
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structural elucidation of a unique binding mode by an intact alphavirus human IgG molecule to a quaternary epitope - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15254BF88AF3A983054BF800648A20D8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365132/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Structural elucidation of a unique binding mode by an intact alphavirus human IgG molecule to a quaternary epitope">
<meta name="citation_author" content="Abhishek Bandyopadhyay">
<meta name="citation_author_institution" content="Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA">
<meta name="citation_author" content="Lauren E Williamson">
<meta name="citation_author_institution" content="The Vanderbilt Center for Antibody Therapeutics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author_institution" content="Department of Pediatrics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author" content="Devika Sirohi">
<meta name="citation_author_institution" content="Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA">
<meta name="citation_author" content="Kevin Bailey">
<meta name="citation_author_institution" content="Institute for Antiviral Research, Utah State University, Logan, UT 84335 USA">
<meta name="citation_author" content="Theron Gilliland, Jr">
<meta name="citation_author_institution" content="The Center for Vaccine Research, University of Pittsburgh, Pittsburgh, PA 165261 USA">
<meta name="citation_author" content="Thomas Klose">
<meta name="citation_author_institution" content="Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA">
<meta name="citation_author" content="Geeta Buda">
<meta name="citation_author_institution" content="Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA">
<meta name="citation_author" content="Andrew Trivette">
<meta name="citation_author_institution" content="The Vanderbilt Center for Antibody Therapeutics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author" content="Chengqun Sun">
<meta name="citation_author_institution" content="The Center for Vaccine Research, University of Pittsburgh, Pittsburgh, PA 165261 USA">
<meta name="citation_author_institution" content="Department of Immunology, University of Pittsburgh, Pittsburgh, PA 165261 USA">
<meta name="citation_author" content="Justin G Julander">
<meta name="citation_author_institution" content="Institute for Antiviral Research, Utah State University, Logan, UT 84335 USA">
<meta name="citation_author" content="William B Klimstra">
<meta name="citation_author_institution" content="The Center for Vaccine Research, University of Pittsburgh, Pittsburgh, PA 165261 USA">
<meta name="citation_author_institution" content="Department of Immunology, University of Pittsburgh, Pittsburgh, PA 165261 USA">
<meta name="citation_author" content="James E Crowe, Jr">
<meta name="citation_author_institution" content="The Vanderbilt Center for Antibody Therapeutics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author_institution" content="Department of Pediatrics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author_institution" content="Department of Pathology, Microbiology and Immunology, Vanderbilt University, Nashville, TN 37232 USA">
<meta name="citation_author" content="Richard J Kuhn">
<meta name="citation_author_institution" content="Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA">
<meta name="citation_author_institution" content="Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, IN 47907 USA">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7716">
<meta name="citation_doi" content="10.1038/s41467-025-60505-x">
<meta name="citation_pmid" content="40830099">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365132/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365132/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365132/pdf/41467_2025_Article_60505.pdf">
<meta name="description" content="Eastern equine encephalitis virus (EEEV) is a mosquito-transmitted alphavirus that can cause severe encephalitis in humans and horses with a high case fatality rate. There are no licensed EEEV vaccines or therapeutics for human use, warranting the ...">
<meta name="og:title" content="Structural elucidation of a unique binding mode by an intact alphavirus human IgG molecule to a quaternary epitope">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Eastern equine encephalitis virus (EEEV) is a mosquito-transmitted alphavirus that can cause severe encephalitis in humans and horses with a high case fatality rate. There are no licensed EEEV vaccines or therapeutics for human use, warranting the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365132/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365132">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-60505-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_60505.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365132%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365132/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365132/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365132/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7716. doi: <a href="https://doi.org/10.1038/s41467-025-60505-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-60505-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structural elucidation of a unique binding mode by an intact alphavirus human IgG molecule to a quaternary epitope</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bandyopadhyay%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Abhishek Bandyopadhyay</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Abhishek Bandyopadhyay</span></h3>
<div class="p">
<sup>1</sup>Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bandyopadhyay%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abhishek Bandyopadhyay</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williamson%20LE%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lauren E Williamson</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lauren E Williamson</span></h3>
<div class="p">
<sup>2</sup>The Vanderbilt Center for Antibody Therapeutics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">
<sup>3</sup>Department of Pediatrics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williamson%20LE%22%5BAuthor%5D" class="usa-link"><span class="name western">Lauren E Williamson</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sirohi%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Devika Sirohi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Devika Sirohi</span></h3>
<div class="p">
<sup>1</sup>Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sirohi%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Devika Sirohi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bailey%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kevin Bailey</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kevin Bailey</span></h3>
<div class="p">
<sup>4</sup>Institute for Antiviral Research, Utah State University, Logan, UT 84335 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bailey%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kevin Bailey</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gilliland%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Theron Gilliland Jr</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Theron Gilliland Jr</span></h3>
<div class="p">
<sup>5</sup>The Center for Vaccine Research, University of Pittsburgh, Pittsburgh, PA 165261 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gilliland%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Theron Gilliland Jr</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klose%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Thomas Klose</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Thomas Klose</span></h3>
<div class="p">
<sup>1</sup>Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klose%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Klose</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buda%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Geeta Buda</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Geeta Buda</span></h3>
<div class="p">
<sup>1</sup>Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buda%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Geeta Buda</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trivette%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Andrew Trivette</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Andrew Trivette</span></h3>
<div class="p">
<sup>2</sup>The Vanderbilt Center for Antibody Therapeutics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trivette%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew Trivette</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Chengqun Sun</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Chengqun Sun</span></h3>
<div class="p">
<sup>5</sup>The Center for Vaccine Research, University of Pittsburgh, Pittsburgh, PA 165261 USA </div>
<div class="p">
<sup>6</sup>Department of Immunology, University of Pittsburgh, Pittsburgh, PA 165261 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chengqun Sun</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Julander%20JG%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Justin G Julander</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Justin G Julander</span></h3>
<div class="p">
<sup>4</sup>Institute for Antiviral Research, Utah State University, Logan, UT 84335 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Julander%20JG%22%5BAuthor%5D" class="usa-link"><span class="name western">Justin G Julander</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klimstra%20WB%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">William B Klimstra</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">William B Klimstra</span></h3>
<div class="p">
<sup>5</sup>The Center for Vaccine Research, University of Pittsburgh, Pittsburgh, PA 165261 USA </div>
<div class="p">
<sup>6</sup>Department of Immunology, University of Pittsburgh, Pittsburgh, PA 165261 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klimstra%20WB%22%5BAuthor%5D" class="usa-link"><span class="name western">William B Klimstra</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crowe%20JE%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">James E Crowe Jr</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">James E Crowe Jr</span></h3>
<div class="p">
<sup>2</sup>The Vanderbilt Center for Antibody Therapeutics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">
<sup>3</sup>Department of Pediatrics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">
<sup>7</sup>Department of Pathology, Microbiology and Immunology, Vanderbilt University, Nashville, TN 37232 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crowe%20JE%22%5BAuthor%5D" class="usa-link"><span class="name western">James E Crowe Jr</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>7,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuhn%20RJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Richard J Kuhn</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Richard J Kuhn</span></h3>
<div class="p">
<sup>1</sup>Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA </div>
<div class="p">
<sup>8</sup>Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, IN 47907 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuhn%20RJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard J Kuhn</span></a>
</div>
</div>
<sup>1,</sup><sup>8,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biological Sciences, Purdue University, West Lafayette, IN 47907 USA </div>
<div id="Aff2">
<sup>2</sup>The Vanderbilt Center for Antibody Therapeutics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div id="Aff3">
<sup>3</sup>Department of Pediatrics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div id="Aff4">
<sup>4</sup>Institute for Antiviral Research, Utah State University, Logan, UT 84335 USA </div>
<div id="Aff5">
<sup>5</sup>The Center for Vaccine Research, University of Pittsburgh, Pittsburgh, PA 165261 USA </div>
<div id="Aff6">
<sup>6</sup>Department of Immunology, University of Pittsburgh, Pittsburgh, PA 165261 USA </div>
<div id="Aff7">
<sup>7</sup>Department of Pathology, Microbiology and Immunology, Vanderbilt University, Nashville, TN 37232 USA </div>
<div id="Aff8">
<sup>8</sup>Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, IN 47907 USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Apr 8; Accepted 2025 May 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365132Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830099/" class="usa-link">40830099</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Eastern equine encephalitis virus (EEEV) is a mosquito-transmitted alphavirus that can cause severe encephalitis in humans and horses with a high case fatality rate. There are no licensed EEEV vaccines or therapeutics for human use, warranting the need to better understand the human immune response against EEEV. Here we present a cryo-EM reconstruction of the chimeric virus, Sindbis (SINV)/EEEV, in complex with a potently neutralizing and efficacious intact human IgG1 antibody in a mouse model of infection and disease. This antibody requires bivalency to recognize a quaternary epitope on the E2 glycoprotein and cross-links two virus spikes across the icosahedral two-fold axis through a unique binding mode. Kinetic analysis of the binding interaction provides insights into this distinguishing feature. Mechanistically, the antibody inhibits viral entry into cells through blockade of receptor binding and early fusion events but does not block egress, thereby, exclusively targeting an epitope found on intact virions. The discovery of the quaternary epitope and unique binding mode recognized by this antibody together advance our understanding of the complexity of antibody-antigen interactions and can aid in vaccine design to elicit recognition of distinct epitopes of clinically relevant alphaviruses.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cryoelectron microscopy, Viral infection</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Structural analyses of antibody-virus complexes offer critical insights into immune recognition mechanisms. In this report, we present how a patient-derived IgG recognizes a quaternary epitope on EEEV particles through strictly bivalent interactions.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The humoral response to viral infections targets a wide range of epitopes on the virion, including quaternary epitopes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>â€“<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Antibody-antigen complexes help elucidate the structural basis for recognition of diverse epitopes by neutralizing antibodies. However, most of these studies used antibody Fab fragments instead of IgG, which limits our understanding of mechanism of action to monovalent antibody-antigen interactions that may inaccurately describe the occupancy of bound sites on virion particles<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>â€“<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. A more limited set of studies investigated intact neutralizing antibodies in complex with virion particles through structural analyses<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. To further our understanding of antibody-antigen interactions, we describe the structural and molecular analyses of a potently neutralizing human monoclonal antibody (mAb) designated EEEV-373 as a bivalent IgG1 molecule in complex with the chimeric virus Sindbis (SINV)/Eastern equine encephalitis virus (EEEV).</p>
<p id="Par4">The encephalitic alphaviruses are mosquito-transmitted viruses and include EEEV, Western (WEEV), and Venezuelan (VEEV) equine encephalitis viruses. EEEV is the most virulent amongst the encephalitic alphaviruses, with a reported human case fatality rate of ~30% and neurological sequelae in ~90% of survivors<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>â€“<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. In the United States, on average 11 human cases of encephalitic disease caused by EEEV are reported each year (<a href="https://www.cdc.gov/easternequineencephalitis/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cdc.gov/easternequineencephalitis/</a>). However, a spike in cases was observed in 2019, in which there were 38 reported human cases and 19 deaths across the United States due to EEEV. Although reported human case numbers decreased in the following years, 19 cases of EEEV neuroinvasive disease were reported in 2024, highlighting the concerns about the potential for larger future EEEV outbreaks. Unfortunately, there are no FDA-approved vaccines or antiviral treatments against EEEV available for human use. In addition, the encephalitic alphaviruses can be aerosolized for use as a potential bioweapon<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Given these significant concerns coupled with the lack of prevention or treatment options (vaccines or antiviral drugs) for humans, EEEV is classified as a NIAID Category B priority pathogen and an USDA/CDC Select Agent.</p>
<p id="Par5">Alphaviruses are enveloped viruses with a positive-sense single-stranded ~11.7â€‰kb RNA genome that encodes both the non-structural and structural proteins<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>â€“<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The alphavirus structural polyprotein encodes five to six structural proteins: capsid, E3, E2, 6K, E1, and TF (transframe). An alphavirus particle consists of a nucleocapsid core (NC) and glycoprotein layer. The NC consists of the capsid protein encapsulating the RNA genome with <em>T</em>â€‰=â€‰4 quasi-icosahedral symmetry and is comprised of 12 pentamers or 30 hexamers around the icosahedral five-fold (i5) or two-fold (i2) vertices, respectively. The NC is further encapsulated by the icosahedral glycoprotein layer consisting of the E2 and E1 single-pass transmembrane glycoproteins embedded in a host-derived lipid bilayer. The E2 and E1 glycoproteins form trimers of heterodimers arranged as 80 trimeric spikes on the surface of an alphavirus particle, with 20 spikes at the icosahedral three-fold (i3) and 60 spikes at the quasi-three-fold (q3) axes<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par6">The E2 glycoprotein forms the outermost layer of the viral particle and occludes most of the underlying E1 protein<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The E2 glycoprotein engages cell-surface receptors and attachment factors, while the E1 glycoprotein lies tangential to the viral membrane and is responsible for endosomal viral fusion<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>â€“<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. The E2 glycoprotein consists of three immunoglobulin-like domains (A, B, and C), of which a â€œcleftâ€ formed between domains A and B is involved in receptor interactions<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>â€“<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. In the past few years, several bona fide entry receptors were identified for alphaviruses through CRISPR/Cas9-based genetic screening approaches. Different receptors were identified for the arthritogenic (<em>i.e</em>., Mxra8<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>) and encephalitic (<em>i.e</em>., VEEV: LDLRAD3<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>) alphaviruses, which may contribute to differences in pathogenesis<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. More recently, members of the low-density lipoprotein receptor (LDLR) family, very low-density lipoprotein receptor (VLDLR)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, apolipoprotein E receptor 2 (ApoER2)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, and LDLR<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> were identified as receptors for EEEV. Domain B also helps prevent premature exposure of the E1 fusion loop<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p>
<p id="Par7">Previous studies have described neutralizing anti-alphavirus antibodies that target different epitopes on the E2 glycoprotein (<em>i.e</em>., domains A, B, and A/B) and can inhibit either cell attachment, receptor binding, viral fusion, virus egress, or a combination of these steps<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>â€“<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. On the other hand, several anti-E1 glycoprotein neutralizing antibodies have been described and appear to primarily block viral egress<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Nevertheless, there are a limited number of studies describing antibodies that specifically target quaternary epitopes on alphavirus particles.</p>
<p id="Par8">The structural analyses described here reveal a quaternary epitope on the E2 glycoprotein targeted by a human monoclonal antibody (mAb) designated EEEV-373. Potent efficacy of EEEV-373 was confirmed in cell culture and a mouse model of EEEV infection. This mAb distinctively binds by cross-linking two q3-spikes diagonally across the i2 axis governed by steric and distance constraints. Kinetic analysis of the antibody-antigen interaction provided mechanistic insights into the strict requirement for bivalent heterodimer binding. These data expand the scope of structure-guided immunogen design for clinically relevant alphaviruses. Furthermore, this study advances our growing knowledge on the complexity of antibody-antigen interactions for therapeutic antibody development.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Identification and characterization of the potently neutralizing and efficacious human mAb EEEV-373</h3>
<p id="Par9">We isolated the human mAb EEEV-373 (IgG1, Îº) from the B cells of an individual with prior documented natural EEEV infection using human B cell hybridoma cell technology. EEEV-373 was selected based on binding reactivity in ELISA to the chimeric virus Sindbis (SINV)/EEEV, EEEV virus-like particles (VLPs), and recombinant EEEV heterodimer (E2/E1) protein. The chimeric virus SINV/EEEV encodes the nonstructural proteins of SINV and the structural proteins of EEEV, which enables safe use of virus particles in biosafety level (BSL)âˆ’2 conditions<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p>
<p id="Par10">To further characterize EEEV-373, we assessed the neutralization activity of hybridoma-cell-derived EEEV-373 IgG against wild-type EEEV (strain FL93-939) under BSL-3 conditions through a plaque reduction neutralization assay (Fig.Â <a href="#Fig1" class="usa-link">1A</a>). EEEV-373 exhibited potent neutralization activity against EEEV, with a half-maximal inhibitory concentration (IC<sub>50</sub>) value of &lt;37 pM (11â€‰ng/mL). A minor residual fraction of non-neutralized virus (2%) was present at the highest concentration tested of 75â€‰nM (22.5â€‰Âµg/mL), indicating incomplete neutralization of EEEV (Fig.Â <a href="#Fig1" class="usa-link">1B</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. EEEVâˆ’373 potently neutralizes and protects against wild-type EEEV.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365132_41467_2025_60505_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/14c6/12365132/6728a90b985e/41467_2025_60505_Fig1_HTML.jpg" loading="lazy" id="d33e605" height="1028" width="749" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Neutralization curves of EEEV-373 (hybridoma-derived; green squares) as bivalent IgG1 molecules against wild-type EEEV strain FL93-939 in a plaque reduction neutralization assay. The previously published irrelevant isotype mAb (rDENV-2D22; black squares) or mouse anti-EEEV ascites fluid (ATCC (+)ve; purple squares) were included as negative or positive controls, respectively, for comparative purposes<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Each curve displays mAb concentration (nM) or ascites fluid dilution on the x-axis and percent relative viral infectivity on the y-axis. Data represent meanâ€‰Â±â€‰SD of technical triplicates of one biological replicate. <strong>B</strong> Percent of EEEV strain FL93-939 residual fraction (y-axis) present at the highest concentration tested (75â€‰nM) of mAb (EEEV-373 [2%], rDENV-2D22 [99%], or ascites fluid [0%]). Data represent the median values of technical triplicates of one biological replicate. <strong>C</strong>â€“<strong>E</strong> EEEV (strain FL93-939; 10<sup>3.3</sup> CCID<sub>50</sub>) was inoculated subcutaneously (s.c.) into C57BL/6 mice. After 24â€‰h, EEEV-373 (green) as bivalent IgG1 molecules was administered intraperitoneally (i.p.) at 10â€‰mg/kg (200â€‰Âµg/mouse; <em>n</em>â€‰=â€‰10). The previously published negative control treatment group, rDENV-2D22 (black; <em>n</em>â€‰=â€‰10), or the mock-inoculated normal controls (grey; <em>n</em>â€‰=â€‰5)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>Â also were included for comparative purposes. <strong>C</strong> EEEV-373 (green circles) or mock-inoculated normal control (grey circles) groups mediated 100% survival compared to the negative control mAb (rDENV-2D22; black circles) treatment group. Survival curves were compared using the log-rank testÂ with Bonferroni multiple comparison correction (*<em>Ïadj</em>â€‰&lt;â€‰0.002). <strong>D</strong> Percent body weight change of each mouse from the EEEV-373 (green circles), rDENV-2D22 (black circles), or mock-inoculated normal control (grey circles) groups over the course of 18 days post-inoculation. <strong>E</strong> Serum was collected 3 days post-inoculation from each mouse of the treatment groups (EEEV-373 [green circles], rDENV-2D22 [black circles], or mock-inoculated normal controls [grey circles]) for quantification of virus titer (log<sub>10</sub> CCID<sub>50</sub>/mL; y-axis) in an infectious cell culture assay. Data represent meanâ€‰Â±â€‰SD. The EEEV-373 and mock-inoculated normal control groups were compared to the rDENV-2D22 negative control treatment group using an ordinary one-way ANOVA with Dunnettâ€™s multiple comparisons test (*<em>Ï</em>â€‰&lt;â€‰0.05).</p></figcaption></figure><p id="Par11">Next, we assessed the in vivo efficacy of EEEV-373 IgG against EEEV (strain FL93â€“939) infection in a subcutaneous (s.c.) treatment challenge model (Figs.Â <a href="#Fig1" class="usa-link">1</a>Câ€“<a href="#Fig1" class="usa-link">E</a>). In this model, EEEV was inoculated s.c. (10<sup>3.3</sup> CCID<sub>50</sub>) and antibody (IgG) was administered intraperitonially (i.p.) 24â€‰h after virus inoculation. Mice were followed for a total of 21 days for survival, and viremia was assessed 3 days after virus inoculation using a plaque assay. In the EEEV-373 treatment group, 100% of the animals survived, compared to a 10% survival rate in theÂ previously published negative control group treated with the recombinantly-derived dengue virus-specific mAb rDENV-2D22 (IgG1, Î»)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> (Fig.Â <a href="#Fig1" class="usa-link">1C</a>). The body weight of the animals corresponded with survival, in which body weight was relatively stable for the EEEV-373 treatment group compared to an observed loss in body weight for the rDENV-2D22 treatment group (Fig.Â <a href="#Fig1" class="usa-link">1D</a>). We also observed a significant reduction in virus titer in the serum to the limit of detection of the assay for animals treated with EEEV-373, compared to those that received rDENV-2D22 (Fig.Â <a href="#Fig1" class="usa-link">1E</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">EEEV-373 requires bivalent interactions for binding and neutralization of SINV/EEEV</h3>
<p id="Par12">To further characterize EEEV-373, we addressed whether bivalency is required for binding in ELISA (Fig.Â <a href="#Fig2" class="usa-link">2A</a>). EEEV-373 as either bivalent IgG or F(abâ€²)2 molecules bound to EEEV VLPs and recombinant EEEV E2/E1 protein. However, binding to recombinant monomeric EEEV E2 or E1 protein was not observed at the concentrations tested, which suggests EEEV-373 preferentially binds to a complex quaternary epitope. Maximal optical density signal was observed for binding to EEEV VLPs compared to EEEV E2/E1 protein, which suggested that there may be an avidity effect conferred by bivalent binding to virus particles. In contrast, EEEV-373 tested as monovalent Fab molecules did not appreciably bind to EEEV VLPs, recombinant EEEV E2/E1, E2, or E1 proteins at the concentrations tested. The lack of binding observed indicates EEEV-373 requires bivalent interactions for stable binding. To further assess the dependence of EEEV-373 on bivalency, we tested the neutralization potency of EEEV-373 against SINV/EEEV (Fig.Â <a href="#Fig2" class="usa-link">2C</a>). Consistent with the binding profile of EEEV-373, the IgG and F(abâ€²)2 formats of the antibody neutralized SINV/EEEV with similar potencies, although aâ€‰~3-fold reduction in IC<sub>50</sub> value was observed for the F(abâ€²)2 format (Fig.Â <a href="#Fig2" class="usa-link">2D</a>). In contrast, EEEV-373 Fab molecules failed to neutralize SINV/EEEV.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. EEEV-373 binds and neutralizes EEEV in a bivalent manner.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365132_41467_2025_60505_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/14c6/12365132/8829d246b2d5/41467_2025_60505_Fig2_HTML.jpg" loading="lazy" id="d33e711" height="790" width="799" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Representative binding curves of EEEV-373 as bivalent IgG1 (left), F(abÊ¹)2 (middle), or monovalent Fab (right) molecules to EEEV virus-like particles (VLPs; green circles), Western equine encephalitis virus (WEEV) VLPs (black circles), recombinant EEEV E2/E1 (maroon circles), monomeric E2 (blue circles) or E1 (light purple circles) protein. Each curve displays mAb concentration (nM) on the x-axis and optical density at 450â€‰nm on the y-axis. Data represent meanâ€‰Â±â€‰SD of technical triplicates and are representative of two independent experiments. <strong>B</strong> Representative binding kinetic curves (black lines) of EEEV-373 as bivalent IgG1 (left), F(abÊ¹)2 (middle), or monovalent Fab (right) molecules to EEEV VLPs using quartz crystal microbalance biosensors (Attana). EEEV VLPs were amine-coupled to chips and quenched with ethanolamine. EEEV-373 then was incubated with the chip for 300â€‰s at the labeled concentration (0.33 to 35â€‰nM) to determine the association rates (<em>k</em><sub><em>on</em></sub>) followed by incubation with buffer to determine the dissociation rates (<em>k</em><sub><em>off</em></sub>). Time (in seconds) is on the x-axis and frequency (in hertz (Hz)) is on the y-axis. For analysis, a bivalent (IgG1 and F(abÊ¹)2) binding model was used, and corresponding fitted curves are shown by the purple lines. <strong>C</strong> Representative neutralization curves of EEEV-373 (green symbols) as bivalent IgG1 (closed circles), F(abâ€²)2 (open circles), or monovalent Fab (open squares) molecules against the chimeric virus, SINV/EEEV. A negative control bivalent IgG1 mAb (black circles) also was included. Each curve displays mAb concentration (nM) on the x-axis and percent relative viral infectivity on the y-axis. Data represent meanâ€‰Â±â€‰SD of technical triplicates and are representative of two independent experiments. <strong>D</strong> Summary table of the half-maximal effective (EC<sub>50</sub>) concentration values for binding of EEEV-373 as bivalent IgG1 (left column), F(abâ€²)2 (middle column), or monovalent Fab (right column) molecules to EEEV VLPs or recombinant EEEV E2/E1 protein, and half-maximal inhibitory (IC<sub>50</sub>) values for neutralization activity against SINV/EEEV. <strong>E</strong> Summary table of kinetic analyses (<em>K</em><sub><em>D</em></sub>, <em>k</em><sub><em>on</em></sub>, <em>k</em><sub><em>off</em></sub>) for EEEV-373 as bivalent IgG1 (left column), F(abâ€²)2 (middle column), or monovalent Fab (right column) molecules.</p></figcaption></figure><p id="Par13">We next assessed the binding kinetics of EEEV-373 IgG, F(abâ€²)2, or Fab molecules with EEEV VLPs using quartz crystal microbalance biosensors (Figs.Â <a href="#Fig2" class="usa-link">2</a>B, <a href="#Fig2" class="usa-link">E</a>). In this assay, we also observed binding of EEEV-373 to EEEV VLPs in a bivalent manner. For binding of EEEV-373 IgG or F(abâ€²)2 molecules to EEEV VLPs, we observed a biphasic association pattern in which there was a fast initial binding interaction (10<sup>5</sup>â€‰Ms<sup>âˆ’1</sup>) followed by a slower second phase interaction (10<sup>âˆ’1</sup>â€‰Ms<sup>âˆ’1</sup>). The <em>K</em><sub><em>D</em></sub>, <em>k</em><sub><em>on</em></sub>, and <em>k</em><sub><em>off</em></sub> values for both IgG and F(abâ€²)2 molecules are similar (Fig.Â <a href="#Fig2" class="usa-link">2E</a>), suggesting that the Fc region of the antibody does not play a major role in the binding interaction. Further supporting this, we also observed a similar association profile for binding of bivalent EEEV-373 IgG and F(abâ€²)2 molecules to EEEV VLPs using biolayer interferometry (Fig.Â <a href="#MOESM1" class="usa-link">S1</a>). In contrast to bivalent molecules, a weak association was observed for Fab molecule binding to EEEV VLPs. This weak association corresponds with the variable detectable signal observed at the highest concentrations of Fab molecules for binding to EEEV VLPs as tested via ELISA (Fig.Â <a href="#Fig2" class="usa-link">2A</a>). However, in comparison, binding of EEEV-373 Fab molecules to EEEV VLPs was not detected using biolayer interferometry (Fig.Â <a href="#MOESM1" class="usa-link">S1</a>). In this case, the interaction was comparable to that of a negative control IgG. A potential explanation for the observed kinetic behavior of Fab molecules may be the inability to form stable antibody-antigen complexes. The biphasic association pattern of bivalent IgG or F(abâ€²)2 molecules indicates that complex formation involves more than one step. The initial interaction may be energetically weak, as reflected by the poor binding of Fab molecules. The avidity effects of bivalent molecules may compensate for this weak interaction by engagement of two binding sites, shifting the equilibrium towards formation of a high affinity complex.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Cryo-EM reconstruction of EEEV-373 as an IgG1 molecule in complex with SINV/EEEV particles</h3>
<p id="Par14">To elucidate the structural determinants of bivalent binding and neutralization, we obtained a three-dimensional cryo-EM reconstruction of SINV/EEEV particles in complex with EEEV-373 as an IgG1 molecule. The reconstruction was determined to ~4.6â€‰Ã… with imposed icosahedral symmetry (Fig.Â <a href="#Fig3" class="usa-link">3A</a>). Although the density of the IgG was weak compared to the glycoprotein densities, a single EEEV-373 IgG molecule connecting two q3 spikes diagonally across the i2 axis was visible (Figs.Â <a href="#Fig3" class="usa-link">3</a>A, <a href="#Fig3" class="usa-link">B</a>). An even weaker and broken density around the i5 axis was also observed. This density likely reflects low IgG occupancy around the i5 axis, suggesting that the predominant mode of binding cross-links the q3 spikes across the i2 axis (Fig.Â <a href="#MOESM1" class="usa-link">S6D</a>). To improve the density of the antibody and fit atomic models, localized reconstruction combined with focused classification and refinement of the two q3 spikes bound to EEEV-373 were performed, providing a global resolution estimate of ~3.8â€‰Ã… (Figs.Â <a href="#Fig3" class="usa-link">3</a>C, <a href="#Fig3" class="usa-link">D</a>, and <a href="#MOESM1" class="usa-link">S7</a>). The overall density of the EEEV-373 IgG molecule at the i2 axis improved, with the local resolution map estimation showing the variable domains resolved significantly better than the constant domains (Figs.Â <a href="#Fig3" class="usa-link">3E, F</a>). The local resolution map estimates the resolution of the variable domains to be ~5â€‰Ã…, whereas that of the constant domains was greater than ~9â€‰Ã…. We did not observe any density for the hinge or Fc region of the IgG molecule.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Cryo-EM reconstruction of SINV/EEEV in complex with a potently neutralizing intact IgG.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365132_41467_2025_60505_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/14c6/12365132/2e49c8afb16a/41467_2025_60505_Fig3_HTML.jpg" loading="lazy" id="d33e840" height="434" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Cryo-EM reconstruction of SINV/EEEV particles in complex with EEEV-373 IgG was determined by single particle averaging with icosahedral symmetry to 4.6â€‰Ã… resolution. The black triangle represents the asymmetric unit, with the symmetry axes depicted by the red pentagon (5-fold axis), triangle (3-fold axis), and circle (2-fold [i2] axis) symbols. The quasi-three-fold (q3) spikes are represented by the blue triangles. The scale bar (in Ã…) represents the radial distance from the center of the virus. <strong>B</strong> A close-up view of the i2 axis displays a single EEEV-373 IgG molecule diagonally spanning across the axis to connect two q3 spikes. The q3 and icosahedral three-fold (i3) spikes are labeled in dark blue text accordingly. In comparison to the SINV/EEEV particle density, the density of the IgG molecule is relatively weak, and to help elucidate the complex, the two arms of the IgG molecule are indicated by the dark blue dotted lines. <strong>C</strong>, <strong>D</strong> Localized reconstruction, focused classification, and refinement with C1 symmetry of the two q3 spikes cross-linked by the EEEV-373 IgG molecule were performed to improve the resolution of the reconstruction. A top (<strong>C</strong>) or side (<strong>D</strong>) view of the improved reconstruction from the i2 axis perspective are illustrated. The q3 spikes, constant and variable domains of EEEV-373 are indicated accordingly. The map is colored according to the coloring of the fitted model in ChimeraX. The grey regions fall slightly outside of the fitted model, while the relative protein domains of E1 (domain I: orange, domain II: light pink, domain III: light orange), E2 (domain A: cyan, domain B: teal, domain C: aqua, Î²-ribbon connector: purple), EEEV-373 heavy chain (green) and light chain (yellow) are colored accordingly. <strong>E</strong>, <strong>F</strong> Local resolution map of the volume as shown in <strong>C</strong> and <strong>D</strong>. <strong>E</strong>, <strong>F</strong> correspond to the viewing directions in <strong>C</strong> and <strong>D</strong>, respectively. The scale bar (in Ã…) represents the resolution gradient.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Identification of a quaternary epitope targeted by EEEV-373 on the E2/E1 glycoproteins</h3>
<p id="Par15">From the cryo-EM reconstruction, we observed that EEEV-373 primarily binds the E2 glycoprotein of SINV/EEEV particles (Fig.Â <a href="#Fig4" class="usa-link">4</a> and <a href="#MOESM1" class="usa-link">S2</a>). The total area buried by EEEV-373 on the E2/E1 glycoprotein is 750.5â€‰Ã…<sup>2</sup>, with the heavy or light chain burying 402â€‰Ã…<sup>2</sup> or 348.5â€‰Ã…<sup>2</sup>, respectively. To determine the binding footprint of EEEV-373 on the E2/E1 glycoproteins, residues within 6â€‰Ã… of the backbones of the fitted atomic model were identified. The binding footprint of EEEV-373 constitutes several residues on the E2 glycoprotein and one residue (K61) within domain II of the E1 glycoprotein. Except for K61, all remaining residues are not conserved amongst other alphaviruses (Fig.Â <a href="#MOESM1" class="usa-link">S4</a>). This finding supports the specificity of EEEV-373 to EEEV, as we did not observe detectable binding to Western equine encephalitis virus (WEEV) VLPs by ELISA at the concentrations tested (Fig.Â <a href="#Fig2" class="usa-link">2A</a>). Based on the binding footprint, the light chain of EEEV-373 binds surface exposed residues within domain B of the E2 glycoprotein, and the heavy chain binds the E2 Î²-ribbon connector and residue K61 of the E1 glycoprotein (Figs.Â <a href="#Fig4" class="usa-link">4</a>C, <a href="#Fig5" class="usa-link">5</a>, and <a href="#MOESM1" class="usa-link">S2</a>). The heavy chain complementarity-determining regions (CDRs) clamp onto the Î²-ribbon connector (residues 230 to 248), which is spatially adjacent to domain B and includes the â€˜arch 2â€™ region<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> (Figs.Â <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, and <a href="#MOESM1" class="usa-link">S2</a>). The Î²-ribbon connector essentially is a long linker connecting domains A to B and B to C, with little secondary structure. Natively, the â€˜arch 2â€™ region of the Î²-ribbon connector forms a narrow cleft-like structure and makes significant interactions with the underlying E1 protein (Figs.Â <a href="#Fig5" class="usa-link">5A, B</a>). Thus, it is conceivable that in the absence of the E1 protein, the Î²-ribbon connector might be flexible, and as a result the cleft of the â€˜arch 2â€™ region would become unstable. EEEV-373 binding shows that CDRH1 and CDRH2 clamp onto the â€˜arch 2â€™ region (Figs.Â <a href="#Fig5" class="usa-link">5C, D</a>), with CDRH3 inserting into the cleft (Figs.Â <a href="#Fig5" class="usa-link">5E, F</a>). The heavy chain interaction with residue K61 of the E1 glycoprotein is largely due to its long side chain, as no other residue in the vicinity of K61 contributes to the binding footprint of EEEV-373. The interactions observed for the heavy chain CDRs with the E2 glycoprotein, suggest that EEEV-373 binding critically depends on the conformational nature of the heterodimer through the interaction of the E2 Î²-ribbon connector with the E1 protein, consistent with the ELISA binding data (Fig.Â <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. EEEV-373 epitope binding footprint on SINV/EEEV particles.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365132_41467_2025_60505_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/14c6/12365132/a52f15497121/41467_2025_60505_Fig4_HTML.jpg" loading="lazy" id="d33e916" height="286" width="700" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> RIVEM (Radial Interpretation of Viral Electron Density Maps) representation of the EEEV-373 IgG binding epitope on the viral surface. Residues within 6â€‰Ã… of the complementarity-determining regions (CDRs) of EEEV-373 IgG are colored yellow and constitute the epitope binding footprint on the viral surface. The scale bar represents the radial distance in Ã… from the center of the virus. <strong>B</strong> Cartoon representation of one arm of the EEEV-373 IgG molecule in complex with a unit of the SINV/EEEV E2/E1 heterodimer. The different protein domains of E1 (domain I: orange, domain II: light pink, domain III: tan), E2 (domain A: cyan, domain B: teal, domain C: aqua, Î²-ribbon connector: light purple), EEEV-373 heavy chain (green; VH: variable domain, CH1: constant domain), and EEEV-373 light chain (yellow; VL: variable domain, CL: constant domain) are labeled accordingly. <strong>C</strong> A close-up view of the interface between EEEV-373 and the SINV/EEEV E2/E1 heterodimer. The heavy (green) or light (yellow) chain CDRs 1-3 are indicated as HCDR1, HCDR2, HCDR3 or LCDR1, LCDR2, LCDR3, respectively. The E2 protein domain A (cyan), domain B (teal), Î²-ribbon connector (light purple), and E1 protein domain II (light pink) are labeled accordingly.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. EEEV-373 binds a conformational epitope on the SINV/EEEV E2 protein.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365132_41467_2025_60505_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/14c6/12365132/57da34c4cc1f/41467_2025_60505_Fig5_HTML.jpg" loading="lazy" id="d33e946" height="972" width="674" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> A close-up view of the E2 protein Î²-ribbon connector (cyan) and the E1 protein domain II (orange) interface, further highlighted by the dotted purple circle. The residues comprising the binding footprint of EEEV-373 are colored red. <strong>B</strong> A side view (rotated ~90Â°) of the E2/E1 interface is shown to highlight the significant interaction of the underlying E1 protein (orange) for the proper conformation and stabilization of the E2 protein Î²-ribbon connector (cyan). Residues involved in this interaction are indicated by the cyan or red spheres. The red spheres indicate the E2 residues that constitute the EEEV-373 epitope binding footprint. A detailed cartoon representation view (<strong>C</strong>) and a close-up, side view (rotated ~90Â°; <strong>D</strong>) of the interactions between the heavy chain complementarity-determining region loops (HCDR1, HCDR2, and HCDR3) of EEEV-373 (green) and the â€˜arch 2â€™ cleft of the E2 protein Î²-ribbon connector (cyan). Specific view of the heavy chain CDR3 loop (HCDR3) of EEEV-373 (green) inserting within the â€˜arch 2â€™ cleft of the E2 protein Î²-ribbon connector (cyan) as shown by the cartoon (<strong>E</strong>) or sphere (<strong>F</strong>) representations. The E2 protein Î²-ribbon connector residues that form the epitope binding footprint of EEEV-373 are indicated by the red spheres. Additional residues on the E2 protein within the immediate vicinity of the footprint are indicated by the cyan spheres. <strong>G</strong> The quality of the model fit to the map in the epitope-paratope region is coloured according to the Q-score. The corresponding color key is shown on the right.</p></figcaption></figure><p id="Par16">To further determine the most functionally relevant interactions in the binding footprint of EEEV-373, we assessed the ability for SINV/EEEV to escape neutralization under antibody selective pressure. In the presence of EEEV-373 at saturating antibody concentrations, we identified escape mutant viruses with a E147K mutation in the E2 protein (Fig.Â <a href="#MOESM1" class="usa-link">S3</a>). This residue corresponds to the Î²-ribbon connector outside of the EEEV-373 binding footprint as determined by cryo-EM (Fig.Â <a href="#MOESM1" class="usa-link">S4</a>). Given the potential flexibility and conformational dependence of the Î²-ribbon connector for binding, the E147K mutation may allosterically alter the conformation of the epitope, ablating the neutralization capabilities of EEEV-373.</p></section><section id="Sec7"><h3 class="pmc_sec_title">EEEV-373 cross-links viral q3 spikes using a preferred binding mode</h3>
<p id="Par17">In the cryo-EM reconstruction of EEEV-373 IgG in complex with SINV/EEEV particles, we predominantly observe one binding orientation with the IgG molecule connecting two q3-spikes across the i2 axis (q3.1â€“q3.3). Based on previous studies, the distance between the CH1 domains (Cys216 [Kabat numbering]) of the arms of an IgG molecule is ~50â€‰Ã…, allowing for intra-virion cross-linking of icosahedral viruses<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. However, since the resolution of the constant domains is ~9â€‰Ã… in our reconstruction, we are unable to accurately determine the position of Cys216, which corresponds to Cys228 for EEEV-373. Instead, we used the last residue on strand G of the CH1 domain<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> (Lys222 in EEEV-373) to determine the distances more accurately between the two arms of the EEEV-373. There are 12 residues that act as a long flexible linker between the last residue on strand G and the cysteine residue that forms the first disulfide bond in the hinge region (Cys222 [Kabat numbering]). The length of this flexible segment when fully stretched is ~48â€‰Ã…. Thus, the distance criteria for the arms of an IgG molecule ranges from ~50 to 100â€‰Ã….</p>
<p id="Par18">In the preferred q3.1â€“q3.3 binding mode of EEEV-373, the distance between Lys222 of the CH1 domains is ~42.5â€‰Ã… (Fig.Â <a href="#Fig6" class="usa-link">6A</a>), which is well within the range (~50â€“100â€‰Ã…) that allows for an IgG molecule to bind two epitopes<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Since the i2 axis of alphaviruses is quasi 6-fold, there are five additional ways EEEV-373 could cross-link viral spikes across the i2 axis (Fig.Â <a href="#Fig6" class="usa-link">6B</a>). To investigate why we observed this preferential q3.1â€“q3.3 mode of binding, we superimposed the atomic model of the E2 glycoprotein bound to an EEEV-373 Fab arm from the q3.1 spike onto the i3 (i3.1 and i3.2) and q3.4 spikes. This fitting shows that in the i3.1â€“i3.2 binding mode, the distance between Lys222 of the CH1 domains of EEEV-373 is ~64â€‰Ã… (Fig.Â <a href="#Fig6" class="usa-link">6C</a>), which may be too distant or require stretching for both arms of the EEEV-373 IgG molecule to simultaneously bind. Additional binding modes lead to severe clashes of the CL domains, such as in the case of cross-linking adjacent i3.1â€“q3.1 spikes (Fig.Â <a href="#Fig6" class="usa-link">6D</a>) or between the q3.3â€“q3.4 spikes (~44.2â€‰Ã…; Fig.Â <a href="#Fig6" class="usa-link">6G</a>). These clashes probably restrain binding of the IgG molecule to the preferred q3.1â€“q3.3 orientation. Interestingly, the q3.1â€“q3.4 binding mode satisfies both the distance criteria (~41â€‰Ã…) and the fitted models do not appear to clash (Fig.Â <a href="#Fig6" class="usa-link">6E</a>). However, the poor resolutions of the CL and the CH1 domains, as estimated from the local resolution maps, suggest these regions are highly flexible. Due to flexibility of the elbow region, movement of the constant domains perpendicular to the pseudo-2-fold axis of the Fab (between the heavy and light chains) can occur<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Therefore, movement of the constant domains could lead to clashes between the two arms of the IgG in the q3.1â€“q3.4 binding mode. In the observed q3.1â€“q3.3 binding mode, this movement does not lead to clashes between the two arms of the IgG molecule. The last binding mode (q3.1â€“i3.2) satisfies the distance criteria (~48.9â€‰Ã…) and does not sterically clash (Fig.Â <a href="#Fig6" class="usa-link">6F</a>), yet we do not observe this mode of binding in the structure. The only difference between q3.1â€“q3.3 and q3.1â€“i3.2 binding modes is the rotation of one Fab arm about 60Â° with respect to the other arm, which may cause tension on the IgG molecule. However, it is not obvious from the structure as to why this mode of binding is not observed. Together, these observations suggest stringent criteria (<em>i.e</em>., distance and angle of binding) are required for EEEV-373 IgG to bind and form intra-virion spike cross-links.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Potential binding modes of EEEV-373 IgG at the icosahedral 2-fold axis of SINV/EEEV particles.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365132_41467_2025_60505_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/14c6/12365132/5d09fe58410e/41467_2025_60505_Fig6_HTML.jpg" loading="lazy" id="d33e1068" height="1158" width="749" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Preferred binding mode orientation of EEEV-373 IgG across the icosahedral 2-fold (i2) axis (q3.1-q3.3). A cartoon representation of the fitted atomic models of the E2 protein (cyan) and EEEV-373 IgG (heavy chain: green, light chain: yellow) at the i2 axis (blue circle) is shown. The E1 protein is omitted for clarity. The four q3 spikes (q3.1-q3.4) and the two i3 spikes (i3.1-i3.2) are labeled in bolded text accordingly. The distance (in Ã…) between the CÎ± atoms of the last residue (Lys222; red spheres) on strain G of the constant domains (CH1) of the two Fab arms of EEEV-373 (42.5â€‰Ã…) is indicated by the dotted line. <strong>B</strong> In addition to the observed binding mode, there are five alternative ways (<strong>C</strong>â€“<strong>G</strong>) EEEV-373 could cross-link viral spikes across the i2 axis as depicted. The distance (in Ã…) between the CÎ± atoms of the last residue (Lys222; red spheres) on strain G of the constant domains (CH1) of the two Fab arms of EEEV-373 are indicated by the dotted lines (<strong>C</strong> [i3.1-i3.2]: 64.0â€‰Ã…; <strong>E</strong> [q3.1-q3.4]: 41.0â€‰Ã…; <strong>F</strong> [q3.1-i3.2]: 48.9â€‰Ã…; <strong>G</strong> [q3.3-q3.4]: 44.2â€‰Ã…). For the i3.1-q3.1 (<strong>D</strong>) or q3.3-q3.4 (<strong>G</strong>) binding modes, steric clashes are observed between the light chain constant domains of EEEV-373 (yellow) as indicated by the black dotted circle (<strong>D</strong>) or the black box (<strong>G</strong>). <strong>E</strong> For the q3.1-q3.4 binding mode, movement of the constant domains as indicated by the purple arrows may lead to steric clashes. <strong>F</strong> For the q3.1-i3.2 binding mode, EEEV-373 may also bind in this orientation. However, a 60Â° rotation of one Fab arm with respect to the other Fab arm may reduce the likelihood of binding. The distance and steric hindrance constraints observed in <strong>C</strong>â€“<strong>G</strong> likely contribute to the observation of EEEV-373 in the preferred binding mode orientation (<strong>A</strong>) with binding of the two Fab arms to the q3.1 and q3.3 spikes across the i2 axis.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">EEEV-373 does not aggregate EEEV VLPs by formation of intra-virion spike cross-links</h3>
<p id="Par19">To functionality validate the observed structural basis of neutralization by EEEV-373, we aimed to elucidate the molecular mechanism of action of EEEV-373 against EEEV. Based on the cryo-EM reconstruction, the observed binding orientation of EEEV-373 displays formation of intra-virion spike cross-links by the IgG molecule. To further corroborate this, we tested the ability of EEEV-373 to aggregate VLPs using dynamic light scattering (DLS) (Fig.Â <a href="#Fig7" class="usa-link">7C</a>). The hydrodynamic diameter of antibody-antigen complexes at different molar ratios can be estimated using DLS. The diameter of the VLP alone was ~70â€‰nm, which is close to the expected value for an alphavirus particle<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. At any of the molar ratios tested, EEEV-373 as IgG or F(abâ€²)2 molecules minimally aggregated VLPs. This finding strongly supports the formation of intra-virion spike cross-links by EEEV-373 binding to two q3 spikes across the i2 axis. At very high molar ratios (&gt;1:100 IgG:VLP), a slight increase in hydrodynamic diameter to ~100â€‰nm is observed. This finding may result from partial binding of EEEV-373 to additional exposed epitopes present on two neighboring virus particles (inter-virion spike cross-links) following complete occupancy of the preferred binding orientation within the same virus particle (intra-virion spike cross-links). Binding to these exposed epitopes may account for the observed weak density around the i5 axis in the icosahedral reconstruction. As expected, we did not detect aggregation of EEEV VLPs by EEEV-373 Fab molecules.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. EEEV-373 inhibits SINV/EEEV entry through receptor blockade and inhibition of early fusion events.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365132_41467_2025_60505_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/14c6/12365132/caf904d5696f/41467_2025_60505_Fig7_HTML.jpg" loading="lazy" id="d33e1135" height="709" width="795" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Competition-binding ELISA to assess the ability of bivalent EEEV-373 (green) or isotype-matched control (black) IgG1 molecules to block ApoER2 (<strong>A</strong>) or VLDLR (<strong>B</strong>) protein receptors from binding to EEEV VLPs at the concentrations indicated (67 to 3â€‰nM; x-axis). The average optical density at 450â€‰nm of the isotype-matched control mAb at each concentration was calculated and used to normalize the percent binding of EEEV-373 to EEEV VLPs (y-axis). Data represent meanâ€‰Â±â€‰SD of technical triplicates of two independent experiments. Data were analyzed at the specified concentrations using multiple unpaired <em>t</em>-tests (***<em>Ï</em>â€‰&lt;â€‰0.000001, **<em>Ï</em>â€‰&lt;â€‰0.0001, *<em>Ï</em>â€‰&lt;â€‰0.001). The exact <em>Ï</em> values are noted for each comparison. <strong>C</strong> Dynamic light scattering (DLS) of EEEV-373 (green) as bivalent IgG1 (top), F(abâ€²)2 (middle), or monovalent Fab (bottom) molecules at different Ab:VLP molar ratios (x-axis). The peak hydrodynamic diameter (nm) is shown on the y-axis. Data represent meanâ€‰Â±â€‰SD of technical duplicates and are representative of two independent experiments. <strong>D</strong> Representative neutralization curves of EEEV-373 (green) as bivalent IgG1 (top), F(abâ€²)2 (middle), or monovalent Fab (bottom) molecules assessed by a post-attachment assay (open circles) or focus reduction neutralization test (FRNT; closed circles). Ab concentration (nM) is on the x-axis and percent relative infectivity is shown on the y-axis. Data represent meanâ€‰Â±â€‰SD of technical duplicates and are representative of two independent experiments. <strong>E</strong> Liposomal fusion assay of DiD-labeled SINV/EEEV particles incubated with liposomes in the presence of EEEV-373 (closed green circles), EEEV-94 (closed cyan circles; positive control), or an isotype-matched control (open black squares; negative control) as bivalent IgG1 (top left), F(abâ€²)2 (top right), or monovalent Fab (bottom left) molecules. Assay controls in which no antibody (closed black circles) or liposomes (closed grey circles) were also included. Data are representative of two independent experiments. <strong>F</strong> An egress inhibition assay was performed to determine the relative RNA copies/ÂµL present in supernatant harvested at either 1â€‰h (left) or 6â€‰h (right) after addition of EEEV-373 (green) or theÂ previously published<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> irrelevant isotype control mAb, rDENV-2D22 (black), as bivalent IgG1 molecules. RNA copies/ÂµL were determined using a standard curve with quantitative EEEV RNA (ATCC). SINV/EEEV RNA levels were compared to theÂ previously published<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> SINV/EEEV only controlÂ group (purple) using an ordinary one-way ANOVA with Dunnettâ€™s multiple comparisons test (ns = not significant).Â The previously published negative control mAb, rDENV-2D22, and SINV/EEEV only control groups were included for comparative purposes<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Data represent meanâ€‰Â±â€‰SD of technical quadruplicates and are representative of two independent experiments.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">EEEV-373 blocks receptor binding to EEEV VLPs</h3>
<p id="Par20">Members of the low-density lipoprotein receptor (LDLR) family, apolipoprotein E receptor 2 (ApoER2)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> and very low-density lipoprotein receptor (VLDLR)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, were recently identified as receptors for EEEV. To assess the ability of EEEV-373 to block receptor interactions, we performed a competition-binding ELISA with ApoER2 and VLDLR (Figs.Â <a href="#Fig7" class="usa-link">7A, B</a>). In this assay, EEEV VLPs were incubated with EEEV-373 or a negative control mAb as IgG1 molecules. ApoER2 or VLDLR then was added to detect whether receptor binding can occur in the presence of mAb. Based on the results, EEEV-373 blocked ~80% or ~50% of the interactions with ApoER2 or VLDLR, respectively, compared to the negative control mAb. Recently, the VLDLR binding footprint on the E2 glycoprotein of EEEV was identified<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The E2 residues making the most significant interactions with VLDLR (H155, K156, and R157) are on the â€˜central archâ€™ region of the Î²-ribbon connector, which is adjacent to domain A. This region is distinct from the EEEV-373 binding site, which is on â€˜arch 2â€™ of the Î²-ribbon connector. Therefore, it is likely that the partial inhibition of VLDLR binding to EEEV by EEEV-373 is through steric blockade of cognate receptor binding or by allosteric modulation of the Î²-ribbon connector. The receptor binding site for ApoER2 on EEEV is not known yet. However, the results observed suggests EEEV-373 may bind within or nearby the receptor binding site to block or sterically hinder ApoER2 binding (Fig.Â <a href="#MOESM1" class="usa-link">S4</a>). As EEEV-373 binds in a preferential manner across the i2 axis, incomplete occupancy of exposed receptor binding sites may occur. Under saturating conditions, 60 out of 80 trimeric spikes are stably occupied with EEEV-373 IgG per virion. For the occupied 60 q3 spikes, only a third of the E2 glycoproteins are bound by IgG, exposing the remaining E2 glycoproteins for receptor binding. This mechanism likely accounts for the observed residual binding of these receptors to EEEV VLPs in the presence of saturating concentrations of EEEV-373.</p></section><section id="Sec10"><h3 class="pmc_sec_title">EEEV-373 partially inhibits SINV/EEEV after virus attachment to cells</h3>
<p id="Par21">We next tested the ability of EEEV-373 to neutralize SINV/EEEV after initial attachment to a Vero cell monolayer using a post-attachment inhibition assay (Fig.Â <a href="#Fig7" class="usa-link">7D</a>), as previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. In this assay, Vero cells were incubated with SINV/EEEV particles at 4â€‰Â°C to allow for virus attachment. EEEV-373 then was added, and the cells were shifted to 37â€‰Â°C to enable virus entry. In comparison to a standard focus reduction neutralization assay (FRNT, Fig.Â <a href="#Fig7" class="usa-link">7D</a>), EEEV-373 still neutralized SINV/EEEV as IgG or F(abâ€²)2 molecules after virus attachment to Vero cell monolayer cultures. However, a reduced potency in neutralization was observed (~8- to &gt;200-fold shift in IC<sub>50</sub> values for IgG or F(abâ€²)2, respectively). This finding may be due to the inaccessibility of all epitopes through interactions between virus particles and cellular receptor or attachment factors on cells, leading to incomplete neutralization. Thus, EEEV-373 may partially neutralize through receptor blocking as previously described and is unable to neutralize as efficiently once the virus has already interacted with cells. Consistent with previous data (Fig.Â <a href="#Fig2" class="usa-link">2C</a>), EEEV-373 Fab molecules did not neutralize SINV/EEEV.</p></section><section id="Sec11"><h3 class="pmc_sec_title">EEEV-373 inhibits initial low-pH-induced fusion of SINV/EEEV with liposomes</h3>
<p id="Par22">We next assessed whether EEEV-373 inhibits low-pH-induced fusion of SINV/EEEV with liposomes as previously described in ref. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. This assay serves as a surrogate for virus fusion with the endosomal membrane during alphaviral infection of cells by measuring the dequenching of DiD (1,1â€²-dioctadecyl-3,3,3â€²,3â€²-tetramethylindodicarbocyanine, 4-chlorobenzenesulfonate salt)-labeled SINV/EEEV particles. In this assay, DiD-labeled SINV/EEEV particles were incubated with an excess of unlabeled liposomes (200â€‰nm in size) and membrane fusion was triggered by acidification with a low-pH buffer (final pH ~5.2). Viral fusion was measured by dilution of DiD within the liposomal membrane, and a subsequent increase in fluorescence intensity over time. Inhibition of SINV/EEEV fusion was measured by the reduction of DiD fluorescence dequenching. Pre-incubation of DiD-labeled SINV/EEEV particles with EEEV-373 IgG or F(abÊ¹)2 molecules led to an initial reduction in relative fluorescence intensity during the first 10-30â€‰s after acidification compared to EEEV-373 Fab molecules or negative controls (<em>i.e</em>., irrelevant controls or absence of antibody) (Fig.Â <a href="#Fig7" class="usa-link">7E</a>). This finding suggests bivalent EEEV-373 molecules inhibit initial fusion of SINV/EEEV particles with liposomes or delay fusion kinetics. As time progressed, however, relative fluorescence levels were comparable to those of the negative controls. The reason for this finding is not entirely known. The inability to completely abolish fusion even at saturating concentrations may occur due to incomplete occupancy of necessary epitopes on the virion to block fusion, which enables the virus to compensate for partial occupancy or EEEV-373 binding and subsequently to fuse. Alternatively, multiple mechanisms of action may be at play for the potent neutralization of SINV/EEEV observed by EEEV-373.</p></section><section id="Sec12"><h3 class="pmc_sec_title">EEEV-373 fails to inhibit SINV/EEEV egress</h3>
<p id="Par23">Lastly, we assessed the ability of EEEV-373 to inhibit egress of SINV/EEEV from infected cells, using an established egress assay<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Briefly, BHK-21 cell culture monolayers were inoculated with SINV/EEEV at an MOI of 1 for 2â€‰h at 37â€‰Â°C. Cells were washed extensively to remove any unbound virus and then incubated with EEEV-373 or the negative control mAb rDENV-2D22 as IgG molecules in the presence of ammonium chloride to prevent de novo infection. Supernatant was harvested following 1 or 6â€‰h to isolate and detect newly made viral RNA molecules by qRT-PCR. Compared with theÂ previously published negative control<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, EEEV-373 did not significantly inhibit SINV/EEEV egress from BHK-21 cells (Fig.Â <a href="#Fig7" class="usa-link">7F</a>).</p>
<p id="Par24">During the alphavirus replication cycle, viral intermediates displaying heterodimers of E2 and E1 are formed on the surface of infected cells prior to egress<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Given the observed preferential binding mode of EEEV-373 as IgG molecules to SINV/EEEV particles and lack of detectable inhibition of viral egress, we hypothesized that EEEV-373 may not bind to the intermediate forms of EEEV during viral infection. To test this model, we transiently transfected cells with the structural polyprotein (capsid-E3-E2-6K-E1) of EEEV strain FL93-939 and assessed binding of EEEV-373 IgG molecules (Fig.Â <a href="#MOESM1" class="usa-link">S8</a>). Compared to the positive control mAb EEEV-106<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, we observed minimal binding of EEEV-373 to the EEEV structural proteins expressed on the cell surface. This observation supports the hypothesis that EEEV-373 binds to a preferred conformation present principally on intact virions.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par25">In this study, we present the structural and molecular basis of epitope recognition and mechanism of action of the potently neutralizing and protective anti-EEEV human mAb EEEV-373. Several aspects of this interaction are interesting from the perspective of therapeutic antibody development and structure-guided immunogen design for vaccine development. First, we present a cryo-EM structure of SINV/EEEV particles in complex with an intact IgG1 molecule. This work adds to a growing but a small list of IgG1-antigen complex structures<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Structures of virus-antibody complexes reveal features of interactions that are not always obvious from the corresponding structures of Fab molecules. For example, the bivalent nature and flexibility of the two arms of the antibody can lead to preferred engagement of sites on the viral surface. Monovalent Fab molecules may or may not recapitulate this preferred recognition aspect of the interaction, and thus these studies may lead to inaccurate representation of Fab occupied viral epitopes. As we see in this reconstruction, EEEV-373 preferentially cross-links the two q3-spikes across the i2 axis.</p>
<p id="Par26">The quaternary epitope revealed by the reconstruction presented here is unique in that the CDRs of EEEV-373 primarily contact residues in domain B and the Î²-ribbon connector of the E2 glycoprotein. The observed heavy chain CDRH2 interaction with residue K61 of the E1 glycoprotein is largely due to its long side chain, as no other surrounding E1 residues contribute to the binding footprint. This binding mode differs from that reported for the murine anti-chikungunya virus (CHIKV) mAb CHK-263 IgG, which binds to a similar area on CHIKV particles<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. At the i2 axis, two CHK-263 IgG molecules engage the four q3 spikes. However, due to the low resolution of the reconstructions, the connectivity between the Fab arms of the two IgGs could not be ascertained. The binding modes of CHK-263 IgG suggested by authors correspond to the q3.1â€“q3.4 and q3.3â€“q3.4 alternative binding modes depicted here for EEEV-373 (Fig.Â <a href="#Fig6" class="usa-link">6</a>). Like EEEV-373, weak density of CHK-263 around the i5 axis was also observed, which the authors interpreted as a flexible interaction. In contrast to EEEV-373, CHK-263 tilts ~17.5Â° towards and makes contacts with the E1 glycoprotein (Fig.Â <a href="#MOESM1" class="usa-link">S5</a>). This difference in the angle of approach may be one of the reasons one EEEV-373 IgG molecule binds diagonally across the i2 axis on SINV/EEEV particles, whereas two CHK-263 IgG molecules are able to bind CHIKV particles. The heavy and light chains of CHK-263 are also flipped with respect to the corresponding chains of EEEV-373. Furthermore, CHK-263 binds as monovalent Fab molecules, which may occur due to recognition of a stabilizing quaternary epitope comprising both proteins of the E2/E1 heterodimer (as K61 and C63 of CHIKV E1 domain II also bind the CDRs of CHIK-263). The â€˜arch 2â€™ cleft of the E2 protein Î²-ribbon connector into which the CDRH3 of EEEV-373 inserts is stabilized by the extensive interactions with the underlying E1 glycoprotein. It is this interaction that renders the epitope conformational and quaternary, which is validated by the ELISA binding data showing that bivalent EEEV-373 does not bind either recombinant monomeric E2 or E1 glycoproteins but binds to the E2/E1 heterodimer and VLPs (Fig.Â <a href="#Fig2" class="usa-link">2A</a>).</p>
<p id="Par27">The bivalency requirement for complex formation is a remarkable feature of EEEV-373. Monovalent Fab molecules of neutralizing antiviral antibodies tend to have lower affinity to viruses compared to their bivalent counterpart due to the absence of avidity effects. However, the weak detectable binding observed for monovalent EEEV-373 Fab molecules is consistent with the requirement for bivalency to neutralize EEEV. The biphasic nature of association observed in the kinetic analyses strongly supports the contribution of avidity effects towards complex formation. The dependence on avidity may stem from the flexibility of the heterodimer epitope, necessitating slower interactions for stable complex formation. Given the lack of significant secondary structural elements in the Î²-ribbon connector, the â€˜arch 2â€™ cleft may undergo structural transitions, leading to flexibility in the epitope. However, this could not be visualized due to the limited resolution of this region. Another explanation may result from binding of one Fab arm of the IgG molecule to the heterodimer epitope during the fast initial binding interaction step, while the other Fab arm transiently samples additional heterodimer epitopes in a slower fashion until recognizing the correct orientation to generate a high affinity complex. In such scenarios, avidity contributions from bivalent engagement of the IgG with virus particles shifts the equilibrium towards a stable complex formation.</p>
<p id="Par28">The relatively high potency and intra-virion cross-linking displayed by EEEV-373 mirrors the activity of another potently neutralizing anti-EEEV antibody, EEEV-106<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. The ability to form intra-virion crosslinks on the icosahedral viral surface appears to correspond well with the high potency of neutralizing antibodies. The intra-virion cross-linking observed for EEEV-373 appears to be structurally constrained as the binding mode of EEEV-373 observed in the reconstruction strongly suggests that there is limited flexibility of the Fab arms with respect to one another. Theoretically, there are six possible ways the spikes across the i2 axis can be cross-linked by EEEV-373 (Fig.Â <a href="#Fig6" class="usa-link">6</a>). As discussed, three of these binding modes (i3.1â€“q3.1, q3.1â€“q3.4, or q3.3â€“q3.4) are not feasible due to distance or steric constraints. However, while the remaining three modes (i3.1â€”i3.2, q3.1â€“q3.3, and q3.1â€“i3.2) satisfy both conditions, only one binding orientation was observed (q3.1â€“q3.3). In the i3.1â€“i3.2 binding mode, the Fab arms must stretch by ~50% to simultaneously bind (Fig.Â <a href="#Fig6" class="usa-link">6D</a>). In the q3.1â€“i3.2 binding mode, IgG binding would require the Fab arms to rotate around the IgG 2-fold axis by ~60Â° with respect to each other (Fig.Â <a href="#Fig6" class="usa-link">6F</a>). It is not obvious from the structure why these two alternative binding modes are not observed. It may be that EEEV-373 transiently interacts with these orientations prior to binding the observed preferred orientation (q3.1â€“q3.3). This outcome may result from an energetically favorable interaction with the q3.1â€“q3.3 binding mode.</p>
<p id="Par29">EEEV-373 neutralizes EEEV by acting at multiple steps in the viral entry pathway (Fig.Â <a href="#Fig7" class="usa-link">7</a>). EEEV-373 reduced ApoER2 or VLDLR binding by 80 or 50%, respectively, which suggests EEEV-373 binds near the receptor binding site or may sterically hinder the virus particles from binding to receptors (Fig.Â <a href="#MOESM1" class="usa-link">S4B</a>). The incomplete extent of receptor blockade may be attributed to the exposure of additional epitopes besides those that are occupied by EEEV-373 IgG1 molecules. The receptor binding site for ApoER2 and VLDLR is thought to overlap on virus particles, as soluble VLDLR blocks EEEV infection of K562 cells overexpressing ApoER2<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The observed differences in receptor blockade by EEEV-373 may be due to differences in binding affinity to virus particles. The apparent affinity of VLDLR to EEEV VLPs is ~2.1 to 15â€‰nM<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, whereas EEEV-373 binds to EEEV VLPs with an affinity of ~14â€‰nM. The strong affinity of VLDLR to EEEV VLPs may enable the receptor to outcompete EEEV-373, which may explain the incomplete blockade of VLDLR. In contrast, the affinity of ApoER2 to EEEV VLPs is not known. However, based on the results, this interaction may have lower affinity due to greater blockade of ApoER2 by EEEV-373.</p>
<p id="Par30">Even after SINV/EEEV attachment to cells, EEEV-373 maintains neutralization activity, albeit at aâ€‰~8-fold reduced potency. Post-attachment neutralization activity suggests EEEV-373 also inhibits viral entry into cells. Furthermore, bivalent EEEV-373 molecules (IgG or F(abâ€²)2) inhibit early fusion events in the endosome, as suggested by its ability to prevent low pH-induced fusion of SINV/EEEV with liposomes. Altogether, EEEV-373 may block or partially stall steps necessary for virus entry and fusion into the cell. The cumulative inhibitory effect of EEEV-373 at multiple steps probably explains the observed potent neutralization of SINV/EEEV and 100% protective efficacy in a mouse subcutaneous challenge model of EEEV (Fig.Â <a href="#Fig1" class="usa-link">1</a>). The bivalent requirement for EEEV-373 to neutralize SINV/EEEV entry may relate to how EEEV-373 binds to virus particles. The intra-virion cross-linking of the q3 spikes across the i2 axis suggests that SINV/EEEV particles may be stabilized, such that the necessary transitional changes for virus entry and fusion are hindered. The weak association and lack of neutralization activity against SINV/EEEV as Fab molecules supports this due to the absence of avidity effects.</p>
<p id="Par31">EEEV-373 IgG1 molecules did not block viral egress from infected cells, and the antibody minimally binds to E2 or E1 proteins expressed at the plasma membrane (Fig.Â <a href="#MOESM1" class="usa-link">S8</a>). These findings strongly suggest that the epitope is not assembled in the proper quaternary formation or is inaccessible during virus particle assembly and egress from the plasma membrane of infected cells. The structural arrangement of glycoproteins on intact virions presents the correct geometry (i.e., distance, angle, and curvature) for EEEV-373 to engage with its epitope, which is otherwise transient or absent on the plasma membrane of transiently transfected or infected cells during budding of virus progeny.</p>
<p id="Par32">EEEV-373 is interesting because of its recognition of a complex dynamic antigenic site. The occurrence of bivalent interactions required for stable binding and efficient neutralization allows the antibody to perform multiple functions with less molecules. Complex epitopes on viral surface antigens are attractive vaccine targets since the neutralizing antibodies that target them tend to be potent. Additionally, antibodies targeting complex conserved epitopes may provide broad protection against different serotypes or lineages of viruses. For example, for flaviviruses, such as dengue (DENV), antibodies that target quaternary epitopes within the E protein dimers show potent neutralization capability against all four DENV serotypes<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Similarly, another potently neutralizing antibody currently in clinical development, designated ZIKV-117, neutralizes Zika virus strains from different lineages and targets a quaternary epitope spanning across E protein dimer-dimer interfaces<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. In contrast to flaviviruses like dengue, Zika and West Nile viruses, the structural characterization of quaternary epitopes on alphaviruses is lacking. Interestingly, cryo-EM reconstructions of alphaviruses in complex with protein receptors have shown the receptor binding site to interact with residues in both the E2 and E1 glycoproteins, thus, rendering alphavirus receptor binding a quaternary interaction<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. To the best of our knowledge, structural characterization of a highly conformationally-dependent epitope on the E2 glycoprotein of a major alphaviral pathogen targeted by a patient-derived human antibody has not been previously observed. The strict dependence on bivalency for binding and neutralization displayed by EEEV-373 highlights surprising intricacies of the antigen-antibody interaction. The elucidation of this complex interaction has important implications for therapeutic antibody developments against viral pathogens. Altogether, the work described here helps inform us on the rational design of immunogens for EEEV and likely other clinically relevant alphaviruses to elicit protective responses to complex epitopes.</p></section><section id="Sec14"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec15"><h3 class="pmc_sec_title">Human subject information</h3>
<p id="Par33">EEEV-373 was isolated from one research subject as previously described in ref. <sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Briefly, peripheral blood mononuclear cells (PBMCs) were isolated from peripheral blood samples by density gradient purification and cryopreserved until use. Written informed consent was given from the subject and protocols were approved by the Institutional Review Board (IRB) at Vanderbilt University Medical Center for the recruitment and collection of blood samples used in this study (8675).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Mouse model</h3>
<p id="Par34">Animal models were performed as previously described in refs. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. C57BL/6 mice were purchased from Jackson Laboratories. All animal procedures were carried out in accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. The protocols were approved by the Institutional Animal Care and Use Committee at the Utah State University IACUC protocol #10025.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Cell lines</h3>
<p id="Par35">Cell lines were maintained as previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Routine mycoplasma detection was performed using a universal mycoplasma detection kit (ATCC) and in all cases gave negative results for the cell lines tested except for the kit positive control reagent.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Viruses and virus-like particles (VLPs)</h3>
<p id="Par36">The chimeric virus Sindbis virus (SINV; TR339)/Eastern equine encephalitis virus (EEEV; strain FL93â€“939) was previously described in ref. <sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Briefly, the structural proteins genes of SINV TR339 were replaced with the structural protein genes of EEEV (FL93-939) under control of the SINV 26S subgenomic promoter in the cDNA clone. For wild-type virus neutralization studies, EEEV (strain FL93-939) was derived from a cDNA clone as previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. For animal studies, the EEEV FL93-939 strain used was obtained from Dr. Robert Tesh, World Reference Center for Emerging Viruses and Arboviruses (University of Texas Medical Branch), Galveston, TX and passaged twice in Vero cells prior to use in mice<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Purified EEEV VLPs were kindly provided by Dr. John Mascola<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Recombinant proteins</h3>
<p id="Par37">Recombinant EEEV E2/E1 and monomeric E1 proteins (strain FL93-939) were expressed and purified as previously described, respectively<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Briefly, the proteins were codon-optimized, synthesized, and cloned into the mammalian expression vector pcDNA3.1(+). Recombinant proteins were produced in Expi293F cells using the ExpiFectamine 293 transfection kit according to manufacturerâ€™s instructions (Thermo Fisher Scientific). Cell supernatant was clarified and purified through a HisTrap excel column (Cytiva) on an Ã„KTA pure 25â€‰M chromatography system. Recombinant EEEV E3E2/E2 (strain v105), which contains a mixture of E3E2 and E2 glycoproteins, was purchased from IBT Bioservices.</p></section><section id="Sec20"><h3 class="pmc_sec_title">SINV/EEEV production and purification</h3>
<p id="Par38">For binding and neutralization assays, SINV/EEEV was produced using the methods previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. For cryo-EM studies, SINV/EEEV was purified according to the previously described protocol<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. For liposomal fusion inhibition studies, SINV/EEEV was purified and labeled with Vybrant DiD cell-labeling solution as previously described in ref. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Human hybridoma antibody generation and sequence analysis</h3>
<p id="Par39">EEEV-373 hybridoma cell lines secreting EEEV-373 IgG was generated, selected, and purified as previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Recombinant mAbs were produced using the ExpiCHO expression system (Thermo Fisher Scientific) as previously described in ref. <sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Antibodies were purified from clarified supernatants using HiTrap MabSelect SuRe (Cytiva) columns on an Ã„KTA Pure 25â€‰M chromatography system and further concentrated using 50â€‰K MWCO AmiconÂ® Ultra centrifugal filter units (MilliporeSigma), desalted, and buffer exchanged with 7â€‰K MWCO Zeba desalting columns (Thermo Fisher Scientific). For the generation of EEEV-373 Fab or F(abâ€²2) molecules, hybridoma mAb was cleaved using the IgdE cysteine enzyme FabALACTICAÂ® (Genovis) as previously described in ref. <sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> or using the IdeS protease (Promega), respectively. Briefly, IgG was incubated with FabALACTICAÂ® or IdeS protease (1 unit/1â€‰Âµg of IgG) overnight at room temperature or 37â€‰Â°C for 30 to 60â€‰min, respectively. Fab or F(abâ€²2) molecules were then purified using the CaptureSelect<sup>TM</sup> IgG-Fc affinity matrix (Thermo Fisher Scientific). For sequence analysis of EEEV-373 (Supplementary TablesÂ <a href="#MOESM1" class="usa-link">2</a>, <a href="#MOESM1" class="usa-link">3</a>), the gene segments and V-region nucleotide percent identity compared to germline were determined using the ImMunoGenetics (IMGT) database as previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Protein EC<sub>50</sub> ELISA</h3>
<p id="Par40">To determine the binding EC<sub>50</sub> values of EEEV-373 to recombinant antigens an ELISA was performed as previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Briefly, EEEV VLPs (2â€‰Âµg/mL) and recombinant EEEV E3E2/E2 (2â€‰Âµg/mL), E2/E1 (3.84â€‰Âµg/mL), or E1 (2â€‰Âµg/mL) proteins were diluted in 1Ã— D-PBS to coat 384-well ELISA plates (Thermo Fisher Scientific) and incubated at 4â€‰Â°C overnight. The plates were aspirated and incubated with blocking solution (2% non-fat dry milk (Bio-Rad), 2% goat serum (Thermo Fisher Scientific) in 1Ã— D-PBS-T [1Ã— D-PBSâ€‰+â€‰0.05% Tween 20]) for 1â€‰h at room temperature. EEEV-373 IgG, F(abâ€²)2, or Fab molecules were initially diluted to 33â€‰nM then serially diluted 3-fold in blocking solution (1% non-fat dry milk, 1% goat serum in 1Ã— D-PBS-T). Abs then were added to the plates and incubated at room temperature for 2â€‰h. The plates then were washed 3Ã— with 1Ã— D-PBS-T and incubated with a solution of secondary antibodies (goat anti-human kappa-HRP and goat-anti-human lambda-HRP [Southern Biotech]) diluted 1:4,000 in blocking solution (1% non-fat dry milk, 1% goat serum in 1Ã— D-PBS-T) for 1â€‰h at room temperature. The plates then were washed 3Ã— with 1Ã— D-PBS-T followed by the addition of One-Step<sup>TM</sup> Ultra-TMB ELISA substrate solution (Thermo Fisher Scientific) for ~5â€‰min. The reaction was stopped with 1â€‰N HCl and optical density then was read at 450â€‰nm with a BioTek<sup>TM</sup> plate reader.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Binding kinetics analysis using a quartz crystal microbalance biosensor</h3>
<p id="Par41">EEEV VLPs (35â€‰Âµg/mL) diluted in 10â€‰mM HEPES buffer (pH 7.4) were immobilized onto LNB carboxyl sensor chips (Attana AB) by amine coupling using EDC (1-ethylâ€“3-[3-dimethylaminopropyl]-carbodiimide hydrochloride) and Sulfo-NHS (<em>N</em>-hydroxysulfosuccinimide) chemistry on the Attana Cell<sup>TM</sup> 200 instrument (Attana AB). For binding assays, first to account for background binding, the sensor chips were injected with running buffer (1Ã— HBS-T). EEEV-373 IgG (33, 8, or 0.33â€‰nM), F(abÊ¹)2 (35, 10, or 0.35â€‰nM), or Fab (35, 10, or 0.35â€‰nM) molecules diluted in running buffer then were injected to observe Ab binding to EEEV VLPs. Between Ab injections, EEEV VLP immobilized LNB carboxyl sensor chips were regenerated using 10â€‰mM glycine pH 3.5 buffer. Assays were performed at a flow rate of 10â€‰ÂµL/min at 22â€‰Â°C using 50- and 600â€‰s association and dissociation times, respectively. Data were collected on the Attester software (Attana AB). A blank LNB carboxyl sensor chip was used as a reference and subtracted using the Evaluation software (Attana AB). Curves were fitted and kinetic parameters calculated using a bivalent (IgG and F(abâ€²)2) binding model using the TraceDrawer software (Ridgeview Instruments AB).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Binding kinetics analysis using biolayer interferometry</h3>
<p id="Par42">Biolayer interferometry was performed using an OctetÂ® HTX system (Sartorius). Following a baseline measurement in kinetics buffer (Sartorius), 10â€‰Âµg/mL of EEEV-373 IgG, F(abâ€²)2, and negative control IgG molecules, or 6.7â€‰Âµg/mL of EEEV-373 and EEEV-94 Fab molecules were separately loaded to protein L biosensors (Sartorius). The biosensors were then dipped into kinetics buffer followed by incubation with EEEV VLPs at concentrations of 0, 0.2, 0.5, 0.8, 1, or 2â€‰nM for 300â€‰s. Dissociation was then measured by biosensor incubation in kinetics buffer for 1800â€‰s. A reference subtraction was performed for each antibody and curves were fitted using a 1:1 binding model. Kinetic analysis was performed using the OctetÂ® Data Analysis v12.2 software (Sartorius).</p></section><section id="Sec25"><h3 class="pmc_sec_title">Cell surface display EEEV binding</h3>
<p id="Par43">Binding analysis of mAbs to cells expressing alphavirus structural proteins has been previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Briefly, Expi293F cells were transiently transfected with a plasmid (pcDNA3.1(+)) containing the structural proteins (capsid-E3-E2-6K-E1) of EEEV (strain FL93-939) using the ExpiFectamine 293 transfection kit according to manufacturerâ€™s protocols (Thermo Fisher Scientific). Cells were incubated at 37â€‰Â°C in a humidified atmosphere of 8% CO<sub>2</sub> for 24â€‰h. Cells then were harvested, fixed with 1% PFA/PBS, washed twice with 1Ã— D-PBS, and stored at 4â€‰Â°C in FACS buffer (1Ã— D-PBS, 2% ultra-low IgG FBS, 2â€‰mM EDTA) until use. Cells were plated at 40â€“50,000 cells/well in 96-well V-bottom plates. EEEV-373, rEEEV-106, or rDENV-2D22 IgG were diluted to 1â€‰Âµg/mL in FACS buffer and incubated with the cells at 4â€‰Â°C for 1â€‰h. Cells then were washed with FACS buffer and incubated with secondary antibodies (anti-human IgG-PE and anti-human IgA-PE [Southern Biotech]) diluted 1:1,000 in FACS buffer for 1â€‰h at 4â€‰Â°C. Cells then were washed in FACS buffer, and the number of events was collected on an IntelliCytÂ® iQue Screener Plus flow cytometer (Sartorius).</p></section><section id="Sec26"><h3 class="pmc_sec_title">Dynamic light scattering (DLS)</h3>
<p id="Par44">DLS was performed to assess antibody-mediated aggregation of virus particles as previously described in ref. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Briefly, EEEV-373 IgG, F(abâ€²)2, or Fab molecules were mixed and incubated at 37â€‰Â°C for 30â€‰min with EEEV VLPs at Ab:VLP molar ratios ranging from 1:1 to 1,000:1. VLPs were incubated without mAb to control for particle size (~70â€‰nm). DLS was then performed at 37â€‰Â°C using a biologics stability screening platform (Uncle) and Uncle version 5.0 analysis software (Unchained Labs). Mean peak intensity was used to determine the hydrodynamic diameter (nm) of EEEV VLPs.</p></section><section id="Sec27"><h3 class="pmc_sec_title">ApoER2 and VLDLR receptor blockade</h3>
<p id="Par45">EEEV VLPs were diluted to 2â€‰Âµg/mL in 1Ã— D-PBS to coat 384-well ELISA plates (Thermo Fisher Scientific) and incubated at 4â€‰Â°C overnight. The plates were aspirated and incubated with blocking solution (2% non-fat dry milk (Bio-Rad), 2% goat serum (Thermo Fisher Scientific) in 1Ã— D-PBS-T [1Ã— D-PBSâ€‰+â€‰0.05% Tween 20]) for 1â€‰h at room temperature. EEEV-373 or a negative control mAb were diluted for a final concentration of 3 to 67â€‰nM in blocking solution (1% non-fat dry milk, 1% goat serum in 1Ã— D-PBS-T). MAbs then were added to the plates and incubated at room temperature for 1â€‰h. Recombinant human apolipoprotein E R2 protein (ApoER2; R&amp;D Systems) or human very low-density lipoprotein receptor (VLDLR; R&amp;D Systems) protein were diluted to a final concentration of 5â€‰Âµg/mL. ApoER2 or VLDLR were then added to the plates and incubated at room temperature for 1â€‰h. The plates then were washed 3Ã— with 1Ã— D-PBS-T and incubated with a solution of secondary antibodies (anti-his-HRP [Thermo Fisher Scientific]) diluted 1:2,000 in blocking solution (1% non-fat dry milk, 1% goat serum in 1Ã— D-PBS-T) for 1â€‰h at room temperature. The plates then were washed 3Ã— with 1Ã— D-PBS-T followed by the addition of One-Step<sup>TM</sup> Ultra-TMB ELISA substrate solution (Thermo Fisher Scientific) for ~5â€‰min. The reaction was stopped with 1â€‰N HCl and optical density then was read at 450â€‰nm with a BioTek<sup>TM</sup> plate reader. Percent binding of ApoER2 or VLDLR was normalized to the average optical density value for binding in the presence of the negative control mAb.</p></section><section id="Sec28"><h3 class="pmc_sec_title">SINV/EEEV neutralization assays</h3>
<p id="Par46">Neutralization assays, including the focus reduction neutralization test (FRNT) and post-attachment assays, were formed as previously described in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Briefly, EEEV-373 IgG, F(abâ€²)2, or Fab molecules were initially diluted to 33â€‰nM then serially diluted 3-fold in medium (DMEM/2% ultra-low IgG FBS/10â€‰mM HEPES). Abs were then incubated with ~100 FFU/well of SINV/EEEV and incubated together for 1â€‰h at 37â€‰Â°C in a humidified atmosphere of 5% CO<sub>2</sub> or 4â€‰Â°C, respectively. Virus foci were detected as previously described.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Liposomal fusion inhibition assay</h3>
<p id="Par47">Liposomes (2â€‰mM) containing 1,2-dioleoyl-sn-glycero-3-phosphocholine (DOPC), 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine (DOPE), sphingomyelin (chicken egg), and cholesterol (ovine wool) were prepared as previously described in ref. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Fusion inhibition was assessed as previously described in ref. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Briefly, Vybrant DiD-labeled SINV/EEEV particles were incubated with EEEV-373 or negative control IgG (9â€‰Âµg), F(abâ€²)2 (6.2â€‰Âµg), or Fab (3â€‰Âµg) molecules for 30â€‰min at room temperature with shaking (300â€‰rpm). Buffer with no Ab or liposome controls were also included. Prepared liposomes (2â€‰mM) then were added, expect for the no liposome control, and loaded onto a SpectraMax iD5 multimode plate reader (Molecular Devices) at 37â€‰Â°C. Sequential injections of 0.1â€‰M MES-0.2â€‰M acetic acid (pH 5.0) were performed using an automated injector and mixed immediately. Fluorescence (excitation at 635â€‰nm, emission at 675â€‰nm) then was read every 10â€‰s for a total of 100â€‰s.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Egress inhibition assay</h3>
<p id="Par48">Egress inhibition was performed as previously described in ref. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Briefly, BHK-21 cells were inoculated with SINV/EEEV (MOI 1) in medium (DMEM/2% ultra-low IgG FBS/10â€‰mM HEPES) and then incubated at 37â€‰Â°C in a humidified atmosphere of 5% CO<sub>2</sub> for 2â€‰h. Cells then were washed 6Ã— with medium containing 25â€‰mM NH<sub>4</sub>Cl. EEEV-373 or rDENV-2D22 IgG were diluted to 10â€‰Âµg/mL in the same medium and added to the cells.Â The previously published negative control mAb, rDENV-2D22 IgG, and SINV/EEEV only control groups performed at the same time were included for comparison<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. The cells then were incubated at 37â€‰Â°C in a humidified atmosphere of 5% CO<sub>2</sub> for up to 6â€‰h. Supernatant was harvested after 1- and 6-hour incubation periods. Viral RNA then was extracted from the supernatant using the MaxwellÂ® Viral Total Nucleic Acid purification kit on the MaxwellÂ® system (Promega). qRT-PCR and quantification of relative RNA copies/ÂµL were performed as previously described using the SuperScript<sup>TM</sup> III Platinum One-Step qRT-PCR kit (Thermo Fisher Scientific)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p></section><section id="Sec31"><h3 class="pmc_sec_title">SINV/EEEV escape mutant generation and analysis</h3>
<p id="Par49">To assess the ability for viral escape in the presence of EEEV-373, a real-time cell analysis (RTCA) assay and xCELLigence RTCA Multiple Plates instrument was used. The RTCA assay was performed by adding DMEM/2% ultra-low IgG FBS/10â€‰mM HEPES to each well of a 96-well E-plate to establish a background reading. Vero cells (~18,000) were then seeded to each well of the E-plate overnight. Continuous monitoring of cell index values was performed every 15â€‰min and analyzed using the RTCA software. Initial establishment of the neutralization potency of EEEV-373 in the RTCA assay was performed by incubating SINV/EEEV at an MOI of ~0.06 with serial three-fold dilutions of EEEV-373 or a negative control mAb starting at 10â€‰Âµg/mL for 1â€‰h at 37â€‰Â°C in 5% CO<sub>2</sub>. The virus:mAb complexes were then added to the Vero cell monolayers and continuously monitored for changes in cell index values over time. Wells containing only medium or SINV/EEEV were included as controls. Normalized cell index values approximately 48â€‰h post-inoculation were determined to establish the percent neutralization. To select for escape mutant viruses, a similar experiment was performed with some modifications. SINV/EEEV at an MOI of ~0.06 was incubated with saturating concentrations (50 and 10â€‰Âµg/mL) of EEEV-373 or a negative control mAb for 1â€‰h at 37â€‰Â°C in 5% CO<sub>2</sub>. The virus:mAb complexes were then added to the Vero cell monolayers and continuously monitored for changes in cell index values over time. Wells in which cytopathic effect was observed in the presence of EEEV-373 compared to control wells were identified as potential escape mutants and supernatant was collected.</p>
<p id="Par50">To verify the observed phenotype, supernatants were incubated in the absence or presence of EEEV-373 (20â€‰Âµg/mL), added to confluent Vero cell monolayers in a 6-well plate, and then incubated at 37â€‰Â°C in 5% CO<sub>2</sub>. Supernatants were then collected, and viral RNA was extracted using a QIAmp viral RNA extraction kit (Qiagen). cDNA corresponding to the E2 protein gene was then generated and amplified using a SuperScript<sup>TM</sup> III One-Step RT-PCR system with Platinum<sup>TM</sup> Taq DNA polymerase (Thermo Fisher Scientific) with the following forward and reverse primers, respectively: 5â€² ATGTGCGTCCTGGCCAATATCACGTTTCC 3â€² and 5â€² GAACAAAACTAGGGCAACCACTGCTGTAGC 3â€². The PCR product was then purified using a QIAquick PCR purification kit (Qiagen). The resulting product was sequenced using long-read Oxford Nanopore technology (Plasmidsaurus). From the obtained sequences, variants were identified and compared to consensus sequences or the corresponding input wild-type SINV/EEEV sequence. Escape mutant viruses were identified as variants obtained only in the presence of EEEV-373.</p></section><section id="Sec32"><h3 class="pmc_sec_title">EEEV plaque reduction neutralization test</h3>
<p id="Par51">The neutralization activity of EEEV-373 against wild-type EEEV (strain FL93-939) was performed as previously described in ref. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Briefly, EEEV-373 (75â€‰nM) was serially diluted 2-fold and incubated with ~100 PFU of EEEV at 37â€‰Â°C for 1â€‰h. The previously publishedÂ anti-EEEV ascites fluid (ATCC) and rDENV-2D22 IgG positive and negative controls, respectively, performed at the same time were included for comparison<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Ab:virus complexes then were added to Vero cell monolayer cultures and incubated at 37â€‰Â°C for 1â€‰h. An agarose overlay then was added and plates were incubated at 37â€‰Â°C for 2 days. For detection of plaques, a neutral red overlay was added.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Mouse subcutaneous EEEV challenge model</h3>
<p id="Par52">Mice were challenged with EEEV by bilateral subcutaneous injections as previously described in refs. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Mice were treated with EEEV-373 at 10â€‰mg/kg via a single intraperitoneal injection 24â€‰h after EEEV challenge. Previously published controls, including rDENV-2D22 IgG and normal controls, performed at the same time were included for comparison<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The mice were monitored for 21 days post-virus inoculation (dpi) for survival, disease signs, and body weight. Serum was collected from the mice 3 dpi to assess virus titers via an infectious cell culture assay as previously described in refs. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Sample preparation for cryo-EM and data collection</h3>
<p id="Par53">SINV/EEEV particles were mixed with EEEV-373 at a ratio of 1:1 (virus:IgG) and 3â€‰ÂµL of the virus-IgG mixture was immediately loaded onto glow discharged lacey carbon grids in a BSL2-containment facility. The grids then were blotted for 3.2â€‰s and plunge frozen in liquid ethane using a Gatan CP3. The entire process was performed under 30â€‰s to avoid potential aggregation of the particles. The frozen grids of the complexes were imaged using a Gatan Bioquantum-K3 camera mounted on a Titan Krios (Thermo Fisher Scientific) microscope. The data collection parameters are listed in Supplementary TableÂ <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec35"><h3 class="pmc_sec_title">Cryo-EM data processing</h3>
<p id="Par54">Data was processed as previously described in ref. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. The resolution of the final map (4.6â€‰Ã…) was calculated using the Gold Standard Fourier Shell Correlation (GSFSC) coefficient at 0.143<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Although the quality of the map was sufficient to resolve the glycoproteins, the density of the bound IgG molecule was quite poor. To improve the density of the bound IgG, localized reconstruction of the i2 was performed. For this, sub-particles at the i2 were extracted and 3D classified into 10 classes without imposing any symmetry (C1) in Scipion<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. The best classes with well-defined structural features were pooled (289,759 sub-particles) and further refined in Relion<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> with C1 symmetry until the refinement converged. At this step, the estimated resolution according to GSFSC coefficient at 0.143 was 4.3â€‰Ã…. Although the quality of the map for the bound IgG improved in comparison to the icosahedral map, it was still not useful for atomic model fitting. Therefore, a mask was applied to the two IgG bound q3 spikes and another round of 3D classification within the masked volume into three classes were carried out in Relion. One class contained with well-defined features and this class (271,460 sub-particles) was further refined using C1 symmetry in Relion. This generated a map of the bound IgG of sufficient quality to begin atomic model fitting. No density was observed for either the hinge or Fc region of the antibody. The resolution of this final map was estimated to be 3.8â€‰Ã… according to the GSFSC coefficient at 0.143. Although the resolution estimation of the map is global, visual inspection revealed a resolution range across the map. Therefore, local resolution estimation of the map was carried out in Relion, which estimated the resolution of the epitope-paratope interaction surface to be around 5â€‰Ã…. The map was sharpened using Autosharpen in Phenix<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. The statistics of the data collection and structure reconstruction are listed in Supplementary TableÂ <a href="#MOESM1" class="usa-link">1</a>. A schematic of the data processing is described in Supplementary Fig.Â <a href="#MOESM1" class="usa-link">7</a>.</p></section><section id="Sec36"><h3 class="pmc_sec_title">Model building and fitting into the cryo-EM density map</h3>
<p id="Par55">The ectodomain models of EEEV E2 and E1 were obtained from the Protein Data Bank (PDB ID: 6MX4). A homology model of EEEV-373 was built using AlphaFold 3. Rigid body fitting of the ectodomains and the Fab arms of the IgG into the 3.8â€‰Ã… map corresponding to the two q3 spikes cross-linked by EEEV-373 were carried out in ChimeraX<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. The fitted models then were further refined against the map using the Real Space Refinement routine in Phenix<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. For the E2 and E1 ectodomains simulated annealing was included in the refinement protocol and the variable domains of the IgG were refined without simulated annealing. The resolution of the map corresponding to the constant domains were significantly worse than the rest of the map. Therefore, the constant domains were fitted as rigid bodies. The quality of the model geometry and the clash score were iteratively improved by a combination of Isolde<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, Coot<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> and real_space_refinement routine in Phenix. The viral surface area buried by EEEV-373 was calculated using ChimeraX. For fitting into the icosahedral map, the refined model of one of the q3 spikes in complex with the Fab arm of the IgG and a model of the ectodomain of the i3 E2/E1 heterodimer (PDB ID: 6MX4) were fitted in UCSF ChimeraX following <em>T</em>â€‰=â€‰4 quasi-symmetry of the asymmetric unit as rigid bodies. Then, the asymmetric unit around the fitted model was extracted using the phenix.map_box command. Only the i3 arm of the model was subjected to the Real_Space_Refine routine in Phenix using default settings against the extracted map corresponding to the asymmetric unit. The model quality was improved as described above. Finally, the refined model is again fitted as a rigid body in the icosahedral map in ChimeraX following <em>T</em>â€‰=â€‰4 quasi-symmetry.</p></section><section id="Sec37"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par56">All statistical analyses were performed using GraphPad Prism version 10 (GraphPad Software). Details about statistical analyses can be found in the figure legends. EC<sub>50</sub> and IC<sub>50</sub> values were calculated after log transformation of the concentration values and non-linear regression analysis using sigmoidal dose-response (variable slope).Â Survival curves were compared using the log-rank test with Bonferroni multiple comparison correction (Ï<sub>adj</sub> = 1-(1-Ï<sub>orig</sub>)<sup>n</sup>; n = number of comparisons).</p></section><section id="Sec38"><h3 class="pmc_sec_title">Inclusion and ethics</h3>
<p id="Par57">The research complied with all relevant ethical regulations as established by the institutional review boards associated with all authors.</p></section><section id="Sec39"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par58">Further information on research design is available in theÂ <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec40"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (163.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (388.2KB, pdf) </sup>
</div></div></section></section><section id="Sec41"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (742.2KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank R. Carnahan at Vanderbilt University for helpful discussions. We thank R. Nargi, C. Gainza, J. Rodriguez, L. Handal, S. Monroig, J. Martinez, E. Villalobos, S. Seeback, J. Reidy, R. Irving, R. Troseth, and L. Myers for technical support. We thank G. DeBellis, K. Moton, T. Martin, T. Hendy, H. Darling, A. Bunnell, B. Haseltine, N. Lee, N. Suazo, and J. Mitchell for administrative assistance. We thank the Purdue cryo-EM facility for equipment access and support. L.E.W. was supported by the NIH grants T32 HL069765 and F31 AI145189. This project received support from the NIAID grants R01 AI095366-10 (R.J.K.), U19 AI142790 (R.J.K. and J.E.C.), R01 AI172156 (J.E.C.), R01 AI41646 (W.B.K.), and the NIH/NIAID contract HHSN272201700041I, Task A51 (J.G.J.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>A.B. and L.E.W. contributed equally to this work. A.B. purified SINV/EEEV particles for complex formation with EEEV-373 and prepared the cryo-EM grids for data collection. A.B. and T.K. collected cryo-EM data and performed data analysis and processing. A.B. generated the reconstruction volumes, refined the atomic models, and performed the structural analyses. L.E.W. isolated, purified, and characterized EEEV-373 IgG, F(abâ€²)2, or Fab molecules. L.E.W. performed all biochemical, biophysical, and BSL-2 neutralization assays. D.S. prepared liposomes and DiD-labeled SINV/EEEV particles, performed the fusion inhibition assay, and interpreted the results. K.B. performed the EEEV mouse studies. G.B. provided and maintained cell culture used for SINV/EEEV particle generation. A.T. analyzed viral escape mutant sequences. C.S. generated the SINV/EEEV chimeric virus. J.G.J. planned and supervised the EEEV mouse studies. T.G. performed and W.B.K. supervised the BSL-3 neutralization assays. J.E.C. and R.J.K. obtained funding and provided supervision. A.B., L.E.W., J.E.C. and R.J.K. wrote and revised the manuscript with inputs from all the authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par59"><em>Nature Communications</em> thanks Scott Pegan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All relevant data are included within the manuscript. The source data underlying Figs.Â <a href="#Fig1" class="usa-link">1</a>, <a href="#Fig2" class="usa-link">2</a>, and <a href="#Fig7" class="usa-link">7</a> and Supplementary Figs.Â <a href="#MOESM1" class="usa-link">1</a>, <a href="#MOESM1" class="usa-link">3</a> are provided as a Source Data file. The cryo-EM maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-43980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-43980</a> (icosahedral reconstruction of SINV/EEEV particles in complex with EEEV-373 IgG1 molecules) and <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-43507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-43507</a> (localized reconstruction map of the q3 spikes crosslinked by EEEV-373). The atomic coordinates have been deposited in the Protein Data Bank under accession codes <a href="https://doi.org/10.2210/pdb9AY1/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9AY1</a> (atomic model of the asymmetric unit) and <a href="https://doi.org/10.2210/pdb8VSV/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8VSV</a> (atomic model corresponding to the localized reconstruction map). Previously published atomic models used and referred to in this study are: <a href="http://doi.org/10.2210/pdb6MX4/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6MX4</a> and <a href="http://doi.org/10.2210/pdb7CVZ/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7CVZ</a>. Materials described in this paper are available for distribution for nonprofit use using templated documents from Association of University Technology Managers â€œToolkit MTAsâ€, available at: <a href="https://autm.net/surveys-and-tools/agreements/material-transfer-agreements/mta-toolkit" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://autm.net/surveys-and-tools/agreements/material-transfer-agreements/mta-toolkit</a>. Further information or requests for reagents may be directed to and fulfilled by the Lead Contact: Dr. Richard J. Kuhn (kuhnr@purdue.edu).Â <a href="#Sec41" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par60">L.E.W. has served as a consultant for BigHat Biosciences. J.E.C. has served as a consultant for Luna Labs USA, Merck Sharp &amp; Dohme Corporation, Emergent Biosolutions, and BTG International Inc, is a former member of the Scientific Advisory Boards of Gigagen (Grifols) and Meissa Vaccines, and is founder of IDBiologics. The laboratory of J.E.C. received unrelated sponsored research agreements from AstraZeneca, Takeda Vaccines, and IDBiologics during the conduct of the study. Vanderbilt University has applied for a patent for the antibody studied in this paper.Â The remaining authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Abhishek Bandyopadhyay, Lauren E. Williamson.</p></div>
<div class="fn p" id="fn3"><p>These authors jointly supervised this work: James E. Crowe Jr., Richard J. Kuhn.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>James E. Crowe, Jr., Email: james.crowe@vumc.org</p>
<p>Richard J. Kuhn, Email: kuhnr@purdue.edu</p></section><section id="sec42"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-60505-x.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Crowe, J. E. Human antibodies for viral infections. <em>Annu Rev. Immunol.</em><strong>40</strong>, 349â€“386 (2022).
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-042718-041309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35113730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crowe,%20J.%20E.%20Human%20antibodies%20for%20viral%20infections.%20Annu%20Rev.%20Immunol.40,%20349%E2%80%93386%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Oliphant, T. &amp; Diamond, M. S. The molecular basis of antibody-mediated neutralization of West Nile virus. <em>Expert Opin. Biol. Ther.</em><strong>7</strong>, 885â€“892 (2007).
</cite> [<a href="https://doi.org/10.1517/14712598.7.6.885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17555373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oliphant,%20T.%20&amp;%20Diamond,%20M.%20S.%20The%20molecular%20basis%20of%20antibody-mediated%20neutralization%20of%20West%20Nile%20virus.%20Expert%20Opin.%20Biol.%20Ther.7,%20885%E2%80%93892%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Rey, F. A. &amp; Lok, S.-M. Common features of enveloped viruses and implications for immunogen design for next-generation vaccines. <em>Cell</em><strong>172</strong>, 1319â€“1334 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.02.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7112304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29522750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rey,%20F.%20A.%20&amp;%20Lok,%20S.-M.%20Common%20features%20of%20enveloped%20viruses%20and%20implications%20for%20immunogen%20design%20for%20next-generation%20vaccines.%20Cell172,%201319%E2%80%931334%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Sevvana, M. &amp; Kuhn, R. J. Mapping the diverse structural landscape of the flavivirus antibody repertoire. <em>Curr. Opin. Virol.</em><strong>45</strong>, 51â€“64 (2020).
</cite> [<a href="https://doi.org/10.1016/j.coviro.2020.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7746604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32801077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sevvana,%20M.%20&amp;%20Kuhn,%20R.%20J.%20Mapping%20the%20diverse%20structural%20landscape%20of%20the%20flavivirus%20antibody%20repertoire.%20Curr.%20Opin.%20Virol.45,%2051%E2%80%9364%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Fox, J. M. et al. Broadly neutralizing alphavirus antibodies bind an epitope on E2 and inhibit entry and egress. <em>Cell</em><strong>163</strong>, 1095â€“1107 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.10.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4659373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26553503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fox,%20J.%20M.%20et%20al.%20Broadly%20neutralizing%20alphavirus%20antibodies%20bind%20an%20epitope%20on%20E2%20and%20inhibit%20entry%20and%20egress.%20Cell163,%201095%E2%80%931107%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Hasan, S. S. et al. Cryo-EM structures of Eastern equine encephalitis virus reveal mechanisms of virus disassembly and antibody neutralization. <em>Cell Rep.</em><strong>25</strong>, 3136â€“3147.e5 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.11.067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6302666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30540945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hasan,%20S.%20S.%20et%20al.%20Cryo-EM%20structures%20of%20Eastern%20equine%20encephalitis%20virus%20reveal%20mechanisms%20of%20virus%20disassembly%20and%20antibody%20neutralization.%20Cell%20Rep.25,%203136%E2%80%933147.e5%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Jin, J. &amp; Simmons, G. Antiviral functions of monoclonal antibodies against chikungunya virus. <em>Viruses</em><strong>11</strong>, 305 (2019).
</cite> [<a href="https://doi.org/10.3390/v11040305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6520934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30925717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20J.%20&amp;%20Simmons,%20G.%20Antiviral%20functions%20of%20monoclonal%20antibodies%20against%20chikungunya%20virus.%20Viruses11,%20305%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zhang, S. et al. A Human antibody neutralizes different flaviviruses by using different mechanisms. <em>Cell Rep.</em><strong>31</strong>, 107584 (2020).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.107584" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32348755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20S.%20et%20al.%20A%20Human%20antibody%20neutralizes%20different%20flaviviruses%20by%20using%20different%20mechanisms.%20Cell%20Rep.31,%20107584%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Zhou, Q. F. et al. Structural basis of Chikungunya virus inhibition by monoclonal antibodies. <em>Proc. Natl. Acad. Sci. USA</em><strong>117</strong>, 27637â€“27645 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.2008051117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7959576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33087569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Q.%20F.%20et%20al.%20Structural%20basis%20of%20Chikungunya%20virus%20inhibition%20by%20monoclonal%20antibodies.%20Proc.%20Natl.%20Acad.%20Sci.%20USA117,%2027637%E2%80%9327645%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Armstrong, P. M. &amp; Andreadis, T. G. Eastern equine encephalitis virus â€” old enemy, new threat. <em>N. Engl. J. Med.</em><strong>368</strong>, 1670â€“1673 (2013).
</cite> [<a href="https://doi.org/10.1056/NEJMp1213696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23635048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Armstrong,%20P.%20M.%20&amp;%20Andreadis,%20T.%20G.%20Eastern%20equine%20encephalitis%20virus%20%E2%80%94%20old%20enemy,%20new%20threat.%20N.%20Engl.%20J.%20Med.368,%201670%E2%80%931673%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Ayres, J. C. &amp; Feemster, R. F. The sequelae of Eastern equine encephalomyelitis. <em>N. Engl. J. Med.</em><strong>240</strong>, 960â€“962 (1949).
</cite> [<a href="https://doi.org/10.1056/NEJM194906162402403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18144801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ayres,%20J.%20C.%20&amp;%20Feemster,%20R.%20F.%20The%20sequelae%20of%20Eastern%20equine%20encephalomyelitis.%20N.%20Engl.%20J.%20Med.240,%20960%E2%80%93962%20(1949)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Lindsey, N. P., Staples, J. E. &amp; Fischer, M. Eastern equine encephalitis virus in the United States, 2003â€“2016. <em>Am. J. Trop. Med. Hyg.</em><strong>98</strong>, 1472â€“1477 (2018).
</cite> [<a href="https://doi.org/10.4269/ajtmh.17-0927" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5953388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29557336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lindsey,%20N.%20P.,%20Staples,%20J.%20E.%20&amp;%20Fischer,%20M.%20Eastern%20equine%20encephalitis%20virus%20in%20the%20United%20States,%202003%E2%80%932016.%20Am.%20J.%20Trop.%20Med.%20Hyg.98,%201472%E2%80%931477%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Lindsey, N. P., Martin, S. W., Staples, J. E. &amp; Fischer, M. Notes from the field: multistate outbreak of Eastern equine encephalitis virus â€” United States, 2019. <em>MMWR Morb. Mortal. Wkly Rep.</em><strong>69</strong>, 50â€“51 (2020).
</cite> [<a href="https://doi.org/10.15585/mmwr.mm6902a4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6973353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31945032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lindsey,%20N.%20P.,%20Martin,%20S.%20W.,%20Staples,%20J.%20E.%20&amp;%20Fischer,%20M.%20Notes%20from%20the%20field:%20multistate%20outbreak%20of%20Eastern%20equine%20encephalitis%20virus%20%E2%80%94%20United%20States,%202019.%20MMWR%20Morb.%20Mortal.%20Wkly%20Rep.69,%2050%E2%80%9351%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Ronca, S. E., Dineley, K. T. &amp; Paessler, S. Neurological sequelae resulting from encephalitic alphavirus infection. <em>Front. Microbiol.</em><strong>7</strong>, 959 (2016).
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.00959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4913092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27379085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ronca,%20S.%20E.,%20Dineley,%20K.%20T.%20&amp;%20Paessler,%20S.%20Neurological%20sequelae%20resulting%20from%20encephalitic%20alphavirus%20infection.%20Front.%20Microbiol.7,%20959%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Sidwell, R. W. &amp; Smee, D. F. Viruses of the Bunya- and Togaviridae families: potential as bioterrorism agents and means of control. <em>Antivir. Res.</em><strong>57</strong>, 101â€“111 (2003).
</cite> [<a href="https://doi.org/10.1016/s0166-3542(02)00203-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12615306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sidwell,%20R.%20W.%20&amp;%20Smee,%20D.%20F.%20Viruses%20of%20the%20Bunya-%20and%20Togaviridae%20families:%20potential%20as%20bioterrorism%20agents%20and%20means%20of%20control.%20Antivir.%20Res.57,%20101%E2%80%93111%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Jose, J., Snyder, J. E. &amp; Kuhn, R. J. A structural and functional perspective of alphavirus replication and assembly. <em>Future Microbiol.</em><strong>4</strong>, 837â€“856 (2009).
</cite> [<a href="https://doi.org/10.2217/fmb.09.59" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2762864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19722838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jose,%20J.,%20Snyder,%20J.%20E.%20&amp;%20Kuhn,%20R.%20J.%20A%20structural%20and%20functional%20perspective%20of%20alphavirus%20replication%20and%20assembly.%20Future%20Microbiol.4,%20837%E2%80%93856%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Zhang, R. et al. 4.4 Ã… cryo-EM structure of an enveloped alphavirus Venezuelan equine encephalitis virus. <em>EMBO J.</em><strong>30</strong>, 3854â€“3863 (2011).
</cite> [<a href="https://doi.org/10.1038/emboj.2011.261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3173789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21829169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20R.%20et%20al.%204.4%20%C3%85%20cryo-EM%20structure%20of%20an%20enveloped%20alphavirus%20Venezuelan%20equine%20encephalitis%20virus.%20EMBO%20J.30,%203854%E2%80%933863%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Button, J. M., Qazi, S. A., Wang, J. C.-Y. &amp; Mukhopadhyay, S. Revisiting an old friend: new findings in alphavirus structure and assembly. <em>Curr. Opin. Virol.</em><strong>45</strong>, 25â€“33 (2020).
</cite> [<a href="https://doi.org/10.1016/j.coviro.2020.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7746636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32683295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Button,%20J.%20M.,%20Qazi,%20S.%20A.,%20Wang,%20J.%20C.-Y.%20&amp;%20Mukhopadhyay,%20S.%20Revisiting%20an%20old%20friend:%20new%20findings%20in%20alphavirus%20structure%20and%20assembly.%20Curr.%20Opin.%20Virol.45,%2025%E2%80%9333%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Mukhopadhyay, S. et al. Mapping the structure and function of the E1 and E2 glycoproteins in alphaviruses. <em>Structure</em><strong>14</strong>, 63â€“73 (2006).
</cite> [<a href="https://doi.org/10.1016/j.str.2005.07.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2757649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16407066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mukhopadhyay,%20S.%20et%20al.%20Mapping%20the%20structure%20and%20function%20of%20the%20E1%20and%20E2%20glycoproteins%20in%20alphaviruses.%20Structure14,%2063%E2%80%9373%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Gibbons, D. L. et al. Multistep regulation of membrane insertion of the fusion peptide of Semliki Forest virus. <em>J. Virol.</em><strong>78</strong>, 3312â€“3318 (2004).
</cite> [<a href="https://doi.org/10.1128/JVI.78.7.3312-3318.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC371068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15016852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibbons,%20D.%20L.%20et%20al.%20Multistep%20regulation%20of%20membrane%20insertion%20of%20the%20fusion%20peptide%20of%20Semliki%20Forest%20virus.%20J.%20Virol.78,%203312%E2%80%933318%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Holmes, A. C., Basore, K., Fremont, D. H. &amp; Diamond, M. S. A molecular understanding of alphavirus entry. <em>PLoS Pathog.</em><strong>16</strong>, e1008876 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1008876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7580943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33091085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holmes,%20A.%20C.,%20Basore,%20K.,%20Fremont,%20D.%20H.%20&amp;%20Diamond,%20M.%20S.%20A%20molecular%20understanding%20of%20alphavirus%20entry.%20PLoS%20Pathog.16,%20e1008876%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Voss, J. E. et al. Glycoprotein organization of chikungunya virus particles revealed by X-ray crystallography. <em>Nature</em><strong>468</strong>, 709â€“712 (2010).
</cite> [<a href="https://doi.org/10.1038/nature09555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21124458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voss,%20J.%20E.%20et%20al.%20Glycoprotein%20organization%20of%20chikungunya%20virus%20particles%20revealed%20by%20X-ray%20crystallography.%20Nature468,%20709%E2%80%93712%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Zhang, R. et al. Mxra8 is a receptor for multiple arthritogenic alphaviruses. <em>Nature</em><strong>557</strong>, 570â€“574 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0121-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5970976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29769725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20R.%20et%20al.%20Mxra8%20is%20a%20receptor%20for%20multiple%20arthritogenic%20alphaviruses.%20Nature557,%20570%E2%80%93574%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Basore, K. et al. Cryo-EM Structure of chikungunya virus in complex with the Mxra8 receptor. <em>Cell</em><strong>177</strong>, 1725â€“1737.e16 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7227486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31080061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Basore,%20K.%20et%20al.%20Cryo-EM%20Structure%20of%20chikungunya%20virus%20in%20complex%20with%20the%20Mxra8%20receptor.%20Cell177,%201725%E2%80%931737.e16%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Ma, H. et al. LDLRAD3 is a receptor for Venezuelan equine encephalitis virus. <em>Nature</em><strong>588</strong>, 308â€“314 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2915-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7769003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33208938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20H.%20et%20al.%20LDLRAD3%20is%20a%20receptor%20for%20Venezuelan%20equine%20encephalitis%20virus.%20Nature588,%20308%E2%80%93314%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Basore, K. et al. Structure of Venezuelan equine encephalitis virus in complex with the LDLRAD3 receptor. <em>Nature</em><strong>598</strong>, 672â€“676 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03963-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8550936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34646020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Basore,%20K.%20et%20al.%20Structure%20of%20Venezuelan%20equine%20encephalitis%20virus%20in%20complex%20with%20the%20LDLRAD3%20receptor.%20Nature598,%20672%E2%80%93676%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Chen, C.-L. et al. Cryo-EM structure of eastern equine encephalitis virus in complex with heparan sulfate analogues. <em>Proc. Natl. Acad. Sci. USA</em><strong>117</strong>, 8890â€“8899 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.1910670117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7183182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32245806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20C.-L.%20et%20al.%20Cryo-EM%20structure%20of%20eastern%20equine%20encephalitis%20virus%20in%20complex%20with%20heparan%20sulfate%20analogues.%20Proc.%20Natl.%20Acad.%20Sci.%20USA117,%208890%E2%80%938899%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Clark, L. E. et al. VLDLR and ApoER2 are receptors for multiple alphaviruses. <em>Nature</em><strong>602</strong>, 475â€“480 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-021-04326-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8808280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34929721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clark,%20L.%20E.%20et%20al.%20VLDLR%20and%20ApoER2%20are%20receptors%20for%20multiple%20alphaviruses.%20Nature602,%20475%E2%80%93480%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Yang, P. et al. Structural basis for VLDLR recognition by Eastern equine encephalitis virus. <em>Nat. Commun.</em><strong>15</strong>, 6548 (2024).</cite> [<a href="https://doi.org/10.1038/s41467-024-50887-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11297306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39095394/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ma, H. et al. The low-density lipoprotein receptor promotes infection of multiple encephalitic alphaviruses. <em>Nat. Commun.</em><strong>15</strong>, 246 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-023-44624-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10764363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38172096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20H.%20et%20al.%20The%20low-density%20lipoprotein%20receptor%20promotes%20infection%20of%20multiple%20encephalitic%20alphaviruses.%20Nat.%20Commun.15,%20246%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Adams, L. J. et al. Structural and functional basis of VLDLR usage by Eastern equine encephalitis virus. <em>Cell S</em><strong>0092</strong>, 01318â€“1 (2024).</cite> [<a href="https://doi.org/10.1016/j.cell.2023.11.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10843625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38176410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams,%20L.%20J.%20et%20al.%20Structural%20and%20functional%20basis%20of%20VLDLR%20usage%20by%20Eastern%20equine%20encephalitis%20virus.%20Cell%20S0092,%2001318%E2%80%931%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Kielian, M. Mechanisms of virus membrane fusion proteins. <em>Annu. Rev. Virol.</em><strong>1</strong>, 171â€“189 (2014).
</cite> [<a href="https://doi.org/10.1146/annurev-virology-031413-085521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26958720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kielian,%20M.%20Mechanisms%20of%20virus%20membrane%20fusion%20proteins.%20Annu.%20Rev.%20Virol.1,%20171%E2%80%93189%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Li, L., Jose, J., Xiang, Y., Kuhn, R. J. &amp; Rossmann, M. G. Structural changes of envelope proteins during alphavirus fusion. <em>Nature</em><strong>468</strong>, 705â€“708 (2010).
</cite> [<a href="https://doi.org/10.1038/nature09546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3057476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21124457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20L.,%20Jose,%20J.,%20Xiang,%20Y.,%20Kuhn,%20R.%20J.%20&amp;%20Rossmann,%20M.%20G.%20Structural%20changes%20of%20envelope%20proteins%20during%20alphavirus%20fusion.%20Nature468,%20705%E2%80%93708%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Earnest, J. T. et al. Neutralizing antibodies against Mayaro virus require Fc effector functions for protective activity. <em>J. Exp. Med.</em><strong>216</strong>, 2282â€“2301 (2019).
</cite> [<a href="https://doi.org/10.1084/jem.20190736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6781005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31337735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Earnest,%20J.%20T.%20et%20al.%20Neutralizing%20antibodies%20against%20Mayaro%20virus%20require%20Fc%20effector%20functions%20for%20protective%20activity.%20J.%20Exp.%20Med.216,%202282%E2%80%932301%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Jin, J. et al. Neutralizing monoclonal antibodies block chikungunya virus entry and release by targeting an epitope critical to viral pathogenesis. <em>Cell Rep.</em><strong>13</strong>, 2553â€“2564 (2015).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2015.11.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4720387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26686638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20J.%20et%20al.%20Neutralizing%20monoclonal%20antibodies%20block%20chikungunya%20virus%20entry%20and%20release%20by%20targeting%20an%20epitope%20critical%20to%20viral%20pathogenesis.%20Cell%20Rep.13,%202553%E2%80%932564%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Jin, J. et al. Neutralizing antibodies inhibit chikungunya virus budding at the plasma membrane. <em>Cell Host Microbe</em><strong>24</strong>, 417â€“428.e5 (2018).
</cite> [<a href="https://doi.org/10.1016/j.chom.2018.07.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30146390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20J.%20et%20al.%20Neutralizing%20antibodies%20inhibit%20chikungunya%20virus%20budding%20at%20the%20plasma%20membrane.%20Cell%20Host%20Microbe24,%20417%E2%80%93428.e5%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Long, F. et al. Cryo-EM structures elucidate neutralizing mechanisms of anti-chikungunya human monoclonal antibodies with therapeutic activity. <em>Proc. Natl. Acad. Sci. USA</em><strong>112</strong>, 13898â€“13903 (2015).
</cite> [<a href="https://doi.org/10.1073/pnas.1515558112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4653152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26504196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Long,%20F.%20et%20al.%20Cryo-EM%20structures%20elucidate%20neutralizing%20mechanisms%20of%20anti-chikungunya%20human%20monoclonal%20antibodies%20with%20therapeutic%20activity.%20Proc.%20Natl.%20Acad.%20Sci.%20USA112,%2013898%E2%80%9313903%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Powell, L. A. et al. Human mAbs broadly protect against arthritogenic alphaviruses by recognizing conserved elements of the Mxra8 receptor-binding site. <em>Cell Host Microbe</em><strong>28</strong>, 699â€“711.e7 (2020).
</cite> [<a href="https://doi.org/10.1016/j.chom.2020.07.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7666055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32783883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Powell,%20L.%20A.%20et%20al.%20Human%20mAbs%20broadly%20protect%20against%20arthritogenic%20alphaviruses%20by%20recognizing%20conserved%20elements%20of%20the%20Mxra8%20receptor-binding%20site.%20Cell%20Host%20Microbe28,%20699%E2%80%93711.e7%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Williamson, L. E. et al. Human antibodies protect against aerosolized Eastern equine encephalitis virus infection. <em>Cell</em><strong>183</strong>, 1884â€“1900.e23 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7806206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33301709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williamson,%20L.%20E.%20et%20al.%20Human%20antibodies%20protect%20against%20aerosolized%20Eastern%20equine%20encephalitis%20virus%20infection.%20Cell183,%201884%E2%80%931900.e23%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Williamson, L. E. et al. Structural constraints link differences in neutralization potency of human anti-Eastern equine encephalitis virus monoclonal antibodies. <em>Proc. Natl. Acad. Sci. USA</em><strong>120</strong>, e22131690120 (2023).</cite> [<a href="https://doi.org/10.1073/pnas.2213690120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10068833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36961925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williamson,%20L.%20E.%20et%20al.%20Structural%20constraints%20link%20differences%20in%20neutralization%20potency%20of%20human%20anti-Eastern%20equine%20encephalitis%20virus%20monoclonal%20antibodies.%20Proc.%20Natl.%20Acad.%20Sci.%20USA120,%20e22131690120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Kim, A. S. et al. Pan-protective anti-alphavirus human antibodies target a conserved E1 protein epitope. <em>Cell</em><strong>184</strong>, 4414â€“4429.e19 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8382027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34416146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20A.%20S.%20et%20al.%20Pan-protective%20anti-alphavirus%20human%20antibodies%20target%20a%20conserved%20E1%20protein%20epitope.%20Cell184,%204414%E2%80%934429.e19%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Williamson, L. E. et al. Therapeutic alphavirus cross-reactive E1 human antibodies inhibit viral egress. <em>Cell</em><strong>184</strong>, 4430â€“4446.e22 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.07.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8418820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34416147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williamson,%20L.%20E.%20et%20al.%20Therapeutic%20alphavirus%20cross-reactive%20E1%20human%20antibodies%20inhibit%20viral%20egress.%20Cell184,%204430%E2%80%934446.e22%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Kim, A. S. et al. Protective antibodies against Eastern equine encephalitis virus bind to epitopes in domains A and B of the E2 glycoprotein. <em>Nat. Microbiol.</em><strong>4</strong>, 187â€“197 (2019).
</cite> [<a href="https://doi.org/10.1038/s41564-018-0286-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6294662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30455470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20A.%20S.%20et%20al.%20Protective%20antibodies%20against%20Eastern%20equine%20encephalitis%20virus%20bind%20to%20epitopes%20in%20domains%20A%20and%20B%20of%20the%20E2%20glycoprotein.%20Nat.%20Microbiol.4,%20187%E2%80%93197%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Fibriansah, G. et al. Dengue virus. Cryo-EM structure of an antibody that neutralizes dengue virus type 2 by locking E protein dimers. <em>Science</em><strong>349</strong>, 88â€“91 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa8651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4672004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26138979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fibriansah,%20G.%20et%20al.%20Dengue%20virus.%20Cryo-EM%20structure%20of%20an%20antibody%20that%20neutralizes%20dengue%20virus%20type%202%20by%20locking%20E%20protein%20dimers.%20Science349,%2088%E2%80%9391%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Sharma, A. et al. The epitope arrangement on flavivirus particles contributes to Mab C10â€™s extraordinary neutralization breadth across Zika and dengue viruses. <em>Cell</em><strong>184</strong>, 6052â€“6066.e18 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.11.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8724787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34852239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20A.%20et%20al.%20The%20epitope%20arrangement%20on%20flavivirus%20particles%20contributes%20to%20Mab%20C10%E2%80%99s%20extraordinary%20neutralization%20breadth%20across%20Zika%20and%20dengue%20viruses.%20Cell184,%206052%E2%80%936066.e18%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Williams, A. F. &amp; Barclay, A. N. The immunoglobulin superfamily-domains for cell surface recognition. <em>Annu Rev. Immunol.</em><strong>6</strong>, 381â€“405 (1988).
</cite> [<a href="https://doi.org/10.1146/annurev.iy.06.040188.002121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3289571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williams,%20A.%20F.%20&amp;%20Barclay,%20A.%20N.%20The%20immunoglobulin%20superfamily-domains%20for%20cell%20surface%20recognition.%20Annu%20Rev.%20Immunol.6,%20381%E2%80%93405%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Chmielewski, D. et al. Chikungunya virus assembly and budding visualized in situ using cryogenic electron tomography. <em>Nat. Microbiol.</em><strong>7</strong>, 1270â€“1279 (2022).
</cite> [<a href="https://doi.org/10.1038/s41564-022-01164-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9930444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35773421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chmielewski,%20D.%20et%20al.%20Chikungunya%20virus%20assembly%20and%20budding%20visualized%20in%20situ%20using%20cryogenic%20electron%20tomography.%20Nat.%20Microbiol.7,%201270%E2%80%931279%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Dejnirattisai, W. et al. A new class of highly potent, broadly neutralizing antibodies isolated from viremic patients infected with dengue virus. <em>Nat. Immunol.</em><strong>16</strong>, 170â€“177 (2015).
</cite> [<a href="https://doi.org/10.1038/ni.3058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4445969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25501631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dejnirattisai,%20W.%20et%20al.%20A%20new%20class%20of%20highly%20potent,%20broadly%20neutralizing%20antibodies%20isolated%20from%20viremic%20patients%20infected%20with%20dengue%20virus.%20Nat.%20Immunol.16,%20170%E2%80%93177%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Hasan, S. S. et al. A human antibody against Zika virus crosslinks the E protein to prevent infection. <em>Nat. Commun.</em><strong>8</strong>, 14722 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms14722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5356071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28300075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hasan,%20S.%20S.%20et%20al.%20A%20human%20antibody%20against%20Zika%20virus%20crosslinks%20the%20E%20protein%20to%20prevent%20infection.%20Nat.%20Commun.8,%2014722%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Sapparapu, G. et al. Neutralizing human antibodies prevent Zika virus replication and fetal disease in mice. <em>Nature</em><strong>540</strong>, 443â€“447 (2016).
</cite> [<a href="https://doi.org/10.1038/nature20564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5583716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27819683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sapparapu,%20G.%20et%20al.%20Neutralizing%20human%20antibodies%20prevent%20Zika%20virus%20replication%20and%20fetal%20disease%20in%20mice.%20Nature540,%20443%E2%80%93447%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gardner, C. L. et al. Heparan sulfate binding by natural Eastern equine encephalitis viruses promotes neurovirulence. <em>Proc. Natl. Acad. Sci. USA</em><strong>108</strong>, 16026â€“16031 (2011).
</cite> [<a href="https://doi.org/10.1073/pnas.1110617108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3179095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21896745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gardner,%20C.%20L.%20et%20al.%20Heparan%20sulfate%20binding%20by%20natural%20Eastern%20equine%20encephalitis%20viruses%20promotes%20neurovirulence.%20Proc.%20Natl.%20Acad.%20Sci.%20USA108,%2016026%E2%80%9316031%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Ko, S. Y. et al. A virus-like particle vaccine prevents equine encephalitis virus infection in nonhuman primates. <em>Sci. Transl. Med.</em><strong>11</strong>, eaav3113 (2019).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.aav3113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31092692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ko,%20S.%20Y.%20et%20al.%20A%20virus-like%20particle%20vaccine%20prevents%20equine%20encephalitis%20virus%20infection%20in%20nonhuman%20primates.%20Sci.%20Transl.%20Med.11,%20eaav3113%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Kafai, N. M. et al. Neutralizing antibodies protect mice against Venezuelan equine encephalitis virus aerosol challenge. <em>J. Exp. Med.</em><strong>219</strong>, e20212532 (2022).
</cite> [<a href="https://doi.org/10.1084/jem.20212532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9195047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35297953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kafai,%20N.%20M.%20et%20al.%20Neutralizing%20antibodies%20protect%20mice%20against%20Venezuelan%20equine%20encephalitis%20virus%20aerosol%20challenge.%20J.%20Exp.%20Med.219,%20e20212532%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Scheres, S. H. W. &amp; Chen, S. Prevention of overfitting in cryo-EM structure determination. <em>Nat. Methods</em><strong>9</strong>, 853â€“854 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.2115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4912033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22842542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scheres,%20S.%20H.%20W.%20&amp;%20Chen,%20S.%20Prevention%20of%20overfitting%20in%20cryo-EM%20structure%20determination.%20Nat.%20Methods9,%20853%E2%80%93854%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>de la Rosa-TrevÃ­n, J. M. et al. Scipion: a software framework toward integration, reproducibility and validation in 3D electron microscopy. <em>J. Struct. Biol.</em><strong>195</strong>, 93â€“99 (2016).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2016.04.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27108186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20la%20Rosa-Trev%C3%ADn,%20J.%20M.%20et%20al.%20Scipion:%20a%20software%20framework%20toward%20integration,%20reproducibility%20and%20validation%20in%203D%20electron%20microscopy.%20J.%20Struct.%20Biol.195,%2093%E2%80%9399%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. <em>Elife</em><strong>7</strong>, e42166 (2018).
</cite> [<a href="https://doi.org/10.7554/eLife.42166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6250425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30412051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zivanov,%20J.%20et%20al.%20New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3.%20Elife7,%20e42166%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in <em>Phenix</em>. <em>Acta Crystallogr. D. Struct. Biol.</em><strong>75</strong>, 861â€“877 (2019).
</cite> [<a href="https://doi.org/10.1107/S2059798319011471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6778852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31588918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liebschner,%20D.%20et%20al.%20Macromolecular%20structure%20determination%20using%20X-rays,%20neutrons%20and%20electrons:%20recent%20developments%20in%20Phenix.%20Acta%20Crystallogr.%20D.%20Struct.%20Biol.75,%20861%E2%80%93877%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. <em>Protein Sci.</em><strong>30</strong>, 70â€“82 (2021).
</cite> [<a href="https://doi.org/10.1002/pro.3943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7737788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32881101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pettersen,%20E.%20F.%20et%20al.%20UCSF%20ChimeraX:%20structure%20visualization%20for%20researchers,%20educators,%20and%20developers.%20Protein%20Sci.30,%2070%E2%80%9382%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Croll, T. I. <em>ISOLDE</em>: a physically realistic environment for model building into low-resolution electron-density maps. <em>Acta Crystallogr. D. Struct. Biol.</em><strong>74</strong>, 519â€“530 (2018).
</cite> [<a href="https://doi.org/10.1107/S2059798318002425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Croll,%20T.%20I.%20ISOLDE:%20a%20physically%20realistic%20environment%20for%20model%20building%20into%20low-resolution%20electron-density%20maps.%20Acta%20Crystallogr.%20D.%20Struct.%20Biol.74,%20519%E2%80%93530%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Emsley, P. &amp; Cowtan, K. <em>Coot</em>: model-building tools for molecular graphics. <em>Acta Crystallogr. D. Biol. Crystallogr.</em><strong>60</strong>, 2126â€“2132 (2004).
</cite> [<a href="https://doi.org/10.1107/S0907444904019158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emsley,%20P.%20&amp;%20Cowtan,%20K.%20Coot:%20model-building%20tools%20for%20molecular%20graphics.%20Acta%20Crystallogr.%20D.%20Biol.%20Crystallogr.60,%202126%E2%80%932132%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (163.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (388.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365132/bin/41467_2025_60505_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (742.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All relevant data are included within the manuscript. The source data underlying Figs.Â <a href="#Fig1" class="usa-link">1</a>, <a href="#Fig2" class="usa-link">2</a>, and <a href="#Fig7" class="usa-link">7</a> and Supplementary Figs.Â <a href="#MOESM1" class="usa-link">1</a>, <a href="#MOESM1" class="usa-link">3</a> are provided as a Source Data file. The cryo-EM maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-43980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-43980</a> (icosahedral reconstruction of SINV/EEEV particles in complex with EEEV-373 IgG1 molecules) and <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-43507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-43507</a> (localized reconstruction map of the q3 spikes crosslinked by EEEV-373). The atomic coordinates have been deposited in the Protein Data Bank under accession codes <a href="https://doi.org/10.2210/pdb9AY1/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9AY1</a> (atomic model of the asymmetric unit) and <a href="https://doi.org/10.2210/pdb8VSV/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8VSV</a> (atomic model corresponding to the localized reconstruction map). Previously published atomic models used and referred to in this study are: <a href="http://doi.org/10.2210/pdb6MX4/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6MX4</a> and <a href="http://doi.org/10.2210/pdb7CVZ/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7CVZ</a>. Materials described in this paper are available for distribution for nonprofit use using templated documents from Association of University Technology Managers â€œToolkit MTAsâ€, available at: <a href="https://autm.net/surveys-and-tools/agreements/material-transfer-agreements/mta-toolkit" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://autm.net/surveys-and-tools/agreements/material-transfer-agreements/mta-toolkit</a>. Further information or requests for reagents may be directed to and fulfilled by the Lead Contact: Dr. Richard J. Kuhn (kuhnr@purdue.edu).Â <a href="#Sec41" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-60505-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_60505.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.2Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365132/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365132/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365132%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365132/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365132/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365132/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830099/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365132/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830099/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365132/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365132/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="uKxidOz6tbmCknGPeoBVEaRf30p7ooma6iGS7alHMdbRTd6Z1rO1nrzlsnD0LAxQ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
