
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Development of a duplex chamber digital PCR to quantify twelve genetically modified maize events - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A0EC8AF1CA9305A0EC00618D0E91.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="gmcrops">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377078/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="GM Crops &amp; Food">
<meta name="citation_title" content="Development of a duplex chamber digital PCR to quantify twelve genetically modified maize events">
<meta name="citation_author" content="Eun-Ha Kim">
<meta name="citation_author_institution" content="Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea">
<meta name="citation_author" content="Youn-Sung Cho">
<meta name="citation_author_institution" content="Planning &amp; Coordination Division, National Institute of Agricultural Sciences, Jeonju, Korea">
<meta name="citation_author" content="Byeori Kim">
<meta name="citation_author_institution" content="Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea">
<meta name="citation_author" content="Yelinn Yoo">
<meta name="citation_author_institution" content="Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea">
<meta name="citation_author" content="Jae-In Lee">
<meta name="citation_author_institution" content="Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea">
<meta name="citation_author" content="Young Soon Kim">
<meta name="citation_author_institution" content="Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea">
<meta name="citation_author" content="Tae-Sung Park">
<meta name="citation_author_institution" content="Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="527">
<meta name="citation_doi" content="10.1080/21645698.2025.2548053">
<meta name="citation_pmid" content="40842205">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377078/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377078/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377078/pdf/KGMC_16_2548053.pdf">
<meta name="description" content="Accurate quantification of genetically modified organisms (GMOs) is essential for regulatory compliance, especially under threshold-based labeling policies. In this study, we developed and validated twelve event-specific duplex chamber- or ...">
<meta name="og:title" content="Development of a duplex chamber digital PCR to quantify twelve genetically modified maize events">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Accurate quantification of genetically modified organisms (GMOs) is essential for regulatory compliance, especially under threshold-based labeling policies. In this study, we developed and validated twelve event-specific duplex chamber- or ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377078/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12377078">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1080/21645698.2025.2548053"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/KGMC_16_2548053.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377078%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12377078/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12377078/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377078/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-gmcrops.png" alt="GM Crops &amp; Food logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to GM Crops &amp; Food" title="Link to GM Crops &amp; Food" shape="default" href="https://doi.org/10.1080/21645698.2025.2548053" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">GM Crops Food</button></div>. 2025 Aug 21;16(1):527–538. doi: <a href="https://doi.org/10.1080/21645698.2025.2548053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1080/21645698.2025.2548053</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22GM%20Crops%20Food%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22GM%20Crops%20Food%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22GM%20Crops%20Food%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22GM%20Crops%20Food%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Development of a duplex chamber digital PCR to quantify twelve genetically modified maize events</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20EH%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Eun-Ha Kim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Eun-Ha Kim</span></h3>
<div class="p">
<sup>a</sup>Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div>Conceptualization, Data curation, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20EH%22%5BAuthor%5D" class="usa-link"><span class="name western">Eun-Ha Kim</span></a>
</div>
</div>
<sup>a,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cho%20YS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Youn-Sung Cho</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Youn-Sung Cho</span></h3>
<div class="p">
<sup>b</sup>Planning &amp; Coordination Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div>Conceptualization, Data curation, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cho%20YS%22%5BAuthor%5D" class="usa-link"><span class="name western">Youn-Sung Cho</span></a>
</div>
</div>
<sup>b,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Byeori Kim</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Byeori Kim</span></h3>
<div class="p">
<sup>a</sup>Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div>Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Byeori Kim</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yoo%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yelinn Yoo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yelinn Yoo</span></h3>
<div class="p">
<sup>a</sup>Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div>Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yoo%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yelinn Yoo</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JI%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jae-In Lee</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jae-In Lee</span></h3>
<div class="p">
<sup>a</sup>Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div>Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JI%22%5BAuthor%5D" class="usa-link"><span class="name western">Jae-In Lee</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20YS%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Young Soon Kim</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Young Soon Kim</span></h3>
<div class="p">
<sup>a</sup>Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div>Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20YS%22%5BAuthor%5D" class="usa-link"><span class="name western">Young Soon Kim</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20TS%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Tae-Sung Park</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Tae-Sung Park</span></h3>
<div class="p">
<sup>a</sup>Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div>Conceptualization, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20TS%22%5BAuthor%5D" class="usa-link"><span class="name western">Tae-Sung Park</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff0001">
<sup>a</sup>Biosafety Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div id="aff0002">
<sup>b</sup>Planning &amp; Coordination Division, National Institute of Agricultural Sciences, Jeonju, Korea</div>
<div class="author-notes p">
<div class="fn" id="an0001">
<sup>✉</sup><p class="display-inline">CONTACT Tae-Sung Park <span>jowonoon@korea.kr</span> Biosafety Division, National Institute of Agricultural Sciences, 370, Nongsaengmyeong-ro, Deokjin-gu, Jeonju-si, Jeonbuk-do, South Korea</p>
</div>
<div class="fn" id="an0002">
<sup>*</sup><p class="display-inline">Eun-Ha Kim and Youn-Sung Cho contributed equally to this work and are co-first authors.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Eun-Ha Kim</span></strong>: <span class="role">Conceptualization, Data curation, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Youn-Sung Cho</span></strong>: <span class="role">Conceptualization, Data curation, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Byeori Kim</span></strong>: <span class="role">Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yelinn Yoo</span></strong>: <span class="role">Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jae-In Lee</span></strong>: <span class="role">Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Young Soon Kim</span></strong>: <span class="role">Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Tae-Sung Park</span></strong>: <span class="role">Conceptualization, Supervision, Writing – review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 21; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). Published with license by Taylor &amp; Francis Group, LLC.</div>
<p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. The terms on which this article has been published allow the posting of the Accepted Manuscript in a repository by the author(s) or with their consent.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12377078  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40842205/" class="usa-link">40842205</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Accurate quantification of genetically modified organisms (GMOs) is essential for regulatory compliance, especially under threshold-based labeling policies. In this study, we developed and validated twelve event-specific duplex chamber- or chip-based digital PCR (cdPCR) methods using microfluidic array plates to quantify GM maize events approved in South Korea. In contrast to conventional real-time PCR, the cdPCR approach allows for absolute quantification without standard curve calibration and incorporates event-specific zygosity ratio correction to improve accuracy of the measurement. The method was evaluated at GMO content levels of 0.9%, 3.0%, and 5.0%. It demonstrated high sensitivity and robustness, with trueness, precision, and reproducibility satisfying the minimum performance criteria recommended by international guidelines. Comparative analysis with real-time quantitative PCR (qPCR) showed comparable accuracy; however, cdPCR provided advantages in cost-efficiency and operational simplicity. These findings support the applicability of duplex cdPCR as a practical and reliable tool for GMO quantification, particularly in national regulatory laboratories and for enforcement of labeling thresholds such as Korea’s 3.0% rule.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>KEYWORDS:</strong> Chamber digital PCR, genetically modified maize, quantitative method</p></section></section><section id="s0001"><h2 class="pmc_sec_title">Introduction</h2>
<p>The advancement of genetically modified (GM) crops has been significantly driven by scientific and technological developments, resulting in traits such as pest resistance, herbicide tolerance, resilience to environmental stress, and enhanced nutritional profiles.<sup><a href="#cit0001" class="usa-link" aria-describedby="cit0001">1</a>,<a href="#cit0002" class="usa-link" aria-describedby="cit0002">2</a></sup> In 2024, GM crops were cultivated on approximately 209.8 million hectares globally. Among these, GM soybeans accounted for 50.0% of the total cultivation area, followed by GM maize (32.5%), GM cotton (11.8%), and GM canola (5.0%).<sup><a href="#cit0003" class="usa-link" aria-describedby="cit0003">3</a></sup></p>
<p>To ensure consumer choice, transparency, and harmonized trade practices, many countries have implemented labeling regulations that require quantification of GM content based on predefined thresholds. For example, South Korea enforces a GMO labeling threshold of 3.0% for food,<sup><a href="#cit0004" class="usa-link" aria-describedby="cit0004">4</a></sup> while the European Union and Japan apply thresholds of 0.9% and 5.0%, respectively.<sup><a href="#cit0005" class="usa-link" aria-describedby="cit0005">5–7</a></sup></p>
<p>Accurate and reliable quantification methods are essential for verifying compliance with such labeling requirements. GM content in food or feed is generally expressed as the ratio of transgene (GM-specific target) to an endogenous reference gene. Currently, real-time quantitative PCR (qPCR) is the most widely used technique for GMO quantification.<sup><a href="#cit0008" class="usa-link" aria-describedby="cit0008">8–12</a></sup> However, qPCR has inherent limitations, including its dependence on standard curves and vulnerability to amplification efficiency variations and PCR inhibitors.<sup><a href="#cit0013" class="usa-link" aria-describedby="cit0013">13–15</a></sup></p>
<p>As an alternative, digital PCR (dPCR) has emerged as a next-generation method for absolute quantification of nucleic acids. In dPCR, the reaction mixture is partitioned into thousands of independent reactions, and after amplification, each partition is classified as positive or negative. The absolute quantity of the target is then calculated using Poisson statistics.<sup><a href="#cit0016" class="usa-link" aria-describedby="cit0016">16</a>,<a href="#cit0017" class="usa-link" aria-describedby="cit0017">17</a></sup> One major advantage of dPCR over qPCR is that it enables absolute quantification without the need for standard curve calibration, thereby improving reproducibility and reducing operator dependency, particularly when working with low target concentrations.<sup><a href="#cit0018" class="usa-link" aria-describedby="cit0018">18</a>,<a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a></sup> Due to these advantages, dPCR has been widely applied to the detection and quantification of GMOs in recent years.</p>
<p>dPCR platforms are broadly classified into droplet-based dPCR (ddPCR) and chamber- or chip-based dPCR (cdPCR) systems.<sup><a href="#cit0016" class="usa-link" aria-describedby="cit0016">16</a>,<a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a>,<a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a></sup> ddPCR offers high partitioning resolution and enhanced sensitivity for detecting low-copy-number targets. It also enables flexible multiplexing through amplitude-based fluorescence clustering.<sup><a href="#cit0015" class="usa-link" aria-describedby="cit0015">15</a>,<a href="#cit0021" class="usa-link" aria-describedby="cit0021">21</a>,<a href="#cit0022" class="usa-link" aria-describedby="cit0022">22</a></sup> By contrast, cdPCR platforms simplify operation by eliminating droplet generation and emulsification steps. Their pre-defined microchamber structure ensures uniform partition volumes, improves thermal uniformity, and minimizes cross-contamination risks. Notably, cdPCR systems support image-based fluorescence detection, allowing precise duplex or multiplex quantification with reduced signal overlap.<sup><a href="#cit0016" class="usa-link" aria-describedby="cit0016">16</a></sup></p>
<p>Employing these platform-specific advantages, various dPCR assays have been developed for the simultaneous quantification of multiple transgenic elements, demonstrating high accuracy, sensitivity, and wide dynamic range.<sup><a href="#cit0015" class="usa-link" aria-describedby="cit0015">15</a>,<a href="#cit0023" class="usa-link" aria-describedby="cit0023">23–25</a></sup> In support of this progress, the European Network of GMO Laboratories and other expert groups have established performance criteria and validation guidelines specifically tailored for dPCR-based GMO quantification.<sup><a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a>,<a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a>,<a href="#cit0026" class="usa-link" aria-describedby="cit0026">26</a></sup></p>
<p>In South Korea, the safety assessment and management of GMOs are regulated under the Act on Transboundary Movements of Living Modified Organisms.<sup><a href="#cit0027" class="usa-link" aria-describedby="cit0027">27</a></sup> The Rural Development Administration is responsible for conducting environmental risk assessments for agricultural GMOs. To support this regulatory framework, the Biosafety Division of the National Institute of Agricultural Sciences (NIAS) verifies detection and quantification methods using qPCR and dPCR techniques for GM crops approved domestically. These validations follow internationally recognized protocols, including those provided by the European Union Reference Laboratory for GM Food and Feed (EURL-GMFF).<sup><a href="#cit0009" class="usa-link" aria-describedby="cit0009">9</a>,<a href="#cit0012" class="usa-link" aria-describedby="cit0012">12</a>,<a href="#cit0026" class="usa-link" aria-describedby="cit0026">26</a>,<a href="#cit0028" class="usa-link" aria-describedby="cit0028">28–30</a></sup></p>
<p>The objective of this study is to develop and validate a duplex cd PCR method for the absolute quantification of twelve GM maize events approved in South Korea. The selection of the 12 GM maize events in this study was based on their regulatory approval for food, feed, or industrial use in South Korea and their relevance to ongoing domestic monitoring programs. The assay aims to support regulatory enforcement of the 3.0% labeling threshold for GMOs<sup><a href="#cit0004" class="usa-link" aria-describedby="cit0004">4</a></sup> by enabling precise quantification without calibration curves and by incorporating experimentally determined zygosity ratios for each GM event. We applied the QuantStudio Absolute Q™ system, which simplifies the dPCR workflow by using pre-manufactured microfluidic chamber plates. Furthermore, our method was evaluated through blind sample testing at regulatory threshold levels (0.9%, 3.0%, and 5.0%) using both cdPCR and qPCR. The results demonstrate that duplex cdPCR is a robust and practical approach for accurate GMO quantification in support of national monitoring and inspection efforts.</p></section><section id="s0002"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="s0002-s2001"><h3 class="pmc_sec_title">Plant Materials</h3>
<p>Twelve GM maize events – DAS01131-3, DAS40278-9, DP004114-3, DP023211-2, DP051291-1, DP098140-6, DP202216-6, MON87403-1, MON87411-9, MON87429-9, MON87460-4, and MON88017-3—and their corresponding non-GM reference materials were provided by the respective developers for in-house method verification conducted at the Biosafety Division of the NIAS, South Korea. Detailed information on each GM maize event, including the approval year in South Korea, intended use (food, feed, or industrial), trait characteristics, and developer, is provided in Supplemental Table S1.</p></section><section id="s0002-s2002"><h3 class="pmc_sec_title">Genomic DNA Extraction</h3>
<p>Maize seeds (both GM and non-GM) were ground into a fine powder using a mortar and pestle under liquid nitrogen. Genomic DNA was extracted and purified using the Maxwell RSC PureFood GMO and Authentication Kit (Promega, Madison, WI, USA), following the manufacturer’s instructions. DNA concentration and purity were assessed using a NanoDrop One spectrophotometer (Thermo Fisher Scientific, Waltham, MA, USA), and double-stranded DNA content was quantified using a Qubit 4 Fluorometer (Thermo Fisher Scientific).</p></section><section id="s0002-s2003"><h3 class="pmc_sec_title">Primers and Probes</h3>
<p>Event-specific primers and probes, as well as those targeting the endogenous reference gene <em>high mobility group a</em> (<em>hmg</em>, GenBank accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/AJ131373.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AJ131373.1</a>), were provided by the respective applicants in accordance with the national regulatory requirements under the LMO ACT of South Korea, which obliges GM developers to submit molecular characterization and detection information for official method verification. The Sequences and target regions are listed in <a href="#t0001" class="usa-link">Table 1</a>. They are also publicly available in international databases such as International Service for the Acquisition of Agri-biotech Applications (ISAAA) and EUginus. For certain GM maize events such as DP004114–3 and DP023211–2, the primers and probes used in this study differ from those specified in the EURL-GMFF validated methods (e.g., QT-EVE-ZM-035 and QT-TAX-ZM-008). Probes targeting GM events were labeled with 6-carboxyfluorescein (FAM), while <em>hmg</em> probes were labeled with either FAM or VIC. The 3′ ends of the probes were modified with either TAMRA or non-fluorescent quencher-minor groove bindeer (NFQ-MGB). All oligonucleotides were synthesized by Bioneer (Daejeon, South Korea).</p>
<section class="tw xbox font-sm" id="t0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Primers and probes used for GM maize event-specific targets and the endogenous reference gene (<em>hmg</em>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1">Event name</th>
<th colspan="2" align="center" rowspan="1">Primers/probes sequence (5’-3’)</th>
<th align="center" colspan="1" rowspan="1">
<sup>a</sup>Length (bp)</th>
</tr></thead>
<tbody>
<tr>
<td colspan="2" align="left" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3</td>
<td align="left" colspan="1" rowspan="1">F- CTA AGA GCT AAG ATT GCG CGG<br><em>R</em>- TTC GGG CCT AAC TTT TGG TG<br><em>p</em>- [6-FAM]-ACA TAT TTT TTG AGG ATA ACA GCA-[NFQ-MGB]</td>
<td align="center" colspan="1" rowspan="1">98</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS40278" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS40278</a>–9</td>
<td align="left" colspan="1" rowspan="1">F- CAC GAA CCA TTG AGT TAC AAT C<br><em>R</em>- TGG TTC ATT GTA TTC TGG CTT TG<br><em>p</em>- [6-FAM]- AGC TAA CCT TCA TTG TAT TCC G-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">98</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS59122" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS59122</a>–7</td>
<td align="left" colspan="1" rowspan="1">F- GGG ATA AGC AAG TAA AAG CGC TC<br><em>R</em>- CCT TAA TTC TCC GCT CAT GAT CAG<br><em>p</em>- [6-FAM]- TTT AAA CTG AAG GCG GGA AAC GAC AA-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">86</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">DP 004114–3</td>
<td align="left" colspan="1" rowspan="1">F- CGT TTG TAG CAC TTG CAC GTA GT<br><em>R</em>- GGT AAC CGC TCT TCC AGT TGA A<br><em>p</em>- [6-FAM]- AAG CTT CAA CAC AGA TC-[NFQ-MGB]</td>
<td align="center" colspan="1" rowspan="1">90</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">DP023211–2</td>
<td align="left" colspan="1" rowspan="1">F- CAT TTT GGA TTG TAA TAT GTG TAC CTC A<br><em>R</em>- CTA GCT CGA CTA GTT AGT TAG ACG CG<br><em>p</em>- [6-FAM]- TTA AAT CTG ACG TGA GGC GC-[NFA-MGB]</td>
<td align="center" colspan="1" rowspan="1">92</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">DP098140–6</td>
<td align="left" colspan="1" rowspan="1">F- GTG TGT ATG TCT CTT TGC TTG GTC TT<br><em>R</em>- GAT TGT CGT TTC CCG CCT TC<br><em>p</em>- [6-FAM]- CTC TAT CGA TCC CCC TCT TTG ATA GTT TAA ACT-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">80</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">DP202216–6</td>
<td align="left" colspan="1" rowspan="1">F- CCA TCT GAG GTC TGC ACT CTC AC<br><em>R</em>- CTC CGC TCA TGA TCA GAT TGT C<br><em>p</em>- [6-FAM]- CAA CAC ACT CAA ACA CTG ATA G-[NFQ-MGB]</td>
<td align="center" colspan="1" rowspan="1">105</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">MON87403–1</td>
<td align="left" colspan="1" rowspan="1">F- CTT TCT TTT TCT CCA TAT TGA CCA TCA TAC<br><em>R</em>- TAC TCC GGA ATG AGT GCT CTG TAT C<br><em>p</em>- [6-FAM]- TCA TTG CGA TCC ACA TTT CCC TAC ATG G-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">88</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">MON87411–9</td>
<td align="left" colspan="1" rowspan="1">F- CTC TGT AAC AGA AAA CAC CAT CTA GAG<br><em>R</em>- ACA AAA GTG AAC TAG TTC TAG GGT AGA T<br><em>p</em>- [6-FAM]- CCG CGT TTA AAC TAT CAG TGT TTA GAG AAT-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">109</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">MON87429–9</td>
<td align="left" colspan="1" rowspan="1">F- CGA GAC AGA CTC AAT GTA TCC GAG ATA CTC<br><em>R</em>- CCA TCA TAC TCA TTG CTG ATC CAT GTA<br><em>p</em>- [6-FAM]- TCC CGG ACA TGA AAC CAA ACA AGA GTG GTC-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">116</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">MON87460–4</td>
<td align="left" colspan="1" rowspan="1">F- CAC GTT GAA GGA AAA TGG ATT G<br><em>R</em>- TCG CGA TCC TCC TCA AAG AC<br><em>p</em>- [6-FAM]- AGG GAG TAT GTA GAT AAA TTT TCA AAG CGT TAG ACG GC-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">82</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">MON88017–3</td>
<td align="left" colspan="1" rowspan="1">F- GAG CAG GAC CTG CAG AAG CT<br><em>R</em>- TCC GGA GTT GAC CAT CCA<br><em>p</em>- [6-FAM]- TCC CGC CTT CAG TTT AAA CAG AGT CGG GT-[TAMRA]</td>
<td align="center" colspan="1" rowspan="1">94</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">
<em>hmg</em> (endogenous reference gene)</td>
<td align="left" colspan="1" rowspan="1">F- TTG GAC TAG AAA TCT CGT GCT GA<br><em>R</em>- GCT ACA TAG GGA GCC TTG TCC T<br><em>p</em>- [VIC]-CAA TCC ACA CAA ACG CAC GCG TA-[NFQ-MGB]</td>
<td align="center" colspan="1" rowspan="1">79</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tfn0001"><p><sup>a</sup>Length denotes the amplicon size in base pairs. In primer/probe sequence, F, R, and P indicate forward primer, reverse primer, and probe, respectively. TAMRA, 6-carboxytetramethylrhodamin NFQ-MGB, non fluorescent quencher-minor groove binder.</p></div></div></section></section><section id="s0002-s2004"><h3 class="pmc_sec_title">Real-Time qPCR Analysis for DP023211–2</h3>
<p>Quantitative PCR (qPCR) was conducted using the QuantStudio 1 Real-Time PCR Instrument (Applied Biosystems by Thermo Fisher, Foster City, CA, USA) to establish calibration curves for the DP023211–2 event. Each 25 μL reaction contained 12.5 μL of 2X Qplex Master Mix (CellSafe, Yougin, South Korea), primers, probes, 10 μL of template DNA, and nuclease-free water. The reagents are detailed in Supplemental Table S2. Two zygosity ratios were applied to calibrate the DNA mixtures: 0.42 (provided by the applicant) and 0.13 (experimentally determined by cdPCR in this study), to simulate 10% GM content based on copy number ratios. The calibration curves were established on at five samples with four experimental replicates. The first point of the calibration curve (S1) was established for a sample containing 10% maize DP023211–2 DNA in a total of 300 ng of non-GM maize DNA. Standards S2 to S5 are prepared by serial dilutions (dilution factor 4 for S2, S4 and S5, and dilution factor 3 for S3). The corresponding amount of DNA in reaction and copy numbers for standard curves are listed in Supplemental Table S3.<sup><a href="#cit0031" class="usa-link" aria-describedby="cit0031">31</a></sup> Thermal cycling conditions were 95°C for 10 min, followed by 45 cycles of 95°C for 15 s and 60°C for 1 min. The same thermal protocol was used for both the DP023211–2 and <em>hmg</em> assays. Blind samples containing 0.9%, 3.0%, and 5.0% GM content were prepared by mixing DP023211–2 DNA with non-GM maize DNA at the corresponding copy number ratios. Copy numbers were calculated from Ct values based on the standard curves. GMO content was expressed as the ratio of DP023211–2 to <em>hmg</em> copy numbers. Accuracy was evaluated as relative bias, and precision was assessed as the relative standard deviation (RSD) from four technical replicates.</p></section><section id="s0002-s2005"><h3 class="pmc_sec_title">cdPCR Analysis</h3>
<p>Simplex and duplex cdPCR assays were performed using the QuantStudio Absolute Q™ Digital PCR System (Applied Biosystems by Thermo Fisher) with 16-well microfluidic array plates (MAP). Reaction mixtures followed the manufacturer’s protocol. For simplex assays, each 9 μL reaction included 2 μL of 5X Absolute Q™ DNA Digital PCR Master Mix, 5 μL of template DNA (1 ng/µl), 0.9 μL of primer mix (10 μM), 0.25 μL of probe (10 μM), and 1.4 μL of distilled water. For duplex assays, the same master mix and DNA volumes were used, with two sets of primers and probes (total 2.3 μL), and 0.7 μL of water. Each well was loaded with 9 μL of reaction mixture and overlaid with 15 μL of separation oil (Thermo Fisher Scientific, A52730), followed by placement of a sealing gasket. Each well contained 20,480 fixed microchambers. Thermal cycling was carried out at 96°C for 5 min, followed by 40 cycles of 96°C for 5 s and 60°C for 15 s. Fluorescence signals were analyzed using the QuantStudio 3D Digital PCR instrument and QuantStudio Analysis Suite Cloud Software. Zygosity ratios for each GM event were calculated using duplex cdPCR by measuring the ratio of event-specific target copy number to <em>hmg</em> copy number in GM DNA samples. Blind DNA mixtures at 0.9%, 3.0%, and 5.0% GM content were analyzed to evaluate the method’s performance in quantifying low-level GMO content using cdPCR. Four experimental replications were run for each cdPCR analysis.</p></section></section><section id="s0003"><h2 class="pmc_sec_title">Results and Discussion</h2>
<section id="s0003-s2001"><h3 class="pmc_sec_title">Comparison of Simplex and Duplex cdPCR Analysis</h3>
<p>GMO quantification is based on the ratio of transgene to endogenous gene copy numbers. Therefore, simultaneous detection within a single reaction mixture using duplex PCR is both practical and cost-effective. To evaluate this, we compared the performance of simplex and duplex cdPCR for two GM maize events: <a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3 and DP910521–2. In the simplex assay, the FAM reporter dye was used to detect both the transgene and the <em>hmg</em> endogenous gene. The measured copy numbers for <a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3 and DP910521–2 transgenes were 439.09 and 427.45 copies/μL, respectively, while <em>hmg</em> copy numbers were 1048.41 and 1174.39 copies/μL (<a href="#t0002" class="usa-link">Table 2</a>). To assess any dye-related bias, <em>hmg</em> quantification was performed using both FAM and VIC dyes. The results showed no statistically significant difference between the two fluorophores (FAM, 1048.41 copies/μL; VIC, 1077.4 copies/μL), as determined by a Student’s t-test (<em>p</em> &gt; .05), indicating that dye selection did not influence <em>hmg</em> quantification.</p>
<section class="tw xbox font-sm" id="t0002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Quantitative results of simplex and duplex cdPCR assays for <a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3 and DP051291–2 event<sup>a</sup>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1">Event</th>
<th align="center" colspan="1" rowspan="1">System</th>
<th align="center" colspan="1" rowspan="1">Target</th>
<th align="center" colspan="1" rowspan="1">Dye</th>
<th align="center" colspan="1" rowspan="1">Total</th>
<th align="center" colspan="1" rowspan="1">Positive</th>
<th align="center" colspan="1" rowspan="1">Conc. Cp/µl</th>
<th align="center" colspan="1" rowspan="1">SD</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="5" align="left" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3</td>
<td align="left" colspan="1" rowspan="1">simplex</td>
<td align="left" colspan="1" rowspan="1">GM</td>
<td align="left" colspan="1" rowspan="1">FAM</td>
<td align="center" colspan="1" rowspan="1">20408</td>
<td align="center" colspan="1" rowspan="1">3526</td>
<td align="center" colspan="1" rowspan="1">439.09</td>
<td align="center" colspan="1" rowspan="1">20.88</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">simplex</td>
<td align="left" colspan="1" rowspan="1"><em>hmg</em></td>
<td align="left" colspan="1" rowspan="1">FAM</td>
<td align="center" colspan="1" rowspan="1">20451.8</td>
<td align="center" colspan="1" rowspan="1">7448</td>
<td align="center" colspan="1" rowspan="1">1048.41</td>
<td align="center" colspan="1" rowspan="1">37.59</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">simplex</td>
<td align="left" colspan="1" rowspan="1"><em>hmg</em></td>
<td align="left" colspan="1" rowspan="1">VIC</td>
<td align="center" colspan="1" rowspan="1">20432.5</td>
<td align="center" colspan="1" rowspan="1">7604</td>
<td align="center" colspan="1" rowspan="1">1077.4</td>
<td align="center" colspan="1" rowspan="1">19.0</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">duplex</td>
<td align="left" colspan="1" rowspan="1">GM</td>
<td align="left" colspan="1" rowspan="1">FAM</td>
<td rowspan="2" align="center" colspan="1">20424.5</td>
<td align="center" colspan="1" rowspan="1">3453</td>
<td align="center" colspan="1" rowspan="1">428.74</td>
<td align="center" colspan="1" rowspan="1">6.86</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>hmg</em></td>
<td align="left" colspan="1" rowspan="1">VIC</td>
<td align="center" colspan="1" rowspan="1">7455</td>
<td align="center" colspan="1" rowspan="1">1051.3</td>
<td align="center" colspan="1" rowspan="1">20.50</td>
</tr>
<tr>
<td rowspan="5" align="left" colspan="1">DP051291–2</td>
<td align="left" colspan="1" rowspan="1">simplex</td>
<td align="left" colspan="1" rowspan="1">GM</td>
<td align="left" colspan="1" rowspan="1">FAM</td>
<td align="center" colspan="1" rowspan="1">20428.8</td>
<td align="center" colspan="1" rowspan="1">3443</td>
<td align="center" colspan="1" rowspan="1">427.45</td>
<td align="center" colspan="1" rowspan="1">18.68</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">simplex</td>
<td align="left" colspan="1" rowspan="1"><em>hmg</em></td>
<td align="left" colspan="1" rowspan="1">FAM</td>
<td align="center" colspan="1" rowspan="1">20407.3</td>
<td align="center" colspan="1" rowspan="1">8135</td>
<td align="center" colspan="1" rowspan="1">1174.39</td>
<td align="center" colspan="1" rowspan="1">18.82</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">simplex</td>
<td align="left" colspan="1" rowspan="1"><em>hmg</em></td>
<td align="left" colspan="1" rowspan="1">VIC</td>
<td align="center" colspan="1" rowspan="1">20437.3</td>
<td align="center" colspan="1" rowspan="1">8152</td>
<td align="center" colspan="1" rowspan="1">1177.98</td>
<td align="center" colspan="1" rowspan="1">24.56</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">duplex</td>
<td align="left" colspan="1" rowspan="1">GM</td>
<td align="left" colspan="1" rowspan="1">FAM</td>
<td rowspan="2" align="center" colspan="1">20421.3</td>
<td align="center" colspan="1" rowspan="1">3329</td>
<td align="center" colspan="1" rowspan="1">411.74</td>
<td align="center" colspan="1" rowspan="1">21.25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>hmg</em></td>
<td align="left" colspan="1" rowspan="1">VIC</td>
<td align="center" colspan="1" rowspan="1">8038</td>
<td align="center" colspan="1" rowspan="1">1157.12</td>
<td align="center" colspan="1" rowspan="1">22.24</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tfn0002"><p><sup>a</sup>Each value represents the average of four experimental replications, except for SD. SD, standard deviation of Conc. Cp/µl (concentration of copy/µl).</p></div></div></section><p>Accurate digital PCR requires efficient partitioning. The MAP 16 Digital PCR Plate used in this study contains 20,480 fixed microchambers per well. The observed positive chamber counts exceeded 20,400, indicating even partitioning. In duplex cdPCR, FAM was used for transgene detection and VIC for <em>hmg</em>. The resulting copy numbers were statistically comparable to those obtained from simplex assays, with no significant difference observed based on Student’s t-test (<em>p</em> &gt; .05), confirming that amplification efficiency was not influenced by dye type or reaction format. These findings validate the use of duplex cdPCR for GMO quantification.</p></section><section id="s0003-s2002"><h3 class="pmc_sec_title">Determination of Zygosity Ratio in GM Maize Events Using cdPCR</h3>
<p>Determination of the zygosity ratio of the non-GM maize is critical for expressing GM content as a haploid genome copy ratio, as emphasized in EU guide lines.<sup><a href="#cit0032" class="usa-link" aria-describedby="cit0032">32–34</a></sup> Maize has a complex tissue structure with variable DNA contributions from maternal and paternal genomes, depending on the hybrid type. The zygosity ratio can vary depending on the donor parent of the transgene and specific seed tissue composition, such as the embryo, endosperm, and seed coat.<sup><a href="#cit0035" class="usa-link" aria-describedby="cit0035">35</a></sup> Corbisier et al.<sup><a href="#cit0034" class="usa-link" aria-describedby="cit0034">34</a></sup> reported that the copy number ratios between GM event-specific targets and taxon-specific reference genes varied notably among 17 certified GM maize reference materials using ddPCR. The observed conversion factors ranged from 0.34 to 0.68, depending on the parental origin of the transgene (male vs. female donor), endosperm contribution, and potential differences in DNA extractability from seed tissues. In line with these findings, our study also showed the zygosity ratio ranged from 0.134 (DP023211–3) to 0.625 (DP202216–6) (<a href="#t0003" class="usa-link">Table 3</a>), indicating hemizygous GM maize materials.</p>
<section class="tw xbox font-sm" id="t0003"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Zygosity ratio of GM event-specific and endogenous <em>hmg</em> targets in positive control samples as determined by cdPCR<sup>a</sup>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" align="center" colspan="1">Event</th>
<th rowspan="2" align="center" colspan="1">Total</th>
<th colspan="3" align="center" rowspan="1">Event specific<hr>
</th>
<th colspan="3" align="center" rowspan="1">Endogenous Reference<hr>
</th>
<th rowspan="2" align="center" colspan="1">Zygosity<br>ratio</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">Positive</th>
<th align="center" colspan="1" rowspan="1">Conc.cp/µl.</th>
<th align="center" colspan="1" rowspan="1">SD</th>
<th align="center" colspan="1" rowspan="1">Positive</th>
<th align="center" colspan="1" rowspan="1">Conc.cp/µl.</th>
<th align="center" colspan="1" rowspan="1">SD</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3</td>
<td align="center" colspan="1" rowspan="1">20424.75</td>
<td align="center" colspan="1" rowspan="1">3402</td>
<td align="center" colspan="1" rowspan="1">421.77</td>
<td align="center" colspan="1" rowspan="1">10.38</td>
<td align="center" colspan="1" rowspan="1">7495</td>
<td align="center" colspan="1" rowspan="1">1058.35</td>
<td align="center" colspan="1" rowspan="1">11.69</td>
<td align="center" colspan="1" rowspan="1">0.399</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS40278" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS40278</a>–9</td>
<td align="center" colspan="1" rowspan="1">20441</td>
<td align="center" colspan="1" rowspan="1">5227</td>
<td align="center" colspan="1" rowspan="1">683.55</td>
<td align="center" colspan="1" rowspan="1">4.34</td>
<td align="center" colspan="1" rowspan="1">10521</td>
<td align="center" colspan="1" rowspan="1">1674.37</td>
<td align="center" colspan="1" rowspan="1">59.25</td>
<td align="center" colspan="1" rowspan="1">0.408</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS59122" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS59122</a>–7</td>
<td align="center" colspan="1" rowspan="1">20198.75</td>
<td align="center" colspan="1" rowspan="1">4606</td>
<td align="center" colspan="1" rowspan="1">598.82</td>
<td align="center" colspan="1" rowspan="1">42.58</td>
<td align="center" colspan="1" rowspan="1">9232</td>
<td align="center" colspan="1" rowspan="1">1413.77</td>
<td align="center" colspan="1" rowspan="1">80.47</td>
<td align="center" colspan="1" rowspan="1">0.424</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DP004114–3</td>
<td align="center" colspan="1" rowspan="1">20413</td>
<td align="center" colspan="1" rowspan="1">5524</td>
<td align="center" colspan="1" rowspan="1">732.52</td>
<td align="center" colspan="1" rowspan="1">104.5</td>
<td align="center" colspan="1" rowspan="1">12023</td>
<td align="center" colspan="1" rowspan="1">2058.52</td>
<td align="center" colspan="1" rowspan="1">43.68</td>
<td align="center" colspan="1" rowspan="1">0.356</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DP023211–2</td>
<td align="center" colspan="1" rowspan="1">20406</td>
<td align="center" colspan="1" rowspan="1">2024</td>
<td align="center" colspan="1" rowspan="1">241.77</td>
<td align="center" colspan="1" rowspan="1">5.23</td>
<td align="center" colspan="1" rowspan="1">11039</td>
<td align="center" colspan="1" rowspan="1">1802.55</td>
<td align="center" colspan="1" rowspan="1">40.66</td>
<td align="center" colspan="1" rowspan="1">0.134</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DP098140–6</td>
<td align="center" colspan="1" rowspan="1">20431.5</td>
<td align="center" colspan="1" rowspan="1">6824</td>
<td align="center" colspan="1" rowspan="1">941.1</td>
<td align="center" colspan="1" rowspan="1">41.67</td>
<td align="center" colspan="1" rowspan="1">10724</td>
<td align="center" colspan="1" rowspan="1">1722.73</td>
<td align="center" colspan="1" rowspan="1">16.81</td>
<td align="center" colspan="1" rowspan="1">0.546</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DP202216–6</td>
<td align="center" colspan="1" rowspan="1">20446.75</td>
<td align="center" colspan="1" rowspan="1">8510</td>
<td align="center" colspan="1" rowspan="1">1245.86</td>
<td align="center" colspan="1" rowspan="1">24.78</td>
<td align="center" colspan="1" rowspan="1">11803</td>
<td align="center" colspan="1" rowspan="1">1993.23</td>
<td align="center" colspan="1" rowspan="1">40.51</td>
<td align="center" colspan="1" rowspan="1">0.625</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MON87403–1</td>
<td align="center" colspan="1" rowspan="1">20457.75</td>
<td align="center" colspan="1" rowspan="1">3984</td>
<td align="center" colspan="1" rowspan="1">501.36</td>
<td align="center" colspan="1" rowspan="1">12.23</td>
<td align="center" colspan="1" rowspan="1">6347</td>
<td align="center" colspan="1" rowspan="1">859.81</td>
<td align="center" colspan="1" rowspan="1">13.99</td>
<td align="center" colspan="1" rowspan="1">0.583</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MON87411–9</td>
<td align="center" colspan="1" rowspan="1">20395.75</td>
<td align="center" colspan="1" rowspan="1">3953</td>
<td align="center" colspan="1" rowspan="1">498.64</td>
<td align="center" colspan="1" rowspan="1">1.75</td>
<td align="center" colspan="1" rowspan="1">6167</td>
<td align="center" colspan="1" rowspan="1">833.55</td>
<td align="center" colspan="1" rowspan="1">10.58</td>
<td align="center" colspan="1" rowspan="1">0.598</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MON87429–9</td>
<td align="center" colspan="1" rowspan="1">20423.25</td>
<td align="center" colspan="1" rowspan="1">3301</td>
<td align="center" colspan="1" rowspan="1">408.26</td>
<td align="center" colspan="1" rowspan="1">35.19</td>
<td align="center" colspan="1" rowspan="1">5942</td>
<td align="center" colspan="1" rowspan="1">795.8</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">0.513</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MON87460–4</td>
<td align="center" colspan="1" rowspan="1">20409.75</td>
<td align="center" colspan="1" rowspan="1">3129</td>
<td align="center" colspan="1" rowspan="1">385.24</td>
<td align="center" colspan="1" rowspan="1">8.23</td>
<td align="center" colspan="1" rowspan="1">6593</td>
<td align="center" colspan="1" rowspan="1">903.02</td>
<td align="center" colspan="1" rowspan="1">14.6</td>
<td align="center" colspan="1" rowspan="1">0.427</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MON88017–3</td>
<td align="center" colspan="1" rowspan="1">20415</td>
<td align="center" colspan="1" rowspan="1">3247</td>
<td align="center" colspan="1" rowspan="1">400.99</td>
<td align="center" colspan="1" rowspan="1">8.91</td>
<td align="center" colspan="1" rowspan="1">5775</td>
<td align="center" colspan="1" rowspan="1">769.75</td>
<td align="center" colspan="1" rowspan="1">10.2</td>
<td align="center" colspan="1" rowspan="1">0.521</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tfn0003"><p><sup>a</sup>Each value represents the average of four experimental replications, except for SD. SD, standard deviation of Conc. Cp/µl (concentration of copy/µl).</p></div></div></section><p>Our results were compared with zygosity ratio values available in the applicant’s reference protocols and the validation report of the GM maize event by the EURL-GMFF (Supplemental Table S4). Among the 12 GM maize events tested, the most notable discrepancies between experimentally determined and reference zygosity ratios were observed in DP023211–2 and DP202216–6. For DP023211–2, the duplex cdPCR method yielded a zygosity ratio of 0.134 (mean of four replicates), while the applicant’s documentation reported a value of 0.42. Similarly, DP202216–6 showed experimentally determined ratios of 0.625, compared to values of 0.99 reported by the EURL-GMFF. In these cases, the reference values were provided as single figures without replicate data or accompanying measures of variability (e.g., standard deviation), precluding formal statistical comparison (e.g., t-test). Nevertheless, the relative differences – ranging from 36% to over 68%—are substantial and analytically meaningful. These discrepancies likely reflect differences in seed batch composition, parental genotype, hybrid type, or tissue-specific DNA content, all of which can influence the effective zygosity ratio observed in quantitative PCR-based analyses.<sup><a href="#cit0034" class="usa-link" aria-describedby="cit0034">34</a></sup> To account for such variation, this study applied duplex cdPCR to determine zygosity ratios directly from seed-derived DNA, enabling a material-specific correction for each event. This approach ensures more accurate quantification when CRMs are unavailable, not matrix-matched, or exhibit batch-to-batch variability.</p></section><section id="s0003-s2003"><h3 class="pmc_sec_title">Quantification of DP02311–3 Blind Samples by qPCR</h3>
<p>To further investigate the impact of the zygosity ratio on quantification, we established a standard curve using qPCR with DNA templates adjusted to match the expected copy ratios corresponding to the two distinct zygosity ratios of DP023211–2. The average slope of the standard curve should range from −3.1 to −3.6, corresponding to an efficiency between 90% and 110%, and the R<sup>2</sup> coefficient should be ≥ 0.99, according to the European Network of GMO Laboratories (ENGL) method acceptance criteria (2008). When using the zygosity ratio of 0.43 provided by the applicants, the slope, R<sup>2</sup>, and efficiency values met the ENGL criteria. However, the Ct value showed high variation at the lowest concentration (S5) of DP023211–2 (Supplemental Fig. S1D). In contrast, when using the zygosity ratio of 0.13 measured via cdPCR in this study, the slope, R<sup>2</sup>, and efficiency values met the ENGL criteria, and the Ct value exhibited less variation across the five concentrations (<a href="#f0001" class="usa-link">Figure 1(B,E)</a>). These results indicate that a zygosity ratio of 0.43 led to an underestimation of GM quantity, affecting PCR amplification efficiency, particularly at the lowest concentration.
</p>
<figure class="fig xbox font-sm" id="f0001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8359/12377078/65ff1b4a7c08/KGMC_A_2548053_F0001_OC.jpg" loading="lazy" height="423" width="800" alt="Figure 1."></p>
<div class="p text-right font-secondary"><a href="figure/f0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Detection of DP023211–2 blind samples using qPCR analysis. (A) Amplification plots for calibrators and blind samples. (B) Standard curves for <em>hmg</em> with concentrations ranging from 572 to 109,890 copies. (C) DP023211–2 blind samples with expected transgene contents of 0.9%, 3.0%, and 5.0% on the <em>hmg</em> standard curve (D) standard curves for DP023211–2 with concentrations ranging from 57 to 10,989 copies. (E) DP023211–2 blind samples expected transgene contents of 0.9%, 3.0%, and 5.0% on the DP023211–2 standard curve. A zygosity ratio of 0.13 was applied to calculate the copy number ratio. Data were means from four replicates.</p></figcaption></figure><p>The DP02311–3 event-specific and <em>hmg</em> qPCR analyses were performed to quantify blind DNA samples with GM levels of 5.0%, 3.0%, and 0.9% (expressed as the % copy number ratio of DP02311–3 to <em>hmg</em>). The measured copy numbers of the DP02311–3 event-specific target and <em>hmg</em>, as well as the % copy number ratio of DP02311–3 to <em>hmg</em>, are shown in <a href="#t0004" class="usa-link">Table 4</a>. The measured GMO % was determined as the ratio of the average copy number of DP02311–3 to that of <em>hmg</em>.</p>
<section class="tw xbox font-sm" id="t0004"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Comparison of quantitative performance between qPCR and cdPCR for GM content analysis of DP023211–2 maize event<sup>a</sup>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
</colgroup>
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"> </th>
<th colspan="5" align="center" rowspan="1">qPCR<hr>
</th>
<th colspan="5" align="center" rowspan="1">cdPCR<hr>
</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"> </th>
<th colspan="2" align="center" rowspan="1">Copy numbers<hr>
</th>
<th colspan="3" align="center" rowspan="1">Content, %<hr>
</th>
<th colspan="2" align="center" rowspan="1">Copy numbers/µl<hr>
</th>
<th colspan="3" align="center" rowspan="1">Content, %<hr>
</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">DP023211–2</th>
<th align="center" colspan="1" rowspan="1">GM</th>
<th align="center" colspan="1" rowspan="1"><em>hmg</em></th>
<th align="center" colspan="1" rowspan="1">Value</th>
<th align="center" colspan="1" rowspan="1">Bias</th>
<th align="left" colspan="1" rowspan="1">RSD</th>
<th align="center" colspan="1" rowspan="1">GM</th>
<th align="center" colspan="1" rowspan="1"><em>hmg</em></th>
<th colspan="1" rowspan="1">Value</th>
<th align="center" colspan="1" rowspan="1">Bias</th>
<th align="center" colspan="1" rowspan="1">RSD</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">0.9%</td>
<td align="center" colspan="1" rowspan="1">456.3</td>
<td align="center" colspan="1" rowspan="1">52691.7</td>
<td align="center" colspan="1" rowspan="1">0.87</td>
<td align="center" colspan="1" rowspan="1">3.33</td>
<td align="left" colspan="1" rowspan="1">1.61</td>
<td align="center" colspan="1" rowspan="1">2.3</td>
<td align="center" colspan="1" rowspan="1">1848.7</td>
<td colspan="1" rowspan="1">0.93</td>
<td align="center" colspan="1" rowspan="1">3.06</td>
<td align="center" colspan="1" rowspan="1">17.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3.0%</td>
<td align="center" colspan="1" rowspan="1">1672.9</td>
<td align="center" colspan="1" rowspan="1">55540.2</td>
<td align="center" colspan="1" rowspan="1">3.01</td>
<td align="center" colspan="1" rowspan="1">0.33</td>
<td align="left" colspan="1" rowspan="1">2.27</td>
<td align="center" colspan="1" rowspan="1">6.7</td>
<td align="center" colspan="1" rowspan="1">1726.2</td>
<td colspan="1" rowspan="1">2.90</td>
<td align="center" colspan="1" rowspan="1">3.25</td>
<td align="center" colspan="1" rowspan="1">17.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5.0%</td>
<td align="center" colspan="1" rowspan="1">2817.5</td>
<td align="center" colspan="1" rowspan="1">55507.3</td>
<td align="center" colspan="1" rowspan="1">5.09</td>
<td align="center" colspan="1" rowspan="1">1.80</td>
<td align="left" colspan="1" rowspan="1">4.63</td>
<td align="center" colspan="1" rowspan="1">11.1</td>
<td align="center" colspan="1" rowspan="1">1736.2</td>
<td colspan="1" rowspan="1">4.78</td>
<td align="center" colspan="1" rowspan="1">4.50</td>
<td align="center" colspan="1" rowspan="1">6.8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tfn0004"><p><sup>a</sup>Each value is an average of four experimental replications, except for Bias and RSD. Bias, deviation from expected value; RSD, relative standard deviation.</p></div></div></section><p>According to the minimum performance requirements for GMO analytical methods, the measured value must have a relative bias of ± 25% (trueness) and precision in terms of a relative standard deviation (RSD) of ≤ 25%.<sup><a href="#cit0026" class="usa-link" aria-describedby="cit0026">26</a>,<a href="#cit0028" class="usa-link" aria-describedby="cit0028">28</a></sup> The measured % copy number ratios were 0.87%, 3.01%, and 5.09%, which were close to the expected values (<a href="#t0004" class="usa-link">Table 4</a>). For all blind samples, the % bias and % RSD of the measured copy number ratios were below 5%. The measured copy numbers and Ct values of the DP02311–3 event-specific target in blind DNA samples were well-aligned with the standard curves of <em>hmg</em> and DP02311–3 (<a href="#f0001" class="usa-link">Figure 1(C,E)</a>), indicating that the qPCR assays precisely measured the % copy number ratio in blind samples.</p></section><section id="s0003-s2004"><h3 class="pmc_sec_title">Quantification of DP02311–3 Blind Samples by cdPCR</h3>
<p>Unlike qPCR, digital PCR (dPCR) enables absolute quantification of the target without the need for a standard curve. Several studies demonstrated that ddPCR platforms provide high analytical performance for regulatory testing of GM events in maize, soybean, and canola.<sup><a href="#cit0015" class="usa-link" aria-describedby="cit0015">15</a>,<a href="#cit0022" class="usa-link" aria-describedby="cit0022">22</a>,<a href="#cit0036" class="usa-link" aria-describedby="cit0036">36–39</a></sup> For instance, Dobnik et al.<sup><a href="#cit0021" class="usa-link" aria-describedby="cit0021">21</a></sup> developed two multiplex ddPCR assays to quantify up to twelve GM maize events simultaneously, achieving limit of quantification (LOQ) as low as 0.058% and bias within ± 25%. Li et al.<sup><a href="#cit0038" class="usa-link" aria-describedby="cit0038">38</a></sup> developed an event-specific duplex ddPCR in DBN9936 maize, demonstrating accurate quantification across blind matrix and genomic DNA samples with RSD and bias within ± 25%. While the ddPCR method typically require emulsification steps and specialized droplet readers, in this study the cdPCR method utilizes pre-fabricated microfluidic plates and eliminates droplet generation, thereby reducing operational complexity.</p>
<p>In this study, we established a duplex cdPCR method for quantifying DP02311–3 blind samples. <a href="#f0002" class="usa-link">Figure 2</a> shows the amplification results for DP02311–3 (<a href="#f0002" class="usa-link">Figure 2(A)</a>) and <em>hmg</em> (<a href="#f0002" class="usa-link">Figure 2(B)</a>) at 100%, 5.0%, 3.0%, and 0.9% GMO content. The measured GM % was determined as the ratio of the average copy number of DP02311–3 to that of <em>hmg</em>, divided by the zygosity ratio (<a href="#t0004" class="usa-link">Table 4</a>). The measured % copy number ratios were 0.93%, 2.90%, and 4.78%, with a % bias of less than 5.0%. The % RSD of the measured copy number ratios for 0.9%, 3.0%, and 5.0% was 17.0%, 17.6%, and 6.8%, respectively, which fell within the acceptable range of ≤ 25%.<sup><a href="#cit0028" class="usa-link" aria-describedby="cit0028">28</a></sup>
</p>
<figure class="fig xbox font-sm" id="f0002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8359/12377078/951226cb118a/KGMC_A_2548053_F0002_OC.jpg" loading="lazy" height="269" width="800" alt="Figure 2."></p>
<div class="p text-right font-secondary"><a href="figure/f0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quantification of the DP023211–2 blind samples using cdPCR analysis. DNA samples from DP023211–2 positive and negative control were mixed to obtain final transgene contents of 100%, 5.0%, 3.0%, and 0.09%. The mixed DNA samples were used as a template to amplify the transgene (A) and <em>hmg</em> (B) on the cdPCR platform.</p></figcaption></figure><p>Both DP02311–3 and <em>hmg</em> qPCR and duplex cdPCR accurately quantified GMO content (<a href="#t0004" class="usa-link">Table 4</a>). However, the results of this study differ from previous reports suggesting that dPCR exhibits higher precision than qPCR in GMO quantification.<sup><a href="#cit0024" class="usa-link" aria-describedby="cit0024">24</a>,<a href="#cit0038" class="usa-link" aria-describedby="cit0038">38</a></sup> In this study, qPCR quantification was conducted after multiple independent runs to adjust DNA concentration, as well as primer and probe concentrations and ratios, to meet the acceptable criteria set by the ENGL.<sup><a href="#cit0028" class="usa-link" aria-describedby="cit0028">28</a></sup> In contrast, cdPCR analysis was performed without such optimization trials, which may explain the higher precision observed in the qPCR results. Additionally, the amount of template DNA used for the blind samples (5.0%, 3.0%, and 0.9%) in qPCR was 150 ng, 90 ng, and 27 ng, respectively, which was significantly higher than the 0.225 ng, 0.135 ng, and 0.0405 ng used in cdPCR. Therefore, duplex cdPCR is a more cost- and time-efficient method without the need for standard curves or extensive optimization steps. Moreover, it enables accurate quantification even in samples with low GMO content, demonstrating its practical applicability. Schematic representations of the developed qPCR and duplex cdPCR analyses to quantify GM maize content were presented in <a href="#f0003" class="usa-link">Figure 3</a>.
</p>
<figure class="fig xbox font-sm" id="f0003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8359/12377078/f6fbf99ea537/KGMC_A_2548053_F0003_B.jpg" loading="lazy" height="217" width="800" alt="Figure 3."></p>
<div class="p text-right font-secondary"><a href="figure/f0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of RT-qPCR and duplex cdPCR analyses for GM maize quantification. ERG, endogenous reference gene.</p></figcaption></figure></section><section id="s0003-s2005"><h3 class="pmc_sec_title">Quantification of Eleven GM Maize Blind Samples by cdPCR</h3>
<p>The established cdPCR method was further applied to quantify the GM maize blind samples (0.9%, 3.0%, and 5.0%) in eleven maize events. The results for the target genes (DAS01131-3, DAS40278-9, DP004114-3, DP051291-1, DP098140-6, DP202216–6, MON87403-1, MON87411-9, MON87429-9, MON87460–4, and MON88017-3) and <em>hmg</em> cdPCR quantification are shown in <a href="#t0005" class="usa-link">Table 5</a>. The GM content was determined as the ratio of the average copy number of the transgene to that of <em>hmg</em>, divided by the zygosity ratio (<a href="#t0002" class="usa-link">Table 2</a>). The measured GM % for each maize event was similar to the expected value. The precision and trueness of quantification were within the limits established by the ENGL,<sup><a href="#cit0028" class="usa-link" aria-describedby="cit0028">28</a></sup> except for <a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3 and MON88017–3. The 0.9% <a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3 sample was measured as 0.721%, with an RSD of 27.89% and a bias of −19.88%, while the 0.9% MON88017–3 sample was measured as 0.669%, with an RSD of 30.38% and a bias of −16.41%. Increased % RSD and % bias in PCR-based GMO quantification analyses were observed in samples with concentrations approaching the minimum value tested, which is consistent with previous reports.<sup><a href="#cit0008" class="usa-link" aria-describedby="cit0008">8</a>,<a href="#cit0015" class="usa-link" aria-describedby="cit0015">15</a>,<a href="#cit0038" class="usa-link" aria-describedby="cit0038">38</a></sup> Therefore, further investigation is needed to determine whether increasing the template amount for the 5.0%, 3.0%, and 0.9% blind samples in cdPCR reduces % RSD and % bias. Nevertheless, this study showed that cdPCR achieved accurate and precise quantification of GM content at 3.0%, a GMO labeling threshold in South Korea<sup><a href="#cit0004" class="usa-link" aria-describedby="cit0004">4</a></sup> without optimization trials, while qPCR required multiple optimization steps (e.g. DNA input, primer/probe concentration) to meet ENGL criteria (data not shown).</p>
<section class="tw xbox font-sm" id="t0005"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Precision and trueness of GM quantification, corrected by zygosity ratio as determined by cdPCR<sup>a</sup>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" align="center" colspan="1">Event</th>
<th align="center" colspan="1" rowspan="1"> </th>
<th rowspan="2" align="center" colspan="1">Total</th>
<th colspan="3" align="center" rowspan="1">Event specific target<hr>
</th>
<th colspan="3" align="center" rowspan="1">
<em>hmg</em><hr>
</th>
<th align="center" colspan="1" rowspan="1"> </th>
<th align="center" colspan="1" rowspan="1"> </th>
<th align="center" colspan="1" rowspan="1"> </th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">
<sup>b</sup>Target<br>GM-levels %</th>
<th align="center" colspan="1" rowspan="1">Positive</th>
<th align="center" colspan="1" rowspan="1">Conc.cp/µl.</th>
<th align="center" colspan="1" rowspan="1">SD</th>
<th align="center" colspan="1" rowspan="1">Positive</th>
<th align="center" colspan="1" rowspan="1">Conc.cp/µl.</th>
<th align="center" colspan="1" rowspan="1">SD</th>
<th align="center" colspan="1" rowspan="1">
<sup>b</sup>Measured GM-level %</th>
<th align="center" colspan="1" rowspan="1">Precision (RSD%)</th>
<th align="center" colspan="1" rowspan="1">Trueness (bias %)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3" align="left" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20426.3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">2.98</td>
<td align="center" colspan="1" rowspan="1">0.57</td>
<td align="center" colspan="1" rowspan="1">7373</td>
<td align="center" colspan="1" rowspan="1">1037.1</td>
<td align="center" colspan="1" rowspan="1">47.96</td>
<td align="center" colspan="1" rowspan="1">0.721</td>
<td align="center" colspan="1" rowspan="1">27.89</td>
<td align="center" colspan="1" rowspan="1">−19.88</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20455.3</td>
<td align="center" colspan="1" rowspan="1">111</td>
<td align="center" colspan="1" rowspan="1">12.54</td>
<td align="center" colspan="1" rowspan="1">0.61</td>
<td align="center" colspan="1" rowspan="1">7563</td>
<td align="center" colspan="1" rowspan="1">1068.7</td>
<td align="center" colspan="1" rowspan="1">24.23</td>
<td align="center" colspan="1" rowspan="1">2.944</td>
<td align="center" colspan="1" rowspan="1">6.19</td>
<td align="center" colspan="1" rowspan="1">−1.85</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20445.3</td>
<td align="center" colspan="1" rowspan="1">191</td>
<td align="center" colspan="1" rowspan="1">21.75</td>
<td align="center" colspan="1" rowspan="1">1.49</td>
<td align="center" colspan="1" rowspan="1">7742</td>
<td align="center" colspan="1" rowspan="1">1101.8</td>
<td align="center" colspan="1" rowspan="1">23.17</td>
<td align="center" colspan="1" rowspan="1">4.954</td>
<td align="center" colspan="1" rowspan="1">9.51</td>
<td align="center" colspan="1" rowspan="1">−0.93</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS40278" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS40278</a>–9</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20463.3</td>
<td align="center" colspan="1" rowspan="1">59</td>
<td align="center" colspan="1" rowspan="1">6.71</td>
<td align="center" colspan="1" rowspan="1">0.86</td>
<td align="center" colspan="1" rowspan="1">10382</td>
<td align="center" colspan="1" rowspan="1">1638.8</td>
<td align="center" colspan="1" rowspan="1">19.29</td>
<td align="center" colspan="1" rowspan="1">1.003</td>
<td align="center" colspan="1" rowspan="1">15.69</td>
<td align="center" colspan="1" rowspan="1">11.44</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20474.8</td>
<td align="center" colspan="1" rowspan="1">190</td>
<td align="center" colspan="1" rowspan="1">21.58</td>
<td align="center" colspan="1" rowspan="1">1.22</td>
<td align="center" colspan="1" rowspan="1">10296</td>
<td align="center" colspan="1" rowspan="1">1618.0</td>
<td align="center" colspan="1" rowspan="1">31.34</td>
<td align="center" colspan="1" rowspan="1">3.067</td>
<td align="center" colspan="1" rowspan="1">3.15</td>
<td align="center" colspan="1" rowspan="1">8.90</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20453.5</td>
<td align="center" colspan="1" rowspan="1">294</td>
<td align="center" colspan="1" rowspan="1">33.54</td>
<td align="center" colspan="1" rowspan="1">0.78</td>
<td align="center" colspan="1" rowspan="1">10237</td>
<td align="center" colspan="1" rowspan="1">1606.9</td>
<td align="center" colspan="1" rowspan="1">27.63</td>
<td align="center" colspan="1" rowspan="1">5.113</td>
<td align="center" colspan="1" rowspan="1">3.84</td>
<td align="center" colspan="1" rowspan="1">2.25</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/DAS59122" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS59122</a>–7</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20361</td>
<td align="center" colspan="1" rowspan="1">46</td>
<td align="center" colspan="1" rowspan="1">6.46</td>
<td align="center" colspan="1" rowspan="1">0.61</td>
<td align="center" colspan="1" rowspan="1">9208</td>
<td align="center" colspan="1" rowspan="1">1393.4</td>
<td align="center" colspan="1" rowspan="1">27.13</td>
<td align="center" colspan="1" rowspan="1">0.878</td>
<td align="center" colspan="1" rowspan="1">11.60</td>
<td align="center" colspan="1" rowspan="1">−2.48</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20437.5</td>
<td align="center" colspan="1" rowspan="1">157</td>
<td align="center" colspan="1" rowspan="1">27.16</td>
<td align="center" colspan="1" rowspan="1">1.5</td>
<td align="center" colspan="1" rowspan="1">9147</td>
<td align="center" colspan="1" rowspan="1">1373.7</td>
<td align="center" colspan="1" rowspan="1">26.96</td>
<td align="center" colspan="1" rowspan="1">3.063</td>
<td align="center" colspan="1" rowspan="1">3.93</td>
<td align="center" colspan="1" rowspan="1">2.09</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20417.8</td>
<td align="center" colspan="1" rowspan="1">288</td>
<td align="center" colspan="1" rowspan="1">44.64</td>
<td align="center" colspan="1" rowspan="1">2.53</td>
<td align="center" colspan="1" rowspan="1">9165</td>
<td align="center" colspan="1" rowspan="1">1379.2</td>
<td align="center" colspan="1" rowspan="1">18.29</td>
<td align="center" colspan="1" rowspan="1">5.620</td>
<td align="center" colspan="1" rowspan="1">2.87</td>
<td align="center" colspan="1" rowspan="1">12.40</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">DP004114–3</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20457</td>
<td align="center" colspan="1" rowspan="1">48</td>
<td align="center" colspan="1" rowspan="1">5.44</td>
<td align="center" colspan="1" rowspan="1">0.36</td>
<td align="center" colspan="1" rowspan="1">12447</td>
<td align="center" colspan="1" rowspan="1">2170.9</td>
<td align="center" colspan="1" rowspan="1">47.2</td>
<td align="center" colspan="1" rowspan="1">0.704</td>
<td align="center" colspan="1" rowspan="1">6.59</td>
<td align="center" colspan="1" rowspan="1">−21.76</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20450.5</td>
<td align="center" colspan="1" rowspan="1">176</td>
<td align="center" colspan="1" rowspan="1">19.98</td>
<td align="center" colspan="1" rowspan="1">1.08</td>
<td align="center" colspan="1" rowspan="1">12423</td>
<td align="center" colspan="1" rowspan="1">2164.9</td>
<td align="center" colspan="1" rowspan="1">41.5</td>
<td align="center" colspan="1" rowspan="1">2.593</td>
<td align="center" colspan="1" rowspan="1">6.07</td>
<td align="center" colspan="1" rowspan="1">−13.55</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20462.5</td>
<td align="center" colspan="1" rowspan="1">285</td>
<td align="center" colspan="1" rowspan="1">32.44</td>
<td align="center" colspan="1" rowspan="1">2.46</td>
<td align="center" colspan="1" rowspan="1">12468</td>
<td align="center" colspan="1" rowspan="1">2176.2</td>
<td align="center" colspan="1" rowspan="1">54.4</td>
<td align="center" colspan="1" rowspan="1">4.189</td>
<td align="center" colspan="1" rowspan="1">9.86</td>
<td align="center" colspan="1" rowspan="1">−16.22</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">DP023211–2</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20408.8</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">2.3</td>
<td align="center" colspan="1" rowspan="1">0.64</td>
<td align="center" colspan="1" rowspan="1">11161</td>
<td align="center" colspan="1" rowspan="1">1848.7</td>
<td align="center" colspan="1" rowspan="1">73.67</td>
<td align="center" colspan="1" rowspan="1">0.86</td>
<td align="center" colspan="1" rowspan="1">17.64</td>
<td align="center" colspan="1" rowspan="1">−4.44</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20446.8</td>
<td align="center" colspan="1" rowspan="1">59</td>
<td align="center" colspan="1" rowspan="1">6.72</td>
<td align="center" colspan="1" rowspan="1">1.06</td>
<td align="center" colspan="1" rowspan="1">10747</td>
<td align="center" colspan="1" rowspan="1">1726.2</td>
<td align="center" colspan="1" rowspan="1">13.2</td>
<td align="center" colspan="1" rowspan="1">2.902</td>
<td align="center" colspan="1" rowspan="1">17.62</td>
<td align="center" colspan="1" rowspan="1">−3.25</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20448.5</td>
<td align="center" colspan="1" rowspan="1">98</td>
<td align="center" colspan="1" rowspan="1">11.12</td>
<td align="center" colspan="1" rowspan="1">0.94</td>
<td align="center" colspan="1" rowspan="1">10787</td>
<td align="center" colspan="1" rowspan="1">1736.2</td>
<td align="center" colspan="1" rowspan="1">55.9</td>
<td align="center" colspan="1" rowspan="1">4.775</td>
<td align="center" colspan="1" rowspan="1">6.80</td>
<td align="center" colspan="1" rowspan="1">−4.50</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">DP098140–6</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20448.3</td>
<td align="center" colspan="1" rowspan="1">57</td>
<td align="center" colspan="1" rowspan="1">6.46</td>
<td align="center" colspan="1" rowspan="1">0.61</td>
<td align="center" colspan="1" rowspan="1">10840</td>
<td align="center" colspan="1" rowspan="1">1748.38</td>
<td align="center" colspan="1" rowspan="1">24.89</td>
<td align="center" colspan="1" rowspan="1">0.676</td>
<td align="center" colspan="1" rowspan="1">12.29</td>
<td align="center" colspan="1" rowspan="1">−24.84</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20445.8</td>
<td align="center" colspan="1" rowspan="1">239</td>
<td align="center" colspan="1" rowspan="1">27.16</td>
<td align="center" colspan="1" rowspan="1">1.5</td>
<td align="center" colspan="1" rowspan="1">10836</td>
<td align="center" colspan="1" rowspan="1">1747.83</td>
<td align="center" colspan="1" rowspan="1">22.78</td>
<td align="center" colspan="1" rowspan="1">2.845</td>
<td align="center" colspan="1" rowspan="1">5.13</td>
<td align="center" colspan="1" rowspan="1">−5.18</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20436.5</td>
<td align="center" colspan="1" rowspan="1">391</td>
<td align="center" colspan="1" rowspan="1">44.66</td>
<td align="center" colspan="1" rowspan="1">2.53</td>
<td align="center" colspan="1" rowspan="1">10818</td>
<td align="center" colspan="1" rowspan="1">1744.6</td>
<td align="center" colspan="1" rowspan="1">21.56</td>
<td align="center" colspan="1" rowspan="1">4.686</td>
<td align="center" colspan="1" rowspan="1">5.39</td>
<td align="center" colspan="1" rowspan="1">−6.28</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">DP202216–6</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20460.5</td>
<td align="center" colspan="1" rowspan="1">82</td>
<td align="center" colspan="1" rowspan="1">9.27</td>
<td align="center" colspan="1" rowspan="1">1.17</td>
<td align="center" colspan="1" rowspan="1">11838</td>
<td align="center" colspan="1" rowspan="1">2000.53</td>
<td align="center" colspan="1" rowspan="1">42.46</td>
<td align="center" colspan="1" rowspan="1">0.741</td>
<td align="center" colspan="1" rowspan="1">16.97</td>
<td align="center" colspan="1" rowspan="1">−17.63</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20460</td>
<td align="center" colspan="1" rowspan="1">334</td>
<td align="center" colspan="1" rowspan="1">38.13</td>
<td align="center" colspan="1" rowspan="1">1.46</td>
<td align="center" colspan="1" rowspan="1">11798</td>
<td align="center" colspan="1" rowspan="1">1989.87</td>
<td align="center" colspan="1" rowspan="1">40.63</td>
<td align="center" colspan="1" rowspan="1">3.066</td>
<td align="center" colspan="1" rowspan="1">6.52</td>
<td align="center" colspan="1" rowspan="1">2.19</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20414</td>
<td align="center" colspan="1" rowspan="1">555</td>
<td align="center" colspan="1" rowspan="1">63.84</td>
<td align="center" colspan="1" rowspan="1">2.32</td>
<td align="center" colspan="1" rowspan="1">11746</td>
<td align="center" colspan="1" rowspan="1">1983.21</td>
<td align="center" colspan="1" rowspan="1">37.05</td>
<td align="center" colspan="1" rowspan="1">5.150</td>
<td align="center" colspan="1" rowspan="1">2.91</td>
<td align="center" colspan="1" rowspan="1">3.00</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">MON87403–1</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20450.3</td>
<td align="center" colspan="1" rowspan="1">37</td>
<td align="center" colspan="1" rowspan="1">4.19</td>
<td align="center" colspan="1" rowspan="1">0.46</td>
<td align="center" colspan="1" rowspan="1">6431</td>
<td align="center" colspan="1" rowspan="1">873.95</td>
<td align="center" colspan="1" rowspan="1">8.61</td>
<td align="center" colspan="1" rowspan="1">0.959</td>
<td align="center" colspan="1" rowspan="1">12.60</td>
<td align="center" colspan="1" rowspan="1">6.54</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">19365.3</td>
<td align="center" colspan="1" rowspan="1">139</td>
<td align="center" colspan="1" rowspan="1">17.41</td>
<td align="center" colspan="1" rowspan="1">7.46</td>
<td align="center" colspan="1" rowspan="1">6674</td>
<td align="center" colspan="1" rowspan="1">869.29</td>
<td align="center" colspan="1" rowspan="1">24.76</td>
<td align="center" colspan="1" rowspan="1">2.944</td>
<td align="center" colspan="1" rowspan="1">6.57</td>
<td align="center" colspan="1" rowspan="1">6.57</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20453.5</td>
<td align="center" colspan="1" rowspan="1">213</td>
<td align="center" colspan="1" rowspan="1">24.26</td>
<td align="center" colspan="1" rowspan="1">1.08</td>
<td align="center" colspan="1" rowspan="1">6539</td>
<td align="center" colspan="1" rowspan="1">891.76</td>
<td align="center" colspan="1" rowspan="1">13.17</td>
<td align="center" colspan="1" rowspan="1">4.665</td>
<td align="center" colspan="1" rowspan="1">4.74</td>
<td align="center" colspan="1" rowspan="1">4.73</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">MON87411–9</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20409.8</td>
<td align="center" colspan="1" rowspan="1">39</td>
<td align="center" colspan="1" rowspan="1">4.46</td>
<td align="center" colspan="1" rowspan="1">0.89</td>
<td align="center" colspan="1" rowspan="1">6275</td>
<td align="center" colspan="1" rowspan="1">850.35</td>
<td align="center" colspan="1" rowspan="1">17.45</td>
<td align="center" colspan="1" rowspan="1">0.877</td>
<td align="center" colspan="1" rowspan="1">23.83</td>
<td align="center" colspan="1" rowspan="1">−2.58</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20417.5</td>
<td align="center" colspan="1" rowspan="1">141</td>
<td align="center" colspan="1" rowspan="1">16.07</td>
<td align="center" colspan="1" rowspan="1">1.54</td>
<td align="center" colspan="1" rowspan="1">6220</td>
<td align="center" colspan="1" rowspan="1">841.09</td>
<td align="center" colspan="1" rowspan="1">23.58</td>
<td align="center" colspan="1" rowspan="1">3.194</td>
<td align="center" colspan="1" rowspan="1">10.62</td>
<td align="center" colspan="1" rowspan="1">6.46</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20436.8</td>
<td align="center" colspan="1" rowspan="1">213</td>
<td align="center" colspan="1" rowspan="1">24.19</td>
<td align="center" colspan="1" rowspan="1">1.27</td>
<td align="center" colspan="1" rowspan="1">6250</td>
<td align="center" colspan="1" rowspan="1">845</td>
<td align="center" colspan="1" rowspan="1">12.96</td>
<td align="center" colspan="1" rowspan="1">4.785</td>
<td align="center" colspan="1" rowspan="1">5.64</td>
<td align="center" colspan="1" rowspan="1">−4.29</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">MON87429–9</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20436.8</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">2.95</td>
<td align="center" colspan="1" rowspan="1">0.33</td>
<td align="center" colspan="1" rowspan="1">5876</td>
<td align="center" colspan="1" rowspan="1">784.78</td>
<td align="center" colspan="1" rowspan="1">18.58</td>
<td align="center" colspan="1" rowspan="1">0.733</td>
<td align="center" colspan="1" rowspan="1">15.15</td>
<td align="center" colspan="1" rowspan="1">−18.59</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20432.5</td>
<td align="center" colspan="1" rowspan="1">94</td>
<td align="center" colspan="1" rowspan="1">10.64</td>
<td align="center" colspan="1" rowspan="1">2.59</td>
<td align="center" colspan="1" rowspan="1">5916</td>
<td align="center" colspan="1" rowspan="1">791.34</td>
<td align="center" colspan="1" rowspan="1">3.79</td>
<td align="center" colspan="1" rowspan="1">2.9444</td>
<td align="center" colspan="1" rowspan="1">21.80</td>
<td align="center" colspan="1" rowspan="1">−3.93</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20444.3</td>
<td align="center" colspan="1" rowspan="1">181</td>
<td align="center" colspan="1" rowspan="1">20.53</td>
<td align="center" colspan="1" rowspan="1">1.75</td>
<td align="center" colspan="1" rowspan="1">5934</td>
<td align="center" colspan="1" rowspan="1">793.71</td>
<td align="center" colspan="1" rowspan="1">13.92</td>
<td align="center" colspan="1" rowspan="1">4.665</td>
<td align="center" colspan="1" rowspan="1">8.13</td>
<td align="center" colspan="1" rowspan="1">0.84</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">MON87460–4</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20445.3</td>
<td align="center" colspan="1" rowspan="1">30</td>
<td align="center" colspan="1" rowspan="1">3.4</td>
<td align="center" colspan="1" rowspan="1">0.75</td>
<td align="center" colspan="1" rowspan="1">6424</td>
<td align="center" colspan="1" rowspan="1">873.13</td>
<td align="center" colspan="1" rowspan="1">20.21</td>
<td align="center" colspan="1" rowspan="1">0.913</td>
<td align="center" colspan="1" rowspan="1">19.32</td>
<td align="center" colspan="1" rowspan="1">−9.28</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20427.8</td>
<td align="center" colspan="1" rowspan="1">93</td>
<td align="center" colspan="1" rowspan="1">10.53</td>
<td align="center" colspan="1" rowspan="1">0.73</td>
<td align="center" colspan="1" rowspan="1">6482</td>
<td align="center" colspan="1" rowspan="1">883.74</td>
<td align="center" colspan="1" rowspan="1">23.18</td>
<td align="center" colspan="1" rowspan="1">2.793</td>
<td align="center" colspan="1" rowspan="1">0.73</td>
<td align="center" colspan="1" rowspan="1">−20.7</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20436.3</td>
<td align="center" colspan="1" rowspan="1">157</td>
<td align="center" colspan="1" rowspan="1">17.82</td>
<td align="center" colspan="1" rowspan="1">0.92</td>
<td align="center" colspan="1" rowspan="1">6363</td>
<td align="center" colspan="1" rowspan="1">863.59</td>
<td align="center" colspan="1" rowspan="1">15.88</td>
<td align="center" colspan="1" rowspan="1">4.837</td>
<td align="center" colspan="1" rowspan="1">0.51</td>
<td align="center" colspan="1" rowspan="1">−16.3</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">MON88017–3</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
<td align="center" colspan="1" rowspan="1">20450</td>
<td align="center" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">2.83</td>
<td align="center" colspan="1" rowspan="1">0.94</td>
<td align="center" colspan="1" rowspan="1">6049</td>
<td align="center" colspan="1" rowspan="1">762.58</td>
<td align="center" colspan="1" rowspan="1">24.01</td>
<td align="center" colspan="1" rowspan="1">0.669</td>
<td align="center" colspan="1" rowspan="1">30.38</td>
<td align="center" colspan="1" rowspan="1">−16.41</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">20420.5</td>
<td align="center" colspan="1" rowspan="1">100</td>
<td align="center" colspan="1" rowspan="1">11.39</td>
<td align="center" colspan="1" rowspan="1">0.24</td>
<td align="center" colspan="1" rowspan="1">5946</td>
<td align="center" colspan="1" rowspan="1">685.63</td>
<td align="center" colspan="1" rowspan="1">94.34</td>
<td align="center" colspan="1" rowspan="1">2.744</td>
<td align="center" colspan="1" rowspan="1">5.45</td>
<td align="center" colspan="1" rowspan="1">−8.5</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1">5.0</td>
<td align="center" colspan="1" rowspan="1">20431.5</td>
<td align="center" colspan="1" rowspan="1">166</td>
<td align="center" colspan="1" rowspan="1">18.91</td>
<td align="center" colspan="1" rowspan="1">0.39</td>
<td align="center" colspan="1" rowspan="1">5956</td>
<td align="center" colspan="1" rowspan="1">757.17</td>
<td align="center" colspan="1" rowspan="1">24.03</td>
<td align="center" colspan="1" rowspan="1">4.551</td>
<td align="center" colspan="1" rowspan="1">3.62</td>
<td align="center" colspan="1" rowspan="1">−9.0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tfn0005"><p><sup>a</sup>Each value represents the average of four experimental replications, except for SD, precision, and trueness. <sup>b</sup> GM % in copy/maize haploid genome copy number × 100. SD, standard deviation of Conc. Cp/µl (concentration of copy/µl); RSD, relative standard deviation; Bias, deviation from expected value.</p></div></div></section><p>Additionally, GM content was calculated using a zygosity ratio of 0.5, assuming that maize is in the market predominantly as a hybrid trait (Supplemental Table S5). For some GMO contents of <a href="https://www.ncbi.nlm.nih.gov/protein/DAS01131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DAS01131</a>–3, DP04114–3, DP023211–2, DP051291–1, DP202216–6, MON87411–9, and MON88017–3, the % RSD and % bias did not meet the acceptance criteria, indicating that accurate determination of the zygosity ratio is crucial for precise GMO quantification.</p>
<p>In conclusion, duplex cdPCR demonstrated comparable performance to qPCR in quantifying DP023211–3. Determination of the zygosity ratio of GM DNA samples was crucial for accurate quantification of GM content. The duplex cdPCR method achieved accurate quantification of twelve GM maize events at GMO levels as low as 0.9% and successfully quantified 3.0% and 5.0% GM content with good precision and trueness. These findings demonstrate the duplex cdPCR method is straightforward and does not require additional optimization trials, unlike qPCR, which requires standard curve establishment. While the present study focused on key performance metrics such as accuracy and precision, we acknowledge that a comprehensive in-house validation will be necessary to support broader regulatory application. Future work will therefore include a full validation of the method in accordance with ENGL and other international guidelines, addressing essential parameters such as limit of detection (LOD), LOQ, robustness, and reproducibility. We also plan to explore participation in inter-laboratory ring trials to further assess the method’s reliability and transferability.</p></section></section><section id="sec14"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SM6672"><div class="caption p"><span>Kim et al_Supplemental Information_clean.docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377078/bin/KGMC_A_2548053_SM6672.docx" data-ga-action="click_feat_suppl" class="usa-link">KGMC_A_2548053_SM6672.docx</a><sup> (310.3KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We acknowledge the applicants for providing the GM reference materials and detection information as part of the national regulatory process in South Korea. EHK is supported by a postdoctoral training program at the Rural Development Administration of South Korea.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This study was supported by the Rural Development Administration of the Republic of Korea [PJ Number: RS-2025-00400888].</p></section><section id="s0004"><h2 class="pmc_sec_title">Disclosure Statement</h2>
<p>This study was conducted for scientific and regulatory purposes only. The use of product names or trademarks is intended solely to provide accurate methodological details and does not imply endorsement by the authors, their affiliated institutions, or the funding agency. All trademarks mentioned herein are the property of their respective owners.</p></section><section id="s0005"><h2 class="pmc_sec_title">Supplementary Material</h2>
<p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/21645698.2025.2548053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1080/21645698.2025.2548053</a></p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="cit0001">
<span class="label">1.</span><cite>Kumar K, Gambhir G, Dass A, Tripathi AK, Singh A, Jha AK, Yadava P, Choudhary M, Rakshit S.. 
Genetically modified crops: current status and future prospects. Planta. 2020;251(4):91. doi: 10.1007/s00425-020-03372-8.
</cite> [<a href="https://doi.org/10.1007/s00425-020-03372-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32236850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Genetically%20modified%20crops:%20current%20status%20and%20future%20prospects&amp;volume=251&amp;issue=4&amp;publication_year=2020&amp;pages=91&amp;pmid=32236850&amp;doi=10.1007/s00425-020-03372-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0002">
<span class="label">2.</span><cite>Oliver MJ. Why we need GMO crops in agriculture. Mo Med. 2014;111(6):492–507. doi: 10.1126/science.1204531.
</cite> [<a href="https://doi.org/10.1126/science.1204531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6173531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25665234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mo%20Med&amp;title=Why%20we%20need%20GMO%20crops%20in%20agriculture&amp;volume=111&amp;issue=6&amp;publication_year=2014&amp;pages=492-507&amp;pmid=25665234&amp;doi=10.1126/science.1204531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0003">
<span class="label">3.</span><cite>AgbioInvestor GM Monitor . Global GM crop area review. Suite 18, Pathhead, Midlothian. 2025.  [Assessed in 2025 May
6]. <a href="https://www.gm.agbioinvestor.com/downloads" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.gm.agbioinvestor.com/downloads</a>.</cite>
</li>
<li id="cit0004">
<span class="label">4.</span><cite>Ministry of Food and Drug Safety (MFDS) . Labeling standards for genetically modified foods. Etc. (Notice No. 2019-98). 2019. <a href="https://www.law.go.kr/LSW/admRulLsInfoP.do?admRulSeq=2100000183336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.law.go.kr/LSW/admRulLsInfoP.do?admRulSeq=2100000183336</a>.</cite>
</li>
<li id="cit0005">
<span class="label">5.</span><cite>European Commission Regulation (EC) No. 1829/2003 . Of the European Parliament and of the council of 22 September, 2003, concerning on genetically modified food and feed. 1829/2003.</cite>
</li>
<li id="cit0006">
<span class="label">6.</span><cite>European Commission Regulation (EC) No. 1830/2003 . Of the European Parliament and of the council of 22 September, 2003, concerning the traceability and labelling of genetically modified organisms and the traceability of food and feed products from genetically modified organisms and amending directive 2001/18/EC. 1830/2003.</cite>
</li>
<li id="cit0007">
<span class="label">7.</span><cite>Ministry of Health, Labor, and Welfare (MHLW) . Guidelines for the labeling of genetically modified foods, etc. Japan; 2001, revised. <a href="https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/kenkou_iryou/shokuhin/idenshi/index_00002.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/kenkou_iryou/shokuhin/idenshi/index_00002.html</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ministry%20of%20Health,%20Labor,%20and%20Welfare%20(MHLW)%20.%20Guidelines%20for%20the%20labeling%20of%20genetically%20modified%20foods,%20etc.%20Japan;%202001,%20revised.%20https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/kenkou_iryou/shokuhin/idenshi/index_00002.html." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0008">
<span class="label">8.</span><cite>Del Gaudio S, Cirillo A, Di Bernardo G, Galderisi U, Cipollaro M. Verification of real-time PCR methods for qualitative and quantitative testing of genetically modified organisms. J Food Qual. 2012;35(6):442–47. doi: 10.1111/jfq.12001.</cite> [<a href="https://doi.org/10.1111/jfq.12001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Food%20Qual&amp;title=Verification%20of%20real-time%20PCR%20methods%20for%20qualitative%20and%20quantitative%20testing%20of%20genetically%20modified%20organisms&amp;volume=35&amp;issue=6&amp;publication_year=2012&amp;pages=442-47&amp;doi=10.1111/jfq.12001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0009">
<span class="label">9.</span><cite>Hübner P, Waiblinger HU, Pietsch K, Brodmann P. Validation of PCR methods for quantitation of genetically modified plants in food. J AOAC Int. 2001;86(6):1855–64. doi: 10.1093/jaoac/84.6.1855.</cite> [<a href="https://doi.org/10.1093/jaoac/84.6.1855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11767156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20AOAC%20Int&amp;title=Validation%20of%20PCR%20methods%20for%20quantitation%20of%20genetically%20modified%20plants%20in%20food&amp;volume=86&amp;issue=6&amp;publication_year=2001&amp;pages=1855-64&amp;pmid=11767156&amp;doi=10.1093/jaoac/84.6.1855&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0010">
<span class="label">10.</span><cite>International Standard (ISO) 21569 . Foodstuffs, methods of analysis for the detection of genetically modified organisms and derived products-qualitative nucleic acid based methods. Geneva, Switzerland: International Organization for Standardization; 2005. (2005/Amd 1:2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?International%20Standard%20(ISO)%2021569%20.%20Foodstuffs,%20methods%20of%20analysis%20for%20the%20detection%20of%20genetically%20modified%20organisms%20and%20derived%20products-qualitative%20nucleic%20acid%20based%20methods.%20Geneva,%20Switzerland:%20International%20Organization%20for%20Standardization;%202005.%20(2005/Amd%201:2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0011">
<span class="label">11.</span><cite>International Standard (ISO) 21570 . Foodstuffs, methods of analysis for the detection of genetically modified organisms and derived products-quantitative nucleic acid based methods. Geneva, Switzerland: International Organization for Standardization; 2005. 
(2005/Amd 1:2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?International%20Standard%20(ISO)%2021570%20.%20Foodstuffs,%20methods%20of%20analysis%20for%20the%20detection%20of%20genetically%20modified%20organisms%20and%20derived%20products-quantitative%20nucleic%20acid%20based%20methods.%20Geneva,%20Switzerland:%20International%20Organization%20for%20Standardization;%202005.%20(2005/Amd%201:2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0012">
<span class="label">12.</span><cite>Grohmann L, Barbante A, Eriksson R, Gatto F, Georgieva T, Huber I, Hulin J, Köppel R, Marchesi U, Marmin L, et al. Guidance document on multiplex real-time PCR methods. Luxembourg: EUR 30708 EN, Publications Office of the European Union; 2021. ISBN 978-92-76-37820-4.</cite> [<a href="https://scholar.google.com/scholar_lookup?Grohmann%20L,%20Barbante%20A,%20Eriksson%20R,%20Gatto%20F,%20Georgieva%20T,%20Huber%20I,%20Hulin%20J,%20K%C3%B6ppel%20R,%20Marchesi%20U,%20Marmin%20L,%20et%20al.%20Guidance%20document%20on%20multiplex%20real-time%20PCR%20methods.%20Luxembourg:%20EUR%2030708%20EN,%20Publications%20Office%20of%20the%20European%20Union;%202021.%20ISBN%20978-92-76-37820-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0013">
<span class="label">13.</span><cite>Bharuthram A, Paximadis M, Picton ACP, Tiemessen CT. Comparison of a quantitative real-time PCR assay and droplet digital PCR for copy number analysis of the CCL4L genes. Infect Genetucs Evol. 2014;25:28–35. doi: 10.1016/j.meegid.2014.03.028.</cite> [<a href="https://doi.org/10.1016/j.meegid.2014.03.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24727646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Genetucs%20Evol&amp;title=Comparison%20of%20a%20quantitative%20real-time%20PCR%20assay%20and%20droplet%20digital%20PCR%20for%20copy%20number%20analysis%20of%20the%20CCL4L%20genes&amp;volume=25&amp;publication_year=2014&amp;pages=28-35&amp;pmid=24727646&amp;doi=10.1016/j.meegid.2014.03.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0014">
<span class="label">14.</span><cite>Iwobi A, Gerdes L, Busch U, Pecoraro S. Droplet digital PCR for routine analysis of genetically modified foods (GMO). Food Control. 2016;69:205–13. doi: 10.1016/j.foodcont.2016.04.048.</cite> [<a href="https://doi.org/10.1016/j.foodcont.2016.04.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Control&amp;title=Droplet%20digital%20PCR%20for%20routine%20analysis%20of%20genetically%20modified%20foods%20(GMO)&amp;volume=69&amp;publication_year=2016&amp;pages=205-13&amp;doi=10.1016/j.foodcont.2016.04.048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0015">
<span class="label">15.</span><cite>Morisset D, Štebih D, Milavec M, Gruden K, Žel J, Heazlewood JL. Quantitative analysis of food and feed samples with droplet digital PCR. PLOS ONE. 2013;8(5):e62583. doi: 10.1371/journal.pone.0062583.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0062583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3642186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23658750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=Quantitative%20analysis%20of%20food%20and%20feed%20samples%20with%20droplet%20digital%20PCR&amp;volume=8&amp;issue=5&amp;publication_year=2013&amp;pages=e62583&amp;pmid=23658750&amp;doi=10.1371/journal.pone.0062583&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0016">
<span class="label">16.</span><cite>Quan P-L, Sauzade M, Brouzes E. dPCR: a technology review. Sensors. 2018;18(4):1271. doi: 10.3390/s18041271.
</cite> [<a href="https://doi.org/10.3390/s18041271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5948698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29677144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sensors&amp;title=dPCR:%20a%20technology%20review&amp;volume=18&amp;issue=4&amp;publication_year=2018&amp;pages=1271&amp;pmid=29677144&amp;doi=10.3390/s18041271&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0017">
<span class="label">17.</span><cite>Vogelstein B, Kinzler KW. Digital PCR. Proc Natl Acad Sci USA. 1999;96(16):9236–41. doi: 10.1073/pnas.96.16.9236.
</cite> [<a href="https://doi.org/10.1073/pnas.96.16.9236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC17763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10430926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Digital%20PCR&amp;volume=96&amp;issue=16&amp;publication_year=1999&amp;pages=9236-41&amp;pmid=10430926&amp;doi=10.1073/pnas.96.16.9236&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0018">
<span class="label">18.</span><cite>Huggett JF, Foy CA, Benes V, Emslie K, Garson JA, Haynes R, Hellemans J, Kubista M, Mueller RD, Nolan T, et al. The digital MIQE guidelines: minimum information for publication of quantitative digital PCR experiments. Clin Chem. 2013;59(6):892–902. doi: 10.1373/clinchem.2013.206375.
</cite> [<a href="https://doi.org/10.1373/clinchem.2013.206375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23570709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&amp;title=The%20digital%20MIQE%20guidelines:%20minimum%20information%20for%20publication%20of%20quantitative%20digital%20PCR%20experiments&amp;volume=59&amp;issue=6&amp;publication_year=2013&amp;pages=892-902&amp;pmid=23570709&amp;doi=10.1373/clinchem.2013.206375&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0019">
<span class="label">19.</span><cite>Pecoraro S, Berben G, Burns M, Corbisier P, De Giacomo M, De Loose M, Dagand E, Dobnik D, Eriksson R, Holst-Jensen A, et al. Overview and recommendations for the application of digital PCR. JRC technical reports. JRC technical reports, EUR 29673 EN, JRC 115736. Luxembourg: Publications Office of the European Union; 2019.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=JRC%20technical%20reports.%20JRC%20technical%20reports,%20EUR%2029673%20EN,%20JRC%20115736&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0020">
<span class="label">20.</span><cite>Demeke T. Considerations for the successful detection and quantification of genetically modified events in grain and food samples using multiplex digital PCR. Foods. 2025;14(1):75. doi: 10.3390/foods14010075.</cite> [<a href="https://doi.org/10.3390/foods14010075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11719948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39796365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=Considerations%20for%20the%20successful%20detection%20and%20quantification%20of%20genetically%20modified%20events%20in%20grain%20and%20food%20samples%20using%20multiplex%20digital%20PCR&amp;volume=14&amp;issue=1&amp;publication_year=2025&amp;pages=75&amp;pmid=39796365&amp;doi=10.3390/foods14010075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0021">
<span class="label">21.</span><cite>Dobnik D, Spilsberg B, Košir AB, Holst-Jensen A, Žel J. Multiplex quantification of 12 European union authorized genetically modified maize lines with droplet digital polymerase chain reaction. Anal Chem. 2015;87(16):8218–26. doi: 10.1021/acs.analchem.5b01208.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.5b01208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26169291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Chem&amp;title=Multiplex%20quantification%20of%2012%20European%20union%20authorized%20genetically%20modified%20maize%20lines%20with%20droplet%20digital%20polymerase%20chain%20reaction&amp;volume=87&amp;issue=16&amp;publication_year=2015&amp;pages=8218-26&amp;pmid=26169291&amp;doi=10.1021/acs.analchem.5b01208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0022">
<span class="label">22.</span><cite>Demeke T, Lee SJ, Eng M. Increasing the efficiency of canola and soybean GMO detection and quantification using multiplex droplet digital PCR. Biology. 2022;11(2):201. doi: 10.3390/biology11020201.
</cite> [<a href="https://doi.org/10.3390/biology11020201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8869681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35205068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biology&amp;title=Increasing%20the%20efficiency%20of%20canola%20and%20soybean%20GMO%20detection%20and%20quantification%20using%20multiplex%20droplet%20digital%20PCR&amp;volume=11&amp;issue=2&amp;publication_year=2022&amp;pages=201&amp;pmid=35205068&amp;doi=10.3390/biology11020201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0023">
<span class="label">23.</span><cite>Köppel R, Bucher T, Bär D, van Belsen F, Ganeshan A. Validation of 13 duplex droplet digital PCR systems for quantitative GMO analysis of most prevalent GMO traits. Eur Food Res Technol. 2018;244(2):313–21. doi: 10.1007/s00217-017-2957-4.</cite> [<a href="https://doi.org/10.1007/s00217-017-2957-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20Food%20Res%20Technol&amp;title=Validation%20of%2013%20duplex%20droplet%20digital%20PCR%20systems%20for%20quantitative%20GMO%20analysis%20of%20most%20prevalent%20GMO%20traits&amp;volume=244&amp;issue=2&amp;publication_year=2018&amp;pages=313-21&amp;doi=10.1007/s00217-017-2957-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0024">
<span class="label">24.</span><cite>Long L, Yan W, He Y, Dong L, Xing Z, Li C, Xia W, Li F. Development of a duplex digital PCR method to quantify five genetically modified soybean events. Food Anal Methods. 2022;15(2):294–306. doi: 10.1007/s12161-021-02104-2.</cite> [<a href="https://doi.org/10.1007/s12161-021-02104-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Anal%20Methods&amp;title=Development%20of%20a%20duplex%20digital%20PCR%20method%20to%20quantify%20five%20genetically%20modified%20soybean%20events&amp;volume=15&amp;issue=2&amp;publication_year=2022&amp;pages=294-306&amp;doi=10.1007/s12161-021-02104-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0025">
<span class="label">25.</span><cite>Noma S, Kikuchi Y, Satou M, Tanaka T, Takiya T, Okusu H, Futo S, Takabatake R, Kitta K, Mano J. Simple, precise, and less biased GMO quantification by multiplexed genetic element-specific digital PCR. J AOAC Int. 2022;105(1):159–66. doi: 10.1093/jaoacint/qsab138.
</cite> [<a href="https://doi.org/10.1093/jaoacint/qsab138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34626115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20AOAC%20Int&amp;title=Simple,%20precise,%20and%20less%20biased%20GMO%20quantification%20by%20multiplexed%20genetic%20element-specific%20digital%20PCR&amp;volume=105&amp;issue=1&amp;publication_year=2022&amp;pages=159-66&amp;pmid=34626115&amp;doi=10.1093/jaoacint/qsab138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0026">
<span class="label">26.</span><cite>Gatto F, Savini C, Baillie CK, Broothaerts W, Burns M, Dagabd E, Debode F, Dobnik D, Grantiņa-Leviņa L, Grohmann L, et al. Definition of minimum performance requirements for analytical methods of GMO testing part 2. In: JRC. Vol. 125975. Luxembourg: Publications office of the European Union; 2023.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=JRC&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0027">
<span class="label">27.</span><cite>LMO Act . Act No. 15868 (partial amendment, 2018. 12. 18). 2018. <a href="https://www.law.go.kr/LSW/lsInfoP.do?efYd=20190612%26lsiSeq=205608#0000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.law.go.kr/LSW/lsInfoP.do?efYd=20190612&amp;lsiSeq=205608#0000</a>.</cite>
</li>
<li id="cit0028">
<span class="label">28.</span><cite>European Network of GMO Laboratories (ENGL) . Definition of minimum performance requirements for analytical methods of GMO testing. JRC technical report. Report No. JRC95544. 2015.</cite>
</li>
<li id="cit0029">
<span class="label">29.</span><cite>Smith M. Validating real-time polymerase chain reaction (PCR) assays. In: Bamford D, Zuckarman M, editors. Encyclopedia of virology. Vol. 5. 4th ed. Amsterdam: Elsevier Ltd; 2021. p. 35–44.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Encyclopedia%20of%20virology&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0030">
<span class="label">30.</span><cite>Weidner C, Frenzel J, Bartsch D, Waiblinger H-U, Mankertz J. Guideline for the verification of digital PCR methods in analytical GMO testing. J Consum Prot Food Saf. 2024;19(3):335–37. doi: 10.1007/s00003-024-01516-6.</cite> [<a href="https://doi.org/10.1007/s00003-024-01516-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Consum%20Prot%20Food%20Saf&amp;title=Guideline%20for%20the%20verification%20of%20digital%20PCR%20methods%20in%20analytical%20GMO%20testing&amp;volume=19&amp;issue=3&amp;publication_year=2024&amp;pages=335-37&amp;doi=10.1007/s00003-024-01516-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0031">
<span class="label">31.</span><cite>Arumuganathan K, Earle ED. Nuclear DNA content of some important plant species. Plant Mol Biol Rep. 1991;9(3):208–18. doi: 10.1007/BF02672069.</cite> [<a href="https://doi.org/10.1007/BF02672069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol%20Biol%20Rep&amp;title=Nuclear%20DNA%20content%20of%20some%20important%20plant%20species&amp;volume=9&amp;issue=3&amp;publication_year=1991&amp;pages=208-18&amp;doi=10.1007/BF02672069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0032">
<span class="label">32.</span><cite>Corbisier P, Barbante A, Berben G, Broothaerts W, De Loose M, Emons H, Georgieva T, Lievens A, Mazzara M, Papazova N, et al. JRC technical reports: recommendation for the unit of measurement and the measuring system to report traceable and comparable results expressing GM content in accordance with EU legislation. 2017.</cite>
</li>
<li id="cit0033">
<span class="label">33.</span><cite>Corbisier P, Emons H. Towards metrologically traceable and comparable results in GM quantification. Anal Bioanal Chem. 2019;411(1):7–11. doi: 10.1007/s00216-018-1457-0.
</cite> [<a href="https://doi.org/10.1007/s00216-018-1457-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6311193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30397759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Bioanal%20Chem&amp;title=Towards%20metrologically%20traceable%20and%20comparable%20results%20in%20GM%20quantification&amp;volume=411&amp;issue=1&amp;publication_year=2019&amp;pages=7-11&amp;pmid=30397759&amp;doi=10.1007/s00216-018-1457-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0034">
<span class="label">34.</span><cite>Corbisier P, Buttinger G, Savini C, Sacco MG, Gatto F, Emons H. Expression of GM content in mass fraction from digital PCR data. Food Control. 2022;133:108626. doi: 10.1016/j.foodcont.2021.108626.
</cite> [<a href="https://doi.org/10.1016/j.foodcont.2021.108626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8756621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35241875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Control&amp;title=Expression%20of%20GM%20content%20in%20mass%20fraction%20from%20digital%20PCR%20data&amp;volume=133&amp;publication_year=2022&amp;pages=108626&amp;pmid=35241875&amp;doi=10.1016/j.foodcont.2021.108626&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0035">
<span class="label">35.</span><cite>Holst-Jensen A, De Loose M, Van den Eede G. Coherence between legal requirements and approaches for detection of genetically modified organisms (GMOs) and their derived products. J Agric Food Chem. 2006;54(8):2799–809. doi: 10.1021/jf052849a.
</cite> [<a href="https://doi.org/10.1021/jf052849a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16608192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Agric%20Food%20Chem&amp;title=Coherence%20between%20legal%20requirements%20and%20approaches%20for%20detection%20of%20genetically%20modified%20organisms%20(GMOs)%20and%20their%20derived%20products&amp;volume=54&amp;issue=8&amp;publication_year=2006&amp;pages=2799-809&amp;pmid=16608192&amp;doi=10.1021/jf052849a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0036">
<span class="label">36.</span><cite>Cottenet G, Blancpain C, Chuah PF. Performance assessment of digital PCR for the quantification of GM-maize and GM-soya events. Anal Bioanal Chem. 2019;411(11):2461–69. doi: 10.1007/s00216-019-01692-7.
</cite> [<a href="https://doi.org/10.1007/s00216-019-01692-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30810790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Bioanal%20Chem&amp;title=Performance%20assessment%20of%20digital%20PCR%20for%20the%20quantification%20of%20GM-maize%20and%20GM-soya%20events&amp;volume=411&amp;issue=11&amp;publication_year=2019&amp;pages=2461-69&amp;pmid=30810790&amp;doi=10.1007/s00216-019-01692-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0037">
<span class="label">37.</span><cite>Demeke T, Eng M. Detection of soybean GMO events using two multiplex droplet digital PCR assays. J AOAC Int. 2025;108(1):23–28. doi: 10.1093/jaoacint/qsae082.
</cite> [<a href="https://doi.org/10.1093/jaoacint/qsae082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11761968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39475430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20AOAC%20Int&amp;title=Detection%20of%20soybean%20GMO%20events%20using%20two%20multiplex%20droplet%20digital%20PCR%20assays&amp;volume=108&amp;issue=1&amp;publication_year=2025&amp;pages=23-28&amp;pmid=39475430&amp;doi=10.1093/jaoacint/qsae082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0038">
<span class="label">38.</span><cite>Li J, Gao H, Li Y, Xiao F, Zhai S, Wu G, Wu Y. Event-specific PCR methods to quantify the genetically modified DBN9936 maize. J Food Compos Anal. 2022;105:104236. doi: 10.1016/j.jfca.2021.104236.</cite> [<a href="https://doi.org/10.1016/j.jfca.2021.104236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Food%20Compos%20Anal&amp;title=Event-specific%20PCR%20methods%20to%20quantify%20the%20genetically%20modified%20DBN9936%20maize&amp;volume=105&amp;publication_year=2022&amp;pages=104236&amp;doi=10.1016/j.jfca.2021.104236&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0039">
<span class="label">39.</span><cite>Verginelli D, Ciuffa S, Spinella K, La Rocca D, Misto M, Quarchioni C, Bonini P, Fusco C, Peroni L, Peddis S, et al. In-house validation of four duplex droplet digital PCR assays to quantify GM soybean events. Foods. 2024;13(24):4011. doi: 10.3390/foods13244011.
</cite> [<a href="https://doi.org/10.3390/foods13244011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11727554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39766954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=In-house%20validation%20of%20four%20duplex%20droplet%20digital%20PCR%20assays%20to%20quantify%20GM%20soybean%20events&amp;volume=13&amp;issue=24&amp;publication_year=2024&amp;pages=4011&amp;pmid=39766954&amp;doi=10.3390/foods13244011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Kim et al_Supplemental Information_clean.docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377078/bin/KGMC_A_2548053_SM6672.docx" data-ga-action="click_feat_suppl" class="usa-link">KGMC_A_2548053_SM6672.docx</a><sup> (310.3KB, docx) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from GM Crops &amp; Food are provided here courtesy of <strong>Taylor & Francis</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1080/21645698.2025.2548053"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/KGMC_16_2548053.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12377078/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12377078/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377078%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377078/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12377078/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12377078/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40842205/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12377078/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40842205/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12377078/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12377078/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="JEzvqY0RptkLf9FoZTljGBput3YracmEBHkEVWxvFHkwuHVh3Pi38HwZSsvSP004">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
