
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            LYMTACs:chimeric small molecules repurpose lysosomal membrane proteins for target protein relocalization and degradation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253B7D8AF23023053B7D003C5ACBE2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370906/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="LYMTACs:chimeric small molecules repurpose lysosomal membrane proteins for target protein relocalization and degradation">
<meta name="citation_author" content="Dhanusha A Nalawansha">
<meta name="citation_author_institution" content="Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Georgios Mazis">
<meta name="citation_author_institution" content="Amgen Research Copenhagen, Ronnegade 8, Copenhagen, Denmark">
<meta name="citation_author" content="Gitte Husemoen">
<meta name="citation_author_institution" content="Amgen Research Copenhagen, Ronnegade 8, Copenhagen, Denmark">
<meta name="citation_author" content="Kate S Ashton">
<meta name="citation_author_institution" content="Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Weixian Deng">
<meta name="citation_author_institution" content="Discovery Proteomics, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Ryan P Wurz">
<meta name="citation_author_institution" content="Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Anh T Tran">
<meta name="citation_author_institution" content="Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Brian A Lanman">
<meta name="citation_author_institution" content="Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Jiansong Xie">
<meta name="citation_author_institution" content="Cytometry and Imaging Sciences, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Robert G Guenette">
<meta name="citation_author_institution" content="Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Shiqian Li">
<meta name="citation_author_institution" content="Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Christopher E Smith">
<meta name="citation_author_institution" content="Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Suresh Archunan">
<meta name="citation_author_institution" content="Syngene Amgen Research &amp; Development Center, Bengaluru, India">
<meta name="citation_author" content="Manoj K Agnihotram">
<meta name="citation_author_institution" content="Syngene Amgen Research &amp; Development Center, Bengaluru, India">
<meta name="citation_author" content="Arghya Sadhukhan">
<meta name="citation_author_institution" content="Syngene Amgen Research &amp; Development Center, Bengaluru, India">
<meta name="citation_author" content="Rajiv Kapoor">
<meta name="citation_author_institution" content="Syngene Amgen Research &amp; Development Center, Bengaluru, India">
<meta name="citation_author" content="Chris Wilde">
<meta name="citation_author_institution" content="Lead Discovery and Characterization, Amgen Research, Thousand Oaks, CA 91320 USA">
<meta name="citation_author" content="Sajjan Koirala">
<meta name="citation_author_institution" content="Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Felipe De Sousa E Melo">
<meta name="citation_author_institution" content="Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_author" content="Patrick Ryan Potts">
<meta name="citation_author_institution" content="Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7812">
<meta name="citation_doi" content="10.1038/s41467-025-63128-4">
<meta name="citation_pmid" content="40841368">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370906/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370906/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370906/pdf/41467_2025_Article_63128.pdf">
<meta name="description" content="Proximity-inducing modalities that co-opt cellular pathways offer new opportunities to regulate oncogenic drivers. Inspired by the success of proximity-based chimeras in both intracellular and extracellular target space, here we describe the ...">
<meta name="og:title" content="LYMTACs:chimeric small molecules repurpose lysosomal membrane proteins for target protein relocalization and degradation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Proximity-inducing modalities that co-opt cellular pathways offer new opportunities to regulate oncogenic drivers. Inspired by the success of proximity-based chimeras in both intracellular and extracellular target space, here we describe the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370906/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370906">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63128-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63128.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370906%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370906/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370906/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370906/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7812. doi: <a href="https://doi.org/10.1038/s41467-025-63128-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63128-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>LYMTACs:chimeric small molecules repurpose lysosomal membrane proteins for target protein relocalization and degradation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nalawansha%20DA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Dhanusha A Nalawansha</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Dhanusha A Nalawansha</span></h3>
<div class="p">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nalawansha%20DA%22%5BAuthor%5D" class="usa-link"><span class="name western">Dhanusha A Nalawansha</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mazis%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Georgios Mazis</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Georgios Mazis</span></h3>
<div class="p">
<sup>2</sup>Amgen Research Copenhagen, Ronnegade 8, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mazis%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Georgios Mazis</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Husemoen%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Gitte Husemoen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Gitte Husemoen</span></h3>
<div class="p">
<sup>2</sup>Amgen Research Copenhagen, Ronnegade 8, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Husemoen%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gitte Husemoen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ashton%20KS%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kate S Ashton</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kate S Ashton</span></h3>
<div class="p">
<sup>3</sup>Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ashton%20KS%22%5BAuthor%5D" class="usa-link"><span class="name western">Kate S Ashton</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Weixian Deng</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Weixian Deng</span></h3>
<div class="p">
<sup>4</sup>Discovery Proteomics, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Weixian Deng</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurz%20RP%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ryan P Wurz</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ryan P Wurz</span></h3>
<div class="p">
<sup>3</sup>Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurz%20RP%22%5BAuthor%5D" class="usa-link"><span class="name western">Ryan P Wurz</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20AT%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Anh T Tran</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Anh T Tran</span></h3>
<div class="p">
<sup>3</sup>Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20AT%22%5BAuthor%5D" class="usa-link"><span class="name western">Anh T Tran</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lanman%20BA%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Brian A Lanman</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Brian A Lanman</span></h3>
<div class="p">
<sup>3</sup>Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lanman%20BA%22%5BAuthor%5D" class="usa-link"><span class="name western">Brian A Lanman</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jiansong Xie</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jiansong Xie</span></h3>
<div class="p">
<sup>5</sup>Cytometry and Imaging Sciences, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiansong Xie</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guenette%20RG%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Robert G Guenette</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Robert G Guenette</span></h3>
<div class="p">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guenette%20RG%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert G Guenette</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Shiqian Li</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Shiqian Li</span></h3>
<div class="p">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shiqian Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20CE%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Christopher E Smith</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Christopher E Smith</span></h3>
<div class="p">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20CE%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher E Smith</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Archunan%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Suresh Archunan</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Suresh Archunan</span></h3>
<div class="p">
<sup>6</sup>Syngene Amgen Research &amp; Development Center, Bengaluru, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Archunan%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suresh Archunan</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Agnihotram%20MK%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Manoj K Agnihotram</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Manoj K Agnihotram</span></h3>
<div class="p">
<sup>6</sup>Syngene Amgen Research &amp; Development Center, Bengaluru, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Agnihotram%20MK%22%5BAuthor%5D" class="usa-link"><span class="name western">Manoj K Agnihotram</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sadhukhan%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Arghya Sadhukhan</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Arghya Sadhukhan</span></h3>
<div class="p">
<sup>6</sup>Syngene Amgen Research &amp; Development Center, Bengaluru, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sadhukhan%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arghya Sadhukhan</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kapoor%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Rajiv Kapoor</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Rajiv Kapoor</span></h3>
<div class="p">
<sup>6</sup>Syngene Amgen Research &amp; Development Center, Bengaluru, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kapoor%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rajiv Kapoor</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilde%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Chris Wilde</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Chris Wilde</span></h3>
<div class="p">
<sup>7</sup>Lead Discovery and Characterization, Amgen Research, Thousand Oaks, CA 91320 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilde%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chris Wilde</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koirala%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Sajjan Koirala</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Sajjan Koirala</span></h3>
<div class="p">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koirala%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sajjan Koirala</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22De%20Sousa%20E%20Melo%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Felipe De Sousa E Melo</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Felipe De Sousa E Melo</span></h3>
<div class="p">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22De%20Sousa%20E%20Melo%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Felipe De Sousa E Melo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Potts%20PR%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Patrick Ryan Potts</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Patrick Ryan Potts</span></h3>
<div class="p">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Potts%20PR%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrick Ryan Potts</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Induced Proximity Platform, Amgen Research, Thousand Oaks, CA USA </div>
<div id="Aff2">
<sup>2</sup>Amgen Research Copenhagen, Ronnegade 8, Copenhagen, Denmark </div>
<div id="Aff3">
<sup>3</sup>Medicinal Chemistry, Amgen Research, Thousand Oaks, CA USA </div>
<div id="Aff4">
<sup>4</sup>Discovery Proteomics, Amgen Research, Thousand Oaks, CA USA </div>
<div id="Aff5">
<sup>5</sup>Cytometry and Imaging Sciences, Amgen Research, Thousand Oaks, CA USA </div>
<div id="Aff6">
<sup>6</sup>Syngene Amgen Research &amp; Development Center, Bengaluru, India </div>
<div id="Aff7">
<sup>7</sup>Lead Discovery and Characterization, Amgen Research, Thousand Oaks, CA 91320 USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 10; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370906  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841368/" class="usa-link">40841368</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Proximity-inducing modalities that co-opt cellular pathways offer new opportunities to regulate oncogenic drivers. Inspired by the success of proximity-based chimeras in both intracellular and extracellular target space, here we describe the development of <span class="text-underline">LY</span>sosome <span class="text-underline">M</span>embrane <span class="text-underline">TA</span>rgeting <span class="text-underline">C</span>himera<span class="text-underline">s</span> (LYMTACs) as a small molecule-based platform that functions intracellularly to modulate the membrane proteome. Conceptually, LYMTACs are heterobifunctional small molecules that co-opt short-lived lysosomal membrane proteins (LMPs) as effectors to deliver targets for lysosomal degradation. We demonstrate that a promiscuous kinase inhibitor-based LYMTAC selectively targets membrane proteins for lysosomal degradation via RNF152, a short-lived LMP. We extend this concept by showing that oncogenic KRAS<sup>G12D</sup> signaling can be potently inhibited by LYMTACs. Mechanistically, LYMTACs display multi-pharmacology and exert their activity through both target relocalization into the lysosome and degradation. We further generalize LYMTACs across various LMPs and thus offer a platform to access challenging membrane proteins through targeted protein relocalization and degradation.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Small molecules, Proteomic analysis, Medicinal chemistry, Confocal microscopy, Lysosomes</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">LYMTACs are heterobifunctional small molecules that take advantage of lysosomal membrane proteins to degrade membrane targets via relocalization and lysosomal degradation, offering a modular approach to drug otherwise intractable membrane proteins.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Plasma membrane-bound proteins play a central role in cellular signaling, and their deregulation has been implicated in a wide variety of diseases<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. While the majority of FDA-approved drugs engage the membrane proteome, a considerable proportion of membrane-bound targets remain undruggable by current targeted approaches, which are largely comprised of antibodies and small molecule inhibitors (SMI)<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Despite the potential clinical benefit in modulating membrane proteins, small molecule and antibody-based therapies often suffer from limitations such as stoichiometric target inactivation, challenges in target engagement due to lack of druggable sites, dose-dependent toxicities, and resistance mechanisms<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Hence, alternative therapeutic strategies are needed to expand the addressable pool of membrane targets more effectively. Of particular interest, targeted protein degradation (TPD) by proteolysis targeting chimeras (PROTACs) has demonstrated several advantages over inhibitor-based therapies<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. For instance, PROTACs can utilize a broader set of target binding ligands, including silent binders or functional binders. Second, degradation of a target removes the entire protein, including its scaffolding functions, essentially mimicking a genetic knockout. Even though PROTACs have been successful at degrading a wide array of cellular proteins, their utility for degrading membrane proteins has not been fully explored<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. This is, in part, because membrane proteins are primarily degraded through lysosomal pathways such as receptor-mediated endocytosis and autophagy and may not always be optimally accessible for PROTACs<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. In recent years, lysosome targeting chimeras (LYTACs), antibody-based PROTACs (AbTACs), and proteolysis targeting antibodies (PROTABs) have emerged as promising proximity-inducing modalities to target secreted and membrane proteins for lysosomal degradation; however, to date, such approaches have mainly relied on the use of antibodies<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Therefore, we anticipate small molecule-based strategies that function intracellularly for the degradation of membrane proteins through lysosomal machineries could expand the scope of TPD modalities.</p>
<p id="Par4">Lysosomal membrane proteins (LMPs) regulate essential cellular processes such as membrane repair, autophagy, phagocytosis, and viral infection<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. A recent study has demonstrated that a subset of LMPs, such as RNF152 and LAPTM4a, are short-lived and are internalized into the lysosome for degradation in a ubiquitin- and endosomal sorting complexes required for transport (ESCRT)-dependent manner, referred to as lysosomal microautophagy<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. We hypothesized that the rapid turnover and lysosomal proximity of LMPs could be co-opted to regulate the fate of proteins of interest (POIs) using proximity-inducing small molecules. We turned this concept into practice and have developed <span class="text-underline">Ly</span>sosome <span class="text-underline">M</span>embrane <span class="text-underline">Ta</span>rgeting <span class="text-underline">C</span>himeras (LYMTACs), heterobifunctional molecules composed of a POI ligand, a linker, and an LMP ligand (Fig. <a href="#Fig1" class="usa-link">1A</a>). Mechanistically, a LYMTAC induces the proximity between a POI and a short-lived LMP, thereby inducing relocalization of the target protein to the lysosomal membrane, promoting its internalization and subsequent degradation. By applying this concept to various membrane kinases and oncogenic KRAS<sup>G12D</sup>, we demonstrate the utility of LYMTACs in targeting membrane proteins for lysosomal clearance.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Development of LYMTAC as a modular tool for lysosomal degradation of membrane proteins.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370906_41467_2025_63128_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7163/12370906/238f2980f1ff/41467_2025_63128_Fig1_HTML.jpg" loading="lazy" id="d33e499" height="1020" width="794" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of the LYMTAC concept. A LYMTAC is a heterobifunctional molecule composed of a POI ligand, a linker and an LMP ligand. LYMTACs co-opt short-lived lysosomal membrane proteins as effectors to deliver target proteins for lysosomal degradation. <strong>B</strong> Cycloheximide (CHX) assay in FLAG-MTH1-RNF152 stably expressing HCT116 cells. Cells were pre-treated with DMSO, 200 nM Bafilomycin A1 (BafA1), or 1 µM ubiquitin E1 inhibitor (TAK-243) for 30 min, followed by co-treatment with 100 µg/mL CHX for 4 h. Cells were harvested at 0 h to include as a control. Cells were lysed and subjected to immunoblotting with MTH1 and vinculin antibodies. Quantified data are representative of two independent experiments. <strong>C</strong> Structure of LYMTAC-1 (promiscuous kinase inhibitor-PEG2-MTH1 ligand) and PROTAC TL-12-186. <strong>D</strong> Quantitative proteomics analysis of LYMTAC-1. HEK293 cells stably expressing FLAG-MTH1-RNF152 were treated with either DMSO or 500 nM LYMTAC-1 for 19 h and subjected to global proteomics analysis. Data are representative of three treatment replicates. Differential expression was assessed with the <strong>limma</strong> moderated <em>t</em>-test (empirical-Bayes framework, <em>two-sided</em>). Reported <em>P</em> values are <em>unadjusted</em>; no correction for multiple comparisons was applied. <strong>E</strong> Quantitative proteomics analysis of PROTAC (TL-12-186). HEK293 cells stably expressing FLAG-MTH1-RNF152 were treated with either DMSO or 500 nM PROTAC for 19 h and subjected to global proteomics analysis. Data are representative of three treatment replicates. Differential expression was assessed with the <strong>limma</strong> moderated <em>t</em>-test (empirical-Bayes framework, <em>two-sided</em>). Reported <em>P</em> values are <em>unadjusted</em>; no correction for multiple comparisons was applied. <strong>F</strong> LYMTAC-1 induced dose-dependent degradation of PTK2. HEK293 cells stably expressing FLAG-MTH1-RNF152 were treated with increasing concentrations of LYMTAC-1 for 19 h and subjected to immunoblotting. Quantified data are representative of three independent experiments. <strong>G</strong> LYMTAC-1 induced dose-dependent degradation of EPHA2. Quantified data are representative of three independent experiments. <strong>H</strong>, <strong>I</strong> HEK293 cells stably expressing MTH1-RNF152 were pre-treated with BafA1 and MG-132 for 30 min, co-treated with DMSO or 500 nM LYMTAC-1 for 19 h, and subjected to immunoblotting with the indicated antibodies. Quantified data are representative of two independent experiments.</p></figcaption></figure><p id="Par5">In this work, we demonstrate that LYMTACs are capable of exerting pharmacological activity through substrate relocalization and sequestration at the lysosome even in the absence of extensive degradation. This dual mode of action translated into a deeper suppression of KRAS<sup>G12D</sup> signaling and inhibition of cancer cell viability compared to reversible KRAS inhibitors. The scope of this platform is further highlighted by establishing that multiple LMPs, such as RNF152, LAMPT4a, and LAPTM5, can serve as effectors for the LYMTAC technology. We anticipate that the chemical modularity and tunability of small molecule LYMTACs will offer new opportunities to optimize such agents for the efficient degradation of diverse membrane-associated proteins.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Development of LYMTAC as a modular tool for lysosomal degradation of membrane proteins</h3>
<p id="Par6">To establish the LYMTAC technology, we used RNF152 as our primary LMP effector for proof-of-concept studies. Due to the lack of selective and well-characterized ligands for RNF152, we generated a model system using MTH1-tagged RNF152 (MTH1-RNF152) to allow for induced proximity to the POI using a selective MTH1 binding ligand<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Since MTH1 ligands have been successfully incorporated into heterobifunctional molecules and tested in chemical-induced dimerization studies, we used the MTH1 tag and its ligands to establish the LYMTAC concept<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Importantly, MTH1 tagging did not alter the turnover rate of RNF152 as shown by almost complete degradation of MTH1-RNF152 after only four hours of protein synthesis inhibition by cycloheximide in HEK293 and HCT116 cells (Fig. <a href="#Fig1" class="usa-link">1B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Furthermore, this rapid turnover of MTH1-RNF152 was both ubiquitin- and lysosome-dependent as evidenced by inhibition by the ubiquitin E1 inhibitor (TAK-243) and lysosomal V-ATPase inhibitor (Bafilomycin A1) (Fig. <a href="#Fig1" class="usa-link">1B</a>).</p>
<p id="Par7">After establishing the MTH1-RNF152 LMP model cellular system, we probed the impact of LYMTAC on target protein degradation. As such, we devised an untargeted approach using a promiscuous kinase inhibitor as the POI ligand<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. We generated a multi-kinase targeting LYMTAC (LYMTAC-1) by linking the promiscuous kinase inhibitor, TL-13-87, to an MTH1-targeting ligand (Fig. <a href="#Fig1" class="usa-link">1C</a>). This non-selective kinase inhibitor binds to a vast array of membrane, cytoplasmic, and nuclear protein kinases, thus allowing us to identify the target space that is most readily addressed by LYMTACs. HEK293 cells stably expressing MTH1-RNF152 were treated with LYMTAC-1 and subjected to global proteomics to identify degradable protein targets in an unbiased manner. A total of six protein kinases were significantly downregulated by LYMTAC-1, validating the ability of LYMTACs to drive target protein degradation (Fig. <a href="#Fig1" class="usa-link">1D</a>). Strikingly, five out of six kinases that were downregulated by LYMTAC-1 were exclusively integral membrane proteins (EPHA2, INSR, EPHB4, ACVR1, TGFBR2)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par8">To directly compare the degradable protein pool of pan-kinase LYMTAC to a PROTAC functionalized with the same pan-kinase inhibitor, HEK293 cells were treated with previously reported pan-kinase PROTAC (TL-12-186) and subjected to global proteomics. Consistent with previous reports, the pan-kinase PROTAC mostly induced degradation of cytosolic and nuclear kinases (Fig. <a href="#Fig1" class="usa-link">1E</a>)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Even though approximately 300 kinases were detected in both experiments, the PROTAC targeted a small fraction of membrane proteins (6 out of 67 downregulated kinases) for degradation. This is in stark contrast to LYMTAC-1, which mainly induced downregulation of membrane protein kinases (5 out of 6 downregulated kinases were membrane kinases) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). This includes ACVR1, INSR, and TGFBR2, which are uniquely degraded by LYMTAC-1 and not by TL-12-186. Thus, LYMTACs provide access to a distinct pool of cellular targets. We speculate that the observed substrate selectivity may be due to the distinct pathways co-opted by the two effectors, RNF152 and CRBN, which engage the lysosomal and proteasomal degradation pathways, respectively. By engaging the endo-lysosomal degradation pathway, LYMTACs may be more suitable for degrading membrane proteins, given that membrane proteins are predominantly degraded via the endo-lysosomal pathway.</p>
<p id="Par9">To corroborate the proteomics results, HEK293 cells stably expressing MTH1-RNF152 were treated with increasing concentrations of LYMTAC-1 and subjected to immunoblotting. Consistent with the proteomics data, LYMTAC-1 induced PTK2 and EPHA2 degradation in a dose-dependent manner, with robust degradation observed at 10 nM (Fig. <a href="#Fig1" class="usa-link">1F</a>, <a href="#Fig1" class="usa-link">G</a>). LYMTAC-1 activity was dependent on RNF152, since the addition of excess MTH1 ligand abrogated PTK2 degradation (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Next, we investigated the route of degradation. Cells were pre-treated with either lysosome (Bafilomycin A1) or proteasome (MG-132) inhibitors, and LYMTAC-1-induced degradation of PTK2 and EPHA2 was assessed. LYMTAC-1-induced PTK2 and EPHA2 degradation was rescued by Bafilomycin A1 but not MG-132, consistent with lysosomal-mediated clearance of these membrane proteins, further confirming the anticipated degradation mechanism of LYMTACs (Fig. <a href="#Fig1" class="usa-link">1H</a>, <a href="#Fig1" class="usa-link">I</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">LYMTACs induce ubiquitylation and lysosomal degradation of KRAS<sup>G12D</sup>
</h3>
<p id="Par10">Having established the ability of LYMTACs to target membrane proteins for degradation, we sought to further investigate the therapeutic potential of these agents in degrading a historically challenging-to-drug protein, KRAS<sup>G12D</sup>. KRAS<sup>G12D</sup> is a membrane-anchored protein that has been associated with the tumorigenic capacity of many cancer types<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. We first utilized a chemical genetic system by co-expressing MTH1-RNF152 effector and HiBiT-FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup> POI (Fig. <a href="#Fig2" class="usa-link">2A</a>)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. We chose to first establish this system using dimerization tags in cells not dependent on KRAS signaling for survival. Endogenously tagged HCT116 cells were generated by knocking in HiBiT-FKBP12<sup>F36V</sup> at the KRAS locus, where gene editing was used to introduce G12D by correcting the G13D mutation that was present naturally in HCT116 cells. For proof of concept, we chose to leverage FKBP12<sup>F36V</sup>-tagged proteins of interest (POIs) as a suitable target protein given the availability of potent and selective ligands for FKBP12<sup>F36V</sup>. Additionally, the use of FKBP12<sup>F36V</sup>-tagging allowed us to quantitatively assess KRAS<sup>G12D</sup> levels and ligand engagement without the complications associated with the use of pharmacologically active inhibitors in cells sensitive to KRAS<sup>G12D</sup> depletion. Next, we designed LYMTAC-2, comprising an FKBP12<sup>F36V</sup> ligand, a connecting linker, and an MTH1 ligand to dimerize the target and effector proteins (Fig. <a href="#Fig2" class="usa-link">2B</a>). In line with our pan-kinase data, RNF152-recruiting LYMTAC-2 induced potent but partial KRAS<sup>G12D</sup> degradation in a dose-dependent manner after 24 h (Fig. <a href="#Fig2" class="usa-link">2C</a>). However, LYMTAC-2-induced KRAS degradation was not observed in the absence of MTH1-RNF152, demonstrating that LYMTAC-induced KRAS degradation is dependent on MTH1-RNF152. To further understand the effect of untagged RNF152 (which mimics the endogenous RNF152) on LYMTAC’s function, we overexpressed varying amounts of untagged RNF152 in HCT116 cells and assessed its impact on LYMTAC-induced KRAS degradation in cells stably expressing MTH1-RNF152. Overexpression of RNF152 did not alter the efficiency of LYMTAC-induced KRAS degradation, even at the highest overexpression level (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. LYMTACs induce ubiquitylation and lysosomal degradation of KRAS<sup>G12D</sup>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370906_41467_2025_63128_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7163/12370906/56544981ae6d/41467_2025_63128_Fig2_HTML.jpg" loading="lazy" id="d33e675" height="826" width="799" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of the chemical genetic system where the target protein is HiBiT-FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup>, and the effector is MTH1-RNF152. <strong>B</strong> Structure of LYMTAC-2 (FKBP12<sup>F36V</sup> ligand-PEG2-MTH1 ligand). <strong>C</strong> HiBiT cellular degradation assay in knock-in HCT116 (HiBiT- FKBP12<sup>F36V</sup>-KRAS<sup>G12D:</sup> (HF-KRAS)) cells in the absence or presence of stably expressing MTH1-RNF152. Cells were treated with increasing concentrations of LYMTAC-2 for 24 h, and HiBiT levels were measured using Nano-Glo® HiBiT lytic reagent. Data are representative of three independent experiments, reported as the mean ± S.E. <strong>D</strong> HCT116 (HF-KRAS) cells stably expressing FLAG-MTH1-RNF152 were pre-treated with BafA1, MG-132, or TAK-243 for 30 min followed by co-treatment with DMSO or 500 nM LYMTAC-2 for 6 h and subjected to immunoblotting with KRAS and tubulin antibodies. Quantified data are representative of two independent experiments. <strong>E</strong> KRAS<sup>G12D</sup> ubiquitylation assay. HA-ubiquitin was transfected into HCT116, pre-treated with 200 nM BafA1 for 30 min, then co-treated with DMSO or 1 µM LYMTAC-2 for 4 h. Next, cells were lysed and subjected to immunoprecipitation with anti-HiBiT antibody. Whole cell lysate was used as the input. The respective blots are probed with HA, MTH1, HiBiT, and vinculin antibodies. Data are representative of three independent experiments.</p></figcaption></figure><p id="Par11">To obtain additional insights on LYMTAC-2-induced KRAS<sup>G12D</sup> degradation, HCT116 cells were pre-treated with inhibitors such as Bafilomycin A1, MG-132, or TAK-243, and then co-treated with LYMTAC-2 and subjected to immunoblotting. LYMTAC-2-mediated KRAS<sup>G12D</sup> degradation was rescued by both ubiquitin E1 (TAK-243) and lysosomal (Bafilomycin A1) inhibitors (Fig. <a href="#Fig2" class="usa-link">2D</a>). However, LYMTAC-induced KRAS<sup>G12D</sup> degradation was not rescued by proteasome inhibition. These data indicate that LYMTAC-2-mediated KRAS<sup>G12D</sup> degradation follows the native ubiquitin- and lysosome-dependent turnover of RNF152. RNF152 is an E3 ubiquitin ligase LMP whose rapid internalization and lysosomal degradation are dependent on its own auto-ubiquitylation<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Thus, we speculated that LYMTAC-2 may drive degradation of KRAS<sup>G12D</sup> through RNF152-dependent ubiquitylation of KRAS<sup>G12D</sup>. To this end, we performed a cellular ubiquitylation assay by transiently expressing HA-ubiquitin in HCT116 cells, followed by immunoprecipitation and immunoblotting. Notably, cells treated with LYMTAC-2 revealed rapid KRAS<sup>G12D</sup> ubiquitylation (Fig. <a href="#Fig2" class="usa-link">2E</a>). Collectively, our data suggest that LYMTACs can lead to the formation of a productive ternary complex with KRAS<sup>G12D</sup> and RNF152, promote KRAS<sup>G12D</sup> ubiquitylation, and induce lysosome-dependent clearance of KRAS<sup>G12D</sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">KRAS<sup>G12D</sup> LYMTACs induce pathway suppression irrespective of KRAS<sup>G12D</sup> degradation</h3>
<p id="Par12">Emerging resistance to KRAS inhibitors (KRASi) is a critical challenge in maximizing the clinical potential of these drugs<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Degradation strategies that remove rather than inhibit oncogenic proteins offer the potential of addressing a number of such resistance mechanisms, which has encouraged our development of proximity-inducing approaches to degrade membrane-associated oncogenic KRAS. Having established and characterized the biochemical activity of KRAS<sup>G12D</sup> LYMTACs, we next investigated the downstream consequences of lysosomal-mediated degradation of KRAS<sup>G12D</sup>. To target endogenous KRAS<sup>G12D</sup> directly, we generated KRAS<sup>G12D</sup> LYMTACs by replacing the FKBP12<sup>F36V</sup> ligand with a pan-KRAS inhibitor (pan-KRASi) (Fig. <a href="#Fig3" class="usa-link">3A</a>, <a href="#Fig3" class="usa-link">B</a>). Extensive research in the TPD field has defined several common structural features that influence the efficacy of PROTAC design such as exit vector orientation, linker length, linker composition, and E3 ligase ligand<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. X-ray crystallographic characterization of the binding mode of the pan-KRAS inhibitor has suggested attachment of the effector ligand via the fused bicyclo-pyrollidine ring could yield active PROTACs and LYMTACs.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. KRAS LYMTACs induce pathway suppression irrespective of KRAS degradation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370906_41467_2025_63128_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7163/12370906/807d100a335d/41467_2025_63128_Fig3_HTML.jpg" loading="lazy" id="d33e790" height="692" width="798" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of complex formation between KRAS<sup>G12D</sup> and MTH1-RNF152 in the presence of KRAS-targeting LYMTAC. <strong>B</strong> Structures of the pan-KRAS inhibitor and PROTAC <strong>C</strong> Structures of LYMTAC-3 and LYMTAC-4. <strong>D</strong> HiBiT cellular degradation assay in FLAG-MTH1-RNF152-stably expressing, knock-in HCT116 (HiBiT-FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup>) cells. Cells were treated with indicated doses of compounds for 24 h and subjected to Nano-Glo® HiBiT lytic assay. Data are representative of three independent experiments, reported as the mean ± S.E. <strong>E</strong> Ternary complex formation assay, AsPC-1 cells stably expressing FLAG-MTH1-RNF152 were pre-treated with BafA1 for 30 min, followed by co-treatment with DMSO or 1 µM LYMTAC-3 or LYMTAC-4 for 4 h and subjected to immunoprecipitation with anti-FLAG antibody. Whole cell lysate was used as the input. The respective blots are probed with KRAS, MTH1, and vinculin antibodies. Data are representative of two independent experiments. <strong>F</strong> AsPC-1 cells stably expressing FLAG-MTH1-RNF152 were treated with DMSO or 100 nM of the indicated compounds for 24 h and subjected to immunoblotting with KRAS, pERK, ERK, and vinculin antibodies. Quantified data are representative of two independent experiments. <strong>G</strong> AsPC-1 cells stably expressing FLAG-MTH1-RNF152 were treated with 100 nM KRAS inhibitor, 100 nM PROTAC, or 100 nM LYMTAC-4 for 6 h. For control experiments with LYMTAC-4, cells were pretreated with MTH1-ligand or BafA1 for 30 min, followed by co-treatment in the absence or presence of 100 nM LYMTAC-4 for 6 h, and subjected to immunoblotting with KRAS, p-ERK, ERK, and vinculin antibodies. Quantified data are representative of two independent experiments. <strong>H</strong> HEK293 cells stably expressing HiBiT-FKBP12-KRAS<sup>G12D</sup> and FLAG-MTH1-RNF152 were pre-treated with 1 µM TAK-243 for 30 min followed by DMSO or 1 µM LYMTAC-2 treatment for 6 h. The respective blots are probed with KRAS, p-ERK, ERK, and tubulin antibodies. Quantified data are representative of two independent experiments.</p></figcaption></figure><p id="Par13">First, we screened KRAS LYMTACs with two different linker compositions for cellular HiBiT- FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup> degradation using HCT116 cells and compared their efficacy to that achieved through PROTAC-mediated degradation (Fig. <a href="#Fig3" class="usa-link">3B</a>, <a href="#Fig3" class="usa-link">C</a>). While our KRAS PROTAC and two LYMTACs displayed varying levels of KRAS<sup>G12D</sup> degradation, rigid linker-based LYMTAC-4 (which also tethered the MTH1 ligand to the pan-KRAS inhibitor via a shorter linker) displayed improved D<sub>max</sub> (61%) and DC<sub>50</sub> (2 nM) values compared to PEG linker-based LYMTAC-3 [D<sub>max</sub> = 48%, DC<sub>50</sub> = 25 nM), Fig. <a href="#Fig3" class="usa-link">3D</a>). Leveraging the more potent LYMTAC-4 in subsequent experiments, we functionally characterized the effect of LYMTAC-induced KRAS<sup>G12D</sup> degradation in AsPC-1 cells, a relevant KRAS<sup>G12D</sup> mutant pancreatic cancer cell line that relies on this oncogene for cell proliferation and survival<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. MTH1-RNF152 was stably introduced into AsPC-1 cells to evaluate ternary complex formation (TCF) by various LYMTACs. The data suggest that LYMTAC-4 induced a more productive ternary complex between KRAS<sup>G12D</sup> and RNF152, compared to LYMTAC-3 (Fig. <a href="#Fig3" class="usa-link">3E</a>). Experience with this system further supported the selection of LYMTAC-4 as an appropriate tool molecule for subsequent investigation. To evaluate the cellular effect of LYMTAC-4, AsPC-1 cells were treated with pan-KRASi, PROTAC, or LYMTAC-4 for 24 h and probed for pathway activity. LYMTAC-4 induced KRAS degradation and robust pERK downregulation at 24 h, comparable to pan-KRASi and PROTAC (Fig. <a href="#Fig3" class="usa-link">3F</a>). Additionally, to test the ability of KRAS LYMTACs to degrade other KRAS variants, we tested LYMTAC-4 in HCT116 harboring the KRAS<sup>G13D</sup> mutation. Consistent with the ability of the KRAS inhibitor to bind G13D, we observed LYMTAC-4 induced KRAS<sup>G13D</sup> degradation in HCT116 cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>).</p>
<p id="Par14">We next sought to further elucidate LYMTAC mode of action and its dependence on ternary complex formation (TCF). This was investigated by treating AsPC-1 cells with LYMTAC-4 for 6 h in the absence or presence of excessive competing MTH1-ligand or Bafilomycin A1. Excess MTH1 ligand abrogated LYMTAC-induced KRAS<sup>G12D</sup> degradation and p-ERK signaling, demonstrating that TCF is required to induce KRAS<sup>G12D</sup> degradation and inhibit downstream signaling (Fig. <a href="#Fig3" class="usa-link">3G</a>). As expected, Bafilomycin A1 treatment rescued LYMTAC-4-induced KRAS<sup>G12D</sup> degradation. Unexpectedly, in Bafilomycin A1-treated cells, LYMTAC-4 still suppressed KRAS<sup>G12D</sup> downstream signaling even in the absence of KRAS<sup>G12D</sup> degradation. To rule out that p-ERK downregulation was due to the pan-KRASi moiety of LYMTAC-4, we stably co-expressed FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup> and MTH1-RNF152 in HEK293 cells and engaged FKBP12-KRAS with a non-inhibitory FKBP12 ligand containing LYMTAC-2 (FKBP12 ligand-PEG2-MTH1 ligand). Strikingly, FKBP12-targeting LYMTAC-2 also suppressed p-ERK signaling irrespective of KRAS ubiquitylation and degradation (Fig. <a href="#Fig3" class="usa-link">3H</a>). Collectively, these data provide compelling evidence that pathway inhibition is not due to the inhibitory effect of pan-KRASi-based LYMTACs but could be attributed to the ability of LYMTACs to affect target biology beyond target degradation. Overall, the data indicate that LYMTAC-induced KRAS ubiquitylation and degradation are not uniformly required for downstream signaling inhibition, while TCF is critical to suppress p-ERK signaling.</p></section><section id="Sec6"><h3 class="pmc_sec_title">KRAS relocalization and lysosomal degradation using LYMTAC lead to deeper pathway suppression and potent cell killing compared to SMI</h3>
<p id="Par15">The observation that signaling downstream of KRAS<sup>G12D</sup> can be modulated by LYMTACs even in the absence of KRAS<sup>G12D</sup> degradation prompted us to further explore whether quantitative relocalization of KRAS from the plasma membrane to the lysosome could be driving this effect (Fig. <a href="#Fig4" class="usa-link">4A</a>). To address this hypothesis, we generated mNeonGreen-KRAS<sup>WT</sup> stably expressing HEK293 cells to allow for subcellular KRAS tracking by confocal microscopy. After generating the cell line, we also verified in a CHX assay that addition of the mNeonGreen tag does not significantly alter the half-life of KRAS (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). Pre-treatment of these cells with Bafilomycin A1 followed by LYMTAC-4 revealed rapid and marked KRAS relocalization from the plasma membrane followed by co-localization with RNF152 and the lysosomal marker LAMP1 (Fig. <a href="#Fig4" class="usa-link">4B</a>). This was specific for LYMTACs, as this phenotype was absent in cells treated with pan-KRASi or PROTAC (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). We performed time lapse imaging in HEK293 cells to illustrate the kinetics of LYMTAC-4 induced KRAS relocalization. Internalization was evident as early as 30 min and gradually increased in a time-dependent manner (Fig. <a href="#Fig4" class="usa-link">4C</a>). We believe that the kinetics of relocalization may depend on several factors, such as the expression levels of MTH1-RNF152 and KRAS in each cell, and the efficiency of ternary complex formation.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. KRAS relocalization and lysosomal degradation using LYMTAC lead to deeper pathway suppression and potent cell killing compared to SMI.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370906_41467_2025_63128_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7163/12370906/320f6f8c57f0/41467_2025_63128_Fig4_HTML.jpg" loading="lazy" id="d33e918" height="1236" width="784" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of LYMTAC-induced KRAS relocalization from plasma membrane to lysosome. <strong>B</strong> KRAS localization analyzed by confocal microscopy. FLAG-MTH1-RNF152-stably expressing HEK293 mNeonGreen-KRAS <sup>WT</sup> cells were pre-treated with 200 nM BafA1 for 30 min and then co-treated with DMSO or 1 µM LYMTAC-4 for 4 h. Cells were fixed, permeabilized, and immunostained with FLAG and LAMP1 antibodies, followed by appropriate secondary fluorophore-conjugated antibodies. Nuclei were stained with Hoechst dye. Data are representative of two independent experiments. <strong>C</strong> Kinetics of LYMTAC-4 induced KRAS relocalization. FLAG-MTH1-RNF152-stably expressing HEK293 mNeonGreen-KRAS <sup>WT</sup> cells were pre-treated with 200 nM BafA1 for 30 min and then co-treated with 1 µM LYMTAC-4. Live cell images were acquired every 30 min up to 4 h. Puncta formation in each cell is highlighted with orange, green and pink arrows at each time point. Data are representative of two independent experiments. <strong>D</strong> Washout experiment in AsPC-1 cells stably expressing FLAG-MTH1-RNF152. Cells were treated with 100 nM pan-KRASi and 100 nM LYMTAC-4 for 4 h, washed three times with PBS and replaced with fresh media in the absence or presence of 5 µM MTH1-ligand for 48 h. Quantified data are representative of two independent experiments. <strong>E</strong> pan-KRASi, PROTAC, and LYMTAC-4 activity in a 5-day cell proliferation assay in AsPC-1 cells stably expressing MTH1-RNF152. Data are representative of three independent experiments, reported as the mean ± S.E.</p></figcaption></figure><p id="Par16">KRAS is a plasma membrane-anchored protein that activates oncogenic signaling at the membrane via multiple effector proteins<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Various attempts to block KRAS trafficking to the plasma membrane have been made, yet these approaches have been largely unsuccessful<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. In addition, membrane-bound KRAS is susceptible to pathway rebound driven by upstream receptor tyrosine kinases such as EGFR, ultimately contributing to therapeutic resistance<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Given that LYMTACs display dual functionality in the inactivation of membrane proteins (i.e., via intracellular relocalization and via subsequent lysosomal degradation), their distinct mechanism of action may offer multiple advantages in overcoming some of the limitations commonly seen with current therapies. We explored this possibility by measuring pathway rebound upon drug withdrawal. AsPC-1 cells were treated with pan-KRASi or LYMTAC-4 for four hours, excess compound was washed away, and fresh media was added for 48 h before analysis. Consistent with the reported KRASi-induced signaling rebound, we observed a p-ERK signal rebound after 48 h in the pan-KRASi-treated samples (Fig. <a href="#Fig4" class="usa-link">4D</a>). Strikingly, LYMTAC-4 induced sustained KRAS<sup>G12D</sup> degradation and delayed rebound of p-ERK signaling compared to pan-KRASi (Fig. <a href="#Fig4" class="usa-link">4D</a>). Excess MTH1 ligand rescued KRAS<sup>G12D</sup> degradation and p-ERK inhibition, indicating that this effect was on target and dependent on RNF152 (Fig. <a href="#Fig4" class="usa-link">4D</a>). Notably LYMTAC-4 suppressed downstream p-ERK signaling similarly to a highly optimized clinically used KRASi, MRTX1133, that does not display p-ERK signaling rebound after washout (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). Given that LYMTAC-4 maintained prolonged signaling inhibition, we sought to further monitor the effect of LYMTACs on AsPC-1 cell proliferation. Importantly, the depth of pathway suppression also translated phenotypically as LYMTAC-4 demonstrated improved cell killing compared to both KRASi and PROTAC (Fig. <a href="#Fig4" class="usa-link">4E</a>). This potent cell killing could be attributed to the multi-pharmacology displayed by LYMTACs via KRAS relocalization, sustained p-ERK inhibition, and lysosomal degradation.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Expansion of the LYMTAC platform to other LMPs</h3>
<p id="Par17">To demonstrate the generality of our platform, we targeted additional LMP effectors beyond RNF152. We focused on LAMPT4a and LAPTM5, two LMPs that associate with the NEDD4-1 E3 ubiquitin ligase at the lysosome membrane through PY motifs (Fig. <a href="#Fig5" class="usa-link">5A</a>)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. After generating cells stably expressing MTH1-tagged LAPTM4a and LAPTM5, we first performed a cycloheximide assay in HEK293 cells to confirm their short-lived nature. The data indicate that both MTH1-LAPTM4a and MTH1-LAPTM5 are rapidly degraded within four hours in HEK293 cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). Similarly to HEK293 cells, HCT116 cells stably expressing MTH1-tagged LAPTM4a or LAPTM5 showed rapid LMP turnover in a ubiquitin and lysosome-dependent manner (Fig. <a href="#Fig5" class="usa-link">5B</a>, <a href="#Fig5" class="usa-link">C</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Expansion of LYMTAC platform to other LMPs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370906_41467_2025_63128_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7163/12370906/42edd7860b16/41467_2025_63128_Fig5_HTML.jpg" loading="lazy" id="d33e1019" height="1005" width="799" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of Lysosome Membrane Proteins (LMPs). <strong>B</strong> CHX assay in FLAG-MTH1-LAPTM4a-stably expressing HCT116 cells. Cells were pre-treated with DMSO, 200 nM Bafilomycin A1 (BafA1), or 1 µM E1 inhibitor (TAK-243) for 30 min, followed by co-treatment with CHX for 4 h. Cells were harvested at 0 h to include as a control. Cells were lysed and subjected to immunoblotting with MTH1 and vinculin antibodies. Quantified data are representative of two independent experiments. <strong>C</strong> CHX assay in FLAG-MTH1-LAPTM5-stably expressing HCT116 cells. Quantified data are representative of two independent experiments. <strong>D</strong> HiBiT cellular degradation assay in FLAG-MTH1-LAPTM4a- and FLAG-MTH1-LAPTM5-stably expressing HCT116 (knock-in-HiBiT- FKBP12<sup>F36V</sup>-KRAS<sup>G12D:</sup> (HF-KRAS)) cells. Cells were treated with increasing concentrations of LYMTAC-2 for 24 h, and HiBiT levels were measured using Nano-Glo® HiBiT lytic reagent. Data are representative of three independent experiments and reported as the mean ± S.E. <strong>E</strong> HCT116 (HF-KRAS) cells stably expressing FLAG-MTH1-LAPTM5 were pre-treated with BafA1, 2 BTZ, or TAK-243 for 30 min, followed by co-treatment with DMSO or 500 nM LYMTAC-2 for 6 h and subjected to immunoblotting with KRAS and tubulin antibodies. Quantified data are representative of two independent experiments. <strong>F</strong> KRAS ubiquitylation assay. HA-ubiquitin was transfected into HCT116 expressing FLAG-MTH1-LAPTM5, which were pre-treated with 200 nM BafA1 for 30 min, followed by co-treatment with DMSO or 1 µM LYMTAC-2 treatment for 4 h. Next, cells were lysed and subjected to immunoprecipitation with anti-HiBiT antibody. Whole cell lysate was used as the input. The respective blots were probed with HA, MTH1, HiBiT, and vinculin antibodies. Data are representative of two independent experiments. <strong>G</strong> AsPC-1 cells stably expressing FLAG-MTH1-LAPTM4a were pre-treated with either pan-KRAS inhibitor or MTH1 ligand, followed by DMSO or 100 nM LYMTAC-4 treatment for 6 h. Cells were lysed and subjected to immunoblotting with KRAS, pERK, and tubulin antibodies. Quantified data are representative of two independent experiments.</p></figcaption></figure><p id="Par18">Having confirmed the fast turnover of MTH1-LAPTM4a and MTH1-LAPTM5, we next asked whether these two LMPs can induce target protein degradation in a LYMTAC-dependent manner. To this end, we utilized HiBit-FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup> knock-in HCT116 cells and stably expressed MTH1-LAPTM4a or MTH1-LAPTM5. Consistent with results seen for MTH1-RNF152, both MTH1-LAPTM4a and MTH1-LAPTM5 induced potent KRAS<sup>G12D</sup> degradation in a LYMTAC-2-dependent manner (Fig. <a href="#Fig5" class="usa-link">5D</a>), validating these two LMPs as effectors for the LYMTAC technology. To verify the mechanism of action for LAPTM5-based LYMTACs, HCT116 cells were treated with lysosomal (Bafilomycin A1), proteasomal (bortezomib), and ubiquitin E1 (TAK-243) inhibitors prior to LYMTAC-2 treatment and subjected to immunoblotting. Consistent with RNF152-recruiting LYMTACs, LAPTM5-induced KRAS<sup>G12D</sup> degradation was rescued by both lysosome and ubiquitin E1 inhibitors, but not proteosome inhibitor (Fig. <a href="#Fig5" class="usa-link">5E</a>). To further understand the ubiquitin-dependent mechanism of action, cellular ubiquitylation assays were performed. As expected, LAPTM5 formed a ternary complex with KRAS<sup>G12D</sup> and induced KRAS<sup>G12D</sup> ubiquitylation in the presence of LYMTAC-2 (Fig. <a href="#Fig5" class="usa-link">5F</a>).</p>
<p id="Par19">Finally, to expand LMPs that can be harnessed for LYMTAC development, we generated MTH1-LAPTM4a expressing AsPC-1 cells and confirmed the rapid degradation of this construct as had been observed with prior LMP (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). We further confirmed that KRAS<sup>G12D</sup> levels were unaffected by overexpressing either MTH1-RNF152 or MTH1-LAPTM4a in the absence of LYMTAC molecules (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). Similarly to RNF152, LAPTM4a formed a productive ternary complex with KRAS<sup>G12D</sup> in the presence of LYMTAC-4 (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>). Moreover, LAPTM4a-recruiting LYMTAC-4 induced KRAS<sup>G12D</sup> degradation and p-ERK inhibition in AsPC-1 cells without affecting the levels of MTH1-LAPTM4a (Fig. <a href="#Fig5" class="usa-link">5G</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). To further verify the mechanism of action, cells were treated with either pan-KRASi or MTH1-ligand prior to LYMTAC-4 treatment. The ligand competition data indicate that LYMTAC-induced KRAS<sup>G12D</sup> degradation and p-ERK inhibition are LAPTM4a-dependent and require a ternary complex between KRAS<sup>G12D</sup> and LAPTM4a.</p>
<p id="Par20">In summary, we describe here a SM-based LYMTAC platform that functions by repurposing short-lived lysosomal membrane protein effectors to induce membrane protein relocalization and subsequent degradation. Using a promiscuous kinase inhibitor-based LYMTAC, we demonstrate that LYMTACs preferentially deliver membrane kinases for lysosomal degradation. The observed substrate selectivity was a differentiating feature of LYMTAC-driven degradation when compared to cytosolic and nuclear protein degradation by a promiscuous kinase inhibitor-based PROTAC. The selectivity of LYMTACs towards membrane-associated proteins is likely explained by their natural turnover in the lysosome and continuous flux in the membrane trafficking pathways that offer opportunities for these membrane proteins to be captured by LMPs. Furthermore, using oncogenic KRAS<sup>G12D</sup> as a therapeutically relevant, membrane-associated target, we demonstrated that LYMTACs induce sustained pathway inhibition and potent cell killing as compared to a KRAS inhibitor. Notably, LYMTAC-induced KRAS relocalization from the plasma membrane to the lysosome is sufficient to suppress downstream signaling. While we observe a lack of complete KRAS degradation (D<sub>max</sub> ~50–60%) with LYMTACs, we observe quantitative relocalization of KRAS from plasma membrane to lysosomes, potent inhibition of p-ERK signaling, and cytotoxicity.</p>
<p id="Par21">Our proof-of-concept results demonstrate that the LYMTAC technology could be an effective strategy to sequester KRAS away from the plasma membrane. This is reminiscent of proximity-inducing modalities that have recently been shown to modulate target protein via non-degradative mechanisms such as relocalization and sequestration<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. In addition to the in-depth characterization of RNF152, we validated the LYMTAC technology with other LMPs such as LAPTM4a and LAPTM5, clearly demonstrating the potential compatibility of many more LMPs with LYMTAC technology. Overall, LYMTACs are a small molecule proximity-based platform that function intracellularly to inhibit oncogenic signaling through multi-pharmacology, including relocalization (occupancy-driven pharmacology) and lysosomal degradation (event-driven pharmacology) of membrane proteins (Fig. <a href="#Fig6" class="usa-link">6</a>). Other modalities such as LYTACs, PROTABs, and AbTACs have also emerged as novel platforms to regulate the membrane proteome. A potential advantage of LYMTAC compared to existing antibody-based technologies would be that 1) LYMTAC is a small molecule-based platform with dual mechanism of action and engages membrane targets from the cytosolic side, and 2) has the potential to target both integral membrane and membrane-anchored proteins for relocalization and lysosomal degradation. However, compared to LYTACs, LYMTACs are unable to target extracellular proteins for degradation.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. LYMTACs display multi-pharmacology.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370906_41467_2025_63128_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7163/12370906/d9aa93cc7204/41467_2025_63128_Fig6_HTML.jpg" loading="lazy" id="d33e1097" height="676" width="675" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>LYMTAC is an SM-based, proximity-inducing modality that functions intracellularly to relocalize and degrade membrane proteins via the lysosome. LYMTACs inhibit POI signaling via relocalization and subsequent lysosomal degradation.</p></figcaption></figure><p id="Par22">While we introduce LYMTAC as a chemical biology-driven proof-of-concept study, these MTH1-derived LYMTAC tool molecules can be utilized in fundamental research to functionally characterize the effect of target protein relocalization and lysosomal degradation. Identification of potent and selective ligands for short-lived LMPs remains a key challenge to advance the LYMTAC technology beyond the chemical biology concept presented here. We speculate that techniques such as DNA-encoded library screening, chemoproteomics, and fragment-based library screening will facilitate the discovery of ligands for multiple lysosomal membrane proteins in the future. Similarly to the challenges encountered with PROTAC delivery in animal models, extensive optimization will be required to deliver LYMTACs into animal models in the future. Future work should focus on understanding the target scope, LMP space (including cell-type specificity), and ligand discovery for LMPs to enable the LYMTAC technology for therapeutic development.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Cell lines</h3>
<p id="Par23">Human embryonic kidney 293 (HEK293), AsPC-1, and HCT116 cells were obtained from ATCC. HEK293 cells were cultured in DMEM (Gibco) supplemented with 10% (v/v) heat-inactivated fetal bovine serum (FBS) and 1% penicillin/streptomycin (Pen/strep) (v/v). AsPC-1 and HCT116 cells were grown in RPMI media supplemented with 10% FBS and 1% Pen/strep. Jump-In™GripTite™ HEK293 cells (A14150) were obtained from Invitrogen and grown in DMEM Glutamax supplemented with 10% FBS, 0.1 mM NEAA, 25 mM HEPES, 1x Pen/strep, 600 µg/mL Geneticin. Cells were cultured at 37 °C and 5% CO<sub>2</sub> in a humidified incubator and regularly screened for mycoplasma contamination.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Antibodies and reagents</h3>
<p id="Par24">Antibodies against MTH1 (43918), FLAG (14793), Vinculin (13901), Tubulin (2144), pERK (9101), ERK (4695), PTK2 (3285), EPHA2 (6997), HA (3724), LAMP1(15665), and GAPDH (2118) were purchased from Cell Signaling Technology. HiBiT antibody (N7200) was obtained from Promega, and KRAS antibody (LS-C175665) was purchased from LSBio. RNF152 antibody (PA5-95817) was purchased from Invitrogen. Alexa Fluor 568 (A11011) and Alexa Fluor 647 (A32728) conjugated secondary antibodies and Hoechst 33342 stain (H3570) were purchased from Invitrogen. Amersham ECL Rabbit IgG, HRP-linked whole secondary Ab (from donkey) (NA934) and Amersham ECL Mouse IgG, HRP-linked whole secondary Ab (from sheep) (NA931) were purchased from Cytiva. RIPA Lysis and Extraction Buffer (89900) and IP-lysis buffer (87787) were purchased from ThermoScientific. Protease inhibitor cocktail (11873580001) was obtained from Roche, and phosphatase inhibitor cocktail (A32957) were obtained from ThermoScientific. 1,10-phenanthroline monohydrate (S5543), N-ethylmaleimide (S3692), and M3814 (S8586) were purchased from Selleckchem. PR-619 (4482) was purchased from Tocris. Nano-Glo® HiBiT lytic detection reagent (N3040) was purchased from Promega, and CyQUANT™ direct cell proliferation assay (<a href="https://www.ncbi.nlm.nih.gov/nuccore/C35011" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">C35011</a>) was obtained from Invitrogen. Bafilomycin A1 (54645), MG-132 (2194) and Cycloheximide (2112) were purchased from Cell Signaling Technology. TAK-243 (HY-100487) and Bortezomib (HY-10227) were obtained from MedChemExpress. Puromycin (ant-pr-1), blasticidin (ant-bl-05), and Geneticin (ant-gn-1) were purchased from InvivoGen. Pierce™ Anti-DYKDDDDK (FLAG) Magnetic Agarose beads (A36797) and 16% Formaldehyde (28906) were purchased from ThermoScientific. Protein G Magnetic Sepharose beads (28951379) were obtained from Cytiva. 8-well Ibidi dishes (80826) were purchased from Ibidi, and 4X NuPAGE™ LDS sample buffer (NP0007) was purchased from Invitrogen.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Plasmids</h3>
<p id="Par25">HA-ubiquitin cDNA in pcDNA3.1+ vector was obtained from GenScript. FLAG-MTH1-RNF152, FLAG-MTH1-LAPTM4a, and FLAG-MTH1-LAPTM5 cDNA were cloned into the pGenLenti-puro vector (GenScript). Lentivirus particles were used to transduce HEK293, HCT116, and AsPC-1 cells and generated stable cells expressing different MTH1 fusions.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Generation of endogenously tagged KRAS<sup>G12D</sup>(HiBiT-FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup>) in HCT116 cells</h3>
<p id="Par26">HCT116 cells have a heterozygous G13D mutation on the endogenous KRAS locus. To generate HCT116 cells with N-terminal HiBiT-FKBP12<sup>F36V-</sup>KRAS<sup>G12D</sup> homozygous clones, a cutting site was designed near the endogenous KRAS G12 locus (guide RNA: 5’-aaacttgtggtagttggagc-3’). The HDR template was designed to contain the BSD-P2A-HiBiT-FKBP12<sup>F36V</sup> insertion and the G12D mutation simultaneously and de novo synthesized at GenScript. To generate the homozygous clones, HCT116 were seeded overnight in 12-well plates at 6×10<sup>5</sup> cells per well overnight. The next day, cells were transfected with a mixture of 4x plasmids containing guide RNA, CRISPR enzyme, HDR template, and i53 HDR enhancer<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Six hours after transfection, cells were passaged 1:8 into 6 well plates with 2.5 mL culture medium with 1 μM M3814 as another HDR enhancer. One day later, the media was replaced with media containing 2 μg/mL puromycin (the guide RNA plasmid contains a puromycin-resistant gene for selection) and 1 μM M3814 for another 2 days. The media was then replaced with media containing 20 μg/mL blasticidin to select cells with endogenous KRAS tagging for 2 days. The cell pool was further recovered in regular medium for 2 days and were used for single cloning. For single cloning, approximately 500 cells were plated on a 10 cm dish in tissue culture media and allowed to grow for approximately one week. Single colonies were isolated and transferred to a 96-well plate using a stereomicroscope and 20 μL pipette tips. Single clones were identified with genomic PCR and subjected to Sanger sequencing to confirm homozygous HiBiT- FKBP12<sup>F36V</sup> tagging and G12D mutation at the N-terminus of endogenous KRAS.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Generation of HEK293, HCT116 (KI HiBiT-FKBP12-KRAS), and AsPC-1 cells stably expressing MTH1 fusions</h3>
<p id="Par27">To generate RNF152-expressing cells for global proteomics analysis, HEK293 cells were transduced with lentivirus particles containing FLAG-MTH1-RNF152 in the presence of polybrene (10 µg/mL). The cells were selected using 2 µg/ml puromycin for 2 weeks to generate stable cells, and MTH1-RNF152 expression was verified using immunoblotting. FLAG-MTH1-LAPTM4a and FLAG-MTH1-LAPTM5 stably expressing HEK293 cells were generated, similar to RNF152. To generate effector-expressing HCT116 (KI HiBiT-FKBP12<sup>F36V-</sup>KRAS<sup>G12D</sup>) cells, cells were separately transduced with lentivirus particles containing FLAG-MTH1-RNF152, FLAG-MTH1-LAPTM4a, and FLAG-MTH1-LAPTM5, selected using 2 µg/mL puromycin for 2 weeks and verified stable expression via immunoblotting. To generate effector-expressing AsPC-1 cells, cells were transduced with lentivirus particles containing FLAG-MTH1-RNF152 or FLAG-MTH1-LAPTM4a, selected using 1.5 µg/mL puromycin for 2 weeks and effector expression was verified via immunoblotting.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Generation of HEK293 cells stably co-expressing HiBiT-FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup> and MTH1-RNF152 fusion</h3>
<p id="Par28">To generate HiBiT-FKBP12<sup>F36V-</sup>KRAS<sup>G12D</sup> stably expressing Jump-In™GripTite™ cells, HiBiT-FKBP12<sup>F36V-</sup>KRAS<sup>G12D</sup> cDNA was cloned into pJTI™ R4 Dest CMV pA vector. This plasmid was co-transfected with pJTI™ R4 Int vector into Jump-In™GripTite™ HEK293 cells. After transfection, cells were selected using 10 µg/mL blasticidin to obtain single cell clones. HiBiT-FKBP12<sup>F36V-</sup>KRAS<sup>G12D</sup> expression was verified by immunoblotting and HiBit assay. To generate effector expressing cells, HiBiT-FKBP12<sup>F36V-</sup>KRAS<sup>G12D</sup> stable cells were transduced with lentivirus particles containing FLAG-MTH1-RNF152 in the presence of 10 µg/mL polybrene. The cells were selected using 2 µg/mL puromycin for 2 weeks to generate stable cells and MTH1-RNF152 expression was verified using immunoblotting.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Generation of HEK293 cells stably co-expressing mNeonGreen-KRAS<sup>WT</sup> and MTH1-RNF152</h3>
<p id="Par29">To generate mNeonGreen-KRAS<sup>WT</sup> stable cells, mNeonGreen-KRAS<sup>WT</sup> cDNA was cloned into pJTI™ R4 Dest CMV pA vector. This plasmid was co-transfected with pJTI™ R4 Int vector into Jump-In™GripTite™ HEK293 cells. After transfection, cells were selected for two weeks using 5 µg/mL blasticidin, and clones were obtained by single-cell sorting. mNeonGreen-KRAS<sup>WT</sup> expression was verified by immunoblotting and microcopy analysis. To generate effector-expressing cells, mNeonGreen-KRAS cells were transduced with lentivirus particles containing FLAG-MTH1-RNF152 in the presence of 10 µg/mL polybrene. The cells were selected using 2 µg/mL puromycin for 2 weeks to generate stable cells, and MTH1-RNF152 expression was verified using immunoblotting.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Cell lysis and immunoblotting</h3>
<p id="Par30">After the desired treatments, cells were washed 2X with PBS, trypsinized, and harvested as cell pellets. Cell pellets were lysed in RIPA buffer supplemented with 1X protease and 1X phosphatase inhibitor cocktail, incubated on ice for 20 min, and the cell lysate was clarified by centrifugation. Lysate was then mixed with 4X NuPAGE™ LDS Sample Buffer supplemented with 10% β-mercaptoethanol, boiled at 95 °C, and proteins were separated on 4–20% Criterion TGX precast gradient gels (Bio-Rad) and transferred to PVDF membrane. Next, the blots were incubated with blocking buffer (5% (w/v) nonfat dry milk in TBST (0.1% Tween-20, 20 mM Tris–HCl pH 7.6, 150 mM NaCl) for 1 h at room temperature, followed by overnight incubation with primary antibodies at 4 °C. After primary antibody incubation, membranes were washed 3X with TBST and incubated with HRP-conjugated secondary antibodies for 1 h at room temperature. Membrane was washed 3X with TBST, developed using ECL or Super Signal West Femto substrate (ThermoFisher), and images were acquired using Bio-Rad ChemiDoc system.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Lysosome membrane protein (LMP) turnover analysis using cycloheximide assays</h3>
<p id="Par31">HCT116 cells stably expressing MTH1-LMP were seeded at a density of 6 × 10<sup>5</sup> cells per well in a 6-well dish. After 24 h, cells were pre-treated with DMSO, 200 nM Bafilomycin A1, and 1 µM E1 inhibitor (TAK-243) for 30 min, followed by co-treatment with 100 µg/mL cycloheximide for 4 h. Cells were harvested at 0 h to include as a control. After 4 h, cells were harvested and lysed with RIPA buffer supplemented with protease inhibitor cocktail and subjected to immunoblotting with the indicated antibodies.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Quantitative proteomics analysis</h3>
<p id="Par32">HEK293 cells stably expressing MTH1-RNF152 were seeded at a density of 5 × 10<sup>5</sup> cells per well in a 6-well dish. After 24 h, cells were treated with DMSO, 500 nM LYMTAC-1, and 500 nM PROTAC (TL-12-186) for 19 h. Each treatment was performed in triplicate. Next, cells were washed 1X with PBS, trypsinized, and harvested. Cell pellet was washed 2X with PBS and subjected to global proteomics analysis.</p></section><section id="Sec19"><h3 class="pmc_sec_title">LC-MS sample preparation</h3>
<p id="Par33">Cell pellets were lysed in 100 µL of lysis buffer (2% SDS, 100 mM Tris-HCl) and sonicated using an Abcam PIXUL sonicator (50 Hz, 20 min). The protein concentration of the lysates was determined by the BCA method (Thermo Fischer Scientific 23225). For each sample, 30 µg of protein was reduced and alkylated by adding Bond-Breaker TCEP (Thermo 77720) to a final concentration of 100 mM and chloroacetamide to a final concentration of 500 mM and incubating at 56 °C for 45 min. The samples were then subjected to the SP3 protocol for clean-up. Briefly, 30 µg of protein was bound to &gt;300 µg of beads in 80% (v/v) ethanol for 15 min, and then washed three times with 80% ethanol and once with acetonitrile. The protein-bound beads were digested with 1.2 µg of Trypsin/Lys-C mix in 50 mM TEAB buffer for 18 h at 37 °C. The beads were removed, and the peptides were dried to eliminate TEAB. Before LC-MS analysis, the samples were resuspended in 5% formic acid.</p></section><section id="Sec20"><h3 class="pmc_sec_title">LC-MS analysis method</h3>
<p id="Par34">LYMTAC and PROTAC treatment experiments were performed using Thermo Scientific Vanquish Neo LC with Thermo Scientific Orbitrap Eclipse or Evosep One LC with Bruker TIMS-TOF-HT.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Vanquish neo-orbitrap eclipse LC-MS method</h3>
<p id="Par35">Liquid Chromatography (LC) Conditions: The chromatographic separation was performed using a Thermo Scientific Vanquish Neo LC system with a Bruker PepSep C18 analytical column (15 cm × 150 µm, 1.5 µm particle size). The mobile phases were Solvent A (0.1% formic acid in water) and Solvent B (80% acetonitrile and 0.1% formic acid in water). The flow rate was set to 1.2 µL/min, and the column was equilibrated with 0% Solvent B before sample injection. The gradient elution was programmed as follows: 0% Solvent B from 0 to 1 min, followed by an increase to 2% Solvent B at 1 min. Subsequently, Solvent B was linearly ramped up to 24% over the course of 84 min(1–85 min). After reaching 24%, the percentage of Solvent B was further increased to 40% over the next 14 min (85-99 min). Finally, Solvent B was rapidly increased to 95% from 99 to 100 min to wash the column and held at 95% for 4 min (100–104 min) before the flow was returned to the initial conditions. Column re-equilibration was conducted at 95% Solvent B for an additional 4 min. The total run time for the method was 104 min, including the gradient, column wash, and re-equilibration steps.</p>
<p id="Par36">Mass Spectrometry (MS) Conditions: Mass spec analysis was performed using a Thermo Scientific Orbirap Eclipse mass spectrometer with a variable window DIA method. The MS1 scans were acquired by Orbitrap at a resolution of 60,000 and a mass range set to m/z 400–1400. The RF lens was set to 60%, and the AGC (Automatic Gain Control) target was normalized to a value of 200%. The absolute AGC value was set at 1.0e6, with a custom maximum injection time of 100 ms.</p>
<p id="Par37">For the data independent MS/MS (tMS²) analysis, the MS2 scans were acquired in Orbitrap at the resolution of 15,000, the fragmentation of precursors was performed by using high-energy collisional dissociation (HCD) mode at the energy of 33%. The quadrupole was used for ion isolation, with isolation windows defined by a pre-determined table (m/z), the variable window definition is in the supplementary material (Table <a href="#MOESM1" class="usa-link">S1</a>). The mass range for the MS/MS scans was set from m/z 200 to m/* 2000. The RF lens was maintained at 50%, and the AGC target was normalized to 1000%. The absolute AGC value was set at 5.00e5.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Evosep One- TIMS-TOF-HT LC-MS method</h3>
<p id="Par38">Peptide separations were performed using the Evosep One 30SPD method (i.e., a throughput of 30 samples per day) on a Bruker PepSep C18 analytical column (15 cm × 150 µm, 1.5 µm particle size). The TIMS-TOF-HT operated in diaPASEF mode, with ion mobility separation spanning an inverse reduced mobility (1/K<sub>0</sub>) range of 0.7–1.3 V·s/cm², mass range from 100-1700 m/z. Both the ramp time and accumulation time were set to 75 ms, and 36 scan windows were acquired per cycle, yielding a total cycle time of 0.97 s.</p></section><section id="Sec23"><h3 class="pmc_sec_title">LC-MS data analysis method</h3>
<p id="Par39">Both variable width window DIA and diaPASEF data were searched against an in-house built human spectral library using DIA-NN (v1.8.2) with swiss-prot canonical human reference proteome (20221016), two pass Neural Network was used and with match between run enabled, protein quant results were analyzed through a custom R script and applied statistical modeling and hypothesis testing using the limma package.</p></section><section id="Sec24"><h3 class="pmc_sec_title">HiBiT cellular degradation assay</h3>
<p id="Par40">MTH1-fusions (MTH1-RNF152, MTH1-LAPTM4a, MTH1-LAPTM5) stably expressing HCT116 (knock-in-HiBiT- FKBP12<sup>F36V</sup>-KRAS<sup>G12D</sup> (HF-KRAS)) cells were seeded at a density of 10,000 cells per well in 96-well plates (Corning 3904). Next day, cells were treated with DMSO or increasing concentrations (2.2 nM to 1000 nM) of KRAS PROTAC, LYMTAC-3, or LYMTAC-4. For HiBiT assays with LYMTAC-2, cells were treated with DMSO or increasing concentrations (0.001 nM to 100 nM) of LYMTAC-2. After 24 h, HiBiT assay was performed using Nano Glo-HiBit lytic reagent according to the manufacturer’s protocol (N3040, Promega). The luminescence readings were recorded using EnVision plate reader under the settings Ultra-Sensitive Luminescence (US LUM) and plotted as a four-parameter non-linear regression curve fit using GraphPad Prism 8.4.3.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Mechanism of action studies using control compounds</h3>
<p id="Par41">HCT116 (HF-KRAS) cells stably expressing MTH1-fusion were seeded at a density of 5 × 10<sup>5</sup> cells per well in 6-well dishes. Next day, cells were pre-treated with 200 nM BafA1, 2 µM BTZ or 10 µM MG-132, and 1 µM TAK-243 for 30 min and co-treated with DMSO or 500 nM LYMTAC-2 for 6 h. For PTK2 and EPHA2 rescue experiments, HEK293 cells were pre-treated with 200 nM BafA1 and 10 µM MG-132 for 30 min and co-treated with DMSO or 500 nM LYMTAC-1 for 19 h. AsPC-1 cells were pre-treated with 200 nM BafA1 for 30 min, followed by 100 nM LYMTAC-4 co-treatment for 6 h. After treatment, cells were lysed in RIPA buffer and subjected to immunoblotting with the indicated antibodies.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Ligand competition experiments</h3>
<p id="Par42">Cells stably expressing MTH1-fusion were seeded at a density of 5 × 10<sup>5</sup> cells per well in 6-well dishes. HEK293 cells stably expressing MTH1-RNF152 were pre-treated with 25 µM MTH1-ligand for 30 min, followed by co-treatment with 500 nM LYMTAC-1 for 19 h. AsPC-1 cells were pretreated with 10 µM MTH1-ligand or 10 µM pan-KRAS inhibitor for 30 min, followed by 100 nM LYMTAC-4 co-treatment for 6 h. After treatment, cells were lysed and subjected to immunoblotting with the indicated antibodies.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Ternary complex formation assay</h3>
<p id="Par43">For ternary complex assays, AsPC-1 cells stably expressing FLAG-MTH1-RNF152 or FLAG-MTH1-LAPTM4a were seeded in 10-cm dishes at a density of 5 × 10<sup>6</sup> cells. Next day, cells were pre-treated with BafA1 for 30 min, followed by co-treatment with DMSO, 1 µM LYMTAC-3, or 1 µM LYMTAC-4 for 4 h. Cells were harvested and lysed in IP-lysis buffer supplemented with protease inhibitor. Next, the cell lysate was subjected to immunoprecipitation with anti-FLAG magnetic beads overnight at 4 °C. Then, beads were washed three times with wash buffer (100 mM Tris-HCl, pH 7.5, 150 mM NaCl, 0.1% Tween-20) and the samples were subjected to SDS-PAGE and Western blotting. The respective blots are probed with the indicated antibodies.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Ubiquitylation assays</h3>
<p id="Par44">To probe LYMTAC-induced KRAS ubiquitylation, HCT116 (HF-KRAS + MTH1 fusion) were seeded in a 10-cm dish at a cell density of 5 × 10<sup>6</sup> cells. Next day, 4 µg HA-ubiquitin was transfected into HCT116 (HF-KRAS + MTH1 fusion) using Lipofectamine 2000. After 24 h, cells were pre-treated with 200 nM BafA1 for 30 min followed by co-treatment with DMSO or 1 µM LYMTAC-2 for 4 h. Cells were then placed on ice, washed with ice-cold 1X PBS, and lysed in IP lysis buffer supplemented with 1X protease inhibitor cocktail, 5 mM 1,10-phenanthroline monohydrate, 10 mM N-ethylmaleimide, and 20 µM PR-619. Then, the lysate was incubated with 2 µg of anti-HiBiT antibody for 20 min at room temperature. HiBiT antibody incubated lysate was next added to 20 µL of protein G Sepharose magnetic beads and incubated overnight. After incubation, beads were washed 3X with wash buffer, and bound proteins were eluted using 2X NuPAGE™ LDS Sample Buffer supplemented with 10% β-ME. Proteins were separated in SDS-PAGE and immunoblotted with respective antibodies.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Confocal microscopy</h3>
<p id="Par45">For microscopy experiments, HEK293 mNeonGreen-KRAS<sup>WT</sup> cells stably expressing FLAG-MTH1-RNF152 were seeded at a density of 15,000 cells per well in 8-well chambered dishes (Ibidi). Next day, cells were pre-treated with 200 nM BafA1 for 30 min and co-treated with 1 µM LYMTAC-4 for 4 h. Then, cells were washed 2X with PBS and fixed with 4% paraformaldehyde in PBS for 15 min at room temperature. After fixing, cells were permeabilized with 0.2% Triton-X-100 in PBS for 10 min at room temperature. Then, cells were incubated with blocking buffer (3% BSA, 0.2% Triton-X-100 in PBS) for 1 h at room temperature, followed by overnight incubation at 4 °C with FLAG (1:300 dilution) and LAMP1 (1:50 dilution) antibodies in antibody dilution buffer (1% BSA, 0.2% Triton-X-100 in PBS). Next day, cells were washed 3X with PBS at room temperature and incubated with secondary antibodies (1:500 dilution, Alexa Fluor 568 and Alexa Fluro 647) at room temperature for 1 h. Next, cells were washed 3X with PBS and stained with Hoechst stain (3 μg/mL) in PBS for 10 min at room temperature, followed by two PBS washes. The cells were imaged using a 63x/1.30 NA glycerol immersion objective lens on an inverted Leica TCS SP8 confocal microscope and images were analyzed using Leica LAS X software.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Time lapse imaging</h3>
<p id="Par46">HEK293 stably transfected with mNeonGreen-KRAS<sup>WT</sup> and MTH1-RNF152 were seeded in a 35 mm glass-bottom dish with 2 mL Dulbecco’s Modified Eagle’s medium (DMEM; Gibco). After 24 h, cells were treated with 200 nM BafA1 and NucBlue™ Live ReadyProbes™ Reagent for 30 min at 37 °C in 5% CO<sub>2</sub> medium. Then the media was replaced with new media (2 mL) containing 1 μM LYMTAC-4 and 200 nM BafA1, and images were acquired every 30 min up to 4 h. The cells were imaged using a 63x/1.30 NA glycerol immersion objective lens on an inverted Leica TCS SP8 confocal microscope, and images were analyzed using Leica LAS X software.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Washout experiment with LYMTAC-4 in AsPC-1</h3>
<p id="Par47">AsPC-1 cells stably expressing RNF152 were seeded at a density of 4 × 10<sup>5</sup> cells per well in 6-well dishes. After 24 h, cells were treated with DMSO, 100 nM pan-KRAS inhibitor, or 100 nM LYMTAC-4 for 4 h, removed media, washed three times with PBS, and replaced with fresh media for 48 h. For LYMTAC-4 treated wells, fresh media was added in the absence or presence of 5 μM MTH1-ligand for 48 h. After 48 h, cells were lysed and subjected to SDS-PAGE and immunoblotting with respective antibodies.</p></section><section id="Sec32"><h3 class="pmc_sec_title">Cell proliferation assays</h3>
<p id="Par48">AsPC-1 cells stably expressing RNF152 were seeded at a density of 2000 cells per well in 96-well plates. Next day, cells were treated with DMSO and dose titrations (0.5 nM to 1000 nM) of pan-KRASi, KRAS PROTAC, or LYMTAC-4. After five days of treatment, cell viability was measured using CyQUANT™ direct cell proliferation assay reagents following the manufacturer’s protocol (<a href="https://www.ncbi.nlm.nih.gov/nuccore/C35011" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">C35011</a>). The fluorescence readings were recorded using EnVision plate reader (settings: FITC_bottom, Ex: FITC FP 480, Em: FITC 535) and plotted as a four-parameter non-linear regression curve fit using GraphPad Prism 8.4.3.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par49">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec34"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (96.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (680.3KB, pdf) </sup>
</div></div></section></section><section id="Sec35"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Ryan Case, Jan Andersson, Olivier Bedel, Rati Verma, Kusal Samarasinghe, Spencer Hill, and Kyle Mangano for helpful discussions. We thank Alex Golkar for supporting final compound characterization. We thank Craig Kiefer for generating graphics for the manuscript. We thank all members of the Induced Proximity Platform for their support.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>D.A.N. and P.R.P. conceived the project and designed experiments. D. A.N., W.D., J.X., S.L., R.G.G., and C.E.S. performed experiments and analyzed data. K.S.A., G.H., G.M., A.T.T., R.P.W., B.A.L., S.A., M.K.A., A.S., and R.K. designed and synthesized compounds. C.W. performed the final compound characterization. D.A.N. wrote the original draft of the manuscript, revised by F.D.S.M., S.K., and P.R.P. and was further revised with input from all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par50"><em>Nature Communications</em> thanks Markella Konstantinidou and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Source data are provided with this paper as a Source Data file. All proteomic data have been uploaded to MaSSIVE repository under the accession: <a href="https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?accession=MSV000096874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MSV000096874</a>. <a href="#Sec35" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par51">As indicated in the affiliations, some authors are employees of Amgen. The remaining authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Dhanusha A. Nalawansha, Email: dnalawan@amgen.com</p>
<p>Patrick Ryan Potts, Email: ryan.potts@amgen.com.</p></section><section id="sec36"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63128-4.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lin, C.-Y. et al. Membrane protein-regulated networks across human cancers. <em>Nat. Commun.</em><strong>10</strong>, 3131 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-10920-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6635409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31311925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20C.-Y.%20et%20al.%20Membrane%20protein-regulated%20networks%20across%20human%20cancers.%20Nat.%20Commun.10,%203131%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Jelokhani-Niaraki, M. Membrane proteins: structure, function and motion. <em>Int. J. Mol. Sci.</em><strong>24</strong>, 468 (2022).
</cite> [<a href="https://doi.org/10.3390/ijms24010468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9820270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36613912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jelokhani-Niaraki,%20M.%20Membrane%20proteins:%20structure,%20function%20and%20motion.%20Int.%20J.%20Mol.%20Sci.24,%20468%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>To’a Salazar, G., Huang, Z., Zhang, N., Zhang, X.-G. &amp; An, Z. Antibody therapies targeting complex membrane proteins. <em>Engineering</em><strong>7</strong>, 1541–1551 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?To%E2%80%99a%20Salazar,%20G.,%20Huang,%20Z.,%20Zhang,%20N.,%20Zhang,%20X.-G.%20&amp;%20An,%20Z.%20Antibody%20therapies%20targeting%20complex%20membrane%20proteins.%20Engineering7,%201541%E2%80%931551%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Hutchings, C. J. Mini-review: antibody therapeutics targeting G protein-coupled receptors and ion channels. <em>Antib. Ther.</em><strong>3</strong>, 257–264 (2020).
</cite> [<a href="https://doi.org/10.1093/abt/tbaa023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7799348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33912796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hutchings,%20C.%20J.%20Mini-review:%20antibody%20therapeutics%20targeting%20G%20protein-coupled%20receptors%20and%20ion%20channels.%20Antib.%20Ther.3,%20257%E2%80%93264%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Santos, R. et al. A comprehensive map of molecular drug targets. <em>Nat. Rev. Drug Discov.</em><strong>16</strong>, 19–34 (2017).
</cite> [<a href="https://doi.org/10.1038/nrd.2016.230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6314433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27910877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santos,%20R.%20et%20al.%20A%20comprehensive%20map%20of%20molecular%20drug%20targets.%20Nat.%20Rev.%20Drug%20Discov.16,%2019%E2%80%9334%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Rodríguez-Nava, C. et al. Mechanisms of action and limitations of monoclonal antibodies and single chain fragment variable (scFv) in the treatment of cancer. <em>Biomedicines</em><strong>11</strong>, 1610 (2023).
</cite> [<a href="https://doi.org/10.3390/biomedicines11061610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10295864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37371712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodr%C3%ADguez-Nava,%20C.%20et%20al.%20Mechanisms%20of%20action%20and%20limitations%20of%20monoclonal%20antibodies%20and%20single%20chain%20fragment%20variable%20(scFv)%20in%20the%20treatment%20of%20cancer.%20Biomedicines11,%201610%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Cruz, E. &amp; Kayser, V. Monoclonal antibody therapy of solid tumors: clinical limitations and novel strategies to enhance treatment efficacy. <em>Biologics: targets Ther.</em><strong>13</strong>, 33–51 (2019).</cite> [<a href="https://doi.org/10.2147/BTT.S166310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6503308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31118560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cruz,%20E.%20&amp;%20Kayser,%20V.%20Monoclonal%20antibody%20therapy%20of%20solid%20tumors:%20clinical%20limitations%20and%20novel%20strategies%20to%20enhance%20treatment%20efficacy.%20Biologics:%20targets%20Ther.13,%2033%E2%80%9351%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Sakamoto, K. M. et al. Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation. <em>Proc. Natl Acad. Sci. USA</em><strong>98</strong>, 8554–8559 (2001).
</cite> [<a href="https://doi.org/10.1073/pnas.141230798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC37474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11438690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sakamoto,%20K.%20M.%20et%20al.%20Protacs:%20chimeric%20molecules%20that%20target%20proteins%20to%20the%20Skp1-Cullin-F%20box%20complex%20for%20ubiquitination%20and%20degradation.%20Proc.%20Natl%20Acad.%20Sci.%20USA98,%208554%E2%80%938559%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Bondeson, D. P. &amp; Crews, C. M. Targeted protein degradation by small molecules. <em>Annu. Rev. Pharmacol. Toxicol.</em><strong>57</strong>, 107–123 (2017).
</cite> [<a href="https://doi.org/10.1146/annurev-pharmtox-010715-103507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5586045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27732798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bondeson,%20D.%20P.%20&amp;%20Crews,%20C.%20M.%20Targeted%20protein%20degradation%20by%20small%20molecules.%20Annu.%20Rev.%20Pharmacol.%20Toxicol.57,%20107%E2%80%93123%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Nalawansha, D. A. &amp; Crews, C. M. PROTACs: an emerging therapeutic modality in precision medicine. <em>Cell Chem. Biol.</em><strong>27</strong>, 998–1014 (2020).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2020.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9424844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32795419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nalawansha,%20D.%20A.%20&amp;%20Crews,%20C.%20M.%20PROTACs:%20an%20emerging%20therapeutic%20modality%20in%20precision%20medicine.%20Cell%20Chem.%20Biol.27,%20998%E2%80%931014%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Guenette, R. G., Yang, S. W., Min, J., Pei, B. &amp; Potts, P. R. Target and tissue selectivity of PROTAC degraders. <em>Chem. Soc. Rev.</em><strong>51</strong>, 5740–5756 (2022).
</cite> [<a href="https://doi.org/10.1039/d2cs00200k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35587208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guenette,%20R.%20G.,%20Yang,%20S.%20W.,%20Min,%20J.,%20Pei,%20B.%20&amp;%20Potts,%20P.%20R.%20Target%20and%20tissue%20selectivity%20of%20PROTAC%20degraders.%20Chem.%20Soc.%20Rev.51,%205740%E2%80%935756%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Samarasinghe, K. T. G. &amp; Crews, C. M. Targeted protein degradation: a promise for undruggable proteins. <em>Cell Chem. Biol.</em><strong>28</strong>, 934–951 (2021).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2021.04.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8286327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34004187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samarasinghe,%20K.%20T.%20G.%20&amp;%20Crews,%20C.%20M.%20Targeted%20protein%20degradation:%20a%20promise%20for%20undruggable%20proteins.%20Cell%20Chem.%20Biol.28,%20934%E2%80%93951%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Winter, G. E. et al. Phthalimide conjugation as a strategy for in vivo target protein degradation. <em>Science</em><strong>348</strong>, 1376–1381 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aab1433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4937790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25999370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Winter,%20G.%20E.%20et%20al.%20Phthalimide%20conjugation%20as%20a%20strategy%20for%20in%20vivo%20target%20protein%20degradation.%20Science348,%201376%E2%80%931381%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Verma, R., Mohl, D. &amp; Deshaies, R. J. Harnessing the power of proteolysis for targeted protein inactivation. <em>Mol. cell</em><strong>77</strong>, 446–460 (2020).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2020.01.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32004468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verma,%20R.,%20Mohl,%20D.%20&amp;%20Deshaies,%20R.%20J.%20Harnessing%20the%20power%20of%20proteolysis%20for%20targeted%20protein%20inactivation.%20Mol.%20cell77,%20446%E2%80%93460%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Simpson, L. M. et al. Target protein localization and its impact on PROTAC-mediated degradation. <em>Cell Chem. Biol.</em><strong>29</strong>, 1482–1504.e7 (2022).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2022.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36075213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simpson,%20L.%20M.%20et%20al.%20Target%20protein%20localization%20and%20its%20impact%20on%20PROTAC-mediated%20degradation.%20Cell%20Chem.%20Biol.29,%201482%E2%80%931504.e7%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Burslem, G. M. et al. The advantages of targeted protein degradation over inhibition: an RTK case study. <em>Cell Chem. Biol.</em><strong>25</strong>, 67–77.e3 (2018).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2017.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5831399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29129716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burslem,%20G.%20M.%20et%20al.%20The%20advantages%20of%20targeted%20protein%20degradation%20over%20inhibition:%20an%20RTK%20case%20study.%20Cell%20Chem.%20Biol.25,%2067%E2%80%9377.e3%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Sardana, R. &amp; Emr, S. D. Membrane protein quality control mechanisms in the endo-lysosome system. <em>Trends Cell Biol.</em><strong>31</strong>, 269–283 (2021).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2020.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33414051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sardana,%20R.%20&amp;%20Emr,%20S.%20D.%20Membrane%20protein%20quality%20control%20mechanisms%20in%20the%20endo-lysosome%20system.%20Trends%20Cell%20Biol.31,%20269%E2%80%93283%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Piper, R. C. &amp; Luzio, J. P. Ubiquitin-dependent sorting of integral membrane proteins for degradation in lysosomes. <em>Curr. Opin. Cell Biol.</em><strong>19</strong>, 459–465 (2007).
</cite> [<a href="https://doi.org/10.1016/j.ceb.2007.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2046217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17689064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Piper,%20R.%20C.%20&amp;%20Luzio,%20J.%20P.%20Ubiquitin-dependent%20sorting%20of%20integral%20membrane%20proteins%20for%20degradation%20in%20lysosomes.%20Curr.%20Opin.%20Cell%20Biol.19,%20459%E2%80%93465%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Nalawansha, D. A., Mangano, K., den Besten, W. &amp; Potts, P. R. TAC-tics for leveraging proximity biology in drug discovery. <em>Chembiochem</em><strong>25</strong>, e202300712 (2024).
</cite> [<a href="https://doi.org/10.1002/cbic.202300712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38015747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nalawansha,%20D.%20A.,%20Mangano,%20K.,%20den%20Besten,%20W.%20&amp;%20Potts,%20P.%20R.%20TAC-tics%20for%20leveraging%20proximity%20biology%20in%20drug%20discovery.%20Chembiochem25,%20e202300712%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Banik, S. M. et al. Lysosome-targeting chimaeras for degradation of extracellular proteins. <em>Nature</em><strong>584</strong>, 291–297 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2545-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7727926/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32728216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banik,%20S.%20M.%20et%20al.%20Lysosome-targeting%20chimaeras%20for%20degradation%20of%20extracellular%20proteins.%20Nature584,%20291%E2%80%93297%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Marei, H. et al. Antibody targeting of E3 ubiquitin ligases for receptor degradation. <em>Nature</em><strong>610</strong>, 182–189 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-05235-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9534761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36131013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marei,%20H.%20et%20al.%20Antibody%20targeting%20of%20E3%20ubiquitin%20ligases%20for%20receptor%20degradation.%20Nature610,%20182%E2%80%93189%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Cotton, A. D., Nguyen, D. P., Gramespacher, J. A., Seiple, I. B. &amp; Wells, J. A. Development of antibody-based PROTACs for the degradation of the cell-surface immune checkpoint protein PD-L1. <em>J. Am. Chem. Soc.</em><strong>143</strong>, 593–598 (2021).
</cite> [<a href="https://doi.org/10.1021/jacs.0c10008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8154509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33395526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cotton,%20A.%20D.,%20Nguyen,%20D.%20P.,%20Gramespacher,%20J.%20A.,%20Seiple,%20I.%20B.%20&amp;%20Wells,%20J.%20A.%20Development%20of%20antibody-based%20PROTACs%20for%20the%20degradation%20of%20the%20cell-surface%20immune%20checkpoint%20protein%20PD-L1.%20J.%20Am.%20Chem.%20Soc.143,%20593%E2%80%93598%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Caianiello, D. F. et al. Bifunctional small molecules that mediate the degradation of extracellular proteins. <em>Nat. Chem. Biol.</em><strong>17</strong>, 947–953 (2021).
</cite> [<a href="https://doi.org/10.1038/s41589-021-00851-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34413525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caianiello,%20D.%20F.%20et%20al.%20Bifunctional%20small%20molecules%20that%20mediate%20the%20degradation%20of%20extracellular%20proteins.%20Nat.%20Chem.%20Biol.17,%20947%E2%80%93953%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Pance, K. et al. Modular cytokine receptor-targeting chimaeras for targeted degradation of cell surface and extracellular proteins. <em>Nat. Biotechnol.</em><strong>41</strong>, 273–281 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-022-01456-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9931583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36138170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pance,%20K.%20et%20al.%20Modular%20cytokine%20receptor-targeting%20chimaeras%20for%20targeted%20degradation%20of%20cell%20surface%20and%20extracellular%20proteins.%20Nat.%20Biotechnol.41,%20273%E2%80%93281%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Schwake, M., Schröder, B. &amp; Saftig, P. Lysosomal membrane proteins and their central role in physiology. <em>Traffic</em><strong>14</strong>, 739–748 (2013).
</cite> [<a href="https://doi.org/10.1111/tra.12056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23387372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwake,%20M.,%20Schr%C3%B6der,%20B.%20&amp;%20Saftig,%20P.%20Lysosomal%20membrane%20proteins%20and%20their%20central%20role%20in%20physiology.%20Traffic14,%20739%E2%80%93748%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Xu, J. The role of lysosomal membrane proteins in autophagy and related diseases. <em>FEBS J.</em><strong>291</strong>, 3762-3785 (2024).
</cite> [<a href="https://doi.org/10.1111/febs.16820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37221945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20J.%20The%20role%20of%20lysosomal%20membrane%20proteins%20in%20autophagy%20and%20related%20diseases.%20FEBS%20J.291,%203762-3785%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Zhang, W. et al. A conserved ubiquitin- and ESCRT-dependent pathway internalizes human lysosomal membrane proteins for degradation. <em>PLoS Biol.</em><strong>19</strong>, e3001361 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3001361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8337054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34297722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.%20et%20al.%20A%20conserved%20ubiquitin-%20and%20ESCRT-dependent%20pathway%20internalizes%20human%20lysosomal%20membrane%20proteins%20for%20degradation.%20PLoS%20Biol.19,%20e3001361%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Zhu, L., Jorgensen, J. R., Li, M., Chuang, Y. S. &amp; Emr, S. D. ESCRTs function directly on the lysosome membrane to downregulate ubiquitinated lysosomal membrane proteins. <em>eLife</em><strong>6</strong>, e26403 (2017).
</cite> [<a href="https://doi.org/10.7554/eLife.26403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5507667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28661397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20L.,%20Jorgensen,%20J.%20R.,%20Li,%20M.,%20Chuang,%20Y.%20S.%20&amp;%20Emr,%20S.%20D.%20ESCRTs%20function%20directly%20on%20the%20lysosome%20membrane%20to%20downregulate%20ubiquitinated%20lysosomal%20membrane%20proteins.%20eLife6,%20e26403%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Wang, Y., et al. LAPTM5 mediates immature B cell apoptosis and B cell tolerance by regulating the WWP2-PTEN-AKT pathway. <em>Proc. Natl Acad. Sci. USA</em><strong>119</strong>, e2205629119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2205629119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9457450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36037365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.,%20et%20al.%20LAPTM5%20mediates%20immature%20B%20cell%20apoptosis%20and%20B%20cell%20tolerance%20by%20regulating%20the%20WWP2-PTEN-AKT%20pathway.%20Proc.%20Natl%20Acad.%20Sci.%20USA119,%20e2205629119%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Veits, G. K. et al. Development of an AchillesTAG degradation system and its application to control CAR-T activity. <em>Curr. Res. Chem. Biol.</em><strong>1</strong>, 100010 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Veits,%20G.%20K.%20et%20al.%20Development%20of%20an%20AchillesTAG%20degradation%20system%20and%20its%20application%20to%20control%20CAR-T%20activity.%20Curr.%20Res.%20Chem.%20Biol.1,%20100010%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Mangano, K. et al. VIPER-TACs leverage viral E3 ligases for disease-specific targeted protein degradation. <em>Cell Chem. Biol.</em><strong>32</strong>, 423–433.e9 (2025).</cite> [<a href="https://doi.org/10.1016/j.chembiol.2025.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40049166/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Huang, H. T. et al. A chemoproteomic approach to query the degradable kinome using a multi-kinase degrader. <em>Cell Chem. Biol.</em><strong>25</strong>, 88–99.e6 (2018).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2017.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6427047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29129717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20H.%20T.%20et%20al.%20A%20chemoproteomic%20approach%20to%20query%20the%20degradable%20kinome%20using%20a%20multi-kinase%20degrader.%20Cell%20Chem.%20Biol.25,%2088%E2%80%9399.e6%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Tapial Mart¡nez, P., L¢pez Navajas, P. &amp; Lietha, D. FAK structure and regulation by membrane interactions and force in focal adhesions. <em>Biomolecules</em><strong>10</strong>, 179 (2020).
</cite> [<a href="https://doi.org/10.3390/biom10020179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7072507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31991559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tapial%20Mart%C2%A1nez,%20P.,%20L%C2%A2pez%20Navajas,%20P.%20&amp;%20Lietha,%20D.%20FAK%20structure%20and%20regulation%20by%20membrane%20interactions%20and%20force%20in%20focal%20adhesions.%20Biomolecules10,%20179%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wilson, K., Shiuan, E. &amp; Brantley-Sieders, D. M. Oncogenic functions and therapeutic targeting of EphA2 in cancer. <em>Oncogene</em><strong>40</strong>, 2483–2495 (2021).
</cite> [<a href="https://doi.org/10.1038/s41388-021-01714-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8035212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33686241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilson,%20K.,%20Shiuan,%20E.%20&amp;%20Brantley-Sieders,%20D.%20M.%20Oncogenic%20functions%20and%20therapeutic%20targeting%20of%20EphA2%20in%20cancer.%20Oncogene40,%202483%E2%80%932495%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Bondeson, D. P. et al. Lessons in PROTAC design from selective degradation with a promiscuous warhead. <em>Cell Chem. Biol.</em><strong>25</strong>, 78–87.e5 (2018).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2017.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5777153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29129718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bondeson,%20D.%20P.%20et%20al.%20Lessons%20in%20PROTAC%20design%20from%20selective%20degradation%20with%20a%20promiscuous%20warhead.%20Cell%20Chem.%20Biol.25,%2078%E2%80%9387.e5%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Donovan, K. A. et al. Mapping the degradable kinome provides a resource for expedited degrader development. <em>Cell</em><strong>183</strong>, 1714–1731.e10 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.10.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10294644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33275901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Donovan,%20K.%20A.%20et%20al.%20Mapping%20the%20degradable%20kinome%20provides%20a%20resource%20for%20expedited%20degrader%20development.%20Cell183,%201714%E2%80%931731.e10%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Moore, A. R., Rosenberg, S. C., McCormick, F., Malek, S. RAS-targeted therapies. <em>Nat. Rev. Drug Discov.</em> (2021).</cite> [<a href="https://doi.org/10.1038/s41573-021-00220-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33990767/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Schwinn, M. K. et al. CRISPR-mediated tagging of endogenous proteins with a luminescent peptide. <em>ACS Chem. Biol.</em><strong>13</strong>, 467–474 (2018).
</cite> [<a href="https://doi.org/10.1021/acschembio.7b00549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28892606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwinn,%20M.%20K.%20et%20al.%20CRISPR-mediated%20tagging%20of%20endogenous%20proteins%20with%20a%20luminescent%20peptide.%20ACS%20Chem.%20Biol.13,%20467%E2%80%93474%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Clackson, T. et al. Redesigning an FKBP-ligand interface to generate chemical dimerizers with novel specificity. <em>Proc. Natl. Acad. Sci. USA</em><strong>95</strong>, 10437–10442 (1998).
</cite> [<a href="https://doi.org/10.1073/pnas.95.18.10437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC27912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9724721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clackson,%20T.%20et%20al.%20Redesigning%20an%20FKBP-ligand%20interface%20to%20generate%20chemical%20dimerizers%20with%20novel%20specificity.%20Proc.%20Natl.%20Acad.%20Sci.%20USA95,%2010437%E2%80%9310442%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Ash, L. J. et al. KRAS: biology, inhibition, and mechanisms of inhibitor resistance. <em>Curr. Oncol. (Tor., Ont.)</em><strong>31</strong>, 2024–2046 (2024).</cite> [<a href="https://doi.org/10.3390/curroncol31040150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11049385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38668053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ash,%20L.%20J.%20et%20al.%20KRAS:%20biology,%20inhibition,%20and%20mechanisms%20of%20inhibitor%20resistance.%20Curr.%20Oncol.%20(Tor.,%20Ont.)31,%202024%E2%80%932046%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Huang, L., Guo, Z., Wang, F. &amp; Fu, L. KRAS mutation: from undruggable to druggable in cancer. <em>Signal Transduct. Target. Ther.</em><strong>6</strong>, 386 (2021).
</cite> [<a href="https://doi.org/10.1038/s41392-021-00780-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8591115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34776511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20L.,%20Guo,%20Z.,%20Wang,%20F.%20&amp;%20Fu,%20L.%20KRAS%20mutation:%20from%20undruggable%20to%20druggable%20in%20cancer.%20Signal%20Transduct.%20Target.%20Ther.6,%20386%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Liu, J., Kang, R. &amp; Tang, D. The KRAS-G12C inhibitor: activity and resistance. <em>Cancer Gene Ther.</em><strong>29</strong>, 875–878 (2022).
</cite> [<a href="https://doi.org/10.1038/s41417-021-00383-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34471232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.,%20Kang,%20R.%20&amp;%20Tang,%20D.%20The%20KRAS-G12C%20inhibitor:%20activity%20and%20resistance.%20Cancer%20Gene%20Ther.29,%20875%E2%80%93878%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Ostrem, J. M. L., Peters, U. &amp; Shokat, K. M. Direct RAS inhibitors turn 10. <em>Nat. Chem. Biol.</em><strong>20</strong>, 1238–1241 (2024).
</cite> [<a href="https://doi.org/10.1038/s41589-024-01691-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39060392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ostrem,%20J.%20M.%20L.,%20Peters,%20U.%20&amp;%20Shokat,%20K.%20M.%20Direct%20RAS%20inhibitors%20turn%2010.%20Nat.%20Chem.%20Biol.20,%201238%E2%80%931241%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zografou-Barredo, N. A., Hallatt, A. J., Goujon-Ricci, J. &amp; Cano, C. A beginner’s guide to current synthetic linker strategies towards VHL-recruiting PROTACs. <em>Bioorg. Med. Chem.</em><strong>88-89</strong>, 117334 (2023).
</cite> [<a href="https://doi.org/10.1016/j.bmc.2023.117334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37224698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zografou-Barredo,%20N.%20A.,%20Hallatt,%20A.%20J.,%20Goujon-Ricci,%20J.%20&amp;%20Cano,%20C.%20A%20beginner%E2%80%99s%20guide%20to%20current%20synthetic%20linker%20strategies%20towards%20VHL-recruiting%20PROTACs.%20Bioorg.%20Med.%20Chem.88-89,%20117334%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Békés, M., Langley, D. R. &amp; Crews, C. M. PROTAC targeted protein degraders: the past is prologue. <em>Nat. Rev. Drug Discov.</em><strong>21</strong>, 181–200 (2022).
</cite> [<a href="https://doi.org/10.1038/s41573-021-00371-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8765495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35042991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B%C3%A9k%C3%A9s,%20M.,%20Langley,%20D.%20R.%20&amp;%20Crews,%20C.%20M.%20PROTAC%20targeted%20protein%20degraders:%20the%20past%20is%20prologue.%20Nat.%20Rev.%20Drug%20Discov.21,%20181%E2%80%93200%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Bond, M. J., Chu, L., Nalawansha, D. A., Li, K. &amp; Crews, C. M. Targeted degradation of oncogenic KRASG12C by VHL-recruiting PROTACs. <em>ACS Cent. Sci.</em><strong>6</strong>, 1367–1375 (2020).
</cite> [<a href="https://doi.org/10.1021/acscentsci.0c00411" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7453568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32875077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bond,%20M.%20J.,%20Chu,%20L.,%20Nalawansha,%20D.%20A.,%20Li,%20K.%20&amp;%20Crews,%20C.%20M.%20Targeted%20degradation%20of%20oncogenic%20KRASG12C%20by%20VHL-recruiting%20PROTACs.%20ACS%20Cent.%20Sci.6,%201367%E2%80%931375%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Gurreri, E. et al. KRAS-dependency in pancreatic ductal adenocarcinoma. mechanisms of escaping in resistance to KRAS inhibitors and perspectives of therapy. <em>Int. J. Mol. Sci.</em><strong>24</strong>, 9313 (2023).
</cite> [<a href="https://doi.org/10.3390/ijms24119313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10253344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37298264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gurreri,%20E.%20et%20al.%20KRAS-dependency%20in%20pancreatic%20ductal%20adenocarcinoma.%20mechanisms%20of%20escaping%20in%20resistance%20to%20KRAS%20inhibitors%20and%20perspectives%20of%20therapy.%20Int.%20J.%20Mol.%20Sci.24,%209313%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Tran, T. H. et al. KRAS interaction with RAF1 RAS-binding domain and cysteine-rich domain provides insights into RAS-mediated RAF activation. <em>Nat. Commun.</em><strong>12</strong>, 1176 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21422-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7895934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33608534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tran,%20T.%20H.%20et%20al.%20KRAS%20interaction%20with%20RAF1%20RAS-binding%20domain%20and%20cysteine-rich%20domain%20provides%20insights%20into%20RAS-mediated%20RAF%20activation.%20Nat.%20Commun.12,%201176%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Henkels, K. M., Rehl, K. M. &amp; Cho, K. -J. Blocking K-Ras interaction with the plasma membrane is a tractable therapeutic approach to inhibit oncogenic K-ras activity. <em>Front. Mol. Biosci.</em><strong>8</strong>, 673096 (2021).
</cite> [<a href="https://doi.org/10.3389/fmolb.2021.673096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8244928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34222333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Henkels,%20K.%20M.,%20Rehl,%20K.%20M.%20&amp;%20Cho,%20K.%20-J.%20Blocking%20K-Ras%20interaction%20with%20the%20plasma%20membrane%20is%20a%20tractable%20therapeutic%20approach%20to%20inhibit%20oncogenic%20K-ras%20activity.%20Front.%20Mol.%20Biosci.8,%20673096%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Haidar, M. &amp; Jacquemin, P. Past and future strategies to inhibit membrane localization of the KRAS oncogene. <em>Int. J. Mol. Sci.</em><strong>22</strong>, 13193 (2021).
</cite> [<a href="https://doi.org/10.3390/ijms222413193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8707736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34947990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haidar,%20M.%20&amp;%20Jacquemin,%20P.%20Past%20and%20future%20strategies%20to%20inhibit%20membrane%20localization%20of%20the%20KRAS%20oncogene.%20Int.%20J.%20Mol.%20Sci.22,%2013193%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Yaeger, R. et al. Molecular Characterization of acquired resistance to KRASG12C-EGFR inhibition in colorectal cancer. <em>Cancer Discov.</em><strong>13</strong>, 41–55 (2023).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-22-0405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9827113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36355783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yaeger,%20R.%20et%20al.%20Molecular%20Characterization%20of%20acquired%20resistance%20to%20KRASG12C-EGFR%20inhibition%20in%20colorectal%20cancer.%20Cancer%20Discov.13,%2041%E2%80%9355%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Holderfield, M. et al. Concurrent inhibition of oncogenic and wild-type RAS-GTP for cancer therapy. <em>Nature</em><strong>629</strong>, 919–926 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07205-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11111408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38589574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holderfield,%20M.%20et%20al.%20Concurrent%20inhibition%20of%20oncogenic%20and%20wild-type%20RAS-GTP%20for%20cancer%20therapy.%20Nature629,%20919%E2%80%93926%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Raina, K. et al. Regulated induced proximity targeting chimeras—RIPTACs—A heterobifunctional small molecule strategy for cancer selective therapies. <em>Cell Chem. Biol.</em><strong>31</strong>, 1490–1502.e42 (2024).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2024.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11371387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39116881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raina,%20K.%20et%20al.%20Regulated%20induced%20proximity%20targeting%20chimeras%E2%80%94RIPTACs%E2%80%94A%20heterobifunctional%20small%20molecule%20strategy%20for%20cancer%20selective%20therapies.%20Cell%20Chem.%20Biol.31,%201490%E2%80%931502.e42%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Ng, C. S. C., Liu, A., Cui, B. &amp; Banik, S. M. Targeted Protein Relocalization via Protein Transport Coupling. <em>Nature</em><strong>633</strong>, 941–951 (2024).</cite> [<a href="https://doi.org/10.1038/s41586-024-07950-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11761438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39294374/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Gibson, W. J. et al. Bifunctional small molecules that induce nuclear localization and targeted transcriptional regulation. <em>J. Am. Chem. Soc.</em><strong>145</strong>, 26028–26037 (2023).
</cite> [<a href="https://doi.org/10.1021/jacs.3c06179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10704550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37992275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibson,%20W.%20J.%20et%20al.%20Bifunctional%20small%20molecules%20that%20induce%20nuclear%20localization%20and%20targeted%20transcriptional%20regulation.%20J.%20Am.%20Chem.%20Soc.145,%2026028%E2%80%9326037%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Sadagopan, A. et al. p53 protein abundance is a therapeutic window across TP53 mutant cancers and is targetable with proximity inducing small molecules. <em>bioRxiv</em>, 2024.07.27.605429 (2024).</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Schulze, C. J. et al. Chemical remodeling of a cellular chaperone to target the active state of mutant KRAS. <em>Science</em><strong>381</strong>, 794–799 (2023).
</cite> [<a href="https://doi.org/10.1126/science.adg9652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10474815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37590355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schulze,%20C.%20J.%20et%20al.%20Chemical%20remodeling%20of%20a%20cellular%20chaperone%20to%20target%20the%20active%20state%20of%20mutant%20KRAS.%20Science381,%20794%E2%80%93799%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Canny, M. D. et al. Inhibition of 53BP1 favors homology-dependent DNA repair and increases CRISPR–Cas9 genome-editing efficiency. <em>Nat. Biotechnol.</em><strong>36</strong>, 95–102 (2018).
</cite> [<a href="https://doi.org/10.1038/nbt.4021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5762392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29176614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Canny,%20M.%20D.%20et%20al.%20Inhibition%20of%2053BP1%20favors%20homology-dependent%20DNA%20repair%20and%20increases%20CRISPR%E2%80%93Cas9%20genome-editing%20efficiency.%20Nat.%20Biotechnol.36,%2095%E2%80%93102%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (96.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (680.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370906/bin/41467_2025_63128_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Source data are provided with this paper as a Source Data file. All proteomic data have been uploaded to MaSSIVE repository under the accession: <a href="https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?accession=MSV000096874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MSV000096874</a>. <a href="#Sec35" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63128-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63128.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370906/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370906/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370906%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370906/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370906/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370906/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841368/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370906/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841368/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370906/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370906/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="EAa44dUQElcidNza0t2pCneNrY2FIxymgU05Rxw6sJqLpkkQDuC9AYyx0DE5If9c">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
