
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Liquid biopsy diagnostics for non-small cell lung cancer via elucidation of tRNA signatures - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532FBF8AF23153052FBF0048CF82E2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commsmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370967/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Medicine">
<meta name="citation_title" content="Liquid biopsy diagnostics for non-small cell lung cancer via elucidation of tRNA signatures">
<meta name="citation_author" content="Zhuokun Feng">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author_institution" content="Molecular Biosciences and Bioengineering Program, College of Tropical Agriculture and Human Resources, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author_institution" content="Genomics and Bioinformatics Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA">
<meta name="citation_author" content="Masaki Nasu">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author" content="Gehan Devendra">
<meta name="citation_author_institution" content="The Queen’s Medical Center, Honolulu, HI USA">
<meta name="citation_author" content="Ayman A Abdul-Ghani">
<meta name="citation_author_institution" content="Pali Momi Medical Center, Honolulu, HI USA">
<meta name="citation_author" content="Owen T M Chan">
<meta name="citation_author_institution" content="Pathology Core Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA">
<meta name="citation_author" content="Jeffrey A Borgia">
<meta name="citation_author_institution" content="RUSH University Medical Center, Chicago, IL USA">
<meta name="citation_author" content="Zitong Gao">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author" content="Hanqiu Zhang">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author" content="Yu Chen">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author" content="Ting Gong">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author" content="Gang Luo">
<meta name="citation_author_institution" content="Illinois Institute of Technology, Chicago, IL USA">
<meta name="citation_author" content="Hua Yang">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author" content="Lang Wu">
<meta name="citation_author_institution" content="Pacific Center for Genome Research, University of Hawaii Cancer Center, Honolulu, HI USA">
<meta name="citation_author" content="Yuanyuan Fu">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author_institution" content="Genomics and Bioinformatics Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA">
<meta name="citation_author_institution" content="Pacific Center for Genome Research, University of Hawaii Cancer Center, Honolulu, HI USA">
<meta name="citation_author" content="Youping Deng">
<meta name="citation_author_institution" content="Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA">
<meta name="citation_author_institution" content="Genomics and Bioinformatics Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA">
<meta name="citation_author_institution" content="Pacific Center for Genome Research, University of Hawaii Cancer Center, Honolulu, HI USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="5">
<meta name="citation_firstpage" content="364">
<meta name="citation_doi" content="10.1038/s43856-025-01068-2">
<meta name="citation_pmid" content="40841456">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370967/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370967/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370967/pdf/43856_2025_Article_1068.pdf">
<meta name="description" content="Lung cancer, particularly non-small cell lung cancer (NSCLC), accounts for about 85% of all lung cancer cases and remains a major global health challenge. Traditional diagnostic methods, such as chest X-rays and low-dose CT scans, have limitations, ...">
<meta name="og:title" content="Liquid biopsy diagnostics for non-small cell lung cancer via elucidation of tRNA signatures">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Lung cancer, particularly non-small cell lung cancer (NSCLC), accounts for about 85% of all lung cancer cases and remains a major global health challenge. Traditional diagnostic methods, such as chest X-rays and low-dose CT scans, have limitations, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370967/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370967">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s43856-025-01068-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/43856_2025_Article_1068.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370967%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370967/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370967/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370967/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commsmed.jpg" alt="Communications Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Medicine" title="Link to Communications Medicine" shape="default" href="https://www.nature.com/commsmed/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Med (Lond)</button></div>. 2025 Aug 21;5:364. doi: <a href="https://doi.org/10.1038/s43856-025-01068-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s43856-025-01068-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Med%20(Lond)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Med%20(Lond)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Med%20(Lond)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Med%20(Lond)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Liquid biopsy diagnostics for non-small cell lung cancer via elucidation of tRNA signatures</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhuokun Feng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhuokun Feng</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">
<sup>2</sup>Molecular Biosciences and Bioengineering Program, College of Tropical Agriculture and Human Resources, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">
<sup>3</sup>Genomics and Bioinformatics Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhuokun Feng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nasu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Masaki Nasu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Masaki Nasu</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nasu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Masaki Nasu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Devendra%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Gehan Devendra</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Gehan Devendra</span></h3>
<div class="p">
<sup>4</sup>The Queen’s Medical Center, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Devendra%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gehan Devendra</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdul-Ghani%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ayman A Abdul-Ghani</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ayman A Abdul-Ghani</span></h3>
<div class="p">
<sup>5</sup>Pali Momi Medical Center, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdul-Ghani%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Ayman A Abdul-Ghani</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20OTM%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Owen T M Chan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Owen T M Chan</span></h3>
<div class="p">
<sup>6</sup>Pathology Core Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20OTM%22%5BAuthor%5D" class="usa-link"><span class="name western">Owen T M Chan</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Borgia%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jeffrey A Borgia</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jeffrey A Borgia</span></h3>
<div class="p">
<sup>7</sup>RUSH University Medical Center, Chicago, IL USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Borgia%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeffrey A Borgia</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zitong Gao</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zitong Gao</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zitong Gao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Hanqiu Zhang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Hanqiu Zhang</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hanqiu Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Yu Chen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Yu Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Ting Gong</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Ting Gong</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Ting Gong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Gang Luo</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Gang Luo</span></h3>
<div class="p">
<sup>8</sup>Illinois Institute of Technology, Chicago, IL USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gang Luo</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Hua Yang</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Hua Yang</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hua Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Lang Wu</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Lang Wu</span></h3>
<div class="p">
<sup>9</sup>Pacific Center for Genome Research, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lang Wu</span></a>
</div>
</div>
<sup>9,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Yuanyuan Fu</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Yuanyuan Fu</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">
<sup>3</sup>Genomics and Bioinformatics Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="p">
<sup>9</sup>Pacific Center for Genome Research, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuanyuan Fu</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>9,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Youping Deng</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Youping Deng</span></h3>
<div class="p">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div class="p">
<sup>3</sup>Genomics and Bioinformatics Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="p">
<sup>9</sup>Pacific Center for Genome Research, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Youping Deng</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>9,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Quantitative Health Sciences, John A. Burns School of Medicine, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div id="Aff2">
<sup>2</sup>Molecular Biosciences and Bioengineering Program, College of Tropical Agriculture and Human Resources, University of Hawaii at Manoa, Honolulu, HI USA </div>
<div id="Aff3">
<sup>3</sup>Genomics and Bioinformatics Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div id="Aff4">
<sup>4</sup>The Queen’s Medical Center, Honolulu, HI USA </div>
<div id="Aff5">
<sup>5</sup>Pali Momi Medical Center, Honolulu, HI USA </div>
<div id="Aff6">
<sup>6</sup>Pathology Core Shared Resource, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div id="Aff7">
<sup>7</sup>RUSH University Medical Center, Chicago, IL USA </div>
<div id="Aff8">
<sup>8</sup>Illinois Institute of Technology, Chicago, IL USA </div>
<div id="Aff9">
<sup>9</sup>Pacific Center for Genome Research, University of Hawaii Cancer Center, Honolulu, HI USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jul 2; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370967  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841456/" class="usa-link">40841456</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Lung cancer, particularly non-small cell lung cancer (NSCLC), accounts for about 85% of all lung cancer cases and remains a major global health challenge. Traditional diagnostic methods, such as chest X-rays and low-dose CT scans, have limitations, including high false-positive rates, radiation risks, and the invasiveness of tissue biopsies. This study aims to develop a non-invasive liquid biopsy approach for early NSCLC diagnosis.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">We developed a machine-learning model to analyze small RNA sequencing data from 1446 tissue samples to identify a diagnostic tRNA signature. This signature was independently validated using the in-house data of 233 plasma exosome samples. Diagnostic performance was assessed using Area Under the Curve (AUC) metrics. Signature tRNAs were then evaluated across various clinical and demographic variables, with further survival analysis and functional studies to explore the molecular role of the signature tRNAs.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">We identify a robust six-tRNA signature with strong diagnostic performance, achieving AUC values of 0.97 in discovery, 0.96 in hold-out validation, and 0.84 in independent validation. The signature effectively distinguishes cancerous from benign samples (AUC = 0.85) and consistently performs across clinical and demographic variables, with AUC values exceeding 0.80, particularly for early-stage lung cancer diagnosis. Additionally, three signature tRNAs demonstrate prognostic value for independent survival prediction. Functional studies suggest potential regulatory roles of specific tRNAs and their associated fragments in tumor metabolism pathways.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">This research underscores the diagnostic power of tRNA signature for NSCLC liquid biopsy and provides epigenetic insights that enhance our understanding of oncogenic molecular pathophysiology.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Non-small-cell lung cancer, Diagnostic markers, Cancer genetics</p></section></section><section class="abstract" id="Abs2"><h2>Plain Language Summary</h2>
<p id="Par5">Lung cancer is one of the leading causes of cancer-related deaths worldwide, and early detection is crucial for improving patient outcomes. This study investigates whether transfer RNAs (tRNAs), key regulators of protein synthesis, can be used as a non-invasive way to detect non-small cell lung cancer (NSCLC). We analyzed large-scale small RNA sequencing datasets derived from diverse NSCLC cohorts and identified six tRNAs that could be combined to accurately distinguish people with lung cancer from healthy individuals, particularly people with early-stage lung cancer. Further analysis suggests that some of these tRNAs may also play a role in tumor growth. These findings represent a promising step toward developing an easier method to screen for lung cancer, potentially improving early diagnosis and patient survival.</p></section><section class="abstract" id="Abs3"><hr class="headless">
<p id="Par6">Feng et al. develop a machine-learning model to identify a tRNA signature to enable the non-invasive early diagnosis of non-small cell lung cancer (NSCLC). A six-tRNA signature is identified and validated in plasma exosome samples, with survival analysis revealing prognostic value for three tRNAs.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par7">Lung cancer remains a principal public health challenge worldwide, marked by its substantial prevalence and mortality rates. As reported in the 2024 Cancer Statistics, there are 234,580 new instances of lung cancer and 125,070 resultant fatalities in the United States alone, establishing lung cancer as the leading cause of cancer-related mortality<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. This malignancy is histologically divided into small cell lung cancer (SCLC) and NSCLC, with the latter accounting for approximately 85% of incidences. NSCLC exhibits notable heterogeneity, subdivided into lung squamous cell carcinoma (LUSC) and lung adenocarcinoma (LUAD). NSCLC presents a high risk for early metastasis, contributing to a grim five-year survival rate of merely 28% within the U.S. population<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. These statistics underscore the importance of early diagnosis in improving patient survival and treatment outcomes.</p>
<p id="Par8">Traditional diagnostic avenues, such as chest radiography and low-dose computed tomography (LDCT), are hindered by considerable drawbacks, including elevated false-positive rates and potential radiation risks<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Moreover, the invasive nature and poor compliance associated with multiple tissue biopsies exacerbate the challenges in early cancer detection<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Thus, there is an urgent necessity for the innovation of non-invasive and accurate diagnostic methodologies for early lung cancer detection.</p>
<p id="Par9">In this context, liquid biopsy emerges as a pioneering approach in cancer screening<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. This methodology facilitates a non-invasive, comprehensive evaluation of an individual’s overall health in real time by analyzing tumor biomarkers present in body fluids such as blood. It holds potential for diagnosis, prognosis, and monitoring of treatment responses in cancer patients. Predominantly, circulating tumor cells, DNA, and exosomes have been identified as key biomarkers in liquid biopsies<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par10">Exosomes, a subset of extracellular vesicles (EVs) with diameters ranging from 30 to 200 nm, serve as crucial mediators in intercellular communication, transporting DNA, mRNA, small non-coding RNA (sncRNA), long non-coding RNA (lncRNA), and proteins. Their important function in many aspects of carcinogenesis and cancer development processes, including angiogenesis, metastasis, epithelial-mesenchymal transition (EMT), and inhibiting antitumor immunity, makes them a focus of liquid biopsy research for lung cancer<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. With advances in next-generation sequencing (NGS) technologies, such as small-RNA sequencing, researchers have identified exosomal sncRNAs as vital mediators of information exchange between cancerous and normal cells in lung cancer<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. For example, <em>miR-30e-3p</em>, <em>miR-30a-3p</em>, <em>miR-181-5p</em>, and <em>miR-361-5p</em> are identified to be the diagnostic biomarkers of LUAD<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>; <em>miR-15b-5p</em>, <em>miR-10b-5p</em>, <em>miR-320b</em> are validated to test LUSC<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>; <em>piR-hsa-26925</em> and <em>piR-hsa-5444</em> are discovered to be the potential biomarkers of LUAD<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>; exosomal tRNA-derived fragments (tRFs) like <em>tRF-Leu-TAA-005</em>, <em>tRF-Asn-GTT-010</em>, <em>tRF-Ala-AGC-036</em>, <em>tRF-Lys-CTT-049</em>, and <em>tRF-Trp-CCA-057</em> are significantly downregulated in NSCLC patients that can serve as promising diagnostic biomarkers for NSCLC<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. As a result, blood exosomal sncRNA has great potential to be developed as the diagnostic signature in detecting NSCLC.</p>
<p id="Par11">Transfer RNA (tRNA), a type of sncRNA, plays an essential role in protein synthesis by transforming amino acids in the mRNA translation process. While it has been traditionally viewed as the housekeeping gene due to its stable expression, recent studies have illuminated the variability in tRNA expression across different organs and between healthy individuals and cancer patients<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Moreover, tRNA and its associated metabolism have been discovered to participate in tumorigenesis and cancer progression, including the processes of tRNA transcription and mutations, tRNAs or its derivatives, tRNA modification, and tRNA aminoacylation<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The transcription of tRNA mediated by RNA polymerase III (Pol III), which is initially inhibited by tumor-suppressing genes, is aggravated by the products of oncogenes<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Telomerase reverse transcriptase (TERT) can exacerbate tRNA transcription by facilitating the binding of Pol III to tRNA genes, thereby enhancing Pol III’s occupancy rate. This process is significantly correlated with the upregulation of several tRNA expressions, such as <em>tRNA</em><sup><em>Lys</em></sup>, <em>tRNA</em><sup><em>Met</em></sup>, and <em>tRNA</em><sup><em>Arg</em></sup>, playing a notable role in cancer progression<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Kuang et al. discover <em>tRNA</em><sup><em>Ile</em></sup>, <em>tRNA</em><sup><em>Pro</em></sup>, and <em>tRNA</em><sup><em>Lys</em></sup> are related to tumor differentiation, and patients with up-regulated expression <em>of mt-tRNA</em><sup><em>Glu</em></sup> and <em>tRNA</em><sup><em>Tyr</em></sup>, or down-regulated expression of <em>tRNA</em><sup><em>Thr</em></sup> and <em>tRNA</em><sup><em>Asn</em></sup> are more likely to relapse<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Furthermore, they also find that <em>tRNA</em><sup><em>Lys</em></sup>, <em>mt-tRNA</em><sup><em>Ser</em></sup>, and <em>tRNA</em><sup><em>Tyr</em></sup> are accurate prognostic biomarkers to predict the survival span of LUAD patients<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par12">Despite the growing evidence in tRNA expression between NSCLC patients and healthy individuals, the development of tRNAs as diagnostic signatures and the exploration of their diagnostic efficacy remain limited. Additionally, the relationship between tRNA biomarkers and blood exosomes in lung cancer has yet to be elucidated, posing a barrier to the clinical application of tRNA signatures in liquid biopsies for early cancer screening.</p>
<p id="Par13">In this study, we address these gaps by employing a machine learning-based approach to identify a six-tRNA diagnostic signature using small RNA sequencing data from multiple global NSCLC cohorts. We then validate the diagnostic performance of this tRNA signature in independent blood exosome datasets, demonstrating its potential utility in non-invasive NSCLC detection. Robustness assessments across various clinical and demographic subgroups, such as age, sex, smoking history, histological subtype, and cancer stage, further confirm the generalizability of the signature. In addition, survival analyses and functional enrichment investigations reveal the prognostic significance of the identified tRNAs and provide insights into their potential regulatory roles in NSCLC pathogenesis.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Data mining from public resources</h3>
<p id="Par14">In this study, we conducted a comprehensive data mining process across the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA) databases to identify small RNA sequencing datasets relevant to NSCLC research. Raw sequencing files in FASTQ format, derived from lung tissue samples, were systematically retrieved from six publicly available study cohorts across both databases. These included four GEO datasets—<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE110907</a> (Korea, URL: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907</a>)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE62182</a> (Canada, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182</a>)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE83527</a> (Canada, URL: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527</a>)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175462</a> (Canada, URL: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462</a>)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>—and two TCGA datasets: TCGA-LUAD and TCGA-LUSC (USA, <a href="https://portal.gdc.cancer.gov/analysis_page?app" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/analysis_page?app</a> = ).</p>
<p id="Par15">In addition to genomic data, corresponding clinical and demographic information was collected, including age, sex, ethnicity, diagnosis, histological subtype, American Joint Committee on Cancer (AJCC) pathological stage, and smoking history. Following data mining, the filtering step was applied to eliminate duplicate and low-quality samples. This process yielded a refined dataset of 1446 samples, including 1173 tumor and 273 normal samples, as detailed in Data <a href="#MOESM4" class="usa-link">S1</a>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Plasma sample collection</h3>
<p id="Par16">Plasma specimens and associated patient information were obtained from RUSH University Medical Center, comprising a cohort of 117 individuals diagnosed with NSCLC, 54 subjects with benign lung conditions, and 62 healthy controls, totaling 233 samples (Table <a href="#Tab1" class="usa-link">1</a>). The samples were collected between February 2010 and June 2019, and all blood samples were collected at the time of initial diagnosis, forming a retrospective convenience series. While some patients underwent surgical biopsy during the diagnostic phase, any additional treatments—including surgery, radiation, or chemotherapy—were conducted only after blood sampling. Although data regarding immunotherapy treatments were unavailable, the timing of blood collection at diagnosis suggests minimal impact from anti-tumor treatments on the plasma samples. The clinicians responsible for the clinical diagnosis were blinded to the tRNA signature analysis results during the diagnostic evaluation. Histopathological and clinical diagnoses were independently determined based on established clinical and pathological criteria, ensuring an unbiased reference standard assessment.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Demographic and clinical characteristics of individuals included in the study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th rowspan="2" colspan="2"></th>
<th colspan="5" rowspan="1">Tissue</th>
<th colspan="1" rowspan="1">Exosome</th>
<th rowspan="2" colspan="1"><em>Total</em></th>
</tr>
<tr>
<th colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE110907</a></th>
<th colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE62182</a></th>
<th colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE83527</a></th>
<th colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175462</a></th>
<th colspan="1" rowspan="1">TCGA</th>
<th colspan="1" rowspan="1">RUSH</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="4" colspan="1"><strong>Diagnosis</strong></td>
<td align="left" colspan="1" rowspan="1">Lung cancer</td>
<td align="left" colspan="1" rowspan="1">48</td>
<td align="left" colspan="1" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">36</td>
<td align="left" colspan="1" rowspan="1">63</td>
<td align="left" colspan="1" rowspan="1">991</td>
<td align="left" colspan="1" rowspan="1">117</td>
<td align="left" colspan="1" rowspan="1">1290(76.83%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Normal</td>
<td colspan="1" rowspan="1">48</td>
<td colspan="1" rowspan="1">21</td>
<td colspan="1" rowspan="1">36</td>
<td colspan="1" rowspan="1">77</td>
<td colspan="1" rowspan="1">91</td>
<td colspan="1" rowspan="1">62</td>
<td colspan="1" rowspan="1">335(19.95%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Benign</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">54</td>
<td colspan="1" rowspan="1">54(3.22%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Total</em></td>
<td colspan="1" rowspan="1">96</td>
<td colspan="1" rowspan="1">56</td>
<td colspan="1" rowspan="1">72</td>
<td colspan="1" rowspan="1">140</td>
<td colspan="1" rowspan="1">1082</td>
<td colspan="1" rowspan="1">233</td>
<td colspan="1" rowspan="1">1679</td>
</tr>
<tr>
<td align="left" rowspan="6" colspan="1"><strong>Age</strong></td>
<td align="left" colspan="1" rowspan="1">&lt;50</td>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">72(5.21%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">50–59</td>
<td colspan="1" rowspan="1">34</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">189</td>
<td colspan="1" rowspan="1">36</td>
<td colspan="1" rowspan="1">259(18.74%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">60–69</td>
<td colspan="1" rowspan="1">28</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">364</td>
<td colspan="1" rowspan="1">97</td>
<td colspan="1" rowspan="1">489(35.38%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">70–79</td>
<td colspan="1" rowspan="1">20</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">379</td>
<td colspan="1" rowspan="1">81</td>
<td colspan="1" rowspan="1">480(34.73%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">≥80</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">70</td>
<td colspan="1" rowspan="1">12</td>
<td colspan="1" rowspan="1">82(5.93%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Total</em></td>
<td colspan="1" rowspan="1">96</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">1055*</td>
<td colspan="1" rowspan="1">231*</td>
<td colspan="1" rowspan="1">1382</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1"><strong>Sex</strong></td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">96</td>
<td align="left" colspan="1" rowspan="1">34</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">437</td>
<td align="left" colspan="1" rowspan="1">124</td>
<td align="left" colspan="1" rowspan="1">821(48.90%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Male</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">22</td>
<td colspan="1" rowspan="1">42</td>
<td colspan="1" rowspan="1">40</td>
<td colspan="1" rowspan="1">645</td>
<td colspan="1" rowspan="1">109</td>
<td colspan="1" rowspan="1">858(51.10%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Total</em></td>
<td colspan="1" rowspan="1">96</td>
<td colspan="1" rowspan="1">56</td>
<td colspan="1" rowspan="1">72</td>
<td colspan="1" rowspan="1">140</td>
<td colspan="1" rowspan="1">1082</td>
<td colspan="1" rowspan="1">233</td>
<td colspan="1" rowspan="1">1679</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><strong>Race</strong></td>
<td align="left" colspan="1" rowspan="1">White</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">799</td>
<td align="left" colspan="1" rowspan="1">163</td>
<td align="left" colspan="1" rowspan="1">962(78.02%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">African American</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">88</td>
<td colspan="1" rowspan="1">70</td>
<td colspan="1" rowspan="1">158(12.81%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Asian</td>
<td colspan="1" rowspan="1">96</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">17</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">113(9.16%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Total</em></td>
<td colspan="1" rowspan="1">96</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">NA</td>
<td colspan="1" rowspan="1">904*</td>
<td colspan="1" rowspan="1">233</td>
<td colspan="1" rowspan="1">1233</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><strong>Histological subtype</strong></td>
<td align="left" colspan="1" rowspan="1">LUAD</td>
<td align="left" colspan="1" rowspan="1">48</td>
<td align="left" colspan="1" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">36</td>
<td align="left" colspan="1" rowspan="1">63</td>
<td align="left" colspan="1" rowspan="1">513</td>
<td align="left" colspan="1" rowspan="1">50</td>
<td align="left" colspan="1" rowspan="1">745(57.84%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">LUSC</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">478</td>
<td colspan="1" rowspan="1">46</td>
<td colspan="1" rowspan="1">524(40.68%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Other types of NSCLC</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">19</td>
<td colspan="1" rowspan="1">19(1.48%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Total</em></td>
<td colspan="1" rowspan="1">48</td>
<td colspan="1" rowspan="1">35</td>
<td colspan="1" rowspan="1">36</td>
<td colspan="1" rowspan="1">63</td>
<td colspan="1" rowspan="1">991</td>
<td colspan="1" rowspan="1">115*</td>
<td colspan="1" rowspan="1">1288</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1"><strong>AJCC pathologic stage</strong></td>
<td align="left" colspan="1" rowspan="1">Stage I</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">41</td>
<td align="left" colspan="1" rowspan="1">507</td>
<td align="left" colspan="1" rowspan="1">82</td>
<td align="left" colspan="1" rowspan="1">688(55.48%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Stage II</td>
<td colspan="1" rowspan="1">6</td>
<td colspan="1" rowspan="1">10</td>
<td colspan="1" rowspan="1">10</td>
<td colspan="1" rowspan="1">13</td>
<td colspan="1" rowspan="1">279</td>
<td colspan="1" rowspan="1">6</td>
<td colspan="1" rowspan="1">324(26.13%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Stage III</td>
<td colspan="1" rowspan="1">12</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">5</td>
<td colspan="1" rowspan="1">164</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">193(15.56%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Stage IV</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">30</td>
<td colspan="1" rowspan="1">1</td>
<td colspan="1" rowspan="1">35(2.82%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Total</em></td>
<td colspan="1" rowspan="1">48</td>
<td colspan="1" rowspan="1">35</td>
<td colspan="1" rowspan="1">26*</td>
<td colspan="1" rowspan="1">60*</td>
<td colspan="1" rowspan="1">980*</td>
<td colspan="1" rowspan="1">91*</td>
<td colspan="1" rowspan="1">1240</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1"><strong>Smoking history</strong></td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">48</td>
<td align="left" colspan="1" rowspan="1">66</td>
<td align="left" colspan="1" rowspan="1">94</td>
<td align="left" colspan="1" rowspan="1">827</td>
<td align="left" colspan="1" rowspan="1">117</td>
<td align="left" colspan="1" rowspan="1">1166(73.61%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">No</td>
<td colspan="1" rowspan="1">82</td>
<td colspan="1" rowspan="1">8</td>
<td colspan="1" rowspan="1">6</td>
<td colspan="1" rowspan="1">46</td>
<td colspan="1" rowspan="1">255</td>
<td colspan="1" rowspan="1">21</td>
<td colspan="1" rowspan="1">418(26.39%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>Total</em></td>
<td colspan="1" rowspan="1">96</td>
<td colspan="1" rowspan="1">56</td>
<td colspan="1" rowspan="1">72</td>
<td colspan="1" rowspan="1">140</td>
<td colspan="1" rowspan="1">1082</td>
<td colspan="1" rowspan="1">138*</td>
<td colspan="1" rowspan="1">1584</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p18"><p>Bold font highlights column and row titles. The total number of cases in specific categories is presented in <em>italic formatting</em>. “*” indicates missing data in certain datasets, with only available information summarized. <em>LUAD</em> lung adenocarcinoma, <em>LUSC</em> lung squamous cell carcinoma, <em>NSCLC</em> non-small cell lung cancer, <em>AJCC</em> American Joint Committee on Cancer.</p></div></div></section><p id="Par17">The ethical acquisition and utilization of human blood samples, along with the clinical data, received approval from the University of Hawaii Human Studies Program, and informed consent was obtained from all patients, under the protocol number 2018-00636.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Exosome isolation and RNA extraction</h3>
<p id="Par18">Exosomes were meticulously isolated from each plasma specimen utilizing the Capturem Extracellular Vesicle Isolation Kit (Mini, Catalog No. 635741, Takara Bio USA, Inc.). From an initial volume of 500 μL of plasma, the exosomes were subsequently eluted in 200 μL of elution buffer and visualized under an electron microscope (Fig. <a href="#MOESM2" class="usa-link">S1</a>). The miRNeasy Serum/Plasma Kit (Catalog No. 217184, Qiagen, Germany) was used to extract total RNA from these exosome preparations. The resultant RNA was eluted in 16 μL of RNase-free water and preserved at a temperature of −50 °C for subsequent analytical and application purposes.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Library preparation and small RNA sequencing</h3>
<p id="Par19">Small RNA sequencing of all 233 extracted RNA samples was conducted by the Genomics and Bioinformatics Shared Resources (GBSR) at the University of Hawaii Cancer Center, employing the QIAseq miRNA Library Kit (Catalog No. 331502, Qiagen, Germany) for library preparation. The sequencing process utilized the NextSeq 500 System (Illumina, USA) to achieve a target of 10 million reads per sample. This strategic approach ensured a comprehensive analysis of the small RNA profiles within each specimen.</p>
<p id="Par20">Data retrieved from public databases, including GEO datasets of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE110907</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE62182</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE83527</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175462</a> and TCGA dataset, were all generated using miRNA-seq based on whole transcriptome sequencing on the Illumina HiSeq 2000 platform (Homo sapiens).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Small RNA sequencing data processing</h3>
<p id="Par21">With reference to the previous pipeline developed by our lab for small RNA seq upstream analysis<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, we developed the pipeline to quantify tRNA for this study. Quality control measures were implemented on all small RNA sequencing raw FASTQ files utilizing FastQC (version 0.11.9) for initial assessment and fastp (version 0.20.0) for adaptor trimming. This procedure was applied uniformly to datasets obtained from public repositories as well as to the independent cohort collected from RUSH University Medical Center. Subsequent alignment of the processed reads was conducted against the Genome Reference Consortium Human Build 38 (GRCh38/hg38) using the Spliced Transcripts Alignment to a Reference (STAR) software (version 2.7.10). To perform gene feature annotation and the quantification of mapped reads specific to tRNA and tRF, FeatureCounts (version 2.0.6) was employed in conjunction with customized Gene Transfer Format (GTF) files for each RNA gene. Specifically, the tRNA GTF file was derived and compiled from hg38 tRNA fasta files available at GtRNAdb (<a href="http://gtrnadb.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gtrnadb.ucsc.edu/</a>). In parallel, tRF annotations were transformed from the hg19 tRF metadata provided by MINTbase (<a href="https://cm.jefferson.edu/MINTbase/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cm.jefferson.edu/MINTbase/</a>) to the hg38 tRF reference using hgLiftOver (<a href="https://genome.ucsc.edu/cgi-bin/hgLiftOver" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genome.ucsc.edu/cgi-bin/hgLiftOver</a>), then summarized into a customized hg38 tRF GTF file. Following annotation, the raw count data for tRNAs were converted into Transcripts Per Million (TPM) format, facilitating downstream analytical processes.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Principal component analysis (PCA)</h3>
<p id="Par22">Gene counts exhibiting low expression, defined as being absent in over 50% of samples within a specific study cohort, were systematically excluded. This threshold sets a balance between retaining relevant features and eliminating genes with low expression levels that might not have substantial discriminatory value, especially in clinical screening applications. The limma package (version 3.56.2) was employed for data normalization to address batch effects across different studies. A preliminary analysis of tRNA expression across all samples was conducted using Principal Component Analysis (PCA), with the resulting expression patterns visualized in a two-dimensional space defined by the first two principal components via ggplot2 package (version 3.4.4) in R. Given the non-normal distribution of PCA results, a rank transformation was applied prior to performing a Multivariate Analysis of Variance (MANOVA). This analysis aimed to identify significant differences in tRNA expression patterns between tumor samples and control groups, which included both normal and benign specimens. Additionally, comparisons of tRNA expression profiles between different NSCLC subtypes were conducted to explore potential subtype-specific variations.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Model development to identify and validate the tRNA diagnostic signature</h3>
<p id="Par23">The methodological framework of this study is structured into three sequential phases: discovery, hold-out validation, and independent validation. Initially, samples from the public database were randomly allocated, with 70% designated for the discovery phase to facilitate model training and the remaining 30% reserved for the hold-out validation phase to assess model performance. Stratified splitting was employed to maintain an equivalent ratio of tumor to control samples as in the original dataset, thereby mitigating bias arising from sample imbalance. Plasma exosome samples procured from RUSH University Medical Center were exclusively applied to the independent validation phase to substantiate the liquid biopsy’s diagnostic potential.</p>
<p id="Par24">In order to guarantee statistical robustness, the Hanley &amp; McNeil formula for AUC-based analysis was used to determine the required sample size for evaluating the tRNA-based biomarker, taking performance validation and feature selection stability into consideration. Assuming an AUC of 0.75 in the discovery phase and 0.80 in the validation phase, with 95% confidence (α = 0.05), 80% power (β = 0.20), and a null hypothesis AUC of 0.5, the minimum required sample sizes were ~90 patients (72 cases, 18 controls, r = 4.3) for discovery, ~44 patients (36 cases, 8 controls, r = 4.3) for hold-out validation, and ~72 patients (36 cases, 36 controls, r = 1.0) for independent validation. Our study exceeded these estimates, including 1012 patients in discovery (821 cases, 191 controls, r = 4.3), 434 in hold-out validation (352 cases, 82 controls, r = 4.3), and 202 in independent validation (117 cases, 116 controls, r = 1.0), ensuring high statistical power.</p>
<p id="Par25">Before conducting our analysis, we performed batch effect correction on the stratified data from different phases using the limma package (version 3.56.2) in R, preparing it for meta-analytical processing. To identify the tRNA signature with the highest diagnostic value, we utilized the MetaIntegrator package (version 2.1.3, <a href="https://github.com/cran/MetaIntegrator" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/cran/MetaIntegrator</a>) in R, which consolidates effect sizes and statistical significance of gene expression across multiple studies<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. This approach facilitates the detection of significant tRNAs demonstrating meaningful effect sizes. Our study adopted stringent filtering criteria: (1) a minimum meta-effect size threshold of 0.2 with a False Discovery Rate (FDR) not exceeding 30% for the combined effect sizes; (2) integration of gene significances through Fisher’s test, requiring an FDR below 0.001; (3) analysis across all six datasets included in the study.</p>
<p id="Par26">Additionally, our methodology incorporates a leave-one-out validation algorithm from the MetaIntegrator package (version 2.1.3), which systematically excludes one dataset at a time to conduct meta-analysis on the remainder. This iterative approach ensures that each dataset is removed once, enabling the identification of a robust tRNA signature resilient to potential biases associated with any single dataset.</p>
<p id="Par27">The final optimization of the gene set was achieved using the forwardSearch and backwardSearch algorithms, which are iterative, greedy-based approaches designed to maximize the predictive performance of the tRNA signature. The forwardSearch algorithm begins with a single tRNA and incrementally adds additional tRNAs from the candidate pool, each selected for its ability to enhance the weighted AUC. Weighted AUC is calculated as the sum of the AUCs for individual datasets, each adjusted by sample size, to prioritize performance across the largest sample coverage. This addition process continues until further tRNA inclusion no longer improves the weighted AUC. In contrast, the backwardSearch algorithm begins with the entire collection of potential tRNAs and systematically eliminates specific tRNAs whose removal results in the greatest rise in weighted AUC. This elimination process continues until no further eliminations improve the weighted AUC.</p>
<p id="Par28">Using the intersection of results from both algorithms produced a robust six-tRNA signature, ensuring a highly optimized selection with minimized redundancy and maximized discriminatory power across datasets. This six-tRNA signature was then validated through ROC and PRC analysis, combining the T-scores of the selected tRNAs to confirm its diagnostic accuracy across multiple phases and datasets.</p>
<p id="Par29">Clinical data were accessible during data preprocessing and statistical analyses, including patient demographics and clinical status. Therefore, rather than directly selecting features based on clinical labeling, model construction was done in a data-driven manner to reduce potential bias. ROC and PRC analyses were used in the validation phase to evaluate the identified signature’s diagnostic performance with known cancer/control classifications.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Assessing robustness and diagnostic efficacy of the tRNA signature</h3>
<p id="Par30">The standardized tRNA signature score (T-score) of each sample was computed to classify samples as oncogenic or non-oncogenic. The T-score calculation relied on the expression levels (TPM-normalized) of the six tRNA signature genes across all samples in the dataset. Specifically, the calculation involves three steps:</p>
<ol class="list" style="list-style-type:decimal">
<li>
<p id="Par31">Geometric Mean Calculation:</p>
<div class="p" id="Par32">The geometric means of up-regulated (<em>S</em><sub><em>up</em></sub>) and down-regulated (<em>S</em><sub><em>down</em></sub>) tRNAs for each sample <em>i</em> were calculated as:<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1332" display="block"><msubsup><mrow><mi>G</mi><mi>e</mi><mi>o</mi><mi>M</mi><mi>e</mi><mi>a</mi><mi>n</mi></mrow><mrow><mi>u</mi><mi>p</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mo>=</mo><msup><mrow><mfenced close=")" open="("><mrow><msub><mrow><mo mathsize="big">∏</mo></mrow><mrow><msub><mrow><mi>j</mi><mi>ϵ</mi><mi>S</mi></mrow><mrow><mi>u</mi><mi>p</mi></mrow></msub></mrow></msub><msubsup><mrow><mi>T</mi><mi>P</mi><mi>M</mi></mrow><mrow><mi>j</mi></mrow><mrow><mi>i</mi></mrow></msubsup></mrow></mfenced></mrow><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mo>∣</mo><msub><mrow><mi>S</mi></mrow><mrow><mi>u</mi><mi>p</mi></mrow></msub><mo>∣</mo></mrow></mfrac></mrow></msup></math></td>
<td class="label">1</td>
</tr></table>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1394" display="block"><msubsup><mrow><mi>G</mi><mi>e</mi><mi>o</mi><mi>M</mi><mi>e</mi><mi>a</mi><mi>n</mi></mrow><mrow><mi>d</mi><mi>o</mi><mi>w</mi><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mo>=</mo><msup><mrow><mfenced close=")" open="("><mrow><msub><mrow><mo mathsize="big">∏</mo></mrow><mrow><msub><mrow><mi>j</mi><mi>ϵ</mi><mi>S</mi></mrow><mrow><mi>d</mi><mi>o</mi><mi>w</mi><mi>n</mi></mrow></msub></mrow></msub><msubsup><mrow><mi>T</mi><mi>P</mi><mi>M</mi></mrow><mrow><mi>j</mi></mrow><mrow><mi>i</mi></mrow></msubsup></mrow></mfenced></mrow><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mo>∣</mo><msub><mrow><mi>S</mi></mrow><mrow><mi>d</mi><mi>o</mi><mi>w</mi><mi>n</mi></mrow></msub><mo>∣</mo></mrow></mfrac></mrow></msup></math></td>
<td class="label">2</td>
</tr></table>
</div>
</li>
<li><div class="p" id="Par33">Preliminary Score: The preliminary score was computed as:<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1467" display="block"><msup><mrow><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi></mrow><mrow><mi>i</mi></mrow></msup><mo>=</mo><msubsup><mrow><mi>G</mi><mi>e</mi><mi>o</mi><mi>M</mi><mi>e</mi><mi>a</mi><mi>n</mi></mrow><mrow><mi>u</mi><mi>p</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mo>−</mo><msubsup><mrow><mi>G</mi><mi>e</mi><mi>o</mi><mi>M</mi><mi>e</mi><mi>a</mi><mi>n</mi></mrow><mrow><mi>d</mi><mi>o</mi><mi>w</mi><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msubsup></math></td>
<td class="label">3</td>
</tr></table>
</div></li>
<li><div class="p" id="Par34">Standardization:<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e1521" display="block"><mi>T</mi><mo>−</mo><msup><mrow><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi></mrow><mrow><mi>i</mi></mrow></msup><mo>=</mo><mfrac><mrow><msup><mrow><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi></mrow><mrow><mi>i</mi></mrow></msup><mo>−</mo><mi>μ</mi></mrow><mrow><mi>σ</mi></mrow></mfrac></math></td>
<td class="label">4</td>
</tr></table>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1556"><mi>μ</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e1565"><mi>σ</mi></math></span> are the mean and standard deviation of all preliminary scores.</div></li>
</ol>
<p id="Par35">To evaluate the discriminatory efficacy of the tRNA signature between cancerous and non-cancerous specimens, ROC and PRC analyses were employed based on the T-scores of samples across different phases. Herein, the AUC and the Area Under the Precision-Recall Curve (AUPRC) were determined, respectively, and summarized by the sample size weight. In the independent validation phase, the signature’s diagnostic accuracy was further scrutinized, contrasting NSCLC with benign and normal samples. Concurrently, the Mann–Whitney U test was utilized to compare T-scores between cancer and control samples across various studies. Similarly, the robustness of the tRNA signature was verified through distinct sample segregations based on age, gender, racial group, cancer subtype, smoking history, and histological stage. We systematically analyzed all available demographic and clinical data across datasets, with missing values excluded from the analysis (labeled as “NA” in Table <a href="#Tab1" class="usa-link">1</a>). The results were visualized using the summaryROCPlot and multiplePRCPlot functions from the MetaIntegrator package (version 2.1.3), along with the pROC (version 1.18.5) and ggplot2 packages (version 3.4.4) in R.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Survival analysis</h3>
<p id="Par36">We conducted survival analysis using retrospective clinical data (vital status, follow-up duration) from the TCGA-LUAD and TCGA-LUSC cohorts (2004–2013), combined with TPM-normalized tRNA expression data. The clinical endpoints included overall survival, defined as the time from diagnosis to death from any cause; and progression-free survival, defined as the time from diagnosis to disease progression or death, whichever occurred first. Patients with incomplete follow-up data or those lost to follow-up before a survival event were excluded, and samples with low expression in over 50% of tRNAs were removed to ensure robust biomarker evaluation.</p>
<p id="Par37">The required sample size for survival analysis was initially estimated using Schoenfeld’s formula for Cox proportional hazards regression, assuming HR = 1.5, a 60% event rate, 80% power, and α = 0.05, yielding a minimum of 80 patients (48 observed death events). To maintain statistical power, we included the full available dataset of 976 NSCLC patients without pre-stratification.</p>
<p id="Par38">We developed a risk score based on the expression levels of tRNAs, assigning scores of 1 for high expression and 0 for low, and determined a risk score cut-off using the Survminer package (version 0.4.9). Kaplan-Meier curves visualized survival implications. A multivariate Cox regression model incorporating risk scores with patient demographics and clinical features (age, cancer subtype, sex, smoking history, AJCC cancer stage) was used to evaluate prognostic factors. The results were visualized with ggplot2 (version 3.4.4), highlighting the prognostic relevance of tRNA in lung cancer.</p>
<p id="Par39">For the analysis of pathway-specific survival, Z-scores for pathway activity were computed by aggregating the TPM-normalized RNA expression values of pathway-associated genes identified through RNAhybrid (version 2.1.2) and KEGG enrichment analysis (using a significance threshold of <em>p</em> &lt; 0.05). A Cox proportional hazards model was employed to calculate risk scores for each sample based on pathway activity. The median risk score was used to categorize samples into high-risk and low-risk groups. Kaplan-Meier survival analysis was then conducted for overall metabolic pathways and individual pathways.</p>
<p id="Par40">After completing all survival analyses, a post-hoc power analysis was conducted to evaluate the statistical power of the study based on the observed hazard ratio (HR) and actual number of survival events. While the initial sample size estimation was based on HR = 1.5, the observed HR ranged from 1.3 to 2.2, prompting a reassessment of statistical adequacy. With 976 NSCLC patients and 390 observed deaths, the post-hoc power remained high ( ~ 95.6% to ~100%), confirming that the study is sufficiently powered to detect the moderate to strong prognostic impact of the tRNA signature.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Exploration of tRF expression data</h3>
<p id="Par41">We adopted limma package (version 3.56.2) to correct the batch effects across various datasets on TPM-normalized tRF expression data. Subsequently, the corrected data underwent meta-analysis using the MetaIntegrator package (version 2.1.3). This analysis focused on identifying tRFs with significant differential expression between cancerous and non-cancerous samples, characterized by a substantial effect size and an FDR of less than 0.05.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Correlation analysis between signature tRNAs and their originated tRFs</h3>
<p id="Par42">A Spearman rank correlation analysis was conducted to investigate the relationship between the expression levels of six signature tRNAs and their corresponding tRFs, utilizing TPM-normalized expression data for both tRNAs and tRFs. This analysis was performed on expression data from both individual datasets and a combined dataset comprising all tissue samples. The analysis outcomes, whether derived from individual or aggregated datasets, were visualized using the pheatmap package in R (version 1.0.12).</p></section><section id="Sec14"><h3 class="pmc_sec_title">tRF-targeted gene identification and functional enrichment</h3>
<p id="Par43">RNAhybrid (version 2.1.2) was utilized on a Linux platform to predict the target genes of two tRF biomarkers derived from our identified signature tRNAs. Target sequences for hybridization were obtained from human mRNA FASTA files (GRCh38/hg38) sourced from GENCODE Human (<a href="https://www.gencodegenes.org/human/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.gencodegenes.org/human/</a>). The criteria for prediction included setting the minimum free energy (MFE) threshold for hybridization between two sequences at −20 kcal/mol and applying a <em>p</em>value cutoff of 0.05. This analysis resulted in the identification of 712 potential target genes for the tRFs.</p>
<p id="Par44">We performed functional enrichment analyses on the predicted target genes using KEGG pathway and Gene Ontology (GO) analysis. KEGG pathways were analyzed with the NIH DAVID Functional Annotation Tool (<a href="https://david.ncifcrf.gov" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://david.ncifcrf.gov</a>), while GO terms were analyzed using the online tools of the Gene Ontology Resource (<a href="https://www.geneontology.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.geneontology.org/</a>). Significantly enriched pathways (<em>p</em> &lt; 0.05) and GO terms (FDR &lt; 0.05) were visualized using ggplot2 (version 3.4.4) in R and Cytoscape (version 3.10.1).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par45">Small RNA sequencing data were obtained from GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE110907</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE62182</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE83527</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175462</a>) and TCGA (TCGA-LUAD and TCGA-LUSC), totaling 1446 lung tissue samples (1173 tumor, 273 normal) and 233 plasma samples from RUSH University Medical Center for independent validation. Upstream analysis included FastQC (version 0.11.9), fastp (version 0.20.0), STAR (version 2.7.10), and FeatureCounts (version 2.0.6), with limma (version 3.56.2) applied for batch effect correction. Statistical analyses were performed in R (version 3.4.4) using standardized bioinformatics workflows. MetaIntegrator (version 2.1.3) identified a six-tRNA diagnostic signature, validated via leave-one-out cross-validation, forwardSearch/backwardSearch optimization, and ROC/PRC analysis. Kaplan-Meier survival analysis and multivariate Cox regression evaluated prognostic relevance, while Spearman correlation examined tRF-tRNA relationships. Functional analysis was conducted using RNAhybrid (version 2.1.2), KEGG (<a href="https://david.ncifcrf.gov" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://david.ncifcrf.gov</a>), and GO enrichment (<a href="https://www.geneontology.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.geneontology.org/</a>), ensuring robustness and reproducibility.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par46">Further information on research design is available in the <a href="#MOESM11" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec18"><h3 class="pmc_sec_title">tRNAs emerge as promising candidates for diagnostic signatures in lung cancer detection</h3>
<p id="Par47">A comprehensive analysis was conducted on a substantial dataset comprising 1679 samples, including 1446 tissue samples sourced from six distinct public datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE110907</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE62182</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE83527</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175462</a>, TCGA-LUAD, and TCGA-LUSC), as well as 233 exosome samples obtained from RUSH University Medical Center (Fig. <a href="#Fig1" class="usa-link">1a</a>). The sample distribution across cancer and control groups (including benign and normal samples) was nearly balanced across datasets, except for TCGA, which predominantly consisted of 991 cancer samples and 91 normal samples. Overall, the study encompassed 1290 cancer samples, 335 normal samples, and 54 benign samples, with detailed demographics, cancer subtypes, the American Joint Committee on Cancer (AJCC) pathologic stages, and smoking history presented in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Principal component analysis (PCA) and study design based on sample distribution.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370967_43856_2025_1068_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5a4/12370967/f726f7bba287/43856_2025_1068_Fig1_HTML.jpg" loading="lazy" id="d33e1669" height="718" width="667" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Sample size and diagnostic classification for each dataset included in the study. Bar colors represent different diagnostic categories. <strong>b</strong> PCA plot of all samples, where each point represents a specimen. Green points indicate control samples (normal and benign), orange points represent cancer samples. The shape of each point corresponds to its dataset of origin. <strong>c</strong> Schematic representation of the study’s methodological framework.</p></figcaption></figure><p id="Par48">Principal Component Analysis (PCA) revealed a significant separation in tRNA expression profiles between control and tumor groups (MANOVA, <em>p</em> = 2.2 × 10<sup>−16</sup>, partial η<sup>2</sup> = 0.23), highlighting the potential of tRNAs as discriminative markers for cancer detection (Fig. <a href="#Fig1" class="usa-link">1b</a>). Furthermore, minimal expression divergence was observed between LUAD and LUSC subtypes (Fig. <a href="#MOESM2" class="usa-link">S2</a>), suggesting that both NSCLC subtypes may share similar oncogenic pathways and cellular processes critical for tumor growth and survival, leading to overlapping molecular signatures.</p></section><section id="Sec19"><h3 class="pmc_sec_title">A six-tRNA signature has been identified and rigorously validated across multiple phases</h3>
<p id="Par49">The signature identification model was constructed using the MetaIntegrator algorithm, which integrates gene significance and effect size, supplemented by leave-one-out analysis to confirm the robustness of the identified markers<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. To ensure the model’s reliability, tissue samples were stratified into 70% for the discovery phase and 30% for hold-out validation, maintaining the original cancer-to-non-cancer ratio. Independent exosome samples were utilized for external validation (Fig. <a href="#Fig1" class="usa-link">1c</a>).</p>
<p id="Par50">During the discovery phase, 117 tRNAs (68 up-regulated and 49 down-regulated) were initially filtered (Data <a href="#MOESM5" class="usa-link">S2</a>). Further refinement using the forwardSearch and backwardSearch algorithms led to the selection of a robust six-tRNA signature, consisting of four up-regulated tRNAs (<em>tRNA-Val-CAC-2-1</em>, <em>tRNA-Leu-AAG-2-3</em>, <em>tRNA-Val-CAC-1-5</em>, and <em>tRNA-Lys-CTT-3-1</em>) and two down-regulated tRNAs (<em>tRNA-Ala-TGC-3-2</em> and <em>tRNA-Asp-GTC-1-1</em>) in tumor samples compared to the control group.</p>
<p id="Par51">The performance of the six-tRNA signature was evaluated using ROC and PRC curves based on the T-score, demonstrating an outstanding AUC of 0.97 in the discovery phase (Fig. <a href="#Fig2" class="usa-link">2a</a>). PRC analysis also showed excellent results, with high AUPRCs of 0.79-1.00 across databases (Fig. <a href="#Fig2" class="usa-link">2b</a>). Significant differences in T-scores between tumor and control samples were confirmed across datasets using Mann–Whitney U tests (Fig. <a href="#Fig2" class="usa-link">2c</a>). The signature was then successfully validated in the hold-out validation phase using the remaining tissue samples, yielding an AUC of 0.96, with a minimal AUPRC of 0.84, and statistically significant differences between tumor and control groups (Fig. <a href="#Fig2" class="usa-link">2d–f</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Performance evaluation of the 6-tRNA signature across different phases.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370967_43856_2025_1068_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5a4/12370967/5e63c3c3257c/43856_2025_1068_Fig2_HTML.jpg" loading="lazy" id="d33e1775" height="523" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>d</strong>, <strong>g</strong> Receiver Operating Characteristic (ROC) curves for T-scores in the discovery, hold-out validation, and independent validation phases, respectively. Curve colors correspond to their respective datasets. AUC, area under the curve; CI, confidence interval. <strong>b</strong>, <strong>e</strong>, <strong>h</strong> Precision-Recall Curves (PRC) for T-scores in the discovery, hold-out validation, and independent validation phases, respectively. Curve colors correspond to their respective datasets. AUPRC, area under the precision-recall curve; CI, confidence interval. <strong>c</strong>, <strong>f</strong>, <strong>i</strong> Boxplots of T-scores across different datasets, comparing tumor and control cohorts during the discovery, hold-out validation, and independent validation phases. Box colors represent different diagnostic categories. Sample sizes (n) for each cohort are displayed below the corresponding boxplots. Statistical comparisons were performed using the Mann–Whitney U test. Exact <em>p</em>-values are displayed above each comparison bar. <strong>j</strong> Heatmap of T-scores for each tRNA biomarker across all datasets and phases. “t” represents the discovery phase, “v” represents the hold-out validation phase, and “RUSH” denotes the independent validation phase. The color scale represents T-score values, ranging from 1.0 (orange) to -1.0 (purple).</p></figcaption></figure><p id="Par52">Further independent validation in a cohort from RUSH University Medical Center confirmed the diagnostic potential of the six-tRNA signature, with an AUC of 0.84 and AUPRC of 0.84, along with statistically significant T-scores (Fig. <a href="#Fig2" class="usa-link">2g–i</a>). This independent validation highlights the signature’s potential for application in liquid biopsy settings. Besides, additional analyses comparing NSCLC patients with benign individuals and with healthy controls within the RUSH cohort showed high AUC values of 0.85 and 0.82, respectively, with significant distinctions in T-scores between cancerous and non-cancerous cohorts (Fig. <a href="#MOESM2" class="usa-link">S3</a>). These findings reinforce the diagnostic precision of the six-tRNA signature in liquid biopsies, particularly in differentiating between malignant and benign conditions.</p>
<p id="Par53">To confirm that the imbalanced sample sizes of cancerous and non-cancerous cases did not introduce bias during model training and validation, a stratified sampling approach was applied to the TCGA dataset. The 991 tumor samples were randomly split into ten subsets, nine of which contained 99 samples each, and one contained 100 samples. Each tumor subset was paired with 91 control samples to form ten balanced groups. The performance of the six-tRNA signature was evaluated using ROC analysis for each group, resulting in a consistent AUC of 0.95 across all groups, further validating the robustness of the signature (Fig. <a href="#MOESM2" class="usa-link">S4</a>).</p>
<p id="Par54">In conclusion, the six-tRNA signature has been successfully identified and validated across multiple phases and datasets (Fig. <a href="#Fig2" class="usa-link">2j</a>, Fig. <a href="#MOESM2" class="usa-link">S5</a>), highlighting its potential as a reliable signature for the detection of NSCLC.</p></section><section id="Sec20"><h3 class="pmc_sec_title">The robustness of the T-score has been substantiated across a spectrum of demographic factors, histological subtypes, smoking statuses, and AJCC pathologic stages</h3>
<p id="Par55">The collected data encompassing age, sex, histological subtype, AJCC pathologic stage, and smoking history of the study cohort are systematically presented in Table <a href="#Tab1" class="usa-link">1</a>. The robustness of the six-tRNA signature’s T-score across diverse categorizations of the samples was assessed by employing ROC, PRC analyses, and Mann–Whitney U tests (Fig. 3, S6). Integration of patient age data from both TCGA and RUSH datasets revealed that the T-score consistently exhibited high performance across various age demographics, as evidenced by an AUC value of 0.86, with no individual age group demonstrating an AUC of below 0.83 (Fig. <a href="#Fig3" class="usa-link">3a</a>). Concurrently, statistical evaluations indicated a significant T-score difference between tumor and control samples across all age ranges (Fig. <a href="#Fig3" class="usa-link">3b</a>). Further validation of the T-score’s robustness was conducted across different sexes, histological subtypes, and smoking statuses, synthesizing data from all available datasets, which suggests a uniformly high efficacy irrespective of these variables (Fig. <a href="#Fig3" class="usa-link">3c–h</a>). Crucially, comparative analyses of the T-score’s effectiveness across various AJCC pathologic stages revealed its remarkable proficiency in diagnosing both early-stage (stages I and II) and advanced-stage (stages III and IV) NSCLC, with each stage demonstrating AUC values of no less than 0.94 (Fig. <a href="#Fig3" class="usa-link">3i</a>). Correspondingly, a significant distinction was observed between the tumor and control samples, underscoring the T-score’s accurate diagnostic performance for early-stage NSCLC (Fig. <a href="#Fig3" class="usa-link">3j</a>). PRC analyses across these factors also demonstrated outstanding performance, with all AUPRC values exceeding 0.90, further supporting the diagnostic robustness of the 6-tRNA signature (Fig. <a href="#MOESM2" class="usa-link">S6</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Evaluation of T-score robustness in stratified analyses by age, sex, histological subtype, smoking history, and AJCC cancer stage in NSCLC patients.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370967_43856_2025_1068_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5a4/12370967/67ee0320c88a/43856_2025_1068_Fig3_HTML.jpg" loading="lazy" id="d33e1864" height="940" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>c</strong>, <strong>e</strong>, <strong>g</strong>, <strong>i</strong> Receiver Operating Characteristic (ROC) curves assessing the discriminatory performance of the tRNA signature across different age groups, sexes, histological subtypes, smoking history categories, and AJCC cancer stages. Curve colors correspond to their respective demographic categories. AUC area under the curve, CI confidence interval. <strong>b</strong>, <strong>d</strong>, <strong>f</strong>, <strong>h</strong>, <strong>j</strong> Boxplots comparing T-scores between tumor and control cohorts across different stratified factors. Box colors represent different diagnostic categories. Sample sizes (n) for each cohort are displayed below the corresponding boxplots. Statistical comparisons were performed using the Mann–Whitney U test. Exact <em>p</em> values are displayed above each comparison bar.</p></figcaption></figure><p id="Par56">In addition to these factors, the T-score was further validated across racial groups. The analysis revealed that the T-score demonstrated excellent diagnostic performance in White and Asian populations, with AUC values exceeding 0.84. However, its diagnostic accuracy was notably lower in Black individuals, reflecting a reduced AUC and diminished statistical significance. This discrepancy may be due to the underrepresentation of Black individuals in the datasets, which limits the model’s ability to capture potential biological variations specific to this population (Fig. <a href="#MOESM2" class="usa-link">S7</a>).</p>
<p id="Par57">In conclusion, the T-score emerges as a potential independent biomarker for NSCLC diagnosis, transcending variations in sex, age groups, histological subtypes, smoking history, and pathologic stages.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Three tRNAs within the signature were validated to have prognostic values that are capable of being independent predictive factors</h3>
<p id="Par58">Upon analyzing data from 976 lung cancer patients (filtered from the original 1,089 cases in the TCGA repository, including 585 TCGA-LUAD and 504 TCGA-LUSC samples), three tRNAs from the identified six-tRNA signature—<em>tRNA-Lys-CTT-3-1</em>, <em>tRNA-Val-CAC-2-1</em>, and <em>tRNA-Leu-AAG-2-3</em>—demonstrated significant prognostic value. Kaplan-Meier survival analysis, followed by log-rank testing, revealed that patients with higher expression levels of these tRNAs exhibited significantly poorer survival outcomes. Specifically, <em>tRNA-Lys-CTT-3-1</em> was associated with a hazard ratio (HR) of 1.37 (<em>p</em> = 2.44 × 10<sup>−3</sup>), <em>tRNA-Val-CAC-2-1</em> with an HR of 1.33 (<em>p</em> = 3.02 × 10<sup>−2</sup>), and <em>tRNA-Leu-AAG-2-3</em> with an HR of 1.30 (<em>p</em> = 1.07 × 10<sup>−2</sup>). These findings suggest that elevated expression of these three tRNAs is linked to a higher risk of mortality, highlighting their potential as independent prognostic biomarkers in lung cancer(Fig. <a href="#Fig4" class="usa-link">4a–c</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Survival analysis of individual tRNA biomarkers and composite risk scores in the TCGA patient cohort.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370967_43856_2025_1068_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5a4/12370967/f478376d85bd/43856_2025_1068_Fig4_HTML.jpg" loading="lazy" id="d33e1927" height="596" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a–c</strong> Kaplan-Meier survival curves illustrating the prognostic impact of individual tRNA biomarkers in NSCLC patients. Curve colors correspond to expression levels, where blue represents low expression and red represents high expression. <strong>d</strong> Kaplan-Meier survival curve showing the effect of the composite risk score on patient prognosis. Curve colors correspond to risk score groups, with yellow indicating the high-risk group and blue indicating the low-risk group. Dashed lines denote the median survival threshold, corresponding to a 50% survival probability for the respective patient cohorts at specific time points. The tables below each plot indicate the number of patients at risk over the 72-month observation period. Statistical significance was determined using the log-rank test. HR hazard ratio, CI confidence interval.</p></figcaption></figure><p id="Par59">To further assess the prognostic significance of these biomarkers, a cumulative risk score was derived by integrating the expression levels of <em>tRNA-Lys-CTT-3-1, tRNA-Val-CAC-2-1</em>, and <em>tRNA-Leu-AAG-2-3</em>. Patients were categorized into high-risk and low-risk groups based on an optimal threshold identified using the Survminer package. Kaplan-Meier survival analysis revealed a statistically significant survival disparity between the two groups (log-rank test, <em>p</em> = 5.41 × 10<sup>−4</sup>, HR = 1.44), with low-risk patients exhibiting a median survival advantage of approximately three years over those in the high-risk group during the 72-month follow-up period. Notably, the cumulative risk score demonstrated superior prognostic performance, providing greater predictive accuracy than any single tRNA alone (Fig. <a href="#Fig4" class="usa-link">4d</a>).</p>
<p id="Par60">Additional validation using multivariate Cox regression analysis confirmed the risk score as an independent prognostic factor for NSCLC, alongside other factors such as pathological subtype, sex, age, smoking history, and AJCC cancer stage (Wald test, <em>p</em> = 6.27 × 10<sup>–4</sup>, HR = 1.80; Table <a href="#Tab2" class="usa-link">2</a>). Similar to previous findings, the lower risk score predicted longer survival, underscoring its potential as an efficient prognostic tool in NSCLC<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Multivariate cox regression analysis of independent predictors of survival in NSCLC patients</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Subgroup</th>
<th colspan="1" rowspan="1">Number</th>
<th colspan="1" rowspan="1">Adjusted HR (95% CI)</th>
<th colspan="1" rowspan="1">
<em>P</em> value</th>
<th colspan="1" rowspan="1">Significance</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Risk_score</strong></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Low</td>
<td align="left" colspan="1" rowspan="1">415</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">High</td>
<td align="left" colspan="1" rowspan="1">576</td>
<td align="left" colspan="1" rowspan="1">1.80 (1.45–1.17)</td>
<td align="left" colspan="1" rowspan="1">6.27E-04</td>
<td align="left" colspan="1" rowspan="1">***</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Subtype</strong></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LUAD</td>
<td align="left" colspan="1" rowspan="1">513</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LUSC</td>
<td align="left" colspan="1" rowspan="1">478</td>
<td align="left" colspan="1" rowspan="1">1.08 (0.86–1.35)</td>
<td align="left" colspan="1" rowspan="1">5.15E-01</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Sex</strong></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">398</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">593</td>
<td align="left" colspan="1" rowspan="1">1.03 (0.83–1.29)</td>
<td align="left" colspan="1" rowspan="1">7.84E-01</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Age</strong></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Age</td>
<td align="left" colspan="1" rowspan="1">991</td>
<td align="left" colspan="1" rowspan="1">1.01 (1.00–1.03)</td>
<td align="left" colspan="1" rowspan="1">1.89E-02</td>
<td align="left" colspan="1" rowspan="1">*</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Smoking_history</strong></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">235</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">756</td>
<td align="left" colspan="1" rowspan="1">0.88 (0.69–1.13)</td>
<td align="left" colspan="1" rowspan="1">3.25E-01</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>AJCC_stage</strong></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Stage I</td>
<td align="left" colspan="1" rowspan="1">507</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1">Reference</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Stage II</td>
<td align="left" colspan="1" rowspan="1">279</td>
<td align="left" colspan="1" rowspan="1">1.52 (1.19–1.95)</td>
<td align="left" colspan="1" rowspan="1">8.56E-04</td>
<td align="left" colspan="1" rowspan="1">***</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Stage III</td>
<td align="left" colspan="1" rowspan="1">164</td>
<td align="left" colspan="1" rowspan="1">2.29 (1.76–2.97)</td>
<td align="left" colspan="1" rowspan="1">7.61E-10</td>
<td align="left" colspan="1" rowspan="1">***</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Stage IV</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">3.28 (2.05–5.24)</td>
<td align="left" colspan="1" rowspan="1">7.17E-07</td>
<td align="left" colspan="1" rowspan="1">***</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p68"><p>The reference category for each variable is indicated as “Reference” under the “Adjusted HR” and “<em>P </em>value” columns, serving as the baseline for comparison. Statistical significance is denoted by asterisks: “*” for <em>p</em> &lt; 0.05; “**” for <em>p</em> &lt; 0.01; “***” for <em>p</em> &lt; 0.001.</p></div></div></section></section><section id="Sec22"><h3 class="pmc_sec_title">Two tRNAs in the signature may exert regulatory influences through associated tRNA fragments</h3>
<p id="Par61">Transfer RNA fragments (tRFs) have emerged as vital regulators in the post-transcriptional process of tumorigenesis through interactions with mRNA molecules<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Despite their recognized roles, the connection between tRFs and their parent tRNA expressions has not been well-characterized. To address this gap, the study investigated the expression patterns correlating tRFs with identified tRNAs. We found 69 tRFs with substantial differential expression between malignant and non-cancerous samples: 18 were up-regulated, and 51 were down-regulated (Fig. <a href="#Fig5" class="usa-link">5a</a>, Data <a href="#MOESM6" class="usa-link">S3</a>). Intriguingly, two up-regulated tRFs, namely <em>5’-tRF-Val-CAC-2-1</em> and <em>5’-tRF-Val-CAC-1-5</em>, were derived from signature tRNAs (Fig. <a href="#Fig5" class="usa-link">5b</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Regulatory impact of signature tRNAs through associated tRNA fragments (tRFs).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370967_43856_2025_1068_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5a4/12370967/35e1d60af7e0/43856_2025_1068_Fig5_HTML.jpg" loading="lazy" id="d33e2249" height="991" width="733" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Heatmap illustrating the expression levels of differentially expressed tRFs across multiple datasets. The color scale represents T-score values, ranging from 1.0 (orange) to –1.0 (purple). <strong>b</strong> Forest plot depicting the meta-analysis of standardized mean differences (SMDs) for two tRFs across different datasets. Each dataset is represented by an individual effect estimate (gray box), with 95% confidence intervals (CIs) shown as blue lines. The size of the gray box reflects the sample size within each dataset. The red diamond represents the overall summary estimate from the meta-analysis. A positive SMD indicates higher expression in tumor samples, while a negative SMD suggests lower expression. <strong>c</strong> Heatmap of Spearman’s rank correlation coefficients between signature tRNAs and their corresponding tRFs, aggregated from all datasets. The color scale represents correlation coefficients ranging from –1 (red) to 1 (dark blue). <strong>d</strong> Functional interaction network illustrating the relationships between two identified tRFs and their associated mRNAs, as determined by KEGG pathway enrichment analysis. Diamonds represent tRFs; ellipses denote mRNAs. The colors of mRNA groups correspond to distinct KEGG pathways. <strong>e</strong> Bar plot summarizing the results of KEGG pathway enrichment analysis. Bar length represents the number of genes enriched in each pathway. Statistical significance is indicated by a <em>p</em> value scale ranging from 0.05 (light green) to 0.01 (gray). <strong>f</strong>, <strong>g</strong>, <strong>h</strong> Bubble plots representing Gene Ontology (GO) enrichment analysis results, detailing biological processes (BP), cellular components (CC), and molecular functions (MF). Bubble size corresponds to the number of enriched genes, and false discovery rate (FDR) values are color-scaled from 0.05 (light green) to 0.01 (gray) to indicate statistical significance.</p></figcaption></figure><p id="Par62">Based on TPM-normalized expression profiles, a Spearman correlation analysis was performed across individual datasets and aggregate samples to see whether there was a correlation between these tRFs and their source tRNAs. <em>5’-tRF-Val-CAC-2-1 / 5’-tRF-Val-CAC-1-5</em> and their parent tRNAs (<em>tRNA-Val-CAC-2-1 / tRNA-Val-CAC-1-5</em>) showed a strong positive association with each other, with correlation coefficients of 0.78 and 0.75 in the entire sample population, respectively (Fig. <a href="#Fig5" class="usa-link">5c</a>). Individual datasets consistently showed this link (Fig. <a href="#MOESM2" class="usa-link">S8</a>), suggesting that the signature tRNAs may use the corresponding tRFs to achieve their regulatory function.</p>
<p id="Par63">Following these results, a strict minimum free energy (MFE) criterion of -20 kcal/mol was applied to the RNAhybrid tool to discover mRNA targets that might interact with the identified tRFs. A total of 957 mRNAs were found to interact with the tRFs in a significant way in our analysis (Data <a href="#MOESM7" class="usa-link">S4</a>). Significant enrichment was found in six important pathways by subsequent KEGG pathway enrichment analysis, with metabolic pathways being the most significantly enriched, comprising 92 identified genes. Other enriched pathways included glycan degradation, homologous recombination, RNA polymerase, adherens junction, and O-glycan biosynthesis. These pathways are crucial for preserving the genomic stability and energy production needed for tumor cell survival and proliferation (Fig. <a href="#Fig5" class="usa-link">5d, e</a>; Data <a href="#MOESM8" class="usa-link">S5</a>)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p>
<p id="Par64">Similarly, Gene Ontology (GO) enrichment analysis identified important gene accumulations in biological processes. According to gene count, the top five processes were metabolic, organic substance metabolic, cellular, and main metabolic processes (Fig. <a href="#Fig5" class="usa-link">5f</a>; Data <a href="#MOESM9" class="usa-link">S6</a>). These fundamental cellular and metabolic processes are often altered in cancer by modifying cellular metabolism and other processes to favor tumorigenesis<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Furthermore, cellular elements such as organelles and intracellular structures were noticeably overrepresented (Fig. <a href="#Fig5" class="usa-link">5g</a>), and molecular function analysis revealed gene aggregation linked to binding and catalytic activities (Fig. <a href="#Fig5" class="usa-link">5h</a>), supporting the various functions of tRFs in cellular function and cancer development.</p>
<p id="Par65">To explore the potential regulatory role of tRFs and their target pathways, we conducted survival analysis on the expression levels of specific metabolic pathways (Fig.6, S9). Kaplan-Meier survival analysis for the overall metabolic pathway demonstrated significant prognostic value in NSCLC (log-rank test, <em>p</em> = 4.26 × 10<sup>−15</sup>, HR = 2.29; Fig. <a href="#Fig6" class="usa-link">6a</a>). KEGG analysis of these tRF target genes highlighted pathways including biosynthesis of cofactors, purine metabolism, oxidative phosphorylation, other types of O-glycan biosynthesis, nucleotide metabolism, lysine degradation, amino acid biosynthesis, and the phosphatidylinositol signaling system (Fig. <a href="#Fig6" class="usa-link">6b</a>, Data <a href="#MOESM10" class="usa-link">S7</a>). Further survival analysis revealed significant prognostic differences across multiple metabolic pathways, including biosynthesis of cofactors (<em>p</em> = 4.17 × 10<sup>−5</sup>, HR = 1.52; Fig. <a href="#Fig6" class="usa-link">6c</a>), purine metabolism (<em>p</em> = 3.21 × 10<sup>−2</sup>, HR = 1.24; Fig. <a href="#Fig6" class="usa-link">6d</a>), oxidative phosphorylation (<em>p</em> = 3.98 ×  10<sup>−2</sup>, HR = 1.23; Fig. <a href="#Fig6" class="usa-link">6e</a>), and alanine, aspartate, and glutamate metabolism (<em>p</em> = 2.06 ×  10<sup>−4</sup>, HR = 1.46; Fig. <a href="#Fig6" class="usa-link">6f</a>). In all cases, lower risk scores correlated with prolonged survival, underscoring a strong association between these metabolic pathways and NSCLC progression.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Survival analysis of tRF-targeted metabolic pathways.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370967_43856_2025_1068_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5a4/12370967/67c9c1825506/43856_2025_1068_Fig6_HTML.jpg" loading="lazy" id="d33e2370" height="945" width="788" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Kaplan-Meier survival curve illustrating the prognostic significance of genes enriched in overall metabolic pathways. Dashed lines indicate the median survival threshold, corresponding to a 50% survival probability for the respective patient cohorts at specific time points. The tables below the plot display the number of patients at risk over the observation period. Statistical significance was determined using the log-rank test. HR hazard ratio, CI confidence interval. <strong>b</strong> KEGG pathway enrichment analysis identifying specific metabolic pathways associated with the enriched genes. Statistical significance is indicated by a p-value scale ranging from 0.05 (light green) to 0.01 (gray). <strong>c–f</strong> Kaplan-Meier survival curves comparing high-risk and low-risk groups based on pathway activity scores for key metabolic pathways, including biosynthesis of cofactors, purine metabolism, oxidative phosphorylation, and alanine, aspartate, and glutamate metabolism. Dashed lines represent the median survival threshold, corresponding to a 50% survival probability for the respective patient cohorts at specific time points. The tables below each plot show the number of patients at risk over the observation period. Statistical significance was determined using the log-rank test. HR hazard ratio, CI confidence interval.</p></figcaption></figure><p id="Par66">In conclusion, our results suggest that tRNAs, through their processing into tRFs and their influence on mRNA stability and translational regulation, play an essential role in modulating metabolic networks crucial for NSCLC tumorigenesis.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par67">This study developed a comprehensive model to analyze small RNA sequencing data from multiple studies, identifying a six-tRNA signature for NSCLC diagnosis. Utilizing 1446 lung tissue samples from five public datasets, the study proceeded through an initial discovery phase for rigorous model training, followed by a hold-out validation phase to confirm the model’s robustness and the efficacy of the gene signature. Independent validation using a cohort of 233 plasma exosome samples further substantiated the diagnostic accuracy of the six-tRNA signature, demonstrating its utility for non-invasive, real-time liquid biopsies. Notably, the diagnostic performance of the signature remained consistently high across various demographics and clinical characteristics, including age, gender, smoking history, cancer subtypes, and histological stages. Furthermore, the signature’s capacity to distinguish between cancerous and benign conditions highlights its noteworthy potential for clinical application in the early screening of NSCLC.</p>
<p id="Par68">tRNAs are classified into three distinct hierarchical levels: isoacceptors, isodecoders, and individual tRNAs, with isoacceptors comprising tRNAs with differing anticodons but carrying the same amino acid, isodecoders sharing identical anticodons but differing in nucleotide sequences, and individual tRNAs maintaining constant sequences but located at different genomic sites. Historically, the focus on tRNAs within cancer research has predominantly been on the differential expression of isoacceptors and the resultant protein variations, given their classical function in amino acid transfer during mRNA translation<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. However, recent advancements have illuminated the important roles of tRNAs in non-canonical functions, especially in epigenetics, such as tRNA fragments’ regulatory effects on gene expression in post-transcription, underscoring the imperative to investigate tRNAs beyond isoacceptors to include isodecoders and individual tRNAs<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Our study represents an effort to identify lung cancer-specific diagnostic tRNA signature at the individual tRNA level, revealing the complex network of non-traditional tRNA functions from an epigenetic standpoint, thereby emphasizing the importance and potential of exploring the multifaceted roles of tRNAs in cancer pathogenesis.</p>
<p id="Par69">Interestingly, our pre-analysis using PCA revealed no significant difference in tRNA expression between the two main NSCLC subtypes, LUAD and LUSC. As essential components of protein synthesis and cellular metabolism, tRNAs are often dysregulated across various cancer types. However, their expression may not differ significantly between closely related subtypes within the same organ. This observation aligns with findings from the public database DBtrend (<a href="https://trend.pmrc.re.kr/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://trend.pmrc.re.kr/</a>)<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, indicating no significant differences in tRNA expression between LUAD and LUSC cohorts. The absence of distinct separation between these subtypes further confirms the feasibility of studying tRNA diagnostic biomarkers across the entire NSCLC spectrum, encompassing both LUAD and LUSC.</p>
<p id="Par70">We included all collected samples in the model development process to strengthen the robustness of our signature identification model. However, this approach presented a challenge of sample imbalance, which is mainly characterized by the greater number of cancerous samples relative to non-cancerous ones. To address this issue and mitigate potential biases in model construction, we implemented several strategies. Firstly, we opted for a stratified splitting approach instead of random sampling for dividing the samples into the discovery and hold-out validation phases, ensuring the maintenance of the original proportion of tumor samples to control samples within the dataset and reducing bias due to uneven sample distribution. Secondly, we utilized the Leave-One-Out (LOO) algorithm within the MetaIntegrator framework, which systematically excludes one dataset at a time for analysis, cycling through until each dataset has been excluded once, thus preventing the dominance of any single study due to its large sample size in influencing the outcome and ensuring that the identified signature is representative across all datasets. Thirdly, to rigorously evaluate the diagnostic performance of our model, we employed the Mann-Whitney U test in conjunction with PRC analysis; the Mann–Whitney U test determines the significance of the identified signature within each dataset for a given phase, while the PRC is more sensitive to the imbalanced samples and provides a more nuanced assessment of diagnostic performance. Lastly, a targeted examination of the TCGA dataset, the main source of sample imbalance, was conducted through the resampling of tumor samples, which were then systematically paired with normal controls, confirming the high diagnostic efficacy of the six-tRNA signature and affirming that no bias was introduced during the model construction. This multifaceted strategy demonstrates our dedication to creating an unbiased and scientifically valid diagnosis methodology.</p>
<p id="Par71">Our six-tRNA signature, initially identified in tissue samples and subsequently validated in both tissue and plasma exosomes, demonstrates outstanding robustness. Exosomes, which can be released into the circulation from various parent cells, are actively generated by tumor cells and can accumulate to concentrations of at least 10<sup>9</sup> vesicles per milliliter of blood. The abundance of exosomes secreted by tumor cells demonstrates a “seeds and plants” relationship between circulating exosomes and malignancies<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. These exosomes act as mediators of cell-to-cell communication at distant sites, carrying functional particles—including DNA, RNA, lipids, proteins, and metabolites—originating from tumor cells and thus facilitating tumor growth and metastasis<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. By validating the tissue-identified signature in exosomes, our results support the biomarker’s ability to accurately detect the presence of NSCLC in a variety of sample types. This versatility highlights its clinical applicability, underscoring its promising potential for diagnostic applications in both invasive and non-invasive settings.</p>
<p id="Par72">The identification of the six signature tRNAs in our study aligns with findings from broader tRNA research in humans. A previous study developed a comprehensive tRNA database (DBtRend, available at <a href="https://trend.pmrc.re.kr/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://trend.pmrc.re.kr/</a>) using a specialized pipeline to analyze human small RNA or microRNA sequencing data from TCGA and GEO datasets<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The signature tRNAs identified in our research directly correspond with their regulatory trends as DBtRend indicates. Specifically, <em>tRNA-Val-CAC-2-1</em>, <em>tRNA-Leu-AAG-2-3</em>, <em>tRNA-Val-CAC-1-5</em>, and <em>tRNA-Lys-CTT-3-1</em> demonstrated significant upregulation in tumor samples from both the TCGA-LUAD and TCGA-LUSC cohorts compared to normal controls<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Additionally, Kuang et al. reported that high expression of <em>tRNA-Leu-AAG</em> is associated with specific pathological subtypes of LUAD, while upregulation of <em>tRNA-Lys-CTT</em> is linked to tumor differentiation<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. This consistency with previous research underscores the reliability of the signature tRNAs identified in our investigation.</p>
<p id="Par73">Three distinct tRNAs—<em>tRNA-Lys-CTT-3-1</em>, <em>tRNA-Val-CAC-2-1</em>, and <em>tRNA-Leu-AAG-2-3</em>—were revealed to be significant predictors of survival in NSCLC patients, supporting the predictive value of the identified tRNA signature. The significance of <em>tRNA-Lys-CTT-3-1</em> in our findings is supported by previous research that has demonstrated its involvement in LUAD prognosis<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. As a useful independent predictor of survival, the composite risk score derived from these tRNAs offers an improved method of patient categorization. Our findings are also comparable to a few prognostic studies, which link worse survival outcomes to advanced cancer stages and advanced age<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Furthermore, while gender and histological subtype had minor effects on survival, our data suggest a tendency for male patients to be at higher risk, which is consistent with research that predominantly attributes these differences to demographic and diagnostic factors<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Likewise, although the difference was not statistically significant between cancer subtypes, LUAD patients tended to have higher survival outcomes than LUSC patients<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Interestingly, no significant survival difference was observed between smokers and non-smokers, likely due to limited smoking history data in TCGA, such as pack-years, cessation duration, and smoking initiation age. Furthermore, treatment regimens, which were not considered, may also impact outcomes. Future studies should incorporate detailed smoking and treatment data, and use prospective, multi-omics approaches to improve tRNA-based survival models.</p>
<p id="Par74">The correlation analysis suggests two signature tRNAs may regulate gene transcription through their derived tRNA fragments (tRFs). Supporting evidence from prior studies shows that variations in the expression of specific tRNA isodecoders may lead to changes in their corresponding tRFs within human tissues<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. These changes are associated with pathological processes in infections, neurodegenerative diseases, and cancers, where increased tRNA turnover rates have been observed in tumor tissues compared to normal tissues, correlating tRF levels with cancer histological stages<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. For example, 5’ tRFs derived from <em>tRNA</em><sup><em>Glu</em></sup>, <em>tRNA</em><sup><em>Ser</em></sup>, <em>tRNA</em><sup><em>Leu</em></sup>, and <em>tRNA</em><sup><em>Arg</em></sup> are notably overexpressed in prostate cancer cell lines, even exceeding the expression levels of many microRNAs (miRNAs)<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Similarly, in HeLa cells, 5’ tRFs from <em>tRNA</em><sup><em>Lys</em></sup>, <em>tRNA</em><sup><em>Val</em></sup>, <em>tRNA</em><sup><em>Gln</em></sup>, and <em>tRNA</em><sup><em>Arg</em></sup> are expressed at levels comparable to abundant miRNAs like miR-21 and the let-7 family<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Consistently, in our study, <em>5’-tRF-Val-CAC-2-1</em> and <em>5’-tRF-Val-CAC-1-5</em> demonstrated high regulation in NSCLC tissue and plasma samples, with a positive correlation to their originating tRNAs. Furthermore, <em>5’-tRF-Val-CAC</em> has been recognized as a potential biomarker in several cancers, including LUAD, oral squamous cell carcinoma, bladder cancer, etc<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Its role in cancer varies; for example, in gastric cancer, it critically regulates MAPK protein expression to influence tumorigenesis<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>; and in breast cancer, it participates in the tRF-17/THBS1/TGF-β1/Smad3 signaling pathway<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>.</p>
<p id="Par75">The KEGG results demonstrated the potential roles of tRF-regulated pathways in NSCLC progression and prognosis. Metabolic pathways identified in the KEGG enrichment analysis, including biosynthesis of cofactors, purine metabolism, oxidative phosphorylation, and alanine, aspartate, and glutamate metabolism, play fundamental roles in cellular energy production, biosynthetic processes, and maintenance of redox balance. These findings suggest that tRF-regulated pathways, particularly those involved in metabolism, contribute essentially to NSCLC progression and prognosis by supporting metabolic reprogramming, energy production, and tumor cell survival.</p>
<p id="Par76">In previous lung cancer studies, the biosynthesis of cofactors is critical for activating enzymes involved in cancer metabolism, tumor proliferation, immune response, transcriptional regulation, and drug resistance<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. For instance, <em>NFS1</em>, a gene enriched in this pathway, is vital in maintaining iron-sulfur clusters, which are essential for multiple cellular proteins. <em>NFS1</em> helps protect lung tumor cells from ferroptosis by mitigating oxidative damage, thereby promoting tumor survival<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Additionally, <em>NME3</em>, identified as a component of this pathway, enhances TLR5-mediated NFκB signaling in response to flagellin, thereby contributing to antitumor immunity in lung cancer patients<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Similarly, the purine metabolism pathway, crucial for nucleotide synthesis, is often upregulated in cancers to meet the increased demands for DNA and RNA synthesis during tumor proliferation<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. This pathway has been explored as a therapeutic target in NSCLC, with inhibitors showing promise in reducing tumor growth. For example, <em>ATIC</em>, a gene enriched in this pathway, has been reported to promote LUAD cell growth and migration by enhancing Myc expression, underscoring its role in tumor progression<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Additionally, oxidative phosphorylation (OXPHOS), the primary ATP-generating pathway, is often reprogrammed in cancers, including NSCLC, to adapt to hypoxic tumor microenvironments. While many cancers predominantly rely on glycolysis<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>, OXPHOS remains an essential mechanism for energy production, especially in NSCLC, supporting tumor cell division, migration, and invasion<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. For instance, <em>ATP5F1B</em>, encoding a mitochondrial ATP synthase subunit, has been shown to inversely correlate with aerobic glycolysis rates in cancer cells. This highlights mitochondrial bioenergetic alterations as key contributors to glucose metabolism in NSCLC<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Furthermore, <em>UQCRC2</em>, another gene in this pathway, has been identified as a prognostic biomarker for LUAD in previous studies<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. The alanine, aspartate, and glutamate metabolism pathway is integral to amino acid biosynthesis, nitrogen balance, and energy generation via the tricarboxylic acid (TCA) cycle<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Dysregulation of this pathway can disrupt nitrogen metabolism, driving the metabolic reprogramming commonly observed in NSCLC<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Genes such as <em>GLS2</em>, enriched in this pathway, play a protective role by reducing oxidative stress in NSCLC cells<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Additionally, <em>ASNS</em> has been implicated in promoting lung tumor cell metastasis via the Wnt pathway and mitochondrial functions, even in the absence of endogenous asparagine production<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par77">The diagnostic potential of tRNAs in NSCLC has been highlighted in the research; nevertheless, certain aspects still require improvement. First, while small RNA-seq can approximate tRNA expression levels, accurately quantifying mature tRNAs from our samples remains challenging due to their inherent secondary structures and extensive post-transcriptional modifications, which can compromise sequencing quality <sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>,<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. Additionally, the sequence similarity between precursor tRNAs, mature tRNAs, and tRNA fragments often results in ambiguous alignments during analysis<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Despite our efforts to fine-tune alignment and annotation parameters with stringent thresholds, ambiguities persist. The emerging technology of tRNA-seq offers a solution to these issues, potentially providing more accurate insights into tRNA biology as more data become available<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. Furthermore, the development of refined tRNA mapping pipelines warrants comparison with our methods to enhance the precision of tRNA mapping<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Second, the small RNA-seq data available in public databases predominantly represent individuals of European descent, with limited representation from African Americans, Pacific Islanders, and Asians. This lack of diversity likely contributes to the unbalanced performance of our model and signature across racial groups, as our analysis showed consistently higher diagnostic accuracy in distinguishing between tumor and control samples in White and Asian populations, while the accuracy was relatively lower in the Black population. Addressing this limitation in future studies by expanding data collection to include a more diverse representation of racial groups will improve the generalizability of our findings and mitigate potential biases arising from genetic and racial differences. Third, our functional analyses indicate that the identified signature tRNAs may regulate gene expression post-transcriptionally through their derivatives, corroborating the prognostic value of specific tRF-targeted pathways, consistent with findings from other cancer studies. However, while our results highlight the potential significance of these pathways, the precise molecular mechanisms mediated by these signature tRNAs remain speculative. Future research should focus on elucidating these mechanisms through experimental validation and benchwork studies, which will not only enhance our understanding of their biological roles but also identify potential therapeutic targets for cancer treatment.</p>
<p id="Par78">In a word, this study underscores the critical role of the tRNA signature in the advancement of diagnostic methodologies for NSCLC, providing encouraging paths for early, non-invasive detection methods.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (478.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">43856_2025_1068_MOESM3_ESM.pdf</a><sup> (32.7KB, pdf) </sup><p>Description of Additional Supplementary files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (81.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (28.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (22.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (81KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (12.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (14.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (14.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (79.1KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This research was supported by grants from the National Institutes of Health (NIH): R01CA230514, R01CA223490, P20GM103466, P30CA071789, P20GM139753, U54GM138062, U54HG013243, T32DK137523, 1UE5HG013826, 3OT2OD032581-01S5-895 and 1OT2OD032581‐02‐PP90Y to Y.D. This work was also supported by NIH grants U24MD015970 and RCC-004UHI-Pilot, awarded to Y.F. Additional support was provided by NIH grant U54MD007601, awarded to M.N. We would like to express our appreciation to Orion S. Rivers and Tina Weatherby Carvalho of the University of Hawaii Pacific Biosciences Research Center Biological Electron Microscope Facility for their expert assistance with electron microscopy.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Z.F.: Conceptualization, Methodology, Formal analysis, Visualization, Writing—Original Draft; M.N.: Validation, Investigation, Resources; G.D., A.A.A.-G., O.T.M.C., J.A.B.: Resources, Writing—Review &amp; Editing; Z.G.: Methodology, Data Curation; H.Z., Y.C., T.G., G.L.: Data Curation; H.Y.: Project administration; L.W.*, Y.F.*, Y.D.*: Conceptualization, Supervision, Project administration, Funding acquisition.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par79"><em>Communications Medicine</em> thanks Christos Kontos and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [<a href="#MOESM1" class="usa-link">Peer reviewer reports</a> are available].</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Publicly available small RNA sequencing data and corresponding clinical information were obtained from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA) databases. The GEO datasets include <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE110907</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE62182</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE83527</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175462</a>. TCGA data from lung cancer cohorts, specifically TCGA-LUAD and TCGA-LUSC, were accessed through the TCGA data portal. Direct links to these datasets are provided in the Methods section. The small RNA sequencing data from the RUSH cohort have been deposited in the database of Genotypes and Phenotypes (dbGaP) under accession number phs004166.v1.p1. The data will be made available to authorized researchers upon controlled access approval. Supplementary Data files accompanying this study include detailed metadata of all analyzed samples (Data <a href="#MOESM4" class="usa-link">S1</a>), statistical filtering results for candidate tRNAs and tRFs (Data <a href="#MOESM5" class="usa-link">S2</a>, <a href="#MOESM6" class="usa-link">S3</a>), predicted target genes of selected tRFs (Data <a href="#MOESM7" class="usa-link">S4</a>), and results of downstream functional enrichment analyses, including KEGG and GO pathway analyses of the predicted target genes (Data <a href="#MOESM8" class="usa-link">S5</a>-<a href="#MOESM10" class="usa-link">7</a>). These supplementary materials provide comprehensive support for the analyses and conclusions presented in the manuscript.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All statistical analyses in this study were performed using publicly available software packages, with optimized parameter settings detailed in the Methods section. All analyses were conducted in accordance with the package’s guidelines and its open-source licensing terms. The primary code is available at <a href="https://github.com/F-UH/Primary-Code/tree/master" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/F-UH/Primary-Code/tree/master</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par80">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Lang Wu, Email: lwu@cc.hawaii.edu.</p>
<p>Yuanyuan Fu, Email: fuy@hawaii.edu.</p>
<p>Youping Deng, Email: dengy@hawaii.edu.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s43856-025-01068-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Siegel, R. L., Giaquinto, A. N. &amp; Jemal, A. Cancer statistics, 2024. <em>CA Cancer J. Clin.</em><strong>74</strong>, 12–49 (2024).
</cite> [<a href="https://doi.org/10.3322/caac.21820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38230766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siegel,%20R.%20L.,%20Giaquinto,%20A.%20N.%20&amp;%20Jemal,%20A.%20Cancer%20statistics,%202024.%20CA%20Cancer%20J.%20Clin.74,%2012%E2%80%9349%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>American Cancer Society. <em>Lung Cancer Survival Rates</em>, <a href="https://www.cancer.org/cancer/types/lung-cancer/about/what-is.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cancer.org/cancer/types/lung-cancer/about/what-is.html</a> (2024).</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Chen, Y., Zitello, E., Guo, R. &amp; Deng, Y. The function of LncRNAs and their role in the prediction, diagnosis, and prognosis of lung cancer. <em>Clin. Transl. Med</em><strong>11</strong>, e367 (2021).
</cite> [<a href="https://doi.org/10.1002/ctm2.367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8021541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33931980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.,%20Zitello,%20E.,%20Guo,%20R.%20&amp;%20Deng,%20Y.%20The%20function%20of%20LncRNAs%20and%20their%20role%20in%20the%20prediction,%20diagnosis,%20and%20prognosis%20of%20lung%20cancer.%20Clin.%20Transl.%20Med11,%20e367%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Zhang, Y., Shi, L., Simoff, M. J., J Wagner, O. &amp; Lavin, J. Biopsy frequency and complications among lung cancer patients in the United States. <em>Lung Cancer Manag</em><strong>9</strong>, LMT40 (2020).
</cite> [<a href="https://doi.org/10.2217/lmt-2020-0022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7729592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33318758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.,%20Shi,%20L.,%20Simoff,%20M.%20J.,%20J%20Wagner,%20O.%20&amp;%20Lavin,%20J.%20Biopsy%20frequency%20and%20complications%20among%20lung%20cancer%20patients%20in%20the%20United%20States.%20Lung%20Cancer%20Manag9,%20LMT40%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Dou, Y. et al. Plasma small ncRNA pair panels as novel biomarkers for early-stage lung adenocarcinoma screening. <em>BMC Genomics</em><strong>19</strong>, 545 (2018).
</cite> [<a href="https://doi.org/10.1186/s12864-018-4862-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6053820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30029594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dou,%20Y.%20et%20al.%20Plasma%20small%20ncRNA%20pair%20panels%20as%20novel%20biomarkers%20for%20early-stage%20lung%20adenocarcinoma%20screening.%20BMC%20Genomics19,%20545%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Fang, R. et al. The evaluation of serum biomarkers for non-small cell lung cancer (NSCLC) diagnosis. <em>Front Physiol.</em><strong>9</strong>, 1710 (2018).
</cite> [<a href="https://doi.org/10.3389/fphys.2018.01710" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6281717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30555348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20R.%20et%20al.%20The%20evaluation%20of%20serum%20biomarkers%20for%20non-small%20cell%20lung%20cancer%20(NSCLC)%20diagnosis.%20Front%20Physiol.9,%201710%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Li, W. et al. Liquid biopsy in lung cancer: Significance in diagnostics, prediction, and treatment monitoring. <em>Mol. Cancer</em><strong>21</strong>, 25 (2022).
</cite> [<a href="https://doi.org/10.1186/s12943-022-01505-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8772097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35057806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20W.%20et%20al.%20Liquid%20biopsy%20in%20lung%20cancer:%20Significance%20in%20diagnostics,%20prediction,%20and%20treatment%20monitoring.%20Mol.%20Cancer21,%2025%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Jóźwicka, T. M., Erdmańska, P. M., Stachowicz-Karpińska, A., Olkiewicz, M. &amp; Jóźwicki, W. Exosomes-promising carriers for regulatory therapy in oncology. <em>Cancers (Basel)</em><strong>16</strong>, 10.3390/cancers16050923 (2024).</cite> [<a href="https://doi.org/10.3390/cancers16050923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10931160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38473285/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Yu, Z. et al. Mitochondria-derived small RNAs as diagnostic biomarkers in lung cancer patients through a novel ratio-based expression analysis methodology. <em>Genes Dis.</em><strong>10</strong>, 1055–1061 (2023).
</cite> [<a href="https://doi.org/10.1016/j.gendis.2022.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10308114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37396544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20Z.%20et%20al.%20Mitochondria-derived%20small%20RNAs%20as%20diagnostic%20biomarkers%20in%20lung%20cancer%20patients%20through%20a%20novel%20ratio-based%20expression%20analysis%20methodology.%20Genes%20Dis.10,%201055%E2%80%931061%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Siriwardhana, C., Khadka, V. S., Chen, J. J. &amp; Deng, Y. Development of a miRNA-seq based prognostic signature in lung adenocarcinoma. <em>BMC Cancer</em><strong>19</strong>, 34 (2019).
</cite> [<a href="https://doi.org/10.1186/s12885-018-5206-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6325795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30621620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siriwardhana,%20C.,%20Khadka,%20V.%20S.,%20Chen,%20J.%20J.%20&amp;%20Deng,%20Y.%20Development%20of%20a%20miRNA-seq%20based%20prognostic%20signature%20in%20lung%20adenocarcinoma.%20BMC%20Cancer19,%2034%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Jin, X. et al. Evaluation of tumor-derived exosomal miRNA as potential diagnostic biomarkers for early-stage non-small cell lung cancer using next-generation sequencing. <em>Clin. Cancer Res</em><strong>23</strong>, 5311–5319 (2017).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-17-0577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28606918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20X.%20et%20al.%20Evaluation%20of%20tumor-derived%20exosomal%20miRNA%20as%20potential%20diagnostic%20biomarkers%20for%20early-stage%20non-small%20cell%20lung%20cancer%20using%20next-generation%20sequencing.%20Clin.%20Cancer%20Res23,%205311%E2%80%935319%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Li, J. et al. PIWI-interacting RNAs are aberrantly expressed and may serve as novel biomarkers for diagnosis of lung adenocarcinoma. <em>Thorac. Cancer</em><strong>12</strong>, 2468–2477 (2021).
</cite> [<a href="https://doi.org/10.1111/1759-7714.14094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8447905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34346164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.%20et%20al.%20PIWI-interacting%20RNAs%20are%20aberrantly%20expressed%20and%20may%20serve%20as%20novel%20biomarkers%20for%20diagnosis%20of%20lung%20adenocarcinoma.%20Thorac.%20Cancer12,%202468%E2%80%932477%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Zheng, B. et al. Plasma exosomal tRNA-derived fragments as diagnostic biomarkers in non-small cell lung cancer. <em>Front Oncol.</em><strong>12</strong>, 1037523 (2022).
</cite> [<a href="https://doi.org/10.3389/fonc.2022.1037523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9659620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36387119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20B.%20et%20al.%20Plasma%20exosomal%20tRNA-derived%20fragments%20as%20diagnostic%20biomarkers%20in%20non-small%20cell%20lung%20cancer.%20Front%20Oncol.12,%201037523%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Hernandez-Alias, X., Benisty, H., Schaefer, M. H. &amp; Serrano, L. Translational efficiency across healthy and tumor tissues is proliferation-related. <em>Mol. Syst. Biol.</em><strong>16</strong>, e9275 (2020).
</cite> [<a href="https://doi.org/10.15252/msb.20199275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7061310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32149479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hernandez-Alias,%20X.,%20Benisty,%20H.,%20Schaefer,%20M.%20H.%20&amp;%20Serrano,%20L.%20Translational%20efficiency%20across%20healthy%20and%20tumor%20tissues%20is%20proliferation-related.%20Mol.%20Syst.%20Biol.16,%20e9275%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Bian, M., Huang, S., Yu, D. &amp; Zhou, Z. tRNA metabolism and lung cancer: Beyond translation. <em>Front Mol. Biosci.</em><strong>8</strong>, 659388 (2021).
</cite> [<a href="https://doi.org/10.3389/fmolb.2021.659388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8516113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34660690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bian,%20M.,%20Huang,%20S.,%20Yu,%20D.%20&amp;%20Zhou,%20Z.%20tRNA%20metabolism%20and%20lung%20cancer:%20Beyond%20translation.%20Front%20Mol.%20Biosci.8,%20659388%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Morton, J. P., Kantidakis, T. &amp; White, R. J. RNA polymerase III transcription is repressed in response to the tumour suppressor ARF. <em>Nucleic Acids Res</em><strong>35</strong>, 3046–3052 (2007).
</cite> [<a href="https://doi.org/10.1093/nar/gkm208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1888803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17439968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morton,%20J.%20P.,%20Kantidakis,%20T.%20&amp;%20White,%20R.%20J.%20RNA%20polymerase%20III%20transcription%20is%20repressed%20in%20response%20to%20the%20tumour%20suppressor%20ARF.%20Nucleic%20Acids%20Res35,%203046%E2%80%933052%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Khattar, E. &amp; Tergaonkar, V. Transcriptional regulation of telomerase reverse transcriptase (TERT) by MYC. <em>Front Cell Dev. Biol.</em><strong>5</strong>, 1 (2017).
</cite> [<a href="https://doi.org/10.3389/fcell.2017.00001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5266692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28184371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khattar,%20E.%20&amp;%20Tergaonkar,%20V.%20Transcriptional%20regulation%20of%20telomerase%20reverse%20transcriptase%20(TERT)%20by%20MYC.%20Front%20Cell%20Dev.%20Biol.5,%201%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kuang, M. et al. tRNA-based prognostic score in predicting survival outcomes of lung adenocarcinomas. <em>Int J. Cancer</em><strong>145</strong>, 1982–1990 (2019).
</cite> [<a href="https://doi.org/10.1002/ijc.32250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30838640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuang,%20M.%20et%20al.%20tRNA-based%20prognostic%20score%20in%20predicting%20survival%20outcomes%20of%20lung%20adenocarcinomas.%20Int%20J.%20Cancer145,%201982%E2%80%931990%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Yu, N. et al. Identification of tumor suppressor miRNAs by integrative miRNA and mRNA sequencing of matched tumor-normal samples in lung adenocarcinoma. <em>Mol. Oncol.</em><strong>13</strong>, 1356–1368 (2019).
</cite> [<a href="https://doi.org/10.1002/1878-0261.12478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6547618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30913346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20N.%20et%20al.%20Identification%20of%20tumor%20suppressor%20miRNAs%20by%20integrative%20miRNA%20and%20mRNA%20sequencing%20of%20matched%20tumor-normal%20samples%20in%20lung%20adenocarcinoma.%20Mol.%20Oncol.13,%201356%E2%80%931368%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Vucic, E. A. et al. Smoking status impacts microRNA mediated prognosis and lung adenocarcinoma biology. <em>BMC Cancer</em><strong>14</strong>, 778 (2014).
</cite> [<a href="https://doi.org/10.1186/1471-2407-14-778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4216369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25342220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vucic,%20E.%20A.%20et%20al.%20Smoking%20status%20impacts%20microRNA%20mediated%20prognosis%20and%20lung%20adenocarcinoma%20biology.%20BMC%20Cancer14,%20778%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Becker-Santos, D. D. et al. Developmental transcription factor NFIB is a putative target of oncofetal miRNAs and is associated with tumour aggressiveness in lung adenocarcinoma. <em>J. Pathol.</em><strong>240</strong>, 161–172 (2016).
</cite> [<a href="https://doi.org/10.1002/path.4765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27357447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becker-Santos,%20D.%20D.%20et%20al.%20Developmental%20transcription%20factor%20NFIB%20is%20a%20putative%20target%20of%20oncofetal%20miRNAs%20and%20is%20associated%20with%20tumour%20aggressiveness%20in%20lung%20adenocarcinoma.%20J.%20Pathol.240,%20161%E2%80%93172%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Cohn, D. E. et al. Reactivation of multiple fetal miRNAs in lung adenocarcinoma. <em>Cancers (Basel)</em><strong>13</strong>, 10.3390/cancers13112686 (2021).</cite> [<a href="https://doi.org/10.3390/cancers13112686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8199175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34072436/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Nakatsu, K., Jijiwa, M., Khadka, V., Nasu, M. &amp; Deng, Y. sRNAfrag: A pipeline and suite of tools to analyze fragmentation in small RNA sequencing data. <em>Brief Bioinform</em><strong>25</strong>, 10.1093/bib/bbad515 (2023).</cite> [<a href="https://doi.org/10.1093/bib/bbad515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10796253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38243693/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Haynes, W. A. et al. Empowering multi-cohort gene expression analysis to increase reproducibility. <em>Pac. Symp. Biocomput</em><strong>22</strong>, 144–153 (2017).
</cite> [<a href="https://doi.org/10.1142/9789813207813_0015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5167529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27896970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haynes,%20W.%20A.%20et%20al.%20Empowering%20multi-cohort%20gene%20expression%20analysis%20to%20increase%20reproducibility.%20Pac.%20Symp.%20Biocomput22,%20144%E2%80%93153%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Gao, Z. et al. Comprehensive landscape of tRNA-derived fragments in lung cancer. <em>Mol. Ther. Oncolytics</em><strong>26</strong>, 207–225 (2022).
</cite> [<a href="https://doi.org/10.1016/j.omto.2022.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9307607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35892120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20Z.%20et%20al.%20Comprehensive%20landscape%20of%20tRNA-derived%20fragments%20in%20lung%20cancer.%20Mol.%20Ther.%20Oncolytics26,%20207%E2%80%93225%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Pinho, S. S. &amp; Reis, C. A. Glycosylation in cancer: Mechanisms and clinical implications. <em>Nat. Rev. Cancer</em><strong>15</strong>, 540–555 (2015).
</cite> [<a href="https://doi.org/10.1038/nrc3982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26289314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinho,%20S.%20S.%20&amp;%20Reis,%20C.%20A.%20Glycosylation%20in%20cancer:%20Mechanisms%20and%20clinical%20implications.%20Nat.%20Rev.%20Cancer15,%20540%E2%80%93555%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Marzio, A. et al. EMSY inhibits homologous recombination repair and the interferon response, promoting lung cancer immune evasion. <em>Cell</em><strong>185</strong>, 10.1016/j.cell.2021.12.005 (2022).</cite> [<a href="https://doi.org/10.1016/j.cell.2021.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8751279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34963055/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Jin, B. et al. MEN1 is a regulator of alternative splicing and prevents R-loop-induced genome instability through suppression of RNA polymerase II elongation. <em>Nucleic Acids Res.</em><strong>51</strong>, 7951–7971 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10450199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37395406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20B.%20et%20al.%20MEN1%20is%20a%20regulator%20of%20alternative%20splicing%20and%20prevents%20R-loop-induced%20genome%20instability%20through%20suppression%20of%20RNA%20polymerase%20II%20elongation.%20Nucleic%20Acids%20Res.51,%207951%E2%80%937971%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kim, A. et al. Notch1 destabilizes the adherens junction complex through upregulation of the Snail family of E-cadherin repressors in non-small cell lung cancer. <em>Oncol. Rep.</em><strong>30</strong>, 1423–1429 (2013).
</cite> [<a href="https://doi.org/10.3892/or.2013.2565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23807483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20A.%20et%20al.%20Notch1%20destabilizes%20the%20adherens%20junction%20complex%20through%20upregulation%20of%20the%20Snail%20family%20of%20E-cadherin%20repressors%20in%20non-small%20cell%20lung%20cancer.%20Oncol.%20Rep.30,%201423%E2%80%931429%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Liang, Y. et al. Stage-associated differences in the serum N- and O-glycan profiles of patients with non-small cell lung cancer. <em>Clin. Proteom.</em><strong>16</strong>, 20 (2019).</cite> [<a href="https://doi.org/10.1186/s12014-019-9240-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6509814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31168300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang,%20Y.%20et%20al.%20Stage-associated%20differences%20in%20the%20serum%20N-%20and%20O-glycan%20profiles%20of%20patients%20with%20non-small%20cell%20lung%20cancer.%20Clin.%20Proteom.16,%2020%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>DeBerardinis, R. J. &amp; Chandel, N. S. Fundamentals of cancer metabolism. <em>Sci. Adv.</em><strong>2</strong>, e1600200 (2016).
</cite> [<a href="https://doi.org/10.1126/sciadv.1600200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4928883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27386546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?DeBerardinis,%20R.%20J.%20&amp;%20Chandel,%20N.%20S.%20Fundamentals%20of%20cancer%20metabolism.%20Sci.%20Adv.2,%20e1600200%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Cruz-Bermúdez, A. et al. Cancer-associated fibroblasts modify lung cancer metabolism involving ROS and TGF-β signaling. <em>Free Radic. Biol. Med</em>. <strong>130</strong>, 163–173 (2019).
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2018.10.450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30391585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cruz-Berm%C3%BAdez,%20A.%20et%20al.%20Cancer-associated%20fibroblasts%20modify%20lung%20cancer%20metabolism%20involving%20ROS%20and%20TGF-%CE%B2%20signaling.%20Free%20Radic.%20Biol.%20Med.%20130,%20163%E2%80%93173%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Orellana, E. A., Siegal, E. &amp; Gregory, R. I. tRNA dysregulation and disease. <em>Nat. Rev. Genet</em><strong>23</strong>, 651–664 (2022).
</cite> [<a href="https://doi.org/10.1038/s41576-022-00501-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11170316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35681060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orellana,%20E.%20A.,%20Siegal,%20E.%20&amp;%20Gregory,%20R.%20I.%20tRNA%20dysregulation%20and%20disease.%20Nat.%20Rev.%20Genet23,%20651%E2%80%93664%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Pan, T. Modifications and functional genomics of human transfer RNA. <em>Cell Res</em><strong>28</strong>, 395–404 (2018).
</cite> [<a href="https://doi.org/10.1038/s41422-018-0013-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5939049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29463900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20T.%20Modifications%20and%20functional%20genomics%20of%20human%20transfer%20RNA.%20Cell%20Res28,%20395%E2%80%93404%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Lee, J.-O., Lee, M. &amp; Chung, Y.-J. DBtRend: A web-server of tRNA expression profiles from small RNA sequencing data in humans. <em>Genes (Basel)</em><strong>12</strong>, 10.3390/genes12101576 (2021).</cite> [<a href="https://doi.org/10.3390/genes12101576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8535304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34680971/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Meng, X., Pan, J., Sun, S. &amp; Gong, Z. Circulating exosomes and their cargos in blood as novel biomarkers for cancer. <em>Transl. Cancer Res.</em><strong>7</strong>, S226–S242 (2018).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Kalluri, R. The biology and function of exosomes in cancer. <em>J. Clin. Invest</em><strong>126</strong>, 1208–1215 (2016).
</cite> [<a href="https://doi.org/10.1172/JCI81135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4811149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27035812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalluri,%20R.%20The%20biology%20and%20function%20of%20exosomes%20in%20cancer.%20J.%20Clin.%20Invest126,%201208%E2%80%931215%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Wild, C. M. et al. Prognostic relevance of tumor-infiltrating immune cells in cervix squamous cell carcinoma. <em>Cancers (Basel)</em><strong>15</strong>, 10.3390/cancers15204952 (2023).</cite> [<a href="https://doi.org/10.3390/cancers15204952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10605287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37894319/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Islam, M. R. et al. Demographic differentials of lung cancer survival in Bangladeshi patients. <em>PLoS One</em><strong>16</strong>, e0261238 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0261238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8664208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34890415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Islam,%20M.%20R.%20et%20al.%20Demographic%20differentials%20of%20lung%20cancer%20survival%20in%20Bangladeshi%20patients.%20PLoS%20One16,%20e0261238%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Asamura, H. et al. A Japanese lung cancer registry study: Prognosis of 13,010 resected lung cancers. <em>J. Thorac. Oncol.</em><strong>3</strong>, 46–52 (2008).
</cite> [<a href="https://doi.org/10.1097/JTO.0b013e31815e8577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18166840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asamura,%20H.%20et%20al.%20A%20Japanese%20lung%20cancer%20registry%20study:%20Prognosis%20of%2013,010%20resected%20lung%20cancers.%20J.%20Thorac.%20Oncol.3,%2046%E2%80%9352%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Foeglé, J. et al. Specific features of non-small cell lung cancer in women: a retrospective study of 1738 cases diagnosed in Bas-Rhin between 1982 and 1997. <em>J. Thorac. Oncol.</em><strong>2</strong>, 466–474 (2007).
</cite> [<a href="https://doi.org/10.1097/01.JTO.0000275340.39960.25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17545840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Foegl%C3%A9,%20J.%20et%20al.%20Specific%20features%20of%20non-small%20cell%20lung%20cancer%20in%20women:%20a%20retrospective%20study%20of%201738%20cases%20diagnosed%20in%20Bas-Rhin%20between%201982%20and%201997.%20J.%20Thorac.%20Oncol.2,%20466%E2%80%93474%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Chang, J. W., Asamura, H., Kawachi, R. &amp; Watanabe, S. -i Gender difference in survival of resected non-small cell lung cancer: Histology-related phenomenon? <em>J. Thorac. Cardiovasc Surg.</em><strong>137</strong>, 807–812 (2009).
</cite> [<a href="https://doi.org/10.1016/j.jtcvs.2008.09.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19327500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang,%20J.%20W.,%20Asamura,%20H.,%20Kawachi,%20R.%20&amp;%20Watanabe,%20S.%20-i%20Gender%20difference%20in%20survival%20of%20resected%20non-small%20cell%20lung%20cancer:%20Histology-related%20phenomenon?%20J.%20Thorac.%20Cardiovasc%20Surg.137,%20807%E2%80%93812%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Anusewicz, D., Orzechowska, M. &amp; Bednarek, A. K. Lung squamous cell carcinoma and lung adenocarcinoma differential gene expression regulation through pathways of Notch, Hedgehog, Wnt, and ErbB signalling. <em>Sci. Rep.</em><strong>10</strong>, 21128 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-77284-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7713208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33273537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anusewicz,%20D.,%20Orzechowska,%20M.%20&amp;%20Bednarek,%20A.%20K.%20Lung%20squamous%20cell%20carcinoma%20and%20lung%20adenocarcinoma%20differential%20gene%20expression%20regulation%20through%20pathways%20of%20Notch,%20Hedgehog,%20Wnt,%20and%20ErbB%20signalling.%20Sci.%20Rep.10,%2021128%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Torres, A. G., Reina, O., Stephan-Otto Attolini, C. &amp; Ribas de Pouplana, L. Differential expression of human tRNA genes drives the abundance of tRNA-derived fragments. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 8451–8456 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1821120116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6486751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30962382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torres,%20A.%20G.,%20Reina,%20O.,%20Stephan-Otto%20Attolini,%20C.%20&amp;%20Ribas%20de%20Pouplana,%20L.%20Differential%20expression%20of%20human%20tRNA%20genes%20drives%20the%20abundance%20of%20tRNA-derived%20fragments.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%208451%E2%80%938456%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Yu, X. et al. tRNA-derived fragments: Mechanisms underlying their regulation of gene expression and potential applications as therapeutic targets in cancers and virus infections. <em>Theranostics</em><strong>11</strong>, 461–469 (2021).
</cite> [<a href="https://doi.org/10.7150/thno.51963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7681095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33391486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20X.%20et%20al.%20tRNA-derived%20fragments:%20Mechanisms%20underlying%20their%20regulation%20of%20gene%20expression%20and%20potential%20applications%20as%20therapeutic%20targets%20in%20cancers%20and%20virus%20infections.%20Theranostics11,%20461%E2%80%93469%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Wu, W. et al. Changes of tRNA-derived fragments by Alzheimer’s disease in cerebrospinal fluid and blood serum. <em>J. Alzheimers Dis.</em><strong>96</strong>, 1285–1304 (2023).
</cite> [<a href="https://doi.org/10.3233/JAD-230412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10832608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37980659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20W.%20et%20al.%20Changes%20of%20tRNA-derived%20fragments%20by%20Alzheimer%E2%80%99s%20disease%20in%20cerebrospinal%20fluid%20and%20blood%20serum.%20J.%20Alzheimers%20Dis.96,%201285%E2%80%931304%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Fu, M. et al. Emerging roles of tRNA-derived fragments in cancer. <em>Mol. Cancer</em><strong>22</strong>, 30 (2023).
</cite> [<a href="https://doi.org/10.1186/s12943-023-01739-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9926655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36782290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20M.%20et%20al.%20Emerging%20roles%20of%20tRNA-derived%20fragments%20in%20cancer.%20Mol.%20Cancer22,%2030%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Lee, Y. S., Shibata, Y., Malhotra, A. &amp; Dutta, A. A novel class of small RNAs: tRNA-derived RNA fragments (tRFs). <em>Genes Dev.</em><strong>23</strong>, 2639–2649 (2009).
</cite> [<a href="https://doi.org/10.1101/gad.1837609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2779758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19933153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20Y.%20S.,%20Shibata,%20Y.,%20Malhotra,%20A.%20&amp;%20Dutta,%20A.%20A%20novel%20class%20of%20small%20RNAs:%20tRNA-derived%20RNA%20fragments%20(tRFs).%20Genes%20Dev.23,%202639%E2%80%932649%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Cole, C. et al. Filtering of deep sequencing data reveals the existence of abundant Dicer-dependent small RNAs derived from tRNAs. <em>RNA</em><strong>15</strong>, 2147–2160 (2009).
</cite> [<a href="https://doi.org/10.1261/rna.1738409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2779667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19850906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cole,%20C.%20et%20al.%20Filtering%20of%20deep%20sequencing%20data%20reveals%20the%20existence%20of%20abundant%20Dicer-dependent%20small%20RNAs%20derived%20from%20tRNAs.%20RNA15,%202147%E2%80%932160%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Zhang, J. et al. Screening and potential role of tRFs and tiRNAs derived from tRNAs in the carcinogenesis and development of lung adenocarcinoma. <em>Oncol. Lett.</em><strong>22</strong>, 506 (2021).
</cite> [<a href="https://doi.org/10.3892/ol.2021.12767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8114470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33986867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.%20et%20al.%20Screening%20and%20potential%20role%20of%20tRFs%20and%20tiRNAs%20derived%20from%20tRNAs%20in%20the%20carcinogenesis%20and%20development%20of%20lung%20adenocarcinoma.%20Oncol.%20Lett.22,%20506%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Dhahbi, J. et al. Profiling of tRNA Halves and YRNA Fragments in Serum and Tissue From Oral Squamous Cell Carcinoma Patients Identify Key Role of 5’ tRNA-Val-CAC-2-1 Half. <em>Front Oncol.</em><strong>9</strong>, 959 (2019).
</cite> [<a href="https://doi.org/10.3389/fonc.2019.00959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6775249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31616639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dhahbi,%20J.%20et%20al.%20Profiling%20of%20tRNA%20Halves%20and%20YRNA%20Fragments%20in%20Serum%20and%20Tissue%20From%20Oral%20Squamous%20Cell%20Carcinoma%20Patients%20Identify%20Key%20Role%20of%205%E2%80%99%20tRNA-Val-CAC-2-1%20Half.%20Front%20Oncol.9,%20959%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Qin, C., Chen, Z.-H., Cao, R., Shi, M.-J. &amp; Tian, Y. Differential Expression Profiles and Bioinformatics Analysis of tRNA-Derived Small RNAs in Muscle-Invasive Bladder Cancer in a Chinese Population. <em>Genes (Basel)</em><strong>13</strong>, 10.3390/genes13040601 (2022).</cite> [<a href="https://doi.org/10.3390/genes13040601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9030102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35456407/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Xu, W. et al. tRF-Val-CAC-016 modulates the transduction of CACNA1d-mediated MAPK signaling pathways to suppress the proliferation of gastric carcinoma. <em>Cell Commun. Signal</em><strong>20</strong>, 68 (2022).
</cite> [<a href="https://doi.org/10.1186/s12964-022-00857-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9118711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35590368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20W.%20et%20al.%20tRF-Val-CAC-016%20modulates%20the%20transduction%20of%20CACNA1d-mediated%20MAPK%20signaling%20pathways%20to%20suppress%20the%20proliferation%20of%20gastric%20carcinoma.%20Cell%20Commun.%20Signal20,%2068%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Mo, D. et al. tRNA-derived fragment tRF-17-79MP9PP attenuates cell invasion and migration via THBS1/TGF-β1/Smad3 axis in breast cancer. <em>Front Oncol.</em><strong>11</strong>, 656078 (2021).
</cite> [<a href="https://doi.org/10.3389/fonc.2021.656078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8072113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33912465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mo,%20D.%20et%20al.%20tRNA-derived%20fragment%20tRF-17-79MP9PP%20attenuates%20cell%20invasion%20and%20migration%20via%20THBS1/TGF-%CE%B21/Smad3%20axis%20in%20breast%20cancer.%20Front%20Oncol.11,%20656078%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Thapa, M. &amp; Dallmann, G. Role of coenzymes in cancer metabolism. <em>Semin Cell Dev. Biol.</em><strong>98</strong>, 44–53 (2020).
</cite> [<a href="https://doi.org/10.1016/j.semcdb.2019.05.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31176736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thapa,%20M.%20&amp;%20Dallmann,%20G.%20Role%20of%20coenzymes%20in%20cancer%20metabolism.%20Semin%20Cell%20Dev.%20Biol.98,%2044%E2%80%9353%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Ju, H.-Q., Lin, J.-F., Tian, T., Xie, D. &amp; Xu, R.-H. NADPH homeostasis in cancer: Functions, mechanisms and therapeutic implications. <em>Signal Transduct. Target Ther.</em><strong>5</strong>, 231 (2020).
</cite> [<a href="https://doi.org/10.1038/s41392-020-00326-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7542157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33028807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ju,%20H.-Q.,%20Lin,%20J.-F.,%20Tian,%20T.,%20Xie,%20D.%20&amp;%20Xu,%20R.-H.%20NADPH%20homeostasis%20in%20cancer:%20Functions,%20mechanisms%20and%20therapeutic%20implications.%20Signal%20Transduct.%20Target%20Ther.5,%20231%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Alvarez, S. W. et al. NFS1 undergoes positive selection in lung tumours and protects cells from ferroptosis. <em>Nature</em><strong>551</strong>, 639–643 (2017).
</cite> [<a href="https://doi.org/10.1038/nature24637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5808442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29168506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alvarez,%20S.%20W.%20et%20al.%20NFS1%20undergoes%20positive%20selection%20in%20lung%20tumours%20and%20protects%20cells%20from%20ferroptosis.%20Nature551,%20639%E2%80%93643%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Flentie, K. et al. Nucleoside diphosphate kinase-3 (NME3) enhances TLR5-induced NFκB activation. <em>Mol. Cancer Res</em>. <strong>16</strong>, 986–999 (2018).
</cite> [<a href="https://doi.org/10.1158/1541-7786.MCR-17-0603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29523766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flentie,%20K.%20et%20al.%20Nucleoside%20diphosphate%20kinase-3%20(NME3)%20enhances%20TLR5-induced%20NF%CE%BAB%20activation.%20Mol.%20Cancer%20Res.%2016,%20986%E2%80%93999%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Yin, J. et al. Potential mechanisms connecting purine metabolism and cancer therapy. <em>Front Immunol.</em><strong>9</strong>, 1697 (2018).
</cite> [<a href="https://doi.org/10.3389/fimmu.2018.01697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6077182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30105018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin,%20J.%20et%20al.%20Potential%20mechanisms%20connecting%20purine%20metabolism%20and%20cancer%20therapy.%20Front%20Immunol.9,%201697%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Niu, N. et al. ATIC facilitates cell growth and migration by upregulating Myc expression in lung adenocarcinoma. <em>Oncol. Lett.</em><strong>23</strong>, 131 (2022).
</cite> [<a href="https://doi.org/10.3892/ol.2022.13251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8895470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35251351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Niu,%20N.%20et%20al.%20ATIC%20facilitates%20cell%20growth%20and%20migration%20by%20upregulating%20Myc%20expression%20in%20lung%20adenocarcinoma.%20Oncol.%20Lett.23,%20131%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Koppenol, W. H., Bounds, P. L. &amp; Dang, C. V. Otto Warburg’s contributions to current concepts of cancer metabolism. <em>Nat. Rev. Cancer</em><strong>11</strong>, 325–337 (2011).
</cite> [<a href="https://doi.org/10.1038/nrc3038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21508971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koppenol,%20W.%20H.,%20Bounds,%20P.%20L.%20&amp;%20Dang,%20C.%20V.%20Otto%20Warburg%E2%80%99s%20contributions%20to%20current%20concepts%20of%20cancer%20metabolism.%20Nat.%20Rev.%20Cancer11,%20325%E2%80%93337%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Kalainayakan, S. P., FitzGerald, K. E., Konduri, P. C., Vidal, C. &amp; Zhang, L. Essential roles of mitochondrial and heme function in lung cancer bioenergetics and tumorigenesis. <em>Cell Biosci.</em><strong>8</strong>, 56 (2018).
</cite> [<a href="https://doi.org/10.1186/s13578-018-0257-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6215344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30410721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalainayakan,%20S.%20P.,%20FitzGerald,%20K.%20E.,%20Konduri,%20P.%20C.,%20Vidal,%20C.%20&amp;%20Zhang,%20L.%20Essential%20roles%20of%20mitochondrial%20and%20heme%20function%20in%20lung%20cancer%20bioenergetics%20and%20tumorigenesis.%20Cell%20Biosci.8,%2056%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>López-Ríos, F. et al. Loss of the mitochondrial bioenergetic capacity underlies the glucose avidity of carcinomas. <em>Cancer Res</em><strong>67</strong>, 9013–9017 (2007).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-07-1678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17909002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez-R%C3%ADos,%20F.%20et%20al.%20Loss%20of%20the%20mitochondrial%20bioenergetic%20capacity%20underlies%20the%20glucose%20avidity%20of%20carcinomas.%20Cancer%20Res67,%209013%E2%80%939017%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Yu, X. et al. Comprehensive proteomic profiling of lung adenocarcinoma: development and validation of an innovative prognostic model. <em>Transl. Cancer Res.</em><strong>13</strong>, 2187–2207 (2024).
</cite> [<a href="https://doi.org/10.21037/tcr-23-1940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11170522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38881920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20X.%20et%20al.%20Comprehensive%20proteomic%20profiling%20of%20lung%20adenocarcinoma:%20development%20and%20validation%20of%20an%20innovative%20prognostic%20model.%20Transl.%20Cancer%20Res.13,%202187%E2%80%932207%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Cluntun, A. A., Lukey, M. J., Cerione, R. A. &amp; Locasale, J. W. Glutamine Metabolism in Cancer: Understanding the Heterogeneity. <em>Trends Cancer</em><strong>3</strong>, 169–180 (2017).
</cite> [<a href="https://doi.org/10.1016/j.trecan.2017.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5383348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28393116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cluntun,%20A.%20A.,%20Lukey,%20M.%20J.,%20Cerione,%20R.%20A.%20&amp;%20Locasale,%20J.%20W.%20Glutamine%20Metabolism%20in%20Cancer:%20Understanding%20the%20Heterogeneity.%20Trends%20Cancer3,%20169%E2%80%93180%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Chen, J., Cui, L., Lu, S. &amp; Xu, S. Amino acid metabolism in tumor biology and therapy. <em>Cell Death Dis.</em><strong>15</strong>, 42 (2024).
</cite> [<a href="https://doi.org/10.1038/s41419-024-06435-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10787762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38218942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20J.,%20Cui,%20L.,%20Lu,%20S.%20&amp;%20Xu,%20S.%20Amino%20acid%20metabolism%20in%20tumor%20biology%20and%20therapy.%20Cell%20Death%20Dis.15,%2042%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Zhou, X. et al. Activated amino acid response pathway generates apatinib resistance by reprograming glutamine metabolism in non-small-cell lung cancer. <em>Cell Death Dis.</em><strong>13</strong>, 636 (2022).
</cite> [<a href="https://doi.org/10.1038/s41419-022-05079-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9304404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35864117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20X.%20et%20al.%20Activated%20amino%20acid%20response%20pathway%20generates%20apatinib%20resistance%20by%20reprograming%20glutamine%20metabolism%20in%20non-small-cell%20lung%20cancer.%20Cell%20Death%20Dis.13,%20636%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Chen, G.-Y., Chao, H.-C., Liao, H.-W., Tsai, I. L. &amp; Kuo, C.-H. Rapid quantification of glutaminase 2 (GLS2)-related metabolites by HILIC-MS/MS. <em>Anal. Biochem</em><strong>539</strong>, 39–44 (2017).
</cite> [<a href="https://doi.org/10.1016/j.ab.2017.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28993139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20G.-Y.,%20Chao,%20H.-C.,%20Liao,%20H.-W.,%20Tsai,%20I.%20L.%20&amp;%20Kuo,%20C.-H.%20Rapid%20quantification%20of%20glutaminase%202%20(GLS2)-related%20metabolites%20by%20HILIC-MS/MS.%20Anal.%20Biochem539,%2039%E2%80%9344%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Cai, D.-J. et al. Asparagine synthetase regulates lung-cancer metastasis by stabilizing the β-catenin complex and modulating mitochondrial response. <em>Cell Death Dis.</em><strong>13</strong>, 566 (2022).
</cite> [<a href="https://doi.org/10.1038/s41419-022-05015-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9226154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35739087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cai,%20D.-J.%20et%20al.%20Asparagine%20synthetase%20regulates%20lung-cancer%20metastasis%20by%20stabilizing%20the%20%CE%B2-catenin%20complex%20and%20modulating%20mitochondrial%20response.%20Cell%20Death%20Dis.13,%20566%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Cozen, A. E. et al. ARM-seq: AlkB-facilitated RNA methylation sequencing reveals a complex landscape of modified tRNA fragments. <em>Nat. Methods</em><strong>12</strong>, 879–884 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4553111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26237225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cozen,%20A.%20E.%20et%20al.%20ARM-seq:%20AlkB-facilitated%20RNA%20methylation%20sequencing%20reveals%20a%20complex%20landscape%20of%20modified%20tRNA%20fragments.%20Nat.%20Methods12,%20879%E2%80%93884%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Torres, A. G. et al. Inosine modifications in human tRNAs are incorporated at the precursor tRNA level. <em>Nucleic Acids Res</em>. <strong>43</strong>, 5145–5157 (2015).
</cite> [<a href="https://doi.org/10.1093/nar/gkv277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4446420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25916855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torres,%20A.%20G.%20et%20al.%20Inosine%20modifications%20in%20human%20tRNAs%20are%20incorporated%20at%20the%20precursor%20tRNA%20level.%20Nucleic%20Acids%20Res.%2043,%205145%E2%80%935157%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Gogakos, T. et al. Characterizing expression and processing of precursor and mature human tRNAs by Hydro-tRNAseq and PAR-CLIP. <em>Cell Rep.</em><strong>20</strong>, 1463–1475 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.07.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5564215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28793268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gogakos,%20T.%20et%20al.%20Characterizing%20expression%20and%20processing%20of%20precursor%20and%20mature%20human%20tRNAs%20by%20Hydro-tRNAseq%20and%20PAR-CLIP.%20Cell%20Rep.20,%201463%E2%80%931475%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Zheng, G. et al. Efficient and quantitative high-throughput tRNA sequencing. <em>Nat. Methods</em><strong>12</strong>, 835–837 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4624326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26214130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20G.%20et%20al.%20Efficient%20and%20quantitative%20high-throughput%20tRNA%20sequencing.%20Nat.%20Methods12,%20835%E2%80%93837%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Hoffmann, A. et al. Accurate mapping of tRNA reads. <em>Bioinformatics</em><strong>34</strong>, 1116–1124 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx756" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29228294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoffmann,%20A.%20et%20al.%20Accurate%20mapping%20of%20tRNA%20reads.%20Bioinformatics34,%201116%E2%80%931124%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (478.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">43856_2025_1068_MOESM3_ESM.pdf</a><sup> (32.7KB, pdf) </sup><p>Description of Additional Supplementary files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (81.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (28.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (22.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (81KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (12.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (14.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (14.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370967/bin/43856_2025_1068_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (79.1KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Publicly available small RNA sequencing data and corresponding clinical information were obtained from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA) databases. The GEO datasets include <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE110907</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE62182</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE83527" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE83527</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175462</a>. TCGA data from lung cancer cohorts, specifically TCGA-LUAD and TCGA-LUSC, were accessed through the TCGA data portal. Direct links to these datasets are provided in the Methods section. The small RNA sequencing data from the RUSH cohort have been deposited in the database of Genotypes and Phenotypes (dbGaP) under accession number phs004166.v1.p1. The data will be made available to authorized researchers upon controlled access approval. Supplementary Data files accompanying this study include detailed metadata of all analyzed samples (Data <a href="#MOESM4" class="usa-link">S1</a>), statistical filtering results for candidate tRNAs and tRFs (Data <a href="#MOESM5" class="usa-link">S2</a>, <a href="#MOESM6" class="usa-link">S3</a>), predicted target genes of selected tRFs (Data <a href="#MOESM7" class="usa-link">S4</a>), and results of downstream functional enrichment analyses, including KEGG and GO pathway analyses of the predicted target genes (Data <a href="#MOESM8" class="usa-link">S5</a>-<a href="#MOESM10" class="usa-link">7</a>). These supplementary materials provide comprehensive support for the analyses and conclusions presented in the manuscript.</p>
<p>All statistical analyses in this study were performed using publicly available software packages, with optimized parameter settings detailed in the Methods section. All analyses were conducted in accordance with the package’s guidelines and its open-source licensing terms. The primary code is available at <a href="https://github.com/F-UH/Primary-Code/tree/master" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/F-UH/Primary-Code/tree/master</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Medicine are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s43856-025-01068-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/43856_2025_Article_1068.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370967/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370967/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370967%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370967/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370967/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370967/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841456/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370967/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841456/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370967/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370967/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="0350jM2eCVYVyB7mlXTJon9XrUdnocncpr3bFG5720S1xZyXYry2i5Akp9nfUJEC">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
