
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Comparative analysis of the mitogenomes of multiple species of Fagaceae, with special focus on Quercus gilva - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152550E78AF424730550E700185C9898.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcps">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363117/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Plant Biology">
<meta name="citation_title" content="Comparative analysis of the mitogenomes of multiple species of Fagaceae, with special focus on Quercus gilva">
<meta name="citation_author" content="Yu Li">
<meta name="citation_author_institution" content="Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China">
<meta name="citation_author" content="Si-Si Zheng">
<meta name="citation_author_institution" content="Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China">
<meta name="citation_author" content="Gregor Kozlowski">
<meta name="citation_author_institution" content="Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China">
<meta name="citation_author_institution" content="Department of Biology and Botanic Garden, University of Fribourg, Chemin du Musée 10, Fribourg, Fribourg, 1700 Switzerland">
<meta name="citation_author_institution" content="Natural History Museum Fribourg, Chemin du Musée 6, Fribourg, 1700 Switzerland">
<meta name="citation_author" content="Yi-Gang Song">
<meta name="citation_author_institution" content="Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1098">
<meta name="citation_doi" content="10.1186/s12870-025-07072-x">
<meta name="citation_pmid" content="40830429">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363117/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363117/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363117/pdf/12870_2025_Article_7072.pdf">
<meta name="description" content="Quercus, as the most abundant and widely distributed genus within the family Fagaceae, has been extensively studied at nuclear genome and plastome. However, mitogenome studies in Quercus remain scarce. In this study, we assemble and annotate the ...">
<meta name="og:title" content="Comparative analysis of the mitogenomes of multiple species of Fagaceae, with special focus on Quercus gilva">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Quercus, as the most abundant and widely distributed genus within the family Fagaceae, has been extensively studied at nuclear genome and plastome. However, mitogenome studies in Quercus remain scarce. In this study, we assemble and annotate the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363117/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12363117">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12870-025-07072-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12870_2025_Article_7072.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12363117%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12363117/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12363117/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363117/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcps.png" alt="BMC Plant Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Plant Biology" title="Link to BMC Plant Biology" shape="default" href="https://bmcplantbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Plant Biol</button></div>. 2025 Aug 19;25:1098. doi: <a href="https://doi.org/10.1186/s12870-025-07072-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12870-025-07072-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Plant%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Comparative analysis of the mitogenomes of multiple species of Fagaceae, with special focus on <em>Quercus gilva</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yu Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yu Li</span></h3>
<div class="p">
<sup>1</sup>Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20SS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Si-Si Zheng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Si-Si Zheng</span></h3>
<div class="p">
<sup>1</sup>Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20SS%22%5BAuthor%5D" class="usa-link"><span class="name western">Si-Si Zheng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kozlowski%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Gregor Kozlowski</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Gregor Kozlowski</span></h3>
<div class="p">
<sup>1</sup>Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China </div>
<div class="p">
<sup>2</sup>Department of Biology and Botanic Garden, University of Fribourg, Chemin du Musée 10, Fribourg, Fribourg, 1700 Switzerland </div>
<div class="p">
<sup>3</sup>Natural History Museum Fribourg, Chemin du Musée 6, Fribourg, 1700 Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kozlowski%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gregor Kozlowski</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20YG%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yi-Gang Song</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yi-Gang Song</span></h3>
<div class="p">
<sup>1</sup>Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20YG%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi-Gang Song</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Eastern China Conservation Centre for Wild Endangered Plant Resources, Shanghai Chenshan Botanical Garden, Shanghai, 201602 China </div>
<div id="Aff2">
<sup>2</sup>Department of Biology and Botanic Garden, University of Fribourg, Chemin du Musée 10, Fribourg, Fribourg, 1700 Switzerland </div>
<div id="Aff3">
<sup>3</sup>Natural History Museum Fribourg, Chemin du Musée 6, Fribourg, 1700 Switzerland </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 16; Accepted 2025 Jul 15; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12363117  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830429/" class="usa-link">40830429</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1"><em>Quercus</em>, as the most abundant and widely distributed genus within the family Fagaceae, has been extensively studied at nuclear genome and plastome. However, mitogenome studies in <em>Quercus</em> remain scarce. In this study, we assemble and annotate the mitogenome of <em>Quercus gilva</em> based on Illumina and Nanopore data. Additionally, we explore the structural features of its mitogenome and provide comprehensive analyses of the phylogeny and evolution of Fagaceae.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">The <em>Q. gilva</em> mitogenome consists of four molecules (three circular molecules and one linear molecule) with 490,015 bp in total length and 45.68% in guanine-cytosine (GC) content. The mitogenome encodes 59 genes, including 37 protein-coding genes (PCGs), 19 transfer RNA genes (tRNAs), and three ribosomal RNA genes (rRNAs). We also examine the repeat sequences, codon usage bias, RNA editing sites, and endosymbiotic gene transfer in the mitogenome. The wide distribution of repeat sequences is a key factor in mitogenome rearrangement. These is widespread gene transfer among the mitogenome, plastome, and nuclear genome of <em>Q. gilva</em>. Comparative genomic analyses of the 11 Fagaceae mitogenomes reveal significant structural variations in size and gene loss. Synteny analysis further indicates extensive genome rearrangements and inversions within the 11 mitogenomes. However, analyses of nucleotide diversity (Pi) and nonsynonymous and synonymous substitution ratio (Ka/Ks) values reveal a low rate of evolution in the mitogenomes of Fagaceae. Finally, phylogenetic analysis based on 12 conserved mitochondrial PCGs of 40 taxa strongly supports the classification of Fabids.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">In this study, the mitogenome of <em>Q. gilva</em> is newly assembled, providing important genomic resources for the phylogeny, resource conservation and development of <em>Quercus</em>. At the same time, the study of structural variation among the mitogenomes of Fagaceae species also help to elucidate the formation mechanism of mitogenome structural diversity.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12870-025-07072-x.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>Quercus</em>, Mitogenome, RNA editing site, Endosymbiotic gene transfer</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par26">Nuclear genome (nDNA), mitochondrial genome (mitogenome, mtDNA), and plastid genome (plastome, cpDNA) are the three independently inherited genomes in higher plants, and they exhibit different structure, function, and molecular evolutionary rate [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The evolutionary trajectories of these genomes have long fascinated many evolutionary biologists [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Unlike the biparentally inherited nuclear genome, organelle genomes (mitogenome and plastome) are uniparentally inherited with endosymbiotic origin [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Over the course of evolution, the gene content of organelle genomes has been significantly reduced, largely due to gene loss or transfer to nuclear genome [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Recent advances in sequencing technologies have opened new avenues for exploring the evolution of plant organelle genomes.</p>
<p id="Par27">Normally, plastomes range from 120 kb to 170 kb in size and are highly conserved with stable circular and quadripartite structure that contains the core genes for photosynthesis [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. In contrast, plant mitogenomes, with the core genes for respiration, exhibit extreme variability in both size (ranging from 66 kb to 12 Mb) and structural topology (master circular molecules, subgenomic circular molecules, linear molecules, and branched multigenomic structures), even among closely related species [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. This variability is further complicated by the presence of highly variable intergenic spacer regions (IGS) and conserved coding sequences (CDSs) [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Notably, plant mitogenomes present an intriguing evolutionary paradox: while they have a remarkably low mutation rate, they exhibit a high rate of structural rearrangements, such as gene duplications, inversions, and translocations [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Endosymbiotic gene transfer (EGT) is a frequent phenomenon among the mitogenome, plastome, and nuclear genome in plants [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Evidence suggests that the gene transfer from plastome to mitogenome occurs in a unidirectional manner, whereas the gene transfer between nuclear genome and mitogenome is bidirectional [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p>
<p id="Par28">Due to these complex properties, the assembly and study of mitogenomes face significant challenges. To date, over 13,000 plastomes have been fully assembled, while fewer than 1,000 plant mitogenomes have been completed [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. To better understand the unique evolutionary features of plant mitogenomes (such as their mutation rates, structural variations, and gene transfer events) and their contributions to plant phylogeny, further studies on a larger number of mitogenomes are essential in the future.</p>
<p id="Par29"><em>Quercus</em>, with approximately 450 species, is divided into two subgenera and eight Sect. [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Due to its high ecological adaptability, <em>Quercus</em> has become a widespread taxon in the Northern Hemisphere [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Recent developments in high-throughput sequencing have greatly facilitated the assembly of <em>Quercus</em> genomes. Numerous studies, focusing on the evolution of nuclear genomes and plastomes (including structure variation, population genomics, introgression, phylogenomics, and so on), have been published in the last decade [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Compared with the plentiful number of nuclear genomes (16 individuals) and plastomes (120 individuals) available for <em>Quercus</em>, only five mitogenomes, belonging to two sections, have been sequenced (<em>Quercus acutissima</em>, <em>Q. cerris</em>, and <em>Q. variabilis</em> in section <em>Cerris</em>; and <em>Q. petraea</em> and <em>Q. robur</em> in section <em>Quercus</em>) (Accessed on November 24, 2024) [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. It will be interesting to explore the structural variation, mutation rate, and phylogenetic relationship of <em>Quercus</em> based on mitogenomes from different sections.</p>
<p id="Par30"><em>Quercus gilva</em>, which belongs to subgenus <em>Cerris</em> section <em>Cyclobalanopsis</em>, is an endangered species with very low genetic diversity in the subtropical evergreen broad-leaved forests of East Asia [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. As an important local quality timber species, its population size and number have declined dramatically in recent decades [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Recently, the nuclear genome and plastome of <em>Q. gilva</em> have been reported, providing an opportunity to compare their mutation rate and explore their gene transfer events [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>–<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p>
<p id="Par31">In this study, we successfully assembled the <em>Q. gilva</em> mitogenome using Illumina short reads and Nanopore long reads, providing the first mitogenome reference for <em>Quercus</em> section <em>Cyclobalanopsis</em>. We examined the structural features of the <em>Q. gilva</em> mitogenome, evaluated endosymbiotic gene transfer events among the plastome, mitogenome, and nuclear genome of <em>Q. gilva</em>, and explored the phylogeny and evolution of <em>Q. gilva</em> and related species within the family Fagaceae. The purpose of this study is to provide valuable genomic resources that will support future research on <em>Quercus</em>, particularly in the areas of species identification, conservation genetics, and evolutionary biology. Furthermore, our findings lay a theoretical foundation for understanding the role of the mitogenomes in the adaptation and speciation of <em>Quercus</em> species, offering new perspectives on their evolutionary trajectories.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plant material, DNA extraction, and sequencing</h3>
<p id="Par32">Fresh and healthy plant leaves of <em>Quercus gilva</em> were sampled from Fujian Province, China (29.94°N, 116.87°E). The voucher specimen was identified by one of the authors (Yi-Gang Song), and deposited in the Chenshan Herbarium (CSH) at Shanghai Chenshan Botanical Garden. High-quality genomic DNA was extracted from fresh leaves using the modified cetyltrimethylammonium bromide (CTAB) protocol [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. DNA quality and concentration were tested by 0.75% agarose gel electrophoresis analysis, NanoDrop One spectrophotometer (Thermo Fisher Scientific), and Qubit 2.0 Fluorometer (Life Technologies, Carlsbad, CA, USA). The genomic DNA was then randomly fragmented using the Covaris ultrasonic disruptor.</p>
<p id="Par33">Paired-end (PE) sequencing libraries with 350 bp insert fragments were performed using the Nextera DNA Flex Library Prep Kit (Illumina, San Diego, CA, USA). The products were then subject to sequencing (each end 150 bp PE reads) on the Illumina HiSeq XTen platform. For Oxford Nanopore sequencing, the sequencing libraries were prepared using the Ligation Sequencing Kit SQK-LSK109 (Oxford Nanopore Technologies, Cambridge, the UK). The purified library was loaded onto primed R9.4 Spot-On Flow Cells for sequencing using the Oxford Nanopore PromethION sequencer (Oxford Nanopore Technologies, Oxford, UK) at Wuhan Benagen Tech Solutions Company Limited, Wuhan, China.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Genome assembly and annotation</h3>
<p id="Par34">In this study, we <em>de novo</em> assembled and annotated organelle genomes of <em>Q. gilva</em>. For the plastome, we completely assembled it by employing the “get_organelle_from_reads.py” command of the GetOrganelle v1.7.6.1 software, with the parameter “-F embplant_pt” [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Moreover, for the mitogenome, we tested three different assembly strategies: GetOrganelle v1.7.6.1, Unicycler v0.5.1, and GSAT v1.10 [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. A rough mitogenome was initially obtained with the other parameter of “-F embplant_mt” by the GetOrganelle v1.7.6.1 software [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Unicycler was also used to perform hybrid assembly [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. We then further assembled the mitogenome by the GSAT v1.10 software based on the rough mitogenome and Nanopore long reads [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. After comparing the assembly completeness and structural consistency of the results from all three approaches, we selected the GSAT-assembled mitogenome for downstream analysis due to its higher accuracy and completeness (Fig. S1 and S2A).</p>
<p id="Par35">Four different assembled graphs, namely OG (organelle graph), MRG (mitochondrial rough graph), MRMG (mitochondrial rough master graph), and MMG (mitochondrial master graph), were obtained through the four steps of graphShort, graphLong, graphSimplification, and graphCorrection respectively. Finally, the final mitogenome was obtained by manually checking and correcting the assembly structure with the Bandage software [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Minimap2 was then used to map the raw high-throughput sequencing data, including both Illumina and Nanopore reads, back to the assembled mitogenome to evaluate the coverage distribution [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>].</p>
<p id="Par36">The assembled organelle genomes were annotated using the online annotation tool GeSeq (<a href="https://chlorobox.mpimp-golm.mpg.de/geseq.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://chlorobox.mpimp-golm.mpg.de/geseq.html</a>) [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. The annotation results were manually reviewed and corrected using the Geneious R9.0.2 software [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. The plant mitogenomes typically encode 24 core genes (including nine NADH dehydrogenase genes, five ATP synthase genes, four ubiquinol cytochrome c reductase genes, three cytochrome c oxidase genes, one cytochrome c biogenesis gene, one maturase gene, and one transport membrane protein gene) and 17 variable genes (including four large subunit of ribosome genes, eleven small subunit of ribosome genes, and two succinate dehydrogenase genes) [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. The open reading frames (ORFs) in the mitogenome were annotated using the gene prediction tool ORF Finder (<a href="https://www.ncbi.nlm.nih.gov/orffinder/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/orffinder/</a>) [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Finally, the genomic structures of the organelle genomes were plotted using the organelle genome drawing program OGDRAW (<a href="https://chlorobox.mpimp-golm.mpg.de/OGDraw.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://chlorobox.mpimp-golm.mpg.de/OGDraw.html</a>) [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. The organelle genome sequences and their annotations were submitted to GenBank with accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PQ862789:PQ862792%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ862789–PQ862792</a> for the mitogenome and <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ876385" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ876385</a> for the plastome, which are scheduled for release on July 24, 2025.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Repeat sequence identification</h3>
<p id="Par37">Based on the sequence length, repeat sequences can be divided into three types, namely simple sequence repeats (SSRs), tandem repeat sequences (TRSs), and dispersed repeat sequences (DRSs). In this study, we analyzed these repeat sequences in the mitogenome. SSRs were detected using the MISA Perl script (<a href="http://pgrc.ipk-gatersleben.de/misa/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://pgrc.ipk-gatersleben.de/misa/</a>) [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. The thresholds of repeat units were set to 10, 5, 4, 3, 3, and 3 for the mono-, di-, tri-, tetra-, penta-, and hexanucleotides, respectively. TRSs were predicted using the Tandem Repeats Finder (TRF) web service (<a href="https://tandem.bu.edu/trf/trf.advanced.submit.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://tandem.bu.edu/trf/trf.advanced.submit.html</a>) with the following parameters: 2, 7, and 7 for matches, mismatches, and indels, respectively, and 50 and 500 for the minimum alignment score and maximum period size, respectively [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Finally, DRSs, including forward (F), reverse (R), complementary (C), and palindromic (P) repeat sequences, were analyzed using the REPuter web service (<a href="https://bibiserv.cebitec.uni-bielefeld.de/reputer" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bibiserv.cebitec.uni-bielefeld.de/reputer</a>) [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. The hamming distance and minimal repeat size were set as 3 and 30 bp, respectively. The visualization of the repeat sequences was accomplished using the “Circlize” package in R.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Codon usage analysis and RNA editing site prediction</h3>
<p id="Par38">The protein-coding genes (PCGs) of the mitogenome were extracted using the Geneious R9.0.2 software [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Subsequently, twenty-eight PCG sequences, which had a length exceeding 300 bp and a start codon of ATG, were chosen for codon usage analysis by the CodonW v1.4.2 software [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. The relative synonymous codon usage (RSCU) values were calculated and drawn the stacked bar plot using the “ggplot2” package in R.</p>
<p id="Par39">The Deepred-Mt software was used to predict potential cytosine (C) to uracil (U) RNA editing sites in the mitogenome based on the convolutional neural network (CNN) model [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. The threshold of the probability was set to 0.9.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Endosymbiotic gene transferred fragments identification</h3>
<p id="Par40">To identify the mitochondrial plastid transferred fragments (MTPTs) and nuclear mitochondrial transferred fragments (NUMTs) of <em>Q. gilva</em>, we compared the mitogenome with the nuclear genome and plastome using the BLASTn v2.9.0 software [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. The plastome was assembled from this study. And the nuclear genome was obtained from the previous study of Wang et al., which has been deposited in the National Genomics Data Center (NGDC) (<a href="https://ngdc.cncb.ac.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn</a>) under the accession number GWHDOCW00000000, with a scheduled public release date of June 27, 2025 [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. The mitogenome was used for building database with makeblastdb, while the nuclear genome and plastome served as query sequences with the criteria of E-value ≤ 1e − 5, identity ≥ 80%, and alignment length ≥ 50 bp. Eventually, the endosymbiotic gene transferred fragments were visualized using the “Circlize” package in R.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Comparative analyses of the mitogenomes of Fagaceae</h3>
<p id="Par41">We downloaded all publicly available mitogenomes of ten Fagaceae species from the National Center of Biotechnology Information (NCBI) database and reannotated them to reduce potential analytical errors. We performed a gene presence-absence analysis across all species and visualized the results using a heatmap generated with the TBtools-II software [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. BLASTn was used for pairwise comparison of the mitogenomes of these 11 Fagaceae species [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Homologous sequences greater than 500 bp as conserved colinear blocks were then retained for collinearity analysis and plotting using the NGenomeSyn v1.41 software [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Not only that, we also downloaded the corresponding plastomes of these ten species. The shared PCGs of the organelle genomes (30 of mitogenomes and 79 of plastomes) across the 11 species were aligned using the MAFFT v7.487 software [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. These PCGs sequences were then used for nucleotide diversity analysis and Ka/Ks analysis. Nucleotide diversity (Pi) values of each PCG were calculated using the DnaSP v6.12.03 software with step size of 10 bp and window length of 100 bp [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Finally, the calculation of the nonsynonymous (Ka) and synonymous (Ks) substitution ratio (Ka/Ks) for each shared PCG was performed for pairwise species comparisons using the KaKs_Calculator v3.0 software [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>], employing the MLWL method [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. A gene was considered to be under positive selection if at least one pairwise comparison yielded a Ka/Ks value greater than 1.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Phylogenetic analysis</h3>
<p id="Par42">We downloaded the mitogenomes of 38 Fabid taxa from the NCBI database, including seven Fagales species, seven Cucurbitales species, eleven Rosales species, and thirteen Fabales species, and performed phylogenetic analyses with the new assembly mitogenome of <em>Q. gilva</em> (Table S1). The complete mitogenome of <em>Arabidopsis thaliana</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_037304" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_037304</a>) was used to root the tree as an outgroup. We extracted the CDSs of 12 shared PCGs (<em>atp4</em>, <em>atp8</em>, <em>nad2</em>, <em>nad4</em>, <em>nad6</em>, <em>nad7</em>, <em>nad9</em>, <em>rps3</em>, <em>matR</em>, <em>cox3</em>, <em>ccmB</em>, and <em>ccmC</em>) from 40 species using the PhyloSuite v1.2.3 software [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. The CDSs were aligned using the MAFFT v7.450 software with default parameters [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. The phylogenetic tree was constructed using the maximum-likelihood (ML) method implemented in the IQ-tree v2.1.3 software with 1,000 ultrafast bootstrap replicates [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. Finally, the phylogenetic tree was visualized and beautified in the FigTree v.1.4.4 software (<a href="http://tree.bio.ed.ac.uk/software/figtree/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://tree.bio.ed.ac.uk/software/figtree/</a>).</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Assembly of the mitogenome of <em>Q. gilva</em>
</h3>
<p id="Par43">Based on Illumina short reads and Nanopore long reads, we successfully assembled the master graphs of the mitogenome with the GSAT software. A total of 615.7, 482.4, and 481.1 kb sequences with 39, 27, and nine contigs and 52, 36, and 12 links were obtained from each step of OG, MRG, and MRMG, respectively (Table S2). The final MMG was 481,113 bp in length, with nine contigs and 12 links (Figure S2A and Table S2). All contigs exhibited high coverage depths, with short-read coverage ranging from 1257.14 X to 3015.83 X and long-read coverage ranging from 400.06 X to 1054.17 X (Table S3). This indicated that all contigs were strongly supported by both short and long reads, further confirming the high quality and accuracy of the mitogenome assembly. Notably, the long-read coverage depths of contig 1 (1054.17 X) and contig 3 (802.43 X) were approximately twice as high as those of the other contigs (Figure S2B and Table S3), suggesting that they likely represent large repeats with two copies.</p>
<p id="Par44">To evaluate the homologous recombination events between subgenomes mediated by these two large repeat sequences, we systematically analyzed the possible recombination pathways in Bandage by integrating the contig graph structure and sequencing read support. After evaluating all possible pathways in the MMG, it was finally found that the mitogenome showed a complex multi-branched structure through repeat-mediated recombination (Figure S2C). Specifically, molecule 1 consists of contig 8; molecule 2 consists of contig 1 (and its repeat copy), contigs 2, 5, and 6; molecule 3 contains contigs 3, 4, and 7; and molecule 4 includes the duplicated contig 3 and contig 9 (Table <a href="#Tab1" class="usa-link">1</a>). These rearrangements reflect the dynamic nature of the mitogenome structure, which likely exists as a mixture of alternative conformations in vivo.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Branched conformation of the <em>Quercus gilva</em> mitogenome</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Molecule</th>
<th align="left" colspan="1" rowspan="1">Path</th>
<th align="left" colspan="1" rowspan="1">Type</th>
<th align="left" colspan="1" rowspan="1">Length (bp)</th>
<th align="left" colspan="1" rowspan="1">GC content (%)</th>
<th align="left" colspan="1" rowspan="1">PCGs</th>
<th align="left" colspan="1" rowspan="1">tRNAs</th>
<th align="left" colspan="1" rowspan="1">rRNAs</th>
<th align="left" colspan="1" rowspan="1">ORFs</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">molecule 1</td>
<td align="left" colspan="1" rowspan="1">contig8</td>
<td align="left" colspan="1" rowspan="1">Circular</td>
<td align="left" colspan="1" rowspan="1">96,997</td>
<td align="left" colspan="1" rowspan="1">47.01</td>
<td align="left" colspan="1" rowspan="1">7(2)</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">43</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">molecule 2</td>
<td align="left" colspan="1" rowspan="1">contig2_1_5_1(copy)_6</td>
<td align="left" colspan="1" rowspan="1">linear</td>
<td align="left" colspan="1" rowspan="1">125,848</td>
<td align="left" colspan="1" rowspan="1">45.19</td>
<td align="left" colspan="1" rowspan="1">11(3)</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">51</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">molecule 3</td>
<td align="left" colspan="1" rowspan="1">contig7_3_4</td>
<td align="left" colspan="1" rowspan="1">Circular</td>
<td align="left" colspan="1" rowspan="1">133,477</td>
<td align="left" colspan="1" rowspan="1">45.98</td>
<td align="left" colspan="1" rowspan="1">6(3)</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">52</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">molecule 4</td>
<td align="left" colspan="1" rowspan="1">contig3(copy)_9</td>
<td align="left" colspan="1" rowspan="1">Circular</td>
<td align="left" colspan="1" rowspan="1">133,693</td>
<td align="left" colspan="1" rowspan="1">44.88</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">47</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Branched</td>
<td align="left" colspan="1" rowspan="1">490,015</td>
<td align="left" colspan="1" rowspan="1">45.68</td>
<td align="left" colspan="1" rowspan="1">37</td>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">193</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">General features of the mitogenome and plastome</h3>
<p id="Par45">The mitogenome consisted of four molecules (molecules 1–4) with 490,015 bp in total length and 45.68% in guanine-cytosine (GC) content. The lengths of circular molecule 1, linear molecule 2, circular molecule 3, and circular molecule 4 were 96,997, 125,848, 133,477, and 133,693 bp, respectively, and the GC contents were 47.01, 45.19, 45.98, and 44.88%, respectively (Figure S2C and Table <a href="#Tab1" class="usa-link">1</a>).</p>
<p id="Par46">The mitogenome annotated a total of 37 protein-coding genes (PCGs) (<em>atp9</em> were multi-copy), 19 transfer RNA genes (tRNAs) (<em>trnM-CAU</em> and <em>trnP-UGG</em> were multi-copy), and three ribosomal RNA genes (rRNAs) (Fig. <a href="#Fig1" class="usa-link">1</a> and Table <a href="#Tab2" class="usa-link">2</a>). Among the 36 unique PCGs, 24 PCGs were identified as core genes, and the other 12 PCGs were classified as variable genes. The 12 variable genes comprised four large subunits of ribosome genes, six small subunits of ribosome genes, and two succinate dehydrogenase gene. Seven genes were cut by introns: <em>rpl2</em> and <em>ccmFc</em> contained a single intron, <em>nad4</em> harbored three introns, while <em>nad1</em>, <em>nad2</em>, <em>nad5</em>, and <em>nad7</em> had four introns. Notably, three PCGs (<em>nad1</em>, <em>nad2</em>, and <em>nad5</em>) were identified as special trans-splicing genes. Detailed information on these genes was provided in Table S4. In addition, a total of 193 open reading frames (ORFs) larger than 300 bp were predicted by the ORF Finder in the <em>Q. gilva</em> mitogenome (Table S5).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363117_12870_2025_7072_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb06/12363117/e113094742a6/12870_2025_7072_Fig1_HTML.jpg" loading="lazy" id="MO1" height="1121" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome map of the <em>Quercus gilva</em> mitogenome. Genes outside the circle and on the line are transcribed in the clockwise direction, while genes inside the circle and below the line are transcribed in the counterclockwise direction. The genes belonging to different functional groups are identified by different colors</p></figcaption></figure><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Gene composition of the <em>Q. gilva</em> mitogenome</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Group of genes</th>
<th align="left" colspan="1" rowspan="1">Number</th>
<th align="left" colspan="1" rowspan="1">Gene name</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">ATP synthase</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">atp1, atp4, atp6, atp8, atp9 (×2)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ubiquinol cytochrome c reductase</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">ccmB, ccmC, ccmFc*, ccmFn</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cytochrome c biogenesis</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">cob</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cytochrome c oxidase</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">cox1, cox2, cox3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Maturases</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">matR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Transport membrane protein</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">mttB</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NADH dehydrogenase</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">nad1****, nad2****, nad3, nad4***, nad4L, nad5****, nad6, nad7****, nad9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Large subunit of ribosome (LSU)</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">rpl2*, rpl5, rpl10, rpl16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Small subunit of ribosome (SSU)</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">rps1, rps3, rps4, rps10*, rps12, rps19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Succinate dehydrogenase</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">sdh3, sdh4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal RNAs</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">rrn5, rrn18, rrn26</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Transfer RNAs</td>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">trnC-GCA, trnD-GUC, trnE-UUC, trnF-GAA, trnG-GCC, trnH-GUG, trnK-UUU, trnM-CAU (×3), trnN-GUU, trnP-UGG (×2), trnQ-UUG, trnS-GCU, trnS-UGA, trnV-GAC, trnW-CCA, trnY-GUA</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p29">
<p><em>Gene (×)</em> Number of copies of multi-copy genes, <em>Bold</em> trans-splicing genes that fragment across multiple branches</p>
<p>*one intron</p>
<p>***three introns</p>
<p>****four introns</p>
</div></div></section><p id="Par47">The plastome assembled in this study was 160,757 bp in length, with a typical circular tetrad structure consisting of a large single copy (LSC) region (90,217 bp) and a small single copy (SSC) region (18,870 bp), separated by a pair of inverted repeats (IR) regions (25,835 bp). A total of 86 PCGs, 37 tRNAs and eight rRNAs were annotated in the plastome (Figure S3).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Repeat sequences</h3>
<p id="Par48">The mitogenome contained a large number of repeat sequences, with their quantity and specific distribution shown in Fig. <a href="#Fig2" class="usa-link">2</a>C. Using the MISA Perl script, a total of 20, 40, 52, and 43 SSRs were identified in the four molecules, respectively (Fig. <a href="#Fig2" class="usa-link">2</a>A and Table S6). SSRs exhibited diverse types; however, hexanucleotide repeats were not detected in any of the four molecules (Fig. <a href="#Fig2" class="usa-link">2</a>A). The tetranucleotide repeats were the most abundant, accounting for 65 loci (40.37%), followed by mono- (40, 24.84%), di- (35, 21.74%), tri- (15, 9.32%), and pentanucleotide repeats (6, 3.73%). Most SSRs were composed of two complementary bases, adenine (A) and thymine (T). Additionally, two, five, four, and five TRSs were detected in the four molecules, respectively (Fig. <a href="#Fig2" class="usa-link">2</a>A and Table S7). The copy number of these TRSs was primarily between two and four, with the size of the basic repeat units ranging from 15 to 45 bp (Table S7). Among DRSs detected by the REPuter, no reverse or complementary repeats in the four molecules, while a total of 69 forward repeats and 50 palindromic repeats were identified respectively (Fig. <a href="#Fig2" class="usa-link">2</a>A and Table S8). The lengths of DRSs were predominantly between 30 and 60 bp (Fig. <a href="#Fig2" class="usa-link">2</a>B). Statistical analysis of the distribution of all repeats showed that most were located in the intergenic spacer regions (IGS), with only a small fraction found within the gene regions (Fig. <a href="#Fig2" class="usa-link">2</a>D).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363117_12870_2025_7072_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb06/12363117/c6a1b673cc47/12870_2025_7072_Fig2_HTML.jpg" loading="lazy" id="MO2" height="506" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The repeats of the <em>Q. gilva</em> mitogenome. (<strong>A</strong>) Total number of repeats across various types; (<strong>B</strong>) Distribution of dispersed repeat sequences by length categories (30–39 bp, 40–49 bp, 50–59 bp, and &gt; 60 bp); (<strong>C</strong>) Circos plot showing the distribution of all repeats. The outermost circle represents the <em>Q. gilva</em> mitogenome, while the layers from the outside to the inside illustrate the distribution of SSRs, TRSs, and DRSs, respectively; (<strong>D</strong>) Proportion of all repeats located in different genomic regions. SSRs, simple sequence repeats; TRSs, tandem repeat sequences; DRSs, dispersed repeat sequences; IGS, intergenic spacer regions</p></figcaption></figure></section><section id="Sec14"><h3 class="pmc_sec_title">Codon usage bias</h3>
<p id="Par49">The codon usage analysis of 28 selected PCGs revealed a universal codon usage preference in the mitogenome. Among the RSCU values calculated for the 64 codons, 30 codons had RSCU values greater than 1 (Table S9). Of these, only three codons (UCC, ACC, and UUG) ended in G/C, while the remaining 27 codons ended in A/U, indicating a strong preference for A/U ending codons (Figure S4 and Table S9). Histidine (His) exhibited the strongest bias, favoring the CAU codon (RSCU = 1.51), followed by Alanine (Ala), which preferred the GCU codon (RSCU = 1.50). Additionally, the most frequently used codons in the mitogenome were UUU (Phe), AUU (Ile), UUC (Phe), and GAA (Glu), each appearing more than 250 times.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Prediction of RNA editing sites</h3>
<p id="Par50">Based on the deep representation learning method (Deepred-Mt), we predicted 491 potential C to U RNA editing sites in the 36 PCGs of the <em>Q. gilva</em> mitogenome (Table S10). Among the PCGs, <em>nad4</em> had the highest number of predicted RNA editing sites, followed by <em>ccmB</em>, while no RNA editing sites were detected in <em>rpl2</em> (Fig. <a href="#Fig3" class="usa-link">3</a>A). Of the 491 RNA editing sites, 461 (93.90%) sites resulted in nonsynonymous mutation, while 30 (6.10%) sites were synonymous mutation. The nonsynonymous editing events caused 12 distinct types of amino acid changes. The most frequent mutation was the replacement of serine (Ser) and proline (Pro) with leucine (Leu), occurring 112 and 105 times, respectively (Fig. <a href="#Fig3" class="usa-link">3</a>B). Furthermore, 227 (46.23%) sites converted hydrophilic amino acids into hydrophobic ones, while 50 (10.18%) reversed this trend, changing hydrophobic amino acids to hydrophilic ones. Notably, three RNA editing sites at <em>ccmFc</em>−1315, <em>nad1</em>−308, and <em>rps10</em>−391 introduced stop codons (Arg→Ter) (Table S10).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363117_12870_2025_7072_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb06/12363117/1aabc036dc7c/12870_2025_7072_Fig3_HTML.jpg" loading="lazy" id="MO3" height="270" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Characteristics of RNA editing sites predicted in the 36 PCGs of the <em>Q. gilva</em> mitogenome. (<strong>A</strong>) The number of RNA editing sites predicted in the PCGs. (<strong>B</strong>) The distribution of amino acid changes resulting from RNA editing events</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Endosymbiotic gene transfer</h3>
<p id="Par51">A total of 11 MTPTs between the mitogenome and plastome of <em>Q. gilva</em> were identified by conducting sequence similarity calculations (Fig. <a href="#Fig4" class="usa-link">4</a> A and Table S11). The total length of the MTPTs was 6,512 bp, representing approximately 1.33% of the mitogenome and 4.05% of the plastome (Table S11). Among them, two MTPTs exceeded 1,000 bp in length, namely MTPT3 (1,322 bp) and MTPT7 (1,149 bp). Five MTPTs were observed to have two copies localized within the IR regions of the plastome, while the remaining MTPTs were located in the LSC region. Annotation results of the MTPTs revealed that these homologous fragments were derived from 14 plastid genes, including six complete genes (<em>trnD-GUC</em>, <em>trnI-CAU</em>, <em>trnV-GAC</em>, <em>trnH-GUG</em>, <em>trnN-GUU</em>, <em>and trnM-CAU</em>) and eight partial genes. Although all PCGs were pseudogenized, the tRNA genes retain functional roles in the mitogenome. Compared with the 11 MTPTs, we also found 1824 NUMTs between the mitogenome and nuclear genome (Fig. <a href="#Fig4" class="usa-link">4</a>B and Table S12). The matched lengths ranged from 50 to 133,698 bp (Table S12). Notably, extensive sequence transfer was observed on chromosome 5 in the nuclear genome, which encompassed nearly the entire mitogenome (Fig. <a href="#Fig4" class="usa-link">4</a>B). However, the origin of these fragments remained to be determined. Thus, these findings indicated a widespread gene transfer among the mitogenome, plastome, and nuclear genome of <em>Q. gilva</em>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363117_12870_2025_7072_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb06/12363117/ae3b81980aca/12870_2025_7072_Fig4_HTML.jpg" loading="lazy" id="MO4" height="335" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representations of MTPTs and NUMTs of <em>Q. gilva</em>. (<strong>A</strong>) Endosymbiotic gene transfer between the mitogenome and plastome. (<strong>B</strong>) Endosymbiotic gene transfer between the mitogenome and nuclear genome. The green, blue, and orange arcs represent the plastome, mitogenome, and nuclear genome, respectively. And the inner colored lines connecting the arcs indicate MTPTs and NUMTs, with different colors representing the level of the sequence identity. MTPT, mitochondrial plastid transferred fragments; NUMT, nuclear mitochondrial transferred fragments</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Comparison and collinearity of the mitogenomes in Fagaceae</h3>
<p id="Par52">The mitogenomes of eleven Fagaceae species, including five genera (<em>Quercus</em>, <em>Fagus</em>, <em>Castanea</em>, <em>Castanopsis</em>, and <em>Lithocarpus</em>), exhibited notable structure variation. Genome sizes ranged from 349,290 bp (<em>Q. petraea</em>) to 592,702 bp (<em>C. carlesii</em>), representing a significant disparity (Table S13). The length of <em>Q. gilva</em> mitogenome was the longest among the existing mitogenomes of <em>Quercus</em>. In contrast, GC content varied within a narrow range of 45.5% (<em>C. carlesii</em>) to 46.1% (<em>Q. petraea</em>), indicating low variability in base composition. Additionally, during the evolutionary process, the mitogenomes had also undergone the loss of PCGs. Our results showed that a total of 30 PCGs were shared among these 11 species (Figure S5). <em>Quercus petraea</em> exhibited the highest number of gene deletions, including two core genes. The number of large and small subunits of ribosome genes varied the most among all mitogenomes. More notably, <em>rps14</em> was unique to <em>Fagus sylvatica</em> and had been lost in all other species.</p>
<p id="Par53">A synteny analysis was also conducted for these 11 species for deeper comparative insights. The results revealed a significant number of homologous collinear blocks among these Fagaceae species, alongside substantial genome rearrangements and inversions (Fig. <a href="#Fig5" class="usa-link">5</a>). Notably, the largest collinear blocks were identified between <em>Q. petraea</em> and <em>Q. robur</em> within the same section, followed by <em>C. mollissima</em> and <em>C. henryi</em> within the same genus. This suggested that closely related species were more likely to retain larger syntenic blocks.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363117_12870_2025_7072_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb06/12363117/d03d8daa3998/12870_2025_7072_Fig5_HTML.jpg" loading="lazy" id="MO5" height="921" width="744" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Collinearity analysis of the mitogenomes of 11 Fagaceae species. The red arc area implies inversion, whereas the gray area represents high homology</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Pi values and Ka/Ks ratio of organelle genomes from Fagaceae</h3>
<p id="Par54">Pi values were calculated for the PCGs shared by the 11 organelle genomes of Fagaceae, respectively, including 30 PCGs of mitogenomes and 79 PCGs of plastomes (Fig. <a href="#Fig6" class="usa-link">6</a>A and Figure S6A). On average, the Pi values of chloroplast genes were higher than those of mitochondrial genes. Moreover, the difference of Pi values of chloroplast genes was greater than that of mitochondrial genes. The <em>rpl22</em> and <em>ycf1</em> in the plastome and <em>atp1</em>, <em>atp9</em>, and <em>cox2</em> in the mitogenome showed relatively high level of Pi values, reflecting their high sequence diversity within the family Fagaceae.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363117_12870_2025_7072_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb06/12363117/fa0c1bc4c0ba/12870_2025_7072_Fig6_HTML.jpg" loading="lazy" id="MO6" height="671" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The Pi values (<strong>A</strong>) and Ka/Ks ratio (<strong>B</strong>) of 30 common mitogenome PCGs among Fagaceae. The box plot represents the distribution of Ka/Ks values for all pairwise comparison in the data. Pi, nucleotide diversity; Ka/Ks, nonsynonymous and synonymous substitution ratio</p></figcaption></figure><p id="Par55">By calculating the Ka/Ks ratio of common PCGs, we compared the evolutionary rates in the organelle genomes of Fagaceae to assess the selection pressure of each PCG (Fig. <a href="#Fig6" class="usa-link">6</a>B and Figure S6B). In general, most PCGs had Ka/Ks values of less than 1 in the organelle genomes. However, there were ten chloroplast PCGs (<em>accD</em>, <em>ccmA</em>, <em>infA</em>, <em>matK</em>, <em>nadH</em>, <em>petA</em>, <em>rpl14</em>, <em>rpl22</em>, <em>rpoC2</em>, and <em>ycf2</em>) and seven mitochondrial PCGs (<em>atp6</em>, <em>atp9</em>, <em>ccmC</em>, <em>ccmFN</em>, <em>matR</em>, <em>nad1</em>, and <em>rps3</em>) in which at least one pairwise comparison had a Ka/Ks value greater than 1, indicating potential positive selection pressure for these PCGs (Table S14 and Table S15). These PCGs with high Pi and Ka/Ks values can be used as potential markers for phylogenetic studies of Fagaceae.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Phylogeny</h3>
<p id="Par56">In order to determine the phylogenetic position of <em>Q. gilva</em>, the phylogenetic tree was reconstructed after sequence alignment based on 12 conserved mitochondrial PCGs from the selected 40 angiosperm species (Fig. <a href="#Fig7" class="usa-link">7</a>). The results showed that the bootstrap support (BS) values of most nodes in the phylogenetic tree were greater than 80, which strongly supported the divisions of the four orders of Fabids. The phylogenetic tree was consistent with the angiosperm phylogeny group IV (APG IV) and plastid phylogenomic angiosperm (PPA II) classification system. In the <em>Quercus</em> genus, the <em>Q. gilva</em> of section <em>Cyclobalanopsis</em> in this study differentiated first at the base of the phylogenetic tree.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12363117_12870_2025_7072_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb06/12363117/b1768370410c/12870_2025_7072_Fig7_HTML.jpg" loading="lazy" id="MO7" height="515" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree of 40 angiosperms based on 12 conserved mitochondrial PCGs. The number at each node is the bootstrap support values</p></figcaption></figure></section></section><section id="Sec20"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec21"><h3 class="pmc_sec_title">The mitogenome of <em>Quercus gilva</em>
</h3>
<p id="Par57">In this study, we successfully assembled the mitogenome of <em>Q. gilva</em> by integrating short-read and long-read sequencing data, providing valuable information for mitogenome research within <em>Quercus</em> and Fagaceae. Compared to other published mitogenomes from closely related species within the family, the size, GC content and gene content of the mitogenome of <em>Q. gilva</em> were in the medium range, indicating a high level of assembly completeness and quality [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. The mitogenome of <em>Q. gilva</em> had a branched structure, which was similar to that of the mitogenome of <em>Q. acutissima</em> [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. The widespread distribution of repeats in this mitogenome was likely to contribute to genome rearrangement, which was a key factor in the existence of mitogenome isoforms [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>–<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. Dispersed repeat sequences were predominantly composed of forward and palindromic repeats, a pattern that was also observed in the mitogenomes of <em>Saposhnikovia divaricate</em> (Apiaceae) and <em>Calophyllum soulattri</em> (Calophyllaceae) [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>, <a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]. Statistical analysis revealed that most of these repeats were located in the IGS, which were generally considered hotspots for genome rearrangement. The presence of repeats within IGS likely enhanced their recombination potential, further contributing to the dynamic evolution and plasticity of mitogenomes [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>].</p>
<p id="Par58">RNA editing was widespread in the mitogenomes of higher plants, where it regulated gene expression by altering transcriptional sequences and played a crucial role in the mitogenomes [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>–<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. The most common form of RNA editing was a deamination reaction that converted C to U at a specific site [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]. In this study, we predicted a total of 491 potential RNA editing sites in 36 PCGs in the <em>Q. gilva</em> mitogenome, all of which exhibited the conversion of C to U. These RNA editing sites were unevenly distributed among different genes, with <em>nad4</em> exhibiting the highest number of editing sites, followed by <em>ccmB</em>. This distribution pattern was consistent with previous findings in <em>Q. acutissima</em> [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Furthermore, the RNA editing sites were mainly located at the second codon position, which was also observed in other plant mitogenomes [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>, <a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. These RNA editing sites provided valuable insights into the regulation of mitochondrial gene expression and function in plants.</p>
<p id="Par59">Endosymbiotic gene transfer in plant cells had been recognized as a significant evolutionary mechanism, facilitating the movement of genetic material between organelle genomes and nuclear genome [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Three main types of EGT events were commonly observed: between mitochondrion and nucleus (NUMT), between chloroplast and nucleus (NUPT), and between mitochondrion and chloroplast (MTPT) [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. We identified 11 MTPTs in the <em>Q. gilva</em> mitogenome, which included six complete chloroplast genes. MTPTs had also been reported in other angiosperms, such as <em>Ipomoea batatas</em> (Convolvulaceae), <em>Cannabis sativa</em> (Cannabaceae), and <em>Paeonia lactiflora</em> (Paeoniaceae) [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>–<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>]. Notably, most of the identified MTPTs were tRNA genes, which were commonly transferred between organelle genomes in angiosperms [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>]. In addition to MTPTs, we also identified 1824 NUMTs in the <em>Q. gilva</em>, further emphasizing the widespread nature of gene transfer. Notably, extensive sequence transfer was observed on chromosome 5, which encompassed nearly the entire mitogenome.</p>
<p id="Par60">These EGT events had profound implications for the evolution of angiosperms [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]. On the one hand, by integrating genetic fragments from one genome into another, EGT events increased genomic complexity and genetic diversity [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>]. On the other hand, some transferred genes were retained and subsequently functionalized, acquiring new biological roles in the recipient genome and enhancing the plant’s adaptability to environmental changes [<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>]. Although the successful establishment of functional genes in the nuclear genome was relatively rare, a large number of non-functional DNA transfer events appeared to be widespread, suggesting that such gene transfers accumulated over evolutionary timescales through neutral processes [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. Therefore, the combination of functional genomics and expression-based validation will be essential to further elucidate the specific roles of EGT events in plant adaptive evolution.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Significant structural variations in the mitogenomes of Fagaceae</h3>
<p id="Par61">The mitogenomes of plants exhibited complex structures and remarkable diversity, which had significantly hindered the progress of mitogenome research [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>, <a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>]. This trend was also evident in Fagaceae, where the number of published plastomes was nearly 20 times greater than that of mitogenomes. The mitogenome size in Fagaceae ranged from 349,290 bp to 592,702 bp, highlighting the substantial size variation not only within the same family but also within the same genus [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>, <a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>]. The number of genes in the mitogenomes was also highly diverse [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]. However, changes in mitogenome size did not correlate with the gene number, as the genome size was primarily influenced by the accumulation of non-coding regions [<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>]. The number of large and small subunits of ribosome genes varied most among all mitogenomes. In addition, three rRNA genes were consistently present: <em>rrn5</em>, <em>rrn18</em> (<em>rrnS</em>), and <em>rrn26</em> (<em>rrnL</em>), which were essential for mitochondrial protein synthesis. These findings strengthened the understanding of mitogenome diversity in Fagaceae.</p>
<p id="Par62">Many studies had traditionally presented plant mitogenomes as having a single circular structure [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>, <a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>, <a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>]. However, with the advancements in sequencing technologies and assembly methods, increasing evidence had shown that mitogenomes exhibited structural polymorphism [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>]. These structures included branching linear structures [<a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>, <a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>], polycyclic structures [<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>, <a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>], and linear structures [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. The mitogenome of <em>Q. gilva</em> assembled in this study also displayed a branching linear structure, consisting of three circular molecules and one linear molecule. This assembly structure was similar to that of the mitogenome of <em>Q. acutissima</em> [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. However, the mitogenomes of <em>F. sylvatica</em> and <em>Lithocarpus litseifolius</em> in Fagaceae were simplified circular structure [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>, <a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. And the mitogenomes of <em>Castanea mollissima</em>, <em>Castanopsis carlesii</em>, and <em>C. henryi</em> possessed complex multi-branched structures [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. These diverse structures were crucial for understanding the evolution of mitogenome architecture in Fagaceae, though the coexistence of multiple molecular structures needed further investigation.</p>
<p id="Par63">A comprehensive genome-wide collinearity analysis of 11 Fagaceae species was performed. The results revealed the complex structural evolution of Fagaceae. The significant number of homologous syntenic blocks highlighted the conserved nature of mitogenomes, while the observed genome rearrangements and inversions underscored the dynamic evolutionary processes of genome structure. Notably, the largest syntenic blocks were identified between <em>Q. petraea</em> and <em>Q. robur</em> within the same section, as well as between <em>C. mollissima</em> and <em>C. henryi</em> within the same genus. These findings strongly suggested that closely related species were more likely to retain larger syntenic regions. Similar patterns had also been observed in the mitogenomes of other plant species [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>, <a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>].</p></section><section id="Sec23"><h3 class="pmc_sec_title">Phylogeny and evolution in the mitogenomes of Fagaceae</h3>
<p id="Par64">We analyzed the Pi values and Ka/Ks ratios of shared PCGs in the mitogenomes and plastomes of Fagaceae, providing valuable insights into the evolution of different species [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>]. The results from both analyses indicated that the evolutionary rate of plastid genes was higher than that of mitochondrial genes, highlighting the relatively lower evolutionary rate of mitogenome [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Genes exhibiting higher Pi and Ka/Ks values (Ka/Ks &gt; 1) were likely involved in adaptive evolutionary processes within these Fagaceae species. Moreover, these genes could serve as potential molecular markers for phylogenetic studies and species identification within Fagaceae. In the mitogenome, <em>atp9</em> was a gene co-screened by the two analyses. The ATP9 protein played a crucial role in proton transport ATPase activity and proton transfer [<a href="#CR116" class="usa-link" aria-describedby="CR116">116</a>]. The evolution of this gene may influence the growth and development of Fagaceae plants.</p>
<p id="Par65">The mitogenomes of plants exhibited a relatively slow evolutionary rate, making them valuable tools for phylogenetic studies [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Our phylogenetic analysis grouped the selected 40 species into distinct taxa corresponding to their respective families and orders. Furthermore, the phylogenetic tree also revealed concordance with both the APG IV and PPA II classification systems [<a href="#CR117" class="usa-link" aria-describedby="CR117">117</a>, <a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>], demonstrating the reliability of using mitogenomes for constructing phylogenetic relationships. However, within the <em>Quercus</em> clade, <em>Quercus gilva</em> from section <em>Cyclobalanopsis</em> differentiated first at the base, a finding that contrasted with results derived from nuclear genome (RAD-seq) and plastome [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>, <a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>]. The different results may be due to the lack of data on the one hand, and may be due to the serious hybridization of <em>Quercus</em> on the other hand [<a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>]. As the dominant species in the subtropical evergreen broad-leaved forests of East Asia (EBLFs), the early divergence of section <em>Cyclobalanopsis</em> may also reflect ancient lineages associated with early ecological events, which provides a new evolutionary context for our mitochondrial genomics findings [<a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>]. Therefore, in order to further explore the phylogenetic relationships and evolutionary biology of this genus, we need to sequence and analyze the mitogenomes of more species of <em>Quercus</em>.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12363117/bin/12870_2025_7072_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7072_MOESM1_ESM.docx</a><sup> (4MB, docx) </sup><p>Supplementary Material 1: Figure S1. Structural graph comparison of the <em>Q. gilva</em> mitogenome assembled using different tools. (A) Mitochondrial genome graph assembled with GetOrganelle v1.7.6.1, with a total length of 388,738 bp; (B) Mitochondrial genome graph assembled with Unicycler v0.5.1, with a total length of 479,114 bp. Figure S2. Schematic representation of the assembly results of the <em>Q. gilva</em> mitogenome. (A) The final MMG graph assembled using the GSAT software; (B) The circular diagram of all contigs, with the green and blue rings representing the coverage depths of short and long reads, respectively; (C) The multi-branched structure graph resulted from resolving the structure of (A). Figure S3. Genome map of the <em>Q. gilva </em>plastome. Genes outside the circle are transcribed in the counter-clockwise direction, whereas those inside the circle are transcribed in the clockwise direction. The genes belonging to different functional groups are identified by different colors. The length and boundary of the LSC, SSC, and the two IR regions are indicated in the inner circle. The dark gray area indicates GC content while the lighter gray corresponds to the AT content of the genomes. LSC, large single copy; SSC, small single copy; IR, inverted repeat. Figure S4. Relative synonymous codon usage (RSCU) values of the <em>Q. gilva </em>mitogenome. Figure S5. Distribution of the PCGs in the mitogenomes of eleven Fagaceae species. Red boxes indicate that the PCG is present while blue boxes indicate that the PCG is absent. PCGs are categorized into core genes and variable genes following the classification of Adams et al. (2003). PCG, protein-coding genes. Figure S6. The Pi values (A) and Ka/Ks ratio (B) of 79 common plastome PCGs among Fagaceae. The box plot represents the distribution of Ka/Ks values for all pairwise comparison in the data. Pi, nucleotide diversity; Ka/Ks, nonsynonymous and synonymous substitution ratio</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12363117/bin/12870_2025_7072_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7072_MOESM2_ESM.xlsx</a><sup> (343.8KB, xlsx) </sup><p>Supplementary Material 2: Table S1. The basic information of selected angiosperms for constructing phylogenetic trees in this study. Table S2. Assembly statistics of different graphs of the <em>Q. gilva </em>mitogenome. Table S3. Contigs of the MMGs of the<em> Q. gilva </em>mitogenome. Table S4. Basic information statistics of genes of the <em>Q. gilva</em> mitogenome. Table S5. Basic information statistics of open reading frames (ORFs) of the <em>Q. gilva</em> mitogenome. Table S6. Simple sequence repeats (SSRs) of the <em>Q. gilva</em> mitogenome. Table S7. Tandem repeat sequences (TRSs) of the <em>Q. gilva </em>mitogenome. Table S8. Dispersed repeat sequences (DRSs) of the <em>Q. gilva </em>mitogenome. Table S9. Relative synonymous codon usage (RSCU) value and codon number of 61 codons of the <em>Q. gilva </em>mitogenome. Table S10. The predicted C to U RNA editing sites in the PCGs of the <em>Q. gilva</em> mitogenome. Table S11. The homologous mitochondrial plastid transferred fragments (MTPTs) identified between the mitogenome and plastome of <em>Q. gilva</em>. Table S12. The homologous nuclear mitochondrial transferred fragments (NUMTs) identified between the mitogenome and nuclear genome of <em>Q. gilva</em>. Table S13. Basic information on the eleven assembled mitogenomes in Fagaceae (access on 2024.11.24). Table S14. Ka/Ks values for pairwise comparisons of mitogenomes across 11 Fagaceae species. Table S15. Ka/Ks values for pairwise comparisons of plastomes across 11 Fagaceae species</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12363117/bin/12870_2025_7072_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">References</a><sup> (30.3KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We want to thank Wuhan Benagen Technology Co., Ltd. for their help on sequencing.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>BS</dt>
<dd><p id="Par4">Bootstrap support</p></dd>
<dt>CDS</dt>
<dd><p id="Par5">Coding sequences</p></dd>
<dt>CTAB</dt>
<dd><p id="Par6">Cetyltrimethylammonium bromide</p></dd>
<dt>DRS</dt>
<dd><p id="Par7">Dispersed repeat sequence</p></dd>
<dt>EGT</dt>
<dd><p id="Par8">Endosymbiotic gene transfer</p></dd>
<dt>IGS</dt>
<dd><p id="Par9">Intergenic spacer region</p></dd>
<dt>IR</dt>
<dd><p id="Par10">Inverted repeats</p></dd>
<dt>Ka/Ks</dt>
<dd><p id="Par11">Nonsynonymous and synonymous substitution ratio</p></dd>
<dt>LSC</dt>
<dd><p id="Par12">Large single copy</p></dd>
<dt>MTPT</dt>
<dd><p id="Par13">Mitochondrial plastid transferred fragment</p></dd>
<dt>NCBI</dt>
<dd><p id="Par14">National center of biotechnology information</p></dd>
<dt>NGDC</dt>
<dd><p id="Par15">National genomics data center</p></dd>
<dt>NUMT</dt>
<dd><p id="Par16">Nuclear mitochondrial transferred fragment</p></dd>
<dt>ORF</dt>
<dd><p id="Par17">Open reading frame</p></dd>
<dt>PCG</dt>
<dd><p id="Par18">Protein-coding gene</p></dd>
<dt>Pi</dt>
<dd><p id="Par19">Nucleotide diversity</p></dd>
<dt>rRNA</dt>
<dd><p id="Par20">Ribosomal RNA</p></dd>
<dt>RSCU</dt>
<dd><p id="Par21">Relative synonymous codon usage</p></dd>
<dt>SSC</dt>
<dd><p id="Par22">Small single copy</p></dd>
<dt>SSR</dt>
<dd><p id="Par23">Simple sequence repeat</p></dd>
<dt>tRNA</dt>
<dd><p id="Par24">Transfer RNA</p></dd>
<dt>TRS</dt>
<dd><p id="Par25">Tandem repeat sequence</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>Y.G. S. conceived and conceptualized the study; Y. L. analyzed the mitogenome and wrote the manuscript; S.S. Z. and Y.G. S. collected the sample; G. K. and Y.G. S. reviewed and revised the manuscript. All authors have read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the National Natural Science Foundation of China (grant number 31901217), the Special Fund for Scientific Research of Shanghai Landscaping and City Appearance Administrative Bureau (grant numbers G192422, G242414, and G242416), and the Science and Technology Development Center, National Forestry and Grassland Administration (grant number KJZXSA202214).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The assembled complete organelle genomes of <em>Quercus gilva</em> has been deposited in GenBank (NCBI), under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PQ862789:PQ862792%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ862789–PQ862792</a> for the mitogenome and <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ876385" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ876385</a> for the plastome.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par66">The collection and usage of plant specimens in current study complied with the IUCN Policy Statement on Research Involving Species at Risk of Extinction and the Convention on the Trade in Endangered Species of Wild Fauna and Flora. Ethical approval was not applicable for this study.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par67">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par68">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Drouin G, Daoud H, Xia JN. Relative rates of synonymous substitutions in the mitochondrial, chloroplast and nuclear genomes of seed plants. Mol Phylogenet Evol. 2008;49(3):827–31.
</cite> [<a href="https://doi.org/10.1016/j.ympev.2008.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18838124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Drouin%20G,%20Daoud%20H,%20Xia%20JN.%20Relative%20rates%20of%20synonymous%20substitutions%20in%20the%20mitochondrial,%20chloroplast%20and%20nuclear%20genomes%20of%20seed%20plants.%20Mol%20Phylogenet%20Evol.%202008;49(3):827%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Wolfe KH, Li WH, Sharp PM. Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs. Proc Natl Acad Sci U S A. 1987;84(24):9054–8.
</cite> [<a href="https://doi.org/10.1073/pnas.84.24.9054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC299690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3480529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolfe%20KH,%20Li%20WH,%20Sharp%20PM.%20Rates%20of%20nucleotide%20substitution%20vary%20greatly%20among%20plant%20mitochondrial,%20chloroplast,%20and%20nuclear%20DNAs.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%201987;84(24):9054%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Muse SV. Examining rates and patterns of nucleotide substitution in plants. Plant Mol Biol. 2000;42(1):25–43.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10688129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muse%20SV.%20Examining%20rates%20and%20patterns%20of%20nucleotide%20substitution%20in%20plants.%20Plant%20Mol%20Biol.%202000;42(1):25%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Bi CW, Sun N, Han FC, Xu KW, Yang Y, Ferguson DK. The first mitogenome of Lauraceae (<em>Cinnamomum chekiangense</em>). Plant Divers. 2023;46(1):144–8.
</cite> [<a href="https://doi.org/10.1016/j.pld.2023.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10851304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38343589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bi%20CW,%20Sun%20N,%20Han%20FC,%20Xu%20KW,%20Yang%20Y,%20Ferguson%20DK.%20The%20first%20mitogenome%20of%20Lauraceae%20(Cinnamomum%20chekiangense).%20Plant%20Divers.%202023;46(1):144%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Hu HY, Sun PC, Yang YZ, Ma JX, Liu JQ. Genome-scale angiosperm phylogenies based on nuclear, plastome, and mitochondrial datasets. J Integr Plant Biol. 2023;65(6):1479–89.
</cite> [<a href="https://doi.org/10.1111/jipb.13455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36647606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20HY,%20Sun%20PC,%20Yang%20YZ,%20Ma%20JX,%20Liu%20JQ.%20Genome-scale%20angiosperm%20phylogenies%20based%20on%20nuclear,%20plastome,%20and%20mitochondrial%20datasets.%20J%20Integr%20Plant%20Biol.%202023;65(6):1479%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wang J, Kan SL, Liao XZ, Zhou JW, Tembrock LR, Daniell H, Jin SX, Wu ZQ. Plant organellar genomes: much done, much more to do. Trends Plant Sci. 2024;29(7):754–69.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2023.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38220520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Kan%20SL,%20Liao%20XZ,%20Zhou%20JW,%20Tembrock%20LR,%20Daniell%20H,%20Jin%20SX,%20Wu%20ZQ.%20Plant%20organellar%20genomes:%20much%20done,%20much%20more%20to%20do.%20Trends%20Plant%20Sci.%202024;29(7):754%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Xiang KL, Wu SD, Lian L, He WC, Peng D, Peng HW, Zhang XN, Li HL, et al. Genomic data and ecological niche modeling reveal an unusually slow rate of molecular evolution in the cretaceous Eupteleaceae. Sci China Life Sci. 2023;67(4):803–16.
</cite> [<a href="https://doi.org/10.1007/s11427-023-2448-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38087029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiang%20KL,%20Wu%20SD,%20Lian%20L,%20He%20WC,%20Peng%20D,%20Peng%20HW,%20Zhang%20XN,%20Li%20HL,%20et%20al.%20Genomic%20data%20and%20ecological%20niche%20modeling%20reveal%20an%20unusually%20slow%20rate%20of%20molecular%20evolution%20in%20the%20cretaceous%20Eupteleaceae.%20Sci%20China%20Life%20Sci.%202023;67(4):803%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hartman H, Fedorov A. The origin of the eukaryotic cell: a genomic investigation. Proc Natl Acad Sci U S A. 2002;99(3):1420–5.
</cite> [<a href="https://doi.org/10.1073/pnas.032658599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC122206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11805300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hartman%20H,%20Fedorov%20A.%20The%20origin%20of%20the%20eukaryotic%20cell:%20a%20genomic%20investigation.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202002;99(3):1420%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Timmis JN, Ayliffe MA, Huang CY, Martin W. Endosymbiotic gene transfer: organelle genomes forge eukaryotic chromosomes. Nat Rev Genet. 2004;5(2):123–35.
</cite> [<a href="https://doi.org/10.1038/nrg1271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14735123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Timmis%20JN,%20Ayliffe%20MA,%20Huang%20CY,%20Martin%20W.%20Endosymbiotic%20gene%20transfer:%20organelle%20genomes%20forge%20eukaryotic%20chromosomes.%20Nat%20Rev%20Genet.%202004;5(2):123%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Lang BF, Burger G. Mitochondrial and eukaryotic origins: a critical review. Adv Bot Res. 2012;63:1–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?Lang%20BF,%20Burger%20G.%20Mitochondrial%20and%20eukaryotic%20origins:%20a%20critical%20review.%20Adv%20Bot%20Res.%202012;63:1%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Martin WF, Garg S, Zimorski V. Endosymbiotic theories for eukaryote origin. Phil Trans R Soc Lond B Biol Sci. 2015;370(1678): 20140330.
</cite> [<a href="https://doi.org/10.1098/rstb.2014.0330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4571569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26323761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin%20WF,%20Garg%20S,%20Zimorski%20V.%20Endosymbiotic%20theories%20for%20eukaryote%20origin.%20Phil%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci.%202015;370(1678):%2020140330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Adams KL, Palmer JD. Evolution of mitochondrial gene content: gene loss and transfer to the nucleus. Mol Phylogenet Evol. 2003;29(3):380–95.
</cite> [<a href="https://doi.org/10.1016/s1055-7903(03)00194-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14615181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams%20KL,%20Palmer%20JD.%20Evolution%20of%20mitochondrial%20gene%20content:%20gene%20loss%20and%20transfer%20to%20the%20nucleus.%20Mol%20Phylogenet%20Evol.%202003;29(3):380%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Chen YM, Guo YW, Xie XM, Wang ZH, Miao LF, Yang ZZ, Jiao YN, Xie CJ, et al. Pangenome-based trajectories of intracellular gene transfers in Poaceae unveil high cumulation in triticeae. Plant Physiol. 2023;193(1):578–94.
</cite> [<a href="https://doi.org/10.1093/plphys/kiad319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10469385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37249052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20YM,%20Guo%20YW,%20Xie%20XM,%20Wang%20ZH,%20Miao%20LF,%20Yang%20ZZ,%20Jiao%20YN,%20Xie%20CJ,%20et%20al.%20Pangenome-based%20trajectories%20of%20intracellular%20gene%20transfers%20in%20Poaceae%20unveil%20high%20cumulation%20in%20triticeae.%20Plant%20Physiol.%202023;193(1):578%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Gualberto JM, Newton KJ. Plant mitochondrial genomes: dynamics and mechanisms of mutation. Annu Rev Plant Biol. 2017;68:225–52.
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-043015-112232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28226235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gualberto%20JM,%20Newton%20KJ.%20Plant%20mitochondrial%20genomes:%20dynamics%20and%20mechanisms%20of%20mutation.%20Annu%20Rev%20Plant%20Biol.%202017;68:225%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Daniell H, Lin CS, Yu M, Chang WJ. Chloroplast genomes: diversity, evolution, and applications in genetic engineering. Genome Biol. 2016;17(1):134.
</cite> [<a href="https://doi.org/10.1186/s13059-016-1004-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4918201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27339192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daniell%20H,%20Lin%20CS,%20Yu%20M,%20Chang%20WJ.%20Chloroplast%20genomes:%20diversity,%20evolution,%20and%20applications%20in%20genetic%20engineering.%20Genome%20Biol.%202016;17(1):134." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Palmer JD. Comparative organization of Chloroplast genomes. Annu Rev Genet. 1985;19:325–54.
</cite> [<a href="https://doi.org/10.1146/annurev.ge.19.120185.001545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3936406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palmer%20JD.%20Comparative%20organization%20of%20Chloroplast%20genomes.%20Annu%20Rev%20Genet.%201985;19:325%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Shaw J, Lickey EB, Schilling EE, Small RL. Comparison of whole chloroplast genome sequences to choose noncoding regions for phylogenetic studies in angiosperms: the tortoise and the hare III. Am J Bot. 2007;94(3):275–88.
</cite> [<a href="https://doi.org/10.3732/ajb.94.3.275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21636401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaw%20J,%20Lickey%20EB,%20Schilling%20EE,%20Small%20RL.%20Comparison%20of%20whole%20chloroplast%20genome%20sequences%20to%20choose%20noncoding%20regions%20for%20phylogenetic%20studies%20in%20angiosperms:%20the%20tortoise%20and%20the%20hare%20III.%20Am%20J%20Bot.%202007;94(3):275%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Wakasugi T, Tsudzuki T, Sugiura M. The genomics of land plant chloroplasts: gene content and alteration of genomic information by RNA editing. Photosynth Res. 2001;70(1):107–18.
</cite> [<a href="https://doi.org/10.1023/A:1013892009589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16228365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wakasugi%20T,%20Tsudzuki%20T,%20Sugiura%20M.%20The%20genomics%20of%20land%20plant%20chloroplasts:%20gene%20content%20and%20alteration%20of%20genomic%20information%20by%20RNA%20editing.%20Photosynth%20Res.%202001;70(1):107%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Alverson AJ, Wei XX, Rice DW, Stern DB, Barry K, Palmer JD. Insights into the evolution of mitochondrial genome size from complete sequences of <em>Citrullus lanatus</em> and <em>Cucurbita pepo</em> (Cucurbitaceae). Mol Biol Evol. 2010;27(6):1436–48.
</cite> [<a href="https://doi.org/10.1093/molbev/msq029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2877997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20118192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alverson%20AJ,%20Wei%20XX,%20Rice%20DW,%20Stern%20DB,%20Barry%20K,%20Palmer%20JD.%20Insights%20into%20the%20evolution%20of%20mitochondrial%20genome%20size%20from%20complete%20sequences%20of%20Citrullus%20lanatus%20and%20Cucurbita%20pepo%20(Cucurbitaceae).%20Mol%20Biol%20Evol.%202010;27(6):1436%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Putintseva YA, Bondar EI, Simonov EP, Sharov VV, Oreshkova NV, Kuzmin DA, Konstantinov YM, Shmakov VN, et al. Siberian larch (<em>Larix sibirica</em> Ledeb.) mitochondrial genome assembled using both short and long nucleotide sequence reads is currently the largest known mitogenome. BMC Genomics. 2020;21(1):654.
</cite> [<a href="https://doi.org/10.1186/s12864-020-07061-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7517811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32972367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Putintseva%20YA,%20Bondar%20EI,%20Simonov%20EP,%20Sharov%20VV,%20Oreshkova%20NV,%20Kuzmin%20DA,%20Konstantinov%20YM,%20Shmakov%20VN,%20et%20al.%20Siberian%20larch%20(Larix%20sibirica%20Ledeb.)%20mitochondrial%20genome%20assembled%20using%20both%20short%20and%20long%20nucleotide%20sequence%20reads%20is%20currently%20the%20largest%20known%20mitogenome.%20BMC%20Genomics.%202020;21(1):654." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Skippington E, Barkman TJ, Rice DW, Palmer JD. Miniaturized mitogenome of the parasitic plant <em>Viscum scurruloideum</em> is extremely divergent and dynamic and has lost all Nad genes. Proc Natl Acad Sci U S A. 2015;112(27):E3515-24.
</cite> [<a href="https://doi.org/10.1073/pnas.1504491112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4500244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26100885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skippington%20E,%20Barkman%20TJ,%20Rice%20DW,%20Palmer%20JD.%20Miniaturized%20mitogenome%20of%20the%20parasitic%20plant%20Viscum%20scurruloideum%20is%20extremely%20divergent%20and%20dynamic%20and%20has%20lost%20all%20Nad%20genes.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202015;112(27):E3515-24." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Sloan DB. One ring to rule them all? Genome sequencing provides new insights into the ‘master circle’ model of plant mitochondrial DNA structure. New Phytol. 2013;200(4):978–85.
</cite> [<a href="https://doi.org/10.1111/nph.12395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24712049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sloan%20DB.%20One%20ring%20to%20rule%20them%20all?%20Genome%20sequencing%20provides%20new%20insights%20into%20the%20%E2%80%98master%20circle%E2%80%99%20model%20of%20plant%20mitochondrial%20DNA%20structure.%20New%20Phytol.%202013;200(4):978%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Zhang S, Wang J, He WC, Kan SL, Liao XZ, Jordan DR, Mace ES, Tao YF, et al. Variation in mitogenome structural conformation in wild and cultivated lineages of sorghum corresponds with domestication history and plastome evolution. BMC Plant Biol. 2023;23(1):91.
</cite> [<a href="https://doi.org/10.1186/s12870-023-04104-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9926791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36782130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Wang%20J,%20He%20WC,%20Kan%20SL,%20Liao%20XZ,%20Jordan%20DR,%20Mace%20ES,%20Tao%20YF,%20et%20al.%20Variation%20in%20mitogenome%20structural%20conformation%20in%20wild%20and%20cultivated%20lineages%20of%20sorghum%20corresponds%20with%20domestication%20history%20and%20plastome%20evolution.%20BMC%20Plant%20Biol.%202023;23(1):91." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kubo T, Newton KJ. Angiosperm mitochondrial genomes and mutations. Mitochondrion. 2008;8(1):5–14.
</cite> [<a href="https://doi.org/10.1016/j.mito.2007.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18065297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kubo%20T,%20Newton%20KJ.%20Angiosperm%20mitochondrial%20genomes%20and%20mutations.%20Mitochondrion.%202008;8(1):5%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Palmer JD, Adams KL, Cho Y, Parkinson CL, Qiu YL, Song KM. Dynamic evolution of plant mitochondrial genomes: mobile genes and introns and highly variable mutation rates. Proc Natl Acad Sci U S A. 2000;97(13):6960–6.
</cite> [<a href="https://doi.org/10.1073/pnas.97.13.6960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC34370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10860957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palmer%20JD,%20Adams%20KL,%20Cho%20Y,%20Parkinson%20CL,%20Qiu%20YL,%20Song%20KM.%20Dynamic%20evolution%20of%20plant%20mitochondrial%20genomes:%20mobile%20genes%20and%20introns%20and%20highly%20variable%20mutation%20rates.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202000;97(13):6960%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Christensen AC. Plant mitochondrial genome evolution can be explained by DNA repair mechanisms. Genome Biol Evol. 2013;5(6):1079–86.
</cite> [<a href="https://doi.org/10.1093/gbe/evt069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3698917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23645599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christensen%20AC.%20Plant%20mitochondrial%20genome%20evolution%20can%20be%20explained%20by%20DNA%20repair%20mechanisms.%20Genome%20Biol%20Evol.%202013;5(6):1079%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Knoop V. The mitochondrial DNA of land plants: peculiarities in phylogenetic perspective. Curr Genet. 2004;46(3):123–39.
</cite> [<a href="https://doi.org/10.1007/s00294-004-0522-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15300404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knoop%20V.%20The%20mitochondrial%20DNA%20of%20land%20plants:%20peculiarities%20in%20phylogenetic%20perspective.%20Curr%20Genet.%202004;46(3):123%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Oda K, Yamato K, Ohta E, Nakamura Y, Takemura M, Nozato N, Akashi K, Kanegae T, et al. Gene organization deduced from the complete sequence of liverwort <em>Marchantia polymorpha</em> mitochondrial DNA: a primitive form of plant mitochondrial genome. J Mol Biol. 1992;223(1):1–7.
</cite> [<a href="https://doi.org/10.1016/0022-2836(92)90708-r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1731062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oda%20K,%20Yamato%20K,%20Ohta%20E,%20Nakamura%20Y,%20Takemura%20M,%20Nozato%20N,%20Akashi%20K,%20Kanegae%20T,%20et%20al.%20Gene%20organization%20deduced%20from%20the%20complete%20sequence%20of%20liverwort%20Marchantia%20polymorpha%20mitochondrial%20DNA:%20a%20primitive%20form%20of%20plant%20mitochondrial%20genome.%20J%20Mol%20Biol.%201992;223(1):1%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Keeling PJ. Horizontal gene transfer in eukaryotes: aligning theory with data. Nat Rev Genet. 2024;25:416–30.
</cite> [<a href="https://doi.org/10.1038/s41576-023-00688-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38263430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keeling%20PJ.%20Horizontal%20gene%20transfer%20in%20eukaryotes:%20aligning%20theory%20with%20data.%20Nat%20Rev%20Genet.%202024;25:416%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Notsu Y, Masood S, Nishikawa T, Kubo N, Akiduki G, Nakazono M, Hirai A, Kadowaki K. The complete sequence of the rice (<em>Oryza sativa</em> L.) mitochondrial genome: frequent DNA sequence acquisition and loss during the evolution of flowering plants. Mol Genet Genomics. 2002;268(4):434–45.
</cite> [<a href="https://doi.org/10.1007/s00438-002-0767-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12471441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Notsu%20Y,%20Masood%20S,%20Nishikawa%20T,%20Kubo%20N,%20Akiduki%20G,%20Nakazono%20M,%20Hirai%20A,%20Kadowaki%20K.%20The%20complete%20sequence%20of%20the%20rice%20(Oryza%20sativa%20L.)%20mitochondrial%20genome:%20frequent%20DNA%20sequence%20acquisition%20and%20loss%20during%20the%20evolution%20of%20flowering%20plants.%20Mol%20Genet%20Genomics.%202002;268(4):434%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Stupar RM, Lilly JW, Town CD, Cheng ZK, Kaul S, Buell CR, Jiang JM. Complex MtDNA constitutes an approximate 620-kb insertion on <em>Arabidopsis thaliana</em> chromosome 2: implication of potential sequencing errors caused by large-unit repeats. Proc Natl Acad Sci U S A. 2001;98(9):5099–103.
</cite> [<a href="https://doi.org/10.1073/pnas.091110398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC33170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11309509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stupar%20RM,%20Lilly%20JW,%20Town%20CD,%20Cheng%20ZK,%20Kaul%20S,%20Buell%20CR,%20Jiang%20JM.%20Complex%20MtDNA%20constitutes%20an%20approximate%20620-kb%20insertion%20on%20Arabidopsis%20thaliana%20chromosome%202:%20implication%20of%20potential%20sequencing%20errors%20caused%20by%20large-unit%20repeats.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202001;98(9):5099%E2%80%93103." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Denk T, Grimm GW, Manos PS, Deng M, Hipp AL et al. An updated infrageneric classification of the oaks: review of previous taxonomic schemes and synthesis of evolutionary patterns. In: Gil-Pelegrín E, editors. Oaks Physiological Ecology. Exploring the Functional Diversity of Genus <em>Quercus</em> L. Tree Physiol. 2017;13–38.</cite>
</li>
<li id="CR33">
<span class="label">33.</span><cite>Cavender-Bares J. Diversification, adaptation, and community assembly of the American oaks (<em>Quercus</em>), a model clade for integrating ecology and evolution. New Phytol. 2019;221(2):669–92.
</cite> [<a href="https://doi.org/10.1111/nph.15450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30368821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cavender-Bares%20J.%20Diversification,%20adaptation,%20and%20community%20assembly%20of%20the%20American%20oaks%20(Quercus),%20a%20model%20clade%20for%20integrating%20ecology%20and%20evolution.%20New%20Phytol.%202019;221(2):669%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Jin DM, Yuan Q, Dai XL, Kozlowski G, Song YG. Enhanced precipitation drove the evolution of subtropical evergreen broad-leaved forests in eastern China since the early miocene: evidence from ring-cupped oaks. J Syst Evol. 2023;62:677–86.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jin%20DM,%20Yuan%20Q,%20Dai%20XL,%20Kozlowski%20G,%20Song%20YG.%20Enhanced%20precipitation%20drove%20the%20evolution%20of%20subtropical%20evergreen%20broad-leaved%20forests%20in%20eastern%20China%20since%20the%20early%20miocene:%20evidence%20from%20ring-cupped%20oaks.%20J%20Syst%20Evol.%202023;62:677%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kremer A. Microevolution of European temperate Oaks in response to environmental changes. C R Biol. 2016;339(7–8):263–7.
</cite> [<a href="https://doi.org/10.1016/j.crvi.2016.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27263361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kremer%20A.%20Microevolution%20of%20European%20temperate%20Oaks%20in%20response%20to%20environmental%20changes.%20C%20R%20Biol.%202016;339(7%E2%80%938):263%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Fu RR, Zhu YX, Liu Y, Feng Y, Lu RS, Li Y, Li P, Kremer A, et al. Genome-wide analyses of introgression between two sympatric Asian oak species. Nat Ecol Evol. 2022;6(7):924–35.
</cite> [<a href="https://doi.org/10.1038/s41559-022-01754-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35513577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu%20RR,%20Zhu%20YX,%20Liu%20Y,%20Feng%20Y,%20Lu%20RS,%20Li%20Y,%20Li%20P,%20Kremer%20A,%20et%20al.%20Genome-wide%20analyses%20of%20introgression%20between%20two%20sympatric%20Asian%20oak%20species.%20Nat%20Ecol%20Evol.%202022;6(7):924%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Kapoor B, Jenkins J, Schmutz J, Zhebentyayeva T, Kuelheim C, Coggeshall M, Heim C, Lasky JR, et al. A haplotype-resolved chromosome-scale genome for <em>Quercus rubra</em> L. provides insights into the genetics of adaptive traits for red oak species. (Bethesda). 2023;G3(11):jkad209.</cite> [<a href="https://doi.org/10.1093/g3journal/jkad209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10627279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37708394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kapoor%20B,%20Jenkins%20J,%20Schmutz%20J,%20Zhebentyayeva%20T,%20Kuelheim%20C,%20Coggeshall%20M,%20Heim%20C,%20Lasky%20JR,%20et%20al.%20A%20haplotype-resolved%20chromosome-scale%20genome%20for%20Quercus%20rubra%20L.%20provides%20insights%20into%20the%20genetics%20of%20adaptive%20traits%20for%20red%20oak%20species.%20(Bethesda).%202023;G3(11):jkad209." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Li Y, Zheng SS, Wang TR, Liu MH, Kozlowski G, Yi LT, Song YG. New insights on the phylogeny, evolutionary history, and ecological adaptation mechanism in cycle-cup oaks based on chloroplast genomes. Ecol Evol. 2024;14(9): e70318.
</cite> [<a href="https://doi.org/10.1002/ece3.70318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11407850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39290669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Y,%20Zheng%20SS,%20Wang%20TR,%20Liu%20MH,%20Kozlowski%20G,%20Yi%20LT,%20Song%20YG.%20New%20insights%20on%20the%20phylogeny,%20evolutionary%20history,%20and%20ecological%20adaptation%20mechanism%20in%20cycle-cup%20oaks%20based%20on%20chloroplast%20genomes.%20Ecol%20Evol.%202024;14(9):%20e70318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Liu X, Zhang WX, Zhang YT, Yang J, Zeng P, Tian ZZ, Sun WB, Cai J. Chromosome-scale genomes of <em>Quercus sichourensis</em> and <em>Quercus Rex</em> provide insights into the evolution and adaptation of Fagaceae. J Genet Genomics. 2024;51(5):554–65.
</cite> [<a href="https://doi.org/10.1016/j.jgg.2024.03.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38575109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20X,%20Zhang%20WX,%20Zhang%20YT,%20Yang%20J,%20Zeng%20P,%20Tian%20ZZ,%20Sun%20WB,%20Cai%20J.%20Chromosome-scale%20genomes%20of%20Quercus%20sichourensis%20and%20Quercus%20Rex%20provide%20insights%20into%20the%20evolution%20and%20adaptation%20of%20Fagaceae.%20J%20Genet%20Genomics.%202024;51(5):554%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wang LL, Li Y, Zheng SS, Kozlowski G, Xu J, Song YG. Complete chloroplast genomes of four oaks from the section <em>Cyclobalanopsis</em> improve the phylogenetic analysis and understanding of evolutionary processes in the genus <em>Quercus</em>. Genes. 2024;15(2): 230.
</cite> [<a href="https://doi.org/10.3390/genes15020230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10888318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38397219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20LL,%20Li%20Y,%20Zheng%20SS,%20Kozlowski%20G,%20Xu%20J,%20Song%20YG.%20Complete%20chloroplast%20genomes%20of%20four%20oaks%20from%20the%20section%20Cyclobalanopsis%20improve%20the%20phylogenetic%20analysis%20and%20understanding%20of%20evolutionary%20processes%20in%20the%20genus%20Quercus.%20Genes.%202024;15(2):%20230." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Zhou YR, Li Y, Yang LH, Kozlowski G, Yi LT, Liu MH, Zheng SS, Song YG. The adaptive evolution of <em>Quercus</em> section <em>Ilex</em> using the Chloroplast genomes of two threatened species. Sci Rep. 2024;14:20577.
</cite> [<a href="https://doi.org/10.1038/s41598-024-71838-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11375091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39232239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20YR,%20Li%20Y,%20Yang%20LH,%20Kozlowski%20G,%20Yi%20LT,%20Liu%20MH,%20Zheng%20SS,%20Song%20YG.%20The%20adaptive%20evolution%20of%20Quercus%20section%20Ilex%20using%20the%20Chloroplast%20genomes%20of%20two%20threatened%20species.%20Sci%20Rep.%202024;14:20577." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Bi QX, Li DX, Zhao Y, Wang MK, Li YC, Liu XJ, Wang LB, Yu HY. Complete mitochondrial genome of <em>Quercus variabilis</em> (Fagales, Fagaceae). Mitochondrial DNA B. 2019;4(2):3927–8.</cite> [<a href="https://doi.org/10.1080/23802359.2019.1687027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7707397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33366255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bi%20QX,%20Li%20DX,%20Zhao%20Y,%20Wang%20MK,%20Li%20YC,%20Liu%20XJ,%20Wang%20LB,%20Yu%20HY.%20Complete%20mitochondrial%20genome%20of%20Quercus%20variabilis%20(Fagales,%20Fagaceae).%20Mitochondrial%20DNA%20B.%202019;4(2):3927%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Liu D, Guo HL, Zhu JL, Qu K, Chen Y, Guo YT, Ding P, Yang HP, et al. Complex physical structure of complete mitochondrial genome of <em>Quercus acutissima</em> (Fagaceae): a significant energy plant. Genes. 2022;13(8): 1321.
</cite> [<a href="https://doi.org/10.3390/genes13081321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9331829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35893058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20D,%20Guo%20HL,%20Zhu%20JL,%20Qu%20K,%20Chen%20Y,%20Guo%20YT,%20Ding%20P,%20Yang%20HP,%20et%20al.%20Complex%20physical%20structure%20of%20complete%20mitochondrial%20genome%20of%20Quercus%20acutissima%20(Fagaceae):%20a%20significant%20energy%20plant.%20Genes.%202022;13(8):%201321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Huang CJ, Zhang YT, Bruce B. In: Chun WY, Huang CJ, editors. Fagaceae. Flora of China, Science Press and Missouri Botanical Garden; 1999. p. 380–400.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Song YG, Wang TR, Lu ZJ, Ge BJ, Zhong X, Li XC, Jin DM, Yuan Q, et al. Population survey combined with genomic-wide genetic variation unravels the endangered status of <em>Quercus gilva</em>. Diversity. 2023;15:230.</cite> [<a href="https://scholar.google.com/scholar_lookup?Song%20YG,%20Wang%20TR,%20Lu%20ZJ,%20Ge%20BJ,%20Zhong%20X,%20Li%20XC,%20Jin%20DM,%20Yuan%20Q,%20et%20al.%20Population%20survey%20combined%20with%20genomic-wide%20genetic%20variation%20unravels%20the%20endangered%20status%20of%20Quercus%20gilva.%20Diversity.%202023;15:230." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Carrero C, Jerome D, Beckman E, Byrne A, Coombes AJ, Deng M, Rodríguez AG, Sam HV, et al. The Red List of Oaks 2020. The Morton Arboretum. Lisle, IL: 2020.</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Sugiura N, Tang DQ, Kurokochi H, Saito Y, Ide Y. Genetic structure of <em>Quercus Gilva</em> Blume in Japan as revealed by chloroplast DNA sequences. Botany. 2015;93(12):870–80.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sugiura%20N,%20Tang%20DQ,%20Kurokochi%20H,%20Saito%20Y,%20Ide%20Y.%20Genetic%20structure%20of%20Quercus%20Gilva%20Blume%20in%20Japan%20as%20revealed%20by%20chloroplast%20DNA%20sequences.%20Botany.%202015;93(12):870%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Wang TR, Ning X, Zheng SS, Li Y, Lu ZJ, Meng HH, Ge BJ, Kozlowski G, et al. Genomic insights into ecological adaptation of Oaks revealed by phylogenomic analysis of multiple species. Plant Divers. 2025;47(1):53–67.</cite> [<a href="https://doi.org/10.1016/j.pld.2024.07.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11873581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40041560/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Zeng QM, Liu B, Lin RQ, Jiang YT, Liu ZJ, Chen SP. The complete Chloroplast genome sequence of <em>Quercus Gilva</em> (Fagaceae). Mitochondrial DNA B. 2019;4(2):2493–4.</cite> [<a href="https://doi.org/10.1080/23802359.2019.1637299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7687598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33365597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20QM,%20Liu%20B,%20Lin%20RQ,%20Jiang%20YT,%20Liu%20ZJ,%20Chen%20SP.%20The%20complete%20Chloroplast%20genome%20sequence%20of%20Quercus%20Gilva%20(Fagaceae).%20Mitochondrial%20DNA%20B.%202019;4(2):2493%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Yang YC, Zhou T, Qian ZQ, Zhao GF. Phylogenetic relationships in Chinese oaks (Fagaceae, <em>Quercus</em>): evidence from plastid genome using low-coverage whole genome sequencing. Genomics. 2021;113(3):1438–47.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2021.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33744343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20YC,%20Zhou%20T,%20Qian%20ZQ,%20Zhao%20GF.%20Phylogenetic%20relationships%20in%20Chinese%20oaks%20(Fagaceae,%20Quercus):%20evidence%20from%20plastid%20genome%20using%20low-coverage%20whole%20genome%20sequencing.%20Genomics.%202021;113(3):1438%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Doyle JJ. A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochem Bull. 1987;19(1):11–5.</cite> [<a href="https://scholar.google.com/scholar_lookup?Doyle%20JJ.%20A%20rapid%20DNA%20isolation%20procedure%20for%20small%20quantities%20of%20fresh%20leaf%20tissue.%20Phytochem%20Bull.%201987;19(1):11%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Jin JJ, Yu WB, Yang JB, Song Y, dePamphilis CW, Yi TS, Li DZ. Getorganelle: a fast and versatile toolkit for accurate de novo assembly of organelle genomes. Genome Biol. 2020;21(1):241.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02154-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7488116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32912315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20JJ,%20Yu%20WB,%20Yang%20JB,%20Song%20Y,%20dePamphilis%20CW,%20Yi%20TS,%20Li%20DZ.%20Getorganelle:%20a%20fast%20and%20versatile%20toolkit%20for%20accurate%20de%20novo%20assembly%20of%20organelle%20genomes.%20Genome%20Biol.%202020;21(1):241." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wick RR, Judd LM, Gorrie CL, Holt KE, Unicycler. Resolving bacterial genome assemblies from short and long sequencing reads. PLoS Comput Biol. 2017;13(6):e1005595.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5481147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28594827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Judd%20LM,%20Gorrie%20CL,%20Holt%20KE,%20Unicycler.%20Resolving%20bacterial%20genome%20assemblies%20from%20short%20and%20long%20sequencing%20reads.%20PLoS%20Comput%20Biol.%202017;13(6):e1005595." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>He WC, Xiang KL, Chen CJ, Wang J, Wu ZQ. Master graph: an essential integrated assembly model for the plant mitogenome based on a graph-based framework. Brief Bioinform. 2023;24(1): bbac522.
</cite> [<a href="https://doi.org/10.1093/bib/bbac522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36644898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20WC,%20Xiang%20KL,%20Chen%20CJ,%20Wang%20J,%20Wu%20ZQ.%20Master%20graph:%20an%20essential%20integrated%20assembly%20model%20for%20the%20plant%20mitogenome%20based%20on%20a%20graph-based%20framework.%20Brief%20Bioinform.%202023;24(1):%20bbac522." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Wick RR, Schultz MB, Zobel J, Holt KE. Bandage: interactive visualization of de novo genome assemblies. Bioinformatics. 2015;31(20):3350–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4595904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26099265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Schultz%20MB,%20Zobel%20J,%20Holt%20KE.%20Bandage:%20interactive%20visualization%20of%20de%20novo%20genome%20assemblies.%20Bioinformatics.%202015;31(20):3350%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34(18):3094–100.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H.%20Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences.%20Bioinformatics.%202018;34(18):3094%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Tillich M, Lehwark P, Pellizzer T, Ulbricht-Jones ES, Fischer A, Bock R, Greiner S. Geseq–versatile and accurate annotation of organelle genomes. Nucleic Acids Res. 2017;45(W1):W6-11.
</cite> [<a href="https://doi.org/10.1093/nar/gkx391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5570176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28486635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tillich%20M,%20Lehwark%20P,%20Pellizzer%20T,%20Ulbricht-Jones%20ES,%20Fischer%20A,%20Bock%20R,%20Greiner%20S.%20Geseq%E2%80%93versatile%20and%20accurate%20annotation%20of%20organelle%20genomes.%20Nucleic%20Acids%20Res.%202017;45(W1):W6-11." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Kearse M, Moir R, Wilson A, Stones-Havas S, Cheung M, Sturrock S, Buxton S, Cooper A, et al. Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics. 2012;28(12):1647–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3371832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22543367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kearse%20M,%20Moir%20R,%20Wilson%20A,%20Stones-Havas%20S,%20Cheung%20M,%20Sturrock%20S,%20Buxton%20S,%20Cooper%20A,%20et%20al.%20Geneious%20basic:%20an%20integrated%20and%20extendable%20desktop%20software%20platform%20for%20the%20organization%20and%20analysis%20of%20sequence%20data.%20Bioinformatics.%202012;28(12):1647%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Rombel IT, Sykes KF, Rayner S, Johnston SA. ORF-finder: a vector for high-throughput gene identification. Gene. 2002;282(1–2):33–41.
</cite> [<a href="https://doi.org/10.1016/s0378-1119(01)00819-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11814675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rombel%20IT,%20Sykes%20KF,%20Rayner%20S,%20Johnston%20SA.%20ORF-finder:%20a%20vector%20for%20high-throughput%20gene%20identification.%20Gene.%202002;282(1%E2%80%932):33%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Greiner S, Lehwark P, Bock R. Organellargenomedraw (OGDRAW) version 1.3.1: expanded toolkit for the graphical visualization of organellar genomes. Nucleic Acids Res. 2019;47(W1):W59-64.
</cite> [<a href="https://doi.org/10.1093/nar/gkz238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30949694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greiner%20S,%20Lehwark%20P,%20Bock%20R.%20Organellargenomedraw%20(OGDRAW)%20version%201.3.1:%20expanded%20toolkit%20for%20the%20graphical%20visualization%20of%20organellar%20genomes.%20Nucleic%20Acids%20Res.%202019;47(W1):W59-64." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Beier S, Thiel T, Münch T, Scholz U, Mascher M. MISA-web: a web server for microsatellite prediction. Bioinformatics. 2017;33(16):2583–5.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5870701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28398459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beier%20S,%20Thiel%20T,%20M%C3%BCnch%20T,%20Scholz%20U,%20Mascher%20M.%20MISA-web:%20a%20web%20server%20for%20microsatellite%20prediction.%20Bioinformatics.%202017;33(16):2583%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Benson G. Tandem repeats finder: a program to analyze DNA sequences. Nucleic Acids Res. 1999;27(2):573–80.
</cite> [<a href="https://doi.org/10.1093/nar/27.2.573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9862982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benson%20G.%20Tandem%20repeats%20finder:%20a%20program%20to%20analyze%20DNA%20sequences.%20Nucleic%20Acids%20Res.%201999;27(2):573%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Kurtz S, Choudhuri JV, Ohlebusch E, Schleiermacher C, Stoye J, Giegerich R. Reputer: the manifold applications of repeat analysis on a genomic scale. Nucleic Acids Res. 2001;29(22):4633–42.
</cite> [<a href="https://doi.org/10.1093/nar/29.22.4633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC92531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11713313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kurtz%20S,%20Choudhuri%20JV,%20Ohlebusch%20E,%20Schleiermacher%20C,%20Stoye%20J,%20Giegerich%20R.%20Reputer:%20the%20manifold%20applications%20of%20repeat%20analysis%20on%20a%20genomic%20scale.%20Nucleic%20Acids%20Res.%202001;29(22):4633%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Wright F. The ‘effective number of codons’ used in a gene. Gene. 1990;87(1):23–9.
</cite> [<a href="https://doi.org/10.1016/0378-1119(90)90491-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2110097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wright%20F.%20The%20%E2%80%98effective%20number%20of%20codons%E2%80%99%20used%20in%20a%20gene.%20Gene.%201990;87(1):23%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Edera AA, Small I, Milone DH, Sanchez-Puerta MV. Deepred-Mt: deep representation learning for predicting C-to-U RNA editing in plant mitochondria. Comput Biol Med. 2021;136: 104682.
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2021.104682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34343887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edera%20AA,%20Small%20I,%20Milone%20DH,%20Sanchez-Puerta%20MV.%20Deepred-Mt:%20deep%20representation%20learning%20for%20predicting%20C-to-U%20RNA%20editing%20in%20plant%20mitochondria.%20Comput%20Biol%20Med.%202021;136:%20104682." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Camacho C, Coulouris G, Avagyan V, Ma N, Papadopoulos J, Bealer L, Madden TL. BLAST+: architecture and applications. BMC Bioinformatics. 2009;10: 421.
</cite> [<a href="https://doi.org/10.1186/1471-2105-10-421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20003500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camacho%20C,%20Coulouris%20G,%20Avagyan%20V,%20Ma%20N,%20Papadopoulos%20J,%20Bealer%20L,%20Madden%20TL.%20BLAST+:%20architecture%20and%20applications.%20BMC%20Bioinformatics.%202009;10:%20421." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Chen CJ, Wu Y, Li JW, Wang X, Zeng ZH, Xu J, Liu YL, Feng JT, et al. TBtools-II: a one for all, all for one bioinformatics platform for biological big-data mining. Mol Plant. 2023;16(11):1733–42.
</cite> [<a href="https://doi.org/10.1016/j.molp.2023.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37740491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20CJ,%20Wu%20Y,%20Li%20JW,%20Wang%20X,%20Zeng%20ZH,%20Xu%20J,%20Liu%20YL,%20Feng%20JT,%20et%20al.%20TBtools-II:%20a%20one%20for%20all,%20all%20for%20one%20bioinformatics%20platform%20for%20biological%20big-data%20mining.%20Mol%20Plant.%202023;16(11):1733%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>He WM, Yang J, Jing Y, Xu L, Yu K, Fang XD. NGenomeSyn: an easy-to-use and flexible tool for publication-ready visualization of syntenic relationships across multiple genomes. Bioinformatics. 2023;39(3): btad121.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10027429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36883694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20WM,%20Yang%20J,%20Jing%20Y,%20Xu%20L,%20Yu%20K,%20Fang%20XD.%20NGenomeSyn:%20an%20easy-to-use%20and%20flexible%20tool%20for%20publication-ready%20visualization%20of%20syntenic%20relationships%20across%20multiple%20genomes.%20Bioinformatics.%202023;39(3):%20btad121." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol. 2013;30(4):772–80.
</cite> [<a href="https://doi.org/10.1093/molbev/mst010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23329690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katoh%20K,%20Standley%20DM.%20MAFFT%20multiple%20sequence%20alignment%20software%20version%207:%20improvements%20in%20performance%20and%20usability.%20Mol%20Biol%20Evol.%202013;30(4):772%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Rozas J, Ferrer-Mata A, Sánchez-DelBarrio JC, Guirao-Rico S, Librado P, Ramos-Onsins SE, Sánchez-Gracia A. DnaSP 6: DNA sequence polymorphism analysis of large data sets. Mol Biol Evol. 2017;34(12):3299–302.
</cite> [<a href="https://doi.org/10.1093/molbev/msx248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29029172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rozas%20J,%20Ferrer-Mata%20A,%20S%C3%A1nchez-DelBarrio%20JC,%20Guirao-Rico%20S,%20Librado%20P,%20Ramos-Onsins%20SE,%20S%C3%A1nchez-Gracia%20A.%20DnaSP%206:%20DNA%20sequence%20polymorphism%20analysis%20of%20large%20data%20sets.%20Mol%20Biol%20Evol.%202017;34(12):3299%E2%80%93302." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Zhang Z. KaKs_Calculator 3.0: calculating selective pressure on coding and non-coding sequences. Genom Proteom Bioinf. 2022;20(3):536–40.</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.12.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9801026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34990803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Z.%20KaKs_Calculator%203.0:%20calculating%20selective%20pressure%20on%20coding%20and%20non-coding%20sequences.%20Genom%20Proteom%20Bioinf.%202022;20(3):536%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Yong YS, Hu SX, Zhong MY, Wen Y, Zhou Y, Ma RX, Jiang XY, Zhang QS. Horizontal gene transfer from Chloroplast to mitochondria of seagrasses in the yellow-Bohai seas. Genomics. 2024;116(5): 110940.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2024.110940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39303860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yong%20YS,%20Hu%20SX,%20Zhong%20MY,%20Wen%20Y,%20Zhou%20Y,%20Ma%20RX,%20Jiang%20XY,%20Zhang%20QS.%20Horizontal%20gene%20transfer%20from%20Chloroplast%20to%20mitochondria%20of%20seagrasses%20in%20the%20yellow-Bohai%20seas.%20Genomics.%202024;116(5):%20110940." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Xiang CY, Gao FL, Jakovlić I, Lei HP, Hu Y, Zhang H, Zou H, Wang GT, et al. Using phylosuite for molecular phylogeny and tree-based analyses. Imeta. 2023;2(1): e87.
</cite> [<a href="https://doi.org/10.1002/imt2.87" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiang%20CY,%20Gao%20FL,%20Jakovli%C4%87%20I,%20Lei%20HP,%20Hu%20Y,%20Zhang%20H,%20Zou%20H,%20Wang%20GT,%20et%20al.%20Using%20phylosuite%20for%20molecular%20phylogeny%20and%20tree-based%20analyses.%20Imeta.%202023;2(1):%20e87." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Minh BQ, Schmidt HA, Chernomor O, Schrempf D, Woodhams MD, von Haeseler A, Lanfear R. IQ-tree 2: new models and efficient methods for phylogenetic inference in the genomic era. Mol Biol Evol. 2020;37(5):1530–4.
</cite> [<a href="https://doi.org/10.1093/molbev/msaa015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Minh%20BQ,%20Schmidt%20HA,%20Chernomor%20O,%20Schrempf%20D,%20Woodhams%20MD,%20von%20Haeseler%20A,%20Lanfear%20R.%20IQ-tree%202:%20new%20models%20and%20efficient%20methods%20for%20phylogenetic%20inference%20in%20the%20genomic%20era.%20Mol%20Biol%20Evol.%202020;37(5):1530%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Guo HL, Liu Q, Chen Y, Niu HY, Zhao QR, Song H, Pang RD, Huang XL, et al. Comprehensive assembly and comparative examination of the full mitochondrial genome in <em>Castanea mollissima</em> Blume. Genomics. 2023;115(6): 110740.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2023.110740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37923179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20HL,%20Liu%20Q,%20Chen%20Y,%20Niu%20HY,%20Zhao%20QR,%20Song%20H,%20Pang%20RD,%20Huang%20XL,%20et%20al.%20Comprehensive%20assembly%20and%20comparative%20examination%20of%20the%20full%20mitochondrial%20genome%20in%20Castanea%20mollissima%20Blume.%20Genomics.%202023;115(6):%20110740." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Mader M, Schroeder H, Schott T, Schöning-Stierand K, Leite Montalvão AP, Liesebach H, Liesebach M, Fussi B, et al. Mitochondrial genome of <em>Fagus sylvatica</em> L. as a source for taxonomic marker development in the Fagales. Plants (Basel). 2020;9(10): 1274.
</cite> [<a href="https://doi.org/10.3390/plants9101274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7650814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32992588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mader%20M,%20Schroeder%20H,%20Schott%20T,%20Sch%C3%B6ning-Stierand%20K,%20Leite%20Montalv%C3%A3o%20AP,%20Liesebach%20H,%20Liesebach%20M,%20Fussi%20B,%20et%20al.%20Mitochondrial%20genome%20of%20Fagus%20sylvatica%20L.%20as%20a%20source%20for%20taxonomic%20marker%20development%20in%20the%20Fagales.%20Plants%20(Basel).%202020;9(10):%201274." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Mishra B, Ulaszewski B, Meger J, Ploch S, Burczyk J, Thines M. A comparison of three circular mitochondrial genomes of <em>Fagus sylvatica</em> from Germany and Poland reveals low variation and complete identity of the gene space. Forests. 2021;12(5): 12:571.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mishra%20B,%20Ulaszewski%20B,%20Meger%20J,%20Ploch%20S,%20Burczyk%20J,%20Thines%20M.%20A%20comparison%20of%20three%20circular%20mitochondrial%20genomes%20of%20Fagus%20sylvatica%20from%20Germany%20and%20Poland%20reveals%20low%20variation%20and%20complete%20identity%20of%20the%20gene%20space.%20Forests.%202021;12(5):%2012:571." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Qiu XY, Tian YQ, Li ZQ, Wu XJ, Xiang ZW, Wang YQ, Li J, Xiao SG. Assembly and characterization analysis of the complete mitochondrial genome of <em>Lithocarpus litseifolius</em> (Hance) Chun. Genet Resour Crop Evol. 2025;72(1):295–313.</cite>
</li>
<li id="CR79">
<span class="label">79.</span><cite>Tu XD, Xin YX, Fu HH, Zhou CY, Liu QL, Tang XH, Zou LH, Liu ZJ, et al. The complete mitochondrial genome of <em>Castanopsis carlesii</em> and <em>Castanea henryi</em> reveals the rearrangement and size differences of mitochondrial DNA molecules. BMC Plant Biol. 2024;24(1):988.
</cite> [<a href="https://doi.org/10.1186/s12870-024-05618-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11492686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39428457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tu%20XD,%20Xin%20YX,%20Fu%20HH,%20Zhou%20CY,%20Liu%20QL,%20Tang%20XH,%20Zou%20LH,%20Liu%20ZJ,%20et%20al.%20The%20complete%20mitochondrial%20genome%20of%20Castanopsis%20carlesii%20and%20Castanea%20henryi%20reveals%20the%20rearrangement%20and%20size%20differences%20of%20mitochondrial%20DNA%20molecules.%20BMC%20Plant%20Biol.%202024;24(1):988." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Han Y, Feng YL, Wang J, Zhu SS, Jin XJ, Wu ZQ, Zhang YH. Comprehensive analysis of the complete mitochondrial genome of <em>Rehmannia chingii</em>: an autotrophic species in the Orobanchaceae family. Genes. 2024;15(1): 98.
</cite> [<a href="https://doi.org/10.3390/genes15010098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10815111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38254987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20Y,%20Feng%20YL,%20Wang%20J,%20Zhu%20SS,%20Jin%20XJ,%20Wu%20ZQ,%20Zhang%20YH.%20Comprehensive%20analysis%20of%20the%20complete%20mitochondrial%20genome%20of%20Rehmannia%20chingii:%20an%20autotrophic%20species%20in%20the%20Orobanchaceae%20family.%20Genes.%202024;15(1):%2098." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Mower JP, Case AL, Floro ER, Willis JH. Evidence against equimolarity of large repeat arrangements and a predominant master circle structure of the mitochondrial genome from a Monkeyflower (<em>Mimulus guttatus</em>) lineage with cryptic CMS. Genome Biol Evol. 2012;4(5):670–86.
</cite> [<a href="https://doi.org/10.1093/gbe/evs042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3381676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22534162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mower%20JP,%20Case%20AL,%20Floro%20ER,%20Willis%20JH.%20Evidence%20against%20equimolarity%20of%20large%20repeat%20arrangements%20and%20a%20predominant%20master%20circle%20structure%20of%20the%20mitochondrial%20genome%20from%20a%20Monkeyflower%20(Mimulus%20guttatus)%20lineage%20with%20cryptic%20CMS.%20Genome%20Biol%20Evol.%202012;4(5):670%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Wynn EL, Christensen AC. Repeats of unusual size in plant mitochondrial genomes: identification, incidence and evolution. G3 Genes|Genomes|Genetics. 2019;9(2):549–59.
</cite> [<a href="https://doi.org/10.1534/g3.118.200948" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6385970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30563833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wynn%20EL,%20Christensen%20AC.%20Repeats%20of%20unusual%20size%20in%20plant%20mitochondrial%20genomes:%20identification,%20incidence%20and%20evolution.%20G3%20Genes%7CGenomes%7CGenetics.%202019;9(2):549%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Xia CY, Li JL, Zuo YW, He P, Zhang H, Zhang XX, Wang BR, Zhang JB, et al. Complete mitochondrial genome of <em>Thuja sutchuenensis</em> and its implications on evolutionary analysis of complex mitogenome architecture in Cupressaceae. BMC Plant Biol. 2023;23(1):84.
</cite> [<a href="https://doi.org/10.1186/s12870-023-04054-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9903464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36750935/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia%20CY,%20Li%20JL,%20Zuo%20YW,%20He%20P,%20Zhang%20H,%20Zhang%20XX,%20Wang%20BR,%20Zhang%20JB,%20et%20al.%20Complete%20mitochondrial%20genome%20of%20Thuja%20sutchuenensis%20and%20its%20implications%20on%20evolutionary%20analysis%20of%20complex%20mitogenome%20architecture%20in%20Cupressaceae.%20BMC%20Plant%20Biol.%202023;23(1):84." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Ni Y, Li JL, Chen HM, Yue JW, Chen PH, Liu C. Comparative analysis of the chloroplast and mitochondrial genomes of <em>Saposhnikovia divaricata</em> revealed the possible transfer of plastome repeat regions into the mitogenome. BMC Genomics. 2022;23(1):570.
</cite> [<a href="https://doi.org/10.1186/s12864-022-08821-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9364500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35945507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ni%20Y,%20Li%20JL,%20Chen%20HM,%20Yue%20JW,%20Chen%20PH,%20Liu%20C.%20Comparative%20analysis%20of%20the%20chloroplast%20and%20mitochondrial%20genomes%20of%20Saposhnikovia%20divaricata%20revealed%20the%20possible%20transfer%20of%20plastome%20repeat%20regions%20into%20the%20mitogenome.%20BMC%20Genomics.%202022;23(1):570." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Cadorna CAE, Pahayo DG, Rey JD. The first mitochondrial genome of <em>Calophyllum soulattri</em> burm.f. Sci Rep. 2024;14(1):5112.
</cite> [<a href="https://doi.org/10.1038/s41598-024-55016-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10907642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38429360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cadorna%20CAE,%20Pahayo%20DG,%20Rey%20JD.%20The%20first%20mitochondrial%20genome%20of%20Calophyllum%20soulattri%20burm.f.%20Sci%20Rep.%202024;14(1):5112." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Sloan DB, Alverson AJ, Chuckalovcak JP, Wu M, McCauley DE, Palmer JD, Taylor DR. Rapid evolution of enormous, multichromosomal genomes in flowering plant mitochondria with exceptionally high mutation rates. PLoS Biol. 2012;10(1):e1001241.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1001241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3260318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22272183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sloan%20DB,%20Alverson%20AJ,%20Chuckalovcak%20JP,%20Wu%20M,%20McCauley%20DE,%20Palmer%20JD,%20Taylor%20DR.%20Rapid%20evolution%20of%20enormous,%20multichromosomal%20genomes%20in%20flowering%20plant%20mitochondria%20with%20exceptionally%20high%20mutation%20rates.%20PLoS%20Biol.%202012;10(1):e1001241." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Binder S, Brennicke A. Gene expression in plant mitochondria: transcriptional and post-transcriptional control. Phil Trans R Soc Lond B. 2023;358(1429):181–9.</cite> [<a href="https://doi.org/10.1098/rstb.2002.1179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1693100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12594926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Binder%20S,%20Brennicke%20A.%20Gene%20expression%20in%20plant%20mitochondria:%20transcriptional%20and%20post-transcriptional%20control.%20Phil%20Trans%20R%20Soc%20Lond%20B.%202023;358(1429):181%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Hao W, Liu GX, Wang WP, Shen W, Zhao YP, Sun JL, Yang QY, Zhang YX, et al. RNA editing and its roles in plant organelles. Front Genet. 2021;12: 757109.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.757109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8511385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34659369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao%20W,%20Liu%20GX,%20Wang%20WP,%20Shen%20W,%20Zhao%20YP,%20Sun%20JL,%20Yang%20QY,%20Zhang%20YX,%20et%20al.%20RNA%20editing%20and%20its%20roles%20in%20plant%20organelles.%20Front%20Genet.%202021;12:%20757109." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Small ID, Schallenberg-Rüdinger M, Takenaka M, Mireau H, Ostersetzer-Biran O. Plant organellar RNA editing: what 30 years of research has revealed. Plant J. 2020;101(5):1040–56.
</cite> [<a href="https://doi.org/10.1111/tpj.14578" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31630458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Small%20ID,%20Schallenberg-R%C3%BCdinger%20M,%20Takenaka%20M,%20Mireau%20H,%20Ostersetzer-Biran%20O.%20Plant%20organellar%20RNA%20editing:%20what%2030%20years%20of%20research%20has%20revealed.%20Plant%20J.%202020;101(5):1040%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Ichinose M, Sugita M. RNA editing and its molecular mechanism in plant organelles. Genes (Basel). 2017;8(1):5.</cite> [<a href="https://doi.org/10.3390/genes8010005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5295000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28025543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ichinose%20M,%20Sugita%20M.%20RNA%20editing%20and%20its%20molecular%20mechanism%20in%20plant%20organelles.%20Genes%20(Basel).%202017;8(1):5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Guo LJ, Lao GR, He LF, Xiao D, Zhan J, Wang AQ. De novo assembly and comparative analysis of mitochondrial genomes of two <em>Pueraria montana</em> varieties. Int J Mol Sci. 2024;25(11): 5656.
</cite> [<a href="https://doi.org/10.3390/ijms25115656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11171644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38891844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20LJ,%20Lao%20GR,%20He%20LF,%20Xiao%20D,%20Zhan%20J,%20Wang%20AQ.%20De%20novo%20assembly%20and%20comparative%20analysis%20of%20mitochondrial%20genomes%20of%20two%20Pueraria%20montana%20varieties.%20Int%20J%20Mol%20Sci.%202024;25(11):%205656." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Liu XY, Zhang DD, Yu ZF, Zeng B. Assembly and analysis of the complete mitochondrial genome of the Chinese wild dwarf almond (<em>Prunus tenella</em>). Front Genet. 2024;14: 1329060.
</cite> [<a href="https://doi.org/10.3389/fgene.2023.1329060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10811783/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38283144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20XY,%20Zhang%20DD,%20Yu%20ZF,%20Zeng%20B.%20Assembly%20and%20analysis%20of%20the%20complete%20mitochondrial%20genome%20of%20the%20Chinese%20wild%20dwarf%20almond%20(Prunus%20tenella).%20Front%20Genet.%202024;14:%201329060." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Li GL, Zhang H, Lin ZM, Li HW, Xu GC, Xu YQ, Ji RC, Luo WB, et al. Comparative analysis of chloroplast and mitochondrial genomes of sweet potato provides evidence of gene transfer. Sci Rep. 2024;14(1):4547.
</cite> [<a href="https://doi.org/10.1038/s41598-024-55150-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10894244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38402284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20GL,%20Zhang%20H,%20Lin%20ZM,%20Li%20HW,%20Xu%20GC,%20Xu%20YQ,%20Ji%20RC,%20Luo%20WB,%20et%20al.%20Comparative%20analysis%20of%20chloroplast%20and%20mitochondrial%20genomes%20of%20sweet%20potato%20provides%20evidence%20of%20gene%20transfer.%20Sci%20Rep.%202024;14(1):4547." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Liu JT, Ni Y, Liu C. Polymeric structure of the <em>Cannabis sativa</em> L. mitochondrial genome identified with an assembly graph model. Gene. 2023;853: 147081.
</cite> [<a href="https://doi.org/10.1016/j.gene.2022.147081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36470482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20JT,%20Ni%20Y,%20Liu%20C.%20Polymeric%20structure%20of%20the%20Cannabis%20sativa%20L.%20mitochondrial%20genome%20identified%20with%20an%20assembly%20graph%20model.%20Gene.%202023;853:%20147081." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Tang P, Ni Y, Li JL, Lu QQ, Liu C, Guo JL. The complete mitochondrial genome of <em>Paeonia lactiflora</em> Pall. (Saxifragales: Paeoniaceae): evidence of gene transfer from chloroplast to mitochondrial genome. Genes. 2024;15(2): 239.
</cite> [<a href="https://doi.org/10.3390/genes15020239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10888214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38397228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20P,%20Ni%20Y,%20Li%20JL,%20Lu%20QQ,%20Liu%20C,%20Guo%20JL.%20The%20complete%20mitochondrial%20genome%20of%20Paeonia%20lactiflora%20Pall.%20(Saxifragales:%20Paeoniaceae):%20evidence%20of%20gene%20transfer%20from%20chloroplast%20to%20mitochondrial%20genome.%20Genes.%202024;15(2):%20239." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Bi CW, Paterson AH, Wang XL, Xu YQ, Wu DY, Qu YS, Jiang AN, Ye QL, et al. Analysis of the complete mitochondrial genome sequence of the diploid cotton <em>Gossypium raimondii</em> by comparative genomics approaches. Biomed Res Int. 2016;2016:5040598.
</cite> [<a href="https://doi.org/10.1155/2016/5040598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5099484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27847816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bi%20CW,%20Paterson%20AH,%20Wang%20XL,%20Xu%20YQ,%20Wu%20DY,%20Qu%20YS,%20Jiang%20AN,%20Ye%20QL,%20et%20al.%20Analysis%20of%20the%20complete%20mitochondrial%20genome%20sequence%20of%20the%20diploid%20cotton%20Gossypium%20raimondii%20by%20comparative%20genomics%20approaches.%20Biomed%20Res%20Int.%202016;2016:5040598." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Portugez S, Martin WF, Hazkani-Covo E. Mosaic mitochondrial-plastid insertions into the nuclear genome show evidence of both non-homologous end joining and homologous recombination. BMC Evol Biol. 2018;18(1):162–71.
</cite> [<a href="https://doi.org/10.1186/s12862-018-1279-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6215612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30390623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Portugez%20S,%20Martin%20WF,%20Hazkani-Covo%20E.%20Mosaic%20mitochondrial-plastid%20insertions%20into%20the%20nuclear%20genome%20show%20evidence%20of%20both%20non-homologous%20end%20joining%20and%20homologous%20recombination.%20BMC%20Evol%20Biol.%202018;18(1):162%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Kleine T, Maier UG, Leister D. DNA transfer from organelles to the nucleus: the idiosyncratic genetics of endosymbiosis. Annu Rev Plant Biol. 2009;60:115–38.
</cite> [<a href="https://doi.org/10.1146/annurev.arplant.043008.092119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19014347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kleine%20T,%20Maier%20UG,%20Leister%20D.%20DNA%20transfer%20from%20organelles%20to%20the%20nucleus:%20the%20idiosyncratic%20genetics%20of%20endosymbiosis.%20Annu%20Rev%20Plant%20Biol.%202009;60:115%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Huang CY, Ayliffe MA, Timmis JN. Direct measurement of the transfer rate of Chloroplast DNA into the nucleus. Nature. 2003;422:72–6.
</cite> [<a href="https://doi.org/10.1038/nature01435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12594458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20CY,%20Ayliffe%20MA,%20Timmis%20JN.%20Direct%20measurement%20of%20the%20transfer%20rate%20of%20Chloroplast%20DNA%20into%20the%20nucleus.%20Nature.%202003;422:72%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Kubo T, Mikami T. Organization and variation of angiosperm mitochondrial genome. Physiol Plant. 2007;129:6–13.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kubo%20T,%20Mikami%20T.%20Organization%20and%20variation%20of%20angiosperm%20mitochondrial%20genome.%20Physiol%20Plant.%202007;129:6%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Palmer JD, Herbon LA. Plant mitochondrial DNA evolves rapidly in structure, but slowly in sequence. J Mol Evol. 1988;28(1–2):87–97.
</cite> [<a href="https://doi.org/10.1007/BF02143500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3148746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palmer%20JD,%20Herbon%20LA.%20Plant%20mitochondrial%20DNA%20evolves%20rapidly%20in%20structure,%20but%20slowly%20in%20sequence.%20J%20Mol%20Evol.%201988;28(1%E2%80%932):87%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Bi CW, Lu N, Xu YQ, He CP, Lu ZH. Characterization and analysis of the mitochondrial genome of common bean (<em>Phaseolus vulgaris</em>) by comparative genomic approaches. Int J Mol Sci. 2020;21(11): 3778.
</cite> [<a href="https://doi.org/10.3390/ijms21113778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7312688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32471098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bi%20CW,%20Lu%20N,%20Xu%20YQ,%20He%20CP,%20Lu%20ZH.%20Characterization%20and%20analysis%20of%20the%20mitochondrial%20genome%20of%20common%20bean%20(Phaseolus%20vulgaris)%20by%20comparative%20genomic%20approaches.%20Int%20J%20Mol%20Sci.%202020;21(11):%203778." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Wu ZQ, Liao XZ, Zhang XN, Tembrock LR, Broz A. Genomic architectural variation of plant mitochondria—a review of multichromosomal structuring. J Syst Evol. 2020;60(1):160–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wu%20ZQ,%20Liao%20XZ,%20Zhang%20XN,%20Tembrock%20LR,%20Broz%20A.%20Genomic%20architectural%20variation%20of%20plant%20mitochondria%E2%80%94a%20review%20of%20multichromosomal%20structuring.%20J%20Syst%20Evol.%202020;60(1):160%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Wu ZQ, Stone JD, Štorchová H, Sloan DB. High transcript abundance, RNA editing, and small RNAs in intergenic regions within the massive mitochondrial genome of the angiosperm <em>Silene noctiflora</em>. BMC Genomics. 2015;16(1): 938.
</cite> [<a href="https://doi.org/10.1186/s12864-015-2155-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4647634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26573088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20ZQ,%20Stone%20JD,%20%C5%A0torchov%C3%A1%20H,%20Sloan%20DB.%20High%20transcript%20abundance,%20RNA%20editing,%20and%20small%20RNAs%20in%20intergenic%20regions%20within%20the%20massive%20mitochondrial%20genome%20of%20the%20angiosperm%20Silene%20noctiflora.%20BMC%20Genomics.%202015;16(1):%20938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Liu X, You Q, Liu MM, Bo C, Zhu YF, Duan YB, Xue JP, Wang DX, et al. Assembly and comparative analysis of the complete mitochondrial genome of <em>Pinellia ternata</em>. Funct Plant Biol. 2024;51(2): FP23256.
</cite> [<a href="https://doi.org/10.1071/FP23256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38316513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20X,%20You%20Q,%20Liu%20MM,%20Bo%20C,%20Zhu%20YF,%20Duan%20YB,%20Xue%20JP,%20Wang%20DX,%20et%20al.%20Assembly%20and%20comparative%20analysis%20of%20the%20complete%20mitochondrial%20genome%20of%20Pinellia%20ternata.%20Funct%20Plant%20Biol.%202024;51(2):%20FP23256." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Yu XX, Feng YL, Zhang J. Characterization of the complete mitochondrial genome of wintersweet (<em>Chimonanthus praecox</em>) and comparative analysis within magnoliids. Life. 2024;14(2): 182.
</cite> [<a href="https://doi.org/10.3390/life14020182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10890521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38398691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20XX,%20Feng%20YL,%20Zhang%20J.%20Characterization%20of%20the%20complete%20mitochondrial%20genome%20of%20wintersweet%20(Chimonanthus%20praecox)%20and%20comparative%20analysis%20within%20magnoliids.%20Life.%202024;14(2):%20182." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Kozik A, Rowan BA, Lavelle D, Berke L, Schranz ME, Michelmore RW, Christensen AC. The alternative reality of plant mitochondrial DNA: one ring does not rule them all. PLoS Genet. 2019;15(8): e1008373.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6742443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31469821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozik%20A,%20Rowan%20BA,%20Lavelle%20D,%20Berke%20L,%20Schranz%20ME,%20Michelmore%20RW,%20Christensen%20AC.%20The%20alternative%20reality%20of%20plant%20mitochondrial%20DNA:%20one%20ring%20does%20not%20rule%20them%20all.%20PLoS%20Genet.%202019;15(8):%20e1008373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Jackman SD, Coombe L, Warren RL, Kirk H, Trinh E, MacLeod T, Pleasance S, Pandoh P, et al. Complete mitochondrial genome of a gymnosperm, Sitka spruce (<em>Picea sitchensis</em>), indicates a complex physical structure. Genome Biol Evol. 2020;12(7):1174–9.
</cite> [<a href="https://doi.org/10.1093/gbe/evaa108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7486957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32449750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackman%20SD,%20Coombe%20L,%20Warren%20RL,%20Kirk%20H,%20Trinh%20E,%20MacLeod%20T,%20Pleasance%20S,%20Pandoh%20P,%20et%20al.%20Complete%20mitochondrial%20genome%20of%20a%20gymnosperm,%20Sitka%20spruce%20(Picea%20sitchensis),%20indicates%20a%20complex%20physical%20structure.%20Genome%20Biol%20Evol.%202020;12(7):1174%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Guo S, Li ZY, Li CL, Liu Y, Liang XL, Qin YM. Assembly and characterization of the complete mitochondrial genome of <em>Ventilago Leiocarpa</em>. Plant Cell Rep. 2024;43(3):77.
</cite> [<a href="https://doi.org/10.1007/s00299-023-03126-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38386216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20S,%20Li%20ZY,%20Li%20CL,%20Liu%20Y,%20Liang%20XL,%20Qin%20YM.%20Assembly%20and%20characterization%20of%20the%20complete%20mitochondrial%20genome%20of%20Ventilago%20Leiocarpa.%20Plant%20Cell%20Rep.%202024;43(3):77." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Shan YY, Li JL, Zhang X, Yu J. The complete mitochondrial genome of <em>Amorphophallus albus</em> and development of molecular markers for five <em>Amorphophallus</em> species based on mitochondrial DNA. Front Plant Sci. 2023;14: 1180417.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1180417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10322194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37416891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shan%20YY,%20Li%20JL,%20Zhang%20X,%20Yu%20J.%20The%20complete%20mitochondrial%20genome%20of%20Amorphophallus%20albus%20and%20development%20of%20molecular%20markers%20for%20five%20Amorphophallus%20species%20based%20on%20mitochondrial%20DNA.%20Front%20Plant%20Sci.%202023;14:%201180417." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Wang L, Liu X, Xu YJ, Zhang ZW, Wei YS, Hu Y, Zheng CB, Qu XY. Assembly and comparative analysis of the first complete mitochondrial genome of a traditional Chinese medicine <em>Angelica biserrata</em> (Shan et Yuan) Yuan et Shan. Int J Biol Macromol. 2024;257(1):128571.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2023.128571" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38052286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20L,%20Liu%20X,%20Xu%20YJ,%20Zhang%20ZW,%20Wei%20YS,%20Hu%20Y,%20Zheng%20CB,%20Qu%20XY.%20Assembly%20and%20comparative%20analysis%20of%20the%20first%20complete%20mitochondrial%20genome%20of%20a%20traditional%20Chinese%20medicine%20Angelica%20biserrata%20(Shan%20et%20Yuan)%20Yuan%20et%20Shan.%20Int%20J%20Biol%20Macromol.%202024;257(1):128571." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Szandar K, Krawczyk K, Myszczyński K, Ślipiko M, Sawicki J, Szczecińska M. Breaking the limits - multichromosomal structure of an early eudicot <em>Pulsatilla patens</em> mitogenome reveals extensive RNA-editing, longest repeats and Chloroplast derived regions among sequenced land plant mitogenomes. BMC Plant Biol. 2022;22(1): 109.
</cite> [<a href="https://doi.org/10.1186/s12870-022-03492-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8905907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35264098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szandar%20K,%20Krawczyk%20K,%20Myszczy%C5%84ski%20K,%20%C5%9Alipiko%20M,%20Sawicki%20J,%20Szczeci%C5%84ska%20M.%20Breaking%20the%20limits%20-%20multichromosomal%20structure%20of%20an%20early%20eudicot%20Pulsatilla%20patens%20mitogenome%20reveals%20extensive%20RNA-editing,%20longest%20repeats%20and%20Chloroplast%20derived%20regions%20among%20sequenced%20land%20plant%20mitogenomes.%20BMC%20Plant%20Biol.%202022;22(1):%20109." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Hao ZG, Zhang ZP, Zhang JN, Cui XF, Li JQ, Luo LX, Li YB. The complete mitochondrial genome of <em>Aglaia odorata</em>, insights into its genomic structure and RNA editing sites. Front Plant Sci. 2024;15: 1362045.
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1362045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10950942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38510436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao%20ZG,%20Zhang%20ZP,%20Zhang%20JN,%20Cui%20XF,%20Li%20JQ,%20Luo%20LX,%20Li%20YB.%20The%20complete%20mitochondrial%20genome%20of%20Aglaia%20odorata,%20insights%20into%20its%20genomic%20structure%20and%20RNA%20editing%20sites.%20Front%20Plant%20Sci.%202024;15:%201362045." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Lu GL, Zhang K, Que YX, Li YF. Assembly and analysis of the first complete mitochondrial genome of <em>Punica granatum</em> and the gene transfer from Chloroplast genome. Front Plant Sci. 2023;14: 1132551.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1132551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37416882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20GL,%20Zhang%20K,%20Que%20YX,%20Li%20YF.%20Assembly%20and%20analysis%20of%20the%20first%20complete%20mitochondrial%20genome%20of%20Punica%20granatum%20and%20the%20gene%20transfer%20from%20Chloroplast%20genome.%20Front%20Plant%20Sci.%202023;14:%201132551." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Hurst LD. The ka/ks ratio: diagnosing the form of sequence evolution. Trends Genet. 2002;18(9):486.
</cite> [<a href="https://doi.org/10.1016/s0168-9525(02)02722-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12175810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hurst%20LD.%20The%20ka/ks%20ratio:%20diagnosing%20the%20form%20of%20sequence%20evolution.%20Trends%20Genet.%202002;18(9):486." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Zancani M, Braidot E, Filippi A, Lippe G. Structural and functional properties of plant mitochondrial F-ATP synthase. Mitochondrion. 2020;53:178–93.
</cite> [<a href="https://doi.org/10.1016/j.mito.2020.06.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32534049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zancani%20M,%20Braidot%20E,%20Filippi%20A,%20Lippe%20G.%20Structural%20and%20functional%20properties%20of%20plant%20mitochondrial%20F-ATP%20synthase.%20Mitochondrion.%202020;53:178%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Li HT, Luo Y, Gan L, Ma PF, Gao LM, Yang JB, Cai J, Gitzendanner MA, et al. Plastid phylogenomic insights into relationships of all flowering plant families. BMC Biol. 2021;19(1): 232.
</cite> [<a href="https://doi.org/10.1186/s12915-021-01166-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8555322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34711223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20HT,%20Luo%20Y,%20Gan%20L,%20Ma%20PF,%20Gao%20LM,%20Yang%20JB,%20Cai%20J,%20Gitzendanner%20MA,%20et%20al.%20Plastid%20phylogenomic%20insights%20into%20relationships%20of%20all%20flowering%20plant%20families.%20BMC%20Biol.%202021;19(1):%20232." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>The Angiosperm Phylogeny Group, Chase MW, Christenhusz MJM, Fay MF, Byng JW, Judd WS, et al. An update of the angiosperm phylogeny group classification for the orders and families of flowering plants: APG IV. Bot J Linn Soc. 2016;181(1):1–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?The%20Angiosperm%20Phylogeny%20Group,%20Chase%20MW,%20Christenhusz%20MJM,%20Fay%20MF,%20Byng%20JW,%20Judd%20WS,%20et%20al.%20An%20update%20of%20the%20angiosperm%20phylogeny%20group%20classification%20for%20the%20orders%20and%20families%20of%20flowering%20plants:%20APG%20IV.%20Bot%20J%20Linn%20Soc.%202016;181(1):1%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Deng M, Jiang XL, Hipp AL, Manos PS, Hahn M. Phylogeny and biogeography of East Asian evergreen oaks (<em>Quercus</em> section <em>Cyclobalanopsis</em>; Fagaceae): insights into the Cenozoic history of evergreen broad-leaved forests in subtropical Asia. Mol Phylogenet Evol. 2018;119:170–81.
</cite> [<a href="https://doi.org/10.1016/j.ympev.2017.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29175095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deng%20M,%20Jiang%20XL,%20Hipp%20AL,%20Manos%20PS,%20Hahn%20M.%20Phylogeny%20and%20biogeography%20of%20East%20Asian%20evergreen%20oaks%20(Quercus%20section%20Cyclobalanopsis;%20Fagaceae):%20insights%20into%20the%20Cenozoic%20history%20of%20evergreen%20broad-leaved%20forests%20in%20subtropical%20Asia.%20Mol%20Phylogenet%20Evol.%202018;119:170%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Ortego J, Gugger PF, Riordan EC, Sork VL. Influence of climatic niche suitability and geographical overlap on hybridization patterns among Southern Californian oaks. J Biogeogr. 2014;41:1895–908.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ortego%20J,%20Gugger%20PF,%20Riordan%20EC,%20Sork%20VL.%20Influence%20of%20climatic%20niche%20suitability%20and%20geographical%20overlap%20on%20hybridization%20patterns%20among%20Southern%20Californian%20oaks.%20J%20Biogeogr.%202014;41:1895%E2%80%93908." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Meng HH, Song YG, Hu GX, Huang PH, Li M, Fang OY, Su RP, Cao GL, et al. Evolution of East Asian subtropical evergreen broad-leaved forests: when and how. J Syst Evol. 2025. 10.1111/jse.70001.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12363117/bin/12870_2025_7072_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7072_MOESM1_ESM.docx</a><sup> (4MB, docx) </sup><p>Supplementary Material 1: Figure S1. Structural graph comparison of the <em>Q. gilva</em> mitogenome assembled using different tools. (A) Mitochondrial genome graph assembled with GetOrganelle v1.7.6.1, with a total length of 388,738 bp; (B) Mitochondrial genome graph assembled with Unicycler v0.5.1, with a total length of 479,114 bp. Figure S2. Schematic representation of the assembly results of the <em>Q. gilva</em> mitogenome. (A) The final MMG graph assembled using the GSAT software; (B) The circular diagram of all contigs, with the green and blue rings representing the coverage depths of short and long reads, respectively; (C) The multi-branched structure graph resulted from resolving the structure of (A). Figure S3. Genome map of the <em>Q. gilva </em>plastome. Genes outside the circle are transcribed in the counter-clockwise direction, whereas those inside the circle are transcribed in the clockwise direction. The genes belonging to different functional groups are identified by different colors. The length and boundary of the LSC, SSC, and the two IR regions are indicated in the inner circle. The dark gray area indicates GC content while the lighter gray corresponds to the AT content of the genomes. LSC, large single copy; SSC, small single copy; IR, inverted repeat. Figure S4. Relative synonymous codon usage (RSCU) values of the <em>Q. gilva </em>mitogenome. Figure S5. Distribution of the PCGs in the mitogenomes of eleven Fagaceae species. Red boxes indicate that the PCG is present while blue boxes indicate that the PCG is absent. PCGs are categorized into core genes and variable genes following the classification of Adams et al. (2003). PCG, protein-coding genes. Figure S6. The Pi values (A) and Ka/Ks ratio (B) of 79 common plastome PCGs among Fagaceae. The box plot represents the distribution of Ka/Ks values for all pairwise comparison in the data. Pi, nucleotide diversity; Ka/Ks, nonsynonymous and synonymous substitution ratio</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12363117/bin/12870_2025_7072_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7072_MOESM2_ESM.xlsx</a><sup> (343.8KB, xlsx) </sup><p>Supplementary Material 2: Table S1. The basic information of selected angiosperms for constructing phylogenetic trees in this study. Table S2. Assembly statistics of different graphs of the <em>Q. gilva </em>mitogenome. Table S3. Contigs of the MMGs of the<em> Q. gilva </em>mitogenome. Table S4. Basic information statistics of genes of the <em>Q. gilva</em> mitogenome. Table S5. Basic information statistics of open reading frames (ORFs) of the <em>Q. gilva</em> mitogenome. Table S6. Simple sequence repeats (SSRs) of the <em>Q. gilva</em> mitogenome. Table S7. Tandem repeat sequences (TRSs) of the <em>Q. gilva </em>mitogenome. Table S8. Dispersed repeat sequences (DRSs) of the <em>Q. gilva </em>mitogenome. Table S9. Relative synonymous codon usage (RSCU) value and codon number of 61 codons of the <em>Q. gilva </em>mitogenome. Table S10. The predicted C to U RNA editing sites in the PCGs of the <em>Q. gilva</em> mitogenome. Table S11. The homologous mitochondrial plastid transferred fragments (MTPTs) identified between the mitogenome and plastome of <em>Q. gilva</em>. Table S12. The homologous nuclear mitochondrial transferred fragments (NUMTs) identified between the mitogenome and nuclear genome of <em>Q. gilva</em>. Table S13. Basic information on the eleven assembled mitogenomes in Fagaceae (access on 2024.11.24). Table S14. Ka/Ks values for pairwise comparisons of mitogenomes across 11 Fagaceae species. Table S15. Ka/Ks values for pairwise comparisons of plastomes across 11 Fagaceae species</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12363117/bin/12870_2025_7072_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">References</a><sup> (30.3KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The assembled complete organelle genomes of <em>Quercus gilva</em> has been deposited in GenBank (NCBI), under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PQ862789:PQ862792%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ862789–PQ862792</a> for the mitogenome and <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ876385" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ876385</a> for the plastome.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Plant Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12870-025-07072-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12870_2025_Article_7072.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12363117/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12363117/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12363117%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12363117/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12363117/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12363117/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830429/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12363117/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830429/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12363117/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12363117/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="s6Gc3wuQIXb8HRdLkRzaX4H8sgEGcCvMrr83ddT0aqKAMUhbAIYnsj0rApTBzodv">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
