
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Graph neural network-based drug-drug interaction prediction - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AF058AF3143305AF050057DA42B8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365209/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Graph neural network-based drug-drug interaction prediction">
<meta name="citation_author" content="Khushnood Abbas">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Zhoukou Normal University, Henan, China">
<meta name="citation_author" content="Chen Hao">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Zhoukou Normal University, Henan, China">
<meta name="citation_author" content="Xu Yong">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Zhoukou Normal University, Henan, China">
<meta name="citation_author" content="Mohammad Kamrul Hasan">
<meta name="citation_author_institution" content="Faculty of Information Science and Technology, Universiti Kebangsaan Malaysia (UKM), Bangi, Selangor, 43600 Malaysia">
<meta name="citation_author" content="Shayla Islam">
<meta name="citation_author_institution" content="Institute of Computer Science and Digital Innovation, UCSI University, Kuala Lumpur, 56000 Malaysia">
<meta name="citation_author" content="Abdul Hadi Abd Rahman">
<meta name="citation_author_institution" content="Faculty of Information Science and Technology, Universiti Kebangsaan Malaysia (UKM), Bangi, Selangor, 43600 Malaysia">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30340">
<meta name="citation_doi" content="10.1038/s41598-025-12936-1">
<meta name="citation_pmid" content="40830547">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365209/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365209/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365209/pdf/41598_2025_Article_12936.pdf">
<meta name="description" content="With the growing variety of pharmacological compounds and the increasing need for polypharmacy, accurately predicting drug-drug interactions (DDIs) is essential to ensure both treatment efficacy and patient safety. Beneficial DDIs can enhance ...">
<meta name="og:title" content="Graph neural network-based drug-drug interaction prediction">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="With the growing variety of pharmacological compounds and the increasing need for polypharmacy, accurately predicting drug-drug interactions (DDIs) is essential to ensure both treatment efficacy and patient safety. Beneficial DDIs can enhance ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365209/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365209">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-12936-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_12936.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365209%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365209/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365209/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365209/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 19;15:30340. doi: <a href="https://doi.org/10.1038/s41598-025-12936-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-12936-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Graph neural network-based drug-drug interaction prediction</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abbas%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Khushnood Abbas</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Khushnood Abbas</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Zhoukou Normal University, Henan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abbas%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Khushnood Abbas</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hao%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chen Hao</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chen Hao</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Zhoukou Normal University, Henan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hao%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chen Hao</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yong%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xu Yong</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xu Yong</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Zhoukou Normal University, Henan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yong%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xu Yong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20MK%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mohammad Kamrul Hasan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mohammad Kamrul Hasan</span></h3>
<div class="p">
<sup>2</sup>Faculty of Information Science and Technology, Universiti Kebangsaan Malaysia (UKM), Bangi, Selangor, 43600 Malaysia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20MK%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohammad Kamrul Hasan</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islam%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shayla Islam</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shayla Islam</span></h3>
<div class="p">
<sup>3</sup>Institute of Computer Science and Digital Innovation, UCSI University, Kuala Lumpur, 56000 Malaysia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islam%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shayla Islam</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rahman%20AHA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Abdul Hadi Abd Rahman</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Abdul Hadi Abd Rahman</span></h3>
<div class="p">
<sup>2</sup>Faculty of Information Science and Technology, Universiti Kebangsaan Malaysia (UKM), Bangi, Selangor, 43600 Malaysia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rahman%20AHA%22%5BAuthor%5D" class="usa-link"><span class="name western">Abdul Hadi Abd Rahman</span></a>
</div>
</div>
<sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Computer Science and Technology, Zhoukou Normal University, Henan, China </div>
<div id="Aff2">
<sup>2</sup>Faculty of Information Science and Technology, Universiti Kebangsaan Malaysia (UKM), Bangi, Selangor, 43600 Malaysia </div>
<div id="Aff3">
<sup>3</sup>Institute of Computer Science and Digital Innovation, UCSI University, Kuala Lumpur, 56000 Malaysia </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 7; Accepted 2025 Jul 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365209  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830547/" class="usa-link">40830547</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">With the growing variety of pharmacological compounds and the increasing need for polypharmacy, accurately predicting drug-drug interactions (DDIs) is essential to ensure both treatment efficacy and patient safety. Beneficial DDIs can enhance therapeutic outcomes. In contrast, adverse interactions may result in toxicity, reduced efficacy, or even fatality. Thus, the accurate prediction of DDIs is paramount. Building on recent advancements in graph neural network (GNN) architectures, this paper extends prior research, such as the SAGE GNN model, Graph Attention Network model, and Graph Diffusion Network model, by integrating advanced techniques such as skip connections, post-processing layers, and optimized training methods. It start from basic GNN to buld more advanced models such as based on Adaptive Graph Diffusion model. Our experimental results shows based on evaluation on 3 different drug-drug interaction dataset that on some evaluation metric basic models outperforms the advanced ones. We have found that GCN with skip connections, GCN with NGNN and SAGE with NGNN give competent accuracy with other baseline models. The corresponding code and datasets used in this study are available on GitHub for reproducibility at: <a href="https://github.com/khushnood/DrugDruginteractionPredictionBasedOnGNN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/khushnood/DrugDruginteractionPredictionBasedOnGNN</a>.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Mathematics and computing, Predictive medicine</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">When multiple drugs are taken simultaneously, the activity and effectiveness of one drug can be affected by another substance, usually another drug. Drug interactions are categorized into synergism and antagonism. Synergism refers to the combined use of different drugs that enhances their therapeutic effect, while antagonism occurs when two drugs interact, leading to side effects such as reduced efficacy and adverse reactions in patients. In fact, adverse reactions caused by drug interactions are quite common. For example, in recent years, many antiallergic drugs, such as loratadine and diphenhydramine, have been combined with macrolides. Serious adverse reactions can occur after the combined use of antibiotics (e.g., erythromycin). According to relevant studies, gastrointestinal reactions and skin allergies are the most common adverse effects of various macrolide antibiotics, with incidence rates as high as 48% and 26%, respectively<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Therefore, researchers are actively focusing on developing computational approaches to identify such interactions. The figure <a href="#Fig1" class="usa-link">1</a> depicts how the drug interaction prediction problem is converted to graph ML-based prediction problem.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/93eb8a2547ad/41598_2025_12936_Fig1_HTML.jpg" loading="lazy" id="MO1" height="363" width="781" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The figure illustrates the process of converting a drug molecule into graph representations, followed by the application of a Graph Neural Network (GNN) to make predictions.</p></figcaption></figure><section id="Sec2"><h3 class="pmc_sec_title">Research background</h3>
<p id="Par3">In recent years, the use of polypharmacy–administering multiple drugs simultaneously–has become a common approach for managing complex diseases such as cancer. This strategy is particularly advantageous for older adults with multiple comorbidities, as it utilizes the synergistic effects of drug combinations to enhance treatment efficacy. However, unintended drug-drug interactions (DDIs) remain a significant concern, posing risks such as adverse side effects and drug toxicity. With the growing demand for multidrug therapies, the need for reliable DDI detection methods has become increasingly urgent. Traditional in vitro and in vivo approaches for identifying DDIs are both expensive and time-consuming, particularly given the vast number of possible drug combinations. Consequently, developing effective strategies for DDI prediction has become a priority in pharmaceutical research. Proactively identifying potential DDIs not only minimizes the occurrence of adverse drug reactions (ADRs) but also streamlines the drug development process, ultimately improving patient safety and therapeutic outcomes.</p>
<p id="Par4">Currently, the variety of drug interactions is vast, yet only a limited number of DDIs are identified through clinical trials and chemical experiments. As a result, some drug interactions remain unnoticed until they are widely used on the market. Often, only after patients experience adverse reactions and report them are these interactions discovered. Although this reactive approach does uncover harmful drug interactions, it comes at the cost of patient health and safety, making it an unsustainable solution. Therefore, there is an urgent need for more proactive approaches to predict drug interactions.</p>
<p id="Par5">In recent years, the accumulation of biological and medical data has led to the widespread application of machine learning in the prediction of DDI. Machine learning algorithms are now being used to automatically extract features from datasets, converting them into biologically meaningful vectors or graph-based data. These features are then analyzed using machine learning models to predict potential drug interactions. Currently, advances in the analysis of graph data structures have led to the development of sophisticated techniques to study graph-based data. This has drawn significant attention to the use of machine learning in this field. One notable method is the calculation of the weighted average of neighboring node information using neural networks, a technique that forms the basis of graph neural networks (GNNs). GNNs have become a powerful tool for processing graph data structures, making them highly suitable for DDI prediction.</p></section><section id="Sec3"><h3 class="pmc_sec_title">Research status</h3>
<p id="Par6">In June 2021, Zhu Jie, Wang Jingxiang, and colleagues introduced a drug efficacy prediction system (DLEPS) based on genetic fingerprinting and deep learning<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. This study employed neural networks to establish connections between compound structures and gene expression profiles, successfully predicting drug interactions and validating these predictions using real clinical datasets. In early 2022, Wang Fei, Lei Xiujuan, Liao Bo, and Wu Fangxiang introduced a multi-kernel graph convolutional network (GCNMK) for drug interaction prediction<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The GCNMK model incorporates two distinct DDI kernels within the graph convolutional layer: one based on an “increasing” DDI graph that captures positive DDI correlations, and the other based on a “decreasing” DDI graph that captures negative DDI correlations. The model demonstrated higher prediction accuracy, with the learned drug features being fed into a block of three fully connected layers for DDI prediction. Gao et al.<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup> introduced AutoDDI, an automated method for predicting drug-drug interactions (DDIs). This approach leverages a reinforcement learning-based search algorithm to design optimal GNN architectures without the need for manual intervention. AutoDDI explores a customized search space of handcrafted GNN models, ensuring efficient and accurate predictions. The method achieves state-of-the-art performance on real-world datasets, demonstrating its effectiveness. Additionally, AutoDDI improves interpretability by accurately identifying the drug substructures involved in DDIs. Geng et al.<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> came up with a graph neural network-based model that leverages a multiscale graph neural network (MGNN) and an attention-based substructure interaction learning module to predict adverse drug-drug interactions accurately. MGDDI addresses existing limitations by effectively capturing molecular substructure variations and detailed interactions between drug pairs, demonstrating superior predictive performance in experimental evaluations. Feng et al.<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup> proposed a graph attention network-based model for drug-drug interaction (DDI) prediction, designed to handle both known drugs within the DDI network and novel drugs. By constructing a five-layer graph attention network that captures k-hop low-dimensional feature representations and employing an MLP predictor, GNN-DDI achieves superior predictive performance. Additionally, it enhances interpretability by identifying specific drug substructures involved in DDIs, providing insights into their underlying mechanisms. Similarly, Han et al.<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> introduced SmileGNN, a novel DDI prediction model that integrates structural features from SMILES data with topological features derived from knowledge graphs using graph neural networks. Experimental results show that SmileGNN outperforms existing models in DDI prediction accuracy, and its credibility is further validated by verifying five of the top ten predicted interactions with the latest database.</p>
<p id="Par7">Liu et al.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> developed DANN-DDI, a deep attention neural network-based framework for predicting missing drug-drug interactions (DDIs). This model integrates multiple drug feature networks and utilizes graph embedding to learn meaningful drug representations, while an attention mechanism is employed to enhance the learning of interactions between drug pairs. Experimental results indicate that DANN-DDI is capable of identifying novel DDIs and associated events, thus improving the robustness and applicability of DDI prediction. In a similar vein, Li et al.<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> introduced HetDDI, a pre-trained heterogeneous graph neural network model designed to enhance DDI prediction. HetDDI combines structural information from drug molecule graphs with rich semantic data from biomedical knowledge graphs, enabling it to effectively learn drug representations from diverse sources of heterogeneous information. Case studies further demonstrate HetDDI’s capacity to predict previously unknown DDIs, confirming its practical applicability in real-world scenarios. Sukanya et al.<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> proposed an innovative method for DDI prediction that leverages rich drug representations sourced from multiple knowledge domains, such as the Drug-Target Interaction Network and chemical structure data. This method, which incorporates techniques like metapath2vec and Graph Auto-Encoders, has been shown to outperform previous models across several benchmark datasets. Shtar et al.<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> introduced two novel methods, Adjacency Matrix Factorization (AMF) and Adjacency Matrix Factorization with Propagation (AMFP), for predicting DDIs. These methods frame the DDI prediction problem as link prediction and utilize artificial neural networks and factor propagation to uncover interactions. Lei et al.<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> presented the Neighborhood Relation-Aware Graph Neural Network (NRAGNN), which improves DDI prediction by incorporating both the topological structure of drug entities and the semantic significance of relationships within the drug knowledge graph. The NRAGNN model showed superior performance on the KEGG-DRUG dataset, achieving significant improvements in various evaluation metrics compared to existing models. Lastly, Feng et al.<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> introduced DPDDI, a method for predicting DDIs that uses a Graph Convolution Network (GCN) to extract network structure features from a DDI network, while a Deep Neural Network (DNN) is employed to perform the prediction. DPDDI outperformed four existing methods, demonstrating the effectiveness of GCN-derived features and the concatenation feature aggregation operator in capturing complex DDI information.</p>
<p id="Par8">Khaled et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> introduce a novel model called the Hypergraph Neural Network (HyGNN) that predicts Drug-Drug Interactions (DDIs) by leveraging only SMILES strings. This model captures the chemical structure similarities of drugs through hypergraph representations, highlighting the importance of structural relationships in drug interactions. Similarly, Junpeng et al.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> present the Multi-layer Adaptive Soft Mask Graph Neural Network (MASMDDI) model, which enhances DDI prediction by capturing interaction information between chemical substructures. The effectiveness of MASMDDI was evaluated using the DrugBank dataset, where it demonstrated promising results in predicting DDIs for previously unknown drugs. In a different approach, Wang et al.<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> introduce AttenSyn, an attention-based deep graph neural network designed for predicting synergistic drug combinations. This model leverages molecular graphs along with an attention-based pooling module to refine the representations of drug pairs, significantly outperforming current methods for anticancer drug synergy prediction. Additionally, AttenSyn provides interpretability by identifying the key drug substructures contributing to synergy, and further validation on two cell lines demonstrates its ability to generalize across different biological contexts. Wu et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> present MKG-FENN, a multimodal knowledge graph fused end-to-end neural network aimed at DDI prediction. MKG-FENN constructs a multimodal knowledge graph and extracts high-order semantic features through a graph neural network, which are then fused in an end-to-end manner, enabling more robust DDI predictions. Yu et al.<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> propose the Substructure-aware Tensor Neural Network (STNN-DDI), which improves the prediction of multiple types of DDIs by mapping drugs into a substructure-substructure interaction (SSI) space. This model not only enhances predictive performance but also offers interpretability by identifying the critical substructure pairs that drive interactions. Deepa et al.<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> introduce the SNF-HCNN framework, which leverages drug data from sources like DrugBank, PubChem, and SIDER. Using a similarity matrix derived from drug features, SNF-HCNN offers a novel approach to predicting DDIs. Similarly, Su et al.<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> introduce DDKG, an attention-based knowledge graph representation learning framework that integrates drug attributes and neighboring information to enhance DDI predictions. Feng et al.<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> present deepMDDI, a deep learning-based model for multi-label DDI prediction. By utilizing a relational graph convolutional network encoder and a tensor-like decoder, deepMDDI performs well in both transductive and inductive settings, successfully identifying new valid DDIs in case studies. Lastly, Wang et al.<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> propose MIRACLE, a multi-view graph contrastive representation learning method that captures both inter-view molecular structures and intra-view drug interaction relationships, significantly improving DDI prediction accuracy. Further DPSP framework<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> predicts drug-drug interactions by constructing novel drug feature vectors using multiple types of drug information and Jaccard similarity, followed by a deep neural network for classification. Researchers<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> proposed NNPS, a neural network-based method for predicting polypharmacy side effects using novel drug feature vectors derived from mono side effects and drug-protein interactions.</p>
<p id="Par9">Further we have analysed the correlated work of drug target interaction prediction problems. How the researchers have solved the drug target interaction problem. Neural Matrix Factorization (ANMF), a novel model for computational drug repositioning that integrates drug-drug and disease-disease similarities, autoencoder-based hidden feature extraction, and Generalized Matrix Factorization is porposed<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. GraphCL-DTA, a model for drug-target binding affinity prediction that leverages graph contrastive learning to improve drug representation quality. GraphCL-DTA performs semantic-preserving data augmentation in the embedding space, allowing for richer, unsupervised representation learning of molecular graphs. Additionally, a new loss function is introduced to directly optimize the uniformity of drug and target embeddings<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Hybrid Attentional Memory Network (HAMN), a deep learning model that combines neighborhood-based and latent factor-based collaborative filtering approaches to leverage both local and global information for drug repositioning. By incorporating a memory-attention mechanism and an autoencoder, along with auxiliary drug and disease information, HAMN addresses the cold start problem<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Further researchers<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> present a multi-task self-supervised learning framework for computational drug repositioning, designed to address the challenge of label sparsity by enhancing drug representation learning. By jointly training a main task for drug-disease association prediction and an auxiliary contrastive learning task using data augmentation. NMFDR<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, a Neural Metric Factorization model for computational drug repositioning, to overcome the expressive limitations of traditional matrix factorization methods. By modeling drug-disease associations through a generalized Euclidean distance in a high-dimensional latent space. The authors<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> propose PUON, it leverages only validated (positive) and unvalidated (unlabeled) drug-disease associations without negative sampling and introduces an Outer Neighborhood-based classifier to capture cross-feature information from latent factors. These various approaches demonstrate the growing importance of graph-based and neural network-driven methodologies in the prediction and understanding of drug interactions, reflecting an ongoing trend toward more advanced, interpretable, and accurate models in computational drug discovery.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Research gaps and challenges</h3>
<p id="Par10">Although significant progress has been made in drug-drug interaction prediction, several challenges and methodological limitations remain.</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par11"><strong>Data Imbalance and Incompleteness:</strong> Drug interaction data usually come from medical experiments, patient records, literature reports, and departmental statistics. However, these data are often inconsistent. Furthermore, while the amount of drug-related data is vast, the subset available for analysis is relatively limited, leading to data imbalance. Many potential drug interactions remain undiscovered due to the scarcity of comprehensive datasets.</p></li>
<li><p id="Par12"><strong>Complex Mechanisms of Interaction:</strong> In the human body, drugs can interact during various phases such as metabolism, absorption, distribution, and excretion. These mechanisms often involve multiple biological processes, making it challenging to develop predictive models that can accurately capture and interpret such complex interactions.</p></li>
<li><p id="Par13"><strong>Non-Euclidian relationship:</strong> One of the primary obstacles is the vast complexity of interactions between drugs, which involves numerous factors such as chemical properties, pharmacodynamics, and pharmacokinetics. The large number of potential drug combinations further complicates the task, as traditional experimental methods for detecting DDIs through in vitro and in vivo testing are both time-consuming and costly. Additionally, many existing models rely heavily on structured data, such as chemical structures or biological pathways, but fail to fully capture the complex, unstructured relationships and nuances between drugs that can lead to interactions.</p></li>
<li><p id="Par14"><strong>Data Fusion Challenges:</strong> Effective drug-drug interaction analysis requires integrating multiple data sources, including multi-modal data like molecular structures, protein interaction networks, and genomic information. Combining these diverse data types to generate meaningful and useful features remains a significant challenge.</p></li>
<li><p id="Par15"><strong>Data Sharing and Cross-Field Collaboration:</strong> Predicting drug interactions often involves collaboration across diverse fields such as biology, chemistry, and medicine. However, issues related to data sharing, intellectual property rights, and the protection of sensitive information hinder cross-field cooperation, making it difficult to access and share crucial datasets.</p></li>
</ul></section><section id="Sec5"><h3 class="pmc_sec_title">Research significance</h3>
<p id="Par16"><strong>Theoretical Significance:</strong> Traditional drug-drug interaction prediction methods primarily rely on the chemical properties of drugs and target protein sequences, among other factors. However, predicting drug interactions remains a complex challenge. The recent rise of graph neural network (GNN) models offers a promising solution by enabling the modeling of intricate graph data, including the three-dimensional structures of drug molecules, chemical bond information, and target proteins. This approach provides richer and more precise feature representations for drug interaction prediction. Moreover, GNN models can capture relationships between drugs and target proteins, offering researchers deeper insights into the mechanisms of drug interactions. Accurate drug interaction predictions can also support personalized medicine, improving treatment outcomes and enhancing medication safety.</p>
<p id="Par17"><strong>Practical Significance:</strong> The application of graph neural network (GNN) models has introduced considerable advantages for researchers using machine learning to predict drug interactions. GNN models significantly improve the accuracy of these predictions, positively influencing drug development, the therapeutic effectiveness of drug combinations, and drug safety. Therefore, using GNN models for drug interaction prediction presents vast research potential and practical implications for the field.</p></section></section><section id="Sec6"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec7"><h3 class="pmc_sec_title">Training results</h3>
<p id="Par18">The experiments are performed five times with different random seeds. Each model is trained for 100 epochs with learning rate=0.001, batch size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/79a20f9df21b/d33e410.gif" loading="lazy" id="d33e410" alt="Inline graphic"></span>, Hidden layer size=128,dropout rate=0.5.input-drop 0.0, edge-drop 0.0, attn-drop 0.0 and so on. We kept the default parameters given by the paper code itself for the baseline models.</p>
<p id="Par19">The training loss curves (Figure <a href="#Fig2" class="usa-link">2</a>) across the three datasets–BIOSNAP, DrugBank, and OGBL-DDI–exhibit consistent downward trends, indicating that all models generally learn meaningful patterns over epochs. In the BIOSNAP dataset, the models display smooth and monotonic convergence, with some achieving noticeably lower final losses. This suggests that BIOSNAP is relatively easier to fit, and some models generalize better on it. In contrast, the DrugBank dataset reveals greater variance in convergence behavior. Certain models converge rapidly, while others exhibit a slower decline or maintain higher loss values throughout training, possibly due to the dataset’s increased complexity or noise. The OGBL-DDI dataset shows the most irregular loss trajectories, with several curves presenting mild fluctuations or delayed convergence. These patterns imply a more challenging optimization landscape, where model architecture significantly affects the training dynamics. Overall, the shape and rate of the loss curves reflect differences in model capacity, optimization stability, and dataset complexity.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/6876226ff4de/41598_2025_12936_Fig2_HTML.jpg" loading="lazy" id="MO2" height="290" width="668" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The experimental results of loss value of each model during training for the three datasets. .</p></figcaption></figure><p id="Par20"><strong>Overview for the results on validation dataset</strong> Table <a href="#Tab1" class="usa-link">1</a> reports the highest validation performance of various models during training across the three datasets: <em>Biosnapddi</em>, <em>Drugbankddi</em>, and <em>Ogbl-ddi</em>, based on Hits@10 through Hits@50. The best three models per metric are color-coded, highlighting leading performance in ranking accuracy.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Validation Metrics during training.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Tab1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/022f1ad99cfc/41598_2025_12936_Tab1_HTML.jpg" loading="lazy" id="MO3" height="476" width="669" alt="graphic file with name 41598_2025_12936_Tab1_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par21">The validation results reveal distinct performance patterns across datasets. For Biosnapddi, models consistently achieve high Hits@n scores, indicating effective identification and ranking of true DDI pairs. Notably, ‘AGDN‘, ‘GCN‘, and ‘MEMAGDN‘ demonstrate strong performance on this dataset. In contrast, all models exhibit very low Hits@n values on the Drugbankddi dataset, typically ranging from 0.010 to 0.030. This suggests a significant challenge in the models’ ability to prioritize true DDI instances within their predictions for this specific dataset. ‘MAGCN‘ shows a marginal advantage at higher ’n’. On the Ogbl-ddi dataset, model performance is moderate, yet more discerning. ‘SAGE‘ and ‘SAGENGNN‘ generally lead in ‘Hits@n‘, with ‘MAGCN‘ and ‘AGDN‘ also performing competitively, achieving hit rates up to 0.630. In essence, while models effectively rank DDI pairs on Biosnapddi, they struggle considerably on Drugbankddi. The Ogbl-ddi dataset offers a more balanced evaluation ground, highlighting varying strengths among models in accurately retrieving relevant drug-drug interactions. </p>
<p id="Par22"><strong>Overview from test data: </strong> Table <a href="#Tab2" class="usa-link">2</a> presents the final performance on test data of various models across three datasets: <em>Biosnapddi</em>, <em>Drugbankddi</em>, and <em>Ogbl-ddi</em>. The evaluation includes five ranking-based metrics, Hits@10 through Hits@50, along with overall classification accuracy. There is a clear distinction between the performance on Accuracy and Hits@n metrics. While many models achieve very high accuracy (often above 0.900), their Hits@n scores, particularly at lower ’n’ values (e.g., @10, @20), are generally quite low, especially for the Biosnapddi and Drugbankddi datasets. This suggests that the models are highly accurate at classifying most DDI pairs (likely negative interactions, which are much more common), but struggle to rank true positive interactions highly within a set of predictions. In DDI prediction, a high Hits@n is often more crucial as it indicates the model’s ability to prioritize relevant interactions.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Model evaluation based on various metrics and datasets.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Tab2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/638de0a7a569/41598_2025_12936_Tab2_HTML.jpg" loading="lazy" id="MO4" height="421" width="669" alt="graphic file with name 41598_2025_12936_Tab2_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par23">The Ogbl-ddi dataset appears to be significantly more challenging for DDI prediction in terms of absolute Hits@n scores, but also shows a greater range and higher maximum Hits@n values, indicating that models have more room for differentiation and improvement on this dataset compared to Biosnapddi and Drugbankddi. The accuracy for Ogbl-ddi is also generally high, but slightly lower than Biosnapddi. For Biosnapddi and Drugbankddi, the Hits@n values are consistently close to 0.000 for most models and ’n’ values. This indicates a substantial challenge in correctly ranking positive DDI instances among many negative ones.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Analysis by dataset</h3>
<section id="Sec9"><h4 class="pmc_sec_title">Biosnapddi dataset</h4>
<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par25">
<strong>Hits@n Performance:</strong> This dataset presents a significant challenge for all models in terms of Hits@n. Most models show <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/526b03e216b2/d33e547.gif" loading="lazy" id="d33e547" alt="Inline graphic"></span> across all Hits@n metrics, up to @40 or even @50.<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par26">
<strong>AGDN</strong> is the only model to show a slight improvement, achieving <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/05b11c26a73d/d33e559.gif" loading="lazy" id="d33e559" alt="Inline graphic"></span> for Hits@30, @40, and @50. This small positive value, while still low, indicates it might be marginally better at retrieving true positives than others.</div></li>
<li><div class="p" id="Par27">
<strong>GATMLP</strong> and <strong>MEMAGDN</strong> also show <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/05b11c26a73d/d33e573.gif" loading="lazy" id="d33e573" alt="Inline graphic"></span> at Hits@50, with MEMAGDN also achieving it at Hits@30 and @40.</div></li>
<li><div class="p" id="Par28">The extremely low Hits@n values across the board suggest that if DDI prediction is about ranking potential interactions, these models are not performing effectively on this dataset.</div></li>
</ul>
</div></li>
<li><div class="p" id="Par29">
<strong>Accuracy Performance:</strong> In contrast to Hits@n, accuracy is remarkably high for most models on Biosnapddi.<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par30">
<strong>AGDN</strong> and <strong>SAGESKIPCONNECTION</strong> share the top accuracy at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/ff6ea3211f23/d33e599.gif" loading="lazy" id="d33e599" alt="Inline graphic"></span>, suggesting excellent performance in classifying DDI vs. non-DDI pairs.</div></li>
<li><div class="p" id="Par31">
<strong>GCNSKIPCONNECTION</strong> and <strong>SAGENGNN</strong> also perform very well with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/9f6e591d601a/d33e613.gif" loading="lazy" id="d33e613" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/3e519b8fadb8/d33e619.gif" loading="lazy" id="d33e619" alt="Inline graphic"></span> respectively.</div></li>
<li><div class="p" id="Par32">Models like <strong>MEMAGDN</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/de74ffe932b0/d33e631.gif" loading="lazy" id="d33e631" alt="Inline graphic"></span>) and <strong>MAGCN</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/01bb97778ed6/d33e640.gif" loading="lazy" id="d33e640" alt="Inline graphic"></span>) show significantly lower accuracy and higher standard deviations, indicating less stable or generally poorer classification performance compared to the others.</div></li>
</ul>
</div></li>
<li><p id="Par33">On Biosnapddi dataset models are very good at classifying (leading to high accuracy). However, they struggle to correctly rank or identify the rare positive interactions (leading to very low Hits@n). For practical DDI prediction where identifying <em>true</em> interactions is paramount, the current models are not effectively addressing this challenge including our proposed models.</p></li>
</ul></section><section id="Sec10"><h4 class="pmc_sec_title">Drugbankddi dataset</h4>
<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par35">
<strong>Hits@n Performance:</strong> Similar to Biosnapddi, the Hits@n scores for Drugbankddi are consistently very low (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/526b03e216b2/d33e675.gif" loading="lazy" id="d33e675" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/05b11c26a73d/d33e681.gif" loading="lazy" id="d33e681" alt="Inline graphic"></span> across all ’n’ values for all models).<ul class="list" style="list-style-type:disc"><li><div class="p" id="Par36">Again, this indicates a struggle in ranking positive DDI interactions. All models show identical (or very nearly identical) Hits@n performance here.</div></li></ul>
</div></li>
<li><div class="p" id="Par37">
<strong>Accuracy Performance:</strong> Accuracy on Drugbankddi is generally lower than on Biosnapddi but still moderately high for several models.<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par38">
<strong>SAGENGNN</strong> performs best in terms of accuracy at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/c585838c7527/d33e708.gif" loading="lazy" id="d33e708" alt="Inline graphic"></span>.</div></li>
<li><div class="p" id="Par39">
<strong>SAGESKIPCONNECTION</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/4751c7d53316/d33e719.gif" loading="lazy" id="d33e719" alt="Inline graphic"></span>) and <strong>SAGE</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/f88591ec01d5/d33e728.gif" loading="lazy" id="d33e728" alt="Inline graphic"></span>) also show strong accuracy.</div></li>
<li><div class="p" id="Par40">
<strong>AGDN</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/8c44f6e48e12/d33e739.gif" loading="lazy" id="d33e739" alt="Inline graphic"></span>) is competitive.</div></li>
<li><div class="p" id="Par41">
<strong>GATMLP</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/40868b944836/d33e750.gif" loading="lazy" id="d33e750" alt="Inline graphic"></span>) and especially <strong>GCNSKIPCONNECTION</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/e8cbe066e8c2/d33e759.gif" loading="lazy" id="d33e759" alt="Inline graphic"></span>) show notably poorer accuracy, with the latter essentially performing at random chance for binary classification.</div></li>
</ul>
</div></li>
<li><p id="Par42">The Drugbankddi dataset also presents a challenge for ranking positive DDIs (evident from the consistently low Hits@n). While overall accuracy is respectable for some models, others perform poorly, suggesting a more balanced or diverse set of challenges compared to Biosnapddi in terms of classification. The significant drop in accuracy for certain models like GCNSKIPCONNECTION suggests it struggles with the underlying patterns in this specific dataset.</p></li>
</ul></section><section id="Sec11"><h4 class="pmc_sec_title">Ogbl-ddi dataset</h4>
<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par44">
<strong>Hits@n Performance:</strong> This is where the models show much more meaningful differentiation and higher absolute Hits@n scores.<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par45">
<strong>GCNNGNNGRAPHCONV</strong> is the standout performer in Hits@n, achieving the highest scores across most ’n’ values: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/28cbaea1c447/d33e794.gif" loading="lazy" id="d33e794" alt="Inline graphic"></span> (@10), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/692b1fa6ac16/d33e800.gif" loading="lazy" id="d33e800" alt="Inline graphic"></span> (@20), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/1891d75dad3e/d33e806.gif" loading="lazy" id="d33e806" alt="Inline graphic"></span> (@30), and the highest at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/6753b1a5c303/d33e812.gif" loading="lazy" id="d33e812" alt="Inline graphic"></span> (@40) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/a10c99605b56/d33e818.gif" loading="lazy" id="d33e818" alt="Inline graphic"></span> (@50). This indicates it is the most effective at retrieving true positive DDIs within the top ranked predictions.</div></li>
<li><div class="p" id="Par46">
<strong>GCNSKIPCONNECTION</strong> is also very strong, particularly at Hits@10 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/5a4891a6290e/d33e829.gif" loading="lazy" id="d33e829" alt="Inline graphic"></span> - highest) and @20 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/ff23fcf6872e/d33e835.gif" loading="lazy" id="d33e835" alt="Inline graphic"></span> - highest). Its performance slightly drops at higher ’n’ compared to GCNNGNNGRAPHCONV.</div></li>
<li><div class="p" id="Par47">Other models like <strong>GCN</strong> (e.g., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/bbc52fcde38e/d33e847.gif" loading="lazy" id="d33e847" alt="Inline graphic"></span> at @50) and <strong>MAGCN</strong> (e.g., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/bbc52fcde38e/d33e856.gif" loading="lazy" id="d33e856" alt="Inline graphic"></span> at @50) also show decent Hits@n.</div></li>
<li><div class="p" id="Par48">
<strong>SAGE</strong>, <strong>SAGENGNN</strong>, and <strong>AGDN</strong> generally have lower Hits@n values on this dataset compared to the top performers.</div></li>
</ul>
</div></li>
<li><div class="p" id="Par49">
<strong>Accuracy Performance:</strong> Accuracy is high across most models on Ogbl-ddi.<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par50">
<strong>SAGE</strong> achieves the highest accuracy at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/d36eef02185a/d33e887.gif" loading="lazy" id="d33e887" alt="Inline graphic"></span>.</div></li>
<li><div class="p" id="Par51">
<strong>MAGCN</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/cfa3c3ec6107/d33e898.gif" loading="lazy" id="d33e898" alt="Inline graphic"></span>) and <strong>MEMAGDN</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/52bc6cf4ece4/d33e907.gif" loading="lazy" id="d33e907" alt="Inline graphic"></span>) also show excellent accuracy.</div></li>
<li><div class="p" id="Par52">
<strong>AGDN</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/a8aa73112483/d33e918.gif" loading="lazy" id="d33e918" alt="Inline graphic"></span>) is also among the top.</div></li>
<li><div class="p" id="Par53">
<strong>GCNSKIPCONNECTION</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/f71bf7ca894e/d33e929.gif" loading="lazy" id="d33e929" alt="Inline graphic"></span>) and <strong>GATMLP</strong> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/690cf0640d6c/d33e938.gif" loading="lazy" id="d33e938" alt="Inline graphic"></span>) have slightly lower accuracy but are still performing well.</div></li>
</ul>
</div></li>
<li><p id="Par54">The Ogbl-ddi <strong>:</strong> dataset appears to be better suited for evaluating the ranking capabilities of DDI models. Models like <strong>GCNNGNNGRAPHCONV</strong> and <strong>GCNSKIPCONNECTION</strong> demonstrate superior ability in retrieving actual DDI instances, indicating their potential for real-world DDI prediction where identifying relevant interactions is key. High accuracy is maintained by most models, suggesting a good overall classification ability.</p></li>
</ul></section></section><section id="Sec12"><h3 class="pmc_sec_title">Analysis by model (across datasets)</h3>
<section id="Sec13"><h4 class="pmc_sec_title">AGDN</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par56"><strong>Hits@n:</strong> Consistently low on Biosnapddi and Drugbankddi (0.000-0.010). Shows moderate performance on Ogbl-ddi (e.g., 0.230 at @50).</p></li>
<li><p id="Par57"><strong>Accuracy:</strong> Excellent on Biosnapddi (0.992), decent on Drugbankddi (0.798), and strong on Ogbl-ddi (0.975).</p></li>
<li><p id="Par58"><strong>Overall:</strong> A generally robust performer in terms of accuracy, but its Hits@n performance is only notable on Ogbl-ddi among the datasets tested.</p></li>
</ul></section><section id="Sec14"><h4 class="pmc_sec_title">GATMLP</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par60"><strong>Hits@n:</strong> Very poor on Biosnapddi and Drugbankddi (mostly 0.000-0.010). Moderate on Ogbl-ddi (e.g., 0.250 at @50).</p></li>
<li><p id="Par61"><strong>Accuracy:</strong> Good on Biosnapddi (0.989), but significantly lower on Drugbankddi (0.614) and slightly lower on Ogbl-ddi (0.937) compared to top models.</p></li>
<li><p id="Par62"><strong>Overall:</strong> Seems to struggle with Drugbankddi dataset in particular. Hits@n are not its strong suit.</p></li>
</ul></section><section id="Sec15"><h4 class="pmc_sec_title">GCN</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par64"><strong>Hits@n:</strong> Very low on Biosnapddi and Drugbankddi. Better on Ogbl-ddi (e.g., 0.350 at @50), performing reasonably well among the models.</p></li>
<li><p id="Par65"><strong>Accuracy:</strong> High on Biosnapddi (0.987), moderate on Drugbankddi (0.736), and good on Ogbl-ddi (0.951).</p></li>
<li><p id="Par66"><strong>Overall:</strong> A solid baseline GNN model, showing average to good performance, particularly for Hits@n on Ogbl-ddi.</p></li>
</ul></section><section id="Sec16"><h4 class="pmc_sec_title">GCNSKIPCONNECTION</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par68"><strong>Hits@n:</strong> Very low on Biosnapddi and Drugbankddi. <strong>Very strong on</strong> Ogbl-ddi, particularly at lower ’n’ values (highest Hits@10 and @20).</p></li>
<li><p id="Par69"><strong>Accuracy:</strong> Excellent on Biosnapddi (0.991), but remarkably poor on Drugbankddi (0.500), and decent on Ogbl-ddi (0.928).</p></li>
<li><p id="Par70"><strong>Overall:</strong> A peculiar model. While it excels at retrieving relevant hits on Ogbl-ddi, its highly variable accuracy across datasets (especially the random performance on Drugbankddi) suggests it might be sensitive to specific dataset characteristics or require more fine-tuning.</p></li>
</ul></section><section id="Sec17"><h4 class="pmc_sec_title">MEMAGDN</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par72"><strong>Hits@n:</strong> Marginally better than GCN on Biosnapddi (0.010 at @30,@40,@50) and consistent with others on Drugbankddi (0.010). Moderate on Ogbl-ddi (e.g., 0.240 at @50).</p></li>
<li><p id="Par73"><strong>Accuracy:</strong> Noticeably lower accuracy on Biosnapddi (0.891) and Drugbankddi (0.738) with higher variance, but high on Ogbl-ddi (0.973).</p></li>
<li><p id="Par74"><strong>Overall:</strong> Its performance is quite inconsistent across datasets, suggesting it might not generalize as well as some other models.</p></li>
</ul></section><section id="Sec18"><h4 class="pmc_sec_title">MAGCN</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par76"><strong>Hits@n:</strong> Very low on Biosnapddi and Drugbankddi. Decent on Ogbl-ddi (e.g., 0.350 at @50).</p></li>
<li><p id="Par77"><strong>Accuracy:</strong> Relatively low and variable accuracy on Biosnapddi (0.888) and Drugbankddi (0.699), but excellent on Ogbl-ddi (0.977).</p></li>
<li><p id="Par78"><strong>Overall:</strong> Similar to MEMAGDN, its performance varies significantly, showing strong accuracy on Ogbl-ddi but struggling on the other two in terms of accuracy.</p></li>
</ul></section><section id="Sec19"><h4 class="pmc_sec_title">SAGE</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par80"><strong>Hits@n:</strong> Very low on Biosnapddi and Drugbankddi. Lower end on Ogbl-ddi (0.220 at @50) compared to top models.</p></li>
<li><p id="Par81"><strong>Accuracy:</strong> High on Biosnapddi (0.980), highest on Drugbankddi among models with 0.801, and highest on Ogbl-ddi (0.981).</p></li>
<li><p id="Par82"><strong>Overall:</strong> SAGE is consistently among the top models for accuracy across all datasets, indicating its strong classification ability. However, its Hits@n performance is generally not as competitive as GCNNGNNGRAPHCONV or GCNSKIPCONNECTION on Ogbl-ddi.</p></li>
</ul></section><section id="Sec20"><h4 class="pmc_sec_title">SAGENGNN</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par84"><strong>Hits@n:</strong> Very low on Biosnapddi and Drugbankddi. Lower end on Ogbl-ddi (0.170 at @50).</p></li>
<li><p id="Par85"><strong>Accuracy:</strong> High on Biosnapddi (0.991), highest on Drugbankddi (0.810), and high on Ogbl-ddi (0.972).</p></li>
<li><p id="Par86"><strong>Overall:</strong> Similar to SAGE, it excels in overall classification accuracy across all datasets, but its ability to rank true positives highly (Hits@n) is not its strongest point.</p></li>
</ul></section><section id="Sec21"><h4 class="pmc_sec_title">SAGESKIPCONNECTION</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par88"><strong>Hits@n:</strong> Very low on Biosnapddi and Drugbankddi. Lower end on Ogbl-ddi (0.240 at @50).</p></li>
<li><p id="Par89"><strong>Accuracy:</strong> Top accuracy on Biosnapddi (0.992) and strong on Drugbankddi (0.807) and Ogbl-ddi (0.964).</p></li>
<li><p id="Par90"><strong>Overall:</strong> A very strong performer in terms of accuracy across all datasets, similar to SAGE and SAGENGNN. Its Hits@n performance on Ogbl-ddi is slightly better than SAGE/SAGENGNN but still not as good as GCNNGNNGRAPHCONV or GCNSKIPCONNECTION.</p></li>
</ul></section></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par91">Accoding to experimental results based on various models, the analysis reveals that while many models achieve high accuracy in classifying drug-drug interaction (DDI) pairs, they struggle significantly with ranking true positive interactions, particularly on the biosnapddi and drugbankddi datasets. The ogbl-ddi dataset, however, shows more differentiation and higher Hits@n scores, indicating a better environment for evaluating ranking capabilities. Models like GCNNGNNGRAPHCONV and GCNSKIPCONNECTION stand out for their performance in Hits@n on ogbl-ddi, while SAGE, SAGENGNN, and SAGESKIPCONNECTION excel in accuracy across all datasets. We have found that not single models outperforms on all the metrics on the datasets which shows the DDI dataset have inherent different characteristics which one model doesn’t fit on all the datasets.This experimental result will provide important references and guidance for drug interaction prediction using graph machine learning, significantly contributing to the sustainable development of drug interaction research and offering valuable insights for real-life drug use. In future research, we can consider the following aspects: </p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par92">Use different models to conduct experiments on various datasets and leverage the characteristics of different models to combine them for discovering better drug interaction prediction models.</p></li>
<li><p id="Par93">Actively collaborate with experts in biology and medicine to update and optimize models based on relevant clinical trial data and pharmacological knowledge.</p></li>
<li><p id="Par94">Investigate methods to improve the interpretability and explainability of the model so that researchers in related fields can understand the model’s prediction results.</p></li>
</ol></section><section id="Sec23"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec24"><h3 class="pmc_sec_title">Graph neural network models</h3>
<section id="Sec25"><h4 class="pmc_sec_title">GAT model</h4>
<p id="Par95">The Graph Attention Network (GAT) represents an advanced graph neural network model designed to leverage the attention mechanism for capturing intricate relationships among nodes within a graph, ultimately producing comprehensive node representations<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Unlike traditional methods, GAT introduces an innovative approach by assigning adaptive attention weights to neighboring nodes, allowing the model to focus on the most relevant connections. This mechanism enables efficient learning without relying on computationally intensive operations, such as matrix inversion, or requiring prior knowledge of the graph’s structure. By incorporating multiple stacked layers, GAT systematically aggregates domain-specific features from the local neighborhood of each node. The detailed architecture of the GAT model is depicted in Figure <a href="#Fig3" class="usa-link">3</a>(a).</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/0c7675f56be0/41598_2025_12936_Fig3_HTML.jpg" loading="lazy" id="MO5" height="425" width="790" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Various graph neural networks used in our study are described in this figure.</p></figcaption></figure></section><section id="Sec26"><h4 class="pmc_sec_title">GCN model</h4>
<p id="Par96">The Graph Convolutional Network (GCN) is a deep learning model designed for processing graph-structured data. It is primarily utilized for tasks such as node classification and link prediction<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. GCN represents graph data using two key components: an adjacency matrix and a node feature matrix. The adjacency matrix encodes the relationships between nodes, where entries typically indicate the presence (1) or absence (0) of connections, while the node feature matrix contains the feature vectors for each node. Through graph convolution operations, GCN propagates and aggregates node features from neighboring nodes, thereby generating enriched node representations. These representations are iteratively refined across multiple layers of graph convolutions, enabling the model to capture both local connectivity and complex relationships. GCN supports parameter sharing and is adaptable to graphs of various structures, providing strong generalization capabilities. Furthermore, GCNs can be integrated with other deep learning models to enhance performance across diverse tasks. The architecture of the GCN model is illustrated in Figure <a href="#Fig3" class="usa-link">3</a>(a).</p></section><section id="Sec27"><h4 class="pmc_sec_title">SAGE model</h4>
<p id="Par97">The SAGE (Sample and Aggregation) model is a graph neural network model developed to learn low-dimensional representations of nodes in graph data. It effectively captures both local and global characteristics of nodes through multilayer neighbor sampling and information aggregation operations<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. It is depicted in Figure <a href="#Fig3" class="usa-link">3</a>(a).</p></section><section id="Sec28"><h4 class="pmc_sec_title">Graph Sage + NGNN (SAGENGNN)</h4>
<p id="Par98">In this model, the GraphSAGE layer and the Nested Graph Neural Network (NGNN) layers are combined sequentially to enhance feature extraction, following a linear architecture inspired by<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p></section><section id="Sec29"><h4 class="pmc_sec_title">TAG model</h4>
<p id="Par99">The Topology Adaptive Graph (TAG) method adapts the topology of graph neural networks to suit the structure of different graph data<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Traditional graph neural networks often assume a fixed topology, which may not be ideal for diverse graph structures. TAG dynamically adjusts the network’s topology to fit the input graph data better and capture local structural information better. It allows each node to adaptively select its neighboring nodes and learn convolutional kernels based on the relationships between nodes. This flexibility enables TAG to better capture local structural information, enhancing the performance of graph neural networks across different types of graph data.</p>
<p id="Par100">To tackle the drug-drug interaction problem, we explored various combinations of these models, systematically evaluating their performance and comparing the results with those of other state-of-the-art approaches.</p></section></section><section id="Sec30"><h3 class="pmc_sec_title">Memory attention graph convolution(MAGCN):</h3>
<p id="Par101">Wang et al.<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> introduced the Graph Inception Diffusion Networks (GIDN) model, which extends graph diffusion mechanisms across diverse feature spaces while incorporating an inception module to mitigate the computational overhead associated with complex network structures. The architecture of the GIDN model is illustrated in Figure <a href="#Fig3" class="usa-link">3</a>(f). The primary components of the model are as follows:</p>
<section id="Sec31"><h4 class="pmc_sec_title">Feature transformation</h4>
<p id="Par102">Node features (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/d75807f75928/d33e1458.gif" loading="lazy" id="d33e1458" alt="Inline graphic"></span>) are first split into smaller blocks corresponding to hop steps. For each block, a transformation is applied. The feature transformation for a block <em>i</em> is defined as:</p>
<table class="disp-formula p" id="Equ9"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/1c0f945ccf16/d33e1467.gif" loading="lazy" id="d33e1467" alt="graphic file with name d33e1467.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/0cba16452b61/d33e1473.gif" loading="lazy" id="d33e1473" alt="Inline graphic"></span> is a learned position embedding for the <em>i</em>-th hop.</p></section><section id="Sec32"><h4 class="pmc_sec_title"> Attention mechanism</h4>
<p id="Par103">The layer computes attention between nodes in a graph. The attention mechanism uses source (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/1c269efe12f7/d33e1486.gif" loading="lazy" id="d33e1486" alt="Inline graphic"></span>) and destination (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/f1f1baa2b9c3/d33e1492.gif" loading="lazy" id="d33e1492" alt="Inline graphic"></span>) node features for computing attention scores.</p>
<p id="Par104">Query, Key, and Value matrices are used to transform node features:</p>
<table class="disp-formula p" id="Equ10"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/ecebc4572edf/d33e1500.gif" loading="lazy" id="d33e1500" alt="graphic file with name d33e1500.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/46b00cde0aa9/d33e1506.gif" loading="lazy" id="d33e1506" alt="Inline graphic"></span> are the learned projection matrices.</p>
<p id="Par105">The attention score computation for each edge is given by:</p>
<table class="disp-formula p" id="Equ11"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/77f1f45a4cda/d33e1514.gif" loading="lazy" id="d33e1514" alt="graphic file with name d33e1514.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/4d510855ba30/d33e1520.gif" loading="lazy" id="d33e1520" alt="Inline graphic"></span> is a learned vector, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/043c0cbe0033/d33e1526.gif" loading="lazy" id="d33e1526" alt="Inline graphic"></span> denotes concatenation. The attention scores are normalized using softmax:</p>
<table class="disp-formula p" id="Equ12"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/e83c43b39711/d33e1532.gif" loading="lazy" id="d33e1532" alt="graphic file with name d33e1532.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/78c1d4a59090/d33e1538.gif" loading="lazy" id="d33e1538" alt="Inline graphic"></span> denotes the set of neighboring nodes of node <em>i</em>.</p>
<p id="Par106">The final node features are aggregated using attention weights:</p>
<table class="disp-formula p" id="Equ13"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/733c50fc4e4f/d33e1550.gif" loading="lazy" id="d33e1550" alt="graphic file with name d33e1550.gif"></td></tr></table></section><section id="Sec33"><h4 class="pmc_sec_title"> Edge features (Optional)</h4>
<p id="Par107">If edge features (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/5d39f3670ccf/d33e1559.gif" loading="lazy" id="d33e1559" alt="Inline graphic"></span>) are provided, they are incorporated into the attention mechanism. Specifically, the edge features modify the attention weights as:</p>
<table class="disp-formula p" id="Equ14"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2c9699df3591/d33e1565.gif" loading="lazy" id="d33e1565" alt="graphic file with name d33e1565.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/0412f80cd2f8/d33e1571.gif" loading="lazy" id="d33e1571" alt="Inline graphic"></span> represents the edge feature vector between nodes <em>i</em> and <em>j</em>.</p></section><section id="Sec34"><h4 class="pmc_sec_title"> Multi-hop neighborhood aggregation</h4>
<p id="Par108">The model aggregates features over multiple hop steps. It updates node features and computes attention for each hop, allowing information to flow over longer distances.</p></section><section id="Sec35"><h4 class="pmc_sec_title"> TAG and Jumping Knowledge Layer</h4>
<p id="Par109">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/8316f350fef7/d33e1591.gif" loading="lazy" id="d33e1591" alt="Inline graphic"></span>
</p></section><section id="Sec36"><h4 class="pmc_sec_title">Output and residual connection</h4>
<p id="Par110">After attention aggregation, the features are transformed through a final linear transformation. The output is:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/4a484a7ec3a0/d33e1601.gif" loading="lazy" id="d33e1601" alt="graphic file with name d33e1601.gif"></td>
<td class="label">1</td>
</tr></table>
<p>If a residual connection is used, the input features are added back to the output, potentially via a transformation matrix:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2491d295a7a6/d33e1608.gif" loading="lazy" id="d33e1608" alt="graphic file with name d33e1608.gif"></td>
<td class="label">2</td>
</tr></table></section><section id="Sec37"><h4 class="pmc_sec_title">Adaptive Graph Diffusion Networks (AGDN)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>
</h4>
<p id="Par111">Sun et al.<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> proposed Adaptive Graph Diffusion Networks (AGDNs) to address overfitting, over-smoothing, and inefficiency in deep Graph Neural Networks. AGDNs perform generalized multi-hop diffusion across different feature spaces using learnable hop-wise weighting mechanisms–Hop-wise Attention and Hop-wise Convolution–while maintaining moderate runtime and complexity. They have also given a variant MEMAGDN.</p></section></section><section id="Sec38"><h3 class="pmc_sec_title">Model optimization</h3>
<section id="Sec39"><h4 class="pmc_sec_title">Model optimization methods<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>
</h4>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par113"><strong>Skip Connection:</strong> Skip connection is a technique employed in neural networks to establish direct connections between different layers. It functions by adding the output of one layer directly to the input of another, enabling information to bypass intermediate layers and propagate more effectively through the network. This approach alleviates the vanishing gradient problem and facilitates the training of deeper networks. Moreover, skip connections allow lower-level features to be retained and utilized by higher layers. In our experiments, as illustrated in Figure <a href="#Fig3" class="usa-link">3</a>(b), we incorporated skip connections into the basic model. This enhancement enables the model to aggregate information from neighbors over multiple hops while mitigating the risk of over-smoothing in node embeddings.</p></li>
<li><p id="Par114"><strong>Post-Processing Layers:</strong> While adding more convolutional layers can be challenging in GNNs, it is also possible to incorporate pre- and post-processing layers that do not perform message passing. Post-processing layers, such as multi-layer perceptrons (MLPs), are applied to node embeddings to convert and refine the nonlinear outputs of the model. MLPs consist of fully connected layers containing multiple neurons and using nonlinear activation functions such as ReLU or Sigmoid. Figure <a href="#Fig3" class="usa-link">3</a>(c) illustrates the model structure with these post-processing layers.</p></li>
<li><p id="Par115"><strong>Enhanced Predictor:</strong> Building on the idea that passing embeddings through MLP layers can enhance performance, we incorporate this concept into a GNN model. Instead of integrating the LinkPredictor as part of the GNN, we define it as a small neural network (“NeuralLinkPredictor”) to perform post-processing independently. This setup optimizes the model’s predictive abilities, as depicted in Figure <a href="#Fig3" class="usa-link">3</a>(d).</p></li>
</ol></section></section><section id="Sec40"><h3 class="pmc_sec_title">Model construction</h3>
<p id="Par117">The AGDN component is a specialized neural network designed for processing graph data by aggregating information from neighboring nodes through diffusion operations. By iteratively updating node representations based on their neighbors, the AGDN effectively captures the graph’s structural information, making it well-suited for understanding complex relationships in graph data.</p>
<p id="Par118">Despite its strengths, the AGDN alone may face limitations in addressing the dynamic and intricate nature of graph topologies encountered in real-world applications. To address this, the TAG model introduces a topology-adaptive mechanism that dynamically adjusts to the graph’s structural variations based on the data’s characteristics. This enables the TAG model to better capture essential topological features, enhancing prediction accuracy.</p>
<p id="Par119">In the MAGCN model, the AGDN, Attention based Neibhour aggregation, and TAGConv layer is applied to the node features, further refining the learned representations by adaptively accounting for the graph’s topology. The final component, the MLP, performs classification or regression tasks on the updated node features. As a robust nonlinear mapping tool, the Jumping Knowledge layer transforms these features into the target output space, enabling accurate and reliable predictions.</p>
<p id="Par120">In summary, the MAGCN model effectively combines the complementary strengths of AGDN, Attention mechanism, TAG, and Jumping Knowledge layer to achieve efficient graph data processing and precise predictive performance. This integrated approach offers wide applicability and has demonstrated strong results in practical applications.</p></section><section id="Sec41"><h3 class="pmc_sec_title">Model training and optimization process</h3>
<section id="Sec42"><h4 class="pmc_sec_title">Problem formulation</h4>
<p id="Par121">During the initial training phase, negative sampling is employed to address the imbalance between positive and negative edges in the graph. For each positive edge (i.e., an existing edge in the training graph), an equal number of negative edges (i.e., non-existent edges) is generated. This approach ensures that each training batch contains a balanced representation of positive and negative edges, facilitating effective learning.</p>
<p id="Par122">The model then generates node embeddings by processing all training edges, transforming the graph data into feature representations that encapsulate the structural and relational information of the graph. These embeddings serve as the input for subsequent prediction tasks.</p>
<p id="Par123">As a result, the problem is formulated as a binary classification task, where the objective is to distinguish between positive and negative edges based on their learned feature representations. This framing aligns with standard machine learning methodologies, allowing the model to leverage well-established classification techniques for edge prediction.</p></section><section id="Sec43"><h4 class="pmc_sec_title">Choice of loss function</h4>
<p id="Par124">The experiment employs binary cross-entropy loss to evaluate model performance. This loss function is particularly suited for binary classification problems, where it measures the discrepancy between two probability distributions: one representing the true labels and the other representing the model’s predictions.</p>
<p id="Par125">In the context of binary classification, each sample is assigned a binary label, typically denoted as 0 or 1, where 0 represents the negative class and 1 denotes the positive class. The model generates a probability value between 0 and 1, indicating the likelihood that a given sample belongs to the positive class. The binary cross-entropy loss evaluates the model’s performance by comparing the predicted probability with the actual true label. The loss function can be mathematically expressed as follows:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2b8f7c652760/d33e1702.gif" loading="lazy" id="d33e1702" alt="graphic file with name d33e1702.gif"></td>
<td class="label">3</td>
</tr></table>
<p>Among them, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/eab8a4e955a3/d33e1709.gif" loading="lazy" id="d33e1709" alt="Inline graphic"></span> is the binary label 0 or 1, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/bb9faaa422fb/d33e1715.gif" loading="lazy" id="d33e1715" alt="Inline graphic"></span> is the predicted probability that the output belongs to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/eab8a4e955a3/d33e1721.gif" loading="lazy" id="d33e1721" alt="Inline graphic"></span> label.</p></section><section id="Sec44"><h4 class="pmc_sec_title">Optimizer selection and parameter settings</h4>
<p id="Par126">The experiment employs the Adam optimizer for model training. Adam, which stands for Adaptive Moment Estimation, is an adaptive learning rate optimization algorithm that combines the benefits of momentum-based gradient descent with the advantages of adaptive learning rates<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. It adjusts the learning rate of each parameter by calculating both the first-order moment estimate (the mean) and the second-order moment estimate (the uncentered variance) of the gradients. This adaptive mechanism allows Adam to effectively handle sparse gradients and noisy data.</p></section><section id="Sec45"><h4 class="pmc_sec_title">Evaluation indicators</h4>
<p id="Par127">The experiment employs two performance evaluation metrics: Hits@K and Accuracy. The Hits@K metric is designed to rank true drug interactions higher than non-interacting drug pairs, emphasizing the model’s ability to correctly identify relevant interactions. Hits@K is particularly useful in ranking tasks, where the goal is to ensure that the true positive interactions are within the top K predictions made by the model. The specific calculation process is as follows:</p>
<p id="Par128">
<strong>1. Hits@K:</strong>
</p>
<p id="Par129">The process begins by calculating the rank of each positive sample based on the predicted scores. If a positive sample’s rank is less than or equal to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2aad2085578d/d33e1746.gif" loading="lazy" id="d33e1746" alt="Inline graphic"></span>, it is considered a “hit” and contributes to the HIT value. Finally, the HIT value is computed by calculating the ratio of the number of hit positive samples to the total number of positive samples, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/665ac1c299c3/d33e1752.gif" loading="lazy" id="d33e1752" alt="Inline graphic"></span>, as follows:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/b67d00dfc297/d33e1758.gif" loading="lazy" id="d33e1758" alt="graphic file with name d33e1758.gif"></td>
<td class="label">4</td>
</tr></table>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/86ffa8427a77/d33e1764.gif" loading="lazy" id="d33e1764" alt="graphic file with name d33e1764.gif"></td>
<td class="label">5</td>
</tr></table>
<p>Next, for each negative sample, its ranking in the predicted scores is calculated. If the rank is greater than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2aad2085578d/d33e1771.gif" loading="lazy" id="d33e1771" alt="Inline graphic"></span>, the negative sample is considered a “hit” and contributes to the HIT value. The HIT value for the negative samples is then computed by calculating the ratio of the number of hit negative representative data to the total number of negative representative data (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/6fd57f503be4/d33e1777.gif" loading="lazy" id="d33e1777" alt="Inline graphic"></span>), as follows:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/b29d2a989e45/d33e1784.gif" loading="lazy" id="d33e1784" alt="graphic file with name d33e1784.gif"></td>
<td class="label">6</td>
</tr></table>
<p>The final HIT value (Hits@K) is the average of the positive HIT value and the negative HIT value:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/edf67f8b41d6/d33e1791.gif" loading="lazy" id="d33e1791" alt="graphic file with name d33e1791.gif"></td>
<td class="label">7</td>
</tr></table>
<p>Among them, it is assumed that there are <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/e1842c352784/d33e1798.gif" loading="lazy" id="d33e1798" alt="Inline graphic"></span> samples, of which <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/665ac1c299c3/d33e1804.gif" loading="lazy" id="d33e1804" alt="Inline graphic"></span> are positive samples and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/6fd57f503be4/d33e1810.gif" loading="lazy" id="d33e1810" alt="Inline graphic"></span> are negative samples. “Number of positive samples with Rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/469685265e52/d33e1816.gif" loading="lazy" id="d33e1816" alt="Inline graphic"></span> K” refers to the number of positive samples with predicted score rankings less than or equal to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2aad2085578d/d33e1823.gif" loading="lazy" id="d33e1823" alt="Inline graphic"></span>. “Number of negative samples with Rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/24c50dfb2a52/d33e1829.gif" loading="lazy" id="d33e1829" alt="Inline graphic"></span> K” refers to the number of negative samples with predicted score rankings greater than or equal to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2aad2085578d/d33e1835.gif" loading="lazy" id="d33e1835" alt="Inline graphic"></span>.</p>
<p id="Par130"><strong>2. Accuracy:</strong> Accuracy is another widely used evaluation metric in classification tasks, which measures the proportion of samples that are correctly classified by the model. The formula for calculating accuracy is as follows:</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/9e4fa6dcb3b6/d33e1845.gif" loading="lazy" id="d33e1845" alt="graphic file with name d33e1845.gif"></td>
<td class="label">8</td>
</tr></table>
<p>where “Number of Correct Predictions” represents the number of samples correctly classified by the model, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/e1842c352784/d33e1852.gif" loading="lazy" id="d33e1852" alt="Inline graphic"></span> represents the total number of samples.</p></section></section><section id="Sec47"><h3 class="pmc_sec_title">Data set selection</h3>
<p id="Par131">In this study, we have used three publicly available drug-drug interaction datasets. These datasets have their own strengths and weaknesses, making them altogether great choices for evaluating models’ performance. The datasets are as follows: </p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par132"><strong>OGB-DDI:</strong> The Ogbl-ddi dataset is a homogeneous, unweighted, undirected graph where each node corresponds to an FDA-approved or experimental drug. Edges denote drug-drug interactions, capturing cases where the combined effect of two drugs differs significantly from their independent effects<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The dataset comprises 4,267 nodes and 1,067,911 edges, with an average node degree of 501. This structure enables the exploration of complex drug-drug interactions.</p></li>
<li><p id="Par133"><strong>Drugbank-DDI:</strong>The DrugBank drug-drug interaction (DDI) dataset is manually curated from FDA and Health Canada drug labels, along with primary literature sources. It includes 191,808 drug-drug interaction pairs involving 1,706 unique drugs and spans 86 distinct interaction types. The associated task is formulated as a multi-class classification problem: given the SMILES representations of two drugs, the goal is to predict the type of interaction between them<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p></li>
<li><p id="Par134"><strong>Biosnap-DDI:</strong>The BIOSNAP dataset comprises 1,322 FDA-approved drugs with 41,520 labeled drug-drug interactions (DDIs). In contrast, the Zhang-DDI dataset includes 548 drugs and 48,584 pairwise DDIs. Both datasets are used for multi-DDI prediction tasks, where the objective is to identify specific interaction types between drug pairs<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p></li>
</ol></section><section id="Sec48"><h3 class="pmc_sec_title">Dataset characteristics</h3>
<p id="Par135">In Figure <a href="#Fig4" class="usa-link">4</a> we have shown ddi interaction adjacency plot for Ogbl-ddi dataset. In Figure <a href="#Fig5" class="usa-link">5</a> we have shown the label distribution for DrugBank ddi and Biosnap DDI. The figure shows these these are balanced class. For Ogbl-ddi dataset only interaction graph were present. To make it balanced we have done global sampling with equal number of true class for training the data. In this section we have anlysed various characteristics of our datasets. Although in the following analysis we have only considered two datasets because the have provided drug smiles in their datasets.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/a76cec09d9a9/41598_2025_12936_Fig4_HTML.jpg" loading="lazy" id="MO6" height="664" width="749" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Whole (un-directed) graph adjacency matrix plot. Each node represents a drug. The dots show there is an interaction between two drugs.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/bbed42ee2d6e/41598_2025_12936_Fig5_HTML.jpg" loading="lazy" id="MO7" height="208" width="668" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This is label distribution of both the datasets. Which means both datasets are balanced.</p></figcaption></figure></section><section id="Sec49"><h3 class="pmc_sec_title">Tonimoto similarity</h3>
<p id="Par136">Tanimoto similarity, is a widely used metric in cheminformatics and drug discovery to quantify the structural similarity between two molecules. Once two molecules are represented by their fingerprints (let’s call them A and B), the Tanimoto coefficient (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/7fd51490fc3e/d33e1930.gif" loading="lazy" id="d33e1930" alt="Inline graphic"></span>) is calculated using the following formula:</p>
<table class="disp-formula p" id="Equ15"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/fe113e12388d/d33e1936.gif" loading="lazy" id="d33e1936" alt="graphic file with name d33e1936.gif"></td></tr></table>
<p>Where:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par137"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/13d6ad983fb9/d33e1945.gif" loading="lazy" id="d33e1945" alt="Inline graphic"></span> is the number of bits (features) that are “on” (present) in <em>both</em> fingerprint A and fingerprint B.</p></li>
<li><p id="Par138"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/cfa86e9251fe/d33e1956.gif" loading="lazy" id="d33e1956" alt="Inline graphic"></span> is the total number of “on” bits in fingerprint A.</p></li>
<li><p id="Par139"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/bb5e56fe4b59/d33e1964.gif" loading="lazy" id="d33e1964" alt="Inline graphic"></span> is the total number of “on” bits in fingerprint B.</p></li>
</ul>
<p>The Tanimoto coefficient ranges from 0 to 1, where:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par140">1 indicates perfect structural similarity (the fingerprints are identical).</p></li>
<li><p id="Par141">0 indicates no shared features (the fingerprints have no “on” bits in common).</p></li>
</ul>
<section id="Sec50"><h4 class="pmc_sec_title">Impact on Drug-Drug Interaction (DDI)</h4>
<p id="Par142">The core principle behind using Tanimoto similarity in these contexts is the <strong>“similarity principle”</strong>: <em>structurally similar molecules are likely to have similar properties and biological activities.</em></p>
<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par143">
<strong>Prediction of DDIs:</strong> Tanimoto similarity is a key tool in predicting potential drug-drug interactions. The assumption is:<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par144">If Drug A interacts with Drug B, and Drug C is structurally very similar to Drug A, then Drug C is also likely to interact with Drug B.</div></li>
<li><div class="p" id="Par145">Similarly, if two drugs have similar molecular structures, they might compete for the same metabolic enzymes (e.g., CYP enzymes) or transporters, leading to altered concentrations and effects of one or both drugs.</div></li>
</ul>
</div></li>
<li><div class="p" id="Par146">
<strong>Mechanisms:</strong> DDIs can arise from various mechanisms (pharmacokinetic or pharmacodynamic). Structural similarity can shed light on:<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par147">
<strong>Pharmacokinetic DDIs:</strong> If two drugs are structurally similar, they might be metabolized by the same enzymes or transported by the same proteins, leading to competitive inhibition, induction, or altered clearance, which can increase or decrease drug levels in the body.</div></li>
<li><div class="p" id="Par148">
<strong>Pharmacodynamic DDIs:</strong> While less direct than pharmacokinetic interactions, highly similar drugs might have similar off-target effects or bind to the same receptors, leading to additive or antagonistic effects.</div></li>
</ul>
</div></li>
</ul>
<p>In Figure <a href="#Fig6" class="usa-link">6</a> we have shown the Tonimoto similarity index for both the datasets.</p>
<figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/92b1718b16a1/41598_2025_12936_Fig6_HTML.jpg" loading="lazy" id="MO8" height="243" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tonimoto similarity between drugs for both the datasets.</p></figcaption></figure></section></section><section id="Sec51"><h3 class="pmc_sec_title">Molecular weight distribution</h3>
<p id="Par149">The Figure <a href="#Fig7" class="usa-link">7</a> shows molecular weight distribution for both the datasets. Molecular weight affects drug-drug and drug-target interactions by influencing absorption, distribution, binding affinity, and permeability. Generally, higher molecular weight can reduce membrane permeability, while also affecting how tightly or selectively a drug binds to its target or interacts with other drugs.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/211aa97bdf74/41598_2025_12936_Fig7_HTML.jpg" loading="lazy" id="MO9" height="237" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Molecular weight distribution for both the datasets.</p></figcaption></figure></section><section id="Sec52"><h3 class="pmc_sec_title">Average hydrophobicity (logP) of each drug pair</h3>
<p id="Par150">Molecular hydrophobicity affects drug-drug and drug-target interactions by influencing binding affinity, membrane permeability, and solubility. Hydrophobic drugs often bind more strongly to hydrophobic pockets on targets but may have poor solubility and higher chances of off-target or nonspecific interactions. In Figure <a href="#Fig8" class="usa-link">8</a> we have shown the average hydrophobicity for two datasets.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365209_41598_2025_12936_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/73011cd93727/41598_2025_12936_Fig8_HTML.jpg" loading="lazy" id="MO10" height="232" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Average hydrophobicity of each drug pair.</p></figcaption></figure></section><section id="Sec53"><h3 class="pmc_sec_title">Data set partitioning</h3>
<p id="Par151">The drug edges in the dataset are segmented based on the specific proteins they target in the body. As a result, the test set consists of drugs that primarily bind to proteins distinct from those in the training and validation sets. This segmentation ensures that the drugs in the test set exhibit different biological mechanisms of action and interact with distinct proteins compared to those in the training and validation sets. Protein-target segmentation, therefore, provides an opportunity to assess the model’s ability to make meaningful and generalizable predictions that are not limited to the specific proteins seen during training<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>.</p></section><section id="Sec54"><h3 class="pmc_sec_title">Experimental tasks</h3>
<p id="Par152">The experimental task focuses on predicting drug-drug interactions (DDIs) by ranking each known positive interaction against approximately 100,000 randomly sampled negative interactions. The primary evaluation metrics are Hits@10, Hits@20, and Hits@50, which measure the proportion of true positive interactions ranked within the top <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2aad2085578d/d33e2068.gif" loading="lazy" id="d33e2068" alt="Inline graphic"></span> predictions. These metrics assess the model’s ability to correctly prioritize true interactions among a large set of candidates. In addition to Hits@K, we also report accuracy to provide a broader view of model performance across the entire prediction space. Prior studies have indicated that evaluating performance at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2f7cc147de92/d33e2074.gif" loading="lazy" id="d33e2074" alt="Inline graphic"></span> is particularly meaningful in DDI prediction tasks, but we extend this analysis to smaller and larger <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c62c/12365209/2aad2085578d/d33e2080.gif" loading="lazy" id="d33e2080" alt="Inline graphic"></span> values for a more comprehensive evaluation.</p></section><section id="Sec55"><h3 class="pmc_sec_title">Experimental environment</h3>
<p id="Par153">The experimental code environment is based on Python version 3.10.8 and PyTorch version 2.1.2+cu118. The version of torch-scatter is 2.1.2+pt21cu118, torch-cluster is 1.6.3+pt21cu118, the version of torch-sparse is 0.6.18+pt21cu118, torch-geometric is 2.6.0, ogbl is 1.3.6, dgl is 2.4.0+cu118, and dgl-life is 0.3.2.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the Key Scientific and Technological Research Projects in Henan Province under Grant 202102210379 and also by Zhoukou Normal University super scientific project grant <em>ZKNUC</em>2018019. We would also like to acknowledge the assistance of language models such as ChatGPT in enhancing the readability and clarity of our manuscript. The use of these tools has significantly contributed to refining the language and ensuring that our research is communicated effectively.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Manuscript Draft, investigation, methodology, Data analysis and processing, Software, Validation, : K. Abbas, C. Hao, M.K. Hasan, X. Yong, S, Islam, A.H. Rahman:. Review and editing, investigation, methodology, Software.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the Key Scientific and Technological Research Projects in Henan Province under Grant 202102210379 and also by Zhoukou Normal University super scientific project grant <em>ZKNUC</em>2018019.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All the data are publically available and described as below: The online dataset can be downloaded using python obg package. The further information is provided on Stanford University website: <a href="https://ogb.stanford.edu/docs/linkprop/#ogbl-ddi" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ogb.stanford.edu/docs/linkprop/#ogbl-ddi</a> The datasets used and/or analyzed during the current study are available in the OGB Repository. The tutorial how to use the data can be found here: <a href="https://ogb.stanford.edu/docs/home/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ogb.stanford.edu/docs/home/</a></p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par154">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Khushnood Abbas, Email: Khushnood.abbas@zknu.edu.cn.</p>
<p>Chen Hao, Email: chenhao10242024@163.com.</p>
<p>Mohammad Kamrul Hasan, Email: mkhasan@ukm.edu.my.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Jianqing, T. Occurrence of adverse reactions related to macrolide antibiotics and guidance measures. <em>North. Pharm.</em> 019 (2022).</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Zhu, J. et al. Prediction of drug efficacy from transcriptional profiles with deep learning. <em>Nat. biotechnology</em><strong>39</strong>, 1444–1452 (2021).</cite> [<a href="https://doi.org/10.1038/s41587-021-00946-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34140681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20J.%20et%20al.%20Prediction%20of%20drug%20efficacy%20from%20transcriptional%20profiles%20with%20deep%20learning.%20Nat.%20biotechnology39,%201444%E2%80%931452%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Wang, F., Lei, X., Liao, B. &amp; Wu, F.-X. Predicting drug-drug interactions by graph convolutional network with multi-kernel. <em>Briefings Bioinforma.</em><strong>23</strong>, bbab511 (2022).</cite> [<a href="https://doi.org/10.1093/bib/bbab511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34864856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20F.,%20Lei,%20X.,%20Liao,%20B.%20&amp;%20Wu,%20F.-X.%20Predicting%20drug-drug%20interactions%20by%20graph%20convolutional%20network%20with%20multi-kernel.%20Briefings%20Bioinforma.23,%20bbab511%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Gao, J., Wu, Z., Al-Sabri, R., Oloulade, B. M. &amp; Chen, J. Autoddi: Drug–drug interaction prediction with automated graph neural network. <em>IEEE J. Biomed. Heal. Informatics</em><strong>28</strong>, 1773–1784. 10.1109/JBHI.2024.3349570 (2024).</cite> [<a href="https://doi.org/10.1109/JBHI.2024.3349570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38446657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20J.,%20Wu,%20Z.,%20Al-Sabri,%20R.,%20Oloulade,%20B.%20M.%20&amp;%20Chen,%20J.%20Autoddi:%20Drug%E2%80%93drug%20interaction%20prediction%20with%20automated%20graph%20neural%20network.%20IEEE%20J.%20Biomed.%20Heal.%20Informatics28,%201773%E2%80%931784.%2010.1109/JBHI.2024.3349570%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Geng, G. et al. Mgddi: A multi-scale graph neural networks for drug–drug interaction prediction. <em>Methods</em><strong>228</strong>, 22–29 (2024).
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2024.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38754712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geng,%20G.%20et%20al.%20Mgddi:%20A%20multi-scale%20graph%20neural%20networks%20for%20drug%E2%80%93drug%20interaction%20prediction.%20Methods228,%2022%E2%80%9329%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Feng, Y.-H. &amp; Zhang, S.-W. Prediction of drug-drug interaction using an attention-based graph neural network on drug molecular graphs. <em>Molecules</em><strong>27</strong> (2022).</cite> [<a href="https://doi.org/10.3390/molecules27093004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9105425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35566354/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Han, X., Xie, R., Li, X. &amp; Li, J. Smilegnn: Drug–drug interaction prediction based on the smiles and graph neural network. <em>Life</em><strong>12</strong> (2022).</cite> [<a href="https://doi.org/10.3390/life12020319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8879716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35207606/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Liu, S. et al. Enhancing drug-drug interaction prediction using deep attention neural networks. <em>IEEE/ACM transactions on computational biology and Bioinformatics</em><strong>20</strong>, 976–985 (2022).</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3172421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35511833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20S.%20et%20al.%20Enhancing%20drug-drug%20interaction%20prediction%20using%20deep%20attention%20neural%20networks.%20IEEE/ACM%20transactions%20on%20computational%20biology%20and%20Bioinformatics20,%20976%E2%80%93985%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Li, Z., Tu, X., Chen, Y. &amp; Lin, W. HetDDI: a pre-trained heterogeneous graph neural network model for drug-drug interaction prediction. <em>Briefings Bioinforma.</em><strong>24</strong>, bbad385 (2023).</cite> [<a href="https://doi.org/10.1093/bib/bbad385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37903412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.,%20Tu,%20X.,%20Chen,%20Y.%20&amp;%20Lin,%20W.%20HetDDI:%20a%20pre-trained%20heterogeneous%20graph%20neural%20network%20model%20for%20drug-drug%20interaction%20prediction.%20Briefings%20Bioinforma.24,%20bbad385%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Purkayastha, S., Mondal, I., Sarkar, S., Goyal, P. &amp; Pillai, J. K. Drug-drug interactions prediction based on drug embedding and graph auto-encoder. In <em>2019 IEEE 19th International Conference on Bioinformatics and Bioengineering (BIBE)</em>, 547–552 (2019).</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Shtar, G., Rokach, L. &amp; Shapira, B. Detecting drug-drug interactions using artificial neural networks and classic graph similarity measures. <em>PloS one</em><strong>14</strong>, e0219796 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0219796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6675052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31369568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shtar,%20G.,%20Rokach,%20L.%20&amp;%20Shapira,%20B.%20Detecting%20drug-drug%20interactions%20using%20artificial%20neural%20networks%20and%20classic%20graph%20similarity%20measures.%20PloS%20one14,%20e0219796%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Lei, Z.-C., Jiang, J.-J., Ma, C.-Z., Zhou, W.-J. &amp; Wang, C.-Z. Drug-drug interaction prediction based on neighborhood relation-aware graph neural network. <em>Comput. Eng. &amp; Sci.</em><strong>46</strong>, 907 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lei,%20Z.-C.,%20Jiang,%20J.-J.,%20Ma,%20C.-Z.,%20Zhou,%20W.-J.%20&amp;%20Wang,%20C.-Z.%20Drug-drug%20interaction%20prediction%20based%20on%20neighborhood%20relation-aware%20graph%20neural%20network.%20Comput.%20Eng.%20&amp;%20Sci.46,%20907%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Feng, Y.-H., Zhang, S.-W. &amp; Shi, J.-Y. Dpddi: a deep predictor for drug-drug interactions. <em>BMC bioinformatics</em><strong>21</strong>, 419 (2020).
</cite> [<a href="https://doi.org/10.1186/s12859-020-03724-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7513481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32972364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng,%20Y.-H.,%20Zhang,%20S.-W.%20&amp;%20Shi,%20J.-Y.%20Dpddi:%20a%20deep%20predictor%20for%20drug-drug%20interactions.%20BMC%20bioinformatics21,%20419%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Saifuddin, K. M., Bumgardner, B., Tanvir, F. &amp; Akbas, E. Hygnn: Drug-drug interaction prediction via hypergraph neural network. In <em>2023 IEEE 39th International Conference on Data Engineering (ICDE)</em>, 1503–1516 (IEEE, 2023).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Lin, J., Hong, B., Cai, Z., Lu, P. &amp; Lin, K. Masmddi: multi-layer adaptive soft-mask graph neural network for drug-drug interaction prediction. <em>Front. Pharmacol.</em><strong>15</strong>, 1369403 (2024).
</cite> [<a href="https://doi.org/10.3389/fphar.2024.1369403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11144894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38831885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20J.,%20Hong,%20B.,%20Cai,%20Z.,%20Lu,%20P.%20&amp;%20Lin,%20K.%20Masmddi:%20multi-layer%20adaptive%20soft-mask%20graph%20neural%20network%20for%20drug-drug%20interaction%20prediction.%20Front.%20Pharmacol.15,%201369403%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Wang, T., Wang, R. &amp; Wei, L. Attensyn: an attention-based deep graph neural network for anticancer synergistic drug combination prediction. <em>J. Chem. Inf. Model.</em><strong>64</strong>, 2854–2862 (2023).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.3c00709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37565997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20T.,%20Wang,%20R.%20&amp;%20Wei,%20L.%20Attensyn:%20an%20attention-based%20deep%20graph%20neural%20network%20for%20anticancer%20synergistic%20drug%20combination%20prediction.%20J.%20Chem.%20Inf.%20Model.64,%202854%E2%80%932862%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wu, D., Sun, W., He, Y., Chen, Z. &amp; Luo, X. Mkg-fenn: A multimodal knowledge graph fused end-to-end neural network for accurate drug-drug interaction prediction. <em>In Proceedings of the AAAI Conference on Artificial Intelligence</em><strong>38</strong>, 10216–10224 (2024).</cite>
</li>
<li id="CR18">
<span class="label">18.</span><cite>Yu, H., Zhao, S. &amp; Shi, J. Stnn-ddi: a substructure-aware tensor neural network to predict drug-drug interactions. <em>Briefings Bioinforma.</em><strong>23</strong>, bbac209 (2022).</cite> [<a href="https://doi.org/10.1093/bib/bbac209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35667078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20H.,%20Zhao,%20S.%20&amp;%20Shi,%20J.%20Stnn-ddi:%20a%20substructure-aware%20tensor%20neural%20network%20to%20predict%20drug-drug%20interactions.%20Briefings%20Bioinforma.23,%20bbac209%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Kumari, D. <em>et al.</em> A study on improving drug-drug interactions prediction using convolutional neural networks. <em>Appl. Soft Comput.</em> 112242 (2024).</cite>
</li>
<li id="CR20">
<span class="label">20.</span><cite>Su, X., Hu, L., You, Z., Hu, P. &amp; Zhao, B. Attention-based knowledge graph representation learning for predicting drug-drug interactions. <em>Briefings in bioinformatics</em><strong>23</strong>, bbac140 (2022).
</cite> [<a href="https://doi.org/10.1093/bib/bbac140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35453147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20X.,%20Hu,%20L.,%20You,%20Z.,%20Hu,%20P.%20&amp;%20Zhao,%20B.%20Attention-based%20knowledge%20graph%20representation%20learning%20for%20predicting%20drug-drug%20interactions.%20Briefings%20in%20bioinformatics23,%20bbac140%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Feng, Y.-H., Zhang, S.-W., Zhang, Q.-Q., Zhang, C.-H. &amp; Shi, J.-Y. deepmddi: A deep graph convolutional network framework for multi-label prediction of drug-drug interactions. <em>Anal. Biochem.</em><strong>646</strong>, (2022).</cite> [<a href="https://doi.org/10.1016/j.ab.2022.114631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35227661/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wang, Y., Min, Y., Chen, X. &amp; Wu, J. Multi-view graph contrastive representation learning for drug-drug interaction prediction. <em>In Proceedings of the web conference</em><strong>2021</strong>, 2921–2933 (2021).</cite>
</li>
<li id="CR23">
<span class="label">23.</span><cite>Masumshah, R., Aghdam, R. &amp; Eslahchi, C. A neural network-based method for polypharmacy side effects prediction. <em>BMC Bioinforma. </em><strong>22 </strong>(2021).</cite> [<a href="https://doi.org/10.1186/s12859-021-04298-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8305591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34303360/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Masumshah, R. &amp; Eslahchi, C. Dpsp: a multimodal deep learning framework for polypharmacy side effects prediction. <em>Bioinforma. Adv.</em> <strong>3</strong> (2023).</cite> [<a href="https://doi.org/10.1093/bioadv/vbad110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10493180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37701676/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Yang, X., lbrahim Zamit, Liu, Y. &amp; He, J. Additional neural matrix factorization model for computational drug repositioning. <em>BMC Bioinforma.</em> <strong>20</strong> (2019).</cite> [<a href="https://doi.org/10.1186/s12859-019-2983-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6694624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31412762/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Yang, X., ke Yang, G. &amp; Chu, J. Graphcl-dta: A graph contrastive learning with molecular semantics for drug-target binding affinity prediction. <em>IEEE J. Biomed. Heal. Informatics</em><strong>28</strong>, 4544–4552 (2023).</cite> [<a href="https://doi.org/10.1109/JBHI.2024.3350666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38190664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20X.,%20ke%20Yang,%20G.%20&amp;%20Chu,%20J.%20Graphcl-dta:%20A%20graph%20contrastive%20learning%20with%20molecular%20semantics%20for%20drug-target%20binding%20affinity%20prediction.%20IEEE%20J.%20Biomed.%20Heal.%20Informatics28,%204544%E2%80%934552%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>He, J., Yang, X., Gong, Z. &amp; lbrahim Zamit. Hybrid attentional memory network for computational drug repositioning. <em>BMC Bioinforma.</em> <strong>21</strong> (2020).</cite> [<a href="https://doi.org/10.1186/s12859-020-03898-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7724880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33297947/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Yang, X., ke Yang, G. &amp; Chu, J. Self-supervised learning for label sparsity in computational drug repositioning. <em>IEEE/ACM Transactions on Comput. Biol. Bioinforma.</em><strong>20</strong>, 3245–3256 (2022).</cite> [<a href="https://doi.org/10.1109/TCBB.2023.3254163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37028367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20X.,%20ke%20Yang,%20G.%20&amp;%20Chu,%20J.%20Self-supervised%20learning%20for%20label%20sparsity%20in%20computational%20drug%20repositioning.%20IEEE/ACM%20Transactions%20on%20Comput.%20Biol.%20Bioinforma.20,%203245%E2%80%933256%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Yang, X., ke Yang, G. &amp; Chu, J. The neural metric factorization for computational drug repositioning. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em><strong>20</strong>, 731–741 (2021).</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3144429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35061591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20X.,%20ke%20Yang,%20G.%20&amp;%20Chu,%20J.%20The%20neural%20metric%20factorization%20for%20computational%20drug%20repositioning.%20IEEE/ACM%20Transactions%20on%20Computational%20Biology%20and%20Bioinformatics20,%20731%E2%80%93741%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Yang, X., ke Yang, G. &amp; Chu, J. The computational drug repositioning without negative sampling. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em><strong>20</strong>, 1506–1517 (2021).</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3212051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36197871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20X.,%20ke%20Yang,%20G.%20&amp;%20Chu,%20J.%20The%20computational%20drug%20repositioning%20without%20negative%20sampling.%20IEEE/ACM%20Transactions%20on%20Computational%20Biology%20and%20Bioinformatics20,%201506%E2%80%931517%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Velickovic, P. et al. Graph attention networks. <em>stat</em><strong>1050</strong>, 10–48550 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Velickovic,%20P.%20et%20al.%20Graph%20attention%20networks.%20stat1050,%2010%E2%80%9348550%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Kipf, T. N. &amp; Welling, M. Semi-supervised classification with graph convolutional networks. arXiv preprint <a href="http://arxiv.org/abs/1609.02907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:1609.02907</a> (2016).</cite>
</li>
<li id="CR33">
<span class="label">33.</span><cite>Hamilton, W., Ying, Z. &amp; Leskovec, J. Inductive representation learning on large graphs. <em>Advances in neural information processing systems</em><strong>30</strong> (2017).</cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>Abu-El-Haija, S. <em>et al.</em> Mixhop: Higher-order graph convolutional architectures via sparsified neighborhood mixing. In Chaudhuri, K. &amp; Salakhutdinov, R. (eds.) <em>Proceedings of the 36th International Conference on Machine Learning, ICML 2019, 9-15 June 2019, Long Beach, California, USA</em>, vol. 97 of <em>Proceedings of Machine Learning Research</em>, 21–29 (PMLR, 2019).</cite>
</li>
<li id="CR35">
<span class="label">35.</span><cite>Du, J., Zhang, S., Wu, G., Moura, J. M. &amp; Kar, S. Topology adaptive graph convolutional networks. arXiv preprint <a href="http://arxiv.org/abs/1710.10370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:1710.10370</a> (2017).</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Wang, Z. <em>et al.</em> Gidn: A lightweight graph inception diffusion network for high-efficient link prediction. arXiv preprint <a href="http://arxiv.org/abs/2210.01301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:2210.01301</a> (2022).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Sun, C., Hu, J., Gu, H., Chen, J. &amp; Yang, M. Adaptive graph diffusion networks. (2020). <a href="http://arxiv.org/abs/2012.15024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:2012.15024</a>.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Siddiqui, A. GNN-Based Link Prediction in Drug-Drug Interaction Networks — medium.com. <em>Medium.com</em> (2023). [Accessed 21-05-2024].</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Kingma, D. P. &amp; Ba, J. Adam: A method for stochastic optimization. arXiv preprint <a href="http://arxiv.org/abs/1412.6980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:1412.6980</a> (2014).</cite>
</li>
<li id="CR40">
<span class="label">40.</span><cite>Wishart, D. S. et al. Drugbank 5.0: a major update to the drugbank database for 2018. <em>Nucleic acids research</em><strong>46</strong>, D1074–D1082 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gkx1037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29126136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wishart,%20D.%20S.%20et%20al.%20Drugbank%205.0:%20a%20major%20update%20to%20the%20drugbank%20database%20for%202018.%20Nucleic%20acids%20research46,%20D1074%E2%80%93D1082%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Guney, E. Reproducible drug repurposing: When similarity does not suffice. In <em>Pacific Symposium on Biocomputing 2017</em>, 132–143 (World Scientific, 2017).</cite> [<a href="https://doi.org/10.1142/9789813207813_0014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27896969/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Ryu, J. Y., Kim, H. U. &amp; Lee, S. Y. Deep learning improves prediction of drug–drug and drug–food interactions. <em>Proc. Natl. Acad. Sci.</em> <strong>115</strong>, E4304–E4311 (2018).</cite> [<a href="https://doi.org/10.1073/pnas.1803294115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5939113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29666228/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yu, Y. et al. Sumgnn: multi-typed drug interaction prediction via efficient knowledge graph summarization. <em>Bioinformatics</em><strong>37</strong>, 2988–2995 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10060701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33769494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20Y.%20et%20al.%20Sumgnn:%20multi-typed%20drug%20interaction%20prediction%20via%20efficient%20knowledge%20graph%20summarization.%20Bioinformatics37,%202988%E2%80%932995%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All the data are publically available and described as below: The online dataset can be downloaded using python obg package. The further information is provided on Stanford University website: <a href="https://ogb.stanford.edu/docs/linkprop/#ogbl-ddi" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ogb.stanford.edu/docs/linkprop/#ogbl-ddi</a> The datasets used and/or analyzed during the current study are available in the OGB Repository. The tutorial how to use the data can be found here: <a href="https://ogb.stanford.edu/docs/home/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ogb.stanford.edu/docs/home/</a></p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-12936-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_12936.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365209/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365209/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365209%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365209/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365209/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365209/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830547/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365209/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830547/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365209/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365209/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="TfZH8AYJhrkGqnTtZ9bA5VGUUuAOusLoLCCSzziiosHUA4to1Q1QoUw9xjKYeMWT">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
