
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The PERPETUAL FLOWERING locus: Necessary but insufficient for genomic prediction of runnerless and other asexual reproduction phenotypes in strawberry - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251BBE8AEEEDD3051BBE003724904D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365471/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="The PERPETUAL FLOWERING locus: Necessary but insufficient for genomic prediction of runnerless and other asexual reproduction phenotypes in strawberry">
<meta name="citation_author" content="Hillel Brukental">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Marta L Bjornson">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Dominique D A Pincot">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Michael A Hardigan">
<meta name="citation_author_institution" content="USDA‐ARS Horticultural Crops Production and Genetic Improvement, Corvallis, Oregon, USA">
<meta name="citation_author" content="Sadikshya Sharma">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Nicolas P Jiménez">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Randi A Famula">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Cindy M Lòpez">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Glenn S Cole">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Mitchell J Feldmann">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_author" content="Steven J Knapp">
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California, USA">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70086">
<meta name="citation_doi" content="10.1002/tpg2.70086">
<meta name="citation_pmid" content="40831148">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365471/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365471/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365471/pdf/TPG2-18-e70086.pdf">
<meta name="description" content="Strawberry (Fragaria × ananassa) reproduces sexually through seeds and asexually through stolons. The ability to cost‐effectively clonally propagate hybrid individuals on a large scale has shaped strawberry breeding and production practices. Despite ...">
<meta name="og:title" content="The PERPETUAL FLOWERING locus: Necessary but insufficient for genomic prediction of runnerless and other asexual reproduction phenotypes in strawberry">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Strawberry (Fragaria × ananassa) reproduces sexually through seeds and asexually through stolons. The ability to cost‐effectively clonally propagate hybrid individuals on a large scale has shaped strawberry breeding and production practices. Despite ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365471/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365471">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70086"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70086.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365471%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365471/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365471/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365471/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70086" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 19;18(3):e70086. doi: <a href="https://doi.org/10.1002/tpg2.70086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70086</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The <em>PERPETUAL FLOWERING</em> locus: Necessary but insufficient for genomic prediction of runnerless and other asexual reproduction phenotypes in strawberry</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brukental%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hillel Brukental</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hillel Brukental</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brukental%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hillel Brukental</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bjornson%20ML%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Marta L Bjornson</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Marta L Bjornson</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bjornson%20ML%22%5BAuthor%5D" class="usa-link"><span class="name western">Marta L Bjornson</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pincot%20DDA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Dominique D A Pincot</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Dominique D A Pincot</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pincot%20DDA%22%5BAuthor%5D" class="usa-link"><span class="name western">Dominique D A Pincot</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hardigan%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Michael A Hardigan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Michael A Hardigan</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Horticultural Crops Production and Genetic Improvement, Corvallis, Oregon, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hardigan%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael A Hardigan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sadikshya Sharma</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sadikshya Sharma</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sadikshya Sharma</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jim%C3%A9nez%20NP%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Nicolas P Jiménez</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Nicolas P Jiménez</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jim%C3%A9nez%20NP%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicolas P Jiménez</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Famula%20RA%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Randi A Famula</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Randi A Famula</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Famula%20RA%22%5BAuthor%5D" class="usa-link"><span class="name western">Randi A Famula</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B2pez%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Cindy M Lòpez</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Cindy M Lòpez</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B2pez%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">Cindy M Lòpez</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cole%20GS%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Glenn S Cole</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Glenn S Cole</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cole%20GS%22%5BAuthor%5D" class="usa-link"><span class="name western">Glenn S Cole</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feldmann%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Mitchell J Feldmann</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Mitchell J Feldmann</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feldmann%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Mitchell J Feldmann</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knapp%20SJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Steven J Knapp</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Steven J Knapp</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knapp%20SJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Steven J Knapp</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270086-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, University of California, Davis, California, USA</div>
<div id="tpg270086-aff-0002">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Horticultural Crops Production and Genetic Improvement, Corvallis, Oregon, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Mitchell J. Feldmann, Department of Plant Sciences, University of California, Davis, CA, USA.  Email: <span>mjfeldmann@ucdavis.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 27; Accepted 2025 Jun 30; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365471  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40831148/" class="usa-link">40831148</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Strawberry (<em>Fragaria</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em>) reproduces sexually through seeds and asexually through stolons. The ability to cost‐effectively clonally propagate hybrid individuals on a large scale has shaped strawberry breeding and production practices. Despite the technical and economic importance of clonal propagation, little is known about the genetic regulation of runnering in strawberry, apart from the pleiotropic effects of <em>PERPETUAL FLOWERING</em> (<em>PF</em>), a dominant gene introgressed from a wild relative that abolishes temperature‐dependent photoperiod sensitivity and incompletely and variably suppresses runnering. Here, we show that runnering phenotypes are heritable and highly variable in strawberry, ranging from runnerless to prolific in short‐day (<em>pfpf</em>) and day‐neutral (<em>PF</em>_) plants. The <em>PF</em> locus was physically mapped to Mb 26.4–27.3 on chromosome 4B and found to explain 22% of the genetic variance for runnering (78% of the heritability was missing). <em>PF</em> was the only runnering‐associated locus identified by genome‐wide association studies among diverse clonal genetic resources and progeny from narrow and wide crosses (1537 individuals). The pleotropic effect of <em>PF</em> on runnering was temporal, variable, and incompletely dominant. Genomic selection was found to be a viable strategy for modifying runnering phenotypes in strawberry. Genomic prediction accuracies ranged from 0.53 to 0.79 for runnering, were greater within than between populations, and increased when corrected for <em>PF</em>. Our study builds the foundation for improving the productivity of strawberry by developing runnerless cultivars for seed‐propagation or reduced runnering cultivars for clonal‐propagation through phenotypic or genomic selection.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270086-list-10001" class="list" style="list-style-type:disc">
<li><p>Variation for runnering is substantial and heritable within seasonal and perpetual flowering strawberry populations.</p></li>
<li><p>The pleiotropic suppression of runnering by the <em>PERPETUAL FLOWERING</em> (<em>PF</em>) allele is continuously variable, temporal, and incompletely dominant.</p></li>
<li><p>The accuracy of breeding value prediction is sufficient for modifying the runner growth characteristics of the strawberry plant by genomic selection.</p></li>
<li><p>The productivity of strawberry can be improved by introducing seed‐propagated cultivars with runnerless and clone‐propagated cultivars with reduced runnering phenotypes.</p></li>
</ul></section><section class="abstract" id="abstract3"><h2>Plain Language Summary</h2>
<p>The ability to clone individual plants has shaped strawberry breeding and production practices, and runner (stolon) growth is well known to home gardeners and farmers alike and important for the propagation of strawberry cultivars. The energy that the strawberry plant diverts into runner growth can decrease fruit yield. To circumvent maximize fruit yield, farmers mechanically trim runners. The genetic factors that control runner growth are complex but mostly unknown apart from <em>PERPETUAL FLOWERING</em>, a gene controlling photoperiod sensitive flowering and affecting runnering. We observed a full spectrum of runner growth phenotypes in seasonal and perpetual flowering populations, found that runnering phenotypes are heritable, and showed that the runner growth of strawberry plants can be modified by phenotypic or genomic selection. Lastly, we identified extreme transgressive segregates for runnering and showed that runnerless and reduced runnering cultivars can be developed to reduce production costs and improve productivity.</p></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270086-dl-0001">
<dt>ARMS</dt>
<dd><p>amplification‐refractory mutation system</p></dd>
<dt>DN</dt>
<dd><p>day‐neutral</p></dd>
<dt>EMM</dt>
<dd><p>estimated marginal mean</p></dd>
<dt>FS</dt>
<dd><p>full sibs</p></dd>
<dt>GEBV</dt>
<dd><p>genomic‐estimated breeding value</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>LD</dt>
<dd><p>linkage disequilibrium</p></dd>
<dt>LMM</dt>
<dd><p>linear mixed model</p></dd>
<dt>
<em>PF</em>
</dt>
<dd><p>
<em>PERPETUAL FLOWERING</em>
</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>REML</dt>
<dd><p>restricted maximum likelihood</p></dd>
<dt>RS</dt>
<dd><p>runner score</p></dd>
<dt>SD</dt>
<dd><p>short‐day</p></dd>
<dt>SDP</dt>
<dd><p>strawberry diversity panel</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>SSR</dt>
<dd><p>simple sequence repeat</p></dd>
</dl>
<section id="tpg270086-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Cultivated strawberry (<em>Fragaria</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em>), a self‐compatible allo‐octoploid (2<em>n</em> = 8<em>x</em> = 56), reproduces sexually through either self‐ or cross‐pollination and asexually through stolons (runners) and plantlets (daughter plants) (Heide et al., <a href="#tpg270086-bib-0037" class="usa-link" aria-describedby="tpg270086-bib-0037">2013</a>; Hytönen &amp; Kurokura, <a href="#tpg270086-bib-0043" class="usa-link" aria-describedby="tpg270086-bib-0043">2020</a>; Voth &amp; Bringhurst, <a href="#tpg270086-bib-0077" class="usa-link" aria-describedby="tpg270086-bib-0077">1990</a>). That reproductive flexibility has shaped breeding and cultivar development practices, which rely on the clonal propagation of parents and progeny (Darrow, <a href="#tpg270086-bib-0017" class="usa-link" aria-describedby="tpg270086-bib-0017">1966</a>; Voth &amp; Bringhurst, <a href="#tpg270086-bib-0077" class="usa-link" aria-describedby="tpg270086-bib-0077">1990</a>). Strawberry cultivars are nearly always clonally propagated hybrid (F<sub>1</sub>) individuals originating from crosses between outbred parents with shared ancestry and complex, intertwined pedigrees (Darrow, <a href="#tpg270086-bib-0017" class="usa-link" aria-describedby="tpg270086-bib-0017">1966</a>; Feldmann, Pincot, Seymour, et al., <a href="#tpg270086-bib-0026" class="usa-link" aria-describedby="tpg270086-bib-0026">2024</a>; Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>; D. D. A. Pincot et al., <a href="#tpg270086-bib-0059" class="usa-link" aria-describedby="tpg270086-bib-0059">2021</a>).</p>
<p>While breeding has traditionally and necessarily focused on improving fruit quality and production traits in strawberry, adequate runner growth and clone yield (nursery production traits) are necessary for the cost‐effective clonal propagation of cultivars, especially those grown on a large‐scale for annual production systems. Uneconomical clone yields have generally not been a problem for modern short‐day (SD) or day‐neutral (DN) cultivars, which are classified by the absence or presence of flowering under long‐days (14.0–14.9 h), respectively, because most runner sufficiently in nursery production environments; however, some runner weakly, particularly modern DN cultivars purposefully selected for reduced runnering in fruit production environments, for example, the reduced runnering DN cultivar UCD Moxie (US Plant Patent #2019/0380247 P1).</p>
<p>Despite the economic and agricultural importance of asexual reproduction in strawberry, stolon and plantlet growth divert energy and nutrients away from sexual reproduction, alter plant architecture, and are unnecessary and unwanted in fruit‐bearing plants (Heide et al., <a href="#tpg270086-bib-0037" class="usa-link" aria-describedby="tpg270086-bib-0037">2013</a>; Hytönen &amp; Kurokura, <a href="#tpg270086-bib-0043" class="usa-link" aria-describedby="tpg270086-bib-0043">2020</a>; Sønsteby et al., <a href="#tpg270086-bib-0069" class="usa-link" aria-describedby="tpg270086-bib-0069">2021</a>; Voth &amp; Bringhurst, <a href="#tpg270086-bib-0077" class="usa-link" aria-describedby="tpg270086-bib-0077">1990</a>). The runners that emerge in fruit production fields are typically removed by hand to maximize fruit yield, whereas flowers that emerge in bare‐root plant (nursery) production fields are commonly removed by hand to maximize daughter plant (clone) yield. The ideal cultivars for clone‐propagated production systems produce economically viable clone yields in nursery production and an absolute bare minimum of runners in fruit production. That ideal can be difficult to achieve in practice because clone and fruit yield are negatively genetically correlated, although that has not been unequivocally substantiated, partly because runners are normally trimmed in fruit‐bearing plants to negate the pleiotropic effect of runner growth on fruit yield. The ideal cultivars for seed‐propagated production systems are runnerless. Although runnerless phenotypes have long been known in the octoploid, the genetic determinants of runnerless phenotypes are unknown and breeding for runnerless and reduced runnering phenotypes has scarcely been documented (Darrow, <a href="#tpg270086-bib-0016" class="usa-link" aria-describedby="tpg270086-bib-0016">1929</a>, <a href="#tpg270086-bib-0017" class="usa-link" aria-describedby="tpg270086-bib-0017">1966</a>).</p>
<p>Little is known about the genetic regulation of asexual reproduction or the heritability of runnering or clone yield variation within SD and DN populations of octoploid strawberry, apart from the pronounced and well documented temporal pleiotropic effects of the <em>PERPETUAL FLOWERING</em> (<em>PF</em>) locus on sexual and asexual reproduction (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>; Bringhurst and Voth, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>; Bringhurst et al., <a href="#tpg270086-bib-0004" class="usa-link" aria-describedby="tpg270086-bib-0004">1989</a>; Castro et al., <a href="#tpg270086-bib-0012" class="usa-link" aria-describedby="tpg270086-bib-0012">2015</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Hossain et al., <a href="#tpg270086-bib-0041" class="usa-link" aria-describedby="tpg270086-bib-0041">2019</a>). Those pleiotropic effects are apparent and exposed when progeny of crosses between SD (<em>pfpf</em>) and DN (<em>PF_</em>) parents are grown under long days (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>).</p>
<p>The DN cultivars developed at University of California (UC) Davis (UCD) and many others around the globe are descendants of a hybrid between the SD cultivar Shasta and the <em>Fragaria virginiana</em> spp. <em>glauca</em> ecotype Wasatch (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst and Voth, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>). That iconic ecotype, collected from the Wasatch Mountains of Utah in 1953 by Royce S. Bringhurst, became the backbone of the UCD DN breeding program (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>; Bringhurst et al., <a href="#tpg270086-bib-0004" class="usa-link" aria-describedby="tpg270086-bib-0004">1989</a>; Feldmann, Pincot, Cole, et al., <a href="#tpg270086-bib-0027" class="usa-link" aria-describedby="tpg270086-bib-0027">2024</a>; J. Hancock et al., <a href="#tpg270086-bib-0031" class="usa-link" aria-describedby="tpg270086-bib-0031">2008</a>, J. F. Hancock et al., <a href="#tpg270086-bib-0033" class="usa-link" aria-describedby="tpg270086-bib-0033">2018</a>; D. D. A. Pincot et al., <a href="#tpg270086-bib-0059" class="usa-link" aria-describedby="tpg270086-bib-0059">2021</a>). Bringhurst et al. (<a href="#tpg270086-bib-0004" class="usa-link" aria-describedby="tpg270086-bib-0004">1989</a>) and Ahmadi et al. (<a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>) showed that DN descendants of the original hybrid (Shasta <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><mo>×</mo></mrow></math></span> Wasatch) inherited a dominant <em>PF</em> allele from Wasatch that is necessary for flowering under long days. The Wasatch <em>PF</em> allele, which has been indispensable and widely used in the development of perpetual flowering (DN) cultivars, doubled fruit yields and transformed strawberry production in California (Feldmann, Pincot, Cole, et al., <a href="#tpg270086-bib-0027" class="usa-link" aria-describedby="tpg270086-bib-0027">2024</a>; J. F. Hancock, <a href="#tpg270086-bib-0032" class="usa-link" aria-describedby="tpg270086-bib-0032">2006</a>; J. F. Hancock et al., <a href="#tpg270086-bib-0033" class="usa-link" aria-describedby="tpg270086-bib-0033">2018</a>).</p>
<p>Although SD and DN plants asexually reproduce under a wide range of conditions, runner proliferation and daughter plant growth are weaker in <em>PF_</em> than <em>pfpf</em> individuals under the long days and high temperatures of summer (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>). Using those conditions, Gaston et al. (<a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>) genetically mapped a quantitative trait locus (QTL) for runnering (<em>PFRU</em>) that colocated with the <em>PF</em> locus. They showed that phenotypic differences for runnering between <em>pfpf</em> and <em>PF</em>_ individuals were exposed under long days and likely caused by the pleiotropic effects of the <em>PF</em> locus. We are highlighting the distinction here because the <em>PF</em> locus was originally identified from the Mendelian distributions of long‐day flowering habit phenotypes (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>; Bringhurst and Voth, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>; Bringhurst et al., <a href="#tpg270086-bib-0004" class="usa-link" aria-describedby="tpg270086-bib-0004">1989</a>), whereas the <em>PFRU</em> QTL was identified from the continuous distributions of runnering and inflorescence count phenotypes (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>).</p>
<p>The effects of <em>PF</em> alleles are photoperiod and temperature dependent (Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Heide et al., <a href="#tpg270086-bib-0037" class="usa-link" aria-describedby="tpg270086-bib-0037">2013</a>; Hytönen &amp; Kurokura, <a href="#tpg270086-bib-0043" class="usa-link" aria-describedby="tpg270086-bib-0043">2020</a>; Koskela et al., <a href="#tpg270086-bib-0046" class="usa-link" aria-describedby="tpg270086-bib-0046">2016</a>; Sønsteby &amp; Heide, <a href="#tpg270086-bib-0068" class="usa-link" aria-describedby="tpg270086-bib-0068">2006</a>); nevertheless, across the normal range of temperatures encountered in environments that dominate fruit production, <em>PFPF</em> and <em>PFpf</em> plants flower continuously, whereas <em>pfpf</em> plants cease flowering in the late spring and early summer as temperatures increase and daylengths approach the summer solstice (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>; Bringhurst et al., <a href="#tpg270086-bib-0004" class="usa-link" aria-describedby="tpg270086-bib-0004">1989</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Heide et al., <a href="#tpg270086-bib-0037" class="usa-link" aria-describedby="tpg270086-bib-0037">2013</a>; Hytönen &amp; Kurokura, <a href="#tpg270086-bib-0043" class="usa-link" aria-describedby="tpg270086-bib-0043">2020</a>). The higher temperatures over the long days of summer typically stimulate runnering in SD and DN plants (Sønsteby and Heide, <a href="#tpg270086-bib-0068" class="usa-link" aria-describedby="tpg270086-bib-0068">2006</a>).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270086-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270086-list-0001" class="list" style="list-style-type:disc">
<li><p>Variation for runnering is substantial and heritable within seasonal and perpetual flowering strawberry populations.</p></li>
<li><p>The pleiotropic suppression of runnering by the <em>PERPETUAL FLOWERING</em> (<em>PF</em>) allele is continuously variable, temporal, and incompletely dominant.</p></li>
<li><p>The accuracy of breeding value prediction is sufficient for modifying the runner growth characteristics of the strawberry plant by genomic selection.</p></li>
<li><p>The productivity of strawberry can be improved by introducing seed‐propagated cultivars with runnerless and clone‐propagated cultivars with reduced runnering phenotypes.</p></li>
</ul></section></section><p>The photoperiod‐dependent inhibition of flowering by the recessive (wildtype) allele in SD (<em>pfpf</em>) plants causes a discrete shift from mixed sexual and asexual reproduction to asexual reproduction only, whereas the dominant (mutant) allele (<em>PF</em>) knocks out the photoperiod‐dependent inhibition of flowering of the wildtype (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>; Bringhurst et al., <a href="#tpg270086-bib-0004" class="usa-link" aria-describedby="tpg270086-bib-0004">1989</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>). DN (<em>PF_</em>) plants flower continuously, exhibit mixed sexual and asexual reproduction over much of their life cycle, and lack the discrete photoperiod‐dependent shift from mixed sexual and asexual reproduction to asexual reproduction only (Sønsteby &amp; Heide, <a href="#tpg270086-bib-0068" class="usa-link" aria-describedby="tpg270086-bib-0068">2006</a>). Sønsteby and Heide (<a href="#tpg270086-bib-0068" class="usa-link" aria-describedby="tpg270086-bib-0068">2006</a>) stressed that DN strawberries are obligatory long‐day plants at high temperatures (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mo>≥</mo></mrow></math></span> 27°C), quantitative long‐day plants at intermediate temperatures, and truly DNs only at cooler temperatures (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><mo>≤</mo></mrow></math></span> 9°C). The term DN, however, has become deeply ingrained in the lexicons of science and commerce to describe perpetual flowering (<em>PF</em>_) plants, and is used throughout this paper for simplicity because temperature‐dependent flowering habit definitions change as temperatures change and because <em>PF</em>_ cultivars encounter a wide range of temperatures over an entire growing season in climates where strawberries are produced on a large scale, for example, coastal California.</p>
<p>While the phenotypic and pleiotropic effects of the <em>PF</em> locus have been widely discussed (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>; Castro et al., <a href="#tpg270086-bib-0012" class="usa-link" aria-describedby="tpg270086-bib-0012">2015</a>; Cockerton et al., <a href="#tpg270086-bib-0014" class="usa-link" aria-describedby="tpg270086-bib-0014">2023</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Koskela et al., <a href="#tpg270086-bib-0046" class="usa-link" aria-describedby="tpg270086-bib-0046">2016</a>; Lewers et al., <a href="#tpg270086-bib-0048" class="usa-link" aria-describedby="tpg270086-bib-0048">2019</a>; Perrotte et al., <a href="#tpg270086-bib-0056" class="usa-link" aria-describedby="tpg270086-bib-0056">2016</a>; Saiga et al., <a href="#tpg270086-bib-0063" class="usa-link" aria-describedby="tpg270086-bib-0063">2022</a>; Salinas et al., <a href="#tpg270086-bib-0064" class="usa-link" aria-describedby="tpg270086-bib-0064">2017</a>; Verma et al., <a href="#tpg270086-bib-0076" class="usa-link" aria-describedby="tpg270086-bib-0076">2017</a>), the genetic determinants of variation for runnering and clone yield within DN (<em>PF_</em>) and SD (<em>pfpf</em>) populations have not. Here, we show that the suppression of runnering is highly variable and quantitative among <em>PF_</em> plants under long days, that runnering is highly variable within populations of SD (<em>pfpf</em>) and DN (<em>PF_</em>) plants where the confounding effects of the <em>PF</em> locus are obviously absent, and that variation for runnering in SD and DN plants spans the entire range, from runnerless to prolific and unrestrained. The genetic determinants of that phenotypic variation are unknown.</p>
<p>We undertook the studies described here to develop a deeper understanding of the genetics of asexual reproduction and explore genomic prediction as a solution to the problem of accurately predicting runnering phenotypes in octoploid strawberry populations that are either fixed or segregating for <em>PF</em> alleles. The heritability of runnering and feasibility of applying genomic selection for runnering has not been previously assessed in strawberry. Our studies were motivated by the prospect of developing highly productive runnerless cultivars for seed‐propagated and near‐runnerless cultivars for clone‐propagated production systems that strike the perfect balance between sexual and asexual reproduction, within the physiological and developmental limits imposed by the biology of the underlying causal genes, including <em>PF</em> (Caruana et al., <a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Koskela et al., <a href="#tpg270086-bib-0046" class="usa-link" aria-describedby="tpg270086-bib-0046">2016</a>; Mouhu et al., <a href="#tpg270086-bib-0054" class="usa-link" aria-describedby="tpg270086-bib-0054">2013</a>; Sønsteby and Heide, <a href="#tpg270086-bib-0068" class="usa-link" aria-describedby="tpg270086-bib-0068">2006</a>).</p></section><section id="tpg270086-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270086-sec-0040"><h3 class="pmc_sec_title">2.1. Plant material</h3>
<p>The plant materials used in our studies included four seed‐propagated F<sub>2</sub> populations, nine seed‐propagated full‐sib (FS) families, and 932 clonally propagated individuals maintained in the UCD germplasm collection. The latter spanned the range of runnering phenotypes observed in strawberry and were selected to construct a strawberry diversity panel (SDP) for genome‐wide association study (GWAS) and genomic prediction study. The SDP included 15 <em>Fragaria chiloensis</em> ecotypes, 24 <em>F. virginiana</em> ecotypes, and 893 <em>Fragaria</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-25" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> individuals (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The <em>Fragaria</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> individuals included 71 UC and 46 non‐UC cultivars developed between 1775 and 2017 and 777 hybrid (F<sub>1</sub>) individuals developed between 1937 and 2021. The origin year, taxa, and pedigrees of these individuals are documented in Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>.</p>
<p>We developed nine FS families by crossing elite SD and DN parents with runner scores (RSs) ranging from 2.2 to 4.0 (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). The parents included three cultivars released by UCD in 2023 (UC Surfline, UC Monarch, and UC Keystone) and 11 additional elite F<sub>1</sub> hybrid individuals selected from FS families developed between 2017 and 2019.</p>
<section class="tw xbox font-sm" id="tpg270086-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Flowering habit (FH)<a href="#tpg270086-tbl1-note-0001" class="usa-link">
<sup>a</sup>
</a> and runner score (RS)<a href="#tpg270086-tbl1-note-0002" class="usa-link">
<sup>b</sup>
</a> phenotypes and A/G single nucleotide polymorphism (SNP) marker (AX‐184947290) genotypes<a href="#tpg270086-tbl1-note-0003" class="usa-link">
<sup>c</sup>
</a> of the parents of F<sub>1</sub> individuals and F<sub>2</sub> and full‐sib families.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th colspan="4" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Female parent</th>
<th colspan="4" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Male parent</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Source</th>
<th align="left" rowspan="1" colspan="1">Source ID</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-27_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><mi>n</mi></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">Parent ID</th>
<th align="left" rowspan="1" colspan="1">FH</th>
<th align="left" rowspan="1" colspan="1">SNP</th>
<th align="left" rowspan="1" colspan="1">RS</th>
<th align="left" rowspan="1" colspan="1">Parent ID</th>
<th align="left" rowspan="1" colspan="1">FH</th>
<th align="left" rowspan="1" colspan="1">SNP</th>
<th align="left" rowspan="1" colspan="1">RS</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">F<sub>1</sub>
</td>
<td align="left" rowspan="1" colspan="1">19A907P022</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">17C321P015</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">5.0</td>
<td align="left" rowspan="1" colspan="1">Royal Royce</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">2.8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">19A907P024</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">17C321P015</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">5.0</td>
<td align="left" rowspan="1" colspan="1">Royal Royce</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">2.8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">19A908P012</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">17C321P015</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">5.0</td>
<td align="left" rowspan="1" colspan="1">55C032P001</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.3</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">19A908P053</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">17C321P015</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">5.0</td>
<td align="left" rowspan="1" colspan="1">55C032P001</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.3</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">F<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">21S950</td>
<td align="left" rowspan="1" colspan="1">48</td>
<td align="left" rowspan="1" colspan="1">19A907P022</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AA</td>
<td align="left" rowspan="1" colspan="1">1.3</td>
<td align="left" rowspan="1" colspan="1">19A907P022</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AA</td>
<td align="left" rowspan="1" colspan="1">1.3</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">21S951</td>
<td align="left" rowspan="1" colspan="1">135</td>
<td align="left" rowspan="1" colspan="1">19A907P024</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.3</td>
<td align="left" rowspan="1" colspan="1">19A907P024</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.3</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">21S952</td>
<td align="left" rowspan="1" colspan="1">48</td>
<td align="left" rowspan="1" colspan="1">19A908P012</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">19A908P012</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">21S953</td>
<td align="left" rowspan="1" colspan="1">43</td>
<td align="left" rowspan="1" colspan="1">19A908P053</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">19A908P053</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">1.0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Full‐sib</td>
<td align="left" rowspan="1" colspan="1">22C315</td>
<td align="left" rowspan="1" colspan="1">46</td>
<td align="left" rowspan="1" colspan="1">UC Surfline</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">3.7</td>
<td align="left" rowspan="1" colspan="1">19B091P078</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C324</td>
<td align="left" rowspan="1" colspan="1">37</td>
<td align="left" rowspan="1" colspan="1">UC Monarch</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">3.8</td>
<td align="left" rowspan="1" colspan="1">19B091P078</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C339</td>
<td align="left" rowspan="1" colspan="1">44</td>
<td align="left" rowspan="1" colspan="1">17C618P658</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
<td align="left" rowspan="1" colspan="1">19B091P078</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C346</td>
<td align="left" rowspan="1" colspan="1">49</td>
<td align="left" rowspan="1" colspan="1">18C430P006</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">4.0</td>
<td align="left" rowspan="1" colspan="1">19B033P006</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">3.0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C409</td>
<td align="left" rowspan="1" colspan="1">49</td>
<td align="left" rowspan="1" colspan="1">UC Keystone</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
<td align="left" rowspan="1" colspan="1">19B033P006</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">3.0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C410</td>
<td align="left" rowspan="1" colspan="1">27</td>
<td align="left" rowspan="1" colspan="1">UC Keystone</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
<td align="left" rowspan="1" colspan="1">18B061P100</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">2.2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C426</td>
<td align="left" rowspan="1" colspan="1">48</td>
<td align="left" rowspan="1" colspan="1">17C121P097</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">2.4</td>
<td align="left" rowspan="1" colspan="1">19C174P009</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">2.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C485</td>
<td align="left" rowspan="1" colspan="1">46</td>
<td align="left" rowspan="1" colspan="1">19C155P035</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">3.6</td>
<td align="left" rowspan="1" colspan="1">19B091P078</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">22C498</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">18B061P005</td>
<td align="left" rowspan="1" colspan="1">DN</td>
<td align="left" rowspan="1" colspan="1">AG</td>
<td align="left" rowspan="1" colspan="1">2.5</td>
<td align="left" rowspan="1" colspan="1">17C693P610</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">GG</td>
<td align="left" rowspan="1" colspan="1">4.0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270086-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270086-tbl1-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">The flowering habits of the parents were classified as short‐day (SD) or day‐neutral (DN) from the absence or presence of flowering, respectively, under long days (13.5–14.5 h) in Winters, CA.</p>
</div>
<div class="fn" id="tpg270086-tbl1-note-0002">
<sup>
<sup>b</sup>
</sup><p class="display-inline">The runner scores of the parents were recorded using an ordinal scale, where 1 = <em>runnerless</em>, 2 = <em>weak runnering</em>, 3 = <em>intermediate runnering</em>, 4 = <em>strong runnering</em>, and 5 = <em>extreme runnering</em> (see Figure <a href="#tpg270086-fig-0003" class="usa-link">3A</a>).</p>
</div>
<div class="fn" id="tpg270086-tbl1-note-0003">
<sup>
<sup>c</sup>
</sup><p class="display-inline">AX‐184947290 is an A/G SNP on a 50,000 Axiom array (Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>). AX‐184947290 is associated with the <em>PF</em> locus and highly predictive of <em>PF</em> genotypes in short‐day and day‐neutral cultivars developed at University of California, Davis, between 1935 and 2012. The physical location of AX‐184947290 on chromosome 4B in the UCD Royal Royce genome is by 27,294,311 (<a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0</a>).</p>
</div>
</div></section><p>F<sub>2</sub> populations (21S950, 21S951, 21S952, and 21S953) were developed by self‐pollinating four runnerless to near‐runnerless F<sub>1</sub> individuals (19A907P022, 19A907P024, 19A908P012, and 19A908P053) developed from crosses between parents with contrasting runnering phenotypes (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). The crosses were 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001. We selected two runnerless F<sub>1</sub> individuals from each FS family: 19A907P022 and 19A907P024 from 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce and 19A908P012 and 19A908P053 from 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). The selected individuals were manually self‐pollinated over the winter of 2020–2021 to produce seeds of two 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce F<sub>2</sub> populations (21S950 and 21S951) and two 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> populations (21S952 and 21S953).</p>
<p>17C321P015 is an SD individual (<em>pfpf</em>) with an aggressive runnering phenotype (RS = 5.0) developed from a cross between Cabrillo and PI551727. PI551727 (CA 1234 Mendocino) is an SD <em>F. chiloensis</em> spp. <em>lucida</em> ecotype with an aggressive runnering phenotype (RS = 5.0) typical of ecotypes of this species. Cabrillo is a DN cultivar (<em>PFpf</em>) with an intermediate runnering phenotype (RS = 3.0) typical of modern DN cultivars. Royal Royce is a DN cultivar (<em>PFpf</em>) with slightly weaker runnering (RS = 2.8) than Cabrillo. 55C032P001 is a runnerless (RS = 1.3), DN individual developed by Royce S. Bringhurst in 1955 from a cross between the SD cultivar Shasta (<em>pfpf</em>; RS = 3.0) and Wasatch, an <em>F. virginiana</em> spp. <em>glauca</em> ecotype. Wasatch is the source of the dominant <em>PF</em> allele that Bringhurst and Voth (<a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>) introgressed into SD genetic backgrounds, starting with Shasta (Ahmadi et al. <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>).</p></section><section id="tpg270086-sec-0050"><h3 class="pmc_sec_title">2.2. Field experiments and phenotyping</h3>
<p>SDP individuals were grown and clonally propagated annually by rooting plantlets in a field nursery in Winters, CA, over several years (2017–2023, excluding 2018). Our study location was the same as that originally used by Ahmadi et al. (<a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>) to discover the <em>PF</em> locus. The annual propagation cycle started with planting two or three bare‐root plants/individual in the field in April and ended with the harvest of rooted plantlets (bare‐root plants) in December or January. The row spacing was 1.0 m with 0.3 m between plots. Runners were trimmed, if necessary, to prevent cross‐contamination between plots. The bare‐root plants were dug from the field in December or January of each year, cleaned, trimmed, and stored at 4°for 6–10 weeks before being replanted to repeat the annual propagation cycle.</p>
<p>Over the course of nursery propagation cycle each year, SDP individuals were phenotyped for photoperiod‐dependent flowering habit and scored for runnering. The flowering habits of individuals were classified as SD or DN by the absence or presence of flowering, respectively, under long daylengths (13.5–14.5 h) in late spring and early fall. SDP individuals were phenotyped for runnering in August of each year using ordinal RSs on a 1–5 scale, where 1 = <em>runnerless</em>, 2 = <em>weak runnering</em>, 3 = <em>intermediate runnering</em>, 4 = <em>strong runnering</em>, and 5 = <em>extreme runnering</em>. The RSs compiled in Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a> are estimated marginal means (EMMs) calculated from phenotypes observed over multiple years in Winters, CA. The number of observations per individual ranged from 1 to 6 (the harmonic mean number of observations per individual was 1.92).</p>
<p>We grew approximately 100 seed‐propagated individuals from the 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 FS families in a Winters, CA, field experiment in 2018–2019. Seedlings of those individuals were grown in a shade house over the summer of 2018, transplanted to the field in September 2018, and phenotyped for runnering in June 2019. Two runnerless individuals were identified and selected within each FS family. Those F<sub>1</sub> individuals were transplanted from the field to a greenhouse in July 2019 and self‐pollinated over the winter of 2019–2020 to create four F<sub>2</sub> populations (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
<p>F<sub>2</sub> progeny (274 individuals from four families) were phenotyped for runner count in a Davis, CA, field experiment in 2021–2022 (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). F<sub>2</sub> seeds were germinated in June 2021; seedlings were transplanted to the field in September 2021; and plants were phenotyped by counting primary, secondary, and tertiary runners on March 26, April 9 and 22, May 6 and 20, and June 3 and 16, 2022. The photoperiods on those dates were 12.3, 13.0, 13.3, 14.1, 14.3, 14.4, and 14.5 h, respectively. The photoperiod on the summer solstice (June 21, 2022) was 14.5 h.</p>
<p>FS progeny (372 individuals from nine families) were phenotyped for RS and count in a Davis, CA, field experiment in 2023‐2024. FS seeds were germinated June 2023, seedlings were transplanted to the field in September 2023, and plants were phenotyped by counting inflorescence number and primary runner number on May 1, June 1, and July 1, 2024. The photoperiods on those dates were 13.5, 14.4, and 14.5 h, respectively. FS progeny were scored for runnering as described for the SDP population on July 1, 2024.</p></section><section id="tpg270086-sec-0060"><h3 class="pmc_sec_title">2.3. Genotyping</h3>
<p>Single nucleotide polymorphism (SNP) genotypes for the SDP population were obtained from previous studies (Feldmann, Pincot, Seymour, et al., <a href="#tpg270086-bib-0026" class="usa-link" aria-describedby="tpg270086-bib-0026">2024</a>; Hardigan et al., <a href="#tpg270086-bib-0036" class="usa-link" aria-describedby="tpg270086-bib-0036">2021</a>; D. D. Pincot et al., <a href="#tpg270086-bib-0057" class="usa-link" aria-describedby="tpg270086-bib-0057">2022</a>). DNA samples of SDP individuals were previously genotyped with either 50,000 or 850,000 Axiom SNP arrays designed by Hardigan et al. (<a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>) and developed in collaboration with the ThermoFisher Affymetrix Expert Design Program (<a href="https://www.thermofisher.com/order/catalog/product/551041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.thermofisher.com/order/catalog/product/551041</a>). Freshly emerged leaves were harvested from F<sub>2</sub> and FS individuals, lyophilized, and powdered for DNA isolation using the E‐Z 96 Plant DNA Kit (Omega Bio‐Tek) as described by Knapp et al. (<a href="#tpg270086-bib-0045" class="usa-link" aria-describedby="tpg270086-bib-0045">2024</a>). DNA samples of these individuals were genotyped with the 50,000 Axiom SNP array Hardigan et al. (<a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>).</p>
<p>The probe DNA sequences for SNPs on the 50,000 and 850,000 Axiom arrays used in our studies were physically anchored to the haplotype‐phased UCD Royal Royce genome in silico (FaRR1; <a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0</a>) and unphased ‘Camarosa’ genome in silico (FaCA1; Edger et al. (<a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>); <a href="https://phytozome-next.jgi.doe.gov/info/Fxananassa_v1_0_a1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/Fxananassa_v1_0_a1</a>). The physical positions of SNPs in the UCD Royal Royce genome are reported throughout this paper. DNA sequences for the SNP probes and physical coordinates are compiled in Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S2</a> for both genomes using the chromosome nomenclature described by Hardigan et al. (<a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>). That nomenclature is cross‐referenced to other linkage group and chromosome nomenclatures in Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S3</a>. Homoeologous chromosomes in the UCD Royal Royce genome were numbered 1–7 and oriented according to the rules used for <em>Fragaria vesca</em>, the A‐genome ancestor (Edger et al., <a href="#tpg270086-bib-0020" class="usa-link" aria-describedby="tpg270086-bib-0020">2018</a>; Shulaev et al., <a href="#tpg270086-bib-0067" class="usa-link" aria-describedby="tpg270086-bib-0067">2011</a>). As previously described (Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>), that was feasible because of synteny among homoeologous A, B, C, and D genome chromosomes. DNA samples that passed quality and quantity control standards were genotyped on a GeneTitan HT Microarray System by ThermoFisher Scientific. SNP genotypes were called using the Affymetrix Axiom Analysis Suite (v1.1.1.66, Affymetrix).</p>
<p>SNP genotypes were curated and filtered for GWAS and other analyses by retaining those with call rates <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><mo>≥</mo></mrow></math></span> 89% and in silico assigned physical positions corroborated by de novo genetic mapping (the latter are identified in Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S2</a>). The genomic relationship matrix was estimated among SDP, F<sub>2</sub>, and FS individuals using the <em>rrBLUP::A.mat()</em> function in the R package “rrBLUP” with a minor allele frequency cutoff of 0.05 (min.MAF = 0.05), a maximum missing data cutoff of 0.8 (max.missing = 0.8), and imputation of missing data (return.imputed = TRUE) (D. D. A. Pincot et al., <a href="#tpg270086-bib-0058" class="usa-link" aria-describedby="tpg270086-bib-0058">2020</a>). After data processing, 41,794 SNPs were retained for analyses of F<sub>2</sub> and FS populations, whereas 42,903 SNPs were retained for analyses of the SDP population. We genotyped 194 SDP individuals with the 850,000 Axiom SNP array (Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>). After data processing, 799,895 were retained for analyses of those individuals, whereas 625,039 were retained for analyses of the subset of 66 UC individuals (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S2</a>).</p>
<p>Genetic relationships among SDP, F<sub>2</sub>, and FS individuals were visualized by constructing a neighbor‐joining tree using the <em>nj()</em> function in the “ape” R package (Paradis et al., <a href="#tpg270086-bib-0055" class="usa-link" aria-describedby="tpg270086-bib-0055">2004</a>) and drawing the tree using the R package “ggtree” (Yu et al., <a href="#tpg270086-bib-0081" class="usa-link" aria-describedby="tpg270086-bib-0081">2017</a>). The Euclidean distance matrix for this analyses was estimated from 49,483 array‐genotyped SNPs using the <em>dist()</em> function in the R “stats” package (R Core Team, <a href="#tpg270086-bib-0061" class="usa-link" aria-describedby="tpg270086-bib-0061">2013</a>).</p></section><section id="tpg270086-sec-0070"><h3 class="pmc_sec_title">2.4. Statistical analyses</h3>
<p>The RSs observed among SDP individuals were analyzed using the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><mrow><mi>l</mi><mi>m</mi><mi>e</mi><mi>r</mi><mo stretchy="false">(</mo><mo stretchy="false">)</mo></mrow></mrow></math></span> linear mixed model (LMM) function in the R package “lme4” (Bates et al. <a href="#tpg270086-bib-0002" class="usa-link" aria-describedby="tpg270086-bib-0002">2015</a>; <a href="https://cran.r-project.org/web/packages/lme4/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/web/packages/lme4/index.html</a>), The LMM for that analysis was as follows:
</p>
<table class="disp-formula p" id="tpg270086-disp-0001"><tr>
<td class="formula"><math id="jats-math-38" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>Y</mi><mi>i</mi></msub><mi>j</mi><mo linebreak="goodbreak">+</mo><msub><mrow><mi>G</mi><mi>Y</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> is the RS for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><mi>i</mi></mrow></math></span>th individual in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-41" display="inline"><mrow><mi>j</mi></mrow></math></span>th year, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><msub><mi>G</mi><mi>i</mi></msub></mrow></math></span> is the fixed effect of the <em>i</em>th individual, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><msub><mi>Y</mi><mi>j</mi></msub></mrow></math></span> is the random effect of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mi>j</mi></mrow></math></span>th year, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-45" display="inline"><mrow><mrow><mi>G</mi><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></span> is the random effect of the interaction between individual and year. A single RS was recorded each year, the composition of the germplasm collection changed year to year, and the number of years per individuals ranged from one to six. The harmonic mean number of replications per individual was 1.92. EMMs for RS were estimated using the R package “emmeans” (Lenth, <a href="#tpg270086-bib-0047" class="usa-link" aria-describedby="tpg270086-bib-0047">2019</a>).</p>
<p>The individual, year, and individual <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-46" display="inline"><mrow><mo>×</mo></mrow></math></span> year variance components and broad‐sense heritability on a clone‐mean basis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-47" display="inline"><mrow><mrow><msup><mi>H</mi><mn>2</mn></msup><mo>=</mo><msub><mi>V</mi><mi>G</mi></msub><mrow><mo stretchy="false">(</mo></mrow><msub><mi>V</mi><mi>G</mi></msub><mo>+</mo><msub><mi>V</mi><mrow><mi>G</mi><mi>Y</mi></mrow></msub><mo>/</mo><mi>r</mi></mrow></mrow></math></span>) were estimated for RS using restricted maximum likelihood (REML) in the R package “lme4” (Bates et al. <a href="#tpg270086-bib-0002" class="usa-link" aria-describedby="tpg270086-bib-0002">2015</a>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-48" display="inline"><mrow><msub><mi>V</mi><mi>G</mi></msub></mrow></math></span> is the REML estimate of the individual variance component, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-49" display="inline"><mrow><msub><mi>V</mi><mrow><mi>G</mi><mi>Y</mi></mrow></msub></mrow></math></span> is the REML estimate of the individual <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-50" display="inline"><mrow><mo>×</mo></mrow></math></span> year variance component, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-51" display="inline"><mrow><mrow><mi>h</mi><mo>=</mo><mn>1.92</mn></mrow></mrow></math></span> is the harmonic mean of the number of observations per individual, calculated using <em>pysch::harmonic.mean()</em> (Revelle, <a href="#tpg270086-bib-0062" class="usa-link" aria-describedby="tpg270086-bib-0062">2021</a>).</p>
<p>Genomic‐estimated narrow‐sense heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-52" display="inline"><mrow><msup><mi>h</mi><mn>2</mn></msup></mrow></math></span>) was estimated for the SDP, F<sub>2</sub>, and FS populations using the <em>rrBLUP:: mixed.solve()</em> function, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-53" display="inline"><mrow><mrow><msup><mi>h</mi><mn>2</mn></msup><mo>=</mo><msub><mi>V</mi><mi>A</mi></msub><mrow><mo>/</mo><mo stretchy="false">(</mo></mrow><msub><mi>V</mi><mi>A</mi></msub><mo>+</mo><msub><mi>V</mi><mrow><mi>G</mi><mi>Y</mi><mo stretchy="false">)</mo></mrow></msub><mo>/</mo><mi>r</mi></mrow></mrow></math></span>) for the SDP population, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-54" display="inline"><mrow><mrow><msup><mi>h</mi><mn>2</mn></msup><mo>=</mo><msub><mi>V</mi><mi>A</mi></msub><mrow><mo>/</mo><mo stretchy="false">(</mo></mrow><msub><mi>V</mi><mi>P</mi></msub></mrow></mrow></math></span> for the F<sub>2</sub> and FS populations, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-55" display="inline"><mrow><msub><mi>V</mi><mi>A</mi></msub></mrow></math></span> is the genomic‐estimated additive genetic variance, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-56" display="inline"><mrow><msub><mi>V</mi><mi>P</mi></msub></mrow></math></span> is the phenotypic variance on an individual‐plant basis.</p>
<p>Several statistics were estimated for the <em>PF</em>‐associated SNP AX‐184947290 using the LMMs described above. The additive and dominance effects of AX‐184947290 were estimated by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-57" display="inline"><mrow><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub><mo>−</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><mn>2</mn></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-58" display="inline"><mrow><mrow><mover accent="true"><mi>d</mi><mo>^</mo></mover><mo>=</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AG</mtext></msub><mo>−</mo><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub><mo>+</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><mn>2</mn></mrow></mrow></math></span>, respectively, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-59" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-60" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AG</mtext></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-61" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub></mrow></math></span> are the respective phenotypic means for AA, AG, and GG SNP genotype. The degree of dominance of AX‐184947290 was estimated by |<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-62" display="inline"><mrow><mrow><mover accent="true"><mi>d</mi><mo>^</mo></mover><mo>/</mo><mover accent="true"><mi>a</mi><mo>^</mo></mover></mrow></mrow></math></span>| (Falconer, <a href="#tpg270086-bib-0023" class="usa-link" aria-describedby="tpg270086-bib-0023">1996</a>; Walsh, <a href="#tpg270086-bib-0078" class="usa-link" aria-describedby="tpg270086-bib-0078">2001</a>). The percentage of the genetic variance explained by AX‐184947290 was estimated by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-63" display="inline"><mrow><mrow><mtext>GVE</mtext><mo>=</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>M</mi><mn>2</mn></msubsup><mo>/</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>A</mi><mn>2</mn></msubsup></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-64" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>M</mi><mn>2</mn></msubsup></mrow></math></span> is the bias‐corrected average semivariance REML estimate of the variance associated with AX‐184947290 (Feldmann et al., <a href="#tpg270086-bib-0024" class="usa-link" aria-describedby="tpg270086-bib-0024">2021</a>, <a href="#tpg270086-bib-0025" class="usa-link" aria-describedby="tpg270086-bib-0025">2022</a>). The percentage of the phenotypic variance explained by AX‐184947290 was estimated by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-65" display="inline"><mrow><mrow><mtext>PVE</mtext><mo>=</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>M</mi><mn>2</mn></msubsup><mo>/</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>P</mi><mn>2</mn></msubsup></mrow></mrow></math></span>.</p></section><section id="tpg270086-sec-0080"><h3 class="pmc_sec_title">2.5. Genome‐wide association study</h3>
<p>GWAS analyses were applied to the SDP population for flowering habit and RS, F<sub>2</sub> population for runner count, and FS population for RS, runner count, and inflorescence count using GEMMA v0.98.1 (Zhou &amp; Stephens, <a href="#tpg270086-bib-0082" class="usa-link" aria-describedby="tpg270086-bib-0082">2012</a>). The RS EMMs and binary flowering habit classification (SD and DN) were used as input for GWAS of the SDP population. The runner count, inflorescence count, and RS observations for individual time points were used as input for GWAS of F<sub>2</sub> and FS populations. The GEMMA ‘lmm2’ function and kinship (K) matrix were used to correct for the population structure (Zhou &amp; Stephens, <a href="#tpg270086-bib-0082" class="usa-link" aria-describedby="tpg270086-bib-0082">2012</a>). The <em>p</em>‐values were adjusted for a false discovery rate (FDR) of 0.05 and reported as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-66" display="inline"><mrow><mrow><mo>−</mo><msub><mi>log</mi><mn>10</mn></msub><mrow><mo stretchy="false">(</mo><mtext>FDR p-value</mtext><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> score. Three lead <em>PF</em>‐associated SNPs (AX‐184947290, AX‐184219235, and AX‐184208850) were used as covariates in GEMMA to search for associations between genetic variants and phenotypes. We tested AX‐184947290 and AX‐184219235 individually; AX‐184947290 and AX‐184219235 in combination; and AX‐184947290, AX‐184219235, and AX‐184208850 in combination. Lastly, we used the <em>GWAS()</em> function of the “rrBLUP” R package for GWAS of SDP individuals genotyped with the 850,000 SNP array using flowering behavior coded as 0 (SD) or 1 (DN). The kinship matrix was estimated using the <em>rrBLUP::A.mat()</em> function and incorporated in the “rrBLUP” analysis to control for population structure.</p></section><section id="tpg270086-sec-0090"><h3 class="pmc_sec_title">2.6. Haplotype analyses</h3>
<p>The linkage phases of 50,000 array‐genotyped SNPs were unknown among individuals in the SDP and F<sub>2</sub> populations. They were inferred in the latter by genetic mapping and by comparing haplotypes inferred among SDP individuals. We used Beagle 5.4v (Browning et al., <a href="#tpg270086-bib-0009" class="usa-link" aria-describedby="tpg270086-bib-0009">2021</a>) with default parameters to infer the haplotypes of 51 SNPs that were associated with the <em>PF</em> locus and had minor allele frequencies (MAFs) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-67" display="inline"><mrow><mo>≥</mo></mrow></math></span> 0.3 in the SDP population. GWAS‐estimated SNP effects with FDR‐corrected <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-68" display="inline"><mrow><mrow><mo>−</mo><msub><mi>log</mi><mn>10</mn></msub><mrow><mo stretchy="false">(</mo><mtext>p-value</mtext><mo stretchy="false">)</mo></mrow><mo>&gt;</mo><mn>2</mn></mrow></mrow></math></span> for flowering habit or RS were selected for phasing. This filtering process yielded 51 SNPs spanning bp 26,529,027 to 29,635,684 on chromosome 4B. The squared correlation coefficient linkage disequilibrium (LD) statistic (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-69" display="inline"><mrow><msup><mi>R</mi><mn>2</mn></msup></mrow></math></span>) was estimated among the 51 SNPs using the R package “snpStats,” version 1.58.0 (<a href="https://bioconductor.org/packages/snpStats" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioconductor.org/packages/snpStats</a>).</p></section><section id="tpg270086-sec-0100"><h3 class="pmc_sec_title">2.7. Genomic prediction analyses</h3>
<p>Genomic relationship matrices (G) and genomic‐estimated breeding values (GEBVs) were estimated by genomic best linear unbiased prediction (G‐BLUP) for RS in the SDP population, runner count in the F<sub>2</sub> population, and runner count and inflorescence count in the FS population using the <em>mixed.solve()</em> function in the R package rrBLUP (Endelman, <a href="#tpg270086-bib-0021" class="usa-link" aria-describedby="tpg270086-bib-0021">2011</a>; Endelman &amp; Jannink, <a href="#tpg270086-bib-0022" class="usa-link" aria-describedby="tpg270086-bib-0022">2012</a>). GEBVs were estimated with and without incorporating the AX‐184947290 SNP as a fixed effect in the G‐BLUP analysis. Genomic predictive ability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-70" display="inline"><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub></mrow></math></span>) and prediction accuracy (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-71" display="inline"><mrow><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub><mo>/</mo><mi>h</mi></mrow></mrow></math></span>) were count within and between strawberry populations by cross‐validation, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-72" display="inline"><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover></mrow></math></span> is the GEBV and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-73" display="inline"><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></math></span> is the EMM. We used the SDP population (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-74" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>932</mn></mrow></mrow></math></span>) and subset of UC individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-75" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>672</mn></mrow></mrow></math></span>) in the SDP population (SDP‐UC) for cross‐validation. Cross‐validation was performed with 1000 Monte Carlo iterations within the SDP, SDP‐UC, and FS populations; between the SDP and FS; and SDP‐UC and FC populations by randomly splitting individuals into training (80%) and validation (20%) subsets (Burman, <a href="#tpg270086-bib-0010" class="usa-link" aria-describedby="tpg270086-bib-0010">1989</a>; Molinaro et al., <a href="#tpg270086-bib-0053" class="usa-link" aria-describedby="tpg270086-bib-0053">2005</a>; Poland &amp; Rutkoski, <a href="#tpg270086-bib-0060" class="usa-link" aria-describedby="tpg270086-bib-0060">2016</a>).</p></section></section><section id="tpg270086-sec-0110"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270086-sec-0120"><h3 class="pmc_sec_title">3.1. Variation for runnering encompasses the entire phenotypic range, from runnerless to prolific</h3>
<p>Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a>, <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a> depict the range of stolon (runner) and plantlet (daughter plant) growth phenotypes observed among several thousand seed‐propagated FS progeny in early‐stage selection nurseries and 3000 clonally propagated SD and DN individuals observed in the UC strawberry germplasm collection over a period of 9 years (2015–2023). That variation was captured by the sample of 932 clonally propagated SDP individuals phenotyped for photoperiod‐dependent flowering habit and runnering in a low‐elevation nursery production environment (Winters, CA; 38.52°N, 121.97°W; 41 m) where photoperiods ranged from 9.5 h to 14.9 h over the March to December growing season (Figure <a href="#tpg270086-fig-0003" class="usa-link">3</a>; Supporting Information<a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The term “runnering” applied here and throughout our paper, though informal, is commonly used in research and commerce to describe stolon (runner) and plantlet (daughter plant) growth phenotypes.</p>
<figure class="fig xbox font-sm" id="tpg270086-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/fb9296846a46/TPG2-18-e70086-g006.jpg" loading="lazy" id="jats-graphic-1" height="747" width="600" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variation for runnering in octoploid strawberry. Unless otherwise noted, the phenotypes shown in these photographs (shot to scale by Fred Greaves for University of California, Davis) were observed on November 11, 2024, Winters, CA. Guadalupe‐Nipomo, an <em>F. chiloensis</em> spp. <em>lucida</em> ecotype carpeting a sand dune near Oso Flaco Beach, CA (August 1963). Frederik 9 and Hinesburg are <em>F. virginiana</em> spp. <em>virginiana</em> ecotypes. Harris Spring and Scott's Creek are <em>F. virginiana</em> spp. <em>platypetala</em> ecotypes. Yaquina A is an <em>F. chiloensis</em> spp. <em>pacifica</em> ecotype. 19A908P012 and 19A908P053 are runnerless 17C321P105 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-76" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 hybrids (July 2020, Winters, CA). 17C321P015 is a hybrid between Cabrillo and PI551727, an <em>F. chiloensis</em> spp. <em>lucida</em> ecotype (December 2017, Winters, CA). 55C032P001 is a runnerless day‐neutral hybrid. Senga Sengana, Elsanta, Tahoe, UC Surfline, and UC Monarch are short‐day cultivars. <em>PF</em> genotypes were predicted from the segregation of <em>PF</em> in full‐sib families and by using a single nucleotide polymorphism (SNP) marker (AX‐184947290) associated with the <em>PF</em> locus. RS is the estimated marginal mean for RS over years. Arrows identify the labeled individuals where unrelated individuals appear in growth boundaries.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270086-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/4d480cf99fc6/TPG2-18-e70086-g008.jpg" loading="lazy" id="jats-graphic-3" height="598" width="600" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variation for runnering among day‐neutral (<em>PFPF</em> and <em>PFpf</em>) individuals known to be descendants of Wasatch, the <em>F. virginiana</em> spp. <em>glauca</em> donor of the <em>PF</em> allele. The Wasatch photograph shows the original <em>PF</em> donor plant collected in July, 1953, from the Wasatch Mountains of Utah (reproduced from an original photograph in the Royce S. Bringhurst Special Collection [UUS COLL MSS 515], Merrill‐Cazier Library, Utah State University; <a href="https://library.usu.edu/archives/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://library.usu.edu/archives/</a>). The phenotypes shown in the other photographs (shot to scale by Fred Greaves for University of California (UC), Davis) were observed on November 11, 2024 in Winters, CA. <em>PF</em> genotypes were predicted using a single nucleotide polymorphism (SNP) marker (AX‐184947290) associated with the <em>PF</em> locus. RS is the estimated marginal mean for runner score over years. Arrows identify the labeled individuals where unrelated individuals appear in growth boundaries. The aerial photograph (upper left corner) displays a subset of the strawberry diversity panel (SDP) and other individuals preserved in the clonal germplasm collection at UC Davis.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270086-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/63ef831cd721/TPG2-18-e70086-g009.jpg" loading="lazy" id="jats-graphic-5" height="690" width="682" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Runner score distributions of strawberry diversity panel individuals classified as short‐day (SD = <em>pfpf</em>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-77" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>479</mn></mrow></mrow></math></span>) or day‐neutral (DN = <em>PF</em>_; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-78" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>441</mn></mrow></mrow></math></span>) over six growing seasons in Winters, CA. (A) Archetypal phenotypes of individuals scored for runnering on an ordinal scale, where 1 = <em>runnerless</em>, 2 = <em>weak runnering</em>, 3 = <em>intermediate runnering</em>, 4 = <em>strong runnering</em>, and 5 = <em>extreme runnering</em> (shot to scale by Fred Greaves for University of California, Davis, July 2024, Winters, CA). (B) Histograms were plotted using the phenotypic means (estimated marginal means (EMMs) = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-79" display="inline"><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></math></span>) for runner scores over years (the harmonic mean number of observations per individual was 1.92). The <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-80" display="inline"><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></math></span> distributions of strawberry diversity panel (SDP) individuals classified as SD or DN are shown in the left column with their group means (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-81" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>SD</mtext></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-82" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>DN</mtext></msub></mrow></math></span>). The <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-83" display="inline"><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></math></span> distributions of SDP individuals with GG, AG, and AA AX‐184947290 single nucleotide polymorphism (SNP) genotypes are shown in the right column of histograms with their genotypic means (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-84" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-85" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AG</mtext></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-86" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub></mrow></math></span>).</p></figcaption></figure><p>The SDP included 15 <em>F. chiloensis</em> ecotypes, 24 <em>F. virginiana</em> ecotypes, and 893 <em>Fragaria</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-87" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> individuals with runner and daughter plant growth (runnering) phenotypes spanning the range found in the octoploid, from runnerless to prolific and unrestrained (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a>, <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). Genetic relationships among these individuals are depicted in Figure <a href="#tpg270086-fig-0004" class="usa-link">4</a>, along with individuals from two exotic <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-88" display="inline"><mrow><mo>×</mo></mrow></math></span> exotic (17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-89" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001) F<sub>2</sub> families, two elite <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-90" display="inline"><mrow><mo>×</mo></mrow></math></span> exotic (17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-91" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce) F<sub>2</sub> families, and nine elite <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-92" display="inline"><mrow><mo>×</mo></mrow></math></span> elite FS families that were phenotyped for runnering in our study (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S4</a> and <a href="#tpg270086-supinfo-0001" class="usa-link">S5</a>). The nine FS families were developed from crosses among 13 elite UC parents (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270086-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/902b6fa69bd5/TPG2-18-e70086-g003.jpg" loading="lazy" id="jats-graphic-7" height="517" width="700" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic relationships among 932 strawberry diversity panel, 270 F<sub>2</sub>, and 372 full‐sib individuals genotyped with an Axiom 50,000 single nucleotide polymorphism (SNP) array. Genetic relationships were estimated from 49,483 SNPs. Clades identified by 21C950 and 21C951 are 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-93" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce F<sub>2</sub> families. Clades identified by 21C952 and 21C953 are 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-94" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> families. Clades identified by 22C numbers are full‐sib families developed from crosses between elite short‐day and day‐neutral individuals between 2017 and 2019 at University of California, Davis. The other branches and nodes identify strawberry diversity panel individuals, including <em>F. chiloensis</em> and <em>F. virginiana</em> ecotypes.</p></figcaption></figure><p>We knew from previous studies that we needed to correct for the photoperiod‐dependent pleiotropic effects of the dominant Wasatch <em>PF</em> allele to identify other loci affecting runnering phenotypes (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>). SDP individuals were classified as SD or DN by the absence or presence of flowering under long‐days (14.0–14.9 h), respectively (Figure <a href="#tpg270086-fig-0003" class="usa-link">3B</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The repeatability of our flowering habit classifications was 99% over six growing seasons (March–December) in Winters, CA (38.52°N). SD individuals were inferred to be homozygous recessive (<em>pfpf</em>), whereas DN individuals were inferred to be heterozygous or homozygous dominant (<em>PF</em>_) for <em>PF</em> alleles (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst &amp; Voth, <a href="#tpg270086-bib-0006" class="usa-link" aria-describedby="tpg270086-bib-0006">1976</a>; Bringhurst et al., <a href="#tpg270086-bib-0004" class="usa-link" aria-describedby="tpg270086-bib-0004">1989</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Perrotte et al., <a href="#tpg270086-bib-0056" class="usa-link" aria-describedby="tpg270086-bib-0056">2016</a>).</p>
<p>SDP individuals were phenotyped for runnering in our clonal propagation nursery in Winters, CA, over six growing seasons (March–December) using an ordinal RS scale, where 1 = <em>runnerless</em>, 2 = <em>weak runnering</em>, 3 = <em>intermediate runnering</em>, 4 = <em>strong runnering</em>, and 5 = <em>extreme runnering</em> (Figure <a href="#tpg270086-fig-0003" class="usa-link">3</a>). The number of years of observation ranged from one to six per individual (the harmonic mean number of years of observation was 1.92). To complement our analyses of full‐season phenotypes of clonally propagated SDP individuals, seed‐propagated exotic <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-95" display="inline"><mrow><mo>×</mo></mrow></math></span> exotic (17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-96" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001) F<sub>2</sub>, elite <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-97" display="inline"><mrow><mo>×</mo></mrow></math></span> exotic (17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-98" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce) F<sub>2</sub>, and elite <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-99" display="inline"><mrow><mo>×</mo></mrow></math></span> elite FS progeny were phenotyped by counting runners under long days (13.0–14.5 h photoperiods in Davis and Winters, CA). The runner count phenotypes of F<sub>2</sub> individuals were recorded on seven dates with photoperiods ranging from 13.0 to 14.5 h (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S4</a>). Our experience in the F<sub>2</sub> study guided the phenotyping schedule applied in the FS study. FS individuals were phenotyped on three dates (May 1, June 1, and July 1) with photoperiods ranging from 13.5 to 14.5 h (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S5</a>). Unless otherwise noted, we are reporting statistics estimated from phenotypes observed on the longest days when phenotypic differences and the temporal effect of the <em>PF</em> locus were the greatest.</p>
<p>The individuals in our study populations sampled genetic variation across the entire diversity range, from exotic (ecotypes of the wild relatives) to elite (modern UC cultivars). Among SDP individuals, 478 were classified as SD and 441 were classified as DN (Figure <a href="#tpg270086-fig-0003" class="usa-link">3B</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The <em>Fragaria</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-100" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> individuals included 80 SD cultivars, 39 DN cultivars, 776 UC cultivars, and other hybrids developed between 1969 and 2021 (the SDP‐UC training population), and 691 UC cultivars and other hybrids developed between 2016 and 2021. The 893 <em>Fragaria</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-101" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> individuals selected for the highly admixed SDP population originated from 554 unique pedigrees (FS families) among 489 parents (Figure <a href="#tpg270086-fig-0004" class="usa-link">4</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The runnering phenotypes shown in Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a> and <a href="#tpg270086-fig-0002" class="usa-link">2</a>, with five exceptions (Guadalupe‐Nipomo, 17C321P015, 19A908P012, 19A908P053, and Wasatch), were observed on November 11, 2024 after a full‐season of runner growth and clonal propagation in Winters, CA, and were shot to scale to ensure accurate visual comparisons (the mother plants were transplanted March 2024). The runners of those individuals were periodically trimmed to prevent cross‐contamination between individuals; hence, the phenotypes shown for wild relatives and other individuals with prolific and unrestrained runnering are less extreme than would have been observed without runner trimming.</p>
<p>Untrimmed phenotypes are shown in Figure <a href="#tpg270086-fig-0001" class="usa-link">1</a> for an <em>F. chiloensis</em> spp. <em>lucida</em> ecotype (Guadalupe‐Nipomo) in situ and 17C321P015 ex situ, a hybrid between the UC DN cultivar Cabrillo and an <em>F. chiloensis</em> spp. <em>lucida</em> ecotype (PI551727) native to Cape Mendocino, CA. The iconic photograph of the Guadalupe‐Nipomo ecotype, captured by Royce S. Bringhurst near Oso Flaco Beach, CA, illustrates the coastal adaptation and prolific and unrestrained runner growth of this wild relative in a sand dune habitat (Bringhurst et al., <a href="#tpg270086-bib-0005" class="usa-link" aria-describedby="tpg270086-bib-0005">1977</a>). The phenotype shown for 17C321P015 depicts a single season of nursery growth from a single mother plant (identified by the arrow) without runner trimming (Figure <a href="#tpg270086-fig-0001" class="usa-link">1</a>). The diameter of the labyrinth of runners and daughter plants emanating from the 17C321P015 mother plant was approximately 3.7 m at the end of the clonal propagation season (December 2017).</p>
<p>At the opposite extreme are the runnerless to near‐runnerless phenotypes of several DN Wasatch descendants, for example, 55C032P001 (55.32‐1), 20C420P006, and UCD Moxie (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a> and <a href="#tpg270086-fig-0002" class="usa-link">2</a>). 55C032P001 is a first‐generation, DN descendant of Shasta <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-102" display="inline"><mrow><mo>×</mo></mrow></math></span> Wasatch and donor of the <em>PF</em> allele found in 65C065P601, Aptos, Brighton, and Hecker, the earliest UC DN cultivars (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst and Voth, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>; D. D. A. Pincot et al., <a href="#tpg270086-bib-0059" class="usa-link" aria-describedby="tpg270086-bib-0059">2021</a>). 65C065P601 (65.65‐601) is a third‐generation DN descendant of Shasta <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-103" display="inline"><mrow><mo>×</mo></mrow></math></span> Wasatch, parent of the DN cultivar Brighton, and grandparent of the DN cultivar Selva (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Bringhurst and Voth, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>). High‐density SNP haplotypes for those individuals facilitated analyses to pinpoint the physical location of the <em>PF</em> locus.</p></section><section id="tpg270086-sec-0130"><h3 class="pmc_sec_title">3.2. The runnering phenotypes of strawberry are highly heritable and stable</h3>
<p>We found runnering phenotypes to be highly heritable, particularly among clonally propagated plants (Table <a href="#tpg270086-tbl-0002" class="usa-link">2</a>). The broad‐sense heritability on a clone‐mean basis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-104" display="inline"><mrow><msup><mi>H</mi><mn>2</mn></msup></mrow></math></span>) for RS was 0.86 in the SDP population (Table <a href="#tpg270086-tbl-0002" class="usa-link">2</a>). Our estimates of narrow‐sense heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-105" display="inline"><mrow><msup><mi>h</mi><mn>2</mn></msup></mrow></math></span>) for RS or count in the SDP, F<sub>2</sub>, and FS populations were more moderate, ranging from 0.37 to 0.60 (Table <a href="#tpg270086-tbl-0002" class="usa-link">2</a>). Several transgressive phenotypes were recovered in our breeding and genetic studies, for example, the runnerless phenotypes of the F<sub>1</sub> parents of our 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-106" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce F<sub>2</sub> populations (19A907P022 and 19A907P024) were transgressive (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>; Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a>, <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a>). The runner scores for both were 1.3, whereas the RSs of the parents were 2.8 and 5.0. Conversely, the runnerless to near‐runnerless phenotypes recovered among 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-107" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 progeny (FS) were anticipated because the 55C032P001 parent was runnerless (RS = 1.3). The runnerless phenotype of 55C032P001 was unaffected by temperature and photoperiod variation over nine field growing seasons (March–December) in Winters, CA (Figure <a href="#tpg270086-fig-0001" class="usa-link">1</a>). Similarly, the phenotypes of 20C420P005 and the other runnerless to near‐runnerless individuals identified in our studies have been stable across years and nursery and fruit production environments (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a> and <a href="#tpg270086-fig-0002" class="usa-link">2</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>).</p>
<section class="tw xbox font-sm" id="tpg270086-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Additive genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-108" display="inline"><mrow><msub><mi>V</mi><mi mathvariant="normal">A</mi></msub></mrow></math></span>), coefficient of additive genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-109" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span>), and heritability of runnering phenotypes in strawberry.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Population<a href="#tpg270086-tbl2-note-0001" class="usa-link"><sup>a</sup></a>
</th>
<th align="left" rowspan="1" colspan="1">Trait<a href="#tpg270086-tbl2-note-0002" class="usa-link"><sup>b</sup></a>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-110_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-110" display="inline"><mrow><mi>n</mi></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-111_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-111" display="inline"><mrow><msub><mi>y</mi><mtext>MIN</mtext></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-112_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-112" display="inline"><mrow><msub><mi>y</mi><mtext>MAX</mtext></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-113_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-113" display="inline"><mrow><msub><mi mathvariant="normal">V</mi><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-114_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-114" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-115_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-115" display="inline"><mrow><msup><mi>h</mi><mn>2</mn></msup></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-116_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-116" display="inline"><mrow><msup><mi>H</mi><mn>2</mn></msup></mrow></math></span></td></tr></table>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Diversity panel</td>
<td align="left" rowspan="1" colspan="1">Runner score</td>
<td align="left" rowspan="1" colspan="1">932</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">15.9</td>
<td align="left" rowspan="1" colspan="1">0.60</td>
<td align="left" rowspan="1" colspan="1">0.86</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">F<sub>2</sub> families</td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">270</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">104</td>
<td align="left" rowspan="1" colspan="1">58.5</td>
<td align="left" rowspan="1" colspan="1">32.7</td>
<td align="left" rowspan="1" colspan="1">0.37</td>
<td align="left" rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Full‐sib families</td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">372</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">98</td>
<td align="left" rowspan="1" colspan="1">183.9</td>
<td align="left" rowspan="1" colspan="1">30.7</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
<td align="left" rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Inflorescence count</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">41</td>
<td align="left" rowspan="1" colspan="1">19.1</td>
<td align="left" rowspan="1" colspan="1">177.7</td>
<td align="left" rowspan="1" colspan="1">0.60</td>
<td align="left" rowspan="1" colspan="1">—</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270086-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270086-tbl2-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Statistics were estimated from clonal replicates of strawberry diversity panel (SDP) individuals, unreplicated seed‐propagated 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-117" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-118" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> progeny, and unreplicated seed‐propagated full‐sib progeny from nine crosses between elite parents with contrasting runnering phenotypes (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-119" display="inline"><mrow><mi>n</mi></mrow></math></span> is the number of individuals in the population. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-120" display="inline"><mrow><msub><mi>y</mi><mtext>MIN</mtext></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-121" display="inline"><mrow><msub><mi>y</mi><mtext>MAX</mtext></msub></mrow></math></span> are phenotypic minimums and maximums, respectively. Genetic variance among clones of individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-122" display="inline"><mrow><msub><mi>V</mi><mi mathvariant="normal">G</mi></msub></mrow></math></span>) and broad‐sense heritability on a clone‐mean basis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-123" display="inline"><mrow><mrow><msup><mi>H</mi><mn>2</mn></msup><mo>=</mo><msub><mi>V</mi><mi mathvariant="normal">G</mi></msub><mo>/</mo><msub><mi>V</mi><mover accent="true"><mi mathvariant="normal">P</mi><mo>¯</mo></mover></msub></mrow></mrow></math></span>) were estimated for runner score among SDP individuals, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-124" display="inline"><mrow><msub><mi>V</mi><mover accent="true"><mi mathvariant="normal">P</mi><mo>¯</mo></mover></msub></mrow></math></span> is the phenotypic variance on a clone‐mean basis. Genomic additive genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-125" display="inline"><mrow><msub><mi>V</mi><mi mathvariant="normal">A</mi></msub></mrow></math></span>) and narrow‐sense heritability on a clone‐mean basis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-126" display="inline"><mrow><mrow><msup><mi>h</mi><mn>2</mn></msup><mo>=</mo><msub><mi>V</mi><mi mathvariant="normal">A</mi></msub><mo>/</mo><msub><mi>V</mi><mover accent="true"><mi mathvariant="normal">P</mi><mo>¯</mo></mover></msub></mrow></mrow></math></span>) were estimated for runner score among SDP individuals. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-127" display="inline"><mrow><msub><mi>V</mi><mi mathvariant="normal">A</mi></msub></mrow></math></span> and narrow‐sense heritability on a individual‐plant basis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-128" display="inline"><mrow><mrow><msup><mi>h</mi><mn>2</mn></msup><mo>=</mo><msub><mi>V</mi><mi mathvariant="normal">A</mi></msub><mo>/</mo><msub><mi>V</mi><mi mathvariant="normal">P</mi></msub></mrow></mrow></math></span>) were estimated for runner count among F<sub>2</sub> progeny and runner and inflorescence count among full‐sib progeny, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-129" display="inline"><mrow><msub><mi>V</mi><mi mathvariant="normal">P</mi></msub></mrow></math></span> is the phenotypic variance on an individual plant basis. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-130" display="inline"><mrow><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub><mo>=</mo><msqrt><msub><mi>V</mi><mi mathvariant="normal">A</mi></msub></msqrt><mo>/</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mo>×</mo><mn>100</mn></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-131" display="inline"><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></math></span> is the population mean.</p>
</div>
<div class="fn" id="tpg270086-tbl2-note-0002">
<sup>
<sup>b</sup>
</sup><p class="display-inline">The flowering habits of single nucleotide polymorphism (SNP) individuals were classified as short‐day or day‐neutral from the absence or presence of flowering, respectively, under long days (13.5–14.5 h) over a period of 6 years in Winters, CA. The runner scores of SDP individuals were similarly recorded over a period 6 years in Winters, CA using an ordinal scale, where 1 = <em>runnerless</em>, 2 = <em>weak runnering</em>, 3 = <em>intermediate runnering</em>, 4 = <em>strong runnering</em>, and 5 = <em>extreme runnering</em> (see Figure <a href="#tpg270086-fig-0003" class="usa-link">3A</a>). F<sub>2</sub> and full‐sib progeny were phenotyped by counting the number of primary runners observed in Davis or Winters, CA, respectively, within 1 week of the summer solstice (the daylength was 14.5 h).</p>
</div>
</div></section><p>Although runnerless and near‐runnerless phenotypes have long been known in so‐called “ever‐bearing” cultivars of octoploid strawberry (Darrow, <a href="#tpg270086-bib-0016" class="usa-link" aria-describedby="tpg270086-bib-0016">1929</a>, <a href="#tpg270086-bib-0017" class="usa-link" aria-describedby="tpg270086-bib-0017">1966</a>), the runnerless phenotype of 55C032P001 and other perpetual flowering descendants of the Wasatch ecotype have apparently never been documented (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a>, <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a>). The emergence of that transgressive phenotype among Shasta <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-132" display="inline"><mrow><mo>×</mo></mrow></math></span> Wasatch progeny must have been unanticipated because both parents are prolific runner producers (Bringhurst and Voth, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>). The mean RS for Shasta was 3.0 in our study (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The original Wasatch ecotype disappeared from the UC collection long ago; however, the ecotype recollected by J. Hancock et al. (<a href="#tpg270086-bib-0030" class="usa-link" aria-describedby="tpg270086-bib-0030">2001</a>) from the original Wasatch population (BT3; PI612491) was found to have the prolific and unrestrained runnering phenotype typical of <em>F. virginiana</em> spp. <em>glauca</em>. The mean RS for BT3 was 4.83 in our study (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>).</p></section><section id="tpg270086-sec-0140"><h3 class="pmc_sec_title">3.3. Physical mapping of the <em>PF</em> locus uncovered the persistence of linkage‐dragged DNA of the wild donor</h3>
<p>We physically mapped the <em>PF</em> locus to the lower arm of chromosome 4B using associations between octoploid genome‐anchored 50,000 array SNPs and long‐day flowering habit classifications of SDP individuals (Figures <a href="#tpg270086-fig-0005" class="usa-link">5A</a> and <a href="#tpg270086-fig-0006" class="usa-link">6A</a>). Statistically significant associations were not observed elsewhere in the genome apart from singletons on chromosomes 4C and 7A that were identified in our initial genome‐wide search and subsequently found to be false‐positives (Figure <a href="#tpg270086-fig-0005" class="usa-link">5A</a>). The lead <em>PF</em>‐associated SNPs were AX‐184219235 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-133" display="inline"><mrow><mrow><mo>−</mo><mi>log</mi><mn>10</mn><mo stretchy="false">(</mo><mi>p</mi><mo stretchy="false">)</mo></mrow></mrow></math></span> = 35.7; bp 27,206,278) and AX‐184947290 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-134" display="inline"><mrow><mrow><mo>−</mo><mtext>log10</mtext><mo stretchy="false">(</mo><mi>p</mi><mo stretchy="false">)</mo></mrow></mrow></math></span> = 34.5; bp 27,294,311).</p>
<figure class="fig xbox font-sm" id="tpg270086-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/e5def8654c9d/TPG2-18-e70086-g004.jpg" loading="lazy" id="jats-graphic-9" height="626" width="692" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plots displaying Axiom 50,000 array‐genotyped SNPs associated with flowering or runnering phenotypes across the octoploid strawberry genome. There are four pairs of plots (A–B, C–D, E–F, and G–H) displaying statistics for genome‐wide association study (GWAS) without fixed effects (upper panel in each pair) and where the <em>PF</em>‐associated single nucleotide polymorphism (SNP) AX‐184947290 was incorporated as a fixed effect (lower panel in each pair). (A–B) GWAS for flowering habit among 932 strawberry diversity panel individuals. (C–D) GWAS for runner score among 932 strawberry diversity panel individuals. (E–F) GWAS for runner count among 183 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-135" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce F<sub>2</sub> and 87 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-136" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> progeny. (G–H) GWAS for runner count among 372 full‐sib progeny developed from crosses between elite UC parents.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270086-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/5fc7077e2a9a/TPG2-18-e70086-g001.jpg" loading="lazy" id="jats-graphic-11" height="815" width="782" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plots displaying Axiom 50,000 or 850,000 array‐genotyped single nucleotide polymorphisms (SNPs) associated with flowering or runnering phenotypes on chromosome 4B. The physical locations of several array‐genotyped SNPs (identified by AX prefixes), previously genetically mapped DNA markers (identified by other labels), and <em>RGA1‐4B</em> (a <em>REPRESSOR OF GA1</em> homoeolog) are shown in the UCD Royal Royce genome. The Manhattan plots shown in (B) and the right hand column of (E–H) display associations identified when AX‐184947290 was used as a fixed effect. (A) SNPs associated with flowering habit observed among 932 strawberry diversity panel (SDP) individuals. The upper rug plot shows the physical positions of 322 Axiom 50,000 array SNPs, whereas the lower rug plot shows the positions of 905 annotated genes spanning Mb 25.0–30.4 in the UCD Royal Royce genome. (B) SNPs associated with flowering habit among 932 SDP individuals identified by GWAS using AX‐184947290 as a fixed effect. (C) SNPs associated with flowering habit among 66 UC individuals genotyped with the 850,000 array. (D) SNPs associated with flowering habit among 194 SDP individuals genotyped with the 850,000 array. (E) SNPs associated with runner score among 932 SDP individuals. (F) SNPs associated with runner count among 183 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-137" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce F<sub>2</sub> and 87 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-138" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> progeny. (G) SNPs associated with runner count among 372 full‐sib progeny developed from crosses between elite UC parents. (H) SNPs associated with inflorescence count among 372 full‐sib progeny developed from crosses between elite UC parents.</p></figcaption></figure><p>The pattern of LD decay for <em>PF</em>‐associated SNPs was mostly smooth in the SDP population (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A</a>). LD steadily decayed upstream of AX‐184947290 to an LD cliff (AX‐184094002; bp 26,496,157) and downstream of AX‐184947290 to the end of the chromosome. There was a trough and irregular pattern of LD decay immediately downstream of AX‐184947290 (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A</a>). Two SNPs downstream of the trough were strongly associated with the <em>PF</em> locus, AX‐184572767 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-139" display="inline"><mrow><mrow><mo>−</mo><mi>log</mi><mn>10</mn><mo stretchy="false">(</mo><mi>p</mi><mo stretchy="false">)</mo></mrow></mrow></math></span> = 27.69; bp 27,930,755) and AX‐184726447 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-140" display="inline"><mrow><mrow><mo>−</mo><mi>log</mi><mn>10</mn><mo stretchy="false">(</mo><mi>p</mi><mo stretchy="false">)</mo></mrow></mrow></math></span> = 32.28; bp 28,070,877). Those associations were likely caused by the highly admixed structure of the SDP population (Hou et al., <a href="#tpg270086-bib-0042" class="usa-link" aria-describedby="tpg270086-bib-0042">2021</a>), not by the segregation of a second locus, as shown below.</p>
<p>Because the effect of the <em>PF</em> locus dwarfs could mask the effects of other loci affecting photoperiod‐dependent flowering, GWAS was repeated using <em>PF</em>‐associated SNPs as fixed effects, specifically AX‐184219235 or AX‐184947290 alone; AX‐184219235 and AX‐184947290 in combination; and AX‐184219235, AX‐184947290, and AX‐184208850 in combination (Figures <a href="#tpg270086-fig-0005" class="usa-link">5B</a> and <a href="#tpg270086-fig-0006" class="usa-link">6B</a>). These were the three most highly predictive <em>PF</em>‐associated SNPs for flowering habit in the SDP. When AX‐184947290 alone was used as a fixed effect for GWAS, the effects of SNP singletons on chromosomes 4C and 7A, the SNP outliers downstream of AX‐184947290 on chromosome 4B (AX‐184572767 and AX‐184726447), and other <em>PF</em>‐associated SNPs on chromosome 4B either nearly or completely disappeared (Figures <a href="#tpg270086-fig-0005" class="usa-link">5B</a> and <a href="#tpg270086-fig-0006" class="usa-link">6B</a>). The effects of AX‐184208850 and AX‐184219235 were slightly above the genetic background when AX‐184947290 was used as a fixed effect and vice versa for the others (Figure <a href="#tpg270086-fig-0006" class="usa-link">6B</a>). Our results suggest that the fixed effects captured haplotypic diversity in the highly admixed SDP population (Hou et al., <a href="#tpg270086-bib-0042" class="usa-link" aria-describedby="tpg270086-bib-0042">2021</a>) and that the pattern of LD decay observed in the SDP population was caused by the segregation of a single gene in strong LD with AX‐184947290 (bp 27,294,311).</p>
<p>We genotyped a small sample of SD and DN UC individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-141" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>66</mn></mrow></mrow></math></span>) and a slightly larger sample of more diverse SD and DN individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-142" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>194</mn></mrow></mrow></math></span>) with an 850,000 SNP array to graphically genotype the Wasatch introgression on chromosome 4B (Figure <a href="#tpg270086-fig-0006" class="usa-link">6C</a>). While GWAS analyses of those small samples were underpowered and insufficient for accurately pinpointing the physical location of <em>PF</em>, the 850,000 array saturated the Wasatch introgression with 2600 SNPs, clearly identified a stable recombination breakpoint upstream of the <em>PF</em> locus (the upper boundary of the Wasatch introgression), and showed that Wasatch DNA has persisted and survived selection for perpetual flowering from the recombination breakpoint to the end of the chromosome since the original hybrid (Shasta <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-143" display="inline"><mrow><mo>×</mo></mrow></math></span> Wasatch) was developed in 1955 (Figure <a href="#tpg270086-fig-0006" class="usa-link">6C</a>). The upper boundary of the Wasatch introgression was marked by an LD cliff where significance dropped to near zero immediately upstream of the 850,000 array SNP AX‐184970075 (bp 26,490,375) (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A,C,D</a>). The recombination breakpoint was predicted to reside between AX‐184284360 (bp 26,484,790) and AX‐184970075 (26,490,375) among individuals genotyped with the 850,000 SNP array. That interval was inside the interval (bp 26,480,809–26,496,157) where the recombination breakpoint was predicted to reside in individuals genotyped with the 50,000 SNP array.</p>
<p>The stability of the upper boundary suggests that the location of crossovers has been highly constrained in Wasatch descendants that inherited the <em>PF</em> allele (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A,C,D</a>). We observed runs‐of‐homozygosity and suppressed recombination (a recombination cold spot) immediately upstream of the recombination breakpoint (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A,C,D</a>). The LD pattern upstream of the recombination breakpoint could correlate with structural variation between SD recipients and the Wasatch donor, pedigree inbreeding caused by backcrossing to closely related SD recipients, or a combination thereof (Ceballos et al., <a href="#tpg270086-bib-0013" class="usa-link" aria-describedby="tpg270086-bib-0013">2018</a>; Wright &amp; Andolfatto, <a href="#tpg270086-bib-0080" class="usa-link" aria-describedby="tpg270086-bib-0080">2008</a>).</p></section><section id="tpg270086-sec-0150"><h3 class="pmc_sec_title">3.4. SNP haplotypes shed light on the <em>PF</em>‐associated LD block found in Wastach descendants</h3>
<p>The SDP population included 423 DN individuals developed at UCD between 1955 and 2021 (Wasatch descendants) and 396 SD individuals developed at UCD between 1935 and 2021 (Figure <a href="#tpg270086-fig-0004" class="usa-link">4</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). With 66 generations of historical recombination (breeding), hybrid offspring of 563 unique pedigrees (FS crosses) among 498 parents in the SDP and FS populations combined, and highly accurate flowering habit classifications of SDP individuals, we anticipated that GWAS might pinpoint the physical position of the gene encoded by <em>PF</em>; however, recombination breakpoints were scarce from the upper boundary of Wasatch introgression (Mb 26.53) to the lower boundary (Mb 27.28–27.29) of the LD block predicted to harbor <em>PF</em> (Figure <a href="#tpg270086-fig-0007" class="usa-link">7</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S6</a>).</p>
<figure class="fig xbox font-sm" id="tpg270086-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/312bf753f688/TPG2-18-e70086-g002.jpg" loading="lazy" id="jats-graphic-13" height="597" width="546" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Haplotypes for 51 single nucleotide polymorphisms (SNPs) flanking the <em>PF</em> locus on chromosome 4B (Mb 26.53–29.64) among 808 strawberry diversity panel (SDP) individuals phenotyped for long day flowering habit. The selected SNPs had minor allele frequencies <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-144" display="inline"><mrow><mrow><mo>≥</mo><mn>0.3</mn></mrow></mrow></math></span> and exceeded the false‐discovery‐rate (FDR)‐corrected significance threshold of 2 for runner score genome‐wide association study (GWAS) in the SDP population (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-145" display="inline"><mrow><mrow><mo>−</mo><msub><mi>log</mi><mn>10</mn></msub></mrow></mrow></math></span>
<em>p</em>‐values <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-146" display="inline"><mrow><mo>≥</mo></mrow></math></span> 2). The heatmap displays squared correlation coefficient (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-147" display="inline"><mrow><msup><mi>R</mi><mn>2</mn></msup></mrow></math></span>) linkage disequilibrium (LD) statistics among the 51 SNPs, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-148" display="inline"><mrow><mrow><msup><mi>R</mi><mn>2</mn></msup><mo>=</mo><mn>1</mn></mrow></mrow></math></span> when LD is perfect (black cells), <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-149" display="inline"><mrow><mrow><msup><mi>R</mi><mn>2</mn></msup><mo>=</mo><mn>0</mn></mrow></mrow></math></span> when LD is absent (white cells), and cell darkness increases as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-150" display="inline"><mrow><msup><mi>R</mi><mn>2</mn></msup></mrow></math></span> increases. The 51‐SNP LD block was split into a 33‐SNP LD block upstream of AX‐184240324 and an 18‐SNP LD block downstream of AX‐ 184947290 (identified by a solid line between the upper and lower LD blocks). Haplotypes are shown for individuals carrying six out of 400 unique haplotypes in the 33‐SNP LD block, including the Wasatch haplotype (W), a common short‐day haplotype (N1), and four others (N2–N5). <em>PF</em> genotypes are shown in parentheses below the names of each individual. Axiom 50,000 array SNP marker names are displayed in the upper panel for a select subset of the 51 SNPs.</p></figcaption></figure><p>To assess the strength and pattern of LD and search for recombination breakpoints, haplotypes of 51 <em>PF</em>‐associated SNPs were phased among 808 SDP individuals (Figure <a href="#tpg270086-fig-0007" class="usa-link">7</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S6</a>). The SNPs selected for this analysis had minor allele frequencies (MAFs) greater than 0.3 and GWAS effects for RS that exceeded a false‐discovery‐rate (FDR) significance threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-151" display="inline"><mrow><mrow><mo>−</mo><msub><mi>log</mi><mn>10</mn></msub><mrow><mo stretchy="false">(</mo><mtext>p-value</mtext><mo stretchy="false">)</mo></mrow><mo>=</mo><mn>2</mn></mrow></mrow></math></span> among SDP population individuals (Figure <a href="#tpg270086-fig-0006" class="usa-link">6C</a>, <a href="#tpg270086-fig-0006" class="usa-link">D</a>). We identified 400 unique 51‐SNP haplotypes among 808 SDP individuals (1616 haplotypes; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S6</a>). The five most common haplotypes were observed 74 to 234 times each and accounted for 700 of the 1616 haplotypes (43.3%), whereas 291 haplotypes were only observed once. The 51‐SNP Wasatch haplotype was observed in 87% of the individuals classified as DN (<em>PF</em>_).</p>
<p>Strong LD was observed among the 33 phased SNPs upstream of AX‐184240324 (Mb 27.29; Figure <a href="#tpg270086-fig-0007" class="usa-link">7</a>). LD was markedly weaker among the 18 phased SNPs downstream of AX‐184947290 (Mb 27.28). We identified 182 unique 33‐SNP haplotypes, including a common Wasatch haplotype (W) in the upper LD block (Figure <a href="#tpg270086-fig-0007" class="usa-link">7</a>). The boundary between the upper and lower LD blocks was supported by the absence of <em>PF</em> segregation among 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-152" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> progeny and among FS progeny developed from crosses between UC Eclipse and SD cultivars (Figure <a href="#tpg270086-fig-0008" class="usa-link">8</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a>). UC Eclipse carries a recombination breakpoint immediately downstream of AX‐184240324; hence, we concluded that <em>PF</em> is found upstream of the upper boundary of the lower LD block. Cabrillo and UCD Finn share that recombination breakpoint (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a>). When combined with insights gained from unique recombination breakpoints among F<sub>2</sub> progeny, SNP haplotypes of SDP individuals, and the presence or absence of <em>PF</em> segregation among F<sub>2</sub> progeny and UC Eclipse <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-153" display="inline"><mrow><mo>×</mo></mrow></math></span> SD FS progeny, we concluded that the <em>PF</em> gene must be found between AX‐184021395 (Mb 26.40) and AX‐184240324 (Mb 27.28) on chromosome 4B (Figures <a href="#tpg270086-fig-0006" class="usa-link">6</a>, <a href="#tpg270086-fig-0007" class="usa-link">7</a>, <a href="#tpg270086-fig-0008" class="usa-link">8</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S6–S8</a>).</p>
<figure class="fig xbox font-sm" id="tpg270086-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/2cd7705c052d/TPG2-18-e70086-g005.jpg" loading="lazy" id="jats-graphic-15" height="496" width="700" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variation for runner count (left column) and Manhattan plots displaying Axiom 50,000 array‐genotyped single nucleotide polymorphisms (SNPs) associated with runner count on chromosome 4B (right column) in four F<sub>2</sub> populations. (A) 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-154" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce F<sub>2</sub> (21S950 originating by self‐pollinating the 19A907P022 F<sub>1</sub>). (B) 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-155" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce F<sub>2</sub> (21S951 originated by self‐pollinating the 19A907P024 F<sub>1</sub>). (C) 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-156" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> (21S952 originated by self‐pollinating the 19A908P012 F<sub>1</sub>). (D) 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-157" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> (21S953 originated by self‐pollinating the 19A908P053 F<sub>1</sub>). The lead SNPs are labeled.</p></figcaption></figure><p>Our search of annotations in UCD Royal Royce genome assemblies (both haplotypes) identified 193 genes spanning bp 26,401,121–27,418,663 on chromosome 4B (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S9</a>). That search did not uncover any genes known or predicted to regulate flowering in angiosperms (Hempel et al., <a href="#tpg270086-bib-0038" class="usa-link" aria-describedby="tpg270086-bib-0038">1997</a>). We did, however, discover a homolog of <em>REPRESSOR OF GA1</em> on chromosome 4B (<em>RGA1‐4B</em>) downstream of <em>PF</em>. <em>RGA1‐4B</em> gene identifiers in the haplotype‐phased UCD Royal Royce genome assemblies are Fxa4Bg203107 (haplotype 2; Mb 28.83–28.94) and Fxa4Bg103085 (haplotype 1; Mb 29.15–29.25). Homoeologs were also identified in the other three genomes (Figure <a href="#tpg270086-supinfo-0001" class="usa-link">S2</a>). Studies in <em>F. vesca</em> identified a <em>REPRESSOR OF GA1</em> (<em>FveRGA1</em>) gene on chromosome 4 as a causal determinant of the runnerless phenotype of Yellow Wonder (Caruana et al., <a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>; Tenreira et al., <a href="#tpg270086-bib-0074" class="usa-link" aria-describedby="tpg270086-bib-0074">2017</a>). Hawaii 4 develops runners under long days, whereas Yellow Wonder does not.</p></section><section id="tpg270086-sec-0160"><h3 class="pmc_sec_title">3.5. Genome‐wide association studies identified a single QTL for runnering</h3>
<p>As anticipated, we observed LD blocks of <em>PF</em>‐coincident RS and count‐associated SNPs on the distal end of chromosome 4B in GWASs of the SDP, F<sub>2</sub>, and FS populations (Figures <a href="#tpg270086-fig-0005" class="usa-link">5C–H</a> and <a href="#tpg270086-fig-0006" class="usa-link">6E–G</a>). Those <em>PF</em>‐associated SNPs identify the <em>PFRU</em> QTL previously identified by genetic mapping (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>). Significant associations with RS or count QTL were not observed elsewhere in the genome (Figure <a href="#tpg270086-fig-0005" class="usa-link">5</a>). That was unanticipated, especially in the SDP and F<sub>2</sub> populations. The absence of QTL was less surprising in the FS population because the parents were elite individuals with runnering phenotypes that are less extreme than the exotic parents of the F<sub>2</sub> populations and exotic individuals in the SDP population (Figure <a href="#tpg270086-fig-0004" class="usa-link">4</a>; Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
<p>Our initial hypothesis was that the runnerless phenotype of 55C032P001 might be caused by a single gene mutation independent of the Wasatch <em>PF</em> mutation. This was based on the finding in woodland strawberry (<em>F. vesca</em>) that the runnerless phenotype was caused by mutations in a single gene (<em>R</em>) independent of the gene (<em>S</em>) that controls the perpetual flowering phenotype (Brown &amp; Wareing, <a href="#tpg270086-bib-0008" class="usa-link" aria-describedby="tpg270086-bib-0008">1965</a>). We did not, however, uncover an independent mutation for the runnerless phenotype in our GWASs of 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-158" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> progeny developed by self‐pollinating runnerless F<sub>1</sub> individuals (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a> and <a href="#tpg270086-fig-0005" class="usa-link">5</a>; Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). Our analyses solely pointed to the <em>PFRU</em> QTL (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>) and showed that the genetic architecture of runnering is complex in octoploid strawberry (Figure <a href="#tpg270086-fig-0008" class="usa-link">8E–G</a>; Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="tpg270086-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Statistics<a href="#tpg270086-tbl3-note-0001" class="usa-link">
<sup>a</sup>
</a> for an A/G single nucleotide polymorphisms (SNP) marker (AX‐184947290) associated with the <em>PF</em> locus on chromosome 4B.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Population<a href="#tpg270086-tbl3-note-0002" class="usa-link"><sup>b</sup></a>
</th>
<th align="left" rowspan="1" colspan="1">Trait<a href="#tpg270086-tbl3-note-0003" class="usa-link"><sup>c</sup></a>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-159_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-159" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-160_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-160" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AG</mtext></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-161_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-161" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-162_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-162" display="inline"><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-163_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-163" display="inline"><mrow><mover accent="true"><mi>d</mi><mo>^</mo></mover></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-164_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-164" display="inline"><mrow><mrow><mrow><mo stretchy="false">|</mo></mrow><mover accent="true"><mi>d</mi><mo>^</mo></mover><mo>/</mo><mover accent="true"><mi>a</mi><mo>^</mo></mover><mrow><mo stretchy="false">|</mo></mrow></mrow></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">PVE (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Diversity panel</td>
<td align="left" rowspan="1" colspan="1">Runner score</td>
<td align="left" rowspan="1" colspan="1">3.7</td>
<td align="left" rowspan="1" colspan="1">3.1</td>
<td align="left" rowspan="1" colspan="1">3.0</td>
<td align="left" rowspan="1" colspan="1">0.4</td>
<td align="left" rowspan="1" colspan="1">−0.2</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">14.9</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pooled F<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">31.5</td>
<td align="left" rowspan="1" colspan="1">20.8</td>
<td align="left" rowspan="1" colspan="1">15.0</td>
<td align="left" rowspan="1" colspan="1">8.3</td>
<td align="left" rowspan="1" colspan="1">−2.4</td>
<td align="left" rowspan="1" colspan="1">0.29</td>
<td align="left" rowspan="1" colspan="1">29.1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">21S950 F<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">16.0</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">21S951 F<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">38.6</td>
<td align="left" rowspan="1" colspan="1">27.9</td>
<td align="left" rowspan="1" colspan="1">20.5</td>
<td align="left" rowspan="1" colspan="1">9.1</td>
<td align="left" rowspan="1" colspan="1">−1.7</td>
<td align="left" rowspan="1" colspan="1">0.19</td>
<td align="left" rowspan="1" colspan="1">24.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">21S952 F<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">27.7</td>
<td align="left" rowspan="1" colspan="1">14.2</td>
<td align="left" rowspan="1" colspan="1">9.9</td>
<td align="left" rowspan="1" colspan="1">8.9</td>
<td align="left" rowspan="1" colspan="1">−4.6</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">31.1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">21S953 F<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">22.1</td>
<td align="left" rowspan="1" colspan="1">13.7</td>
<td align="left" rowspan="1" colspan="1">10.8</td>
<td align="left" rowspan="1" colspan="1">5.6</td>
<td align="left" rowspan="1" colspan="1">−2.8</td>
<td align="left" rowspan="1" colspan="1">0.49</td>
<td align="left" rowspan="1" colspan="1">22.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Full‐sib families</td>
<td align="left" rowspan="1" colspan="1">Runner score</td>
<td align="left" rowspan="1" colspan="1">3.6</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
<td align="left" rowspan="1" colspan="1">2.8</td>
<td align="left" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">0.0</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">7.1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Runner count</td>
<td align="left" rowspan="1" colspan="1">50.7</td>
<td align="left" rowspan="1" colspan="1">37.6</td>
<td align="left" rowspan="1" colspan="1">28.5</td>
<td align="left" rowspan="1" colspan="1">11.1</td>
<td align="left" rowspan="1" colspan="1">−2.0</td>
<td align="left" rowspan="1" colspan="1">0.18</td>
<td align="left" rowspan="1" colspan="1">15.0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Inflorescence count</td>
<td align="left" rowspan="1" colspan="1">0.0</td>
<td align="left" rowspan="1" colspan="1">4.5</td>
<td align="left" rowspan="1" colspan="1">10.6</td>
<td align="left" rowspan="1" colspan="1">−5.3</td>
<td align="left" rowspan="1" colspan="1">−0.8</td>
<td align="left" rowspan="1" colspan="1">0.14</td>
<td align="left" rowspan="1" colspan="1">46.7</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270086-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270086-tbl3-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-165" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-166" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AG</mtext></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-167" display="inline"><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub></mrow></math></span> are estimates of SNP marker genotypic means. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-168" display="inline"><mrow><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub><mo>−</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><mn>2</mn></mrow></mrow></math></span> is the additive effect, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-169" display="inline"><mrow><mrow><mover accent="true"><mi>d</mi><mo>^</mo></mover><mo>=</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AG</mtext></msub><mo>−</mo><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>GG</mtext></msub><mo>+</mo><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>AA</mtext></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><mn>2</mn></mrow></mrow></math></span> is the dominance effect, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-170" display="inline"><mrow><mrow><mrow><mo stretchy="false">|</mo></mrow><mover accent="true"><mi>d</mi><mo>^</mo></mover><mo>/</mo><mover accent="true"><mi>a</mi><mo>^</mo></mover><mrow><mo stretchy="false">|</mo></mrow></mrow></mrow></math></span> is the degree‐of‐dominance of the AX‐184947290 SNP marker locus. PVE is an estimate of the phenotypic variances explained by the SNP marker locus.</p>
</div>
<div class="fn" id="tpg270086-tbl3-note-0002">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Statistics were estimated from clonal replicates of strawberry diversity panel individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-171" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>932</mn></mrow></mrow></math></span>), unreplicated seed‐propagated 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-172" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce (21S950 and 21S951) and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-173" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 (21S952 and 21S953) F<sub>2</sub> progeny (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-174" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>270</mn></mrow></mrow></math></span>), and unreplicated seed‐propagated full‐sib progeny (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-175" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>372</mn></mrow></mrow></math></span>) developed from nine crosses between elite parents with contrasting runnering phenotypes (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
</div>
<div class="fn" id="tpg270086-tbl3-note-0003">
<sup>
<sup>b</sup>
</sup><p class="display-inline">The flowering habits of SDP individuals were classified as short‐day or day‐neutral from the absence or presence of flowering, respectively, under long daylengths (13.5–14.5 h) in Winters, CA. The runner scores of strawberry diversity panel (SDP) individuals were recorded over a period of 6 years in Winters, CA using an ordinal scale, where 1 = <em>runnerless</em>, 2 = <em>weak runnering</em>, 3 = <em>intermediate runnering</em>, 4 = <em>strong runnering</em>, and 5 = <em>extreme runnering</em> (see Figure <a href="#tpg270086-fig-0003" class="usa-link">3A</a>). F<sub>2</sub> and full‐sib progeny were phenotyped by counting the number of primary runners observed in Davis or Winters, CA, respectively, within 1 week of the summer solstice (the daylength was 14.5 h).</p>
</div>
</div></section><p>The statistics reported throughout this paper for runner count in the F<sub>2</sub> and FS populations were estimated from phenotypes observed at the summer solstice when the photoperiod was 14.5 h (Figures <a href="#tpg270086-fig-0005" class="usa-link">5</a>, <a href="#tpg270086-fig-0006" class="usa-link">6</a>, <a href="#tpg270086-fig-0008" class="usa-link">8</a>, and <a href="#tpg270086-fig-0009" class="usa-link">9</a>). We selected that time point because the effect of <em>PF</em> locus on runnering was greatest at the summer solstice (Figure <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The temporal effect of the <em>PF</em> locus is shown in Manhattan plots developed for GWAS analyses of phenotypes observed among F<sub>2</sub> progeny on seven dates, from March 26 (12.3 h photoperiod) to July 6, 2022 (14.5 h photoperiod) (Figure <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). As anticipated, the pleiotropic effect of the <em>PF</em> locus on runnering was weakest and nonsignificant early in the season (March 26 –May 6) and steadily increased through the summer solstice (June 21).</p>
<figure class="fig xbox font-sm" id="tpg270086-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e54f/12365471/4bbdb5bc056d/TPG2-18-e70086-g007.jpg" loading="lazy" id="jats-graphic-17" height="661" width="302" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270086-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variation for runner score, runner count, and inflorescence count among 372 full‐sib progeny from nine families phenotyped July 1, 2024 in Davis, CA (14.5 h photoperiod) from seedlings transplanted on October 15, 2023. <em>PF</em> and the <em>PF</em>‐associated single nucleotide polymorphism (SNP) AX‐18497290 segregated in this population. The parents were elite UC short‐day and day‐neutral individuals (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). (A–E) AX‐184947290 SNP genotypes are color coded dark gray (AA), light gray (AG), and orange (GG). (A) The runner count by runner score distribution. (B) The runner count by inflorescence count distribution. (C–E) The runner count distributions among AA, AG, and GG individuals.</p></figcaption></figure><p>The patterns of LD decay for RS or count‐associated SNPs on chromosome 4B differed in the SDP, F<sub>2</sub>, and FS populations because of differences in population structure, pedigree inbreeding, recombination, and selection history (Figure <a href="#tpg270086-fig-0006" class="usa-link">6E–G</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S6–S8</a>). They provided different pieces to the puzzle for understanding the pleiotropic effects of the <em>PF</em> locus on runnering in elite and exotic genetic backgrounds (Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>). The patterns of LD decay for flowering‐habit‐ and RS‐associated SNPs on chromosome 4B were virtually identical in the highly admixed SDP population (Figure <a href="#tpg270086-fig-0004" class="usa-link">4</a> Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S6</a>). The lead SNP AX‐184947290 was predicted to be closest to the gene encoded by <em>PF</em> (Figure <a href="#tpg270086-fig-0006" class="usa-link">6E</a>). Using that SNP as a predictor, the <em>PF</em> locus was estimated to explain approximately 22.1% of the genetic variance and 14.9% of the phenotypic variance for RS in the SDP population (Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>). As observed for flowering habit, when AX‐184947290 was used as a fixed effect, the effects of other <em>PF</em>‐associated SNPs disappeared (Figures <a href="#tpg270086-fig-0005" class="usa-link">5D</a> and <a href="#tpg270086-fig-0006" class="usa-link">6E</a>).</p>
<p>The strongest GWAS signals and smoothest patterns of LD decay for runnering were observed in the F<sub>2</sub> populations (Figures <a href="#tpg270086-fig-0005" class="usa-link">5</a>, <a href="#tpg270086-fig-0006" class="usa-link">6</a>, and <a href="#tpg270086-fig-0008" class="usa-link">8A–D</a>). Figures <a href="#tpg270086-fig-0005" class="usa-link">5E</a> and <a href="#tpg270086-fig-0006" class="usa-link">6F</a> display Manhattan plots for analyses of the F<sub>2</sub> populations combined. Figure <a href="#tpg270086-fig-0008" class="usa-link">8</a> displays the runner count phenotypic distributions and chromosome 4B Manhattan plots for analyses of individual F<sub>2</sub> populations. The combined F<sub>2</sub> population analysis produced sharper and cleaner GWAS signals than those observed in the SDP or FS populations because the phenotypes of the parents were extreme, haplotypic variation was minimum, and the parents carried unique recombination breakpoints (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a> and <a href="#tpg270086-supinfo-0001" class="usa-link">S8</a>). We observed an LD cliff immediately downstream of AX‐184942790 in combined F<sub>2</sub> and individual 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-176" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> analyses with comparatively flat LD beyond the LD cliff to the end of the chromosome, in contrast to the choppy pattern of LD decay observed immediately downstream of AX‐184942790 in the SDP population (Figures <a href="#tpg270086-fig-0006" class="usa-link">6E</a>,‐F and <a href="#tpg270086-fig-0008" class="usa-link">8B–D</a>).</p>
<p>We discovered that individuals in the 21S950 (17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-177" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce) F<sub>2</sub> family were homozygous for Wasatch alleles from the upper boundary of the Wasatch introgression (AX‐184251339; bp 25,006,055) to AX‐184418344 (bp 27,417,624), the 50,000 array SNP immediately downstream of AX‐184947290 (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S8</a>). There was an absence of runner‐count associated SNPs across the Mb 25.0–30.4 genomic segment in that F<sub>2</sub> family; hence, we concluded that the gene was likely located upstream of bp 27,417,624 (Figure <a href="#tpg270086-fig-0008" class="usa-link">8A</a>). By contrast, significant runner count‐associated SNPs were observed across the entire genomic segment (Mb 25.0–30.3) in the other 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-178" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce F<sub>2</sub> family (21S951). High‐density graphical genotypes and haplotypes showed that AX‐184947290 and numerous other SNPs upstream of AX‐184418344 (bp 27,417,624) segregated in that F<sub>2</sub> family (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a> and <a href="#tpg270086-supinfo-0001" class="usa-link">S8</a>). The LD blocks transmitted in those F<sub>2</sub> individuals were wide, thereby creating a comparatively flat pattern of LD decay across the entire genomic segment in the 21S951 F<sub>2</sub> family (Figure <a href="#tpg270086-fig-0008" class="usa-link">8B</a>).</p>
<p>The runnerless 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-179" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>1</sub> individuals that we selected and self‐pollinated (19A908P012 and 19A908P053) were discovered to be homozygous upstream of AX‐184198749 (bp 26,691,998) and downstream of AX‐184947290 (bp 27,294,311) (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a> and <a href="#tpg270086-supinfo-0001" class="usa-link">S8</a>), and that genomic segment segregated in both of those F<sub>2</sub> families (21S952 and 21S953; Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>). Fourteen SNPs in the segregating genomic segment were significantly associated with runner count (Figure <a href="#tpg270086-fig-0008" class="usa-link">8C,D</a>). Hence, when combined with our association studies in the 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-180" display="inline"><mrow><mo>×</mo></mrow></math></span> UCD Royal Royce F<sub>2</sub> families, we concluded that the gene encoded by <em>PF</em> must be located upstream of AX‐184418344 (bp 27,417,624) and proximal to AX‐184947290 (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a> and <a href="#tpg270086-supinfo-0001" class="usa-link">S8</a>).</p>
<p>Although the dominant Wasatch <em>PF</em> allele partially suppresses runnering under long days, we found that <em>PF</em>_ individuals can have dramatically different runnering phenotypes and that runnering is not always suppressed in <em>PF</em>_ individuals, for example, the runnering phenotype of 21C221P054 (<em>PFpf</em>) was extreme (RS = 5.0) and virtually identical to that of the wild parent (Figures <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a>, <a href="#tpg270086-fig-0008" class="usa-link">8</a>, and <a href="#tpg270086-fig-0009" class="usa-link">9</a>). At the other extreme, 20C420P006 (<em>PFpf</em>) was runnerless (RS = 1.0; Figure <a href="#tpg270086-fig-0002" class="usa-link">2</a>). Our analyses of individual F<sub>2</sub> populations showed that the phenotypic variation observed for runnering among SD (<em>pfpf</em>) and DN (<em>PF</em>_) individuals cannot be explained by the segregation of <em>PF</em> alone (Figures <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a>, and <a href="#tpg270086-fig-0008" class="usa-link">8</a>; Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>). The runner count distributions were strikingly similar among the four F<sub>2</sub> families; however, phenotypes ranged from runnerless to prolific in three of the four (21S951, 21S952, and 21S953) and weak (10 runners/plant was the minimum) to prolific in the other (21S950) (Figure <a href="#tpg270086-fig-0008" class="usa-link">8</a>). GWAS for runner count showed that the 21S950 F<sub>2</sub> family did not segregate for the <em>PF</em> gene, whereas the other three F<sub>2</sub> families did (Figure <a href="#tpg270086-fig-0008" class="usa-link">8</a>; Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a> and <a href="#tpg270086-supinfo-0001" class="usa-link">S8</a>).</p>
<p>The FS population combined progeny from nine crosses between elite parents with runnering phenotypes typical of commercially important cultivars (RSs in the 2.5–4.0 range; Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>; Figure <a href="#tpg270086-fig-0009" class="usa-link">9</a>). The individuals in those families were phenotyped for runner and inflorescence count. Association studies of the latter shed light on the pleiotropic effect of the <em>PF</em> locus on runner count in elite genetic backgrounds (Figure <a href="#tpg270086-fig-0006" class="usa-link">6H</a>). Significant inflorescence count–associated SNPs identify the boundaries of the <em>PF</em>‐associated LD blocks transmitted by the parents. The pattern of LD decay for flowering habit was comparatively flat from Mb 26.5 to 28.0 among FSs because of limited recombination. We observed a distinct LD cliff upstream of <em>PF</em> aligned with the stable recombination coldspot found in Wasatch descendants (Figure <a href="#tpg270086-fig-0006" class="usa-link">6H</a>). That pattern clearly shows that LD blocks spanning at least a 0.5 Mb upstream and downstream of the <em>PF</em> locus (Mb 27.3) are typically transmitted in segregating populations, partly as a function of limited recombination and partly as a function of shared ancestry (pedigree inbreeding) among elite parents (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>) As anticipated, the effect of the <em>PF</em> locus was approximately three‐fold greater for inflorescence than runner count (Figure <a href="#tpg270086-fig-0006" class="usa-link">6G,H</a>; Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>). Consequently, the LD decay for runner count was comparatively flat and lacked LD cliffs in the FS population (Figure <a href="#tpg270086-fig-0006" class="usa-link">6G</a>).</p>
<p>We found that RS was a directionally accurate predictor of runner count, particularly among individuals in the lower tails of both distributions (Figure <a href="#tpg270086-fig-0009" class="usa-link">9</a>). The runner counts of seed‐propagated FS progeny observed near the summer solstice (14.5 photoperiod) were highly correlated with RS (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-181" display="inline"><mrow><mrow><mi>r</mi><mo>=</mo><mn>0.86</mn></mrow></mrow></math></span>). Notably, the runner count ranges were narrowest among FS progeny with RSs in the runnerless to near‐runnerless range (1–2). The runnering phenotypes of at least 18.6% of the FS progeny were transgressive (1 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-182" display="inline"><mrow><mo>≤</mo></mrow></math></span> RS <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-183" display="inline"><mrow><mo>≤</mo></mrow></math></span> 2). Using the <em>PF</em>‐associated SNP AX‐184947290 as a predictor, of 58 individuals with RSs = 2, seven were <em>PFPF</em>, 29 were <em>PFpf</em>, and 22 were <em>pfpf</em>; hence, reduced runnering phenotypes were recovered among both SD and DN individuals.</p></section><section id="tpg270086-sec-0170"><h3 class="pmc_sec_title">3.6. The pleiotropic effect of the <em>PERPETUAL FLOWERING</em> locus on runnering across a broad cross‐section of octoploid genetic resources</h3>
<p>Our data support the hypothesis that the <em>PFRU</em> QTL for runnering previously identified by Gaston et al. (<a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>) was likely caused by the pleitropic effect of a single gene (<em>PF</em>). Using the lead SNP identified by GWAS (AX‐184947290), we estimated that the <em>PF</em> locus explained 7.1%–31.1% of the phenotypic variation for RS or count (Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>). The Wasatch allele has a dominant (Mendelian) effect on long‐day flowering habit; however, the pleiotropic effects of the <em>PF</em> allele on runnering are quantitative and incompletely dominant: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-184" display="inline"><mrow><mrow><mrow><mo stretchy="false">|</mo></mrow><mover accent="true"><mi>d</mi><mo>^</mo></mover><mo>/</mo><mover accent="true"><mi>a</mi><mo>^</mo></mover><mrow><mo stretchy="false">|</mo></mrow></mrow></mrow></math></span> ranged from 0.18 in the elite <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-185" display="inline"><mrow><mo>×</mo></mrow></math></span> elite FS population to 0.51 in the SDP population (Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>).</p>
<p>The <em>PF</em> locus is well‐known to affect the timing of the photoperiod‐dependent inhibition of flowering and discrete transition from sexual to asexual reproduction in SD (<em>pfpf</em>) plants (Ahmadi et al., <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Sønsteby and Heide, <a href="#tpg270086-bib-0068" class="usa-link" aria-describedby="tpg270086-bib-0068">2006</a>). While neither are observed in DN (<em>PF</em>_) plants (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Sønsteby and Heide, <a href="#tpg270086-bib-0068" class="usa-link" aria-describedby="tpg270086-bib-0068">2006</a>), both SD and DN plants runner under a wide range of conditions, including short and long days, and when producing fruit, which is why runner trimming is commonly practiced and necessary to promote flowering and maximize yield in fruit production (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a>, <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a>). We observed the complete range of phenotypes, from runnerless (RS = 1.0) to extreme runnering (RS = 5.0), among the diverse sample of SD and DN individuals assembled for the SDP and parents of F<sub>2</sub> families segregating for <em>PF</em> alleles and runnering phenotypes (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>; Figure <a href="#tpg270086-fig-0003" class="usa-link">3</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>).</p>
<p>The runnering phenotypes of most of the wild species ecotypes in our study were aggressive (RS &gt; 4.0), as exemplified by Frederik 9, Hinesburg, Harris Spring, and Yaquina A in Figure <a href="#tpg270086-fig-0001" class="usa-link">1</a> (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The median RS was <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-186" display="inline"><mrow><mrow><mover accent="true"><mi>y</mi><mo>∼</mo></mover><mo>=</mo><mn>4.8</mn></mrow></mrow></math></span> among <em>F. chiloensis</em> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-187" display="inline"><mrow><mrow><mover accent="true"><mi>y</mi><mo>∼</mo></mover><mo>=</mo><mn>4.8</mn></mrow></mrow></math></span> among <em>F. virginiana</em> ecotypes. The RS means for 33 of the 38 octoploid ecotypes phenotyped in our study were in the 4.0–5.0 range, and only one (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-188" display="inline"><mrow><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mo>=</mo><mn>3.0</mn></mrow></mrow></math></span>; Scott's Creek) was less than the SDP population mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-189" display="inline"><mrow><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mo>=</mo><mn>3.38</mn></mrow></mrow></math></span>).</p>
<p>The RS distribution was negatively skewed among SD and approximately normal among DN individuals in the SDP population (Figure <a href="#tpg270086-fig-0003" class="usa-link">3</a>). The RS mean was significantly greater for SD (<em>pfpf</em>) individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-190" display="inline"><mrow><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mrow><mi>S</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>3.68</mn></mrow></mrow></math></span>) than DN (<em>PF</em>_) individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-191" display="inline"><mrow><mrow><msub><mover accent="true"><mi>y</mi><mo>¯</mo></mover><mtext>DN</mtext></msub><mo>=</mo><mn>3.07</mn></mrow></mrow></math></span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-192" display="inline"><mrow><mrow><mi>p</mi><mo>≤</mo><mn>0.0001</mn></mrow></mrow></math></span>). Similarly, the RS median was greater for SD individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-193" display="inline"><mrow><mrow><msub><mover accent="true"><mi>y</mi><mo>∼</mo></mover><mtext>SD</mtext></msub><mo>=</mo><mn>3.95</mn></mrow></mrow></math></span>) than DN individuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-194" display="inline"><mrow><mrow><msub><mover accent="true"><mi>y</mi><mo>∼</mo></mover><mtext>DN</mtext></msub><mo>=</mo><mn>2.97</mn></mrow></mrow></math></span>) (Figure <a href="#tpg270086-fig-0003" class="usa-link">3</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). These differences broadly reflect the pleiotropic effect of the <em>PF</em> locus on runnering over an entire growing season in our low‐elevation nursery production (clonal propagation) environment (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a> and <a href="#tpg270086-fig-0002" class="usa-link">2</a>).</p>
<p>As shown for a broad cross‐section of octoploid genetic resources, DN (<em>PF_</em>) individuals tend to produce fewer runners than SD (<em>pfpf</em>) individuals in nursery production (Figure <a href="#tpg270086-fig-0003" class="usa-link">3</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). Of 478 SD individuals in the SDP population, 97.5% had RSs in the 2.5–5.0 range. The 12 SD individuals in the weak runnering range (1.0–2.5) include the cultivars Elsanta (RS = 1.2), Howard 17 (RS = 2.0), Senga Sengana (RS = 2.0), and Fairfax (RS = 2.0). The other eight are UC individuals developed in 2018 that were purposefully selected for reduced runnering (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>).</p>
<p>We observed a wide range of runnering phenotypes among DN individuals known to be homozygous or heterozygous for the dominant Wasatch <em>PF</em> allele, for example, 20C420P006 (<em>PFpf</em>; RS = 1.0), UCD Moxie (<em>PFpf</em>; RS = 2.0), UC Eclipse (<em>PFPF</em>; RS = 3.4), and Monterey (<em>PFpf</em>; RS = 4.0) (Figure <a href="#tpg270086-fig-0002" class="usa-link">2</a>). Of 444 individuals classified as DN in the SDP population, 80.0% had runners scores in the 2.5–5.0 range (Figure <a href="#tpg270086-fig-0003" class="usa-link">3</a>; S1). The percentage of individuals in the weak runnering range (1.0–2.5) was greater for DN (20.0%) than SD (2.5%) individuals. Weak runnering DN cultivars were uncommon (the runnering phenotypes of 30 of the 40 DN cultivars screened in our study were intermediate to strong).</p>
<p>DN cultivars carrying the Wasatch <em>PF</em> allele have occasionally been described as poor runner producers, but cannot be broadly classified as poor runnering, for example, Hossain et al. (<a href="#tpg270086-bib-0041" class="usa-link" aria-describedby="tpg270086-bib-0041">2019</a>) described Albion (<em>PFpf</em>) as a “poor runner producer” and depicted a runnerless Albion plant. Our data do not support that characterization. Albion produced an above average number of runners (RS = 3.3), runners in fruit and bare‐root plant production environments, and has been clonally propagated on a large‐scale for commercial production since 1997 (Figure <a href="#tpg270086-fig-0002" class="usa-link">2</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). Our data show that the runnering phenotypes of DN (<em>PF</em>_) individuals temporally vary and span the entire range, from runnerless to extreme (Figures <a href="#tpg270086-fig-0003" class="usa-link">3</a>, <a href="#tpg270086-fig-0008" class="usa-link">8</a>, and <a href="#tpg270086-fig-0009" class="usa-link">9</a>).</p></section><section id="tpg270086-sec-0180"><h3 class="pmc_sec_title">3.7. Accuracy of genomic selection for runnering</h3>
<p>The <em>PF</em> locus explained 22.1% of the genetic variance for runnering in the SDP population and was found to be necessary but insufficient for predicting runnering phenotypes in strawberry populations segregating for dominant and recessive <em>PF</em> alleles (Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>; Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a> and <a href="#tpg270086-fig-0002" class="usa-link">2</a>). With as much as 78% of the heritability for runnering missing (unexplained by segregation of the <em>PF</em> locus), we explored the merit of applying genomic selection for runnering by testing different combinations of training and validation populations with and without correcting for the fixed effect of the <em>PF</em> locus (Table <a href="#tpg270086-tbl-0004" class="usa-link">4</a>). We specifically sought to identify how and where genomic selection for runnering might be cost effectively applied in a strawberry breeding pipeline to identify runnerless individuals as selection candidates for seed‐propagated cultivars and near‐runnerless individuals as selection candidates for clone‐propagated cultivars. The strategy for the latter is to identify individuals near the lower limit of commercially viable clonal propagation, specifically those with RSs in the 2.0–2.5 range (Figures <a href="#tpg270086-fig-0003" class="usa-link">3</a> and <a href="#tpg270086-fig-0009" class="usa-link">9</a>).</p>
<section class="tw xbox font-sm" id="tpg270086-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Genomic predictive ability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-195" display="inline"><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub></mrow></math></span>) and prediction accuracy (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-196" display="inline"><mrow><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub><mo>/</mo><mi>h</mi></mrow></mrow></math></span>) for runner score or count within and between strawberry populations estimated by cross‐validation<a href="#tpg270086-tbl4-note-0001" class="usa-link">
<sup>a</sup>
</a> with and without correcting for the effect of the <em>PF</em> locus<a href="#tpg270086-tbl4-note-0002" class="usa-link">
<sup>b</sup>
</a>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Population<a href="#tpg270086-tbl4-note-0003" class="usa-link"><sup>c</sup></a>
</th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">PF Uncorrected</th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">PF Corrected</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Training</th>
<th align="left" rowspan="1" colspan="1">Validation</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-197_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-197" display="inline"><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-198_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-198" display="inline"><mrow><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub><mo>/</mo><mi>h</mi></mrow></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-199_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-199" display="inline"><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-200_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-200" display="inline"><mrow><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub><mo>/</mo><mi>h</mi></mrow></mrow></math></span></td></tr></table>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">SDP</td>
<td align="left" rowspan="1" colspan="1">SDP</td>
<td align="left" rowspan="1" colspan="1">0.59</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
<td align="left" rowspan="1" colspan="1">0.60</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SDP‐UC</td>
<td align="left" rowspan="1" colspan="1">SDP‐UC</td>
<td align="left" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.50</td>
<td align="left" rowspan="1" colspan="1">0.66</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FS</td>
<td align="left" rowspan="1" colspan="1">FS</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
<td align="left" rowspan="1" colspan="1">0.58</td>
<td align="left" rowspan="1" colspan="1">0.79</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SDP</td>
<td align="left" rowspan="1" colspan="1">FS</td>
<td align="left" rowspan="1" colspan="1">0.33</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">0.43</td>
<td align="left" rowspan="1" colspan="1">0.60</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SDP‐UC</td>
<td align="left" rowspan="1" colspan="1">FS</td>
<td align="left" rowspan="1" colspan="1">0.25</td>
<td align="left" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">0.39</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270086-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270086-tbl4-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Cross‐validation was done by randomly drawing 80% of the individuals within a training population to predict genomic‐estimated breeding values (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-201" display="inline"><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover></mrow></math></span>) of validation population individuals. The prediction accuracy was estimated by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-202" display="inline"><mrow><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub><mo>/</mo><mi>h</mi></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-203" display="inline"><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub></mrow></math></span> is the simple correlation between the observed phenotype (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-204" display="inline"><mrow><mi>y</mi></mrow></math></span>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-205" display="inline"><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-206" display="inline"><mrow><mrow><mi>h</mi><mo>=</mo><msqrt><msup><mi>h</mi><mn>2</mn></msup></msqrt></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-207" display="inline"><mrow><msup><mi>h</mi><mn>2</mn></msup></mrow></math></span> is the genomic‐estimated narrow‐sense heritability, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-208" display="inline"><mrow><mi>y</mi></mrow></math></span> was the either the runner count for unreplicated seed‐propagated full‐sib (FS) individuals or the estimated marginal mean for runner score for clonally propagated strawberry diversity panel (SDP) individuals over years. Cross‐validation statistics were estimated from 1000 randomly drawn samples of training population individuals with replacement. When predicting within a population (e.g., within the SDP population), <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-209" display="inline"><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover></mrow></math></span> was estimated for individuals that were not sampled in each iteration (20% of the individuals within the population). When predicting between populations, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-210" display="inline"><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover></mrow></math></span> was estimated for every individual in the validation population.</p>
</div>
<div class="fn" id="tpg270086-tbl4-note-0002">
<sup>
<sup>b</sup>
</sup><p class="display-inline">The <em>PF</em>‐associated single nucleotide polymorphism (SNP) AX‐184947290 was used as a fixed effect to correct for the effect of the <em>PF</em> locus on runner score or count.</p>
</div>
<div class="fn" id="tpg270086-tbl4-note-0003">
<sup>
<sup>c</sup>
</sup><p class="display-inline">The populations are the SDP <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-211" display="inline"><mrow><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>=</mo><mn>932</mn></mrow></mrow></math></span>), a population‐specific subset of cultivars and other elite SDP individuals developed at the University of California (UC), Davis (SDP‐UC; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-212" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>672</mn></mrow></mrow></math></span>), 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-213" display="inline"><mrow><mo>×</mo></mrow></math></span> Royal Royce and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-214" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> families (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-215" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>270</mn></mrow></mrow></math></span>), and nine FS families (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-216" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>372</mn></mrow></mrow></math></span>) developed from crosses between elite UC parents.</p>
</div>
</div></section><p>Because the SDP and FS family training populations segregated for dominant and recessive <em>PF</em> alleles, we used a <em>PF</em>‐associated SNP marker (AX‐184947290) to correct for the fixed effect of the <em>PF</em> locus (Tables <a href="#tpg270086-tbl-0001" class="usa-link">1</a> and <a href="#tpg270086-tbl-0004" class="usa-link">4</a>). The inclusion of AX‐184947290 as a fixed effect increased the accuracy of genomic prediction across the training and validation population combinations tested (Table <a href="#tpg270086-tbl-0004" class="usa-link">4</a>). The accuracy increase gained by <em>PF</em>‐correction was greater for the between population combinations (training <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-217" display="inline"><mrow><mo>→</mo></mrow></math></span> validation = SDP <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-218" display="inline"><mrow><mo>→</mo></mrow></math></span> FS and SDP‐UC <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-219" display="inline"><mrow><mo>→</mo></mrow></math></span> FS) than the within population combinations tested in our study, for example, SDP <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-220" display="inline"><mrow><mo>→</mo></mrow></math></span> SDP (Table <a href="#tpg270086-tbl-0004" class="usa-link">4</a>). As anticipated, directly correcting for the fixed effect of the <em>PF</em> locus is necessary for maximizing genomic prediction accuracy when selecting for runnering in populations segregating for dominant and recessive <em>PF</em> alleles (Table <a href="#tpg270086-tbl-0004" class="usa-link">4</a>).</p>
<p>The clonal genetic resources associated with strawberry breeding programs typically comprise permanent to semipermanent individuals, both exotic and elite, and a continually evolving sample of selected individuals spanning several generations of within FS family selection. The 932 individuals selected for the SDP originated from 554 unique pedigrees (FS families) among 489 parents (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). The phenotypic data underlying breeding program‐associated clonal genetic resources are continually updated, thereby creating ever‐increasing samples of training population data to inform parent and genomic selection (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). We used the SDP to assess whether RS observations accumulated year‐over‐year among genetically diverse individuals could be used to accurately estimate breeding values for runner count among seed‐propagated elite <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-221" display="inline"><mrow><mo>×</mo></mrow></math></span> elite FS progeny (selection candidates before clonal propagation). We discovered that breeding values for RS were predicted with outstanding accuracy within the SDP population per se (the <em>PF</em>‐corrected estimate was <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-222" display="inline"><mrow><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub><mo>/</mo><mi>h</mi><mo>=</mo><mn>0.77</mn></mrow></mrow></math></span>). The prediction accuracy was lower when the SDP was used as the training population for predicting breeding values among elite <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-223" display="inline"><mrow><mo>×</mo></mrow></math></span> elite FS progeny (SDP <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-224" display="inline"><mrow><mo>→</mo></mrow></math></span> FS; Table <a href="#tpg270086-tbl-0004" class="usa-link">4</a>). We note here that physiological and methodological differences could partly account for the slightly lower SDP to FS prediction accuracy. SDP individuals were clonally propagated and phenotyped in late summer using an ordinal scale, whereas FS individuals were seed propagated and phenotyped at the summer solstice for runner count.</p>
<p>To further explore the approach of using phenotypes of genetic resources (e.g., the SDP population) for genomic prediction of runnering breeding values in elite segregating populations, we tested a training population comprising 776 SDP individuals with UC pedigrees starting from 1969 onward (SDP‐UC), the year of origin of the first Wasatch‐derived DN cultivars, specifically Hecker, Aptos, and Brighton (Bringhurst and Voth, <a href="#tpg270086-bib-0007" class="usa-link" aria-describedby="tpg270086-bib-0007">1980</a>). The SDP‐UC subset was created by eliminating 156 pre‐1969 UC, non‐UC, and other exotic individuals from the SDP (File S1). Because accuracy depends on the closeness of genetic relationships and patterns of LD between training and validation populations (Habier et al., <a href="#tpg270086-bib-0029" class="usa-link" aria-describedby="tpg270086-bib-0029">2007</a>; Hickey et al., <a href="#tpg270086-bib-0039" class="usa-link" aria-describedby="tpg270086-bib-0039">2014</a>; A. J. Lorenz &amp; Smith, <a href="#tpg270086-bib-0050" class="usa-link" aria-describedby="tpg270086-bib-0050">2015</a>), our hypothesis was that accuracy could be increased by using a UC‐specific subset of SDP training population individuals to predict the breeding values of FS progeny developed with elite UC parents. We discovered that RS breeding values were predicted with acceptable accuracy within the SDP‐UC training population per se (SDP‐UC <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-225" display="inline"><mrow><mo>→</mo></mrow></math></span> SDP‐UC); however, to our surprise, genomic prediction accuracy was lower within the SDP‐UC than the SDP training population, for example, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-226" display="inline"><mrow><mrow><msub><mi>r</mi><mrow><mover accent="true"><mi>a</mi><mo>^</mo></mover><mo>,</mo><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow></msub><mo>/</mo><mi>h</mi></mrow></mrow></math></span> decreased from 0.77 in the SDP <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-227" display="inline"><mrow><mo>→</mo></mrow></math></span> SDP cross‐validation to 0.66 in the SDP‐UC <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-228" display="inline"><mrow><mo>→</mo></mrow></math></span> SDP‐UC cross‐validation (Table <a href="#tpg270086-tbl-0004" class="usa-link">4</a>). This might be partly explained by decreased genetic variation in the SDP‐UC population.</p>
<p>We attributed the decreased accuracy of the SDP‐UC training population to the reduction of individuals from the upper and lower tails of the RS distribution and associated decreases in allelic diversity among loci underlying extreme phenotypes eliminated by artificial selection (Figure <a href="#tpg270086-fig-0009" class="usa-link">9</a>). While adding genetically distant individuals to a training population often decreases genomic prediction accuracy, we suspect that the effect of eliminating genetically distant individuals from the training population was offset by the loss of allelic variation needed for accurate genomic prediction (Habier et al., <a href="#tpg270086-bib-0029" class="usa-link" aria-describedby="tpg270086-bib-0029">2007</a>; Hickey et al., <a href="#tpg270086-bib-0039" class="usa-link" aria-describedby="tpg270086-bib-0039">2014</a>; A. J. Lorenz &amp; Smith, <a href="#tpg270086-bib-0050" class="usa-link" aria-describedby="tpg270086-bib-0050">2015</a>). Selection against runnering phenotypes at both extremes (RS &lt; 2.0 and RS &gt; 4.0) has narrowed genetic variation for runnering among modern cultivars, including those in the SDP‐UC subset. Our analysis suggests that augmenting the training population with extreme phenotypes can increase prediction accuracy, similar to what has been proposed in similar breeding scenarios where selection has significantly changed allele frequencies and eliminated unfavorable alleles (Brandariz &amp; Bernardo, <a href="#tpg270086-bib-0003" class="usa-link" aria-describedby="tpg270086-bib-0003">2018</a>; Knapp et al., <a href="#tpg270086-bib-0045" class="usa-link" aria-describedby="tpg270086-bib-0045">2024</a>).</p>
<p>The prediction accuracy for runner count within the FS training population (FS <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-229" display="inline"><mrow><mo>→</mo></mrow></math></span> FS) was nearly identical to that observed within the SDP training population (Table <a href="#tpg270086-tbl-0004" class="usa-link">4</a>). This suggested that phenotyping a nominal number of seed‐propagated (unreplicated) FS individuals for runner count should facilitate the accurate prediction of breeding values of unphenotyped FS individuals, particularly with continual updating of the training population (Hickey et al., <a href="#tpg270086-bib-0039" class="usa-link" aria-describedby="tpg270086-bib-0039">2014</a>; Jannink et al., <a href="#tpg270086-bib-0044" class="usa-link" aria-describedby="tpg270086-bib-0044">2010</a>; A. Lorenz &amp; Nice, <a href="#tpg270086-bib-0049" class="usa-link" aria-describedby="tpg270086-bib-0049">2017</a>). The approach of counting runners among a subset of individuals within FS families, although effective for building a training population and applying genomic selection, is time consuming, cost prohibitive, and unnecessary for identifying individuals with runnerless to near‐runnerless phenotypes, primarily because rapid visual phenotyping and phenotypic selection among seed‐propagated FS individuals was found to be highly effective (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a> and <a href="#tpg270086-fig-0002" class="usa-link">2</a>).</p></section></section><section id="tpg270086-sec-0190"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Our studies were undertaken to assess the feasibility of identifying runnerless and reduced runnering selection candidates in segregating populations. Although the genetic mechanisms underlying variation for runnering were found to be complex (a quantitative black box), the broad‐sense heritability, stability, and reproducibility of runnering phenotypes were exceptional in our studies and showed that the natural genetic variation needed to develop runnerless or reduced runnering cultivars is prevalent and heritable in octoploid strawberry (Figures <a href="#tpg270086-fig-0001" class="usa-link">1</a>, <a href="#tpg270086-fig-0002" class="usa-link">2</a>, <a href="#tpg270086-fig-0003" class="usa-link">3</a> and <a href="#tpg270086-fig-0009" class="usa-link">9</a>; Table <a href="#tpg270086-tbl-0002" class="usa-link">2</a>). We recovered runnerless and reduced runnering phenotypes at moderately high frequencies and found them to be stable and reproducible despite the complex genetic architecture of runnering.</p>
<p>The pleiotropic effect of the <em>PF</em> locus on runnering was found to be highly variable, temporal, and incompletely dominant, even though the effect of the <em>PF</em> allele on flowering is highly stable, predictable, and dominant (Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270086-supinfo-0001" class="usa-link">S2</a>; Ahmadi et al. <a href="#tpg270086-bib-0001" class="usa-link" aria-describedby="tpg270086-bib-0001">1990</a>; Gaston et al. <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Hossain et al. <a href="#tpg270086-bib-0041" class="usa-link" aria-describedby="tpg270086-bib-0041">2019</a>). From a practical breeding perspective, the pleiotropic effect of the <em>PF</em> locus on runnering is inconsequential because progeny are destined to be SD (<em>pfpf</em>) or DN (<em>PF_</em>). To state this another way, when selecting for runnering within SD or DN populations per se, the outcome of selection hinges on genetic variation caused by loci other than <em>PF</em> (Table <a href="#tpg270086-tbl-0003" class="usa-link">3</a>). Those other loci underlie the quantitative variation and missing heritability for runnering uncovered in our study (Table <a href="#tpg270086-tbl-0002" class="usa-link">2</a>).</p>
<p>Our physical mapping study facilitated the integration and cross‐referencing of findings from earlier genetic mapping studies of the <em>PF</em> locus (Honjo et al., <a href="#tpg270086-bib-0040" class="usa-link" aria-describedby="tpg270086-bib-0040">2016</a>; Perrotte et al., <a href="#tpg270086-bib-0056" class="usa-link" aria-describedby="tpg270086-bib-0056">2016</a>; Sargent et al., <a href="#tpg270086-bib-0065" class="usa-link" aria-describedby="tpg270086-bib-0065">2016</a>; R. Spigler et al., <a href="#tpg270086-bib-0070" class="usa-link" aria-describedby="tpg270086-bib-0070">2008</a>, R. B. Spigler et al., <a href="#tpg270086-bib-0071" class="usa-link" aria-describedby="tpg270086-bib-0071">2010</a>; Saiga et al., <a href="#tpg270086-bib-0063" class="usa-link" aria-describedby="tpg270086-bib-0063">2022</a>). This was previously difficult because of differences in linkage group and chromosome nomenclatures; the absence, inaccessibility, or nontransferability of earlier DNA marker information; and the absence of highly contiguous octoploid reference genomes and physically anchored DNA markers (Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>, <a href="#tpg270086-bib-0036" class="usa-link" aria-describedby="tpg270086-bib-0036">2021</a>). Those difficulties were eliminated with the assembly and annotation of highly contiguous, haplotype‐phased octoploid genomes, the development of octoploid genome‐anchored 50,000 and 850,000 SNP arrays and medium‐density genotyping platforms, the discovery that a significant percentage of short DNA sequences could be unambiguously aligned to octoploid reference genomes, the development of a chromosome nomenclature aligned with the evolutionary history and organization of the four genomes, and the development of a Rosetta Stone for cross‐referencing linkage group and chromosome nomenclatures (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S3</a>; Tennessen et al., <a href="#tpg270086-bib-0073" class="usa-link" aria-describedby="tpg270086-bib-0073">2014</a>; Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>, <a href="#tpg270086-bib-0034" class="usa-link" aria-describedby="tpg270086-bib-0034">2023</a>; Session &amp; Rokhsar, <a href="#tpg270086-bib-0066" class="usa-link" aria-describedby="tpg270086-bib-0066">2023</a>).</p>
<p>When DNA sequences for previously genetically mapped <em>PF</em>‐linked simple sequence repeat (SSR) and amplification‐refractory mutation system (ARMS) markers were aligned to the UCD Royal Royce genome (Honjo et al., <a href="#tpg270086-bib-0040" class="usa-link" aria-describedby="tpg270086-bib-0040">2016</a>; Perrotte et al., <a href="#tpg270086-bib-0056" class="usa-link" aria-describedby="tpg270086-bib-0056">2016</a>; Sargent et al., <a href="#tpg270086-bib-0065" class="usa-link" aria-describedby="tpg270086-bib-0065">2016</a>; R. Spigler et al., <a href="#tpg270086-bib-0070" class="usa-link" aria-describedby="tpg270086-bib-0070">2008</a>, R. B. Spigler et al., <a href="#tpg270086-bib-0071" class="usa-link" aria-describedby="tpg270086-bib-0071">2010</a>; Saiga et al., <a href="#tpg270086-bib-0063" class="usa-link" aria-describedby="tpg270086-bib-0063">2022</a>), we discovered that they are located upstream or downstream of the physical position predicted for the <em>PF</em> locus (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A</a>). Our analyses shed light on why fine‐scale genetic mapping of the <em>PF</em> locus has been challenging. We showed that recombination has been limited and that wild donor DNA has persisted in the LD block predicted to harbor the <em>PF</em> locus (Figure <a href="#tpg270086-fig-0008" class="usa-link">8</a>). The high density and even distribution of physically anchored SNPs on the 50,000 and 850,000 arrays used in our studies enabled us to saturate chromosome 4B with DNA markers and graphically genotype the wild donor genomic segment that has persisted in modern DN descendants of Wasatch (Figure <a href="#tpg270086-fig-0006" class="usa-link">6</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S7</a>).</p>
<p>Using the DNA sequences for previously genetically mapped SSR and ARMS markers, the Rosetta Stone, and Royal Royce genome as a physical reference, we found that upward of 10 linkage group nomenclatures have been used to describe the location of DNA markers linked to <em>PF</em> on chromosome 4B (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A</a>; Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S3</a>). The linkage group identifiers used in those earlier genetic mapping studies are LGIVb‐f (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Perrotte et al., <a href="#tpg270086-bib-0056" class="usa-link" aria-describedby="tpg270086-bib-0056">2016</a>), IV‐T‐1 (Castro et al., <a href="#tpg270086-bib-0012" class="usa-link" aria-describedby="tpg270086-bib-0012">2015</a>; Honjo et al., <a href="#tpg270086-bib-0040" class="usa-link" aria-describedby="tpg270086-bib-0040">2016</a>), LG‐4A (Verma et al., <a href="#tpg270086-bib-0076" class="usa-link" aria-describedby="tpg270086-bib-0076">2017</a>), 4A (Cockerton et al., <a href="#tpg270086-bib-0014" class="usa-link" aria-describedby="tpg270086-bib-0014">2023</a>), LG4‐D (Davik et al., <a href="#tpg270086-bib-0018" class="usa-link" aria-describedby="tpg270086-bib-0018">2015</a>), LG4b (Sargent et al., <a href="#tpg270086-bib-0065" class="usa-link" aria-describedby="tpg270086-bib-0065">2016</a>), Fvb4‐4 (Edger et al., <a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>; Saiga et al., <a href="#tpg270086-bib-0063" class="usa-link" aria-describedby="tpg270086-bib-0063">2022</a>), LGIV (R. B. Spigler et al., <a href="#tpg270086-bib-0071" class="usa-link" aria-describedby="tpg270086-bib-0071">2010</a>), LG19 (R. Spigler et al., <a href="#tpg270086-bib-0070" class="usa-link" aria-describedby="tpg270086-bib-0070">2008</a>), and possibly LG28 (Weebadde et al., <a href="#tpg270086-bib-0079" class="usa-link" aria-describedby="tpg270086-bib-0079">2008</a>). Several of these linkage group nomenclatures, despite variation in acronyms and symbols, were found to have correctly identified chromosome 4, the B genome, or both (Figure <a href="#tpg270086-fig-0006" class="usa-link">6A</a>). While the discordance among linkage group nomenclatures can often be reconciled through physical mapping of DNA sequences linked to previously genetically or physically mapped loci, our analysis highlights the critical need for the community‐wide adoption of a universal chromosome nomenclature to advance genetic studies in strawberry.</p>
<p>The <em>PF</em>‐associated SNPs identified in the present study and <em>PF</em>‐associated SSR and ARMS markers identified in previous studies are imperfect predictors of <em>PF</em> genotypes because of massive haplotype diversity, suppressed recombination in the <em>PF</em>‐associated LD block, and unresolved causal variation (Figures <a href="#tpg270086-fig-0006" class="usa-link">6</a> and <a href="#tpg270086-fig-0007" class="usa-link">7</a>; Perrotte et al. <a href="#tpg270086-bib-0056" class="usa-link" aria-describedby="tpg270086-bib-0056">2016</a>; Salinas et al. <a href="#tpg270086-bib-0064" class="usa-link" aria-describedby="tpg270086-bib-0064">2017</a>; Honjo et al. <a href="#tpg270086-bib-0040" class="usa-link" aria-describedby="tpg270086-bib-0040">2016</a>; Saiga et al. <a href="#tpg270086-bib-0063" class="usa-link" aria-describedby="tpg270086-bib-0063">2022</a>). Several <em>PF</em>‐associated SNPs identified in the present study predicted the flowering habit of SDP individuals with similar accuracy (Figures <a href="#tpg270086-fig-0006" class="usa-link">6</a> and <a href="#tpg270086-fig-0007" class="usa-link">7</a>). The A allele for the AX‐184947290 SNP (found in the Wasatch haplotype) was observed in 332 out of 379 SDP individuals classified as DN (87.6%) (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S1</a>). Similarly, the AX‐184947290 GG homozygote was observed in 345 out of 416 SDP individuals classified as SD (82.9%). The prediction accuracy of AX‐184947290 was only marginally greater when non‐UC individuals were trimmed from the analysis (88.5% for DN and 86.7% for SD individuals). That trend was virtually identical for several other SNPs and SNP haplotypes in the upper LD block, for example, the 33‐SNP Wasatch haplotype (W) was observed in 84.6% of the SDP individuals classified as DN (<em>PF</em>_) in the SDP population, including 55C031P001, 65C065P601, and UC DN and summer‐plant cultivars (Supporting Information <a href="#tpg270086-supinfo-0001" class="usa-link">S6</a>). The obvious solution to the prediction problem is to identify the causal gene and mutation underlying <em>PF</em>; however, despite extensive previous research, the causal variant remains elusive.</p>
<p>Studies beyond those presented here are needed to understand how runnerless and near‐runnerless phenotypes affect yield. We hypothesized that runnering and fruit yield are negatively genetically correlated and that fruit yields can be maximized by minimizing runner proliferation. The pleiotropic effect of runnering on fruit yield, however, has not yet been commercially documented because runners are mechanically trimmed in production fields, thereby masking the negative pleiotropic effect of runner and daughter plant growth on flowering and fruit yield. One thing is certain, reduced runnering significantly decreases runner trimming costs for fruit growers. This has been substantiated by commercial testing of UCD Moxie, a reduced runnering DN cultivar that produces high yields of marketable fruit and requires substantially less runner trimming than typical DN cultivars (Figure <a href="#tpg270086-fig-0002" class="usa-link">2</a>).</p>
<p>Tenreira et al. (<a href="#tpg270086-bib-0074" class="usa-link" aria-describedby="tpg270086-bib-0074">2017</a>) showed that the runnerless phenotype of the <em>F. vesca</em> cultivar Yellow Wonder was caused by “a deletion mutation in a gene coding for a GA20‐oxidase enzyme needed for GA biosynthesis.” Caruana et al. (<a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>) induced a recessive mutation (<em>suppressor of runnerless‐1</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-230" display="inline"><mrow><mo stretchy="false">[</mo></mrow></math></span>
<em>srl‐1</em>]) in Yellow Wonder that eliminated the runnerless phenotype. The <em>srl‐1</em> phenotype was subsequently found to be caused by a point mutation (G <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-231" display="inline"><mrow><mo>→</mo></mrow></math></span> A) that created a stop codon in <em>FveRGA1</em>, <em>FveRGA1</em>, which encodes a <em>DELLA</em> protein. They showed that the mutation affects the relative expression of <em>FveRGA1</em> in leaves and concluded “that <em>FveRGA1</em> likely represses GA signaling in multiple tissues and developmental stages” and that <em>FveRGA1</em> plays an important role in “regulating axillary meristem identity in strawberry” (Caruana et al., <a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>). We originally hypothesized that the runnerless phenotype of 55C032P001 might be caused by a single gene mutation, for example, by an <em>RGA1</em> mutation; however, our results did not support that hypothesis.</p>
<p>While the pleiotropic effect of the <em>PF</em> locus on runnering was predictable and anticipated (Gaston et al., <a href="#tpg270086-bib-0028" class="usa-link" aria-describedby="tpg270086-bib-0028">2013</a>; Hossain et al., <a href="#tpg270086-bib-0041" class="usa-link" aria-describedby="tpg270086-bib-0041">2019</a>), the absence of large‐effect loci for runnering was not. The identification of single gene mutations with large effects on runnering in future studies seems plausible, particularly since <em>RGA1</em> and other genes affecting flowering and runnering have been identified and others are bound to emerge (Caruana et al., <a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>; Hytönen &amp; Kurokura, <a href="#tpg270086-bib-0043" class="usa-link" aria-describedby="tpg270086-bib-0043">2020</a>; Koskela et al., <a href="#tpg270086-bib-0046" class="usa-link" aria-describedby="tpg270086-bib-0046">2016</a>). We identified a homolog of one of the more promising candidate genes (<em>RGA1</em>) in the fringe of the <em>PF</em> LD block (<em>RGA1‐4B</em>; Fxa4Bg103085; bp 28,885,347–28,887,164; Figure <a href="#tpg270086-supinfo-0001" class="usa-link">S2</a>). While intriguing, SNPs in LD with <em>RGA1‐4B</em> were 1.59 Mb downstream of <em>PF</em> (27,294,311) and only very weakly associated with runner count. Whether natural <em>RGA1‐4B</em> allelic variants affect runnering remains unknown. This gene, however, is an interesting target for modifying runner growth characteristics by genetic engineering or gene editing (Caruana et al., <a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>).</p>
<p>Strawberry production could migrate, at least partly, toward seed‐propagated cultivars where runners are unnecessary. This would necessitate significant changes in current breeding and nursery production practices; specifically, a shift from the clonal propagation of hybrid individuals between outbred (partially inbred) parents to seed propagation of single‐cross hybrids between inbred parents, the formation and exploitation of heterotic groups, the implementation of inbred–hybrid breeding schemes, and the development of a hybrid seed production system (Feldmann, Pincot, Seymour, et al., <a href="#tpg270086-bib-0026" class="usa-link" aria-describedby="tpg270086-bib-0026">2024</a>). Seed propagation of hybrid individuals through apomixis could be even more game‐changing (Conner &amp; Ozias‐Akins, <a href="#tpg270086-bib-0015" class="usa-link" aria-describedby="tpg270086-bib-0015">2017</a>; Mahlandt et al., <a href="#tpg270086-bib-0052" class="usa-link" aria-describedby="tpg270086-bib-0052">2023</a>; Underwood &amp; Mercier, <a href="#tpg270086-bib-0075" class="usa-link" aria-describedby="tpg270086-bib-0075">2022</a>), eliminate the need for implementing resource‐intensive inbred‐hybrid breeding schemes, and facilitate less resource‐intensive within family selection schemes that have produced substantial genetic gains and can directly exploit elite genetics without inbred line development (Feldmann, Pincot, Seymour, et al., <a href="#tpg270086-bib-0026" class="usa-link" aria-describedby="tpg270086-bib-0026">2024</a>; Feldmann, Pincot, Cole, et al., <a href="#tpg270086-bib-0027" class="usa-link" aria-describedby="tpg270086-bib-0027">2024</a>).</p></section><section id="tpg270086-sec-0200"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Hillel Brukental</strong>: Conceptualization; Data curation; formal analysis; investigation; methodology; Project administration; visualization; writing—original draft; writing—review and editing. <strong>Marta L. Bjornson</strong>: Conceptualization; data curation; formal analysis; supervision; validation; visualization; writing—review and editing. <strong>Dominique D.A. Pincot</strong>: Conceptualization; Data curation; methodology; Resources; writing—review and editing. <strong>Michael A. Hardigan</strong>: Conceptualization; Data curation; formal analysis; methodology; Resources; writing—review and editing. <strong>Sadikshya Sharma</strong>: Data curation; formal analysis; writing—review and editing. <strong>Nicolas P. Jiménez</strong>: Data curation; Formal analysis; Resources; writing—review and editing. <strong>Randi A. Famula</strong>: Conceptualization; Data curation;project administration; resources; supervision; writing—review and editing. <strong>Cindy M. López</strong>: Data curation; investigation; resources; writing—review and editing. <strong>Glenn S. Cole</strong>: Conceptualization; data curation; investigation; Formal analysis; Funding acquisition; methodology; Supervision; project administration; writing—review and editing. <strong>Mitchell J. Feldmann</strong>: data curation; formal analysis; funding acquisition; investigation; methodology; project administration; supervision; validation; visualization; writing—original draft; writingreview and editing. <strong>Steven J. Knapp</strong>: Conceptualization; Data curation; formal analysis; funding acquisition; investigation; methodology; project administration; resources; supervision; visualization; writingreview and editing.</p></section><section id="tpg270086-sec-0220"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="tpg270086-sec-0230"><h2 class="pmc_sec_title">SUPPORTING INFORMATION</h2>
<p>The phenotypic and genotypic data generated in this study have been deposited at Zenodo and are freely available (<a href="https://doi.org/10.5281/zenodo.13937028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.13937028</a>).</p></section><section id="sec23"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270086-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270086-supl-0001"><div class="caption p">
<p>
<strong>Supplemental File S1</strong>. Origin years, species, pedigrees, flowering habit classifications, runner score phenotypic means, and AX‐184947290 SNP genotypes for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>932</mn></mrow></mrow></math></span> octoploid strawberry diversity panel (SDP) individuals. The SDP included 15 <em>F. chiloensis</em>, 24 <em>F. virginiana</em>, and 893 <em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> clonally propagated individuals. SDP individuals were classified as short‐day (SD = 0) or day‐neutral (DN = 1). The runner scores of SDP individuals were recorded on clonally propagated plants in Winters, CA using an ordinal scale, where 1 = runnerless, 2 = weak runnering, 3 = intermediate runnering, 4 = strong runnering, and 5 = extreme runnering. The runner score estimated marginal means (EMMs) of SDP individuals were estimated from <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mrow><mi>r</mi><mo>=</mo><mn>1.92</mn></mrow></mrow></math></span> observations, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mi>r</mi></mrow></math></span> is the harmonic mean number of observations per individual (the data were unbalanced and some individuals were observed only once). Tabs 2 and 3 present the unique crosses (554) and unique parents (489) of the 932 SDP individuals.</p>
<p>
<strong>Supplemental File S2</strong>. The physical positions of 50K and 850K Axiom array SNPs corroborated by genetic mapping. SNPs were physically anchored to the ‘Camarosa’ (FaCA1; Edger et al. (<a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>); <a href="https://phytozome-next.jgi.doe.gov/info/Fxananassa_v1_0_a1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/Fxananassa_v1_0_a1</a>) and ‘UCD Royal Royce’ (FaRR1; <a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0</a>) genomes in silico. Chromosomes were numbered using the nomenclature of Hardigan et al. (<a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>) and cross‐referenced to the nomenclature of Edger et al. (<a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>). This database includes the physical positions of SNPs identified by BLAST in the ‘Camarosa’ and ‘Royal Royce’ genomes, DNA sequences of the SNP probes, chromosome assignments and physical positions of SNPs corroborated by genetic mapping, and associated information.</p>
<p>
<strong>Supplemental File S3</strong>. A Rosetta stone for cross‐referencing linkage group and chromosome nomenclatures in octoploid strawberry. This database includes several previously published chromosome nomenclatures (Edger et al., <a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>; Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>; Tennessen et al., <a href="#tpg270086-bib-0073" class="usa-link" aria-describedby="tpg270086-bib-0073">2014</a>) and the chromosome nomenclature proposed by Session and Rokhsar (<a href="#tpg270086-bib-0066" class="usa-link" aria-describedby="tpg270086-bib-0066">2023</a>).</p>
<p>
<strong>Supplemental File S4</strong>. Pedigrees, runner count phenotypes, and AX‐184947290 SNP genotypes for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>87</mn></mrow></mrow></math></span> seed‐propagated 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>183</mn></mrow></mrow></math></span> seed‐propagated 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ F<sub>2</sub> progeny observed March 26, April 9, April 22, May 6, May 20, June 3, and July 6, 2023 in Davis, CA (the respective photoperiods on those dates were 12.3, 13.0, 13.3, 14.1, 14.3, 14.1, and 14.5 hr). Four F<sub>2</sub> families (21S950, 21S951, 21S952, and 21S953) were developed by self‐pollinating runnerless to near‐runnerless F<sub>1</sub> individuals (19A907P022, 19A907P024, 19A908P012, and 19A908P053) originating in 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ full‐sib families (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
<p>
<strong>Supplemental File S5</strong>. Pedigrees, runner count, runner score, and inflorescence count phenotypes, and AX‐184947290 SNP genotypes for 372 seed‐propagated full‐sib progeny observed May 1, June 1, and July 1, 2024 in Winters, CA (the respective photoperiods on those dates were 13.5, 14.4, and 14.5 hr). The population included nine full‐sib families developed from crosses among 13 elite UC parents (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
<p>
<strong>Supplemental File S6</strong>. Haplotypes for 51 phased SNPs among 808 SDP individuals. This database includes 1,606 0,1‐coded haplotypes among 808 SDP individuals classified as either short‐day (SD = 0) or day‐neutral (DN = 1).</p>
<p>
<strong>Supplemental File S7</strong>. Graphical genotypes for 50K Axiom array‐genotyped SNPs segregating between Mb 25.0 and 30.4 on chromosome 4B in 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ F<sub>2</sub> families. This database includes runnering phenotypes, SNP genotypes, and the ordered physical positions for 140 SNP loci among 279 individuals within four F<sub>2</sub> families (21S950, 21S951, 21S952, and 21S953). The predicted associations of SNPs with <em>PF</em> alleles were used to color code cells in the graphical genotype table: peach for <em>pf/pf</em>, pale green for <em>PF/pf</em>, and dark green for <em>PF/PF</em>.</p>
<p>
<strong>Supplemental File S8</strong>. Genotypes and haplotypes for 15 SNPs associated with the <em>PF</em> locus on chromosome 4B among selected samples of three individuals each from an 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ F<sub>2</sub> family (21S950) and two 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> families (21S952 and 21S953). The SNPs spanned bp 26,389,744 (AX‐184068183) to bp 27,597,497 (AX‐184160850). The predicted associations of SNPs with <em>PF</em> alleles were used to color code cells in the graphical genotype table: peach for <em>pf/pf</em>, pale green for <em>PF/pf</em>, and dark green for <em>PF/PF</em>.</p>
<p>
<strong>Supplemental File S9</strong>. The predicted functions of 173 annotated genes spanning bp 27,597,497 to 27,418,663 on chromosome 4B in the ‘UCD Royal Royce’ reference genome (FaRR1; <a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0</a>).</p>
<p>
<strong>Supplemental Figure S1</strong>. Manhattan plots depicting the photoperiod‐dependent effect of the <em>PF</em> locus on runner count among 270 F<sub>2</sub> progeny phenotyped between March 26 and July 6 2022 in Davis, CA. F<sub>2</sub> individuals were genotyped with an Axiom 50K SNP array. Manhattan plots are shown for genome‐wide association studies of phenotypes observed (A) March 26 (12.3 hr), (B) April 9 (13.0 hr), (C) April 22 (13.3 hr), (D) May 6 (14.1 hr) (E) May 20 (14.3 hr), (F) June 3 (14.1 hr), and (G) July 6 (14.5 hr) among 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> progeny.</p>
<p>
<strong>Supplemental Figure S2</strong>. Synteny of <em>REPRESSOR OF GA1</em> (<em>RGA1</em>) homoeologs on chromosome 4 in the A, B, C, and D genomes of octoploid strawberry (<em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em>) and A genome of woodland strawberry (<em>F. vesca</em>). The ortholog identified in <em>F. vesca</em> (<em>FveRGA1</em>) by Caruana et al. (<a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>) is shown below <em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> chromosome 4 haplotypes. Homoelogs found in the octoploid are identified by the prefix Fxa. Homology was calculated using GENESPACE (Lovell et al., <a href="#tpg270086-bib-0051" class="usa-link" aria-describedby="tpg270086-bib-0051">2022</a>). The synteny plot was drawn using JCVI (Tang et al., <a href="#tpg270086-bib-0072" class="usa-link" aria-describedby="tpg270086-bib-0072">2024</a>). The synteny plot was developed using genes annotated in the haplotype‐phased assemblies of the ‘UCD Royal Royce’ genome, a day‐neutral <em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> cultivar (<a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0.)" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0.)</a>
</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365471/bin/TPG2-18-e70086-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70086-s001.zip</a><sup> (53.9MB, zip) </sup>
</div></div></section></section><section id="tpg270086-sec-0210" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was supported by a Vaadia‐BARD Postdoctoral Fellowship awarded to Hillel Brukental by the United States–Israel Binational Agricultural Research and Development Fund (Award #FI‐ 637‐2023) and by grants awarded to Steven J. Knapp and Mitchell J. Feldmann from the United Stated Department of Agriculture (<a href="https://doi.org/10.13039/100000199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.13039/100000199</a>) National Institute of Food and Agriculture (NIFA) Specialty Crops Research Initiative (#2017‐51181B6833 and #2022‐51181‐38328‐0) and California Strawberry Commission (<a href="https://doi.org/10.13039/100006760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.13039/100006760</a>). We greatly appreciate the outstanding support of the field superintendents and staff at the UC Davis Wolfskill Experiment Orchard (Winters, CA) and UC Davis Vegetable Crops (Davis, CA).</p></section><section id="notes1"><p>


Brukental, H.
, 
Bjornson, M. L.
, 
Pincot, D. D. A.
, 
Hardigan, M. A.
, 
Sharma, S.
, 
Jiménez, N. P.
, 
Famula, R. A.
, 
Lòpez, C. M.
, 
Cole, G. S.
, 
Feldmann, M. J.
, &amp; 
Knapp, S. J.
 (2025). The <em>PERPETUAL FLOWERING</em> locus: Necessary but insufficient for genomic prediction of runnerless and other asexual reproduction phenotypes in strawberry. The Plant Genome, 18, e70086. 10.1002/tpg2.70086

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270086-note-0001"><p>Assigned to Associate Editor Awais Khan.</p></div></div></section></section><section id="tpg270086-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270086-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270086-bib-0001">
<cite>

Ahmadi, H.
, 
Bringhurst, R. S
, &amp; 
Voth, V.
 (1990). Modes of inheritance of photoperiodism in <em>Fragaria</em>
. Journal of the American Society for Horticultural Science, 115(1), 146–152. 10.21273/JASHS.115.1.146
</cite> [<a href="https://doi.org/10.21273/JASHS.115.1.146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=Modes%20of%20inheritance%20of%20photoperiodism%20in%20Fragaria&amp;author=H.%20Ahmadi&amp;author=R.%20S%20Bringhurst&amp;author=V.%20Voth&amp;volume=115&amp;issue=1&amp;publication_year=1990&amp;pages=146-152&amp;doi=10.21273/JASHS.115.1.146&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0002">
<cite>

Bates, D.
, 
Mächler, M.
, 
Bolker, B.
, &amp; 
Walker, S.
 (2015). Fitting linear mixed‐effects models using lme4. Journal of Statistical Software, 67, 1–48. 10.18637/jss.v067.i01
</cite> [<a href="https://doi.org/10.18637/jss.v067.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Fitting%20linear%20mixed%E2%80%90effects%20models%20using%20lme4&amp;author=D.%20Bates&amp;author=M.%20M%C3%A4chler&amp;author=B.%20Bolker&amp;author=S.%20Walker&amp;volume=67&amp;publication_year=2015&amp;pages=1-48&amp;doi=10.18637/jss.v067.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0003">
<cite>

Brandariz, S. P.
, &amp; 
Bernardo, R.
 (2018). Maintaining the accuracy of genomewide predictions when selection has occurred in the training population. Crop Science, 58, 1226–1231. 10.2135/cropsci2017.11.0682
</cite> [<a href="https://doi.org/10.2135/cropsci2017.11.0682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Maintaining%20the%20accuracy%20of%20genomewide%20predictions%20when%20selection%20has%20occurred%20in%20the%20training%20population&amp;author=S.%20P.%20Brandariz&amp;author=R.%20Bernardo&amp;volume=58&amp;publication_year=2018&amp;pages=1226-1231&amp;doi=10.2135/cropsci2017.11.0682&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0004">
<cite>

Bringhurst, R.
, 
Ahmadi, H.
, &amp; 
Voth, V.
 (1989). Inheritance of the day‐neutral trait in strawberries. Acta Horticulturae, 265, 35–42. 10.17660/ActaHortic.1989.265.2
</cite> [<a href="https://doi.org/10.17660/ActaHortic.1989.265.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Horticulturae&amp;title=Inheritance%20of%20the%20day%E2%80%90neutral%20trait%20in%20strawberries&amp;author=R.%20Bringhurst&amp;author=H.%20Ahmadi&amp;author=V.%20Voth&amp;volume=265&amp;publication_year=1989&amp;pages=35-42&amp;doi=10.17660/ActaHortic.1989.265.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0005">
<cite>

Bringhurst, R.
, 
Hancock, J.
, &amp; 
Voth, V.
 (1977). The beach strawberry, an important natural resource. California Agriculture, 31, 10.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=California%20Agriculture&amp;title=The%20beach%20strawberry,%20an%20important%20natural%20resource&amp;author=R.%20Bringhurst&amp;author=J.%20Hancock&amp;author=V.%20Voth&amp;volume=31&amp;publication_year=1977&amp;pages=10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0006">
<cite>

Bringhurst, R.
, &amp; 
Voth, V.
 (1976). Origin and evolutionary potentiality of the day‐neutral trait in octoploid <em>Fragaria</em>
. Genetics, 83, s10.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Origin%20and%20evolutionary%20potentiality%20of%20the%20day%E2%80%90neutral%20trait%20in%20octoploid%20Fragaria&amp;author=R.%20Bringhurst&amp;author=V.%20Voth&amp;volume=83&amp;publication_year=1976&amp;pages=s10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0007">
<cite>

Bringhurst, R.
, &amp; 
Voth, V.
 (1980). Six new strawberry varieties released. California Agriculture, 34, 12–15.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=California%20Agriculture&amp;title=Six%20new%20strawberry%20varieties%20released&amp;author=R.%20Bringhurst&amp;author=V.%20Voth&amp;volume=34&amp;publication_year=1980&amp;pages=12-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0008">
<cite>

Brown, T.
, &amp; 
Wareing, P.
 (1965). The genetical control of the everbearing habit and three other characters in varieties of <em>Fragaria vesca</em>
. Euphytica, 14, 97–112. 10.1007/BF00032819
</cite> [<a href="https://doi.org/10.1007/BF00032819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=The%20genetical%20control%20of%20the%20everbearing%20habit%20and%20three%20other%20characters%20in%20varieties%20of%20Fragaria%20vesca&amp;author=T.%20Brown&amp;author=P.%20Wareing&amp;volume=14&amp;publication_year=1965&amp;pages=97-112&amp;doi=10.1007/BF00032819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0009">
<cite>

Browning, B. L.
, 
Tian, X.
, 
Zhou, Y.
, &amp; 
Browning, S. R.
 (2021). Fast two‐stage phasing of large‐scale sequence data. American Journal of Human Genetics, 108, 1880–1890. 10.1016/j.ajhg.2021.08.005

</cite> [<a href="https://doi.org/10.1016/j.ajhg.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8551421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34478634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Fast%20two%E2%80%90stage%20phasing%20of%20large%E2%80%90scale%20sequence%20data&amp;author=B.%20L.%20Browning&amp;author=X.%20Tian&amp;author=Y.%20Zhou&amp;author=S.%20R.%20Browning&amp;volume=108&amp;publication_year=2021&amp;pages=1880-1890&amp;pmid=34478634&amp;doi=10.1016/j.ajhg.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0010">
<cite>

Burman, P.
 (1989). A comparative study of ordinary cross‐validation, v‐fold cross‐validation and the repeated learning‐testing methods. Biometrika, 76, 503–514. 10.2307/2336116
</cite> [<a href="https://doi.org/10.2307/2336116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=A%20comparative%20study%20of%20ordinary%20cross%E2%80%90validation,%20v%E2%80%90fold%20cross%E2%80%90validation%20and%20the%20repeated%20learning%E2%80%90testing%20methods&amp;author=P.%20Burman&amp;volume=76&amp;publication_year=1989&amp;pages=503-514&amp;doi=10.2307/2336116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0011">
<cite>

Caruana, J. C.
, 
Sittmann, J. W.
, 
Wang, W.
, &amp; 
Liu, Z.
 (2018). Suppressor of runnerless encodes a DELLA protein that controls runner formation for asexual reproduction in strawberry. Molecular Plant, 11, 230–233. 10.1016/j.molp.2017.11.001

</cite> [<a href="https://doi.org/10.1016/j.molp.2017.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29158170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Suppressor%20of%20runnerless%20encodes%20a%20DELLA%20protein%20that%20controls%20runner%20formation%20for%20asexual%20reproduction%20in%20strawberry&amp;author=J.%20C.%20Caruana&amp;author=J.%20W.%20Sittmann&amp;author=W.%20Wang&amp;author=Z.%20Liu&amp;volume=11&amp;publication_year=2018&amp;pages=230-233&amp;pmid=29158170&amp;doi=10.1016/j.molp.2017.11.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0012">
<cite>

Castro, P.
, 
Bushakra, J. M.
, 
Stewart, P.
, 
Weebadde, C. K.
, 
Wang, D.
, 
Hancock, J. F.
, 
Finn, C. E.
, 
Luby, J. J.
, &amp; 
Lewers, K. S.
 (2015). Genetic mapping of day‐neutrality in cultivated strawberry. Molecular Breeding, 35, 79. 10.1007/s11032-015-0250-4
</cite> [<a href="https://doi.org/10.1007/s11032-015-0250-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Genetic%20mapping%20of%20day%E2%80%90neutrality%20in%20cultivated%20strawberry&amp;author=P.%20Castro&amp;author=J.%20M.%20Bushakra&amp;author=P.%20Stewart&amp;author=C.%20K.%20Weebadde&amp;author=D.%20Wang&amp;volume=35&amp;publication_year=2015&amp;pages=79&amp;doi=10.1007/s11032-015-0250-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0013">
<cite>

Ceballos, F. C.
, 
Joshi, P. K.
, 
Clark, D. W.
, 
Ramsay, M.
, &amp; 
Wilson, J. F.
 (2018). Runs of homozygosity: Windows into population history and trait architecture. Nature Reviews Genetics, 19(4), 220–234. 10.1038/nrg.2017.109
</cite> [<a href="https://doi.org/10.1038/nrg.2017.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29335644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Runs%20of%20homozygosity:%20Windows%20into%20population%20history%20and%20trait%20architecture&amp;author=F.%20C.%20Ceballos&amp;author=P.%20K.%20Joshi&amp;author=D.%20W.%20Clark&amp;author=M.%20Ramsay&amp;author=J.%20F.%20Wilson&amp;volume=19&amp;issue=4&amp;publication_year=2018&amp;pages=220-234&amp;pmid=29335644&amp;doi=10.1038/nrg.2017.109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0014">
<cite>

Cockerton, H. M.
, 
Nellist, C. F.
, 
Hytönen, T.
, 
Litthauer, S.
, 
Hopson, K.
, 
Whitehouse, A.
, 
Sobczyk, M.
, &amp; 
Harrison, R. J.
 (2023). Epistatic modifiers influence the expression of continual flowering in strawberry. Plants, People, Planet, 5, 70–81. 10.1002/ppp3.10300
</cite> [<a href="https://doi.org/10.1002/ppp3.10300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants,%20People,%20Planet&amp;title=Epistatic%20modifiers%20influence%20the%20expression%20of%20continual%20flowering%20in%20strawberry&amp;author=H.%20M.%20Cockerton&amp;author=C.%20F.%20Nellist&amp;author=T.%20Hyt%C3%B6nen&amp;author=S.%20Litthauer&amp;author=K.%20Hopson&amp;volume=5&amp;publication_year=2023&amp;pages=70-81&amp;doi=10.1002/ppp3.10300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0015">
<cite>

Conner, J. A.
, &amp; 
Ozias‐Akins, P.
 (2017). Apomixis: Engineering the ability to harness hybrid vigor in crop plants. In Schmidt A. (Ed.), Plant germline development: Methods and protocols (pp. 17–34). Springer. 10.1007/978-1-4939-7286-9_2
</cite> [<a href="https://doi.org/10.1007/978-1-4939-7286-9_2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28936646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Plant%20germline%20development:%20Methods%20and%20protocols&amp;author=J.%20A.%20Conner&amp;author=P.%20Ozias%E2%80%90Akins&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0016">
<cite>

Darrow, G. M.
 (1929). Development of runners and runner plants in the strawberry. USDA Technical Bulletin, 122, 1–28.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=USDA%20Technical%20Bulletin&amp;title=Development%20of%20runners%20and%20runner%20plants%20in%20the%20strawberry&amp;author=G.%20M.%20Darrow&amp;volume=122&amp;publication_year=1929&amp;pages=1-28&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0017"><cite>

Darrow, G. M.
 (1966). 
<em>The strawberry: History, breeding and physiology</em>. Holt, Rinehart &amp; Winston.</cite></li>
<li id="tpg270086-bib-0018">
<cite>

Davik, J.
, 
Sargent, D. J.
, 
Brurberg, M. B.
, 
Lien, S.
, 
Kent, M.
, &amp; 
Alsheikh, M.
 (2015). A ddRAD based linkage map of the cultivated strawberry, <em>Fragaria ananassa</em>
. PLoS One, 10, e0137746. 10.1371/journal.pone.0137746

</cite> [<a href="https://doi.org/10.1371/journal.pone.0137746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4580419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26398886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20ddRAD%20based%20linkage%20map%20of%20the%20cultivated%20strawberry,%20Fragaria%20ananassa&amp;author=J.%20Davik&amp;author=D.%20J.%20Sargent&amp;author=M.%20B.%20Brurberg&amp;author=S.%20Lien&amp;author=M.%20Kent&amp;volume=10&amp;publication_year=2015&amp;pages=e0137746&amp;pmid=26398886&amp;doi=10.1371/journal.pone.0137746&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0019">
<cite>

Edger, P. P.
, 
Poorten, T. J.
, 
VanBuren, R.
, 
Hardigan, M. A.
, 
Colle, M.
, 
McKain, M. R.
, 
Smith, R. D.
, 
Teresi, S. J.
, 
Nelson, A. D.
, 
Wai, C. M. W.
, 
Alger, E. I.
, 
Bird, K. A.
, 
Yocca, A. E.
, 
Pumplin, N.
, 
Ou, S.
, 
Ben‐Zvi, G.
, 
Brodt, A.
, 
Baruch, K.
, 
Swale, T.
, &amp; 
Knapp, S. J.
 (2019). Origin and evolution of the octoploid strawberry genome. Nature Genetics, 51, 541–547. 10.1038/s41588-019-0356-4

</cite> [<a href="https://doi.org/10.1038/s41588-019-0356-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6882729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30804557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Origin%20and%20evolution%20of%20the%20octoploid%20strawberry%20genome&amp;author=P.%20P.%20Edger&amp;author=T.%20J.%20Poorten&amp;author=R.%20VanBuren&amp;author=M.%20A.%20Hardigan&amp;author=M.%20Colle&amp;volume=51&amp;publication_year=2019&amp;pages=541-547&amp;pmid=30804557&amp;doi=10.1038/s41588-019-0356-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0020">
<cite>

Edger, P. P.
, 
VanBuren, R.
, 
Colle, M.
, 
Poorten, T. J.
, 
Wai, C. M.
, 
Niederhuth, C. E.
, 
Alger, E. I.
, 
Ou, S.
, 
Acharya, C. B.
, 
Wang, J.
, 
Callow, P.
, 
McKain, M. R.
, 
Shi, J.
, 
Collier, C.
, 
Xiong, Z.
, 
Mower, J. P.
, 
Slovin, J. P.
, 
Hytönen, T.
, 
Jiang, N.
, &amp; 
Knapp, S. J.
 (2018). Single‐molecule sequencing and optical mapping yields an improved genome of woodland strawberry (<em>Fragaria vesca</em>) with chromosome‐scale contiguity. Gigascience, 7, gix124. 10.1093/gigascience/gix124

</cite> [<a href="https://doi.org/10.1093/gigascience/gix124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5801600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29253147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Single%E2%80%90molecule%20sequencing%20and%20optical%20mapping%20yields%20an%20improved%20genome%20of%20woodland%20strawberry%20(Fragaria%20vesca)%20with%20chromosome%E2%80%90scale%20contiguity&amp;author=P.%20P.%20Edger&amp;author=R.%20VanBuren&amp;author=M.%20Colle&amp;author=T.%20J.%20Poorten&amp;author=C.%20M.%20Wai&amp;volume=7&amp;publication_year=2018&amp;pages=gix124&amp;pmid=29253147&amp;doi=10.1093/gigascience/gix124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0021">
<cite>

Endelman, J. B.
 (2011). Ridge regression and other kernels for genomic selection with R package rrBLUP. The Plant Genome, 4. 10.3835/plantgenome2011.08.0024
</cite> [<a href="https://doi.org/10.3835/plantgenome2011.08.0024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Ridge%20regression%20and%20other%20kernels%20for%20genomic%20selection%20with%20R%20package%20rrBLUP&amp;author=J.%20B.%20Endelman&amp;volume=4&amp;publication_year=2011&amp;doi=10.3835/plantgenome2011.08.0024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0022">
<cite>

Endelman, J. B.
, &amp; 
Jannink, J.‐L
 (2012). Shrinkage estimation of the realized relationship matrix. G3 Genes Genomes|Genetics, 2, 1405–1413. 10.1534/g3.112.004259

</cite> [<a href="https://doi.org/10.1534/g3.112.004259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3484671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23173092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%20Genomes%7CGenetics&amp;title=Shrinkage%20estimation%20of%20the%20realized%20relationship%20matrix&amp;author=J.%20B.%20Endelman&amp;author=J.%E2%80%90L%20Jannink&amp;volume=2&amp;publication_year=2012&amp;pages=1405-1413&amp;pmid=23173092&amp;doi=10.1534/g3.112.004259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0023"><cite>

Falconer, D. S.
 (1996). 
Introduction to quantitative genetics
. Pearson Education India.</cite></li>
<li id="tpg270086-bib-0024">
<cite>

Feldmann, M. J.
, 
Piepho, H.‐P.
, 
Bridges, W. C.
, &amp; 
Knapp, S. J.
 (2021). Average semivariance yields accurate estimates of the fraction of marker‐associated genetic variance and heritability in complex trait analyses. PLoS Genetics, 17, e1009762.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8425577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34437540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Average%20semivariance%20yields%20accurate%20estimates%20of%20the%20fraction%20of%20marker%E2%80%90associated%20genetic%20variance%20and%20heritability%20in%20complex%20trait%20analyses&amp;author=M.%20J.%20Feldmann&amp;author=H.%E2%80%90P.%20Piepho&amp;author=W.%20C.%20Bridges&amp;author=S.%20J.%20Knapp&amp;volume=17&amp;publication_year=2021&amp;pages=e1009762&amp;pmid=34437540&amp;doi=10.1371/journal.pgen.1009762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0025">
<cite>

Feldmann, M. J.
, 
Piepho, H.‐P.
, &amp; 
Knapp, S. J.
 (2022). Average semivariance directly yields accurate estimates of the genomic variance in complex trait analyses. G3 Genes|Genomes|Genetics, 12, jkac080. 10.1093/g3journal/jkac080
</cite> [<a href="https://doi.org/10.1093/g3journal/jkac080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9157152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35442424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Average%20semivariance%20directly%20yields%20accurate%20estimates%20of%20the%20genomic%20variance%20in%20complex%20trait%20analyses&amp;author=M.%20J.%20Feldmann&amp;author=H.%E2%80%90P.%20Piepho&amp;author=S.%20J.%20Knapp&amp;volume=12&amp;publication_year=2022&amp;pmid=35442424&amp;doi=10.1093/g3journal/jkac080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0026">
<cite>

Feldmann, M. J.
, 
Pincot, D. D.
, 
Seymour, D. K.
, 
Famula, R. A.
, 
Jiménez, N. P.
, 
López, C. M.
, 
Cole, G. S.
, &amp; 
Knapp, S. J.
 (2024). A dominance hypothesis argument for historical genetic gains and the fixation of heterosis in octoploid strawberry. Genetics, 228(4), iyae159. 10.1093/genetics/iyae159
</cite> [<a href="https://doi.org/10.1093/genetics/iyae159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11631417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39385702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=A%20dominance%20hypothesis%20argument%20for%20historical%20genetic%20gains%20and%20the%20fixation%20of%20heterosis%20in%20octoploid%20strawberry&amp;author=M.%20J.%20Feldmann&amp;author=D.%20D.%20Pincot&amp;author=D.%20K.%20Seymour&amp;author=R.%20A.%20Famula&amp;author=N.%20P.%20Jim%C3%A9nez&amp;volume=228&amp;issue=4&amp;publication_year=2024&amp;pmid=39385702&amp;doi=10.1093/genetics/iyae159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0027">
<cite>

Feldmann, M. J.
, 
Pincot, D. D. A.
, 
Cole, G. S.
, &amp; 
Knapp, S. J.
 (2024). Genetic gains underpinning a little‐known strawberry green revolution. Nature Communications, 15, 2468. 10.1038/s41467-024-46421-6
</cite> [<a href="https://doi.org/10.1038/s41467-024-46421-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10951273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38504104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Genetic%20gains%20underpinning%20a%20little%E2%80%90known%20strawberry%20green%20revolution&amp;author=M.%20J.%20Feldmann&amp;author=D.%20D.%20A.%20Pincot&amp;author=G.%20S.%20Cole&amp;author=S.%20J.%20Knapp&amp;volume=15&amp;publication_year=2024&amp;pages=2468&amp;pmid=38504104&amp;doi=10.1038/s41467-024-46421-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0028">
<cite>

Gaston, A.
, 
Perrotte, J.
, 
Lerceteau‐Köhler, E.
, 
Rousseau‐Gueutin, M.
, 
Petit, A.
, 
Hernould, M.
, 
Rothan, C.
, &amp; 
Denoyes, B.
 (2013). PFRU, a single dominant locus regulates the balance between sexual and asexual plant reproduction in cultivated strawberry. Journal of Experimental Botany, 64, 1837–1848. 10.1093/jxb/ert047

</cite> [<a href="https://doi.org/10.1093/jxb/ert047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23554259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=PFRU,%20a%20single%20dominant%20locus%20regulates%20the%20balance%20between%20sexual%20and%20asexual%20plant%20reproduction%20in%20cultivated%20strawberry&amp;author=A.%20Gaston&amp;author=J.%20Perrotte&amp;author=E.%20Lerceteau%E2%80%90K%C3%B6hler&amp;author=M.%20Rousseau%E2%80%90Gueutin&amp;author=A.%20Petit&amp;volume=64&amp;publication_year=2013&amp;pages=1837-1848&amp;pmid=23554259&amp;doi=10.1093/jxb/ert047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0029">
<cite>

Habier, D.
, 
Fernando, R. L.
, &amp; 
Dekkers, J.
 (2007). The impact of genetic relationship information on genome‐assisted breeding values. Genetics, 177(4), 2389–2397. 10.1534/genetics.107.081190

</cite> [<a href="https://doi.org/10.1534/genetics.107.081190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2219482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18073436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20impact%20of%20genetic%20relationship%20information%20on%20genome%E2%80%90assisted%20breeding%20values&amp;author=D.%20Habier&amp;author=R.%20L.%20Fernando&amp;author=J.%20Dekkers&amp;volume=177&amp;issue=4&amp;publication_year=2007&amp;pages=2389-2397&amp;pmid=18073436&amp;doi=10.1534/genetics.107.081190&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0030">
<cite>

Hancock, J.
, 
Finn, C.
, 
Hokanson, S.
, 
Luby, J.
, 
Goulart, B.
, 
Demchak, K.
, 
Callow, P.
, 
Serce, S.
, 
Schilder, A.
, &amp; 
Hummer, K. E.
 (2001). A multistate comparison of native octoploid strawberries from North and South America. Journal of the American Society for Horticultural Science, 126(5), 579–586. 10.21273/JASHS.126.5.579
</cite> [<a href="https://doi.org/10.21273/JASHS.126.5.579" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=A%20multistate%20comparison%20of%20native%20octoploid%20strawberries%20from%20North%20and%20South%20America&amp;author=J.%20Hancock&amp;author=C.%20Finn&amp;author=S.%20Hokanson&amp;author=J.%20Luby&amp;author=B.%20Goulart&amp;volume=126&amp;issue=5&amp;publication_year=2001&amp;pages=579-586&amp;doi=10.21273/JASHS.126.5.579&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0031">
<cite>

Hancock, J. F.
, 
Weebadde, C.
, &amp; 
Serçe, S.
 (2008). Challenges faced by day‐neutral strawberry breeders in the continental climates of the Eastern United States and Canada. HortScience, 43, 1635–1636. 10.21273/HORTSCI.43.6.1635
</cite> [<a href="https://doi.org/10.21273/HORTSCI.43.6.1635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=HortScience&amp;title=Challenges%20faced%20by%20day%E2%80%90neutral%20strawberry%20breeders%20in%20the%20continental%20climates%20of%20the%20Eastern%20United%20States%20and%20Canada&amp;author=J.%20F.%20Hancock&amp;author=C.%20Weebadde&amp;author=S.%20Ser%C3%A7e&amp;volume=43&amp;publication_year=2008&amp;pages=1635-1636&amp;doi=10.21273/HORTSCI.43.6.1635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0032">
<cite>

Hancock, J. F.
 (2006). California public strawberry breeders: A perfect marriage of genetics and culture. HortScience, 41, 16. 10.21273/HORTSCI.41.1.16
</cite> [<a href="https://doi.org/10.21273/HORTSCI.41.1.16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=HortScience&amp;title=California%20public%20strawberry%20breeders:%20A%20perfect%20marriage%20of%20genetics%20and%20culture&amp;author=J.%20F.%20Hancock&amp;volume=41&amp;publication_year=2006&amp;pages=16&amp;doi=10.21273/HORTSCI.41.1.16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0033">
<cite>

Hancock, J. F.
, 
Edger, P. P.
, 
Callow, P. W.
, 
Herlache, T.
, &amp; 
Finn, C. E.
 (2018). Generating a unique germplasm base for the breeding of day‐neutral strawberry cultivars. HortScience, 53(7), 1069–1071. 10.21273/HORTSCI12840-18
</cite> [<a href="https://doi.org/10.21273/HORTSCI12840-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=HortScience&amp;title=Generating%20a%20unique%20germplasm%20base%20for%20the%20breeding%20of%20day%E2%80%90neutral%20strawberry%20cultivars&amp;author=J.%20F.%20Hancock&amp;author=P.%20P.%20Edger&amp;author=P.%20W.%20Callow&amp;author=T.%20Herlache&amp;author=C.%20E.%20Finn&amp;volume=53&amp;issue=7&amp;publication_year=2018&amp;pages=1069-1071&amp;doi=10.21273/HORTSCI12840-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0034">
<cite>

Hardigan, M. A.
, 
Feldmann, M. J.
, 
Carling, J.
, 
Zhu, A.
, 
Kilian, A.
, 
Famula, R. A.
, 
Cole, G. S.
, &amp; 
Knapp, S. J.
 (2023). A medium‐density genotyping platform for cultivated strawberry using DArTag technology. The Plant Genome, 16(4), e20399. 10.1002/tpg2.20399

</cite> [<a href="https://doi.org/10.1002/tpg2.20399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37940627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=A%20medium%E2%80%90density%20genotyping%20platform%20for%20cultivated%20strawberry%20using%20DArTag%20technology&amp;author=M.%20A.%20Hardigan&amp;author=M.%20J.%20Feldmann&amp;author=J.%20Carling&amp;author=A.%20Zhu&amp;author=A.%20Kilian&amp;volume=16&amp;issue=4&amp;publication_year=2023&amp;pages=e20399&amp;pmid=37940627&amp;doi=10.1002/tpg2.20399&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0035">
<cite>

Hardigan, M. A.
, 
Feldmann, M. J.
, 
Lorant, A.
, 
Bird, K. A.
, 
Famula, R.
, 
Acharya, C.
, 
Cole, G.
, 
Edger, P. P.
, &amp; 
Knapp, S. J.
 (2020). Genome synteny has been conserved among the octoploid progenitors of cultivated strawberry over millions of years of evolution. Frontiers in Plant Science, 10, 1789. 10.3389/fpls.2019.01789

</cite> [<a href="https://doi.org/10.3389/fpls.2019.01789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7020885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32158449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%20synteny%20has%20been%20conserved%20among%20the%20octoploid%20progenitors%20of%20cultivated%20strawberry%20over%20millions%20of%20years%20of%20evolution&amp;author=M.%20A.%20Hardigan&amp;author=M.%20J.%20Feldmann&amp;author=A.%20Lorant&amp;author=K.%20A.%20Bird&amp;author=R.%20Famula&amp;volume=10&amp;publication_year=2020&amp;pages=1789&amp;pmid=32158449&amp;doi=10.3389/fpls.2019.01789&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0036">
<cite>

Hardigan, M. A.
, 
Lorant, A.
, 
Pincot, D. D. A.
, 
Feldmann, M. J.
, 
Famula, R. A.
, 
Acharya, C. B.
, 
Lee, S.
, 
Verma, S.
, 
Whitaker, V. M.
, 
Bassil, N.
, 
Zurn, J.
, 
Cole, G. S.
, 
Bird, K.
, 
Edger, P. P.
, &amp; 
Knapp, S. J.
 (2021). Unraveling the complex hybrid ancestry and domestication history of cultivated strawberry. Molecular Biology and Evolution, 38, 2285–2305. 10.1093/molbev/msab024

</cite> [<a href="https://doi.org/10.1093/molbev/msab024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8136507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33507311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Unraveling%20the%20complex%20hybrid%20ancestry%20and%20domestication%20history%20of%20cultivated%20strawberry&amp;author=M.%20A.%20Hardigan&amp;author=A.%20Lorant&amp;author=D.%20D.%20A.%20Pincot&amp;author=M.%20J.%20Feldmann&amp;author=R.%20A.%20Famula&amp;volume=38&amp;publication_year=2021&amp;pages=2285-2305&amp;pmid=33507311&amp;doi=10.1093/molbev/msab024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0037">
<cite>

Heide, O.
, 
Stavang, J.
, &amp; 
Sønsteby, A.
 (2013). Physiology and genetics of flowering in cultivated and wild strawberries—‐A review. The Journal of Horticultural Science and Biotechnology, 88, 1–18. 10.1080/14620316.2013.11512930
</cite> [<a href="https://doi.org/10.1080/14620316.2013.11512930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Horticultural%20Science%20and%20Biotechnology&amp;title=Physiology%20and%20genetics%20of%20flowering%20in%20cultivated%20and%20wild%20strawberries%E2%80%94%E2%80%90A%20review&amp;author=O.%20Heide&amp;author=J.%20Stavang&amp;author=A.%20S%C3%B8nsteby&amp;volume=88&amp;publication_year=2013&amp;pages=1-18&amp;doi=10.1080/14620316.2013.11512930&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0038">
<cite>

Hempel, F. D.
, 
Weigel, D.
, 
Mandel, M. A.
, 
Ditta, G.
, 
Zambryski, P. C.
, 
Feldman, L. J.
, &amp; 
Yanofsky, M. F.
 (1997). Floral determination and expression of floral regulatory genes in <em>Arabidopsis</em>
. Development, 124, 3845–3853. 10.1242/dev.124.19.3845

</cite> [<a href="https://doi.org/10.1242/dev.124.19.3845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9367440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Floral%20determination%20and%20expression%20of%20floral%20regulatory%20genes%20in%20Arabidopsis&amp;author=F.%20D.%20Hempel&amp;author=D.%20Weigel&amp;author=M.%20A.%20Mandel&amp;author=G.%20Ditta&amp;author=P.%20C.%20Zambryski&amp;volume=124&amp;publication_year=1997&amp;pages=3845-3853&amp;pmid=9367440&amp;doi=10.1242/dev.124.19.3845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0039">
<cite>

Hickey, J. M.
, 
Dreisigacker, S.
, 
Crossa, J.
, 
Hearne, S.
, 
Babu, R.
, 
Prasanna, B. M.
, 
Grondona, M.
, 
Zambelli, A.
, 
Windhausen, V. S.
, 
Mathews, K.
, &amp; 
Gorjanc, G.
 (2014). Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation. Crop Science, 54, 1476–1488. 10.2135/cropsci2013.03.0195
</cite> [<a href="https://doi.org/10.2135/cropsci2013.03.0195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Evaluation%20of%20genomic%20selection%20training%20population%20designs%20and%20genotyping%20strategies%20in%20plant%20breeding%20programs%20using%20simulation&amp;author=J.%20M.%20Hickey&amp;author=S.%20Dreisigacker&amp;author=J.%20Crossa&amp;author=S.%20Hearne&amp;author=R.%20Babu&amp;volume=54&amp;publication_year=2014&amp;pages=1476-1488&amp;doi=10.2135/cropsci2013.03.0195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0040">
<cite>

Honjo, M.
, 
Nunome, T.
, 
Kataoka, S.
, 
Yano, T.
, 
Hamano, M.
, 
Yamazaki, H.
, 
Yamamoto, T.
, 
Morishita, M.
, &amp; 
Yui, S.
 (2016). Simple sequence repeat markers linked to the everbearing flowering gene in long‐day and day‐neutral cultivars of the octoploid cultivated strawberry <em>Fragaria x ananassa</em>
. Euphytica, 209, 291–303. 10.1007/s10681-015-1626-6
</cite> [<a href="https://doi.org/10.1007/s10681-015-1626-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Simple%20sequence%20repeat%20markers%20linked%20to%20the%20everbearing%20flowering%20gene%20in%20long%E2%80%90day%20and%20day%E2%80%90neutral%20cultivars%20of%20the%20octoploid%20cultivated%20strawberry%20Fragaria%20x%20ananassa&amp;author=M.%20Honjo&amp;author=T.%20Nunome&amp;author=S.%20Kataoka&amp;author=T.%20Yano&amp;author=M.%20Hamano&amp;volume=209&amp;publication_year=2016&amp;pages=291-303&amp;doi=10.1007/s10681-015-1626-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0041">
<cite>

Hossain, M. R.
, 
Natarajan, S.
, 
Kim, H.‐T.
, 
Jesse, D. M. I.
, 
Lee, C.‐G.
, 
Park, J.‐I.
, &amp; 
Nou, I.‐S.
 (2019). High density linkage map construction and QTL mapping for runner production in allo‐octoploid strawberry <em>Fragaria x ananassa</em> based on ddRAD‐seq derived SNPs. Scientific Reports, 9, 3275. 10.1038/s41598-019-39808-9

</cite> [<a href="https://doi.org/10.1038/s41598-019-39808-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6397268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30824841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=High%20density%20linkage%20map%20construction%20and%20QTL%20mapping%20for%20runner%20production%20in%20allo%E2%80%90octoploid%20strawberry%20Fragaria%20x%20ananassa%20based%20on%20ddRAD%E2%80%90seq%20derived%20SNPs&amp;author=M.%20R.%20Hossain&amp;author=S.%20Natarajan&amp;author=H.%E2%80%90T.%20Kim&amp;author=D.%20M.%20I.%20Jesse&amp;author=C.%E2%80%90G.%20Lee&amp;volume=9&amp;publication_year=2019&amp;pages=3275&amp;pmid=30824841&amp;doi=10.1038/s41598-019-39808-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0042">
<cite>

Hou, K.
, 
Bhattacharya, A.
, 
Mester, R.
, 
Burch, K. S.
, &amp; 
Pasaniuc, B.
 (2021). On powerful GWAS in admixed populations. Nature Genetics, 53, 1631–1633. 10.1038/s41588-021-00953-5

</cite> [<a href="https://doi.org/10.1038/s41588-021-00953-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8939372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34824480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=On%20powerful%20GWAS%20in%20admixed%20populations&amp;author=K.%20Hou&amp;author=A.%20Bhattacharya&amp;author=R.%20Mester&amp;author=K.%20S.%20Burch&amp;author=B.%20Pasaniuc&amp;volume=53&amp;publication_year=2021&amp;pages=1631-1633&amp;pmid=34824480&amp;doi=10.1038/s41588-021-00953-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0043">
<cite>

Hytönen, T.
, &amp; 
Kurokura, T.
 (2020). Control of flowering and runnering in strawberry. The Horticulture Journal, 89, 96–107. 10.2503/hortj.UTD-R011
</cite> [<a href="https://doi.org/10.2503/hortj.UTD-R011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Horticulture%20Journal&amp;title=Control%20of%20flowering%20and%20runnering%20in%20strawberry&amp;author=T.%20Hyt%C3%B6nen&amp;author=T.%20Kurokura&amp;volume=89&amp;publication_year=2020&amp;pages=96-107&amp;doi=10.2503/hortj.UTD-R011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0044">
<cite>

Jannink, J.‐L.
, 
Lorenz, A. J.
, &amp; 
Iwata, H.
 (2010). Genomic selection in plant breeding: From theory to practice. Briefings in Functional Genomics, 9, 166–177. 10.1093/bfgp/elq001

</cite> [<a href="https://doi.org/10.1093/bfgp/elq001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20156985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20in%20Functional%20Genomics&amp;title=Genomic%20selection%20in%20plant%20breeding:%20From%20theory%20to%20practice&amp;author=J.%E2%80%90L.%20Jannink&amp;author=A.%20J.%20Lorenz&amp;author=H.%20Iwata&amp;volume=9&amp;publication_year=2010&amp;pages=166-177&amp;pmid=20156985&amp;doi=10.1093/bfgp/elq001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0045">
<cite>

Knapp, S. J.
, 
Cole, G. S.
, 
Pincot, D. D. A.
, 
Dilla‐Ermita, C. J.
, 
Bjornson, M.
, 
Famula, R. A.
, 
Gordon, T. R.
, 
Harshman, J. M.
, 
Henry, P. M.
, &amp; 
Feldmann, M. J.
 (2024). Transgressive segregation, hopeful monsters, and phenotypic selection drove rapid genetic gains and breakthroughs in predictive breeding for quantitative resistance to <em>Macrophomina</em> in strawberry. Horticulture Research, 11, uhad289. 10.1093/hr/uhad289

</cite> [<a href="https://doi.org/10.1093/hr/uhad289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10939388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38487295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Transgressive%20segregation,%20hopeful%20monsters,%20and%20phenotypic%20selection%20drove%20rapid%20genetic%20gains%20and%20breakthroughs%20in%20predictive%20breeding%20for%20quantitative%20resistance%20to%20Macrophomina%20in%20strawberry&amp;author=S.%20J.%20Knapp&amp;author=G.%20S.%20Cole&amp;author=D.%20D.%20A.%20Pincot&amp;author=C.%20J.%20Dilla%E2%80%90Ermita&amp;author=M.%20Bjornson&amp;volume=11&amp;publication_year=2024&amp;pages=uhad289&amp;pmid=38487295&amp;doi=10.1093/hr/uhad289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0046">
<cite>

Koskela, E. A.
, 
Sønsteby, A.
, 
Flachowsky, H.
, 
Heide, O. M.
, 
Hanke, M.‐V.
, 
Elomaa, P.
, &amp; 
Hytönen, T.
 (2016). TERMINAL FLOWER1 is a breeding target for a novel everbearing trait and tailored flowering responses in cultivated strawberry (<em>Fragaria x ananassa</em> Duch.). Plant Biotechnology Journal, 14, 1852–1861. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.12545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.12545</a>

</cite> [<a href="https://doi.org/10.1111/pbi.12545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5069601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26940366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=TERMINAL%20FLOWER1%20is%20a%20breeding%20target%20for%20a%20novel%20everbearing%20trait%20and%20tailored%20flowering%20responses%20in%20cultivated%20strawberry%20(Fragaria%20x%20ananassa%20Duch.)&amp;author=E.%20A.%20Koskela&amp;author=A.%20S%C3%B8nsteby&amp;author=H.%20Flachowsky&amp;author=O.%20M.%20Heide&amp;author=M.%E2%80%90V.%20Hanke&amp;volume=14&amp;publication_year=2016&amp;pages=1852-1861&amp;pmid=26940366&amp;doi=10.1111/pbi.12545&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0047"><cite>

Lenth, R.
 (2019). 
emmeans: Estimated Marginal Means, Aka Least‐Squares Means (R package version 1.4.3.01) [Computer software]
. CRAN <a href="https://cran.r-project.org/web/packages/emmeans/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/web/packages/emmeans/index.html</a>
</cite></li>
<li id="tpg270086-bib-0048">
<cite>

Lewers, K.
, 
Castro, P.
, 
Hancock, J.
, 
Weebadde, C.
, 
Die, J.
, &amp; 
Rowland, L.
 (2019). Evidence of epistatic suppression of repeat fruiting in cultivated strawberry. BMC Plant Biology, 19, 386. 10.1186/s12870-019-1984-7

</cite> [<a href="https://doi.org/10.1186/s12870-019-1984-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6729047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31488054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Evidence%20of%20epistatic%20suppression%20of%20repeat%20fruiting%20in%20cultivated%20strawberry&amp;author=K.%20Lewers&amp;author=P.%20Castro&amp;author=J.%20Hancock&amp;author=C.%20Weebadde&amp;author=J.%20Die&amp;volume=19&amp;publication_year=2019&amp;pages=386&amp;pmid=31488054&amp;doi=10.1186/s12870-019-1984-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0049">
<cite>

Lorenz, A. J.
, &amp; 
Nice, L.
 (2017). Training population design and resource allocation for genomic selection in plant breeding. In Varshney R. K., Roorkiwal M. &amp; Sorrells M. E. (Eds.), Genomic selection for crop improvement: New molecular breeding strategies for crop improvement (pp. 7–22). Springer. 10.1007/978-3-319-63170-7_2
</cite> [<a href="https://doi.org/10.1007/978-3-319-63170-7_2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Genomic%20selection%20for%20crop%20improvement:%20New%20molecular%20breeding%20strategies%20for%20crop%20improvement&amp;author=A.%20J.%20Lorenz&amp;author=L.%20Nice&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0050">
<cite>

Lorenz, A. J.
, &amp; 
Smith, K. P.
 (2015). Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley. Crop Science, 55(6), 2657–2667. 10.2135/cropsci2014.12.0827
</cite> [<a href="https://doi.org/10.2135/cropsci2014.12.0827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Adding%20genetically%20distant%20individuals%20to%20training%20populations%20reduces%20genomic%20prediction%20accuracy%20in%20barley&amp;author=A.%20J.%20Lorenz&amp;author=K.%20P.%20Smith&amp;volume=55&amp;issue=6&amp;publication_year=2015&amp;pages=2657-2667&amp;doi=10.2135/cropsci2014.12.0827&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0051">
<cite>

Lovell, J. T.
, 
Sreedasyam, A.
, 
Schranz, M. E.
, 
Wilson, M.
, 
Carlson, J. W.
, 
Harkess, A.
, 
Emms, D.
, 
Goodstein, D. M.
, &amp; 
Schmutz, J.
 (2022). GENESPACE tracks regions of interest and gene copy number variation across multiple genomes. eLife, 11, e78526. 10.7554/eLife.78526

</cite> [<a href="https://doi.org/10.7554/eLife.78526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9462846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36083267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=GENESPACE%20tracks%20regions%20of%20interest%20and%20gene%20copy%20number%20variation%20across%20multiple%20genomes&amp;author=J.%20T.%20Lovell&amp;author=A.%20Sreedasyam&amp;author=M.%20E.%20Schranz&amp;author=M.%20Wilson&amp;author=J.%20W.%20Carlson&amp;volume=11&amp;publication_year=2022&amp;pages=e78526&amp;pmid=36083267&amp;doi=10.7554/eLife.78526&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0052">
<cite>

Mahlandt, A.
, 
Singh, D. K.
, &amp; 
Mercier, R.
 (2023). Engineering apomixis in crops. Theoretical and Applied Genetics, 136(6), 131. 10.1007/s00122-023-04357-3

</cite> [<a href="https://doi.org/10.1007/s00122-023-04357-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10195744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37199785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Engineering%20apomixis%20in%20crops&amp;author=A.%20Mahlandt&amp;author=D.%20K.%20Singh&amp;author=R.%20Mercier&amp;volume=136&amp;issue=6&amp;publication_year=2023&amp;pages=131&amp;pmid=37199785&amp;doi=10.1007/s00122-023-04357-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0053">
<cite>

Molinaro, A. M.
, 
Simon, R.
, &amp; 
Pfeiffer, R. M.
 (2005). Prediction error estimation: A comparison of resampling methods. Bioinformatics, 21, 3301–3307. 10.1093/bioinformatics/bti499

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15905277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prediction%20error%20estimation:%20A%20comparison%20of%20resampling%20methods&amp;author=A.%20M.%20Molinaro&amp;author=R.%20Simon&amp;author=R.%20M.%20Pfeiffer&amp;volume=21&amp;publication_year=2005&amp;pages=3301-3307&amp;pmid=15905277&amp;doi=10.1093/bioinformatics/bti499&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0054">
<cite>

Mouhu, K.
, 
Kurokura, T.
, 
Koskela, E. A.
, 
Albert, V. A.
, 
Elomaa, P.
, &amp; 
Hytönen, T.
 (2013). The <em>Fragaria vesca</em> homolog of SUPPRESSOR OF OVEREXPRESSION OF CONSTANS1 represses flowering and promotes vegetative growth. The Plant Cell, 25, 3296–3310. 10.1105/tpc.113.115055

</cite> [<a href="https://doi.org/10.1105/tpc.113.115055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3809533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24038650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=The%20Fragaria%20vesca%20homolog%20of%20SUPPRESSOR%20OF%20OVEREXPRESSION%20OF%20CONSTANS1%20represses%20flowering%20and%20promotes%20vegetative%20growth&amp;author=K.%20Mouhu&amp;author=T.%20Kurokura&amp;author=E.%20A.%20Koskela&amp;author=V.%20A.%20Albert&amp;author=P.%20Elomaa&amp;volume=25&amp;publication_year=2013&amp;pages=3296-3310&amp;pmid=24038650&amp;doi=10.1105/tpc.113.115055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0055">
<cite>

Paradis, E.
, 
Claude, J.
, &amp; 
Strimmer, K.
 (2004). APE: Analyses of phylogenetics and evolution in R language. Bioinformatics, 20, 289–290. 10.1093/bioinformatics/btg412

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btg412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14734327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=APE:%20Analyses%20of%20phylogenetics%20and%20evolution%20in%20R%20language&amp;author=E.%20Paradis&amp;author=J.%20Claude&amp;author=K.%20Strimmer&amp;volume=20&amp;publication_year=2004&amp;pages=289-290&amp;pmid=14734327&amp;doi=10.1093/bioinformatics/btg412&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0056">
<cite>

Perrotte, J.
, 
Gaston, A.
, 
Potier, A.
, 
Petit, A.
, 
Rothan, C.
, &amp; 
Denoyes, B.
 (2016). Narrowing down the single homoeologous FaPFRU locus controlling flowering in cultivated octoploid strawberry using a selective mapping strategy. Plant Biotechnology Journal, 14, 2176–2189. 10.1111/pbi.12574

</cite> [<a href="https://doi.org/10.1111/pbi.12574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5095798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27168086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Narrowing%20down%20the%20single%20homoeologous%20FaPFRU%20locus%20controlling%20flowering%20in%20cultivated%20octoploid%20strawberry%20using%20a%20selective%20mapping%20strategy&amp;author=J.%20Perrotte&amp;author=A.%20Gaston&amp;author=A.%20Potier&amp;author=A.%20Petit&amp;author=C.%20Rothan&amp;volume=14&amp;publication_year=2016&amp;pages=2176-2189&amp;pmid=27168086&amp;doi=10.1111/pbi.12574&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0057">
<cite>

Pincot, D. D. A.
, 
Feldmann, M. J.
, 
Hardigan, M. A.
, 
Vachev, M. V.
, 
Henry, P. M.
, 
Gordon, T. R.
, 
Bjornson, M.
, 
Rodriguez, A.
, 
Cobo, N.
, 
Famula, R. A.
, 
Cole, G. S.
, 
Coaker, G. L.
, &amp; 
Knapp, S. J.
 (2022). Novel <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-232" display="inline"><mrow><mrow><mi>F</mi><mi>u</mi><mi>s</mi><mi>a</mi><mi>r</mi><mi>i</mi><mi>u</mi><mi>m</mi></mrow></mrow></math></span> wilt resistance genes uncovered in natural and cultivated strawberry populations are found on three non‐homoeologous chromosomes. Theoretical and Applied Genetics, 135(6), 2121–2145.
</cite> [<a href="https://doi.org/10.1007/s00122-022-04102-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9205853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35583656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;author=D.%20D.%20A.%20Pincot&amp;author=M.%20J.%20Feldmann&amp;author=M.%20A.%20Hardigan&amp;author=M.%20V.%20Vachev&amp;author=P.%20M.%20Henry&amp;volume=135&amp;issue=6&amp;publication_year=2022&amp;pages=2121-2145&amp;pmid=35583656&amp;doi=10.1007/s00122-022-04102-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0058">
<cite>

Pincot, D. D. A.
, 
Hardigan, M. A.
, 
Cole, G. S.
, 
Famula, R. A.
, 
Henry, P. M.
, 
Gordon, T. R.
, &amp; 
Knapp, S. J.
 (2020). Accuracy of genomic selection and long‐term genetic gain for resistance to verticillium wilt in strawberry. The Plant Genome, 13, e20054.
</cite> [<a href="https://doi.org/10.1002/tpg2.20054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33217217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Accuracy%20of%20genomic%20selection%20and%20long%E2%80%90term%20genetic%20gain%20for%20resistance%20to%20verticillium%20wilt%20in%20strawberry&amp;author=D.%20D.%20A.%20Pincot&amp;author=M.%20A.%20Hardigan&amp;author=G.%20S.%20Cole&amp;author=R.%20A.%20Famula&amp;author=P.%20M.%20Henry&amp;volume=13&amp;publication_year=2020&amp;pages=e20054&amp;pmid=33217217&amp;doi=10.1002/tpg2.20054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0059">
<cite>

Pincot, D. D. A.
, 
Ledda, M.
, 
Feldmann, M. J.
, 
Hardigan, M. A.
, 
Poorten, T. J.
, 
Runcie, D. E.
, 
Heffelfinger, C.
, 
Dellaporta, S. L.
, 
Cole, G. S.
, &amp; 
Knapp, S. J.
 (2021). Social network analysis of the genealogy of strawberry: Retracing the wild roots of heirloom and modern cultivars. G3 Genes|Genomes|Genetics, 11, jkab015. 10.1093/g3journal/jkab015
</cite> [<a href="https://doi.org/10.1093/g3journal/jkab015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8022721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33772307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Social%20network%20analysis%20of%20the%20genealogy%20of%20strawberry:%20Retracing%20the%20wild%20roots%20of%20heirloom%20and%20modern%20cultivars&amp;author=D.%20D.%20A.%20Pincot&amp;author=M.%20Ledda&amp;author=M.%20J.%20Feldmann&amp;author=M.%20A.%20Hardigan&amp;author=T.%20J.%20Poorten&amp;volume=11&amp;publication_year=2021&amp;pmid=33772307&amp;doi=10.1093/g3journal/jkab015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0060">
<cite>

Poland, J.
, &amp; 
Rutkoski, J.
 (2016). Advances and challenges in genomic selection for disease resistance. Annual Review of Phytopathology, 54, 79–98. 10.1146/annurev-phyto-080615-100056
</cite> [<a href="https://doi.org/10.1146/annurev-phyto-080615-100056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27491433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Phytopathology&amp;title=Advances%20and%20challenges%20in%20genomic%20selection%20for%20disease%20resistance&amp;author=J.%20Poland&amp;author=J.%20Rutkoski&amp;volume=54&amp;publication_year=2016&amp;pages=79-98&amp;pmid=27491433&amp;doi=10.1146/annurev-phyto-080615-100056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0061"><cite>
R Core Team
. (2013). 
R: A language and environment for statistical computing
. R Foundation for Statistical Computing. <a href="http://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.R‐project.org/</a>
</cite></li>
<li id="tpg270086-bib-0062"><cite>

Revelle, W.
 (2021). psych: Procedures for Psychological, Psychometric, and Personality Research (R package version 2.1.3) [Computer software]. CRAN. <a href="https://CRAN.R-project.org/package=psych" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=psych</a>
</cite></li>
<li id="tpg270086-bib-0063">
<cite>

Saiga, S.
, 
Tada, M.
, 
Segawa, T.
, 
Sugihara, Y.
, 
Nishikawa, M.
, 
Makita, N.
, 
Sakamoto, M.
, 
Tanaka, K.
, 
Wada, T.
, &amp; 
Takagi, H.
 (2022). NGS‐based genome wide association study helps to develop co‐dominant marker for the physical map‐based locus of PFRU controlling flowering in cultivated octoploid strawberry. Euphytica, 219, 6. 10.1007/s10681-022-03132-7
</cite> [<a href="https://doi.org/10.1007/s10681-022-03132-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=NGS%E2%80%90based%20genome%20wide%20association%20study%20helps%20to%20develop%20co%E2%80%90dominant%20marker%20for%20the%20physical%20map%E2%80%90based%20locus%20of%20PFRU%20controlling%20flowering%20in%20cultivated%20octoploid%20strawberry&amp;author=S.%20Saiga&amp;author=M.%20Tada&amp;author=T.%20Segawa&amp;author=Y.%20Sugihara&amp;author=M.%20Nishikawa&amp;volume=219&amp;publication_year=2022&amp;pages=6&amp;doi=10.1007/s10681-022-03132-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0064">
<cite>

Salinas, N. R.
, 
Zurn, J. D.
, 
Mathey, M.
, 
Mookerjee, S.
, 
Denoyes, B.
, 
Perrotte, J.
, 
Potier, A.
, 
Finn, C. E.
, 
Hancock, J. F.
, 
Stewart, P.
, &amp; 
Bassil, N. V.
 (2017). Validation of molecular markers associated with perpetual flowering in octoploid <em>Fragaria</em> germplasm. Molecular Breeding, 37, 70. 10.1007/s11032-017-0672-2
</cite> [<a href="https://doi.org/10.1007/s11032-017-0672-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Validation%20of%20molecular%20markers%20associated%20with%20perpetual%20flowering%20in%20octoploid%20Fragaria%20germplasm&amp;author=N.%20R.%20Salinas&amp;author=J.%20D.%20Zurn&amp;author=M.%20Mathey&amp;author=S.%20Mookerjee&amp;author=B.%20Denoyes&amp;volume=37&amp;publication_year=2017&amp;pages=70&amp;doi=10.1007/s11032-017-0672-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0065">
<cite>

Sargent, D.
, 
Yang, Y.
, 
Šurbanovski, N.
, 
Bianco, L.
, 
Buti, M.
, 
Velasco, R.
, 
Giongo, L.
, &amp; 
Davis, T.
 (2016). HaploSNP affinities and linkage map positions illuminate subgenome composition in the octoploid, cultivated strawberry (<em>Fragaria x ananassa</em>). Plant Science, 242, 140–150. 10.1016/j.plantsci.2015.07.004

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2015.07.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26566832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=HaploSNP%20affinities%20and%20linkage%20map%20positions%20illuminate%20subgenome%20composition%20in%20the%20octoploid,%20cultivated%20strawberry%20(Fragaria%20x%20ananassa)&amp;author=D.%20Sargent&amp;author=Y.%20Yang&amp;author=N.%20%C5%A0urbanovski&amp;author=L.%20Bianco&amp;author=M.%20Buti&amp;volume=242&amp;publication_year=2016&amp;pages=140-150&amp;pmid=26566832&amp;doi=10.1016/j.plantsci.2015.07.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0066">
<cite>

Session, A. M.
, &amp; 
Rokhsar, D. S.
 (2023). Transposon signatures of allopolyploid genome evolution. Nature Communications, 14, 3180. 10.1038/s41467-023-38560-z
</cite> [<a href="https://doi.org/10.1038/s41467-023-38560-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10235133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37263993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Transposon%20signatures%20of%20allopolyploid%20genome%20evolution&amp;author=A.%20M.%20Session&amp;author=D.%20S.%20Rokhsar&amp;volume=14&amp;publication_year=2023&amp;pages=3180&amp;pmid=37263993&amp;doi=10.1038/s41467-023-38560-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0067">
<cite>

Shulaev, V.
, 
Sargent, D. J.
, 
Crowhurst, R. N.
, 
Mockler, T. C.
, 
Folkerts, O.
, 
Delcher, A. L.
, 
Jaiswal, P.
, 
Mockaitis, K.
, 
Liston, A.
, 
Mane, S. P.
, 
Burns, P.
, 
Davis, T. M.
, 
Slovin, J. P.
, 
Bassil, N.
, 
Hellens, R. P.
, 
Evans, C.
, 
Harkins, T.
, 
Kodira, C.
, &amp; 
Folta, K. M.
 (2011). The genome of woodland strawberry (<em>Fragaria vesca</em>). Nature Genetics, 43, 109–116. 10.1038/ng.740

</cite> [<a href="https://doi.org/10.1038/ng.740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3326587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21186353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=The%20genome%20of%20woodland%20strawberry%20(Fragaria%20vesca)&amp;author=V.%20Shulaev&amp;author=D.%20J.%20Sargent&amp;author=R.%20N.%20Crowhurst&amp;author=T.%20C.%20Mockler&amp;author=O.%20Folkerts&amp;volume=43&amp;publication_year=2011&amp;pages=109-116&amp;pmid=21186353&amp;doi=10.1038/ng.740&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0068">
<cite>

Sønsteby, A.
, &amp; 
Heide, O. M.
 (2006). Dormancy relations and flowering of the strawberry cultivars Korona and Elsanta as influenced by photoperiod and temperature. Scientia Horticulturae, 110, 57–67. 10.1016/j.scienta.2006.06.012
</cite> [<a href="https://doi.org/10.1016/j.scienta.2006.06.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientia%20Horticulturae&amp;title=Dormancy%20relations%20and%20flowering%20of%20the%20strawberry%20cultivars%20Korona%20and%20Elsanta%20as%20influenced%20by%20photoperiod%20and%20temperature&amp;author=A.%20S%C3%B8nsteby&amp;author=O.%20M.%20Heide&amp;volume=110&amp;publication_year=2006&amp;pages=57-67&amp;doi=10.1016/j.scienta.2006.06.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0069">
<cite>

Sønsteby, A.
, 
Woznicki, T. L.
, &amp; 
Heide, O. M.
 (2021). Effects of runner removal and partial defoliation on the growth and yield performance of ‘Favori’ everbearing strawberry plants. Horticulturae, 7, 215. 10.3390/horticulturae7080215
</cite> [<a href="https://doi.org/10.3390/horticulturae7080215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulturae&amp;title=Effects%20of%20runner%20removal%20and%20partial%20defoliation%20on%20the%20growth%20and%20yield%20performance%20of%20%E2%80%98Favori%E2%80%99%20everbearing%20strawberry%20plants&amp;author=A.%20S%C3%B8nsteby&amp;author=T.%20L.%20Woznicki&amp;author=O.%20M.%20Heide&amp;volume=7&amp;publication_year=2021&amp;pages=215&amp;doi=10.3390/horticulturae7080215&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0070">
<cite>

Spigler, R. B.
, 
Lewers, K.
, 
Main, D.
, &amp; 
Ashman, T.
 (2008). Genetic mapping of sex determination in a wild strawberry, <em>Fragaria virginiana</em>, reveals earliest form of sex chromosome. Heredity, 101(6), 507–517. 10.1038/hdy.2008.100

</cite> [<a href="https://doi.org/10.1038/hdy.2008.100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18797475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genetic%20mapping%20of%20sex%20determination%20in%20a%20wild%20strawberry,%20Fragaria%20virginiana,%20reveals%20earliest%20form%20of%20sex%20chromosome&amp;author=R.%20B.%20Spigler&amp;author=K.%20Lewers&amp;author=D.%20Main&amp;author=T.%20Ashman&amp;volume=101&amp;issue=6&amp;publication_year=2008&amp;pages=507-517&amp;pmid=18797475&amp;doi=10.1038/hdy.2008.100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0071">
<cite>

Spigler, R. B.
, 
Lewers, K. S.
, 
Johnson, A. L.
, &amp; 
Ashman, T.‐L.
 (2010). Comparative mapping reveals autosomal origin of sex chromosome in octoploid <em>Fragaria virginiana</em>
. Journal of Heredity, 101(suppl_1), S107–S117. 10.1093/jhered/esq001

</cite> [<a href="https://doi.org/10.1093/jhered/esq001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20421320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=Comparative%20mapping%20reveals%20autosomal%20origin%20of%20sex%20chromosome%20in%20octoploid%20Fragaria%20virginiana&amp;author=R.%20B.%20Spigler&amp;author=K.%20S.%20Lewers&amp;author=A.%20L.%20Johnson&amp;author=T.%E2%80%90L.%20Ashman&amp;volume=101&amp;issue=1&amp;publication_year=2010&amp;pages=S107-S117&amp;pmid=20421320&amp;doi=10.1093/jhered/esq001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0072">
<cite>

Tang, H.
, 
Krishnakumar, V.
, 
Zeng, X.
, 
Xu, Z.
, 
Taranto, A.
, 
Lomas, J. S.
, 
Zhang, Y.
, 
Huang, Y.
, 
Wang, Y.
, 
Yim, W. C.
, 
Zhang, J.
, &amp; 
Zhang, X.
 (2024). JCVI: A versatile toolkit for comparative genomics analysis. iMeta, 3, e211. 10.1002/imt2.211

</cite> [<a href="https://doi.org/10.1002/imt2.211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11316928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39135687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iMeta&amp;title=JCVI:%20A%20versatile%20toolkit%20for%20comparative%20genomics%20analysis&amp;author=H.%20Tang&amp;author=V.%20Krishnakumar&amp;author=X.%20Zeng&amp;author=Z.%20Xu&amp;author=A.%20Taranto&amp;volume=3&amp;publication_year=2024&amp;pages=e211&amp;pmid=39135687&amp;doi=10.1002/imt2.211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0073">
<cite>

Tennessen, J. A.
, 
Govindarajulu, R.
, 
Ashman, T.‐L.
, &amp; 
Liston, A.
 (2014). Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps. Genome Biology and Evolution, 6, 3295–3313. 10.1093/gbe/evu261

</cite> [<a href="https://doi.org/10.1093/gbe/evu261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4986458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25477420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=Evolutionary%20origins%20and%20dynamics%20of%20octoploid%20strawberry%20subgenomes%20revealed%20by%20dense%20targeted%20capture%20linkage%20maps&amp;author=J.%20A.%20Tennessen&amp;author=R.%20Govindarajulu&amp;author=T.%E2%80%90L.%20Ashman&amp;author=A.%20Liston&amp;volume=6&amp;publication_year=2014&amp;pages=3295-3313&amp;pmid=25477420&amp;doi=10.1093/gbe/evu261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0074">
<cite>

Tenreira, T.
, 
Lange, M. J. P.
, 
Lange, T.
, 
Bres, C.
, 
Labadie, M.
, 
Monfort, A.
, 
Hernould, M.
, 
Rothan, C.
, &amp; 
Denoyes, B.
 (2017). A specific gibberellin 20‐oxidase dictates the flowering‐runnering decision in diploid strawberry. The Plant Cell, 29, 2168–2182. 10.1105/tpc.16.00949

</cite> [<a href="https://doi.org/10.1105/tpc.16.00949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5635972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28874507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=A%20specific%20gibberellin%2020%E2%80%90oxidase%20dictates%20the%20flowering%E2%80%90runnering%20decision%20in%20diploid%20strawberry&amp;author=T.%20Tenreira&amp;author=M.%20J.%20P.%20Lange&amp;author=T.%20Lange&amp;author=C.%20Bres&amp;author=M.%20Labadie&amp;volume=29&amp;publication_year=2017&amp;pages=2168-2182&amp;pmid=28874507&amp;doi=10.1105/tpc.16.00949&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0075">
<cite>

Underwood, C. J.
, &amp; 
Mercier, R.
 (2022). Engineering apomixis: Clonal seeds approaching the fields. Annual Review of Plant Biology, 73(1), 201–225. 10.1146/annurev-arplant-102720-013958
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-102720-013958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35138881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Engineering%20apomixis:%20Clonal%20seeds%20approaching%20the%20fields&amp;author=C.%20J.%20Underwood&amp;author=R.%20Mercier&amp;volume=73&amp;issue=1&amp;publication_year=2022&amp;pages=201-225&amp;pmid=35138881&amp;doi=10.1146/annurev-arplant-102720-013958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0076">
<cite>

Verma, S.
, 
Zurn, J. D.
, 
Salinas, N.
, 
Mathey, M. M.
, 
Denoyes, B.
, 
Hancock, J. F.
, 
Finn, C. E.
, 
Bassil, N. V.
, &amp; 
Whitaker, V. M.
 (2017). Clarifying sub‐genomic positions of QTLs for flowering habit and fruit quality in U.S. strawberry (<em>Fragaria x ananassa</em>) breeding populations using pedigree‐based QTL analysis. Horticulture Research, 4, 17062. 10.1038/hortres.2017.62

</cite> [<a href="https://doi.org/10.1038/hortres.2017.62" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5676184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29138689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Clarifying%20sub%E2%80%90genomic%20positions%20of%20QTLs%20for%20flowering%20habit%20and%20fruit%20quality%20in%20U.S.%20strawberry%20(Fragaria%20x%20ananassa)%20breeding%20populations%20using%20pedigree%E2%80%90based%20QTL%20analysis&amp;author=S.%20Verma&amp;author=J.%20D.%20Zurn&amp;author=N.%20Salinas&amp;author=M.%20M.%20Mathey&amp;author=B.%20Denoyes&amp;volume=4&amp;publication_year=2017&amp;pages=17062&amp;pmid=29138689&amp;doi=10.1038/hortres.2017.62&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0077">
<cite>

Voth, V.
, &amp; 
Bringhurst, R. S.
 (1990). Culture and physiological manipulation of California strawberries. HortScience, 25, 889–892. 10.21273/HORTSCI.25.8.889
</cite> [<a href="https://doi.org/10.21273/HORTSCI.25.8.889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=HortScience&amp;title=Culture%20and%20physiological%20manipulation%20of%20California%20strawberries&amp;author=V.%20Voth&amp;author=R.%20S.%20Bringhurst&amp;volume=25&amp;publication_year=1990&amp;pages=889-892&amp;doi=10.21273/HORTSCI.25.8.889&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0078">
<cite>

Walsh, B.
 (2001). Quantitative genetics in the age of genomics. Theoretical Population Biology, 59, 175–184. 10.1006/tpbi.2001.1512

</cite> [<a href="https://doi.org/10.1006/tpbi.2001.1512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11444958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20Population%20Biology&amp;title=Quantitative%20genetics%20in%20the%20age%20of%20genomics&amp;author=B.%20Walsh&amp;volume=59&amp;publication_year=2001&amp;pages=175-184&amp;pmid=11444958&amp;doi=10.1006/tpbi.2001.1512&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0079">
<cite>

Weebadde, C.
, 
Wang, D.
, 
Finn, C.
, 
Lewers, K.
, 
Luby, J.
, 
Bushakra, J.
, 
Sjulin, T.
, &amp; 
Hancock, J.
 (2008). Using a linkage mapping approach to identify QTL for day‐neutrality in the octoploid strawberry. Plant Breeding, 127, 94–101. 10.1111/j.1439-0523.2007.01430.x
</cite> [<a href="https://doi.org/10.1111/j.1439-0523.2007.01430.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Using%20a%20linkage%20mapping%20approach%20to%20identify%20QTL%20for%20day%E2%80%90neutrality%20in%20the%20octoploid%20strawberry&amp;author=C.%20Weebadde&amp;author=D.%20Wang&amp;author=C.%20Finn&amp;author=K.%20Lewers&amp;author=J.%20Luby&amp;volume=127&amp;publication_year=2008&amp;pages=94-101&amp;doi=10.1111/j.1439-0523.2007.01430.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0080">
<cite>

Wright, S. I.
, &amp; 
Andolfatto, P.
 (2008). The impact of natural selection on the genome: Emerging patterns in <em>Drosophila</em> and <em>Arabidopsis</em>
. Annual Review of Ecology, Evolution, and Systematics, 39, 193–213. 10.1146/annurev.ecolsys.39.110707.173342
</cite> [<a href="https://doi.org/10.1146/annurev.ecolsys.39.110707.173342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Ecology,%20Evolution,%20and%20Systematics&amp;title=The%20impact%20of%20natural%20selection%20on%20the%20genome:%20Emerging%20patterns%20in%20Drosophila%20and%20Arabidopsis&amp;author=S.%20I.%20Wright&amp;author=P.%20Andolfatto&amp;volume=39&amp;publication_year=2008&amp;pages=193-213&amp;doi=10.1146/annurev.ecolsys.39.110707.173342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0081">
<cite>

Yu, G.
, 
Smith, D. K.
, 
Zhu, H.
, 
Guan, Y.
, &amp; 
Lam, T. T.‐Y.
 (2017). ggtree: An R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution, 8, 28–36. 10.1111/2041-210X.12628
</cite> [<a href="https://doi.org/10.1111/2041-210X.12628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Ecology%20and%20Evolution&amp;title=ggtree:%20An%20R%20package%20for%20visualization%20and%20annotation%20of%20phylogenetic%20trees%20with%20their%20covariates%20and%20other%20associated%20data&amp;author=G.%20Yu&amp;author=D.%20K.%20Smith&amp;author=H.%20Zhu&amp;author=Y.%20Guan&amp;author=T.%20T.%E2%80%90Y.%20Lam&amp;volume=8&amp;publication_year=2017&amp;pages=28-36&amp;doi=10.1111/2041-210X.12628&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270086-bib-0082">
<cite>

Zhou, X.
, &amp; 
Stephens, M.
 (2012). Genome‐wide efficient mixed‐model analysis for association studies. Nature Genetics, 44, 821–824.
</cite> [<a href="https://doi.org/10.1038/ng.2310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3386377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22706312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Genome%E2%80%90wide%20efficient%20mixed%E2%80%90model%20analysis%20for%20association%20studies&amp;author=X.%20Zhou&amp;author=M.%20Stephens&amp;volume=44&amp;publication_year=2012&amp;pages=821-824&amp;pmid=22706312&amp;doi=10.1038/ng.2310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Supplemental File S1</strong>. Origin years, species, pedigrees, flowering habit classifications, runner score phenotypic means, and AX‐184947290 SNP genotypes for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>932</mn></mrow></mrow></math></span> octoploid strawberry diversity panel (SDP) individuals. The SDP included 15 <em>F. chiloensis</em>, 24 <em>F. virginiana</em>, and 893 <em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> clonally propagated individuals. SDP individuals were classified as short‐day (SD = 0) or day‐neutral (DN = 1). The runner scores of SDP individuals were recorded on clonally propagated plants in Winters, CA using an ordinal scale, where 1 = runnerless, 2 = weak runnering, 3 = intermediate runnering, 4 = strong runnering, and 5 = extreme runnering. The runner score estimated marginal means (EMMs) of SDP individuals were estimated from <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mrow><mi>r</mi><mo>=</mo><mn>1.92</mn></mrow></mrow></math></span> observations, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mi>r</mi></mrow></math></span> is the harmonic mean number of observations per individual (the data were unbalanced and some individuals were observed only once). Tabs 2 and 3 present the unique crosses (554) and unique parents (489) of the 932 SDP individuals.</p>
<p>
<strong>Supplemental File S2</strong>. The physical positions of 50K and 850K Axiom array SNPs corroborated by genetic mapping. SNPs were physically anchored to the ‘Camarosa’ (FaCA1; Edger et al. (<a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>); <a href="https://phytozome-next.jgi.doe.gov/info/Fxananassa_v1_0_a1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/Fxananassa_v1_0_a1</a>) and ‘UCD Royal Royce’ (FaRR1; <a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0</a>) genomes in silico. Chromosomes were numbered using the nomenclature of Hardigan et al. (<a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>) and cross‐referenced to the nomenclature of Edger et al. (<a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>). This database includes the physical positions of SNPs identified by BLAST in the ‘Camarosa’ and ‘Royal Royce’ genomes, DNA sequences of the SNP probes, chromosome assignments and physical positions of SNPs corroborated by genetic mapping, and associated information.</p>
<p>
<strong>Supplemental File S3</strong>. A Rosetta stone for cross‐referencing linkage group and chromosome nomenclatures in octoploid strawberry. This database includes several previously published chromosome nomenclatures (Edger et al., <a href="#tpg270086-bib-0019" class="usa-link" aria-describedby="tpg270086-bib-0019">2019</a>; Hardigan et al., <a href="#tpg270086-bib-0035" class="usa-link" aria-describedby="tpg270086-bib-0035">2020</a>; Tennessen et al., <a href="#tpg270086-bib-0073" class="usa-link" aria-describedby="tpg270086-bib-0073">2014</a>) and the chromosome nomenclature proposed by Session and Rokhsar (<a href="#tpg270086-bib-0066" class="usa-link" aria-describedby="tpg270086-bib-0066">2023</a>).</p>
<p>
<strong>Supplemental File S4</strong>. Pedigrees, runner count phenotypes, and AX‐184947290 SNP genotypes for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>87</mn></mrow></mrow></math></span> seed‐propagated 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mrow><mi>n</mi><mo>=</mo><mn>183</mn></mrow></mrow></math></span> seed‐propagated 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ F<sub>2</sub> progeny observed March 26, April 9, April 22, May 6, May 20, June 3, and July 6, 2023 in Davis, CA (the respective photoperiods on those dates were 12.3, 13.0, 13.3, 14.1, 14.3, 14.1, and 14.5 hr). Four F<sub>2</sub> families (21S950, 21S951, 21S952, and 21S953) were developed by self‐pollinating runnerless to near‐runnerless F<sub>1</sub> individuals (19A907P022, 19A907P024, 19A908P012, and 19A908P053) originating in 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ full‐sib families (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
<p>
<strong>Supplemental File S5</strong>. Pedigrees, runner count, runner score, and inflorescence count phenotypes, and AX‐184947290 SNP genotypes for 372 seed‐propagated full‐sib progeny observed May 1, June 1, and July 1, 2024 in Winters, CA (the respective photoperiods on those dates were 13.5, 14.4, and 14.5 hr). The population included nine full‐sib families developed from crosses among 13 elite UC parents (Table <a href="#tpg270086-tbl-0001" class="usa-link">1</a>).</p>
<p>
<strong>Supplemental File S6</strong>. Haplotypes for 51 phased SNPs among 808 SDP individuals. This database includes 1,606 0,1‐coded haplotypes among 808 SDP individuals classified as either short‐day (SD = 0) or day‐neutral (DN = 1).</p>
<p>
<strong>Supplemental File S7</strong>. Graphical genotypes for 50K Axiom array‐genotyped SNPs segregating between Mb 25.0 and 30.4 on chromosome 4B in 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ F<sub>2</sub> families. This database includes runnering phenotypes, SNP genotypes, and the ordered physical positions for 140 SNP loci among 279 individuals within four F<sub>2</sub> families (21S950, 21S951, 21S952, and 21S953). The predicted associations of SNPs with <em>PF</em> alleles were used to color code cells in the graphical genotype table: peach for <em>pf/pf</em>, pale green for <em>PF/pf</em>, and dark green for <em>PF/PF</em>.</p>
<p>
<strong>Supplemental File S8</strong>. Genotypes and haplotypes for 15 SNPs associated with the <em>PF</em> locus on chromosome 4B among selected samples of three individuals each from an 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ F<sub>2</sub> family (21S950) and two 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> families (21S952 and 21S953). The SNPs spanned bp 26,389,744 (AX‐184068183) to bp 27,597,497 (AX‐184160850). The predicted associations of SNPs with <em>PF</em> alleles were used to color code cells in the graphical genotype table: peach for <em>pf/pf</em>, pale green for <em>PF/pf</em>, and dark green for <em>PF/PF</em>.</p>
<p>
<strong>Supplemental File S9</strong>. The predicted functions of 173 annotated genes spanning bp 27,597,497 to 27,418,663 on chromosome 4B in the ‘UCD Royal Royce’ reference genome (FaRR1; <a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0</a>).</p>
<p>
<strong>Supplemental Figure S1</strong>. Manhattan plots depicting the photoperiod‐dependent effect of the <em>PF</em> locus on runner count among 270 F<sub>2</sub> progeny phenotyped between March 26 and July 6 2022 in Davis, CA. F<sub>2</sub> individuals were genotyped with an Axiom 50K SNP array. Manhattan plots are shown for genome‐wide association studies of phenotypes observed (A) March 26 (12.3 hr), (B) April 9 (13.0 hr), (C) April 22 (13.3 hr), (D) May 6 (14.1 hr) (E) May 20 (14.3 hr), (F) June 3 (14.1 hr), and (G) July 6 (14.5 hr) among 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> ‘UCD Royal Royce’ and 17C321P015 <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span> 55C032P001 F<sub>2</sub> progeny.</p>
<p>
<strong>Supplemental Figure S2</strong>. Synteny of <em>REPRESSOR OF GA1</em> (<em>RGA1</em>) homoeologs on chromosome 4 in the A, B, C, and D genomes of octoploid strawberry (<em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em>) and A genome of woodland strawberry (<em>F. vesca</em>). The ortholog identified in <em>F. vesca</em> (<em>FveRGA1</em>) by Caruana et al. (<a href="#tpg270086-bib-0011" class="usa-link" aria-describedby="tpg270086-bib-0011">2018</a>) is shown below <em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> chromosome 4 haplotypes. Homoelogs found in the octoploid are identified by the prefix Fxa. Homology was calculated using GENESPACE (Lovell et al., <a href="#tpg270086-bib-0051" class="usa-link" aria-describedby="tpg270086-bib-0051">2022</a>). The synteny plot was drawn using JCVI (Tang et al., <a href="#tpg270086-bib-0072" class="usa-link" aria-describedby="tpg270086-bib-0072">2024</a>). The synteny plot was developed using genes annotated in the haplotype‐phased assemblies of the ‘UCD Royal Royce’ genome, a day‐neutral <em>F</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline"><mrow><mo>×</mo></mrow></math></span>
<em>ananassa</em> cultivar (<a href="https://phytozome-next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0.)" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/FxananassaRoyalRoyce_v1_0.)</a>
</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365471/bin/TPG2-18-e70086-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70086-s001.zip</a><sup> (53.9MB, zip) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70086"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70086.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365471/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365471/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365471%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365471/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365471/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365471/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40831148/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365471/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40831148/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365471/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365471/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xfBQI3OIh3Mk7WPQK48LZMdi5Qy4rdZWjvRMDsudxYeMgmadLOqqDIkuWBOjt3b9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
