
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Consolidated 3‑Fold Isotopic Lens for Probing RNAs - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E480548AEEB3630580540030D5D373.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371485/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="Consolidated 3‑Fold Isotopic Lens for Probing RNAs">
<meta name="citation_author" content="Solomon K Attionu">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland 20742, United States">
<meta name="citation_author" content="Lukasz T Olenginski">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Colorado, Boulder, Colorado 80309, United States">
<meta name="citation_author" content="Frances P Stump">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland 20742, United States">
<meta name="citation_author" content="Theodore K Dayie">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland 20742, United States">
<meta name="citation_publication_date" content="2025 Jun 17">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="694">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.5c00075">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371485/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371485/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371485/pdf/bg5c00075.pdf">
<meta name="description" content="Undesired scalar and dipolar couplings are two major interactions that complicate structural and dynamic studies in solution nuclear magnetic resonance (NMR) spectroscopy. Recent developments in site-specific isotopic labeling technologies have gone ...">
<meta name="og:title" content="Consolidated 3‑Fold Isotopic Lens for Probing RNAs">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Undesired scalar and dipolar couplings are two major interactions that complicate structural and dynamic studies in solution nuclear magnetic resonance (NMR) spectroscopy. Recent developments in site-specific isotopic labeling technologies have gone ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371485/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371485">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.5c00075"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg5c00075.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371485%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371485/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371485/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371485/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Jun 17;5(4):694–705. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.5c00075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.5c00075</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Consolidated
3‑Fold Isotopic Lens for Probing
RNAs</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Attionu%20SK%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Solomon K Attionu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Solomon K Attionu</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Biochemistry, University
of Maryland, College
Park, Maryland
20742, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Attionu%20SK%22%5BAuthor%5D" class="usa-link"><span class="name western">Solomon K Attionu</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olenginski%20LT%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lukasz T Olenginski</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lukasz T Olenginski</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Biochemistry, University of Colorado, Boulder, Colorado
80309, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olenginski%20LT%22%5BAuthor%5D" class="usa-link"><span class="name western">Lukasz T Olenginski</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stump%20FP%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Frances P Stump</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Frances P Stump</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Biochemistry, University
of Maryland, College
Park, Maryland
20742, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stump%20FP%22%5BAuthor%5D" class="usa-link"><span class="name western">Frances P Stump</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dayie%20TK%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Theodore K Dayie</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Theodore K Dayie</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Biochemistry, University
of Maryland, College
Park, Maryland
20742, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dayie%20TK%22%5BAuthor%5D" class="usa-link"><span class="name western">Theodore K Dayie</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Chemistry and Biochemistry, University
of Maryland, College
Park, Maryland
20742, United States
</div>
<div id="aff2">
<sup>‡</sup>
Department
of Biochemistry, University of Colorado, Boulder, Colorado
80309, United States
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>dayie@umd.edu</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 12; Accepted 2025 May 28; Revised 2025 May 27; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371485</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Undesired scalar and dipolar couplings are two major
interactions
that complicate structural and dynamic studies in solution nuclear
magnetic resonance (NMR) spectroscopy. Recent developments in site-specific
isotopic labeling technologies have gone a long way toward alleviating
these problems. While some nuclei have intrinsic properties that make
them suitable for specific NMR experiments, these same properties
render them inefficient in other experiments. Site-specific isotopic
labeling facilitates the controlled incorporation of isotopes to enable
facile analysis of RNAs. Here, we describe the synthesis and incorporation
of [1′-<sup>13</sup>C, 2-<sup>19</sup>F, 7-<sup>15</sup>N]
adenosine 5′-triphosphate into the 9 kDa rRNA, thus expanding the applications of previously
synthesized [2-<sup>13</sup>C, 7-<sup>15</sup>N]-adenosine 5′-triphosphate,
with the added benefit of <sup>19</sup>F incorporation. We utilized
these <sup>13</sup>C and <sup>15</sup>N probes to characterize the
structural dynamic features within this RNA, and <sup>19</sup>F was
used to monitor binding interactions. Finally, we leveraged the chemical
shielding anisotropy-dominated relaxation of <sup>15</sup>N7-adenosine
for straightforward analysis of <em>R</em>
<sub>1</sub> and <em>R</em>
<sub>1ρ</sub> rates.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> NMR spectroscopy, site-specific isotopic labeling, chemical shielding anisotropy (CSA), dipole−dipole
coupling, fluorine</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0011.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/d1ce88f2df3c/bg5c00075_0011.jpg" loading="lazy" height="279" width="635" alt="graphic file with name bg5c00075_0011.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/d1ce88f2df3c/bg5c00075_0009.jpg" loading="lazy" id="tgr1" height="279" width="635" alt="graphic file with name bg5c00075_0009.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Among the several biophysical techniques
for the structural determination
of RNAs, nuclear magnetic resonance (NMR) spectroscopy remains an
attractive tool, as it provides structural information at the atomic
level.<em>
−



</em> Nonetheless, some inherent problems are associated with NMR. These
include line broadening, signal overlap, and spectral overcrowding
as molecular weight increases.<em>
−

</em> To limit these problems, spectroscopists
have developed several stable isotope labeling technologies (e.g., <sup>2</sup>H, <sup>13</sup>C, <sup>15</sup>N, and <sup>19</sup>F).<em>
−







</em> They choose the isotope to probe by NMR spectroscopy based on practical
and efficiency considerations. For example, the low gyromagnetic ratio
of <sup>15</sup>N nucleus translates into relatively sharper lines
at high molecular weights.<em>
,
</em> But again, the low
gyromagnetic ratio makes <sup>15</sup>N less sensitive to external
magnetic fields, resulting in weaker NMR signals. In contrast, <sup>19</sup>F not only has a high gyromagnetic
ratio that makes it very sensitive to such fields but also a high chemical shift dispersion (at least six
times that of <sup>1</sup>H) that potentially mitigates spectral crowding. Nonetheless, the large chemical shielding anisotropy
(CSA) of <sup>19</sup>F leads to extensive line broadening in <sup>19</sup>F NMR at high molecular weights and high magnetic field strengths. This is a significant limitation to the use
of <sup>19</sup>F NMR for studying larger biomolecules. Conversely,
the very narrow line widths of the <sup>15</sup>N nucleus make it
an attractive candidate for use in large RNAs, regardless of its inherently
low sensitivity. In addition to <sup>15</sup>N and <sup>19</sup>F, <sup>13</sup>C has a gyromagnetic ratio 1/4th that of <sup>1</sup>H, also translating to sharp lines, albeit broader
than <sup>15</sup>N. Nonetheless, the line width of <sup>13</sup>C
in the ribose ring is relatively sharper due to its small CSA values.
This makes ribose <sup>13</sup>C desirable for experiments that require
relatively high magnetic field strengths. Given that some of the limitations
of <sup>15</sup>N, <sup>13</sup>C, and <sup>19</sup>F are offset by
each other, consolidating the properties of all three into a single
probe would likely enhance its overall utility in studying RNAs by
NMR spectroscopy.</p>
<p>For instance, we recently incorporated <sup>15</sup>N and <sup>13</sup>C into the same nucleotide. We synthesized
and incorporated
[2-<sup>13</sup>C, 7-<sup>15</sup>N]-adenosine 5′-triphosphate
into ∼20 kDa human hepatitis B ε RNA viral element. With <sup>1</sup>H–<sup>15</sup>N heteronuclear
single quantum coherence (HSQC) experiments, we obtained well-resolved peaks, with all expected signals accounted
for. We further characterized this RNA with <sup>15</sup>N-edited
NOESY HSQC and <sup>13</sup>C spin-relaxation
experiments.<em>
,,
</em> In
another previous work, researchers have shown that the substitution
of adenosine-H2 with <sup>19</sup>F does not significantly perturb
RNA’s physical or structural properties. It is unclear if we can ascribe this observation to two
opposing effects: Is <sup>19</sup>F’s ability to destabilize
Watson–Crick base pairing counteracted by its stable base π-stacking?<em>
,
</em> Another noteworthy observation is that because <sup>19</sup>F polarizes
uridine nucleotides, researchers can readily identify different RNA
secondary structural motifs. A similar
observation was documented by Sochor et al. Imino protons of uridines involved in canonical Watson–Crick
base pairing to 2-fluoroadenosine showed significant chemical shift
perturbations (CSPs) from the regular A-U base pair. Additionally,
the presence of fluorine at the H2 position makes it very sensitive
to its chemical environment.<em>
,
</em> With such a dual <sup>13</sup>C–<sup>19</sup>F label, Kreutz and co-workers characterized
RNA-small molecule interactions and detected low-populated dynamic
processes in RNA on the microsecond time scale, otherwise invisible
in a <sup>1</sup>H spectrum, and different RNA secondary structural
motifs.
</p>
<p>In this work, we therefore
extend those dual labels to a triple <sup>15</sup>N–<sup>13</sup>C-<sup>19</sup>F site-specific label.
This site-specific triple label exploits the favorable spectroscopic
properties of <sup>19</sup>F and thus may enhance its versatility.
We synthesized and applied [1′-<sup>13</sup>C, 2-<sup>19</sup>F, 7-<sup>15</sup>N] adenosine 5′-triphosphate to a 9 kDa rRNA. This new isotopic labeling pattern
enables us to qualitatively tease out the effects of <sup>19</sup>F substitution on nuclei even up to five bonds away and analyze the
structural dynamics and interactions of RNA with ligands.</p></section><section id="sec2"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Theoretical Simulations</h3>
<p>The chemical shielding tensors
(CST) for <sup>1</sup>H8-<sup>15</sup>N7 and H1′–C1′
spin pairs were calculated using density functional theory (DFT) methods,
as described in previous work (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Tables S1 and S2</a>).<em>
−


</em> Using these CST parameters, we computed the <em>R</em>
<sub>2</sub> relaxation rates using <a href="#eq1" class="usa-link">eqs 
</a> and  for the <sup>1</sup>H8–<sup>15</sup>N7 and <sup>1</sup>H1–<sup>13</sup>C1′ spin
pair in A (<a href="#fig6" class="usa-link">Figure 
</a>A) assuming isotropic tumbling. We assumed
an isolated N7–H8 and H1′–C1′ spin pair,
where dipolar contributions from neighboring nuclei were ignored.
For bond distances between H8–N7 and H1′–C1′
in A, we used 2.1 and 1.1 Å, respectively, based on the helical
region of the PDB structure 6VAR, the human hepatitis B encapsidation
signal ε RNA. We repeated these
calculations using SPINACH software to consider every significant
relaxation, cross-relaxation, and cross-correlation process in a rigid
system with two spin-1/2 nuclei.
</p>
<table class="disp-formula p" id="eq1"><tr>
<td class="formula"><math id="d1e470" display="block"><msub><mi>R</mi><mn>2</mn></msub><mo>=</mo><mrow><mo stretchy="true">(</mo><mfrac><mi>A</mi><mn>2</mn></mfrac><mo stretchy="true">)</mo></mrow><msub><mrow><mo stretchy="false">[</mo><mn>4</mn><mi>J</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow><mo>+</mo><mn>3</mn><mi>J</mi><mo stretchy="false">(</mo><mi>ω</mi></mrow><mi mathvariant="normal">N</mi></msub><mo stretchy="false">)</mo><mo>+</mo><msub><mrow><mn>6</mn><mi>J</mi><mo stretchy="false">(</mo><mi>ω</mi></mrow><mrow><mi mathvariant="normal">H</mi><mn>8</mn></mrow></msub><mo>+</mo><msub><mi>ω</mi><mi mathvariant="normal">N</mi></msub><mo stretchy="false">)</mo><mo>+</mo><msub><mrow><mn>6</mn><mi>J</mi><mo stretchy="false">(</mo><mi>ω</mi></mrow><mrow><mi mathvariant="normal">H</mi><mn>8</mn></mrow></msub><mo stretchy="false">)</mo><mo>+</mo><msub><mrow><mi>J</mi><mo stretchy="false">(</mo><mi>ω</mi></mrow><mrow><mi mathvariant="normal">H</mi><mn>8</mn></mrow></msub><mo>−</mo><msub><mi>ω</mi><mi mathvariant="normal">N</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">]</mo><mo>+</mo><mrow><mo stretchy="true">(</mo><mfrac><mi>B</mi><mn>6</mn></mfrac><mo stretchy="true">)</mo></mrow><mrow><mo stretchy="false">[</mo><mn>4</mn><mi>J</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow><mo>+</mo><msub><mrow><mn>3</mn><mi>J</mi><mo stretchy="false">(</mo><mi>ω</mi></mrow><mi mathvariant="normal">N</mi></msub><mo stretchy="false">)</mo></mrow><mo stretchy="false">]</mo><mo>+</mo><msub><mi>R</mi><mi>ex</mi></msub></math></td>
<td class="label">1</td>
</tr></table>
<table class="disp-formula p" id="eq2"><tr>
<td class="formula"><math id="d1e602" display="block" indentalign="left" indentshift="1.0em" indentshiftfirst="0em" linebreak="auto"><mi>A</mi><mo>=</mo><msup><mrow><mo stretchy="true">(</mo><mfrac><msub><mi>μ</mi><mn>0</mn></msub><mrow><mn>4</mn><mi>π</mi></mrow></mfrac><mo stretchy="true">)</mo></mrow><mn>2</mn></msup><msup><mrow><mo stretchy="true">(</mo><mfrac><mrow><msub><mi>γ</mi><mi mathvariant="normal">N</mi></msub><msub><mi>γ</mi><mi mathvariant="normal">H</mi></msub><mi>ℏ</mi></mrow><msub><mrow><mn>2</mn><msup><mi>r</mi><mn>3</mn></msup></mrow><mi>NH</mi></msub></mfrac><mo stretchy="true">)</mo></mrow><mn>2</mn></msup><mo>,</mo><mspace width="0.25em"></mspace><mi>B</mi><mo>=</mo><mfrac><mrow><mo stretchy="false">(</mo><msup><msub><mrow><mi mathvariant="normal">Δ</mi><mi>σ</mi></mrow><mi mathvariant="normal">N</mi></msub><mn>2</mn></msup><msup><msub><mi>B</mi><mn>0</mn></msub><mn>2</mn></msup><msup><msub><mi>γ</mi><mi mathvariant="normal">N</mi></msub><mn>2</mn></msup><mo stretchy="false">)</mo></mrow><mn>3</mn></mfrac><mo>,</mo><mo linebreak="goodbreak" lspace="0em" rspace="1em">⁣</mo><mi>J</mi><mrow><mo stretchy="false">(</mo><mi>ω</mi><mo stretchy="false">)</mo></mrow><mo>=</mo><mfrac><msub><mi>τ</mi><mi mathvariant="normal">c</mi></msub><mrow><mn>5</mn><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>+</mo><msup><mrow><msub><mrow><mo stretchy="false">(</mo><mi>ω</mi><mi>τ</mi></mrow><mi mathvariant="normal">c</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo stretchy="false">)</mo></mrow></mrow></mfrac></math></td>
<td class="label">2</td>
</tr></table>
<table class="disp-formula p" id="eq3"><tr>
<td class="formula"><math id="d1e718" display="block" indentshift="1.0em" indentshiftfirst="0em" linebreak="auto"><msub><mrow><mi mathvariant="normal">Δ</mi><mi>σ</mi></mrow><mi mathvariant="normal">N</mi></msub><mo>=</mo><msqrt><mrow><mo stretchy="false">(</mo><msubsup><mi>σ</mi><mi>X</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi>σ</mi><mi>Y</mi><mn>2</mn></msubsup><mo>−</mo><msub><mi>σ</mi><mi>X</mi></msub><msub><mi>σ</mi><mi>Y</mi></msub><mo stretchy="false">)</mo></mrow></msqrt><mo>,</mo><mspace width="0.25em"></mspace><msub><mi>σ</mi><mi>X</mi></msub><mo>=</mo><msub><mi>σ</mi><mi mathvariant="italic">ZZ</mi></msub><mo>−</mo><msub><mi>σ</mi><mi mathvariant="italic">XX</mi></msub><mo>,</mo><mspace width="0.25em"></mspace><msub><mi>σ</mi><mi>Y</mi></msub><mo>=</mo><msub><mi>σ</mi><mi mathvariant="italic">ZZ</mi></msub><mo>−</mo><msub><mi>σ</mi><mi mathvariant="italic">YY</mi></msub></math></td>
<td class="label">3</td>
</tr></table>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/f0376e6ff9a8/bg5c00075_0006.jpg" loading="lazy" id="gr8" height="494" width="627" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Theoretical simulation for <sup>15</sup>N7-A and <sup>13</sup>C1′-A line widths at correlation time
(<strong>τ</strong>
<sub>c</sub>) of 5 ns and varying magnetic field
strengths using
SPINACH software.
<sup>15</sup>N7 transverse
relaxation rates (<em>R</em>
<sub>2</sub>) increase quadratically
with increasing magnetic field strength. (B) Experimental <em>R</em>
<sub>1ρ</sub> for <sup>13</sup>C1′-A<sub>F/H</sub> in fluorinated RNA (magenta) and nonfluorinated RNA (orange). <em>R</em>
<sub>1ρ</sub> for <sup>15</sup>N7-A<sub>F</sub> in
fluorinated RNA (green). Magenta, green, and orange dashed lines represent
the mean for <sup>13</sup>C1′-A<sub>F</sub>, <sup>15</sup>N7-A<sub>F</sub>, and <sup>13</sup>C1′-A<sub>H</sub>, respectively.
(C) Longitudinal relaxation rates (<em>R</em>
<sub>1</sub>)
for <sup>15</sup>N7-A<sub>F</sub>. All relaxation experiments were
conducted at 298 K. (D) <em>R</em>
<sub>2</sub>/<em>R</em>
<sub>1</sub> values for <sup>15</sup>N7-A<sub>F</sub>. The dashed
black lines in (C) and (D) represent the average relaxation rates.</p></figcaption></figure><p>σ<sub>
<em>XX</em>
</sub>, σ<sub>
<em>YY</em>
</sub>, and σ<sub>
<em>ZZ</em>
</sub> are the principal components of the traceless chemical
shielding tensor in the Haeberlen notation,<em>
,
</em> ordered as follows: | σ<sub>
<em>ZZ</em>
</sub>-σ<sub>
<em>iso</em>
</sub>| ≥ | σ<sub>
<em>XX</em>
</sub>-σ<sub>
<em>iso</em>
</sub>| ≥ | σ<sub>
<em>YY</em>
</sub>-σ<sub>
<em>iso</em>
</sub>|; <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e827" display="inline"><msub><mi>σ</mi><mi>iso</mi></msub><mo>=</mo><mfrac><mn>1</mn><mn>3</mn></mfrac><mrow><mo stretchy="false">(</mo><msub><mi>σ</mi><mi mathvariant="italic">XX</mi></msub><mo>+</mo><msub><mi>σ</mi><mi mathvariant="italic">YY</mi></msub><mo>+</mo><msub><mi>σ</mi><mi mathvariant="italic">ZZ</mi></msub><mo stretchy="false">)</mo></mrow></math>
</span>. Here, μ<sub>0</sub> is the permeability
of the vacuum, γ<sub>N</sub> and γ<sub>H</sub> are the
gyromagnetic ratios of <sup>15</sup>N and <sup>1</sup>H respectively,
ℏ is Planck’s constant divided by 2π, <em>r</em>
<sub>NH</sub> is the internuclear distance between <sup>15</sup>N7 and <sup>1</sup>H8, Δ<strong>σ</strong>
<sub>N</sub> (<a href="#eq3" class="usa-link">eq 
</a>) is the chemical
shift anisotropy of <sup>15</sup>N, <em>B</em>
<sub>o</sub> is the static magnetic field strength, <em>J</em>(ω)
is the spectral density function at a given frequency assuming isotropic
tumbling, <strong>τ</strong>
<sub>c</sub> is the rotational correlation
time, ω is the angular frequency of the nucleus, and <em>R</em>
<sub>ex</sub> is the chemical exchange rate constant. The
internuclear distance and magnetic field strength were adjusted accordingly
when simulating relaxation rates for C1′ in A.</p></section><section id="sec2.2"><h3 class="pmc_sec_title">DFT Calculations</h3>
<p>Quantum chemistry calculations were
carried out using ORCA<em>
,
</em> and the automated fragmentation
(AFNMR) package<em>
,
</em> as described earlier.<em>
−


</em>
</p></section><section id="sec2.3"><h3 class="pmc_sec_title">General Procedures</h3>
<p>A detailed description of the synthesis
of [2-<sup>13</sup>C, 7-<sup>15</sup>N]-adenine and [7-<sup>15</sup>N]-guanine was described previously.<em>
,
</em> Commercially
available reagents were used for the syntheses of the nucleobases
without further purification. Isotopically labeled [<sup>15</sup>N]-sodium
nitrite was purchased from Sigma-Aldrich (St. Louis, MO).</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Reagents</h3>
<p>[1-<sup>13</sup>C]-ribose was purchased from
Omicron Biochemicals (IN). The enzymes used in the one-pot coupling
reaction of [1-<sup>13</sup>C]-ribose to an isotopically labeled nucleobase
were ribokinase (RK), phosphoribosyl pyrophosphate synthetase (PRPPS),
and adenine phosphoribosyl transferase (APRT). These enzymes were
purified and expressed by methods we have previously described.<em>
,−

</em> Thermostable pyrophosphatase was purchased from New
England Biolabs (Ipswich, MA). Bovine serum albumin, creatine kinase
(CK), and myokinase (MK) were purchased from ThermoFisher Scientific
(Waltham, MA). A boronate column with Affi-Gel Boronate Gel (BioRad)
was used in the purification of isotopically labeled nucleoside triphosphates.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">Nucleoside Triphosphate Synthesis</h3>
<p>The nucleoside triphosphates
used in the enzymatic transcription of both RNA constructs (fluorinated
and nonfluorinated) were synthesized as described previously.<em>
,
</em> All reactions were carried out at 37 °C in synthesis buffer
(100 mM creatine phosphate, 50 mM HEPES pH 8, 10 mM magnesium chloride,
10 mM dithiothreitol, 100 mM potassium chloride, and 0.1 mg/mL bovine
serum albumin) and a dATP regeneration system (0.5 mM dATP, 100 mM
creatine phosphate, 0.05 mg/mL creatine kinase, and 10 mU/μL
myokinase). The reaction was initiated with 10 μU/μL RK,
10 μU/μL adenine phosphoribosyl transferase, 10 μU/μL
PRPPS, 0.1 mg/mL thermal-stable inorganic pyrophosphatase, 8 mM fluorinated
and nonfluorinated adenine, and 7 mM [1-<sup>13</sup>C]-ribose. On
completion in 24 h, all of the reactions were purified with a boronate
resin column at 4 °C.<em>
,,
</em>
</p></section><section id="sec2.6"><h3 class="pmc_sec_title">RNA Preparation</h3>
<p>The DNA template used for transcription
is 5′-mGmGC GAC TTC ACC CGA AGG TGT GAC G C CTA TAG TGA GTC
GTA TTA G-3′. The T7 RNA polymerase promoter sequence was 5′-CTA
TAG TGA GTC GTA TTA G-3′. DNA template was purchased from Integrated
DNA Technologies (IDT) (Coralville, IA). 10 mL <em>in vitro</em> transcription reaction was carried out at 37 °C using a transcription
buffer (40 mM Tris–HCl pH 8.0, 1 mM spermidine and 0.01% Triton
X-100), 0.3 μM single strand DNA template, 80 mg/mL PEG, 1 mM
DTT, 2 U/μL thermostable inorganic pyrophosphatase, 10 mg/mL
T7 RNA polymerase, a total of 10 mM rNTPs (2.5 mM each of unlabeled
GTP, CTP, UTP, and either <strong>18</strong> or <strong>19</strong>), and
10 mM MgCl<sub>2</sub>. The concentrations of rNTPs and MgCl<sub>2</sub> were chosen following optimization in small (50 μL) reactions.
After 3 h, the reaction was quenched by adding 1 mL of 0.5 mM EDTA
(pH 8). 10 mL of phenol–chloroform was added to the aqueous
transcription reaction. The suspension was spun in a centrifuge at
10,500 rpm for 15 min. Upon completion, the aqueous RNA layer (top
layer) was collected into a clean falcon tube with a pipet, followed
by the addition of 1 mL of 3 M NaOAc and 30 mL of ice-chilled absolute
ethanol. This solution was left in the refrigerator overnight at −20
°C. The solid precipitate formed was isolated from the solution
by centrifugation at 10,500 rpm for 1 h. The aqueous solution was
decanted, leaving behind a white solid crude RNA residue that was
resuspended in a 1× TBE buffer. The crude RNA solution was purified
via preparative polyacrylamide (10%) denaturing gel electrophoresis,
followed by extraction of RNA from the gel using the crush and soak
method. The collected RNA was exchanged
with deionized water and lyophilized. The RNA pellets were dissolved
in 270 μL of D<sub>2</sub>O and 30 μL of 10× NMR
buffer (200 mM Na<sub>3</sub>PO<sub>4</sub> pH 6.5, 1 M NaCl, 5 mM
DTT, 2 mM EDTA pH 8, 0.27 mM NaN<sub>3</sub>, 0.1 mM DSS). The RNA
solutions were heated to 95 °C for about 3 min and rapidly cooled
on ice for 5 min. Concentrations of RNA were calculated using a molar
absorption coefficient for the rRNA of 281 mM<sup>–1</sup> cm<sup>–1</sup>. The concentrations
of the purified unbound rRNA
samples were ≈0.6 mM. The concentrations of the purified paromomycin-bound rRNA samples were 0.3 mM.</p></section><section id="sec2.7"><h3 class="pmc_sec_title">NMR Spectroscopy</h3>
<p>All NMR experiments were conducted
at 14.4 T (600 MHz <sup>1</sup>H frequency) on a Bruker Avance III
HD spectrometer equipped with a triple-resonance shielded z-gradient
probe. Two-dimensional two-bond (<sup>2</sup>
<em>J</em>
<sub>H8N9</sub>) <sup>1</sup>H–<sup>15</sup>N HSQC data for fluorinated
and nonfluorinated RNAs were collected in D<sub>2</sub>O at 277 and
298 K. Two-dimensional one-bond (<sup>1</sup>
<em>J</em>
<sub>H1′C1</sub>) <sup>1</sup>H–<sup>13</sup>C HSQC data
for fluorinated and nonfluorinated RNAs were collected in D<sub>2</sub>O at 277 and 298 K. <sup>13</sup>C <em>R</em>
<sub>1ρ</sub> relaxation data on fluorinated rRNA were collected in D<sub>2</sub>O at 298 K using standard Bruker
pulse programs. For the <sup>13</sup>C <em>R</em>
<sub>1ρ</sub> experiment, relaxation delays of 0.004 (x2), 0.016, 0.02, 0,024,
0.036, and 0.064 s were used. <sup>15</sup>N <em>R</em>
<sub>1</sub> and <em>R</em>
<sub>1ρ</sub> relaxation data
on fluorinated rRNA were also
collected in D<sub>2</sub>O at 298 K using standard Bruker pulse programs. For the <sup>15</sup>N <em>R</em>
<sub>1</sub> experiment, relaxation delays of 0.02 (x2), 0.04, 0.06, 0.08,
0.12, 0.16, 0.2, 0.24, and 0.3 s were used. For the <sup>15</sup>N <em>R</em>
<sub>1ρ</sub> experiment, relaxation delays of 0.004
(x2), 0.016, 0.02, 0.024, 0.036, and 0.064 s were used. <sup>15</sup>N <em>R</em>
<sub>1ρ</sub> relaxation data on nonfluorinated rRNA were also collected in D<sub>2</sub>O at 298 K using standard Bruker pulse programs. Relaxation delays of 0.004 (x2), 0.016, 0.02, 0.024, 0.036,
and 0.064 s were used. Spin-lock strength of 1.9 kHz was applied for
all <em>R</em>
<sub>1ρ</sub> measurements. Two-dimensional
(2D) <sup>1</sup>H–<sup>1</sup>H NOESY was conducted on the
fluorinated RNA with 300 ms mixing time, 64 scans, and 2048 ×
1024 complex data points in the <sup>1</sup>H dimensions at 298 K.
2D imino <sup>1</sup>H–<sup>1</sup>H NOESY was conducted on
both fluorinated and nonfluorinated RNAs with 250 ms mixing time,
64 scans, and 2048 × 400 complex data points in the <sup>1</sup>H dimensions at 283 K. 2D <sup>13</sup>C- and <sup>15</sup>N-edited
NOESY HSQC data were collected on the fluorinated RNA with a 250 ms
mixing time at 298 K. 2D <sup>15</sup>N-edited NOESY HSQC experiment
was carried out with 40 scans per increment with spectral widths of
14 ppm (2048 complex data points) in the direct <sup>1</sup>H-dimension
and 14 ppm (200 complex data points) in the indirect <sup>1</sup>H-dimension. <sup>13</sup>C-edited NOESY HSQC experiment was carried out with 64 scans
per increment with spectral widths of 14 ppm (2048 complex data points)
in the direct <sup>1</sup>H-dimension and 14 ppm (200 complex data
points) in the indirect <sup>1</sup>H-dimension. <sup>1</sup>H–<sup>1</sup>H TOCSY was collected for the fluorinated RNA with a mixing
time of 120 ms at 298 K. This experiment was carried out with 128
scans per increment with spectral widths of 12 ppm (2048 complex data
points) in the direct <sup>1</sup>H-dimension and 10 ppm (234 complex
data points) in the indirect <sup>1</sup>H-dimension. 2D <sup>13</sup>C-edited TOCSY HSQC experiment was performed with 64 scans per increment,
with spectral widths of 16 ppm (2048 complex data points) in the direct <sup>1</sup>H-dimension and 16 ppm (200 complex data points) in the indirect <sup>1</sup>H-dimension. All NMR data were processed and analyzed by Bruker
Topspin Software version 4.1.4.</p></section><section id="sec2.8"><h3 class="pmc_sec_title">UV Melting Experiment</h3>
<p>1 μM solutions of <strong>18</strong> and <strong>19</strong>-labeled RNA were prepared in a buffer
containing 15 mM sodium cacodylate (pH 6.4), 25 mM NaCl, and 0.1 mM
EDTA (pH 8), such that their absorbances were between 0.1 and 1. Samples
were then transferred into clean cuvettes and placed in a ultraviolet
(UV) spectrophotometer with the temperature change set at 1 °C
per minute. The temperature range was set from 4 to 95 °C. The
absorbances were recorded as a function of varying temperature. The
melting curves were fit by using the Marquardt nonlinear least-squares
method. Melting point and thermodynamic parameters such as Δ<em>G</em>
<sup>298K</sup> were extracted using the van’t Hoff
expression and nonlinear curve fitting previously described for unimolecular
hairpin formation by Luteran et al. and
Siegfried and Bevilacqua. The heat-up
(4 to 95 °C) and cool-down (95 to 4 °C) absorbances were
used as duplicates for data analysis.</p></section><section id="sec2.9"><h3 class="pmc_sec_title">Mass Spectrometry</h3>
<p>Mass spectrometry was conducted using
a JEOL AccuTOF-CS electrospray instrument (ESI) with positive mode
ionization for nucleobase and negative mode ionization for nucleoside
5′-triphosphate.</p></section><section id="sec2.10"><h3 class="pmc_sec_title">Synthesis of [1′-<sup>13</sup>C, 2-<sup>19</sup>F, 7-<sup>15</sup>N] Adenosine 5′-Triphosphate</h3>
<p>The synthesis
of the atom-specifically labeled [2-<sup>19</sup>F, 7-<sup>15</sup>N] adenine probe was carried out in 10 chemical steps, starting with
the sodium ethoxide (C<sub>2</sub>H<sub>5</sub>ONa)-mediated cyclization
of guanidine <strong>1</strong> and ethylcyanoacetate <strong>2</strong> to
yield diaminopyrimidinone 3 (<a href="#sch1" class="usa-link">Scheme 
</a>). Nitrosylation of <strong>3</strong> with commercially
available <sup>15</sup>N-sodium nitrite (Na<sup>15</sup>NO<sub>2</sub>) then installed <sup>15</sup>N at the 5-position to afford compound <strong>4</strong>.</p>
<figure class="fig xbox font-sm" id="sch1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/740953f960cb/bg5c00075_0007.jpg" loading="lazy" id="gr1" height="289" width="700" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The nitroso group in <strong>4</strong> was reduced
by sodium dithionite
(Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>) to form triaminopyrimidinone <strong>5</strong>, which was then cyclized by reacting with formamide (CH<sub>3</sub>NO) and formic acid (CH<sub>3</sub>COOH) to provide guanine <strong>6</strong>. Chlorination of <strong>6</strong> was achieved using the Vilsmeier–Haack
reaction to yield intermediate <strong>7</strong> followed by subsequent
reactions with CH<sub>3</sub>COOH and sodium hydroxide (NaOH) to yield <strong>8</strong> and <strong>9</strong>. The chloro group in <strong>9</strong> was
converted to an azido group by the addition of sodium azide (NaN<sub>3</sub>) to afford <strong>10</strong>. In the second-to-last step, a
diazotization reaction with tetrafluoroboric acid (HBF<sub>4</sub>) provided fluorinated <strong>11</strong>. Finally, the azido group
in <strong>11</strong> was reduced by sodium borohydride (NaBH<sub>4</sub>) to yield the desired [7-<sup>15</sup>N]-2-fluoroadenine <strong>12</strong>. Taken together, our synthetic route provided <strong>12</strong> in
10 chemical steps with an overall yield of 5% (cf 12% for previously
synthesized phosphoramidites and 5% nonhydrolyzable 2FATP) and without the need for chromatographic purification
(<a href="#sch1" class="usa-link">Scheme 
</a>).</p>
<p>We
then adapted our established one-pot enzymatic reaction<em>
,,,,
</em> to couple the newly synthesized [7-<sup>15</sup>N]-2-fluoroadenine <strong>12</strong> with [1-<sup>13</sup>C]-<span class="font-variant-small-caps">d</span>-ribose followed by subsequent phosphorylation to the corresponding
adenosine 5′-triphosphate (ATP) (<a href="#sch2" class="usa-link">Scheme 
</a>). In brief, ribokinase (RK, E.C. 2.7.1.15)
phosphorylates [1-<sup>13</sup>C]-<span class="font-variant-small-caps">d</span>-ribose <strong>13</strong> at its position O5 to yield <strong>14</strong>. Then, phosphoribosyl
pyrophosphate synthetase (PRPPS, E.C. 2.7.6.1) pyrophosphorylates <strong>14</strong> at its O1′ site to form 5-phospho-<span class="font-variant-small-caps">d</span>-ribosyl-α-1-pyrophosphate <strong>15</strong>. Adenine phosphoribosyl transferase (APRT, E.C. 2.4.2.7)
then facilitates the nucleophilic attack of the C1′ position
of <strong>15</strong> by N9 of <strong>12</strong> to afford the 5′-monophosphate <strong>16</strong>. Finally, myokinase (MK, E.C. 2.7.4.3) phosphorylates <strong>16</strong> to yield 5′-diphosphate <strong>17</strong>, which is
phosphorylated a final time by creatine kinase (CK, E.C. 2.7.3.2)
to form the desired [1′-<sup>13</sup>C, 2-<sup>19</sup>F, 7-<sup>15</sup>N] ATP <strong>18</strong> in 95% yield (<a href="#sch2" class="usa-link">Scheme 
</a>). The characterization of compounds <strong>9</strong>, <strong>10</strong>, <strong>11</strong>, <strong>12</strong>, and <strong>18</strong> is shown in the Supporting Information (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S1–S11</a>).</p>
<figure class="fig xbox font-sm" id="sch2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/932c0bb101a7/bg5c00075_0008.jpg" loading="lazy" id="gr2" height="308" width="700" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section></section><section id="sec3"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">NMR Spectroscopy of Atom-Specifically Labeled RNA</h3>
<p>We
synthesized building block <strong>18</strong> for NMR analysis of RNA.
To this end, we used our atom-specifically labeled <strong>18</strong> along with unlabeled guanosine (G), cytidine (C), and uridine (U)
5′-triphosphates to make a 27-nucleotide (nt) fluorinated RNA
construct by DNA-templated RNA polymerase-based in vitro transcription
(<a href="#fig1" class="usa-link">Figure 
</a>A). This RNA
is derived from the aminoacyl site of the rRNA in and is therefore critical for protein synthesis
and bacterial survival. We will refer
to this RNA as the A-site throughout the text. A one-dimensional (1D) <sup>19</sup>F NMR spectrum of <strong>18</strong>-labeled RNA showed all
four fluorine signals corresponding to each of the modified adenosine
nucleotides although two peaks downfield overlap with each other (<a href="#fig1" class="usa-link">Figure 
</a>B). As a control
(<a href="#fig1" class="usa-link">Figure 
</a>A), we made
a nonfluorinated (i.e., unmodified) RNA construct with our recently
reported [2-<sup>13</sup>C, 7-<sup>15</sup>N]-adenosine into [1′,2-<sup>13</sup>C<sub>2</sub>, 7-<sup>15</sup>N]-ATP <strong>19</strong>.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/b1771994af42/bg5c00075_0001.jpg" loading="lazy" id="gr3" height="1093" width="750" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Validation
of atom-specifically labeled RNA constructs. (A) Secondary
structure of the 27-nt A-site RNA made from <em>in vitro</em> transcription with either <strong>18</strong> (left) or <strong>19</strong> (right) incorporated. Nucleotides labeled with <strong>18</strong> or <strong>19</strong> are numbered and shown in green or black, respectively.
(B) 1D <sup>19</sup>F NMR spectra at 298 K of the <strong>18</strong>-labeled
A-site with (magenta) or without (green) paromomycin. (C) Optical
melting data from the <strong>18</strong>- and <strong>19</strong>-labeled
A-site with the extracted melting temperature (<em>T</em>
<sub>m</sub>) and Gibbs free energy of folding at 298 K (ΔG<sup>298K</sup>). Comparison of the imino region of <sup>1</sup>H NMR
for (D) <strong>19</strong>-labeled A-site RNA and (E) <strong>18</strong>-labeled A-site RNA with and without paromomycin at 298 K. <sup>1</sup>H–<sup>1</sup>H NOESY NMR at 283 K showing imino–imino <sup>1</sup>H cross peaks in (F) <strong>19</strong>-labeled A-site RNA and
(G) <strong>18</strong>-labeled A-site RNA. Asterisks (*) indicate proton
signal with the largest chemical shift perturbation between <strong>18</strong>- and <strong>19</strong>-labeled A-site RNA as confirmed by
imino–imino NOE contacts in F and G.</p></figcaption></figure><p>The H2 position in purines is sensitive to changes
in chemical
environment such as drug binding due to its direct involvement in
π-stacking interactions in an RNA. Given that fluorine is also
extremely sensitive to changes in its environment owing to the uneven
distribution of electrons in its valence p-orbitals, the substitution
of <sup>1</sup>H with <sup>19</sup>F at the 2-position should be an
effective probe to validate binding of cognate ligands. Not surprisingly,
we observed significant chemical shift perturbations in the <sup>19</sup>F NMR spectrum upon addition of the well-characterized bulge-binding
A-site ligand paromomycin (PAR) to <strong>18</strong>-labeled RNA (<a href="#fig1" class="usa-link">Figure 
</a>B).</p>
<p>Optical
melting experiments with these two RNAs confirmed that <sup>19</sup>F labeling did not significantly perturb A-site thermal stability
or folding thermodynamics (<a href="#fig1" class="usa-link">Figure 
</a>C), in agreement with previous reports. Optical melting experiments provided overall
thermodynamics comparison between the fluorinated and nonfluorinated
RNAs. To ascertain local site-specific differences between both RNAs,
we monitored changes in the H-bonded imino proton signals upon fluorination
of the RNA. The imino proton of U18 (U18 H3), which is base-paired
to A9 within the regular A-helix, appeared to be more shielded in
the fluorinated RNA (<a href="#fig1" class="usa-link">Figure 
</a>D,E). This is likely due to the electrostatic
repulsions between the fluorine atom of fluorinated adenosine (F2)
and oxygen (O2) of uridine. This observation agrees with previous
reports on HIV TAR RNA. While we observed
more imino <sup>1</sup>H signals in the fluorinated RNA, the NMR imino
melt experiment revealed no differences in the melting profiles for
both fluorinated and nonfluorinated RNAs (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12</a>).</p>
<p>Upon titration of PAR, both RNA constructs undergo
CSPs (<a href="#fig1" class="usa-link">Figure 
</a>D,E), indicating that the fluorinated RNA retained
the binding
properties of the nonfluorinated RNA (<a href="#fig1" class="usa-link">Figure 
</a>B). Again, both RNA samples had nearly similar
imino–imino proton NOE contacts (<a href="#fig1" class="usa-link">Figure 
</a>F,G). For example, even though U18 H3 was
more shielded in the fluorinated RNA, the pattern of its NOE cross
peak with the preceding G17 H1 was comparable to that of the nonfluorinated
RNA (<a href="#fig1" class="usa-link">Figure 
</a>F,G).</p>
<p>As a second application, we implemented
two-dimensional (2D) HSQC
experiments using one (<sup>1</sup>
<em>J</em>
<sub>H1′C1′</sub>)- and two (<sup>2</sup>
<em>J</em>
<sub>H8N7</sub>)-bond couplings
for coherence transfer in both fluorinated (i.e., <strong>18</strong>-labeled)
and nonfluorinated (i.e., <strong>19</strong>-labeled) RNA. As expected,
all four H8–N7 (<a href="#fig2" class="usa-link">Figure 
</a>A) and H1′–C1′ (<a href="#fig2" class="usa-link">Figure 
</a>B) resonances were observed. In the <sup>1</sup>H–<sup>15</sup>N experiment, fluorine incorporation
deshielded the <sup>15</sup>N–N7 atoms but shielded the <sup>1</sup>H atoms (<a href="#fig2" class="usa-link">Figure 
</a>A). Nucleotides A20 and A21 exhibited the largest CSPs between the
fluorinated and nonfluorinated RNAs in both <sup>15</sup>N and <sup>13</sup>C HQSCs (<a href="#fig2" class="usa-link">Figure 
</a>C,D).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/6d576cccd5d7/bg5c00075_0002.jpg" loading="lazy" id="gr4" height="530" width="627" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Carbon and Nitrogen NMR parameters of
fluorinated and nonfluorinated
RNA. (A) 2D <sup>1</sup>H–<sup>15</sup>N HSQC experiments using
the two-bond (<sup>2</sup>
<em>J</em>
<sub>H8N7</sub>) coupling
for coherence transfer (inset) to show all H8–N7 resonances.
(B) 2D <sup>1</sup>H–<sup>13</sup>C HSQC experiments using
the one-bond (<sup>1</sup>
<em>J</em>
<sub>H1′C1′</sub>) coupling for coherence transfer (inset) to show all four H1′–C1′
resonances. Both HSQC experiments were conducted at 298 K. Chemical
shift differences were calculated as Δδ = <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e1590" display="inline"><msqrt><msup><mrow><mo stretchy="false">(</mo><mi mathvariant="normal">Δ</mi><msub><mi>δ</mi><mi mathvariant="normal">H</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="false">(</mo><mi>α</mi><mi mathvariant="normal">Δ</mi><msub><mi>δ</mi><mi>N/C</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></msqrt></math>
</span> between <strong>18</strong>- and <strong>19</strong>-labeled RNAs in (C) <sup>1</sup>H–<sup>15</sup>N and (D) <sup>1</sup>H–<sup>13</sup>C HSQCs, where Δδ<sub>H</sub> and Δδ<sub>N/C</sub> are the changes in proton and nitrogen/carbon
chemical shifts, respectively, and α is the ratio of the gyromagnetic
ratio between nitrogen/carbon and proton. All NMR spectra are annotated
with resonance assignments.</p></figcaption></figure><p>To qualitatively assess the effect of fluorine
on the spectroscopic
properties of nearby nuclei in more detail, we repeated our HSQC experiments
after titration with PAR and at a reduced temperature. Any <sup>19</sup>F-induced effect on relaxation will be amplified when bound to PAR
or at lower temperatures. Indeed, titration with PAR resulted in the
disappearance of two H8–N7 resonances in the nonfluorinated
RNA spectrum (<a href="#fig3" class="usa-link">Figure 
</a>A) while only a single signal was lost in the fluorinated construct
(<a href="#fig3" class="usa-link">Figure 
</a>B). In the
case of H1′–C1′ resonances, only one nonfluorinated
RNA signal was lost upon PAR titration (<a href="#fig3" class="usa-link">Figure 
</a>C), while the fluorinated RNA retained all
four resonances (<a href="#fig3" class="usa-link">Figure 
</a>D). However, in all cases, we observed attenuation of the signal
intensity of A20 or the complete disappearance of its signal. This
agrees with previous reports that have shown nucleotide A20 to be
a dynamic hotspot for aminoglycoside binding. At a lower temperature of 10 °C, only one WT H8–N7 resonance
disappeared (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S13A</a>), while all four
signals were retained in the fluorinated construct (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S13B</a>). The H1′–C1′ resonances,
on the other hand, were mostly unaffected (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S13C,D</a>).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/cce03d124b22/bg5c00075_0003.jpg" loading="lazy" id="gr5" height="512" width="627" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ligand titrations in fluorinated and nonfluorinated RNA.
2D <sup>1</sup>H–<sup>15</sup>N HSQC experiments with PAR titrated
against (A) <strong>19-</strong>labeled and (B) <strong>18</strong>-labeled
A-site. 2D <sup>1</sup>H–<sup>13</sup>C HSQC experiments with
PAR titrated against (C) <strong>19</strong>-labeled and (D) <strong>18</strong>-labeled A-site. All HSQC experiments were conducted at 298 K. All
NMR spectra are annotated with resonance assignments.</p></figcaption></figure><p>We also performed a 2D <sup>1</sup>H–<sup>1</sup>H total
correlation spectroscopy (TOCSY) experiment to reveal all uridine and cytidine H5/H6 cross peaks. Given our
previous findings that fluorination of adenosine at the 2-position
alters the chemical environment of H8–N7 and H1′–C1′
resonances, we hypothesized that fluorine may perturb additional nearby
resonances relative to the unfluorinated RNA. Given that all adenosine
nucleotides are in or near the bulge, any <sup>19</sup>F-induced chemical
shift perturbations (CSPs) would likely affect uridine and cytidine
nucleotides that are near this structural motif. Indeed, comparison
of fluorinated and nonfluorinated spectra shows CSPs for H5/H6 nuclei
in bulge nucleotides C6 and C8, as well as the adjacent but overlapping
U5/C10/U23 (<a href="#fig4" class="usa-link">Figure 
</a>A). However, CSPs were also observed for nucleotides C10–U12
that extend into the top of the upper helix and into the tetraloop.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/7eb1ffbffafa/bg5c00075_0004.jpg" loading="lazy" id="gr6" height="889" width="627" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TOCSY
experiments in fluorinated and nonfluorinated RNA. (A) 2D <sup>1</sup>H–<sup>1</sup>H TOCSY experiments in the <strong>18</strong>-labeled
and <strong>19</strong>-labeled A-site. (B) 2D <sup>1</sup>H–<sup>1</sup>H TOCSY experiments with PAR titrated against the <strong>19</strong>-labeled A-site. (C) 2D <sup>1</sup>H–<sup>1</sup>H TOCSY
experiments with PAR titrated against the <strong>18</strong>-labeled
A-site. All TOCSY experiments were conducted at 298 K. All NMR spectra
are annotated with resonance assignments. CSPs mapped onto the secondary
structure of the 27-nt A-site RNA are shown to the right of each respective
spectrum.</p></figcaption></figure><p>To further explore this observation that <sup>19</sup>F-induced
CSPs localize mainly to bulge nucleotides, we titrated PAR against
WT RNA. Again, paromomycin binds to the A-rich bulge region of the
nonfluorinated rRNA. Here, we observed a set of CSPs similar to those
of bulge nucleotides C6, C8, and U5/C10/U23 (<a href="#fig4" class="usa-link">Figure 
</a>B). As before, CSPs to upper helix nucleotides
C10 and C11 were also observed, as were CSPs to lower helix nucleotides
C3, C24, and C26 (<a href="#fig4" class="usa-link">Figure 
</a>B). As with the WT RNA, a very similar set of CSPs was observed
for the fluorinated RNA upon titration of PAR (<a href="#fig4" class="usa-link">Figure 
</a>C). Taken together, these data establish
a set of shared <sup>19</sup>F- and PAR-induced CSPs that correspond
to H5/H6 nuclei from uridines and cytidines in and around the bulge
region (<a href="#fig4" class="usa-link">Figure 
</a>).
Given that only adenosines were fluorinated (<strong>18</strong>-labeled
RNA), we expected the most significant CSPs for nucleotides near adenosines.
Similarly, the binding pocket for PAR in the nonfluorinated RNA (<strong>19</strong>-labeled RNA) is the A-rich bulge. Hence, we also expected major CSPs upon PAR binding to be localized
to the nucleotides near adenosines.</p></section><section id="sec3.2"><h3 class="pmc_sec_title">Chemical Shift Assignments and Structure of 18-Labeled RNA</h3>
<p>Peak patterns observed in both <sup>1</sup>H–<sup>13</sup>C and <sup>1</sup>H–<sup>15</sup>N HSQCs (<a href="#fig2" class="usa-link">Figure 
</a>A,B) were offset but identical
for both RNA constructs (i.e., <strong>18</strong>-labeled and <strong>19</strong>-labeled RNAs). Hence, we performed a series of experiments to assess
our preliminary assignments for the fluorinated RNA, which was solely
based on the peak patterns for the WT RNA.</p>
<p>We performed a <sup>1</sup>H–<sup>1</sup>H NOESY experiment on the fluorinated
RNA to obtain intranucleotide adenosine H8–H1′ correlations
(<a href="#fig5" class="usa-link">Figure 
</a>A). However,
given severe signal overlap among aromatic, ribose, and H5 of pyrimidine
hydrogen atoms, we employed a series of <sup>13</sup>C- and <sup>15</sup>N-edited experiments to simplify data analysis. <sup>15</sup>N7-edited
NOESY provides both intranucleotide H8–H1′ cross peaks
and internucleotide cross peaks between H8 and neighboring protons
in the 3′ to 5′ direction. Similarly, <sup>13</sup>C1′
edited NOESY produces intranucleotide H1′–H8 cross peaks
and internucleotide cross peaks between H1′ and neighboring
aromatic protons in the 5′ to 3′ direction. A set of
shared cross peaks between both <sup>13</sup>C1′- and <sup>15</sup>N7-edited NOESY experiments unambiguously represents intranucleotide
H1′–H8 correlation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S14</a>). In addition, <sup>13</sup>C1′- and <sup>15</sup>N7-edited
NOESY experiments are complementary techniques in that they permit
analysis of internucleotide correlations in either 5′ to 3′
or 3′ to 5′ directions, respectively.
</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371485_bg5c00075_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d3c5/12371485/ef40d8ec51c6/bg5c00075_0005.jpg" loading="lazy" id="gr7" height="923" width="750" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Regular <sup>1</sup>H–<sup>1</sup>H NOESY spectrum of <strong>18</strong>-labeled RNA with a mixing time of 250 ms. Red and blue
shaded regions in A are edited by <sup>13</sup>C and <sup>15</sup>N-nuclei, respectively, for facile analyses in (B–D). (B) <sup>13</sup>C1′-edited NOESY HSQC showing both intramolecular
H8–H1′ contacts in adenosine and some intermolecular
contacts to adenosine H8. (C) <sup>15</sup>N7-edited NOESY-HSQC showing
both intramolecular H8–H1′ contacts in adenosine, and
some intermolecular contacts to adenosine H8. Asterisks (*) indicate
shared cross peaks between <sup>15</sup>N7- and <sup>13</sup>C1′-edited
NOESY experiments. (D) <sup>13</sup>C1<sup>′</sup> TOCSY-HQSC
showing H1′–H2′ correlations within adenosines
with the 2′-endo sugar pucker conformation. All experiments
were conducted at 298 K.</p></figcaption></figure><p>First, we performed both <sup>13</sup>C1′-
and <sup>15</sup>N7-edited NOESY HSQC experiments that enabled facile
correlation
of <sup>13</sup>C1′ to their corresponding intranucleotide <sup>15</sup>N7 between their shared proton resonances, even in the absence
of chemical shift assignments (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S15</a>). These correlations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S15</a>) corroborate
our preliminary assignments for the fluorinated RNA (<a href="#fig2" class="usa-link">Figure 
</a>A,B). Furthermore, we observed
the expected set of shared cross peaks between <sup>15</sup>N7- and <sup>13</sup>C1′-edited NOESY experiments (highlighted by the magenta
asterisks*), indicating intranucleotide H1′–H8 correlations
within the adenosines (<a href="#fig5" class="usa-link">Figure 
</a>B,C). Additionally, we observed internucleotide cross peaks
in both <sup>15</sup>N7- and <sup>13</sup>C1′-edited NOESY
experiments (<a href="#fig5" class="usa-link">Figure 
</a>B,C).</p>
<p>Second, we performed an H1′–H2′
TOCSY experiment
that provides structural information about sugar puckering in a nucleotide.
For nucleotides in 2′-endo (B-DNA like) conformation, H1′
and H2′ have sizable three-bond scalar coupling (<sup>3</sup>
<em>J</em>
<sub>H1′H2′</sub> ≈8 Hz),
while those in 3′-endo (RNA like) have small scalar coupling
(<sup>3</sup>
<em>J</em>
<sub>H1′H2′</sub> &lt;
2 Hz). Hence, nucleotides that show H1′–H2′
TOCSY cross peaks in RNA are more likely to be in nonhelical regions
of RNA. Again, this experiment was limited to only adenosine using <sup>13</sup>C1′ editing. Out of the four adenosines in the fluorinated
RNA, we observed only two H1′–H2′ TOCSY cross
peaks (<a href="#fig5" class="usa-link">Figure 
</a>D).
These correspond to nucleotides A21 and A20, located in the A-rich
bulge. H1′–H2′ dihedral angles measured for A21
and A20 in the deposited PDB structure (PDB ID <a href="https://www.rcsb.org/structure/1PBR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1PBR</a>) were 155.2°
and 101.2°, respectively. Interestingly, these were the nucleotides
that exhibited the largest CSPs in <a href="#fig2" class="usa-link">Figure 
</a>C,D. This raises
the following question: Is there a link between CSPs observed in <a href="#fig2" class="usa-link">Figure 
</a>C,D and nucleotide flexibility?</p></section><section id="sec3.3"><h3 class="pmc_sec_title">
<sup>13</sup>C and <sup>15</sup>N NMR Spin-Relaxation Experiments</h3>
<p>Given the anticipated flexibility of the bulge, we used our newly
synthesized probe <strong>18</strong> to monitor the RNA dynamics. Such
RNA motions at different time scales are implicated in its various
cellular processes such as ligand sensing and signaling, viral genome
packaging, and catalysis. Longitudinal
(<em>R</em>
<sub>1</sub>) and transverse (<em>R</em>
<sub>2</sub>) relaxation rates measure the rates at which magnetization
decays along the longitudinal and transverse planes, respectively. In addition, <em>R</em>
<sub>2</sub> is
directly proportional to the line width of an NMR signal and the spin-relaxation
rate in a rotating frame (<em>R</em>
<sub>1ρ</sub>).
</p>
<p>While <sup>1</sup>H and <sup>19</sup>F
are very sensitive to external magnetic fields, they become challenging
to work with at high magnetic fields and molecular weights due to
their fast relaxation rates from dipolar interactions and CSA, respectively.
Nonetheless, extensive relaxation experiments have been conducted
on <sup>1</sup>H and <sup>19</sup>F for small protein and RNA constructs.<em>
,
</em>
</p>
<p>The <sup>1</sup>H8–<sup>15</sup>N7 spin pair is isolated
from other active nuclei in our site-specific label, making it an
attractive system to probe. Our simulations for an RNA with a correlation
time of 5 ns at 600 MHz showed that an isolated <sup>1</sup>H1′–<sup>13</sup>C1′ spin pair exhibits ∼2-fold broader line
widths for <sup>13</sup>C1′ nuclei compared with <sup>15</sup>N7 nuclei in an isolated <sup>1</sup>H8–<sup>15</sup>N7 spin
pair (<a href="#fig6" class="usa-link">Figure 
</a>A). In agreement with our simulations, the
average experimental <sup>15</sup>N line width was ∼2×
narrower than <sup>13</sup>C (<a href="#fig6" class="usa-link">Figure 
</a>B). Additionally, we observed attenuated <em>R</em>
<sub>1ρ</sub> values for A21 and A20 in both <sup>13</sup>C
and <sup>15</sup>N <em>R</em>
<sub>1ρ</sub> experiments,
indicating increased internal motions within both nucleotides. We
observed the same trends for <em>R</em>
<sub>1ρ</sub> in both the <sup>15</sup>N7 and <sup>13</sup>C1′ relaxation
rates (<a href="#fig6" class="usa-link">Figure 
</a>B).
The measured <sup>13</sup>C1′ <em>R</em>
<sub>1ρ</sub> values were identical for corresponding adenosines in the fluorinated
and nonfluorinated RNAs (<a href="#fig6" class="usa-link">Figure 
</a>B), except for A21.</p>
<p>Interestingly, there was no significant difference
between measured <em>R</em>
<sub>1</sub> values (<a href="#fig6" class="usa-link">Figure 
</a>C). Like <em>R</em>
<sub>2</sub>, the attenuated <em>R</em>
<sub>2</sub>/<em>R</em>
<sub>1</sub> ratios indicate
fast local motions, with the added benefit of removing variations
in local CSAs and dipolar interactions. Again, both A20 and A21 had
the smallest R<sub>2</sub>/<em>R</em>
<sub>1</sub> values,
indicating fast motions within those nucleotides (<a href="#fig6" class="usa-link">Figure 
</a>D).</p></section></section><section id="sec4"><h2 class="pmc_sec_title">Discussion</h2>
<p>
<sup>1</sup>H NMR is routinely used for
structural studies due
to its high sensitivity to magnetic fields. However, the use of <sup>1</sup>H NMR in biomolecular systems is limited owing to low chemical
shift dispersion and line broadening. <sup>19</sup>F, on the other
hand, has better chemical shift dispersion in addition to its high
sensitivity. Unfortunately, the high CSA of <sup>19</sup>F also limits
its application in larger biomolecules at high magnetic fields due
to large line broadening. Conversely, <sup>13</sup>C and <sup>15</sup>N have lower gyromagnetic ratios, which render them less sensitive
to external magnetic fields, yet possess sharper NMR line widths.
A chemical probe that can establish a “middle ground”
by leveraging the benefits of all three nuclei shows promise to be
versatile for studying both small and large RNAs.</p>
<p>First, we
demonstrated in a series of NMR experiments that the
incorporation of <sup>19</sup>F into RNA is not completely innocuous.
While we do not observe significant changes in thermodynamic properties
between <strong>18</strong>- and <strong>19</strong>-labeled RNA, we observe
these changes in the magnetic properties of nuclei proximal and distal
to the site of fluorination. Furthermore, these changes are not limited
to the fluorinated nucleotide but are also seen in neighboring nuclei,
as evidenced by the <sup>1</sup>H–<sup>1</sup>H TOCSY experiment.
Additionally, <sup>19</sup>F can simplify an otherwise highly overlapped
spectrum. Consequently, we showcase the ability of <sup>19</sup>F
to detect the binding of cognate ligands to RNA by monitoring the
CSPs. It is worth noting that the application of <sup>19</sup>F is
not limited to only small RNAs, as recent studies have shown that
installing <sup>13</sup>C–<sup>19</sup>F pairs in RNA nucleobases
reduces the directly attached <sup>13</sup>C line width by a factor
of 2–5, allowing its application to large RNAs.<em>
,
</em> A suite of elegant experiments has also been designed using <sup>19</sup>F as a probe to measure various protein dynamics properties
that can be implemented in RNA studies.
</p>
<p>Second, we demonstrated the application of the site-specific
label
to simplify an otherwise overlapped <sup>1</sup>H–<sup>1</sup>H NOESY spectrum. We observed all adenosine intranucleotide H8–H1′
cross peaks in both <sup>13</sup>C- and <sup>15</sup>N-edited NOESY
experiments that were subsequently used to confirm preliminary chemical
shift assignments. In addition, we observed TOCSY cross peaks for
H1′- H2′ correlations in the <sup>13</sup>C-edited TOCSY
HSQC experiment. The presence/absence of these correlations provides
structural information about the ribose sugar pucker. Taken together,
our labeling scheme enables complementary probing of structural features
within the RNA in the 5′ to 3′ or 3′ to 5′
direction.</p>
<p>Finally, we performed spin-relaxation experiments
on the <strong>18</strong>-labeled RNA. We obtained ∼2-fold sharper
lines for <sup>15</sup>N7 compared with <sup>13</sup>C1′ from
our simulations
for an RNA tumbling at 5 ns at a magnetic field strength of 600 MHz.
Our experimental results corroborated this prediction, where we measured
∼2-fold increase in the average line widths of <sup>13</sup>C1′ compared with <sup>15</sup>N7, under the same experimental
conditions. We also show that there is no significant difference in
the transverse relaxation rates for the fluorinated and nonfluorinated
RNAs, except for nucleotide A21.</p>
<p>Taken together, we have demonstrated
the utility of the novel triple
labeling pattern to obtain invaluable structural and dynamics information
on a 27-mer RNA. Our data also suggest that there may be a link between
chemical shift changes induced by <sup>19</sup>F incorporation and
nucleotide flexibility for the RNA system we investigated. While we
did not fully leverage the capability of <sup>19</sup>F in this study,
we intend to do so by enriching the C2 position with <sup>13</sup>C, as reported recently. This, in turn,
should produce very sharp lines in <sup>13</sup>C as the two relaxation
mechanisms of dipole–dipole relaxation and chemical shielding
anisotropy almost cancel out.
</p></section><section id="sec18"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371485/bin/bg5c00075_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00075_si_001.pdf</a><sup> (2.3MB, pdf) </sup>
</div></div></section></section><section id="ACK-d14e2134-autogenerated" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank
Prof David A. Case (Rutgers Univ) for providing us
with the chemical shielding tensors we used in our simulations.</p></section><section id="notes-1"><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00075?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00075</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Detailed description of chemical synthesis and characterization
data; additional NMR data for RNA; additional resonance assignment
strategies; chemical shift tensors and atomic coordinates for theoretical
simulations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00075/suppl_file/bg5c00075_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes-2"><p>S.K.A.: Conceptualization,
investigation, writing. L.T.O.: Conceptualization, investigation,
and writing. F.P.S.: Investigation and writing. T.K.D.: Conceptualization,
investigation, writing, and supervision.</p></section><section id="notes-3"><p>We acknowledge
support from NIH (U54 AI17660-TKD) and NSF (DBI1040158-TKD).</p></section><section id="notes-4"><p>The authors
declare no competing financial interest.</p></section><section id="spn1"><p>Published
as part of <em>ACS Bio &amp; Med Chem Au</em>
<a href="https://pubs.acs.org/curated-content?journal=abmcb8&amp;ref=feature" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">special issue</a> “Juneteenth 2025”.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Fürtig B., Richter C., Wöhnert J., Schwalbe H.. NMR Spectroscopy of
RNA. ChemBioChem. 2003;4(10):936–962. doi: 10.1002/cbic.200300700.</cite> [<a href="https://doi.org/10.1002/cbic.200300700" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14523911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemBioChem&amp;title=NMR%20Spectroscopy%20of%20RNA&amp;author=B.%20F%C3%BCrtig&amp;author=C.%20Richter&amp;author=J.%20W%C3%B6hnert&amp;author=H.%20Schwalbe&amp;volume=4&amp;issue=10&amp;publication_year=2003&amp;pages=936-962&amp;pmid=14523911&amp;doi=10.1002/cbic.200300700&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Dethoff E. A., Chugh J., Mustoe A. M., Al-Hashimi H. M.. Functional
Complexity and Regulation through RNA Dynamics. Nature. 2012;482(7385):322–330. doi: 10.1038/nature10885.</cite> [<a href="https://doi.org/10.1038/nature10885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3320162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22337051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional%20Complexity%20and%20Regulation%20through%20RNA%20Dynamics&amp;author=E.%20A.%20Dethoff&amp;author=J.%20Chugh&amp;author=A.%20M.%20Mustoe&amp;author=H.%20M.%20Al-Hashimi&amp;volume=482&amp;issue=7385&amp;publication_year=2012&amp;pages=322-330&amp;pmid=22337051&amp;doi=10.1038/nature10885&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Sekhar A., Kay L. E.. NMR Paves the Way for Atomic Level Descriptions of
Sparsely Populated, Transiently Formed Biomolecular Conformers. Proc. Natl. Acad. Sci. U.S.A. 2013;110(32):12867–12874. doi: 10.1073/pnas.1305688110.</cite> [<a href="https://doi.org/10.1073/pnas.1305688110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3740838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23868852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=NMR%20Paves%20the%20Way%20for%20Atomic%20Level%20Descriptions%20of%20Sparsely%20Populated,%20Transiently%20Formed%20Biomolecular%20Conformers&amp;author=A.%20Sekhar&amp;author=L.%20E.%20Kay&amp;volume=110&amp;issue=32&amp;publication_year=2013&amp;pages=12867-12874&amp;pmid=23868852&amp;doi=10.1073/pnas.1305688110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Ganser L. R., Kelly M. L., Herschlag D., Al-Hashimi H. M.. The Roles
of Structural Dynamics in the Cellular Functions of RNAs. Nat. Rev. Mol. Cell Biol. 2019;20(8):474–489. doi: 10.1038/s41580-019-0136-0.</cite> [<a href="https://doi.org/10.1038/s41580-019-0136-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7656661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31182864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=The%20Roles%20of%20Structural%20Dynamics%20in%20the%20Cellular%20Functions%20of%20RNAs&amp;author=L.%20R.%20Ganser&amp;author=M.%20L.%20Kelly&amp;author=D.%20Herschlag&amp;author=H.%20M.%20Al-Hashimi&amp;volume=20&amp;issue=8&amp;publication_year=2019&amp;pages=474-489&amp;pmid=31182864&amp;doi=10.1038/s41580-019-0136-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Marušič M., Schlagnitweit J., Petzold K.. RNA Dynamics by NMR Spectroscopy. ChemBioChem. 2019;20(21):2685–2710. doi: 10.1002/cbic.201900072.</cite> [<a href="https://doi.org/10.1002/cbic.201900072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6899578/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30997719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemBioChem&amp;title=RNA%20Dynamics%20by%20NMR%20Spectroscopy&amp;author=M.%20Maru%C5%A1i%C4%8D&amp;author=J.%20Schlagnitweit&amp;author=K.%20Petzold&amp;volume=20&amp;issue=21&amp;publication_year=2019&amp;pages=2685-2710&amp;pmid=30997719&amp;doi=10.1002/cbic.201900072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Varani G., Aboul-ela F., Allain F. H.-T.. NMR Investigation of RNA Structure. Prog Nucl Magn Reson Spectrosc. 1996;29:51–127. doi: 10.1016/0079-6565(96)01028-X.</cite> [<a href="https://doi.org/10.1016/0079-6565(96)01028-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog%20Nucl%20Magn%20Reson%20Spectrosc&amp;title=NMR%20Investigation%20of%20RNA%20Structure&amp;author=G.%20Varani&amp;author=F.%20Aboul-ela&amp;author=F.%20H.-T.%20Allain&amp;volume=29&amp;publication_year=1996&amp;pages=51-127&amp;doi=10.1016/0079-6565(96)01028-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Keyhani S., Goldau T., Blümler A., Heckel A., Schwalbe H.. Chemo-Enzymatic
Synthesis of Position-Specifically Modified RNA for Biophysical Studies
Including Light Control and NMR Spectroscopy. Angew. Chem., Int. Ed. 2018;57(37):12017–12021. doi: 10.1002/anie.201807125.</cite> [<a href="https://doi.org/10.1002/anie.201807125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30007102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=Chemo-Enzymatic%20Synthesis%20of%20Position-Specifically%20Modified%20RNA%20for%20Biophysical%20Studies%20Including%20Light%20Control%20and%20NMR%20Spectroscopy&amp;author=S.%20Keyhani&amp;author=T.%20Goldau&amp;author=A.%20Bl%C3%BCmler&amp;author=A.%20Heckel&amp;author=H.%20Schwalbe&amp;volume=57&amp;issue=37&amp;publication_year=2018&amp;pages=12017-12021&amp;pmid=30007102&amp;doi=10.1002/anie.201807125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Dayie K. T.. Key Labeling
Technologies to Tackle Sizeable Problems in RNA Structural Biology. Int. J. Mol. Sci. 2008;9(7):1214–1240. doi: 10.3390/ijms9071214.</cite> [<a href="https://doi.org/10.3390/ijms9071214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2635727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19325801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Key%20Labeling%20Technologies%20to%20Tackle%20Sizeable%20Problems%20in%20RNA%20Structural%20Biology&amp;author=K.%20T.%20Dayie&amp;volume=9&amp;issue=7&amp;publication_year=2008&amp;pages=1214-1240&amp;pmid=19325801&amp;doi=10.3390/ijms9071214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Becette O., Olenginski L. T., Dayie T. K.. Solid-Phase Chemical Synthesis of
Stable Isotope-Labeled RNA to Aid Structure and Dynamics Studies by
NMR Spectroscopy. Molecules. 2019;24(19):3476.  doi: 10.3390/molecules24193476.</cite> [<a href="https://doi.org/10.3390/molecules24193476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6804060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31557861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Solid-Phase%20Chemical%20Synthesis%20of%20Stable%20Isotope-Labeled%20RNA%20to%20Aid%20Structure%20and%20Dynamics%20Studies%20by%20NMR%20Spectroscopy&amp;author=O.%20Becette&amp;author=L.%20T.%20Olenginski&amp;author=T.%20K.%20Dayie&amp;volume=24&amp;issue=19&amp;publication_year=2019&amp;pages=3476&amp;pmid=31557861&amp;doi=10.3390/molecules24193476&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Asadi-Atoi P., Barraud P., Tisne C., Kellner S.. Benefits of Stable
Isotope Labeling in RNA Analysis. Biol. Chem. 2019;400:847–865. doi: 10.1515/hsz-2018-0447.</cite> [<a href="https://doi.org/10.1515/hsz-2018-0447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30893050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Chem.&amp;title=Benefits%20of%20Stable%20Isotope%20Labeling%20in%20RNA%20Analysis&amp;author=P.%20Asadi-Atoi&amp;author=P.%20Barraud&amp;author=C.%20Tisne&amp;author=S.%20Kellner&amp;volume=400&amp;publication_year=2019&amp;pages=847-865&amp;pmid=30893050&amp;doi=10.1515/hsz-2018-0447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>

Le, M. T.
; 
Brown, R. E.
; 
Simon, A. E.
; 
Dayie, T. K.
. 
In Vivo, Large-Scale
Preparation of Uniformly <sup>15</sup>N- and Site-Specifically <sup>13</sup>C-Labeled Homogeneous, Recombinant RNA for NMR Studies. In Methods in Enzymology; Elsevier, 2015; Vol. 565, pp 495–535.
</cite> [<a href="https://doi.org/10.1016/bs.mie.2015.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26577743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;publication_year=2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Lu K., Miyazaki Y., Summers M. F.. Isotope Labeling Strategies for NMR
Studies of RNA. J. Biomol. NMR. 2010;46(1):113–125. doi: 10.1007/s10858-009-9375-2.</cite> [<a href="https://doi.org/10.1007/s10858-009-9375-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2797625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19789981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20NMR&amp;title=Isotope%20Labeling%20Strategies%20for%20NMR%20Studies%20of%20RNA&amp;author=K.%20Lu&amp;author=Y.%20Miyazaki&amp;author=M.%20F.%20Summers&amp;volume=46&amp;issue=1&amp;publication_year=2010&amp;pages=113-125&amp;pmid=19789981&amp;doi=10.1007/s10858-009-9375-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Liu Y., Holmstrom E., Zhang J., Yu P., Wang J., Dyba M. A., Chen D., Ying J., Lockett S., Nesbitt D. J., Ferré-D’Amaré A. R., Sousa R., Stagno J. R., Wang Y. X.. Synthesis and Applications
of RNAs with Position-Selective Labelling and Mosaic Composition. Nature. 2015;522(7556):368–372. doi: 10.1038/nature14352.</cite> [<a href="https://doi.org/10.1038/nature14352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4800989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25938715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Synthesis%20and%20Applications%20of%20RNAs%20with%20Position-Selective%20Labelling%20and%20Mosaic%20Composition&amp;author=Y.%20Liu&amp;author=E.%20Holmstrom&amp;author=J.%20Zhang&amp;author=P.%20Yu&amp;author=J.%20Wang&amp;volume=522&amp;issue=7556&amp;publication_year=2015&amp;pages=368-372&amp;pmid=25938715&amp;doi=10.1038/nature14352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>

Alvarado, L. J.
; 
Longhini, A. P.
; 
LeBlanc, R. M.
; 
Chen, B.
; 
Kreutz, C.
; 
Dayie, T.
K.
. 
Chemo-Enzymatic
Synthesis of Selectively <sup>13</sup>C/<sup>15</sup>N-Labeled RNA
for NMR Structural and Dynamics Studies. In Methods in Enzymology; Elsevier, 2014; Vol. 549, pp 133–162.
</cite> [<a href="https://doi.org/10.1016/B978-0-12-801122-5.00007-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5879494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25432748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>

Scott, L. G.
; 
Hennig, M.
. 

<sup>19</sup>F-Site-Specific-Labeled
Nucleotides for Nucleic Acid Structural Analysis by NMR. In Methods in Enzymology; Elsevier, 2016; Vol. 566, pp 59–87.
</cite> [<a href="https://doi.org/10.1016/bs.mie.2015.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26791976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Marchant J., Bax A., Summers M. F.. Accurate
Measurement of Residual Dipolar Couplings
in Large RNAs by Variable Flip Angle NMR. J.
Am. Chem. Soc. 2018;140(22):6978–6983. doi: 10.1021/jacs.8b03298.</cite> [<a href="https://doi.org/10.1021/jacs.8b03298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6021016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29757635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Accurate%20Measurement%20of%20Residual%20Dipolar%20Couplings%20in%20Large%20RNAs%20by%20Variable%20Flip%20Angle%20NMR&amp;author=J.%20Marchant&amp;author=A.%20Bax&amp;author=M.%20F.%20Summers&amp;volume=140&amp;issue=22&amp;publication_year=2018&amp;pages=6978-6983&amp;pmid=29757635&amp;doi=10.1021/jacs.8b03298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Schnieders R., Richter C., Warhaut S., de Jesus V., Keyhani S., Duchardt-Ferner E., Keller H., Wöhnert J., Kuhn L. T., Breeze A. L., Bermel W., Schwalbe H., Fürtig B.. Evaluation
of 15N-Detected H–N Correlation
Experiments on Increasingly Large RNAs. J. Biomol.
NMR. 2017;69(1):31–44. doi: 10.1007/s10858-017-0132-7.</cite> [<a href="https://doi.org/10.1007/s10858-017-0132-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28879611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20NMR&amp;title=Evaluation%20of%2015N-Detected%20H%E2%80%93N%20Correlation%20Experiments%20on%20Increasingly%20Large%20RNAs&amp;author=R.%20Schnieders&amp;author=C.%20Richter&amp;author=S.%20Warhaut&amp;author=V.%20de%20Jesus&amp;author=S.%20Keyhani&amp;volume=69&amp;issue=1&amp;publication_year=2017&amp;pages=31-44&amp;pmid=28879611&amp;doi=10.1007/s10858-017-0132-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>

Cross, T. A.


<sup>15</sup>N NMR. In Studies in Physical and Theoretical
Chemistry; Elsevier, 1998; Vol. 84, pp 218–235.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Studies%20in%20Physical%20and%20Theoretical%20Chemistry&amp;publication_year=1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Scott L. G., Geierstanger B. H., Williamson J. R., Hennig M.. Enzymatic Synthesis
and 19F NMR Studies of 2-Fluoroadenine- Substituted RNA. J. Am. Chem. Soc. 2004;126(38):11776–11777. doi: 10.1021/ja047556x.</cite> [<a href="https://doi.org/10.1021/ja047556x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15382896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Enzymatic%20Synthesis%20and%2019F%20NMR%20Studies%20of%202-Fluoroadenine-%20Substituted%20RNA&amp;author=L.%20G.%20Scott&amp;author=B.%20H.%20Geierstanger&amp;author=J.%20R.%20Williamson&amp;author=M.%20Hennig&amp;volume=126&amp;issue=38&amp;publication_year=2004&amp;pages=11776-11777&amp;pmid=15382896&amp;doi=10.1021/ja047556x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>

Rastinejad, F.
; 
Evilia, C.
; 
Lu, P.
. 
Studies of Nucleic
Acids and Their Protein Interactions by <sup>19</sup>F NMR. In Methods in Enzymology; Elsevier, 1995; Vol. 261, pp 560–575.
</cite> [<a href="https://doi.org/10.1016/s0076-6879(95)61025-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8569512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;publication_year=1995&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Boeszoermenyi A., Chhabra S., Dubey A., Radeva D. L., Burdzhiev N. T., Chanev C. D., Petrov O. I., Gelev V. M., Zhang M., Anklin C., Kovacs H., Wagner G., Kuprov I., Takeuchi K., Arthanari H.. Aromatic 19F-13C TROSY: A Background-Free
Approach to Probe Biomolecular Structure, Function, and Dynamics. Nat. Methods. 2019;16(4):333–340. doi: 10.1038/s41592-019-0334-x.</cite> [<a href="https://doi.org/10.1038/s41592-019-0334-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6549241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30858598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Aromatic%2019F-13C%20TROSY:%20A%20Background-Free%20Approach%20to%20Probe%20Biomolecular%20Structure,%20Function,%20and%20Dynamics&amp;author=A.%20Boeszoermenyi&amp;author=S.%20Chhabra&amp;author=A.%20Dubey&amp;author=D.%20L.%20Radeva&amp;author=N.%20T.%20Burdzhiev&amp;volume=16&amp;issue=4&amp;publication_year=2019&amp;pages=333-340&amp;pmid=30858598&amp;doi=10.1038/s41592-019-0334-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Dayie T. K., Olenginski L. T., Taiwo K. M.. Isotope labels combined
with solution
NMR spectroscopy make visible the invisible conformations of small-to-large
RNAs. Chem. Rev. 2022;122:9357–9394. doi: 10.1021/acs.chemrev.1c00845.</cite> [<a href="https://doi.org/10.1021/acs.chemrev.1c00845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9136934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35442658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=Isotope%20labels%20combined%20with%20solution%20NMR%20spectroscopy%20make%20visible%20the%20invisible%20conformations%20of%20small-to-large%20RNAs&amp;author=T.%20K.%20Dayie&amp;author=L.%20T.%20Olenginski&amp;author=K.%20M.%20Taiwo&amp;volume=122&amp;publication_year=2022&amp;pages=9357-9394&amp;pmid=35442658&amp;doi=10.1021/acs.chemrev.1c00845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Olenginski L. T., Dayie T. K.. Chemo-Enzymatic
Synthesis of [2–13C, 7–15
N]-ATP for Facile NMR Analysis of RNA. Monatsh.
Chem. 2020;151(9):1467–1473. doi: 10.1007/s00706-020-02667-6.</cite> [<a href="https://doi.org/10.1007/s00706-020-02667-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Monatsh.%20Chem.&amp;title=Chemo-Enzymatic%20Synthesis%20of%20%5B2%E2%80%9313C,%207%E2%80%9315%20N%5D-ATP%20for%20Facile%20NMR%20Analysis%20of%20RNA&amp;author=L.%20T.%20Olenginski&amp;author=T.%20K.%20Dayie&amp;volume=151&amp;issue=9&amp;publication_year=2020&amp;pages=1467-1473&amp;doi=10.1007/s00706-020-02667-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Bodenhausen G., Ruben D. J.. Natural Abundance Nitrogen-15 NMR
by Enhanced Heteronuclear
Spectroscopy. Chem. Phys. Lett. 1980;69(1):185–189. doi: 10.1016/0009-2614(80)80041-8.</cite> [<a href="https://doi.org/10.1016/0009-2614(80)80041-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Lett.&amp;title=Natural%20Abundance%20Nitrogen-15%20NMR%20by%20Enhanced%20Heteronuclear%20Spectroscopy&amp;author=G.%20Bodenhausen&amp;author=D.%20J.%20Ruben&amp;volume=69&amp;issue=1&amp;publication_year=1980&amp;pages=185-189&amp;doi=10.1016/0009-2614(80)80041-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Ikura M., Kay L. E., Tschudin R., Bax A.. Three-Dimensional NOESY-HMQC
Spectroscopy of a 13C-Labeled Protein. J. Magn. Reson. 1990;86(1):204–209. doi: 10.1016/0022-2364(90)90227-Z.</cite> [<a href="https://doi.org/10.1016/0022-2364(90)90227-Z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.&amp;title=Three-Dimensional%20NOESY-HMQC%20Spectroscopy%20of%20a%2013C-Labeled%20Protein&amp;author=M.%20Ikura&amp;author=L.%20E.%20Kay&amp;author=R.%20Tschudin&amp;author=A.%20Bax&amp;volume=86&amp;issue=1&amp;publication_year=1990&amp;pages=204-209&amp;doi=10.1016/0022-2364(90)90227-Z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>LeBlanc R. M., Longhini A. P., Tugarinov V., Dayie T. K.. NMR probing of invisible
excited states using selectively labeled RNAs. J. Biomol. NMR. 2018;71:165–172. doi: 10.1007/s10858-018-0184-3.</cite> [<a href="https://doi.org/10.1007/s10858-018-0184-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29858959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20NMR&amp;title=NMR%20probing%20of%20invisible%20excited%20states%20using%20selectively%20labeled%20RNAs&amp;author=R.%20M.%20LeBlanc&amp;author=A.%20P.%20Longhini&amp;author=V.%20Tugarinov&amp;author=T.%20K.%20Dayie&amp;volume=71&amp;publication_year=2018&amp;pages=165-172&amp;pmid=29858959&amp;doi=10.1007/s10858-018-0184-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Broom A. D., Amarnath V., Vince R., Brownell J.. Poly­(2-Fluoroadenylic
Acid). The Role of Basicity in the Stabilization of Complementary
Helices. Biochim. Biophys. Acta. 1979;563:508–517. doi: 10.1016/0005-2787(79)90069-8.</cite> [<a href="https://doi.org/10.1016/0005-2787(79)90069-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&amp;title=Poly%C2%AD(2-Fluoroadenylic%20Acid).%20The%20Role%20of%20Basicity%20in%20the%20Stabilization%20of%20Complementary%20Helices&amp;author=A.%20D.%20Broom&amp;author=V.%20Amarnath&amp;author=R.%20Vince&amp;author=J.%20Brownell&amp;volume=563&amp;publication_year=1979&amp;pages=508-517&amp;pmid=37911&amp;doi=10.1016/0005-2787(79)90069-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Battaglia M. R., Buckingham A. D., Williams J. H.. The Electric Quadrupole Moments of
Benzene and Hexafluorobenzene. Chem. Phys. Lett. 1981;78(3):421–423. doi: 10.1016/0009-2614(81)85228-1.</cite> [<a href="https://doi.org/10.1016/0009-2614(81)85228-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Lett.&amp;title=The%20Electric%20Quadrupole%20Moments%20of%20Benzene%20and%20Hexafluorobenzene&amp;author=M.%20R.%20Battaglia&amp;author=A.%20D.%20Buckingham&amp;author=J.%20H.%20Williams&amp;volume=78&amp;issue=3&amp;publication_year=1981&amp;pages=421-423&amp;doi=10.1016/0009-2614(81)85228-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Becette O. B., Zong G., Chen B., Taiwo K. M., Case D. A., Dayie T. K.. Solution NMR Readily Reveals Distinct Structural Folds
and Interactions in Doubly 13 C- and 19 F-Labeled
RNAs. Sci. Adv. 2020;6(41):eabc6572.  doi: 10.1126/sciadv.abc6572.</cite> [<a href="https://doi.org/10.1126/sciadv.abc6572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7541061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33028531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Solution%20NMR%20Readily%20Reveals%20Distinct%20Structural%20Folds%20and%20Interactions%20in%20Doubly%2013%20C-%20and%2019%20F-Labeled%20RNAs&amp;author=O.%20B.%20Becette&amp;author=G.%20Zong&amp;author=B.%20Chen&amp;author=K.%20M.%20Taiwo&amp;author=D.%20A.%20Case&amp;volume=6&amp;issue=41&amp;publication_year=2020&amp;pages=eabc6572&amp;pmid=33028531&amp;doi=10.1126/sciadv.abc6572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Sochor F., Silvers R., Müller D., Richter C., Fürtig B., Schwalbe H.. 
19F-Labeling
of the Adenine H2-Site to Study
Large RNAs by NMR Spectroscopy. J. Biomol. NMR. 2016;64(1):63–74. doi: 10.1007/s10858-015-0006-9.</cite> [<a href="https://doi.org/10.1007/s10858-015-0006-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26704707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20NMR&amp;title=19F-Labeling%20of%20the%20Adenine%20H2-Site%20to%20Study%20Large%20RNAs%20by%20NMR%20Spectroscopy&amp;author=F.%20Sochor&amp;author=R.%20Silvers&amp;author=D.%20M%C3%BCller&amp;author=C.%20Richter&amp;author=B.%20F%C3%BCrtig&amp;volume=64&amp;issue=1&amp;publication_year=2016&amp;pages=63-74&amp;pmid=26704707&amp;doi=10.1007/s10858-015-0006-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Nußbaumer F., Plangger R., Roeck M., Kreutz C.. Aromatic 19F– 13C TROSY[19F, 13C]-Pyrimidine Labeling for NMR Spectroscopy of RNA. Angew. Chem., Int. Ed. 2020;59(39):17062–17069. doi: 10.1002/anie.202006577.</cite> [<a href="https://doi.org/10.1002/anie.202006577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7540360/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32558232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=Aromatic%2019F%E2%80%93%2013C%20TROSY%EE%97%B8%5B19F,%2013C%5D-Pyrimidine%20Labeling%20for%20NMR%20Spectroscopy%20of%20RNA&amp;author=F.%20Nu%C3%9Fbaumer&amp;author=R.%20Plangger&amp;author=M.%20Roeck&amp;author=C.%20Kreutz&amp;volume=59&amp;issue=39&amp;publication_year=2020&amp;pages=17062-17069&amp;pmid=32558232&amp;doi=10.1002/anie.202006577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Hoe W.-M., Cohen A. J., Handy N. C.. Assessment
of a New Local Exchange
Functional OPTX. Chem. Phys. Lett. 2001;341(3–4):319–328. doi: 10.1016/S0009-2614(01)00581-4.</cite> [<a href="https://doi.org/10.1016/S0009-2614(01)00581-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Lett.&amp;title=Assessment%20of%20a%20New%20Local%20Exchange%20Functional%20OPTX&amp;author=W.-M.%20Hoe&amp;author=A.%20J.%20Cohen&amp;author=N.%20C.%20Handy&amp;volume=341&amp;issue=3%E2%80%934&amp;publication_year=2001&amp;pages=319-328&amp;doi=10.1016/S0009-2614(01)00581-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Jensen F.. Segmented
Contracted Basis Sets Optimized for Nuclear Magnetic Shielding. J. Chem. Theory Comput. 2015;11(1):132–138. doi: 10.1021/ct5009526.</cite> [<a href="https://doi.org/10.1021/ct5009526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Segmented%20Contracted%20Basis%20Sets%20Optimized%20for%20Nuclear%20Magnetic%20Shielding&amp;author=F.%20Jensen&amp;volume=11&amp;issue=1&amp;publication_year=2015&amp;pages=132-138&amp;pmid=26574211&amp;doi=10.1021/ct5009526&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Bannwarth C., Ehlert S., Grimme S.. GFN2-XTBAn
Accurate and Broadly
Parametrized Self-Consistent Tight-Binding Quantum Chemical Method
with Multipole Electrostatics and Density-Dependent Dispersion Contributions. J. Chem. Theory Comput. 2019;15(3):1652–1671. doi: 10.1021/acs.jctc.8b01176.</cite> [<a href="https://doi.org/10.1021/acs.jctc.8b01176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30741547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=GFN2-XTB%EE%97%B8An%20Accurate%20and%20Broadly%20Parametrized%20Self-Consistent%20Tight-Binding%20Quantum%20Chemical%20Method%20with%20Multipole%20Electrostatics%20and%20Density-Dependent%20Dispersion%20Contributions&amp;author=C.%20Bannwarth&amp;author=S.%20Ehlert&amp;author=S.%20Grimme&amp;volume=15&amp;issue=3&amp;publication_year=2019&amp;pages=1652-1671&amp;pmid=30741547&amp;doi=10.1021/acs.jctc.8b01176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Attionu S. K., Dill R., Summers M. F., Case D. A., Marchant J., Dayie T. K.. Selective [9–15N] Guanosine for NMR
Studies of Large RNAs. ChemBioChem. 2025:2500206.  doi: 10.1002/cbic.202500206.</cite> [<a href="https://doi.org/10.1002/cbic.202500206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12177700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40320375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemBioChem&amp;title=Selective%20%5B9%E2%80%9315N%5D%20Guanosine%20for%20NMR%20Studies%20of%20Large%20RNAs&amp;author=S.%20K.%20Attionu&amp;author=R.%20Dill&amp;author=M.%20F.%20Summers&amp;author=D.%20A.%20Case&amp;author=J.%20Marchant&amp;publication_year=2025&amp;pages=2500206&amp;pmid=40320375&amp;doi=10.1002/cbic.202500206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Peng J. W., Wagner G.. Mapping of Spectral
Density Functions Using Heteronuclear
NMR Relaxation Measurements. J. Magn. Reson. 1992;98(2):308–332. doi: 10.1016/0022-2364(92)90135-T.</cite> [<a href="https://doi.org/10.1016/0022-2364(92)90135-T" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.&amp;title=Mapping%20of%20Spectral%20Density%20Functions%20Using%20Heteronuclear%20NMR%20Relaxation%20Measurements&amp;author=J.%20W.%20Peng&amp;author=G.%20Wagner&amp;volume=98&amp;issue=2&amp;publication_year=1992&amp;pages=308-332&amp;doi=10.1016/0022-2364(92)90135-T&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>LeBlanc R. M., Kasprzak W. K., Longhini A. P., Olenginski L. T., Abulwerdi F., Ginocchio S., Shields B., Nyman J., Svirydava M., Del Vecchio C., Ivanic J., Schneekloth J. S., Shapiro B. A., Dayie T. K., Le Grice S. F. J.. Structural Insights
of the Conserved “Priming Loop” of Hepatitis B Virus
Pre-Genomic RNA. J. Biomol. Struct. Dyn. 2022;40(20):9761–9773. doi: 10.1080/07391102.2021.1934544.</cite> [<a href="https://doi.org/10.1080/07391102.2021.1934544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10167916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34155954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Structural%20Insights%20of%20the%20Conserved%20%E2%80%9CPriming%20Loop%E2%80%9D%20of%20Hepatitis%20B%20Virus%20Pre-Genomic%20RNA&amp;author=R.%20M.%20LeBlanc&amp;author=W.%20K.%20Kasprzak&amp;author=A.%20P.%20Longhini&amp;author=L.%20T.%20Olenginski&amp;author=F.%20Abulwerdi&amp;volume=40&amp;issue=20&amp;publication_year=2022&amp;pages=9761-9773&amp;pmid=34155954&amp;doi=10.1080/07391102.2021.1934544&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Hogben H. J., Krzystyniak M., Charnock G. T. P., Hore P. J., Kuprov I.. Spinach –
A Software Library for Simulation of Spin Dynamics in Large Spin Systems. J. Magn. Reson. 2011;208(2):179–194. doi: 10.1016/j.jmr.2010.11.008.</cite> [<a href="https://doi.org/10.1016/j.jmr.2010.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21169043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.&amp;title=Spinach%20%E2%80%93%20A%20Software%20Library%20for%20Simulation%20of%20Spin%20Dynamics%20in%20Large%20Spin%20Systems&amp;author=H.%20J.%20Hogben&amp;author=M.%20Krzystyniak&amp;author=G.%20T.%20P.%20Charnock&amp;author=P.%20J.%20Hore&amp;author=I.%20Kuprov&amp;volume=208&amp;issue=2&amp;publication_year=2011&amp;pages=179-194&amp;pmid=21169043&amp;doi=10.1016/j.jmr.2010.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>

Ulrich, H.

High Resolution NMR in
Solids Selective Averaging; Academic Press: New York, 1976.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=High%20Resolution%20NMR%20in%20Solids%20Selective%20Averaging&amp;publication_year=1976&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Harris R. K., Becker E. D., De Menezes S. M. C., Granger P., Hoffman R. E., Zilm K. W.. Further Conventions
for NMR Shielding and Chemical
Shifts (IUPAC Recommendations 2008) Magn. Reson.
Chem. 2008;46(6):582–598. doi: 10.1002/mrc.2225.</cite> [<a href="https://doi.org/10.1002/mrc.2225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18407566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Magn.%20Reson.%20Chem.&amp;title=Further%20Conventions%20for%20NMR%20Shielding%20and%20Chemical%20Shifts%20(IUPAC%20Recommendations%202008)&amp;author=R.%20K.%20Harris&amp;author=E.%20D.%20Becker&amp;author=S.%20M.%20C.%20De%20Menezes&amp;author=P.%20Granger&amp;author=R.%20E.%20Hoffman&amp;volume=46&amp;issue=6&amp;publication_year=2008&amp;pages=582-598&amp;pmid=18407566&amp;doi=10.1002/mrc.2225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Neese F., Wennmohs F., Becker U., Riplinger C.. The ORCA Quantum
Chemistry Program Package. J. Chem. Phys. 2020;152(22):224108.  doi: 10.1063/5.0004608.</cite> [<a href="https://doi.org/10.1063/5.0004608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32534543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=The%20ORCA%20Quantum%20Chemistry%20Program%20Package&amp;author=F.%20Neese&amp;author=F.%20Wennmohs&amp;author=U.%20Becker&amp;author=C.%20Riplinger&amp;volume=152&amp;issue=22&amp;publication_year=2020&amp;pages=224108&amp;pmid=32534543&amp;doi=10.1063/5.0004608&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Neese F.. Software Update:
The ORCA Program SystemVersion 5.0. WIREs Comput. Mol. Biosci. 2022;12(5):e1606.  doi: 10.1002/wcms.1606.</cite> [<a href="https://doi.org/10.1002/wcms.1606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=WIREs%20Comput.%20Mol.%20Biosci.&amp;title=Software%20Update:%20The%20ORCA%20Program%20System%EE%97%B8Version%205.0&amp;author=F.%20Neese&amp;volume=12&amp;issue=5&amp;publication_year=2022&amp;pages=e1606&amp;doi=10.1002/wcms.1606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Swails J., Zhu T., He X., Case D. A.. AFNMR: Automated Fragmentation Quantum
Mechanical Calculation of NMR Chemical Shifts for Biomolecules. J. Biomol. NMR. 2015;63(2):125–139. doi: 10.1007/s10858-015-9970-3.</cite> [<a href="https://doi.org/10.1007/s10858-015-9970-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6556433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26232926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20NMR&amp;title=AFNMR:%20Automated%20Fragmentation%20Quantum%20Mechanical%20Calculation%20of%20NMR%20Chemical%20Shifts%20for%20Biomolecules&amp;author=J.%20Swails&amp;author=T.%20Zhu&amp;author=X.%20He&amp;author=D.%20A.%20Case&amp;volume=63&amp;issue=2&amp;publication_year=2015&amp;pages=125-139&amp;pmid=26232926&amp;doi=10.1007/s10858-015-9970-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Case D. A.. Using Quantum
Chemistry to Estimate Chemical Shifts in Biomolecules. Biophys. Chem. 2020;267:106476.  doi: 10.1016/j.bpc.2020.106476.</cite> [<a href="https://doi.org/10.1016/j.bpc.2020.106476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7686263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33035752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20Chem.&amp;title=Using%20Quantum%20Chemistry%20to%20Estimate%20Chemical%20Shifts%20in%20Biomolecules&amp;author=D.%20A.%20Case&amp;volume=267&amp;publication_year=2020&amp;pages=106476&amp;pmid=33035752&amp;doi=10.1016/j.bpc.2020.106476&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Taiwo K. M., Olenginski L. T., Nußbaumer F., Nam H., Hilber S., Kreutz C., Dayie T. K.. Synthesis of [7–15N]-GTPs for RNA Structure and Dynamics by NMR Spectroscopy. Monatsh. Chem. 2022;153(3):293–299. doi: 10.1007/s00706-022-02892-1.</cite> [<a href="https://doi.org/10.1007/s00706-022-02892-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8948113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35400760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Monatsh.%20Chem.&amp;title=Synthesis%20of%20%5B7%E2%80%9315N%5D-GTPs%20for%20RNA%20Structure%20and%20Dynamics%20by%20NMR%20Spectroscopy&amp;author=K.%20M.%20Taiwo&amp;author=L.%20T.%20Olenginski&amp;author=F.%20Nu%C3%9Fbaumer&amp;author=H.%20Nam&amp;author=S.%20Hilber&amp;volume=153&amp;issue=3&amp;publication_year=2022&amp;pages=293-299&amp;pmid=35400760&amp;doi=10.1007/s00706-022-02892-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Longhini A. P., LeBlanc R. M., Becette O., Salguero C., Wunderlich C. H., Johnson B. A., D’Souza V. M., Kreutz C., Dayie T. K.. Chemo-Enzymatic
Synthesis of Site-Specific Isotopically Labeled Nucleotides for Use
in NMR Resonance Assignment, Dynamics and Structural Characterizations. Nucleic Acids Res. 2016;44(6):e52.  doi: 10.1093/nar/gkv1333.</cite> [<a href="https://doi.org/10.1093/nar/gkv1333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4824079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26657632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Chemo-Enzymatic%20Synthesis%20of%20Site-Specific%20Isotopically%20Labeled%20Nucleotides%20for%20Use%20in%20NMR%20Resonance%20Assignment,%20Dynamics%20and%20Structural%20Characterizations&amp;author=A.%20P.%20Longhini&amp;author=R.%20M.%20LeBlanc&amp;author=O.%20Becette&amp;author=C.%20Salguero&amp;author=C.%20H.%20Wunderlich&amp;volume=44&amp;issue=6&amp;publication_year=2016&amp;pages=e52&amp;pmid=26657632&amp;doi=10.1093/nar/gkv1333&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Alvarado L. J., LeBlanc R. M., Longhini A. P., Keane S. C., Jain N., Yildiz Z. F., Tolbert B. S., D’Souza V. M., Summers M. F., Kreutz C., Dayie T. K.. Regio-Selective
Chemical-Enzymatic Synthesis of Pyrimidine Nucleotides Facilitates
RNA Structure and Dynamics Studies. ChemBioChem. 2014;15(11):1573–1577. doi: 10.1002/cbic.201402130.</cite> [<a href="https://doi.org/10.1002/cbic.201402130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4127085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24954297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemBioChem&amp;title=Regio-Selective%20Chemical-Enzymatic%20Synthesis%20of%20Pyrimidine%20Nucleotides%20Facilitates%20RNA%20Structure%20and%20Dynamics%20Studies&amp;author=L.%20J.%20Alvarado&amp;author=R.%20M.%20LeBlanc&amp;author=A.%20P.%20Longhini&amp;author=S.%20C.%20Keane&amp;author=N.%20Jain&amp;volume=15&amp;issue=11&amp;publication_year=2014&amp;pages=1573-1577&amp;pmid=24954297&amp;doi=10.1002/cbic.201402130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Arthur P. K., Alvarado L. J., Dayie T. K.. Expression, Purification and Analysis
of the Activity of Enzymes from the Pentose Phosphate Pathway. Protein Expression Purif. 2011;76(2):229–237. doi: 10.1016/j.pep.2010.11.008.</cite> [<a href="https://doi.org/10.1016/j.pep.2010.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4125576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21111048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expression%20Purif.&amp;title=Expression,%20Purification%20and%20Analysis%20of%20the%20Activity%20of%20Enzymes%20from%20the%20Pentose%20Phosphate%20Pathway&amp;author=P.%20K.%20Arthur&amp;author=L.%20J.%20Alvarado&amp;author=T.%20K.%20Dayie&amp;volume=76&amp;issue=2&amp;publication_year=2011&amp;pages=229-237&amp;pmid=21111048&amp;doi=10.1016/j.pep.2010.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>

Petrov, A.
; 
Wu, T.
; 
Puglisi, E. V.
; 
Puglisi, J. D.
. 
RNA Purification by Preparative
Polyacrylamide Gel Electrophoresis. In Methods
in Enzymology; Elsevier, 2013; Vol. 530, pp 315–330.
</cite> [<a href="https://doi.org/10.1016/B978-0-12-420037-1.00017-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24034329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Dayie K. T., Wagner G.. Relaxation-Rate Measurements
for 15N–1H Groups with Pulsed-Field Gradients
and Preservation of Coherence
Pathways. J. Magn. Reson., Ser. A. 1994;111(1):121–126. doi: 10.1006/jmra.1994.1236.</cite> [<a href="https://doi.org/10.1006/jmra.1994.1236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.,%20Ser.%20A&amp;title=Relaxation-Rate%20Measurements%20for%2015N%E2%80%931H%20Groups%20with%20Pulsed-Field%20Gradients%20and%20Preservation%20of%20Coherence%20Pathways&amp;author=K.%20T.%20Dayie&amp;author=G.%20Wagner&amp;volume=111&amp;issue=1&amp;publication_year=1994&amp;pages=121-126&amp;doi=10.1006/jmra.1994.1236&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Luteran E. M., Kahn J. D., Paukstelis P. J.. Stability of the PH-Dependent Parallel-Stranded
d­(CGA) Motif. Biophys. J. 2020;119(8):1580–1589. doi: 10.1016/j.bpj.2020.09.002.</cite> [<a href="https://doi.org/10.1016/j.bpj.2020.09.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7642291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32966760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Stability%20of%20the%20PH-Dependent%20Parallel-Stranded%20d%C2%AD(CGA)%20Motif&amp;author=E.%20M.%20Luteran&amp;author=J.%20D.%20Kahn&amp;author=P.%20J.%20Paukstelis&amp;volume=119&amp;issue=8&amp;publication_year=2020&amp;pages=1580-1589&amp;pmid=32966760&amp;doi=10.1016/j.bpj.2020.09.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>

Siegfried, N. A.
; 
Bevilacqua, P. C.
. 
Thinking Inside
the Box: Designing, Implementing, and Interpreting Thermodynamic Cycles
to Dissect Cooperativity in RNA and DNA Folding. In Methods in Enzymology; Elsevier, 2009; Vol. 455, pp 365–393.
</cite> [<a href="https://doi.org/10.1016/S0076-6879(08)04213-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19289213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Juen F., Glänzer D., Plangger R., Kugler V., Fleischmann J., Stefan E., Case D. A., Kovacs H., Kwaku Dayie T., Kreutz C.. Enhanced TROSY Effect in [2-19F, 2-13C] Adenosine and ATP Analogs Facilitates NMR Spectroscopy
of Very Large Biological RNAs in Solution. Angew.
Chem., Int. Ed. 2024;63(9):e202316273.  doi: 10.1002/anie.202316273.</cite> [<a href="https://doi.org/10.1002/anie.202316273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10922520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38185473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=Enhanced%20TROSY%20Effect%20in%20%5B2-19F,%202-13C%5D%20Adenosine%20and%20ATP%20Analogs%20Facilitates%20NMR%20Spectroscopy%20of%20Very%20Large%20Biological%20RNAs%20in%20Solution&amp;author=F.%20Juen&amp;author=D.%20Gl%C3%A4nzer&amp;author=R.%20Plangger&amp;author=V.%20Kugler&amp;author=J.%20Fleischmann&amp;volume=63&amp;issue=9&amp;publication_year=2024&amp;pages=e202316273&amp;pmid=38185473&amp;doi=10.1002/anie.202316273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>LeBlanc R. M., Longhini A. P., Le Grice S. F. J., Johnson B. A., Dayie T. K.. Combining
Asymmetric 13C-Labeling and Isotopic Filter/Edit NOESY:
A Novel Strategy for Rapid and Logical RNA Resonance Assignment. Nucleic Acids Res. 2017;45(16):e146.  doi: 10.1093/nar/gkx591.</cite> [<a href="https://doi.org/10.1093/nar/gkx591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5766159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28934505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Combining%20Asymmetric%2013C-Labeling%20and%20Isotopic%20Filter/Edit%20NOESY:%20A%20Novel%20Strategy%20for%20Rapid%20and%20Logical%20RNA%20Resonance%20Assignment&amp;author=R.%20M.%20LeBlanc&amp;author=A.%20P.%20Longhini&amp;author=S.%20F.%20J.%20Le%20Grice&amp;author=B.%20A.%20Johnson&amp;author=T.%20K.%20Dayie&amp;volume=45&amp;issue=16&amp;publication_year=2017&amp;pages=e146&amp;pmid=28934505&amp;doi=10.1093/nar/gkx591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Fourmy D., Recht M. I., Blanchard S. C., Puglisi J. D.. Structure of the
A Site of 16 S Ribosomal RNA Complexed with an Aminoglycoside Antibiotic. Science. 1996;274(5291):1367–1371. doi: 10.1126/science.274.5291.1367.</cite> [<a href="https://doi.org/10.1126/science.274.5291.1367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8910275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structure%20of%20the%20A%20Site%20of%2016%20S%20Ribosomal%20RNA%20Complexed%20with%20an%20Aminoglycoside%20Antibiotic&amp;author=D.%20Fourmy&amp;author=M.%20I.%20Recht&amp;author=S.%20C.%20Blanchard&amp;author=J.%20D.%20Puglisi&amp;volume=274&amp;issue=5291&amp;publication_year=1996&amp;pages=1367-1371&amp;pmid=8910275&amp;doi=10.1126/science.274.5291.1367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Tsai A., Uemura S., Johansson M., Puglisi E. V., Marshall R. A., Aitken C. E., Korlach J., Ehrenberg M., Puglisi J. D.. The Impact of Aminoglycosides on
the Dynamics of Translation
Elongation. Cell Rep. 2013;3(2):497–508. doi: 10.1016/j.celrep.2013.01.027.</cite> [<a href="https://doi.org/10.1016/j.celrep.2013.01.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3766726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23416053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=The%20Impact%20of%20Aminoglycosides%20on%20the%20Dynamics%20of%20Translation%20Elongation&amp;author=A.%20Tsai&amp;author=S.%20Uemura&amp;author=M.%20Johansson&amp;author=E.%20V.%20Puglisi&amp;author=R.%20A.%20Marshall&amp;volume=3&amp;issue=2&amp;publication_year=2013&amp;pages=497-508&amp;pmid=23416053&amp;doi=10.1016/j.celrep.2013.01.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Shaka A.J, Lee C.J, Pines A. Iterative schemes
for bilinear operators;
application to spin decoupling. J. Magn. Reson. 1988;77:274–293. doi: 10.1016/0022-2364(88)90178-3.</cite> [<a href="https://doi.org/10.1016/0022-2364(88)90178-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.&amp;title=Iterative%20schemes%20for%20bilinear%20operators;%20application%20to%20spin%20decoupling&amp;author=A.J%20Shaka&amp;author=C.J%20Lee&amp;author=A%20Pines&amp;volume=77&amp;publication_year=1988&amp;pages=274-293&amp;doi=10.1016/0022-2364(88)90178-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Wijmenga S. S., Van Buuren B. N. M.. The
Use of NMR Methods for Conformational Studies of
Nucleic Acids. Prog. Nucl. Magn. Reson. Spectrosc. 1998;32(4):287–387. doi: 10.1016/S0079-6565(97)00023-X.</cite> [<a href="https://doi.org/10.1016/S0079-6565(97)00023-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog.%20Nucl.%20Magn.%20Reson.%20Spectrosc.&amp;title=The%20Use%20of%20NMR%20Methods%20for%20Conformational%20Studies%20of%20Nucleic%20Acids&amp;author=S.%20S.%20Wijmenga&amp;author=B.%20N.%20M.%20Van%20Buuren&amp;volume=32&amp;issue=4&amp;publication_year=1998&amp;pages=287-387&amp;doi=10.1016/S0079-6565(97)00023-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Al-Hashimi H. M., Walter N. G.. RNA Dynamics: It Is about Time. Curr. Opin Struct. Biol. 2008;18(3):321–329. doi: 10.1016/j.sbi.2008.04.004.</cite> [<a href="https://doi.org/10.1016/j.sbi.2008.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2580758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18547802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin%20Struct.%20Biol.&amp;title=RNA%20Dynamics:%20It%20Is%20about%20Time&amp;author=H.%20M.%20Al-Hashimi&amp;author=N.%20G.%20Walter&amp;volume=18&amp;issue=3&amp;publication_year=2008&amp;pages=321-329&amp;pmid=18547802&amp;doi=10.1016/j.sbi.2008.04.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Palmer A. G.. NMR Characterization
of the Dynamics of Biomacromolecules. Chem.
Rev. 2004;104(8):3623–3640. doi: 10.1021/cr030413t.</cite> [<a href="https://doi.org/10.1021/cr030413t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15303831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=NMR%20Characterization%20of%20the%20Dynamics%20of%20Biomacromolecules&amp;author=A.%20G.%20Palmer&amp;volume=104&amp;issue=8&amp;publication_year=2004&amp;pages=3623-3640&amp;pmid=15303831&amp;doi=10.1021/cr030413t&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>

Massi, F.
; 
Peng, J.
W.
. 
Characterizing Protein
Dynamics with NMR R 1ρ Relaxation Experiments. In Methods in Molecular Biology; Springer Nature, 2018; Vol. 1688, pp 205–221.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-7386-6_10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29151211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Molecular%20Biology&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Overbeck J. H., Kremer W., Sprangers R.. A Suite of 19F Based Relaxation
Dispersion Experiments to Assess Biomolecular Motions. J. Biomol. NMR. 2020;74(12):753–766. doi: 10.1007/s10858-020-00348-4.</cite> [<a href="https://doi.org/10.1007/s10858-020-00348-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7701166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32997265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20NMR&amp;title=A%20Suite%20of%2019F%20Based%20Relaxation%20Dispersion%20Experiments%20to%20Assess%20Biomolecular%20Motions&amp;author=J.%20H.%20Overbeck&amp;author=W.%20Kremer&amp;author=R.%20Sprangers&amp;volume=74&amp;issue=12&amp;publication_year=2020&amp;pages=753-766&amp;pmid=32997265&amp;doi=10.1007/s10858-020-00348-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Juen M. A., Wunderlich C. H., Nußbaumer F., Tollinger M., Kontaxis G., Konrat R., Hansen D. F., Kreutz C.. Excited States
of Nucleic Acids Probed by Proton Relaxation Dispersion NMR Spectroscopy. Angew. Chem., Int. Ed. 2016;55(39):12008–12012. doi: 10.1002/anie.201605870.</cite> [<a href="https://doi.org/10.1002/anie.201605870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5082494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27533469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=Excited%20States%20of%20Nucleic%20Acids%20Probed%20by%20Proton%20Relaxation%20Dispersion%20NMR%20Spectroscopy&amp;author=M.%20A.%20Juen&amp;author=C.%20H.%20Wunderlich&amp;author=F.%20Nu%C3%9Fbaumer&amp;author=M.%20Tollinger&amp;author=G.%20Kontaxis&amp;volume=55&amp;issue=39&amp;publication_year=2016&amp;pages=12008-12012&amp;pmid=27533469&amp;doi=10.1002/anie.201605870&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371485/bin/bg5c00075_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00075_si_001.pdf</a><sup> (2.3MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.5c00075"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg5c00075.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371485/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371485/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371485%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371485/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371485/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371485/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371485/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371485/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371485/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="D4YO0aBCosOXuvKTFuCAZutg7PeQb9Iig2HyRa53Amsm7K8od1MG7ujfeh1uaBKh">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
