
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Cryo-EM structure of ribosome from pathogenic protozoa Entamoeba histolytica reveals unique features of its architecture - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152545E58AF31C430545E5004A011052.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368212/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Cryo-EM structure of ribosome from pathogenic protozoa Entamoeba histolytica reveals unique features of its architecture">
<meta name="citation_author" content="Shivani Sharma">
<meta name="citation_author_institution" content="Structural Biology &amp; Translation Regulation laboratory, UNESCO-DBT Regional Centre for Biotechnology, NCR Biotech Science Cluster, Faridabad, Haryana (Delhi-NCR) India">
<meta name="citation_author" content="Shalini Mishra">
<meta name="citation_author_institution" content="Structural Biology Laboratory, School of Life Science, Jawaharlal Nehru University, New Delhi, India">
<meta name="citation_author" content="Samudrala Gourinath">
<meta name="citation_author_institution" content="Structural Biology Laboratory, School of Life Science, Jawaharlal Nehru University, New Delhi, India">
<meta name="citation_author" content="Prem S Kaushal">
<meta name="citation_author_institution" content="Structural Biology &amp; Translation Regulation laboratory, UNESCO-DBT Regional Centre for Biotechnology, NCR Biotech Science Cluster, Faridabad, Haryana (Delhi-NCR) India">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7758">
<meta name="citation_doi" content="10.1038/s41467-025-62767-x">
<meta name="citation_pmid" content="40835601">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368212/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368212/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368212/pdf/41467_2025_Article_62767.pdf">
<meta name="description" content="Entamoeba histolytica, an anaerobic protozoan parasite, is the causative agent of amoebiasis, bloody diarrhea, and liver abscesses in humans. Amoebiasis is more predominant in tropical areas with poor sanitation conditions, and it remains the fourth ...">
<meta name="og:title" content="Cryo-EM structure of ribosome from pathogenic protozoa Entamoeba histolytica reveals unique features of its architecture">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Entamoeba histolytica, an anaerobic protozoan parasite, is the causative agent of amoebiasis, bloody diarrhea, and liver abscesses in humans. Amoebiasis is more predominant in tropical areas with poor sanitation conditions, and it remains the fourth ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368212/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368212">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62767-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62767.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368212%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368212/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368212/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368212/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 20;16:7758. doi: <a href="https://doi.org/10.1038/s41467-025-62767-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62767-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Cryo-EM structure of ribosome from pathogenic protozoa <em>Entamoeba histolytica</em> reveals unique features of its architecture</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Shivani Sharma</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Shivani Sharma</span></h3>
<div class="p">
<sup>1</sup>Structural Biology &amp; Translation Regulation laboratory, UNESCO-DBT Regional Centre for Biotechnology, NCR Biotech Science Cluster, Faridabad, Haryana (Delhi-NCR) India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shivani Sharma</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mishra%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Shalini Mishra</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Shalini Mishra</span></h3>
<div class="p">
<sup>2</sup>Structural Biology Laboratory, School of Life Science, Jawaharlal Nehru University, New Delhi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mishra%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shalini Mishra</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gourinath%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Samudrala Gourinath</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Samudrala Gourinath</span></h3>
<div class="p">
<sup>2</sup>Structural Biology Laboratory, School of Life Science, Jawaharlal Nehru University, New Delhi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gourinath%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Samudrala Gourinath</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaushal%20PS%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Prem S Kaushal</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Prem S Kaushal</span></h3>
<div class="p">
<sup>1</sup>Structural Biology &amp; Translation Regulation laboratory, UNESCO-DBT Regional Centre for Biotechnology, NCR Biotech Science Cluster, Faridabad, Haryana (Delhi-NCR) India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaushal%20PS%22%5BAuthor%5D" class="usa-link"><span class="name western">Prem S Kaushal</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Structural Biology &amp; Translation Regulation laboratory, UNESCO-DBT Regional Centre for Biotechnology, NCR Biotech Science Cluster, Faridabad, Haryana (Delhi-NCR) India </div>
<div id="Aff2">
<sup>2</sup>Structural Biology Laboratory, School of Life Science, Jawaharlal Nehru University, New Delhi, India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 May 20; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368212  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835601/" class="usa-link">40835601</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1"><em>Entamoeba histolytica</em>, an anaerobic protozoan parasite, is the causative agent of amoebiasis, bloody diarrhea, and liver abscesses in humans. Amoebiasis is more predominant in tropical areas with poor sanitation conditions, and it remains the fourth leading cause of death due to a protozoan infection. <em>E. histolytica</em> life cycle spans between an infective ‘cyst stage’ and an active disease-causing ‘trophozoite stage’. We have isolated ribosomes from the trophozoite stage of <em>E. histolytica</em>. Here, we report single particle cryo-EM structures of the 53S ribosome large subunit (LSU), and 75S associated ribosomes, with P-tRNA, A/P and P/E tRNAs, and with paromomycin antibiotic, at 2.8 Å to 3.4 Å resolution. The <em>E. histolytica</em> possesses a reduced ribosome with a conserved core, and the periphery evolved with species-specific unique features. The most notable features are the presence of the rRNA triple helix near the peptide exit tunnel, the expansion segment H88ES102 near the exit site on LSU, and unique insertions in r-proteins. Furthermore, the 75S ribosome paromomycin complex structure provides the atomic details of its interactions. These structures provide insights into the evolutionary adaptation of the <em>E. histolytica</em> translational machinery and may be explored further for amoebicidal therapeutic intervention.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cryoelectron microscopy, Ribosome</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2"><em>Entamoeba histolytica</em> causes bloody diarrhea and liver abscesses in humans. Here, the authors report the cryo-EM structure of <em>E. histolytica</em> ribosomes from its disease-causing, trophozoite state and reveal several unique features, such as the rRNA triple helix motif.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3"><em>Entamoeba histolytica</em>, as its name suggests (histo-lytic = tissue destroyer), is an anaerobic parasite responsible for intestinal infection that results in bloody diarrhea, known as invasive amoebiasis or invasive colitis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Occasionally, the parasite reaches into the bloodstream and spreads through the body, most frequently ending up in the liver, where it causes liver abscesses. Rarely, it reaches the lung, brain, and spleen and causes abscesses, which can be fatal if untreated<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. <em>E. histolytica</em> infects ~35–50 million people worldwide and roughly kills 55,000 people each year (<a href="https://www.cdc.gov/dpdx/amebiasis/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cdc.gov/dpdx/amebiasis/index.html</a>)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The life cycle of <em>E. histolytica</em> spans the active disease-causing “trophozoite stage,” which exists only in the host, and the infective “cyst stage,” which survives outside the host. Infection occurs mainly via ingesting mature cyst from fecally contaminated food, water, or hands<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>.</p>
<p id="Par4">Earlier studies showed that ribosomes from <em>Entamoeba invadens</em>, a non-pathogenic model organism for <em>E. histolytica</em>, assemble in a rod-shaped helical structure known as chromatoid bodies during the cyst stage<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. In contrast, its trophozoite stage contains typical ribosomes having monosomes, ribosomal large subunit (LSU), and ribosomal small subunit (SSU)<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. <em>E. histolytica</em> possesses a reduced ribosome as its monosome, LSU, and SSU, which sediments at 75S, 53S, and 36S, respectively (S—Svedberg unit)<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> and their respective rRNA sediments at 25S, 17S, 5.8S, 5S, and 4S<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Conversely, human ribosomes monosome, LSU, and SSU sediments, at the 80S, 60S, and 40S, respectively, and possess four rRNA species 28S, 18S, 5.8S, and 5S<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Whereas the bacterial ribosome, monosome, LSU, and SSU sediments at the 70S, 50S, and 30S, respectively, and their rRNAs are 23S, 16S, and 5S<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par5">Protein synthesis in <em>Entamoeba</em> trophozoites was first reported by Carter et al.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The purified ribosomes form helices of the cyst, and polysomes from growing trophozoites appeared identical<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. However, the crude cyst extract was inefficient in protein synthesis, and it became efficient only after adding the soluble fractions from trophozoites<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Emetine was introduced as an amoebicidal drug that binds to the <em>E. histolytica</em> ribosome and inhibits protein synthesis<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. However, within two decades, <em>E. histolytica</em> mutants resistant to emetine were also reported<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Currently, ribosome-targeting aminoglycoside antibiotic “paromomycin” and other drugs are used as amoebicidal<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The high-resolution ribosome structures from other protozoans<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> have revealed the species-specific unique features of their architecture and protein synthesis. The atomic details for <em>E. histolytica</em> ribosome architecture are absent, as no structure is available.</p>
<p id="Par6">Here, we report the single particle cryo-EM structures of <em>E. histolytica</em> 53S, ribosome large subunit, at 2.8 Å resolution, 75S translating ribosome with bound tRNA at P-site, 75S with bound tRNAs at A/P- and P/E-sites, and 75S with bound paromomycin antibiotic and tRNAs at A/P- and P/E-sites. All the ribosome structures are reported at 3.0–3.4 Å resolutions. These structures reveal several unique features specific to <em>E. histolytica</em> ribosome architecture, such as the rRNA triple helix motif, expansion segment defined as H88ES102, and unique insertion in r-proteins. Further, the structure of 75S with bound paromomycin provided an atomic illustration of its binding pocket.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec3"><h3 class="pmc_sec_title">
<em>E. histolytica</em> ribosome</h3>
<p id="Par7"><em>The E. histolytica</em> ribosomes purified through sucrose density gradient centrifugation (SDGC) showed three prominent peaks (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). The fractions analyzed on 1% agarose gel stained with ethidium bromide showed two prominent bands, suggesting that it contains ribosomal RNA (rRNA), most likely corresponding to the 25S and 17S rRNAs (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>), fractions corresponding to the peaks P1, P2, and P3 (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>), analyzed through negative staining electron microscopy, clearly showed that peak P1 contains irregular shape and size particles (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>), which might be degraded ribosomes. Peak P2 contains a ribosomal large subunit (LSU) like particle, which is 53S in <em>Entamoeba</em><sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c</a>). Peak P3 contains the associated ribosome like particles (Supplementary Fig. <a href="#MOESM1" class="usa-link">1d</a>), which is 75S in <em>Entamoeba</em><sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, and small helical segments were also visible (Supplementary Fig. <a href="#MOESM1" class="usa-link">1d</a>), which might be fragments of helical ribosomal assemblies, observed earlier by Morgan &amp; Uzman<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, Further, peak P2 sample screening in cryo-EM conditions showed a uniform ribosomal distribution with optimum ice thickness (Supplementary Fig. <a href="#MOESM1" class="usa-link">1e</a>), suggesting the samples are ideal for high-resolution cryo-EM data collection.</p></section><section id="Sec4"><h3 class="pmc_sec_title">
<em>E. histolytica</em> ribosomal large subunit contains a rRNA triple helix motif</h3>
<p id="Par8">For ribosomal large subunit (LSU), the single particle reconstruction performed using RELION 3.1.4 has yielded an overall 2.8 Å resolution cryo-EM map from finally selected 378,061 particles (Fig. <a href="#Fig1" class="usa-link">1a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). The local resolution calculated using RELION showed that the core of the ribosomal LSU has a resolution of up to 2.5 Å, whereas the surface has nearly 4.5 Å resolution (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). The cryo-EM map clearly shows the high-resolution features, such as resolved r-protein alpha helices and rRNA double helices (Fig. <a href="#Fig1" class="usa-link">1a, e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>, Supplementary Movie <a href="#MOESM3" class="usa-link">1</a>). Remarkably, we could clearly see the rRNA triple helix motif in the LSU cryo-EM map, which is located near the peptide exit tunnel (PET) (Fig. <a href="#Fig1" class="usa-link">1a</a>). The rRNA triple helix is composed of three helices, two parallel and one antiparallel helix. The three nucleotides are on the same plane. The nucleotide bases of the middle helix that have purine nucleotide interact through Watson and Crick base pairing (–) with the nucleotide bases of one of the helices, forming a classical double helical structure. The third helix’s nucleotides interact with the double helix’s major groove through Hoogsteen base pairing (•). Therefore, it is a major groove RNA triple helix. Four such nucleotide triplets U3326–A3414•U3311, U3325–A3425•U3312, U3324–A3426•U3313, and C3323–G3417•U3314 constitutes the triple helix architecture (Fig. <a href="#Fig1" class="usa-link">1b–d</a>, Supplementary Movie <a href="#MOESM3" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Cryo-EM structure of <em>E. histolytica</em> 53S ribosome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/3ba84549e720/41467_2025_62767_Fig1_HTML.jpg" loading="lazy" id="d33e476" height="379" width="700" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The 2.8 Å resolution cryo-EM map of 53S ribosomal large subunit, rRNA (sky blue), r-proteins (royal blue), and third helix of rRNA triple helix (purple) are shown. The red circle shows the boundary of the triple helix. Central perturbance (CP), large subunit (LSU), and peptide exit tunnel (PET) are labeled. A dotted circle that is 50% transparent shows the location of PET in LSU. <strong>b</strong>, <strong>c</strong> rRNA triple helix, Watson and Crick pairing helices (sky blue), and third helix interacting to the major groove (purple), backbone in ribbon, and nucleotides in filled atoms are shown, and 5′ and 3′end are labeled. <strong>b</strong>, <strong>c</strong> are related by a rotation of 45° along the Y-axis. <strong>d</strong> A 2D diagram for the triple helix color code and labeling is the same as for (<strong>b</strong>, <strong>c</strong>). The <strong>−</strong> represents Watson and Crick base-paired nucleotide, the square box flanked by line represents Hoogsteen base-paired nucleotides, and circle flanked by a line represents nucleotides in close proximity. An animation is shown in Supplementary Movie <a href="#MOESM3" class="usa-link">1</a>. <strong>e</strong> Four base triples that constitute the triple helix are shown in the sticks, and cryo-EM density is shown on the surface with 85% transparency.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">The overall architecture of the <em>E. histolytica</em> 75S ribosome</h3>
<p id="Par9">Some of the high-resolution features similar to the large subunit cryo-EM map (Fig. <a href="#Fig1" class="usa-link">1</a>) were also visible in the 75S ribosome cryo-EM map (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">3</a>–<a href="#MOESM1" class="usa-link">5</a>). The 75S ribosome showed the presence of mRNA, nascent peptide, and bound tRNAs at P-site in one class and A/P and P/E-sites in another class (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>, and Supplementary Figs. <a href="#MOESM1" class="usa-link">3</a>–<a href="#MOESM1" class="usa-link">5</a>, Supplementary Movies <a href="#MOESM4" class="usa-link">2</a> and <a href="#MOESM5" class="usa-link">3</a>), suggesting that these translation factors were co-purified during ribosome purification and <em>E. histolytica</em> ribosomes were in an active translating state.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Cryo-EM structure of <em>E. histolytica</em> 75S with tRNA bound at P-site in the ribosome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/cd6df3ebc93c/41467_2025_62767_Fig2_HTML.jpg" loading="lazy" id="d33e568" height="905" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong> Composite cryo-EM maps of the 75S P-tRNA ribosome reconstructed from the focused 3D refinement of LSU, SSU body, and SSU head. LSU rRNA (light blue) and SSU rRNA (light yellow), r-proteins (different colors), P-tRNA (red), and mRNA (purple) are shown. <strong>a</strong>, <strong>b</strong> are related by ~180° rotation along the Y-axis and 35° rotation along the Z-axis. <strong>c</strong>, <strong>d</strong> The cryo-EM maps of <strong>a</strong>, <strong>b</strong> are replaced by the atomic coordinates. <strong>e</strong> The composite map of the 75S ribosome with P-tRNA is sliced through the peptide exit tunnel, NP-nascent peptide (green), mRNA (purple), and P-tRNA (red) are shown. <strong>f</strong> Only the mRNA, P-tRNA, and NP of the 75S ribosome are shown; the second residue from the CCA end of the P-tRNA, which looks like Arginine, is also shown. An animation is provided in Supplementary Movie <a href="#MOESM4" class="usa-link">2</a>.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Cryo-EM structure of <em>E. histolytica</em> 75S with tRNAs bound at A/P- and P/E-site in the ribosome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/af14ee9680e8/41467_2025_62767_Fig3_HTML.jpg" loading="lazy" id="d33e595" height="202" width="700" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Composite cryo-EM maps of 75S ribosome with A/P- and P/E-tRNAs and mRNA. LSU (sky blue) and SSU (khaki) with 80% transparency, A/P-tRNA (salmon), P/E-tRNA (orange), and mRNA (purple) are shown. <strong>b</strong> Only the translational factors with coordinates and cryo-EM map with 70% transparency are shown. <strong>c</strong> A comparison of A/P- and P/E-tRNAs from this study with A-, P-, and E-tRNAs from <em>Giardia intestinalis</em> (PDB ID 8G4S) is shown. An animation is provided in Supplementary Movie <a href="#MOESM5" class="usa-link">3</a>.</p></figcaption></figure><p id="Par10">The LSU cryo-EM map at 2.8 Å resolution and 75S ribosome bound with P-tRNA focused refinement maps at 3.0–3.4 Å resolution were used for initial model building. The cryo-EM maps of 75S clearly showed two extra densities, one adjacent to H88 and another at the 3′ end of 25S rRNA after modeling the annotated 25S rRNA sequence 3459 nucleotides (nt) long in the NCBI database (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/BDEQ01000001.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BDEQ01000001.1</a>). We could unambiguously build nearly 70 nt in H88 and 44 extra nt at the 3′ end (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). The extra nt in H88 appears to be an <em>E. histolytica</em>-specific expansion segment situated near the L1 stalk and E-site of the LSU, and therefore, we named it H88ES102 (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig4" class="usa-link">4</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). We have reannotated the length of 25S rRNA to 3504 nt long after building 44 extra nt at the 3′ end.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Secondary structure diagram of the <em>E. histolytica</em> LSU rRNA and its comparison with human counterparts.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/b51df1b7f033/41467_2025_62767_Fig4_HTML.jpg" loading="lazy" id="d33e669" height="1250" width="781" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> 2D representation of the 25S, 5.8S and 5S rRNAs colored by domains. D0 (Sandy brown), DI (purple dark), DII (blue), DIII (purple light), DIV (yellow), DV (pink), DVI (green), 5.8S (sienna), and 5S (light green) are shown and labeled with a Roman number. The rRNA that could not be modeled is shown in gray. Secondary structures were drawn using the R2DT tool from RNA Central and Adobe Illustrator, and human templates were obtained for comparison from <a href="http://apollo.chemistry.gatech.edu/RibosomeGallery/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://apollo.chemistry.gatech.edu/RibosomeGallery/index.html</a>. A detailed 2D diagram for 25S is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>, and rRNA nt building details are provided in Supplementary Table <a href="#MOESM1" class="usa-link">2</a>. <strong>b</strong>, <strong>c</strong> Comparative analysis of <em>E. histolytica</em> 25S rRNA with the human 28S rRNA (PDB ID: 4UG0<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>). Conserved regions are shown in a color similar to that of (<strong>a</strong>). The rRNA helices present in humans, which are either missing or disordered in <em>E. histolytica</em>, are shown in dark gray. The triple helix and H88ES102 are also labeled. A comparison of individual <em>E. histolytica</em> 25S rRNA domains with the human 28S rRNA domains (gray) (PDB ID: 4UG0<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>) is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>.</p></figcaption></figure><p id="Par11">Approximately 90.26% of 25S nt (3163 nt out of 3504 nt), 92.9% of 5.8S nt (144 nt out of 155 nt), and 100% 5S nt (117 nt) of LSU rRNAs were modeled in the cryo-EM map (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a> and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Similarly, 74.06% of 17S nt (1442 nt out of 1947 nt) was modeled for SSU rRNA in cryo-EM map (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a> and Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). Overall, <em>E. histolytica</em> rRNA retains a conserved core and an altered surface (Figs. <a href="#Fig4" class="usa-link">4b, c</a> and <a href="#Fig5" class="usa-link">5b, c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a> and Supplementary Tables <a href="#MOESM1" class="usa-link">2</a> and <a href="#MOESM1" class="usa-link">3</a>). The cryo-EM maps did not show any extra density that could be attributed to the 4S rRNA, as observed previously<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>; maybe its binding is highly dynamic.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Secondary structure diagram of the <em>E. histolytica</em> SSU rRNA and its comparison with human counterparts.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/8d33888bdb09/41467_2025_62767_Fig5_HTML.jpg" loading="lazy" id="d33e758" height="1250" width="690" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> 2D representation of the 17S rRNAs colored by domains. 5′ (blue), C (brown), 3′M (pink), 3′m (green). The rRNA that could not be modeled is shown in gray. The secondary structures were drawn using the R2DT tool from RNA Central and Adobe Illustrator, and human templates for comparison were obtained from the RiboVision suite (<a href="http://apollo.chemistry.gatech.edu/RiboVision" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://apollo.chemistry.gatech.edu/RiboVision</a>). A detailed 2D diagram for 25S is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>, and statistics in Supplementary Table <a href="#MOESM1" class="usa-link">3</a>. <strong>b</strong>, <strong>c</strong> Comparisons of <em>E. histolytica</em> 17S rRNA with the human 18S rRNA (PDB ID: 4UG0<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>). Regions either missing or disordered in 17S rRNA are shown in dark gray; other color schemes are similar to (<strong>a</strong>). A detailed comparison of individual domains is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">9b</a>.</p></figcaption></figure><p id="Par12">We were able to build 40 LSU and 31 SSU r-proteins using the focused refinement cryo-EM maps for LSU, SSU body, and SSU head of 75S ribosome with bound P-tRNA and LSU map of individual subunit (Fig. <a href="#Fig2" class="usa-link">2</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a>–<a href="#MOESM1" class="usa-link">13</a>, Supplementary Tables <a href="#MOESM1" class="usa-link">2</a> and <a href="#MOESM1" class="usa-link">3</a>). High-resolution features in our cryo-EM maps allow us to assign the correct isoform for r-protein (Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a>–<a href="#MOESM1" class="usa-link">12</a>) and to build r-protein extensions/insertions (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). There were discrepancies in the annotations of ribosomal protein in the whole genome sequence available at NCBI with GENBANK ID: <a href="https://www.ncbi.nlm.nih.gov/nuccore/BDEQ01000001.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BDEQ01000001.1</a> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/BDEQ01000001.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/nuccore/BDEQ01000001.1</a>). Such as protein eL13 has six protein IDs, and uL13 has one protein ID. Sequence alignment showed that four IDs annotated to eL13, out of the six IDs, show similarity to the uL13 r-protein. Thus, these IDs were misannotated as eL13 (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). The genome database does not annotate any protein ID for eL22 and eL29. However, after a sequence search, human eL22 and eL29 showed significant sequence identity with <em>E. histolytica</em> protein IDs EHI_141940, EHI_023220, respectively. Further, our cryo-EM maps clearly showed density that could be attributed to these two proteins (Fig. <a href="#Fig2" class="usa-link">2a–d</a>). The eL29 is shorter (64 aa) and shares only 33% identity with its human (122 aa) counterpart. Similarly, the SSU r-protein eS8 and RACK1 were not annotated in the <em>E. histolytica</em> genome database. Based on sequence similarity with human counterpart and cryo-EM density in our map, we could assign the protein IDs <a href="https://www.ncbi.nlm.nih.gov/protein/GAT98092" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GAT98092</a> and <a href="https://www.ncbi.nlm.nih.gov/protein/GAT91985" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GAT91985</a> to eS8 and RACK1 <em>E. histolytica</em> r-proteins, respectively (Fig. <a href="#Fig2" class="usa-link">2a–d</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">
<em>E. histolytica</em> rRNA is reduced compared to its human counterpart</h3>
<p id="Par13">The LSU of <em>E. histolytica</em> consists of 25S, 5.8S, and 5S rRNAs with 3504, 155, and 117 nt in length, respectively<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>, Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Its human counterpart has elaborate rRNAs, 28S, 5.8S, and 5S, with 5070, 157, and 121 nt in length, respectively (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. On comparing both, the overall architecture of the rRNA core is conserved, whereas the surface helices are reduced compared to their human counterpart (Fig. <a href="#Fig4" class="usa-link">4b, c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>). The <em>E. histolytica</em> 25S rRNA can be subdivided into the domain system adopted for human 28S rRNA (<a href="http://apollo.chemistry.gatech.edu/RibosomeGallery/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://apollo.chemistry.gatech.edu/RibosomeGallery/index.html</a>)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, which comprises of conserved core termed as domain 0, along with six other domains (DI to DVI) (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>).</p>
<p id="Par14">Domain I of the 25S rRNA in <em>E. histolytica</em>, consisting of 1–769 nt, is shorter than the human counterpart, which is 1313 nt long (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). The difference in length is primarily due to the absence of ESs, H25ES7d, e, f, g, h, and the shortening of ESs, H25ES7a and b, which are ~96 nt long in <em>E. histolytica</em> as compared to ~245 nt in humans (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>). The ES H25ES7c is shorter and adopts a different conformation than its human counterpart. The 25S rRNA H15 remains partially disordered in our cryo-EM maps, which is longer (87 nt) in <em>E. histolytica</em> compared to the 28S rRNA of the human ribosome (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>).</p>
<p id="Par15">Domain II of the 25S rRNA in <em>E. histolytica</em> is from 794 to 1566 nt (772 nt long), which is 257 nt shorter than human Domain II (from 1317 to 2346 nt) (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>), and adopts a similar conformation. The ESs, 30ES9, 31ES9a, 45ES15, and 45ES15a are absent altogether, whereas H45, 30ES9, and H31are truncated in <em>E. histolytica</em> compared to human counterparts (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>).</p>
<p id="Par16">Domain III comprises 1587–2079 nt (492 nt in length) in <em>E. histolytica</em>,25S rRNA. This domain is ~33 nt longer than the domain III in human 28S rRNA, which is 459 nt in length (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). The longer size can be attributed to the longer H54ES20a, H55, and H47, whereas the H54ES20a and H55 remain partially disordered in our cryo-EM maps (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>).</p>
<p id="Par17">Domain IV spans from 2110 to 2408 nt (298 nt in length), which is drastically reduced in <em>E. histolytica</em> 25S rRNA compared to 977 nt long in its human (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). This domain is mainly composed of H63 and its expansion segment ES27. The enormous difference in size is due to the absence of 668 nt, constituting two major expansion segments 63ES27a and b (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>). These ESs, 63ES27a and b, have been known to play a crucial role in recruiting factors at the peptide exit tunnel and ribosome assembly<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup><sup>,</sup> and their deletion is lethal in higher eukaryotes<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p>
<p id="Par18">Domain V, which harbors the peptidyl transferase center (PTC)<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, spans from 2476 to 3121 nt (645 nt long) (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>) and has a conserved PTC region. Although it has a similar nt number (654 nt in humans), it shows significant alterations in the L1 stalk region. The L1 stalk region is composed of H76, H77, and H78, which are disordered in our map, whereas the constituting ESs (ES30 and ES30a) are absent in <em>E. histolytica</em> rRNA. The r-protein, uL1, also remains disordered in our cryo-EM maps. The H79 and its ESs, ES31, ES31a, and ES31b, are shorter compared to their human counterpart (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>). The H88 possesses a 70 nt long <em>E. histolytica</em>-specific ES, H88ES102 (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>), and makes a unique interaction with the insertion segment of eL13 r-protein (Fig. <a href="#Fig2" class="usa-link">2c, d</a>). The H88ES102 region remains disordered in our 53S LSU cryo-EM map, suggesting it gets ordered only after 75S formation. Recently, it has been suggested that the H88 plays a role in phase 2 of the 60S ribosomal subunit biogenesis, the assembly of the central protuberance (CP). H88 and the L1 stalk provide two nucleation sites for the compaction and assembly of the rRNAs H80-H87, and are involved in forming CP and 5S RNP<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Whether the <em>E. histolytica</em> H88 expansion H88ES102 has any role in modulating the CP assembly remains to be investigated.</p>
<p id="Par19">Domain VI of <em>E. histolytica</em> comprises 3146 to 3504 nt (358 nt long), shorter than the human domain VI (497 nt long) (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). The reduction in length is mainly due to reduced H98 and H101 and their compact expansion segments, 98ES39, 98ES39a, 98ES39b, and 101ES41 (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>, <a href="#MOESM1" class="usa-link">9a</a>). The H98 adopts a different conformation in <em>E. histolytica</em> 25S rRNA, which leads to a unique rRNA motif formation, the triple helix rRNA motif (Fig. <a href="#Fig4" class="usa-link">4c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>). The 5.8S and 5S are mostly conserved, with 155 nt and 117 nt in length, respectively. (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>).</p>
<p id="Par20">Similar to LSU 25S rRNA, the overall architecture of the <em>E. histolytica</em> SSU rRNA has a conserved core region (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a> and <a href="#MOESM1" class="usa-link">9b</a>). The peripheral helices are either missing or reduced in length compared to their human counterpart (Fig. <a href="#Fig5" class="usa-link">5b, c</a> Supplementary Fig. <a href="#MOESM1" class="usa-link">9b</a>). The SSU rRNA is 17S<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, and consists of 1947 nt. In contrast, humans have 18S rRNA, which has a length of 1869 nt (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). The 17S rRNA is also made up of four domains: 5′ domain (5′), central (C) domain, 3′ major (3′M) domain, and 3′ minor (3′m) domain (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">S8</a> and <a href="#MOESM1" class="usa-link">S9b</a>). The 5′ domain spans from 1 to 602 nt, shorter than its human (658 nt long) counterpart (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>), mainly because of missing expansion segments, h9es3c and b. Meanwhile, h16 and h17 remain disordered in our cryo-EM map (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">S8</a> and <a href="#MOESM1" class="usa-link">S9b</a>). The 17S rRNA, C domain consists from 603 to 1162 nt (559 nt long), which is longer in <em>E. histolytica</em> compared to humans (534 nt long) 18S rRNA (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>), mainly because of an insertion in surface helices 21es6c and 26es7, which remains disordered in our cryo-EM map. The 21es6b, which is shorter in 17S rRNA, also remains a disorder. The 3′M domain comprises 1171–1801 nt (630 nt in length) of 17S rRNA, which is exceptionally longer than humans (497 nt long) 18S rRNA (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). The longer length is mainly attributed to ESs present in h39, h40, and h41es10, which remain partially disordered in our cryo-EM map (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a> and <a href="#MOESM1" class="usa-link">9b</a>). The 3′m domain encompasses from 1802 to 1947 nt (145 nt in length) and is shorter than its human (169 nt long) counterpart (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). The shorter length is mainly because of h44, the major helix in the 3′m domain, which remains partially disordered in our cryo-EM maps (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a> and <a href="#MOESM1" class="usa-link">9b</a>). The 3′ M and 3′m are domains that constitute the decoding center on the SSU, where codon and anticodon interaction takes place during protein synthesis<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, retaining a conserved core with alteration in surface helices in <em>E. histolytica</em>.</p>
<p id="Par21">The ribosomal catalytic sites, the decoding center<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup><sub>,</sub> the peptidyl transferase center, and the polypeptide exit tunnel<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> are highly conserved in <em>E. histolytica</em>. The tRNA binding sites, A-, P- and E-sites on both the subunits are also conserved (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>) with minor variations, particularly in the r-proteins; uS13, uS19, eS25, uL16, and eL42, which constitute the tRNA binding sites (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). Although the core of these proteins is conserved, the terminal end remains disordered compared to their human counterparts (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>), even in the tRNA-bound state (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). Interestingly, the loop region of the uL16, which interacts with the acceptor arm of the P-tRNA adopts a slightly different conformation and remains partially disordered in both <em>E. histolytica</em> and humans despite having the same amino acid sequence (Supplementary Fig. <a href="#MOESM1" class="usa-link">15d</a>). C-terminus of eL42 is slightly shorter in <em>E. histolytica</em> than in humans (Supplementary Fig. <a href="#MOESM1" class="usa-link">15e</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">
<em>E. histolytica</em> r-protein possesses species-specific insertions/extensions</h3>
<p id="Par22"><em>E. histolytica</em> r-protein possesses overall conserved domains compared to its human counterpart, with deletions and insertions/extensions in a few r-proteins (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). The uL3, uL18, eL6, eL14, eL29, eL37 and eS17 are truncated, whereas uL4, eL9, eL13 eL18 eL20, uL24, eL31, uS3, eS19, and eS21 carries <em>E. histolytica</em>-specific insertions/extensions (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). The cryo-EM density for these extensions/insertions were evident in the map (Fig. <a href="#Fig2" class="usa-link">2a–d</a>). We could build these extensions/insertions (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). Intriguingly, uL24 of <em>E. histolytica</em>, which has a nearly ~80 amino acid long insertion, occupies the position of the human eL28 protein. The eL28 is absent in <em>E. histolytica</em> but present in humans (Supplementary Fig. <a href="#MOESM1" class="usa-link">16a</a>). The uL24 extension appears specific to <em>E. histolytica</em> as it  is shorter in other eukaryotes and protozoa (Supplementary Fig. <a href="#MOESM1" class="usa-link">16b</a>). Similarly, a ~90 amino acid residue insertion in the eL13 seems species-specific (Supplementary Fig. <a href="#MOESM1" class="usa-link">17a</a>) and is absent in other eukaryotes and protozoa (Supplementary Fig. <a href="#MOESM1" class="usa-link">17b</a>).</p>
<p id="Par23">The <em>E. histolytica</em> r-proteins, eL6, eS8, and uS17 loops adopt a slightly different conformation, despite sharing a conserved domain with humans (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). The uL10, uL11, eL28, eS31, and eS32 present in human ribosomes are absent in <em>E. histolytica</em>. Some <em>Entamoeba</em>-specific insertions/extensions in r-proteins, eS4, eS6, eS7, eS25, and eS26, could not be modeled in our cryo-EM map, maybe because of their flexible nature.</p></section><section id="Sec8"><h3 class="pmc_sec_title">
<em>E. histolytica</em> intersubunit bridges and conformational changes in LSU upon association</h3>
<p id="Par24">Intersubunit bridges at the interface side of the SSU and LSU<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> hold the ribosomal subunits together. The bridges were analyzed using Chimera X. The residues within a 4 Å distance at the interface side of LSU and SSU were selected. The intersubunit bridges were compared with those of human and another protozoan, <em>Giardia lamblia</em> bridges<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> (Fig. <a href="#Fig6" class="usa-link">6a</a> and Supplementary Table <a href="#MOESM1" class="usa-link">5</a>). We were able to analyze 13 bonafide bridges, whereas the bridge eB13 remains disordered in the 75S ribosome cryo-EM map. The <em>E. histolytica</em> ribosome has retained most of the bacterial-like bridges, B1b/c, B2a, B2b, B2c, B2e, B3, B4, B5, B6, B7a, B7b/a, and B8, and also acquired a few eukaryotic-specific bridges, eB11, eB12. A notable difference was the distinct conformation of the C-terminus of uS17 (Supplementary Fig. <a href="#MOESM1" class="usa-link">18a</a>), which makes a unique interaction with eL19 r-protein near bridge eB12. This interaction appears to be <em>E. histolytica</em>-specific; therefore, we have named this an eB12a bridge. The eS32 protein is absent in <em>E. histolytica;</em> consequently, the bridge eB14 is missing. The bridge B1a, eB8, and eB13 could not be observed because the interacting residues remain disordered in our cryo-EM map.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. <em>E. histolytica</em> ribosome intersubunit bridges and conformational changes occur in LSU upon association.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/fc10796cc5b2/41467_2025_62767_Fig6_HTML.jpg" loading="lazy" id="d33e1298" height="883" width="750" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The ribosomal small subunit (left) and large subunit (right) are shown in light khaki and blue, respectively. The residues involved in intersubunit bridges are shown in the atomic sphere, and bridges are labeled. The bridges shown in gray color were not observed in the 75S ribosome. A detail of interacting residues is provided in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>. <strong>b</strong> The LSU coordinates built using 75S cryo-EM maps (light blue) docked into the LSU Cryo-EM map (gray) are shown. The rRNA helices that become ordered upon association with the SSU are labeled.</p></figcaption></figure><p id="Par25">Docking the LSU coordinates built in the 75S ribosome cryo-EM into the 2.8 Å cryo-EM map of the LSU alone, showed that the rRNA helices, mainly on the interface side, become ordered upon association with SSU (Fig. <a href="#Fig6" class="usa-link">6b</a>). The most notable is the <em>E. histolytica</em>-specific surface ES H88ES102 and insertions in eL13, which become ordered upon association (Fig. <a href="#Fig7" class="usa-link">7</a>). Other rRNA helices, H34, H43, H44, H69, H76, H79ES31b, H84, H101 also become ordered in the 75S associated ribosome. The H34, H69, and H101 lead to the formation of intersubunit bridges B4, B2a, and eB11, respectively, upon association with SSU (Fig. <a href="#Fig6" class="usa-link">6a</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Co-evolution of rRNA and r-proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/c5dd3c09f30b/41467_2025_62767_Fig7_HTML.jpg" loading="lazy" id="d33e1327" height="760" width="750" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The <em>E. histolytica</em> LSU is shown on the solvent side in the middle, and regions of rRNA and r-protein co-evolution are zoomed and shown in the surrounding area in roughly matching orientation. The rRNA (blue) is shown in ribbon and surface view with 80% transparency. The r-proteins in the ribbon (light blue) and extension/insertion (red) are shown. Central perturbance (CP) and peptide exit tunnel (PET) are labeled. A circle that is 50% transparent shows the location of PET in LSU. A full 3D model of r-protein with extensions is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">
<em>E. histolytica</em> rRNA and r-protein have co-evolved</h3>
<p id="Par26"><em>E. histolytica</em> ribosomes have evolved with a conserved core and alterations mainly on the solvent-exposed surface. The co-evolution of r-protein and rRNA is a common feature observed earlier in the structure of eukaryotic ribosomes<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The r-proteins, such as uL4, uL9, eL13, eL18, eL20, uL24, and eL31, possess prominent <em>E. histolytica</em>-specific extensions/insertions in their sequences (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). These are mainly localized on the solvent side surface of the LSU (Fig. <a href="#Fig7" class="usa-link">7</a>). On the other hand, the LSU 25S rRNA is compact and reduced in size compared to human 28S rRNA. The small size of 25S rRNA is mainly because some eukaryotic-specific expansion segments are missing (Fig. <a href="#Fig4" class="usa-link">4b, c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>). To stabilize the reduced size of rRNA, which has opted a different conformation for surface helices, <em>E. histolytica</em> has evolved with extensions, which contain mostly positively charged amino acid residues and mainly adopts α helical secondary structure (Fig. <a href="#Fig7" class="usa-link">7</a>). The rRNA helices sandwich these α helices, which are localized in the vicinity of the ribosomal surface (Fig. <a href="#Fig7" class="usa-link">7</a>). For example, uL4 extension sandwich between H25ES7b and H25ES7c, eL9 insertion interacts with H42. Most intriguingly, the eL13 extension is sandwiched between H18ES43 and H25ES7a, with the <em>E. histolytica</em>-specific helix H88ES102. Similarly, the uL24 extension sandwich between H45 and H46, and eL31 interacts with the H94, H99 and H98ES39b, which constitutes the rRNA triple helix, an <em>E. histolytica</em>-specific rRNA motif (Fig. <a href="#Fig7" class="usa-link">7</a>). In summary, it suggests that the <em>E. histolytica</em> LSU rRNA helices and r-proteins have co-evolved to provide a compact and stable architecture, which might be crucial for its survival.</p></section><section id="Sec10"><h3 class="pmc_sec_title">
<em>E. histolytica</em> possesses a triple helix motif in its 25S rRNA</h3>
<p id="Par27">RNA triple helix, naturally found in biological machineries, plays a vital role and provides a catalytic scaffold, structural stability, metal binding site, etc.<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. RNA triple helix is found in telomerase, an RNP enzyme that maintains the chromosome end, where RNA triple helix is crucial for its catalytic activity<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Spliceosome and Group II introns RNA triple helix coordinate the catalytic Mg<sup>2+</sup> ions during splicing<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. In riboswitches, the RNA triple helix provides a ligand-binding scaffold<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. It is also reported in humans that metastasis-associated lung adenocarcinoma transcript 1 (MALAT1), which is endowed with a blunt end triple helix (no 3′overhang), inhibits the rapid phase of nuclear RNA decay and also upregulates translation<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Kaposi’s sarcoma-associated herpesvirus (KSHV) adopts a triple RNA helix motif in lytic phage and protects its genome from decay<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. However, there are no reports of rRNA adopting a triple helical conformation.</p>
<p id="Par28">Interestingly, we found a distinctly visible cryo-EM density for a triple helix in the <em>E. histolytica</em> LSU map (Figs. <a href="#Fig1" class="usa-link">1a–e</a> and <a href="#Fig4" class="usa-link">4</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>, <a href="#MOESM1" class="usa-link">9a</a>, Supplementary Movie <a href="#MOESM3" class="usa-link">1</a>). To the best of our knowledge, this is the first report of an RNA triple helix present in ribosomal RNA. The triple helix is present in DVI, towards the 3′ end of 25S rRNA, on the surface, solvent side, of the LSU near the PET (Figs. <a href="#Fig4" class="usa-link">4c</a> and <a href="#Fig8" class="usa-link">8</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">9a</a>). Compared with the human 28S rRNA, we observed that <em>E. histolytica</em> 3′ is shorter by 16 nt compared with its human counterpart (Fig. <a href="#Fig8" class="usa-link">8</a>). The human 28S rRNA 3′ position overlaps with the <em>E. histolytica</em> triple helix position on 25S rRNA (Figs. <a href="#Fig4" class="usa-link">4c</a> and <a href="#Fig8" class="usa-link">8</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>). The other eukaryotes, <em>Saccharomyces cerevisiae</em> and protozoans, <em>Plasmodium falciparum</em>, <em>Leishmania donovani</em>, and <em>Giardia lamblia</em>, also possess an extended 3′ end, and overall, a conserved architecture of domain VI to that of human rRNA domain VI (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. Comparative analysis of LSU rRNA domain VI.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/29c33bf58fc0/41467_2025_62767_Fig8_HTML.jpg" loading="lazy" id="d33e1519" height="498" width="675" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> 25S rRNA domain VI from <em>E. histolytica</em> (green) and the triple helix motif (sky blue and magenta). <strong>b</strong> The 28S rRNA, domain VI from human (PDB ID: 4UG0<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>) (gray), and the extra 3′ rRNA nucleotide (black) are shown, and major helices are labeled. A comparative analysis of Domain VI is provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>.</p></figcaption></figure><p id="Par29">As the 3′ end is a target site for 3′ to 5′ exonucleases, it is speculative to say that <em>E. histolytica</em> has remodeled its 25S rRNA by shortening its length and adopting a different conformation, the triple helix, to protect its rRNA from human 3′ to 5′ exonucleases. It will most likely be similar to what was reported earlier for KSHV<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. However, it needs experimental evidence.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Cryo-EM structure of <em>E. histolytica</em> 75S ribosome bound with paromomycin</h3>
<p id="Par30">The cryo-EM single particle reconstruction yielded a final 3.2 Å resolution map after performing focused refinement of the SSU body (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). We could clearly see the cryo-EM density that could be attributed to paromomycin (PAR), an aminoglycoside (AG) antibiotic (Fig. <a href="#Fig9" class="usa-link">9</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). The PAR binds to a conserved binding pocket, the major groove of h44 in the SSU body (Fig. <a href="#Fig9" class="usa-link">9b–d</a>), as earlier reported in <em>E. coli</em><sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> and <em>Leishmania donovani</em><sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> (Fig. <a href="#Fig10" class="usa-link">10</a>). PAR binding flips out the universally conserved residues A1902 and A1903 (A1492 and A1493 in <em>E. coli</em> numbering) (Figs. <a href="#Fig9" class="usa-link">9c, e</a> and <a href="#Fig10" class="usa-link">10</a>), which otherwise are flipped out only when there is a cognate codon-anticodon interaction at A-site of the decoding center. In normal situations, the flipped-out conformation of A1902 and A1903 (A1492 and A1493 <em>in E. coli</em> numbering) stabilizes the cognate tRNA at the A-site and facilitates the correct tRNA selection<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. In PAR-bound state, these two nucleotides, A1902 and A1903 (A1492 and A1493 <em>in E. coli</em> numbering), remain flipped out permanently, thus facilitating the selection of both cognate and non-cognate (incorrect) tRNA at the A-site in the decoding center, leading to error-prone translation<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. The PAR binding pocket is highly conserved, where ring 1 of PAR is involved in stacking interactions with G1901 (G1491 in <em>E. coli</em> numbering) and hydrogen bonds with G1901 and A1902. The ring II of PAR makes three hydrogen bonds with A1903, G1904, and U1905 (A1493, G1494, and U1495 in <em>E. coli</em> numbering), whereas ring III of PAR makes a single hydrogen bond with C1810 (G1407 in <em>E. coli</em>). The ring IV of PAR also makes a single hydrogen bond with G1808 (G1399 in <em>E. coli</em> numbering) (Figs. <a href="#Fig9" class="usa-link">9e, f</a> and <a href="#Fig10" class="usa-link">10</a>). The H69 of LSU and uS12 protein further stabilize the PAR binding pocket. The rRNA nucleotide modifications within the PAR binding pocket have been shown to play a role in stabilizing PAR binding<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. However, we did not model any modifications in our cryo-EM map; thus, these observations could not be made. The PAR binding pocket is highly conserved in <em>E. histolytica</em>, <em>L. donovani</em>, and <em>E. coli</em>. In contrast, the corresponding pocket in humans is similar with a few notable differences (Fig. <a href="#Fig10" class="usa-link">10</a>). The base stacking interaction between ring I of PAR with G1901 (G1491 in <em>E. coli</em>) appears to be the major stabilizing interaction, where G1901 makes Watson Crick base pair with C1812 (C1409 in <em>E. coli</em>) and provides a stable platform for base stacking interaction with ring I of PAR. Whereas in humans, the corresponding residue A1823 adopts a slightly different conformation and does not base pair with the opposite strand nucleotide, which is C1710 (Fig. <a href="#Fig10" class="usa-link">10</a>). Therefore, in humans, the base stacking interaction would be disfavored upon PAR binding.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9. Cryo-EM structure of 75S ribosome in complex with paromomycin antibiotics.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/96132802197c/41467_2025_62767_Fig9_HTML.jpg" loading="lazy" id="d33e1664" height="483" width="800" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Cryo-EM maps of 3D focus refinement of SSU body (olive), a thumbnail with 75S cryo-EM map in transparent on the left is shown. <strong>b</strong> The map of the (<strong>a</strong>) rotated by ~90° along the Y-axis with 70 % transparency, and atomic models of SSU body (olive), mRNA (red), anticodon stem-loop of A/P-tRNA (salmon), P/E-tRNA (orange), and paromomycin cryo-EM density (magenta) are shown. <strong>c</strong> A zoomed paromomycin binding pocket with transparent cryo-EM map is shown. <strong>d</strong> Paromomycin coordinates fitted in cryo-EM density (transparent magenta) are shown. <strong>e</strong> Paromomycin binding pocket residues within 4 Å distance are shown. <strong>f</strong> A 2D diagram of the paromomycin binding pocket residues within 4 Å is shown.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10. Helix h44, the paromomycin binding pocket.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368212_41467_2025_62767_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0ff/12368212/1320589f31ac/41467_2025_62767_Fig10_HTML.jpg" loading="lazy" id="d33e1685" height="541" width="800" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(Top panel) Paromomycin (purple) bound to the h44 binding pocket in ribosomes from <em>E. histolytica</em> (from this study), <em>L. donovani</em> (PDB ID: 6AZ1), and <em>E. coli</em> (PDB ID: 7K00) are shown. A corresponding region in humans (PDB ID: 8g61) is also shown in the left panel. (Bottom panel) A 2D diagram of helix 44 paromomycin binding pocket. Universally conserved residues A1492 and A1493 (<em>E. coli</em> numbering) are shown with a green background. Other crucial residues for paromomycin are shown with blue and red backgrounds.</p></figcaption></figure><p id="Par31">In summary, the <em>E. histolytica</em> ribosomes isolated from the disease-causing trophozoite stage using sucrose density gradient centrifugation showed prominent peaks corresponding to the 75S ribosome and its 53S large subunit. Single particle reconstruction and model building have elucidated the atomic details of the 53S and 7SS ribosomes with tRNAs, mRNA, and nascent peptide and revealed <em>E. histolytica</em>-specific features such as triple helix motif, H88ES102, and insertion/extensions in r-proteins. The <em>E. histolytica</em> retains a conserved core with modifications in solvent-exposed surface. <em>E. histolytica</em> might have evolved with a triple RNA helix motif to protect its ribosome from host RNase attack. The cryo-EM structure of 75S ribosome in complex with paromomycin antibiotic has illustrated the atomic details of its binding pocket in <em>E. histolytica</em>. This study opens a new avenue to investigate protein synthesis in <em>E. histolytica</em>, which is vaguely understood, and to exploit the unique species-specific features of its translation machinery to design novel amoebicidal therapeutics.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">
<em>Entamoeba histolytica</em> cell growth</h3>
<p id="Par32">Trophozoites of <em>E. histolytica</em> strain HM1: IMSS clone-6 were grown axenically in TYI-S-33 medium at 35 °C as described earlier by Diamond et al.<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. To get a sufficient amount of ribosomes, the culture was expanded into a 75 cm<sup>2</sup> cell culture, plug seal flask (NEST) with 250 ml complete media following the earlier protocol by Gautam et al.<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. In brief, TYI-S-33 media was prepared by mixing potassium phosphate dibasic 1.0 g, potassium phosphate monobasic 0.6 g, biosate peptone 30.0 g, glucose 10.0 g, sodium chloride 2.0 g, L-cysteine hydrochloride 1.0 g, ascorbic acid 1.0 g, and ferric ammonium citrate 22.8 mg in 700 ml of double distilled water and pH was adjusted to 6.8 using 5 N NaOH. The volume was made up to 900 ml (10 units) and filtered using a filter assembly (Corning 500 ml, filter system). The cells were grown in TYI-S-33 medium complemented with 15% adult bovine serum, 1X Diamond vitamin mix, 125 µl of 250 U/ml Benzylpenicillin, and 0.25 mg/ml Streptomycin per 90 ml of medium (complete media). After 60 h, the media was removed, and 1X Phosphate Buffer Saline (PBS pH 7.4) was added to the confluent culture, chilled on ice for 7 min, and then harvested by centrifugation at 1200 × <em>g</em> for 7 min at 4 °C.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Ribosome isolation and purification</h3>
<p id="Par33">Ribosome isolation was done similarly, as reported previously in ref. <sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, with minor modifications. The <em>E. histolytica</em> cells were resuspended in lysis buffer (20 mM) HEPES-sodium salt buffer, pH 7.4, 40 mM KOAc, 10 mM Mg(OAc)<sub>2</sub>, 250 mM sucrose, 3 mM DTT, 1 mM PMSF, 1X Protease inhibitor cocktail (cOmplete, Roche), RNasin (Thermo,1:40 dilution) and flash frozen in liquid nitrogen. The cell lysis was performed using Retsch Mixer Mill MM500 at 30 Hz, 1 min/cycle, and a total of 8 cycles. 2 U/ml DNase I (Thermo Scientific) was added to the cell lysate and incubated on ice for 45 min. The cell debris was cleared by centrifugation at 18,626 × <em>g</em> for 45 min at 4 °C. The cleared cell lysate was layered on a 1.1 M sucrose cushion (20 mM HEPES-sodium salt buffer pH 7.4, 150 mM KOAc, 10 mM Mg(OAc)<sub>2</sub>, 1.1 M sucrose, 3 mM DTT) in equal volume, followed by ultra-centrifugation at 100,000 × <em>g</em> for 16 h at 4 °C (Hitachi P70AT2 rotor). The pellet was then resuspended in buffer A (20 mM HEPES-sodium salt buffer, pH 7.4, 150 mM KOAc, 10 mM Mg(OAc)<sub>2</sub>, 3 mM DTT) and centrifuged at 18,626 × <em>g</em> 30 min, 4 °C. The absorbance of the supernatant was measured at 260 nm, and 10 OD units were layered on a 10%–50% sucrose gradient in gradient buffer (20 mM HEPES-sodium salt buffer, pH 7.4, 150 mM KOAc, 10 mM Mg(OAc)<sub>2</sub>, 3 mM DTT) prepared using Gradient Master from Biocomp Instruments. Gradients were centrifuged at 256,400 × <em>g</em> for 8 h at 4 °C (Hitachi, P40ST rotor). Afterward, the gradients were fractionated using a Gilson Fractionator from BioComp Instruments (Supplementary Fig.<a href="#MOESM1" class="usa-link">1</a>). The fractions were then analyzed using 1% agarose gel with 0.06% bleach (Supplementary Fig.<a href="#MOESM1" class="usa-link">1</a>). The fractions corresponding to peaks were then pooled and concentrated using 100 kDa cut-off amicon from Merck Millipore, followed by buffer exchange with ribosome storage buffer (20 mM HEPES-sodium salt buffer pH 7.4, 100 mM KOAc, 10 mM Mg(OAc)<sub>2</sub>, 10 mM NH<sub>4</sub>OAc, 3 mM DTT). The concentration of the sample was determined by absorbance at 260 nm, and ribosomes were stored at −80 °C for future use.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Electron microscopy</h3>
<p id="Par34">The 300 mesh Copper grids with a thin carbon layer on the top (CF300-Cu) from Electron Microscopy Science (EMS) were glow-discharged for 50 s in a JEOL glow discharge unit. A 3 µl sample from peak 1(Supplementary Fig.<a href="#MOESM1" class="usa-link">1a</a>) of sucrose gradient fractionation was applied on the grid’s carbon side and washed three times with Mili-Q water. The grid was stained with 1% uranyl acetate solution. The grids for peaks 2 and 3 (Supplementary Fig.<a href="#MOESM1" class="usa-link">1a</a>) were prepared following a similar protocol. The grids were analyzed in a 120 kVa JEOL JEM1400 electron microscope at Advanced Technology Platform Centre (ATPC), Regional Centre for Biotechnology (RCB), Faridabad (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b–d</a>).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Cryo-electron microscopy</h3>
<p id="Par35">A 3 µl of 53S ribosome (1 µg/µl, peak 1) sample was applied to a quantifoil Cu 1.2/1.3 grids from Ted Pella, Inc. and plunge freeze using GATAN CP3 plunger after 3-s blotting at 4 °C temperature and 80% humidity. The cryo-EM grid was mounted in a GATAN 626 cryo-holder and analyzed using a JEOL JEM2200FS microscope equipped with a GATAN K2 summit direct electron detector camera at ATPC, RCB, Faridabad. At optimum ice thickness, the data were collected in a Movie mode at 1.649 e<sup>−</sup>/Å<sup>2</sup>/frame with 30 frames per Movie stack.</p>
<p id="Par36">After initial cryo-EM condition optimization at ATPC, RCB, the final high-resolution data were collected at a 300 kVa Titan Krios microscope equipped with Falcon 3 direct electron detector camera (ThermoFisher) at the National Electron Cryo-Microscopy Facility, Bangalore Life Science Cluster (BLiSc), Bangalore. The data sets were collected in an electron integration Movie mode. For the 53S ribosomal subunit, 3470 Movie stacks were collected with 25 frames per Movie stack at 75,000× magnification with 1.07 Å pixel and an electron dose of 1.34 e<sup>-</sup>/Å<sup>2</sup>/frame in electron integration mode using a Falcon 3 electron detector camera (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). For 75S ribosome, 5724 Movie stacks (dataset 1) and 6116 Movie stacks (dataset 2) were collected in a similar setting as that of 53S ribosomal subunit data collection, with a similar electron dose of 1.34 e<sup>-</sup>/Å<sup>2</sup>/frame 75,000× magnification with 1.07 Å pixel (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>).</p></section><section id="Sec17"><h3 class="pmc_sec_title">75S ribosome paromomycin complex EM data collection</h3>
<p id="Par37">The purified 75S ribosomes, stored at −80 °C, were thawed and spun at 20,000 × <em>g</em> for 10 min. 200 nM of 75S ribosomes were incubated with a 1000-time excess of PAR (0.2 mM) at 25 °C for 15 min and immediately used for EM analysis. Initial sample screening was performed by following steps similar to those of the associated 75S ribosome. The samples were screened at room temperature, negative staining using JEOL JEM1400 120 kVa electron microscope, and at cryo-condition using JEOL JEM2200FS 200 kVa electron microscope equipped with GATAN K2 camera at ATPC, RCB Faridabad. High-resolution cryo-EM data were collected using a 300 kVa Titan Krios microscope equipped with a Falcon 3 direct electron detector camera (ThermoFisher) at the National Electron Cryo-Microscopy Facility, Bangalore Life Science Cluster (BLiSc), Bangalore. A total of 4,688 Movie stacks were collected with 32 frames per Movie stack at 75,000× magnification with 1.07 Å pixel with an electron dose of 1.09 e<sup>-</sup>/Å<sup>2</sup>/frame in electron integration mode using a Falcon 3 electron detector camera (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Single particle reconstruction</h3>
<p id="Par38">RELION 3.1.4<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> was used for single particle reconstruction of all three, 53S, 75S and 75S-PAR, ribosome 3D reconstruction (Supplementary Figs. <a href="#MOESM1" class="usa-link">S2</a>–<a href="#MOESM1" class="usa-link">S5</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). The Movie stacks were drift corrected using RELION’s own implementation, and single micrographs were generated. The micrographs were CTF corrected using CTFFIND4<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Initially, particles were picked manually, and 2D reference images were generated for automatic particle picking. 763,731 auto-picked particles for the 53S ribosomal subunit were subjected to 2D classification, and the best classes with 554,832 particles were selected for 3D classification. A 60 Å low-pass filtered 80S ribosome cryo-EM map (EMDB ID 2660) from <em>Plasmodium falciparum</em><sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> was used as a reference map. The three best 3D classes, consisting of 447,323 particles, were subjected to 3D refinement, which yielded a consensus map of 3.1 Å resolution after post-processing. The gold-standard FSC = 0.143 criterion<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> was used for resolution estimation throughout data processing. These 447,323 particles were subjected to CTF refinement and Bayesian polishing, followed by 3D refinement and post-processing, which improved the overall map quality and resolution to 2.8 Å (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Further, the particles were subjected to a 3D classification into 3 classes without alignment, and one class with 378,061 particles, which has homogeneous particles, was selected. The particles were 3D refined and post-processed to 2.8 Å resolution. The gold-standard FSC = 0.143 criterion<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> was used for final resolution estimation. Local resolution was estimated using ResMap<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, embedded in RELION. The final map was auto-sharpened using Phenix.auto_sharpen<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p>
<p id="Par39">Similar steps to those of 53S data processing were followed for the 75S ribosome refinement. For dataset 1, 100,475 particles were auto-picked using 2D reference images generated after manual particle picking and 2D classification. The auto-picked particles were subjected to 2D classification, and the best classes consisting of 45,379 particles were selected for 3D refinement, and a consensus cryo-EM map was generated at 3.6 Å resolution. A low-pass filtered to 60 Å cryo-EM map (EMDB ID 2660) <em>from Plasmodium falciparum</em><sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> was used as a reference map in the 3D refinement. The particles were polished using CTF refinement and Bayesian polishing, which further improved the overall resolution to 3.3 Å. A 3D classification into 3 classes without alignment was performed on the polished particles. The best class with 37,355 (82%), was subjected to 3D refinement and post-processing, which yielded a 3.3 Å resolution cryo-EM map (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). For dataset 2, similar data processing steps were followed. 96,868 particles were selected after 2D and 3D classification, which yielded a 3.0 Å resolution map after particle polishing (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>).</p>
<p id="Par40">The polished particles from data 1 and data 2, a total of 142,247 particles, were merged, and a consensus map at 3.0 Å resolution was generated. Further, a 3D classification into 4 classes without alignment was performed. Two best classes emerged, class 1 having 39,958 (29%) particles with tRNA bound A/P- and P/E-sites of the ribosome, and another class (class 4) having 53,764 (38%) particles with tRNA bound at the P-site of the ribosome. Meanwhile, 29% of particles yield a low-resolution class, and 2% of particles remain unaligned. The 3D refinement and post-processing yielded a 3.1 Å resolution cryo-EM map for both classes. A partial signal subtraction followed by focused refinement<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> was performed to further improve the quality and resolution of the LSU, SSU body, and SSU head map. After post-processing, the large subunit resolution improved to 3 Å for both classes. Whereas the class 1, SSU body and SSU head resolved to 3.2 Å and 3.4 Å resolution, respectively. Similarly, the class 4, SSU body and SSU head resolved to 3.1 Å and 3.4 Å resolution, respectively (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). The resolution was estimated using the gold-standard FSC = 0.143 criterion<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> and local resolution was calculated using ResMap<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). The Phenix.auto_sharpen was used to auto-sharpen the final maps<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p>
<p id="Par41">The 75S-PAR data was processed by following similar steps. Finally, the selected 54,889 particles yielded a 3.0 Å resolution cryo-EM map after particle polishing and 3D classification without alignment. A partial signal subtraction followed by focused refinement<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> for SSU body was performed<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> to improve the local resolution (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Model building and structure analysis</h3>
<p id="Par42">rRNA and r-protein coordinates were built by coalescing both approaches, which are template-based and de novo model building. The atomic coordinates of <em>Toxoplasma gondii</em> (PDB ID: 5XXB)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> and <em>Homo sapiens</em> (PDB ID: 4UG0<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>) were used for template-based modeling, where ribosomal rRNAs, 28S, 5S, and 5.8S were rigid-body docked into 53S cryo-EM using Chimera<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. By convention, 28S rRNA was divided into 6 domains, and each was rigid-body docked (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). The regions not fitting into the cryo-EM maps were deleted, and the nucleotide residues were mutated to the <em>E. histolytica</em> 25S rRNA, 5.8S, and 5S rRNA sequences using COOT v.0.9.3<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. The nucleotide sequences were manually built in the cryo-EM map using COOT for the remaining segments of rRNA. The r-protein modules were initially built using Model Angelo<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> without the FASTA sequence, followed by threading of the <em>Entamoeba</em> sequence on the carbon backbone using PHYRE2<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. The iterative steps of manual model building, real-space refinement, and geometry regularization were done in COOT, then phenix.real_space_refinement was used for the flexible refinement<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>.</p>
<p id="Par43">For model building in the 75S, the atomic coordinates of the 53S ribosomal subunit were rigid docked in cryo-EM map of the LSU and followed by flexible refinement using phenix.real_space_refinement. The atomic coordinates of the 36S ribosomal subunit were built by following steps similar to that of the 53S model building in the 53S cryo-EM map described above. The final coordinates of the 53S and 36S were rigid-body docked in the 75S cryo-EM map and followed by flexible refinement using phenix.real_space_refinement. The model-building statistics are given in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>. Unless otherwise stated, all figures were prepared in Chimera and ChimeraX<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. The RNA 2D diagram was prepared using the R2DT tool<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> at RNA Central using <em>Saccharomyces cerevisiae</em> and <em>Homo sapiens</em> as templates, RNAPDBee<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>–<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, and Adobe Illustrator were also used.</p>
<p id="Par44">Similarly, for the model building of the 75S-PAR complex, the coordinates of individual subunits were docked, followed by rigid body and flexible fitting using Phenix. The coordinates of PAR were docked from the <em>E. coli</em> ribosome PAR complex, PDB 7K00. Figure <a href="#Fig9" class="usa-link">9f</a> was generated using Discovery Studio<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par45">Further information on research design is available in the <a href="#MOESM7" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (30.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62767_MOESM2_ESM.pdf</a><sup> (97.3KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (108.8MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM4_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (79.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (89.5MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM6_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 4</a><sup> (65.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer review file</a><sup> (171.7KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge the National Electron Cryo-Microscopy facility at BLiSc, C-CAMP, Bangalore, and a big thanks to Drs. Vinothkumar and Sucharita for cryo-EM data collection. We acknowledge the Advanced Technology Platform Centre at the Regional Centre for Biotechnology, Faridabad, and thank Dr. Reena and Mr. Madhava for help in the initial cryo-EM sample screening. We acknowledge the computing resource at the Indian Biological Data Centre (IBDC), Faridabad, used for data processing. We thank all PSK lab members for proofreading the manuscript and providing valuable insights. S.S. acknowledges the fellowship from DBT. S.M. acknowledges a Fellowship from UGC and ICMR. This work is supported by a research grant, BT/PR45101/DRUG/134/121/2022, awarded to S.G. and P.S.K. from the Department of Biotechnology, India.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>P.S.K. and S.G. conceived the research project. S.M. and S.S. grew the <em>E. histolytica</em> pellet. S.S. purified and characterized the ribosomes. P.S.K. and S.S. performed the single particle reconstruction, model building, and structure analysis. P.S.K. and S.S. wrote the manuscript with input from S.M. and S.G.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par46"><em>Nature Communications</em> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The cryo-EM maps have been deposited in the <a href="https://www.ebi.ac.uk/emdb/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB</a> and the atomic coordinates have been deposited in the wwPDB (<a href="https://www.wwpdb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.wwpdb.org</a>) with accession codes: <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64694" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64694</a> and <a href="https://doi.org/10.2210/pdb9V1i/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9v1i</a> for ribosomal large subunit (LSU), <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64713</a> and <a href="https://doi.org/10.2210/pdb9V2l/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9v21</a> for 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64697</a> and <a href="https://doi.org/10.2210/pdb9V1l/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V1L</a> for large subunit (LSU) of 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64695</a> and <a href="https://doi.org/10.2210/pdb9V1J/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V1J</a> for small subunit (body) of 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64696</a> and <a href="https://doi.org/10.2210/pdb9V1K/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V1K</a> for small subunit (head) of 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64716</a> and <a href="https://doi.org/10.2210/pdb9V1K/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V24</a> for 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64717</a> and <a href="https://doi.org/10.2210/pdb9V25/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V25</a> for large subunit (LSU) of 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64719" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64719</a> and <a href="https://doi.org/10.2210/pdb9V27/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V27</a> for small subunit (body) of 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64720</a> and <a href="https://doi.org/10.2210/pdb9V28/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V28</a> for small subunit (head) of 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64718</a> and <a href="https://doi.org/10.2210/pdb9V26/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V26</a> for 75S ribosome with A/P- and P/E-tRNAs bound to antibiotic paromomycin and <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64721</a> and <a href="https://doi.org/10.2210/pdb9V29/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V29</a> for small subunit (body) of 75S ribosome with A/P- and P/E-tRNAs bound to antibiotic paromomycin.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par47">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62767-x.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Stanley, S. L. Amoebiasis. <em> Lancet</em><strong>361</strong>, 1025–1034 (2003).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(03)12830-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12660071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanley,%20S.%20L.%20Amoebiasis.%20Lancet361,%201025%E2%80%931034%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Nespola, B. et al. First case of amebic liver abscess 22 years after the first occurrence. <em>Parasite</em><strong>22</strong>, 20 (2015).
</cite> [<a href="https://doi.org/10.1051/parasite/2015020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4472968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26088504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nespola,%20B.%20et%20al.%20First%20case%20of%20amebic%20liver%20abscess%2022%20years%20after%20the%20first%20occurrence.%20Parasite22,%2020%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ryan, K. J., Ray, C. G. &amp; Sherris, J. C. <em>Sherris Medical Microbiology: An Introduction to Infectious Diseases</em><a href="https://hslic-unm.on.worldcat.org/oclc/52358530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hslic-unm.on.worldcat.org/oclc/52358530</a> (McGraw-Hill, 2004).</cite>
</li>
<li id="CR4">
<span class="label">4.</span><cite>Shirley, D. A. T., Farr, L., Watanabe, K. &amp; Moonah, S. A review of the global burden, new diagnostics, and current Therapeutics for amebiasis. <em>Open Forum Infect. Dis.</em><strong>5</strong>, ofy161 (2018).
</cite> [<a href="https://doi.org/10.1093/ofid/ofy161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6055529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30046644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shirley,%20D.%20A.%20T.,%20Farr,%20L.,%20Watanabe,%20K.%20&amp;%20Moonah,%20S.%20A%20review%20of%20the%20global%20burden,%20new%20diagnostics,%20and%20current%20Therapeutics%20for%20amebiasis.%20Open%20Forum%20Infect.%20Dis.5,%20ofy161%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Barker, D. C. &amp; Deutsch, K. The chromatoid body of Entamoeba invadens. <em>Exp. Cell Res</em>. <strong>15</strong>, 604–610 (1958).</cite> [<a href="https://doi.org/10.1016/0014-4827(58)90108-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13609637/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Morgan, R. S. &amp; Uzman, B. G. Nature of the packing of ribosomes within chromatoid bodies. <em>Science</em><strong>152</strong>, 214–216 (1966).
</cite> [<a href="https://doi.org/10.1126/science.152.3719.214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12325350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgan,%20R.%20S.%20&amp;%20Uzman,%20B.%20G.%20Nature%20of%20the%20packing%20of%20ribosomes%20within%20chromatoid%20bodies.%20Science152,%20214%E2%80%93216%20(1966)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Morgan, R. S., Slayter, H. S. &amp; Weller, D. L. Isolation of ribosomes from cysts of Entamoeba invadens. <em>J. Cell Biol.</em><strong>36</strong>, 45–51 (1968).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/19866725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgan,%20R.%20S.,%20Slayter,%20H.%20S.%20&amp;%20Weller,%20D.%20L.%20Isolation%20of%20ribosomes%20from%20cysts%20of%20Entamoeba%20invadens.%20J.%20Cell%20Biol.36,%2045%E2%80%9351%20(1968)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Lake, J. A. &amp; Slayter, H. S. Three dimensional structure of the chromatoid body of Entamoeba invadens. <em>Nature</em><strong>227</strong>, 1032–1037 (1970).
</cite> [<a href="https://doi.org/10.1038/2271032a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5449766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lake,%20J.%20A.%20&amp;%20Slayter,%20H.%20S.%20Three%20dimensional%20structure%20of%20the%20chromatoid%20body%20of%20Entamoeba%20invadens.%20Nature227,%201032%E2%80%931037%20(1970)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lake, J. A. &amp; Slayter, H. S. Three-dimensional structure of the chromatoid body helix of Entamoeba invadens. <em>J. Mol. Biol.</em><strong>66</strong>, 271–282 (1972).
</cite> [<a href="https://doi.org/10.1016/0022-2836(72)90478-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5038453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lake,%20J.%20A.%20&amp;%20Slayter,%20H.%20S.%20Three-dimensional%20structure%20of%20the%20chromatoid%20body%20helix%20of%20Entamoeba%20invadens.%20J.%20Mol.%20Biol.66,%20271%E2%80%93282%20(1972)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Morgan, R. S. Structure of ribosomes of chromatoid bodies: Three-dimensional Fourier synthesis at low resolution. <em>Science</em><strong>162</strong>, 670–81 (1968).
</cite> [<a href="https://doi.org/10.1126/science.162.3854.670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4176903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgan,%20R.%20S.%20Structure%20of%20ribosomes%20of%20chromatoid%20bodies:%20Three-dimensional%20Fourier%20synthesis%20at%20low%20resolution.%20Science162,%20670%E2%80%9381%20(1968)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Price, M., Specht, C., Boudreau, R. E., Shaw, D. E. &amp; Weller, D. L. Preliminary characterization of ribosomes of Entamoeba invadens. <em>Mol. Biochem. Parasitol.</em><strong>8</strong>, 137–143 (1983).
</cite> [<a href="https://doi.org/10.1016/0166-6851(83)90005-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6348536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Price,%20M.,%20Specht,%20C.,%20Boudreau,%20R.%20E.,%20Shaw,%20D.%20E.%20&amp;%20Weller,%20D.%20L.%20Preliminary%20characterization%20of%20ribosomes%20of%20Entamoeba%20invadens.%20Mol.%20Biochem.%20Parasitol.8,%20137%E2%80%93143%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Albach, R. A., Prachayasittikul, V. &amp; Heebner, G. M. Isolation and characterization of RNA of Entamoeba histolytica. <em>Mol. Biochem. Parasitol.</em><strong>12</strong>, 261–272 (1984).
</cite> [<a href="https://doi.org/10.1016/0166-6851(84)90083-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6090899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Albach,%20R.%20A.,%20Prachayasittikul,%20V.%20&amp;%20Heebner,%20G.%20M.%20Isolation%20and%20characterization%20of%20RNA%20of%20Entamoeba%20histolytica.%20Mol.%20Biochem.%20Parasitol.12,%20261%E2%80%93272%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Khatter, H., Myasnikov, A. G., Natchiar, S. K. &amp; Klaholz, B. P. Structure of the human 80S ribosome. <em>Nature</em><strong>520</strong>, 640–645 (2015).
</cite> [<a href="https://doi.org/10.1038/nature14427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25901680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khatter,%20H.,%20Myasnikov,%20A.%20G.,%20Natchiar,%20S.%20K.%20&amp;%20Klaholz,%20B.%20P.%20Structure%20of%20the%20human%2080S%20ribosome.%20Nature520,%20640%E2%80%93645%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Nelson, D. L. &amp; Cox, M. M. <em>Lehninger Principles of Biochemistry</em> 7th edn, Vol. 2 (W.H. Free Co., 2017).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Carter, W. A., Levy, H. B. &amp; Diamond, L. S. Protein synthesis by amoebal ribosomes. <em>Nature</em><strong>213</strong>, 10.1038/213722b0 (1967).</cite> [<a href="https://doi.org/10.1038/213722b0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4291696/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kusamrarn, T., Vinijchaikul, K. &amp; Bailey, G. B. Comparison of the structure and function of polysomal and helical ribosomes from Entamoeba Invadens. <em>J. Cell Biol.</em><strong>65</strong>, 540–548 (1975).
</cite> [<a href="https://doi.org/10.1083/jcb.65.3.540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2109441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1133116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kusamrarn,%20T.,%20Vinijchaikul,%20K.%20&amp;%20Bailey,%20G.%20B.%20Comparison%20of%20the%20structure%20and%20function%20of%20polysomal%20and%20helical%20ribosomes%20from%20Entamoeba%20Invadens.%20J.%20Cell%20Biol.65,%20540%E2%80%93548%20(1975)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>ENTNER, N. &amp; GROLLMAN, A. P. Inhibition of protein synthesis: a mechanism of amebicide action of emetine and other structurally related compounds. <em>J. Protozool.</em><strong>20</strong>, 160–163 (1973).
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.1973.tb06025.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4347870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?ENTNER,%20N.%20&amp;%20GROLLMAN,%20A.%20P.%20Inhibition%20of%20protein%20synthesis:%20a%20mechanism%20of%20amebicide%20action%20of%20emetine%20and%20other%20structurally%20related%20compounds.%20J.%20Protozool.20,%20160%E2%80%93163%20(1973)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>ENTNER, N. Emetine binding to ribosomes of Entamoeba histolytica—inhibition of protein synthesis and amebicidal action. <em>J. Protozool.</em><strong>26</strong>, 324–328 (1979).
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.1979.tb02789.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/226696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?ENTNER,%20N.%20Emetine%20binding%20to%20ribosomes%20of%20Entamoeba%20histolytica%E2%80%94inhibition%20of%20protein%20synthesis%20and%20amebicidal%20action.%20J.%20Protozool.26,%20324%E2%80%93328%20(1979)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Orozco, E., de la Cruz Hernández, F. &amp; Rodríguez, M. A. Isolation and characterization of Entamoeba histolytica mutants resistant to emetine. <em>Mol. Biochem. Parasitol.</em><strong>15</strong>, 49–59 (1985).
</cite> [<a href="https://doi.org/10.1016/0166-6851(85)90028-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2859522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orozco,%20E.,%20de%20la%20Cruz%20Hern%C3%A1ndez,%20F.%20&amp;%20Rodr%C3%ADguez,%20M.%20A.%20Isolation%20and%20characterization%20of%20Entamoeba%20histolytica%20mutants%20resistant%20to%20emetine.%20Mol.%20Biochem.%20Parasitol.15,%2049%E2%80%9359%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Gonzales, M. L. M., Dans, L. F. &amp; Sio-Aguilar, J. Antiamoebic drugs for treating amoebic colitis. <em>Cochrane Database Syst. Rev.</em><strong>2019</strong>, 10.1002/14651858.CD006085.pub3 (2019).</cite> [<a href="https://doi.org/10.1002/14651858.CD006085.pub3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6326239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30624763/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>World Health Organization. WHO model prescribing information: drugs used in parasitic diseases, 2nd ed. World Health Organization. <a href="https://iris.who.int/handle/10665/41765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://iris.who.int/handle/10665/41765</a> (1995).</cite>
</li>
<li id="CR22">
<span class="label">22.</span><cite>Hashem, Y. et al. High-resolution cryo-electron microscopy structure of the Trypanosoma brucei ribosome. <em>Nature</em><strong>494</strong>, 385–389 (2013).
</cite> [<a href="https://doi.org/10.1038/nature11872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3659406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23395961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hashem,%20Y.%20et%20al.%20High-resolution%20cryo-electron%20microscopy%20structure%20of%20the%20Trypanosoma%20brucei%20ribosome.%20Nature494,%20385%E2%80%93389%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Wong, W. et al. Cryo-EM structure of the Plasmodium falciparum 80S ribosome bound to the anti-protozoan drug emetine. <em>eLife</em><strong>2014</strong>, e03080 (2014).</cite> [<a href="https://doi.org/10.7554/eLife.03080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24913268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong,%20W.%20et%20al.%20Cryo-EM%20structure%20of%20the%20Plasmodium%20falciparum%2080S%20ribosome%20bound%20to%20the%20anti-protozoan%20drug%20emetine.%20eLife2014,%20e03080%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Shalev-Benami, M. et al. Atomic resolution snapshot of Leishmania ribosome inhibition by the aminoglycoside paromomycin. <em>Nat. Commun.</em><strong>8</strong>, 1589 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-01664-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5693986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29150609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shalev-Benami,%20M.%20et%20al.%20Atomic%20resolution%20snapshot%20of%20Leishmania%20ribosome%20inhibition%20by%20the%20aminoglycoside%20paromomycin.%20Nat.%20Commun.8,%201589%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Li, Z. et al. Cryo-EM structures of the 80S ribosomes from human parasites Trichomonas vaginalis and Toxoplasma gondii. <em>Cell Res.</em><strong>27</strong>, 1275–1288 (2017).
</cite> [<a href="https://doi.org/10.1038/cr.2017.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5630675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28809395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.%20et%20al.%20Cryo-EM%20structures%20of%20the%2080S%20ribosomes%20from%20human%20parasites%20Trichomonas%20vaginalis%20and%20Toxoplasma%20gondii.%20Cell%20Res.27,%201275%E2%80%931288%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Matzov, D. et al. Cryo-EM structure of the highly atypical cytoplasmic ribosome of Euglena gracilis. <em>Nucleic Acids Res.</em><strong>48</strong>, 11750–11761 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7672448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33091122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matzov,%20D.%20et%20al.%20Cryo-EM%20structure%20of%20the%20highly%20atypical%20cytoplasmic%20ribosome%20of%20Euglena%20gracilis.%20Nucleic%20Acids%20Res.48,%2011750%E2%80%9311761%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Hiregange, D. G. et al. Cryo-EM structure of the ancient eukaryotic ribosome from the human parasite Giardia lamblia. <em>Nucleic Acids Res.</em><strong>50</strong>, 1770–1782 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkac046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8860606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35100413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hiregange,%20D.%20G.%20et%20al.%20Cryo-EM%20structure%20of%20the%20ancient%20eukaryotic%20ribosome%20from%20the%20human%20parasite%20Giardia%20lamblia.%20Nucleic%20Acids%20Res.50,%201770%E2%80%931782%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Majumdar, S. et al. Insights into translocation mechanism and ribosome evolution from cryo-EM structures of translocation intermediates of Giardia intestinalis. <em>Nucleic Acids Res.</em><strong>51</strong>, 3436–3451 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10123126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36912103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Majumdar,%20S.%20et%20al.%20Insights%20into%20translocation%20mechanism%20and%20ribosome%20evolution%20from%20cryo-EM%20structures%20of%20translocation%20intermediates%20of%20Giardia%20intestinalis.%20Nucleic%20Acids%20Res.51,%203436%E2%80%933451%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Eiler, D. R. et al. The Giardia lamblia ribosome structure reveals divergence in several biological pathways and the mode of emetine function. <em>Structure</em>10.1016/j.str.2023.12.015 (2024).</cite> [<a href="https://doi.org/10.1016/j.str.2023.12.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10997490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38242118/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Petrov, A. S. et al. Secondary structures of rRNAs from all three domains of life. <em>PLoS ONE</em><strong>9</strong>, e88222 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0088222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3914948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24505437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petrov,%20A.%20S.%20et%20al.%20Secondary%20structures%20of%20rRNAs%20from%20all%20three%20domains%20of%20life.%20PLoS%20ONE9,%20e88222%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Petrov, A. S. et al. Secondary structure and domain architecture of the 23S and 5S rRNAs. <em>Nucleic Acids Res.</em><strong>41</strong>, 7522–7535 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gkt513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3753638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23771137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petrov,%20A.%20S.%20et%20al.%20Secondary%20structure%20and%20domain%20architecture%20of%20the%2023S%20and%205S%20rRNAs.%20Nucleic%20Acids%20Res.41,%207522%E2%80%937535%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Shankar, V. et al. RRNA expansion segment 27Lb modulates the factor recruitment capacity of the yeast ribosome and shapes the proteome. <em>Nucleic Acids Res.</em><strong>48</strong>, 3244–3256 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7102955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31960048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shankar,%20V.%20et%20al.%20RRNA%20expansion%20segment%2027Lb%20modulates%20the%20factor%20recruitment%20capacity%20of%20the%20yeast%20ribosome%20and%20shapes%20the%20proteome.%20Nucleic%20Acids%20Res.48,%203244%E2%80%933256%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Fujii, K., Susanto, T. T., Saurabh, S. &amp; Barna, M. Decoding the function of expansion segments in ribosomes. <em>Mol. Cell</em><strong>72</strong>, 1013–1020.e6 (2018).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2018.11.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6407129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30576652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujii,%20K.,%20Susanto,%20T.%20T.,%20Saurabh,%20S.%20&amp;%20Barna,%20M.%20Decoding%20the%20function%20of%20expansion%20segments%20in%20ribosomes.%20Mol.%20Cell72,%201013%E2%80%931020.e6%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Pfeffer, S. et al. Structure and 3D arrangement of endoplasmic reticulum membrane-associated ribosomes. <em>Structure</em><strong>20</strong>, 508–518 (2012).</cite> [<a href="https://doi.org/10.1016/j.str.2012.06.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22819217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pfeffer,%20S.%20et%20al.%20Structure%20and%203D%20arrangement%20of%20endoplasmic%20reticulum%20membrane-associated%20ribosomes.%20Structure20,%20508%E2%80%93518%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Ban, N., Nissen, P., Hansen, J., Moore, P. B. &amp; Steitz, T. A. The complete atomic structure of the large ribosomal subunit at 2.4 Å resolution. <em>Science</em><strong>289</strong>, 905–920 (2000).
</cite> [<a href="https://doi.org/10.1126/science.289.5481.905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10937989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ban,%20N.,%20Nissen,%20P.,%20Hansen,%20J.,%20Moore,%20P.%20B.%20&amp;%20Steitz,%20T.%20A.%20The%20complete%20atomic%20structure%20of%20the%20large%20ribosomal%20subunit%20at%202.4%20%C3%85%20resolution.%20Science289,%20905%E2%80%93920%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Kater, L. et al. Construction of the Central Protuberance and L1 Stalk during 60S Subunit Biogenesis. <em>Mol. Cell</em><strong>79</strong>, 615–628.e5 (2020).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2020.06.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32668200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kater,%20L.%20et%20al.%20Construction%20of%20the%20Central%20Protuberance%20and%20L1%20Stalk%20during%2060S%20Subunit%20Biogenesis.%20Mol.%20Cell79,%20615%E2%80%93628.e5%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Wimberly, B. T. et al. Structure of the 30S ribosomal subunit. <em>Nature</em><strong>407</strong>, 327–339 (2000).</cite> [<a href="https://doi.org/10.1038/35030006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11014182/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Jenner, L. et al. Crystal structure of the 80S yeast ribosome. <em>Curr. Opin. Struct. Biol.</em><strong>22</strong>, 10.1016/j.sbi.2012.07.013 (2012).</cite> [<a href="https://doi.org/10.1016/j.sbi.2012.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22884264/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Wang, X. et al. Coevolution of ribosomal RNA expansion segment 7Land assembly factor Noc2p specializes the ribosome biogenesis pathway between Saccharomyces cerevisiae and Candida albicans. <em>Nucleic Acids Res.</em><strong>49</strong>, 4655–4667 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8096215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33823547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20Coevolution%20of%20ribosomal%20RNA%20expansion%20segment%207Land%20assembly%20factor%20Noc2p%20specializes%20the%20ribosome%20biogenesis%20pathway%20between%20Saccharomyces%20cerevisiae%20and%20Candida%20albicans.%20Nucleic%20Acids%20Res.49,%204655%E2%80%934667%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Conrad, N. K. The emerging role of triple helices in RNA biology. <em>Wiley Interdiscip. Rev. RNA</em><strong>5</strong>, 10.1002/wrna.1194 (2014).</cite> [<a href="https://doi.org/10.1002/wrna.1194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4721660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24115594/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Brown, J. A. Unraveling the structure and biological functions of RNA triple helices. <em>Wiley Interdiscip. Rev. RNA</em><strong>11</strong>, 10.1002/wrna.1598 (2020).</cite> [<a href="https://doi.org/10.1002/wrna.1598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7583470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32441456/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Nguyen, T. H. D. et al. Cryo-EM structure of substrate-bound human telomerase holoenzyme. <em>Nature</em><strong>557</strong>, 190–195 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0062-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6223129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29695869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20T.%20H.%20D.%20et%20al.%20Cryo-EM%20structure%20of%20substrate-bound%20human%20telomerase%20holoenzyme.%20Nature557,%20190%E2%80%93195%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Bertram, K. et al. Cryo-EM structure of a human spliceosome activated for step 2 of splicing. <em>Nature</em><strong>542</strong>, 318–323 (2017).
</cite> [<a href="https://doi.org/10.1038/nature21079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28076346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bertram,%20K.%20et%20al.%20Cryo-EM%20structure%20of%20a%20human%20spliceosome%20activated%20for%20step%202%20of%20splicing.%20Nature542,%20318%E2%80%93323%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Qu, G. et al. Structure of a group II intron in complex with its reverse transcriptase. <em>Nat. Struct. Mol. Biol.</em><strong>23</strong>, 549–557 (2016).
</cite> [<a href="https://doi.org/10.1038/nsmb.3220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4899178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27136327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qu,%20G.%20et%20al.%20Structure%20of%20a%20group%20II%20intron%20in%20complex%20with%20its%20reverse%20transcriptase.%20Nat.%20Struct.%20Mol.%20Biol.23,%20549%E2%80%93557%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Huang, L., Wang, J., Wilson, T. J. &amp; Lilley, D. M. J. Structure of the Guanidine III Riboswitch. <em>Cell Chem. Biol.</em><strong>24</strong>, 1407–1415.e2 (2017).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2017.08.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5696562/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28988949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20L.,%20Wang,%20J.,%20Wilson,%20T.%20J.%20&amp;%20Lilley,%20D.%20M.%20J.%20Structure%20of%20the%20Guanidine%20III%20Riboswitch.%20Cell%20Chem.%20Biol.24,%201407%E2%80%931415.e2%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Brown, J. A. et al. Structural insights into the stabilization of MALAT1 noncoding RNA by a bipartite triple helix. <em>Nat. Struct. Mol. Biol.</em><strong>21</strong>, 633–640 (2014).
</cite> [<a href="https://doi.org/10.1038/nsmb.2844" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4096706/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24952594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20J.%20A.%20et%20al.%20Structural%20insights%20into%20the%20stabilization%20of%20MALAT1%20noncoding%20RNA%20by%20a%20bipartite%20triple%20helix.%20Nat.%20Struct.%20Mol.%20Biol.21,%20633%E2%80%93640%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Mitton-Fry, R. M., DeGregorio, S. J., Wang, J., Steitz, T. A. &amp; Steitz, J. A. Poly(A) tail recognition by a viral RNA element through assembly of a triple helix. <em>Science</em><strong>330</strong>, 1244–1247 (2010).
</cite> [<a href="https://doi.org/10.1126/science.1195858" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3074936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21109672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitton-Fry,%20R.%20M.,%20DeGregorio,%20S.%20J.,%20Wang,%20J.,%20Steitz,%20T.%20A.%20&amp;%20Steitz,%20J.%20A.%20Poly(A)%20tail%20recognition%20by%20a%20viral%20RNA%20element%20through%20assembly%20of%20a%20triple%20helix.%20Science330,%201244%E2%80%931247%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Watson, Z. L. et al. Structure of the bacterial ribosome at 2 Å resolution. <em>eLife</em><strong>9</strong>, 1–62 (2020).</cite> [<a href="https://doi.org/10.7554/eLife.60482" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7550191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32924932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Watson,%20Z.%20L.%20et%20al.%20Structure%20of%20the%20bacterial%20ribosome%20at%202%20%C3%85%20resolution.%20eLife9,%201%E2%80%9362%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Carter, A. P. et al. Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics. <em>Nature</em><strong>407</strong>, 340–348 (2000).</cite> [<a href="https://doi.org/10.1038/35030019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11014183/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Diamond, L. S., Harlow, D. R. &amp; Cunnick, C. C. A new medium for the axenic cultivation of entamoeba histolytica and other entamoeba. <em>Trans. R. Soc. Trop. Med. Hyg.</em><strong>72</strong>, 431–432 (1978).
</cite> [<a href="https://doi.org/10.1016/0035-9203(78)90144-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/212851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diamond,%20L.%20S.,%20Harlow,%20D.%20R.%20&amp;%20Cunnick,%20C.%20C.%20A%20new%20medium%20for%20the%20axenic%20cultivation%20of%20entamoeba%20histolytica%20and%20other%20entamoeba.%20Trans.%20R.%20Soc.%20Trop.%20Med.%20Hyg.72,%20431%E2%80%93432%20(1978)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gautam, G., Ali, M. S., Bhattacharya, A. &amp; Gourinath, S. EhFP10: a FYVE family GEF interacts with myosin IB to regulate cytoskeletal dynamics during endocytosis in Entamoeba histolytica. <em>PLoS Pathog.</em><strong>15</strong>, e1007573 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1007573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6396940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30779788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gautam,%20G.,%20Ali,%20M.%20S.,%20Bhattacharya,%20A.%20&amp;%20Gourinath,%20S.%20EhFP10:%20a%20FYVE%20family%20GEF%20interacts%20with%20myosin%20IB%20to%20regulate%20cytoskeletal%20dynamics%20during%20endocytosis%20in%20Entamoeba%20histolytica.%20PLoS%20Pathog.15,%20e1007573%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kumar, N., Sharma, S. &amp; Kaushal, P. S. Cryo-EM structure of the mycobacterial 70S ribosome in complex with ribosome hibernation promotion factor RafH. <em>Nat. Commun.</em><strong>15</strong>, 638 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-44879-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10799931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38245551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20N.,%20Sharma,%20S.%20&amp;%20Kaushal,%20P.%20S.%20Cryo-EM%20structure%20of%20the%20mycobacterial%2070S%20ribosome%20in%20complex%20with%20ribosome%20hibernation%20promotion%20factor%20RafH.%20Nat.%20Commun.15,%20638%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. <em>eLife</em><strong>7</strong>, e42166 (2018).
</cite> [<a href="https://doi.org/10.7554/eLife.42166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6250425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30412051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zivanov,%20J.%20et%20al.%20New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3.%20eLife7,%20e42166%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Rohou, A. &amp; Grigorieff, N. CTFFIND4: Fast and accurate defocus estimation from electron micrographs. <em>J. Struct. Biol.</em><strong>192</strong>, 216–221 (2015).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2015.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6760662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26278980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rohou,%20A.%20&amp;%20Grigorieff,%20N.%20CTFFIND4:%20Fast%20and%20accurate%20defocus%20estimation%20from%20electron%20micrographs.%20J.%20Struct.%20Biol.192,%20216%E2%80%93221%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Rosenthal, P. B. &amp; Henderson, R. Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. <em>J. Mol. Biol.</em><strong>333</strong>, 721–745 (2003).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2003.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14568533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenthal,%20P.%20B.%20&amp;%20Henderson,%20R.%20Optimal%20determination%20of%20particle%20orientation,%20absolute%20hand,%20and%20contrast%20loss%20in%20single-particle%20electron%20cryomicroscopy.%20J.%20Mol.%20Biol.333,%20721%E2%80%93745%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Kucukelbir, A., Sigworth, F. J. &amp; Tagare, H. D. Quantifying the local resolution of cryo-EM density maps. <em>Nat. Methods</em><strong>11</strong>, 63–65 (2014).
</cite> [<a href="https://doi.org/10.1038/nmeth.2727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3903095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24213166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kucukelbir,%20A.,%20Sigworth,%20F.%20J.%20&amp;%20Tagare,%20H.%20D.%20Quantifying%20the%20local%20resolution%20of%20cryo-EM%20density%20maps.%20Nat.%20Methods11,%2063%E2%80%9365%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Terwilliger, T. C., Sobolev, O. V., Afonine, P. V. &amp; Adams, P. D. Automated map sharpening by maximization of detail and connectivity. <em>Acta Crystallogr. Sect. D Struct. Biol.</em><strong>74</strong>, 545–559 (2018).
</cite> [<a href="https://doi.org/10.1107/S2059798318004655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Terwilliger,%20T.%20C.,%20Sobolev,%20O.%20V.,%20Afonine,%20P.%20V.%20&amp;%20Adams,%20P.%20D.%20Automated%20map%20sharpening%20by%20maximization%20of%20detail%20and%20connectivity.%20Acta%20Crystallogr.%20Sect.%20D%20Struct.%20Biol.74,%20545%E2%80%93559%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Bai, X. C., Rajendra, E., Yang, G., Shi, Y. &amp; Scheres, S. H. W. Sampling the conformational space of the catalytic subunit of human g-secretase. <em>eLife</em><strong>4</strong>, e11182 (2015).
</cite> [<a href="https://doi.org/10.7554/eLife.11182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4718806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26623517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bai,%20X.%20C.,%20Rajendra,%20E.,%20Yang,%20G.,%20Shi,%20Y.%20&amp;%20Scheres,%20S.%20H.%20W.%20Sampling%20the%20conformational%20space%20of%20the%20catalytic%20subunit%20of%20human%20g-secretase.%20eLife4,%20e11182%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Nguyen, T. H. D. et al. Cryo-EM structure of the yeast U4/U6.U5 tri-snRNP at 3.7 Å resolution. <em>Nature</em><strong>530</strong>, 298–302 (2016).
</cite> [<a href="https://doi.org/10.1038/nature16940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4762201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26829225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20T.%20H.%20D.%20et%20al.%20Cryo-EM%20structure%20of%20the%20yeast%20U4/U6.U5%20tri-snRNP%20at%203.7%20%C3%85%20resolution.%20Nature530,%20298%E2%80%93302%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. <em>J. Comput. Chem.</em><strong>25</strong>, 1605–1612 (2004).
</cite> [<a href="https://doi.org/10.1002/jcc.20084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pettersen,%20E.%20F.%20et%20al.%20UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis.%20J.%20Comput.%20Chem.25,%201605%E2%80%931612%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Casañal, A., Lohkamp, B. &amp; Emsley, P. Current developments in Coot for macromolecular model building of electron cryo-microscopy and crystallographic data. <em>Protein Sci.</em><strong>29</strong>, 1069–1078 (2020).
</cite> [<a href="https://doi.org/10.1002/pro.3791" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7096722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31730249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Casa%C3%B1al,%20A.,%20Lohkamp,%20B.%20&amp;%20Emsley,%20P.%20Current%20developments%20in%20Coot%20for%20macromolecular%20model%20building%20of%20electron%20cryo-microscopy%20and%20crystallographic%20data.%20Protein%20Sci.29,%201069%E2%80%931078%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Jamali, K. et al. Automated model building and protein identification in cryo-EM maps. <em>Nature</em><strong>628</strong>, 450–457 (2024).</cite> [<a href="https://doi.org/10.1038/s41586-024-07215-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11006616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38408488/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Kelley, L. A., Mezulis, S., Yates, C. M., Wass, M. N. &amp; Sternberg, M. J. E. The Phyre2 web portal for protein modeling, prediction and analysis. <em>Nat. Protoc.</em><strong>10</strong>, 845–858 (2015).
</cite> [<a href="https://doi.org/10.1038/nprot.2015.053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5298202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25950237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelley,%20L.%20A.,%20Mezulis,%20S.,%20Yates,%20C.%20M.,%20Wass,%20M.%20N.%20&amp;%20Sternberg,%20M.%20J.%20E.%20The%20Phyre2%20web%20portal%20for%20protein%20modeling,%20prediction%20and%20analysis.%20Nat.%20Protoc.10,%20845%E2%80%93858%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. <em>Acta Crystallogr. Sect. D Struct. Biol.</em><strong>74</strong>, 531–544 (2018).
</cite> [<a href="https://doi.org/10.1107/S2059798318006551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Afonine,%20P.%20V.%20et%20al.%20Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography.%20Acta%20Crystallogr.%20Sect.%20D%20Struct.%20Biol.74,%20531%E2%80%93544%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. <em>Protein Sci.</em><strong>27</strong>, 14–25 (2018).
</cite> [<a href="https://doi.org/10.1002/pro.3235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28710774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goddard,%20T.%20D.%20et%20al.%20UCSF%20ChimeraX:%20meeting%20modern%20challenges%20in%20visualization%20and%20analysis.%20Protein%20Sci.27,%2014%E2%80%9325%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Sweeney, B. A. et al. R2DT is a framework for predicting and visualising RNA secondary structure using templates. <em>Nat. Commun.</em><strong>12</strong>, 3494 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23555-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8190129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34108470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sweeney,%20B.%20A.%20et%20al.%20R2DT%20is%20a%20framework%20for%20predicting%20and%20visualising%20RNA%20secondary%20structure%20using%20templates.%20Nat.%20Commun.12,%203494%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Antczak, M. et al. New algorithms to represent complex pseudoknotted RNA structures in dot-bracket notation. <em>Bioinformatics</em><strong>34</strong>, 1304–1312 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5905660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29236971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Antczak,%20M.%20et%20al.%20New%20algorithms%20to%20represent%20complex%20pseudoknotted%20RNA%20structures%20in%20dot-bracket%20notation.%20Bioinformatics34,%201304%E2%80%931312%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Zok, T. et al. RNApdbee 2.0: multifunctional tool for RNA structure annotation. <em>Nucleic Acids Res.</em><strong>46</strong>, W30–W35 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gky314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29718468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zok,%20T.%20et%20al.%20RNApdbee%202.0:%20multifunctional%20tool%20for%20RNA%20structure%20annotation.%20Nucleic%20Acids%20Res.46,%20W30%E2%80%93W35%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Antczak, M. et al. RNApdbee—a webserver to derive secondary structures from pdb files of knotted and unknotted RNAs. <em>Nucleic Acids Res.</em><strong>42</strong>, W368–W372 (2014).
</cite> [<a href="https://doi.org/10.1093/nar/gku330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24771339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Antczak,%20M.%20et%20al.%20RNApdbee%E2%80%94a%20webserver%20to%20derive%20secondary%20structures%20from%20pdb%20files%20of%20knotted%20and%20unknotted%20RNAs.%20Nucleic%20Acids%20Res.42,%20W368%E2%80%93W372%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Miyata, T. Discovery studio modeling environment. <em>Ensemble</em><strong>17</strong>, 1–382 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Miyata,%20T.%20Discovery%20studio%20modeling%20environment.%20Ensemble17,%201%E2%80%93382%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (30.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62767_MOESM2_ESM.pdf</a><sup> (97.3KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (108.8MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM4_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (79.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (89.5MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM6_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 4</a><sup> (65.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368212/bin/41467_2025_62767_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer review file</a><sup> (171.7KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The cryo-EM maps have been deposited in the <a href="https://www.ebi.ac.uk/emdb/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB</a> and the atomic coordinates have been deposited in the wwPDB (<a href="https://www.wwpdb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.wwpdb.org</a>) with accession codes: <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64694" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64694</a> and <a href="https://doi.org/10.2210/pdb9V1i/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9v1i</a> for ribosomal large subunit (LSU), <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64713</a> and <a href="https://doi.org/10.2210/pdb9V2l/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9v21</a> for 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64697</a> and <a href="https://doi.org/10.2210/pdb9V1l/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V1L</a> for large subunit (LSU) of 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64695</a> and <a href="https://doi.org/10.2210/pdb9V1J/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V1J</a> for small subunit (body) of 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64696</a> and <a href="https://doi.org/10.2210/pdb9V1K/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V1K</a> for small subunit (head) of 75S ribosome with P-tRNA, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64716</a> and <a href="https://doi.org/10.2210/pdb9V1K/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V24</a> for 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64717</a> and <a href="https://doi.org/10.2210/pdb9V25/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V25</a> for large subunit (LSU) of 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64719" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64719</a> and <a href="https://doi.org/10.2210/pdb9V27/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V27</a> for small subunit (body) of 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64720</a> and <a href="https://doi.org/10.2210/pdb9V28/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V28</a> for small subunit (head) of 75S ribosome with A/P- and P/E-tRNAs, <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64718</a> and <a href="https://doi.org/10.2210/pdb9V26/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V26</a> for 75S ribosome with A/P- and P/E-tRNAs bound to antibiotic paromomycin and <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-64721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-64721</a> and <a href="https://doi.org/10.2210/pdb9V29/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V29</a> for small subunit (body) of 75S ribosome with A/P- and P/E-tRNAs bound to antibiotic paromomycin.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62767-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62767.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (14.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368212/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368212/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368212%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368212/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368212/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368212/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835601/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368212/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835601/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368212/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368212/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="gHgV3v16V2M3wYS9lNqdaY8hmYY9BkLZeGidqBDbhLTT76MSKua7XoSF2ehCNYfE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
