
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Novel thrombolytic protease from the rare halophile Brachybacterium paraconglomeratum: bioprocessing and in vitro application - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152553DE8AF44EF30553DE000234EAAB.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcbiot">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360004/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Biotechnology">
<meta name="citation_title" content="Novel thrombolytic protease from the rare halophile Brachybacterium paraconglomeratum: bioprocessing and in vitro application">
<meta name="citation_author" content="Mandragutti Teja">
<meta name="citation_author_institution" content="Department of Biotechnology, Andhra University, Visakhapatnam, 530 003 India">
<meta name="citation_author" content="Tura Safawo Jarso">
<meta name="citation_author_institution" content="Department of Biology (Applied Genetics and Biotechnology Stream), College of Natural Sciences, Salale University, Fiche, 255 Ethiopia">
<meta name="citation_author" content="Dokka Muni Kumar">
<meta name="citation_author_institution" content="Department of Biochemistry, School of Allied and Healthcare Sciences, Malla Reddy University, Hyderabad, 500 100 India">
<meta name="citation_author" content="Sudhakar Godi">
<meta name="citation_author_institution" content="Department of Human Genetics, Andhra University, Visakhapatnam, 530 003 India">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="85">
<meta name="citation_doi" content="10.1186/s12896-025-00967-5">
<meta name="citation_pmid" content="40826056">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360004/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360004/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360004/pdf/12896_2025_Article_967.pdf">
<meta name="description" content="Alkaline proteases are hydrolytic enzymes that play a crucial role in various biological processes. Proteases produced by halophiles showed exceptional efficiency in breaking down the complex structures ranging from pigments to extracellular ...">
<meta name="og:title" content="Novel thrombolytic protease from the rare halophile Brachybacterium paraconglomeratum: bioprocessing and in vitro application">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Alkaline proteases are hydrolytic enzymes that play a crucial role in various biological processes. Proteases produced by halophiles showed exceptional efficiency in breaking down the complex structures ranging from pigments to extracellular ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360004/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12360004">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12896-025-00967-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12896_2025_Article_967.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360004%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12360004/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12360004/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360004/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcbiot.png" alt="BMC Biotechnology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Biotechnology" title="Link to BMC Biotechnology" shape="default" href="https://bmcbiotechnol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Biotechnol</button></div>. 2025 Aug 18;25:85. doi: <a href="https://doi.org/10.1186/s12896-025-00967-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12896-025-00967-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Biotechnol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Biotechnol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Biotechnol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Biotechnol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Novel thrombolytic protease from the rare halophile <em>Brachybacterium paraconglomeratum</em>: bioprocessing and in vitro application</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Teja%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mandragutti Teja</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mandragutti Teja</span></h3>
<div class="p">
<sup>1</sup>Department of Biotechnology, Andhra University, Visakhapatnam, 530 003 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Teja%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mandragutti Teja</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarso%20TS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Tura Safawo Jarso</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tura Safawo Jarso</span></h3>
<div class="p">
<sup>2</sup>Department of Biology (Applied Genetics and Biotechnology Stream), College of Natural Sciences, Salale University, Fiche, 255 Ethiopia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarso%20TS%22%5BAuthor%5D" class="usa-link"><span class="name western">Tura Safawo Jarso</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20DM%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Dokka Muni Kumar</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Dokka Muni Kumar</span></h3>
<div class="p">
<sup>3</sup>Department of Biochemistry, School of Allied and Healthcare Sciences, Malla Reddy University, Hyderabad, 500 100 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20DM%22%5BAuthor%5D" class="usa-link"><span class="name western">Dokka Muni Kumar</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Godi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sudhakar Godi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sudhakar Godi</span></h3>
<div class="p">
<sup>4</sup>Department of Human Genetics, Andhra University, Visakhapatnam, 530 003 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Godi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sudhakar Godi</span></a>
</div>
</div>
<sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biotechnology, Andhra University, Visakhapatnam, 530 003 India </div>
<div id="Aff2">
<sup>2</sup>Department of Biology (Applied Genetics and Biotechnology Stream), College of Natural Sciences, Salale University, Fiche, 255 Ethiopia </div>
<div id="Aff3">
<sup>3</sup>Department of Biochemistry, School of Allied and Healthcare Sciences, Malla Reddy University, Hyderabad, 500 100 India </div>
<div id="Aff4">
<sup>4</sup>Department of Human Genetics, Andhra University, Visakhapatnam, 530 003 India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 7; Accepted 2025 Apr 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12360004  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826056/" class="usa-link">40826056</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Alkaline proteases are hydrolytic enzymes that play a crucial role in various biological processes. Proteases produced by halophiles showed exceptional efficiency in breaking down the complex structures ranging from pigments to extracellular proteins, tissue proteins to tumors, and thrombin clots. There is a huge global demand for naturally occurring thrombolytic proteases to treat intravascular thrombosis, as they are cost-effective and have minimal detrimental effects, making them widely used in the biomedical applications.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">The production of thrombolytic protease from the rare halophile <em>Brachybacterium paraconglomeratum</em> strain M4 and its in-vitro thrombolytic activity is investigated for the first time. On skim milk agar, a distinct zone of casein hydrolysis was observed, in submerged fermentation a substantial protease production was noted. The enzyme was purified through a four-step purification process following an initial precipitation with 60% ammonium sulfate, subsequent ion-exchange chromatography using DEAE-cellulose, and final gel-filtration chromatography utilizing Sephadex G-100. As a result, a protein with a specific activity of 300 ± 32 U/mg was obtained with a purification fold of 19 and a recovery percentage of 38.2%. The molecular mass of the purified enzyme was 14.4 kDa via SDS-PAGE while MALDI/MS analysis further revealed a 131 amino acid sequence with an isoelectric point of 8.55 and on comparison, the strain M4 alkaline protease aligns with the proteasome subunit alpha. The protease is classified as a serine protease based on the inhibition by PMSF and its activity profile. The in-vitro thrombolytic assay revealed that the purified enzyme achieved a clot lysis rate of 65 ± 3%, performing effectively in comparison to the standard.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">The findings indicate that strain M4 is an efficient producer of thrombolytic protease and the purified form effectively dissolves thrombin clots. Currently, the studies are underway to explore its potential for biomedical applications and industrial-scale production.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12896-025-00967-5.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Halophile, <em>Brachybacterium paraconglomeratum</em>, Bioprocessing, MALDI-TOFF, Proteasome, Thrombolytic activity</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">The major complications such as deep vein thrombosis (DVT), pulmonary embolism (PE), and ischemic strokes in the global population are due to the formation of blood clots (thrombosis) in the vascular system. The risk factors for this illness include hypertension, smoking, corticosteroids, physical inactivity, oral contraceptives and low awareness about thrombosis in the general population [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The alkaline proteases sourced from microbes are used more than 40% in hydrolyzing blood clots from the blood vessels and their stability in physiological conditions, making them ideal for various biomedical applications [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. The research is ongoing to optimize or develop low molecular weight thrombolytic proteases to treat this illness because of their reduced side effects, better distribution, and faster clearance from the vessels.</p>
<p id="Par5">The studies proved that the halophilic bacteria that thrive in unique biological niches within marine ecosystems can produce extracellular, thermostable proteases with distinct characteristics and capabilities. Therefore, researchers have turned their attention to the enzymatic potential of halophiles, intending to discover low molecular weight proteases as a significant challenge [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Notably, alkaline proteases derived from halophiles, such as <em>Bacillus</em> sp. and <em>Pseudomonas</em> sp., are known for their high molecular weight but have been explored across various sectors, including food, agriculture, and pharmaceuticals due to their robust activity under alkaline conditions [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>].</p>
<p id="Par6">Microbial thrombolytic proteases, such as streptokinase (SK) and staphylokinase (SAK), have been extensively studied for their efficacy in clot dissolution. Despite their proven effectiveness, several gaps remain in fully understanding and harnessing their potential [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. While these enzymes are associated with minimal side effects, challenges persist in optimizing their efficacies in clinical application, particularly in conditions like myocardial infarction and deep vein thrombosis. Key gaps include a need for improved delivery methods to enhance efficiency, better characterization for long-term effects and expanded research on enzyme penetration activities.</p>
<p id="Par7">The underlying mechanisms of thrombolysis were fibrinolysis and plasminogen activation have been explored however, more research is needed to unravel the nuances of these mechanisms in different pathological conditions [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Furthermore, microbial proteases are increasingly employed in treating myocardial infarction, ischemic stroke, and targeted thrombolytic therapies in combination with other treatment modalities, but this is still in its infancy [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Other widely known plasmin-like proteins include lumbrokinase from earthworms [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>] and fibrolase from snake venom [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>] both of which face limitations such as reduced stability under physiological conditions and in-vivo inhibition. Addressing these gaps is essential for the successful integration of microbial proteases into broader clinical, veterinary, and other biomedical applications.</p>
<p id="Par8">Therefore, the key objectives of this study were to bioprocess the protease from the strain M4, evaluate its stability, identify the enzyme through sequence based analysis, and assess its robustness in thrombolytic applications. The study also sought to address the crucial problem of improving the stability of the enzyme in harsh conditions, underscoring its potential for use in both clinical and industrial settings.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Chemicals and reagents</h3>
<p id="Par9">All chemicals and reagents used in this study were of analytical grade. The substrates, buffers, and other laboratory chemicals were sourced from Hi Media and SRL Chemicals, Mumbai, India. The reagents of SDS-PAGE, DEAE-cellulose and Sephadex G (100,200) and streptokinase (positive control) were obtained from Sigma-Aldrich chemicals, Benguluru, India. The matrix reagents and standards of the MALDI-TOF analysis were procured from Bruker online, Mumbai, India.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Bacteria</h3>
<p id="Par10">Several halophilic bacteria were isolated from the estuarine mangrove region of the Bay of Bengal near Suryalanka, India (15.858952°N, 80.511521°E). The objective was to study their diversity and investigate novel applications in human health and environmental sustainability. During the biochemical screening, the strain M4 demonstrated the highest proteolytic activity among the isolates, leading its selection for further investigation of thrombolytic proteases. The strain M4 was subjected to 16S rRNA gene sequencing and identified as <em>Brachybacterium paraconglomeratum</em> by CSIR-IMTECH, India, and assigned with the accession ID: MTCC 13074. The 16S rRNA gene sequence was subsequently deposited in the NCBI GenBank under the accession ID: <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW899045" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW899045</a>. Following isolation, the isolate was lyophilized, for further research and analysis.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Confirmative assay</h3>
<p id="Par11">In the biochemical tests, the isolate M4 demonstrated the highest casein hydrolytic activity. To validate this, the enzyme producing strain M4 was cultured on sterilized skim milk agar (SMA) medium prepared with 0.5% peptone, 1.0% powdered skimmed milk, 5% NaCl, and 2.5% agar-agar, adjusted to pH 8. Following 24 h of incubation at 37 °C, the plates were examined for a clear zone of hydrolysis [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p></section><section id="Sec6"><h3 class="pmc_sec_title">Production media</h3>
<p id="Par12">The isolate was cultivated in 500 mL Erlenmeyer flasks containing 250 mL sterile optimized media that include glucose,1.5% (w/v); peptone, 1.0% (w/v); 1% NaCl (w/v); inorganic salt solution, 5% (v/v) [(MgSO₄, 0.5% (w/v); KH<sub>2</sub>PO<sub>4</sub>, 0.5% (w/v); and FeSO<sub>4</sub>, 0.01% (w/v)] and adjusted to pH 8.0. For the production, 1% inoculum was introduced and cultured in a shaking incubator (Remi, Mumbai, India) at 160 rpm for 48 h. To obtain a clear supernatant, the broth was centrifuged at 705 g for 20 min at 4 °C. The supernatant was stored at 2–8 °C for further investigations [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Enzyme activity</h3>
<p id="Par13">The supernatant was subjected to protease activity following the Tsuchida et al. method [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. A 200 µL aliquot of crude enzyme extract was incubated with 0.5 mL of 1% (<em>w</em>/<em>v</em>) casein in 50 mM phosphate buffer for 20 min at 40 °C, pH 7.8. The reaction was halted by adding 1 mL of 10% (<em>w</em>/<em>v</em>) TCA and was left for 15 min at 37 °C. After centrifugation at 10,000 rpm for 5 min, the supernatant was mixed with 2.5 mL of 0.44 M Na<sub>2</sub>CO<sub>3</sub> and 1 mL of 2-fold diluted Folin-Ciocalteu’s phenol (FC) reagent. After 30 min of incubation, the absorbance was measured at 660 nm against a reagent blank. A standard graph was created using tyrosine (0–100 µg/mL). Under standard conditions, the enzyme unit of protease activity was defined as the amount of enzyme required to liberate 1 µg of tyrosine per minute per milliliter.</p>
<p id="Par14">Additionally, the total protein content was determined using the Lowry et al. [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] method with bovine serum albumin (BSA) as the reference protein. The specific activity (U/mg) and relative activity (U/mL) were also calculated, as these metrics are crucial for characterizing the protease.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Purification of thrombolytic protease</h3>
<p id="Par15">The enzyme purification followed a four-step method as described by Lakshmi et al. [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Initially, the crude enzyme extract is gradually saturated with ammonium sulfate (10–100%) followed by dialysis against a Tris-HCl buffer (pH 7.5) to remove excess salt. Subsequently, the dialyzed sample was subjected to ion-exchange chromatography using a DEAE-cellulose column (30 × 2.0 cm). Glycine-NaOH buffer (pH 10.0) was used to pre-equilibrate the column. Fractions of 5 mL were collected at a flow rate of 60 mL/h. The unbound active fractions were combined, concentrated, dialyzed against distilled water at 4 °C and lyophilized. Following that, the bound protein was eluted via a linear gradient of 0.1 to 1.0 M NaCl, with 5 mL fractions collected at a flow rate of 60 mL/h. The fractions were dissolved in Glycine-NaOH buffer (pH 10.0), and loaded onto a sephadex G-100 column (30 × 1.5 cm) pre-equilibrated with Glycine-NaOH buffer (pH 10.0). The fraction (2 mL) was eluted with the same buffer and collected at a flow rate of 12 mL/h. At every level the protein content of these fractions was determined at 280 nm absorbance and the enzyme was assayed using casein as the substrate [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p></section><section id="Sec9"><h3 class="pmc_sec_title">SDS-PAGE and zymography</h3>
<p id="Par16">The protein samples were subjected to SDS-PAGE (sodium dodecyl sulfate-polyacrylamide gel electrophoresis) according to the Laemmli (1970) method [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], using a discontinuous buffer system consisting of a stacking gel and a resolving gel. Following electrophoresis, the gel was stained with coomassie brilliant blue R-250 and destained with methanol, acetic acid, and water in the ratio 50:10:40 to visualize the separated protein bands.</p>
<p id="Par17">The Nicodeme et al. (2005) method was used to create the activity gel, with some changes. Co-polymerization of 10% polyacrylamide gel was done with 1% casein [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. The samples were run at 100 V in a non-reducing buffer with no denaturation. To remove SDS, the gel was washed twice in 2.5% Triton-X-100 for 30 min before being incubated in Glycine- NaOH buffer, pH-10.0 for 1 h at 37 °C. The gel was stained for 30 min, with 0.5% coomassie brilliant blue R-250 followed by destaining with methanol, acetic acid, and water in the ratio of 50:10:40. Glycoprotein staining was also done simultaneously.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Determination of molecular weight</h3>
<p id="Par18">The molecular mass of the enzyme was determined by one-dimensional SDS-PAGE under both reducing (with 10% 2-mercaptoethanol) and non-reducing conditions, with some modifications, using a recombinant protein marker (puregene protein) as a reference. The migration distances of standard proteins were plotted, enabling the calculation of molecular weight of the strain M4 protease [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par19">The method of Anderson et al. [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] for molecular sieve chromatography with sephadex G-200 was used to estimate the molecular weight of the strain M4 protease. Sephadex was swelled in 0.1 M sodium phosphate buffer, pH 7.6, and packed in a 1.65 × 94 cm column (bed volume 185 mL). The same buffer was used to equilibrate and develop the column. Blue Dextran was used to calculate the void volume (Vo). Collected 1 mL fractions were analyzed, and the elution volume (Ve) was plotted against log molecular weight.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Characterization of the purified protease</h3>
<p id="Par20">The stability of protease produced by the strain M4 was evaluated under various biochemical conditions. Substrate specificity was determined on different substrates, including casein, gelatin, ovalbumin, and bovine serum albumin. The effect of pH on protease activity was analyzed across a broad pH range (2.0–12.0) using appropriate buffer systems. Additionally, the enzyme’s thermal stability was assessed at temperatures ranging from 30 °C to 80 °C [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>].</p>
<p id="Par21">Enzyme inhibition studies were conducted by pre-incubating the enzyme with specific inhibitors, including phenylmethylsulfonyl fluoride (PMSF), iodoacetic acid (IAA), β-mercaptoethanol, and ethylenediaminetetraacetic acid (EDTA), at concentrations of 5 mM and 10 mM. The metal ion stability was evaluated by incubating the enzyme with various metal ions (CuCl<sub>2</sub>, CaCl<sub>2</sub>, MnCl<sub>2</sub>, ZnCl<sub>2</sub>, NiCl<sub>2</sub>, HgCl<sub>2</sub>, MgCl<sub>2</sub>, CoCl<sub>2</sub>, and PbCl<sub>2</sub>) at the same concentrations. In all experiments, an untreated enzyme control was maintained, and the residual activity percentage was determined for comparison [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>].</p></section><section id="Sec12"><h3 class="pmc_sec_title">Matrix-Assisted Laser Desorption/Ionization-Time of Flight Mass Spectra (MALDI-TOF/MS)</h3>
<p id="Par22">MALDI-TOF/MS ULTRAFLEX III (Bruker Daltonics, Germany) was utilized for the rapid and precise analysis of peptide sequences. Gel bands containing protein samples underwent peptic digestion with trypsin as per the method described by Gao et al. [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The sample preparation involved mixing 1.5 µL of the digested sample with 1.5 µL of α-cyano-4-hydroxycinnamic acid matrix, and 1.5 µL of the resulting mixture was then spotted onto the MALDI plate. Following air drying, analysis was conducted and external calibration was performed using standard peptide (PEPMIX) supplied by Bruker. Subsequent analysis was carried out using FLEX ANALYSIS SOFTWARE (Version 3.3) in reflectron ion mode. Masses obtained in the PMF were then submitted for mascot search to identify the protein sequence.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Protein sequence analysis</h3>
<p id="Par23">Utilizing the p-BLAST programme and CLUSTAL W software, the protease sequence of the strain M4 was evaluated by comparing it to the various protease sequences found in the protein database. The tree was created using the neighbor-joining algorithm [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] and the other methods, used in building the phylogenetic tree were the minimum evolution method used to infer the evolutionary history, and the maximum composite likelihood method to calculate the evolutionary distances.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Thrombolytic activity</h3>
<p id="Par24">An in-vitro clot lysis model was developed to investigate the thrombolytic activity of protease extracted from the strain M4 [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Freshly collected chicken blood from processing units was transferred into pre-weighed vials, approximately 500 µL per vial. The tubes were then incubated at 37 °C for 45 min for the formation of the clot. Then the residual serum was washed away, leaving only the clot in the tubes. The initial clot weight was recorded. Next, 100 µL of the purified enzyme was added to each tube. The tubes were then incubated at 37 °C for 90 min for clot lysis. After the incubation period, the fluid obtained as a result of clot lysis was separated, and the remaining clot was weighed once more. For calculation, the same procedure was conducted simultaneously using streptokinase as a positive control and phosphate-buffered saline (PBS) alone as a negative control. To determine the percentage of clot lysis, the difference between the weights recorded before and after clot disruption was taken into account. By contrasting with streptokinase and PBS, the percentage of weight loss in each tube was taken into consideration as proof of the thrombolytic activity of the purified enzyme.</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/a702a9d148e6/d33e429.gif" loading="lazy" id="d33e429" alt="graphic file with name d33e429.gif"></td></tr></table>
<table class="disp-formula p" id="Equ2"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/3a4c3fe42e60/d33e437.gif" loading="lazy" id="d33e437" alt="graphic file with name d33e437.gif"></td></tr></table></section><section id="Sec15"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par27">All determinations were performed in triplicates, and the activities are presented as the mean standard deviation (mean ± SD). Statistical analysis was performed using excel and origin data analysis tool (Origin 2023).</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Screening of proteolytic activity</h3>
<p id="Par28">The biological properties and other potential applications of <em>B. paraconglomeratum</em> strain M4 have been published previously [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. The isolate M4 demonstrated proteolytic activity on skim milk agar, as indicated by a clear proteolytic zone around the colony (Fig. <a href="#Fig1" class="usa-link">1</a>A and B). The isolate M4 was subsequently subjected to submerged fermentation for protease production.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360004_12896_2025_967_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/ecd027e59059/12896_2025_967_Fig1_HTML.jpg" loading="lazy" id="d33e479" height="366" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Detection of proteolytic activity of strain M4. <strong>B</strong> Zone of hydrolysis on a skim milk agar plate by the strain M4</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Protease activity</h3>
<p id="Par30">The submerged fermented broth was centrifuged and the supernatant was used as the source of enzyme. The filtrates’ aggregate protein content was (13 ± 1 mg/mL), total activity (216 ± 3 U/mL) and the specific activity (16 ± 16 U/mg) was determined.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Purification of protease</h3>
<p id="Par31">The purification of protease from the strain M4 is detailed in Table <a href="#Tab1" class="usa-link">1</a>. Using 60% ammonium sulfate saturation, maximum protein precipitation with high specific activity was achieved. Ion-exchange chromatography on a DEAE-cellulose column resulted in 8.82-fold purification with 48.2% yield and specific activity of 143 ± 06 U/mg. The DEAE-cellulose and G-100 elution profiles pointed to the fractions 20–45, which exhibited the greatest enzyme activity, were chosen for further study. Following G-100 filtration, the enzyme achieved an 18.56-fold purification, with a 38.2% yield and a specific activity of 300 ± 32 U/mg, resulting in an overall ~ 19-fold purification and 38% enzyme recovery from the supernatant.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of the purification process detailing enzyme activity, purity levels, and variability (SD) in the results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Preparation</th>
<th align="left" colspan="1" rowspan="1">Volume (mL)</th>
<th align="left" colspan="1" rowspan="1">Total protein (mg)</th>
<th align="left" colspan="1" rowspan="1">Total activity units/mL</th>
<th align="left" colspan="1" rowspan="1">Specific activity U/mg protein</th>
<th align="left" colspan="1" rowspan="1">Yield %</th>
<th align="left" colspan="1" rowspan="1">Purification fold</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Crude</td>
<td align="center" colspan="1" rowspan="1">200</td>
<td align="center" colspan="1" rowspan="1">2650 ± 16</td>
<td align="center" colspan="1" rowspan="1">43,270 ± 89</td>
<td align="center" colspan="1" rowspan="1">16 ± 16</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">60% ammonium sulphate fractionation</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="center" colspan="1" rowspan="1">640 ± 7</td>
<td align="center" colspan="1" rowspan="1">21,340 ± 50</td>
<td align="center" colspan="1" rowspan="1">33 ± 32</td>
<td align="left" colspan="1" rowspan="1">49.3</td>
<td align="left" colspan="1" rowspan="1">2.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DEAE-cellulose ion-exchange chromatography</td>
<td align="center" colspan="1" rowspan="1">50</td>
<td align="center" colspan="1" rowspan="1">145 ± 5</td>
<td align="center" colspan="1" rowspan="1">20,870 ± 41</td>
<td align="center" colspan="1" rowspan="1">143 ± 26</td>
<td align="left" colspan="1" rowspan="1">48.2</td>
<td align="left" colspan="1" rowspan="1">8.82</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gelfiltration chromatography (Sephadex G-100)</td>
<td align="center" colspan="1" rowspan="1">30</td>
<td align="center" colspan="1" rowspan="1">55 ± 6</td>
<td align="center" colspan="1" rowspan="1">16,540 ± 35</td>
<td align="center" colspan="1" rowspan="1">300 ± 32</td>
<td align="left" colspan="1" rowspan="1">38.2</td>
<td align="left" colspan="1" rowspan="1">18.56</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec20"><h3 class="pmc_sec_title">Homogeneity and molecular weight determination</h3>
<p id="Par33">The purification was confirmed by a single protein band found in the Sephadex G-100 fraction on a 10% gel (Fig. <a href="#Fig2" class="usa-link">2</a>A, <a href="#MOESM1" class="usa-link">S1</a>A). A Coomassie Blue stained band was observed in the zymogram, showing a clear zone of casein hydrolysis around the protein band corresponding to the protease (Fig. <a href="#Fig2" class="usa-link">2</a>A). PAS staining showed no reaction, confirming the absence of carbohydrate moieties in the protease (Fig. <a href="#Fig2" class="usa-link">2</a>A). The apparent molecular mass of the purified protease was 14.4 kDa (Fig. <a href="#Fig2" class="usa-link">2</a>B, <a href="#MOESM1" class="usa-link">S1</a>B) via reducing and non-reducing conditions of SDS-PAGE whereas it was estimated at approx. 14.4 kDa by Sephadex G-200 gel filtration. The molecular weight of the enzyme was calculated from the elution volume ratio (<em>Ve</em>/<em>Vo</em>, where <em>Ve</em> is the elution volume of the enzyme, and <em>Vo</em> is the void volume of the column) versus the log molecular weight.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360004_12896_2025_967_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/1fab7bcfe778/12896_2025_967_Fig2_HTML.jpg" loading="lazy" id="d33e634" height="385" width="744" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Native PAGE validating the purification and purified protease of the strain M4 showing a distinct band corresponding to its active form, confirmed through substrate activity staining. Lane-1: Ammonium sulfate fraction, Lane-2: Ion exchange fraction, Lane-3: Gel filtration fraction, Lane-4: PAS staining, Lane 5: Zymogram: Specific/activity staining. <strong>B</strong> SDS-PAGE on 5–15% slab gel showing protein ladder with molecular weights in kDa and the purified protease of the strain M4 showing a distinct band corresponding to its molecular weight</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Activity profile of the strain M4 protease</h3>
<p id="Par35">The protease produced by the strain M4 demonstrated the highest substrate specificity towards casein, exhibiting an activity of 216 ± 0.84. The optimum pH for their activity was determined to be pH 10.0, with 98% relative activity. Among the tested temperature range of 30–80 °C, the maximum activity was observed at 70 °C. The presence of divalent metal ions, including Ca²⁺, Mg²⁺, Mn²⁺, and Cu²⁺ at a concentration of 10 mM, enhanced the enzyme’s activity. In contrast, PMSF at a concentration of 5 mM completely inhibited the activity, while other inhibitors had minimal impact on the enzyme activity. The results were compared with the prior studies and are summarized in Table <a href="#Tab2" class="usa-link">2</a>, which thus classifies the strain M4 protease as serine kind of protease.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Important biochemical findings of the strain M4 protease compared with selected microbial proteases</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Microorganism</th>
<th align="left" colspan="1" rowspan="1">Type</th>
<th align="left" colspan="1" rowspan="1">Substrate specificity</th>
<th align="left" colspan="1" rowspan="1">Optimal pH</th>
<th align="left" colspan="1" rowspan="1">Optimal (Temp<sup>o</sup>)</th>
<th align="left" colspan="1" rowspan="1">Inhibitor</th>
<th align="left" colspan="1" rowspan="1">Metal ions</th>
<th align="left" colspan="1" rowspan="1">Specific activity U/mg</th>
<th align="left" colspan="1" rowspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Serratia marcescens</em>
</td>
<td align="left" colspan="1" rowspan="1">Metallo proteinase</td>
<td align="left" colspan="1" rowspan="1">Casein</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">
<p>5 mM 1,10-Phenathroline</p>
<p>5 mM EDTA</p>
</td>
<td align="left" colspan="1" rowspan="1">1 mM Ni<sup>2+</sup>
</td>
<td align="left" colspan="1" rowspan="1">42,388</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Actinomadura viridilutea</em>
</td>
<td align="left" colspan="1" rowspan="1">ASP</td>
<td align="left" colspan="1" rowspan="1">Casein</td>
<td align="center" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">PMSF, DFP</td>
<td align="left" colspan="1" rowspan="1">2 mM Ca<sup>2+</sup>Mg<sup>2+</sup> Fe<sup>2+</sup>
</td>
<td align="left" colspan="1" rowspan="1">67,200 ± 615</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Bacillus halodurans</em>
</td>
<td align="left" colspan="1" rowspan="1">ASP</td>
<td align="left" colspan="1" rowspan="1">Casein</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">50</td>
<td align="left" colspan="1" rowspan="1">EDTA</td>
<td align="left" colspan="1" rowspan="1">Ca<sup>2+</sup>, Mg<sup>2+</sup> Mn<sup>2+</sup>
</td>
<td align="left" colspan="1" rowspan="1">509.84</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Aspergillus niger</em>
</td>
<td align="left" colspan="1" rowspan="1">Metallo protease</td>
<td align="left" colspan="1" rowspan="1">Casein</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">EDTA</td>
<td align="left" colspan="1" rowspan="1">5 mM Ca<sup>2+</sup>
</td>
<td align="left" colspan="1" rowspan="1">165.92</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Bacillus subtilis</em>
</td>
<td align="left" colspan="1" rowspan="1">Serine protease</td>
<td align="left" colspan="1" rowspan="1">Casein</td>
<td align="center" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">55</td>
<td align="left" colspan="1" rowspan="1">5 mM EDTA</td>
<td align="left" colspan="1" rowspan="1">Mg<sup>2+</sup>
</td>
<td align="left" colspan="1" rowspan="1">769.1</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Stremptoyces nogalator</em>
</td>
<td align="left" colspan="1" rowspan="1">Cysteine protease</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">EDTA, PMSF</td>
<td align="left" colspan="1" rowspan="1">Mn<sup>2+</sup>
</td>
<td align="left" colspan="1" rowspan="1">78.65</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Stremptoyces ambofaciens</em>
</td>
<td align="left" colspan="1" rowspan="1">Novel</td>
<td align="left" colspan="1" rowspan="1">Casein</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">20.8</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">* M4 protease</td>
<td align="left" colspan="1" rowspan="1">ASP</td>
<td align="left" colspan="1" rowspan="1">Casein</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">PMSF</td>
<td align="left" colspan="1" rowspan="1">Mn<sup>2+</sup> Ca<sup>2+</sup> Cu<sup>2+</sup>Mg<sup>2+</sup>
</td>
<td align="left" colspan="1" rowspan="1">300 ± 32</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p44"><p>ASP—Alkaline serine protease, * M4 protease produced by <em>B. paraconglomeratum</em></p></div></div></section></section><section id="Sec22"><h3 class="pmc_sec_title">MALDI-TOF/Peptide Mass Fingerprinting (PMF)</h3>
<p id="Par37">The MALDI-TOF analysis of the purified alkaline protease revealed a molecular mass of 14.4 kDa, as expected mass for this enzyme. The mass spectra (Fig. <a href="#Fig3" class="usa-link">3</a>) showed a prominent peptide peaks between 500 and 1500 Da confirming the molecular weight obtained from SDS-PAGE analysis.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360004_12896_2025_967_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/67afd0dbad3f/12896_2025_967_Fig3_HTML.jpg" loading="lazy" id="d33e976" height="491" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MALDI-TOF spectrum of the peptide fragments. The x-axis represents the mass-to-charge ratio (<em>m</em>/<em>z</em>), indicating the molecular weights of detected ions, while the y-axis shows the relative intensity, reflecting the abundance of each ion species</p></figcaption></figure><p id="Par38">The PMF revealed partial homology with proteasome subunit alpha of the <em>Arthrobacter</em> sp. with a score of 42 in the PMF database. The total amino acids are 131 and the isoelectric point was determined to be 8.55. The identified peaks and their corresponding masses provided insights into the enzyme’s sequence, confirming it as a serine protease.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Amino acid sequence of the strain M4 protease</h3>
<p id="Par39">The amino acid sequence of the strain M4 protease is as follows: &gt;MKDRADFARKGIARGRSVVKIAFAVGKYNEFESLQAGVRYADVRGSYDREDVTAGLASIADEKFIVMGGQADKVAETVEGGQIPASALEVAVLDRASESTRGSRRAFRRLGDDEAALLSKEN.</p>
<p id="Par40">The PIR information on the amino acid sequence and residue specific data for the enzyme characterization are highlighted in the Fig. <a href="#Fig4" class="usa-link">4</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360004_12896_2025_967_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/6a3b557bf875/12896_2025_967_Fig4_HTML.jpg" loading="lazy" id="d33e986" height="698" width="732" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The protein information resource (PIR) data on the amino acid sequence and total residue composition of purified protease of the strain M4</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">Phylogenetic analysis</h3>
<p id="Par41">The phylogenetic analysis of the strain M4 alkaline protease was constructed using the data from ClustalW, protein database searches and p-BLAST, revealing approximately 57% sequence homology with the unclassified <em>Arthrobacter</em> proteasome subunit alpha (Fig. <a href="#Fig5" class="usa-link">5</a>). However, further analysis is needed to confirm functional similarities with the established <em>Arthrobacter</em> proteins.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360004_12896_2025_967_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/19f61d372a76/12896_2025_967_Fig5_HTML.jpg" loading="lazy" id="d33e1000" height="323" width="732" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The phylogenetic tree was constructed using the neighbor-joining method to illustrate the evolutionary relationship between the protease of <em>B. paraconglomeratum</em> strain M4, and other closely related proteases of the bacterial species</p></figcaption></figure></section><section id="Sec25"><h3 class="pmc_sec_title">Thrombolytic activity</h3>
<p id="Par45">The in-vitro thrombolytic activity of the strain M4 protease was assessed through blood clot lysis in three separate vials, as shown in Fig. <a href="#Fig6" class="usa-link">6</a>. The results indicated that purified protease achieved 65 ± 3% clot lysis, while the positive control, streptokinase (Std), showed 82 ± 5%, and the negative control (PBS) exhibited less than 10 ± 4% clot lysis, as shown in Fig. <a href="#Fig7" class="usa-link">7</a>. Based on these results, the purified protease of the strain M4 demonstrates promising potential and is expected to perform comparably to the standard after optimizing the thrombolytic conditions.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/f94b460f96e3/12896_2025_967_Fig6_HTML.jpg" loading="lazy" id="d33e1033" height="1150" width="1547" alt="Fig. 6"></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>In-vitro thrombolytic activity of the strain M4 protease. <strong>A</strong> Blood samples were allowed to coagulate completely in separate vials, reaching 100% coagulation. <strong>B</strong> The vials were then inverted to show only the blood clots. <strong>C</strong> Alkaline protease (AP), standard streptokinase (ST), and a negative control (NC) were added to the blood clots and incubated at 37 °C for 45 min. <strong>D</strong> Clot dissolution was observed with AP and ST, indicating thrombolytic activity, while no lysis was seen with the NC</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360004_12896_2025_967_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1028/12360004/6bdfb957ebc1/12896_2025_967_Fig7_HTML.jpg" loading="lazy" id="d33e1043" height="443" width="744" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Depicting the percentage of clot lysis across the thrombolytic agents tested in the study</p></figcaption></figure></section></section><section id="Sec26"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par48">Alkaline proteases in halophiles are typically produced as a response to the high-salt environments, aiding in nutrient acquisition, stress adaptation, and the breakdown of complex proteins under extreme conditions. This study reports the production of thrombolytic protease from rare halophile <em>B. paraconglomeratum</em> for the first time. Under the controlled conditions, the strain M4 efficiently demonstrated the hydrolytic activity. In purification, ammonium sulfate precipitation was used effectively to purify the protease, a common method for protein concentration across animal, plant, and microbial sources. Similar precipitation approaches have been applied to various proteases, including extracellular protease from <em>Serratia marcescens</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] and venom-derived proteases from <em>Nemopilema nomurai</em> [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Likewise, <em>Actinomadura viridilutea</em> DZ50’s keratinolytic protease exhibited comparable precipitation rates and high enzyme activity [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p>
<p id="Par49">Similarly, DEAE-cellulose and Sephadex G-100 help to refine the enzyme and achieve higher purity, yielding consistent results across different microbial sources involving <em>Bacillus halodurans</em> [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>], <em>Bacillus caseinilyticus</em>, <em>Aspergillus flavus</em> [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>], and <em>Stenotrophomonas maltophilia</em> [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. The reports on serine protease from <em>Periplaneta americana</em> and <em>Bacillus alveayuensis</em> showed yields from 12 to 46%, highlighting the need to optimize conditions for each strain and column gel size [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. In total a 19-fold increase in protein purity, aligned with Lakshmi et al. [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] confirms the reliability and reproducibility of the purification methods across various studies.</p>
<p id="Par50">The activity profile of the strain M4 protease closely aligns with the activities of protease produced by the <em>Serratia</em> and <em>Bacillus</em> species [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. When compared with the existing data presented in Table <a href="#Tab2" class="usa-link">2</a>, the strain M4 protease demonstrates higher activity under extreme temperature and pH. The observed enzymatic activity in the presence of divalent metal ions further supports its versatility, while its complete inhibition by PMSF strongly indicates that it belongs to the serine type protease.</p>
<p id="Par51">The homogeneity and molecular weights of alkaline proteases ranging from 15 to 44 kDa were obtained from various studies and evaluated via complementary techniques such as SDS-PAGE and gel filtration (G-200) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. <em>B. paraconglomeratum</em> produced a 14.4 kDa subunit of proteolytic complex, similar to the serine protease from <em>B. cereus</em> [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The precise mass peaks were identified for the peptide fragments of the subunit alpha proteasome complex using MALDI-TOF and once again confirmed the molecular mass of the enzyme [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Over the past three decades, the combination of MALDI/PMF and MASCOT, PROFOUND tools has proven highly effective for peptide identification [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. The amino acid sequence and functional domain conservation together showed 57% sequence homology with the unclassified <em>Arthrobacter</em> proteasome subunit alpha, suggesting a shared ancestry and potentially similar functions [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Hence, the biochemical analysis, G-200 gel filtration findings, MALDI analysis and PMF data were in complete agreement in unveiling the enzyme’s key characteristics.</p>
<p id="Par52">The observations of in-vitro activity are consistent with previous reports on therapeutic agents from <em>B. cereus</em> [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] and <em>Indigofera cassioides</em> extracts [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>], indicating that the protease could have applications in preventing myocardial infarction. The strain M4 protease, as a bacterial activator similar to streptokinase, functions by converting plasminogen into plasmin, leading to the dissolution of fibrin clots. This mechanism firmly places the extracted protease within the category of first-generation thrombolytic agents [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. The strain M4 protease exhibits impressive clot lysis activity on par with the commercial proteases as highlighted in Table <a href="#Tab3" class="usa-link">3</a>. Gene ontology predicts that the strain M4 protease retains endopeptidase activity, essential for breaking down of internal peptide bonds [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] during proteolysis and cellular protein degradation.</p>
<section class="tw xbox font-sm" id="Tab3"><h3 class="obj_head">Table 3.</h3>
<div class="caption p"><p>Thrombolytic activities and mode of action of commercial proteases and the experimental protease of the strain M4</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Protease</th>
<th align="left" colspan="1" rowspan="1">Source</th>
<th align="left" colspan="1" rowspan="1">Thrombolytic activity %</th>
<th align="left" colspan="1" rowspan="1">Usage</th>
<th align="left" colspan="1" rowspan="1">Mechanism</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Streptase<sup>®</sup>
</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptococcus</em> spp.</td>
<td align="left" colspan="1" rowspan="1">101 ± 4</td>
<td align="left" colspan="1" rowspan="1">Therapeutic drug</td>
<td align="left" colspan="1" rowspan="1">Activates plasminogen cascade</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Heberkinase</td>
<td align="left" colspan="1" rowspan="1">Recom., <em>E. coli</em>
</td>
<td align="left" colspan="1" rowspan="1">92 ± 2</td>
<td align="left" colspan="1" rowspan="1">Therapeutic drug</td>
<td align="left" colspan="1" rowspan="1">Converts plasminogen directly</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Staphylokinase</td>
<td align="left" colspan="1" rowspan="1">
<em>Staphylococcus aureus</em>
</td>
<td align="left" colspan="1" rowspan="1">46</td>
<td align="left" colspan="1" rowspan="1">Clinical trails</td>
<td align="left" colspan="1" rowspan="1">Plasminogen activation</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Urokinase</td>
<td align="left" colspan="1" rowspan="1">Urine</td>
<td align="left" colspan="1" rowspan="1">80</td>
<td align="left" colspan="1" rowspan="1">Therapeutic drug</td>
<td align="left" colspan="1" rowspan="1">Plasminogen activation</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Nattokinase</td>
<td align="left" colspan="1" rowspan="1">
<em>Bacillus subtilis</em>
</td>
<td align="left" colspan="1" rowspan="1">80</td>
<td align="left" colspan="1" rowspan="1">Clinical, dietary</td>
<td align="left" colspan="1" rowspan="1">Direct fibrin degradation</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Lumbrokinase</td>
<td align="left" colspan="1" rowspan="1">
<em>Lumbricus rubellus</em>
</td>
<td align="left" colspan="1" rowspan="1">92</td>
<td align="left" colspan="1" rowspan="1">Clinical</td>
<td align="left" colspan="1" rowspan="1">Direct fibrin breakdown</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fibrolase</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>Agkistrodon</em>
</p>
<p>
<em>contortrix contortrix</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">Clinical</td>
<td align="left" colspan="1" rowspan="1">Direct fibrin cleavage</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M4 Protease</td>
<td align="left" colspan="1" rowspan="1">
<em>Brachybacterium paraconglomeratum</em>
</td>
<td align="left" colspan="1" rowspan="1">65 ± 4</td>
<td align="left" colspan="1" rowspan="1">Under research</td>
<td align="left" colspan="1" rowspan="1">Endopeptidase activity</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par53">In this study, the halophilic <em>B. paraconglomeratum</em> produces an alkaline protease of low molecular weight, making it an ideal candidate for therapeutic applications. The further research will delve into the detailed enzyme kinetics, strain improvement for better yield of enzyme, optimization of thrombolytic conditions and progressive exploration of toxicity and preclinical studies.</p></section><section id="Sec27"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par55">This research highlights a novel source for safer and more effective thrombolytic agent. We are the first to demonstrate the thrombolytic potential of the proteasome complex from rare <em>B. paraconglomeratum</em> strain M4. The proteolytic activity was validated on skim milk agar. The enzyme purified through the chromatography process exhibited strong purity levels and reached a final specific activity of 300 ± 32 U/mg. The molecular mass analysis confirmed the protease to be 14.4 kDa, aligning structurally and functionally with known proteasome subunit alpha proteins. MALDI–TOF and PMF analysis identified 131 amino acids in the alpha peptide. In-vitro, the alkaline protease demonstrated significant clot lysis (65 ± 3%) which was comparable to streptokinase thereby classifying it as a first-generation thrombolytic agent. This in-vitro study demonstrates that after optimization, toxicity and in-vivo studies this low molecular weight protease can effectively use as therapeutic agent for cardiovascular and cerebrovascular diseases.</p></section><section id="Sec28"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12360004/bin/12896_2025_967_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (159.3KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>All the authors are extremely grateful to the Department of Biotechnology and Analytical Laboratory (DST) at Andhra University for providing the necessary facilities to complete this work.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.T. conceptualized, performed the experiments, and initially drafted the research article and revised the manuscript for intellectual content. G.S: supervision, review and validation. T.S.J and D.M.K. drafted, proofread, and reviewed the manuscript for intellectual content. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The authors did not receive any specific grants or funds for this research.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated during the current study is provided within the manuscript or supplementary information files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par56">Ethical clearance was not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par57">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par58">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Mandragutti Teja, Email: tejaaron75@gmail.com.</p>
<p>Tura Safawo Jarso, Email: turajarso21@gmail.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Agarwal S, Lee AD, Raju RS, Stephen E. Venous thromboembolism: A problem in the Indian/Asian population? Indian J Uro. 2009;25:11–6. 10.4103/0970-1591.45531.</cite> [<a href="https://doi.org/10.4103/0970-1591.45531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2684304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19468423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agarwal%20S,%20Lee%20AD,%20Raju%20RS,%20Stephen%20E.%20Venous%20thromboembolism:%20A%20problem%20in%20the%20Indian/Asian%20population?%20Indian%20J%20Uro.%202009;25:11%E2%80%936.%2010.4103/0970-1591.45531." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Naveed M, Nadeem F, Mehmood T, Bilal M, Anwar Z, Amjad F. Protease—A versatile and ecofriendly biocatalyst with multi-industrial applications: an updated review. Catal Lett. 2021;151:307–23.</cite> [<a href="https://scholar.google.com/scholar_lookup?Naveed%20M,%20Nadeem%20F,%20Mehmood%20T,%20Bilal%20M,%20Anwar%20Z,%20Amjad%20F.%20Protease%E2%80%94A%20versatile%20and%20ecofriendly%20biocatalyst%20with%20multi-industrial%20applications:%20an%20updated%20review.%20Catal%20Lett.%202021;151:307%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ibrahim EN, Ma K. Industrial applications of thermostable enzymes from extremophilic microorganisms. Curr Biochem Eng. 2017;4(2):75–98. 10.2174/2212711904666170405123414.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ibrahim%20EN,%20Ma%20K.%20Industrial%20applications%20of%20thermostable%20enzymes%20from%20extremophilic%20microorganisms.%20Curr%20Biochem%20Eng.%202017;4(2):75%E2%80%9398.%2010.2174/2212711904666170405123414." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Haddar A, Agrebi R, Bougatef A, Hmidet N, Sellami-Kamoun A, Nasri M. Two detergent stable alkaline serine-proteases from Bacillus mojavensis A21: purification, characterization and potential application as a laundry detergent additive. Bioresour Technol. 2009;100(13):3366–73.
</cite> [<a href="https://doi.org/10.1016/j.biortech.2009.01.061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19269812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haddar%20A,%20Agrebi%20R,%20Bougatef%20A,%20Hmidet%20N,%20Sellami-Kamoun%20A,%20Nasri%20M.%20Two%20detergent%20stable%20alkaline%20serine-proteases%20from%20Bacillus%20mojavensis%20A21:%20purification,%20characterization%20and%20potential%20application%20as%20a%20laundry%20detergent%20additive.%20Bioresour%20Technol.%202009;100(13):3366%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gupta R, Beg QK, Lorenz P. Bacterial alkaline proteases: molecular approaches and industrial applications. Appl Microbiol Biotechnol. 2002;59(1):15–32.
</cite> [<a href="https://doi.org/10.1007/s00253-002-0975-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12073127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta%20R,%20Beg%20QK,%20Lorenz%20P.%20Bacterial%20alkaline%20proteases:%20molecular%20approaches%20and%20industrial%20applications.%20Appl%20Microbiol%20Biotechnol.%202002;59(1):15%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Cuiling W, Dan L, Xinghao Y, Ribang W, Jiang Z, Jiafeng H, Hailun H. Improving production of protease from Pseudoalteromonas Sp. CSN423 by random mutagenesis. Mar Biotechnol. 2016;18(5):610–8. 10.1007/s10126-016-9721-9.</cite> [<a href="https://doi.org/10.1007/s10126-016-9721-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27752851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cuiling%20W,%20Dan%20L,%20Xinghao%20Y,%20Ribang%20W,%20Jiang%20Z,%20Jiafeng%20H,%20Hailun%20H.%20Improving%20production%20of%20protease%20from%20Pseudoalteromonas%20Sp.%20CSN423%20by%20random%20mutagenesis.%20Mar%20Biotechnol.%202016;18(5):610%E2%80%938.%2010.1007/s10126-016-9721-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Jackson KW, Esmon N, Tang J, Streptokinase. Staphylokinase Methods Enzymol. 1981;80:87–394. 10.1016/s0076-6879(81)80033-x.</cite> [<a href="https://doi.org/10.1016/s0076-6879(81)80033-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7043195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackson%20KW,%20Esmon%20N,%20Tang%20J,%20Streptokinase.%20Staphylokinase%20Methods%20Enzymol.%201981;80:87%E2%80%93394.%2010.1016/s0076-6879(81)80033-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Akhtar T, Hoq MM, Mazid MA. Bacterial proteases as thrombolytics and fibrinolytics. Dhaka Univ J Pharm Sci. 2017;16(2):255–69.</cite> [<a href="https://scholar.google.com/scholar_lookup?Akhtar%20T,%20Hoq%20MM,%20Mazid%20MA.%20Bacterial%20proteases%20as%20thrombolytics%20and%20fibrinolytics.%20Dhaka%20Univ%20J%20Pharm%20Sci.%202017;16(2):255%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Hadjidj R, Badis A, Mechri S, Eddouaouda K, Khelouia L, Annane R, Hattab M, Jaouadi B. Purification, biochemical and molecular characterization of novel protease from <em>Bacillus licheniformis</em> strain K7A. Int J Biol Macromol. 2009;114:1033–48.</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2018.03.167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29605249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hadjidj%20R,%20Badis%20A,%20Mechri%20S,%20Eddouaouda%20K,%20Khelouia%20L,%20Annane%20R,%20Hattab%20M,%20Jaouadi%20B.%20Purification,%20biochemical%20and%20molecular%20characterization%20of%20novel%20protease%20from%20Bacillus%20licheniformis%20strain%20K7A.%20Int%20J%20Biol%20Macromol.%202009;114:1033%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Diwan D, Usmani Z, Sharma M, Nelson JW, Thakur VK, Christie G, Molina G, Gupta VK. Thrombolytic enzymes of microbial origin: A review. Int J Mol Sci. 2021;22(19):10468. 10.3390/ijms221910468.
</cite> [<a href="https://doi.org/10.3390/ijms221910468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8508633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34638809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diwan%20D,%20Usmani%20Z,%20Sharma%20M,%20Nelson%20JW,%20Thakur%20VK,%20Christie%20G,%20Molina%20G,%20Gupta%20VK.%20Thrombolytic%20enzymes%20of%20microbial%20origin:%20A%20review.%20Int%20J%20Mol%20Sci.%202021;22(19):10468.%2010.3390/ijms221910468." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Mihara H, Sumi H, Yoneta T, Mizumoto H, Ikeda R, Seiki M, Maruyama M. A novel fibrinolytic enzyme extracted from the earthworm, <em>Lumbricus rubellus</em>. Jpn J Physiol. 1991;41(3):461–72. 10.2170/jjphysiol.41.461.
</cite> [<a href="https://doi.org/10.2170/jjphysiol.41.461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1960890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mihara%20H,%20Sumi%20H,%20Yoneta%20T,%20Mizumoto%20H,%20Ikeda%20R,%20Seiki%20M,%20Maruyama%20M.%20A%20novel%20fibrinolytic%20enzyme%20extracted%20from%20the%20earthworm,%20Lumbricus%20rubellus.%20Jpn%20J%20Physiol.%201991;41(3):461%E2%80%9372.%2010.2170/jjphysiol.41.461." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Chen HÁ, Guan AÁ, Markland FS Jr. Immunological properties of the fibrinolytic enzyme (fibrolase) from Southern Copperhead (<em>Agkistrodon contortrix contortrix</em>) venom and its purification by immunoaffinity chromatography. Toxicon. 1991;29(6):683–94. 10.1016/0041-0101(91)90060-5.
</cite> [<a href="https://doi.org/10.1016/0041-0101(91)90060-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1926169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20H%C3%81,%20Guan%20A%C3%81,%20Markland%20FS%20Jr.%20Immunological%20properties%20of%20the%20fibrinolytic%20enzyme%20(fibrolase)%20from%20Southern%20Copperhead%20(Agkistrodon%20contortrix%20contortrix)%20venom%20and%20its%20purification%20by%20immunoaffinity%20chromatography.%20Toxicon.%201991;29(6):683%E2%80%9394.%2010.1016/0041-0101(91)90060-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Wehr HM, Frank JF. Standard methods for the examination of dairy products. 17th ed. Washington, DC: American Public Health Association; 2004. 10.2105/9780875530024.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wehr%20HM,%20Frank%20JF.%20Standard%20methods%20for%20the%20examination%20of%20dairy%20products.%2017th%20ed.%20Washington,%20DC:%20American%20Public%20Health%20Association;%202004.%2010.2105/9780875530024." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Taneja K, Bajaj BK, Kumar S, Dilbaghi N. Production, purification and characterization of fibrinolytic enzyme from <em>Serratia</em> Sp. KG-2-1 using optimized media. 3 Biotech. 2017;7(3):184. 10.1007/s13205-017-0808-4.
</cite> [<a href="https://doi.org/10.1007/s13205-017-0808-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5491434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28664371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taneja%20K,%20Bajaj%20BK,%20Kumar%20S,%20Dilbaghi%20N.%20Production,%20purification%20and%20characterization%20of%20fibrinolytic%20enzyme%20from%20Serratia%20Sp.%20KG-2-1%20using%20optimized%20media.%203%20Biotech.%202017;7(3):184.%2010.1007/s13205-017-0808-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Tsuchida O, Yamagota Y, Ishizuka J, Arai J, Yamada J, Takeuchi M, Ichishima E. An alkaline proteinase of an alkalophilic <em>Bacillus</em> Sp. Curr Microbiol. 1986;14:7–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?Tsuchida%20O,%20Yamagota%20Y,%20Ishizuka%20J,%20Arai%20J,%20Yamada%20J,%20Takeuchi%20M,%20Ichishima%20E.%20An%20alkaline%20proteinase%20of%20an%20alkalophilic%20Bacillus%20Sp.%20Curr%20Microbiol.%201986;14:7%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Lowry OH, Rosebrough NJ, Farr AL, Randall RL. Protein measurement with the Folin phenol reagent. J Biol Chem. 1951;193(1):265–75.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/14907713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lowry%20OH,%20Rosebrough%20NJ,%20Farr%20AL,%20Randall%20RL.%20Protein%20measurement%20with%20the%20Folin%20phenol%20reagent.%20J%20Biol%20Chem.%201951;193(1):265%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Lakshmi B, Muni Kumar D, Hemalatha K. Purification and characterization of alkaline protease with novel properties from <em>Bacillus cereus</em> strain S8. J Genet Eng Biotechnol. 2018;16(2):295–304. 10.1016/j.jgeb.2018.05.009.
</cite> [<a href="https://doi.org/10.1016/j.jgeb.2018.05.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6354005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30733738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lakshmi%20B,%20Muni%20Kumar%20D,%20Hemalatha%20K.%20Purification%20and%20characterization%20of%20alkaline%20protease%20with%20novel%20properties%20from%20Bacillus%20cereus%20strain%20S8.%20J%20Genet%20Eng%20Biotechnol.%202018;16(2):295%E2%80%93304.%2010.1016/j.jgeb.2018.05.009." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Bayoudh A, Gharsallah N, Chamkha M, Dhouib A, Ammar S, Nasri M. Purification and characterization of an alkaline protease from <em>Pseudomonas aeruginosa</em> MN1. J Indust Microbiol Biotechnol. 2000;24(4):291–5. 10.1038/sj.jim.2900822.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bayoudh%20A,%20Gharsallah%20N,%20Chamkha%20M,%20Dhouib%20A,%20Ammar%20S,%20Nasri%20M.%20Purification%20and%20characterization%20of%20an%20alkaline%20protease%20from%20Pseudomonas%20aeruginosa%20MN1.%20J%20Indust%20Microbiol%20Biotechnol.%202000;24(4):291%E2%80%935.%2010.1038/sj.jim.2900822." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Thangam EB, Rajkumar GS. Purification and characterization of alkaline protease from <em>Alcaligenes faecalis</em>. Biotechnol Appl Biochem. 2002;35(2):149. 10.1042/ba20010013.
</cite> [<a href="https://doi.org/10.1042/ba20010013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11916457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thangam%20EB,%20Rajkumar%20GS.%20Purification%20and%20characterization%20of%20alkaline%20protease%20from%20Alcaligenes%20faecalis.%20Biotechnol%20Appl%20Biochem.%202002;35(2):149.%2010.1042/ba20010013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Laemmli UK. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature. 1970;227(5259):680–5. 10.1038/227680a0.
</cite> [<a href="https://doi.org/10.1038/227680a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5432063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laemmli%20UK.%20Cleavage%20of%20structural%20proteins%20during%20the%20assembly%20of%20the%20head%20of%20bacteriophage%20T4.%20Nature.%201970;227(5259):680%E2%80%935.%2010.1038/227680a0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Nicodeme M, Grill JP, Humbert G, Gaillard JL. Extracellular protease activity of different <em>Pseudomonas</em> strains: dependence of proteolytic activity on culture conditions. J Appl Microbiol. 2005;99(3):641–8.
</cite> [<a href="https://doi.org/10.1111/j.1365-2672.2005.02634.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16108806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicodeme%20M,%20Grill%20JP,%20Humbert%20G,%20Gaillard%20JL.%20Extracellular%20protease%20activity%20of%20different%20Pseudomonas%20strains:%20dependence%20of%20proteolytic%20activity%20on%20culture%20conditions.%20J%20Appl%20Microbiol.%202005;99(3):641%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Marshall T. Sodium Dodecyl sulfate polyacrylamide gel electrophoresis of serum after protein denaturation in the presence or absence of 2-mercaptoethanol. Clin Chem. 1984;30(3):475–9. 10.1093/clinchem/30.3.475.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/6697503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marshall%20T.%20Sodium%20Dodecyl%20sulfate%20polyacrylamide%20gel%20electrophoresis%20of%20serum%20after%20protein%20denaturation%20in%20the%20presence%20or%20absence%20of%202-mercaptoethanol.%20Clin%20Chem.%201984;30(3):475%E2%80%939.%2010.1093/clinchem/30.3.475." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Anderson DMW, Dea ICM, Hendrie A. Molecular-sieve chromatography. Sel Annu Rev Anal. 1971;1:1–40. 10.1039/as9710100001.</cite> [<a href="https://scholar.google.com/scholar_lookup?Anderson%20DMW,%20Dea%20ICM,%20Hendrie%20A.%20Molecular-sieve%20chromatography.%20Sel%20Annu%20Rev%20Anal.%201971;1:1%E2%80%9340.%2010.1039/as9710100001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Gao R, Shi T, Liu X, Zhao M, Cui H, Yuan L. Purification and characterisation of a salt-stable protease from the halophilic archaeon <em>Halogranum rubrum</em>. J Sci Food Agric. 2016;97(5):1412–9. 10.1002/jsfa.7879.
</cite> [<a href="https://doi.org/10.1002/jsfa.7879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27378532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20R,%20Shi%20T,%20Liu%20X,%20Zhao%20M,%20Cui%20H,%20Yuan%20L.%20Purification%20and%20characterisation%20of%20a%20salt-stable%20protease%20from%20the%20halophilic%20archaeon%20Halogranum%20rubrum.%20J%20Sci%20Food%20Agric.%202016;97(5):1412%E2%80%939.%2010.1002/jsfa.7879." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Darabi M, Seddigh S. Bioinformatic characterization of aspartic protease (AP) enzyme in seed plants. Plant Syst Evol. 2015;301(10):2399–417. 10.1007/s00606-015-1236-8.</cite> [<a href="https://scholar.google.com/scholar_lookup?Darabi%20M,%20Seddigh%20S.%20Bioinformatic%20characterization%20of%20aspartic%20protease%20(AP)%20enzyme%20in%20seed%20plants.%20Plant%20Syst%20Evol.%202015;301(10):2399%E2%80%93417.%2010.1007/s00606-015-1236-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Prasad S, Kashyap RS, Deopujari JY, Purohit HJ, Taori GM, Daginawala HF. Development of an in vitro model to study clot Lysis activity of thrombolytic drugs. Thromb J. 2006;4(1):14. 10.1186/1477-9560-4-14.
</cite> [<a href="https://doi.org/10.1186/1477-9560-4-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1570448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16968529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prasad%20S,%20Kashyap%20RS,%20Deopujari%20JY,%20Purohit%20HJ,%20Taori%20GM,%20Daginawala%20HF.%20Development%20of%20an%20in%20vitro%20model%20to%20study%20clot%20Lysis%20activity%20of%20thrombolytic%20drugs.%20Thromb%20J.%202006;4(1):14.%2010.1186/1477-9560-4-14." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Mandragutti T, Tura SJ, Sudhakar G, Sharmila B, Beulah K. Physicochemical characterization of polyhydroxybutyrate (PHB) produced by the rare halophile <em>Brachybacterium paraconglomeratum</em> MTCC 13074. Microb Cell Fact. 2024;23(1):59. 10.1186/s12934-024-02324-1.
</cite> [<a href="https://doi.org/10.1186/s12934-024-02324-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10882773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38388436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mandragutti%20T,%20Tura%20SJ,%20Sudhakar%20G,%20Sharmila%20B,%20Beulah%20K.%20Physicochemical%20characterization%20of%20polyhydroxybutyrate%20(PHB)%20produced%20by%20the%20rare%20halophile%20Brachybacterium%20paraconglomeratum%20MTCC%2013074.%20Microb%20Cell%20Fact.%202024;23(1):59.%2010.1186/s12934-024-02324-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Mandragutti T, Sudhakar G. Selective isolation and genomic characterization of biopolymer producer—A novel feature of halophile <em>Brachybacterium paraconglomeratum</em> MTCC 13074. J Genet Eng Biotechnol. 2023;21:2. 10.1186/s43141-023-00484-y.
</cite> [<a href="https://doi.org/10.1186/s43141-023-00484-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9975135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36853280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mandragutti%20T,%20Sudhakar%20G.%20Selective%20isolation%20and%20genomic%20characterization%20of%20biopolymer%20producer%E2%80%94A%20novel%20feature%20of%20halophile%20Brachybacterium%20paraconglomeratum%20MTCC%2013074.%20J%20Genet%20Eng%20Biotechnol.%202023;21:2.%2010.1186/s43141-023-00484-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Wan MH. Screening, characterization, and cloning of a solvent-tolerant protease from <em>Serratia marcescens</em> MH6. J Microbiol Biotechnol. 2010;20(5):881–8. 10.4014/jmb.0910.10038.
</cite> [<a href="https://doi.org/10.4014/jmb.0910.10038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20519911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan%20MH.%20Screening,%20characterization,%20and%20cloning%20of%20a%20solvent-tolerant%20protease%20from%20Serratia%20marcescens%20MH6.%20J%20Microbiol%20Biotechnol.%202010;20(5):881%E2%80%938.%2010.4014/jmb.0910.10038." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Yu C, Li R, Yin X, Yu H, Li P. Synergistic effect of proteinase activity by purification and identification of toxic protease from <em>Nemopilema nomurai</em>. Front Pharmacol. 2021;12:791847. 10.3389/fphar.2021.791847.
</cite> [<a href="https://doi.org/10.3389/fphar.2021.791847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8660593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34899353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20C,%20Li%20R,%20Yin%20X,%20Yu%20H,%20Li%20P.%20Synergistic%20effect%20of%20proteinase%20activity%20by%20purification%20and%20identification%20of%20toxic%20protease%20from%20Nemopilema%20nomurai.%20Front%20Pharmacol.%202021;12:791847.%2010.3389/fphar.2021.791847." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Ben EM, Zaraî JN, Rekik H, Omrane M, Mechri S, Moujehed E, Kourdali S, Hattab EM, Badis A, Bejar S, et al. Biochemical and molecular characterization of new keratinolytic protease from <em>Actinomadura viridilutea</em> DZ50. Int J Biol Macromol. 2016;92:299–315. 10.1016/j.ijbiomac.2016.07.009.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2016.07.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27387016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ben%20EM,%20Zara%C3%AE%20JN,%20Rekik%20H,%20Omrane%20M,%20Mechri%20S,%20Moujehed%20E,%20Kourdali%20S,%20Hattab%20EM,%20Badis%20A,%20Bejar%20S,%20et%20al.%20Biochemical%20and%20molecular%20characterization%20of%20new%20keratinolytic%20protease%20from%20Actinomadura%20viridilutea%20DZ50.%20Int%20J%20Biol%20Macromol.%202016;92:299%E2%80%93315.%2010.1016/j.ijbiomac.2016.07.009." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Annamalai N, Rajeswari MV, Thavasi R, Vijayalakshmi S, Balasubramanian T. Optimization, purification and characterization of novel thermostable, haloalkaline, solvent-stable protease from <em>Bacillus halodurans</em> CAS6 using marine shellfish wastes: A potential additive for detergent and antioxidant synthesis. Bioprocess Biosyst Eng. 2013;36(7):873–83. 10.1007/s00449-012-0820-3.
</cite> [<a href="https://doi.org/10.1007/s00449-012-0820-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22983319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Annamalai%20N,%20Rajeswari%20MV,%20Thavasi%20R,%20Vijayalakshmi%20S,%20Balasubramanian%20T.%20Optimization,%20purification%20and%20characterization%20of%20novel%20thermostable,%20haloalkaline,%20solvent-stable%20protease%20from%20Bacillus%20halodurans%20CAS6%20using%20marine%20shellfish%20wastes:%20A%20potential%20additive%20for%20detergent%20and%20antioxidant%20synthesis.%20Bioprocess%20Biosyst%20Eng.%202013;36(7):873%E2%80%9383.%2010.1007/s00449-012-0820-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Muthulakshmi C, Gomathi D, Kumar DG. Production, purification and characterization of protease by <em>Aspergillus flavus</em> under solid state fermentation. Jordan J Biol Sci. 2011;147(621):1–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?Muthulakshmi%20C,%20Gomathi%20D,%20Kumar%20DG.%20Production,%20purification%20and%20characterization%20of%20protease%20by%20Aspergillus%20flavus%20under%20solid%20state%20fermentation.%20Jordan%20J%20Biol%20Sci.%202011;147(621):1%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Waghmare SR, Gurav AA, Mali SA, Nadaf NH, Jadhav DB, Sonawane KD. Purification and characterization of novel organic solvent-tolerant 98 kda alkaline protease from isolated <em>Stenotrophomonas maltophilia</em> strain SK. Protein Expr Purif. 2015;107:1–6. 10.1016/j.pep.2014.11.002.
</cite> [<a href="https://doi.org/10.1016/j.pep.2014.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25462807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waghmare%20SR,%20Gurav%20AA,%20Mali%20SA,%20Nadaf%20NH,%20Jadhav%20DB,%20Sonawane%20KD.%20Purification%20and%20characterization%20of%20novel%20organic%20solvent-tolerant%2098%C2%A0kda%20alkaline%20protease%20from%20isolated%20Stenotrophomonas%20maltophilia%20strain%20SK.%20Protein%20Expr%20Purif.%202015;107:1%E2%80%936.%2010.1016/j.pep.2014.11.002." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Annamalai N, Rajeswari MV, Balasubramanian T. Extraction, purification and application of thermostable and halostable alkaline protease from <em>Bacillus alveayuensis</em> CAS5 using marine wastes. Food Bioprod Process. 2014;92(4):335–42. 10.1016/j.fbp.2013.08.009.</cite> [<a href="https://scholar.google.com/scholar_lookup?Annamalai%20N,%20Rajeswari%20MV,%20Balasubramanian%20T.%20Extraction,%20purification%20and%20application%20of%20thermostable%20and%20halostable%20alkaline%20protease%20from%20Bacillus%20alveayuensis%20CAS5%20using%20marine%20wastes.%20Food%20Bioprod%20Process.%202014;92(4):335%E2%80%9342.%2010.1016/j.fbp.2013.08.009." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Sathishkumar R, Ananthan G, Raghunathan C. Production and characterization of haloalkaline protease from ascidian-associated <em>Virgibacillus halodenitrificans</em> RSK CAS1 using marine wastes. Ann Microbiol. 2015;65(3):1481–93.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sathishkumar%20R,%20Ananthan%20G,%20Raghunathan%20C.%20Production%20and%20characterization%20of%20haloalkaline%20protease%20from%20ascidian-associated%20Virgibacillus%20halodenitrificans%20RSK%20CAS1%20using%20marine%20wastes.%20Ann%20Microbiol.%202015;65(3):1481%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Mothe T, Sultanpuram VR. Production, purification and characterization of a thermotolerant alkaline Serine protease from a novel species <em>Bacillus caseinilyticus</em>. 3 Biotech. 2016;6(1):53. 10.1007/s13205-016-0377-y.
</cite> [<a href="https://doi.org/10.1007/s13205-016-0377-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4752951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28330122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mothe%20T,%20Sultanpuram%20VR.%20Production,%20purification%20and%20characterization%20of%20a%20thermotolerant%20alkaline%20Serine%20protease%20from%20a%20novel%20species%20Bacillus%20caseinilyticus.%203%20Biotech.%202016;6(1):53.%2010.1007/s13205-016-0377-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Maruthiah T, Somanath B, Jasmin JV, Immanuel G, Palavesam A. Production, purification and characterization of halophilic organic solvent-tolerant protease from marine crustacean shell wastes and its efficacy on deproteinization. 3 Biotech. 2016;6(2):157. 10.1007/s13205-016-0474-y.
</cite> [<a href="https://doi.org/10.1007/s13205-016-0474-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4961627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28330229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maruthiah%20T,%20Somanath%20B,%20Jasmin%20JV,%20Immanuel%20G,%20Palavesam%20A.%20Production,%20purification%20and%20characterization%20of%20halophilic%20organic%20solvent-tolerant%20protease%20from%20marine%20crustacean%20shell%20wastes%20and%20its%20efficacy%20on%20deproteinization.%203%20Biotech.%202016;6(2):157.%2010.1007/s13205-016-0474-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Chandrasekaran M, Chandrasekar R, Chun SC, Sathiyabama M. Isolation, characterization And molecular three-dimensional structural predictions of metalloprotease from a phytopathogenic fungus, <em>Alternaria Solani (Ell. And Mart.) Sor</em>. J Biosci Bioeng. 2016;122(2):131–9. 10.1016/j.jbiosc.2015.12.021.
</cite> [<a href="https://doi.org/10.1016/j.jbiosc.2015.12.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26924427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chandrasekaran%20M,%20Chandrasekar%20R,%20Chun%20SC,%20Sathiyabama%20M.%20Isolation,%20characterization%20And%20molecular%20three-dimensional%20structural%20predictions%20of%20metalloprotease%20from%20a%20phytopathogenic%20fungus,%20Alternaria%20Solani%20(Ell.%20And%20Mart.)%20Sor.%20J%20Biosci%20Bioeng.%202016;122(2):131%E2%80%939.%2010.1016/j.jbiosc.2015.12.021." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Cañas B, López-Ferrer D, Ramos-Fernández A, Camafeita E, Calvo E. Mass spectrometry technologies for proteomics. Brief Funct Genomic Proteomic. 2006;4(4):295–320.
</cite> [<a href="https://doi.org/10.1093/bfgp/eli002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17202122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ca%C3%B1as%20B,%20L%C3%B3pez-Ferrer%20D,%20Ramos-Fern%C3%A1ndez%20A,%20Camafeita%20E,%20Calvo%20E.%20Mass%20spectrometry%20technologies%20for%20proteomics.%20Brief%20Funct%20Genomic%20Proteomic.%202006;4(4):295%E2%80%93320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Xia Z, Gale WL, Chang X, Langenau D, Patiño R, Maule AG, Densmore LD. Phylogenetic sequence analysis, Recombinant expression, and tissue distribution of a channel catfish Estrogen receptor Β. Gen Comp Endocrinol. 2000;118(1):139–49.
</cite> [<a href="https://doi.org/10.1006/gcen.1999.7447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10753576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia%20Z,%20Gale%20WL,%20Chang%20X,%20Langenau%20D,%20Pati%C3%B1o%20R,%20Maule%20AG,%20Densmore%20LD.%20Phylogenetic%20sequence%20analysis,%20Recombinant%20expression,%20and%20tissue%20distribution%20of%20a%20channel%20catfish%20Estrogen%20receptor%20%CE%92.%20Gen%20Comp%20Endocrinol.%202000;118(1):139%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Kumar SS, Sabu A. Fibrinolytic enzymes for thrombolytic therapy. Adv Exp Med Biol. 2019;1148:345–81. 10.1007/978-981-13-7709-9_15.
</cite> [<a href="https://doi.org/10.1007/978-981-13-7709-9_15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31482506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar%20SS,%20Sabu%20A.%20Fibrinolytic%20enzymes%20for%20thrombolytic%20therapy.%20Adv%20Exp%20Med%20Biol.%202019;1148:345%E2%80%9381.%2010.1007/978-981-13-7709-9_15." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Narasimhan MK, Chandrasekaran M, Rajesh M. Fibrinolytic enzyme production by newly isolated <em>Bacillus cereus</em> SRM-001 with enhanced in-vitro blood clot Lysis potential. J Gen Appl Microbiol. 2015;61(5):157–64. 10.2323/jgam.61.157.
</cite> [<a href="https://doi.org/10.2323/jgam.61.157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26582284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Narasimhan%20MK,%20Chandrasekaran%20M,%20Rajesh%20M.%20Fibrinolytic%20enzyme%20production%20by%20newly%20isolated%20Bacillus%20cereus%20SRM-001%20with%20enhanced%20in-vitro%20blood%20clot%20Lysis%20potential.%20J%20Gen%20Appl%20Microbiol.%202015;61(5):157%E2%80%9364.%2010.2323/jgam.61.157." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Sudhakar P, Kumar RS, Kothai R, Arul B. In-vitro thrombolytic and in-vivo cardioprotective activities of methanolic extract of <em>Indigofera cassioides</em> Rottl. Ex. DC. Indian J Pharm Sci. 2022;84(2):319–27.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sudhakar%20P,%20Kumar%20RS,%20Kothai%20R,%20Arul%20B.%20In-vitro%20thrombolytic%20and%20in-vivo%20cardioprotective%20activities%20of%20methanolic%20extract%20of%20Indigofera%20cassioides%20Rottl.%20Ex.%20DC.%20Indian%20J%20Pharm%20Sci.%202022;84(2):319%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Shen M, Wang Y, Hu F, Lv L, Chen K, Xing G. Thrombolytic agents: nanocarriers in targeted release. Molecules. 2021;26(22):6776. 10.3390/molecules26226776.
</cite> [<a href="https://doi.org/10.3390/molecules26226776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8619279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34833868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20M,%20Wang%20Y,%20Hu%20F,%20Lv%20L,%20Chen%20K,%20Xing%20G.%20Thrombolytic%20agents:%20nanocarriers%20in%20targeted%20release.%20Molecules.%202021;26(22):6776.%2010.3390/molecules26226776." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Whisstock JC, Lesk AM. Prediction of protein function from protein sequence and structure. Q Rev Biophys. 2003;36(3):307–40. 10.1017/s0033583503003901.
</cite> [<a href="https://doi.org/10.1017/s0033583503003901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15029827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whisstock%20JC,%20Lesk%20AM.%20Prediction%20of%20protein%20function%20from%20protein%20sequence%20and%20structure.%20Q%20Rev%20Biophys.%202003;36(3):307%E2%80%9340.%2010.1017/s0033583503003901." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Kotb E, Alabdalall AH, Alsayed MA, Alghamdi AI, Alkhaldi E, AbdulAzeez S, Borgio JF. Isolation, screening, and identification of alkaline protease-producing bacteria and application of the most potent enzyme from <em>Bacillus</em> Sp. Mar64. Fermentation. 2023;9(7):637. 10.3390/fermentation9070637.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kotb%20E,%20Alabdalall%20AH,%20Alsayed%20MA,%20Alghamdi%20AI,%20Alkhaldi%20E,%20AbdulAzeez%20S,%20Borgio%20JF.%20Isolation,%20screening,%20and%20identification%20of%20alkaline%20protease-producing%20bacteria%20and%20application%20of%20the%20most%20potent%20enzyme%20from%20Bacillus%20Sp.%20Mar64.%20Fermentation.%202023;9(7):637.%2010.3390/fermentation9070637." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Kalpana Devi M, Rasheedha Banu A, Gnanaprabhal GR, Pradeep BV, Palaniswamy M. Purification, characterization of alkaline protease enzyme from native isolate <em>Aspergillus Niger</em> and its compatibility with commercial detergents. Indian J Sci Technol. 2008;1(7):1–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kalpana%20Devi%20M,%20Rasheedha%20Banu%20A,%20Gnanaprabhal%20GR,%20Pradeep%20BV,%20Palaniswamy%20M.%20Purification,%20characterization%20of%20alkaline%20protease%20enzyme%20from%20native%20isolate%20Aspergillus%20Niger%20and%20its%20compatibility%20with%20commercial%20detergents.%20Indian%20J%20Sci%20Technol.%202008;1(7):1%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Mitra P, Chakrabartty PK. An extracellular protease with depilation activity from <em>Streptomyces nogalator</em>. J Sci Ind Res. 2005;64:978–83. 10.17485/ijst/2008/v1i7.8.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mitra%20P,%20Chakrabartty%20PK.%20An%20extracellular%20protease%20with%20depilation%20activity%20from%20Streptomyces%20nogalator.%20J%20Sci%20Ind%20Res.%202005;64:978%E2%80%9383.%2010.17485/ijst/2008/v1i7.8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Bajaj BK, Sharma P. An alkali-thermotolerant extracellular protease from a newly isolated Streptomyces Sp. DP2. N Biotechnol. 2011;28(6):725–32. 10.1016/j.nbt.2011.01.001.
</cite> [<a href="https://doi.org/10.1016/j.nbt.2011.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21232644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bajaj%20BK,%20Sharma%20P.%20An%20alkali-thermotolerant%20extracellular%20protease%20from%20a%20newly%20isolated%20Streptomyces%20Sp.%20DP2.%20N%20Biotechnol.%202011;28(6):725%E2%80%9332.%2010.1016/j.nbt.2011.01.001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Mahboubi A, Sadjady SK, Mirzaei Saleh Abadi M, Azadi S, Solaimanian R. Biological activity analysis of native and Recombinant streptokinase using clot Lysis and chromogenic substrate assay. Iran J Pharm Res. 2012;11(4):1087–93.
</cite> [<a href="/articles/PMC3813160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24250541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahboubi%20A,%20Sadjady%20SK,%20Mirzaei%20Saleh%20Abadi%20M,%20Azadi%20S,%20Solaimanian%20R.%20Biological%20activity%20analysis%20of%20native%20and%20Recombinant%20streptokinase%20using%20clot%20Lysis%20and%20chromogenic%20substrate%20assay.%20Iran%20J%20Pharm%20Res.%202012;11(4):1087%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Alakbaree M, Zakaria Z, Ahmed H, Zawawi N. In-vitro characterization of thrombolytic activity of staphylokinase produced by <em>Staphylococcus aureus</em> S1. In: Proceedings of the 1st Samarra International Conference for Pure and Applied Sciences, Samarra, Iraq, 23–24 March 2021; AIP Publishing: Melville, NY, USA. 2022;2394:1. 10.1063/5.0121711</cite>
</li>
<li id="CR53">
<span class="label">53.</span><cite>Gurewich V, Pannell R, Louie S, Kelley P, Suddith RL, Greenlee R. Effective and fibrin-specific clot Lysis by a Zymogen precursor form of urokinase (pro-urokinase). A study in vitro and in two animal species. J Clin Invest. 1984;73(6):1731–9. 10.1172/JCI111381.
</cite> [<a href="https://doi.org/10.1172/JCI111381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC437085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6725557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gurewich%20V,%20Pannell%20R,%20Louie%20S,%20Kelley%20P,%20Suddith%20RL,%20Greenlee%20R.%20Effective%20and%20fibrin-specific%20clot%20Lysis%20by%20a%20Zymogen%20precursor%20form%20of%20urokinase%20(pro-urokinase).%20A%20study%20in%20vitro%20and%20in%20two%20animal%20species.%20J%20Clin%20Invest.%201984;73(6):1731%E2%80%939.%2010.1172/JCI111381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Pinontoan R, Purnomo JS, Avissa EB. In-vitro and in-silico analyses of the thrombolytic potential of green Kiwifruit. Sci Rep. 2024;14(1):13799. 10.1038/s41598-024-64160-y.
</cite> [<a href="https://doi.org/10.1038/s41598-024-64160-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11178772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38877048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinontoan%20R,%20Purnomo%20JS,%20Avissa%20EB.%20In-vitro%20and%20in-silico%20analyses%20of%20the%20thrombolytic%20potential%20of%20green%20Kiwifruit.%20Sci%20Rep.%202024;14(1):13799.%2010.1038/s41598-024-64160-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Dickey A, Wang N, Cooper E, Tull L, Breedlove D, Mason H, Liu D, Wang KY. Transient expression of Lumbrokinase (PI239) in tobacco (Nicotiana tabacum) using a geminivirus-based single replicon system dissolves fibrin and blood clots. Evid Based Complement Alternat Med. 2017;2017(6093017). 10.1155/2017/6093017.</cite> [<a href="https://doi.org/10.1155/2017/6093017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5592424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28932252/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Markland FS, Swenson S, Fibrolase. Trials and tribulations. Toxins. 2010;2(4):793–808. 10.3390/toxins2040793.
</cite> [<a href="https://doi.org/10.3390/toxins2040793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3153196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22069611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Markland%20FS,%20Swenson%20S,%20Fibrolase.%20Trials%20and%20tribulations.%20Toxins.%202010;2(4):793%E2%80%93808.%2010.3390/toxins2040793." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12360004/bin/12896_2025_967_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (159.3KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated during the current study is provided within the manuscript or supplementary information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Biotechnology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12896-025-00967-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12896_2025_Article_967.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12360004/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12360004/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360004%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360004/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12360004/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12360004/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826056/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12360004/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826056/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12360004/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12360004/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xxvsvj28UbV7c9ylPDmNSryjTLnlSgUjFTK9Fbp9pWY0MZKQXGpBgsYWxgWeYur3">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
