
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A hybrid adversarial autoencoder-graph network model with dynamic fusion for robust scRNA-seq clustering - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BC618AF4380305BC610019F276DF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359927/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="A hybrid adversarial autoencoder-graph network model with dynamic fusion for robust scRNA-seq clustering">
<meta name="citation_author" content="Binhua Tang">
<meta name="citation_author_institution" content="Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, 213200 Nanjing, China">
<meta name="citation_author_institution" content="Shanghai Key Laboratory of Data Science, Fudan University, Shanghai, 200438 China">
<meta name="citation_author_institution" content="BGI Research, Changzhou, Jiangsu 213299 China">
<meta name="citation_author" content="Yingying Feng">
<meta name="citation_author_institution" content="Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, 213200 Nanjing, China">
<meta name="citation_author" content="Xinyu Gao">
<meta name="citation_author_institution" content="Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, 213200 Nanjing, China">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="749">
<meta name="citation_doi" content="10.1186/s12864-025-11941-y">
<meta name="citation_pmid" content="40826008">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359927/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359927/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359927/pdf/12864_2025_Article_11941.pdf">
<meta name="description" content="Single-cell RNA sequencing (scRNA-seq) allows the exploration of biological heterogeneity among different cell types within tissues at a single-cell resolution. Cell clustering serves as a foundation for scRNA-seq data analysis and provides new ...">
<meta name="og:title" content="A hybrid adversarial autoencoder-graph network model with dynamic fusion for robust scRNA-seq clustering">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Single-cell RNA sequencing (scRNA-seq) allows the exploration of biological heterogeneity among different cell types within tissues at a single-cell resolution. Cell clustering serves as a foundation for scRNA-seq data analysis and provides new ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359927/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359927">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11941-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11941.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359927%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359927/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359927/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359927/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 18;26:749. doi: <a href="https://doi.org/10.1186/s12864-025-11941-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11941-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A hybrid adversarial autoencoder-graph network model with dynamic fusion for robust scRNA-seq clustering</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Binhua Tang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Binhua Tang</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, 213200 Nanjing, China </div>
<div class="p">
<sup>2</sup>Shanghai Key Laboratory of Data Science, Fudan University, Shanghai, 200438 China </div>
<div class="p">
<sup>3</sup>BGI Research, Changzhou, Jiangsu 213299 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Binhua Tang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yingying Feng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yingying Feng</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, 213200 Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yingying Feng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xinyu Gao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xinyu Gao</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, 213200 Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinyu Gao</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, 213200 Nanjing, China </div>
<div id="Aff2">
<sup>2</sup>Shanghai Key Laboratory of Data Science, Fudan University, Shanghai, 200438 China </div>
<div id="Aff3">
<sup>3</sup>BGI Research, Changzhou, Jiangsu 213299 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 25; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359927  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826008/" class="usa-link">40826008</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Single-cell RNA sequencing (scRNA-seq) allows the exploration of biological heterogeneity among different cell types within tissues at a single-cell resolution. Cell clustering serves as a foundation for scRNA-seq data analysis and provides new insights into the heterogeneity of cells within complex tissues. However, the inherent features of scRNA-seq data, such as heterogeneity, sparsity, and high dimensionality, pose significant technical challenges for effective cell clustering.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">Here, we present a novel deep clustering method, scCAGN, based on an adversarial autoencoder (AAE) and a cross-attention graph convolutional network (GCN), to address the above challenges in scRNA-seq data analysis. Specifically, to enhance data reconstruction, scCAGN utilizes adversarial autoencoders to augment encoder capabilities. Graph feature representations obtained via a GCN were integrated using a dynamic information fusion mechanism, yielding enhanced feature representations. In addition, scCAGN combines three different loss functions to optimize clustering performance through a joint clustering approach. By leveraging a unique information fusion and joint mechanism, scCAGN extracts deep cell features without labeled information, thus improving cell classification efficiency. Our findings show that scCAGN surpasses the existing methods in clustering performance across eight typical scRNA-seq datasets, achieving a maximum Normalized Mutual Information (NMI) improvement of 11.94%, notably reaching an NMI of 0.9732 in the QS_diaphragm dataset. It showed an average NMI improvement of 13% across the eight benchmark datasets, surpassing the lowest-performing method. Further ablation and hyperparameter analyses validated the robustness of the proposed method. The code is available at: <a href="http://github.com/gladex/scCAGN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://github.com/gladex/scCAGN</a>.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">scCAGN integrates AAE and cross-attention GCN with dynamic fusion, achieving state-of-the-art scRNA-seq clustering (0.9732 NMI, 13% average gain) across eight datasets. Validated via ablation and hyperparameter tests, it advances label-free cell discovery and enables further multimodal integration to dissect cellular heterogeneity.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11941-y.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> ScRNA-seq, Self-supervised clustering, GCN, Adversarial AE, Cross-attention mechanism</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">The introduction of scRNA-seq technology has enabled researchers to measure gene expression levels more precisely at the single-cell level, thereby identifying cellular heterogeneity [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Identifying cell types plays a crucial role in revealing cellular heterogeneity and in understanding complex mechanisms [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Previous methods for cell-type identification typically involve dimensionality reduction techniques, such as t-distributed Stochastic Neighbor Embedding (t-SNE), followed by machine learning methods like K-means clustering for clustering analysis [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Several toolkits, such as Seurat [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], have been developed to facilitate rapid dimensionality reduction and cell-type identification.</p>
<p id="Par5">With the advancement of deep learning technologies, various deep clustering methods for scRNA-seq data have recently been proposed [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. scGNN introduces cell graphs into multiple autoencoders integrated with a GCN to learn the embeddings of the topological graph [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. DESC employs autoencoders to learn low-dimensional representations of data, followed by deep embedding clustering for soft assignments, ultimately optimizing cluster centers and data distributions [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. scDSC integrates structural information into deep clustering using a ZINB model-based autoencoder and graph neural networks [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. DeepCCI combines the learned cell embeddings from autoencoders and GCNs, and projects them into a shared embedding space, thereby completing cell clustering through a joint embedding approach [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. scDFN extracts cell information from scRNA-seq data using improved graph autoencoders, optimizing model clustering performance through information fusion and a self-supervised mechanism [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>].</p>
<p id="Par6">Despite significant progress made by deep clustering methods in cell-type identification, certain limitations still exist [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. For instance, many autoencoder-based methods focus solely on hidden layer features and do not fully utilize the information from the decoder layer, and single hidden layer features are inadequate for fully capturing the deep relationships between cells [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. In addition, most existing methods employ simple linear superposition when integrating embedding representations from different modules, overlooking the complexity and differences in feature structures, which limit the enhancement of model performance [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par7">In summary, this work presents a novel self-supervised clustering approach that integrates an adversarial autoencoder and a graph convolutional network employing a cross-attention mechanism. First, the AAE employs a discriminator to guide the encoder in learning a more structured latent space that aligns with the distribution of real data, thereby enhancing the encoder to capture and learn embedding representations. Subsequently, the model employs a GCN to obtain local and global insights from the graph data. A multi-head attention mechanism is used to integrate these insights, enabling the model to prioritize salient features. Finally, dual constraints guide the model in learning a distribution with higher confidence, thereby optimizing cell-type annotation. The method performance was assessed by comparison with eight state-of-the-art methods using eight typical scRNA-seq datasets. The experimental results reveal that the proposed scCAGN achieves a higher level of clustering performance than competing methods. In addition, ablation studies and hyperparameter analyses further validate the robustness of the proposed method.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Preprocessing for the raw scRNA-seq datasets</h3>
<p id="Par8">During the preprocessing of the raw scRNA-seq data, we first filtered and performed quality control, keeping only genes that were expressed as non-zero in more than 1% of the cells and cells that expressed non-zero values in more than 1% of the genes. Subsequently, each cell type was normalized and standardized. Finally, the genes were ranked based on their standard deviation, and the <em>M</em> genes with the highest variance were selected for subsequent analysis to reduce the influence of noise. Following data preprocessing, Euclidean distances between cells were computed, and cell selection was performed using the K-nearest neighbor (KNN) algorithm. A graph representing cell similarities was constructed and served as the input for the graph network module.</p></section><section id="Sec4"><h3 class="pmc_sec_title">The main framework of the proposed scCAGN method</h3>
<p id="Par9">In the absence of label information for guidance, to better leverage the intrinsic data and constraint information to analyze scRNA-seq data, we propose a novel deep clustering algorithm called scCAGN, based on AAE and a cross-attention mechanism within the GCN.</p>
<p id="Par10">As shown in Fig. <a href="#Fig1" class="usa-link">1</a>, the method comprises three main components: an adversarial autoencoder, cross-attention-based graph information fusion, and deep dual-constraint clustering. First, we used the preprocessed data <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/d2c6fb245681/d33e286.gif" loading="lazy" id="d33e286" alt="Inline graphic"></span> as part of adversarial autoencoder pretraining to learn the latent feature representation of the data by minimizing the reconstruction loss and adversarial loss. Then, the feature representations learned by the adversarial autoencoder and graph convolutional network were fused into a richer feature representation using the cross-attention mechanism. Finally, we designed a joint supervision mechanism that guides the model to adjust the clustering representation and optimize the clustering results by learning a soft label distribution with higher confidence.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/286e3d5d6e93/12864_2025_11941_Fig1_HTML.jpg" loading="lazy" id="d33e313" height="557" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Framework diagram of the proposed scCAGN. (<strong>a</strong>) Data preprocessing module, (<strong>b</strong>) Adversarial Autoencoder module, (<strong>c</strong>) cross-attention mechanism-based graph information fusion module, and (<strong>d</strong>) deep dual-constraint clustering module</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">The structure of adversarial Autoencoder</h3>
<p id="Par12">The AAE module aims to optimize the generative and structural properties of the embedded representation in the latent space by introducing an adversarial training mechanism. The AAE comprises three components: encoder, decoder, and discriminator. The encoder and decoder minimize data reconstruction loss, whereas the encoder and discriminator form a generative adversarial network. The discriminator guides the encoder to learn a more structured latent space that aligns with the distribution of the real data.</p>
<p id="Par13">Assuming that the AAE has <em>l</em> layers, the feature representations learned at the <em>l</em>-th layer by the encoder and decoder are denoted as:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/f30053ba1963/d33e327.gif" loading="lazy" id="d33e327" alt="graphic file with name d33e327.gif"></td>
<td class="label">1</td>
</tr></table>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/6f4c4b11f3ab/d33e333.gif" loading="lazy" id="d33e333" alt="graphic file with name d33e333.gif"></td>
<td class="label">2</td>
</tr></table>
<p id="Par14">where <em>f</em>(·) and <em>g</em>(·) represent the activation functions used in the fully-connected layer; <em>W</em><sub><em>e</em></sub><sup>(<em>l</em>)</sup>, <em>b</em><sub><em>e</em></sub><sup>(<em>l</em>)</sup>, <em>W</em><sub><em>d</em></sub><sup>(<em>l</em>)</sup> and <em>b</em><sub><em>d</em></sub><sup>(<em>l</em>)</sup> denote the weights and biases of the <em>l</em>-th layer in the encoder and decoder, respectively; <em>Enc</em><sup>(0)</sup> denotes the preprocessed data <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/d6d925ef4bd3/d33e400.gif" loading="lazy" id="d33e400" alt="Inline graphic"></span>, and <em>Dec</em><sup>(0)</sup> is the feature representation after information fusion <em>Z</em><sup><em>ca.</em></sup>. The output of the decoder is the reconstructed data <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/d2c6fb245681/d33e417.gif" loading="lazy" id="d33e417" alt="Inline graphic"></span>, and the loss function is denoted as:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/06b14ec2a765/d33e423.gif" loading="lazy" id="d33e423" alt="graphic file with name d33e423.gif"></td>
<td class="label">3</td>
</tr></table>
<p id="Par15">where <em>N</em> refers to the sample count.</p>
<p id="Par16">We sampled a random noise vector <em>fake_z</em> from a normal Gaussian distribution <em>N</em>(0,1) as a negative sample for the discriminator. The latent representation, <em>z</em>, generated by the encoder was used as a positive sample for the discriminator. Through adversarial training, the encoder and discriminator engage in a continuous game, where the encoder aims to generate positive samples that better approximate the normal Gaussian distribution, whereas the discriminator aims to more accurately identify the negative samples. The objective loss function of the discriminator is expressed as:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/e24f56231500/d33e445.gif" loading="lazy" id="d33e445" alt="graphic file with name d33e445.gif"></td>
<td class="label">4</td>
</tr></table>
<p id="Par17">And the objective loss function of the encoder,</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/a86f7172811c/d33e453.gif" loading="lazy" id="d33e453" alt="graphic file with name d33e453.gif"></td>
<td class="label">5</td>
</tr></table>
<p id="Par18">where <em>D</em>(·) denotes the discriminator’s decision result, and E[·] for the mean squared error (MSE) loss.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Graph information fusion based on a cross-attention mechanism</h3>
<p id="Par19">To delineate the spatial relationships inherent in the scRNA-seq data, a K-nearest neighbor (KNN) algorithm was utilized to generate an adjacency matrix. The K-nearest neighbors of each cell were determined by computing the Euclidean distances. Then, we built a cell graph based on the similarity between these cells, where nodes represent individual cells and edges for the relationship weights between cells. The constructed cell graph and gene expression matrix were fed into a GCN module to learn low-dimensional embeddings and topological structures for each node from the graph structure. The node embedding representation learned by the GCN at layer <em>l</em> is:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/a1da66c1d975/d33e471.gif" loading="lazy" id="d33e471" alt="graphic file with name d33e471.gif"></td>
<td class="label">6</td>
</tr></table>
<p id="Par20">where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/c574e98d2afc/d33e479.gif" loading="lazy" id="d33e479" alt="Inline graphic"></span> denotes the degree matrix of the adjacency matrix, <em>I</em> is the identity diagonal matrix of adjacency <em>A</em> for each node, and <em>Z</em><sup>(<em>l−1</em>)</sup> propagates information between nodes and updates the node representations through <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/8ffadc2d6d15/d33e499.gif" loading="lazy" id="d33e499" alt="Inline graphic"></span>.</p>
<p id="Par21">Considering that AAE and GCN each have complementary advantages in the representations they learn, we have fused these two types of representations to obtain a more comprehensive embedding representation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/f95b42e3f708/d33e507.gif" loading="lazy" id="d33e507" alt="Inline graphic"></span>,</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/65f725e0202f/d33e513.gif" loading="lazy" id="d33e513" alt="graphic file with name d33e513.gif"></td>
<td class="label">7</td>
</tr></table>
<p id="Par22">where <em>φ</em> denotes a balancing parameter to adjust the outputs of the encoder and graph convolutional network (<em>φ =</em> 0.5); <em>α</em> and <em>β</em> are the regulating parameters for the component contribution.</p>
<p id="Par23">We introduced an information fusion approach via a multi-head attention mechanism to address the limitation of GCNs in capturing fixed-weight spatial relationships and to integrate different modules, concurrently focusing on and learning the most relevant input features. Specifically, the latent representation <em>z</em> of the encoder was taken as the query <em>Q</em> and the output <em>Z</em><sup>(<em>l</em>)</sup> of the GCN as the key <em>K</em> and value <em>V</em>. We calculated the attention scores between <em>Q</em> and <em>K</em> using scaled dot-product attention, and then performed a weighted sum of the attention scores with <em>V</em> to obtain the output of a single attention head, denoted as <em>head</em><sub><em>m</em></sub>. Finally, we concatenated the outputs of all attention heads to obtain a more enriched feature representation, <em>Z</em><sup><em>ca.</em></sup>.</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/d0f162094a0f/d33e578.gif" loading="lazy" id="d33e578" alt="graphic file with name d33e578.gif"></td>
<td class="label">8</td>
</tr></table>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/700d1ebb7b6c/d33e584.gif" loading="lazy" id="d33e584" alt="graphic file with name d33e584.gif"></td>
<td class="label">9</td>
</tr></table>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/e2228cc18817/d33e590.gif" loading="lazy" id="d33e590" alt="graphic file with name d33e590.gif"></td>
<td class="label">10</td>
</tr></table>
<p id="Par24">where <em>k</em> is the dimension of the key data and <em>m</em> the number of attention heads in multi-head attention. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/d987f64a98c6/d33e604.gif" loading="lazy" id="d33e604" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/ae50859a6b43/d33e610.gif" loading="lazy" id="d33e610" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/41eddfbc04ca/d33e616.gif" loading="lazy" id="d33e616" alt="Inline graphic"></span>are the weights for <em>Q</em>, <em>K</em>, and <em>V</em> in the <em>m</em>-th head, respectively, while <em>W</em><sup><em>ca.</em></sup> is the weight used when concatenating the attention outputs from all heads. The fused feature representation <em>Z</em><sup><em>ca.</em></sup> serves as the input to the decoder.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Deep double-constrained clustering</h3>
<p id="Par25">To enhance the clustering representation and optimize the clustering performance without labeled guidance, we employed a double-constrained clustering algorithm that uses the Student’s t-distribution as the kernel function. This algorithm evaluates the similarity between data points in the embedding space by estimating the soft membership of each data point to the cluster centers in the latent representation <em>z</em>, defined as:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/60fdea51edd1/d33e655.gif" loading="lazy" id="d33e655" alt="graphic file with name d33e655.gif"></td>
<td class="label">11</td>
</tr></table>
<p id="Par26">where <em>q</em><sub><em>ij</em></sub> represents the probability of assigning cell <em>i</em> to cluster <em>j</em>, <em>z</em><sub><em>i</em></sub> is the data point of the <em>i</em>-th cell in the latent representation <em>z</em>, <em>µ</em><sub><em>j</em></sub> is the centroid of cell cluster <em>j</em>, <em>k</em> is the number of cell clusters, and <em>α</em> is the degree of freedom (the default is 1). Additionally, we defined the target distribution <em>P</em> through the clustering distribution <em>Q</em>, aiming to learn from <em>Q</em> and approach the high-confidence distribution <em>P</em> to optimize the clustering. The distribution is defined as:</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/6a410d2abf1f/d33e716.gif" loading="lazy" id="d33e716" alt="graphic file with name d33e716.gif"></td>
<td class="label">12</td>
</tr></table>
<p id="Par27">where <em>τ</em> is a temperature parameter. A reduction in <em>τ</em> results in a sharper distribution <em>P</em>, signifying an increased confidence in the cluster assignment for a given sample. The GCN module produces soft clustering labels, represented as distribution <em>T</em>, which are optimized via double constraints against target distribution <em>P</em>. The objective loss function is denoted as:</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/f4cd5f211972/d33e740.gif" loading="lazy" id="d33e740" alt="graphic file with name d33e740.gif"></td>
<td class="label">13</td>
</tr></table>
<p id="Par28">The specific training process of the proposed model is described in Algorithm 1, and its combined objective loss function is as follows:</p>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/d3bb6693df29/d33e748.gif" loading="lazy" id="d33e748" alt="graphic file with name d33e748.gif"></td>
<td class="label">14</td>
</tr></table>
<p id="Par29">where <em>ε</em>, <em>µ</em>, and <em>θ</em> are the parameters that balance the contributions of the loss function.</p>
<figure class="fig xbox font-sm" id="Figa"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Figa_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/72d02abc9951/12864_2025_11941_Figa_HTML.jpg" loading="lazy" id="d33e770" height="660" width="669" alt="graphic file with name 12864_2025_11941_Figa_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="figure/Figa/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Algorithm 1: The proposed scCAGN model</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Evaluation metrics for the experiments</h3>
<p id="Par30">To evaluate the clustering performance of scCAGN on scRNA-seq datasets, we employed the Normalized Mutual Information (NMI) and Adjusted Rand Index (ARI) as the primary metrics to assess model performance. NMI was used to measure the similarity between the predicted and true values, and ARI to assess the overlap between the predicted and actual clusters. Both measure the consistency between the predicted and true labels, with higher values indicating better clustering performance. Additionally, we also use the Completeness and Adjusted Mutual Information (AMI) as supplementary metrics to provide a more comprehensive assessment of model performance. The detailed formulae and definitions are provided in the supplement.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Experimental settings and datasets</h3>
<p id="Par31">In this study, scCAGN was developed using Python 3.9.20 and PyTorch 2.3.1 + cu121. The default number of highly variable genes <em>M</em> was set to 2000, with the encoder’s hidden layer dimensions of 100, 200, and 200. The hidden layers in the decoder are the reverse of the encoder, and the dimensions of the latent representation are eight.</p>
<p id="Par32">The default value of <em>K</em> for constructing the cell graph using KNN was 10, and we employed the Adamax optimizer with a learning rate of 0.001 for training and updating the encoder. In the information fusion mechanism, we use a multi-head attention with 4 attention heads; in the double-constrained clustering module, the default value of the temperature parameter <em>τ</em> is set to 0.5; in the objective loss function, the hyperparameters <em>ε</em>, <em>µ</em>, and <em>θ</em> were empirically set to 0.0001, 0.5, and 0.5, respectively, based on established practices from analogous frameworks in the literature and preliminary experimental validation.</p>
<p id="Par33">scCAGN employs the ground-truth cell type count as the predefined cluster number to ensure fair benchmarking against alternative methods while mitigating performance biases arising from cluster number estimation inaccuracies. Meanwhile, to ensure diversity of the experimental data, we collected 10 single-cell RNA sequencing datasets from various tissues and sequencing platforms, as detailed in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of the ten typical scRNA-seq datasets chosen from diverse tissues and sequencing platforms</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/4ffed7975818/12864_2025_11941_Tab1_HTML.jpg" loading="lazy" id="d33e814" height="482" width="760" alt="graphic file with name 12864_2025_11941_Tab1_HTML.jpg"></p>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Clustering performance comparison with representative methods</h3>
<p id="Par35">We compared scCAGN with eight advanced competing methods, including scTAG, scDFN, and DeepCCI, on eight scRNA-seq datasets to evaluate their clustering performance [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. To ensure fairness of the experiments, all methods used the default parameters or settings recommended by the authors, and the input data for each clustering method underwent the same preprocessing.</p>
<p id="Par36">Figure <a href="#Fig2" class="usa-link">2</a> shows the performance of nine methods on the eight scRNA-seq datasets. The experimental results indicated that scCAGN achieved the highest average NMI (0.8918) and ARI (0.8942) across the benchmark datasets, with an average NMI improvement of 3.5–13.5% and an average ARI improvement of 7.5–31.73%.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/243dc7400a02/12864_2025_11941_Fig2_HTML.jpg" loading="lazy" id="d33e847" height="551" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance comparison of nine clustering methods on eight benchmark datasets, (<strong>a</strong>) The NMI statistics of nine methods across the eight datasets; (<strong>b</strong>) Boxplot of the mean performance of the nine methods on NMI and ARI; (<strong>c</strong>) The ARI statistics of nine methods across the eight datasets</p></figcaption></figure><p id="Par37">Specifically, scCAGN performed the best on seven datasets; however, on the Muraro dataset, the NMI (0.8558) was lower than that of scLEGA (0.8630) and scSimGCL (0.8632) because of the presence of significant data noise, and our model was not fine-tuned separately for each dataset. Across the eight datasets, compared with the baseline method DeepCCI, scCAGN showed the greatest improvement in the “Qs_muscle” dataset, with NMI and ARI increasing by 11.94% and 30.33%, respectively; it achieved the highest NMI value of 0.9732 on the “QS_Diaphragm” dataset. In general, our method demonstrated superior performance compared to other methods.</p>
<p id="Par38">The distributions of the NMI and ARI values for scCAGN and the eight alternative methods across the eight datasets are shown in Fig. <a href="#Fig2" class="usa-link">2</a>b. The results demonstrate our model exhibits a stable performance across various datasets, further validating its superior robustness.</p>
<p id="Par39">Furthermore, to intuitively assess the ability of scCAGN to learn low-dimensional representations from high-dimensional data, we projected the learned embedded representations into a two-dimensional space. Figure <a href="#Fig3" class="usa-link">3</a> illustrates the visualization of cell clusters identified by the seven clustering methods across the two datasets, where each point represents a cell and each color denotes a different cell type. The results indicate the clusters obtained by our model exhibit greater compactness within individual clusters and greater dispersion between clusters, further demonstrating the superiority of clustering performance.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/a83882d94072/12864_2025_11941_Fig3_HTML.jpg" loading="lazy" id="d33e868" height="614" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of dimensionality-reduction visualizations of the seven clustering methods across two datasets (Qs_trachea and Qs_muscle)</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Effectiveness validation of the GCN module</h3>
<p id="Par41">To evaluate the effectiveness of the GCN module, we compared it with similar variants to assess its performance in clustering tasks. Specifically, within the same framework, we evaluated the impact of the GCN, Graph Attention Networks (GAT), and Graph Sample and Aggregated (GraphSAGE) embedding models on the clustering results.</p>
<p id="Par42">The experimental results in Fig. <a href="#Fig4" class="usa-link">4</a>(b) indicate that the performance of the GCN achieves optimal values across five different datasets, demonstrating its effectiveness in capturing local structures and adjacency information of nodes. In contrast, while GAT is capable of learning the relationships between nodes, the introduction of the cross-attention mechanism can lead to overfitting, thereby limiting its ability to capture and utilize local structural information, resulting in less meaningful feature representations. Additionally, the aggregation method employed by GraphSAGE relies on the transformation and aggregation of neighboring node features; thus, its application to datasets with significant differences in neighbor features or high-dimensional data may fail to effectively preserve the higher-order information of the graph structure. In summary, the GCN proved to be more effective in learning structural information from graph data, exhibiting superior clustering performance compared to other variants, which further underscores the sensitivity and adaptability of the GCN architecture to clustering outcomes.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/ece43faf545b/12864_2025_11941_Fig4_HTML.jpg" loading="lazy" id="d33e897" height="503" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>a</strong>) Impact of varying numbers of attention heads <em>h</em> (0, 1, 2, 4 and 8) on the clustering performance of scCAGN across three different datasets; (<strong>b</strong>) The effect of replacing GCN in the model with other similar variants (GAT, GraphSAGE) on the clustering performance of scCAGN across five different datasets</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Ablation analysis of the proposed scCAGN</h3>
<p id="Par44">An ablation study across five datasets was performed to evaluate the efficacy of the model architecture and individual contributions of each module to the clustering performance of the proposed scCAGN. The experiments included: (1) removing the discriminator (without AAE); (2) removing the cross-attention mechanism (without CA); (3) scCAGN: (Ours, AAE + GCN + CA); (4) implementing information fusion using concatenation operation (CA replaced by Concat); and (5) implementing information fusion using multiplication operations (CA replaced by Multiply).</p>
<p id="Par45">The results, as depicted in Fig. <a href="#Fig5" class="usa-link">5</a>(a), demonstrate that our proposed method achieves the best performance in terms of NMI and ARI across the five datasets and various ablation experiment settings. Specifically, on the “Qs_Trachea” dataset, the NMI metric improved by 10.63% compared with the worst-performing ablation setup. As shown in Fig. <a href="#Fig5" class="usa-link">5</a>(b), our method outperforms the other approaches in terms of median values and result variability, exhibiting greater robustness. This further validates the stability and superiority of the proposed method in terms of the performance.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/382343bd6bc8/12864_2025_11941_Fig5_HTML.jpg" loading="lazy" id="d33e924" height="452" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ablation study of the proposed scCAGN architecture. (<strong>a</strong>) Comparison of NMI and ARI across five datasets under various ablation settings. (<strong>b</strong>) Box plots of NMI and ARI distributions under different ablation settings</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Impacts of different loss functions on experimental performance</h3>
<p id="Par46">To investigate the effectiveness of joint loss clustering and the contribution of the three loss functions <em>L</em><sub><em>dc</em></sub>, <em>L</em><sub><em>res</em></sub>, and <em>L</em><sub><em>g</em></sub> to clustering performance, we performed further ablation experiments to arbitrarily permute and combine the three loss functions. The performance was evaluated on two real scRNA-seq datasets, Muraro and Qs_diaphragm, and the results are shown in Table <a href="#Tab2" class="usa-link">2</a>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Impact of different objective loss functions on the model performance</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Tab2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/675a8978ae3f/12864_2025_11941_Tab2_HTML.jpg" loading="lazy" id="d33e958" height="486" width="765" alt="graphic file with name 12864_2025_11941_Tab2_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par48">The analysis results indicate that in joint loss clustering, when the double-constraint clustering loss <em>L</em><sub><em>dc</em></sub> is not considered, the average values of the NMI and ARI are 0.8949 and 0.8957, respectively. Regardless of how other loss functions are combined for clustering, there is no significant improvement in the clustering performance. When the reconstruction loss <em>L</em><sub><em>res</em></sub> or adversarial loss <em>L</em><sub><em>g</em></sub> is not considered, the average NMI remains the same as when <em>L</em><sub><em>dc</em></sub> is not considered, whereas the average ARI increases by 1%. However, when the three losses were combined for clustering, the average NMI and ARI significantly improved to 0.9145 and 0.9371, respectively.</p>
<p id="Par49">In summary, we conclude that when clustering a model using any two loss functions in combination or a single loss function, the clustering performance is not remarkable. However, a combination of the three loss functions significantly improves the clustering performance. Therefore, the combined use of the three loss functions has a positive impact on the clustering performance of the proposed scCAGN.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Hyperparameter analysis</h3>
<p id="Par50">In this study, we systematically investigated the selection rationale and conducted sensitivity analyses of critical hyperparameters to strengthen the methodological interpretability and robustness of the proposed framework. The proposed scCAGN utilizes a multi-head attention mechanism for information integration. A comparative analysis across the three datasets was performed to evaluate the impact of attention head counts on clustering performance. The determination of feasible attention head counts is based on the dimensionality of the latent representation, <em>z</em>. Figure <a href="#Fig4" class="usa-link">4</a>(a) presents the impact of using different numbers of attention heads (0, 1, 2, 4, and 8) on the performance of scCAGN. The results show that when <em>heads</em> = 4, scCAGN achieved optimal NMI scores across the three datasets.</p>
<p id="Par51">The analysis revealed that because of the high dimensionality and sparsity characteristic of scRNA-seq data, when the attention-head count is set to 0, 1, or 2, the model cannot learn the underlying relationships from multiple perspectives and fails to effectively extract feature information from high-dimensional data. When the attention-head count was increased to eight, the model became more complex. As each attention head focuses on different parts of the data features in parallel, the learned information becomes too dispersed, and it is difficult to concentrate on key features, leading to redundancy, which degrades the model performance.</p>
<p id="Par52">The temperature parameter <em>τ</em> in our model regulates the sharpness of the soft clustering distribution <em>Q</em>, critically influencing its alignment with the target distribution <em>P</em> and ultimately determining clustering performance. A systematic grid search evaluation was performed across five benchmark datasets from diverse sequencing platforms to assess <em>τ</em> values within {0.1, 0.3, 0.5, 1.0, 2.0}. As illustrated in Figs. <a href="#Fig6" class="usa-link">6</a>(a) and 6(b), <em>τ =</em> 0.5 yielded optimal or near-optimal NMI and ARI scores across most datasets. Notably, performance exhibited minimal sensitivity to τ variations within this range, with average fluctuations limited to 4.98% (NMI) and 4.92% (ARI), confirming robust parameter stability. We therefore adopted τ = 0.5 as the default configuration to balance clustering accuracy and training consistency. Complete experimental results are documented in Supplementary Sect. 3.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359927_12864_2025_11941_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c575/12359927/52b1f494016e/12864_2025_11941_Fig6_HTML.jpg" loading="lazy" id="d33e1044" height="534" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>a</strong>) and (<strong>b</strong>) illustrate the variation trends of NMI and ARI for scCAGN across five benchmark datasets under distinct temperature parameter configurations. (<strong>c</strong>) and (<strong>d</strong>) comparatively present the NMI and ARI outcomes of all evaluated models on three datasets, where the horizontal axis label “x1-x2” denotes models trained on sub-batch x1 and evaluated on sub-batch x2. This cross-batch validation protocol rigorously assesses generalization performance under batch distribution shifts</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Impact of batch effect on clustering performance</h3>
<p id="Par53">To evaluate the robustness of the proposed method under batch variation scenarios, three benchmark datasets with explicit batch annotations—Dendritic, Retina(19), and Retina(5)—were selected. Each dataset was partitioned into two sub-batches (x1 and x2) to establish a cross-batch validation protocol. Specifically, clustering performance was assessed under both “x1→x2” (trained on x1, tested on x2) and “x2→x1” configurations to simulate typical batch distribution discrepancies.</p>
<p id="Par54">As illustrated in Figs. <a href="#Fig6" class="usa-link">6</a>(c) and 6(d), the scCAGN framework achieved optimal or near-optimal performance in NMI and ARI across all datasets and batch combinations, demonstrating superior cross-batch generalization capacity. Notably, scCAGN exhibited exceptional robustness in the Dendritic and Retina(19) datasets, outperforming all baseline methods. For example, in the Dendritic dataset, scCAGN surpassed the second-best model by maximum margins of 4.5% (NMI) and 8.1% (ARI) under cross-batch testing conditions. These results indicate that scCAGN effectively extracts discriminative expression features invariant to batch effects, maintaining stable clustering performance without requiring explicit batch correction modules. The framework’s inherent ability to mitigate batch-related perturbations highlights its potential for practical deployment in multi-protocol scRNA-seq studies.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Comparative analysis of computational efficiency</h3>
<p id="Par55">Comparative analysis of computational efficiency</p>
<p id="Par56">We systematically evaluated the computational efficiency of scCAGN by comparing its runtime and memory usage with multiple advanced methods across five datasets from different sequencing platforms. The results indicate that scCAGN exhibits moderate computational overhead compared to lightweight models, primarily due to the integration of multi-head attention mechanisms and graph convolution modules. While these components enhance feature representation and spatial dependency modeling, they increase computational complexity, particularly during graph construction and cross-view information fusion. Conversely, scCAGN demonstrates superior memory efficiency, maintaining significantly lower memory footprints across all datasets compared to existing methods. This trade-off between runtime and memory utilization reflects our prioritization of clustering accuracy and spatial modeling capability. Detailed experimental results are provided in Supplementary Sect. 4.</p>
<p id="Par57">Overall, scCAGN substantially outperforms rapid algorithms like Louvain and Leiden in clustering performance while achieving better memory control than scMAE, scLEGA, and scSimGCL under comparable accuracy levels. Future work will focus on optimizing model architecture and inference workflows to reduce runtime while preserving spatial modeling capabilities, thereby enhancing the framework’s applicability in large-scale data analyses.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Discussion and conclusions</h2>
<p id="Par58">This study proposed a new self-supervised clustering approach, named scCAGN, which leverages AAE and GCN, interconnected through cross-attention, for cell-type identification in scRNA-seq data.</p>
<p id="Par59">Initially, high-dimensional scRNA-seq data underwent preprocessing, encompassing the normalization and selection of highly variable genes. Subsequently, a discriminator, which is integrated with an encoder to constitute a generative adversarial network, is introduced. Latent and graph feature representations were extracted from the data using the AAE and GCN, respectively. The feature representations derived from the AAE and GCN were further integrated using an information fusion technique. The similarity between feature representations is computed in parallel using a multi-head attention mechanism, allowing for the dynamic weighting of the graph feature representation. This enables the model to automatically prioritize salient features during information fusion, thereby generating more meaningful representations. Finally, we employed a joint loss function to constrain and adjust the soft label distribution output by the encoder and optimize the clustering results.</p>
<p id="Par60">To evaluate the performance of the scCAGN method comprehensively, we compared it with other typical methods from various perspectives. The experiments demonstrated scCAGN exhibited outstanding clustering performance across eight datasets, outperforming eight state-of-the-art clustering methods designed for the scRNA-seq data. The ablation and hyperparameter analyses further confirmed the robustness of our approach.</p>
<p id="Par61">This work advances unsupervised analysis of scRNA-seq data by providing a scalable, end-to-end solution that bypasses reliance on labeled data. Future studies could explore multimodal integration with spatial or epigenetic data, while theoretical investigations into the interplay between adversarial training and graph attention mechanisms may further refine model interpretability. The analytical frameworks with enhanced interpretability will further improve the transparency and reliability of the model.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12359927/bin/12864_2025_11941_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (639.7KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors sincerely thank the editors and anonymous reviewers for their valuable time and helpful suggestions.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>BHT, YYF and XYG drafted the manuscript and performed the coding and analysis. BHT and YYF revised the manuscript. BHT led the project. All authors proofread and approved the final version.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The work was supported in part by the research fund (No. B240203012) from the Key Laboratory of Maritime Intelligent Cyberspace Technology (Ministry of Education of China), Hohai University, China.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The benchmark data can be accessed from Gene Expression Omnibus with accession numbers of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65525" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE65525</a> (Mouse_ES_cell), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE94333" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE94333</a> (Adam), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74672" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74672</a> (Romanov), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE85241" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE85241</a> (Muraro), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE23341" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE23341</a> (Qs_Trachea+Qs_Diaphragm), Yan+Qx_muscle (<a href="https://github.com/JiangBioLab/DeepCCI/tree/main/Cluster_model/dataset" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/JiangBioLab/DeepCCI/tree/main/Cluster_model/dataset</a>), and Qs_Muscle (<a href="https://github.com/Philyzh8/scTAG/tree/master/data/Quake_Smart-seq2_Limb_Muscle" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Philyzh8/scTAG/tree/master/data/Quake_Smart-seq2_Limb_Muscle</a>).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par62">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par63">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par64">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Papalexi E, Satija R. Single-cell RNA sequencing to explore immune cell heterogeneity. Nat Rev Immunol. 2018;18(1):35–45.
</cite> [<a href="https://doi.org/10.1038/nri.2017.76" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28787399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Papalexi%20E,%20Satija%20R.%20Single-cell%20RNA%20sequencing%20to%20explore%20immune%20cell%20heterogeneity.%20Nat%20Rev%20Immunol.%202018;18(1):35%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Van de Sande B, Lee JS, Mutasa-Gottgens E, Naughton B, Bacon W, Manning J, Wang Y, Pollard J, Mendez M, Hill J, et al. Applications of single-cell RNA sequencing in drug discovery and development. Nat Rev Drug Discovery. 2023;22(6):496–520.
</cite> [<a href="https://doi.org/10.1038/s41573-023-00688-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10141847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37117846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20de%20Sande%20B,%20Lee%20JS,%20Mutasa-Gottgens%20E,%20Naughton%20B,%20Bacon%20W,%20Manning%20J,%20Wang%20Y,%20Pollard%20J,%20Mendez%20M,%20Hill%20J,%20et%20al.%20Applications%20of%20single-cell%20RNA%20sequencing%20in%20drug%20discovery%20and%20development.%20Nat%20Rev%20Drug%20Discovery.%202023;22(6):496%E2%80%93520." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Qiu P. Embracing the dropouts in single-cell RNA-seq analysis. Nat Commun. 2020;11(1):1169.
</cite> [<a href="https://doi.org/10.1038/s41467-020-14976-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32127540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu%20P.%20Embracing%20the%20dropouts%20in%20single-cell%20RNA-seq%20analysis.%20Nat%20Commun.%202020;11(1):1169." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Stassen SV, Siu DMD, Lee KCM, Ho JWK, So HKH, Tsia KK. PARC: ultrafast and accurate clustering of phenotypic data of millions of single cells. Bioinformatics. 2020;36(9):2778–86.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7203756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31971583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stassen%20SV,%20Siu%20DMD,%20Lee%20KCM,%20Ho%20JWK,%20So%20HKH,%20Tsia%20KK.%20PARC:%20ultrafast%20and%20accurate%20clustering%20of%20phenotypic%20data%20of%20millions%20of%20single%20cells.%20Bioinformatics.%202020;36(9):2778%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Petegrosso R, Li Z, Kuang R. Machine learning and statistical methods for clustering single-cell RNA-sequencing data. Brief Bioinform. 2019;21(4):1209–23.</cite> [<a href="https://doi.org/10.1093/bib/bbz063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31243426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petegrosso%20R,%20Li%20Z,%20Kuang%20R.%20Machine%20learning%20and%20statistical%20methods%20for%20clustering%20single-cell%20RNA-sequencing%20data.%20Brief%20Bioinform.%202019;21(4):1209%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Slovin S, Carissimo A, Panariello F, Grimaldi A, Bouché V, Gambardella G, Cacchiarelli D. Single-Cell RNA Sequencing Analysis: A Step-by-Step Overview. In: RNA Bioinformatics. Edited by Picardi E. New York, NY: Springer US; 2021:343–365. 10.1007/978-1-0716-1307-8_19.</cite> [<a href="https://doi.org/10.1007/978-1-0716-1307-8_19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33835452/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Brendel M, Su C, Bai Z, Zhang H, Elemento O, Wang F. Application of deep learning on single-cell RNA sequencing data analysis: A review. Genom Proteom Bioinform. 2022;20(5):814–35.  10.1016/j.gpb.2022.11.011.</cite> [<a href="https://doi.org/10.1016/j.gpb.2022.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10025684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36528240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brendel%20M,%20Su%20C,%20Bai%20Z,%20Zhang%20H,%20Elemento%20O,%20Wang%20F.%20Application%20of%20deep%20learning%20on%20single-cell%20RNA%20sequencing%20data%20analysis:%20A%20review.%20Genom%20Proteom%20Bioinform.%202022;20(5):814%E2%80%9335.%C2%A0%2010.1016/j.gpb.2022.11.011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Ciortan M, Defrance M. GNN-based embedding for clustering scRNA-seq data. Bioinformatics. 2021;38(4):1037–44.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ciortan%20M,%20Defrance%20M.%20GNN-based%20embedding%20for%20clustering%20scRNA-seq%20data.%20Bioinformatics.%202021;38(4):1037%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Gan Y, Chen Y, Xu G, Guo W, Zou G. Deep enhanced constraint clustering based on contrastive learning for scRNA-seq data. Brief Bioinform. 2023. 10.1093/bib/bbad222.
</cite> [<a href="https://doi.org/10.1093/bib/bbad222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37313714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gan%20Y,%20Chen%20Y,%20Xu%20G,%20Guo%20W,%20Zou%20G.%20Deep%20enhanced%20constraint%20clustering%20based%20on%20contrastive%20learning%20for%20scRNA-seq%20data.%20Brief%20Bioinform.%202023.%2010.1093/bib/bbad222." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Gao H, Zhang B, Liu L, Li S, Gao X, Yu B. A universal framework for single-cell multi-omics data integration with graph convolutional networks. Brief Bioinform. 2023. 10.1093/bib/bbad081
</cite> [<a href="https://doi.org/10.1093/bib/bbad081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10199767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36929841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20H,%20Zhang%20B,%20Liu%20L,%20Li%20S,%20Gao%20X,%20Yu%20B.%20A%20universal%20framework%20for%20single-cell%20multi-omics%20data%20integration%20with%20graph%20convolutional%20networks.%20Brief%20Bioinform.%202023.%2010.1093/bib/bbad081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wang J, Ma A, Chang Y, Gong J, Jiang Y, Qi R, Wang C, Fu H, Ma Q, Xu D. scGNN is a novel graph neural network framework for single-cell RNA-seq analyses. Nat Commun. 2021;12(1):1882.  10.1038/s41467-021-22197-x.
</cite> [<a href="https://doi.org/10.1038/s41467-021-22197-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7994447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33767197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Ma%20A,%20Chang%20Y,%20Gong%20J,%20Jiang%20Y,%20Qi%20R,%20Wang%20C,%20Fu%20H,%20Ma%20Q,%20Xu%20D.%20scGNN%20is%20a%20novel%20graph%20neural%20network%20framework%20for%20single-cell%20RNA-seq%20analyses.%20Nat%20Commun.%202021;12(1):1882.%C2%A0%2010.1038/s41467-021-22197-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Luo Z, Xu C, Zhang Z, Jin W. A topology-preserving dimensionality reduction method for single-cell RNA-seq data using graph autoencoder. Sci Rep. 2021;11(1):20028.
</cite> [<a href="https://doi.org/10.1038/s41598-021-99003-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8501122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34625592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo%20Z,%20Xu%20C,%20Zhang%20Z,%20Jin%20W.%20A%20topology-preserving%20dimensionality%20reduction%20method%20for%20single-cell%20RNA-seq%20data%20using%20graph%20autoencoder.%20Sci%20Rep.%202021;11(1):20028." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Li X, Wang K, Lyu Y, Pan H, Zhang J, Stambolian D, Susztak K, Reilly MP, Hu G, Li M. Deep learning enables accurate clustering with batch effect removal in single-cell RNA-seq analysis. Nat Commun. 2020;11(1):2338.
</cite> [<a href="https://doi.org/10.1038/s41467-020-15851-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7214470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32393754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Wang%20K,%20Lyu%20Y,%20Pan%20H,%20Zhang%20J,%20Stambolian%20D,%20Susztak%20K,%20Reilly%20MP,%20Hu%20G,%20Li%20M.%20Deep%20learning%20enables%20accurate%20clustering%20with%20batch%20effect%20removal%20in%20single-cell%20RNA-seq%20analysis.%20Nat%20Commun.%202020;11(1):2338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Gan Y, Huang X, Zou G, Zhou S, Guan J. Deep structural clustering for single-cell RNA-seq data jointly through autoencoder and graph neural network. Brief Bioinform. 2022;23(2). 10.1093/bib/bbac018.</cite> [<a href="https://doi.org/10.1093/bib/bbac018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35172334/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Yang W, Wang P, Luo M, Cai Y, Xu C, Xue G, Jin X, Cheng R, Que J, Pang F, et al. DeepCCI: a deep learning framework for identifying cell–cell interactions from single-cell RNA sequencing data. Bioinformatics. 2023. 10.1093/bioinformatics/btad596
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10558043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37740953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20W,%20Wang%20P,%20Luo%20M,%20Cai%20Y,%20Xu%20C,%20Xue%20G,%20Jin%20X,%20Cheng%20R,%20Que%20J,%20Pang%20F,%20et%20al.%20DeepCCI:%20a%20deep%20learning%20framework%20for%20identifying%20cell%E2%80%93cell%20interactions%20from%20single-cell%20RNA%20sequencing%20data.%20Bioinformatics.%202023.%2010.1093/bioinformatics/btad596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Liu T, Jia C, Bi Y, Guo X, Zou Q, Li F. scDFN: enhancing single-cell RNA-seq clustering with deep fusion networks. Brief Bioinform. 2024. 10.1093/bib/bbae486.
</cite> [<a href="https://doi.org/10.1093/bib/bbae486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11456827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39373051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20T,%20Jia%20C,%20Bi%20Y,%20Guo%20X,%20Zou%20Q,%20Li%20F.%20scDFN:%20enhancing%20single-cell%20RNA-seq%20clustering%20with%20deep%20fusion%20networks.%20Brief%20Bioinform.%202024.%C2%A010.1093/bib/bbae486." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Tian T, Zhang J, Lin X, Wei Z, Hakonarson H. Model-based deep embedding for constrained clustering analysis of single cell RNA-seq data. Nat Commun. 2021;12(1):1873.
</cite> [<a href="https://doi.org/10.1038/s41467-021-22008-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7994574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33767149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian%20T,%20Zhang%20J,%20Lin%20X,%20Wei%20Z,%20Hakonarson%20H.%20Model-based%20deep%20embedding%20for%20constrained%20clustering%20analysis%20of%20single%20cell%20RNA-seq%20data.%20Nat%20Commun.%202021;12(1):1873." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Wang X, Hu Z, Yu T, Wang Y, Wang R, Wei Y, Shu J, Ma J, Li Y. Con-AAE: contrastive cycle adversarial autoencoders for single-cell multi-omics alignment and integration. Bioinformatics. 2023. 10.1093/bioinformatics/btad162
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10101696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36975610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Hu%20Z,%20Yu%20T,%20Wang%20Y,%20Wang%20R,%20Wei%20Y,%20Shu%20J,%20Ma%20J,%20Li%20Y.%20Con-AAE:%20contrastive%20cycle%20adversarial%20autoencoders%20for%20single-cell%20multi-omics%20alignment%20and%20integration.%20Bioinformatics.%202023.%2010.1093/bioinformatics/btad162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wang Y, Yu Z, Li S, Bian C, Liang Y, Wong K-C, Li X. scBGEDA: deep single-cell clustering analysis via a dual denoising autoencoder with bipartite graph ensemble clustering. Bioinformatics. 2023. 10.1093/bioinformatics/btad075.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9925104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36734596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20Yu%20Z,%20Li%20S,%20Bian%20C,%20Liang%20Y,%20Wong%20K-C,%20Li%20X.%20scBGEDA:%20deep%20single-cell%20clustering%20analysis%20via%20a%20dual%20denoising%20autoencoder%20with%20bipartite%20graph%20ensemble%20clustering.%20Bioinformatics.%202023.%2010.1093/bioinformatics/btad075." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Tan D, Yang C, Wang J, Su Y, Zheng C. scAMAC: self-supervised clustering of scRNA-seq data based on adaptive multi-scale autoencoder. Brief Bioinform. 2024. 10.1093/bib/bbae068.
</cite> [<a href="https://doi.org/10.1093/bib/bbae068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10905526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38426327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan%20D,%20Yang%20C,%20Wang%20J,%20Su%20Y,%20Zheng%20C.%20scAMAC:%20self-supervised%20clustering%20of%20scRNA-seq%20data%20based%20on%20adaptive%20multi-scale%20autoencoder.%20Brief%20Bioinform.%202024.%C2%A010.1093/bib/bbae068." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Wang H-Y, Zhao J-P, Zheng C-H, Su Y-S. scGMAAE: gaussian mixture adversarial autoencoders for diversification analysis of scRNA-seq data. Brief Bioinform. 2023. 10.1093/bib/bbac585.
</cite> [<a href="https://doi.org/10.1093/bib/bbac585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36592058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20H-Y,%20Zhao%20J-P,%20Zheng%20C-H,%20Su%20Y-S.%20scGMAAE:%20gaussian%20mixture%20adversarial%20autoencoders%20for%20diversification%20analysis%20of%20scRNA-seq%20data.%20Brief%20Bioinform.%202023.%2010.1093/bib/bbac585." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Tran B, Tran D, Nguyen H, Ro S, Nguyen T. scCAN: single-cell clustering using autoencoder and network fusion. Sci Rep. 2022;12(1):10267. 10.1038/s41598-022-14218-6.
</cite> [<a href="https://doi.org/10.1038/s41598-022-14218-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9206025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35715568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tran%20B,%20Tran%20D,%20Nguyen%20H,%20Ro%20S,%20Nguyen%20T.%20scCAN:%20single-cell%20clustering%20using%20autoencoder%20and%20network%20fusion.%20Sci%20Rep.%202022;12(1):10267.%2010.1038/s41598-022-14218-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Feng X, Xiu Y-H, Long H-X, Wang Z-T, Bilal A, Yang L-M. Advancing single-cell RNA-seq data analysis through the fusion of multi-layer perceptron and graph neural network. Brief Bioinform. 2024. 10.1093/bib/bbad481</cite> [<a href="https://doi.org/10.1093/bib/bbad481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10764207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38171931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng%20X,%20Xiu%20Y-H,%20Long%20H-X,%20Wang%20Z-T,%20Bilal%20A,%20Yang%20L-M.%20Advancing%20single-cell%20RNA-seq%20data%20analysis%20through%20the%20fusion%20of%20multi-layer%20perceptron%20and%20graph%20neural%20network.%20Brief%20Bioinform.%202024.%2010.1093/bib/bbad481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Fang Z, Zheng R, Li M. scMAE: a masked autoencoder for single-cell RNA-seq clustering. Bioinformatics. 2024.  10.1093/bioinformatics/btae020.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10832357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38230824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang%20Z,%20Zheng%20R,%20Li%20M.%20scMAE:%20a%20masked%20autoencoder%20for%20single-cell%20RNA-seq%20clustering.%20Bioinformatics.%202024.%C2%A0%2010.1093/bioinformatics/btae020." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Liu Z, Liang Y, Wang G, Zhang T. scLEGA: an attention-based deep clustering method with a tendency for low expression of genes on single-cell RNA-seq data. Brief Bioinform. 2024. 10.1093/bib/bbae371.
</cite> [<a href="https://doi.org/10.1093/bib/bbae371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11281828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39060167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Z,%20Liang%20Y,%20Wang%20G,%20Zhang%20T.%20scLEGA:%20an%20attention-based%20deep%20clustering%20method%20with%20a%20tendency%20for%20low%20expression%20of%20genes%20on%20single-cell%20RNA-seq%20data.%20Brief%20Bioinform.%202024.%2010.1093/bib/bbae371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Yu Z, Lu Y, Wang Y, Tang F, Wong K-C, Li X. ZINB-based graph embedding autoencoder for single-cell RNA-seq interpretations. Proceedings of the AAAI Conference on Artificial Intelligence. 2022;36(4):4671–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yu%20Z,%20Lu%20Y,%20Wang%20Y,%20Tang%20F,%20Wong%20K-C,%20Li%20X.%20ZINB-based%20graph%20embedding%20autoencoder%20for%20single-cell%20RNA-seq%20interpretations.%20Proceedings%20of%20the%20AAAI%20Conference%20on%20Artificial%20Intelligence.%202022;36(4):4671%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Zhang Z, Liu Y, Xiao M, Wang K, Huang Y, Bian J, Yang R, Li F. Graph contrastive learning as a versatile foundation for advanced scRNA-seq data analysis. Brief Bioinform. 2024. 10.1093/bib/bbae558
</cite> [<a href="https://doi.org/10.1093/bib/bbae558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11530284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39487083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Z,%20Liu%20Y,%20Xiao%20M,%20Wang%20K,%20Huang%20Y,%20Bian%20J,%20Yang%20R,%20Li%20F.%20Graph%20contrastive%20learning%20as%20a%20versatile%20foundation%20for%20advanced%20scRNA-seq%20data%20analysis.%20Brief%20Bioinform.%202024.%2010.1093/bib/bbae558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359927/bin/12864_2025_11941_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (639.7KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The benchmark data can be accessed from Gene Expression Omnibus with accession numbers of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65525" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE65525</a> (Mouse_ES_cell), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE94333" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE94333</a> (Adam), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74672" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74672</a> (Romanov), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE85241" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE85241</a> (Muraro), <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE23341" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE23341</a> (Qs_Trachea+Qs_Diaphragm), Yan+Qx_muscle (<a href="https://github.com/JiangBioLab/DeepCCI/tree/main/Cluster_model/dataset" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/JiangBioLab/DeepCCI/tree/main/Cluster_model/dataset</a>), and Qs_Muscle (<a href="https://github.com/Philyzh8/scTAG/tree/master/data/Quake_Smart-seq2_Limb_Muscle" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Philyzh8/scTAG/tree/master/data/Quake_Smart-seq2_Limb_Muscle</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11941-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11941.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359927/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359927/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359927%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359927/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359927/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359927/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826008/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359927/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826008/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359927/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359927/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="PoCCv8dNjJkP4X0ckhXUwyohvsS3cCqHJfjRXrzAKy00E8Ky0ODgXV20RZbynWYz">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
