
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Elucidating structural and molecular requirements of somatostatin subtype-4 agonist bound complexes using quantum mechanics approaches - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152543E98AF2EFB30543E9004BB978BF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="rscsd">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332849/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Organic &amp; Biomolecular Chemistry">
<meta name="citation_title" content="Elucidating structural and molecular requirements of somatostatin subtype-4 agonist bound complexes using quantum mechanics approaches">
<meta name="citation_author" content="Olivia Slater">
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, School of Pharmacy, Southern Illinois University, Edwardsville, IL 62026, USA, Email: mkontoy@siue.edu">
<meta name="citation_author" content="Maria Kontoyianni">
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, School of Pharmacy, Southern Illinois University, Edwardsville, IL 62026, USA, Email: mkontoy@siue.edu">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="23">
<meta name="citation_issue" content="33">
<meta name="citation_firstpage" content="7611">
<meta name="citation_doi" content="10.1039/d5ob00775e">
<meta name="citation_pmid" content="40778438">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332849/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332849/">
<meta name="description" content="Somatotropin-release Inhibiting Factor (SRIF) binds somatostatin subtype-4 receptor (sst4) within the neocortex to increase amyloid beta catabolism, and possibly effects neuronal plaque formation and disease progression of Alzheimer&#x27;s disease. ...">
<meta name="og:title" content="Elucidating structural and molecular requirements of somatostatin subtype-4 agonist bound complexes using quantum mechanics approaches">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Somatotropin-release Inhibiting Factor (SRIF) binds somatostatin subtype-4 receptor (sst4) within the neocortex to increase amyloid beta catabolism, and possibly effects neuronal plaque formation and disease progression of Alzheimer&#x27;s disease. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332849/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332849">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1039/d5ob00775e"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332849%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332849/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332849/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332849/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-rscsd.png" alt="Royal Society of Chemistry logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Royal Society of Chemistry" title="Link to Royal Society of Chemistry" shape="default" href="https://doi.org/10.1039/d5ob00775e" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Org Biomol Chem</button></div>. 2025 Aug 8;23(33):7611–7622. doi: <a href="https://doi.org/10.1039/d5ob00775e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/d5ob00775e</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Org%20Biomol%20Chem%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Org%20Biomol%20Chem%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Org%20Biomol%20Chem%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Org%20Biomol%20Chem%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Elucidating structural and molecular requirements of somatostatin subtype-4 agonist bound complexes using quantum mechanics approaches</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Slater%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Olivia Slater</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Olivia Slater</span></h3>
<div class="p">
<sup>a</sup>Department of Pharmaceutical Sciences, School of Pharmacy, Southern Illinois University, Edwardsville, IL 62026, USA, Email: mkontoy@siue.edu</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Slater%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olivia Slater</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kontoyianni%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Maria Kontoyianni</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Maria Kontoyianni</span></h3>
<div class="p">
<sup>a</sup>Department of Pharmaceutical Sciences, School of Pharmacy, Southern Illinois University, Edwardsville, IL 62026, USA, Email: mkontoy@siue.edu</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kontoyianni%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria Kontoyianni</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="affa">
<sup>a</sup>Department of Pharmaceutical Sciences, School of Pharmacy, Southern Illinois University, Edwardsville, IL 62026, USA, Email: mkontoy@siue.edu</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 10; Accepted 2025 Jul 30; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>This journal is © The Royal Society of Chemistry</div>
<p>This article is licensed under a <a href="https://creativecommons.org/licenses/by-nc/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-Non Commercial 3.0 Unported Licence</a>. You can use material from this article in other publications without requesting further permissions from the RSC, provided that the correct acknowledgement is given and it is not used for commercial purposes.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332849  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40778438/" class="usa-link">40778438</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Somatotropin-release Inhibiting Factor (SRIF) binds somatostatin subtype-4 receptor (sst4) within the neocortex to increase amyloid beta catabolism, and possibly effects neuronal plaque formation and disease progression of Alzheimer's disease. Recently, sst4 in complex with SRIF or a small molecule agonist were resolved using cryo-EM, and mutagenesis identified amino acid residues that contributed to activity. In the present study, we used mixed Quantum Mechanics/Molecular Mechanics (QM/MM) to refine the experimental sst4 complexes and decipher agonist-sst4 interactions. SRIF rendered the complex more stable, while interactions with asparagines 199 and 293 seen in the experimental complexes were lost and replaced with Gln279. We also addressed long-standing questions related to sst4 agonist binding. Toward that end, we used quantum mechanics, molecular docking, and QM/MM refinement methods and employed pair-wise comparisons or matched molecular pairs to explore the effects of chemical substitutions on ligand properties and energetics. We identified amino acid interactions that were consistent among all refined complexes, and those that differentiated high affinity binders. Finally, we considered several parameters to discern which correlated best with affinity, and highlighted aspects often overlooked or yet to be explored in order to enhance lead optimization outcomes.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>Somatotropin-release Inhibiting Factor (SRIF) binds somatostatin subtype-4 receptor (sst4) within the neocortex to increase amyloid beta catabolism, and possibly effects neuronal plaque formation and disease progression of Alzheimer's disease.<a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-ga.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/fe04f4f57bc4/d5ob00775e-ga.jpg" loading="lazy" id="ga" height="473" width="611" alt="graphic file with name d5ob00775e-ga.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Alzheimer's disease (AD) is the most common form of age-related dementia that affects over 55 million people globally. Disease management costs an estimated 1.3 trillion US dollars annually, according to the World Health Organization. Main neuropathological hallmarks of AD include loss of neurons and synapses, neurofibrillary tangles, and neuronal plaques deposited between neurons due to accumulated amyloid-beta (Aβ) peptide starting in the neocortex. Brain neurons express amyloid-precursor protein (APP) which is cleaved by β- and γ-secretases to release Aβ. Under normal physiological conditions, Aβ is rapidly catabolized by neprilysin and cleared from the brain. Neprilysin levels decrease with age to promote Aβ dyshomeostasis and plaque formation.<sup><a href="#cit1" class="usa-link" aria-describedby="cit1">1–3</a></sup></p>
<p>Somatostatin, or Somatotropin-Release Inhibiting Factor (SRIF), is a cyclic peptide of either 14 or 28 amino acid residues that affects the Aβ degradation pathway.<sup><a href="#cit4" class="usa-link" aria-describedby="cit4">4</a></sup> SRIF-14 is the main isoform expressed in the central nervous system (CNS) where it modulates neuroplasticity and neurotransmission by interacting with inhibitory G-protein coupled receptors of the somatostatin receptor (sst) family. Within the neocortex, SRIF binds sst4 to stimulate neprilysin and Aβ catabolism.<sup><a href="#cit5" class="usa-link" aria-describedby="cit5">5–8</a></sup></p>
<p>Poor bioavailability due to high molecular weight and short plasma half-life limit the potential of SRIF-14 as a CNS drug. Researchers have developed peptidomimetics and non-peptide agonists with chemical features mimicking SRIF residues important for potency (Phe7), and target binding (Trp8 and Lys9).<sup><a href="#cit9" class="usa-link" aria-describedby="cit9">9,10</a></sup> Recently, sst4 complexes bound by SRIF-14 and a small molecule agonist were resolved using cryo-Electron Microscopy (cryo-EM).<sup><a href="#cit11" class="usa-link" aria-describedby="cit11">11</a></sup> Both ligands formed electrostatic interactions with Asp126 through either the Lys9 of SRIF-14 or the moiety mimicking Lys9 of the small molecule, and Asn 199 with the main chain atoms of SRIF or the terminal amide of the small molecule (<a href="#fig1" class="usa-link">Fig. 1</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">Fig. 1. Dipole interactions observed in experimental complexes of sst4 bound by (a) SRIF-14 and (b) small molecule agonists (PDB IDs: 7XMS and 7XMT).</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/252ff85a71bd/d5ob00775e-f1.jpg" loading="lazy" height="345" width="760" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Internal ligands were developed through a structure-based design strategy. The earlier non-peptide agonists incorporated aromatic, heteroaromatic, and imidazole to mimic Phe7, Trp8, and Lys9 attached to the 1,2,4-triazole scaffold, whereas our most recent work incorporated 3-thio-1,2,4-triazole.<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12–14</a></sup> Initial efforts modelled agonist binding to sst4 using model-built structures with the β<sub>2</sub>-adrenergic and μ-opioid X-ray structures as templates.<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15–19</a></sup> Most recently we employed the cryo-EM sst4 structures to dock the thio-triazole series.<sup><a href="#cit20" class="usa-link" aria-describedby="cit20">20</a></sup> High and moderate affinity ligands formed bidentate interactions between Asp126 and the imidazolyl and indolyl hydrogens. Alternatively, the Lys9 mimetic imidazole hydrogen-bonded with Asp126, whereas the indole or sulfur interacted with Gln279. Hydrophobic and van der Waals (vdW) interactions involved Leu123, Leu200, Trp202, Phe211, Phe275, and Leu297. In certain cases, the 3-thio-1,2,4-triazole was positioned near Asn199. For the most part, low affinity binders formed fewer interactions and adopted more strained conformations.<sup><a href="#cit20" class="usa-link" aria-describedby="cit20">20</a></sup></p>
<p>In this work, we capitalized on experimental information and computational resources to address long-standing questions about the chemical features of agonists that render high affinity for sst4. We employed quantum mechanics (QM) and Quantum Mechanics/Molecular Mechanics (QM/MM) calculations to investigate quantum effects on agonist and receptor static structures. The questions we aimed to address were: (1) Can we decipher which amino acids are essential for binding with sst4? (2) What is the significance of the sulfur atom? (3) Does affinity correlate with certain parameters? (4) What are the best practices moving forward? Toward 1, we used experimental bound complexes (PDB IDs: 7XMS and 7XMT),<sup><a href="#cit11" class="usa-link" aria-describedby="cit11">11</a></sup> or with the ligands removed. We performed QM/MM calculations of sst4's binding pocket in the bound and unbound configurations to observe changes energetically, and to identify which interactions were maintained, lost, or gained due to refinement. Toward 2, we compared QM-optimized ligand geometries of matched molecular pairs. Toward 3, optimized ligand conformations were docked into the refined sst4 binding pockets and docked solutions were optimized with QM/MM. Refined complexes were compared in a pair-wise manner to identify consistent binding patterns of high affinity ligands. We looked at energies of refined complexes and properties of optimized ligands to explore correlations with affinity. Collectively, these results gave insights toward 4 and how to best approach future lead optimization endeavors for this particular system.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="sec3"><h3 class="pmc_sec_title">2.1. Structure preparation</h3>
<p>Sst4 in complex with SRIF or small molecule were extracted from the Protein Data Bank (PDB IDs: 7XMS and 7XMT). Hydrogens were added with ProPka<sup><a href="#cit21" class="usa-link" aria-describedby="cit21">21</a></sup> and bond orders were reassigned using the Cambridge Crystallographic Database.<sup><a href="#cit22" class="usa-link" aria-describedby="cit22">22</a></sup> Missing side-chains were added using Prime and tautomeric states were assigned with Epik at pH 7.4 ± 2.0. Ligands were either maintained to represent bound configurations or extracted to obtain unbound configurations of sst4. To relax the structures and to alleviate crystal packing, while maintaining observed conformations, agonist-bound and unbound structures were minimized with 1000 steps of steepest descent, followed by 2000 conjugate gradient with rms of 0.1 to resolve poor contacts. Subsequently, short molecular dynamics (MD) simulations were carried out using the CHARMM force field within Discovery Studio (BIOVIA, Dassault Systèmes, Discovery Studio Modeling Environment 2023, San Diego). The systems were heated to target temperature 300 K for 10 ps, followed by 100 ps equilibration. The final production stage was 1 ns and used the constant-volume thermodynamic ensemble with Leapfrog Verlet integration and SHAKE<sup><a href="#cit23" class="usa-link" aria-describedby="cit23">23</a></sup> constraints applied.</p></section><section id="sec4"><h3 class="pmc_sec_title">2.2. Optimizing conformations of sst4 binding pockets</h3>
<p>Schrodinger's Qsite was used to perform QM/MM calculations (QSite, Schrödinger, LLC, New York, NY, 2024).<sup><a href="#cit24" class="usa-link" aria-describedby="cit24">24,25</a></sup> We defined the binding pockets as amino acids within 10 Å of SRIF or the small molecule agonist which totaled 107 or 129 residues, respectively. We considered three definitions of the QM region that included (1) ligand atoms only, (2) ligand atoms plus side-chain atoms within 2 Å of the ligand, and (3) ligand atoms plus side-chain atoms within 6 Å of the ligand. We have reported values for the largest definition (ligand plus 6 Å). The DFT-B3LYP method with 6-31G basis set was used to treat the QM region, which employed the minimization method with a maximum of 100 iterations. The net charge of the QM region was equal to zero with a spin multiplicity of one. The OPLS_2005 force field was used to define the potential energy function for the MM region, with dielectric constant set to 1.0. We employed a maximum of 1000 minimization cycles using conjugate gradient algorithm with initial step size of 0.05 or 0.10 and a maximum step size of 1.0. An energy change criterion of 0.1 and a gradient criterion of 0.01 defined convergence for the MM region.</p></section><section id="sec5"><h3 class="pmc_sec_title">2.3. Calculating properties of sst4 binding pockets</h3>
<p>MM, electronic, and the total QM/MM energies of the binding pockets were taken directly from QM/MM calculations. We performed single point energy calculations of the optimized conformations using MOPAC (Molecular Orbital PACkage).<sup><a href="#cit26" class="usa-link" aria-describedby="cit26">26,27</a></sup> Briefly, MOPAC performs semiempirical calculations using NDDO (neglect of diatomic differential overlap) theory. We employed the PM7 Hamiltonian and RHF wavefunction to calculate atomic electrostatic potential (ESP) charges fit to atom centers and total self-consistent field (SCF) energies. Calculations included all amino acid residues that defined the binding cavities (107 for SRIF-sst4 and 129 for small molecule-sst4 complexes). We compared the bound and unbound conformations, each before and after optimization, to visually inspect changes in conformation, and calculated RMSD's of side-chain heavy atoms using per-residue alignments of side-chain heavy atoms.</p></section><section id="sec6"><h3 class="pmc_sec_title">2.4. QM optimization of small molecule conformations</h3>
<p>Ligand conformations were optimized within Biovia's Discovery Studio 2023. Atom types were assigned using MMFF, and the CHARMM forcefield was applied. Initial structures were energy minimized using steepest descent followed by conjugate gradient, with a maximum of 200 steps each and rms gradients set to 0.1 and 0.01, respectively. Minimized conformations were used as initial structures for systematic searches to obtain low energy conformations. A maximum of 1000 conformations per ligand were generated with an energy threshold of 20 kcal mol<sup>−1</sup>, RMSD cutoff of zero and torsion increments of 15 degrees. Each conformation was energy minimized using conjugate gradient with rms 0.1. Conformations with the highest solvent accessible surface area were analyzed. Selected conformations underwent geometry optimization using Schrodinger's Jaguar.<sup><a href="#cit28" class="usa-link" aria-describedby="cit28">28</a></sup> We employed density functional theory (DFT) calculations using B3LYP-D3/6-31G** basis set. The usual nonrelativistic Hamiltonian was used and the SCF spin was treated automatically. Accurate SCF level was used and a maximum 48 iterations and energy change of 5 × 10<sup>−5</sup> Hartrees defined convergence. The QM optimized ligands were subsequently docked into the four configurations of sst4's binding pockets obtained as described above. We allowed a maximum of 30 poses per ligand. In addition, we extracted SRIF and the small molecule ligands from refined complexes and performed single point energy (SPE) calculations <em>in vacuum</em> to study electrostatic properties of bound conformations. We calculated ESP partial charges, net molecular charges, and mapped ESPs onto the isodensity surfaces. We included SRIF-14 resolved from solution NMR in these experiments because Thr10 was not resolved in the cryo-EM complex (PDB ID: 2MI1).<sup><a href="#cit29" class="usa-link" aria-describedby="cit29">29</a></sup> Both methods give similar conformations of the peptide, and we found properties of either structure led to the same conclusions.</p></section><section id="sec7"><h3 class="pmc_sec_title">2.5. Calculating ligand properties</h3>
<p>We tested other methods and basis sets to compare energies with results of B3LYP-D3/6-31G**, and the same conclusions were drawn. Therefore, conformational energies were taken from the final geometry optimization step. Gibbs free energy was calculated using the equation Δ<em>G</em> = <em>RT</em> ln <em>K</em><sub>d</sub>, where <em>K</em><sub>d</sub> = 1/10<sup>−p<em>K</em><sub>a</sub></sup>. Temperature was set to 310 Kelvin, and p<em>K</em><sub>a</sub>'s were calculated using quantum chemical approaches within Jaguar, as described above, with the acidity constant corresponding to the most ionizable group in each molecule. Differences in binding free energy (ΔΔ<em>G</em>) between ligands within a pair (L1 and L2) were calculated as ΔΔ<em>G</em> = Δ<em>G</em>L2 − Δ<em>G</em>L1, where ΔΔ<em>G</em> &gt; 0 kcal mol<sup>−1</sup> indicated L1 had higher affinity. Atomic ESPs fit to atom centers were calculated using a spherical grid, and ESP surface maps were generated with grid density set to 20 pts per Å. ESP-properties that employed statistical measures of variance, maximum, minimum, and mean values were calculated and included local polarity (<em>Π</em><sub>pol</sub>) and charge balance (<em>υ</em><sub>bal</sub>) parameters. To calculate solvation energies, optimized ligands were submitted for single-point energy (SPE) calculations that employed the Poisson Boltzmann Finite element method solvent model.<sup><a href="#cit30" class="usa-link" aria-describedby="cit30">30</a></sup> Acidity constants (p<em>K</em><sub>a</sub>'s) were calculated using Jaguar's Micro-p<em>K</em><sub>a</sub> method<sup><a href="#cit31" class="usa-link" aria-describedby="cit31">31</a></sup> within Schrodinger's p<em>K</em><sub>a</sub> prediction module. Briefly, Micro-p<em>K</em><sub>a</sub> calculations include solvation free energies of protonated and deprotonated species, fitting charged species to experimental data with surface tension corrections trained on common functional groups applied for neutral species. Gas-phase deprotonation energy represents the free energy difference (Δ<em>G</em><sub>g</sub>) between the ligand's protonated and deprotonated states. The summation of Δ<em>G</em><sub>g</sub> and solvation free energies of neutral and ionic species equals the free energy of deprotonation (Δ<em>G</em><sub>a</sub>) which is used to calculate the molecule's p<em>K</em><sub>a</sub> as p<em>K</em><sub>a</sub> = Δ<em>G</em><sub>a</sub>/2.303RT. An empirical correction is applied after <em>ab initio</em> calculations of p<em>K</em><sub>a</sub>'s to better agree with experimental. We employed the automatic search to consider all p<em>K</em><sub>a</sub> atoms, and compared accurate and quick SCF levels which gave approximately equivalent p<em>K</em><sub>a</sub>'s for optimized ligand geometries.<sup><a href="#cit32" class="usa-link" aria-describedby="cit32">32</a></sup> A schematic representation of the workflow is shown in <a href="#fig2" class="usa-link">Fig. 2</a>.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Fig. 2. Schematic representation of the computational workflow.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/2ccf786ca601/d5ob00775e-f2.jpg" loading="lazy" height="517" width="756" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section></section><section id="sec8"><h2 class="pmc_sec_title">3. Results and discussion</h2>
<section id="sec9"><h3 class="pmc_sec_title">3.1. Changes in experimental structures after refinement</h3>
<section id="sec10"><h4 class="pmc_sec_title">3.1.1. Calculated properties of sst4 binding pockets</h4>
<p>We first examined properties of experimental binding pockets refined with QM/MM (<a href="#tab1" class="usa-link">Table 1</a>; PDB IDs: 7XMS and 7XMT). MM energy equaled the sum of stretch, bend, and torsion energies across the interface. Electronic energy equaled the sum of one- and two-electron terms plus nuclear repulsion within the QM region. The SCF total energy was calculated semiempirically. SRIF-sst4 complex had MM energy of −3.7 × 10<sup>3</sup> kcal mol<sup>−1</sup>, electronic energy of −7.2 × 10<sup>7</sup> kcal mol<sup>−1</sup>, and total energy equaled −7.9 × 10<sup>6</sup> kcal mol<sup>−1</sup> which were the lowest energies of all configurations. When the peptide was removed, the pocket MM energy was −2.4 × 10<sup>3</sup> kcal mol<sup>−1</sup>, electronic energy was −2.9 × 10<sup>7</sup> kcal mol<sup>−1</sup>, and total energy equaled −4.5 × 10<sup>6</sup> kcal mol<sup>−1</sup>. Small molecule-sst4 complex had MM energy of −2.6 × 10<sup>3</sup> kcal mol<sup>−1</sup>, electronic energy of −4.6 × 10<sup>7</sup> kcal mol<sup>−1</sup>, and total energy equal to −5.7 × 10<sup>6</sup> kcal mol<sup>−1</sup>. Small molecule bound <em>versus</em> unbound configurations had the same MM energies, but electronic and total energies were higher for the unbound configuration and equaled −3.2 × 10<sup>7</sup> kcal mol<sup>−1</sup> and −4.6 × 10<sup>6</sup> kcal mol<sup>−1</sup>, respectively. As expected, all energies were lower for bound configurations, while cavity volumes decreased when either ligand was bound, suggesting conformational changes upon binding. In addition, decreased pocket volumes upon binding suggest tighter and more specific bound complexes (<a href="#tab1" class="usa-link">Table 1</a>). Our results indicate both agonists stabilized the receptor conformation, but SRIF rendered a more stable complex compared to the small molecule. For residues within the QM region, the small molecule complex carried a net charge of −5.9 where the ligand contributed −4.5, <em>versus</em> the peptide complex which had a net charge of −25.7 with SRIF-14 being neutral.</p>
<section class="tw xbox font-sm" id="tab1"><h5 class="obj_head">Table 1. Calculated properties of sst4 binding pockets in bound and unbound configurations after QM/MM refinement.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" valign="bottom" colspan="1"> </th>
<th colspan="2" align="left" rowspan="1">Small molecule/sst4 (PDB ID: 7XMT)</th>
<th colspan="2" align="left" rowspan="1">SRIF-14/sst4 (PDB ID: 7XMS)</th>
</tr>
<tr>
<th rowspan="1" colspan="1">Bound</th>
<th rowspan="1" colspan="1">Unbound</th>
<th rowspan="1" colspan="1">Bound</th>
<th rowspan="1" colspan="1">Unbound</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">MM energy<a href="#tab1fna" class="usa-link"><sup>a</sup></a> (kcal mol<sup>−1</sup>)</td>
<td rowspan="1" colspan="1">−2.6 × 10<sup>3</sup>
</td>
<td rowspan="1" colspan="1">−2.6 × 10<sup>3</sup>
</td>
<td rowspan="1" colspan="1">−3.7 × 10<sup>3</sup>
</td>
<td rowspan="1" colspan="1">−2.4 × 10<sup>3</sup>
</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Electronic energy<a href="#tab1fnb" class="usa-link"><sup>b</sup></a> (kcal mol<sup>−1</sup>)</td>
<td rowspan="1" colspan="1">−4.6 × 10<sup>7</sup>
</td>
<td rowspan="1" colspan="1">−3.2 × 10<sup>7</sup>
</td>
<td rowspan="1" colspan="1">−7.2 × 10<sup>7</sup>
</td>
<td rowspan="1" colspan="1">−2.9 × 10<sup>7</sup>
</td>
</tr>
<tr>
<td rowspan="1" colspan="1">SCF total energy<a href="#tab1fnc" class="usa-link"><sup>c</sup></a> (kcal mol<sup>−1</sup>)</td>
<td rowspan="1" colspan="1">−5.7 × 10<sup>6</sup>
</td>
<td rowspan="1" colspan="1">−4.6 × 10<sup>6</sup>
</td>
<td rowspan="1" colspan="1">−7.9 × 10<sup>6</sup>
</td>
<td rowspan="1" colspan="1">−4.5 × 10<sup>6</sup>
</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Cavity volume (Å<sup>3</sup>)</td>
<td rowspan="1" colspan="1">1.8 × 10<sup>4</sup>
</td>
<td rowspan="1" colspan="1">1.9 × 10<sup>4</sup>
</td>
<td rowspan="1" colspan="1">1.5 × 10<sup>4</sup>
</td>
<td rowspan="1" colspan="1">1.8 × 10<sup>4</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tab1fna">
<sup>a</sup><p class="display-inline">MM energy included QM and MM regions.</p>
</div>
<div class="fn" id="tab1fnb">
<sup>b</sup><p class="display-inline">Electronic energy included atoms within the QM region only.</p>
</div>
<div class="fn" id="tab1fnc">
<sup>c</sup><p class="display-inline">Semiempirical SCF total energy included all atoms within the binding pocket.</p>
</div>
</div></section></section><section id="sec11"><h4 class="pmc_sec_title">3.1.2. Structural superimpositions</h4>
<p>We superimposed refined complexes with respective experimental structures. Rotation of Asp126 improved bond angles with Lys9 or the Lys9 mimetic of both agonists. However, larger changes were observed within the small molecule complex. Leu123, Met130, and Phe131 located at the bottom of the pocket moved away from the small molecule. ECL2 was slightly translated towards the ligand which mainly affected side-chain rotamers. Comparison of the refined structures revealed that receptor conformations were highly similar. The greatest differences were between ECL2, which was rotated approximately 45 degrees upward to accommodate SRIF. The small molecule complex had ECL2 spanning across the pocket entrance, almost like open <em>versus</em> closed. Within 6 Å of the ligand, however, there were only small differences in rotamers that we attributed to the size of SRIF which occupied more space compared to the small molecule or unbound cavities.</p></section><section id="sec12"><h4 class="pmc_sec_title">3.1.3. Determining amino acids of importance in binding affinity</h4>
<p>As mentioned in the Introduction, the main electrostatics observed in the SRIF-bound experimental structure were with Asp126 and Asn199, while Phe6 formed hydrophobic interactions with Asn293. Similarly, the small molecule interacted with Asp126 and Asn199, with its phenyl ring sitting near Leu123, Val103, and Leu297.</p>
<p>To determine residues most essential for agonist binding, we assessed which interactions were maintained or gained upon refinement and identified the amino acids that interacted with SRIF and the small molecule. Interactions with Asp126 were maintained, interactions with Gln279 were gained, and both agonists formed hydrophobic interactions with Leu123 and Leu297 in both refined complexes. SRIF's Lys9 maintained the interaction with Asp126 and the bond distance decreased from 2.7 Å to 1.8 Å. Leu297 formed a hydrophobic interaction with the hydrocarbon side-chain of Lys9. Tryptophan 8 hydrogen-bonded with the backbone carbonyl of Leu123. There was also a polar-π interaction formed between Gln279 and Trp8 (<a href="#fig3" class="usa-link">Fig. 3a</a>). The small molecule maintained the salt-bridge with Asp126 and the bond distance decreased from 2.5 Å to 1.8 Å. The sulfonamide and Gln279 formed a polar vdW side-chain interaction also. Naphthyl and Phe211's side-chain became oriented edge-to-face (<a href="#fig3" class="usa-link">Fig. 3b</a>). Backbone atoms of residues within ECL2 underwent small translations, and asparagine residues had different rotamers in refined complexes, hence interactions with Asn199 were not maintained. The side-chain RMSD's for the SRIF- and small molecule-bound complexes before and after refinement are in Table S1. Amino acid positioning changes are larger in the SRIF-complex when compared to the small molecule, suggesting tighter binding.</p>
<figure class="fig xbox font-sm" id="fig3"><h5 class="obj_head">Fig. 3. Superimpositions of (a) experimental (orange) and QM/MM treated (blue) complexes of sst4 bound by SRIF-14 (PDB ID: 7XMS) and (b) experimental (green) and QM/MM treated (purple) complexes of sst4 bound by a small molecule agonist (PDB ID: 7XMT).</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/17fe16b21ad5/d5ob00775e-f3.jpg" loading="lazy" height="382" width="790" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>We extracted SRIF and the small molecule ligand from refined complexes and performed single point energy calculations <em>in vacuum</em> to study electrostatic properties of bound conformations. We calculated ESP partial charges, net molecular charges, and mapped ESPs onto the isodensity surfaces. SRIF had net molecular charge of +3 whereas the small molecule had +1. The Trp8 groups carried similar positive partial charges equal to 0.152 (indole) and 0.188 (naphthyl) in SRIF and small molecule, respectively. The distribution of charge was affected by the indole-NH which had a more localized positive charge and the rest of the atoms were neutral. In contrast, the naphthyl had slight positive charge across both rings because of the high electron withdrawing effect of the sulfonamide. The basic functionalities (NH<sub>3</sub><sup>+</sup>) had equal positive partial charges of 0.560 (SRIF) and 0.581 (small molecule). Similar observations were made for the bound SRIF and small molecule. Specifically, charges for the Trp8 and Lys9 groups in SRIF were 0.026 and 0.967, respectively, whereas in the small molecule charges were 0.031 and 0.851 for the Trp8 and Lys9 mimetic groups. These results show net atomic charges were similar for functional groups that interacted with sst4.</p>
<p>In summary, SRIF-sst4 was the more stable complex. QM/MM refinement alleviated packing at the bottom of the cavity, and improved lengths and angles of non-bonded interactions between ligands and receptor residues. The binding cavities were highly similar except for ECL2. Residues Leu123, Asp126, Gln279, and Leu297 formed interactions with Trp8 and Lys9 moieties of bound agonists. Interactions with Asn199 and Asn293, seen in the experimental structures, were not observed in the refined complexes. Localized positive charge density about Lys9 was deemed essential, and positive partial charges of Trp8 contributed to affinity.</p></section></section><section id="sec13"><h3 class="pmc_sec_title">3.2. Sulfur atom effects on physicochemical properties</h3>
<p>We used matched molecular pairs to explore the effects of sulfur. Chemical structures and properties are given in comparisons 1–8 of <a href="#tab2" class="usa-link">Table 2</a>. We compared low energy ligand conformations using ESP partial charges, solvation free energy (Δ<em>G</em><sub>solv</sub>), acidity constant (p<em>K</em><sub>a</sub>), and <em>A</em> log <em>P</em>. Affinity is affected by solvation/desolvation and we addressed it using Δ<em>G</em><sub>solv</sub>, which measured the free energy difference between the gas and solution phases, where more negative Δ<em>G</em><sub>solv</sub> indicated higher solubility. Given that ionization affects ligand partitioning and permeability, we considered p<em>K</em><sub>a</sub> as well. p<em>K</em><sub>a</sub> Represented the propensity of Lys9 mimetic (imidazole) to become deprotonated in water and was calculated by subtracting the gas-phase deprotonation energy from the summed free energy changes of solvation of the neutral and charged species <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/9279aa7dec9f/d5ob00775e-t1.jpg" loading="lazy" id="ugt1" alt="Inline graphic">. We anticipated sulfur would affect electronic character because of the difference in electronegativity <em>versus</em> carbon or oxygen, reflected by differences in ESPs, Δ<em>G</em><sub>solv</sub>, or p<em>K</em><sub>a</sub>. We also anticipated conformational effects due to increased size of the sulfur atom. Optimized ligands were docked into the refined cryo-EM structures of sst4 to observe whether sulfur affected how ligands interacted with the macromolecule. Sulfur increases solubility in thiourea 7<em>versus</em> urea 2 and results in a greater tendency of imidazole to become deprotonated when compared to urea (comparison 1). 3-Oxy-triazole 230 displays same solubility as 3-thio-triazole 42; however, oxygen's electronegativity increases the propensity of imidazole to deprotonate, contrary to lower electron-withdrawing sulfur (comparison 2). Notably, sulfur-containing ligands in <a href="#tab2" class="usa-link">Table 2</a> display higher solubility unless the benzene carries strong electron-withdrawing substituents such as trifluoromethyl or fluorine (comparisons 6 and 8). On the other hand, sulfur being more electronegative than carbon lowers the p<em>K</em><sub>a</sub>, with the exception of highly electronegative substituents on the benzene such as mesylate and trifluoromethyl. The average summed partial charges of functional groups presumed to interact with Asp126 were affected by the sulfur, but the atoms acting as hydrogen bond donors (Nε2 hydrogen or urea hydrogens) maintained similar positive partial charges between 0.310 and 0.401. Sulfur compounds had uncharged imidazoles at brain pH (p<em>K</em><sub>a</sub> &lt; 7.0), <em>versus</em> urea 2 of comparison 1 with p<em>K</em><sub>a</sub> equal to 7.9, and triazole 207 of comparison 4 with p<em>K</em><sub>a</sub> of 7.5.</p>
<section class="tw xbox font-sm" id="tab2"><h4 class="obj_head">Table 2. Chemical structures and calculated properties of matched molecular pairs.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead>
<tr><th colspan="9" rowspan="1">
<a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-u1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/2df1d7c9eb5b/d5ob00775e-u1.jpg" loading="lazy" id="ugr1" height="212" width="772" alt="graphic file with name d5ob00775e-u1.jpg"></a>
</th></tr>
<tr>
<th rowspan="1" colspan="1">Comparison</th>
<th align="left" rowspan="1" colspan="1">Compound</th>
<th rowspan="1" colspan="1">R</th>
<th rowspan="1" colspan="1">X</th>
<th rowspan="1" colspan="1">Y</th>
<th rowspan="1" colspan="1">Sst4 <em>K</em><sub>i</sub> (nM)</th>
<th rowspan="1" colspan="1">Δ<em>G</em><sub>solv</sub> (kcal mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">Imidazole, p<em>K</em><sub>a</sub>
</th>
<th rowspan="1" colspan="1">
<em>A</em> log <em>P</em>
</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 1</td>
<td rowspan="1" colspan="1">2</td>
<td rowspan="1" colspan="1">O</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">14</td>
<td rowspan="1" colspan="1">−21</td>
<td rowspan="1" colspan="1">7.9</td>
<td rowspan="1" colspan="1">4.5</td>
</tr>
<tr>
<td rowspan="1" colspan="1">7</td>
<td rowspan="1" colspan="1">S</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">70</td>
<td rowspan="1" colspan="1">−26</td>
<td rowspan="1" colspan="1">5.9</td>
<td rowspan="1" colspan="1">4.8</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 2</td>
<td rowspan="1" colspan="1">230</td>
<td rowspan="1" colspan="1">OCH<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">387</td>
<td rowspan="1" colspan="1">−24</td>
<td rowspan="1" colspan="1">5.0</td>
<td rowspan="1" colspan="1">3.5</td>
</tr>
<tr>
<td rowspan="1" colspan="1">42</td>
<td rowspan="1" colspan="1">SCH<sub>2</sub>
</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td align="left" rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">31</td>
<td rowspan="1" colspan="1">−24</td>
<td rowspan="1" colspan="1">6.2</td>
<td rowspan="1" colspan="1">4.0</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 3</td>
<td rowspan="1" colspan="1">180</td>
<td rowspan="1" colspan="1">C</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">1.7</td>
<td rowspan="1" colspan="1">−21</td>
<td rowspan="1" colspan="1">5.8</td>
<td rowspan="1" colspan="1">3.9</td>
</tr>
<tr>
<td rowspan="1" colspan="1">93</td>
<td rowspan="1" colspan="1">S</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">0.9</td>
<td rowspan="1" colspan="1">−23</td>
<td rowspan="1" colspan="1">5.7</td>
<td rowspan="1" colspan="1">4.6</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 4</td>
<td rowspan="1" colspan="1">207</td>
<td rowspan="1" colspan="1">C</td>
<td rowspan="1" colspan="1">3-SO<sub>2</sub>CH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">1.0</td>
<td rowspan="1" colspan="1">−21</td>
<td rowspan="1" colspan="1">7.5</td>
<td rowspan="1" colspan="1">3.4</td>
</tr>
<tr>
<td rowspan="1" colspan="1">95</td>
<td rowspan="1" colspan="1">S</td>
<td rowspan="1" colspan="1">3-SO<sub>2</sub>CH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">0.7</td>
<td rowspan="1" colspan="1">−27</td>
<td rowspan="1" colspan="1">6.5</td>
<td rowspan="1" colspan="1">4.1</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 5</td>
<td rowspan="1" colspan="1">208</td>
<td rowspan="1" colspan="1">C</td>
<td rowspan="1" colspan="1">3-SO<sub>2</sub>CH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">0.7</td>
<td rowspan="1" colspan="1">−27</td>
<td rowspan="1" colspan="1">5.8</td>
<td rowspan="1" colspan="1">3.6</td>
</tr>
<tr>
<td rowspan="1" colspan="1">96</td>
<td rowspan="1" colspan="1">S</td>
<td rowspan="1" colspan="1">3-SO<sub>2</sub>CH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">1.0</td>
<td rowspan="1" colspan="1">−30</td>
<td rowspan="1" colspan="1">6.7</td>
<td rowspan="1" colspan="1">4.3</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 6</td>
<td rowspan="1" colspan="1">196</td>
<td rowspan="1" colspan="1">C</td>
<td rowspan="1" colspan="1">3-CF<sub>3</sub>
</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">0.7</td>
<td rowspan="1" colspan="1">−25</td>
<td rowspan="1" colspan="1">4.1</td>
<td rowspan="1" colspan="1">5.0</td>
</tr>
<tr>
<td rowspan="1" colspan="1">98</td>
<td rowspan="1" colspan="1">S</td>
<td rowspan="1" colspan="1">3-CF<sub>3</sub>
</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">1.3</td>
<td rowspan="1" colspan="1">−21</td>
<td rowspan="1" colspan="1">6.1</td>
<td rowspan="1" colspan="1">5.7</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 7</td>
<td rowspan="1" colspan="1">201</td>
<td rowspan="1" colspan="1">C</td>
<td rowspan="1" colspan="1">3-OCH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">0.6</td>
<td rowspan="1" colspan="1">−26</td>
<td rowspan="1" colspan="1">5.4</td>
<td rowspan="1" colspan="1">5.1</td>
</tr>
<tr>
<td rowspan="1" colspan="1">97</td>
<td rowspan="1" colspan="1">S</td>
<td rowspan="1" colspan="1">3-OCH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">0.6</td>
<td rowspan="1" colspan="1">−27</td>
<td rowspan="1" colspan="1">4.7</td>
<td rowspan="1" colspan="1">4.6</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Comparison 8</td>
<td rowspan="1" colspan="1">184</td>
<td rowspan="1" colspan="1">C</td>
<td rowspan="1" colspan="1">4-F</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">0.9</td>
<td rowspan="1" colspan="1">−27</td>
<td rowspan="1" colspan="1">6.5</td>
<td rowspan="1" colspan="1">4.3</td>
</tr>
<tr>
<td rowspan="1" colspan="1">99</td>
<td rowspan="1" colspan="1">S</td>
<td rowspan="1" colspan="1">4-F</td>
<td rowspan="1" colspan="1">6-F</td>
<td rowspan="1" colspan="1">0.6</td>
<td rowspan="1" colspan="1">−20</td>
<td rowspan="1" colspan="1">5.0</td>
<td rowspan="1" colspan="1">5.0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Docking results showed ligand pairs had similar conformations and orientations within the binding pocket, with the exception of comparison 2 ligands (<a href="#fig4" class="usa-link">Fig. 4</a>). <em>In vacuum</em>, oxy-triazole 230's conformation was extended (<a href="#fig4" class="usa-link">Fig. 4a</a>). Within the docked complex, the imidazole hydrogen-bonded with Asp126 with bond distance equal to 1.7 Å and the indole was positioned near Leu123. The benzyl side-chain was positioned higher and did not form vdW interactions. It should be noted that the binding mode of 230 (<a href="#fig4" class="usa-link">Fig. 4b</a>) was similar to refined SRIF-sst4 with Trp8 moiety hydrogen bonding to Leu123 backbone carbonyl and Lys9 moiety donating a hydrogen to the carboxylate of Asp126. The conformations of thio-triazole 42 were similar <em>in vacuum</em> and within the receptor cavity, but imidazole side-chain was rotated 120° (<a href="#fig4" class="usa-link">Fig. 4c and d</a>). Compound 42's imidazole hydrogen-bonded with the side-chain of Asp126 with bond distance equal to 1.7 Å, the indole NH hydrogen-bonded with the backbone carbonyl of Phe211 at approximately 2.0 Å distance. The benzyl group was positioned lower in the pocket and was involved in vdW interactions with Trp112 and His294.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Fig. 4. Sulfur <em>versus</em> oxygen. (a) QM-optimized geometries with mapped ESPs of compound 230 (3-oxy-1,2,4-triazole). (b) Compound 230 docked into the refined sst4 binding pocket. (c) QM-optimized geometries with mapped ESPs of compound 42 (3-thio-1,2,4-triazole). (d) Compound 42 docked into the refined sst4 binding pocket.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/fb4ad108556b/d5ob00775e-f4.jpg" loading="lazy" height="648" width="700" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>In summary, sulfur compounds maintained similar net atomic charges to SRIF and small molecule agonists, with positive partial charges of functional groups meant to interact with Asp126. The sulfur atom affected solubility for the most part, acid–base character of the Lys9 mimetic, and binding modes of oxy- <em>versus</em> thio-triazole ligands. In contrast, sulfur did not affect how ligands bound sst4 when comparing sulfur <em>versus</em> carbon.</p></section><section id="sec14"><h3 class="pmc_sec_title">3.3. Determinants of affinity for sst4 agonists</h3>
<section id="sec15"><h4 class="pmc_sec_title">3.3.1. Binding modes of sst4 agonists</h4>
<p>In a preceding section, we discussed docking results of molecular pairs carrying sulfur <em>versus</em> carbon or oxygen focusing on effects of sulfur. Herein, we present our analyses of all refined docked solutions in order to identify consistent binding patterns. Most ligands shared the same binding mode as 42 (<a href="#fig4" class="usa-link">Fig. 4d</a>), including triazoles 135, 234, 213, thiotriazole 95, and thioureas 9 and 24 as examples (Fig. S1). These ligands formed a dipole interaction between imidazole or amine and Asp126 carboxylate, while indole hydrogen-bonded with the Phe211 backbone carbonyl oxygen or Gln279. The benzyl pointed towards His294 that formed interactions dependent upon placement and substitutions to the functional group. Ligands that formed similar interactions to SRIF and compound 230 had moderate affinity (<em>K</em><sub>i</sub> &lt; 100 nM). Urea compound 2 maintained the bidentate interaction as previously reported.<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15</a></sup></p>
<p>It is possible for ligands to have more than one binding mode, which we observed for ligands with phenyl or benzyl at positions 3 and 5 of the thiotriazole. In the first binding pattern, compounds 90 (<em>K</em><sub>i</sub> = 2.6 nM) and 233 (<em>K</em><sub>i</sub> = 6 nM) had the thiol or carbonyl oxygen of the sulfone forming a hydrogen bond with Gln279 with bond distances of approximately 2.8 Å, and the Lys9 mimetic hydrogen-bonded with Asp126 with bond distances approximately 1.8 Å and 1.9 Å, respectively. The 4-fluoro-benzyl groups formed edge-to-face π−π interactions with Phe211, and additional vdW and hydrophobic interactions involved Leu123, Phe131, Ile181, Leu200, and Leu297. The sulfur atom of compound 122 (<em>K</em><sub>i</sub> = 243 nM) was positioned slightly lower and pointed down, thus it did not interact with Gln279 but other interactions were maintained (<a href="#fig5" class="usa-link">Fig. 5a</a>). In the second binding pattern, the positions of the benzyl or phenyl groups were flipped. The phenylthiol of compound 90 and the phenylsulfonyl of 233 oriented away from Gln279 and formed hydrophobic and vdW interactions with Ile181, Leu200, and Phe211, and the imidazoles maintained hydrogen bonds with Asp126 (<a href="#fig5" class="usa-link">Fig. 5b</a>). The second binding pattern was not observed for compound 122 which had moderate affinity.</p>
<figure class="fig xbox font-sm" id="fig5"><h5 class="obj_head">Fig. 5. Compounds with multiple binding modes. (a) Binding mode 1 with compounds 90 (orange), 233 (green), and 122 (purple) docked into refined sst4. (b) Binding mode 2 of high affinity compounds 90 (orange) and 233 (green) docked into refined sst4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/474ab1203bc7/d5ob00775e-f5.jpg" loading="lazy" height="337" width="734" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec16"><h4 class="pmc_sec_title">3.3.2. Critical amino acids</h4>
<p>We compared ligands in a pair-wise manner where only one substituent differed. We first examined docking results before and after refinement using RMSD's of ligand atoms and binding pocket residues. Results showed only minor changes of less than approximately 1 Å and 2 Å, respectively, which gave us confidence in our docked complexes. Between pairs, we evaluated amino acid interactions, per residue and absolute sums of complex partial charges, and electronic energies. All ligands maintained the strong dipole interaction with Asp126 and a hydrophobic interaction with Leu200. High affinity ligands (<em>K</em><sub>i</sub> &lt; 10 nM) also hydrogen-bonded with either Gln279 or the backbone carbonyl of Phe211 and formed hydrophobic or vdW interactions with three to five residues where Leu123 and Leu297 remained consistent.</p>
<p>We were able to explain observed affinities for ligand pairs that we could not explain previously using the identified amino acids. Disubstituted triazoles 213 (<em>K</em><sub>i</sub> = 99 nM) and 214 (<em>K</em><sub>i</sub> = 717 nM) differed by a 6-fluoro substitution to the indole which lowered affinity sevenfold. Ligand conformations were similar and both ligands had imidazole forming a hydrogen bond with Asp126, but triazole and indole groups were positioned differently within the binding pocket (<a href="#fig6" class="usa-link">Fig. 6a</a>). Unsubstituted indole hydrogen-bonded with Phe211 backbone carbonyl with bond distance of 2.1 Å, and formed vdW interactions with Phe131 and Phe211 side-chains. The triazole of 214 was positioned lower in the pocket, and 6F-indole was extended up toward the pocket entrance and thus it did not hydrogen bond with amino acid residues. Thiotriazoles 105 (<em>K</em><sub>i</sub> = 945 nM) and 107 (<em>K</em><sub>i</sub> = 959 nM) had quinoline at position 5 of the triazole as the Trp8 mimetic, and either an amine or imidazole as the basic functional group. Within the binding pocket, ligand conformations were similar, but 105 was positioned slightly lower compared with 107 and the benzyl-thiol moieties were positioned perpendicular to one another (<a href="#fig6" class="usa-link">Fig. 6b</a>). These differences did not affect amino acid interactions. Both ligands had the basic functionality hydrogen bonding with Asp126, and forming hydrophobic or vdW interactions with Phe131, Phe275, and Leu297. Quinoline and triazole aromatic rings were within the same plane (flat) and parallel, and the quinoline nitrogen pointed down and away from Gln279 or Phe211.</p>
<figure class="fig xbox font-sm" id="fig6"><h5 class="obj_head">Fig. 6. Amino acid interactions explained differences in observed affinities when ligands were docked into the QM/MM-refined sst4 binding pocket. Superimpositions of (a) di-substituted triazoles 213 (orange; <em>K</em><sub>i</sub> = 99 nM) and 214 (purple; <em>K</em><sub>i</sub> = 717 nM) and (b) 5-(2-quinolyl)-3-thio-1,2,4-triazoles 105 (green; <em>K</em><sub>i</sub> = 945 nM) and 107 (purple; <em>K</em><sub>i</sub> = 959 nM) within the refined sst4 binding pocket.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332849_d5ob00775e-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7ff/12332849/8d49aa015a7c/d5ob00775e-f6.jpg" loading="lazy" height="388" width="795" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec17"><h4 class="pmc_sec_title">3.3.3. Energies and ligand properties that correlated with affinity</h4>
<p>We looked at Gibbs free energies, properties of ligand ESPs<sup><a href="#cit33" class="usa-link" aria-describedby="cit33">33</a></sup> and acidity constants to discern which correlated best with affinity using 35 ligands with QM-optimized geometries (chemical structures in Fig. S1). Differences in binding free energies for representative ligand pairs do not display consistent correlation with affinity (Table S2). A more negative Δ<em>G</em> for a ligand under investigation would be suggestive of higher affinity and favorable binding. Even though there is a direct relationship between Gibbs free energy and p<em>K</em><sub>a</sub>, inherent challenges such as: (1) changes in charge states of both receptor and ligand upon binding, (2) pH dependence of receptor–ligand binding,<sup><a href="#cit34" class="usa-link" aria-describedby="cit34">34</a></sup> (3) one ionizable group on the ligand <em>versus</em> “distant” p<em>K</em><sub>a</sub> changes on the macromolecule, and (4) desolvation penalty are not taken into consideration. As can be seen, most ligands carry an imidazole, yet their varying affinities and preceding sections indicate factors beyond direct electrostatic perturbation are in effect. Finally, multi-electron systems suffer from the complexity of repulsive forces between all electron pairs, in addition to Coulomb repulsion terms which need to be approximated. Properties of ligand ESPs included absolute p<em>K</em><sub>a</sub> of Lys9 mimetics, charge balance, and separation parameters. The balance parameter (<em>υ</em><sub>bal</sub>) quantified the strength of positive and negative electrostatic potentials across the molecular surface by adding the variance of positive and negative charges <em>versus</em> the total charge variance. The maximum value of <em>υ</em><sub>bal</sub> is 0.250 which meant positive or negative regions of a molecule could form equally strong interactions. Inversely, <em>υ</em><sub>bal</sub> of 0.000 indicated only positive or negative regions of a molecule likely form strong interactions. Local polarity (<em>Π</em><sub>pol</sub>) was a measure of charge separation taken as the average deviation from the mean surface potential. Ranges and average values of the properties are given in <a href="#tab3" class="usa-link">Table 3</a>. There is a trend between charge separation and affinity, where <em>Π</em><sub>pol</sub> increased as affinity increased. We interpreted this relationship as higher affinity with increased localized positive charge of either the imidazole or the amine cations. We confirmed this trend as a true positive relationship using external ligands reported by Boehringer Ingelheim<sup><a href="#cit35" class="usa-link" aria-describedby="cit35">35,36</a></sup> that are structurally dissimilar to the triazole series (Fig. S2) and we calculated Pearson's correlation coefficient (<em>r</em>) and Kendall's <em>τ</em> (data not shown). Correlation (<em>r</em>) increased as affinity increased, and <em>Π</em><sub>pol</sub> and affinity were not related when compounds had <em>K</em><sub>i</sub> &gt; 100 nM.</p>
<section class="tw xbox font-sm" id="tab3"><h5 class="obj_head">Table 3. Averages and ranges of calculated properties of sst4 agonists.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th colspan="2" rowspan="1"> </th>
<th rowspan="1" colspan="1">
<em>K</em>
<sub>i</sub> &lt; 2 nM (<em>n</em> = 12)</th>
<th rowspan="1" colspan="1">2 nM ≤ <em>K</em><sub>i</sub> &lt; 20 nM (<em>n</em> = 12)</th>
<th rowspan="1" colspan="1">20 nM ≤ <em>K</em><sub>i</sub> &lt; 100 nM (<em>n</em> = 8)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="2" valign="top" colspan="1">Charge separation (<em>Π</em><sub>pol</sub>)</td>
<td rowspan="1" colspan="1">Range</td>
<td rowspan="1" colspan="1">15–20</td>
<td rowspan="1" colspan="1">15–21</td>
<td rowspan="1" colspan="1">15–22</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Average</td>
<td rowspan="1" colspan="1">18</td>
<td rowspan="1" colspan="1">17</td>
<td rowspan="1" colspan="1">16</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">Charge balance (<em>ν</em><sub>bal</sub>)</td>
<td rowspan="1" colspan="1">Range</td>
<td rowspan="1" colspan="1">0.21–0.25</td>
<td rowspan="1" colspan="1">0.20–0.25</td>
<td rowspan="1" colspan="1">0.21–0.25</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Average</td>
<td rowspan="1" colspan="1">0.23</td>
<td rowspan="1" colspan="1">0.24</td>
<td rowspan="1" colspan="1">0.24</td>
</tr>
<tr>
<td rowspan="2" valign="top" colspan="1">p<em>K</em><sub>a</sub> (Lys9 mimetic)</td>
<td rowspan="1" colspan="1">Range</td>
<td rowspan="1" colspan="1">4.1–7.5</td>
<td rowspan="1" colspan="1">4.0–8.4</td>
<td rowspan="1" colspan="1">4.2–9.1</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Average</td>
<td rowspan="1" colspan="1">6.1</td>
<td rowspan="1" colspan="1">7.5</td>
<td rowspan="1" colspan="1">6.3</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In summary, we identified the same binding modes reported in our previous studies,<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14,20</a></sup> but additional refinement of docked complexes with QM/MM helped clarify which interactions were most important or showed an additional interaction that helped differentiate high from medium affinity binders. The bidentate interaction we had reported previously was slightly altered by QM/MM refinement, which revealed that the indole interaction with Asp126 was replaced by a hydrogen bond with the backbone carbonyl of Leu123, similar to refined SRIF-sst4 complex. Amino acids Gln279 or Phe211, Leu123, and Leu297 differentiated high affinity agonists, whereas electrostatics with Asp126 and hydrophobic interactions with Leu200 were consistent across affinity ranges. The ligand property <em>Π</em><sub>pol</sub> correlated with affinity, while calculated energies did not.</p></section></section><section id="sec18"><h3 class="pmc_sec_title">3.4. Best practices moving forward</h3>
<p>When we initiated this project we expected: (1) more energetically favorable configurations for the SRIF- and small molecule-bound complexes when compared to unbound ones, and decreased cavity volumes for the bound complexes; (2) optimized bonding interactions with the main electrostatics residues, namely Asp126 and Asn199, reported in the refined experimental complexes; (3) sulfur would impact ligand conformations and to an extent binding modes, lipophilicity, and possibly electronic character; (4) higher affinity triazole and thiotriazole derivatives should form tighter complexes <em>versus</em> lower affinity ones. Even though we indeed saw favorable energetics for the refined experimental complexes, Asn199 did not participate in binding interactions after refinement. Sulfur and carbon triazoles displayed similar binding modes, which differed from oxy-triazoles. Sulfur increased solubility and lowered the p<em>K</em><sub>a</sub>, not consistently but contingent upon substitutions on the nearby benzene. Finally, the highest affinity ligands showed additional electrostatics and vdW interactions when compared to those with lower affinity.</p>
<p>Delving deeper beyond our expectations, and contrary to popular belief that post-docking refinement is necessary to achieve accurate docked solutions, our results indicate that extensive conformational sampling does not enhance accuracy. What matters instead are physically feasible starting conformations. Understanding molecular recognition is complex and multi-dimensional. Experimental structures are accurate to a degree, but not absolute representations of reality. Besides packing and unresolved parts of the structure, density mapping is approximated at times. On the other hand, high level theory such as the calculations presented herein promise accuracy for the particular area of the receptor under consideration, however they are not reflective of the actual binding events necessarily. Thus, factoring in the stage of the project, <em>i.e.</em> hit <em>versus</em> lead <em>versus</em> late candidate, against the computational cost of high level methods would be advisable. Besides, a global minimum is not representative of the bioactive ligand conformation and attaining the lowest energy complexes with QM/MM may give the investigator confidence in their approach, but changes occurring beyond the MM region are not considered. Drug discovery enterprises very seldom focus on protonation state changes of receptors and ligands and/or the pH-dependence of ligand–receptor binding. However, these changes impact ionizable groups in macromolecules and small molecules and could affect the nature of the final docked complex. For this system in particular, refinement helped identify critical amino acids but did not offer a correlation to affinity. It is our belief there is a lot to be uncovered at the physiological level for sst4. Given that electrostatics and even vdW required to render affinity are rather limited, conformational rearrangements upon ligand binding likely result in revised interactions of a residue with the ligand and within the receptor partial charges. These changes may extend beyond the immediate vicinity of the cavity. The fact we observed a trend between charge separation and affinity is indicative of the relevance of ionizable groups and local or distant electrostatic scenarios. Lead optimization efforts include optimizing ligand physicochemical properties related to pharmacokinetics, while ensuring potency is not impacted greatly. What we have seen here is that additional ligand characteristics such as <em>Π</em><sub>pol</sub> would be invaluable in ascertaining which lead scaffolds should be advanced in the pipeline during lead optimization campaigns.</p></section></section><section id="sec19"><h2 class="pmc_sec_title">4. Conclusions</h2>
<p>QM/MM refinement of the cryo-EM structures led to more stable bound complexes when compared to unbound ones, while the pocket volume decreased upon binding. Between SRIF and the small molecule, SRIF rendered a more stable complex. Binding interactions with Asn199 and Asn293 seen in the experimental structures were lost and replaced with Gln279. In an effort to identify critical amino acids for agonist binding, we compared bound configurations of several known sst4 ligands and identified Asp126, Gln279, and Leu297 interactions were consistent among refined complexes. To shed light on the effect of sulfur in the compound series under investigation, we considered several physicochemical properties of matched molecular pairs. Results showed sulfur affected solubility, ligand p<em>K</em><sub>a</sub>'s and binding modes. We also aimed at correlating affinity with certain parameters for sst4 agonists. Gibbs free energies, properties of ligand ESPs and acidity constants were considered to discern which correlated best with affinity. There was no correlation between energetic properties of ligands or protein–ligand complexes and affinity, but we identified a strong relationship between <em>Π</em><sub>pol</sub> and affinity for sst4 agonists of different chemical classes. Further, Asp126, Gln279 or Phe211, Leu123, and Leu297 were important for high affinity of the triazole and thio-triazole derivatives considered herein. Moving forward, we advise: (1) less focus be given in post-docking refinement and more on physically viable starting conformations for better docking accuracy; (2) balancing the computational cost of high level theory and the extent to which models reflect reality with the stage of the drug discovery project; (3) deeper understanding of the system, protonation changes, and effects of altered interactions upon binding and beyond the cavity; and (4) consideration of ligand properties in lead optimization beyond the classical physicochemical ones.</p></section><section id="sec20"><h2 class="pmc_sec_title">Conflicts of interest</h2>
<p>There are no conflicts to declare.</p></section><section id="sec21"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="OB-023-D5OB00775E-s001"><div class="caption p"><span>OB-023-D5OB00775E-s001</span></div>
<div class="media p" id="d67e1462"><div class="caption">
<a href="/articles/instance/12332849/bin/OB-023-D5OB00775E-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">OB-023-D5OB00775E-s001.pdf</a><sup> (176.6KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This research was supported in part by the National Institute on Aging of the National Institutes of Health under Award Number R01AG047858 and SIUE's URCA program. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></section><section id="sec22"><h2 class="pmc_sec_title">Data availability</h2>
<p>All relevant data are within the paper and SI. The datasets supporting this article have been uploaded as part of the SI and are presented in Fig. S1. Table S1 provides the side-chain RMSD’s for the SRIF-sst4 and small molecule-sst4 complexes before and after refinement. Table S2 shows the differences in binding free energies for representative ligand pairs, with compounds in Fig. S2 used to validate the correlations between ligand properties and affinity.</p>
<p>Supplementary information is available. See DOI: <a href="https://doi.org/10.1039/d5ob00775e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/d5ob00775e</a>.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="cit1">
<cite>Kumar U. Expression of somatostatin receptor subtypes (SSTR1-5) in Alzheimer's disease brain: an immunohistochemical analysis. Neuroscience. 2005;134(2):525–538. doi: 10.1016/j.neuroscience.2005.04.001.</cite> [<a href="https://doi.org/10.1016/j.neuroscience.2005.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15961235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=Expression%20of%20somatostatin%20receptor%20subtypes%20(SSTR1-5)%20in%20Alzheimer's%20disease%20brain:%20an%20immunohistochemical%20analysis&amp;author=U.%20Kumar&amp;volume=134&amp;issue=2&amp;publication_year=2005&amp;pages=525-538&amp;pmid=15961235&amp;doi=10.1016/j.neuroscience.2005.04.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit2">
<cite>Viollet C. Lepousez G. Loudes C. Videau C. Simon A. Epelbaum J. Somatostatinergic systems in brain: networks and functions. Mol. Cell. Endocrinol. 2008;286(1–2):75–87. doi: 10.1016/j.mce.2007.09.007.</cite> [<a href="https://doi.org/10.1016/j.mce.2007.09.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17997029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Endocrinol.&amp;title=Somatostatinergic%20systems%20in%20brain:%20networks%20and%20functions&amp;author=C.%20Viollet&amp;author=G.%20Lepousez&amp;author=C.%20Loudes&amp;author=C.%20Videau&amp;author=A.%20Simon&amp;volume=286&amp;issue=1%E2%80%932&amp;publication_year=2008&amp;pages=75-87&amp;pmid=17997029&amp;doi=10.1016/j.mce.2007.09.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit3">
<cite>Saito T. Iwata N. Tsubuki S. Takaki Y. Takano J. Huang S. M. Suemoto T. Higuchi M. Saido T. C. Somatostatin regulates brain amyloid beta peptide Abeta42 through modulation of proteolytic degradation. Nat. Med. 2005;11(4):434–439. doi: 10.1038/nm1206.</cite> [<a href="https://doi.org/10.1038/nm1206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15778722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=Somatostatin%20regulates%20brain%20amyloid%20beta%20peptide%20Abeta42%20through%20modulation%20of%20proteolytic%20degradation&amp;author=T.%20Saito&amp;author=N.%20Iwata&amp;author=S.%20Tsubuki&amp;author=Y.%20Takaki&amp;author=J.%20Takano&amp;volume=11&amp;issue=4&amp;publication_year=2005&amp;pages=434-439&amp;pmid=15778722&amp;doi=10.1038/nm1206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit4">
<cite>Brazeau P. Vale W. Burgus R. Ling N. Butcher M. Rivier J. Guillemin R. Hypothalamic polypeptide that inhibits the secretion of immunoreactive pituitary growth hormone. Science. 1973;179(68):77–79. doi: 10.1126/science.179.4068.77.</cite> [<a href="https://doi.org/10.1126/science.179.4068.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4682131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Hypothalamic%20polypeptide%20that%20inhibits%20the%20secretion%20of%20immunoreactive%20pituitary%20growth%20hormone&amp;author=P.%20Brazeau&amp;author=W.%20Vale&amp;author=R.%20Burgus&amp;author=N.%20Ling&amp;author=M.%20Butcher&amp;volume=179&amp;issue=68&amp;publication_year=1973&amp;pages=77-79&amp;pmid=4682131&amp;doi=10.1126/science.179.4068.77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit5">
<cite>Kumar U. Grant M. Somatostatin and somatostatin receptors. Results Probl. Cell Differ. 2010;50:137–184. doi: 10.1007/400_2009_29.</cite> [<a href="https://doi.org/10.1007/400_2009_29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19859675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Results%20Probl.%20Cell%20Differ.&amp;title=Somatostatin%20and%20somatostatin%20receptors&amp;author=U.%20Kumar&amp;author=M.%20Grant&amp;volume=50&amp;publication_year=2010&amp;pages=137-184&amp;pmid=19859675&amp;doi=10.1007/400_2009_29&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit6">
<cite>Moller L. N. Stidsen C. E. Hartmann B. Holst J. J. Somatostatin receptors. Biochim. Biophys. Acta. 2003;1616(1):1–84. doi: 10.1016/S0005-2736(03)00235-9.</cite> [<a href="https://doi.org/10.1016/S0005-2736(03)00235-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14507421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&amp;title=Somatostatin%20receptors&amp;author=L.%20N.%20Moller&amp;author=C.%20E.%20Stidsen&amp;author=B.%20Hartmann&amp;author=J.%20J.%20Holst&amp;volume=1616&amp;issue=1&amp;publication_year=2003&amp;pages=1-84&amp;pmid=14507421&amp;doi=10.1016/S0005-2736(03)00235-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit7">
<cite>Bruno J. F. Xu Y. Song J. Berelowitz M. Molecular cloning and functional expression of a brain-specific somatostatin receptor. Proc. Natl. Acad. Sci. U. S. A. 1992;89(23):11151–11155. doi: 10.1073/pnas.89.23.11151.</cite> [<a href="https://doi.org/10.1073/pnas.89.23.11151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC50507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1360663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Molecular%20cloning%20and%20functional%20expression%20of%20a%20brain-specific%20somatostatin%20receptor&amp;author=J.%20F.%20Bruno&amp;author=Y.%20Xu&amp;author=J.%20Song&amp;author=M.%20Berelowitz&amp;volume=89&amp;issue=23&amp;publication_year=1992&amp;pages=11151-11155&amp;pmid=1360663&amp;doi=10.1073/pnas.89.23.11151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit8">
<cite>Hama E. Saido T. C. Etiology of sporadic Alzheimer's disease: somatostatin, neprilysin, and amyloid beta peptide. Med. Hypotheses. 2005;65(3):498–500. doi: 10.1016/j.mehy.2005.02.045.</cite> [<a href="https://doi.org/10.1016/j.mehy.2005.02.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15921860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Hypotheses&amp;title=Etiology%20of%20sporadic%20Alzheimer's%20disease:%20somatostatin,%20neprilysin,%20and%20amyloid%20beta%20peptide&amp;author=E.%20Hama&amp;author=T.%20C.%20Saido&amp;volume=65&amp;issue=3&amp;publication_year=2005&amp;pages=498-500&amp;pmid=15921860&amp;doi=10.1016/j.mehy.2005.02.045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit9">
<cite>Lewis I. Bauer W. Albert R. Chandramouli N. Pless J. Weckbecker G. Bruns C. A novel somatostatin mimic with broad somatotropin release inhibitory factor receptor binding and superior therapeutic potential. J. Med. Chem. 2003;46(12):2334–2344. doi: 10.1021/jm021093t.</cite> [<a href="https://doi.org/10.1021/jm021093t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12773038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=A%20novel%20somatostatin%20mimic%20with%20broad%20somatotropin%20release%20inhibitory%20factor%20receptor%20binding%20and%20superior%20therapeutic%20potential&amp;author=I.%20Lewis&amp;author=W.%20Bauer&amp;author=R.%20Albert&amp;author=N.%20Chandramouli&amp;author=J.%20Pless&amp;volume=46&amp;issue=12&amp;publication_year=2003&amp;pages=2334-2344&amp;pmid=12773038&amp;doi=10.1021/jm021093t&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit10">
<cite>Crider A. M. Witt K. A. Somatostatin sst4 ligands: chemistry and pharmacology. Mini-Rev. Med. Chem. 2007;7(3):213–220. doi: 10.2174/138955707780059880.</cite> [<a href="https://doi.org/10.2174/138955707780059880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17346214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mini-Rev.%20Med.%20Chem.&amp;title=Somatostatin%20sst4%20ligands:%20chemistry%20and%20pharmacology&amp;author=A.%20M.%20Crider&amp;author=K.%20A.%20Witt&amp;volume=7&amp;issue=3&amp;publication_year=2007&amp;pages=213-220&amp;pmid=17346214&amp;doi=10.2174/138955707780059880&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit11">
<cite>Zhao W. Han S. Qiu N. 
et al., Structural insights into ligand recognition and selectivity of somatostatin receptors. Cell Res. 2022;32:761–772. doi: 10.1038/s41422-022-00679-x.</cite> [<a href="https://doi.org/10.1038/s41422-022-00679-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9343605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35739238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=et%20al.,%20Structural%20insights%20into%20ligand%20recognition%20and%20selectivity%20of%20somatostatin%20receptors&amp;author=W.%20Zhao&amp;author=S.%20Han&amp;author=N.%20Qiu&amp;volume=32&amp;publication_year=2022&amp;pages=761-772&amp;pmid=35739238&amp;doi=10.1038/s41422-022-00679-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit12">
<cite>Ankersen M. Crider A. M. Liu S. Ho B. Andersen H. S. Stidsen C. E. Discovery of a novel non-peptide somatostatin agonist with SST4 selectivity. J. Am. Chem. Soc. 1998;120(7):1368–1373. doi: 10.1021/ja973325x.</cite> [<a href="https://doi.org/10.1021/ja973325x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Discovery%20of%20a%20novel%20non-peptide%20somatostatin%20agonist%20with%20SST4%20selectivity&amp;author=M.%20Ankersen&amp;author=A.%20M.%20Crider&amp;author=S.%20Liu&amp;author=B.%20Ho&amp;author=H.%20S.%20Andersen&amp;volume=120&amp;issue=7&amp;publication_year=1998&amp;pages=1368-1373&amp;doi=10.1021/ja973325x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit13">
<cite>Daryaei I. Sandoval K. Witt K. Kontoyianni M. Michael Crider A. Discovery of a 3,4,5-trisubstituted-1,2,4-triazole agonist with high affinity and selectivity at the somatostatin subtype-4 (sst(4)) receptor. MedChemComm. 2018;9(12):2083–2090. doi: 10.1039/C8MD00388B.</cite> [<a href="https://doi.org/10.1039/C8MD00388B" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6336083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30746066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MedChemComm&amp;title=Discovery%20of%20a%203,4,5-trisubstituted-1,2,4-triazole%20agonist%20with%20high%20affinity%20and%20selectivity%20at%20the%20somatostatin%20subtype-4%20(sst(4))%20receptor&amp;author=I.%20Daryaei&amp;author=K.%20Sandoval&amp;author=K.%20Witt&amp;author=M.%20Kontoyianni&amp;author=A.%20Michael%20Crider&amp;volume=9&amp;issue=12&amp;publication_year=2018&amp;pages=2083-2090&amp;pmid=30746066&amp;doi=10.1039/C8MD00388B&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit14">
<cite>Neumann W. L. Sandoval K. E. Mobayen S. Minaeian M. Kukielski S. G. Srabony K. N. Frare R. Slater O. Farr S. A. Niehoff M. L. Hospital A. Kontoyianni M. Crider A. M. Witt K. A. Synthesis and structure-activity relationships of 3,4,5-trisubstituted-1,2,4-triazoles: high affinity and selective somatostatin receptor-4 agonists for Alzheimer's disease treatment. RSC Med. Chem. 2021;12(8):1352–1365. doi: 10.1039/D1MD00044F.</cite> [<a href="https://doi.org/10.1039/D1MD00044F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8372316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34458738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Med.%20Chem.&amp;title=Synthesis%20and%20structure-activity%20relationships%20of%203,4,5-trisubstituted-1,2,4-triazoles:%20high%20affinity%20and%20selective%20somatostatin%20receptor-4%20agonists%20for%20Alzheimer's%20disease%20treatment&amp;author=W.%20L.%20Neumann&amp;author=K.%20E.%20Sandoval&amp;author=S.%20Mobayen&amp;author=M.%20Minaeian&amp;author=S.%20G.%20Kukielski&amp;volume=12&amp;issue=8&amp;publication_year=2021&amp;pages=1352-1365&amp;pmid=34458738&amp;doi=10.1039/D1MD00044F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit15">
<cite>Liu Z. Crider A. M. Ansbro D. Hayes C. Kontoyianni M. A structure-based approach to understanding somatostatin receptor-4 agonism (sst4) J. Chem. Inf. Model. 2012;52(1):171–186. doi: 10.1021/ci200375j.</cite> [<a href="https://doi.org/10.1021/ci200375j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22148589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=A%20structure-based%20approach%20to%20understanding%20somatostatin%20receptor-4%20agonism%20(sst4)&amp;author=Z.%20Liu&amp;author=A.%20M.%20Crider&amp;author=D.%20Ansbro&amp;author=C.%20Hayes&amp;author=M.%20Kontoyianni&amp;volume=52&amp;issue=1&amp;publication_year=2012&amp;pages=171-186&amp;pmid=22148589&amp;doi=10.1021/ci200375j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit16">
<cite>Slater O. Kontoyianni M. A computational study of somatostatin subtype-4 agonist binding. SN Appl. Sci. 2022;4:140. doi: 10.1007/s42452-022-04968-3.</cite> [<a href="https://doi.org/10.1007/s42452-022-04968-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=SN%20Appl.%20Sci.&amp;title=A%20computational%20study%20of%20somatostatin%20subtype-4%20agonist%20binding&amp;author=O.%20Slater&amp;author=M.%20Kontoyianni&amp;volume=4&amp;publication_year=2022&amp;pages=140&amp;doi=10.1007/s42452-022-04968-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit17">
<cite>Rasmussen S. G. Choi H. J. Fung J. J. Pardon E. Casarosa P. Chae P. S. Devree B. T. Rosenbaum D. M. Thian F. S. Kobilka T. S. Schnapp A. Konetzki I. Sunahara R. K. Gellman S. H. Pautsch A. Steyaert J. Weis W. I. Kobilka B. K. Structure of a nanobody-stabilized active state of the beta(2) adrenoceptor. Nature. 2011;469(7329):175–180. doi: 10.1038/nature09648.</cite> [<a href="https://doi.org/10.1038/nature09648" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3058308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21228869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20a%20nanobody-stabilized%20active%20state%20of%20the%20beta(2)%20adrenoceptor&amp;author=S.%20G.%20Rasmussen&amp;author=H.%20J.%20Choi&amp;author=J.%20J.%20Fung&amp;author=E.%20Pardon&amp;author=P.%20Casarosa&amp;volume=469&amp;issue=7329&amp;publication_year=2011&amp;pages=175-180&amp;pmid=21228869&amp;doi=10.1038/nature09648&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit18">
<cite>Huang W. Manglik A. Venkatakrishnan A. J. Laeremans T. Feinberg E. N. Sanborn A. L. Kato H. E. Livingston K. E. Thorsen T. S. Kling R. C. Granier S. Gmeiner P. Husbands S. M. Traynor J. R. Weis W. I. Steyaert J. Dror R. O. Kobilka B. K. Structural insights into micro-opioid receptor activation. Nature. 2015;524(7565):315–321. doi: 10.1038/nature14886.</cite> [<a href="https://doi.org/10.1038/nature14886" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4639397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26245379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural%20insights%20into%20micro-opioid%20receptor%20activation&amp;author=W.%20Huang&amp;author=A.%20Manglik&amp;author=A.%20J.%20Venkatakrishnan&amp;author=T.%20Laeremans&amp;author=E.%20N.%20Feinberg&amp;volume=524&amp;issue=7565&amp;publication_year=2015&amp;pages=315-321&amp;pmid=26245379&amp;doi=10.1038/nature14886&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit19">
<cite>Negi A. Zhou J. Sweeney S. Murphy P. V. Ligand design for somatostatin receptor isoforms 4 and 5. Eur. J. Med. Chem. 2019;163:148–159. doi: 10.1016/j.ejmech.2018.11.030.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2018.11.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30503939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Ligand%20design%20for%20somatostatin%20receptor%20isoforms%204%20and%205&amp;author=A.%20Negi&amp;author=J.%20Zhou&amp;author=S.%20Sweeney&amp;author=P.%20V.%20Murphy&amp;volume=163&amp;publication_year=2019&amp;pages=148-159&amp;pmid=30503939&amp;doi=10.1016/j.ejmech.2018.11.030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit20">
<cite>Crider A. M. Hospital A. Sandoval K. E. Neumann W. L. Kukielski S. Garic L. Ingold K. Dunahoo M. Srabony K. N. Frare R. Slater O. Peel N. Kontoyianni M. Witt K. A. 3-Thio-3,4,5-trisubstituted-1,2,4-triazoles: high affinity somatostatin receptor-4 agonist synthesis and structure-activity relationships. RSC Med. Chem. 2025;16:945–960. doi: 10.1039/D4MD00597J.</cite> [<a href="https://doi.org/10.1039/D4MD00597J" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11660163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39713483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Med.%20Chem.&amp;title=3-Thio-3,4,5-trisubstituted-1,2,4-triazoles:%20high%20affinity%20somatostatin%20receptor-4%20agonist%20synthesis%20and%20structure-activity%20relationships&amp;author=A.%20M.%20Crider&amp;author=A.%20Hospital&amp;author=K.%20E.%20Sandoval&amp;author=W.%20L.%20Neumann&amp;author=S.%20Kukielski&amp;volume=16&amp;publication_year=2025&amp;pages=945-960&amp;pmid=39713483&amp;doi=10.1039/D4MD00597J&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit21">
<cite>Sondergaard C. R. Olsson M. H. Rostkowski M. Jensen J. H. Improved Treatment of Ligands and Coupling Effects in Empirical Calculation and Rationalization of pKa Values. J. Chem. Theory Comput. 2011;7(7):2284–2295. doi: 10.1021/ct200133y.</cite> [<a href="https://doi.org/10.1021/ct200133y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26606496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Improved%20Treatment%20of%20Ligands%20and%20Coupling%20Effects%20in%20Empirical%20Calculation%20and%20Rationalization%20of%20pKa%20Values&amp;author=C.%20R.%20Sondergaard&amp;author=M.%20H.%20Olsson&amp;author=M.%20Rostkowski&amp;author=J.%20H.%20Jensen&amp;volume=7&amp;issue=7&amp;publication_year=2011&amp;pages=2284-2295&amp;pmid=26606496&amp;doi=10.1021/ct200133y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit22">
<cite>Groom C. R. Bruno I. J. Lightfoot M. P. Ward S. C. The Cambridge Structural Database. Acta Crystallogr., Sect. B: Struct. Sci., Cryst. Eng. Mater. 2016;72(Pt 2):171–179. doi: 10.1107/S2052520616003954.</cite> [<a href="https://doi.org/10.1107/S2052520616003954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4822653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27048719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.,%20Sect.%20B:%20Struct.%20Sci.,%20Cryst.%20Eng.%20Mater.&amp;title=The%20Cambridge%20Structural%20Database&amp;author=C.%20R.%20Groom&amp;author=I.%20J.%20Bruno&amp;author=M.%20P.%20Lightfoot&amp;author=S.%20C.%20Ward&amp;volume=72&amp;issue=Pt%202&amp;publication_year=2016&amp;pages=171-179&amp;pmid=27048719&amp;doi=10.1107/S2052520616003954&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit23">
<cite>Ryckaert J.-P. Ciccotti G. Berendsen H. J. C. Numerical integration of cartesian equations of motion of a system with constraints - molecular dynamics of N-alkanes. J. Comput. Phys. 1977;23(3):327–341. doi: 10.1016/0021-9991(77)90098-5.</cite> [<a href="https://doi.org/10.1016/0021-9991(77)90098-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Phys.&amp;title=Numerical%20integration%20of%20cartesian%20equations%20of%20motion%20of%20a%20system%20with%20constraints%20-%20molecular%20dynamics%20of%20N-alkanes&amp;author=J.-P.%20Ryckaert&amp;author=G.%20Ciccotti&amp;author=H.%20J.%20C.%20Berendsen&amp;volume=23&amp;issue=3&amp;publication_year=1977&amp;pages=327-341&amp;doi=10.1016/0021-9991(77)90098-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit24">
<cite>Murphy R. B. Philipp D. M. Friesner R. A. A Mixed Quantum Mechanics/Molecular Mechanics (QM/MM) Method for Large-Scale Modeling of Chemistry in Protein Environments. J. Comput. Chem. 2000;21:1442–1457. doi: 10.1002/1096-987X(200012)21:16&amp;#x0003c;1442::AID-JCC3&amp;#x0003e;3.0.CO;2-O.</cite> [<a href="https://doi.org/10.1002/1096-987X(200012)21:16&amp;#x0003c;1442::AID-JCC3&amp;#x0003e;3.0.CO;2-O" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=A%20Mixed%20Quantum%20Mechanics/Molecular%20Mechanics%20(QM/MM)%20Method%20for%20Large-Scale%20Modeling%20of%20Chemistry%20in%20Protein%20Environments&amp;author=R.%20B.%20Murphy&amp;author=D.%20M.%20Philipp&amp;author=R.%20A.%20Friesner&amp;volume=21&amp;publication_year=2000&amp;pages=1442-1457&amp;doi=10.1002/1096-987X(200012)21:16&amp;#x0003c;1442::AID-JCC3&amp;#x0003e;3.0.CO;2-O&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit25">
<cite>Philipp D. M. Friesner R. A. Mixed ab initio QM/MM modeling using frozen orbitals and tests with alanine dipeptide and tetrapeptide. J. Comput. Chem. 1999;20:1468–1494. doi: 10.1002/(SICI)1096-987X(19991115)20:14&amp;#x0003c;1468::AID-JCC2&amp;#x0003e;3.0.CO;2-0.</cite> [<a href="https://doi.org/10.1002/(SICI)1096-987X(19991115)20:14&amp;#x0003c;1468::AID-JCC2&amp;#x0003e;3.0.CO;2-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=Mixed%20ab%20initio%20QM/MM%20modeling%20using%20frozen%20orbitals%20and%20tests%20with%20alanine%20dipeptide%20and%20tetrapeptide&amp;author=D.%20M.%20Philipp&amp;author=R.%20A.%20Friesner&amp;volume=20&amp;publication_year=1999&amp;pages=1468-1494&amp;doi=10.1002/(SICI)1096-987X(19991115)20:14&amp;#x0003c;1468::AID-JCC2&amp;#x0003e;3.0.CO;2-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit26">
<cite>Stewart J. J. MOPAC: a semiempirical molecular orbital program. J. Comput. Aided Mol. Des. 1990;4(1):1–105. doi: 10.1007/BF00128336.</cite> [<a href="https://doi.org/10.1007/BF00128336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2197373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Aided%20Mol.%20Des.&amp;title=MOPAC:%20a%20semiempirical%20molecular%20orbital%20program&amp;author=J.%20J.%20Stewart&amp;volume=4&amp;issue=1&amp;publication_year=1990&amp;pages=1-105&amp;pmid=2197373&amp;doi=10.1007/BF00128336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit27">
<cite>
Stewart J. J. P., MOPAC2016, Stewart Computational Chemistry, Colorado Springs, 2016</cite> [<a href="https://scholar.google.com/scholar_lookup?title=MOPAC2016&amp;author=J.%20J.%20P.%20Stewart&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit28">
<cite>Bochevarov A. D. Harder E. Hughes T. F. Greenwood J. R. Braden D. A. Philipp D. M. Rinaldo D. Halls M. D. Zhang J. Friesner R. A. Jaguar: A High-Performance Quantum Chemistry Software Program with Strengths in Life and Materials Sciences. Int. J. Quantum Chem. 2013;113:2110–2142. doi: 10.1002/qua.24481.</cite> [<a href="https://doi.org/10.1002/qua.24481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Quantum%20Chem.&amp;title=Jaguar:%20A%20High-Performance%20Quantum%20Chemistry%20Software%20Program%20with%20Strengths%20in%20Life%20and%20Materials%20Sciences&amp;author=A.%20D.%20Bochevarov&amp;author=E.%20Harder&amp;author=T.%20F.%20Hughes&amp;author=J.%20R.%20Greenwood&amp;author=D.%20A.%20Braden&amp;volume=113&amp;publication_year=2013&amp;pages=2110-2142&amp;doi=10.1002/qua.24481&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit29">
<cite>Anoop A. Ranganathan S. Dhaked B. D. Jha N. N. Pratihar S. Ghosh S. Sahay S. Kumar S. Das S. Kombrabail M. Agarwal K. Jacob R. S. Singru P. Bhaumik P. Padinhateeri R. Kumar A. Maji S. K. Elucidating the role of disulfide bond on amyloid formation and fibril reversibility of somatostatin-14: relevance to its storage and secretion. J. Biol. Chem. 2014;289(24):16884–16903. doi: 10.1074/jbc.M114.548354.</cite> [<a href="https://doi.org/10.1074/jbc.M114.548354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4059132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24782311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Elucidating%20the%20role%20of%20disulfide%20bond%20on%20amyloid%20formation%20and%20fibril%20reversibility%20of%20somatostatin-14:%20relevance%20to%20its%20storage%20and%20secretion&amp;author=A.%20Anoop&amp;author=S.%20Ranganathan&amp;author=B.%20D.%20Dhaked&amp;author=N.%20N.%20Jha&amp;author=S.%20Pratihar&amp;volume=289&amp;issue=24&amp;publication_year=2014&amp;pages=16884-16903&amp;pmid=24782311&amp;doi=10.1074/jbc.M114.548354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit30">
<cite>Fogolari F. Brigo A. Molinari H. The Poisson-Boltzmann equation for biomolecular electrostatics: a tool for structural biology. J. Mol. Recognit. 2002;15(6):377–392. doi: 10.1002/jmr.577.</cite> [<a href="https://doi.org/10.1002/jmr.577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12501158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Recognit.&amp;title=The%20Poisson-Boltzmann%20equation%20for%20biomolecular%20electrostatics:%20a%20tool%20for%20structural%20biology&amp;author=F.%20Fogolari&amp;author=A.%20Brigo&amp;author=H.%20Molinari&amp;volume=15&amp;issue=6&amp;publication_year=2002&amp;pages=377-392&amp;pmid=12501158&amp;doi=10.1002/jmr.577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit31">
<cite>Klicić J. J. Friesner R. A. Liu S.-Y. Guida W. C. Accurate prediction of acidity constants in aqueous solution via density functional theory and self-consistent reaction field methods. J. Phys. Chem. A. 2002;106:1327–1335. doi: 10.1021/jp012533f.</cite> [<a href="https://doi.org/10.1021/jp012533f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20A&amp;title=Accurate%20prediction%20of%20acidity%20constants%20in%20aqueous%20solution%20via%20density%20functional%20theory%20and%20self-consistent%20reaction%20field%20methods&amp;author=J.%20J.%20Klici%C4%87&amp;author=R.%20A.%20Friesner&amp;author=S.-Y.%20Liu&amp;author=W.%20C.%20Guida&amp;volume=106&amp;publication_year=2002&amp;pages=1327-1335&amp;doi=10.1021/jp012533f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit32">
<cite>Bochevarov A. D. Watson M. A. Greenwood J. R. Philipp D. M. Multiconformation, Density Functional Theory-Based pK(a) Prediction in Application to Large, Flexible Organic Molecules with Diverse Functional Groups. J. Chem. Theory Comput. 2016;12(12):6001–6019. doi: 10.1021/acs.jctc.6b00805.</cite> [<a href="https://doi.org/10.1021/acs.jctc.6b00805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27951674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Multiconformation,%20Density%20Functional%20Theory-Based%20pK(a)%20Prediction%20in%20Application%20to%20Large,%20Flexible%20Organic%20Molecules%20with%20Diverse%20Functional%20Groups&amp;author=A.%20D.%20Bochevarov&amp;author=M.%20A.%20Watson&amp;author=J.%20R.%20Greenwood&amp;author=D.%20M.%20Philipp&amp;volume=12&amp;issue=12&amp;publication_year=2016&amp;pages=6001-6019&amp;pmid=27951674&amp;doi=10.1021/acs.jctc.6b00805&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit33">
<cite>Politzer P. Murray J. S. Computational prediction of condensed phase properties from statistical characterization of molecular surface electrostatic potentials. Fluid Phase Equilib. 2001;185:129–137. doi: 10.1016/S0378-3812(01)00463-0.</cite> [<a href="https://doi.org/10.1016/S0378-3812(01)00463-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fluid%20Phase%20Equilib.&amp;title=Computational%20prediction%20of%20condensed%20phase%20properties%20from%20statistical%20characterization%20of%20molecular%20surface%20electrostatic%20potentials&amp;author=P.%20Politzer&amp;author=J.%20S.%20Murray&amp;volume=185&amp;publication_year=2001&amp;pages=129-137&amp;doi=10.1016/S0378-3812(01)00463-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit34">
<cite>Onufriev A. V. Alexov E. Protonation and pK changes in protein-ligand binding. Q. Rev. Biophys. 2013;46(2):181–209. doi: 10.1017/S0033583513000024.</cite> [<a href="https://doi.org/10.1017/S0033583513000024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4437766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23889892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Q.%20Rev.%20Biophys.&amp;title=Protonation%20and%20pK%20changes%20in%20protein-ligand%20binding&amp;author=A.%20V.%20Onufriev&amp;author=E.%20Alexov&amp;volume=46&amp;issue=2&amp;publication_year=2013&amp;pages=181-209&amp;pmid=23889892&amp;doi=10.1017/S0033583513000024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit35"><cite>
Mazzaferro R., Ferrara M., Giovannini R., Lingard I. and Rudolf K., Morpholine and 1,4-oxazepane amides as somatostatin receptor subtype 4 (sstr4) agonists, WO2016075240A1, 2016</cite></li>
<li id="cit36"><cite>
Giovannini R., Cui Y., Doods H., Ferrara M., Just S., Kuelzer R., Lingard I., Mazzaferro R. and Rudolf K., New somatostatin receptor subtype 4 (sstr4) agonists, WO2014184275A1, 2014</cite></li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>OB-023-D5OB00775E-s001</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332849/bin/OB-023-D5OB00775E-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">OB-023-D5OB00775E-s001.pdf</a><sup> (176.6KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All relevant data are within the paper and SI. The datasets supporting this article have been uploaded as part of the SI and are presented in Fig. S1. Table S1 provides the side-chain RMSD’s for the SRIF-sst4 and small molecule-sst4 complexes before and after refinement. Table S2 shows the differences in binding free energies for representative ligand pairs, with compounds in Fig. S2 used to validate the correlations between ligand properties and affinity.</p>
<p>Supplementary information is available. See DOI: <a href="https://doi.org/10.1039/d5ob00775e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/d5ob00775e</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Organic &amp; Biomolecular Chemistry are provided here courtesy of <strong>Royal Society of Chemistry</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1039/d5ob00775e"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332849/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332849/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332849%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332849/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332849/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332849/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40778438/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332849/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40778438/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332849/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332849/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wvCTmhTWAozRCdbEF3HDOs7D0ae08QKzxbmnawzzduyv8NjfcBBtDNuFxTqOjP9G">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
