
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            AlphaFold modeling uncovers global structural features of class I and class II fungal hydrophobins - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152543E98AF2EFB30543E90022EF298C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375992/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="AlphaFold modeling uncovers global structural features of class I and class II fungal hydrophobins">
<meta name="citation_author" content="Li‐Yen Yang">
<meta name="citation_author_institution" content="School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author" content="Daniel J Hicks">
<meta name="citation_author_institution" content="School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author" content="Paul S Russo">
<meta name="citation_author_institution" content="School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author_institution" content="School of Materials Science and Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author" content="Andrew C McShan">
<meta name="citation_author_institution" content="School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70279">
<meta name="citation_doi" content="10.1002/pro.70279">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375992/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375992/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375992/pdf/PRO-34-e70279.pdf">
<meta name="description" content="Hydrophobins are a family of small fungal proteins that self‐assemble at hydrophobic–hydrophilic interfaces. Hydrophobins not only play crucial roles in filamentous fungal growth and development but also have attracted substantial attention due to ...">
<meta name="og:title" content="AlphaFold modeling uncovers global structural features of class I and class II fungal hydrophobins">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Hydrophobins are a family of small fungal proteins that self‐assemble at hydrophobic–hydrophilic interfaces. Hydrophobins not only play crucial roles in filamentous fungal growth and development but also have attracted substantial attention due to ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375992/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375992">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70279"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70279.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375992%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375992/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375992/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375992/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 25;34(9):e70279. doi: <a href="https://doi.org/10.1002/pro.70279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70279</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
AlphaFold modeling uncovers global structural features of class I and class II fungal hydrophobins</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20LY%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Li‐Yen Yang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Li‐Yen Yang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20LY%22%5BAuthor%5D" class="usa-link"><span class="name western">Li‐Yen Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hicks%20DJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Daniel J Hicks</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Daniel J Hicks</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hicks%20DJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel J Hicks</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Russo%20PS%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Paul S Russo</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Paul S Russo</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>School of Materials Science and Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Russo%20PS%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul S Russo</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McShan%20AC%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Andrew C McShan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Andrew C McShan</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McShan%20AC%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew C McShan</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70279-aff-0001">
<sup>
<sup>1</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div id="pro70279-aff-0002">
<sup>
<sup>2</sup>
</sup>School of Materials Science and Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Andrew C. McShan, School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, GA 30332, USA. 
Email: <span>andrew.mcshan@chemistry.gatech.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 19; Received 2025 Feb 13; Accepted 2025 Aug 11; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375992</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Hydrophobins are a family of small fungal proteins that self‐assemble at hydrophobic–hydrophilic interfaces. Hydrophobins not only play crucial roles in filamentous fungal growth and development but also have attracted substantial attention due to their unique material properties. Structural characterization of class I and class II hydrophobins to date has been limited to a handful of proteins. While machine‐learning‐based structure prediction methods have the potential to exponentially expand our ability to define global structure–function relationships of biomolecules, they have not yet been extensively applied to hydrophobins. Here, we apply a suite of bioinformatics tools including Rosetta, AlphaFold, FoldMason, and Foldseek toward analysis, modeling, classification, and global comparison of class I and class II hydrophobins. We first probe the structural and energetic features of experimental class I and class II structures available in the Protein Data Bank. Using previously solved X‐ray and NMR structures, we benchmark the ability of AlphaFold to predict class I and class II hydrophobin folds. We explore the physicochemical properties of more than 7,000 class I and class II hydrophobins in the UniProt database. Then, using AlphaFold models, we classify the structural universe of all known class I and class II hydrophobins into six distinct clades. We also uncover putative non‐canonical features of hydrophobins, including extended N‐terminal tails, five disulfide bonds, polyhydrophobins, and non‐hydrophobin proteins containing hydrophobin‐like folds. Finally, we examine the ability of AlphaFold and Chai‐1 to model hydrophobin membrane binding, conformational changes, and self‐assembly of class I rodlets and class II meshes. Together, our results highlight that AlphaFold not only accurately models and enables the global comparison of features within the hydrophobin protein family but also uncovers new properties that can be further evaluated with experimentation.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> alphafold, computational biology, foldmason, foldseek, hydrophobins, protein structure, rosetta</p></section></section><section id="pro70279-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Hydrophobins are small proteins (typically &lt;15 kDa) that play roles in filamentous fungal growth and development (Ball, Kwan, &amp; Sunde, <a href="#pro70279-bib-0010" class="usa-link" aria-describedby="pro70279-bib-0010">2020</a>; Wessels, <a href="#pro70279-bib-0138" class="usa-link" aria-describedby="pro70279-bib-0138"><sup>1997</sup></a>; Wessels, de Vries, Asgeirsdottir, &amp; Schuren, <a href="#pro70279-bib-0137" class="usa-link" aria-describedby="pro70279-bib-0137"><sup>1991</sup></a>; Wessels, de Vries, Asgeirsdóttir, &amp; Springer, <a href="#pro70279-bib-0139" class="usa-link" aria-describedby="pro70279-bib-0139"><sup>1991</sup></a>). Hydrophobins have been referred to as nature's Janus particles since they are amphipathic molecules with opposing hydrophobic and hydrophilic surfaces (Karadkar et al., <a href="#pro70279-bib-0056" class="usa-link" aria-describedby="pro70279-bib-0056">2023</a>). The amphipathic properties of hydrophobins underpin their ability to transition from soluble to insoluble forms that self‐assemble at hydrophobic–hydrophilic interfaces (i.e., air‐water, air‐oil, air‐membrane, water‐solid, and fungal‐host) (Cheung, <a href="#pro70279-bib-0024" class="usa-link" aria-describedby="pro70279-bib-0024">2012</a>; Szilvay et al., <a href="#pro70279-bib-0115" class="usa-link" aria-describedby="pro70279-bib-0115"><sup>2007</sup></a>; Wang et al., <a href="#pro70279-bib-0134" class="usa-link" aria-describedby="pro70279-bib-0134"><sup>2005</sup></a>). Hydrophobins are involved in a wide range of cellular processes in fungi, such as the formation of aerial hyphae, spores, and fruiting bodies (Ren et al., <a href="#pro70279-bib-0103" class="usa-link" aria-describedby="pro70279-bib-0103">2013</a>; van Wetter et al., <a href="#pro70279-bib-0126" class="usa-link" aria-describedby="pro70279-bib-0126"><sup>2000</sup></a>; Wessels, <a href="#pro70279-bib-0138" class="usa-link" aria-describedby="pro70279-bib-0138"><sup>1997</sup></a>), the attachment of hyphae to hydrophobic surfaces (Wösten et al., <a href="#pro70279-bib-0146" class="usa-link" aria-describedby="pro70279-bib-0146">1994</a>), and evasion of host immune responses (Aimanianda et al., <a href="#pro70279-bib-0003" class="usa-link" aria-describedby="pro70279-bib-0003">2009</a>). For example, soluble hydrophobins secreted at the tip of the submerged hyphae diffuse into the aqueous environment and self‐assemble at the medium‐air interface, which results in a decrease in the water surface tension that allows hyphae to grow in the air (Cai et al., <a href="#pro70279-bib-0020" class="usa-link" aria-describedby="pro70279-bib-0020">2021</a>; Wösten et al., <a href="#pro70279-bib-0147" class="usa-link" aria-describedby="pro70279-bib-0147"><sup>1999</sup></a>). Due to their unique material‐like properties, hydrophobins also have a variety of industrial and medical applications, such as surface functionalization, altering wettability, amphipathic solid films, deicing, emulsification, antifouling, foaming, shape‐shifting materials, and aerogel coating (Khalesi et al., <a href="#pro70279-bib-0061" class="usa-link" aria-describedby="pro70279-bib-0061">2015</a>; Rojas‐Osnaya et al., <a href="#pro70279-bib-0107" class="usa-link" aria-describedby="pro70279-bib-0107"><sup>2024</sup></a>; Stanzione et al., <a href="#pro70279-bib-0112" class="usa-link" aria-describedby="pro70279-bib-0112"><sup>2022</sup></a>; Szilvay et al., <a href="#pro70279-bib-0115" class="usa-link" aria-describedby="pro70279-bib-0115"><sup>2007</sup></a>; Zhang et al., <a href="#pro70279-bib-0152" class="usa-link" aria-describedby="pro70279-bib-0152"><sup>2019</sup></a>). Example applications include hydrophobin‐coated nanoparticles for drug delivery (Valo et al., <a href="#pro70279-bib-0123" class="usa-link" aria-describedby="pro70279-bib-0123">2010</a>; Zhao et al., <a href="#pro70279-bib-0154" class="usa-link" aria-describedby="pro70279-bib-0154"><sup>2016</sup></a>) and the modification of plastic materials for biodegradation (Khatua et al., <a href="#pro70279-bib-0062" class="usa-link" aria-describedby="pro70279-bib-0062">2024</a>; Piscitelli et al., <a href="#pro70279-bib-0099" class="usa-link" aria-describedby="pro70279-bib-0099"><sup>2017</sup></a>; Stanzione et al., <a href="#pro70279-bib-0112" class="usa-link" aria-describedby="pro70279-bib-0112"><sup>2022</sup></a>).</p>
<p>Hydrophobins are typically classified into two distinct families, class I (PFAM PF01185) and class II (PFAM PF06766), based on solvent solubility, amino acid hydropathy patterns, and cysteine residue spacing within the amino acid sequence (Kershaw &amp; Talbot, <a href="#pro70279-bib-0059" class="usa-link" aria-describedby="pro70279-bib-0059">1998</a>; Rojas‐Osnaya et al., <a href="#pro70279-bib-0107" class="usa-link" aria-describedby="pro70279-bib-0107"><sup>2024</sup></a>; Wösten, <a href="#pro70279-bib-0144" class="usa-link" aria-describedby="pro70279-bib-0144"><sup>2001</sup></a>). Arguably, the most well‐studied hydrophobins from a functional perspective are the class I EAS from <em>Neurospora crassa</em> and the class II HFBI / HFBII molecules from <em>Trichoderma reesei</em> (Hakanpää, Szilvay, et al., <a href="#pro70279-bib-0044" class="usa-link" aria-describedby="pro70279-bib-0044">2006</a>; Linder et al., <a href="#pro70279-bib-0075" class="usa-link" aria-describedby="pro70279-bib-0075"><sup>2001</sup></a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>; Mackay et al., <a href="#pro70279-bib-0079" class="usa-link" aria-describedby="pro70279-bib-0079"><sup>2001</sup></a>; Magarkar et al., <a href="#pro70279-bib-0080" class="usa-link" aria-describedby="pro70279-bib-0080"><sup>2014</sup></a>; Nakari‐Setälä et al., <a href="#pro70279-bib-0091" class="usa-link" aria-describedby="pro70279-bib-0091"><sup>1996</sup></a>; Nakari‐Setälä et al., <a href="#pro70279-bib-0090" class="usa-link" aria-describedby="pro70279-bib-0090"><sup>1997</sup></a>). Phylogenetic and structural analyses suggest that the class I family can further be separated into proteins originating from Ascomycota (class IA) and Basidiomycota (class IB) fungal phyla (Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038">2017</a>; Kenward et al., <a href="#pro70279-bib-0058" class="usa-link" aria-describedby="pro70279-bib-0058"><sup>2020</sup></a>). A third family termed class III (or intermediate/mixed) has also been proposed, though it is sometimes considered a special case within class I / class II (Jensen et al., <a href="#pro70279-bib-0049" class="usa-link" aria-describedby="pro70279-bib-0049">2010</a>). Most hydrophobins contain eight cysteine residues that form four intramolecular disulfide bonds thought to stabilize the tertiary fold (Ren et al., <a href="#pro70279-bib-0103" class="usa-link" aria-describedby="pro70279-bib-0103">2013</a>; Wösten, <a href="#pro70279-bib-0144" class="usa-link" aria-describedby="pro70279-bib-0144"><sup>2001</sup></a>). The cysteine residues/disulfide bonds may (for class II) or may not (for class I) play direct roles in self‐assembly (de Vocht et al., <a href="#pro70279-bib-0028" class="usa-link" aria-describedby="pro70279-bib-0028"><sup>2000</sup></a>; Kershaw et al., <a href="#pro70279-bib-0060" class="usa-link" aria-describedby="pro70279-bib-0060"><sup>2005</sup></a>; Sallada et al., <a href="#pro70279-bib-0109" class="usa-link" aria-describedby="pro70279-bib-0109"><sup>2018</sup></a>). Hydrophobins typically contain an N‐terminal secretion signal peptide (cleaved to furnish the mature hydrophobin), an intrinsically disordered N‐terminal tail, and the core folded hydrophobin domain (Kottmeier et al., <a href="#pro70279-bib-0064" class="usa-link" aria-describedby="pro70279-bib-0064">2011</a>). Generally, the core folded domain contains a mix of random coils/loops, short α‐helices, and anti‐parallel β‐sheet secondary structural elements that form around a central β‐barrel stabilized by disulfide bonds (Ball, Pham, et al., <a href="#pro70279-bib-0011" class="usa-link" aria-describedby="pro70279-bib-0011">2020</a>; Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038"><sup>2017</sup></a>; Hakanpää et al., <a href="#pro70279-bib-0043" class="usa-link" aria-describedby="pro70279-bib-0043"><sup>2004</sup></a>; Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069"><sup>2008</sup></a>). The hydrophobin loops are typically referred to with nomenclature defining the intercysteine residues (Rojas‐Osnaya et al., <a href="#pro70279-bib-0107" class="usa-link" aria-describedby="pro70279-bib-0107">2024</a>). For example, C7‐C8 defines the loop formed by the residues spanning between Cys 7 and Cys 8 (Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069">2008</a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>; Tanaka et al., <a href="#pro70279-bib-0116" class="usa-link" aria-describedby="pro70279-bib-0116"><sup>2014</sup></a>).</p>
<p>There have been 12 class I and 9 class II experimental structures solved by either X‐ray crystallography or solution NMR spectroscopy (Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a> – EAS, DewA, MPG1, SC16, SLH4, WI1, PC1, and RodA for class I; HFBI, HFBII, and NC2 for class II). The class I hydrophobin EAS from <em>Neurospora crassa</em> contains a four‐stranded β‐barrel core with surrounding β‐strands and dynamic disordered regions where charged residues (i.e., Asp and Lys) and hydrophobic residues (i.e., Ile, Ala, Leu) are localized on opposing surfaces (Kwan et al., <a href="#pro70279-bib-0070" class="usa-link" aria-describedby="pro70279-bib-0070">2006</a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>; Mackay et al., <a href="#pro70279-bib-0079" class="usa-link" aria-describedby="pro70279-bib-0079"><sup>2001</sup></a>). Likewise, the class II hydrophobins HFBI and HFBII from <em>Trichoderma reesei</em> contain a four‐stranded β‐barrel core flanked by an α‐helix and conformationally labile loops with opposing hydrophilic and hydrophobic surfaces (Hakanpää et al., <a href="#pro70279-bib-0043" class="usa-link" aria-describedby="pro70279-bib-0043">2004</a>; Hakanpää, Linder, et al., <a href="#pro70279-bib-0042" class="usa-link" aria-describedby="pro70279-bib-0042"><sup>2006</sup></a>; Hakanpää, Szilvay, et al., <a href="#pro70279-bib-0044" class="usa-link" aria-describedby="pro70279-bib-0044"><sup>2006</sup></a>). Some hydrophobins undergo post‐translational modifications, such as glycosylation (de Vocht et al., <a href="#pro70279-bib-0029" class="usa-link" aria-describedby="pro70279-bib-0029"><sup>1998</sup></a>; Linder et al., <a href="#pro70279-bib-0076" class="usa-link" aria-describedby="pro70279-bib-0076"><sup>2005</sup></a>). Glycosylation is not a strict requirement for self‐assembly since recombinant non‐glycosylated hydrophobins expressed in <em>E. coli</em> can readily self‐assemble and not all hydrophobins are glycosylated (de Vocht et al., <a href="#pro70279-bib-0029" class="usa-link" aria-describedby="pro70279-bib-0029"><sup>1998</sup></a>; Linder et al., <a href="#pro70279-bib-0076" class="usa-link" aria-describedby="pro70279-bib-0076"><sup>2005</sup></a>). Many secreted hydrophobins exist as soluble monomeric, dimeric, or tetrameric species when not in direct contact with hydrophobic–hydrophilic interfaces (Kisko et al., <a href="#pro70279-bib-0063" class="usa-link" aria-describedby="pro70279-bib-0063">2008</a>; Mackay et al., <a href="#pro70279-bib-0079" class="usa-link" aria-describedby="pro70279-bib-0079"><sup>2001</sup></a>; Morris et al., <a href="#pro70279-bib-0089" class="usa-link" aria-describedby="pro70279-bib-0089"><sup>2013</sup></a>; Wang, Graveland‐Bikker, et al., <a href="#pro70279-bib-0132" class="usa-link" aria-describedby="pro70279-bib-0132"><sup>2004</sup></a>).</p>
<p>Upon contact with a hydrophobic–hydrophilic interface, structural changes have been reported to accompany hydrophobin self‐assembly (de Vocht et al., <a href="#pro70279-bib-0029" class="usa-link" aria-describedby="pro70279-bib-0029"><sup>1998</sup></a>; de Vocht et al., <a href="#pro70279-bib-0027" class="usa-link" aria-describedby="pro70279-bib-0027"><sup>2002</sup></a>; Kallio et al., <a href="#pro70279-bib-0054" class="usa-link" aria-describedby="pro70279-bib-0054"><sup>2007</sup></a>; Pham et al., <a href="#pro70279-bib-0097" class="usa-link" aria-describedby="pro70279-bib-0097"><sup>2018</sup></a>; Sunde et al., <a href="#pro70279-bib-0114" class="usa-link" aria-describedby="pro70279-bib-0114"><sup>2008</sup></a>; Wang et al., <a href="#pro70279-bib-0131" class="usa-link" aria-describedby="pro70279-bib-0131"><sup>2002</sup></a>; Wang, Permentier, et al., <a href="#pro70279-bib-0133" class="usa-link" aria-describedby="pro70279-bib-0133"><sup>2004</sup></a>). Class I hydrophobins form monolayers of amyloid‐like fibrils (i.e., rodlets) rich in β‐sheet characteristics that undergo amyloid‐specific interactions with Congo red and Thioflavin‐T (de Vries et al., <a href="#pro70279-bib-0031" class="usa-link" aria-describedby="pro70279-bib-0031"><sup>1993</sup></a>; Kwan et al., <a href="#pro70279-bib-0070" class="usa-link" aria-describedby="pro70279-bib-0070"><sup>2006</sup></a>; Mackay et al., <a href="#pro70279-bib-0079" class="usa-link" aria-describedby="pro70279-bib-0079"><sup>2001</sup></a>; Shanmugam et al., <a href="#pro70279-bib-0110" class="usa-link" aria-describedby="pro70279-bib-0110"><sup>2019</sup></a>). The interaction between the stacked β‐sheet of the amyloid‐like hydrophobin fibrils and the dyes generates unique spectroscopic properties: gold‐green birefringence from Congo red and a fluorescence emission maximum shift to 485 nm from Thioflavin‐T (Biancalana et al., <a href="#pro70279-bib-0014" class="usa-link" aria-describedby="pro70279-bib-0014">2009</a>; Howie et al., <a href="#pro70279-bib-0046" class="usa-link" aria-describedby="pro70279-bib-0046"><sup>2008</sup></a>). Class I hydrophobins have been shown to transition from a right‐hand twisted β‐structure in monomeric forms to a relaxed β‐structure in the assembled amyloid‐like rodlets (de Vocht et al., <a href="#pro70279-bib-0027" class="usa-link" aria-describedby="pro70279-bib-0027"><sup>2002</sup></a>; Pham et al., <a href="#pro70279-bib-0097" class="usa-link" aria-describedby="pro70279-bib-0097"><sup>2018</sup></a>). The rodlets have been characterized as unbranched fibrils of 10 nm in width and 35–250 nm in length that spontaneously assemble at hydrophobic–hydrophilic interfaces and then associate laterally to form amphipathic monolayers (Dempsey &amp; Beever, <a href="#pro70279-bib-0032" class="usa-link" aria-describedby="pro70279-bib-0032">1979</a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>; Terauchi et al., <a href="#pro70279-bib-0120" class="usa-link" aria-describedby="pro70279-bib-0120"><sup>2020</sup></a>). The disordered N‐terminal tail seems to be dispensable for self‐assembly (Vergunst &amp; Langelaan, <a href="#pro70279-bib-0129" class="usa-link" aria-describedby="pro70279-bib-0129">2022</a>). For the class I hydrophobin EAS, the C3‐C4 loop is not required for rodlet formation and surface activity (Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069">2008</a>). Instead, mutagenesis and biophysical assays, supported by the generation of a chimeric hydrophobin, have identified that the hydrophobic C7‐C8 loop (residues F72‐N76 corresponding to sequence FLIIN for EAS) is responsible for rodlet formation, enabling data‐driven in silico modeling of the cross‐β assembled rodlet structure (Kwan et al., <a href="#pro70279-bib-0070" class="usa-link" aria-describedby="pro70279-bib-0070">2006</a>; Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069"><sup>2008</sup></a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>).</p>
<p>The architecture of amphipathic monolayers formed by the class II hydrophobins is in stark contrast to the rodlets formed by class I molecules (Lo et al., <a href="#pro70279-bib-0077" class="usa-link" aria-describedby="pro70279-bib-0077">2014</a>). Class II assembled monolayers are not amyloid‐like rodlets, but instead form mesh‐like structures with repeating polygonal/hexagonal arrangements of 1 to 2 nm thickness and 20 to 90 nm diameter (Chang et al., <a href="#pro70279-bib-0023" class="usa-link" aria-describedby="pro70279-bib-0023">2020</a>; Magarkar et al., <a href="#pro70279-bib-0080" class="usa-link" aria-describedby="pro70279-bib-0080"><sup>2014</sup></a>; Ren et al., <a href="#pro70279-bib-0104" class="usa-link" aria-describedby="pro70279-bib-0104"><sup>2014</sup></a>; Szilvay et al., <a href="#pro70279-bib-0115" class="usa-link" aria-describedby="pro70279-bib-0115"><sup>2007</sup></a>). Structures of HFBI and HFBII in the presence of detergents and polystyrene nanoparticles provide insights into binding to hydrophobic surfaces and multimerization: HFBI and HFBII form dimers via their hydrophobic surfaces, which then laterally interact at hydrophilic interfaces to assemble into monolayers (Hakanpää, Szilvay, et al., <a href="#pro70279-bib-0044" class="usa-link" aria-describedby="pro70279-bib-0044">2006</a>; Kallio et al., <a href="#pro70279-bib-0054" class="usa-link" aria-describedby="pro70279-bib-0054"><sup>2007</sup></a>; Kallio &amp; Rouvinen, <a href="#pro70279-bib-0055" class="usa-link" aria-describedby="pro70279-bib-0055"><sup>2011</sup></a>). Class II hydrophobins have been suggested to undergo an increase in α‐helical content during transitions from monomeric to the assembled form (Askolin et al., <a href="#pro70279-bib-0008" class="usa-link" aria-describedby="pro70279-bib-0008">2006</a>). Deletions of residues in certain loops (i.e., the C4‐C5 or C7‐C8 loops) delay self‐assembly of class II hydrophobins (Gallo et al., <a href="#pro70279-bib-0037" class="usa-link" aria-describedby="pro70279-bib-0037">2023</a>; Lienemann et al., <a href="#pro70279-bib-0074" class="usa-link" aria-describedby="pro70279-bib-0074"><sup>2013</sup></a>; Valsecchi et al., <a href="#pro70279-bib-0124" class="usa-link" aria-describedby="pro70279-bib-0124"><sup>2019</sup></a>). For both class I and class II hydrophobins, the self‐assembly process may be modulated by pH, temperature, ionic strength, pressure, detergent, acids, or organic solvents (Gravagnuolo et al., <a href="#pro70279-bib-0041" class="usa-link" aria-describedby="pro70279-bib-0041">2016</a>; Pham et al., <a href="#pro70279-bib-0097" class="usa-link" aria-describedby="pro70279-bib-0097"><sup>2018</sup></a>; Wösten &amp; de Vocht, <a href="#pro70279-bib-0145" class="usa-link" aria-describedby="pro70279-bib-0145"><sup>2000</sup></a>).</p>
<p>The diverse biological functions and materials applications of hydrophobins call for more studies on predicting structure–function relationships that underpin the many features of hydrophobin activity. Previous reports have applied computational tools spanning molecular dynamics simulations, structure modeling, and bioinformatic analysis of sequence/physicochemical properties toward advancing our understanding of hydrophobins (Bouqellah &amp; Farag, <a href="#pro70279-bib-0018" class="usa-link" aria-describedby="pro70279-bib-0018">2023</a>; Chang et al., <a href="#pro70279-bib-0023" class="usa-link" aria-describedby="pro70279-bib-0023"><sup>2020</sup></a>; de Simone et al., <a href="#pro70279-bib-0026" class="usa-link" aria-describedby="pro70279-bib-0026"><sup>2012</sup></a>; Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038"><sup>2017</sup></a>; Jensen et al., <a href="#pro70279-bib-0049" class="usa-link" aria-describedby="pro70279-bib-0049"><sup>2010</sup></a>; Mereghetti &amp; Wade, <a href="#pro70279-bib-0083" class="usa-link" aria-describedby="pro70279-bib-0083"><sup>2011</sup></a>). Machine‐learning‐based tools for structure prediction, such as AlphaFold (Abramson et al., <a href="#pro70279-bib-0001" class="usa-link" aria-describedby="pro70279-bib-0001">2024</a>; Agarwal &amp; McShan, <a href="#pro70279-bib-0002" class="usa-link" aria-describedby="pro70279-bib-0002"><sup>2024</sup></a>; Jumper et al., <a href="#pro70279-bib-0051" class="usa-link" aria-describedby="pro70279-bib-0051"><sup>2021</sup></a>), have the potential to provide novel insights into (i) hydrophobins where experimental structures are unavailable, and (ii) the global features of all known hydrophobins. To date, the ability of these tools toward global characterization, classification, and discovery of new hydrophobin properties has not yet been thoroughly evaluated. Although the abilities of AlphaFold are already established for many proteins (Abramson et al., <a href="#pro70279-bib-0001" class="usa-link" aria-describedby="pro70279-bib-0001">2024</a>; Agarwal &amp; McShan, <a href="#pro70279-bib-0002" class="usa-link" aria-describedby="pro70279-bib-0002"><sup>2024</sup></a>; Akdel et al., <a href="#pro70279-bib-0004" class="usa-link" aria-describedby="pro70279-bib-0004"><sup>2022</sup></a>; Jumper et al., <a href="#pro70279-bib-0051" class="usa-link" aria-describedby="pro70279-bib-0051"><sup>2021</sup></a>), application to hydrophobins is especially important because they are difficult to crystallize.</p>
<p>Here, we use a suite of bioinformatics tools including Rosetta, AlphaFold, Chai‐1, FoldMason, and Foldseek for analysis, structural modeling, classification, and global comparison of class I and class II hydrophobins. We first probe the structural and energetic features of experimental class I and class II hydrophobin structures available in the Protein Data Bank. Using previously solved X‐ray and NMR structures, we benchmark the ability of AlphaFold to predict class I and class II hydrophobin folds across a range of targets. We explore the physicochemical properties of all class I and class II hydrophobins in the UniProt database. Then, using AlphaFold models, we evaluate the structural universe of class I and class II hydrophobins. AlphaFold models also allow us to uncover putative non‐canonical features of hydrophobins, which are compared to the canonical features. Finally, we examine the ability of AlphaFold and Chai‐1 to model self‐assembly of class I rodlets and class II meshes. Together, our results highlight that AlphaFold accurately models both class I and class II hydrophobins with some limitations, and offer working models that provide insights into new putative hydrophobin properties.</p></section><section id="pro70279-sec-0002"><h2 class="pmc_sec_title">2. METHODS</h2>
<section id="pro70279-sec-0003"><h3 class="pmc_sec_title">2.1. Curation of class I and class II hydrophobin sequences and structures</h3>
<p>The UniProt database was mined for class I and class II hydrophobins sequences with in‐house scripts combining UniProt's programmatic access and Biopython (UniProt Consortium, <a href="#pro70279-bib-0006" class="usa-link" aria-describedby="pro70279-bib-0006">2008</a>). Class I and class II sequences were distinguished with <em>Family &amp; Domains</em> search query terms “fungal hydrophobin family” (for class I) and “cerato‐ulmin hydrophobin family” (for class II). Ultimately, a total of 5414 class I and 1442 class II hydrophobin amino acid sequences were obtained in FASTA format. Using UniProt accession numbers, we also mined the RCSB Protein Data Bank (PDB) and AlphaFold Protein Structure Database for all class I and class II hydrophobin atomic structures and models, respectively (Berman et al., <a href="#pro70279-bib-0013" class="usa-link" aria-describedby="pro70279-bib-0013">2000</a>; Varadi et al., <a href="#pro70279-bib-0127" class="usa-link" aria-describedby="pro70279-bib-0127"><sup>2022</sup></a>). From the PDB, we obtained a total of 12 fungal class I and 9 class II structures, either from X‐ray crystallography or solution NMR, in.pdb format (Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a>).</p></section><section id="pro70279-sec-0004"><h3 class="pmc_sec_title">2.2. Analysis of hydrophobin electrostatics, energetics, and physicochemical properties</h3>
<p>PDB files were analyzed in PyMOL v3.0.4. Electrostatic surface potentials were calculated in PyMOL with the Adaptive Poisson–Boltzmann Solver (APBS) Plugin (Jurrus et al., <a href="#pro70279-bib-0052" class="usa-link" aria-describedby="pro70279-bib-0052">2018</a>). Structures were prepared with the PDB<a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2PQR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PQR</a> method (Dolinsky et al., <a href="#pro70279-bib-0034" class="usa-link" aria-describedby="pro70279-bib-0034">2004</a>), APBS maps were calculated with 0.5 grid spacing, and contour scaling in the Connolly surface method ranged from −5 kT/e (red, negative) to +5 kT/e (blue, positive).</p>
<p>Total energies of all 12 fungal class I and 9 class II structures were determined in the <em>Rosetta</em> computational modeling suite (Alford et al., <a href="#pro70279-bib-0005" class="usa-link" aria-describedby="pro70279-bib-0005">2017</a>). For comparison with other small, globular proteins, analogous calculations were performed on separate SH3 domains (<em>n</em> = 13 with PDB IDs <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=1K76" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1K76</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=1NYF" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1NYF</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=1ZLM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1ZLM</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2A08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2A08</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2HDA" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2HDA</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2VVK" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2VVK</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=3C0C" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3C0C</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=3I35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3I35</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=3UA6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3UA6</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=4JZ4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JZ4</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=5NV1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5NV1</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=7A2J" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7A2J</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=7JT9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7JT9</a>) and acyl carrier proteins (<em>n</em> = 8 with PDB IDs <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=1HY8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1HY8</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=1L0H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1L0H</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=1T8K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1T8K</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2EHS" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2EHS</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2FAC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2FAC</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2K92" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2K92</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2L0Q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2L0Q</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=7E42" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7E42</a>). The REF2015 forcefield of <em>Rosetta</em> v2021.16 was used to minimize bond angles/geometry and side‐chain rotamer conformations with Idealize and Relax applications, respectively (Nivón et al., <a href="#pro70279-bib-0092" class="usa-link" aria-describedby="pro70279-bib-0092">2013</a>). Per‐residue contributions to the total energy were determined by the Residue Energy Breakdown application (Leman et al., <a href="#pro70279-bib-0072" class="usa-link" aria-describedby="pro70279-bib-0072">2020</a>). The following code was used:</p>
<pre id="pro70279-code-0001"><kbd>


idealize_jd2.static.macosclangrelease ‐s *.pdb


relax.static.macosclangrelease ‐s *_0001.pdb ‐ex1 ‐ex2


python3 extract_scores.py


residue_energy_breakdown.static.macosclangrelease ‐s *_0001_0001.pdb ‐out:file:silent energy_breakdown.sc


awk '{print $4, $7, $28, $NF'} energy_breakdown.sc | sort ‐nk3 &gt; sorted_energy_breakdown_totalscore.txt


</kbd></pre>
<p>Solvent‐accessible surface area (SASA) values were determined from the Rosetta idealized/relaxed PDB files using PyMOL with the command:</p>
<pre id="pro70279-code-0002"><kbd>


set dot_solvent, 1; set dot_density, 4; set solvent_radius, 1.4; get_area


</kbd></pre>
<p>The percentage of amino acid types in each sequence was determined using in‐house scripts utilizing Biopython's Bio.PDB PDBParser module.</p>
<p>For characterization of physicochemical properties: sequence length of full‐length class I and class I hydrophobins was obtained from UniProt and then evaluated with Biopython's Bio.Seq module. The total count of cysteine amino acids was determined over full‐length sequences with Biopython's Bio.SeqUtils.ProtParam Protein Analysis module (Wilkins et al., <a href="#pro70279-bib-0141" class="usa-link" aria-describedby="pro70279-bib-0141">1999</a>). Isoelectric point was predicted with Biopython's Bio.SeqUtils.IsoelectricPoint module according to Bjellqvist's method (Bjellqvist et al., <a href="#pro70279-bib-0015" class="usa-link" aria-describedby="pro70279-bib-0015">1993</a>). The GRAVY (Grand Average of Hydropathy) score was determined according to the Kyte and Doolittle method (Kyte &amp; Doolittle, <a href="#pro70279-bib-0071" class="usa-link" aria-describedby="pro70279-bib-0071">1982</a>) with Biopython's Bio.SeqUtils.ProtParam Protein Analysis module.</p>
<p>All scripts are freely available (see Data Availability section).</p></section><section id="pro70279-sec-0005"><h3 class="pmc_sec_title">2.3. 
AlphaFold2 benchmark of class I and class II hydrophobin structures</h3>
<p>AlphaFold2 models from the AlphaFold Protein Structure Database were compared with experimental structures from the PDBs of 21 experimental hydrophobin structures (Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a>). SWISS‐MODEL's Structure Assessment tool (<a href="https://swissmodel.expasy.org/assess" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://swissmodel.expasy.org/assess</a>) was used to determine Cα root‐mean‐square deviation (Cα RMSD, Å), Cα local distance difference test (lDDT‐Cα), and template modeling score (TM‐score) (Kabsch, <a href="#pro70279-bib-0053" class="usa-link" aria-describedby="pro70279-bib-0053">1976</a>; Mariani et al., <a href="#pro70279-bib-0081" class="usa-link" aria-describedby="pro70279-bib-0081"><sup>2013</sup></a>; Waterhouse et al., <a href="#pro70279-bib-0135" class="usa-link" aria-describedby="pro70279-bib-0135"><sup>2024</sup></a>; Zhang &amp; Skolnick, <a href="#pro70279-bib-0153" class="usa-link" aria-describedby="pro70279-bib-0153"><sup>2004</sup></a>). For NMR ensembles, values are reported for the NMR model with the lowest Cα RMSD relative to the AlphaFold2 model.</p></section><section id="pro70279-sec-0006"><h3 class="pmc_sec_title">2.4. Structure prediction of class I and class II hydrophobins</h3>
<p>Several hydrophobins or proteins containing hydrophobin‐like domains did not have precalculated AlphaFold models available in the AlphaFold Protein Structure Database. In these cases, we generated models with AlphaFold3 using default parameters (Abramson et al., <a href="#pro70279-bib-0001" class="usa-link" aria-describedby="pro70279-bib-0001">2024</a>). Random seeds were used, and the top model was chosen based on overall pTM score.</p>
<p>For benchmarking hydrophobin membrane binding and self‐assembly, AlphaFold3 modeling was performed on Georgia Tech's high‐performance cluster as briefly described below.</p>
<p>Example.json file for running AlphaFold3 with 104 different model systems with 8 class I hydrophobin sequences (EAS Δ15) and 50 oleic acid molecules (OLA):</p>
<pre id="pro70279-code-0003"><kbd>


{


 "name": "EAS_detergent_octamer",


 "modelSeeds": [


  99993, 18, 1981, 39, 950, 12, 946, 955, 972, 41, 24, 985, 970, 21, 944, 8, 989, 956, 937, 962,


  19 960 25976 444974 11 378947 53 929 936 49 923 927 910 7 929 16 991 986 4 33,


  959, 10, 22, 17, 31, 969, 962, 25, 9782517, 3, 853, 845, 832, 819, 775, 780, 753, 735, 718, 640,


  598 590 579 550 505 473 462 430 111402 365 350 325 310 284 276 264 252 231,


  223, 207, 195, 182, 19024551, 140, 132, 116, 1032, 91, 77, 66, 60, 44, 36, 28, 15, 6, 2, 1, 5, 13,


  9, 17, 27, 38


 ],


 "sequences": [


  { "protein": { "id": "A", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "protein": { "id": "B", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "protein": { "id": "C", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "protein": { "id": "D", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "protein": { "id": "E", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "protein": { "id": "F", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "protein": { "id": "G", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "protein": { "id": "H", "sequence": "SATTIGPNTCSIDDYKPYCCQSMSGSASLGCVVGVIGSQCGASVKCCKDDVTNTGNSFLIINAANCVA" } },


  { "ligand": { "id": "OA", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OB", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OC", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OD", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OE", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OF", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OG", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OH", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OI", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OJ", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OK", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OL", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OM", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ON", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OO", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OP", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OQ", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OR", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OS", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OT", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OU", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OV", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OW", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OX", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OY", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "OZ", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZA", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZB", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZC", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZD", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZE", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZF", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZG", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZH", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZI", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "ZJ", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PA", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PB", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PC", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PD", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PE", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PF", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PG", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PH", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PI", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PJ", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PK", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PL", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PM", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PN", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PO", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PP", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PQ", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PR", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PS", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PT", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PU", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PV", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PW", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PX", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PY", "ccdCodes": ["OLA"] } },


  { "ligand": { "id": "PZ", "ccdCodes": ["OLA"] } }


 ],


 "dialect": "alphafold3",


 "version": 1


}


</kbd></pre>
<p>Example of the SLURM format submission script:</p>
<pre id="pro70279-code-0004"><kbd>


#!/bin/bash


#SBATCH ‐‐job‐name=af3 # name of the job


#SBATCH ‐‐account=accountname # charge account


#SBATCH ‐‐time=72:00:00 # Time limit hrs:min:sec


#SBATCH ‐o slurm_run_%j_af3_output.txt


#SBATCH ‐e slurm_run_%j_af3_log.txt


#SBATCH ‐‐mem=80GB


#SBATCH ‐q inferno # QOS name


#SBATCH ‐‐gres=gpu:A100:1


# go into the submission directory


cd $SLURM_SUBMIT_DIR


# run AF3


input='./af_input'


output='./af_output'


echo $input


echo $output


# run the af3 structure prediction script


apptainer exec ‐‐bind "$input":/root/af_input ‐‐bind "$output":/root/af_output ‐‐bind /storage/cedar/cedar0/cedarp‐amcshan3‐0/alphafold3_param:/root/models ‐‐bind /storage/cedar/cedar0/cedarp‐amcshan3‐0/alphafold3_database:/root/public_databases ‐‐nv /storage/cedar/cedar0/cedarp‐amcshan3‐0/alphafold3/image.sif python /storage/cedar/cedar0/cedarp‐amcshan3‐0/alphafold3/run_alphafold.py ‐‐json_path=/root/af_input/EAS.json ‐‐model_dir=/root/models ‐‐db_dir=/root/public_databases ‐‐output_dir=/root/af_output


</kbd></pre>
<p>Chai‐1, an AlphaFold3 mimic (Discovery et al., <a href="#pro70279-bib-0033" class="usa-link" aria-describedby="pro70279-bib-0033">2024</a>), was also used to model self‐assembly of EAS Δ15 using the Chai‐1 webserver (<a href="https://lab.chaidiscovery.com/dashboard" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://lab.chaidiscovery.com/dashboard</a>). Five copies of EAS Δ15 were input to Chai‐1 together with distance restraints based on the expected amyloid‐like structure of EAS Δ15 (Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078">2012</a>):</p>
<section class="tw xbox font-sm" id="jats-table-wrap-1"><div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Type</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Chain 1</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Residue index 1</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Chain 2</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Residue index 2</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Distance (Å)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">E</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">E</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">20</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">E</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">E</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">20</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">E</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">C</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">D</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">E</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">11.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">F58</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">12</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">I60</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">N53</td>
<td align="left" valign="top" rowspan="1" colspan="1">12</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">T52</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">12</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Contact</td>
<td align="left" valign="top" rowspan="1" colspan="1">A</td>
<td align="left" valign="top" rowspan="1" colspan="1">T54</td>
<td align="left" valign="top" rowspan="1" colspan="1">B</td>
<td align="left" valign="top" rowspan="1" colspan="1">L59</td>
<td align="left" valign="top" rowspan="1" colspan="1">12</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/jats-table-wrap-1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="pro70279-sec-0007"><h3 class="pmc_sec_title">2.5. Generation of sequence and structure‐based dendrograms</h3>
<p>To generate a sequence‐based dendrogram, a multiple sequence alignment was built from all 6,920 hydrophobin sequences using Clustal Omega v1.2.4 (Sievers et al., <a href="#pro70279-bib-0111" class="usa-link" aria-describedby="pro70279-bib-0111">2011</a>). Inclusion of full‐length sequences resulted in a noisy alignment due to the disordered N‐terminal region and signal sequence. Thus, amino acids preceding the first cysteine were omitted as described previously (Linder et al., <a href="#pro70279-bib-0076" class="usa-link" aria-describedby="pro70279-bib-0076">2005</a>). Next, TrimAI was used to optimize the alignment by removing sequences with excessive gaps, resulting in a total of 6,754 sequences (Capella‐Gutiérrez et al., <a href="#pro70279-bib-0021" class="usa-link" aria-describedby="pro70279-bib-0021">2009</a>). Finally, IQ‐TREE 2 was used to construct the sequence tree (Minh et al., <a href="#pro70279-bib-0086" class="usa-link" aria-describedby="pro70279-bib-0086">2020</a>). The commands for the pipeline are as follows:</p>
<pre id="pro70279-code-0005"><kbd>


python3 make_fasta_trimcys.py


./clustal_omega ‐i hydrophobin_extra_trimmedCys.fasta ‐o hydrophobin_trimCys.aln ‐v ‐‐outfmt=clu –force


cd trimAl_Linux_x86‐64


./trimal ‐in hydrophobin_trimCys.aln ‐out hydrophobin_trimCys.trim ‐automated1


./iqtree3_intel ‐s hydrophobin_trimCys.trim


</kbd></pre>
<p>The resulting Newick format output was analyzed to generate an unrooted sequence‐based dendrogram in Interactive Tree of Life (iTOL) v7 (<a href="https://itol.embl.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://itol.embl.de/</a>) (Letunic &amp; Bork, <a href="#pro70279-bib-0073" class="usa-link" aria-describedby="pro70279-bib-0073">2024</a>).</p>
<p>To generate a structure‐based dendrogram, a total of 6,920 AlphaFold models of fungal class I and class II hydrophobins were passed through the multiple structure alignment tool FoldMason (<a href="https://github.com/steineggerlab/foldmason" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/steineggerlab/foldmason</a>) (Gilchrist et al., <a href="#pro70279-bib-0039" class="usa-link" aria-describedby="pro70279-bib-0039">2024</a>). There was no filtering/preprocessing before or after running FoldMason. FoldMason was run in the basic MSA workflow mode as follows:</p>
<pre id="pro70279-code-0006"><kbd>


foldmason easy‐msa *.pdb result.fasta tmpFolder ‐‐report‐mode 1


</kbd></pre>
<p>The resulting Newick format output was analyzed to generate an unrooted structure‐based dendrogram in iTOL v7 (<a href="https://itol.embl.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://itol.embl.de/</a>) (Letunic &amp; Bork, <a href="#pro70279-bib-0073" class="usa-link" aria-describedby="pro70279-bib-0073">2024</a>).</p></section><section id="pro70279-sec-0008"><h3 class="pmc_sec_title">2.6. Identification of non‐canonical hydrophobins</h3>
<p>To identify examples of putative hydrophobins with five disulfide bonds, we used in‐house scripts to filter all hydrophobin sequences containing 10 total cysteines with a sequence length of less than 250 amino acids. We then visualized the corresponding AlphaFold models to check whether all disulfide bonds were predicted to be oxidized as well as their placement relative to canonical hydrophobins. To identify examples of putative hydrophobins with extended N‐terminal tails, we used in‐house scripts to filter all hydrophobin sequences containing at least 70 amino acids before the first Cys residue. We then visualized the corresponding AlphaFold models to check whether a hydrophobin fold was present. To identify examples of polyhydrophobins, we used in‐house scripts to filter all hydrophobin sequences containing “trihydrophobin” or “pentahydrophobins” in the UniProt annotation. We also manually checked AlphaFold models of sequences &gt;250 amino acids in length. We then visualized the corresponding AlphaFold models to check whether multiple hydrophobin folds were present. For comparison of non‐canonical hydrophobins with canonical hydrophobins, sequence alignments were performed with Clustal Omega v1.2.4 and processed with ESPript 3 (Robert &amp; Gouet, <a href="#pro70279-bib-0106" class="usa-link" aria-describedby="pro70279-bib-0106">2014</a>; Sievers et al., <a href="#pro70279-bib-0111" class="usa-link" aria-describedby="pro70279-bib-0111"><sup>2011</sup></a>).</p></section><section id="pro70279-sec-0009"><h3 class="pmc_sec_title">2.7. Identification of hydrophobin‐like domains within other proteins</h3>
<p>To identify non‐hydrophobin proteins containing hydrophobin‐like domains, class I hydrophobins (DewA‐PDB 2LSH, EAS‐PDB 2FMC, MPG1‐PDB 2 2N4O, PC1‐PDB 6E08, RodA‐PDB 6GCJ, SC16‐PDB 2NBH, SLH4‐PDB 5W0Y, WI1‐PDB 6E9M) and class II hydrophobins (HFBI‐PDB 2FZ6, HFBII‐1R2M, NC2‐4AOG) were used as input to search eight structural databases (AlphaFold/Proteome v4, AlphaFold/Swiss‐Prot v4, AlphaFold/UniProt50 v4, CATH50 v4.3.0, GMGCL 2024, MGnify‐ESM30 v1, PDB100 20,240,101, BFMD 20240623) with Foldseek in 3Di/AA mode with the iterative search option without any taxonomic filters (<a href="https://search.foldseek.com/search" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://search.foldseek.com/search</a>) (van Kempen et al., <a href="#pro70279-bib-0125" class="usa-link" aria-describedby="pro70279-bib-0125"><sup>2024</sup></a>). All hits that were canonical hydrophobins were filtered out with in‐house scripts and manual inspection. Cα RMSD values between reference structures and the hydrophobin‐like domains were determined in PyMOL v3.0.4. Sequence alignments were performed with Clustal Omega 1.2.4 and processed with ESPript 3 (Robert &amp; Gouet, <a href="#pro70279-bib-0106" class="usa-link" aria-describedby="pro70279-bib-0106">2014</a>; Sievers et al., <a href="#pro70279-bib-0111" class="usa-link" aria-describedby="pro70279-bib-0111"><sup>2011</sup></a>). MSA coverage heatmaps were obtained using ColabFold v1.5.5: AlphaFold2 using MMseqs2 (Mirdita et al., <a href="#pro70279-bib-0087" class="usa-link" aria-describedby="pro70279-bib-0087">2022</a>) at the following link:</p>
<p>
<a href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb</a>.</p></section><section id="pro70279-sec-0010"><h3 class="pmc_sec_title">2.8. Prediction of aggregation potential and phase separation potential</h3>
<p>The aggregation potential of putative hydrophobins was performed with the AggreProt Predictor webserver v1 (<a href="https://loschmidt.chemi.muni.cz/aggreprot/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://loschmidt.chemi.muni.cz/aggreprot/</a>) (Planas‐Iglesias et al., <a href="#pro70279-bib-0100" class="usa-link" aria-describedby="pro70279-bib-0100">2024</a>). Canonical or non‐canonical hydrophobin sequences were input into the server and jobs were submitted using default parameters. The potential for hydrophobins to undergo phase separation was determined using the ParSe tool v2 (Ibrahim et al., <a href="#pro70279-bib-0048" class="usa-link" aria-describedby="pro70279-bib-0048">2023</a>). Canonical or non‐canonical hydrophobin sequences were input into two different server sites (<a href="https://stevewhitten.github.io/Parse_v2_FASTA/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://stevewhitten.github.io/Parse_v2_FASTA/</a> and <a href="https://stevewhitten.github.io/Parse_v2_web/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://stevewhitten.github.io/Parse_v2_web/</a>) using default parameters.</p></section></section><section id="pro70279-sec-0011"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="pro70279-sec-0012"><h3 class="pmc_sec_title">3.1. Class I hydrophobins exhibit greater structural diversity than class II hydrophobins</h3>
<p>As a baseline for defining structure–function relationships, we first compared and contrasted the features of class I and class II fungal hydrophobins with previously determined experimental structures available in the PDB (Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a>) (Berman et al., <a href="#pro70279-bib-0013" class="usa-link" aria-describedby="pro70279-bib-0013">2000</a>). We visualized the general structural features of eight class I hydrophobins (MPG1, EAS, SC16, DewA, SLH4, PC1, WI1, and RodA) and three class II hydrophobins (HFBI, HFBII, and NC2) (Figure <a href="#pro70279-fig-0001" class="usa-link">1</a>, Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S1</a> and <a href="#pro70279-supitem-0001" class="usa-link">S2</a>) Vergunst et al., <a href="#pro70279-bib-0128" class="usa-link" aria-describedby="pro70279-bib-0128"><sup>2022</sup></a>. Both class I and class II hydrophobins showed a centralized β‐barrel core composed of anti‐parallel β‐sheets stabilized by four disulfide bonds. Both class I and class II hydrophobins exhibited amphipathic surfaces with a charged hydrophilic face and a mostly neutral hydrophobic face. Electrostatic surface potentials as determined with the APBS method revealed that the precise surface electrostatic potential profile differs from protein to protein (Unni et al., <a href="#pro70279-bib-0122" class="usa-link" aria-describedby="pro70279-bib-0122">2011</a>). For example, the hydrophilic face of DewA showed a more localized electrostatic surface potential, whereas it was more evenly distributed for MPG1 (Figure <a href="#pro70279-fig-0001" class="usa-link">1</a>). The hydrophobic faces of both PC1 and WI1 were more negatively charged relative to other class I hydrophobins (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S1</a>). As previously reported (Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038">2017</a>), class I hydrophobins showed greater structural diversity than class II hydrophobins, as highlighted by differences in the number/arrangement of α‐helices flanking the core domain, the number/arrangement of β‐strands in the core domain, and the length/orientation of the intercysteine loops (C1‐C2, C3‐C4, C4‐C5, C7‐C8). Class II hydrophobins HFBI, HFBII, and NC2 had highly similar folds and electrostatic surface potentials around the central β‐barrel composed of four anti‐parallel β‐strands flanked by a single α‐helix (Figure <a href="#pro70279-fig-0001" class="usa-link">1</a>). The hydrophobic face of HFBI and HFBII, which is lined by the C1‐C2, C3‐C4, and C7‐C8 loops, has been shown to provide a surface for hydrophobin oligomerization as well as association with different types of detergents (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S2</a>). Together, this analysis highlights previous reports that class I hydrophobins exhibit greater structural diversity than class II hydrophobins, and unique features underpin their differences in self‐assembly (Linder et al., <a href="#pro70279-bib-0076" class="usa-link" aria-describedby="pro70279-bib-0076">2005</a>; Wösten, <a href="#pro70279-bib-0144" class="usa-link" aria-describedby="pro70279-bib-0144"><sup>2001</sup></a>).</p>
<figure class="fig xbox font-sm" id="pro70279-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2287/12375992/bd171770ba36/PRO-34-e70279-g001.jpg" loading="lazy" id="jats-graphic-1" height="488" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70279-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>General structural features of class I and class II hydrophobins. Examples of class I (left) and class II (right) hydrophobin structures from the Protein Data Bank. Cartoon representations are shown with β‐strands in cyan, α‐helices in pink, and coils in gray. Disulfide bonds are shown as yellow sticks. Each cartoon is also accompanied by an electrostatic surface visualization calculated with the Adaptive Poisson–Boltzmann Solver (APBS plugin of PyMOL) (Unni et al., <a href="#pro70279-bib-0122" class="usa-link" aria-describedby="pro70279-bib-0122">2011</a>). The contour scale for the APBS visualization is −5 kT/e (red, negative) to +5 kT/e (blue, positive). The hydrophobic (white) and hydrophilic (dark blue/dark red) faces of the amphipathic hydrophobin surface are noted. The following structures are shown: MPG1 from <em>Magnaporthe oryzae</em> (PDB ID 2N4O‐Class I) (Pham et al., <a href="#pro70279-bib-0096" class="usa-link" aria-describedby="pro70279-bib-0096">2016</a>), EAS from <em>Neurospora crassa</em> (PDB ID 2FMC‐Class I) (Kwan et al., <a href="#pro70279-bib-0070" class="usa-link" aria-describedby="pro70279-bib-0070">2006</a>), SC16 from <em>Schizophyllum commune</em> (PDB ID 2NBH‐Class I) (Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038">2017</a>), DewA from <em>Aspergillus nidulans</em> (PDB ID 2LSH‐Class I) (Morris et al., <a href="#pro70279-bib-0089" class="usa-link" aria-describedby="pro70279-bib-0089">2013</a>), HFBI from <em>Trichoderma reesei</em> (PDB ID 2FZ6‐Class II) (Hakanpää, Szilvay, et al., <a href="#pro70279-bib-0044" class="usa-link" aria-describedby="pro70279-bib-0044">2006</a>), HFBII from <em>Trichoderma reesei</em> (PDB ID 1R2M‐Class II) (Hakanpää, Szilvay, et al., <a href="#pro70279-bib-0044" class="usa-link" aria-describedby="pro70279-bib-0044">2006</a>), and NC2 from <em>Neurospora crassa</em> (PDB ID 4AOG‐Class II) (Ren et al., <a href="#pro70279-bib-0104" class="usa-link" aria-describedby="pro70279-bib-0104">2014</a>). The N‐terminus (“N”), C‐terminus (“C”), and secondary structure elements (α‐helix, β‐strands, and intercysteine loops) are noted.</p></figcaption></figure></section><section id="pro70279-sec-0013"><h3 class="pmc_sec_title">3.2. Van der Waals attractive forces, coulombic electrostatics, and backbone hydrogen bonds stabilize class I and class II hydrophobin structures</h3>
<p>To evaluate the types of molecular forces that stabilize class I and class II hydrophobin folds, we idealized bond angles/geometry, performed side‐chain rotamer energy minimization, and then scored each structure within the REF2015 forcefield of the <em>Rosetta</em> modeling suite (Alford et al., <a href="#pro70279-bib-0005" class="usa-link" aria-describedby="pro70279-bib-0005">2017</a>; Leman et al., <a href="#pro70279-bib-0072" class="usa-link" aria-describedby="pro70279-bib-0072"><sup>2020</sup></a>). This allowed us to determine which of <em>Rosetta</em>'s 19 physics‐ and knowledge‐based energy terms contribute favorably or unfavorably to the total energy of the different classes of hydrophobins. Results were highly self‐consistent within the structures evaluated (Figure <a href="#pro70279-fig-0002" class="usa-link">2a</a>). The total <em>Rosetta</em> energy values determined from the idealized and relaxed structures were −151.5 ± 47.7 Rosetta energy units (REU) over the 12 class I structures and −142.5 ± 16.8 REU across the 9 class II structures. The REF2015 energy terms that contributed most favorably to hydrophobin structures were Lennard‐Jones attractive forces (fa_atr), Coulombic electrostatic interactions (fa_elec), and short−/long‐range hydrogen bonding within the backbone (hbond_sr_bb and hbond_lr_bb). The REF2015 energy terms that contributed most unfavorably to hydrophobin structures were the Lazaridis–Karplus solvation energy (fa_sol), internal energy of side‐chain rotamers as derived from Dunbrack's statistics (fa_dun), Lennard‐Jones repulsive forces (fa_rep), and reference energies for each amino acid (ref). We also probed whether the residue‐residue interactions that most contributed to total energies were localized to specific regions of the hydrophobin structure. Broadly speaking, visualization of the top five pairs of interactions that contributed most favorably to the total energy reveals highly localized hydrophobic and electrostatic interactions between residues within the disulfide bond stabilized β‐sheet core (Figure <a href="#pro70279-fig-0002" class="usa-link">2b,c</a>). Together, these results highlight self‐consistent patterns of Van der Waals attractive forces (i.e., dipole–dipole, dipole‐induced dipole, London dispersion), electrostatic interactions, and backbone hydrogen bonds that contribute to the overall favorable energetics of class I and class II hydrophobin monomeric structures.</p>
<figure class="fig xbox font-sm" id="pro70279-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2287/12375992/09d6a467507d/PRO-34-e70279-g003.jpg" loading="lazy" id="jats-graphic-3" height="871" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70279-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>
<em>Rosetta</em> evaluation of the molecular forces contributing to hydrophobin stability. (a) Histogram of contributions of each <em>Rosetta</em> REF2015 score function energy term to the total energy of class I and class II hydrophobins from the Protein Data Bank. Data are mean ± standard deviation where error bars are derived from separate calculations of different hydrophobin structures; <em>n</em> = 12 for class I and <em>n</em> = 9 for class II (see Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a>). (b) and (c) show summaries of the top five molecular interactions (residue‐residue pairs) for each structure: MPG1 from <em>Magnaporthe oryzae</em> (PDB ID 2N4O‐Class I) (Pham et al., <a href="#pro70279-bib-0096" class="usa-link" aria-describedby="pro70279-bib-0096">2016</a>), SC16 from <em>Schizophyllum commune</em> (PDB ID 2NBH‐Class I) (Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038">2017</a>), DewA from <em>Aspergillus nidulans</em> (PDB ID 2LSH‐Class I) (Morris et al., <a href="#pro70279-bib-0089" class="usa-link" aria-describedby="pro70279-bib-0089">2013</a>), HFBI from <em>Trichoderma reesei</em> (PDB ID 2FZ6‐Class II) (Hakanpää, Szilvay, et al., <a href="#pro70279-bib-0044" class="usa-link" aria-describedby="pro70279-bib-0044">2006</a>), HFBII from <em>Trichoderma reesei</em> (PDB ID 1R2M‐Class II) (Hakanpää et al., <a href="#pro70279-bib-0043" class="usa-link" aria-describedby="pro70279-bib-0043">2004</a>), and NC2 from <em>Neurospora crassa</em> (PDB ID 4AOG‐Class II) (Ren et al., <a href="#pro70279-bib-0104" class="usa-link" aria-describedby="pro70279-bib-0104">2014</a>). Disulfide bonds are shown as yellow sticks.</p></figcaption></figure><p>We next asked whether the forces that stabilize hydrophobins were similar to those that stabilize other small globular proteins. The primarily β‐sheet SH3 domains and the primarily α‐helical acyl carrier proteins were used as model systems (Farmer et al., <a href="#pro70279-bib-0036" class="usa-link" aria-describedby="pro70279-bib-0036">2019</a>; Kurochkina &amp; Guha, <a href="#pro70279-bib-0068" class="usa-link" aria-describedby="pro70279-bib-0068"><sup>2012</sup></a>). The REF2015 energy terms that contributed most favorably (i.e., fa_atr, fa_elec, hbond_sr_bb, hbond_lr_bb) and least favorably (i.e., fa_sol) to hydrophobin structures were highly consistent with forces contributing to SH3 domain and acyl carrier protein stability (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S3</a>). Like hydrophobins, electrostatic surface potentials of SH3 domains showed opposing hydrophilic and hydrophobic surfaces, consistent with reports that SH3 ligand binding sites have a hydrophobic surface flanked by charged loops (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S3</a>) (Booker et al., <a href="#pro70279-bib-0016" class="usa-link" aria-describedby="pro70279-bib-0016">1993</a>). In contrast, acyl carrier proteins exhibited a highly charged (mostly negative) surface lacking a major hydrophobic face, consistent with reports that negatively charged acyl carrier proteins interact with positively charged surfaces of cognate enzymes (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S3</a>) (Barajas et al., <a href="#pro70279-bib-0012" class="usa-link" aria-describedby="pro70279-bib-0012">2016</a>). The SASAs and percentage of polar/charged/hydrophobic residues across hydrophobins, SH3 domain, and acyl carrier protein structures were also relatively consistent except for class I hydrophobins that showed a moderate increase in the SASA that could be attributed to the flexible disordered loops. According to this analysis, the forces stabilizing hydrophobins are similar to those underlying stability in other small, globular proteins. The unique features attributed to hydrophobin function are likely linked to a combination of their amphipathic nature, conformational plasticity, and sequence/structure features present in the hydrophobic intercysteine loops.</p></section><section id="pro70279-sec-0014"><h3 class="pmc_sec_title">3.3. Physicochemical properties of class I and class II hydrophobins are well delineated</h3>
<p>To study the global features of hydrophobins, we evaluated the physicochemical properties of all class I and class II hydrophobins extracted from the UniProt database. As others have done at a smaller scale (Bouqellah &amp; Farag, <a href="#pro70279-bib-0018" class="usa-link" aria-describedby="pro70279-bib-0018">2023</a>; Rineau et al., <a href="#pro70279-bib-0105" class="usa-link" aria-describedby="pro70279-bib-0105"><sup>2017</sup></a>; Zhao et al., <a href="#pro70279-bib-0155" class="usa-link" aria-describedby="pro70279-bib-0155"><sup>2021</sup></a>), we computationally probed sequence length, cysteine count, isoelectric point, and GRAVY score of each sequence. Most class I and class II hydrophobins had sequence lengths between 100 and 140, with a larger distribution of lengths observed for class I hydrophobins (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S4a</a>). Cysteine counts of class I and class II hydrophobins were heavily skewed to 8, consistent with four disulfide bonds within the core folded domain (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S4b</a>). Predicted isoelectric points (pI) were found to be clustered between 4 and 6 for both class I and class II hydrophobins (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S4c</a>). The range of predicted pI values is consistent with experimental zeta potential measurements (Terauchi et al., <a href="#pro70279-bib-0119" class="usa-link" aria-describedby="pro70279-bib-0119">2022</a>; Yang et al., <a href="#pro70279-bib-0150" class="usa-link" aria-describedby="pro70279-bib-0150"><sup>2019</sup></a>). Given that the intracellular pH of filamentous fungi ranges from 7.4 to 7.7 (Bagar et al., <a href="#pro70279-bib-0009" class="usa-link" aria-describedby="pro70279-bib-0009">2009</a>), most cytoplasmic hydrophobins would be negatively charged. Of note, the surface pH along growing hyphae ranges from 5 to 6.3, where hydrophobins could exhibit a less negative charge (Xiong et al., <a href="#pro70279-bib-0148" class="usa-link" aria-describedby="pro70279-bib-0148">2022</a>). GRAVY scores, a measure of hydropathy where a higher positive value indicates greater hydrophobicity (Kyte &amp; Doolittle, <a href="#pro70279-bib-0071" class="usa-link" aria-describedby="pro70279-bib-0071">1982</a>), were primarily populated in the range from 0 to 1 for both class I and class II hydrophobins (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S4d</a>). These GRAVY scores highlight the unique hydrophobic properties of hydrophobins relative to the mean GRAVY scores across the proteomes of bacteria, archaea, and eukaryotes that range from −0.5 to −0.3 (Jin et al., <a href="#pro70279-bib-0050" class="usa-link" aria-describedby="pro70279-bib-0050">2021</a>). Together, this global analysis shows that the physicochemical properties of class I and class II hydrophobins are well delineated and similar between the different classes.</p></section><section id="pro70279-sec-0015"><h3 class="pmc_sec_title">3.4. 
AlphaFold2 robustly predicts structures of both class I and class II hydrophobins</h3>
<p>We next benchmarked the ability of AlphaFold2 to predict structures of monomeric class I and class II hydrophobins. Toward quantitative evaluation, we used several structural comparison metrics, such as Cα RMSD, lDDT‐Cα, and TM‐score, to compare AlphaFold2 models with 21 experimental X‐ray or NMR structures (Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a>) (Kabsch, <a href="#pro70279-bib-0053" class="usa-link" aria-describedby="pro70279-bib-0053">1976</a>; Mariani et al., <a href="#pro70279-bib-0081" class="usa-link" aria-describedby="pro70279-bib-0081"><sup>2013</sup></a>; Waterhouse et al., <a href="#pro70279-bib-0135" class="usa-link" aria-describedby="pro70279-bib-0135"><sup>2024</sup></a>; Zhang &amp; Skolnick, <a href="#pro70279-bib-0153" class="usa-link" aria-describedby="pro70279-bib-0153"><sup>2004</sup></a>). Cα RMSD measures the mean distance between corresponding atoms and represents a global superposition where values &lt;1.5 Å represent the highest quality predictions, values from 1.5 to 2.5 Å represent high‐quality predictions, values between 2.5 to 4 Å represent intermediate‐quality predictions, and values above 4 Å represent failed predictions (Bornot et al., <a href="#pro70279-bib-0017" class="usa-link" aria-describedby="pro70279-bib-0017">2009</a>; McPartlon &amp; Xu, <a href="#pro70279-bib-0082" class="usa-link" aria-describedby="pro70279-bib-0082"><sup>2023</sup></a>). Because RMSD can be insensitive to local errors and may not reliably distinguish partially correct models from incorrect ones, multiple metrics of comparison are often used to quantify the accuracy of predicted models relative to native conformations (Jumper et al., <a href="#pro70279-bib-0051" class="usa-link" aria-describedby="pro70279-bib-0051">2021</a>; Kufareva &amp; Abagyan, <a href="#pro70279-bib-0067" class="usa-link" aria-describedby="pro70279-bib-0067"><sup>2012</sup></a>; Olechnovič et al., <a href="#pro70279-bib-0094" class="usa-link" aria-describedby="pro70279-bib-0094"><sup>2019</sup></a>). To better assess prediction accuracy, complementary metrics, such as TM‐score and lDDT‐Cα, are often used. TM‐score measures the mean distance between Cα atoms scaled by a length‐dependent distance parameter and represents a global superposition where values above 0.6 represent likely accurate predictions (Xu &amp; Zhang, <a href="#pro70279-bib-0149" class="usa-link" aria-describedby="pro70279-bib-0149">2010</a>; Zhang &amp; Skolnick, <a href="#pro70279-bib-0153" class="usa-link" aria-describedby="pro70279-bib-0153"><sup>2004</sup></a>). lDDT‐Cα measures the mean fraction of preserved all‐atom distance for Cα atoms and presents local comparisons within defined tolerance thresholds where values above 0.6 represent likely accurate predictions (Jumper et al., <a href="#pro70279-bib-0051" class="usa-link" aria-describedby="pro70279-bib-0051">2021</a>; Mariani et al., <a href="#pro70279-bib-0081" class="usa-link" aria-describedby="pro70279-bib-0081"><sup>2013</sup></a>). Within our benchmarking dataset, we observed Cα RMSD values of 2.2 ± 0.6 for class I and 1.4 ± 1.1 for class II, lDDT‐Cα values of 0.71 ± 0.07 for class I and 0.84 ± 0.08 for class II, and TM‐score values of 0.80 ± 0.07 for class I and 0.90 ± 0.09 for class II (Figure <a href="#pro70279-fig-0003" class="usa-link">3a</a>). While several hydrophobin AlphaFold models fall outside the Cα RMSD quality threshold, almost all models were within the lDDT‐Cα and TM‐score cut‐off scores for moderate to high‐quality predictions, supporting the proper prediction of the global hydrophobin fold (Figure <a href="#pro70279-fig-0003" class="usa-link">3a</a>).</p>
<figure class="fig xbox font-sm" id="pro70279-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2287/12375992/bbe4ceb233f0/PRO-34-e70279-g004.jpg" loading="lazy" id="jats-graphic-5" height="956" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70279-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>AlphaFold2 accurately predicts structures of class I and class II hydrophobins. (a) Results from benchmarking of AlphaFold2 on 21 class I and class II hydrophobin experimental structures. Root‐mean‐square deviation (Cα RMSD, Å), Local Distance Difference Test (lDDT‐Cα), and Template modeling score (TM‐score) for AlphaFold2 models relative to experimental structures are summarized. Error bars are derived from the mean ± the standard deviation across comparison of all structures within each class (see Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a>). The dotted black lines in each graph denote standard cut‐offs that mark a good prediction (Cα RMSD &lt;2.5 Å, lDDT‐Cα &gt;0.6, TM‐score &gt;0.6). (b) Overlay of AlphaFold2 models (blue) versus experimental structure (green) for class I hydrophobins. PDB IDs: <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2FMC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2FMC</a> (EAS from <em>Neurospora crassa</em>) (Kwan et al., <a href="#pro70279-bib-0070" class="usa-link" aria-describedby="pro70279-bib-0070">2006</a>), 6E98 (PC1 from <em>Phanerochaete carnosa</em>) (Kenward et al., <a href="#pro70279-bib-0058" class="usa-link" aria-describedby="pro70279-bib-0058">2020</a>), 2LSH (DewA from <em>Aspergillus nidulans</em>) (Morris et al., <a href="#pro70279-bib-0089" class="usa-link" aria-describedby="pro70279-bib-0089">2013</a>). (c) Overlay of AlphaFold2 models (blue) versus experimental structures (green) for class II hydrophobins. PDB IDs: <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2GVM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2GVM</a> (HFBI from <em>Trichoderma reesei</em>) (Hakanpää, Szilvay, et al., <a href="#pro70279-bib-0044" class="usa-link" aria-describedby="pro70279-bib-0044">2006</a>), 1R2M (HFBII from <em>Trichoderma reesei</em>) (Hakanpää et al., <a href="#pro70279-bib-0043" class="usa-link" aria-describedby="pro70279-bib-0043">2004</a>), 4AOG (NC2 from <em>Neurospora crassa</em>) (Ren et al., <a href="#pro70279-bib-0104" class="usa-link" aria-describedby="pro70279-bib-0104">2014</a>). For each AlphaFold3 model, the Cα RMSD, lDDT‐Cα, and TM‐score values relative to the experiment structure are noted. For clarity, experimental structures derived from solution NMR (PDB IDs <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2FMC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2FMC</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=6E98" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6E98</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2LSH" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2LSH</a>, <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=4AOG" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4AOG</a>) are shown with only the model with the lowest Cα RMSD value relative to the AlphaFold2 model.</p></figcaption></figure><p>Visual inspections of AlphaFold2 models versus experimental structures for class I hydrophobins and class II hydrophobins highlight the robustness of the structure prediction within the β‐barrel core (Figure <a href="#pro70279-fig-0003" class="usa-link">3b,c</a>, Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S5</a> and <a href="#pro70279-supitem-0001" class="usa-link">S6</a>). In terms of per‐residue predicted local‐distance difference test (pLDDT) confidence scores, AlphaFold2 models of class II hydrophobins generally exhibit higher per‐residue pLDDT values (indicating greater confidence) relative to class I hydrophobins, consistent with the enhanced structural diversity, sequence diversity, and flexibility of class I proteins. There were some notable deficiencies in AlphaFold's ability to model the loop regions for some of the hydrophobins (for example, it does well for PC1, HFBI, and HFBII, but poorly for DewA, EAS, and NC2) (Figure <a href="#pro70279-fig-0003" class="usa-link">3b,c</a>). Difficulty in modeling the flexible loops of hydrophobins is an important potential limitation, given that the C7‐C8 loop of class I (Kwan et al., <a href="#pro70279-bib-0070" class="usa-link" aria-describedby="pro70279-bib-0070">2006</a>; Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069"><sup>2008</sup></a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>) and C4‐C5 / C7‐C8 loops of class II hydrophobins (Gallo et al., <a href="#pro70279-bib-0037" class="usa-link" aria-describedby="pro70279-bib-0037">2023</a>; Lienemann et al., <a href="#pro70279-bib-0074" class="usa-link" aria-describedby="pro70279-bib-0074"><sup>2013</sup></a>; Valsecchi et al., <a href="#pro70279-bib-0124" class="usa-link" aria-describedby="pro70279-bib-0124"><sup>2019</sup></a>) are important for self‐assembly. Both NMR dynamic measurements and MD simulations suggest these loops are highly flexible and unlikely to adopt a single well‐ordered conformation for monomeric hydrophobins in solution (Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069">2008</a>; Mackay et al., <a href="#pro70279-bib-0079" class="usa-link" aria-describedby="pro70279-bib-0079"><sup>2001</sup></a>; Nolle et al., <a href="#pro70279-bib-0093" class="usa-link" aria-describedby="pro70279-bib-0093"><sup>2023</sup></a>; Vergunst &amp; Langelaan, <a href="#pro70279-bib-0129" class="usa-link" aria-describedby="pro70279-bib-0129"><sup>2022</sup></a>). Other investigators have observed that AlphaFold assigns lower pLDDT values to disordered regions of proteins with high sequence diversity (Akdel et al., <a href="#pro70279-bib-0004" class="usa-link" aria-describedby="pro70279-bib-0004">2022</a>). Together, these results reveal that AlphaFold2 seems to predict structures of the monomeric states of both class I and class II hydrophobins with moderate to high confidence, but may have some limitations in modeling the flexible, functional loops of hydrophobins.</p></section><section id="pro70279-sec-0016"><h3 class="pmc_sec_title">3.5. 
AlphaFold models enable global classification of the structural universe of class I and class II hydrophobins</h3>
<p>To uncover global structural features for class I and class II hydrophobins, we compared AlphaFold models of all class I and class II hydrophobins using the multiple structure alignment tool FoldMason (Gilchrist et al., <a href="#pro70279-bib-0039" class="usa-link" aria-describedby="pro70279-bib-0039">2024</a>). We then prepared an unrooted structure‐based dendrogram using the iTOL tool (Letunic &amp; Bork, <a href="#pro70279-bib-0073" class="usa-link" aria-describedby="pro70279-bib-0073">2024</a>). The dendrogram highlights the impressive structural diversity of class I hydrophobins with five unique clades showing clear separation in the tree (Figure <a href="#pro70279-fig-0004" class="usa-link">4</a>). For example, as previously noted from phylogenetic analysis of hydrophobin sequences (Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038">2017</a>; Linder et al., <a href="#pro70279-bib-0076" class="usa-link" aria-describedby="pro70279-bib-0076"><sup>2005</sup></a>; Tanaka et al., <a href="#pro70279-bib-0117" class="usa-link" aria-describedby="pro70279-bib-0117"><sup>2022</sup></a>), global structural comparisons resulted in the subdivision of class I hydrophobin sequences into those originating from Ascomycota or Basidiomycota fungal phyla (class IA and class IB, respectively) (Figure <a href="#pro70279-fig-0004" class="usa-link">4</a>). Class IA differed from Class IB due to subtle differences in the central anti‐parallel β‐sheet “core” structure. Class IB proteins originating from Basidiomycota were further split into three similar but unique clades that exhibit differences in the central β‐barrel structure and intercysteine loops (Figure <a href="#pro70279-fig-0004" class="usa-link">4</a>). Structures within intermediate/mixed folds were also identified as a unique clade (Figure <a href="#pro70279-fig-0004" class="usa-link">4</a>) (Jensen et al., <a href="#pro70279-bib-0049" class="usa-link" aria-describedby="pro70279-bib-0049">2010</a>). In stark contrast, class II hydrophobins were localized into a single clade originating from Ascomycota, consistent with high structural conservation (Figures <a href="#pro70279-fig-0001" class="usa-link">1</a> and <a href="#pro70279-fig-0004" class="usa-link">4</a>) (Hakanpää et al., <a href="#pro70279-bib-0043" class="usa-link" aria-describedby="pro70279-bib-0043">2004</a>; Ren et al., <a href="#pro70279-bib-0104" class="usa-link" aria-describedby="pro70279-bib-0104"><sup>2014</sup></a>). To gain insights into how hydrophobin sequence conservation is linked to structure conservation, we generated a sequence‐based dendrogram of all hydrophobins. In contrast to the six well‐defined clades of the structure‐based dendrogram, the sequence‐based dendrogram had many more subclades despite clear disambiguation between class IA, class IB, and class II hydrophobin classes (Figure <a href="#pro70279-fig-0004" class="usa-link">4</a>, Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S7</a>). This suggests much greater hydrophobin sequence diversity relative to structural diversity. Sequence alignments showed that class II ascomycetes contained high conservation for the canonical cysteine residues, core residues in the β‐barrel structure, the C3‐C4 loop, and the C7‐C8 loop but limited conservation in the C1‐C2 loop, C4‐C5 loop, and α1 helix (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S8</a>). Conserved residues in class IA and class IIB hydrophobins were primarily localized to the canonical cysteine residues and core residues in the β‐barrel structure with low conservation in the C3‐C4 loop and C4‐5 loop (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S8</a>). The C7‐C8 showed some sequence conservation for class IB but not class IA. Together, these results suggest that AlphaFold models can be used toward global classification of class I and class II hydrophobins into six distinct clades that exhibit unique sequence and structure features.</p>
<figure class="fig xbox font-sm" id="pro70279-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2287/12375992/7724a245f907/PRO-34-e70279-g002.jpg" loading="lazy" id="jats-graphic-7" height="432" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70279-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure‐based dendrogram of AlphaFold2 models classifies class I and class II hydrophobins into six distinct clades. Global comparison of AlphaFold2 models was performed with multiple structure alignment software FoldMason (Gilchrist et al., <a href="#pro70279-bib-0039" class="usa-link" aria-describedby="pro70279-bib-0039">2024</a>). An unrooted structure‐based dendrogram was generated by iTOL, resulting in 6,920 leaves (i.e., structural models) broken up into six main clades highlighted with different colored circles (Letunic &amp; Bork, <a href="#pro70279-bib-0073" class="usa-link" aria-describedby="pro70279-bib-0073">2024</a>). Representative AlphaFold2 models are shown for each clade colored per‐residue by pLDDT score. The flexible N‐terminal tails are not shown for clarity. Each structure is labeled with its corresponding UniProt accession number. Class I hydrophobins are subdivided into class IA and class IB based on ascomycetes or basidiomycetes origins, respectively.</p></figcaption></figure></section><section id="pro70279-sec-0017"><h3 class="pmc_sec_title">3.6. 
AlphaFold models enable identification of non‐canonical hydrophobin features</h3>
<p>With AlphaFold models in hand, we searched for non‐canonical hydrophobin features that could result in unique functional properties. Several putative non‐canonical features are described below and illustrated in Figure <a href="#pro70279-fig-0005" class="usa-link">5</a>. First, while canonical hydrophobins contain a cysteine count of 8 resulting in four disulfide bonds (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S4b</a>), we identified hydrophobins with 10 cysteines that were predicted by AlphaFold to have an additional fifth oxidized disulfide bond. An example is UniProt ID #A0A9P5TUV7, a putative class I hydrophobin from <em>Rhodocollybia butyracea</em> (Figure <a href="#pro70279-fig-0005" class="usa-link">5</a>). A total of 112 class I and 17 class II hydrophobins with 10 cysteines were identified, suggesting hydrophobins with 10 cysteines are relatively rare. The class I sequences with 10 cysteines were biased toward <em>Peniophora, Collybiopsis, Armillaria</em>, and <em>Talaromyces</em> fungal genera. The class II sequences with 10 cysteines were biased toward <em>Tolypocladium, Fusarium, Akanthomyces</em>, and <em>Phomopsis</em> fungal genera. For class II hydrophobins, the new disulfide bond in UniProt ID #A0A8K0J0P6 could be primitively termed the C9‐C10 disulfide since it precedes the canonical C7‐C8 disulfide and is spatially placed adjacent to the core hydrophobin fold (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S9</a>). The structure and sequence for A0A8K0J0P6 were partially conserved across the canonical class II hydrophobins HFBI, HFBII, and NC2 such that we expect it to be functionally active. For class I hydrophobins, despite the almost perfect spatial overlap of the four canonical disulfide bonds with four of the disulfide bonds for the putative novel hydrophobin, the sequence mapping of Cys connectivity was different (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S10</a> – for example, the canonical C1‐C6 disulfide connectivity maps to the new C4‐C7 disulfide connectivity). Given these differences between class I hydrophobins with 8 and 10 cysteines, we predicted the aggregation‐prone sequence of non‐canonical class I hydrophobins with five disulfide bonds relative to canonical class I hydrophobins using the in silico tool AggreProt (Planas‐Iglesias et al., <a href="#pro70279-bib-0100" class="usa-link" aria-describedby="pro70279-bib-0100">2024</a>). AggreProt correctly identified the C7‐C8 loop as the likely aggregation site for canonical class I hydrophobins EAS, MPG1, and SC16 (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S11</a>). For non‐canonical class I hydrophobins A0A9P5TUV7, A0A5E3WUH3, A0A5E3X438, a sequence spanning C7 to C10 was identified (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S11</a> – note that here the Cys residue numbering is different as described above). Finally, the non‐canonical class I hydrophobins exhibited an amphipathic nature with one hydrophobin and one hydrophilic surface, suggesting that they are bona fide hydrophobins (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S11</a>). The ability of these new putative hydrophobins to self‐assemble into functional rodlets remains to be shown experimentally. Thus, taken with a grain of salt, the putative new class I hydrophobins with five disulfide bonds seem to exhibit unique sequence features despite an overall EAS‐like hydrophobin fold. AlphaFold placed the new disulfide bond spatially near the canonical C3‐C4 disulfide (for class I) and the canonical C1‐C6 disulfide (for class II) where it could impact the folding properties, stability, or aggregation features of the hydrophobin β‐barrel core (Figure <a href="#pro70279-fig-0005" class="usa-link">5</a>, Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S9–S11</a>).</p>
<figure class="fig xbox font-sm" id="pro70279-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2287/12375992/ead0216b1f43/PRO-34-e70279-g006.jpg" loading="lazy" id="jats-graphic-9" height="404" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70279-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>AlphaFold models reveal non‐canonical features of hydrophobins. An AlphaFold3 model of UniProt ID A0A9P5TUV7 (a class I hydrophobin from <em>Rhodocollybia butyracea</em> with five disulfide bonds—note that the nomenclature of the Cys pairs is different from the canonical class I hydrophobins). AlphaFold Protein Structure Database accession numbers are: AF‐A0A0F4GET0‐F1 (a class II hydrophobin from <em>Zymoseptoria brevis</em> with a Gly, Ser, Asn rich extended disordered N‐terminal domain), AF‐<a href="https://www.ncbi.nlm.nih.gov/protein/Q9UVI4" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9UVI4</a>‐F1 (a class II trihydrophobin TH1 from <em>Claviceps fusiformis</em>), AF‐<a href="https://www.ncbi.nlm.nih.gov/protein/Q8J1W4" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q8J1W4</a>‐F1 (a class II pentahydrophobin CPPH1 from <em>Claviceps purpurea</em>), and AF‐G8A517‐F1 (a class I hydrophobin from <em>Flammulina velutipes</em> that contains a Fic/DOC‐like domain). The dotted box highlights Protein Data Bank accession numbers: PDB ID <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=4BHU" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4BHU</a> (bacterial hydrophobin BslA from <em>Bacillus subtilis</em>) (Hobley et al., <a href="#pro70279-bib-0045" class="usa-link" aria-describedby="pro70279-bib-0045">2013</a>) and PDB ID <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=5MKD" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5MKD</a> (bacterial hydrophobin Ywea from <em>Bacillus subtilis</em>) (Arnaouteli et al., <a href="#pro70279-bib-0007" class="usa-link" aria-describedby="pro70279-bib-0007">2023</a>). AlphaFold models are colored by per‐residue pLDDT score.</p></figcaption></figure><p>Second, we identified hydrophobins with an extended, disordered N‐terminal tail (independent of the secretion signal). On average, canonical hydrophobins contain a 15 to 25 residue long signal peptide followed by a few (&lt;25) disordered residues before the beginning of the central β‐barrel core. We defined non‐canonical hydrophobins with extended disordered N‐terminal tails to have at least 70 residues preceding the first Cys in the sequence, which forms the boundary of the β‐barrel core (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S12a</a>). An example is UniProt ID #A0A0F4GET0, a putative class II hydrophobin from <em>Zymoseptoria brevis</em> (Figure <a href="#pro70279-fig-0005" class="usa-link">5</a>). Many extended N‐terminal tails were predicted by AlphaFold to be primarily disordered, although low confidence secondary structure elements were modeled within some sequences (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S12b</a>). A total of 143 class I and 60 class II hydrophobins with extended N‐terminal tails were identified. The class I sequences with extended N‐terminal tails were biased toward <em>Armillaria, Aspergillus</em>, and <em>Rhizoctonia</em> fungal genera. The class II sequences with extended N‐terminal tails were biased toward <em>Trichoderma, Cercospora</em>, and <em>Ophiocordyceps</em> fungal genera. Amino acid sequence alignment revealed that, in general, beyond the signal peptide, there is limited conservation of residues within the extended N‐terminal tail for both class I and class II hydrophobins (Supplementary Alignments <a href="#pro70279-supitem-0001" class="usa-link">S1</a> and <a href="#pro70279-supitem-0001" class="usa-link">S2</a>). However, apart from the extended disordered N‐terminal tail, there was more structural and sequence similarity within the β‐barrel core of the hydrophobin domain, suggesting hydrophobins with extended N‐terminal tails are fully functional proteins (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S12b</a>, Supplementary Alignments <a href="#pro70279-supitem-0001" class="usa-link">S1</a> and <a href="#pro70279-supitem-0001" class="usa-link">S2</a>). The biological function of the disordered N‐terminal tail is poorly understood but it does not seem to be strictly required for self‐assembly (Vergunst &amp; Langelaan, <a href="#pro70279-bib-0129" class="usa-link" aria-describedby="pro70279-bib-0129">2022</a>). Long, disordered N‐terminal tails could perform regulatory functions analogous to those observed for other proteins: they could regulate conformational changes via allosteric coupling effects (Tee et al., <a href="#pro70279-bib-0118" class="usa-link" aria-describedby="pro70279-bib-0118">2020</a>), act as intramolecular entropic bristles limiting undesired protein aggregation (Graña‐Montes et al., <a href="#pro70279-bib-0040" class="usa-link" aria-describedby="pro70279-bib-0040">2014</a>; Yu &amp; Sukenik, <a href="#pro70279-bib-0151" class="usa-link" aria-describedby="pro70279-bib-0151"><sup>2023</sup></a>), or provide binding sites for interactions with modulatory proteins (Převorovský et al., <a href="#pro70279-bib-0101" class="usa-link" aria-describedby="pro70279-bib-0101">2011</a>). We found that non‐canonical class II (and to a lesser extent class I) hydrophobins with extended N‐terminal sequences were rich in Gly, Ser, and Asn residues that promote phase separation (Ibrahim et al., <a href="#pro70279-bib-0048" class="usa-link" aria-describedby="pro70279-bib-0048">2023</a>) (Supplementary Alignment <a href="#pro70279-supitem-0001" class="usa-link">S1</a> and <a href="#pro70279-supitem-0001" class="usa-link">S2</a>). The potential for the non‐canonical hydrophobins with extended N‐terminal tails to undergo phase separation was predicted with the <em>in silico</em> tool ParSe (Ibrahim et al., <a href="#pro70279-bib-0048" class="usa-link" aria-describedby="pro70279-bib-0048">2023</a>), which supported that extended N‐terminal hydrophobin sequences likely contain features for phase separation (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S13</a>). Taken together, these results allow us to hypothesize that some non‐canonical hydrophobins with extended disordered N‐terminal tails might have the ability to undergo phase separation that promotes or alters their functionality.</p>
<p>Third, we identified polyhydrophobin sequences containing three (trihydrophobin) to five (pentahydrophobin) hydrophobin domains within a single polypeptide chain. Examples include UniProt ID #<a href="https://www.ncbi.nlm.nih.gov/protein/Q9UVI4" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9UVI4</a>, a class II trihydrophobin CFTH1 from <em>Claviceps fusiformis</em>, and #<a href="https://www.ncbi.nlm.nih.gov/protein/Q8J1W4" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q8J1W4</a>, a class II pentahydrophobin CPPH1 from <em>Claviceps purpurea</em> (Figure <a href="#pro70279-fig-0005" class="usa-link">5</a>). A total of 12 class I (11 trihydrophobin, 1 pentahydrophobin) and 126 class II (124 trihydrophobin, 2 pentahydrophobin) multidomain hydrophobins were identified. The class I sequences with multidomain hydrophobins were biased toward <em>Fusarium</em> fungal genera. The class II sequences with multidomain hydrophobins were biased toward <em>Fusarium, Trichoderma, Gibberella</em>, and <em>Claviceps</em> fungal genera. As highlighted for CPPH1 from <em>Claviceps purpurea</em>, a more in‐depth analysis shows that the individual hydrophobin units of the class II pentahydrophobin were highly conserved in both sequence and structure with canonical monomeric hydrophobins, such as HFBII (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S14a–d</a>). Thus, we expect that multidomain hydrophobins can self‐assemble, although how the multimerization could influence the assembly process or monolayer structures remains to be seen. These findings are consistent with previous reports that have identified and characterized trihydrophobins and pentahydrophobins, rich in Gly/Asp repeat linkers, as functional self‐assembling hydrophobins from <em>Claviceps</em> with potential roles in the formation of aerial hyphae (De Vries et al., <a href="#pro70279-bib-0030" class="usa-link" aria-describedby="pro70279-bib-0030">1999</a>; Mey et al., <a href="#pro70279-bib-0084" class="usa-link" aria-describedby="pro70279-bib-0084"><sup>2003</sup></a>). Future studies should evaluate the potential of fungal proteases to cleave polyhydrophobins into monomeric hydrophobins.</p>
<p>Finally, although not present within our fungal hydrophobin testing set, non‐canonical bacterial hydrophobins, such as BslA and Ywea, have been reported (Arnaouteli et al., <a href="#pro70279-bib-0007" class="usa-link" aria-describedby="pro70279-bib-0007">2023</a>; Hobley et al., <a href="#pro70279-bib-0045" class="usa-link" aria-describedby="pro70279-bib-0045"><sup>2013</sup></a>). BslA and Ywea adopt an immunoglobulin‐like β‐sandwich fold, similar to the amyloid‐forming protein β2‐microglobulin (Wilkinson et al., <a href="#pro70279-bib-0142" class="usa-link" aria-describedby="pro70279-bib-0142">2023</a>). AlphaFold was able to predict bacterial hydrophobin folds with moderate to high accuracy within the immunoglobulin‐like core; however, notable differences were observed in the cap region of the bacterial hydrophobins that are reported to be important for assembly (Arnaouteli et al., <a href="#pro70279-bib-0007" class="usa-link" aria-describedby="pro70279-bib-0007">2023</a>; Hobley et al., <a href="#pro70279-bib-0045" class="usa-link" aria-describedby="pro70279-bib-0045"><sup>2013</sup></a>) (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S15</a>).</p>
<p>Together, these results highlight that AlphaFold models of hydrophobins, together with sequence analysis and <em>in silico</em> functional prediction tools, can reveal putative non‐canonical features and have the potential to generate tested hypotheses to guide future <em>in vitro, in situ</em>, and <em>in vitro</em> experiments to disambiguate their functionalities.</p></section><section id="pro70279-sec-0018"><h3 class="pmc_sec_title">3.7. 
AlphaFold models uncover hydrophobin‐like domains within non‐hydrophobin proteins</h3>
<p>We also identified hydrophobin domains within a single polypeptide chain that harbored separate well‐folded globular domains outside of the primary hydrophobin fold. An example is UniProt ID #G8A517, a class I hydrophobin from <em>Flammulina velutipes</em> that contains a Fic/DOC‐like domain (Figure <a href="#pro70279-fig-0005" class="usa-link">5</a>). Although it remains to be experimentally tested, the presence of the Fic/DOC‐like domain could promote AMPylation of proteins at hydrophobic‐hydrophobic surfaces where the hydrophobin self‐assembles (Veyron et al., <a href="#pro70279-bib-0130" class="usa-link" aria-describedby="pro70279-bib-0130">2018</a>). Intrigued by the possibility of non‐hydrophobin domains covalently linked in a single polypeptide chain to hydrophobin‐like domains, we used canonical hydrophobin structures as “bait” proteins within Foldseek, which searches several structural databases (AlphaFold/Proteome, AlphaFold/Swiss‐Prot, AlphaFold/UniProt50, CATH50, GMGCL MGnify‐ESM30, PDB100) to identify non‐hydrophobin proteins that contain domains resembling class I or class II hydrophobins (van Kempen et al., <a href="#pro70279-bib-0125" class="usa-link" aria-describedby="pro70279-bib-0125">2024</a>). Many canonical hydrophobins were identified but not considered for analysis since the goal was to identify hydrophobin‐like folds within other proteins. Instead, hits were defined solely as structures that contained separate well‐folded globular domains in addition to a canonical hydrophobin‐like fold.</p>
<p>For class II, 25 total hits were identified using HFBI, HFBII, and NC2 structures as bait (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S16</a>). For class I, 62 total hits were identified using DewA, EAS, MPG1, PC1, RodA, SC16, SL14, and WI1 structures as bait (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S16</a>). Notably, most hit proteins were derived from fungal organisms (i.e., <em>Leucoagaricus</em> sp. and <em>Aspergillus</em> sp.). However, a few hits were derived from non‐fungal organisms (i.e., dinoflagellate symbiotic alga <em>Zooxanthella</em>, diatom <em>Skeletonema</em>, and plant <em>Brassica</em>). Many hits were shared and consistently identified across the similar bait proteins, but there were also unique hits from some bait proteins (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S16</a>). The hydrophobin‐like domains had Cα RMSD values ranging from 0.5 to 5 Å to the bait class I or class II hydrophobin structures, indicating high to low structural similarity. Examples of hits are described below. Using SC16 as the bait for class I hydrophobin‐like domains, we identified a putative 60S ribosomal protein L27‐A from <em>Leucoagaricus</em> sp. (UniProt #A0A137QB05), a putative Chaperonin from <em>Aspergillus fumigatus</em> (UniProt #A0A0J5PMS5), a putative Nucleoporin from Nup54 from <em>Leucoagaricus leucothites</em> (UniProt #A0A8H5FVP6), and a putative BTB domain‐containing protein from <em>Moniliophthora roreri</em> (UniProt #A0A0W0FA47) (Figure <a href="#pro70279-fig-0006" class="usa-link">6</a>, Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S17</a> for other examples). Using HFBI as the bait for class II hydrophobin‐like domains, we identified a putative Ferric reductase transmembrane component 3 from <em>Beauveria bassiana</em> (UniProt #A0A0A2W2K4), a putative serine/threonine protein kinase BUD32 from <em>Sporothrix insectorum</em> (UniProt #A0A162IG95), a putative 2‐polyprenyl‐6‐methoxyphenol hydroxylase‐like oxidoreductase from <em>Aureobasidium pullulans</em> (UniProt #A0A4S9H610), a putative Thiopurine methyltransferase from <em>Pyrenophora tritici‐repentis</em> (UniProt #A0A2W1CXJ2) (Figure <a href="#pro70279-fig-0006" class="usa-link">6</a>, Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S18</a> for other examples). In general, the hydrophobin‐like domains from polypeptide chains within other proteins were found to be well conserved in both sequence and structure with monomeric class I and class II hydrophobins. For example, most of the identified hydrophobin‐like domains contained the four intramolecular disulfide bonds and intercysteine loops that resembled canonical hydrophobins, suggesting many of the newly identified non‐canonical hydrophobins might be functional for self‐assembly (Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S19</a> and <a href="#pro70279-supitem-0001" class="usa-link">S20</a>). One caveat of this analysis is that many of the non‐canonical hydrophobin‐like domains had moderate pLDDT values (50–70), suggesting only some confidence in the predicted hydrophobin fold. We suspect this may be due to limitations in sequence coverage and/or template identity in the multiple sequence alignment (MSA). This is supported by heat‐map representations of the MSA coverage and plots of predicted lDDT per‐residue position for a hydrophobin domain with low average pLDDT (UniProt #A0A162IG95) and high average pLDDT (UniProt #A0A2H1FZK2) (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S21</a>).</p>
<figure class="fig xbox font-sm" id="pro70279-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2287/12375992/9e340cc18fef/PRO-34-e70279-g005.jpg" loading="lazy" id="jats-graphic-11" height="408" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70279-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>AlphaFold models uncover hydrophobin‐like domains within non‐hydrophobin proteins. Comparison of experimental hydrophobin structures (magenta) overlaid with AlphaFold models of proteins containing hydrophobin‐like domains (colored by pLDDT score). Examples for proteins containing class I hydrophobin‐like domains: AF‐A0A137QB05‐F1 (60S ribosomal protein L27‐A from <em>Leucoagaricus</em> sp.), AF‐A0A0J5PMS5‐F1 (Chaperonin from <em>Aspergillus fumigatus</em>), AlphaFold3 model of UniProt ID A0A8H5FVP6 (Nucleoporin Nup54 from <em>Leucoagaricus leucothites</em>), and AF‐A0A0W0FA47‐F1 (BTB domain‐containing protein from <em>Moniliophthora roreri</em>). Examples for proteins containing class II hydrophobin‐like domains: AF‐A0A0A2W2K4‐F1 (Ferric reductase transmembrane component 3 from <em>Beauveria bassiana</em>), A0A162IG95‐F1 (serine/threonine protein kinase BUD32 from <em>Sporothrix insectorum</em>), AF‐A0A4S9H610‐F1 (2‐polyprenyl‐6‐methoxyphenol hydroxylase‐like oxidoreductase from <em>Aureobasidium pullulans</em>), and AF‐A0A2W1CXJ2‐F1 (Thiopurine methyltransferase from <em>Pyrenophora tritici‐repentis</em>). Cα RMSD values (Ångströms, Å) each hydrophobin relative to the hydrophobin‐like domain are noted. Reference hydrophobins are colored magenta: SC16 for class I (PDB ID <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2NBH" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2NBH</a>) and HFBI (PDB ID <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2FZ6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2FZ6</a>) for class II. AlphaFold models are colored by per‐residue pLDDT score.</p></figcaption></figure><p>Together, these results suggest a combination of databases containing AlphaFold models (the AlphaFold structure database) and robust structural alignment software (Foldseek) is useful at identifying novel putative proteins with hydrophobin‐like domains. The role of the hydrophobin‐like domain in these putative proteins is unclear and requires experimental testing. We posit that the hydrophobin‐like domains either help localize these proteins at the fungal membrane‐water surface or promote oligomerization.</p></section><section id="pro70279-sec-0019"><h3 class="pmc_sec_title">3.8. Limitations in AlphaFold and Chai‐1 modeling of hydrophobin self‐assembly</h3>
<p>Another outstanding question is whether AlphaFold3 and other machine‐learning‐based structure prediction software, such as Chai‐1, can generate plausible models of hydrophobin binding to membranes and/or hydrophobin self‐assembly (Abramson et al., <a href="#pro70279-bib-0001" class="usa-link" aria-describedby="pro70279-bib-0001">2024</a>; Discovery et al., <a href="#pro70279-bib-0033" class="usa-link" aria-describedby="pro70279-bib-0033"><sup>2024</sup></a>; Evans et al., <a href="#pro70279-bib-0035" class="usa-link" aria-describedby="pro70279-bib-0035"><sup>2022</sup></a>; Jumper et al., <a href="#pro70279-bib-0051" class="usa-link" aria-describedby="pro70279-bib-0051"><sup>2021</sup></a>; Krishna et al., <a href="#pro70279-bib-0065" class="usa-link" aria-describedby="pro70279-bib-0065"><sup>2024</sup></a>). To address this, we performed AlphaFold3 modeling of class I EAS<sub>Δ15</sub> and class II HFBII in three different scenarios: (i) 8 hydrophobin copies in the absence of a membrane‐mimetic; (ii) 1 copy of hydrophobin with 50 copies of oleic acid; and (iii) 8 hydrophobin copies with 50 copies of oleic acid. Oleic acid was used to mimic a membrane‐like hydrophobic surface. Relative to the experimental structure of class I EAS<sub>Δ15</sub> (Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069">2008</a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>), AlphaFold3 predicted conformational changes in the C3‐C4 and C7‐C8 functional loops in the presence of oleic acid or multiple hydrophobin copies (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S22a–c</a>). Further, the hydrophobic surface of EAS was embedded into the oleic acid membrane, with the hydrophilic surface providing contacts between hydrophobin monomers. Even after running calculations with more than 100 different model seeds, AlphaFold3 was not able to predict the amyloid‐like fibril structure expected of EAS<sub>Δ15</sub> (Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069">2008</a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>). Relative to the experimental structure of class II HFBII (Hakanpää et al., <a href="#pro70279-bib-0043" class="usa-link" aria-describedby="pro70279-bib-0043">2004</a>; Kallio et al., <a href="#pro70279-bib-0054" class="usa-link" aria-describedby="pro70279-bib-0054"><sup>2007</sup></a>; Linder et al., <a href="#pro70279-bib-0075" class="usa-link" aria-describedby="pro70279-bib-0075"><sup>2001</sup></a>), AlphaFold3 did not predict conformational changes in the C3‐C4 and C7‐C8 functional loops of HFBII in the presence of oleic acid or multiple hydrophobin copies (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S23a–c</a>). The hydrophobic surface of HFBII was also embedded into the membrane, with the hydrophilic surface providing contacts between hydrophobin monomers. AlphaFold did not predict the mesh‐like structure expected of HFBII, nor did it predict oligomerization mechanisms as observed in the crystal structure of HFBII in the presence of detergents (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S23d</a>) (Hakanpää et al., <a href="#pro70279-bib-0043" class="usa-link" aria-describedby="pro70279-bib-0043">2004</a>; Kallio et al., <a href="#pro70279-bib-0054" class="usa-link" aria-describedby="pro70279-bib-0054"><sup>2007</sup></a>; Linder et al., <a href="#pro70279-bib-0075" class="usa-link" aria-describedby="pro70279-bib-0075"><sup>2001</sup></a>). We also attempted to force the amyloid‐like fibril structure of class I EAS<sub>Δ15</sub> by performing structure modeling in Chai‐1 using distance restraints between TNT (residues 52 to 54 based on numbering in PDB <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2K6A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2K6A</a>) and FLI (residues 58 to 60 based on numbering in PDB <a href="http://bioinformatics.org/firstglance/fgij//fg.htm?mol=2K6A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2K6A</a>) (Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069">2008</a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>). Relative to the experimental structure of EAS<sub>Δ15</sub> (Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069">2008</a>; Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078"><sup>2012</sup></a>), Chai‐1 models with distance restraints showed a massive conformational change in the C3‐C4 and C7‐C8 functional loops with increased β‐sheet content but still did not resemble the expected amyloid‐like fibril structure (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S24</a>). Taken together, these results highlight that AlphaFold and Chai‐1 models of hydrophobins have the potential to provide insights into binding of hydrophobic surfaces but have difficulty modeling amyloid‐like fibrils and mesh‐like oligomeric states. These limitations of modern structure prediction software have been reported for other systems (Pinheiro et al., <a href="#pro70279-bib-0098" class="usa-link" aria-describedby="pro70279-bib-0098">2021</a>).</p></section></section><section id="pro70279-sec-0020"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Here, we apply a suite of computational tools with AlphaFold models at the helm to analyze, model, classify, and compare the global features of fungal class I and class II hydrophobins. To our knowledge, this work represents the most comprehensive application of these bioinformatics tools to date for the exploration of structure–function relationships in hydrophobins. Based on quantitative analysis of molecular interactions within the <em>Rosetta</em> REF2015 forcefield, both class I and class II hydrophobins are stabilized by attractive forces (representative of hydrophobic interactions), electrostatics (representative of dipole interactions and charged residue interactions), and backbone hydrogen bonding; these forces also stabilize other small, globular proteins (Figure <a href="#pro70279-fig-0002" class="usa-link">2</a>, Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S3</a>). The consistency between the results is striking given the sequence and structural diversity within the class I hydrophobin family and differences between class I and class II folds. Nonetheless, in each case, the residue‐residue pairs defining the total energy are highly located within the core β‐sheet. These findings set a baseline for future <em>Rosetta</em> analysis that could be useful for defining how mutations destabilize monomeric hydrophobin states (i.e., ∆∆G of mutation via <em>Rosetta</em>'s <em>cartesian_ddg</em> application) (Park et al., <a href="#pro70279-bib-0095" class="usa-link" aria-describedby="pro70279-bib-0095">2016</a>) or using computational site‐directed mutagenesis to engineer tunable features into hydrophobins (i.e., <em>Rosetta</em>'s mutagenesis protocol, Rosetta design) (Thieker et al., <a href="#pro70279-bib-0121" class="usa-link" aria-describedby="pro70279-bib-0121">2022</a>).</p>
<p>While AlphaFold has revolutionized the structure prediction field, it is not assumed that it will perform well at every modeling task. Benchmarking of different families of proteins is essential to understand the strengths and weaknesses of AlphaFold and its algorithmic cousins (Agarwal &amp; McShan, <a href="#pro70279-bib-0002" class="usa-link" aria-describedby="pro70279-bib-0002">2024</a>; Akdel et al., <a href="#pro70279-bib-0004" class="usa-link" aria-describedby="pro70279-bib-0004"><sup>2022</sup></a>; Chakravarty &amp; Porter, <a href="#pro70279-bib-0022" class="usa-link" aria-describedby="pro70279-bib-0022"><sup>2022</sup></a>). Our benchmark shows that AlphaFold is robust at predicting structures of monomeric class I and class II hydrophobins but may have some limitations in modeling the functional intercysteine loops (Figure <a href="#pro70279-fig-0003" class="usa-link">3</a>). It may be possible to combine AlphaFold modeling with <em>Rosetta</em> relax protocols or all‐atom MD simulations to evaluate the conformational landscapes of loops with lower pLDDT values. AlphaFold's training procedure involved a self‐distillation set together with known structures from the PDB (maximum release date of 30 April 2018) (Jumper et al., <a href="#pro70279-bib-0051" class="usa-link" aria-describedby="pro70279-bib-0051">2021</a>). Furthermore, AlphaFold was not trained on structures solved by NMR methods (Abramson et al., <a href="#pro70279-bib-0001" class="usa-link" aria-describedby="pro70279-bib-0001">2024</a>; Jumper et al., <a href="#pro70279-bib-0051" class="usa-link" aria-describedby="pro70279-bib-0051"><sup>2021</sup></a>). This matters here because many hydrophobin structures in the PDB were determined by NMR (Supplementary Table <a href="#pro70279-supitem-0001" class="usa-link">S1</a>). Since many hydrophobin structures evaluated in our benchmark were either released after the training or are solution NMR structures, our results suggest that AlphaFold has not simply learned to predict the same structures it was trained on but is a bona fide predictor of monomeric class I and class II hydrophobin folds. As new hydrophobin structures are determined, these conclusions should be further evaluated.</p>
<p>Classification of hydrophobins is important because each subclass exhibits unique functional features (i.e., rodlet formation vs. mesh formation), and comparisons between classes can provide information on evolutionary mechanisms (Ren et al., <a href="#pro70279-bib-0103" class="usa-link" aria-describedby="pro70279-bib-0103">2013</a>; Sunde et al., <a href="#pro70279-bib-0114" class="usa-link" aria-describedby="pro70279-bib-0114"><sup>2008</sup></a>; Wösten, <a href="#pro70279-bib-0144" class="usa-link" aria-describedby="pro70279-bib-0144"><sup>2001</sup></a>; Wösten &amp; de Vocht, <a href="#pro70279-bib-0145" class="usa-link" aria-describedby="pro70279-bib-0145"><sup>2000</sup></a>). Class I and class II hydrophobins have been historically analyzed and classified by sequence‐based phylogenetic analysis (Gandier et al., <a href="#pro70279-bib-0038" class="usa-link" aria-describedby="pro70279-bib-0038">2017</a>; Jensen et al., <a href="#pro70279-bib-0049" class="usa-link" aria-describedby="pro70279-bib-0049"><sup>2010</sup></a>; Linder et al., <a href="#pro70279-bib-0076" class="usa-link" aria-describedby="pro70279-bib-0076"><sup>2005</sup></a>; Whiteford &amp; Spanu, <a href="#pro70279-bib-0140" class="usa-link" aria-describedby="pro70279-bib-0140"><sup>2002</sup></a>). Here, a combination of AlphaFold2 modeling and multiple structure alignment at scale by FoldMason (Gilchrist et al., <a href="#pro70279-bib-0039" class="usa-link" aria-describedby="pro70279-bib-0039">2024</a>) allowed us to construct a detailed unrooted dendrogram of hydrophobins based on structural comparison. Six distinct clades were identified: one for class IA ascomycetes, one for class II ascomycetes, one mixed clade for ascomycetes and basidiomycetes, and three separate clades for class IB basidiomycetes (Figure <a href="#pro70279-fig-0004" class="usa-link">4</a>). Together, these clades represent new insights into global structural features of class I and class II fungal hydrophobins structural universe. Representative structures highlight an interesting dichotomy: while the hydrophobin core and loops are quite diverse, the disulfide bonded β‐sheet core is an underlying commonality where subtle differences could fine‐tune protein stability and dynamics. We expect that upon identification of new hydrophobin sequences, AlphaFold modeling, and together with FoldMason, can provide a robust, structure‐based classification scheme. Finally, structure‐based and sequence‐based dendrograms can provide useful information on how hydrophobins have evolved in fungi. For example, it has been proposed that class II hydrophobins evolved independently of class I hydrophobins in a case of convergent evolution (Whiteford &amp; Spanu, <a href="#pro70279-bib-0140" class="usa-link" aria-describedby="pro70279-bib-0140">2002</a>). Class IA hydrophobins and some clades of class IB hydrophobins showed a very similar core β‐barrel/α‐helix fold typical of hydrophobins despite being in separate evolutionary clades with distinct sequence features (Figure <a href="#pro70279-fig-0004" class="usa-link">4</a>, Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S7</a> and <a href="#pro70279-supitem-0001" class="usa-link">S8</a>). The strong functional selection for maintaining a specific architecture, even if sequences diverge widely, implies that similar structures have evolved independently in distant fungal groups to perform similar roles, which is a hallmark of convergent evolution.</p>
<p>Through analysis of AlphaFold models and the application of protein structure alignment at scale with Foldseek (van Kempen et al., <a href="#pro70279-bib-0125" class="usa-link" aria-describedby="pro70279-bib-0125"><sup>2024</sup></a>), we also identified relatively rare non‐canonical features of hydrophobins: hydrophobins with extended N‐terminal tails, hydrophobins with five disulfide bonds, polyhydrophobins, and non‐hydrophobin proteins containing hydrophobin‐like folds (Figures <a href="#pro70279-fig-0005" class="usa-link">5</a> and <a href="#pro70279-fig-0006" class="usa-link">6</a>). While some of these features are supported by previous experiments (i.e., trihydrophobins and pentahydrophobins) (De Vries et al., <a href="#pro70279-bib-0030" class="usa-link" aria-describedby="pro70279-bib-0030">1999</a>; Mey et al., <a href="#pro70279-bib-0084" class="usa-link" aria-describedby="pro70279-bib-0084"><sup>2003</sup></a>), many of these putative features or hypothetical proteins remain to be tested. Our data allow us to posit the following: extended N‐terminal tails may be involved in phase separation (Supplementary Figure <a href="#pro70279-supitem-0001" class="usa-link">S13</a>); the additional disulfide bond could impart increased stability into the hydrophobin fold (Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S9</a> and <a href="#pro70279-supitem-0001" class="usa-link">S10</a>); and the possibility exists that the presence of hydrophobin‐like folds within diverse families of fungal proteins could influence their biological function or activity in meaningful ways (Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S17</a> and <a href="#pro70279-supitem-0001" class="usa-link">S18</a>). In particular, the identification and characterization of hydrophobin domains in a single polypeptide chain with non‐hydrophobin domains could inform the design of many types of bifunctional proteins for a myriad of applications. The emergence of new copies of hydrophobin folds within other proteins in some cases could arise from gene duplication events that have been proposed in many hydrophobin‐containing fungal species (Karlsson et al., <a href="#pro70279-bib-0057" class="usa-link" aria-describedby="pro70279-bib-0057">2007</a>; Kubicek et al., <a href="#pro70279-bib-0066" class="usa-link" aria-describedby="pro70279-bib-0066"><sup>2008</sup></a>; Mgbeahuruike et al., <a href="#pro70279-bib-0085" class="usa-link" aria-describedby="pro70279-bib-0085"><sup>2013</sup></a>).</p>
<p>Several important areas of structure–function relationships of hydrophobins have not been probed here but should be addressed in future work. The first is that AlphaFold and Chai modeling of hydrophobin self‐assembly and membrane binding seems to be a major limitation (Supplementary Figures <a href="#pro70279-supitem-0001" class="usa-link">S22–S24</a>). In addition, hydrophobins are highly dynamic proteins that sample conformationally diverse ensembles in solution (Macindoe et al., <a href="#pro70279-bib-0078" class="usa-link" aria-describedby="pro70279-bib-0078">2012</a>; Mackay et al., <a href="#pro70279-bib-0079" class="usa-link" aria-describedby="pro70279-bib-0079"><sup>2001</sup></a>; Vergunst &amp; Langelaan, <a href="#pro70279-bib-0129" class="usa-link" aria-describedby="pro70279-bib-0129"><sup>2022</sup></a>). The “static” structures analyzed and modeled here provide useful insights but may not represent the full picture. Several groups are working on adapting AlphaFold toward prediction of conformational ensembles by clustering or subsampling of the multiple sequence alignments (Bowman, <a href="#pro70279-bib-0019" class="usa-link" aria-describedby="pro70279-bib-0019">2024</a>; Monteiro da Silva et al., <a href="#pro70279-bib-0088" class="usa-link" aria-describedby="pro70279-bib-0088"><sup>2024</sup></a>; Sala et al., <a href="#pro70279-bib-0108" class="usa-link" aria-describedby="pro70279-bib-0108"><sup>2023</sup></a>; Wayment‐Steele et al., <a href="#pro70279-bib-0136" class="usa-link" aria-describedby="pro70279-bib-0136"><sup>2024</sup></a>). Once these algorithms are refined and benchmarked to address a wide range of systems, it will be interesting to see them applied to class I and class II hydrophobins. The modeling is expected to be improved by the direct integration of structure prediction tools with experimental measurements of dynamics through the use of NMR spectroscopy, neutron spin echo spectroscopy, or small angle neutron scattering (Chinnam et al., <a href="#pro70279-bib-0025" class="usa-link" aria-describedby="pro70279-bib-0025">2023</a>; Huang &amp; Montelione, <a href="#pro70279-bib-0047" class="usa-link" aria-describedby="pro70279-bib-0047"><sup>2024</sup></a>; Stingaciu, <a href="#pro70279-bib-0113" class="usa-link" aria-describedby="pro70279-bib-0113"><sup>2022</sup></a>). For future studies, several approaches that have helped for other amyloid‐like assemblies may prove useful (Ragonis‐Bachar et al., <a href="#pro70279-bib-0102" class="usa-link" aria-describedby="pro70279-bib-0102">2024</a>; Wojciechowska et al., <a href="#pro70279-bib-0143" class="usa-link" aria-describedby="pro70279-bib-0143"><sup>2024</sup></a>), (i) performing thousands of different predictions with randomized seeds followed by sorting based on ipTM score (Abramson et al., <a href="#pro70279-bib-0001" class="usa-link" aria-describedby="pro70279-bib-0001">2024</a>), (ii) performing predictions with varying chain copy numbers, (iii) including different types of lipids in the prediction to generate an artificial hydrophobic–hydrophilic interface (Kallio et al., <a href="#pro70279-bib-0054" class="usa-link" aria-describedby="pro70279-bib-0054">2007</a>), and (iv) providing explicit residue‐residue restraints from experimental mutagenesis (Gallo et al., <a href="#pro70279-bib-0037" class="usa-link" aria-describedby="pro70279-bib-0037">2023</a>; Kwan et al., <a href="#pro70279-bib-0070" class="usa-link" aria-describedby="pro70279-bib-0070"><sup>2006</sup></a>; Kwan et al., <a href="#pro70279-bib-0069" class="usa-link" aria-describedby="pro70279-bib-0069"><sup>2008</sup></a>; Lienemann et al., <a href="#pro70279-bib-0074" class="usa-link" aria-describedby="pro70279-bib-0074"><sup>2013</sup></a>; Valsecchi et al., <a href="#pro70279-bib-0124" class="usa-link" aria-describedby="pro70279-bib-0124"><sup>2019</sup></a>). Conducted against a backdrop of steadily improving computational tools and with these limitations in mind, AlphaFold seems ready to uncover new biology, provide new testable hypotheses, and aid in the engineering of proteins.</p></section><section id="pro70279-sec-0025"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Li‐Yen Yang:</strong> Investigation. <strong>Daniel J. Hicks:</strong> Conceptualization; investigation. <strong>Paul S. Russo:</strong> Writing – review and editing; supervision. <strong>Andrew C. McShan:</strong> Conceptualization; investigation; writing – original draft; writing – review and editing; methodology; formal analysis; supervision; visualization.</p></section><section id="pro70279-sec-0022"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare that there are no competing interests.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70279-supitem-0001"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375992/bin/PRO-34-e70279-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70279-s001.docx</a><sup> (24.1MB, docx) </sup>
</div></div></section></section><section id="pro70279-sec-0021" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>A.C.M. acknowledges start‐up funding from the Georgia Institute of Technology and the Shurl and Kay Curci Foundation under award GR00027161. P.S.R. and D.J.H. acknowledge support from the Hightower Family through the Georgia Institute of Technology Foundation. L.‐Y.Y. acknowledges funds from the Taiwan Ministry of Education Government Scholarship to Study Abroad program. This research was supported in part through research cyberinfrastructure resources and services provided by the Partnership for an Advanced Computing Environment (PACE) at the Georgia Institute of Technology, Atlanta, Georgia, USA.</p></section><section id="notes1"><p>

Yang L‐Y, Hicks DJ, Russo PS, McShan AC. 
AlphaFold modeling uncovers global structural features of class I and class II fungal hydrophobins. Protein Science. 2025;34(9):e70279. 10.1002/pro.70279

</p>
<section id="pro70279-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="pro70279-note-0001"><p>
<strong>Review Editor:</strong> Nir Ben‐Tal</p></div></div></section></section><section id="pro70279-sec-0024"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All AlphaFold models, scripts used to generate the data, and source data are freely provided on GitHub at <a href="https://github.com/mcshanlab/Yang_hydrophobins_2025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/mcshanlab/Yang_hydrophobins_2025</a>.</p></section><section id="pro70279-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70279-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70279-bib-0001">
<cite>
Abramson J, Adler J, Dunger J, Evans R, Green T, Pritzel A, et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Nature. 2024;630:493–500.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203&amp;author=J%20Abramson&amp;author=J%20Adler&amp;author=J%20Dunger&amp;author=R%20Evans&amp;author=T%20Green&amp;volume=630&amp;publication_year=2024&amp;pages=493-500&amp;pmid=38718835&amp;doi=10.1038/s41586-024-07487-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0002">
<cite>
Agarwal V, McShan AC. The power and pitfalls of AlphaFold2 for structure prediction beyond rigid globular proteins. Nat Chem Biol. 2024;20:950–959.
</cite> [<a href="https://doi.org/10.1038/s41589-024-01638-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11956457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38907110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Chem%20Biol&amp;title=The%20power%20and%20pitfalls%20of%20AlphaFold2%20for%20structure%20prediction%20beyond%20rigid%20globular%20proteins&amp;author=V%20Agarwal&amp;author=AC%20McShan&amp;volume=20&amp;publication_year=2024&amp;pages=950-959&amp;pmid=38907110&amp;doi=10.1038/s41589-024-01638-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0003">
<cite>
Aimanianda V, Bayry J, Bozza S, Kniemeyer O, Perruccio K, Elluru SR, et al. Surface hydrophobin prevents immune recognition of airborne fungal spores. Nature. 2009;460:1117–1121.
</cite> [<a href="https://doi.org/10.1038/nature08264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19713928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Surface%20hydrophobin%20prevents%20immune%20recognition%20of%20airborne%20fungal%20spores&amp;author=V%20Aimanianda&amp;author=J%20Bayry&amp;author=S%20Bozza&amp;author=O%20Kniemeyer&amp;author=K%20Perruccio&amp;volume=460&amp;publication_year=2009&amp;pages=1117-1121&amp;pmid=19713928&amp;doi=10.1038/nature08264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0004">
<cite>
Akdel M, Pires DEV, Pardo EP, Jänes J, Zalevsky AO, Mészáros B, et al. A structural biology community assessment of AlphaFold2 applications. Nat Struct Mol Biol. 2022;29:1056–1067.
</cite> [<a href="https://doi.org/10.1038/s41594-022-00849-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9663297/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36344848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Mol%20Biol&amp;title=A%20structural%20biology%20community%20assessment%20of%20AlphaFold2%20applications&amp;author=M%20Akdel&amp;author=DEV%20Pires&amp;author=EP%20Pardo&amp;author=J%20J%C3%A4nes&amp;author=AO%20Zalevsky&amp;volume=29&amp;publication_year=2022&amp;pages=1056-1067&amp;pmid=36344848&amp;doi=10.1038/s41594-022-00849-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0005">
<cite>
Alford RF, Leaver‐Fay A, Jeliazkov JR, O'Meara MJ, DiMaio FP, Park H, et al. The Rosetta all‐atom energy function for macromolecular modeling and design. J Chem Theory Comput. 2017;13:3031–3048.
</cite> [<a href="https://doi.org/10.1021/acs.jctc.7b00125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5717763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28430426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=The%20Rosetta%20all%E2%80%90atom%20energy%20function%20for%20macromolecular%20modeling%20and%20design&amp;author=RF%20Alford&amp;author=A%20Leaver%E2%80%90Fay&amp;author=JR%20Jeliazkov&amp;author=MJ%20O'Meara&amp;author=FP%20DiMaio&amp;volume=13&amp;publication_year=2017&amp;pages=3031-3048&amp;pmid=28430426&amp;doi=10.1021/acs.jctc.7b00125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0007">
<cite>
Arnaouteli S, Bamford NC, Brandani GB, Morris RJ, Schor M, Carrington JT, et al. Lateral interactions govern self‐assembly of the bacterial biofilm matrix protein BslA. Proc Natl Acad Sci USA. 2023;120:e2312022120.
</cite> [<a href="https://doi.org/10.1073/pnas.2312022120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7615278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37903266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Lateral%20interactions%20govern%20self%E2%80%90assembly%20of%20the%20bacterial%20biofilm%20matrix%20protein%20BslA&amp;author=S%20Arnaouteli&amp;author=NC%20Bamford&amp;author=GB%20Brandani&amp;author=RJ%20Morris&amp;author=M%20Schor&amp;volume=120&amp;publication_year=2023&amp;pages=e2312022120&amp;pmid=37903266&amp;doi=10.1073/pnas.2312022120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0008">
<cite>
Askolin S, Linder M, Scholtmeijer K, Tenkanen M, Penttilä M, de Vocht ML, et al. Interaction and comparison of a class I hydrophobin from Schizophyllum commune and class II hydrophobins from Trichoderma reesei. Biomacromolecules. 2006;7:1295–1301.
</cite> [<a href="https://doi.org/10.1021/bm050676s" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16602752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomacromolecules&amp;title=Interaction%20and%20comparison%20of%20a%20class%20I%20hydrophobin%20from%20Schizophyllum%20commune%20and%20class%20II%20hydrophobins%20from%20Trichoderma%20reesei&amp;author=S%20Askolin&amp;author=M%20Linder&amp;author=K%20Scholtmeijer&amp;author=M%20Tenkanen&amp;author=M%20Penttil%C3%A4&amp;volume=7&amp;publication_year=2006&amp;pages=1295-1301&amp;pmid=16602752&amp;doi=10.1021/bm050676s&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0009">
<cite>
Bagar T, Altenbach K, Read ND, Bencina M. Live‐cell imaging and measurement of intracellular pH in filamentous fungi using a genetically encoded ratiometric probe. Eukaryot Cell. 2009;8:703–712.
</cite> [<a href="https://doi.org/10.1128/EC.00333-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2681602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19286983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&amp;title=Live%E2%80%90cell%20imaging%20and%20measurement%20of%20intracellular%20pH%20in%20filamentous%20fungi%20using%20a%20genetically%20encoded%20ratiometric%20probe&amp;author=T%20Bagar&amp;author=K%20Altenbach&amp;author=ND%20Read&amp;author=M%20Bencina&amp;volume=8&amp;publication_year=2009&amp;pages=703-712&amp;pmid=19286983&amp;doi=10.1128/EC.00333-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0010">
<cite>
Ball SR, Kwan AH, Sunde M. Hydrophobin rodlets on the fungal Cell Wall. Curr Top Microbiol Immunol. 2020;425:29–51.
</cite> [<a href="https://doi.org/10.1007/82_2019_186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31875265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Top%20Microbiol%20Immunol&amp;title=Hydrophobin%20rodlets%20on%20the%20fungal%20Cell%20Wall&amp;author=SR%20Ball&amp;author=AH%20Kwan&amp;author=M%20Sunde&amp;volume=425&amp;publication_year=2020&amp;pages=29-51&amp;pmid=31875265&amp;doi=10.1007/82_2019_186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0011">
<cite>
Ball SR, Pham CLL, Lo V, Morris VK, Kwan AH, Sunde M. Formation of amphipathic amyloid monolayers from fungal Hydrophobin proteins. Methods Mol Biol. 2020;2073:55–72.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9869-2_4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31612436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=Formation%20of%20amphipathic%20amyloid%20monolayers%20from%20fungal%20Hydrophobin%20proteins&amp;author=SR%20Ball&amp;author=CLL%20Pham&amp;author=V%20Lo&amp;author=VK%20Morris&amp;author=AH%20Kwan&amp;volume=2073&amp;publication_year=2020&amp;pages=55-72&amp;pmid=31612436&amp;doi=10.1007/978-1-4939-9869-2_4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0012">
<cite>
Barajas JF, Finzel K, Valentic TR, Shakya G, Gamarra N, Martinez D, et al. Structural and biochemical analysis of protein‐protein interactions between the acyl‐carrier protein and product template domain. Angew Chem Int Ed Engl. 2016;55:13005–13009.
</cite> [<a href="https://doi.org/10.1002/anie.201605401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5161458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27653519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew%20Chem%20Int%20Ed%20Engl&amp;title=Structural%20and%20biochemical%20analysis%20of%20protein%E2%80%90protein%20interactions%20between%20the%20acyl%E2%80%90carrier%20protein%20and%20product%20template%20domain&amp;author=JF%20Barajas&amp;author=K%20Finzel&amp;author=TR%20Valentic&amp;author=G%20Shakya&amp;author=N%20Gamarra&amp;volume=55&amp;publication_year=2016&amp;pages=13005-13009&amp;pmid=27653519&amp;doi=10.1002/anie.201605401&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0013">
<cite>
Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, et al. The protein data Bank. Nucleic Acids Res. 2000;28:235–242.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20protein%20data%20Bank&amp;author=HM%20Berman&amp;author=J%20Westbrook&amp;author=Z%20Feng&amp;author=G%20Gilliland&amp;author=TN%20Bhat&amp;volume=28&amp;publication_year=2000&amp;pages=235-242&amp;pmid=10592235&amp;doi=10.1093/nar/28.1.235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0014">
<cite>
Biancalana M, Makabe K, Koide A, Koide S. Molecular mechanism of thioflavin‐T binding to the surface of β‐rich peptide self‐assemblies. J Mol Biol. 2009;385:1052–1063.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2008.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2664162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19038267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Molecular%20mechanism%20of%20thioflavin%E2%80%90T%20binding%20to%20the%20surface%20of%20%CE%B2%E2%80%90rich%20peptide%20self%E2%80%90assemblies&amp;author=M%20Biancalana&amp;author=K%20Makabe&amp;author=A%20Koide&amp;author=S%20Koide&amp;volume=385&amp;publication_year=2009&amp;pages=1052-1063&amp;pmid=19038267&amp;doi=10.1016/j.jmb.2008.11.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0015">
<cite>
Bjellqvist B, Hughes GJ, Pasquali C, Paquet N, Ravier F, Sanchez JC, et al. The focusing positions of polypeptides in immobilized pH gradients can be predicted from their amino acid sequences. Electrophoresis. 1993;14:1023–1031.
</cite> [<a href="https://doi.org/10.1002/elps.11501401163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8125050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Electrophoresis&amp;title=The%20focusing%20positions%20of%20polypeptides%20in%20immobilized%20pH%20gradients%20can%20be%20predicted%20from%20their%20amino%20acid%20sequences&amp;author=B%20Bjellqvist&amp;author=GJ%20Hughes&amp;author=C%20Pasquali&amp;author=N%20Paquet&amp;author=F%20Ravier&amp;volume=14&amp;publication_year=1993&amp;pages=1023-1031&amp;pmid=8125050&amp;doi=10.1002/elps.11501401163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0016">
<cite>
Booker GW, Gout I, Downing AK, Driscoll PC, Boyd J, Waterfield MD, et al. Solution structure and ligand‐binding site of the SH3 domain of the p85 alpha subunit of phosphatidylinositol 3‐kinase. Cell. 1993;73:813–822.
</cite> [<a href="https://doi.org/10.1016/0092-8674(93)90259-s" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7684655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Solution%20structure%20and%20ligand%E2%80%90binding%20site%20of%20the%20SH3%20domain%20of%20the%20p85%20alpha%20subunit%20of%20phosphatidylinositol%203%E2%80%90kinase&amp;author=GW%20Booker&amp;author=I%20Gout&amp;author=AK%20Downing&amp;author=PC%20Driscoll&amp;author=J%20Boyd&amp;volume=73&amp;publication_year=1993&amp;pages=813-822&amp;pmid=7684655&amp;doi=10.1016/0092-8674(93)90259-s&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0017">
<cite>
Bornot A, Etchebest C, de Brevern AG. A new prediction strategy for long local protein structures using an original description. Proteins. 2009;76:570–587.
</cite> [<a href="https://doi.org/10.1002/prot.22370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19241475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=A%20new%20prediction%20strategy%20for%20long%20local%20protein%20structures%20using%20an%20original%20description&amp;author=A%20Bornot&amp;author=C%20Etchebest&amp;author=AG%20de%20Brevern&amp;volume=76&amp;publication_year=2009&amp;pages=570-587&amp;pmid=19241475&amp;doi=10.1002/prot.22370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0018">
<cite>
Bouqellah NA, Farag PF. In silico evaluation, phylogenetic analysis, and structural modeling of the class II Hydrophobin family from different fungal phytopathogens. Microorganisms. 2023;11:2632.
</cite> [<a href="https://doi.org/10.3390/microorganisms11112632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10672791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38004644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=In%20silico%20evaluation,%20phylogenetic%20analysis,%20and%20structural%20modeling%20of%20the%20class%20II%20Hydrophobin%20family%20from%20different%20fungal%20phytopathogens&amp;author=NA%20Bouqellah&amp;author=PF%20Farag&amp;volume=11&amp;publication_year=2023&amp;pages=2632&amp;pmid=38004644&amp;doi=10.3390/microorganisms11112632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0019">
<cite>
Bowman GR. AlphaFold and protein folding: not dead yet! The frontier is conformational ensembles. Annu Rev Biomed Data Sci. 2024;7:51–57.
</cite> [<a href="https://doi.org/10.1146/annurev-biodatasci-102423-011435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11892350/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38603560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biomed%20Data%20Sci&amp;title=AlphaFold%20and%20protein%20folding:%20not%20dead%20yet!%20The%20frontier%20is%20conformational%20ensembles&amp;author=GR%20Bowman&amp;volume=7&amp;publication_year=2024&amp;pages=51-57&amp;pmid=38603560&amp;doi=10.1146/annurev-biodatasci-102423-011435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0020">
<cite>
Cai F, Zhao Z, Gao R, Chen P, Ding M, Jiang S, et al. The pleiotropic functions of intracellular hydrophobins in aerial hyphae and fungal spores. PLoS Genet. 2021;17:e1009924.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8635391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34788288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&amp;title=The%20pleiotropic%20functions%20of%20intracellular%20hydrophobins%20in%20aerial%20hyphae%20and%20fungal%20spores&amp;author=F%20Cai&amp;author=Z%20Zhao&amp;author=R%20Gao&amp;author=P%20Chen&amp;author=M%20Ding&amp;volume=17&amp;publication_year=2021&amp;pages=e1009924&amp;pmid=34788288&amp;doi=10.1371/journal.pgen.1009924&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0021">
<cite>
Capella‐Gutiérrez S, Silla‐Martínez JM, Gabaldón T. trimAl: a tool for automated alignment trimming in large‐scale phylogenetic analyses. Bioinformatics. 2009;25:1972–1973.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=trimAl:%20a%20tool%20for%20automated%20alignment%20trimming%20in%20large%E2%80%90scale%20phylogenetic%20analyses&amp;author=S%20Capella%E2%80%90Guti%C3%A9rrez&amp;author=JM%20Silla%E2%80%90Mart%C3%ADnez&amp;author=T%20Gabald%C3%B3n&amp;volume=25&amp;publication_year=2009&amp;pages=1972-1973&amp;pmid=19505945&amp;doi=10.1093/bioinformatics/btp348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0022">
<cite>
Chakravarty D, Porter LL. AlphaFold2 fails to predict protein fold switching. Protein Sci. 2022;31:e4353.
</cite> [<a href="https://doi.org/10.1002/pro.4353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9134877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35634782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=AlphaFold2%20fails%20to%20predict%20protein%20fold%20switching&amp;author=D%20Chakravarty&amp;author=LL%20Porter&amp;volume=31&amp;publication_year=2022&amp;pages=e4353&amp;pmid=35634782&amp;doi=10.1002/pro.4353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0023">
<cite>
Chang HJ, Choi H, Na S. Predicting the self‐assembly film structure of class II hydrophobin NC2 and estimating its structural characteristics. Colloids Surf B Biointerfaces. 2020;195:111269.
</cite> [<a href="https://doi.org/10.1016/j.colsurfb.2020.111269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32739772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Colloids%20Surf%20B%20Biointerfaces&amp;title=Predicting%20the%20self%E2%80%90assembly%20film%20structure%20of%20class%20II%20hydrophobin%20NC2%20and%20estimating%20its%20structural%20characteristics&amp;author=HJ%20Chang&amp;author=H%20Choi&amp;author=S%20Na&amp;volume=195&amp;publication_year=2020&amp;pages=111269&amp;pmid=32739772&amp;doi=10.1016/j.colsurfb.2020.111269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0024">
<cite>
Cheung DL. Molecular simulation of hydrophobin adsorption at an oil‐water interface. Langmuir. 2012;28:8730–8736.
</cite> [<a href="https://doi.org/10.1021/la300777q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22591377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Langmuir&amp;title=Molecular%20simulation%20of%20hydrophobin%20adsorption%20at%20an%20oil%E2%80%90water%20interface&amp;author=DL%20Cheung&amp;volume=28&amp;publication_year=2012&amp;pages=8730-8736&amp;pmid=22591377&amp;doi=10.1021/la300777q&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0025">
<cite>
Chinnam NB, Syed A, Hura GL, Hammel M, Tainer JA, Tsutakawa SE. Combining small angle X‐ray scattering (SAXS) with protein structure predictions to characterize conformations in solution. Methods Enzymol. 2023;678:351–376.
</cite> [<a href="https://doi.org/10.1016/bs.mie.2022.09.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10132260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36641214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol&amp;title=Combining%20small%20angle%20X%E2%80%90ray%20scattering%20(SAXS)%20with%20protein%20structure%20predictions%20to%20characterize%20conformations%20in%20solution&amp;author=NB%20Chinnam&amp;author=A%20Syed&amp;author=GL%20Hura&amp;author=M%20Hammel&amp;author=JA%20Tainer&amp;volume=678&amp;publication_year=2023&amp;pages=351-376&amp;pmid=36641214&amp;doi=10.1016/bs.mie.2022.09.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0026">
<cite>
de Simone A, Kitchen C, Kwan AH, Sunde M, Dobson CM, Frenkel D. Intrinsic disorder modulates protein self‐assembly and aggregation. Proc Natl Acad Sci USA. 2012;109:6951–6956.
</cite> [<a href="https://doi.org/10.1073/pnas.1118048109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3344965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22509003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Intrinsic%20disorder%20modulates%20protein%20self%E2%80%90assembly%20and%20aggregation&amp;author=A%20de%20Simone&amp;author=C%20Kitchen&amp;author=AH%20Kwan&amp;author=M%20Sunde&amp;author=CM%20Dobson&amp;volume=109&amp;publication_year=2012&amp;pages=6951-6956&amp;pmid=22509003&amp;doi=10.1073/pnas.1118048109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0027">
<cite>
de Vocht ML, Reviakine I, Ulrich W‐P, Bergsma‐Schutter W, Wösten HAB, Vogel H, et al. Self‐assembly of the hydrophobin SC3 proceeds via two structural intermediates. Protein Sci. 2002;11:1199–1205.
</cite> [<a href="https://doi.org/10.1110/ps.4540102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2373556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11967376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Self%E2%80%90assembly%20of%20the%20hydrophobin%20SC3%20proceeds%20via%20two%20structural%20intermediates&amp;author=ML%20de%20Vocht&amp;author=I%20Reviakine&amp;author=W%E2%80%90P%20Ulrich&amp;author=W%20Bergsma%E2%80%90Schutter&amp;author=HAB%20W%C3%B6sten&amp;volume=11&amp;publication_year=2002&amp;pages=1199-1205&amp;pmid=11967376&amp;doi=10.1110/ps.4540102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0028">
<cite>
de Vocht ML, Reviakine I, Wösten HA, Brisson A, Wessels JG, Robillard GT. Structural and functional role of the disulfide bridges in the hydrophobin SC3. J Biol Chem. 2000;275:28428–28432.
</cite> [<a href="https://doi.org/10.1074/jbc.M000691200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10829014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structural%20and%20functional%20role%20of%20the%20disulfide%20bridges%20in%20the%20hydrophobin%20SC3&amp;author=ML%20de%20Vocht&amp;author=I%20Reviakine&amp;author=HA%20W%C3%B6sten&amp;author=A%20Brisson&amp;author=JG%20Wessels&amp;volume=275&amp;publication_year=2000&amp;pages=28428-28432&amp;pmid=10829014&amp;doi=10.1074/jbc.M000691200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0029">
<cite>
de Vocht ML, Scholtmeijer K, van der Vegte EW, de Vries OMH, Sonveaux N, Wösten HAB, et al. Structural characterization of the Hydrophobin SC3, as a monomer and after self‐assembly at hydrophobic/hydrophilic interfaces. Biophys J. 1998;74:2059–2068.
</cite> [<a href="https://doi.org/10.1016/s0006-3495(98)77912-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1299546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9545064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Structural%20characterization%20of%20the%20Hydrophobin%20SC3,%20as%20a%20monomer%20and%20after%20self%E2%80%90assembly%20at%20hydrophobic/hydrophilic%20interfaces&amp;author=ML%20de%20Vocht&amp;author=K%20Scholtmeijer&amp;author=EW%20van%20der%20Vegte&amp;author=OMH%20de%20Vries&amp;author=N%20Sonveaux&amp;volume=74&amp;publication_year=1998&amp;pages=2059-2068&amp;pmid=9545064&amp;doi=10.1016/s0006-3495(98)77912-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0030">
<cite>
De Vries OM, Moore S, Arntz C, Wessels JG, Tudzynski P. Identification and characterization of a tri‐partite hydrophobin from Claviceps fusiformis. A novel type of class II hydrophobin. Eur J Biochem. 1999;262:377–385.
</cite> [<a href="https://doi.org/10.1046/j.1432-1327.1999.00387.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10336622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&amp;title=Identification%20and%20characterization%20of%20a%20tri%E2%80%90partite%20hydrophobin%20from%20Claviceps%20fusiformis.%20A%20novel%20type%20of%20class%20II%20hydrophobin&amp;author=OM%20De%20Vries&amp;author=S%20Moore&amp;author=C%20Arntz&amp;author=JG%20Wessels&amp;author=P%20Tudzynski&amp;volume=262&amp;publication_year=1999&amp;pages=377-385&amp;pmid=10336622&amp;doi=10.1046/j.1432-1327.1999.00387.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0031">
<cite>
de Vries OMH, Fekkes MP, Wösten HAB, Wessels JGH. Insoluble hydrophobin complexes in the walls of Schizophyllum commune and other filamentous fungi. Arch Microbiol. 1993;159:330–335.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Arch%20Microbiol&amp;title=Insoluble%20hydrophobin%20complexes%20in%20the%20walls%20of%20Schizophyllum%20commune%20and%20other%20filamentous%20fungi&amp;author=OMH%20de%20Vries&amp;author=MP%20Fekkes&amp;author=HAB%20W%C3%B6sten&amp;author=JGH%20Wessels&amp;volume=159&amp;publication_year=1993&amp;pages=330-335&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0032">
<cite>
Dempsey GP, Beever RE. Electron microscopy of the rodlet layer of Neurospora crassa conidia. J Bacteriol. 1979;140:1050–1062.
</cite> [<a href="https://doi.org/10.1128/jb.140.3.1050-1062.1979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC216752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/93107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Electron%20microscopy%20of%20the%20rodlet%20layer%20of%20Neurospora%20crassa%20conidia&amp;author=GP%20Dempsey&amp;author=RE%20Beever&amp;volume=140&amp;publication_year=1979&amp;pages=1050-1062&amp;pmid=93107&amp;doi=10.1128/jb.140.3.1050-1062.1979&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0033"><cite>
Discovery C, Boitreaud J, Dent J, McPartlon M, Meier J, Reis V, et al. Chai‐1: decoding the molecular interactions of life.
2024. 
</cite></li>
<li id="pro70279-bib-0034">
<cite>
Dolinsky TJ, Nielsen JE, McCammon JA, Baker NA. PDB2PQR: an automated pipeline for the setup of Poisson–Boltzmann electrostatics calculations. Nucleic Acids Res. 2004;32:W665–W667.
</cite> [<a href="https://doi.org/10.1093/nar/gkh381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC441519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15215472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=PDB2PQR:%20an%20automated%20pipeline%20for%20the%20setup%20of%20Poisson%E2%80%93Boltzmann%20electrostatics%20calculations&amp;author=TJ%20Dolinsky&amp;author=JE%20Nielsen&amp;author=JA%20McCammon&amp;author=NA%20Baker&amp;volume=32&amp;publication_year=2004&amp;pages=W665-W667&amp;pmid=15215472&amp;doi=10.1093/nar/gkh381&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0035">
<cite>
Evans R, O'Neill M, Pritzel A, Antropova N, Senior A, Green T, et al. Protein complex prediction with AlphaFold‐multimer. bioRxiv. 2022;2021.10.04.463034.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Protein%20complex%20prediction%20with%20AlphaFold%E2%80%90multimer&amp;author=R%20Evans&amp;author=M%20O'Neill&amp;author=A%20Pritzel&amp;author=N%20Antropova&amp;author=A%20Senior&amp;publication_year=2022&amp;pages=2021.10.04.463034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0036">
<cite>
Farmer R, Thomas CM, Winn PJ. Structure, function and dynamics in acyl carrier proteins. PLoS One. 2019;14:e0219435.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0219435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6619796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31291335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Structure,%20function%20and%20dynamics%20in%20acyl%20carrier%20proteins&amp;author=R%20Farmer&amp;author=CM%20Thomas&amp;author=PJ%20Winn&amp;volume=14&amp;publication_year=2019&amp;pages=e0219435&amp;pmid=31291335&amp;doi=10.1371/journal.pone.0219435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0037">
<cite>
Gallo M, Luti S, Baroni F, Baccelli I, Cilli EM, Cicchi C, et al. Plant defense elicitation by the Hydrophobin Cerato‐Ulmin and correlation with its structural features. Int J Mol Sci. 2023;24:2251.
</cite> [<a href="https://doi.org/10.3390/ijms24032251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9916430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36768573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Plant%20defense%20elicitation%20by%20the%20Hydrophobin%20Cerato%E2%80%90Ulmin%20and%20correlation%20with%20its%20structural%20features&amp;author=M%20Gallo&amp;author=S%20Luti&amp;author=F%20Baroni&amp;author=I%20Baccelli&amp;author=EM%20Cilli&amp;volume=24&amp;publication_year=2023&amp;pages=2251&amp;pmid=36768573&amp;doi=10.3390/ijms24032251&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0038">
<cite>
Gandier J‐A, Langelaan DN, Won A, O'Donnell K, Grondin JL, Spencer HL, et al. Characterization of a Basidiomycota hydrophobin reveals the structural basis for a high‐similarity class I subdivision. Sci Rep. 2017;7:45863.
</cite> [<a href="https://doi.org/10.1038/srep45863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5385502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28393921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Characterization%20of%20a%20Basidiomycota%20hydrophobin%20reveals%20the%20structural%20basis%20for%20a%20high%E2%80%90similarity%20class%20I%20subdivision&amp;author=J%E2%80%90A%20Gandier&amp;author=DN%20Langelaan&amp;author=A%20Won&amp;author=K%20O'Donnell&amp;author=JL%20Grondin&amp;volume=7&amp;publication_year=2017&amp;pages=45863&amp;pmid=28393921&amp;doi=10.1038/srep45863&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0039"><cite>
Gilchrist CLM, Mirdita M, Steinegger M. Multiple Protein Structure Alignment at Scale with FoldMason. <em>bioRxiv</em>; 2024.08.01.606130.</cite></li>
<li id="pro70279-bib-0040">
<cite>
Graña‐Montes R, Marinelli P, Reverter D, Ventura S. N‐terminal protein tails act as aggregation protective entropic bristles: the SUMO case. Biomacromolecules. 2014;15:1194–1203.
</cite> [<a href="https://doi.org/10.1021/bm401776z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24564702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomacromolecules&amp;title=N%E2%80%90terminal%20protein%20tails%20act%20as%20aggregation%20protective%20entropic%20bristles:%20the%20SUMO%20case&amp;author=R%20Gra%C3%B1a%E2%80%90Montes&amp;author=P%20Marinelli&amp;author=D%20Reverter&amp;author=S%20Ventura&amp;volume=15&amp;publication_year=2014&amp;pages=1194-1203&amp;pmid=24564702&amp;doi=10.1021/bm401776z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0041">
<cite>
Gravagnuolo AM, Longobardi S, Luchini A, Appavou M‐S, de Stefano L, Notomista E, et al. Class I Hydrophobin Vmh2 adopts atypical mechanisms to self‐assemble into functional amyloid fibrils. Biomacromolecules. 2016;17:954–964.
</cite> [<a href="https://doi.org/10.1021/acs.biomac.5b01632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomacromolecules&amp;title=Class%20I%20Hydrophobin%20Vmh2%20adopts%20atypical%20mechanisms%20to%20self%E2%80%90assemble%20into%20functional%20amyloid%20fibrils&amp;author=AM%20Gravagnuolo&amp;author=S%20Longobardi&amp;author=A%20Luchini&amp;author=M%E2%80%90S%20Appavou&amp;author=L%20de%20Stefano&amp;volume=17&amp;publication_year=2016&amp;pages=954-964&amp;pmid=26828412&amp;doi=10.1021/acs.biomac.5b01632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0042">
<cite>
Hakanpää J, Linder M, Popov A, Schmidt A, Rouvinen J. Hydrophobin HFBII in detail: ultrahigh‐resolution structure at 0.75 a. Acta Crystallogr D Biol Crystallogr. 2006;62:356–367.
</cite> [<a href="https://doi.org/10.1107/S0907444906000862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16552136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Hydrophobin%20HFBII%20in%20detail:%20ultrahigh%E2%80%90resolution%20structure%20at%200.75%20a&amp;author=J%20Hakanp%C3%A4%C3%A4&amp;author=M%20Linder&amp;author=A%20Popov&amp;author=A%20Schmidt&amp;author=J%20Rouvinen&amp;volume=62&amp;publication_year=2006&amp;pages=356-367&amp;pmid=16552136&amp;doi=10.1107/S0907444906000862&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0043">
<cite>
Hakanpää J, Paananen A, Askolin S, Nakari‐Setälä T, Parkkinen T, Penttilä M, et al. Atomic resolution structure of the HFBII hydrophobin, a self‐assembling amphiphile. J Biol Chem. 2004;279:534–539.
</cite> [<a href="https://doi.org/10.1074/jbc.M309650200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14555650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Atomic%20resolution%20structure%20of%20the%20HFBII%20hydrophobin,%20a%20self%E2%80%90assembling%20amphiphile&amp;author=J%20Hakanp%C3%A4%C3%A4&amp;author=A%20Paananen&amp;author=S%20Askolin&amp;author=T%20Nakari%E2%80%90Set%C3%A4l%C3%A4&amp;author=T%20Parkkinen&amp;volume=279&amp;publication_year=2004&amp;pages=534-539&amp;pmid=14555650&amp;doi=10.1074/jbc.M309650200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0044">
<cite>
Hakanpää J, Szilvay GR, Kaljunen H, Maksimainen M, Linder M, Rouvinen J. Two crystal structures of Trichoderma reesei hydrophobin HFBI‐‐the structure of a protein amphiphile with and without detergent interaction. Protein Sci. 2006;15:2129–2140.
</cite> [<a href="https://doi.org/10.1110/ps.062326706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2242604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16882996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Two%20crystal%20structures%20of%20Trichoderma%20reesei%20hydrophobin%20HFBI%E2%80%90%E2%80%90the%20structure%20of%20a%20protein%20amphiphile%20with%20and%20without%20detergent%20interaction&amp;author=J%20Hakanp%C3%A4%C3%A4&amp;author=GR%20Szilvay&amp;author=H%20Kaljunen&amp;author=M%20Maksimainen&amp;author=M%20Linder&amp;volume=15&amp;publication_year=2006&amp;pages=2129-2140&amp;pmid=16882996&amp;doi=10.1110/ps.062326706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0045">
<cite>
Hobley L, Ostrowski A, Rao FV, Bromley KM, Porter M, Prescott AR, et al. BslA is a self‐assembling bacterial hydrophobin that coats the Bacillus subtilis biofilm. Proc Natl Acad Sci USA. 2013;110:13600–13605.
</cite> [<a href="https://doi.org/10.1073/pnas.1306390110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3746881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23904481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=BslA%20is%20a%20self%E2%80%90assembling%20bacterial%20hydrophobin%20that%20coats%20the%20Bacillus%20subtilis%20biofilm&amp;author=L%20Hobley&amp;author=A%20Ostrowski&amp;author=FV%20Rao&amp;author=KM%20Bromley&amp;author=M%20Porter&amp;volume=110&amp;publication_year=2013&amp;pages=13600-13605&amp;pmid=23904481&amp;doi=10.1073/pnas.1306390110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0046">
<cite>
Howie AJ, Brewer DB, Howell D, Jones AP. Physical basis of colors seen in Congo red‐stained amyloid in polarized light. Lab Investig. 2008;88:232–242.
</cite> [<a href="https://doi.org/10.1038/labinvest.3700714" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18166974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Investig&amp;title=Physical%20basis%20of%20colors%20seen%20in%20Congo%20red%E2%80%90stained%20amyloid%20in%20polarized%20light&amp;author=AJ%20Howie&amp;author=DB%20Brewer&amp;author=D%20Howell&amp;author=AP%20Jones&amp;volume=88&amp;publication_year=2008&amp;pages=232-242&amp;pmid=18166974&amp;doi=10.1038/labinvest.3700714&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0047">
<cite>
Huang YJ, Montelione GT. Hidden structural states of proteins revealed by conformer selection with AlphaFold‐NMR. bioRxiv. 2024;2024.06.26.600902.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Hidden%20structural%20states%20of%20proteins%20revealed%20by%20conformer%20selection%20with%20AlphaFold%E2%80%90NMR&amp;author=YJ%20Huang&amp;author=GT%20Montelione&amp;publication_year=2024&amp;pages=2024.06.26.600902&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0048">
<cite>
Ibrahim AY, Khaodeuanepheng NP, Amarasekara DL, Correia JJ, Lewis KA, Fitzkee NC, et al. Intrinsically disordered regions that drive phase separation form a robustly distinct protein class. J Biol Chem. 2023;299:102801.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2022.102801" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9860499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36528065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Intrinsically%20disordered%20regions%20that%20drive%20phase%20separation%20form%20a%20robustly%20distinct%20protein%20class&amp;author=AY%20Ibrahim&amp;author=NP%20Khaodeuanepheng&amp;author=DL%20Amarasekara&amp;author=JJ%20Correia&amp;author=KA%20Lewis&amp;volume=299&amp;publication_year=2023&amp;pages=102801&amp;pmid=36528065&amp;doi=10.1016/j.jbc.2022.102801&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0049">
<cite>
Jensen BG, Andersen MR, Pedersen MH, Frisvad JC, Søndergaard I. Hydrophobins from aspergillus species cannot be clearly divided into two classes. BMC Res Notes. 2010;3:344.
</cite> [<a href="https://doi.org/10.1186/1756-0500-3-344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3020181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21182770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Res%20Notes&amp;title=Hydrophobins%20from%20aspergillus%20species%20cannot%20be%20clearly%20divided%20into%20two%20classes&amp;author=BG%20Jensen&amp;author=MR%20Andersen&amp;author=MH%20Pedersen&amp;author=JC%20Frisvad&amp;author=I%20S%C3%B8ndergaard&amp;volume=3&amp;publication_year=2010&amp;pages=344&amp;pmid=21182770&amp;doi=10.1186/1756-0500-3-344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0050">
<cite>
Jin Y‐T, Jin T‐Y, Zhang Z‐L, Ye Y‐N, Deng Z, Wang J, et al. Quantitative elucidation of associations between nucleotide identity and physicochemical properties of amino acids and the functional insight. Comput Struct Biotechnol J. 2021;19:4042–4048.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2021.07.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8346530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34527183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Struct%20Biotechnol%20J&amp;title=Quantitative%20elucidation%20of%20associations%20between%20nucleotide%20identity%20and%20physicochemical%20properties%20of%20amino%20acids%20and%20the%20functional%20insight&amp;author=Y%E2%80%90T%20Jin&amp;author=T%E2%80%90Y%20Jin&amp;author=Z%E2%80%90L%20Zhang&amp;author=Y%E2%80%90N%20Ye&amp;author=Z%20Deng&amp;volume=19&amp;publication_year=2021&amp;pages=4042-4048&amp;pmid=34527183&amp;doi=10.1016/j.csbj.2021.07.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0051">
<cite>
Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596:583–589.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=J%20Jumper&amp;author=R%20Evans&amp;author=A%20Pritzel&amp;author=T%20Green&amp;author=M%20Figurnov&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0052">
<cite>
Jurrus E, Engel D, Star K, Monson K, Brandi J, Felberg LE, et al. Improvements to the APBS biomolecular solvation software suite. Protein Sci. 2018;27:112–128.
</cite> [<a href="https://doi.org/10.1002/pro.3280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28836357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Improvements%20to%20the%20APBS%20biomolecular%20solvation%20software%20suite&amp;author=E%20Jurrus&amp;author=D%20Engel&amp;author=K%20Star&amp;author=K%20Monson&amp;author=J%20Brandi&amp;volume=27&amp;publication_year=2018&amp;pages=112-128&amp;pmid=28836357&amp;doi=10.1002/pro.3280&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0053">
<cite>
Kabsch W. A solution for the best rotation to relate two sets of vectors. Acta Crystallogr Sect A. 1976;32:922–923.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20Sect%20A&amp;title=A%20solution%20for%20the%20best%20rotation%20to%20relate%20two%20sets%20of%20vectors&amp;author=W%20Kabsch&amp;volume=32&amp;publication_year=1976&amp;pages=922-923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0054">
<cite>
Kallio JM, Linder MB, Rouvinen J. Crystal structures of hydrophobin HFBII in the presence of detergent implicate the formation of fibrils and monolayer films. J Biol Chem. 2007;282:28733–28739.
</cite> [<a href="https://doi.org/10.1074/jbc.M704238200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17636262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Crystal%20structures%20of%20hydrophobin%20HFBII%20in%20the%20presence%20of%20detergent%20implicate%20the%20formation%20of%20fibrils%20and%20monolayer%20films&amp;author=JM%20Kallio&amp;author=MB%20Linder&amp;author=J%20Rouvinen&amp;volume=282&amp;publication_year=2007&amp;pages=28733-28739&amp;pmid=17636262&amp;doi=10.1074/jbc.M704238200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0055">
<cite>
Kallio JM, Rouvinen J. Amphiphilic nanotubes in the crystal structure of a biosurfactant protein hydrophobin HFBII. Chem Commun. 2011;47:9843–9845.</cite> [<a href="https://doi.org/10.1039/c1cc13139g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21808803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Commun&amp;title=Amphiphilic%20nanotubes%20in%20the%20crystal%20structure%20of%20a%20biosurfactant%20protein%20hydrophobin%20HFBII&amp;author=JM%20Kallio&amp;author=J%20Rouvinen&amp;volume=47&amp;publication_year=2011&amp;pages=9843-9845&amp;pmid=21808803&amp;doi=10.1039/c1cc13139g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0056">
<cite>
Karadkar S, Tiwari A, Chaskar AC. Recent advancements in Janus nanoparticle‐based biosensing platforms. Int Nano Lett. 2023;13:93–115.
</cite> [<a href="https://doi.org/10.1007/s40089-022-00385-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9676883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36438713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20Nano%20Lett&amp;title=Recent%20advancements%20in%20Janus%20nanoparticle%E2%80%90based%20biosensing%20platforms&amp;author=S%20Karadkar&amp;author=A%20Tiwari&amp;author=AC%20Chaskar&amp;volume=13&amp;publication_year=2023&amp;pages=93-115&amp;pmid=36438713&amp;doi=10.1007/s40089-022-00385-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0057">
<cite>
Karlsson M, Stenlid J, Olson A. Two hydrophobin genes from the conifer pathogen Heterobasidion annosum are expressed in aerial hyphae. Mycologia. 2007;99:227–231.
</cite> [<a href="https://doi.org/10.3852/mycologia.99.2.227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17682775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Two%20hydrophobin%20genes%20from%20the%20conifer%20pathogen%20Heterobasidion%20annosum%20are%20expressed%20in%20aerial%20hyphae&amp;author=M%20Karlsson&amp;author=J%20Stenlid&amp;author=A%20Olson&amp;volume=99&amp;publication_year=2007&amp;pages=227-231&amp;pmid=17682775&amp;doi=10.3852/mycologia.99.2.227&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0058">
<cite>
Kenward C, Vergunst KL, Langelaan DN. Expression, purification, and refolding of diverse class IB hydrophobins. Protein Expr Purif. 2020;176:105732.
</cite> [<a href="https://doi.org/10.1016/j.pep.2020.105732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32866612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expr%20Purif&amp;title=Expression,%20purification,%20and%20refolding%20of%20diverse%20class%20IB%20hydrophobins&amp;author=C%20Kenward&amp;author=KL%20Vergunst&amp;author=DN%20Langelaan&amp;volume=176&amp;publication_year=2020&amp;pages=105732&amp;pmid=32866612&amp;doi=10.1016/j.pep.2020.105732&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0059">
<cite>
Kershaw MJ, Talbot NJ. Hydrophobins and repellents: proteins with fundamental roles in fungal morphogenesis. Fungal Genet Biol. 1998;23:18–33.
</cite> [<a href="https://doi.org/10.1006/fgbi.1997.1022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9501475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet%20Biol&amp;title=Hydrophobins%20and%20repellents:%20proteins%20with%20fundamental%20roles%20in%20fungal%20morphogenesis&amp;author=MJ%20Kershaw&amp;author=NJ%20Talbot&amp;volume=23&amp;publication_year=1998&amp;pages=18-33&amp;pmid=9501475&amp;doi=10.1006/fgbi.1997.1022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0060">
<cite>
Kershaw MJ, Thornton CR, Wakley GE, Talbot NJ. Four conserved intramolecular disulphide linkages are required for secretion and cell wall localization of a hydrophobin during fungal morphogenesis. Mol Microbiol. 2005;56:117–125.
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2005.04547.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15773983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=Four%20conserved%20intramolecular%20disulphide%20linkages%20are%20required%20for%20secretion%20and%20cell%20wall%20localization%20of%20a%20hydrophobin%20during%20fungal%20morphogenesis&amp;author=MJ%20Kershaw&amp;author=CR%20Thornton&amp;author=GE%20Wakley&amp;author=NJ%20Talbot&amp;volume=56&amp;publication_year=2005&amp;pages=117-125&amp;pmid=15773983&amp;doi=10.1111/j.1365-2958.2005.04547.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0061">
<cite>
Khalesi M, Gebruers K, Derdelinckx G. Recent advances in fungal Hydrophobin towards using in industry. Protein J. 2015;34:243–255.
</cite> [<a href="https://doi.org/10.1007/s10930-015-9621-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26208665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20J&amp;title=Recent%20advances%20in%20fungal%20Hydrophobin%20towards%20using%20in%20industry&amp;author=M%20Khalesi&amp;author=K%20Gebruers&amp;author=G%20Derdelinckx&amp;volume=34&amp;publication_year=2015&amp;pages=243-255&amp;pmid=26208665&amp;doi=10.1007/s10930-015-9621-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0062">
<cite>
Khatua S, Simal‐Gandara J, Acharya K. Myco‐remediation of plastic pollution: current knowledge and future prospects. Biodegradation. 2024;35:249–279.
</cite> [<a href="https://doi.org/10.1007/s10532-023-10053-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10950981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37665521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biodegradation&amp;title=Myco%E2%80%90remediation%20of%20plastic%20pollution:%20current%20knowledge%20and%20future%20prospects&amp;author=S%20Khatua&amp;author=J%20Simal%E2%80%90Gandara&amp;author=K%20Acharya&amp;volume=35&amp;publication_year=2024&amp;pages=249-279&amp;pmid=37665521&amp;doi=10.1007/s10532-023-10053-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0063">
<cite>
Kisko K, Szilvay GR, Vainio U, Linder MB, Serimaa R. Interactions of hydrophobin proteins in solution studied by small‐angle X‐ray scattering. Biophys J. 2008;94:198–206.
</cite> [<a href="https://doi.org/10.1529/biophysj.107.112359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2134873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17827247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Interactions%20of%20hydrophobin%20proteins%20in%20solution%20studied%20by%20small%E2%80%90angle%20X%E2%80%90ray%20scattering&amp;author=K%20Kisko&amp;author=GR%20Szilvay&amp;author=U%20Vainio&amp;author=MB%20Linder&amp;author=R%20Serimaa&amp;volume=94&amp;publication_year=2008&amp;pages=198-206&amp;pmid=17827247&amp;doi=10.1529/biophysj.107.112359&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0064">
<cite>
Kottmeier K, Ostermann K, Bley T, Rödel G. Hydrophobin signal sequence mediates efficient secretion of recombinant proteins in Pichia pastoris. Appl Microbiol Biotechnol. 2011;91:133–141.
</cite> [<a href="https://doi.org/10.1007/s00253-011-3246-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21484207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=Hydrophobin%20signal%20sequence%20mediates%20efficient%20secretion%20of%20recombinant%20proteins%20in%20Pichia%20pastoris&amp;author=K%20Kottmeier&amp;author=K%20Ostermann&amp;author=T%20Bley&amp;author=G%20R%C3%B6del&amp;volume=91&amp;publication_year=2011&amp;pages=133-141&amp;pmid=21484207&amp;doi=10.1007/s00253-011-3246-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0065">
<cite>
Krishna R, Wang J, Ahern W, Sturmfels P, Venkatesh P, Kalvet I, et al. Generalized biomolecular modeling and design with RoseTTAFold all‐atom. Science 384:eadl2528. 2024;384:eadl2528.
</cite> [<a href="https://doi.org/10.1126/science.adl2528" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38452047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Generalized%20biomolecular%20modeling%20and%20design%20with%20RoseTTAFold%20all%E2%80%90atom&amp;author=R%20Krishna&amp;author=J%20Wang&amp;author=W%20Ahern&amp;author=P%20Sturmfels&amp;author=P%20Venkatesh&amp;volume=384&amp;publication_year=2024&amp;pages=eadl2528&amp;pmid=38452047&amp;doi=10.1126/science.adl2528&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0066">
<cite>
Kubicek CP, Baker S, Gamauf C, Kenerley CM, Druzhinina IS. Purifying selection and birth‐and‐death evolution in the class II hydrophobin gene families of the ascomycete Trichoderma/Hypocrea. BMC Evol Biol. 2008;8:4.
</cite> [<a href="https://doi.org/10.1186/1471-2148-8-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2253510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18186925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol%20Biol&amp;title=Purifying%20selection%20and%20birth%E2%80%90and%E2%80%90death%20evolution%20in%20the%20class%20II%20hydrophobin%20gene%20families%20of%20the%20ascomycete%20Trichoderma/Hypocrea&amp;author=CP%20Kubicek&amp;author=S%20Baker&amp;author=C%20Gamauf&amp;author=CM%20Kenerley&amp;author=IS%20Druzhinina&amp;volume=8&amp;publication_year=2008&amp;pages=4&amp;pmid=18186925&amp;doi=10.1186/1471-2148-8-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0067">
<cite>
Kufareva I, Abagyan R. Methods of protein structure comparison. Methods Mol Biol. 2012;857:231–257.
</cite> [<a href="https://doi.org/10.1007/978-1-61779-588-6_10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4321859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22323224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=Methods%20of%20protein%20structure%20comparison&amp;author=I%20Kufareva&amp;author=R%20Abagyan&amp;volume=857&amp;publication_year=2012&amp;pages=231-257&amp;pmid=22323224&amp;doi=10.1007/978-1-61779-588-6_10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0068">
<cite>
Kurochkina N, Guha U. SH3 domains: modules of protein–protein interactions. Biophys Rev. 2012;5:29–39.
</cite> [<a href="https://doi.org/10.1007/s12551-012-0081-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5418429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28510178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20Rev&amp;title=SH3%20domains:%20modules%20of%20protein%E2%80%93protein%20interactions&amp;author=N%20Kurochkina&amp;author=U%20Guha&amp;volume=5&amp;publication_year=2012&amp;pages=29-39&amp;pmid=28510178&amp;doi=10.1007/s12551-012-0081-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0069">
<cite>
Kwan AH, Macindoe I, Vukasin PV, Morris VK, Kass I, Gupte R, et al. The Cys3‐Cys4 loop of the hydrophobin EAS is not required for rodlet formation and surface activity. J Mol Biol. 2008;382:708–720.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2008.07.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18674544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=The%20Cys3%E2%80%90Cys4%20loop%20of%20the%20hydrophobin%20EAS%20is%20not%20required%20for%20rodlet%20formation%20and%20surface%20activity&amp;author=AH%20Kwan&amp;author=I%20Macindoe&amp;author=PV%20Vukasin&amp;author=VK%20Morris&amp;author=I%20Kass&amp;volume=382&amp;publication_year=2008&amp;pages=708-720&amp;pmid=18674544&amp;doi=10.1016/j.jmb.2008.07.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0070">
<cite>
Kwan AHY, Winefield RD, Sunde M, Matthews JM, Haverkamp RG, Templeton MD, et al. Structural basis for rodlet assembly in fungal hydrophobins. Proc Natl Acad Sci USA. 2006;103:3621–3626.
</cite> [<a href="https://doi.org/10.1073/pnas.0505704103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1533775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16537446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Structural%20basis%20for%20rodlet%20assembly%20in%20fungal%20hydrophobins&amp;author=AHY%20Kwan&amp;author=RD%20Winefield&amp;author=M%20Sunde&amp;author=JM%20Matthews&amp;author=RG%20Haverkamp&amp;volume=103&amp;publication_year=2006&amp;pages=3621-3626&amp;pmid=16537446&amp;doi=10.1073/pnas.0505704103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0071">
<cite>
Kyte J, Doolittle RF. A simple method for displaying the hydropathic character of a protein. J Mol Biol. 1982;157:105–132.
</cite> [<a href="https://doi.org/10.1016/0022-2836(82)90515-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7108955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=A%20simple%20method%20for%20displaying%20the%20hydropathic%20character%20of%20a%20protein&amp;author=J%20Kyte&amp;author=RF%20Doolittle&amp;volume=157&amp;publication_year=1982&amp;pages=105-132&amp;pmid=7108955&amp;doi=10.1016/0022-2836(82)90515-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0072">
<cite>
Leman JK, Weitzner BD, Lewis SM, Adolf‐Bryfogle J, Alam N, Alford RF, et al. Macromolecular modeling and design in Rosetta: recent methods and frameworks. Nat Methods. 2020;17:665–680.
</cite> [<a href="https://doi.org/10.1038/s41592-020-0848-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7603796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32483333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Macromolecular%20modeling%20and%20design%20in%20Rosetta:%20recent%20methods%20and%20frameworks&amp;author=JK%20Leman&amp;author=BD%20Weitzner&amp;author=SM%20Lewis&amp;author=J%20Adolf%E2%80%90Bryfogle&amp;author=N%20Alam&amp;volume=17&amp;publication_year=2020&amp;pages=665-680&amp;pmid=32483333&amp;doi=10.1038/s41592-020-0848-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0073">
<cite>
Letunic I, Bork P. Interactive tree of life (iTOL) v6: recent updates to the phylogenetic tree display and annotation tool. Nucleic Acids Res. 2024;52:W78–W82.
</cite> [<a href="https://doi.org/10.1093/nar/gkae268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38613393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Interactive%20tree%20of%20life%20(iTOL)%20v6:%20recent%20updates%20to%20the%20phylogenetic%20tree%20display%20and%20annotation%20tool&amp;author=I%20Letunic&amp;author=P%20Bork&amp;volume=52&amp;publication_year=2024&amp;pages=W78-W82&amp;pmid=38613393&amp;doi=10.1093/nar/gkae268&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0074">
<cite>
Lienemann M, Gandier J‐A, Joensuu JJ, Iwanaga A, Takatsuji Y, Haruyama T, et al. Structure‐function relationships in Hydrophobins: probing the role of charged side chains. Appl Environ Microbiol. 2013;79:5533–5538.
</cite> [<a href="https://doi.org/10.1128/AEM.01493-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3754166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23835172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Structure%E2%80%90function%20relationships%20in%20Hydrophobins:%20probing%20the%20role%20of%20charged%20side%20chains&amp;author=M%20Lienemann&amp;author=J%E2%80%90A%20Gandier&amp;author=JJ%20Joensuu&amp;author=A%20Iwanaga&amp;author=Y%20Takatsuji&amp;volume=79&amp;publication_year=2013&amp;pages=5533-5538&amp;pmid=23835172&amp;doi=10.1128/AEM.01493-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0075">
<cite>
Linder M, Selber K, Nakari‐Setälä T, Qiao M, Kula MR, Penttilä M. The hydrophobins HFBI and HFBII from Trichoderma reesei showing efficient interactions with nonionic surfactants in aqueous two‐phase systems. Biomacromolecules. 2001;2:511–517.
</cite> [<a href="https://doi.org/10.1021/bm0001493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11749214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomacromolecules&amp;title=The%20hydrophobins%20HFBI%20and%20HFBII%20from%20Trichoderma%20reesei%20showing%20efficient%20interactions%20with%20nonionic%20surfactants%20in%20aqueous%20two%E2%80%90phase%20systems&amp;author=M%20Linder&amp;author=K%20Selber&amp;author=T%20Nakari%E2%80%90Set%C3%A4l%C3%A4&amp;author=M%20Qiao&amp;author=MR%20Kula&amp;volume=2&amp;publication_year=2001&amp;pages=511-517&amp;pmid=11749214&amp;doi=10.1021/bm0001493&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0076">
<cite>
Linder MB, Szilvay GR, Nakari‐Setälä T, Penttilä ME. Hydrophobins: the protein‐amphiphiles of filamentous fungi. FEMS Microbiol Rev. 2005;29:877–896.
</cite> [<a href="https://doi.org/10.1016/j.femsre.2005.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16219510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol%20Rev&amp;title=Hydrophobins:%20the%20protein%E2%80%90amphiphiles%20of%20filamentous%20fungi&amp;author=MB%20Linder&amp;author=GR%20Szilvay&amp;author=T%20Nakari%E2%80%90Set%C3%A4l%C3%A4&amp;author=ME%20Penttil%C3%A4&amp;volume=29&amp;publication_year=2005&amp;pages=877-896&amp;pmid=16219510&amp;doi=10.1016/j.femsre.2005.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0077">
<cite>
Lo VC, Ren Q, Pham CLL, Morris VK, Kwan AH, Sunde M. Fungal Hydrophobin proteins produce self‐assembling protein films with diverse structure and chemical stability. Nanomaterials. 2014;4:827–843.
</cite> [<a href="https://doi.org/10.3390/nano4030827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5304692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28344251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nanomaterials&amp;title=Fungal%20Hydrophobin%20proteins%20produce%20self%E2%80%90assembling%20protein%20films%20with%20diverse%20structure%20and%20chemical%20stability&amp;author=VC%20Lo&amp;author=Q%20Ren&amp;author=CLL%20Pham&amp;author=VK%20Morris&amp;author=AH%20Kwan&amp;volume=4&amp;publication_year=2014&amp;pages=827-843&amp;pmid=28344251&amp;doi=10.3390/nano4030827&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0078">
<cite>
Macindoe I, Kwan AH, Ren Q, Morris VK, Yang W, Mackay JP, et al. Self‐assembly of functional, amphipathic amyloid monolayers by the fungal hydrophobin EAS. Proc Natl Acad Sci USA. 2012;109:E804–E811.
</cite> [<a href="https://doi.org/10.1073/pnas.1114052109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3325668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22308366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Self%E2%80%90assembly%20of%20functional,%20amphipathic%20amyloid%20monolayers%20by%20the%20fungal%20hydrophobin%20EAS&amp;author=I%20Macindoe&amp;author=AH%20Kwan&amp;author=Q%20Ren&amp;author=VK%20Morris&amp;author=W%20Yang&amp;volume=109&amp;publication_year=2012&amp;pages=E804-E811&amp;pmid=22308366&amp;doi=10.1073/pnas.1114052109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0079">
<cite>
Mackay JP, Matthews JM, Winefield RD, Mackay LG, Haverkamp RG, Templeton MD. The hydrophobin EAS is largely unstructured in solution and functions by forming amyloid‐like structures. Structure. 2001;9:83–91.
</cite> [<a href="https://doi.org/10.1016/s0969-2126(00)00559-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11250193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=The%20hydrophobin%20EAS%20is%20largely%20unstructured%20in%20solution%20and%20functions%20by%20forming%20amyloid%E2%80%90like%20structures&amp;author=JP%20Mackay&amp;author=JM%20Matthews&amp;author=RD%20Winefield&amp;author=LG%20Mackay&amp;author=RG%20Haverkamp&amp;volume=9&amp;publication_year=2001&amp;pages=83-91&amp;pmid=11250193&amp;doi=10.1016/s0969-2126(00)00559-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0080">
<cite>
Magarkar A, Mele N, Abdel‐Rahman N, Butcher S, Torkkeli M, Serimaa R, et al. Hydrophobin film structure for HFBI and HFBII and mechanism for accelerated film formation. PLoS Comput Biol. 2014;10:e1003745.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1003745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4117420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25079355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Hydrophobin%20film%20structure%20for%20HFBI%20and%20HFBII%20and%20mechanism%20for%20accelerated%20film%20formation&amp;author=A%20Magarkar&amp;author=N%20Mele&amp;author=N%20Abdel%E2%80%90Rahman&amp;author=S%20Butcher&amp;author=M%20Torkkeli&amp;volume=10&amp;publication_year=2014&amp;pages=e1003745&amp;pmid=25079355&amp;doi=10.1371/journal.pcbi.1003745&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0081">
<cite>
Mariani V, Biasini M, Barbato A, Schwede T. lDDT: a local superposition‐free score for comparing protein structures and models using distance difference tests. Bioinformatics. 2013;29:2722–2728.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3799472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23986568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=lDDT:%20a%20local%20superposition%E2%80%90free%20score%20for%20comparing%20protein%20structures%20and%20models%20using%20distance%20difference%20tests&amp;author=V%20Mariani&amp;author=M%20Biasini&amp;author=A%20Barbato&amp;author=T%20Schwede&amp;volume=29&amp;publication_year=2013&amp;pages=2722-2728&amp;pmid=23986568&amp;doi=10.1093/bioinformatics/btt473&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0082">
<cite>
McPartlon M, Xu J. An end‐to‐end deep learning method for protein side‐chain packing and inverse folding. Proc Natl Acad Sci USA. 2023;120:e2216438120.
</cite> [<a href="https://doi.org/10.1073/pnas.2216438120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10266014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37253017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=An%20end%E2%80%90to%E2%80%90end%20deep%20learning%20method%20for%20protein%20side%E2%80%90chain%20packing%20and%20inverse%20folding&amp;author=M%20McPartlon&amp;author=J%20Xu&amp;volume=120&amp;publication_year=2023&amp;pages=e2216438120&amp;pmid=37253017&amp;doi=10.1073/pnas.2216438120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0083">
<cite>
Mereghetti P, Wade RC. Diffusion of hydrophobin proteins in solution and interactions with a graphite surface. BMC Biophys. 2011;4:9.
</cite> [<a href="https://doi.org/10.1186/2046-1682-4-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3114038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21595866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biophys&amp;title=Diffusion%20of%20hydrophobin%20proteins%20in%20solution%20and%20interactions%20with%20a%20graphite%20surface&amp;author=P%20Mereghetti&amp;author=RC%20Wade&amp;volume=4&amp;publication_year=2011&amp;pages=9&amp;pmid=21595866&amp;doi=10.1186/2046-1682-4-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0084">
<cite>
Mey G, Correia T, Oeser B, Kershaw MJ, Garre V, Arntz C, et al. Structural and functional analysis of an oligomeric hydrophobin gene from Claviceps purpurea. Mol Plant Pathol. 2003;4:31–41.
</cite> [<a href="https://doi.org/10.1046/j.1364-3703.2003.00138.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20569360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant%20Pathol&amp;title=Structural%20and%20functional%20analysis%20of%20an%20oligomeric%20hydrophobin%20gene%20from%20Claviceps%20purpurea&amp;author=G%20Mey&amp;author=T%20Correia&amp;author=B%20Oeser&amp;author=MJ%20Kershaw&amp;author=V%20Garre&amp;volume=4&amp;publication_year=2003&amp;pages=31-41&amp;pmid=20569360&amp;doi=10.1046/j.1364-3703.2003.00138.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0085">
<cite>
Mgbeahuruike AC, Kovalchuk A, Chen H, Ubhayasekera W, Asiegbu FO. Evolutionary analysis of hydrophobin gene family in two wood‐degrading basidiomycetes, Phlebia brevispora and Heterobasidion annosum s.l. BMC Evol Biol. 2013;13:240.
</cite> [<a href="https://doi.org/10.1186/1471-2148-13-240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3879219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24188142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol%20Biol&amp;title=Evolutionary%20analysis%20of%20hydrophobin%20gene%20family%20in%20two%20wood%E2%80%90degrading%20basidiomycetes,%20Phlebia%20brevispora%20and%20Heterobasidion%20annosum%20s.l&amp;author=AC%20Mgbeahuruike&amp;author=A%20Kovalchuk&amp;author=H%20Chen&amp;author=W%20Ubhayasekera&amp;author=FO%20Asiegbu&amp;volume=13&amp;publication_year=2013&amp;pages=240&amp;pmid=24188142&amp;doi=10.1186/1471-2148-13-240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0086">
<cite>
Minh BQ, Schmidt HA, Chernomor O, Schrempf D, Woodhams MD, von Haeseler A, et al. IQ‐TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. Mol Biol Evol. 2020;37:1530–1534.
</cite> [<a href="https://doi.org/10.1093/molbev/msaa015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=IQ%E2%80%90TREE%202:%20new%20models%20and%20efficient%20methods%20for%20phylogenetic%20inference%20in%20the%20genomic%20era&amp;author=BQ%20Minh&amp;author=HA%20Schmidt&amp;author=O%20Chernomor&amp;author=D%20Schrempf&amp;author=MD%20Woodhams&amp;volume=37&amp;publication_year=2020&amp;pages=1530-1534&amp;pmid=32011700&amp;doi=10.1093/molbev/msaa015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0087">
<cite>
Mirdita M, Schütze K, Moriwaki Y, Heo L, Ovchinnikov S, Steinegger M. ColabFold: making protein folding accessible to all. Nat Methods. 2022;19:679–682.
</cite> [<a href="https://doi.org/10.1038/s41592-022-01488-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9184281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35637307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=ColabFold:%20making%20protein%20folding%20accessible%20to%20all&amp;author=M%20Mirdita&amp;author=K%20Sch%C3%BCtze&amp;author=Y%20Moriwaki&amp;author=L%20Heo&amp;author=S%20Ovchinnikov&amp;volume=19&amp;publication_year=2022&amp;pages=679-682&amp;pmid=35637307&amp;doi=10.1038/s41592-022-01488-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0088">
<cite>
Monteiro da Silva G, Cui JY, Dalgarno DC, Lisi GP, Rubenstein BM. High‐throughput prediction of protein conformational distributions with subsampled AlphaFold2. Nat Commun. 2024;15:2464.
</cite> [<a href="https://doi.org/10.1038/s41467-024-46715-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10973385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38538622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=High%E2%80%90throughput%20prediction%20of%20protein%20conformational%20distributions%20with%20subsampled%20AlphaFold2&amp;author=G%20Monteiro%20da%20Silva&amp;author=JY%20Cui&amp;author=DC%20Dalgarno&amp;author=GP%20Lisi&amp;author=BM%20Rubenstein&amp;volume=15&amp;publication_year=2024&amp;pages=2464&amp;pmid=38538622&amp;doi=10.1038/s41467-024-46715-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0089">
<cite>
Morris VK, Kwan AH, Sunde M. Analysis of the structure and conformational states of DewA gives insight into the assembly of the fungal hydrophobins. J Mol Biol. 2013;425:244–256.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2012.10.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23137797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Analysis%20of%20the%20structure%20and%20conformational%20states%20of%20DewA%20gives%20insight%20into%20the%20assembly%20of%20the%20fungal%20hydrophobins&amp;author=VK%20Morris&amp;author=AH%20Kwan&amp;author=M%20Sunde&amp;volume=425&amp;publication_year=2013&amp;pages=244-256&amp;pmid=23137797&amp;doi=10.1016/j.jmb.2012.10.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0090">
<cite>
Nakari‐Setälä T, Aro N, Ilmén M, Muñoz G, Kalkkinen N, Penttilä M. Differential expression of the vegetative and spore‐bound hydrophobins of Trichoderma reesei‐‐cloning and characterization of the hfb2 gene. Eur J Biochem. 1997;248:415–423.
</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1997.00415.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9346297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&amp;title=Differential%20expression%20of%20the%20vegetative%20and%20spore%E2%80%90bound%20hydrophobins%20of%20Trichoderma%20reesei%E2%80%90%E2%80%90cloning%20and%20characterization%20of%20the%20hfb2%20gene&amp;author=T%20Nakari%E2%80%90Set%C3%A4l%C3%A4&amp;author=N%20Aro&amp;author=M%20Ilm%C3%A9n&amp;author=G%20Mu%C3%B1oz&amp;author=N%20Kalkkinen&amp;volume=248&amp;publication_year=1997&amp;pages=415-423&amp;pmid=9346297&amp;doi=10.1111/j.1432-1033.1997.00415.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0091">
<cite>
Nakari‐Setälä T, Aro N, Kalkkinen N, Alatalo E, Penttilä M. Genetic and biochemical characterization of the Trichoderma reesei hydrophobin HFBI. Eur J Biochem. 1996;235:248–255.
</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1996.00248.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8631337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&amp;title=Genetic%20and%20biochemical%20characterization%20of%20the%20Trichoderma%20reesei%20hydrophobin%20HFBI&amp;author=T%20Nakari%E2%80%90Set%C3%A4l%C3%A4&amp;author=N%20Aro&amp;author=N%20Kalkkinen&amp;author=E%20Alatalo&amp;author=M%20Penttil%C3%A4&amp;volume=235&amp;publication_year=1996&amp;pages=248-255&amp;pmid=8631337&amp;doi=10.1111/j.1432-1033.1996.00248.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0092">
<cite>
Nivón LG, Moretti R, Baker D. A pareto‐optimal refinement method for protein design scaffolds. PLoS One. 2013;8:e59004.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0059004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3614904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23565140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20pareto%E2%80%90optimal%20refinement%20method%20for%20protein%20design%20scaffolds&amp;author=LG%20Niv%C3%B3n&amp;author=R%20Moretti&amp;author=D%20Baker&amp;volume=8&amp;publication_year=2013&amp;pages=e59004&amp;pmid=23565140&amp;doi=10.1371/journal.pone.0059004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0093">
<cite>
Nolle F, Starke LJ, Griffo A, Lienemann M, Jacobs K, Seemann R, et al. Hydrophobin bilayer as water impermeable protein membrane. Langmuir. 2023;39:13790–13800.
</cite> [<a href="https://doi.org/10.1021/acs.langmuir.3c01006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10552762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37726241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Langmuir&amp;title=Hydrophobin%20bilayer%20as%20water%20impermeable%20protein%20membrane&amp;author=F%20Nolle&amp;author=LJ%20Starke&amp;author=A%20Griffo&amp;author=M%20Lienemann&amp;author=K%20Jacobs&amp;volume=39&amp;publication_year=2023&amp;pages=13790-13800&amp;pmid=37726241&amp;doi=10.1021/acs.langmuir.3c01006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0094">
<cite>
Olechnovič K, Monastyrskyy B, Kryshtafovych A, Venclovas Č. Comparative analysis of methods for evaluation of protein models against native structures. Bioinformatics. 2019;35:937–944.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6419896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30169622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Comparative%20analysis%20of%20methods%20for%20evaluation%20of%20protein%20models%20against%20native%20structures&amp;author=K%20Olechnovi%C4%8D&amp;author=B%20Monastyrskyy&amp;author=A%20Kryshtafovych&amp;author=%C4%8C%20Venclovas&amp;volume=35&amp;publication_year=2019&amp;pages=937-944&amp;pmid=30169622&amp;doi=10.1093/bioinformatics/bty760&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0095">
<cite>
Park H, Bradley P, Greisen P, Liu Y, Mulligan VK, Kim DE, et al. Simultaneous optimization of biomolecular energy functions on features from small molecules and macromolecules. J Chem Theory Comput. 2016;12:6201–6212.
</cite> [<a href="https://doi.org/10.1021/acs.jctc.6b00819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5515585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27766851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Simultaneous%20optimization%20of%20biomolecular%20energy%20functions%20on%20features%20from%20small%20molecules%20and%20macromolecules&amp;author=H%20Park&amp;author=P%20Bradley&amp;author=P%20Greisen&amp;author=Y%20Liu&amp;author=VK%20Mulligan&amp;volume=12&amp;publication_year=2016&amp;pages=6201-6212&amp;pmid=27766851&amp;doi=10.1021/acs.jctc.6b00819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0096">
<cite>
Pham CLL, Rey A, Lo V, Soulès M, Ren Q, Meisl G, et al. Self‐assembly of MPG1, a hydrophobin protein from the rice blast fungus that forms functional amyloid coatings, occurs by a surface‐driven mechanism. Sci Rep. 2016;6:25288.
</cite> [<a href="https://doi.org/10.1038/srep25288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4855151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27142249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Self%E2%80%90assembly%20of%20MPG1,%20a%20hydrophobin%20protein%20from%20the%20rice%20blast%20fungus%20that%20forms%20functional%20amyloid%20coatings,%20occurs%20by%20a%20surface%E2%80%90driven%20mechanism&amp;author=CLL%20Pham&amp;author=A%20Rey&amp;author=V%20Lo&amp;author=M%20Soul%C3%A8s&amp;author=Q%20Ren&amp;volume=6&amp;publication_year=2016&amp;pages=25288&amp;pmid=27142249&amp;doi=10.1038/srep25288&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0097">
<cite>
Pham CLL, Rodríguez de Francisco B, Valsecchi I, Dazzoni R, Pillé A, Lo V, et al. Probing structural changes during self‐assembly of surface‐active Hydrophobin proteins that form functional amyloids in fungi. J Mol Biol. 2018;430:3784–3801.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2018.07.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30096347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Probing%20structural%20changes%20during%20self%E2%80%90assembly%20of%20surface%E2%80%90active%20Hydrophobin%20proteins%20that%20form%20functional%20amyloids%20in%20fungi&amp;author=CLL%20Pham&amp;author=B%20Rodr%C3%ADguez%20de%20Francisco&amp;author=I%20Valsecchi&amp;author=R%20Dazzoni&amp;author=A%20Pill%C3%A9&amp;volume=430&amp;publication_year=2018&amp;pages=3784-3801&amp;pmid=30096347&amp;doi=10.1016/j.jmb.2018.07.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0098">
<cite>
Pinheiro F, Santos J, Ventura S. AlphaFold and the amyloid landscape. J Mol Biol. 2021;433:167059.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2021.167059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34023402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=AlphaFold%20and%20the%20amyloid%20landscape&amp;author=F%20Pinheiro&amp;author=J%20Santos&amp;author=S%20Ventura&amp;volume=433&amp;publication_year=2021&amp;pages=167059&amp;pmid=34023402&amp;doi=10.1016/j.jmb.2021.167059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0099">
<cite>
Piscitelli A, Cicatiello P, Gravagnuolo AM, Sorrentino I, Pezzella C, Giardina P. Applications of functional amyloids from fungi: surface modification by class I Hydrophobins. Biomolecules. 2017;7:45.
</cite> [<a href="https://doi.org/10.3390/biom7030045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5618226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28672843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Applications%20of%20functional%20amyloids%20from%20fungi:%20surface%20modification%20by%20class%20I%20Hydrophobins&amp;author=A%20Piscitelli&amp;author=P%20Cicatiello&amp;author=AM%20Gravagnuolo&amp;author=I%20Sorrentino&amp;author=C%20Pezzella&amp;volume=7&amp;publication_year=2017&amp;pages=45&amp;pmid=28672843&amp;doi=10.3390/biom7030045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0100">
<cite>
Planas‐Iglesias J, Borko S, Swiatkowski J, Elias M, Havlasek M, Salamon O, et al. AggreProt: a web server for predicting and engineering aggregation prone regions in proteins. Nucleic Acids Res. 2024;52:W159–W169.
</cite> [<a href="https://doi.org/10.1093/nar/gkae420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38801076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=AggreProt:%20a%20web%20server%20for%20predicting%20and%20engineering%20aggregation%20prone%20regions%20in%20proteins&amp;author=J%20Planas%E2%80%90Iglesias&amp;author=S%20Borko&amp;author=J%20Swiatkowski&amp;author=M%20Elias&amp;author=M%20Havlasek&amp;volume=52&amp;publication_year=2024&amp;pages=W159-W169&amp;pmid=38801076&amp;doi=10.1093/nar/gkae420&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0101">
<cite>
Převorovský M, Atkinson SR, Ptáčková M, McLean JR, Gould K, Folk P, et al. N‐termini of fungal CSL transcription factors are disordered, enriched in regulatory motifs and inhibit DNA binding in fission yeast. PLoS One. 2011;6:e23650.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0023650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3155561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21858190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=N%E2%80%90termini%20of%20fungal%20CSL%20transcription%20factors%20are%20disordered,%20enriched%20in%20regulatory%20motifs%20and%20inhibit%20DNA%20binding%20in%20fission%20yeast&amp;author=M%20P%C5%99evorovsk%C3%BD&amp;author=SR%20Atkinson&amp;author=M%20Pt%C3%A1%C4%8Dkov%C3%A1&amp;author=JR%20McLean&amp;author=K%20Gould&amp;volume=6&amp;publication_year=2011&amp;pages=e23650&amp;pmid=21858190&amp;doi=10.1371/journal.pone.0023650&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0102">
<cite>
Ragonis‐Bachar P, Axel G, Blau S, Ben‐Tal N, Kolodny R, Landau M. What can AlphaFold do for antimicrobial amyloids?
Proteins. 2024;92(2):265–281.
</cite> [<a href="https://doi.org/10.1002/prot.26618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37855235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=What%20can%20AlphaFold%20do%20for%20antimicrobial%20amyloids?&amp;author=P%20Ragonis%E2%80%90Bachar&amp;author=G%20Axel&amp;author=S%20Blau&amp;author=N%20Ben%E2%80%90Tal&amp;author=R%20Kolodny&amp;volume=92&amp;issue=2&amp;publication_year=2024&amp;pages=265-281&amp;pmid=37855235&amp;doi=10.1002/prot.26618&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0103">
<cite>
Ren Q, Kwan AH, Sunde M. Two forms and two faces, multiple states and multiple uses: properties and applications of the self‐assembling fungal hydrophobins. Biopolymers. 2013;100:601–612.
</cite> [<a href="https://doi.org/10.1002/bip.22259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23913717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biopolymers&amp;title=Two%20forms%20and%20two%20faces,%20multiple%20states%20and%20multiple%20uses:%20properties%20and%20applications%20of%20the%20self%E2%80%90assembling%20fungal%20hydrophobins&amp;author=Q%20Ren&amp;author=AH%20Kwan&amp;author=M%20Sunde&amp;volume=100&amp;publication_year=2013&amp;pages=601-612&amp;pmid=23913717&amp;doi=10.1002/bip.22259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0104">
<cite>
Ren Q, Kwan AH, Sunde M. Solution structure and interface‐driven self‐assembly of NC2, a new member of the class II hydrophobin proteins. Proteins. 2014;82:990–1003.
</cite> [<a href="https://doi.org/10.1002/prot.24473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24218020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Solution%20structure%20and%20interface%E2%80%90driven%20self%E2%80%90assembly%20of%20NC2,%20a%20new%20member%20of%20the%20class%20II%20hydrophobin%20proteins&amp;author=Q%20Ren&amp;author=AH%20Kwan&amp;author=M%20Sunde&amp;volume=82&amp;publication_year=2014&amp;pages=990-1003&amp;pmid=24218020&amp;doi=10.1002/prot.24473&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0105">
<cite>
Rineau F, Lmalem H, Ahren D, Shah F, Johansson T, Coninx L, et al. Comparative genomics and expression levels of hydrophobins from eight mycorrhizal genomes. Mycorrhiza. 2017;27:383–396.
</cite> [<a href="https://doi.org/10.1007/s00572-016-0758-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28066872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mycorrhiza&amp;title=Comparative%20genomics%20and%20expression%20levels%20of%20hydrophobins%20from%20eight%20mycorrhizal%20genomes&amp;author=F%20Rineau&amp;author=H%20Lmalem&amp;author=D%20Ahren&amp;author=F%20Shah&amp;author=T%20Johansson&amp;volume=27&amp;publication_year=2017&amp;pages=383-396&amp;pmid=28066872&amp;doi=10.1007/s00572-016-0758-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0106">
<cite>
Robert X, Gouet P. Deciphering key features in protein structures with the new ENDscript server. Nucleic Acids Res. 2014;42:W320–W324.
</cite> [<a href="https://doi.org/10.1093/nar/gku316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24753421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Deciphering%20key%20features%20in%20protein%20structures%20with%20the%20new%20ENDscript%20server&amp;author=X%20Robert&amp;author=P%20Gouet&amp;volume=42&amp;publication_year=2014&amp;pages=W320-W324&amp;pmid=24753421&amp;doi=10.1093/nar/gku316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0107">
<cite>
Rojas‐Osnaya J, Quintana‐Quirino M, Espinosa‐Valencia A, Bravo AL, Nájera H. Hydrophobins: multitask proteins. Front Phys. 2024;12:1393340.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Phys&amp;title=Hydrophobins:%20multitask%20proteins&amp;author=J%20Rojas%E2%80%90Osnaya&amp;author=M%20Quintana%E2%80%90Quirino&amp;author=A%20Espinosa%E2%80%90Valencia&amp;author=AL%20Bravo&amp;author=H%20N%C3%A1jera&amp;volume=12&amp;publication_year=2024&amp;pages=1393340&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0108">
<cite>
Sala D, Engelberger F, Mchaourab HS, Meiler J. Modeling conformational states of proteins with AlphaFold. Curr Opin Struct Biol. 2023;81:102645.
</cite> [<a href="https://doi.org/10.1016/j.sbi.2023.102645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37392556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Struct%20Biol&amp;title=Modeling%20conformational%20states%20of%20proteins%20with%20AlphaFold&amp;author=D%20Sala&amp;author=F%20Engelberger&amp;author=HS%20Mchaourab&amp;author=J%20Meiler&amp;volume=81&amp;publication_year=2023&amp;pages=102645&amp;pmid=37392556&amp;doi=10.1016/j.sbi.2023.102645&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0109">
<cite>
Sallada ND, Dunn KJ, Berger BW. A structural and functional role for disulfide bonds in a class II Hydrophobin. Biochemistry. 2018;57:645–653.
</cite> [<a href="https://doi.org/10.1021/acs.biochem.7b01166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29277996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=A%20structural%20and%20functional%20role%20for%20disulfide%20bonds%20in%20a%20class%20II%20Hydrophobin&amp;author=ND%20Sallada&amp;author=KJ%20Dunn&amp;author=BW%20Berger&amp;volume=57&amp;publication_year=2018&amp;pages=645-653&amp;pmid=29277996&amp;doi=10.1021/acs.biochem.7b01166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0110">
<cite>
Shanmugam N, Baker MODG, Ball SR, Steain M, Pham CLL, Sunde M. Microbial functional amyloids serve diverse purposes for structure, adhesion and defence. Biophys Rev. 2019;11:287–302.
</cite> [<a href="https://doi.org/10.1007/s12551-019-00526-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6557962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31049855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20Rev&amp;title=Microbial%20functional%20amyloids%20serve%20diverse%20purposes%20for%20structure,%20adhesion%20and%20defence&amp;author=N%20Shanmugam&amp;author=MODG%20Baker&amp;author=SR%20Ball&amp;author=M%20Steain&amp;author=CLL%20Pham&amp;volume=11&amp;publication_year=2019&amp;pages=287-302&amp;pmid=31049855&amp;doi=10.1007/s12551-019-00526-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0111">
<cite>
Sievers F, Wilm A, Dineen D, Gibson TJ, Karplus K, Li W, et al. Fast, scalable generation of high‐quality protein multiple sequence alignments using Clustal omega. Mol Syst Biol. 2011;7:539.
</cite> [<a href="https://doi.org/10.1038/msb.2011.75" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3261699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21988835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Syst%20Biol&amp;title=Fast,%20scalable%20generation%20of%20high%E2%80%90quality%20protein%20multiple%20sequence%20alignments%20using%20Clustal%20omega&amp;author=F%20Sievers&amp;author=A%20Wilm&amp;author=D%20Dineen&amp;author=TJ%20Gibson&amp;author=K%20Karplus&amp;volume=7&amp;publication_year=2011&amp;pages=539&amp;pmid=21988835&amp;doi=10.1038/msb.2011.75&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0112">
<cite>
Stanzione I, Pitocchi R, Pennacchio A, Cicatiello P, Piscitelli A, Giardina P. Innovative surface bio‐functionalization by fungal hydrophobins and their engineered variants. Front Mol Biosci. 2022;9:959166.
</cite> [<a href="https://doi.org/10.3389/fmolb.2022.959166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9403755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36032682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Mol%20Biosci&amp;title=Innovative%20surface%20bio%E2%80%90functionalization%20by%20fungal%20hydrophobins%20and%20their%20engineered%20variants&amp;author=I%20Stanzione&amp;author=R%20Pitocchi&amp;author=A%20Pennacchio&amp;author=P%20Cicatiello&amp;author=A%20Piscitelli&amp;volume=9&amp;publication_year=2022&amp;pages=959166&amp;pmid=36032682&amp;doi=10.3389/fmolb.2022.959166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0113">
<cite>
Stingaciu L‐R. Study of protein dynamics via neutron spin Echo spectroscopy. J Vis Exp. 2022;182: e61862.</cite> [<a href="https://doi.org/10.3791/61862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35499345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Vis%20Exp&amp;title=Study%20of%20protein%20dynamics%20via%20neutron%20spin%20Echo%20spectroscopy&amp;author=L%E2%80%90R%20Stingaciu&amp;volume=182&amp;publication_year=2022&amp;pages=e61862&amp;pmid=35499345&amp;doi=10.3791/61862&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0114">
<cite>
Sunde M, Kwan AHY, Templeton MD, Beever RE, Mackay JP. Structural analysis of hydrophobins. Micron. 2008;39:773–784.
</cite> [<a href="https://doi.org/10.1016/j.micron.2007.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17875392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Micron&amp;title=Structural%20analysis%20of%20hydrophobins&amp;author=M%20Sunde&amp;author=AHY%20Kwan&amp;author=MD%20Templeton&amp;author=RE%20Beever&amp;author=JP%20Mackay&amp;volume=39&amp;publication_year=2008&amp;pages=773-784&amp;pmid=17875392&amp;doi=10.1016/j.micron.2007.08.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0115">
<cite>
Szilvay GR, Paananen A, Laurikainen K, Vuorimaa E, Lemmetyinen H, Peltonen J, et al. Self‐assembled hydrophobin protein films at the air‐water interface: structural analysis and molecular engineering. Biochemistry. 2007;46:2345–2354.
</cite> [<a href="https://doi.org/10.1021/bi602358h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17297923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Self%E2%80%90assembled%20hydrophobin%20protein%20films%20at%20the%20air%E2%80%90water%20interface:%20structural%20analysis%20and%20molecular%20engineering&amp;author=GR%20Szilvay&amp;author=A%20Paananen&amp;author=K%20Laurikainen&amp;author=E%20Vuorimaa&amp;author=H%20Lemmetyinen&amp;volume=46&amp;publication_year=2007&amp;pages=2345-2354&amp;pmid=17297923&amp;doi=10.1021/bi602358h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0116">
<cite>
Tanaka T, Tanabe H, Uehara K, Takahashi T, Abe K. Involvement of hydrophobic amino acid residues in C7‐C8 loop of aspergillus oryzae hydrophobin RolA in hydrophobic interaction between RolA and a polyester. Biosci Biotechnol Biochem. 2014;78:1693–1699.
</cite> [<a href="https://doi.org/10.1080/09168451.2014.932684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25273133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biosci%20Biotechnol%20Biochem&amp;title=Involvement%20of%20hydrophobic%20amino%20acid%20residues%20in%20C7%E2%80%90C8%20loop%20of%20aspergillus%20oryzae%20hydrophobin%20RolA%20in%20hydrophobic%20interaction%20between%20RolA%20and%20a%20polyester&amp;author=T%20Tanaka&amp;author=H%20Tanabe&amp;author=K%20Uehara&amp;author=T%20Takahashi&amp;author=K%20Abe&amp;volume=78&amp;publication_year=2014&amp;pages=1693-1699&amp;pmid=25273133&amp;doi=10.1080/09168451.2014.932684&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0117">
<cite>
Tanaka T, Terauchi Y, Yoshimi A, Abe K. Aspergillus Hydrophobins: physicochemical properties, biochemical properties, and functions in solid polymer degradation. Microorganisms. 2022;10:1498.
</cite> [<a href="https://doi.org/10.3390/microorganisms10081498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9394342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35893556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Aspergillus%20Hydrophobins:%20physicochemical%20properties,%20biochemical%20properties,%20and%20functions%20in%20solid%20polymer%20degradation&amp;author=T%20Tanaka&amp;author=Y%20Terauchi&amp;author=A%20Yoshimi&amp;author=K%20Abe&amp;volume=10&amp;publication_year=2022&amp;pages=1498&amp;pmid=35893556&amp;doi=10.3390/microorganisms10081498&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0118">
<cite>
Tee W‐V, Guarnera E, Berezovsky IN. Disorder driven allosteric control of protein activity. Curr Res Struct Biol. 2020;2:191–203.
</cite> [<a href="https://doi.org/10.1016/j.crstbi.2020.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8244471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34235479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Res%20Struct%20Biol&amp;title=Disorder%20driven%20allosteric%20control%20of%20protein%20activity&amp;author=W%E2%80%90V%20Tee&amp;author=E%20Guarnera&amp;author=IN%20Berezovsky&amp;volume=2&amp;publication_year=2020&amp;pages=191-203&amp;pmid=34235479&amp;doi=10.1016/j.crstbi.2020.09.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0119">
<cite>
Terauchi Y, Nagayama M, Tanaka T, Tanabe H, Yoshimi A, Nanatani K, et al. Adsorption kinetics and self‐assembled structures of aspergillus oryzae Hydrophobin RolA on hydrophobic and charged solid surfaces. Appl Environ Microbiol. 2022;88:e0208721.
</cite> [<a href="https://doi.org/10.1128/aem.02087-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8939330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35108098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Adsorption%20kinetics%20and%20self%E2%80%90assembled%20structures%20of%20aspergillus%20oryzae%20Hydrophobin%20RolA%20on%20hydrophobic%20and%20charged%20solid%20surfaces&amp;author=Y%20Terauchi&amp;author=M%20Nagayama&amp;author=T%20Tanaka&amp;author=H%20Tanabe&amp;author=A%20Yoshimi&amp;volume=88&amp;publication_year=2022&amp;pages=e0208721&amp;pmid=35108098&amp;doi=10.1128/aem.02087-21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0120">
<cite>
Terauchi Y, Tanaka T, Mitsuishi M, Yabu H, Yoshimi A, Nantani K, et al. Analysis of the self‐assembly process of aspergillus oryzae hydrophobin RolA by Langmuir‐Blodgett method. Biosci Biotechnol Biochem. 2020;84:678–685.
</cite> [<a href="https://doi.org/10.1080/09168451.2019.1706443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31876261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biosci%20Biotechnol%20Biochem&amp;title=Analysis%20of%20the%20self%E2%80%90assembly%20process%20of%20aspergillus%20oryzae%20hydrophobin%20RolA%20by%20Langmuir%E2%80%90Blodgett%20method&amp;author=Y%20Terauchi&amp;author=T%20Tanaka&amp;author=M%20Mitsuishi&amp;author=H%20Yabu&amp;author=A%20Yoshimi&amp;volume=84&amp;publication_year=2020&amp;pages=678-685&amp;pmid=31876261&amp;doi=10.1080/09168451.2019.1706443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0121">
<cite>
Thieker DF, Maguire JB, Kudlacek ST, Leaver‐Fay A, Lyskov S, Kuhlman B. Stabilizing proteins, simplified: a Rosetta‐based webtool for predicting favorable mutations. Protein Sci. 2022;31:e4428.
</cite> [<a href="https://doi.org/10.1002/pro.4428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9490798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36173174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Stabilizing%20proteins,%20simplified:%20a%20Rosetta%E2%80%90based%20webtool%20for%20predicting%20favorable%20mutations&amp;author=DF%20Thieker&amp;author=JB%20Maguire&amp;author=ST%20Kudlacek&amp;author=A%20Leaver%E2%80%90Fay&amp;author=S%20Lyskov&amp;volume=31&amp;publication_year=2022&amp;pages=e4428&amp;pmid=36173174&amp;doi=10.1002/pro.4428&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0006">
<cite>
UniProt Consortium
. The universal protein resource (UniProt). Nucleic Acids Res. 2008;36:D190–D195.
</cite> [<a href="https://doi.org/10.1093/nar/gkm895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2238893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18045787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20universal%20protein%20resource%20(UniProt&amp;volume=36&amp;publication_year=2008&amp;pages=D190-D195&amp;pmid=18045787&amp;doi=10.1093/nar/gkm895&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0122">
<cite>
Unni S, Huang Y, Hanson RM, Tobias M, Krishnan S, Li WW, et al. Web servers and services for electrostatics calculations with APBS and PDB2PQR. J Comput Chem. 2011;32:1488–1491.
</cite> [<a href="https://doi.org/10.1002/jcc.21720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3062090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21425296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&amp;title=Web%20servers%20and%20services%20for%20electrostatics%20calculations%20with%20APBS%20and%20PDB2PQR&amp;author=S%20Unni&amp;author=Y%20Huang&amp;author=RM%20Hanson&amp;author=M%20Tobias&amp;author=S%20Krishnan&amp;volume=32&amp;publication_year=2011&amp;pages=1488-1491&amp;pmid=21425296&amp;doi=10.1002/jcc.21720&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0123">
<cite>
Valo HK, Laaksonen PH, Peltonen LJ, Linder MB, Hirvonen JT, Laaksonen TJ. Multifunctional hydrophobin: toward functional coatings for drug nanoparticles. ACS Nano. 2010;4:1750–1758.
</cite> [<a href="https://doi.org/10.1021/nn9017558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20210303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Nano&amp;title=Multifunctional%20hydrophobin:%20toward%20functional%20coatings%20for%20drug%20nanoparticles&amp;author=HK%20Valo&amp;author=PH%20Laaksonen&amp;author=LJ%20Peltonen&amp;author=MB%20Linder&amp;author=JT%20Hirvonen&amp;volume=4&amp;publication_year=2010&amp;pages=1750-1758&amp;pmid=20210303&amp;doi=10.1021/nn9017558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0124">
<cite>
Valsecchi I, Lai JI, Stephen‐Victor E, Pillé A, Beaussart A, Lo V, et al. Assembly and disassembly of aspergillus fumigatus conidial rodlets. Cell Surf. 2019;5:100023.
</cite> [<a href="https://doi.org/10.1016/j.tcsw.2019.100023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7389560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32743139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Surf&amp;title=Assembly%20and%20disassembly%20of%20aspergillus%20fumigatus%20conidial%20rodlets&amp;author=I%20Valsecchi&amp;author=JI%20Lai&amp;author=E%20Stephen%E2%80%90Victor&amp;author=A%20Pill%C3%A9&amp;author=A%20Beaussart&amp;volume=5&amp;publication_year=2019&amp;pages=100023&amp;pmid=32743139&amp;doi=10.1016/j.tcsw.2019.100023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0125">
<cite>
van Kempen M, Kim SS, Tumescheit C, Mirdita M, Lee J, Gilchrist CLM, et al. Fast and accurate protein structure search with Foldseek. Nat Biotechnol. 2024;42:243–246.
</cite> [<a href="https://doi.org/10.1038/s41587-023-01773-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10869269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37156916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Fast%20and%20accurate%20protein%20structure%20search%20with%20Foldseek&amp;author=M%20van%20Kempen&amp;author=SS%20Kim&amp;author=C%20Tumescheit&amp;author=M%20Mirdita&amp;author=J%20Lee&amp;volume=42&amp;publication_year=2024&amp;pages=243-246&amp;pmid=37156916&amp;doi=10.1038/s41587-023-01773-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0126">
<cite>
van Wetter MA, Wösten HA, Wessels JG. SC3 and SC4 hydrophobins have distinct roles in formation of aerial structures in dikaryons of Schizophyllum commune. Mol Microbiol. 2000;36:201–210.
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.2000.01848.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10760177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=SC3%20and%20SC4%20hydrophobins%20have%20distinct%20roles%20in%20formation%20of%20aerial%20structures%20in%20dikaryons%20of%20Schizophyllum%20commune&amp;author=MA%20van%20Wetter&amp;author=HA%20W%C3%B6sten&amp;author=JG%20Wessels&amp;volume=36&amp;publication_year=2000&amp;pages=201-210&amp;pmid=10760177&amp;doi=10.1046/j.1365-2958.2000.01848.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0127">
<cite>
Varadi M, Anyango S, Deshpande M, Nair S, Natassia C, Yordanova G, et al. AlphaFold protein structure database: massively expanding the structural coverage of protein‐sequence space with high‐accuracy models. Nucleic Acids Res. 2022;50:D439–D444.
</cite> [<a href="https://doi.org/10.1093/nar/gkab1061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34791371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=AlphaFold%20protein%20structure%20database:%20massively%20expanding%20the%20structural%20coverage%20of%20protein%E2%80%90sequence%20space%20with%20high%E2%80%90accuracy%20models&amp;author=M%20Varadi&amp;author=S%20Anyango&amp;author=M%20Deshpande&amp;author=S%20Nair&amp;author=C%20Natassia&amp;volume=50&amp;publication_year=2022&amp;pages=D439-D444&amp;pmid=34791371&amp;doi=10.1093/nar/gkab1061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0128">
<cite>
Vergunst KL, Kenward C, Langelaan DN. Characterization of the structure and self‐assembly of two distinct class IB hydrophobins. Appl Microbiol Biotechnol. 2022;106:7831–7843.
</cite> [<a href="https://doi.org/10.1007/s00253-022-12253-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36329133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=Characterization%20of%20the%20structure%20and%20self%E2%80%90assembly%20of%20two%20distinct%20class%20IB%20hydrophobins&amp;author=KL%20Vergunst&amp;author=C%20Kenward&amp;author=DN%20Langelaan&amp;volume=106&amp;publication_year=2022&amp;pages=7831-7843&amp;pmid=36329133&amp;doi=10.1007/s00253-022-12253-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0129">
<cite>
Vergunst KL, Langelaan DN. The N‐terminal tail of the hydrophobin SC16 is not required for rodlet formation. Sci Rep. 2022;12:366.
</cite> [<a href="https://doi.org/10.1038/s41598-021-04223-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8748815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35013607/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=The%20N%E2%80%90terminal%20tail%20of%20the%20hydrophobin%20SC16%20is%20not%20required%20for%20rodlet%20formation&amp;author=KL%20Vergunst&amp;author=DN%20Langelaan&amp;volume=12&amp;publication_year=2022&amp;pages=366&amp;pmid=35013607&amp;doi=10.1038/s41598-021-04223-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0130">
<cite>
Veyron S, Peyroche G, Cherfils J. FIC proteins: from bacteria to humans and back again. Pathog Dis. 2018;76(2).</cite> [<a href="https://doi.org/10.1093/femspd/fty012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29617857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathog%20Dis&amp;title=FIC%20proteins:%20from%20bacteria%20to%20humans%20and%20back%20again&amp;author=S%20Veyron&amp;author=G%20Peyroche&amp;author=J%20Cherfils&amp;publication_year=2018&amp;pages=76(2)&amp;pmid=29617857&amp;doi=10.1093/femspd/fty012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0131">
<cite>
Wang X, de Vocht ML, de Jonge J, Poolman B, Robillard GT. Structural changes and molecular interactions of hydrophobin SC3 in solution and on a hydrophobic surface. Protein Sci. 2002;11:1172–1181.
</cite> [<a href="https://doi.org/10.1110/ps.4530102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2373550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11967373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Structural%20changes%20and%20molecular%20interactions%20of%20hydrophobin%20SC3%20in%20solution%20and%20on%20a%20hydrophobic%20surface&amp;author=X%20Wang&amp;author=ML%20de%20Vocht&amp;author=J%20de%20Jonge&amp;author=B%20Poolman&amp;author=GT%20Robillard&amp;volume=11&amp;publication_year=2002&amp;pages=1172-1181&amp;pmid=11967373&amp;doi=10.1110/ps.4530102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0132">
<cite>
Wang X, Graveland‐Bikker JF, De Kruif CG, Robillard GT. Oligomerization of hydrophobin SC3 in solution: from soluble state to self‐assembly. Protein Sci. 2004;13:810–821.
</cite> [<a href="https://doi.org/10.1110/ps.03367304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2286737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14978312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Oligomerization%20of%20hydrophobin%20SC3%20in%20solution:%20from%20soluble%20state%20to%20self%E2%80%90assembly&amp;author=X%20Wang&amp;author=JF%20Graveland%E2%80%90Bikker&amp;author=CG%20De%20Kruif&amp;author=GT%20Robillard&amp;volume=13&amp;publication_year=2004&amp;pages=810-821&amp;pmid=14978312&amp;doi=10.1110/ps.03367304&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0133">
<cite>
Wang X, Permentier HP, Rink R, Kruijtzer JAW, Liskamp RMJ, et al. Probing the self‐assembly and the accompanying structural changes of hydrophobin SC3 on a hydrophobic surface by mass spectrometry. Biophys J. 2004;87:1919–1928.
</cite> [<a href="https://doi.org/10.1529/biophysj.104.041616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1304595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15345568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Probing%20the%20self%E2%80%90assembly%20and%20the%20accompanying%20structural%20changes%20of%20hydrophobin%20SC3%20on%20a%20hydrophobic%20surface%20by%20mass%20spectrometry&amp;author=X%20Wang&amp;author=HP%20Permentier&amp;author=R%20Rink&amp;author=JAW%20Kruijtzer&amp;author=RMJ%20Liskamp&amp;volume=87&amp;publication_year=2004&amp;pages=1919-1928&amp;pmid=15345568&amp;doi=10.1529/biophysj.104.041616&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0134">
<cite>
Wang X, Shi F, Wösten HAB, Hektor H, Poolman B, Robillard GT. The SC3 hydrophobin self‐assembles into a membrane with distinct mass transfer properties. Biophys J. 2005;88:3434–3443.
</cite> [<a href="https://doi.org/10.1529/biophysj.104.057794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1305490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15749774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=The%20SC3%20hydrophobin%20self%E2%80%90assembles%20into%20a%20membrane%20with%20distinct%20mass%20transfer%20properties&amp;author=X%20Wang&amp;author=F%20Shi&amp;author=HAB%20W%C3%B6sten&amp;author=H%20Hektor&amp;author=B%20Poolman&amp;volume=88&amp;publication_year=2005&amp;pages=3434-3443&amp;pmid=15749774&amp;doi=10.1529/biophysj.104.057794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0135">
<cite>
Waterhouse AM, Studer G, Robin X, Bienert S, Tauriello G, Schwede T. The structure assessment web server: for proteins, complexes and more. Nucleic Acids Res. 2024;52:W318–W323.
</cite> [<a href="https://doi.org/10.1093/nar/gkae270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38634802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20structure%20assessment%20web%20server:%20for%20proteins,%20complexes%20and%20more&amp;author=AM%20Waterhouse&amp;author=G%20Studer&amp;author=X%20Robin&amp;author=S%20Bienert&amp;author=G%20Tauriello&amp;volume=52&amp;publication_year=2024&amp;pages=W318-W323&amp;pmid=38634802&amp;doi=10.1093/nar/gkae270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0136">
<cite>
Wayment‐Steele HK, Ojoawo A, Otten R, Apitz JM, Pitsawong W, Hömberger M, et al. Predicting multiple conformations via sequence clustering and AlphaFold2. Nature. 2024;625:832–839.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06832-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10808063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37956700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Predicting%20multiple%20conformations%20via%20sequence%20clustering%20and%20AlphaFold2&amp;author=HK%20Wayment%E2%80%90Steele&amp;author=A%20Ojoawo&amp;author=R%20Otten&amp;author=JM%20Apitz&amp;author=W%20Pitsawong&amp;volume=625&amp;publication_year=2024&amp;pages=832-839&amp;pmid=37956700&amp;doi=10.1038/s41586-023-06832-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0137">
<cite>
Wessels J, de Vries O, Asgeirsdottir S, Schuren F. Hydrophobin genes involved in formation of aerial hyphae and fruit bodies in Schizophyllum. Plant Cell. 1991;3:793–799.
</cite> [<a href="https://doi.org/10.1105/tpc.3.8.793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12324614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Hydrophobin%20genes%20involved%20in%20formation%20of%20aerial%20hyphae%20and%20fruit%20bodies%20in%20Schizophyllum&amp;author=J%20Wessels&amp;author=O%20de%20Vries&amp;author=S%20Asgeirsdottir&amp;author=F%20Schuren&amp;volume=3&amp;publication_year=1991&amp;pages=793-799&amp;pmid=12324614&amp;doi=10.1105/tpc.3.8.793&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0138">
<cite>
Wessels JG. Hydrophobins: proteins that change the nature of the fungal surface. Adv Microb Physiol. 1997;38:1–45.
</cite> [<a href="https://doi.org/10.1016/s0065-2911(08)60154-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8922117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Microb%20Physiol&amp;title=Hydrophobins:%20proteins%20that%20change%20the%20nature%20of%20the%20fungal%20surface&amp;author=JG%20Wessels&amp;volume=38&amp;publication_year=1997&amp;pages=1-45&amp;pmid=8922117&amp;doi=10.1016/s0065-2911(08)60154-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0139">
<cite>
Wessels JG, de Vries OM, Asgeirsdóttir SA, Springer J. The thn mutation of Schizophyllum commune, which suppresses formation of aerial hyphae, affects expression of the Sc3 hydrophobin gene. J Gen Microbiol. 1991;137:2439–2445.
</cite> [<a href="https://doi.org/10.1099/00221287-137-10-2439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1770359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Microbiol&amp;title=The%20thn%20mutation%20of%20Schizophyllum%20commune,%20which%20suppresses%20formation%20of%20aerial%20hyphae,%20affects%20expression%20of%20the%20Sc3%20hydrophobin%20gene&amp;author=JG%20Wessels&amp;author=OM%20de%20Vries&amp;author=SA%20Asgeirsd%C3%B3ttir&amp;author=J%20Springer&amp;volume=137&amp;publication_year=1991&amp;pages=2439-2445&amp;pmid=1770359&amp;doi=10.1099/00221287-137-10-2439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0140">
<cite>
Whiteford JR, Spanu PD. Hydrophobins and the interactions between fungi and plants. Mol Plant Pathol. 2002;3:391–400.
</cite> [<a href="https://doi.org/10.1046/j.1364-3703.2002.00129.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20569345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant%20Pathol&amp;title=Hydrophobins%20and%20the%20interactions%20between%20fungi%20and%20plants&amp;author=JR%20Whiteford&amp;author=PD%20Spanu&amp;volume=3&amp;publication_year=2002&amp;pages=391-400&amp;pmid=20569345&amp;doi=10.1046/j.1364-3703.2002.00129.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0141">
<cite>
Wilkins MR, Gasteiger E, Bairoch A, Sanchez JC, Williams KL, Appel RD, et al. Protein identification and analysis tools in the ExPASy server. Methods Mol Biol. 1999;112:531–552.
</cite> [<a href="https://doi.org/10.1385/1-59259-584-7:531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10027275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=Protein%20identification%20and%20analysis%20tools%20in%20the%20ExPASy%20server&amp;author=MR%20Wilkins&amp;author=E%20Gasteiger&amp;author=A%20Bairoch&amp;author=JC%20Sanchez&amp;author=KL%20Williams&amp;volume=112&amp;publication_year=1999&amp;pages=531-552&amp;pmid=10027275&amp;doi=10.1385/1-59259-584-7:531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0142">
<cite>
Wilkinson M, Gallardo RU, Martinez RM, Guthertz N, So M, Aubrey LD, et al. Disease‐relevant β2‐microglobulin variants share a common amyloid fold. Nat Commun. 2023;14:1190.
</cite> [<a href="https://doi.org/10.1038/s41467-023-36791-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9981686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36864041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Disease%E2%80%90relevant%20%CE%B22%E2%80%90microglobulin%20variants%20share%20a%20common%20amyloid%20fold&amp;author=M%20Wilkinson&amp;author=RU%20Gallardo&amp;author=RM%20Martinez&amp;author=N%20Guthertz&amp;author=M%20So&amp;volume=14&amp;publication_year=2023&amp;pages=1190&amp;pmid=36864041&amp;doi=10.1038/s41467-023-36791-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0143"><cite>
Wojciechowska AW, Wojciechowski JW, Kotulska M. Non‐standard proteins in the lenses of AlphaFold3 – case study of amyloids. <em>bioRxiv</em>; 2024, 2024.07.09.602655.</cite></li>
<li id="pro70279-bib-0144">
<cite>
Wösten HA. Hydrophobins: multipurpose proteins. Ann Rev Microbiol. 2001;55:625–646.
</cite> [<a href="https://doi.org/10.1146/annurev.micro.55.1.625" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11544369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Rev%20Microbiol&amp;title=Hydrophobins:%20multipurpose%20proteins&amp;author=HA%20W%C3%B6sten&amp;volume=55&amp;publication_year=2001&amp;pages=625-646&amp;pmid=11544369&amp;doi=10.1146/annurev.micro.55.1.625&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0145">
<cite>
Wösten HA, de Vocht ML. Hydrophobins, the fungal coat unravelled. Biochim Biophys Acta. 2000;1469:79–86.
</cite> [<a href="https://doi.org/10.1016/s0304-4157(00)00002-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10998570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Hydrophobins,%20the%20fungal%20coat%20unravelled&amp;author=HA%20W%C3%B6sten&amp;author=ML%20de%20Vocht&amp;volume=1469&amp;publication_year=2000&amp;pages=79-86&amp;pmid=10998570&amp;doi=10.1016/s0304-4157(00)00002-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0146">
<cite>
Wösten HA, Schuren FH, Wessels JG. Interfacial self‐assembly of a hydrophobin into an amphipathic protein membrane mediates fungal attachment to hydrophobic surfaces. EMBO J. 1994;13:5848–5854.
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1994.tb06929.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC395559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7813424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J&amp;title=Interfacial%20self%E2%80%90assembly%20of%20a%20hydrophobin%20into%20an%20amphipathic%20protein%20membrane%20mediates%20fungal%20attachment%20to%20hydrophobic%20surfaces&amp;author=HA%20W%C3%B6sten&amp;author=FH%20Schuren&amp;author=JG%20Wessels&amp;volume=13&amp;publication_year=1994&amp;pages=5848-5854&amp;pmid=7813424&amp;doi=10.1002/j.1460-2075.1994.tb06929.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0147">
<cite>
Wösten HA, van Wetter MA, Lugones LG, van der Mei HC, Busscher HJ, Wessels JG. How a fungus escapes the water to grow into the air. Curr Biol. 1999;9:85–88.
</cite> [<a href="https://doi.org/10.1016/s0960-9822(99)80019-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10021365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=How%20a%20fungus%20escapes%20the%20water%20to%20grow%20into%20the%20air&amp;author=HA%20W%C3%B6sten&amp;author=MA%20van%20Wetter&amp;author=LG%20Lugones&amp;author=HC%20van%20der%20Mei&amp;author=HJ%20Busscher&amp;volume=9&amp;publication_year=1999&amp;pages=85-88&amp;pmid=10021365&amp;doi=10.1016/s0960-9822(99)80019-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0148">
<cite>
Xiong B‐J, Stanley CE, Dusny C, Schlosser D, Harms H, Wick LY. pH distribution along growing fungal hyphae at microscale. J Fungi (Basel). 2022;8:599.
</cite> [<a href="https://doi.org/10.3390/jof8060599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9224906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35736082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Fungi%20(Basel)&amp;title=pH%20distribution%20along%20growing%20fungal%20hyphae%20at%20microscale&amp;author=B%E2%80%90J%20Xiong&amp;author=CE%20Stanley&amp;author=C%20Dusny&amp;author=D%20Schlosser&amp;author=H%20Harms&amp;volume=8&amp;publication_year=2022&amp;pages=599&amp;pmid=35736082&amp;doi=10.3390/jof8060599&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0149">
<cite>
Xu J, Zhang Y. How significant is a protein structure similarity with TM‐score = 0.5?
Bioinformatics. 2010;26:889–895.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2913670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20164152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=How%20significant%20is%20a%20protein%20structure%20similarity%20with%20TM%E2%80%90score%E2%80%89=%E2%80%890.5?&amp;author=J%20Xu&amp;author=Y%20Zhang&amp;volume=26&amp;publication_year=2010&amp;pages=889-895&amp;pmid=20164152&amp;doi=10.1093/bioinformatics/btq066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0150">
<cite>
Yang J, Wang B, Ge L, Yang X, Wang X, Dai Y, et al. The enhancement of surface activity and nanoparticle stability through the alteration of charged amino acids of HGFI. Colloids Surf B Biointerfaces. 2019;175:703–712.
</cite> [<a href="https://doi.org/10.1016/j.colsurfb.2018.12.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30594056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Colloids%20Surf%20B%20Biointerfaces&amp;title=The%20enhancement%20of%20surface%20activity%20and%20nanoparticle%20stability%20through%20the%20alteration%20of%20charged%20amino%20acids%20of%20HGFI&amp;author=J%20Yang&amp;author=B%20Wang&amp;author=L%20Ge&amp;author=X%20Yang&amp;author=X%20Wang&amp;volume=175&amp;publication_year=2019&amp;pages=703-712&amp;pmid=30594056&amp;doi=10.1016/j.colsurfb.2018.12.044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0151">
<cite>
Yu F, Sukenik S. Structural preferences shape the entropic force of disordered protein ensembles. J Phys Chem B. 2023;127:4235–4244.
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.3c00698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10201532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37155239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Phys%20Chem%20B&amp;title=Structural%20preferences%20shape%20the%20entropic%20force%20of%20disordered%20protein%20ensembles&amp;author=F%20Yu&amp;author=S%20Sukenik&amp;volume=127&amp;publication_year=2023&amp;pages=4235-4244&amp;pmid=37155239&amp;doi=10.1021/acs.jpcb.3c00698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0152">
<cite>
Zhang X, Blalock B, Huberty W, Chen Y, Hung F, Russo PS. Microbubbles and oil droplets stabilized by a class II Hydrophobin in Marinelike environments. Langmuir. 2019;35:4380–4386.
</cite> [<a href="https://doi.org/10.1021/acs.langmuir.8b03777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30873841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Langmuir&amp;title=Microbubbles%20and%20oil%20droplets%20stabilized%20by%20a%20class%20II%20Hydrophobin%20in%20Marinelike%20environments&amp;author=X%20Zhang&amp;author=B%20Blalock&amp;author=W%20Huberty&amp;author=Y%20Chen&amp;author=F%20Hung&amp;volume=35&amp;publication_year=2019&amp;pages=4380-4386&amp;pmid=30873841&amp;doi=10.1021/acs.langmuir.8b03777&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0153">
<cite>
Zhang Y, Skolnick J. Scoring function for automated assessment of protein structure template quality. Proteins. 2004;57:702–710.
</cite> [<a href="https://doi.org/10.1002/prot.20264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15476259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Scoring%20function%20for%20automated%20assessment%20of%20protein%20structure%20template%20quality&amp;author=Y%20Zhang&amp;author=J%20Skolnick&amp;volume=57&amp;publication_year=2004&amp;pages=702-710&amp;pmid=15476259&amp;doi=10.1002/prot.20264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0154">
<cite>
Zhao L, Xu H, Li Y, Song D, Wang X, Qiao M, et al. Novel application of hydrophobin in medical science: a drug carrier for improving serum stability. Sci Rep. 2016;6:26461.
</cite> [<a href="https://doi.org/10.1038/srep26461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4876437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27212208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Novel%20application%20of%20hydrophobin%20in%20medical%20science:%20a%20drug%20carrier%20for%20improving%20serum%20stability&amp;author=L%20Zhao&amp;author=H%20Xu&amp;author=Y%20Li&amp;author=D%20Song&amp;author=X%20Wang&amp;volume=6&amp;publication_year=2016&amp;pages=26461&amp;pmid=27212208&amp;doi=10.1038/srep26461&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70279-bib-0155">
<cite>
Zhao Z, Cai F, Gao R, Ding M, Jiang S, Chen P, et al. At least three families of hyphosphere small secreted cysteine‐rich proteins can optimize surface properties to a moderately hydrophilic state suitable for fungal attachment. Environ Microbiol. 2021;23:5750–5768.
</cite> [<a href="https://doi.org/10.1111/1462-2920.15413" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8596622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33538393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&amp;title=At%20least%20three%20families%20of%20hyphosphere%20small%20secreted%20cysteine%E2%80%90rich%20proteins%20can%20optimize%20surface%20properties%20to%20a%20moderately%20hydrophilic%20state%20suitable%20for%20fungal%20attachment&amp;author=Z%20Zhao&amp;author=F%20Cai&amp;author=R%20Gao&amp;author=M%20Ding&amp;author=S%20Jiang&amp;volume=23&amp;publication_year=2021&amp;pages=5750-5768&amp;pmid=33538393&amp;doi=10.1111/1462-2920.15413&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375992/bin/PRO-34-e70279-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70279-s001.docx</a><sup> (24.1MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All AlphaFold models, scripts used to generate the data, and source data are freely provided on GitHub at <a href="https://github.com/mcshanlab/Yang_hydrophobins_2025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/mcshanlab/Yang_hydrophobins_2025</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70279"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70279.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375992/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375992/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375992%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375992/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375992/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375992/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375992/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375992/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375992/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ZBIAovIZNq5IJj0wpcR8TT71CIPGON26nM8K36WC7amHaoqGtZ5d4LjrMubVmj4b">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
