
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Chemically modified tRNA enhances the translation capacity of mRNA rich in cognate codons - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525375D8AF1CB6305375D0042681598.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373798/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Chemically modified tRNA enhances the translation capacity of mRNA rich in cognate codons">
<meta name="citation_author" content="Liangzhen Dong">
<meta name="citation_author_institution" content="State Key Laboratory of Natural and Biomimetic Drugs, Department of Molecular and Cellular Pharmacology, School of Pharmaceutical Sciences, Peking University, Beijing, China">
<meta name="citation_author" content="Jiayu Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Natural and Biomimetic Drugs, Department of Molecular and Cellular Pharmacology, School of Pharmaceutical Sciences, Peking University, Beijing, China">
<meta name="citation_author" content="Qing Xia">
<meta name="citation_author_institution" content="State Key Laboratory of Natural and Biomimetic Drugs, Department of Molecular and Cellular Pharmacology, School of Pharmaceutical Sciences, Peking University, Beijing, China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7825">
<meta name="citation_doi" content="10.1038/s41467-025-62981-7">
<meta name="citation_pmid" content="40846860">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373798/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373798/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373798/pdf/41467_2025_Article_62981.pdf">
<meta name="description" content="Although messenger RNA (mRNA) vaccines have been employed to prevent the spread of COVID-19, they are still limited by instability and low translation capacity. Alterations in tRNA abundance and modification, linking codon optimality, impact mRNA ...">
<meta name="og:title" content="Chemically modified tRNA enhances the translation capacity of mRNA rich in cognate codons">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Although messenger RNA (mRNA) vaccines have been employed to prevent the spread of COVID-19, they are still limited by instability and low translation capacity. Alterations in tRNA abundance and modification, linking codon optimality, impact mRNA ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373798/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373798">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62981-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62981.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373798%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373798/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373798/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373798/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7825. doi: <a href="https://doi.org/10.1038/s41467-025-62981-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62981-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Chemically modified tRNA enhances the translation capacity of mRNA rich in cognate codons</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Liangzhen Dong</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Liangzhen Dong</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Natural and Biomimetic Drugs, Department of Molecular and Cellular Pharmacology, School of Pharmaceutical Sciences, Peking University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Liangzhen Dong</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jiayu Wang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jiayu Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Natural and Biomimetic Drugs, Department of Molecular and Cellular Pharmacology, School of Pharmaceutical Sciences, Peking University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiayu Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Qing Xia</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Qing Xia</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Natural and Biomimetic Drugs, Department of Molecular and Cellular Pharmacology, School of Pharmaceutical Sciences, Peking University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qing Xia</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Natural and Biomimetic Drugs, Department of Molecular and Cellular Pharmacology, School of Pharmaceutical Sciences, Peking University, Beijing, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 16; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373798  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846860/" class="usa-link">40846860</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Although messenger RNA (mRNA) vaccines have been employed to prevent the spread of COVID-19, they are still limited by instability and low translation capacity. Alterations in tRNA abundance and modification, linking codon optimality, impact mRNA stability and protein output in a codon-dependent manner, suggesting tRNA as a potential translation enhancer. Here, we report a strategy named tRNA-plus to augment translation via artificially modulating tRNA availability. Overexpression of specific tRNAs enhances the stability and translation efficiency of SARS-CoV-2 Spike mRNA, boosting protein levels up to 4.7-fold. Additionally, chemically synthesized tRNAs bearing multiple site-specific modifications, particularly at the anticodon-loop and TΨC-loop, exhibit on average ~4-fold higher decoding efficacy than unmodified tRNAs, along with increased stability and reduced immunotoxicity. Furthermore, codelivery of Spike mRNA vaccine and tRNA through lipid nanoparticles elicits augmented humoral and cellular immune responses in vivo. These results presented here provide a general approach to elevate mRNA translation potency, with applications in diverse translation-based fields.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> tRNAs, Chemical modification, RNA vaccines</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The Covid-19 vaccinations showed the potential for mRNA vaccines; however, the technology still has low translation capacity. Here, the authors report on a method of improving mRNA stability and translation by overexpressing tRNAs significantly bosting protein production.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">mRNA vaccines have been proven to be viable tools during the coronavirus disease 2019 (COVID-19) pandemic, owing to their safety, ease of design and synthesis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. However, conventional mRNA molecules tend to be chemically unstable and exhibit low protein expression efficacy, thus compromising their clinical applications as vaccines and therapeutics<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. To address these challenges, efforts to improve the translation capacity of mRNA are necessary.</p>
<p id="Par4">mRNA translation output is determined by the overall balance between translation and degradation<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Existing strategies have primarily focused on optimizing mRNA vectors for better adaptation to translation machinery or reduced recognition by degradation complexes. For instance, incorporation of 2’-O-methylated Cap analogs, such as Cap1 and Cap2, enhances the recognition by translation initiation factor eIF4E, conferring high initiation probability to mRNA<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Codon optimization, combined with increased secondary structure that extends mRNA lifetime, has been shown to enhance protein expression<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. The use of multi-capped or branched-poly(A) tailed structures promotes the stability and translation efficiency of mRNA by augmenting the interaction with eIF4E or poly(A)-binding protein, while blocking the accessibility of decapping or deadenylation enzymes<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Similarly, circular RNA (circRNA), which is resistant to exonuclease cleavage, also promises to broaden the pharmacokinetic window of mRNA<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. However, these RNAs are inevitably limited by issues related to RNA design, modifications incompatibility, synthesis, yields and scalability. Therefore, we assumed that a translation enhancer could augment protein output without any complicated manipulations in mRNA sequence and structure.</p>
<p id="Par5">Transfer RNAs (tRNA), the primary interpreters of genetic codes, dynamically vary in their abundance and modification (tRNA availability), modulating the translation efficiency and accuracy of mRNA in a codon-specific manner<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. To ensure effective protein synthesis and maintain cellular homeostasis, the anticodon demand of mRNAs must be balanced with the cellular tRNA supply<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Distinct tRNA signatures correlated with the codon usage of highly expressed proteins have been well characterized in diverse human tissues, during proliferation and differentiation, as well as under stress conditions<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. In general, codons pairing to high-abundance tRNAs (optimal) are translated more rapidly than those read by low-abundance tRNAs (non-optimal)<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. This concept is termed codon optimality, determined (in part) by the tRNA availability<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. In addition to the canonical decoding function, tRNA is also a major determinant of mRNA stability (Fig. <a href="#Fig1" class="usa-link">1a</a>). mRNA translation and degradation are closely coupled as the state of translating ribosomes are strictly sensed by degradation machinery<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Specifically, when a ribosome encounters a non-optimal codon, it may pause transiently due to the shortage of cognate tRNA, leaving the A and E sites empty and enabling the engagement of deadenylation complex to accelerate mRNA degradation<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Importantly, specific tRNAs in the P sites can serve as the adapters to recruit the regulatory complex<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Upregulation of tRNA abundance or modification has been demonstrated to improve codon optimality, and thereby enhances the stability and translation of mRNA rich in cognate codons, associated with tumor metastasis, antibody production in plasma cells, activation and proliferation of T cells<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Given tRNA is such an essential translation machinery, we postulated that artificial modulation of tRNA availability could achieve the efficient translation of cognate codon-rich mRNAs, termed the tRNA-plus strategy (Fig. <a href="#Fig1" class="usa-link">1b</a>). Recently, AAV- or LNP-delivered engineered tRNAs have emerged as potential therapeutics for premature termination codons (PTC) diseases, promising the in vivo safety and efficacy of tRNAs<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. We further hypothesized that inclusion of tRNA into mRNA vaccine could augment its translation capacity and immunogenicity.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Screens of tRNA in promoting spike protein expression.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373798_41467_2025_62981_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30ba/12373798/2c3587915f05/41467_2025_62981_Fig1_HTML.jpg" loading="lazy" id="d33e355" height="999" width="710" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic diagram illustrating the relationship between codon optimality, tRNA availability and mRNA stability. <strong>b</strong> Schematics of the tRNA-plus strategy, showing that supply of tRNA could contribute to faster translation elongation. Figure was generated using BioRender. <strong>c</strong> Codon usage analysis of Spike (B.1.1.529) mRNA. <strong>d</strong> Codon stable coefficient (CSC) analysis of Spike mRNA. <strong>e</strong> Combined analysis of the codon usage and CSC of Spike mRNA. The calculated equation and stability score were shown. <strong>f</strong> Differential profiling of tRNA isodecoders expression in SARS-CoV-2-infected versus uninfected Vero E6 cells. Scattered points on the right of the red line represent upregulated tRNAs in cells after viral infection, while those on the left represent downregulated ones. Threshold: |log2FC | &gt; 1 and Padj &lt; 0.05. <strong>g</strong> Primary tRNA library consisted of four classes of tRNAs, with those labeled in gray serving as the control. Figure was generated using BioRender. <strong>h</strong> Workflow for the screening of tRNAs to increase spike protein expression, assessed by the intensity of enhanced green fluorescence protein (EGFP) fluorescence and western blotting (WB). Constructs expressing Spike-E2A-EGFP and natural tRNA are shown. Figure was generated using BioRender. <strong>i</strong> Determination of the optimal ratio of Spike to tRNA plasmid, ranging from 1:1 to 1:4. Representative fluorescence images and quantification of the relative fluorescence intensity of EGFP using ImageJ software (<em>n</em> = 3 biological replicates). Blank vector was used as the control. Scale bars, 100 μm. <strong>j</strong> Relative expression of spike protein normalized to the control in (<strong>i</strong>). Data are presented as the mean value of two biological replicates. <strong>k</strong> Quantification of spike protein expression following overexpression of different tRNAs (<em>n</em> = 3 biological replicates). Statistical significance was analysed using one-way ANOVA analysis with Dunnett’s correction (<strong>i, k</strong>). NS, not significant. Data are presented as mean ± s.d. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par6">In this study, we systematically scored codon contributions to the stability of SARS-CoV-2 Spike mRNA through a combined analysis of codon usage and codon optimality, which linked codon content to mRNA stability. Our results demonstrated that tRNAs corresponding to high-scoring codons were of significant capability in promoting protein expression. In addition, since modifications are crucial for tRNA functionality, we engineered and synthesized tRNAs with multiple site-specific modifications, specifically at the anticodon loop–which modulate the decoding rate and fidelity–and the TΨC loop that enhance the affinity for elongation factor<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Furthermore, we leveraged an LNP system for codelivery of Spike mRNA and tRNA, termed the mRNA-tRNA vaccine, to assess the spike-specific humoral and cellular immune responses in vivo.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Selection of potential tRNAs to promote Spike mRNA translation</h3>
<p id="Par7">The differential expression of genomes and long-term coevolution between codons and tRNAs shape host codon usage bias, implying that the codon adaptation extent of an exogenous gene could influence its protein expression in the host<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Codon usage analysis showed that ten codons (TTC, CTG, GTG, ATC, AAC, AGC, GCC, ACC, AAG, TAC) were 1.67- to 2.65-fold more extensively used in SARS-CoV-2 Spike (B.1.1.529) mRNA compared to the host, suggesting higher demands for the corresponding tRNA anticodons (Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Codons can be divided into optimal and non-optimal categories according to their codon optimality and codon stable coefficient (CSC)—defined as the Pearson correlation coefficient between mRNA stability and codon occurrence<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Increasing codon optimality has been shown to enhance the stability and translation efficiency of mRNAs rich in cognate codons, thereby elevating protein output<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The CSC analysis revealed that Spike mRNA consisted of 856 optimal codons and 414 non-optimal codons (Fig. <a href="#Fig1" class="usa-link">1d</a>). To quantify the contribution of each codon to mRNA stability, a scoring system based on the joint analysis of codon frequency and CSC was established. These stability scores represent the cumulative regulatory effects of codon content on mRNA translation (Fig. <a href="#Fig1" class="usa-link">1e</a> and Supplementary Table. <a href="#MOESM1" class="usa-link">1</a>). Accordingly, tRNAs corresponding to codons (either optimal or non-optimal) with high absolute scores were thought to have the potential to promote the translation of target mRNAs.</p>
<p id="Par8">Considering that multiple tRNA isodecoders can recognize the same codon, selection of tRNA variants with high decoding efficiency is essential<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Engineered tRNAs derived from natural tRNAs with altered anticodons could achieve PTC suppression, and their readthrough efficacy may be indicative of the decoding ability of cognate tRNAs<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. This suggests that tRNA isodecoders demonstrating high readthrough activity should be prioritized. For instance, among the multiple tRNA<sup>Ala</sup>AGC isodecoders, tRNA<sup>Ala</sup>AGC-2-1 was ultimately selected based on the high readthrough efficiency of its engineered tRNA-CUA variant, high cellular abundance, and favorable tRNAScan-SE scores<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Notably, several hyperutilized codons, which lack complementary tRNA genes based on the Watson-Crick base-pairing rules, such as GCC (tRNA<sup>Ala</sup>GGC), ACC (tRNA<sup>Thr</sup>GGT), and CCC (tRNA<sup>Pro</sup>GGG), are decoded via wobble base-pairing of tRNAs, implying a possibility that conversion of Wobble-tRNAs (referred to near-cognate tRNAs) into Watson-Crick-tRNA (referred to cognate tRNAs) could improve their decoding efficiency.</p>
<p id="Par9">In addition, tRNA profiling of SARS-CoV-2-infected versus uninfected Vero E6 cells revealed significant upregulation of several tRNA isodecoders, including tRNA<sup>Val</sup>TAC-2-1, tRNA<sup>His</sup>GTG-1-1/-2-1, tRNA<sup>Gly</sup>GCC-1-1/-5-1, tRNA<sup>Glu</sup>TTC-3-1, and tRNA<sup>Glu</sup>CTC-1-1, probably to meet the increased anticodon demand during viral amplification (Fig. <a href="#Fig1" class="usa-link">1f</a>). Conversely, certain tRNA molecules showed decreased abundance upon viral infection, suggesting repressed transcription or active degradation as the plausible mechanism leading to such drastic downregulation<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> (Fig. <a href="#Fig1" class="usa-link">1f</a>). Furthermore, the SARS-CoV-2 non-structural protein 1 (Nsp1) has been shown to inhibit host translation while fostering viral translation<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, which indicated that the reduced host translation demand may drive this alteration of tRNA levels. Collectively, the primary tRNA library consisting of 34 tRNAs was divided into optimal, less optimal, non-optimal, and less non-optimal tRNAs according to the stability scores of their cognate codons (Fig. <a href="#Fig1" class="usa-link">1g</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Overexpression of tRNA increases spike protein synthesis</h3>
<p id="Par10">To evaluate tRNA functionality in promoting the translation of Spike mRNA, constructs expressing Spike-2A-EGFP and natural tRNA were co-transfected into HEK293T cells (Fig. <a href="#Fig1" class="usa-link">1h</a>). As tRNA abundance is crucial for ribosome decoding efficiency<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, a 1:4 ratio of Spike to tRNA was found to outperform other ratios and was applied to subsequent screens (Fig. <a href="#Fig1" class="usa-link">1i-j</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>). Our results showed that many of these tRNAs improved spike protein expression, with optimal tRNAs being more effective than non-optimal tRNAs (Fig. <a href="#Fig1" class="usa-link">1k</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2b-c</a>). For optimal tRNAs, the overexpression of several tRNAs, including tRNA<sup>Phe</sup>GAA-3-1, tRNA<sup>Leu</sup>CAG-1-1, and tRNA<sup>Ala</sup>GGC-2-1, showed a considerable improvement in protein synthesis, resulting in 3.5- to 4.7-fold greater protein levels compared to the control. Among non-optimal tRNAs, tRNA<sup>Pro</sup>AGG-1-1 led to the highest increase in protein expression, followed by tRNA<sup>His</sup>GTG-2-1. Notably, tRNA<sup>Ala</sup>GGC-2-1, derived from tRNA<sup>Ala</sup>AGC-2-1, exhibited superior performance compared to its original sequence, consistent with previous findings that codons decoded via wobble pairing are translated more slowly than those read by Watson-Crick pairing<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. This suggested that the existing sequences of natural tRNAs may not be optimized for maximal translation efficiency. For tRNAs varied during viral infection, overexpressing upregulated tRNAs (tRNA<sup>Gly</sup>GCC-5-1, tRNA<sup>Glu</sup>CTC-1-1, tRNA<sup>Glu</sup>TTC-3-1, and tRNA<sup>His</sup>GTG−2-1/-1-1) yielded higher protein production than those of downregulated tRNAs (tRNA<sup>Glu</sup>CTC-3-1, tRNA<sup>Sec</sup>TCA-1-1), with tRNA<sup>His</sup>GTG-2-1 being the most potent among them. Such enhanced efficacy was absent when tRNAs with low tRNAScan-SE scores (tRNA<sup>Phe</sup>GAA-11-1) or low absolute stability scores (tRNA<sup>Tyr</sup>ATA-1-1, tRNA<sup>Gln</sup>CTG-6-1) were overexpressed (Fig. <a href="#Fig1" class="usa-link">1k</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2b-c</a>). Furthermore, given the documented variations in tRNA pool composition across distinct cell types<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, we then selected functionally verified tRNAs to validate the effects originating from HEK293T system in HepG2 cells. Consistently, overexpression of tRNAs (including tRNA<sup>Phe</sup>GAA-3-1, tRNA<sup>Ile</sup>GAT-1-1, tRNA<sup>Leu</sup>CAG-1-1, and tRNA<sup>Ala</sup>GGC-2-1) significantly enhanced spike protein production, demonstrating the feasibility of tRNA-plus in other cellular systems (Supplementary Fig. <a href="#MOESM1" class="usa-link">2d</a>). Moreover, a positive correlation between protein levels and actual or absolute stability scores (r = 0.52 or 0.54, respectively) was observed, particularly for the functional tRNAs (r = 0.68 or 0.63, respectively), supporting that tRNAs correlated with high-scoring codons held the promise for enhancing protein expression (Fig. <a href="#Fig2" class="usa-link">2a, b</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. tRNA-plus enhances the translation of codon-optimized Spike mRNA and packaging of Spike pseudovirus.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373798_41467_2025_62981_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30ba/12373798/cdb5d66275db/41467_2025_62981_Fig2_HTML.jpg" loading="lazy" id="d33e620" height="1021" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong> Relationship between spike protein expression and actual or absolute stability scores of codons corresponding to all or functional tRNAs (<em>n</em> = 3 biological replicates). <strong>c</strong> Schematic diagram illustrating more rapid translation resulting from the combination of codon-optimized mRNA and tRNA overexpression. Figure was generated using BioRender. <strong>d</strong> Codon usage analysis of codon-optimized Spike mRNA (S-OP). <strong>e</strong> Joint analysis of the codon usage and CSC of S-OP. The stability score was shown. <strong>f</strong> Representative western blot images showing increases in spike protein expression following tRNA overexpression. Left, S-OP; right, S-WT. The experiment was independently repeated three times with consistent results. <strong>g</strong> Left, schematics depicting that spike pseudovirus infects host cells through S1-ACE2 interaction; right, constructs expressing spike protein and EGFP-2A-Nluc, are used for pseudoviral packaging. Figure was generated using BioRender. <strong>h</strong> Representative western blot images and quantification of spike protein expression from PMD2.G and PCE vector (<em>n</em> = 3 biological replicates). <strong>i</strong>, <strong>j</strong> Representative fluorescence images of pseudoviral packaging in HEK293T cells and viral infection in HEK293T-ACE2 cells. Scale bars, 100 μm. The experiment was independently repeated three times with consistent results. <strong>k</strong> Schematic diagram showing that stably overexpressing tRNA could increase spike protein expression and viral packaging. Figure was generated using BioRender. <strong>l</strong> Expression of spike protein in HEK293T and HEK293T-tL cells. Representative fluorescence images and quantification of the MFI of EGFP by flow cytometry (<em>n</em> = 3 biological replicates). Scale bars, 100 μm. <strong>m</strong> Representative fluorescence images of HEK293T-ACE2 cells infected with pseudovirus and titration of pseudovirus titer based on the luminescence. Data are presented as the mean value of two biological replicates. HEK293T-ACE2 cells only or HEK293T cells incubated with pseudovirus served as the negative control. Scale bars, 250 μm. <em>P</em> values were calculated using an unpaired two-tailed <em>t</em>-test (<strong>h, l</strong>). Data are presented as mean ± s.d. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">tRNA overexpression synergizes with codon optimization</h3>
<p id="Par11">Given that codon optimization modulates the codon usage of an exogenous gene, whereas tRNA-plus alters the decoding rates of cognate codons, we anticipated that combination of both strategies could synergistically realize higher protein production than exclusive utility (Fig. <a href="#Fig2" class="usa-link">2c</a>). Spike-OP (codon-optimized) mRNA containing 1021 optimal codons and 249 non-optimal codons showed higher stability score than Spike-WT according to the scoring system (Fig. <a href="#Fig2" class="usa-link">2d, e</a>). Likewise, tRNA<sup>Phe</sup>GAA, tRNA<sup>Leu</sup>CAG, tRNA<sup>Val</sup>CAC, and tRNA<sup>Ala</sup>GGC were considered effective for improving Spike-OP translation. These, along with tRNA<sup>His</sup>GTG-2-1, were chosen for subsequent verification (Fig. <a href="#Fig2" class="usa-link">2d, e</a>). As expected, overexpressing these tRNAs enhanced protein expression irrespective of codon optimization, with Spike-OP producing much more protein than Spike-WT, implying an increased upper limit of protein expression capacity (Fig. <a href="#Fig2" class="usa-link">2f</a>).</p>
<p id="Par12">In addition, to determine whether tRNA-plus could improve viral packaging efficiency, a lentivirus-based spike pseudovirus packaging system was established, in which spike protein was expressed from either a lentivirus envelope vector (PMD2.G) or an Epstein-Barr Virus (EBV)-based vector (PCE) (Fig. <a href="#Fig2" class="usa-link">2g</a>). Compared to PMD2.G control, the PCE vector, which contains the EBNA1-OriP element known to facilitate the transcription of downstream genes<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, resulted in a considerable increase in pseudoviral packaging by elevating spike protein expression (Fig. <a href="#Fig2" class="usa-link">2h–j</a>). This suggested spike protein served as a crucial determinant of viral packaging. Similarly, stable overexpression of tRNA<sup>Leu</sup>CAG-1-1 significantly boosted both spike protein production and viral packaging, leading to a 4.62-fold increase in viral titer (Fig. <a href="#Fig2" class="usa-link">2k–m</a>). Together, these findings demonstrated the efficacy of tRNA-plus strategy in enhancing protein expression, which was further augmented through combination with other approaches like codon-optimization.</p></section><section id="Sec6"><h3 class="pmc_sec_title">tRNA-plus enhances the stability and translation efficiency of Spike mRNA</h3>
<p id="Par13">Translation affects the stability of mRNA in a codon-dependent manner<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> that is closely correlated to codon composition and optimality<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Optimal and non-optimal codons tend to pair with tRNAs that are highly or poorly expressed, respectively<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, suggesting that tRNA may also be a major determinant of mRNA stability. To assess whether changes in tRNA levels impact mRNA stability, α-amanitin-mediated or Tet-off-induced mRNA stability measurements were performed in tRNA-overexpression (tRNA-OE) and control cells, both pre-transfected with spike plasmids (Fig. <a href="#Fig3" class="usa-link">3a</a>–d). Cells stably overexpressing tRNA<sup>Leu</sup>CAG-1-1 or tRNA<sup>Ala</sup>GGC-2-1 were described as tL or tA, respectively (Fig. <a href="#Fig3" class="usa-link">3e, f</a>). As expected, both transient and consistent upregulation of tRNAs enhanced Spike mRNA stability, with observed inter-group differences indicating a codon-specific effect (Fig. <a href="#Fig3" class="usa-link">3b, c</a>, g, h).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Overexpressed tRNA increases the stability and translation efficiency of mRNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373798_41467_2025_62981_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30ba/12373798/0879b04c5931/41467_2025_62981_Fig3_HTML.jpg" loading="lazy" id="d33e790" height="988" width="706" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic illustration of how transient tRNA overexpression affects mRNA stability using α-amanitin-mediated mRNA decay assay or Tet-off-induced transcription stops of Spike mRNA. Figure was generated using BioRender. <strong>b</strong>, <strong>c</strong> Quantification of Spike mRNA decay rate in each group with transient tRNA overexpression (<em>n</em> = 3 biological replicates). <strong>d</strong> Schematic illustration of how stable tRNA overexpression affects mRNA stability using α-amanitin-mediated mRNA decay assay or doxycycline induced transcription stops of Spike mRNA. Figure was generated using BioRender. <strong>e</strong> Quantification of the tRNA mapped reads for tRNA<sup>Leu</sup>CAG-1 and tRNA<sup>Leu</sup>CAG-2 in WT and tL cells using tRNA sequencing (<em>n</em> = 3 biological replicates). <strong>f</strong> Quantification of the relative expression of tRNA<sup>Ala</sup>GGC-2-1 in WT and tA cells using qPCR (<em>n</em> = 3 biological replicates). <strong>g</strong>, <strong>h</strong> Measurements of mRNA decay rate in each group with stable tRNA overexpression (<em>n</em> = 4 biological replicates). <strong>i</strong> Workflow illustration of how stable tRNA overexpression affects the mRNA translation efficiency, as determined by quantifying the protein output for a given input of mRNA and ribosomal occupancy in Spike mRNA. Figure was generated using BioRender. <strong>j</strong>, Representative western blot images and quantification of spike protein expression, normalized to WT cells (plasmid, <em>n</em> = 4 biological replicates; mRNA, <em>n</em> = 5 biological replicates). Left, Spike plasmid; right, IVT-Spike mRNA. <strong>k</strong> Ribosome density profiles of Spike mRNA translation monitored by ribosome profiling in WT, HEK293T-tL, and HEK293T-tA cells (<em>n</em> = 2 biological replicates). Red dashed lines denote the start and stop codons of the Spike coding sequence (CDS). RPM, reads per million mapped reads. Statistical significance was analysed using unpaired two-tailed <em>t</em>-test (<strong>e, f</strong>) or one-way ANOVA with Dunnett’s correlation (<strong>b</strong>, <strong>c</strong>, <strong>g</strong>, <strong>h</strong>, <strong>j</strong>). NS, not significant. Data are presented as mean ± s.d. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par14">To determine whether this effect was limited to mRNA with a specific degree of codon-optimization, EGFP variants with distinct codon usages (EGFP-OP and DOP-1/2/3) were tested in HEK293T-mCherry cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>). Consistent with their stability scores, EGFP variants harboring higher proportions of optimal codons exhibited higher mRNA levels than those with fewer optimal codons (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>). Likewise, overexpression of tRNAs increased the mRNA stability and protein production of all EGFP variants, especially the codon-deoptimized form such as EGFP-DOP-3, indicating that tRNA-plus functioned similarly to codon-optimization (Supplementary Fig. <a href="#MOESM1" class="usa-link">3c, d</a>). In addition, to exclude the interference caused by simultaneous alteration of multiple codons in translation, we performed a codon mutagenesis experiment in which every optimal AAG codon in the EGFP-OP coding sequence was switched into the synonymous non-optimal AAA codon, resulting in a less optimal variant EGFP-AAA (Supplementary Fig. <a href="#MOESM1" class="usa-link">3e</a>). In contrast to the tRNA<sup>Val</sup>CAC control, tRNA<sup>Lys</sup>TTT overexpression demonstrated a considerable improvement in EGFP-AAA expression than EGFP-OP, implying a codon-specific effect of upregulated tRNAs on target mRNA (Supplementary Fig. <a href="#MOESM1" class="usa-link">3f–h</a>).</p>
<p id="Par15">Since altered tRNA levels directly impact the decoding rate of cognate codons, quantitative protein expression assays and ribosome profiling were conducted in tRNA-OE and WT cells to assess the changes in translation efficiency (Fig. <a href="#Fig3" class="usa-link">3i</a>). We observed that stable overexpression of tRNA<sup>Leu</sup>CAG-1-1 and tRNA<sup>Ala</sup>GGC-2-1 substantially increased spike protein output in terms of the same plasmid or mRNA input, which indicated a markedly enhanced translation efficiency (Fig. <a href="#Fig3" class="usa-link">3j</a>). Moreover, ribosome profiling revealed that overexpressed tRNA conferred higher ribosomal occupancy to Spike mRNA, with distinct ribosomal signatures for different tRNAs, confirming the augmented translation capacity of target mRNA (Fig. <a href="#Fig3" class="usa-link">3k</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">4a, b</a>). Additionally, to investigate whether tRNA upregulation had an impact on gene expression at both RNA and protein levels, we carried out RNA-seq experiment and quantitative proteomics in tRNA-OE cells and WT control cells. Differential expression analysis revealed that neither tRNA<sup>Leu</sup>CAG-1-1-OE nor tRNA<sup>Ala</sup>GGC-2-1-OE induced global changes in RNA or protein expressions (Supplementary Fig. <a href="#MOESM1" class="usa-link">4c–j</a>). However, since previous studies demonstrated that upregulation of tRNA<sup>Glu</sup>UUC globally modulated the translation of endogenous mRNA in a codon-specific manner<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, we speculated that this effect may depend on certain tRNA species and tRNA expression duration. In particular, consistent tRNA overexpression could potentially drive the transcriptomic and proteomic reprogramming through persistent codon-anticodon interactions. Ultimately, we concluded that alterations in tRNA levels were coupled with other translation machinery to coordinately regulate the translation and degradation of mRNA rich in cognate codons.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Modifications are necessary for RNA functionality</h3>
<p id="Par16">Given that tRNA-plus showed significant efficacy in enhancing protein expression, we postulated that combining tRNA into SARS-CoV-2 Spike mRNA vaccine could augment its translation capacity, thereby generating mRNA-tRNA vaccines that produce more antigen and elicit stronger immune responses. For mRNA-tRNA-based vaccines, three components are necessary: (1) mRNAs with efficient translation capacity, (2) tRNAs with high decoding ability, and (3) an LNP codelivery system.</p>
<p id="Par17">Modified nucleotides are commonly incorporated into in vitro-transcribed (IVT) mRNAs to reduce immunogenicity and improve translation capacity<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Consistent with previous findings<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, we demonstrated that complete substitution of U with m<sup>1</sup>Ψ achieved the highest encoding efficiency among various modifications. Replacement of A with Z decreased the translation capacity of mRNA, which was compensated for by converting the poly(Z) tail into poly(A) tail. Although N<sup>6</sup>-methylA (m<sup>6</sup>A) and N<sup>7</sup>-methylG (m<sup>7</sup>G) were compatible with T7 RNA polymerase, they unexpectedly abolished the encoding ability of mRNA, suggesting the site-specific functional regulation of these modifications (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a–c</a>).</p>
<p id="Par18">Similarly, the decoding capacity of tRNA is determined by its primary sequence, overlaid by extensive posttranscriptional modifications, some of which may improve its decoding efficiency<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Hypomodified tRNAs exhibit reduced decoding accuracy and rates due to the unstable codon-anticodon pairing with mRNAs. For instance, hypomodification of m<sup>1</sup>G37 leads to +1 frameshifting and dysregulation of mRNA abundance by nonsense-mediated decay<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. To assess the function of <em>in vitro-</em>synthesized (IVS) tRNAs in enhancing spike protein expression, IVS-tRNAs were co-transfected into HEK293T cells with either S-2P-Nluc or S-2P mRNA at a mass ratio of 1:1 (Supplementary Fig. <a href="#MOESM1" class="usa-link">5d, e</a>). tRNA<sup>Leu</sup>UUA-1-1 derived from tRNA<sup>Leu</sup>CAG-1-1 specifically recognized the nonsense codon UAA serving as the control. By contrast, unmodified tRNAs were associated with a reduction in spike protein expression, perhaps owing to the transient increase in unmodified tRNAs competitively inhibiting the access of functionalized cognate tRNAs to the ribosomal A-sites, resulting in diminished overall decoding rates of cognate codons (Supplementary Fig. <a href="#MOESM1" class="usa-link">5f–h</a>). This indicates that appropriate modifications are essential for tRNA functionality.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Engineering modified suppressor tRNA for high decoding efficiency</h3>
<p id="Par19">Considering the overall performance of tRNAs, tRNA<sup>Leu</sup>CAG-1-1 (denoted as Lt) was selected to investigate the effects of site-specific modifications on its decoding ability. Modifications that have been frequently studied, including those located at the D-loop (Gm18), anticodon loop (m<sup>5</sup>C34, Cm34, m<sup>1</sup>G37), and TΨC-loop (Ψ55, m<sup>1</sup>A58), were incorporated into the sequence of sup-tRNA<sup>Leu</sup>CUA (denoted as Lst), derived from Lt with altered anticodon, resulting in the respective variants Lst-1 ~ 6 (Fig. <a href="#Fig4" class="usa-link">4a, b</a>). For the initial screen, sup-tRNAs and EGFP<sup>39UAG</sup> mRNA were co-transfected into HEK293T cells, and the readthrough activity of sup-tRNA could reflect the decoding ability of cognate natural tRNA (Fig. <a href="#Fig4" class="usa-link">4c</a>). As expected, modifications in the anticodon loop markedly improved the readthrough ability of Lst, with Lst-3 (Cm34) exhibiting the highest efficiency, probably due to the wobble modification Cm34 that enhances the codon-anticodon interactions on ribosomes<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. In addition, m<sup>1</sup>A58, which has been shown to increase the affinity of tRNAs for the elongation factor eEF1A<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, also contributed to a substantial PTC suppression (Fig. <a href="#Fig4" class="usa-link">4d, e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Chemically modified sup-tRNA shows high decoding efficiency and stability.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373798_41467_2025_62981_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30ba/12373798/c91d539e855b/41467_2025_62981_Fig4_HTML.jpg" loading="lazy" id="d33e1017" height="989" width="741" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Left, Schematic diagram of tRNA<sup>Leu</sup>CAG with modified nucleotide substitutions in the D-loop (Gm18), the anticodon loop (m<sup>5</sup>C34, Cm34, m<sup>1</sup>G37), and the TΨC-loop (Ψ55, m<sup>1</sup>A58) highlighted in red. The double-headed arrows linking two modifications or nucleotides with amino acid indicate the potential interactions between these positions. Middle, the chemical structures of modified nucleotides. Right, sup-tRNA<sup>Leu</sup>CUA (Lst) with single and multiple modifications are listed in table (Lst-1 to Lst-13). <strong>b</strong> Sup-tRNA<sup>Leu</sup>CUA derived from tRNA<sup>Leu</sup>CAG recognizes UAG, whose readthrough efficacy can partially reflect the decoding ability of tRNA<sup>Leu</sup>CAG. Figure was generated using BioRender. <strong>c</strong> Screening of IVS-sup-tRNAs with IVT-EGFP<sup>39UAG</sup> mRNA in HEK293T cells. The readthrough efficiency of sup-tRNAs was determined from the fluorescence of PTC-EGFP normalized to WT-EGFP. IVT, in vitro-transcribed mRNA; IVS, in vitro-synthesized tRNA. Figure was generated using BioRender. <strong>d</strong> The readthrough efficiency of sup-tRNAs. Lst-UUA served as a negative control (<em>n</em> = 3 biological replicates). <strong>e</strong> Representative western blotting images in (<strong>d</strong>). <strong>f</strong>, Aminoacylation efficacy of sup-tRNAs determined by the in vitro aminoacylation assay (<em>n</em> = 3 biological replicates). <strong>g</strong>, Stability of sup-tRNAs determined by the method based on the HEK293T-SecNluc<sup>133UAG</sup> cell line (see Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>). Left, Relative luminescence of SecNluc protein in the supernatants over time, normalized to that of 2-hr point (<em>n</em> = 3 biological replicates); right, half-lives of tRNAs determined by one phase decay nonlinear regression in Prism (GraphPad) are listed. Statistical significance was analysed using one-way ANOVA with Dunnett’s correlation (<strong>d</strong>, <strong>f</strong>). NS, not significant. Data are presented as mean ± s.d. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par20">Since interactions between modified nucleotides or between nucleotides and charged amino acids residues are thought to facilitate tRNA functionality<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, multiple functionally relevant modifications were then simultaneously introduced into tRNA (Fig. <a href="#Fig4" class="usa-link">4a</a>). For Cm34-modified tRNAs, the further incorporated modifications led to a significant increase in PTC suppression efficiency, particularly m<sup>1</sup>G37 that promotes the translation fidelity and decoding rate of tRNA via stabilizing tRNA-ribosome interactions<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> (Fig. <a href="#Fig4" class="usa-link">4d, e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>). Besides, m<sup>1</sup>G37 markedly compensated for the reduced aminoacylation efficacy resulting from Cm34, which elevated the concentrations of functional tRNAs (Fig. <a href="#Fig4" class="usa-link">4f</a>).</p>
<p id="Par21">Next, to assess whether these introduced modifications could strengthen tRNA stability, HEK293T cells constitutively expressed Secretory (Sec) Nluc<sup>133UAG</sup> were used. Briefly, the readthrough of SecNluc<sup>133UAG</sup> mRNA occurred only in the presence of sup-tRNA-CUA, and thus tRNA levels could be inferred from the luminescence of SecNluc protein. This enabled tRNA half-life to be determined from real-time monitoring of SecNluc expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>). Notably, we observed that m<sup>1</sup>A58-modified tRNAs, such as Lst-6 (t<sub>1/2</sub> 6.117 h) and Lst-11 (t<sub>1/2</sub> 5.721 h), were more stable than Lst (t<sub>1/2</sub> 5.368 h), perhaps owing to the increased affinity for eEF1A that prevented modified-tRNA from degradation by the rapid tRNA decay (RTD) pathway<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Moreover, the interplay between Gm18 and Ψ55 that has been demonstrated to stabilize the L-shaped structure of tRNA<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, further lengthened tRNA half-life, such as Lst-13 (t<sub>1/2</sub> 7.529 h) (Fig. <a href="#Fig4" class="usa-link">4g</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6c</a>). Conversely, incorporation of any other modifications into m<sup>5</sup>C34-modified tRNA markedly reduced PTC readthrough, along with the attenuated aminoacylation efficacy and decreased stability (Fig. <a href="#Fig4" class="usa-link">4d–g</a>). Together, these findings indicate that modifications within the anticodon loop exert the strongest effect on the decoding capacity of tRNA, whereas modifications outside of this region enhance the stability of tRNA.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Modifications confer enhanced decoding efficiency to natural tRNA</h3>
<p id="Par22">To test the applicability of conclusions drawn from sup-tRNAs, identical modifications were introduced into Lt-CAG, resulting in a series of variants Lt1 ~ Lt13. In the initial evaluation, tRNAs were co-transfected into HEK293T cells with 5 × CUG-mCherry mRNA using Trans-It (Fig. <a href="#Fig5" class="usa-link">5a</a>). Under normal circumstances, the successive CUG codons located in the 5’ of mRNA impede the translocation of ribosomes due to the shortage of cognate tRNAs. Upon tRNA overexpression, ribosomes can pass through this region more easily, contributing to faster translation. For a given tRNA input, a tRNA with stronger decoding ability could achieve higher mCherry expression (Fig. <a href="#Fig5" class="usa-link">5a</a>). Similarly, modifications in the anticodon loop significantly improved the decoding ability of Lt, with Lt-4 (m<sup>1</sup>G37) being the most potent among single-modified variants, probably because of its high aminoacylation efficiency (Fig. <a href="#Fig5" class="usa-link">5b–d</a>). Moreover, the overexpression of multiple-modified tRNAs, including Lt-8, Lt-10, Lt-12, and Lt-13, demonstrated a 1.96- to 2.22-fold increase in the decoding efficiency (Fig. <a href="#Fig5" class="usa-link">5b–d</a>). In contrast to Lst, m<sup>5</sup>C34-modified tRNAs performed better than Cm34-modified tRNAs, possibly resulting from the structural deviations between Lt and Lst (Fig. <a href="#Fig5" class="usa-link">5b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7b</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Modifications confer enhanced decoding efficiency and reduced immunogenicity to natural tRNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373798_41467_2025_62981_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30ba/12373798/8f341fef0190/41467_2025_62981_Fig5_HTML.jpg" loading="lazy" id="d33e1178" height="681" width="799" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Top, workflow of assessing the decoding ability of tRNAs with 5 × CUG-mCherry mRNA in HEK293T cells. Middle and bottom, schematic diagram illustrating how this method works. Briefly, in normal situation, the successive CUG codons impede the movement of ribosomes due to the lack of tRNALeuCAG. Overexpression of cognate tRNAs alleviates the ribosomal traffic jam in this region and contributes to more rapid translation, the extent of which is associated with the decoding ability of input tRNAs. Figure was generated using BioRender. <strong>b</strong>, <strong>c</strong> Representative fluorescent images and quantification of the expression of mCherry using flow cytometry, normalized to that of Lt-CAG. Lst-UUA and Lst-CUA served as the negative controls (<em>n </em>= 3 biological replicates). Scale bars, 100 μm. <strong>d</strong> Aminoacylation efficacy of tRNAs determined by the in vitro aminoacylation assay (<em>n</em> = 3 biological replicates). <strong>e</strong>, <strong>f</strong> HEK293T-TLR8-NF-κB-EGFP cell line was used to measure the immunotoxicity of IVS-tRNA, as indicated from the percentage of EGFP positive cells (<em>n</em> = 3 biological replicates). Left, 0.2 μg of IVS-tRNA; right, 0.4 μg of IVS-tRNA. Figure was generated using BioRender. <strong>g</strong> TLR-dependent activation by IVS-tRNA was monitored in HEK293T-TLR8 cells with IL-8 secretion as the quantitative indicators (<em>n</em> = 3 biological replicates). Left, 0.2 μg of IVS-tRNA; right, 0.4 μg of IVS-tRNA. <strong>h</strong> Schematic diagram showing that tRNA immunogenicity was evaluated in M0 macrophages derived from THP-1 cells pretreated with PMA. R848, an agonist that activates both TLR7 and TLR8, served as a positive control. Mock, mock-transfected cells (<em>n</em> = 3 biological replicates). Figure was generated using BioRender. Statistical significance was analysed using one-way ANOVA with Dunnett’s correlation (<strong>b</strong>, <strong>d</strong>, <strong>f</strong>, <strong>h</strong>). NS, not significant. Data are presented as mean ± s.d. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par23">Irrespective of tRNA stability and structure, the total decoding efficiency of tRNA can be simplified into two aspects: (1) the aminoacylation efficiency, reflecting the abundance of charged-tRNAs (t<sub>1</sub>); and (2) the partial decoding efficiency, estimated as the ratio of protein output (p<sub>0</sub>) to charged-tRNA (t<sub>1</sub>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a</a>). A more striking positive correlation was found between the aminoacylation efficiency and total decoding efficiency of Lst (r = 0.89) compared to Lt (r = 0.62) that was restricted to single- and double-modified tRNAs (Supplementary Fig. <a href="#MOESM1" class="usa-link">7c-d</a>). Although Lst and Lt showed no obvious correlation in aminoacylation, their respective total and partial decoding efficiency showed significant positive correlations (r = 0.80 and 0.92, respectively), supporting that the readthrough activity of sup-tRNA could reflect the decoding ability of cognate natural tRNA (Supplementary Fig. <a href="#MOESM1" class="usa-link">7e-g</a>).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Modifications confer reduced immunogenicity to tRNA</h3>
<p id="Par24">Synthetic tRNAs may stimulate innate immunity through activation of Toll-like receptors, specifically TLR7 and TLR8, which are activated by viral or single-stranded RNA<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. To assess whether position-specific modifications of tRNA reduce its immune toxicity, HEK293T-TLR8 cells consistently harboring a nuclear factor-kappa B (NF-κB) reporter system and HEK293T-TLR8 cells were employed (Fig. <a href="#Fig5" class="usa-link">5e</a>–g). Compared to unmodified tRNA<sup>Leu</sup>CAG, variants with individual modifications (such as Gm18, m<sup>1</sup>G37, Ψ55, or m<sup>1</sup>A58) exhibited reduced immunogenicity (Fig. <a href="#Fig5" class="usa-link">5f, g</a>). For m<sup>5</sup>C34- or Cm34-modified tRNAs, additional modifications further decreased TLR8-dependent activation, particularly in the Lt-12 and Lt-13 variants (Fig. <a href="#Fig5" class="usa-link">5f, g</a>). This enhanced suppression likely arises from either the Gm18-Ψ55 interaction enhancing tRNA structural rigidity, or the global immune inhibitory Gm18 modification suppressing TLR signaling<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. To better simulate the recognition of tRNA by immune cells, a model was established based on THP-1 cells that, after phorbol 12-myristate 13-acetate (PMA) treatment, differentiated into M0 macrophages with highly expressed TLR7/8. Consistently, among all single-modified tRNAs, Gm18-, m<sup>1</sup>G37-, or Ψ55-modified variants marginally activated TLR7/8 to the extent of mock transfection (Fig. <a href="#Fig5" class="usa-link">5h</a>). Notably, 2’-O-methylation of C34 in tRNAs (such as Lt-3, Lt-9, Lt-11) also reduced the immunotoxicity (Fig. <a href="#Fig5" class="usa-link">5h</a>). Given the sequence-specific recognition mechanisms of TLR7/8<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, these immune inhibitory effects may be specific to the tRNA<sup>Leu</sup>CAG sequence, and whether these modifications confer similar reductions in immunogenicity across diverse tRNA sequences remains to be determined.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Codelivery of mRNA and tRNA using lipid nanoparticles</h3>
<p id="Par25">A lipid-based formulation for the co-encapsulation of mRNA and tRNA was established and characterized as follows: diameter (80 ~ 90 nm), polydispersity index (PDI) below 0.2, encapsulation efficiency above 95%, mass ratio of mRNA to tRNA (1:1) (Fig. <a href="#Fig6" class="usa-link">6a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">8a–c</a>). LNPs also showed batch-to-batch reproducibility in terms of their encapsulated RNAs and absorption efficiency in HEK293T cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">8d–g</a>). For vaccines, sufficient antigen expression is a crucial determinant for eliciting effective immune responses. Therefore, we evaluated the protein expression of LNP mRNA-tRNA preparations in HEK293T cells, using T1 (Lst-UUA) as the benchmark. These LNPs encapsulated with Lt-CAG, Lt-4, Lt-8, Lt-9, Lt-11, Lt-12, or Lt-13 were described as T2 ~ T8, respectively (Supplementary Fig. <a href="#MOESM1" class="usa-link">8f</a>). By contrast, LNPs encapsulated with Lt, including T2, T3, T6, and T7, exhibited higher protein expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">8h</a>). Notably, within the restricted lifetime of Spike mRNA, T7 showed the highest antigen concentration among the five LNP preparations (Fig. <a href="#Fig6" class="usa-link">6b, c</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. mRNA-tRNA vaccines elicited stronger humoral and cellular immunity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373798_41467_2025_62981_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30ba/12373798/0acab0c46fc2/41467_2025_62981_Fig6_HTML.jpg" loading="lazy" id="d33e1358" height="991" width="736" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Left, schematic diagram showing that LNP formulation consisted of tRNA, Spike-mRNA, ionizable lipid, PEG2000-DMG, DSPC and cholesterol. Right, abbreviation of LNPs and tRNAs. T1 (Lst-UUA) was used as the benchmark vaccine. Placebo (blank LNP) was used as a negative control. Figure was generated using BioRender. <strong>b</strong>, <strong>c</strong> Dynamic changes of RNA levels and antigen concentrations of LNP formulations transfected in HEK293T cells (<em>n</em> = 3 biological replicates). <strong>d</strong> Schematic diagram of the mRNA-tRNA vaccination, antibody detection, and splenic lymphocytes detection in BALB/c mice. Figure was generated using BioRender. <strong>e</strong> Measurement of the IgG antibody endpoint GMTs elicited by the mRNA-tRNA vaccines using ELISA (<em>n</em> = 8 mice in each group). <strong>f</strong> Quantification of the NT50 of mRNA-tRNA immunized mouse sera by Spike B.1.1.529 pseudovirus neutralizing assay. The sera were collected 2 weeks after the booster injection (T1 ~ T7, <em>n</em> = 8 mice in each group; placebo, <em>n</em> = 3 mice). <strong>g</strong> ELISpot assay measurement of the SARS-CoV-2 Spike-specific IFN-γ, TNF-α, IL-2, and IL-4 responses of splenocytes from immunized mice. Mouse spleens were isolated and dissociated 5 weeks post boost-injection (T1 ~ T7, <em>n </em>= 8 mice in each group; placebo, <em>n</em> = 3 mice). <strong>h</strong> FACS analysis results showing the percentages and cell counts of CD4<sup>+</sup> Tem cells producing IL-2, and IL-4 after stimulation with Spike-peptide pool (<em>n</em> = 8 mice in each group). <strong>i</strong> FACS analysis results showing the percentages and cell counts of CD8<sup>+</sup> Tem cells producing IFN-γ and TNF-α after stimulation (<em>n </em>= 8 mice in each group). Statistical significance was analysed using one-way ANOVA with Dunnett’s correlation (<strong>e</strong>–<strong>i</strong>) or two-way ANOVA with Dunnett’s correlation (<strong>c</strong>). *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, ns, not significant. Data are presented as mean ± s.d. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec12"><h3 class="pmc_sec_title">In vivo immunogenicity of mRNA-tRNA vaccines</h3>
<p id="Par26">The ability of mRNA-tRNA vaccines to induce increased immunogenicity in vivo was then tested. Mice were immunized intramuscularly with two doses of vaccines at an interval of two weeks, and the levels of anti-spike IgG, neutralizing antibodies and spike-specific interferon-γ (IFN-γ), TNF-α, interleukin-2 (IL-2), and IL-4 secreting T cells were assessed (Fig. <a href="#Fig6" class="usa-link">6d</a>). Consistent with the decoding ability of the formulated tRNAs, all mRNA-tRNA vaccines were able to elicit more robust antibody responses than the benchmark vaccine, with T7 being the most effective. Specifically, T7 showed a 1.51-fold increase in IgG titer (9.8 × 10<sup>5</sup> versus 6.5 × 10<sup>5</sup>) and a 2.52-fold increase in NT50 (Fig. <a href="#Fig6" class="usa-link">6e, f</a>).</p>
<p id="Par27">To compare the cellular immune responses among these groups, splenocytes of immunized mice were harvested 5 weeks after booster injection and then stimulated with a SARS-CoV-2 Spike peptide pool, using DMSO as the negative control. Cytokine secretion by T cells was quantified by ELISpot assays or by intracellular cytokine staining of effector memory T cells (Tem, CD44<sup>+</sup>CD62L<sup>-</sup>). ELISpot results showed robust secretion of IFN-γ, TNF, IL-2, and IL-4 in all groups, with higher secretion of IL-2 than IL-4, demonstrating the induction of a Th1-biased cellular immune response. Notably, T7 resulted in a significant increase in IL-2 and IL-4 secretion (Fig. <a href="#Fig6" class="usa-link">6g</a>). Consistent with ELISpot results, quantification of the flow cytometry results indicated that the proportions and numbers of CD4<sup>+</sup> and CD8<sup>+</sup> TEM cells, as well as CD4<sup>+</sup> TEM cells producing IL-2 and IL-4, were increased in T7. This suggested a stronger spike-specific CD4<sup>+</sup> and CD8<sup>+</sup> T cell response elicited by T7 compared to the benchmark T1. Unexpectedly, the percentages of CD8<sup>+</sup> TEM cells secreting IFN-γ and TNF-α were slightly increased in T1 and T2, probably owing to the formulated unmodified tRNAs with relatively high immunogenicity acting as adjuvants to enhance the immune response (Fig. <a href="#Fig6" class="usa-link">6h, i</a>, and Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>-<a href="#MOESM1" class="usa-link">10</a>). Collectively, these results demonstrated that mRNA-tRNA vaccines induced augmented humoral and cellular immune responses.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par28">Here, we harnessed the potential of tRNA as a translation enhancer to develop the tRNA-plus strategy, which demonstrated substantial improvements in protein expression through synergistically enhancing mRNA stability and translational efficiency. We also systematically engineered and synthesized tRNA bearing multiple chemical modifications that conferred high decoding ability to tRNA. When combined with Spike mRNA vaccines, tRNA-plus contributed to augmented spike-specific humoral and cellular immunity in vivo. In contrast to current strategies that focus on optimizing mRNA vector, such as circular<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, self-amplifying<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>, multi-capped and multi-tailed RNAs<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, we offer a unique perspective: the translation machinery can be artificially modulated to increase translation capacity without any complicated alterations in mRNA sequence or structure.</p>
<p id="Par29">Our findings show that tRNA-plus is functionally akin to codon optimization (Fig. <a href="#Fig2" class="usa-link">2f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), as both increase the optimality of target mRNA. In comparison, codon optimization adjusts codon composition to adapt host tRNA signatures, yet is constrained by tRNA supply<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>; whereas tRNA-plus improves cognate codon optimality via upregulating tRNA availability. This underscores the potency of combining both strategies for synergistically promoting protein expression. However, excessively fast elongation rates may impair co-translational folding by interfering with chaperones binding to nascent chains, particularly in unstructured regions such as linkers between structured domains, resulting in the accumulation of misfolded, non-functional proteins<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Furthermore, upregulating rare tRNA abundance increases protein misfolding<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>, implying potential negative effects of tRNA overexpression on protein quality. Although non-optimal codons are typically associated with slow translation and accelerated mRNA degradation, their frequent occurrence within secondary structures capable of co-translational folding suggests they could facilitate correct protein folding through slowing ribosomal elongation rates<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. In agreement with previous studies<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, we demonstrate that tRNA-plus enhances the stability, translation efficiency, and ribosomal occupancy of cognate codon-rich mRNA. This can be elucidated as follows: (1) the upregulated tRNAs are more available to the ribosomal A-sites, allowing more efficient decoding at cognate codons; (2) as translation elongation and initiation are tightly coupled, the increased elongation rates alleviate the congestion of ribosomes around the start codon, enabling more ribosomes to enter the elongation phase<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>; (3) the increased ribosomal occupancy synergizes with the improved codon optimality, protecting mRNA from degradation by decay machinery<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. It is noteworthy that altering tRNA levels not only affect the decoding of cognate codons, but may also influence the decoding of both near-cognate and non-cognate codons via wobble pairing<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Moreover, as the moving ribosome is sensed by deadenylation complexes<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, excessive ribosome loads can easily cause ribosomal collisions that accelerate mRNA decay<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Therefore, the future establishment of a comprehensive analysis method, considering additional factors including mRNA secondary structures, translation initiation probability, codon distribution, composition of host tRNA pool, and activity of aminoacyl-tRNA synthetase, will be helpful for maximizing the efficacy of tRNA-plus<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>.</p>
<p id="Par30">Our results regarding chemically modified tRNAs indicate that modifications incorporated in the anticodon loop contribute significantly to the decoding efficiency, particularly at the wobble position; whereas those outside of the anticodon loop regulate other properties such as affinity for eEF1A, stability, and immunogenicity. Combination of functionally relevant modifications or nucleotides with three-dimensional base interactions further improves tRNA decoding capacity<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. This offers a paradigm and methods for optimizing tRNA functionality that can be applied in the development of tRNA-based therapeutics. However, given the &gt; 150 types of modifications, the modified tRNA design space is prohibitively large, especially for cytoplasmic tRNA with an average of 13 modifications<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Thus, the development of tRNA sequencing technologies for comprehensive coverage of modifications within each type of tRNA molecule will be beneficial. One limitation of this research is that tRNA only contributes to 30−50% elevation in linear mRNA translation, probably due to the inherent instability of mRNA and the limited ability of partially functionalized tRNA. This can be compensated for via upregulating tRNA abundance, replacing with more functionalized tRNA, and combining multiple tRNAs. Additionally, the use of mRNA with extended lifetime can expand the translation duration of tRNA<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, enabling further augmentation of mRNA functions. Although our findings demonstrate the augmented immune responses elicited by Spike mRNA-tRNA vaccines, the protection studies against SARS-CoV-2 challenge were not conducted due to the limited availability of live virus. Furthermore, the long-term safety and efficacy of co-delivered tRNA molecules require further comprehensive evaluation.</p>
<p id="Par31">Given tRNA is such a fundamental translation machinery that serves dual functions in modulating mRNA translation and stability, the tRNA-plus strategy can theoretically be applied to various translation-based scenarios, including circRNA- or saRNA-based vaccines, in vivo CAR-T therapy<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, and stem cell fate decision (e.g., by combining transcriptional factor mRNA with appropriate tRNA)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Moreover, we envision that cells with customized tRNA signatures will hold promises for recombinant protein expression, antibody production, and viral packaging.</p></section><section id="Sec14"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec15"><h3 class="pmc_sec_title">Plasmids construction</h3>
<p id="Par32">Sequence information for all constructs is provided in Supplementary information and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>. To evaluate the function of tRNA in increasing spike protein expression, sequences encoding Spike-WT and Spike-OP (SARS-CoV-2 Omicron B.1.1.529) with EGFP downstream were inserted into the pcDNA3.1-CMV backbone using the Uniclone One Step Seamless Cloning Kit (Genesand) following the manufacturer’s instructions. For 7sk-driven tRNA plasmids, the tRNA fragments were PCR-amplified through a pair of complementary primers and assembled into the pcDNA3.1-7sk backbone via the Gibson assembly strategy. The tRNA gene sequences were obtained from the genomic tRNA database (<a href="http://gtrnadb.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gtrnadb.ucsc.edu/</a>). To evaluate the decoding ability of tRNA isodecoders, engineered tRNAs with altered anticodon were then PCR-amplified through a pair of complementary primers and assembled into the pcDNA3.1-7sk-EF1a-mCherry backbone via the Gibson assembly strategy. For T7-driven mRNA constructs, elements including the T7 promoter (TAATACGACTCACTATA<span class="text-underline">AG</span>), 5’UTR (human beta-globin), multiple cloning site (MCS), 3’UTR (AES-mtRNR1), and poly(A)-tail (100 A) were de novo synthesized in Nanjing Genscript Company and cloned into an Epstein-Barr Virus (EBV)-based vector (PCE). Sequences encoding S-2P-WT, S-2P-WT-Nluc (2 P: K983P, V984P), Flag-EGFP-Myc, Flag-EGFP<sup>39UAG</sup>-Myc, 5 × CUG-mCherry-Myc were inserted into the MCS of PCE-mRNA vector via Gibson assembly, respectively. For packaging of Spike pseudovirus, sequence of Spike-OP was inserted into a lentiviral envelope vector (PMD2.G) or a PCE vector. The sequence encoding EGFP-P2A-Nluc, serving as the viral genome, was cloned into the pLenti backbone. For Tet-Off induced mRNA stability measurement, elements including the Tet-responsive transactivator (tTA-Advanced), tight Tetracycline-response elements promoter (PTight), Spike-E2A-EGFP were cloned into a pcDNA3.1 Vector. To establish stable cell lines, sequences encoding tRNA, SecNluc<sup>133UAG</sup>, TLR8, and NF-κB-EGFP were PCR-amplified and assembled into the pLenti backbone carrying resistance genes for either hygromycin or puromycin, respectively. For construction of EGFP with different codon usages, EGFP-OP, EGFP-DOP-1, EGFP-DOP-2, EGFP-DOP-3, and EGFP-AAA were synthesized and cloned into the pcDNA3.1-CMV backbone, respectively.</p></section><section id="Sec16"><h3 class="pmc_sec_title">In vitro transcription of linear mRNA</h3>
<p id="Par33">Sequence information for all mRNA products is provided in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>. DNA plasmids were linearized through double enzymatic digestion using <em>Hind III</em> (Thermo Fisher Scientific) and type IIS restriction enzyme <em>BsmI</em> (Thermo Fisher Scientific). Linearized templates with free-ending poly(A)-tail were then separated by agarose gel electrophoresis and purified with the FastPure Gel DNA Extraction Mini Kit (Vazyme). Capped and tailed mRNA constructs, including S-2P-WT, S-2P-Nluc, Flag-EGFP-Myc, Flag-EGFP<sup>39UAG</sup>-Myc, 5 × CUG-mCherry-Myc, were synthesized using the HiSynth T7 Co-transcription RNA Synthesis Kit (Syngenebio) with complete replacement of uridine with m<sup>1</sup>Ψ. To study the influence of different modifications in the translation capacity of mRNA, Flag-EGFP-Myc mRNAs were transcribed either without or with one of the following modifications: modified U (Ψ, m<sup>1</sup>Ψ, mo5U, m3-mo5U), modified C (m5C), modified A (m6A, Z), modified G (m7G) (Syngenebio). The EGFP (m6A)-poly(A)-mRNA and EGFP (Z)-poly(A)-mRNA were generated from a PCR-amplified template without poly(A)-tail, and were subjected to co-capped transcription using HiSynth T7 Co-transcription RNA Synthesis Kit (Syngenebio) and tailing using <em>E. coli</em> poly (A) polymerase (Novoprotein). These resulting mRNAs were purified using VAHTS RNA Clean Beads (Vazyme). mRNA concentrations were quantified using NanoDrop.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par34">HEK293T (CRL-3216), THP-1, HepG2 cell lines were maintained in our laboratory. The HEK293T-hACE2 cell line was purchased from Biodragon Inc. (Beijing, China). The HEK293T-tRNA<sup>Leu</sup>CAG (HEK293T-tL), HEK293T-tRNA<sup>Ala</sup>GGC (HEK293T-tA), HEK293T-mCherry, HEK293T-SecNluc<sup>133UAG</sup>, HEK293T-TLR8 and HEK293T-TLR8-NF-κB-EGFP cell lines were generated in our laboratory. These HEK293T cell lines and HepG2 cells were cultured in Dulbecco’s Modified Eagle Medium (Gibco) with 10% fetal bovine serum (FBS) (Excell), supplemented with 1% penicillin-streptomycin (D10 medium), in a 5% CO2 incubator at 37 °C. The THP-1 cells were maintained in RPMI-1640 medium (Gibco) with 10% FBS (Gibco) and 55 mM 2-mercaptoethanol (Sigma-Aldrich) (R10 medium) in a 5% CO2 incubator at 37 °C.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Generation of stable cell lines</h3>
<p id="Par35">For lentivirus generation, HEK293T cells were seeded onto 100 mm dishes and grown to approximately 70% confluency the next day. 1.5 µg of pRSV-Rev (Addgene, plasmid no. 12253), 3.5 µg of pMD2.G (Addgene, plasmid no. 12259) and 5 µg of pMDLg/pRRE (Addgene, plasmid no. 12251) packaging vectors were co-transfected with 10 µg of the appropriate pLenti-backboned targeting plasmids using 60 µl of polyethylenimine (PEI, Polysciences). After 8 h, the media were exchanged. Culture supernatants were harvested 48 h post-transfection, centrifuged at 1000 × <em>g</em> for 10 min, and filtered through a 0.45 µm filter (Millipore). Viral particles were precipitated at 4 °C using PEG-it virus precipitation solution (SBI) for another 24 h. The lentiviral particles were then obtained by centrifugation at 4000 × <em>g</em> for 30 min, resuspension with DMEM in 1/50 of the supernatant volume (50 ×  concentrated), and preservation in -80 °C. For cell transduction, 50 ~ 100 µl of the appropriate virus was used to infect HEK293T cells in the presence of polybrene (8 µg ml<sup>-1</sup>, Macgene). Medium was replaced 24 h later. Antibiotic selection was performed with hygromycin (300 µg ml<sup>-1</sup>) or puromycin (10 µg ml<sup>-1</sup>) for 2-7 days alongside untransduced control cells, starting at 48 h after transduction. Stable cell lines were established as follows: HEK293T-tL, HEK293T-tA, HEK293T-mCherry, HEK293T-SecNluc<sup>133UAG</sup>, HEK293T-TLR8, and HEK293T-TLR8-NF-κB-EGFP. For tRNA stable overexpression constructs, tRNA<sup>Leu</sup>CAG-1-1 and tRNA<sup>Ala</sup>GGC-2-1 (tRNA<sup>Ala</sup>AGC-2-1 with anticodon altered into GGC) sequences were used.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Western blotting</h3>
<p id="Par36">Cellular lysates were collected using ice-cold RIPA buffer with protease and phosphatase inhibitors (Solarbio). Cellular debris was removed by centrifugation at 16,000 ×g for 12 min at 4 °C. Samples were boiled with loading buffer, separated by SDS-PAGE, transferred to a polyvinylidene difluoride membrane (Millipore), blocked, and incubated with target-specific antibody: (1) anti-SARS-CoV-2 Spike (SinoBio, 40592-MM117; 1:3000), anti-GAPDH (SinoBio, 10094-MM13; 1:10,000); (2) anti-Flag-tag (Biodragon, B1084; 1:5000), anti-β-Tubulin (Biodragon, B1031; 1:10,000). Subsequently, the primary antibodies were bound to IRDye® 680RD Goat anti-Mouse IgG Secondary Antibody (LICOR, 926-68070; 1:15,000). The membrane was then imaged using the Odyssey® Sa Infrared Imaging System at Peking University State Key Laboratory of Natural and Biomimetic Drugs. The grayscale intensity of the western blot bands was quantified using ImageJ software.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Spike B.1.1.529 pseudovirus production and titration</h3>
<p id="Par37">The generation of lentiviral-based Spike pseudovirus was performed as previously described<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> with some modifications. In brief, PMDLg-PRRE, pRSV-Rev, PCE-CMV-S-OP, and pLenti-EGFP-Nluc plasmids were co-transfected into HEK293T cells using PEI (Polysciences). The media were replaced after 8 h, and supernatants containing pseudovirus were collected at 48-60 h after transfection. The supernatants were filtered, precipitated, concentrated 100-fold, and stored at -80 °C until use. For titration of pseudovirus, HEK293T cells and HEK293T-ACE2 cells were seeded at a density of 4 × 10<sup>4</sup> cells per well in D10 medium in 96-well plates precoated with poly-L-lysine (Millipore). The next day, pseudovirus was serially diluted 2-fold in D10 medium, starting at a 1:10 dilution, and incubated with cells. Untreated HEK293T-ACE2 cells (cell-only) and HEK293T cells treated with pseudovirus (virus-only) were used as the negative controls. After incubation at 37 °C for 48–60 h, the supernatant of each well was aspirated and replaced with 50 µl of fresh medium and 50 µl of Nano-Glo Luciferase reagent (Promega). Luminescence was then measured on a TriStar<sup>2</sup>S LB942 Multimode Microplate Reader, and viral titer was determined by plotting RLUs against virus dilution.</p></section><section id="Sec21"><h3 class="pmc_sec_title">RT-qPCR</h3>
<p id="Par38">First, the medium was removed and 500 µl of TRIzol reagent (Invitrogen) was added to each well of 12-well plate. Total RNA was extracted from the cell lysate using the Direct-zol RNA MiniPrep Kit (Zymo Research) according to the manufacturer’s protocol. The isolated RNA was quantified using Nanodrop. Then, 1 μg of total RNA was reverse transcribed into cDNA using Hifair® II 1st Strand cDNA Synthesis Kit (Yeason). Quantitative PCR was performed using Hieff® SYBR Green Master Mix (Yeason) on a LightCycler­® 480 Instrument II. The qPCR primers for GAPDH, 18S rRNA, Spike, EGFP-OP, DOP-1, DOP-2, DOP-3, mCherry, and tRNA<sup>Ala</sup>GGC-2-1 are provided in Supplementary Table <a href="#MOESM1" class="usa-link">4</a>. This analysis was performed using ∆∆C<sub>T</sub> method. Gene expressions of Spike and tRNA<sup>Ala</sup>GGC-2-1 were normalized to GAPDH, while expressions of EGFP-OP, DOP-1, DOP-2, and DOP-3 were normalized to mCherry.</p></section><section id="Sec22"><h3 class="pmc_sec_title">mRNA decay assay</h3>
<p id="Par39">Cells were seeded at 4 × 10<sup>5</sup> cells per well in 12-well plates in D10 medium. Twenty-four h after seeding, cells were transfected with plasmids using PEI (Polysciences). Thirty-six hours after transfection, the media were spiked with 10 μg ml<sup>-1</sup> of α-amanitin (APExBIO) to selectively inhibit the transcriptional activity of eukaryotic RNA polymerase II. In terms of Tet-Off Spike plasmid, doxycycline (MCE) was added to a final concentration of 0.5 μg ml<sup>-1</sup> to selectively turn off the transcription of Spike mRNA. Total RNA was then extracted using the Direct-zol RNA MiniPrep Kit (Zymo Research) at 0-hr and 8-hr time points after α-amanitin or doxycycline treatment. To measure the stability of Spike mRNA upon transient tRNA overexpression, a mixture of spike plasmid and tRNA plasmid was transfected into HEK293T cells. As for stably overexpressing tRNA, spike plasmid alone was transfected into HEK293T-mCherry (WT, control), HEK293T-tL, and HEK293T-tA cells. For EGFP mRNA stability assessments, EGFP variants were transfected into HEK293T-mCherry cells with tRNA plasmid. QPCR was employed to detect the relative quantities of target transcripts at different time points using 18S rRNA, GAPDH, or mCherry as an endogenous control.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Transfer RNA sequencing</h3>
<p id="Par40">For tRNA profiling, 3 × 10<sup>6</sup> HEK293T-mCherry (control), HEK293T-tL and HEK293T-tA cells were subjected to total RNA extraction using TRIzol reagent (Invitrogen). The subsequent tRNA sequencing was conducted by CloudSeq Inc. (Shanghai, China). Briefly, total RNA was subjected to small-RNA ( &lt; 200nt) extraction using the MirVana Isolation Kit (Thermo Fisher Scientific). The enriched small RNAs were then deacylated in 100 mM Tris-HCl (pH 9.0) and 1 mM EDTA at 37 °C for 30 min. tRNA libraries were constructed with the GenSeq® Small RNA Library Prep Kit (GenSeq, Inc) according to the manufacturer’s instructions. All libraries were size-selected (170-210 bp) and sequenced in a HiSeq platform (Illumina).</p>
<p id="Par41">For analysis, standard human reference libraries were adapted from the tRNAScan-SE library by appending CCA tails to the tRNAs from the genomic tRNA database (GtRNAdb, <a href="http://gtrnadb.ucsc.edu" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gtrnadb.ucsc.edu</a>; Hsapi19). Isodecoders with identical scores were consolidated for ease of identity assignment. Raw reads were generated after sequencing, image analysis, base calling and quality filtering on Illumina sequencer. Q30 was used to perform quality control. The adaptor sequences were trimmed and the adaptor-trimmed-reads (≥15nt) were left by the cutadapt software (v1.9.3). Then, trimmed reads from all samples were aligned using bowtie2 software to the aforementioned tRNA library with sensitive options. Mapped reads on each tRNA were counted using samtools. DESeq2 was used to detect statistical differences in tRNA expression between samples or conditions.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Ribosome profiling and data analysis</h3>
<p id="Par42">For ribosome profiling, ten-million HEK293T-mCherry (control), HEK293T-tL and HEK293T-tA cells were transfected with 2 µg of spike plasmid using PEI (Polysciences), respectively. Thirty-six h after transfection, cells were quickly washed with ice-cold PBS supplemented with 100 µg ml<sup>-1</sup> cycloheximide (Sigma-Aldrich), followed by being scraped off the plate in 2 ml cycloheximide-containing PBS, centrifuged, and flash-frozen. Subsequent ribosome profiling was conducted by CloudSeq Inc. (Shanghai, China), following the protocol for GenSeq® Ribo Profile Kit (GenSeq, Inc). Briefly, cycloheximide-treated cells were lysed with lysis buffer and digested with nuclease. Ribosome protected fragments (RPFs) were obtained through size exclusion chromatography, size selection by polyacrylamide gel-electrophoresis (PAGE), and rRNA removal. The purified RNA fragments were then end-repaired, ligated with 3’ adapters and reverse-transcribed into cDNA, which was subjected to PAGE-purification, circularization, and PCR-amplification. The resulting libraries were purified and sequenced in a NovaSeq platform (Illumina).</p>
<p id="Par43">For analysis, single-end reads were harvested from the sequencer and were quality controlled by Q30. Adapter sequences were removed from raw sequencing data using cutadapt software (v1.9.3). Tophat2 software was consequently used to algin the resulting high-quality clean reads to the human (GRCh38) reference genome. HTSeq software (v0.9.1) was then used to get the raw counts. Aligned reads were employed to predict the offset by riboWaltz. The ribosome density within each gene was calculated through custom scripts as previously described<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The Spike transcript (B.1.1.529) coverage is represented as RPM.</p></section><section id="Sec25"><h3 class="pmc_sec_title">RNA-seq and data analysis</h3>
<p id="Par44">For RNA sequencing, 3 × 10<sup>6</sup> HEK293T-mCherry (control), HEK293T-tL and HEK293T-tA cells were subjected to total RNA extraction using TRIzol reagent (Invitrogen). Then, mRNA sequencing was performed by CloudSeq Inc. (Shanghai, China). Briefly, mRNA was purified using GenSeq® mRNA Purification Kit (GenSeq, Inc.) and library was constructed using GenSeq® Directional RNA Library Prep Kit (GenSeq, Inc.) according to the manufacture’s protocol. The mRNA was fragmented into ~300 nt in length, and subjected to first-strand cDNA synthesis using reverse transcriptase and random hexamer primers, and second-strand cDNA synthesis in 2<sup>nd</sup> Strand Synthesis Buffer with dUTP Mix. The resulting double stranded cDNA fragments undergone end-repair, dA-tailing, and adapters ligation were then PCR amplified and purified to obtain the final libraries. Ultimately, libraries were sequenced on an Illumina HiSeq sequencer (Illumina) in paired-end 150 bp mode.</p>
<p id="Par45">Paired-end reads generated from sequencer were firstly quality controlled by Q30. After 3’ adaptor-trimming and substandard reads removing via cutadapt (v1.9.3), the high-quality clean reads were mapped against the reference genome using hisat2 (v2.0.4). Subsequently, the raw counts were obtained using HTSeq (v0.9.1) and normalized using edgeR. Differentially expressed mRNAs were identified based on adjusted <em>p</em> value and fold-change thresholds.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Proteomic 4D label free and data analysis</h3>
<p id="Par46">For proteomic analysis, HEK293T-mCherry (control), HEK293T-tL and HEK293T-tA cells (100 mm dishes, approximately 1 × 10<sup>7</sup> cells) were firstly washed with ice-cold PBS twice, and scraped off the dishes in 2 ml ice-cold PBS, followed by centrifugation and rapid-freezing in liquid nitrogen. Subsequent proteomic analysis was performed by CloudSeq Inc. (Shanghai, China) using 4D label-free quantification. Briefly, proteins were alkylated with iodoacetamide and digested into peptides using trypsin overnight. Then, peptides were separated by liquid chromatography (Easy nLC 1000 system, Thermo Fisher Scientific) and analyzed by LC-MS/MS (timsTOF Pro 2 Mass spectrometer with nano electrospray ionization source, Bruker).</p>
<p id="Par47">Raw mass spectrometry data were processed using MaxQuant software (v1.5.2.8), and peptides and protein identification was performed using the Maxquant algorithm, with relative quantitative information (LFQ) determined. Database searching used the UniProt database with the following parameters: peptide and protein false discovery rate (FDR) ≤ 0.01; minimum peptide length = 7 amino acids; minimum threshold of protein quantification = 2. LFQ values were transformed by log10 and standardized by quantile (limma package). Differential peptide and protein expression was assessed using volcano plots based on <em>p</em> value and fold-change thresholds.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Synthesis of tRNA oligonucleotides</h3>
<p id="Par48">Sequence information for all tRNA oligonucleotides is provided in Supplementary Table <a href="#MOESM1" class="usa-link">3</a>. The tRNA oligonucleotides were chemically synthesized using an LK-48E DNA/RNA synthesizer (Biolino, Shanghai) with controlled pore glass (CPG) solid supports. The RNA phosphoramidite monomers (ChemGenes, USA) were employed as 0.1 M solutions in anhydrous acetonitrile for oligonucleotide coupling. Specifically, during the synthesis cycles of oligos, the coupling reactions were promoted with a 1:1 (v/v) mixture solution of 0.25 M 5-benzylthio-1H-tetrazole (BTT) and 0.3 M 5-(ethylthio)-1H-tetrazole (ETT). The coupling steps were performed twice for 5 min each time for the canonical A, U, C, and G amidities, whereas 10 min each time for the modified monomers. Following the last coupling, the solid supports were pre-aminolysed with 40% diethylamine, washed with 90% acetonitrile, and treated with 28% ammonia aqueous solution at 60 °C for 3 h to remove the base-labile protecting groups and cleave the RNA oligos from CPG. The above dry residues of oligos were then incubated with triethylamine trihydrofluoride at 60 °C for 2 h to remove 2’-O-TBDMS protecting group. Final purification was conducted by high performance liquid chromatography (HPLC) (Waters 2795) and product identity was confirmed using an LCQ Fleet Ion Trap Mass Spectrometer (Thermo Fisher Scientific). All tRNA oligos featured a 5’ phosphorylation and a 3’ CCA tail. The synthesized natural internal modifications of tRNA<sup>Leu</sup>CAG and suppressor tRNA<sup>Leu</sup>CUA include Gm18, m<sup>5</sup>C34, Cm34, m<sup>1</sup>G37, Ψ55, and m<sup>1</sup>A58.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Expression and purification of hcLeuRS</h3>
<p id="Par49">The recombinant plasmid pET-30a-hcLARS was constructed by Nanjing GenScript Company. The expression plasmid was transformed into chemically competent <em>E. coli</em> BL21 (DE3) cells and a single colony was inoculated into TB medium containing the relevant antibiotic at 37 °C until OD600 reaching ~1.2. The protein expression was induced with isopropyl β-D-thiogalactoside at 15°C for 16 h. After induction, cells were harvested by centrifugation and cellular pellets were resuspended with lysis buffer followed by sonication. The suspension was centrifugated at 10,000 × g for 60 min at 4 °C, and the resulting supernatant was purified by affinity chromatography using a Ni Column<sup>+</sup> Superdex 200 column. Target protein was sterilized by 0.22 μm filter before stored in aliquots. The concentration was determined by Bradford protein assay with BSA as the standard. The protein purity and molecular weight were determined by standard SDS-PAGE along with Western blot confirmation.</p></section><section id="Sec29"><h3 class="pmc_sec_title">In vitro aminoacylation assay of tRNA</h3>
<p id="Par50">In vitro aminoacylation assay was conducted as previously described<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. For the Leu system, 10 nM hcLARS synthetase was incubated with 10 μg of tRNA oligonucleotides in reaction buffer (50 mM Tris-HCl, 10 mM MgCl<sub>2</sub>, 20 mM KCl, 2 mM DTT, pH 7.4) supplemented with 0.1 mM ATP, 1 mM L-Leucine, and 0.1 mg ml<sup>-1</sup> BSA at 37 °C for 20 min. The aminoacylation efficiency was determined by the production of AMP using AMP-Glo<sup>TM</sup> Assay (Promega) according to the manufacturer’s instructions. Non-cognate tRNA<sup>Phe</sup>GAA was used as the negative control.</p></section><section id="Sec30"><h3 class="pmc_sec_title">tRNA stability</h3>
<p id="Par51">HEK293T-SecNluc<sup>133UAG</sup> cells were seeded at 4 × 10<sup>4</sup> cells per well in 96-well plates and grown to 80% confluence the next day. Cells were transfected with 0.2 μg of tRNA<sup>Leu</sup>CUA or tRNA<sup>Leu</sup>UUA (negative control) in triplicates using Lipofectamine 3000 (Thermo Fisher Scientific). The medium was refreshed after 6 h, and supernatants containing SecNluc protein were collected at an interval of 2 h. Luminescence was then measured using Nano-Glo Luciferase reagent (Promega) on a TriStar<sup>2</sup>S LB942 Multimode Microplate Reader. The abundance of tRNA was estimated as the relative luminescence normalized to the 2-hr time point. The half-life of tRNA was determined by one-phase decay model in Prism (GraphPad).</p></section><section id="Sec31"><h3 class="pmc_sec_title">tRNA immunostimulatory activity</h3>
<p id="Par52">For evaluation of the tRNA-induced stimulation of TLR7 and TLR8, human TLR stably transformed HEK293 cells are well established as previously described<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. HEK293T-TLR8 cells and HEK293T-TLR8-NFκB-EGFP cells were seeded at 4 × 10<sup>4</sup> cells per well in 96-well plates and cultured overnight until reaching 90% confluence. Then, cells were transfected with 0.2 μg and 0.4 μg of IVS-tRNA using Lipofectamine 3000 (Thermo Fisher Scientific) or 0.2 μg and 0.4 μg of Resiquimod (R848, HARVEYBIO) as a control agonist of the TLR7-TLR8 signaling pathway. After 36 h, IL-8 in the culture supernatants of HEK293T-TLR8 cells was measured using human IL-8/CXCL8 ELISA Kit (SinoBio), and the percentage of EGFP positive HEK293T-TLR8-NFκB-EGFP cells was quantified using flow cytometry.</p>
<p id="Par53">THP-1 cells (4 ×  10<sup>5</sup> cells per well) were plated in 48-well plates and differentiated into macrophage-like cells with 50 nM phorbol 12-myristate 13-acetate (PMA, Sigma-Aldrich). After 12 h of PMA treatment, cells were rinsed with PBS and maintained in fresh medium for an additional 48 h. After resting, cells were transfected with 1 μg of IVS-tRNA using the TransIT®-mRNA Transfection Kit (Mirus Bio) or 1 μg of R848 as the positive control. Twenty-four h post-transfection, TNF-α in supernatants was determined using human TNF-alpha ELISA Kit (SinoBio) according to the manufacturer’s instructions.</p></section><section id="Sec32"><h3 class="pmc_sec_title">LNP mRNA-tRNA formulation and characterization</h3>
<p id="Par54">LNP mRNA-tRNA formulations were produced by D-Nano Therapeutics (Beijing, China), as previously described<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> with some modifications. In brief, lipids were dissolved in ethanol at the following molar percentage ratios: 50% SM102 (Sinopeg), 10% DSPC (Lipoid), 38.5% cholesterol (Nippon Fine Chemical), and 1.5% PEG-DMG (JenKem). The lipid mixture was combined with an acidification buffer of 25 mM sodium acetate (pH 5.0) containing mRNA and tRNA at a volume ratio of 4:1 (aqueous:ethanol) using a microfluidic mixer (Aitesen). The ratio of nitrogen presents on the ionizable N:P ratio was set to 3.0. Formulations were dialyzed against 20 mM Tris (pH 7.4) in dialysis cassettes (MWCO 8-14 kDa, Gentihold) overnight. Diluted LNPs were concentrated using the ultra-centrifugal filters (MWCO 100 kDa, Millipore), and then passed through a 0.22 μm filter and stored at 4 °C. Particle size and polydispersity index (PDI) were characterized using a Zetasizer Pro (Malvern Instruments). Concentrations and encapsulation efficiency of mRNA-tRNA were determined using the Equalbit RNA HS Assay Kit (Vazyme). All LNPs were found to be 80 ~ 90 nm in size, with encapsulation efficiency greater than 95%.</p>
<p id="Par55">To determine the ratio of formulated RNAs in each LNP mRNA-tRNA preparation, 3 μg of LNP was first disrupted using TRIzol reagent (Invitrogen), and total RNA was extracted using the Direct-zol RNA MicroPrep kit (ZYMO Research). A certain volume of the extracted RNA was then reverse transcribed into cDNA, followed by a 100-fold dilution for further qPCR detection. LNP mRNA alone was used as a control. To assess the absorption efficiency of LNP formulations, HEK293T cells, pre-seeded in 12- or 24-well plates to reach 90% confluence, were transfected with LNPs. Twenty-four h post-transfection, total RNA was extracted using the Direct-zol RNA MiniPrep kit (ZYMO Research), and the abundance of Spike mRNA was quantified by RT-qPCR. After 36 h, cellular lysates and culture supernatants were harvested for quantifying spike protein expression by western-blotting or by ELISA using SARS-CoV-2 Omicron (B.1.1.529) variant Spike ELISA Kit (SinoBio) according to the manufacturer’s protocol.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Mice</h3>
<p id="Par56">Animal experiments were proven by the Institutional Animal Care and Use Committee (IACUC) of Peking University Health Science Center under animal protocol no. DLASBD0667. The animal experiments were conducted in compliance with Institutional Animal Care and Use Committee policies and the National Institutes of Health (NIH) guidelines. The mice were housed three to five per cage and kept on under pathogen-free (SPF) conditions (12 h light–dark cycle with ad libitum food and water at the temperature of 18–23 °C with 40–60% humidity) in the Department of Laboratory Animal Science of Peking University Health Science Center.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Mouse vaccination</h3>
<p id="Par57">For mouse vaccination, the BALB/c (female, 6-8 weeks old) mice were intramuscularly immunized (i.m.) with 10 μg of LNP mRNA-tRNA vaccine or a placebo (LNP only) in a 100 µl volume, followed by a second administration to boost the immune responses after 2 weeks. The sera of immunized mice were collected to detect the SARS-CoV-2 spike-specific IgG and neutralizing antibodies at 2- and 5-week post boost. Five weeks after the boost, the splenocytes of vaccinated mice were harvested to determine the cytokine production by T cells.</p></section><section id="Sec35"><h3 class="pmc_sec_title">Measurement of anti-spike IgG antibody by ELISA</h3>
<p id="Par58">All immunized mouse serum samples were heat-inactivated at 56 °C for 30 min before use. The anti-Spike IgG antibodies were measured using the mouse SARS-CoV-2 Spike (Omicron) IgG Antibody ELISA Kit (Vazyme) according to the manufacturer’s instructions. Briefly, serial 3-fold dilutions (in sample dilution buffer) of mouse sera, starting at a ratio of 1:2400, were added to the pre-coated ELISA plates and incubated at 37 °C for 60 min. After five washes with wash buffer, enzyme-linked antibody working solution was added to the plates and incubated at 37 °C for 30 min. Then, the plates were washed 5 times with wash buffer, and TMB substrates (100 µl per well) were added, followed by incubation for 10 min. Then, the ELISA stop buffer was added to stop the reaction. Finally, the absorbance (450/630 nm) was measured with an AMR-100 microplate reader. The IgG endpoint GMTs were defined as the dilution fold, at which the optical density exceeded 2.1× background (negative control).</p></section><section id="Sec36"><h3 class="pmc_sec_title">Pseudovirus-based neutralization assay</h3>
<p id="Par59">To determine the NT50 of immunized mouse serum, HEK293T-hACE2 cells were seeded in the 96-well plates pre-coated with PDL at a density of 4 × 10<sup>4</sup> cells per well. The cells were then incubated for approximately 16 h until they reached over 80% confluence, suitable for pseudovirus infection. The mouse serum was diluted in a 3-fold series, starting from a 1:40 dilution, and incubated with SARS-CoV-2 spike pseudovirus at 37 °C for 60 min. D10 medium without serum served as the positive control, while D10 medium without pseudovirus served as the negative control. Subsequently, the supernatant of HEK293T-hACE2 cells was removed, and the mixture of serum and pseudovirus was added to each well. After incubation for 24 h, the culture supernatant was replaced with fresh D10 medium. Forty-eight h later, luminescence was measured using the Nano-Glo Luciferase Assay (Promega), reflecting the degree of pseudovirus infection. The NT50, defined as the fold dilution that inhibits more than 50% of pseudovirus infection compared to the positive control, was determined by four-parameter nonlinear regression in Prism (GraphPad).</p></section><section id="Sec37"><h3 class="pmc_sec_title">ELISpot assay</h3>
<p id="Par60">ELISpot assay of splenocytes was performed using the mouse IFN-γ, TNF-α, IL-2, and IL-4 ELISpot PLUS Kit (MabTech) according to the manufacturer’s instructions. Briefly, 2.5 × 10<sup>5</sup> splenocytes were stimulated with a SARS-CoV-2 S peptide pool (15-mer sequences with 11-amino acid overlap covering the full-length Spike, total 315 peptides; SinoBio) at the final concentration of 2 μg ml<sup>-1</sup>. DMSO served as the negative control, while the cocktail of PMA (Sigma-Aldrich) and Ionomycin (Yeason) was used as the positive control. After 24 h of stimulation at 37 °C, the cells were washed away, and the secreted cytokines were bound with the following target-specific biotinylated antibodies: anti-mouse IFN-γ (R4-6A2-biotin), anti-mouse TNF-α (MT11B10-biotin), anti-mouse IL-2 (5H4-biotin), anti-mouse IL-4 (BVD6-24G2-biotin). After 2 h of incubation at room temperature, the plates were washed with 1 × PBS, and incubated with streptavidin alkaline phosphatase for 60 min at room temperature. The plates were developed using BCIP/NBT substrate until distinct spots emerged. The spots were counted using an AID vSpot Spectrum EliSpot/FluoroSpot Reader. The results were normalized to SFC per million cells.</p></section><section id="Sec38"><h3 class="pmc_sec_title">Flow cytometry analysis of T cells</h3>
<p id="Par61">For multiparameter intracellular cytokine staining assays, splenocytes were seeded at 3.5 × 10<sup>6</sup> cells per well in 200 µl of R10 medium in the 96-well plate. Each group was treated with mock (200 µl of R10; negative control), pooled Spike peptides (4 μg ml<sup>-1</sup>), and 50 ng ml<sup>-1</sup> PMA (Sigma-Aldrich) plus 2 μM ionomycin (Yeason) (positive control) and incubated at 37 °C for 5 h. The GolgiPlug transport inhibitor (BD) was added simultaneously. After stimulation, the cells were washed with PBS prior to staining with the LIVE/DEAD Viability kit (Thermo Fisher Scientific) for 30 min at 4°C. Cells were then washed and suspended in Fc block (Biolegend, clone 93) for 10 min, followed by surface staining with the following antibodies on ice: anti-CD45 (Thermo Fisher Scientific, clone 30-F11), anti-CD3ε (Thermo Fisher Scientific, clone 145-2C11), anti-CD4 (Thermo Fisher Scientific, clone RM4-5), anti-CD8α (Thermo Fisher Scientific, clone 53-6.7), anti-CD44 (Thermo Fisher Scientific, clone IM7), and anti-CD62L (Thermo Fisher Scientific, clone Mel-14). After 20 min of incubation, the cells were washed with fluorescence-activated cell sorting (FACS) buffer (PBS with 0.5% BSA) and then fixed and permeabilized using a BD CytoPerm fixation and permeabilization solution kit (BD). Cells were washed in perm and wash solution, followed by intracellular staining (40 min, 4 °C) using a cocktail of the following antibodies: anti-IFN-γ (Thermo Fisher Scientific, clone XMG1.2), anti-TNF-α (Biolegend, clone MP6-XT22), anti-IL-2 (Thermo Fisher Scientific, clone JES6-5H4), and anti-IL-4 (Thermo Fisher Scientific, clone 11B11). Finally, the cells were washed in perm/wash solution and suspended in FACS buffer, acquired on a CytoFLEX LX cytometer (Beckman Colter Life Sciences). Data were analyzed using FlowJo v10 software.</p></section><section id="Sec39"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par62">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec40"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information File</a><sup> (44.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (109.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (813.8KB, pdf) </sup>
</div></div></section></section><section id="Sec41"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (25MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was financially supported by the National Key R&amp;D Program of China, Frontier Biotechnology (Grant No. 2023YFC3403300), and the National Natural Science Foundation of China (Grant No. U23A20106). Illustrations were created with BioRender.com.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.D. and Q.X. conceived the idea. L.D. established the methodology and designed the experiments. L.D. and J.W. performed the experiments and offered the original images. L.D. and J.W. organized the results, analysed the data, prepared the figures, wrote and edited the manuscript. Q.X. supervised the study.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par63"><em>Nature Communications</em> thanks Marc Torrent and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data from the Ribosome profiling, tRNA sequencing and RNA-seq have been deposited in the Gene Expression Omnibus (GEO) under the accession numbers GSE301911, GSE301914, GSE301916. Data from tRNA profiling of Vero E6 cells infected with SARS-CoV-2 virus have been deposited in Figshare (10.6084/m9.figshare.29510411). The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the iProX partner repository with the dataset identifier PXD065927. Source data is available for Figs. <a href="#Fig1" class="usa-link">1</a>–<a href="#Fig6" class="usa-link">6</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">1</a>-<a href="#MOESM1" class="usa-link">9</a> in the associated source data file. <a href="#Sec41" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par64">Q.X. and L.D. et al are listed as co-inventors on patent regarding use of tRNA in promoting protein coding ability of mRNA (PCT/CN2025/109246 and CN118995711). Q.X. is the founder of QiXia Decdoe and holds equity in this company. The other authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Liangzhen Dong, Jiayu Wang.</p></div>
</div></section><section id="sec42"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62981-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Baden, L. R. et al. Efficacy and Safety of the mRNA-1273 SARS-CoV-2 Vaccine. <em>N. Engl. J. Med</em>. <strong>384</strong>, 403–416 (2021).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2035389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7787219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33378609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baden,%20L.%20R.%20et%20al.%20Efficacy%20and%20Safety%20of%20the%20mRNA-1273%20SARS-CoV-2%20Vaccine.%20N.%20Engl.%20J.%20Med.%20384,%20403%E2%80%93416%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Polack, F. P. et al. Safety and Efficacy of the BNT162b2 mRNA Covid-19 Vaccine. <em>N. Engl. J. Med</em>. <strong>383</strong>, 2603–2615 (2020).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2034577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7745181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33301246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Polack,%20F.%20P.%20et%20al.%20Safety%20and%20Efficacy%20of%20the%20BNT162b2%20mRNA%20Covid-19%20Vaccine.%20N.%20Engl.%20J.%20Med.%20383,%202603%E2%80%932615%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Pardi, N., Hogan, M. J., Porter, F. W. &amp; Weissman, D. mRNA vaccines — a new era in vaccinology. <em>Nat. Rev. Drug Discov.</em><strong>17</strong>, 261–279 (2018).
</cite> [<a href="https://doi.org/10.1038/nrd.2017.243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5906799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29326426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pardi,%20N.,%20Hogan,%20M.%20J.,%20Porter,%20F.%20W.%20&amp;%20Weissman,%20D.%20mRNA%20vaccines%20%E2%80%94%20a%20new%20era%20in%20vaccinology.%20Nat.%20Rev.%20Drug%20Discov.17,%20261%E2%80%93279%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Mauger, D. M. et al. mRNA structure regulates protein expression through changes in functional half-life. <em>Proc. Natl. Acad. Sci. Usa.</em><strong>116</strong>, 24075–24083 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1908052116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6883848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31712433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mauger,%20D.%20M.%20et%20al.%20mRNA%20structure%20regulates%20protein%20expression%20through%20changes%20in%20functional%20half-life.%20Proc.%20Natl.%20Acad.%20Sci.%20Usa.116,%2024075%E2%80%9324083%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Garneau, N. L., Wilusz, J. &amp; Wilusz, C. J. The highways and byways of mRNA decay. <em>Nat. Rev. Mol. Cell Biol.</em><strong>8</strong>, 113–126 (2007).
</cite> [<a href="https://doi.org/10.1038/nrm2104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17245413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garneau,%20N.%20L.,%20Wilusz,%20J.%20&amp;%20Wilusz,%20C.%20J.%20The%20highways%20and%20byways%20of%20mRNA%20decay.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.8,%20113%E2%80%93126%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Züst, R. et al. Ribose 2′-O-methylation provides a molecular signature for the distinction of self and non-self mRNA dependent on the RNA sensor Mda5. <em>Nat. Immunol.</em><strong>12</strong>, 137–143 (2011).
</cite> [<a href="https://doi.org/10.1038/ni.1979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3182538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21217758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Z%C3%BCst,%20R.%20et%20al.%20Ribose%202%E2%80%B2-O-methylation%20provides%20a%20molecular%20signature%20for%20the%20distinction%20of%20self%20and%20non-self%20mRNA%20dependent%20on%20the%20RNA%20sensor%20Mda5.%20Nat.%20Immunol.12,%20137%E2%80%93143%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Despic, V. &amp; Jaffrey, S. R. mRNA ageing shapes the Cap2 methylome in mammalian mRNA. <em>Nature</em><strong>614</strong>, 358–366 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-022-05668-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9891201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36725932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Despic,%20V.%20&amp;%20Jaffrey,%20S.%20R.%20mRNA%20ageing%20shapes%20the%20Cap2%20methylome%20in%20mammalian%20mRNA.%20Nature614,%20358%E2%80%93366%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zhang, H. et al. Algorithm for optimized mRNA design improves stability and immunogenicity. <em>Nature</em><strong>621</strong>, 396–403 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06127-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10499610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37130545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20H.%20et%20al.%20Algorithm%20for%20optimized%20mRNA%20design%20improves%20stability%20and%20immunogenicity.%20Nature621,%20396%E2%80%93403%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Chen, H. et al. Chemical and topological design of multicapped mRNA and capped circular RNA to augment translation. <em>Nat. Biotechnol.</em>10.1038/s41587-024-02393-y (2024).
</cite> [<a href="https://doi.org/10.1038/s41587-024-02393-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11929619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39313647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20H.%20et%20al.%20Chemical%20and%20topological%20design%20of%20multicapped%20mRNA%20and%20capped%20circular%20RNA%20to%20augment%20translation.%20Nat.%20Biotechnol.10.1038/s41587-024-02393-y%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Chen, H. et al. Branched chemically modified poly(A) tails enhance the translation capacity of mRNA. <em>Nat. Biotechnol.</em><strong>43</strong>, 194–203 (2025).
</cite> [<a href="https://doi.org/10.1038/s41587-024-02174-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11416571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38519719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20H.%20et%20al.%20Branched%20chemically%20modified%20poly(A)%20tails%20enhance%20the%20translation%20capacity%20of%20mRNA.%20Nat.%20Biotechnol.43,%20194%E2%80%93203%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Passmore, L. A. &amp; Coller, J. Roles of mRNA poly(A) tails in regulation of eukaryotic gene expression. <em>Nat. Rev. Mol. Cell Biol.</em><strong>23</strong>, 93–106 (2022).
</cite> [<a href="https://doi.org/10.1038/s41580-021-00417-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7614307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34594027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Passmore,%20L.%20A.%20&amp;%20Coller,%20J.%20Roles%20of%20mRNA%20poly(A)%20tails%20in%20regulation%20of%20eukaryotic%20gene%20expression.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.23,%2093%E2%80%93106%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Qu, L. et al. Circular RNA vaccines against SARS-CoV-2 and emerging variants. <em>Cell</em><strong>185</strong>, 1728–1744.e16 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.03.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8971115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35460644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qu,%20L.%20et%20al.%20Circular%20RNA%20vaccines%20against%20SARS-CoV-2%20and%20emerging%20variants.%20Cell185,%201728%E2%80%931744.e16%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Chen, R. et al. Engineering circular RNA for enhanced protein production. <em>Nat. Biotechnol.</em><strong>41</strong>, 262–272 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-022-01393-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9931579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35851375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20R.%20et%20al.%20Engineering%20circular%20RNA%20for%20enhanced%20protein%20production.%20Nat.%20Biotechnol.41,%20262%E2%80%93272%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Dana, A. &amp; Tuller, T. The effect of tRNA levels on decoding times of mRNA codons. <em>Nucleic Acids Res</em><strong>42</strong>, 9171–9181 (2014).
</cite> [<a href="https://doi.org/10.1093/nar/gku646" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4132755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25056313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dana,%20A.%20&amp;%20Tuller,%20T.%20The%20effect%20of%20tRNA%20levels%20on%20decoding%20times%20of%20mRNA%20codons.%20Nucleic%20Acids%20Res42,%209171%E2%80%939181%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Gorochowski, T. E., Ignatova, Z., Bovenberg, R. A. L. &amp; Roubos, J. A. Trade-offs between tRNA abundance and mRNA secondary structure support smoothing of translation elongation rate. <em>Nucleic Acids Res.</em><strong>43</strong>, 3022–3032 (2015).
</cite> [<a href="https://doi.org/10.1093/nar/gkv199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4381083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25765653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorochowski,%20T.%20E.,%20Ignatova,%20Z.,%20Bovenberg,%20R.%20A.%20L.%20&amp;%20Roubos,%20J.%20A.%20Trade-offs%20between%20tRNA%20abundance%20and%20mRNA%20secondary%20structure%20support%20smoothing%20of%20translation%20elongation%20rate.%20Nucleic%20Acids%20Res.43,%203022%E2%80%933032%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Goodarzi, H. et al. Modulated expression of specific tRNAs drives gene expression and cancer progression. <em>Cell</em><strong>165</strong>, 1416–1427 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.05.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4915377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27259150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goodarzi,%20H.%20et%20al.%20Modulated%20expression%20of%20specific%20tRNAs%20drives%20gene%20expression%20and%20cancer%20progression.%20Cell165,%201416%E2%80%931427%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Gingold, H. &amp; Pilpel, Y. Determinants of translation efficiency and accuracy. <em>Mol. Syst. Biol.</em><strong>7</strong>, 481 (2011).
</cite> [<a href="https://doi.org/10.1038/msb.2011.14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3101949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21487400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gingold,%20H.%20&amp;%20Pilpel,%20Y.%20Determinants%20of%20translation%20efficiency%20and%20accuracy.%20Mol.%20Syst.%20Biol.7,%20481%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Dittmar, K. A., Goodenbour, J. M. &amp; Pan, T. Tissue-Specific Differences in Human Transfer RNA Expression. <em>PLoS Genet</em><strong>2</strong>, e221 (2006).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.0020221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1713254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17194224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dittmar,%20K.%20A.,%20Goodenbour,%20J.%20M.%20&amp;%20Pan,%20T.%20Tissue-Specific%20Differences%20in%20Human%20Transfer%20RNA%20Expression.%20PLoS%20Genet2,%20e221%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Gao, L. et al. Selective gene expression maintains human tRNA anticodon pools during differentiation. <em>Nat. Cell Biol.</em><strong>26</strong>, 100–112 (2024).
</cite> [<a href="https://doi.org/10.1038/s41556-023-01317-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10791582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38191669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20L.%20et%20al.%20Selective%20gene%20expression%20maintains%20human%20tRNA%20anticodon%20pools%20during%20differentiation.%20Nat.%20Cell%20Biol.26,%20100%E2%80%93112%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Gingold, H. et al. A Dual Program for Translation Regulation in Cellular Proliferation and Differentiation. <em>Cell</em><strong>158</strong>, 1281–1292 (2014).
</cite> [<a href="https://doi.org/10.1016/j.cell.2014.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25215487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gingold,%20H.%20et%20al.%20A%20Dual%20Program%20for%20Translation%20Regulation%20in%20Cellular%20Proliferation%20and%20Differentiation.%20Cell158,%201281%E2%80%931292%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Torrent, M., Chalancon, G., de Groot, N. S., Wuster, A. &amp; Madan Babu, M. Cells alter their tRNA abundance to selectively regulate protein synthesis during stress conditions. <em>Sci. Signal.</em><strong>11</strong>, eaat6409 (2018).
</cite> [<a href="https://doi.org/10.1126/scisignal.aat6409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6130803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30181241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torrent,%20M.,%20Chalancon,%20G.,%20de%20Groot,%20N.%20S.,%20Wuster,%20A.%20&amp;%20Madan%20Babu,%20M.%20Cells%20alter%20their%20tRNA%20abundance%20to%20selectively%20regulate%20protein%20synthesis%20during%20stress%20conditions.%20Sci.%20Signal.11,%20eaat6409%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Presnyak, V. et al. Codon Optimality Is a Major Determinant of mRNA Stability. <em>Cell</em><strong>160</strong>, 1111–1124 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.02.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4359748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25768907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Presnyak,%20V.%20et%20al.%20Codon%20Optimality%20Is%20a%20Major%20Determinant%20of%20mRNA%20Stability.%20Cell160,%201111%E2%80%931124%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Pechmann, S. &amp; Frydman, J. Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding. <em>Nat. Struct. Mol. Biol.</em><strong>20</strong>, 237–243 (2013).
</cite> [<a href="https://doi.org/10.1038/nsmb.2466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3565066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23262490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pechmann,%20S.%20&amp;%20Frydman,%20J.%20Evolutionary%20conservation%20of%20codon%20optimality%20reveals%20hidden%20signatures%20of%20cotranslational%20folding.%20Nat.%20Struct.%20Mol.%20Biol.20,%20237%E2%80%93243%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Hanson, G. &amp; Coller, J. Codon optimality, bias and usage in translation and mRNA decay. <em>Nat. Rev. Mol. Cell Biol.</em><strong>19</strong>, 20–30 (2018).
</cite> [<a href="https://doi.org/10.1038/nrm.2017.91" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6594389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29018283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanson,%20G.%20&amp;%20Coller,%20J.%20Codon%20optimality,%20bias%20and%20usage%20in%20translation%20and%20mRNA%20decay.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.19,%2020%E2%80%9330%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Bae, H. &amp; Coller, J. Codon optimality-mediated mRNA degradation: Linking translational elongation to mRNA stability. <em>Mol. Cell</em><strong>82</strong>, 1467–1476 (2022).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2022.03.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10111967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35452615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bae,%20H.%20&amp;%20Coller,%20J.%20Codon%20optimality-mediated%20mRNA%20degradation:%20Linking%20translational%20elongation%20to%20mRNA%20stability.%20Mol.%20Cell82,%201467%E2%80%931476%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Zhu, X., Cruz, V. E., Zhang, H., Erzberger, J. P. &amp; Mendell, J. T. Specific tRNAs promote mRNA decay by recruiting the CCR4-NOT complex to translating ribosomes. <em>Science</em><strong>386</strong>, eadq8587 (2024).
</cite> [<a href="https://doi.org/10.1126/science.adq8587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11583848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39571015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20X.,%20Cruz,%20V.%20E.,%20Zhang,%20H.,%20Erzberger,%20J.%20P.%20&amp;%20Mendell,%20J.%20T.%20Specific%20tRNAs%20promote%20mRNA%20decay%20by%20recruiting%20the%20CCR4-NOT%20complex%20to%20translating%20ribosomes.%20Science386,%20eadq8587%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Martin, S. et al. Oligodendrocyte differentiation alters tRNA modifications and codon optimality-mediated mRNA decay. <em>Nat. Commun.</em><strong>13</strong>, 5003 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-32766-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9411196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36008413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin,%20S.%20et%20al.%20Oligodendrocyte%20differentiation%20alters%20tRNA%20modifications%20and%20codon%20optimality-mediated%20mRNA%20decay.%20Nat.%20Commun.13,%205003%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Giguère, S. et al. Antibody production relies on the tRNA inosine wobble modification to meet biased codon demand. <em>Science</em><strong>383</strong>, 205–211 (2024).
</cite> [<a href="https://doi.org/10.1126/science.adi1763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10954030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38207021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gigu%C3%A8re,%20S.%20et%20al.%20Antibody%20production%20relies%20on%20the%20tRNA%20inosine%20wobble%20modification%20to%20meet%20biased%20codon%20demand.%20Science383,%20205%E2%80%93211%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Liu, Y. et al. tRNA-m1A modification promotes T cell expansion via efficient MYC protein synthesis. <em>Nat. Immunol.</em><strong>23</strong>, 1433–1444 (2022).
</cite> [<a href="https://doi.org/10.1038/s41590-022-01301-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36138184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20tRNA-m1A%20modification%20promotes%20T%20cell%20expansion%20via%20efficient%20MYC%20protein%20synthesis.%20Nat.%20Immunol.23,%201433%E2%80%931444%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Wang, J. et al. AAV-delivered suppressor tRNA overcomes a nonsense mutation in mice. <em>Nature</em><strong>604</strong>, 343–348 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-04533-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9446716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35322228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.%20et%20al.%20AAV-delivered%20suppressor%20tRNA%20overcomes%20a%20nonsense%20mutation%20in%20mice.%20Nature604,%20343%E2%80%93348%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Albers, S. et al. Engineered tRNAs suppress nonsense mutations in cells and in vivo. <em>Nature</em><strong>618</strong>, 842–848 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06133-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10284701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37258671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Albers,%20S.%20et%20al.%20Engineered%20tRNAs%20suppress%20nonsense%20mutations%20in%20cells%20and%20in%20vivo.%20Nature618,%20842%E2%80%93848%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Coller, J. &amp; Ignatova, Z. tRNA therapeutics for genetic diseases. <em>Nat. Rev. Drug Discov.</em><strong>23</strong>, 108–125 (2024).
</cite> [<a href="https://doi.org/10.1038/s41573-023-00829-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38049504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coller,%20J.%20&amp;%20Ignatova,%20Z.%20tRNA%20therapeutics%20for%20genetic%20diseases.%20Nat.%20Rev.%20Drug%20Discov.23,%20108%E2%80%93125%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Suzuki, T. The expanding world of tRNA modifications and their disease relevance. <em>Nat. Rev. Mol. Cell Biol.</em><strong>22</strong>, 375–392 (2021).
</cite> [<a href="https://doi.org/10.1038/s41580-021-00342-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33658722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suzuki,%20T.%20The%20expanding%20world%20of%20tRNA%20modifications%20and%20their%20disease%20relevance.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.22,%20375%E2%80%93392%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Delaunay, S., Helm, M. &amp; Frye, M. RNA modifications in physiology and disease: towards clinical applications. <em>Nat. Rev. Genet</em><strong>25</strong>, 104–122 (2024).
</cite> [<a href="https://doi.org/10.1038/s41576-023-00645-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37714958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delaunay,%20S.,%20Helm,%20M.%20&amp;%20Frye,%20M.%20RNA%20modifications%20in%20physiology%20and%20disease:%20towards%20clinical%20applications.%20Nat.%20Rev.%20Genet25,%20104%E2%80%93122%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Liu, F. et al. ALKBH1-Mediated tRNA Demethylation Regulates Translation. <em>Cell</em><strong>167</strong>, 816–828.e16 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.09.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5119773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27745969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20F.%20et%20al.%20ALKBH1-Mediated%20tRNA%20Demethylation%20Regulates%20Translation.%20Cell167,%20816%E2%80%93828.e16%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gustafsson, C., Govindarajan, S. &amp; Minshull, J. Codon bias and heterologous protein expression. <em>Trends Biotechnol.</em><strong>22</strong>, 346–353 (2004).
</cite> [<a href="https://doi.org/10.1016/j.tibtech.2004.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15245907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gustafsson,%20C.,%20Govindarajan,%20S.%20&amp;%20Minshull,%20J.%20Codon%20bias%20and%20heterologous%20protein%20expression.%20Trends%20Biotechnol.22,%20346%E2%80%93353%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Wu, Q. et al. Translation affects mRNA stability in a codon-dependent manner in human cells. <em>eLife</em><strong>8</strong>, e45396 (2019).
</cite> [<a href="https://doi.org/10.7554/eLife.45396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6529216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31012849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Q.%20et%20al.%20Translation%20affects%20mRNA%20stability%20in%20a%20codon-dependent%20manner%20in%20human%20cells.%20eLife8,%20e45396%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Geslain, R. &amp; Pan, T. Functional analysis of human tRNA isodecoders. <em>J. Mol. Biol.</em><strong>396</strong>, 821–831 (2010).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2009.12.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2822071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20026070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geslain,%20R.%20&amp;%20Pan,%20T.%20Functional%20analysis%20of%20human%20tRNA%20isodecoders.%20J.%20Mol.%20Biol.396,%20821%E2%80%93831%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Lueck, J. D. et al. Engineered transfer RNAs for suppression of premature termination codons. <em>Nat. Commun.</em><strong>10</strong>, 822 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-08329-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6379413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30778053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lueck,%20J.%20D.%20et%20al.%20Engineered%20transfer%20RNAs%20for%20suppression%20of%20premature%20termination%20codons.%20Nat.%20Commun.10,%20822%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Chan, P. P., Lin, B. Y., Mak, A. J. &amp; Lowe, T. M. tRNAscan-SE 2.0: improved detection and functional classification of transfer RNA genes. <em>Nucleic Acids Res</em>. <strong>49</strong>, 9077–9096 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8450103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34417604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chan,%20P.%20P.,%20Lin,%20B.%20Y.,%20Mak,%20A.%20J.%20&amp;%20Lowe,%20T.%20M.%20tRNAscan-SE%202.0:%20improved%20detection%20and%20functional%20classification%20of%20transfer%20RNA%20genes.%20Nucleic%20Acids%20Res.%2049,%209077%E2%80%939096%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Schubert, K. et al. SARS-CoV-2 Nsp1 binds the ribosomal mRNA channel to inhibit translation. <em>Nat. Struct. Mol. Biol.</em><strong>27</strong>, 959–966 (2020).
</cite> [<a href="https://doi.org/10.1038/s41594-020-0511-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32908316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schubert,%20K.%20et%20al.%20SARS-CoV-2%20Nsp1%20binds%20the%20ribosomal%20mRNA%20channel%20to%20inhibit%20translation.%20Nat.%20Struct.%20Mol.%20Biol.27,%20959%E2%80%93966%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Aguilar Rangel, M., Stein, K. &amp; Frydman, J. A machine learning approach uncovers principles and determinants of eukaryotic ribosome pausing. <em>Sci. Adv.</em><strong>10</strong>, eado0738 (2024).
</cite> [<a href="https://doi.org/10.1126/sciadv.ado0738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11488575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39423268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aguilar%20Rangel,%20M.,%20Stein,%20K.%20&amp;%20Frydman,%20J.%20A%20machine%20learning%20approach%20uncovers%20principles%20and%20determinants%20of%20eukaryotic%20ribosome%20pausing.%20Sci.%20Adv.10,%20eado0738%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Längle-Rouault, F. et al. Up to 100-fold increase of apparent gene expression in the presence of Epstein-Barr virus orip sequences and EBNA1: implications of the nuclear import of plasmids. <em>J. Virol.</em><strong>72</strong>, 6181 (1998).
</cite> [<a href="https://doi.org/10.1128/jvi.72.7.6181-6185.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC110430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9621086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%A4ngle-Rouault,%20F.%20et%20al.%20Up%20to%20100-fold%20increase%20of%20apparent%20gene%20expression%20in%20the%20presence%20of%20Epstein-Barr%20virus%20orip%20sequences%20and%20EBNA1:%20implications%20of%20the%20nuclear%20import%20of%20plasmids.%20J.%20Virol.72,%206181%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Cannarozzi, G. et al. A role for codon order in translation dynamics. <em>Cell</em><strong>141</strong>, 355–367 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.02.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20403329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cannarozzi,%20G.%20et%20al.%20A%20role%20for%20codon%20order%20in%20translation%20dynamics.%20Cell141,%20355%E2%80%93367%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Bazzini, A. A. et al. Codon identity regulates mRNA stability and translation efficiency during the maternal-to-zygotic transition. <em>EMBO J.</em><strong>35</strong>, 2087 (2016).
</cite> [<a href="https://doi.org/10.15252/embj.201694699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5048347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27436874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bazzini,%20A.%20A.%20et%20al.%20Codon%20identity%20regulates%20mRNA%20stability%20and%20translation%20efficiency%20during%20the%20maternal-to-zygotic%20transition.%20EMBO%20J.35,%202087%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Earnest-Noble, L. B. et al. Two isoleucyl tRNAs that decode synonymous codons divergently regulate breast cancer metastatic growth by controlling translation of proliferation-regulating genes. <em>Nat. Cancer</em><strong>3</strong>, 1484–1497 (2022).
</cite> [<a href="https://doi.org/10.1038/s43018-022-00469-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11323107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36510010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Earnest-Noble,%20L.%20B.%20et%20al.%20Two%20isoleucyl%20tRNAs%20that%20decode%20synonymous%20codons%20divergently%20regulate%20breast%20cancer%20metastatic%20growth%20by%20controlling%20translation%20of%20proliferation-regulating%20genes.%20Nat.%20Cancer3,%201484%E2%80%931497%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Karikó, K. et al. Incorporation of pseudouridine into mRNA yields superior nonimmunogenic vector with increased translational capacity and biological stability. <em>Mol. Ther.</em><strong>16</strong>, 1833–1840 (2008).
</cite> [<a href="https://doi.org/10.1038/mt.2008.200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2775451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18797453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karik%C3%B3,%20K.%20et%20al.%20Incorporation%20of%20pseudouridine%20into%20mRNA%20yields%20superior%20nonimmunogenic%20vector%20with%20increased%20translational%20capacity%20and%20biological%20stability.%20Mol.%20Ther.16,%201833%E2%80%931840%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Andries, O. et al. N1-methylpseudouridine-incorporated mRNA outperforms pseudouridine-incorporated mRNA by providing enhanced protein expression and reduced immunogenicity in mammalian cell lines and mice. <em>J. Control. Release</em><strong>217</strong>, 337-44 (2015).</cite> [<a href="https://doi.org/10.1016/j.jconrel.2015.08.051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26342664/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Pan, T. Modifications and functional genomics of human transfer RNA. <em>Cell Res</em>. <strong>28</strong>, 395–404 (2018).
</cite> [<a href="https://doi.org/10.1038/s41422-018-0013-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5939049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29463900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20T.%20Modifications%20and%20functional%20genomics%20of%20human%20transfer%20RNA.%20Cell%20Res.%2028,%20395%E2%80%93404%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Björk, G. R., Wikström, P. M. &amp; Byström, A. S. Prevention of Translational Frameshifting by the Modified Nucleoside 1-Methylguanosine. <em>Science</em><strong>244</strong>, 986–989 (1989).
</cite> [<a href="https://doi.org/10.1126/science.2471265" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2471265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bj%C3%B6rk,%20G.%20R.,%20Wikstr%C3%B6m,%20P.%20M.%20&amp;%20Bystr%C3%B6m,%20A.%20S.%20Prevention%20of%20Translational%20Frameshifting%20by%20the%20Modified%20Nucleoside%201-Methylguanosine.%20Science244,%20986%E2%80%93989%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Rosselló-Tortella, M. et al. Epigenetic loss of the transfer RNA-modifying enzyme TYW2 induces ribosome frameshifts in colon cancer. <em>Proc. Natl. Acad. Sci. USA</em><strong>117</strong>, 20785 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.2003358117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7456070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32778592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rossell%C3%B3-Tortella,%20M.%20et%20al.%20Epigenetic%20loss%20of%20the%20transfer%20RNA-modifying%20enzyme%20TYW2%20induces%20ribosome%20frameshifts%20in%20colon%20cancer.%20Proc.%20Natl.%20Acad.%20Sci.%20USA117,%2020785%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Ranjan, N. &amp; Rodnina, M. V. tRNA wobble modifications and protein homeostasis. <em>Translation</em><strong>4</strong>, e1143076 (2016).
</cite> [<a href="https://doi.org/10.1080/21690731.2016.1143076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4909423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27335723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ranjan,%20N.%20&amp;%20Rodnina,%20M.%20V.%20tRNA%20wobble%20modifications%20and%20protein%20homeostasis.%20Translation4,%20e1143076%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Hernandez-Alias, X. et al. Single-read tRNA-seq analysis reveals coordination of tRNA modification and aminoacylation and fragmentation. <em>Nucleic Acids Res.</em><strong>51</strong>, e17–e17 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9943672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36537222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hernandez-Alias,%20X.%20et%20al.%20Single-read%20tRNA-seq%20analysis%20reveals%20coordination%20of%20tRNA%20modification%20and%20aminoacylation%20and%20fragmentation.%20Nucleic%20Acids%20Res.51,%20e17%E2%80%93e17%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Nguyen, H. A., Hoffer, E. D. &amp; Dunham, C. M. Importance of a tRNA anticodon loop modification and a conserved, noncanonical anticodon stem pairing in tRNACGGPro for decoding. <em>J. Biol. Chem.</em><strong>294</strong>, 5281 (2019).
</cite> [<a href="https://doi.org/10.1074/jbc.RA119.007410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6462517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30782843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20H.%20A.,%20Hoffer,%20E.%20D.%20&amp;%20Dunham,%20C.%20M.%20Importance%20of%20a%20tRNA%20anticodon%20loop%20modification%20and%20a%20conserved,%20noncanonical%20anticodon%20stem%20pairing%20in%20tRNACGGPro%20for%20decoding.%20J.%20Biol.%20Chem.294,%205281%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Dewe, J. M., Whipple, J. M., Chernyakov, I., Jaramillo, L. N. &amp; Phizicky, E. M. The yeast rapid tRNA decay pathway competes with elongation factor 1A for substrate tRNAs and acts on tRNAs lacking one or more of several modifications. <em>RNA</em><strong>18</strong>, 1886–1896 (2012).
</cite> [<a href="https://doi.org/10.1261/rna.033654.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3446711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22895820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dewe,%20J.%20M.,%20Whipple,%20J.%20M.,%20Chernyakov,%20I.,%20Jaramillo,%20L.%20N.%20&amp;%20Phizicky,%20E.%20M.%20The%20yeast%20rapid%20tRNA%20decay%20pathway%20competes%20with%20elongation%20factor%201A%20for%20substrate%20tRNAs%20and%20acts%20on%20tRNAs%20lacking%20one%20or%20more%20of%20several%20modifications.%20RNA18,%201886%E2%80%931896%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Subramanian, M., Srinivasan, T. &amp; Sudarsanam, D. Examining the Gm18 and m1G Modification positions in tRNA sequences. <em>Genomics Inform.</em><strong>12</strong>, 71 (2014).
</cite> [<a href="https://doi.org/10.5808/GI.2014.12.2.71" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4099351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25031570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Subramanian,%20M.,%20Srinivasan,%20T.%20&amp;%20Sudarsanam,%20D.%20Examining%20the%20Gm18%20and%20m1G%20Modification%20positions%20in%20tRNA%20sequences.%20Genomics%20Inform.12,%2071%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kim, S. H. et al. Three-dimensional structure of yeast phenylalanine transfer RNA: folding of the polynucleotide chain. <em>Science</em><strong>179</strong>, 285–288 (1973).
</cite> [<a href="https://doi.org/10.1126/science.179.4070.285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4566654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20H.%20et%20al.%20Three-dimensional%20structure%20of%20yeast%20phenylalanine%20transfer%20RNA:%20folding%20of%20the%20polynucleotide%20chain.%20Science179,%20285%E2%80%93288%20(1973)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Karikó, K., Buckstein, M., Ni, H. &amp; Weissman, D. Suppression of RNA recognition by toll-like receptors: the impact of nucleoside modification and the evolutionary origin of RNA. <em>Immunity</em><strong>23</strong>, 165–175 (2005).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2005.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16111635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karik%C3%B3,%20K.,%20Buckstein,%20M.,%20Ni,%20H.%20&amp;%20Weissman,%20D.%20Suppression%20of%20RNA%20recognition%20by%20toll-like%20receptors:%20the%20impact%20of%20nucleoside%20modification%20and%20the%20evolutionary%20origin%20of%20RNA.%20Immunity23,%20165%E2%80%93175%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Jöckel, S. et al. The 2′-O-methylation status of a single guanosine controls transfer RNA–mediated Toll-like receptor 7 activation or inhibition. <em>J. Exp. Med</em><strong>209</strong>, 235–241 (2012).
</cite> [<a href="https://doi.org/10.1084/jem.20111075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3280869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22312111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?J%C3%B6ckel,%20S.%20et%20al.%20The%202%E2%80%B2-O-methylation%20status%20of%20a%20single%20guanosine%20controls%20transfer%20RNA%E2%80%93mediated%20Toll-like%20receptor%207%20activation%20or%20inhibition.%20J.%20Exp.%20Med209,%20235%E2%80%93241%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Freund, I. et al. 2’-O-methylation within prokaryotic and eukaryotic tRNA inhibits innate immune activation by endosomal Toll-like receptors but does not affect recognition of whole organisms. <em>RNA</em><strong>25</strong>, 869–880 (2019).
</cite> [<a href="https://doi.org/10.1261/rna.070243.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6573781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31019095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Freund,%20I.%20et%20al.%202%E2%80%99-O-methylation%20within%20prokaryotic%20and%20eukaryotic%20tRNA%20inhibits%20innate%20immune%20activation%20by%20endosomal%20Toll-like%20receptors%20but%20does%20not%20affect%20recognition%20of%20whole%20organisms.%20RNA25,%20869%E2%80%93880%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Heil, F. et al. Species-specific recognition of single-stranded RNA via toll-like receptor 7 and 8. <em>Science</em><strong>303</strong>, 1526–1529 (2004).
</cite> [<a href="https://doi.org/10.1126/science.1093620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14976262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heil,%20F.%20et%20al.%20Species-specific%20recognition%20of%20single-stranded%20RNA%20via%20toll-like%20receptor%207%20and%208.%20Science303,%201526%E2%80%931529%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Tanji, H. et al. Toll-like receptor 8 senses degradation products of single-stranded RNA. <em>Nat. Struct. Mol. Biol.</em><strong>22</strong>, 109–115 (2015).
</cite> [<a href="https://doi.org/10.1038/nsmb.2943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25599397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanji,%20H.%20et%20al.%20Toll-like%20receptor%208%20senses%20degradation%20products%20of%20single-stranded%20RNA.%20Nat.%20Struct.%20Mol.%20Biol.22,%20109%E2%80%93115%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Bloom, K., van den Berg, F. &amp; Arbuthnot, P. Self-amplifying RNA vaccines for infectious diseases. <em>Gene Ther.</em><strong>28</strong>, 117–129 (2021).
</cite> [<a href="https://doi.org/10.1038/s41434-020-00204-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7580817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33093657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bloom,%20K.,%20van%20den%20Berg,%20F.%20&amp;%20Arbuthnot,%20P.%20Self-amplifying%20RNA%20vaccines%20for%20infectious%20diseases.%20Gene%20Ther.28,%20117%E2%80%93129%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Yu, C.-H. et al. Codon usage influences the local rate of translation elongation to regulate co-translational protein folding. <em>Mol. Cell</em><strong>59</strong>, 744–754 (2015).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.07.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4561030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26321254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20C.-H.%20et%20al.%20Codon%20usage%20influences%20the%20local%20rate%20of%20translation%20elongation%20to%20regulate%20co-translational%20protein%20folding.%20Mol.%20Cell59,%20744%E2%80%93754%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Zhou, M., Wang, T., Fu, J., Xiao, G. &amp; Liu, Y. Nonoptimal codon usage influences protein structure in intrinsically disordered regions. <em>Mol. Microbiol</em><strong>97</strong>, 974–987 (2015).
</cite> [<a href="https://doi.org/10.1111/mmi.13079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4636118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26032251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20M.,%20Wang,%20T.,%20Fu,%20J.,%20Xiao,%20G.%20&amp;%20Liu,%20Y.%20Nonoptimal%20codon%20usage%20influences%20protein%20structure%20in%20intrinsically%20disordered%20regions.%20Mol.%20Microbiol97,%20974%E2%80%93987%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Zhang, G., Hubalewska, M. &amp; Ignatova, Z. Transient ribosomal attenuation coordinates protein synthesis and co-translational folding. <em>Nat. Struct. Mol. Biol.</em><strong>16</strong>, 274–280 (2009).
</cite> [<a href="https://doi.org/10.1038/nsmb.1554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19198590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20G.,%20Hubalewska,%20M.%20&amp;%20Ignatova,%20Z.%20Transient%20ribosomal%20attenuation%20coordinates%20protein%20synthesis%20and%20co-translational%20folding.%20Nat.%20Struct.%20Mol.%20Biol.16,%20274%E2%80%93280%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Bicknell, A. A. et al. Attenuating ribosome load improves protein output from mRNA by limiting translation-dependent mRNA decay. <em>Cell Rep.</em><strong>43</strong>, 114098 (2024).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.114098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38625793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bicknell,%20A.%20A.%20et%20al.%20Attenuating%20ribosome%20load%20improves%20protein%20output%20from%20mRNA%20by%20limiting%20translation-dependent%20mRNA%20decay.%20Cell%20Rep.43,%20114098%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Gamble, C. E., Brule, C. E., Dean, K. M., Fields, S. &amp; Grayhack, E. J. Adjacent Codons Act in Concert to Modulate Translation Efficiency in Yeast. <em>Cell</em><strong>166</strong>, 679–690 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.05.070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4967012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27374328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gamble,%20C.%20E.,%20Brule,%20C.%20E.,%20Dean,%20K.%20M.,%20Fields,%20S.%20&amp;%20Grayhack,%20E.%20J.%20Adjacent%20Codons%20Act%20in%20Concert%20to%20Modulate%20Translation%20Efficiency%20in%20Yeast.%20Cell166,%20679%E2%80%93690%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Rurik, J. G. et al. CAR T cells produced in vivo to treat cardiac injury. <em>Science</em><strong>375</strong>, 91–96 (2022).
</cite> [<a href="https://doi.org/10.1126/science.abm0594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9983611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34990237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rurik,%20J.%20G.%20et%20al.%20CAR%20T%20cells%20produced%20in%20vivo%20to%20treat%20cardiac%20injury.%20Science375,%2091%E2%80%9396%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Ng, A. H. M. et al. A comprehensive library of human transcription factors for cell fate engineering. <em>Nat. Biotechnol.</em><strong>39</strong>, 510–519 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0742-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33257861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ng,%20A.%20H.%20M.%20et%20al.%20A%20comprehensive%20library%20of%20human%20transcription%20factors%20for%20cell%20fate%20engineering.%20Nat.%20Biotechnol.39,%20510%E2%80%93519%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Crawford, K. H. D. et al. Protocol and Reagents for Pseudotyping Lentiviral Particles with SARS-CoV-2 Spike Protein for Neutralization Assays. <em>Viruses</em><strong>12</strong>, 513 (2020).
</cite> [<a href="https://doi.org/10.3390/v12050513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7291041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32384820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crawford,%20K.%20H.%20D.%20et%20al.%20Protocol%20and%20Reagents%20for%20Pseudotyping%20Lentiviral%20Particles%20with%20SARS-CoV-2%20Spike%20Protein%20for%20Neutralization%20Assays.%20Viruses12,%20513%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Ding, W. et al. Chimeric design of pyrrolysyl-tRNA synthetase/tRNA pairs and canonical synthetase/tRNA pairs for genetic code expansion. <em>Nat. Commun.</em><strong>11</strong>, 3154 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-16898-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7308279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32572025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20W.%20et%20al.%20Chimeric%20design%20of%20pyrrolysyl-tRNA%20synthetase/tRNA%20pairs%20and%20canonical%20synthetase/tRNA%20pairs%20for%20genetic%20code%20expansion.%20Nat.%20Commun.11,%203154%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Hassett, K. J. et al. Optimization of Lipid Nanoparticles for Intramuscular Administration of mRNA Vaccines. <em>Mol. Ther. Nucleic Acids</em><strong>15</strong>, 1–11 (2019).
</cite> [<a href="https://doi.org/10.1016/j.omtn.2019.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6383180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30785039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hassett,%20K.%20J.%20et%20al.%20Optimization%20of%20Lipid%20Nanoparticles%20for%20Intramuscular%20Administration%20of%20mRNA%20Vaccines.%20Mol.%20Ther.%20Nucleic%20Acids15,%201%E2%80%9311%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information File</a><sup> (44.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (109.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (813.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373798/bin/41467_2025_62981_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (25MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data from the Ribosome profiling, tRNA sequencing and RNA-seq have been deposited in the Gene Expression Omnibus (GEO) under the accession numbers GSE301911, GSE301914, GSE301916. Data from tRNA profiling of Vero E6 cells infected with SARS-CoV-2 virus have been deposited in Figshare (10.6084/m9.figshare.29510411). The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the iProX partner repository with the dataset identifier PXD065927. Source data is available for Figs. <a href="#Fig1" class="usa-link">1</a>–<a href="#Fig6" class="usa-link">6</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">1</a>-<a href="#MOESM1" class="usa-link">9</a> in the associated source data file. <a href="#Sec41" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62981-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62981.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373798/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373798/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373798%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373798/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373798/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373798/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846860/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373798/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846860/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373798/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373798/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="uOvLI8evTvctWz8A3MajbjzmSSFJRe6ajyUxpXvmlTrsJEF4NQ9jnDHGD21lnR9b">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
