
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Dissecting pOXA-48 fitness effects in clinical Enterobacterales using plasmid-wide CRISPRi screens - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B9598AF3FCD305B959002CCE7ACA.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361523/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Dissecting pOXA-48 fitness effects in clinical Enterobacterales using plasmid-wide CRISPRi screens">
<meta name="citation_author" content="Alicia Calvo-Villamañán">
<meta name="citation_author_institution" content="Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain">
<meta name="citation_author" content="Jorge Sastre-Dominguez">
<meta name="citation_author_institution" content="Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain">
<meta name="citation_author" content="Álvaro Barrera-Martín">
<meta name="citation_author_institution" content="Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain">
<meta name="citation_author" content="Coloma Costas">
<meta name="citation_author_institution" content="Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain">
<meta name="citation_author" content="Álvaro San Millan">
<meta name="citation_author_institution" content="Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain">
<meta name="citation_author_institution" content="Centro de Investigación Biológica en Red de Epidemiología y Salud Pública, Instituto de Salud Carlos III, Madrid, Spain">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7700">
<meta name="citation_doi" content="10.1038/s41467-025-63082-1">
<meta name="citation_pmid" content="40825783">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361523/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361523/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361523/pdf/41467_2025_Article_63082.pdf">
<meta name="description" content="Conjugative plasmids are the main vehicle for the spread of antimicrobial resistance (AMR) genes in clinical bacteria. AMR plasmids allow bacteria to survive antibiotic treatments, but they also produce physiological alterations in their hosts that ...">
<meta name="og:title" content="Dissecting pOXA-48 fitness effects in clinical Enterobacterales using plasmid-wide CRISPRi screens">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Conjugative plasmids are the main vehicle for the spread of antimicrobial resistance (AMR) genes in clinical bacteria. AMR plasmids allow bacteria to survive antibiotic treatments, but they also produce physiological alterations in their hosts that ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361523/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12361523">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63082-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63082.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12361523%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12361523/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12361523/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361523/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7700. doi: <a href="https://doi.org/10.1038/s41467-025-63082-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63082-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Dissecting pOXA-48 fitness effects in clinical Enterobacterales using plasmid-wide CRISPRi screens</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvo-Villama%C3%B1%C3%A1n%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Alicia Calvo-Villamañán</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Alicia Calvo-Villamañán</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvo-Villama%C3%B1%C3%A1n%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alicia Calvo-Villamañán</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sastre-Dominguez%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jorge Sastre-Dominguez</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jorge Sastre-Dominguez</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sastre-Dominguez%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jorge Sastre-Dominguez</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barrera-Mart%C3%ADn%20%C3%81%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Álvaro Barrera-Martín</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Álvaro Barrera-Martín</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barrera-Mart%C3%ADn%20%C3%81%22%5BAuthor%5D" class="usa-link"><span class="name western">Álvaro Barrera-Martín</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costas%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Coloma Costas</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Coloma Costas</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costas%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Coloma Costas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22San%20Millan%20%C3%81%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Álvaro San Millan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Álvaro San Millan</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain </div>
<div class="p">
<sup>2</sup>Centro de Investigación Biológica en Red de Epidemiología y Salud Pública, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22San%20Millan%20%C3%81%22%5BAuthor%5D" class="usa-link"><span class="name western">Álvaro San Millan</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Microbial Biotechnology, Centro Nacional de Biotecnología (CNB), Consejo Superior de Investigaciones Científicas (CSIC), Madrid, Spain </div>
<div id="Aff2">
<sup>2</sup>Centro de Investigación Biológica en Red de Epidemiología y Salud Pública, Instituto de Salud Carlos III, Madrid, Spain </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 24; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12361523  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40825783/" class="usa-link">40825783</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Conjugative plasmids are the main vehicle for the spread of antimicrobial resistance (AMR) genes in clinical bacteria. AMR plasmids allow bacteria to survive antibiotic treatments, but they also produce physiological alterations in their hosts that commonly translate into fitness costs. Despite the key role of plasmid-associated fitness effects in AMR evolution, their origin and molecular bases remain poorly understood. In this study, we introduce plasmid-wide CRISPR interference (CRISPRi) screens as a tool to dissect plasmid-associated fitness effects. We design and perform CRISPRi screens targeting the globally distributed carbapenem resistance plasmid pOXA-48 in 13 different multidrug resistant clinical Enterobacterales. Our results reveal that pOXA-48 gene-level effects are conserved across clinical strains, and expose the key role of the carbapenemase-encoding gene, <em>bla</em><sub>OXA-48</sub>, as the main culprit for pOXA-48 fitness costs. Moreover, our results highlight the relevance of postsegregational killing systems in pOXA-48 vertical transmission, and uncover new genes implicated in pOXA-48 stability (<em>pri, korC</em>, DNDJGHEP_13 and 14 and H-NS). This study sheds new light on the biology and evolution of carbapenem resistant Enterobacterales and endorses CRISPRi screens as a powerful method for studying plasmid-mediated AMR.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Bacterial genetics, Clinical microbiology, Functional genomics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">This study investigates the effects of the carbapenem resistance plasmid pOXA-48 in clinical enterobacteria. Using CRISPRi screens, the authors revealed that the carbapenemase OXA-48 has a role in the plasmid-associated fitness costs.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Antimicrobial Resistance (AMR) is quickly becoming one of the most concerning global health problems, with 1-1.28 M deaths worldwide associated with AMR infections each year<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Studies have estimated that in a few decades AMR infections will be the main cause of non-natural death, increasing the yearly toll to 10 M deaths associated with AMR<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> and causing an annual loss of 1% of the world GDP<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. AMR is particularly alarming in the clinical setting due to the generalised use of antibiotics in hospitals, which expedites the evolution of antimicrobial resistant bacteria<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In the past decades, a growing number of studies has focused on studying AMR evolution in clinical bacteria<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Still, the vast majority of scientific studies on AMR evolution to date have been performed with laboratory bacterial strains, which don’t fully represent the complexity of clinical ones. We are therefore in dire need of more scientific studies performed in clinical strains, ones that are able to capture AMR in its full, real-world complexity<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par4">AMR can be achieved through different mechanisms<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Out of them, conjugative plasmids play an essential role in the spread of AMR among clinical Enterobacterales<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. These extrachromosomal genetic elements often carry AMR genes and have the ability to be transferred horizontally to cells that are nearby, promoting the dissemination of AMR genes across bacterial populations<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Plasmids can be double-edged swords for the bacteria that carry them. On the one hand, they carry genes that can be useful in the right conditions (e.g., a specific AMR gene when a given antibiotic is prescribed to a patient). On the other hand, they alter the fine-tuned metabolism of the cell, often causing changes to the fitness of their bacterial hosts<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Multiple studies have shown plasmids to produce variable fitness effects across bacterial hosts<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. These host-specific responses to carrying the same plasmid help explain why plasmids are frequently associated with a small fraction of hosts, regardless of their often wide host range.</p>
<p id="Par5">Plasmid-associated fitness costs arise from genetic conflicts between the novel plasmid genes and the genetic background of the host<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The molecular mechanisms behind these interactions are multifactorial and poorly understood<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Plasmid’s costs can originate during the different phases of their life-cycle in the cell: from plasmid reception<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, to replication<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, gene expression<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> and conjugation<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. However, recent studies suggest that the main source of plasmid fitness costs are the interactions between plasmid-encoded proteins and the host bacterium<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. For example, plasmids often encode for homologues of bacterial regulators which they can use to re-tune the regulatory networks of the bacterium to their advantage<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. While beneficial to plasmid survival, these changes in metabolic flux might affect essential cellular processes and create a fitness cost for the host. Similarly, by encoding alleles of genes also present in the bacterial chromosome, plasmids might disturb the optimal protein concentration, leading to a fitness cost<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Finally, by encoding homologues of highly connected bacterial proteins, multiple metabolic processes might be affected at once in the host<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Several studies have centred on trying to understand how plasmid fitness costs play a role in plasmid evolution. A trade-off between horizontal and vertical transmission of the plasmid has been proposed: a very efficient plasmid conjugation machinery tends to be costly to the plasmid host<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. A second trade-off between the expression of resistance genes and vertical transmission has also been proposed: in the absence of antibiotics the costs associated with plasmids have been shown to often arise from the expression of their resistance genes<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. pOXA-48 is a highly-conjugative plasmid of great clinical relevance, which promotes the dissemination of the carbapenemase-encoding gene <em>bla</em><sub>OXA-48</sub> across clinical Enterobacterales<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Carbapenemases are able to hydrolyse carbapenems, which are used in hospitals as last-resort treatments of multidrug resistant (MDR) infections. Because of this, pOXA-48 dissemination in the nosocomial context represents a global public-health threat. Despite its order-wide host range, clinical data shows the plasmid to be strongly associated with specific high-risk <em>Klebsiella pneumoniae</em> clones (such as those belonging to sequence type ST11<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>).</p>
<p id="Par6">The aim of this study is to uncover the origin and molecular basis of pOXA-48-associated fitness effects at the gene level. Specifically, we investigated the fitness effects associated with silencing each gene in the plasmid in different clinical strains of Enterobacterales using CRISPR interference screens (CRISPRi). Our results show that although carrying pOXA-48 leads to very different fitness responses in its different hosts, the gene-level effects are conserved amongst them. We confirm the importance of the toxin-antitoxin (TA) system PemI-PemK in the vertical transmission of the plasmid in clinical strains, and highlight the previously unknown role of plasmid-encoded modulators of expression in pOXA-48 stability in <em>E. coli</em>. Our data also suggests that pOXA-48 might be better adapted to <em>K. pneumoniae</em> than to <em>Escherichia coli</em>, which could explain its strong association in the clinic with the first species. Most importantly, we show that the carbapenemase OXA-48 is the main responsible for the fitness costs associated with the plasmid.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Designing a pOXA-48-wide CRISPRi screen for clinical Enterobacterales</h3>
<p id="Par7">We started by designing and building a CRISPRi set-up to study the gene-level fitness effects of pOXA-48 in clinical strains of Enterobacterales. CRISPRi works by using dCas9, a DNA-binding protein that can be directed to virtually any region of DNA guided by a small RNA molecule named single-guide RNA (sgRNA). Once bound to DNA, dCas9 inhibits the expression of the targeted region (Fig. <a href="#Fig1" class="usa-link">1A</a>). In CRISPRi screens, a pool of bacteria is built so that each individual bacterium carries dCas9 programmed to silence an individual gene through a specific sgRNA. This pool of bacteria can then be propagated, and the fitness effects of each gene in the given context can be determined through the change in frequency of gene-specific sgRNAs in the population (Fig. <a href="#Fig1" class="usa-link">1B</a>). For the pOXA-48-wide CRISPRi screen, we designed 5 sgRNAs targeting each plasmid gene and up to 10 sgRNAs per intergenic region (depending on the size of the region, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Each sgRNA was cloned in pFR56apm, a vector expressing the CRISPRi machinery under DAPG induction. The resulting 568-plasmid library was named OXAlib.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. CRISPRi screens set-up and fitness effects of pOXA-48.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361523_41467_2025_63082_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/960b/12361523/30471ab9de9f/41467_2025_63082_Fig1_HTML.jpg" loading="lazy" id="d33e427" height="735" width="798" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Depiction of the mechanism of CRISPRi screens and fitness costs associated with carrying pOXA-48 in the strains under study. <strong>A</strong> Schematic representation of CRISPRi gene-silencing and the set-up of CRISPRi screens. Values in pOXA-48 correspond to Kb. Each individual arrow represents an sgRNA targeting the region, and the orientation of the arrow indicates the orientation of dCas9 when binding to the DNA strand (RNAP stands for RNA polymerase). <strong>B</strong> Schematic representation of CRISPRi screens. Each colour corresponds to bacteria with a different pOXA-48 gene silenced (<em>i.e</em>., carries pFR56apm with an sgRNA programmed against said gene). Bacteria that carry a guide silencing a beneficial gene will decrease in frequency in the population during the screening, lowering the amount of the guide over time (e.g., red cells). On the other hand, bacteria carrying guides that silence costly genes will be enriched during the screening (e.g., light blue cells). <strong>C</strong> Unrooted phylogenetic tree of the Enterobacterales clinical strains used in this study. Multi Locus Sequence Types (STs) are indicated. <strong>D</strong> Distribution of pOXA-48 fitness effects (Relative Area Under the Growth Curve (see “Methods”), <em>n</em> = 16, error bars correspond to Standard Deviation) in the clinical strains selected for the CRISPRi screening. Asterisks indicate significant cost of pOXA-48 in each of the strains (two-sided paired t-tests against each relative plasmid-free strain after Bonferroni correction; <em>n</em> = 16; <em>p</em> &lt; 0.05).</p></figcaption></figure><p id="Par8">We selected a collection of 13 clinical strains of <em>K. pneumoniae</em> (<em>n</em> = 8) and <em>E. coli</em> (<em>n</em> = 5), representative of the diversity of MDR Enterobacterales isolated from hospitalised patients at a large hospital in Madrid<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, to perform the CRISPRi screen. This collection included clones that are typically associated with pOXA-48 in our hospital and others, such as <em>K. pneumoniae</em> ST11, ST307 and <em>E. coli</em> ST10<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> (Fig. <a href="#Fig1" class="usa-link">1C</a>). As a control we also included <em>E. coli</em> J53, a lab strain derived from K12<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. For more information on the strains used, consult the supplementary data (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>).</p>
<p id="Par9">We first determined the fitness effects of pOXA-48 (variant K8, GenBank <a href="https://www.ncbi.nlm.nih.gov/nuccore/MT441554" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT441554</a>, belonging to plasmid taxonomy unit L/M [IncL]) in each individual host, by comparing the growth of pOXA-48 carrying and pOXA-48-free isogenic versions of each strain (Fig. <a href="#Fig1" class="usa-link">1D</a>). As previously described, the plasmid produced variable fitness effects in the 14 strains<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p>
<p id="Par10">One of the limitations of using CRISPRi in wild type, clinical strains is that the efficiency of CRISPRi silencing can vary between strains. These differences can be due to several factors, such as different levels of expression of the CRISPRi machinery in each strain, different guide strength for each guide in the library, or aspects intrinsic to the CRISPR machinery that vary from strain to strain. Therefore, prior to performing the CRISPRi screens, we confirmed the efficiency of dCas9-mediated silencing in all strains using two different approaches. First, we targeted the essential gene <em>rpsL</em><sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> with a guide expressed from the pFR56apm vector and confirmed a strong reduction in the viability of all strains (SFig. <a href="#MOESM1" class="usa-link">1</a>). Second, we introduced a GFP-producing version of pOXA-48 (pOXA-48A_nuc::sfGFP) in each strain and measured the reduction in GFP expression associated with targeting the <em>gfp</em> gene by flow cytometry (SFig. <a href="#MOESM1" class="usa-link">2</a>). Our results confirmed that the CRISPRi machinery is highly active in all strains.</p></section><section id="Sec4"><h3 class="pmc_sec_title">CRISPRi screens reveal conserved pOXA-48 gene-level effects across clinical strains</h3>
<p id="Par11">We introduced the OXAlib library in the 14 pOXA-48-carrying strains and performed the CRISPRi screens with continuous dCas9 expression for ~85 generations (72 h). The screens were performed in each strain individually, in triplicate, both without antibiotic pressure and in the presence of sub-inhibitory concentrations of ertapenem, to which pOXA-48 confers resistance. Due to the differences in MIC for the different strains, different concentrations of ertapenem were used for each strain (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). To determine the fitness effects of pOXA-48-encoded genes, we collected samples at 0, ~25, ~55 and ~85 generations and measured the fitness effect associated with silencing each gene individually. In CRISPRi screens, changes in the distribution of each sgRNA in the pool throughout time are used as proxy for the fitness effects associated with silencing the gene they target. These changes are represented as the log2 fold-change (log2FC) of each sgRNA in the population. For simplicity, we refer to log2FC as “genes scores” in the rest of this manuscript. The temporal gene score dynamics of each gene were analysed for each time-point (SFig. <a href="#MOESM1" class="usa-link">3</a>, <a href="#MOESM1" class="usa-link">4</a>). The data analysed in the rest of this paper corresponds to ~85 generations, as that’s when the effect of each sgRNA was most visible. In parallel, the stability in the population of both pOXA-48 and pFR56apm were also assessed (SFig. <a href="#MOESM1" class="usa-link">5</a>).</p>
<p id="Par12">Despite the variable fitness effects produced by pOXA-48 across the clinical strains (Fig. <a href="#Fig1" class="usa-link">1D</a>), we observed that silencing each pOXA-48 gene individually led to similar fitness effects across strains (Fig. <a href="#Fig2" class="usa-link">2A</a>). To confirm this observation, we correlated the gene scores of all the genes and intergenic regions between all the strains per experimental condition. Pearson correlation indexes (<em>r</em>) resulted in highly positive values for most of the pairwise comparisons between strains both in absence and presence of ertapenem (0.25 &lt; <em>r</em> &lt; 0.90, median = 0.64 without ertapenem; −0.07 &lt; <em>r</em> &lt; 0.91, median = 0.66 with ertapenem. Figure <a href="#Fig2" class="usa-link">2B</a>; Supplementary Data <a href="#MOESM5" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. pOXA-48-encoded genes produce similar fitness effects across clinical strains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361523_41467_2025_63082_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/960b/12361523/ae8396f280f6/41467_2025_63082_Fig2_HTML.jpg" loading="lazy" id="d33e559" height="702" width="798" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representation of the fitness effects associated with silencing each gene in pOXA-48 individually in the presence and absence of antibiotic pressure, and how conserved those effects are amongst strains. <strong>A</strong> Heatmap of CRISPRi gene scores at the end of the screening (t = 72 h, ~85 generations). Fitness effects associated with silencing each individual gene / intergenic region in the absence of selection (top panel) and in the presence of ertapenem (bottom panel). The score for each gene / intergenic region corresponds to the median log<sub>2</sub> fold-change of the sgRNAs targeting the element in the population (see “Methods”). Blue shades indicate those genes which silencing is detrimental (i.e., genes that are beneficial in that condition), whereas yellow shades correspond to genes which silencing is beneficial during the experiment (i.e., genes that are costly in that condition). Coding regions of pOXA-48 are shown with different colours indicating common functional clusters as shown on top. Genes and surrounding intergenic regions showing a significant deviation in the gene scores in both conditions are highlighted in the top. A fully annotated version of the figure is available in SFig. <a href="#MOESM1" class="usa-link">7</a>. <strong>B</strong> Pearson correlation matrices between strains’ gene scores at the end of the screening without (left) and with ertapenem (right). Blue shades indicate positive Pearson correlation coefficients, meaning similar tendencies in the overall screening results between the strains. Circle sizes indicate correlation absolute values from 0 to 1 (blue) or −1 to 0 (red).</p></figcaption></figure><p id="Par13">Finally, we aimed to confirm that the overall gene-level effects revealed in CRISPRi screens were able to recapitulate the fitness effects produced by the entire pOXA-48 plasmid. Our results showed that the median gene scores of plasmid-targeting sgRNAs in the absence of ertapenem (compared to the control guides) anticorrelated with the relative AUC (Area Under the growth Curve) of pOXA-48-carrying strains (<em>R</em> = −0.73, <em>P</em> value = 0.0045). Therefore, silencing plasmid genes is more beneficial as the plasmid increases in cost. These results confirmed that our CRISPRi data is able to reflect overall plasmid fitness effects (SFig. <a href="#MOESM1" class="usa-link">6</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Analysing pOXA-48 gene-level fitness effects</h3>
<p id="Par14">To identify which genes play a significant role in pOXA-48 fitness effects, we performed a permutation test to pinpoint the genes whose silencing led to statistically significant changes in frequency both in presence and absence of ertapenem (Fig. <a href="#Fig3" class="usa-link">3A</a>). These genes were (i) <em>bla</em><sub>OXA-48</sub>, the β-lactamase gene; (ii) <em>repA</em>, the replication initiation gene, and <em>repC</em>, transcriptional repressor involved in plasmid copy number control (in further experiments we study <em>repA</em> as representative of the <em>repCBA</em> locus); (iii) <em>pri</em>, which is involved in conjugation, and shows homology to DNA primases; (iv) <em>korC</em>, a transcriptional repressor with unknown function in pOXA-48; (v) a H-NS-like family regulator gene; (vi) an operon with unknown function formed by DNDJGHEP_0013 and _0014, predicted to be an ABC-transporter substrate binding protein and an Xre-like transcriptional regulator, respectively; and (vii) <em>pemI</em>, the antitoxin of the <em>pemI-pemK</em> system, responsible for post-segregational killing in case of plasmid mis-segregation<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. In line with our previous results (Fig. <a href="#Fig2" class="usa-link">2</a>, SFig. <a href="#MOESM1" class="usa-link">7</a>), the genes showing significant effects were highly conserved across strains (SFig. <a href="#MOESM1" class="usa-link">8</a>). Interestingly, the response was more generally conserved in <em>K. pneumoniae</em> than in <em>E. coli</em> (Welch two sample t-test, t = −2.1694 <em>P</em> value = 0.04234 without ertapenem; two-sided Kruskal-Wallis test, chi-squared = 5.6326, <em>P</em> value = 0.01763 with ertapenem; SFig. <a href="#MOESM1" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Role of different plasmid genes in pOXA-48 biology.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361523_41467_2025_63082_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/960b/12361523/25f588c3d37f/41467_2025_63082_Fig3_HTML.jpg" loading="lazy" id="d33e684" height="512" width="767" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identification of pOXA-48 genes producing significant fitness effects and their effect on plasmid stability. <strong>A</strong> Volcano plots obtained from two-sided permutation test assays in absence of selection (left) and in the presence of ertapenem (right). The Y axis shows the logarithmic transformation of the adjusted <em>P</em> value (FDR) from the permutation test, while the x axis indicates the median gene score (log<sub>2</sub> fold-change values for sgRNAs targeting each gene from CRISPRi screens). Grey points indicate genes and intergenic regions which silencing was non-significant (i.e., −0.5 &lt; gene score &lt; 0.5; and <em>P</em> &gt; 0.05). Genes and intergenic regions which silencing was detrimental in each condition are indicated in red, whereas genes which silencing was beneficial are coloured in blue. The labels of genes encoding hypothetical proteins are shown in grey. <strong>B</strong> pOXA-48 stability after 24 h of CRISPRi gene silencing of individual plasmid genes (<em>X</em> axis) in a subset of 5 strains (Y axis; 2 <em>E. coli</em>: EC22 and C288; and 3 <em>K. pneumoniae</em>: KPN18, KPN11 and KPN10) from our screenings (in presence of apramycin to select for pFR56apm). <strong>C</strong> pOXA-48 Copy Number measured silencing individual plasmid genes relative to control guide values for each strain tested (Log<sub>10</sub> scale). Negative values indicate PCN decrease when silencing the indicated gene. Each point shows an individual replicate (<em>n</em> = 6). Bars correspond to median relative PCN values for all the replicates in each of the strains tested. Bars shaded in blue correspond to <em>K. pneumoniae</em> strains, and in pink to <em>E. coli</em> strains.</p></figcaption></figure><p id="Par15">Our analysis detected genes that, when silenced, produced a significant advantage or disadvantage for the bacterial host. In the absence of ertapenem, silencing five of the genes led to a negative effect (<em>pri</em>, H-NS, DNDJGHEP_0013 and _0014, and <em>pemI</em>), while silencing of <em>bla</em><sub>OXA-48</sub> and <em>repA</em> led to a positive fitness effect (Fig. <a href="#Fig3" class="usa-link">3A</a>, left panel). In the presence of ertapenem, silencing of all seven genes led to a detrimental effect (Fig. <a href="#Fig3" class="usa-link">3A</a>, right panel).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Investigating the role of individual genes on plasmid stability</h3>
<p id="Par16">Our results showed that most of the genes producing a significant signal when silenced led to a fitness disadvantage. For <em>bla</em><sub>OXA-48</sub>, the fitness disadvantage in the presence of ertapenem can be easily explained because the OXA-48 carbapenemase is responsible for the resistance phenotype. In the case of <em>pemI</em>, silencing the antitoxin gene probably leads to a PemK-mediated toxic effect (although this is likely mitigated by the polar silencing of <em>pemK</em>, since <em>pemI-pemK</em> form an operon<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>). The effect of PemK is analysed in depth in the following section. For the remaining genes, we hypothesised that the fitness defects could be due to a reduction in plasmid stability. Specifically, in the presence of ertapenem, plasmid loss would produce an obvious growth defect, since the cells would no longer produce the OXA-48 carbapenemase. Moreover, plasmid loss should also lead to a PemK-mediated post-segregational killing effect, and this should hold true both in presence and absence of ertapenem. Therefore, we decided to investigate the potential role of silencing these genes in pOXA-48 stability. We individually cloned the strongest sgRNA from the CRISPRi screen for each gene in pFR56apm. Then, we transformed the pFR56apm plasmids individually in a subset of the strains used in our screen, namely <em>K. pneumoniae</em> KPN10 (ST1427), KPN11 (ST307) and KPN18 (ST11), and <em>E. coli</em> C288 (ST744) and EC22 (ST131). Using these constructions we investigated how blocking the expression of each gene affected plasmid stability.</p>
<p id="Par17">We blocked the expression of each gene individually over a 24 h growth cycle and measured the stability of pOXA-48 using replica plating (Fig. <a href="#Fig3" class="usa-link">3B</a>). We confirmed the phenotypic results by measuring pOXA-48 copy number in the same populations by qPCR (Fig. <a href="#Fig3" class="usa-link">3C</a>). Our results showed that silencing all the genes under study, except <em>bla</em><sub>OXA-48</sub>, led to a reduction in pOXA-48 stability compared to the non-targeting control guide at least in one of the strains (one-tailed z-tests after Bonferroni adjustment, <em>n</em> = 5, <em>p</em> &lt; 0.05). The most dramatic decrease in plasmid stability was mediated by silencing <em>repA</em>, which led to the complete curing of the plasmid in 4/5 of our strains. This was to be expected, given the role of RepA as the replication initiation protein of pOXA-48. Surprisingly, <em>pri</em> seems to play a similar role in plasmid stability as <em>repA</em>. This gene had been described as essential for conjugation in a similar plasmid to pOXA-48 belonging to the plasmid taxonomic unit L/M (IncM)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. However, its potential activity as a DNA primase could play a role in plasmid replication as well.</p>
<p id="Par18">H-NS, <em>korC</em> and the DNDJGHEP_0013 and _0014 genes produced a stronger effect in pOXA-48 stability in <em>E. coli</em> than in <em>K. pneumoniae</em>. These results are further supported by a parallel study working with a transposon insertion library of pOXA-48 in <em>E. coli</em> MG1655, in which Baffert et al. showed that KorC and DNDJGHEP_0014 (<em>orf20</em>) are essential for plasmid stability<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The specific mechanisms by which these genes affect plasmid stability are difficult to assess. H-NS, <em>korC</em> and DNDJGHEP_0014 (which forms an operon with DNDJGHEP_0013 and is therefore also silenced by guides targeting this upstream gene) encode for transcriptional regulators. Further work will be needed to clarify the transcriptional effects associated with silencing these genes and how they impact plasmid stability. Interestingly, although silencing these genes produced a negative effect in most <em>K. pneumoniae</em> strains in our CRISPRi screens, they affected pOXA-48 stability to a lower extent in this second experiment. However, it is important to remember that the results of silencing individual genes were obtained after a single growth cycle (8-10 generations), while the CRISPRi screens were performed for 85 generations. Finally, and as expected, <em>pemI</em> produced a very modest effect on plasmid stability, and <em>bla</em><sub>OXA-48</sub> produced no effect at all.</p></section><section id="Sec7"><h3 class="pmc_sec_title">PemK enhances vertical transmission of pOXA-48</h3>
<p id="Par19">Previous studies performed in laboratory <em>E. coli</em> strains had shown that PemK is responsible for the fitness defects associated with pOXA-48 loss<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. We hypothesised that this effect should also hold true in MDR clinical strains of Enterobacterales. To test this hypothesis, we first cloned <em>pemK</em> in an expression vector under the control of the pTet promoter. We introduced this vector in <em>K. pneumoniae</em> KPN10 and <em>E. coli</em> EC22. Interestingly, even if the expression of <em>pemK</em> was not induced, we could not obtain viable transformants for KPN10 carrying the intact vector, suggesting that even a leaky expression of <em>pemK</em> is lethal in this strain. We tried to transform the vector in a different <em>K. pneumoniae</em> strain, KPN11, with the same result. For <em>E. coli</em> EC22, we could transform this vector, probably due to a tighter pTet repression in this species. Our results showed that <em>pemK</em> expression led to a dramatic growth defect in this strain (SFig. <a href="#MOESM1" class="usa-link">9</a>). These results confirm that PemK produces a toxic effect in our wild type clinical strains.</p>
<p id="Par20">To further confirm the relevance of PemK in pOXA-48 stability, we constructed a version of pOXA-48 lacking the toxin gene <em>pemK</em> (pOXA-48∆<em>pemK</em>), and introduced it in <em>K. pneumoniae</em> KPN10, KPN11 and KPN18, and <em>E. coli</em> C288 and EC22. We performed plasmid stability assays for 5 days to determine the stability of both wt-pOXA-48 and pOXA-48∆<em>pemK</em> in the different strains (Fig. <a href="#Fig4" class="usa-link">4A</a>). Wild-type pOXA-48 was fully stable in <em>K. pneumoniae</em> and in <em>E. coli</em> EC22, but it was slightly unstable in <em>E. coli</em> C288. pOXA-48∆<em>pemK</em> was surprisingly stable in <em>K. pneumoniae</em>, only starting to be lost in KPN11 and KPN18 after 4 and 3 days, respectively. Contrary, pOXA-48∆<em>pemK</em> was very unstable in <em>E. coli</em>, being lost up to 90% in EC22. Both pOXA-48 and pOXA-48∆<em>pemK</em> showed an erratic behaviour in <em>E. coli</em> C288. The fact that pOXA-48 does not produce a fitness cost in this strain, and therefore there is a small benefit associated with losing it, may help explain this behaviour (Fig. <a href="#Fig1" class="usa-link">1D</a>). Together, these results suggest that pOXA-48 segregation might be more efficient in <em>K. pneumoniae</em> than in <em>E. coli</em>. This hypothesis will have to be tested in future studies.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Role of PemK in pOXA-48 stability.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361523_41467_2025_63082_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/960b/12361523/d4d6f096d88c/41467_2025_63082_Fig4_HTML.jpg" loading="lazy" id="d33e920" height="288" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The PemI-PemK TA system promotes pOXA-48 vertical transmission. <strong>A</strong> Stability of pOXA-48 (darker lines) and pOXA-48∆<em>pemK</em> (lighter dashed lines) in the different clinical strains over 5 days in LB culture (with a daily 1:1000 dilution). <strong>B</strong> pOXA-48 and pOXA-48∆<em>pemK</em> stability after 24 h of CRISPRi gene silencing of individual plasmid genes.</p></figcaption></figure><p id="Par21">Finally, we decided to test the effect of silencing the three genes producing partial plasmid instability when silenced (<em>korC</em>, H-NS, and DNDJGHEP_0013-0014) in the pOXA-48∆<em>pemK</em> plasmid, including the <em>repA</em> guide as control. Our hypothesis was that if PemK is eliminating the cells where pOXA-48 is lost, once the <em>pemK</em> gene is removed, we should be able to observe a decrease in plasmid stability associated with silencing these genes. We performed these experiments in <em>K. pneumoniae</em> KPN10, KPN11 and <em>E. coli</em> EC22. In line with our hypothesis, we observed an increase in plasmid loss associated with silencing these three genes (Fig. <a href="#Fig4" class="usa-link">4B</a>). These results confirmed the relevance of <em>korC</em> and the DNDJGHEP_0013-14 operon in pOXA-48 stability in both species. The H-NS gene, on the other hand, seems to produce a species-specific effect. This observation is supported by our previous results (Fig. <a href="#Fig3" class="usa-link">3B</a>).</p>
<p id="Par22">Taken together, our results underline the key role of the PemI-PemK TA system promoting pOXA-48 vertical transmission in clinical Enterobacterales.</p></section><section id="Sec8"><h3 class="pmc_sec_title">
<em>bla</em><sub>OXA-48</sub> is responsible for pOXA-48 fitness costs</h3>
<p id="Par23">Only two pOXA-48 genes produce a significant fitness benefit when silenced in the absence of ertapenem, <em>repA</em> and <em>bla</em><sub>OXA-48</sub>. The results for <em>repA</em> are puzzling, since we have demonstrated that <em>repA</em> silencing produces a rapid plasmid loss, which leads to a toxic effect (Fig. <a href="#Fig3" class="usa-link">3B, C</a>). In fact, when looking at the CRISPRi screen results, <em>repA</em> is the only gene with significant effect that changes in sign over time. Silencing <em>repA</em> is associated with a fitness defect in the first time point and it becomes beneficial later on (SFig. <a href="#MOESM1" class="usa-link">10</a>). We comment on these counterintuitive results in more depth in the discussion section. Results for <em>bla</em><sub>OXA-48</sub>, on the other hand, are more straightforward. Previous reports had already indicated that <em>bla</em><sub>OXA-48</sub> expression was associated with a fitness cost in a clinical Enterobacterales strain<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Therefore, we decided to study <em>bla</em><sub>OXA-48</sub> effects in more detail.</p>
<p id="Par24">First, we correlated the CRISPRi score of <em>bla</em><sub>OXA-48</sub> with the relative growth of pOXA-48-carrying strains, obtaining a strong negative relationship (Fig. <a href="#Fig5" class="usa-link">5</a> A, Pearson R-squared = −0.91, <em>P</em> value = 0.0016, for <em>K. pneumoniae</em> and Pearson R-squared = −0.88, <em>P</em> value = 0.02, for <em>E. coli</em>). This correlation was in fact stronger than the one obtained including the scores of all the genes in the CRISPRi screen (SFig. <a href="#MOESM1" class="usa-link">6</a>), as well as stronger than the correlation obtained with the subset of genes producing significant fitness effects (SFigs. <a href="#MOESM1" class="usa-link">11</a>, <a href="#MOESM1" class="usa-link">12</a>). Therefore, these results strongly suggest that the main contributor to pOXA-48 cost is the <em>bla</em><sub>OXA-48</sub> gene.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Role of <em>bla</em><sub>OXA-48</sub> in pOXA-48 fitness effects.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361523_41467_2025_63082_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/960b/12361523/12095eace860/41467_2025_63082_Fig5_HTML.jpg" loading="lazy" id="d33e1097" height="356" width="799" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The <em>bla</em><sub>OXA-48</sub> gene is responsible for pOXA-48-associated fitness costs. <strong>A</strong> Two-sided Pearson correlation between the relative Area Under the growth Curve (AUC) of pOXA-48-carrying strains compared to their pOXA-48-free counterparts (shown in Fig. <a href="#Fig1" class="usa-link">1D</a>) and the gene score (log<sub>2</sub> fold change) associated with silencing <em>bla</em><sub>OXA-48</sub> in the CRISPRi screen. Lines indicate the best fit for the correlation of each species. Shadowed areas indicate the 95% CI. Each colour represents a species. <strong>B</strong> Plasmid cost associated with carrying pOXA-48 and pOXA-48∆<em>bla</em><sub>OXA-48</sub> in different clinical strains of Enterobacterales measured as relative AUC of pOXA-48-carrying compared to pOXA-48-free clones. Colours of strain names in the x-axis are again indicative of species: in pink <em>E. coli</em>, in blue <em>K. pneumoniae</em>, and in black <em>C. freundii</em>. Asterisks indicate significant cost/benefit of either pOXA-48 or pOXA-48∆<em>bla</em><sub>OXA-48</sub> in each of the strains (two-sided paired t-tests against each relative plasmid-free strain after Bonferroni correction; n = 8, except for KPN11, KPN10, KPN18, K163, EC22, C288 carrying pOXA-48, in which <em>n</em> = 16; <em>p</em> &lt; 0.05; error bars correspond to Standard Deviation).</p></figcaption></figure><p id="Par25">To experimentally validate these results, we used a version of plasmid pOXA-48 with a small deletion affecting the <em>bla</em><sub>OXA-48</sub> gene (pOXA-48∆<em>bla</em><sub>OXA-48</sub>). This plasmid was recently recovered from a clinical strain and does not produce a functional OXA-48 carbapenemase, but is otherwise isogenic to pOXA-48<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. pOXA-48∆<em>bla</em><sub>OXA-48</sub> has a small deletion of 199nt that comprises the full promoter region of <em>bla</em><sub>OXA-48</sub> and the first 36nt of the gene. Transcription of a truncated <em>bla</em><sub>OXA-48</sub> mRNA takes place, possibly from a secondary promoter, but levels of transcription are very low (between 1.8-8.4% of the wt <em>bla</em><sub>OXA-48</sub> expression) and does not lead to a resistant phenotype (SFig. <a href="#MOESM1" class="usa-link">13</a>). To further confirm that our previous results are generalisable, and not due to our choice of initial set of strains of work, we conjugated pOXA-48 and pOXA-48∆<em>bla</em><sub>OXA-48</sub> in a second set of clinical Enterobacterales including <em>Klebsiella</em> spp., <em>E. coli</em>, and <em>Citrobacter freundii</em> clones (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). As previously described<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, pOXA-48 produced variable fitness effects across the collection (average relative AUC = 0.938, SD = 0.075), causing a significant reduction in relative AUC in 7 out of 13 strains (paired t-tests against each relative plasmid-free strain after Bonferroni correction; n = 8-16; p &lt; 0.05, Fig. <a href="#Fig5" class="usa-link">5B</a>). Interestingly, and in line with the hypothesis of <em>bla</em><sub>OXA-48</sub> being responsible for pOXA-48-associated fitness costs, pOXA-48∆<em>bla</em><sub>OXA-48</sub> produced milder fitness effects in this collection (average relative AUC = 1.002, SD = 0.059), leading to significant costs in 2 of the strains and to significant fitness advantages in 2 other strains (paired t-tests against each relative plasmid-free strain after Bonferroni correction; n = 8-16; p &lt; 0.05, Fig. <a href="#Fig5" class="usa-link">5B</a>). Taken together, these results strongly suggest that the main cause of the fitness cost produced by pOXA-48 is the expression of the <em>bla</em><sub>OXA-48</sub> gene.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par26">Unveiling the complex interactions between a plasmid and its bacterial hosts is essential for understanding plasmid ecology and evolution, and therefore for deciphering the dynamics of AMR in real-life scenarios, such as the hospital set-up or the patient’s gut. To better understand these interactions, in this work we study the fitness effect of each gene in pOXA-48, using 13 different clinically relevant clones of Enterobacterales as hosts. To our knowledge, this is the first study to date to perform such an extensive screen in clinical strains, which allowed us to discover clinically-relevant interactions that could be missed by performing the experiments in laboratory strains. Our results showed that although pOXA-48 produces a wide range of fitness effects across the clinical clones (Fig. <a href="#Fig1" class="usa-link">1</a>), the effects of silencing each plasmid gene are overall conserved (Fig. <a href="#Fig2" class="usa-link">2</a>). This difference is driven by the fact that most of the gene-level fitness effects detected were “post-segregational killing” deleterious effects, associated with plasmid loss (Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>). These effects emerge in the CRISPRi screen due to the silencing of genes associated with plasmid stability, but they do not necessarily contribute to the fitness effects of the wild-type plasmid with a correct partitioning. However, these results highlight the efficiency of plasmid-encoded mechanisms, such as the <em>pemI-pemK</em> TA system, promoting vertical plasmid transmission in clinical strains (Fig. <a href="#Fig4" class="usa-link">4</a>). Concurrently, while these effects are qualitatively generalisable to all the strains, the degree to which each gene plays a role differs in each of the clinical strains. This could potentially be explained by the extensive pangenome of these strains (in particular their plasmidome, with each strain carrying very different sets of natural plasmids - whose gene-content could complement the silenced genes in a strain-to-strain manner). We claim that these observations highlight the need to perform genome-wide CRISPRi screens in clinical strains to better capture all potential genetic interactions in the cell. Finally, we could show that despite pOXA-48 carrying more than 90 genes and CDS, the fitness costs associated with this plasmid could be explained almost entirely by the expression of the carabapenemase gene <em>bla</em><sub>OXA-48</sub> (Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<p id="Par27">This study shows the potential of using CRISPRi screens to deepen our understanding of plasmid biology at a gene level. For example, we showed that apart from the previously known role in plasmid conjugation<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, Pri plays a role in plasmid replication, likely as a primase (Fig. <a href="#Fig3" class="usa-link">3B, C</a>). We also unveil the importance, particularly in <em>E. coli</em>, of three modulators of gene expression: KorC, DNDJGHEP_0014 and an H-NS-like protein. Individually silencing the genes coding for those proteins led to plasmid loss in <em>E. coli</em> (Fig. <a href="#Fig3" class="usa-link">3B, C</a>). Plasmids are known to encode for transcription regulators that control the expression of genes on both the plasmid and the host chromosome, thus manipulating the host cell behaviour to their benefit. Examples of such plasmid-encoded regulators are VirBR, which promotes transmission of IncX3 plasmids by upregulating their Type IV secretion system<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, RsmQ, which switches its host from a motile to a sessile lifestyle, thus promoting direct contact between individuals and promoting conjugation of the plasmid pQBR103<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, or several plasmid-encoded H-NS-like proteins, which have been shown to, amongst other functions, control the extracellular matrix composition of its host to regulate biofilm formation<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The precise role of KorC, DNDJGHEP_0014 and the H-NS-like protein, more specifically the plasmid and/or host genes under their modulation, will be explored in more detail in the future. The importance of KorC and DNDJGHEP_0014 (<em>orf20</em>) for pOXA-48 stability was however also observed in <em>E. coli</em> MG1655 in an elegant parallel study by Baffert et al. working with a transposon insertion library of pOXA-48<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. In this simultaneous study exploring the role of each plasmid gene in conjugation, the authors were not able to obtain pOXA-48 mutants disrupting <em>korC</em>. In addition, the authors showed that plasmid mutants disrupting DNDJGHEP_0014/<em>orf20</em> were only stable when an intact copy of the gene was provided in <em>trans</em>. Baffert et al.’s results further support the hypothesis of these genes playing a role in pOXA-48 stability. Likewise, we observed that although deletion of the H-NS-like protein from pOXA-48 is possible, it has a strong effect on plasmid stability in <em>E. coli</em> (SFig. <a href="#MOESM1" class="usa-link">14</a>). We also show that, as previously described<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, the PemI-PemK TA system promotes the vertical transmission of pOXA-48 to cell progeny. However, we observed that the role PemI-PemK play in vertical plasmid transmission varied among clones. pOXA-48∆<em>pemK</em> was stably transmitted to the progeny in the 3 <em>K. pneumoniae</em> strains tested but not in 2 <em>E. coli</em> strains (Fig. <a href="#Fig4" class="usa-link">4A</a>). Given that our data indicated that PemK is toxic both in <em>K. pneumoniae</em> and <em>E. coli</em>, these results suggest that plasmid segregation may be more efficient in the first species, which could help to explain why pOXA-48 is more strongly associated with <em>K. pneumoniae</em> than <em>E. coli</em> in the clinic<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. We hypothesise that this higher stability could be explained by the fact that pOXA-48’s plasmid copy number is higher in <em>K. pneumoniae</em> than in <em>E. coli</em> (<em>K. pneumoniae</em>, mean = 2.21 copies; <em>E. coli</em> mean = 1.41), as we recently showed<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Alternatively, this difference could be due to the partition machinery being more efficient in the first species than the latter, which would suggest that pOXA-48 is better adapted to the <em>K. pneumoniae</em> than <em>E. coli</em>. Finally, it could also be possible that additional TA systems in pOXA-48 may explain the differences in plasmid stability between species. In fact, Baffert et al. just reported the existence of a second TA system in pOXA-48, composed of a toxin, DqlB, and a non-coding RNA <em>trans</em>-acting antitoxin, <em>agrB</em><sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. This system had not been previously annotated in pOXA-48 due to its small size, and is located in the intergenic region between <em>repA</em> and <em>trbB</em>. Because of the lack of annotation, we did not design guides specifically targeting this region in the OXAlib. However, three of our <em>repA-trbB</em> intergenic guides target the antitoxin <em>agrB</em> (SFig. <a href="#MOESM1" class="usa-link">15A</a>). In line with the results from Baffert et al., we observed a dramatic decrease in the gene score of the guides silencing the antitoxin throughout our CRISPRi screens (SFig. <a href="#MOESM1" class="usa-link">15B-C</a>), supporting the role of TrbB-<em>agrB</em> as a second TA system in pOXA-48. The effect associated with silencing <em>agrB</em> was however similar for <em>K. pneumoniae</em> and <em>E. coli</em> strains, so it did not help to explain the higher stability of pOXA-48 in <em>K. pneumoniae</em>.</p>
<p id="Par28">In our CRISPRi screens only two genes produced a fitness benefit when silenced in the absence of antibiotic selection: <em>repA</em> and <em>bla</em><sub>OXA-48</sub>. The results for <em>repA</em> are difficult to explain, and highlight some of the limitations of our set-up. On the one hand, it is possible that a small subpopulation of the cells that carried a <em>repA</em> guide survived an early pOXA-48 loss in the experiment, overcoming the PemK and Dlbq toxic effect. This subpopulation would be free of the cost associated with pOXA-48 carriage and could therefore proliferate in the pool of cells. This would explain the rapid initial decline of guides targeting <em>repA</em> in the population, and their subsequent enrichment in later time-points (SFig. <a href="#MOESM1" class="usa-link">10</a>). On the other hand, it has been shown that weak and unpredictable CRISPRi gene repression can arise from targeting CDSs under negative regulatory feedback loops<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Rep proteins are known to follow this type of self-regulation and this could also contribute to the seemingly erratic behaviour of guides targeting the <em>repA locus</em><sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>.</p>
<p id="Par29">The main finding of this work is the fact that the gene encoding for the β-lactamase OXA-48 is the main contributor to pOXA-48’s fitness cost across clinical strains (Fig. <a href="#Fig5" class="usa-link">5</a>). Β-lactamases had already been shown to have an effect on bacterial cell physiology and lead to a decrease in fitness<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. These effects may have different origins. First, they could be produced by the β-lactamase’s signal peptide<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, responsible for folding, stability and translocation of the protein to the periplasm<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. By leading to a host-specific inefficient processing of the protein and subsequent envelope stress, the signal peptide effectively dictates the host-range of some β-lactamases<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, which could explain why plasmids encoding for these genes are more costly in some hosts than others. Second, the fitness costs associated with β-lactamases have also been associated with their residual DD-endopeptidase activity, which leads to the hydrolysis of the peptide-cross bridge and alters the structure and stability of the cell wall<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. This means that carrying plasmids such as pOXA-48 can lead to a severe destabilisation of the bacterial envelope which can have downstream repercussions, such as fitness costs and even collateral sensitivity to other antibiotics<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. This also paints a somewhat hopeful picture about the dissemination of these plasmids in the population: since the main source of cost of the plasmid is the AMR gene itself, in the absence of antibiotic treatment the cells will be under pressure to either lose the plasmid or mutate <em>bla</em><sub>OXA-48</sub>. The latter was already observed in a previous study analysing within-patient evolution of pOXA-48, where a small deletion affecting the <em>bla</em><sub>OXA-48</sub> gene and abolishing the AMR phenotype was associated with reduction of plasmid cost in the gut microbiota of a hospitalised patient<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Lastly, this also opens a gateway to explore new ways to exploit these changes in the cell envelope as potential new antimicrobial strategies, ones highly specific to cells carrying genes within the β-lactamase family<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p>
<p id="Par30">In summary, our study highlights the role of different plasmid-encoded genes in the biology of pOXA-48 and opens new doors to explore alternative ways to fight AMR in Enterobacterales. Our results also highlight the usefulness of using high-throughput techniques to explore previously unknown biological phenomena at a genetic level. And lastly, given that previous works indicate that both the fitness and transcriptomics response to carrying plasmids are highly host-specific<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, our study paves the way to performing genome-wide CRISPRi screens in order to dissect plasmid-bacteria interactions in clinical bacteria. Set-ups such as this, although complex and labour-intensive, will help us unveil aspects of plasmid biology with applications in real-life scenarios.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Bacterial strains and culture conditions</h3>
<p id="Par31">All clinical strains used in this study were obtained from the previously characterised R-GNOSIS collection of over 28,000 rectal samples collected from patients admitted to the Hospital Universitario Ramón y Cajal in Madrid between the years of 2014 and 2016 (R-GNOSIS-FP7-HEALTH-F3-2011-282512)).</p>
<p id="Par32">For the CRISPRi screens, a subset of these strains was selected based on previous works. First, a collection of strains that did not carry pOXA-48 naturally and where pOXA-48 transconjugants had been obtained<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, and, secondly, a collection of strains that carried pOXA-48 naturally and where pOXA-48 had been successfully cured<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Out of these two collections, strains were selected based on sensibility to apramycin - antibiotic resistance gene used for selection of the OXAlib library -, conjugation rate efficiency, belonging to a sequence type of clinical relevance, their plasmid profile and, lastly, activity of dCas9 - tested by measuring cell death by silencing the essential gene rpsL, as in Cui et al<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. and sfGFP (Supplementary Methods, SFig. <a href="#MOESM1" class="usa-link">1</a>, <a href="#MOESM1" class="usa-link">2</a>)). The resulting selection comprises 14 strains, 8 <em>K. pneumoniae</em> strains and 6 <em>E. coli</em> strains (including the lab strain J53) (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). The clinical strains belong to STs of clinical relevance, namely <em>K. pneumoniae</em> ST1427 (KPN01, KPN07 and KPN10), ST11 (KPN15, KPN18 and K153) and ST307 (KPN11 and K163), and <em>E. coli</em> ST10 (EC23), ST131 (EC08 and EC22), ST744 (C288) and ST4669 (EC05).</p>
<p id="Par33">The variant of pOXA-48 used in this work is pOXA-48_K8 (accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/MT441554" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT441554</a>), and its sequence and complete genomes of the strains used in this study were obtained by the lab in previous works, available under the BioProjects PRJNA641166 and PRJNA803387 in the SRA repository of the National Centre for Biotechnology Information (NCBI).</p>
<p id="Par34">All experiments were performed in Lennox lysogeny broth (LB), supplemented with 15 g/L of agar when indicated. The different antibiotics used throughout the study were amoxicillin / clavulanic acid at a proportion of 5 mg amoxicillin / 1 mg clavulanic acid, ertapenem, and apramycin. 50 µM of 2,4-diacetylphloroglucinol (DAPG) was used for induction of dCas9.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Calculation of pOXA-48 fitness effects in different clinical strains</h3>
<p id="Par35">The fitness effect associated with pOXA-48 in each strain was calculated by comparing differences in the Area Under the Curve (AUC) of isogenic plasmid-carrying and plasmid-free clones, as described in DelaFuente et al.<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Raw data can be found in the supplementary materials (Supplementary Data <a href="#MOESM6" class="usa-link">4</a>).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Designing the OXAlib library</h3>
<p id="Par36">dCas9 is reprogrammed to target a specific position using guide RNAs (sgRNAs), the design of which needs to follow a specific set of rules in bacteria<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. (i) First, it has been shown that the specific sequence of the sgRNA has an effect on the activity of dCas9. (ii) Due to design limitations intrinsic to the CRISPRi technique, sgRNAs targeting coding regions have to be limited to the coding strand. The rules for efficiently targeting non-coding regions of DNA are not known, and therefore sgRNAs targeting those regions should be designed to target both strands of DNA. (iii) If dCas9 is directed to a gene within an operon, this binding will have a polar effect on the downstream genes. (iv) It is also important to avoid potential off-targets resulting from a permissiveness of dCas9 to bind to similar regions of DNA to bona fide targets.</p>
<p id="Par37">The OXAlib library was designed using the CRISPR@Pasteur design tool (<a href="https://crispr-browser.pasteur.cloud/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://crispr-browser.pasteur.cloud/</a>) which is trained on <em>E. coli</em> activity data, and which has been successfully used to design such libraries in different species of Enterobacterales<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The tool was adapted to our setup (looking for targets on a plasmid and for off-targets on a set of genomes - see Code Availability). Using this tool, the 5 best-performing sgRNAs were selected for each gene in pOXA-48_K8 (called pOXA-48 in this paper). sgRNAs targeting both DNA strands were also designed against intergenic regions of the plasmid (up to 5 targeting each strand, depending on the size of the intergenic region). The logic behind targeting both strands in intergenic regions is to attempt to capture effects of targeting any possible genetic elements present in the regions. 20 non-targeting control sgRNAs were also designed, amounting to a total of 568 sgRNAs (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>).</p>
<p id="Par38">Guides were selected with the previously described sorting strategy and ordered for synthesis as ssDNA oligos from Twist Bioscience. The sequence of each individual oligo is as follows: 5’-ACAGCTAGCTCAGTCCTAGGTATAATACTAGT-(20nt guide sequence)-GTTTTAGAGCTAGAAATAGCAAGTTAAAATAAGGCTAGT-3’.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Cloning the OXAlib library</h3>
<p id="Par39">The ssDNA oligo pool was amplified using primers AC74 and AC75 (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>) for subsequent cloning into pFR56apm<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. pFR56apm is based on pFR56<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>, with an apramycin resistance gene as a selection marker. This antibiotic permits selection of the vector in all the different clinical strains (which due to their plasmidsome are naturally resistant to several antibiotics). Amplification was performed using New England Biolabs’ Phusion High-Fidelity DNA polymerase, with the HF buffer, DMSO and the recommended protocol. To minimise PCR amplification biases of specific oligos in the pool, the reaction was limited to 9 cycles (1 cycle for generating dsDNA oligos and 8 cycles of amplification), and performed with 1 ng of template and 50 pmol of each primer. The resulting product was gel purified. The pFR56apm vector was digested with <em>BsaI</em> and gel purified. The OXAlib plasmid library was assembled using the Gibson Assembly method<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>, using New England Biolabs’ NEBuilder HiFi DNA Assembly, 200 ng of total DNA and a ratio of 1:5 vector:insert. Cloning was performed in <em>E. coli</em> MG1655. Quality control of the final OXAlib library can be found in the Supplementary Methods.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Conjugating the OXAlib library into clinical strains</h3>
<p id="Par40">pFR56apm is a mobilisable plasmid - meaning it can be conjugated to compatible hosts, but requires expression of the conjugation machinery in <em>trans</em>. The OXAlib library was minipreped from <em>E. coli</em> MG1655 and transformed (with a coverage &gt; 1000x of OXAlib) into <em>E. coli</em> ß3914ø + pTA-MOB<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. This strain is auxotrophic for diaminopimelic acid (DAP), allowing for its counter-selection as a donor strain.</p>
<p id="Par41">Conjugation of the OXAlib from ß3914ø to the different strains was performed by mixing 10<sup>8</sup> donor cells with 10<sup>8</sup> of recipient cells (ensuring a coverage &gt; 1000x of OXAlib) for 3 h. After conjugation, the cells were selected in apramycin (50 µg/mL) and let grow overnight at room temperature. Absence of DAP counter-selects against ß3914ø. Conjugation efficiency was assessed using serial dilutions and a good coverage of the OXAlib library achieved for each strain (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>).</p></section><section id="Sec16"><h3 class="pmc_sec_title">CRISPRi screens</h3>
<section id="Sec17"><h4 class="pmc_sec_title">Experiment</h4>
<p id="Par42">First of all, it is crucial when working with CRISPRi screens to ensure that each step guarantees a good coverage of the library (<em>i.e</em>., that no individual guides are lost from the population due to over-dilution). To ensure this, we always work with an excess of cells of at least 1000x the size of OXAlib.</p>
<p id="Par43">Strains carrying the OXAlib library were diluted 1:1000 in fresh LB / LB supplemented with ertapenem (subinhibitory concentrations of the antibiotics were used, Supplementary Methods, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). After 1 h, DAPG (50 µM) was added to the cultures for dCas9 induction. Every 8 h, a 1:1000 dilution was performed in fresh LB + DAPG / LB + DAPG + ertapenem. Prior to each dilution, OD was measured (to calculate number of generations). At 24, 48 and 72 h, apart from the dilution and OD measurement, a miniprep was performed using 10 mL of culture. The resulting pool of plasmids correspond to the timepoints at ~25, ~55 and ~85 generations. All screens were performed in triplicate (3 biological replicates), in aerobic conditions, at 37 °C, with shaking (250 r.p.m.).</p>
<p id="Par44">Simultaneously, to assess whether the CRISPRi machinery encoded in pFR56apm is still active after 85 generations of growth, we grew the different strains harbouring pOXA-48::sfGFP and pFR56apm loaded with an sgRNA targeting sfGFP for 85 generations with induction of the CRISPRi machinery (DAPG 50 µm), then diluted the cultures in LB / LB supplemented with DAPG, grew the resulting sub-cultures overnight and measured sfGFP expression using cytometry. Our results suggest that the CRISPRi machinery is fully active until the end of the CRISPRi experiments performed in this manuscript (SFig. <a href="#MOESM1" class="usa-link">16</a>).</p></section><section id="Sec18"><h4 class="pmc_sec_title">Illumina sample preparation and sequencing strategy</h4>
<p id="Par45">Library sequencing was performed as previously described<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, with some small adaptations. The PCR primers used for the nested PCR can be found in the supplementary materials (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). To maintain good coverage of the OXAlib library, 100 ng of plasmid miniprep were used as template for each reaction. To minimise PCR biases, 19 cycles were used in the 1st PCR, and 13 cycles on the 2nd. Each replicate was given a different i5-i7 index combination, achieved by amplification with specific pairs of primers. Lastly, Illumina sequencing quality depends strongly on the variability of the regions sequenced. To bypass variability issues arising from amplicons sharing a large 5’ identical sequence, 3 different staggered primers were used in the 1st PCR.</p>
<p id="Par46">Each sample was quantified using a bioanalyzer. 150 ng of each sample were pooled together and sequenced using the standard protocol of a NextSeq 500 benchtop sequencer (Illumina).</p></section></section><section id="Sec19"><h3 class="pmc_sec_title">Data analysis</h3>
<section id="Sec20"><h4 class="pmc_sec_title">Quality control of Illumina sequencing</h4>
<p id="Par47">First, a QC of the raw Illumina data sequences (55 bp amplicon sequencing) was performed using FastQC v.0.11.9<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> and MultiQC v.1.11<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. After confirming high quality per base score in the guide region (mean Phred score &gt; 30), the sequences were used in downstream analyses.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Guide count per sample</h4>
<p id="Par48">For getting each individual guide count in each sample, each fastq file was parsed using Bash (see Code Availability). Briefly, knowing the sequence of the promoter preceding the guides in each read, 15 bp of it were used to match the reads position in which the guide began, and the following 20 bp (i.e., the sgRNA) were extracted. For each sample, the number of times each individual guide appeared was calculated using Bash commands. The list of guides was sorted, each unique occurrence was counted, and the output was reformatted to save each strain guides count in an individual csv file (see Source Data). Within the individual csv counts files, the guides with less counts than 5 for timepoints 0 and with less counts than 2 for the rest of the timepoints were filtered. The results of all the samples were merged in the same table, and only those guides present in the plasmid library were kept. The sequencing coverage for each guide was calculated as the median of the counts of guides (1 sgRNA per read; avoiding outliers from guides counted in excess) multiplied by 5 (as there are 5 guides per gene). The counts for the three replicates per time point were merged after checking the reproducibility of the replicates in each condition (LB + DAPG or LB + DAPG+ertapenem).</p></section><section id="Sec22"><h4 class="pmc_sec_title">Gene score calculation</h4>
<p id="Par49">For calculating the gene scores (median log2 fold change by gene) a script which took the table with the replicates merged by sample, and calculated the log2FC for each gene, then centreing the values by removing the median of control guides, was written based on the code of Rousset et al.<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> (<a href="https://gitlab.pasteur.fr/dbikard/ecocg/-/blob/master/Notebook.ipynb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gitlab.pasteur.fr/dbikard/ecocg/-/blob/master/Notebook.ipynb</a>).</p></section><section id="Sec23"><h4 class="pmc_sec_title">Phylogenetic analysis</h4>
<p id="Par50">The phylogenetic distances between the different strains were calculated using Mashtree v.1.2.0<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> (NJ method) and visualised using the iTOL web server (<a href="https://itol.embl.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://itol.embl.de/</a>). Phylogenetic information was used to represent the strains clustered by relatedness in the heatmap (Fig. <a href="#Fig2" class="usa-link">2A</a>).</p></section><section id="Sec24"><h4 class="pmc_sec_title">Conservation of CRISPRi screening results in different strains</h4>
<p id="Par51">To analyse whether the effect of the OXAlib library on the strains tested was conserved, we studied the similarity between strains gene scores at the end of the screening. We calculated the correlation values (Pearson correlation coefficients) between the strains based on all the gene scores at the end of the screening for each condition by using the corrplot package v.0.95 in R. Additionally, we compared the genes whose silencing had the most notable effect in each strain. To do so, we listed the significant genes and intergenic regions for each strain and condition calculated from individual permutation tests for either LB + DAPG or LB + DAPG+ertapenem (see following Methods section). With these, we measured the Sørensen-Dice Similarity Coefficient (SDC) for all pairwise comparisons between strains of the same species screened in the same condition. The SDC was calculated using the following formula:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e1685" display="block"><mi>S</mi><mi>D</mi><mi>C</mi><mo>=</mo><mfrac><mrow><mn>2</mn><mi mathvariant="normal">⋅</mi><mo>∣</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>x</mi></mrow></msub><msub><mrow><mspace width="0.25em"></mspace><mo>∩</mo><mi>G</mi></mrow><mrow><mi>y</mi></mrow></msub><mo>∣</mo></mrow><mrow><mo>∣</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>x</mi></mrow></msub><mo>∣</mo><mo>+</mo><mo>∣</mo><msub><mrow><mi>G</mi></mrow><mrow><mi>y</mi></mrow></msub><mo>∣</mo></mrow></mfrac></math></td></tr></table>
<p>Where |<em>G</em><sub><em>x</em></sub>| and |<em>G</em><sub><em>y</em></sub>| are respectively the number of significant genes in strain X and Y (i.e. the number of elements compared from each strain), and |<em>G</em><sub><em>x</em></sub> ∩ <em>G</em><sub><em>y</em></sub>| is the number of common significant genes between X and Y strains in the given condition (i.e. full set of significant genes for both strains in either ertapenem or no antibiotics).</p></section><section id="Sec25"><h4 class="pmc_sec_title">Permutation tests for gene importance</h4>
<p id="Par52">For checking statistically significant differences from the score of each gene from the distribution of gene scores in the CRISPRi screen, a permutation test for all the strains considered together per each condition (LB + DAPG or LB + DAPG + ERTA) was carried out. Briefly, the mean of each gene score was calculated considering all the strains from the screens (<em>n</em> = 14). The difference of this value and the mean of the scores from the rest of the group was assessed as the statistics of interest in the permutation test. The same statistic was calculated for 100000 iterations (i.e., minimum computable <em>P</em> value = 0.00001) taking random samples (<em>n</em> = 14) in each iteration. This number of permutations was selected to get more precise probabilities, as the <em>P</em> values of some genes resulted marginally significant when performing the permutation tests with fewer repeats (1000−10000). Lastly, the number of times that the permuted statistic was more extreme than the mean of each gene (i.e., the <em>P</em> value) was calculated. The same test was conducted for each species, as well as for each strain, for both experimental conditions. Finally, the p-values were adjusted by FDR. To report the statistical results from the permutation tests as well as the CRISPRi screening results, Volcano Plots were represented to simultaneously show each gene <em>P</em> value and its score using the R package EnhancedVolcano v.1.18.0. As in this type of screen, most genes showed scores close to 0. Thus, a log2FC low threshold (i.e., +−0.5) was set, close to the centre of the distribution. For the <em>P</em> value threshold, the default one (10e-06; i.e., <em>P</em> value &lt; 0.05) was selected.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Ridge regression model for plasmid cost</h4>
<p id="Par53">To simultaneously consider the contribution of multiple genes to pOXA-48 cost, a multiple regression model with Ridge regularisation was developed using the glmnet v.4.1-7 R package separately for both <em>E. coli</em> and <em>K. pneumoniae</em>. First, feature selection of all the genes and intergenic regions tested was performed, as there were 8 observations of plasmid cost for <em>K. pneumoniae</em> and 6 observations of plasmid cost for <em>E. coli</em> (n=strains), and 106 explanatory variables (<em>n</em>=genes &amp; intergenic regions included in CRISPRi screen). Then, only those genes with significant <em>P</em> values in the permutation test performed for each of the species (i.e., expected to be more informative) in absence of antibiotics during the CRISPRi screen were included in the model. From these, intergenic regions were kept out, as their effect introduced multicollinearity issues by showing similar behaviour as their surrounding genes in the CRISPRi screening (e.g., <em>bla</em><sub>OXA-48</sub> intergenic surrounding regions). Multicollinearity between explanatory variables was assessed after feature selection using the R package corrplot v.0.95. Due to the limited number of observations for each species, the models were trained using Leave-One-Out Cross Validation. Moreover, to include all the genes resulting from the feature selection process, Ridge regularisation was selected, as the regression coefficients are not shrunk to 0. Optimal parameters from the trained models were selected to then build the final models. The ridge coefficients for each gene, and the R-squared for each species were calculated from the final models tuned with the optimal parameters (lambda [λ]; SFig. <a href="#MOESM1" class="usa-link">8</a>).</p></section><section id="Sec27"><h4 class="pmc_sec_title">Plasmid cost based on guide distribution</h4>
<p id="Par54">To infer the plasmid cost in each strain, a proxy based on the general effect of blocking the plasmid genes was employed. The effect of each plasmid gene on bacterial fitness is expected to be reflected on the CRISPRi screen in absence of antibiotics, as guides which block costly genes that cause deleterious effects are expected to show higher scores, and vice versa. Then, if the plasmid is highly costly in a strain, the general effect of blocking its genes should reflect this. Thus, the general effects of blocking the plasmid genes were correlated with the relative fitness of plasmid-carrying strains. The difference between the mean effect of blocking pOXA-48 genes (mean CRISPRi gene scores for each strain) and the mean effect of the control guides, which are supposed to have no effect, was calculated. Then, these values were correlated (Spearman method) with the fitness effect of pOXA-48 in each strain.</p></section></section><section id="Sec28"><h3 class="pmc_sec_title">Validation of CRISPRi screen’s results</h3>
<section id="Sec29"><h4 class="pmc_sec_title">Strain selection</h4>
<p id="Par55">From the initial set of strains used in this study, two <em>E. coli</em> (C288, EC22) and three <em>K. pneumoniae strains</em> (KPN10, KPN11 and KPN18) were chosen for validation of results based on the overall strongest fold-changes observed.</p></section><section id="Sec30"><h4 class="pmc_sec_title">Cloning of individual sgRNAs in pFR56apm and conjugation into clinical strains</h4>
<p id="Par56">Cloning of individual sgRNAs (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>) in pFR56apm was performed using Golden Gate assembly as in Rousset et al.<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. All cloning was performed in <em>E. coli</em> MG1655 and followed by conjugation into the clinical strains as described previously.</p></section><section id="Sec31"><h4 class="pmc_sec_title">bla<sub>OXA-48</sub> associated fitness effects</h4>
<p id="Par57">To validate the results associated with silencing the gene <em>bla</em><sub>OXA-48</sub>, growth curves were performed to study plasmid fitness effects as explained before in this method section. This time, a second set of strains was selected from the RGNOSIS collection, to avoid overfitting to the original strains used in the screen. These strains were once again selected based on the fitness effects associated with the plasmid, as described in Fernández-Calvet et al.<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. A variant of pOXA-48 lacking a complete <em>bla</em><sub>OXA-48</sub> was identified previously in the lab<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. This variant was conjugated on the cured version of the strain, obtaining three isogenic set-ups for each strain: the wild-type carrying pOXA-48, a pOXA-48-free cured version, and a pOXA-48∆<em>bla</em><sub>OXA-48</sub> transconjugant.</p></section><section id="Sec32"><h4 class="pmc_sec_title">Plasmid stability</h4>
<p id="Par58">Plasmid stability of pOXA-48 and pOXA-48 versions was measured by growing cells carrying the plasmid in LB over subsequent periods of 24 h. After each cycle, cells were plated in LB, guaranteeing the isolation of individual colonies. Cells were then streaked in presence and absence of ertapenem using replica plating. Percentages of plasmid loss / plasmid presence were calculated by comparing growth in presence and absence of selection. Each experiment was performed in triplicate. Number of 24 h cycles used in each experiment are indicated throughout the text. To statistically analyse whether the stability of pOXA-48 was significantly affected in at least one strain when silencing individual genes (Fig. <a href="#Fig3" class="usa-link">3B</a>), we performed a one-tailed z-test for each guide. Briefly, we calculated the maximum deviation from the control group (guide “control”) per individual guide, from which we inferred their Z-score. Finally, we corrected the <em>P</em> values by Bonferroni adjustment (<em>n</em> = 8).</p></section><section id="Sec33"><h4 class="pmc_sec_title">qPCR for PCN calculation</h4>
<p id="Par59">Changes in pOXA-48’s PCN were calculated for the different clinical strains carrying pFR56apm programmed to silence different plasmid genes. In order to do so, the different strains were grown in triplicates in LB supplemented with apramycin 50 µg/mL and DAPG 50 µM for 24 h. 50 µL samples were taken of each replicate, centrifuged to collect the pellet and boiled for 10 min. 2 µL of each boiling reaction were used per qPCR reaction, using the NZYSupreme qPCR Green Master Mix (2X), ROX plus kit from NZYTech. Targeted plasmid and chromosome genes were <em>bla</em><sub>OXA-48</sub> (amplicon size 100 bp; efficiency 107,47%) and <em>dnaE</em> (chromosomal gene with 1 copy, amplicon size 200 bp, efficiency 105.79%), respectively. Efficiency was calculated (taking into account amplicon size difference) as in San Millán et al<em>.</em><sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. The amplification conditions were: 5 min of initial denaturation (95 °C), followed by 30 cycles of 15 s denaturation, 30 s annealing (55 °C) and 30 s extension (60 °C). The relative PCN was calculated as in San Millán et al<em>.</em><sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Raw data for qPCR can be found in the supplementary data (Supplementary Data <a href="#MOESM6" class="usa-link">4</a>).</p></section><section id="Sec34"><h4 class="pmc_sec_title">Lambda-Red recombination for construction of deletion mutants</h4>
<p id="Par60">A version of pOXA-48 lacking <em>pemK</em> was built using lambda-red recombination<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. The recombination cassette with extremes matching the beginning and end of <em>pemK</em>, and a kanamycin gene flanked by FRT sides was amplified from pKD4, as described in Datsenko et al.<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Primers used for construction of the cassette and validation of the construct can be found in the supplementary data (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>).</p></section></section><section id="Sec35"><h3 class="pmc_sec_title">Statistics &amp; reproducibility</h3>
<p id="Par61">All statistical analyses were performed using RStudio v.2024.12.1 + 563 and R v.4.3. Parametric or non-parametric tests were used depending on the structure of the data (see each specific subsection analysis). Tests were controlled for multiple comparison testing by applying FDR or Bonferroni adjustment (see each specific subsection analysis). To analyze correlation data, we used Pearson correlation for normally distributed data. No statistical method was used to predetermine sample size. No data were excluded from the analyses. The experiments were not randomised. The Investigators were not blinded to allocation during experiments and outcome assessment.</p></section><section id="Sec36"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par62">Further information on research design is available in the <a href="#MOESM7" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec37"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (4.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63082_MOESM2_ESM.pdf</a><sup> (51.6KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (221.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (187KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (424.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (3.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (130.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (975.8KB, pdf) </sup>
</div></div></section></section><section id="Sec38"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (450.3KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the R-GNOSIS group, the volunteers and the medical staff from the Hospital Universitario Ramón y Cajal (Madrid, Spain) involved in the sample isolation process. We thank David Bikard and the Institut Pasteur for plasmid pFR56. We thank the <em>Unidad de Genómica del Parque Científico de Madrid</em> for helping with the Illumina sequencing. We thank Sarah Bigot for the kind gift of plasmid pOXA-48A_nuc::sfGFP. This work was supported by the European Research Council (ERC) under the European Union’s Horizon Europe research and innovation programme (ERC-2022-CoG Project 101086992 – PLAS-FIGHTER) and by MCIN/AEI/10.13039/501100011033 and the European Union NextGenerationEU/PRTR (Project PCI2021- 122062- 2 A). A.C.V. was funded by an EMBO postdoctoral fellowship (ALTF 322-2022). We acknowledge financial support from the Spanish State Research Agency, AEI/10.13039/501100011033, through the “Severo Ochoa” Programme for Centres of Excellence in R&amp;D [SEV-2013-0347, SEV-2017-0712, CEX2023-001386-S].</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>A.C.V., J.S.D. and A.S.M. conceptualised the study. A.C.V., J.S.D., A.BM. and A.S.M. designed the methodology. J.S.D. performed the bioinformatic analyses. A.CV. A-BM. and C.C.R. performed the experiments. A.C.V., J.S.D. and A.S.M. contributed to data analysis. A.S.M. was responsible for funding acquisition and supervision. A.C.V., J.S.D. and A.S.M. wrote the original draft and undertook the reviewing and editing process. All authors supervised and approved the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par63"><em>Nature Communications</em> thanks Enas Newire, and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The sequences generated during this project are available under the BioProject ID <span class="text-underline">PRJNA1203587</span> in the SRA repository of the National Centre for Biotechnology Information (NCBI). Source data for all figures is provided with this paper (see Supplementary Data <a href="#MOESM5" class="usa-link">3</a> and <a href="#MOESM6" class="usa-link">4</a>, Source Data, Supplementary Information). <a href="#Sec38" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All the code developed for the analyses included in this work can be found at the <span class="text-underline">Github repository</span><sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par64">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Alicia Calvo-Villamañán, Jorge Sastre-Dominguez.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Alicia Calvo-Villamañán, Email: aliciacalvovillamanan@gmail.com.</p>
<p>Álvaro San Millan, Email: asanmillan@cnb.csic.es.</p></section><section id="sec39"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63082-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Review on Antimicrobial Resistance. <em>Antimicrobial Resistance - Tackling a Crisis for the Health and Wealth of Nations</em>. (2014).</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Naghavi, M. et al. Global burden of bacterial antimicrobial resistance 1990–2021: a systematic analysis with forecasts to 2050. <em> Lancet</em><strong>404</strong>, 1199–1226 (2024).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(24)01867-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11718157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39299261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naghavi,%20M.%20et%20al.%20Global%20burden%20of%20bacterial%20antimicrobial%20resistance%201990%E2%80%932021:%20a%20systematic%20analysis%20with%20forecasts%20to%202050.%20Lancet404,%201199%E2%80%931226%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ho, C. S. et al. Antimicrobial resistance: a concise update. <em>Lancet Microbe</em><strong>0</strong>, 100947 (2024).</cite> [<a href="https://doi.org/10.1016/j.lanmic.2024.07.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39305919/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>World Bank. <em>Drug-Resistant Infections: A Threat to Our Economic Future</em>. (2017).</cite>
</li>
<li id="CR5">
<span class="label">5.</span><cite>Baquero, F. et al. Evolutionary Pathways and Trajectories in Antibiotic Resistance. <em>Clin. Microbiol. Rev</em>. 10.1128/CMR.00050-19 (2021).</cite> [<a href="https://doi.org/10.1128/CMR.00050-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8404696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34190572/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Song, S. et al. Adaptive evolution of carbapenem-resistant hypervirulent Klebsiella pneumoniae in the urinary tract of a single patient. <em>Proc. Natl. Acad. Sci. USA.</em><strong>121</strong>, e2400446121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2400446121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11363291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39150777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20S.%20et%20al.%20Adaptive%20evolution%20of%20carbapenem-resistant%20hypervirulent%20Klebsiella%20pneumoniae%20in%20the%20urinary%20tract%20of%20a%20single%20patient.%20Proc.%20Natl.%20Acad.%20Sci.%20USA.121,%20e2400446121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Dorado-Morales, P., Garcillán-Barcia, M. P., Lasa, I. &amp; Solano, C. Fitness cost evolution of natural plasmids of staphylococcus aureus. <em>mBio</em><strong>12</strong>, 10.1128/mbio.03094-20 (2021).</cite> [<a href="https://doi.org/10.1128/mBio.03094-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8545097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33622733/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hernando-Amado, S. et al. Ciprofloxacin resistance rapidly declines in nfxB defective clinical strains of Pseudomonas aeruginosa. <em>Nat. Commun.</em><strong>16</strong>, 4992 (2025).
</cite> [<a href="https://doi.org/10.1038/s41467-025-60330-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12137949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40467543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hernando-Amado,%20S.%20et%20al.%20Ciprofloxacin%20resistance%20rapidly%20declines%20in%20nfxB%20defective%20clinical%20strains%20of%20Pseudomonas%20aeruginosa.%20Nat.%20Commun.16,%204992%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Souque, C., Ojeda, I. G. &amp; Baym, M. From petri dishes to patients to populations: scales and evolutionary mechanisms driving antibiotic resistance. <em>Annu. Rev. Microbiol.</em><strong>78</strong>, 361–382 (2024).
</cite> [<a href="https://doi.org/10.1146/annurev-micro-041522-102707" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39141706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Souque,%20C.,%20Ojeda,%20I.%20G.%20&amp;%20Baym,%20M.%20From%20petri%20dishes%20to%20patients%20to%20populations:%20scales%20and%20evolutionary%20mechanisms%20driving%20antibiotic%20resistance.%20Annu.%20Rev.%20Microbiol.78,%20361%E2%80%93382%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Smith, W. P. J., Wucher, B. R., Nadell, C. D. &amp; Foster, K. R. Bacterial defences: mechanisms, evolution and antimicrobial resistance. <em>Nat. Rev. Microbiol.</em><strong>21</strong>, 519–534 (2023).
</cite> [<a href="https://doi.org/10.1038/s41579-023-00877-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37095190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20W.%20P.%20J.,%20Wucher,%20B.%20R.,%20Nadell,%20C.%20D.%20&amp;%20Foster,%20K.%20R.%20Bacterial%20defences:%20mechanisms,%20evolution%20and%20antimicrobial%20resistance.%20Nat.%20Rev.%20Microbiol.21,%20519%E2%80%93534%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Alonso-del et al. Antimicrobial resistance level and conjugation permissiveness shape plasmid distribution in clinical enterobacteria. <em>Proc. Natl. Acad. Sci.</em><strong>120</strong>, e2314135120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2314135120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10741383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38096417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alonso-del%20et%20al.%20Antimicrobial%20resistance%20level%20and%20conjugation%20permissiveness%20shape%20plasmid%20distribution%20in%20clinical%20enterobacteria.%20Proc.%20Natl.%20Acad.%20Sci.120,%20e2314135120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Smillie, C., Garcillán-Barcia, M. P., Francia, M. V., Rocha, E. P. C. &amp; de la Cruz, F. Mobility of plasmids. <em>Microbiol. Mol. Biol. Rev.</em><strong>74</strong>, 434–452 (2010).
</cite> [<a href="https://doi.org/10.1128/MMBR.00020-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2937521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20805406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smillie,%20C.,%20Garcill%C3%A1n-Barcia,%20M.%20P.,%20Francia,%20M.%20V.,%20Rocha,%20E.%20P.%20C.%20&amp;%20de%20la%20Cruz,%20F.%20Mobility%20of%20plasmids.%20Microbiol.%20Mol.%20Biol.%20Rev.74,%20434%E2%80%93452%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>San Millan, A. &amp; MacLean, R. C. Fitness costs of plasmids: a limit to plasmid transmission. <em>Microbiol. Spectr</em>. <strong>5</strong>, 10 (2017).</cite> [<a href="https://doi.org/10.1128/microbiolspec.mtbp-0016-2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11687550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28944751/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Porse, A., Schønning, K., Munck, C. &amp; Sommer, M. O. A. Survival and evolution of a large multidrug resistance plasmid in new clinical bacterial hosts. <em>Mol. Biol. Evol.</em><strong>33</strong>, 2860–2873 (2016).
</cite> [<a href="https://doi.org/10.1093/molbev/msw163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5062321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27501945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Porse,%20A.,%20Sch%C3%B8nning,%20K.,%20Munck,%20C.%20&amp;%20Sommer,%20M.%20O.%20A.%20Survival%20and%20evolution%20of%20a%20large%20multidrug%20resistance%20plasmid%20in%20new%20clinical%20bacterial%20hosts.%20Mol.%20Biol.%20Evol.33,%202860%E2%80%932873%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Hall, J. P. J. et al. Plasmid fitness costs are caused by specific genetic conflicts enabling resolution by compensatory mutation. <em>PLOS Biol.</em><strong>19</strong>, e3001225 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3001225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8544851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34644303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hall,%20J.%20P.%20J.%20et%20al.%20Plasmid%20fitness%20costs%20are%20caused%20by%20specific%20genetic%20conflicts%20enabling%20resolution%20by%20compensatory%20mutation.%20PLOS%20Biol.19,%20e3001225%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Fernández-Calvet, A. et al. The distribution of fitness effects of plasmid pOXA-48 in clinical enterobacteria. <em>Microbiology</em><strong>169</strong>, 001369 (2023).
</cite> [<a href="https://doi.org/10.1099/mic.0.001369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10433420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37505800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fern%C3%A1ndez-Calvet,%20A.%20et%20al.%20The%20distribution%20of%20fitness%20effects%20of%20plasmid%20pOXA-48%20in%20clinical%20enterobacteria.%20Microbiology169,%20001369%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Baltrus, D. A. Exploring the costs of horizontal gene transfer. <em>Trends Ecol. Evol.</em><strong>28</strong>, 489–495 (2013).
</cite> [<a href="https://doi.org/10.1016/j.tree.2013.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23706556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baltrus,%20D.%20A.%20Exploring%20the%20costs%20of%20horizontal%20gene%20transfer.%20Trends%20Ecol.%20Evol.28,%20489%E2%80%93495%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Billane, K., Harrison, E., Cameron, D. &amp; Brockhurst, M. A. Why do plasmids manipulate the expression of bacterial phenotypes?. <em>Philos. Trans. R. Soc. B Biol. Sci.</em><strong>377</strong>, 20200461 (2021).</cite> [<a href="https://doi.org/10.1098/rstb.2020.0461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8628079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34839708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Billane,%20K.,%20Harrison,%20E.,%20Cameron,%20D.%20&amp;%20Brockhurst,%20M.%20A.%20Why%20do%20plasmids%20manipulate%20the%20expression%20of%20bacterial%20phenotypes?.%20Philos.%20Trans.%20R.%20Soc.%20B%20Biol.%20Sci.377,%2020200461%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Hall, R. J., Snaith, A. E., Thomas, M. J. N., Brockhurst, M. A. &amp; McNally, A. Multidrug resistance plasmids commonly reprogram the expression of metabolic genes in Escherichia coli. <em>mSystems</em><strong>9</strong>, e01193–23 (2024).
</cite> [<a href="https://doi.org/10.1128/msystems.01193-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10949484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38376169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hall,%20R.%20J.,%20Snaith,%20A.%20E.,%20Thomas,%20M.%20J.%20N.,%20Brockhurst,%20M.%20A.%20&amp;%20McNally,%20A.%20Multidrug%20resistance%20plasmids%20commonly%20reprogram%20the%20expression%20of%20metabolic%20genes%20in%20Escherichia%20coli.%20mSystems9,%20e01193%E2%80%9323%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Thompson, C. M. A. et al. Plasmids manipulate bacterial behaviour through translational regulatory crosstalk. <em>PLOS Biol.</em><strong>21</strong>, e3001988 (2023).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3001988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9928087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36787297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thompson,%20C.%20M.%20A.%20et%20al.%20Plasmids%20manipulate%20bacterial%20behaviour%20through%20translational%20regulatory%20crosstalk.%20PLOS%20Biol.21,%20e3001988%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Prensky, H., Gomez-Simmons, A., Uhlemann, A.-C. &amp; Lopatkin, A. Conjugation dynamics depend on both the plasmid acquisition cost and the fitness cost. <em>Mol. Syst. Biol.</em><strong>17</strong>, e9913 (2021).
</cite> [<a href="https://doi.org/10.15252/msb.20209913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7919528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33646643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prensky,%20H.,%20Gomez-Simmons,%20A.,%20Uhlemann,%20A.-C.%20&amp;%20Lopatkin,%20A.%20Conjugation%20dynamics%20depend%20on%20both%20the%20plasmid%20acquisition%20cost%20and%20the%20fitness%20cost.%20Mol.%20Syst.%20Biol.17,%20e9913%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Petrova, V., Chitteni-Pattu, S., Drees, J. C., Inman, R. B. &amp; Cox, M. M. An SOS inhibitor that binds to free RecA protein: the PsiB protein. <em>Mol. Cell</em><strong>36</strong>, 121–130 (2009).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2009.07.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2761248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19818715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petrova,%20V.,%20Chitteni-Pattu,%20S.,%20Drees,%20J.%20C.,%20Inman,%20R.%20B.%20&amp;%20Cox,%20M.%20M.%20An%20SOS%20inhibitor%20that%20binds%20to%20free%20RecA%20protein:%20the%20PsiB%20protein.%20Mol.%20Cell36,%20121%E2%80%93130%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Park, C. &amp; Zhang, J. High expression hampers horizontal gene transfer. <em>Genome Biol. Evol.</em><strong>4</strong>, 523–532 (2012).
</cite> [<a href="https://doi.org/10.1093/gbe/evs030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3342876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22436996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park,%20C.%20&amp;%20Zhang,%20J.%20High%20expression%20hampers%20horizontal%20gene%20transfer.%20Genome%20Biol.%20Evol.4,%20523%E2%80%93532%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Koraimann, G. &amp; Wagner, M. A. Social behavior and decision making in bacterial conjugation. <em>Front. Cell. Infect. Microbiol</em>. <strong>4</strong>, 54 (2014).</cite> [<a href="https://doi.org/10.3389/fcimb.2014.00054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4010749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24809026/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Toribio-Celestino, L. &amp; Millan, A. S. Plasmid–bacteria associations in the clinical context. <em>Trends Microbiol</em>. <strong>0</strong>, 10.1016/j.tim.2025.04.011 (2025).</cite> [<a href="https://doi.org/10.1016/j.tim.2025.04.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40374465/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Toribio-Celestino, L. et al. A plasmid-chromosome crosstalk in multidrug resistant enterobacteria. <em>Nat. Commun.</em><strong>15</strong>, 10859 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-55169-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11686079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39738078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Toribio-Celestino,%20L.%20et%20al.%20A%20plasmid-chromosome%20crosstalk%20in%20multidrug%20resistant%20enterobacteria.%20Nat.%20Commun.15,%2010859%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Cohen, O., Gophna, U. &amp; Pupko, T. The complexity hypothesis revisited: connectivity rather than function constitutes a barrier to horizontal gene transfer. <em>Mol. Biol. Evol.</em><strong>28</strong>, 1481–1489 (2011).
</cite> [<a href="https://doi.org/10.1093/molbev/msq333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21149642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cohen,%20O.,%20Gophna,%20U.%20&amp;%20Pupko,%20T.%20The%20complexity%20hypothesis%20revisited:%20connectivity%20rather%20than%20function%20constitutes%20a%20barrier%20to%20horizontal%20gene%20transfer.%20Mol.%20Biol.%20Evol.28,%201481%E2%80%931489%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Turner, P. E., Cooper, V. S. &amp; Lenski, R. E. Tradeoff between horizontal and vertical modes of transmission in bacterial plasmids. <em>Evol. Int. J. Org. Evol.</em><strong>52</strong>, 315–329 (1998).</cite> [<a href="https://doi.org/10.1111/j.1558-5646.1998.tb01634.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28568337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Turner,%20P.%20E.,%20Cooper,%20V.%20S.%20&amp;%20Lenski,%20R.%20E.%20Tradeoff%20between%20horizontal%20and%20vertical%20modes%20of%20transmission%20in%20bacterial%20plasmids.%20Evol.%20Int.%20J.%20Org.%20Evol.52,%20315%E2%80%93329%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Vogwill, T. &amp; MacLean, R. C. The genetic basis of the fitness costs of antimicrobial resistance: a meta-analysis approach. <em>Evol. Appl.</em><strong>8</strong>, 284–295 (2015).
</cite> [<a href="https://doi.org/10.1111/eva.12202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4380922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25861386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vogwill,%20T.%20&amp;%20MacLean,%20R.%20C.%20The%20genetic%20basis%20of%20the%20fitness%20costs%20of%20antimicrobial%20resistance:%20a%20meta-analysis%20approach.%20Evol.%20Appl.8,%20284%E2%80%93295%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Rajer, F. &amp; Sandegren, L. The role of antibiotic resistance genes in the fitness cost of multiresistance plasmids. <em>mBio</em>10.1128/mbio.03552-21 (2022).</cite> [<a href="https://doi.org/10.1128/mbio.03552-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8764527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35038907/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>DelaFuente, J. et al. Within-patient evolution of plasmid-mediated antimicrobial resistance. <em>Nat. Ecol. Evol.</em><strong>6</strong>, 1980–1991 (2022).
</cite> [<a href="https://doi.org/10.1038/s41559-022-01908-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7613874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36303001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?DelaFuente,%20J.%20et%20al.%20Within-patient%20evolution%20of%20plasmid-mediated%20antimicrobial%20resistance.%20Nat.%20Ecol.%20Evol.6,%201980%E2%80%931991%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Poirel, L., Bonnin, R. A. &amp; Nordmann, P. Genetic features of the widespread plasmid coding for the carbapenemase OXA-48. <em>Antimicrob. Agents Chemother.</em><strong>56</strong>, 559–562 (2012).
</cite> [<a href="https://doi.org/10.1128/AAC.05289-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3256075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22083465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poirel,%20L.,%20Bonnin,%20R.%20A.%20&amp;%20Nordmann,%20P.%20Genetic%20features%20of%20the%20widespread%20plasmid%20coding%20for%20the%20carbapenemase%20OXA-48.%20Antimicrob.%20Agents%20Chemother.56,%20559%E2%80%93562%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Rozwandowicz, M. et al. Plasmids carrying antimicrobial resistance genes in Enterobacteriaceae. <em>J. Antimicrob. Chemother.</em><strong>73</strong>, 1121–1137 (2018).
</cite> [<a href="https://doi.org/10.1093/jac/dkx488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29370371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rozwandowicz,%20M.%20et%20al.%20Plasmids%20carrying%20antimicrobial%20resistance%20genes%20in%20Enterobacteriaceae.%20J.%20Antimicrob.%20Chemother.73,%201121%E2%80%931137%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>David, S. et al. Epidemic of carbapenem-resistant Klebsiella pneumoniae in Europe is driven by nosocomial spread. <em>Nat. Microbiol.</em><strong>4</strong>, 1919–1929 (2019).
</cite> [<a href="https://doi.org/10.1038/s41564-019-0492-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7244338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31358985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?David,%20S.%20et%20al.%20Epidemic%20of%20carbapenem-resistant%20Klebsiella%20pneumoniae%20in%20Europe%20is%20driven%20by%20nosocomial%20spread.%20Nat.%20Microbiol.4,%201919%E2%80%931929%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>León-Sampedro, R. et al. Pervasive transmission of a carbapenem resistance plasmid in the gut microbiota of hospitalized patients. <em>Nat. Microbiol.</em><strong>6</strong>, 606–616 (2021).
</cite> [<a href="https://doi.org/10.1038/s41564-021-00879-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33782584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%C3%B3n-Sampedro,%20R.%20et%20al.%20Pervasive%20transmission%20of%20a%20carbapenem%20resistance%20plasmid%20in%20the%20gut%20microbiota%20of%20hospitalized%20patients.%20Nat.%20Microbiol.6,%20606%E2%80%93616%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Matsumura, Y., Peirano, G. &amp; Pitout, J. D. D. Complete genome sequence of Escherichia coli J53, an azide-resistant laboratory strain used for conjugation experiments. <em>Genome Announc</em>. 10.1128/genomea.00433-18 (2018).</cite> [<a href="https://doi.org/10.1128/genomeA.00433-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5968721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29798920/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Alonso-del et al. Variability of plasmid fitness effects contributes to plasmid persistence in bacterial communities. <em>Nat. Commun.</em><strong>12</strong>, 2653 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-22849-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8113577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33976161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alonso-del%20et%20al.%20Variability%20of%20plasmid%20fitness%20effects%20contributes%20to%20plasmid%20persistence%20in%20bacterial%20communities.%20Nat.%20Commun.12,%202653%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Cui, L. et al. A CRISPRi screen in E. coli reveals sequence-specific toxicity of dCas9. <em>Nat. Commun.</em><strong>9</strong>, 1–10 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-04209-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5954155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29765036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui,%20L.%20et%20al.%20A%20CRISPRi%20screen%20in%20E.%20coli%20reveals%20sequence-specific%20toxicity%20of%20dCas9.%20Nat.%20Commun.9,%201%E2%80%9310%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Reuter, A. et al. Targeted-antibacterial-plasmids (TAPs) combining conjugation and CRISPR/Cas systems achieve strain-specific antibacterial activity. <em>Nucleic Acids Res</em><strong>49</strong>, 3584–3598 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8034655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33660775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reuter,%20A.%20et%20al.%20Targeted-antibacterial-plasmids%20(TAPs)%20combining%20conjugation%20and%20CRISPR/Cas%20systems%20achieve%20strain-specific%20antibacterial%20activity.%20Nucleic%20Acids%20Res49,%203584%E2%80%933598%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Tsuchimoto, S. &amp; Ohtsubo, E. Autoregulatiou by cooperative binding of the PemI and PemK proteins to the promoter region of the pem operon. <em>Mol. Gen. Genet. MGG</em><strong>237</strong>, 81–88 (1993).
</cite> [<a href="https://doi.org/10.1007/BF00282787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8455570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsuchimoto,%20S.%20&amp;%20Ohtsubo,%20E.%20Autoregulatiou%20by%20cooperative%20binding%20of%20the%20PemI%20and%20PemK%20proteins%20to%20the%20promoter%20region%20of%20the%20pem%20operon.%20Mol.%20Gen.%20Genet.%20MGG237,%2081%E2%80%9388%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Dmowski, M., Gołębiewski, M. &amp; Kern-Zdanowicz, I. Characteristics of the conjugative transfer system of the IncM plasmid pCTX-M3 and Identification of its putative regulators. <em>J. Bacteriol</em>. 10.1128/jb.00234-18 (2018).</cite> [<a href="https://doi.org/10.1128/JB.00234-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6112013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29986941/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Baffert, Y. et al. Genetic determinants of pOXA-48 plasmid maintenance and propagation in Escherichia coli. Preprint at <em>bioRxiv</em>10.1101/2025.01.23.634512 (2025).</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Ma, T. et al. VirBR, a transcription regulator, promotes IncX3 plasmid transmission, and persistence of blaNDM-5 in zoonotic bacteria. <em>Nat. Commun.</em><strong>15</strong>, 5498 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-49800-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11214620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38944647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20T.%20et%20al.%20VirBR,%20a%20transcription%20regulator,%20promotes%20IncX3%20plasmid%20transmission,%20and%20persistence%20of%20blaNDM-5%20in%20zoonotic%20bacteria.%20Nat.%20Commun.15,%205498%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Benomar, S., Di Venanzio, G. &amp; Feldman, M. F. Plasmid-encoded H-NS controls extracellular matrix composition in a modern Acinetobacter baumannii urinary isolate. <em>J. Bacteriol</em>. 10.1128/jb.00277-21 (2021).</cite> [<a href="https://doi.org/10.1128/JB.00277-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8508102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34398664/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Tsuchimoto, S., Ohtsubo, H. &amp; Ohtsubo, E. Two genes, pemK and pemI, responsible for stable maintenance of resistance plasmid R100. <em>J. Bacteriol.</em><strong>170</strong>, 1461–1466 (1988).
</cite> [<a href="https://doi.org/10.1128/jb.170.4.1461-1466.1988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC210989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2832364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsuchimoto,%20S.,%20Ohtsubo,%20H.%20&amp;%20Ohtsubo,%20E.%20Two%20genes,%20pemK%20and%20pemI,%20responsible%20for%20stable%20maintenance%20of%20resistance%20plasmid%20R100.%20J.%20Bacteriol.170,%201461%E2%80%931466%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Vigouroux, A., Oldewurtel, E., Cui, L., Bikard, D. &amp; van Teeffelen, S. Tuning dCas9’s ability to block transcription enables robust, noiseless knockdown of bacterial genes. <em>Mol. Syst. Biol.</em><strong>14</strong>, e7899 (2018).
</cite> [<a href="https://doi.org/10.15252/msb.20177899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5842579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29519933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vigouroux,%20A.,%20Oldewurtel,%20E.,%20Cui,%20L.,%20Bikard,%20D.%20&amp;%20van%20Teeffelen,%20S.%20Tuning%20dCas9%E2%80%99s%20ability%20to%20block%20transcription%20enables%20robust,%20noiseless%20knockdown%20of%20bacterial%20genes.%20Mol.%20Syst.%20Biol.14,%20e7899%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Rousset, F. et al. Genome-wide CRISPR-dCas9 screens in E. coli identify essential genes and phage host factors. <em>PLOS Genet</em><strong>14</strong>, e1007749 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1007749" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6242692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30403660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rousset,%20F.%20et%20al.%20Genome-wide%20CRISPR-dCas9%20screens%20in%20E.%20coli%20identify%20essential%20genes%20and%20phage%20host%20factors.%20PLOS%20Genet14,%20e1007749%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>del Solar, G., Giraldo, R., Ruiz-Echevarría, M. J., Espinosa, M. &amp; Díaz-Orejas, R. Replication and control of circular bacterial plasmids. <em>Microbiol. Mol. Biol. Rev.</em><strong>62</strong>, 434–464 (1998).
</cite> [<a href="https://doi.org/10.1128/mmbr.62.2.434-464.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC98921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9618448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?del%20Solar,%20G.,%20Giraldo,%20R.,%20Ruiz-Echevarr%C3%ADa,%20M.%20J.,%20Espinosa,%20M.%20&amp;%20D%C3%ADaz-Orejas,%20R.%20Replication%20and%20control%20of%20circular%20bacterial%20plasmids.%20Microbiol.%20Mol.%20Biol.%20Rev.62,%20434%E2%80%93464%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Marciano, D. C., Karkouti, O. Y. &amp; Palzkill, T. A fitness cost associated with the antibiotic resistance enzyme SME-1 β-lactamase. <em>Genetics</em><strong>176</strong>, 2381 (2007).
</cite> [<a href="https://doi.org/10.1534/genetics.106.069443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17565956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marciano,%20D.%20C.,%20Karkouti,%20O.%20Y.%20&amp;%20Palzkill,%20T.%20A%20fitness%20cost%20associated%20with%20the%20antibiotic%20resistance%20enzyme%20SME-1%20%CE%B2-lactamase.%20Genetics176,%202381%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Socha, R. D., Chen, J. &amp; Tokuriki, N. The molecular mechanisms underlying hidden phenotypic variation among metallo-β-lactamases. <em>J. Mol. Biol.</em><strong>431</strong>, 1172–1185 (2019).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2019.01.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30769117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Socha,%20R.%20D.,%20Chen,%20J.%20&amp;%20Tokuriki,%20N.%20The%20molecular%20mechanisms%20underlying%20hidden%20phenotypic%20variation%20among%20metallo-%CE%B2-lactamases.%20J.%20Mol.%20Biol.431,%201172%E2%80%931185%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>López, C., Ayala, J. A., Bonomo, R. A., González, L. J. &amp; Vila, A. J. Protein determinants of dissemination and host specificity of metallo-β-lactamases. <em>Nat. Commun.</em><strong>10</strong>, 3617 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-11615-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6689000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31399590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez,%20C.,%20Ayala,%20J.%20A.,%20Bonomo,%20R.%20A.,%20Gonz%C3%A1lez,%20L.%20J.%20&amp;%20Vila,%20A.%20J.%20Protein%20determinants%20of%20dissemination%20and%20host%20specificity%20of%20metallo-%CE%B2-lactamases.%20Nat.%20Commun.10,%203617%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Fernández, A. et al. Expression of OXA-type and SFO-1 β-lactamases induces changes in peptidoglycan composition and affects bacterial fitness. <em>Antimicrob. Agents Chemother.</em><strong>56</strong>, 1877–1884 (2012).
</cite> [<a href="https://doi.org/10.1128/AAC.05402-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3318342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22290977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fern%C3%A1ndez,%20A.%20et%20al.%20Expression%20of%20OXA-type%20and%20SFO-1%20%CE%B2-lactamases%20induces%20changes%20in%20peptidoglycan%20composition%20and%20affects%20bacterial%20fitness.%20Antimicrob.%20Agents%20Chemother.56,%201877%E2%80%931884%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tu, M. M. et al. Exploiting the fitness cost of metallo-β-lactamase expression can overcome antibiotic resistance in bacterial pathogens. <em>Nat. Microbiol.</em><strong>10</strong>, 53–65 (2025).
</cite> [<a href="https://doi.org/10.1038/s41564-024-01883-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39747690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tu,%20M.%20M.%20et%20al.%20Exploiting%20the%20fitness%20cost%20of%20metallo-%CE%B2-lactamase%20expression%20can%20overcome%20antibiotic%20resistance%20in%20bacterial%20pathogens.%20Nat.%20Microbiol.10,%2053%E2%80%9365%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Herencias, C. et al. β-lactamase expression induces collateral sensitivity in Escherichia coli. <em>Nat. Commun.</em><strong>15</strong>, 4731 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-49122-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11148083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38830889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herencias,%20C.%20et%20al.%20%CE%B2-lactamase%20expression%20induces%20collateral%20sensitivity%20in%20Escherichia%20coli.%20Nat.%20Commun.15,%204731%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Furniss, R. C. D. et al. Breaking antimicrobial resistance by disrupting extracytoplasmic protein folding. <em>eLife</em><strong>11</strong>, e57974 (2022).
</cite> [<a href="https://doi.org/10.7554/eLife.57974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8863373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35025730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Furniss,%20R.%20C.%20D.%20et%20al.%20Breaking%20antimicrobial%20resistance%20by%20disrupting%20extracytoplasmic%20protein%20folding.%20eLife11,%20e57974%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Dunn, S., Carrilero, L., Brockhurst, M. &amp; McNally, A. Limited and strain-specific transcriptional and growth responses to acquisition of a multidrug resistance plasmid in genetically diverse Escherichia coli lineages. <em>mSystems</em>, 10.1128/msystems.00083-21 (2021).</cite> [<a href="https://doi.org/10.1128/mSystems.00083-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8092126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33906912/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Buckner, M. M. C. et al. Clinically relevant plasmid-host interactions indicate that transcriptional and not genomic modifications ameliorate fitness costs of klebsiella pneumoniae carbapenemase-carrying plasmids. <em>mBio.</em>10.1128/mbio.02303-17 (2018).</cite> [<a href="https://doi.org/10.1128/mBio.02303-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5915730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29691332/" class="usa-link">PubMed</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>DelaFuente, J., Rodriguez-Beltran, J. &amp; San Millan, A. Methods to Study Fitness and Compensatory Adaptation in Plasmid-Carrying Bacteria. <em>Methods Mol. Biol. Clifton NJ</em><strong>2075</strong>, 371–382 (2020).</cite> [<a href="https://doi.org/10.1007/978-1-4939-9877-7_26" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31584176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?DelaFuente,%20J.,%20Rodriguez-Beltran,%20J.%20&amp;%20San%20Millan,%20A.%20Methods%20to%20Study%20Fitness%20and%20Compensatory%20Adaptation%20in%20Plasmid-Carrying%20Bacteria.%20Methods%20Mol.%20Biol.%20Clifton%20NJ2075,%20371%E2%80%93382%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Calvo-Villamañán, A. et al. On-target activity predictions enable improved CRISPR–dCas9 screens in bacteria. <em>Nucleic Acids Res</em><strong>48</strong>, e64–e64 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa294" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7293049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32352514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calvo-Villama%C3%B1%C3%A1n,%20A.%20et%20al.%20On-target%20activity%20predictions%20enable%20improved%20CRISPR%E2%80%93dCas9%20screens%20in%20bacteria.%20Nucleic%20Acids%20Res48,%20e64%E2%80%93e64%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Rousset, F. et al. The impact of genetic diversity on gene essentiality within the Escherichia coli species. <em>Nat. Microbiol</em>. 10.1038/s41564-020-00839-y (2021).</cite> [<a href="https://doi.org/10.1038/s41564-020-00839-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33462433/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Gibson, D. G. et al. Enzymatic assembly of DNA molecules up to several hundred kilobases. <em>Nat. Methods</em><strong>6</strong>, 343–345 (2009).
</cite> [<a href="https://doi.org/10.1038/nmeth.1318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19363495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibson,%20D.%20G.%20et%20al.%20Enzymatic%20assembly%20of%20DNA%20molecules%20up%20to%20several%20hundred%20kilobases.%20Nat.%20Methods6,%20343%E2%80%93345%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Le Roux, F., Binesse, J., Saulnier, D. &amp; Mazel, D. Construction of a vibrio splendidus mutant lacking the metalloprotease gene vsm by use of a novel counterselectable suicide vector. <em>Appl. Environ. Microbiol.</em><strong>73</strong>, 777–784 (2007).
</cite> [<a href="https://doi.org/10.1128/AEM.02147-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1800747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17122399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%20Roux,%20F.,%20Binesse,%20J.,%20Saulnier,%20D.%20&amp;%20Mazel,%20D.%20Construction%20of%20a%20vibrio%20splendidus%20mutant%20lacking%20the%20metalloprotease%20gene%20vsm%20by%20use%20of%20a%20novel%20counterselectable%20suicide%20vector.%20Appl.%20Environ.%20Microbiol.73,%20777%E2%80%93784%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Strand, T. A., Lale, R., Degnes, K. F., Lando, M. &amp; Valla, S. A New and Improved Host-Independent Plasmid System for RK2-Based Conjugal Transfer. <em>PLOS ONE</em><strong>9</strong>, e90372 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0090372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3940858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24595202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strand,%20T.%20A.,%20Lale,%20R.,%20Degnes,%20K.%20F.,%20Lando,%20M.%20&amp;%20Valla,%20S.%20A%20New%20and%20Improved%20Host-Independent%20Plasmid%20System%20for%20RK2-Based%20Conjugal%20Transfer.%20PLOS%20ONE9,%20e90372%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Andrews S. FastQC: a quality control tool for high throughput sequence data. <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a> (2010).</cite>
</li>
<li id="CR65">
<span class="label">65.</span><cite>Ewels, P., Magnusson, M., Lundin, S. &amp; Käller, M. MultiQC: summarize analysis results for multiple tools and samples in a single report. <em>Bioinformatics</em><strong>32</strong>, 3047–3048 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5039924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27312411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ewels,%20P.,%20Magnusson,%20M.,%20Lundin,%20S.%20&amp;%20K%C3%A4ller,%20M.%20MultiQC:%20summarize%20analysis%20results%20for%20multiple%20tools%20and%20samples%20in%20a%20single%20report.%20Bioinformatics32,%203047%E2%80%933048%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Katz, L. S. et al. Mashtree: a rapid comparison of whole genome sequence files. <em>J. Open Source Softw.</em><strong>4</strong>, 1762 (2019).</cite> [<a href="https://doi.org/10.21105/joss.01762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9380445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35978566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katz,%20L.%20S.%20et%20al.%20Mashtree:%20a%20rapid%20comparison%20of%20whole%20genome%20sequence%20files.%20J.%20Open%20Source%20Softw.4,%201762%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>San Millan, A., Heilbron, K. &amp; MacLean, R. C. Positive epistasis between co-infecting plasmids promotes plasmid survival in bacterial populations. <em>ISME J.</em><strong>8</strong>, 601–612 (2014).
</cite> [<a href="https://doi.org/10.1038/ismej.2013.182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3930321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24152711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?San%20Millan,%20A.,%20Heilbron,%20K.%20&amp;%20MacLean,%20R.%20C.%20Positive%20epistasis%20between%20co-infecting%20plasmids%20promotes%20plasmid%20survival%20in%20bacterial%20populations.%20ISME%20J.8,%20601%E2%80%93612%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Datsenko, K. A. &amp; Wanner, B. L. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. <em>Proc. Natl. Acad. Sci.</em><strong>97</strong>, 6640–6645 (2000).
</cite> [<a href="https://doi.org/10.1073/pnas.120163297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC18686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10829079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Datsenko,%20K.%20A.%20&amp;%20Wanner,%20B.%20L.%20One-step%20inactivation%20of%20chromosomal%20genes%20in%20Escherichia%20coli%20K-12%20using%20PCR%20products.%20Proc.%20Natl.%20Acad.%20Sci.97,%206640%E2%80%936645%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Sastre-Dominguez, J. Dissecting pOXA-48 fitness effects in clinical Enterobacterales using plasmid-wide CRISPRi screens. 10.5281/zenodo.16609943.</cite> [<a href="https://doi.org/10.1038/s41467-025-63082-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12361523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40825783/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (4.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63082_MOESM2_ESM.pdf</a><sup> (51.6KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (221.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (187KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (424.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (3.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (130.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (975.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361523/bin/41467_2025_63082_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (450.3KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The sequences generated during this project are available under the BioProject ID <span class="text-underline">PRJNA1203587</span> in the SRA repository of the National Centre for Biotechnology Information (NCBI). Source data for all figures is provided with this paper (see Supplementary Data <a href="#MOESM5" class="usa-link">3</a> and <a href="#MOESM6" class="usa-link">4</a>, Source Data, Supplementary Information). <a href="#Sec38" class="usa-link">Source data</a> are provided with this paper.</p>
<p>All the code developed for the analyses included in this work can be found at the <span class="text-underline">Github repository</span><sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63082-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63082.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12361523/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12361523/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12361523%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361523/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12361523/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12361523/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40825783/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12361523/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40825783/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12361523/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12361523/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wI0HSq3UfvBBYAHEIXlcFV5kDLV8WzBCA47A4ZUIegY7xZN09dvLQfIiU1Etbv8S">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
