
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The effect of WWTP products amendments on Phaseolus vulgaris rhizosphere and its ability to inactivate clarithromycin - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532F1D8AF228D3052F1D000CCF2FF1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373788/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="The effect of WWTP products amendments on Phaseolus vulgaris rhizosphere and its ability to inactivate clarithromycin">
<meta name="citation_author" content="Lucie Kotrbová">
<meta name="citation_author_institution" content="Department of Ecosystem, Faculty of Science, University of South Bohemia, České Budějovice, Czech Republic">
<meta name="citation_author_institution" content="Institute of Soil Biology and Biogeochemistry, Biology Centre Academy of Sciences of the Czech Republic, České Budějovice, Czech Republic">
<meta name="citation_author" content="Kateřina Grabicová">
<meta name="citation_author_institution" content="Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic">
<meta name="citation_author" content="Helena Švecová">
<meta name="citation_author_institution" content="Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic">
<meta name="citation_author" content="Andrea Vojs Staňová">
<meta name="citation_author_institution" content="Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic">
<meta name="citation_author_institution" content="Department of Analytical Chemistry, Faculty of Natural Sciences, Comenius University in Bratislava, Bratislava, Slovak Republic">
<meta name="citation_author" content="Martina Petrlíková">
<meta name="citation_author_institution" content="Institute of Soil Biology and Biogeochemistry, Biology Centre Academy of Sciences of the Czech Republic, České Budějovice, Czech Republic">
<meta name="citation_author" content="Roman Grabic">
<meta name="citation_author_institution" content="Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic">
<meta name="citation_author" content="Radka Kodešová">
<meta name="citation_author_institution" content="Department of Soil Science and Soil Protection, Faculty of Agrobiology, Czech University of Life Sciences Prague, Prague, Czech Republic">
<meta name="citation_author" content="Alica Chroňáková">
<meta name="citation_author_institution" content="Institute of Soil Biology and Biogeochemistry, Biology Centre Academy of Sciences of the Czech Republic, České Budějovice, Czech Republic">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30950">
<meta name="citation_doi" content="10.1038/s41598-025-14953-6">
<meta name="citation_pmid" content="40846880">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373788/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373788/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373788/pdf/41598_2025_Article_14953.pdf">
<meta name="description" content="With increasing efforts to reuse wastewater treatment plant (WWTP) products in agriculture, assessing their impact on soil-plant systems is crucial, while the effects of accompanying antibiotic residues on soil microbial communities have not yet ...">
<meta name="og:title" content="The effect of WWTP products amendments on Phaseolus vulgaris rhizosphere and its ability to inactivate clarithromycin">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="With increasing efforts to reuse wastewater treatment plant (WWTP) products in agriculture, assessing their impact on soil-plant systems is crucial, while the effects of accompanying antibiotic residues on soil microbial communities have not yet ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373788/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373788">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-14953-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_14953.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373788%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373788/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373788/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373788/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 22;15:30950. doi: <a href="https://doi.org/10.1038/s41598-025-14953-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-14953-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The effect of WWTP products amendments on <em>Phaseolus vulgaris</em> rhizosphere and its ability to inactivate clarithromycin</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kotrbov%C3%A1%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lucie Kotrbová</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lucie Kotrbová</span></h3>
<div class="p">
<sup>1</sup>Department of Ecosystem, Faculty of Science, University of South Bohemia, České Budějovice, Czech Republic </div>
<div class="p">
<sup>2</sup>Institute of Soil Biology and Biogeochemistry, Biology Centre Academy of Sciences of the Czech Republic, České Budějovice, Czech Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kotrbov%C3%A1%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucie Kotrbová</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grabicov%C3%A1%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Kateřina Grabicová</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kateřina Grabicová</span></h3>
<div class="p">
<sup>3</sup>Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grabicov%C3%A1%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kateřina Grabicová</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C5%A0vecov%C3%A1%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Helena Švecová</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Helena Švecová</span></h3>
<div class="p">
<sup>3</sup>Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C5%A0vecov%C3%A1%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Helena Švecová</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sta%C5%88ov%C3%A1%20AV%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Andrea Vojs Staňová</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Andrea Vojs Staňová</span></h3>
<div class="p">
<sup>3</sup>Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic </div>
<div class="p">
<sup>4</sup>Department of Analytical Chemistry, Faculty of Natural Sciences, Comenius University in Bratislava, Bratislava, Slovak Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sta%C5%88ov%C3%A1%20AV%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrea Vojs Staňová</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petrl%C3%ADkov%C3%A1%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Martina Petrlíková</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Martina Petrlíková</span></h3>
<div class="p">
<sup>2</sup>Institute of Soil Biology and Biogeochemistry, Biology Centre Academy of Sciences of the Czech Republic, České Budějovice, Czech Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petrl%C3%ADkov%C3%A1%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martina Petrlíková</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grabic%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Roman Grabic</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Roman Grabic</span></h3>
<div class="p">
<sup>3</sup>Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grabic%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Roman Grabic</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kode%C5%A1ov%C3%A1%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Radka Kodešová</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Radka Kodešová</span></h3>
<div class="p">
<sup>5</sup>Department of Soil Science and Soil Protection, Faculty of Agrobiology, Czech University of Life Sciences Prague, Prague, Czech Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kode%C5%A1ov%C3%A1%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Radka Kodešová</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chro%C5%88%C3%A1kov%C3%A1%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Alica Chroňáková</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Alica Chroňáková</span></h3>
<div class="p">
<sup>2</sup>Institute of Soil Biology and Biogeochemistry, Biology Centre Academy of Sciences of the Czech Republic, České Budějovice, Czech Republic </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chro%C5%88%C3%A1kov%C3%A1%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alica Chroňáková</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Ecosystem, Faculty of Science, University of South Bohemia, České Budějovice, Czech Republic </div>
<div id="Aff2">
<sup>2</sup>Institute of Soil Biology and Biogeochemistry, Biology Centre Academy of Sciences of the Czech Republic, České Budějovice, Czech Republic </div>
<div id="Aff3">
<sup>3</sup>Faculty of Fisheries and Protections of Waters, University of South Bohemia in České Budějovice, South Bohemian Research Centre of Aquaculture and Biodiversity of Hydrocenoses, Vodňany, Czech Republic </div>
<div id="Aff4">
<sup>4</sup>Department of Analytical Chemistry, Faculty of Natural Sciences, Comenius University in Bratislava, Bratislava, Slovak Republic </div>
<div id="Aff5">
<sup>5</sup>Department of Soil Science and Soil Protection, Faculty of Agrobiology, Czech University of Life Sciences Prague, Prague, Czech Republic </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 11; Accepted 2025 Aug 4; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373788  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846880/" class="usa-link">40846880</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">With increasing efforts to reuse wastewater treatment plant (WWTP) products in agriculture, assessing their impact on soil-plant systems is crucial, while the effects of accompanying antibiotic residues on soil microbial communities have not yet been adequately studied. This study focuses on clarithromycin (CLR), highly present in wastewater, and investigates the CLR-degradation potential of plant-associated microorganisms. <em>Phaseolus vulgaris</em> plants were grown in raised beds filled with Haplic Cambisol and amended with or without WWTP products (treated wastewater, biosolid, or composted biosolid), as a source of CLR residues. The rhizosphere microbiomes after biosolid amendments was significantly enriched by <em>Pseudomonadaceae</em> as assessed by 16S rRNA metagenomics and cultures enriched by CLR revealed dominance of <em>Proteobacteria</em>. However, no degradation of CLR by microbial consortia or enrichment cultures was observed, suggesting the multiplication of CLR-resistant bacteria with other resistance mechanisms. Cultivation-based approach combined with antibiotic modulation assays and subsequent LC-MS analysis confirmed the complete CLR removal by seven phylogenetic groups of actinomycetes in vitro. The proportion of isolates indicated that the rhizosphere is a natural reservoir for CLR-inactivating microorganisms; however, the amendment of soils with WWTP products can significantly increase their abundance and diversity.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-14953-6.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Biodegradation, Macrolides, Soil microbiome, Antimicrobial resistance, <em>Streptomyces</em>, Micropollutants</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Microbiology, Environmental sciences, Microbial ecology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The rhizosphere plays a crucial role in plant health and the functioning of the soil ecosystem. Complex root-microbe interactions improve plant fitness by facilitating nutrient uptake (nitrogen fixation, phosphorus solubilization, siderophore production), producing various growth-promoting hormones that help reduce the negative effects of environmental stressors such as drought and salinity<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> and by providing a broad spectrum of antimicrobial compounds that act as a barrier against pathogen invasion<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. In addition to these fundamental functions, various plant-associated microorganisms degrade a wide range of anthropogenic pollutants in the environment. This capability is extensively utilized in phytoremediation practices, particularly within constructed wetland<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The micropollutants are ubiquitous and enter the soil and thus the rhizosphere through human activities. One of the main sources of pharmaceutical contaminants are municipal and industrial wastewater<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Recently, the focus has shifted to the circular economy and the efficient use of by-products from wastewater treatment plants (WWTPs) in agriculture. Examples include the use of biosolids as plant fertilizer and the use of treated wastewater (TWW) for irrigation. This practice raises important questions about the impact of pharmaceutical residues in WWTP products on the soil and plant microbiome, and their subsequent fate in the environment.</p>
<p id="Par4">The range and extent of pharmaceuticals present in WWTP products is related to the proportion of their consumption in the population<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. One of the most worrying are antibiotics, as multiresistant pathogens are rapidly developing. Macrolide antibiotics are critically important antibiotics in human therapy<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>and as such they are among the most commonly prescribed antibiotics in Central and Western Europe<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. They comprise a broad spectrum of molecules consisting of a macrocyclic lactone ring with attached deoxy-sugar or amino-sugar residues and have a bacteriostatic effect by inhibiting proteosynthesis. Their spectrum of activity depends on the size of the macrolactone ring (macrolides with 12- to 16-membered rings) and the presence or absence of cladinose. In human medicine, macrolides mainly comprises the well-known representative erythromycin (natural product, ERY) and its derivatives clarithromycin (CLR) and azithromycin (AZI).</p>
<p id="Par5">In WWTP, the reduction of macrolide antibiotic concentrations involves both biotic and abiotic processes. While mechanisms such as sorption to sewage sludge and photodegradation via UV disinfection contribute to macrolide attenuation, microbial degradation in activated sludge systems is the main route for their removal<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Nevertheless, these processes are often insufficient to achieve complete elimination, resulting in the persistence of macrolide residues in the final effluents and sludge-derived products. The concentrations of the various macrolides in TWW then range from nanograms to micrograms per liter<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Occasional deviations are observed in TWW from pharmaceutical companies<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. In biosolids, macrolide concentrations are found in tens of nanograms per gram<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Thus, their concentrations often exceed the PNEC (Predicted No Effect Concentration) values, which were developed to assess the risks of pollutants to biological systems and the risk associated with the spread of antibiotic resistance mechanisms (PNEC<sub>res</sub>)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> in surface waters. The most commonly prescribed in the EU is CLR<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, 40% of which is excreted in active form<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. CLR exceeded the PNEC values in 32.1% of 123,761 environmental samples analysed worldwide from various sources (hospital wastewater 66% &gt; municipal wastewater 63% &gt; surface water 5%<sup>20</sup>) and was identified as the macrolide with the highest risk quotient (RQ) associated with resistance development during wastewater treatment<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par6">With the WWTP products, macrolides enter the freshwater and soils, resulting often in alarming concentrations in both matrices<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Their fate in the receiving soil is influenced by various soil properties such as pH, cation exchange capacity, clay and organic matter resulting in high sorption of macrolides to soil particles and in their high persistence in the environment<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. However, experimental designs comparing sterile and non-sterile soils have also shown significant role of microorganisms in degradation of ERY<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> indicated often by sharp decrease of half-lives, DT<sub>50</sub>. In addition, previous exposure of soil to macrolides indicated increased degradation and selection of degrading soil microbiota in a field study<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. What is more, direct in vitro degradation of ERY by soil and cave bacteria has occasionally been demonstrated<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. In addition, WWTP products have previously been associated with CLR-degrading microorganisms<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> and may increase the degradative capacity of plant-associated indigenous microbiome. However, none of such in vitro experiments were conducted with CLR. Furthermore, strain should be considered as non-pathogenic, contaminant-tolerant microbe with high environmental adaptability for use in remediation strategy. In this context, spore-forming filamentous bacteria of the genus <em>Streptomyces</em> are of great interest. Streptomycetes are typical soil saprophytic bacteria with a high metabolic versatility known for production of variety of extracellular enzymes that ensure broad spectrum of degradation capacities<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par7">Despite the recognized potential of biodegrading microorganisms in the remediation of contaminated sites, there is little comprehensive information on the soil-plant microbiome responsible for CLR degradation in both uncontaminated and contaminated soils. Therefore, in our study, we aimed to characterize rhizosphere bacterial composition and its ability to degrade macrolide antibiotic CLR as affected by soil management type. We hypothesize, that the application of WWTP products containing residues of CLR will enhance degradation capacity of rhizosphere microbiome. We expected that higher and continuous CLR preasure will select CLR-degrading microorganisms. Although many Gram-negatives were previously associated with degradation of macrolides, we hypothesise that Gram-positive filamentous-like bacteria typical for the soil environment are abundant CLR-degrader as well. To achieve our goals, we characterized rhizosphere of plants grown in two year mesocosm trial representing four different management practices: CMW was irrigated with tap water (control soil), CME was irrigated with TWW, CMB was amended with biosolid, and CMC was amended with composted biosolid. In the crop-rotation system, rhizosphere of beans (<em>Phaseolus vulgaris</em>) grown in the second year of the trial (year 2022) was collected. Different approaches can be used to obtain and characterize degradative microorganisms: isolation of microbial consortia enhancing synergistic mechanisms<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>or isolation of pure cultures<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. In addition, prior enrichment of the microbial consortium with macrolides at high concentrations under co-metabolic conditions can increase the yield of degrading microorganisms<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Here, we followed all the approaches. First, we characterized the bacterial composition of the rhizosphere, and evaluated the degradation of CLR by whole microbial consortia (DG1) and by microbial consorcia enhanced by increasing enrichment of CLR (DG2). Secondly, in view of previous studies showing the importance of actinomycetes in the degradation of non-macrolide antibiotics and other various pollutants and their potential in bioremediation<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>we focused on the CLR inactivating actinomycete cultures.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Occurrence of selected macrolides in WWTP products and treated soils</h3>
<p id="Par8">The treated wastewater contained all three macrolides tested (ERY, CLR, AZI), with concentrations being lowest for ERY (median values 27 and 21 ng L<sup>− 1</sup> in years 2021 and 2022, respectively) and the highest for CLR (median values 280 and 230 ng L<sup>− 1</sup> in years 2021 and 2022, respectively). Concentrations were comparable in both years, with maximum values in April and October (start and end of vegetation season). In contrast, only AZI and CLR were detected in biosolid, and, surprisingly, concentrations varied from year to year with higher values in 2022. Only AZI was detected in composted biosolids. The tap water contained trace concentrations of CLR (5.9 ng L<sup>− 1</sup> and 1.5 ng L<sup>− 1</sup> in years 2021 and 2022, respectively). The concentrations of macrolides in the treated soils and in the water discharged from soil beds were below the limits of quantification. The results are shown in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Concentrations of tested macrolides in matrices (TWW, cambisol, tap water, biosolid and composted biosolid).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">TWW 4–11/2021 (<em>n</em> = 30), min-max (median) values [ng L<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Cambisol (<em>n</em> = 2), min-max (median) values [ng g<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Composted biosolid (<em>n</em> = 2),min-max (median) values [ng g<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Biosolid (<em>n</em> = 2), min-max (median) values [ng g<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Tap water 9/2021 (<em>n</em> = 2), min-max (median) values [ng L<sup>− 1</sup>]</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="6" rowspan="1">Year 2021</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Azithromycin</td>
<td align="left" colspan="1" rowspan="1">10–220 (64)</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
<td align="left" colspan="1" rowspan="1">9.4–20 (15)</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Clarithromycin</td>
<td align="left" colspan="1" rowspan="1">140–670 (280)</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
<td align="left" colspan="1" rowspan="1">7.9–19 (13)</td>
<td align="left" colspan="1" rowspan="1">5.6–6.2 (5.9)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Erythromycin</td>
<td align="left" colspan="1" rowspan="1">10–41 (27)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">TWW 4–9/2022 (<em>n</em> = 28), min-max (median) values [ng L<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Cambisol, min-max (median) values [ng g<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Composted biosolid (<em>n</em> = 3), min-max (median) values [ng L<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Biosolid (<em>n</em> = 3), min-max (median) [ng g<sup>− 1</sup>]</th>
<th align="left" colspan="1" rowspan="1">Tap water 4/2022–11/2022 (<em>n</em> = 3), min-max (median) values [ng L<sup>− 1</sup>]</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="6" rowspan="1">Year 2022</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Azithromycin</td>
<td align="left" colspan="1" rowspan="1">21–170 (76)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">24–30 (25)</td>
<td align="left" colspan="1" rowspan="1">270–290 (280)</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Clarithromycin</td>
<td align="left" colspan="1" rowspan="1">98–660 (230)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
<td align="left" colspan="1" rowspan="1">41–47 (44)</td>
<td align="left" colspan="1" rowspan="1">1.2–1.7 (1.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Erythromycin</td>
<td align="left" colspan="1" rowspan="1">5.6–78 (21)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
<td align="left" colspan="1" rowspan="1">&lt; LOQ</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p11"><p>LOQ values are provided in Supplementary Table S8. Abbreviations: LOQ – limits of quantification.</p></div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Rhizosphere Microbiome as affected by soil management</h3>
<p id="Par10">As expected, soil amendments with biosolid (CMB) and compost (CMC) altered nutrient content when compared to the control (CMW, Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>). Significant increase were observed in dissolved phosphorus (DP) content (<em>p &lt; 0.001</em> in both cases) and carbon content in microbial biomass (C<sub>mic</sub>; <em>p</em> &lt; 0.05 in CMC). Irrigation with TWW (CME) had no effect on the observed parameters. More details are shown in Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>.</p>
<p id="Par11">These shifts were not reflected in the changes in bacterial α-diversity of respective rhizosphere (Supplementary Figure <a href="#MOESM2" class="usa-link">S2</a>a), as neither the Shannon index nor Chao 1 differed between managements (<em>p &gt; 0.05</em> in all cases, Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>). However, the effect of management was evident in prokaryotic β-diversity as assessed by NMDS analyses. Both biosolids amendments (CMC and CMB) formed separate clusters, while CMW and CME showed a greater degree of similarity (Supplementary Figure <a href="#MOESM2" class="usa-link">S2</a>b).</p>
<p id="Par12">The prokaryotic communities of the rhizosphere were dominated by <em>Proteobacteria</em> (50.9% on average), followed by <em>Actinobacteriota</em> (15.4%). The most abundant families were <em>Commamonadaceae</em>,<em> Nitrosomonadaceae</em>, <em>Pseudomonadaceae</em>, <em>SC-I-84</em>, <em>Gemmatimonadaceae</em> and <em>Xanthobacteraceae</em> (Fig. <a href="#Fig1" class="usa-link">1</a>a, b). Differential abundance analysis (DESeq2) showed the differences between the prokaryotic communities at family level as affected by soil management. <em>Thermoactinomycetaceae</em> increased their relative abundance in all soils amended by WWTP products (CME, CMC and CMB) compared to the control, while <em>Oxalobacteraceae</em> were increased only in CMB. The largest taxonomic shift was observed for the <em>Pseudomonadaceae</em>, whose relative abundance increased almost ninefold in CMC and CMB compared to the control (Fig. <a href="#Fig1" class="usa-link">1</a>c).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373788_41598_2025_14953_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6658/12373788/f8e18bf02345/41598_2025_14953_Fig1_HTML.jpg" loading="lazy" id="d33e761" height="447" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative abundance of prokaryotic taxa in rhizosphere and enrichment. (<strong>a</strong>) Heatmap at the level of phylum. (<strong>b</strong>) Heatmap at the level of family. Only taxa comprising relative abundance higher than 1.5% in more than 20% of samples are shown. (<strong>c</strong>) Differential abundance analyses (DESeq2) performed on the level of family. Only statistically significant results are shown (<em>p &lt; 0.05</em>). Abbreviations: CMW – soil irrigated with tap water, CME –soil irrigated with TWW, CMC – soil amended with composted biosolid and irrigate with TWW, CMB – soil amended with biosolid and irrigated with tap water; the treatment type supplemented with -E indicates the rhizosphere samples enriched for four months with clarithromycin in liquid minimal medium.</p></figcaption></figure><p id="Par14">We did not observed any significant differences between managements in the culturable bacteria in the rhizosphere on non-selective medium, however, we observed changes on medium supplemented with CLR (<em>p = 0.712</em>). The CLR-resistant subpopulation was comparable in CMW, CME and CMC, but was significantly larger in CMB compared to CMW (<em>p = 0.011</em>, Fig. <a href="#Fig2" class="usa-link">2</a>). The prevalence of CLR-resistant bacteria was three times higher in CMB than in the others (4.2% in CMB and 1.1–1.5% in the other managements). The results of the statistical analysis can be found in Supplementary Table S3.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373788_41598_2025_14953_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6658/12373788/d32811a7d2bf/41598_2025_14953_Fig2_HTML.jpg" loading="lazy" id="d33e786" height="629" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cultivable bacteria in rhizosphere. logCFU per gram of wet weight of rhizosphere (<em>n</em> = 3) after 6 days of cultivation at 28 °C. Signif. codes: 0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1. ). Abbreviations: CMW – soil irrigated with tap water, CME –soil irrigated with TWW, CMC – soil amended with composted biosolid and irrigate with TWW, CMB – soil amended with biosolid and irrigated with tap water.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Rhizosphere Microbiome as such showed no measurable degradation of CLR</h3>
<p id="Par16">The results of the liquid chromatography mass spectrometry (LC-MS) analysis (first degradation experiment, DG1) showed that the microbial population of the rhizosphere did not degrade CLR under the given experimental conditions (Supplementary Figure S3). Only the CMW and CME management types were included in DG1, as we assumed the previous selective pressure of CLR present in TWW used for continuous long-term irrigation of plants (e.g. growing seasons in years 2021 and 2022). Both management regimes showed a similar response to CLR as the sole carbon source. The total number of bacterial cells slightly decreased on day 3 and increased on day 11 compared to the control without CLR. The number of cultivable bacteria on day 6 were 9.6 × 10<sup>5</sup> and 5.9 × 10<sup>5</sup> CFU per mL in CMW and CME, respectively.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The enrichment cultures revealed amplification of resistant bacteria but no degradation of CLR</h3>
<p id="Par17">To increase the probability of obtaining a CLR-degrading culture, rhizosphere microbial communities were enriched with increasing concentrations of CLR under co-metabolic conditions for up to four months. All management types were included in the enrichment experiment (enriched cultures are referred here as CMW-E, CME-E, CMC-E and CMB-E). Once the final concentration of CLR reached the 1 mg L<sup>− 1</sup>, the viability of the enriched cultures was checked by standard cultivation on agar plates. The CFU values were similar regardless of the management (<em>p = 0.780</em>; Supplementary Table S4). Although we expected an activation of the degradation mechanisms under the given experimental conditions, no degradation activity was revealed from LC-MS data. The number of cultivable bacteria on day 6 of degradation experiment using enriched cultures (DG2) showed no effect of soil management (only minor differences, <em>p</em> = 0.169). The total cell counts were also similar on day 3, but surprisingly, the total cell counts increased on day 11 of the experiment in the CMW-E, while it remained stable in the other enriched cultures). The results of DG2 are shown in Supplementary Figure S4.</p>
<p id="Par18">To characterize the enriched microbial population, we performed sequencing of 16S rDNA gene amplicons from cultures obtained at the final enrichment concentration (1 mg L<sup>− 1</sup>). Soil management had no effect on the α-diversity of prokaryotic communities (both Shannon and Chao 1 indices <em>p</em> &gt; 0.05, Supplementary Figure S<a href="#MOESM1" class="usa-link">2</a>s). As expected, the microbial communities of the enriched cultures differed significantly from the original rhizosphere. Enrichment with CLR reduced α-diversity (<em>p &lt; 0.001</em> in both indices) and altered the structure of the microbial communities significantly (Supplementary Figure S<a href="#MOESM1" class="usa-link">2</a>a, c). The most abundant phylum was <em>Proteobacteria</em> (91.4%) with minor participation of <em>Bdellovibrionota</em> (3.2%), <em>Verrucomicrobiota</em> (1.4%) and <em>Armatimonadota</em> (1%). Proteobacteria increased their relative abundance almost one-fold in enrichments when compared to original rhizosphere (Fig. <a href="#Fig1" class="usa-link">1</a>a) and were represented mainly by families <em>Comamonadaceae</em> (28%), <em>Pseudomonadaceae</em> (26.9%), <em>Xanthomonadaceae</em> (15.2%), <em>Chromobacteriaceae</em> (9%), and <em>Caulobacteraceae</em> (4%), followed by <em>Bdellovibrionaceae</em> (3.2%) (Fig. <a href="#Fig1" class="usa-link">1</a>b; Supplementary Table S5). Community structure was the most different in CMC-E in comparison to other enrichments, which clustered together(Supplementary Figure <a href="#MOESM2" class="usa-link">S2</a>c). DESeq2 analysis (<em>p &lt; 0.05</em>) revealed that the increase of relative abundance of <em>Sphingomonadaceae</em> was associated with soils irrigated with TWW (both CME-E and CMC-E), and <em>Oxalobacteraceae</em>, whose relative abundance was lower in CME-E and CMB-E in comparison to control (CMW-E, Fig. <a href="#Fig1" class="usa-link">1</a>c).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Pure isolates of actinomycetes from rhizosphere inactivate CLR</h3>
<p id="Par19">A total of 111 isolates were obtained from the rhizosphere with typical actinomycetes morphology, which could grow on CLR-supplemented medium. The isolates were tested using solid antibiotic modulation assay (AMA) and yielded 32 CLR-inactivating actinomycetes, of which 28 were unique (39.6% of the total). The lowest proportion was observed in CMW (7.7% of 13 isolates in total) and a comparable proportion in other soil managements: 30% (50 in total), 27.7% (22 in total) and 27% (26 in total) for CME, CMC and CMB, respectively. In addition, another 23 isolates, of which 14 were unique, showed an ambiguous inactivating response. Taxonomic assignment of 16S rRNA sequences of the total 42 isolates with positive and ambiguous response in solid-AMA revealed three bacterial genera of CLR-inactivating actinomycetes: <em>Streptomyces</em> (39 isolates), <em>Nocardiopsis</em> (2 isolates) and <em>Nocardioides</em> (1 isolate) (Fig. <a href="#Fig3" class="usa-link">3</a>). Phylogenetic analysis revealed six clades of <em>Streptomyces</em> sp., dominated by isolates of <em>S. albidoflavus</em> group, <em>S. xiamenensis</em> and <em>S. anulatus</em> group. The strongest inactivation activity showed isolates identified as <em>S. xiamenensis</em> (CME and CMC), <em>S. intermedius</em> group (CMB), <em>S. albidoflavus</em> group (all soil management types) and <em>Nocardiopsis alba</em> (CMB), while the isolates with ambiguous response were identified as <em>S. anulatus</em> group (CME, CMC and CMB), <em>S. zaomyceticus</em> (CME), <em>S</em>. <em>nigrescens</em> group (CME) and <em>Nocardioides albus</em> (CME).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373788_41598_2025_14953_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6658/12373788/9fc11b020bc4/41598_2025_14953_Fig3_HTML.jpg" loading="lazy" id="d33e943" height="521" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Maximum-likelihood (PhyML) 16S rRNA gene phylogenetic tree of rhizosphere isolates with positive or ambiguous results in antibiotic modulation assay. Soil management type of isolates are distinguished by colour of the isolate ID (BCCO 10_XXXX). The inner ring shows the clade assignment, the outer two rings show the results of solid- and liquid-antibiotic modulation assay. Abbreviations: CMW – soil irrigated with tap water, CME –soil irrigated with TWW, CMC – soil amended with composted biosolid and irrigate with TWW, CMB – soil amended with biosolid and irrigated with tap water.</p></figcaption></figure><p id="Par21">The isolates were further tested using liquid-AMA, the selection process included the affiliation to a phylogenetic clade, type of soil management and intensity of CLR-inactivation in solid-AMA. Finally, a total of 30 isolates were tested, of which 70% showed a significant decrease in CLR concentration in the spent medium. The results of both antibiotic modulation assays are shown in Fig. <a href="#Fig3" class="usa-link">3</a>. 67% of the isolates gave consistent results in both types of assays.</p>
<p id="Par22">While some of CLR-inactivating <em>Streptomyces</em> clades gave identical results regardless of soil management type (<em>S. albidoflavus</em> group), some of the highly represented clades showed the management-specific response. Isolates of the <em>S. anulatus</em> group inactivated CLR more efficiently when isolated from rhizosphere irrigated with TWW. In addition, <em>S. xiamenensis</em> isolates were only recovered from the rhizosphere irrigated with TWW (CME, CMC). The accuracy of liquid-AMA was verified by measuring the residual concentration of CLR in the spent medium by LC-MS for selected isolates (Table <a href="#Tab2" class="usa-link">2</a>). Total removal (100%) of CLR was confirmed for isolates with strong liquid-AMA activity (formation of no inhibition zone). False-positive results were ruled out by biomass sorption test, which showed no sorption of CLR to microbial biomass (biomass of isolates BCCO 10_2486, BCCO 10_2487, BCCO 10_2488 and BCCO 10_2496 tested).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Results for the isolates, whose biodegradation were verified by LC-HRMS.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolate ID</th>
<th align="left" colspan="1" rowspan="1">Soil management</th>
<th align="left" colspan="1" rowspan="1">The closest relatives (% sequence identity, accession no.)</th>
<th align="left" colspan="1" rowspan="1">Solid-AMA (decrease of zone size in mm)</th>
<th align="left" colspan="1" rowspan="1">Liquid-AMA (Visible zone of inhibition in mm)</th>
<th align="left" colspan="1" rowspan="1">Biomass sorption assay</th>
<th align="left" colspan="1" rowspan="1">Average initial concentration of CLR (µg L<sup>− 1</sup>) ± SD (<em>n</em> = 3), LOQ (min-max) = 1.8–4.8</th>
<th align="left" colspan="1" rowspan="1">Removal efficiency LC-HRMS (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2490</td>
<td align="left" colspan="1" rowspan="1">CMB</td>
<td align="left" colspan="1" rowspan="1">
<em>Nocardiopsis alba (99.79%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_026340.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_026340.1</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">++</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">NT</td>
<td align="center" colspan="1" rowspan="1">1000 ± 60</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2489</td>
<td align="left" colspan="1" rowspan="1">CMB</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptomyces intermedius/ koyangensis (99.72%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_041103.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_041103.1</a>/ <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP031742.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP031742.1</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">++</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">NT</td>
<td align="center" colspan="1" rowspan="1">1000 ± 0</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2485</td>
<td align="left" colspan="1" rowspan="1">CME</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptomyces koyangensis (99.72%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_025662.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_025662.1</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">NT</td>
<td align="center" colspan="1" rowspan="1">1000 ± 0</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2486</td>
<td align="left" colspan="1" rowspan="1">CME</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptomyces anulatus/ pretensis/ praecox/ durocortorensis (100.00%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_112527.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_112527.1</a>/ <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_125619.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_125619.1</a>/ <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_112358.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_112358.1</a>/  <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW582863.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW582863.1</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">+/-</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">not sorbed</td>
<td align="center" colspan="1" rowspan="1">1100 ± 0</td>
<td align="left" colspan="1" rowspan="1">98.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2496</td>
<td align="left" colspan="1" rowspan="1">CME</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptomyces resistomycificus/ griseochromogenes (100.00%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_042100.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_042100.1</a>/ <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_042102.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_042102.1</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">++</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">not sorbed</td>
<td align="center" colspan="1" rowspan="1">1000 ± 60</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2488</td>
<td align="left" colspan="1" rowspan="1">CME</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptomyces xiamenensis (99.51%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP009922.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP009922.3</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">+++</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">not sorbed</td>
<td align="center" colspan="1" rowspan="1">1000 ± 0</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2487</td>
<td align="left" colspan="1" rowspan="1">CME</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptomyces griseus (99.93%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU048540.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU048540.1</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">++</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">not sorbed</td>
<td align="center" colspan="1" rowspan="1">1100 ± 60</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BCCO_10_2492</td>
<td align="left" colspan="1" rowspan="1">CMC</td>
<td align="left" colspan="1" rowspan="1">
<em>Streptomyces xiamenensis (99.20%</em>,<em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP009922.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP009922.3</a>)</em>
</td>
<td align="left" colspan="1" rowspan="1">+++</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">NT</td>
<td align="center" colspan="1" rowspan="1">1000 ± 0</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p30"><p>Experiments in solid-AMA were performed on MH agar in 28 °C for 3 days, while in liquid-AMA in MH broth in 28 °C for 5 days. Abbreviation: +/- decrease in ZD by &lt; 3 mm, + decrease in ZD by 4–7 mm, ++ decrease in ZD by 8–14 mm, +++ decrease in ZD by &gt; 15 mm; NT - not tested; CMW – soil irrigated with tap water, CME –soil irrigated with TWW, CMC – soil amended with composted biosolid and irrigate with TWW, CMB – soil amended with biosolid and irrigated with tap water.</p></div></div></section></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">Given the severe effort for reuse of WWTP products, there is a lack of research addressing the impact of the WWTP products on the rhizosphere microbiome of edible plants and the fate of certain micropollutants. We focused on the biodegradation of the second most common antibiotic, clarithromycin. As expected, CLR was detected in TWW in significantly higher concentrations than other macrolides, following the data about the prescriptions (provided in Supplementary Table S6) with seasonal peak in autumn. The concentrations of macrolides in biosolids varied between years, being higher in 2022, especially for AZI. The different occurrence patterns of tested macrolides in the matrices underlines the fact that the individual macrolide compounds have different physicochemical properties, e.g. higher sorption of AZI<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. In TWW and biosolid, CLR exceeded PNEC<sub>res</sub> (liquid and soil<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, suggesting the selection and spread of CLR-resistant bacteria. CLR has previously been identified as one of the main contaminants among antibiotic residues in the aquatic environment<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and is one of the main micropollutant requiring special attention in wastewater treatment<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Despite the presence of macrolides in WWTP products, they did not reach measurable levels in the receiving soils even after two years of application. This may be caused by degradation of macrolides by soil microorganisms, however also by their leakage due to heavy rainfalls in the summer season<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Trace concentrations of CLR found in tap water are alarming (1.2–6.2 ng L<sup>− 1</sup>, Table <a href="#Tab1" class="usa-link">1</a>). Previously<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, found various pharmaceuticals, including macrolides, in the drinking water reservoir (Švihov, the Czech Republic) downstream of outlet of TWW. This highlight the limitations of wastewater treatment methods and emphasize the importance of use of advanced treatment technologies. In contrast, no macrolides were detected in Cambisol prior to the construction of the soil mesocosms.</p>
<p id="Par25">The rhizosphere is not only a hotspot of microbial activity in the soil, but also a pool of microorganisms that can establish closer relationships with plants or directly colonize their edible parts. Giving the increasing importance of the circular economy, many studies have focused on assessing the safety of WWTP products for agricultural use, but little is known about the effects of WWTP products on the functional attributes of plant-associated microbiome. In our study, we used a wide range of methods to obtain comprehensive information on the effects of WWTP products on the bacterial community in the rhizosphere of <em>P. vulgaris</em> and their potential to degrade CLR. First, using 16S rRNA metagenomics, we found a high resilience of bacteria to the use of TWW for irrigation over two years, similar to that described for bacteria in bulk soil after one year of the same management<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. On the other hand, amendment of both types of biosolids to the rhizosphere significantly increased the relative abundance of <em>Pseudomonadaceae</em> compared to the control. This is probably related to the improved nutrient status of the soil (i.e. increased DP content), and is also consistent with the high abundance of <em>Pseudomonadaceae</em> previously found in the rhizosphere of <em>P. vulgaris</em> grown in Phaeozem by Medina-Pérez et al.<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. These changes in the structure of the bacterial community were not reflected in the structure of the enriched cultures. Secondly, the cultivation approaches showed the effect of biosolid on the prevalence of CLR-resistant rhizosphere bacteria, which was not observed with composted biosolid or TWW (Fig. <a href="#Fig2" class="usa-link">2</a>). Previously, an increase in resistant bacteria in vitro after biosolids addition had not been observed, however, studies tended to focus on bacteria considered indicative of fecal contamination rather than native soil microbiome<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Third, by characterizing the CLR-inactivating streptomycetes isolated from the rhizosphere, we found differences in the observed bacterial taxa and in the CLR-inactivation efficiency related to the type of WWTP product introduced into the soil. The diversity of CLR-inactivating streptomycetes was highest in the rhizosphere irrigated with TWW (seven clades), followed by CMB (four), CMC (three) and the lowest in the untreated rhizosphere (one). The degradation of CLR were also verified for eight strains with positive CLR inactivation. This suggests that agricultural soils harbour indigenous streptomycetes that can degrade CLR and that this capacity is enhanced by the addition of WWTP products. This is probably related to the macrolide residues contained in WWTP products and persistent selection pressure on the rhizosphere microbiome. The knowledge about the occurrence of streptomycetes in WWTP products is limited. However, Streptomycetes are thought to frequently colonize humans<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a> </sup>and the phylotypes found in this study have previously been associated with human tissue<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<p id="Par26">We aimed to investigate CLR-inactivation activity and taxonomic assignment of microorganisms in the rhizosphere of <em>P. vulgaris</em> grown in Cambisol under different management regimes: with or without amendments of TWW or biosolids. To characterise and obtain the CLR-degrading microorganisms in the experimental rhizosphere, we pursued two complementary approaches. The first approach involved the isolation of bacterial consortia from the rhizosphere that utilise a variety of microbial enzymes and cell-to-cell signaling pathways to degrade a broad range of pollutants. In addition, to select for CLR-degrading microbial community, we also performed enrichment of the rhizosphere culture with CLR in increasing concentration up to four months. In the second approach, pure <em>Streptomyces</em>-like cultures were isolated from the rhizosphere using standard cultivation techniques. Both approaches are widely used, but here the isolation of CLR-degrading microbial consortia failed, although the viability of the consortia were verified (cultivation on agar plates and microscopy, Supplementary Figures S3-S4). The rhizosphere as an environment with abundant labile organic substrates selects for copiotrophs with faster growth rate<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Some of them were previously identified as ERY degraders in the aquatic environment<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Shen et al.<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> identified <em>Proteobacteria</em> and <em>Bacteroidetes</em> as one of the ERY degraders in soil. Copiotrophs dominated in the enriched cultures too (<em>Comamonadaceae</em>, <em>Pseudomonadaceae</em> and <em>Xanthomonadaceae</em>), and likely employed other CLR-resistance mechanisms for self-protection than CLR-degradation. These could be methylation of the macrolide target within the cell (mediated by <em>erm</em> genes), protection of the target by the production ABC-F protein (mediated by <em>msr</em> genes) or the use of broad-spectrum efflux pumps to expel macrolides from the cell<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. However, these were not the subject of this study.</p>
<p id="Par27">We performed the isolation on agar with CLR-concentration of 1 mg L<sup>− 1</sup>, which is below the epidemiological cut-off value (ECCOF) presented by The European Committee on Antimicrobial Susceptibility Testing for mycobacteria (as representatives of actinomycetes)<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a> </sup>and below the clinical breakpoint values set for <em>Nocardia</em> and other actinomycetes<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. This ensured to retrieve actinomycetes posing various CLR-resistance mechanisms including the biodegradation. By the combination of solid and liquid AMA, we obtained representatives of three actinomycetes genera capable of inactivating CLR, which clustered into eight phylogenetic clades. The most of strains were assigned to the genus <em>Streptomyces</em>. In some cases (33%), the differences in the CLR removal efficiency between solid-AMA and liquid-AMA occurred, which could be explained by specific growth requirements, which are difficult to meet in the screening stage. Streptomycetes are typical soil saprophytes, often present active in the rhizosphere of various plants. They prefer to grow in a more structured environment for the complex cellular development and spore formation and have a long generation time<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Therefore, in the enriched cultures, they were probably overgrown by fast-growers. Although they were previously isolated from human tissue, they are not considered obligate pathogens except those causing endemic actinomycetoma<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. On the other hand, they are the main producers of natural antibiotics, including macrolides. Therefore, they may produce enzymes that modify and inactivate the antibiotic for self-defence (glycosylases encoded by <em>ole</em> genes<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Moreover, the predictions of the annotated genome sequences of streptomycetes suggest that they may also utilize homologues of hydrolytic enzymes such as erythromycin esterases (Ere) or α/β-hydrolases (Est) able to irreversibly break down the macrolactone ring, the central structure of the macrolides. The action of previously unknown or overlooked genes coding for new enzymes involved in resistance mechanisms cannot be ruled out<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p>
<p id="Par28">The biodegradation of antibiotics is an ancient tool that provides self-protection for antibiotic-producing bacteria and serves as a natural mechanism for detoxifying the environment<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. At the same time, some of the genes encoding antibiotic-degrading enzymes also confer resistance to antibiotics in pathogenic bacteria<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. This dual role of antibiotic-degrading genes needs to be considered and risk assessment related to their spread in the environment requires further investigation. Some of the resistance genes are being monitored globally, especially in clinically important pathogenic species both in clinical setting and in the environment<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Previous studies have shown that the pool of resistance genes differs between pathogenic and antibiotic-producing taxa and that there is a phylogenetic barrier to their transmission<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. This argues for the use of antibiotic-degrading <em>Streptomyces</em> species (natural soil dwellers) in bioremediation to reduce environmental concentrations and thus slow the development and spread of antibiotic resistance in contaminated environments.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par29">The increasing environmental pollution by antibiotics requires a comprehensive understanding of their fate in the environment. The role of the rhizosphere in the degradation of antibiotics is undisputed, but the underlying mechanisms are still largely unknown. Our results show that the rhizosphere of beans (<em>Phaseolus vulgaris</em>) grown in Haplic Cambisol serves as a natural reservoir for actinomycetes that can inactivate the semi-synthetic macrolide antibiotic clarithromycin. However, the application of WWTP products as fertilizer or as alternative water source for irrigation led to a significant increase in the abundance and taxonomic diversity of CLR-degrading actinomycetes. This could be due to the selection pressure exerted by the macrolide concentrations exceeding the PNEC<sub>res</sub> in the matrices used, treated wastewater and biosolid. Through a comprehensive assessment of the rhizosphere microbiome and soil chemistry, we found that fertilisation with both biosolids altered the bacterial community structure and dissolved phosphorus content, while composted biosolid also increased soil microbial biomass. Although we did not find significant differences in either the diversity or the structure of the rhizosphere bacterial community after the sole application of treated wastewater compared to the control, the isolate-level approach provided insight into the fundamental changes at the functional level - the abundance and phylogenetic representation of CLR-degrading actinomycetes increased substantially. Our study emphasises the importance of employing both culture-dependent and culture-independent approaches in studies focused on antibiotic degradation. Furthermore, while degradation of antibiotics is a natural detoxification of soil bacteria, it is also a mechanism of antibiotic resistance. This dual role needs to be further investigated to fully understand its impact and assess its potential for bioremediation.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Chemicals used</h3>
<p id="Par30">Clarithromycin (CLR) with a purity of more than 98.0% was purchased from Tokyo Chemical Industry Co., Ltd. (Tokyo, Japan). The stock solution (2 mg mL<sup>− 1</sup>) was prepared in LC-MS grade ethanol (Merck, Darmstadt, Germany) and filter sterilized (nylon membrane, Thermo Scientific, NY, USA). For the purpose of degradation experiments with bacterial consortia in a growth medium containing CLR as the sole carbon source, CLR was dissolved directly in the culture medium at a final concentration of 1 mg L<sup>− 1</sup> and the medium was then filter sterilized. Nalidixic acid (Merck, Darmstadt, Germany), dissolved in deionized water, and cycloheximide (Merck, Darmstadt, Germany) dissolved in dimethyl sulfoxide (Merck, Darmstadt, Germany) were used to prepare agar media for the isolation of actinomyceter to avoid the growth of fast-growing bacteria nd fungi, respectively. Ultra-pure water (Milli-Q<sup>®</sup> Direct 8, Merck, Germany) and acetonitrile (Merck, LC-MS grade) acidified with formic acid (Merck, LC-MS grade) were used for LC-MS analysis.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Experimental set-up</h3>
<p id="Par31">The two-year experiment was conducted at the WWTP in Hrdějovice (Czech Republic) with a current load of 112,000 population equivalents. In spring 2021, raised beds (1.5 × 1.0 × 0.8 m, i.e. width x depth x height) were constructed as previously described<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> and filled with Haplic Cambisol developed on paragneiss (sandy loam soil). We managed the soil beds in four different ways: (a) CMW was irrigated with tap water (control), (b) CME was irrigated with treated wastewater (WWTP effluent), (c) CMC contained the wet equivalent of 133 t of composted biosolids per hectare and was irrigated with tap water, and (d) CMB contained the wet equivalent of 44 t of stabilized biosolids per hectare and was irrigated with tap water. The experiment started in 2021 and continued in 2022 with one modification: Irrigation with tap water was replaced by irrigation with wastewater for CMC. The amendments of composted biosolids and biosolids were made in the first half of April in 2021 and 2022 (in equal amounts each year). The compost, which contained 15–30% (by volume) of biosolids, was obtained from the composting company DIWENDYS s.r.o. (České Budějovice, Czech Republic). Sanitized and stabilized biosolids (thermophilic anaerobic digestion) were obtained directly from WWTP. The chemical and microbial parameters of the soil were previously described by<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. In 2021, the beds were cropped with maize (<em>Zea mays</em> L. convar. Saccharata Koern), in 2022 with potatoes (<em>Solanum tuberosum</em> L.; April – July), soybeans (<em>Glycine max</em> L.; April – August) and beans (<em>Phaseolus vulgaris</em> L.; July – October). The experimental setup is shown schematically in Supplementary Figure S5.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Soil and rhizosphere sampling</h3>
<p id="Par32">Bulk soil and rhizosphere samples of <em>P. vulgaris</em> were collected at the time of harvest in October 2022. Bulk soil samples were taken from the topsoil (0–10 cm depth) in four replicates in an approximate amount of 0.5 kg and immediately transported to the laboratory on ice. For the collection of the rhizosphere, the roots of <em>P. vulgaris</em> including the soil and rhizosphere were excavated and transported on ice to the laboratory, where the bulk soil was removed and only the rhizosphere was collected (in three replicates). Both bulk soil and rhizosphere were then separately homogenized by sieving through a 2-mm sieve and stored at -76 °C for further molecular analyses. The bulk soil samples were immediately analysed for nutrients in the soil water extract and dry matter content. Samples of the rhizosphere for microbial cultivation analyses and degradation experiments were used immediately. The bulk soil used for the preparation of the enrichment medium (see below) was stored at 4 °C.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Soil characteristics</h3>
<p id="Par33">Dissolved organic carbon, nitrogen and phosphorous (DOC, DN and DP, respectively) in soil samples were determined as previously described by<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Briefly, 10 g of homogenised bulk soil sample were shaken in 50 mL of deionized water (45 min) and centrifuged (1370xg for 30 min). The supernatant was filtered through 0.45 μm membrane (Whatman, ME 25/21 ST). DOC and DN in the filtrate were measured using a TOC analyser (model TOC-LCPH/CPN, Shimadzu), DP was determined spectrophotometrically<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Microbial biomass carbon (C<sub>mic</sub>) and nitrogen (N<sub>mic</sub>) were analyzed by fumigation-extraction method as described in<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Bacterial colony forming units in rhizosphere and isolation of actinobacteria</h3>
<p id="Par34">Five grams of homogenized rhizosphere samples were suspended in 45 mL of 0.2% Calgon (sodium hexametaphosphate, Penta, Praha, Czech Republic) and sonicated for 4 min to detach the bacterial cells from the soil particles. The suspensions were serially diluted in sterile saline and plated on R2A agar (ready-to-use, Himedia, Mumbai, India) with or without selection of CLR (1 mg L<sup>− 1</sup>). The bacteria were cultivated at 28 °C. All media were additionally supplemented with nalidixic acid (25 mg L<sup>− 1</sup>) and cycloheximide (50 mg L<sup>− 1</sup>) to prevent the growth of fast-growing bacteria and fungi, respectively. All soil samples were analysed in triplicate. Filamentous-like bacteria were picked from the R2A agar plates supplemented with CLR after 6 or more days of cultivation and stored as spore suspensions in 15% (v/v) glycerol stocks at -76 °C for long-term preservation. The isolates have been deposited in the Culture Collection of Actinomycetes of the Biology Centre Collection of Organisms (BCCO, <a href="http://www.actinomycetes.bcco.cz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.actinomycetes.bcco.cz</a>).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Degradation experiment with rhizosphere-bacterial consortia (DG1)</h3>
<p id="Par35">As we expected the highest concentration of CLR in TWW, the first degradation experiment with freshly collected and homogenized rhizosphere was performed in triplicate with CMW and CME in October 2022 (here referred to as DG1). Glassware and mineral medium (MM) were prepared as described<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Experiments were performed in 100 mL Erlenmeyer flasks filled with 50 mL of MM and CLR as the sole carbon source at a final concentration of 0.6 mg C L<sup>− 1</sup>. The medium was prepared 1 day before the experiment, filtered with a 0.2 μm membrane and stored at 4 °C. To standardize the size of the inoculum, 5 g of rhizosphere was suspended in 45 mL of MM, sonicated for 2 min, and the total number of bacterial cells in the suspension was estimated by DAPI (4′, 6-diamidino-2-phenylindole) staining and microscopic counting<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The suspension was then diluted with sterile MM and used as an inoculum to reach a final concentration of 10<sup>5</sup> bacterial cells mL<sup>− 1</sup>. A sterile, non-inoculated MM with (MM + CLR) or without (MM0) supplementation of CLR served as abiotic controls. The experiment was carried out for 11 days (previously shown as sufficient length for degradation of CLR in liquid conditions<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> at 25 °C and in the dark in a rotary shaker (120 rpm). The number of bacterial cells was measured on day 3 and day 11 as described above. The number of CFU was estimated on day 6 by plating a serially diluted suspension on R2A agar supplemented with CLR (1 mg L<sup>− 1</sup>) and nalidixic acid (25 mg L<sup>− 1</sup>), colonies were read after 6 days of cultivation at 28 °C in the dark. Samples for the measurement of CLR concentrations were collected aseptically in a sterile laminar flow hood with a sterile syringe and needle on day 0 immediately after inoculation and at the end of the experiment (day 11).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Degradation experiment with enrichmed cultures (DG2)</h3>
<p id="Par36">The second degradation experiment required the preparation of enrichment cultures. Enrichment was performed by all management types (CMW, CME, CMC and CMB) in triplicate as follows: Five grams of freshly collected and homogenized rhizosphere was weighed into 100 mL Erlenmeyer flask and suspended in 45 mL MM<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The experiment was performed at 25 °C in a rotary shaker (120 rpm) in the dark with an initial CLR concentration of 0.01 mg L<sup>− 1</sup>. To acclimate microbial consortia CLR concentration was gradually increased over a period of 14 days by 0.124 mg L<sup>− 1</sup> up to final 1 mg L<sup>− 1</sup> of CLR after 4 months and continued simultaneously with the refreshment of the culture. The enrichment medium for the refreshing cultures was prepared by sterilization (autoclaving) of 5 g of bulk soil of the corresponding soil management, suspended in 45 mL of fresh MM. The cooled medium was supplemented with sterile CLR solution and inoculated with 5% (2.5 mL of the culture from previous 14 days-period. The microbial community obtained 7 days after reaching a CLR concentration of 1 mg L-1 is referred to herein as the enriched culture (denoted CMW-E, CME-E, CMC-E and CMB-E). To estimate the CFU in the enriched culture, 1 mL of the suspension was serially diluted and plated on the R2A agar supplemented with CLR (1 mg L<sup>− 1</sup>) and nalidixic acid (25 µg L<sup>− 1</sup>). The bacterial community structure of the enriched culture was assessed by 16S rRNA gene amplicon sequencing (see below).</p>
<p id="Par37">The enriched cultures served as inoculants for the second degradation experiment (here referred to as <strong>DG2</strong>), using the same experimental setup as described for DG1, with the only exception that the initial concentration of CLR as the sole carbon source in DG2 was increased to 1.64 mg L<sup>− 1</sup> which represent 1 mg C L<sup>− 1</sup>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">CLR-inactivation experiments using pure bacterial cultures</h3>
<p id="Par38">The bacterial isolates obtained from the rhizosphere were tested for their ability to inactivate CLR. Prior to testing, isolates were revived on yeast malt extract agar (ISP medium no. 2) and refreshed on oatmeal agar (ISP medium no. 3; both media preparation as described by<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> for 7 days at 28 °C.</p>
<p id="Par39">Initially, all isolates were tested on solid medium using the antibiotic modulation assay (here <strong>solid-AMA</strong>) described by<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> with minor modifications. Briefly, CLR (2 mg L<sup>− 1</sup>) was applied to the centre of Mueller-Hinton agar plates (Dulab, Czech Republic; 9 cm diameter) in a volume of 5 µl and soaked for 2 h. Then a loop of the fully grown strain of interest was inoculated from the edge to the centre of the agar plate and cultured for 72 h at 28 °C in the dark. To demonstrate the inactivation ability of the test strain, agar plates were overlaid with an overnight culture of <em>Staphylococcus aureus</em> DSM 346 (100 µl of 0.5 McFarland) resuspended in 10 mL of Mueller-Hinton soft agar (MH broth, Carl Roth, Germany; 1% agar, VWR, Pennsylvania, USA). The soft agar was cooled to 42 °C before inoculation. The mixture was poured evenly onto the plate and cultured at 37 °C for 21 h. Each isolate was tested in two replicates. The visible decrease in the zone of inhibition of <em>S. aureus</em> near the tested culture, indicating a decrease in CLR concentration, was assumed to be a CLR-positive inactivation reaction. The intensity of inactivation was categorised as strong (&gt; 15 mm), medium (8–14 mm), low (3–7 mm) or ambiguous (0–3 mm) based on the decrease in the inhibition zone diameter near the tested culture.</p>
<p id="Par40">Secondly, the positive results of the solid-AMA were verified in a nutrient-rich liquid medium (MH broth) supplemented with CLR (1 mg L<sup>− 1</sup>) (referred to here as <strong>liquid-AMA).</strong> The experiment was performed in 100 mL Erlenmeyer flasks covered with cellulose stoppers and aluminium foil in a final volume of 50 mL. The 72-hour pre-culture grown in MH broth served as inoculum. 200 µL of the preculture (washed twice with sterile saline), diluted to 2 McFarland scale values, was inoculated into the medium and cultured for up to 5 days at 28 °C in a rotary shaker (140 rpm) in the dark. Each isolate and the abiotic control were tested in triplicate. On day five, the spent medium was filter-sterilized using a regenerated cellulose membrane with a pore size of 0.2 μm (Chromafil, Macherey-Nagel, Düren, Germany). The spent medium was applied to a sterile paper disc (50 µL), followed by the disc diffusion method performed on MH agar (Dulab, Czech Republic). The development of the inhibition zone of the indicator strain formed by the CLR residue in the spent medium was compared with the inhibition of CLR in the one of abiotic control. If a decrease in the diameter of the inhibition zone was observed compared to the control, the CLR was inactivated by the given culture. <em>Bacillus subtilis</em> CCM 1718 served as the indicator strain here, as it formed larger inhibition zones with easily readable results. The results of the selected strains that showed the highest CLR inactivation and belonged to different clade or soil management (see below) were verified by LC-HRMS (spent medium collected on day 0 and 5 of cultivation).</p>
<p id="Par41">To evaluate and exclude the effects of sorption of CLR on bacterial cells (false-positive results), a biomass sorption control was performed in triplicate as follows: 5-day-old cultures were autoclaved and then subjected to liquid-AMA assay as described above. The inhibition zone of indicator strain formed by CLR residue in the spent medium of tested isolates, identical to the inhibition zone of abiotic control, was assumed as not-sorbed CLR. The representative strains selected for the biomass sorption test were chosen based on their growth morphology in the liquid medium (cloudy appearance or growth in clumps; 4 isolates in total).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Analytics of clarithromycin in degradation experiments</h3>
<p id="Par42">The concentrations of macrolide compounds (CLR, AZI and ERY) in complex samples (treated wastewater, tap water, Cambisol, composted biosolid, biosolid, soil in the beds and water drained from the beds) were analysed by liquid chromatography connected with high-resolution mass spectrometry (LC/HRMS) using the methods described in detail in our previous publication<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>.</p>
<p id="Par43">Samples from the degradation experiments DG1 and DG2 as well as liquid-AMA were immediately filter sterilized through a regenerated cellulose membrane with a pore size of 0.2 μm (Chromafil, Macherey-Nagel, Duren, Germany) and stored at -20 °C. The CLR concentration was quantified by LC-MS as follows: 5 ng of an internal standard (Clarithromycin-D3, Toronto Research Chemicals, Canada) was added to 500 µL of the sample. The samples were analysed by HPLC with MS/MS detection (TSQ Quantiva, Thermo Fisher Scientific). The separation of the analyte was performed at Arion polar C18 column (Chromservis, Czech Republic) by the gradient of acetonitrile in water (LC-MS grade, Merck, Darmstadt, both acidified with 0.1% of formic acid). Detailed information on the analysis can be found in Supplementary Table S7. Limits of quantifications (LOQ, ng mL<sup>− 1</sup>) of macrolides in all the matrices are provided in Supplementary Table S8.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Bacterial pure cultures identification and phylogeny tree construction</h3>
<p id="Par44">All bacterial isolates with positive or ambiguous response in solid-AMA were identified by 16S rRNA gene similarity. DNA was extracted from the bacterial biomass using the Nucleospin Microbial DNA Kit (Macherey-Nagel, Düren, Germany) according to the manufacturer’s instructions for Gram-positive bacteria. PCR amplification was performed as previously described<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. Sequences were edited and assembled in Geneious v 8.1.9 (Biomatters, Auckland, New Zealand) and compared with GenBank sequences of type strains using the Basic Local Alignment Search Tool<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> to identify the most closely related species. To exclude identical strains from the following analytical methods, we compared the genetic fingerprint of selected isolated from the same soil management and assigned to the same species (clade) using box-PCR<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>.</p>
<p id="Par45">The phylogenetic tree was constructed in Geneious using the ClustalW alignment and the GTR + I + G model using PhyML. The final visualization was done in iTOL v. 6.9.1<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. The reference sequences for the phylogenetic tree were downloaded from Genbank and are listed in Supplementary Table S9.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Prokaryotic community analyses by 16S rRNA gene amplicon sequencing</h3>
<p id="Par46">Genomic DNA from the rhizosphere and enriched cultures was extracted using the NucleoSpin<sup>®</sup>Soil Kit (Macherey-Nagel, Düren, Germany) according to the manufacturer’s instructions with a final elution volume of 60 µL. The quality and concentration of the DNA was checked using the NanoDropTM One UV-Vis spectrophotometer (ThermoFisher Scientific, Waltham, USA). The extracted DNA was stored at –18 °C.</p>
<p id="Par47">For bacterial community structure analysis, the hypervariable region V4 of the bacterial 16S rRNA gene was amplified using primer pair 515F (5′-GTGCCAGCMGCCGCGGTAA-3′) and 806R (5′-GGACTACHVGGGTWTCTAAT-3′) designed by<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. PCR reactions were performed in duplicate using FastStart PCR Master (Roche, Basel, Switzerland) and Bovine Serum Albumin (BSA, final concentration 0.6 mg mL<sup>− 1</sup>; ThermoFisher Scientific, Waltham, USA) in the final volume of 25 µL. The reaction conditions were as follows: initial denaturation 94 °C for 5 min, 28 cycles of 94 °C for 45 s, 50 °C for 45 s, 72 °C for 45 s and final elongation at 72 °C for 10 min. The purity and amount of DNA in the PCR product was checked on an agarose gel (2%). Pooled replicates were then submitted for paired-end sequencing (2 × 153 bp) using an Illumina MiniSeq platform (UIL, Chicago, USA).</p>
<p id="Par48">Raw paired-end sequences were demultiplexed by the sequencing facility. The sequences were filtered (low quality sequences with a maximum expected error of 2), denoised, merged and analysed for the presence of chimeric sequences with final Amplicon Sequencing Variant (ASV) clustering using the DADA2 pipeline<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>,<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. ASV taxonomy was assigned using the SILVA 16S rRNA gene database (v.138.1)<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup> with a confidence threshold of 99% using the RDP classifier<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par49">The effects of management on soil chemical properties and the number of CFU from the rhizosphere and degradation experiments were evaluated by one-way ANOVA using the <em>aov</em> function of R<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. The effects of soil management and cultivation day in DG1 and DG2 on the number of bacterial cells and the effects of management and soil origin (either rhizosphere or enrichment) on the α-diversity of the respective microbiome were evaluated by two-way ANOVA. If ANOVA results were significant, a pairwise post-hoc comparison was performed with Tukey’s adjustment of p-values using the TukeyHSD function. Data were checked for normal distribution using the Shapiro-Wilk test and log-transformed where necessary.</p>
<p id="Par50">Soil microbial community data were analysed in R using phyloseq v.1.44.0<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. Alpha diversity was estimated using the Chao1 richness index and the Shannon index<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup> from datasets rarefied to minimal sequencing depth (2742 sequences). The effects of soil management and microbiome origin (either rhizosphere or enrichment) on both indices were assessed by a two-way ANOVA of logarithmized values, followed by a pairwise post-hoc comparison with Tukey adjustment of p-values. The effect of soil management on bacterial communities were visualized using non-metric multidimensional scaling (NMDS) of Bray-Curtis dissimilarities of square-root transformed ASV relative abundances. To determine the bacterial families that were significantly affected by soil management, differential gene expression analysis (DESeq2)<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup> was used on non-rarefied and untransformed datasets. All graphical outputs were generated using the ggplot2 package<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup> in the R environment (v. 4.3.1)<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. All statistical tests were evaluated at significance level of α = 0.05.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373788/bin/41598_2025_14953_MOESM1_ESM.pptx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (758.9KB, pptx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373788/bin/41598_2025_14953_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (56.7KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors thank Kateřina Petříčková from the Institute of Immunology and Microbiology, 1st Faculty of Medicine, Charles University in Prague, for providing culture stocks of <em>Staphylococcus aureus</em> DSM 346 and <em>Bacillus subtilis</em> CCM 1718; and Jiří Petrásek and Veronika Jílková from the Laboratory of Soil Organic Matter, Institute of Soil Biology and Biogeochemistry, BC CAS for chemical analysis. ČEVAK, p.l.c company and to city of České Budějovice are thanked for the possibility to experiment in the WWTP area. Aleš Klement, Miroslav Fér, and Antonín Nikodým from the Czech University of Life Sciences are thanked for the construction of the soil mesocosms and for controlling the experiment.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.K. – conceptualisation, investigation, methodology, formal analysis, visualisation, writing – original draft; K.G. – data curation, formal analysis, writing – review and editing; H.Š. – data curation, formal analysis; A.V.S. – data curation, formal analysis; M.P. – formal analysis, methodology; R.G. – investigation, formal analysis, writing – review and editing; R.K. – funding acquisition, writing – review and editing; A.C. – data curation, investigation, writing – original draft.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data generated or analysed during this study are included in this published article (and its Supplementary Information files). The 16S rRNA metagenomic sequence data obtained from microbial consortia have been deposited in the European Nucleotide Archive (ENA) at EMBL-EBI under accession number PRJEB75333 and are available at the following URL: https://www.ebi.ac.uk/ena/browser/view/PRJEB75333. 16S rRNA sequences of actinomycete isolates obtained in this study have been deposited in the GenBank under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PQ423591:PQ423632%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ423591-PQ423632</a> (Supplementary Table S9).</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par51">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Chieb, M. &amp; Gachomo, E. W. The role of plant growth promoting rhizobacteria in plant drought stress responses. <em>BMC Plant. Biol.</em><strong>23</strong>, 407 (2023).
</cite> [<a href="https://doi.org/10.1186/s12870-023-04403-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10464363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37626328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chieb,%20M.%20&amp;%20Gachomo,%20E.%20W.%20The%20role%20of%20plant%20growth%20promoting%20rhizobacteria%20in%20plant%20drought%20stress%20responses.%20BMC%20Plant.%20Biol.23,%20407%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Eichmann, R., Richards, L. &amp; Schäfer, P. Hormones as go-betweens in plant Microbiome assembly. <em>Plant J.</em><strong>105</strong>, 518–541 (2021).
</cite> [<a href="https://doi.org/10.1111/tpj.15135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8629125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33332645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eichmann,%20R.,%20Richards,%20L.%20&amp;%20Sch%C3%A4fer,%20P.%20Hormones%20as%20go-betweens%20in%20plant%20Microbiome%20assembly.%20Plant%20J.105,%20518%E2%80%93541%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Li, J., Wang, C., Liang, W. &amp; Liu, S. Rhizosphere microbiome: the emerging barrier in Plant-Pathogen interactions. <em>Front Microbiol</em><strong>12</strong>, (2021).</cite> [<a href="https://doi.org/10.3389/fmicb.2021.772420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8586421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34777326/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Wang, J. et al. A review on microorganisms in constructed wetlands for typical pollutant removal: species, function, and diversity. <em>Front Microbiol</em><strong>13</strong>, 845725 (2022).</cite> [<a href="https://doi.org/10.3389/fmicb.2022.845725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9016276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35450286/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Khasawneh, O. F. S. &amp; Palaniandy, P. Occurrence and removal of pharmaceuticals in wastewater treatment plants. <em>Process Saf. Environ. Prot.</em><strong>150</strong>, 532–556 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Khasawneh,%20O.%20F.%20S.%20&amp;%20Palaniandy,%20P.%20Occurrence%20and%20removal%20of%20pharmaceuticals%20in%20wastewater%20treatment%20plants.%20Process%20Saf.%20Environ.%20Prot.150,%20532%E2%80%93556%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Vodyanitskii, Y. N. &amp; Yakovlev, A. S. Contamination of soils and groundwater with new organic micropollutants: A review. <em>Eurasian Soil. Sci.</em><strong>49</strong>, 560–569 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Vodyanitskii,%20Y.%20N.%20&amp;%20Yakovlev,%20A.%20S.%20Contamination%20of%20soils%20and%20groundwater%20with%20new%20organic%20micropollutants:%20A%20review.%20Eurasian%20Soil.%20Sci.49,%20560%E2%80%93569%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Baz-Lomba, J. A. et al. Comparison of pharmaceutical, illicit drug, alcohol, nicotine and caffeine levels in wastewater with sale, seizure and consumption data for 8 European cities. <em>BMC Public. Health</em>. <strong>16</strong>, 1035 (2016).
</cite> [<a href="https://doi.org/10.1186/s12889-016-3686-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5045646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27716139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baz-Lomba,%20J.%20A.%20et%20al.%20Comparison%20of%20pharmaceutical,%20illicit%20drug,%20alcohol,%20nicotine%20and%20caffeine%20levels%20in%20wastewater%20with%20sale,%20seizure%20and%20consumption%20data%20for%208%20European%20cities.%20BMC%20Public.%20Health.%2016,%201035%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>WHO. <em>WHO’s List of Medically Important Antimicrobials: A Risk Management Tool for Mitigating Antimicrobial Resistance Due To Non-Human Use</em> (Geneva, 2024).</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>European Centre for Disease Prevention and Control. Antimicrobial consumption in the EU/EEA (ESAC-Net) - Annual Epidemiological Report for 2022. Stockholm: ECDC. (2023).</cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>Zhao, R. et al. Deciphering of microbial community and antibiotic resistance genes in activated sludge reactors under high selective pressure of different antibiotics. <em>Water Res.</em><strong>151</strong>, 388–402 (2019).
</cite> [<a href="https://doi.org/10.1016/j.watres.2018.12.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30616051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20R.%20et%20al.%20Deciphering%20of%20microbial%20community%20and%20antibiotic%20resistance%20genes%20in%20activated%20sludge%20reactors%20under%20high%20selective%20pressure%20of%20different%20antibiotics.%20Water%20Res.151,%20388%E2%80%93402%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Rodriguez-Mozaz, S. et al. Antibiotic residues in final effluents of European wastewater treatment plants and their impact on the aquatic environment. <em>Environ. Int.</em><strong>140</strong>, 105733 (2020).
</cite> [<a href="https://doi.org/10.1016/j.envint.2020.105733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32353669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodriguez-Mozaz,%20S.%20et%20al.%20Antibiotic%20residues%20in%20final%20effluents%20of%20European%20wastewater%20treatment%20plants%20and%20their%20impact%20on%20the%20aquatic%20environment.%20Environ.%20Int.140,%20105733%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Bielen, A. et al. Negative environmental impacts of antibiotic-contaminated effluents from pharmaceutical industries. <em>Water Res.</em><strong>126</strong>, 79–87 (2017).
</cite> [<a href="https://doi.org/10.1016/j.watres.2017.09.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28923406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bielen,%20A.%20et%20al.%20Negative%20environmental%20impacts%20of%20antibiotic-contaminated%20effluents%20from%20pharmaceutical%20industries.%20Water%20Res.126,%2079%E2%80%9387%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Ivanová, L. et al. Pharmaceuticals and illicit drugs – A new threat to the application of sewage sludge in agriculture. <em>Sci. Total Environ.</em><strong>634</strong>, 606–615 (2018).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2018.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29635203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ivanov%C3%A1,%20L.%20et%20al.%20Pharmaceuticals%20and%20illicit%20drugs%20%E2%80%93%20A%20new%20threat%20to%20the%20application%20of%20sewage%20sludge%20in%20agriculture.%20Sci.%20Total%20Environ.634,%20606%E2%80%93615%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Yang, L. et al. Application of biosolids drives the diversity of antibiotic resistance genes in soil and lettuce at harvest. <em>Soil. Biol. Biochem.</em><strong>122</strong>, 131–140 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yang,%20L.%20et%20al.%20Application%20of%20biosolids%20drives%20the%20diversity%20of%20antibiotic%20resistance%20genes%20in%20soil%20and%20lettuce%20at%20harvest.%20Soil.%20Biol.%20Biochem.122,%20131%E2%80%93140%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Bengtsson-Palme, J. et al. Elucidating selection processes for antibiotic resistance in sewage treatment plants using metagenomics. <em>Sci. Total Environ.</em><strong>572</strong>, 697–712 (2016).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2016.06.228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27542633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bengtsson-Palme,%20J.%20et%20al.%20Elucidating%20selection%20processes%20for%20antibiotic%20resistance%20in%20sewage%20treatment%20plants%20using%20metagenomics.%20Sci.%20Total%20Environ.572,%20697%E2%80%93712%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Wolters, B. et al. Biosolids for safe land application: does wastewater treatment plant size matters when considering antibiotics, pollutants, microbiome, mobile genetic elements and associated resistance genes? <em>Environ. Microbiol.</em><strong>24</strong>, 1573–1589 (2022).
</cite> [<a href="https://doi.org/10.1111/1462-2920.15938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9306954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35192222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolters,%20B.%20et%20al.%20Biosolids%20for%20safe%20land%20application:%20does%20wastewater%20treatment%20plant%20size%20matters%20when%20considering%20antibiotics,%20pollutants,%20microbiome,%20mobile%20genetic%20elements%20and%20associated%20resistance%20genes?%20Environ.%20Microbiol.24,%201573%E2%80%931589%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Bengtsson-Palme, J. &amp; Larsson, D. G. J. Concentrations of antibiotics predicted to select for resistant bacteria: proposed limits for environmental regulation. <em>Environ. Int.</em><strong>86</strong>, 140–149 (2016).
</cite> [<a href="https://doi.org/10.1016/j.envint.2015.10.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26590482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bengtsson-Palme,%20J.%20&amp;%20Larsson,%20D.%20G.%20J.%20Concentrations%20of%20antibiotics%20predicted%20to%20select%20for%20resistant%20bacteria:%20proposed%20limits%20for%20environmental%20regulation.%20Environ.%20Int.86,%20140%E2%80%93149%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Adriaenssens, N. et al. Consumption of macrolides, Lincosamides and streptogramins in the community, European union/european economic area, 1997–2017. <em>J. Antimicrob. Chemother.</em><strong>76</strong>, 30–36 (2021).
</cite> [<a href="https://doi.org/10.1093/jac/dkab175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8314107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34312653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adriaenssens,%20N.%20et%20al.%20Consumption%20of%20macrolides,%20Lincosamides%20and%20streptogramins%20in%20the%20community,%20European%20union/european%20economic%20area,%201997%E2%80%932017.%20J.%20Antimicrob.%20Chemother.76,%2030%E2%80%9336%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Baumann, M. et al. Aquatic toxicity of the macrolide antibiotic clarithromycin and its metabolites. <em>Chemosphere</em><strong>120</strong>, 192–198 (2015).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2014.05.089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25051235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baumann,%20M.%20et%20al.%20Aquatic%20toxicity%20of%20the%20macrolide%20antibiotic%20clarithromycin%20and%20its%20metabolites.%20Chemosphere120,%20192%E2%80%93198%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Booth, A., Aga, D. S. &amp; Wester, A. L. Retrospective analysis of the global antibiotic residues that exceed the predicted no effect concentration for antimicrobial resistance in various environmental matrices. <em>Environ. Int.</em><strong>141</strong>, 105796 (2020).
</cite> [<a href="https://doi.org/10.1016/j.envint.2020.105796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32422499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Booth,%20A.,%20Aga,%20D.%20S.%20&amp;%20Wester,%20A.%20L.%20Retrospective%20analysis%20of%20the%20global%20antibiotic%20residues%20that%20exceed%20the%20predicted%20no%20effect%20concentration%20for%20antimicrobial%20resistance%20in%20various%20environmental%20matrices.%20Environ.%20Int.141,%20105796%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Monahan, C., Morris, D., Nag, R. &amp; Cummins, E. Risk ranking of macrolide antibiotics – Release levels, resistance formation potential and ecological risk. <em>Sci. Total Environ.</em><strong>859</strong>, 160022 (2023).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2022.160022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36368382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Monahan,%20C.,%20Morris,%20D.,%20Nag,%20R.%20&amp;%20Cummins,%20E.%20Risk%20ranking%20of%20macrolide%20antibiotics%20%E2%80%93%20Release%20levels,%20resistance%20formation%20potential%20and%20ecological%20risk.%20Sci.%20Total%20Environ.859,%20160022%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wilkinson, J. L. et al. Pharmaceutical pollution of the world ’ s rivers. 119, 1–10 (2022).</cite> [<a href="https://doi.org/10.1073/pnas.2113947119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8872717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35165193/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Yi, X., Lin, C., Ong, E. J. L., Wang, M. &amp; Zhou, Z. Occurrence and distribution of trace levels of antibiotics in surface waters and soils driven by non-point source pollution and anthropogenic pressure. <em>Chemosphere</em><strong>216</strong>, 213–223 (2019).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2018.10.087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30368086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yi,%20X.,%20Lin,%20C.,%20Ong,%20E.%20J.%20L.,%20Wang,%20M.%20&amp;%20Zhou,%20Z.%20Occurrence%20and%20distribution%20of%20trace%20levels%20of%20antibiotics%20in%20surface%20waters%20and%20soils%20driven%20by%20non-point%20source%20pollution%20and%20anthropogenic%20pressure.%20Chemosphere216,%20213%E2%80%93223%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>García-Valverde, M. et al. Long-term effects on the agroecosystem of using reclaimed water on commercial crops. <em>Science Total Environment</em><strong>859</strong>, 160462 (2023).</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2022.160462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36435246/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Baranauskaite-Fedorova, I. &amp; Dvarioniene, J. Management of macrolide antibiotics (Erythromycin, clarithromycin and Azithromycin) in the environment: A case study of environmental pollution in Lithuania. <em>Water (Switzerland)</em>. <strong>15</strong>, 10 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Baranauskaite-Fedorova,%20I.%20&amp;%20Dvarioniene,%20J.%20Management%20of%20macrolide%20antibiotics%20(Erythromycin,%20clarithromycin%20and%20Azithromycin)%20in%20the%20environment:%20A%20case%20study%20of%20environmental%20pollution%20in%20Lithuania.%20Water%20(Switzerland).%2015,%2010%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Mejías, C., Martín, J., Santos, J. L., Aparicio, I. &amp; Alonso, E. Occurrence of pharmaceuticals and their metabolites in sewage sludge and soil: A review on their distribution and environmental risk assessment. <em>Trends Environ. Anal. Chem.</em><strong>30</strong>, e00125 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mej%C3%ADas,%20C.,%20Mart%C3%ADn,%20J.,%20Santos,%20J.%20L.,%20Aparicio,%20I.%20&amp;%20Alonso,%20E.%20Occurrence%20of%20pharmaceuticals%20and%20their%20metabolites%20in%20sewage%20sludge%20and%20soil:%20A%20review%20on%20their%20distribution%20and%20environmental%20risk%20assessment.%20Trends%20Environ.%20Anal.%20Chem.30,%20e00125%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kodešová, R. et al. Pharmaceuticals’ sorptions relative to properties of thirteen different soils. <em>Sci. Total Environ.</em><strong>511</strong>, 435–443 (2015).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2014.12.088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25569579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kode%C5%A1ov%C3%A1,%20R.%20et%20al.%20Pharmaceuticals%E2%80%99%20sorptions%20relative%20to%20properties%20of%20thirteen%20different%20soils.%20Sci.%20Total%20Environ.511,%20435%E2%80%93443%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Rodríguez-López, L. et al. Clarithromycin as soil and environmental pollutant: Adsorption-desorption processes and influence of pH. <em>Environ Res</em><strong>233</strong>, 116520 (2023).</cite> [<a href="https://doi.org/10.1016/j.envres.2023.116520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37390951/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kodešová, R. et al. An analysis of the dissipation of pharmaceuticals under thirteen different soil conditions. <em>Sci. Total Environ.</em><strong>544</strong>, 369–381 (2016).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2015.11.085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26657382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kode%C5%A1ov%C3%A1,%20R.%20et%20al.%20An%20analysis%20of%20the%20dissipation%20of%20pharmaceuticals%20under%20thirteen%20different%20soil%20conditions.%20Sci.%20Total%20Environ.544,%20369%E2%80%93381%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Pan, M. &amp; Chu, L. M. Adsorption and degradation of five selected antibiotics in agricultural soil. <em>Sci. Total Environ.</em><strong>545–546</strong>, 48–56 (2016).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2015.12.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26745292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20M.%20&amp;%20Chu,%20L.%20M.%20Adsorption%20and%20degradation%20of%20five%20selected%20antibiotics%20in%20agricultural%20soil.%20Sci.%20Total%20Environ.545%E2%80%93546,%2048%E2%80%9356%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Shen, D. et al. The fate of erythromycin in soils and its effect on soil microbial community structure. <em>Sci. Total Environ.</em><strong>820</strong>, 153373 (2022).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2022.153373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35081411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20D.%20et%20al.%20The%20fate%20of%20erythromycin%20in%20soils%20and%20its%20effect%20on%20soil%20microbial%20community%20structure.%20Sci.%20Total%20Environ.820,%20153373%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Wei, L. et al. Transformation of erythromycin during secondary effluent soil aquifer recharging: removal contribution and degradation path. <em>J. Environ. Sci. (China)</em>. <strong>51</strong>, 173–180 (2017).
</cite> [<a href="https://doi.org/10.1016/j.jes.2016.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28115128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20L.%20et%20al.%20Transformation%20of%20erythromycin%20during%20secondary%20effluent%20soil%20aquifer%20recharging:%20removal%20contribution%20and%20degradation%20path.%20J.%20Environ.%20Sci.%20(China).%2051,%20173%E2%80%93180%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Topp, E., Renaud, J., Sumarah, M. &amp; Sabourin, L. Reduced persistence of the macrolide antibiotics erythromycin, clarithromycin and Azithromycin in agricultural soil following several years of exposure in the field. <em>Sci. Total Environ.</em><strong>562</strong>, 136–144 (2016).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2016.03.210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27096634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Topp,%20E.,%20Renaud,%20J.,%20Sumarah,%20M.%20&amp;%20Sabourin,%20L.%20Reduced%20persistence%20of%20the%20macrolide%20antibiotics%20erythromycin,%20clarithromycin%20and%20Azithromycin%20in%20agricultural%20soil%20following%20several%20years%20of%20exposure%20in%20the%20field.%20Sci.%20Total%20Environ.562,%20136%E2%80%93144%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Zhou, P. et al. Cellular metabolism network of Bacillus Thuringiensis related to erythromycin stress and degradation. <em>Ecotoxicol. Environ. Saf.</em><strong>160</strong>, 328–341 (2018).
</cite> [<a href="https://doi.org/10.1016/j.ecoenv.2018.05.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29857237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20P.%20et%20al.%20Cellular%20metabolism%20network%20of%20Bacillus%20Thuringiensis%20related%20to%20erythromycin%20stress%20and%20degradation.%20Ecotoxicol.%20Environ.%20Saf.160,%20328%E2%80%93341%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang, Y. S., Zheng, X. C., Hu, Q. W. &amp; Zheng, Y. G. Degradation of abamectin by newly isolated Stenotrophomonas maltophilia ZJB-14120 and characterization of its abamectin-tolerance mechanism. <em>Res. Microbiol.</em><strong>166</strong>, 408–418 (2015).
</cite> [<a href="https://doi.org/10.1016/j.resmic.2015.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25957243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20S.,%20Zheng,%20X.%20C.,%20Hu,%20Q.%20W.%20&amp;%20Zheng,%20Y.%20G.%20Degradation%20of%20abamectin%20by%20newly%20isolated%20Stenotrophomonas%20maltophilia%20ZJB-14120%20and%20characterization%20of%20its%20abamectin-tolerance%20mechanism.%20Res.%20Microbiol.166,%20408%E2%80%93418%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bhullar, K. et al. Antibiotic resistance is prevalent in an isolated cave Microbiome. <em>PLoS One</em>. <strong>7</strong>, 1–11 (2012).</cite> [<a href="https://doi.org/10.1371/journal.pone.0034953" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3324550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22509370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhullar,%20K.%20et%20al.%20Antibiotic%20resistance%20is%20prevalent%20in%20an%20isolated%20cave%20Microbiome.%20PLoS%20One.%207,%201%E2%80%9311%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Zhang, W., Qiu, L., Gong, A. &amp; Yuan, X. Isolation and characterization of a high-efficiency erythromycin A-degrading ochrobactrum sp. strain. <em>Mar. Pollut Bull.</em><strong>114</strong>, 896–902 (2017).
</cite> [<a href="https://doi.org/10.1016/j.marpolbul.2016.10.076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27863881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.,%20Qiu,%20L.,%20Gong,%20A.%20&amp;%20Yuan,%20X.%20Isolation%20and%20characterization%20of%20a%20high-efficiency%20erythromycin%20A-degrading%20ochrobactrum%20sp.%20strain.%20Mar.%20Pollut%20Bull.114,%20896%E2%80%93902%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Chenxi, W., Spongberg, A. L. &amp; Witter, J. D. Determination of the persistence of pharmaceuticals in biosolids using liquid-chromatography tandem mass spectrometry. <em>Chemosphere</em><strong>73</strong>, 511–518 (2008).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2008.06.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18674794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chenxi,%20W.,%20Spongberg,%20A.%20L.%20&amp;%20Witter,%20J.%20D.%20Determination%20of%20the%20persistence%20of%20pharmaceuticals%20in%20biosolids%20using%20liquid-chromatography%20tandem%20mass%20spectrometry.%20Chemosphere73,%20511%E2%80%93518%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Terzic, S. et al. Biotransformation of macrolide antibiotics using enriched activated sludge culture: kinetics, transformation routes and ecotoxicological evaluation. <em>J. Hazard. Mater.</em><strong>349</strong>, 143–152 (2018).
</cite> [<a href="https://doi.org/10.1016/j.jhazmat.2018.01.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29414746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Terzic,%20S.%20et%20al.%20Biotransformation%20of%20macrolide%20antibiotics%20using%20enriched%20activated%20sludge%20culture:%20kinetics,%20transformation%20routes%20and%20ecotoxicological%20evaluation.%20J.%20Hazard.%20Mater.349,%20143%E2%80%93152%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Devanshi, S., Shah, K. R., Arora, S. &amp; Saxena, S. Actinomycetes as an environmental scrubber. in Crude Oil (eds Abdel-Raouf, M. E. &amp; El-Keshawy, M. H.) (IntechOpen, Rijeka, doi:10.5772/intechopen.99187. (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Devanshi,%20S.,%20Shah,%20K.%20R.,%20Arora,%20S.%20&amp;%20Saxena,%20S.%20Actinomycetes%20as%20an%20environmental%20scrubber.%20in%20Crude%20Oil%20(eds%20Abdel-Raouf,%20M.%20E.%20&amp;%20El-Keshawy,%20M.%20H.)%20(IntechOpen,%20Rijeka,%20doi:10.5772/intechopen.99187.%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Özel Duygan, B. D., Gaille, C., Fenner, K. &amp; van der Meer, J. R. Assessing antibiotics biodegradation and effects at Sub-inhibitory concentrations by quantitative microbial community Deconvolution. <em>Front. Environ. Sci.</em><strong>9</strong>, 1–17 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?%C3%96zel%20Duygan,%20B.%20D.,%20Gaille,%20C.,%20Fenner,%20K.%20&amp;%20van%20der%20Meer,%20J.%20R.%20Assessing%20antibiotics%20biodegradation%20and%20effects%20at%20Sub-inhibitory%20concentrations%20by%20quantitative%20microbial%20community%20Deconvolution.%20Front.%20Environ.%20Sci.9,%201%E2%80%9317%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Ren, J. et al. Biodegradation of erythromycin by Delftia lacustris RJJ-61 and characterization of its erythromycin esterase. <em>J. Basic. Microbiol.</em><strong>61</strong>, 55–62 (2021).
</cite> [<a href="https://doi.org/10.1002/jobm.202000613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33332633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ren,%20J.%20et%20al.%20Biodegradation%20of%20erythromycin%20by%20Delftia%20lacustris%20RJJ-61%20and%20characterization%20of%20its%20erythromycin%20esterase.%20J.%20Basic.%20Microbiol.61,%2055%E2%80%9362%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Sauvêtre, A. et al. Enrichment of endophytic actinobacteria in roots and rhizomes of miscanthus × giganteus plants exposed to diclofenac and sulfamethoxazole. <em>Environ. Sci. Pollut. Res.</em><strong>27</strong>, 11892–11904 (2020).</cite> [<a href="https://doi.org/10.1007/s11356-020-07609-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31981026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sauv%C3%AAtre,%20A.%20et%20al.%20Enrichment%20of%20endophytic%20actinobacteria%20in%20roots%20and%20rhizomes%20of%20miscanthus%20%C3%97%20giganteus%20plants%20exposed%20to%20diclofenac%20and%20sulfamethoxazole.%20Environ.%20Sci.%20Pollut.%20Res.27,%2011892%E2%80%9311904%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Mawang, C. I., Azman, A. S., Fuad, A. S. M. &amp; Ahamad, M. Actinobacteria: an eco-friendly and promising technology for the bioaugmentation of contaminants. <em>Biotechnol. Rep.</em><strong>32</strong>, e00679 (2021).</cite> [<a href="https://doi.org/10.1016/j.btre.2021.e00679" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8503819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34660214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mawang,%20C.%20I.,%20Azman,%20A.%20S.,%20Fuad,%20A.%20S.%20M.%20&amp;%20Ahamad,%20M.%20Actinobacteria:%20an%20eco-friendly%20and%20promising%20technology%20for%20the%20bioaugmentation%20of%20contaminants.%20Biotechnol.%20Rep.32,%20e00679%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Ajibola, A. S. &amp; Zwiener, C. Occurrence and risk assessment of antibiotic residues in sewage sludge of two Nigerian hospital wastewater treatment plants. <em>Water Air Soil. Pollut</em>. <strong>233</strong>, 405 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ajibola,%20A.%20S.%20&amp;%20Zwiener,%20C.%20Occurrence%20and%20risk%20assessment%20of%20antibiotic%20residues%20in%20sewage%20sludge%20of%20two%20Nigerian%20hospital%20wastewater%20treatment%20plants.%20Water%20Air%20Soil.%20Pollut.%20233,%20405%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>European Commission. Proposal for a Directive on Urban Wastewater Treatment. Document 52022PC0541. vol. 0345. (2022).</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Chroňáková, A. et al. Arenosol epieuric and haplic Cambisol show a similar level of resilience of microbial communities when irrigated with treated wastewater in a temperate climate. <em>Appl. Soil. Ecol.</em><strong>204</strong>, 105693 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chro%C5%88%C3%A1kov%C3%A1,%20A.%20et%20al.%20Arenosol%20epieuric%20and%20haplic%20Cambisol%20show%20a%20similar%20level%20of%20resilience%20of%20microbial%20communities%20when%20irrigated%20with%20treated%20wastewater%20in%20a%20temperate%20climate.%20Appl.%20Soil.%20Ecol.204,%20105693%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Datel, J. V. &amp; Hrabankova, A. Pharmaceuticals load in the Svihov water reservoir (Czech Republic) and impacts on quality of treated drinking water. <em>Water (Switzerland)</em><strong>12</strong>, 1387 (2020).</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Medina-Pérez, G. et al. Bacterial Communities in the Rhizosphere of Common Bean Plants (Phaseolus vulgaris L.) Grown in an Arable Soil Amended with TiO2 Nanoparticles. <em>Agronomy</em> 14, (2024).</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Rahube, T. O. et al. Impact of fertilizing with Raw or anaerobically digested sewage sludge on the abundance of antibiotic-resistant coliforms, antibiotic resistance genes, and pathogenic bacteria in soil and on vegetables at harvest. <em>Appl. Environ. Microbiol.</em><strong>80</strong>, 6898–6907 (2014).
</cite> [<a href="https://doi.org/10.1128/AEM.02389-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4249014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25172864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rahube,%20T.%20O.%20et%20al.%20Impact%20of%20fertilizing%20with%20Raw%20or%20anaerobically%20digested%20sewage%20sludge%20on%20the%20abundance%20of%20antibiotic-resistant%20coliforms,%20antibiotic%20resistance%20genes,%20and%20pathogenic%20bacteria%20in%20soil%20and%20on%20vegetables%20at%20harvest.%20Appl.%20Environ.%20Microbiol.80,%206898%E2%80%936907%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Herbrík, A. et al. A human Lung-Associated streptomyces sp. TR1341 produces various secondary metabolites responsible for virulence, cytotoxicity and modulation of immune response. <em>Front. Microbiol.</em><strong>10</strong>, 1–17 (2020).</cite> [<a href="https://doi.org/10.3389/fmicb.2019.03028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6978741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32010093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herbr%C3%ADk,%20A.%20et%20al.%20A%20human%20Lung-Associated%20streptomyces%20sp.%20TR1341%20produces%20various%20secondary%20metabolites%20responsible%20for%20virulence,%20cytotoxicity%20and%20modulation%20of%20immune%20response.%20Front.%20Microbiol.10,%201%E2%80%9317%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kotrbová, L. et al. Evaluation and comparison of antibiotic susceptibility profiles of streptomyces spp. From clinical specimens revealed common and region-dependent resistance patterns. <em>Sci. Rep.</em><strong>12</strong>, 1–12 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-13094-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9174267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35672429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kotrbov%C3%A1,%20L.%20et%20al.%20Evaluation%20and%20comparison%20of%20antibiotic%20susceptibility%20profiles%20of%20streptomyces%20spp.%20From%20clinical%20specimens%20revealed%20common%20and%20region-dependent%20resistance%20patterns.%20Sci.%20Rep.12,%201%E2%80%9312%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>López, J. L. et al. Growth rate is a dominant factor predicting the rhizosphere effect. <em>ISME J.</em><strong>17</strong>, 1396–1405 (2023).
</cite> [<a href="https://doi.org/10.1038/s41396-023-01453-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10432406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37322285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez,%20J.%20L.%20et%20al.%20Growth%20rate%20is%20a%20dominant%20factor%20predicting%20the%20rhizosphere%20effect.%20ISME%20J.17,%201396%E2%80%931405%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Kim, Y. H. &amp; Cerniglia, C. E. Influence of erythromycin a on the microbial populations in aquaculture sediment microcosms. <em>Aquat. Toxicol.</em><strong>73</strong>, 230–241 (2005).
</cite> [<a href="https://doi.org/10.1016/j.aquatox.2005.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15935863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20Y.%20H.%20&amp;%20Cerniglia,%20C.%20E.%20Influence%20of%20erythromycin%20a%20on%20the%20microbial%20populations%20in%20aquaculture%20sediment%20microcosms.%20Aquat.%20Toxicol.73,%20230%E2%80%93241%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Golkar, T., Zielinski, M. &amp; Berghuis, A. M. Look and outlook on enzyme-mediated macrolide resistance. <em>Front. Microbiol.</em><strong>9</strong>, 1–15 (2018).
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.01942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6109786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30177927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Golkar,%20T.,%20Zielinski,%20M.%20&amp;%20Berghuis,%20A.%20M.%20Look%20and%20outlook%20on%20enzyme-mediated%20macrolide%20resistance.%20Front.%20Microbiol.9,%201%E2%80%9315%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Vester, B. &amp; Douthwaite, S. Macrolide resistance conferred by base substitutions in 23S rRNA. <em>Antimicrob. Agents Chemother.</em><strong>45</strong>, 1–12 (2001).
</cite> [<a href="https://doi.org/10.1128/AAC.45.1.1-12.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC90232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11120937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vester,%20B.%20&amp;%20Douthwaite,%20S.%20Macrolide%20resistance%20conferred%20by%20base%20substitutions%20in%2023S%20rRNA.%20Antimicrob.%20Agents%20Chemother.45,%201%E2%80%9312%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>European Committee on Antimicrobial Susceptibility Testing. Data from the EUCAST MIC distribution website, last accessedSeptember, <a href="https://www.eucast.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.eucast.org</a> (2022).</cite>
</li>
<li id="CR58">
<span class="label">58.</span><cite>CLSI. M24. Susceptibility Testing of Mycobacteria, and Other Aerobic Actinomycetes; Approved Standard-2nd Ed. CLSI guidlines vol. 31. (2011).</cite>
</li>
<li id="CR59">
<span class="label">59.</span><cite>Shepherd, M. D., Kharel, M. K., Bosserman, M. A. &amp; Rohr, J. Laboratory maintenance of streptomyces species. <em>Curr. Protoc. Microbiol.</em><strong>1–10</strong>10.1002/9780471729259.mc10e01s18 (2010).</cite> [<a href="https://doi.org/10.1002/9780471729259.mc10e01s18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2950629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20812215/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Quirós, L. M., Carbajo, R. J., Braña, A. F. &amp; Salas, J. A. Glycosylation of macrolide antibiotics. Purification and kinetic studies of a macrolide glycosyltransferase from streptomyces antibioticus. <em>J. Biol. Chem.</em><strong>275</strong>, 11713–11720 (2000).
</cite> [<a href="https://doi.org/10.1074/jbc.275.16.11713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10766792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quir%C3%B3s,%20L.%20M.,%20Carbajo,%20R.%20J.,%20Bra%C3%B1a,%20A.%20F.%20&amp;%20Salas,%20J.%20A.%20Glycosylation%20of%20macrolide%20antibiotics.%20Purification%20and%20kinetic%20studies%20of%20a%20macrolide%20glycosyltransferase%20from%20streptomyces%20antibioticus.%20J.%20Biol.%20Chem.275,%2011713%E2%80%9311720%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Dhindwal, P. et al. A neglected and emerging antimicrobial resistance gene encodes for a serine-dependent macrolide esterase. <em>Proc. Natl. Acad. Sci. U S A</em>. <strong>120</strong>, 15–17 (2023).</cite> [<a href="https://doi.org/10.1073/pnas.2219827120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9974460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36791107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dhindwal,%20P.%20et%20al.%20A%20neglected%20and%20emerging%20antimicrobial%20resistance%20gene%20encodes%20for%20a%20serine-dependent%20macrolide%20esterase.%20Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A.%20120,%2015%E2%80%9317%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Baquero, F., Coque, T. M. &amp; Martínez, J. L. Natural detoxification of antibiotics in the environment: A one health perspective. <em>Front. Microbiol.</em><strong>13</strong>, 1–11 (2022).</cite> [<a href="https://doi.org/10.3389/fmicb.2022.1062399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9730888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36504820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baquero,%20F.,%20Coque,%20T.%20M.%20&amp;%20Mart%C3%ADnez,%20J.%20L.%20Natural%20detoxification%20of%20antibiotics%20in%20the%20environment:%20A%20one%20health%20perspective.%20Front.%20Microbiol.13,%201%E2%80%9311%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Nor Amdan, N. A., Shahrulzamri, N. A. &amp; Hashim, R. Mohamad jamil, N. Understanding the evolution of macrolides resistance: A mini review. <em>J. Glob Antimicrob. Resist.</em><strong>38</strong>, 368–375 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jgar.2024.07.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39117142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nor%20Amdan,%20N.%20A.,%20Shahrulzamri,%20N.%20A.%20&amp;%20Hashim,%20R.%20Mohamad%20jamil,%20N.%20Understanding%20the%20evolution%20of%20macrolides%20resistance:%20A%20mini%20review.%20J.%20Glob%20Antimicrob.%20Resist.38,%20368%E2%80%93375%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Zhang, A. N. et al. An omics-based framework for assessing the health risk of antimicrobial resistance genes. <em>Nat. Commun.</em><strong>12</strong>, 1–11 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-25096-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8346589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34362925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20A.%20N.%20et%20al.%20An%20omics-based%20framework%20for%20assessing%20the%20health%20risk%20of%20antimicrobial%20resistance%20genes.%20Nat.%20Commun.12,%201%E2%80%9311%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Aminov, R. I. &amp; Mackie, R. I. Evolution and ecology of antibiotic resistance genes. <em>FEMS Microbiol. Lett.</em><strong>271</strong>, 147–161 (2007).
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2007.00757.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17490428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aminov,%20R.%20I.%20&amp;%20Mackie,%20R.%20I.%20Evolution%20and%20ecology%20of%20antibiotic%20resistance%20genes.%20FEMS%20Microbiol.%20Lett.271,%20147%E2%80%93161%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Fér, M., Kodešová, R., Klement, A. &amp; Nikodem, A. The impact of treated wastewater and biosolids from the municipal wastewater treatment plant on water and carbon dioxide effluxes from soils. <em>J. Hydrology Hydromechanics</em>. <strong>70</strong>, 276–283 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?F%C3%A9r,%20M.,%20Kode%C5%A1ov%C3%A1,%20R.,%20Klement,%20A.%20&amp;%20Nikodem,%20A.%20The%20impact%20of%20treated%20wastewater%20and%20biosolids%20from%20the%20municipal%20wastewater%20treatment%20plant%20on%20water%20and%20carbon%20dioxide%20effluxes%20from%20soils.%20J.%20Hydrology%20Hydromechanics.%2070,%20276%E2%80%93283%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Fér, M., Kodešová, R., Nikodem, A., Klement, A. &amp; Pavlů, L. Carbon dioxide and water effluxes from soils affected by reclaimed wastewater and sludge from the wastewater treatment plant during the three-year period. <em>Biol. (Bratisl)</em>. 10.1007/s11756-023-01588-z (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?F%C3%A9r,%20M.,%20Kode%C5%A1ov%C3%A1,%20R.,%20Nikodem,%20A.,%20Klement,%20A.%20&amp;%20Pavl%C5%AF,%20L.%20Carbon%20dioxide%20and%20water%20effluxes%20from%20soils%20affected%20by%20reclaimed%20wastewater%20and%20sludge%20from%20the%20wastewater%20treatment%20plant%20during%20the%20three-year%20period.%20Biol.%20(Bratisl).%2010.1007/s11756-023-01588-z%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Murphy, J. &amp; Riley, J. P. A modified single solution method for the determination of phosphate in natural waters. <em>Anal. Chim. Acta</em>. <strong>27</strong>, 31–36 (1962).</cite> [<a href="https://scholar.google.com/scholar_lookup?Murphy,%20J.%20&amp;%20Riley,%20J.%20P.%20A%20modified%20single%20solution%20method%20for%20the%20determination%20of%20phosphate%20in%20natural%20waters.%20Anal.%20Chim.%20Acta.%2027,%2031%E2%80%9336%20(1962)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Bloem, J., Veninga, M. &amp; Shepherd, J. Fully automatic determination of soil bacterium numbers, cell volumes, and frequencies of dividing cells by confocal laser scanning microscopy and image analysis. <em>Appl. Environ. Microbiol.</em><strong>61</strong>, 926–936 (1995).
</cite> [<a href="https://doi.org/10.1128/aem.61.3.926-936.1995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1388375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16534976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bloem,%20J.,%20Veninga,%20M.%20&amp;%20Shepherd,%20J.%20Fully%20automatic%20determination%20of%20soil%20bacterium%20numbers,%20cell%20volumes,%20and%20frequencies%20of%20dividing%20cells%20by%20confocal%20laser%20scanning%20microscopy%20and%20image%20analysis.%20Appl.%20Environ.%20Microbiol.61,%20926%E2%80%93936%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Shirling, E. B. &amp; Gottlieb, D. Methods for characterization of streptomyces species1. <em>Int. J. Syst. Evol. Microbiol.</em><strong>16</strong>, 313–340 (1966).</cite> [<a href="https://scholar.google.com/scholar_lookup?Shirling,%20E.%20B.%20&amp;%20Gottlieb,%20D.%20Methods%20for%20characterization%20of%20streptomyces%20species1.%20Int.%20J.%20Syst.%20Evol.%20Microbiol.16,%20313%E2%80%93340%20(1966)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Kelsic, E. D., Zhao, J., Vetsigian, K. &amp; Kishony, R. Counteraction of antibiotic production and degradation stabilizes microbial communities. <em>Nature</em><strong>521</strong>, 516–519 (2015).
</cite> [<a href="https://doi.org/10.1038/nature14485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4551410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25992546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelsic,%20E.%20D.,%20Zhao,%20J.,%20Vetsigian,%20K.%20&amp;%20Kishony,%20R.%20Counteraction%20of%20antibiotic%20production%20and%20degradation%20stabilizes%20microbial%20communities.%20Nature521,%20516%E2%80%93519%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Kodešová, R. et al. Contamination of water, soil, and plants by micropollutants from reclaimed wastewater and sludge from a wastewater treatment plant. <em>Sci. Total Environ.</em><strong>907</strong>, 167965 (2024).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2023.167965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37866592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kode%C5%A1ov%C3%A1,%20R.%20et%20al.%20Contamination%20of%20water,%20soil,%20and%20plants%20by%20micropollutants%20from%20reclaimed%20wastewater%20and%20sludge%20from%20a%20wastewater%20treatment%20plant.%20Sci.%20Total%20Environ.907,%20167965%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Petříčková, K. et al. Evolution of cyclizing 5-aminolevulinate synthases in the biosynthesis of actinomycete secondary metabolites: outcomes for genetic screening techniques. <em>Front. Microbiol.</em><strong>6</strong>, 1–15 (2015).
</cite> [<a href="https://doi.org/10.3389/fmicb.2015.00814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4525017/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26300877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pet%C5%99%C3%AD%C4%8Dkov%C3%A1,%20K.%20et%20al.%20Evolution%20of%20cyclizing%205-aminolevulinate%20synthases%20in%20the%20biosynthesis%20of%20actinomycete%20secondary%20metabolites:%20outcomes%20for%20genetic%20screening%20techniques.%20Front.%20Microbiol.6,%201%E2%80%9315%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. Basic local alignment search tool. <em>J. Mol. Biol.</em><strong>215</strong>, 403–410 (1990).
</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altschul,%20S.%20F.,%20Gish,%20W.,%20Miller,%20W.,%20Myers,%20E.%20W.%20&amp;%20Lipman,%20D.%20J.%20Basic%20local%20alignment%20search%20tool.%20J.%20Mol.%20Biol.215,%20403%E2%80%93410%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Lanoot, B. et al. BOX-PCR fingerprinting as a powerful tool to reveal synonymous names in the genus streptomyces. Emended descriptions are proposed for the species streptomyces.cinereorectus, S. fradiae, S. tricolor, S. colombiensis, S. filamentosus, S. vinaceus and S. phaeopurpureus. <em>Syst. Appl. Microbiol.</em><strong>27</strong>, 84–92 (2004).
</cite> [<a href="https://doi.org/10.1078/0723-2020-00257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15053325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lanoot,%20B.%20et%20al.%20BOX-PCR%20fingerprinting%20as%20a%20powerful%20tool%20to%20reveal%20synonymous%20names%20in%20the%20genus%20streptomyces.%20Emended%20descriptions%20are%20proposed%20for%20the%20species%20streptomyces.cinereorectus,%20S.%20fradiae,%20S.%20tricolor,%20S.%20colombiensis,%20S.%20filamentosus,%20S.%20vinaceus%20and%20S.%20phaeopurpureus.%20Syst.%20Appl.%20Microbiol.27,%2084%E2%80%9392%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Ciccarelli, F. D., Doerks, T., Creevey, C. J. &amp; Snel, B. Bork, P. of a highly resolved tree of life. <em>Sci. (1979)</em>. <strong>311</strong>, 1283–1287 (2006).</cite> [<a href="https://doi.org/10.1126/science.1123061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16513982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ciccarelli,%20F.%20D.,%20Doerks,%20T.,%20Creevey,%20C.%20J.%20&amp;%20Snel,%20B.%20Bork,%20P.%20of%20a%20highly%20resolved%20tree%20of%20life.%20Sci.%20(1979).%20311,%201283%E2%80%931287%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Caporaso, J. G. et al. Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample. <em>Proc. Natl. Acad. Sci. U S A</em>. <strong>108</strong>, 4516–4522 (2011).
</cite> [<a href="https://doi.org/10.1073/pnas.1000080107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3063599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20534432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caporaso,%20J.%20G.%20et%20al.%20Global%20patterns%20of%2016S%20rRNA%20diversity%20at%20a%20depth%20of%20millions%20of%20sequences%20per%20sample.%20Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A.%20108,%204516%E2%80%934522%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Callahan, B. J. et al. DADA2: High-resolution sample inference from illumina amplicon data. <em>Nat. Methods</em>. <strong>13</strong>, 581–583 (2016).
</cite> [<a href="https://doi.org/10.1038/nmeth.3869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4927377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27214047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callahan,%20B.%20J.%20et%20al.%20DADA2:%20High-resolution%20sample%20inference%20from%20illumina%20amplicon%20data.%20Nat.%20Methods.%2013,%20581%E2%80%93583%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Callahan, B. J. et al. Replication and refinement of a vaginal microbial signature of preterm birth in two Racially distinct cohorts of US women. <em>Proc. Natl. Acad. Sci. U S A</em>. <strong>114</strong>, 9966–9971 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1705899114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5604014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28847941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callahan,%20B.%20J.%20et%20al.%20Replication%20and%20refinement%20of%20a%20vaginal%20microbial%20signature%20of%20preterm%20birth%20in%20two%20Racially%20distinct%20cohorts%20of%20US%20women.%20Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A.%20114,%209966%E2%80%939971%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Quast, C. et al. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. <em>Nucleic Acids Res.</em><strong>41</strong>, D590–D596 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gks1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23193283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quast,%20C.%20et%20al.%20The%20SILVA%20ribosomal%20RNA%20gene%20database%20project:%20improved%20data%20processing%20and%20web-based%20tools.%20Nucleic%20Acids%20Res.41,%20D590%E2%80%93D596%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Wang, Q., Garrity, G. M., Tiedje, J. M. &amp; Cole, J. R. Naive bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy. <em>Appl. Environ. Microbiol.</em><strong>73</strong>, 5261–5267 (2007).
</cite> [<a href="https://doi.org/10.1128/AEM.00062-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.,%20Garrity,%20G.%20M.,%20Tiedje,%20J.%20M.%20&amp;%20Cole,%20J.%20R.%20Naive%20bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy.%20Appl.%20Environ.%20Microbiol.73,%205261%E2%80%935267%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>R Core Team. R: A Language and Environment for Statistical Computing. (2021).</cite>
</li>
<li id="CR83">
<span class="label">83.</span><cite>McMurdie, P. J. &amp; Holmes, S. Phyloseq: an R package for reproducible interactive analysis and graphics of Microbiome census data. <em>PLoS One</em>. <strong>8</strong>, e61217 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0061217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3632530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23630581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMurdie,%20P.%20J.%20&amp;%20Holmes,%20S.%20Phyloseq:%20an%20R%20package%20for%20reproducible%20interactive%20analysis%20and%20graphics%20of%20Microbiome%20census%20data.%20PLoS%20One.%208,%20e61217%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Hugerth, L. W. &amp; Andersson, A. F. Analysing microbial community composition through amplicon sequencing: from sampling to hypothesis testing. <em>Front. Microbiol.</em><strong>8</strong>, 1561 (2017).
</cite> [<a href="https://doi.org/10.3389/fmicb.2017.01561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5591341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28928718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hugerth,%20L.%20W.%20&amp;%20Andersson,%20A.%20F.%20Analysing%20microbial%20community%20composition%20through%20amplicon%20sequencing:%20from%20sampling%20to%20hypothesis%20testing.%20Front.%20Microbiol.8,%201561%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated Estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20Estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Wickham, H. <em>Ggplot2: Elegant Graphics for Data Analysis</em> (Springer-, 2016). 10.1007/978-3-319-24277-4</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373788/bin/41598_2025_14953_MOESM1_ESM.pptx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (758.9KB, pptx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373788/bin/41598_2025_14953_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (56.7KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analysed during this study are included in this published article (and its Supplementary Information files). The 16S rRNA metagenomic sequence data obtained from microbial consortia have been deposited in the European Nucleotide Archive (ENA) at EMBL-EBI under accession number PRJEB75333 and are available at the following URL: https://www.ebi.ac.uk/ena/browser/view/PRJEB75333. 16S rRNA sequences of actinomycete isolates obtained in this study have been deposited in the GenBank under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PQ423591:PQ423632%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ423591-PQ423632</a> (Supplementary Table S9).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-14953-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_14953.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373788/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373788/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373788%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373788/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373788/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373788/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846880/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373788/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846880/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373788/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373788/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="HlwGOl2yuddNCj6TBq2vrR3XMHXDO4qhG9BEUu6pGMmnRZVCd8Kez39aqeJ1Juos">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
