
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Data-driven frameworks to robustly predict solubility parameter of diverse polymers - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4838B8AEEF72305838B003D2537FD.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375797/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Data-driven frameworks to robustly predict solubility parameter of diverse polymers">
<meta name="citation_author" content="Raouf Hassan">
<meta name="citation_author_institution" content="Civil Engineering Department, College of Engineering, Imam Mohammad Ibn Saud Islamic University (IMSIU), Riyadh, 13318 Saudi Arabia">
<meta name="citation_author" content="Mohammad Reza Kazemi">
<meta name="citation_author_institution" content="Process Engineering Department, Bandar Imam Petrochemical Company (BIPC), Mahshahr, Iran">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="31157">
<meta name="citation_doi" content="10.1038/s41598-025-12758-1">
<meta name="citation_pmid" content="40851024">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375797/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375797/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375797/pdf/41598_2025_Article_12758.pdf">
<meta name="description" content="This study intends to effectively forecast solubility parameter of diverse polymers by creating machine learning models that can grasp the complex relationships between essential input factors like molecular weight, melting point, boiling point, ...">
<meta name="og:title" content="Data-driven frameworks to robustly predict solubility parameter of diverse polymers">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="This study intends to effectively forecast solubility parameter of diverse polymers by creating machine learning models that can grasp the complex relationships between essential input factors like molecular weight, melting point, boiling point, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375797/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375797">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-12758-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_12758.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375797%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375797/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375797/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375797/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 25;15:31157. doi: <a href="https://doi.org/10.1038/s41598-025-12758-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-12758-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Data-driven frameworks to robustly predict solubility parameter of diverse polymers</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hassan%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Raouf Hassan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Raouf Hassan</span></h3>
<div class="p">
<sup>1</sup>Civil Engineering Department, College of Engineering, Imam Mohammad Ibn Saud Islamic University (IMSIU), Riyadh, 13318 Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hassan%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Raouf Hassan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kazemi%20MR%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mohammad Reza Kazemi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mohammad Reza Kazemi</span></h3>
<div class="p">
<sup>2</sup>Process Engineering Department, Bandar Imam Petrochemical Company (BIPC), Mahshahr, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kazemi%20MR%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohammad Reza Kazemi</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Civil Engineering Department, College of Engineering, Imam Mohammad Ibn Saud Islamic University (IMSIU), Riyadh, 13318 Saudi Arabia </div>
<div id="Aff2">
<sup>2</sup>Process Engineering Department, Bandar Imam Petrochemical Company (BIPC), Mahshahr, Iran </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 21; Accepted 2025 Jul 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375797  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40851024/" class="usa-link">40851024</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">This study intends to effectively forecast solubility parameter of diverse polymers by creating machine learning models that can grasp the complex relationships between essential input factors like molecular weight, melting point, boiling point, liquid molar volume, radius of gyration, dielectric constant, dipole moment, refractive index, van der Waals area and reduced volume, and parachor, alongside the target variable, which is solubility coefficient of polymers. The goal is to create strong models that accurately capture these intricate relationships to facilitate accurate forecasts of the solubility parameter for polymers. Multiple machine learning algorithms, ranging from basic methods like Linear Regression to advanced techniques such as Artificial Neural Networks (ANNs), Ridge Regression, Lasso Regression, Support Vector Machines (SVMs), Linear Regression, Random Forests (RFs), Gradient Boosting Machines (GBM), K-Nearest Neighbors (KNN), Elastic Net, Decision Trees, Light Gradient Boosting Machine (LightGBM), Categorical Boosting (CatBoost), Convolutional Neural Networks (CNNs), and Extreme Gradient Boosting (XGBoost) were utilized. These methods were utilized to create data-driven models that adeptly seize the intricate connections between input characteristics and output variable, facilitating precise predictions of the solubility parameter for polymers. The efficacy of the developed models was rigorously evaluated using statistical metrics such as R², RMSE, and MRD%, along with visual tools including cross-plots, deviation plots, and SHAP analysis to enhance interpretability and predictive reliability. To guarantee the dataset’s reliability, consisting of 1,799 datapoints on the solubility parameter of polymers, the Monte Carlo outlier detection algorithm was utilized. This stage verified the dataset’s accuracy and appropriateness for model training and evaluation. Results indicated that the models CatBoost, ANN, and CNN surpassed other techniques, attaining superior accuracy shown by the highest R-squared values and the lowest error rates. Sensitivity analysis showed that every input feature impacted the target variable, while SHAP analysis determined that dielectric constant was the most significant factor influencing the solubility parameter of polymers. These results highlight the efficiency of the utilized machine learning methods and emphasize the vital importance of these input parameters in establishing the solubility parameter of polymers. This method not only verifies that the models can make accurate predictions but also provides valuable insights into the impact of input features on solubility parameters of polymers, enhancing algorithm interpretability and scientific understanding.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-12758-1.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Polymers, Solubility, Machine learning, Data-driven models, SHAP analysis</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Chemistry, Engineering, Materials science, Mathematics and computing</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">In the fields of pharmaceutical, petroleum, chemical and polymer engineering, scientists and engineers prioritize identifying the most suitable solvents per explicit industrial applications. Solubility parameter provides a critical measure of a solvent tendency to solve a given solute. The idea of the solubility factor was formalized in the 1930s by Hildebrand, building on earlier work by Scatchard on cohesive energy density. This theory has since been extensively studied and applied across various chemical processes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Subsequent contributions by Hildebrand<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> and Hansen<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup> further refined the notion by introducing additional parameters and expanding its applicability. The solubility parameter has proven instrumental in polymer processing, surface treatments and coating technologies<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par3">The theoretical foundation of the solubility parameter lies in molecular interaction and chemical structure energies<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In the case of similarity of solubility factors for two molecules, the miscibility between one molecule (solution or liquid) and another (solid) tends to improve<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. This principle, known as “like dissolves like” underpins many solvent selection strategies<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. In condensed phases, intermolecular forces, primarily cohesive forces and van der Waals attractions, govern molecular behavior<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Hildebrand<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> expanded on earlier cohesive energy theories by introducing the concept of the solubility factor, which relates to the energy density of cohesion of materials. Generally, the solubility parameter can be determined as the square root of the cohesive pressure and is often called the Hildebrand solubility parameter<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. However, the Hildebrand parameter’s inability to account for hydrogen bonding limits its application to slightly polar or nonpolar materials deprived of hydrogen bonding<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. To overcome this limitation, Hansen introduced solubility parameter of three-component type, which takes the account of dipolar, nonpolar and hydrogen-bonding interactions<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Hansen solubility parameters (HSP) have found widespread use in some industries such as coatings and paints, where solvent-polymer interactions should be essentially taken into account<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Additionally, HSP has been extended to represent materials like nanoparticles, pigments DNA, and others, enabling the analysis of interactions with plasticizers, solvents, foodstuffs, fragrance chemicals and more<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par4">Recognizing the importance of solubility parameter, numerous approaches have been developed to calculate or estimate it. Equation-of-state (EOS)-based methods have been extensively considered for this purpose over the past years<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Group contribution (GC) methods, pioneered by van Krevelen<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, enable the assessment of partial solubility parameters for pure organic and polymer compounds.</p>
<p id="Par5">Quantitative structure-property/activity relationship (QSPR/QSAR) method, leveraging classification/regression techniques, is broadly employed in engineering and chemical science to forecast physical characteristics<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Although QSPR/QSAR models deliver accurate results, determining molecular descriptors involves time-intensive processes such as optimization, structure drawing and calculation, which may pose challenges for researchers lacking advanced computational chemistry expertise.</p>
<p id="Par6">This study employs machine learning (ML), known as a multidisciplinary arena combining ideas from probability theory, statistics, approximation theory, and other domains<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, to develop predictive models for the solubility parameter of diverse polymer solutions using a QSPR strategy. Advanced ML algorithms, including linear regression, convolutional and artificial neural networks, lasso and ridge regression, support vector machines, elastic net, random forests, gradient boosting, k-nearest neighbors, decision trees, extreme gradient boosting, categorical boosting, light gradient boosting, and Gaussian processes, are used. The Monte Carlo outlier detection algorithm ensures dataset suitability for model training. Model performance is assessed using multiple metrics and visualization tools, while SHAP values provide insights into the impact of essential characteristics on solubility estimations. A detailed description of the methodology is provided in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/d0be6385349f/41598_2025_12758_Fig1_HTML.jpg" loading="lazy" id="d33e293" height="422" width="794" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Modeling solubility parameter of polymers via various data driven methods.</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Machine learning backgrounds</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Convolutional neural network</h3>
<p id="Par8">A Convolutional Neural Network (CNN) is a type of deep learning algorithm tailored to handle structured data, including images and time-series data. These networks excel in visual computing applications, like object detection, image classification, and intricate pattern recognition<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The structure of a CNN is composed of several layers, with pooling layers, convolutional layers, and completely linked layers playing crucial roles<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Convolutional layers employ filters or kernels to detect fundamental characteristics such as lines and patterns, which are subsequently processed by deeper layers to identify more complex patterns, such as shapes or objects. Pooling layers help reduce data dimensionality, allowing the network to focus on the primary significant features<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. CNNs are usually trained using the backpropagation algorithm, which enables the network to fine-tune its weights by comparing predictions with actual outcomes<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. These networks are highly effective in various fields, including facial recognition, medical imaging, and general image analysis, and can autonomously learn hierarchical features with little human input. A key advantage of CNNs is their ability to generalize well to new, previously unseen data, making them particularly efficient for tasks involving large datasets<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Recent improvements in hardware, especially the adoption of Graphics Processing Units (GPUs), have greatly enhanced the training speed of these networks, further boosting their widespread use<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">K-nearest neighbors</h3>
<p id="Par9">The K-Nearest Neighbors (KNN) method is among the most commonly used supervised learning methods in machine learning. It operates by evaluating distances between data points to make predictions or classifications<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. When estimating the value or class of a new data point, KNN identifies the K closest points from training dataset and determines prediction based on majority label among these K neighbors<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. This process usually involves distance metrics such as Euclidean distance, meaning that closer data points have a stronger influence on the outcome. A key advantage of KNN is its non-parametric nature, which means it does not need a clear training or learning stage<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Rather than constructing a complex predictive model, KNN directly uses the training data to make predictions at runtime. However, this also leads to one of its main drawbacks: the algorithm can be computationally expensive during the prediction phase, especially with large datasets, since it must determine distances for every data point in the training set<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par10">Additionally, execution of KNN is highly responsive to the choice of the K factor and method used to measure distances. Selecting an inappropriate K value or distance metric can greatly influence results accuracy<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Despite these limitations, KNN is extensively applied in different fields, like text classification, and junk email filtering, due to its simplicity and effectiveness in many scenarios<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Lasso regression</h3>
<p id="Par11">Lasso Regression is a linear regression algorithm designed for feature selection and mitigating overfitting in regression models. The name “Lasso” stands for " Selection Operator and Least Absolute Shrinkage”<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. This technique incorporates an L1 regularization to reduce features coefficients, bringing certain coefficients to zero and effectively removing the corresponding features from the model<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. This process allows for automatic feature selection, as less important or irrelevant features are discarded, leading to a model that is easier to understand and interpret.</p>
<p id="Par12">A major benefit of Lasso Regression is its capability to automatically identify relevant features and optimize the model, particularly when dealing with datasets that include numerous irrelevant or highly correlated features<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. It is especially effective in high-dimensional scenarios in situations with a substantial amount of characteristics, and some may be redundant or non-informative. By simplifying the model, Lasso Regression improves the model’s capability to apply to novel, unencountered data<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p>
<p id="Par13">Despite its advantages, Lasso Regression has some drawbacks. It may struggle when features are highly correlated, as it tends to randomly select one feature from a group of correlated predictors and disregard the others<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. In such situations, alternative approaches like Ridge Regression might be more suitable. Nevertheless, Lasso Regression continues to be an influential instrument in machine learning and data assessment, providing a reliable method for constructing streamlined models in cases with intricate feature relationships<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Decision tree</h3>
<p id="Par14">The Decision Tree algorithm is a frequently used technique in machine learning for solving regression and classification problems. It constructs a model as a hierarchical tree structure, with each node symbolizing a decision or condition that partitions the data into subsets<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The tree is built using the input features, with the process continuing until it reaches leaf nodes, which provide the final predictions or outcomes<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. While building the tree, metrics such as Gini Index or Entropy are employed as splitting criteria to minimize impurity and enhance the accuracy of classifications at each step<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par15">One of the primary strengths of decision trees is their interpretability. The logic behind the decisions made by the model can be easily traced and understood, rendering them appropriate for uses that require transparency is important<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Moreover, decision trees are capable of managing both numerical and categorical data without the need for feature scaling, adding to their versatility<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p>
<p id="Par16">However, decision trees have some limitations. They are susceptible to overfitting, particularly when handling noisy data, since the model might grasp irrelevant patterns or anomalies found in the training dataset<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. To mitigate this issue, methods such as ensemble methods or pruning are often utilized. These approaches help simplify the tree and improve its capacity to apply to unknown data, thereby enhancing the general durability and efficiency of the model<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Linear regression</h3>
<p id="Par17">Linear Regression is among the most frequently utilized machine learning methods for representing the relationship between a dependent parameter and a single or several independent variables. This method operates under the assumption that there is a straight relationship between the input characteristics and the output labels<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The primary objective of linear regression is to identify a line or hyperplane that most suitable for the data. Mathematically, this is represented by a linear equation, where the coefficients are optimized using techniques such like the least squares approach<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par18">Linear regression finds applications in various domains, including predicting real estate prices, forecasting product sales, and analyzing economic trends. One of its key advantages is its simplicity and interpretability, as the results are straightforward to understand and explain<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Additionally, when the connection between parameters is genuinely linear, linear regression can generate highly accurate predictions<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>.</p>
<p id="Par19">However, linear regression may struggle with datasets that exhibit complex or nonlinear relationships, leading to issues such as underfitting or overfitting. In such cases, more advanced models, such as polynomial regression or nonlinear methods, might be more appropriate<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. In spite of these constraints, linear regression remains a powerful instrument for comprehension and forecasting outcomes in scenarios where the relationships between variables are approximately linear<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Gradient boosting machine</h3>
<p id="Par20">The Gradient Boosting Machine (GBM) is a potent machine learning algorithm designed to tackle regression and classification jobs. It leverages boosting technique, which enhances predictive performance by combining several fragile models into a more robust one ensemble<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. In GBM, models are constructed sequentially, with every following model aimed at correcting the errors of those before it. This repetitive method makes use of the Gradient Descent improvement method to minimize prediction errors by targeting the residuals (differences between actual and predicted values) at each step<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p>
<p id="Par21">At its core, during each training phase, GBM ensures that the new model specifically addresses the weaknesses of the previous one, progressively improving overall accuracy. A primary benefit of GBM is its capacity to manage intricate, nonlinear datasets effectively, resulting in high predictive precision<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. By relying on weak models and gradually refining them, GBM achieves strong generalization capabilities, rendering it appropriate for various uses, such as fraud detection, sales forecasting, and disease diagnosis<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>.</p>
<p id="Par22">Despite its strengths, GBM does have some drawbacks. It can be sensitive to noisy data, which may lead to overfitting if not properly managed. Additionally, the algorithm typically requires more training time compared to other methods due to its sequential nature. Another challenge lies in parameter tuning, as selecting optimal hyperparameters for GBM can be complex and computationally demanding<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Nevertheless, when properly configured, GBM remains a highly effective tool to address different real-life issues<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Support vector machine</h3>
<p id="Par23">Support Vector Machine (SVM) is a robust machine learning method primarily utilized for classification jobs, although it can likewise be modified for regression jobs. The core objective of SVM is to create a perfect decision boundary, referred to as a hyperplane, that efficiently divides data points that pertain to various categories<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. In classification, this hyperplane is designed to maximize margin between classes, ensuring the most effective separation possible. For datasets that are not linearly separable, SVM utilizes kernel functions to transform the data into a higher-dimensional space, enabling linear separation in this transformed space<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>.</p>
<p id="Par24">A major advantage of SVM is its ability to manage intricate, nonlinear relationships within data by leveraging various kernel functions, like radial basis function (RBF) kernels or polynomial. This makes SVM particularly effective in high-dimensional spaces, where it has demonstrated strong predictive accuracy<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. As a result, SVM has been effectively utilized in various fields, such as facial recognition, illness diagnosis, and text categorization<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p>
<p id="Par25">However, SVM does have notable limitations. It is highly sensitive to parameter settings, especially the choice of kernel function and the tuning of regularization parameter C, which manages the balance between obtaining a low error on the training dataset and maintaining a smooth decision boundary<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Additionally, training an SVM model may require significant computational resources, especially for extensive datasets, as it entails addressing a quadratic optimization challenge. This requirement for significant computational resources and time can make SVM less practical for very large-scale applications. In spite of these difficulties, SVM continues to be an effective instrument for numerous machine learning applications, especially when dealing with complex, high-dimensional data<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Categorical boosting</h3>
<p id="Par26">Categorical Boosting, commonly known as CatBoost, is an innovative machine learning method aimed at enhancing the efficacy of classification models, particularly when working with datasets that include categorical attributes<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. In contrast to conventional boosting algorithms like XGBoost and LightGBM, which mainly target numerical data and necessitate preprocessing steps such as One-Hot Encoding for categorical variables, Categorical Boosting directly manages categorical features without extensive preprocessing requirements. This is accomplished via cutting-edge encoding techniques that are incorporated into the model’s structure<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p>
<p id="Par27">CatBoost integrates Gradient Boosting concepts with unique feature engineering methods specifically designed for categorical data. This approach removes the necessity for manual encoding or changing of categorical variables, thereby decreasing preprocessing time and complexity<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The algorithm utilizes advanced techniques like target-oriented encoding and mixtures of categorical attributes to efficiently grasp the connections between categories and the target variable<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p>
<p id="Par28">A major benefit of Categorical Boosting is its capacity to improve model precision while streamlining the process. By directly training on categorical data, the model can utilize information from various classes at once, resulting in enhanced classification effectiveness<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. This is especially beneficial in situations where data shows batch traits or when the dataset includes many categorical variables. Moreover, CatBoost’s method of managing categorical features guarantees that the model stays strong and efficient, even when dealing with high-cardinality data<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p>
<p id="Par29">In conclusion, Categorical Boosting provides an effective approach for classification tasks by overcoming the challenges of conventional boosting algorithms when working with categorical data. Its capacity to enhance preprocessing efficiency, along with its robust predictive capabilities, renders it a useful resource in multiple areas, such as customer segmentation, fraud identification, and natural language processing. Nonetheless, similar to other intricate models, it might need precise adjustment of hyperparameters for optimal performance<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Artificial neural network</h3>
<p id="Par30">An Artificial Neural Network (ANN) is a computing framework intended to mimic information functioning abilities of the human mind. It is made up of linked processing units known as neurons, which function similarly to biological neurons. Each neuron receives inputs from other neurons, processes them using a mathematical operation such as an activation function and then sends an output to subsequent neurons<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. ANNs are especially proficient at addressing intricate jobs like pattern identification and forecasting, and data classification, as they have the ability to learn from datasets and progressively enhance their results gradually<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p>
<p id="Par31">Typically, ANNs are structured in multiple layers, forming what is commonly referred to as a Multilayer Perceptron (MLP). These layers include a layer for input, several hidden layers, and a layer for output<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. The input layer takes in data, while the hidden layers handle processing it through various transformations, and output layer provides the final result. In the training period, algorithms like backpropagation are employed to adjust the weights between neurons. This adjustment minimizes prediction errors and gradually enhances the network’s accuracy by fine-tuning its internal parameters<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p>
<p id="Par32">ANNs are especially well-suited for handling intricate, nonlinear problems that demand substantial computational power. As a result, they are widely used in various areas like processing of language, recognition of image and speech, and many other areas within data science<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Their capacity to represent intricate connections and adjust to new data makes them an effective instrument for resolving real-world problems across a variety of industries<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">10. Extreme gradient boosting</h3>
<p id="Par33">Extreme Gradient Boosting (XGBoost) is an exceptionally powerful and robust machine learning method celebrated for its skill in addressing intricate regression and classification challenges, particularly in extensive datasets. XGBoost, as an enhanced version of the gradient boosting framework, builds upon conventional boosting methods by merging various weak models usually decision trees for building a robust forecasting model<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. In this algorithm, every a new tree is formed to address the errors of the earlier one, gradually enhancing the predictions and moving closer to the best solution.</p>
<p id="Par34">A key attribute of XGBoost is its incorporation of regularization methods, which aid in avoiding overfitting and improving the model’s generalization abilities. This renders it especially efficient in managing noisy or high-dimensional data. XGBoost is renowned for its speed and efficiency, capable of providing very accurate predictions even when managing large data sets<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Utilizing parallel processing, the algorithm greatly shortens computation duration, making it highly appropriate for practical applications that deal with extensive datasets<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>.</p>
<p id="Par35">XGBoost is highly effective in handling complex and unstructured data, providing strong solutions for different activities, such as recognition of image and processing of natural language. Its extensive customizability enables users to adjust multiple parameters, including learning rate, tree depth, and regularization strength, to enhance performance for particular applications. These features have resulted in its extensive use in both scholarly research and commercial applications, including data mining contests such as those on Kaggle, where it has repeatedly shown exceptional performance<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>.</p>
<p id="Par36">To summarize, XGBoost is acknowledged as one of the greatest preferred algorithms in machine learning because of its effectiveness, precision, and versatility. Its capacity to manage intricate datasets while ensuring high performance has established its status as a preferred approach for addressing prediction and classification problems across various fields<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">11. Light gradient boosting machine</h3>
<p id="Par37">The LightGBM is a cutting-edge machine learning algorithm specifically designed to handle large and complex datasets efficiently. It represents an optimized variant of the gradient boosting method, offering significant benefits like quicker training speed, reduced memory usage, and improved accuracy compared to similar algorithms like XGBoost<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. One of the key innovations of LightGBM is its unique approach to building decision trees using “leaf-wise” growth rather than the conventional “level-wise” method. This strategy allows the algorithm to create deeper trees more effectively, focusing on splits that contribute the most to performance of model, thereby enhancing accuracy<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par38">A major strength of LightGBM excels due to its capability to train models rapidly, even on massive datasets. By leveraging techniques such as parallel processing and batch sampling, LightGBM achieves superior performance while minimizing computational resources. Additionally, it is highly efficient in managing both discrete and continuous data, all while consuming significantly less memory than other gradient boosting frameworks. These features make LightGBM particularly well-suited for applications involving large-scale data modeling, such as fraud detection, sales forecasting, and improving search engine rankings<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>.</p>
<p id="Par39">Due to its efficiency, scalability, and high performance, LightGBM has gained widespread adoption across various industries and research domains. Its ability to deliver accurate predictions with reduced resource requirements makes it serves as a perfect option for addressing practical issues where data size and complexity are significant challenges. Overall, LightGBM stands out as a powerful tool for building robust predictive models in large-scale applications<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">1 elastic net</h3>
<p id="Par40">Elastic Net is a regression approach that merges the benefits of two well-known regularization techniques: Lasso regression and Ridge regression. This combined strategy is especially useful for datasets that exhibit high dimensionality or significant correlations between features. Lasso regression prioritizes feature selection by forcing the coefficients of less significant features to zero, whereas Ridge regression combats overfitting by imposing penalties on large coefficients and lowering model variance<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Elastic Net combines these two approaches to find a balance between selecting features and applying regularization, which makes it particularly advantageous in situations where the feature count is greater than the observation count<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>.</p>
<p id="Par41">A significant benefit of Elastic Net is its capacity to manage highly correlated features more efficiently than using Lasso or Ridge separately. In cases where several features are highly correlated, Lasso typically chooses one feature at random and disregards the rest, while Elastic Net can incorporate clusters of correlated features into the model. This renders Elastic Net ideal for datasets that exhibit intricate relationships among variables, as seen in fields like genomics, finance, or image processing<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>.</p>
<p id="Par42">The algorithm attains this equilibrium through a weighted mix of the L1 and L2 penalties. By adjusting the mixing parameter, which governs the balance of each regularization technique, Elastic Net can enhance the compromise between selecting features and minimizing variance. This adaptability enables Elastic Net to enhance the precision of forecasts and generalization of models, particularly when handling numerous features or multicollinearity<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>.</p>
<p id="Par43">In conclusion, Elastic Net is an effective method for regression issues that involve high-dimensional data or related attributes. Its capacity to integrate the advantages of Lasso and Ridge regression renders it a perfect option for improving model performance in intricate datasets, guaranteeing strength and dependability in numerous practical applications<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">13. Ridge regression</h3>
<p id="Par44">Ridge Regression is a regularization method employed in machine learning to avoid overfitting in regression models, especially when working with high-dimensional datasets. It is particularly useful in circumstances where the quantity of features surpasses the amount of observations or when there are significant correlations between the features<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. In Ridge Regression, a penalty component is included in loss function to prevent the model from giving overly large weights to any individual feature. This penalty is determined by summing the squares of the weight values, promoting the model to allocate importance more uniformly among all features by diminishing the size of the coefficients<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>.</p>
<p id="Par45">The hyperparameter, λ (lambda), referred to as the regularization coefficient, regulates the regularization strength in Ridge Regression. An increased value of λ enforces a more significant penalty on the model, resulting in reduced weights and a more straightforward model that performs better on new data. On the other hand, a smaller λ enables the model to align more precisely with the training data, potentially leading to overfitting if not adequately adjusted. By modifying λ, a balance between bias and variance can be achieved, allowing the model to stay both adaptable and strong<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>.</p>
<p id="Par46">Ridge Regression is highly valued due to its user-friendliness and efficiency, rendering it a favored option in data analysis endeavors that include numerous features and intricate connections among them. It is especially beneficial when dealing with noisy data since it minimizes the influence of superfluous or redundant features by driving their coefficients closer to zero. This aids in enhancing the model’s forecasting accuracy and consistency, particularly in cases where more intricate models could overfit the data. In general, Ridge Regression offers an effective method for enhancement the generalization abilities of regression models when dealing with difficult datasets<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">14. Random forest</h3>
<p id="Par47">Random Forest is a robust machine learning algorithm that leverages an ensemble of decision trees to predict and analyze data. As a kind of ensemble learning technique, it integrates the forecasts from various weak models (individual decision trees) to create a greater precision and stable overall model. In Random Forest, instead of constructing a single decision tree, numerous trees are built, each using a random subset of data<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. The ultimate prediction is decided via a voting process for classification tasks (majority vote) or averaging for regression tasks, based on the outputs of all the individual trees<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>,<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>.</p>
<p id="Par48">One of the main benefits of Random Forest is its capability to significantly reduce overfitting, a common issue with individual decision trees. By introducing randomness in both the data samples and the characteristics employed to construct each tree, the algorithm ensures diversity among the trees, which enhances the model’s generalization capability. This randomness prevents any single tree from becoming too complex or overly tailored to training data, thereby enhancing the model’s effectiveness on unfamiliar data<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>.</p>
<p id="Par49">Another notable feature of Random Forest is its capacity to manage high-dimensional datasets with noisy and correlated features. The algorithm’s inherent ability to work with subsets of features during the tree-building process ensures that it can effectively manage intricate relationships within the data. This makes Random Forest particularly well-suited for a broad variety of applications, comprising acknowledgment of pattern, processing of image, and forecasting of financial<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>.</p>
<p id="Par50">In conclusion, Random Forest is an adaptable and robust tool for tasks involving both classification and regression. Its ability to handle complex, noisy datasets while reducing overfitting and improving accuracy has made it a popular choice across various domains, from data science to real-world problem-solving scenarios<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Data description and analysis</h2>
<p id="Par51">The dataset utilized in this study originates from the DIPPR 801 database, maintained by the Design Institute for Physical Properties (DIPPR) of the American Institute of Chemical Engineers (AIChE)<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. Constructed by Design Institute for Physical Properties (DIPPR) to address industry demands for reliable real estate information, DIPPR 801 database is renowned for its rigorous curation and validation processes. These processes ensure the consistency and precision of the information<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. For this study, the independent parameters were extracted from the physical constants of the aforementioned database, which encompasses solubility factors of 1,889 polymeric compounds. Concerning this, the input variables include molecular weight, melting point, boiling point, liquid molar volume, dielectric constant, radius of gyration, refractive index, van der Waals area, reduced volume, and parachor, while the output factor refers to the solubility parameter of polymers, which is predicted using the QSPR approach developed through machine learning models. This comprehensive dataset enables the creation of sophisticated models that can accurately predict solubility parameters, thereby facilitating advancements in polymer science and engineering.</p>
<p id="Par52">It is crucial to note that the solubility parameters of polymers were experimentally determined, as reported in the literature, based on a range of physicochemical properties and structural descriptors. The resulting experimental dataset consists of 1,799 records, which have been systematically divided into three subsets to facilitate model development and evaluation. Specifically, approximately 70% of the data (1,259 data points) is designated for training the predictive models, while 15% (270 data points) is reserved for testing and another 15% (270 data points) for validation. This strategic partitioning of the dataset ensures a rigorous and reliable framework for developing and assessing models aimed at accurately predicting the solubility parameters of polymers. Such an approach improves generalizability and robustness of the models, rendering them more relevant to real-world polymer processing scenarios.</p>
<p id="Par53">The input parameters for the predictive model were carefully selected by identifying the key factors that significantly influence the solubility parameter of polymers. These parameters include a comprehensive set of physicochemical properties, such as molecular weight, melting point, boiling point, liquid molar volume, dipole moment, radius of gyration, dielectric constant, van der Waals area, refractive index, and reduced volume, and parachor. Each of these properties holds a crucial position in deciding behavior and interactions of polymers, thereby directly or indirectly affecting their solubility characteristics. For example, molecular weight has been shown to influence polymer solubility by affecting chain entanglement and free volume, with higher molecular weights typically reducing solubility due to increased chain stiffness and lower segmental mobility<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Similarly, dielectric constant reflects the polarity of the polymer, which directly affects interactions with polar or nonpolar solvents<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. Boiling and melting points are indicators of intermolecular forces, such as van der Waals or hydrogen bonding, which also influence solubility behavior<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>.</p>
<p id="Par54">In this study, the solubility parameter of polymers serves as the primary output variable, which the model aims to predict accurately based on aforementioned input parameters. To provide insights into the range, distribution, and relationships between the input parameters and the solubility parameter, scatter matrix plots have been generated and are presented in Fig. <a href="#Fig2" class="usa-link">2</a>. These visualizations offer a valuable overview of the data, highlighting patterns, correlations, and potential outliers, which are crucial for grasping the foundational framework of the dataset and guiding the development of an effective predictive model.</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/e213ddcddd78/41598_2025_12758_Fig2a_HTML.jpg" loading="lazy" id="d33e803" height="1589" width="788" alt="Fig. 2"></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/bbcb1a2f1d8f/41598_2025_12758_Fig2b_HTML.jpg" loading="lazy" id="d33e804" height="1601" width="790" alt="Fig. 2"></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/e241445c28ed/41598_2025_12758_Fig2c_HTML.jpg" loading="lazy" id="d33e805" height="1599" width="790" alt="Fig. 2"></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/3ded4ce4261b/41598_2025_12758_Fig2d_HTML.jpg" loading="lazy" id="d33e806" height="1607" width="790" alt="Fig. 2"></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/83a063a5b6b7/41598_2025_12758_Fig2e_HTML.jpg" loading="lazy" id="d33e807" height="782" width="787" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Scatter matrix diagrams: Relationships between variables.</p></figcaption></figure><p id="Par56">Figure <a href="#Fig3" class="usa-link">3</a> presents the Pearson correlation coefficients for all parameter pairs analyzed in this study. The Pearson correlation coefficient r is calculated using the following formula:</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/bf1bda61fff4/41598_2025_12758_Fig3_HTML.jpg" loading="lazy" id="d33e842" height="633" width="708" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation coefficient matrix: Two-way relationships among variables.</p></figcaption></figure><p id="Par57"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/19d3f1c4232c/41598_2025_12758_Article_IEq1.gif" loading="lazy" alt="Inline graphic"></span>2</p>
<p id="Par58">In the context of correlation analysis, <em>Z</em>ˉ represents the mean of the second variable, while <em>Z</em> denotes the second variable itself.</p>
<p id="Par59">According to the information shown in Fig. <a href="#Fig3" class="usa-link">3</a>, it is clear that every input variable shows a connection with polymers solubility parameter. Significantly, boiling point, melting point, dipole moment, dielectric constant and refractive index, show a beneficial effect, while molecular weight, liquid molar volume, van der Waals area, radius of gyration and reduced volume, and parachor present an adverse correlation with the solubility parameter.</p>
<p id="Par61">Prior to developing machine learning models to predict the solubility parameter of polymers, it is essential to ensure data reliability through the management of outliers. This research uses the Monte Carlo Outlier Detection (MCOD) algorithm, which effectively identifies outliers in extensive datasets by integrating random selection using density-driven techniques. To improve data quality, we applied a Monte Carlo Outlier Detection (MCOD) algorithm, which combines random sampling with local density estimation to identify and exclude outliers. This technique helps reduce computational complexity while retaining the representativeness of the dataset for model training. A boxplot summarizing the data distribution after applying MCOD is shown in Fig. <a href="#Fig4" class="usa-link">4</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h3 class="obj_head">Fig. 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/e150d34c6009/41598_2025_12758_Fig4_HTML.jpg" loading="lazy" id="d33e863" height="626" width="794" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Identifying outliers utilizing the monte carlo algorithm, (B) Boxplot of data distribution.</p></figcaption></figure><p id="Par62">Figure <a href="#Fig4" class="usa-link">4</a> presents a boxplot representation of databank employed in this research, showcasing the data’s distribution and defining the range considered appropriate for model creation. As illustrated, most data points reside within the allowable range, indicating excellent data quality. In this study, each data point from the collected dataset was used to train the machine learning methods. This method ensures that the models are built on a thorough databank, thus improving their ability to efficiently generalize to new, unknown databank. By including full spectrum of data, the algorithms are more adept at recognizing the fundamental patterns and discrepancies, which leads to more dependable and precise forecasts for determining the solubility parameter of polymers in different solvents.</p></section><section id="Sec18"><h2 class="pmc_sec_title">Results and discussions</h2>
<p id="Par64">In this part, we perform a thorough evaluation of the dependability and efficacy of created data-driven models for forecasting the solubility parameter of various polymers. In order to achieve this, we utilize different machine learning methods, like ridge regression, Convolutional Neural Networks (CNN), linear regression, Artificial Neural Networks (ANN), elastic net, Support Vector Regression (SVR), lasso regression, Random Forest (RF), Gradient Boosting Machines (GBM), K-Nearest Neighbors (KNN), Decision Trees (DT), LightGBM, XGBoost, and CatBoost.</p>
<p id="Par65">In order to evaluate the effectiveness of these models, a range of metrics are employed, like R-squared (R²), average squared error (MSE), mean relative deviation percentage (MRD%), and standard deviation of residuals (σ). The meanings of these metrics are outlined below:</p>
<section class="tw xbox font-sm" id="Taba"><div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/bf8a077446cd/41598_2025_12758_Article_IEq2.gif" loading="lazy" alt="Inline graphic"></span>
</th>
<th align="left" colspan="1" rowspan="1">2</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/8bac9b2135ed/41598_2025_12758_Article_IEq3.gif" loading="lazy" alt="Inline graphic"></span>
</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/e1f1cffd774b/41598_2025_12758_Article_IEq4.gif" loading="lazy" alt="Inline graphic"></span>
</td>
<td align="center" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/8d169c436e23/41598_2025_12758_Article_IEq5.gif" loading="lazy" alt="Inline graphic"></span>
</td>
<td align="center" colspan="1" rowspan="1">5</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Taba/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par67">Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/991b2f612fbb/41598_2025_12758_Article_IEq6.gif" loading="lazy" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/0a0745c91248/41598_2025_12758_Article_IEq7.gif" loading="lazy" alt="Inline graphic"></span> represent predicted and actual target values, while N denotes the total number of data points in dataset.</p>
<p id="Par68">As per Table <a href="#Tab1" class="usa-link">1</a>, the highest-performing models recognized are ANN, CNN, and Catboost. Table <a href="#Tab1" class="usa-link">1</a> validates their exceptional performance, as CNN records the highest R² values (training: 0.941, validation: 0.838, and testing: 0.914), the lowest RMSE (training: 1079, validation 1415, and testing: 1329), and the lowest MRD% (training: 3.8%, validation:4.3%, and testing: 4.6%). Likewise, ANN and Catboost show outstanding performance, exhibiting high R² metrics, minimal MSE, and relatively low MRD%, which affirms their reliability in predictive tasks.</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Assessment metrics for machine learning models: performance during all segments.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Model</th>
<th align="left" colspan="3" rowspan="1">
<em>R</em>²</th>
<th align="left" colspan="3" rowspan="1">RMSE</th>
<th align="left" colspan="3" rowspan="1">MRD%</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Validation</th>
<th align="left" colspan="1" rowspan="1">Testing</th>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Validation</th>
<th align="left" colspan="1" rowspan="1">Testing</th>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Validation</th>
<th align="left" colspan="1" rowspan="1">Testing</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Linear Regression</td>
<td align="center" colspan="1" rowspan="1">0.751</td>
<td align="center" colspan="1" rowspan="1">0.745</td>
<td align="center" colspan="1" rowspan="1">0.763</td>
<td align="center" colspan="1" rowspan="1">2223</td>
<td align="center" colspan="1" rowspan="1">2040</td>
<td align="center" colspan="1" rowspan="1">2242</td>
<td align="center" colspan="1" rowspan="1">11.5</td>
<td align="center" colspan="1" rowspan="1">11.3</td>
<td align="center" colspan="1" rowspan="1">6.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ridge Regression</td>
<td align="center" colspan="1" rowspan="1">0.751</td>
<td align="center" colspan="1" rowspan="1">0.748</td>
<td align="center" colspan="1" rowspan="1">0.763</td>
<td align="center" colspan="1" rowspan="1">2223</td>
<td align="center" colspan="1" rowspan="1">2029</td>
<td align="center" colspan="1" rowspan="1">2238</td>
<td align="center" colspan="1" rowspan="1">13.3</td>
<td align="center" colspan="1" rowspan="1">12.8</td>
<td align="center" colspan="1" rowspan="1">6.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Lasso Regression</td>
<td align="center" colspan="1" rowspan="1">0.751</td>
<td align="center" colspan="1" rowspan="1">0.751</td>
<td align="center" colspan="1" rowspan="1">0.765</td>
<td align="center" colspan="1" rowspan="1">2224</td>
<td align="center" colspan="1" rowspan="1">2016</td>
<td align="center" colspan="1" rowspan="1">2232</td>
<td align="center" colspan="1" rowspan="1">11.4</td>
<td align="center" colspan="1" rowspan="1">16.7</td>
<td align="center" colspan="1" rowspan="1">6.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Elastic Net</td>
<td align="center" colspan="1" rowspan="1">0.751</td>
<td align="center" colspan="1" rowspan="1">0.749</td>
<td align="center" colspan="1" rowspan="1">0.764</td>
<td align="center" colspan="1" rowspan="1">2223</td>
<td align="center" colspan="1" rowspan="1">2023</td>
<td align="center" colspan="1" rowspan="1">2236</td>
<td align="center" colspan="1" rowspan="1">15.6</td>
<td align="center" colspan="1" rowspan="1">14.3</td>
<td align="center" colspan="1" rowspan="1">6.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gradient Boosting</td>
<td align="center" colspan="1" rowspan="1">0.987</td>
<td align="center" colspan="1" rowspan="1">0.884</td>
<td align="center" colspan="1" rowspan="1">0.888</td>
<td align="center" colspan="1" rowspan="1">503</td>
<td align="center" colspan="1" rowspan="1">1376</td>
<td align="center" colspan="1" rowspan="1">1540</td>
<td align="center" colspan="1" rowspan="1">2.0</td>
<td align="center" colspan="1" rowspan="1">4.3</td>
<td align="center" colspan="1" rowspan="1">5.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Random Forest</td>
<td align="center" colspan="1" rowspan="1">0.987</td>
<td align="center" colspan="1" rowspan="1">0.884</td>
<td align="center" colspan="1" rowspan="1">0.888</td>
<td align="center" colspan="1" rowspan="1">503</td>
<td align="center" colspan="1" rowspan="1">1376</td>
<td align="center" colspan="1" rowspan="1">1540</td>
<td align="center" colspan="1" rowspan="1">2.0</td>
<td align="center" colspan="1" rowspan="1">4.3</td>
<td align="center" colspan="1" rowspan="1">5.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">XGBoost</td>
<td align="center" colspan="1" rowspan="1">1.000</td>
<td align="center" colspan="1" rowspan="1">0.827</td>
<td align="center" colspan="1" rowspan="1">0.828</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="center" colspan="1" rowspan="1">1678</td>
<td align="center" colspan="1" rowspan="1">1910</td>
<td align="center" colspan="1" rowspan="1">0.0</td>
<td align="center" colspan="1" rowspan="1">5.1</td>
<td align="center" colspan="1" rowspan="1">4.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LightGBM</td>
<td align="center" colspan="1" rowspan="1">0.960</td>
<td align="center" colspan="1" rowspan="1">0.847</td>
<td align="center" colspan="1" rowspan="1">0.849</td>
<td align="center" colspan="1" rowspan="1">887</td>
<td align="center" colspan="1" rowspan="1">1577</td>
<td align="center" colspan="1" rowspan="1">1786</td>
<td align="center" colspan="1" rowspan="1">1.7</td>
<td align="center" colspan="1" rowspan="1">4.6</td>
<td align="center" colspan="1" rowspan="1">5.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CatBoost</td>
<td align="center" colspan="1" rowspan="1">0.961</td>
<td align="center" colspan="1" rowspan="1">0.870</td>
<td align="center" colspan="1" rowspan="1">0.893</td>
<td align="center" colspan="1" rowspan="1">875</td>
<td align="center" colspan="1" rowspan="1">1455</td>
<td align="center" colspan="1" rowspan="1">1503</td>
<td align="center" colspan="1" rowspan="1">3.3</td>
<td align="center" colspan="1" rowspan="1">4.3</td>
<td align="center" colspan="1" rowspan="1">6.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SVR</td>
<td align="center" colspan="1" rowspan="1">0.904</td>
<td align="center" colspan="1" rowspan="1">0.884</td>
<td align="center" colspan="1" rowspan="1">0.863</td>
<td align="center" colspan="1" rowspan="1">1382</td>
<td align="center" colspan="1" rowspan="1">1373</td>
<td align="center" colspan="1" rowspan="1">1705</td>
<td align="center" colspan="1" rowspan="1">2.7</td>
<td align="center" colspan="1" rowspan="1">3.3</td>
<td align="center" colspan="1" rowspan="1">4.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KNN</td>
<td align="center" colspan="1" rowspan="1">0.877</td>
<td align="center" colspan="1" rowspan="1">0.801</td>
<td align="center" colspan="1" rowspan="1">0.792</td>
<td align="center" colspan="1" rowspan="1">1560</td>
<td align="center" colspan="1" rowspan="1">1803</td>
<td align="center" colspan="1" rowspan="1">2097</td>
<td align="center" colspan="1" rowspan="1">4.2</td>
<td align="center" colspan="1" rowspan="1">5.6</td>
<td align="center" colspan="1" rowspan="1">4.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Decision Tree</td>
<td align="center" colspan="1" rowspan="1">1.000</td>
<td align="center" colspan="1" rowspan="1">0.744</td>
<td align="center" colspan="1" rowspan="1">0.695</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">2045</td>
<td align="center" colspan="1" rowspan="1">2540</td>
<td align="center" colspan="1" rowspan="1">0.0</td>
<td align="center" colspan="1" rowspan="1">7.2</td>
<td align="center" colspan="1" rowspan="1">3.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ANN</td>
<td align="center" colspan="1" rowspan="1">0.968</td>
<td align="center" colspan="1" rowspan="1">0.838</td>
<td align="center" colspan="1" rowspan="1">0.914</td>
<td align="center" colspan="1" rowspan="1">798</td>
<td align="center" colspan="1" rowspan="1">1626</td>
<td align="center" colspan="1" rowspan="1">1351</td>
<td align="center" colspan="1" rowspan="1">3.0</td>
<td align="center" colspan="1" rowspan="1">4.3</td>
<td align="center" colspan="1" rowspan="1">4.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CNN</td>
<td align="center" colspan="1" rowspan="1">0.941</td>
<td align="center" colspan="1" rowspan="1">0.877</td>
<td align="center" colspan="1" rowspan="1">0.917</td>
<td align="center" colspan="1" rowspan="1">1079</td>
<td align="center" colspan="1" rowspan="1">1415</td>
<td align="center" colspan="1" rowspan="1">1329</td>
<td align="center" colspan="1" rowspan="1">3.8</td>
<td align="center" colspan="1" rowspan="1">4.3</td>
<td align="center" colspan="1" rowspan="1">4.6</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par69">In contrast, simpler models like Linear Regression, Decision Tree, and XGBoost demonstrate significantly lower performance metrics when compared to more advanced models. These models achieve R² values ranging from 0.70 to 0.83, which are notably lower than those of top-performing models. Additionally, they exhibit substantially higher Root Mean Squared Error (RMSE) values, varying between 1910 and 2540 indicating larger deviations from actual values.</p>
<p id="Par70">The Decision Tree and XGBoost models, in particular, display perfect performance on the training data (R² = 1.0000, MSE = 0), but this is indicative of severe overfitting. As a result, its performance deteriorates markedly on validation and test sets, revealing poor generalization capabilities. Similarly, the KNN model demonstrates relatively acceptable R² values (0.88 for training, 0.80 for validation, and 0.79 for testing). However, its higher MRD% values in both validation (5.6%) and test sets (4%) suggest that it incurs greater relative errors compared to the leading models. These results highlight the constraints of these models in effectively capturing complex patterns within the data and emphasize the significance of selecting more sophisticated algorithms for improved predictive performance.</p>
<p id="Par71">In summary, CNN, ANN, and Catboost emerge as the most precise and reliable frameworks for predicting the solubility parameter of diverse polymers, as evidenced by their robust performance across all evaluation metrics. Their superior performance is consistently demonstrated across all evaluation metrics, showcasing their ability to effectively manage the intricacies and fluctuations present in polymer datasets. These gradient boosting-based algorithms not only achieve high predictive accuracy but also exhibit strong generalization abilities, creating them well-suited for this challenging job. The robustness of these models regarding metrics like R², Root Mean Squared Error (RMSE), and Mean Relative Deviation (MRD%) further solidifies their position as leading choices for solubility parameter prediction in polymer science applications.</p>
<p id="Par73">This study utilizes graphical techniques, including relative deviation values and crossplots, to evaluate the efficiency of machine learning models designed for forecasting the solubility parameter of various polymers. These visualization methods play an essential role in enhancing evaluation process by providing clear and intuitive comparisons of model performance. By plotting predicted solubility parameter values against actual values, these graphical tools facilitate a deeper understanding of the models’ accuracy and reliability. They also help identify any discrepancies or error patterns that may not be immediately apparent from numerical metrics alone.</p>
<p id="Par74">Relative deviation values offer a quantitative measure of the disparities between forecasted and real outcomes, enabling a more nuanced evaluation of model precision. Crossplots, on the other hand, visually represent these deviations, allowing for easy identification of trends, outliers, or systematic biases in the predictions. Together, these techniques offer valuable perspectives on the strength and consistency of models, reinforcing overall assessment of their ability to accurately predict the solubility parameter of polymers across diverse datasets. This graphical approach ensures a comprehensive and reliable evaluation framework, supporting the development of effective predictive models in polymer science.</p>
<p id="Par75">Figure <a href="#Fig5" class="usa-link">5</a> provides a comparative visualization of actual data points versus modeled data points based on the data point index for all developed models, covering all phases for the solubility parameters of various polymers. As depicted, the actual and predicted values for CatBoost, ANN, and CNN closely overlap, highlighting that these machine learning algorithms outperform all other methods evaluated in this study. This near-perfect alignment underscores their superior ability to generalize across different datasets.</p>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig5a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/ebe2489b2c3b/41598_2025_12758_Fig5a_HTML.jpg" loading="lazy" id="d33e1341" height="1130" width="752" alt="Fig. 5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig5b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/93aa42d9865e/41598_2025_12758_Fig5b_HTML.jpg" loading="lazy" id="d33e1342" height="1135" width="752" alt="Fig. 5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig5c_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/5f0c860b0f8c/41598_2025_12758_Fig5c_HTML.jpg" loading="lazy" id="d33e1343" height="1137" width="752" alt="Fig. 5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig5d_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/5cfee099d17f/41598_2025_12758_Fig5d_HTML.jpg" loading="lazy" id="d33e1344" height="1134" width="752" alt="Fig. 5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig5e_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/83c88708e621/41598_2025_12758_Fig5e_HTML.jpg" loading="lazy" id="d33e1345" height="746" width="751" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison between actual and forecasted values: all segments for all data-driven models.</p></figcaption></figure><p id="Par76">Furthermore, Fig. <a href="#Fig6" class="usa-link">6</a> presents cross-plots comparing actual versus estimated values for all machine learning algorithms. A dense clustering of datapoints around the line y = x is clearly visible for CatBoost, ANN, and CNN, reinforcing their exceptional foretelling accuracy. The fitted lines derived from these cross-plot points for the three algorithms closely adhere to line y = x, indicating a strong linear correlation between real solubility values and estimations generated by these models.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig6a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/9ae329f3e611/41598_2025_12758_Fig6a_HTML.jpg" loading="lazy" id="d33e1355" height="949" width="710" alt="Fig. 6"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig6b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/00c7e924dfc4/41598_2025_12758_Fig6b_HTML.jpg" loading="lazy" id="d33e1356" height="705" width="710" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cross plots: Modeled versus real points for all data across all data-driven algorithms.</p></figcaption></figure><p id="Par77">These visual analyses collectively demonstrate the robustness and reliability of CatBoost, ANN, and CNN in predicting the solubility factors of diverse polymers. The high degree of correspondence between actual and predicted values not only validates exceptional performance of these algorithms but also confirms their suitability for real-world applications in polymer science and engineering.</p>
<p id="Par78">Figure <a href="#Fig7" class="usa-link">7</a> presents scatter plots that show relative errors of developed algorithms in predicting the solubility parameters of various polymers. The distribution of these error values around the x-axis is particularly evident for CatBoost, ANN, and CNN models. This illustration emphasizes the slight difference between their predictions and the real solubility values, reinforcing strong alignment between the predicted and true values. Such a tight clustering of errors around zero indicates that these models exhibit high precision and accuracy in their predictions.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig7a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/9c5c47c700c8/41598_2025_12758_Fig7a_HTML.jpg" loading="lazy" id="d33e1366" height="946" width="708" alt="Fig. 7"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig7b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/c04da2ca9aa1/41598_2025_12758_Fig7b_HTML.jpg" loading="lazy" id="d33e1367" height="702" width="708" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Percentage of relative deviation: Training, testing, and validation phases for all data-driven models.</p></figcaption></figure><p id="Par79">Figure <a href="#Fig8" class="usa-link">8</a> provides an additional layer of insight by showcasing the prediction distribution capabilities of all developed machine learning models across all phases. Notably, estimation profiles for CatBoost, ANN, and CNN show greater consistency across these phases compared to other methods. This consistency suggests that these Models excel not only on the training dataset but also generalize efficiently to unfamiliar testing and validation datasets.</p>
<figure class="fig xbox font-sm" id="Fig8"><h3 class="obj_head">Fig. 8.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig8a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/67cb22355a1d/41598_2025_12758_Fig8a_HTML.jpg" loading="lazy" id="d33e1377" height="953" width="708" alt="Fig. 8"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig8b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/40905427e61a/41598_2025_12758_Fig8b_HTML.jpg" loading="lazy" id="d33e1378" height="705" width="708" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of frequencies: All phases for all data-driven algorithms.</p></figcaption></figure><p id="Par80">The ability of CatBoost, ANN, and CNN to maintain consistent performance across different phases further validates their robustness and reliability. These findings collectively confirm that these three models are the most effective and dependable choices for estimating the solubility parameters of diverse polymers within the context of this study. Their superior performance makes them highly suitable for practical applications in polymer science and related fields.</p>
<p id="Par85">Evaluating the importance of input features is a critical step in comprehending how different factors affect the prediction of solubility parameters for diverse polymers using machine learning models. In this research, the SHAP (Shapley Additive Explanations) method is employed to assess feature importance, offering a robust framework for interpreting the complexities of high-performance models. Rooted in game theory, the SHAP method assigns contribution values to every datapoint based on its input features, thereby enhancing algorithm interpretability at aggregate and individual levels. This approach offers important perspectives on the role of each input variable in shaping target output in this case, the solubility parameters of polymers.</p>
<p id="Par86">Figure <a href="#Fig9" class="usa-link">9</a> presents the SHAP values for the input parameters and their corresponding importance, as established by the CNN model. The features are ranked from highest to lowest according to their mean SHAP values, with higher ranks indicating a greater influence on the solubility parameter estimations. The SHAP analysis identifies the dielectric constant as the most influential feature in determining the solubility parameter of polymers. This observation is consistent with the physicochemical interpretation that the dielectric constant reflects the polarity of a material. Polymers with higher dielectric constants tend to have stronger dipole–dipole interactions and enhanced compatibility with polar solvents, leading to higher solubility parameters. This relationship has been previously reported in polymer-solvent compatibility studies<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>,<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>, where higher polarity generally correlates with greater cohesive energy density. Conversely, materials with low dielectric constants exhibit weaker intermolecular interactions and thus lower solubility parameters, which the model accurately captures.</p>
<figure class="fig xbox font-sm" id="Fig9"><h3 class="obj_head">Fig. 9.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375797_41598_2025_12758_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/178a/12375797/33f1539147f8/41598_2025_12758_Fig9_HTML.jpg" loading="lazy" id="d33e1417" height="562" width="750" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SHAP analysis based on the CNN model.</p></figcaption></figure><p id="Par87">In addition to dielectric constant, other top-ranking features such as molecular weight, liquid molar volume, and melting point also contribute significantly. Molecular weight typically exhibits a negative correlation with solubility due to the increased rigidity and reduced free volume in higher molecular weight polymers, which hinders solvation<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Similarly, higher liquid molar volumes often indicate bulkier molecules, resulting in lower cohesive energy density and reduced solubility. Melting point, as an indicator of crystalline strength and intermolecular forces, tends to have a positive correlation, suggesting that polymers with higher melting points often possess stronger intermolecular cohesion, which influences their solubility characteristics<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. These insights reinforce the interpretability of the model and its alignment with established physicochemical principles.</p>
<p id="Par88">By providing a detailed breakdown of feature contributions, the SHAP method not only enhances the transparency of the Random Forest model but also offers actionable insights for improving predictive accuracy and understanding the underlying mechanisms governing polymer solubility. This analysis serves as a foundation for optimizing polymer design and processing strategies, making it an invaluable tool in the field of polymer science and engineering.</p>
<p id="Par90">While several recent studies have employed machine learning to predict solubility parameters of polymers<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>,<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>, many of these efforts suffer from limitations such as the use of relatively small or domain-specific datasets, limited model diversity (e.g., reliance on linear models or single-layer ANNs), and a lack of interpretability analysis regarding feature importance. In contrast, the present study utilizes a significantly larger and more diverse dataset (1,799 polymers) extracted from the DIPPR 801 database, covering a broad range of physicochemical descriptors. Moreover, we implement and compare a wide variety of both classical and advanced machine learning models including CNN, CatBoost, and ANN and evaluate them comprehensively using multiple performance metrics and visualization techniques. Importantly, the incorporation of SHAP analysis allows us to go beyond predictive accuracy and offer insights into the relative importance and physical relevance of each input feature, thus improving model transparency and scientific interpretability. These advancements differentiate this work from prior studies and address common weaknesses in earlier approaches.</p></section><section id="Sec19"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par91">This paper is centered on development of data-driven models leveraging a wide array of machine learning algorithms to estimate solubility parameters of diverse polymers. The algorithms employed include Random Forests (RFs), Linear Regression, K-Nearest Neighbors (KNN), Convolutional Neural Networks (CNNs), Artificial Neural Networks (ANNs), Ridge Regression, Lasso Regression, Elastic Net, Support Vector Machines (SVMs), Extreme Gradient Boosting (XGBoost), Decision Trees (DTs), Light Gradient Boosting Machines (LightGBM), Gradient Boosting Machines (GBMs), and Categorical Boosting (CatBoost). These models were trained using key physicochemical descriptors such as molecular weight, melting point, liquid molar volume, radius of gyration, boiling point, dielectric constant, dipole moment, van der Waals area, refractive index and reduced volume, and parachor. To assess the effectiveness of the developed models, a combination of quantitative metrics and visualization tools was utilized. The Monte Carlo outlier detection method was utilized to guarantee the databank’s reliability, confirming that most of the information points were suitable for validation and training purposes. The superior performance of CatBoost, ANN, and CNN can be attributed to their inherent ability to capture complex, non-linear relationships and intricate feature interactions within the dataset. CatBoost, as a gradient boosting method tailored for handling categorical and numerical features, efficiently models non-linearities while mitigating overfitting through ordered boosting and advanced regularization. ANN and CNN, on the other hand, are deep learning architectures capable of learning hierarchical patterns and internal representations without explicit feature engineering. In particular, CNNs leverage local connectivity and weight sharing mechanisms that help identify spatial correlations and subtle dependencies among features. These capabilities allow the models to generalize better across the diverse physicochemical properties present in the polymer dataset, resulting in more accurate and robust predictions. Further analysis revealed that specific physicochemical properties, including boiling point, melting point, dipole moment, dielectric constant, and refractive index, significantly influence the solubility favtors of polymers. Some of these factors exhibited positive relationships with solubility, while others showed negative correlations. This finding underscores the models’ capability to successfully capture intricate and nonlinear connections present in the dataset, reinforcing their suitability for predictive applications in this domain. To enhance interpretability, SHAP (Shapley Additive Explanations) values were employed to clarify contributions of individual features to the predictions. The analysis identified the dielectric constant as the most influential factor affecting the solubility parameters of diverse polymers. This insight not only validates the robustness of the selected models but also provides deeper understanding of the critical elements influencing polymer solubility. In conclusion, this comprehensive approach not only demonstrates the effectiveness of CatBoost, ANN, and CNN in accurately predicting polymer solubility parameters but also offers valuable understanding of the fundamental processes motivating these predictions. Through the integration of sophisticated machine learning methods with interpretable feature importance analysis, this research opens the door for enhanced precision and transparency in future research, enabling more informed decisions in polymer science and engineering applications.</p></section><section id="Sec20"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375797/bin/41598_2025_12758_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (36.5KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors express their gratitude to the Deanship of Scientific Research at Imam Mohammad Ibn Saud Islamic University (IMSIU) for supporting this research project (grant number IMSIU-DDRSP2502).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Raouf Hassan: Data curation management, Modeling, Data curation, Drafted the initial manuscript, Reviewed and edited the manuscript.Mohammad Reza Kazemi: Conceptualization, Analyses, Reviewed and edited the manuscript, Supervision.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding statement</h2>
<p>This research received financial support from the Deanship of Scientific Research at Imam Mohammad Ibn Saud Islamic University (IMSIU) (grant number IMSIU-DDRSP2502).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data that support the findings of this study are available from the corresponding author, Mohammad Reza Kezemi, upon reasonable request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par92">The authors declare no competing interests.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Authors contribution statement</h3>
<p id="Par93">Raouf Hassan: Data management curation, Modeling, Curation of data, Created the first draft of the manuscript, Evaluated and revised the manuscript.</p>
<p id="Par94">Mohammad Reza Kazemi: Conceptualization, Analysis, Manuscript review and editing, Oversight.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Scatchard, G. Equilibria in Non-electrolyte solutions in relation to the vapor pressures and densities of the components. <em>Chem. Rev.</em><strong>8</strong> (2), 321–333 (1931).</cite> [<a href="https://scholar.google.com/scholar_lookup?Scatchard,%20G.%20Equilibria%20in%20Non-electrolyte%20solutions%20in%20relation%20to%20the%20vapor%20pressures%20and%20densities%20of%20the%20components.%20Chem.%20Rev.8%20(2),%20321%E2%80%93333%20(1931)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Hildebrand, J. H. &amp; Scott, R. L. <em>The Solubility of Nonelectrolytes, Reinhold Pub</em>3 (Co., 1950).</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Hildebrand, J. H. A critique of the theory of solubility of non-electrolytes. <em>Chem. Rev.</em><strong>44</strong> (1), 37–45 (1949).
</cite> [<a href="https://doi.org/10.1021/cr60137a003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18125399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hildebrand,%20J.%20H.%20A%20critique%20of%20the%20theory%20of%20solubility%20of%20non-electrolytes.%20Chem.%20Rev.44%20(1),%2037%E2%80%9345%20(1949)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Hansen, C. M. 50 years with solubility parameters—past and future. <em>Prog. Org. Coat.</em><strong>51</strong> (1), 77–84 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20C.%20M.%2050%20years%20with%20solubility%20parameters%E2%80%94past%20and%20future.%20Prog.%20Org.%20Coat.51%20(1),%2077%E2%80%9384%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Sun, R. &amp; Elabd, Y. A. Synthesis and high alkaline chemical stability of polyionic liquids with methylpyrrolidinium, methylpiperidinium, methylazepanium, methylazocanium, and Methylazonanium cations. <em>ACS Macro Lett.</em><strong>8</strong> (5), 540–545 (2019).
</cite> [<a href="https://doi.org/10.1021/acsmacrolett.9b00039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35619359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20R.%20&amp;%20Elabd,%20Y.%20A.%20Synthesis%20and%20high%20alkaline%20chemical%20stability%20of%20polyionic%20liquids%20with%20methylpyrrolidinium,%20methylpiperidinium,%20methylazepanium,%20methylazocanium,%20and%20Methylazonanium%20cations.%20ACS%20Macro%20Lett.8%20(5),%20540%E2%80%93545%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Barton, A. F. M. <em>CRC Handbook of Solubility Parameters and Other Cohesion Parameters</em> (Routledge, 2017).</cite>
</li>
<li id="CR7">
<span class="label">7.</span><cite>Hansen, C. M. <em>The Three Dimensional Solubility Parameter</em>14 (Copenhagen, 1967).</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Schwaighofer, A. et al. Accurate solubility prediction with error bars for electrolytes: A machine learning approach. <em>J. Chem. Inf. Model.</em><strong>47</strong> (2), 407–424 (2007).
</cite> [<a href="https://doi.org/10.1021/ci600205g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17243756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwaighofer,%20A.%20et%20al.%20Accurate%20solubility%20prediction%20with%20error%20bars%20for%20electrolytes:%20A%20machine%20learning%20approach.%20J.%20Chem.%20Inf.%20Model.47%20(2),%20407%E2%80%93424%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Cheng, T. et al. Binary classification of aqueous solubility using support vector machines with reduction and recombination feature selection. <em>J. Chem. Inf. Model.</em><strong>51</strong> (2), 229–236 (2011).
</cite> [<a href="https://doi.org/10.1021/ci100364a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3047290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21214224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20T.%20et%20al.%20Binary%20classification%20of%20aqueous%20solubility%20using%20support%20vector%20machines%20with%20reduction%20and%20recombination%20feature%20selection.%20J.%20Chem.%20Inf.%20Model.51%20(2),%20229%E2%80%93236%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Lozada, M., Monfort, J. P. &amp; del Río, F. Calculation of solubility parameters from an equation of state. <em>Chem. Phys. Lett.</em><strong>45</strong> (1), 130–133 (1977).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lozada,%20M.,%20Monfort,%20J.%20P.%20&amp;%20del%20R%C3%ADo,%20F.%20Calculation%20of%20solubility%20parameters%20from%20an%20equation%20of%20state.%20Chem.%20Phys.%20Lett.45%20(1),%20130%E2%80%93133%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Panayiotou, C. Solubility parameter revisited: an equation-of-state approach for its Estimation. <em>Fluid. Phase. Equilibria</em>. <strong>131</strong> (1–2), 21–35 (1997).</cite> [<a href="https://scholar.google.com/scholar_lookup?Panayiotou,%20C.%20Solubility%20parameter%20revisited:%20an%20equation-of-state%20approach%20for%20its%20Estimation.%20Fluid.%20Phase.%20Equilibria.%20131%20(1%E2%80%932),%2021%E2%80%9335%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Goharshadi, E. K. &amp; Hesabi, M. Estimation of solubility parameter using equations of state. <em>J. Mol. Liq.</em><strong>113</strong> (1–3), 125–132 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Goharshadi,%20E.%20K.%20&amp;%20Hesabi,%20M.%20Estimation%20of%20solubility%20parameter%20using%20equations%20of%20state.%20J.%20Mol.%20Liq.113%20(1%E2%80%933),%20125%E2%80%93132%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Stefanis, E., Tsivintzelis, I. &amp; Panayiotou, C. The partial solubility parameters: an equation-of-state approach. <em>Fluid. Phase. Equilibria</em>. <strong>240</strong> (2), 144–154 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?Stefanis,%20E.,%20Tsivintzelis,%20I.%20&amp;%20Panayiotou,%20C.%20The%20partial%20solubility%20parameters:%20an%20equation-of-state%20approach.%20Fluid.%20Phase.%20Equilibria.%20240%20(2),%20144%E2%80%93154%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Van Krevelen, D. W., Te, K. &amp; Nijenhuis <em>Properties of Polymers: their Correlation with Chemical Structure; their Numerical Estimation and Prediction from Additive Group Contributions</em> (Elsevier, 2009).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Wang, B. et al. Prediction of minimum ignition energy from molecular structure using quantitative structure–property relationship (QSPR) models. <em>Ind. Eng. Chem. Res.</em><strong>56</strong> (1), 47–51 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20B.%20et%20al.%20Prediction%20of%20minimum%20ignition%20energy%20from%20molecular%20structure%20using%20quantitative%20structure%E2%80%93property%20relationship%20(QSPR)%20models.%20Ind.%20Eng.%20Chem.%20Res.56%20(1),%2047%E2%80%9351%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Jiao, Z. et al. Review of recent developments of quantitative structure-property relationship models on fire and explosion-related properties. <em>Process Saf. Environ. Prot.</em><strong>129</strong>, 280–290 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jiao,%20Z.%20et%20al.%20Review%20of%20recent%20developments%20of%20quantitative%20structure-property%20relationship%20models%20on%20fire%20and%20explosion-related%20properties.%20Process%20Saf.%20Environ.%20Prot.129,%20280%E2%80%93290%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Jiao, Z. et al. <em>Machine Learning and Deep Learning in Chemical Health and Safety: a Systematic Review of Techniques and Applications</em>27p. 316–334 (ACS Chemical Health &amp; Safety, 2020). 6.</cite>
</li>
<li id="CR18">
<span class="label">18.</span><cite>Morimoto, M. et al. Convolutional neural networks for fluid flow analysis: toward effective metamodeling and low dimensionalization. <em>Theoret. Comput. Fluid Dyn.</em><strong>35</strong> (5), 633–658 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Morimoto,%20M.%20et%20al.%20Convolutional%20neural%20networks%20for%20fluid%20flow%20analysis:%20toward%20effective%20metamodeling%20and%20low%20dimensionalization.%20Theoret.%20Comput.%20Fluid%20Dyn.35%20(5),%20633%E2%80%93658%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Yu, Y. et al. Torsional capacity evaluation of RC beams using an improved bird swarm algorithm optimised 2D convolutional neural network. <em>Eng. Struct.</em><strong>273</strong>, 115066 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yu,%20Y.%20et%20al.%20Torsional%20capacity%20evaluation%20of%20RC%20beams%20using%20an%20improved%20bird%20swarm%20algorithm%20optimised%202D%20convolutional%20neural%20network.%20Eng.%20Struct.273,%20115066%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Wei, S. et al. A quantum convolutional neural network on NISQ devices. <em>AAPPS Bull.</em><strong>32</strong>, 1–11 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wei,%20S.%20et%20al.%20A%20quantum%20convolutional%20neural%20network%20on%20NISQ%20devices.%20AAPPS%20Bull.32,%201%E2%80%9311%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Li, J. &amp; Yang, H. The underwater acoustic target timbre perception and recognition based on the auditory inspired deep convolutional neural network. <em>Appl. Acoust.</em><strong>182</strong>, 108210 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.%20&amp;%20Yang,%20H.%20The%20underwater%20acoustic%20target%20timbre%20perception%20and%20recognition%20based%20on%20the%20auditory%20inspired%20deep%20convolutional%20neural%20network.%20Appl.%20Acoust.182,%20108210%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Khandokar, I. et al. <em>Handwritten Character Recognition Using Convolutional Neural Network</em>. IOP Publishing. <strong>1918 </strong>(4), 042152 (2021).</cite>
</li>
<li id="CR23">
<span class="label">23.</span><cite>Lu, J. et al. Enhanced K-nearest neighbor for intelligent fault diagnosis of rotating machinery. <em>Appl. Sci.</em><strong>11</strong> (3), 919 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lu,%20J.%20et%20al.%20Enhanced%20K-nearest%20neighbor%20for%20intelligent%20fault%20diagnosis%20of%20rotating%20machinery.%20Appl.%20Sci.11%20(3),%20919%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kang, S. K-nearest neighbor learning with graph neural networks. <em>Mathematics</em><strong>9</strong> (8), 830 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kang,%20S.%20K-nearest%20neighbor%20learning%20with%20graph%20neural%20networks.%20Mathematics9%20(8),%20830%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Gallego, A. J., Rico-Juan, J. R. &amp; Valero-Mas, J. J. Efficient k-nearest neighbor search based on clustering and adaptive k values. <em>Pattern Recogn.</em><strong>122</strong>, 108356 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gallego,%20A.%20J.,%20Rico-Juan,%20J.%20R.%20&amp;%20Valero-Mas,%20J.%20J.%20Efficient%20k-nearest%20neighbor%20search%20based%20on%20clustering%20and%20adaptive%20k%20values.%20Pattern%20Recogn.122,%20108356%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Assegie, T. A. An optimized K-Nearest neighbor based breast cancer detection. <em>J. Rob. Control (JRC)</em>. <strong>2</strong> (3), 115–118 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Assegie,%20T.%20A.%20An%20optimized%20K-Nearest%20neighbor%20based%20breast%20cancer%20detection.%20J.%20Rob.%20Control%20(JRC).%202%20(3),%20115%E2%80%93118%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Lee, J. H., Shi, Z. &amp; Gao, Z. On LASSO for predictive regression. <em>J. Econ.</em><strong>229</strong> (2), 322–349 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.%20H.,%20Shi,%20Z.%20&amp;%20Gao,%20Z.%20On%20LASSO%20for%20predictive%20regression.%20J.%20Econ.229%20(2),%20322%E2%80%93349%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Tibshirani, R. Regression shrinkage and selection via the Lasso. <em>J. Royal Stat. Soc. Ser. B: Stat. Methodol.</em><strong>58</strong> (1), 267–288 (1996).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tibshirani,%20R.%20Regression%20shrinkage%20and%20selection%20via%20the%20Lasso.%20J.%20Royal%20Stat.%20Soc.%20Ser.%20B:%20Stat.%20Methodol.58%20(1),%20267%E2%80%93288%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Hastie, T. <em>The Elements of Statistical Learning: Data Mining, Inference, and Prediction</em> (Springer, 2009).</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Altelbany, S. Evaluation of ridge, elastic net and Lasso regression methods in precedence of multicollinearity problem: a simulation study. <em>J. Appl. Econ. Bus. Stud.</em><strong>5</strong> (1), 131–142 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Altelbany,%20S.%20Evaluation%20of%20ridge,%20elastic%20net%20and%20Lasso%20regression%20methods%20in%20precedence%20of%20multicollinearity%20problem:%20a%20simulation%20study.%20J.%20Appl.%20Econ.%20Bus.%20Stud.5%20(1),%20131%E2%80%93142%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Gholami, H. et al. Modeling land susceptibility to wind erosion hazards using LASSO regression and graph convolutional networks. <em>Front. Environ. Sci.</em><strong>11</strong>, 1187658 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gholami,%20H.%20et%20al.%20Modeling%20land%20susceptibility%20to%20wind%20erosion%20hazards%20using%20LASSO%20regression%20and%20graph%20convolutional%20networks.%20Front.%20Environ.%20Sci.11,%201187658%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ghiasi, M. M. &amp; Zendehboudi, S. Application of decision tree-based ensemble learning in the classification of breast cancer. <em>Comput. Biol. Med.</em><strong>128</strong>, 104089 (2021).
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2020.104089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33338982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghiasi,%20M.%20M.%20&amp;%20Zendehboudi,%20S.%20Application%20of%20decision%20tree-based%20ensemble%20learning%20in%20the%20classification%20of%20breast%20cancer.%20Comput.%20Biol.%20Med.128,%20104089%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Schidler, A. &amp; Szeider, S. SAT-based decision tree learning for large data sets. <em>J. Artif. Intell. Res.</em><strong>80</strong>, 875–918 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Schidler,%20A.%20&amp;%20Szeider,%20S.%20SAT-based%20decision%20tree%20learning%20for%20large%20data%20sets.%20J.%20Artif.%20Intell.%20Res.80,%20875%E2%80%93918%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Ren, Q. et al. Lithology identification using principal component analysis and particle swarm optimization fuzzy decision tree. <em>J. Petrol. Sci. Eng.</em><strong>220</strong>, 111233 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ren,%20Q.%20et%20al.%20Lithology%20identification%20using%20principal%20component%20analysis%20and%20particle%20swarm%20optimization%20fuzzy%20decision%20tree.%20J.%20Petrol.%20Sci.%20Eng.220,%20111233%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Qin, Z. et al. <em>Are neural rankers still outperformed by gradient boosted decision trees?</em>.</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Breiman, L. <em>Classification and Regression Trees</em> (Wadsworth International Group, 1984).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Rokach, L. &amp; Maimon, O. Top-down induction of decision trees classifiers-a survey. <em>IEEE Trans. Syst. Man. Cybernetics Part. C (Applications Reviews)</em>. <strong>35</strong> (4), 476–487 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rokach,%20L.%20&amp;%20Maimon,%20O.%20Top-down%20induction%20of%20decision%20trees%20classifiers-a%20survey.%20IEEE%20Trans.%20Syst.%20Man.%20Cybernetics%20Part.%20C%20(Applications%20Reviews).%2035%20(4),%20476%E2%80%93487%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Patel, R. K. et al. <em>A Logistic Regression and Decision Tree Based Hybrid Approach to Predict Alzheimer’s Disease</em>. 2023 International Conference on Computational Intelligence and Sustainable Engineering Solutions (CISES). IEEE, 2023.</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Alita, D., Putra, A. D. &amp; Darwis, D. Analysis of classic assumption test and multiple linear regression coefficient test for employee structural office recommendation. <em>IJCCS (Indonesian J. Comput. Cybernetics Systems)</em>. <strong>15</strong> (3), 295–306 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Alita,%20D.,%20Putra,%20A.%20D.%20&amp;%20Darwis,%20D.%20Analysis%20of%20classic%20assumption%20test%20and%20multiple%20linear%20regression%20coefficient%20test%20for%20employee%20structural%20office%20recommendation.%20IJCCS%20(Indonesian%20J.%20Comput.%20Cybernetics%20Systems).%2015%20(3),%20295%E2%80%93306%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Goldsmith-Pinkham, P., Hull, P. &amp; Kolesár, M. Contamination bias in linear regressions. <em>Am. Econ. Rev.</em><strong>114</strong> (12), 4015–4051 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Goldsmith-Pinkham,%20P.,%20Hull,%20P.%20&amp;%20Koles%C3%A1r,%20M.%20Contamination%20bias%20in%20linear%20regressions.%20Am.%20Econ.%20Rev.114%20(12),%204015%E2%80%934051%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Rahimov, N. &amp; Dilmurod, K. <em>The application of multiple linear regression algorithm and python for crop yield prediction in agriculture</em>. <em>Harv. Educational Sci. Rev.</em>, <strong>2</strong>(1). (2022).</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Guo, L. et al. Data-driven power flow calculation method: A lifting dimension linear regression approach. <em>IEEE Trans. Power Syst.</em><strong>37</strong> (3), 1798–1808 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Guo,%20L.%20et%20al.%20Data-driven%20power%20flow%20calculation%20method:%20A%20lifting%20dimension%20linear%20regression%20approach.%20IEEE%20Trans.%20Power%20Syst.37%20(3),%201798%E2%80%931808%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Cheshmehzangi, A. et al. A hierarchical study for urban statistical indicators on the prevalence of COVID-19 in Chinese City clusters based on multiple linear regression (MLR) and polynomial best subset regression (PBSR) analysis. <em>Sci. Rep.</em><strong>12</strong> (1), p1964 (2022).</cite> [<a href="https://doi.org/10.1038/s41598-022-05859-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8817036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35121784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheshmehzangi,%20A.%20et%20al.%20A%20hierarchical%20study%20for%20urban%20statistical%20indicators%20on%20the%20prevalence%20of%20COVID-19%20in%20Chinese%20City%20clusters%20based%20on%20multiple%20linear%20regression%20(MLR)%20and%20polynomial%20best%20subset%20regression%20(PBSR)%20analysis.%20Sci.%20Rep.12%20(1),%20p1964%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhou, J. et al. Developing a hybrid model of Jaya algorithm-based extreme gradient boosting machine to estimate blast-induced ground vibrations. <em>Int. J. Rock Mech. Min. Sci.</em><strong>145</strong>, 104856 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20J.%20et%20al.%20Developing%20a%20hybrid%20model%20of%20Jaya%20algorithm-based%20extreme%20gradient%20boosting%20machine%20to%20estimate%20blast-induced%20ground%20vibrations.%20Int.%20J.%20Rock%20Mech.%20Min.%20Sci.145,%20104856%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Rawat, R. et al. <em>Analysis of Darknet Traffic for Criminal Activities Detection Using TF-IDF and Light Gradient Boosted Machine Learning Algorithm</em>. Innovations in electrical and electronic engineering: Proceedings of ICEEE 2021, Springer Singapore. 671-681 (2021).</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Osman, M. et al. Ml-lgbm: A machine learning model based on light gradient boosting machine for the detection of version number attacks in rpl-based networks. <em>IEEE Access.</em><strong>9</strong>, 83654–83665 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Osman,%20M.%20et%20al.%20Ml-lgbm:%20A%20machine%20learning%20model%20based%20on%20light%20gradient%20boosting%20machine%20for%20the%20detection%20of%20version%20number%20attacks%20in%20rpl-based%20networks.%20IEEE%20Access.9,%2083654%E2%80%9383665%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Bibault, J. E., Chang, D. T. &amp; Xing, L. Development and validation of a model to predict survival in colorectal cancer using a gradient-boosted machine. <em>Gut</em><strong>70</strong> (5), 884–889 (2021).
</cite> [<a href="https://doi.org/10.1136/gutjnl-2020-321799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32887732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bibault,%20J.%20E.,%20Chang,%20D.%20T.%20&amp;%20Xing,%20L.%20Development%20and%20validation%20of%20a%20model%20to%20predict%20survival%20in%20colorectal%20cancer%20using%20a%20gradient-boosted%20machine.%20Gut70%20(5),%20884%E2%80%93889%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Perez, B. C. et al. Prediction performance of linear models and gradient boosting machine on complex phenotypes in outbred mice. <em>G3</em><strong>12</strong> (4), jkac039 (2022).
</cite> [<a href="https://doi.org/10.1093/g3journal/jkac039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8982369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35166767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez,%20B.%20C.%20et%20al.%20Prediction%20performance%20of%20linear%20models%20and%20gradient%20boosting%20machine%20on%20complex%20phenotypes%20in%20outbred%20mice.%20G312%20(4),%20jkac039%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Gaye, B., Zhang, D. &amp; Wulamu, A. Improvement of support vector machine algorithm in big data background. <em>Math. Probl. Eng.</em><strong>2021</strong> (1), 5594899 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gaye,%20B.,%20Zhang,%20D.%20&amp;%20Wulamu,%20A.%20Improvement%20of%20support%20vector%20machine%20algorithm%20in%20big%20data%20background.%20Math.%20Probl.%20Eng.2021%20(1),%205594899%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Zhang, H. et al. Distance-based support vector machine to predict DNA N6-methyladenine modification. <em>Curr. Bioinform.</em><strong>17</strong> (5), 473–482 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20H.%20et%20al.%20Distance-based%20support%20vector%20machine%20to%20predict%20DNA%20N6-methyladenine%20modification.%20Curr.%20Bioinform.17%20(5),%20473%E2%80%93482%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Rodríguez-Pérez, R. &amp; Bajorath, J. Evolution of support vector machine and regression modeling in chemoinformatics and drug discovery. <em>J. Comput. Aided Mol. Des.</em><strong>36</strong> (5), 355–362 (2022).
</cite> [<a href="https://doi.org/10.1007/s10822-022-00442-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9325859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35304657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodr%C3%ADguez-P%C3%A9rez,%20R.%20&amp;%20Bajorath,%20J.%20Evolution%20of%20support%20vector%20machine%20and%20regression%20modeling%20in%20chemoinformatics%20and%20drug%20discovery.%20J.%20Comput.%20Aided%20Mol.%20Des.36%20(5),%20355%E2%80%93362%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Li, C. et al. Stability prediction of hard rock pillar using support vector machine optimized by three metaheuristic algorithms. <em>Int. J. Min. Sci. Technol.</em><strong>33</strong> (8), 1019–1036 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.%20et%20al.%20Stability%20prediction%20of%20hard%20rock%20pillar%20using%20support%20vector%20machine%20optimized%20by%20three%20metaheuristic%20algorithms.%20Int.%20J.%20Min.%20Sci.%20Technol.33%20(8),%201019%E2%80%931036%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Jun, Z. <em>The Development and Application of Support Vector Machine</em>. Journal of Physics: Conference Series, IOP Publishing. <strong>1748</strong> (5), 052006 (2021).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Dasi, H., Ying, Z. &amp; Yang, B. Predicting the consumed heating energy at residential buildings using a combination of categorical boosting (CatBoost) and Meta heuristics algorithms. <em>J. Building Eng.</em><strong>71</strong>, 106584 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dasi,%20H.,%20Ying,%20Z.%20&amp;%20Yang,%20B.%20Predicting%20the%20consumed%20heating%20energy%20at%20residential%20buildings%20using%20a%20combination%20of%20categorical%20boosting%20(CatBoost)%20and%20Meta%20heuristics%20algorithms.%20J.%20Building%20Eng.71,%20106584%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Rondinella, F. et al. Improved predictions of asphalt concretes’ dynamic modulus and phase angle using decision-tree based categorical boosting model. <em>Constr. Build. Mater.</em><strong>400</strong>, 132709 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rondinella,%20F.%20et%20al.%20Improved%20predictions%20of%20asphalt%20concretes%E2%80%99%20dynamic%20modulus%20and%20phase%20angle%20using%20decision-tree%20based%20categorical%20boosting%20model.%20Constr.%20Build.%20Mater.400,%20132709%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Hussain, S. et al. A novel feature engineered-CatBoost-based supervised machine learning framework for electricity theft detection. <em>Energy Rep.</em><strong>7</strong>, 4425–4436 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hussain,%20S.%20et%20al.%20A%20novel%20feature%20engineered-CatBoost-based%20supervised%20machine%20learning%20framework%20for%20electricity%20theft%20detection.%20Energy%20Rep.7,%204425%E2%80%934436%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Wang, Y. <em>Personality Type Prediction Using Decision Tree, Gbdt, and Cat Boost</em>. 2022 International conference on big data, information and computer network (BDICN), IEEE. 552-558 (2022).</cite>
</li>
<li id="CR58">
<span class="label">58.</span><cite>Kim, J. S. Covid-19 prediction and detection using machine learning algorithms: catboost and linear regression. <em>Am. J. Theoretical Appl. Stat.</em><strong>10</strong> (5), 208 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kim,%20J.%20S.%20Covid-19%20prediction%20and%20detection%20using%20machine%20learning%20algorithms:%20catboost%20and%20linear%20regression.%20Am.%20J.%20Theoretical%20Appl.%20Stat.10%20(5),%20208%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Lalwani, P. &amp; Ramasamy, G. <em>Hybrid Principal Component Analysis Using Boosting Classification Techniques: Categorical Boosting</em>. International Conference on Data Science and Applications, Springer Nature Singapore. 229-242 (2023).</cite>
</li>
<li id="CR60">
<span class="label">60.</span><cite>Linka, K. &amp; Kuhl, E. A new family of constitutive artificial neural networks towards automated model discovery. <em>Comput. Methods Appl. Mech. Eng.</em><strong>403</strong>, 115731 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Linka,%20K.%20&amp;%20Kuhl,%20E.%20A%20new%20family%20of%20constitutive%20artificial%20neural%20networks%20towards%20automated%20model%20discovery.%20Comput.%20Methods%20Appl.%20Mech.%20Eng.403,%20115731%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Indira, D. et al. [Retracted] improved artificial neural network with state order dataset Estimation for brain Cancer cell diagnosis. <em>Biomed. Res. Int.</em><strong>2022</strong> (1), 7799812 (2022).
</cite> [<a href="https://doi.org/10.1155/2022/7799812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9038414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35480141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Indira,%20D.%20et%20al.%20%5BRetracted%5D%20improved%20artificial%20neural%20network%20with%20state%20order%20dataset%20Estimation%20for%20brain%20Cancer%20cell%20diagnosis.%20Biomed.%20Res.%20Int.2022%20(1),%207799812%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="/articles/PMC10769766/" class="text-red">Retracted</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Kurani, A. et al. A comprehensive comparative study of artificial neural network (ANN) and support vector machines (SVM) on stock forecasting. <em>Annals Data Sci.</em><strong>10</strong> (1), 183–208 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kurani,%20A.%20et%20al.%20A%20comprehensive%20comparative%20study%20of%20artificial%20neural%20network%20(ANN)%20and%20support%20vector%20machines%20(SVM)%20on%20stock%20forecasting.%20Annals%20Data%20Sci.10%20(1),%20183%E2%80%93208%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Liu, Q. et al. Prediction of chloride diffusivity in concrete using artificial neural network: modelling and performance evaluation. <em>Constr. Build. Mater.</em><strong>268</strong>, 121082 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Q.%20et%20al.%20Prediction%20of%20chloride%20diffusivity%20in%20concrete%20using%20artificial%20neural%20network:%20modelling%20and%20performance%20evaluation.%20Constr.%20Build.%20Mater.268,%20121082%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Jawad, J., Hawari, A. H. &amp; Zaidi, S. J. Artificial neural network modeling of wastewater treatment and desalination using membrane processes: A review. <em>Chem. Eng. J.</em><strong>419</strong>, 129540 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jawad,%20J.,%20Hawari,%20A.%20H.%20&amp;%20Zaidi,%20S.%20J.%20Artificial%20neural%20network%20modeling%20of%20wastewater%20treatment%20and%20desalination%20using%20membrane%20processes:%20A%20review.%20Chem.%20Eng.%20J.419,%20129540%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Ali, Z. A. et al. eXtreme gradient boosting algorithm with machine learning: A review. <em>Acad. J. Nawroz Univ.</em><strong>12</strong> (2), 320–334 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ali,%20Z.%20A.%20et%20al.%20eXtreme%20gradient%20boosting%20algorithm%20with%20machine%20learning:%20A%20review.%20Acad.%20J.%20Nawroz%20Univ.12%20(2),%20320%E2%80%93334%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Montomoli, J. et al. Machine learning using the extreme gradient boosting (XGBoost) algorithm predicts 5-day delta of SOFA score at ICU admission in COVID-19 patients. <em>J. Intensive Med.</em><strong>1</strong> (02), 110–116 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jointm.2021.09.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8531027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36785563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Montomoli,%20J.%20et%20al.%20Machine%20learning%20using%20the%20extreme%20gradient%20boosting%20(XGBoost)%20algorithm%20predicts%205-day%20delta%20of%20SOFA%20score%20at%20ICU%20admission%20in%20COVID-19%20patients.%20J.%20Intensive%20Med.1%20(02),%20110%E2%80%93116%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Liu, Y. et al. Measuring distance using ultra-wideband radio technology enhanced by extreme gradient boosting decision tree (XGBoost). <em>Autom. Constr.</em><strong>126</strong>, 103678 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Measuring%20distance%20using%20ultra-wideband%20radio%20technology%20enhanced%20by%20extreme%20gradient%20boosting%20decision%20tree%20(XGBoost).%20Autom.%20Constr.126,%20103678%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Haumahu, J. P., Permana, S. D. H. &amp; Yaddarabullah, Y. <em>Fake News Classification for Indonesian News Using Extreme Gradient Boosting (XGBoost)</em>. IOP Conference Series: Materials Science and Engineering, IOP Publishing.<strong> 1098</strong> (5), 052081 (2021).</cite>
</li>
<li id="CR69">
<span class="label">69.</span><cite>Saldanha, M. et al. Configurable fast block partitioning for VVC intra coding using light gradient boosting machine. <em>IEEE Trans. Circuits Syst. Video Technol.</em><strong>32</strong> (6), 3947–3960 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Saldanha,%20M.%20et%20al.%20Configurable%20fast%20block%20partitioning%20for%20VVC%20intra%20coding%20using%20light%20gradient%20boosting%20machine.%20IEEE%20Trans.%20Circuits%20Syst.%20Video%20Technol.32%20(6),%203947%E2%80%933960%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Rufo, D. D. et al. Diagnosis of diabetes mellitus using gradient boosting machine (LightGBM). <em>Diagnostics</em><strong>11</strong> (9), 1714 (2021).
</cite> [<a href="https://doi.org/10.3390/diagnostics11091714" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8467876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34574055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rufo,%20D.%20D.%20et%20al.%20Diagnosis%20of%20diabetes%20mellitus%20using%20gradient%20boosting%20machine%20(LightGBM).%20Diagnostics11%20(9),%201714%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Alghushairy, O. et al. Machine learning-based model for accurate identification of druggable proteins using light extreme gradient boosting. <em>J. Biomol. Struct. Dynamics</em>. <strong>42</strong> (22), 12330–12341 (2024).</cite> [<a href="https://doi.org/10.1080/07391102.2023.2269280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37850427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alghushairy,%20O.%20et%20al.%20Machine%20learning-based%20model%20for%20accurate%20identification%20of%20druggable%20proteins%20using%20light%20extreme%20gradient%20boosting.%20J.%20Biomol.%20Struct.%20Dynamics.%2042%20(22),%2012330%E2%80%9312341%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Sanders, W. et al. Data-driven flood alert system (FAS) using extreme gradient boosting (XGBoost) to forecast flood stages. <em>Water</em><strong>14</strong> (5), 747 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sanders,%20W.%20et%20al.%20Data-driven%20flood%20alert%20system%20(FAS)%20using%20extreme%20gradient%20boosting%20(XGBoost)%20to%20forecast%20flood%20stages.%20Water14%20(5),%20747%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Ahrens, A., Hansen, C. &amp; Schaffer, M. <em>LASSOPACK: Stata module for lasso, square-root lasso, elastic net, ridge, adaptive lasso estimation and cross-validation.</em> (2024).</cite>
</li>
<li id="CR74">
<span class="label">74.</span><cite>Ayllón, S., Brugarolas, P. &amp; Lado, S. <em>New estimates for inequality of opportunity in Europe using elastic net algorithms</em>. <em>Appl. Econ. Lett.</em>, : pp. 1–5. (2024).</cite>
</li>
<li id="CR75">
<span class="label">75.</span><cite>Alharthi, A. M. <em>EMBEDDED FEATURE SELECTION METHODS WITH HIGH DIMENSIONALITY FOR ELASTIC NET AND LOGISTIC REGRESSION MODELS.</em> (2022).</cite>
</li>
<li id="CR76">
<span class="label">76.</span><cite>La Tour, T. D. et al. Feature-space selection with banded ridge regression. <em>NeuroImage</em><strong>264</strong>, 119728 (2022).
</cite> [<a href="https://doi.org/10.1016/j.neuroimage.2022.119728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9807218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36334814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?La%20Tour,%20T.%20D.%20et%20al.%20Feature-space%20selection%20with%20banded%20ridge%20regression.%20NeuroImage264,%20119728%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Abdulhafedh, A. Comparison between common statistical modeling techniques used in research, including: discriminant analysis vs logistic regression, ridge regression vs LASSO, and decision tree vs random forest. <em>Open. Access. Libr. J.</em><strong>9</strong> (2), 1–19 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abdulhafedh,%20A.%20Comparison%20between%20common%20statistical%20modeling%20techniques%20used%20in%20research,%20including:%20discriminant%20analysis%20vs%20logistic%20regression,%20ridge%20regression%20vs%20LASSO,%20and%20decision%20tree%20vs%20random%20forest.%20Open.%20Access.%20Libr.%20J.9%20(2),%201%E2%80%9319%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Patil, P. et al. <em>Uniform Consistency of cross-validation Estimators for high-dimensional Ridge Regression</em>. International conference on artificial intelligence and statistics, PMLR. 3178-3186 (2021).</cite>
</li>
<li id="CR79">
<span class="label">79.</span><cite>Bai, J. et al. Multinomial random forest. <em>Pattern Recogn.</em><strong>122</strong>, 108331 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bai,%20J.%20et%20al.%20Multinomial%20random%20forest.%20Pattern%20Recogn.122,%20108331%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Breiman, L. Random forests. <em>Mach. Learn.</em><strong>45</strong>, 5–32 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Breiman,%20L.%20Random%20forests.%20Mach.%20Learn.45,%205%E2%80%9332%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Hastie, T., Tibshirani, R. &amp; Friedman, J. <em>The elements of statistical learning</em>. Citeseer. (2009).</cite>
</li>
<li id="CR82">
<span class="label">82.</span><cite>Wu, Y. &amp; Chang, Y. <em>Ransomware Detection on Linux Using Machine Learning with Random Forest Algorithm</em> (Authorea Preprints, 2024).</cite>
</li>
<li id="CR83">
<span class="label">83.</span><cite>He, S. et al. Predictive modeling of groundwater nitrate pollution and evaluating its main impact factors using random forest. <em>Chemosphere</em><strong>290</strong>, 133388 (2022).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2021.133388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34952022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20S.%20et%20al.%20Predictive%20modeling%20of%20groundwater%20nitrate%20pollution%20and%20evaluating%20its%20main%20impact%20factors%20using%20random%20forest.%20Chemosphere290,%20133388%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Zhou, J., Huang, S. &amp; Qiu, Y. Optimization of random forest through the use of MVO, GWO and MFO in evaluating the stability of underground entry-type excavations. <em>Tunn. Undergr. Space Technol.</em><strong>124</strong>, 104494 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20J.,%20Huang,%20S.%20&amp;%20Qiu,%20Y.%20Optimization%20of%20random%20forest%20through%20the%20use%20of%20MVO,%20GWO%20and%20MFO%20in%20evaluating%20the%20stability%20of%20underground%20entry-type%20excavations.%20Tunn.%20Undergr.%20Space%20Technol.124,%20104494%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Bloxham, J. C. et al. Proper use of the DIPPR 801 database for creation of models, methods, and processes. <em>J. Chem. Eng. Data</em>. <strong>66</strong> (1), 3–10 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bloxham,%20J.%20C.%20et%20al.%20Proper%20use%20of%20the%20DIPPR%20801%20database%20for%20creation%20of%20models,%20methods,%20and%20processes.%20J.%20Chem.%20Eng.%20Data.%2066%20(1),%203%E2%80%9310%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Hansen, C. M. <em>Hansen solubility parameters: a user’s handbook</em> (CRC, 2007).</cite>
</li>
<li id="CR87">
<span class="label">87.</span><cite>Barton, A. F. <em>CRC Handbook of Solubility Parameters and Other Cohesion Parameters</em> (Routledge, 2017).</cite>
</li>
<li id="CR88">
<span class="label">88.</span><cite>Chi, M. et al. Atomistic descriptors for machine learning models of solubility parameters for small molecules and polymers. <em>Polymers</em><strong>14</strong> (1), 26 (2021).
</cite> [<a href="https://doi.org/10.3390/polym14010026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8747575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35012054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chi,%20M.%20et%20al.%20Atomistic%20descriptors%20for%20machine%20learning%20models%20of%20solubility%20parameters%20for%20small%20molecules%20and%20polymers.%20Polymers14%20(1),%2026%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Ethier, J. G. et al. Predicting phase behavior of linear polymers in solution using machine learning. <em>Macromolecules</em><strong>55</strong> (7), 2691–2702 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ethier,%20J.%20G.%20et%20al.%20Predicting%20phase%20behavior%20of%20linear%20polymers%20in%20solution%20using%20machine%20learning.%20Macromolecules55%20(7),%202691%E2%80%932702%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375797/bin/41598_2025_12758_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (36.5KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author, Mohammad Reza Kezemi, upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-12758-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_12758.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (16.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375797/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375797/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375797%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375797/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375797/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375797/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40851024/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375797/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40851024/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375797/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375797/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="eGnP0OxszEnjfqXkkkqYjyH4gQgm37yg3pGlPNE0w4LmWfm12xlB79Fe3hGieaU4">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
