
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Targeted CRISPR screens reveal genes essential for Cryptosporidium survival in the host intestine - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152547A28AF34FB30547A200120447EA.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368253/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Targeted CRISPR screens reveal genes essential for Cryptosporidium survival in the host intestine">
<meta name="citation_author" content="Lucy C Watson">
<meta name="citation_author_institution" content="Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="Katarzyna A Sala">
<meta name="citation_author_institution" content="Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="Netanya Bernitz">
<meta name="citation_author_institution" content="Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="Lotta Baumgärtel">
<meta name="citation_author_institution" content="Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="Mitchell A Pallett">
<meta name="citation_author_institution" content="Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="N Bishara Marzook">
<meta name="citation_author_institution" content="Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="Lorian Cobra Straker">
<meta name="citation_author_institution" content="Electron Microscopy Science Technology Platform, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="Duo Peng">
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub, San Francisco, CA UK">
<meta name="citation_author" content="Lucy Collinson">
<meta name="citation_author_institution" content="Electron Microscopy Science Technology Platform, The Francis Crick Institute, London, UK">
<meta name="citation_author" content="Adam Sateriale">
<meta name="citation_author_institution" content="Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7749">
<meta name="citation_doi" content="10.1038/s41467-025-63012-1">
<meta name="citation_pmid" content="40835841">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368253/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368253/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368253/pdf/41467_2025_Article_63012.pdf">
<meta name="description" content="The Cryptosporidium parasite is one of the leading causes of diarrheal morbidity and mortality in children, and adolescent infections are associated with chronic malnutrition. There are no vaccines available for protection and only one drug approved ...">
<meta name="og:title" content="Targeted CRISPR screens reveal genes essential for Cryptosporidium survival in the host intestine">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The Cryptosporidium parasite is one of the leading causes of diarrheal morbidity and mortality in children, and adolescent infections are associated with chronic malnutrition. There are no vaccines available for protection and only one drug approved ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368253/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368253">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63012-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63012.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368253%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368253/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368253/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368253/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 20;16:7749. doi: <a href="https://doi.org/10.1038/s41467-025-63012-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63012-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Targeted CRISPR screens reveal genes essential for <em>Cryptosporidium</em> survival in the host intestine</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Watson%20LC%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lucy C Watson</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lucy C Watson</span></h3>
<div class="p">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Watson%20LC%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucy C Watson</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sala%20KA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Katarzyna A Sala</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Katarzyna A Sala</span></h3>
<div class="p">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sala%20KA%22%5BAuthor%5D" class="usa-link"><span class="name western">Katarzyna A Sala</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bernitz%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Netanya Bernitz</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Netanya Bernitz</span></h3>
<div class="p">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bernitz%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Netanya Bernitz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baumg%C3%A4rtel%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Lotta Baumgärtel</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lotta Baumgärtel</span></h3>
<div class="p">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baumg%C3%A4rtel%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lotta Baumgärtel</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pallett%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Mitchell A Pallett</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mitchell A Pallett</span></h3>
<div class="p">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pallett%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Mitchell A Pallett</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marzook%20NB%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">N Bishara Marzook</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">N Bishara Marzook</span></h3>
<div class="p">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marzook%20NB%22%5BAuthor%5D" class="usa-link"><span class="name western">N Bishara Marzook</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Straker%20LC%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Lorian Cobra Straker</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Lorian Cobra Straker</span></h3>
<div class="p">
<sup>2</sup>Electron Microscopy Science Technology Platform, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Straker%20LC%22%5BAuthor%5D" class="usa-link"><span class="name western">Lorian Cobra Straker</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Duo Peng</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Duo Peng</span></h3>
<div class="p">
<sup>3</sup>Chan Zuckerberg Biohub, San Francisco, CA UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Duo Peng</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collinson%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lucy Collinson</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lucy Collinson</span></h3>
<div class="p">
<sup>2</sup>Electron Microscopy Science Technology Platform, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collinson%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucy Collinson</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sateriale%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Adam Sateriale</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Adam Sateriale</span></h3>
<div class="p">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sateriale%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Adam Sateriale</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Cryptosporidiosis Laboratory, The Francis Crick Institute, London, UK </div>
<div id="Aff2">
<sup>2</sup>Electron Microscopy Science Technology Platform, The Francis Crick Institute, London, UK </div>
<div id="Aff3">
<sup>3</sup>Chan Zuckerberg Biohub, San Francisco, CA UK </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 9; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368253  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835841/" class="usa-link">40835841</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The <em>Cryptosporidium</em> parasite is one of the leading causes of diarrheal morbidity and mortality in children, and adolescent infections are associated with chronic malnutrition. There are no vaccines available for protection and only one drug approved for treatment that has limited efficacy. A major barrier to developing new therapeutics is a lack of foundational knowledge of <em>Cryptosporidium</em> biology, including which parasite genes are essential for survival and virulence. Here, we iteratively improve the tools for genetically manipulating <em>Cryptosporidium</em> and develop a targeted CRISPR-based screening method to rapidly assess how the loss of individual parasite genes influence survival in vivo. Using this method, we examine the parasite’s pyrimidine salvage pathway and a set of leading <em>Cryptosporidium</em> vaccine candidates. From this latter group, using inducible knockout, we determined the parasite gene known as Cp23 to be essential for survival in vivo. Parasites deficient in Cp23 were able to replicate within and emerge from infected epithelial cells, yet unable to initiate gliding motility which is required for the reinfection of neighbouring cells. The targeted screening method presented here is highly versatile and will enable researchers to more rapidly expand the knowledge base for <em>Cryptosporidium</em> infection biology, paving the way for new therapeutics.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Microbiology, Parasitology, Parasite biology, Pathogens</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Cryptosporidium is an important threat to public health, yet it lacks a robust genetic toolkit. Here, Watson et al. introduce a targeted CRISPR-based screening method to identify parasite genes that are essential for its survival within the intestine.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Diarrhoeal related infections are a major cause of morbidity and mortality in children around the world<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Cryptosporidiosis is consistently found to be one of the leading causes of moderate-to-severe diarrhoeal disease in infants<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Unlike other diarrhoeal diseases that are attributed with high incidence rates, such as rotavirus or <em>Shigella</em>, there are no effective drugs or vaccines for <em>Cryptosporidium</em>. Nitazoxanide, the only Food and Drug Administration approved drug for treatment, is not effective in immunocompromised individuals and only partially effective in children, the patient population that needs intervention the most<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. One reason for this scarcity of therapeutics is a historic lack of effective systems to study the parasite. While there have been recent improvements in genetic manipulation<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, drug target identification<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, and animal models of infection<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, our comprehension of <em>Cryptosporidium</em> biology remains very basic. Specifically, there is a limited understanding of the <em>Cryptosporidium</em> genes that contribute to parasite fitness and survival, genes that would be the most suitable targets for therapeutic intervention.</p>
<p id="Par4">Reverse genetic approaches have been instrumental in identifying parasite genes that influence survival and convey fitness in other Apicomplexan parasites<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. <em>Toxoplasma</em>, which has historically served as a facile model for Apicomplexa, has benefited from a high transfection efficiency coupled with non-homologous end joining (NHEJ) to be at the forefront of Apicomplexa CRISPR screening<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Like <em>Cryptosporidium</em>, the <em>Plasmodium</em> parasite lacks NHEJ pathways, so CRISPR-Cas9 driven double stranded breaks can only be fixed by homologous repair. This lack of NHEJ can be leveraged to implement precise CRISPR screening, yet to date, no such screening method exists in <em>Cryptosporidium</em>. Here, we overcome the current technical barriers for <em>Cryptosporidium</em> genetic manipulation to develop a reproducible method for pooled in vivo CRISPR screens.</p>
<p id="Par5"><em>Cryptosporidium</em> has a highly streamlined genome and lacks many basic metabolic pathways. Because of this, it is thought to be heavily reliant on its host cell for nutrients. This is particularly evident in the nucleotide salvage pathway, where <em>Cryptosporidium</em> is believed to be capable of scavenging both nucleotides and nucleotide precursors from the host enterocyte<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Despite a compact genome, both purine and pyrimidine salvage pathways show evidence of redundancy. The purine salvage pathway has been well studied in <em>Cryptosporidium</em>, revealing that many genes in the pathway can be independently eliminated, resulting in little or no deficit in parasite growth<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Surprisingly, this includes inosine monophosphate dehydrogenase (IMPDH), a gene once considered to be a crucial drug target based on metabolic mapping<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The absence of a growth defect in parasites that lack IMPDH suggests that <em>Cryptosporidium</em> can not only synthesise purine nucleotides, but also take them up directly from the infected host cell. Less is known about the pyrimidine salvage pathway in <em>Cryptosporidium</em>, however, there is still evidence of redundancy. Thymidine kinase (TK) and dihydrofolate reductase-thymidine synthase (DHFR-TS) both convert their respective substrates (deoxythymidine or deoxyuridine monophosphate) to deoxythymidine monophosphate. For this reason, TK and DHFR-TS are both independently non-essential for DNA synthesis and parasite growth<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Using our in vivo pooled CRISPR screen, we discovered that most of the pyrimidine salvage genes significantly contribute to parasite fitness within the intestine.</p>
<p id="Par6">It is widely accepted that protective immunity can be acquired to <em>Cryptosporidium</em>, as incidences of <em>Cryptosporidium</em> infections decrease with age<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, and experimental infection with attenuated parasites leads to protection in calves and mice<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Considering the success of the vaccination campaign for rotavirus<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, a diarrhoeal pathogen with a similar patient population and pathogenesis, a <em>Cryptosporidium</em> vaccine is predicted to greatly reduce morbidity and mortality in children. Numerous protein antigens have been associated with protection in humans and several prominent surface proteins have been suggested as vaccine candidates<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Yet, if and how these proteins contribute to parasite fitness is unclear. Here, we assess the relative fitness contributions during infection for a panel of proposed <em>Cryptosporidium</em> vaccine candidates (collected from the literature). As Cp23 (also known as the <em>Cryptosporidium</em> immunodominant antigen 23) is one of the leading vaccine candidates, and strongly correlated with protection in humans<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, we investigated how this protein specifically contributes to parasite virulence and fitness.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Iterative improvements to improve <em>Cryptosporidium</em> genetic manipulation</h3>
<p id="Par7">To develop CRISPR screening, we first sought to improve <em>Cryptosporidium</em> transfection efficiency. To transfect <em>Cryptosporidium</em>, dormant oocysts are treated with bile salts and warmed to body temperature to simulate the conditions of a human intestine. Under these conditions, the environmentally hardy oocysts ‘excyst’, releasing four motile and transfectable sporozoites that readily invade intestinal epithelial cells. To optimise transfection, we transfected <em>Cryptosporidium parvum</em> parasites with a vector that contained both luminescent (NanoLuciferase) and fluorescent (mCherry) reporters under constitutive expression and allowed the transfected parasites to infect an intestinal cell (HCT8) monolayer. Numerous electroporation programmes were trialled using transient expression, with FL115 demonstrating the highest luminescence (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). Furthermore, various combinations of bile salts and incubation media were tested for their effect on parasite transfection. Notably, incubation in sodium taurocholate led to higher transfection efficiency compared to the deoxy form, sodium taurodeoxycholate (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>). Combined, these adjustments resulted in more than a 50-fold improvement in transfection efficiency (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c, d</a>).</p>
<p id="Par8">As <em>Cryptosporidium</em> parasites lack NHEJ, CRISPR driven injury of the genome is required to drive homologous recombination for genetic manipulation. Although this represents a significant barrier to high-throughput screening, the parasite’s lack of NHEJ very likely contributes to its uniquely high level of genetic editing specificity. Indeed, an ‘off-target’ genomic insertion has yet to be reported by <em>Cryptosporidium</em> researchers. Further, the parasite predominantly exists in a haploid state during its life cycle and has a minimal requirement of homology for efficient editing. To determine the smallest reliable allowance for efficient repair, we tested the efficiency of editing using differing length homology arms to integrate a nanoluciferase gene into the dispensable TK locus. A positive correlation between the length of the homology arms and the efficiency of repair was observed, with 50 bp (the longest length tested) being the most efficient, while no integration was observed in the absence of CRISPR driven injury of TK (Supplementary Fig. <a href="#MOESM1" class="usa-link">1e</a>).</p>
<p id="Par9">To allow for pooled genetic knockout screens, we reasoned that a one-plasmid approach would be required, where a single plasmid delivers the Cas9-expression vector, a target specific gRNA, and a segment of DNA to be inserted at the target genomic locus (repair DNA). The repair DNA contains homologous flanks surrounding an expression cassette with the selectable marker and a genetic barcode for identification. Further, to enable high-throughput creation of targeted libraries, we developed a two-step Golden Gate assembly method, where first a 300 bp segment containing all gene-specific (unique) DNA is integrated into the Cas9-expression vector. In the second step of the assembly, a non-unique expression cassette is inserted that will integrate into the genome, enabling the use of virtually any combination of selection markers or reporters (Fig. <a href="#Fig1" class="usa-link">1a</a>). To fit all unique DNA into a 300 bp oligonucleotide, we used a 50 bp gRNA which simultaneously serves as one of the homology sites for genomic integration. To test the feasibility of this approach, we generated a vector targeting TK, a gene that is known to be dispensable for parasite growth. The TK targeting vector was transfected into <em>Cryptosporidium parvum (C. parvum)</em> sporozoites that were used to infect genetically immunocompromised (interferon gamma deficient <em>Ifnγ</em><sup><em>–/–</em></sup>) mice and luminescence was measured in mouse faecal material 7 days post transfection, comparable to what is observed when using a separate DNA repair template and a Cas9-expression vector (Fig. <a href="#Fig1" class="usa-link">1b</a>). As we redefined the method to generate <em>Cryptosporidium</em> transgenics, we confirmed the approach’s specificity via whole genome sequencing, and as expected, the only genome alteration observed was at the targeted site within the coding region of the TK gene (Fig. <a href="#Fig1" class="usa-link">1c</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. In vivo CRISPR screen of the <em>Cryptosporidium</em> pyrimidine salvage pathway.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368253_41467_2025_63012_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9703/12368253/df28e5195500/41467_2025_63012_Fig1_HTML.jpg" loading="lazy" id="d33e549" height="893" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic illustrates how a knockout vector is generated. The first Golden Gate reaction occurs between a Cas9 expression plasmid and a 300 bp unique segment (containing the two 50 bp homology arms (one of which serves dual function as the gRNA), a unique DNA barcode, BsmBI restriction enzyme sites and the tracrRNA). The second Golden Gate reaction, using the BsmBI restriction enzyme sites, inserts a variable selection/reporter cassette to generate a complete knockout vector. The knockout vector then contains all the machinery to disrupt a gene of interest by inserting a variable selection cassette and barcode at the genomic locus. <strong>b</strong> A knockout vector targeting thymidine kinase (cgd5_4440) was generated and transfected into <em>C. parvum</em> sporozoites that were used to infect Ifnγ<sup>–/–</sup> mice under paromomycin selection. Faecal samples were collected and the luminescence in faecal material was monitored. Data shows the mean faecal luminescence from a pooled cage sample ( ± SEM of 2 technical replicates), <em>n</em> = 4 mice. <strong>c</strong> Alignment of reads from whole genome sequencing of the thymidine kinase knockout strain to the <em>C. parvum</em> IOWAII genome at the site of insertion. Note the complete lack of alignment to the PAM site, which is removed by the homologous repair event. <strong>d</strong> Overview of CRISPR screening method. Following construction of KO vectors (detailed in 1a), sporozoites are transfected with gene specific KO vectors and used to infect mice. Specific barcodes are then amplified via high fidelity PCR and used to calculate fold enrichment (log2[%barcode(output) / %barcodes(input)]) which measures the relative fitness contribution of each gene. <strong>e</strong> Mouse faecal material was collected, and luminescence was monitored in the pooled sample from the pyrimidine salvage pathway CRISPR screens at 1 and 2 KO vectors per gene. Data shows the mean faecal luminescence from a pooled cage sample ( ± SEM of 2 technical replicates), <em>n</em> = 4 mice. <strong>f</strong> Rank ordered fold enrichment scores from the 1 and 2 KO vectors per gene pyrimidine salvage CRISPR screens. The colour indicates the relative fitness contribution, with dark purple showing high fitness conferring and dark green showing low fitness conferring. <strong>g</strong> Comparison of the fold enrichment scores between the 1 and 2 KO vectors per gene pyrimidine salvage CRISPR screens. Confidence refers to the inverse of the 95% confidence interval when comparing the log2 fold change scores from each screen (see methods).</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">CRISPR KO screen of the pyrimidine salvage pathway</h3>
<p id="Par10">To assess the feasibility and reproducibility of a recombination-based in vivo screen, we designed a pilot screen targeting 11 genes in the parasite’s pyrimidine salvage pathway (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). We ran parallel screens employing either 1 or 2 targeting vectors per gene, hence 11 or 22 gRNAs, respectively. Targeted KO vectors were transfected into <em>C. parvum</em> sporozoites, which were propagated in <em>Ifnγ</em><sup><em>–/–</em></sup> mice under paromomycin selection (Fig. <a href="#Fig1" class="usa-link">1d</a>). Luminescence from the integrated nanoluciferase reporter was detected in mouse faecal material 8 days post transfection, in both the 1 and 2 KO vector per gene pyrimidine salvage screens (Fig. <a href="#Fig1" class="usa-link">1e</a>). From the output, faecal material from the infected mice, parasites were purified, their DNA extracted, and the barcodes amplified via high fidelity PCR. From the input, transfected parasites used to infect the mice, DNA was extracted and the barcodes amplified via high fidelity PCR. Both the output and input barcodes were sequenced and used to calculate fold enrichment scores for each gene. Fold enrichment scores serve as a measure of relative fitness, genes whose barcodes show a negative enrichment are presumed to be important to parasite survival and therefore highly fitness conferring (Fig. <a href="#Fig1" class="usa-link">1f</a>). Importantly, fold enrichment scores between the 1 and 2 KO vector screens were strongly correlated, achieving an R<sup>2</sup> of 0.842 (Fig. <a href="#Fig1" class="usa-link">1g</a>). A separate biological repeat of the 1 KO vector screen using a separate shipment of Cryptosporidium parvum IOWAII parasites also showed good correlation, demonstrating reproducibility of the method (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Of the 11 genes included in the pyrimidine salvage screen, only 3 were low fitness conferring: dihydrofolate reductase—thymidylate synthase (DHFR-TS) (cgd4_4460), thymidine kinase (TK) (cgd5_4440) and dCMP deaminase (cgd2_2780). As mentioned previously, TK and DHFR-TS play redundant roles in synthesis of <em>Cryptosporidium</em> dTMP, making them independently non-essential for DNA synthesis and parasite growth. Thus, the results of our pilot screen effectively mirror those of previous experiments attempting individual knockouts of these genes in vivo<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Refining diCRE-mediated genetic editing in <em>Cryptosporidium</em>
</h3>
<p id="Par11">To validate our results from the pyrimidine salvage screen, we sought to use an inducible Cre-recombinase system to remove genes at their genetic locus. This method has been previously adapted for use in <em>C. parvum</em>, employing the conditional split recombinase (diCRE), which dimerises upon rapamycin addition and excises DNA between loxP recombination sites, one of which is embedded within an artificial intron introduced into the target gene’s coding sequence<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. This system demonstrated high efficiency, but low-to-moderate activity in the absence of the rapamycin induction (i.e. leakiness). To refine this system, we tested a validated intron from the male gamete fusion factor HAP2 (cgd8_2220). Insertion of this intron within the nanoluciferase reporter with and without the loxP recombination site did not affect the measured luminescence during in vitro infection (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>). Using this validated intron/loxP combination, we generated a stable <em>C. parvum</em> diCRE parasite line targeting TK with the diCRE subunits, FRB-Cre60 and FKBP-Cre59, expressed under the same promoter, separated via a self-cleaving T2A peptide (TK-T2A-diCRE) which allows for ribosomes to ‘skip’ during translation and create two separate polypeptides from the same expression cassette (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>). However, when investigating the excision dynamics of this <em>C. parvum</em> TK-T2A-diCRE line, by infecting HCT8 monolayers in the presence or absence of rapamycin, we noted a high level of excision in the absence of rapamycin, similar to what has been reported previously (Supplementary Fig. <a href="#MOESM1" class="usa-link">3c</a>). We reasoned that the T2A skip peptide may not be functioning properly and causing background induction in <em>Cryptosporidium</em>. Consequently, we generated a stable transgenic parasite line targeting TK where the diCRE segments were under independent aldolase and tubulin promoters (TK-diCRE) (Supplementary Fig. <a href="#MOESM1" class="usa-link">3d</a>). When investigating the excision dynamics of this TK-diCRE line, we observed complete excision of the loxP-flanked TK segment at 24 h post rapamycin induction, and no measurable excision in the non-induced controls (Supplementary Fig. <a href="#MOESM1" class="usa-link">3e</a>). Using these TK-diCRE parasites (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>), we performed a time course to measure the dynamics of knockout at both the DNA and protein level (via a C-terminal HA tag). DNA excision had not occurred by 8 h post rapamycin treatment but was complete by 24 h, and no excision was detected in the non-induced control (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>). Likewise, protein levels started to decrease by 12 h post rapamycin treatment and were approximately 95% reduced compared to the non-induced controls by 24 h. In contrast, the protein levels remained high throughout the time course in the non-induced controls (Supplementary Fig. <a href="#MOESM1" class="usa-link">4c, d</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Ribonucleotide reductase is required for parasite DNA replication and survival</h3>
<p id="Par12">Ribonucleotide reductases (RNRs) catalyse the conversion of nucleotides to deoxynucleotides, an essential step for DNA synthesis in all organisms. In our pyrimidine salvage screen, RNR showed the lowest fold enrichment score, indicating a high effect on parasite fitness. Using a direct approach to disrupt the endogenous locus we were unable to derive RNR deficient parasites (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>), Further, our first attempt to create RNR-diCRE parasites was unsuccessful, likely due to the addition of a C-terminal HA epitope tag. It has been suggested that the C-terminus of RNR is required for its function in other organisms and this may hold true for <em>Cryptosporidium</em><sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The second attempt to generate RNR-diCRE parasites, without the C-terminal HA epitope tag, was successful (Supplementary Fig. <a href="#MOESM1" class="usa-link">4e</a>). In vitro, rapamycin induced DNA excision of both TK- and RNR-diCRE parasites appears to follow similar dynamics, although we note some partial excision in the RNR-diCRE parasites at later timepoints (Fig. <a href="#Fig2" class="usa-link">2a, b</a>). Further, functional deletion of both TK and RNR was confirmed using 5-ethynyl2´-deoxyuridine (EdU), a thymidine analogue. Thymidine kinase is required for EdU phosphorylation and incorporation, and it has been shown before that loss of the TK gene in <em>Cryptosporidium</em> leads to a failure to incorporate EdU<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In contrast, RNR deletion will not block EdU phosphorylation, but rather incorporation, as DNA synthesis cannot occur without other deoxynucleotides. We observed a complete lack of EdU incorporation in rapamycin treated TK and RNR-diCRE parasites, while EdU incorporation was observed in the non-induced and wildtype controls (Supplementary Fig. <a href="#MOESM1" class="usa-link">4f</a>). In vitro, deletion of TK led to no growth defect, whereas deletion of RNR strongly attenuated parasite growth by 24 h post infection (Fig. <a href="#Fig2" class="usa-link">2c</a>). Similarly, rapamycin treatment of TK-diCRE infected mice lead to no growth defect, but rapamycin treatment of RNR-diCRE infected mice completely inhibited parasite growth (Fig. <a href="#Fig2" class="usa-link">2d</a>), demonstrating that RNR is indeed essential for parasite survival, further underlining the reliability of the in vivo CRISPR screening method.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Validating screen results with diCRE mediated excision.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368253_41467_2025_63012_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9703/12368253/23fee896585f/41467_2025_63012_Fig2_HTML.jpg" loading="lazy" id="d33e713" height="1245" width="746" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>,<strong> b</strong> TK-diCRE (shown in green) and ribonucleotide reductase (RNR) -diCRE (shown in purple) parasites were generated and used to infect an HCT8 monolayer in the presence or absence of rapamycin. Genomic DNA was extracted at 6, 24, and 48 h, and diagnostic PCRs confirmed the level of excision at the given time points. Data shown is representative of 2 biological replicates. <strong>c</strong> HCT8 cell monolayers infected with TK-diCRE and RNR-diCRE parasites in the presence or absence of rapamycin. At 6, 24, and 48 h the monolayers were fixed, stained, and the number of parasites per host nuclei was quantified. Box plot whiskers show the minimum and maximum values, the box shows the 25th to 75th percentile and the line shows the median. Representative images are shown, with nuclei in blue (Hoechst), and parasites in green (Vicia villosa lectin). Scale bar = 30 μm. Data shown is representative of 2 biological replicates. Significance was determined using a two-tailed unpaired <em>t</em> test. <strong>d</strong> TK-diCRE or RNR-diCRE parasites were used to infect Ifnγ<sup>–/–</sup> mice, which were either treated with rapamycin or vehicle (DMSO) in their drinking water starting at day 2 post infection. Data shows the mean faecal luminescence from a pooled cage sample ( ± SEM of 2 technical replicates), <em>n</em> = 2 mice per condition. Note that a cell culture dish in the figure indicates an in vitro experiment, while a mouse silhouette indicates an in vivo experiment.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">CRISPR KO screen of <em>Cryptosporidium</em> vaccine candidates</h3>
<p id="Par13">Both the 1 and 2 KO vector per gene pyrimidine salvage screens were reproducible, but to circumvent potential low efficiency gRNAs, we chose to screen the leading <em>Cryptosporidium</em> vaccine candidates with 2 KO vectors per gene. In total, 11 vaccine candidates (22 KO vectors) were chosen due to their reported surface location and implication of immunogenicity and/or immune protection in the literature (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Luminescence from the nanoluciferase reporter was detected in mouse faecal material around 7 days post transfection in both screens (Fig. <a href="#Fig3" class="usa-link">3a</a>). As before, parasites were purified, DNA extracted, and barcodes were amplified via high fidelity PCR. The sequenced barcode counts were used to calculate fold enrichment scores for each gene (Fig. <a href="#Fig3" class="usa-link">3b</a>), and importantly these scores were comparable, achieving an R<sup>2</sup> of 0.611 (Fig. <a href="#Fig3" class="usa-link">3c</a>). Of the 11 genes in the vaccine candidate screen, only 2 were low fitness conferring: cgd6_1660 (thrombospondin repeat protein 11 (TSP11)) and cgd6_32 (apical glycoprotein 1 (AGP1)). The remaining 9 genes appeared to confer some level of fitness. Akey et al. recently demonstrated that AGP1 was dispensable for parasite survival, while apical glycoprotein 2 (AGP2) (cgd7_4330) was likely essential, both phenotypes that our pooled in vivo CRISPR screen recapitulated<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. In vivo CRISPR screen of <em>Cryptosporidium</em> vaccine candidates.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368253_41467_2025_63012_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9703/12368253/eb08ad7cc246/41467_2025_63012_Fig3_HTML.jpg" loading="lazy" id="d33e786" height="825" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Mouse faecal material was collected and luminescence was monitored during infection. Each replicate was conducted with 2 KO vectors per gene. Data shows the mean faecal luminescence from a pooled cage sample ( ± SEM of 2 technical replicates), <em>n</em> = 5 Ifnγ<sup>–/–</sup> mice for screen 1 and n = 3 Ifnγ<sup>–/–</sup> mice for screen 2. <strong>b</strong> Rank ordered fold enrichment scores from the vaccine candidate CRISPR screens. The colour indicates the relative fitness contribution of a gene, with dark purple being high fitness conferring and dark green being low fitness conferring. <strong>c</strong> Comparison of the fold enrichment scores between the replicate CRISPR screens for each gene. Confidence refers to the inverse of the 95% confidence interval when comparing the log2 fold change scores from each screen (see “Methods”). <strong>d</strong> Mouse faecal material was collected and luminescence was monitored during infection. Data shows the mean faecal luminescence from a pooled cage sample ( ± SEM of 2 technical replicates), <em>n</em> = 5 Ifnγ<sup>–/–</sup> mice. Barcodes from KO parasites could be easily monitored over time (<strong>e</strong>) and within individual mice (<strong>f</strong>).</p></figcaption></figure><p id="Par14">To gain more insight into growth and competition of pooled KO parasites, we conducted an additional 1 KO vector per gene vaccine candidate screen, where instead of amplifying DNA barcodes from parasites purified around the peak of infection, we amplified barcodes directly from individual faecal collections (Fig. <a href="#Fig3" class="usa-link">3d</a>). This alternative method offered greater temporal and spatial resolution of infection where fold enrichment scores could be easily and non-invasively monitored throughout the experiment from either pooled collections or individual mice (Fig. <a href="#Fig3" class="usa-link">3e, f</a>). Individual mice demonstrated considerable heterogeneity at day 8 of infection, which decreased over time and became more uniform by day 12. Again, this screen found that AGP1 and TSP11 were low fitness conferring relative to the other genes within the cohort.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Immunodominant antigen 23 is required for host cell invasion</h3>
<p id="Par15">Immunodominant antigen 23 (Cp23) (cgd4_3620) is one of the leading <em>Cryptosporidium</em> vaccine candidates, having been originally discovered in 1986<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Despite nearly 40 years of research, there is very little known about Cp23’s role and function during <em>Cryptosporidium</em> infection. In our vaccine candidate screen, Cp23 displayed a low fold enrichment score, indicating a moderate-to-high effect on parasite fitness. To investigate the function of Cp23, we attempted to create a Cp23 knockout line of <em>Cryptosporidium</em> parasites but were unsuccessful (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). We also made an attempt to epitope tag the endogenous locus of Cp23 at the C-terminus and were unsuccessful at generating viable parasites, suggesting the epitope tag may interfere with an essential function (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). We were, however, successful in generating an inducible knockout Cp23-diCRE parasite line (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>). In vitro, the rapamycin induced DNA excision had started by 6 h and was complete by 24 h, with no excision observed in the non-induced controls (Fig. <a href="#Fig4" class="usa-link">4a, b</a>). Loss of Cp23 was confirmed at the protein level using a commercially available monoclonal antibody. In vitro, rapamycin treatment caused a significant decrease in parasite growth after 24 h (Fig. <a href="#Fig4" class="usa-link">4c</a>) and rapamycin treatment of Cp23-diCRE infected mice completely blocked parasite growth (Fig. <a href="#Fig4" class="usa-link">4d</a>), further supporting that Cp23 is indeed essential for parasite survival.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Immunodominant antigen 23 is essential and required for reinvasion of host cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368253_41467_2025_63012_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9703/12368253/5798e91644d0/41467_2025_63012_Fig4_HTML.jpg" loading="lazy" id="d33e912" height="932" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong> HCT8 cell monolayers infected with Cp23-diCRE in the presence or absence of rapamycin. At 6, 24, and 48 h, gDNA was extracted and diagnostic PCRs confirmed the level of excision at the given timepoint. Data shown is representative of 2 biological replicates<strong>. c</strong> HCT8 cell monolayers infected with Cp23-diCRE in the presence or absence of rapamycin. At 6, 24, and 48 h, the monolayer was fixed and stained and the parasite per host nuclei was quantified. Box plot whiskers show the minimum and maximum values, the box shows the 25th to 75th percentile and the line shows the median. Representative images are shown, with nuclei in blue (Hoechst), and parasites in green (Vicia villosa lectin). Scale bar = 30 μm. Data shown is representative of two biological replicates. Significance was determined using a two-tailed unpaired <em>t</em> test. <strong>d</strong>. Ifnγ<sup>–/–</sup> mice were infected with 50,000 Cp23-diCRE parasites and treated with rapamycin or DMSO control in their drinking water at 2 days post infection. Data shows the mean faecal luminescence from a pooled cage sample ( ± SEM of 2 technical replicates), <em>n</em> = 2 mice per condition. <strong>e</strong> Super resolution microscopy at 24 h post infection in vitro with Cp23-diCRE parasites in the presence or absence of rapamycin; green (helix pomatia agglutinin (HPA)), parasite; magenta, (sytox), nuclei; red (Cp23 Ab), Cp23. Scale bar = 2 μm. Data shown is representative of two biological replicates. <strong>f</strong> Expansion microscopy of the <em>C. parvum</em> asexual stages: sporozoite, merozoite, trophozoite and meront, and the sexual stages: macrogamont (female) and microgamete (male); green (helix pomatia agglutinin (HPA)); blue (N-hydroxysuccinimide (NHS ester); magenta (sytox); red (αCp23). Scale bar = 5 μm. Median expansion factor of 4.5. <strong>g</strong> Transmission electron microscopy of an excysted and unexcysted <em>C. parvum</em> sporozoite coupled with immunogold labelling with Cp23 antibody. Scale bar = 300 nm<strong>. h</strong> Permeabilisation assay of <em>C. parvum</em> sporozoites. The permeabilised condition used 0.1% Triton X-100 and the non-permeabilised condition used PBS. Grey (tryptophan synthase (αTrpB); green (Vicia villosa lectin); red (αCp23); blue (Hoechst). All images were taken using the same settings and exposure. Scale bar = 5 μm. Data shown is representative of two biological replicates. <strong>i</strong> HCT8 cell monolayers infected with either Cp23-diCRE or wildtype parasites in the presence or absence of rapamycin, and at 22 h post infection life stages were quantified. Data shows 2 biological replicates ± sd. <em>n</em> = 438 Cp23-diCRE - RAP, n = 173 Cp23-diCRE + RAP, <em>n</em> = 213 wildtype—RAP, <em>n</em> = 203 wildtype + RAP. <strong>j</strong>,<strong> k</strong> Live imaging of the reinvasion event was carried out at 18 h post infection of an HCT8 monolayer with Cp23-diCRE parasites in the presence or absence of rapamycin. All live microscopy data shown is from 2 biological replicates with <em>n</em> = 9 egress events recorded for each condition. Gliding distance was only measured for merozoites with clear movement following egress (<em>n</em> = 18 for DMSO and <em>n</em> = 5 for RAP). Movement of individual merozoites was tracked using Fiji software. Scatter plots show the mean ± sd with significance determined using a two-tailed unpaired t-test. Representative images are shown in (<strong>k</strong>) with movies in supplementary data. Scale bar = 8 μm. Note that a cell culture dish in the figure indicates an in vitro experiment, while a mouse silhouette indicates an in vivo experiment.</p></figcaption></figure><p id="Par16">Cp23 has been reported to be on the surface of the sporozoite<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, in the sporozoite trails<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, and is predicted to localise to the micronemes<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. To localise Cp23 at high resolution, we used a commercial antibody that we genetically validated with our Cp23-diCRE parasites (Fig. <a href="#Fig4" class="usa-link">4e</a>), coupled with expansion and super-resolution microscopy. This revealed Cp23 was expressed at the parasite’s pellicle throughout the life cycle (Fig. <a href="#Fig4" class="usa-link">4f</a>). To resolve the location further, we carried out transmission electron microscopy (TEM) with the immunogold labelled Cp23 antibody (Fig. <a href="#Fig4" class="usa-link">4g</a>). In excysted and unexcysted sporozoites, Cp23 again appeared to primarily localise to the parasite pellicle, either at the plasma membrane or inner membrane complex (IMC). The immunogold-TEM also revealed that even in unexcysted sporozoites, Cp23 was not present in the parasite’s micronemes. Further, we found no evidence that Cp23 was present in sporozoite trails (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>). When permeabilised with Triton X-100, sporozoites demonstrated faint staining with the Cp23 antibody (Fig. <a href="#Fig4" class="usa-link">4h</a>). Without permeabilisation, <em>C. parvum</em> sporozoites, surprisingly, demonstrated either a complete lack of Cp23 signal or a heighted intensity of signal throughout the parasite (32.5% no signal and 67.5% heightened signal parasites) (Fig. <a href="#Fig4" class="usa-link">4h</a>). In these high intensity parasites, we can detect the internal control antibody (CpTrpB - <em>Cryptosporidium</em> tryptophan synthase beta) at low levels. This suggests that (1) high intensity Cp23 parasites are weakly permeabilised, (2) permeabilisation with Triton X-100 affects Cp23 localisation, likely by disrupting its membrane association, and (3) Cp23 is likely not exposed on the surface of the sporozoite.</p>
<p id="Par17">Within our synchronised in vitro model of infection, <em>C. parvum</em> parasites egress from their infected cell and re-invade nearby epithelial cells around 18 h post infection, starting another round of asexual reproduction. By 22 h, this reinvasion event is mostly complete. To determine the biological function of Cp23, we examined our Cp23-diCRE parasites in the context of this reinfection event, when DNA excision and depletion of Cp23 has started. At 22 h, Cp23-diCRE parasites without rapamycin treatment were mostly newly invaded life stages (1n or 2n = 81% of parasites observed, n = 356/438). When Cp23 was ablated by rapamycin, fewer parasites observed were newly invaded life stages (1n or 2n = 45%, <em>n</em> = 77/173) (Fig. <a href="#Fig4" class="usa-link">4i</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6c, e</a>). In wildtype controls, both in the presence and absence of rapamycin, again, most of the life stages observed were newly invaded (1n or 2n = 83%, n = 176/203 and 171/213 respectively) (Fig. <a href="#Fig4" class="usa-link">4i</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6d, e</a>). We hypothesised that the reduced percentage of newly invaded parasites when Cp23 was ablated might be due to a defect in reinvasion. To examine this more closely, live microscopy was carried out in the presence and absence of rapamycin (Fig. <a href="#Fig4" class="usa-link">4j, k</a>, Supplementary Movies <a href="#MOESM3" class="usa-link">S1</a>–<a href="#MOESM6" class="usa-link">4</a>). When Cp23 was ablated, merozoites could egress, but were then unable to move to another cell to initiate reinvasion. Apicomplexan parasites, such as <em>Cryptosporidium</em>, are known to use a method of locomotion called gliding motility, where parasites secrete proteins that are then bound by their own surface receptors, allowing for forward propulsion using an actomyosin-based complex (referred to as the glideosome)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Loss of Cp23 in <em>Cryptosporidium</em> appears to specifically block this gliding motility that is essential for reinfection.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par18">Reverse genetic screens play an important role in molecular biology and have transformed the field for many pathogens. Here, we overcome the current technical barriers to develop a pooled in vivo CRISPR KO method that allows for reverse genetic screening in <em>Cryptosporidium</em>. As <em>Cryptosporidium</em> lacks the molecular machinery for NHEJ, our method had to employ homologous recombination to create stable transgenics. We leveraged two important features of <em>Cryptosporidium</em> genetics: (1) the parasite spends nearly all of its life cycle in a haploid form, thus requiring only one recombination event, and (2) there is a minimal length requirement for homologous recombination, in fact 30 bp of DNA flanking the Cas9 cut site appears to be sufficient in our experiments (although efficiency increases with the size of DNA flanks). Our two-step cloning approach for generating these targeted KO vectors is designed to be versatile, allowing for an unlimited range of selection markers and reporters that can be inserted into the genome, which will allow researchers to develop new and innovative screens.</p>
<p id="Par19">Within an infected host, <em>Cryptosporidium</em> undergoes multiple rounds of asexual reproduction before differentiating into sexual forms (male and female) that unite to allow for genetic recombination. During the sexual cycle, transgenic parasites in these CRISPR screens can mate and this has the potential to influence results. This risk is partially mitigated by two factors: (1) the parasite must complete at least three rounds of asexual replication prior to sexual differentiation and potential recombination. This gives ample time for any genes that are detrimental to parasite fitness to exert their effect; (2) <em>Cryptosporidium</em> has a relatively high rate of recombination, which should allow for genes, even those in close proximity, to segregate in a nearly random fashion<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. In both the pyrimidine salvage and vaccine candidate screens, we identified high and low fitness conferring genes in close proximity to each other (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). Despite these mitigating factors, there is the potential for sublethal gene knockouts to exert synergistic or antagonistic effects and this possibility must be considered during the design process and while analysing and interpreting results.</p>
<p id="Par20">Another important factor to consider is the amplifying effect of relative fitness screens within an in vivo model of infection. Genes that have a mild effect on parasite survival and fitness can be outcompeted within a larger pool. This appears to be the case for uracil phosphoribosyltransferase (UPRT, cgd1_1900), which catalyses the conversion of uracil and phosphoribosylpyrophosphate to uridine monophosphate. UPRT has recently been shown to be non-essential to parasite survival, yet the authors noted that their UPRT KO <em>Cryptosporidium</em> parasites appeared to have a growth defect<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. In our in vivo screens, UPRT had a negative fold enrichment, indicating that UPRT contributes highly to parasite fitness and this likely reflects the observed growth defect. In contrast to UPRT, dCMP deaminase (cgd2_2780) had little effect on parasite fitness. dCMP deaminase catalyses the conversion of deoxycytidine-monophosphate (dCMP) to deoxyuridine-monophosphate (dUMP), which is then transformed into deoxythymidine-monophosphate (dTMP) by DHFR-TS. Thymidine kinase can also produce dTMP, likely explaining why the loss of dCMP deaminase had little effect on parasite fitness.</p>
<p id="Par21">Most of the genes in the vaccine candidate CRISPR screen were identified as fitness conferring, with the exception of apical glycoprotein 1 (AGP1) and thrombospondin repeat protein 11 (TSP11). Akey et al. previously showed AGP1 was dispensable for parasite survival and AGP2 was likely essential, both phenotypes that were recapitulated within our screens<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Recently, it has been suggested that there may be redundancy in the <em>Cryptosporidium</em> thrombospondin protein family (TSPs), as there are 12 members, many with similar domain structures<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Although the <em>Cryptosporidium</em> TSPs currently have an unknown function, orthologous Apicomplexan TSP genes play a role in adhesion and motility. We included two TSPs in the vaccine candidate screen: (1) TSP8 (cgd6_780, also known as MIC1) and (2) TSP11 (cgd6_1660). TSP8/MIC1 is a known micronemal protein<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> and was identified as high fitness conferring. In contrast, TSP11 was dispensable, demonstrating that there is at least some redundancy within the <em>Cryptosporidium</em> TSP protein family. We also confirmed GP40 (cgd6_1080, also known as GP60) and GP900 (cgd7_4020), antigens that are associated with protection from reinfection in humans, influence parasite fitness in vivo<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par22">Cp23 is one of the leading cryptosporidiosis vaccine candidates, and here we demonstrated that it is highly fitness conferring, both in vitro and in vivo. Cp23 was first identified in 1986<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, and recent studies have revealed Cp23 recognising IgA and IgG are correlated with protection against <em>Cryptosporidium</em> infection in humans<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Until now, the function and precise localisation of Cp23 has been unclear. Here we validate a commercial Cp23 monoclonal antibody using diCRE mediated excision of the target gene, then use a combination of ultrastructure expansion, super-resolution, and transmission electron microscopy, to describe the localisation of Cp23 throughout the parasite life cycle at high resolution. Although our localisation agrees with previously published studies that detects Cp23 at the pellicle<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, our data suggests that Cp23 may not be exposed at the surface of the sporozoite. Cp23 has no detectable signal peptide, transmembrane domain or GPI-anchor, yet was recently demonstrated to be both myristoylated and palmitoylated<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Myristylation is a non-reversible lipid modifications that is commonly found in proteins that are anchored to internal membranes in other Apicomplexan parasites<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. There are, however, some exceptions to this rule. TgMIC7 is a micronemal protein in <em>Toxoplasma</em> that is myristoylated prior to trafficking to the parasite surface<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Yet, TgMIC7 contains a transmembrane domain that likely facilitates entry into the secretory pathway, whereas Cp23 does not. While we cannot rule out trafficking of Cp23 to the outer membrane of the parasite, our data supports an internal localisation.</p>
<p id="Par23">Despite this internal localisation, it is clear Cp23 IgA and IgG have both been correlated with protection in the clinic (Gilchrist et al.,<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>). One interpretation of this correlation is that antibodies recognising Cp23 arise from severe <em>Cryptosporidium</em> infection(s) where cell-based adaptive immunity against the parasite has developed. Controlled experiments using a natural murine model of <em>Cryptosporidium</em> suggest that antibody-based immunity is dispensable for resolution of infection<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Further, we found that the genetically validated Cp23 antibody used in this study was unable to block parasite attachment or invasion in vitro (Sup. Fig. <a href="#MOESM1" class="usa-link">8</a>). However, it has been reported that antibodies to Cp23 may have a protective effect in vivo, thus we cannot definitively rule out the possibility of Cp23-directed protection, whether through antibody or cell-based immunity<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Certainly, more investigation is warranted given we now know Cp23’s essential role in parasite motility and survival.</p>
<p id="Par24">With the targeted in vivo CRISPR screening method developed here, we assessed the fitness contributions of 22 <em>Cryptosporidium</em> genes. This number is near to the total number of <em>Cryptosporidium</em> genes with a genetically verified impact on parasite virulence or survival, prior to this study. As this screening technology develops and improves, we anticipate greater throughput, allowing for the assessment of a variety of phenotypes on a larger scale. This rapid assessment of phenotypes will allow us to expand the knowledge base for <em>Cryptosporidium</em> and explore basic biology that is essential for developing more effective interventions.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<p id="Par25">All research in this manuscript was done in accordance with all relevant ethical regulations and animal work was approved by Home Office under project license PP8575470.</p>
<section id="Sec11"><h3 class="pmc_sec_title">Plasmid design and construction</h3>
<p id="Par26">Genomic sites for CRISPR directed repair were predicted using a customised version of EuPaGDT that retrieved 50 bp of flanking DNA surrounding the predicted PAM site<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Golden Gate Assembly or Gibson Assembly was used to generate all vectors used in this work. Golden Gate Assembly used, BsaI, BbsI-HF or BsmBI (New England BioLabs (NEB)) restriction enzymes. Gibson Assembly used HiFi DNA Assembly (NEB). To generate KO vectors for CRISPR screening, 2 consecutive Golden Gate reactions were performed. Firstly, between a Cas9-U6 vector and a 300 bp unique segment containing 50 bp homology arms (one of which contained the 50 bp gRNA), tracrRNA and a DNA barcode, creating a Cas9-unique plasmid. Secondly, between this Cas9-unique plasmid and an interchangeable selection cassette, in turn creating the KO vector. To generate diCRE mediated inducible knockouts, a gene of interest (GOI) was recodonised and cloned into a LoxP-Nluc-NeoR-diCRE vector via Gibson Assembly. To generate Cas9-U6-gRNA vectors, a 20 bp gRNA was cloned into the Cas9-U6-BsaI vector via Golden Gate Assembly<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Culturing host cells</h3>
<p id="Par27">Human ileocecal adenocarcinoma cells (HCT8) were cultured in RPMI-1640 medium (Gibco) supplemented with 10% heat-inactivated foetal bovine serum (Merck), 120 U/mL penicillin (Life Technologies) and 0.1% amphotericin B (Gibco) at 37 °C under 5% CO<sub>2.</sub> Cells were passaged at approximately 70% confluence using 0.25% trypsin-EDTA (Gibco). Cells were used for experiments between passage numbers 5 and 25. For super resolution and ultrastructure expansion microscopy, cells were seeded in 24-well plates containing coverslips. For high throughput microscopy quantifications, cells were seeded in black 96-well clear bottom tissue culture-treated plates (Corning). For transient transfections using luminescence, cells were seeded in 24-well tissue culture-treated plates (Corning). For live imaging, cells were seeded in μ-slide 8-well high chambers (Ibidi).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Oocysts and excystation</h3>
<p id="Par28"><em>C. parvum</em> IOWAII strain oocysts were purchased from Bunch Grass Farm (Deary, ID). Oocysts were stored at 4 °C and used within 3 months of the date of isolation. The oocysts were excysted by incubating on ice with 1% sodium hypochlorite (VWR) in H<sub>2</sub>0 for 5 min, followed by incubating with 0.75% sodium taurocholate (Merck) in RPMI-1640 medium with 1% foetal bovine serum at 37 °C (10 min for cell monolayer infections without transfection and 50 min prior for transfection). For in vitro infections, primed oocysts were used to infect HCT8 monolayers. To proceed with transfections, more than 60% of oocysts had to have excysted, observed using light microscopy at 50 min post sodium taurocholate treatment (Eclipse TS2R Nikon). For Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>, 0.75% sodium taurocholate was substituted with either 0.75% sodium deoxycholate (Sigma) or 0.75% sodium taurodeoxycholate (Sigma), and 1% RPMI-1640 was substituted with PBS.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Generation of transgenic parasites</h3>
<p id="Par29">Excysted sporozoites were suspended in Lonza SF buffer, combined with the appropriate DNA and electroporated using programme ‘FL115’ on an AMAXA Nucleofactor 4D electroporator (Lonza) (FL115 was used for all transfections unless otherwise stated). For transient transfections in vitro, 5.0 × 10<sup>6</sup> oocysts were excysted and sporozoites were transfected in the 20 μL 16-well Nucleocuvette Strip format with 20μg of plasmid. For generating stable transgenic parasites in vivo, 2.5 × 10<sup>7</sup> oocysts were excysted and sporozoites were transfected in the 100 μL Nucleocuvette Vessel format with 20 μg of Cas9-U6-gRNA plasmid and 40μg of repair cassette containing the 50 bp homology arms (Supplementary Table <a href="#MOESM1" class="usa-link">3</a> for all gRNA/homology primers used for individual genetic modifications in this study). For CRISPR screens, 5.0 × 10<sup>6</sup> sporozoites were excysted and sporozoites were transfected with 30 μg of KO vector for each gene (when 2 KO vectors were used, 15 μg of each KO vector was used). Transfections to knockout individual genes were performed separately (when 2 KO vectors per gene were used, transfections of KO vectors for the same gene were performed together). Parasites were pooled post transfection in 1% RPMI-1640 (Gibco) to infect mice and 4 or 5 mice were infected for each screen (Supplementary Tables <a href="#MOESM1" class="usa-link">4</a>–<a href="#MOESM1" class="usa-link">6</a> for all gRNA/homology primers used for individual genetic modifications in this study).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Mouse model of infection</h3>
<p id="Par30">Interferon gamma deficient (<em>Ifnγ</em><sup><em>–/–</em></sup>) mice were bred and housed in pathogen-free conditions in the Biological Research Facility at The Francis Crick Institute. Mice of both sexes were used for experiments and mice were infected between the ages of 4wks to 8wks. To increase infection efficiency, mice were pretreated with an antibiotic cocktail: 1 g/L ampicillin (Merck), 1 g/L streptomycin (Merck) and 0.5 g/L vancomycin (Cambridge Biosciences) in their drinking water for 3–10 days prior to infection with transfected sporozoites. Before infection, mice received saturated sodium bicarbonate (Thermo Fisher Scientific) via oral gavage to neutralise their stomach acid. A second oral gavage was then undertaken 5 min thereafter with transfected sporozoites. Neither oral gavage exceeded the maximum volume of 0.1 mL per 10 g of body weight. For selection of transgenic parasites, 16 mg/mL paromomycin (BioServ) was administered in the mice’s drinking water. To induce excision when diCRE-expressing transgenic parasites were used in vivo, 0.05 mg/mL rapamycin (Stratech Scientific) was administered in the mice’s drinking water. No drug treatment lasted more than 3 consecutive weeks. During murine infections, faecal material was collected daily and stored at 4 °C. All experiments involving mice was done under the care and supervision of the Francis Crick Institute veterinary and Biological Research Facility staff, under protocols approved under project license PP8575470.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Measuring parasite shedding by nanoluciferase</h3>
<p id="Par31">Faecal material was collected and 20 mg was lysed in faecal lysis buffer (50 mM Tris-hydrochloric acid, 10% glycerol, 1% Triton X-100, 2 mM dithiothreitol, 2 mM ethylenediaminetetraacetic acid (EDTA)). The lysate was clarified and combined with an equal volume of a 1:50 Nano-Glo Luciferase Assay Substrate: Nano-Glo Luciferase Assay Buffer (Promega). Luminescence was read at 200 gain on the BioTek Cytation5 (Agilent Technologies).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Isolation of oocysts from mouse faeces</h3>
<p id="Par32">Faecal collections from the peak of infection were pooled, combined with cold water and filtered through a 250 mm mesh. The faecal suspension was then mixed 1:1 with saturated sucrose and pelleted by centrifugation at 1000 × <em>g</em> for 10 min. Oocysts, located in the supernatant, were washed with cold water and pelleted by centrifugation at 1000 g for 5 min. A caesium gradient was used to isolate 750μL of oocysts. Pure oocysts were washed with saline and stored for up to 6 months at 4 °C in saline.</p></section><section id="Sec18"><h3 class="pmc_sec_title">CRISPR screening barcoding</h3>
<p id="Par33">To obtain barcodes from the output, barcodes could either be extracted from the peak of infection or directly from daily faecal samples. To do so from the peak of infection, oocysts were purified from pooled faecal material (5–7 days surrounding the peak of infection), excysted and gDNA was extracted using the DNeasy Blood &amp; Tissue Kit (Qiagen). To do so from daily faecal samples, DNA was extracted from 100 mg of faeces using the QIAamp PowerFecal Pro DNA Kit (Qiagen). Barcodes were obtained from the input material (100 μL of the pool of transfected parasites used to infect the mice) using the QIAquick PCR Purification Kit (Qiagen). Barcodes from the input and output were amplified with high fidelity KAPA polymerase (Roche) (25 cycles, annealing 68 °C and extension 15 seconds), and amplicon sequencing flanks were added with high fidelity KAPA polymerase (Roche) (10 cycles, 68 °C annealing and 15 seconds extension). The success of the nested PCR was confirmed via gel electrophoresis and barcodes were submitted for amplicon sequencing to The Genomics Science Technology Platform at The Francis Crick Institute. Illumina MiSeq platform with a paired-end 250 bp run configuration on a Nano flow cell was used.</p></section><section id="Sec19"><h3 class="pmc_sec_title">CRISPR screening analysis</h3>
<p id="Par34">The 7 bp barcode between the barcode primer binding sites was bioinformatically extracted from both input and output samples. Barcodes for each gene were counted and the percentage of the barcode in the pool calculated for both the input and output. Using the % barcode in the pool for the input and output, fold changes of all genes were calculated:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e1219" display="block"><msub><mrow><mi>l</mi><mi>o</mi><mi>g</mi></mrow><mrow><mn>2</mn></mrow></msub><mspace width="0.16em"></mspace><mi>f</mi><mi>o</mi><mi>l</mi><mi>d</mi><mspace width="0.16em"></mspace><mi>c</mi><mi>h</mi><mi>a</mi><mi>n</mi><mi>g</mi><mi>e</mi><mo>=</mo><msub><mrow><mi>l</mi><mi>o</mi><mi>g</mi></mrow><mrow><mn>2</mn></mrow></msub><mfenced close="]" open="["><mrow><mfrac><mrow><mi>%</mi><mspace width="0.25em"></mspace><mi>b</mi><mi>a</mi><mi>r</mi><mi>c</mi><mi>o</mi><mi>d</mi><mi>e</mi><mspace width="0.16em"></mspace><mi>i</mi><mi>n</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>u</mi><mi>t</mi><mi>p</mi><mi>u</mi><mi>t</mi></mrow><mrow><mi>%</mi><mspace width="0.25em"></mspace><mi>b</mi><mi>a</mi><mi>r</mi><mi>c</mi><mi>o</mi><mi>d</mi><mi>e</mi><mspace width="0.16em"></mspace><mi>i</mi><mi>n</mi><mspace width="0.16em"></mspace><mi>i</mi><mi>n</mi><mi>p</mi><mi>u</mi><mi>t</mi></mrow></mfrac></mrow></mfenced></math></td></tr></table></section><section id="Sec20"><h3 class="pmc_sec_title">Immunofluorescence microscopy</h3>
<p id="Par35">At the required timepoint, HCT8 cell monolayers were washed with 1X PBS, fixed with 4% PFA/PBS (Alfa Aesar) for 15 min, permeabilised with 0.25% Triton X-100 (Merck) for 10 min and blocked with 4% BSA/PBS (Merck) overnight at 4 °C. Primary antibodies were incubated in 1% BSA/PBS for 2 h and then cell monolayers washed 5 times with 1X PBS. Secondary antibodies were incubated for 1 h in 1% BSA/PBS along with the fluorescein labelled 1:4000 Vicia villosa lectin (VVL) (Vector Lab) or 1:1000 Helix pomatia agglutinin (HPA) (Invitrogen) that stains parasites. Nuclei were stained by incubating with 1:10,000 Hoechst 33342 (Invitrogen) in 1X PBS for 5 min. For the EdU assays, 10 mM EdU was incubated from 28 to 32 h post infection and stained as the described in the Click-iT EdU Cell Proliferation Kit for Imaging (<a href="https://www.ncbi.nlm.nih.gov/nuccore/C10340" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">C10340</a>, Invitrogen). Stained monolayers were washed with 1X PBS, the coverslips mounted with ProLong Gold Antifade (Thermo Fisher Scientific) and visualised using a VisiTech instant super resolution imaging system (VT-iSIM). Alternatively, when super resolution was not required, the BioTek Cytation5 (Agilent Technologies) was used to visualise the stained monolayer. See Supplementary Table <a href="#MOESM1" class="usa-link">7</a> for a full list of antibodies used in this work.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Ultrastructure expansion microscopy</h3>
<p id="Par36">This protocol has been adapted from Liffner and Absalon, 2021. For the expansion of sporozoites, coverslips were coated with 0.1 mg/mL poly-D-lysine for 1 h and then washed twice with 1X PBS. Excysted sporozoites in 1% RPMI-1640 were added and allowed to adhere for 10 min at 37 °C. Adhered sporozoites or cell monolayers to expand were fixed with 4% PFA/PBS for 15 min at 37 °C. Protein crosslinking prevention was performed by adding 1.4% formaldehyde/ 2% acrylamide in 1X PBS to each sample, then incubating overnight at 37 °C. To perform gelation, TEMED and APS were added to a monomer solution (19% w/w sodium acrylate / 10% v/v acrylamide / 0.1% v/v N,N’-methylenebisacrylamide in 1X PBS) and pipetted under each coverslip in a pre-cooled humid chamber which was then incubated for 5 min on ice. To complete gelation, the humid chamber was incubated for 1 h at 37 °C. To separate gels from the coverslips, the coverslips were incubated with a denaturation buffer (200 mM SDS, 200 mM NaCl, 50 mM Tris in water, pH 9) for 15 min. To complete the denaturation of the sample, the gel was incubated in a denaturation buffer for 90 min at 95 °C. To expand the sample, the gel was incubated with dH<sub>2</sub>0 for 30 min, three times in total at room temperature (RT). Following the first round of expansion, the sample was shrunk by incubating with 1X PBS, 2 times in total at RT. To block the sample, 2% BSA/PBS was incubated for 1 h at RT. To stain the sample, primary antibodies were incubated in 2% BSA/PBS overnight at RT. Gels were then washed 3 times in total with 0.5% Tween 20/PBS for 10 min. Directly conjugated and secondary antibodies were incubated in 1X PBS for 3 h at RT. The gel was then washed 3 times in total with 0.5% Tween20/PBS for 10 min. A second round of expansion took place by incubating the gel with dH<sub>2</sub>0 for 30 min, three times at RT. The gel was measured to calculate the expansion factor, mounted onto a 0.1 mg/mL poly-D-lysine coated 60 mm dish and visualised on the VT-iSIM microscope.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Immunogold electron microscopy</h3>
<p id="Par37">1 × 10<sup>7</sup> oocysts were excysted for 1 h, pelleted and re-suspended in fixative (8% formaldehyde in 0.4 M HEPES buffer, pH 7.4) for 15 min at RT. Samples were washed with 0.2 M HEPES and a secondary fixation (2% formaldehyde + 0.05% glutaraldehyde in 0.2 M HEPES) step was carried out at 4 <sup>o</sup>C overnight. Samples were dehydrated and infiltrated with LR white resin at −20 <sup>o</sup>C overnight. Prior to polymerisation, samples were brought back to RT for 1 h and then polymerised at 60 <sup>o</sup>C for 24 h. The samples were sectioned using a Leica UC7 ultramicrotome with a 45° Diatome diamond knife, achieving sections of 70 nm thickness. The sections were collected on nickel grids and immunogold labelled. To do so, the samples were quenched with PBS/glycine for 2 min three times and then blocked with 1% BSA/PBS for 5 min at RT. A 1:10 dilution of the primary Cp23 antibody was incubated in 1% BSA/PBS for 1 h and then samples were washed twice with 0.1% BSA/PBS. A 1:50 dilution of the protein-A gold bound to 10 nm gold particles (PAG-10) was incubated with the sample in 0.1% BSA/PBS for 20 min and then washed twice with PBS. A post fixation step was carried out with 1% glutaraldehyde for 5 min and samples were washed with Milli-Q H<sub>2</sub>0 for 1 min 6 times in total. Samples were incubated in 1% uranyl acetate for 10 min and air dried. Transmission electron microscopy was carried out on the 120-kV JEOL JEM-1400Flash Electron Microscope (JEOL Ltd., Welwyn Garden City, UK) with a JEOL Matataki Flash camera.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Live microscopy</h3>
<p id="Par38">At 18 h post-infection, phase contrast imaging was performed using an Eclipse Ts2R microscope (Nikon) with a 40X/0.55 NA Ph1 ADL objective (Nikon), digital sight 10 camera (Nikon) and a TPi-TCSX (Tokai Hit) heated stage set at 37 <sup>o</sup>C. Images were acquired at 25fps for 1 h to capture egressing merozoites. Images were imported into ImageJ2 Version 2 and the number of gliding merozoites was manually observed. Image sequences were converted to movies to acquire short videos of egressing merozoites.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Permeabilisation assay</h3>
<p id="Par39">Coverslips placed in 24-well plates were coated with 0.1 mg/mL poly-D-lysine for 1 h and washed twice with 1X PBS. Excysted sporozoites in serum free RPMI-1640 were allowed to adhere for 10 min at 37 <sup>o</sup>C. Adhered sporozoites were fixed with 1% PFA/PBS for 20 min at RT. For the permeabilised condition, 0.1% Triton X-100 was used to permeabilise for 10 min at RT. For the non-permeabilised condition, 1X PBS was instead incubated for 10 min at RT. Both conditions were blocked with 4% BSA/PBS overnight at 4 °C. Primary antibodies were incubated in 1% BSA/PBS for 2 h at RT. The coverslips were washed 5 times and secondary antibodies were incubated in 1% BSA/PBS for 1 h at RT. All coverslips were mounted with ProLong Gold Antifade (Thermo Fisher Scientific) and visualised using a VT-iSIM.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Attachment assay</h3>
<p id="Par40">Coverslips placed in 24-well plates were coated with 0.025 mg/mL poly-D-lysine for 1 h and washed twice with 1X PBS. 60,000 oocysts per well were excysted and sporozoites were added and spun at 80 g for 1 min to adhere. Attachment was performed in Ringer’s solution (10 mM Hepes pH 6.7, 10 mM Glucose, 2 mM CaCl<sub>2</sub>, 1 mM MgCl<sub>2</sub>, 3 mM KCl, 3 mM NaH<sub>2</sub>PO<sub>4</sub> and 155 mM NaCl) for 15 min at 37 °C. Sporozoites were fixed using 8% PFA/PBS leak in resulting in 4% PFA/PBS/Ringer’s fixation for 15 min at RT. Sporozoites were washed 3 times in total with 1X PBS and blocked with 4% BSA/PBS overnight at 4 °C. Sporozoites were stained with 1:5000 Helix pomatia agglutinin (HPA) in 1% BSA/PBS for 1 h at RT. The BioTek Cytation5 (Agilent Technologies) was used to visualise the attached sporozoites. Images were exported as TIFs into ImageJ2 Version 2 and the below macro was used to analyse the data:</p>
<p id="Par41">run(“Subtract Background…”, “rolling=50”);</p>
<p id="Par42">setAutoThreshold(“Default dark”);</p>
<p id="Par43">//run(“Threshold…”);</p>
<p id="Par44">setAutoThreshold(“Triangle dark”);</p>
<p id="Par45">setOption(“BlackBackground”, true);</p>
<p id="Par46">run(“Convert to Mask”);</p>
<p id="Par47">run(“Median…”, “radius=2”); run(“Analyze Particles…”, “size=50-10000 circularity=0.00-1 display summarize overlay”);</p></section><section id="Sec26"><h3 class="pmc_sec_title">Neutralisation assay</h3>
<p id="Par48">HCT8 cells were seeded to confluency in 96-well clear bottom tissue culture-treated plates (Corning). An 8-point twofold dilution series of the Cp23 antibody (Stratech, LS-C137378) and the IgG isotype control Stratech, GTX35009 + 0.02% Proclin 300 (Merck, 48912-U) was prepared (0.02 mg/mL–0.0002 mg/mL) in 96-well non-tissue culture-treated plates. 25,000 oocysts per well were excysted and incubated with the antibody dilution series for 5 min at 37 °C. Post antibody incubation, the sporozoites were added to the 96-well plate containing HCT8 cells in 1% RPMI-1640. At 24 h post-infection, cell monolayers were washed with 1X PBS, fixed with 4% PFA/PBS (Alfa Aesar) for 15 min, permeabilised with 0.25% Triton X-100 (Sigma) for 10 min and blocked with 4% BSA/PBS (Merck) overnight at 4 <sup>o</sup>C. Parasites were stained with 1:4000 Vicia villosa lectin (VVL) and host nuclei were stained with 1:10,000 Hoechst 33342 (Invitrogen) in 1% BSA/PBS for 1 h at RT. Cell monolayers were washed and visualised on the BioTek Cytation5 (Agilent Technologies). 2 × 2 tiled images per well were acquired with the ×20 objective. The Gen5 analysis software (Agilent Technologies) was used to count the number of host cell nuclei and parasites for parasite per nuclei calculations.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par49">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S1</a><sup> (6.2MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM4_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S2</a><sup> (7.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S3</a><sup> (13MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM6_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S4</a><sup> (9.1MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63012_MOESM7_ESM.pdf</a><sup> (91.8KB, pdf) </sup><p>Description Of Additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (203.4KB, pdf) </sup>
</div></div></section></section><section id="Sec29"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data File</a><sup> (30.4KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Pippa Hawes of The Francis Crick Electron Microscopy Scientific Technology Platform for her helpful discussions and Elena Rodrigues of the Cryptosporidiosis Laboratory for preparing <em>Cryptosporidium</em> samples for electron microscopy. We also thank Nicholas Chisholm and other members of the Biological Research Facility, as well as the Genomics Scientific Technology Platform at The Francis Crick Institute for their contributions to this work, and Rodrigo Baptista of Houston Methodist for his insight and helpful conversations. Lastly, we thank VEuPathDB for maintenance of critical databases that were essential for this work. This work was supported by the Francis Crick Institute, which receives its core funding from Cancer Research UK, the Medical Research Council, and the Wellcome Trust (CC2063), and by a UKRI grant awarded to A.S. (101042783).</p></section><section id="notes1"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par50"><em>Nature Communications</em> thanks Wesley Van Voorhis, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding provided by The Francis Crick Institute.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All raw data used for graphs is available in the Source Data file. <a href="#Sec29" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par51">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec30"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63012-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Troeger, C. et al. Estimates of global, regional, and national morbidity, mortality, and aetiologies of diarrhoeal diseases: A systematic analysis for the Global Burden of Disease Study 2015. <em> Lancet Infect. Dis.</em><strong>17</strong>, 909–948 (2017).
</cite> [<a href="https://doi.org/10.1016/S1473-3099(17)30276-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5589208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28579426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Troeger,%20C.%20et%20al.%20Estimates%20of%20global,%20regional,%20and%20national%20morbidity,%20mortality,%20and%20aetiologies%20of%20diarrhoeal%20diseases:%20A%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202015.%20Lancet%20Infect.%20Dis.17,%20909%E2%80%93948%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Troeger, C. et al. Estimates of the global, regional, and national morbidity, mortality, and aetiologies of diarrhoea in 195 countries: A systematic analysis for the Global Burden of Disease Study 2016. <em> Lancet Infect. Dis.</em><strong>18</strong>, 1211–1228 (2018).
</cite> [<a href="https://doi.org/10.1016/S1473-3099(18)30362-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6202444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30243583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Troeger,%20C.%20et%20al.%20Estimates%20of%20the%20global,%20regional,%20and%20national%20morbidity,%20mortality,%20and%20aetiologies%20of%20diarrhoea%20in%20195%20countries:%20A%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202016.%20Lancet%20Infect.%20Dis.18,%201211%E2%80%931228%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Kotloff, K. L. et al. Burden and aetiology of diarrhoeal disease in infants and young children in developing countries (the Global Enteric Multicenter Study, GEMS): a prospective, case-control study. <em>Lancet</em><strong>382</strong>, 209–222 (2013).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(13)60844-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23680352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kotloff,%20K.%20L.%20et%20al.%20Burden%20and%20aetiology%20of%20diarrhoeal%20disease%20in%20infants%20and%20young%20children%20in%20developing%20countries%20(the%20Global%20Enteric%20Multicenter%20Study,%20GEMS):%20a%20prospective,%20case-control%20study.%20Lancet382,%20209%E2%80%93222%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kotloff, K. L. et al. The incidence, aetiology, and adverse clinical consequences of less severe diarrhoeal episodes among infants and children residing in low-income and middle-income countries: a 12-month case-control study as a follow-on to the Global Enteric Multicenter Study (GEMS). <em> Lancet Glob. Health</em><strong>7</strong>, e568–e584 (2019).
</cite> [<a href="https://doi.org/10.1016/S2214-109X(19)30076-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6484777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31000128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kotloff,%20K.%20L.%20et%20al.%20The%20incidence,%20aetiology,%20and%20adverse%20clinical%20consequences%20of%20less%20severe%20diarrhoeal%20episodes%20among%20infants%20and%20children%20residing%20in%20low-income%20and%20middle-income%20countries:%20a%2012-month%20case-control%20study%20as%20a%20follow-on%20to%20the%20Global%20Enteric%20Multicenter%20Study%20(GEMS).%20Lancet%20Glob.%20Health7,%20e568%E2%80%93e584%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Abubakar, I., Aliyu, S. H., Arumugam, C., Usman, N. K. &amp; Hunter, P. R. Treatment of cryptosporidiosis in immunocompromised individuals: Systematic review and meta-analysis. <em>Br. J. Clin. Pharm.</em><strong>63</strong>, 387–393 (2007).</cite> [<a href="https://doi.org/10.1111/j.1365-2125.2007.02873.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2203234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17335543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abubakar,%20I.,%20Aliyu,%20S.%20H.,%20Arumugam,%20C.,%20Usman,%20N.%20K.%20&amp;%20Hunter,%20P.%20R.%20Treatment%20of%20cryptosporidiosis%20in%20immunocompromised%20individuals:%20Systematic%20review%20and%20meta-analysis.%20Br.%20J.%20Clin.%20Pharm.63,%20387%E2%80%93393%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Amadi, B. et al. High dose prolonged treatment with nitazoxanide is not effective for cryptosporidiosis in HIV positive Zambian children: A randomised controlled trial. <em>BMC Infect. Dis.</em><strong>9</strong>, 195 (2009).
</cite> [<a href="https://doi.org/10.1186/1471-2334-9-195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2794874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19954529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amadi,%20B.%20et%20al.%20High%20dose%20prolonged%20treatment%20with%20nitazoxanide%20is%20not%20effective%20for%20cryptosporidiosis%20in%20HIV%20positive%20Zambian%20children:%20A%20randomised%20controlled%20trial.%20BMC%20Infect.%20Dis.9,%20195%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Vinayak, S. et al. Genetic modification of the diarrhoeal pathogen Cryptosporidium parvum. <em>Nature</em><strong>523</strong>, 477–480 (2015).
</cite> [<a href="https://doi.org/10.1038/nature14651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4640681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26176919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vinayak,%20S.%20et%20al.%20Genetic%20modification%20of%20the%20diarrhoeal%20pathogen%20Cryptosporidium%20parvum.%20Nature523,%20477%E2%80%93480%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Caldwell, N. et al. Cryptosporidium lysyl-tRNA synthetase inhibitors define the interplay between solubility and permeability required to achieve efficacy. <em>Sci. Transl. Med.</em><strong>16</strong>, eadm8631 (2024).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.adm8631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7617456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39441903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caldwell,%20N.%20et%20al.%20Cryptosporidium%20lysyl-tRNA%20synthetase%20inhibitors%20define%20the%20interplay%20between%20solubility%20and%20permeability%20required%20to%20achieve%20efficacy.%20Sci.%20Transl.%20Med.16,%20eadm8631%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Manjunatha, U. H. et al. Cryptosporidium PI(4)K inhibitor EDI048 is a gut-restricted parasiticidal agent to treat paediatric enteric cryptosporidiosis. <em>Nat. Microbiol</em><strong>9</strong>, 2817–2835 (2024).
</cite> [<a href="https://doi.org/10.1038/s41564-024-01810-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11522000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39379634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manjunatha,%20U.%20H.%20et%20al.%20Cryptosporidium%20PI(4)K%20inhibitor%20EDI048%20is%20a%20gut-restricted%20parasiticidal%20agent%20to%20treat%20paediatric%20enteric%20cryptosporidiosis.%20Nat.%20Microbiol9,%202817%E2%80%932835%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ajiboye, J. et al. Identification of potent and orally efficacious phosphodiesterase inhibitors in Cryptosporidium parvum-infected immunocompromised male mice. <em>Nat. Commun.</em><strong>15</strong>, 8272 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-52658-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11436873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39333545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ajiboye,%20J.%20et%20al.%20Identification%20of%20potent%20and%20orally%20efficacious%20phosphodiesterase%20inhibitors%20in%20Cryptosporidium%20parvum-infected%20immunocompromised%20male%20mice.%20Nat.%20Commun.15,%208272%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Sateriale, A. et al. A genetically tractable, natural mouse model of cryptosporidiosis offers insights into host protective immunity. <em>Cell Host Microbe</em><strong>26</strong>, 135–146.e135 (2019).
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6617386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31231045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sateriale,%20A.%20et%20al.%20A%20genetically%20tractable,%20natural%20mouse%20model%20of%20cryptosporidiosis%20offers%20insights%20into%20host%20protective%20immunity.%20Cell%20Host%20Microbe26,%20135%E2%80%93146.e135%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sidik, S. M. et al. A genome-wide CRISPR screen in toxoplasma identifies essential apicomplexan genes. <em>Cell</em><strong>166</strong>, 1423–1435.e1412 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.08.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5017925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27594426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sidik,%20S.%20M.%20et%20al.%20A%20genome-wide%20CRISPR%20screen%20in%20toxoplasma%20identifies%20essential%20apicomplexan%20genes.%20Cell166,%201423%E2%80%931435.e1412%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Young, J. et al. A CRISPR platform for targeted in vivo screens identifies Toxoplasma gondii virulence factors in mice. <em>Nat. Commun.</em><strong>10</strong>, 3963 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-11855-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6722137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31481656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Young,%20J.%20et%20al.%20A%20CRISPR%20platform%20for%20targeted%20in%20vivo%20screens%20identifies%20Toxoplasma%20gondii%20virulence%20factors%20in%20mice.%20Nat.%20Commun.10,%203963%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Bushell, E. et al. Functional profiling of a plasmodium genome reveals an abundance of essential genes. <em>Cell</em><strong>170</strong>, 260–272.e268 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.06.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5509546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28708996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bushell,%20E.%20et%20al.%20Functional%20profiling%20of%20a%20plasmodium%20genome%20reveals%20an%20abundance%20of%20essential%20genes.%20Cell170,%20260%E2%80%93272.e268%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Butterworth, S. et al. High-throughput identification of Toxoplasma gondii effector proteins that target host cell transcription. <em>Cell Host Microbe</em><strong>31</strong>, 1748–1762.e1748 (2023).
</cite> [<a href="https://doi.org/10.1016/j.chom.2023.09.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12033024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37827122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Butterworth,%20S.%20et%20al.%20High-throughput%20identification%20of%20Toxoplasma%20gondii%20effector%20proteins%20that%20target%20host%20cell%20transcription.%20Cell%20Host%20Microbe31,%201748%E2%80%931762.e1748%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Smith, T. A., Lopez-Perez, G. S., Herneisen, A. L., Shortt, E. &amp; Lourido, S. Screening the Toxoplasma kinome with high-throughput tagging identifies a regulator of invasion and egress. <em>Nat. Microbiol.</em><strong>7</strong>, 868–881 (2022).
</cite> [<a href="https://doi.org/10.1038/s41564-022-01104-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9167752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35484233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20T.%20A.,%20Lopez-Perez,%20G.%20S.,%20Herneisen,%20A.%20L.,%20Shortt,%20E.%20&amp;%20Lourido,%20S.%20Screening%20the%20Toxoplasma%20kinome%20with%20high-throughput%20tagging%20identifies%20a%20regulator%20of%20invasion%20and%20egress.%20Nat.%20Microbiol.7,%20868%E2%80%93881%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Pawlowic, M. C. et al. Genetic ablation of purine salvage in Cryptosporidium parvum reveals nucleotide uptake from the host cell. <em>Proc. Natl. Acad. Sci.</em><strong>116</strong>, 21160–21165 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1908239116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6800313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31570573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pawlowic,%20M.%20C.%20et%20al.%20Genetic%20ablation%20of%20purine%20salvage%20in%20Cryptosporidium%20parvum%20reveals%20nucleotide%20uptake%20from%20the%20host%20cell.%20Proc.%20Natl.%20Acad.%20Sci.116,%2021160%E2%80%9321165%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Striepen, B. et al. Gene transfer in the evolution of parasite nucleotide biosynthesis. <em>Proc. Natl. Acad. Sci. USA</em><strong>101</strong>, 3154–3159 (2004).
</cite> [<a href="https://doi.org/10.1073/pnas.0304686101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC365759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14973196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Striepen,%20B.%20et%20al.%20Gene%20transfer%20in%20the%20evolution%20of%20parasite%20nucleotide%20biosynthesis.%20Proc.%20Natl.%20Acad.%20Sci.%20USA101,%203154%E2%80%933159%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Jefferies, R. et al. Target validation of the inosine monophosphate dehydrogenase (IMPDH) gene in Cryptosporidium using Phylomer® peptides. <em>Exp. Parasitol.</em><strong>148</strong>, 40–48 (2015).
</cite> [<a href="https://doi.org/10.1016/j.exppara.2014.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25447124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jefferies,%20R.%20et%20al.%20Target%20validation%20of%20the%20inosine%20monophosphate%20dehydrogenase%20(IMPDH)%20gene%20in%20Cryptosporidium%20using%20Phylomer%C2%AE%20peptides.%20Exp.%20Parasitol.148,%2040%E2%80%9348%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Haque, R. et al. Prospective case-control study of the association between common enteric protozoal parasites and diarrhea in Bangladesh. <em>Clin. Infect. Dis.</em><strong>48</strong>, 1191–1197 (2009).
</cite> [<a href="https://doi.org/10.1086/597580" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2883291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19323634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haque,%20R.%20et%20al.%20Prospective%20case-control%20study%20of%20the%20association%20between%20common%20enteric%20protozoal%20parasites%20and%20diarrhea%20in%20Bangladesh.%20Clin.%20Infect.%20Dis.48,%201191%E2%80%931197%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Sow, S. O. et al. The Burden of Cryptosporidium Diarrheal Disease among Children &lt;24 Months of Age in Moderate/High Mortality Regions of Sub-Saharan Africa and South Asia, Utilizing Data from the Global Enteric Multicenter Study (GEMS). <em>PLoS Negl. Trop. Dis.</em><strong>10</strong>, e0004729 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pntd.0004729" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4878811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27219054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sow,%20S.%20O.%20et%20al.%20The%20Burden%20of%20Cryptosporidium%20Diarrheal%20Disease%20among%20Children%20&lt;24%20Months%20of%20Age%20in%20Moderate/High%20Mortality%20Regions%20of%20Sub-Saharan%20Africa%20and%20South%20Asia,%20Utilizing%20Data%20from%20the%20Global%20Enteric%20Multicenter%20Study%20(GEMS).%20PLoS%20Negl.%20Trop.%20Dis.10,%20e0004729%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Jenkins, M., Higgins, J., Kniel, K., Trout, J. &amp; Fayer, R. Protection of calves against cryptosporiosis by oral inoculation with gamma-irradiated Cryptosporidium parvum oocysts. <em>J. Parasitol.</em><strong>90</strong>, 1178–1180 (2004).
</cite> [<a href="https://doi.org/10.1645/GE-3333RN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15562625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jenkins,%20M.,%20Higgins,%20J.,%20Kniel,%20K.,%20Trout,%20J.%20&amp;%20Fayer,%20R.%20Protection%20of%20calves%20against%20cryptosporiosis%20by%20oral%20inoculation%20with%20gamma-irradiated%20Cryptosporidium%20parvum%20oocysts.%20J.%20Parasitol.90,%201178%E2%80%931180%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Yen, C. et al. Rotavirus vaccines: current status and future considerations. <em>Hum. Vaccin Immunother.</em><strong>10</strong>, 1436–1448 (2014).
</cite> [<a href="https://doi.org/10.4161/hv.28857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4185955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24755452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yen,%20C.%20et%20al.%20Rotavirus%20vaccines:%20current%20status%20and%20future%20considerations.%20Hum.%20Vaccin%20Immunother.10,%201436%E2%80%931448%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>O’Ryan, M. Rotavirus Vaccines: a story of success with challenges ahead. <em>F1000Res</em><strong>6</strong>, 1517 (2017).
</cite> [<a href="https://doi.org/10.12688/f1000research.11912.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5580433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28928954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Ryan,%20M.%20Rotavirus%20Vaccines:%20a%20story%20of%20success%20with%20challenges%20ahead.%20F1000Res6,%201517%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Gilchrist C. A. et al. Specific Cryptosporidium antigens associate with reinfection immunity and protection from cryptosporidiosis. <em>J Clin Invest</em><strong>133</strong>, (2023).</cite> [<a href="https://doi.org/10.1172/JCI166814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10425216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37347553/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Manque Patricio, A. et al. Identification and Immunological Characterization of Three Potential Vaccinogens against Cryptosporidium Species. <em>Clin. Vaccin. Immunol.</em><strong>18</strong>, 1796–1802 (2011).</cite> [<a href="https://doi.org/10.1128/CVI.05197-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3209039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21918117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manque%20Patricio,%20A.%20et%20al.%20Identification%20and%20Immunological%20Characterization%20of%20Three%20Potential%20Vaccinogens%20against%20Cryptosporidium%20Species.%20Clin.%20Vaccin.%20Immunol.18,%201796%E2%80%931802%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Askari, N. et al. Evaluation of recombinant P23 protein as a vaccine for passive immunization of newborn calves against Cryptosporidium parvum. <em>Parasite Immunol.</em><strong>38</strong>, 282–289 (2016).
</cite> [<a href="https://doi.org/10.1111/pim.12317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27012710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Askari,%20N.%20et%20al.%20Evaluation%20of%20recombinant%20P23%20protein%20as%20a%20vaccine%20for%20passive%20immunization%20of%20newborn%20calves%20against%20Cryptosporidium%20parvum.%20Parasite%20Immunol.38,%20282%E2%80%93289%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Tandel, J. et al. Genetic ablation of a female-specific apetala 2 transcription factor blocks oocyst shedding in cryptosporidium parvum. <em>mBio</em><strong>14</strong>, e0326122 (2023).
</cite> [<a href="https://doi.org/10.1128/mbio.03261-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10233709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36786597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tandel,%20J.%20et%20al.%20Genetic%20ablation%20of%20a%20female-specific%20apetala%202%20transcription%20factor%20blocks%20oocyst%20shedding%20in%20cryptosporidium%20parvum.%20mBio14,%20e0326122%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Shaw, S. et al. Genetic crosses within and between species of Cryptosporidium. <em>Proc. Natl. Acad. Sci.</em><strong>121</strong>, e2313210120 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2313210120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10769859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38147547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaw,%20S.%20et%20al.%20Genetic%20crosses%20within%20and%20between%20species%20of%20Cryptosporidium.%20Proc.%20Natl.%20Acad.%20Sci.121,%20e2313210120%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Cohen, E. A., Gaudreau, P., Brazeau, P. &amp; Langelier, Y. Specific inhibition of herpesvirus ribonucleotide reductase by a nonapeptide derived from the carboxy terminus of subunit 2. <em>Nature</em><strong>321</strong>, 441–443 (1986).
</cite> [<a href="https://doi.org/10.1038/321441a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3012360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cohen,%20E.%20A.,%20Gaudreau,%20P.,%20Brazeau,%20P.%20&amp;%20Langelier,%20Y.%20Specific%20inhibition%20of%20herpesvirus%20ribonucleotide%20reductase%20by%20a%20nonapeptide%20derived%20from%20the%20carboxy%20terminus%20of%20subunit%202.%20Nature321,%20441%E2%80%93443%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Dutia, B. M., Frame, M. C., Subak-Sharpe, J. H., Clark, W. N. &amp; Marsden, H. S. Specific inhibition of herpesvirus ribonucleotide reductase by synthetic peptides. <em>Nature</em><strong>321</strong>, 439–441 (1986).
</cite> [<a href="https://doi.org/10.1038/321439a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3012359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dutia,%20B.%20M.,%20Frame,%20M.%20C.,%20Subak-Sharpe,%20J.%20H.,%20Clark,%20W.%20N.%20&amp;%20Marsden,%20H.%20S.%20Specific%20inhibition%20of%20herpesvirus%20ribonucleotide%20reductase%20by%20synthetic%20peptides.%20Nature321,%20439%E2%80%93441%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Akey, M. E., Xu, R., Ravindran, S., Funkhouser-Jones, L. &amp; Sibley, L. D. Apical Secretory Glycoprotein Complex Contributes to Cell Attachment and Entry by Cryptosporidium parvum. <em>mBio</em><strong>14</strong>, e0306422 (2023).
</cite> [<a href="https://doi.org/10.1128/mbio.03064-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9973360/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36722968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akey,%20M.%20E.,%20Xu,%20R.,%20Ravindran,%20S.,%20Funkhouser-Jones,%20L.%20&amp;%20Sibley,%20L.%20D.%20Apical%20Secretory%20Glycoprotein%20Complex%20Contributes%20to%20Cell%20Attachment%20and%20Entry%20by%20Cryptosporidium%20parvum.%20mBio14,%20e0306422%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Ungar, B. L. &amp; Nash, T. E. Quantification of specific antibody response to Cryptosporidium antigens by laser densitometry. <em>Infect. Immun.</em><strong>53</strong>, 124–128 (1986).
</cite> [<a href="https://doi.org/10.1128/iai.53.1.124-128.1986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC260085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3522424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ungar,%20B.%20L.%20&amp;%20Nash,%20T.%20E.%20Quantification%20of%20specific%20antibody%20response%20to%20Cryptosporidium%20antigens%20by%20laser%20densitometry.%20Infect.%20Immun.53,%20124%E2%80%93128%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Mead, J. R., Arrowood, M. J. &amp; Sterling, C. R. Antigens of Cryptosporidium sporozoites recognized by immune sera of infected animals and humans. <em>J. Parasitol.</em><strong>74</strong>, 135–143 (1988).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3282049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mead,%20J.%20R.,%20Arrowood,%20M.%20J.%20&amp;%20Sterling,%20C.%20R.%20Antigens%20of%20Cryptosporidium%20sporozoites%20recognized%20by%20immune%20sera%20of%20infected%20animals%20and%20humans.%20J.%20Parasitol.74,%20135%E2%80%93143%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Arrowood, M. J., Sterling, C. R. &amp; Healey, M. C. Immunofluorescent microscopical visualization of trails left by gliding Cryptosporidium parvum sporozoites. <em>J. Parasitol.</em><strong>77</strong>, 315–317 (1991).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2010865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arrowood,%20M.%20J.,%20Sterling,%20C.%20R.%20&amp;%20Healey,%20M.%20C.%20Immunofluorescent%20microscopical%20visualization%20of%20trails%20left%20by%20gliding%20Cryptosporidium%20parvum%20sporozoites.%20J.%20Parasitol.77,%20315%E2%80%93317%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Guérin, A. et al. Cryptosporidium uses multiple distinct secretory organelles to interact with and modify its host cell. <em>Cell Host Microbe</em><strong>31</strong>, 650–664.e656 (2023).
</cite> [<a href="https://doi.org/10.1016/j.chom.2023.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36958336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu%C3%A9rin,%20A.%20et%20al.%20Cryptosporidium%20uses%20multiple%20distinct%20secretory%20organelles%20to%20interact%20with%20and%20modify%20its%20host%20cell.%20Cell%20Host%20Microbe31,%20650%E2%80%93664.e656%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Keeley, A. &amp; Soldati, D. The glideosome: a molecular machine powering motility and host-cell invasion by Apicomplexa. <em>Trends Cell Biol.</em><strong>14</strong>, 528–532 (2004).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2004.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15450974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keeley,%20A.%20&amp;%20Soldati,%20D.%20The%20glideosome:%20a%20molecular%20machine%20powering%20motility%20and%20host-cell%20invasion%20by%20Apicomplexa.%20Trends%20Cell%20Biol.14,%20528%E2%80%93532%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kimball, A. et al. Mendelian segregation and high recombination rates facilitate genetic analyses in Cryptosporidium parvum. <em>PLOS Genet.</em><strong>20</strong>, e1011162 (2024).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1011162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11213348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38885280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kimball,%20A.%20et%20al.%20Mendelian%20segregation%20and%20high%20recombination%20rates%20facilitate%20genetic%20analyses%20in%20Cryptosporidium%20parvum.%20PLOS%20Genet.20,%20e1011162%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>John, A. et al. Conservation, abundance, glycosylation profile, and localization of the TSP protein family in Cryptosporidium parvum. <em>J. Biol. Chem.</em><strong>299</strong>, 103006 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2023.103006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10034466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36775128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?John,%20A.%20et%20al.%20Conservation,%20abundance,%20glycosylation%20profile,%20and%20localization%20of%20the%20TSP%20protein%20family%20in%20Cryptosporidium%20parvum.%20J.%20Biol.%20Chem.299,%20103006%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Putignani, L. et al. The thrombospondin-related protein CpMIC1 (CpTSP8) belongs to the repertoire of micronemal proteins of Cryptosporidium parvum. <em>Mol. Biochem Parasitol.</em><strong>157</strong>, 98–101 (2008).
</cite> [<a href="https://doi.org/10.1016/j.molbiopara.2007.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17981348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Putignani,%20L.%20et%20al.%20The%20thrombospondin-related%20protein%20CpMIC1%20(CpTSP8)%20belongs%20to%20the%20repertoire%20of%20micronemal%20proteins%20of%20Cryptosporidium%20parvum.%20Mol.%20Biochem%20Parasitol.157,%2098%E2%80%93101%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Enriquez, F. J. &amp; Riggs, M. W. Role of immunoglobulin A monoclonal antibodies against P23 in controlling murine Cryptosporidium parvum infection. <em>Infect. Immun.</em><strong>66</strong>, 4469–4473 (1998).
</cite> [<a href="https://doi.org/10.1128/iai.66.9.4469-4473.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC108540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9712802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Enriquez,%20F.%20J.%20&amp;%20Riggs,%20M.%20W.%20Role%20of%20immunoglobulin%20A%20monoclonal%20antibodies%20against%20P23%20in%20controlling%20murine%20Cryptosporidium%20parvum%20infection.%20Infect.%20Immun.66,%204469%E2%80%934473%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Haserick, J. R., Klein, J. A., Costello, C. E. &amp; Samuelson, J. Cryptosporidium parvum vaccine candidates are incompletely modified with O-linked-N-acetylgalactosamine or contain N-terminal N-myristate and S-palmitate. <em>PLoS One</em><strong>12</strong>, e0182395 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0182395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5549699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28792526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haserick,%20J.%20R.,%20Klein,%20J.%20A.,%20Costello,%20C.%20E.%20&amp;%20Samuelson,%20J.%20Cryptosporidium%20parvum%20vaccine%20candidates%20are%20incompletely%20modified%20with%20O-linked-N-acetylgalactosamine%20or%20contain%20N-terminal%20N-myristate%20and%20S-palmitate.%20PLoS%20One12,%20e0182395%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Schlott, A. C., Holder, A. A. &amp; Tate, E. W. N-myristoylation as a drug target in malaria: Exploring the role of N-myristoyltransferase substrates in the inhibitor mode of action. <em>ACS Infect. Dis.</em><strong>4</strong>, 449–457 (2018).
</cite> [<a href="https://doi.org/10.1021/acsinfecdis.7b00203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29363940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schlott,%20A.%20C.,%20Holder,%20A.%20A.%20&amp;%20Tate,%20E.%20W.%20N-myristoylation%20as%20a%20drug%20target%20in%20malaria:%20Exploring%20the%20role%20of%20N-myristoyltransferase%20substrates%20in%20the%20inhibitor%20mode%20of%20action.%20ACS%20Infect.%20Dis.4,%20449%E2%80%93457%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Broncel, M. et al. Profiling of myristoylation in Toxoplasma gondii reveals an N-myristoylated protein important for host cell penetration. <em>eLife</em><strong>9</strong>, e57861 (2020).
</cite> [<a href="https://doi.org/10.7554/eLife.57861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7373427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32618271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Broncel,%20M.%20et%20al.%20Profiling%20of%20myristoylation%20in%20Toxoplasma%20gondii%20reveals%20an%20N-myristoylated%20protein%20important%20for%20host%20cell%20penetration.%20eLife9,%20e57861%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Alvarez-Jarreta, J. et al. VEuPathDB: The eukaryotic pathogen, vector and host bioinformatics resource center in 2023. <em>Nucleic Acids Res.</em><strong>52</strong>, D808–D816 (2023).</cite> [<a href="https://doi.org/10.1093/nar/gkad1003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37953350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alvarez-Jarreta,%20J.%20et%20al.%20VEuPathDB:%20The%20eukaryotic%20pathogen,%20vector%20and%20host%20bioinformatics%20resource%20center%20in%202023.%20Nucleic%20Acids%20Res.52,%20D808%E2%80%93D816%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Pawlowic, M. C., Vinayak, S., Sateriale, A., Brooks, C. F. &amp; Striepen, B. Generating and maintaining transgenic cryptosporidium parvum parasites. <em>Curr. Protoc. Microbiol</em><strong>46</strong>, 20b.22.21–20b.22.32 (2017).</cite> [<a href="https://doi.org/10.1002/cpmc.33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5556942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28800157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pawlowic,%20M.%20C.,%20Vinayak,%20S.,%20Sateriale,%20A.,%20Brooks,%20C.%20F.%20&amp;%20Striepen,%20B.%20Generating%20and%20maintaining%20transgenic%20cryptosporidium%20parvum%20parasites.%20Curr.%20Protoc.%20Microbiol46,%2020b.22.21%E2%80%9320b.22.32%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S1</a><sup> (6.2MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM4_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S2</a><sup> (7.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S3</a><sup> (13MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM6_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie S4</a><sup> (9.1MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63012_MOESM7_ESM.pdf</a><sup> (91.8KB, pdf) </sup><p>Description Of Additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (203.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368253/bin/41467_2025_63012_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data File</a><sup> (30.4KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All raw data used for graphs is available in the Source Data file. <a href="#Sec29" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63012-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63012.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368253/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368253/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368253%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368253/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368253/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368253/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835841/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368253/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835841/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368253/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368253/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="7KVaiM680WDHOTUyZv11g0HkwayWoHrQKKXFfm2aRJVcXnhRnvXU5YLMFa7ZKFVW">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
