
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Skin in the game: a review of computational models of the skin - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152552628AF438330552620001B0A67C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="biodmin">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366154/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BioData Mining">
<meta name="citation_title" content="Skin in the game: a review of computational models of the skin">
<meta name="citation_author" content="Seda Ceylan">
<meta name="citation_author_institution" content="Department of Bioengineering, Adana Alparslan Türkeş Science and Technology University, Adana, Türkiye">
<meta name="citation_author" content="Didem Demir">
<meta name="citation_author_institution" content="Department of Chemistry and Chemical Process Technologies, Tarsus University, Mersin, Türkiye">
<meta name="citation_author" content="Cayla Harris">
<meta name="citation_author_institution" content="NICE research group, School of Computer Science and Electronic Engineering, University of Surrey, Guildford, United Kingdom">
<meta name="citation_author" content="Semih Latif İpek">
<meta name="citation_author_institution" content="Department of Food Engineering, Adana Alparslan Türkeş Science and Technology University, Adana, Türkiye">
<meta name="citation_author_institution" content="Department of Chemistry, Kütahya Dumlupınar University, Kütahya, Türkiye">
<meta name="citation_author" content="Vasileios Vavourakis">
<meta name="citation_author_institution" content="Department of Medical Physics and Biomedical Engineering, University College London, London, UK">
<meta name="citation_author_institution" content="In Silico Modelling Group, Department of Mechanical and Manufacturing Engineering, University of Cyprus, Nicosia, Cyprus">
<meta name="citation_author" content="Marco Manca">
<meta name="citation_author_institution" content="SCimPulse Foundation, Geleen, The Netherlands">
<meta name="citation_author" content="Sandrine Dubrac">
<meta name="citation_author_institution" content="Department of Dermatology, Venereology and Allergology, Medical University of Innsbruck, Innsbruck, Austria">
<meta name="citation_author" content="Roman Bauer">
<meta name="citation_author_institution" content="NICE research group, School of Computer Science and Electronic Engineering, University of Surrey, Guildford, United Kingdom">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="18">
<meta name="citation_firstpage" content="55">
<meta name="citation_doi" content="10.1186/s13040-025-00471-8">
<meta name="citation_pmid" content="40830504">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366154/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366154/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366154/pdf/13040_2025_Article_471.pdf">
<meta name="description" content="With the vast advances in computing technology, computational (or in silico) modelling has emerged as a transformative tool in dermatology. These findings can provide novel insights into complex biological processes and aid in the development of ...">
<meta name="og:title" content="Skin in the game: a review of computational models of the skin">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="With the vast advances in computing technology, computational (or in silico) modelling has emerged as a transformative tool in dermatology. These findings can provide novel insights into complex biological processes and aid in the development of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366154/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366154">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13040-025-00471-8"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13040_2025_Article_471.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366154%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366154/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366154/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366154/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-biodmin.png" alt="BioData Mining logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BioData Mining" title="Link to BioData Mining" shape="default" href="https://biodatamining.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BioData Min</button></div>. 2025 Aug 19;18:55. doi: <a href="https://doi.org/10.1186/s13040-025-00471-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13040-025-00471-8</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BioData%20Min%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BioData%20Min%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BioData%20Min%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BioData%20Min%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Skin in the game: a review of computational models of the skin</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ceylan%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Seda Ceylan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Seda Ceylan</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering, Adana Alparslan Türkeş Science and Technology University, Adana, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ceylan%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Seda Ceylan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demir%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Didem Demir</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Didem Demir</span></h3>
<div class="p">
<sup>2</sup>Department of Chemistry and Chemical Process Technologies, Tarsus University, Mersin, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demir%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Didem Demir</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harris%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Cayla Harris</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Cayla Harris</span></h3>
<div class="p">
<sup>3</sup>NICE research group, School of Computer Science and Electronic Engineering, University of Surrey, Guildford, United Kingdom </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harris%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cayla Harris</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C4%B0pek%20SL%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Semih Latif İpek</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Semih Latif İpek</span></h3>
<div class="p">
<sup>4</sup>Department of Food Engineering, Adana Alparslan Türkeş Science and Technology University, Adana, Türkiye </div>
<div class="p">
<sup>5</sup>Department of Chemistry, Kütahya Dumlupınar University, Kütahya, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C4%B0pek%20SL%22%5BAuthor%5D" class="usa-link"><span class="name western">Semih Latif İpek</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vavourakis%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Vasileios Vavourakis</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Vasileios Vavourakis</span></h3>
<div class="p">
<sup>6</sup>Department of Medical Physics and Biomedical Engineering, University College London, London, UK </div>
<div class="p">
<sup>7</sup>In Silico Modelling Group, Department of Mechanical and Manufacturing Engineering, University of Cyprus, Nicosia, Cyprus </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vavourakis%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vasileios Vavourakis</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manca%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Marco Manca</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Marco Manca</span></h3>
<div class="p">
<sup>8</sup>SCimPulse Foundation, Geleen, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manca%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marco Manca</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dubrac%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Sandrine Dubrac</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Sandrine Dubrac</span></h3>
<div class="p">
<sup>9</sup>Department of Dermatology, Venereology and Allergology, Medical University of Innsbruck, Innsbruck, Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dubrac%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sandrine Dubrac</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bauer%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Roman Bauer</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Roman Bauer</span></h3>
<div class="p">
<sup>3</sup>NICE research group, School of Computer Science and Electronic Engineering, University of Surrey, Guildford, United Kingdom </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bauer%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Roman Bauer</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Bioengineering, Adana Alparslan Türkeş Science and Technology University, Adana, Türkiye </div>
<div id="Aff2">
<sup>2</sup>Department of Chemistry and Chemical Process Technologies, Tarsus University, Mersin, Türkiye </div>
<div id="Aff3">
<sup>3</sup>NICE research group, School of Computer Science and Electronic Engineering, University of Surrey, Guildford, United Kingdom </div>
<div id="Aff4">
<sup>4</sup>Department of Food Engineering, Adana Alparslan Türkeş Science and Technology University, Adana, Türkiye </div>
<div id="Aff5">
<sup>5</sup>Department of Chemistry, Kütahya Dumlupınar University, Kütahya, Türkiye </div>
<div id="Aff6">
<sup>6</sup>Department of Medical Physics and Biomedical Engineering, University College London, London, UK </div>
<div id="Aff7">
<sup>7</sup>In Silico Modelling Group, Department of Mechanical and Manufacturing Engineering, University of Cyprus, Nicosia, Cyprus </div>
<div id="Aff8">
<sup>8</sup>SCimPulse Foundation, Geleen, The Netherlands </div>
<div id="Aff9">
<sup>9</sup>Department of Dermatology, Venereology and Allergology, Medical University of Innsbruck, Innsbruck, Austria </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 16; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366154  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830504/" class="usa-link">40830504</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">With the vast advances in computing technology, computational (or in silico) modelling has emerged as a transformative tool in dermatology. These findings can provide novel insights into complex biological processes and aid in the development of innovative therapeutic and regenerative strategies for the skin. Modelling combines experimental data and knowledge across multiple disciplines, serving as a common framework to elucidate the workings of the skin. From a biomedical perspective, the mechanisms of skin diseases can be studied by simulating cellular interactions and signalling pathways. Computational investigations of these mechanisms can be categorised into two distinct approaches: data-driven and model-based. Data-driven approaches allow the diagnosis of skin diseases on the basis of data collection via imaging or feedback from portable sensors, often yielding performance exceeding that of their human counterparts. Model-based methods are well suited to address topics such as skin cell biology and biomechanics, contributing to wound healing and skin cancer research. Furthermore, such modelling has found utility in the development of virtual skin models and skin-on-chip devices, enabling the prediction of skin responses to various substances, including cosmetics and drugs. In the realm of dermatological surgery, computational tools have been instrumental in optimizing surgical planning and improving clinical outcomes. While significant advancements have been made, challenges such as data availability, model validation, and interdisciplinary collaboration persist. This review highlights the current state-of-the-art in computational modeling in dermatology, identifies key challenges, and outlines its prospects.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Skin, Dermis, Epidermis, Modelling, Skin biomechanics, Computational dermatology, In silico</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">The skin: a complex organ</h2>
<p id="Par2">The skin, the largest organ of the body, is a multilayered barrier that provides both physical and immunological protection. It plays a central role in numerous aspects of essential physiological function, including protection from germs and UV radiation, and the regulation of body temperature. These essential roles make it the subject of extensive biomedical and pharmaceutical research.</p>
<p id="Par3">The multiple layers of this complex organ include the epidermis, the dermis, and the hypodermis (subcutaneous tissue) (Fig. <a href="#Fig1" class="usa-link">1</a>). The epidermis is the outermost layer of the skin, providing the first protective barrier, and consists of four to five distinct layers: the <em>stratum basale</em>, <em>stratum spinosum</em>, <em>stratum granulosum</em>, <em>stratum lucidum</em> (only in soles and palms), and <em>stratum corneum</em>. Keratinocytes, melanocytes, Langerhans cells, and Merkel cells constitute the epidermis [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The dermis consists of two distinct areas: the superficial papillary dermis and the deeper reticular dermis. These areas contain collagen, mostly type I or type III, and elastic fibres such as elastin and fibrillin microfibrils [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Various cell types, including fibroblasts, immune cells (e.g., macrophages, mast cells), adipocytes and Schwann cells, populate the dermis, which also includes skin appendages such as hair follicles, sweat glands (eccrine), and apocrine and sebaceous glands [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The hypodermis is composed of highly-vascularized, loose, areolar connective tissue, consisting of adipocytes, fibroblasts and macrophages, embedded in an extracellular matrix rich in molecules such as proteoglycans and glycosaminoglycans [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366154_13040_2025_471_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d724/12366154/913d9eedf804/13040_2025_471_Fig1_HTML.jpg" loading="lazy" id="MO1" height="222" width="708" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Anatomical skin layers and the basic content of each layer in terms of structure, composition and cells present</p></figcaption></figure><p id="Par4">Given the range and impact of skin diseases around the world, research is essential to advance the understanding of skin pathophysiology and to develop new treatments.</p>
<p id="Par5">Skin diseases are recognized as important public health problems affecting millions of people, involving high costs for individuals and societies, and are among the leading causes of the global burden of disease. To date, more than 3,000 skin diseases, both acute and chronic, affecting individuals of all ages, sexes, ethnicities and social conditions have been described. Among these diseases, some can cause a significant deterioration in quality of life and represent substantial costs for patients [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<p id="Par6">Skin diseases encompass a spectrum of conditions, such as [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par7"><strong>Inflammation</strong>: acne, dermatitis, eczema, hives, pityriasis rosea, and psoriasis</p></li>
<li><p id="Par8"><strong>Viral infections</strong>: herpes simplex, herpes zoster, and warts</p></li>
<li><p id="Par9"><strong>Bacterial infections</strong>: impetigo, folliculitis, cellulitis, and Lyme disease</p></li>
<li><p id="Par10"><strong>Fungal infections</strong>: athlete’s foot, jock itch, and ringworm</p></li>
<li><p id="Par11"><strong>Pigmentation disorders</strong>: age spots, freckles, melasma, and vitiligo</p></li>
<li><p id="Par12"><strong>Tumors and cancers</strong>: basal cell carcinoma, squamous cell carcinoma, and malignant melanoma</p></li>
<li><p id="Par13"><strong>Other conditions</strong>: traumatic injuries, rosacea, spider veins, and varicose veins</p></li>
</ul>
<p>Among these, dermatitis, psoriasis, and melanoma are associated with some of the highest global values for Disability-Adjusted Life Years (DALYs) [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>].</p>
<p id="Par14">Research and development for the treatment of skin diseases worldwide is considerable. Given the extensive financial and temporal investments required for the development and introduction of a new drug to the market, combined with the long periods between preclinical testing and drug approval, the use of accurate in vitro models in early stages of drug development is becoming increasingly common [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. A prerequisite to drug development is a deep understanding of pathogenic processes that help identify key targets in the skin. Mathematical modelling of skin diseases can help provide tools for understanding complex and dynamic pathological mechanisms. The power of mathematical models, coupled with in vitro experiments, can reveal previously unidentified regulatory mechanisms [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].</p>
<p id="Par15">Computational modelling, which encompasses mathematical modelling, is a discipline that aims to simulate complex systems by employing mathematics, physics, and computer science. A computational model contains a multitude of variables that define the characteristics of a system under study. Simulations are conducted by adjusting the variables, individually or in combination, and observing the outcomes. This enables scientists to perform thousands of simulated experiments in a very short time. The results of these simulated experiments can then be used to assess and compare both existing and novel hypotheses. Computational models can aid in the study of a biological system at multiple levels and across different scales. These include models of disease aetiology, such as molecular processes, cell–cell interactions, and changes affecting tissues and organs. This review summarises the current techniques of computational modelling dedicated to studying the skin, with applications in research into fundamental skin properties, diseases, healing, and ageing. We aim to define the current technologies already available and reveal the direction in which research and development is heading. This presents a compilation of studies on modelling the biological factors and physical structure of skin tissue, all of which play an active role in skin disease modelling studies. Accurate modelling of these diverse biological and physical properties of the skin is highly important for understanding the aetiology of skin diseases, developing treatment strategies, and testing new treatment methods. Examples of these properties are described in Fig. <a href="#Fig2" class="usa-link">2</a>, which provides a schematic representation of the physical and biological properties that are frequently employed in the context of skin modelling studies. In summary, the success of research on skin diseases is based on accurate modelling of the complex biological and physical structure of the skin, which will be explored and evaluated within the scope of this review.</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366154_13040_2025_471_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d724/12366154/136e318f5e7f/13040_2025_471_Fig2_HTML.jpg" loading="lazy" id="MO2" height="261" width="709" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Computational model perspective for skin problems summarized by illustrating the main biological and physical properties of skin</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Data-driven vs. model-based computational models</h2>
<p id="Par16">Fundamentally, there are two main types of approaches for computational modelling. The traditional type denotes those that are primarily <em>model-based</em>, relying on established theoretical frameworks and mathematical equations to represent the underlying phenomena. Clearly, these require some prior insight and knowledge of the complex system at hand. In the case of the skin, the model could incorporate equations describing diffusion in different layers of the skin, the fact that there are different layers and cell types, or some assumptions underlying collagen dynamics. Hence, while these models provide valuable insights, they can be challenging to conceive, as they often require significant domain expertise and can be limited by the accuracy of the underlying assumptions.</p>
<p id="Par17">In contrast, <em>data-driven</em> models have emerged as powerful alternatives, particularly with the proliferation of large datasets and advancements in computing power. These models learn patterns directly from data without explicit theoretical assumptions. They excel at capturing complex relationships and making predictions in scenarios where the underlying mechanisms are poorly understood. For example, in the context of skin research, automated image analysis methods for the diagnosis of melanoma have been developed and successfully employed in clinical practice, as explained below.</p>
<p id="Par18">Both model-based and data-driven approaches have advantages and disadvantages (Table <a href="#Tab1" class="usa-link">1</a>), and the modeller can choose the one that leverages existing data and background knowledge to achieve the given goals of the research [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. The availability of suitable hardware and software should also be considered. Different computational approaches can fit better with different resources. For example, mechanistic models can often be more computationally demanding since they are designed to capture the actual underlying mechanisms. In contrast, data-driven models are usually agnostic to the specific object of the research, and significant optimization has often been conducted to leverage these rather generic models for various applications. For example, artificial neural networks (ANNs) have gained significant attention because of their ability to model a wide range of data, including images, languages or videos. Along those lines, various open-source machine-learning (ML) and deep-learning (DL) libraries exist that allow the utilisation of a wide range of computer hardware [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Analogously, mechanistic simulators, such as in agent-based modelling (ABM) [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], are available to simulate skin biomechanics and biology. These methods often enable highly efficient simulations [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Notably, a given computational model can also be hybrid, leveraging advantages of both approaches for different model components.
</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Model-based vs. data-driven approaches</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Category</th>
<th align="left" colspan="1" rowspan="1">Model-based (MB)</th>
<th align="left" colspan="1" rowspan="1">Data-driven (DD)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1"><strong>Advantages</strong></td>
<td align="left" colspan="1" rowspan="1">-Provides explainable results based on established science</td>
<td align="left" colspan="1" rowspan="1">-Can discover patterns that may not have been identified before</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Useful for hypothesis testing and understanding mechanisms</td>
<td align="left" colspan="1" rowspan="1">-Can analyse large, complex datasets effectively</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Relatively data- and compute-efficient</td>
<td align="left" colspan="1" rowspan="1">-Adaptive to new data, enabling continual improvement</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Generally, more straightforward to validate and gain regulatory approval due to transparency</td>
<td align="left" colspan="1" rowspan="1">-Highly effective for predictive analytics and diagnostics</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Scalability allows more granular and accurate representation of complex systems</td>
<td align="left" colspan="1" rowspan="1">-Fast computations or/and low operational cost once model trained</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><strong>Disadvantages</strong></td>
<td align="left" colspan="1" rowspan="1">-Model accuracy: Based on assumptions that may reduce performance in benchmark comparisons</td>
<td align="left" colspan="1" rowspan="1">-Often lacks interpretability, rendering it a “black box” approach</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Less adaptable to new, unforeseen patterns of data</td>
<td align="left" colspan="1" rowspan="1">-Demand for large, high-quality datasets</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Limited by the accuracy of underlying theoretical models, verification is challenging</td>
<td align="left" colspan="1" rowspan="1">-Susceptible to biases present in the training data</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Often requires significant domain expertise and dedicated training of user</td>
<td align="left" colspan="1" rowspan="1">-High computational resource requirements, especially during model training</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1"><strong>Data types</strong></td>
<td align="left" colspan="1" rowspan="1">-Experimentally and specifically measured variables (e.g., concentrations, rates, biomechanical measures)</td>
<td align="left" colspan="1" rowspan="1">-Large and multi-dimensional (e.g., images, time series, omics, sensor data)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Structured, parameter-focused</td>
<td align="left" colspan="1" rowspan="1">-Often unstructured or semi-structured. Labelled or semi-labelled</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1"><strong>Collection instruments and data acquisition</strong></td>
<td align="left" colspan="1" rowspan="1">-Lab equipment, assays, physiological sensors</td>
<td align="left" colspan="1" rowspan="1">-Wearables, scanners, photo/video cameras</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-In vitro/in vivo experiments, controlled experiments, theoretical analysis</td>
<td align="left" colspan="1" rowspan="1">-Real-world environments, observational studies, clinical trials</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1"><strong>Analysis &amp; processing methods</strong></td>
<td align="left" colspan="1" rowspan="1">-Differential equations, system dynamics, rule-based models, agent-based modelling, finite element method, molecular dynamics simulation</td>
<td align="left" colspan="1" rowspan="1">-Machine learning, AI, statistical inference, data mining</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Solvers of ordinary/partial differential equations (ODE/PDE) solvers, simulation software</td>
<td align="left" colspan="1" rowspan="1">-Neural networks, clustering, regression, principal-component analysis (PCA)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">-Experimental validation, sensitivity analysis</td>
<td align="left" colspan="1" rowspan="1">-Cross-validation, prediction accuracy</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p22"><p>This table lists advantages, disadvantages and other aspects of these approaches. Please note that this comparison is not exhaustive</p></div></div></section><p id="Par19">When domain experts understand the capabilities and utilisation of computational modelling and simulations, it becomes possible to test different hypotheses. Multiple approaches are usually required for a certain complex system to be understood, given that one never has access to full information at all levels. Hence, there are often gaps in our understanding that render various hypotheses possible. Using computational modelling, one can implement and compare such hypotheses and assess their consequences. In this way, it becomes possible to generate predictions that could then be experimentally verified to identify the most suitable hypothesis.</p></section><section id="Sec3"><h2 class="pmc_sec_title">Computational modelling in systems biology</h2>
<p id="Par20">Biological systems are often relatively complex; thus, to gain insights into how they function, it is crucial to obtain experimental information from different data modalities, such as omics, imaging, or biomechanical data. The availability of powerful and precise technologies to extract such data has enabled the generation of large and detailed sets of information at different spatial and temporal levels. Given the associated complexities, it is necessary to employ advanced quantitative methodologies to capture and make sense of these data. In addition to mathematical and analytical methods, computational approaches have gained significant attraction in the biological and medical communities, boosted by the accelerating performance gains of modern computer hardware and software. Notably, many computational models address open questions in the context of highly complex systems and multidisciplinary settings and thus foster the exchange of ideas across traditional fields.</p>
<p id="Par21">From a conceptual perspective, computational modelling can be understood as a generic approach to simplify and formulate a model with predictive power. It is an iterative approach, as the level of detail and depth of the model can be refined in close interaction with experimental research. It is through this interaction that it becomes possible to make sense of highly complex datasets, which otherwise would surpass the abilities of the human mind. Figure <a href="#Fig3" class="usa-link">3</a> visualizes the computational approach in this iterative fashion.</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366154_13040_2025_471_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d724/12366154/3ecca5c05bd6/13040_2025_471_Fig3_HTML.jpg" loading="lazy" id="MO3" height="272" width="709" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The approach of computational modelling in computational biology research (adapted from [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]). A given biological system, such as the skin, is modelled on the basis of an initial set of model parameters. Conditional upon satisfactory validation, novel and experimentally verifiable predictions can be generated and compared with new experimental data. If successful, a more detailed and comprehensive computational model can be devised in an iterative manner</p></figcaption></figure><p id="Par22">The starting point of the circular approach is the selection of the system to be modelled. For instance, this type of system could be a cancerous skin sample. The next step is to create a computer model of the system, which requires certain assumptions and simplifications. The model is subsequently evaluated on a validation dataset that must be new— i.e., no information from the validation dataset can be used in the creation of the model. Depending on the validation results (such as accuracy of cancer diagnosis), the model must be revised and improved. This can be achieved by amending the initial assumptions and increasing the level of detail of the model. If the validation is satisfactory, novel hypotheses should be generated. Experimental verification will then provide information on whether the model needs revision or whether the system can be studied at greater depth.</p>
<p id="Par23">Computational modelling is a generic methodology that aims to produce an idealized representation (such as using a mechanistic or data-driven approach, see <a href="#Sec2" class="usa-link">Data-driven vs. model-based computational models</a> section) of a complex system via continuous improvement through tight interactions with the experimental work of the system under consideration. Skin research can help address various research questions, such as those concerning skin cancer diagnosis. Notably, computer models often also generate experimentally verifiable predictions that allow assessment of the accuracy and realism of the model. Moreover, they can reduce the need for experimental research (including the use of animals and other ethically challenging methods) [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Despite the impressive capabilities and insights that computational methods have delivered in biomedicine, challenges in bridging gaps between wet-lab research and computational modelling remain present. Along those lines, computer scientists may lack expertise in biological and medical domains. On the other hand, it is often challenging for researchers without a strong computer science background to make use of custom research software. We address this gap here by providing a general overview of work in dermatology.</p>
<p id="Par24">Given the extensive availability of software to employ computational modelling for a wide range of tasks, it is not surprising that a plethora of applications in systems biology have been addressed. Because of the central role that the skin plays in human physiology and well-being, extensive computational research has focused on understanding its role in health and disease. While it is beyond the scope of this review to describe this research comprehensively, we categorize and provide an overview of the most prevalent categories in the following sections.</p></section><section id="Sec4"><h2 class="pmc_sec_title">Computational models in skin research and applications</h2>
<section id="Sec5"><h3 class="pmc_sec_title">Characterization of skin biophysics</h3>
<p id="Par25">One of the main purposes of computational modelling is to gain insights into the workings of complex systems. Computer models can bringing together existing knowledge and establish basic dynamics of the underlying processes. Moreover, they can be used to test and compare different hypotheses. In clinical investigations, diagnosis, and numerical modelling, understanding the mechanical characteristics of soft tissues and organs is crucial, which highlights the importance of exploring these attributes in the context of skin. In dermatology, significant fundamental research has been conducted via computational modelling. The review by Limbert [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>] summarizes a number of seminal works employing a spectrum of different computational and mathematical techniques to model skin biophysics. This work presents various model-based approaches that consider detailed biophysical laws. However, current developments in AI techniques will likely play an increasingly important role in fundamental skin models. Limbert’s review investigates such data-driven methods that help generate predictions that can then be used to optimize therapeutic interventions. However, as in many other biomedical fields, data availability is a significant bottleneck, so interdisciplinary initiatives are needed to inform and improve relevant computational models. Mechanical qualities vary from person to person, and they change with time and under various circumstances. Gaining knowledge of these material properties adds to what can be included in computer simulations and organ prediction diagnostics. Characterizing the mechanical behaviour of skin and understanding the underlying mechanisms of deformation at different spatial scales is essential in a broad spectrum of applications, such as surgery, cosmetics, forensics, biomimetics, and engineering of protective gear or artificial grafts [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Therefore, accurate methods that can measure the mechanical characteristics of skin tissue are needed.</p>
<p id="Par26">In Dwivedi et al. [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], an innovative approach was proposed to determine the mass density and elasticity of soft tissues by analysing their mechanical vibration response when stimulated externally by sound waves or a mechanical indenter. Soft sheets are subjected to a vibration test to gauge how they react to a continuous range of stimulation frequencies. Using 3D digital image correlation (DIC) and two high-speed cameras, frequency responses are gathered. To analytically simulate the free vibrations of continuous layered human skin and neoprene artificial sheet samples, an effective theoretical model was constructed. Finite element (FE) modelling, as well as comparisons with DIC measurements and frequency tests, demonstrated the excellent validity and correctness of the analytical simulations. This analytical approach was converted into a numerical program and was shown to reproduce the experimental measurements in the simulation. This study effectively demonstrates how experimental and in silico methods can be combined to generate highly realistic skin biophysics models. Nevertheless, how different skin properties (e.g., across age and skin types) can be measured in a non-invasive manner and modelled remains to be investigated.</p>
<p id="Par27">Changes in the mechanical properties of the skin can be indicators of disease, highlighting the potential that computational models of skin biomechanics have in global health. A significant example is how elevated blood glucose levels associated with type 2 diabetes mellitus (T2DM) alters the skin’s mechanical characteristics. Bermudez et al. [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] compared the biomechanics of maximum stress load capabilities and elastic modulus to be significantly lower in murine skin affected by diabetes mellitus than in healthy individuals. Dwivedi et al. [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] also observed higher levels of anisotropy, meaning higher degrees of parallelism between collagen fibres, as well as general collagen loss in the skin of mice with T2DM, contributing to the observation in a reduction of skin stiffness.</p>
<p id="Par28">Understanding the unique mechanical properties of skin is not only important in their response to health and disease, but also for engineering tools and technology that has the intention to work with these properties. Printable or flexible sensors are a field of medical innovation with groundbreaking potential, specifically in the realm of skin biophysics, where monitoring devices can be adapted to the unique, biological terrain by the specific design. Implementing medical technology into an adaptable medium that can be worn or integrated onto a physical body allows for easier and more compatible health monitoring.</p>
<p id="Par29">Zhu et al. [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>] created a real-time toolpath adaptation system for in situ 3D printing by estimating the motion and deformation of the target surface. A hydrogel-based sensor was printed via this printing technology on a porcine lung that had been deformed by breathing. Through electrical impedance tomography, a non-invasive method of measuring cardiac output and pulmonary artery pressure via a radiation-free medical imaging approach [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>], the sensor continuously offers spatial mapping of deformation while adhering to the tissue surface. By building a training dataset of point clouds with pointwise correspondence among data samples on the basis of 3D scans, they were able to determine the linear shape basis model of the surface deformation via a machine learning method. Owing to its additive manufacturing capabilities, adaptive 3D printing technology of this type may improve robot-assisted medical treatments by enabling autonomous and direct printing of biological materials and wearable electronics on and within the human body.</p>
<p id="Par30">Similar methods can be used in the context of 3D printing these biosensors directly onto the skin for various monitoring approaches. Internal temperature monitoring is fundamental in a range of health and disease, whether they identify localized temperature shifts that may indicate issues with inflammation or circulation, or more generalised shifts associated with thermoregulation or infection. Current work is focused on incorporating conductive polymers into printable mediums. Wei et al. [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>] implemented these polymers into biocompatible and mechanically robust microneedles that can penetrate into the skin with a depth, diameter and angle optimised in their studies. Hwang et al. [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>] created a skin-conformal temperature sensor array less than 2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e645"><mi>μ</mi></math></span>m thick, allowing for comfort and mechanically appropriate deformation of the arrays when attached to skin.</p>
<p id="Par31">Strong knowledge in skin biophysics is crucial for understanding the modalities of drugs and other chemical penetration into the tissue, which influences their toxicity and treatment efficacy. In this context, computational modelling can help increase the cost-effectiveness of drug delivery or assess the risk of various exposure types. Drug information is frequently found within clinical narratives, and drug regimens are subject to frequent changes due to factors such as intolerance or insurance-related issues, which complicates accurate modelling. In a research study, scientists created an informatics framework that uses natural language processing and ML techniques to determine patient drug exposure histories from electronic medical records [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. In line with this, several studies have employed computational modelling to study the uptake of various molecules in the skin.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The diseased and damaged skin</h3>
<p id="Par32">Skin pathology is unique in that it is directly observable via visual assessment. However, current diagnostic procedures still require assessment by formal medical personnel. This requirement for direct assessment by trained medical professionals is a common barrier in medicine in terms of accessibility, cost and resource availability, which is particularly problematic in remote or poorer regions of the world. Given the ease of observing the skin, skin diseases are ideal candidates for incorporating machine learning tools into modern medical diagnostics to improve accessibility and reduce the workload on the current medical workforce.</p>
<p id="Par33">There has been a recent surge of interest in the use of images of healthy and diseased skin under different conditions to train AI models for diagnostic potential. As often observed when training different models, a significant bottleneck for this approach is the availability of training data. Pattern recognition algorithms require vast amounts of data demonstrating images that are classified as healthy or diseased. Owing to the potential demand, there are few complete datasets that possess the extent of information required to train a model for skin disease and damage classification.</p>
<p id="Par34">A database for 1,000 research participants, 200 healthy volunteers and 800 dermatology patients with facial skin concerns was constructed for the study [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. To categorize patients according to their skin conditions, an expert method has been devised (majority class). Here, an ANN that had been pretrained was chosen to construct the expert system. Since the dataset was not balanced, i.e., the number of samples varied significantly across skin conditions, a modified learning algorithm was employed. This algorithm treats the minority class as noise or interference during the training phase. Although the ANN achieved a high degree of accuracy, more assessment parameters suggest that it could be enhanced to more accurately predict the results. More sensitive and accurate findings may be obtained by increasing the number of factors and the number of patients or healthy volunteers, which will ultimately result in an effective classifier [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Notably, the number of research participants was significant and could be further expanded in number to improve the expert method in future studies. Overall, his study highlights the potential of using ANNs for effective skin diagnostics.</p>
<p id="Par35">CNN algorithms can be used to extract higher- and higher-level representations of image content and are thus used for image classification; in dermatology, they extend towards the classification of skin lesions and the identification of whole skin types. Photographs of normal, oily, and dry skin were included in the collection; there were 112 photographs of normal skin, 120 images of oily skin, and 97 images of dry skin. A total of 1,316 pictures were obtained after the Contrast Limited Adaptive Histogram Equalisation (CLAHE) approach was used to improve image quality, and data augmentation by rotation was used to increase dataset variation. The accuracy of the model was 89.70% when tested on unknown data and verified via cross-validations [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>].</p>
<p id="Par36">An excellent study by Shah et al. [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] explored the role of machine learning techniques, such as ANNs and convolutional neural networks (CNNs), in identifying skin cancer. These methods are inspired by biological neural networks, where information is processed between a point of input and output. In this context, computational neural networks can take in the information provided through images of the skin, process and recognize patterns about this information, and provide an output as a classification of the condition of the skin. Shah et al. explored currently available ANN methods, where the network is made up of multiple layers of nodes, allowing different weights of importance assigned to each input. The added layers constitute deep learning, and combined with different weights of information being relayed at each layer, they provide a network that processes the images and identifies patterns in the texture and boundaries of the provided dataset of skin samples.</p>
<p id="Par37">Some ANN techniques incorporate additional information, such as colour, symmetry, patient background and history as parameters. CNNs expand on this concept while distributing their weights between nodes evenly to allow for a reduction in parameters and a lower computational cost. This allows for higher efficiency in the image processing steps, such as hair and colour removal, and improved extraction of topological features that either deviate from healthy skin. The efficacy and performance of ANNs and CNNs in the early and efficient diagnosis of skin cancer are determined by evaluating studies on the detection of the disease. The findings of this study, which showed the potential for these technologies to increase accuracy in skin cancer diagnosis, showed that ANNs and CNNs were effective in early skin cancer identification when various datasets and hybrid models were used, as well as the ability of CNNs to discriminate between benign and malignant skin malignancies. This novel application of deep learning techniques for skin cancer detection may be key in saving time and effort in future diagnostic processes and highlights the vital need for automated systems for skin lesion recognition [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>], with automation of skin cancer diagnosis procedures and the application of CNNs yielding more accurate categorization [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>].</p>
<p id="Par38">The identification of pathological skin markers via dermoscopy is laborious, especially in healthcare settings with limited resources [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Recent developments in computer vision [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] have helped overcome these limitations through the development of numerous smartphone-based apps or smartphone images [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] for early skin disease prediction. In a recent study, they processed over 45,000 high-quality photos from the HAM10000 dataset, which covers seven types of skin disorders, via a collection of sixteen unique DL-based CNN models. With the use of lesion photos, their algorithms were able to accurately diagnose and categorize skin conditions.</p>
<p id="Par39">As it has been demonstrated by research on grokking (a delayed generalization occurring long after achieving near zero training error) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>], CNNs always benefit from larger dataset, as long as the computational infrastructure supports an arbitrarily deep architecture. It is, however, common in biomedical sciences to operate in settings where data resources are fragmented and access is limited. Thus, a valuable question to answer concerns what the minimum dataset size would be to effectively train a CNN. The answer is not straightforward, as the convergence of the testing and training errors is heavily affected by the task and the data distributions, and it can differ based on the network architecture (e.g. WideResNet [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>], Xception [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>], Pyramidal [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>], etc.) and trade-offs based on the computational infrastructure. There are techniques to enhance learning performance when faced with data scarcity. When computational infrastructure is not the limiting factor, the most promising one is transfer learning [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>], a technique where data extraneous to the original task but similar in nature are introduced in early learning stages to learn general features, which are in later training steps fine-tuned on the smaller request dataset. Alternatively, data augmentation techniques are available to improve attributes and size of training datasets [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>].</p>
<p id="Par40">Classification of lesions becomes challenging when physical representations of different pathologes manifest extremely similarly. For example, there are several similarities between melanoma and melanocytic nevi, which poses significant challenges for differentiation between the two [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Malibari et al. [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] demonstrated the effectiveness of the Improved Whale Optimization Algorithm (IWOA) with deep neural network (DNN) models in skin cancer detection and classification. The IWOA is used in this process to choose the DNN settings effectively. With a maximum accuracy of 99.90%, the comparative analysis findings demonstrated the excellent performance of the proposed optimal deep neural network (ODNN) single-bond CADSCC (Computer Aided Diagnosis model for Skin Cancer Detection and Classification) model over other previous techniques. They report that the ODNN–CADSCC model includes a number of processes, including DNN classification, U-Net segmentation, SqueezeNet-based feature extraction, Wiener filtering (WF)-based preprocessing, and IWOA-based parameter optimization. The WF-based preprocessing technique is the main method used by the ODNN–CADSCC model to eliminate noise from the dataset. In addition, the U-Net segmentation method is used to identify questionable lesions in dermoscopic images of skin cancer samples. The SqueezeNet model is utilized to produce an array of feature vectors. Finally, an IWOA model combined with a DNN model is used for picture classification.</p>
<p id="Par41">The absence of transparent decision-making in AI-based computer-aided diagnosis (CAD) systems is a major obstacle to their successful integration into routine clinical procedures. While widely used eXplainable AI (XAI) techniques offer insights into these essentially opaque algorithms, these explanations are typically complex and difficult to understand. Given the inherent ambiguity of the description of the underlying medical condition, choices about the malignancy of skin lesions on the basis of dermoscopic images require further clarity in their explanation. To support these predictions, different studies have developed AI frameworks for biomedical image analysis that offer multimodal concept-based explanations in the form of simple-to-read verbal explanations and visual maps [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>].</p>
<p id="Par42">Kulkarni [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>] presented a pathology-based computational method using a DNN architecture for disease-specific survival prediction. In contrast to existing approaches that use genetic markers, this novel method uses digital pictures, which eliminates the requirements for sample transportation and modification, and thus is an affordable and accessible advancement.</p>
<p id="Par43">Another characterisation study investigated the effects of arsenic on skin [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Arsenic is a highly poisonous molecule present in the environment (air pollution), with long-term exposure to low quantities through food and water resulting in skin sores and cancer. This encouraged researchers to use AI to establish a remedy and comprehend the effects of arsenic through the study of skin photos, with half depicting individuals with symptoms of intoxication and the other half showing mixed images of skin unaffected by arsenic. The ultimate goal of this approach was to identify symptoms of arsenic intoxication faster and treat patients more accurately. Moreover, scientists, physicians, and other experts may be interested in this dataset, such as machine learning researchers with an interest in training AI models to distinguish between conditions resembling arsenic contamination from other diseases.</p>
<p id="Par44">The development of characterization methods that identify affected against healthy image examples for a specific disease demonstrates a significant milestone. With some pathologically unique conditions manifesting similarly to the observer, development of models that can distinguish not only health from disease, but different diseased skin conditions from each other is the next significant step for global health initiatives in the context of skin. Examples of such categorization of pathological skin markers are further explored in this review.</p>
<p id="Par45">This dataset provided by Emu et al. [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], demonstrates an example dataset that is well suited for the training of AI models that identify skin lesions induced by arsenic exposure. Work has already been done to extend this to other skin conditions, such as skin cancer, which is one of the greatest global healthcare burdens. Skin cancer, including both nonmelanoma skin cancer (NMSC) and malignant melanoma, is a frequent malignancy in populations with European ancestry and is becoming more prevalent [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. According to the US Skin Cancer Foundation, more Americans are affected by skin cancer annually than by combined diagnoses of all other types of cancer [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Given the high risk of metastasis in skin cancer patients, early detection and affordable and accessible care are considered serious health threats.</p>
<p id="Par46">Another classification study was carried out on the basis of photos of burned skin by Yıldız et al. [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. They aimed to find an autonomous classifier that can exploit an artificial intelligence-based mobile application to assess burn severity from real-time images of skin burns. To this end, the application segments the burn image region and classifies the burns as first-, second-, or third-degree burns. A smartphone application named “Burn Wound Detection” was created to identify burn photos for the first time in Türkiye by this group [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>].</p>
<p id="Par47">According to the still scarce published evidence, models improve specialist care accuracy when paired with human doctors [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] albeit with a caveat concerning equality of care, due to an exacerbation of racial bias. Indeed that human-machine interaction should be carefully thought and designed in the delivery of augmented care is reinforced by several experiences [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Available evidences are still largely indecisive, as the same models can produce very positive results in test settings they have been explicitly designed for, whilst resulting significantly inferior in settings that have more general designs [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>], although a thought should be spared for the very limited size of the experience. It is worth stressing how benchmarking in healthcare is not as straightforward as in other engineering settings, due to the messy nature of human interactions and trajectories bringing to an episode of care, and even demonstrating the superiority of specialist vs general practitioners performances has proven tricky [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. More research should thus focus on fair and meaningful performance testing in healthcare, engaging with the medical and patient communities to better understand how bias and randomness emerge in the episode of care.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Skin wound healing</h3>
<p id="Par48">An important aspect when defining the elements of biological phenomena that a mathematical model should encompass pertains to the primary guiding principles that the model should be as complex as necessary on the basis of (i) the research question and hypothesis to interrogate and (ii) the quantity and resolution of experimental data available to study skin mechano-biology. Furthermore, choosing the appropriate model for skin wound healing should consider the type of injury, the type of wound and the recovery phase. Although existing models of wound healing, skin tissue remodelling, and angiogenesis in the dermis rely on certain simplifying assumptions, the specific structure of the resulting model can differ significantly. There have been several excellent reviews on mathematical and computational models dedicated to wound healing [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. However, a concise overview of the in silico modelling works for skin wound healing that have been developed over the last 15 years is presented below.</p>
<p id="Par49">The pioneering paper of Flynn [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>] presented a unique FE model of wound closure for simulating the incision, excision and closure of skin flaps. They modelled different excision shapes (i.e., circular, elliptical, fusiform and lazy S-plasty cuts that are relevant to skin plastic surgery) to numerically predict the surrounding stress field, the closure force along the sutured edge and the size of any extrusions. Their work demonstrated the potential of mechanistic FE models such that they can be explored to determine the optimum excision shape that minimizes the risk of skin scarring after skin closure and suturing.</p>
<p id="Par50">Similarly, Capek et al. [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>] developed a plane-stress model to evaluate the forces required for suturing selected skin wounds (specifically elliptical shapes). They performed a numerical analysis via FEs and in vivo experiments to estimate the forces induced during epidermal wound closure of elliptical wounds of varying sizes, both with and without a skin pre-stress effect. The major contribution of their work was that the amount of force required to close a wound firmly was strongly correlated with the size of the elliptical wound and skin pre-stress. Extending from the aforementioned 2D models, Chanda and Unnikrishnan [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>] developed a 3D two-layered computational model of the skin (epidermis and dermis) that considered a diamond-shaped wound with a varying cross-section while also placing numerical sutures to simulate wound closure effectively. In contrast to the viscoelastic model of Capek [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], Chanda and Unnikrishnan modelled the skin as a hyperelastic material and explicitly incorporated skin prestressing. Their contribution was primarily in numerically estimating the force required for each suture to close a skin wound, which was estimated at approximately 5 Newtons with a maximum value at the centrE of the wound, where the suture was observed to require an approximately fourfold greater pull force than the first end suture. The work of Chanda and Unnikrishnan is promising in that, given the geometrical features of a wound (depth, opening), the biomechanical properties of the skin (elasticity and pre-stress) and the suture properties (wire material and cross-sectional area), a computational model can be used to estimate the force requirements of a wound suture.</p>
<p id="Par51">Valero et al. [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>] presented a continuum-based FE model of epidermal wound healing that included major biological (blood vessels and fibroblasts), biochemical (nutrients and growth factors) and biomechanical (cell traction forces and the extracellular matrix) factors that are pertinent to wound healing. The major novelty of this contribution, compared with previous works from the same group of investigators, concerns the coupling of healing-induced angiogenesis and contraction of the wound and nonlinear effects (large deformations) using an updated Lagrangian FE methodology. A key finding of Valero et al. was that an elliptical wound vascularizes two days earlier than a circular wound, despite both experiencing similar contraction levels.</p>
<p id="Par52">Agent-based modelling (ABM) approaches have been a significant emerging research angle for the investigation of modelling wound-healing. McDougall et al. [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>] presented a dermal wound healing ABM simulation method that included key extracellular materials (i.e., collagen and fibrin density and alignment), with the fibroblasts considered discrete units. They investigated in silico how manipulation of wound healing-related growth factors can modulate scar tissue formation and explored the effects of tissue remodelling on collagen alignment. In contrast to the established differential equation-based models of inflammation and wound healing, Ziraldo et al. [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>] developed a computational modelling platform using ABM representation to facilitate the construction of tissue-realistic models appropriate for wound healing. Despite not being directly linked to skin healing, their hybrid model was designed to simulate pressure ulcer formation in spinal cord-injured and uninjured patients to identify control points that reduce stress caused by tissue ischaemia or reperfusion. A few years later, Boon et al. [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>] proposed a cell-based ABM model to simulate the dynamics of tissue contraction in dermal wounds; this model was more advanced since it accounts for the ingress of leukocytes and the regeneration of collagen. Their cell-based model was capable of replicating the biological processes that take place during wound contracture while also combining with the role of the immune system during healing. The key takeaway of Boon’s work was that the amount of final contraction was positively correlated with the migration velocity of leukocytes, suggesting that the immune system functions well to prevent contractures.</p>
<p id="Par53">In the same year, Vermolen and Gefen [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>] proposed a phenomenological 2D model to simulate the mechanical forces exerted by myofibroblasts onto the extracellular matrix, as is evident in tissue remodelling and the contractures of burns. Their model was capable of recapitulating tissue contracture, i.e., the permanent contraction of tissue after wound healing, and was used to probe the interplay between chemokines and fibroblast-experienced strains and stresses. Later, Vermolen [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>] formulated a Bayesian framework for estimating correlations and likelihoods from wound healing simulations to interrogate the mechanisms of tissue contraction. Their interesting modelling approach assesses the uncertainty of a hybrid in silico model. It combines a cell-based model with a continuum-based FE model for wound contraction and explores the correlations between the final contraction of a wound and key parameters such as tissue elasticity, force exerted by fibroblasts, death rate and differentiation rate of the cells involved in wound healing and angiogenesis.</p>
<p id="Par54">Singh and Chanda [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] proposed a personalized model for clinical wound healing via an FE procedure to numerically estimate the biomechanical stresses and strains generated due to the natural tension of the skin during progressive wound healing. Motivated by the development of a simulation tool for pre-surgical planning and robotic surgery for performing wound closure operations, they simulated different stages of clotting. They investigated the effect of the biaxial natural tension of the skin on wound closure by studying a range of diabetic, venous, pressure, and ulcerative wounds. Singh and Chanda also studied the magnitude of biomechanical forces developing in skin tissue during the different phases of healing in human skin. Later, Bai and Zeng [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>] developed a 3D geometric model of an epithelial monolayer sheet that considers the biomechanics of epithelial cells and the intercellular interactions between neighbouring cells at cell–cell junctions and the cell–substrate adhesions between epidermal cells and the hypodermis (substrate). The key findings of their modelling work are that the efficacy with which a skin wound closes is related to the initial gap size, as with the early work of Capek et al. [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], whereas enhancing the lamellipodial protrusion of epithelial cells was found to accelerate wound closure. Additionally, Bai and Zeng [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>] reported that biomechanical stresses are concentrated at the wound edge of the epithelial monolayer, whereas the normal cellular stress dominates the shear stress at the cell–cell interface.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Skin permeation</h3>
<p id="Par55">Understanding how substances permeate the skin is critical in fields such as pharmacology, dermatology, and materials science. In particular, it can help develop safe and effective topical drug delivery systems and skincare products [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. By computationally simulating the complex interactions that govern the movement of substances through the skin, researchers can study and predict permeation rates, optimize drug delivery systems, and evaluate the safety of cosmetic and pharmaceutical formulations.</p>
<p id="Par56">One widely used approach in skin permeation modelling is the use of mathematical frameworks based on Fick’s laws of diffusion. These models assume that permeation occurs as a diffusion-driven process and often represent the skin as a series of distinct layers, each with specific permeability coefficients. While such models are computationally efficient, they are limited by their reliance on simplifying assumptions that may not fully account for the heterogeneous and anisotropic nature of the skin.</p>
<p id="Par57">More advanced models, such as molecular dynamics (MD) simulations and machine learning-based approaches, can overcome these limitations. MD simulations provide a detailed, atomistic perspective on how permeants interact with skin lipids and proteins, enabling the exploration of mechanisms at the nanoscale. On the other hand, machine learning models leverage large datasets from experimental permeation studies to identify patterns and make predictions. These data-driven models are particularly useful in situations where traditional mechanistic models fail or where empirical data are scarce. Notably, combining different approaches to leverage their individual advantages is possible. Zhang et al. [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>] presented a computational workflow that integrates MD modelling, statistical methods named quantitative structure–property relations (QSPRs), and quantum chemistry calculations. They demonstrate good accordance with multiple in vitro experimental datasets and provide open-source software to implement their approach. Other recent studies focused more on the data-driven side of computational modelling and demonstrated high accuracies with respect to skin permeability prediction [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Overall, the field has reached a state where permeability predictions are often accurate, and their usage for clinical applications needs to be corroborated. Close collaboration with the pharmaceutical industry has the potential to play a seminal role in this context.</p>
<p id="Par58">Notably, challenges similar to those faced by other computational skin models remain. These include accounting for inter-individual variability in skin properties such as age, hydration levels, and disease states. Additionally, the dynamic nature of the skin, including processes such as metabolism and active transport, makes it difficult to create comprehensive and universally applicable models. Moreover, modelling and predicting the permeation of product formulations comprising mixtures of different substances is problematic, as these mixtures may entail complex interactions across ingredients.</p>
<p id="Par59">Despite these challenges, advancements in computational power, data availability, and algorithmic sophistication continue to increase the accuracy and utility of skin permeation models. As these tools evolve, they hold the promise of accelerating the development of safer, more effective trans-dermal therapies and improving the understanding of the role of the skin as a barrier and interface with the external environment.</p></section><section id="Sec9"><h3 class="pmc_sec_title">The ageing skin</h3>
<p id="Par60">Research into skin ageing has gained significant traction in recent years. The skin is an organ where ageing is easily visible [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. A person’s face may provide considerable non-verbal information, including gender, age, emotion, etc. [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. Such information is also detectable in pictures of human faces. Numerous methods have been created by researchers to extract this type of information from facial photos for use in a variety of computer vision applications. In addition, the effects of genetic and environmental factors can be evaluated with various computational techniques to provide further understanding of skin ageing processes.</p>
<p id="Par61">In a recent study by Want et al. [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>], researchers conducted a thorough examination of the transcriptional regulatory landscape of canine skin ageing. Weighted gene co-expression network analysis (WGCNA) is a useful bioinformatic tool used to identify correlations and magnitudes of these correlations between gene expression levels to identify patterns in which genes are expressed together. This study identifies gene interactions in mechanistic pathways and gene modules associated with ageing. Alterations in the expression of these co-expressed genes were verified in aged human skin via single-cell RNA sequencing (scRNA-seq) analysis. Notably, the cell types with the greatest differential gene expression throughout ageing are fibroblasts, mitotic cells, spinous cells, and basal cells.</p>
<p id="Par62">Combining techniques of computational gene regulatory network (GRN) construction with WGCNA, Wang et al. [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>] identified target genes by examining the frequency of expression of transcription factor (TF) binding motifs. By comparing highly enriched TFs inside the computational GRNs to genes with high degrees of connectivity via WGCNA, the genes CTCF and RAD21 were identified as key regulators in ageing skin, which were further correlated with shifts in gene expression patterns associated with age-dependent blood parameters. An approach to categorize human facial photos into four age groups via ANNs and facial skin ageing characteristics was proposed by Jagtap et al. [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. Wrinkle analysis and the Local Gabor Binary Pattern histogram (LGBPH) were used to extract the ageing aspects of facial skin. LGBPH works on the basis of applying local binary patterns, which is a method of describing image texture on the basis of spatial patterns of pixels, onto Gabor filters, which analyse the direction frequency for texture analysis. The results from LGBPH were then used to identify wrinkles and calculate wrinkle density for wrinkle analysis. Varying ratios of the LGBPH results were then used to train feed-forward neural networks back propagation, which are ANNs where the information is processed in one direction but sends feedback about error margins to adjust the weights for the next iteration, improving the way the algorithm learns to the characteristics of agEing. Using facial images from the PAL facial database, the suggested age classification framework was evaluated and trained. The results demonstrate a significant improvement in age classification accuracy, reaching 94.17% and 93.75% for males and females, respectively [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. The mechanical characteristics were also related to wrinkles with ageing.</p>
<p id="Par63">The potential of physics-based skin modelling for combating skin ageing, diseases and trauma has been previously reviewed [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. This review highlights that the physical mechanisms underlying experimental observations can be elucidated by analytical models of skin wrinkles, but these models soon become overly restrictive when investigating naturally complex biological structures, non-uniform loading conditions, or more complex non-linear materials. Selected distinguished modelling works about the skin are outlined in Table <a href="#Tab2" class="usa-link">2</a>.
</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Computational models related to skin wrinkles</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Method or computational model</th>
<th align="left" colspan="1" rowspan="1">Target and result of modelling</th>
<th align="left" colspan="1" rowspan="1">Ref</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">3D agent-based model of stable epidermal structure with well-organized layers</td>
<td align="left" colspan="1" rowspan="1">Stability of layered structure depends on cell supply rate from the basal layer. Also explores dermal stiffness effects, barrier function, and corn (clavus) formation</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Experimentally based computational model of compression-induced macroscopic skin wrinkles</td>
<td align="left" colspan="1" rowspan="1">Wrinkle depth increases significantly under 20–30% strain; number of layers and prestress influence wrinkle formation</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Experimental apparatus to analyse morphology of abrupt wrinkle shift</td>
<td align="left" colspan="1" rowspan="1">Around age 33, skin wrinkling pace increases rapidly, shifting appearance from stratum corneum to epidermal wrinkles</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Finite-element analyses on microrelief and etched wrinkles</td>
<td align="left" colspan="1" rowspan="1">Fine microrelief helps prevent deep creases (deeper than 0.35 mm)</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Finite strain parametric finite element model of skin</td>
<td align="left" colspan="1" rowspan="1">Microrelief is the dominant factor in wrinkle characteristics with modest elastic modulus ratios</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3D finite element models with 2–6 anatomically motivated layers</td>
<td align="left" colspan="1" rowspan="1">Simulates wrinkle formations and evaluates age-related effects on dynamic surface wrinkles</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Wrinkle detection using skeleton lines and novel loss function</td>
<td align="left" colspan="1" rowspan="1">Improves subtle wrinkle detection; shares a dataset of 1021 Chinese face images to support research</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Machine learning implementation of the Baumann Skin Type Indicator (BSTI) using InceptionV3</td>
<td align="left" colspan="1" rowspan="1">Achieves 84% classification accuracy for skin types (e.g., wrinkle/tight, pigmented, dry/oily, sensitive/resistant)</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Wrinkle Oriented Active Appearance Model (WOAAM)</td>
<td align="left" colspan="1" rowspan="1">Encodes wrinkle form/texture into interpretable vectors; uses KDE to generate age-representative wrinkle patterns</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Finite element method for wound geometry simulation</td>
<td align="left" colspan="1" rowspan="1">Simulates wound healing stress/strain in various wound types to support surgical and robotic planning</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Biofidelic skin phantom with DIC testing and FE modeling</td>
<td align="left" colspan="1" rowspan="1">Estimates mechanical response of elliptical wounds, guiding safe closure techniques in surgery/robotics</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par64">In particular, wrinkled skin sits at the crossroad of interest between dermatopathology, healthy ageing/longevity research, and cosmetics, their appeal attested by the variety of novel technologies available on the market with the promise of delay or even reversal of the wrinkling process [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, most research in the field is performed on animal models, realistic in vitro models being scarcely available and under active development [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] as a consequence of the complexity of the underlying biology and physics. When developing computational models of wrinkling skin, it must be taken into account that the number of structural layers can vary from 2 to 6 depending on the district of interest, the biophysics of forces expressed by skin muscles (and locomotor mechanics when relevant), the dermal-epidermal junction interface; the thickness of the viable epidermis, the reticular dermis, and the papillary dermis [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Skin microbiota</h3>
<p id="Par65">The topic of skin microbiota is important in skin research and dermatology. The interactions between microbial organisms and the skin play an important role in human health and disease. Although relevant computational modelling work is still in its infancy, insightful studies that investigate skin microbial communities using computational modelling exist. Along those lines, the recent work of Lee et al. [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>] reviews in silico skin studies and provides a comprehensive picture of the field’s state of the art. Notably, although only a relatively small number of studies is described, these cover both mechanistic approaches [<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>] as well as data-driven models [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>, <a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]. A more substantial literature exists for computational models of gut microbiota, some of which is also reviewed by Lee et al..</p>
<p id="Par66">The study of Jaiswal et al. [<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>] provides a data-driven, computational framework called SkinBug that incorporates a database to model and predict metabolic activity of the skin microbiome. Their tool yields intriguing information on various aspects of metabolic variability across skin sites, highlighting the need for site-specific predictions of skin microbiomes. Moreover, the capability to predict the impact of skincare products and medical ointments on skin metabolism is very valuable for biomedical applications and understanding of toxicity.</p>
<p id="Par67">We highlight that a wide range of computational methods have been employed to model skin microbiome data and gain insightful results. For instance, the study by Wei et al. [<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>] employed network-based modelling to show that there are more microbial species in moist skin sites than in sebaceous skin sites. Moreover, they show that skin microbiota are relatively well-conserved over time. Overall, their results suggest that well-established network analysis methods are effective for researching the complexity and stability of the human skin microbial community.</p>
<p id="Par68">Given that the skin is, after the gut, the organ with the largest number and diversity of bacteria in the human body, computer models are very well-suited to address the entailed complexity. However, surprisingly little computational work has been conducted in terms of skin microbiota research. In particular, very few relevant mechanistic models exist. More interdisciplinary collaboration across domain experts and more data sharing practices will help bridge current gaps, and facilitate research that can shine light on the workings of the skin microbiome. In future, it is conceivable that more advanced models of skin microbiota will inform treatment strategies and yield personalised medicine, reflecting similar trends as in the context of the gut microbiome [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>].</p></section><section id="Sec11"><h3 class="pmc_sec_title">In silico approaches for virtual clinical trials and digital twins in dermatology</h3>
<p id="Par69">Evidence accumulation and systematization are key bottlenecks in timely market governance, especially when dealing with highly regulated domains, such as healthcare. Thus, it comes as no surprise that players in the biomedical field would regard the acceleration and augmentation of research and innovation practices by digital means very keenly, with the first speculations on the potential impact of computer technologies in medical practice dating back to the 1960 s [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>] and studies of the feasibility of internet trials dating to the early 2000 s [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. Progress has been steady over the period, albeit not glamorous, with one of the first demonstrations of the feasibility of applying ML to medicine, the CASNET model (a consultation program for glaucoma) already in the 1970 s [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] and the Physiome (“physio” = life + “ome” = as a whole) project first presented in a report from the Commission on Bioengineering in Physiology to the International Union of Physiological Sciences (IUPS) Council at the 32nd World Congress in Glasgow in 1993. However, it is only with the growing availability of computing resources and new models of procurement thereof that the field experienced a true coming of age at the turn of the millennium.</p>
<p id="Par70">Clinical trials are estimated to be at a value of USD 80.7 billion in 2023, with a growth projection at a compound annual growth rate (CAGR) of 6.49% from 2024 to 2030, up to USD 123.5 billion by then, with a current revenue share of 50.3% for the North American market [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. In the case of dermatology, the market valuation for cosmetics should also be considered: the cosmetics market was valued at USD 617.20 billion in 2023 and growing at a CAGR of 8.81% from 2024–2033 [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>].</p>
<p id="Par71">This highlights the impact that randomized controlled trials (RTCs) in applications of dermatology and skin research have on the global scale, both in healthcare and socioeconomic implications. Thus, work into improving the pipeline of concept for application is imperative.</p>
<p id="Par72">RTC augmentation practices can aid in reducing uncertainty and/or improving the generalizability of RTCs either by increasing the degree of uncertainty or by better defining the potential outcomes ahead of the study to improve the confidence of analyses and interpretations. Although commonly adopted, well explored and beyond the scope of the manuscript, here, we focus less on augmentation strategies based on data pooling and instead explore the following procedures for computational model-based augmentation: “digital twins” and “synthetic data”.</p>
<p id="Par73">Digital twins are virtual, computer-based models of entities or systems that can be used to simulate behavioUr or/and forecast the trajectory of the physical item it represents [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>]. The concept of these virtual copies is their ability to replicate substantial internal dynamics on the basis of data gathered from the actual system of interest. This enables observation of how the digital entity would respond and react in different situations. This type of synthetic data, which is obtained artificially through mathematical models and algorithms instead of real-world data, allows predictions to be made and data about the physical entity to be collected without being subjected to direct investigation [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>].</p>
<p id="Par74">Digital twins work on the premise that, while providing information, they are also frequently updated with real-time information from their physical twin. This allows for consistent parameter updates to provide the most adjusted and optimized versions of the digital twin for the best possible performance. Digital twins conceptually originated for engineering purposes, but their undeniable potential in personalized medicine has driven significant advancements in healthcare applications. The ability to model the cellular and physiological interactions of biochemical pathways provides many opportunities in healthcare fields such as disease management, drug targeting, patient monitoring and more [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>].</p>
<p id="Par75">With respect to dermatology, as early as 1997, it was estimated that a digital image could effectively substitute for a dermatological visit in more than 80% of cases [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>], which further improved the use of digital methods with the advent of modern deep learning technologies [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>]. This, however, comes with caveats concerning the generalizability of performance results [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>], as is often the case for medical ML, because of a lack of standardization for reporting methods and benchmarks, offering a view of a biomedical domain readily amenable to acceleration and improvement of practices by computational methods.</p>
<p id="Par76">Nevertheless, dermatology and skin biology are surprisingly untouched by efforts in the space of digital twins [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>]. What is needed to establish a digital twin ecosystem capable of impacting the medical market? First, it requires open or otherwise properly governed and auditable data collection. The human skin is a complex organ with multiple layers, and structural complexity is needed to sustain immune, sensing, and repair functions. Creating models requires access to reliable data, and guaranteeing the quality of models further requires provenance, veracity, and mode of use certifications. Data repositories should include codified knowledge about structure, variability in anatomy, age, sex, clinical reports, etc.</p>
<p id="Par77">Furthermore, when hypothesis-driven models are considered, the computational biology community should embrace standardisation practices that are widely accepted to significantly contribute to the success of ML research and deployment. As Python has been the vehicle for the generation of a competitive and accessible arena for ML, by ensuring a low entry barrier for the development of early idea prototypes and sufficient interoperability to benchmark alternative solutions against each other, digital twins ought to leave behind the heroic practice of developing small models from scratch and should embrace one or more powerful model development suites that are being built and maintained by a handful of communities [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>].</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion &amp; outlook</h2>
<p id="Par78">Computational and mathematical models have yielded fundamental insights into many complex systems in biology, bioengineering and biomedicine. It is therefore unsurprising that researchers of the skin are increasingly employing computational tools to help better understand processes of the skin, as well as skin-related applications and pertinent biomedical topics. Here, we provide a comprehensive overview of relevant lines of work, highlighting the strengths and weaknesses of different computational methodologies.</p>
<p id="Par79">As dermatology, and science in general, becomes more interdisciplinary and multifaceted, the depth and variety of computerized techniques will doubtlessly advance further. Nevertheless, it emerges as a bigger picture where complementary approaches allow one to address different goals. Conveying the capabilities of such methods is paramount for their future use and cross-fertilization across scientific disciplines. As we presented in the second chapter of this review two main classes of computational approaches to simulating and understanding the complex biophysics/biomechanics of the skin are available: <em>model-based</em> and <em>data-driven</em> approaches (Table <a href="#Tab1" class="usa-link">1</a>) – these are the two extremes of a spectrum, and hybrid models can exploit both classes respectively. On the one end, the class of <em>model-based</em> (or, as some refer to them as physics-based models) approaches, they are typically grounded in continuum mechanics and thermodynamics to explicitly represent skin’s multi-layered structure (i.e., epidermis, dermis, hypodermis) and its key biological constituents, via FE modelling for example. Their primary strength is founded in their high interpretability to provide mechanistic insights and perform simulations of pertinent biological scenarios (e.g., graft surgical interventions or transdermal drug delivery) even with limited data. However, such models are attributed to be computationally intensive when simulating large or highly detailed structures of the skin, which limits their use in real-time applications; while also formulating accurate mathematical models that can capture the intricate biology of the skin requires specialized biomechanical and biological testing to inform such a model. On the other end, data-based models are primarily based using ML or DL techniques, which bypass the need to provide explicit mathematical or biophysical rules, and they learn by “reading” patterns from experimental or clinical evidence (e.g., histological data/images, cutometer readings, etc.). The literature survey presented in the sections above demonstrates that data-based models can identify complex, often non-intuitive, correlations within high-dimensional biological datasets, while also permitting significant faster predictions, which makes them suitable for real-time applications (e.g., rapid disease skin diagnosis diagnosis, or computer-aided surgery. Despite of this however, data-based models are seen as “black boxes” and offer little insight into the underlying biological mechanisms of the skin. Equally important is the fact that the performance of these models is heavily constrained by the quantity and quality of the data employed for the model training, while they often struggle to generalise to biological conditions as for instance in rare skin diseases or in diverse ethnicities. Approaches that utilise different types of methods can help address context-specific issues, such as for instance the lack of experimental data on particular model parameters. Along those lines, model-based methods often utilise pre-existing background information to improve the accuracy of a given computational model. On the other hand, ML and DL can help identify patterns based on empirical data. It is therefore the task of the investigator to determine the right balance and utilise the most suitable method for any given problem in skin research. Thus, the choice of model (or models) depends on balancing accuracy, computational resources, and accomplishing specific research or clinical objectives. As mentioned above, a new alternative that is gaining traction is the use of hybrid models that combine elements of multiple models, which may fall in either of the two classes. These hybrid models can leverage the strengths of each class to better simulate the multifaceted nature of skin biomechanics, by incorporating imaging data (MRI, ultrasound, histology) into the model and permit personalised and high-fidelity simulations of the skin biophysics.</p>
<p id="Par80">Despite the wealth of current computational techniques and knowledge, significant challenges with respect to real-world applications remain. Along those lines, the skin is a highly personal organ, and it is important to avoid data biases in terms of age, race or sex. Failure to avoid such biases will necessarily lead to biases in computational methods as well, which can lead to misdiagnosis and potentially dangerous medical advice. Hence, any biomedically relevant computational dermatology approach should aim for personalised parametrisation. Ideally, non-invasive, cost-effective and quick methods to measure and assess relevant properties are needed. Moreover, given the complex network of interactions in the body, many data modalities are likely important for optimizing medical impact. Among these, omics data, lifestyle-related data and even weather-related information can, in principle, play a role in determining the current and likely future state of the skin.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Future perspective</h2>
<p id="Par81">In the future, we hope and expect the gap between theoretical and experimental dermatology to narrow, so that methodological challenges can be reduced and hypotheses, including their implications, can be thoroughly assessed. A core requisite for such increased interdisciplinary cooperation resides in the implementation of user-friendly and adaptable software. Along those lines, open-source software plays a crucial role. We highlight the need for cross-disciplinary interactions and the exchange of ideas, which enables truly user-friendly software. The reason is that only when users are directly involved in the development of the software are necessary features and customizable functionalities appropriately acknowledged, supported and realized. We have demonstrated relevant activities and outputs [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>] and aim to continue working toward a future with stronger interdisciplinary, cost-efficient and ethical toolsets.</p>
<p id="Par82">In addition to software, more widespread sharing and use of common datasets is crucial. In other areas of computational biology, such as computational neuroscience, large amounts of neuroimaging data or electrical activity recordings can be made freely available [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. This availability not only enables more scientists to conduct research but also serves the purposes of reproducibility, benchmarking, and standardisation. Furthermore, sharing these data and resources on a broader scale can ensure that dermatological research remains dynamic, open, and ever-evolving. This will help address pressing challenges in the field, including improving diagnostic accuracy, personalizing treatments, and developing cost-effective, ethical solutions to skin-related health problems. In summary, the future of dermatology can be shaped by a combination of advanced computational models, collaborative software development, and a shared commitment to open data, ultimately contributing to more efficient, equitable, and effective scientific progress.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The article is based upon work from COST Action NETSKINMODELS CA21108 supported by COST (European Cooperation in Science and Technology).</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>SC and RB designed and lead the project. All authors wrote, reviewed and edited the manuscript. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>No datasets were generated or analysed during the current study.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par83">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Yousef H, Alhajj M, Sharma S. Anatomy, Skin (Integument), Epidermis. Treasure Island (FL): StatPearls Publishing; 2022.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/29262154/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Saiwaeo S, Arwatchananukul S, Mungmai L, Preedalikit W, Aunsri N. Human skin type classification using image processing and deep learning approaches. Heliyon. 2023;9(11):e21176. 10.1016/j.heliyon.2023.e21176.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2023.e21176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10656243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38027689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saiwaeo%20S,%20Arwatchananukul%20S,%20Mungmai%20L,%20Preedalikit%20W,%20Aunsri%20N.%20Human%20skin%20type%20classification%20using%20image%20processing%20and%20deep%20learning%20approaches.%20Heliyon.%202023;9(11):e21176.%2010.1016/j.heliyon.2023.e21176." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Brown TM, Krishnamurthy K. Histology, Dermis. Treasure Island (FL): StatPearls Publishing; 2022.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/30570967/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Richard MA, et al. Prevalence of most common skin diseases in Europe: a population-based study. J Eur Acad Dermatol Venereol. 2022;36(7):1088–96. 10.1111/JDV.18050.
</cite> [<a href="https://doi.org/10.1111/jdv.18050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9415115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35274366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richard%20MA,%20et%20al.%20Prevalence%20of%20most%20common%20skin%20diseases%20in%20Europe:%20a%20population-based%20study.%20J%20Eur%20Acad%20Dermatol%20Venereol.%202022;36(7):1088%E2%80%9396.%2010.1111/JDV.18050." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Tabassum N, Hamdani M. Plants used to treat skin diseases. Pharmacogn Rev. 2014;8(15):52. 10.4103/0973-7847.125531.
</cite> [<a href="https://doi.org/10.4103/0973-7847.125531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3931201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24600196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tabassum%20N,%20Hamdani%20M.%20Plants%20used%20to%20treat%20skin%20diseases.%20Pharmacogn%20Rev.%202014;8(15):52.%2010.4103/0973-7847.125531." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Stanton DN, Ganguli-Indra G, Indra AK, Karande P. Bioengineered Efficacy Models of Skin Disease: Advances in the Last 10 Years. Pharmaceutics. 2022;14(2). 10.3390/PHARMACEUTICS14020319.</cite> [<a href="https://doi.org/10.3390/pharmaceutics14020319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8877988/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35214050/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Tanaka RJ, Ono M. Skin Disease Modelling from a Mathematical Perspective. J Invest Dermatol. 2013;133(6):1472–8. 10.1038/JID.2013.69.
</cite> [<a href="https://doi.org/10.1038/jid.2013.69" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23514928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanaka%20RJ,%20Ono%20M.%20Skin%20Disease%20Modelling%20from%20a%20Mathematical%20Perspective.%20J%20Invest%20Dermatol.%202013;133(6):1472%E2%80%938.%2010.1038/JID.2013.69." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Roberts PA, Gaffney EA, Luthert PJ, Foss AJE, Byrne HM. Mathematical and computational models of the retina in health, development and disease. Prog Retin Eye Res. 2016;53:48–69.
</cite> [<a href="https://doi.org/10.1016/j.preteyeres.2016.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27063291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roberts%20PA,%20Gaffney%20EA,%20Luthert%20PJ,%20Foss%20AJE,%20Byrne%20HM.%20Mathematical%20and%20computational%20models%20of%20the%20retina%20in%20health,%20development%20and%20disease.%20Prog%20Retin%20Eye%20Res.%202016;53:48%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Bauer R, Cicero AFG, Manca M. Data driven and model based computational futures in cardiovascular practice. Front Cardiovasc Med. 2023;10(Frontiers Media SA):1336340.
</cite> [<a href="https://doi.org/10.3389/fcvm.2023.1336340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10711627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38089768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bauer%20R,%20Cicero%20AFG,%20Manca%20M.%20Data%20driven%20and%20model%20based%20computational%20futures%20in%20cardiovascular%20practice.%20Front%20Cardiovasc%20Med.%202023;10(Frontiers%20Media%20SA):1336340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Nguyen G, et al. Machine learning and deep learning frameworks and libraries for large-scale data mining: a survey. Artif Intell Rev. 2019;52:77–124.</cite> [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20G,%20et%20al.%20Machine%20learning%20and%20deep%20learning%20frameworks%20and%20libraries%20for%20large-scale%20data%20mining:%20a%20survey.%20Artif%20Intell%20Rev.%202019;52:77%E2%80%93124." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Railsback SF, Grimm V. Agent-based and individual-based modelling: a practical introduction. Princeton (NJ): Princeton University Press; 2019.</cite>
</li>
<li id="CR12">
<span class="label">12.</span><cite>Breitwieser L, et al. BioDynaMo: a modular platform for high-performance agent-based simulation. Bioinformatics. 2022;38(2):453–60.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8723141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34529036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breitwieser%20L,%20et%20al.%20BioDynaMo:%20a%20modular%20platform%20for%20high-performance%20agent-based%20simulation.%20Bioinformatics.%202022;38(2):453%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kang S, Kahan S, McDermott J, Flann N, Shmulevich I. Biocellion: accelerating computer simulation of multicellular biological system models. Bioinformatics. 2014;30(21):3101–8.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4609016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25064572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang%20S,%20Kahan%20S,%20McDermott%20J,%20Flann%20N,%20Shmulevich%20I.%20Biocellion:%20accelerating%20computer%20simulation%20of%20multicellular%20biological%20system%20models.%20Bioinformatics.%202014;30(21):3101%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Limbert G. Mathematical and computational modelling of skin biophysics: a review. Proc R Soc A Math Phys Eng Sci. 2017;473(2203):20170257.</cite> [<a href="https://doi.org/10.1098/rspa.2017.0257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5549575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28804267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Limbert%20G.%20Mathematical%20and%20computational%20modelling%20of%20skin%20biophysics:%20a%20review.%20Proc%20R%20Soc%20A%20Math%20Phys%20Eng%20Sci.%202017;473(2203):20170257." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Chaudhary M, Khan A, Gupta M. Skin Ageing: Pathophysiology and Current Market Treatment Approaches. Curr Aging Sci. 2020;13(1):22–30. 10.2174/1567205016666190809161115.
</cite> [<a href="https://doi.org/10.2174/1567205016666190809161115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7403684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31530270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaudhary%20M,%20Khan%20A,%20Gupta%20M.%20Skin%20Ageing:%20Pathophysiology%20and%20Current%20Market%20Treatment%20Approaches.%20Curr%20Aging%20Sci.%202020;13(1):22%E2%80%9330.%2010.2174/1567205016666190809161115." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Youn J, Kim D, Kwak H, Kim DS. Tissue-scale in vitro epithelial wrinkling and wrinkle-to-fold transition. Nat Commun. 2024;15:7118. 10.1038/s41467-024-51437-z.
</cite> [<a href="https://doi.org/10.1038/s41467-024-51437-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11333725/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39160171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Youn%20J,%20Kim%20D,%20Kwak%20H,%20Kim%20DS.%20Tissue-scale%20in%20vitro%20epithelial%20wrinkling%20and%20wrinkle-to-fold%20transition.%20Nat%20Commun.%202024;15:7118.%2010.1038/s41467-024-51437-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Zhao Y, Feng B, Lee J, Lu N, Pierce DM. A multi-layered computational model for wrinkling of human skin predicts aging effects. J Mech Behav Biomed Mater. 2020;103:103552. 10.1016/j.jmbbm.2019.103552.
</cite> [<a href="https://doi.org/10.1016/j.jmbbm.2019.103552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32090947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Y,%20Feng%20B,%20Lee%20J,%20Lu%20N,%20Pierce%20DM.%20A%20multi-layered%20computational%20model%20for%20wrinkling%20of%20human%20skin%20predicts%20aging%20effects.%20J%20Mech%20Behav%20Biomed%20Mater.%202020;103:103552.%2010.1016/j.jmbbm.2019.103552." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Pissarenko A, Meyers MA. The materials science of skin: Analysis, characterization, and modelling. Prog Mater Sci. 2020;110:100634. 10.1016/j.pmatsci.2019.100634.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pissarenko%20A,%20Meyers%20MA.%20The%20materials%20science%20of%20skin:%20Analysis,%20characterization,%20and%20modelling.%20Prog%20Mater%20Sci.%202020;110:100634.%2010.1016/j.pmatsci.2019.100634." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Bermudez DM, Herdrich BJ, Xu J, Lind R, Beason DP, Mitchell ME, Soslowsky LJ, Liechty KW. Impaired biomechanical properties of diabetic skin implications in pathogenesis of diabetic wound complications. Am J Pathol. 2011;178(5):2215–23. 10.1016/j.ajpath.2011.01.015.
</cite> [<a href="https://doi.org/10.1016/j.ajpath.2011.01.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3081147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21514435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bermudez%20DM,%20Herdrich%20BJ,%20Xu%20J,%20Lind%20R,%20Beason%20DP,%20Mitchell%20ME,%20Soslowsky%20LJ,%20Liechty%20KW.%20Impaired%20biomechanical%20properties%20of%20diabetic%20skin%20implications%20in%20pathogenesis%20of%20diabetic%20wound%20complications.%20Am%20J%20Pathol.%202011;178(5):2215%E2%80%9323.%2010.1016/j.ajpath.2011.01.015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Dwivedi K, Lakhani P, Sihota P, Tikoo K, Kumar S, Kumar N. The multiscale characterization and constitutive modelling of healthy and type 2 diabetes mellitus Sprague Dawley rat skin. Acta Biomater. 2023;158:324–46. 10.1016/j.actbio.2022.12.037.
</cite> [<a href="https://doi.org/10.1016/j.actbio.2022.12.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36565785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dwivedi%20K,%20Lakhani%20P,%20Sihota%20P,%20Tikoo%20K,%20Kumar%20S,%20Kumar%20N.%20The%20multiscale%20characterization%20and%20constitutive%20modelling%20of%20healthy%20and%20type%202%20diabetes%20mellitus%20Sprague%20Dawley%20rat%20skin.%20Acta%20Biomater.%202023;158:324%E2%80%9346.%2010.1016/j.actbio.2022.12.037." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Emu IA, et al. Arsenicskinimagebd: a comprehensive image dataset to classify affected and healthy skin of arsenic-affected people. Data Brief. 2024;52: 110016. 10.1016/j.dib.2023.110016.
</cite> [<a href="https://doi.org/10.1016/j.dib.2023.110016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10827410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38293578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emu%20IA,%20et%20al.%20Arsenicskinimagebd:%20a%20comprehensive%20image%20dataset%20to%20classify%20affected%20and%20healthy%20skin%20of%20arsenic-affected%20people.%20Data%20Brief.%202024;52:%20110016.%2010.1016/j.dib.2023.110016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Liu M, et al. Modelling drug exposure data in electronic medical records: an application to warfarin. AMIA Annu Symp Proc. 2011;2011(Ml):815–23.
</cite> [<a href="/articles/PMC3243123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22195139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20M,%20et%20al.%20Modelling%20drug%20exposure%20data%20in%20electronic%20medical%20records:%20an%20application%20to%20warfarin.%20AMIA%20Annu%20Symp%20Proc.%202011;2011(Ml):815%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Ohno K, Kobayashi Y, Uesaka M, et al. A computational model of the epidermis with the deformable dermis and its application to skin diseases. Sci Rep. 2021;11:13234.
</cite> [<a href="https://doi.org/10.1038/s41598-021-92540-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8225835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34168195/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Flynn C, McCormack BAO. Finite element modelling of forearm skin wrinkling. Ski Res Technol. 2008;14(3):261–9.</cite> [<a href="https://doi.org/10.1111/j.1600-0846.2008.00289.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19159370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flynn%20C,%20McCormack%20BAO.%20Finite%20element%20modelling%20of%20forearm%20skin%20wrinkling.%20Ski%20Res%20Technol.%202008;14(3):261%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kuwazuru O, Miyamoto K, Yoshikawa N, Imayama S. Skin wrinkling morphology changes suddenly in the early 30s. Ski Res Technol. 2012;18(4):495–503.</cite> [<a href="https://doi.org/10.1111/j.1600-0846.2011.00598.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22233477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuwazuru%20O,%20Miyamoto%20K,%20Yoshikawa%20N,%20Imayama%20S.%20Skin%20wrinkling%20morphology%20changes%20suddenly%20in%20the%20early%2030s.%20Ski%20Res%20Technol.%202012;18(4):495%E2%80%93503." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Shiihara Y, Sato M, Hara Y, Iwai I, Yoshikawa N. Microrelief suppresses large wrinkling appearance: an in silico study. Ski Res Technol. 2015;21(2):184–91.</cite> [<a href="https://doi.org/10.1111/srt.12175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25470358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shiihara%20Y,%20Sato%20M,%20Hara%20Y,%20Iwai%20I,%20Yoshikawa%20N.%20Microrelief%20suppresses%20large%20wrinkling%20appearance:%20an%20in%20silico%20study.%20Ski%20Res%20Technol.%202015;21(2):184%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Limbert G, Kuhl E. On skin microrelief and the emergence of expression microwrinkles. Soft Matter. 2018;14(8):1292–300.
</cite> [<a href="https://doi.org/10.1039/c7sm01969f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29319711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Limbert%20G,%20Kuhl%20E.%20On%20skin%20microrelief%20and%20the%20emergence%20of%20expression%20microwrinkles.%20Soft%20Matter.%202018;14(8):1292%E2%80%93300." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Liu Z, Qi Q, Wang S, Zhai G. A novel approach to the detection of facial wrinkles: database, detection algorithm, and evaluation metrics. Comput Biol Med. 2024;174: 108431.
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2024.108431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38626507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Z,%20Qi%20Q,%20Wang%20S,%20Zhai%20G.%20A%20novel%20approach%20to%20the%20detection%20of%20facial%20wrinkles:%20database,%20detection%20algorithm,%20and%20evaluation%20metrics.%20Comput%20Biol%20Med.%202024;174:%20108431." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kalbande D, Dhamane S, Bhope R. Implementation of Baumann Skin Type Indicator Using Machine Learning. In: International Conference on Multi-Strategy Learning Environment. Dehradun, India: Springer, Singapore; 2024. pp. 63–74.</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Martin V, Séguier R, Porcheron A, Morizot F. Face aging simulation with a new wrinkle oriented active appearance model. Multimed Tools Appl. 2019;78:6309–27.</cite> [<a href="https://scholar.google.com/scholar_lookup?Martin%20V,%20S%C3%A9guier%20R,%20Porcheron%20A,%20Morizot%20F.%20Face%20aging%20simulation%20with%20a%20new%20wrinkle%20oriented%20active%20appearance%20model.%20Multimed%20Tools%20Appl.%202019;78:6309%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Singh G, Chanda A. Biomechanical modelling of progressive wound healing: a computational study. Biomed Eng Adv. 2022;4:100055.</cite> [<a href="https://scholar.google.com/scholar_lookup?Singh%20G,%20Chanda%20A.%20Biomechanical%20modelling%20of%20progressive%20wound%20healing:%20a%20computational%20study.%20Biomed%20Eng%20Adv.%202022;4:100055." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Chanda A, Upchurch W. Biomechanical modelling of wounded skin. J Compos Sci. 2018;2(4):69.</cite> [<a href="https://scholar.google.com/scholar_lookup?Chanda%20A,%20Upchurch%20W.%20Biomechanical%20modelling%20of%20wounded%20skin.%20J%20Compos%20Sci.%202018;2(4):69." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Apalla Z, Nashan D, Weller RB, Castellsagué X. Skin cancer: epidemiology, disease burden, pathophysiology, diagnosis, and therapeutic approaches. Dermatol Ther (Heidelb). 2017;7:5–19.
</cite> [<a href="https://doi.org/10.1007/s13555-016-0165-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5289116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28150105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Apalla%20Z,%20Nashan%20D,%20Weller%20RB,%20Castellsagu%C3%A9%20X.%20Skin%20cancer:%20epidemiology,%20disease%20burden,%20pathophysiology,%20diagnosis,%20and%20therapeutic%20approaches.%20Dermatol%20Ther%20(Heidelb).%202017;7:5%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Das K, et al. Machine learning and its application in skin cancer. Int J Environ Res Public Health. 2021. 10.3390/ijerph182413409.</cite> [<a href="https://doi.org/10.3390/ijerph182413409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8705277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34949015/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Shah A, et al. A comprehensive study on skin cancer detection using artificial neural network (ANN) and convolutional neural network (CNN). Clin eHealth. 2023;6:76–84. 10.1016/j.ceh.2023.08.002.</cite> [<a href="https://scholar.google.com/scholar_lookup?Shah%20A,%20et%20al.%20A%20comprehensive%20study%20on%20skin%20cancer%20detection%20using%20artificial%20neural%20network%20(ANN)%20and%20convolutional%20neural%20network%20(CNN).%20Clin%20eHealth.%202023;6:76%E2%80%9384.%2010.1016/j.ceh.2023.08.002." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bhargav S, Sowmya V, Syama S, Lekshmi S. Skin cancer detection using Machine Learning. In: Proc. 2nd IEEE Int. Conf. Disruptive Technol. Multi-Disciplinary Res. Appl. CENTCON 2022, vol. 2. 2022. pp. 119–24. 10.1109/CENTCON56610.2022.10051495.</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Shahin M, Chen FF, Hosseinzadeh A, Khodadadi Koodiani H, Shahin A, Ali Nafi O. A smartphone-based application for an early skin disease prognosis: towards a lean healthcare system via computer-based vision. Adv Eng Inform. 2023;57: 102036. 10.1016/j.aei.2023.102036.</cite> [<a href="https://scholar.google.com/scholar_lookup?Shahin%20M,%20Chen%20FF,%20Hosseinzadeh%20A,%20Khodadadi%C2%A0Koodiani%20H,%20Shahin%20A,%20Ali%C2%A0Nafi%20O.%20A%20smartphone-based%20application%20for%20an%20early%20skin%20disease%20prognosis:%20towards%20a%20lean%20healthcare%20system%20via%20computer-based%20vision.%20Adv%20Eng%20Inform.%202023;57:%20102036.%2010.1016/j.aei.2023.102036." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Medhat S, Abdel-Galil H, Aboutabl AE, Saleh H. Skin cancer diagnosis using convolutional neural networks for smartphone images: a comparative study. J Radiat Res Appl Sci. 2022;15(1):262–7. 10.1016/j.jrras.2022.03.008.</cite> [<a href="https://scholar.google.com/scholar_lookup?Medhat%20S,%20Abdel-Galil%20H,%20Aboutabl%20AE,%20Saleh%20H.%20Skin%20cancer%20diagnosis%20using%20convolutional%20neural%20networks%20for%20smartphone%20images:%20a%20comparative%20study.%20J%20Radiat%20Res%20Appl%20Sci.%202022;15(1):262%E2%80%937.%2010.1016/j.jrras.2022.03.008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Humayun AI, Balestriero R, Baraniuk R. Deep Networks Always Grok and Here is Why. In International Conference on Machine Learning. PMLR. 2024. pp. 20722-745.</cite>
</li>
<li id="CR40">
<span class="label">40.</span><cite>Zagoruyko S, Komodakis N. Wide Residual Networks. In British Machine Vision Conference 2016. British Machine Vision Association. Accessed 17 Apr 2025.</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite>Chollet F. Xception: Deep learning with depthwise separable convolutions. In: Proc. IEEE Conf. Comput. Vis. Pattern Recognit. (CVPR). Honolulu, HI, USA: IEEE; 2017. pp. 1251–8.</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Han D, Kim J, Kim J. Deep pyramidal residual networks. In: Proc. IEEE Conf. Comput. Vis. Pattern Recognit. (CVPR). 2017. pp. 5927–35.</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Yosinski J, Clune J, Bengio Y, Lipson H. How transferable are features in deep neural networks? In: <em>Adv. Neural Inf. Process. Syst.</em>Cambridge: MIT Press; 2014. pp. 3320–8.</cite>
</li>
<li id="CR44">
<span class="label">44.</span><cite>Weiss K, Khoshgoftaar TM, Wang D. A survey of transfer learning. J Big Data. 2016;3(9). 10.1186/s40537-016-0043-6.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Shorten C, Khoshgoftaar TM. A survey on image data augmentation for deep learning. J Big Data. 2019;6(60). 10.1186/s40537-019-0197-0.</cite> [<a href="https://doi.org/10.1186/s40537-021-00492-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8287113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34306963/" class="usa-link">PubMed</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Malibari AA, et al. Optimal deep neural network-driven computer aided diagnosis model for skin cancer. Comput Electr Eng. 2022;103: 108318. 10.1016/j.compeleceng.2022.108318.</cite> [<a href="https://scholar.google.com/scholar_lookup?Malibari%20AA,%20et%20al.%20Optimal%20deep%20neural%20network-driven%20computer%20aided%20diagnosis%20model%20for%20skin%20cancer.%20Comput%20Electr%20Eng.%202022;103:%20108318.%2010.1016/j.compeleceng.2022.108318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Zhang L, Zhang J, Gao W, Bai F, Li N, Ghadimi N. A deep learning outline aimed at prompt skin cancer detection utilizing gated recurrent unit networks and improved orca predation algorithm. Biomed Signal Process Control. 2024;90:105858. 10.1016/j.bspc.2023.105858.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20L,%20Zhang%20J,%20Gao%20W,%20Bai%20F,%20Li%20N,%20Ghadimi%20N.%20A%20deep%20learning%20outline%20aimed%20at%20prompt%20skin%20cancer%20detection%20utilizing%20gated%20recurrent%20unit%20networks%20and%20improved%20orca%20predation%20algorithm.%20Biomed%20Signal%20Process%20Control.%202024;90:105858.%2010.1016/j.bspc.2023.105858." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Altamimi A, Alrowais F, Karamti H, Umer M, Cascone L, Ashraf I. An improved skin lesion detection solution using multistep preprocessing features and NASNet transfer learning model. Image Vis Comput. 2024;144: 104969. 10.1016/j.imavis.2024.104969.</cite> [<a href="https://scholar.google.com/scholar_lookup?Altamimi%20A,%20Alrowais%20F,%20Karamti%20H,%20Umer%20M,%20Cascone%20L,%20Ashraf%20I.%20An%20improved%20skin%20lesion%20detection%20solution%20using%20multistep%20preprocessing%20features%20and%20NASNet%20transfer%20learning%20model.%20Image%20Vis%20Comput.%202024;144:%20104969.%2010.1016/j.imavis.2024.104969." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Chanda D, Onim MSH, Nyeem H, Ovi TB, Naba SS. DCENSnet: A new deep convolutional ensemble network for skin cancer classification. Biomed Signal Process Control. 2024;89:105757. 10.1016/j.bspc.2023.105757.</cite> [<a href="https://scholar.google.com/scholar_lookup?Chanda%20D,%20Onim%20MSH,%20Nyeem%20H,%20Ovi%20TB,%20Naba%20SS.%20DCENSnet:%20A%20new%20deep%20convolutional%20ensemble%20network%20for%20skin%20cancer%20classification.%20Biomed%20Signal%20Process%20Control.%202024;89:105757.%2010.1016/j.bspc.2023.105757." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Sies K, et al. Past and present of computer-assisted dermoscopic diagnosis: performance of a conventional image analyser versus a convolutional neural network in a prospective dataset of 1,981 skin lesions. Eur J Cancer. 2020;135:39–46. 10.1016/j.ejca.2020.04.043.
</cite> [<a href="https://doi.org/10.1016/j.ejca.2020.04.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32534243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sies%20K,%20et%20al.%20Past%20and%20present%20of%20computer-assisted%20dermoscopic%20diagnosis:%20performance%20of%20a%20conventional%20image%20analyser%20versus%20a%20convolutional%20neural%20network%20in%20a%20prospective%20dataset%20of%201,981%20skin%20lesions.%20Eur%20J%20Cancer.%202020;135:39%E2%80%9346.%2010.1016/j.ejca.2020.04.043." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Ramprasad MVS, Nagesh SSV, Sahith V, Lankalapalli RK. Hierarchical agglomerative clustering-based skin lesion detection with region based neural networks classification. Meas Sensors. 2023;29: 100865. 10.1016/j.measen.2023.100865.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ramprasad%20MVS,%20Nagesh%20SSV,%20Sahith%20V,%20Lankalapalli%20RK.%20Hierarchical%20agglomerative%20clustering-based%20skin%20lesion%20detection%20with%20region%20based%20neural%20networks%20classification.%20Meas%20Sensors.%202023;29:%20100865.%2010.1016/j.measen.2023.100865." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kulkarni PM, et al. Deep learning based on standard H &amp;e images of primary melanoma tumors identifies patients at risk for visceral recurrence and death. Clin Cancer Res. 2020;26(5):1126–34. 10.1158/1078-0432.CCR-19-1495.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-19-1495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8142811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31636101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kulkarni%20PM,%20et%20al.%20Deep%20learning%20based%20on%20standard%20H%20&amp;e%20images%20of%20primary%20melanoma%20tumors%20identifies%20patients%20at%20risk%20for%20visceral%20recurrence%20and%20death.%20Clin%20Cancer%20Res.%202020;26(5):1126%E2%80%9334.%2010.1158/1078-0432.CCR-19-1495." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Groh M, Badri O, Daneshjou R, et al. Deep learning-aided decision support for diagnosis of skin disease across skin tones. Nat Med. 2024;30:573–83. 10.1038/s41591-023-02728-3.
</cite> [<a href="https://doi.org/10.1038/s41591-023-02728-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10878981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38317019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Groh%20M,%20Badri%20O,%20Daneshjou%20R,%20et%20al.%20Deep%20learning-aided%20decision%20support%20for%20diagnosis%20of%20skin%20disease%20across%20skin%20tones.%20Nat%20Med.%202024;30:573%E2%80%9383.%2010.1038/s41591-023-02728-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Tschandl P, Rinner C, Apalla Z, et al. Human–computer collaboration for skin cancer recognition. Nat Med. 2020;26:1229–34. 10.1038/s41591-020-0942-0.
</cite> [<a href="https://doi.org/10.1038/s41591-020-0942-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32572267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tschandl%20P,%20Rinner%20C,%20Apalla%20Z,%20et%20al.%20Human%E2%80%93computer%20collaboration%20for%20skin%20cancer%20recognition.%20Nat%20Med.%202020;26:1229%E2%80%9334.%2010.1038/s41591-020-0942-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Menzies SW, Sinz C, Menzies M, Lo SN, Yolland W, Lingohr J, et al. Comparison of humans versus mobile phone-powered artificial intelligence for the diagnosis and management of pigmented skin cancer in secondary care: a multicentre, prospective, diagnostic, clinical trial. Lancet Digit Health. 2023;5(10):e679–91. 10.1016/S2589-7500(23)00130-9.
</cite> [<a href="https://doi.org/10.1016/S2589-7500(23)00130-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37775188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menzies%20SW,%20Sinz%20C,%20Menzies%20M,%20Lo%20SN,%20Yolland%20W,%20Lingohr%20J,%20et%20al.%20Comparison%20of%20humans%20versus%20mobile%20phone-powered%20artificial%20intelligence%20for%20the%20diagnosis%20and%20management%20of%20pigmented%20skin%20cancer%20in%20secondary%20care:%20a%20multicentre,%20prospective,%20diagnostic,%20clinical%20trial.%20Lancet%20Digit%20Health.%202023;5(10):e679%E2%80%9391.%2010.1016/S2589-7500(23)00130-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Chen SC, Bravata DM, Weil E, Olkin I. A comparison of dermatologists’ and primary care physicians’ accuracy in diagnosing melanoma: a systematic review. Arch Dermatol. 2001;137(12):1627–34. 10.1001/archderm.137.12.1627.
</cite> [<a href="https://doi.org/10.1001/archderm.137.12.1627" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11735713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20SC,%20Bravata%20DM,%20Weil%20E,%20Olkin%20I.%20A%20comparison%20of%20dermatologists%E2%80%99%20and%20primary%20care%20physicians%E2%80%99%20accuracy%20in%20diagnosing%20melanoma:%20a%20systematic%20review.%20Arch%20Dermatol.%202001;137(12):1627%E2%80%9334.%2010.1001/archderm.137.12.1627." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Fahmy T, Elhendawi H, Elsharkawy WB, Reicha FM. AC conductivity and dielectric relaxation of chitosan/poly (vinyl alcohol) biopolymer polyblend. Bull Mater Sci. 2020;43:1–10.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fahmy%20T,%20Elhendawi%20H,%20Elsharkawy%20WB,%20Reicha%20FM.%20AC%20conductivity%20and%20dielectric%20relaxation%20of%20chitosan/poly%20(vinyl%20alcohol)%20biopolymer%20polyblend.%20Bull%20Mater%20Sci.%202020;43:1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Sherratt JA, Dallon JC. Theoretical models of wound healing: past successes and future challenges. C R Biol. 2002;325(5):557–64.
</cite> [<a href="https://doi.org/10.1016/s1631-0691(02)01464-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12187641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sherratt%20JA,%20Dallon%20JC.%20Theoretical%20models%20of%20wound%20healing:%20past%20successes%20and%20future%20challenges.%20C%20R%20Biol.%202002;325(5):557%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Guerra A, Belinha J, Jorge RN. Modelling skin wound healing angiogenesis: A review. J Theor Biol. 2018;459:1–17.
</cite> [<a href="https://doi.org/10.1016/j.jtbi.2018.09.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30240579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guerra%20A,%20Belinha%20J,%20Jorge%20RN.%20Modelling%20skin%20wound%20healing%20angiogenesis:%20A%20review.%20J%20Theor%20Biol.%202018;459:1%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Flynn C. Finite element models of wound closure. J Tissue Viability. 2010;19(4):137–49.
</cite> [<a href="https://doi.org/10.1016/j.jtv.2009.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19942434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flynn%20C.%20Finite%20element%20models%20of%20wound%20closure.%20J%20Tissue%20Viability.%202010;19(4):137%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Capek L, Jacquet E, Dzan L, Simunek A. The analysis of forces needed for the suturing of elliptical skin wounds. Med Biol Eng Comput. 2012;50:193–8.
</cite> [<a href="https://doi.org/10.1007/s11517-011-0857-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22201040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capek%20L,%20Jacquet%20E,%20Dzan%20L,%20Simunek%20A.%20The%20analysis%20of%20forces%20needed%20for%20the%20suturing%20of%20elliptical%20skin%20wounds.%20Med%20Biol%20Eng%20Comput.%202012;50:193%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Chanda A, Unnikrishnan V. A realistic 3D computational model of the closure of skin wound with interrupted sutures. J Mech Med Biol. 2017;17(01):1750025.</cite> [<a href="https://scholar.google.com/scholar_lookup?Chanda%20A,%20Unnikrishnan%20V.%20A%20realistic%203D%20computational%20model%20of%20the%20closure%20of%20skin%20wound%20with%20interrupted%20sutures.%20J%20Mech%20Med%20Biol.%202017;17(01):1750025." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Valero C, Javierre E, García-Aznar JM, Gómez-Benito MJ. Numerical modelling of the angiogenesis process in wound contraction. Biomech Model Mechanobiol. 2013;12:349–60.
</cite> [<a href="https://doi.org/10.1007/s10237-012-0403-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22584608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valero%20C,%20Javierre%20E,%20Garc%C3%ADa-Aznar%20JM,%20G%C3%B3mez-Benito%20MJ.%20Numerical%20modelling%20of%20the%20angiogenesis%20process%20in%20wound%20contraction.%20Biomech%20Model%20Mechanobiol.%202013;12:349%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>McDougall S, Dallon J, Sherratt J, Maini P. Fibroblast migration and collagen deposition during dermal wound healing: mathematical modelling and clinical implications. Philos Trans R Soc A Math Phys Eng Sci. 1843;2006(364):1385–405.</cite> [<a href="https://doi.org/10.1098/rsta.2006.1773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16766351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDougall%20S,%20Dallon%20J,%20Sherratt%20J,%20Maini%20P.%20Fibroblast%20migration%20and%20collagen%20deposition%20during%20dermal%20wound%20healing:%20mathematical%20modelling%20and%20clinical%20implications.%20Philos%20Trans%20R%20Soc%20A%20Math%20Phys%20Eng%20Sci.%201843;2006(364):1385%E2%80%93405." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Ziraldo C, Mi Q, An G, Vodovotz Y. Computational modelling of inflammation and wound healing. Adv Wound Care. 2013;2(9):527–37.</cite> [<a href="https://doi.org/10.1089/wound.2012.0416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3816782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24527362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ziraldo%20C,%20Mi%20Q,%20An%20G,%20Vodovotz%20Y.%20Computational%20modelling%20of%20inflammation%20and%20wound%20healing.%20Adv%20Wound%20Care.%202013;2(9):527%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Boon WM, Koppenol DC, Vermolen FJ. A multiagent cell-based model for wound contraction. J Biomech. 2016;49(8):1388–401.
</cite> [<a href="https://doi.org/10.1016/j.jbiomech.2015.11.058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26805459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boon%20WM,%20Koppenol%20DC,%20Vermolen%20FJ.%20A%20multiagent%20cell-based%20model%20for%20wound%20contraction.%20J%20Biomech.%202016;49(8):1388%E2%80%93401." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Vermolen FJ, Gefen A. Semistochastic cell-level computational modelling of cellular forces. Biomech Model Mechanobiol. 2015;14(6):1181-95.</cite> [<a href="https://doi.org/10.1007/s10237-015-0664-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25776519/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Vermolen F. Uncertainty Assessment of a Hybrid Cell-Continuum Based Model for Wound Contraction. In: Numerical Mathematics and Advanced Applications ENUMATH 2017. Bergen, Norway: Springer Cham; 2019. pp. 247–55.</cite>
</li>
<li id="CR69">
<span class="label">69.</span><cite>Bai J, Zeng X. Computational modelling and simulation of epithelial wound closure. Sci Rep. 2023;13(1):6265.
</cite> [<a href="https://doi.org/10.1038/s41598-023-33111-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10110613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37069231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bai%20J,%20Zeng%20X.%20Computational%20modelling%20and%20simulation%20of%20epithelial%20wound%20closure.%20Sci%20Rep.%202023;13(1):6265." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Goel A, Pendlington R, Glavin S, Chen T, Belsey NA. Characterization of skin penetration pathways using stimulated Raman scattering microscopy. Eur J Pharm Biopharm. 2024;204:114518.
</cite> [<a href="https://doi.org/10.1016/j.ejpb.2024.114518" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39362383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goel%20A,%20Pendlington%20R,%20Glavin%20S,%20Chen%20T,%20Belsey%20NA.%20Characterization%20of%20skin%20penetration%20pathways%20using%20stimulated%20Raman%20scattering%20microscopy.%20Eur%20J%20Pharm%20Biopharm.%202024;204:114518." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Zhang D, Deacon BN, Li W, Lian G, Chen T. A computational workflow for end-to-end simulation of percutaneous absorption. Int J Pharm. 2025;670: 125084.
</cite> [<a href="https://doi.org/10.1016/j.ijpharm.2024.125084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39681219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20D,%20Deacon%20BN,%20Li%20W,%20Lian%20G,%20Chen%20T.%20A%20computational%20workflow%20for%20end-to-end%20simulation%20of%20percutaneous%20absorption.%20Int%20J%20Pharm.%202025;670:%20125084." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Toan DN, Thanh ND, Truong MX, Van DT, Thanh NN. Design, synthesis, molecular docking study and molecular dynamics simulation of new coumarin-pyrimidine hybrid compounds having anticancer and antidiabetic activity. Med Chem Res. 2023;32(6):1143–62.
</cite> [<a href="https://doi.org/10.1007/s00044-023-03060-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10119543/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37305206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Toan%20DN,%20Thanh%20ND,%20Truong%20MX,%20Van%20DT,%20Thanh%20NN.%20Design,%20synthesis,%20molecular%20docking%20study%20and%20molecular%20dynamics%20simulation%20of%20new%20coumarin-pyrimidine%20hybrid%20compounds%20having%20anticancer%20and%20antidiabetic%20activity.%20Med%20Chem%20Res.%202023;32(6):1143%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Ita K, Roshanaei S. Artificial intelligence for skin permeability prediction: deep learning. J Drug Target. 2024;32(3):334–46.
</cite> [<a href="https://doi.org/10.1080/1061186X.2024.2309574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38258521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ita%20K,%20Roshanaei%20S.%20Artificial%20intelligence%20for%20skin%20permeability%20prediction:%20deep%20learning.%20J%20Drug%20Target.%202024;32(3):334%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Shakel Z, Lima SAC, Reis S. Strategies to Make Human Skin Models based on Cellular Senescence for Ageing Research. Ageing Res Rev. 2024;100:102430.
</cite> [<a href="https://doi.org/10.1016/j.arr.2024.102430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39032611/" class="usa-link">PubMed</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Dai X, et al. Decreased oxidative stress response and oxidant detoxification of skin during aging. Mech Ageing Dev. 2023;216: 111878. 10.1016/j.mad.2023.111878.
</cite> [<a href="https://doi.org/10.1016/j.mad.2023.111878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37827221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dai%20X,%20et%20al.%20Decreased%20oxidative%20stress%20response%20and%20oxidant%20detoxification%20of%20skin%20during%20aging.%20Mech%20Ageing%20Dev.%202023;216:%20111878.%2010.1016/j.mad.2023.111878." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Quan T. Molecular insights of human skin epidermal and dermal aging. J Dermatol Sci. 2023;112(2):48–53. 10.1016/j.jdermsci.2023.08.006.
</cite> [<a href="https://doi.org/10.1016/j.jdermsci.2023.08.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37661473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quan%20T.%20Molecular%20insights%20of%20human%20skin%20epidermal%20and%20dermal%20aging.%20J%20Dermatol%20Sci.%202023;112(2):48%E2%80%9353.%2010.1016/j.jdermsci.2023.08.006." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Cao L, Lee B, Lee B-H, Lee S, Kim H-R. Sargahydroquinoic acid from Sargassum macrocarpum attenuates TNF-<img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d724/12366154/f74ccd99c859/13040_2025_471_Article_IEq2.gif" loading="lazy" alt="Inline graphic"> and UV-induced skin aging in human dermal fibroblasts. Algal Res. 2024;78:103410. 10.1016/j.algal.2024.103410.</cite>
</li>
<li id="CR78">
<span class="label">78.</span><cite>Wang X-M, et al. Network-based analysis identifies key regulatory transcription factors involved in skin aging. Exp Gerontol. 2023;178: 112202. 10.1016/j.exger.2023.112202.
</cite> [<a href="https://doi.org/10.1016/j.exger.2023.112202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37178875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X-M,%20et%20al.%20Network-based%20analysis%20identifies%20key%20regulatory%20transcription%20factors%20involved%20in%20skin%20aging.%20Exp%20Gerontol.%202023;178:%20112202.%2010.1016/j.exger.2023.112202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Jagtap J, Kokare M. Human age classification using facial skin aging features and artificial neural network. Cogn Syst Res. 2016;40:116–28. 10.1016/j.cogsys.2016.05.002.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jagtap%20J,%20Kokare%20M.%20Human%20age%20classification%20using%20facial%20skin%20aging%20features%20and%20artificial%20neural%20network.%20Cogn%20Syst%20Res.%202016;40:116%E2%80%9328.%2010.1016/j.cogsys.2016.05.002." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Alagić A, et al. Application of artificial intelligence in the analysis of the facial skin health condition. IFAC-PapersOnLine. 2022;55(4):31–7. 10.1016/j.ifacol.2022.06.005.38620941
</cite> [<a href="https://scholar.google.com/scholar_lookup?Alagi%C4%87%20A,%20et%20al.%20Application%20of%20artificial%20intelligence%20in%20the%20analysis%20of%20the%20facial%20skin%20health%20condition.%20IFAC-PapersOnLine.%202022;55(4):31%E2%80%937.%2010.1016/j.ifacol.2022.06.005.38620941" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Yıldız M, et al. Segmentation and classification of skin burn images with artificial intelligence: Development of a mobile application. Burns. 2024. 10.1016/j.burns.2024.01.007.</cite> [<a href="https://doi.org/10.1016/j.burns.2024.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38331663/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Zhu Z, Park HS, McAlpine MC. 3D printed deformable sensors. Sci Adv. 2020;6(25):1–10. 10.1126/sciadv.aba5575.</cite> [<a href="https://doi.org/10.1126/sciadv.aba5575" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7299613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32596461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20Z,%20Park%20HS,%20McAlpine%20MC.%203D%20printed%20deformable%20sensors.%20Sci%20Adv.%202020;6(25):1%E2%80%9310.%2010.1126/sciadv.aba5575." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Proença M. Noninvasive hemodynamic monitoring by electrical impedance tomography. Lausanne: EPFL; 2017.</cite>
</li>
<li id="CR84">
<span class="label">84.</span><cite>Wei Q, Rojas D, Wang Q, Zapata-Pérez R, Xuan X, Molinero-Fernández Á, Crespo GA, Cuartero M. Wearable 3D-Printed Microneedle Sensor for Intradermal Temperature Monitoring. ACS Sens. 2025. 10.1021/acssensors.4c03681.</cite> [<a href="https://doi.org/10.1021/acssensors.4c03681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12210253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40230020/" class="usa-link">PubMed</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Hwang C, Choi J-G, Pang C, Kim M-S, Park S. Skin-Conformal Ag Flake-Decorated PEDOT:PSS Sensor Arrays for Spatially Resolved Body Temperature Monitoring. *Small* 2024;2412675. 10.1002/smll.202412675.</cite> [<a href="https://doi.org/10.1002/smll.202412675" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40424019/" class="usa-link">PubMed</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Ambinder EP. A history of the shift toward full computerization of medicine. J Oncol Pract. 2005;1(2):54.
</cite> [<a href="https://doi.org/10.1200/jop.2005.1.2.54" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2793587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20871680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ambinder%20EP.%20A%20history%20of%20the%20shift%20toward%20full%20computerization%20of%20medicine.%20J%20Oncol%20Pract.%202005;1(2):54." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Petrini C, Mannelli C, Riva L, Gainotti S, Gussoni G. Decentralized clinical trials (DCTs): A few ethical considerations. Front Public Heal. 2022;10:1081150.</cite> [<a href="https://doi.org/10.3389/fpubh.2022.1081150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9797802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36590004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petrini%20C,%20Mannelli%20C,%20Riva%20L,%20Gainotti%20S,%20Gussoni%20G.%20Decentralized%20clinical%20trials%20(DCTs):%20A%20few%20ethical%20considerations.%20Front%20Public%20Heal.%202022;10:1081150." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Weiss S, Kulikowski CA, Safir A. Glaucoma consultation by computer. Comput Biol Med. 1978;8(1):25–40.
</cite> [<a href="https://doi.org/10.1016/0010-4825(78)90011-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/620517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weiss%20S,%20Kulikowski%20CA,%20Safir%20A.%20Glaucoma%20consultation%20by%20computer.%20Comput%20Biol%20Med.%201978;8(1):25%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>GVResearch GV. Clinical Trials Market Size, Share &amp; Trends Analysis Report By Phase (Phase I, Phase II, Phase III, Phase IV), By Study Design, By Indication (Pain Management, Oncology, CNS Condition, Diabetes, Obesity), By Region, And Segment Forecasts, 2022 - 2030. p. 120, 2023. [Online]. <a href="https://www.grandviewresearch.com/industry-analysis/global-clinical-trials-market" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.grandviewresearch.com/industry-analysis/global-clinical-trials-market#</a>. Accessed 12 Aug 2025.</cite>
</li>
<li id="CR90">
<span class="label">90.</span><cite>Chouhan N, Vig H, Deshmukh R. Cosmetics Market. Allied Mark Res. 2021;1(February):277.</cite> [<a href="https://scholar.google.com/scholar_lookup?Chouhan%20N,%20Vig%20H,%20Deshmukh%20R.%20Cosmetics%20Market.%20Allied%20Mark%20Res.%202021;1(February):277." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Grieves M. Conceptual ideal for PLM. Presentation for the Product Lifecycle Management (PLM) Center. Michigan: University of Michigan; 2002.</cite> [<a href="https://scholar.google.com/scholar_lookup?Grieves%20M.%20Conceptual%20ideal%20for%20PLM.%20Presentation%20for%20the%20Product%20Lifecycle%20Management%20(PLM)%20Center.%20Michigan:%20University%20of%20Michigan;%202002." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Katsoulakis E, et al. Digital twins for health: a scoping review. NPJ Digit Med. 2024;7(1): 77.
</cite> [<a href="https://doi.org/10.1038/s41746-024-01073-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10960047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38519626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katsoulakis%20E,%20et%20al.%20Digital%20twins%20for%20health:%20a%20scoping%20review.%20NPJ%20Digit%20Med.%202024;7(1):%2077." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Zhang K, et al. Concepts and applications of digital twins in healthcare and medicine. Patterns. 2024;5(8).</cite> [<a href="https://doi.org/10.1016/j.patter.2024.101028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11368703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39233690/" class="usa-link">PubMed</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Kvedar JC, et al. The substitution of digital images for dermatologic physical examination. Arch Dermatol. 1997;133(2):161–7.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9041828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kvedar%20JC,%20et%20al.%20The%20substitution%20of%20digital%20images%20for%20dermatologic%20physical%20examination.%20Arch%20Dermatol.%201997;133(2):161%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Puri P, et al. Deep learning for dermatologists: Part II. Current applications. J Am Acad Dermatol. 2022;87(6):1352–60.
</cite> [<a href="https://doi.org/10.1016/j.jaad.2020.05.053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7669658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32428608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Puri%20P,%20et%20al.%20Deep%20learning%20for%20dermatologists:%20Part%20II.%20Current%20applications.%20J%20Am%20Acad%20Dermatol.%202022;87(6):1352%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Daneshjou R, et al. Disparities in dermatology AI performance on a diverse, curated clinical image set. Sci Adv. 2022;8(32).</cite> [<a href="https://doi.org/10.1126/sciadv.abq6147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9374341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35960806/" class="usa-link">PubMed</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Akbarialiabad H, Pasdar A, Murrell DF. Digital twins in dermatology, current status, and the road ahead. NPJ Digit Med. 2024;7(1):228.
</cite> [<a href="https://doi.org/10.1038/s41746-024-01220-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11347578/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39187568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akbarialiabad%20H,%20Pasdar%20A,%20Murrell%20DF.%20Digital%20twins%20in%20dermatology,%20current%20status,%20and%20the%20road%20ahead.%20NPJ%20Digit%20Med.%202024;7(1):228." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Ghaffarizadeh A, Heiland R, Friedman SH, Mumenthaler SM, Macklin P. PhysiCell: An open source physics-based cell simulator for 3-D multicellular systems. PLoS Comput Biol. 2018;14(2):e1005991.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005991" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5841829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29474446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghaffarizadeh%20A,%20Heiland%20R,%20Friedman%20SH,%20Mumenthaler%20SM,%20Macklin%20P.%20PhysiCell:%20An%20open%20source%20physics-based%20cell%20simulator%20for%203-D%20multicellular%20systems.%20PLoS%20Comput%20Biol.%202018;14(2):e1005991." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Markiewicz CJ, et al. The OpenNeuro resource for sharing of neuroscience data. Elife. 2021;10:e71774.
</cite> [<a href="https://doi.org/10.7554/eLife.71774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8550750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34658334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Markiewicz%20CJ,%20et%20al.%20The%20OpenNeuro%20resource%20for%20sharing%20of%20neuroscience%20data.%20Elife.%202021;10:e71774." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Lee J, Wu H, Mannan A, Nakamura Y, Amagai M, Irvine A, Tanaka R. Towards the next generation of in silico modelling of dynamic host-microbiota interactions in the skin. JID Innov. 2025;5:100385.</cite> [<a href="https://doi.org/10.1016/j.xjidi.2025.100385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12221595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40606013/" class="usa-link">PubMed</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Montgomery V, Wood-Yang A, Styczynski M, Prausnitz M. Feasibility of engineered <em>Bacillus subtilis</em> for use as a microbiome-based topical drug delivery platform. Bioeng Transl Med. 2024;9: e10645.
</cite> [<a href="https://doi.org/10.1002/btm2.10645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11256169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39036074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Montgomery%20V,%20Wood-Yang%20A,%20Styczynski%20M,%20Prausnitz%20M.%20Feasibility%20of%20engineered%20Bacillus%20subtilis%20for%20use%20as%20a%20microbiome-based%20topical%20drug%20delivery%20platform.%20Bioeng%20Transl%20Med.%202024;9:%20e10645." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Miyano T, Irvine A, Tanaka R. Model-based meta-analysis to optimize <em>Staphylococcus aureus</em>–targeted therapies for atopic dermatitis. JID Innov. 2022;2: 100110.
</cite> [<a href="https://doi.org/10.1016/j.xjidi.2022.100110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9214323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35757782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyano%20T,%20Irvine%20A,%20Tanaka%20R.%20Model-based%20meta-analysis%20to%20optimize%20Staphylococcus%20aureus%E2%80%93targeted%20therapies%20for%20atopic%20dermatitis.%20JID%20Innov.%202022;2:%20100110." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Jaiswal S, Agarwal S, Thodum P, Sharma V. SkinBug: an artificial intelligence approach to predict human skin microbiome-mediated metabolism of biotics and xenobiotics. Iscience. 2021;24:101925.</cite> [<a href="https://doi.org/10.1016/j.isci.2020.101925" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7772573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33385118/" class="usa-link">PubMed</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Wei Y, Zhang S, Zhao C, Yang F, Pan Q. Network-based modeling for analyzing the human skin microbiome. In: 2010 IEEE International Conference on Bioinformatics and Biomedicine Workshops (BIBMW). Hong Kong, China: IEEE; 2010. pp. 119–23.</cite>
</li>
<li id="CR105">
<span class="label">105.</span><cite>Dakal T, Xu C, Kumar A. Advanced computational tools, artificial intelligence and machine-learning approaches in gut microbiota and biomarker identification. Front Med Technol. 2025;6:1434799.
</cite> [<a href="https://doi.org/10.3389/fmedt.2024.1434799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12037385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40303946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dakal%20T,%20Xu%20C,%20Kumar%20A.%20Advanced%20computational%20tools,%20artificial%20intelligence%20and%20machine-learning%20approaches%20in%20gut%20microbiota%20and%20biomarker%20identification.%20Front%20Med%20Technol.%202025;6:1434799." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No datasets were generated or analysed during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BioData Mining are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13040-025-00471-8"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13040_2025_Article_471.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366154/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366154/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366154%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366154/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366154/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366154/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830504/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366154/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830504/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366154/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366154/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="c8EAFXmNN1Rt8TOGlZ7EgGlE2Jlrck1f8Mysab7sEzmbKPJh3Akbg69hEsFzBRQi">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
