
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Plant negative-strand RNA virus phosphoprotein condensates exploit host trafficking and lipid synthesis for viral factory assembly - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152545448AF31493054544000A4FE4A2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366689/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="Plant negative-strand RNA virus phosphoprotein condensates exploit host trafficking and lipid synthesis for viral factory assembly">
<meta name="citation_author" content="Zhiyi Wang">
<meta name="citation_author_institution" content="Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Jingyi Zhang">
<meta name="citation_author_institution" content="Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Jilei Huang">
<meta name="citation_author_institution" content="Instrumental Analysis and Research Center, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Gan Sha">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Xinyue Song">
<meta name="citation_author_institution" content="Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Xue Cao">
<meta name="citation_author_institution" content="Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Zhenchen Yan">
<meta name="citation_author_institution" content="Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Chuanhe Liu">
<meta name="citation_author_institution" content="Instrumental Analysis and Research Center, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Siping Chen">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Ziying Li">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Xiuqin Huang">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Qingjun Xie">
<meta name="citation_author_institution" content="Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Plant Molecular Breeding, College of Agriculture, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Xin Yang">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Guohui Zhou">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author" content="Tong Zhang">
<meta name="citation_author_institution" content="Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_author_institution" content="National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="34">
<meta name="citation_firstpage" content="eadx7905">
<meta name="citation_doi" content="10.1126/sciadv.adx7905">
<meta name="citation_pmid" content="40834074">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366689/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366689/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366689/pdf/sciadv.adx7905.pdf">
<meta name="description" content="RNA viruses often remodel host intracellular membranes to establish specialized replication compartments through viral protein–induced phase separation. However, the mechanisms underlying membrane remodeling and the characteristics that render these ...">
<meta name="og:title" content="Plant negative-strand RNA virus phosphoprotein condensates exploit host trafficking and lipid synthesis for viral factory assembly">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="RNA viruses often remodel host intracellular membranes to establish specialized replication compartments through viral protein–induced phase separation. However, the mechanisms underlying membrane remodeling and the characteristics that render these ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366689/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366689">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adx7905"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adx7905.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366689%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366689/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366689/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366689/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 20;11(34):eadx7905. doi: <a href="https://doi.org/10.1126/sciadv.adx7905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adx7905</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Plant negative-strand RNA virus phosphoprotein condensates exploit host trafficking and lipid synthesis for viral factory assembly</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhiyi Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhiyi Wang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>3</sup></sup>National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Visualization, Writing - original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiyi Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jingyi Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jingyi Zhang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Data curation, Formal analysis, Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jingyi Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jilei Huang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jilei Huang</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Instrumental Analysis and Research Center, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Data curation, Formal analysis, Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jilei Huang</span></a>
</div>
</div>
<sup>4,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sha%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Gan Sha</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Gan Sha</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>3</sup></sup>National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Data curation, Formal analysis, Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sha%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gan Sha</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Xinyue Song</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xinyue Song</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinyue Song</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Xue Cao</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Xue Cao</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xue Cao</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zhenchen Yan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zhenchen Yan</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>3</sup></sup>National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhenchen Yan</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Chuanhe Liu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Chuanhe Liu</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Instrumental Analysis and Research Center, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chuanhe Liu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Siping Chen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Siping Chen</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Siping Chen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Ziying Li</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Ziying Li</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Ziying Li</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Xiuqin Huang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Xiuqin Huang</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiuqin Huang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Qingjun Xie</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Qingjun Xie</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>5</sup></sup>Guangdong Provincial Key Laboratory of Plant Molecular Breeding, College of Agriculture, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Resources, Visualization, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qingjun Xie</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Xin Yang</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Xin Yang</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin Yang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Guohui Zhou</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Guohui Zhou</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Project administration, Resources, Supervision, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guohui Zhou</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Tong Zhang</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Tong Zhang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="p">
<sup><sup>3</sup></sup>National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.</div>
<div>Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Supervision, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tong Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Guangdong Basic Research Center of Excellence for Precise Breeding of Future Crops, South China Agricultural University, Guangzhou 510642, China.</div>
<div id="aff2">
<sup><sup>2</sup></sup>Guangdong Province Key Laboratory of Microbial Signals and Disease Control, College of Plant Protection, South China Agricultural University, Guangzhou 510642, China.</div>
<div id="aff3">
<sup><sup>3</sup></sup>National Key Laboratory of Green Pesticide, South China Agricultural University, Guangzhou 510642, China.</div>
<div id="aff4">
<sup><sup>4</sup></sup>Instrumental Analysis and Research Center, South China Agricultural University, Guangzhou 510642, China.</div>
<div id="aff5">
<sup><sup>5</sup></sup>Guangdong Provincial Key Laboratory of Plant Molecular Breeding, College of Agriculture, South China Agricultural University, Guangzhou 510642, China.</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>zhangtong@scau.edu.cn</span> (T.Z.); <span>ghzhou@scau.edu.cn</span> (G.Z.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Zhiyi Wang</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Visualization, Writing - original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jingyi Zhang</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jilei Huang</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Gan Sha</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xinyue Song</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xue Cao</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Zhenchen Yan</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Chuanhe Liu</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Siping Chen</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ziying Li</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiuqin Huang</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Qingjun Xie</span></strong>: <span class="role">Resources, Visualization, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xin Yang</span></strong>: <span class="role">Supervision, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Guohui Zhou</span></strong>: <span class="role">Project administration, Resources, Supervision, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Tong Zhang</span></strong>: <span class="role">Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Supervision, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 28; Accepted 2025 Jul 9; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution License 4.0 (CC BY).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution license</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366689  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40834074/" class="usa-link">40834074</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>RNA viruses often remodel host intracellular membranes to establish specialized replication compartments through viral protein–induced phase separation. However, the mechanisms underlying membrane remodeling and the characteristics that render these sites conducive to replication remain poorly understood, particularly in plant negative-strand RNA viruses. Here, we demonstrate that the phosphoprotein (P) of rice stripe mosaic virus (RSMV) forms biomolecular condensates via liquid-liquid phase separation (LLPS) to recruit essential components for viral replication factories (VFs). We identify a direct interaction between RSMV P and adenosine diphosphate (ADP) ribosylation factor 1 (OsARF1C), a crucial regulator of the coatomer protein I (COP I) vesicle transport pathway that is vital for viral replication. This interaction indirectly recruits OsARF1C’s partner, phosphatidylinositol 4-kinase beta (OsPI4KB), which drives localized phosphatidylinositol-4 phosphate (PI4P) synthesis. Concurrently, the P protein modulates its aggregates and LLPS droplets through PI4P, thereby expanding the replication site and enhancing viral replication.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>RSMV P forms phase-separated condensates recruiting COP I to synthesize PI4P, expanding viral replication sites for infection.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Biomolecular condensates, driven by liquid-liquid phase separation (LLPS), are diverse membraneless organelles within cells that can move freely without being confined by a double-layer membrane. These structures play crucial roles in various cellular processes in eukaryotic cells, including growth, development, and signal transduction (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>–<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). Identified biomolecular condensates in cells include the nucleolus, Cajal bodies, processing bodies, stress granules, and more (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>, <a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>). Research indicates that the formation of LLPS is primarily regulated by intrinsically disordered regions (IDRs) of proteins. Furthermore, LLPS is influenced by posttranslational modifications of proteins, host factors, and changes in cellular environments (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>, <a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>, <a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>).</p>
<p>During the infection of host cells, many viruses create unique organelles that consist of viral proteins, host factors, and nucleic acids. These organelles, commonly referred to as viral inclusion bodies, viroplasms, or viral factories (VFs), play crucial roles in viral replication, transcription, and assembly (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>). The formation of VFs exhibits a high level of dynamic adaptability, rapidly assembling and disassembling throughout the viral infection process. Materials continuously flow and exchange within and outside these inclusion bodies, suggesting a close resemblance to biomolecular condensates driven by LLPS (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>). A growing body of evidence shows that viruses use their encoded proteins to undergo LLPS, establishing liquid condensates that serve as VFs. For instance, positive-sense RNA viruses, such as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which encodes the nucleocapsid (N) protein (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>–<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>), as well as human norovirus (HuNoV) that encodes RNA-dependent RNA polymerase (RdRp) (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>), are known to form such structures. Negative-sense RNA viruses, including Ebola virus (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>), vesicular stomatitis virus (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>), respiratory syncytial virus (RSV) (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>), and rabies virus (RABV) (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>, <a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>), also participate in LLPS through proteins such as N protein and phosphoprotein (P), while double-stranded RNA viruses such as rotavirus, which encodes the nonstructural protein NSP5, are capable of forming viroplasms essential for viral replication (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>). Similar mechanisms have recently been corroborated in plant viruses, where inhibiting the phase separation of the tomato spotted wilt virus N protein effectively blocked viral infection (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>). Notable progress has been made in understanding LLPS of plant viral proteins, suggesting that viral condensates may enhance virus fitness by facilitating biomolecule concentration and improving the efficiency of energy-intensive processes such as RNA replication (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). The barley yellow striate mosaic virus (BYSMV) P protein dynamically coordinates viral replication and transcription through phosphorylation-modulated phase separation dynamics (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). The tomato yellow mottle-associated virus uses its N and P proteins to assemble viral replication factories exhibiting LLPS properties (<a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>). The tomato bushy stunt virus (TBSV) p33 and the carnation Italian ringspot virus p36 facilitate viral replication factory assembly via LLPS (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>). These findings highlight the dual role of viral protein–driven LLPS in assembling inclusion bodies and expediting viral replication. However, the regulatory factors and precise mechanisms governing LLPS-mediated assembly of viral replication factories remain key unresolved questions in the field.</p>
<p>RNA viruses often modify intracellular membranes to create replication sites characterized by unique lipids and proteins distinct from those of the original organelles (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>, <a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>). Phosphoinositides (PIPs) are recognized lipids involved in the replication of various RNA viruses (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>–<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>). They are produced through the phosphorylation of phosphatidylinositol (PI) at the D-3, D-4, and/or D-5 positions of the hydroxyl groups in its inositol ring (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>–<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). Phosphatidylinositol 4-phosphate (PI4P) is the predominant PIP in cells, widely distributed across various cellular membranes, with the highest abundance at the trans-Golgi network (TGN) (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). PI4P is synthesized by phosphatidylinositol 4-kinases (PI4Ks). Mammals possess four PI4K isoforms, namely, the III isoforms PI4KA and PI4KB, and the II isoforms PI4K2A and PI4K2B (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>, <a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>, <a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>). In plants, only the III isoforms PI4KA/B have been confirmed to exhibit kinase activity in generating PI4P (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). PI4KA primarily localizes to the endoplasmic reticulum (ER), regulating the exit of transport vesicles from the ER, while PI4KB governs transport at the Golgi apparatus and from the Golgi to the plasma membrane (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>). Studies have shown that many animal RNA viruses can directly recruit PI4KB, relying on its activity to generate abundant PI4P to facilitate viral replication. In addition, plant positive-sense RNA viruses, such as TBSV, can enhance their own replication by hijacking different vesicular transport pathways or directly recruiting PI4K to produce ample PI4P (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>, <a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>, <a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>–<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>). Furthermore, several studies indicate that viruses can indirectly recruit the PI4KB-PI4P pathway by enlisting other host factors. Among these, research suggests that adenosine diphosphate (ADP) ribosylation factor 1 (ARF1), a key factor in coatomer protein I (COP I) vesicle formation, can recruit PI4KB to the Golgi apparatus, stimulating PI4KB to generate PI4P for the budding of Golgi-derived transport vesicles (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>, <a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>, <a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). To date, it remains unclear whether PIPs are necessary for the replication of plant negative-sense RNA viruses and how they are recruited or synthesized at replication sites.</p>
<p>Rice stripe mosaic virus (RSMV) is a notable pathogen responsible for considerable losses in rice production (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). It belongs to the family Rhabdoviridae, genus <em>Cytorhabdovirus</em>, and has a genome comprising a negative-sense single-stranded RNA of ~13 kb (<a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>). The complementary strand of the genome encodes seven nonoverlapping open reading frames that produce the classic N, P, matrixprotein (M), glycoprotein (G), and largeprotein (L) proteins of rhabdoviruses, along with two accessory proteins (<a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>–<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). Our recent study successfully established a mini-replicon (MR) system for RSMV that replicates and transcribes within <em>Nicotiana benthamiana</em> cells. Through this system, we found that the P protein encoded by RSMV can form granules, which are crucial for viral replication (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). Understanding the composition of the RSMV VFs and analyzing the factors that influence their dynamic adaptability may help in developing a theoretical model for the replication of plant negative-sense RNA viruses as well as provide new targets for disease prevention and control. In this study, we demonstrated that the P protein encoded by RSMV forms the core of the VFs through LLPS. Moreover, we identified that the P protein indirectly recruits OsPI4KB into the VFs by interacting with OsARF1C, a key factor in the vesicle transport pathway. This interaction facilitates the generation and accumulation of PI4P within the VFs. The presence of PI4P enhances the formation of larger granules during P protein LLPS, thereby providing RSMV with expanded replication sites and accelerating the viral infection process.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">The P protein is a crucial component of the VFs formed during RSMV infection</h3>
<p>In our previous study, we identified that the RSMV-encoded P protein forms VF-like inclusion bodies that are critical for viral replication (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). Electron microscopy revealed numerous ring-shaped VFs in the cytoplasm, encircling newly synthesized viral ribonucleoproteins (RNPs) (fig. S1A). Immunolabeling with a gold-conjugated P-specific antibody demonstrated a concentrated distribution of gold particles within the VFs, in contrast to their sparse presence in other virus-infected regions (<a href="#F1" class="usa-link">Fig. 1A</a>). Quantitative analysis of gold-labeled P protein within designated regions further substantiated these observations (<a href="#F1" class="usa-link">Fig. 1B</a>), highlighting the notable role of the P protein in RSMV VF architecture.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. RSMV P protein enriches in VFs and undergoes LLPS in vivo.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/a574ab7010c3/sciadv.adx7905-f1.jpg" loading="lazy" height="738" width="750" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Immunosorbent electron microscopy analysis of P subcellular localization patterns in noninfected and RSMV-infected rice stem cells at 30 days post inoculation (dpi). P is labeled with 12-nm gold particles. The blue-labeled areas represent virus particles (V), while the yellow-labeled areas represent VFs. Scale bars, 200 nm. (<strong>B</strong>) The number of P protein colloidal gold particles in both V and VF regions. Gold particles were counted in 30 cells; statistical analysis was conducted using Student’s <em>t</em> test, with *** indicating <em>P</em> &lt; 0.001. (<strong>C</strong>) Localization of GFP-P and mCherry-HDEL expressed in <em>N. benthamiana</em> leaves at 55 hours post infiltration (hpi). Scale bar, 20 μm. (<strong>D</strong>) Confocal images showing the fusion of two GFP-P granules in <em>N. benthamiana</em> leaf epidermal cells, with red arrows indicating granule fusion and yellow arrows indicating granule split. Scale bar, 2 μm. (<strong>E</strong>) Confocal images illustrating FRAP of GFP-P granules in <em>N. benthamiana</em> leaves at 55 hpi. Scale bar, 5 μm. (<strong>F</strong>) FRAP recovery curves for GFP-P granules, with the intensity of each granule normalized to its prebleach fluorescence. Data are presented as means ± SD of 10 granules. (<strong>G</strong>) Representative confocal images of GFP-P droplets before and after treatment with 10% HEX for 10 min in <em>N. benthamiana</em> leaves. Scale bars, 20 μm. (<strong>H</strong>) Subcellular distribution of BFP-P, N-GFP, and mCherry-L in rice protoplasts. Scale bars, 10 μm.</p></figcaption></figure><p>To characterize the material properties of the RSMV P, we examined the subcellular localization of green fluorescent protein (GFP)–P in <em>N. benthamiana</em> leaves. Consistent with previous findings (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>), GFP-P formed granules on the ER (<a href="#F1" class="usa-link">Fig. 1C</a>). These granules exhibited dynamic movement within the cytoplasm, undergoing fusion and division (<a href="#F1" class="usa-link">Fig. 1D</a> and movie S1). Fluorescence recovery after photobleaching (FRAP) revealed that ~36% of the GFP-P granule signal recovered within 120 s, indicating rapid protein redistribution between membraneless granules and cellular proteins (<a href="#F1" class="usa-link">Fig. 1, E and F</a>). Treatment with 1,6-hexanediol (HEX), a chemical inhibitor of liquid-like droplets, effectively dispersed the GFP-P granules (<a href="#F1" class="usa-link">Fig. 1G</a>). These findings suggest that the RSMV P protein forms liquid-like granules in vivo.</p>
<p>During rhabdovirus infection, viroplasms primarily comprise N, P, and L proteins for viral replication and transcription, and P functions as a central protein that anchors the polymerase complexes to the N-RNA templates (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). Our investigation focused on the potential role of RSMV P in facilitating the recruitment of N and L proteins for efficient RNA synthesis. Coexpression experiments in rice protoplasts revealed that N-GFP was incorporated into blue fluorescent protein (BFP)-P granules but not free BFP (fig. S1B). Similarly, BFP-P bodies were found to recruit mCherry-L, rather than mCherry (fig. S1C). Coexpression of N-GFP, mCherry-L, and BFP-P in rice protoplasts resulted in the formation of spherical granules, while coexpression with BFP led to even distribution in the cytoplasm (<a href="#F1" class="usa-link">Fig. 1H</a>). These results indicate that the RSMV P protein acts as a scaffold protein, initiating granule formation and facilitating the recruitment of N protein into condensed RNP complexes.</p></section><section id="sec4"><h3 class="pmc_sec_title">RSMV P undergoes phase separation in vitro</h3>
<p>The spherical GFP-P foci exhibited characteristics indicative of typical phase-separated biomolecular condensates. In vitro experiments confirmed the phase separation of the purified recombinant GFP-P protein, resulting in the formation of spherical droplets (<a href="#F2" class="usa-link">Fig. 2A</a>). Both the number and size of droplets increased with higher GFP-P concentrations and lower NaCl concentrations (<a href="#F2" class="usa-link">Fig. 2B</a>). HEX treatment effectively dispersed the GFP-P droplets (<a href="#F2" class="usa-link">Fig. 2C</a>). FRAP analysis revealed a recovery of ~50% of GFP-P fluorescent signals within 550 s (<a href="#F2" class="usa-link">Fig. 2, D and E</a>). Low-complexity regions or IDRs are well-established drivers of phase separation (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>). Previous findings identified three IDRs in the RSMV P protein, with IDR2 playing a crucial role in the formation of GFP-P droplets (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). The deletion of IDR2 from the recombinant protein (GFP-P<sup>mIDR2</sup>) inhibited spherical droplet formation, underscoring the importance of IDR2 in GFP-P condensate formation (fig. S2). These results validate the occurrence of LLPS of the RSMV P protein in vitro.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. P protein undergoes LLPS and recruits the N protein and genomic RNA in vitro.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/f4ae615f7d99/sciadv.adx7905-f2.jpg" loading="lazy" height="780" width="750" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Confocal images showing droplets formed by GFP-P at a concentration of 10 μM in 100 mM NaCl. Scale bar, 10 μm. (<strong>B</strong>) Confocal images demonstrating phase separation of GFP-P at various concentrations of GFP-P and NaCl. Scale bars, 20 μm. (<strong>C</strong>) Confocal images illustrating GFP-P droplets before and after treatment with 5% HEX for 1 min. Scale bars, 20 μm. (<strong>D</strong>) Confocal images displaying FRAP of GFP-P droplets in vitro at a concentration of 10 μM in 100 mM NaCl. Scale bar, 2 μm. (<strong>E</strong>) FRAP recovery curve of GFP-P droplets, with data represented as the means ± SD of 10 droplets. (<strong>F</strong>) Incorporation of mCherry-N into GFP-P droplets, where free GFP could not form droplets to recruit mCherry-N. Scale bars, 20 μm. (<strong>G</strong>) Confocal images representing the incorporation of mCherry-N and Cy5-Trailer of the RSMV genome into GFP-P droplets, where free GFP was unable to form droplets to recruit mCherry-N and Cy5-Trailer. Scale bars, 20 μm.</p></figcaption></figure><p>The ability of BFP-P granules to recruit N-GFP protein in vivo (<a href="#F1" class="usa-link">Fig. 1H</a>) prompted an investigation into the potential for RSMV P–formed droplets to concentrate RSMV N protein and genomic RNAs in vitro. Purified mCherry-N with free GFP did not undergo LLPS; however, incubation with GFP-P resulted in the gradual incorporation of mCherry-N into GFP-P–formed droplets (<a href="#F2" class="usa-link">Fig. 2F</a>). Subsequent experiments using a 227-nucleotide Cy5-labeled 5′ trailer of the RSMV genome demonstrated that both mCherry-N and Cy5-Trailer were incorporated into GFP-P droplets, whereas they remained evenly distributed when incubated with GFP alone (<a href="#F2" class="usa-link">Fig. 2G</a>). These findings collectively indicate that RSMV P–formed liquid droplets have the capacity to encapsulate RSMV N protein and the 5′ trailer of the RSMV genome in vitro.</p></section><section id="sec5"><h3 class="pmc_sec_title">RSMV P interacts with a key component of the COP I, OsARF1C</h3>
<p>A yeast two-hybrid (Y2H) screening was used to find host factors implicated in RSMV replication by interacting with the P protein. Considering the ER localization of the P protein, OsARF1C emerged as a potential interacting partner, as it plays a crucial role in the COP I vesicle transport pathway (<a href="#F3" class="usa-link">Fig. 3A</a>). This interaction was validated through protein affinity isolation (<a href="#F3" class="usa-link">Fig. 3B</a>). Bimolecular fluorescence complementation (BiFC) further demonstrated that the P protein interacts with OsARF1C in the cytoplasm, forming punctate granules (<a href="#F3" class="usa-link">Fig. 3C</a>). Subsequently, we generated OsARF1C overexpression transgenic lines (<em>OsARF1C-OE</em>) with a Myc tag fused to its C terminus. Coimmunoprecipitation (co-IP) assays indicated that the RSMV P protein was able to precipitate with OsARF1C-Myc following RSMV infection (<a href="#F3" class="usa-link">Fig. 3D</a>), confirming the existence of the OsARF1C-P protein complex in planta.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. Interaction of OsARF1C with P and its role in viral MR replication.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/226d5bc11a63/sciadv.adx7905-f3.jpg" loading="lazy" height="644" width="800" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) The interaction between P and OsARF1C was assessed using a Y2H assay. Yeast strain Y2HGold cultures, cotransformed with the indicated plasmids, were grown on SD-Trp-Leu-His-Ade selection medium. (<strong>B</strong>) In vitro glutathione <em>S</em>-transferase (GST) pull-down assays were conducted to explore the interaction between P and OsARF1C. (<strong>C</strong>) BiFC assay validated the interaction in rice protoplasts. Scale bars, 10 μm. (<strong>D</strong>) Co-IP assays confirmed the interaction between OsARF1C and RSMV P. WT and OsARF1C-OE plants were mock- or RSMV-inoculated and were harvested at 15 dpi. Total proteins were immunoprecipitated with anti-P beads. Input and IP proteins were analyzed by protein gel blot analysis with anti-Myc and anti-P antibodies. (<strong>E</strong>, <strong>G</strong>, and <strong>I</strong>) RT-qPCR analysis of minigenome RNA replication in the MR system at 6 dpi with various treatments. <em>NbPP2A</em> was used as an internal control gene. The values represent means ± SD (<em>n</em> = 4). ** and *** indicate highly significant difference (<em>P</em> &lt; 0.01 and <em>P</em> &lt; 0.001, respectively) based on the Tukey-Kramer post hoc test. (<strong>F</strong>, <strong>H</strong>, and <strong>J</strong>) Accumulation of GFP and RFP in the MR-inoculated <em>N. benthamiana</em> leaves at 6 dpi was measured via Western blotting, with relative protein band intensities normalized against that of actin.</p></figcaption></figure><p>The rice genome contains six ARF1 members: OsARF1A, OsARF1B, OsARF1C, OsARF1D, OsARF1E, and OsARF1F (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>). A comparative analysis of protein sequences among these six OsARF1 variants revealed that all ARF1 proteins consist of 181 amino acids and demonstrate remarkable sequence conservation (fig. S3A). BiFC assays indicated that the RSMV P protein interacts with all OsARF1 family members (fig. S3B). Consequently, OsARF1C was selected as a representative for further functional investigations. The transcription levels of OsARF1 remained unchanged during RSMV infection compared to healthy plants (fig. S3C).</p></section><section id="sec6"><h3 class="pmc_sec_title">The activity and homeostasis of OsARF1 are critical for RSMV infection</h3>
<p>To investigate the impact of ARF1 on RSMV infections in plants, we used tobacco rattle virus (TRV)–based virus-induced gene silencing to down-regulate <em>ARF1</em> in <em>N. benthamiana</em> plants (fig. S3D). Subsequently, the silenced plants were inoculated with the RSMV MR system (fig. S3E) (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). A notable reduction in both genomic RNA and reporter protein levels of MR was observed in NbARF1-silenced plants (<a href="#F3" class="usa-link">Fig. 3, E and F</a>), indicating the critical role of ARF1 in RSMV MR replication.</p>
<p>Brefeldin A (BFA) inhibits protein secretion from the Golgi apparatus to the ER by disrupting ARF protein nucleotide exchange reactions (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>). We tested the effect of BFA and observed that its treatment notably reduced OsARF1C expression and impaired colocalization between the viral P protein and OsARF1C (fig. S3F). Moreover, BFA treatment markedly decreased BFP-P granule size compared to the control (fig. S3, G and H). Then, <em>N. benthamiana</em> leaves were infected with RSMV MR for 6 days, followed by a 24-hour treatment with BFA. Subsequent reverse transcription quantitative polymerase chain reaction (RT-qPCR) analysis demonstrated a significant reduction in MR RNA accumulation (<a href="#F3" class="usa-link">Fig. 3G</a>). Simultaneously, the levels of GFP and red fluorescent protein (RFP) decreased (<a href="#F3" class="usa-link">Fig. 3H</a>). These findings suggest that the BFA-mediated inhibition of ARF1 function adversely affects the replication of RSMV MR.</p>
<p>ARFs function as molecular switches, transitioning between an inert state bound to guanosine diphosphate (GDP) and an activated state bound to guanosine triphosphate (GTP) (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>). We generated a GDP-locked mutant of OsARF1C, designated OsARF1C<sup>T31N</sup>, in which the threonine at position 31 was substituted with asparagine, and a GTP-locked mutant of OsARF1C, denoted OsARF1C<sup>Q71L</sup>, where the glutamine at position 71 was replaced with leucine (fig. S3A). BiFC analysis revealed that both mutants maintained the interactions with P proteins (fig. S4A). Overexpression of OsARF1C<sup>T31N</sup> or OsARF1C<sup>Q71L</sup> in <em>N. benthamiana</em> significantly reduced MR RNA accumulation (<a href="#F3" class="usa-link">Fig. 3I</a>). Western blot analysis corroborated the decreased levels of GFP and RFP in coinfiltrated tissues (<a href="#F3" class="usa-link">Fig. 3J</a>). Wild-type (WT) OsARF1C overexpression also diminished RNA and reporter protein levels of the RSMV MR, suggesting a potential role for ARF1 in viral disease resistance (<a href="#F3" class="usa-link">Fig. 3, I and J</a>).</p>
<p>In a manner consistent with the <em>OsARF1C-OE</em>, we generated transgenic rice lines that overexpressed OsARF1C<sup>T31N</sup> and OsARF1C<sup>Q71L</sup>, respectively. The overexpressing plants were then inoculated with RSMV. Compared to WT plants, these transgenic lines exhibited milder disease symptoms (<a href="#F4" class="usa-link">Fig. 4, A to C</a>) and showed reduced levels of viral RNA (<a href="#F4" class="usa-link">Fig. 4, E to G</a>) and N and P proteins (<a href="#F4" class="usa-link">Fig. 4, I to K</a>). In addition, we developed OsARF1 RNA interference (RNAi) lines targeting conserved sequences of six <em>OsARF1</em> genes. These RNAi lines demonstrated stunted growth and ultimately died (<a href="#F4" class="usa-link">Fig. 4D</a>), highlighting the critical role of ARF1 in plant development. Upon RSMV infection, the RNAi lines displayed attenuated viral phenotypes (<a href="#F4" class="usa-link">Fig. 4D</a>), with decreased levels of viral RNA and proteins (<a href="#F4" class="usa-link">Fig. 4, H and L</a>). Collectively, these findings indicate that the activity and homeostasis of OsARF1 are essential for RSMV replication.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. The role of OsARF1C in RSMV infection.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/f953ce620e90/sciadv.adx7905-f4.jpg" loading="lazy" height="620" width="800" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong> to <strong>C</strong>) Phenotypes of mock- and RSMV-inoculated WT, <em>OsARF1C-OE</em>, <em>OsARF1C</em><sup>T31N</sup><em>-OE</em>, and <em>OsARF1C</em><sup>Q71L</sup><em>-OE</em> transgenic rice plants at 35 dpi. Scale bars, 15 cm. (<strong>D</strong>) Phenotypes of mock- and RSMV-inoculated WT and <em>OsARF1-RNAi</em> transgenic rice plants at 20 dpi. Scale bars, 15 cm. (<strong>E</strong> to <strong>H</strong>) Accumulation of RSMV RNA was assessed in RSMV-inoculated WT, <em>OsARF1C-OE</em>, <em>OsARF1C</em><sup>T31N</sup><em>-OE</em>, and <em>OsARF1C</em><sup>Q71L</sup><em>-OE</em> transgenic rice plants at 35 dpi, as well as in RSMV-inoculated WT and <em>OsARF1-RNAi</em> transgenic rice plants at 20 dpi, by RT-qPCR. Shown values represent the means ± SD (<em>n</em> = 4). Different letters indicate significant differences (<em>P</em> &lt; 0.05) based on the Tukey-Kramer post hoc test. (<strong>I</strong> to <strong>L</strong>) Accumulation of RSMV N and P proteins was assessed in RSMV-inoculated WT, <em>OsARF1C-OE</em>, <em>OsARF1C</em><sup>T31N</sup><em>-OE</em>, and <em>OsARF1C</em><sup>Q71L</sup><em>-OE</em> transgenic rice plants at 35 dpi, as well as in RSMV-inoculated WT and <em>OsARF1-RNAi</em> transgenic rice plants at 20 dpi, using Western blot assays. The relative intensity of protein bands was normalized against tubulin.</p></figcaption></figure></section><section id="sec7"><h3 class="pmc_sec_title">RSMV P recruits OsPI4KB into granules via OsARF1C</h3>
<p>To elucidate the interaction mechanism between OsARF1C and the RSMV P protein, we investigated their subcellular localization rice protoplasts. OsARF1C–yellow fluorescent protein (YFP) localized to the cytoplasm, whereas BFP-P formed punctate granules. Coexpression revealed partial colocalization, with a subset of OsARF1C-YFP overlapping with the punctate granules (<a href="#F5" class="usa-link">Fig. 5A</a>). Furthermore, both the GDP-locked mutant and the GTP-locked mutant can be recruited into punctate granules formed by BFP-P (fig. S4B). In vitro investigations demonstrated that purified OsARF1C-mCherry did not exhibit LLPS with free GFP. However, coincubation with GFP-P resulted in the integration of OsARF1C-mCherry into droplets formed by the P protein (<a href="#F5" class="usa-link">Fig. 5B</a>). These findings suggest that the P protein has the capacity to recruit OsARF1C.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5. The P protein of RSMV hijacks OsPI4KB into its granules via OsARF1C.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/9e55495c88a9/sciadv.adx7905-f5.jpg" loading="lazy" height="862" width="750" alt="Fig. 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Confocal images showing the subcellular localization of P and OsARF1C in rice protoplasts. Scale bars, 10 μm. (<strong>B</strong>) Confocal images demonstrating the incorporation of OsARF1C-mCherry into GFP-P droplets. Free GFP was unable to form droplets to recruit OsARF1C-mCherry. Scale bars, 20 μm. (<strong>C</strong>) Confocal images illustrating the colocalization of BFP-P, GFP-PI4KB, and OsARF1C-RFP in rice protoplasts. Scale bars, 10 μm. Zoomed area scale bars, 1 μm. The figures on the right panel represent the normalized fluorescence intensities of the BFP, GFP, and RFP channels along the dashed white lines in the merged images.</p></figcaption></figure><p>ARF1 is crucial for protein and lipid transport in mammalian cells, specifically mediating PI4P synthesis through its interaction with PI4KB. Here, ARF1 facilitates and stimulates the generation of PI4P (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>). BiFC assays revealed an interaction between OsARF1C and OsPI4KB (fig. S4C). Subcellular localization studies demonstrated colocalization of OsPI4KB with OsARF1C and its mutant forms, OsARF1C<sup>T31N</sup> and OsARF1C<sup>Q71L</sup> (fig. S4D). Further analysis of colocalization with RSMV P protein showed that the P protein can recruit both OsARF1 and OsPI4KB into punctate granules. However, when OsARF1C<sup>T31N</sup> was present, the P protein could recruit OsARF1 but not OsPI4KB. In contrast, with OsARF1<sup>Q71L</sup>, neither of the proteins were recruited, and the fluorescence intensity analysis confirmed these observations (<a href="#F5" class="usa-link">Fig. 5C</a>). These results indicate that the recruitment of OsPI4KB by RSMV P protein is dependent on the activity of OsARF1C.</p></section><section id="sec8"><h3 class="pmc_sec_title">OsPI4KB generates PI4P to facilitate RSMV infection</h3>
<p>The function of OsPI4KB in RSMV replication was investigated using the RSMV MR system in <em>N. benthamiana</em> under various experimental conditions. PIK93, a PI4K inhibitor (<a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>), was applied to RSMV MR–infected tobacco leaves at 6 days postinoculation (dpi). RT-qPCR analysis performed 24 hours later revealed a significant reduction in MR RNA accumulation upon the application of PIK93 (<a href="#F6" class="usa-link">Fig. 6A</a>), which was associated with corresponding decreases in GFP and RFP protein levels (<a href="#F6" class="usa-link">Fig. 6B</a>). Subsequently, the TRV system was used to silence <em>NbPI4KB</em> expression (fig. S5A). When challenged with the RSMV MR system, <em>NbPI4KB</em>-silenced plants exhibited impaired viral replication, characterized by diminished levels of MR RNA and GFP/RFP proteins compared to β-glucuronidase (GUS)-silenced control plants (<a href="#F6" class="usa-link">Fig. 6, C and D</a>).</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Fig. 6. OsPI4KB generates PI4P to facilitate RSMV infection.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/473d719f2713/sciadv.adx7905-f6.jpg" loading="lazy" height="792" width="800" alt="Fig. 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) RT-qPCR analysis of minigenome RNA replication in the MR system at 6 dpi with PIK93 treatments. <em>NbPP2A</em> served as an internal control gene. The values shown are the means ± SD (<em>n</em> = 4). Student’s <em>t</em> test was used for analyses, and * indicates that the difference between the data is significant (<em>P</em> &lt; 0.05). (<strong>B</strong> and <strong>D</strong>) Accumulation of GFP and RFP in the MR-inoculated <em>N. benthamiana</em> leaves at 6 dpi was determined by Western blot assay. The relative protein band intensity was normalized against that of actin. (<strong>C</strong>) RT-qPCR analysis of minigenome RNA replication at 6 dpi in TRV-induced gene silenced <em>N. benthamiana</em> leaves, using <em>NbPP2A</em> as an internal control gene. The values shown are the means ± SD (<em>n</em> = 4). Student’s <em>t</em> test was used for analyses, and * indicates significant differences (<em>P</em> &lt; 0.05). (<strong>E</strong> and <strong>H</strong>) Phenotypes of mock-inoculated and RSMV-inoculated WT, <em>ospi4kb</em>, <em>GH1-OE</em>, and <em>gh1</em> transgenic rice plants at 35 dpi. Scale bars, 15 cm. (<strong>F</strong> and <strong>I</strong>) Accumulation of RSMV RNA in WT, <em>ospi4kb</em>, <em>GH1-OE</em>, and <em>gh1</em> transgenic rice plants inoculated with RSMV at 35 dpi was determined by RT-qPCR. The values shown are the means ± SD (<em>n</em> = 4). Different letters or * indicate significant differences (<em>P</em> &lt; 0.05) based on the Tukey-Kramer post hoc test. (<strong>G</strong> and <strong>J</strong>) Accumulation of RSMV N and P proteins in <em>ospi4kb</em>, <em>GH1-OE</em>, and <em>gh1</em> transgenic rice plants at 35 dpi was determined by Western blot assay. The relative protein band intensity was normalized against that of tubulin.</p></figcaption></figure><p>In addition, CRISPR-Cas9–generated <em>ospi4kb</em> mutants (fig. S5B) were subjected to RSMV infection. Comparative analysis revealed that <em>ospi4kb</em> mutant plants exhibited milder dwarfism upon infection (<a href="#F6" class="usa-link">Fig. 6E</a>), accompanied by decreased levels of RSMV RNA and N/P protein (<a href="#F6" class="usa-link">Fig. 6, F and G</a>). The rice <em>GH1</em> (GRAIN NUMBER AND PLANT HEIGHT1), a PI4P phosphohydrolase with a suppressor of actin (SAC) domain, plays a negative regulatory role in PI4P levels (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>). While <em>gh1</em> mutants with increased PI4P exhibit pronounced disease symptoms, plants overexpressing GH1 (<em>GH1-OE</em>) displayed symptoms comparable to those of WT plants (<a href="#F6" class="usa-link">Fig. 6H</a>). The <em>gh1</em> mutants showed elevated levels of RSMV RNA and N/P protein, in contrast to the similar levels observed in <em>GH1-OE</em> and WT plants (<a href="#F6" class="usa-link">Fig. 6, I and J</a>). These findings suggest that PI4P facilitates RSMV infection.</p></section><section id="sec9"><h3 class="pmc_sec_title">PI4P accumulates within the RSMV VFs</h3>
<p>We postulate that RSMV VFs are characterized by a high abundance of PI4P, as indicated by the recruitment of OsPI4KB. To investigate the potential binding of the P protein to PI4P, we used the molecular docking performed by Chai-1, indicating the interaction between P and PI4P (fig. S6). Furthermore, we performed a lipid-protein binding assay. Purified GFP, GFP-P, and OsARF1C-GFP were incubated on strips containing PI4P, PI, and buffer, with the PI4P biosensor (GFP-human four-phosphate-adaptor protein 1-pleckstrin homology domain, GFP-hFAPP1-PH) serving as a positive control (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a><em>,</em>
<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>). Results demonstrated that GFP-P selectively bound to PI4P, exhibiting a higher binding capacity than the GFP-hFAPP1-PH control (<a href="#F7" class="usa-link">Fig. 7A</a>). In addition, we conducted an in vivo analysis of PI4P and RSMV P colocalization, using GFP-hFAPP1-PH as a PI4P marker in plant cells. Our findings indicated PI4P accumulation within P protein granules in rice protoplasts (<a href="#F7" class="usa-link">Fig. 7B</a>).</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Fig. 7. Enrichment of P-binding PI4P in VFs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/c1f290b27313/sciadv.adx7905-f7.jpg" loading="lazy" height="669" width="720" alt="Fig. 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Analysis of the interaction between RSMV P and PI4P using a lipid binding assay. PI was included as a control, where the binding of hFAPP1-PH to PI4P served as a positive control. (<strong>B</strong>) Confocal microscopy images demonstrate the colocalization of P and PI4P in rice protoplasts, with GFP-hFAPP1-PH indicating the PI4P distribution. Scale bars, 10 μm. (<strong>C</strong>) Immunosorbent electron microscopy analysis showing the subcellular localization of P and PI4P in noninfected versus RSMV-infected rice stem cells at 30 dpi, with P and PI4P labeled using 10- and 18-nm gold particles, respectively. Scale bars, 500 nm. N, noninfected area; V, virion. (<strong>D</strong>) Quantification of P protein and PI4P colloidal gold particles in mock, noninfected, virion, and VF regions. Gold particle counts were performed in 30 cells. *** indicates a highly significant difference (<em>P</em> &lt; 0.001) based on the Tukey-Kramer post hoc test.</p></figcaption></figure><p>To investigate the distribution of PI4P in rice cells infected with RSMV, we used immunoelectron microscopy on samples from both mock-infected and RSMV-infected plants. Using gold particles of two distinct sizes to label the P protein and PI4P, we observed that while PI4P was uniformly distributed in mock-infected cells, RSMV-infected cells exhibited significant enrichment of gold particles within VFs. This enrichment indicates substantial aggregation of both P protein and PI4P within these structures. Conversely, few gold particles were detected in regions resembling virions, and virus-free areas retained a uniform PI4P distribution similar to that in mock-infected cells (<a href="#F7" class="usa-link">Fig. 7, C and D</a>). These findings provide conclusive evidence of substantial PI4P accumulation within RSMV-induced VFs.</p></section><section id="sec10"><h3 class="pmc_sec_title">PI4P enhances viral replication by influencing the P granules size</h3>
<p>To investigate how PI4P regulates the formation of RSMV VFs, we transiently overexpressed OsPI4KA and OsPI4KB in <em>N. benthamiana</em> to elevate cellular PI4P levels (fig. S7A). Confocal microscopy revealed that GFP-P formed larger granules when coexpressed with either OsPI4KA or OsPI4KB compared to the controls (<a href="#F8" class="usa-link">Fig. 8A</a>). Statistical analysis of granule diameters supported the hypothesis that increased PI4P levels promote LLPS of the RSMV P protein (<a href="#F8" class="usa-link">Fig. 8B</a>). Furthermore, we used the TRV system to silence PI4K or ARF1 to inhibit PI4P synthesis and observed GFP-P granule formation in <em>N. benthamiana</em> cells (fig. S7, B to D). The results indicated that silencing PI4K or ARF1 significantly reduced GFP-P granule size compared to controls (<a href="#F8" class="usa-link">Fig. 8, C and D</a>), suggesting that PI4P positively regulates RSMV P protein LLPS in vivo.</p>
<figure class="fig xbox font-sm" id="F8"><h4 class="obj_head">Fig. 8. Modulation of P granule size by PI4P in vivo and in vitro.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f8.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/2145c4075600/sciadv.adx7905-f8.jpg" loading="lazy" height="672" width="750" alt="Fig. 8."></a></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Confocal images depicting GFP-P granules following coinfiltration of GFP-P with PI4KA/B in <em>N. benthamiana</em> leaves. Scale bars, 10 μm. (<strong>B</strong>) Statistical analysis of GFP-P granule diameter in panel (A), with at least 50 GFP-P granules measured. *** denoting a highly significant difference (<em>P</em> &lt; 0.001) based on the Tukey-Kramer post hoc test. (<strong>C</strong>) Confocal images illustrating GFP-P granules in <em>N. benthamiana</em> plants treated with TRV-GUS, TRV-NbARF1, TRV-NbPI4KA, and TRV-NbPI4KB, respectively. Scale bars, 10 μm. (<strong>D</strong>) Statistical analysis of GFP-P granule diameter in panel (C), with a minimum of 150 GFP-P granules measured. *** indicates a highly significant difference (<em>P</em> &lt; 0.001) based on the Tukey-Kramer post hoc test. (<strong>E</strong>) Confocal images of GFP-P droplets in buffers with PI or PI4P, respectively, with buffer concentration at 5 μM in 100 mM NaCl. Scale bars, 10 μm. (<strong>F</strong>) Statistical analysis of GFP-P droplet diameter in panel (E), involving at least 180 GFP-P droplets measured. *** indicating a highly significant difference (<em>P</em> &lt; 0.001) based on the Tukey-Kramer post hoc test. n.s., not significant. (<strong>G</strong>) Confocal images revealing the distribution of PI4P in GFP-P droplets, at a concentration of 10 μM in 100 mM NaCl. GFP-hFAPP1-PH indicates the location of PI4P. Scale bars, 10 μm.</p></figcaption></figure><p>We next investigated the influence of PI4P on the LLPS of the P protein in vitro. Purified GFP-P protein supplemented with PI4P, but not with PI, induced larger spherical LLPS droplets compared to the buffer control (<a href="#F8" class="usa-link">Fig. 8E</a>), resulting in a significant increase in droplet diameter (<a href="#F8" class="usa-link">Fig. 8F</a>). To verify the spatial arrangement of PI4P, we used GFP-hFAPP1-PH, a PI4P-specific probe, alongside BFP-P proteins. Control trials lacking PI4P exhibited no GFP emission within the BFP-P droplets. However, upon the introduction of PI4P, droplet enlargement occurred, accompanied by overlapping fluorescence signals from both GFP and BFP (<a href="#F8" class="usa-link">Fig. 8G</a>). This observation suggests a homogeneous distribution of PI4P and enhanced LLPS of the P protein.</p></section></section><section id="sec11"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>Biomolecular condensates formed through LLPS not only regulate diverse cellular functions but also play an important role in the formation of VFs/inclusion bodies/viroplasms during virus infection. For instance, the N protein of SARS-CoV-2 undergoes LLPS to sequester components of the viral replication machinery (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>). Similarly, the RdRp of HuNoV forms distinct condensates exhibiting LLPS properties, which include sustained polymerase activity and the recruitment of crucial viral replication elements (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). In the case of plant viruses, although not directly implicated in VF formation, the movement protein p26 of pea enation mosaic virus 2 has been observed to undergo phase separation and interact with cellular factors, thereby modulating viral replication (<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>). In addition, the P protein encoded by BYSMV demonstrates LLPS and contributes to viral replication (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>, <a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>). Here, we observed numerous membraneless spherical regions in rice cells infected with RSMV through electron microscopy, with assembled viral RNPs found to radiate outward, indicating that these spherical regions are VFs of RSMV. Further immunoelectron microscopy revealed that the P protein is predominantly localized within the VFs, while its concentration is relatively low in fully assembled RNPs or mature viral particles, suggesting that the P protein is a core component of RSMV VFs. We have also confirmed the LLPS characteristics of the P protein in both in vitro and in vivo experiments, demonstrating its ability to recruit key components for viral replication, including the N protein and viral RNA. These findings underscore the role of the P protein in the construction of RSMV VFs through LLPS.</p>
<p>Previous studies on rhabdoviruses infecting animals and plants, such as RABV (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>) and BYSMV (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>), have indicated that their P proteins contain IDRs capable of undergoing LLPS. This LLPS process in the P protein plays a crucial role in recruiting viral RNAs, viral proteins, and host factors necessary for viral replication, effectively acting as a hub for efficient viral replication. Furthermore, the membraneless organelles formed through phase separation of the P protein may serve to exclude proteins with antiviral properties from the site of viral replication.</p>
<p>The occurrence of protein LLPS is regulated by various factors, including protein concentration, salt concentration, pH, temperature, and solution crowding (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>, <a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). To our knowledge, this is the first report demonstrating the promoting effect of lipids on protein phase separation. We demonstrated that the addition of lipids to the RSMV P protein in vitro LLPS system revealed that PI4P, rather than PI, facilitates the formation and enlargement of P protein droplets. This finding broadens our understanding of the regulatory factors governing protein LLPS and suggests a crucial role for PIPs in viral replication. However, it remains unclear how PIPs regulate the LLPS of viral replication–related proteins and whether PIPs exert a similar promoting effect on the LLPS of other nonviral proteins, warranting further investigation. Recent studies underscore the importance of phase separation in plant disease resistance. Our research demonstrates that PI4P lipids regulate protein phase separation, presenting a potential target for developing disease-resistant crops. However, direct lipid pathway manipulation may compromise plant growth due to their intricate functions. Precise CRISPR editing of the rice <em>RBL1</em> (RESISTANCE TO BLAST1) gene, which modulates PI(4,5)P2, provides a strategic approach to balancing disease resistance and developmental processes (<a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>). Complementary chemical interventions, such as the small molecule Z9, effectively suppress viral nucleoprotein condensate formation, substantially inhibiting viral infection (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>). These advances translate phase separation insights into sophisticated, sustainable strategies for crop protection.</p>
<p>PIPs play a critical role in viral replication; however, viruses and their proteins use diverse mechanisms to hijack PIPs. The first mechanism involves the direct binding of PIPs by membrane-localized viral proteins. For instance, the H7 protein of the vaccinia virus can directly bind to PI3P and PI4P to facilitate viral infection (<a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>). Similarly, the group-specific antigen protein (Gag) of HIV directly interacts with PI(4,5)P2 (<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>). Moreover, the P10 protein of rice black streak dwarf virus can bind to PI(3,5)P2 within its insect host, inhibiting antiviral autophagy and promoting viral replication (<a href="#R62" class="usa-link" aria-describedby="R62"><em>62</em></a>). The second mechanism entails viral proteins recruiting PI kinases to synthesize specific PIPs necessary for viral propagation. For example, the nonstructural protein NS5A of hepatitis C virus (HCV) interacts directly with PI4KA, and mutations at the interaction site notably inhibit HCV replication (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>). In HCV-infected cells, there is a notable increase in PI4P levels without a corresponding elevation in PI4KA transcription, indicating that HCV hijacks PI4KA to replication sites, using its kinase activity to generate more PI4P, which is conducive to viral replication (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>). Similarly, human RSV recruits PI4KB to its replication complex via the N protein to generate PI4P, thereby enhancing viral replication (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>). In addition, the replication protein p33 of TBSV interacts directly with the host Vps34, a type of phosphatidylinositol 3-kinase (PI3K), establishing a viral replication microenvironment enriched with PI3P that facilitates viral replication (<a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>). The third mechanism, akin to the findings of this study, involves viral proteins recruiting cellular trafficking proteins to indirectly recruit PI kinases for localized PIP synthesis, thereby promoting viral replication. TBSV can also exploit the retromer complex proteins via its replication protein p33, facilitating the transport of phosphatidylserine decarboxylase Psd2, PI3K, and PI4K through the retromer trafficking pathway to the viral replication complex (<a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>). Consequently, a complex enriched with phosphatidylethanolamine, PI3P, and PI4P is formed, ultimately enhancing viral replication (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>).</p>
<p>In eukaryotic cells, the membrane-bound ARF1 protein plays a crucial role in the replication of various RNA viruses. This study confirmed the interaction between RSMV P protein and OsARF1, which leads to the indirect recruitment of OsPI4KB for the formation of RSMV VFs. Both silencing and overexpressing OsARF1 were found to be detrimental to RSMV replication. The inhibition of OsARF1 expression disrupts the recruitment of the P protein to PI4KB, thereby inhibiting viral infection. In addition, ARF1 not only functions as a key protein in COP I vesicle transport but also plays a critical role in plant defense (<a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>). Overexpression of ARF1 in tobacco and rice plants enhances their resistance, leading to a notable increase in the expression of resistance-related genes, up-regulation of salicylic acid accumulation, and induction of cell death (<a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>, <a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>). Furthermore, the overexpression of ARF1 in both rice and tobacco plants substantially improves resistance against pathogens, which may explain the observed inhibition of RSMV infection in rice plants overexpressing OsARF1C (<a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>). On the other hand, the stable transport of PI4K by ARF1 relies on the switching of its activity states (associated with GDP or GTP binding). The inactive mutant (OsARF1C<sup>T31N</sup>) fails to transport PI4K, while the constitutively active mutant (OsARF1C<sup>Q71L</sup>) disrupts Golgi and ER integrity (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>), also leading to decreased PI4K recruitment ability. Our findings convincingly demonstrate that both mutants of OsARF1C lose the capacity to facilitate the indirect recruitment of OsPI4KB by P protein, resulting in the inhibition of RSMV infection. Moreover, silencing ARF1 disrupts intracellular vesicular trafficking, hindering the transportation of cargo proteins and small molecules within cells, thereby impeding plant development and potentially leading to premature death (<a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>). The study also observed stunted growth in <em>OsARF1-RNAi</em> plants, with diminished plant vigor possibly compromising viral replication. Therefore, maintaining the homeostasis of ARF1-like vesicle transport proteins in host cells is crucial for successful infection of RSMV.</p>
<p>PI4P is dynamically generated in vivo primarily by phosphorylating PIP through PI4K. Subsequently, Sac1 phosphatase recognizes and dephosphorylates PI4P back to PIP. In rice, GH1 functions as the Sac1 homolog, negatively regulating PI4P (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>). The <em>gh1</em> mutant rice plants showed significantly increased RSMV accumulation, indicating a role for PI4P in RSMV replication. However, overexpressing <em>GH1</em> did not notably impede viral replication. There are two possibilities to consider. First, the basal level of RSMV infection does not rely on a high accumulation of cytoplasmic PI4P. Even with <em>GH1</em> overexpression leading to a decrease in cellular PI4P content, the remaining PI4P is adequate for supporting the establishment of RSMV VFs and viral infection. Second, the P protein of RSMV indirectly recruits PI4KB to onsite synthesis of PI4P at the VFs to promote viral replication. The LLPS of the P protein aids in sequestering PI4P within the VFs. Therefore, despite <em>GH1</em> overexpression affecting only free PI4P in the cell, it does not disrupt the PI4P within the VFs, thus leaving RSMV infection unaffected.</p>
<p>In summary, our results demonstrate that the RSMV P protein directly interacts with OsARF1C, a crucial component of the COP I vesicle transport pathway essential for viral replication. This interaction indirectly recruits OsARF1C’s partner, OsPI4KB, which synthesizes PI4P and binds to the P protein. Meanwhile, the P protein modulates its aggregates and LLPS droplets via PI4P, leading to the formation of a larger replication site for the virus (<a href="#F9" class="usa-link">Fig. 9</a>). These findings provide potential targets for effectively controlling similar viruses, offering promising strategies for disease prevention and control.</p>
<figure class="fig xbox font-sm" id="F9"><h3 class="obj_head">Fig. 9. A proposed model illustrating how RSMV hijacks host trafficking and lipid synthesis for VF assembly.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366689_sciadv.adx7905-f9.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0cc6/12366689/9ac8a2137571/sciadv.adx7905-f9.jpg" loading="lazy" height="543" width="700" alt="Fig. 9."></a></p>
<div class="p text-right font-secondary"><a href="figure/F9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>In healthy rice cells (left), ARF1 and COP I coats facilitate trafficking to the ERGIC and Golgi/TGN compartments, with PI4KB enzymes recruited to Golgi membranes by OsARF1 to produce PI4P lipids. In RSMV-infected rice cells (right), the P protein recruits OsARF1-PI4KB to establish a PI4P-enriched RSMV VF, binding PI4P to induce P LLPS droplets that expand the VF area, thereby enhancing RSMV replication efficiency.</p></figcaption></figure></section><section id="sec12"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec13"><h3 class="pmc_sec_title">Plant materials and growth conditions</h3>
<p>The rice plants used in this study for overexpression, RNAi, and the mutant lines of <em>OsARF1C</em> and <em>OsPI4KB</em> were of the cultivar Zhonghua 11 background. In contrast, the transgenic lines of RSMV P were derived from the cultivar Nipponbare background, while the overexpression and mutant lines associated with <em>GH1</em> were based on the cultivar GC background. Transgenic rice plants overexpressing <em>OsARF1C-OE</em>, <em>OsARF1C</em><sup>T31N</sup><em>-OE</em>, <em>OsARF1C</em><sup>Q71L</sup><em>-OE</em>, and <em>OsARF1-RNAi</em> were generated in this study. The <em>ospi4kb</em> mutant lines were developed using CRISPR-Cas9 technology at the Biogle Genome Editing Center (Jiangsu, China). Seeds of the RSMV P transgenic lines were obtained as described previously (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). In addition, seeds for <em>GH1-OE</em> and <em>gh1</em> mutant lines were provided by H. Lin at the Institute for Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>). All rice plants were grown in a greenhouse maintained at 28°C with 60 ± 5% relative humidity and exposure to natural sunlight. <em>N. benthamiana</em> plants, including transgenic lines expressing RFP–histone 2B (H2B) as a nuclear marker, were cultivated in growth chambers at 25°C, under a photoperiod of 16 hours of light followed by 8 hours of darkness.</p></section><section id="sec14"><h3 class="pmc_sec_title">RSMV inoculation</h3>
<p>RSMV was maintained through serial transmission by leafhoppers (<em>Recilia dorsalis</em>) within rice plants, as previously described (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). Two-week-old seedlings were exposed to either RSMV-carrying or virus-free (mock) leafhoppers, with two to three insects per plant for a duration of 3 days. Following this exposure, the presence of the virus in RSMV-infected rice plants or in viruliferous <em>R. dorsalis</em> was confirmed using RT-qPCR or reverse transcription-recombinase aided amplification (RT-RAA)–CRISPR-Cas12a visual detection as previously described (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>, <a href="#R67" class="usa-link" aria-describedby="R67"><em>67</em></a>).</p></section><section id="sec15"><h3 class="pmc_sec_title">Plasmid constructions</h3>
<p>All primers used for cloning experiments are listed in table S1. The coding sequences of the RSMV genes were amplified using cDNA derived from RSMV-infected rice plants as a template. The coding sequence of hFAPP1-PH from humans was amplified using the cDNA of human tissues as a template. The RSMV MR system, which includes pRSMV-MR, pGD-NPL, pGD-VSRs, pCambia1300-GFP-P, and pCambia1300-N-GFP, was previously constructed (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). All genes from rice and tobacco were amplified using cDNA extracted from the respective plants. Point mutants of OsARF1C were generated using the QuickChange mutagenesis method.</p>
<p>To obtain the plasmids pGADT7-AD-OsARF1C, pGBKT7-BD-P, pGEXT-P, pMBP-OsARF1C, pRHV-cMYC-OsARF1C/OsARF1C<sup>T31N</sup>/OsARF1C<sup>Q71L</sup>, pTRV2-NbARF1/NbPI4KB/NbPI4KA, pCambia-1300-GFP-hFAPP1-PH, pCV-nYFP-OsARF1B/D/F, and pCV-cYFP-OsPI4KB, the full-length coding sequences of the targeted genes were cloned and individually inserted into the vectors using the In-Fusion cloning kit (Vazyme Biotech, C112). In addition, to obtain the plasmids pGD-BFP-P, pGD-OsARF1C/OsARF1C<sup>T31N</sup>/OsARF1-C<sup>Q71L</sup>-RFP, pGD-GFP-OsPI4KB, pET28a-GFP-hFAPP1-PH, pET28a-GFP-P, and pET28a-OsARF1C-GFP, the full-length coding sequences of the targeted genes were cloned and overlapped with GFP, BFP, or RFP sequences and subsequently inserted into the vector using the In-Fusion cloning kit (Vazyme Biotech, C112).</p>
<p>To obtain pEarleyGate202-YC-P, pEarleyGate202-YN-OsARF1C, pBA-35S-FlagMyc4-OsARF1C/OsARF1C<sup>T31N</sup>/OsARF1C<sup>Q71L</sup>, and pEarleyGate102-OsARF1C-YFP, the full-length coding sequences of P, OsARF1C, OsARF1C<sup>T31N</sup>, or OsARF1C<sup>Q71L</sup> were individually cloned into the pENTR/D-TOPO vector (Thermo Fisher Scientific, K240020). Subsequently, the cloned sequences were transferred into the corresponding destination vectors using a Gateway LR reaction kit in accordance with the manufacturer’s instructions (Thermo Fisher Scientific, K240020).</p></section><section id="sec16"><h3 class="pmc_sec_title">RNA extraction and RT-qPCR</h3>
<p>Total RNA was extracted from rice tissue samples using the Total RNA Extraction Reagent (Vazyme Biotech, R401). To detect relative gene expression levels in rice and <em>N. benthamiana</em>, we synthesized cDNA from the isolated total RNA using an oligo(dT) primer and reverse transcriptase (Vazyme Biotech, R312-01). Quantitative reactions were conducted on a CFX96 Touch Real-Time PCR Detection System (Bio-Rad), using the ChamQ SYBR qPCR Master Mix (Vazyme Biotech, Q712-02). At least three independent biological replicates were performed for each experimental condition, with each replicate analyzed in technical triplicate. For assessing the accumulation levels of MR and RSMV genomic RNA in <em>N. benthamiana</em> and rice, one-step RT-qPCR reactions were performed by amplifying fragments that spanned two genes to detect replication levels, using the HiScript II One-Step qRT-PCR SYBR Green Kit (Vazyme Biotech, P612-01). The expression levels of <em>OsEF1a</em> (for rice) or <em>NbPP2A</em> (for <em>N. benthamiana</em>) served as internal controls, and relative RNA accumulation levels were calculated using the 2<sup>-ΔΔC(t)</sup> method (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>). Primers used for RT-qPCR are listed in table S1.</p></section><section id="sec17"><h3 class="pmc_sec_title">Transmission electron microscope observation</h3>
<p>RSMV infected leaves from rice plants were fixed, dehydrated, and embedded, and thin sections were cut as described previously (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). Ultrathin sections (~80 nm) were prepared from the embedded tissues using a Leica UC7 Ultramicrotome and were collected on 3-mm copper slot grids. These sections were then stained with uranyl acetate and lead citrate solution. The stained sections were examined and imaged using a Thermo Fisher Scientific Talos L120C transmission electron microscope operating at an accelerating voltage of 120 kV.</p>
<p>For immunoelectron microscopy, rice leaves were prefixed with 4% paraformaldehyde (in 100 mM phosphate buffer, pH 7.0) and 0.1% glutaraldehyde for 2 hours, followed by postfixation with 2% paraformaldehyde overnight at 4°C. After treatment with 0.1 M glycine for 1 hour, the fixed leaves were dehydrated in graded ethanol at 4°C and embedded in LR White (GE Healthcare, 14381). The embedded leaves were polymerized at −20°C using ultraviolet light. Sections measuring 100 nm were cut and mounted on slot nickel grids using a Leica UC7 Ultramicrotome. The sections were subsequently mounted on formal-supported nickel single slot grids. Initially, the sections were incubated in 100 mM phosphate buffer for 15 min, followed by incubation in a blocking buffer [1% bovine serum albumin (Sigma-Aldrich, V900933) in 100 mM phosphate buffer, pH 7.0] for another 15 min. For single immunosorbent electron microscopy observation of RSMV P protein, sections were incubated with a rabbit antibody specific to RSMV P (1:100, v:v) for 1 hour at room temperature, followed by incubation with a 12-nm gold-conjugated goat anti-rabbit immunoglobulin G (IgG) secondary antibody (Jackson, 111-205-144) solution (1:100, v:v) for an additional hour. In the case of double immunosorbent electron microscopy observation for RSMV P and PI4P, sections were incubated with a mouse antibody against PI4P (Echelon Biosciences, Z-P004; 1 mg/ml) and a rabbit antibody specific to RSMV P (1:100, v:v). Primary and secondary antibodies were visualized using 10-nm colloidal gold goat anti-mouse IgG (Sigma-Aldrich, G7777) and 18-nm colloidal gold donkey anti-rabbit IgG (Jackson, 711-215-152), respectively. After labeling, sections were stained with uranyl acetate and lead citrate prior to examination under the Talos L120C electron microscope. To count the numbers of P and PI4P labeling gold particles, we set a threshold for each image based on signal strength. Images were then converted to binary format and masked, and the number of particles was counted using Fiji/ImageJ.</p></section><section id="sec18"><h3 class="pmc_sec_title">Recombinant protein expression and purification</h3>
<p>Recombinant proteins were expressed in <em>Escherichia coli</em> Rosetta (DE3) cells by cultivating transformed cells in LB medium at 37°C until reaching an OD<sub>600</sub> (optical density at 600 nm) of 0.6. Protein expression was subsequently induced by adding 0.5 mM isopropyl-β-<span class="font-variant-small-caps">d</span>-1-thiogalactopyranoside and incubating overnight at 16°C. Protein purification was then conducted at 4°C.</p>
<p>For the purification of GFP, GFP-P, BFP-P, OsARF1-GFP, OsARF1C-mCherry, mCherry-N, and GFP-hFAPP1-PH, the corresponding bacterial cells were collected and resuspended in a buffer containing 30 mM tris-HCl (pH 6.8), 500 mM NaCl, 10% glycerol, and 20 mM imidazole. The cells were lysed by sonication, and after centrifugation, the supernatants were affinity-extracted using HisPur Ni-NTA Resin (Thermo Fisher Scientific, 88221) and subsequently dialyzed into a protein storage buffer consisting of 30 mM tris-HCl (pH 6.8) and 500 mM NaCl. The proteins were then further purified by size exclusion chromatography with a Superdex 200 column (GE Healthcare, USA) in the protein storage buffer. The peak fractions containing the proteins were collected and concentrated using a 30-kDa molecular weight cutoff (MWCO) Centricon (Millipore, UFC9030-08). The final purified protein concentration was determined using the Bradford assay and SDS–polyacrylamide gel electrophoresis (PAGE). The purified proteins were then snap-frozen in small aliquots using liquid nitrogen and stored at −80°C.</p>
<p>To purify glutathione <em>S</em>-transferase (GST)–P and maltose binding protein (MBP)-OsARF1C proteins, bacterial cells were collected, resuspended in 1× phosphate-buffered saline (PBS) buffer (pH 7.0), and lysed by sonication. Following centrifugation, the supernatants were affinity-extracted using Glutathione Sepharose 4B (GE Healthcare, 17075601) and Amylose Resin (New England Biolabs, E8021V), respectively. Subsequently, the purified proteins were concentrated using a 30-kDa MWCO Centricon (Millipore, UFC903008).</p></section><section id="sec19"><h3 class="pmc_sec_title">In vitro liquid droplet reconstitution assay</h3>
<p>For the phase separation of GFP-P or BFP-P, the proteins were diluted to the desired concentrations using 30 mM tris-HCl (pH 6.8), with a final NaCl concentration of 50 mM unless otherwise specified. For the phase separation of mCherry-N/OsARF1C-mCherry in combination with GFP-P, the proteins were mixed and then diluted to the desired concentrations in 30 mM tris-HCl (pH 6.8). In the case of PI4P with GFP-P, GFP-P was diluted in a buffer containing PI4P [30 mM tris-HCl (pH 6.8) and 100 μM PI4P/PI (Echelon Biosciences, P-4016/P-0016)]. Similarly, for the phase separation of PI4P with GFP-hFAPP1-PH and BFP-P, the two proteins were mixed and diluted in a buffer containing PI4P [30 mM tris-HCl (pH 6.8) and 100 μM PI4P/PI (Echelon Biosciences, P-4016/P-0016)]. Thoroughly mixed reactions were transferred into a 384-well plate, and after 1 hour of incubation at room temperature, the resulting droplets in the wells were observed using confocal microscopy (Leica TCS SP8). For the synthesis of Cy5-labeled RNAs, 5 to 10 μg of DNA templates containing the T7 promoter–driven RSMV trailer sequence were used as templates for in vitro transcription, using the T7 High Yield RNA Synthesis Kit (Yeasen, 10623ES10) according to the manufacturer’s protocols. The final concentrations of adenosine triphosphate, cytidine triphosphate, GTP, uridine triphosphate (UTP), and Cy5-UTP (Yeasen, 10623ES10) in the reaction mixtures were 1.75, 1.75, 1.75, 0.875, and 0.175 mM, respectively. The reactants were mixed gently and incubated at 37°C for 2 hours, followed by the addition of RNase-Free DNase I for 15 min to eliminate DNA templates. Subsequently, 0.1 volumes of 3 M sodium acetate (pH 5.2) and 2.5 volumes of 100% ethanol were added, and the mixture was stored at −20°C for over 4 hours. After centrifugation, the precipitated RNA was washed with 75% ethanol, resuspended in nuclease-free water, and heated at 95°C for 5 min. The phase separation of GFP-P, mCherry-N, and Cy5-labeled RNAs was performed by mixing the indicated proteins with Cy5-labeled RNAs, followed by dilution with buffer [30 mM tris-HCl (pH 7.5) and 1 mM dithiothreitol (DTT)] to achieve the desired concentrations.</p>
<p>BFP, GFP, and mCherry fluorescence were excited/visualized under 405 nm/415 to 488 nm, 488 nm/500 to 530 nm, and 543 nm/588 to 641 nm, respectively. To quantify the number and size of GFP-P droplets, we established a threshold for each image based on signal strength. Subsequently, the images were converted to binary format and masked, and the number and size of droplets were quantified using Fiji/ImageJ.</p></section><section id="sec20"><h3 class="pmc_sec_title">FRAP assay</h3>
<p>FRAP of GFP-P granules in <em>N. benthamiana</em> leaf epidermal cells was conducted using a Leica TCS SP8 confocal microscope equipped with a 40× objective lens. The GFP-P granules were subjected to photobleaching with a laser intensity of 100% at 488 nm over 50 iterations. Fluorescence recovery was monitored every 3 s for a duration of 120 s following the bleaching process. Ten droplets were used to obtain the FRAP recovery curves.</p>
<p>FRAP of GFP-P droplets in a 384-well plate was conducted using a Leica TCS SP8 confocal microscope equipped with a 100× oil-immersion objective. The central region of each GFP-P droplet was subjected to bleaching using a laser intensity of 100% at 488 nm, with a total of 100 iterations. Fluorescence recovery was monitored every 50 s for a duration of 550 s following the bleaching process. Ten droplets were used to obtain the FRAP recovery curves.</p></section><section id="sec21"><h3 class="pmc_sec_title">Y2H assay</h3>
<p>For the Y2H assay, different combinations of pGBK and pGAD plasmids were transformed into the yeast strain Y2HGold. The transformants were initially cultivated on synthetic defined (SD)–Trp/−Leu plates for 2 days. Subsequently, the transformants were cultivated on SD/-Leu/-Trp (SD-L-T) medium and lastly cultivated on SD/-Leu/-Trp-His-Ade (SD-L-T-H-A) selection medium to determine the protein-protein interaction. Yeast cells were photographed 3 days postincubation at 30°C.</p></section><section id="sec22"><h3 class="pmc_sec_title">GST pull-down assay</h3>
<p>A total of 3 μg of purified GST-P protein was mixed with 6 μg of purified MBP-OsARF1C or MBP protein and incubated with 20 μl of Glutathione Sepharose 4B beads (GE Healthcare, 17075601) for 2 hours at 4°C in binding buffer [50 mM tris-HCl (pH 7.5), 150 mM NaCl, and 0.25% Triton X-100]. After washing four times with binding buffer, the resin-bound proteins were released by boiling for 5 min, and the protein samples were analyzed using an immunoblot assay with anti-GST and anti-MBP antibodies.</p></section><section id="sec23"><h3 class="pmc_sec_title">Co-IP assay</h3>
<p>Immunoprecipitation (IP) was conducted using protein extracts derived from <em>OsARF1C-OE</em> or WT plants with RSMV or mock infection. At 15 dpi, the rice leaves were ground in liquid nitrogen, and protein samples were extracted using an IP buffer consisting of 40 mM tris-HCl (pH 6.8), 150 mM NaCl, 5 mM MgCl<sub>2</sub>, 2 mM EDTA, 5 mM DTT, 2% glycerol, 0.1% Triton X-100, and protease inhibitor cocktail from 5 g of leaves. Following a 10-min incubation at 4°C, the mixtures were centrifuged at 12,000<em>g</em> for 10 min. The supernatant was then mixed with 1 μl of anti-P antibody and incubated for 2 hours at 4°C, followed by the addition of 20 μl of Protein A+G Agarose (Beyotime Technology, P2055). The immunoprecipitants were washed four times with 1× PBS and resuspended in 20 μl of 2× SDS-PAGE loading buffer, which consisted of 500 mM tris-HCl (pH 6.8), 50% glycerol, 10% SDS, 2% β-mercaptoethanol, and 1% bromophenol blue. The protein samples were then boiled at 95°C for 10 min and separated using a 10% SDS-PAGE gel for subsequent Western blot analysis.</p></section><section id="sec24"><h3 class="pmc_sec_title">Subcellular localization and BiFC assay</h3>
<p><em>N. benthamiana</em> and rice protoplasts were used in BiFC and subcellular localization assays. Various combinations of expression vectors were transiently expressed in <em>N. benthamiana</em> leaves via agroinfiltration for 55 to 60 hours or transfected into rice protoplasts for 12 to 16 hours. YFP fluorescence indicated in vivo protein interactions. RFP-H2B, expressed in transgenic <em>N. benthamiana</em> leaves, served as a nuclear marker. Cyan fluorescent protein/BFP, GFP, YFP, and RFP/mCherry fluorescence were excited/visualized under 405 nm/415 to 488 nm, 488 nm/500 to 530 nm, 514 nm/525 to 580 nm, and 543 nm/588 to 641 nm, respectively.</p></section><section id="sec25"><h3 class="pmc_sec_title">Chemical treatment</h3>
<p>To inhibit GFP-P droplets in vivo and vitro, we treated 10% of HEX for 5 min in GFP-P–expressed <em>N. benthamiana</em> leaves and added 5% of HEX to GFP-P–contained elution buffer for 1 min before observation.</p>
<p><em>N. benthamiana</em> leaves were infiltrated with the RSMV MR system to inhibit ARF1 and PI4KB activity. At 5 dpi, the leaves received treatments with dimethyl sulfoxide (DMSO), BFA (Aladdin, B102375; final concentration of 50 μg/ml), or PIK93 (Aladdin, P126888; final concentration of 100 μM). Following a 24-hour incubation, the cells were examined using confocal microscopy.</p></section><section id="sec26"><h3 class="pmc_sec_title">P protein–PI4P complex modeling</h3>
<p>The Chai-1 modeling framework (<a href="http://www.chaidiscovery.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.chaidiscovery.com</a>) was used for predicting protein-ligand complex structures. Protein sequences in FASTA format and ligands in SMILES format were inputted into Chai-1, which combines protein folding prediction with ligand docking simulation. Five candidate structures were generated for each complex and ranked based on structural feasibility and interaction quality. Key binding interactions such as hydrogen bonds, hydrophobic contacts, and potential covalent binding sites were identified. The final model was visualized using ChimeraX 1.9, highlighting binding sites and key interactions to elucidate the ligand’s binding mode effectively.</p></section><section id="sec27"><h3 class="pmc_sec_title">Protein-lipid binding assay</h3>
<p>PI4P or PI was dissolved in a buffer consisting of chloroform, methanol, and water in a 1:2:0.8 ratio, achieving a concentration of 1 mM. This solution was then spotted onto a nitrocellulose membrane (Whatman Protran, BA83) at a predetermined concentration. Subsequently, the nitrocellulose membranes were blocked with 1% fatty acid–free milk in 1× tris-buffered saline (20 mM tris and 150 mM NaCl, pH 8.0) for 1 hour. Following the blocking step, the membranes were incubated with purified His-tagged proteins (GFP-P, ARF1C-GFP, GFP-hFAPP1-PH, and GFP) at a concentration of 2 μg/μl for 3 hours at room temperature. After washing, the membranes were incubated with mouse monoclonal anti–6× His tag (Abmart) for 2 hours, followed by incubation with goat anti-mouse IgG–horseradish peroxidase antibody (Mei5bio) for 1 hour at room temperature. Last, the strips were analyzed using a chemiluminescence gel imaging system (Bio-Rad, USA).</p></section><section id="sec28"><h3 class="pmc_sec_title">Western blot and protein quantification analysis</h3>
<p>Individual protein samples were extracted using radioimmunoprecipitation assay lysis buffer. The samples were centrifuged at 13,400<em>g</em> for 2 min, and the upper liquid phase of each sample was analyzed by SDS-PAGE. The PageRuler Prestained Protein Ladder (Thermo Fisher Scientific, 26616) was used to determine the protein sizes. The separated protein bands were transferred to polyvinylidene difluoride membranes (Millipore, Billerica, MA) and detected using antibodies against RSMV N (from J. Wu), RSMV P (generated in this study, produced by Abmart), His (Abmart, <a href="https://www.ncbi.nlm.nih.gov/nuccore/M30111" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">M30111</a>), GST (Abmart, <a href="https://www.ncbi.nlm.nih.gov/nuccore/M20007" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">M20007</a>), MBP (Abmart, <a href="https://www.ncbi.nlm.nih.gov/nuccore/M20051" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">M20051</a>), Myc (Abmart, <a href="https://www.ncbi.nlm.nih.gov/nuccore/M20002" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">M20002</a>), GFP (Abmart, M20004), RFP (ABclonel, WH133973), tubulin (Abmart, <a href="https://www.ncbi.nlm.nih.gov/nuccore/M20045" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">M20045</a>), or actin (Abmart, <a href="https://www.ncbi.nlm.nih.gov/nuccore/M20009" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">M20009</a>). For protein band quantification, the signal intensity was quantified using Fiji/ImageJ. The actin or tubulin bands served as the loading control, and the intensity of individual protein bands was normalized against that of the actin band from the same sample.</p></section><section id="sec29"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p>All experiments were independently repeated more than three times, yielding consistent results. No data were excluded from our studies. Differences were analyzed using either one-way or two-way analysis of variance (ANOVA), followed by Tukey’s multiple comparisons test. A <em>P</em> value of less than 0.05 was considered significant. All analyses were conducted using SPSS version 20 (SPSS, Inc., Chicago, IL, USA).</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank H. Lin from the Institute for Plant Physiology and Ecology at the Chinese Academy of Sciences for providing the <em>gh1</em> and <em>GH1-OE</em> rice seeds. We also thank J. Wu from Zhejiang University for supplying the specific antibody for the RSMV N protein.</p>
<section id="sec30"><p><strong>Funding:</strong> National Natural Science Foundation of China 32222071 (T.Z.). National Key Research and Development Program of China 2022YFA1304400 (T.Z.). Guangzhou Science and Technology Program 2023B03J1313 (T.Z.).</p></section><section id="sec31"><p><strong>Author contributions:</strong> Conceptualization: T.Z. Methodology: Z.W., J.Z., J.H., G.S., and T.Z. Investigation: Z.W., J.Z., J.H., X.S., X.C., S.C., Z.Y., C.L., Z.L., and X.H. Formal analysis: Z.W., J.Z., J.H., and G.S. Resources: T.Z. and G.Z. Data curation: Z.W., J.Z., J.H., and G.S. Writing—original draft: Z.W. and T.Z. Writing—review and editing: T.Z., G.Z., Q.X., and X.Y. Visualization: Z.W. and T.Z. Supervision: T.Z., G.Z., and X.Y. Project administration: T.Z. and G.Z. Funding acquisition: T.Z.</p></section><section id="sec32"><p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec33"><p><strong>Data and materials availability:</strong> All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials.</p></section></section><section id="sec34"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec35"><h3 class="pmc_sec_title">The PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Figs. S1 to S7</p>
<p>Table S1</p>
<p>Legend for movie S1</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366689/bin/sciadv.adx7905_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adx7905_sm.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="sec36"><h3 class="pmc_sec_title">Other Supplementary Material for this manuscript includes the following:</h3>
<section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p"><p>Movie S1</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366689/bin/sciadv.adx7905_movie_s1.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adx7905_movie_s1.zip</a><sup> (1.1MB, zip) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Alberti S., 
Phase separation in biology. Curr. Biol.
27, 
R1097–R1102 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cub.2017.08.069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29065286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Phase%20separation%20in%20biology&amp;volume=27&amp;publication_year=2017&amp;pages=R1097-R1102&amp;pmid=29065286&amp;doi=10.1016/j.cub.2017.08.069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Banani S. F., Lee H. O., Hyman A. A., Rosen M. K., 
Biomolecular condensates: Organizers of cellular biochemistry. Nat. Rev. Mol. Cell Biol.
18, 
285–298 (2017).
</cite> [<a href="https://doi.org/10.1038/nrm.2017.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7434221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28225081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=Biomolecular%20condensates:%20Organizers%20of%20cellular%20biochemistry&amp;volume=18&amp;publication_year=2017&amp;pages=285-298&amp;pmid=28225081&amp;doi=10.1038/nrm.2017.7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Alberti S., Gladfelter A., Mittag T., 
Considerations and challenges in studying liquid-liquid phase separation and biomolecular condensates. Cell
176, 
419–434 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.12.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6445271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30682370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Considerations%20and%20challenges%20in%20studying%20liquid-liquid%20phase%20separation%20and%20biomolecular%20condensates&amp;volume=176&amp;publication_year=2019&amp;pages=419-434&amp;pmid=30682370&amp;doi=10.1016/j.cell.2018.12.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Bamunusinghe D., Hemenway C. L., Nelson R. S., Sanderfoot A. A., Ye C. M., Silva M. A. T., Payton M., Verchot-Lubicz J., 
Analysis of potato virus X replicase and TGBp3 subcellular locations. Virology
393, 
272–285 (2009).
</cite> [<a href="https://doi.org/10.1016/j.virol.2009.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19729179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Analysis%20of%20potato%20virus%20X%20replicase%20and%20TGBp3%20subcellular%20locations&amp;volume=393&amp;publication_year=2009&amp;pages=272-285&amp;pmid=19729179&amp;doi=10.1016/j.virol.2009.08.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Sagan S. M., Weber S. C., 
Let’s phase it: Viruses are master architects of biomolecular condensates. Trends Biochem. Sci.
48, 
229–243 (2023).
</cite> [<a href="https://doi.org/10.1016/j.tibs.2022.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36272892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=Let%E2%80%99s%20phase%20it:%20Viruses%20are%20master%20architects%20of%20biomolecular%20condensates&amp;volume=48&amp;publication_year=2023&amp;pages=229-243&amp;pmid=36272892&amp;doi=10.1016/j.tibs.2022.09.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Rhine K., Vidaurre V., Myong S., 
RNA Droplets. Annu. Rev. Biophys.
49, 
247–265 (2020).
</cite> [<a href="https://doi.org/10.1146/annurev-biophys-052118-115508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7695521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32040349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Biophys.&amp;title=RNA%20Droplets&amp;volume=49&amp;publication_year=2020&amp;pages=247-265&amp;pmid=32040349&amp;doi=10.1146/annurev-biophys-052118-115508&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Kato M., Han T. W., Xie S., Shi K., Du X., Wu L. C., Mirzaei H., Goldsmith E. J., Longgood J., Pei J., Grishin N. V., Frantz D. E., Schneider J. W., Chen S., Li L., Sawaya M. R., Eisenberg D., Tycko R., McKnight S. L., 
Cell-free formation of RNA granules: Low complexity sequence domains form dynamic fibers within hydrogels. Cell
149, 
753–767 (2012).
</cite> [<a href="https://doi.org/10.1016/j.cell.2012.04.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6347373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22579281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Cell-free%20formation%20of%20RNA%20granules:%20Low%20complexity%20sequence%20domains%20form%20dynamic%20fibers%20within%20hydrogels&amp;volume=149&amp;publication_year=2012&amp;pages=753-767&amp;pmid=22579281&amp;doi=10.1016/j.cell.2012.04.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Heinrich B. S., Cureton D. K., Rahmeh A. A., Whelan S. P., 
Protein expression redirects vesicular stomatitis virus RNA synthesis to cytoplasmic inclusions. PLOS Pathog.
6, 
e1000958 (2010).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1000958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2891829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20585632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog.&amp;title=Protein%20expression%20redirects%20vesicular%20stomatitis%20virus%20RNA%20synthesis%20to%20cytoplasmic%20inclusions&amp;volume=6&amp;publication_year=2010&amp;pages=e1000958&amp;pmid=20585632&amp;doi=10.1371/journal.ppat.1000958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Dolnik O., Gerresheim G. K., Biedenkopf N., 
New perspectives on the biogenesis of viral inclusion bodies in negative-sense RNA virus infections. Cells
10, 
1460 (2021).
</cite> [<a href="https://doi.org/10.3390/cells10061460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8230417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34200781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=New%20perspectives%20on%20the%20biogenesis%20of%20viral%20inclusion%20bodies%20in%20negative-sense%20RNA%20virus%20infections&amp;volume=10&amp;publication_year=2021&amp;pages=1460&amp;pmid=34200781&amp;doi=10.3390/cells10061460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Savastano A., Ibáñez de Opakua A., Rankovic M., Zweckstetter M., 
Nucleocapsid protein of SARS-CoV-2 phase separates into RNA-rich polymerase-containing condensates. Nat. Commun.
11, 
6041 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-19843-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7699647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33247108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Nucleocapsid%20protein%20of%20SARS-CoV-2%20phase%20separates%20into%20RNA-rich%20polymerase-containing%20condensates&amp;volume=11&amp;publication_year=2020&amp;pages=6041&amp;pmid=33247108&amp;doi=10.1038/s41467-020-19843-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Wang J., Shi C., Xu Q., Yin H., 
SARS-CoV-2 nucleocapsid protein undergoes liquid-liquid phase separation into stress granules through its N-terminal intrinsically disordered region. Cell Discov.
7, 
5 (2021).
</cite> [<a href="https://doi.org/10.1038/s41421-020-00240-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7817956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33479219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Discov.&amp;title=SARS-CoV-2%20nucleocapsid%20protein%20undergoes%20liquid-liquid%20phase%20separation%20into%20stress%20granules%20through%20its%20N-terminal%20intrinsically%20disordered%20region&amp;volume=7&amp;publication_year=2021&amp;pages=5&amp;pmid=33479219&amp;doi=10.1038/s41421-020-00240-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Chen H., Cui Y., Han X., Hu W., Sun M., Zhang Y., Wang P. H., Song G., Chen W., Lou J., 
Liquid-liquid phase separation by SARS-CoV-2 nucleocapsid protein and RNA. Cell Res.
30, 
1143–1145 (2020).
</cite> [<a href="https://doi.org/10.1038/s41422-020-00408-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7477871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32901111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=Liquid-liquid%20phase%20separation%20by%20SARS-CoV-2%20nucleocapsid%20protein%20and%20RNA&amp;volume=30&amp;publication_year=2020&amp;pages=1143-1145&amp;pmid=32901111&amp;doi=10.1038/s41422-020-00408-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Kaundal S., Anish R., Ayyar B. V., Shanker S., Kaur G., Crawford S. E., Pollet J., Stossi F., Estes M. K., Prasad B. V. V., 
RNA-dependent RNA polymerase of predominant human norovirus forms liquid-liquid phase condensates as viral replication factories. Sci. Adv.
10, 
eadp9333 (2024).
</cite> [<a href="https://doi.org/10.1126/sciadv.adp9333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11661447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39705355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=RNA-dependent%20RNA%20polymerase%20of%20predominant%20human%20norovirus%20forms%20liquid-liquid%20phase%20condensates%20as%20viral%20replication%20factories&amp;volume=10&amp;publication_year=2024&amp;pages=eadp9333&amp;pmid=39705355&amp;doi=10.1126/sciadv.adp9333&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Fang J., Castillon G., Phan S., McArdle S., Hariharan C., Adams A., Ellisman M. H., Deniz A. A., Saphire E. O., 
Spatial and functional arrangement of Ebola virus polymerase inside phase-separated viral factories. Nat. Commun.
14, 
4159 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-39821-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10345124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37443171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Spatial%20and%20functional%20arrangement%20of%20Ebola%20virus%20polymerase%20inside%20phase-separated%20viral%20factories&amp;volume=14&amp;publication_year=2023&amp;pages=4159&amp;pmid=37443171&amp;doi=10.1038/s41467-023-39821-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Heinrich B. S., Maliga Z., Stein D. A., Hyman A. A., Whelan S. P. J., 
Phase transitions drive the formation of vesicular stomatitis virus replication compartments. mBio
9, 
e02290-17 (2018).
</cite> [<a href="https://doi.org/10.1128/mBio.02290-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6123442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30181255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Phase%20transitions%20drive%20the%20formation%20of%20vesicular%20stomatitis%20virus%20replication%20compartments&amp;volume=9&amp;publication_year=2018&amp;pages=e02290-17&amp;pmid=30181255&amp;doi=10.1128/mBio.02290-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Galloux M., Risso-Ballester J., Richard C. A., Fix J., Rameix-Welti M. A., Eléouët J. F., 
Minimal elements required for the formation of respiratory syncytial virus cytoplasmic inclusion bodies in vivo and in vitro. mBio
11, 
e01202-20 (2020).
</cite> [<a href="https://doi.org/10.1128/mBio.01202-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7512546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32963000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Minimal%20elements%20required%20for%20the%20formation%20of%20respiratory%20syncytial%20virus%20cytoplasmic%20inclusion%20bodies%20in%20vivo%20and%20in%20vitro&amp;volume=11&amp;publication_year=2020&amp;pages=e01202-20&amp;pmid=32963000&amp;doi=10.1128/mBio.01202-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Nikolic J., Lagaudrière-Gesbert C., Scrima N., Blondel D., Gaudin Y., 
Structure and function of Negri Bodies. Adv. Exp. Med. Biol.
1215, 
111–127 (2019).
</cite> [<a href="https://doi.org/10.1007/978-3-030-14741-9_6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31317498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Exp.%20Med.%20Biol.&amp;title=Structure%20and%20function%20of%20Negri%20Bodies&amp;volume=1215&amp;publication_year=2019&amp;pages=111-127&amp;pmid=31317498&amp;doi=10.1007/978-3-030-14741-9_6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Nikolic J., Le Bars R., Lama Z., Scrima N., Lagaudrière-Gesbert C., Gaudin Y., Blondel D., 
Negri bodies are viral factories with properties of liquid organelles. Nat. Commun.
8, 
58 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-00102-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5498545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28680096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Negri%20bodies%20are%20viral%20factories%20with%20properties%20of%20liquid%20organelles&amp;volume=8&amp;publication_year=2017&amp;pages=58&amp;pmid=28680096&amp;doi=10.1038/s41467-017-00102-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Geiger F., Acker J., Papa G., Wang X., Arter W. E., Saar K. L., Erkamp N. A., Qi R., Bravo J. P., Strauss S., Krainer G., Burrone O. R., Jungmann R., Knowles T. P., Engelke H., Borodavka A., 
Liquid-liquid phase separation underpins the formation of replication factories in rotaviruses. EMBO J.
40, 
e107711 (2021).
</cite> [<a href="https://doi.org/10.15252/embj.2021107711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8561643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34524703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=Liquid-liquid%20phase%20separation%20underpins%20the%20formation%20of%20replication%20factories%20in%20rotaviruses&amp;volume=40&amp;publication_year=2021&amp;pages=e107711&amp;pmid=34524703&amp;doi=10.15252/embj.2021107711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Zan N., Li J., Yao J., Wu S., Li J., Chen F., Song B., Song R., 
Rational design of phytovirucide inhibiting nucleocapsid protein aggregation in tomato spotted wilt virus. Nat. Commun.
16, 
2034 (2025).
</cite> [<a href="https://doi.org/10.1038/s41467-025-57281-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11868578/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40016246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Rational%20design%20of%20phytovirucide%20inhibiting%20nucleocapsid%20protein%20aggregation%20in%20tomato%20spotted%20wilt%20virus&amp;volume=16&amp;publication_year=2025&amp;pages=2034&amp;pmid=40016246&amp;doi=10.1038/s41467-025-57281-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>May J. P., 
Plant viruses and biomolecular condensates: Novel perspectives in virus replication strategies. New Phytol.
243, 
1636–1638 (2024).
</cite> [<a href="https://doi.org/10.1111/nph.19778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38655603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Plant%20viruses%20and%20biomolecular%20condensates:%20Novel%20perspectives%20in%20virus%20replication%20strategies&amp;volume=243&amp;publication_year=2024&amp;pages=1636-1638&amp;pmid=38655603&amp;doi=10.1111/nph.19778&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Fang X., Gao Q., Zang Y., Qiao J. H., Gao D. M., Xu W. Y., Wang Y., Li D., Wang X. B., 
Host casein kinase 1-mediated phosphorylation modulates phase separation of a rhabdovirus phosphoprotein and virus infection. eLife
11, 
e74884 (2021).</cite> [<a href="https://doi.org/10.7554/eLife.74884" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8887900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35191833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Host%20casein%20kinase%201-mediated%20phosphorylation%20modulates%20phase%20separation%20of%20a%20rhabdovirus%20phosphoprotein%20and%20virus%20infection&amp;volume=11&amp;publication_year=2021&amp;pages=e74884&amp;pmid=35191833&amp;doi=10.7554/eLife.74884&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Liang Y., Zhang X., Wu B., Wang S., Kang L., Deng Y., Xie L., Li Z., 
Actomyosin-driven motility and coalescence of phase-separated viral inclusion bodies are required for efficient replication of a plant rhabdovirus. New Phytol.
240, 
1990–2006 (2023).
</cite> [<a href="https://doi.org/10.1111/nph.19255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37735952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Actomyosin-driven%20motility%20and%20coalescence%20of%20phase-separated%20viral%20inclusion%20bodies%20are%20required%20for%20efficient%20replication%20of%20a%20plant%20rhabdovirus&amp;volume=240&amp;publication_year=2023&amp;pages=1990-2006&amp;pmid=37735952&amp;doi=10.1111/nph.19255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Lin W., Nagy P. D., 
Co-opted cytosolic proteins form condensate substructures within membranous replication organelles of a positive-strand RNA virus. New Phytol.
243, 
1917–1935 (2024).
</cite> [<a href="https://doi.org/10.1111/nph.19691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38515267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Co-opted%20cytosolic%20proteins%20form%20condensate%20substructures%20within%20membranous%20replication%20organelles%20of%20a%20positive-strand%20RNA%20virus&amp;volume=243&amp;publication_year=2024&amp;pages=1917-1935&amp;pmid=38515267&amp;doi=10.1111/nph.19691&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Belov G. A., Sztul E., 
Rewiring of cellular membrane homeostasis by picornaviruses. J. Virol.
88, 
9478–9489 (2014).
</cite> [<a href="https://doi.org/10.1128/JVI.00922-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4136358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24920802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&amp;title=Rewiring%20of%20cellular%20membrane%20homeostasis%20by%20picornaviruses&amp;volume=88&amp;publication_year=2014&amp;pages=9478-9489&amp;pmid=24920802&amp;doi=10.1128/JVI.00922-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Salonen A., Ahola T., Kääriäinen L., 
Viral RNA replication in association with cellular membranes. Curr. Top. Microbiol. Immunol.
285, 
139–173 (2005).
</cite> [<a href="https://doi.org/10.1007/3-540-26764-6_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7120253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15609503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Top.%20Microbiol.%20Immunol.&amp;title=Viral%20RNA%20replication%20in%20association%20with%20cellular%20membranes&amp;volume=285&amp;publication_year=2005&amp;pages=139-173&amp;pmid=15609503&amp;doi=10.1007/3-540-26764-6_5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Hsu N. Y., Ilnytska O., Belov G., Santiana M., Chen Y. H., Takvorian P. M., Pau C., van der Schaar H., Kaushik-Basu N., Balla T., Cameron C. E., Ehrenfeld E., van Kuppeveld F. J., Altan-Bonnet N., 
Viral reorganization of the secretory pathway generates distinct organelles for RNA replication. Cell
141, 
799–811 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.03.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2982146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20510927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Viral%20reorganization%20of%20the%20secretory%20pathway%20generates%20distinct%20organelles%20for%20RNA%20replication&amp;volume=141&amp;publication_year=2010&amp;pages=799-811&amp;pmid=20510927&amp;doi=10.1016/j.cell.2010.03.050&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Li Z., Guo D., Qin Y., Chen M., 
PI4KB on inclusion bodies formed by ER membrane remodeling facilitates replication of human parainfluenza virus type 3. Cell Rep.
29, 
2229–2242.e4 (2019).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.10.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7104050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31747597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=PI4KB%20on%20inclusion%20bodies%20formed%20by%20ER%20membrane%20remodeling%20facilitates%20replication%20of%20human%20parainfluenza%20virus%20type%203&amp;volume=29&amp;publication_year=2019&amp;pages=2229-2242.e4&amp;pmid=31747597&amp;doi=10.1016/j.celrep.2019.10.052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Feng Z., Inaba J. I., Nagy P. D., 
The retromer is co-opted to deliver lipid enzymes for the biogenesis of lipid-enriched tombusviral replication organelles. Proc. Natl. Acad. Sci. U.S.A.
118, 
e2016066118 (2021).
</cite> [<a href="https://doi.org/10.1073/pnas.2016066118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7817191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33376201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=The%20retromer%20is%20co-opted%20to%20deliver%20lipid%20enzymes%20for%20the%20biogenesis%20of%20lipid-enriched%20tombusviral%20replication%20organelles&amp;volume=118&amp;publication_year=2021&amp;pages=e2016066118&amp;pmid=33376201&amp;doi=10.1073/pnas.2016066118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Phillips S. E., Vincent P., Rizzieri K. E., Schaaf G., Bankaitis V. A., Gaucher E. A., 
The diverse biological functions of phosphatidylinositol transfer proteins in eukaryotes. Crit. Rev. Biochem. Mol. Biol.
41, 
21–49 (2006).
</cite> [<a href="https://doi.org/10.1080/10409230500519573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16455519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol.&amp;title=The%20diverse%20biological%20functions%20of%20phosphatidylinositol%20transfer%20proteins%20in%20eukaryotes&amp;volume=41&amp;publication_year=2006&amp;pages=21-49&amp;pmid=16455519&amp;doi=10.1080/10409230500519573&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>W. F. Boss, A. J. Davis, Y. J. Im, R. M. Galvão, I. Perera, “Phosphoinositide Metabolism: Towards an Understanding of Subcellular Signaling” in <em>Biology of Inositols and Phosphoinositides: Subcellular Biochemistry</em>, A. L. Majumder, B. B. Biswas, Eds. (Springer, 2006), pp. 181–205.</cite> [<a href="https://doi.org/10.1007/0-387-27600-9_8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17121276/" class="usa-link">PubMed</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Balla T., 
Phosphoinositides: Tiny lipids with giant impact on cell regulation. Physiol. Rev.
93, 
1019–1137 (2013).
</cite> [<a href="https://doi.org/10.1152/physrev.00028.2012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3962547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23899561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Rev.&amp;title=Phosphoinositides:%20Tiny%20lipids%20with%20giant%20impact%20on%20cell%20regulation&amp;volume=93&amp;publication_year=2013&amp;pages=1019-1137&amp;pmid=23899561&amp;doi=10.1152/physrev.00028.2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Tan J., Brill J. A., 
Cinderella story: PI4P goes from precursor to key signaling molecule. Crit. Rev. Biochem. Mol. Biol.
49, 
33–58 (2014).
</cite> [<a href="https://doi.org/10.3109/10409238.2013.853024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24219382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol.&amp;title=Cinderella%20story:%20PI4P%20goes%20from%20precursor%20to%20key%20signaling%20molecule&amp;volume=49&amp;publication_year=2014&amp;pages=33-58&amp;pmid=24219382&amp;doi=10.3109/10409238.2013.853024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Dornan G. L., McPhail J. A., Burke J. E., 
Type III phosphatidylinositol 4 kinases: Structure, function, regulation, signalling and involvement in disease. Biochem. Soc. Trans.
44, 
260–266 (2016).
</cite> [<a href="https://doi.org/10.1042/BST20150219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26862213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Soc.%20Trans.&amp;title=Type%20III%20phosphatidylinositol%204%20kinases:%20Structure,%20function,%20regulation,%20signalling%20and%20involvement%20in%20disease&amp;volume=44&amp;publication_year=2016&amp;pages=260-266&amp;pmid=26862213&amp;doi=10.1042/BST20150219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Balla A., Balla T., 
Phosphatidylinositol 4-kinases: Old enzymes with emerging functions. Trends Cell Biol.
16, 
351–361 (2006).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2006.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16793271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol.&amp;title=Phosphatidylinositol%204-kinases:%20Old%20enzymes%20with%20emerging%20functions&amp;volume=16&amp;publication_year=2006&amp;pages=351-361&amp;pmid=16793271&amp;doi=10.1016/j.tcb.2006.05.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Mueller-Roeber B., Pical C., 
Inositol phospholipid metabolism in Arabidopsis. Characterized and putative isoforms of inositol phospholipid kinase and phosphoinositide-specific phospholipase C. Plant Physiol.
130, 
22–46 (2002).
</cite> [<a href="https://doi.org/10.1104/pp.004770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC166537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12226484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Inositol%20phospholipid%20metabolism%20in%20Arabidopsis.%20Characterized%20and%20putative%20isoforms%20of%20inositol%20phospholipid%20kinase%20and%20phosphoinositide-specific%20phospholipase%20C&amp;volume=130&amp;publication_year=2002&amp;pages=22-46&amp;pmid=12226484&amp;doi=10.1104/pp.004770&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Boura E., Nencka R., 
Phosphatidylinositol 4-kinases: Function, structure, and inhibition. Exp. Cell Res.
337, 
136–145 (2015).
</cite> [<a href="https://doi.org/10.1016/j.yexcr.2015.03.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26183104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Cell%20Res.&amp;title=Phosphatidylinositol%204-kinases:%20Function,%20structure,%20and%20inhibition&amp;volume=337&amp;publication_year=2015&amp;pages=136-145&amp;pmid=26183104&amp;doi=10.1016/j.yexcr.2015.03.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Berger K. L., Cooper J. D., Heaton N. S., Yoon R., Oakland T. E., Jordan T. X., Mateu G., Grakoui A., Randall G., 
Roles for endocytic trafficking and phosphatidylinositol 4-kinase III alpha in hepatitis C virus replication. Proc. Natl. Acad. Sci. U.S.A.
106, 
7577–7582 (2009).
</cite> [<a href="https://doi.org/10.1073/pnas.0902693106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2678598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19376974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Roles%20for%20endocytic%20trafficking%20and%20phosphatidylinositol%204-kinase%20III%20alpha%20in%20hepatitis%20C%20virus%20replication&amp;volume=106&amp;publication_year=2009&amp;pages=7577-7582&amp;pmid=19376974&amp;doi=10.1073/pnas.0902693106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Ishikawa-Sasaki K., Sasaki J., Taniguchi K., 
A complex comprising phosphatidylinositol 4-kinase IIIβ, ACBD3, and Aichi virus proteins enhances phosphatidylinositol 4-phosphate synthesis and is critical for formation of the viral replication complex. J. Virol.
88, 
6586–6598 (2014).
</cite> [<a href="https://doi.org/10.1128/JVI.00208-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4054359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24672044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&amp;title=A%20complex%20comprising%20phosphatidylinositol%204-kinase%20III%CE%B2,%20ACBD3,%20and%20Aichi%20virus%20proteins%20enhances%20phosphatidylinositol%204-phosphate%20synthesis%20and%20is%20critical%20for%20formation%20of%20the%20viral%20replication%20complex&amp;volume=88&amp;publication_year=2014&amp;pages=6586-6598&amp;pmid=24672044&amp;doi=10.1128/JVI.00208-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Sasaki J., Ishikawa K., Arita M., Taniguchi K., 
ACBD3-mediated recruitment of PI4KB to picornavirus RNA replication sites. EMBO J.
31, 
754–766 (2012).
</cite> [<a href="https://doi.org/10.1038/emboj.2011.429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3273392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22124328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=ACBD3-mediated%20recruitment%20of%20PI4KB%20to%20picornavirus%20RNA%20replication%20sites&amp;volume=31&amp;publication_year=2012&amp;pages=754-766&amp;pmid=22124328&amp;doi=10.1038/emboj.2011.429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Téoulé F., Brisac C., Pelletier I., Vidalain P. O., Jégouic S., Mirabelli C., Bessaud M., Combelas N., Autret A., Tangy F., Delpeyroux F., Blondel B., 
The Golgi protein ACBD3, an interactor for poliovirus protein 3A, modulates poliovirus replication. J. Virol.
87, 
11031–11046 (2013).
</cite> [<a href="https://doi.org/10.1128/JVI.00304-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3807280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23926333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&amp;title=The%20Golgi%20protein%20ACBD3,%20an%20interactor%20for%20poliovirus%20protein%203A,%20modulates%20poliovirus%20replication&amp;volume=87&amp;publication_year=2013&amp;pages=11031-11046&amp;pmid=23926333&amp;doi=10.1128/JVI.00304-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Xiao X., Lei X., Zhang Z., Ma Y., Qi J., Wu C., Xiao Y., Li L., He B., Wang J., 
Enterovirus 3A facilitates viral replication by promoting phosphatidylinositol 4-kinase IIIβ-ACBD3 interaction. J. Virol.
91, 
e00791-17 (2017).
</cite> [<a href="https://doi.org/10.1128/JVI.00791-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5599760/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28701404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&amp;title=Enterovirus%203A%20facilitates%20viral%20replication%20by%20promoting%20phosphatidylinositol%204-kinase%20III%CE%B2-ACBD3%20interaction&amp;volume=91&amp;publication_year=2017&amp;pages=e00791-17&amp;pmid=28701404&amp;doi=10.1128/JVI.00791-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Tai A. W., Benita Y., Peng L. F., Kim S. S., Sakamoto N., Xavier R. J., Chung R. T., 
A functional genomic screen identifies cellular cofactors of hepatitis C virus replication. Cell Host Microbe
5, 
298–307 (2009).
</cite> [<a href="https://doi.org/10.1016/j.chom.2009.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2756022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19286138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=A%20functional%20genomic%20screen%20identifies%20cellular%20cofactors%20of%20hepatitis%20C%20virus%20replication&amp;volume=5&amp;publication_year=2009&amp;pages=298-307&amp;pmid=19286138&amp;doi=10.1016/j.chom.2009.02.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Wang Z., Chen B., Zhang T., Zhou G., Yang X., 
Rice stripe mosaic disease: Characteristics and control strategies. Front. Microbiol.
12, 
715223 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.715223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8358444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34394065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Rice%20stripe%20mosaic%20disease:%20Characteristics%20and%20control%20strategies&amp;volume=12&amp;publication_year=2021&amp;pages=715223&amp;pmid=34394065&amp;doi=10.3389/fmicb.2021.715223&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Yang X., Huang J., Liu C., Chen B., Zhang T., Zhou G., 
Rice stripe mosaic virus, a novel cytorhabdovirus infecting rice via leafhopper transmission. Front. Microbiol.
7, 
2140 (2016).
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.02140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210121/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28101087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Rice%20stripe%20mosaic%20virus,%20a%20novel%20cytorhabdovirus%20infecting%20rice%20via%20leafhopper%20transmission&amp;volume=7&amp;publication_year=2016&amp;pages=2140&amp;pmid=28101087&amp;doi=10.3389/fmicb.2016.02140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Huang X., Wang J., Chen S., Liu S., Li Z., Wang Z., Chen B., Zhang C., Zhang Y., Wu J., Yang X., Xie Q., Li F., An H., Huang J., Li H., Liu C., Wu X., Liu D. X., Yang X., Zhou G., Zhang T., 
Rhabdovirus encoded glycoprotein induces and harnesses host antiviral autophagy for maintaining its compatible infection. Autophagy
20, 
275–294 (2024).
</cite> [<a href="https://doi.org/10.1080/15548627.2023.2252273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10813567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37656054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=Rhabdovirus%20encoded%20glycoprotein%20induces%20and%20harnesses%20host%20antiviral%20autophagy%20for%20maintaining%20its%20compatible%20infection&amp;volume=20&amp;publication_year=2024&amp;pages=275-294&amp;pmid=37656054&amp;doi=10.1080/15548627.2023.2252273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Chen S., Zhong X., Wang Z., Chen B., Huang X., Xu S., Yang X., Zhou G., Zhang T., 
Rice stripe mosaic virus hijacks rice heading-related gene to promote the overwintering of its insect vector. J. Integr. Plant Biol.
66, 
2000–2016 (2024).
</cite> [<a href="https://doi.org/10.1111/jipb.13722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38923382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Integr.%20Plant%20Biol.&amp;title=Rice%20stripe%20mosaic%20virus%20hijacks%20rice%20heading-related%20gene%20to%20promote%20the%20overwintering%20of%20its%20insect%20vector&amp;volume=66&amp;publication_year=2024&amp;pages=2000-2016&amp;pmid=38923382&amp;doi=10.1111/jipb.13722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Wang J. Z. Z., Song X., Chen S., Cao X., Yang X., Zhou G., Zhang T., 
Rice stripe mosaic virus encoded phosphoprotein forms viral factory-like granules and is crucial for viral replication. Phytopathol. Res.
7, 
23 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathol.%20Res.&amp;title=Rice%20stripe%20mosaic%20virus%20encoded%20phosphoprotein%20forms%20viral%20factory-like%20granules%20and%20is%20crucial%20for%20viral%20replication&amp;volume=7&amp;publication_year=2025&amp;pages=23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Jackson A. O., Dietzgen R. G., Goodin M. M., Bragg J. N., Deng M., 
Biology of plant rhabdoviruses. Annu. Rev. Phytopathol.
43, 
623–660 (2005).
</cite> [<a href="https://doi.org/10.1146/annurev.phyto.43.011205.141136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16078897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Phytopathol.&amp;title=Biology%20of%20plant%20rhabdoviruses&amp;volume=43&amp;publication_year=2005&amp;pages=623-660&amp;pmid=16078897&amp;doi=10.1146/annurev.phyto.43.011205.141136&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Muthamilarasan M., Mangu V. R., Zandkarimi H., Prasad M., Baisakh N., 
Structure, organization and evolution of ADP-ribosylation factors in rice and foxtail millet and their expression in rice. Sci. Rep.
6, 
24008 (2016).
</cite> [<a href="https://doi.org/10.1038/srep24008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4838888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27097755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Structure,%20organization%20and%20evolution%20of%20ADP-ribosylation%20factors%20in%20rice%20and%20foxtail%20millet%20and%20their%20expression%20in%20rice&amp;volume=6&amp;publication_year=2016&amp;pages=24008&amp;pmid=27097755&amp;doi=10.1038/srep24008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Donaldson J. G., Finazzi D., Klausner R. D., 
Brefeldin A inhibits Golgi membrane-catalysed exchange of guanine nucleotide onto ARF protein. Nature
360, 
350–352 (1992).
</cite> [<a href="https://doi.org/10.1038/360350a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1448151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Brefeldin%20A%20inhibits%20Golgi%20membrane-catalysed%20exchange%20of%20guanine%20nucleotide%20onto%20ARF%20protein&amp;volume=360&amp;publication_year=1992&amp;pages=350-352&amp;pmid=1448151&amp;doi=10.1038/360350a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Dascher C., Balch W. E., 
Dominant inhibitory mutants of ARF1 block endoplasmic reticulum to Golgi transport and trigger disassembly of the Golgi apparatus. J. Biol. Chem.
269, 
1437–1448 (1994).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/8288610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Dominant%20inhibitory%20mutants%20of%20ARF1%20block%20endoplasmic%20reticulum%20to%20Golgi%20transport%20and%20trigger%20disassembly%20of%20the%20Golgi%20apparatus&amp;volume=269&amp;publication_year=1994&amp;pages=1437-1448&amp;pmid=8288610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Balla A., Tuymetova G., Toth B., Szentpetery Z., Zhao X., Knight Z. A., Shokat K., Steinbach P. J., Balla T., 
Design of drug-resistant alleles of type-III phosphatidylinositol 4-kinases using mutagenesis and molecular modeling. Biochemistry
47, 
1599–1607 (2008).
</cite> [<a href="https://doi.org/10.1021/bi7017927" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18205404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Design%20of%20drug-resistant%20alleles%20of%20type-III%20phosphatidylinositol%204-kinases%20using%20mutagenesis%20and%20molecular%20modeling&amp;volume=47&amp;publication_year=2008&amp;pages=1599-1607&amp;pmid=18205404&amp;doi=10.1021/bi7017927&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Guo T., Chen H. C., Lu Z. Q., Diao M., Chen K., Dong N. Q., Shan J. X., Ye W. W., Huang S., Lin H. X., 
A SAC phosphoinositide phosphatase controls rice development via hydrolyzing PI4P and PI(4,5)P<sub>2</sub>. Plant Physiol.
182, 
1346–1358 (2020).
</cite> [<a href="https://doi.org/10.1104/pp.19.01131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31882455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=A%20SAC%20phosphoinositide%20phosphatase%20controls%20rice%20development%20via%20hydrolyzing%20PI4P%20and%20PI(4,5)P2&amp;volume=182&amp;publication_year=2020&amp;pages=1346-1358&amp;pmid=31882455&amp;doi=10.1104/pp.19.01131&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Zewe J. P., Miller A. M., Sangappa S., Wills R. C., Goulden B. D., Hammond G. R. V., 
Probing the subcellular distribution of phosphatidylinositol reveals a surprising lack at the plasma membrane. J. Cell Biol.
219, 
e201906127 (2020).
</cite> [<a href="https://doi.org/10.1083/jcb.201906127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32211893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=Probing%20the%20subcellular%20distribution%20of%20phosphatidylinositol%20reveals%20a%20surprising%20lack%20at%20the%20plasma%20membrane&amp;volume=219&amp;publication_year=2020&amp;pages=e201906127&amp;pmid=32211893&amp;doi=10.1083/jcb.201906127&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Balla A., Tuymetova G., Tsiomenko A., Várnai P., Balla T., 
A plasma membrane pool of phosphatidylinositol 4-phosphate is generated by phosphatidylinositol 4-kinase type-III alpha: Studies with the PH domains of the oxysterol binding protein and FAPP1. Mol. Biol. Cell
16, 
1282–1295 (2005).
</cite> [<a href="https://doi.org/10.1091/mbc.E04-07-0578" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC551492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15635101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=A%20plasma%20membrane%20pool%20of%20phosphatidylinositol%204-phosphate%20is%20generated%20by%20phosphatidylinositol%204-kinase%20type-III%20alpha:%20Studies%20with%20the%20PH%20domains%20of%20the%20oxysterol%20binding%20protein%20and%20FAPP1&amp;volume=16&amp;publication_year=2005&amp;pages=1282-1295&amp;pmid=15635101&amp;doi=10.1091/mbc.E04-07-0578&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Brown S. L., Garrison D. J., May J. P., 
Phase separation of a plant virus movement protein and cellular factors support virus-host interactions. PLOS Pathog.
17, 
e1009622 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1009622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8483311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34543360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog.&amp;title=Phase%20separation%20of%20a%20plant%20virus%20movement%20protein%20and%20cellular%20factors%20support%20virus-host%20interactions&amp;volume=17&amp;publication_year=2021&amp;pages=e1009622&amp;pmid=34543360&amp;doi=10.1371/journal.ppat.1009622&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Fang X. D., Yan T., Gao Q., Cao Q., Gao D. M., Xu W. Y., Zhang Z. J., Ding Z. H., Wang X. B., 
A cytorhabdovirus phosphoprotein forms mobile inclusions trafficked on the actin/ER network for viral RNA synthesis. J. Exp. Bot.
70, 
4049–4062 (2019).
</cite> [<a href="https://doi.org/10.1093/jxb/erz195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6685698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=A%20cytorhabdovirus%20phosphoprotein%20forms%20mobile%20inclusions%20trafficked%20on%20the%20actin/ER%20network%20for%20viral%20RNA%20synthesis&amp;volume=70&amp;publication_year=2019&amp;pages=4049-4062&amp;pmid=31020313&amp;doi=10.1093/jxb/erz195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Sha G., Sun P., Kong X., Han X., Sun Q., Fouillen L., Zhao J., Li Y., Yang L., Wang Y., Gong Q., Zhou Y., Zhou W., Jain R., Gao J., Huang R., Chen X., Zheng L., Zhang W., Qin Z., Zhou Q., Zeng Q., Xie K., Xu J., Chiu T.-Y., Guo L., Mortimer J. C., Boutté Y., Li Q., Kang Z., Ronald P. C., Li G., 
Genome editing of a rice CDP-DAG synthase confers multipathogen resistance. Nature
618, 
1017–1023 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06205-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11575942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37316672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome%20editing%20of%20a%20rice%20CDP-DAG%20synthase%20confers%20multipathogen%20resistance&amp;volume=618&amp;publication_year=2023&amp;pages=1017-1023&amp;pmid=37316672&amp;doi=10.1038/s41586-023-06205-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Kolli S., Meng X., Wu X., Shengjuler D., Cameron C. E., Xiang Y., Deng J., 
Structure-function analysis of vaccinia virus H7 protein reveals a novel phosphoinositide binding fold essential for poxvirus replication. J. Virol.
89, 
2209–2219 (2015).
</cite> [<a href="https://doi.org/10.1128/JVI.03073-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4338866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25473060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&amp;title=Structure-function%20analysis%20of%20vaccinia%20virus%20H7%20protein%20reveals%20a%20novel%20phosphoinositide%20binding%20fold%20essential%20for%20poxvirus%20replication&amp;volume=89&amp;publication_year=2015&amp;pages=2209-2219&amp;pmid=25473060&amp;doi=10.1128/JVI.03073-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Wen Y., Feigenson G. W., Vogt V. M., Dick R. A., 
Mechanisms of PI(4,5)P<sub>2</sub> enrichment in HIV-1 viral membranes. J. Mol. Biol.
432, 
5343–5364 (2020).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2020.07.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8262684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32739462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Mechanisms%20of%20PI(4,5)P2%20enrichment%20in%20HIV-1%20viral%20membranes&amp;volume=432&amp;publication_year=2020&amp;pages=5343-5364&amp;pmid=32739462&amp;doi=10.1016/j.jmb.2020.07.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Wang H., Zhang J., Liu H., Wang M., Dong Y., Zhou Y., Wong S. M., Xu K., Xu Q., 
A plant virus hijacks phosphatidylinositol-3,5-bisphosphate to escape autophagic degradation in its insect vector. Autophagy
19, 
1128–1143 (2023).
</cite> [<a href="https://doi.org/10.1080/15548627.2022.2116676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10012956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36093594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=A%20plant%20virus%20hijacks%20phosphatidylinositol-3,5-bisphosphate%20to%20escape%20autophagic%20degradation%20in%20its%20insect%20vector&amp;volume=19&amp;publication_year=2023&amp;pages=1128-1143&amp;pmid=36093594&amp;doi=10.1080/15548627.2022.2116676&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Feng Z., Xu K., Kovalev N., Nagy P. D., 
Recruitment of Vps34 PI3K and enrichment of PI3P phosphoinositide in the viral replication compartment is crucial for replication of a positive-strand RNA virus. PLOS Pathog.
15, 
e1007530 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1007530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6342326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30625229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog.&amp;title=Recruitment%20of%20Vps34%20PI3K%20and%20enrichment%20of%20PI3P%20phosphoinositide%20in%20the%20viral%20replication%20compartment%20is%20crucial%20for%20replication%20of%20a%20positive-strand%20RNA%20virus&amp;volume=15&amp;publication_year=2019&amp;pages=e1007530&amp;pmid=30625229&amp;doi=10.1371/journal.ppat.1007530&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Coemans B., Takahashi Y., Berberich T., Ito A., Kanzaki H., Matsumura H., Saitoh H., Tsuda S., Kamoun S., Sági L., Swennen R., Terauchi R., 
High-throughput in planta expression screening identifies an ADP-ribosylation factor (ARF1) involved in non-host resistance and R gene-mediated resistance. Mol. Plant Pathol.
9, 
25–36 (2008).
</cite> [<a href="https://doi.org/10.1111/j.1364-3703.2007.00440.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6640410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18705881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant%20Pathol.&amp;title=High-throughput%20in%20planta%20expression%20screening%20identifies%20an%20ADP-ribosylation%20factor%20(ARF1)%20involved%20in%20non-host%20resistance%20and%20R%20gene-mediated%20resistance&amp;volume=9&amp;publication_year=2008&amp;pages=25-36&amp;pmid=18705881&amp;doi=10.1111/j.1364-3703.2007.00440.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Stefano G., Renna L., Chatre L., Hanton S. L., Moreau P., Hawes C., Brandizzi F., 
In tobacco leaf epidermal cells, the integrity of protein export from the endoplasmic reticulum and of ER export sites depends on active COPI machinery. Plant J.
46, 
95–110 (2006).
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2006.02675.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16553898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=In%20tobacco%20leaf%20epidermal%20cells,%20the%20integrity%20of%20protein%20export%20from%20the%20endoplasmic%20reticulum%20and%20of%20ER%20export%20sites%20depends%20on%20active%20COPI%20machinery&amp;volume=46&amp;publication_year=2006&amp;pages=95-110&amp;pmid=16553898&amp;doi=10.1111/j.1365-313X.2006.02675.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Joshi R., Ramanarao M. V., Lee S., Kato N., Baisakh N., 
Ectopic expression of ADP ribosylation factor 1 (<em>SaARF1</em>) from smooth cordgrass (<em>Spartina alterniflora</em> Loisel) confers drought and salt tolerance in transgenic rice and <em>Arabidopsis</em>. PCTOC
117, 
17–30 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=PCTOC&amp;title=Ectopic%20expression%20of%20ADP%20ribosylation%20factor%201%20(SaARF1)%20from%20smooth%20cordgrass%20(Spartina%20alterniflora%20Loisel)%20confers%20drought%20and%20salt%20tolerance%20in%20transgenic%20rice%20and%20Arabidopsis&amp;volume=117&amp;publication_year=2014&amp;pages=17-30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Wang J., Huang X., Chen S., Chen J., Liang Z., Chen B., Yang X., Zhou G., Zhang T., 
On-site and visual detection of sorghum mosaic virus and rice stripe mosaic virus based on reverse transcription-recombinase-aided amplification and CRISPR/Cas12a. Front. Genome Ed.
5, 
1124794 (2023).
</cite> [<a href="https://doi.org/10.3389/fgeed.2023.1124794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9895793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36741944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Genome%20Ed.&amp;title=On-site%20and%20visual%20detection%20of%20sorghum%20mosaic%20virus%20and%20rice%20stripe%20mosaic%20virus%20based%20on%20reverse%20transcription-recombinase-aided%20amplification%20and%20CRISPR/Cas12a&amp;volume=5&amp;publication_year=2023&amp;pages=1124794&amp;pmid=36741944&amp;doi=10.3389/fgeed.2023.1124794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Livak K. J., Schmittgen T. D., 
Analysis of relative gene expression data using real-time quantitative PCR and the 2<sup>-ΔΔCT</sup> method. Methods
25, 
402–408 (2001).
</cite> [<a href="https://doi.org/10.1006/meth.2001.1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202-%CE%94%CE%94CT%20method&amp;volume=25&amp;publication_year=2001&amp;pages=402-408&amp;pmid=11846609&amp;doi=10.1006/meth.2001.1262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Figs. S1 to S7</p>
<p>Table S1</p>
<p>Legend for movie S1</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366689/bin/sciadv.adx7905_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adx7905_sm.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Movie S1</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366689/bin/sciadv.adx7905_movie_s1.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adx7905_movie_s1.zip</a><sup> (1.1MB, zip) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adx7905"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adx7905.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366689/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366689/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366689%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366689/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366689/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366689/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40834074/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366689/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40834074/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366689/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366689/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="5R369j0uUMcTicutsLcwAR0I7e07n1kdGmEmMdxFo01QZsMrvMSMk68QL6hXAL5j">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
