
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5312FD8AEF2EE30512FD00080AB806.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375763/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma">
<meta name="citation_author" content="Xiaowei Gu">
<meta name="citation_author_institution" content="Department of Hepatobiliary and Pancreatic Surgery (Ward 1), Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 China">
<meta name="citation_author_institution" content="Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China">
<meta name="citation_author" content="Fuzhong Liu">
<meta name="citation_author_institution" content="Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China">
<meta name="citation_author" content="Junfeng Zhang">
<meta name="citation_author_institution" content="Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China">
<meta name="citation_author" content="Nigeerayi Nuermaimaiti">
<meta name="citation_author_institution" content="Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China">
<meta name="citation_author" content="Wenjia Guo">
<meta name="citation_author_institution" content="Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China">
<meta name="citation_author" content="Xiaogang Dong">
<meta name="citation_author_institution" content="Department of Hepatobiliary and Pancreatic Surgery (Ward 1), Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 China">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="31170">
<meta name="citation_doi" content="10.1038/s41598-025-17102-1">
<meta name="citation_pmid" content="40850954">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375763/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375763/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375763/pdf/41598_2025_Article_17102.pdf">
<meta name="description" content="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic ...">
<meta name="og:title" content="Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375763/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375763">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-17102-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_17102.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375763%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375763/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375763/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375763/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 25;15:31170. doi: <a href="https://doi.org/10.1038/s41598-025-17102-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-17102-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xiaowei Gu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xiaowei Gu</span></h3>
<div class="p">
<sup>1</sup>Department of Hepatobiliary and Pancreatic Surgery (Ward 1), Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 China </div>
<div class="p">
<sup>2</sup>Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaowei Gu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Fuzhong Liu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Fuzhong Liu</span></h3>
<div class="p">
<sup>2</sup>Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fuzhong Liu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Junfeng Zhang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Junfeng Zhang</span></h3>
<div class="p">
<sup>2</sup>Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junfeng Zhang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nuermaimaiti%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Nigeerayi Nuermaimaiti</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Nigeerayi Nuermaimaiti</span></h3>
<div class="p">
<sup>2</sup>Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nuermaimaiti%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nigeerayi Nuermaimaiti</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Wenjia Guo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Wenjia Guo</span></h3>
<div class="p">
<sup>2</sup>Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenjia Guo</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Xiaogang Dong</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Xiaogang Dong</span></h3>
<div class="p">
<sup>1</sup>Department of Hepatobiliary and Pancreatic Surgery (Ward 1), Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaogang Dong</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Hepatobiliary and Pancreatic Surgery (Ward 1), Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 China </div>
<div id="Aff2">
<sup>2</sup>Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, 830000 People’s Republic of China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 25; Accepted 2025 Aug 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375763  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40850954/" class="usa-link">40850954</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4<sup>hi</sup>BPTF<sup>hi</sup>E-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-17102-1.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Single-cell, Intrahepatic cholangiocarcinoma, Cancer stem cell, Intercellular signaling pathways, Cell communication</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biliary tract cancer, Cancer stem cells</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">ICC is a malignant tumor of the liver characterized by high tumor heterogeneity and poor prognosis. However, there is still a limited understanding of the cellular diversity and molecular basis of interactions between tumor cells and niche cells in the surrounding microenvironment, which hinders the development of targeted therapies for this disease<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>.</p>
<p id="Par3">ICC arises from the epithelial cells of intrahepatic and extrahepatic bile ducts<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Current studies primarily focus on the genomic and transcriptomic levels, revealing key gene mutations such as TP53, KRAS, C1orf4 (ARID1A), IDH1/2 mutations, and FGFR gene fusions, as well as aberrant signaling pathways driving ICC pathogenesis<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. A prominent feature of ICC is the embedding of tumor cells within a dense stroma, with the tumor microenvironment playing a pivotal role in disease progression, invasion, and metastasis<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par4">Single-cell technologies, particularly single-cell RNA sequencing, play a vital role in studying the tumor microenvironment (TME) and uncovering the etiology and potential mechanisms of human diseases, including cancer<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Advances in scRNA-seq technology allow us to compare differences between tumor cells and normal cells, thereby gaining deeper insights into complex intercellular communication networks and related biological processes<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Aberrant intercellular communication may lead to uncontrolled cell proliferation, driving tumor initiation and progression<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. However, the specific mechanisms by which intercellular communication signals promote the growth of ICC tumor cells remain poorly understood.</p>
<p id="Par5">Intercellular communication is a central mechanism in processes such as cell fate determination, proliferation, migration, and homeostasis<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. One of the key challenges in intercellular communication is the depiction of signaling pathways that regulate cancer<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Numerous studies indicate that signaling pathways mutually regulate each other by triggering a series of intracellular signaling events. In certain tumors, there exist subpopulations of tumor stem cells with self-renewal and differentiation abilities, which are considered major drivers of intra-tumor and inter-tumor heterogeneity<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Intercellular communication plays a crucial role in maintaining the state of these CSCs and influencing their differentiation fate within the tumor and in interaction with the surrounding environment<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p>
<p id="Par6">In this study, we utilized an scRNA-seq dataset analyzing a total of 62,944 cells from two ICC tissue samples and two adjacent normal tissue samples. We identified 19 intercellular signaling pathways that were abnormally activated in the tumor cell clusters of ICC. Furthermore, we discovered a cancer stem cell subpopulation capable of secreting aberrant intercellular signaling molecules, providing valuable insights for differentiating cellular mechanisms and developing potential effective immunotherapy strategies.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">scRNA-seq data</h3>
<p id="Par7">The scRNA-seq data from the ICC patients were derived from the GEO database (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138709" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE138709</a>)<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The dataset includes samples from five ICC tissues and three adjacent normal tissues. For further analysis, we selected paired samples of two ICC tissues and two adjacent normal tissues from two patients, extracting high-quality scRNA-seq data.</p></section><section id="Sec4"><h3 class="pmc_sec_title">scRNA-Seq analysis and external validation</h3>
<p id="Par8">We used R software (v.4.4.0) and the Seurat package (v.5.0.3) to read the data from each sample and perform quality control. Cells with gene detection numbers between 200 and 2500 and mitochondrial gene expression below 5% were retained, while low-quality cells were excluded. After quality control, a total of 15,463 high-quality cells from four samples were preserved for subsequent analyses. We normalized the single-cell counts matrix data using the “NormalizeData” function and identified highly variable genes using the “FindVariableFeatures” function. Subsequently, we conducted dimensionality reduction on the top 30 principal components using principal component analysis (PCA), extracting the most significant individual variations in the principal components to uncover key features.</p>
<p id="Par9">We downloaded and analyzed bulk RNA sequencing data from The Cancer Genome Atlas cholangiocarcinoma (TCGA-CHOL) cohort to validate the expression levels of candidate genes in tumor versus normal samples. Comparative analyses were further performed using the GEPIA web server (<a href="http://gepia.cancer-pku.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gepia.cancer-pku.cn/</a>), which integrates transcriptomic data from both the TCGA and GTEx projects.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Identification of signature genes</h3>
<p id="Par10">We utilized the “FindAllMarkers” function and the Wilcoxon rank-sum test (lnFC &gt; 0.25, <em>p</em> &lt; 0.05, and min.pct &gt; 0.1) to calculate differentially expressed genes between cell clusters and other cells.</p></section><section id="Sec6"><h3 class="pmc_sec_title">inferCNV analysis</h3>
<p id="Par11">To detect large-scale copy number variations in tumor cells, we utilized the inferCNV package (v.1.20.0) to compare the gene expression levels of each tumor cell with a set of normal adjacent cells, inferring the chromosomal variation of tumor cell clusters by analyzing gene expression intensities at different genomic locations to explore potential chromosomal alterations<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Single-cell trajectory analysis</h3>
<p id="Par12">To understand gene expression changes during the process of cell differentiation, we constructed differentiation trajectories of epithelial cells using Monocle 2 (v.2.20.0)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. We performed pseudotime analysis to order individual cells based on biological processes such as differentiation and transition, employing default parameters and the DDRTree algorithm for trajectory construction. Additionally, we analyzed branching fate-determining genes using the Branch Expression Analysis Model (BEAM).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Cell–cell interaction analysis</h3>
<p id="Par13">We used CellChat (v.1.6.1) to analyze intercellular interactions based on the single-cell gene expression matrix<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. CellChat integrates prior knowledge of ligands, receptors, and their cofactors to calculate communication probabilities between cells. We aggregated ligand-receptor interactions for each signaling pathway, calculated communication probabilities at the pathway level, and visualized the intercellular communication network. CellChat provides ligand-receptor databases for both humans and mice; this study utilized the "CellChatDB.human" for data import.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Single-cell transcription factor analysis</h3>
<p id="Par14">Through transcription factor analysis (SCENIC)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, the regulatory strength of key transcription factors (TFs) within each candidate cell cluster was evaluated, and potential core regulatory TFs were identified. During the analysis, the SCENIC R package was used to calculate the activity of genes regulated by each TF, and the area under the recovery curve (AUC) values were used to quantify the functions of these TFs. To further compare TF activity differences across cell clusters, differential analysis was performed using the limma R package.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par15">The human intrahepatic cholangiocarcinoma (ICC) cell line HUCCT1 was purchased from Procell Life Science &amp; Technology Co., Ltd. (China). Cells were cultured in RPMI-1640 medium (Procell, China) supplemented with 10% fetal bovine serum (FBS), 100 U/mL penicillin, and 100 μg/mL streptomycin. All cultures were maintained at 37 °C in a humidified incubator with 5% CO<sub>2</sub>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Cell transfection and quantitative real-time PCR (qRT-PCR)</h3>
<p id="Par16">To achieve BPTF knockdown, HUCCT1 cells were transfected with BPTF-specific small interfering RNAs (siRNAs). The siRNAs, control sequences, and transfection vectors were designed and synthesized by GenePharma Co., Ltd. (Suzhou, China). Transfection was performed using Lipofectamine 3000 reagent (Thermo Fisher Scientific, USA) according to the manufacturer’s instructions. Cells were harvested 48 h post-transfection for further analysis. Total RNA was extracted from siRNA-transfected and non-transfected HUCCT1 cells using the HiPure Total RNA Mini Kit (Magen, China), and complementary DNA (cDNA) was synthesized using the PrimeScript™ RT Master Mix (Perfect Real Time) kit (TaKaRa, Japan). mRNA expression was analyzed by quantitative PCR using the GS AntiQ qPCR SYBR Green Fast Mix (Universal) kit (Genesand, China).</p></section><section id="Sec12"><h3 class="pmc_sec_title">In vitro functional assays</h3>
<p id="Par17">Cell viability was evaluated using the Cell Counting Kit-8 (CCK-8) assay. Briefly, 0.5–1.0 × 10<sup>4</sup> cells suspended in 100 μL complete medium were seeded into 96-well plates. After incubation, 10 μL of CCK-8 solution was added to each well and incubated at 37 °C. Absorbance was measured at 450 nm using a microplate reader. The impact of BPTF knockdown on cell migration was preliminarily assessed using a wound-healing assay. In brief, a scratch was created using a sterile 200-μL pipette tip across a confluent monolayer of cells in a 6-well plate. Detached cells were removed by washing with sterile PBS, and the remaining adherent cells were cultured in serum-free medium. Cell migration into the wound area was imaged and recorded at 24 h.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Multiplex immunofluorescence (mIF) staining</h3>
<p id="Par18">The tumor tissue sections used in this study were provided by the Department of Pathology at our institution. To assess the expression levels of target proteins (e.g., CXCR4, BPTF), tissue samples were processed using multiplex immunofluorescence (mIF) staining. Paraffin-embedded sections were first dewaxed and subjected to antigen retrieval. To minimize nonspecific binding, the sections were blocked with serum. Primary antibodies targeting the genes of interest were then applied at 4 °C and incubated overnight. The following day, after washing, the sections were incubated at room temperature for 1–2 h with fluorescence-labeled secondary antibodies and counterstained with DAPI to label the cell nuclei. Finally, the sections were mounted and imaged using either a fluorescence microscope or a confocal microscope to observe the expression and spatial distribution of the target proteins.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Statistical analyses</h3>
<p id="Par19">In this study, all data were analyzed and visualized using R (v. 4.4.0). The t-test and Wilcoxon test were used to assess the differences between cell clusters, while the AUCell algorithm was employed to calculate transcription factor activity. A <em>p</em>-value of &lt; 0.05 was considered statistically significant in all statistical tests.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Declaration</h3>
<p id="Par20">All methods described in this study were conducted in accordance with the guidelines and regulations of your journal.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Analysis of the transcriptional states of epithelial and tumor cells in ICC and cell type annotation</h3>
<p id="Par21">To investigate the transcriptional states of individual cells in ICC and normal bile duct tissues, we performed a comprehensive analysis of the scRNA-seq data obtained from the GEO database<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. We extracted scRNA-seq data from two paired ICC samples and adjacent non-tumor tissue samples for analysis. After integrating the scRNA-seq datasets and performing quality control, we constructed a cellular map consisting of 15,463 individual cells.</p>
<p id="Par22">Through analysis, these cells were divided into 16 distinct cell clusters (Fig. <a href="#Fig1" class="usa-link">1</a>A). We annotated the 16 cell clusters by analyzing the expression patterns of classical cell marker genes<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> (Fig. <a href="#Fig1" class="usa-link">1</a>B). Based on this marker gene expression, we further classified these 16 clusters into seven distinct cell types, offering a clearer understanding of the cellular composition within the ICC samples (Fig. <a href="#Fig1" class="usa-link">1</a>C). In clusters 7, 14, and 15, we observed that tumor cells and epithelial cells in ICC tissues exhibited overlapping marker gene expressions, which created challenges in differentiating between the two cell types (Fig. <a href="#Fig1" class="usa-link">1</a>B). Consequently, we temporarily designated clusters 7, 14, and 15 in ICC samples as "epithelial/tumor" cells, while in normal bile duct tissues, clusters 7, 14, and 15 were labeled as epithelial cells (Fig. <a href="#Fig1" class="usa-link">1</a>C). Further analysis demonstrated a marked increase in the proportion of "epithelial/tumor" cells in ICC samples when compared to adjacent non-tumor tissue samples (Fig. <a href="#Fig1" class="usa-link">1</a>D). These findings suggest a complex relationship between epithelial cells and tumor cells during the progression of ICC.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/be448cec02fc/41598_2025_17102_Fig1_HTML.jpg" loading="lazy" id="MO1" height="910" width="730" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The expression profiles of 15,463 single cells from paired ICC and adjacent non-tumor tissues. (<strong>A</strong>) UMAP plots derived from single-cell analysis show the distribution of cells from ICC and normal sample groups (left). Different cell clusters from ICC and normal sample groups are shown (right). (<strong>B</strong>) Bubble plots of the marker genes expressed in the major cell types. Dot color reflects expression level and dot size represents the percent of cells expressing marker genes in different cell types. (<strong>C</strong>) UMAP plot of 15,463 cells profiled in the present study colored by major cell types. (<strong>D</strong>) The proportion of major cell types in the four samples.</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Identification of epithelial/tumor cells and chromosomal copy number variation analysis in ICC</h3>
<p id="Par23">To precisely differentiate "epithelial/tumor" cells in ICC tissues, we carried out an in-depth analysis involving 800 "epithelial/tumor" cells from ICC samples and 272 epithelial cells from adjacent normal tissues, totaling 1072 cells. We performed dimensionality reduction analysis using Uniform Manifold Approximation and Projection (UMAP), dividing the cells into 12 distinct subpopulations (Fig. <a href="#Fig2" class="usa-link">2</a>A). In normal samples, the cells were categorized into 11 cell clusters (Fig. <a href="#Fig2" class="usa-link">2</a>B), while the cells in ICC samples were divided into 10 cell clusters (Fig. <a href="#Fig2" class="usa-link">2</a>C). Among these cell clusters, we observed that clusters 0, 1, 2, 3, 5, 6, 7, 9, and 11 were present in both normal and ICC samples. Notably, clusters 8 and 10 were exclusive to normal samples, while cluster 4 was specific to ICC samples. Therefore, we annotated clusters 0, 1, 2, 3, 5, 6, 7, 9, and 11 from normal samples, as well as clusters 8 and 10, as epithelial cells.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/03d3c63a1186/41598_2025_17102_Fig2_HTML.jpg" loading="lazy" id="MO2" height="823" width="737" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identification of the characteristics of epithelial and tumor cells. (<strong>A</strong>) UMAP plots from single-cell analysis show the different “epithelial/tumor” cell clusters in the ICC and normal sample groups. (<strong>B</strong>) UMAP plots from single-cell analysis show the different epithelial cell clusters in the normal sample group. (C) UMAP plots from single-cell analysis show the different "epithelial/tumor" cell clusters in the ICC group. (<strong>D</strong>) Violin plots depicting the expression of ICC tumor marker genes KRAS and MET across different cell clusters. (<strong>E</strong>) Violin plots showing the expression of KRAS and MET in the ICC and normal sample groups. (<strong>F</strong>) UMAP plots illustrating the expression of ICC tumor marker genes KRAS and MET across different cell clusters. (<strong>G</strong>) Violin plots depicting the expression of ICC tumor marker genes KRAS and MET in different cell clusters. (<strong>H</strong>) Heatmap showing the results of CNV analysis. The upper heatmap represents the CNV profile of reference cells, while the lower heatmap represents the CNV profile of target cells. Red indicates CNV amplification, blue indicates CNV deletion and the depth of color represents the magnitude of CNV variation. (<strong>I</strong>) Box plot illustrating the CNV levels across distinct cell clusters (Mann–Whitney U test, *** <em>p</em> &lt; 0.001).</p></figcaption></figure><p id="Par24">Further analysis revealed that clusters 0, 1, 2, 3, 4, 5, 6, 7, 9, 11, and 12 exhibited elevated expression levels of KRAS and MET (Fig. <a href="#Fig2" class="usa-link">2</a>D–F), KRAS and MET are among the most frequently mutated oncogenes in cancer<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, indicating that the cells in these clusters may possess tumor characteristics. In contrast, clusters 8 and 10 showed typical expressions of epithelial cell markers while lacking any tumor-specific markers, confirming that these clusters correspond to normal epithelial cells (Fig. <a href="#Fig2" class="usa-link">2</a>G).</p>
<p id="Par25">Using InferCNV analysis, we explored the chromosomal copy number variations (CNV) in these cells. By comparing with the 272 epithelial cells from normal bile duct tissues, we inferred chromosomal variations at different genomic locations. The results showed significant levels of CNV elevation in clusters 0, 1, 2, 3, 4, 5, 6, 7, 9, and 11 in ICC samples (Fig. <a href="#Fig2" class="usa-link">2</a>H and I). Thus, based on these findings, we annotated clusters 0, 1, 2, 3, 5, 6, 7, 9, and 11, as well as cluster 4, in ICC samples as tumor cells.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Pseudotime analysis of tumor cell differentiation trajectories and activation of key genes</h3>
<p id="Par26">To further explore the differentiation trajectories among different cell subpopulations, we performed pseudotime analysis using Monocle 2, identifying six distinct cell states (Fig. <a href="#Fig3" class="usa-link">3</a>A and B). By analyzing fate 1 and fate 2 of the cells, we could observe the developmental trajectory of epithelial cells toward malignant cells. In these trajectories, two cell clusters (8 and 10) represent the differentiation trajectory of epithelial cells, primarily associated with cell states 5 and 6. The differentiation trajectories of the other 10 cell clusters are mainly associated with the differentiation of tumor cells, reflecting the trajectories of cell states 1, 2, 3, and 4. From the pseudotime plot, we can see that cells in normal tissues, especially clusters 8 and 10, are at the starting stage of the differentiation trajectory, while cells in tumor tissues, particularly cluster 4, are at the terminal stage of the differentiation trajectory (Fig. <a href="#Fig3" class="usa-link">3</a>C–E). This indicates that the differentiation process of tumor cells drives their malignant transformation. Moreover, gene expression analysis revealed that multiple genes were significantly activated in tumor cells, which are closely related to specific behaviors of tumor cells. For instance, KRAS and MET are associated with cell proliferation and display dynamic expression along the differentiation trajectory from epithelial to malignant cells (Fig. <a href="#Fig3" class="usa-link">3</a>F). PIK3CA contributes to anti-apoptotic mechanisms in tumor cells, whereas CDKN2A inhibits cell proliferation and promotes apoptosis (Fig. <a href="#Fig3" class="usa-link">3</a>G). The activation of these genes further supports our findings regarding the biological functional changes of tumor cells during the differentiation process.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/755504974e03/41598_2025_17102_Fig3_HTML.jpg" loading="lazy" id="MO3" height="831" width="691" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pseudotime trajectory of all epithelial cells. (<strong>A</strong>) Displaying the beginnings and endings of pseudo-time trajectories. The colors from dark to light represent the order of pseudo-time. (<strong>B</strong>) Differentiation trajectory of epithelial cells, with each color representing a different cell state. (<strong>C</strong>) Differentiation trajectory of epithelial cells, with each color representing a sample type. (<strong>D</strong>) Differentiation trajectory of epithelial cells, with each color representing a cluster. (<strong>E</strong>) Differentiation trajectories for each individual cell cluster. (<strong>F</strong>) Differentiation trajectories of MET and KRAS. (<strong>G</strong>) Heatmap showing the expression of genes with significantly increased activity.</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Analysis of communication between epithelial tumor cell (E-T) clusters and identification of key signaling pathways</h3>
<p id="Par27">ICC originates from epithelial cells and undergoes complex genomic and epigenetic changes during its transformation into tumor cells. Some early-stage cells retain their original epithelial characteristics<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Based on this, we defined the specific cell populations shared by both normal and tumor samples (clusters 0, 1, 2, 3, 5, 6, 7, 9, and 11) as epithelial tumor (E-T) cells and labeled them sequentially as C0-E-T, C1-E-T, C2-E-T, C3-E-T, C5-E-T, C6-E-T, C7-E-T, C9-E-T, and C11-E-T. These coexisting E-T cell populations may play a pivotal role during the early stages of tumorigenesis and are of significant importance for elucidating the oncogenic mechanisms underlying ICC.</p>
<p id="Par28">To further investigate the intercellular communication among these E-T clusters, we constructed a communication map between the 12 cell clusters using the CellChat R package<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Through the analysis of these clusters, we have identified signaling pathways that are significantly enriched in both outgoing and incoming signals (Fig. <a href="#Fig4" class="usa-link">4</a>A). Among them, the MIF, MK, and SPP1 signaling pathways were the top three contributors (Fig. <a href="#Fig4" class="usa-link">4</a>B). Using heatmap analysis, we visualized the centrality scores of the MIF, MK, and SPP1 signaling pathways in different clusters. This reveals that clusters C7 and C10 are the main influencers in the MIF signaling pathway; clusters C2, C5, and C7 are the primary influencers in the MK signaling pathway; and clusters C4 and C7 are the main influencers in the SPP1 signaling pathway (Fig. <a href="#Fig4" class="usa-link">4</a>C). By further analyzing the ligand-receptor pairs (L–R pairs) of each signaling pathway, we found that MIF-(CD74 + CXCR4), MK-(MDK + NCL), and SPP1-(SPP1 + CD44) are the most contributive L-R pairs in their respective signaling pathways. These ligand-receptor pairs play a crucial role in facilitating communication among tumor cells and are presented in Fig. <a href="#Fig4" class="usa-link">4</a>D and E.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/2e2cfc947a2c/41598_2025_17102_Fig4_HTML.jpg" loading="lazy" id="MO4" height="936" width="748" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cellular communication network of the 12 cell clusters. (<strong>A</strong>) The network centrality scores are displayed via a heatmap, showing the top 19 incoming (left) and outgoing (right) signals across 12 cell clusters. (<strong>B</strong>) A circle plot displays the complex interactions between different cell populations within the top three signaling pathways. (<strong>C</strong>) A heatmap displays the top three signaling pathways. (<strong>D</strong>) The overall contribution of each ligand-receptor pair to the signaling pathways. (<strong>E</strong>) A circle plot displays the complex interactions between different cell populations for the most important ligand-receptor pairs.</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Analysis of major signaling pathways and communication networks between epithelial cells and E-T clusters</h3>
<p id="Par29">Among the 11 key epithelial cell clusters and the E–T cluster, we identified the top three significantly enriched intercellular signaling pathways: MIF, MK, and SPP1 (Fig. <a href="#Fig5" class="usa-link">5</a>A and B). These signaling pathways transmit information between cell clusters through specific ligand-receptor pairs, particularly MIF-(CD74 + CXCR4), MK-(MDK + NCL), and SPP1-(SPP1 + CD44) (Fig. <a href="#Fig5" class="usa-link">5</a>C). In these pathways, certain cell clusters secrete signaling molecules MIF, MK, and SPP1, while the target cell clusters express their corresponding receptors (CD74, CXCR4, MDK, NCL, and CD44) (Fig. <a href="#Fig5" class="usa-link">5</a>D). Through these interactions, a highly complex and dynamic communication network is established between cell clusters<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. This process is crucial in the tumor microenvironment, particularly in solid tumors such as ICC, where tumor cells not only rely on intrinsic genomic alterations<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> but also continuously interact with surrounding cells to shape the microenvironment and promote tumorigenesis<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/91f1d80621f5/41598_2025_17102_Fig5_HTML.jpg" loading="lazy" id="MO5" height="827" width="747" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cellular communication between epithelial cells and E-T clusters. (<strong>A</strong>)The network centrality scores are displayed in a heatmap, showing the top 3 incoming (left) and outgoing (right) signals across 11 cell clusters. (<strong>B</strong>) A chord diagram illustrates the complex interactions between cell populations. (<strong>C</strong>) The overall contribution of each ligand-receptor pair to the signaling pathways. (<strong>D</strong>) Violin plots show the gene expression of ligand-receptor pairs involved in the three signaling pathways.</p></figcaption></figure></section><section id="Sec22"><h3 class="pmc_sec_title">Analysis of abnormal signaling pathways and cell communication patterns in ICC tumor clusters</h3>
<p id="Par30">Through the analysis of ten tumor cell subpopulations extracted from ICC samples, we identified 19 abnormally activated intercellular signaling pathways (Fig. <a href="#Fig6" class="usa-link">6</a>A). These signaling pathways were grouped based on their functions: GROUP 1: Inflammatory Regulation and Immune Response; GROUP 2: Vascular Function or Angiogenesis Regulation; GROUP 3: Related to Cell Proliferation and Migration; GROUP 4: Related to Growth Factors (Fig. <a href="#Fig6" class="usa-link">6</a>B).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/52dc5b9a79ee/41598_2025_17102_Fig6_HTML.jpg" loading="lazy" id="MO6" height="931" width="750" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cellular communication between tumor clusters. (<strong>A</strong>) The network centrality scores are displayed in a heatmap, showing the top 19 incoming (left) and outgoing (right) signals across 10 cell clusters. (<strong>B</strong>) A scatter plot shows the division of 19 signaling pathways into four groups. (<strong>C</strong>–<strong>E</strong>) Figures show the network centrality scores (heatmap), ligand-receptor pairs (bar chart), and gene expression of receptor-ligand pairs (violin plots) for the top three signaling pathways. (<strong>F</strong> and <strong>G</strong>) River plots display the communication patterns coordinating secretory cell clusters with outgoing signaling pathways and target cell clusters with incoming signaling pathways.</p></figcaption></figure><p id="Par31">We focused on analyzing the top three signaling pathways that significantly contribute to the tumor microenvironment—MK, MIF, and SPP1 (Fig. <a href="#Fig6" class="usa-link">6</a>C–E). Among them, the main senders of the MIF signaling pathway are the C1-E-T cluster and the C4-Tumor cluster, while the main receiver is the C10-E-T cluster (Fig. <a href="#Fig6" class="usa-link">6</a>D). MIF signaling relies on the L-R ligand-receptor pair MIF-(CD74 + CXCR4), where CXCR4 is highly expressed in the C7-E-T, C9-E-T, and C11-E-T clusters, and these clusters widely express receptor and ligand genes throughout the tumor population (Fig. <a href="#Fig6" class="usa-link">6</a>D). Using the same analytical approach, we also examined the activities of the MK and SPP1 signaling pathways (Fig. <a href="#Fig6" class="usa-link">6</a>C and E). We found that the C7-E-T cluster plays a particularly prominent role in these major signaling pathways, serving as a key influencer across multiple signaling routes.</p>
<p id="Par32">We further identified different communication patterns between secretory cells and target cells to reveal the complex interactions between cells. For example, output pattern 1 in the C4 tumor cluster coordinates the sending of signals such as SPP1, MIF, VISFATIN, EGF, EDN, and GDF (Fig. <a href="#Fig6" class="usa-link">6</a>F); whereas input pattern 4 in the C4 tumor cluster coordinates the reception of signals such as MIF and GALECTIN (Fig. <a href="#Fig6" class="usa-link">6</a>G). The analysis of these communication patterns reveals the complex mechanisms of interactions among tumor cell populations, providing insights into the potential roles of different tumor subclusters<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p></section><section id="Sec23"><h3 class="pmc_sec_title">The critical role of BPTF and CXCR4 in cancer stem cell–driven tumor progression</h3>
<p id="Par33">We used the SCENIC method to study the role of transcription factors (TFs) in tumor progression<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. We identified 12 transcription factors targeting the MIF-(CD74 + CXCR4) ligand-receptor pair, with BPTF (Bromodomain PHD Finger Transcription Factor) identified as a key regulator of CXCR4 (Fig. <a href="#Fig7" class="usa-link">7</a>A). By analyzing the ranking of genes within the module, the height of cumulative curves, and the area under the curve (AUC), we calculated the activity level of BPTF using AUCell (Fig. <a href="#Fig7" class="usa-link">7</a>B). The results showed that BPTF expression was significantly increased in tumor tissues compared to normal tissues (Fig. <a href="#Fig7" class="usa-link">7</a>C), particularly enriched in the C7-E-T cluster (Fig. <a href="#Fig7" class="usa-link">7</a>D). Notably, CXCR4 is an important marker of CSCs<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, and BPTF plays a critical role in the self-renewal and differentiation of CSCs<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The expression of both CXCR4 and BPTF in the C7-E-T cluster further indicates that this cluster possesses CSCs characteristics. Therefore, we designated the C7-E-T cluster as the CXCR4<sup>hi</sup>BPTF<sup>hi</sup>E-T subpopulation. Through this analysis, we identified a potential CSCs subpopulation and explored the transcriptional regulatory mechanisms of the MIF signaling molecules secreted by this subpopulation, providing new insights into cell communication and tumor biology during ICC progression.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/902b629b9301/41598_2025_17102_Fig7_HTML.jpg" loading="lazy" id="MO7" height="512" width="742" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Transcription factor analysis of tumor clusters. (<strong>A</strong>) The regulatory network of transcription factors that target genes involved in the MIF-(CD74 + CXCR4) ligand-receptor pair. (<strong>B</strong>) AUCell quantification of BPTF transcription factor activity. (<strong>C</strong>) Violin plots showing the expression levels of BPTF across different samples. (<strong>D</strong>) Box plots displaying BPTF expression across different cell clusters.</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">Functional validation of BPTF in the malignant progression of ICC</h3>
<p id="Par34">CSCs play a pivotal role in maintaining tumor heterogeneity, promoting disease progression, and conferring therapeutic resistance. To investigate the oncogenic role of the CSC-associated marker BPTF in ICC, we conducted gene knockdown experiments in the ICC cell line HUCCT1 and RBE using three different small interfering RNAs (siRNAs) targeting BPTF. Reverse transcription quantitative PCR (RT-qPCR) confirmed the knockdown efficiency of each siRNA, showing a significant reduction in BPTF mRNA levels (Fig. <a href="#Fig8" class="usa-link">8</a>A).Among these, the two siRNAs exhibiting the highest knockdown efficiency, siBPTF-1 and siBPTF-3, were selected and subsequently renamed si-1 and si-2 for further experiments.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375763_41598_2025_17102_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39be/12375763/97cd31f321c2/41598_2025_17102_Fig8_HTML.jpg" loading="lazy" id="MO8" height="806" width="742" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BPTF Knockdown Impairs ICC Cell Functions. (<strong>A</strong>) RT-qPCR analysis demonstrating efficient knockdown of BPTF mRNA in HUCCT1 cells (left) and RBE cells (right) transfected with three distinct BPTF-targeting siRNAs compared to the negative control (NC). *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001. (<strong>B</strong>) CCK-8 assays showing reduced proliferative capacity of HUCCT1 cells (left) and RBE cells (right) following BPTF knockdown. Cell viability was assessed at 24, 48, 72, and 96 h post-transfection. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001. (<strong>C</strong>) Wound healing assays indicating impaired migratory ability of HUCCT1 cells (left) and RBE cells (right) upon BPTF knockdown. Representative images were captured at 0 and 24 h after scratching. (<strong>D</strong>) Quantification of migration rates in the wound healing assays shown in (<strong>C</strong>). Bar graphs display data for HUCCT1 cells (left) and RBE cells (right). *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001. (<strong>E</strong>) Multiplex immunofluorescence (mIF) images illustrating the co-expression of MIF, CD74, CXCR4, and BPTF in intrahepatic cholangiocarcinoma (ICC) tumor tissues and matched adjacent non-tumor tissues. (<strong>F</strong>) Bar graphs comparing the mean fluorescence intensity of MIF, CD74, CXCR4, and BPTF between normal and tumor tissues. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001.</p></figcaption></figure><p id="Par35">To assess the impact of BPTF deficiency on cellular behavior, we first performed a CCK-8 assay to evaluate cell proliferation. The results demonstrated that BPTF silencing significantly suppressed the proliferative capacity of HUCCT1 cells (Fig. <a href="#Fig8" class="usa-link">8</a>B). Furthermore, wound-healing assays revealed that BPTF knockdown also impaired the migratory ability of this cell line (Fig. <a href="#Fig8" class="usa-link">8</a>C and D). We speculate that the downregulation of BPTF may compromise the tumorigenic potential driven by CSCs, thereby attenuating the proliferative and migratory tendencies of tumor cells. These findings suggest that BPTF may serve as a key driver of ICC progression by maintaining CSC-like properties.</p>
<p id="Par36">To further validate the conclusions drawn from our single-cell transcriptomic analyses, we collected pathological tissue sections from ten ICC patients treated at our center, including tumor samples and their matched adjacent non-tumor tissues. Multiplex immunofluorescence (mIF) staining was performed to examine the spatial distribution and protein expression levels of MIF, CD74, CXCR4, and BPTF—key signaling axis components identified in the single-cell analysis. The results demonstrated that the co-expression levels of these markers were markedly higher in tumor tissues compared to adjacent normal tissues, providing additional evidence supporting the activation of these critical genes in ICC (Fig. <a href="#Fig8" class="usa-link">8</a>E and F and Supplemental Fig. <a href="#MOESM1" class="usa-link">S1</a>A).</p></section><section id="Sec25"><h3 class="pmc_sec_title">The role of the MIF signaling pathway in ICC tumor cells and its regulatory mechanisms of key genes</h3>
<p id="Par37">The MIF cytokine forms a complex with the type II transmembrane receptor CD74 and CXCR4, thereby activating multiple signaling pathways<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. To investigate the role of the MIF signaling pathway in tumor cells, we analyzed 1,072 epithelial/tumor cells. The results showed that MYC, CDK4 and ODC1 were significantly overexpressed in tumor cells, and these genes are key members of the MYC pathway (Supplemental Fig. <a href="#MOESM1" class="usa-link">S2</a>A–C). To investigate the potential function of BPTF in tumor cells, we performed data analysis using the Gene Transcription Regulation Database (<a href="https://gtrd20-06.biouml.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gtrd20-06.biouml.org/</a>). The results indicated that BPTF is a co-regulatory factor essential for MYC function and can modulate MYC transcriptional activity.</p>
<p id="Par38">We further identified marker genes enriched in the tumor cell clusters (Supplemental Fig. <a href="#MOESM1" class="usa-link">S2</a>D–F), discovering a total of 8 genes that participate in cancer occurrence, development, and metastasis through various pathways and mechanisms. These genes include: FOSL1: promoting tumor cell proliferation; HGF: promoting tumor cell invasion and metastasis; CD55: helping tumor cells evade immune surveillance; CDKN2A: associated with cell cycle regulation and apoptosis; FDCSP: related to immune regulation. The roles of these genes in different cancers vary depending on the signaling pathways and cellular processes involved. Some genes (such as FOSL1) can simultaneously affect multiple malignant phenotypes, indicating their significant roles in cancer progression and potential interactions and cross-effects.</p>
<p id="Par39">The above research indicates that the MIF signaling pathway may promote malignant phenotypes by activating the corresponding receptors in target cell clusters, such as MYC, CDK4, and ODC1. This finding provides important clues for further understanding the role of MIF in ICC progression and suggests its potential therapeutic target value in regulating tumor cell behavior<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par40">Although we have some understanding of the tumor microenvironment in ICC<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, elucidating the key molecular pathways in tumorigenesis requires complex models to analyze the details of cellular interactions in the tumor microenvironment. The specific mechanisms of signaling pathways regulating ICC remain an unresolved issue. Abnormal intercellular communication may lead to uncontrolled cell proliferation, driving tumorigenesis and progression<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Intercellular interactions are crucial for maintaining the state of tumor stem cell subpopulations and regulating their differentiation fate<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Increasing evidence confirms that epithelial-mesenchymal transition (EMT) plays a critical role in the progression of epithelial tumors from initiation to various stages, and it is expected to become an important direction for novel targeted therapies<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. During the transition of epithelial tumors from normal epithelium to tumor cells, some cells exhibit both epithelial and tumor characteristics, which constitute the E-T cell cluster, providing unique insights into the study of tumorigenesis and progression<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<p id="Par41">This study aims to provide a panoramic view of single-cell transcriptomics in ICC and investigate a series of signaling pathways that mutually regulate through intracellular signaling events. We identified 19 aberrantly activated signaling pathways in the E-T and tumor cell clusters, with the C7-E-T cluster playing a significant role in intercellular signaling through pathways like MIF, MK, and SPP1. The multifunctional pro-inflammatory protein MIF-related pathways play a key role, as MIF is closely associated with tumorigenesis, angiogenesis, and metastasis in various cancer phenotypes<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. MIF binds to the cell surface CD74, and in certain circumstances, CD74 forms complexes with CXCR2 or CXCR4, enhancing the biological effects of MIF, activating downstream signaling pathways, and regulating the survival, invasion, and metastasis of tumor cells, thereby providing mechanistic support for tumor progression<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p>
<p id="Par42">TFs are intracellular proteins that play a key role in regulating gene expression. They control the transcription process by specifically recognizing and binding to particular sequences on DNA<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Through transcription factor analysis, we identified BPTF as a key transcription factor that plays an important role in gene regulation. Some studies indicate that BPTF helps maintain the unique epigenetic state of CSCs<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p>
<p id="Par43">ICC is a highly aggressive malignancy characterized by pronounced cellular heterogeneity and a complex TME. While previous studies have primarily focused on genetic and epigenetic alterations as the principal drivers of tumorigenesis, these mechanisms alone fail to fully explain the dynamic progression of ICC. Increasing evidence suggests that intercellular communication plays a pivotal role in regulating tumor behavior and therapeutic responses.</p>
<p id="Par44">CSCs are a small subset of tumor cells with self-renewal and persistent proliferation capabilities, driving tumorigenesis, metastasis, and maintaining tumor heterogeneity<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. In-depth exploration of the regulatory mechanisms of CSCs, especially the key signaling pathways involved, can reveal the role of CSCs in maintaining tumor heterogeneity and driving tumor progression<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Through intercellular communication analysis, we found that signaling in the MIF pathway primarily operates through the ligand-receptor pair MIF-(CD74 + CXCR4)<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Studies show that CXCR4 is highly expressed in certain populations of tumor stem cells, which possess strong self-renewal and tumor-forming capabilities<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Although components of the MIF pathway have been reported in various malignancies, this study provides the first integrative characterization of its role in ICC at single-cell resolution. Notably, we further linked this extracellular signaling axis to downstream oncogenic programs such as MYC activation, suggesting a potential role for this pathway in bridging external TME cues with intracellular transcriptional reprogramming.</p>
<p id="Par45">To functionally validate this mechanism, we focused on BPTF, a chromatin remodeling factor highly expressed in the CXCR4<sup>hi</sup> subpopulation and implicated in the regulation of CSCs. In vitro experiments demonstrated that BPTF knockdown significantly inhibited the proliferation and migration of ICC cells, supporting its role in maintaining stem-like properties and tumor aggressiveness. These results indicate that BPTF may function not only as a downstream effector of MIF signaling but also as a promising therapeutic target.</p>
<p id="Par46">To enhance the translational relevance of our findings, we conducted multiplex immunofluorescence staining in an independent clinical cohort comprising tumor and paired adjacent non-tumor tissues from ICC patients. This analysis revealed co-expression of MIF, CD74, CXCR4, and BPTF at the protein level within tumor tissues, further supporting the activation of the MIF signaling axis in ICC.</p>
<p id="Par47">However, we are also fully aware that the current validation has clear limitations: the present experiments only indirectly suggest that the CXCR4<sup>hi</sup>BPTF<sup>hi</sup> E-T subpopulation may possess cancer stem cell characteristics such as self-renewal, multipotent differentiation, and high tumorigenicity. We fully acknowledge the necessity of additional experiments to rigorously confirm the stemness properties of this subpopulation. In future studies, we plan to perform CXCR4<sup>hi</sup> cell sorting and BPTF overexpression or knockdown rescue experiments to further elucidate the mechanistic role of this subpopulation in ICC progression. In parallel, we will focus on investigating its interactions with tumor microenvironment components, such as immune cells and fibroblasts. Future studies should incorporate animal models and larger, multi-center clinical cohorts to evaluate the robustness and clinical feasibility of these findings. These planned efforts will help address the current limitations and provide a more comprehensive understanding of the biological significance and therapeutic potential of this cancer stem cell subpopulation in ICC.</p>
<p id="Par48">Despite these limitations, the high concordance observed between transcriptomic and proteomic data in independent samples strongly supports the reliability of our conclusions and lays a solid foundation for subsequent functional investigations. Collectively, our findings provide valuable insights into the development of targeted therapeutic strategies against TME- and CSC-related pathways in ICC.</p>
<p id="Par49">In conclusion, the single-cell transcriptomic atlas reveals the characteristics of the TME in ICC and its adjacent tissues. By studying the key mediators in the MIF signaling pathway, we identified an aberrantly activated CSCs subcluster, namely the CXCR4<sup>hi</sup>BPTF<sup>hi</sup>E-T subcluster. The abnormally secreted signaling molecules from this subcluster may provide important clues for further investigation into their biological functions and molecular mechanisms, aiding in the formulation or improvement of treatment strategies for ICC.</p></section><section id="Sec120"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375763/bin/41598_2025_17102_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (536.1KB, pdf) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Xiaogang Dong contributed to study concept and design. Xiaowei Gu and Fuzhong Liu collected data. Junfeng Zhang, Nigeerayi Nuermaimaiti, and Wenjia Guo contributed to analyze the data. Xiaowei Gu and Fuzhong Liu contributed to prepare the manuscript. Xiaogang Dong provided critical feedback on methods, supervised the study. All authors approved the final version to be published.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was financially supported by the open project of Xinjiang Uygur Autonomous Region Key Laboratory (2022D04021), the Xinjiang Uygur Autonomous Region Natural Science Foundation (2022D01C290) and the Tianchi Young Talent Doctoral Program (2022TCYCDXG).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data is presented by the public database. The scRNA-seq data used in this study are openly available from the GEO databas (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>). Transcriptional regulation analysis from the Gene Transcription Regulation Database (<a href="https://gtrd20-06.biouml.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gtrd20-06.biouml.org/</a>). In addition, gene expression data and survival analyses based on TCGA-CHOL (The Cancer Genome Atlas—Cholangiocarcinoma) cohort and GEPIA (<a href="http://gepia.cancer-pku.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gepia.cancer-pku.cn/</a>) were used to further validate our findings.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par50">The authors declare no competing interests.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Ethical approval and consent to participate</h3>
<p id="Par51">This study was approved by the Ethics Committee of Affiliated Cancer Hospital of Xinjiang Medical University (Approval No. K-2024217) in accordance with the Declaration of Helsinki, and written informed consent was obtained from all participants or their legal guardians.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Wang, J., Liu, S., Cao, Y. &amp; Chen, Y. Overcoming treatment resistance in cholangiocarcinoma: current strategies, challenges, and prospects. <em>Front Cell Dev. Biol.</em><strong>12</strong>, 1408852. 10.3389/fcell.2024.1408852 (2024).
</cite> [<a href="https://doi.org/10.3389/fcell.2024.1408852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11327014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39156971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.,%20Liu,%20S.,%20Cao,%20Y.%20&amp;%20Chen,%20Y.%20Overcoming%20treatment%20resistance%20in%20cholangiocarcinoma:%20current%20strategies,%20challenges,%20and%20prospects.%20Front%20Cell%20Dev.%20Biol.12,%201408852.%2010.3389/fcell.2024.1408852%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Moeini, A., Sia, D., Bardeesy, N., Mazzaferro, V. &amp; Llovet, J. M. Molecular pathogenesis and targeted therapies for intrahepatic cholangiocarcinoma. <em>Clin. Cancer Res.</em><strong>22</strong>, 291–300. 10.1158/1078-0432.Ccr-14-3296 (2016).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-14-3296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12224570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26405193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moeini,%20A.,%20Sia,%20D.,%20Bardeesy,%20N.,%20Mazzaferro,%20V.%20&amp;%20Llovet,%20J.%20M.%20Molecular%20pathogenesis%20and%20targeted%20therapies%20for%20intrahepatic%20cholangiocarcinoma.%20Clin.%20Cancer%20Res.22,%20291%E2%80%93300.%2010.1158/1078-0432.Ccr-14-3296%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Wei, M. et al. Multiple cellular origins and molecular evolution of intrahepatic cholangiocarcinoma. <em>Cancer Lett.</em><strong>379</strong>, 253–261. 10.1016/j.canlet.2016.02.038 (2016).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2016.02.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26940139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20M.%20et%20al.%20Multiple%20cellular%20origins%20and%20molecular%20evolution%20of%20intrahepatic%20cholangiocarcinoma.%20Cancer%20Lett.379,%20253%E2%80%93261.%2010.1016/j.canlet.2016.02.038%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Varghese, T. P., John, A. &amp; Mathew, J. Revolutionizing cancer treatment: The role of radiopharmaceuticals in modern cancer therapy. <em>Precis. Radiat. Oncol.</em><strong>8</strong>, 145–152. 10.1002/pro6.1239 (2024).
</cite> [<a href="https://doi.org/10.1002/pro6.1239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11935213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40336977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varghese,%20T.%20P.,%20John,%20A.%20&amp;%20Mathew,%20J.%20Revolutionizing%20cancer%20treatment:%20The%20role%20of%20radiopharmaceuticals%20in%20modern%20cancer%20therapy.%20Precis.%20Radiat.%20Oncol.8,%20145%E2%80%93152.%2010.1002/pro6.1239%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Shen, G. et al. Bridging chronic inflammation and digestive cancer: The critical role of innate lymphoid cells in tumor microenvironments. <em>Int. J. Biol. Sci.</em><strong>20</strong>, 4799–4818. 10.7150/ijbs.96338 (2024).
</cite> [<a href="https://doi.org/10.7150/ijbs.96338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11414386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39309440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20G.%20et%20al.%20Bridging%20chronic%20inflammation%20and%20digestive%20cancer:%20The%20critical%20role%20of%20innate%20lymphoid%20cells%20in%20tumor%20microenvironments.%20Int.%20J.%20Biol.%20Sci.20,%204799%E2%80%934818.%2010.7150/ijbs.96338%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Huang, D. et al. Advances in single-cell RNA sequencing and its applications in cancer research. <em>J. Hematol. Oncol.</em><strong>16</strong>, 98. 10.1186/s13045-023-01494-6 (2023).
</cite> [<a href="https://doi.org/10.1186/s13045-023-01494-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10463514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37612741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20D.%20et%20al.%20Advances%20in%20single-cell%20RNA%20sequencing%20and%20its%20applications%20in%20cancer%20research.%20J.%20Hematol.%20Oncol.16,%2098.%2010.1186/s13045-023-01494-6%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Ye, B. et al. Single-cell RNA sequencing identifies a novel proliferation cell type affecting clinical outcome of pancreatic ductal adenocarcinoma. <em>Front Oncol.</em><strong>13</strong>, 1236435. 10.3389/fonc.2023.1236435 (2023).
</cite> [<a href="https://doi.org/10.3389/fonc.2023.1236435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10433893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37601684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20B.%20et%20al.%20Single-cell%20RNA%20sequencing%20identifies%20a%20novel%20proliferation%20cell%20type%20affecting%20clinical%20outcome%20of%20pancreatic%20ductal%20adenocarcinoma.%20Front%20Oncol.13,%201236435.%2010.3389/fonc.2023.1236435%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Chen, Y., Anwar, M., Wang, X., Zhang, B. &amp; Ma, B. Integrative transcriptomic and single-cell analysis reveals IL27RA as a key immune regulator and therapeutic indicator in breast cancer. <em>Discov. Oncol.</em><strong>16</strong>, 977. 10.1007/s12672-025-02811-w (2025).
</cite> [<a href="https://doi.org/10.1007/s12672-025-02811-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12127260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40450602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.,%20Anwar,%20M.,%20Wang,%20X.,%20Zhang,%20B.%20&amp;%20Ma,%20B.%20Integrative%20transcriptomic%20and%20single-cell%20analysis%20reveals%20IL27RA%20as%20a%20key%20immune%20regulator%20and%20therapeutic%20indicator%20in%20breast%20cancer.%20Discov.%20Oncol.16,%20977.%2010.1007/s12672-025-02811-w%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Zhang, J. et al. Single-cell transcriptome sequencing reveals aberrantly activated inter-tumor cell signaling pathways in the development of clear cell renal cell carcinoma. <em>J. Transl. Med.</em><strong>22</strong>, 37. 10.1186/s12967-023-04818-9 (2024).
</cite> [<a href="https://doi.org/10.1186/s12967-023-04818-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10775677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38191424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.%20et%20al.%20Single-cell%20transcriptome%20sequencing%20reveals%20aberrantly%20activated%20inter-tumor%20cell%20signaling%20pathways%20in%20the%20development%20of%20clear%20cell%20renal%20cell%20carcinoma.%20J.%20Transl.%20Med.22,%2037.%2010.1186/s12967-023-04818-9%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wu, Z. et al. Mitochondrial-related drug resistance lncRNAs as prognostic biomarkers in laryngeal squamous cell carcinoma. <em>Discov. Oncol.</em><strong>15</strong>, 785. 10.1007/s12672-024-01690-x (2024).
</cite> [<a href="https://doi.org/10.1007/s12672-024-01690-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11655928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39692950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Z.%20et%20al.%20Mitochondrial-related%20drug%20resistance%20lncRNAs%20as%20prognostic%20biomarkers%20in%20laryngeal%20squamous%20cell%20carcinoma.%20Discov.%20Oncol.15,%20785.%2010.1007/s12672-024-01690-x%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Su, J. et al. Cell-cell communication: new insights and clinical implications. <em>Signal Transduct. Target Ther.</em><strong>9</strong>, 196. 10.1038/s41392-024-01888-z (2024).
</cite> [<a href="https://doi.org/10.1038/s41392-024-01888-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11382761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39107318/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20J.%20et%20al.%20Cell-cell%20communication:%20new%20insights%20and%20clinical%20implications.%20Signal%20Transduct.%20Target%20Ther.9,%20196.%2010.1038/s41392-024-01888-z%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>An, L., Yu, R., Han, Y. &amp; Zhou, Z. Decoding the intercellular communication network during tumorigenesis. <em>Cancer Biol. Med.</em><strong>19</strong>, 265–272. 10.20892/j.issn.2095-3941.2021.0558 (2021).
</cite> [<a href="https://doi.org/10.20892/j.issn.2095-3941.2021.0558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8958884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34783465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?An,%20L.,%20Yu,%20R.,%20Han,%20Y.%20&amp;%20Zhou,%20Z.%20Decoding%20the%20intercellular%20communication%20network%20during%20tumorigenesis.%20Cancer%20Biol.%20Med.19,%20265%E2%80%93272.%2010.20892/j.issn.2095-3941.2021.0558%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Shimokawa, M. et al. Visualization and targeting of LGR5+ human colon cancer stem cells. <em>Nature</em><strong>545</strong>, 187–192 (2017).
</cite> [<a href="https://doi.org/10.1038/nature22081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28355176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shimokawa,%20M.%20et%20al.%20Visualization%20and%20targeting%20of%20LGR5+%20human%20colon%20cancer%20stem%20cells.%20Nature545,%20187%E2%80%93192%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Shibata, M. &amp; Hoque, M. O. Targeting cancer stem cells: a strategy for effective eradication of cancer. <em>Cancers</em><strong>11</strong>, 732 (2019).
</cite> [<a href="https://doi.org/10.3390/cancers11050732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6562442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31137841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shibata,%20M.%20&amp;%20Hoque,%20M.%20O.%20Targeting%20cancer%20stem%20cells:%20a%20strategy%20for%20effective%20eradication%20of%20cancer.%20Cancers11,%20732%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Razi, S. et al. The role of tumor microenvironment on cancer stem cell fate in solid tumors. <em>Cell Commun. Signal</em><strong>21</strong>, 143. 10.1186/s12964-023-01129-w (2023).
</cite> [<a href="https://doi.org/10.1186/s12964-023-01129-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10273768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37328876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Razi,%20S.%20et%20al.%20The%20role%20of%20tumor%20microenvironment%20on%20cancer%20stem%20cell%20fate%20in%20solid%20tumors.%20Cell%20Commun.%20Signal21,%20143.%2010.1186/s12964-023-01129-w%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zhang, M. et al. Single-cell transcriptomic architecture and intercellular crosstalk of human intrahepatic cholangiocarcinoma. <em>J. Hepatol.</em><strong>73</strong>, 1118–1130. 10.1016/j.jhep.2020.05.039 (2020).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2020.05.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32505533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20M.%20et%20al.%20Single-cell%20transcriptomic%20architecture%20and%20intercellular%20crosstalk%20of%20human%20intrahepatic%20cholangiocarcinoma.%20J.%20Hepatol.73,%201118%E2%80%931130.%2010.1016/j.jhep.2020.05.039%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Jerby-Arnon, L. et al. A cancer cell program promotes T cell exclusion and resistance to checkpoint blockade. <em>Cell</em><strong>175</strong>, 984-997.e924. 10.1016/j.cell.2018.09.006 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6410377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30388455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jerby-Arnon,%20L.%20et%20al.%20A%20cancer%20cell%20program%20promotes%20T%20cell%20exclusion%20and%20resistance%20to%20checkpoint%20blockade.%20Cell175,%20984-997.e924.%2010.1016/j.cell.2018.09.006%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Qiu, X. et al. Reversed graph embedding resolves complex single-cell trajectories. <em>Nat. Methods</em><strong>14</strong>, 979–982. 10.1038/nmeth.4402 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5764547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28825705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu,%20X.%20et%20al.%20Reversed%20graph%20embedding%20resolves%20complex%20single-cell%20trajectories.%20Nat.%20Methods14,%20979%E2%80%93982.%2010.1038/nmeth.4402%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Jin, S. et al. Inference and analysis of cell-cell communication using cell chat. <em>Nat. Commun.</em><strong>12</strong>, 1088. 10.1038/s41467-021-21246-9 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21246-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20S.%20et%20al.%20Inference%20and%20analysis%20of%20cell-cell%20communication%20using%20cell%20chat.%20Nat.%20Commun.12,%201088.%2010.1038/s41467-021-21246-9%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Aibar, S. et al. SCENIC: Single-cell regulatory network inference and clustering. <em>Nat. Methods</em><strong>14</strong>, 1083–1086. 10.1038/nmeth.4463 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aibar,%20S.%20et%20al.%20SCENIC:%20Single-cell%20regulatory%20network%20inference%20and%20clustering.%20Nat.%20Methods14,%201083%E2%80%931086.%2010.1038/nmeth.4463%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Song, G. et al. Single-cell transcriptomic analysis suggests two molecularly subtypes of intrahepatic cholangiocarcinoma. <em>Nat. Commun.</em><strong>13</strong>, 1642. 10.1038/s41467-022-29164-0 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-29164-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8960779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35347134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20G.%20et%20al.%20Single-cell%20transcriptomic%20analysis%20suggests%20two%20molecularly%20subtypes%20of%20intrahepatic%20cholangiocarcinoma.%20Nat.%20Commun.13,%201642.%2010.1038/s41467-022-29164-0%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Cecchi, F., Rabe, D. C. &amp; Bottaro, D. P. Targeting the HGF/Met signalling pathway in cancer. <em>Eur. J. Cancer</em><strong>46</strong>, 1260–1270. 10.1016/j.ejca.2010.02.028 (2010).
</cite> [<a href="https://doi.org/10.1016/j.ejca.2010.02.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3412517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20303741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cecchi,%20F.,%20Rabe,%20D.%20C.%20&amp;%20Bottaro,%20D.%20P.%20Targeting%20the%20HGF/Met%20signalling%20pathway%20in%20cancer.%20Eur.%20J.%20Cancer46,%201260%E2%80%931270.%2010.1016/j.ejca.2010.02.028%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Johnson, C., Burkhart, D. L. &amp; Haigis, K. M. Classification of KRAS-activating mutations and the implications for therapeutic intervention. <em>Cancer Discov.</em><strong>12</strong>, 913–923. 10.1158/2159-8290.Cd-22-0035 (2022).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-22-0035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8988514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35373279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnson,%20C.,%20Burkhart,%20D.%20L.%20&amp;%20Haigis,%20K.%20M.%20Classification%20of%20KRAS-activating%20mutations%20and%20the%20implications%20for%20therapeutic%20intervention.%20Cancer%20Discov.12,%20913%E2%80%93923.%2010.1158/2159-8290.Cd-22-0035%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Choi, J. H. &amp; Thung, S. N. Recent advances in pathology of intrahepatic cholangiocarcinoma. <em>Cancers</em><strong>16</strong>, 1537 (2024).
</cite> [<a href="https://doi.org/10.3390/cancers16081537" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11048541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38672619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi,%20J.%20H.%20&amp;%20Thung,%20S.%20N.%20Recent%20advances%20in%20pathology%20of%20intrahepatic%20cholangiocarcinoma.%20Cancers16,%201537%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Shao, X., Lu, X., Liao, J., Chen, H. &amp; Fan, X. New avenues for systematically inferring cell-cell communication: Through single-cell transcriptomics data. <em>Protein Cell</em><strong>11</strong>, 866–880. 10.1007/s13238-020-00727-5 (2020).
</cite> [<a href="https://doi.org/10.1007/s13238-020-00727-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7719148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32435978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shao,%20X.,%20Lu,%20X.,%20Liao,%20J.,%20Chen,%20H.%20&amp;%20Fan,%20X.%20New%20avenues%20for%20systematically%20inferring%20cell-cell%20communication:%20Through%20single-cell%20transcriptomics%20data.%20Protein%20Cell11,%20866%E2%80%93880.%2010.1007/s13238-020-00727-5%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Warren, E. A. K. &amp; Maithel, S. K. Molecular pathology for cholangiocarcinoma: A review of actionable genetic targets and their relevance to adjuvant &amp; neoadjuvant therapy, staging, follow-up, and determination of minimal residual disease. <em>Hepatobiliary Surg. Nutr.</em><strong>13</strong>, 29–38. 10.21037/hbsn-22-563 (2024).
</cite> [<a href="https://doi.org/10.21037/hbsn-22-563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10839719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38322206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Warren,%20E.%20A.%20K.%20&amp;%20Maithel,%20S.%20K.%20Molecular%20pathology%20for%20cholangiocarcinoma:%20A%20review%20of%20actionable%20genetic%20targets%20and%20their%20relevance%20to%20adjuvant%20&amp;%20neoadjuvant%20therapy,%20staging,%20follow-up,%20and%20determination%20of%20minimal%20residual%20disease.%20Hepatobiliary%20Surg.%20Nutr.13,%2029%E2%80%9338.%2010.21037/hbsn-22-563%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Fabris, L., Sato, K., Alpini, G. &amp; Strazzabosco, M. The tumor microenvironment in cholangiocarcinoma progression. <em>Hepatology</em><strong>73</strong>(Suppl 1), 75–85. 10.1002/hep.31410 (2021).
</cite> [<a href="https://doi.org/10.1002/hep.31410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7714713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32500550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fabris,%20L.,%20Sato,%20K.,%20Alpini,%20G.%20&amp;%20Strazzabosco,%20M.%20The%20tumor%20microenvironment%20in%20cholangiocarcinoma%20progression.%20Hepatology73(Suppl%201),%2075%E2%80%9385.%2010.1002/hep.31410%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Van de Sande, B. et al. A scalable SCENIC workflow for single-cell gene regulatory network analysis. <em>Nat. Protoc.</em><strong>15</strong>, 2247–2276. 10.1038/s41596-020-0336-2 (2020).
</cite> [<a href="https://doi.org/10.1038/s41596-020-0336-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32561888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20de%20Sande,%20B.%20et%20al.%20A%20scalable%20SCENIC%20workflow%20for%20single-cell%20gene%20regulatory%20network%20analysis.%20Nat.%20Protoc.15,%202247%E2%80%932276.%2010.1038/s41596-020-0336-2%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Hermann, P. C. et al. Distinct populations of cancer stem cells determine tumor growth and metastatic activity in human pancreatic cancer. <em>Cell Stem Cell</em><strong>1</strong>, 313–323. 10.1016/j.stem.2007.06.002 (2007).
</cite> [<a href="https://doi.org/10.1016/j.stem.2007.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18371365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hermann,%20P.%20C.%20et%20al.%20Distinct%20populations%20of%20cancer%20stem%20cells%20determine%20tumor%20growth%20and%20metastatic%20activity%20in%20human%20pancreatic%20cancer.%20Cell%20Stem%20Cell1,%20313%E2%80%93323.%2010.1016/j.stem.2007.06.002%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Gelmini, S., Mangoni, M., Serio, M., Romagnani, P. &amp; Lazzeri, E. The critical role of SDF-1/CXCR4 axis in cancer and cancer stem cells metastasis. <em>J. Endocrinol. Invest.</em><strong>31</strong>, 809–819. 10.1007/BF03349262 (2008).
</cite> [<a href="https://doi.org/10.1007/BF03349262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18997494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gelmini,%20S.,%20Mangoni,%20M.,%20Serio,%20M.,%20Romagnani,%20P.%20&amp;%20Lazzeri,%20E.%20The%20critical%20role%20of%20SDF-1/CXCR4%20axis%20in%20cancer%20and%20cancer%20stem%20cells%20metastasis.%20J.%20Endocrinol.%20Invest.31,%20809%E2%80%93819.%2010.1007/BF03349262%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zhao, X. et al. BPTF promotes hepatocellular carcinoma growth by modulating hTERT signaling and cancer stem cell traits. <em>Redox Biol</em><strong>20</strong>, 427–441. 10.1016/j.redox.2018.10.018 (2019).
</cite> [<a href="https://doi.org/10.1016/j.redox.2018.10.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6230923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30419422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20X.%20et%20al.%20BPTF%20promotes%20hepatocellular%20carcinoma%20growth%20by%20modulating%20hTERT%20signaling%20and%20cancer%20stem%20cell%20traits.%20Redox%20Biol20,%20427%E2%80%93441.%2010.1016/j.redox.2018.10.018%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Frey, W. D. et al. BPTF maintains chromatin accessibility and the self-renewal capacity of mammary gland stem cells. <em>Stem Cell Rep.</em><strong>9</strong>, 23–31. 10.1016/j.stemcr.2017.04.031 (2017).</cite> [<a href="https://doi.org/10.1016/j.stemcr.2017.04.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5783326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28579392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frey,%20W.%20D.%20et%20al.%20BPTF%20maintains%20chromatin%20accessibility%20and%20the%20self-renewal%20capacity%20of%20mammary%20gland%20stem%20cells.%20Stem%20Cell%20Rep.9,%2023%E2%80%9331.%2010.1016/j.stemcr.2017.04.031%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Klasen, C. et al. MIF promotes B cell chemotaxis through the receptors CXCR4 and CD74 and ZAP-70 signaling. <em>J. Immunol.</em><strong>192</strong>, 5273–5284 (2014).
</cite> [<a href="https://doi.org/10.4049/jimmunol.1302209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24760155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klasen,%20C.%20et%20al.%20MIF%20promotes%20B%20cell%20chemotaxis%20through%20the%20receptors%20CXCR4%20and%20CD74%20and%20ZAP-70%20signaling.%20J.%20Immunol.192,%205273%E2%80%935284%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wang, Q. et al. IL1RN and PRRX1 as a prognostic biomarker correlated with immune infiltrates in colorectal cancer: Evidence from bioinformatic analysis. <em>Int. J. Genom.</em><strong>2022</strong>, 2723264. 10.1155/2022/2723264 (2022).</cite> [<a href="https://doi.org/10.1155/2022/2723264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9726255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36483329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.%20et%20al.%20IL1RN%20and%20PRRX1%20as%20a%20prognostic%20biomarker%20correlated%20with%20immune%20infiltrates%20in%20colorectal%20cancer:%20Evidence%20from%20bioinformatic%20analysis.%20Int.%20J.%20Genom.2022,%202723264.%2010.1155/2022/2723264%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang, J. &amp; Ilyas, S. Targeting the tumor microenvironment in cholangiocarcinoma: Implications for therapy. <em>Exp. Opin. Investig. Drugs</em><strong>30</strong>, 429–438. 10.1080/13543784.2021.1865308 (2021).</cite> [<a href="https://doi.org/10.1080/13543784.2021.1865308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8096665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33322977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.%20&amp;%20Ilyas,%20S.%20Targeting%20the%20tumor%20microenvironment%20in%20cholangiocarcinoma:%20Implications%20for%20therapy.%20Exp.%20Opin.%20Investig.%20Drugs30,%20429%E2%80%93438.%2010.1080/13543784.2021.1865308%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Li, Z., Li, J., Bai, X., Huang, X. &amp; Wang, Q. Tumor microenvironment as a complex milieu driving cancer progression: A mini review. <em>Clin. Transl. Oncol.</em><strong>27</strong>, 1943–1952. 10.1007/s12094-024-03697-w (2025).
</cite> [<a href="https://doi.org/10.1007/s12094-024-03697-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12033186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39342061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.,%20Li,%20J.,%20Bai,%20X.,%20Huang,%20X.%20&amp;%20Wang,%20Q.%20Tumor%20microenvironment%20as%20a%20complex%20milieu%20driving%20cancer%20progression:%20A%20mini%20review.%20Clin.%20Transl.%20Oncol.27,%201943%E2%80%931952.%2010.1007/s12094-024-03697-w%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Ebrahim, T., Ebrahim, A. S. &amp; Kandouz, M. Diversity of intercellular communication modes: A cancer biology perspective. <em>Cells</em>10.3390/cells13060495 (2024).
</cite> [<a href="https://doi.org/10.3390/cells13060495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10969453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38534339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ebrahim,%20T.,%20Ebrahim,%20A.%20S.%20&amp;%20Kandouz,%20M.%20Diversity%20of%20intercellular%20communication%20modes:%20A%20cancer%20biology%20perspective.%20Cells10.3390/cells13060495%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Meng, K. &amp; Lu, H. Clinical application of high-LET radiotherapy combined with immunotherapy in malignant tumors. <em>Precis. Radiat. Oncol.</em><strong>8</strong>, 42–46. 10.1002/pro6.1225 (2024).
</cite> [<a href="https://doi.org/10.1002/pro6.1225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11935250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40336565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20K.%20&amp;%20Lu,%20H.%20Clinical%20application%20of%20high-LET%20radiotherapy%20combined%20with%20immunotherapy%20in%20malignant%20tumors.%20Precis.%20Radiat.%20Oncol.8,%2042%E2%80%9346.%2010.1002/pro6.1225%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Li, K. H. et al. Clinical outcomes of multisite moderate to high dose radiotherapy for patients with metastatic melanoma. <em>Precis. Radiat. Oncol.</em><strong>8</strong>, 62–69. 10.1002/pro6.1224 (2024).
</cite> [<a href="https://doi.org/10.1002/pro6.1224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11934978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40336647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20K.%20H.%20et%20al.%20Clinical%20outcomes%20of%20multisite%20moderate%20to%20high%20dose%20radiotherapy%20for%20patients%20with%20metastatic%20melanoma.%20Precis.%20Radiat.%20Oncol.8,%2062%E2%80%9369.%2010.1002/pro6.1224%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Chen, Y. et al. Prognostic value of preoperative modified Glasgow prognostic score in predicting overall survival in breast cancer patients: A retrospective cohort study. <em>Oncol. Lett.</em><strong>29</strong>, 180. 10.3892/ol.2025.14926 (2025).
</cite> [<a href="https://doi.org/10.3892/ol.2025.14926" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11843409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39990808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.%20et%20al.%20Prognostic%20value%20of%20preoperative%20modified%20Glasgow%20prognostic%20score%20in%20predicting%20overall%20survival%20in%20breast%20cancer%20patients:%20A%20retrospective%20cohort%20study.%20Oncol.%20Lett.29,%20180.%2010.3892/ol.2025.14926%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Lue, H. et al. Macrophage migration inhibitory factor (MIF) promotes cell survival by activation of the Akt pathway and role for CSN5/JAB1 in the control of autocrine MIF activity. <em>Oncogene</em><strong>26</strong>, 5046–5059. 10.1038/sj.onc.1210318 (2007).
</cite> [<a href="https://doi.org/10.1038/sj.onc.1210318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17310986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lue,%20H.%20et%20al.%20Macrophage%20migration%20inhibitory%20factor%20(MIF)%20promotes%20cell%20survival%20by%20activation%20of%20the%20Akt%20pathway%20and%20role%20for%20CSN5/JAB1%20in%20the%20control%20of%20autocrine%20MIF%20activity.%20Oncogene26,%205046%E2%80%935059.%2010.1038/sj.onc.1210318%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Wang, Q., Wang, H., Ding, Y., Wan, M. &amp; Xu, M. The role of adipokines in pancreatic cancer. <em>Front Oncol.</em><strong>12</strong>, 926230. 10.3389/fonc.2022.926230 (2022).
</cite> [<a href="https://doi.org/10.3389/fonc.2022.926230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9305334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35875143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.,%20Wang,%20H.,%20Ding,%20Y.,%20Wan,%20M.%20&amp;%20Xu,%20M.%20The%20role%20of%20adipokines%20in%20pancreatic%20cancer.%20Front%20Oncol.12,%20926230.%2010.3389/fonc.2022.926230%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Penticuff, J. C., Woolbright, B. L., Sielecki, T. M., Weir, S. J. &amp; Taylor, J. A. MIF family proteins in genitourinary cancer: Tumorigenic roles and therapeutic potential. <em>Nat. Rev. Urol.</em><strong>16</strong>, 318–328. 10.1038/s41585-019-0171-9 (2019).
</cite> [<a href="https://doi.org/10.1038/s41585-019-0171-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30914802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Penticuff,%20J.%20C.,%20Woolbright,%20B.%20L.,%20Sielecki,%20T.%20M.,%20Weir,%20S.%20J.%20&amp;%20Taylor,%20J.%20A.%20MIF%20family%20proteins%20in%20genitourinary%20cancer:%20Tumorigenic%20roles%20and%20therapeutic%20potential.%20Nat.%20Rev.%20Urol.16,%20318%E2%80%93328.%2010.1038/s41585-019-0171-9%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zeng, X. et al. Effects of micronutrients and macronutrients on risk of allergic disease in the European population: A Mendelian randomization study. <em>Food Agric. Immunol.</em><strong>35</strong>, 2442369 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20X.%20et%20al.%20Effects%20of%20micronutrients%20and%20macronutrients%20on%20risk%20of%20allergic%20disease%20in%20the%20European%20population:%20A%20Mendelian%20randomization%20study.%20Food%20Agric.%20Immunol.35,%202442369%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Yadav, A. K. &amp; Desai, N. S. Cancer stem cells: Acquisition, characteristics, therapeutic implications, targeting strategies and future prospects. <em>Stem Cell Rev. Rep.</em><strong>15</strong>, 331–355. 10.1007/s12015-019-09887-2 (2019).
</cite> [<a href="https://doi.org/10.1007/s12015-019-09887-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30993589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yadav,%20A.%20K.%20&amp;%20Desai,%20N.%20S.%20Cancer%20stem%20cells:%20Acquisition,%20characteristics,%20therapeutic%20implications,%20targeting%20strategies%20and%20future%20prospects.%20Stem%20Cell%20Rev.%20Rep.15,%20331%E2%80%93355.%2010.1007/s12015-019-09887-2%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Li, Y. R., Fang, Y., Lyu, Z., Zhu, Y. &amp; Yang, L. Exploring the dynamic interplay between cancer stem cells and the tumor microenvironment: Implications for novel therapeutic strategies. <em>J. Transl. Med.</em><strong>21</strong>, 686. 10.1186/s12967-023-04575-9 (2023).
</cite> [<a href="https://doi.org/10.1186/s12967-023-04575-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10546755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37784157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.%20R.,%20Fang,%20Y.,%20Lyu,%20Z.,%20Zhu,%20Y.%20&amp;%20Yang,%20L.%20Exploring%20the%20dynamic%20interplay%20between%20cancer%20stem%20cells%20and%20the%20tumor%20microenvironment:%20Implications%20for%20novel%20therapeutic%20strategies.%20J.%20Transl.%20Med.21,%20686.%2010.1186/s12967-023-04575-9%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375763/bin/41598_2025_17102_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (536.1KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data is presented by the public database. The scRNA-seq data used in this study are openly available from the GEO databas (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>). Transcriptional regulation analysis from the Gene Transcription Regulation Database (<a href="https://gtrd20-06.biouml.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gtrd20-06.biouml.org/</a>). In addition, gene expression data and survival analyses based on TCGA-CHOL (The Cancer Genome Atlas—Cholangiocarcinoma) cohort and GEPIA (<a href="http://gepia.cancer-pku.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gepia.cancer-pku.cn/</a>) were used to further validate our findings.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-17102-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_17102.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375763/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375763/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375763%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375763/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375763/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375763/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40850954/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375763/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40850954/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375763/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375763/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="SnUM2K1fNBC8kyfmx8bS60bfDf2jEA93P0c9JxXrpFrLdxA2fyIigSN6Tqd6ljtV">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
