
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Novel environment exposure drives temporally defined and region-specific chromatin accessibility and gene expression changes in the hippocampus - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253D468AF26863053D46000DD33E54.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370903/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Novel environment exposure drives temporally defined and region-specific chromatin accessibility and gene expression changes in the hippocampus">
<meta name="citation_author" content="Lisa Traunmüller">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Erin E Duffy">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Hanqing Liu">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Junior Fellow, Society of Fellows, Harvard University, Boston, MA USA">
<meta name="citation_author" content="Stella Sanalidou">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Sebastian Krüttner">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Present Address: Biology of Adversity Project, Broad Institute of MIT and Harvard, Cambridge, MA USA">
<meta name="citation_author" content="Elena G Assad">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Senmiao Sun">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Program in Neuroscience, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Naeem S Pajarillo">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Nancy Niu">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Eric C Griffith">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Michael E Greenberg">
<meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7787">
<meta name="citation_doi" content="10.1038/s41467-025-63029-6">
<meta name="citation_pmid" content="40841540">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370903/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370903/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370903/pdf/41467_2025_Article_63029.pdf">
<meta name="description" content="Exposure to novel environments (NE) induces structural and functional changes in multiple brain areas, including the hippocampus, driven in part by changes in gene expression. However, the cell-type-specific transcriptional and chromatin responses ...">
<meta name="og:title" content="Novel environment exposure drives temporally defined and region-specific chromatin accessibility and gene expression changes in the hippocampus">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Exposure to novel environments (NE) induces structural and functional changes in multiple brain areas, including the hippocampus, driven in part by changes in gene expression. However, the cell-type-specific transcriptional and chromatin responses ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370903/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370903">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63029-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63029.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370903%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370903/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370903/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370903/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7787. doi: <a href="https://doi.org/10.1038/s41467-025-63029-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63029-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Novel environment exposure drives temporally defined and region-specific chromatin accessibility and gene expression changes in the hippocampus</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Traunm%C3%BCller%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lisa Traunmüller</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lisa Traunmüller</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Traunm%C3%BCller%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lisa Traunmüller</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duffy%20EE%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Erin E Duffy</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Erin E Duffy</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duffy%20EE%22%5BAuthor%5D" class="usa-link"><span class="name western">Erin E Duffy</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hanqing Liu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hanqing Liu</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>2</sup>Junior Fellow, Society of Fellows, Harvard University, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hanqing Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanalidou%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Stella Sanalidou</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Stella Sanalidou</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanalidou%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stella Sanalidou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kr%C3%BCttner%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sebastian Krüttner</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sebastian Krüttner</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>4</sup>Present Address: Biology of Adversity Project, Broad Institute of MIT and Harvard, Cambridge, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kr%C3%BCttner%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sebastian Krüttner</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Assad%20EG%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Elena G Assad</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Elena G Assad</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Assad%20EG%22%5BAuthor%5D" class="usa-link"><span class="name western">Elena G Assad</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Senmiao Sun</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Senmiao Sun</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>3</sup>Program in Neuroscience, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Senmiao Sun</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pajarillo%20NS%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Naeem S Pajarillo</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Naeem S Pajarillo</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pajarillo%20NS%22%5BAuthor%5D" class="usa-link"><span class="name western">Naeem S Pajarillo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Niu%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Nancy Niu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Nancy Niu</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Niu%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nancy Niu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Griffith%20EC%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Eric C Griffith</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Eric C Griffith</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Griffith%20EC%22%5BAuthor%5D" class="usa-link"><span class="name western">Eric C Griffith</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Greenberg%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Michael E Greenberg</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Michael E Greenberg</span></h3>
<div class="p">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Greenberg%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael E Greenberg</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Neurobiology, Harvard Medical School, Boston, MA USA </div>
<div id="Aff2">
<sup>2</sup>Junior Fellow, Society of Fellows, Harvard University, Boston, MA USA </div>
<div id="Aff3">
<sup>3</sup>Program in Neuroscience, Harvard Medical School, Boston, MA USA </div>
<div id="Aff4">
<sup>4</sup>Present Address: Biology of Adversity Project, Broad Institute of MIT and Harvard, Cambridge, MA USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 3; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370903  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841540/" class="usa-link">40841540</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on March 1, 2025: "<a href="/articles/PMC11888189/" class="usa-link">
Novel environment exposure drives temporally defined and region-specific chromatin accessibility and gene expression changes in the hippocampus
</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Exposure to novel environments (NE) induces structural and functional changes in multiple brain areas, including the hippocampus, driven in part by changes in gene expression. However, the cell-type-specific transcriptional and chromatin responses to NE remain poorly understood. We employed single-nucleus multiomics and bulk RNA-seq of the hippocampal DG, CA3, and CA1 regions of male mice to profile gene expression and chromatin accessibility following NE exposure. We observed region-specific responses in excitatory neurons and diverse transcriptional changes in inhibitory and non-neuronal cells. NE-regulated genes were enriched for secreted factors, and their cell-type-specific receptor expression highlighted candidate signaling pathways involved in learning and memory. We identified thousands of cell-type-specific chromatin accessibility changes, with coordinated expression and accessibility patterns implicating FOS/AP-1 as a key regulator. These data provide a rich resource of chromatin accessibility and gene expression profiles across hippocampal cell types in response to NE, a physiological stimulus affecting learning and memory.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Genetics of the nervous system, Cellular neuroscience</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Hippocampal neurons adapt to experience through changes in gene expression and chromatin accessibility. Here, authors show that novel environment exposure induces region- and cell-type specific transcriptional changes coordinated by FOS/AP-1.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The mammalian brain is comprised of neuronal and non-neuronal cell types that assemble into a network of functional neural circuits. In addition to cell-intrinsic developmental mechanisms, network assembly and refinement are driven in part by genetic programs that are activated in response to postnatal sensory experience<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. In this regard, post-mitotic neurons continue to adapt their gene expression programs to an ever-changing environment to encode novel experiences and, ultimately, memories. Both cell culture-based studies as well as in vivo stimulation paradigms, including light stimulation of the visual cortex of dark-reared mice and seizure paradigms in the hippocampus, have shown that synaptic activity and/or growth factor signaling induces the transcription of a largely common set of early-response genes (ERGs) that predominantly encode transcription factors (TFs), including FOS and EGR1<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. These ERG TFs in turn regulate a secondary wave of diverse cell-type-specific late-response gene (LRG) programs, which often encode synaptic effector proteins and secreted neuromodulatory proteins tailored to the function of specific cell types within neural circuits<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. A limitation of these prior studies is that the stimuli used to induce the gene expression programs have largely been non-physiological, and the specific features and diversity of gene expression programs that are activated by naturalistic stimuli remained to be identified.</p>
<p id="Par4">Exposure to a novel naturalistic environment (NE) (also referred to as enriched environment) has gained attention as a mode of sensory stimulation that positively influences cognition, improves spatial memory and motor performance, and reduces anxiety<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. In rodents, continuous early-life NE exposure is associated with enhanced performance in learning and memory-associated tasks during adulthood<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Moreover, prolonged novel environment exposure has also been shown to improve cognitive function during aging<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> and both attenuate disease progression and ameliorate behavioral deficits in diverse mouse models of neurodevelopmental and neurodegenerative conditions<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Previous efforts have characterized the gene expression outcomes that accompany structural changes after multiple weeks to months of NE exposure<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. However, the gene regulatory elements and the subsequent genes expression changes that initiate NE-dependent circuit remodeling remain largely uncharacterized. Indeed, to date, there has been little systematic exploration of acute gene expression programs in the hippocampus in response to naturalistic stimuli.</p>
<p id="Par5">To bridge this gap in knowledge, we employed both region-specific RNA sequencing (RNA-seq) from the three main excitatory regions of the mouse hippocampus (Cornu Ammonis 1 (CA1), CA3, and dentate gyrus (DG)) and single-nucleus multiome sequencing (snMultiome-seq) to systematically characterize acute NE-responsive gene expression and chromatin accessibility patterns in the mouse hippocampus. We found that, whereas induction of ERG TFs was observed across all three hippocampal regions and in multiple cell types, novel experience-induced genes were expressed in a highly region- and cell-type-specific manner and were enriched for secreted proteins that function as neuromodulators and likely regulate neuronal connectivity and plasticity. We also profiled NE-responsive genomic cis-regulatory regions, uncovering a widespread role for AP-1 factors in these cell-type-specific responses. Taken together, our findings identify the chromatin accessibility and gene expression landscape following NE exposure across 14 cell types of the hippocampus and provide a rich resource for the functional interrogation of activity-dependent genes and the gene regulatory elements that drive their expression in the context of spatial learning and memory paradigms. These datasets are accessible via our accompanying web-based searchable database, where users can input gene names and extract plots of gene expression and FOS/AP-1 accessibility across all cell classes and times of NE exposure (<a href="https://greenberg.hms.harvard.edu/project/novel-environment-gene-database/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://greenberg.hms.harvard.edu/project/novel-environment-gene-database/</a>).</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Transcriptional landscape following NE exposure</h3>
<p id="Par6">To characterize hippocampal gene expression kinetics in response to brief NE exposure, we transferred adult C57BL/6 J mice from their home cage (HC) to a novel context for 30 min, followed by a return to HC (Fig. <a href="#Fig1" class="usa-link">1A</a>). Immunostaining for FOS and EGR1, two well-characterized ERG TFs, confirmed robust ERG induction within two hours of NE exposure (Fig. <a href="#Fig1" class="usa-link">1B</a>, Fig. <a href="#MOESM1" class="usa-link">S1A</a>), as expected<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Having confirmed the efficiency of this stimulation paradigm, we employed two complementary approaches to systematically characterize NE-induced gene expression programs. First, bulk RNA-seq was used to achieve deep sequencing coverage of both nuclear and cytoplasmic RNA from samples collected following a time course of brief NE exposure. To this end, mice were sacrificed at ten time points, spanning a 24 h period following the start of a 30 min NE exposure (with the exception of the 15 min time point, where mice were immediately sacrificed after 15 min NE exposure). Hippocampal tissue was micro-dissected into CA1, CA3, and DG regions and libraries prepared for deep RNA sequencing (Fig. <a href="#MOESM1" class="usa-link">S1B</a>). Principal component analysis (PCA) effectively separated bulk RNA-seq samples by anatomical region, highlighting cell-class-specific gene expression programs and the reproducibility of our microdissection (Figs. <a href="#Fig1" class="usa-link">1C</a>, <a href="#MOESM1" class="usa-link">S1C</a>). Second, to profile cell-type-specific gene expression and chromatin accessibility changes, hippocampal nuclei were isolated 1 h post-NE exposure for single-nucleus 10X multiome (snRNA + snATAC) sequencing. Following dimensionality reduction, the BRAIN Initiative Cell Census Network (BICCN) snRNA-seq atlas was used to assign 14 major cell populations (Fig. <a href="#Fig1" class="usa-link">1D</a>), which could be further clustered into 33 sub-class cell populations (Fig. <a href="#MOESM1" class="usa-link">S2A</a>). After applying quality control metrics (see Methods, Fig. <a href="#MOESM1" class="usa-link">S2B–D</a>), we obtained transcriptomes from a total of 66,345 nuclei, with the resulting cell clusters well integrated between HC and NE samples (Fig. <a href="#MOESM1" class="usa-link">S2B</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. NE exposure stimulates hippocampal ERG activity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370903_41467_2025_63029_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9924/12370903/4376c20b9e60/41467_2025_63029_Fig1_HTML.jpg" loading="lazy" id="d33e448" height="675" width="698" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Overview of experimental design. <strong>B</strong> Representative immunofluorescence images of FOS and EGR1 protein levels in CA1, CA3, and DG from HC and 90 min following a 30 min NE exposure. Scale bar = 20 µm. <em>n</em> = 1 biological replicate. <strong>C</strong> Principal component (PC) analysis of read depth-normalized RNA-seq counts from the top 2000 most variable genes across CA1, CA3, and DG. Each point is a sample colored by region and time following the start of a 30 min NE exposure (except for 15 min). HC home cage. <em>n</em> = 4 independent biological replicates per time point except HC (<em>n</em> = 5) and CA1 18 h NE (<em>n</em> = 3). <strong>D</strong> UMAP visualization of snRNA-seq from nuclei from all collected timepoints, with cell type information overlaid. CGEGaba GABAergic neurons originating from the caudal ganglionic eminence (e.g., VIP and CCK), CRGlut Cajal-Retzius glutamatergic neurons, ENTGlut entorhinal glutamatergic neurons, IMN immature neurons (e.g., adult-born stem cells), MGEGaba GABAergic neurons originating from the medial ganglionic eminence (e.g., PV and SST), Oligo oligodendrocytes, OtherNN other non-neuronal cells, PPPGlut glutamatergic neurons originating from the presubiculum, parasubiculum, and postsubiculum, SUBGlut glutamatergic neurons originating from the subiculum. <em>n</em> = 66,345 cells, 2 independent biological replicates per time point.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Region-specific differences in home cage gene expression</h3>
<p id="Par7">We further characterized region-specific gene expression differences in HC animals as an additional test of data quality. Hierarchical clustering of differentially expressed genes identified in pairwise comparisons between HC CA1, CA3, and DG bulk RNA-seq samples was used to define region-enriched genes (Fig. <a href="#Fig2" class="usa-link">2A</a>, Fig. <a href="#MOESM1" class="usa-link">S3A</a>). Our snRNA-seq dataset confirmed that excitatory neurons were major contributors to region-specific gene expression differences. In addition, within CA1 and CA3, MGE-derived GABAergic neurons (e.g., PV and SST inhibitory neurons), immature neurons, and non-neuronal cells (e.g., oligodendrocytes) contribute to the region-specific differences (Fig. <a href="#Fig2" class="usa-link">2A</a>). These cell-type- and region-specific gene expression differences strongly align with example genes from prior studies (Fig. <a href="#MOESM1" class="usa-link">S3B</a>)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, with CA1-, CA3-, and DG-enriched gene expression signatures reflecting their specialized roles in synaptic signaling and neuronal function (Figs. <a href="#Fig2" class="usa-link">2B, C</a>, Gene Ontology <a href="#MOESM1" class="usa-link">S3C</a>) that are then further modified by neuronal activity.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Region-specific differences in HC gene expression.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370903_41467_2025_63029_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9924/12370903/db607bcaf5a2/41467_2025_63029_Fig2_HTML.jpg" loading="lazy" id="d33e513" height="836" width="682" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Heatmap of row-normalized z-scores from bulk RNA-seq and snRNA-seq expression across all regions and major cell types, respectively. Each bulk and snRNA-seq column represents a biological replicate (<em>n</em> = 5, <em>n</em> = 2, respectively) separated by region. Bulk RNA-seq CA1-, CA3-, and DG-enriched genes were determined by k-means clustering. Significant genes were defined as DESeq2 P<sub>adj</sub> &lt; 0.05 (using a two-sided Wald test with Benjamini-Hochberg multiple hypothesis correction) and Fold Change |FC | &gt; 2. <strong>B</strong> Merged genome browser tracks (top), box and whisker plots (bottom left) of bulk RNA-seq DSEq2 depth-normalized counts, and violin plots (bottom right) of snRNA-seq counts for <em>Gabrg3</em> (CA1-enriched), <em>Chgb</em> (CA3-enriched), and <em>C1ql3</em> (DG-enriched). Box plots are shown as median ± IQR (whiskers = 1.5*IQR). Violin plots are shown with mean as a dash. <strong>C</strong> Schematic of region-enriched synaptic genes in CA1, CA3, and DG based on differential gene expression analysis and Gene Ontology.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Time-resolved NE-driven gene expression changes</h3>
<p id="Par8">Having characterized basal hippocampal region-specific gene expression differences, we next investigated the nature of NE-driven gene expression responses. Differential gene expression analysis comparing each region-specific bulk RNA-seq dataset to the appropriate HC sample revealed 704 unique genes that were significantly differentially expressed following NE exposure (Figs. <a href="#Fig3" class="usa-link">3A</a>, <a href="#MOESM1" class="usa-link">S4A</a>, <a href="#MOESM1" class="usa-link">S5A</a>, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>, P<sub>adj</sub> &lt; 0.05 and abs (fold change) &gt; 1.2, see Methods), whereas snRNA-seq analysis after 1 h of NE exposure identified 5210 unique genes as differentially regulated (Figs. <a href="#Fig3" class="usa-link">3B</a>, <a href="#MOESM1" class="usa-link">S4</a>B, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>, P<sub>adj</sub> &lt; 0.05 and abs (fold change) &gt; 1.2). Intriguingly, excitatory neuron classes such as CA1, CA3 and DG exhibited the most pronounced changes in our snRNA-seq dataset, which are the most abundant cell classes in the mouse hippocampus, in addition to oligodendrocytes (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. NE-driven gene expression changes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370903_41467_2025_63029_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9924/12370903/10c62772212f/41467_2025_63029_Fig3_HTML.jpg" loading="lazy" id="d33e596" height="1011" width="763" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Jitter plot of log<sub>2</sub>(fold change) for significantly differentially expressed genes (DEG). DESeq2 Padj &lt;0.05, two-sided Wald test with Benjamini-Hochberg correction, |FC | &gt; 1.2 in CA1, CA3 and DG by bulk RNA-seq across NE exposure time points. Red: upregulated; blue: downregulated. <strong>B</strong> Jitter plot of significant DEGs (Padj &lt;0.05, two-sided Wilcoxon Rank Sum Test with Bonferroni correction, |FC | &gt; 1.2) in major cell types by snRNA-seq at 1 h after NE (30 min NE + 30 min HC). Red: upregulated; blue: downregulated. <strong>C</strong> Heatmap of bulk RNA-seq log<sub>2</sub>(fold change) in CA1, CA3, and DG across NE exposure. Genes were defined as early ( &lt; 2 h)-response genes based on the first time point at which |FC | &gt; 1.2 and separated into CA1-, CA3-, and DG-enriched or Common based on whether fold change requirements were satisfied in one or more than one region, respectively. <strong>D</strong> Heatmap as in (<strong>C</strong>), showing early-response transcription factors. Every other row labeled; full gene list in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. <strong>E</strong> Merged genome browser tracks (top) and bulk RNA-seq line plots of DESeq2 depth-normalized counts for <em>Fos</em> in CA1, CA3, and DG. Lines = mean ± SEM; black = Padj &lt;0.05; gray = Padj &gt; 0.05 (two-sided Wald test with Benjamini-Hochberg correction). <strong>F</strong> Line plots of bulk RNA-seq counts for <em>Nr4a3</em> (CA1), <em>Atoh8</em> (CA3), and <em>Bcl6</em> (DG). Lines = mean ± SEM; black = Padj 0.05 (two-sided Wald test with Benjamini-Hochberg correction).</p></figcaption></figure><p id="Par9">Of the 704 differentially regulated genes in our bulk sequencing comparisons, 570 genes show significant differential expression within 2 h of stimulus onset (Figs. <a href="#Fig3" class="usa-link">3C</a>, <a href="#MOESM1" class="usa-link">S5B</a>), with ERG genes being enriched for GO terms related to transcription factor activity (Fig. <a href="#Fig3" class="usa-link">3D</a>, Gene Ontology Fig. <a href="#MOESM1" class="usa-link">S5C</a>, Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). Clustering these genes based on their region-specific induction kinetics revealed a subset of early-response genes that were significantly induced across all three regions (Fig. <a href="#Fig3" class="usa-link">3C, D</a>), including well-characterized ERG TF family members (e.g. <em>Fos, Fosb, Npas4, Nr4a1</em>, and <em>Egr3</em>) (Fig. <a href="#Fig3" class="usa-link">3D, E</a>). By contrast, other early-response TFs exhibited regional specificity, including the ERG TFs <em>Nr4a3</em>, <em>Atf4</em>, and <em>Klf4</em> (CA1-induced), the bHLH TF <em>Atoh8</em> (CA3-induced), and <em>Bcl6</em> (DG-induced), a transcriptional repressor of multiple developmental pathways that is thought to promote neurogenesis<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> (Fig. <a href="#Fig3" class="usa-link">3D, F</a>). We also identified many NE-induced synaptic factors, including the plasticity-related proteins <em>Arc</em> and <em>Homer1</em> (common), the synaptic vesicle protein <em>Synpr</em> (CA1-induced), the regulator of synaptic membrane exocytosis <em>Rims4</em> (CA3-induced), and <em>Dact1</em> (DG-induced), a member of the Wnt pathway (Fig. <a href="#MOESM1" class="usa-link">S5B</a>). Activity-regulated genes that showed significant expression changes in both bulk and scRNA-seq were mostly regulated in excitatory neurons, with a subset showing induction across inhibitory and non-neuronal cell types (Fig. <a href="#MOESM1" class="usa-link">S5D</a>).</p>
<p id="Par10">Coordinated ERG expression has been observed in individual cells in the visual cortex in response to light stimulation following dark rearing<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. We therefore analyzed the correlation of commonly induced ERG expression across single excitatory neurons in CA1, CA3, and DG. ERGs exhibited significantly stronger pair-wise correlations within individual cells compared to similarly expressed, non-induced genes (Fig. <a href="#MOESM1" class="usa-link">S5E</a>). Our results thus show that coordinate ERG expression is a consistent phenomenon across multiple brain regions and occurs in response to a subtle, naturalistic stimulus such as exposure to a novel environment.</p>
<p id="Par11">ERG TFs induce a second wave of LRGs that drive structural and functional changes related to plasticity<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. We identified 134 LRGs by bulk RNA-seq that showed significant differential expression ≥ 2 h after brief NE exposure (Fig. <a href="#Fig4" class="usa-link">4A</a>, P<sub>adj</sub> &lt; 0.05 and abs (fold change) &gt; 1.2, see Methods). Most LRGs are specifically induced in one hippocampal region, such as <em>Dclk3</em> (DG-induced), a serine-threonine kinase thought to regulate the activity of many TFs and chromatin remodelers<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, <em>Mgp</em> (CA3-induced), a matrix-associated protein and regulator of the BMP growth factor pathway<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, and <em>Ttl</em> (CA1-induced), which enables tubulin-tyrosine ligase activity that is required for microtubule organization and axon extension (Fig. <a href="#Fig4" class="usa-link">4B</a>)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. NE-driven late-response genes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370903_41467_2025_63029_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9924/12370903/52b2a010a5d9/41467_2025_63029_Fig4_HTML.jpg" loading="lazy" id="d33e759" height="1015" width="726" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Heatmap of log<sub>2</sub>(fold change) in gene expression relative to HC at various time points following 30 min NE exposure in CA1, CA3, and DG by bulk RNA-seq. Genes were defined as late (≥2 h)-response genes based on the first time point at which |FC | &gt; 1.2 (excluding early-response genes from Fig. <a href="#Fig3" class="usa-link">3C</a>) and separated into CA1-, CA3-, and DG-enriched or Common based on whether fold change requirements were satisfied in one or more than one region, respectively. Number of genes in each cluster is indicated on the left. <strong>B</strong> Line plots of bulk RNA-seq DESeq2 depth-normalized counts for <em>Ttl1</em>, <em>Mgp</em>, and <em>Dclk3</em> following NE exposure in CA1, CA3, and DG, respectively. Lines = mean ± SEM; black = Padj 0.05 (two-sided Wald test with Benjamini-Hochberg correction). <strong>C</strong> Heatmap of log<sub>2</sub>(fold change) in gene expression of all significant NE-regulated genes that are annotated secreted factors relative to HC at each time point of NE exposure in CA1, CA3, and DG by bulk RNA-seq. <strong>D</strong> Heatmap of secreted factors (ligands) from (<strong>C</strong>) with their receptors. Left grid shows ligand-receptor pair, with dark shading indicating whether ligand expression was upregulated (dark red) or downregulated (dark blue) in response to NE. Background shading indicates the hippocampal region of ligand NE gene expression regulation (common, CA1- CA3- or DG-enriched). Right heatmap shows row-normalized z-scores of cognate receptor expression from HC snRNA-seq across all major cell types, ordered by cell type of maximum receptor expression.</p></figcaption></figure><p id="Par12">A subset of genes in CA1 and CA3 showed statistically significant expression changes only at 8 h following NE exposure. However, inspection of normalized counts across the full time course revealed that these changes accumulate gradually over time, peak at 8 h, and return to baseline by 18 h, which is consistent with a sustained but transient transcriptional response (Figs. <a href="#Fig4" class="usa-link">4A</a>, <a href="#MOESM1" class="usa-link">S6A–C</a>). Such changes include the gradual upregulation of <em>Camkk2</em>, encoding a kinase involved in the regulation of long-term potentiation at hippocampal CA1 synapses, a process required for learning and memory<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, and the down-regulation of the transcription factor ETS proto-oncogene 1 (<em>Ets1</em>), which has been linked to axonal growth<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. We further validated the CA3-specific induction after 8 h of NE exposure for <em>Mgp</em> via Fluorescent In Situ Hybridization (FISH, Fig. <a href="#MOESM1" class="usa-link">S6D-F</a>, Supplementary Data <a href="#MOESM6" class="usa-link">4</a>). Thus, short-term exposure to a novel environment leads to prolonged transcriptional changes in a number of cell class-specific genes that may contribute to the encoding of experiences in the hippocampus.</p>
<p id="Par13">To further characterize time-resolved gene expression dynamics following NE exposure, we compared our bulk RNA-seq and snRNA-seq datasets. While the two approaches do not show perfect concordance, we reasoned that discrepancies may arise from differences in the abundance of mature versus nascent transcripts: bulk RNA-seq captures primarily cytoplasmic, spliced RNAs, whereas snRNA-seq is enriched for nuclear, unprocessed pre-mRNAs. To test whether newly transcribed pre-mRNAs displayed similar expression patterns by bulk RNA-seq and snRNA-seq, we quantified reads in our bulk RNA-seq data that aligned to introns, which would enrich for unspliced pre-mRNAs, and performed DEG analysis between HC and 1 h brief NE. The directionality of fold changes was preserved between our bulk- and snRNA-sequencing datasets, and the results showed a reliable and statistically significant correlation between the two datasets (Fig. <a href="#MOESM1" class="usa-link">S7A–D</a>). This correlation was strongest in CA1, which is likely due to a greater recruitment of activated cells in this region compared to CA3 and DG as indicated by greater ERG induction (Fig. <a href="#Fig3" class="usa-link">3D, E</a>). We further validated the experience-dependent induction of mature, but not immature, <em>Egr3</em> RNA and demonstrated the sensitivity and specificity of our datasets by including <em>Grin1</em>, a gene with stable expression, as a negative control using FISH (Fig. <a href="#MOESM1" class="usa-link">S8</a>). These results suggest that the apparent discrepancy in the number of NE-responsive genes in bulk RNA-seq versus snRNA-seq reflects, at least in part, differences in RNA species captured by each method rather than opposing regulatory programs.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Activity-dependent regulation of secreted late-response genes following NE exposure</h3>
<p id="Par14">Overall, upregulated LRGs were enriched for gene ontology functions related to extracellular secretion, (Fig. <a href="#MOESM1" class="usa-link">S5C</a>, Supplementary Data <a href="#MOESM4" class="usa-link">2</a>, Extracellular region, FDR = 0.00803867; extracellular space, FDR = 0.04722025), such as <em>Ccn1</em> (common), a secreted factor involved in cell adhesion, migration, and neuroinflammation processes critical for synaptic plasticity and tissue repair, as well as <em>Inhba</em> (CA1-induced), a member of the transforming growth factor-β (TGFβ) superfamily, <em>Plat</em> (CA3-enriched), which encodes the protein tPA that cleaves neurotransmitter receptors, and the endoplasmic reticulum calcium-binding chaperone <em>Calr</em> (DG-induced) (Fig. <a href="#Fig4" class="usa-link">4C</a>). Novel environment exposure thus triggers region-specific LRG programs that may promote circuit-specific synaptic plasticity.</p>
<p id="Par15">To explore whether secreted factors induced in excitatory neurons could influence neighboring neuronal and non-neuronal cell types, we mapped the expression of known cognate receptors<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> of NE-dependent secreted factors across cell types in our single-nucleus RNA-seq (snRNA-seq) data, revealing receptor expression across multiple hippocampal cell types (Fig. <a href="#Fig4" class="usa-link">4D</a>). Interestingly, the receptors for <em>Ccn1</em>, a NE-induced cell adhesion factor, are mainly expressed in microglia, consistent with prior reports that CCN1 activates this cell type<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. This tight temporal regulation of <em>Ccn1</em> expression may thus contribute to proper hippocampal function insofar as temporally constrained microglial activation is important for dendritic spine phagocytosis, whereas prolonged activation of microglia has been associated with neurodegenerative conditions<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. By contrast, brain-derived neurotrophic factor (<em>Bdnf</em>, CA1-upregulated) and neurotrophin-3 <em>(Ntf3</em>, DG-downregulated) bind to receptors in excitatory, inhibitory, and non-neuronal cells, which may in part explain their widespread roles in promoting neuronal survival, differentiation, and synaptic connectivity<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Intriguingly, we find that receptors for CA3-induced ligands tend to be restricted to excitatory neurons, whereas receptors for CA1-induced ligands (e.g., <em>Bdnf</em> and <em>Inhba</em>) are broadly expressed across many cell types, suggesting that secreted factors that are NE-induced in excitatory neurons may coordinate a wide range of cellular responses to promote aspects of learning and memory, not only in neighboring neurons but also non-neuronal cells.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Cell-type-specific NE-induced gene expression programs</h3>
<p id="Par16">We next sought to explore the cell-type-specificity of NE-regulated gene expression by snRNA-seq. Specifically, we classified genes based on whether their expression changes were common across multiple cell classes (significant in any combination of two cell types: excitatory, inhibitory, or non-neuronal), or specifically enriched in one of these classes. We observed 1762 genes that showed common regulation across multiple cell classes, 7996 genes that were excitatory-enriched, 27 genes that were inhibitory-enriched, 244 genes that were enriched in non-neuronal cells, and 14 genes that were enriched in both inhibitory and non-neuronal cells (Fig. <a href="#Fig5" class="usa-link">5</a>). For example, membrane palmitoylated protein 6 (<em>Mpp6</em>), a membrane protein suggested to be required for synaptic organization and function, was induced in MGE-derived parvalbumin+ and somatostatin+ inhibitory neurons<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. By contrast, CGE inhibitory neurons showed preferential induction of F-box and WD repeat domain-containing 7 (<em>Fbxw7</em>), which is required for neuronal development and signaling by regulating the stability of scaffolding proteins such as Disrupted in schizophrenia1 (DISC1)<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. The histone deacetylase 4 (<em>Hdac4</em>), which mediates energy metabolism, was upregulated in astrocytes<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Finally, <em>Adipor2</em> (Adiponectin receptor protein 2), a key hormone for the regulation of glucose and lipid metabolism<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> that has been linked to reduced extinction learning in contextual fear conditioning is selectively induced in oligodendrocytes<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Our findings indicate that different cell classes in the hippocampus have specialized functions in responding to NE exposure, with excitatory neurons showing the most pronounced induction of gene expression and non-neuronal cells and inhibitory neurons displaying significant responses as well.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Cell-type-specific NE-induced gene expression programs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370903_41467_2025_63029_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9924/12370903/0214fdb26118/41467_2025_63029_Fig5_HTML.jpg" loading="lazy" id="d33e968" height="838" width="683" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Heatmap of log<sub>2</sub>(fold change) in gene expression relative to HC at 1 h following the initiation of a 30 min NE exposure in all major cell types by snRNA-seq. Genes were separated based on their induction in one or more cell classes (excitatory, inhibitory, or non-neuronal), with the number of significant genes specific for each cell class also indicated. <strong>B</strong> Dot plot of the top common or excitatory-specific induced genes. Dot color indicates log<sub>2</sub>(fold change); size indicates -log<sub>10</sub>(P<sub>adj</sub>). Black outline indicates P<sub>adj</sub> &lt; 0.05 using a two-sided Wilcoxon ranked-sum test with Bonferroni multiple hypothesis correction. <strong>C</strong> Dot plot of the top induced genes in non-neuronal cell types. Dot color indicates log<sub>2</sub>(fold change); size indicates -log<sub>10</sub>(P<sub>adj</sub>). Black outline indicates P<sub>adj</sub> &lt; 0.05 using a two-sided Wald test with Benjamini-Hochberg multiple hypothesis correction. <strong>D</strong> Dot plot of the top induced genes in inhibitory neuron cell types that are not significantly induced in any excitatory cell type. Dot color indicates log<sub>2</sub>(fold change); size indicates -log<sub>10</sub>(P<sub>adj</sub>). Black outline indicates P<sub>adj</sub> &lt; 0.05 using a two-sided Wilcoxon ranked-sum test with Bonferroni multiple hypothesis correction.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Continuous NE exposure boosts expression of LRGs</h3>
<p id="Par17">We were surprised to find that very few previously characterized LRGs showed significant gene expression changes 6 h after a brief (30 min) exposure to NE (Figs. <a href="#Fig3" class="usa-link">3</a>A, <a href="#MOESM1" class="usa-link">S4A</a>), as robust LRG expression at this time point has previously been observed in cultured neurons exposed to membrane-depolarizing agents<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> and growth factor stimulation<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> as well as in vivo in response a kainic acid-induced seizure<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. To determine if exposure to a novel environment for a longer period of time was more effective at inducing LRGs in the hippocampus, we transferred adult C57BL/6 J mice from their home cage (HC) to a novel context for 6 h (termed continuous NE, cNE) and prepared bulk or snMultiome-seq libraries as described above (Figs. <a href="#Fig6" class="usa-link">6</a>A, <a href="#MOESM1" class="usa-link">S9A</a>). This longer exposure to novelty led to the identification of differentially expressed genes in both bulk RNA-seq and snRNA-seq (2371 and 1921 genes, respectively) (Figs. <a href="#Fig6" class="usa-link">6</a>B, <a href="#MOESM1" class="usa-link">S9B, C</a>). As observed in response to brief NE, LRGs in response to cNE were largely non-overlapping between CA1, CA3 and DG (Fig. <a href="#Fig6" class="usa-link">6C, D</a>). Bulk RNA-seq showed more LRGs induced in the DG compared to CA1 and CA3 in response to cNE (Figs. <a href="#Fig3" class="usa-link">3</a>A, <a href="#MOESM1" class="usa-link">S9B, D</a>). Upregulated genes were enriched for GO terms associated with responses to extracellular stimuli, while downregulated genes were enriched for terms related to synaptic function and energy metabolism, particularly in the DG (Fig. <a href="#Fig6" class="usa-link">6E</a>). Intriguingly, we identified secreted factors whose expression was modulated by cNE as measured by bulk RNA-seq, which revealed both overlapping (<em>Inhba, Wnt9a</em>) and distinct (<em>Fbn1, Lama1, Slit2</em>) ligands compared to brief NE (Figs. <a href="#Fig6" class="usa-link">6F</a>, <a href="#MOESM1" class="usa-link">S10A–C</a>). These ligands bind receptors that are expressed across diverse excitatory, inhibitory and non-neuronal cell types, suggesting their potential to regulate tissue-wide cellular functions in response to NE (Fig. <a href="#MOESM1" class="usa-link">S10A, B</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Continuous NE exposure boosts LRG expression.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370903_41467_2025_63029_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9924/12370903/7bfdc68a4fc5/41467_2025_63029_Fig6_HTML.jpg" loading="lazy" id="d33e1096" height="1010" width="670" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Overview of experimental design. <em>n</em> = 4 independent biological replicates for bulk RNA-seq. <em>n</em> = 2 biological replicates for snMultiome-seq (<strong>B</strong>) Left: jitter plot of log<sub>2</sub>(fold change) of significant (DESeq2 P<sub>adj</sub> &lt; 0.05 using a two-sided Wald test with Benjamini-Hochberg multiple hypothesis correction &amp; |FC | &gt; 1.2) differentially expressed genes in CA1, CA3, and DG following cNE exposure. Right: jitter plot of log<sub>2</sub>(fold change) of significant (P<sub>adj</sub> &lt; 0.05 using Wilcoxon Rank Sum Test with Bonferroni multiple hypothesis correction, |FC | &gt; 1.2) differentially expressed genes in all major cell types by snRNA-seq following cNE exposure. Upregulated and downregulated genes are plotted as positive and negative values on the y-axis, respectively. <strong>C</strong> Venn diagram of overlap in differentially expressed genes following cNE exposure in CA1, CA3, and DG. <strong>D</strong> Heatmap of log<sub>2</sub>(fold change) in gene expression relative to HC in response to cNE exposure in CA1, CA3, and DG. Genes were separated into upregulated or downregulated clusters by k-means clustering, with the numbers of genes in each cluster indicated on the left. <strong>E</strong> Dot plot of the top enriched GO terms for upregulated and downregulated genes in (<strong>D</strong>), separated by hippocampal region. P<sub>adj</sub> calculated using Fisher’s exact test followed by FDR multiple hypothesis correction. <strong>F</strong> Merged genome browser tracks (left) and box and whisker plots of bulk RNA-seq DESeq2 depth-normalized counts (right) for <em>Inhba</em> and <em>Lama1</em> following brief 6 h and continuous 6 h novel environment exposure (bNE and cNE, respectively). Box plots shown as median ± IQR (whiskers = 1.5*IQR).</p></figcaption></figure><p id="Par18">We considered whether the differences between brief and continuous NE reflected fundamentally distinct gene expression programs induced by different stimulation durations<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, or whether continuous NE leads to a cumulative increase in gene expression over time as RNA gradually accumulates in the cytoplasm. In this regard, we found that the directionality of expression changes between brief NE and cNE were not markedly different. Rather, the magnitude of expression changes was greater in response to cNE. For example, <em>Inhba</em>, a gene encoding a protein involved in hormone secretion, was significantly induced in CA1 in response to both brief and continuous NE, but we observed a larger induction following 6 h of cNE (Fig. <a href="#Fig6" class="usa-link">6F</a>). In contrast, we observed a significant induction of <em>Lama1</em>, encoding a laminin subunit thought to mediate cell attachment and migration<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, in DG, but only in response to cNE. Moreover, focusing on our snRNA-seq datasets, which preferentially capture newly transcribed RNA, we found largely overlapping gene expression changes between brief and continuous NE exposure (Fig. <a href="#MOESM1" class="usa-link">S10D</a>), consistent with similar gene expression responses to these two conditions. The strong correlation between nuclear RNA and total RNA changes observed at 1 h (Fig. <a href="#MOESM1" class="usa-link">S7C, D</a>) was no longer observed after 6 h of continuous NE exposure (Fig. <a href="#MOESM1" class="usa-link">S11A–D</a>), further arguing that the gene expression changes observed by bulk RNA-seq are due to a shift in cytoplasmic mRNA levels.</p>
<p id="Par19">Together, these data support the hypothesis that cNE causes repeated activation of activity-dependent ERGs, either in the same cells or in a larger population of neurons, leading to a gradual cytoplasmic build-up of late-response transcripts that may be important for shaping long-term hippocampal circuit connectivity. Notably, we further compared our cNE bulk dataset to gene expression changes in response to kainic acid-induced seizures, a commonly used stimulus in genomic identification of activity-dependent gene expression<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. These stimulation paradigms induce vastly different magnitudes and directionality of gene regulation. For example, when we focused our attention on a select list of kainic acid-induced candidate genes, we discovered that NE also induced genes such as <em>Scg2, Inhba, Pcsk1 or Nptx2</em> but downregulated <em>Lonrf1, Cgref1</em> and <em>Adpgk</em> (Fig. <a href="#MOESM1" class="usa-link">S12</a>). This differential response suggests that, while seizures trigger a broad and robust activation of genes linked to stress and excitotoxicity, NE exposure may instead promote more selective transcriptional programs, potentially favoring pathways involved in neuroplasticity, adaptation, and homeostasis.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Accessibility of AP-1 sites predicts NE-induced gene upregulation</h3>
<p id="Par20">We next examined NE-regulated chromatin accessibility changes in our snMultiome datasets to gain insight into the regulatory mechanisms that mediate these novel experience-activated gene programs. In this regard, cell clustering based on chromatin accessibility largely matched that based on gene expression (Fig. <a href="#MOESM1" class="usa-link">S13A</a>), where cell clusters from distinct NE time points were well integrated and passed all quality control metrics (Fig. <a href="#MOESM1" class="usa-link">S13B</a>). Differential accessibility analysis identified thousands of regions exhibiting altered accessibility relative to HC in response to 1 h brief NE and 6 h cNE across different cell types, with many NE-regulated peaks showing strong specificity for only one or a few cell types (Figs. <a href="#Fig7" class="usa-link">7A</a>, <a href="#MOESM1" class="usa-link">S13C</a>). For example, ATAC peaks surrounding the <em>Scg2</em> gene body showed a mix of two types of peaks: 1) NE-induced putative enhancer regions that are open across many cell types, or 2) NE-induced cell-class-specific, open regions for CA1 or CA3 pyramidal neurons (Fig. <a href="#Fig7" class="usa-link">7B</a>). Motif enrichment analysis revealed strong enrichment for a number of ERG TF-binding motifs at sites that exhibited increased accessibility in CA1 and CA3 pyramidal neurons as well as DG granule cells in response to NE exposure (Fig. <a href="#MOESM1" class="usa-link">S14A</a>). Strikingly, the FOS/AP-1 motif exhibited the highest enrichment in all three excitatory cell types at 1 h NE, and still exhibited strong enrichment in CA1 and CA3 pyramidal neurons at 6 h cNE with no enrichment in DG granule cells, suggesting a cell class-specific regulation of chromatin regions over time in the mouse hippocampus (Fig. <a href="#MOESM1" class="usa-link">S14A</a>). We further utilized ChromVAR<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, a computational method that analyzes single-nuclei chromatin accessibility data to identify TFs whose binding motifs display variable accessibility across cells. We estimated the fraction of cells in each population that increased the accessibility of regions in response to the 1 h NE stimulation paradigm (Fig. <a href="#MOESM1" class="usa-link">S13D, E</a>, Methods) and identified that the FOS/AP-1 motif was highly enriched in peaks of neuronal populations and de-enriched in peaks of oligodendrocytes or microglia in response to NE (Fig. <a href="#Fig7" class="usa-link">7C, D</a>, Fig. <a href="#MOESM1" class="usa-link">S13</a>E, <a href="#MOESM1" class="usa-link">S14</a>B, <a href="#MOESM1" class="usa-link">S15A</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Chromatin accessibility changes in response to NE at single-cell resolution.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370903_41467_2025_63029_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9924/12370903/3211326a7831/41467_2025_63029_Fig7_HTML.jpg" loading="lazy" id="d33e1239" height="644" width="683" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Heatmaps of overall log<sub>10</sub>(counts per million) peak accessibility (left) and NE-induced log<sub>2</sub>(fold change) (right) across major cell types from snATAC-seq after homecage, 1 h brief, or 6 h continuous NE exposure (H, 1 &amp; 6, respectively). Cell clusters were defined using the BICCN Brain Atlas. CA1/CA3 excitatory neurons, Inhib inhibitory neurons, RHP Glut retrohippocampal glutamatergic neurons, Astro astrocytes, Other NonN other non-neuronal types. Numbers indicate peak counts per category. <strong>B</strong> Genome browser tracks of chromatin accessibility surrounding the <em>Scg2</em> locus, merged by cell type. Red and green shading highlights regions of CA1- or CA3-specific NE-induced opening (1 h brief or 6 h continuous), and grey shading marks common accessible regions. <strong>C</strong> UMAP of FOS/AP-1 motif accessibility using ChromVAR scores in HC, 1 h brief and 6 h continuous NE conditions. <strong>D</strong> Box and whisker plots of FOS/AP-1 ChromVAR scores in major cell types. Boxes show median ± IQR; whiskers = 1.5× IQR. Significance determined via two-sided Mann–Whitney U test with Benjamini-Hochberg (BH) correction: <em>P</em> &lt; 0.05 (*)<em>, P</em> &lt; <em>0.0001</em> (****). Exact values for each comparison are provided in Supplementary Data <a href="#MOESM5" class="usa-link">3</a>. <strong>E</strong> FOS/AP-1 ChromVAR scores stratified by gene expression changes (upregulated: red; downregulated: blue; not significant: grey) in snRNA-seq after 1 h NE. Within each cell type, differences were assessed using a two-sided Kruskal–Wallis test with Benjamini-Hochberg correction. For cell classes with a significant overall effect (FDR &lt; 0.05), post-hoc pairwise comparisons were performed using the Wilcoxon rank-sum tests with Benjamini-Hochberg correction: <em>P</em> &lt; 1e–10 (****). Boxes show median ± IQR; whiskers = 1.5× IQR.</p></figcaption></figure><p id="Par21">To investigate whether increased FOS/AP-1 motif accessibility predicts gene induction, we first binned cells into quantiles based on their ChromVAR scores, with higher scores indicating greater NE-induced accessibility of FOS/AP-1 motifs (Fig. <a href="#MOESM1" class="usa-link">S15B</a>, single gene example; Supplementary Data <a href="#MOESM5" class="usa-link">3</a>). We then compared NE-induced gene expression between cells with high versus low ChromVAR scores across all cell classes. We first focused our analysis on a curated set of 11 FOS/AP-1 target genes identified in prior studies<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Of these, only 4 showed robust upregulation in our bulk RNA-seq dataset following cNE exposure (Fig. <a href="#MOESM1" class="usa-link">S12</a>), and these genes also had the highest increases in FOS/AP-1 ChromVAR scores after NE stimulation. In contrast, the 4 target genes that were downregulated and the 4 that were not significantly changed in expression showed lower FOS/AP-1 motif accessibility overall. Notably, the unchanged genes did exhibit modest but significant increases in ChromVAR scores, though these were consistently smaller than those seen for the upregulated targets (Fig. <a href="#MOESM1" class="usa-link">S15C</a>). More broadly, when examining all genes upregulated by NE in snRNA-seq, we observed elevated FOS/AP-1 ChromVAR scores in their respective cell types (Fig. <a href="#Fig7" class="usa-link">7E</a>). Together, these findings indicate a robust relationship between NE-induced opening of FOS/AP-1 motifs and regulated gene expression, reinforcing the idea that FOS/AP-1 motif accessibility is a strong predictor of NE-induced gene expression.</p>
<p id="Par22">Overall, our data suggest that the presence of accessible FOS/AP-1 motifs is a critical factor for determining the transcriptional response to NE in many cell types and that FOS binding to these sites likely mediates gene induction by these regulatory elements across cell types. More generally, our findings reveal three distinct classes of chromatin accessibility: regions that are universally open across all cell types, those specific to certain cell types under baseline conditions, and regions that become accessible only in response to novel environments, often in a cell-type-specific manner. The diversity of these accessible chromatin regions in our dataset further highlights the potential of gene regulatory elements to undergo chromatin remodeling in response to not only basal neuronal circuit activity, but also to acute physiological stimuli.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par23">Exposure to novel environments modulates neural connectivity over time, leading to improved cognitive function and the attenuation of neurological disease progression. This structural and functional plasticity involves key regions of the hippocampus, including DG, CA3, and CA1, which have distinct but interconnected roles in encoding, processing, and storing information. Previous studies have demonstrated that ERG induction occurs in these regions in response to NE exposure, yet the downstream transcriptomic changes have largely been examined only after days to weeks of prolonged exposure, leaving a gap in our understanding of the molecular mechanisms by which NE acts via ERGs to acutely regulate neuroplasticity<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p>
<p id="Par24">Using a combination of bulk RNA-seq and single-nucleus multiomic sequencing (snRNA-seq + snATAC-seq), our investigation fills a critical gap in knowledge by focusing on the gene regulatory and mRNA transcript changes that occur in mouse DG, CA3, and CA1 during the first 24 h after NE exposure. We observe a rapid induction of ERG TFs across all three hippocampal regions and multiple cell types within 2 h of NE exposure. In contrast, LRGs exhibited region- and cell-type-specific expression patterns, including genes enriched for protein functions related to secreted factors that signal to both neuronal and non-neuronal cells. Prolonged NE exposure led to a more pronounced induction of LRGs as measured by bulk RNA-seq. Together, these findings suggest that sustained NE exposure drives repeated activation of these transcriptional programs, leading to the cytoplasmic accumulation of LRG transcripts.</p>
<p id="Par25">Our multiomic data also revealed that novel environment (NE) exposure induces cell-type-specific changes in chromatin accessibility correlated with cell-type-specific gene expression changes. Notably, regions with increased accessibility are enriched for FOS/AP-1 binding sites, suggesting a central role for the FOS/AP-1 family of TFs in orchestrating these gene expression changes in neurons. Although Fos is a commonly induced TF across many cell types, we find that its chromatin binding sites and associated late-response genes exhibit notable cell-type-specificity. This finding raises an open question about how a widely expressed TF like FOS/AP-1 can drive such distinct transcriptional programs in different cell types. Our findings therefore represent a rich resource to enable the functional characterization of the cell-type-specific LRGs that drive long-term structural changes in the hippocampus following NE exposure, as well as the <em>cis</em>-regulatory elements that drive these cell-type-specific gene expression changes.</p>
<p id="Par26">This resource contributes to a more comprehensive understanding of the early molecular response to naturalistic stimuli in the hippocampus. While substantial progress has been made in generating atlases of cell-type-specific gene expression and chromatin state<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, these previously collected datasets largely reflect conditions in naïve animals, thereby missing transient gene expression programs induced by salient environmental stimuli. Our study thus provides an important resource for future investigation into the functional consequences of these gene expression programs in the context of experience-dependent plasticity and learning.</p>
<p id="Par27">For example, we identified specific ligand-receptor pairs whose expression is induced in response to NE exposure and which may act in a coordinated fashion across cell types. <em>Ccn1</em>, which encodes a secreted factor upregulated in CA1, CA3, and DG, is notable for its role in activating microglia, a cell type that is essential for clearing debris and promoting synaptic plasticity but which requires precise temporal control to avoid prolonged activation linked to neurodegeneration. Similarly, <em>Bdnf</em> and <em>Ntf3</em>, which bind to receptors expressed in multiple hippocampal cell types, may facilitate broad neuronal survival and synaptic remodeling, supporting learning and memory functions. These ligand-receptor interactions highlight a mechanism by which excitatory neurons may regulate neighboring cell responses, potentially shaping microenvironmental conditions in ways that support circuit-specific plasticity. Our identification of NE-induced ligands and their cell type-specific receptors will enable researchers to dissect the precise roles these signaling molecules play in coordinating cell-cell interactions during spatial memory formation.</p>
<p id="Par28">Future researchers can leverage the integration of two different methodologies of this resource to strategically design functional studies on the relationship between NE exposure, chromatin opening, and gene expression in specific cell types, as well as nascent transcription vs. cytoplasmic accumulation of transcript levels over time. For example, our searchable database accompanying this manuscript allows researchers without prior computational experience to search for genes of interest and identify key cell types and time points of NE exposure when designing their experiments (Fig. <a href="#MOESM1" class="usa-link">S1D</a>). Furthermore, recent advances in technologies that enable the high-throughput interrogation of <em>cis</em>-regulatory element activity in specific cell types<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> could be used to clarify the contribution of these genomic elements to cell-type-specific NE-driven gene expression programs. Notably, the rapid activation of FOS/AP-1-related transcriptional programs in response to sensory stimuli appears to be a conserved mechanism across brain regions<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> and species<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, suggesting that this ERG program is fundamental to a wide range of behavioral and cognitive processes.</p>
<p id="Par29">Certain limitations of the study should be considered when interpreting the results. First, the microdissection of hippocampal regions for bulk whole cell RNA-seq provides a heterogeneous mix of cell types, and does not differentiate between mRNA localized to the nucleus, cytoplasm, dendrites or axons. Moreover, hippocampal tissue was collected irrespective of the dorso-ventral axis; thus gene expression changes that are occurring along this axis may be masked in our analysis<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. While snMultiome-seq offers complementary information about gene expression and chromatin accessibility within specific cell types, this method captures nuclear RNA, making it difficult to detect genes that are transiently transcribed and rapidly exported to the cytoplasm, including many ERGs. This limitation could result in the underrepresentation of these genes in our dataset despite their robust transcriptional activity. Additionally, our snMultiome dataset was analyzed at the single-cell level to capture the often sparse and cell class–specific gene induction following NE exposure. While this approach increases sensitivity, it may also result in a higher false-positive rate for genes with modest fold changes. To aid interpretation and experimental planning, we encourage readers to consult our interactive database, which provides integrated information from both bulk and single-nucleus RNA-seq, as well as ChromVAR score quartiles for each gene.</p>
<p id="Par30">In summary, we present a comprehensive dataset characterizing the dynamic chromatin accessibility and gene expression landscape following acute NE exposure. The accompanying searchable database provides a valuable resource for further exploration of activity-dependent genes and putative gene regulatory elements that will advance tool development for the targeted manipulation of AP1-dependent gene expression programs that contribute to novel environment encoding, as well as learning and memory.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Mice</h3>
<p id="Par31">Mice were handled according to protocols approved by the Harvard University Standing Committee on Animal Care and were in accordance with federal guidelines. Wild-type C57/BL6 (JAX 000664) were housed in a 12 h light/dark cycle with ad libitum access to food. Temperatures range from 67 to 74 degrees Fahrenheit and humidity between 35 and 65%. For all sequencing studies, male mice were used to avoid the confounding effects of estrogen cycles observed in females, but for histology and fluorescence in situ hybridization experiments female mice were used. While staining for ERGs in female mice yielded similar results (Figs. <a href="#Fig1" class="usa-link">1B</a>, <a href="#MOESM1" class="usa-link">S1A</a>), supporting the relevance of these pathways across sexes, this approach limits our ability to directly translate our findings to both sexes.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Novel environment paradigm</h3>
<p id="Par32">Adult mice (8–10 weeks of age) were single housed and brought to the room in which behavior was conducted 2–3 nights prior to exposure to novel environment. During this time, mice were housed in a 12 h light/dark cycle with ad libitum access to food. For novel environment stimulation, mice were placed in a square arena (16” x 16”) containing a running wheel, tunnel, falcon tubes, Lego pyramid, and a tube rack for 30 min, returned to their home cage, and brain tissue was harvested at indicated times after the start of novel environment (except for 15 and 30 min timepoints for which tissue was harvested without return to homecage). For 6 h continuous novel environment stimulation, mice were placed into the arena containing toys and hydro gel for 6 h. Arena and toys were cleaned prior to experimentation and in between animals with 70% EtOH. Importantly, tissue was always collected before the animals entered the dark cycle, so the gene expression changes observed are likely not activated in response to changes in response to light state. We collected replicates from the same time point at different times of day to control for gene expression changes driven by circadian rhythms rather than NE exposure. Our initial dataset also included bulk RNA-seq following 12 h of NE exposure, but this time point was later excluded as all mice were harvested at the same time immediately before the dark cycle began and therefore gene expression changes due to NE exposure and circadian rhythms could not be deconvoluted. For both bulk RNA-sequencing and snRNA-seq time course, all timepoints were collected on the same day, corresponding to one replicate, and deep RNA sequencing libraries of replicates were prepared together to account for batch effects. Separate animals were used for bulk and snMultiome experiments.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Microdissection of hippocampus</h3>
<p id="Par33">Mice were deeply anesthetized with isoflurane, decapitated, and brains were placed in a stainless-steel brain matrix on ice to cut 1 mm sections of anterior to posterior hippocampus. Sections were transferred into ice-cold PBS and Cornu Ammonis (CA)1, CA3 and Dentate Gyrus (DG) were separated under a dissection microscope. Tissue was frozen in liquid nitrogen and stored at −80 °C for further processing and RNA extraction. For single-nuclei multiomic-sequencing, whole hippocampi were dissected on an ice-cold metal block and immediately processed for nuclei preparation (see below).</p></section><section id="Sec15"><h3 class="pmc_sec_title">RNA isolation and library preparation for bulk RNA-seq</h3>
<p id="Par34">Tissue was flash-frozen and resuspended in 500 µL TRIzol reagent (Life Technologies), followed by trituration with a 26 G needle. RNA was chloroform extracted, an equal volume of 100% ethanol was added, and RNA was purified using the RNeasy Micro Kit (Qiagen) according to the manufacturer’s instruction, including the on-column DNase digest. RNA concentration was assessed by Nanodrop, and RNA-seq libraries were prepared from 10 ng of total RNA using the SMARTer Stranded Total RNA-seq Pico Input Mammalian V2 kit (Takara Bio) according to the manufacturer’s instructions. Samples were multiplexed with Illumina TruSeq HT barcodes and sequenced on a NovaSeq X 25B with paired-end 2 × 150-nt reads. All samples were sequenced to a depth of at least 20 M reads.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Bulk RNA-seq analysis</h3>
<p id="Par35">Illumina adapters were trimmed with Cutadapt (version 1.14) and aligned using Hisat2<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> (version 2.1.0) to the <em>Mus musculus</em> genome (GRCm39) and transcriptome (Ensembl). Alignments and analysis were performed on the Orchestra2 high performance computing cluster through Harvard Medical School. Aligned BAM files were sorted using Picard Tools (version 2.8.0); stranded bedGraphs were generated using STAR<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> (version 2.7.0 f); and reads were quantified over annotated exons using FeatureCounts<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> (Subread version 2.0.6). Lowly expressed genes were filtered for counts per million &gt;1 in at least five samples using edgeR (version 4.0.16).</p>
<p id="Par36">Differential expression analysis was performed using DESeq2<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Time points from the same region were analyzed together and both time of NE exposure and library preparation batch effects were considered in the design ( ~ time + batch). Fold change shrinkage was applied using apeglm, as it reduces the noise coming from low-count or highly variable genes and improves visualization and ranking by reducing the impact of outliers<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Significant genes were defined as p<sub>adj</sub> &lt; 0.05 and abs(log<sub>2</sub>(fold change)) &gt; log<sub>2</sub>(1.2) after shrinkage. For HC gene expression comparisons in bulk RNA-seq, DESeq2 was run on pairwise comparisons (CA1 vs. CA3, CA1 vs. DG, CA3 vs. DG) and fold change shrinkage was applied. Significant genes were defined as p<sub>adj</sub> &lt; 0.05 and abs(log<sub>2</sub>(fold change)) &gt; log<sub>2</sub>(2) after shrinkage. Depth-normalized counts were generated using DESeq2. Gene Ontology (GO) enrichment analysis was performed using gProfiler2 in R (version 0.2.3), with a custom background of expressed genes based on expression-filtered RNA-seq genes and false discovery rate (FDR)  &lt;  0.05. Heatmaps were generated using ComplexHeatmap (version 2.18.0)<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. TFs and secreted factors were defined by the GO terms for DNA-binding transcription factor activity (GO:0003700) and secretion (GO:0046903), respectively. Synaptic genes were defined using the SynGO database<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Pairs of secreted factors with known receptors were identified using CellTalkDB (<a href="http://tcm.zju.edu.cn/celltalkdb/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://tcm.zju.edu.cn/celltalkdb/</a>).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Single-cell deconvolution from bulk RNA-seq data</h3>
<p id="Par37">The SCDC R package (version 0.0.0.9000)<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> was used to approximate the distribution of cell types in our bulk RNA-seq, using single-cell RNA-seq data from the Allen Brain Atlas as a cell type reference<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Raw counts data from samples in the reference dataset were normalized, and the distribution of cell types present in each of the bulk RNA-seq samples in this study was determined using SCDC.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Single-nuclei multiome nuclei preparation</h3>
<p id="Par38">Tissue was homogenized in 5 ml Homogenization buffer in 7 ml glass dounce with 10 strokes loose and 10 strokes tight, after which 320 µl 5% Igepal was added and dounced 8x with a tight pestle. Homogenate was filtered through a 40 µm strainer and 5 ml of Working Solution was added and mixed. Samples were underlaid with 1 ml 30% and 1 ml 40% Iodixanol solution in a Beckman Coulter Ultra-Clear tube (#344059) and centrifuged in an ultra-centrifuge for 18 min at 10.000 g at 4 C (7600 RPM on SW41). During centrifugation DNA low-binding collection tubes were coated with 300 µl of 30% BSA for 5 min. 500 µl to 1 mL of nuclei were collected at the 30–40% interface into the coated tube. Nuclei were washed by adding 500 µl of nuclei collection/wash buffer and centrifuged for 10 min at 500 g at 4 C in a swing bucket. Pellet was resuspended in 500 µl nuclei collection/wash buffer and nuclei were counted using a disposable hemocytometer. 150.000 nuclei were transferred into a new 30% BSA coated tube, a final concentration of 0.01% Digitonin and 0.1% Tween-20 was added and nuclei were incubated on ice for 10 min. Sample was centrifuged in a swing bucket at 500 g for 13 min at 4 C, supernatant was removed leaving ˜20–30 µl behind. 250 µl of Dilute Nuclei Buffer was added carefully along the side of the tube wall without disturbing the nuclei pellet and sample was centrifuged at 500 g for 10 min at 4 C in swing bucket. Supernatant was carefully removed without disturbing the pellet to ˜7 µl and pellet was resuspended in remaining buffer. Nuclei were counted on hemocytometer and 16.000–20.000 nuclei were used as input for tagmentation reaction and 10X Chromium controller loading.</p>
<p id="Par39"><span class="text-underline">Homogenization Buffer (HB)</span>: 20 mM Tricine-KOH pH = 7.9, 250 mM Sucrose, 25 mM KCl, 5 mM MgCl<sub>2</sub>, add fresh on day of: 1 mM DTT (P2325 Life Technologies), 0.15 mM Spermine (S1141 Sigma), 0.5 mM Spermidine (<a href="https://www.ncbi.nlm.nih.gov/protein/S02116" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02116</a> Sigma), RNAse protector inhibitor 40U/µl (#3335402001 Roche) and cOmplete Mini EDTA-free protease tablet (#1183617001 Roche)</p>
<p id="Par40"><span class="text-underline">Diluent</span>: 120 mM Tricine-KOH pH=7.9, 150 mM KCl, 30 mM MgCl<sub>2</sub></p>
<p id="Par41"><span class="text-underline">Working Solution</span>: 40 mL Iodixanol (D-1556 Sigma) and 8 mL Diluent</p>
<p id="Par42"><span class="text-underline">30% Iodixanol</span>: 6.75 mL Working Solution + 4.5 mL HB</p>
<p id="Par43"><span class="text-underline">40% Iodixanol</span>: 9.6 mL Working Solution + 2.4 ml HB</p>
<p id="Par44"><span class="text-underline">ATAC RSB Buffer</span>: 10 mM Tris-Hcl pH=7.4, 10 mM NaCl, 3 mM MgCl<sub>2</sub></p>
<p id="Par45"><span class="text-underline">Nuclei collection/wash buffer</span>: for 5 mL use 4.52 mL ATAC RSB Buffer, 350 µl 30% BSA (A-8577 Sigma), 5 µl 1 M DTT and 125 µl Protector RNAse inhibitor</p>
<p id="Par46"><span class="text-underline">Dilute Nuclei Buffer</span>: 50 µl Nuclei Buffer (20X from 10X Genomics kit), 1 µl 1 M DTT, 25 µl Protector RNAse inhibitor and 924 µl Nuclease-free H<sub>2</sub>O.</p>
<p id="Par47">The DNA and RNA libraries were generated according to the manufacturer’s recommended protocol (<a href="https://www.10xgenomics.com/support/single-cell-multiome-atac-plus-gene-expression" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.10xgenomics.com/support/single-cell-multiome-atac-plus-gene-expression</a>). 10x multiome ATAC–seq and RNA-sequencing (RNA-seq) libraries were sequenced together on the Illumina NovaSeq X systems using paired-end 2 × 150 bp reads to a depth of around 50,000 reads per cell for each modality.</p></section><section id="Sec19"><h3 class="pmc_sec_title">snRNA-seq analysis</h3>
<p id="Par48">Raw sequencing data were processed using Cell Ranger ARC (v2.0.2, 10x Genomics) under default settings. Reads were mapped to mouse mm10 genome, with the GENCODE vm23 GTF file as used in the BICCN atlas<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. We performed unsupervised clustering with RNA UMI counts based on the SCANPY analysis pipeline<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> and best practice steps<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. In brief, we first filtered for low-quality nuclei by requiring ≥1000 ATAC fragments and ≥200 genes detected per nuclei and &lt;5% mitochondria RNA reads. We then performed a first-pass clustering for quality control. The top 3000 highly variable genes were identified for principal component analysis (PCA). Top 30 PCs were used to perform K-nearest neighbor (k = 25) graph-based Leiden clustering (resolution=1). Putative multiplets were predicted using Scrublet<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>, and Leiden clusters with mean doublets score &gt; 0.2 were marked as doublets. A large majority (87–92%) of cells generated from Cell Ranger ARC for each replicate passed these filtering steps. We then integrated our high-quality cells with hippocampal formation (HPF) cells selected from the whole-mouse brain atlas dataset from BICCN using Seurat (v4.0.0)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Each dataset was normalized using SCTransform with regression of mitochondrial read percentage. The top 3000 variable features were used to identify integration anchors with FindIntegrationAnchors (dims = 1:30), followed by data integration with IntegrateData. PCA was performed on the integration matrix, and the first 30 PCs were used to construct a shared nearest neighbor (SNN) graph and identify clusters via modularity optimization (FindNeighbors and FindClusters, resolution = 0.4). We annotated 14 major cell types, and 33 sub-class level cell types based on the integration results as the cell type nomenclature used in this study. To visualize clusters and annotations, we used scVI package<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> (v1.2.0 n_hidden = 512, n_latent = 40, n_layers = 2, use sample as batch_key) to learn a latent space and perform UMAP embedding<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> to generate the visualization coordinates for scatter plots. For differential expression analysis, cells within each sample were down-sampled to the lowest number of nuclei within that cell type, and differential expression was calculated between HC and NE using the Wilcoxon rank-sum test for statistical significance, followed by multiple hypothesis correction with Bonferroni adjustment. Significant genes were defined as P<sub>adj</sub> &lt; 0.05 and abs(fold change) &gt; 1.2.</p></section><section id="Sec20"><h3 class="pmc_sec_title">snATAC-seq analysis</h3>
<p id="Par49">We used SnapATAC2 (2.5.0)<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> to process fragment files generated from Cell Ranger ARC to make the cell-by-peak h5ad file and bigwig files for each cell type. We used the PoissonVI model<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> from the scVI-tools<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> to learn a cell-by-peak based latent embedding. Differential peak accessibility analysis was performed within the PoissonVI framework using peaks accessible in &gt;1% of all cells. Significant differentially accessible peaks based on the PoissonVI model were defined as FDR &lt; 0.05, abs(mean log<sub>2</sub>(fold change)) &gt;0.3 and abs(empirical log<sub>2</sub>(fold change)) &gt; 0.3.</p>
<p id="Par50">For Homer (v4.11)<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> motif analysis, we used default Homer motif datasets and selected the top 3000 most differentially accessible peaks based on their accessibility fold change between two conditions and resized them into 500 bp bins. We then performed findMotifsGenome.pl and used one condition as foreground (1 h NE and 6 h cNE) and the other as background (HC) to test for motif enrichment.</p>
<p id="Par51">For ChromVAR analysis<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, we used the whole cell-by-peak matrix as input and scanned motifs across the JASPAR2024 motif database<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. We then performed a two-sided t-test with Benjamini-Hochberg FDR correction on the cell-level ChromVAR scores and selected significant differential motifs based on an FDR &lt; 0.05. Similar motifs belonging to the same motif family were merged, and the most differential motif was selected to represent the motif family (e.g., FOS for the AP-1 family). To test whether the variance of cell-level gene expression can be explained by a motif’s ChromVAR score (e.g., Fos), we grouped cells into five equal-sized bins based on their ChromVAR score and performed a one-way ANOVA test with FDR correction using the Pingouin package (0.5.5)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. For genes with significant ANOVA results (FDR &lt; 0.05), we further performed a post-hoc Tukey pairwise test to determine the relationship between the ChromVAR score and gene expression value. For this analysis, we used log<sub>10</sub> (1 + transformed count per million (CPM)) values for the gene expression.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Estimation of induced nuclei within a population</h3>
<p id="Par52">Using the motif-specific ChromVAR scores for each cell described above, cells were defined as “activated” based on a one-tailed T-test using the HC ChromVAR scores as the null distribution followed by multiple hypothesis correction with Bonferroni adjustment, where activated cells were defined by p<sub>adj</sub> &lt; 0.05. A ratio of activated cells (i.e. those cells with significantly higher ChromVAR scores than the HC distribution) over total cells was then calculated for each motif and cell type. Motifs were plotted if any cell type showed &gt;1% of cells activated in any cell type.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Histology</h3>
<p id="Par53">Mice were deeply anesthetized with 90 mg/kg ketamine and 50 mg/kg xylazine by intraperitoneal injection, and transcardially perfused with ice-cold PBS followed by 4% paraformaldehyde (PFA) in PBS. Dissected brains were post-fixed overnight at 4 °C, next incubated overnight in 15% sucrose prepared in PBS, and then overnight in 30% sucrose in PBS at 4 °C. Brains were sectioned on a vibratome LeicaVT1000 at 40 µm. Free-floating hippocampal sections were permeabilized with 0.25% Triton in 1X PBS for 1 h, and blocked (5% bovine serum albumin, 0.3% Triton 100X, and 10% donkey serum in PBS 1X) for 2 h. Blocking solution was replaced by primary antibody solution (0.1% Triton 100X and 10% donkey serum in PBS 1X). Sections were incubated overnight at room temperature with the following primary antibodies: Egr1 anti-rabbit (1:2000, Cell Signaling Technology #4153S) and c-Fos anti-guinea pig (1:3000, Synaptic Systems, #226308). Brain slices were subsequently washed 3x in 0.1% Triton 100X in PBS for 15 min each. Sections were incubated in secondary antibody solution (0.1% Triton 100X in PBS) for 2 h at room temperature with the following secondary antibodies: Cy3 donkey anti-rabbit IgG (1:1000, Jackson ImmunoResearch, #711-165-152) and Cy5 donkey anti-guinea pig IgG (1:1000, Jackson ImmunoResearch, #706-175-148). Slices were washed twice at room temperature with 0.1% Triton 100X in PBS and then counterstained with 4’,6-diamidine-2’-phenylindole dihydrochloride (DAPI) (1:5000, Sigma Millipore, #MBD0015) in PBS for 5 min. Finally, brain slices were washed twice in 1X PBS. Images were acquired with 10X and 40X objectives, at 1024 × 1024 pixel resolution, and 16-bit depth on a ZEISS LSM800 confocal microscope.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Fluorescent in-situ hybridization (FISH)</h3>
<p id="Par54">All solutions were prepared in RNase-free PBS and water. Mice were perfused as described above. Dissected brains were post-fixed overnight at 4 <sup>o</sup>C, incubated overnight in 15% sucrose, and subsequently overnight in 30% sucrose at 4 <sup>o</sup>C until brains sunk. Brains were sectioned on a vibratome at 30 µm. Sections were mounted on RNase-free SuperFrost Plus slides (ThermoFisher) and probed against the genes using the RNAscope Multiplex Fluorescent Assay v2 protocol (ACDBio, Catalogue #323110). RNAscope, was performed following manufacturer’s instructions, with the exception of hydrogen peroxide solution from the kit which was replaced with 30% hydrogen peroxide solution in water (Sigma Aldrich, Catalogue #216763), and Protease IV was used for 19 min. The following probes were used: Egr3-C1 (Catalogue #431101), Egr3-O2-C2 (Catalogue #1824281-C2), Grin1-C3 (Catalogue #431611-C3), Homer1-O1-C1 (Catalogue #557951), Mgp-C3 (Catalogue #463381-C3), Scg2-C2 (Catalogue #477691-C2). Sections were counterstained with 4’,6-diamidine-2’-phenylindole dihydrochloride (DAPI) (1:5000, Sigma Millipore, # MBD0015) in PBS for 5 min. Finally, brain slices were washed in PBS 1X.</p>
<p id="Par55">Images were acquired with 10X and 40X objectives, at 1024 × 1024 pixel resolution, and 16-bit depth on a ZEISS LSM800 confocal microscope.</p></section><section id="Sec24"><h3 class="pmc_sec_title">FISH analysis</h3>
<p id="Par56">For subsequent FISH intensity analyses, we utilized CellProfiler<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. To segment all DAPI-positive cells within the corresponding DAPI channel, we first trained a new segmentation model on our acquired 40x hippocampal images using Cellpose human-in-the-loop training<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>,<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. This trained model was then integrated into CellProfiler via the “runCellpose” macro in a source-built installation of CellProfiler 5.0.0.1b. Model parameters were set as follows: an expected diameter of 0, a cell probability threshold of −2.0, a minimum object size of 6 pixels, and a flow threshold of 0.6.</p>
<p id="Par57">The identified DAPI-positive masks were subsequently expanded by 10 pixels in diameter to account for the peri-nuclear cytoplasmic region. To reduce background signal, we calculated the lower quartile intensity for each RNAscope probe and subtracted this value from the corresponding images. The corrected images were then used to compute median intensity values within each DAPI-positive mask for all RNAscope probes analyzed using Cell Profiler’s Measure Image Intensity function. The default behavior in CellProfiler is to rescale the image intensities from 0 to 1 by dividing all pixels in the image by the maximum possible intensity value defined by the image metadata. Statistical analysis was performed using a custom-built Python script, assuming that median intensities were not normally distributed and using Pairwise Mann-Whitney U- test with Bonferroni correction.</p>
<p id="Par58">The trained DAPI model weights, CellProfiler pipeline, and custom downstream Python scripts for figure generation and statistical analyses are available for download at <a href="https://github.com/skruettn/Traunmueller_et_al._2025/tree/Greenberg_lab" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/skruettn/Traunmueller_et_al._2025/tree/Greenberg_lab</a>. 10.5281/zenodo.15997560.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par59">Further information on research design is available in the <a href="#MOESM7" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (147.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63029_MOESM2_ESM.pdf</a><sup> (52.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (88.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (305.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (54MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (21.7MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The Greenberg Laboratory is supported by the Allen Discovery Center Program, a Paul G. Allen Frontiers Group advised program of the Paul G. Allen Family Foundation and the Tang-Yang Autism Center at Harvard Medical School. L.T. was supported by an EMBO Postdoctoral Fellowship, Long-term Human Frontiers Science Program Fellowship and the William Randolph Hearst Fund. E.E.D. was a Damon Runyon-National Mah Jongg League, Inc. Breakthrough Scientist supported by the Damon Runyon Cancer Research Foundation and was supported by the Warren Alpert Distinguished Scholar Award and a Mahoney Postdoctoral Fellowship. H.L. was supported by the Harvard Society of Fellows. S.K. was supported by a Y. Eva Tan Fellowship of the Tan Yang Autism Center at Harvard University, M.E.G. was supported by funding from NINDS R01 NS115965. The funders had no role in study design, data collection and analysis, or the decision to publish or preparation of the manuscript. We thank members of the Greenberg laboratory for helpful discussions on the manuscript. We thank the Harvard Medical School Neurobiology Department for consultation and instrument availability that supported this work. The mouse and falcon tube in Figs. <a href="#Fig1" class="usa-link">1</a>A and <a href="#Fig6" class="usa-link">6A</a>, and Ion Channel in Fig. <a href="#Fig2" class="usa-link">2C</a> were drawn with SciDraw.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.T., E.E.D., and M.E.G. conceptualized the study and designed the experiments. E.E.D., L.T., and N.N. performed behavioral assays. L.T. performed hippocampal dissections. E.E.D., L.T., E.G.A., N.P. and N.N. performed bulk RNA-seq assays. L.T., E.E.D., S.Sun and H.L. performed snMultiome-seq assays. E.E.D., L.T. and H.L. performed bioinformatic analyses. S.Sanalidou performed immunohistochemistry, fluorescence in-situ hybridization and microscopy. SK generated and trained the models used for fluorescence in situ hybridization analysis. E.E.D., L.T., E.C.G. and M.E.G. drafted the manuscript, with input from all co-authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par60"><em>Nature Communications</em> thanks Eran Mukamel and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All bulk RNA and snMultiome sequencing data generated in this study have been deposited in the Gene Expression Omnibus database under accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE283483" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE283483</a>. Our web-based searchable database is available from (<a href="https://greenberg.hms.harvard.edu/project/novel-environment-gene-database/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://greenberg.hms.harvard.edu/project/novel-environment-gene-database/</a>).</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All code used in this study was previously published and is cited in the Methods section, except for the trained DAPI model weights, CellProfiler pipeline, and custom downstream Python scripts for figure generation and statistical analyses, which are available for download at <a href="https://github.com/skruettn/Traunmueller_et_al._2025/tree/Greenberg_lab" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/skruettn/Traunmueller_et_al._2025/tree/Greenberg_lab</a>. 10.5281/zenodo.15997560</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par61">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Lisa Traunmüller, Erin E. Duffy.</p></div>
</div></section><section id="sec27"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63029-6.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Yap, E.-L. &amp; Greenberg, M. E. Activity-Regulated Transcription: Bridging the Gap between Neural Activity and Behavior. <em>Neuron</em><strong>100</strong>, 330–348 (2018).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.10.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6223657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30359600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yap,%20E.-L.%20&amp;%20Greenberg,%20M.%20E.%20Activity-Regulated%20Transcription:%20Bridging%20the%20Gap%20between%20Neural%20Activity%20and%20Behavior.%20Neuron100,%20330%E2%80%93348%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Kaminska, B., Kaczmarek, L. &amp; Chaudhuri, A. Visual Stimulation Regulates the Expression of Transcription Factors and Modulates the Composition of AP-1 in Visual Cortexa. <em>J. Neurosci.</em><strong>16</strong>, 3968–3978 (1996).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.16-12-03968.1996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6578611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8656291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaminska,%20B.,%20Kaczmarek,%20L.%20&amp;%20Chaudhuri,%20A.%20Visual%20Stimulation%20Regulates%20the%20Expression%20of%20Transcription%20Factors%20and%20Modulates%20the%20Composition%20of%20AP-1%20in%20Visual%20Cortexa.%20J.%20Neurosci.16,%203968%E2%80%933978%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Majdan, M. &amp; Shatz, C. J. Effects of visual experience on activity-dependent gene regulation in cortex. <em>Nat. Neurosci.</em><strong>9</strong>, 650–659 (2006).
</cite> [<a href="https://doi.org/10.1038/nn1674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16582906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Majdan,%20M.%20&amp;%20Shatz,%20C.%20J.%20Effects%20of%20visual%20experience%20on%20activity-dependent%20gene%20regulation%20in%20cortex.%20Nat.%20Neurosci.9,%20650%E2%80%93659%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Mardinly, A. R. et al. Sensory experience regulates cortical inhibition by inducing IGF1 in VIP neurons. <em>Nature</em><strong>531</strong>, 371–375 (2016).
</cite> [<a href="https://doi.org/10.1038/nature17187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4823817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26958833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mardinly,%20A.%20R.%20et%20al.%20Sensory%20experience%20regulates%20cortical%20inhibition%20by%20inducing%20IGF1%20in%20VIP%20neurons.%20Nature531,%20371%E2%80%93375%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Hrvatin, S. et al. Single-cell analysis of experience-dependent transcriptomic states in the mouse visual cortex. <em>Nat. Neurosci.</em><strong>21</strong>, 120–129 (2018).
</cite> [<a href="https://doi.org/10.1038/s41593-017-0029-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5742025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29230054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hrvatin,%20S.%20et%20al.%20Single-cell%20analysis%20of%20experience-dependent%20transcriptomic%20states%20in%20the%20mouse%20visual%20cortex.%20Nat.%20Neurosci.21,%20120%E2%80%93129%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Yap, E.-L. et al. Bidirectional perisomatic inhibitory plasticity of a Fos neuronal network. <em>Nature</em><strong>590</strong>, 115–121 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-020-3031-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7864877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33299180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yap,%20E.-L.%20et%20al.%20Bidirectional%20perisomatic%20inhibitory%20plasticity%20of%20a%20Fos%20neuronal%20network.%20Nature590,%20115%E2%80%93121%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lin, Y. et al. Activity-dependent regulation of inhibitory synapse development by Npas4. <em>Nature</em><strong>455</strong>, 1198–1204 (2008).
</cite> [<a href="https://doi.org/10.1038/nature07319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2637532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18815592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20Y.%20et%20al.%20Activity-dependent%20regulation%20of%20inhibitory%20synapse%20development%20by%20Npas4.%20Nature455,%201198%E2%80%931204%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Tyssowski, K. M. et al. Different Neuronal Activity Patterns Induce Different Gene Expression Programs. <em>Neuron</em><strong>98</strong>, 530–546.e11 (2018).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5934296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29681534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tyssowski,%20K.%20M.%20et%20al.%20Different%20Neuronal%20Activity%20Patterns%20Induce%20Different%20Gene%20Expression%20Programs.%20Neuron98,%20530%E2%80%93546.e11%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Spiegel, I. et al. Npas4 regulates excitatory-inhibitory balance within neural circuits through cell type-specific gene programs. <em>Cell</em><strong>157</strong>, 1216–1229 (2014).
</cite> [<a href="https://doi.org/10.1016/j.cell.2014.03.058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4089405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24855953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spiegel,%20I.%20et%20al.%20Npas4%20regulates%20excitatory-inhibitory%20balance%20within%20neural%20circuits%20through%20cell%20type-specific%20gene%20programs.%20Cell157,%201216%E2%80%931229%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ataman, B. et al. Evolution of Osteocrin as an activity-regulated factor in the primate brain. <em>Nature</em><strong>539</strong>, 242–247 (2016).
</cite> [<a href="https://doi.org/10.1038/nature20111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5499253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27830782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ataman,%20B.%20et%20al.%20Evolution%20of%20Osteocrin%20as%20an%20activity-regulated%20factor%20in%20the%20primate%20brain.%20Nature539,%20242%E2%80%93247%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Ma, Y. et al. Activity-Dependent Transcriptional Program in NGN2+ Neurons Enriched for Genetic Risk for Brain-Related Disorders. <em>Biol. Psychiatry</em><strong>95</strong>, 187–198 (2024).
</cite> [<a href="https://doi.org/10.1016/j.biopsych.2023.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10787819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37454787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20Y.%20et%20al.%20Activity-Dependent%20Transcriptional%20Program%20in%20NGN2+%20Neurons%20Enriched%20for%20Genetic%20Risk%20for%20Brain-Related%20Disorders.%20Biol.%20Psychiatry95,%20187%E2%80%93198%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>van Praag, H., Kempermann, G. &amp; Gage, F. H. Neural consequences of enviromental enrichment. <em>Nat. Rev. Neurosci.</em><strong>1</strong>, 191–198 (2000).
</cite> [<a href="https://doi.org/10.1038/35044558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11257907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Praag,%20H.,%20Kempermann,%20G.%20&amp;%20Gage,%20F.%20H.%20Neural%20consequences%20of%20enviromental%20enrichment.%20Nat.%20Rev.%20Neurosci.1,%20191%E2%80%93198%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Hüttenrauch, M., Salinas, G. &amp; Wirths, O. Effects of Long-Term Environmental Enrichment on Anxiety, Memory, Hippocampal Plasticity and Overall Brain Gene Expression in C57BL6 Mice. <em>Front. Mol. Neurosci.</em><strong>9</strong>, 62 (2016).
</cite> [<a href="https://doi.org/10.3389/fnmol.2016.00062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4971077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27536216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%BCttenrauch,%20M.,%20Salinas,%20G.%20&amp;%20Wirths,%20O.%20Effects%20of%20Long-Term%20Environmental%20Enrichment%20on%20Anxiety,%20Memory,%20Hippocampal%20Plasticity%20and%20Overall%20Brain%20Gene%20Expression%20in%20C57BL6%20Mice.%20Front.%20Mol.%20Neurosci.9,%2062%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Dijkhuizen, S. et al. Impact of enriched environment on motor performance and learning in mice. <em>Sci. Rep.</em><strong>14</strong>, 5962 (2024).
</cite> [<a href="https://doi.org/10.1038/s41598-024-56568-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10933351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38472324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dijkhuizen,%20S.%20et%20al.%20Impact%20of%20enriched%20environment%20on%20motor%20performance%20and%20learning%20in%20mice.%20Sci.%20Rep.14,%205962%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Escorihuela, R. M. et al. Early Environmental Stimulation Produces Long-Lasting Changes on β-Adrenoceptor Transduction System. <em>Neurobiol. Learn. Mem.</em><strong>64</strong>, 49–57 (1995).
</cite> [<a href="https://doi.org/10.1006/nlme.1995.1043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7582812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Escorihuela,%20R.%20M.%20et%20al.%20Early%20Environmental%20Stimulation%20Produces%20Long-Lasting%20Changes%20on%20%CE%B2-Adrenoceptor%20Transduction%20System.%20Neurobiol.%20Learn.%20Mem.64,%2049%E2%80%9357%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Leon, M. &amp; Woo, C. Environmental Enrichment and Successful Aging. <em>Front. Behav. Neurosci</em>. <strong>12</strong>. 10.3389/fnbeh.2018.00155 (2018).</cite> [<a href="https://doi.org/10.3389/fnbeh.2018.00155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6065351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30083097/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Ball, N. J., Mercado, E. &amp; Orduña, I. Enriched Environments as a Potential Treatment for Developmental Disorders: A Critical Assessment. <em>Front. Psychol.</em><strong>10</strong>. 10.3389/fpsyg.2019.00466 (2019).</cite> [<a href="https://doi.org/10.3389/fpsyg.2019.00466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6414413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30894830/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Nithianantharajah, J. &amp; Hannan, A. J. Enriched environments, experience-dependent plasticity and disorders of the nervous system. <em>Nat. Rev. Neurosci.</em><strong>7</strong>, 697–709 (2006).
</cite> [<a href="https://doi.org/10.1038/nrn1970" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16924259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nithianantharajah,%20J.%20&amp;%20Hannan,%20A.%20J.%20Enriched%20environments,%20experience-dependent%20plasticity%20and%20disorders%20of%20the%20nervous%20system.%20Nat.%20Rev.%20Neurosci.7,%20697%E2%80%93709%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Krüttner, S. et al. Absence of familiarity triggers hallmarks of autism in mouse model through aberrant tail-of-striatum and prelimbic cortex signaling. <em>Neuron</em><strong>110</strong>, 1468–1482.e5 (2022).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2022.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35219402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kr%C3%BCttner,%20S.%20et%20al.%20Absence%20of%20familiarity%20triggers%20hallmarks%20of%20autism%20in%20mouse%20model%20through%20aberrant%20tail-of-striatum%20and%20prelimbic%20cortex%20signaling.%20Neuron110,%201468%E2%80%931482.e5%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Espeso-Gil, S. et al. Environmental Enrichment Induces Epigenomic and Genome Organization Changes Relevant for Cognition. <em>Front. Mol. Neurosci.</em><strong>14</strong>, 664912 (2021).
</cite> [<a href="https://doi.org/10.3389/fnmol.2021.664912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8131874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34025350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Espeso-Gil,%20S.%20et%20al.%20Environmental%20Enrichment%20Induces%20Epigenomic%20and%20Genome%20Organization%20Changes%20Relevant%20for%20Cognition.%20Front.%20Mol.%20Neurosci.14,%20664912%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Rampon, C. et al. Effects of environmental enrichment on gene expression in the brain. <em>Proc. Natl Acad. Sci.</em><strong>97</strong>, 12880–12884 (2000).
</cite> [<a href="https://doi.org/10.1073/pnas.97.23.12880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC18858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11070096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rampon,%20C.%20et%20al.%20Effects%20of%20environmental%20enrichment%20on%20gene%20expression%20in%20the%20brain.%20Proc.%20Natl%20Acad.%20Sci.97,%2012880%E2%80%9312884%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zhang, T.-Y. et al. Environmental enrichment increases transcriptional and epigenetic differentiation between mouse dorsal and ventral dentate gyrus. <em>Nat. Commun.</em><strong>9</strong>, 298 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-017-02748-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5775256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29352183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20T.-Y.%20et%20al.%20Environmental%20enrichment%20increases%20transcriptional%20and%20epigenetic%20differentiation%20between%20mouse%20dorsal%20and%20ventral%20dentate%20gyrus.%20Nat.%20Commun.9,%20298%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Ali, A. E. A., Wilson, Y. M. &amp; Murphy, M. A single exposure to an enriched environment stimulates the activation of discrete neuronal populations in the brain of the fos-tau-lacZ mouse. <em>Neurobiol. Learn. Mem.</em><strong>92</strong>, 381–390 (2009).
</cite> [<a href="https://doi.org/10.1016/j.nlm.2009.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19450699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ali,%20A.%20E.%20A.,%20Wilson,%20Y.%20M.%20&amp;%20Murphy,%20M.%20A%20single%20exposure%20to%20an%20enriched%20environment%20stimulates%20the%20activation%20of%20discrete%20neuronal%20populations%20in%20the%20brain%20of%20the%20fos-tau-lacZ%20mouse.%20Neurobiol.%20Learn.%20Mem.92,%20381%E2%80%93390%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Jaeger, B. N. et al. A novel environment-evoked transcriptional signature predicts reactivity in single dentate granule neurons. <em>Nat. Commun.</em><strong>9</strong>, 3084 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-05418-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6079101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30082781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaeger,%20B.%20N.%20et%20al.%20A%20novel%20environment-evoked%20transcriptional%20signature%20predicts%20reactivity%20in%20single%20dentate%20granule%20neurons.%20Nat.%20Commun.9,%203084%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Pollina, E. A. et al. A NPAS4–NuA4 complex couples synaptic activity to DNA repair. <em>Nature</em><strong>614</strong>, 732–741 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-05711-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9946837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36792830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pollina,%20E.%20A.%20et%20al.%20A%20NPAS4%E2%80%93NuA4%20complex%20couples%20synaptic%20activity%20to%20DNA%20repair.%20Nature614,%20732%E2%80%93741%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Cembrowski, M. S., Wang, L., Sugino, K., Shields, B. C. &amp; Spruston, N. Hipposeq: a comprehensive RNA-seq database of gene expression in hippocampal principal neurons. <em>eLife</em><strong>5</strong>, e14997 (2016).
</cite> [<a href="https://doi.org/10.7554/eLife.14997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4846374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27113915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cembrowski,%20M.%20S.,%20Wang,%20L.,%20Sugino,%20K.,%20Shields,%20B.%20C.%20&amp;%20Spruston,%20N.%20Hipposeq:%20a%20comprehensive%20RNA-seq%20database%20of%20gene%20expression%20in%20hippocampal%20principal%20neurons.%20eLife5,%20e14997%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. <em>Nature</em><strong>624</strong>, 317–332 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06812-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao,%20Z.%20et%20al.%20A%20high-resolution%20transcriptomic%20and%20spatial%20atlas%20of%20cell%20types%20in%20the%20whole%20mouse%20brain.%20Nature624,%20317%E2%80%93332%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Bonnefont, J. et al. Cortical Neurogenesis Requires Bcl6-Mediated Transcriptional Repression of Multiple Self-Renewal-Promoting Extrinsic Pathways. <em>Neuron</em><strong>103</strong>, 1096–1108.e4 (2019).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2019.06.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6859502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31353074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonnefont,%20J.%20et%20al.%20Cortical%20Neurogenesis%20Requires%20Bcl6-Mediated%20Transcriptional%20Repression%20of%20Multiple%20Self-Renewal-Promoting%20Extrinsic%20Pathways.%20Neuron103,%201096%E2%80%931108.e4%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Vierbuchen, T. et al. AP-1 Transcription Factors and the BAF Complex Mediate Signal-Dependent Enhancer Selection. <em>Mol. Cell</em><strong>68</strong>, 1067–1082.e12 (2017).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.11.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5744881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29272704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vierbuchen,%20T.%20et%20al.%20AP-1%20Transcription%20Factors%20and%20the%20BAF%20Complex%20Mediate%20Signal-Dependent%20Enhancer%20Selection.%20Mol.%20Cell68,%201067%E2%80%931082.e12%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Stroud, H. et al. An Activity-Mediated Transition in Transcription in Early Postnatal Neurons. <em>Neuron</em><strong>107</strong>, 874–890.e8 (2020).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2020.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7486250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32589877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stroud,%20H.%20et%20al.%20An%20Activity-Mediated%20Transition%20in%20Transcription%20in%20Early%20Postnatal%20Neurons.%20Neuron107,%20874%E2%80%93890.e8%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Galvan, L. et al. The striatal kinase DCLK3 produces neuroprotection against mutant huntingtin. <em>Brain</em><strong>141</strong>, 1434–1454 (2018).
</cite> [<a href="https://doi.org/10.1093/brain/awy057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5917821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29534157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galvan,%20L.%20et%20al.%20The%20striatal%20kinase%20DCLK3%20produces%20neuroprotection%20against%20mutant%20huntingtin.%20Brain141,%201434%E2%80%931454%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Wallin, R., Cain, D., Hutson, S. M., Sane, D. C. &amp; Loeser, R. Modulation of the binding of matrix Gla protein (MGP) to bone morphogenetic protein-2 (BMP-2). <em>Thromb. Haemost.</em><strong>84</strong>, 1039–1044 (2000).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11154111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wallin,%20R.,%20Cain,%20D.,%20Hutson,%20S.%20M.,%20Sane,%20D.%20C.%20&amp;%20Loeser,%20R.%20Modulation%20of%20the%20binding%20of%20matrix%20Gla%20protein%20(MGP)%20to%20bone%20morphogenetic%20protein-2%20(BMP-2).%20Thromb.%20Haemost.84,%201039%E2%80%931044%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Song, W., Cho, Y., Watt, D. &amp; Cavalli, V. Tubulin-tyrosine Ligase (TTL)-mediated Increase in Tyrosinated α-Tubulin in Injured Axons Is Required for Retrograde Injury Signaling and Axon Regeneration*. <em>J. Biol. Chem.</em><strong>290</strong>, 14765–14775 (2015).
</cite> [<a href="https://doi.org/10.1074/jbc.M114.622753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4505541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25911101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20W.,%20Cho,%20Y.,%20Watt,%20D.%20&amp;%20Cavalli,%20V.%20Tubulin-tyrosine%20Ligase%20(TTL)-mediated%20Increase%20in%20Tyrosinated%20%CE%B1-Tubulin%20in%20Injured%20Axons%20Is%20Required%20for%20Retrograde%20Injury%20Signaling%20and%20Axon%20Regeneration*.%20J.%20Biol.%20Chem.290,%2014765%E2%80%9314775%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Mizuno, K. et al. Calcium/calmodulin kinase kinase β has a male-specific role in memory formation. <em>Neuroscience</em><strong>145</strong>, 393–402 (2007).
</cite> [<a href="https://doi.org/10.1016/j.neuroscience.2006.11.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17207577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mizuno,%20K.%20et%20al.%20Calcium/calmodulin%20kinase%20kinase%20%CE%B2%20has%20a%20male-specific%20role%20in%20memory%20formation.%20Neuroscience145,%20393%E2%80%93402%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Peters, M. et al. Loss of Ca2+/Calmodulin Kinase Kinase β Affects the Formation of Some, But Not All, Types of Hippocampus-Dependent Long-Term Memory. <em>J. Neurosci.</em><strong>23</strong>, 9752–9760 (2003).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.23-30-09752.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6740881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14586002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peters,%20M.%20et%20al.%20Loss%20of%20Ca2+/Calmodulin%20Kinase%20Kinase%20%CE%B2%20Affects%20the%20Formation%20of%20Some,%20But%20Not%20All,%20Types%20of%20Hippocampus-Dependent%20Long-Term%20Memory.%20J.%20Neurosci.23,%209752%E2%80%939760%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gu, M. et al. The Transcription Factor Ets1 Influences Axonal Growth via Regulation of Lcn2. <em>Mol. Neurobiol.</em><strong>61</strong>, 971–981 (2024).
</cite> [<a href="https://doi.org/10.1007/s12035-023-03616-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10861751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37672148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu,%20M.%20et%20al.%20The%20Transcription%20Factor%20Ets1%20Influences%20Axonal%20Growth%20via%20Regulation%20of%20Lcn2.%20Mol.%20Neurobiol.61,%20971%E2%80%93981%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>CellTalkDB::About <a href="http://tcm.zju.edu.cn/celltalkdb/about.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://tcm.zju.edu.cn/celltalkdb/about.php</a>.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Yan, L. et al. Single and Compound Knock-outs of MicroRNA (miRNA)−155 and Its Angiogenic Gene Target <em>CCN1</em> in Mice Alter Vascular and Neovascular Growth in the Retina via Resident Microglia*. <em>J. Biol. Chem.</em><strong>290</strong>, 23264–23281 (2015).
</cite> [<a href="https://doi.org/10.1074/jbc.M115.646950" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4645596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26242736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20L.%20et%20al.%20Single%20and%20Compound%20Knock-outs%20of%20MicroRNA%20(miRNA)%E2%88%92155%20and%20Its%20Angiogenic%20Gene%20Target%20CCN1%20in%20Mice%20Alter%20Vascular%20and%20Neovascular%20Growth%20in%20the%20Retina%20via%20Resident%20Microglia*.%20J.%20Biol.%20Chem.290,%2023264%E2%80%9323281%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Gao, C., Jiang, J., Tan, Y. &amp; Chen, S. Microglia in neurodegenerative diseases: mechanism and potential therapeutic targets. <em>Signal Transduct. Target. Ther.</em><strong>8</strong>, 359 (2023).
</cite> [<a href="https://doi.org/10.1038/s41392-023-01588-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10514343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37735487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20C.,%20Jiang,%20J.,%20Tan,%20Y.%20&amp;%20Chen,%20S.%20Microglia%20in%20neurodegenerative%20diseases:%20mechanism%20and%20potential%20therapeutic%20targets.%20Signal%20Transduct.%20Target.%20Ther.8,%20359%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wang, C. S., Kavalali, E. T. &amp; Monteggia, L. M. BDNF signaling in context: From synaptic regulation to psychiatric disorders. <em>Cell</em><strong>185</strong>, 62–76 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8741740/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34963057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20C.%20S.,%20Kavalali,%20E.%20T.%20&amp;%20Monteggia,%20L.%20M.%20BDNF%20signaling%20in%20context:%20From%20synaptic%20regulation%20to%20psychiatric%20disorders.%20Cell185,%2062%E2%80%9376%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Gelfo, F., Mandolesi, L., Serra, L., Sorrentino, G. &amp; Caltagirone, C. The Neuroprotective Effects of Experience on Cognitive Functions: Evidence from Animal Studies on the Neurobiological Bases of Brain Reserve. <em>Neuroscience</em><strong>370</strong>, 218–235 (2018).
</cite> [<a href="https://doi.org/10.1016/j.neuroscience.2017.07.065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28827089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gelfo,%20F.,%20Mandolesi,%20L.,%20Serra,%20L.,%20Sorrentino,%20G.%20&amp;%20Caltagirone,%20C.%20The%20Neuroprotective%20Effects%20of%20Experience%20on%20Cognitive%20Functions:%20Evidence%20from%20Animal%20Studies%20on%20the%20Neurobiological%20Bases%20of%20Brain%20Reserve.%20Neuroscience370,%20218%E2%80%93235%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Saitoh, Y. et al. Involvement of membrane palmitoylated protein 6 (MPP6) in synapses of mouse cerebrum. <em>Histochem. Cell Biol.</em><strong>163</strong>, 1–12 (2025).</cite> [<a href="https://doi.org/10.1007/s00418-025-02378-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12075274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40360818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saitoh,%20Y.%20et%20al.%20Involvement%20of%20membrane%20palmitoylated%20protein%206%20(MPP6)%20in%20synapses%20of%20mouse%20cerebrum.%20Histochem.%20Cell%20Biol.163,%201%E2%80%9312%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yalla, K. et al. FBXW7 regulates DISC1 stability via the ubiquitin-proteosome system. <em>Mol. Psychiatry</em><strong>23</strong>, 1278–1286 (2018).
</cite> [<a href="https://doi.org/10.1038/mp.2017.138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5984089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28727686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yalla,%20K.%20et%20al.%20FBXW7%20regulates%20DISC1%20stability%20via%20the%20ubiquitin-proteosome%20system.%20Mol.%20Psychiatry23,%201278%E2%80%931286%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Wu, K. L. H. et al. Butyrate reduction and HDAC4 increase underlie maternal high fructose-induced metabolic dysfunction in hippocampal astrocytes in female rats. <em>J. Nutr. Biochem</em><strong>126</strong>, 109571 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jnutbio.2024.109571" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38199310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20K.%20L.%20H.%20et%20al.%20Butyrate%20reduction%20and%20HDAC4%20increase%20underlie%20maternal%20high%20fructose-induced%20metabolic%20dysfunction%20in%20hippocampal%20astrocytes%20in%20female%20rats.%20J.%20Nutr.%20Biochem126,%20109571%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Bjursell, M. et al. Opposing Effects of Adiponectin Receptors 1 and 2 on Energy Metabolism. <em>Diabetes</em><strong>56</strong>, 583–593 (2007).
</cite> [<a href="https://doi.org/10.2337/db06-1432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17327425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bjursell,%20M.%20et%20al.%20Opposing%20Effects%20of%20Adiponectin%20Receptors%201%20and%202%20on%20Energy%20Metabolism.%20Diabetes56,%20583%E2%80%93593%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Zhang, D. et al. Adiponectin regulates contextual fear extinction and intrinsic excitability of dentate gyrus granule neurons through AdipoR2 receptors. <em>Mol. Psychiatry</em><strong>22</strong>, 1044–1055 (2017).
</cite> [<a href="https://doi.org/10.1038/mp.2016.58" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5491689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27137743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20D.%20et%20al.%20Adiponectin%20regulates%20contextual%20fear%20extinction%20and%20intrinsic%20excitability%20of%20dentate%20gyrus%20granule%20neurons%20through%20AdipoR2%20receptors.%20Mol.%20Psychiatry22,%201044%E2%80%931055%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Tullai, J. W. et al. Immediate-Early and Delayed Primary Response Genes Are Distinct in Function and Genomic Architecture*. <em>J. Biol. Chem.</em><strong>282</strong>, 23981–23995 (2007).
</cite> [<a href="https://doi.org/10.1074/jbc.M702044200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2039722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17575275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tullai,%20J.%20W.%20et%20al.%20Immediate-Early%20and%20Delayed%20Primary%20Response%20Genes%20Are%20Distinct%20in%20Function%20and%20Genomic%20Architecture*.%20J.%20Biol.%20Chem.282,%2023981%E2%80%9323995%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Ibarra, I. L. et al. Comparative chromatin accessibility upon BDNF stimulation delineates neuronal regulatory elements. <em>Mol. Syst. Biol.</em><strong>18</strong>, e10473 (2022).
</cite> [<a href="https://doi.org/10.15252/msb.202110473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9396287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35996956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ibarra,%20I.%20L.%20et%20al.%20Comparative%20chromatin%20accessibility%20upon%20BDNF%20stimulation%20delineates%20neuronal%20regulatory%20elements.%20Mol.%20Syst.%20Biol.18,%20e10473%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Ichikawa-Tomikawa, N. et al. Laminin α1 is essential for mouse cerebellar development. <em>Matrix Biol.</em><strong>31</strong>, 17–28 (2012).
</cite> [<a href="https://doi.org/10.1016/j.matbio.2011.09.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3259268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21983115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ichikawa-Tomikawa,%20N.%20et%20al.%20Laminin%20%CE%B11%20is%20essential%20for%20mouse%20cerebellar%20development.%20Matrix%20Biol.31,%2017%E2%80%9328%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Schep, A. N., Wu, B., Buenrostro, J. D. &amp; Greenleaf, W. J. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. <em>Nat. Methods</em><strong>14</strong>, 975–978 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5623146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28825706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schep,%20A.%20N.,%20Wu,%20B.,%20Buenrostro,%20J.%20D.%20&amp;%20Greenleaf,%20W.%20J.%20chromVAR:%20inferring%20transcription-factor-associated%20accessibility%20from%20single-cell%20epigenomic%20data.%20Nat.%20Methods14,%20975%E2%80%93978%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Zeisel, A. et al. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq. <em>Science</em><strong>347</strong>, 1138–1142 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa1934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25700174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeisel,%20A.%20et%20al.%20Cell%20types%20in%20the%20mouse%20cortex%20and%20hippocampus%20revealed%20by%20single-cell%20RNA-seq.%20Science347,%201138%E2%80%931142%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Hrvatin, S. et al. A scalable platform for the development of cell-type-specific viral drivers. <em>eLife</em><strong>8</strong>, e48089 (2019).
</cite> [<a href="https://doi.org/10.7554/eLife.48089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6776442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31545165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hrvatin,%20S.%20et%20al.%20A%20scalable%20platform%20for%20the%20development%20of%20cell-type-specific%20viral%20drivers.%20eLife8,%20e48089%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Vormstein-Schneider, D. et al. Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans. <em>Nat. Neurosci.</em><strong>23</strong>, 1629–1636 (2020).
</cite> [<a href="https://doi.org/10.1038/s41593-020-0692-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8015416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32807948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vormstein-Schneider,%20D.%20et%20al.%20Viral%20manipulation%20of%20functionally%20distinct%20interneurons%20in%20mice,%20non-human%20primates%20and%20humans.%20Nat.%20Neurosci.23,%201629%E2%80%931636%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Furlanis, E. et al. An enhancer-AAV toolbox to target and manipulate distinct interneuron subtypes. <em>Neuron</em><strong>113</strong>, 1525–1547.e15 (2025).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2025.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40403705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Furlanis,%20E.%20et%20al.%20An%20enhancer-AAV%20toolbox%20to%20target%20and%20manipulate%20distinct%20interneuron%20subtypes.%20Neuron113,%201525%E2%80%931547.e15%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Yee, C. et al. An activity-regulated transcriptional program directly drives synaptogenesis. <em>Nat. Neurosci.</em><strong>27</strong>, 1695–1707 (2024).
</cite> [<a href="https://doi.org/10.1038/s41593-024-01728-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11374667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39103556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yee,%20C.%20et%20al.%20An%20activity-regulated%20transcriptional%20program%20directly%20drives%20synaptogenesis.%20Nat.%20Neurosci.27,%201695%E2%80%931707%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Kurmangaliyev, Y. Z., Yoo, J., Valdes-Aleman, J., Sanfilippo, P. &amp; Zipursky, S. L. Transcriptional Programs of Circuit Assembly in the <em>Drosophila</em> Visual System. <em>Neuron</em><strong>108</strong>, 1045–1057.e6 (2020).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2020.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33125872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kurmangaliyev,%20Y.%20Z.,%20Yoo,%20J.,%20Valdes-Aleman,%20J.,%20Sanfilippo,%20P.%20&amp;%20Zipursky,%20S.%20L.%20Transcriptional%20Programs%20of%20Circuit%20Assembly%20in%20the%20Drosophila%20Visual%20System.%20Neuron108,%201045%E2%80%931057.e6%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kim, D., Paggi, J. M., Park, C., Bennett, C. &amp; Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. <em>Nat. Biotechnol.</em><strong>37</strong>, 907–915 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Paggi,%20J.%20M.,%20Park,%20C.,%20Bennett,%20C.%20&amp;%20Salzberg,%20S.%20L.%20Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype.%20Nat.%20Biotechnol.37,%20907%E2%80%93915%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em><strong>29</strong>, 15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29,%2015%E2%80%9321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <em>Bioinformatics</em><strong>30</strong>, 923–930 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao,%20Y.,%20Smyth,%20G.%20K.%20&amp;%20Shi,%20W.%20featureCounts:%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features.%20Bioinformatics30,%20923%E2%80%93930%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Zhu, A., Ibrahim, J. G. &amp; Love, M. I. Heavy-tailed prior distributions for sequence count data: removing the noise and preserving large differences. <em>Bioinformatics</em><strong>35</strong>, 2084–2092 (2019).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6581436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20A.,%20Ibrahim,%20J.%20G.%20&amp;%20Love,%20M.%20I.%20Heavy-tailed%20prior%20distributions%20for%20sequence%20count%20data:%20removing%20the%20noise%20and%20preserving%20large%20differences.%20Bioinformatics35,%202084%E2%80%932092%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Gu, Z. Complex heatmap visualization. <em>iMeta</em><strong>1</strong>, e43 (2022).
</cite> [<a href="https://doi.org/10.1002/imt2.43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu,%20Z.%20Complex%20heatmap%20visualization.%20iMeta1,%20e43%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Koopmans, F. et al. SynGO: An Evidence-Based, Expert-Curated Knowledge Base for the Synapse. <em>Neuron</em><strong>103</strong>, 217–234.e4 (2019).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2019.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6764089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31171447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koopmans,%20F.%20et%20al.%20SynGO:%20An%20Evidence-Based,%20Expert-Curated%20Knowledge%20Base%20for%20the%20Synapse.%20Neuron103,%20217%E2%80%93234.e4%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Dong, M. et al. SCDC: bulk gene expression deconvolution by multiple single-cell RNA sequencing references. <em>Brief. Bioinform.</em><strong>22</strong>, 416–427 (2020).</cite> [<a href="https://doi.org/10.1093/bib/bbz166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7820884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31925417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20M.%20et%20al.%20SCDC:%20bulk%20gene%20expression%20deconvolution%20by%20multiple%20single-cell%20RNA%20sequencing%20references.%20Brief.%20Bioinform.22,%20416%E2%80%93427%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Wolf, F. A., Angerer, P. &amp; Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. <em>Genome Biol.</em><strong>19</strong>, 15 (2018).
</cite> [<a href="https://doi.org/10.1186/s13059-017-1382-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5802054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolf,%20F.%20A.,%20Angerer,%20P.%20&amp;%20Theis,%20F.%20J.%20SCANPY:%20large-scale%20single-cell%20gene%20expression%20data%20analysis.%20Genome%20Biol.19,%2015%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Luecken, M. D. &amp; Theis, F. J. Current best practices in single-cell RNA-seq analysis: a tutorial. <em>Mol. Syst. Biol.</em><strong>15</strong>, e8746 (2019).
</cite> [<a href="https://doi.org/10.15252/msb.20188746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6582955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31217225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luecken,%20M.%20D.%20&amp;%20Theis,%20F.%20J.%20Current%20best%20practices%20in%20single-cell%20RNA-seq%20analysis:%20a%20tutorial.%20Mol.%20Syst.%20Biol.15,%20e8746%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Heumos, L. et al. Best practices for single-cell analysis across modalities. <em>Nat. Rev. Genet.</em><strong>24</strong>, 550–572 (2023).
</cite> [<a href="https://doi.org/10.1038/s41576-023-00586-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10066026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37002403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heumos,%20L.%20et%20al.%20Best%20practices%20for%20single-cell%20analysis%20across%20modalities.%20Nat.%20Rev.%20Genet.24,%20550%E2%80%93572%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: Computational Identification of Cell Doublets in Single-Cell Transcriptomic Data. <em>Cell Syst.</em><strong>8</strong>, 281–291.e9 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cels.2018.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6625319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30954476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolock,%20S.%20L.,%20Lopez,%20R.%20&amp;%20Klein,%20A.%20M.%20Scrublet:%20Computational%20Identification%20of%20Cell%20Doublets%20in%20Single-Cell%20Transcriptomic%20Data.%20Cell%20Syst.8,%20281%E2%80%93291.e9%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Liu, H. et al. Single-cell DNA methylome and 3D multi-omic atlas of the adult mouse brain. <em>Nature</em><strong>624</strong>, 366–377 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06805-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20H.%20et%20al.%20Single-cell%20DNA%20methylome%20and%203D%20multi-omic%20atlas%20of%20the%20adult%20mouse%20brain.%20Nature624,%20366%E2%80%93377%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Gayoso, A. et al. A Python library for probabilistic analysis of single-cell omics data. <em>Nat. Biotechnol.</em><strong>40</strong>, 163–166 (2022).
</cite> [<a href="https://doi.org/10.1038/s41587-021-01206-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35132262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gayoso,%20A.%20et%20al.%20A%20Python%20library%20for%20probabilistic%20analysis%20of%20single-cell%20omics%20data.%20Nat.%20Biotechnol.40,%20163%E2%80%93166%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>McInnes, L., Healy, J. &amp; Melville, J. UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction. Preprint at arXiv, 10.48550/arXiv.1802.03426. (2020).</cite>
</li>
<li id="CR72">
<span class="label">72.</span><cite>Zhang, K., Zemke, N. R., Armand, E. J. &amp; Ren, B. A fast, scalable and versatile tool for analysis of single-cell omics data. <em>Nat. Methods</em><strong>21</strong>, 217–227 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-023-02139-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10864184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38191932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20K.,%20Zemke,%20N.%20R.,%20Armand,%20E.%20J.%20&amp;%20Ren,%20B.%20A%20fast,%20scalable%20and%20versatile%20tool%20for%20analysis%20of%20single-cell%20omics%20data.%20Nat.%20Methods21,%20217%E2%80%93227%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Martens, L. D., Fischer, D. S., Yépez, V. A., Theis, F. J. &amp; Gagneur, J. Modeling fragment counts improves single-cell ATAC-seq analysis. <em>Nat. Methods</em><strong>21</strong>, 28–31 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-023-02112-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10776385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38049697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martens,%20L.%20D.,%20Fischer,%20D.%20S.,%20Y%C3%A9pez,%20V.%20A.,%20Theis,%20F.%20J.%20&amp;%20Gagneur,%20J.%20Modeling%20fragment%20counts%20improves%20single-cell%20ATAC-seq%20analysis.%20Nat.%20Methods21,%2028%E2%80%9331%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Heinz, S. et al. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. <em>Mol. Cell</em><strong>38</strong>, 576–589 (2010).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2010.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2898526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heinz,%20S.%20et%20al.%20Simple%20combinations%20of%20lineage-determining%20transcription%20factors%20prime%20cis-regulatory%20elements%20required%20for%20macrophage%20and%20B%20cell%20identities.%20Mol.%20Cell38,%20576%E2%80%93589%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Rauluseviciute, I. et al. JASPAR 2024: 20th anniversary of the open-access database of transcription factor binding profiles. <em>Nucleic Acids Res</em><strong>52</strong>, D174–D182 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37962376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rauluseviciute,%20I.%20et%20al.%20JASPAR%202024:%2020th%20anniversary%20of%20the%20open-access%20database%20of%20transcription%20factor%20binding%20profiles.%20Nucleic%20Acids%20Res52,%20D174%E2%80%93D182%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Vallat, R. Pingouin: statistics in Python. <em>J. Open Source Softw.</em><strong>3</strong>, 1026 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Vallat,%20R.%20Pingouin:%20statistics%20in%20Python.%20J.%20Open%20Source%20Softw.3,%201026%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Carpenter, A. E. et al. CellProfiler: image analysis software for identifying and quantifying cell phenotypes. <em>Genome Biol.</em><strong>7</strong>, R100 (2006).
</cite> [<a href="https://doi.org/10.1186/gb-2006-7-10-r100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1794559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17076895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carpenter,%20A.%20E.%20et%20al.%20CellProfiler:%20image%20analysis%20software%20for%20identifying%20and%20quantifying%20cell%20phenotypes.%20Genome%20Biol.7,%20R100%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Pachitariu, M. &amp; Stringer, C. Cellpose 2.0: how to train your own model. <em>Nat. Methods</em><strong>19</strong>, 1634–1641 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-022-01663-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9718665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36344832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pachitariu,%20M.%20&amp;%20Stringer,%20C.%20Cellpose%202.0:%20how%20to%20train%20your%20own%20model.%20Nat.%20Methods19,%201634%E2%80%931641%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Stringer, C., Wang, T., Michaelos, M. &amp; Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. <em>Nat. Methods</em><strong>18</strong>, 100–106 (2021).
</cite> [<a href="https://doi.org/10.1038/s41592-020-01018-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33318659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stringer,%20C.,%20Wang,%20T.,%20Michaelos,%20M.%20&amp;%20Pachitariu,%20M.%20Cellpose:%20a%20generalist%20algorithm%20for%20cellular%20segmentation.%20Nat.%20Methods18,%20100%E2%80%93106%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (147.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63029_MOESM2_ESM.pdf</a><sup> (52.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (88.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (305.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (54MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370903/bin/41467_2025_63029_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (21.7MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All bulk RNA and snMultiome sequencing data generated in this study have been deposited in the Gene Expression Omnibus database under accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE283483" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE283483</a>. Our web-based searchable database is available from (<a href="https://greenberg.hms.harvard.edu/project/novel-environment-gene-database/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://greenberg.hms.harvard.edu/project/novel-environment-gene-database/</a>).</p>
<p>All code used in this study was previously published and is cited in the Methods section, except for the trained DAPI model weights, CellProfiler pipeline, and custom downstream Python scripts for figure generation and statistical analyses, which are available for download at <a href="https://github.com/skruettn/Traunmueller_et_al._2025/tree/Greenberg_lab" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/skruettn/Traunmueller_et_al._2025/tree/Greenberg_lab</a>. 10.5281/zenodo.15997560</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63029-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63029.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370903/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370903/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370903%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370903/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370903/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370903/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841540/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370903/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841540/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370903/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370903/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="tbvX4iLRxxiyO27cPA8X5HEj6RWCygQU3bPEOEbo0nE3Yp2ACE4pkmskaBDB0NRG">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
