
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Valorization of feral pigs in the tropics, from the genetic characterization to the re-domestication - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15254CEE8AF3CB43054CEE004FBD5B2B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361484/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Valorization of feral pigs in the tropics, from the genetic characterization to the re-domestication">
<meta name="citation_author" content="Nicolas Degras">
<meta name="citation_author_institution" content="INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France">
<meta name="citation_author" content="Yann Labrune">
<meta name="citation_author_institution" content="GenPhySE, Université de Toulouse, INRAE, ENVT, 31326 Castanet Tolosan, France">
<meta name="citation_author" content="Nathalie Mandonnet">
<meta name="citation_author_institution" content="INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France">
<meta name="citation_author" content="Mélain Bructer">
<meta name="citation_author_institution" content="INRAE Centre Antilles-Guyane, UE-PTEA, 97170 Petit-Bourg, Guadeloupe France">
<meta name="citation_author" content="Katia Benony">
<meta name="citation_author_institution" content="INRAE Centre Antilles-Guyane, UE-PTEA, 97170 Petit-Bourg, Guadeloupe France">
<meta name="citation_author" content="Claudine Vertueux-Degras">
<meta name="citation_author_institution" content="Chambre d’Agriculture de Martinique, 97232 Le Lamentin, Martinique France">
<meta name="citation_author" content="Gisèle Alexandre">
<meta name="citation_author_institution" content="INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France">
<meta name="citation_author" content="Nausicaa Poullet">
<meta name="citation_author_institution" content="INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France">
<meta name="citation_author" content="Jean-Luc Gourdine">
<meta name="citation_author_institution" content="INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30276">
<meta name="citation_doi" content="10.1038/s41598-025-15649-7">
<meta name="citation_pmid" content="40825833">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361484/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361484/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361484/pdf/41598_2025_Article_15649.pdf">
<meta name="description" content="Feral pigs may serve as a valuable genetic resource for the future, offering potential interesting gene pool for adaptation to climate change and the preservation of biodiversity. The main objective of this study was to identify the genetic ...">
<meta name="og:title" content="Valorization of feral pigs in the tropics, from the genetic characterization to the re-domestication">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Feral pigs may serve as a valuable genetic resource for the future, offering potential interesting gene pool for adaptation to climate change and the preservation of biodiversity. The main objective of this study was to identify the genetic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361484/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12361484">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-15649-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_15649.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12361484%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12361484/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12361484/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361484/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 19;15:30276. doi: <a href="https://doi.org/10.1038/s41598-025-15649-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-15649-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Valorization of feral pigs in the tropics, from the genetic characterization to the re-domestication</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Degras%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Nicolas Degras</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Nicolas Degras</span></h3>
<div class="p">
<sup>1</sup>INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Degras%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicolas Degras</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Labrune%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yann Labrune</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yann Labrune</span></h3>
<div class="p">
<sup>2</sup>GenPhySE, Université de Toulouse, INRAE, ENVT, 31326 Castanet Tolosan, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Labrune%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yann Labrune</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mandonnet%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Nathalie Mandonnet</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nathalie Mandonnet</span></h3>
<div class="p">
<sup>1</sup>INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mandonnet%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nathalie Mandonnet</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bructer%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mélain Bructer</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mélain Bructer</span></h3>
<div class="p">
<sup>3</sup>INRAE Centre Antilles-Guyane, UE-PTEA, 97170 Petit-Bourg, Guadeloupe France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bructer%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mélain Bructer</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benony%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Katia Benony</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Katia Benony</span></h3>
<div class="p">
<sup>3</sup>INRAE Centre Antilles-Guyane, UE-PTEA, 97170 Petit-Bourg, Guadeloupe France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benony%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katia Benony</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vertueux-Degras%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Claudine Vertueux-Degras</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Claudine Vertueux-Degras</span></h3>
<div class="p">
<sup>4</sup>Chambre d’Agriculture de Martinique, 97232 Le Lamentin, Martinique France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vertueux-Degras%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Claudine Vertueux-Degras</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alexandre%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Gisèle Alexandre</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Gisèle Alexandre</span></h3>
<div class="p">
<sup>1</sup>INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alexandre%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gisèle Alexandre</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poullet%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Nausicaa Poullet</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Nausicaa Poullet</span></h3>
<div class="p">
<sup>1</sup>INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poullet%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nausicaa Poullet</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gourdine%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jean-Luc Gourdine</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jean-Luc Gourdine</span></h3>
<div class="p">
<sup>1</sup>INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gourdine%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">Jean-Luc Gourdine</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>INRAE Centre Antilles-Guyane, UR-ASSET, 97170 Petit-Bourg, Guadeloupe France </div>
<div id="Aff2">
<sup>2</sup>GenPhySE, Université de Toulouse, INRAE, ENVT, 31326 Castanet Tolosan, France </div>
<div id="Aff3">
<sup>3</sup>INRAE Centre Antilles-Guyane, UE-PTEA, 97170 Petit-Bourg, Guadeloupe France </div>
<div id="Aff4">
<sup>4</sup>Chambre d’Agriculture de Martinique, 97232 Le Lamentin, Martinique France </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 14; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12361484  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40825833/" class="usa-link">40825833</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Feral pigs may serve as a valuable genetic resource for the future, offering potential interesting gene pool for adaptation to climate change and the preservation of biodiversity. The main objective of this study was to identify the genetic structure of feral pigs from the Caribbean island of Martinique, measure the inbreeding rate of a Creole population re-domesticated in 2016 from captured feral pigs, and evaluate its evolution to the present day. We hypothesized that feral pigs, like Creole breeds of the Americas, have been shaped by a unique cross-breeding process linked to the historical context of the Caribbean. A total of 121 animals were genotyped and 76 were compared with referenced mainstream genotypes and Creole breeds from the Americas. Re-domestication efforts were carried out through a holistic approach, involving researchers, farmers, consultants, and development actors. The results showed that feral and semi-feral pigs in Martinique belong to the creole pig breeds, with more than 20% Iberian genetic admixture. The majority of domesticated pigs groups studied including re-domesticated creole pigs exhibit a significantly lower proportion of runs of homozygosity compared to feral pigs, suggesting a better control of inbreeding, thanks to structured breeding programs. The chosen conservation strategy was the result of a consensus between scientific evidence, practical experience, and field feasibility. A rotational mating system using sire lines among a minimum of five farmers was adopted. This approach is expected to generate, within five years, a pedigree containing information on a total of at least 150 reproducers on 8 generations.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Feral pig, Admixture, Re-domestication, Conservation strategy, Martinique</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Animal breeding, Population genetics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">The regional concentration of livestock farming and the expansion of international trade - such as off-farm feed production (e.g. soybean meal) - have created new geochemical cycles<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup> that have global impacts on the environment and biodiversity<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Livestock farming systems, which play a major role in biomass transformation and protein production, represent both a critical factor and a driving force in addressing the interconnected challenges of food security, climate change, and biodiversity conservation<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The use and management of local breeds can contribute to meeting these challenges<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, particularly through the conservation of « rustic » animal genetic resources. These breeds are well-adapted – or adaptable – to local conditions<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, and they also carry potentially valuable alleles that could benefit the future of their species<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. The Caribbean region provides a unique context for studying local genetic resources and their use in agro-ecological farming systems. Its complex history – including the Taino culture, European colonization and the transatlantic slave trade, as well as Indian migrations – has fostered extensive interactions between continents (Africa, Europe, Asia)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. These interactions led to the emergence and development of Creole breeds (CR) in low input systems, such as the Creole pig of Martinique (14°N, 61°W), in the French overseas departments of the Caribbean. The Taino, the first inhabitants of the Caribbean islands, did not domesticate <em>Sus scrofa domesticus.</em> This specie was introduced only after Columbus’s second voyage in 1493, and likely reached Martinique during the fourth voyage in 1502<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Over time, these Iberian pig populations gave rise to a diverse range of genotypes across the Americas-Caribbean region, now referred to as “Creole”. Prior to the rise of intensive livestock farming in the 1970s, the CR pig breed played a key role in the subsistence economy of smallholder farmers in Martinique<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Nowadays, CR pig farming is in the minority, having largely been replaced by exotic genotypes selected for higher production and reproduction traits.</p>
<p id="Par3">The Regional Natural Park of Martinique (PNRM) has identified some feral pigs in the mountainous regions and aims to develop a niche market for local pig breeds through the re-domestication of these feral pigs. To achieve this, a program involving various stakeholders has been established, using a holistic dashboard approach<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Since 2016, feral pigs have been captured, re-domesticated by volunteer farmers, and bred to increase the domestic population. In 2022, the French National Research Institute for Agriculture, Food and Environment (INRAE) collected blood samples from these re-domesticated feral pigs to measure the inbreeding coefficient and monitor the genetic evolution of the population. The aims of the present study were (i) to identify the genetic structure of feral pig population in Martinique; (ii) to measure the inbreeding rate of the re-domesticated creole population sampled in 2022 and assess its evolution compared to the founder population sampled in 2016; (iii) to suggest practical breeding schemes encompassing genetic characterization, the re-domestication process, and inbreeding management.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Inbreeding of pig populations of Martinique</h3>
<p id="Par4">Figure <a href="#Fig1" class="usa-link">1</a> shows the inbreeding coefficients of the different pig groups sampled in 2016 and 2022. The results show that feral and semi-feral pigs exhibit the highest inbreeding levels. In contrast, pigs from professional and family farms in Martinique and Guadeloupe display low inbreeding coefficients, comparable to those of Creole pigs raised at INRAE. Highly significant differences (a: <em>P</em> &lt; 0.001) were observed between the inbreeding coefficients of feral pigs (FM) and pigs from family farms in Martinique (FamM). Pigs from the three creole pig farmers (Breeder 1, Breeder 2, Breeder 3) show intermediate inbreeding levels – higher than those of farmed pigs but lower than those of feral and semi-feral. Notably, the FamP group (pigs from specific family farms in Martinique) shows a relatively high inbreeding coefficient compared to other domesticated pigs. Re-domesticated creole pigs from Breeder 1 and Breeder 2, along with FamP pigs - potential ancestors of some re-domesticated feral pigs – display significantly higher inbreeding coefficients than pigs from family farms (FamM) and professional farms (ProM) in Martinique (b, c, d: <em>P</em> &lt; 0.001; e, f, h: <em>P</em> &lt; 0.01).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361484_41598_2025_15649_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4585/12361484/8a2ab2fd99eb/41598_2025_15649_Fig1_HTML.jpg" loading="lazy" id="d33e318" height="390" width="754" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Inbreeding coefficient (F) per farm. P-value (P). <em>a</em>, <em>b</em>, <em>c</em>,<em> d: P &lt; 0</em>,<em>001; e</em>,<em> f</em>,<em> g: P &lt; 0</em>,<em>01 ; h : P &lt; 0</em>,<em>05.</em> Breeder 1,2,3 : Re-domesticated feral pigs by three volunteer farmers in the PNRM program; FM: Feral pigs are animals captured by hunters or PNRM staff; SfM: Pigs crossed with feral pigs by farmers and kept on their farms; FamM: Pigs reared in family farms in Martinique; FamP: Pigs raised on family farms that are potential parents of CRM re-domesticated feral pigs; ProM: Pigs that are reared by pig professional farmers; FamG: Pigs reared in family farms in Guadeloupe; INRAE: Creole pigs that are conserved at the INRAE experimental facilities. Each boxplot represents, from top to bottom: the maximum value, the third quartile, the median (in bold), the first quartile, and the minimum value.</p></figcaption></figure><p id="Par6">Figure <a href="#Fig2" class="usa-link">2</a> illustrates the proportion of the genome in ROH for local pigs across various farms. A total of 194 ROH segments were identified in 65 individuals out of the 121 pigs genotyped in Martinique and Guadeloupe. The ROH results follow the same trend as the inbreeding coefficient, confirming a significant difference between domestic and feral pigs. Domesticated pigs from professional farms (ProM), family farms (FamM and FamP), as well as re-domesticated creole pig from feral population raised by Breeder 1 and Breeder 2, exhibit significantly fewer ROH regions compared to feral pigs (FM) (a: <em>P</em> &lt; 0.001; b, c, d, e: <em>P</em> &lt; 0.01). The short length of the box plots for domesticated creole pigs and feral pigs suggests relatively older inbreeding events. In contrast, the extended length of the box plot for the semi-feral group (SfM) indicates more recent inbreeding.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361484_41598_2025_15649_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4585/12361484/323b74677a47/41598_2025_15649_Fig2_HTML.jpg" loading="lazy" id="d33e365" height="357" width="752" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proportion of the local pigs’genome in ROHs in per farm. <em>P-value (P). a: P &lt; 0</em>,<em>001; b</em>,<em> c</em>,<em> d</em>,<em> e: P &lt; 0</em>,<em>01; f</em>,<em> g: P &lt; 0</em>,<em>005. 194 ROH detected in 65 individuals among the 121 individuals genotyped in Martinique and Guadeloupe.</em> Breeder 1,2,3 : Re-domesticated feral pigs by three volunteer farmers in the PNRM program; FM: Feral pigs are animals captured by hunters or PNRM staff; SfM: Pigs crossed with feral pigs by farmers and kept on their farms; FamM: Pigs reared in family farms in Martinique; FamP: Pigs raised on family farms that are potential parents of CRM re-domesticated feral pigs; ProM: Pigs that are reared by pig professional farmers; FamG: Pigs reared in family farms in Guadeloupe; INRAE: Creole pigs that are conserved at the INRAE experimental facilities. Each boxplot represents, from top to bottom: the maximum value, the third quartile, the median (in bold), the first quartile, and the minimum value.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Genetic diversity between pigs of Martinique</h3>
<p id="Par8">Figure <a href="#Fig3" class="usa-link">3</a> presents the results of the PCA analysis. Genetically similar individuals appear closer together based on the 13,578 SNPs markers used. The analysis revealed that the pigs assumed to belong to CR breed actually represent a wide range of genotypes. Based on Fig. <a href="#Fig3" class="usa-link">3</a>, ten distinct groups can be identified. Among the feral pigs, two groups emerge: one that is genetically distant from domesticated pigs, and another that is closer to domesticated creole pigs. The domesticated creole pigs in Martinique are divided into three groups: one group is closely related to two feral individuals, suggesting a potential kinship; another group is genetically similar to nine pigs from family farms in Martinique (FamP), indicating possible familial ties; and a third group is genetically distinct from both of the above, pointing to different family origins. In addition, there is a large cluster consisting of pigs from professional and family farms in both Martinique and Guadeloupe, along with Creole pigs from the PTEA facility. Within the family- farmed pigs in Martinique, one subgroup stands out as genetically differentiated from the main cluster. The semi-feral pigs form three distinct groups. Two of these are positioned genetically between the feral pigs and those from family and professional farms, reinforcing their intermediate status. The third group includes two individuals closely related to pigs from family farms in Martinique.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361484_41598_2025_15649_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4585/12361484/92dc357965cf/41598_2025_15649_Fig3_HTML.jpg" loading="lazy" id="d33e385" height="416" width="755" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA of genetic distances between animals genotyped in Martinique and Guadeloupe (The two French Caribbean islands). Breeder 1,2,3 : Re-domesticated feral pigs by three volunteer farmers in the PNRM program; FID: Family identifier, FM: Feral pigs are animals captured by hunters or PNRM staff; SfM: Pigs crossed with feral pigs by farmers and kept on their farms; FamM: Pigs reared in family farms in Martinique; FamP: Pigs raised on family farms that are potential parents of CRM re-domesticated feral pigs; ProM: Pigs that are reared by pig professional farmers; FamG: Pigs reared in family farms in Guadeloupe; INRAE: Creole pigs that are conserved at the INRAE experimental facilities.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Genetic admixture of pigs of Martinique</h3>
<p id="Par10">The analysis of the genetic structure of the different pig populations studied was conducted using a variable number of clusters (K), ranging from 7 to 20. The cross-validation error reveals a minimum at K = 10, suggesting that the genetic structure of the dataset is best explained by ten ancestral origins. Based on these results, we chose K = 10 to estimate ancestry coefficients, capturing the contribution of seven mainstream breeds along with Chinese ancestry. The results of the unsupervised analysis at K = 10 (Fig. <a href="#Fig4" class="usa-link">4</a>) show that the genetic structure of Chinese breeds (Jinhua, Jinguha), Iberian, Duroc, Piétrain, Hamshire, Gascon, and to a lesser extent, Large White and Landrace, is relatively homogeneous. These European and Chinese breeds were therefore used as reference genotypes to study the genetic structure of pigs in Martinique - particularly the feral population.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361484_41598_2025_15649_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4585/12361484/d35757d47245/41598_2025_15649_Fig4_HTML.jpg" loading="lazy" id="d33e412" height="991" width="687" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Admixture: From top to bottom (<strong>a</strong>) unsupervised (K = 10); (<strong>b</strong>) supervised admixture (K = 8), (<strong>c</strong>) supervised admixture (K = 10).</p></figcaption></figure><p id="Par12">The supervised analysis at K = 10 shows that similar to Creole pig populations across the Americas, the feral and semi-feral pigs of Martinique carry a notable Iberian genetic component (&gt; 20%). Estimates of ancestral composition show that this Iberian contribution is, on average, higher in feral pigs from Martinique than in semi-feral pigs from Martinique or Creole pigs from Guadeloupe, the other French Caribbean island (Table <a href="#Tab2" class="usa-link">2</a>). Specifically, the main genetic components of feral pigs in Martinique derive from the Landrace and Iberian breeds, whereas Creole pigs in Guadeloupe show stronger influence from the Large White and Landrace breeds. Notably, similar to the Creole pigs of Brazil, the feral pigs of Martinique also display a Chinese genetic component (&gt; 10%). In contrast, pigs from family and professional farms in Martinique exhibit a strong genetic influence from the Large White and Duroc breeds, with minimal Iberian ancestry.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Predicted genetic composition using ancestry coefficients from supervised analysis of 10 clusters.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Geographical regions or breeds<sup>1</sup>
</th>
<th align="left" colspan="1" rowspan="1">Iberian</th>
<th align="left" colspan="1" rowspan="1">Landrace</th>
<th align="left" colspan="1" rowspan="1">Large White</th>
<th align="left" colspan="1" rowspan="1">Duroc</th>
<th align="left" colspan="1" rowspan="1">Chinese</th>
<th align="left" colspan="1" rowspan="1">Jinhua</th>
<th align="left" colspan="1" rowspan="1">Jiangquhai</th>
<th align="left" colspan="1" rowspan="1">Gascon</th>
<th align="left" colspan="1" rowspan="1">Hampshire</th>
<th align="left" colspan="1" rowspan="1">Piétrain</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="11" rowspan="1">Martinique<sup>1</sup>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Feral</td>
<td align="center" colspan="1" rowspan="1">0.24</td>
<td align="center" colspan="1" rowspan="1">0.56</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.06</td>
<td align="center" colspan="1" rowspan="1">0.11</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Semi-feral</td>
<td align="center" colspan="1" rowspan="1">0.21</td>
<td align="center" colspan="1" rowspan="1">0.18</td>
<td align="center" colspan="1" rowspan="1">0.33</td>
<td align="center" colspan="1" rowspan="1">0.06</td>
<td align="center" colspan="1" rowspan="1">0.09</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.05</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Family</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.09</td>
<td align="center" colspan="1" rowspan="1">0.46</td>
<td align="center" colspan="1" rowspan="1">0.23</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Family parents</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">1.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Professional</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.10</td>
<td align="center" colspan="1" rowspan="1">0.39</td>
<td align="center" colspan="1" rowspan="1">0.30</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.16</td>
</tr>
<tr><td align="left" colspan="11" rowspan="1">Guadeloupe<sup>2</sup>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> INRAE</td>
<td align="center" colspan="1" rowspan="1">0.20</td>
<td align="center" colspan="1" rowspan="1">0.13</td>
<td align="center" colspan="1" rowspan="1">0.37</td>
<td align="center" colspan="1" rowspan="1">0.07</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.04</td>
<td align="center" colspan="1" rowspan="1">0.04</td>
<td align="center" colspan="1" rowspan="1">0.10</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Family</td>
<td align="center" colspan="1" rowspan="1">0.06</td>
<td align="center" colspan="1" rowspan="1">0.18</td>
<td align="center" colspan="1" rowspan="1">0.41</td>
<td align="center" colspan="1" rowspan="1">0.08</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">North America<sup>3.4</sup>
</td>
<td align="center" colspan="1" rowspan="1">0.36</td>
<td align="center" colspan="1" rowspan="1">0.04</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.34</td>
<td align="center" colspan="1" rowspan="1">0.18</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
</tr>
<tr><td align="left" colspan="11" rowspan="1">South America<sup>3.5</sup>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Feral</td>
<td align="center" colspan="1" rowspan="1">0.52</td>
<td align="center" colspan="1" rowspan="1">0.08</td>
<td align="center" colspan="1" rowspan="1">0.06</td>
<td align="center" colspan="1" rowspan="1">0.18</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.06</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Creole</td>
<td align="center" colspan="1" rowspan="1">0.54</td>
<td align="center" colspan="1" rowspan="1">0.07</td>
<td align="center" colspan="1" rowspan="1">0.04</td>
<td align="center" colspan="1" rowspan="1">0.15</td>
<td align="center" colspan="1" rowspan="1">0.08</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.07</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Brasil</td>
<td align="center" colspan="1" rowspan="1">0.52</td>
<td align="center" colspan="1" rowspan="1">0.05</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.16</td>
<td align="center" colspan="1" rowspan="1">0.17</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mediterranean<sup>3.7</sup>
</td>
<td align="center" colspan="1" rowspan="1">0.30</td>
<td align="center" colspan="1" rowspan="1">0.30</td>
<td align="center" colspan="1" rowspan="1">0.28</td>
<td align="center" colspan="1" rowspan="1">0.04</td>
<td align="center" colspan="1" rowspan="1">0.01</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.02</td>
<td align="center" colspan="1" rowspan="1">0.03</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p48"><p><sup>1</sup>See Table <a href="#Tab1" class="usa-link">1</a> for more information about the origin of the breeds. In this table, the category “Chinese” includes similar breeds such as Meishan and Xiang Pig.</p></div></div></section></section><section id="Sec6"><h3 class="pmc_sec_title">Breeding scheme selected for ex situ conservation programme</h3>
<p id="Par14">Figure <a href="#Fig5" class="usa-link">5</a> shows the main principles of the breeding scheme for the conservation of feral pigs. A low inbreeding rate should be maintained through rotational mating of sire lines, involving a minimum of five farmers. This minimum number was agreed upon through stakeholders’ discussions, balancing the need for at least four distinct herd groups for conservation purposes and the practical limitations of small-scale farming systems (i.e., a maximum of two boars and six sows per farm). Each circle in the Fig. <a href="#Fig5" class="usa-link">5</a> represents either an individual farm or a group of herds. The following rules have been validated by the stakeholders: (i) minimum genetic diversity: the number of sire and dam lines must be at least 10; each herd or group of herds must maintain these lines by replacing boars with their sons and sows with their daughters; (ii) avoiding inbreeding: sows must be mated with boars born in different herds or herd groups; mating animals born within the same herd is strictly prohibited; (iii) high turnover of sires: the number of boars annually should be maximized, and their replacement rate should be as rapid as possible; boars should be used for only one year and then replaced by their male offspring; in contrast, sows should be retained for as long as feasible, ideally for up to eight litters. By the end of the fourth rotation in this circular mating scheme, the pedigree of the creole pig breed in Martinique is expected to include data on at least 150 reproducers across eight generations.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361484_41598_2025_15649_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4585/12361484/a695351ff41b/41598_2025_15649_Fig5_HTML.jpg" loading="lazy" id="d33e437" height="642" width="696" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Rotational mating plan with maximum avoidance of inbreeding. Each small circle E represent a herd or a regional group of herds, arrows represent the rotation mating (from the first to the fifth generations for maximum avoidance of inbreeding scheme).</p></figcaption></figure></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par16">It is now well established that the use and management of local farm animal genetic resources can help address many of the global challenges we face, by preserving genotypes that may be essential in the future<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The creole pig breed, in particular, has been shaped by a unique crossbreeding history closely tied to the region’s colonial and agricultural past<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The main strengths of the Creole breeds lie in their adaptation to the humid tropical environment. These include heat tolerance<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>an enhanced ability to utilize tropical forages and crops, better mobilization of body reserves, and greater resistance to ecto- and endo-parasites than exotic genotypes<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par17">In our study, we detected a genetic structure comprising 10 ancestral origins. These findings are consistent with previously published data<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>identifying 8 ancestral origins, which highlight the extreme complexity of pig genetic structure across the Americas. Although they share the same name, Creole (CR) pigs differ between American territories, as they result from complex admixture processes involving breeds of diverse origins – shaped by each territory’s unique colonial history between the 15th and 19th centuries<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Despite their genetic diversity, CR pigs share a common feature: a predominant Iberian genetic contribution<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>ranging from 13 to 72%. In the present study, estimates of ancestral composition show that feral pigs in Martinique, like many pig populations across the Americas, carry a substantial proportion of Iberian ancestry – averaging 24%. Interestingly, the presence of feral pigs in Martinique was already noted in the 17th century. Labat<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> described them as follows: “<em>The maroon pigs found on the islands are of two kinds</em>,<em> and it is easy to distinguish between them. Those of the Spanish breed</em>,<em> that is to say</em>,<em> of those first introduced by the Spaniards at the beginning of their discoveries</em>,<em> are short and stocky</em>,<em> with large heads and short snouts.The second breed comes from the domestic pigs that escaped from the pens where they were kept. They are no different from those in France</em>,<em> from where their ancestors were brought</em>,<em> and it does not appear that these two breeds have intermingled…</em>”. Our results show that the proportion of Iberian ancestry is higher in feral pigs from Martinique than in the Creole pigs from Guadeloupe (24 vs. 20%), the other Francophone territory in the Caribbean. This difference may be attributed to the more isolated breeding of feral pigs, which live in the wild and are less exposed to crossbreeding compared to farm-raised pigs<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Contrary to the expectations or beliefs of some farmers, our analysis reveals that most pigs from family and professional farms have little to no Iberian genetic contribution. In cases where no pedigree information was available, we selected animals for genotyping based on their phenotypic resemblance to Creole pigs. This study highlights the value of genomic tools in identifying the genetic origins of animals in the absence of pedigree data. Indeed, genetic markers such as SNPs provide a powerful means to trace and evaluate the ancestry of sampled animals.</p>
<p id="Par18">Our analyses suggest a notable contribution of Chinese ancestry in the genetic makeup of feral and semi-feral pigs in Martinique, ranging from 9 to 11%. This could be explained by: i) indirect introgression through the incorporation of Chinese genes into European breeds to improve prolificity<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>; ii) direct introgression via the historical or more recent importation of Chinese pig genotypes into Martinique. Historical records support this hypothesis. Pigs from Asia were introduced to regions of the Americas during maritime trade between 1685 and 1841. Labat<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> reported, “<em>The French ships that stopped at the islands on their return from Siam and China brought another pig breed - short legged</em>,<em> with very little hair and a very large belly”</em>. Additionally, in rural regions of Guadeloupe and Martinique, elderly people use the term “Siam pigs” to refer to the Creole pig (Gourdine, personal communication).</p>
<p id="Par19">Our results on inbreeding management show that re-domesticated creole pigs from Breeders 2 and 3 have lower inbreeding coefficients than feral pigs (FM), but a significantly higher coefficients than pigs from family farms (FamM) and professional farms (ProM) in Martinique. These differences can be explained in two ways. First, the re-domesticated creole pigs likely exhibit intermediate levels of inbreeding because they originated from a mix of FM and FamM pigs. Second, the higher inbreeding levels observed in these re-domesticated pigs, compared to FamM and ProM pigs, may be due to the lack of formal breeding plans. In contrast, pigs from FamM and ProM farms benefit from more structured breeding practices, which aim to minimize inbreeding. As shown by Zhao et al.<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, although various breeding strategies exist, they generally aim to support reproduction while limiting the accumulation of inbreeding within the population. In contrast, the absence of structured breeding plans for re-domesticated creole pig – partly due to change in PNRM governance in 2016<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> – negatively impact the breeding practices of Breeders 2 and 3. During this period, these pigs were not managed under controlled breeding protocols, likely increasing the probability of mating between closely related individuals. The lack of genealogical information for rare breeds increases the risk of unintentional inbreeding by farmers<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par20">Overall, the majority of domesticated pig groups - including re-domesticated creole pigs – exhibit a significantly lower proportion of runs of homozygosity (ROH) compared to feral pigs. ROH is a recognized method for analyzing inbreeding in animal genetic resources<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. These results suggest that the management of animal genetic resources through structured breeding plans contributes to better control of inbreeding. Windig and Kaal-Lansbergen<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> proposed a herd management strategy based on a rotational breeding system, in which each herd serves as a donor for another in a cyclic manner. This approach has been shown to effectively limit the accumulation of inbreeding while remaining practical for implementation in conservation breeding programs. We drew inspiration from these studies<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> to engage with stakeholders in discussing conservation strategies.</p>
<p id="Par21">The holistic approach used in this study was both iterative and adaptive. All partners recognized the genetic uniqueness of the feral pigs in Martinique, which have adapted to the natural conditions of the tropical humid island. This breed could provide many products (marketable or not) and fulfil different ecological or social roles<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Moreover, the feral pig could play a role in climate-smart livestock systems and contribute to human health benefit<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Within this framework, researchers provided a foundation of scientific rigor in support of the public interest – specifically, in the management of the feral pig in Martinique as a common good. This collaborative and constructive environment, established gradually over time, led to the development of a conservation management plan aimed at reducing the risk of losing valuable genetic traits. While methods for conserving genetic diversity remain an active area of genetic research<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>many are based on simulation studies. These methods can be effective and widely applied – but only if they are adapted to the specific structure and functioning of the farms involved<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. The breeding scheme in the present study resulted from a consensus between scientific recommendations and the partners’ knowledge of the regional context – including the dynamics of breeder networks, the capacity for genetic material exchange, and the advantages and limitations of the proposed genetic management strategies. In our breeding scheme of Fig. <a href="#Fig5" class="usa-link">5</a>, during the first year and in line with Fig. <a href="#Fig3" class="usa-link">3</a>, it may be beneficial for each of the five illustrated farms to include a sample of individuals representing the five identified Creole pig groups in Martinique (two FM and three SfM). This initial distribution would ensure genetic diversity from the outset. This method of conservation breeding has been showed to be one of the most effective alternatives for avoiding high levels of inbreeding, particularly in the absence of detailed pedigree information from pedigree or high-density genotyping data<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. However, rotational mating methods alone may not be sufficient to maintain genetic diversity when the population carries deleterious inbreeding load<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In the long term, it will be essential to structure the mating plan using for instance the optimal contribution selection theory<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. This approach has been shown to allow for the selection of animals to improve traits on interest - such as meat quality - while maintaining low levels of inbreeding<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par22">In the present study, we carried out a genetic characterization of the feral pig population in Martinique. It is an essential step for the conservation and potential valorisation of local breed resources. Our findings show that the feral and semi-feral pigs share a genetic structure comparable to that of Creole breeds across the Americas, notably marked by a high proportion of Iberian genetic inheritance. These results also highlight the relevance of the saying “<em>do not judge a book by its cover</em>”. They show how genetic markers serves as powerful tools for objectively distinguishing “true Creole” pigs from those that merely resemble them phenotypically – underscoring the importance of molecular tools in guiding conservation and breeding efforts.</p></section><section id="Sec41"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Location and genotypes sampling</h3>
<p id="Par23">This research was carried out in Martinique, in collaboration with the staff of the animal experimental station, the Tropical Livestock Platform for Agroecology (PTEA) of the French National Research Institute for Agriculture, Food and Environment (INRAE). This experimental unit is accredited for animal experimentation under number A971802, and the involved staff members are trained in animal experimentation and welfare. Animals were treated in accordance with the guidelines and regulations for animal experimentation set by the French Ministry of Agriculture. The protocol (#69-2012-2) was approved by the Ministry of Higher Education and Research, following the advice of the Animal Care and Use Committee of French West Indies and Guyana (N°069). The authors confirm compliance with the ARRIVE guidelines<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Informed consent was obtained from all farm owners prior to animal blood sampling.</p>
<p id="Par24">The animals studied were distributed across 16 of the 34 districts of the island, primarily in areas where agriculture activities predominate. Figure <a href="#Fig6" class="usa-link">6</a> shows the geographical distribution of the genotyped pigs in Martinique according to their origin (feral, semi-feral, pigs reared on family farms, or pigs from professional pig farms). Tourist areas, urban zones, and regions with declining agricultural activity were not included in the study.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361484_41598_2025_15649_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4585/12361484/878db1b5a920/41598_2025_15649_Fig6_HTML.jpg" loading="lazy" id="d33e635" height="482" width="711" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Geographical distribution of pigs genotyped in Martinique.</p></figcaption></figure></section><section id="Sec10"><h3 class="pmc_sec_title">Holistic approach</h3>
<p id="Par26">The overall objective of the present project required scientific and technical expertise across several fields, including genetic characterization, small population management, and farmers’ practices. To develop a comprehensive set of specifications - such as conservation schemes for pigs - we adopted a holistic approach. Each partner contributed complementary skills: project coordination was led by the consultancy firm Ruralité Multiservices; technical expertise and territory knowledge were provided by the Agricultural Chamber of Martinique, the PNRM, local farmers, and the pig cooperative COOPMAR; and scientific and technical expertise came from INRAE<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Experimental protocols – including animal capture, sanitary management, breeding conditions, and mating management - were discussed and regularly updated based on the following principle: proposal originated from scientific expertise; while implementation was guided by the on-the-ground knowledge of the other partners.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Animals studied and data used</h3>
<p id="Par27">A non-negligible portion of pig farming in Martinique operates within the popular economy<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> beyond the reach of official oversight and public support. According to the previous investigations by the PNRM, feral pigs roam freely in the mountainous regions of the north of the island. Some of these pigs were captured and raised either in family backyards or in small-scale pig farming operations (referred to as semi-feral pigs). As a result, it was necessary to conduct a survey across the territory of Martinique to identify farmers who may be rearing these animals<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par28">The pigs genotyped in this study came from two distinct groups. The first group includes animals genotyped during surveys conducted across Martinique in 2016. These surveys aimed to identify breeders potentially raising creole pigs. Following these investigations, 76 pigs were genotyped: 7 feral pigs from Martinique, 8 semi-feral pigs, 52 farmed pigs from Martinique, 6 farmed pigs from Guadeloupe and 3 from the PTEA experimental flock of INRAE. All pigs were considered by their owners to belong to the Creole breed, based on morphological aspects such as predominantly black or ash-grey coats, broad and forward-drooping ears, and in some cases, the presence of neck wattles. The second group consists of 45 pigs genotyped in 2022, potentially descended from pigs identified as feral in Martinique in 2016. The animals had not been managed under a specific breeding plan. The purpose of this genotyping was to measure the inbreeding rate of 2022 re-domesticated feral pig population and to assess its genetic evolution since 2016. Blood samples for DNA extraction were collected via jugular venipuncture into two 6-ml tubes containing anticoagulant per pig and stored at 4 °C. After visiting all the farms, the samples were transferred to the laboratory, where they were stored at -20 °C and later transported using dry ice to LABOGENA-DNA - the French reference laboratory for livestock genotyping (ISO 170025 accredited, Jouy-en-Josas, France).</p>
<p id="Par29">To investigate the potential genetic origins of these pigs, a total of 538 reference mainstream genotypes were used. These data were source from various studies and institutions: (i) 374 pig genotypes from the Americas-Caribbean region and Europe<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>; (ii) 4 genotypes of CR pigs from Guadeloupe studied by Rosé et al.<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> and maintained at the INRAE experimental facilities in Guadeloupe (16°N, 61°W); (iii) 6 pigs considered to be of the CR breed, identified on family farms in Guadeloupe through surveys<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> and (iv) 62 Piétrain and 26 Gascon genotypes provided by INRAE UMR GenPhySE. Table <a href="#Tab1" class="usa-link">1</a> summarizes the pig genotypes used in this study.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Origins of the pig genotypes used in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Geographical regions or breeds</th>
<th align="left" colspan="1" rowspan="1">Number of genotypes</th>
<th align="left" colspan="1" rowspan="1">Abbreviation used</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="3" rowspan="1">Martinique<sup>1</sup>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Creole</td>
<td align="center" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">CRM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Feral</td>
<td align="center" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">FM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Semi-feral</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">SfM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Family</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">FamM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Family parents</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">FamP</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Professional</td>
<td align="center" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">ProM</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">Guadeloupe<sup>2</sup>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> INRAE</td>
<td align="center" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">INRAE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Family</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">FamG</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">South America<sup>3,4</sup>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Feral</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">FSA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Semi-feral</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">SfSA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Creole</td>
<td align="center" colspan="1" rowspan="1">110</td>
<td align="left" colspan="1" rowspan="1">CR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Brasil</td>
<td align="center" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">BR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">North America<sup>3,5</sup>
</td>
<td align="center" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">USA</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">Mainstream breeds</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Duroc<sup>3</sup>
</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">DU</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Hampshire<sup>3</sup>
</td>
<td align="center" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">HA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Large White<sup>3</sup>
</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">LW</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Landrace<sup>3</sup>
</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">LR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Iberian<sup>3</sup>
</td>
<td align="center" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">IB</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Piétrain<sup>6</sup>
</td>
<td align="center" colspan="1" rowspan="1">62</td>
<td align="left" colspan="1" rowspan="1">PI</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Gascon<sup>6</sup>
</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">GA</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">Other breeds or other regions</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Mediterranean<sup>3,7</sup>
</td>
<td align="center" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">Med</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Wild boar<sup>3</sup>
</td>
<td align="center" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1">WB</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Chinese<sup>3,8</sup>
</td>
<td align="center" colspan="1" rowspan="1">58</td>
<td align="left" colspan="1" rowspan="1">CH</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p37">
<p><sup>1</sup>Martinique: genotypes are from our study; Creole are feral pigs re-domesticated by three volunteer farmers in the PNRM program; feral pigs are animals captured by hunters or PNRM staff; semi-feral pigs are Pigs crossed with feral pigs by farmers and kept on their farms; family corresponds to pigs reared in family farms; family parents correspond to pigs raised on family farms that are potential parents of CRM re-domesticated feral pigs; professional corresponds to pigs that are reared by pig professional farmers.</p>
<p><sup>2</sup>Guadeloupe: INRAE corresponds to 3 Creole pigs that are conserved at the INRAE experimental facilities and 4 genotypes from the study of Rose et al.<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>; Family corresponds to pigs reared in family farms.</p>
<p><sup>3</sup>Genotypes were obtained from the study of Burgos-Paz et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p><sup>4</sup>South America: feral corresponds to genotypes of feral and semi-feral pigs from Argentina; Creole corresponds to genotypes of pigs from Argentina, Bolivia, Colombia, Costa Rica, Cuba, Ecuador, Guatemala, Mexico, Peru, and Uruguay; Brasil corresponds to genotypes of 9 Moura pigs from Concordia, 10 Monteiro pigs from Poconé, 9 Piau pigs from Bahia and 2 Nilo pigs from Goias.</p>
<p><sup>5</sup>North America: genotypes are 7 Ossabaw pigs from the Ossabaw island, 10 Yucatan pigs from Indiana and 15 Guinea hog pigs from several USA locations.</p>
<p><sup>6</sup>Piétrain genotypes were obtained from INRAE.</p>
<p><sup>7</sup>Mediterranean corresponds to 14 Bisaro pigs from Portugal, 4 Canarian pigs from Spain, 4 Black Sicilian pigs from Italy.</p>
<p><sup>8</sup>Chinese corresponds to 13 Xiang Pig, 11 Jiangquhai pigs, 17 Jinhua pigs and 17 Meishan pigs from China.</p>
</div></div></section><p id="Par31">First, we applied quality control filtering to the genotype data using Plink, excluding SNPs and individuals based on missingness, minor allele frequency, and Hardy-Weindberg equilibrium thresholds. We then performed linkage disequilibrium (LD) pruning to remove correlated SNPs and retain only independent markers. Then we performed an Admixture analysis in unsupervised mode, meaning that no ancestral reference genotypes were defined beforehand. The software grouped individuals based on their genetic similarities, with the results displayed as color bands on a graph. This first analysis provides a preliminary view of the genetic composition of the studied samples by comparing them to the various genotypes included in the dataset. Next, based on the optimal number of clusters determined using cross-validation, we performed a supervised Admixture analysis, in which ancestral reference genotypes were specified. This approach enabled us to estimate the relative genetic composition of our study samples by comparing them reference populations. The reference populations were selected based on the study by Burgos-Paz et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>who investigated the genetic structure of Creole pig populations across the Americas. Key ancestral breeds, identified in that study included Iberian, Landrace, Large White, Duroc, Hampshire, and Chinese pigs. To these, we added Piétrain and Gascon genotypes, provided by the INRAE GenPhySE laboratory. These additional genotypes were included based on surveys conducted in 2016, in which breeders in Martinique reported their potential use in crossbreeding<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. The Table <a href="#Tab2" class="usa-link">2</a> shows the genetic composition using ancestry coefficients from supervised analysis of 10 clusters. Due to the strong genetic similarities observed in the admixture analysis with K = 10 (Fig. <a href="#Fig4" class="usa-link">4</a>), the “Chinese” group in Table <a href="#Tab2" class="usa-link">2</a> includes both Meishan and Xiang Pig breeds.</p></section><section id="Sec67"><h3 class="pmc_sec_title">Genotyping and quality control</h3>
<p id="Par32">DNA extraction and blood plasma genotyping were performed by LABOGENA-DNA. The initial genotyping was carried out using the Illumina PorcineSNP60 BeadChip, which contains 61,565 markers. The second round of genotyping, performed in 2022, used the custom PORC-XT_60K_V2 porcine chip, developed by Labogena<sup>®</sup> in partnership with Choice Genetics. This chip is also based on Illumina’s<sup>®</sup> Infinium technology and includes 57,811 SNP markers. The reference genome used was <em>Sus scrofa</em> Build 11.1. Quality control of the genotyping data was performed using PLINK 1.09. Markers were excluded if they had more than 5% missing genotypes, a minor allele frequency (MAF) below 1%, or significant deviation from Hardy-Weinberg equilibrium. Additionally, individuals with more than 5% missing genotype data were also excluded. After quality control, a total of 35,657 SNPs across 583 genotypes were retained. Prior to conducting Principal Component Analysis (PCA) and Admixture analysis, we performed linkage disequilibrium (LD) pruning using the --indep flag of PLINK with the following parameters: window size of 60 SNPs, step size of 5 SNPs, and a variance inflation factor threshold of 2. The resulting dataset included 13,578 SNP and 583 genotypes. This LD pruning step helps to reduce redundancy by eliminating SNPSs in strong LD, ensuring that the remaining markers represent more independent genetic variation and thereby improving the accuracy of subsequent analyses<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p></section><section id="Sec456"><h3 class="pmc_sec_title">Calculations and statistical analysis</h3>
<p id="Par33">Principal Component Analysis (PCA), used to estimate genetic distances between individuals in our sample, was performed using PLINK version 1.09<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> with the --pca parameter. This analysis allows for the visualization of genetic similarities and differences among pigs sampled in Martinique. Genetic distances between two individuals were calculated based on kinship coefficients, estimated from allele frequencies at various loci distributed across the 18 autosomes. These distances reflect both shared ancestry and broader population-level genetic divergence.</p>
<p id="Par34">The inbreeding coefficient was calculated using the --het flag with PLINK 1.09. This analysis computes the observed and expected counts of homozygous autosomal genotype for each sample. The method-of-moments <em>F</em> coefficient was estimated using the formula:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4585/12361484/5365cd304056/d33e1334.gif" loading="lazy" id="d33e1334" alt="graphic file with name d33e1334.gif"></td></tr></table>
<p id="Par35">Initially, an ANOVA test was conducted<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. While the Shapiro-Wilk test confirmed the normality of residuals, Levene’s test indicated significant variance differences among groups, violating the assumption of homogeneity. As a result, a non-parametric approach was adopted. The <em>p-value</em> was estimated using the Kruskal-Wallis test, followed by Dunn’s post-hoc test with Bonferroni correction<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>using R software version 4.2.1<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, and the FSA package version 0.9.6<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par36">Runs of homozygosity (ROH) are one of the primary methods used to assess inbreeding in animal genetic resources<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. ROHs are defined as continuous chromosomal segments in which all loci exhibit homozygous genotypes<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. These homozygous sequences, presumed to originate from a common ancestor, vary in length depending on the level of inbreeding. Short ROHs typically indicate ancient inbreeding, while long ROHs are suggestive of more recent inbreeding events<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. The ROH parameters used in this study were based on the recommendations of Ceballos et al.<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> and Manunza et al.<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. ROHs were detected using PLINK 1.09 with the following settings: a minimum ROH length of 1,000 kb (--homozyg-kb 1000), a minimum of 50 SNPs per ROH (--homozyg-snp 50), a maximum of one heterozygous site allowed per ROH (--homozyg-het 1), and a minimum SNP density of 1 SNP per 100 kb (--homozyg-density 100). The <em>p-value</em> for ROH-based inbreeding differences among groups were estimated using ANOVA. The Shapiro-Wilk test confirmed normality of the residuals, and the Levene’s test indicated homogeneity of variance among groups. Tukey’s HSD test<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> and Student’s <em>t</em>-test were used for pairwise comparisons, with a significance level of 5%, all performed in R software version 4.2.1<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p>
<p id="Par37">Ancestry proportions for each individual were estimated using ADMIXTURE software version 1.3.0<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The optimal number of clusters was determined by minimizing the cross-validation error, using the --cv flag in ADMIXTURE and testing cluster numbers from 7 to 20. This analysis was performed on 121 pig genotypes from Martinique and Guadeloupe to identify the ancestral origins of the sampled populations. To improve estimation accuracy in ADMIXTURE analysis, it is recommended to exclude closely related individuals. Therefore, re-domesticated pigs from Martinique (CRM) were removed, as some individuals from familial farms in Martinique (FamM), as well as from feral (FM), and semi-feral (SfM) populations, were likely close relatives. The final ADMIXTURE analysis was performed on 538 genotypes and 13,578 SNPs.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the PNRM and the Martinique Territorial Community, whose contributions are gratefully acknowledged. We also acknowledge funding from the EU, the Region of Guadeloupe (through the AgroEcoDiv project) and the CPER CRB-PSA project. The authors extend their thanks to R. Brithmer, S. Hoche-Balustre, J. Louis-Sidney and C. Biamba from the PNRM for their administrative support, W. Delyon and I. Semjen from the Ruralité-Multiservices for conducting the surveys, A. Limery and M. Cyrille from COOPMAR and V. Gauthier from the Agricultural Chamber of Martinique for their technical assistance. We are also grateful to Y. Labrune and J. Riquet from the research unit GenPhySE for their advices on conducting the genetic analyses. Finally, we warmly thank the farmers who generously gave their time to engage with and participate in the project – especially the pig farmers Duventru, Nino, Montluc and Audinay - for their insightful discussions, assistance, and overall contribution to the project.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>N.D., G.A. and J-L.G. conceived and designed the project. M.B., K.B. and C.V-D. managed the collection of data. M.B. and K.B. mainly collected the samples and participated to meeting with farmers. N.D. and Y.L. analysed the data. N.D., Y.L. and J-L.G. analysed the data, discussed and interpreted the results. N.D. and J-L.G. wrote the original draft. N.P and N.M. reviewed the paper. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>Sequence data that support the findings of this study have been deposited and are openly available on Research.Data.gouv with the following DOI: https://doi.org/10.57745/HWCCD5 . We also used the published data from Burgos-Paz et al. 2013 available at Dryad: https://doi.org/10.5061/dryad.t1r3d .</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethical approval</h3>
<p id="Par40">The authors confirm compliance with the ARRIVE guidelines. All measurements and observations on animals were performed in accordance with the current law on animal experimentation and ethics (#69-2016-1 from the Animal Care and Use Committee of French West Indies and Guyana) and the experimental protocol was approved by the French Ministry of Agriculture and Fisheries (#A971-18-02) under the direction of J. Fleury (INRA-PTEA).</p></section><section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par39">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lassaletta, L. et al. Food and feed trade as a driver in the global nitrogen cycle: 50-year trends. <em>Biogeochemistry</em><strong>118</strong>, 225–241. 10.1007/s10533-013-9923-4 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lassaletta,%20L.%20et%20al.%20Food%20and%20feed%20trade%20as%20a%20driver%20in%20the%20global%20nitrogen%20cycle:%2050-year%20trends.%20Biogeochemistry118,%20225%E2%80%93241.%2010.1007/s10533-013-9923-4%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Poore, J. &amp; Nemecek, T. Reducing food’s environmental impacts through producers and consumers. <em>Science</em><strong>360</strong>, 987–992. 10.1126/science.aaq0216 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aaq0216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29853680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poore,%20J.%20&amp;%20Nemecek,%20T.%20Reducing%20food%E2%80%99s%20environmental%20impacts%20through%20producers%20and%20consumers.%20Science360,%20987%E2%80%93992.%2010.1126/science.aaq0216%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Dourmad, J. Y., Salaün, Y., Lebret, B. &amp; Riquet, J. Diversité des productions porcines En France. <em>Innov. Agron.</em><strong>68</strong>, 151–170. 10.15454/VUZNNY (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dourmad,%20J.%20Y.,%20Sala%C3%BCn,%20Y.,%20Lebret,%20B.%20&amp;%20Riquet,%20J.%20Diversit%C3%A9%20des%20productions%20porcines%20En%20France.%20Innov.%20Agron.68,%20151%E2%80%93170.%2010.15454/VUZNNY%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Hoffmann, I. Adaptation to climate change – exploring the potential of locally adapted breeds. <em>Animal</em><strong>7</strong>, 346–362. 10.1017/S1751731113000815 (2013).
</cite> [<a href="https://doi.org/10.1017/S1751731113000815" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23739476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoffmann,%20I.%20Adaptation%20to%20climate%20change%20%E2%80%93%20exploring%20the%20potential%20of%20locally%20adapted%20breeds.%20Animal7,%20346%E2%80%93362.%2010.1017/S1751731113000815%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Bruford, M. W. et al. Prospects and challenges for the conservation of farm animal genomic resources, 2015–2025. <em>Front. Genet.</em><strong>6</strong>10.3389/fgene.2015.00314 (2015).</cite> [<a href="https://doi.org/10.3389/fgene.2015.00314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4612686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26539210/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Phocas, F. et al. Quels programmes d’amélioration génétique des Animaux pour des systèmes d’élevage agro-écologiques ? <em>INRA Prod. Anim.</em><strong>30</strong>, 31–46. 10.20870/productions-animales.2017.30.1.2232 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Phocas,%20F.%20et%20al.%20Quels%20programmes%20d%E2%80%99am%C3%A9lioration%20g%C3%A9n%C3%A9tique%20des%20Animaux%20pour%20des%20syst%C3%A8mes%20d%E2%80%99%C3%A9levage%20agro-%C3%A9cologiques%20?%20INRA%20Prod.%20Anim.30,%2031%E2%80%9346.%2010.20870/productions-animales.2017.30.1.2232%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Ozier Lafontaine, H. &amp; Lesueur Jannoyer, M. <em>Sustainable Agriculture Reviews 14: Agroecology and Global Change</em> (Springer [Allemagne], 2014).</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Rodero Serrano, E., Rodero Franganillo, A. &amp; Delgado-Bermejo, J. V. Primitive andalusian livestock and their implications in the discovery of America. <em>Arch. Zootec.</em><strong>41</strong>(154), 383–400 (1992).</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>Barrau, J. Sur Le «cochon-planche» des petites Antilles. <em>J. Agric. Tradit Bot. Appliquée</em>. <strong>25</strong>, 195–201. 10.3406/jatba.1978.3770 (1978).</cite> [<a href="https://scholar.google.com/scholar_lookup?Barrau,%20J.%20Sur%20Le%20%C2%ABcochon-planche%C2%BB%20des%20petites%20Antilles.%20J.%20Agric.%20Tradit%20Bot.%20Appliqu%C3%A9e.%2025,%20195%E2%80%93201.%2010.3406/jatba.1978.3770%20(1978)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Gourdine, J. L. et al. <em>Research Framework for the Development of Creole Pig’s Niche Market in Martinique: a Holistic Approach</em> Vol. 52 (Le Gosier, 2016).</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Gourdine, J. L. et al. Genotype by environment interactions for performance and thermoregulation responses in growing pigs. <em>J. Anim. Sci.</em><strong>97</strong>, 3699–3713. 10.1093/jas/skz245 (2019).
</cite> [<a href="https://doi.org/10.1093/jas/skz245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6735898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31351442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gourdine,%20J.%20L.%20et%20al.%20Genotype%20by%20environment%20interactions%20for%20performance%20and%20thermoregulation%20responses%20in%20growing%20pigs.%20J.%20Anim.%20Sci.97,%203699%E2%80%933713.%2010.1093/jas/skz245%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Poullet, N. et al. Effect of feed restriction and refeeding on performance and metabolism of European and Caribbean growing pigs in a tropical climate. <em>Sci. Rep.</em><strong>9</strong>, 4878. 10.1038/s41598-019-41145-w (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-41145-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6424954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30890729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poullet,%20N.%20et%20al.%20Effect%20of%20feed%20restriction%20and%20refeeding%20on%20performance%20and%20metabolism%20of%20European%20and%20Caribbean%20growing%20pigs%20in%20a%20tropical%20climate.%20Sci.%20Rep.9,%204878.%2010.1038/s41598-019-41145-w%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Gautier, M. &amp; Naves, M. Footprints of selection in the ancestral admixture of a new world Creole cattle breed. <em>Mol. Ecol.</em><strong>20</strong>, 3128–3143. 10.1111/j.1365-294X.2011.05163.x (2011).
</cite> [<a href="https://doi.org/10.1111/j.1365-294X.2011.05163.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21689193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gautier,%20M.%20&amp;%20Naves,%20M.%20Footprints%20of%20selection%20in%20the%20ancestral%20admixture%20of%20a%20new%20world%20Creole%20cattle%20breed.%20Mol.%20Ecol.20,%203128%E2%80%933143.%2010.1111/j.1365-294X.2011.05163.x%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Burgos-Paz, W. et al. Porcine colonization of the americas: a 60k SNP story. <em>Heredity</em><strong>110</strong>, 321–330. 10.1038/hdy.2012.109 (2013).
</cite> [<a href="https://doi.org/10.1038/hdy.2012.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3607178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23250008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burgos-Paz,%20W.%20et%20al.%20Porcine%20colonization%20of%20the%20americas:%20a%2060k%20SNP%20story.%20Heredity110,%20321%E2%80%93330.%2010.1038/hdy.2012.109%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Naves, M., Alexandre, G., Leimbacher, F., Mandonnet, N. &amp; Menendez-Buxadera, A. Les ruminants domestiques de La caraïbe: Le point Sur Les ressources génétiques et Leur exploitation. <em>INRAE Prod. Anim.</em><strong>14</strong>, 181–192. 10.20870/productions-animales.2001.14.3.3738 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Naves,%20M.,%20Alexandre,%20G.,%20Leimbacher,%20F.,%20Mandonnet,%20N.%20&amp;%20Menendez-Buxadera,%20A.%20Les%20ruminants%20domestiques%20de%20La%20cara%C3%AFbe:%20Le%20point%20Sur%20Les%20ressources%20g%C3%A9n%C3%A9tiques%20et%20Leur%20exploitation.%20INRAE%20Prod.%20Anim.14,%20181%E2%80%93192.%2010.20870/productions-animales.2001.14.3.3738%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Labat, J.-B. (1663–1738) A. du texte. <em>Nouveau voyage aux isles de l’Amérique. T. 1 /, contenant l’histoire naturelle de ces pays, l’origine, les moeurs, la religion et le gouvernement des habitans anciens et modernes, les guerres et les événemens singuliers qui y sont arrivez… le commerce et les manufactures qui y sont établies… [Par le P. J.-B. Labat]</em>. (1722).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Gourdine, J. L., Silou, F. &amp; Lebrun, A. Investigaciones Para evaluar La diversidad En productores de Cerdos criollos de Guadeloupe. (2009).</cite>
</li>
<li id="CR18">
<span class="label">18.</span><cite>Chen, H. et al. Introgression of Eastern Chinese and Southern Chinese haplotypes contributes to the improvement of fertility and immunity in European modern pigs. <em>GigaScience</em><strong>9</strong>10.1093/gigascience/giaa014 (2020).</cite> [<a href="https://doi.org/10.1093/gigascience/giaa014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7059266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32141510/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Zhao, F. et al. Genetic gain and inbreeding from simulation of different genomic mating schemes for pig improvement. <em>J. Anim. Sci. Biotechnol.</em><strong>14</strong>, 87. 10.1186/s40104-023-00872-x (2023).
</cite> [<a href="https://doi.org/10.1186/s40104-023-00872-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10262571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37309010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20F.%20et%20al.%20Genetic%20gain%20and%20inbreeding%20from%20simulation%20of%20different%20genomic%20mating%20schemes%20for%20pig%20improvement.%20J.%20Anim.%20Sci.%20Biotechnol.14,%2087.%2010.1186/s40104-023-00872-x%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Degras, N., Labatut, J. &amp; Gourdine, J. L. État des lieux de la gestion de la population porcine férale de Martinique pour une gestion durable du porc féral et des écosystèmes forestiers martiniquais. (GIP Ecofor, 2023).</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Roberts, K. S. &amp; Lamberson, W. R. Relationships among and variation within rare breeds of swine. <em>J. Anim. Sci.</em><strong>93</strong>, 3810–3813. 10.2527/jas.2015-9001 (2015).
</cite> [<a href="https://doi.org/10.2527/jas.2015-9001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26440160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roberts,%20K.%20S.%20&amp;%20Lamberson,%20W.%20R.%20Relationships%20among%20and%20variation%20within%20rare%20breeds%20of%20swine.%20J.%20Anim.%20Sci.93,%203810%E2%80%933813.%2010.2527/jas.2015-9001%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Meyermans, R., Gorssen, W., Buys, N. &amp; Janssens, S. How to study runs of homozygosity using PLINK? A guide for analyzing medium density SNP data in livestock and pet species. <em>BMC Genom.</em><strong>21</strong>, 94. 10.1186/s12864-020-6463-x (2020).</cite> [<a href="https://doi.org/10.1186/s12864-020-6463-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6990544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31996125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meyermans,%20R.,%20Gorssen,%20W.,%20Buys,%20N.%20&amp;%20Janssens,%20S.%20How%20to%20study%20runs%20of%20homozygosity%20using%20PLINK?%20A%20guide%20for%20analyzing%20medium%20density%20SNP%20data%20in%20livestock%20and%20pet%20species.%20BMC%20Genom.21,%2094.%2010.1186/s12864-020-6463-x%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Peripolli, E. et al. Runs of homozygosity: current knowledge and applications in livestock. <em>Anim. Genet.</em><strong>48</strong>, 255–271. 10.1111/age.12526 (2017).
</cite> [<a href="https://doi.org/10.1111/age.12526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27910110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peripolli,%20E.%20et%20al.%20Runs%20of%20homozygosity:%20current%20knowledge%20and%20applications%20in%20livestock.%20Anim.%20Genet.48,%20255%E2%80%93271.%2010.1111/age.12526%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Windig, J. &amp; Kaal-Lansbergen, L. An effective rotational mating scheme for inbreeding reduction in captive populations illustrated by the rare sheep breed Kempisch Heideschaap. <em>Anim. Int. J. Anim. Biosci.</em><strong>2</strong>, 1733–1741. 10.1017/S1751731108003029 (2008).</cite> [<a href="https://doi.org/10.1017/S1751731108003029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22444078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Windig,%20J.%20&amp;%20Kaal-Lansbergen,%20L.%20An%20effective%20rotational%20mating%20scheme%20for%20inbreeding%20reduction%20in%20captive%20populations%20illustrated%20by%20the%20rare%20sheep%20breed%20Kempisch%20Heideschaap.%20Anim.%20Int.%20J.%20Anim.%20Biosci.2,%201733%E2%80%931741.%2010.1017/S1751731108003029%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Nozieres-Petit, M. O. &amp; Lauvie, A. Diversité des contributions des systèmes dʼélevage de races locales. Les points de vue des éleveurs de Trois races ovines méditerranéennes. <em>Cah Agric.</em><strong>27</strong>, 65003. 10.1051/cagri/2018039 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nozieres-Petit,%20M.%20O.%20&amp;%20Lauvie,%20A.%20Diversit%C3%A9%20des%20contributions%20des%20syst%C3%A8mes%20d%CA%BC%C3%A9levage%20de%20races%20locales.%20Les%20points%20de%20vue%20des%20%C3%A9leveurs%20de%20Trois%20races%20ovines%20m%C3%A9diterran%C3%A9ennes.%20Cah%20Agric.27,%2065003.%2010.1051/cagri/2018039%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Erb, K. H. et al. Exploring the biophysical option space for feeding the world without deforestation. <em>Nat. Commun.</em><strong>7</strong>, 11382. 10.1038/ncomms11382 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms11382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4838894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27092437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erb,%20K.%20H.%20et%20al.%20Exploring%20the%20biophysical%20option%20space%20for%20feeding%20the%20world%20without%20deforestation.%20Nat.%20Commun.7,%2011382.%2010.1038/ncomms11382%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Shafer, A. B. A. et al. Genomics and the challenging translation into conservation practice. <em>Trends Ecol. Evol.</em><strong>30</strong>, 78–87. 10.1016/j.tree.2014.11.009 (2015).
</cite> [<a href="https://doi.org/10.1016/j.tree.2014.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25534246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shafer,%20A.%20B.%20A.%20et%20al.%20Genomics%20and%20the%20challenging%20translation%20into%20conservation%20practice.%20Trends%20Ecol.%20Evol.30,%2078%E2%80%9387.%2010.1016/j.tree.2014.11.009%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Wang, J., Santiago, E. &amp; Caballero, A. Prediction and estimation of effective population size. <em>Heredity</em><strong>117</strong>, 193–206. 10.1038/hdy.2016.43 (2016).
</cite> [<a href="https://doi.org/10.1038/hdy.2016.43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5026755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27353047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.,%20Santiago,%20E.%20&amp;%20Caballero,%20A.%20Prediction%20and%20estimation%20of%20effective%20population%20size.%20Heredity117,%20193%E2%80%93206.%2010.1038/hdy.2016.43%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Verrier, E. et al. Assessing the risk status of livestock breeds: a multi-indicator method applied to 178 French local breeds belonging to ten species. <em>Anim. Genet. Resour. Génétiques Anim. Genéticos Anim.</em><strong>57</strong>, 105–118. 10.1017/S2078633615000260 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Verrier,%20E.%20et%20al.%20Assessing%20the%20risk%20status%20of%20livestock%20breeds:%20a%20multi-indicator%20method%20applied%20to%20178%20French%20local%20breeds%20belonging%20to%20ten%20species.%20Anim.%20Genet.%20Resour.%20G%C3%A9n%C3%A9tiques%20Anim.%20Gen%C3%A9ticos%20Anim.57,%20105%E2%80%93118.%2010.1017/S2078633615000260%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Meuwissen, T. H. &amp; Sonesson, A. K. Maximizing the response of selection with a predefined rate of inbreeding: overlapping generations. <em>J. Anim. Sci.</em><strong>76</strong>, 2575–2583. 10.2527/1998.76102575x (1998).
</cite> [<a href="https://doi.org/10.2527/1998.76102575x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9814896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen,%20T.%20H.%20&amp;%20Sonesson,%20A.%20K.%20Maximizing%20the%20response%20of%20selection%20with%20a%20predefined%20rate%20of%20inbreeding:%20overlapping%20generations.%20J.%20Anim.%20Sci.76,%202575%E2%80%932583.%2010.2527/1998.76102575x%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Theodorou, K. &amp; Couvet, D. Genetic management of captive populations: the advantages of circular mating. <em>Conserv. Genet.</em><strong>11</strong>, 2289–2297. 10.1007/s10592-010-0114-8 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Theodorou,%20K.%20&amp;%20Couvet,%20D.%20Genetic%20management%20of%20captive%20populations:%20the%20advantages%20of%20circular%20mating.%20Conserv.%20Genet.11,%202289%E2%80%932297.%2010.1007/s10592-010-0114-8%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>de Rochambeau, H. &amp; Chevalet, C. Minimisation des coefficients de consanguinité moyens Dans les petites populations d’animaux domestiques. <em>Génétique Sélection Évolution</em>. <strong>17</strong>, 459 (1985).
</cite> [<a href="https://doi.org/10.1186/1297-9686-17-4-459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2713977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22879220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Rochambeau,%20H.%20&amp;%20Chevalet,%20C.%20Minimisation%20des%20coefficients%20de%20consanguinit%C3%A9%20moyens%20Dans%20les%20petites%20populations%20d%E2%80%99animaux%20domestiques.%20G%C3%A9n%C3%A9tique%20S%C3%A9lection%20%C3%89volution.%2017,%20459%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Caballero, A., Bravo, I. &amp; Wang, J. Inbreeding load and purging: implications for the short-term survival and the conservation management of small populations. <em>Heredity</em><strong>118</strong>, 177–185. 10.1038/hdy.2016.80 (2017).
</cite> [<a href="https://doi.org/10.1038/hdy.2016.80" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5234482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27624114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caballero,%20A.,%20Bravo,%20I.%20&amp;%20Wang,%20J.%20Inbreeding%20load%20and%20purging:%20implications%20for%20the%20short-term%20survival%20and%20the%20conservation%20management%20of%20small%20populations.%20Heredity118,%20177%E2%80%93185.%2010.1038/hdy.2016.80%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Grundy, B., Villanueva, B. &amp; Woolliams, J. A. Dynamic selection for maximizing response with constrained inbreeding in schemes with overlapping generations. <em>Anim. Sci.</em><strong>70</strong>, 373–382. 10.1017/S1357729800051717 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Grundy,%20B.,%20Villanueva,%20B.%20&amp;%20Woolliams,%20J.%20A.%20Dynamic%20selection%20for%20maximizing%20response%20with%20constrained%20inbreeding%20in%20schemes%20with%20overlapping%20generations.%20Anim.%20Sci.70,%20373%E2%80%93382.%2010.1017/S1357729800051717%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Gourdine, J. L., Sørensen, A. C. &amp; Rydhmer, L. There is room for selection in a small local pig breed when using optimum contribution selection: a simulation study. <em>J. Anim. Sci.</em><strong>90</strong>, 76–84. 10.2527/jas.2011-3898 (2012).
</cite> [<a href="https://doi.org/10.2527/jas.2011-3898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21841085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gourdine,%20J.%20L.,%20S%C3%B8rensen,%20A.%20C.%20&amp;%20Rydhmer,%20L.%20There%20is%20room%20for%20selection%20in%20a%20small%20local%20pig%20breed%20when%20using%20optimum%20contribution%20selection:%20a%20simulation%20study.%20J.%20Anim.%20Sci.90,%2076%E2%80%9384.%2010.2527/jas.2011-3898%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Percie du Sert. The ARRIVE guidelines 2.0: updated guidelines for reporting animal research. <em>PLoS Biol.</em><strong>18</strong>, e3000410 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3000410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7360023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32663219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Percie%20du%20Sert.%20The%20ARRIVE%20guidelines%202.0:%20updated%20guidelines%20for%20reporting%20animal%20research.%20PLoS%20Biol.18,%20e3000410%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Ndiaye, S. Underpinnings and dynamics of the popular economy: A reading from Senegal. <em>RECMA</em> 52–65 (2021).</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Gourdine, J. L. et al. Characterizing practices and the sanitary status of farms of Creole and Creole-like pigs in Martinique. <em>Rev. D’élevage Médecine Vét Pays Trop.</em><strong>71</strong>, 33–39. 10.19182/remvt.31346 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gourdine,%20J.%20L.%20et%20al.%20Characterizing%20practices%20and%20the%20sanitary%20status%20of%20farms%20of%20Creole%20and%20Creole-like%20pigs%20in%20Martinique.%20Rev.%20D%E2%80%99%C3%A9levage%20M%C3%A9decine%20V%C3%A9t%20Pays%20Trop.71,%2033%E2%80%9339.%2010.19182/remvt.31346%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Rosé, R. et al. Interactions between sire family and production environment (temperate vs. tropical) on performance and thermoregulation responses in growing pigs. <em>J. Anim. Sci.</em><strong>95</strong>, 4738–4751. 10.2527/jas2017.1611 (2017).
</cite> [<a href="https://doi.org/10.2527/jas2017.1611" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6292322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29293699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ros%C3%A9,%20R.%20et%20al.%20Interactions%20between%20sire%20family%20and%20production%20environment%20(temperate%20vs.%20tropical)%20on%20performance%20and%20thermoregulation%20responses%20in%20growing%20pigs.%20J.%20Anim.%20Sci.95,%204738%E2%80%934751.%2010.2527/jas2017.1611%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. <em>Am. J. Hum. Genet.</em><strong>81</strong>, 559–575. 10.1086/519795 (2007).
</cite> [<a href="https://doi.org/10.1086/519795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Purcell,%20S.%20et%20al.%20PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.%20Am.%20J.%20Hum.%20Genet.81,%20559%E2%80%93575.%2010.1086/519795%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Kozak, M. &amp; Piepho, H. P. What’s normal anyway? Residual plots are more telling than significance tests when checking ANOVA assumptions. <em>J. Agron. Crop Sci.</em><strong>204</strong>, 86–98. 10.1111/jac.12220 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kozak,%20M.%20&amp;%20Piepho,%20H.%20P.%20What%E2%80%99s%20normal%20anyway?%20Residual%20plots%20are%20more%20telling%20than%20significance%20tests%20when%20checking%20ANOVA%20assumptions.%20J.%20Agron.%20Crop%20Sci.204,%2086%E2%80%9398.%2010.1111/jac.12220%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>MacFarland, T. W., Yates, J. M. &amp; Kruskal–Wallis, H. -Test for oneway analysis of variance (ANOVA) by ranks. In <em>Introduction to Nonparametric Statistics for the Biological Sciences Using R</em> (eds MacFarland, T. W. &amp; Yates, J. M.) 177–211 10.1007/978-3-319-30634-6_6 (Springer International Publishing, 2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?MacFarland,%20T.%20W.,%20Yates,%20J.%20M.%20&amp;%20Kruskal%E2%80%93Wallis,%20H.%20-Test%20for%20oneway%20analysis%20of%20variance%20(ANOVA)%20by%20ranks.%20In%20Introduction%20to%20Nonparametric%20Statistics%20for%20the%20Biological%20Sciences%20Using%20R%20(eds%20MacFarland,%20T.%20W.%20&amp;%20Yates,%20J.%20M.)%20177%E2%80%93211%2010.1007/978-3-319-30634-6_6%20(Springer%20International%20Publishing,%202016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>R Core Team. R: The R Project for Statistical Computing. <a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r-project.org/</a> (2025).</cite>
</li>
<li id="CR44">
<span class="label">44.</span><cite>Ogle, D., Doll, J., Wheeler, A. &amp; Dinno, A. F. S. A. Simple Fisheries Stock Assessment Methods. R package version 0.9.6, <a href="https://fishr-core-team.github.io/FSA/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://fishr-core-team.github.io/FSA/</a> (Accessed 4 Sep 25) (2025).</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Gibson, J. et al. Options and Strategies for the Conservation of Farm Animal Genetic Resources: Report of an International Workshop, AGROPOLIS, Montpellier, France, 7–10 November 2005. (2006).</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Schiavo, G. et al. Runs of homozygosity provide a genome landscape picture of inbreeding and genetic history of European autochthonous and commercial pig breeds. <strong>52</strong>, 155–170 10.1111/age.13045 (2021).</cite> [<a href="https://doi.org/10.1111/age.13045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33544919/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Broman, K. W. &amp; Weber, J. L. Long homozygous chromosomal segments in reference families from the centre d’étude du polymorphisme Humain. <em>Am. J. Hum. Genet.</em><strong>65</strong>, 1493–1500. 10.1086/302661 (1999).
</cite> [<a href="https://doi.org/10.1086/302661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1288359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10577902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Broman,%20K.%20W.%20&amp;%20Weber,%20J.%20L.%20Long%20homozygous%20chromosomal%20segments%20in%20reference%20families%20from%20the%20centre%20d%E2%80%99%C3%A9tude%20du%20polymorphisme%20Humain.%20Am.%20J.%20Hum.%20Genet.65,%201493%E2%80%931500.%2010.1086/302661%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Ceballos, F. C., Hazelhurst, S. &amp; Ramsay, M. Assessing runs of homozygosity: a comparison of SNP array and whole genome sequence low coverage data. <em>BMC Genom.</em><strong>19</strong>, 106. 10.1186/s12864-018-4489-0 (2018).</cite> [<a href="https://doi.org/10.1186/s12864-018-4489-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5789638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29378520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ceballos,%20F.%20C.,%20Hazelhurst,%20S.%20&amp;%20Ramsay,%20M.%20Assessing%20runs%20of%20homozygosity:%20a%20comparison%20of%20SNP%20array%20and%20whole%20genome%20sequence%20low%20coverage%20data.%20BMC%20Genom.19,%20106.%2010.1186/s12864-018-4489-0%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Manunza, A. et al. Romanian wild boars and Mangalitza pigs have a European ancestry and harbour genetic signatures compatible with past population bottlenecks. <em>Sci. Rep.</em><strong>6</strong>, 29913 (2016).
</cite> [<a href="https://doi.org/10.1038/srep29913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4945946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27418428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manunza,%20A.%20et%20al.%20Romanian%20wild%20boars%20and%20Mangalitza%20pigs%20have%20a%20European%20ancestry%20and%20harbour%20genetic%20signatures%20compatible%20with%20past%20population%20bottlenecks.%20Sci.%20Rep.6,%2029913%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Miller, R. G. <em>Simultaneous Statistical Inference</em>. 10.1007/978-1-4613-8122-8. (Springer New York, 1981).</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Tukey, J. W. Comparing individual means in the analysis of variance. <em>Biometrics</em><strong>5</strong>, 99–114. 10.2307/3001913 (1949).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/18151955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tukey,%20J.%20W.%20Comparing%20individual%20means%20in%20the%20analysis%20of%20variance.%20Biometrics5,%2099%E2%80%93114.%2010.2307/3001913%20(1949)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Alexander, D. H., Novembre, J. &amp; Lange, K. Fast model-based Estimation of ancestry in unrelated individuals. <em>Genome Res.</em><strong>19</strong>, 1655–1664. 10.1101/gr.094052.109 (2009).
</cite> [<a href="https://doi.org/10.1101/gr.094052.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alexander,%20D.%20H.,%20Novembre,%20J.%20&amp;%20Lange,%20K.%20Fast%20model-based%20Estimation%20of%20ancestry%20in%20unrelated%20individuals.%20Genome%20Res.19,%201655%E2%80%931664.%2010.1101/gr.094052.109%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Sequence data that support the findings of this study have been deposited and are openly available on Research.Data.gouv with the following DOI: https://doi.org/10.57745/HWCCD5 . We also used the published data from Burgos-Paz et al. 2013 available at Dryad: https://doi.org/10.5061/dryad.t1r3d .</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-15649-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_15649.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12361484/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12361484/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12361484%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361484/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12361484/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12361484/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40825833/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12361484/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40825833/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12361484/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12361484/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MLFnUm9YXntQaVLcgSJaW9WvM81jodXcwERbrg3yL4O2YQcHtpu4GlS95IZ0j6ux">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
