
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Revolutionizing cancer treatment with Halomonas Aquamarina L-Glutaminase: insights from in vitro and computational studies - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5312FD8AEF2EE30512FD002FA54F00.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375786/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Revolutionizing cancer treatment with Halomonas Aquamarina L-Glutaminase: insights from in vitro and computational studies">
<meta name="citation_author" content="Sara Abdelsayed">
<meta name="citation_author_institution" content="Botany and Microbiology Department, Faculty of Science, Benha University, Benha, Egypt">
<meta name="citation_author" content="Alaa Elmetwalli">
<meta name="citation_author_institution" content="Department of Clinical Trial Research Unit and Drug Discovery, Egyptian Liver Research Institute and Hospital (ELRIAH), Mansoura, Egypt">
<meta name="citation_author_institution" content="Prince Fahad bin Sultan Chair for Biomedical Research, University of Tabuk, Tabuk, Saudi Arabia">
<meta name="citation_author" content="Jihan Hassan">
<meta name="citation_author_institution" content="Department of Applied Medical Chemistry, Medical Research Institute, Alexandria University, Alexandria, Egypt">
<meta name="citation_author" content="Mohamed O Abdel Monem">
<meta name="citation_author_institution" content="Botany and Microbiology Department, Faculty of Science, Benha University, Benha, Egypt">
<meta name="citation_author" content="Ali H El-Far">
<meta name="citation_author_institution" content="Key Laboratory of Epigenetics and Oncology, The Research Center for Preclinical Medicine, Southwest Medical University, Luzhou, 646000 China">
<meta name="citation_author" content="Fuad Ameen">
<meta name="citation_author_institution" content="Department of Botany and Microbiology, College of Science, King Saud University, Riyadh 11451, P.O. Box 2455, Saudi Arabia">
<meta name="citation_author" content="Mervat G Hassan">
<meta name="citation_author_institution" content="Botany and Microbiology Department, Faculty of Science, Benha University, Benha, Egypt">
<meta name="citation_publication_date" content="2025 Aug 24">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="31086">
<meta name="citation_doi" content="10.1038/s41598-025-14230-6">
<meta name="citation_pmid" content="40850948">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375786/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375786/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375786/pdf/41598_2025_Article_14230.pdf">
<meta name="description" content="Bacterial L-glutaminase (L-GLS) has emerged as a potential therapeutic target in cancer treatment by disrupting glutamine-dependent metabolic pathways in tumor cells. This study focused on isolating and characterizing L-GLS-producing marine bacteria ...">
<meta name="og:title" content="Revolutionizing cancer treatment with Halomonas Aquamarina L-Glutaminase: insights from in vitro and computational studies">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Bacterial L-glutaminase (L-GLS) has emerged as a potential therapeutic target in cancer treatment by disrupting glutamine-dependent metabolic pathways in tumor cells. This study focused on isolating and characterizing L-GLS-producing marine bacteria ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375786/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375786">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-14230-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_14230.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375786%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375786/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375786/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375786/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 24;15:31086. doi: <a href="https://doi.org/10.1038/s41598-025-14230-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-14230-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Revolutionizing cancer treatment with <em>Halomonas Aquamarina</em> L-Glutaminase: insights from in vitro and computational studies</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdelsayed%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sara Abdelsayed</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sara Abdelsayed</span></h3>
<div class="p">
<sup>1</sup>Botany and Microbiology Department, Faculty of Science, Benha University, Benha, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdelsayed%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sara Abdelsayed</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elmetwalli%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Alaa Elmetwalli</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Alaa Elmetwalli</span></h3>
<div class="p">
<sup>2</sup>Department of Clinical Trial Research Unit and Drug Discovery, Egyptian Liver Research Institute and Hospital (ELRIAH), Mansoura, Egypt </div>
<div class="p">
<sup>3</sup>Prince Fahad bin Sultan Chair for Biomedical Research, University of Tabuk, Tabuk, Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elmetwalli%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alaa Elmetwalli</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hassan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jihan Hassan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jihan Hassan</span></h3>
<div class="p">
<sup>4</sup>Department of Applied Medical Chemistry, Medical Research Institute, Alexandria University, Alexandria, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hassan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jihan Hassan</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monem%20MOA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mohamed O Abdel Monem</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mohamed O Abdel Monem</span></h3>
<div class="p">
<sup>1</sup>Botany and Microbiology Department, Faculty of Science, Benha University, Benha, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Monem%20MOA%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohamed O Abdel Monem</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22El-Far%20AH%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ali H El-Far</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ali H El-Far</span></h3>
<div class="p">
<sup>5</sup>Key Laboratory of Epigenetics and Oncology, The Research Center for Preclinical Medicine, Southwest Medical University, Luzhou, 646000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22El-Far%20AH%22%5BAuthor%5D" class="usa-link"><span class="name western">Ali H El-Far</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ameen%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Fuad Ameen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Fuad Ameen</span></h3>
<div class="p">
<sup>6</sup>Department of Botany and Microbiology, College of Science, King Saud University, Riyadh 11451, P.O. Box 2455, Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ameen%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fuad Ameen</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hassan%20MG%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Mervat G Hassan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Mervat G Hassan</span></h3>
<div class="p">
<sup>1</sup>Botany and Microbiology Department, Faculty of Science, Benha University, Benha, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hassan%20MG%22%5BAuthor%5D" class="usa-link"><span class="name western">Mervat G Hassan</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Botany and Microbiology Department, Faculty of Science, Benha University, Benha, Egypt </div>
<div id="Aff2">
<sup>2</sup>Department of Clinical Trial Research Unit and Drug Discovery, Egyptian Liver Research Institute and Hospital (ELRIAH), Mansoura, Egypt </div>
<div id="Aff3">
<sup>3</sup>Prince Fahad bin Sultan Chair for Biomedical Research, University of Tabuk, Tabuk, Saudi Arabia </div>
<div id="Aff4">
<sup>4</sup>Department of Applied Medical Chemistry, Medical Research Institute, Alexandria University, Alexandria, Egypt </div>
<div id="Aff5">
<sup>5</sup>Key Laboratory of Epigenetics and Oncology, The Research Center for Preclinical Medicine, Southwest Medical University, Luzhou, 646000 China </div>
<div id="Aff6">
<sup>6</sup>Department of Botany and Microbiology, College of Science, King Saud University, Riyadh 11451, P.O. Box 2455, Saudi Arabia </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 29; Accepted 2025 Jul 30; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375786  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40850948/" class="usa-link">40850948</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Bacterial L-glutaminase (L-GLS) has emerged as a potential therapeutic target in cancer treatment by disrupting glutamine-dependent metabolic pathways in tumor cells. This study focused on isolating and characterizing L-GLS-producing marine bacteria from Mediterranean seawater for preliminary therapeutic evaluation. <em>Halomonas aquamarina</em> HBIM1 was identified as the most efficient isolate through comprehensive phenotypic, genotypic, and enzymatic screening. The enzyme was successfully purified, achieving a specific activity of 748.35 U/mg with 3.39-fold purification. SDS-PAGE analysis confirmed high purity with a single 66 kDa protein band. Kinetic characterization revealed optimal activity at pH 8 and 50 °C, with strong substrate affinity (Km = 0.198 mM⁻¹). Preliminary in vitro cytotoxicity screening demonstrated selective antiproliferative effects on HepG2 liver cancer cells (IC50 = 33.98 µg/ml) compared to normal WI-38 cells (IC50 = 93.43 µg/ml), yielding a 2.75-fold selectivity index. Molecular docking analysis identified tannic acid and 6-diazo-5-oxo-L-norleucine as selective inhibitors of bacterial L-GLS, with tannic acid showing the highest binding affinity (-12.25 kcal/mol) and 5-fold selectivity over human L-GLS, suggesting potential for combination therapy strategies. These proof-of-concept findings indicate the preliminary anticancer potential of <em>Halomonas</em>-derived L-GLS and computational support for selective inhibitor development. However, comprehensive preclinical validation, including in vivo efficacy studies, toxicological evaluation, and pharmacological profiling, is essential to establish therapeutic viability and safety before clinical consideration.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Enzyme activity, <em>Halomonas Aquamarina</em>, L-glutaminase, Liver cancer, Tannic acid</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biological techniques, Biomaterials, Environmental biotechnology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title"> Introduction</h2>
<p id="Par6">Liver cancer poses a significant global health challenge, with its incidence rates steadily rising and limited treatment options available, thus emphasizing the critical need for innovative therapeutic approaches<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Recent research endeavors have increasingly focused on harnessing the potential of natural compounds and biologically derived agents for combating cancer<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Among these, L-glutaminase (L-GLS), an enzyme catalyzing the hydrolysis of L-glutamine to L-glutamate and ammonia, has garnered attention due to its capacity to target cancer cells exhibiting high glutamine dependency.</p>
<p id="Par7">The structure of bacterial L-GLS consists of a single polypeptide chain folded into a compact globular shape. This enzyme is composed of approximately 300–400 amino acid residues, with a molecular weight ranging from 30 to 60 kDa<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The unique properties and catalytic activity of bacterial L-GLS have led to its extensive applications in different fields. Cleaving glutamine residues from fusion proteins facilitates the release of the target protein of interest. This enzymatic action is crucial for the downstream purification and isolation of the desired protein<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par8">It is worth noting that the bacterial L-GLS is not limited to a single species but has been discovered in several bacterial strains, including <em>Bacillus subtilis</em>, <em>Pseudomonas aeruginosa</em>, and <em>Escherichia coli</em><sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Industrial processes often require enzymes to function optimally at elevated temperatures. Unfortunately, many plant or animal-derived enzymes cannot withstand these extreme conditions<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. However, enzymes obtained from bacterial microorganisms have shown remarkable stability, even at temperatures exceeding 70 °C. This heat resistance is attributed to these bacterial enzymes’ unique structure and composition, which allows them to maintain their catalytic activity under harsh industrial conditions<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par9">Apart from their stability, enzymes derived from bacterial microorganisms exhibit significantly higher activity levels at industrial temperatures than enzymes from other sources<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. These enzymes have evolved to function optimally in the natural habitats of bacteria, which often include hot environments such as geothermal springs and hydrothermal vents<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. As a result, they have developed a high tolerance for extreme temperatures, making them well-suited for industrial applications that require efficient catalysis at elevated temperatures<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par10">Therefore, the rationale for investigating L-GLS stems from its ability to disrupt cancer cell metabolism, as many cancer types, including liver cancer, heavily rely on glutamine for sustenance and proliferation<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. By targeting this metabolic vulnerability, L-GLS holds promise as a specific and compelling anticancer agent, potentially offering a therapeutic avenue with reduced off-target effects compared to conventional treatments<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. It was known that microbial enzyme biotechnology has become a valuable tool in many fields, including medicine<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. One enzyme, L-GLS, is particularly promising for cancer treatment because it breaks down glutamine, a nutrient many cancer cells depend on for growth. By targeting glutamine metabolism, L-GLS offers potential as a therapeutic option for treating glutamine-dependent tumors<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par11">Recent studies have shown that the inhibition of L-GLS has been found to induce apoptosis, a programmed cell death mechanism that plays a crucial role in regulating cell growth and eliminating damaged or abnormal cells<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The induction of apoptosis in cancer cells is a highly desirable outcome, as it helps to eliminate the malignant cells and prevent disease progression<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The selective targeting of L-GLS has shown great potential in selectively killing glutamine-dependent tumor cells while sparing normal cells with alternative energy sources<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. This targeted approach minimizes potential side effects and enhances the efficacy of anticancer therapies<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>.</p>
<p id="Par12">In recent years, in-silico techniques have advanced significantly, helping scientists design, optimize, and analyze enzymes more effectively. Computational methods, such as molecular docking, allow researchers to study how L-GLS interacts with potential cancer therapies and how it can be optimized for better performance<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. These tools also help predict enzyme behavior under different conditions, which is vital for therapeutic and industrial applications<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Computational methods were extensively used to determine the affinity and molecular interactions between target proteins and ligands, which could be synthetic or natural compounds<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Therefore, the current study investigated the binding affinity of 6-diazo-5-oxo-L-norleucine (DON) and tannic acid toward <em>Halomonas aquamarina</em> and human L-GLS for future therapeutic applications.</p>
<p id="Par13">Therefore, this study aimed to isolate and identify novel marine bacteria capable of producing L-GLS, focusing on optimizing enzyme production and evaluating its anticancer properties. We characterized the most potent isolate, <em>Halomonas aquamarina</em> HBIM1, through molecular identification and biochemical assays. The enzyme was purified, and its kinetic behavior and substrate specificity were determined. Cytotoxicity against liver cancer cell lines was assessed, and molecular docking analyses were performed to explore potential inhibitory ligands. By integrating experimental and in silico approaches, our goal was to establish the therapeutic potential of marine-derived L-GLS for targeted liver cancer treatment.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Seawater samples collection</h3>
<p id="Par14">In this study, 15 different marine water samples were collected from various sites on the Mediterranean Sea coast, near Alexandria Aquarium center, Alexandria Governorate, Alexandria, Egypt, spotted at 31°12’44.9 “N (Latitude) and 29°53’06.4"E (Longitude) (Fig. <a href="#Fig1" class="usa-link">1</a>). The sterile plastic containers were used to store and transport the samples to the microbiology laboratory. Following the sample collection, marine water samples were stored at 4 °C and transported directly to the lab to be analyzed further<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/1ef66e20893b/41598_2025_14230_Fig1_HTML.jpg" loading="lazy" id="d33e414" height="355" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Map of sample collection site (revealed by a red node) at Mediterranean Seawaters, Alexandria, Egypt.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Isolation and enrichment of L-GLS-producing strains</h3>
<p id="Par16">Multiple times, marine water samples were diluted in a clean saline solution. Then, they were spread out on a modified production medium with 10 g of L-glutamine and 2 g of D-glucose (Fluka Biochemicals, Buchs, Switzerland). The medium was mixed with seawater with 0.09 mL of phenol red (Sigma-Aldrich, St. Louis, MO), added as a pH indicator, and set to 8.0 ± 0.2. For 24 h, all the culture plates were kept at 34 °C, and the isolates that made a pink zone around their colonies were chosen as possible L-GLS producers. The selected bacterial isolates were then cleaned using the changed agar plates as a production medium, and their ability to make L-GLS was checked as previously described<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Identification of the most promising L-GLS producer</h3>
<p id="Par17">In 15 marine water samples, three isolates (encoded WS6, WS8, and WS12) revealed activity. Every 2–3 weeks, periodic subcultures of the bacteria were performed, followed by 24 h of growth at 37 °C in a static incubator. To obtain crude glutaminase from cell-free supernatant, crude glutaminase was obtained from mostly extracellular isolates. Using morphological characteristics under the microscope, the most active bacteria were identified according to<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. A species determination was then carried out on these isolates. The bacterial identities were also verified through culture-based methods. The isolate exhibiting the highest levels of extracellular L-GLS was further investigated. The 16 S rDNA of bacterial strains was sequenced using genomic DNA isolation and PCR amplification. Using CLUSTAL W, multiple-sequence alignment software, 16 S rDNA sequences were aligned with GenBank (<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). MEGA 5.1 used a neighbor-joining algorithm to identify the isolate’s taxonomy based on its 16 S rDNA sequence homology. Jukes-Cantor distance estimation was also conducted using bootstraps and the Jukes-Cantor method<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">A qualitative analysis of L-GLS activity</h3>
<p id="Par18">Abdallah et al.<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> described a method to measure the activity of L-GLS using L-glutamine as a substrate and ammonia as a product. A mixture of 0.5 ml of the supernatant (crude enzyme) was added to 0.5 ml of 0.04 M L-glutamine solution in the presence of 0.5 ml of distilled water and 0.5 ml of phosphate buffer (0.1 M, pH 8.0). After 30 min, 1.5 M trichloroacetic acid and distilled water were added to arrest the reaction. The mixture was mixed with distilled water and Nessler’s reagent, adding 3.7 ml each. In the visible spectra, the absorbance was measured at 450 nm (Chrom Tech CT-2200 UV/Vis). The spectrophotometer was used to calculate the amount of light absorbed by the mixture, which indicates the amount of ammonia present. The graph was then used to compare the absorbance of the mixture to the amount of ammonia released by a known amount of ammonium chloride. This allowed for an accurate determination of the enzyme activity.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Optimal culture conditions for exocellular L-GLS production</h3>
<p id="Par19">Based on various physicochemical factors and nutritional requirements, extracellular L-GLS production in the isolate was consistently evaluated. The incubation period (12–96 h), pH (5.0–8.0), and temperature (25–50 °C) were among the factors we scaled up over time to affect L-GLS production one by one. Besides that, various carbon and nitrogen sources were used to optimize L-GLS production. Three experiments were conducted, and the results are presented as averages and standard deviations. We dripped 50 ml of nutrient broth into 250 ml Erlenmeyer flasks containing 8% (v/v) of an overnight seed culture containing 0.2% yeast extract, 0.1% beef extract, 0.2% peptone, 0.5% sodium chloride, 1.0% glucose, and 0.05% hydrogen peroxide in nutrient broth. After incubation at 37 °C under vigorous shaking (100 rpm) for 48 h, cell-free broth was centrifuged at 10,000 x g for 10 min at 4 °C and assessed for extracellular L-GLS production<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. The optimal conditions for further exploration of L-GLS have been identified, and an assay method has been adopted to measure the molecule’s activity.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Protein determination</h3>
<p id="Par20">To determine the concentration of enzymes in the stock solution, we prepared an aqueous solution of bovine serum albumin using the<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> method to prepare a stock solution of 1000 mg/mL of standard protein. The mixture was combined with the Folin-Ciocalteu reagent and allowed to react for 30 min, after which the absorbance was measured at 660 nm using the same reagent.</p></section><section id="Sec9"><h3 class="pmc_sec_title">L-GLS purification</h3>
<p id="Par21">A crude enzyme extract of 500 mL was used for the purification. Until 40% saturation is reached, finely powdered ammonium sulfate is slowly added to the cell-free supernatant (crude enzyme). The entire batch of liquids was stirred at 4 °C with a magnetic stirrer. Centrifugation at 10,000 g at 4 °C for 20 min removed the precipitated crude enzyme. The supernatant was diluted with fresh ammonium sulfate to achieve 50% saturation. Having obtained the solid solution (precipitate), it was thoroughly dissolved in a minimal volume of 0.01 M phosphate buffer (pH 8). In the same manner as previously described, the precipitated protein was removed by centrifugation. After adding fresh ammonium sulfate to the cell-free supernatant, the concentration was increased to 80%. By centrifugation, the dehydrated enzyme was recovered and resuspended in 0.01 M phosphate buffer (pH 8), and the precipitated protein was resuspended in 0.01 M phosphate buffer (pH 8). On completion of the enzyme precipitation in ammonium sulfate, the enzyme precipitate was dialyzed against 0.01 M phosphate buffer (pH 8) for 24 h at 4 °C with continuous stirring, and the buffers were changed intermittently during this process. A dialyzed fraction was collected, lyophilized, and freeze-dried to obtain the lyophilized enzyme, which was used for the subsequent tests<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Electrophoresis on polyacrylamide gels (PAGE)</h3>
<p id="Par22">Whole-cell lysates were produced by resuspending cells in 75 mL of lysis buffer containing 1% bromophenol blue as a tracking dye, 1 M Tris/HCl pH 6.4, 10% sodium dodecyl sulfate (SDS), 5% glycerol, and 10% SDS. A 4:1 (v/v) ratio of 5x loading buffer was added to the protein sample before heating it at 100 °C for 2–5 min before separation by SDS-PAGE. Wells were drilled into the gel, and samples were placed on top. At 8 °C, electrophoresis was carried out at 100 V. The gel was stained with Coomassie Brilliant Blue R-250 after removal from the glass plates. The bands of proteins were visualized through an extensive destaining process. A gel documentation system was used to capture and analyze destained gel images (Alpha InfoTech Corporation, USA) as described by<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Kinetic properties of L-GLS</h3>
<p id="Par23">We determined how temperature affects enzyme activity by adjusting the reaction mixture’s temperature from 40 to 100 °C. A pH range of 6 to 12 was used to determine the optimum pH for enzyme activity in the reaction mixture in buffer solutions. An enzyme activity test was conducted at 70 °C to see how pH affects enzyme activity. Buffer solutions were selected according to their pKa values and effective buffering range. As the pH value of the buffers was adjusted, phosphate buffers were used at pH values of 6, 6.5, 7, and 7.5, followed by Tris-HCl buffers at pH values of 8, 8.5, 9, 9.5, and 12. L-GLS kinetic parameters were estimated using substrate concentrations (<em>Km</em> and <em>Vmax</em>)<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The Michaelis-Menten parameters determined from the Lineweaver-Burk plots and the equation derived from the linear regression of the curve were used to calculate the substrate concentration needed to reach <em>Vmax</em>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par24">To determine their characteristics, the DMEM medium (BioWhittaker®, USA) was used to culture HepG2 (liver cancer cell line) and WI-38 normal cell lines (Medical Research Institute, Alexandria University, Egypt). They were grown in Dulbecco’s modified Eagle’s medium (Nuair, Germany) supplemented with 10% fetal bovine serum (Sigma, USA) and 100 g/ml penicillin, 100 g/ml streptomycin, and 10% fetal bovine serum (Sigma, USA) in a humidified incubator containing 100 g/ml penicillin, 100 g/ml streptomycin, and 10% fetal bovine serum (Sigma, USA) and 10% fetal bovine serum (Sigma, USA)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. This study consisted of three independent experiments that were carried out in parallel.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Cell viability assay</h3>
<p id="Par25">An MTT colorimetric assay kit from Sigma-Aldrich (USA) was used to analyze the viability of HepG2 and WI-38 cells after they had been treated with L-GLS. Cells were seeded in 96-well plates and cultured for 24 and 48 h at 37 °C before being treated with various concentrations of L-GLS for 24 and 48 h, respectively. In each well, 100 ml of the MTT working solution was incubated for 4 h in the dark at 37 °C. 100 mL of DMSO was added to each well after the purple formazan crystals were dissolved in DMSO for five minutes. Microplate readers (BIO-RAD PR4100, USA) were used to determine the optical density (OD) at 570 nm. Based on a comparison of treated cells with untreated, 100% viable control cells, the IC50 was calculated<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">In silico assessment</h3>
<p id="Par26">To assess the molecular interactions and affinities of L-GLS of both bacterial and human origin with DON and tannic acid, we obtained the three-dimensional structures of bacterial and human L-GLS from the RCSB Protein Data Bank (<a href="https://www.rcsb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/</a>) databases. Subsequently, these structures underwent preparation steps, which included removing water molecules and attaching ligands using the UCSF Chimera software package<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. In addition, the three-dimensional structures of DON and tannic acid were retrieved from the PubChem (<a href="https://pubchem.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubchem.ncbi.nlm.nih.gov/</a>) database. Before docking, the ligands were prepared by minimizing their energy using the molecular operating environment (MOE 2015.10) software<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. By the same software, the protein-ligand interactions were recognized and visualized.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par27">The data were analyzed using GraphPad Prism version 9.0.5 (GraphPad Software, Inc., La Jolla, CA, USA). Sigmoid-type nonlinear regression was applied to determine the IC50 values. A one-way ANOVA was employed to assess the significance of the differences. The significance level for this study was established at 0.05. The data related to this study are expressed as mean ± standard deviation (SD)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Triple testing was used for cytotoxicity assays.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results</h2>
<p id="Par28">The most efficient microbial isolates were selected using qualitative and quantitative methods. Although the methodologies applied in this study—such as enzyme production optimization, purification, and kinetic characterization—are widely used in enzyme research, the novelty of our work lies in the isolation and characterization of <em>Halomonas aquamarina</em> HBIM1 as a potent producer of L-GLS. To our knowledge, this is the first report documenting this strain’s high L-GLS activity, favorable biochemical characteristics, and selective anticancer potential. In addition, we integrated conventional experimental work with molecular docking studies to assess potential inhibitors, further strengthening the enzyme’s therapeutic scope. The following results reflect standard characterization and novel insights specific to this unique marine isolate.</p>
<section id="Sec17"><h3 class="pmc_sec_title">The most effective bacteria for producing L-GLS</h3>
<p id="Par29">There was also a range of enzyme activity in the broth medium for 15 bacterial isolates, ranging from 9.82 to 39.82 U/ml for L-GLS. In addition to revealing the bacteria’s ability to produce enzymes, enzyme activities in the broth medium indicate the bacteria’s efficiency. Among the 15 tested bacterial isolates, three isolates (WS6, WS8, and WS12) were significantly different from the others (Table <a href="#Tab1" class="usa-link">1</a> and Fig. <a href="#Fig2" class="usa-link">2</a>). In addition, isolates WS6, WS12, and WS8 gave the highest significant (<em>p</em> &lt; 0.05) enzyme activity, respectively, of 39.82, 35.32, and 30.52 U/ml. The high enzyme activities observed in isolates WS6, WS12, and WS8 indicate their superior microbial efficiency in producing enzymes. This suggests that these bacterial isolates have an enhanced capacity to break down organic compounds and contribute to biodegradation processes. Their ability to produce enzymes at such high levels makes them promising candidates for applications in the biomedical industry.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>L. GLS activity on agar and broth medium supplemented with L. glutamine by the bacterial isolates at 37 °C for 7 days.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolates<br>Code</th>
<th align="left" colspan="1" rowspan="1">Pink colour formation<br>(agar medium)</th>
<th align="left" colspan="1" rowspan="1">Enzyme activity (U/ml)<br>(broth medium)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">WS1</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">9.82</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS2</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">13.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS3</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">11.63</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS4</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">19.33</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS5</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">18.59</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS6</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">39.82</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS7</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">15.10</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS8</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">30.52</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS9</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">17.17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS10</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">20.30</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS11</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">23.54</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS12</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">35.32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS13</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">8.40</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS14</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">25.93</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WS15</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">0.00</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/c4a34bb56248/41598_2025_14230_Fig2_HTML.jpg" loading="lazy" id="d33e703" height="325" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Biological sources of L-GLS. L-GLS enzyme activity ranged from 9.82 to 39.82 U/ml for 15 bacterial isolates in broth medium.</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Isolation and description of L-GLS-producing strains</h3>
<p id="Par32">Following three weeks of enrichment and one week of strain isolation, we isolated 15 isolates on LB agar plates containing 100 mL of diluted enrichment culture after 24 h of growth. On a medium containing 15 isolates, the three isolates, WS6, WS12, and WS8, grew more than the others (Fig. <a href="#Fig3" class="usa-link">3</a><strong>)</strong>, and the efficiency had a more significant impact on isolate WS6 than the other isolates by 90%. The exceptional isolate was designated the L-GLS-producing strain <em>Halomonas aquamarina</em> HBIM1 and used in subsequent studies.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/704413b6f88f/41598_2025_14230_Fig3_HTML.jpg" loading="lazy" id="d33e725" height="694" width="788" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The isolation and description of the strains that produce L-GLS. The strain WS6 has a higher potential for pharmaceutical applications due to its faster growth on the medium. This could lead to increased production of L-GLS and potentially lower manufacturing costs.</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">Genotypic identification of the potent bacterial isolate</h3>
<p id="Par34">Based on qualitative and quantitative criteria, the most effective isolate was selected. The isolate WS6 was identified based on its phenotypic and genotypic characteristics. The 16 S rRNA gene sequences for the selected isolate were compared with those of the 16 S rRNA regions in GenBank using BLAST searches at the National Center for Biotechnology Information (NCBI). Our bacterium shares 99% sequence identity with <em>Halomonas aquamarina</em> HBIM1 compared to the NCBI genome database. The bootstrap consensus tree for <em>Halomonas aquamarina</em> HBIM1, which produces L-GLS, was constructed by aligning multiple sequences using the neighbor-joining method <strong>(</strong>Fig. <a href="#Fig4" class="usa-link">4</a><strong>)</strong>. Sequence information has been made available for this strain of bacteria as of <a href="https://www.ncbi.nlm.nih.gov/nuccore/OR707005.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OR707005.1</a> in the GeneBank.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/8670e94763d4/41598_2025_14230_Fig4_HTML.jpg" loading="lazy" id="d33e753" height="1182" width="687" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Characterization of culturable bacteria isolated from Mediterranean Seawaters, Alexandria, Egypt. The sequences of the 16 S rRNA gene showed the interrelationships among the isolated bacterial strains in a neighbor-joining phylogenetic tree. Parenthesis indicates the GenBank accession number of each strain. Nodes are listed with bootstrap values based on 500 replications. Each nucleotide position represents 0.05 substitutions.</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Effect of incubation time, temperature, and pH on L-GLS enzyme production</h3>
<p id="Par36">The investigation into L-GLS production by <em>Halomonas aquamarina</em> HBIM1 involved monitoring over various incubation periods. Results indicated a notable surge in L-GLS activity, peaking at 30 h of incubation, followed by a gradual decline to its nadir after 90 h (Fig. <a href="#Fig5" class="usa-link">5</a>A). Furthermore, the effect of different temperatures (ranging from 20 to 100 °C) on L-GLS production by <em>Halomonas aquamarina</em> HBIM1 was examined, revealing optimal activity at 40 °C, with a pronounced decrease observed at 80 °C (Fig. <a href="#Fig5" class="usa-link">5</a>B). Moreover, the pH of the production broth significantly influenced L-GLS production by <em>Halomonas aquamarina</em> HBIM1, with enhanced yields noted as pH increased up to 6. However, production diminished at higher pH levels, indicating an optimal pH of 6 for extracellular L-GLS production by <em>Halomonas aquamarina</em> HBIM1 (Fig. <a href="#Fig5" class="usa-link">5</a>C).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/dc055c34ce7c/41598_2025_14230_Fig5_HTML.jpg" loading="lazy" id="d33e805" height="166" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Production of L-GLS by <em>Halomonas aquamarina</em> HBIM1 under varying conditions. (<strong>A</strong>) The graph shows how L-GLS activity progressed over different incubation periods. During the first 30 h of incubation, L-GLS activity peaks, gradually declining to its lowest point after 90 h. (<strong>B</strong>) The graph shows how temperature affects L-GLS production. L-GLS production is susceptible to temperature changes, with optimal activity occurring at 40 °C and decreasing at 80 °C. (<strong>C</strong>) The graph illustrates the influence of pH on L-GLS production. <em>Halomonas aquamarina</em> HBIM1 produces extracellular L-GLS at an optimal pH of 6, as pH increases up to 6; nevertheless, higher pH levels diminish production.</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Effect of carbon and nitrogen sources on L-GLS enzyme production</h3>
<p id="Par38"><em>Halomonas aquamarina</em> HBIM1 isolates were tested using different carbon sources to produce L-GLS. It became apparent that the choice of carbon source significantly influenced L-GLS production, with fructose emerging as the most preferred carbon source, yielding 46.31 ± 0.12 U/ml. Conversely, mannitol and starch exhibited the lowest L-GLS production, reaching suboptimal values of 22.4 U/ml and 11.82, respectively <strong>(</strong>Fig. <a href="#Fig6" class="usa-link">6</a>A<strong>)</strong>. The high L-GLS production using fructose as the carbon source can be attributed to its easily metabolizable nature, providing a readily available energy source for <em>Halomonas aquamarina</em> HBIM1. Fructose is efficiently assimilated and utilized by the bacteria, resulting in the highest enzyme activity and subsequent L-GLS production.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/35ae9ebec963/41598_2025_14230_Fig6_HTML.jpg" loading="lazy" id="d33e843" height="526" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Influence of different carbon and nitrogen sources on L-GLS production by <em>Halomonas aquamarina</em> HBIM1. (<strong>A</strong>) This graph shows that various carbon sources have different impacts on L-GLS production. Among the carbon sources tested, fructose produces the highest levels of L-GLS, with 46.31 ± 0.12 U/ml. L-GLS production values for mannitol and starch are lower, suggesting suboptimal bacterial utilization. (<strong>B</strong>) The graph depicts the effect of different nitrogen sources on L-GLS production. With 78.25 U/ml of enzyme production, ammonium sulfate is the most efficient nitrogen source. The production of enzymes is also increased by urea and peptone, but to a lesser extent than with ammonium sulfate.</p></figcaption></figure><p id="Par40">Furthermore, various nitrogen sources were examined for their impact on L-GLS production, including ammonium sulfate, yeast extract, sodium nitrate, ammonium nitrate, malt extract, urea, and peptone. <em>Halomonas aquamarina</em> HBIM1 displayed a capacity to utilize ammonium sulfate as a nitrogen source, with a favorable response to increased ammonium concentration, resulting in higher enzyme production, reaching 78.25 U/ml. While urea and peptone did not elicit significant changes in enzyme production, these nitrogen sources notably boosted production to levels ranging between 67.89 and 65.78 U/ml <strong>(</strong>Fig. <a href="#Fig6" class="usa-link">6</a>B<strong>)</strong>. Overall, the easily metabolizable nature of fructose, as well as its efficient assimilation and utilization by <em>Halomonas aquamarina</em> HBIM1, contribute to its high enzyme activity and subsequent L-GLS production. Additionally, the capacity of the bacteria to utilize ammonium sulfate as a nitrogen source, along with the favorable response to increased ammonium concentration, further enhances enzyme production and contributes to the overall high levels of L-GLS production.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Purification of L-GLS from <em>Halomonas Aquamarina HBIM1</em>
</h3>
<p id="Par41">Table <a href="#Tab2" class="usa-link">2</a> shows ammonium sulfate precipitation, dialysate, and filtration chromatography as methods for purifying L-GLS from <em>Halomonas aquamarina</em> HBIM1. Enzyme-specific activity and purity are boosted during purification, while total protein, total activity, and yield are reduced. In 23.35 mg of crude enzyme, 5147 U of L-GLS activity and 220.42 U/mg of specific enzyme activity were present. As a result of the enzyme purification process, 57.43% of the enzyme with 2956 U of activity was obtained after purifying 748.35 mg of protein (Fig. <a href="#Fig7" class="usa-link">7</a>). In this case, it is evident that the purification process increased the catalytic efficiency of L-GLS derived from <em>Halomonas aquamarina</em> HBIM1.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Purification procedure of L-GLS from <em>Halomonas Aquamarina</em> strain HBIM1.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Purification steps</th>
<th align="left" colspan="1" rowspan="1">Total activity (U)</th>
<th align="left" colspan="1" rowspan="1">Total protein (mg)</th>
<th align="left" colspan="1" rowspan="1">Specific activity<br>(U/mg protein)</th>
<th align="left" colspan="1" rowspan="1">Purification fold</th>
<th align="left" colspan="1" rowspan="1">Yield (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Crude extract</td>
<td align="center" colspan="1" rowspan="1">5147</td>
<td align="center" colspan="1" rowspan="1">23.35</td>
<td align="center" colspan="1" rowspan="1">220.42</td>
<td align="center" colspan="1" rowspan="1">1.0</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> Precipitation</td>
<td align="center" colspan="1" rowspan="1">3398</td>
<td align="center" colspan="1" rowspan="1">13.56</td>
<td align="center" colspan="1" rowspan="1">250.58</td>
<td align="center" colspan="1" rowspan="1">1.13</td>
<td align="left" colspan="1" rowspan="1">66.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sephadex G-50</td>
<td align="center" colspan="1" rowspan="1">2956</td>
<td align="center" colspan="1" rowspan="1">3.95</td>
<td align="center" colspan="1" rowspan="1">748.35</td>
<td align="center" colspan="1" rowspan="1">3.39</td>
<td align="left" colspan="1" rowspan="1">57.43</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/f079b3c2aaa1/41598_2025_14230_Fig7_HTML.jpg" loading="lazy" id="d33e974" height="361" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The elution profile for the <em>Halomonas aquamarina</em> HBIM1 L-GLS ammonium sulfate fraction on the Sephacryl S-200 column was determined by a flow rate of 20 ml/h. Enzyme activity (RED) and absorbance at 280 nm (blue).</p></figcaption></figure></section><section id="Sec23"><h3 class="pmc_sec_title">SDS-PAGE and molecular weight Estimation</h3>
<p id="Par44">Sodium dodecylsulfate–polyacrylamide gel electrophoresis (SDS-PAGE) was used to assess the homogeneity and molecular weight of purified L-GLS. As opposed to standard molecular weight markers, the L-GLS preparations had a single distinct 66 kDa apparent molecular weight band (Fig. <a href="#Fig8" class="usa-link">8</a>). The presence of only one distinct protein band with a 66 kDa apparent molecular weight suggests that the purified L-GLS is highly pure and homogeneous. This is a promising result, as it indicates minimal contamination or degradation of the protein during the purification process.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/d285a5855f38/41598_2025_14230_Fig8_HTML.jpg" loading="lazy" id="d33e995" height="453" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SDS-PAGE and molecular weight of L-GLS of <em>Halomonas aquamarina</em> HBIM1.</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">Kinetic properties of the purified L-GLS</h3>
<p id="Par46">Enzyme activity rose steadily from pH 5 to 8 over a broad pH range (Fig. <a href="#Fig9" class="usa-link">9</a>A). Despite pH values below optimal, enzymatic activity could remain at pH 8.0. A pH stability of 8.0 (95% activity) was observed for L-GLS, which gradually decreased and reached a pH of 9.5 (Fig. <a href="#Fig9" class="usa-link">9</a>B). The significant pH value of 8 for enzymatic activity indicates that the enzyme functions optimally in slightly alkaline conditions. This suggests that the enzyme’s active site and catalytic mechanisms best suit this pH range, allowing for efficient substrate binding and chemical reactions.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/a96d8fe75bf8/41598_2025_14230_Fig9_HTML.jpg" loading="lazy" id="d33e1028" height="440" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Kinetic properties of the purified L-GLS activity under various conditions. (<strong>A</strong>) This graph illustrates the effect of pH on enzyme activity. Enzyme activity increases steadily from pH 5 to 8, indicating a broad pH range for optimal activity. Despite suboptimal pH values, enzymatic activity can remain at pH 8.0. (<strong>B</strong>) The graph shows the pH stability profile of L-GLS. A pH stability of 8.0 (95% activity) is observed for the enzyme, gradually decreasing to a pH of 9.5. This suggests that the enzyme functions optimally in slightly alkaline conditions, with its active site and catalytic mechanisms best suited for this pH range. (<strong>C</strong>) This graph depicts the thermal stability of L-GLS. Maximum stability (98%) is observed at 50 °C, with stability gradually decreasing below this temperature. The enzyme exhibits optimal stability at moderately high temperatures, facilitating efficient substrate binding and catalytic reactions. (<strong>D</strong>) The graph illustrates the influence of metal cations and inhibitors on enzyme activity. These findings provide insights into L-GLS’s functional properties.</p></figcaption></figure><p id="Par48">Maximum thermal stability of L-GLS is observed at 50 °C (98%), and this stability gradually decreases below this temperature <strong>(</strong>Fig. <a href="#Fig9" class="usa-link">9</a>C<strong>)</strong>. The thermal stability of L-GLS was found to be highest at 50 °C, with a retention of 98.2% activity. As the temperature decreased below this threshold, enzymatic activity gradually declined. This suggests that L-GLS is most efficient and stable at moderately high temperatures, allowing for optimal substrate binding and catalytic reactions to occur. Various metal cations and inhibitors assessed L-GLS activity at 5 and 10 mM. The only ions studied to show a substantial increase in activity at both 2 and 5 mM were Mn, K, Ca, and Mg <strong>(</strong>Fig. <a href="#Fig9" class="usa-link">9</a>D<strong>)</strong>. The fact that Cu and Hg may reduce enzyme activity suggests that the vicinal sulfhydryl groups of the L-GLS enzyme are essential for effective catalysis and that the metal cations and inhibitors used in this study must be able to bind to these sulfhydryl groups to affect enzyme activity.</p></section><section id="Sec25"><h3 class="pmc_sec_title">3.9 Relative rate of hydrolysis of different substrates</h3>
<p id="Par49">Hydrolysis is a chemical reaction in which a molecule is broken down into smaller molecules with water. L-GLS tested substrates of L-glutamine, L-glutamic acid, D-asparagine, and D-glutamine. The results indicated that L-glutamine was hydrolyzed faster than others (100%), and D-asparagine was the lowest one (5%) (Fig. <a href="#Fig10" class="usa-link">10</a>). These results can help understand chemical reactions involved in various biological processes. Furthermore, these hydrolysis rates can significantly affect protein synthesis, nitrogen metabolism, and neurotransmitter production processes. The faster hydrolysis rate of L-glutamine suggests its importance in these processes, while the slower rate of D-asparagine may indicate a more limited role in biological reactions. The difference in hydrolysis rates between L-glutamine and D-asparagine could be attributed to the structural differences between the two molecules. L-glutamine may have a more favorable arrangement of atoms, allowing for easier bonding with water molecules and, thus, faster hydrolysis. Additionally, specific enzymes or catalysts in the biological system could also play a role in accelerating the hydrolysis of L-glutamine compared to D-asparagine.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/836b5929d967/41598_2025_14230_Fig10_HTML.jpg" loading="lazy" id="d33e1066" height="1575" width="762" alt="Fig. 10"></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The relative rate of hydrolysis of different substrates. Hydrolysis of L-glutamine was faster (100%), and that of D-asparagine was lowest (5%).</p></figcaption></figure></section><section id="Sec26"><h3 class="pmc_sec_title">Steady-state kinetic analysis of L-GLS</h3>
<p id="Par51">An enzyme’s Km value is affected by its substrate and environmental conditions at the measurement time. <em>Halomonas aquamarina</em> HBIM1 kinetic constants have been determined using Michaelis-Menten’s behavior when L-glutamine is the substrate. Km and Vmax are calculated by plotting 1/V and 1/[S] in the Lineweaver-Burk plot. As shown in <strong>(</strong>Fig. <a href="#Fig11" class="usa-link">11</a>A<strong>)</strong>, an L-GLS of <em>Halomonas aquamarina</em> HBIM1 had a <em>Km</em> and a <em>Vmax</em> of 0.198 mM<sup>− 1</sup> and 0.04 µmole/ml/min, respectively. These kinetic constants provide insights into the efficiency and effectiveness of the enzyme in its catalytic role.</p>
<figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/3e69aa5afea0/41598_2025_14230_Fig11_HTML.jpg" loading="lazy" id="d33e1121" height="534" width="669" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Km values for L-glutamine obtained using <em>Halomonas aquamarina</em> strain HBIM1 L-GLS as substrate. (<strong>B</strong>) Arrhenius plot of L-GLS activity for <em>Halomonas aquamarina</em> strain HBIM1, plotted against 1/Temperature. A standard assay reaction mixture was used to measure enzyme activity at different temperatures. Based on three independent experiments, each point represents the average. (<strong>C</strong>) L-GLS exhibited a high degree of efficiency after 4 h of incubation.</p></figcaption></figure><p id="Par53">The 20–80 °C temperature range, the Arrhenius plots indicated a linear relationship between Ea and temperature. The activation energy (Ea) was calculated at 4.89 kcal/mol <strong>(</strong>Fig. <a href="#Fig11" class="usa-link">11</a>B<strong>).</strong> The calculated activation energy value of 4.89 kcal/mol indicates the minimum energy required for the reaction. This suggests that at higher temperatures within the 20–80 °C range, the reaction becomes more favorable and proceeds faster, increasing L-GLS efficiency. Regarding L-GLS efficiency from marine water (Fig. <a href="#Fig11" class="usa-link">11</a>C), it was revealed that L-GLS increased significantly after 4 h of incubation. This suggests that the incubation process positively impacts L-GLS performance. This indicates that the incubation process is likely activating the L-GLS cells, allowing them to become more active and efficient. Additionally, the longer the incubation process, the more efficient the L-GLS cells become.</p></section><section id="Sec27"><h3 class="pmc_sec_title">L-GLS enhances cytotoxicity in liver cancer cell line</h3>
<p id="Par54">In WI-38 and HepG2 cells treated for 48 h, L-GLS had antiproliferative effects on the cells. L-GLS IC50 values were determined using dose-response curves. In comparison to control cells, L-GLS-treated cells inhibited cell proliferation dose-dependently. According to HepG2 cells, the IC50 for L-GLS was 33.98 µg/ml. WI-38 cells, however, exhibited IC50 values of 93.43 µg/ml <strong>(</strong>Fig. <a href="#Fig12" class="usa-link">12</a>A-B<strong>)</strong>. The difference in IC50 values between HepG2 and WI-38 cells could be attributed to variations in the expression levels of the target enzyme, L-GLS, in each cell line. HepG2 cells may have higher levels of L-GLS, making them more sensitive to the inhibitory effects of L-GLS than WI-38 cells. Overall, these results suggest that L-GLS may be a promising target for developing novel therapeutic strategies for liver cancer.</p>
<figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/59954e864eb7/41598_2025_14230_Fig12_HTML.jpg" loading="lazy" id="d33e1165" height="313" width="669" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Antiproliferative effects of L-GLS on (<strong>A</strong>). WI-38 and (<strong>B</strong>) HepG2 cells, with IC50 values determined using dose-response curves. HepG2 cells exhibit an IC50 of 33.98 g/ml, whereas WI-38 cells exhibit an IC50 of 93.43 g/ml. L-GLS expression levels may vary between the two cell lines, with HepG2 cells possibly having higher levels of L-GLS, making them more susceptible to L-GLS inhibition.</p></figcaption></figure></section><section id="Sec28"><h3 class="pmc_sec_title">In Silico assessment</h3>
<p id="Par56">Molecular docking scores and human and bacterial L-GLS interaction against 6-diazo-5-oxo-L-norleucine and tannic acid were quantified and illustrated in Figs. <a href="#Fig13" class="usa-link">13</a> and <a href="#Fig14" class="usa-link">14</a>. The molecular docking studies provided significant insights into DON and tannic acid’s binding affinities and molecular interactions with bacterial (<em>Halomonas aquamarina</em> L-GLS) and human glutaminase. These analyses revealed notable differences in ligand specificity, binding energy, and the interaction profiles for each compound, shedding light on their potential as selective inhibitors.</p>
<figure class="fig xbox font-sm" id="Fig13"><h4 class="obj_head">Fig. 13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/436ef3e7fec6/41598_2025_14230_Fig13_HTML.jpg" loading="lazy" id="d33e1192" height="945" width="766" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Molecular docking scores and interactions between human and bacterial L-GLS and inhibitors of 6-diazo-5-oxo-L-norleucine. the binding affinity of 6-diazo-5-oxo-L-norleucine with <em>Halomonas aquamarina</em> L-GLS (score = -5.37 kcal/mol) compared to human L-GLS (score = -4.02 kcal/mol).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig14"><h4 class="obj_head">Fig. 14.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375786_41598_2025_14230_Fig14_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6a5/12375786/a04af82b700d/41598_2025_14230_Fig14_HTML.jpg" loading="lazy" id="d33e1206" height="945" width="710" alt="Fig. 14"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The docking scores of tannic acid with <em>Halomonas aquamarina</em> L-GLS (-12.25 kcal/mol) compared to human GLS (-11.26 kcal/mol). The higher binding affinity of tannic acid with bacterial L-GLS suggests its potency as an inhibitor for GLS, mainly targeting cancer cell growth and proliferation.</p></figcaption></figure><p id="Par59">In the case of DON, the docking scores revealed a stronger binding affinity for bacterial L-GLS (-5.37 kcal/mol) compared to human GLS (-4.02 kcal/mol), indicating a preference for the bacterial enzyme. Detailed analysis of the binding interactions revealed that key residues, including THR486 and GLN514, play a critical role in stabilizing the ligand through hydrogen bond donor and acceptor interactions, with energies of -0.7 and − 3.5 kcal/mol, respectively. Additionally, ASP511 emerged as a pivotal residue contributing to the ligand’s binding stability within the bacterial GLS binding pocket. In contrast, DON exhibited weaker and less energetically favorable interactions with human GLS, which could be attributed to differences in the structural configuration of the binding site, suggesting its potential as a selective inhibitor for bacterial GLS.</p>
<p id="Par60">Tannic acid demonstrated even higher binding affinities than DON, with scores of -12.25 kcal/mol for bacterial GLS and − 11.26 kcal/mol for human GLS, establishing it as the most potent ligand in this study. The docking results highlighted its ability to form multiple hydrogen bonds with critical residues such as GLN514, ASP511, and GLU540 in bacterial GLS, with binding energies ranging from − 0.7 to -3.7 kcal/mol. Furthermore, MET510 was identified as a residue facilitating π-π stacking interactions, further enhancing the ligand’s stabilization within the binding pocket. While tannic acid also interacted with key residues in human GLS, the interactions were weaker regarding distance and energy, indicating a degree of specificity toward the bacterial enzyme.</p>
<p id="Par61">Structural analysis of the binding pockets provided insights into the observed differences in binding affinities. The bacterial GLS binding pocket exhibited a higher density of polar residues, such as GLN514 and ASP511, facilitating stronger hydrogen bonding interactions with the ligands. This feature likely contributed to the enhanced binding of tannic acid and DON to bacterial GLS. On the other hand, the binding pocket of human GLS revealed slightly less optimal spatial alignments of residues, which could account for the reduced binding affinities and weaker interactions. These structural variations underline the selective nature of these inhibitors and their potential for targeted inhibition of bacterial GLS.</p>
<section id="Sec256"><h4 class="pmc_sec_title">Comparative binding affinity and selectivity profiling</h4>
<p id="Par62">Our comprehensive molecular docking analysis revealed significant selectivity differences between bacterial and human L-GLS for both tested inhibitors, providing crucial evidence for targeted therapeutic applications (Table <a href="#Tab3" class="usa-link">3</a>). Tannic acid demonstrated the highest binding affinity to <em>Halomonas aquamarina</em> HBIM1 L-GLS (-12.25 kcal/mol) compared to human L-GLS (-11.26 kcal/mol), yielding a selectivity index of 5.0 (ΔΔG = 0.99 kcal/mol), while DON showed preferential binding to bacterial L-glutaminase (-5.37 kcal/mol) over human enzyme (-4.02 kcal/mol) with a selectivity index of 3.5 (ΔΔG = 1.35 kcal/mol). The binding energy differences translate to significant selectivity at physiological conditions, with calculated dissociation constants revealing tannic acid’s Kd values of approximately 1.2 nM for bacterial enzyme versus 6.0 nM for human enzyme, and DON’s Kd values of 85 µM versus 298 µM, respectively. Detailed structural analysis revealed that bacterial L-GLS exhibits a 23% higher concentration of polar residues (GLN514, ASP511, GLU540) within the 5Å binding radius, facilitating the formation of 4–6 hydrogen bonds with tannic acid compared to only 2–3 bonds with the human enzyme, while enhanced π-π stacking interactions with MET510 (3.2 Å distance) are uniquely present in the bacterial enzyme. Energy decomposition analysis for tannic acid binding revealed that hydrogen bonds contribute − 8.3 kcal/mol (68%) to bacterial enzyme binding versus − 6.9 kcal/mol (61%) to human enzyme binding, with Van der Waals interactions contributing − 2.8 kcal/mol and π-π stacking adding − 1.15 kcal/mol specifically to bacterial enzyme selectivity. Cross-docking validation with multiple conformations confirmed bacterial selectivity in 94% of docking poses, with lower conformational flexibility observed for bacterial-bound complexes (RMSD = 0.8 Å) compared to human-bound complexes (RMSD = 1.4 Å). The calculated therapeutic index (TI = IC50 human/IC50 bacterial) of 5.0 for tannic acid exceeds the minimum threshold (TI ≥ 3) typically required for selective therapeutic agents, indicating that tannic acid can achieve selective bacterial L-GLS inhibition at concentrations that minimally affect normal human L-GLS function, thereby providing a substantial therapeutic window for targeted anticancer therapy with reduced treatment-associated toxicity.</p>
<section class="tw xbox font-sm" id="Tab3"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p>Comprehensive selectivity analysis of L-Glutaminase Inhibitors.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Parameter</th>
<th align="left" colspan="2" rowspan="1">Tannic Acid</th>
<th align="left" colspan="2" rowspan="1">DON</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Bacterial</td>
<td align="left" colspan="1" rowspan="1">Human</td>
<td align="left" colspan="1" rowspan="1">Bacterial</td>
<td align="left" colspan="1" rowspan="1">Human</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Binding Affinity</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Binding Energy (kcal/mol)</td>
<td align="left" colspan="1" rowspan="1">-12.25</td>
<td align="left" colspan="1" rowspan="1">-11.26</td>
<td align="left" colspan="1" rowspan="1">-5.37</td>
<td align="left" colspan="1" rowspan="1">-4.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Estimated Kd (nM)</td>
<td align="left" colspan="1" rowspan="1">1.2</td>
<td align="left" colspan="1" rowspan="1">6.0</td>
<td align="left" colspan="1" rowspan="1">85,000</td>
<td align="left" colspan="1" rowspan="1">298,000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Selectivity Index*</td>
<td align="left" colspan="1" rowspan="1">5.0</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">3.5</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Structural Interactions</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hydrogen Bonds</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">π-π Stacking</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Polar Contacts</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Energy Decomposition</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H-bond Energy (kcal/mol)</td>
<td align="left" colspan="1" rowspan="1">-8.3</td>
<td align="left" colspan="1" rowspan="1">-6.9</td>
<td align="left" colspan="1" rowspan="1">-3.8</td>
<td align="left" colspan="1" rowspan="1">-2.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VdW Energy (kcal/mol)</td>
<td align="left" colspan="1" rowspan="1">-2.8</td>
<td align="left" colspan="1" rowspan="1">-3.1</td>
<td align="left" colspan="1" rowspan="1">-1.2</td>
<td align="left" colspan="1" rowspan="1">-1.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">π-π Energy (kcal/mol)</td>
<td align="left" colspan="1" rowspan="1">-1.15</td>
<td align="left" colspan="1" rowspan="1">-1.26</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pocket Properties</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Polar Residue Density (%)</td>
<td align="left" colspan="1" rowspan="1">67</td>
<td align="left" colspan="1" rowspan="1">44</td>
<td align="left" colspan="1" rowspan="1">67</td>
<td align="left" colspan="1" rowspan="1">44</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Binding Volume (Å<sup>3</sup>)</td>
<td align="left" colspan="1" rowspan="1">850</td>
<td align="left" colspan="1" rowspan="1">920</td>
<td align="left" colspan="1" rowspan="1">850</td>
<td align="left" colspan="1" rowspan="1">920</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Validation Metrics</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Consistency Score (%)</td>
<td align="left" colspan="1" rowspan="1">94</td>
<td align="left" colspan="1" rowspan="1">87</td>
<td align="left" colspan="1" rowspan="1">91</td>
<td align="left" colspan="1" rowspan="1">78</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RMSD (Å)</td>
<td align="left" colspan="1" rowspan="1">0.8</td>
<td align="left" colspan="1" rowspan="1">1.4</td>
<td align="left" colspan="1" rowspan="1">1.1</td>
<td align="left" colspan="1" rowspan="1">1.8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p78"><p>*Selectivity index calculated as relative binding affinity (bacterial/human).</p></div></div></section></section></section></section><section id="Sec30"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par64">In the quest for novel biocatalysts with diverse biomedical applications, microbial enzymes have emerged as key players due to their versatility and efficiency in catalyzing biochemical reactions<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Enzyme biotechnology has garnered significant attention in recent years owing to its potential to revolutionize various sectors, including the pharmaceutical and biomedical fields. Among microbial enzymes<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>L-GLS stands out as a promising candidate for biotechnological applications, given its pivotal role in catalyzing the hydrolysis of L-glutamine, a ubiquitous amino acid with diverse physiological functions<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par65">While conventional techniques were employed for enzyme screening and characterization, this study significantly advances the field by identifying and characterizing <em>Halomonas aquamarina</em> HBIM1—a previously unexploited marine bacterium—as a high-yield producer of L-GLS with therapeutic potential. Unlike commonly studied strains such as <em>Bacillus subtilis</em> or <em>Escherichia coli</em>, <em>H. aquamarina</em> adds diversity to the microbial enzyme repertoire and offers yield, thermal stability, and pH tolerance advantages.</p>
<p id="Par66">Here, the study presented provides a comprehensive investigation into the production, purification, characterization, and potential therapeutic applications of L-GLS derived from <em>Halomonas aquamarina</em> HBIM1. The findings shed light on several crucial aspects of L-GLS production, including the most effective bacterial isolates, optimal conditions for enzyme production, purification methods, kinetic properties, substrate hydrolysis rates, and cytotoxic effects on liver cancer cells. Additionally, in silico assessments highlight potential inhibitors of L-GLS, paving the way for targeted anticancer therapies.</p>
<p id="Par67">Firstly, the study identifies WS6 as a highly efficient producer of L-GLS, with significantly higher enzyme activities than other isolates. This finding is consistent with previous studies demonstrating the variability in enzyme production among bacterial strains<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. In addition, previous studies have also reported on the variability in enzyme production among bacterial strains<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. For instance, Senthilkumar et al.<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> demonstrated significant differences in the enzyme activities of various bacterial isolates. Additionally, Mostafa et al.<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> highlighted the prevalence of <em>Halomonas</em> species in L-GLS production, supporting the current study’s findings regarding <em>Halomonas meridiana</em>’s efficiency in enzyme production. Furthermore, genotypic identification confirms the selected isolate, WS6, as <em>Halomonas aquamarina</em> HBIM1, exhibiting 99% sequence identity with known L-GLS-producing strains. This aligns with previous research highlighting the prevalence of marine species in L-GLS production<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p>
<p id="Par68">The study also elucidates the optimal conditions for L-GLS production, including incubation time, temperature, and pH. The observed surge in enzyme activity at 30 h of incubation, optimal activity at 40 °C, and pH 6 reflects the adaptability of <em>Halomonas aquamarina</em> HBIM1 to varying environmental conditions. These findings corroborate studies emphasizing the influence of incubation parameters on enzyme production in microbial systems<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p>
<p id="Par69">From a therapeutic perspective, the ability of L-GLS to remain active at elevated temperatures and physiological conditions is particularly relevant for targeting glutamine-dependent tumor cells. Tumors often highly depend on glutamine for growth and survival, and stable L-GLS enzymes can effectively degrade glutamine in the tumor microenvironment. This property positions bacterial L-GLS as a potential candidate for enzyme-based therapies to inhibit glutamine metabolism in glutamine-addicted cancers. The dual applicability of bacterial L-GLS in industrial and therapeutic fields highlights its versatility and underscores the need for further studies to explore and optimize its functional properties for specific applications.</p>
<p id="Par70">In therapeutic contexts, L-GLS revealed promise in targeting glutamine-dependent tumor cells, which rely heavily on glutamine for survival and proliferation<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The enzyme’s ability to remain active in the warmer conditions of tumor microenvironments ensures its effectiveness in depleting glutamine, thereby inhibiting cancer cell growth. Additionally, its stability aligns well with hyperthermia-based cancer therapies, where localized heating enhances treatment efficacy<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. So, future research is needed to optimize L-GLS performance in tumor-like conditions and industrial systems to harness its potential fully in these fields. Moreover, the study evaluates the impact of carbon and nitrogen sources on L-GLS production, revealing fructose and ammonium sulfate as the most favorable substrates. This aligns with previous research demonstrating the preference of microbial enzymes for specific carbon and nitrogen sources due to their metabolic pathways<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Furthermore, a Study by Ariaeenejad et al.<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> emphasized the influence of incubation parameters on enzyme production in microbial systems, aligning with the observed surge in enzyme activity at specific incubation times in the current study. Furthermore, Banerjee et al.<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> have investigated the impact of carbon and nitrogen sources on enzyme production, corroborating the preference for specific substrates observed in the current study.</p>
<p id="Par71">Furthermore, the purification process enhances the catalytic efficiency and purity of L-GLS, as evidenced by SDS-PAGE analysis. A single distinct protein band with a molecular weight of 66 kDa indicates high purity and minimal contamination. This finding underscores the importance of purification techniques in obtaining biologically active enzymes for downstream applications<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Amiri et al.<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> have highlighted the importance of purification techniques in obtaining biologically active enzymes, which resonates with the purification methods employed and the SDS-PAGE analysis conducted in the current study. These findings underscore the significance of ensuring enzyme purity for downstream applications.</p>
<p id="Par72">The kinetic analysis provides valuable insights into the enzyme’s behavior, including pH, thermal stability, and substrate affinity. The observed pH optimum of 8 and thermal stability at 50 °C suggest the enzyme’s suitability for biotechnological applications requiring moderate pH and temperature conditions. The steady-state kinetic analysis also reveals a Km of 0.198 mM<sup>− 1</sup> and Vmax of 0.04 µmole/ml/min, indicating efficient substrate binding and turnover rates. Studies on enzyme kinetics by various researchers have provided insights into the behavior of microbial enzymes under different conditions<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The observed pH and thermal stability of L-GLS in the current study align with previous findings on other microbial enzymes<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p>
<p id="Par73">Furthermore, these findings revealed that the Km and Vmax values of L-GLS from <em>Halomonas aquamarina</em> HBIM1 fall within the typical values reported for microbial L-GLS enzymes<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Specifically, the Vmax value, on the other hand, suggests a high catalytic efficiency, positioning this L-GLS as a promising candidate for both industrial and therapeutic applications<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Compared to other microbial enzymes, the thermal stability of L-GLS was favorable, with activity maintained even at elevated temperatures<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. This stability and pH tolerance highlight the enzyme’s potential for therapeutic applications where variable environmental conditions may be encountered. Additionally, the determined Km and Vmax values provide valuable information on the enzyme’s catalytic efficiency, similar to findings reported in other enzymatic studies. Research by Akram et al.<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> explored the potential of microbial enzymes in various industrial processes, highlighting their wide-ranging applications in biofuel production, food processing, and pharmaceuticals. Similarly, a study by Sodhi et al.<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup> investigated the biocatalytic properties of microbial enzymes, emphasizing their advantages over chemical catalysts in terms of specificity, environmental sustainability, and cost-effectiveness.</p>
<p id="Par74">To further highlight the distinctiveness of <em>Halomonas aquamarina</em> HBIM1, a direct comparison was made against other well-studied L-GLS-producing strains <strong>(</strong>Table <a href="#Tab4" class="usa-link">4</a><strong>)</strong>. The enzyme yield from HBIM1 reached 39.82 U/ml, exceeding values reported for <em>Halomonas meridiana</em> (32.5 U/ml)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup><em>Streptomyces avermitilis</em> (28.4 U/ml)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>and <em>Bacillus subtilis</em> (25.6 U/ml)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. In terms of enzymatic efficiency, <em>Halomonas aquamarina</em> HBIM1 displayed a superior specific activity of 748.35 U/mg, compared to 523.6 U/mg and 480.1 U/mg in <em>Halomonas meridiana</em> and <em>Streptomyces avermitilis</em>, respectively. The enzyme’s kinetic constant (Km = 0.198 mM⁻¹) also reflects a higher substrate affinity than those reported for similar bacterial sources (Km &gt; 0.24 mM⁻¹). Moreover, <em>Halomonas aquamarina</em> HBIM1-derived L-GLS revealed optimal catalytic behavior at pH 8 and 50 °C, aligning it better with physiological and industrial conditions.</p>
<section class="tw xbox font-sm" id="Tab4"><h3 class="obj_head">Table 4.</h3>
<div class="caption p"><p>Comparative biochemical and cytotoxic properties of L-glutaminase enzymes from various microbial sources.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Parameter</th>
<th align="left" colspan="1" rowspan="1">
<em>H. aquamarina</em> HBIM1<br>(This study)</th>
<th align="left" colspan="1" rowspan="1">
<em>H. meridiana</em>
<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>
</th>
<th align="left" colspan="1" rowspan="1">
<em>S. avermitilis</em>
<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>
</th>
<th align="left" colspan="1" rowspan="1">
<em>B. subtilis</em>
<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Enzyme yield (U/ml)</td>
<td align="left" colspan="1" rowspan="1">
<strong>39.82</strong>
</td>
<td align="left" colspan="1" rowspan="1">32.5</td>
<td align="left" colspan="1" rowspan="1">28.4</td>
<td align="left" colspan="1" rowspan="1">25.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Specific activity (U/mg)</td>
<td align="left" colspan="1" rowspan="1">
<strong>748.35</strong>
</td>
<td align="left" colspan="1" rowspan="1">523.6</td>
<td align="left" colspan="1" rowspan="1">480.1</td>
<td align="left" colspan="1" rowspan="1">410.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Optimal pH</td>
<td align="left" colspan="1" rowspan="1">
<strong>8.0</strong>
</td>
<td align="left" colspan="1" rowspan="1">7.5</td>
<td align="left" colspan="1" rowspan="1">7.0</td>
<td align="left" colspan="1" rowspan="1">7.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Optimal temperature (°C)</td>
<td align="left" colspan="1" rowspan="1">
<strong>50</strong>
</td>
<td align="left" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">40</td>
<td align="left" colspan="1" rowspan="1">37</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Km (mM⁻¹)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0.198</strong>
</td>
<td align="left" colspan="1" rowspan="1">0.244</td>
<td align="left" colspan="1" rowspan="1">0.285</td>
<td align="left" colspan="1" rowspan="1">0.315</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>IC50</p>
<p>(HepG2, µg/ml)</p>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>33.98</strong>
</td>
<td align="left" colspan="1" rowspan="1">48.9</td>
<td align="left" colspan="1" rowspan="1">52.3</td>
<td align="left" colspan="1" rowspan="1">60.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>IC50</p>
<p>(WI-38, µg/ml)</p>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>93.43</strong>
</td>
<td align="left" colspan="1" rowspan="1">Not reported</td>
<td align="left" colspan="1" rowspan="1">Not reported</td>
<td align="left" colspan="1" rowspan="1">Not reported</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p96"><p> Data highlight the superior performance of <em>Halomonas Aquamarina</em> HBIM1 in terms of production yield, specific activity, enzyme stability, substrate affinity, and selective anticancer activity.</p></div></div></section><p id="Par76">The study also explores the cytotoxic effects of L-GLS on liver cancer cells, demonstrating dose-dependent inhibition of cell proliferation. The differential IC50 values between HepG2 and WI-38 cells highlight the potential therapeutic utility of L-GLS in targeting cancer cells with varying sensitivities to the enzyme. The investigation of cytotoxic effects and therapeutic applications of microbial enzymes in cancer treatment has been a subject of interest in recent research<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The differential IC50 values observed in the current study reflect varying sensitivities of different cell lines to L-GLS, reminiscent of findings reported in studies on other anticancer enzymes<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Research conducted by Awad et al.<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> focused on the importance of L-GLS in cancer therapy, demonstrating its ability to induce apoptosis in cancer cells by depleting intracellular glutamine levels. Furthermore, the research by Abu-Tahon et al.<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> explored the therapeutic potential of L-GLS in cancer treatment, demonstrating its efficacy in sensitizing cancer cells to chemotherapy and radiation therapy.</p>
<p id="Par77">Critically, the cytotoxicity assay revealed that <em>Halomonas aquamarina</em> HBIM1’s L-GLS exerts a more potent antiproliferative effect on HepG2 liver cancer cells (IC50 = 33.98 µg/ml) than other microbial L-GLS enzymes, which typically require higher doses to achieve comparable effects. Importantly, the IC50 for normal WI-38 cells was significantly higher (93.43 µg/ml), indicating a strong therapeutic window <strong>(</strong>Table <a href="#Tab4" class="usa-link">4</a><strong>)</strong>. Together, these comparative metrics establish <em>Halomonas aquamarina</em> HBIM1 as a novel source of L-GLS and a highly promising candidate for therapeutic and industrial applications.</p>
<p id="Par78">The selectivity of L-GLS for cancer cells over normal cells suggests its potential for targeted cancer therapy, minimizing off-target effects. These findings are consistent with previous studies demonstrating the ability of glutaminase to reduce tumor growth by depriving cancer cells of glutamine, an essential nutrient for their survival and proliferation<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Regarding therapeutic implications, the IC50 values obtained in this study indicate that L-GLS may be a viable candidate for further investigation as a cancer treatment. Its selectivity and potency offer promising avenues for developing enzyme-based therapies targeting glutamine metabolism in cancer cells. Therefore, future studies should further investigate the in-vivo efficacy of L-GLS, exploring its potential as part of a combination therapy with existing cancer treatments to enhance therapeutic outcomes<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>.</p>
<p id="Par79">Finally, in silico assessments identify potential inhibitors of L-GLS, including 6-diazo-5-oxo-L-norleucine and tannic acid, with higher binding affinities to bacterial L-GLS compared to human L-GLS. These findings hold promise for developing targeted anticancer therapies leveraging bacterial L-GLS inhibition. L-GLS inhibitors such as 6-diazo-5-oxo-L-norleucine and tannic acid work by binding to L-GLS enzymes, preventing them from functioning normally. This inhibition of L-GLS specifically targets their metabolic pathways, which are crucial for their survival and proliferation. By selectively inhibiting L-GLS, this compound can potentially disrupt cancer cells that rely on bacterial metabolism, opening up new possibilities for targeted anticancer therapies<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. These findings align with previous studies highlighting the importance of residues like GLN514 and ASP511 in bacterial GLS for ligand stabilization<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. The role of π-π stacking interactions, such as those observed with MET510, has been extensively discussed in earlier research on natural inhibitors<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. The observed docking scores and interaction profiles support the hypothesis that tannic acid and DON may serve as selective inhibitors for bacterial GLS, with tannic acid demonstrating superior binding potential. Overall, these results provide a strong foundation for understanding the molecular basis of ligand specificity in bacterial versus human GLS. The observed selectivity of tannic acid for bacterial L-GLS over human L-GLS (0.99 kcal/mol binding energy difference) provides crucial evidence for its potential as a targeted anticancer therapy. This selectivity is attributed to the higher density of polar residues in the bacterial L-GLS binding pocket, facilitating stronger hydrogen bonding interactions, while the human L-GLS binding pocket exhibits less optimal spatial alignments. This differential binding profile suggests that tannic acid can preferentially target cancer cells utilizing bacterial L-GLS while minimizing adverse effects on normal cellular L-GLS activity.</p>
<p id="Par80">It was known that glutaminase plays a crucial role in cancer cell metabolism by catalyzing the conversion of glutamine to glutamate, which fuels various biosynthetic pathways essential for tumor growth and survival<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. Inhibition of glutaminase activity has emerged as a promising strategy for cancer therapy, particularly in tumors reliant on glutamine metabolism<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>,<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. As described in the study, the docking analysis identified tannic acid as a potential inhibitor of L-GLS. The higher binding affinity of tannic acid to L-GLS suggests its ability to inhibit GLS activity could disrupt cancer cell metabolism and hinder tumor progression. Leveraging tannic acid’s inhibitory effects on L-GLS could offer a targeted approach for disrupting cancer cell metabolism while minimizing off-target effects on normal tissues<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. The findings regarding potential inhibitors of L-GLS in the current study align with similar computational studies targeting other enzymes<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup> and highlight the effectiveness and reliability of in silico assessments in drug discovery and development.</p>
<p id="Par81">It is vital to acknowledge the study’s limitations. Firstly, this study utilized a one-factor-at-a-time (OFAT) approach to optimize key parameters, including incubation time, temperature, pH, and carbon and nitrogen sources for L-GLS enzyme production. While this approach provided valuable insights, it does not account for potential interactions among variables. Future studies should employ advanced statistical experimental designs, such as response surface methodology (RSM) or factorial design, to systematically evaluate these interactions and optimize the process more efficiently. Incorporating such models would enable a more comprehensive understanding of synergistic effects between variables and ensure a more robust and reproducible optimization process. While this study successfully evaluated the docking scores and interactions of the ligands with the target proteins, ligand geometry optimization was not performed after obtaining the compounds. This step is crucial for ensuring the ligands are in their energetically favorable conformations, which can significantly enhance the reliability and accuracy of docking predictions. Before docking analysis, ligand geometry optimization will be incorporated in future studies using appropriate computational tools, such as density functional theory (DFT) or molecular mechanics. This addition will allow for a more precise evaluation of binding interactions and contribute to a deeper understanding of the ligand-protein dynamics.</p></section><section id="Sec31"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par82">Our study comprehensively investigated the production, purification, and preliminary therapeutic evaluation of L-GLS derived from <em>Halomonas aquamarina</em> HBIM1. Key findings include the identification of WS6 as a highly efficient bacterial isolate, optimal production conditions utilizing fructose and ammonium sulfate as preferred carbon and nitrogen sources, and successful purification methods that enhanced catalytic efficiency and purity as confirmed by SDS-PAGE analysis. Kinetic analysis revealed optimal pH and thermal stability conditions for L-GLS activity, with favorable substrate affinity (Km = 0.198 mM⁻¹). In vitro cytotoxicity evaluation demonstrated selective antiproliferative effects on HepG2 liver cancer cells (IC50 = 33.98 µg/ml) compared to normal WI-38 cells (IC50 = 93.43 µg/ml), indicating a 2.75-fold selectivity index that warrants further investigation. However, these preliminary in vitro findings require comprehensive validation through preclinical studies to establish in vivo efficacy, safety profile, and pharmacological properties before considering therapeutic applications. Computational analysis identified tannic acid and 6-diazo-5-oxo-L-norleucine (DON) as potential selective inhibitors with higher binding affinities to bacterial L-GLS compared to human L-glutaminase (selectivity indices of 5.0 and 3.5, respectively), suggesting possibilities for combination therapy approaches that merit preclinical validation. While our findings provide proof-of-concept evidence for the anticancer potential of <em>Halomonas aquamarina</em> L-GLS, extensive preclinical studies, including in vivo efficacy evaluation, comprehensive toxicological assessment, pharmacokinetic profiling, and immunogenicity studies, are essential prerequisites before clinical translation.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization, Alaa Elmetwalli, Mohamed O. Abdel Monem, Mervat G. Hassan, Fuad Ameen; Formal analysis, Alaa Elmetwalli, Jihan Hassan, Mohamed O. Abdel Monem, Mervat G. Hassan, Sara Abdelsayed; Investigation, Alaa Elmetwalli, Jihan Hassan, Mervat G. Hassan.; Project administration, Alaa Elmetwalli, Mohamed O. Abdel Monem, Mervat G. Hassan, Fuad Ameen; Software, Alaa Elmetwalli, Jihan Hassan, Ali H. El-Far, Mervat G. Hassan; Validation Alaa Elmetwalli, Jihan Hassan, Mohamed O. Abdel Monem, Mervat G. Hassan, Fuad Ameen; Visualization, Alaa Elmetwalli, Ali H. El-Far, Mervat G. Hassan, Sara Abdelsayed; Writing—original draft, Alaa Elmetwalli.; Writing—review and editing, Alaa Elmetwalli.; All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by The Science, Technology &amp; Innovation Funding Authority (STDF) in cooperation with The Egyptian Knowledge Bank (EKB). The authors extend their appreciation to the ongoing research funding program, (ORF-2025-364), King Saud University, Riyadh, Saudi Arabia.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and/or analyzed during the current study are available in the GeneBank repository with accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/OR707005.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OR707005.1</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par83">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par84">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par85">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Sara Abdelsayed and Alaa Elmetwalli shared first authorship.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Petrowsky, H. et al. Modern therapeutic approaches for the treatment of malignant liver tumours. <em>Nat. Rev. Gastroenterol. Hepatol.</em><strong>17</strong>, 755–772 (2020).
</cite> [<a href="https://doi.org/10.1038/s41575-020-0314-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32681074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petrowsky,%20H.%20et%20al.%20Modern%20therapeutic%20approaches%20for%20the%20treatment%20of%20malignant%20liver%20tumours.%20Nat.%20Rev.%20Gastroenterol.%20Hepatol.17,%20755%E2%80%93772%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Huang, M., Lu, J. J. &amp; Ding, J. Natural products in cancer therapy: Past, present and future. <em>Nat. Prod. Bioprospect</em>. <strong>11</strong>, 5–13 (2021).
</cite> [<a href="https://doi.org/10.1007/s13659-020-00293-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7933288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33389713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20M.,%20Lu,%20J.%20J.%20&amp;%20Ding,%20J.%20Natural%20products%20in%20cancer%20therapy:%20Past,%20present%20and%20future.%20Nat.%20Prod.%20Bioprospect.%2011,%205%E2%80%9313%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Attia, A. A. et al. Amygdalin potentiates the anti-cancer effect of Sorafenib on Ehrlich Ascites carcinoma and ameliorates the associated liver damage. <em>Sci. Rep.</em><strong>12</strong>, 1–9 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-10517-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9021277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35444229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Attia,%20A.%20A.%20et%20al.%20Amygdalin%20potentiates%20the%20anti-cancer%20effect%20of%20Sorafenib%20on%20Ehrlich%20Ascites%20carcinoma%20and%20ameliorates%20the%20associated%20liver%20damage.%20Sci.%20Rep.12,%201%E2%80%939%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Chi, H. et al. Characterization of a novel and glutaminase-free type II L-asparaginase from Corynebacterium glutamicum and its acrylamide alleviation efficiency in potato chips. <em>Int. J. Biol. Macromol.</em><strong>221</strong>, 1384–1393 (2022).
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2022.09.162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36130640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chi,%20H.%20et%20al.%20Characterization%20of%20a%20novel%20and%20glutaminase-free%20type%20II%20L-asparaginase%20from%20Corynebacterium%20glutamicum%20and%20its%20acrylamide%20alleviation%20efficiency%20in%20potato%20chips.%20Int.%20J.%20Biol.%20Macromol.221,%201384%E2%80%931393%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Zhou, Y. et al. Rational engineering and insight for a L-glutaminase activity reduced type II L-asparaginase from Bacillus licheniformis and its antileukemic activity in vitro. <em>Int. J. Biol. Macromol.</em><strong>257</strong>, 128690 (2024).
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2023.128690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20Rational%20engineering%20and%20insight%20for%20a%20L-glutaminase%20activity%20reduced%20type%20II%20L-asparaginase%20from%20Bacillus%20licheniformis%20and%20its%20antileukemic%20activity%20in%20vitro.%20Int.%20J.%20Biol.%20Macromol.257,%20128690%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Barzkar, N., Sohail, M., Tamadoni Jahromi, S., Nahavandi, R. &amp; Khodadadi, M. Marine microbial L-glutaminase: from pharmaceutical to food industry. <em>Appl. Microbiol. Biotechnol.</em><strong>105</strong>, 4453–4466 (2021).
</cite> [<a href="https://doi.org/10.1007/s00253-021-11356-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34043082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barzkar,%20N.,%20Sohail,%20M.,%20Tamadoni%20Jahromi,%20S.,%20Nahavandi,%20R.%20&amp;%20Khodadadi,%20M.%20Marine%20microbial%20L-glutaminase:%20from%20pharmaceutical%20to%20food%20industry.%20Appl.%20Microbiol.%20Biotechnol.105,%204453%E2%80%934466%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Debnath, T., Kujur, R. R. A., Mitra, R. &amp; Das, S. K. Diversity of microbes in hot springs and their sustainable use. <em>Microb. Divers. Ecosyst. Sustain. Biotechnol. Appl. Vol. 1. Microb. Divers. Norm. Extrem. Environ.</em> 159–186 (2019).</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Haki, G. D. &amp; Rakshit, S. K. Developments in industrially important thermostable enzymes: A review. <em>Bioresour Technol.</em><strong>89</strong>, 17–34 (2003).
</cite> [<a href="https://doi.org/10.1016/s0960-8524(03)00033-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12676497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haki,%20G.%20D.%20&amp;%20Rakshit,%20S.%20K.%20Developments%20in%20industrially%20important%20thermostable%20enzymes:%20A%20review.%20Bioresour%20Technol.89,%2017%E2%80%9334%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Feller, G. Protein stability and enzyme activity at extreme biological temperatures. <em>J. Phys. Condens. Matter</em>. <strong>22</strong>, 323101 (2010).
</cite> [<a href="https://doi.org/10.1088/0953-8984/22/32/323101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21386475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feller,%20G.%20Protein%20stability%20and%20enzyme%20activity%20at%20extreme%20biological%20temperatures.%20J.%20Phys.%20Condens.%20Matter.%2022,%20323101%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Nigam, P. S. Microbial enzymes with special characteristics for biotechnological applications. <em>Biomolecules</em><strong>3</strong>, 597–611 (2013).
</cite> [<a href="https://doi.org/10.3390/biom3030597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4030947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24970183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nigam,%20P.%20S.%20Microbial%20enzymes%20with%20special%20characteristics%20for%20biotechnological%20applications.%20Biomolecules3,%20597%E2%80%93611%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Sanchez, S. &amp; Demain, A. L. Enzymes and bioconversions of industrial, pharmaceutical, and biotechnological significance. <em>Org. Process. Res. Dev.</em><strong>15</strong>, 224–230 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sanchez,%20S.%20&amp;%20Demain,%20A.%20L.%20Enzymes%20and%20bioconversions%20of%20industrial,%20pharmaceutical,%20and%20biotechnological%20significance.%20Org.%20Process.%20Res.%20Dev.15,%20224%E2%80%93230%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Kumar, A., Mukhia, S. &amp; Kumar, R. Industrial applications of cold-adapted enzymes: Challenges, innovations and future perspective. <em>3 Biotech.</em><strong>11</strong>, 426 (2021).
</cite> [<a href="https://doi.org/10.1007/s13205-021-02929-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8421504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34567931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20A.,%20Mukhia,%20S.%20&amp;%20Kumar,%20R.%20Industrial%20applications%20of%20cold-adapted%20enzymes:%20Challenges,%20innovations%20and%20future%20perspective.%203%20Biotech.11,%20426%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Lukey, M. J., Wilson, K. F. &amp; Cerione, R. A. Therapeutic strategies impacting cancer cell glutamine metabolism. <em>Future Med. Chem.</em><strong>5</strong>, 1685–1700 (2013).
</cite> [<a href="https://doi.org/10.4155/fmc.13.130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4154374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24047273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lukey,%20M.%20J.,%20Wilson,%20K.%20F.%20&amp;%20Cerione,%20R.%20A.%20Therapeutic%20strategies%20impacting%20cancer%20cell%20glutamine%20metabolism.%20Future%20Med.%20Chem.5,%201685%E2%80%931700%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Nguyen, H. A., Su, Y. &amp; Lavie, A. Design and characterization of erwinia chrysanthemi L-asparaginase variants with diminished L-glutaminase activity. <em>J. Biol. Chem.</em><strong>291</strong>, 17664–17676 (2016).
</cite> [<a href="https://doi.org/10.1074/jbc.M116.728485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5016162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27354283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20H.%20A.,%20Su,%20Y.%20&amp;%20Lavie,%20A.%20Design%20and%20characterization%20of%20erwinia%20chrysanthemi%20L-asparaginase%20variants%20with%20diminished%20L-glutaminase%20activity.%20J.%20Biol.%20Chem.291,%2017664%E2%80%9317676%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Orabi, H., El-Fakharany, E., Abdelkhalek, E. &amp; Sidkey, N. Production, optimization, purification, characterization, and anti-cancer application of extracellular L-glutaminase produced from the marine bacterial isolate. <em>Prep Biochem. Biotechnol.</em><strong>50</strong>, 408–418 (2020).
</cite> [<a href="https://doi.org/10.1080/10826068.2019.1703193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31846380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orabi,%20H.,%20El-Fakharany,%20E.,%20Abdelkhalek,%20E.%20&amp;%20Sidkey,%20N.%20Production,%20optimization,%20purification,%20characterization,%20and%20anti-cancer%20application%20of%20extracellular%20L-glutaminase%20produced%20from%20the%20marine%20bacterial%20isolate.%20Prep%20Biochem.%20Biotechnol.50,%20408%E2%80%93418%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Maurya, D. K., Kumar, A., Chaurasiya, U., Hussain, T. &amp; Singh, S. K. Modern era of microbial biotechnology: opportunities and future prospects. in <em>Microbiomes and plant health</em> 317–343Elsevier, (2021).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Fidelito, G. et al. Multi-substrate metabolic tracing reveals marked heterogeneity and dependency on fatty acid metabolism in human prostate cancer. <em>Mol. Cancer Res.</em><strong>21</strong>, 359–373 (2023).
</cite> [<a href="https://doi.org/10.1158/1541-7786.MCR-22-0796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36574015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fidelito,%20G.%20et%20al.%20Multi-substrate%20metabolic%20tracing%20reveals%20marked%20heterogeneity%20and%20dependency%20on%20fatty%20acid%20metabolism%20in%20human%20prostate%20cancer.%20Mol.%20Cancer%20Res.21,%20359%E2%80%93373%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Pandian, S. R. K., Deepak, V., Nellaiah, H. &amp; Sundar, K. PEG–PHB-glutaminase nanoparticle inhibits cancer cell proliferation in vitro through glutamine deprivation. <em>Vitr Cell. Dev. Biol.</em><strong>51</strong>, 372–380 (2015).</cite> [<a href="https://doi.org/10.1007/s11626-014-9840-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25424834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pandian,%20S.%20R.%20K.,%20Deepak,%20V.,%20Nellaiah,%20H.%20&amp;%20Sundar,%20K.%20PEG%E2%80%93PHB-glutaminase%20nanoparticle%20inhibits%20cancer%20cell%20proliferation%20in%20vitro%20through%20glutamine%20deprivation.%20Vitr%20Cell.%20Dev.%20Biol.51,%20372%E2%80%93380%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Kuo, M. T., Chen, H. H. W., Feun, L. G. &amp; Savaraj, N. Targeting the proline–glutamine–asparagine–arginine metabolic axis in amino acid starvation cancer therapy. <em>Pharmaceuticals</em><strong>14</strong>, 72 (2021).
</cite> [<a href="https://doi.org/10.3390/ph14010072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7830038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33477430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuo,%20M.%20T.,%20Chen,%20H.%20H.%20W.,%20Feun,%20L.%20G.%20&amp;%20Savaraj,%20N.%20Targeting%20the%20proline%E2%80%93glutamine%E2%80%93asparagine%E2%80%93arginine%20metabolic%20axis%20in%20amino%20acid%20starvation%20cancer%20therapy.%20Pharmaceuticals14,%2072%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Carneiro, B. A. &amp; El-Deiry, W. S. Targeting apoptosis in cancer therapy. <em>Nat. Rev. Clin. Oncol.</em><strong>17</strong>, 395–417 (2020).
</cite> [<a href="https://doi.org/10.1038/s41571-020-0341-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8211386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32203277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carneiro,%20B.%20A.%20&amp;%20El-Deiry,%20W.%20S.%20Targeting%20apoptosis%20in%20cancer%20therapy.%20Nat.%20Rev.%20Clin.%20Oncol.17,%20395%E2%80%93417%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Elmetwalli, A. et al. Diarylheptanoids/sorafenib as a potential anticancer combination against hepatocellular carcinoma: the p53/MMP9 axis of action. <em>Naunyn Schmiedebergs Arch. Pharmacol</em> 1–17 (2023).</cite> [<a href="https://doi.org/10.1007/s00210-023-02470-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10497687/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37145126/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Matés, J. M. et al. Metabolic reprogramming of cancer by chemicals that target glutaminase isoenzymes. <em>Curr. Med. Chem.</em><strong>27</strong>, 5317–5339 (2020).
</cite> [<a href="https://doi.org/10.2174/0929867326666190416165004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31038055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mat%C3%A9s,%20J.%20M.%20et%20al.%20Metabolic%20reprogramming%20of%20cancer%20by%20chemicals%20that%20target%20glutaminase%20isoenzymes.%20Curr.%20Med.%20Chem.27,%205317%E2%80%935339%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kao, T. W., Chuang, Y. C., Lee, H. L., Kuo, C. C. &amp; Shen, Y. A. Therapeutic targeting of Glutaminolysis as a novel strategy to combat cancer stem cells. <em>Int. J. Mol. Sci.</em><strong>23</strong>, 15296 (2022).
</cite> [<a href="https://doi.org/10.3390/ijms232315296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9737183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36499623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kao,%20T.%20W.,%20Chuang,%20Y.%20C.,%20Lee,%20H.%20L.,%20Kuo,%20C.%20C.%20&amp;%20Shen,%20Y.%20A.%20Therapeutic%20targeting%20of%20Glutaminolysis%20as%20a%20novel%20strategy%20to%20combat%20cancer%20stem%20cells.%20Int.%20J.%20Mol.%20Sci.23,%2015296%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kumar, M. et al. Identification of small molecule inhibitors of RAD52 for breast cancer therapy: in Silico approach. <em>J. Biomol. Struct. Dyn.</em><strong>42</strong>, 4605–4618 (2024).
</cite> [<a href="https://doi.org/10.1080/07391102.2023.2220822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37288783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20M.%20et%20al.%20Identification%20of%20small%20molecule%20inhibitors%20of%20RAD52%20for%20breast%20cancer%20therapy:%20in%20Silico%20approach.%20J.%20Biomol.%20Struct.%20Dyn.42,%204605%E2%80%934618%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Gupta, D. et al. A comprehensive review on role of Aurora kinase inhibitors (AKIs) in cancer therapeutics. <em>Int J. Biol. Macromol</em><strong>130913</strong> (2024).</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.130913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38508544/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Gupta, D. et al. Identification of polypharmacological anticancerous molecules against Aurora kinase family of proteins. <em>J. Cell. Biochem.</em><strong>123</strong>, 719–735 (2022).
</cite> [<a href="https://doi.org/10.1002/jcb.30214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35040172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta,%20D.%20et%20al.%20Identification%20of%20polypharmacological%20anticancerous%20molecules%20against%20Aurora%20kinase%20family%20of%20proteins.%20J.%20Cell.%20Biochem.123,%20719%E2%80%93735%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Agu, P. C. et al. Molecular Docking as a tool for the discovery of molecular targets of nutraceuticals in diseases management. <em>Sci. Rep.</em><strong>13</strong>, 13398 (2023).
</cite> [<a href="https://doi.org/10.1038/s41598-023-40160-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10435576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37592012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agu,%20P.%20C.%20et%20al.%20Molecular%20Docking%20as%20a%20tool%20for%20the%20discovery%20of%20molecular%20targets%20of%20nutraceuticals%20in%20diseases%20management.%20Sci.%20Rep.13,%2013398%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Elmetwalli, A. et al. Evaluation of Bacillus Aryabhattai B8W22 peroxidase for phenol removal in waste water effluents. <em>BMC Microbiol.</em><strong>23</strong>, 1–13 (2023).
</cite> [<a href="https://doi.org/10.1186/s12866-023-02850-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10148497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37120512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elmetwalli,%20A.%20et%20al.%20Evaluation%20of%20Bacillus%20Aryabhattai%20B8W22%20peroxidase%20for%20phenol%20removal%20in%20waste%20water%20effluents.%20BMC%20Microbiol.23,%201%E2%80%9313%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Hasan, S. F., Elsoud, M. M. A., Sidkey, N. M. &amp; Elhateir, M. M. Production and characterization of polyhydroxybutyrate bioplastic precursor from Parageobacillus toebii using low-cost substrates and its potential antiviral activity. <em>Int J. Biol. Macromol</em><strong>129915</strong> (2024).</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.129915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38325682/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>El-Hadedy, D. E., MH, S., HH, E. &amp; Mm, S. Enhanced production of extracellular L-methioninase by entire cell immobilization of streptomyces MDMMH4 and examination of its utilization as an antioxidant agent. <em>Pak J. Pharm. Sci</em><strong>36</strong>, (2023).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/37599484/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Abellan-Schneyder, I. et al. Primer, pipelines, parameters: Issues in 16S rRNA gene sequencing. <em>Msphere</em><strong>6</strong>, 10–1128 (2021).</cite> [<a href="https://doi.org/10.1128/mSphere.01202-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8544895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33627512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abellan-Schneyder,%20I.%20et%20al.%20Primer,%20pipelines,%20parameters:%20Issues%20in%2016S%20rRNA%20gene%20sequencing.%20Msphere6,%2010%E2%80%931128%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Abdallah, N. A., Amer, S. K. &amp; Habeeb, M. K. Production, purification and characterization of L-glutaminase enzyme from streptomyces avermitilis. <em>Afr. J. Microbiol. Res.</em><strong>14</strong>, 1184–1190 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abdallah,%20N.%20A.,%20Amer,%20S.%20K.%20&amp;%20Habeeb,%20M.%20K.%20Production,%20purification%20and%20characterization%20of%20L-glutaminase%20enzyme%20from%20streptomyces%20avermitilis.%20Afr.%20J.%20Microbiol.%20Res.14,%201184%E2%80%931190%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Katikala, P. K., Bobbarala, V., Tadimalla, P. &amp; Guntuku, G. S. Screening of L-glutaminase producing marine bacterial cultures for extracellular production of L-glutaminase. <em>Int. J. ChemTech Res.</em><strong>1</strong>, 1232–1235 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Katikala,%20P.%20K.,%20Bobbarala,%20V.,%20Tadimalla,%20P.%20&amp;%20Guntuku,%20G.%20S.%20Screening%20of%20L-glutaminase%20producing%20marine%20bacterial%20cultures%20for%20extracellular%20production%20of%20L-glutaminase.%20Int.%20J.%20ChemTech%20Res.1,%201232%E2%80%931235%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Jayabalan, R. et al. Extracellular L-glutaminase production by marine brevundimonas diminuta MTCC 8486. <em>Int. J. Appl. Bioeng.</em><strong>4</strong>, 19–24 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jayabalan,%20R.%20et%20al.%20Extracellular%20L-glutaminase%20production%20by%20marine%20brevundimonas%20diminuta%20MTCC%208486.%20Int.%20J.%20Appl.%20Bioeng.4,%2019%E2%80%9324%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Reda, F. M. Kinetic properties of streptomyces Canarius L-Glutaminase and its anticancer efficiency. <em>Brazilian J. Microbiol.</em><strong>46</strong>, 957–968 (2015).</cite> [<a href="https://doi.org/10.1590/S1517-838246420130847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4704638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26691453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reda,%20F.%20M.%20Kinetic%20properties%20of%20streptomyces%20Canarius%20L-Glutaminase%20and%20its%20anticancer%20efficiency.%20Brazilian%20J.%20Microbiol.46,%20957%E2%80%93968%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>El-Sewedy, T. et al. Hepatocellular carcinoma cells: Activity of amygdalin and Sorafenib in targeting ampk/mtor and BCL-2 for anti-angiogenesis and apoptosis cell death. <em>BMC Complement. Med. Ther.</em><strong>23</strong>, 1–17 (2023).</cite> [<a href="https://doi.org/10.1186/s12906-023-04142-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10508032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37726740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Sewedy,%20T.%20et%20al.%20Hepatocellular%20carcinoma%20cells:%20Activity%20of%20amygdalin%20and%20Sorafenib%20in%20targeting%20ampk/mtor%20and%20BCL-2%20for%20anti-angiogenesis%20and%20apoptosis%20cell%20death.%20BMC%20Complement.%20Med.%20Ther.23,%201%E2%80%9317%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Elmetwalli, A. et al. Probiotic-derived silver nanoparticles target mTOR/MMP-9/BCL-2/dependent AMPK activation for hepatic cancer treatment. <em>Med. Oncol.</em><strong>41</strong>, 106 (2024).
</cite> [<a href="https://doi.org/10.1007/s12032-024-02330-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10995097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38575697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elmetwalli,%20A.%20et%20al.%20Probiotic-derived%20silver%20nanoparticles%20target%20mTOR/MMP-9/BCL-2/dependent%20AMPK%20activation%20for%20hepatic%20cancer%20treatment.%20Med.%20Oncol.41,%20106%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>El-Shehawy, A. A. et al. Thymoquinone, piperine, and Sorafenib combinations attenuate liver and breast cancers progression: Epigenetic and molecular Docking approaches. <em>BMC Complement. Med. Ther.</em><strong>23</strong>, 1–21 (2023).</cite> [<a href="https://doi.org/10.1186/s12906-023-03872-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9985300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36870998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Shehawy,%20A.%20A.%20et%20al.%20Thymoquinone,%20piperine,%20and%20Sorafenib%20combinations%20attenuate%20liver%20and%20breast%20cancers%20progression:%20Epigenetic%20and%20molecular%20Docking%20approaches.%20BMC%20Complement.%20Med.%20Ther.23,%201%E2%80%9321%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Al Balawi, A. N., Eldiasty, J. G., Mosallam, S. A. E. R., El-Alosey, A. R. &amp; Elmetwalli, A. Assessing multi-target antiviral and antioxidant activities of natural compounds against SARS-CoV-2: an integrated in vitro and in Silico study. <em>Bioresour Bioprocess.</em><strong>11</strong>, 108 (2024).
</cite> [<a href="https://doi.org/10.1186/s40643-024-00822-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11602940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39604740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al%20Balawi,%20A.%20N.,%20Eldiasty,%20J.%20G.,%20Mosallam,%20S.%20A.%20E.%20R.,%20El-Alosey,%20A.%20R.%20&amp;%20Elmetwalli,%20A.%20Assessing%20multi-target%20antiviral%20and%20antioxidant%20activities%20of%20natural%20compounds%20against%20SARS-CoV-2:%20an%20integrated%20in%20vitro%20and%20in%20Silico%20study.%20Bioresour%20Bioprocess.11,%20108%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Vilar, S., Cozza, G. &amp; Moro, S. Medicinal chemistry and the molecular operating environment (MOE): Application of QSAR and molecular Docking to drug discovery. <em>Curr. Top. Med. Chem.</em><strong>8</strong>, 1555–1572 (2008).
</cite> [<a href="https://doi.org/10.2174/156802608786786624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19075767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vilar,%20S.,%20Cozza,%20G.%20&amp;%20Moro,%20S.%20Medicinal%20chemistry%20and%20the%20molecular%20operating%20environment%20(MOE):%20Application%20of%20QSAR%20and%20molecular%20Docking%20to%20drug%20discovery.%20Curr.%20Top.%20Med.%20Chem.8,%201555%E2%80%931572%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Bewick, V., Cheek, L. &amp; Ball, J. Statistics review 9: One-way analysis of variance. <em>Crit. Care</em>. <strong>8</strong>, 1–7 (2004).
</cite> [<a href="https://doi.org/10.1186/cc2836" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC420045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15025774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bewick,%20V.,%20Cheek,%20L.%20&amp;%20Ball,%20J.%20Statistics%20review%209:%20One-way%20analysis%20of%20variance.%20Crit.%20Care.%208,%201%E2%80%937%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Rafeeq, H. et al. Esterases as emerging biocatalysts: mechanistic insights, genomic and metagenomic, immobilization, and biotechnological applications. <em>Biotechnol. Appl. Biochem.</em><strong>69</strong>, 2176–2194 (2022).
</cite> [<a href="https://doi.org/10.1002/bab.2277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34699092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rafeeq,%20H.%20et%20al.%20Esterases%20as%20emerging%20biocatalysts:%20mechanistic%20insights,%20genomic%20and%20metagenomic,%20immobilization,%20and%20biotechnological%20applications.%20Biotechnol.%20Appl.%20Biochem.69,%202176%E2%80%932194%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Tandon, S., Sharma, A., Singh, S., Sharma, S. &amp; Sarma, S. J. Therapeutic enzymes: Discoveries, production and applications. <em>J. Drug Deliv Sci. Technol.</em><strong>63</strong>, 102455 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tandon,%20S.,%20Sharma,%20A.,%20Singh,%20S.,%20Sharma,%20S.%20&amp;%20Sarma,%20S.%20J.%20Therapeutic%20enzymes:%20Discoveries,%20production%20and%20applications.%20J.%20Drug%20Deliv%20Sci.%20Technol.63,%20102455%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Binod, P. et al. Recent developments in l-glutaminase production and applications–An overview. <em>Bioresour Technol.</em><strong>245</strong>, 1766–1774 (2017).
</cite> [<a href="https://doi.org/10.1016/j.biortech.2024.131676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39447500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Binod,%20P.%20et%20al.%20Recent%20developments%20in%20l-glutaminase%20production%20and%20applications%E2%80%93An%20overview.%20Bioresour%20Technol.245,%201766%E2%80%931774%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Srinivasan, P. et al. Production and purification of laccase by Bacillus sp. using millet husks and its pesticide degradation application. <em>3 Biotech.</em><strong>9</strong>, 396 (2019).
</cite> [<a href="https://doi.org/10.1007/s13205-019-1900-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6789060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31656734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Srinivasan,%20P.%20et%20al.%20Production%20and%20purification%20of%20laccase%20by%20Bacillus%20sp.%20using%20millet%20husks%20and%20its%20pesticide%20degradation%20application.%203%20Biotech.9,%20396%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Gomaa, E. Z. Production, characterization, and antitumor efficiency of l-glutaminase from halophilic bacteria. <em>Bull. Natl. Res. Cent.</em><strong>46</strong>, 10 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gomaa,%20E.%20Z.%20Production,%20characterization,%20and%20antitumor%20efficiency%20of%20l-glutaminase%20from%20halophilic%20bacteria.%20Bull.%20Natl.%20Res.%20Cent.46,%2010%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Mostafa, Y. S. et al. L-glutaminase synthesis by marine Halomonas Meridiana isolated from the red sea and its efficiency against colorectal cancer cell lines. <em>Molecules</em><strong>26</strong>, 1963 (2021).
</cite> [<a href="https://doi.org/10.3390/molecules26071963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8037810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33807313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mostafa,%20Y.%20S.%20et%20al.%20L-glutaminase%20synthesis%20by%20marine%20Halomonas%20Meridiana%20isolated%20from%20the%20red%20sea%20and%20its%20efficiency%20against%20colorectal%20cancer%20cell%20lines.%20Molecules26,%201963%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Mousumi, D. &amp; Dayanand, A. Production and antioxidant attribute of L-glutaminase from streptomyces enissocaesilis DMQ-24. <em>Int. J. Latest Res. Sci. Technol.</em><strong>2</strong>, 1–9 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mousumi,%20D.%20&amp;%20Dayanand,%20A.%20Production%20and%20antioxidant%20attribute%20of%20L-glutaminase%20from%20streptomyces%20enissocaesilis%20DMQ-24.%20Int.%20J.%20Latest%20Res.%20Sci.%20Technol.2,%201%E2%80%939%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Gehlot, P., Kumar, M. &amp; Pareek, N. Production and purification of glutaminase free L-asparaginase from Lysinibacillus fusiformis and its appraisal in acrylamide mitigation of starchy foods. <em>Mater. Today Proc.</em><strong>69</strong>, 64–73 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gehlot,%20P.,%20Kumar,%20M.%20&amp;%20Pareek,%20N.%20Production%20and%20purification%20of%20glutaminase%20free%20L-asparaginase%20from%20Lysinibacillus%20fusiformis%20and%20its%20appraisal%20in%20acrylamide%20mitigation%20of%20starchy%20foods.%20Mater.%20Today%20Proc.69,%2064%E2%80%9373%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Abhini, K. N., Rajan, A. B., Zuhara, K. F. &amp; Sebastian, D. Response surface methodological optimization of l-asparaginase production from the medicinal plant endophyte acinetobacter baumannii ZAS1. <em>J. Genet. Eng. Biotechnol.</em><strong>20</strong>, 22 (2022).
</cite> [<a href="https://doi.org/10.1186/s43141-022-00309-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8828825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35138483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abhini,%20K.%20N.,%20Rajan,%20A.%20B.,%20Zuhara,%20K.%20F.%20&amp;%20Sebastian,%20D.%20Response%20surface%20methodological%20optimization%20of%20l-asparaginase%20production%20from%20the%20medicinal%20plant%20endophyte%20acinetobacter%20baumannii%20ZAS1.%20J.%20Genet.%20Eng.%20Biotechnol.20,%2022%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Arévalo-Tristancho, E., Díaz, L. E., Cortázar, J. E. &amp; Valero, M. F. Production and characterization of L-Asparaginases of isolated from the Arauca riverbank (Colombia). <em>Open Microbiol. J</em><strong>13</strong>, (2019).</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Wang, B., Pei, J., Xu, S., Liu, J. &amp; Yu, J. A glutamine tug-of-war between cancer and immune cells: Recent advances in unraveling the ongoing battle. <em>J. Exp. Clin. Cancer Res.</em><strong>43</strong>, 74 (2024).
</cite> [<a href="https://doi.org/10.1186/s13046-024-02994-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10921613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38459595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20B.,%20Pei,%20J.,%20Xu,%20S.,%20Liu,%20J.%20&amp;%20Yu,%20J.%20A%20glutamine%20tug-of-war%20between%20cancer%20and%20immune%20cells:%20Recent%20advances%20in%20unraveling%20the%20ongoing%20battle.%20J.%20Exp.%20Clin.%20Cancer%20Res.43,%2074%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tang, X. et al. Selective enhanced cytotoxicity of amino acid deprivation for cancer therapy using thermozyme functionalized nanocatalyst. <em>J. Nanobiotechnol.</em><strong>22</strong>, 53 (2024).</cite> [<a href="https://doi.org/10.1186/s12951-024-02326-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10848425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38326899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20X.%20et%20al.%20Selective%20enhanced%20cytotoxicity%20of%20amino%20acid%20deprivation%20for%20cancer%20therapy%20using%20thermozyme%20functionalized%20nanocatalyst.%20J.%20Nanobiotechnol.22,%2053%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Srivastava, N., Srivastava, M., Mishra, P. K., Ramteke, P. W. &amp; Singh, R. L. <em>New and Future Developments in Microbial Biotechnology and Bioengineering: from Cellulose To Cellulase: Strategies To Improve Biofuel Production</em> (Elsevier, 2019).</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Ariaeenejad, S. et al. Highly efficient computationally derived novel metagenome α-amylase with robust stability under extreme denaturing conditions. <em>Front. Microbiol.</em><strong>12</strong>, 713125 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.713125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8437397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34526977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ariaeenejad,%20S.%20et%20al.%20Highly%20efficient%20computationally%20derived%20novel%20metagenome%20%CE%B1-amylase%20with%20robust%20stability%20under%20extreme%20denaturing%20conditions.%20Front.%20Microbiol.12,%20713125%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Banerjee, S., Maiti, T. K. &amp; Roy, R. N. Production, purification, and characterization of cellulase from acinetobacter Junii GAC 16.2, a novel cellulolytic gut isolate of Gryllotalpa africana, and its effects on cotton fiber and sawdust. <em>Ann. Microbiol.</em><strong>70</strong>, 1–16 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Banerjee,%20S.,%20Maiti,%20T.%20K.%20&amp;%20Roy,%20R.%20N.%20Production,%20purification,%20and%20characterization%20of%20cellulase%20from%20acinetobacter%20Junii%20GAC%2016.2,%20a%20novel%20cellulolytic%20gut%20isolate%20of%20Gryllotalpa%20africana,%20and%20its%20effects%20on%20cotton%20fiber%20and%20sawdust.%20Ann.%20Microbiol.70,%201%E2%80%9316%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Guajardo, N. &amp; Schrebler, R. A. Upstream and downstream bioprocessing in enzyme technology. <em>Pharmaceutics</em><strong>16</strong>, 38 (2023).
</cite> [<a href="https://doi.org/10.3390/pharmaceutics16010038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10818583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38258049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guajardo,%20N.%20&amp;%20Schrebler,%20R.%20A.%20Upstream%20and%20downstream%20bioprocessing%20in%20enzyme%20technology.%20Pharmaceutics16,%2038%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Amiri, S., Mokarram, R. R., Khiabani, M. S., Bari, M. R. &amp; Khaledabad, M. A. Characterization of antimicrobial peptides produced by Lactobacillus acidophilus LA-5 and bifidobacterium lactis BB-12 and their inhibitory effect against foodborne pathogens. <em>Lwt</em><strong>153</strong>, 112449 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Amiri,%20S.,%20Mokarram,%20R.%20R.,%20Khiabani,%20M.%20S.,%20Bari,%20M.%20R.%20&amp;%20Khaledabad,%20M.%20A.%20Characterization%20of%20antimicrobial%20peptides%20produced%20by%20Lactobacillus%20acidophilus%20LA-5%20and%20bifidobacterium%20lactis%20BB-12%20and%20their%20inhibitory%20effect%20against%20foodborne%20pathogens.%20Lwt153,%20112449%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Priyadarshini, A., Sahoo, M. M., Raut, P. R., Mahanty, B. &amp; Sahoo, N. K. Kinetic modelling and process engineering of phenolics microbial and enzymatic biodegradation: a current outlook and challenges. <em>J. Water Process. Eng.</em><strong>44</strong>, 102421 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Priyadarshini,%20A.,%20Sahoo,%20M.%20M.,%20Raut,%20P.%20R.,%20Mahanty,%20B.%20&amp;%20Sahoo,%20N.%20K.%20Kinetic%20modelling%20and%20process%20engineering%20of%20phenolics%20microbial%20and%20enzymatic%20biodegradation:%20a%20current%20outlook%20and%20challenges.%20J.%20Water%20Process.%20Eng.44,%20102421%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Bhatt, P. et al. New insights into the degradation of synthetic pollutants in contaminated environments. <em>Chemosphere</em><strong>268</strong>, 128827 (2021).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2020.128827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33162154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhatt,%20P.%20et%20al.%20New%20insights%20into%20the%20degradation%20of%20synthetic%20pollutants%20in%20contaminated%20environments.%20Chemosphere268,%20128827%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Qeshmi, F. I., Homaei, A., Fernandes, P. &amp; Javadpour, S. Marine microbial L-asparaginase: Biochemistry, molecular approaches and applications in tumor therapy and in food industry. <em>Microbiol. Res.</em><strong>208</strong>, 99–112 (2018).
</cite> [<a href="https://doi.org/10.1016/j.micres.2018.01.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29551216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qeshmi,%20F.%20I.,%20Homaei,%20A.,%20Fernandes,%20P.%20&amp;%20Javadpour,%20S.%20Marine%20microbial%20L-asparaginase:%20Biochemistry,%20molecular%20approaches%20and%20applications%20in%20tumor%20therapy%20and%20in%20food%20industry.%20Microbiol.%20Res.208,%2099%E2%80%93112%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Saleem, R. &amp; Ahmed, S. Characterization of a new L-Glutaminase produced by achromobacter xylosoxidans RSHG1, isolated from an expired hydrolyzed L-Glutamine sample. <em>Catalysts</em><strong>11</strong>, 1262 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Saleem,%20R.%20&amp;%20Ahmed,%20S.%20Characterization%20of%20a%20new%20L-Glutaminase%20produced%20by%20achromobacter%20xylosoxidans%20RSHG1,%20isolated%20from%20an%20expired%20hydrolyzed%20L-Glutamine%20sample.%20Catalysts11,%201262%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Bagewadi, Z. K. et al. Molecular expression, purification and structural characterization of Recombinant L-Glutaminase from streptomyces Roseolus. <em>Int. J. Biol. Macromol.</em><strong>273</strong>, 133142 (2024).
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.133142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38889830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bagewadi,%20Z.%20K.%20et%20al.%20Molecular%20expression,%20purification%20and%20structural%20characterization%20of%20Recombinant%20L-Glutaminase%20from%20streptomyces%20Roseolus.%20Int.%20J.%20Biol.%20Macromol.273,%20133142%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Mosallatpour, S., Aminzadeh, S., Shamsara, M. &amp; Hajihosseini, R. Novel halo-and thermo-tolerant Cohnella sp. A01 L-glutaminase: Heterologous expression and biochemical characterization. <em>Sci. Rep.</em><strong>9</strong>, 19062 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-55587-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6910923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31836796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mosallatpour,%20S.,%20Aminzadeh,%20S.,%20Shamsara,%20M.%20&amp;%20Hajihosseini,%20R.%20Novel%20halo-and%20thermo-tolerant%20Cohnella%20sp.%20A01%20L-glutaminase:%20Heterologous%20expression%20and%20biochemical%20characterization.%20Sci.%20Rep.9,%2019062%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Beckett, A. &amp; Gervais, D. What makes a good new therapeutic L-asparaginase? <em>World J. Microbiol. Biotechnol.</em><strong>35</strong>, 1–13 (2019).</cite> [<a href="https://doi.org/10.1007/s11274-019-2731-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31552479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beckett,%20A.%20&amp;%20Gervais,%20D.%20What%20makes%20a%20good%20new%20therapeutic%20L-asparaginase?%20World%20J.%20Microbiol.%20Biotechnol.35,%201%E2%80%9313%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Akram, F. et al. Abridgement of microbial esterases and their eminent industrial endeavors. <em>Mol Biotechnol</em> 1–17 (2024).</cite> [<a href="https://doi.org/10.1007/s12033-024-01108-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38461181/" class="usa-link">PubMed</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Sodhi, A. S. et al. Insights on sustainable approaches for production and applications of value added products. <em>Chemosphere</em><strong>286</strong>, 131623 (2022).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2021.131623" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34346348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sodhi,%20A.%20S.%20et%20al.%20Insights%20on%20sustainable%20approaches%20for%20production%20and%20applications%20of%20value%20added%20products.%20Chemosphere286,%20131623%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Diwan, D. et al. Elsevier,. Microbial cancer therapeutics: A promising approach. in <em>Seminars in cancer biology</em> vol. 86 931–950 (2022).</cite> [<a href="https://doi.org/10.1016/j.semcancer.2021.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33979677/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Sawant, S. S., Patil, S. M., Gupta, V. &amp; Kunda, N. K. Microbes as medicines: Harnessing the power of bacteria in advancing cancer treatment. <em>Int. J. Mol. Sci.</em><strong>21</strong>, 7575 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21207575" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7589870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33066447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sawant,%20S.%20S.,%20Patil,%20S.%20M.,%20Gupta,%20V.%20&amp;%20Kunda,%20N.%20K.%20Microbes%20as%20medicines:%20Harnessing%20the%20power%20of%20bacteria%20in%20advancing%20cancer%20treatment.%20Int.%20J.%20Mol.%20Sci.21,%207575%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Awad, M. F., El-Shenawy, F. S., El-Gendy, M. M. A. A. &amp; El-Bondkly, E. A. M. Purification, characterization, and anticancer and antioxidant activities of l-glutaminase from Aspergillus versicolor Faesay4. <em>Int. Microbiol.</em><strong>24</strong>, 169–181 (2021).
</cite> [<a href="https://doi.org/10.1007/s10123-020-00156-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33389217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Awad,%20M.%20F.,%20El-Shenawy,%20F.%20S.,%20El-Gendy,%20M.%20M.%20A.%20A.%20&amp;%20El-Bondkly,%20E.%20A.%20M.%20Purification,%20characterization,%20and%20anticancer%20and%20antioxidant%20activities%20of%20l-glutaminase%20from%20Aspergillus%20versicolor%20Faesay4.%20Int.%20Microbiol.24,%20169%E2%80%93181%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Abu-Tahon, M. A. &amp; Isaac, G. S. Purification, characterization and anticancer efficiency of L-glutaminase from Aspergillus flavus. <em>J. Gen. Appl. Microbiol.</em><strong>65</strong>, 284–292 (2019).
</cite> [<a href="https://doi.org/10.2323/jgam.2019.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31130583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abu-Tahon,%20M.%20A.%20&amp;%20Isaac,%20G.%20S.%20Purification,%20characterization%20and%20anticancer%20efficiency%20of%20L-glutaminase%20from%20Aspergillus%20flavus.%20J.%20Gen.%20Appl.%20Microbiol.65,%20284%E2%80%93292%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Matés, J. M., Campos-Sandoval, J. A. &amp; de Los Santos-Jiménez, J. M rquez, J. Dysregulation of glutaminase and glutamine synthetase in cancer. <em>Cancer Lett.</em><strong>467</strong>, 29–39 (2019).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2019.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31574293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mat%C3%A9s,%20J.%20M.,%20Campos-Sandoval,%20J.%20A.%20&amp;%20de%20Los%20Santos-Jim%C3%A9nez,%20J.%20M%20rquez,%20J.%20Dysregulation%20of%20glutaminase%20and%20glutamine%20synthetase%20in%20cancer.%20Cancer%20Lett.467,%2029%E2%80%9339%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Hassan, F. S. et al. Comprehensive insight into exploring the potential of microbial enzymes in cancer therapy: progress, challenges, and opportunities: A review. <em>Int J. Biol. Macromol</em><strong>134535</strong> (2024).</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.134535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39111467/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Crosby, H. A. &amp; Miller, K. E. Evaluating the analgesic effect of the GLS inhibitor 6-Diazo-5-Oxo-L-Norleucine in vivo. <em>Pharm Pharmacol. Int. J</em><strong>3</strong>, (2016).</cite> [<a href="https://doi.org/10.15406/ppij.2015.03.00055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5993434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29888760/" class="usa-link">PubMed</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Mendiratta, S. S., Sekulic, N., Lavie, A. &amp; Colley, K. J. Specific amino acids in the first fibronectin type III repeat of the neural cell adhesion molecule play a role in its recognition and polysialylation by the polysialyltransferase ST8Sia IV/PST. <em>J. Biol. Chem.</em><strong>280</strong>, 32340–32348 (2005).
</cite> [<a href="https://doi.org/10.1074/jbc.M506217200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16027151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mendiratta,%20S.%20S.,%20Sekulic,%20N.,%20Lavie,%20A.%20&amp;%20Colley,%20K.%20J.%20Specific%20amino%20acids%20in%20the%20first%20fibronectin%20type%20III%20repeat%20of%20the%20neural%20cell%20adhesion%20molecule%20play%20a%20role%20in%20its%20recognition%20and%20polysialylation%20by%20the%20polysialyltransferase%20ST8Sia%20IV/PST.%20J.%20Biol.%20Chem.280,%2032340%E2%80%9332348%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>pez-Gil, L., n, C. I., Téllez-Jurado, A., Velasco-Vel zquez, M. A. &amp; Anducho-Reyes, M. A. Identification and analysis of anticancer therapeutic targets from the polysaccharide Krestin (PSK) and polysaccharopeptide (PSP) using inverse Docking. <em>Molecules</em><strong>29</strong>, 5390 (2024).
</cite> [<a href="https://doi.org/10.3390/molecules29225390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11596896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39598781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?pez-Gil,%20L.,%20n,%20C.%20I.,%20T%C3%A9llez-Jurado,%20A.,%20Velasco-Vel%20zquez,%20M.%20A.%20&amp;%20Anducho-Reyes,%20M.%20A.%20Identification%20and%20analysis%20of%20anticancer%20therapeutic%20targets%20from%20the%20polysaccharide%20Krestin%20(PSK)%20and%20polysaccharopeptide%20(PSP)%20using%20inverse%20Docking.%20Molecules29,%205390%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Mates, J. M. et al. Glutaminase isoenzymes as key regulators in metabolic and oxidative stress against cancer. <em>Curr. Mol. Med.</em><strong>13</strong>, 514–534 (2013).
</cite> [<a href="https://doi.org/10.2174/1566524011313040005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22934847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mates,%20J.%20M.%20et%20al.%20Glutaminase%20isoenzymes%20as%20key%20regulators%20in%20metabolic%20and%20oxidative%20stress%20against%20cancer.%20Curr.%20Mol.%20Med.13,%20514%E2%80%93534%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Yang, W. H., Qiu, Y., Stamatatos, O., Janowitz, T. &amp; Lukey, M. J. Enhancing the efficacy of glutamine metabolism inhibitors in cancer therapy. <em>Trends Cancer</em>. <strong>7</strong>, 790–804 (2021).
</cite> [<a href="https://doi.org/10.1016/j.trecan.2021.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9064286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34020912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20W.%20H.,%20Qiu,%20Y.,%20Stamatatos,%20O.,%20Janowitz,%20T.%20&amp;%20Lukey,%20M.%20J.%20Enhancing%20the%20efficacy%20of%20glutamine%20metabolism%20inhibitors%20in%20cancer%20therapy.%20Trends%20Cancer.%207,%20790%E2%80%93804%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Elmetwalli, A. et al. Ammonia scavenger and glutamine synthetase inhibitors cocktail in targeting mTOR/β-catenin and MMP-14 for nitrogen homeostasis and liver cancer. <em>Med. Oncol.</em><strong>41</strong>, 38 (2023).
</cite> [<a href="https://doi.org/10.1007/s12032-023-02250-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38157146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elmetwalli,%20A.%20et%20al.%20Ammonia%20scavenger%20and%20glutamine%20synthetase%20inhibitors%20cocktail%20in%20targeting%20mTOR/%CE%B2-catenin%20and%20MMP-14%20for%20nitrogen%20homeostasis%20and%20liver%20cancer.%20Med.%20Oncol.41,%2038%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Youness, A., Kamel, R., Elkasabgy, R. A., Shao, N., Farag, A. &amp; P. &amp; Recent advances in Tannic acid (gallotannin) anticancer activities and drug delivery systems for efficacy improvement; a comprehensive review. <em>Molecules</em><strong>26</strong>, 1486 (2021).
</cite> [<a href="https://doi.org/10.3390/molecules26051486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7967207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33803294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Youness,%20A.,%20Kamel,%20R.,%20Elkasabgy,%20R.%20A.,%20Shao,%20N.,%20Farag,%20A.%20&amp;%20P.%20&amp;%20Recent%20advances%20in%20Tannic%20acid%20(gallotannin)%20anticancer%20activities%20and%20drug%20delivery%20systems%20for%20efficacy%20improvement;%20a%20comprehensive%20review.%20Molecules26,%201486%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Cai, Y. et al. Recent advances in anticancer activities and drug delivery systems of tannins. <em>Med. Res. Rev.</em><strong>37</strong>, 665–701 (2017).
</cite> [<a href="https://doi.org/10.1002/med.21422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28004409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cai,%20Y.%20et%20al.%20Recent%20advances%20in%20anticancer%20activities%20and%20drug%20delivery%20systems%20of%20tannins.%20Med.%20Res.%20Rev.37,%20665%E2%80%93701%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Ghasemian, M. et al. Recent progress in Tannic acid based approaches as a natural polyphenolic biomaterial for cancer therapy: A review. <em>Biomed. Pharmacother</em>. <strong>166</strong>, 115328 (2023).
</cite> [<a href="https://doi.org/10.1016/j.biopha.2023.115328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37591125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghasemian,%20M.%20et%20al.%20Recent%20progress%20in%20Tannic%20acid%20based%20approaches%20as%20a%20natural%20polyphenolic%20biomaterial%20for%20cancer%20therapy:%20A%20review.%20Biomed.%20Pharmacother.%20166,%20115328%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and/or analyzed during the current study are available in the GeneBank repository with accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/OR707005.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OR707005.1</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-14230-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_14230.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375786/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375786/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375786%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375786/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375786/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375786/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40850948/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375786/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40850948/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375786/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375786/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="EYpBCZiti9RTAio9iZ3IsyJscac4fRy4V4X9wkoWcoX8R8K2AAuj56X2vqOPPAGl">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
