
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AECC8AF310A305AECC00258F2DC6.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcbiology">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366216/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Biology">
<meta name="citation_title" content="Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals">
<meta name="citation_author" content="Thi Tuong Vi Van">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Trung Hieu Tran">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Hue Hanh Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Van Thien Chi Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Dac Ho Vo">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Giang Thi Huong Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Trong Hieu Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Kim Sang To">
<meta name="citation_author_institution" content="Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Anh Luan Nguyen">
<meta name="citation_author_institution" content="Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Cao Hong An Tran">
<meta name="citation_author_institution" content="Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thanh Xuan Jasmine">
<meta name="citation_author_institution" content="Medic Medical Center, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Loan Vo">
<meta name="citation_author_institution" content="Medic Medical Center, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Huong Thoang Nai">
<meta name="citation_author_institution" content="Medic Medical Center, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thuy Trang Tran">
<meta name="citation_author_institution" content="Medic Medical Center, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="My Hoang Truong">
<meta name="citation_author_institution" content="Medic Medical Center, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Ngan Chau Tran">
<meta name="citation_author_institution" content="Medic Medical Center, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Loc Le">
<meta name="citation_author_institution" content="Thai Nguyen National General Hospital, Thai Nguyen, Vietnam">
<meta name="citation_author" content="Thi Hong Nhung Nguyen">
<meta name="citation_author_institution" content="Thai Nguyen National General Hospital, Thai Nguyen, Vietnam">
<meta name="citation_author" content="Ngoc Hieu Tu">
<meta name="citation_author_institution" content="Buon Ma, Thuot Medical University Hospital, Buon Ma Thuot, Vietnam">
<meta name="citation_author" content="Thanh Son Tran">
<meta name="citation_author_institution" content="Buon Ma, Thuot Medical University Hospital, Buon Ma Thuot, Vietnam">
<meta name="citation_author" content="Bao Toan Le">
<meta name="citation_author_institution" content="Can Tho Oncology Hospital, Can Tho, Vietnam">
<meta name="citation_author" content="Van Phong Tang">
<meta name="citation_author_institution" content="Can Tho Oncology Hospital, Can Tho, Vietnam">
<meta name="citation_author" content="Pham Thanh Nhan Nguyen">
<meta name="citation_author_institution" content="Da Nang Oncology Hospital, Da Nang, Vietnam">
<meta name="citation_author" content="Khac Tien Nguyen">
<meta name="citation_author_institution" content="Nghe An Oncology Hospital, Nghe An, Vietnam">
<meta name="citation_author" content="Van Chien Ho">
<meta name="citation_author_institution" content="Nghe An Oncology Hospital, Nghe An, Vietnam">
<meta name="citation_author" content="Xuan Vinh Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Nhu Nhat Tan Doan">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Trang Tran">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Minh Thu Tran">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Vu Uyen Tran">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Minh Phong Le">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Luyen Vu">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Ba Linh Tieu">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Huu Tam Phuc Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Luu Hong Dang Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Ngoc Minh Phan">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Van Phan">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Thanh Thuy Do">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Thi Huyen Dao">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Hung Sang Tang">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Duy Sinh Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Hoa Giang">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Minh Duy Phan">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Hoai-Nghia Nguyen">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Duc Hieu Vo">
<meta name="citation_author_institution" content="Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam">
<meta name="citation_author" content="Le Son Tran">
<meta name="citation_author_institution" content="Medical Genetics Institute, Ho Chi Minh, Vietnam">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="23">
<meta name="citation_firstpage" content="259">
<meta name="citation_doi" content="10.1186/s12915-025-02371-z">
<meta name="citation_pmid" content="40830871">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366216/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366216/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366216/pdf/12915_2025_Article_2371.pdf">
<meta name="description" content="Breast cancer (BC) remains the second leading cause of cancer-related mortality among women worldwide. Liquid biopsy based on circulating tumor DNA (ctDNA) offers a promising noninvasive approach for early detection; however, differentiating ...">
<meta name="og:title" content="Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Breast cancer (BC) remains the second leading cause of cancer-related mortality among women worldwide. Liquid biopsy based on circulating tumor DNA (ctDNA) offers a promising noninvasive approach for early detection; however, differentiating ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366216/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366216">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12915-025-02371-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12915_2025_Article_2371.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366216%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366216/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366216/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366216/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcbiology.png" alt="BMC Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Biology" title="Link to BMC Biology" shape="default" href="https://bmcbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Biol</button></div>. 2025 Aug 20;23:259. doi: <a href="https://doi.org/10.1186/s12915-025-02371-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12915-025-02371-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20TTV%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Thi Tuong Vi Van</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Thi Tuong Vi Van</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20TTV%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Tuong Vi Van</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TH%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Trung Hieu Tran</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Trung Hieu Tran</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TH%22%5BAuthor%5D" class="usa-link"><span class="name western">Trung Hieu Tran</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20THH%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Thi Hue Hanh Nguyen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Thi Hue Hanh Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20THH%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Hue Hanh Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20VTC%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Van Thien Chi Nguyen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Van Thien Chi Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20VTC%22%5BAuthor%5D" class="usa-link"><span class="name western">Van Thien Chi Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vo%20DH%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Dac Ho Vo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Dac Ho Vo</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vo%20DH%22%5BAuthor%5D" class="usa-link"><span class="name western">Dac Ho Vo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20GTH%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Giang Thi Huong Nguyen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Giang Thi Huong Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20GTH%22%5BAuthor%5D" class="usa-link"><span class="name western">Giang Thi Huong Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20TH%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Trong Hieu Nguyen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Trong Hieu Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20TH%22%5BAuthor%5D" class="usa-link"><span class="name western">Trong Hieu Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22To%20KS%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Kim Sang To</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Kim Sang To</span></h3>
<div class="p">
<sup>2</sup>Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22To%20KS%22%5BAuthor%5D" class="usa-link"><span class="name western">Kim Sang To</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20AL%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Anh Luan Nguyen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Anh Luan Nguyen</span></h3>
<div class="p">
<sup>2</sup>Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20AL%22%5BAuthor%5D" class="usa-link"><span class="name western">Anh Luan Nguyen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20CHA%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Cao Hong An Tran</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Cao Hong An Tran</span></h3>
<div class="p">
<sup>2</sup>Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20CHA%22%5BAuthor%5D" class="usa-link"><span class="name western">Cao Hong An Tran</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jasmine%20TX%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Thanh Xuan Jasmine</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Thanh Xuan Jasmine</span></h3>
<div class="p">
<sup>3</sup>Medic Medical Center, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jasmine%20TX%22%5BAuthor%5D" class="usa-link"><span class="name western">Thanh Xuan Jasmine</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vo%20TL%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Thi Loan Vo</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Thi Loan Vo</span></h3>
<div class="p">
<sup>3</sup>Medic Medical Center, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vo%20TL%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Loan Vo</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nai%20THT%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Thi Huong Thoang Nai</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Thi Huong Thoang Nai</span></h3>
<div class="p">
<sup>3</sup>Medic Medical Center, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nai%20THT%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Huong Thoang Nai</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TT%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Thuy Trang Tran</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Thuy Trang Tran</span></h3>
<div class="p">
<sup>3</sup>Medic Medical Center, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TT%22%5BAuthor%5D" class="usa-link"><span class="name western">Thuy Trang Tran</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Truong%20MH%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">My Hoang Truong</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">My Hoang Truong</span></h3>
<div class="p">
<sup>3</sup>Medic Medical Center, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Truong%20MH%22%5BAuthor%5D" class="usa-link"><span class="name western">My Hoang Truong</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20NC%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Ngan Chau Tran</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Ngan Chau Tran</span></h3>
<div class="p">
<sup>3</sup>Medic Medical Center, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20NC%22%5BAuthor%5D" class="usa-link"><span class="name western">Ngan Chau Tran</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20TL%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Thi Loc Le</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Thi Loc Le</span></h3>
<div class="p">
<sup>4</sup>Thai Nguyen National General Hospital, Thai Nguyen, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20TL%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Loc Le</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20THN%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Thi Hong Nhung Nguyen</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Thi Hong Nhung Nguyen</span></h3>
<div class="p">
<sup>4</sup>Thai Nguyen National General Hospital, Thai Nguyen, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20THN%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Hong Nhung Nguyen</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tu%20NH%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Ngoc Hieu Tu</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Ngoc Hieu Tu</span></h3>
<div class="p">
<sup>5</sup>Buon Ma, Thuot Medical University Hospital, Buon Ma Thuot, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tu%20NH%22%5BAuthor%5D" class="usa-link"><span class="name western">Ngoc Hieu Tu</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TS%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Thanh Son Tran</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Thanh Son Tran</span></h3>
<div class="p">
<sup>5</sup>Buon Ma, Thuot Medical University Hospital, Buon Ma Thuot, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TS%22%5BAuthor%5D" class="usa-link"><span class="name western">Thanh Son Tran</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20BT%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Bao Toan Le</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Bao Toan Le</span></h3>
<div class="p">
<sup>6</sup>Can Tho Oncology Hospital, Can Tho, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20BT%22%5BAuthor%5D" class="usa-link"><span class="name western">Bao Toan Le</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20VP%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Van Phong Tang</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Van Phong Tang</span></h3>
<div class="p">
<sup>6</sup>Can Tho Oncology Hospital, Can Tho, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20VP%22%5BAuthor%5D" class="usa-link"><span class="name western">Van Phong Tang</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20PTN%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Pham Thanh Nhan Nguyen</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Pham Thanh Nhan Nguyen</span></h3>
<div class="p">
<sup>7</sup>Da Nang Oncology Hospital, Da Nang, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20PTN%22%5BAuthor%5D" class="usa-link"><span class="name western">Pham Thanh Nhan Nguyen</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20KT%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Khac Tien Nguyen</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Khac Tien Nguyen</span></h3>
<div class="p">
<sup>8</sup>Nghe An Oncology Hospital, Nghe An, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20KT%22%5BAuthor%5D" class="usa-link"><span class="name western">Khac Tien Nguyen</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20VC%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Van Chien Ho</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Van Chien Ho</span></h3>
<div class="p">
<sup>8</sup>Nghe An Oncology Hospital, Nghe An, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20VC%22%5BAuthor%5D" class="usa-link"><span class="name western">Van Chien Ho</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20XV%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Xuan Vinh Nguyen</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Xuan Vinh Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20XV%22%5BAuthor%5D" class="usa-link"><span class="name western">Xuan Vinh Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doan%20NNT%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Nhu Nhat Tan Doan</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Nhu Nhat Tan Doan</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doan%20NNT%22%5BAuthor%5D" class="usa-link"><span class="name western">Nhu Nhat Tan Doan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TT%22%5BAuthor%5D" class="usa-link" aria-describedby="id28"><span class="name western">Thi Trang Tran</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Thi Trang Tran</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TT%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Trang Tran</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TMT%22%5BAuthor%5D" class="usa-link" aria-describedby="id29"><span class="name western">Thi Minh Thu Tran</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Thi Minh Thu Tran</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TMT%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Minh Thu Tran</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20VU%22%5BAuthor%5D" class="usa-link" aria-describedby="id30"><span class="name western">Vu Uyen Tran</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Vu Uyen Tran</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20VU%22%5BAuthor%5D" class="usa-link"><span class="name western">Vu Uyen Tran</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20MP%22%5BAuthor%5D" class="usa-link" aria-describedby="id31"><span class="name western">Minh Phong Le</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Minh Phong Le</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20MP%22%5BAuthor%5D" class="usa-link"><span class="name western">Minh Phong Le</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vu%20TL%22%5BAuthor%5D" class="usa-link" aria-describedby="id32"><span class="name western">Thi Luyen Vu</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Thi Luyen Vu</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vu%20TL%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Luyen Vu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tieu%20BL%22%5BAuthor%5D" class="usa-link" aria-describedby="id33"><span class="name western">Ba Linh Tieu</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">Ba Linh Tieu</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tieu%20BL%22%5BAuthor%5D" class="usa-link"><span class="name western">Ba Linh Tieu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20HTP%22%5BAuthor%5D" class="usa-link" aria-describedby="id34"><span class="name western">Huu Tam Phuc Nguyen</span></a><div hidden="hidden" id="id34">
<h3><span class="name western">Huu Tam Phuc Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20HTP%22%5BAuthor%5D" class="usa-link"><span class="name western">Huu Tam Phuc Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20LHD%22%5BAuthor%5D" class="usa-link" aria-describedby="id35"><span class="name western">Luu Hong Dang Nguyen</span></a><div hidden="hidden" id="id35">
<h3><span class="name western">Luu Hong Dang Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20LHD%22%5BAuthor%5D" class="usa-link"><span class="name western">Luu Hong Dang Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phan%20NM%22%5BAuthor%5D" class="usa-link" aria-describedby="id36"><span class="name western">Ngoc Minh Phan</span></a><div hidden="hidden" id="id36">
<h3><span class="name western">Ngoc Minh Phan</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phan%20NM%22%5BAuthor%5D" class="usa-link"><span class="name western">Ngoc Minh Phan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Phan%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id37"><span class="name western">Thi Van Phan</span></a><div hidden="hidden" id="id37">
<h3><span class="name western">Thi Van Phan</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Phan%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Van Phan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Do%20TTT%22%5BAuthor%5D" class="usa-link" aria-describedby="id38"><span class="name western">Thi Thanh Thuy Do</span></a><div hidden="hidden" id="id38">
<h3><span class="name western">Thi Thanh Thuy Do</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Do%20TTT%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Thanh Thuy Do</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dao%20TH%22%5BAuthor%5D" class="usa-link" aria-describedby="id39"><span class="name western">Thi Huyen Dao</span></a><div hidden="hidden" id="id39">
<h3><span class="name western">Thi Huyen Dao</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dao%20TH%22%5BAuthor%5D" class="usa-link"><span class="name western">Thi Huyen Dao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20HS%22%5BAuthor%5D" class="usa-link" aria-describedby="id40"><span class="name western">Hung Sang Tang</span></a><div hidden="hidden" id="id40">
<h3><span class="name western">Hung Sang Tang</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20HS%22%5BAuthor%5D" class="usa-link"><span class="name western">Hung Sang Tang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20DS%22%5BAuthor%5D" class="usa-link" aria-describedby="id41"><span class="name western">Duy Sinh Nguyen</span></a><div hidden="hidden" id="id41">
<h3><span class="name western">Duy Sinh Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20DS%22%5BAuthor%5D" class="usa-link"><span class="name western">Duy Sinh Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Giang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id42"><span class="name western">Hoa Giang</span></a><div hidden="hidden" id="id42">
<h3><span class="name western">Hoa Giang</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Giang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hoa Giang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phan%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id43"><span class="name western">Minh Duy Phan</span></a><div hidden="hidden" id="id43">
<h3><span class="name western">Minh Duy Phan</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phan%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Minh Duy Phan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20HN%22%5BAuthor%5D" class="usa-link" aria-describedby="id44"><span class="name western">Hoai-Nghia Nguyen</span></a><div hidden="hidden" id="id44">
<h3><span class="name western">Hoai-Nghia Nguyen</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20HN%22%5BAuthor%5D" class="usa-link"><span class="name western">Hoai-Nghia Nguyen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vo%20DH%22%5BAuthor%5D" class="usa-link" aria-describedby="id45"><span class="name western">Duc Hieu Vo</span></a><div hidden="hidden" id="id45">
<h3><span class="name western">Duc Hieu Vo</span></h3>
<div class="p">
<sup>2</sup>Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vo%20DH%22%5BAuthor%5D" class="usa-link"><span class="name western">Duc Hieu Vo</span></a>
</div>
</div>
<sup>2,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20LS%22%5BAuthor%5D" class="usa-link" aria-describedby="id46"><span class="name western">Le Son Tran</span></a><div hidden="hidden" id="id46">
<h3><span class="name western">Le Son Tran</span></h3>
<div class="p">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20LS%22%5BAuthor%5D" class="usa-link"><span class="name western">Le Son Tran</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Medical Genetics Institute, Ho Chi Minh, Vietnam </div>
<div id="Aff2">
<sup>2</sup>Ho Chi Minh City Oncology Hospital, Ho Chi Minh, Vietnam </div>
<div id="Aff3">
<sup>3</sup>Medic Medical Center, Ho Chi Minh, Vietnam </div>
<div id="Aff4">
<sup>4</sup>Thai Nguyen National General Hospital, Thai Nguyen, Vietnam </div>
<div id="Aff5">
<sup>5</sup>Buon Ma, Thuot Medical University Hospital, Buon Ma Thuot, Vietnam </div>
<div id="Aff6">
<sup>6</sup>Can Tho Oncology Hospital, Can Tho, Vietnam </div>
<div id="Aff7">
<sup>7</sup>Da Nang Oncology Hospital, Da Nang, Vietnam </div>
<div id="Aff8">
<sup>8</sup>Nghe An Oncology Hospital, Nghe An, Vietnam </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 20; Accepted 2025 Aug 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366216Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830871/" class="usa-link">40830871</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background
</h3>
<p id="Par12">Breast cancer (BC) remains the second leading cause of cancer-related mortality among women worldwide. Liquid biopsy based on circulating tumor DNA (ctDNA) offers a promising noninvasive approach for early detection; however, differentiating malignant tumors from benign abnormalities remains a significant challenge.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par13">Here, we developed a multimodal approach to analyze cfDNA methylation and fragmentomic patterns in 273 BC patients, 108 individuals with benign breast conditions, and 134 healthy controls. Genome-wide analyses revealed distinct cfDNA copy number alterations and cytosine-enriched cleavage sites in BC patients. Targeted sequencing further revealed unique methylation patterns, including hypermethylation in <em>GPR126</em>, <em>KLF3</em>, and <em>TLR10</em> and hypomethylation in <em>TOP1</em> and <em>MAFB</em>. Our machine-learning model achieved an AUC of 0.90, with 93.6% specificity and 62.1â€“66.3% sensitivity for stage Iâ€“II cancers. In symptomatic populations, sensitivities were 50.0%, 68.2%, and 64.7% for BI-RADS categories 3, 4, and 5, respectively, with 96.1% specificity.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par14">These findings underscore the potential of cfDNA biomarkers to enhance BC detection and reduce the rate of unnecessary biopsies.
</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12915-025-02371-z.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Breast cancer, CfDNA, Benign abnormalities, Methylation and fragmentomic</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par41">Breast cancer (BC) is the second most frequent cancer globally, accounting for 11.6% of all cancer cases in 2022 [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. It is the most common malignancy among women, with about 2.3 million new cases and 660,000 deaths reported in the same year [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Projections indicate that by 2040, the global burden of BC will surpass 3 million new cases and 1 million deaths annually [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Early detection is critical for improving prognosis, survival rates, and overall patient outcomes by facilitating timely clinical interventions [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Therefore, advancing and optimizing effective screening and diagnostic techniques for BC is essential to improving survival outcomes.</p>
<p id="Par42">Mammography and ultrasonography are the most widely employed methods for BC detection, with mammography recognized as the gold standard for early detection [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Despite its proven effectiveness, mammography has several limitations. These include reduced tumor visibility in dense fibroglandular tissue and the risks associated with repeated radiation exposure [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Additionally, the sensitivity of mammography is approximately 86.9%, leaving certain BC types undetected [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Some breast tumors, such as benign lumps, integrate into the natural breast architecture, further complicating detection. In young women and those with dense breast tissue, mammography often yields a high false-positive rate, leading to unnecessary follow-up procedures, including biopsies, as recommended by clinical guidelines [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Among benign breast conditions, fibroadenomas are the most common, followed by fibrocystic disease [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Fibroadenomas present imaging characteristics that overlap with a variety of other benign and malignant breast lesions, contributing to their significant representation among breast biopsies [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>].
</p>
<p id="Par43">Magnetic resonance imaging (MRI) has been proposed as a promising alternative to mammography, particularly because it eliminates the risks of radiation exposure [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. However, its application in clinical practice is limited by factors such as low specificity, complex image interpretation, and the absence of standardized diagnostic guidelines. As a result, breast MRI is generally reserved for women at high risk of BC [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. The Breast Imaging Reporting and Data System (BI-RADS) categories are frequently utilized alongside imaging modalities to evaluate malignancy risk. These categories provide a spectrum of risk, ranging fromâ€‰&gt;â€‰0 toâ€‰â‰¤â€‰2% for BI-RADS 3,â€‰&gt;â€‰2 toâ€‰&lt;â€‰95% for BI-RADS 4, andâ€‰â‰¥â€‰95% for BI-RADS 5 [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Despite their utility, distinguishing benign breast lesions from malignant tumors remains a significant challenge in clinical practice, underscoring the need for improved diagnostic tools and techniques.</p>
<p id="Par44">The limitations of current BC diagnostic methods underscore the need for non-invasive approaches capable of accurately diagnosing BC and differentiating between malignant and benign tumors. In this context, liquid biopsy has emerged as a promising diagnostic method, offering higher sensitivity and specificity for early BC detection [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Liquid biopsy relies on the analysis of epigenetic and genetic alterations in circulating tumor cells (CTCs), circulating tumor DNA (ctDNA), cell-free DNA (cfDNA), mRNAs, microRNAs (miRNAs), and proteins, which can be detected in peripheral blood or other body fluids using advanced technologies [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Several studies have demonstrated the potential of ctDNA as a biomarker for identifying specific mutations associated with BC. However, the sensitivity of mutation-based ctDNA detection methods remains limited due to the lack of universally prevalent mutations in BC [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Recent advancements in DNA methylation analysis within liquid biopsies offer a promising alternative. Methylation markers are particularly advantageous due to their stability, relevance across diverse cancer types, and ability to provide comprehensive insights into cancer biology, including early detection [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Despite these advantages, most methylation-based studies have not included benign lesions, and the overall performance in BC detection remains suboptimal. High rates of false positives and false negatives persist, largely due to benign conditions that can release DNA with methylation or mutation patterns resembling those of cancer, thereby reducing specificity [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>].</p>
<p id="Par45">To address these challenges, we previously developed the SPOT-MAS (Screening for the Presence of Tumor by DNA Methylation and Size) multimodal assay to analyze genome-wide cfDNA methylomics and fragmentomics, with the goal of enhancing BC detection rates [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. However, our previous study did not fully address the differentiation of BC from benign lesions, which limited its applicability for general population screening, where patients with benign lesions are common [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par46">In the current study, we expanded the feature space by incorporating previously unexplored features, including specific 21-mer end motifs around cfDNA cleavage sites, as well as methylation patterns and copy number aberrations across 450 targeted genomic regions. Importantly, we conducted pairwise comparisons between 273 non-metastatic BC patients and 108 patients with benign lesions, as well as 134 healthy individuals, to identify multiple BC-specific signatures. These signatures were then used to build a classifier model to improve the differentiation of early-stage BC from both healthy individuals and patients with benign lesions (Fig. <a href="#Fig1" class="usa-link">1</a>). To further demonstrate the utility of our model, we assessed its performance in detecting BC using an external validation cohort of patients diagnosed with BI-RADS 3â€“5 lesions through standard-of-care (SOC) imaging tests (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366216_12915_2025_2371_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbff/12366216/47da867cecd7/12915_2025_2371_Fig1_HTML.jpg" loading="lazy" id="MO1" height="1036" width="708" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of study design. The plasma samples in the discovery cohort underwent the SPOT-MAS assay. Features of cfDNA (Methylation, End motif, Fragment length, Copy number) were collected from both the target and the genome-wide fraction after sequencing. A multi-feature model was developed using these features to distinguish BC patients from non-cancer (benign and healthy) individuals. To validate this model, we recruited 119 symptomatic participants who had been diagnosed with BI-RADS 3-5 by mammography. The participants in the validation cohort also underwent the SPOT-MAS assay, which was similar to those in the discovery cohort and confirmed by fine needle aspiration (FNA) afterward</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Clinical characteristics of BC patients, individuals with benign breast disease and healthy participants in the discovery cohort</h3>
<p id="Par47">We recruited 273 BC patients, 108 patients with benign breast lumps, and 134 healthy subjects as a discovery cohort to profile cfDNA signatures capable of differentiating BC patients from non-cancer individuals (Table <a href="#Tab1" class="usa-link">1</a>). All BC patients were diagnosed following abnormal imaging results (mammography or ultrasound), with malignancy confirmed through subsequent tissue biopsy. Patients in the benign group were confirmed to have non-malignant breast conditions, such as fibroadenomas or fibrocystic changes. Both benign patients and healthy participants underwent mammography as part of their annual health check-ups and were monitored for 12 months to confirm their cancer-free status. The clinical characteristics of participants in the BC, benign, and control groups are summarized in Table <a href="#Tab1" class="usa-link">1</a> and detailed in Additional file 1:Table S1.
</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of the clinical features of participants in discovery cohort</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="5" colspan="2">Clinical Information</th>
<th align="left" colspan="10" rowspan="1"><strong>Discovery cohort (BC: 273; Benign: 108; Healthy: 134)</strong></th>
</tr>
<tr>
<th align="left" colspan="5" rowspan="1"><strong>Train set</strong></th>
<th align="left" colspan="5" rowspan="1"><strong>Test set</strong></th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">BC</th>
<th align="left" colspan="2" rowspan="1">Noncancer</th>
<th align="left" rowspan="2" colspan="2">
<em>p</em>-value(Mann-Whitney U test)</th>
<th align="left" colspan="1" rowspan="1">BC</th>
<th align="left" colspan="2" rowspan="1">Noncancer</th>
<th align="left" rowspan="2" colspan="2">
<em>p</em>-value(Mann-Whitney U test)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">BC<em>N</em>=143</th>
<th align="left" colspan="1" rowspan="1">Benign<em>N</em>=52</th>
<th align="left" colspan="1" rowspan="1">Healthy<em>N</em>=65</th>
<th align="left" colspan="1" rowspan="1">BC<em>N</em>=130</th>
<th align="left" colspan="1" rowspan="1">Benign<em>N</em>=56</th>
<th align="left" colspan="1" rowspan="1">Healthy<em>N</em>=69</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">N(%)</th>
<th align="left" colspan="1" rowspan="1">N(%)</th>
<th align="left" colspan="1" rowspan="1">N(%)</th>
<th align="left" colspan="1" rowspan="1">BC vs. Benign</th>
<th align="left" colspan="1" rowspan="1">BC vs. Healthy</th>
<th align="left" colspan="1" rowspan="1">N(%)</th>
<th align="left" colspan="1" rowspan="1">N(%)</th>
<th align="left" colspan="1" rowspan="1">N(%)</th>
<th align="left" colspan="1" rowspan="1">BC vs. Benign</th>
<th align="left" colspan="1" rowspan="1">BC vs. Healthy</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="3" colspan="1">Age</td>
<td align="left" colspan="1" rowspan="1">Median</td>
<td align="left" colspan="1" rowspan="1">49.0</td>
<td align="left" colspan="1" rowspan="1">39.5</td>
<td align="left" colspan="1" rowspan="1">49.0</td>
<td align="left" rowspan="3" colspan="1">4.47E-06</td>
<td align="left" rowspan="3" colspan="1">1.00E+00</td>
<td align="left" colspan="1" rowspan="1">53.5</td>
<td align="left" colspan="1" rowspan="1">41.5</td>
<td align="left" colspan="1" rowspan="1">49.0</td>
<td align="left" rowspan="3" colspan="1">5.69E-05</td>
<td align="left" rowspan="3" colspan="1">7.95E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Min</td>
<td align="left" colspan="1" rowspan="1">28.0</td>
<td align="left" colspan="1" rowspan="1">22.0</td>
<td align="left" colspan="1" rowspan="1">25.0</td>
<td align="left" colspan="1" rowspan="1">24.0</td>
<td align="left" colspan="1" rowspan="1">20.0</td>
<td align="left" colspan="1" rowspan="1">22.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Max</td>
<td align="left" colspan="1" rowspan="1">75.0</td>
<td align="left" colspan="1" rowspan="1">68.0</td>
<td align="left" colspan="1" rowspan="1">67.0</td>
<td align="left" colspan="1" rowspan="1">77.0</td>
<td align="left" colspan="1" rowspan="1">70.0</td>
<td align="left" colspan="1" rowspan="1">70.0</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">Stage</td>
<td align="left" colspan="1" rowspan="1">I, IA, IB</td>
<td align="left" colspan="1" rowspan="1">34(23.8)</td>
<td align="left" rowspan="4" colspan="1"></td>
<td align="left" rowspan="4" colspan="1"></td>
<td align="left" rowspan="4" colspan="2"></td>
<td align="left" colspan="1" rowspan="1">29 (22.3)</td>
<td align="left" rowspan="4" colspan="1"></td>
<td align="left" rowspan="4" colspan="1"></td>
<td align="left" rowspan="4" colspan="2"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">II, IIA, IIB</td>
<td align="left" colspan="1" rowspan="1">77(53.8)</td>
<td align="left" colspan="1" rowspan="1">83 (63.8)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">III, IIIA</td>
<td align="left" colspan="1" rowspan="1">21(14.7)</td>
<td align="left" colspan="1" rowspan="1">15 (11.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">11(7.7)</td>
<td align="left" colspan="1" rowspan="1">3 (2.3)</td>
</tr>
<tr>
<td align="left" rowspan="6" colspan="1">Subtypes</td>
<td align="left" colspan="1" rowspan="1">Luminal A</td>
<td align="left" colspan="1" rowspan="1">27(18.9)</td>
<td align="left" rowspan="6" colspan="1"></td>
<td align="left" rowspan="6" colspan="1"></td>
<td align="left" rowspan="6" colspan="2"></td>
<td align="left" colspan="1" rowspan="1">22 (16.9)</td>
<td align="left" rowspan="6" colspan="1"></td>
<td align="left" rowspan="6" colspan="1"></td>
<td align="left" rowspan="6" colspan="2"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Luminal B</td>
<td align="left" colspan="1" rowspan="1">43(30.1)</td>
<td align="left" colspan="1" rowspan="1">46 (35.4)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Luminal B-HER2</td>
<td align="left" colspan="1" rowspan="1">27(18.9)</td>
<td align="left" colspan="1" rowspan="1">25 (19.2)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HER2</td>
<td align="left" colspan="1" rowspan="1">22(15.4)</td>
<td align="left" colspan="1" rowspan="1">21 (16.2)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TNBC</td>
<td align="left" colspan="1" rowspan="1">14(9.8)</td>
<td align="left" colspan="1" rowspan="1">11(8.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">10(7.0)</td>
<td align="left" colspan="1" rowspan="1">5(3.8)</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Subtypes_Benign</td>
<td align="left" colspan="1" rowspan="1">Fibroadenomas</td>
<td align="left" rowspan="2" colspan="1"></td>
<td align="left" colspan="1" rowspan="1">36(69.2)</td>
<td align="left" rowspan="2" colspan="1"></td>
<td align="left" rowspan="2" colspan="2"></td>
<td align="left" rowspan="2" colspan="1"></td>
<td align="left" colspan="1" rowspan="1">38(67.9)</td>
<td align="left" rowspan="2" colspan="1"></td>
<td align="left" rowspan="2" colspan="2"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fibrocystic changes</td>
<td align="left" colspan="1" rowspan="1">16(30.8)</td>
<td align="left" colspan="1" rowspan="1">18(32.1)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par48">Participants in the discovery cohort were randomly divided into a training set and a test set (Table <a href="#Tab1" class="usa-link">1</a>). The training set included 143 BC patients, 52 benign patients, and 65 healthy individuals. In this set, both BC patients and healthy participants had a median age of 49 years (<em>p</em>-valueâ€‰=â€‰1.00, Mannâ€“Whitney <em>U</em> test). By contrast, the benign group had a median age of 39.5 years (range 22â€“68), which was significantly younger than the BC group (<em>p</em>-valueâ€‰â‰¤â€‰0.05, Mannâ€“Whitney <em>U</em> test). Most BC patients in the training set were diagnosed at early stages, with 23.8% at stage I, 53.8% at stage II, and 14.7% at stage IIIA. Staging information was unavailable for 7.7% of patients, though all were confirmed by specialists to have non-metastatic tumors. Regarding BC subtypes, 18.9% were luminal A, 30.1% luminal B, 18.9% luminal B-HER2, 15.4% HER2, and 9.8% triple-negative breast cancer (TNBC). In the benign group, 69.2% were diagnosed with fibroadenomas and 30.8% with fibrocystic changes.</p>
<p id="Par49">The test set consisted of 130 BC patients, 56 benign patients, and 69 healthy individuals (Table <a href="#Tab1" class="usa-link">1</a>). The median ages of participants in the BC, benign, and healthy groups in the test set were comparable to those in the training set. Similar to the training set, most BC patients in the test set presented with early-stage tumors, with 22.3% at stage I and 63.8% at stage II. The distributions of BC subtypes and benign conditions in the test set were consistent with those in the training set (Table <a href="#Tab1" class="usa-link">1</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Shallow genome-wide sequencing reveals distinct CNA and motif end signatures in cfDNA of BC patients compared to benign and healthy individuals</h3>
<p id="Par50">Our prior research using shallow genome-wide sequencing revealed distinct plasma CNA and EM signatures in BC patients compared to healthy individuals [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. However, a comparative analysis between BC and benign patients has not been previously conducted. To identify ctDNA signatures capable of distinguishing BC from both healthy and benign individuals, we examined overlapping significant CNA and EM features across two pairwise comparisons: BC versus benign and BC versus healthy.</p>
<p id="Par51">To investigate the CNA patterns in cfDNA sequencing reads from the whole-genome fraction were aligned to the human reference genome, segmented into 2691 bins of 1 Mb each. The DNA copy number was calculated for each bin. In BC patients compared to benign patients, we identified 863 bins with a significant gain and 592 bins with a significant loss across 22 chromosomes in BC patients (<em>p</em>-valueâ€‰â‰¤â€‰0.05, Mannâ€“Whitney <em>U</em> test, Benjaminiâ€“Hochberg correction, Fig. <a href="#Fig2" class="usa-link">2</a>A, C). When comparing CNA values between BC patients and healthy individuals, we identified only 5 bins with a significant gain and 4 bins with a significant loss in BC patients (<em>p</em>-valueâ€‰â‰¤â€‰0.05, Mannâ€“Whitney <em>U</em> test, Benjaminiâ€“Hochberg correction, Fig. <a href="#Fig2" class="usa-link">2</a>B, D). Notably, 5 bins overlapped between the two pairwise comparisons (Fig. <a href="#Fig2" class="usa-link">2</a>E). Among these, two bins with a significant gain were located on chromosomes 7 and 16 (Fig. <a href="#Fig2" class="usa-link">2</a>F), while three bins with a significant loss were observed on chromosomes 4, 5, and 12 in BC patients (Fig. <a href="#Fig2" class="usa-link">2</a>G).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366216_12915_2025_2371_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbff/12366216/f3a1090b458e/12915_2025_2371_Fig2_HTML.jpg" loading="lazy" id="MO2" height="739" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis of genome-wide copy number aberration (CNA) in cfDNA. Scatter plot shows log2 fold change of DNA copy number in each bin across 22 chromosomes of 273 BC patients versus 108 benign patients (<strong>A</strong>) or 134 healthy subjects (<strong>B</strong>) in the discovery cohort. Each dot represents a bin identified as gain (red), loss (blue), or no change (grey) in the copy number. Proportions of different CNA bins in each chromosome for BC versus benign patients (<strong>C</strong>) and BC versus healthy individuals (<strong>D</strong>). <strong>E</strong> Venn diagram indicates the number of significant bins overlapped between two pair-wise comparisons (BC versus benign and BC versus healthy). Boxplots showing 2 gain bins (<strong>F</strong>) and 3 loss bins (<strong>G</strong>) of copy number in BC patients compared to benign or healthy individuals</p></figcaption></figure><p id="Par52">We next compared the frequencies of all 256 possible 4-mer EMs in cfDNA fragments from BC, benign, and healthy individuals. We identified 18 EMs that were differentially enriched in BC cfDNA compared to benign or healthy individuals (Fig. <a href="#Fig3" class="usa-link">3</a>A). Of these, 15 EMs displayed consistent enrichment patterns across both pairwise comparisons (<em>p</em>-valueâ€‰â‰¤â€‰0.05, Mannâ€“Whitney <em>U</em> test, Benjaminiâ€“Hochberg correction, Fig. <a href="#Fig3" class="usa-link">3</a>A). Among the differentially enriched EMs, 8 (highlighted in red, Fig. <a href="#Fig3" class="usa-link">3</a>A) were more frequently observed in BC patients than in benign or healthy individuals, while 7 (highlighted in blue, Fig. <a href="#Fig3" class="usa-link">3</a>A) were less frequently observed in BC patients compared to these groups. Interestingly, all EMs significantly enriched in BC tended to start with cytosine (C), including CGCC, CCCC, CGCT, CGAC, CGCA, CCCA, CCAC, and CCAG (Fig. <a href="#Fig3" class="usa-link">3</a>A, B). In contrast, 5 out of 7 EMs with reduced frequencies in BC began with guanine (G), including GTGT, GTTA, GTTC, GTTG, and GTTT, while the remaining 2 EMs began with cytosine (C) and thymine (T): CTTG and TTTG (Fig. <a href="#Fig3" class="usa-link">3</a>A, C).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366216_12915_2025_2371_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbff/12366216/b12175d73fa7/12915_2025_2371_Fig3_HTML.jpg" loading="lazy" id="MO3" height="975" width="779" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distinct end motif patterns of plasma cfDNA in BC, benign and healthy individuals. <strong>A</strong> Heatmap shows log2 fold change of 256 4-mer end motifs in BC patients compared to benign or healthy subjects (end motifs were highlighted in red for higher frequency and blue for lower frequency in BC patients). Boxplots showing 8 increased EMs (<strong>B</strong>) and 7 decreased EMs (<strong>C</strong>) of copy number in BC patients compared to benign patients or healthy controls. <strong>D</strong> Heatmap indicates log2 fold change of motif 21 between BC versus benign patients and BC versus healthy subjects. <strong>E</strong> Boxplots show 8 significant ME21s in BC patients compared to benign or healthy individuals</p></figcaption></figure><p id="Par53">Beyond the 4-mer EMs, a previous study [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] demonstrated that nucleotide motifs in regions 10 bp downstream and upstream of cfDNA cleavage sites are consistently conserved in healthy controls, while tumor-derived cfDNA fragments exhibit aberrant changes in these motifs. Building on this foundation, we investigated the nucleotide composition (A/T/G/C) within aâ€‰Â±â€‰10 bp interval surrounding the 5â€² end of each cfDNA fragment (a total of 21 positions, with the fragment start site indexed as position 0) from BC, benign, and healthy subjects. To distinguish this feature from the 4-mer EM, we designated it as Motif 21 (ME21). Our analysis identified 8 significant ME21s shared by both pairwise comparisons (<em>p</em>-valueâ€‰â‰¤â€‰0.05, Mannâ€“Whitney <em>U</em> test, Benjaminiâ€“Hochberg correction, Fig. <a href="#Fig3" class="usa-link">3</a>D). Among these, 2 ME21s were located at positionâ€‰âˆ’â€‰1, where we observed significant enrichment of thymine (T) and a reduced frequency of Guanine (G) (Fig. <a href="#Fig3" class="usa-link">3</a>D,E). Another two ME21s were located at positionâ€‰âˆ’â€‰2, where we identified a notable increase in cytosine (C) and a decrease in guanine (G) (Fig. <a href="#Fig3" class="usa-link">3</a>D,E). The remaining 4 significant ME21s exhibited an increased proportion of cytosine (C) at positionsâ€‰âˆ’â€‰3, 0, 1, and 2 (Fig. <a href="#Fig3" class="usa-link">3</a>D,E).</p>
<p id="Par54">Altogether, our genome-wide sequencing analysis reveals significant changes in CNA and ME that differentiate BC patients from both healthy individuals and those with benign lesions. These findings underscore their potential utility as biomarkers for differential diagnosis.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Identification of methylation changes and CNA in target regions for distinguishing BC patients from benign and healthy individuals</h3>
<p id="Par55">DNA methylation alterations are critical epigenetic modifications that regulate the expression of cancer-associated genes, playing a crucial role in cancer carcinogenesis [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. To explore this, we profiled methylation changes across 450 target regions selected for their importance in the transcriptional regulation of cancer-associated genes. Among these regions, we identified three DMRs, <em>HIVEP2_GPR126</em>, <em>KLF3_TLR10</em>, and <em>MAFB_TOP1</em>, that were consistent across two pairwise comparisons in the discovery cohort (<em>p</em>-valueâ€‰â‰¤â€‰0.05, Mannâ€“Whitney <em>U</em> test, Benjaminiâ€“Hochberg correction, Fig. <a href="#Fig4" class="usa-link">4</a>A).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366216_12915_2025_2371_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbff/12366216/dc00b84254e1/12915_2025_2371_Fig4_HTML.jpg" loading="lazy" id="MO4" height="614" width="778" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis of targeted methylation and copy number in plasma cfDNA. <strong>A</strong> Volcano plot shows log2 fold change and significance methylation (âˆ’log10 Benjamini- Hochberg adjusted <em>p</em>-value from Mann-Whitney U test) of 450 target regions when comparing BC patients to benign or healthy controls in the discovery cohort. There are 3 DMRs (adjusted <em>p-</em>value
&lt; 0.05) and overlapped across these two pairwise comparisons, color-coded by genomic locations (highlighted in red for log2FC&gt;0 and blue for log2FC&lt;0). <strong>B</strong> Boxplots showing 3 regions (HIVEP2_GPR126, KLF3_TLR10, and MAFB_TOP1) in BC patients, benign patients, and healthy participants. <strong>C</strong> Heatmap shows log2 fold change of 156 regions with significant copy number in BC patients compared to benign or healthy individuals. The number of significant regions in each pairwise comparison and overlapped regions was indicated via Venn diagram (<strong>D</strong>). Boxplots showing the top 5 with significantly raised CNA values (<strong>E</strong>) and the top 5 with significantly reduced CNA values (<strong>F</strong>) of BC patients compared to benign or healthy individuals</p></figcaption></figure><p id="Par56">BC patients exhibited significant hypermethylation in the <em>HIVEP2_GPR126</em> region, which are associated with <em>HIVEP2</em> and <em>GPR126</em>, and in the <em>KLF3_TLR10</em> region, which regulates <em>KLF3</em> and <em>TLR10</em> (Log<sub>2</sub>FCâ€‰&gt;â€‰0, Fig. <a href="#Fig4" class="usa-link">4</a>A,B). Conversely, hypomethylation was observed in the <em>MAFB_TOP1</em> region, which are associated with <em>MAFB</em> and <em>TOP1</em> (Log<sub>2</sub>FCâ€‰&lt;â€‰0; Fig. <a href="#Fig4" class="usa-link">4</a>A,B). Notably, genes such as <em>GPR126</em>, <em>KLF3</em>, <em>TLR10</em>, and <em>TOP1</em> are well-documented for their roles in BC cell differentiation [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>â€“<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Thus, our methylation analysis identified three significant DMRs associated with the transcriptional regulation of BC-related genes, highlighting their potential as biomarkers to distinguish BC from benign or healthy individuals.</p>
<p id="Par57">DNA methylation changes are often associated with CNAs, as both can reflect underlying changes in gene regulation [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. To complement the methylation analysis, we assessed CNAs in the same 450 target regions among BC, benign, and healthy individuals. This localized analysis of CNAs, distinct from genome-wide CNA studies, was termed as â€œT_CNA.â€ Of these 450 regions, 156 (34.7%) exhibited significant CNAs across the two pairwise comparisons (<em>p</em>-valueâ€‰â‰¤â€‰0.05, Mannâ€“Whitney <em>U</em> test, Benjaminiâ€“Hochberg correction, Fig. <a href="#Fig4" class="usa-link">4</a>C,D). Of these, 109/156 regions (69.9%) displayed a significant T_CNA gain (Log<sub>2</sub>FCâ€‰&gt;â€‰0, Fig. <a href="#Fig4" class="usa-link">4</a>C), while 47/156 regions (30.1%) exhibited a significant T_CNA loss (Log<sub>2</sub>FCâ€‰&lt;â€‰0, Fig. <a href="#Fig4" class="usa-link">4</a>C) in BC patients compared to benign or healthy controls. The top 5 regions with a significant T_CNA gain in BC patients included <em>SOX17_1</em>, <em>RYR2</em>, <em>HOXD8_HOXD9</em>, <em>SOX17_2</em>, and <em>HOXD8</em> (Fig. <a href="#Fig4" class="usa-link">4</a>E). In contrast, regions with a significant T_CNA loss included <em>TPO_SNTG2</em>, <em>PPP1R3C</em>, <em>C19orf25_APC2</em>, <em>RHOBTB2_TNFRSF10B</em>, and <em>BAIAP2_CHMP6 </em>(Fig. <a href="#Fig4" class="usa-link">4</a>F).</p>
<p id="Par58">Our study identifies three key DMRs and several regions with significant T_CNA alterations, underscoring their potential utility as biomarkers for BC diagnosis. Alterations in methylation and copy number at key regions may provide valuable insights into the regulatory mechanisms underlying BC progression.</p></section><section id="Sec6"><h3 class="pmc_sec_title">A multimodal analysis combining signatures from both the target and genome-wide fraction to enhance the accuracy of BC detection</h3>
<p id="Par59">The identification of multiple significant signatures from both the targeted and genome-wide fractions motivated the development of a classification model to discriminate BC patients from individuals with benign conditions and healthy controls.</p>
<p id="Par60">To construct this model, we divided the discovery cohort dataset into training and test sets. Dimensionality reduction was applied to the training dataset to remove highly correlated features using the Kendall rank correlation coefficient (<em>r</em>â€‰â‰¥â€‰0.7). Subsequently, we performed tenfold cross-validation to identify stable features, employing the SelectKBest method to retain the top 500 features based on their recurrence across iterations (Fig. <a href="#Fig5" class="usa-link">5</a>A).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366216_12915_2025_2371_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbff/12366216/b1f514fac2e4/12915_2025_2371_Fig5_HTML.jpg" loading="lazy" id="MO5" height="958" width="779" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Model construction and evaluation of multi feature model. <strong>A</strong> Model construction workflow. Receiver Operating Characteristic (ROC) curves showing the performances of the multi-feature models with the classification BC versus noncancer (<strong>B</strong>), BC versus benign (<strong>C</strong>), and BC versus healthy (<strong>D</strong>). <strong>E</strong> Bar plot displaying sensitivity and specificity values in the train and the test set</p></figcaption></figure><p id="Par61">We assessed the performance of five machine learning algorithms, including LR, SVM, DT, RF, and XGB, using the HyperClassifierSearch function with tenfold cross-validation and default parameters. Among these, XGB outperformed the other algorithms (Additional file 1:Fig.S1). After selecting XGB as the optimal algorithm, we fine-tuned its hyperparameters using tenfold cross-validation to optimize model performance. The finalized model was evaluated on the test set, achieving outstanding performance metrics.</p>
<p id="Par62">In the training set, our multimodal model achieved area under the ROC curve (AUC) values of 0.97 (95% CI 0.95â€“0.99), 0.95 (95% CI 0.92â€“0.98), and 0.98 (95% CI 0.96â€“0.99) for discriminating BC patients from non-cancer individuals (Fig. <a href="#Fig5" class="usa-link">5</a>B), benign lesion cases (Fig. <a href="#Fig5" class="usa-link">5</a>C), and healthy controls (Fig. <a href="#Fig5" class="usa-link">5</a>D), respectively. Validation on the test set demonstrated robust performance for pairwise classifications, with AUC values of 0.90 (95% CI 0.87â€“0.94), 0.88 (95% CI 0.83â€“0.94), and 0.92 (95% CI 0.88â€“0.96) for the same classifications (Fig. <a href="#Fig5" class="usa-link">5</a>Bâ€“D).</p>
<p id="Par63">To further enhance clinical utility, we implemented a cutoff value of 0.88 to ensure a specificity of at least 95%, thereby minimizing false-positive rates. Under this threshold, the model achieved a sensitivity of 80.4% for detecting BC patients, with specificities of 97.4%, 96.2%, and 98.5% for differentiating BC patients from benign cases, and healthy controls, respectively (Fig. <a href="#Fig5" class="usa-link">5</a>E). When applied to the test set, the model maintained high specificities of 93.6%, 91.1%, and 95.7% for the same classifications, achieving an overall sensitivity of 66.2% for detecting BC patients (Fig. <a href="#Fig5" class="usa-link">5</a>E).</p>
<p id="Par64">To better understand the contribution of cfDNA signatures in our breast cancer detection model, we applied SHAP analysis to identify the top 20 most impactful features [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Among these, the proportion of Guanine (G) at position 0 of cfDNA fragments emerged as the most influential feature (Additional file 1:Fig.S3). Additionally, the proportions of Adenine (A), Guanine (G), and Thymine (T) at positionâ€‰âˆ’â€‰1 were also ranked among the top 20 features. These nucleotide patterns at cleavage sitesâ€”particularly the enrichment of A and T and depletion of Gâ€”may reflect the activity of specific endonucleases and chromatin structural changes [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p>
<p id="Par65">This multimodal approach effectively integrates targeted and genome-wide signatures, demonstrating strong potential for distinguishing BC patients from non-cancer, benign, and healthy individuals. The high specificity and competitive sensitivity suggest its promise for clinical applications.</p></section><section id="Sec7"><h3 class="pmc_sec_title">The multimodal assay enabled effective detection of BC patients at early stages and with heterogeneous molecular subtypes</h3>
<p id="Par66">Detecting early-stage BC is particularly challenging due to the low levels of ctDNA present in the bloodstream [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The model demonstrated robust performance in detecting stage I, II, and III tumors, with an AUC of 0.96 (95% CI 0.92â€“0.99), 0.98 (95% CI 0.96â€“0.99), and 0.98 (95% CI 0.97â€“1.00), respectively, on the training set (Fig. <a href="#Fig6" class="usa-link">6</a>A). Validation on the test set showed AUCs of 0.92 (95% CI 0.87â€“0.97) for stage I, 0.90 (95% CI 0.85â€“0.94) for stages II, and 0.90 (95% CI 0.83â€“0.97) for stages III (Fig. <a href="#Fig6" class="usa-link">6</a>B). Sensitivities increased incrementally with disease stage, detecting BC patients in stages I, II, and III with sensitivities of 62.1%, 66.3%, and 66.7%, respectively (Fig. <a href="#Fig6" class="usa-link">6</a>C).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366216_12915_2025_2371_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbff/12366216/98a18377f5f5/12915_2025_2371_Fig6_HTML.jpg" loading="lazy" id="MO6" height="513" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of the model in BC detection at different stages or with different subtypes. ROC curves showing the performance of the model for BC stage I to III in the train set (<strong>A</strong>) and the test set (<strong>B</strong>). <strong>C</strong> Bar plot showing the sensitivities of the model according to stages I-â€“III. ROC curves showing the performance of the model on different BC subtypes in the train set (<strong>D</strong>) and the test set (<strong>E</strong>). <strong>F</strong> Bar plot showing the sensitivities of the model according to subtypes of BC patients</p></figcaption></figure><p id="Par67">BC is a highly heterogeneous disease, comprising diverse subtypes with distinct genetic, molecular, and clinical characteristics [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. To assess the ability of model to identify these subtypes, we evaluated its performance across five molecular subtype groups: Luminal A, Luminal B, Luminal B-HER2, HER2-enriched, and TNBC. The model exhibited excellent performance in the training set, achieving AUC values ranging from 0.95 to 0.99 for all subtypes (Fig. <a href="#Fig6" class="usa-link">6</a>D). In the test set, the model maintained strong performance, with AUC values ranging from 0.91 to 0.92 (Fig. <a href="#Fig6" class="usa-link">6</a>E). The Luminal B-HER2 subtype demonstrated the lowest test set performance, with an AUC of 0.86 (95% CI 0.76â€“0.94) (Fig. <a href="#Fig6" class="usa-link">6</a>E). Sensitivity analysis revealed that the model effectively differentiated between molecular subtypes in the test set. The highest sensitivity was observed for Luminal B (69.6%), followed by TNBC (63.6%), Luminal A (63.6%), HER2-enriched (61.9%), and Luminal B-HER2 (60.0%) (Fig. <a href="#Fig6" class="usa-link">6</a>F).</p>
<p id="Par68">Our multimodal assay achieved high accuracy in detecting early-stage BC and demonstrated strong performance across heterogeneous molecular subtypes. These results highlight the potential of this approach as a robust diagnostic tool for early detection and molecular classification of BC.</p></section><section id="Sec8"><h3 class="pmc_sec_title">The multimodal assay demonstrated consistently high performance in a validation cohort of participants with breast lesions suspected of malignancy</h3>
<p id="Par69">To further validate the performance of the multimodal model in a diagnostic setting, we recruited 119 participants presenting lesions suspected of BC, classified as BI-RADS 3 to 5 following mammography. These participants, forming the validation cohort, were referred for diagnostic testing via FNA to confirm lesion types. Blood samples for the multimodal assay were collected prior to the FNA procedure. The assay results were subsequently compared with FNA outcomes and tissue biopsy findings from participants who underwent surgery.</p>
<p id="Par70">Among the 119 patients, 43 (36.1%) were confirmed to have BC based on FNA results. Clinical characteristics of the cohort are detailed in Additional file 1:Table S2 and Table S3. Of the 63 patients with BI-RADS 3, 4 (6.3%) were diagnosed with malignant lesions, while 59 (93.7%) had benign lesions, including 46 fibroadenomas, 3 cases of fibrocystic changes, 6 cysts, 1 atypical hyperplasia, and 3 unavailable subtype information cases (Additional file 1:Table S2 and Fig. <a href="#Fig7" class="usa-link">7</a>A). In contrast, higher proportions of patients with BI-RADS 4 (59.5%) and BI-RADS 5 (89.5%) were confirmed to have malignant lesions. Among the 43 BC patients, 36 (83.7%) were diagnosed at early stages (Iâ€“II), while 7 (16.3%) were diagnosed at late stages (III) (Additional file 1:Table S2 and Fig. <a href="#Fig7" class="usa-link">7</a>A). Evaluation of the multimodal model on this cohort demonstrated an overall sensitivity of 65.1%, with sensitivity of 50.0% for BI-RADS 3, 68.2% for BI-RADS 4, and 64.7% for BI-RADS 5 patients in identifying BC (Fig. <a href="#Fig7" class="usa-link">7</a>B). Notably, the model achieved high specificity of 96.1% in differentiating benign individuals from BC patients, including 100% specificity for BI-RADS 5 classification. Among BI-RADS 4 patients, one benign case (a fibroadenoma) was misclassified as malignant, but the model still maintained a specificity of 93.3% for this category (Fig. <a href="#Fig7" class="usa-link">7</a>B). Similarly, in the BI-RADS 3 group, two benign cases (a cyst and a fibroadenoma) were misclassified as malignant, yielding a specificity of 96.6% (Fig. <a href="#Fig7" class="usa-link">7</a>B). Stratification of BC patients by stage revealed that the model detected BC with sensitivities of 53.3% for stage I, 76.2% for stage II, and 57.1% for stage III (Fig. <a href="#Fig7" class="usa-link">7</a>C). Moreover, the model exhibited high sensitivity for certain molecular subtypes, achieving the highest sensitivity (80%) for both Luminal A and HER2-enriched subtypes. Detection sensitivity for the Luminal B-HER2 and Luminal B subtypes was slightly lower, at 66.7% and 50%, respectively (Fig. <a href="#Fig7" class="usa-link">7</a>D).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366216_12915_2025_2371_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbff/12366216/983d16847118/12915_2025_2371_Fig7_HTML.jpg" loading="lazy" id="MO7" height="614" width="779" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of the model in the validation cohort. <strong>A</strong> Heatmap shows the demographic details of 119 symptomatic participants in the validation cohort. <strong>B</strong> Bar plot showing sensitivities and specificities of the model according to BI-RADS lesions. <strong>C</strong> Bar plot indicates sensitivities of BC patients in different stages. <strong>D</strong> Bar plot displaying sensitivities of BC patients in different subtypes</p></figcaption></figure><p id="Par71">These findings demonstrate that the multimodal assay effectively differentiates benign from malignant lesions in a diagnostic setting, achieving high specificity for differentiating benign lesions from early-stage tumors. This highlights its potential as a robust companion tool complementing SOC screening tests in clinical practice.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par72">Recent studies have highlighted liquid biopsy as a promising approach for early BC detection. However, distinguishing benign lesions from malignant tumors remains a significant challenge [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>â€“<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. In this study, our multimodal analysis of cfDNA characteristics demonstrated robust performance in differentiating BC patients not only from healthy controls but also from individuals with benign conditions. This distinction has the potential to reduce false-positive rates, thereby minimizing unnecessary biopsies in clinical practice. Furthermore, our protocol integrates both targeted enrichment and shallow genome-wide sequencing (0.5X) from a single 10 mL blood sample and library preparation (Fig. <a href="#Fig1" class="usa-link">1</a>). This â€œall-in-oneâ€ approach allows for the simultaneous profiling of multiple cfDNA signatures while keeping testing costs low compared to other assays [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], thus supporting its potential implementation in routine clinical practice to differentiate breast cancer from benign and healthy cases.</p>
<p id="Par73">BC is a heterogeneous disease, characterized by variations in the genomic, epigenomic, transcriptomic, and proteomic profiles of cancer cells [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Additionally, at early stages, the concentration of ctDNA in the bloodstream is extremely low, posing a challenge for detection [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. To address these complexities, we previously analyzed multiple features of cfDNA, including methylation and fragmentomic patterns, to identify a broader spectrum of BC-specific signatures. In this study, we expanded on this approach by integrating three genome-wide signaturesâ€”CNA, EM, and ME21, and two targeted signaturesâ€”TMD and targeted CNA (T_CNA).</p>
<p id="Par74">Aberrant DNA copy number changes in breast cancer primarily arise from genomic instability during DNA replicationâ€”a hallmark of cancer. These alterations, involving the gain or loss of DNA segments, often confer growth or survival advantages to tumor cells [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Our statistical analysis identified significant CNAs across several chromosomes, consistent with previously reported findings. For instance, gain of chromosome 16 has been associated with high nuclear grade, larger tumor size, advanced stage, increased expression of topoisomerase IIÎ±, and poorer overall survival in breast cancer [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Furthermore, luminal B-like and HER2-overexpressing subtypes frequently exhibit gains in 1q, 8q, 16p, and 17qâ€”features characteristic of the â€œfirestormâ€ CNA profile [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. In our genome-wide fraction, CNA analysis revealed significant alterations across all 22 chromosomes when comparing BC patients with benign conditions, with more pronounced CNA signatures than those observed between BC and healthy subjects. Benign breast disease may represent an intermediate stage in the progression from normal tissue to cancer [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>], which could explain the complex and variable patterns seen in this group. Furthermore, inflammation in benign breast tissue may contribute to genomic instability, reflecting changes in the tumor microenvironment [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Similarly, altered fragmentation patterns in cfDNA likely result from tumor-specific chromatin accessibility and nucleosome positioning, influenced by the cancer cellâ€™s epigenetic state and apoptosis dynamics [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. During apoptosis, DNA in cancer cells and normal cells cleaves differently, resulting in variations in the DNA patterns at the end of each fragment. Previous studies have demonstrated that 4-mer end motifs can be leveraged for detecting ctDNA in the plasma of patients with hepatocellular carcinoma and other cancers [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. In this study, we identified distinct end motif patterns distinguishing BC from healthy controls as well as from patients with benign patients. Notably, all enriched end motifs in BC began with â€œC,â€ consistent with findings from our previous multi-cancer early detection study, which included BC [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Similarly, Jin et al. identified three 4-mer end motifs (CCCA, CCTG, and CCAG) as preferred end motifs in hepatocellular carcinoma [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Serpas et al. further suggested that <em>DNASE1L3</em> may contribute to the generation of the â€œCCCAâ€ end motif [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. In a related study, Zhu et al. [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] employed an experimental model using the HL60 human leukemia cell line undergoing apoptosis to investigate DNA fragmentation patterns via shallow whole-genome sequencing. Their results revealed that C-end motif preferences (e.g., fragments beginning with cytosine) were retained both intracellularly and extracellularly in apoptotic cells, even in the absence of <em>DNASE1L3</em>. Interestingly, they also observed a reduced abundance of C-end motifs in cfDNA from cancer patients. One possible explanation is that while <em>DNASE1L3</em> may be downregulated in tumors, other nucleasesâ€”such as <em>DNASE1</em>, <em>DFFB</em>, or additional endonucleasesâ€”could compensate or preferentially cleave at C-rich regions, contributing to the observed motif enrichment in vivo [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. Although the HL60 apoptosis model offers valuable mechanistic insights under controlled in vitro conditions, it does not fully capture the biological complexity of cfDNA fragmentation in vivo. In the physiological setting, factors such as tumor heterogeneity, the diversity of cell death pathways, and dynamic interactions within the tumor microenvironment collectively influence nuclease activity and chromatin accessibility. These findings underscore the need for appropriate in vivo models in future studies to disentangle the relative contributions of specific nucleases and chromatin context to cfDNA fragmentomic patterns in cancer.</p>
<p id="Par75">Previous studies have also shown that cfDNA fragment start and end sites contain distinct nucleotide motifs that can differentiate tumor-derived fragments [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. For example, Chandrananda et al. demonstrated that 10-bp motifs upstream and downstream of the cfDNA cleavage site exhibit consistent patterns in cancer-derived cfDNA [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Consistently, we identified significant ME21 signatures enriched in BC compared to benign patients or healthy individuals. This finding highlights the nonrandom fragmentation process of BC-derived ctDNA, underscoring its potential as a marker to discriminate benign lesions from malignant tumors. The proportions of Guanine (G) at position 0, and Adenine (A), Guanine (G), and Thymine (T) at positionâ€‰âˆ’â€‰1, were identified as key cfDNA features contributing to model performance (Additional file 1:Fig.S3). These nucleotide patterns at cfDNA cleavage sitesâ€”particularly the enrichment of A and T and the depletion of Gâ€”are thought to reflect the enzymatic activity of endonucleases and tumor-specific chromatin architecture. Supporting this, Bai et al. [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>] demonstrated that cfDNA is initially cleaved intracellularly by <em>DFFB</em> and <em>DNASE1L3</em>, which preferentially generate A-end and C-end fragments, respectively. Subsequent extracellular cleavage is mediated by <em>DNASE1L3</em> and <em>DNASE1</em>, with <em>DNASE1</em> favoring the generation of T-end fragments [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. In addition, a previous study reported substantial intratumoral heterogeneity in the expression of DNase family genes across 33 tumor types, which may contribute to the distinct nucleotide profiles observed at cleavage sites in breast cancer-derived ctDNA [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>].</p>
<p id="Par76">Across 450 target regions, we identified three DMRs (<em>HIVEP2_GPR126</em>, <em>KLF3_TLR10</em>, and <em>MAFB_TOP1</em>, Fig. <a href="#Fig4" class="usa-link">4</a>A) between BC and both benign patients and healthy subjects. Aberrant DNA methylation in breast cancer often reflects transcriptional silencing of tumor suppressor genes or activation of oncogenic pathways, which is mirrored in the plasma cfDNA methylation landscape [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Among the hypermethylated targets in our study, <em>GPR126</em>, a member of the adhesion G protein-coupled receptor family, plays a key role in the progression of triple-negative breast cancer [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. We also observed hypermethylation in regulatory regions associated with <em>KLF3</em> and <em>TLR10</em>, consistent with previous studies that identified both genes as tumor suppressors [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>] and reported their downregulation in BC patients compared to healthy controls [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Similarly, decreased expression of several toll-like receptors (<em>TLRs</em>), including <em>TLR10</em>, has been documented in BC tissues relative to normal tissues [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. In addition, we observed hypomethylation of genomic regions associated with <em>TOP1</em>, aligning with reports of elevated <em>TOP1</em> expression in BC patients, as described by Tsyganov et al. [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Moreover, we investigated methylation differences between breast tumor and normal tissues using methylation array data from the TCGA-BRCA project. Out of 354,454 CpG sites analyzed, 262,289 were found to be significantly differentially methylated between tumor and normal samples (<em>p</em>-valueâ€‰â‰¤â€‰0.05, <em>t</em>-test, Benjaminiâ€“Hochberg correction). These CpG sites were annotated to 19,163 genes. When compared to the 584 genes annotated from our 450 targeted regions, 542 genes overlapped with those identified in the TCGA dataset. Notably, all six genes corresponding to the three most statistically significant cfDNA-derived regions (<em>HIVEP2_GPR126, KLF3_TLR10</em>, and <em>MAFB_TOP1</em>) also overlapped with the TCGA data (Additional file 1:Fig.S4A). In addition, two key genes identified through SHAP analysisâ€”<em>SNAI1</em> and <em>UBE2V1</em>â€”were also present among the TCGA-derived differentially methylated genes (Additional file 1:Fig.S4B). These results support the biological relevance of our identified methylation signatures and suggest their potential involvement in breast cancer occurrence and progression.</p>
<p id="Par77">In the target fraction, we observed several regions with significant copy number changes in BC compared to both benign patients and healthy subjects. Previous studies have reported that associations between somatic CNA and DNA methylation can be either positive or negative. In contrast, associations between CNA and gene expression may occur independently, as DNA methylation does not strongly mediate this relationship [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. In our study, we identified regions with T_CNA alterations localized in non-differentially methylated regions of BC. Prior research indicates that the copy number variant of <em>SOX17</em> influences both methylation and gene expression in BC [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p>
<p id="Par78">Our multimodal approach demonstrated that combining distinct cfDNA features enhances sensitivity for BC detection while maintaining high specificity in distinguishing benign and healthy classifications. Although the model performed well on the training set, with a sensitivity of 80.4% (Fig. <a href="#Fig5" class="usa-link">5</a>E), sensitivity declined to 66.2% on the test set (Fig. <a href="#Fig5" class="usa-link">5</a>E). In this study, our multi-omics analysis resulted in a high-dimensional feature space, which could increase the risk of overfitting during model construction [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. To mitigate this, we implemented a train-test split combined with tenfold cross-validation within the training set (Fig. <a href="#Fig5" class="usa-link">5</a>A). This strategy enabled robust hyperparameter tuning and stable feature selection across folds, thereby reducing the risk of overfitting to any specific subset. Notably, the test set was completely withheld during model development and used only once for final evaluation, ensuring an unbiased assessment of the modelâ€™s generalization performance. This approach allows for efficient use of limited data while maintaining the integrity of performance evaluation. Previous studies have emphasized the effectiveness of cross-validation in mitigating overfitting and highlighted the importance of independent test set evaluation to avoid optimistic bias in performance estimation [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. To evaluate model stability and assess the risk of overfitting, we performed bootstrap resampling across 1000 training epochs on the discovery cohort (Fig. <a href="#Fig5" class="usa-link">5</a>A). For each of 10 bootstrap iterations, an XGBoost classifier was trained on a resampled dataset drawn with replacement from the discovery cohort. Model performance was monitored across 10 independently bootstrapped models. Beyond 200 epochs, log-loss remained consistently low (Additional file 1:Fig.S5A), and classification accuracy remained high (Additional file 1:Fig.S5B), indicating stable model behavior. Notably, both log-loss and accuracy stabilized early in the training process, suggesting efficient convergence and obviating the need for extended training beyond 200 epochs (Additional file 1:Fig.S5). The narrow and stable performance gap between training and validation sets further supports good model generalizability and indicates an absence of significant overfitting (Additional file 1:Fig.S5B). Despite these challenges, our performance in screening BC was highly competitive, particularly in the context of benign lesion inclusion. Comparatively, the Galleri test, which utilizes a panel ofâ€‰&gt;â€‰100,000 informative methylation loci via targeted whole-genome bisulfite sequencing of plasma DNA to detectâ€‰&gt;â€‰50 cancer types, achieved a specificity of 99.3%. However, BC exhibited the lowest sensitivity, with detection rates of less than 25% for stage I and approximately 50% for stage II cases in caseâ€“control studies [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Compared to our MCED test (SPOT-MAS) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], which utilizes features selected through comparisons across multiple cancer types, the multi-feature model presented in this study demonstrated superior performance (66.2% vs. 49.3%). These findings support our hypothesis that focusing on BC-specific features for training a machine-learning model enhances its performance for detecting a particular cancer type.</p>
<p id="Par79">In this study, we demonstrated that the performance of the multi-feature model was not subtype dependent. Breast cancer is a biologically heterogeneous disease comprising several molecular subtypes, including Luminal A, Luminal B, HER2-enriched, and triple-negative breast cancer (TNBC), each characterized by distinct molecular alterations, biological behaviors, and clinical outcomes [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Approximately 20% of Luminal B tumors are HER2-positive, a group classified as Luminal Bâ€“HER2 (ER<sup>+</sup>/HER2<sup>+</sup>)[<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. In our study, the lower sensitivity observed for this subtype in test set (60%) may reflect subtype-specific differences in ctDNA shedding dynamics and epigenetic features. Less aggressive or more indolent subtypes, such as Luminal A and Luminal B (ER<sup>+</sup>/HER2<sup>-</sup>), tend to release lower levels of ctDNA into circulation. In contrast, more aggressive subtypes, including HER2-enriched and TNBC, are associated with higher ctDNA release due to increased tumor proliferation and necrosis [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. Although Luminal Bâ€“HER2 tumors are more proliferative than Luminal A [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>], they may still shed less ctDNA than HER2-enriched non-luminal or TNBC subtypes, which exhibit more aggressive tumor biology and higher ctDNA abundance [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>, <a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. This biological heterogeneity likely contributes to differences in detection sensitivity across subtypes. Our multimodal approach, which integrates a range of ctDNA-based features, may help mitigate this variability by capturing a broader spectrum of molecular signatures unique to each subtype. Notably, TNBC exhibited a high sensitivity of 63.6% in the test set (Fig. <a href="#Fig6" class="usa-link">6</a>F), aligning with previous studies that emphasize the utility of ctDNA in diagnosing TNBC across early and advanced stages [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>].</p>
<p id="Par80">To further validate our multimodal model, we evaluated its performance in detecting BC within a high-risk cohort of participants with breast lesions by mammography and recommended for FNA for diagnostic confirmation. The malignancy rates for BIRADS-3, BIRADS-4, and BIRADS-5 were 6.3%, 59.5%, and 89.5%, respectively (Additional file 1:Table S2), consistent with previous reports [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. These malignancy rates suggest that the high-risk cohort reflects real-world clinical scenarios. The model accurately identified 50.0% (2/4) of BC cases in the BIRADS-3 group, 68.2% (15/22) in BIRADS-4, and 64.7% (11/17) in BIRADS-5 (Fig. <a href="#Fig7" class="usa-link">7</a>B). Overall, the model achieved a sensitivity of 65.1% (Fig. <a href="#Fig7" class="usa-link">7</a>B) while maintaining a high specificity of 96.1% (Fig. <a href="#Fig7" class="usa-link">7</a>B) in distinguishing BC from benign lesions across BIRADS categories. This performance is consistent with that observed in the original test set, which demonstrated a sensitivity of 66.2% (<em>p</em>-valueâ€‰=â€‰1.0, Fisherâ€™s exact test) and a specificity of 91.1% (<em>p</em>-valueâ€‰=â€‰0.28, Fisherâ€™s exact test) (Fig. <a href="#Fig5" class="usa-link">5</a>E). These results indicate that the model maintains robust and consistent performance when prospectively validated in an independent external cohort. Moreover, these results highlight the potential of the multimodal model, trained on BC-specific signatures, to enhance differentiation between malignant and benign lesions and reduce the need for invasive diagnostic procedures in high-risk populations.</p>
<p id="Par81">While sensitivity for stage I breast cancer remains moderate (53â€“62%), this performance surpasses that of several published cfDNA-based assays and reflects inherent biological challenges related to minimal ctDNA shedding in early disease [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Despite these challenges, our test achieved a sensitivity of 62â€“66.3% for stage Iâ€“II breast cancers (Fig. <a href="#Fig6" class="usa-link">6</a>C and Additional file 1:Table S4)â€”substantially higher than other reported ctDNA-based assays, such as Galleri [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>] (25â€“50%), CancerSEEK (33%), DELFI (57%) [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>], and SPOT-MAS MCED (49.3%) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. A nucleosome profiling-based cfDNA assay developed by Han et al. evaluated 173 malignant, 158 benign, and 102 healthy samples, reporting a sensitivity of 70.8% but a relatively low specificity of 76.5% in distinguishing breast cancer from benign lesions [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. In contrast, our assay demonstrated a unique strength in differentiating breast cancers from benign breast conditions, achieving a high specificity of 96.1% (Fig. <a href="#Fig7" class="usa-link">7</a>B and Additional file 1:Table S4) in a validation cohort representative of real-world high-risk populations. This highlights its potential utility in clinical settings where benign breast lesions are common and often lead to diagnostic uncertainty. Recently, Guardant Revealâ„¢ has been approved for the detection of minimal residual disease (MRD) and recurrence monitoring in early-stage cancers, although it is not primarily intended for early detection or population-level screening. In exploratory analyses involving breast cancer, the assay demonstrated a sensitivity of 79% for detecting distant recurrence but only 13% for localized disease, while maintaining a specificity of 100% [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. These findings highlight the persistent challenge of detecting ctDNA shed by early-stage breast tumors.</p>
<p id="Par82">Despite these promising results, our study has several limitations. First, while the multimodal model showed strong classification performance, the validation cohort sample size was limited. Larger cohorts are needed for further validation before applying the model in real-world screening settings. Second, 16.3% of breast cancer patients were without subtype information (7/43) in the validation cohort (Additional file 1:Fig.S2 and Table S2). These patients underwent initial imaging at study hospitals but chose different facilities for subsequent tests and surgeries, limiting access to complete histological data. In addition, the limited sample size for each molecular subtype remains a constraint, particularly for Luminal Bâ€“HER2, HER2-enriched, and triple-negative breast cancer (TNBC). As reported by Clarke et al. [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>], the distribution of breast cancer subtypes varies by age and population. Among Asian women, HRâº/HER2â» tumors (Luminal A and Luminal B) represent approximately 55% of cases, HRâº/HER2âº (Luminal Bâ€“HER2) account forâ€‰~â€‰12%, HER2-enriched (HRâ»/HER2âº) forâ€‰~â€‰8%, and TNBC forâ€‰~â€‰9%[<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Consequently, acquiring sufficient numbers of early-stage samples from these less common subtypes poses a significant challenge for subtype-specific model development and validation. Future validation of our model in more diverse populations and clinical contexts, and evaluation using real-world data, is required to confirm the robustness of our assay. Currently, we are conducting a prospective study, K-ACCELERATE (<a href="https://clinicaltrials.gov/ct2/show/NCT06391749" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT06391749</a>), which aims to recruit 1000 participants presenting with symptoms or signs suggestive of the five most common cancer types, including breast cancer. This study will provide an opportunity to assess the diagnostic performance of our model in a high-risk diagnostic population and evaluate its sensitivity across molecular subtypes. Third, while our protocol, which combines targeted and shallow genome-wide sequencing (Fig. <a href="#Fig1" class="usa-link">1</a>), enables low testing costs, it does not capture the full spectrum of molecular signatures associated with breast cancer. We are actively incorporating deep sequencing to comprehensively profile the methylome and fragmentome, with the goal of integrating additional features into our framework. This expanded feature set is expected to further improve detection sensitivity and enhance biological interpretability. We believe this iterative strategy will enable the progressive refinement of our assayâ€™s performance while maintaining its clinical feasibility. Finally, while our assay leverages shallow whole-genome sequencing and an integrated library preparation protocol to reduce per-sample costs, we acknowledge that a comprehensive cost-effectiveness analysisâ€”including direct comparison with existing modalities such as mammography in terms of cost-per-test, infrastructure requirements, and workflow integrationâ€”is needed to fully assess clinical scalability. Future prospective studies are warranted to evaluate these parameters in real-world settings.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par83">Our findings underscore the potential of multimodal plasma cfDNA analysis to identify novel biomarkers for accurately distinguishing BC patients from benign lesions and healthy individuals. This capability could reduce false-positive rates and unnecessary biopsies, offering significant clinical utility for high-risk populations.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Patient enrollment</h3>
<p id="Par84">This study included a discovery cohort of 515 female participants, consisting of 273 BC patients, 108 individuals with benign breast disease, and 134 healthy controls. BC diagnoses were confirmed through abnormal mammography findings followed by tissue biopsy. Disease staging was determined using the TNM (Tumor, Node, Metastasis) classification system, based on guidelines from the American Joint Committee on Cancer (Version VIII) and the International Union for Cancer Control. To emphasize the importance of early cancer detection, only patients with non-metastatic BC (stages Iâ€“IIIA) were included in the study.</p>
<p id="Par85">Participants with benign breast disease and healthy controls underwent routine mammography as part of their annual health check-ups. Diagnoses of benign breast disease were confirmed at the time of inclusion and were characterized as non-malignant lumps, including fibroadenomas and fibrocystic changes. These individuals were monitored for 12 months to confirm their cancer-free status. Participants for the discovery cohort were recruited between October 2021 and December 2023 from Ho Chi Minh City Oncology Hospital and the Medical Genetics Institute in Ho Chi Minh City, Vietnam.</p>
<p id="Par86">An external validation cohort was also recruited, comprising 119 female participants identified with BI-RADS 3â€“5 lesions through mammography. All participants underwent fine needle aspiration (FNA) for diagnostic confirmation. This validation cohort was recruited between April 2024 and April 2025 from Thai Nguyen National General Hospital, Can Tho Oncology Hospital, Nghe An Oncology Hospital, Da Nang Oncology Hospital and Buon Ma Thuot Medical University Hospital, Vietnam.</p>
<p id="Par87">The study complied with the ethical principles set forth in the Declaration of Helsinki. The research protocol was approved by the Ethics Committees of all participating institutions, with ethics review number 294/BVUB-HÄÄÄ for the discovery cohort and 460/HÄÄÄ-ÄHYD for the validation cohort. Written informed consent was obtained from all participants before sample collection. Importantly, all samples were collected prior to the initiation of any therapeutic interventions to ensure the integrity of the analysis.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Study design</h3>
<p id="Par88">Plasma samples were isolated from 273 BC patients, 108 individuals with benign breast disease, and 134 healthy controls in the discovery cohort. We employed the SPOT-MAS assay, as described in our previous publication [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], to simultaneously analyze multiple cfDNA signatures from both targeted and genome-wide fractions. To enhance BC detection, we extended the analysis of 21-mer motif ends (ME21), targeted methylation density (TMD) and copy number aberration (T_CNA) signatures in targeted regions (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<p id="Par89">Distinct cfDNA signatures were profiled from the targeted and genome-wide fractions in two pairwise comparisons: (1)Â BC versus benign disease, and (2) BC versus healthy subjects. Our analysis prioritized signatures exhibiting consistent trends across both comparisons in BC patients. After identifying BC-specific signatures, we developed a multi-feature model capable of differentiating BC patients from individuals with benign lesions and healthy controls (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<p id="Par90">Finally, the predictive model was validated using an independent validation cohort, which had not been included in the training or testing phases. This cohort comprised 119 individuals with BI-RADS 3â€“5 lesions identified via mammography. Plasma samples from these participants were analyzed using the SPOT-MAS assay, and the model predicted the presence or absence of ctDNA signals. To confirm diagnoses, all participants underwent FNA, which served as the reference standard for determining cancer status. Following FNA, 43 participants were diagnosed with benign breast disease, while 76 participants were confirmed as having BC. The performance of model was evaluated by comparing its predictions with FNA-confirmed diagnoses (Fig. <a href="#Fig1" class="usa-link">1</a>).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Isolation of cfDNA</h3>
<p id="Par91">Blood (10 mL) was collected in Cell-Free DNA BCT tubes (Streck, USA) and processed via two-step centrifugation as previously described [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The resulting plasma fractions were aliquoted and stored atâ€‰âˆ’â€‰80 Â°C for long-term preservation. cfDNA was extracted from 1 mL plasma using the MagMAX Cell-Free DNA Isolation Kit (ThermoFisher, USA) and quantified with the QuantiFluor dsDNA System (Promega, USA).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Multimodal analysis of genome-wide and targeted fractions of cfDNA</h3>
<p id="Par92">The cfDNA samples isolated for this study were analyzed using the SPOT-MAS assay, as previously described [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The workflow involves three key steps:</p>
<p id="Par93"><em>Step 1</em>: Bisulfite conversion was performed using the EZ DNA Methylation-Gold Kit (Zymo Research, USA), and libraries were prepared with the xGenâ„¢ Methyl-Seq DNA Library Prep Kit (IDT, USA) using Adaptaseâ„¢ technology.</p>
<p id="Par94"><em>Step 2</em>: The library is subsequently subjected to hybridization to selectively enrich the target fraction, encompassing 450 cancer-specific regions, as detailed in the design and construction of the capture panel [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The non-targeted whole-genome fraction is recovered by collecting the flow-through and re-hybridizing it with probes targeting the adapter sequences of the DNA library. Both the target capture and whole-genome fractions are sequenced to depths of about 52X and 0.55X, respectively, generating 100-bp paired-end reads with a sequencing depth of 20 million reads per fraction. Pre-processing of the data generates five distinct cfDNA feature sets: CNA, EM, ME21, TMD, and T_CNA.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Copy number aberration analysis (CNA)</h3>
<p id="Par95">CNA analysis was performed using the QDNAseq R package [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>] with a 1-Mb segmentation strategy. Regions of low mappability and Duke blacklist regions were excluded to ensure data quality. Read counts per bin were calculated using the â€œbinReadCountsâ€ function followed by GC content correction with â€œestimateCorrectionâ€ and â€œcorrectBins.â€ To refine the CNA features, we applied â€œnormalizeBinsâ€ for normalization and â€œsmoothOutlierBinsâ€ to address outliers. This process yielded a final feature vector comprising 2691 bins.</p></section><section id="Sec17"><h3 class="pmc_sec_title">End motif analysis (EM)</h3>
<p id="Par96">Library preparation with Adaptaseâ„¢ added a random 5â€² tail to reverse reads, precluding accurate 5â€² end identification. Therefore, EM features were calculated based on the 5â€² coordinates of forward reads, as previously described [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p></section><section id="Sec18"><h3 class="pmc_sec_title">End motif 21 analysis (ME21)</h3>
<p id="Par97">To define the 21-bp end-motif feature (ME21), we extracted a 21-nucleotide sequence from the 5â€² end of each R1 sequencing read. This sequence consisted of 11 bp from within the read and 10 bp extending into adjacent genomic regions. A position weight matrix (PWM) of dimensions 4â€‰Ã—â€‰21 was constructed, where each row corresponded to a nucleotide (A, T, G, or C) and each column represented a position within the 21-bp sequence [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. The PWM entries captured the proportional frequency of each nucleotide at each position. The matrix was subsequently flattened into a 1â€‰Ã—â€‰84 vector, which served as an input feature for further analysis.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Targeted methylation density analysis (TMD)</h3>
<p id="Par98">Paired-end reads were trimmed using Trimmomatic v0.32 (HEADCROP), aligned with Bismark v0.22.3, and processed with Samtools v1.15 and Bedtools v2.28. Methylation calling was performed using Bismark methylation extractor [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><math id="d33e1957" display="block"><mrow><mi>M</mi><mi>e</mi><mi>t</mi><mi>h</mi><mi>y</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mspace width="0.277778em"></mspace><mi>r</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mo>=</mo><mfrac><mrow><mi>m</mi><mi>e</mi><mi>t</mi><mi>h</mi><mi>y</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>c</mi><mi>y</mi><mi>t</mi><mi>o</mi><mspace width="0.277778em"></mspace><mo>sin</mo><mspace width="0.277778em"></mspace><mi>e</mi><mspace width="0.277778em"></mspace><mfenced close=")" open="("><mi>c</mi></mfenced></mrow><mrow><mi>m</mi><mi>e</mi><mi>t</mi><mi>h</mi><mi>y</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>C</mi><mspace width="0.277778em"></mspace><mo>+</mo><mspace width="0.277778em"></mspace><mi>u</mi><mi>n</mi><mi>m</mi><mi>e</mi><mi>t</mi><mi>h</mi><mi>y</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>C</mi></mrow></mfrac></mrow></math></td></tr></table>
<p id="Par99">Methylation fold changes between BC and benign or healthy samples were calculated for each target region. Differentially methylated regions were identified by comparing BC and benign/healthy samples using a Benjaminiâ€“Hochberg adjusted <em>p</em>-value threshold ofâ€‰â‰¤â€‰0.05 (âˆ’â€‰log<sub>10</sub> adjusted <em>p</em>â€‰â‰¥â€‰1.301).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Targeted copy number aberration analysis (T_CNA)</h3>
<p id="Par100">The 450-targeted regions (T_CNA) feature was constructed similarly to the common genome-wide CNA feature. Read counts across these 450 regions were normalized, and copy numbers were estimated using the <em>QDNAseq</em> package, with minor modifications to adapt the settings for the targeted regions.</p>
<p id="Par101"><em>Step 3</em>: The cfDNA features were input into a machine learning algorithm to generate predictive outcomes. The discovery cohort dataset was divided into training and testing sets, with the training set further split into two pairwise comparisons: cancer versus benign and cancer versus healthy, for feature selection. To minimize redundancy, Kendall rank correlation (<em>r</em>â€‰â‰¥â€‰0.7) was applied to remove highly correlated features. Cross-validation with 10 folds was used to identify stable features in each pairwise comparison. The <em>SelectKBest</em> method was employed to filter the top 500 features in each iteration, based on scores from the"<em>f_classif</em>"function. Stable features were selected if they appeared consistently across all 500 feature sets from the iterations. The final feature set was formed by combining the selected features from both pairwise comparisons.</p>
<p id="Par102">Five machine learning algorithmsâ€”Logistic Regression (LR), Support Vector Machine (SVM), Decision Tree (DT), Random Forest (RF), and Extreme Gradient Boosting (XGB)â€”were evaluated. To identify the best algorithm, we used the <em>HyperclassifierSearch</em> function with tenfold cross-validation and default parameters. After selecting the optimal algorithm, its hyperparameters were fine-tuned, and a classification model was constructed to differentiate BC patients from benign and healthy individuals. To evaluate model stability and assess the risk of overfitting, we performed bootstrap resampling with 1000 training epochs on the discovery cohort. In each of 10 bootstrap iterations, a dataset of equal size was generated by sampling with replacement from the original data. A classifier was trained on each resampled dataset for 1000 boosting rounds. During training, both log loss and accuracy were tracked across epochs using the bootstrap training data and the corresponding out-of-bag (OOB) samples for validation, providing insight into convergence behavior and potential overfitting.</p>
<section id="Sec21"><h4 class="pmc_sec_title">Statistical analysis</h4>
<p id="Par103">The Mannâ€“Whitney <em>U</em> test was employed to evaluate statistically significant differences between BC and benign samples, as well as between BC and healthy samples. DNA methylation data from 1075 breast tumor tissues and 124 adjacent non-tumor tissues were obtained from the TCGA-BRCA project via the TCGA data portal (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>). Differentially methylated CpG sites were identified using the Studentâ€™s <em>t</em> test. To account for multiple comparisons, <em>p</em>-values were adjusted using the Benjaminiâ€“Hochberg correction, with a significance threshold set at <em>Î±</em>â€‰â‰¤â€‰0.05. Sensitivity and specificity values between the test set and the validation cohort were compared using Fisherâ€™s exact test, with a significance threshold also set at <em>Î±</em>â€‰â‰¤â€‰0.05.</p>
<p id="Par104">Receiver operating characteristic (ROC) curves were constructed, and the area under the ROC curve (AUC), along with 95% confidence intervals (CI), was calculated to evaluate the discriminative performance of features. We applied SHapley Additive exPlanations (SHAP) analysis to identify the top 20 most impactful features of the model [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. All statistical analyses were conducted using Python version 3.11.2 (Python Software Foundation, USA).</p></section></section></section><section id="Sec22"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366216/bin/12915_2025_2371_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12915_2025_2371_MOESM1_ESM.zip</a><sup> (1MB, zip) </sup><p>Additional file 1:Â Table S1-S4 and Figure S1-S5. Table S1. Detailed clinical information of all participants in the discovery cohort. Table S2. Summary of participantsâ€™ clinical features in the validation cohort. Table S3. Detailed clinical information of all participants in the validation cohort. Table S4. Comparison of model performance. Fig.S1. Comparison of Area Under the Curve (AUC) values obtained from tenfold cross-validation across five machine learning algorithms (Error bars representâ€‰Â±â€‰1 standard deviation across tenfold cross-validation). Fig. S2. The pie chart shows the percentages according to stages in BC patients (A), subtypes in BC patients (B), and subtypes of benign patients (C). Fig. S3. SHapley Additive exPlanations (SHapley Additive exPlanations (SHAP) analysis reveals the feature importances in the machine learning model.) of the model in training set. Fig. S4. Overlapping genes between 450 target regions and TCGA databaseâ€‹. Fig. S5. Log loss and accuracy across bootstrapped models on the discovery cohort.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements
</h2>
<p>We sincerely thank all participants for their contribution to this study, as well as the clinics and hospitals that provided support for patient consultations and sample collection.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>BC</dt>
<dd><p id="Par15">Breast cancer</p></dd>
<dt>MRI</dt>
<dd><p id="Par16">Magnetic resonance imaging</p></dd>
<dt>BI-RADS</dt>
<dd><p id="Par17">Breast Imaging Reporting and Data System</p></dd>
<dt>CTCs</dt>
<dd><p id="Par18">Circulating tumor cells</p></dd>
<dt>ctDNA</dt>
<dd><p id="Par19">Circulating tumor DNA</p></dd>
<dt>cfDNA</dt>
<dd><p id="Par20">Cell-free DNA</p></dd>
<dt>miRNAs</dt>
<dd><p id="Par21">MicroRNAs</p></dd>
<dt>SPOT-MAS</dt>
<dd><p id="Par22">Screening for the Presence of Tumor by DNA Methylation and Size</p></dd>
<dt>CNA</dt>
<dd><p id="Par23">Copy number aberration</p></dd>
<dt>EM</dt>
<dd><p id="Par24">End motif</p></dd>
<dt>ME21</dt>
<dd><p id="Par25">Motif 21</p></dd>
<dt>AUC</dt>
<dd><p id="Par26">Area under the ROC curve</p></dd>
<dt>FNA</dt>
<dd><p id="Par27">Fine needle aspiration</p></dd>
<dt>TMD</dt>
<dd><p id="Par28">Targeted methylation density</p></dd>
<dt>T_CNA</dt>
<dd><p id="Par29">Targeted copy number aberration</p></dd>
<dt>DMRs</dt>
<dd><p id="Par30">Differentially methylated regions</p></dd>
<dt>TLRs</dt>
<dd><p id="Par31">Toll-like receptors</p></dd>
<dt>MRD</dt>
<dd><p id="Par32">Minimal residual disease</p></dd>
<dt>TNM</dt>
<dd><p id="Par33">Tumor, Node, Metastasis</p></dd>
<dt>LR</dt>
<dd><p id="Par34">Logistic regression</p></dd>
<dt>SVM</dt>
<dd><p id="Par35">Support Vector Machine</p></dd>
<dt>DT</dt>
<dd><p id="Par36">Decision Tree</p></dd>
<dt>RF</dt>
<dd><p id="Par37">Random Forest</p></dd>
<dt>XGB</dt>
<dd><p id="Par38">Extreme Gradient Boosting</p></dd>
<dt>OOB</dt>
<dd><p id="Par39">Out-of-bag</p></dd>
<dt>SHAP</dt>
<dd><p id="Par40">SHapley Additive exPlanations</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authorsâ€™ contributions</h2>
<p>KST, ALN, CHAT, TXJ, TLV1, THTN, TTT, MHT, NCT, TLL, THNN, NHT, TST, BTL, VPT, PTNN, KTN, VCH, TBL, HTPN, LHDN, NMP, TVP, TTTD, THD, HST performed patient consultancy and screening. TTVV, THHN, VTCN, DHV, TTT, TMTT, VUT, MPL, TLV2 performed formal analysis. TTVV, THT, THN, XVN, NNTD performed data curation. DSN, HG, MDP, HNN, LST performed the methodology. DSN, HG, MDP, HNN, DHV, LST performed conceptualization. TTVV, THGN, LST performed writing-original draft. TTVV, THGN, LST performed writing-review and editing. TLV1 corresponding to Thi Loan Vo and TLV2 corresponding to Thi Luyen Vu. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by Gene Solutions.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability
</h2>
<p>DNA-seq from this study have been deposited in NCBI SRA as a Bioproject with accession number PRJNA1296750 [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. All data generated or analyzed during this study are included in this published article, its supplementary information files, and publicly available repositories. The source code for this study is publicly available in the GitHub repository at <a href="https://github.com/tuongvi2259/Breast_Cancer_Project" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tuongvi2259/Breast_Cancer_Project</a> and Zenodo at 10.5281/zenodo.16146086[<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>].</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par105">The research protocol was approved by the Ethics Committees of all participating institutions, with ethics review number 294/BVUB-HÄÄÄ for the discovery cohort and 460/HÄÄÄ-ÄHYD for the validation cohort. Informed written consent was obtained from each participant in accordance with the Declaration of Helsinki.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par106">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par107">The authors, including LST, DSN, HG, MDP, and HNN, hold equity in Gene Solutions. We confirm that this does not impact on our compliance with the journal's policies regarding data and material sharing.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Duc Hieu Vo, Email: hieuvobvub@gmail.com.</p>
<p>Le Son Tran, Email: leson1808@gmail.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References
</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Bray F, Laversanne M, Sung H, Ferlay J, Siegel RL, Soerjomataram I, et al. Global cancer statistics 2022: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J Clin. 2024;74(3):229â€“63.
</cite> [<a href="https://doi.org/10.3322/caac.21834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38572751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bray%20F,%20Laversanne%20M,%20Sung%20H,%20Ferlay%20J,%20Siegel%20RL,%20Soerjomataram%20I,%20et%20al.%20Global%20cancer%20statistics%202022:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries.%20CA%20Cancer%20J%20Clin.%202024;74(3):229%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Arnold M, Morgan E, Rumgay H, Mafra A, Singh D, Laversanne M, et al. Current and future burden of breast cancer: global statistics for 2020 and 2040. Breast. 2022;66:15â€“23.
</cite> [<a href="https://doi.org/10.1016/j.breast.2022.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9465273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36084384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arnold%20M,%20Morgan%20E,%20Rumgay%20H,%20Mafra%20A,%20Singh%20D,%20Laversanne%20M,%20et%20al.%20Current%20and%20future%20burden%20of%20breast%20cancer:%20global%20statistics%20for%202020%20and%202040.%20Breast.%202022;66:15%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Merino Bonilla JA, Torres Tabanera M, Ros Mendoza LH. Breast cancer in the 21st century: from early detection to new therapies. Radiologia. 2017;59(5):368â€“79.
</cite> [<a href="https://doi.org/10.1016/j.rx.2017.06.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28712528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merino%20Bonilla%20JA,%20Torres%20Tabanera%20M,%20Ros%20Mendoza%20LH.%20Breast%20cancer%20in%20the%2021st%20century:%20from%20early%20detection%20to%20new%20therapies.%20Radiologia.%202017;59(5):368%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Gerami R, Sadeghi Joni S, Akhondi N, Etemadi A, Fosouli M, Eghbal AF, et al. A literature review on the imaging methods for breast cancer. Int J Physiol Pathophysiol Pharmacol. 2022;14(3):171â€“6.
</cite> [<a href="/articles/PMC9301184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35891932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gerami%20R,%20Sadeghi%20Joni%20S,%20Akhondi%20N,%20Etemadi%20A,%20Fosouli%20M,%20Eghbal%20AF,%20et%20al.%20A%20literature%20review%20on%20the%20imaging%20methods%20for%20breast%20cancer.%20Int%20J%20Physiol%20Pathophysiol%20Pharmacol.%202022;14(3):171%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gilbert FJ, Tucker L, Young KC. Digital breast tomosynthesis (DBT): a review of the evidence for use as a screening tool. Clin Radiol. 2016;71(2):141â€“50.
</cite> [<a href="https://doi.org/10.1016/j.crad.2015.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26707815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gilbert%20FJ,%20Tucker%20L,%20Young%20KC.%20Digital%20breast%20tomosynthesis%20(DBT):%20a%20review%20of%20the%20evidence%20for%20use%20as%20a%20screening%20tool.%20Clin%20Radiol.%202016;71(2):141%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>TabÃ¡r L, Vitak B, Chen TH, Yen AM, Cohen A, Tot T, et al. Swedish two-county trial: impact of mammographic screening on breast cancer mortality during 3 decades. Radiology. 2011;260(3):658â€“63.
</cite> [<a href="https://doi.org/10.1148/radiol.11110469" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21712474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tab%C3%A1r%20L,%20Vitak%20B,%20Chen%20TH,%20Yen%20AM,%20Cohen%20A,%20Tot%20T,%20et%20al.%20Swedish%20two-county%20trial:%20impact%20of%20mammographic%20screening%20on%20breast%20cancer%20mortality%20during%203%20decades.%20Radiology.%202011;260(3):658%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lehman CD, Arao RF, Sprague BL, Lee JM, Buist DS, Kerlikowske K, et al. National performance benchmarks for modern screening digital mammography: update from the breast cancer surveillance consortium. Radiology. 2017;283(1):49â€“58.
</cite> [<a href="https://doi.org/10.1148/radiol.2016161174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5375631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27918707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lehman%20CD,%20Arao%20RF,%20Sprague%20BL,%20Lee%20JM,%20Buist%20DS,%20Kerlikowske%20K,%20et%20al.%20National%20performance%20benchmarks%20for%20modern%20screening%20digital%20mammography:%20update%20from%20the%20breast%20cancer%20surveillance%20consortium.%20Radiology.%202017;283(1):49%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Salem DS, Kamal RM, Mansour SM, Salah LA, Wessam R. Breast imaging in the young: the role of magnetic resonance imaging in breast cancer screening, diagnosis and follow-up. J Thorac Dis. 2013;5 Suppl 1(Suppl 1):S9â€“18.
</cite> [<a href="https://doi.org/10.3978/j.issn.2072-1439.2013.05.02" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3695543/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23819032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salem%20DS,%20Kamal%20RM,%20Mansour%20SM,%20Salah%20LA,%20Wessam%20R.%20Breast%20imaging%20in%20the%20young:%20the%20role%20of%20magnetic%20resonance%20imaging%20in%20breast%20cancer%20screening,%20diagnosis%20and%20follow-up.%20J%20Thorac%20Dis.%202013;5%20Suppl%201(Suppl%201):S9%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Boral S, Jagtap SV. Clinicohistopathological study of benign breast lesions in surgically excised specimens in a tertiary care hospital. J Cancer Res Ther. 2023;19(Supplement):S116â€“20.
</cite> [<a href="https://doi.org/10.4103/jcrt.JCRT_688_20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37147991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boral%20S,%20Jagtap%20SV.%20Clinicohistopathological%20study%20of%20benign%20breast%20lesions%20in%20surgically%20excised%20specimens%20in%20a%20tertiary%20care%20hospital.%20J%20Cancer%20Res%20Ther.%202023;19(Supplement):S116%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Klinger K, Bhimani C, Shames J, Sevrukov A. Fibroadenoma: from imaging evaluation to treatment. 2019.</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Brown AL, Vijapura C, Patel M, De La Cruz A, Wahab R. Breast cancer in dense breasts: detection challenges and supplemental screening opportunities. Radiographics. 2023;43(10): e230024.
</cite> [<a href="https://doi.org/10.1148/rg.230024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37792590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown%20AL,%20Vijapura%20C,%20Patel%20M,%20De%20La%20Cruz%20A,%20Wahab%20R.%20Breast%20cancer%20in%20dense%20breasts:%20detection%20challenges%20and%20supplemental%20screening%20opportunities.%20Radiographics.%202023;43(10):%20e230024." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Jaglan P, Dass R, Duhan M. Breast cancer detection techniques: issues and challenges. Journal of The Institution of Engineers (India): Series B. 2019;100(4):379â€“86.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jaglan%20P,%20Dass%20R,%20Duhan%20M.%20Breast%20cancer%20detection%20techniques:%20issues%20and%20challenges.%20Journal%20of%20The%20Institution%20of%20Engineers%20(India):%20Series%20B.%202019;100(4):379%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Merkkola-von Schantz PA, Jahkola TA, Krogerus LA, Hukkinen KS, Kauhanen SM. Should we routinely analyze reduction mammaplasty specimens? J Plast Reconstr Aesthet Surg. 2017;70(2):196â€“202.
</cite> [<a href="https://doi.org/10.1016/j.bjps.2016.10.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27908767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merkkola-von%20Schantz%20PA,%20Jahkola%20TA,%20Krogerus%20LA,%20Hukkinen%20KS,%20Kauhanen%20SM.%20Should%20we%20routinely%20analyze%20reduction%20mammaplasty%20specimens?%20J%20Plast%20Reconstr%20Aesthet%20Surg.%202017;70(2):196%E2%80%93202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Duque G, Manterola C, Otzen T, Arias C, Palacios D, Mora M, et al. Cancer biomarkers in liquid biopsy for early detection of breast cancer: a systematic review. Clin Med Insights Oncol. 2022;16: 11795549221134831.
</cite> [<a href="https://doi.org/10.1177/11795549221134831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9634213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36338263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duque%20G,%20Manterola%20C,%20Otzen%20T,%20Arias%20C,%20Palacios%20D,%20Mora%20M,%20et%20al.%20Cancer%20biomarkers%20in%20liquid%20biopsy%20for%20early%20detection%20of%20breast%20cancer:%20a%20systematic%20review.%20Clin%20Med%20Insights%20Oncol.%202022;16:%2011795549221134831." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Freitas AJA, Causin RL, Varuzza MB, Calfa S, Hidalgo Filho CMT, Komoto TT, et al. Liquid biopsy as a tool for the diagnosis, treatment, and monitoring of breast cancer. Int J Mol Sci. 2022. 10.3390/ijms23179952.
</cite> [<a href="https://doi.org/10.3390/ijms23179952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9456236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36077348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Freitas%20AJA,%20Causin%20RL,%20Varuzza%20MB,%20Calfa%20S,%20Hidalgo%20Filho%20CMT,%20Komoto%20TT,%20et%20al.%20Liquid%20biopsy%20as%20a%20tool%20for%20the%20diagnosis,%20treatment,%20and%20monitoring%20of%20breast%20cancer.%20Int%20J%20Mol%20Sci.%202022.%2010.3390/ijms23179952." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Cohen JD, Li L, Wang Y, Thoburn C, Afsari B, Danilova L, et al. Detection and localization of surgically resectable cancers with a multi-analyte blood test. Science. 2018;359(6378):926â€“30.
</cite> [<a href="https://doi.org/10.1126/science.aar3247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6080308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cohen%20JD,%20Li%20L,%20Wang%20Y,%20Thoburn%20C,%20Afsari%20B,%20Danilova%20L,%20et%20al.%20Detection%20and%20localization%20of%20surgically%20resectable%20cancers%20with%20a%20multi-analyte%20blood%20test.%20Science.%202018;359(6378):926%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Manoochehri M, Borhani N, GerhÃ¤user C, Assenov Y, SchÃ¶nung M, Hielscher T, et al. DNA methylation biomarkers for noninvasive detection of triple-negative breast cancer using liquid biopsy. Int J Cancer. 2023;152(5):1025â€“35.
</cite> [<a href="https://doi.org/10.1002/ijc.34337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36305646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manoochehri%20M,%20Borhani%20N,%20Gerh%C3%A4user%20C,%20Assenov%20Y,%20Sch%C3%B6nung%20M,%20Hielscher%20T,%20et%20al.%20DNA%20methylation%20biomarkers%20for%20noninvasive%20detection%20of%20triple-negative%20breast%20cancer%20using%20liquid%20biopsy.%20Int%20J%20Cancer.%202023;152(5):1025%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Liu J, Zhao H, Huang Y, Xu S, Zhou Y, Zhang W, et al. Genome-wide cell-free DNA methylation analyses improve accuracy of non-invasive diagnostic imaging for early-stage breast cancer. Mol Cancer. 2021;20(1):36.
</cite> [<a href="https://doi.org/10.1186/s12943-021-01330-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7893735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33608029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20J,%20Zhao%20H,%20Huang%20Y,%20Xu%20S,%20Zhou%20Y,%20Zhang%20W,%20et%20al.%20Genome-wide%20cell-free%20DNA%20methylation%20analyses%20improve%20accuracy%20of%20non-invasive%20diagnostic%20imaging%20for%20early-stage%20breast%20cancer.%20Mol%20Cancer.%202021;20(1):36." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Zhang Y, Cheng L. DNA methylation analysis of peripheral blood mononuclear cells in diagnosing breast cancer from benign breast lesions. J Transl Med. 2024;22(1):1070.
</cite> [<a href="https://doi.org/10.1186/s12967-024-05842-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11600940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39605043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Cheng%20L.%20DNA%20methylation%20analysis%20of%20peripheral%20blood%20mononuclear%20cells%20in%20diagnosing%20breast%20cancer%20from%20benign%20breast%20lesions.%20J%20Transl%20Med.%202024;22(1):1070." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Adashek JJ, Kato S, Lippman SM, Kurzrock R. The paradox of cancer genes in non-malignant conditions: implications for precision medicine. Genome Med. 2020;12(1):16.
</cite> [<a href="https://doi.org/10.1186/s13073-020-0714-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7027240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32066498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adashek%20JJ,%20Kato%20S,%20Lippman%20SM,%20Kurzrock%20R.%20The%20paradox%20of%20cancer%20genes%20in%20non-malignant%20conditions:%20implications%20for%20precision%20medicine.%20Genome%20Med.%202020;12(1):16." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Pham TMQ, Phan TH, Jasmine TX, Tran TTT, Huynh LAK, Vo TL, et al. Multimodal analysis of genome-wide methylation, copy number aberrations, and end motif signatures enhances detection of early-stage breast cancer. Front Oncol. 2023. 10.3389/fonc.2023.1127086.
</cite> [<a href="https://doi.org/10.3389/fonc.2023.1127086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10200909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37223690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pham%20TMQ,%20Phan%20TH,%20Jasmine%20TX,%20Tran%20TTT,%20Huynh%20LAK,%20Vo%20TL,%20et%20al.%20Multimodal%20analysis%20of%20genome-wide%20methylation,%20copy%20number%20aberrations,%20and%20end%20motif%20signatures%20enhances%20detection%20of%20early-stage%20breast%20cancer.%20Front%20Oncol.%202023.%2010.3389/fonc.2023.1127086." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Nguyen VTC, Nguyen TH, Doan NNT, Pham TMQ, Nguyen GTH, Nguyen TD, et al. Multimodal analysis of methylomics and fragmentomics in plasma cell-free DNA for multi-cancer early detection and localization. Elife. 2023;12: RP89083.
</cite> [<a href="https://doi.org/10.7554/eLife.89083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10567114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37819044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20VTC,%20Nguyen%20TH,%20Doan%20NNT,%20Pham%20TMQ,%20Nguyen%20GTH,%20Nguyen%20TD,%20et%20al.%20Multimodal%20analysis%20of%20methylomics%20and%20fragmentomics%20in%20plasma%20cell-free%20DNA%20for%20multi-cancer%20early%20detection%20and%20localization.%20Elife.%202023;12:%20RP89083." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Chandrananda D, Thorne NP, Bahlo M. High-resolution characterization of sequence signatures due to non-random cleavage of cell-free DNA. BMC Med Genomics. 2015;8:29.
</cite> [<a href="https://doi.org/10.1186/s12920-015-0107-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4469119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26081108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chandrananda%20D,%20Thorne%20NP,%20Bahlo%20M.%20High-resolution%20characterization%20of%20sequence%20signatures%20due%20to%20non-random%20cleavage%20of%20cell-free%20DNA.%20BMC%20Med%20Genomics.%202015;8:29." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Esteller M. Epigenetics in cancer. N Engl J Med. 2008;358(11):1148â€“59.
</cite> [<a href="https://doi.org/10.1056/NEJMra072067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18337604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Esteller%20M.%20Epigenetics%20in%20cancer.%20N%20Engl%20J%20Med.%202008;358(11):1148%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>An W, Lin H, Ma L, Zhang C, Zheng Y, Cheng Q, et al. Progesterone activates GPR126 to promote breast cancer development via the Gi pathway. Proc Natl Acad Sci U S A. 2022;119(15): e2117004119.
</cite> [<a href="https://doi.org/10.1073/pnas.2117004119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9169622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35394864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?An%20W,%20Lin%20H,%20Ma%20L,%20Zhang%20C,%20Zheng%20Y,%20Cheng%20Q,%20et%20al.%20Progesterone%20activates%20GPR126%20to%20promote%20breast%20cancer%20development%20via%20the%20Gi%20pathway.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202022;119(15):%20e2117004119." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Zhu KY, Tian Y, Li YX, Meng QX, Ge J, Cao XC, et al. The functions and prognostic value of KrÃ¼ppel-like factors in breast cancer. Cancer Cell Int. 2022;22(1): 23.
</cite> [<a href="https://doi.org/10.1186/s12935-022-02449-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8760734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35033064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20KY,%20Tian%20Y,%20Li%20YX,%20Meng%20QX,%20Ge%20J,%20Cao%20XC,%20et%20al.%20The%20functions%20and%20prognostic%20value%20of%20Kr%C3%BCppel-like%20factors%20in%20breast%20cancer.%20Cancer%20Cell%20Int.%202022;22(1):%2023." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Shi S, Xu C, Fang X, Zhang Y, Li H, Wen W, et al. Expression profile of Toll-like receptors in human breast cancer. Mol Med Rep. 2020;21(2):786â€“94.
</cite> [<a href="https://doi.org/10.3892/mmr.2019.10853" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6947885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31789409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20S,%20Xu%20C,%20Fang%20X,%20Zhang%20Y,%20Li%20H,%20Wen%20W,%20et%20al.%20Expression%20profile%20of%20Toll-like%20receptors%20in%20human%20breast%20cancer.%20Mol%20Med%20Rep.%202020;21(2):786%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Tsyganov MM, Ibragimova MK, Garbukov EY, Tsydenova IA, Gaptulbarova KA, Dolgasheva DS, et al. Predictive and prognostic significance of mRNA expression and DNA copies aberrations of ERCC1, RRM1, TOP1, TOP2A, TUBB3, TYMS, and GSTP1 genes in patients with breast cancer. Diagnostics. 2022. 10.3390/diagnostics12020405.
</cite> [<a href="https://doi.org/10.3390/diagnostics12020405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8871321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35204496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsyganov%20MM,%20Ibragimova%20MK,%20Garbukov%20EY,%20Tsydenova%20IA,%20Gaptulbarova%20KA,%20Dolgasheva%20DS,%20et%20al.%20Predictive%20and%20prognostic%20significance%20of%20mRNA%20expression%20and%20DNA%20copies%20aberrations%20of%20ERCC1,%20RRM1,%20TOP1,%20TOP2A,%20TUBB3,%20TYMS,%20and%20GSTP1%20genes%20in%20patients%20with%20breast%20cancer.%20Diagnostics.%202022.%2010.3390/diagnostics12020405." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Seal DB, Das V, Goswami S, De RK. Estimating gene expression from DNA methylation and copy number variation: a deep learning regression model for multi-omics integration. Genomics. 2020;112(4):2833â€“41.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2020.03.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32234433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seal%20DB,%20Das%20V,%20Goswami%20S,%20De%20RK.%20Estimating%20gene%20expression%20from%20DNA%20methylation%20and%20copy%20number%20variation:%20a%20deep%20learning%20regression%20model%20for%20multi-omics%20integration.%20Genomics.%202020;112(4):2833%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ponce-Bobadilla AV, Schmitt V, Maier CS, Mensing S, Stodtmann S. Practical guide to SHAP analysis: explaining supervised machine learning model predictions in drug development. Clin Transl Sci. 2024;17(11): e70056.
</cite> [<a href="https://doi.org/10.1111/cts.70056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11513550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39463176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ponce-Bobadilla%20AV,%20Schmitt%20V,%20Maier%20CS,%20Mensing%20S,%20Stodtmann%20S.%20Practical%20guide%20to%20SHAP%20analysis:%20explaining%20supervised%20machine%20learning%20model%20predictions%20in%20drug%20development.%20Clin%20Transl%20Sci.%202024;17(11):%20e70056." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Corces MR, Granja JM, Shams S, Louie BH, Seoane JA, Zhou W, et al. The chromatin accessibility landscape of primary human cancers. Science. 2018. 10.1126/science.aav1898.
</cite> [<a href="https://doi.org/10.1126/science.aav1898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6408149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30361341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Corces%20MR,%20Granja%20JM,%20Shams%20S,%20Louie%20BH,%20Seoane%20JA,%20Zhou%20W,%20et%20al.%20The%20chromatin%20accessibility%20landscape%20of%20primary%20human%20cancers.%20Science.%202018.%2010.1126/science.aav1898." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Bettegowda C, Sausen M, Leary RJ, Kinde I, Wang Y, Agrawal N, et al. Detection of circulating tumor DNA in early- and late-stage human malignancies. Sci Transl Med. 2014;6(224): 224ra24.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.3007094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4017867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24553385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bettegowda%20C,%20Sausen%20M,%20Leary%20RJ,%20Kinde%20I,%20Wang%20Y,%20Agrawal%20N,%20et%20al.%20Detection%20of%20circulating%20tumor%20DNA%20in%20early-%20and%20late-stage%20human%20malignancies.%20Sci%20Transl%20Med.%202014;6(224):%20224ra24." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Fumagalli C, Barberis M. Breast cancer heterogeneity. Diagnostics (Basel). 2021. 10.3390/diagnostics11091555.
</cite> [<a href="https://doi.org/10.3390/diagnostics11091555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8468623/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34573897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fumagalli%20C,%20Barberis%20M.%20Breast%20cancer%20heterogeneity.%20Diagnostics%20(Basel).%202021.%2010.3390/diagnostics11091555." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Matsutani A, Udagawa C, Matsunaga Y, Nakamura S, Zembutsu H. Liquid biopsy for the detection of clinical biomarkers in early breast cancer: new insights and challenges. Pharmacogenomics. 2020;21(5):359â€“67.
</cite> [<a href="https://doi.org/10.2217/pgs-2019-0130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32284011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matsutani%20A,%20Udagawa%20C,%20Matsunaga%20Y,%20Nakamura%20S,%20Zembutsu%20H.%20Liquid%20biopsy%20for%20the%20detection%20of%20clinical%20biomarkers%20in%20early%20breast%20cancer:%20new%20insights%20and%20challenges.%20Pharmacogenomics.%202020;21(5):359%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Tay TKY, Tan PH. Liquid biopsy in breast cancer: a focused review. Arch Pathol Lab Med. 2021;145(6):678â€“86.
</cite> [<a href="https://doi.org/10.5858/arpa.2019-0559-RA" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32045277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tay%20TKY,%20Tan%20PH.%20Liquid%20biopsy%20in%20breast%20cancer:%20a%20focused%20review.%20Arch%20Pathol%20Lab%20Med.%202021;145(6):678%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Lone SN, Nisar S, Masoodi T, Singh M, Rizwan A, Hashem S, et al. Liquid biopsy: a step closer to transform diagnosis, prognosis and future of cancer treatments. Mol Cancer. 2022;21(1):79.
</cite> [<a href="https://doi.org/10.1186/s12943-022-01543-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8932066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35303879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lone%20SN,%20Nisar%20S,%20Masoodi%20T,%20Singh%20M,%20Rizwan%20A,%20Hashem%20S,%20et%20al.%20Liquid%20biopsy:%20a%20step%20closer%20to%20transform%20diagnosis,%20prognosis%20and%20future%20of%20cancer%20treatments.%20Mol%20Cancer.%202022;21(1):79." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Guo L, Kong D, Liu J, Zhan L, Luo L, Zheng W, et al. Breast cancer heterogeneity and its implication in personalized precision therapy. Exp Hematol Oncol. 2023;12(1):3.
</cite> [<a href="https://doi.org/10.1186/s40164-022-00363-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9830930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36624542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20L,%20Kong%20D,%20Liu%20J,%20Zhan%20L,%20Luo%20L,%20Zheng%20W,%20et%20al.%20Breast%20cancer%20heterogeneity%20and%20its%20implication%20in%20personalized%20precision%20therapy.%20Exp%20Hematol%20Oncol.%202023;12(1):3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Garofoli M, Maiorano BA, Bruno G, Giordano G, Falagario UG, Necchi A, et al. Circulating tumor DNA: a new research frontier in urological oncology from localized to metastatic disease. Eur Urol Oncol. 2024;8(3):805-17.</cite> [<a href="https://doi.org/10.1016/j.euo.2024.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39627072/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Fridlyand J, Snijders AM, Ylstra B, Li H, Olshen A, Segraves R, et al. Breast tumor copy number aberration phenotypes and genomic instability. BMC Cancer. 2006;6: 96.
</cite> [<a href="https://doi.org/10.1186/1471-2407-6-96" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1459181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16620391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fridlyand%20J,%20Snijders%20AM,%20Ylstra%20B,%20Li%20H,%20Olshen%20A,%20Segraves%20R,%20et%20al.%20Breast%20tumor%20copy%20number%20aberration%20phenotypes%20and%20genomic%20instability.%20BMC%20Cancer.%202006;6:%2096." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Nakopoulou L, Panayotopoulou EG, Giannopoulou I, Tsirmpa I, Katsarou S, Mylona E, et al. Extra copies of chromosomes 16 and X in invasive breast carcinomas are related to aggressive phenotype and poor prognosis. J Clin Pathol. 2007;60(7):808â€“15.
</cite> [<a href="https://doi.org/10.1136/jcp.2006.037838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1995780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17596546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakopoulou%20L,%20Panayotopoulou%20EG,%20Giannopoulou%20I,%20Tsirmpa%20I,%20Katsarou%20S,%20Mylona%20E,%20et%20al.%20Extra%20copies%20of%20chromosomes%2016%20and%20X%20in%20invasive%20breast%20carcinomas%20are%20related%20to%20aggressive%20phenotype%20and%20poor%20prognosis.%20J%20Clin%20Pathol.%202007;60(7):808%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Grifoni A, Alonzi T, Alter G, Noonan DM, Landay AL, Albini A, et al. Impact of aging on immunity in the context of COVID-19, HIV, and tuberculosis. Front Immunol. 2023;14:1146704.
</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1146704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10246744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37292210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grifoni%20A,%20Alonzi%20T,%20Alter%20G,%20Noonan%20DM,%20Landay%20AL,%20Albini%20A,%20et%20al.%20Impact%20of%20aging%20on%20immunity%20in%20the%20context%20of%20COVID-19,%20HIV,%20and%20tuberculosis.%20Front%20Immunol.%202023;14:1146704." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Figueroa JD, Gierach GL, Duggan MA, Fan S, Pfeiffer RM, Wang Y, et al. Risk factors for breast cancer development by tumor characteristics among women with benign breast disease. Breast Cancer Res. 2021;23(1):34.
</cite> [<a href="https://doi.org/10.1186/s13058-021-01410-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7977564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33736682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Figueroa%20JD,%20Gierach%20GL,%20Duggan%20MA,%20Fan%20S,%20Pfeiffer%20RM,%20Wang%20Y,%20et%20al.%20Risk%20factors%20for%20breast%20cancer%20development%20by%20tumor%20characteristics%20among%20women%20with%20benign%20breast%20disease.%20Breast%20Cancer%20Res.%202021;23(1):34." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Huang A, Cao S, Tang L. The tumor microenvironment and inflammatory breast cancer. J Cancer. 2017;8(10):1884â€“91.
</cite> [<a href="https://doi.org/10.7150/jca.17595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5556652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28819386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20A,%20Cao%20S,%20Tang%20L.%20The%20tumor%20microenvironment%20and%20inflammatory%20breast%20cancer.%20J%20Cancer.%202017;8(10):1884%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Angeles AK, Janke F, Bauer S, Christopoulos P, Riediger AL, SÃ¼ltmann H. Liquid biopsies beyond mutation calling: genomic and epigenomic features of cell-free DNA in cancer. Cancers (Basel). 2021. 10.3390/cancers13225615.
</cite> [<a href="https://doi.org/10.3390/cancers13225615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8616179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34830770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Angeles%20AK,%20Janke%20F,%20Bauer%20S,%20Christopoulos%20P,%20Riediger%20AL,%20S%C3%BCltmann%20H.%20Liquid%20biopsies%20beyond%20mutation%20calling:%20genomic%20and%20epigenomic%20features%20of%20cell-free%20DNA%20in%20cancer.%20Cancers%20(Basel).%202021.%2010.3390/cancers13225615." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Jiang P, Sun K, Peng W, Cheng SH, Ni M, Yeung PC, et al. Plasma DNA end-motif profiling as a fragmentomic marker in cancer, pregnancy, and transplantation. Cancer Discov. 2020;10(5):664â€“73.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-19-0622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32111602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20P,%20Sun%20K,%20Peng%20W,%20Cheng%20SH,%20Ni%20M,%20Yeung%20PC,%20et%20al.%20Plasma%20DNA%20end-motif%20profiling%20as%20a%20fragmentomic%20marker%20in%20cancer,%20pregnancy,%20and%20transplantation.%20Cancer%20Discov.%202020;10(5):664%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Jin C, Liu X, Zheng W, Su L, Liu Y, Guo X, et al. Characterization of fragment sizes, copy number aberrations and 4-mer end motifs in cell-free DNA of hepatocellular carcinoma for enhanced liquid biopsy-based cancer detection. Mol Oncol. 2021;15(9):2377â€“89.
</cite> [<a href="https://doi.org/10.1002/1878-0261.13041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8410516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34133846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20C,%20Liu%20X,%20Zheng%20W,%20Su%20L,%20Liu%20Y,%20Guo%20X,%20et%20al.%20Characterization%20of%20fragment%20sizes,%20copy%20number%20aberrations%20and%204-mer%20end%20motifs%20in%20cell-free%20DNA%20of%20hepatocellular%20carcinoma%20for%20enhanced%20liquid%20biopsy-based%20cancer%20detection.%20Mol%20Oncol.%202021;15(9):2377%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Serpas L, Chan RWY, Jiang P, Ni M, Sun K, Rashidfarrokhi A, et al. Dnase1l3 deletion causes aberrations in length and end-motif frequencies in plasma DNA. Proc Natl Acad Sci U S A. 2019;116(2):641â€“9.
</cite> [<a href="https://doi.org/10.1073/pnas.1815031116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6329986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30593563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Serpas%20L,%20Chan%20RWY,%20Jiang%20P,%20Ni%20M,%20Sun%20K,%20Rashidfarrokhi%20A,%20et%20al.%20Dnase1l3%20deletion%20causes%20aberrations%20in%20length%20and%20end-motif%20frequencies%20in%20plasma%20DNA.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202019;116(2):641%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhu D, Wang H, Wu W, Geng S, Zhong G, Li Y, et al. Circulating cell-free DNA fragmentation is a stepwise and conserved process linked to apoptosis. BMC Biol. 2023;21(1):253.
</cite> [<a href="https://doi.org/10.1186/s12915-023-01752-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10642009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37953260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20D,%20Wang%20H,%20Wu%20W,%20Geng%20S,%20Zhong%20G,%20Li%20Y,%20et%20al.%20Circulating%20cell-free%20DNA%20fragmentation%20is%20a%20stepwise%20and%20conserved%20process%20linked%20to%20apoptosis.%20BMC%20Biol.%202023;21(1):253." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Zhu Z, Chen T, Zhang M, Shi X, Yu P, Liu J, et al. Dynamic profiling of cell-free DNA fragmentation uncovers postprandial metabolic and immune alterations. Hum Genomics. 2025;19(1):27.
</cite> [<a href="https://doi.org/10.1186/s40246-025-00739-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11921681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40102951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20Z,%20Chen%20T,%20Zhang%20M,%20Shi%20X,%20Yu%20P,%20Liu%20J,%20et%20al.%20Dynamic%20profiling%20of%20cell-free%20DNA%20fragmentation%20uncovers%20postprandial%20metabolic%20and%20immune%20alterations.%20Hum%20Genomics.%202025;19(1):27." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Markus H, Chandrananda D, Moore E, Mouliere F, Morris J, Brenton JD, et al. Refined characterization of circulating tumor DNA through biological feature integration. Sci Rep. 2022;12(1):1928.
</cite> [<a href="https://doi.org/10.1038/s41598-022-05606-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8816939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35121756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Markus%20H,%20Chandrananda%20D,%20Moore%20E,%20Mouliere%20F,%20Morris%20J,%20Brenton%20JD,%20et%20al.%20Refined%20characterization%20of%20circulating%20tumor%20DNA%20through%20biological%20feature%20integration.%20Sci%20Rep.%202022;12(1):1928." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Zhou Z, Ma ML, Chan RWY, Lam WKJ, Peng W, Gai W, et al. Fragmentation landscape of cell-free DNA revealed by deconvolutional analysis of end motifs. Proc Natl Acad Sci U S A. 2023;120(17): e2220982120.
</cite> [<a href="https://doi.org/10.1073/pnas.2220982120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10151549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37075072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20Z,%20Ma%20ML,%20Chan%20RWY,%20Lam%20WKJ,%20Peng%20W,%20Gai%20W,%20et%20al.%20Fragmentation%20landscape%20of%20cell-free%20DNA%20revealed%20by%20deconvolutional%20analysis%20of%20end%20motifs.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202023;120(17):%20e2220982120." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Bai Q, He X, Hu T. Pan-cancer analysis of the deoxyribonuclease gene family. Mol Clin Oncol. 2023;18(3): 19.
</cite> [<a href="https://doi.org/10.3892/mco.2023.2615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9926046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36798465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bai%20Q,%20He%20X,%20Hu%20T.%20Pan-cancer%20analysis%20of%20the%20deoxyribonuclease%20gene%20family.%20Mol%20Clin%20Oncol.%202023;18(3):%2019." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Szyf M, Pakneshan P, Rabbani SA. DNA methylation and breast cancer. Biochem Pharmacol. 2004;68(6):1187â€“97.
</cite> [<a href="https://doi.org/10.1016/j.bcp.2004.04.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15313416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szyf%20M,%20Pakneshan%20P,%20Rabbani%20SA.%20DNA%20methylation%20and%20breast%20cancer.%20Biochem%20Pharmacol.%202004;68(6):1187%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Sun W, Bunn P, Jin C, Little P, Zhabotynsky V, Perou CM, et al. The association between copy number aberration, DNA methylation and gene expression in tumor samples. Nucleic Acids Res. 2018;46(6):3009â€“18.
</cite> [<a href="https://doi.org/10.1093/nar/gky131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5887505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29529299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20W,%20Bunn%20P,%20Jin%20C,%20Little%20P,%20Zhabotynsky%20V,%20Perou%20CM,%20et%20al.%20The%20association%20between%20copy%20number%20aberration,%20DNA%20methylation%20and%20gene%20expression%20in%20tumor%20samples.%20Nucleic%20Acids%20Res.%202018;46(6):3009%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Gao C, Li H, Liu C, Wu J, Zhou C, Liu L, et al. Determination of genetic and epigenetic modifications-related prognostic biomarkers of breast cancer: genome high-throughput data analysis. J Oncol. 2021;2021: 2021:2143362.
</cite> [<a href="https://doi.org/10.1155/2021/2143362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8455195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34557230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20C,%20Li%20H,%20Liu%20C,%20Wu%20J,%20Zhou%20C,%20Liu%20L,%20et%20al.%20Determination%20of%20genetic%20and%20epigenetic%20modifications-related%20prognostic%20biomarkers%20of%20breast%20cancer:%20genome%20high-throughput%20data%20analysis.%20J%20Oncol.%202021;2021:%202021:2143362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Shang M, Chang C, Pei Y, Guan Y, Chang J, Li H. Potential management of circulating tumor DNA as a biomarker in triple-negative breast cancer. J Cancer. 2018;9(24):4627â€“34.
</cite> [<a href="https://doi.org/10.7150/jca.28458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6299380/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30588246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shang%20M,%20Chang%20C,%20Pei%20Y,%20Guan%20Y,%20Chang%20J,%20Li%20H.%20Potential%20management%20of%20circulating%20tumor%20DNA%20as%20a%20biomarker%20in%20triple-negative%20breast%20cancer.%20J%20Cancer.%202018;9(24):4627%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Simon RM, Subramanian J, Li MC, Menezes S. Using cross-validation to evaluate predictive accuracy of survival risk classifiers based on high-dimensional data. Brief Bioinform. 2011;12(3):203â€“14.
</cite> [<a href="https://doi.org/10.1093/bib/bbr001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3105299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21324971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simon%20RM,%20Subramanian%20J,%20Li%20MC,%20Menezes%20S.%20Using%20cross-validation%20to%20evaluate%20predictive%20accuracy%20of%20survival%20risk%20classifiers%20based%20on%20high-dimensional%20data.%20Brief%20Bioinform.%202011;12(3):203%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Yates LA, Aandahl Z, Richards SA, Brook BW. Cross validation for model selection: a review with examples from ecology. Ecol Monogr. 2023;93(1): e1557.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yates%20LA,%20Aandahl%20Z,%20Richards%20SA,%20Brook%20BW.%20Cross%20validation%20for%20model%20selection:%20a%20review%20with%20examples%20from%20ecology.%20Ecol%20Monogr.%202023;93(1):%20e1557." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Liu MC, Oxnard GR, Klein EA, Swanton C, Seiden MV. Sensitive and specific multi-cancer detection and localization using methylation signatures in cell-free DNA. Ann Oncol. 2020;31(6):745â€“59.
</cite> [<a href="https://doi.org/10.1016/j.annonc.2020.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8274402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33506766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20MC,%20Oxnard%20GR,%20Klein%20EA,%20Swanton%20C,%20Seiden%20MV.%20Sensitive%20and%20specific%20multi-cancer%20detection%20and%20localization%20using%20methylation%20signatures%20in%20cell-free%20DNA.%20Ann%20Oncol.%202020;31(6):745%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Harbeck N. Insights into biology of luminal HER2 vs. enriched HER2 subtypes: therapeutic implications. Breast. 2015;24(Suppl 2):S44â€“8.
</cite> [<a href="https://doi.org/10.1016/j.breast.2015.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26321480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harbeck%20N.%20Insights%20into%20biology%20of%20luminal%20HER2%20vs.%20enriched%20HER2%20subtypes:%20therapeutic%20implications.%20Breast.%202015;24(Suppl%202):S44%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Magbanua MJM, Brown Swigart L, Ahmed Z, Sayaman RW, Renner D, Kalashnikova E, et al. Clinical significance and biology of circulating tumor DNA in high-risk early-stage HER2-negative breast cancer receiving neoadjuvant chemotherapy. Cancer Cell. 2023;41(6):1091-102.e4.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2023.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10330514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37146605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magbanua%20MJM,%20Brown%20Swigart%20L,%20Ahmed%20Z,%20Sayaman%20RW,%20Renner%20D,%20Kalashnikova%20E,%20et%20al.%20Clinical%20significance%20and%20biology%20of%20circulating%20tumor%20DNA%20in%20high-risk%20early-stage%20HER2-negative%20breast%20cancer%20receiving%20neoadjuvant%20chemotherapy.%20Cancer%20Cell.%202023;41(6):1091-102.e4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Cheang MC, Chia SK, Voduc D, Gao D, Leung S, Snider J, et al. Ki67 index, HER2 status, and prognosis of patients with luminal B breast cancer. J Natl Cancer Inst. 2009;101(10):736â€“50.
</cite> [<a href="https://doi.org/10.1093/jnci/djp082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2684553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19436038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheang%20MC,%20Chia%20SK,%20Voduc%20D,%20Gao%20D,%20Leung%20S,%20Snider%20J,%20et%20al.%20Ki67%20index,%20HER2%20status,%20and%20prognosis%20of%20patients%20with%20luminal%20B%20breast%20cancer.%20J%20Natl%20Cancer%20Inst.%202009;101(10):736%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Coombes RC, Page K, Salari R, Hastings RK, Armstrong A, Ahmed S, et al. Personalized detection of circulating tumor DNA antedates breast cancer metastatic recurrence. Clin Cancer Res. 2019;25(14):4255â€“63.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-18-3663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30992300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coombes%20RC,%20Page%20K,%20Salari%20R,%20Hastings%20RK,%20Armstrong%20A,%20Ahmed%20S,%20et%20al.%20Personalized%20detection%20of%20circulating%20tumor%20DNA%20antedates%20breast%20cancer%20metastatic%20recurrence.%20Clin%20Cancer%20Res.%202019;25(14):4255%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Dawson SJ, Tsui DW, Murtaza M, Biggs H, Rueda OM, Chin SF, et al. Analysis of circulating tumor DNA to monitor metastatic breast cancer. N Engl J Med. 2013;368(13):1199â€“209.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1213261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23484797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dawson%20SJ,%20Tsui%20DW,%20Murtaza%20M,%20Biggs%20H,%20Rueda%20OM,%20Chin%20SF,%20et%20al.%20Analysis%20of%20circulating%20tumor%20DNA%20to%20monitor%20metastatic%20breast%20cancer.%20N%20Engl%20J%20Med.%202013;368(13):1199%E2%80%93209." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Wiratkapun C, Bunyapaiboonsri W, Wibulpolprasert B, Lertsithichai P. Biopsy rate and positive predictive value for breast cancer in BI-RADS category 4 breast lesions. J Med Assoc Thai. 2010;93(7):830â€“7.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/20649064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wiratkapun%20C,%20Bunyapaiboonsri%20W,%20Wibulpolprasert%20B,%20Lertsithichai%20P.%20Biopsy%20rate%20and%20positive%20predictive%20value%20for%20breast%20cancer%20in%20BI-RADS%20category%204%20breast%20lesions.%20J%20Med%20Assoc%20Thai.%202010;93(7):830%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Spick C, Bickel H, Polanec SH, Baltzer PA. Breast lesions classified as probably benign (BI-RADS 3) on magnetic resonance imaging: a systematic review and meta-analysis. Eur Radiol. 2018;28(5):1919â€“28.
</cite> [<a href="https://doi.org/10.1007/s00330-017-5127-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5882619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29168006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spick%20C,%20Bickel%20H,%20Polanec%20SH,%20Baltzer%20PA.%20Breast%20lesions%20classified%20as%20probably%20benign%20(BI-RADS%203)%20on%20magnetic%20resonance%20imaging:%20a%20systematic%20review%20and%20meta-analysis.%20Eur%20Radiol.%202018;28(5):1919%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>SÃ¡nchez-MartÃ­n V, LÃ³pez-LÃ³pez E, Reguero-Paredes D, Godoy-Ortiz A, DomÃ­nguez-Recio ME, JimÃ©nez-RodrÃ­guez B, et al. Comparative study of droplet-digital PCR and absolute Q digital PCR for ctDNA detection in early-stage breast cancer patients. Clin Chim Acta. 2024;552: 117673.
</cite> [<a href="https://doi.org/10.1016/j.cca.2023.117673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38007055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?S%C3%A1nchez-Mart%C3%ADn%20V,%20L%C3%B3pez-L%C3%B3pez%20E,%20Reguero-Paredes%20D,%20Godoy-Ortiz%20A,%20Dom%C3%ADnguez-Recio%20ME,%20Jim%C3%A9nez-Rodr%C3%ADguez%20B,%20et%20al.%20Comparative%20study%20of%20droplet-digital%20PCR%20and%20absolute%20Q%20digital%20PCR%20for%20ctDNA%20detection%20in%20early-stage%20breast%20cancer%20patients.%20Clin%20Chim%20Acta.%202024;552:%20117673." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Cristiano S, Leal A, Phallen J, Fiksel J, Adleff V, Bruhm DC, et al. Genome-wide cell-free DNA fragmentation in patients with cancer. Nature. 2019;570(7761):385â€“9.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1272-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6774252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31142840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cristiano%20S,%20Leal%20A,%20Phallen%20J,%20Fiksel%20J,%20Adleff%20V,%20Bruhm%20DC,%20et%20al.%20Genome-wide%20cell-free%20DNA%20fragmentation%20in%20patients%20with%20cancer.%20Nature.%202019;570(7761):385%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Han BW, Cai GX, Liu Q, Yang X, Guo ZW, Huang LM, et al. Noninvasive discrimination of benign and malignant breast lesions using genome-wide nucleosome profiles of plasma cell-free DNA. Clin Chim Acta. 2021;520:95â€“100.
</cite> [<a href="https://doi.org/10.1016/j.cca.2021.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34107314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20BW,%20Cai%20GX,%20Liu%20Q,%20Yang%20X,%20Guo%20ZW,%20Huang%20LM,%20et%20al.%20Noninvasive%20discrimination%20of%20benign%20and%20malignant%20breast%20lesions%20using%20genome-wide%20nucleosome%20profiles%20of%20plasma%20cell-free%20DNA.%20Clin%20Chim%20Acta.%202021;520:95%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Janni W, Rack B, Friedl TWP, Hartkopf AD, WiesmÃ¼ller L, Pfister K, et al. Detection of minimal residual disease and prediction of recurrence in breast cancer using a plasma-only circulating tumor DNA assay. ESMO Open. 2025;10(4): 104296.
</cite> [<a href="https://doi.org/10.1016/j.esmoop.2025.104296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11982450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40120523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Janni%20W,%20Rack%20B,%20Friedl%20TWP,%20Hartkopf%20AD,%20Wiesm%C3%BCller%20L,%20Pfister%20K,%20et%20al.%20Detection%20of%20minimal%20residual%20disease%20and%20prediction%20of%20recurrence%20in%20breast%20cancer%20using%20a%20plasma-only%20circulating%20tumor%20DNA%20assay.%20ESMO%20Open.%202025;10(4):%20104296." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Clarke CA, Keegan TH, Yang J, Press DJ, Kurian AW, Patel AH, et al. Age-specific incidence of breast cancer subtypes: understanding the black-white crossover. J Natl Cancer Inst. 2012;104(14):1094â€“101.
</cite> [<a href="https://doi.org/10.1093/jnci/djs264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3640371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22773826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clarke%20CA,%20Keegan%20TH,%20Yang%20J,%20Press%20DJ,%20Kurian%20AW,%20Patel%20AH,%20et%20al.%20Age-specific%20incidence%20of%20breast%20cancer%20subtypes:%20understanding%20the%20black-white%20crossover.%20J%20Natl%20Cancer%20Inst.%202012;104(14):1094%E2%80%93101." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Mouliere F, Chandrananda D, Piskorz AM, Moore EK, Morris J, Ahlborn LB, et al. Enhanced detection of circulating tumor DNA by fragment size analysis. Sci Transl Med. 2018. 10.1126/scitranslmed.aat4921.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.aat4921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6483061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30404863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mouliere%20F,%20Chandrananda%20D,%20Piskorz%20AM,%20Moore%20EK,%20Morris%20J,%20Ahlborn%20LB,%20et%20al.%20Enhanced%20detection%20of%20circulating%20tumor%20DNA%20by%20fragment%20size%20analysis.%20Sci%20Transl%20Med.%202018.%2010.1126/scitranslmed.aat4921." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals. NCBI Bioproject Accession: PRJNA1296750. 2025. <a href="https://www.ncbi.nlm.nih.gov/search/all/?term=PRJNA1296750" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/search/all/?term=PRJNA1296750</a>.</cite> [<a href="https://doi.org/10.1186/s12915-025-02371-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830871/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Van TTV. Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals. Zenodo; 2025.</cite> [<a href="https://doi.org/10.1186/s12915-025-02371-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12366216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830871/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366216/bin/12915_2025_2371_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12915_2025_2371_MOESM1_ESM.zip</a><sup> (1MB, zip) </sup><p>Additional file 1:Â Table S1-S4 and Figure S1-S5. Table S1. Detailed clinical information of all participants in the discovery cohort. Table S2. Summary of participantsâ€™ clinical features in the validation cohort. Table S3. Detailed clinical information of all participants in the validation cohort. Table S4. Comparison of model performance. Fig.S1. Comparison of Area Under the Curve (AUC) values obtained from tenfold cross-validation across five machine learning algorithms (Error bars representâ€‰Â±â€‰1 standard deviation across tenfold cross-validation). Fig. S2. The pie chart shows the percentages according to stages in BC patients (A), subtypes in BC patients (B), and subtypes of benign patients (C). Fig. S3. SHapley Additive exPlanations (SHapley Additive exPlanations (SHAP) analysis reveals the feature importances in the machine learning model.) of the model in training set. Fig. S4. Overlapping genes between 450 target regions and TCGA databaseâ€‹. Fig. S5. Log loss and accuracy across bootstrapped models on the discovery cohort.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>DNA-seq from this study have been deposited in NCBI SRA as a Bioproject with accession number PRJNA1296750 [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. All data generated or analyzed during this study are included in this published article, its supplementary information files, and publicly available repositories. The source code for this study is publicly available in the GitHub repository at <a href="https://github.com/tuongvi2259/Breast_Cancer_Project" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tuongvi2259/Breast_Cancer_Project</a> and Zenodo at 10.5281/zenodo.16146086[<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>].</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12915-025-02371-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12915_2025_Article_2371.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.5Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366216/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366216/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366216%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366216/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366216/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366216/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830871/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366216/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830871/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366216/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366216/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="e3QNGkqXGOj91bEF0GzgLxUlQ0Iz9NSytk2aG0y20k3vlyrNtfzNuTQGGrUCNqab">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
