
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Enhancing the prediction accuracy of groundnut yield by integrating significant markers and modeling genotype × environment interaction - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251F2A8AEF5683051F2A000A784830.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375858/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Enhancing the prediction accuracy of groundnut yield by integrating significant markers and modeling genotype × environment interaction">
<meta name="citation_author" content="Nelson Lubanga">
<meta name="citation_author_institution" content="Institute of Biological, Environmental and Rural Sciences, Aberystwyth University, Aberystwyth, UK">
<meta name="citation_author" content="Velma Okaron">
<meta name="citation_author_institution" content="Department of Agricultural Production, School of Agricultural Sciences, College of Agricultural and Environmental Sciences, Makerere University, Kampala, Uganda">
<meta name="citation_author" content="Davis M Gimode">
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT)–Nairobi, Nairobi, Kenya">
<meta name="citation_author" content="Reyna Persa">
<meta name="citation_author_institution" content="Agronomy Department, University of Florida, Gainesville, Florida, USA">
<meta name="citation_author" content="James Mwololo">
<meta name="citation_author_institution" content="ICRISAT ‐ Lilongwe, Lilongwe, Malawi">
<meta name="citation_author" content="David K Okello">
<meta name="citation_author_institution" content="National Semi‐Arid Resources Research Institute, Serere, Uganda">
<meta name="citation_author" content="Mildred Ochwo Ssemakula">
<meta name="citation_author_institution" content="Department of Agricultural Production, School of Agricultural Sciences, College of Agricultural and Environmental Sciences, Makerere University, Kampala, Uganda">
<meta name="citation_author" content="Thomas L Odong">
<meta name="citation_author_institution" content="Department of Agricultural Production, School of Agricultural Sciences, College of Agricultural and Environmental Sciences, Makerere University, Kampala, Uganda">
<meta name="citation_author" content="Wilfred Abincha">
<meta name="citation_author_institution" content="Kenya Agricultural and Livestock Research Organization (KALRO), City Square, Kenya">
<meta name="citation_author" content="Damaris A Odeny">
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT)–Nairobi, Nairobi, Kenya">
<meta name="citation_author" content="Diego Jarquin">
<meta name="citation_author_institution" content="Agronomy Department, University of Florida, Gainesville, Florida, USA">
<meta name="citation_publication_date" content="2025 Aug 24">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70105">
<meta name="citation_doi" content="10.1002/tpg2.70105">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375858/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375858/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375858/pdf/TPG2-18-e70105.pdf">
<meta name="description" content="Multi‐environment trials are routinely conducted in plant breeding to capture the genotype‐by‐environment interaction (G × E) effects. Significant G × E could alter the response pattern of genotypes (the change in rankings of genotypes), ...">
<meta name="og:title" content="Enhancing the prediction accuracy of groundnut yield by integrating significant markers and modeling genotype × environment interaction">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Multi‐environment trials are routinely conducted in plant breeding to capture the genotype‐by‐environment interaction (G × E) effects. Significant G × E could alter the response pattern of genotypes (the change in rankings of genotypes), ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375858/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375858">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70105"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70105.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375858%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375858/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375858/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375858/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70105" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 24;18(3):e70105. doi: <a href="https://doi.org/10.1002/tpg2.70105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70105</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Enhancing the prediction accuracy of groundnut yield by integrating significant markers and modeling genotype × environment interaction</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lubanga%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Nelson Lubanga</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Nelson Lubanga</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute of Biological, Environmental and Rural Sciences, Aberystwyth University, Aberystwyth, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lubanga%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nelson Lubanga</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okaron%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Velma Okaron</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Velma Okaron</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agricultural Production, School of Agricultural Sciences, College of Agricultural and Environmental Sciences, Makerere University, Kampala, Uganda</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okaron%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Velma Okaron</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gimode%20DM%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Davis M Gimode</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Davis M Gimode</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT)–Nairobi, Nairobi, Kenya</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gimode%20DM%22%5BAuthor%5D" class="usa-link"><span class="name western">Davis M Gimode</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Persa%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Reyna Persa</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Reyna Persa</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Agronomy Department, University of Florida, Gainesville, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Persa%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Reyna Persa</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mwololo%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">James Mwololo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">James Mwololo</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>ICRISAT ‐ Lilongwe, Lilongwe, Malawi</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mwololo%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">James Mwololo</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okello%20DK%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">David K Okello</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">David K Okello</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>National Semi‐Arid Resources Research Institute, Serere, Uganda</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okello%20DK%22%5BAuthor%5D" class="usa-link"><span class="name western">David K Okello</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ssemakula%20MO%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Mildred Ochwo Ssemakula</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Mildred Ochwo Ssemakula</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agricultural Production, School of Agricultural Sciences, College of Agricultural and Environmental Sciences, Makerere University, Kampala, Uganda</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ssemakula%20MO%22%5BAuthor%5D" class="usa-link"><span class="name western">Mildred Ochwo Ssemakula</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Odong%20TL%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Thomas L Odong</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Thomas L Odong</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agricultural Production, School of Agricultural Sciences, College of Agricultural and Environmental Sciences, Makerere University, Kampala, Uganda</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Odong%20TL%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas L Odong</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abincha%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Wilfred Abincha</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Wilfred Abincha</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Kenya Agricultural and Livestock Research Organization (KALRO), City Square, Kenya</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abincha%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wilfred Abincha</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Odeny%20DA%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Damaris A Odeny</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Damaris A Odeny</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT)–Nairobi, Nairobi, Kenya</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Odeny%20DA%22%5BAuthor%5D" class="usa-link"><span class="name western">Damaris A Odeny</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarquin%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Diego Jarquin</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Diego Jarquin</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Agronomy Department, University of Florida, Gainesville, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarquin%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Diego Jarquin</span></a>
</div>
</div>
<sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270105-aff-0001">
<sup>
<sup>1</sup>
</sup>Institute of Biological, Environmental and Rural Sciences, Aberystwyth University, Aberystwyth, UK</div>
<div id="tpg270105-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Agricultural Production, School of Agricultural Sciences, College of Agricultural and Environmental Sciences, Makerere University, Kampala, Uganda</div>
<div id="tpg270105-aff-0003">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT)–Nairobi, Nairobi, Kenya</div>
<div id="tpg270105-aff-0004">
<sup>
<sup>4</sup>
</sup>Agronomy Department, University of Florida, Gainesville, Florida, USA</div>
<div id="tpg270105-aff-0005">
<sup>
<sup>5</sup>
</sup>ICRISAT ‐ Lilongwe, Lilongwe, Malawi</div>
<div id="tpg270105-aff-0006">
<sup>
<sup>6</sup>
</sup>National Semi‐Arid Resources Research Institute, Serere, Uganda</div>
<div id="tpg270105-aff-0007">
<sup>
<sup>7</sup>
</sup>Kenya Agricultural and Livestock Research Organization (KALRO), City Square, Kenya</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Damaris A. Odeny, International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT)–Nairobi, P.O Box 39063‐00623 Nairobi, Kenya. Email: <span>Damaris.Odeny@icrisat.org</span>
, Diego Jarquin, Agronomy Department, University of Florida, 2089 Mccarty Hall B, P.O. Box 110500, Gainesville, FL 32611‐7011, USA. Email: <span>jhernandezjarqui@ufl.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 11; Received 2025 Mar 7; Accepted 2025 Jul 26; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375858</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Multi‐environment trials are routinely conducted in plant breeding to capture the genotype‐by‐environment interaction (G × E) effects. Significant G × E could alter the response pattern of genotypes (the change in rankings of genotypes), subsequently complicating the selection process. Four genomic prediction (GP) models were assessed in three groundnut yield‐related traits: pod yield (PY), seed weight (SW), and 100 seed weight (SW100), across four environments. The models, M1 (environment + line), M2 (environment + line + genomic), M3 (environment + line + genomic + genomic × environment interaction), and M4 (environment + line + genomic + genomic × environment interaction + significant markers), were tested using four cross‐validation (CV) schemes (CV2, CV1, CV0, and CV00), each simulating different practical breeding scenarios. The results revealed that models incorporating marker data (M2, M3, and M4) consistently improved predictive ability in comparison to the phenotypic model (M1). Incorporating G × E (M3 and M4) further improved predictive ability and reduced residual and environmental variances. The inclusion of significant markers and G × E was more advantageous in CV1 and CV00 scenarios, demonstrating that this strategy is especially useful when phenotypic data for the target genotypes is limited or unavailable. Across the CV schemes, predictive ability was higher in CV2, suggesting that including additional information on the performance of genotypes in known environments can increase the accuracy of selecting superior genotypes in breeding programs. Integrating significant markers and modeling G × E in GP models could be an effective approach in groundnut breeding programs to accelerate genetic gains.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270105-list-10001" class="list" style="list-style-type:disc">
<li><p>Including marker data in genomic prediction (GP) models significantly increased the predictive ability.</p></li>
<li><p>Incorporating genotype‐by‐environment interaction (G × E) in GP models improved the predictive ability and reduced residual variances.</p></li>
<li><p>Integrating significant markers and modeling G × E in GP models enhanced the predictive ability.</p></li>
</ul></section><section class="abstract" id="abstract3"><h2>Plain Language Summary</h2>
<p>Genotype‐by‐environment interaction (G × E) is an important consideration in plant breeding, particularly for crops like groundnut, where environmental variability can significantly influence the yield. Effectively modeling G × E can enhance the ability to predict how different genotypes perform across diverse environments, which is essential for identifying promising genotypes that are high‐yielding and stable. Using molecular marker data in genomic prediction models improves the accuracy of selecting the best genotypes. Including significant markers as fixed effects in these models can further increase predictive ability by making use of known genetic information. This approach allows for the direct utilization of known genetic information, thereby refining the selection of superior genotypes in breeding programs. Our findings showed that this approach enhanced the predictive ability and can be implemented in groundnut breeding programs to select superior high‐yielding genotypes.</p></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270105-dl-0001">
<dt>BLUEs</dt>
<dd><p>best linear unbiased estimates</p></dd>
<dt>CGM</dt>
<dd><p>crop growth models</p></dd>
<dt>CV</dt>
<dd><p>cross‐validation</p></dd>
<dt>EC</dt>
<dd><p>environmental covariables</p></dd>
<dt>G × E</dt>
<dd><p>genotype‐by‐environment interaction</p></dd>
<dt>G × Y</dt>
<dd><p>genotype‐by‐year</p></dd>
<dt>GBLUP</dt>
<dd><p>genomic best linear unbiased prediction</p></dd>
<dt>GP</dt>
<dd><p>genomic prediction</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>ICRISAT</dt>
<dd><p>nternational Crops Research Institute for the Semi‐Arid Tropics</p></dd>
<dt>M × E</dt>
<dd><p>marker × environment interaction</p></dd>
<dt>MET</dt>
<dd><p>multi‐environment trial</p></dd>
<dt>PY</dt>
<dd><p>pod yield</p></dd>
<dt>SSA</dt>
<dd><p>sub‐Saharan Africa</p></dd>
<dt>SW</dt>
<dd><p>seed weight</p></dd>
<dt>WGP</dt>
<dd><p>whole genome prediction</p></dd>
</dl>
<section id="tpg270105-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Groundnut (<em>Arachis hypogaea L</em>.) is a vital crop primarily cultivated by smallholder farmers in tropical and subtropical countries for food security and income. It plays a significant role in sub‐Saharan Africa (SSA), where it is a major source of dietary protein (Toomer, <a href="#tpg270105-bib-0040" class="usa-link" aria-describedby="tpg270105-bib-0040">2018</a>), healthy fats (Mora‐Escobedo et al., <a href="#tpg270105-bib-0028" class="usa-link" aria-describedby="tpg270105-bib-0028">2015</a>), essential vitamins (Arya et al., <a href="#tpg270105-bib-0001" class="usa-link" aria-describedby="tpg270105-bib-0001">2016</a>), and micronutrients (Kurapati et al., <a href="#tpg270105-bib-0019" class="usa-link" aria-describedby="tpg270105-bib-0019">2021</a>). Despite its significance in SSA, groundnut yields remain remarkably low, averaging ∼1017 kg/ha compared to 4505 kg/ha recorded in the United States (FAO, <a href="#tpg270105-bib-0013" class="usa-link" aria-describedby="tpg270105-bib-0013">2022</a>; USDA‐NASS, <a href="#tpg270105-bib-0041" class="usa-link" aria-describedby="tpg270105-bib-0041">2021</a>). Groundnut yield losses in SSA have been further worsened by climate change effects (Yeleliere et al., <a href="#tpg270105-bib-0045" class="usa-link" aria-describedby="tpg270105-bib-0045">2023</a>) that include prolonged dry periods, heavy rainfall, hail damage, increased temperatures, and the emergence of pests and diseases (Batley &amp; Edwards, <a href="#tpg270105-bib-0004" class="usa-link" aria-describedby="tpg270105-bib-0004">2016</a>). Developing climate‐resilient groundnut varieties is therefore an important breeding objective for improving yields in SSA (Asibuo et al., <a href="#tpg270105-bib-0002" class="usa-link" aria-describedby="tpg270105-bib-0002">2018</a>; Tabe‐O et al., <a href="#tpg270105-bib-0037" class="usa-link" aria-describedby="tpg270105-bib-0037">2023</a>).</p>
<p>Evaluating advanced breeding lines in multi‐environment trials (METs) is a crucial step in the varietal development process (Janila et al., <a href="#tpg270105-bib-0047" class="usa-link" aria-describedby="tpg270105-bib-0047">2013</a>). The METs enable the assessment of genotype stability through studying genotype‐by‐environment interaction (G × E) (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>). Significant G × E occurs when genotypes respond differently to changes in the environment, potentially complicating the selection of high‐yielding groundnut genotypes (Asibuo et al., <a href="#tpg270105-bib-0002" class="usa-link" aria-describedby="tpg270105-bib-0002">2018</a>). Similarly, significant G × E can alter the relative performance rankings of genotypes across environments.</p>
<p>Genomic selection (GS) involves the application of genomic prediction (GP) models to estimate breeding values of genotypes based on genome‐wide markers (Bernardo, <a href="#tpg270105-bib-0005" class="usa-link" aria-describedby="tpg270105-bib-0005">1994</a>; Meuwissen et al., <a href="#tpg270105-bib-0026" class="usa-link" aria-describedby="tpg270105-bib-0026">2001</a>). The accurate capture of G × E enables the identification of superior and stable genotypes across different environments (Crossa et al., <a href="#tpg270105-bib-0009" class="usa-link" aria-describedby="tpg270105-bib-0009">2006</a>; Crossa et al., <a href="#tpg270105-bib-0048" class="usa-link" aria-describedby="tpg270105-bib-0048">2014</a>). Single‐environment GP models ignore G × E, and are therefore unreliable (Crossa et al., <a href="#tpg270105-bib-0011" class="usa-link" aria-describedby="tpg270105-bib-0011">2017</a>) as they fail to capture the heterogeneity of genetic variance across environments, or the imperfect genetic correlation among environments (Lopez‐Cruz et al., <a href="#tpg270105-bib-0021" class="usa-link" aria-describedby="tpg270105-bib-0021">2015</a>). Modeling covariance matrices by incorporating G × E allows for the utilization of information from correlated environments, thereby enhancing the accuracy of predicting genotypes (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>).</p>
<p>The accuracy of GP models is evaluated using random cross‐validation (CV) methods that partition the dataset into training and testing sets (Pérez‐Cabal et al., <a href="#tpg270105-bib-0031" class="usa-link" aria-describedby="tpg270105-bib-0031">2012</a>). The CV2 method, in which known genotypes are evaluated in known environments with unknown combinations of genotype and environments, is particularly well‐suited for scenarios involving incomplete field trials (sparse testing) (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>). The primary objective of CV2 is to predict the performance of genotypes in environments where they have not yet been tested. An alternative method, CV1, predicts the performance of untested genotypes in known environments (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>). In the CV1 scenario, the accuracy of prediction is influenced by the genetic relatedness between the training and testing sets. A third alternative is CV0, which is used to predict the performance of tested genotypes in new environments where they have not been evaluated (Jarquín et al., <a href="#tpg270105-bib-0017" class="usa-link" aria-describedby="tpg270105-bib-0017">2017</a>). Another method, CV00, predicts untested genotypes in unknown environments, and is therefore, the most challenging strategy (Jarquín et al., <a href="#tpg270105-bib-0017" class="usa-link" aria-describedby="tpg270105-bib-0017">2017</a>).</p>
<p>The first statistical framework for analyzing G × E was the Finlay–Wilkinson model, also known as the joint regression analysis that was first introduced by Yates and Cochran (<a href="#tpg270105-bib-0044" class="usa-link" aria-describedby="tpg270105-bib-0044">1938</a>) and later published by Finlay and Wilkinson (<a href="#tpg270105-bib-0014" class="usa-link" aria-describedby="tpg270105-bib-0014">1963</a>). In this method, the performance of each genotype in individual environments is regressed against an environmental index, which represents the overall effect of each environment within the trial. This index is typically defined as the mean performance of all genotypes in a given environment. Burgueño et al. (<a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>) were the first to use the genomic best linear unbiased prediction (GBLUP) model in a multi‐environment context where G × E was modeled with different variance and covariances among environments (factor analytic). However, the model does not incorporate environmental covariables (EC), which limits its ability to predict performance in new environments. Jarquín et al. (<a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>) introduced the Bayesian reaction‐norm model incorporating G × E as functions of markers and ECs. In the Bayesian reaction‐norm model, the main effects of markers and EC, as well as their interactions (marker × EC), are introduced using covariance structures that are functions of markers and EC. This model extends the standard GBLUP by fitting all markers, ECs, and marker × EC interaction effects as random effects, modeled jointly through a multiplicative covariance structure. Specifically, the interaction term follows a multivariate normal distribution with covariance defined as the Hadamard (element‐wise) product of the genomic and environmental relationship matrices. Including EC in this model allows predicting genotypes in new environments.</p>
<p>Lopez‐Cruz et al. (<a href="#tpg270105-bib-0021" class="usa-link" aria-describedby="tpg270105-bib-0021">2015</a>) proposed a marker × environment interaction (M × E) model, in which marker effects and genomic values are partitioned into components that are stable across environments (main effects) and others that are environment‐specific (interactions). The M × E model estimates the phenotypic correlation between any two environments as the ratio of variance component estimates. The covariance (and thus the correlation) between environments is derived from the variance components associated with the genetic effects across environments. Since variance components are constrained to be nonnegative, this formulation inherently restricts the covariance (and thus correlation) between environments to be positive. Cuevas et al. (<a href="#tpg270105-bib-0012" class="usa-link" aria-describedby="tpg270105-bib-0012">2016</a>) extended the M × E model by using a nonlinear (Gaussian) kernel to model G × E. Similarly, Crossa, los Campos, et al. (<a href="#tpg270105-bib-0010" class="usa-link" aria-describedby="tpg270105-bib-0010">2016</a>) extended the M × E model using priors that produce shrinkage and variable selection, that is, Bayesian ridge regression and BayesB.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270105-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270105-list-0001" class="list" style="list-style-type:disc">
<li><p>Including marker data in genomic prediction (GP) models significantly increased the predictive ability.</p></li>
<li><p>Incorporating genotype‐by‐environment interaction (G × E) in GP models improved the predictive ability and reduced residual variances.</p></li>
<li><p>Integrating significant markers and modeling G × E in GP models enhanced the predictive ability.</p></li>
</ul></section></section><p>In this study, we implemented the reaction‐norm model (Jarquín et al., <a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>) by incorporating the interaction between markers and EC. Previous studies have demonstrated that GP models that include significant markers associated with traits of interest as fixed effects, in combination with genome‐wide markers, achieve higher accuracy than standard GP models such as multiple linear regression, G‐BLUP, Bayesian Lasso, and Bayes Cπ (Rutkoski et al., <a href="#tpg270105-bib-0035" class="usa-link" aria-describedby="tpg270105-bib-0035">2014</a>). Spindel et al. (<a href="#tpg270105-bib-0036" class="usa-link" aria-describedby="tpg270105-bib-0036">2016</a>) presented a method where genome‐wide association study (GWAS) was conducted on a training set, and markers passing a threshold were modeled as fixed effects in the RR‐BLUP model. This approach, which they termed GS + de novo GWAS, improved the prediction accuracy by 10% compared to six other GP and marker‐assisted selection (MAS) approaches in rice (<em>Oryza sativa</em> L.) and <em>Capsicum annuum</em>. Kim et al. (<a href="#tpg270105-bib-0018" class="usa-link" aria-describedby="tpg270105-bib-0018">2022</a>) also reported an improved prediction accuracy of 5.3% for capsaicinoid contents in <em>Capsicum annuum</em>. These instances provide evidence that including significant markers as fixed effects in GP models has the potential to improve the prediction accuracy. The current study incorporated significant markers from previous GWAS analyses (Okaron et al., <a href="#tpg270105-bib-0029" class="usa-link" aria-describedby="tpg270105-bib-0029">2024</a>) with the following objectives: (1) study the effects of modeling G × E in GP models for groundnut yield and (2) determine whether incorporating significant markers from GWAS as fixed effects in GP models could enhance predictive ability.</p></section><section id="tpg270105-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270105-sec-0040"><h3 class="pmc_sec_title">2.1. Germplasm</h3>
<p>A total of 192 advanced breeding lines sourced from the International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT) ‐ Malawi were used in this study (Table <a href="#tpg270105-supinfo-0001" class="usa-link">S1</a>). ICRISAT utilises extensive germplasm collections to generate high‐yielding and climate‐resilient varieties in Asia and SSA. The field trial evaluations were conducted in four locations: Serere (1.4994° N, 33.5490° E) and Nakabango (0°19′36″ N, 33°53′36″ E) in Uganda, and Chitala (13°40′59″ S, 34°16′0″ E) and Chitedze (13.9815° S, 33.6372° E) in Malawi. The trials were planted in a 14 × 25 Alpha Lattice design with two replicates. Each plot was 1 × 0.9 m, comprising 3 m‐long rows with 45 cm inter‐row spacing, 60 cm between plots, and 2 m between replicated blocks. Phenotyping was conducted at Chitedze and Chitala (Malawi) during the 2020/21 cropping season (December–April), and at Serere and Nakabango (Uganda) during the long rainy season of 2021. These sites represent major groundnut‐producing agroecological zones in Eastern and Southern Africa.</p></section><section id="tpg270105-sec-0050"><h3 class="pmc_sec_title">2.2. Phenotyping, genotyping, and quality control</h3>
<p>All genotypes were harvested at the end of the growing season and evaluated for PY and other yield‐associated traits, including SW and SW100 in the four environments. Details of the plant material, experimental design, phenotyping and genotyping procedure are described in our recent study by Okaron et al. (<a href="#tpg270105-bib-0029" class="usa-link" aria-describedby="tpg270105-bib-0029">2024</a>). In brief, pods from each plot were harvested and sun‐dried to a moisture content below 13%, then weighed using a weighing scale, and recorded in grams as PY. For SW, seeds from each plot were shelled and weighed in grams. Additionally, 100 kernels were randomly selected from each plot, weighed, and the data recorded in grams as SW100.</p>
<p>SNP filtering involved removing all markers with &gt;20% missing values and a minor allele frequency below 5%. SNPs with a call rate lower than 80% and heterozygosity larger than 95% were also excluded. The final retained genotypes contained 38,853 SNPs. Filtered SNPs were converted to numeric allele classes (0, 1, and 2) for homozygous minor, heterozygous and homozygous major alleles, respectively using Plink 1.9 (Purcell et al., <a href="#tpg270105-bib-0033" class="usa-link" aria-describedby="tpg270105-bib-0033">2007</a>).</p></section><section id="tpg270105-sec-0060"><h3 class="pmc_sec_title">2.3. Phenotypic analysis</h3>
<p>The phenotypic analyses were performed in two stages. The first stage involved calculating the best linear unbiased estimates (BLUEs) for each genotype within each environment using Equation (<a href="#tpg270105-disp-0001" class="usa-link">1</a>). The BLUEs for each trait in each environment were extracted using ASReml‐R Version 4.2 (Butler et al., <a href="#tpg270105-bib-0008" class="usa-link" aria-describedby="tpg270105-bib-0008">2023</a>).</p>
<table class="disp-formula p" id="tpg270105-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><msub><mi>y</mi><mi>ikl</mi></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>L</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>R</mi><mi>k</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mi>y</mi></mrow></math></span>
<em>
<sub>ikl</sub>
</em> represents the <em>i</em>th genotype observed in the the <em>l</em>th incomplete block nested within <em>k</em>th replicate<em>. μ</em> is the overall mean (intercept), <em>L<sub>i</sub>
</em> is the fixed effect of the <em>i</em>th genotype, <em>R<sub>k</sub>
</em> is the random effect of the <em>k</em>th replicate, assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mi>R</mi></mrow></math></span>
<em>
<sub>k</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>R</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>R</mi><mn>2</mn></msubsup></mrow></math></span> is the variance of the replicate. <em>N</em>(.,.) denotes a normal density, iid stands for independent and identically distributed observations, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub></mrow></math></span> is the random effect of the <em>l</em>th block nested within the <em>k</em>th replicate such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>B</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>B</mi><mn>2</mn></msubsup></mrow></math></span> is the variance of the block effect nested within replication, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></math></span> is the random error term with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>k</mi><mi>l</mi></mrow></msub><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mrow></math></span> as the error variance.</p>
<p>To estimate the broad‐sense heritability (<em>H</em>
<sup>2</sup>) for each trait at each environment, we extracted the variances of each effect using the following model whereby genotype, replicate, and the block within replicate were considered random effects:</p>
<table class="disp-formula p" id="tpg270105-disp-0002"><tr>
<td class="formula"><math id="jats-math-11" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>k</mi><mi>l</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>L</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>R</mi><mi>k</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></math></span> represents the observed value, <em>μ</em> is the intercept, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><msub><mi>L</mi><mi>i</mi></msub></mrow></math></span> is the random effect of the <em>i</em>th genotype, assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mi>L</mi></mrow></math></span>
<em>
<sub>i</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the genotype variance, <em>R<sub>k</sub>
</em> is the random effect of the <em>k</em>th replicate, assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mi>R</mi></mrow></math></span>
<em>
<sub>k</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>R</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>R</mi><mn>2</mn></msubsup></mrow></math></span> is the variance of the replicate, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub></mrow></math></span> is the random effect of the <em>l</em>th block nested within the <em>k</em>th replicate such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mrow><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>B</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>B</mi><mn>2</mn></msubsup></mrow></math></span> is the variance of the block effect nested within replication, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub></mrow></math></span> is the random error term with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the error variance. Broad‐sense heritability was then calculated using Equation (<a href="#tpg270105-disp-0003" class="usa-link">3</a>):
</p>
<table class="disp-formula p" id="tpg270105-disp-0003"><tr>
<td class="formula"><math id="jats-math-25" display="block"><mrow><mrow><msup><mi>H</mi><mn>2</mn></msup><mo linebreak="badbreak">=</mo><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>e</mi><mo>/</mo><mi>r</mi></mrow><mn>2</mn></msubsup></mrow></mfrac></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow></math></span> is genotypic variance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></math></span> is error variance, and <em>r</em> is the number of replicates within each environment.</p>
<p>Broad‐sense heritability across the environment was calculated using the following linear model:
</p>
<table class="disp-formula p" id="tpg270105-disp-0004"><tr>
<td class="formula"><math id="jats-math-28" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>L</mi><mi mathvariant="normal">i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>R</mi><mi>k</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub><mo linebreak="goodbreak">+</mo><mi>L</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mi>y</mi></mrow></math></span>
<em>
<sub>ijkl</sub>
</em> represents the <em>i</em>th genotype observed in the <em>j</em>th environment at the <em>l</em>th incomplete block nested within <em>k</em>th replicate<em>. μ</em> is the overall mean (intercept), <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><msub><mi>L</mi><mi>i</mi></msub></mrow></math></span> is the random effect of the <em>i</em>th genotype, assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mi>L</mi></mrow></math></span>
<em>
<sub>i</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the genotype variance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><msub><mi>E</mi><mi>j</mi></msub></mrow></math></span> is the random effect of the <em>j</em>th environment, assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><mi>E</mi></mrow></math></span>
<em>
<sub>j</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>E</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>E</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the variance of the environment, <em>R<sub>k</sub>
</em> is the random effect of the <em>k</em>th replicate, assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-38" display="inline"><mrow><mi>R</mi></mrow></math></span>
<em>
<sub>k</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>R</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>R</mi><mn>2</mn></msubsup></mrow></math></span> is the variance of the replicate, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-41" display="inline"><mrow><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub></mrow></math></span> is the random effect of the <em>l</em>th block nested within the <em>k</em>th replicate such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><mrow><msub><mi>B</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>B</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>B</mi><mn>2</mn></msubsup></mrow></math></span> is the variance component of the block effect nested within replication, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mrow><mi>L</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></span> is the G × E of the <em>i</em>th genotype in <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-45" display="inline"><mrow><mrow><mi>L</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>L</mi><mi>E</mi></mrow><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-46" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>L</mi><mi>E</mi></mrow><mn>2</mn></msubsup></mrow></math></span> is the G × E variance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-47" display="inline"><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></math></span> is the random error term with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-48" display="inline"><mrow><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the error variance of the non‐explained variability by the previous model terms. Broad‐sense heritability was then calculated on an entry‐mean basis using the following equation:
</p>
<table class="disp-formula p" id="tpg270105-disp-0005"><tr>
<td class="formula"><math id="jats-math-49" display="block"><mrow><mrow><msup><mi>H</mi><mn>2</mn></msup><mo linebreak="badbreak">=</mo><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>L</mi><mi>E</mi><mo>/</mo><mi>n</mi></mrow><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>e</mi><mo>/</mo><mi>n</mi><mo>×</mo><mi>r</mi></mrow><mn>2</mn></msubsup></mrow></mfrac></mrow></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-50" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow></math></span> is genotypic variance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-51" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>L</mi><mi mathvariant="normal">E</mi></mrow><mn>2</mn></msubsup></mrow></math></span> is the G × E variance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-52" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is error variance, and <em>r</em> is the number of replicates within each environment and n is the number of environments.</p></section><section id="tpg270105-sec-0070"><h3 class="pmc_sec_title">2.4. Prediction models</h3>
<p>The second stage of the analysis involved using the BLUEs obtained in the first stage (Equation <a href="#tpg270105-disp-0001" class="usa-link">1</a>) to fit four different prediction models described below in Equations (<a href="#tpg270105-disp-0006" class="usa-link">(6)</a>, <a href="#tpg270105-disp-0007" class="usa-link">(7)</a>, <a href="#tpg270105-disp-0008" class="usa-link">(8)</a>, <a href="#tpg270105-disp-0009" class="usa-link">(9)</a>).</p>
<section id="tpg270105-sec-0080"><h4 class="pmc_sec_title">2.4.1. M1: Environment + line (E + L)</h4>
<p>In this model, the response variable corresponds to the adjusted phenotypes (<em>y</em>
<sub>ij</sub>) (BLUEs) obtained after fitting the corresponding mixed linear model as previously described in Equation (<a href="#tpg270105-disp-0001" class="usa-link">1</a>). <em>y</em>
<sub>ij</sub> represents the response of the <em>i</em>th genotype at the <em>j</em>th environment. In this case, the environment and line effects are considered random outcomes:
</p>
<table class="disp-formula p" id="tpg270105-disp-0006"><tr>
<td class="formula"><math id="jats-math-53" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>L</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>where <em>L<sub>i</sub>
</em> is the random effect of the <em>i</em>th line assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-54" display="inline"><mrow><mi>L</mi></mrow></math></span>
<em>
<sub>i</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-55" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-56" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>L</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the variance component of the genotype effect, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-57" display="inline"><mrow><msub><mi>E</mi><mi>j</mi></msub></mrow></math></span> is the random effect of the <em>j</em>th environment such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-58" display="inline"><mrow><mrow><msub><mi>E</mi><mi>j</mi></msub><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>E</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-59" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>E</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the variance component of the environment effect, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-60" display="inline"><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> is the random error term, such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-61" display="inline"><mrow><mi>e</mi></mrow></math></span>
<em>
<sub>ij</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-62" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-63" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the error variance.</p></section><section id="tpg270105-sec-0090"><h4 class="pmc_sec_title">2.4.2. M2: Environment, line, and genomic main effects (E + L + G)</h4>
<p>This model is an extension of M1 (Equation <a href="#tpg270105-disp-0006" class="usa-link">6</a>). It considers the inclusion of the random genomic effect of the <em>i</em>th genotype <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-64" display="inline"><mrow><msub><mi>g</mi><mi mathvariant="normal">i</mi></msub></mrow></math></span>, which approximates its additive genetic value using molecular marker information. This model component was defined by the regression on <em>p</em> marker covariates,
</p>
<table class="disp-formula p" id="tpg270105-disp-0007"><tr>
<td class="formula"><math id="jats-math-65" display="block"><mrow><mrow><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="badbreak">=</mo><munderover><mo>∑</mo><mrow><mi>m</mi><mo>=</mo><mn>1</mn></mrow><mi>p</mi></munderover><msub><mi>x</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub><msub><mi>b</mi><mi>m</mi></msub></mrow></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-66" display="inline"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mi>m</mi></mrow></msub></mrow></math></span> is the genomic information (marker matrix) of the <em>i</em>th genotype at the <em>m</em>th marker, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-67" display="inline"><mrow><msub><mi>b</mi><mi>m</mi></msub></mrow></math></span> is the corresponding marker effect. Marker effects are considered as iid draws from normal distributions of the form <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-68" display="inline"><mrow><mi>b</mi></mrow></math></span>
<em>
<sub>m</sub>
</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-69" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mover><mo>∼</mo><mi>iid</mi></mover><mspace width="0.28em"></mspace><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>b</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-70" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>b</mi><mn>2</mn></msubsup><mspace width="0.28em"></mspace></mrow></mrow></math></span> as the corresponding variance component, which is homogenous to all markers. The vector of genomic effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-71" display="inline"><mrow><mrow><mi>g</mi><mo>=</mo><mo stretchy="false">{</mo><msub><mi>g</mi><mi>i</mi></msub><mo stretchy="false">}</mo></mrow></mrow></math></span> follows a multivariate normal density with zero mean and variance‐covariance matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-72" display="inline"><mrow><mrow><mrow><mi>cov</mi><mspace width="0.28em"></mspace><mspace width="0.28em"></mspace></mrow><mrow><mo stretchy="false">(</mo><mi>g</mi><mo stretchy="false">)</mo></mrow><mo>=</mo><mi>G</mi><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mspace width="0.28em"></mspace></mrow></mrow></math></span> such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-73" display="inline"><mrow><mrow><mi>g</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>G</mi><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>.</p>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-74" display="inline"><mrow><mrow><mi>G</mi><mspace width="0.28em"></mspace><mi mathvariant="normal">α</mi><mspace width="0.28em"></mspace><mfrac><mrow><mi>X</mi><msup><mi>X</mi><mo>′</mo></msup></mrow><mi>p</mi></mfrac></mrow></mrow></math></span> is the genomic relationship matrix, <em>X</em> is the centered and standardized (by columns) matrix of molecular markers, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-75" display="inline"><mrow><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mo>=</mo><mi>p</mi><msubsup><mi mathvariant="normal">σ</mi><mi>b</mi><mn>2</mn></msubsup></mrow></mrow></math></span> is the genomic variance (additive genetic variance). The resulting model becomes:
</p>
<table class="disp-formula p" id="tpg270105-disp-0008"><tr>
<td class="formula"><math id="jats-math-76" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>L</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(8)</td>
</tr></table>
<p>where the genotype effects of the vector of genomic random effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-77" display="inline"><mrow><mi>g</mi></mrow></math></span> are correlated such that Model 2 allows the borrowing of information across genotypes. A disadvantage of this model is that the genomic estimated breeding values for a genotype tested in different environments are the same regardless of the environment. To allow a specific genomic effect in each environment, G × E was included as shown in the Model M3.</p></section><section id="tpg270105-sec-0100"><h4 class="pmc_sec_title">2.4.3. M3: Environment, line, genomic, and genomic × environment interaction effects (E + L + G + G × E)</h4>
<p>This model is an extension of M2 (Equation <a href="#tpg270105-disp-0008" class="usa-link">8</a>) by adding the interaction between the markers and environments (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-78" display="inline"><mrow><mrow><mi>g</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> via covariance structures as described by Jarquín et al. (<a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>). This model is an extension of the GBLUP model where the interactions between the markers and each environment are indirectly modeled using the following linear predictor:
</p>
<table class="disp-formula p" id="tpg270105-disp-0009"><tr>
<td class="formula"><math id="jats-math-79" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>L</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>g</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(9)</td>
</tr></table>
<p>where the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-80" display="inline"><mrow><mrow><mi>g</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></span> term corresponds to the interaction between the genetic value of the<em> i</em>th genotype and the <em>j</em>th environment. This interaction term is assumed to follow a multivariate normal distribution such that
</p>
<table class="disp-formula p" id="tpg270105-disp-0010"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f5b9/12375858/acb7a6b5acd3/TPG2-18-e70105-e001.jpg" loading="lazy" alt="2.4.3."></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-82" display="inline"><mrow><msub><mi>Z</mi><mi>g</mi></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-83" display="inline"><mrow><msub><mi>Z</mi><mi>E</mi></msub></mrow></math></span> are the incidence matrices that connect genotypes and environments with phenotypes, respectively. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-84" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>g</mi><mi>E</mi></mrow><mn>2</mn></msubsup></mrow></math></span> is the variance component associated with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-85" display="inline"><mrow><mrow><mi>g</mi><mi>E</mi></mrow></mrow></math></span> and “°” represents the Hadamard product (element‐by‐element product) of the two covariance structures representing the genetic information and the the environmental effects.</p></section><section id="tpg270105-sec-0110"><h4 class="pmc_sec_title">2.4.4. M4: Environment, line, genomic, genomic × environment interaction effects, and significant GWAS SNPs as fixed effects (E + L + G + G × E + S)</h4>
<p>GWAS was previously conducted by Okaron et al. (<a href="#tpg270105-bib-0029" class="usa-link" aria-describedby="tpg270105-bib-0029">2024</a>) using the same set of SNPs that were employed in the GP models. The significant SNPs (<em>p</em> &lt; 0.005) that were associated with PY, SW, and SW100 from the BLINK model were selected to improve GP. Wang and Zhang (<a href="#tpg270105-bib-0042" class="usa-link" aria-describedby="tpg270105-bib-0042">2021</a>) demonstrated that the BLINK model offers higher statistical power compared to other GWAS models. Based on this criterion, the number of selected markers as fixed effects for each trait was 111 (PY), 187 (SW), and 96 (SW100).</p>
<p>The M3 model was extended by incorporating the significant markers as fixed effects as follows:
</p>
<table class="disp-formula p" id="tpg270105-disp-0011"><tr>
<td class="formula"><math id="jats-math-86" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>L</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>g</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>S</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(10)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-87" display="inline"><mrow><msub><mi>S</mi><mi>i</mi></msub></mrow></math></span> is the fixed genomic effect associated with the <em>i</em>th genotype.
</p>
<table class="disp-formula p" id="tpg270105-disp-0012"><tr><td class="formula"><math id="jats-math-88" display="block"><mrow><mrow><msub><mi>S</mi><mi>i</mi></msub><mo linebreak="badbreak">=</mo><munderover><mo>∑</mo><mrow><mi>u</mi><mo>=</mo><mn>1</mn></mrow><mi>t</mi></munderover><msub><mi>x</mi><mrow><mi>i</mi><mi>u</mi></mrow></msub><msub><mi>b</mi><mi>u</mi></msub></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-89" display="inline"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mi>u</mi></mrow></msub></mrow></math></span> is the genotype at the <em>u</em>th (<em>u </em>= 1, 2,.., <em>t</em>) selected marker of the <em>i</em>th individual, <em>t</em> is the number of associated markers selected for inclusion as fixed‐effect covariates and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-90" display="inline"><mrow><msub><mi>b</mi><mi>u</mi></msub></mrow></math></span> is the fixed additive effect of the <em>u</em>th significant marker.</p></section></section><section id="tpg270105-sec-0120"><h3 class="pmc_sec_title">2.5. Model assessment under different CV schemes</h3>
<p>To assess the performance of the four GP models, four prediction strategies commonly used by breeders were considered. These prediction scenarios mimic different realistic problems that breeders face at different stages of the breeding pipeline. In all the CVs, predictive ability was calculated as the correlation between observed and predicted values within the same environment. A random five‐fold partitioning of the entire population was used for CV1 and CV2, and predictive ability was calculated as the average correlation between predicted and observed values of genotypes within the same environment. In CV0 and CV00, the “leave one environment out” approach was used to predict future (unknown) environments.</p>
<p>In CV2, a fivefold CV was implemented, where 20% of the phenotypic values were randomly assigned to each fold, one fold was used as a testing set, and the remaining four folds (80%) were used as the training set for model calibration. This process was repeated 10 times for all the five folds, and the correlations between the predicted and observed values within the same environment computed for all environments.</p>
<p>Under CV1, a five‐fold CV approach was also used, where the dataset was randomly divided into five equal subsets, each containing 20% of the genotypes. One fold was used as the validation set, while the remaining four folds (comprising 80% of the genotypes) were used as the training set. Predictions were generated separately for each fold. Predictive ability was assessed as the correlation between predicted and observed values within the same environment. This process was repeated across 10 random fivefold partitions.</p>
<p>CV0 was done by masking the phenotypic information of all genotypes at each environment (one at a time), then using the remaining environments as the training set. In CV00 scheme, both the phenotypic information of the testing environment and the phenotypic information of the genotype to be predicted were deleted from all environments (one at a time).</p>
<p>All the prediction models were analyzed using the BGLR package (Pérez &amp; de los Campos, <a href="#tpg270105-bib-0030" class="usa-link" aria-describedby="tpg270105-bib-0030">2014</a>).</p></section></section><section id="tpg270105-sec-0130"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270105-sec-0140"><h3 class="pmc_sec_title">3.1. Phenotypic data analysis</h3>
<p>Mean phenotypic values and broad‐sense heritability (<em>H</em>
<sup>2</sup>) for PY, SW, and SW100 across the four environments that had been reported in Okaron et al. (<a href="#tpg270105-bib-0029" class="usa-link" aria-describedby="tpg270105-bib-0029">2024</a>) are also summarized in Table <a href="#tpg270105-tbl-0001" class="usa-link">1</a>. The mean phenotypic values for PY, SW, and SW100 ranged from 46.97 to 328.85 g, 22.73 to 218.19 g, and 28.08 to 43.01 g, respectively (Figure <a href="#tpg270105-fig-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="tpg270105-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Mean phenotypic values and heritability of pod yield (PY), seed weight (SW), and 100 seed weight (SW100) in the four environments.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Environment</th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">PY</th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">SW</th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">SW100</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Mean (g)</th>
<th align="left" rowspan="1" colspan="1">
<em>H</em>
<sup>2</sup>
</th>
<th align="left" rowspan="1" colspan="1">Mean (g)</th>
<th align="left" rowspan="1" colspan="1">
<em>H</em>
<sup>2</sup>
</th>
<th align="left" rowspan="1" colspan="1">Mean (g)</th>
<th align="left" rowspan="1" colspan="1">
<em>H</em>
<sup>2</sup>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Chitala</td>
<td align="center" rowspan="1" colspan="1">328.85</td>
<td align="center" rowspan="1" colspan="1">0.63</td>
<td align="center" rowspan="1" colspan="1">218.19</td>
<td align="center" rowspan="1" colspan="1">0.67</td>
<td align="center" rowspan="1" colspan="1">38.69</td>
<td align="center" rowspan="1" colspan="1">0.76</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Chitedze</td>
<td align="center" rowspan="1" colspan="1">117.08</td>
<td align="center" rowspan="1" colspan="1">0.58</td>
<td align="center" rowspan="1" colspan="1">62.53</td>
<td align="center" rowspan="1" colspan="1">0.77</td>
<td align="center" rowspan="1" colspan="1">28.08</td>
<td align="center" rowspan="1" colspan="1">0.56</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Nakabango</td>
<td align="center" rowspan="1" colspan="1">46.97</td>
<td align="center" rowspan="1" colspan="1">0.85</td>
<td align="center" rowspan="1" colspan="1">22.73</td>
<td align="center" rowspan="1" colspan="1">0.73</td>
<td align="center" rowspan="1" colspan="1">40.72</td>
<td align="center" rowspan="1" colspan="1">0.10</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Serere</td>
<td align="center" rowspan="1" colspan="1">99.69</td>
<td align="center" rowspan="1" colspan="1">0.39</td>
<td align="center" rowspan="1" colspan="1">41.83</td>
<td align="center" rowspan="1" colspan="1">0.42</td>
<td align="center" rowspan="1" colspan="1">43.01</td>
<td align="center" rowspan="1" colspan="1">0.28</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Across environments</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1">0.42</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1">0.31</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1">0.44</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270105-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="tpg270105-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f5b9/12375858/962011b65690/TPG2-18-e70105-g004.jpg" loading="lazy" id="jats-graphic-1" height="304" width="400" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270105-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Trait phenotypic variance component estimation and proportion of the total variance explained for the three traits (pod yield [PY], seed weight [SW], and 100 seed weight [SW100]). G × E, genotype‐by‐environment.</p></figcaption></figure><p>The estimated <em>H</em>
<sup>2</sup> across environments indicated influences of genetic and environmental impacts and their interactions across environments. SW showed the highest proportion of variance explained by G × E at 16%, while SW100 had the highest proportion of variance explained by the residual at 52% (Figure <a href="#tpg270105-fig-0001" class="usa-link">1</a>). The corresponding narrow sense h<sup>2</sup> estimates were 0.38, 0.31, and 0.58 for PY, SW, and SW100, respectively.</p></section><section id="tpg270105-sec-0150"><h3 class="pmc_sec_title">3.2. Evaluation of the variance components for each model</h3>
<p>The relative percentage of variability explained by the different model terms is shown in Figure <a href="#tpg270105-fig-0002" class="usa-link">2</a>. Adding G × E effects and significant markers as fixed effects reduced the variability captured by the environment and residual terms (Figure <a href="#tpg270105-fig-0002" class="usa-link">2</a>). As expected, the main effects of the environment consistently explained most of the total variance for all traits. The most complex model, M4, had the lowest variance explained by the environment effect, while M1 had the highest in all the traits. This suggests that M4 may more effectively capture G × E than the other models. Relative to model M1, the inclusion of G × E reduced residual variance by 2.9% for PY, 4.0% for SW, and 4.9% for SW100. A comparable reduction was observed in environmental variance, with decreases of 4.8%, 5.4%, and 4.3% for the respective traits (Figure <a href="#tpg270105-fig-0002" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="tpg270105-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f5b9/12375858/6b3d7fab7b89/TPG2-18-e70105-g005.jpg" loading="lazy" id="jats-graphic-3" height="348" width="560" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270105-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Percentage of explained variance components accounted for pod yield (PY), seed weight (SW), and 100 seed weight (SW100) for the four models (M1, M2, M3, and M4). G × E, genotype‐by‐environment.</p></figcaption></figure></section><section id="tpg270105-sec-0160"><h3 class="pmc_sec_title">3.3. Predictive ability</h3>
<p>The mean predictive abilities of all models across the evaluated traits are presented in Figures <a href="#tpg270105-fig-0003" class="usa-link">3</a>, <a href="#tpg270105-fig-0004" class="usa-link">4</a>, <a href="#tpg270105-fig-0005" class="usa-link">5</a>, <a href="#tpg270105-fig-0006" class="usa-link">6</a>. In general, the predictive ability was higher in CV2 compared to CV1, CV0, and CV00, implying that including more information on observed genotypes and known environments in the training set can enhance model performance. Incorporating the G × E interaction in model M3 increased predictive ability compared to the main effect models, that is, M1 and M2, for all traits under CV1 and CV2 schemes (Figures <a href="#tpg270105-fig-0003" class="usa-link">3</a> &amp; <a href="#tpg270105-fig-0004" class="usa-link">4</a>). The inclusion of significant markers as fixed effects, along with G × E, improved the predictive ability in both CV1 and CV00 scenarios (Figures <a href="#tpg270105-fig-0003" class="usa-link">3</a> &amp; <a href="#tpg270105-fig-0005" class="usa-link">5</a>). This finding suggests that accounting for known associated loci can improve the predictive ability by capturing a greater proportion of the underlying genetic variance.</p>
<figure class="fig xbox font-sm" id="tpg270105-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f5b9/12375858/39ac3f83f200/TPG2-18-e70105-g006.jpg" loading="lazy" id="jats-graphic-5" height="258" width="600" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270105-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predictive abilities under CV1 (where CV is cross‐validation) scheme of the four models for pod yield (PY), seed weight (SW), and 100 seed weight (SW100) at Chitala, Chitedze, Nakabango, and Serere.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270105-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f5b9/12375858/fc95f7a9cfc2/TPG2-18-e70105-g001.jpg" loading="lazy" id="jats-graphic-7" height="258" width="600" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270105-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predictive abilities under CV2 (where CV is cross‐validation) scheme for pod yield (PY), seed weight (SW), and 100 seed weight (SW100) at Chitala, Chitedze, Nakabango, and Serere.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270105-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f5b9/12375858/ce063117b4f6/TPG2-18-e70105-g003.jpg" loading="lazy" id="jats-graphic-9" height="258" width="600" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270105-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predictive abilities under CV00 (where CV is cross‐validation) scheme for pod yield (PY), seed weight (SW), and 100 seed weight (SW100) at Chitala, Chitedze, Nakabango, and Serere.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270105-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f5b9/12375858/dca8283ae575/TPG2-18-e70105-g002.jpg" loading="lazy" id="jats-graphic-11" height="259" width="600" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270105-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predictive abilities under CV0 (where CV is cross‐validation) scheme for pod yield (PY), seed weight (SW), and 100 seed weight (SW100) at Chitala, Chitedze, Nakabango, and Serere.</p></figcaption></figure><section id="tpg270105-sec-0170"><h4 class="pmc_sec_title">3.3.1. Predictive ability under CV1</h4>
<p>All models incorporating marker information exhibited higher predictive ability compared to the phenotype model (M1). Overall, models that incorporated G × E exhibited higher predictive ability than the main‐effect models M1 and M2 (Figure <a href="#tpg270105-fig-0003" class="usa-link">3</a>; Table <a href="#tpg270105-supinfo-0001" class="usa-link">S2</a>). Additionally, incorporating significant markers as fixed effects in the GP model that accounted for G × E further enhanced predictive ability across all traits and environments (Figure <a href="#tpg270105-fig-0003" class="usa-link">3</a>). Model M4 consistently demonstrated the highest predictive ability, whereas M1 showed the lowest across all traits. The range in predictive ability was −0.15 to 0.73, −0.15 to 0.67, and −0.11 to 0.77 for PY, SW, and SW100, respectively (Figure <a href="#tpg270105-fig-0003" class="usa-link">3</a>; Table <a href="#tpg270105-supinfo-0001" class="usa-link">S2</a>).</p></section><section id="tpg270105-sec-0180"><h4 class="pmc_sec_title">3.3.2. Predictive ability under CV2</h4>
<p>All traits exhibited moderate to high predictive abilities. Models that incorporated marker information consistently outperformed the phenotypic Model (M1), for all traits except SW100. Overall, the interaction models M3 and M4 did not offer a significant improvement in predictive ability compared to the main‐effect models M1 and M2 (Figure <a href="#tpg270105-fig-0004" class="usa-link">4</a>). The predictive ability ranged from 0.40 to 0.78, 0.35 to 0.71, and 0.69 to 0.90 for PY, SW, and SW100, respectively (Figure <a href="#tpg270105-fig-0004" class="usa-link">4</a>; Table <a href="#tpg270105-supinfo-0001" class="usa-link">S2</a>).</p></section><section id="tpg270105-sec-0190"><h4 class="pmc_sec_title">3.3.3. Predictive ability under CV00</h4>
<p>Models that utilized significant markers as fixed effects consistently outperformed the phenotype model (M1), highlighting the value of including known genomic information (Figure <a href="#tpg270105-fig-0005" class="usa-link">5</a>). As observed in CV1, models that accounted for G × E and included significant markers as fixed effects outperformed the main‐effect models M1 and M2 across all traits, except SW100. The predictive ability ranged from −0.04 to 0.37, −0.05 to 0.35, and −0.05 to 0.45 for PY, SW, and SW100, respectively (Figure <a href="#tpg270105-fig-0005" class="usa-link">5</a>; Table <a href="#tpg270105-supinfo-0001" class="usa-link">S2</a>).</p></section><section id="tpg270105-sec-0200"><h4 class="pmc_sec_title">3.3.4. Predictive ability under CV0</h4>
<p>Among the models evaluated, the main‐effect model M2 consistently demonstrated superior predictive performance compared to M1, M3, and M4 (Figure <a href="#tpg270105-fig-0006" class="usa-link">6</a>; Table <a href="#tpg270105-supinfo-0001" class="usa-link">S2</a>). Similar to CV2, the interaction models M3 and M4 did not provide additional predictive benefit compared to the main‐effect models M1 and M2 (Figure <a href="#tpg270105-fig-0006" class="usa-link">6</a>; Table <a href="#tpg270105-supinfo-0001" class="usa-link">S2</a>). Predictive ability ranged from 0.22 to 0.43 for PY, 0.13 to 0.37 for SW, and 0.10 to 0.84 for SW100 (Figure <a href="#tpg270105-fig-0006" class="usa-link">6</a>).</p></section></section></section><section id="tpg270105-sec-0210"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>The current study exploited existing phenotypic and genotypic datasets for advanced breeding lines evaluated across four different locations to enhance GP for yield‐related traits in groundnut. The results demonstrated the power of modeling G × E to improve GP in groundnut breeding programs. GP models based on METs are inherently complex due to the heterogeneity of genetic variance and imperfect genetic correlations across environments, non‐genetic noise, and high‐dimensional marker data (Jarquín et al., <a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>). Modeling and exploitation of G × E remains one of the major challenges in the analysis of METs in plant breeding and variety testing (Crossa, los Campos, et al., <a href="#tpg270105-bib-0010" class="usa-link" aria-describedby="tpg270105-bib-0010">2016</a>). The use of EC to model the environment component explicitly has been previously shown to increase prediction accuracies in multiple environments (Jarquín et al., <a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>), and this also applies to our work. The reaction norm model used in the current study allowed modeling the main and interaction effects of markers and EC using covariance structures (Jarquín et al., <a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>). The M3 model was in all cases superior to or not different from the main effect models (M1 and M2). These results are consistent with other studies that incorporated G × E, resulting in higher predictive accuracy compared to univariate models (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>; Crossa, Burgueño, et al., <a href="#tpg270105-bib-0009" class="usa-link" aria-describedby="tpg270105-bib-0009">2006</a>; Cuevas et al., <a href="#tpg270105-bib-0012" class="usa-link" aria-describedby="tpg270105-bib-0012">2016</a>). The improved predictive ability could be due to the decreased amount of variance captured by both the environment and residual terms in all the traits as compared to the traditional GP model (M2) and the phenotypic‐based model M1. In other studies, incorporating G × E was reported to reduce the proportion of variability attributed to the environment and residual terms in soybean (Canella Vieira et al., <a href="#tpg270105-bib-0049" class="usa-link" aria-describedby="tpg270105-bib-0049">2022</a>). Mageto et al. (<a href="#tpg270105-bib-0025" class="usa-link" aria-describedby="tpg270105-bib-0025">2020</a>) observed that including G × E led to approximately 30% reduction in the residual variance component in maize. Lubanga et al. (<a href="#tpg270105-bib-0022" class="usa-link" aria-describedby="tpg270105-bib-0022">2025</a>) demonstrated that models incorporating G × E effects exhibited the lowest mean square error across various sparse testing designs when compared to models that included only main effects in cassava.</p>
<p>The benefit of using molecular marker information was reflected in the current study, where models M2, M3, and M4 revealed a higher predictive ability than M1 (without marker information). Lubanga et al. (<a href="#tpg270105-bib-0024" class="usa-link" aria-describedby="tpg270105-bib-0024">2023</a>) reported higher prediction accuracies for models incorporating markers compared to pedigree and phenotypic‐based models. In GBLUP, molecular markers are used to estimate relatedness between individuals and represent realized genomic relationships rather than the expected relationships (Bernardo, <a href="#tpg270105-bib-0005" class="usa-link" aria-describedby="tpg270105-bib-0005">1994</a>; Bernardo &amp; Yu, <a href="#tpg270105-bib-0006" class="usa-link" aria-describedby="tpg270105-bib-0006">2007</a>). This makes it possible to capture distant relationships and variation in sibling relationships due to Mendelian sampling, leading to more accurate estimates of additive genetic variance and breeding values (Goddard et al., <a href="#tpg270105-bib-0015" class="usa-link" aria-describedby="tpg270105-bib-0015">2011</a>).</p>
<p>CV schemes are useful for assessing the predictive ability of GP models in a multi‐environment context (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>; Jarquín et al., <a href="#tpg270105-bib-0017" class="usa-link" aria-describedby="tpg270105-bib-0017">2017</a>). The four CV schemes used in this study simulated different realistic problems that breeders face, that is, CV2, CV1, CV0, and CV00 (Jarquín et al., <a href="#tpg270105-bib-0017" class="usa-link" aria-describedby="tpg270105-bib-0017">2017</a>; Persa et al., <a href="#tpg270105-bib-0032" class="usa-link" aria-describedby="tpg270105-bib-0032">2021</a>). Across all models, predictive ability was consistently higher under CV2 compared to CV1, CV0, and CV00, due to the additional benefit of incorporating observed genotypes in known environments, even though the combinations of genotype and environments are unknown. However, CV2 adds time to the generation interval due to the additional field testing requirement for all the selection candidates (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>). Although CV1 enables the selection of lines without field testing, it allows for a reduction in the generation interval, and this generally leads to lower predictive ability compared to CV2. Breeding programs therefore need to prioritize accordingly to avoid compromising the annual rate of genetic progress (Burgueño et al., <a href="#tpg270105-bib-0007" class="usa-link" aria-describedby="tpg270105-bib-0007">2012</a>; Jarquín et al., <a href="#tpg270105-bib-0017" class="usa-link" aria-describedby="tpg270105-bib-0017">2017</a>).</p>
<p>Predicting new environments is a more difficult task (Jarquín et al., <a href="#tpg270105-bib-0017" class="usa-link" aria-describedby="tpg270105-bib-0017">2017</a>). This study used CV0 and CV00 schemes for predicting in untested environments. These scenarios are common for most breeding programs that have trials in multiple locations with the need to predict the performance of superior genotypes in future environments. Predictive performance was consistently low under the CV00 scheme, which represents the most challenging CV scenario.</p>
<p>Among the traits, modeling G × E in SW100 did not improve the results of modeling the main genotype and EC matrices alone. It is possible that the EC used explained only a limited proportion of the across environment interaction for this trait, and for this reason the model incorporating G × E, when fitting covariance matrices for the environment and marker by environment interaction, did not improve predictive ability in comparison to the main effect models.</p>
<p>The findings of the current study align with earlier reports indicating that high heritability of the target trait results in improved predictive ability (Lubanga et al., <a href="#tpg270105-bib-0023" class="usa-link" aria-describedby="tpg270105-bib-0023">2021</a>; Zhang et al., <a href="#tpg270105-bib-0046" class="usa-link" aria-describedby="tpg270105-bib-0046">2017</a>). These findings imply that enhancing heritability of the target trait is a significant component in training populations to improve prediction accuracies in the GS pipeline design. Groundnut breeders will need to consider expanding the number of locations and replications, and testing in multiple years to increase heritability. The current strengthening of breeding networks such as the Groundnut Improvement Network for Africa (<a href="http://gnut4africa.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://gnut4africa.org</a>) and the Africa Dryland Crops Improvement Network in Africa can be further leveraged to increase testing locations and make GS practical.</p>
<p>The current results further validate earlier studies that reported improved predictive ability using significant markers from trait association studies as fixed effects in prediction models (Spindel et al., <a href="#tpg270105-bib-0036" class="usa-link" aria-describedby="tpg270105-bib-0036">2016</a>). The implemented model (M4) accounted for G × E by modeling interactions between markers and ECs using covariance functions, and incorporating trait‐associated markers from a previous study as fixed effects. The results revealed that including significant markers as fixed effects, along with modeling G × E, improved predictive ability across all traits in CV1 and CV00. However, no clear benefit was observed in CV2 and CV0, perhaps due to the availability of abundant training data leading to the high predictive ability. These results suggest that the advantage of incorporating significant markers and modeling G × E would be more pronounced when predicting unknown genotypes in new environments. The higher influence of the significant markers incorporated as fixed effects in the model could be due to a lower proportion of the total variance associated with the trait explained by the random portion of the model. We, therefore, strongly recommend the addition of trait‐associated markers where available to improve GP in groundnut breeding programs. These findings are consistent with those of Li et al. (<a href="#tpg270105-bib-0020" class="usa-link" aria-describedby="tpg270105-bib-0020">2019</a>) in maize and Kim et al. (<a href="#tpg270105-bib-0018" class="usa-link" aria-describedby="tpg270105-bib-0018">2022</a>) in <em>Capsicum annuum</em>, both of which reported improved predictive ability when significant markers were included as fixed effects. Similarly, Rice and Lipka (<a href="#tpg270105-bib-0034" class="usa-link" aria-describedby="tpg270105-bib-0034">2019</a>) simulated 216 traits with varying genetic architectures and identified 60 cases in which the inclusion of fixed‐effect markers led to improved prediction accuracy. They recommended that the performance of models incorporating significant markers as fixed effects be evaluated on a trait‐by‐trait basis prior to implementation in breeding programs.</p>
<p>The findings of the present study will require future validation based on some of the limitations. First, the phenotypic data for the three traits were collected over a single season across the four experimental sites located in Uganda and Malawi. Consequently, genotype‐by‐year (G × Y) interaction effects were not assessed, which may have limited the robustness of the dataset. Consideration of G × E in the context of G × Y remains an important challenge for breeders, especially those from small breeding programs (Monteverde et al., <a href="#tpg270105-bib-0027" class="usa-link" aria-describedby="tpg270105-bib-0027">2019</a>). Second, the current study used the reaction norm model proposed by Jarquín et al. (<a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>) to analyze data and predict genomic performance of PY, SW, and SW100. The assumption of the model is that the relationships between molecular markers and EC are linear, which is a major limitation since interactions between genes and environmental conditions may take many different forms. The reaction norm model uses the Gaussian prior that does not induce variable selection, and the shrinkage induced by Gaussian prior density may not be particularly appropriate when markers or ECs have large effects. For instance, a study by Monteverde et al. (<a href="#tpg270105-bib-0027" class="usa-link" aria-describedby="tpg270105-bib-0027">2019</a>) on G × Y in grain yield and quality of rice that integrated molecular markers and EC demonstrated that partial least squares model (Wold et al., <a href="#tpg270105-bib-0043" class="usa-link" aria-describedby="tpg270105-bib-0043">2001</a>) gave better prediction accuracies than reaction norm model (Jarquín et al., <a href="#tpg270105-bib-0016" class="usa-link" aria-describedby="tpg270105-bib-0016">2014</a>).</p>
<p>Finally, this study did not evaluate varying levels of significance thresholds for molecular marker selection. Future work should focus on fitting models with greater biological realism that can accommodate non‐linear and more complex genotype responses across a wide range of environments. Promising directions include (i) the reaction norm model with penalized factorial regression (Avagyan et al., <a href="#tpg270105-bib-0003" class="usa-link" aria-describedby="tpg270105-bib-0003">2025</a>), (ii) integrating crop growth models (CGM) with whole genome prediction (WGP) (CGM–WGP) (Technow et al., <a href="#tpg270105-bib-0038" class="usa-link" aria-describedby="tpg270105-bib-0038">2015</a>), (iii) using models that apply priors that produce variable selection (which cannot be directly implemented under the reaction norm model) (Crossa, los Campos, et al., <a href="#tpg270105-bib-0010" class="usa-link" aria-describedby="tpg270105-bib-0010">2016</a>), and (iv) integrating random regressions on known and latent EC (Tolhurst et al., <a href="#tpg270105-bib-0039" class="usa-link" aria-describedby="tpg270105-bib-0039">2022</a>). Future research should also investigate the effects of different marker threshold levels to determine the optimal balance between model complexity and predictive ability, and to provide guidance on the most effective threshold for marker inclusion.</p></section><section id="tpg270105-sec-0220"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>This study demonstrates the importance of incorporating G × E and significant markers in GP. G × E remains one of the most critical challenges in accurately estimating breeding values for agronomic traits, as it can obscure the true genetic potential of genotypes across diverse environmental conditions. The higher predictive ability observed in CV2 compared to the other scenarios suggests that incorporating more information from known genotypes and environments could enhance the prediction accuracy. The inclusion of significant markers as fixed effects and G × E was more advantageous in CV1 and CV00, demonstrating that this strategy is especially useful when phenotypic data for the target genotypes is limited or unavailable. Collectively, these results are promising, especially when predicting newly developed genotypes in unknown environments.</p></section><section id="tpg270105-sec-0230"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Nelson Lubanga</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; project administration; writing—original draft; writing—review and editing. <strong>Velma Okaron</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; project administration; writing—original draft; writing—review and editing. <strong>Davis M. Gimode</strong>: Resources. <strong>Reyna Persa</strong>: Data curation; formal analysis; methodology. <strong>James Mwololo</strong>: Resources; writing—review and editing. <strong>David K. Okello</strong>: Resources. <strong>Mildred Ochwo Ssemakula</strong>: Resources. <strong>Thomas L. Odong</strong>: Resources. <strong>Wilfred Abincha</strong>: Formal analysis; resources. <strong>Damaris A. Odeny</strong>: Conceptualization; funding acquisition; investigation; methodology; project administration; resources; supervision; writing—review and editing. <strong>Diego Jarquin</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; project administration; resources; software; validation; visualization; writing—review and editing.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270105-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270105-supitem-0001"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375858/bin/TPG2-18-e70105-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70105-s002.docx</a><sup> (30.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270105-supitem-0002"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375858/bin/TPG2-18-e70105-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70105-s001.docx</a><sup> (24.4KB, docx) </sup>
</div></div></section></section><section id="tpg270105-sec-0240" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>The first two authors are thankful to the staff of the International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT) in Malawi and the national staff in Uganda who helped in the data collection. This work was supported by the Food and Agriculture Organization (FAO) through the benefit sharing fund of the International Treaty on Plant Genetics for Food and Agriculture. Supplemental funding was provided by the Scholarship Advert for Short Term Academic Mobility (SCIFSA) and the Regional Universities’ Forum for Capacity Building in Agriculture (RUFORUM) for Velma Okaron Ph.D fellowship. The testing and cross‐validation of GP models was made possible through the analytical resources provided by the Institute of Biological, Environmental and Rural Sciences (IBERS) High‐Performance Computer Cluster.</p></section><section id="notes1"><p>


Lubanga, N.
, 
Okaron, V.
, 
Gimode, D. M.
, 
Persa, R.
, 
Mwololo, J.
, 
Okello, D. K.
, 
Ssemakula, M. O.
, 
Odong, T. L.
, 
Abincha, W.
, 
Odeny, D. A.
, &amp; 
Jarquin, D.
 (2025). Enhancing the prediction accuracy of groundnut yield by integrating significant markers and modeling genotype × environment interaction. The Plant Genome, 18, e70105. 10.1002/tpg2.70105

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270105-note-0001">
<p>
<strong>Present address</strong>
</p>
<p>Davis M. Gimode, CGIAR Standing Panel on Impact Assessment, Alliance of Bioversity International and CIAT, Palmira Campus Km 17 vía Cali–Palmira, Palmira, Colombia.</p>
</div>
<div class="fn p" id="tpg270105-note-0002"><p>Assigned to Associate Editor Jianming Yu.</p></div>
<div class="fn p" id="tpg270105-note-0003"><p>Nelson Lubanga and Velma Okaron are equal first authors.</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Damaris A. Odeny, Email: Damaris.Odeny@icrisat.org.</p>
<p>Diego Jarquin, Email: jhernandezjarqui@ufl.edu.</p></section><section id="tpg270105-sec-0260"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The R scripts and the phenotypic and genomic datasets used to perform the analysis are publicly available on the figshare repository: <a href="https://figshare.com/articles/dataset/Datasets_and_R_scripts_used_to_perform_Genomic_prediction_in_multi-environment_groundnut_traisl/27324150?file=50058195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Datasets_and_R_scripts_used_to_perform_Genomic_prediction_in_multi‐environment_groundnut_traisl/27324150?file=50058195</a>.</p></section><section id="tpg270105-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270105-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270105-bib-0001">
<cite>

Arya, S. S.
, 
Salve, A. R.
, &amp; 
Chauhan, S.
 (2016). Peanuts as functional food: A review. Journal of Food Science and Technology, 53(1), 31–41. 10.1007/s13197-015-2007-9

</cite> [<a href="https://doi.org/10.1007/s13197-015-2007-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4711439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26787930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Science%20and%20Technology&amp;title=Peanuts%20as%20functional%20food:%20A%20review&amp;author=S.%20S.%20Arya&amp;author=A.%20R.%20Salve&amp;author=S.%20Chauhan&amp;volume=53&amp;issue=1&amp;publication_year=2016&amp;pages=31-41&amp;pmid=26787930&amp;doi=10.1007/s13197-015-2007-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0002">
<cite>

Asibuo, J. Y.
, 
Forpoh, A. S.
, &amp; 
Akromah, R.
 (2018). Genotype × environment interactions of groundnut (<em>Arachis hypogaea</em> L.) for pod yield. Ecological Genetics and Genomics, 7–8, 27–32. 10.1016/j.egg.2018.03.001
</cite> [<a href="https://doi.org/10.1016/j.egg.2018.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecological%20Genetics%20and%20Genomics&amp;title=Genotype%20%C3%97%20environment%20interactions%20of%20groundnut%20(Arachis%20hypogaea%20L.)%20for%20pod%20yield&amp;author=J.%20Y.%20Asibuo&amp;author=A.%20S.%20Forpoh&amp;author=R.%20Akromah&amp;volume=7%E2%80%938&amp;publication_year=2018&amp;pages=27-32&amp;doi=10.1016/j.egg.2018.03.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0003">
<cite>

Avagyan, V.
, 
Boer, M. P.
, 
Solin, J.
, 
van Dijk, A. D. J.
, 
Bustos‐Korts, D.
, 
van Rossum, B.‐J.
, 
Ramakers, J. J. C.
, 
van Eeuwijk, F.
, &amp; 
Kruijer, W.
 (2025). Penalized factorial regression as a flexible and computationally attractive reaction norm model for prediction in the presence of GxE. Theoretical and Applied Genetics, 138(4), Article 88. 10.1007/s00122-025-04865-4

</cite> [<a href="https://doi.org/10.1007/s00122-025-04865-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11953130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40155554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Penalized%20factorial%20regression%20as%20a%20flexible%20and%20computationally%20attractive%20reaction%20norm%20model%20for%20prediction%20in%20the%20presence%20of%20GxE&amp;author=V.%20Avagyan&amp;author=M.%20P.%20Boer&amp;author=J.%20Solin&amp;author=A.%20D.%20J.%20van%20Dijk&amp;author=D.%20Bustos%E2%80%90Korts&amp;volume=138&amp;issue=4&amp;publication_year=2025&amp;pages=88&amp;pmid=40155554&amp;doi=10.1007/s00122-025-04865-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0004">
<cite>

Batley, J.
, &amp; 
Edwards, D.
 (2016). The application of genomics and bioinformatics to accelerate crop improvement in a changing climate. Current Opinion in Plant Biology, 30, 78–81. 10.1016/j.pbi.2016.02.002

</cite> [<a href="https://doi.org/10.1016/j.pbi.2016.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26926905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=The%20application%20of%20genomics%20and%20bioinformatics%20to%20accelerate%20crop%20improvement%20in%20a%20changing%20climate&amp;author=J.%20Batley&amp;author=D.%20Edwards&amp;volume=30&amp;publication_year=2016&amp;pages=78-81&amp;pmid=26926905&amp;doi=10.1016/j.pbi.2016.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0005">
<cite>

Bernardo, R.
 (1994). Prediction of maize single‐cross performance using RFLPs and information from related hybrids. Crop Science, 34(1), 20–25. 10.2135/cropsci1994.0011183x003400010003x
</cite> [<a href="https://doi.org/10.2135/cropsci1994.0011183x003400010003x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Prediction%20of%20maize%20single%E2%80%90cross%20performance%20using%20RFLPs%20and%20information%20from%20related%20hybrids&amp;author=R.%20Bernardo&amp;volume=34&amp;issue=1&amp;publication_year=1994&amp;pages=20-25&amp;doi=10.2135/cropsci1994.0011183x003400010003x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0006">
<cite>

Bernardo, R.
, &amp; 
Yu, J.
 (2007). Prospects for genomewide selection for quantitative traits in maize. Crop Science, 47(3), 1082–1090. 10.2135/cropsci2006.11.0690
</cite> [<a href="https://doi.org/10.2135/cropsci2006.11.0690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Prospects%20for%20genomewide%20selection%20for%20quantitative%20traits%20in%20maize&amp;author=R.%20Bernardo&amp;author=J.%20Yu&amp;volume=47&amp;issue=3&amp;publication_year=2007&amp;pages=1082-1090&amp;doi=10.2135/cropsci2006.11.0690&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0007">
<cite>

Burgueño, J.
, 
de los Campos, G.
, 
Weigel, K.
, &amp; 
Crossa, J.
 (2012). Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers. Crop Science, 52(2), 707–719. 10.2135/cropsci2011.06.0299
</cite> [<a href="https://doi.org/10.2135/cropsci2011.06.0299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20prediction%20of%20breeding%20values%20when%20modeling%20genotype%20%C3%97%20environment%20interaction%20using%20pedigree%20and%20dense%20molecular%20markers&amp;author=J.%20Burgue%C3%B1o&amp;author=G.%20de%20los%20Campos&amp;author=K.%20Weigel&amp;author=J.%20Crossa&amp;volume=52&amp;issue=2&amp;publication_year=2012&amp;pages=707-719&amp;doi=10.2135/cropsci2011.06.0299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0008">
<cite>

Butler, D. G.
, 
Cullis, B. R.
, 
Gilmour, A. R.
, 
Gogel, B. G.
, &amp; 
Thompson, R.
 (2023). ASReml‐R reference manual (4.2.). VSN International Ltd.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ASReml%E2%80%90R%20reference%20manual%20(4.2.)&amp;author=D.%20G.%20Butler&amp;author=B.%20R.%20Cullis&amp;author=A.%20R.%20Gilmour&amp;author=B.%20G.%20Gogel&amp;author=R.%20Thompson&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0049">
<cite>

Canella Vieira, C.
, 
Persa, R.
, 
Chen, P.
, &amp; 
Jarquin, D.
 (2022). Incorporation of soil‐derived covariates in progeny testing and line selection to enhance genomic prediction accuracy in soybean breeding. Frontiers in Genetics, 13, 905824. 10.3389/fgene.2022.905824

</cite> [<a href="https://doi.org/10.3389/fgene.2022.905824" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9493273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36159995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Incorporation%20of%20soil%E2%80%90derived%20covariates%20in%20progeny%20testing%20and%20line%20selection%20to%20enhance%20genomic%20prediction%20accuracy%20in%20soybean%20breeding&amp;author=C.%20Canella%20Vieira&amp;author=R.%20Persa&amp;author=P.%20Chen&amp;author=D.%20Jarquin&amp;volume=13&amp;publication_year=2022&amp;pages=905824&amp;pmid=36159995&amp;doi=10.3389/fgene.2022.905824&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0009">
<cite>

Crossa, J.
, 
Burgueño, J.
, 
Cornelius, P. L.
, 
McLaren, G.
, 
Trethowan, R.
, &amp; 
Krishnamachari, A.
 (2006). Modeling genotype × environment interaction using additive genetic covariances of relatives for predicting breeding values of wheat genotypes. Crop Science, 46(4), 1722–1733. 10.2135/cropsci2005.11-0427
</cite> [<a href="https://doi.org/10.2135/cropsci2005.11-0427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Modeling%20genotype%20%C3%97%20environment%20interaction%20using%20additive%20genetic%20covariances%20of%20relatives%20for%20predicting%20breeding%20values%20of%20wheat%20genotypes&amp;author=J.%20Crossa&amp;author=J.%20Burgue%C3%B1o&amp;author=P.%20L.%20Cornelius&amp;author=G.%20McLaren&amp;author=R.%20Trethowan&amp;volume=46&amp;issue=4&amp;publication_year=2006&amp;pages=1722-1733&amp;doi=10.2135/cropsci2005.11-0427&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0010">
<cite>

Crossa, J.
, 
de los Campos, G.
, 
Maccaferri, M.
, 
Tuberosa, R.
, 
Burgueño, J.
, &amp; 
Pérez‐Rodríguez, P.
 (2016). Extending the marker × environment interaction model for genomic‐enabled prediction and genome‐wide association analysis in durum wheat. Crop Science, 56(5), 2193–2209. 10.2135/cropsci2015.04.0260
</cite> [<a href="https://doi.org/10.2135/cropsci2015.04.0260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Extending%20the%20marker%20%C3%97%20environment%20interaction%20model%20for%20genomic%E2%80%90enabled%20prediction%20and%20genome%E2%80%90wide%20association%20analysis%20in%20durum%20wheat&amp;author=J.%20Crossa&amp;author=G.%20de%20los%20Campos&amp;author=M.%20Maccaferri&amp;author=R.%20Tuberosa&amp;author=J.%20Burgue%C3%B1o&amp;volume=56&amp;issue=5&amp;publication_year=2016&amp;pages=2193-2209&amp;doi=10.2135/cropsci2015.04.0260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0048">
<cite>

Crossa, J.
, 
Pérez, P.
, 
Hickey, J.
, 
Burgueño, J.
, 
Ornella, L.
, 
Cerón‐Rojas, J.
, 
Zhang, X.
, 
Dreisigacker, S.
, 
Babu, R.
, 
Li, Y.
, 
Bonnett, D.
, &amp; 
Mathews, K.
 (2014). Genomic prediction in CIMMYT maize and wheat breeding programs. Heredity, 112(1), 48–60. 10.1038/hdy.2013.16

</cite> [<a href="https://doi.org/10.1038/hdy.2013.16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3860161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23572121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genomic%20prediction%20in%20CIMMYT%20maize%20and%20wheat%20breeding%20programs&amp;author=J.%20Crossa&amp;author=P.%20P%C3%A9rez&amp;author=J.%20Hickey&amp;author=J.%20Burgue%C3%B1o&amp;author=L.%20Ornella&amp;volume=112&amp;issue=1&amp;publication_year=2014&amp;pages=48-60&amp;pmid=23572121&amp;doi=10.1038/hdy.2013.16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0011">
<cite>

Crossa, J.
, 
Pérez‐Rodríguez, P.
, 
Cuevas, J.
, 
Montesinos‐López, O.
, 
Jarquín, D.
, 
de los Campos, G.
, 
Burgueño, J.
, 
González‐Camacho, J. M.
, 
Pérez‐Elizalde, S.
, 
Beyene, Y.
, 
Dreisigacker, S.
, 
Singh, R.
, 
Zhang, X.
, 
Gowda, M.
, 
Roorkiwal, M.
, 
Rutkoski, J.
, &amp; 
Varshney, R. K.
 (2017). Genomic selection in plant breeding: methods, models, and perspectives. Trends in Plant Science, 22(11), 961–975. 10.1016/j.tplants.2017.08.011

</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28965742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Genomic%20selection%20in%20plant%20breeding:%20methods,%20models,%20and%20perspectives&amp;author=J.%20Crossa&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=J.%20Cuevas&amp;author=O.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=D.%20Jarqu%C3%ADn&amp;volume=22&amp;issue=11&amp;publication_year=2017&amp;pages=961-975&amp;pmid=28965742&amp;doi=10.1016/j.tplants.2017.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0012">
<cite>

Cuevas, J.
, 
Crossa, J.
, 
Soberanis, V.
, 
Pérez‐Elizalde, S.
, 
Pérez‐Rodríguez, P.
, 
de los Campos, G.
, 
Montesinos‐López, O. A.
, &amp; 
Burgueño, J.
 (2016). Genomic prediction of genotype × environment interaction kernel regression models. The Plant Genome, 9(3), plantgenome2016.03.0024. 10.3835/plantgenome2016.03.0024
</cite> [<a href="https://doi.org/10.3835/plantgenome2016.03.0024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27902799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20prediction%20of%20genotype%20%C3%97%20environment%20interaction%20kernel%20regression%20models&amp;author=J.%20Cuevas&amp;author=J.%20Crossa&amp;author=V.%20Soberanis&amp;author=S.%20P%C3%A9rez%E2%80%90Elizalde&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;volume=9&amp;issue=3&amp;publication_year=2016&amp;pmid=27902799&amp;doi=10.3835/plantgenome2016.03.0024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0013">
<cite>
FAO
. (2022). World food and agriculture—Statistical yearbook 2022. FAO. 10.4060/cc2211en
</cite> [<a href="https://doi.org/10.4060/cc2211en" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=World%20food%20and%20agriculture%E2%80%94Statistical%20yearbook%202022&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0014">
<cite>

Finlay, K.
, &amp; 
Wilkinson, G.
 (1963). The analysis of adaptation in a plant‐breeding programme. Australian Journal of Agricultural Research, 14(6), 742. 10.1071/AR9630742
</cite> [<a href="https://doi.org/10.1071/AR9630742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20Journal%20of%20Agricultural%20Research&amp;title=The%20analysis%20of%20adaptation%20in%20a%20plant%E2%80%90breeding%20programme&amp;author=K.%20Finlay&amp;author=G.%20Wilkinson&amp;volume=14&amp;issue=6&amp;publication_year=1963&amp;pages=742&amp;doi=10.1071/AR9630742&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0015">
<cite>

Goddard, M. E.
, 
Hayes, B. J.
, &amp; 
Meuwissen, T. H. E.
 (2011). Using the genomic relationship matrix to predict the accuracy of genomic selection. Journal of Animal Breeding and Genetics, 128(6), 409–421. 10.1111/j.1439-0388.2011.00964.x

</cite> [<a href="https://doi.org/10.1111/j.1439-0388.2011.00964.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22059574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Animal%20Breeding%20and%20Genetics&amp;title=Using%20the%20genomic%20relationship%20matrix%20to%20predict%20the%20accuracy%20of%20genomic%20selection&amp;author=M.%20E.%20Goddard&amp;author=B.%20J.%20Hayes&amp;author=T.%20H.%20E.%20Meuwissen&amp;volume=128&amp;issue=6&amp;publication_year=2011&amp;pages=409-421&amp;pmid=22059574&amp;doi=10.1111/j.1439-0388.2011.00964.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0047">
<cite>

Janila, P.
, 
Nigam, S. N.
, 
Pandey, M. K.
, 
Nagesh, P.
, &amp; 
Varshney, R. K.
 (2013). Groundnut improvement: use of genetic and genomic tools. Frontiers in Plant Science, 4, 23. 10.3389/fpls.2013.00023

</cite> [<a href="https://doi.org/10.3389/fpls.2013.00023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3580887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23443056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Groundnut%20improvement:%20use%20of%20genetic%20and%20genomic%20tools&amp;author=P.%20Janila&amp;author=S.%20N.%20Nigam&amp;author=M.%20K.%20Pandey&amp;author=P.%20Nagesh&amp;author=R.%20K.%20Varshney&amp;volume=4&amp;publication_year=2013&amp;pages=23&amp;pmid=23443056&amp;doi=10.3389/fpls.2013.00023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0016">
<cite>

Jarquín, D.
, 
Crossa, J.
, 
Lacaze, X.
, 
Du Cheyron, P.
, 
Daucourt, J.
, 
Lorgeou, J.
, 
Piraux, F.
, 
Guerreiro, L.
, 
Pérez, P.
, 
Calus, M.
, 
Burgueño, J.
, &amp; 
de los Campos, G.
 (2014). A reaction norm model for genomic selection using high‐dimensional genomic and environmental data. Theoretical and Applied Genetics, 127(3), 595–607. 10.1007/s00122-013-2243-1

</cite> [<a href="https://doi.org/10.1007/s00122-013-2243-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3931944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24337101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=A%20reaction%20norm%20model%20for%20genomic%20selection%20using%20high%E2%80%90dimensional%20genomic%20and%20environmental%20data&amp;author=D.%20Jarqu%C3%ADn&amp;author=J.%20Crossa&amp;author=X.%20Lacaze&amp;author=P.%20Du%20Cheyron&amp;author=J.%20Daucourt&amp;volume=127&amp;issue=3&amp;publication_year=2014&amp;pages=595-607&amp;pmid=24337101&amp;doi=10.1007/s00122-013-2243-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0017">
<cite>

Jarquín, D.
, 
Lemes da Silva, C.
, 
Gaynor, R. C.
, 
Poland, J.
, 
Fritz, A.
, 
Howard, R.
, 
Battenfield, S.
, &amp; 
Crossa, J.
 (2017). Increasing genomic‐enabled prediction accuracy by modeling genotype × environment interactions in Kansas wheat. The Plant Genome, 10(2), plantgenome2016.12.0130. 10.3835/plantgenome2016.12.0130
</cite> [<a href="https://doi.org/10.3835/plantgenome2016.12.0130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28724062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Increasing%20genomic%E2%80%90enabled%20prediction%20accuracy%20by%20modeling%20genotype%20%C3%97%20environment%20interactions%20in%20Kansas%20wheat&amp;author=D.%20Jarqu%C3%ADn&amp;author=C.%20Lemes%20da%20Silva&amp;author=R.%20C.%20Gaynor&amp;author=J.%20Poland&amp;author=A.%20Fritz&amp;volume=10&amp;issue=2&amp;publication_year=2017&amp;pmid=28724062&amp;doi=10.3835/plantgenome2016.12.0130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0018">
<cite>

Kim, G. W.
, 
Hong, J.‐P.
, 
Lee, H.‐Y.
, 
Kwon, J.‐K.
, 
Kim, D.‐A.
, &amp; 
Kang, B.‐C.
 (2022). Genomic selection with fixed‐effect markers improves the prediction accuracy for capsaicinoid contents in <em>Capsicum annuum</em>
. Horticulture Research, 9, uhac204. 10.1093/hr/uhac204

</cite> [<a href="https://doi.org/10.1093/hr/uhac204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9714256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36467271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Genomic%20selection%20with%20fixed%E2%80%90effect%20markers%20improves%20the%20prediction%20accuracy%20for%20capsaicinoid%20contents%20in%20Capsicum%20annuum&amp;author=G.%20W.%20Kim&amp;author=J.%E2%80%90P.%20Hong&amp;author=H.%E2%80%90Y.%20Lee&amp;author=J.%E2%80%90K.%20Kwon&amp;author=D.%E2%80%90A.%20Kim&amp;volume=9&amp;publication_year=2022&amp;pages=uhac204&amp;pmid=36467271&amp;doi=10.1093/hr/uhac204&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0019">
<cite>

Kurapati, S.
, 
Kommineni, R.
, 
Variath, M. T.
, 
Manohar, S. S.
, 
Vemulapalli, P.
, 
Vemireddy, L. N. R.
, &amp; 
Pasupuleti, J.
 (2021). Localization and gene action studies for kernel iron and zinc concentration in groundnut (<em>Arachis hypogaea</em> L.). Euphytica, 217(7), Article 143. 10.1007/s10681-021-02872-2
</cite> [<a href="https://doi.org/10.1007/s10681-021-02872-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Localization%20and%20gene%20action%20studies%20for%20kernel%20iron%20and%20zinc%20concentration%20in%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=S.%20Kurapati&amp;author=R.%20Kommineni&amp;author=M.%20T.%20Variath&amp;author=S.%20S.%20Manohar&amp;author=P.%20Vemulapalli&amp;volume=217&amp;issue=7&amp;publication_year=2021&amp;pages=143&amp;doi=10.1007/s10681-021-02872-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0020">
<cite>

Li, D.
, 
Xu, Z.
, 
Gu, R.
, 
Wang, P.
, 
Lyle, D.
, 
Xu, J.
, 
Zhang, H.
, &amp; 
Wang, G.
 (2019). Correction: Enhancing genomic selection by fitting large‐effect SNPs as fixed effects and a genotype‐by‐environment effect using a maize BC1F3:4population. PLoS One, 14(12), e0226592. 10.1371/journal.pone.0226592

</cite> [<a href="https://doi.org/10.1371/journal.pone.0226592" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6907745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31830105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Correction:%20Enhancing%20genomic%20selection%20by%20fitting%20large%E2%80%90effect%20SNPs%20as%20fixed%20effects%20and%20a%20genotype%E2%80%90by%E2%80%90environment%20effect%20using%20a%20maize%20BC1F3:4population&amp;author=D.%20Li&amp;author=Z.%20Xu&amp;author=R.%20Gu&amp;author=P.%20Wang&amp;author=D.%20Lyle&amp;volume=14&amp;issue=12&amp;publication_year=2019&amp;pages=e0226592&amp;pmid=31830105&amp;doi=10.1371/journal.pone.0226592&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0021">
<cite>

Lopez‐Cruz, M.
, 
Crossa, J.
, 
Bonnett, D.
, 
Dreisigacker, S.
, 
Poland, J.
, 
Jannink, J.‐L.
, 
Singh, R. P.
, 
Autrique, E.
, &amp; 
de los Campos, G.
 (2015). Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model. G3 Genes|Genomes|Genetics, 5(4), 569–582. 10.1534/g3.114.016097

</cite> [<a href="https://doi.org/10.1534/g3.114.016097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4390573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25660166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Increased%20prediction%20accuracy%20in%20wheat%20breeding%20trials%20using%20a%20marker%20%C3%97%20environment%20interaction%20genomic%20selection%20model&amp;author=M.%20Lopez%E2%80%90Cruz&amp;author=J.%20Crossa&amp;author=D.%20Bonnett&amp;author=S.%20Dreisigacker&amp;author=J.%20Poland&amp;volume=5&amp;issue=4&amp;publication_year=2015&amp;pages=569-582&amp;pmid=25660166&amp;doi=10.1534/g3.114.016097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0022">
<cite>

Lubanga, N.
, 
Ifie, B. E.
, 
Persa, R.
, 
Dieng, I.
, 
Rabbi, I. Y.
, &amp; 
Jarquin, D.
 (2025). Sparse testing designs for optimizing resource allocation in multi‐environment cassava breeding trials. The Plant Genome, 18(1), e20558. 10.1002/tpg2.20558

</cite> [<a href="https://doi.org/10.1002/tpg2.20558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11800058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39912128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Sparse%20testing%20designs%20for%20optimizing%20resource%20allocation%20in%20multi%E2%80%90environment%20cassava%20breeding%20trials&amp;author=N.%20Lubanga&amp;author=B.%20E.%20Ifie&amp;author=R.%20Persa&amp;author=I.%20Dieng&amp;author=I.%20Y.%20Rabbi&amp;volume=18&amp;issue=1&amp;publication_year=2025&amp;pages=e20558&amp;pmid=39912128&amp;doi=10.1002/tpg2.20558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0023">
<cite>

Lubanga, N.
, 
Massawe, F.
, &amp; 
Mayes, S.
 (2021). Genomic and pedigree‐based predictive ability for quality traits in tea (<em>Camellia sinensis</em> (L.) O. Kuntze). Euphytica, 217(3), Article 32. 10.1007/s10681-021-02774-3
</cite> [<a href="https://doi.org/10.1007/s10681-021-02774-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Genomic%20and%20pedigree%E2%80%90based%20predictive%20ability%20for%20quality%20traits%20in%20tea%20(Camellia%20sinensis%20(L.)%20O.%20Kuntze)&amp;author=N.%20Lubanga&amp;author=F.%20Massawe&amp;author=S.%20Mayes&amp;volume=217&amp;issue=3&amp;publication_year=2021&amp;pages=32&amp;doi=10.1007/s10681-021-02774-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0024">
<cite>

Lubanga, N.
, 
Massawe, F.
, 
Mayes, S.
, 
Gorjanc, G.
, &amp; 
Bančič, J.
 (2023). Genomic selection strategies to increase genetic gain in tea breeding programs. The Plant Genome, 16(1), e20282. 10.1002/tpg2.20282

</cite> [<a href="https://doi.org/10.1002/tpg2.20282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36349831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20strategies%20to%20increase%20genetic%20gain%20in%20tea%20breeding%20programs&amp;author=N.%20Lubanga&amp;author=F.%20Massawe&amp;author=S.%20Mayes&amp;author=G.%20Gorjanc&amp;author=J.%20Ban%C4%8Di%C4%8D&amp;volume=16&amp;issue=1&amp;publication_year=2023&amp;pages=e20282&amp;pmid=36349831&amp;doi=10.1002/tpg2.20282&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0025">
<cite>

Mageto, E. K.
, 
Crossa, J.
, 
Pérez‐Rodríguez, P.
, 
Dhliwayo, T.
, 
Palacios‐Rojas, N.
, 
Lee, M.
, 
Guo, R.
, 
San Vicente, F.
, 
Zhang, X.
, &amp; 
Hindu, V.
 (2020). Genomic prediction with genotype by environment interaction analysis for kernel zinc concentration in tropical maize germplasm. G3 Genes|Genomes|Genetics, 10(8), 2629–2639. 10.1534/g3.120.401172

</cite> [<a href="https://doi.org/10.1534/g3.120.401172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7407456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32482728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Genomic%20prediction%20with%20genotype%20by%20environment%20interaction%20analysis%20for%20kernel%20zinc%20concentration%20in%20tropical%20maize%20germplasm&amp;author=E.%20K.%20Mageto&amp;author=J.%20Crossa&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=T.%20Dhliwayo&amp;author=N.%20Palacios%E2%80%90Rojas&amp;volume=10&amp;issue=8&amp;publication_year=2020&amp;pages=2629-2639&amp;pmid=32482728&amp;doi=10.1534/g3.120.401172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0026">
<cite>

Meuwissen, T. H. E.
, 
Hayes, B. J.
, &amp; 
Goddard, M. E.
 (2001). Prediction of total genetic value using genome‐wide dense marker maps. Genetics, 157(4), 1819–1829. 10.1093/genetics/157.4.1819

</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Prediction%20of%20total%20genetic%20value%20using%20genome%E2%80%90wide%20dense%20marker%20maps&amp;author=T.%20H.%20E.%20Meuwissen&amp;author=B.%20J.%20Hayes&amp;author=M.%20E.%20Goddard&amp;volume=157&amp;issue=4&amp;publication_year=2001&amp;pages=1819-1829&amp;pmid=11290733&amp;doi=10.1093/genetics/157.4.1819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0027">
<cite>

Monteverde, E.
, 
Gutierrez, L.
, 
Blanco, P.
, 
Pérez de Vida, F.
, 
Rosas, J. E.
, 
Bonnecarrère, V.
, 
Quero, G.
, &amp; 
McCouch, S.
 (2019). Integrating molecular markers and environmental covariates to interpret genotype by environment interaction in rice (<em>Oryza sativa</em> L.) grown in subtropical areas. G3 Genes|Genomes|Genetics, 9(5), 1519–1531. 10.1534/g3.119.400064

</cite> [<a href="https://doi.org/10.1534/g3.119.400064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6505146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30877079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Integrating%20molecular%20markers%20and%20environmental%20covariates%20to%20interpret%20genotype%20by%20environment%20interaction%20in%20rice%20(Oryza%20sativa%20L.)%20grown%20in%20subtropical%20areas&amp;author=E.%20Monteverde&amp;author=L.%20Gutierrez&amp;author=P.%20Blanco&amp;author=F.%20P%C3%A9rez%20de%20Vida&amp;author=J.%20E.%20Rosas&amp;volume=9&amp;issue=5&amp;publication_year=2019&amp;pages=1519-1531&amp;pmid=30877079&amp;doi=10.1534/g3.119.400064&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0028">
<cite>

Mora‐Escobedo, R.
, 
Hernández‐Luna, P.
, 
Joaquín‐Torres, I. C.
, 
Ortiz‐Moreno, A.
, &amp; 
Robles‐Ramírez, M.
, &amp; 
del, C.
 (2015). Physicochemical properties and fatty acid profile of eight peanut varieties grown in Mexico. CyTA—Journal of Food, 13(2), 300–304. 10.1080/19476337.2014.971345
</cite> [<a href="https://doi.org/10.1080/19476337.2014.971345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CyTA%E2%80%94Journal%20of%20Food&amp;title=Physicochemical%20properties%20and%20fatty%20acid%20profile%20of%20eight%20peanut%20varieties%20grown%20in%20Mexico&amp;author=R.%20Mora%E2%80%90Escobedo&amp;author=P.%20Hern%C3%A1ndez%E2%80%90Luna&amp;author=I.%20C.%20Joaqu%C3%ADn%E2%80%90Torres&amp;author=A.%20Ortiz%E2%80%90Moreno&amp;author=M.%20Robles%E2%80%90Ram%C3%ADrez&amp;volume=13&amp;issue=2&amp;publication_year=2015&amp;pages=300-304&amp;doi=10.1080/19476337.2014.971345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0029">
<cite>

Okaron, V.
, 
Mwololo, J.
, 
Gimode, D. M.
, 
Okello, D. K.
, 
Avosa, M.
, 
Clevenger, J.
, 
Korani, W.
, 
Ssemakula, M. O.
, 
Odong, T. L.
, &amp; 
Odeny, D. A.
 (2024). Using cross‐country datasets for association mapping in <em>Arachis hypogaea</em> L. The Plant Genome, 17(4), e20515. 10.1002/tpg2.20515

</cite> [<a href="https://doi.org/10.1002/tpg2.20515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11628922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39404458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Using%20cross%E2%80%90country%20datasets%20for%20association%20mapping%20in%20Arachis%20hypogaea%20L&amp;author=V.%20Okaron&amp;author=J.%20Mwololo&amp;author=D.%20M.%20Gimode&amp;author=D.%20K.%20Okello&amp;author=M.%20Avosa&amp;volume=17&amp;issue=4&amp;publication_year=2024&amp;pages=e20515&amp;pmid=39404458&amp;doi=10.1002/tpg2.20515&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0030">
<cite>

Pérez, P.
, &amp; 
de los Campos, G.
 (2014). Genome‐wide regression and prediction with the BGLR statistical package. Genetics, 198(2), 483–495. 10.1534/genetics.114.164442

</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%E2%80%90wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package&amp;author=P.%20P%C3%A9rez&amp;author=G.%20de%20los%20Campos&amp;volume=198&amp;issue=2&amp;publication_year=2014&amp;pages=483-495&amp;pmid=25009151&amp;doi=10.1534/genetics.114.164442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0031">
<cite>

Pérez‐Cabal, M. A.
, 
Vazquez, A. I.
, 
Gianola, D.
, 
Rosa, G. J. M.
, &amp; 
Weigel, K. A.
 (2012). Accuracy of genome‐enabled prediction in a dairy cattle population using different cross‐validation layouts. Frontiers in Genetics, 3, Article 27. 10.3389/fgene.2012.00027

</cite> [<a href="https://doi.org/10.3389/fgene.2012.00027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3288819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22403583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Accuracy%20of%20genome%E2%80%90enabled%20prediction%20in%20a%20dairy%20cattle%20population%20using%20different%20cross%E2%80%90validation%20layouts&amp;author=M.%20A.%20P%C3%A9rez%E2%80%90Cabal&amp;author=A.%20I.%20Vazquez&amp;author=D.%20Gianola&amp;author=G.%20J.%20M.%20Rosa&amp;author=K.%20A.%20Weigel&amp;volume=3&amp;publication_year=2012&amp;pages=27&amp;pmid=22403583&amp;doi=10.3389/fgene.2012.00027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0032">
<cite>

Persa, R.
, 
Grondona, M.
, &amp; 
Jarquin, D.
 (2021). Development of a genomic prediction pipeline for maintaining comparable sample sizes in training and testing sets across prediction schemes accounting for the genotype‐by‐environment interaction. Agriculture, 11(10), 932. 10.3390/agriculture11100932
</cite> [<a href="https://doi.org/10.3390/agriculture11100932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agriculture&amp;title=Development%20of%20a%20genomic%20prediction%20pipeline%20for%20maintaining%20comparable%20sample%20sizes%20in%20training%20and%20testing%20sets%20across%20prediction%20schemes%20accounting%20for%20the%20genotype%E2%80%90by%E2%80%90environment%20interaction&amp;author=R.%20Persa&amp;author=M.%20Grondona&amp;author=D.%20Jarquin&amp;volume=11&amp;issue=10&amp;publication_year=2021&amp;pages=932&amp;doi=10.3390/agriculture11100932&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0033">
<cite>

Purcell, S.
, 
Neale, B.
, 
Todd‐Brown, K.
, 
Thomas, L.
, 
Ferreira, M. A. R.
, 
Bender, D.
, 
Maller, J.
, 
Sklar, P.
, 
de Bakker, P. I. W.
, 
Daly, M. J.
, &amp; 
Sham, P. C.
 (2007). PLINK: A tool set for whole‐genome association and population‐based linkage analyses. The American Journal of Human Genetics, 81(3), 559–575. 10.1086/519795

</cite> [<a href="https://doi.org/10.1086/519795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&amp;title=PLINK:%20A%20tool%20set%20for%20whole%E2%80%90genome%20association%20and%20population%E2%80%90based%20linkage%20analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd%E2%80%90Brown&amp;author=L.%20Thomas&amp;author=M.%20A.%20R.%20Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0034">
<cite>

Rice, B.
, &amp; 
Lipka, A. E.
 (2019). Evaluation of RR‐BLUP genomic selection models that incorporate peak genome‐wide association study signals in maize and sorghum. The Plant Genome, 12(1), 180052. 10.3835/plantgenome2018.07.0052
</cite> [<a href="https://doi.org/10.3835/plantgenome2018.07.0052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30951091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Evaluation%20of%20RR%E2%80%90BLUP%20genomic%20selection%20models%20that%20incorporate%20peak%20genome%E2%80%90wide%20association%20study%20signals%20in%20maize%20and%20sorghum&amp;author=B.%20Rice&amp;author=A.%20E.%20Lipka&amp;volume=12&amp;issue=1&amp;publication_year=2019&amp;pages=180052&amp;pmid=30951091&amp;doi=10.3835/plantgenome2018.07.0052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0035">
<cite>

Rutkoski, J. E.
, 
Poland, J. A.
, 
Singh, R. P.
, 
Huerta‐Espino, J.
, 
Bhavani, S.
, 
Barbier, H.
, 
Rouse, M. N.
, 
Jannink, J.
, &amp; 
Sorrells, M. E.
 (2014). Genomic selection for quantitative adult plant stem rust resistance in wheat. The Plant Genome, 7(3), plantgenome2014.02.0006. 10.3835/plantgenome2014.02.0006
</cite> [<a href="https://doi.org/10.3835/plantgenome2014.02.0006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20for%20quantitative%20adult%20plant%20stem%20rust%20resistance%20in%20wheat&amp;author=J.%20E.%20Rutkoski&amp;author=J.%20A.%20Poland&amp;author=R.%20P.%20Singh&amp;author=J.%20Huerta%E2%80%90Espino&amp;author=S.%20Bhavani&amp;volume=7&amp;issue=3&amp;publication_year=2014&amp;doi=10.3835/plantgenome2014.02.0006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0036">
<cite>

Spindel, J. E.
, 
Begum, H.
, 
Akdemir, D.
, 
Collard, B.
, 
Redoña, E.
, 
Jannink, J.‐L.
, &amp; 
McCouch, S.
 (2016). Genome‐wide prediction models that incorporate de novo GWAS are a powerful new tool for tropical rice improvement. Heredity, 116(4), 395–408. 10.1038/hdy.2015.113

</cite> [<a href="https://doi.org/10.1038/hdy.2015.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4806696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26860200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genome%E2%80%90wide%20prediction%20models%20that%20incorporate%20de%20novo%20GWAS%20are%20a%20powerful%20new%20tool%20for%20tropical%20rice%20improvement&amp;author=J.%20E.%20Spindel&amp;author=H.%20Begum&amp;author=D.%20Akdemir&amp;author=B.%20Collard&amp;author=E.%20Redo%C3%B1a&amp;volume=116&amp;issue=4&amp;publication_year=2016&amp;pages=395-408&amp;pmid=26860200&amp;doi=10.1038/hdy.2015.113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0037">
<cite>

Tabe‐Ojong, M. P. J.
, 
Lokossou, J. C.
, 
Gebrekidan, B.
, &amp; 
Affognon, H. D.
 (2023). Adoption of climate‐resilient groundnut varieties increases agricultural production, consumption, and smallholder commercialization in West Africa. Nature Communications, 14(1), 5175. 10.1038/s41467-023-40781-1
</cite> [<a href="https://doi.org/10.1038/s41467-023-40781-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10449883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37620315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Adoption%20of%20climate%E2%80%90resilient%20groundnut%20varieties%20increases%20agricultural%20production,%20consumption,%20and%20smallholder%20commercialization%20in%20West%20Africa&amp;author=M.%20P.%20J.%20Tabe%E2%80%90Ojong&amp;author=J.%20C.%20Lokossou&amp;author=B.%20Gebrekidan&amp;author=H.%20D.%20Affognon&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=5175&amp;pmid=37620315&amp;doi=10.1038/s41467-023-40781-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0038">
<cite>

Technow, F.
, 
Messina, C. D.
, 
Totir, L. R.
, &amp; 
Cooper, M.
 (2015). Integrating crop growth models with whole genome prediction through approximate bayesian computation. PLoS One, 10(6), e0130855. 10.1371/journal.pone.0130855

</cite> [<a href="https://doi.org/10.1371/journal.pone.0130855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4488317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26121133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Integrating%20crop%20growth%20models%20with%20whole%20genome%20prediction%20through%20approximate%20bayesian%20computation&amp;author=F.%20Technow&amp;author=C.%20D.%20Messina&amp;author=L.%20R.%20Totir&amp;author=M.%20Cooper&amp;volume=10&amp;issue=6&amp;publication_year=2015&amp;pages=e0130855&amp;pmid=26121133&amp;doi=10.1371/journal.pone.0130855&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0039">
<cite>

Tolhurst, D. J.
, 
Gaynor, R. C.
, 
Gardunia, B.
, 
Hickey, J. M.
, &amp; 
Gorjanc, G.
 (2022). Genomic selection using random regressions on known and latent environmental covariates. Theoretical and Applied Genetics, 135(10), 3393–3415. 10.1007/s00122-022-04186-w

</cite> [<a href="https://doi.org/10.1007/s00122-022-04186-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9519718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36066596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genomic%20selection%20using%20random%20regressions%20on%20known%20and%20latent%20environmental%20covariates&amp;author=D.%20J.%20Tolhurst&amp;author=R.%20C.%20Gaynor&amp;author=B.%20Gardunia&amp;author=J.%20M.%20Hickey&amp;author=G.%20Gorjanc&amp;volume=135&amp;issue=10&amp;publication_year=2022&amp;pages=3393-3415&amp;pmid=36066596&amp;doi=10.1007/s00122-022-04186-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0040">
<cite>

Toomer, O. T.
 (2018). Nutritional chemistry of the peanut (<em>Arachis hypogaea)</em>
. Critical Reviews in Food Science and Nutrition, 58(17), 3042–3053. 10.1080/10408398.2017.1339015

</cite> [<a href="https://doi.org/10.1080/10408398.2017.1339015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28662347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Critical%20Reviews%20in%20Food%20Science%20and%20Nutrition&amp;title=Nutritional%20chemistry%20of%20the%20peanut%20(Arachis%20hypogaea)&amp;author=O.%20T.%20Toomer&amp;volume=58&amp;issue=17&amp;publication_year=2018&amp;pages=3042-3053&amp;pmid=28662347&amp;doi=10.1080/10408398.2017.1339015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0041"><cite>
USDA‐NASS
. (2021). 
U.S. Department of Agriculture. 2020
. <a href="https://www.nass.usda.gov/Newsroom/archive/2021/index.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nass.usda.gov/Newsroom/archive/2021/index.php</a>
</cite></li>
<li id="tpg270105-bib-0042">
<cite>

Wang, J.
, &amp; 
Zhang, Z.
 (2021). GAPIT Version 3: Boosting power and accuracy for genomic association and prediction. Genomics, Proteomics &amp; Bioinformatics, 19(4), 629–640. 10.1016/j.gpb.2021.08.005
</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9121400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34492338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics,%20Proteomics%20&amp;%20Bioinformatics&amp;title=GAPIT%20Version%203:%20Boosting%20power%20and%20accuracy%20for%20genomic%20association%20and%20prediction&amp;author=J.%20Wang&amp;author=Z.%20Zhang&amp;volume=19&amp;issue=4&amp;publication_year=2021&amp;pages=629-640&amp;pmid=34492338&amp;doi=10.1016/j.gpb.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0043">
<cite>

Wold, S.
, 
Sjöström, M.
, &amp; 
Eriksson, L.
 (2001). PLS‐regression: A basic tool of chemometrics. Chemometrics and Intelligent Laboratory Systems, 58(2), 109–130. 10.1016/S0169-7439(01)00155-1
</cite> [<a href="https://doi.org/10.1016/S0169-7439(01)00155-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemometrics%20and%20Intelligent%20Laboratory%20Systems&amp;title=PLS%E2%80%90regression:%20A%20basic%20tool%20of%20chemometrics&amp;author=S.%20Wold&amp;author=M.%20Sj%C3%B6str%C3%B6m&amp;author=L.%20Eriksson&amp;volume=58&amp;issue=2&amp;publication_year=2001&amp;pages=109-130&amp;doi=10.1016/S0169-7439(01)00155-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0044">
<cite>

Yates, F.
, &amp; 
Cochran, W. G.
 (1938). The analysis of groups of experiments. The Journal of Agricultural Science, 28(4), 556–580. 10.1017/S0021859600050978
</cite> [<a href="https://doi.org/10.1017/S0021859600050978" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Agricultural%20Science&amp;title=The%20analysis%20of%20groups%20of%20experiments&amp;author=F.%20Yates&amp;author=W.%20G.%20Cochran&amp;volume=28&amp;issue=4&amp;publication_year=1938&amp;pages=556-580&amp;doi=10.1017/S0021859600050978&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0045">
<cite>

Yeleliere, E.
, 
Antwi‐Agyei, P.
, &amp; 
Baffour‐Ata, F.
 (2023). Impacts of climate change on the yields of leguminous crops in the Guinea Savanna agroecological zone of Ghana. Regional Sustainability, 4(2), 139–149. 10.1016/j.regsus.2023.04.002
</cite> [<a href="https://doi.org/10.1016/j.regsus.2023.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Regional%20Sustainability&amp;title=Impacts%20of%20climate%20change%20on%20the%20yields%20of%20leguminous%20crops%20in%20the%20Guinea%20Savanna%20agroecological%20zone%20of%20Ghana&amp;author=E.%20Yeleliere&amp;author=P.%20Antwi%E2%80%90Agyei&amp;author=F.%20Baffour%E2%80%90Ata&amp;volume=4&amp;issue=2&amp;publication_year=2023&amp;pages=139-149&amp;doi=10.1016/j.regsus.2023.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270105-bib-0046">
<cite>

Zhang, A.
, 
Wang, H.
, 
Beyene, Y.
, 
Semagn, K.
, 
Liu, Y.
, 
Cao, S.
, 
Cui, Z.
, 
Ruan, Y.
, 
Burgueño, J.
, 
San Vicente, F.
, 
Olsen, M.
, 
Prasanna, B. M.
, 
Crossa, J.
, 
Yu, H.
, &amp; 
Zhang, X.
 (2017). Effect of trait heritability, training population size and marker density on genomic prediction accuracy estimation in 22 bi‐parental tropical maize populations. Frontiers in Plant Science, 8, 1916. 10.3389/fpls.2017.01916

</cite> [<a href="https://doi.org/10.3389/fpls.2017.01916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5683035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29167677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Effect%20of%20trait%20heritability,%20training%20population%20size%20and%20marker%20density%20on%20genomic%20prediction%20accuracy%20estimation%20in%2022%20bi%E2%80%90parental%20tropical%20maize%20populations&amp;author=A.%20Zhang&amp;author=H.%20Wang&amp;author=Y.%20Beyene&amp;author=K.%20Semagn&amp;author=Y.%20Liu&amp;volume=8&amp;publication_year=2017&amp;pages=1916&amp;pmid=29167677&amp;doi=10.3389/fpls.2017.01916&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375858/bin/TPG2-18-e70105-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70105-s002.docx</a><sup> (30.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375858/bin/TPG2-18-e70105-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70105-s001.docx</a><sup> (24.4KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The R scripts and the phenotypic and genomic datasets used to perform the analysis are publicly available on the figshare repository: <a href="https://figshare.com/articles/dataset/Datasets_and_R_scripts_used_to_perform_Genomic_prediction_in_multi-environment_groundnut_traisl/27324150?file=50058195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Datasets_and_R_scripts_used_to_perform_Genomic_prediction_in_multi‐environment_groundnut_traisl/27324150?file=50058195</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70105"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70105.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375858/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375858/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375858%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375858/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375858/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375858/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375858/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375858/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375858/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Qr4ymS5SWBqIJbhoxIOuqMsXszuhAlRpqhUczkIrSZw2UtHRZTBpaZx5fEBrZPCs">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
