
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Plasticity of the gut microbiome of golden snub-nosed monkeys (Rhinopithecus roxellana) in response to seasonal variation in diet - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E483F38AEEFFF30583F3005E51CFA0.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjbiofilms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373803/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Biofilms and Microbiomes">
<meta name="citation_title" content="Plasticity of the gut microbiome of golden snub-nosed monkeys (Rhinopithecus roxellana) in response to seasonal variation in diet">
<meta name="citation_author" content="Mingyi Zhang">
<meta name="citation_author_institution" content="School of Ecology and Nature Conservation, Beijing Forestry University, Beijing, China">
<meta name="citation_author_institution" content="State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author_institution" content="College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Xiaochen Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Hui Yao">
<meta name="citation_author_institution" content="Shennongjia National Park, Muyu, Hubei China">
<meta name="citation_author" content="Ying Shen">
<meta name="citation_author_institution" content="State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author_institution" content="College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Yang Teng">
<meta name="citation_author_institution" content="State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author_institution" content="College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Paul A Garber">
<meta name="citation_author_institution" content="International Centre of Biodiversity and Primate Conservation, Dali University, Dali, Yunnan China">
<meta name="citation_author_institution" content="Department of Anthropology and Program in Ecology, Evolution, and Conservation Biology, University of Illinois, Urbana, IL USA">
<meta name="citation_author" content="Huijuan Pan">
<meta name="citation_author_institution" content="School of Ecology and Nature Conservation, Beijing Forestry University, Beijing, China">
<meta name="citation_author" content="Ming Li">
<meta name="citation_author_institution" content="State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="11">
<meta name="citation_firstpage" content="169">
<meta name="citation_doi" content="10.1038/s41522-025-00806-7">
<meta name="citation_pmid" content="40846855">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373803/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373803/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373803/pdf/41522_2025_Article_806.pdf">
<meta name="description" content="The effects of seasonal fluctuations in food availability on gut microbiome composition, diversity, and function present significant challenges to animals with hard-to-digest diets. Here, we investigate seasonal variation the gut microbiome of wild ...">
<meta name="og:title" content="Plasticity of the gut microbiome of golden snub-nosed monkeys (Rhinopithecus roxellana) in response to seasonal variation in diet">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The effects of seasonal fluctuations in food availability on gut microbiome composition, diversity, and function present significant challenges to animals with hard-to-digest diets. Here, we investigate seasonal variation the gut microbiome of wild ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373803/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373803">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41522-025-00806-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41522_2025_Article_806.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373803%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373803/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373803/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373803/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjbiofilms.jpg" alt="NPJ Biofilms and Microbiomes logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Biofilms and Microbiomes" title="Link to NPJ Biofilms and Microbiomes" shape="default" href="http://www.nature.com/npjbiofilms/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Biofilms Microbiomes</button></div>. 2025 Aug 22;11:169. doi: <a href="https://doi.org/10.1038/s41522-025-00806-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41522-025-00806-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Biofilms%20Microbiomes%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Plasticity of the gut microbiome of golden snub-nosed monkeys (<em>Rhinopithecus roxellana</em>) in response to seasonal variation in diet</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mingyi Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mingyi Zhang</span></h3>
<div class="p">
<sup>1</sup>School of Ecology and Nature Conservation, Beijing Forestry University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="p">
<sup>3</sup>College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingyi Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xiaochen Wang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiaochen Wang</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaochen Wang</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hui Yao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hui Yao</span></h3>
<div class="p">
<sup>4</sup>Shennongjia National Park, Muyu, Hubei China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hui Yao</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ying Shen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ying Shen</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="p">
<sup>3</sup>College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Shen</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Teng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yang Teng</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yang Teng</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="p">
<sup>3</sup>College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Teng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yang Teng</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garber%20PA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Paul A Garber</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Paul A Garber</span></h3>
<div class="p">
<sup>5</sup>International Centre of Biodiversity and Primate Conservation, Dali University, Dali, Yunnan China </div>
<div class="p">
<sup>6</sup>Department of Anthropology and Program in Ecology, Evolution, and Conservation Biology, University of Illinois, Urbana, IL USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garber%20PA%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul A Garber</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Huijuan Pan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Huijuan Pan</span></h3>
<div class="p">
<sup>1</sup>School of Ecology and Nature Conservation, Beijing Forestry University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huijuan Pan</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ming Li</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ming Li</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Ming Li</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Ecology and Nature Conservation, Beijing Forestry University, Beijing, China </div>
<div id="Aff2">
<sup>2</sup>State Key Laboratory of Animal Biodiversity Conservation and Integrated Pest Management, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div id="Aff3">
<sup>3</sup>College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China </div>
<div id="Aff4">
<sup>4</sup>Shennongjia National Park, Muyu, Hubei China </div>
<div id="Aff5">
<sup>5</sup>International Centre of Biodiversity and Primate Conservation, Dali University, Dali, Yunnan China </div>
<div id="Aff6">
<sup>6</sup>Department of Anthropology and Program in Ecology, Evolution, and Conservation Biology, University of Illinois, Urbana, IL USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 10; Accepted 2025 Aug 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373803  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846855/" class="usa-link">40846855</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The effects of seasonal fluctuations in food availability on gut microbiome composition, diversity, and function present significant challenges to animals with hard-to-digest diets. Here, we investigate seasonal variation the gut microbiome of wild golden snub-nosed monkeys (<em>Rhinopithecus roxellana</em>), a foregut fermenting primate, using metagenomics and metatranscriptomics data. We reconstructed 578 metagenome-assembled genomes (MAGs), 76.5% of which did not have exact matches in reference databases, highlighting the novelty of their gut microbiota. The gut microbiome of wild golden snub-nosed monkeys exhibited high diversity and enrichment in plant secondary compound metabolism during summer, while in winter it was enriched with enzymes that function in lichen polysaccharide degradation and Lachnospiraceae, which is important for energy balance. Captive monkeys on a consistent diet showed minimal seasonal variation in gut microbiome composition. Habitat changes also affected golden snub-nosed monkey microbiota community assembly and carbon cycling pathways. These findings underscore the gut microbiome’s plasticity in meeting host dietary needs under varying environmental conditions.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Evolution, Microbiology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">The ability to obtain sufficient nutrient and energy intake for growth, reproduction, and maintaining a healthy immune system is a fundamental challenge for wild animals<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Numerous studies have shown that species have evolved behavioral and physiological strategies, including hibernation, seasonal reproduction, shifts in diet and range use in response to temporal changes in food distribution and availability, and increased mobilization of fat stores to meet changing nutritional and reproductive demands<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In addition, abiotic factors such as seasonal variation in temperature, rainfall, and photoperiod can serve as triggers that affect animal physiology and result in seasonal shifts in metabolic requirements and activity patterns<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par3">In this regard, advances in microbiome research have demonstrated the significant role and plasticity of the host gut microbiome in adjusting to changing social and environmental conditions<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. For instance, a study on wild semi-provisioned Tibetan macaques (<em>Macaca thibetana</em>) found that during the spring, their gut microbiome was enriched in bacteria such as <em>Prevotella</em>, which functioned in carbohydrate and energy metabolism. In the winter, their gut microbiome showed an increase in <em>Succinivibrio</em>, which is reported to facilitate more efficient absorption of difficult-to-digest structural carbohydrates including cellulose and hemicellulose<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Similarly, a study of Ethiopian geladas (<em>Theropithecus gelada</em>), identified seasonal differences in the composition of their gut microbiome that were attributed to the combined effects of changes in food availability and temperature<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. During the rainy season, when grasses dominated their diet, fiber-degrading and fermenting bacteria such as <em>Prevotellaceae</em> and <em>Bacteroidaceae</em> dominated the gelada gut microbiome. In contrast, during the dry season, starch-degrading bacteria were more prominent. Moreover, during cold and dry periods, the gelada gut microbiome experienced an increase in bacteria related to energy, amino acid, and lipid metabolism (i.e. <em>Victivallales</em>, <em>Christensenellaceae</em> and <em>Methanobrevibacter</em>). These microbes communities appeared to play an important role in regulating host body temperature and offsetting the elevated energetic demands of remaining thermoneutral<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. In this regard, a controlled study of laboratory mice exposed to cold stress over a 10-day period (temperatures of 6 °C) found that their microbiome became enriched in bacteria of the genus <em>Firmicutes</em>. When this enriched microbiome was transplanted into germ-free mice, there was evidence of remodeling of fat stores and intestinal tissues associated with an increase in the ability of the mice to withstand periods of high energy demand associated with cold temperatures<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par4">In the present study, we examined seasonal differences in the gut microbiome of the golden snub-nosed monkey (<em>Rhinopithecus roxellana</em>, subfamily Colobinae), including individuals from a wild population inhabiting Shennongjia National Park and a captive population housed at the Beijing Wildlife Park. Golden snub-nosed monkeys are an Endangered species of foregut fermenting primates that inhabit high-altitude mountainous temperate forests (from 1500 to 3400 m) across central China. This region is characterized by seasonal extremes in temperature<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, with nighttime winter temperatures averaging 0 °C and dropping to as low as −14 °C. During the winter, golden snub-nosed monkeys can encounter severe snowstorms, and snow cover can last up to 4 months<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. These conditions make foraging difficult and increase the challenge of maintaining an energy balance and remaining thermoneutral.</p>
<p id="Par5">To improve habituation and ensure individuals have sufficient year-round food, some wild populations are semi-provisioned, receiving limited food supplementation from local management authorities. In the Shennongjia population, golden snub-nosed monkeys primarily feed on young leaves and buds during the summer, which account for more than 60% of their feeding time. In winter, they consume lichen and tree bark, which are typically considered low-nutrient foods, accounting for approximately 40% of feeding time<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> (Supplementary Table <a href="#MOESM2" class="usa-link">1</a>). At a site in the Qinling Mountains, Guo et al.<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> found that wild golden snub-nosed monkeys increased their energy intake by a factor of 1.8 in winter compared to spring by exploiting a high lipid and high carbohydrate diet (e.g., bark from <em>Morus australis</em>, <em>Litsea pungens</em> as lipid sources, and bark from <em>Malus baccata</em>n and <em>Cornus hemsleyi</em> as carbohydrate sources)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, which can offset the increased winter energy requirements of remaining thermoneutral. Similarly, Hou et al. reported that these monkeys lost an average of 14% of their body weight during winter, a loss that was compensated for by increased daily energy intake during summer and fall, with much of this energy being stored as fat<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>.</p>
<p id="Par6">Traditionally, studies investigating the primate gut microbiome have been primarily based on 16S rRNA data. This approach, while generating data on microbiota species composition and diversity, offers limited insight into the potential functionalities of the gut microbiota community. By contrast, metagenomic and metatranscriptomic sequencing offer a more detailed profile of the gut microbiome by including more accurate microbiota identification, classification, and comprehensive genomic information<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Using metagenomic and metatranscriptomic sequencing enabled us to gain greater insight into the metabolic potential, functional features, and contributions of the microbiome to the host’s ability to respond to seasonal shifts in diet and nutrient intake. Additionally, the metagenome-assembled genomes (MAGs) technique has gained popularity for reconstructing bacterial draft genomes through metagenomic binning<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Compared to traditional methods that rely on database annotations, MAGs enable the reconstruction of gut microbiome genomes from rare community members, offering significant advantages in discovering new microbes and expanding our understanding of unknown microbial characteristics and functions<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p>
<p id="Par7">Thus in the current study, we sought to reconstruct MAGs based on fecal metagenomic reads and to investigate gene expression profiles through metatranscriptomics, aiming to explore the gut microbiome at the genome level in both captice and wild golden snub-nosed monkeys. We collected 29 winter and 17 summer fecal samples from a wild population of golden snub-nosed monkeys. For 12 individuals, we collected both winter and summer samples, these are referred to as the same individual group (<em>n</em> = 12). We also collected winter (<em>n</em> = 10) and summer (<em>n</em> = 7) fecal samples from individuals in a captive population of golden snub-nosed monkeys that consume a more consistent diet throughout the year. In total, we reconstructed 578 non-redundant MAGs from the fecal metagenomes of golden snub-nosed monkeys, 76.5% of which did not have exact matches in reference databases. Finally, we annotated 4959 KEGG Orthologs (KOs) from the metatranscriptome and performed paired analyses with the metagenomic data. Our results provide new insight into how changes in the gut microbiome respond to seasonal variation in diet and energy demands for golden snub-nosed monkeys and other wild animals.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Construction of MAGs and annotation of fungi</h3>
<p id="Par8">In total, we obtained 2,213,582,166 raw reads from 63 fecal samples, generating 960 Gb of raw data. After quality filtering and the removal of host sequences, 951 Gb (mean ± SD: 15.09 ± 0.89) of short reads were used for the overall metagenome assemblies. For the metatranscriptome, we obtained 404,818,270 raw reads from 22 samples, resulting in 119.5 Gb (mean ± SD: 5.43 ± 0.42) of data after quality filtering and removal of host sequences (Fig. <a href="#Fig1" class="usa-link">1A</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Analysis pipeline for this study and the phylogenetic tree of nonredundant MAGs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373803_41522_2025_806_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a343/12373803/3b4623ed0bce/41522_2025_806_Fig1_HTML.jpg" loading="lazy" id="d33e434" height="975" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Pipeline for constructing metagenome-assembled genomes (MAGs) and metatranscriptomics gene expression profiles of golden snub-nosed monkeys. <strong>B</strong> Geographic distribution of the 63 fecal samples from the golden snub-nosed monkeys. Metagenomic sequencing data were obtained from 29 fecal samples in the wild winter group and 17 fecal samples in the wild summer group. For 12 wild golden snub-nosed monkeys, samples were collected in both seasons, designating them as the ‘same individual group’. The captive winter group included ten samples, and the captive summer group included seven samples, all collected at Beijing Wildlife Park. <strong>C</strong> Phylogenetic tree of the 578 nonredundant MAGs. Clades are colored according to taxonomic phylum. From the inner to the outer layers: rectangular symbols indicate MAG novelty status: green for novel MAGs and yellow for known MAGs; Stacked bars represent genome completeness (orange); Heatmap shows the N50 values of contigs, with colors ranging from light yellow to dark yellow indicating increasing N50 length. Bar plots show the mean relative abundance of each MAG in the four groups, captive summer group (light blue), wild summer group (red), captive winter group (purple), and wild winter group (dark blue).</p></figcaption></figure><p id="Par9">Microbiome genomes were constructed from the metagenomic sequencing data obtained from the 63 samples described above. Metagenomic sequencing generated 3602 MAGs with a threshold of &gt;50% completeness and contamination of ≤20% (Supplementary Table <a href="#MOESM2" class="usa-link">4</a>). These reconstructed gut microbiome genomes were then compiled and dereplicated at 99% of the average nucleotide identity (ANI), which resulted in a final set of 578 non-redundant MAGs (NR-MAGs) (Fig. <a href="#Fig1" class="usa-link">1C</a>). All of the NR-MAGs met the medium-quality criteria (more than 50% completeness and &lt;5% contamination), with 398 classified as high-quality (more than 90% completeness and &lt;5% contamination)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The 578 NR-MAGs were subsequently classified into taxa using the Genome Taxonomy Database Toolkit (GTDB-Tk), with 574 and 4 assigned as bacteria and archaea, respectively. Of all NR-MAGs, only 136 (23.5%) could be assigned to known species (mean ± SD: 10.44% ± 11.92%), and 442 (76.5%) could not be assigned to the species level (mean ± SD: 56.78% ± 15.47%). Additionally, 14 MAGs (2.4%) could not be assigned to the genus level (mean ± SD: 0.8% ± 0.83%), and 1 MAG could not be assigned to the family level (Fig. <a href="#Fig1" class="usa-link">1C</a>, Supplementary Table <a href="#MOESM2" class="usa-link">5</a>). Over 70% of the genomes could not be assigned to an existing species using the GTDB-tk database, confirming that the majority of the golden snub-nosed monkey gut microbiota lack representation in current reference databases.</p>
<p id="Par10">The all NR-MAGs were distributed across 17 phyla, 22 classes, 44 orders, 78 families, and 284 genera. The most abundant MAGs in the gut microbiome of the golden snub-nosed monkeys were primarily from the phylum Bacteroidota. Among the top ten most abundant MAGs, five belonged to the phylum Bacteroidota (mean ± SD: 13.43% ± 7.83%), and nine were classified as novel MAGs (mean ± SD: 15.81% ± 9.97%). This reflected the novelty and potential for further exploration of the gut microbiome of golden snub-nosed monkeys.</p>
<p id="Par11">A total of 243 fungal species were identified in the gut of golden snub-nosed monkeys. These spanned five phyla, 23 classes, 57 orders, and 104 families, the majority (66 families, 63.5%) of which belonged to the phylum Ascomycota (Supplementary Table <a href="#MOESM2" class="usa-link">6</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Gut microbiome community differences between captive and wild groups</h3>
<p id="Par12">To identify clustering patterns and diversity in the gut microbiome of wild golden snub-nosed monkeys, we performed a PCoA analysis based on Bray-Curtis distances. We found the gut microbiome of wild populations showed distinct separation between different seasons, while the captive population exhibited a more convergent trend across seasons Fig. <a href="#Fig2" class="usa-link">2A</a>. Compared to PCoA, however, enterotype analysis, which does not rely on grouping information, can better reflect the actual existing clustering effect of the gut Compared to PCoA, however, enterotype analysis, which does not rely on grouping information, can better reflect the actual existing clustering effect of the gut microbiome in high-dimensional data space in high-dimensional data space<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Our results showed that all individuals in the captive group, regardless of season, belonged to Enterotype 1 (Fig. <a href="#Fig2" class="usa-link">2B</a>). In contrast, the wild summer group was predominantly classified as Enterotype 2 (76.5%), while the wild winter group was predominantly classified as Enterotype 3 (70%) (Fig. <a href="#Fig2" class="usa-link">2B</a>). These results indicated that enterotypes are associated with environmental conditions and seasonal fluctuation in diet and energy requirements, which is important for the management of captive animals and their reintroduction of individuals into the wild.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Seasonal influences on the gut microbial community characteristics and assembly mechanisms of golden snub-nosed monkeys.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373803_41522_2025_806_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a343/12373803/e4d9831b0e10/41522_2025_806_Fig2_HTML.jpg" loading="lazy" id="d33e531" height="995" width="799" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Principal coordinates analysis (PCoA) revealed four distinct clusters of microbial communities belonging to the four groups. <strong>B</strong> Principal coordinate analysis plots based on the Jensen-Shannon distance between samples, revealing three enterotypes in the gut microbiota. The lines connected to the center of each ellipse correspond to the group affiliation. <strong>C</strong> The box plots show the alpha diversity (Simpson index) of the four groups. ∗:<em>p</em> &lt; 0.05. <strong>D</strong> The Venn diagram shows the number of MAGs shared and uniquely present among the wild summer, wild winter, captive summer, and captive winter groups. <strong>E–H</strong> Co-occurrence network among core species of microbiota of the four golden snub-nosed monkey groups. <strong>E</strong> wild summer group, <strong>F</strong> wild winter group, <strong>G</strong> captive summer group, <strong>H</strong> captive winter group. <strong>I–L</strong> Relative importance of different bacterial community assembly mechanisms of the gut microbiota in four groups. <strong>I</strong> wild summer group, <strong>J</strong> wild winter group, <strong>K</strong> captive summer group, <strong>L</strong> captive winter group.</p></figcaption></figure><p id="Par13">The alpha diversity of the gut microbiome community composition (i.e., taxonomic composition) in wild golden snub-nosed monkeys was significantly higher in summer than in winter (Simpson index, <em>p</em> &lt; 0.05). No such seasonal differences were detected in the captive study group (Fig. <a href="#Fig2" class="usa-link">2C</a>). These results indicated that the effects of seasonal changes on the gut microbial of golden snub-nosed monkeys were more pronounced in wild populations, whereas captive groups were less affected by seasonal variations. A Venn diagram revealed that all four groups (wild winter and summer and captive winter and summer) shared 188 MAGs (32.53%) (Fig. <a href="#Fig2" class="usa-link">2D</a>). The captive group harbored 23 unique MAGs, 12 of which could be assigned to the known species. The wild group contained 24 unique MAGs, all of which lacked species-level annotation. Notably, both winter wild and captive groups exhibited more group-unqiue MAGs than did summer groups (wild winter: 21, captive winter: 13 vs. wild summer: 3, captive summer: 10) (Fig. <a href="#Fig2" class="usa-link">2D</a>). Among the MAGs unique to the wild winter group, most were taxonomically assigned to the orders Oscillospirales and Lachnospirales.</p>
<p id="Par14">Compared to the standard alpha/beta diversity metrics widely used in microbial ecology, co-occurrence networks can provide a powerful approach to exploring microbial ecosystem stability<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Co-occurrence network analyses revealed significant seasonal differences in the complexity of the gut microbiome network structure in the wild golden snub-nosed monkeys. The wild summer group exhibited greater robustness (average clustering coefficient: 0.101, nodes: 375, edges: 680), whereas the wild winter group showed lower community stability compared to the summer group (average clustering coefficient: 0.039, nodes: 419, edges: 242) (Fig. <a href="#Fig2" class="usa-link">2E, F</a>). Additionally, under different environmental conditions, regardless of season, the co-occurrence networks of the wild monkeys were significantly sparser, with fewer edges and nodes, compared to those of the captive monkeys (winter/summer, average clustering coefficient: 0.212/0.308, nodes: 400/375, edges: 4250/5955) (Fig. <a href="#Fig2" class="usa-link">2E–H</a>).</p>
<p id="Par15">Unraveling the drivers controlling gut microbial community assembly is a central issue in microbial ecology<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Specifically, we partitioned 578 non-redundant MAGs into a series of phylogenetic bins (≤12 MAGs per bin) to evaluate the role of different bacterial lineages in community assembly processes. Our results revealed that dispersal limitation (DL) was the predominant assembly process shaping the gut microbial of golden snub-nosed monkeys. DL accounted for the majority of community assembly dynamics across all groups (wild summer: 92.5%, wild winter: 92.6%, captive summer: 79%, captive winter: 82%). In contrast, homogeneous selection (HoS) and heterogeneous selection played relatively minor roles, contributing approximately 2.6–15% and 2–4.9%, respectively (Fig. <a href="#Fig2" class="usa-link">2I–L</a>). In the case of the wild snub-nosed monkeys, although the overall contribution of deterministic processes to gut microbial community assembly remained unchanged from summer to winter, the proportion of heterogeneous selection (HeS summer: 4.9%; winter: 2.7%) decreased, while the proportion of HoS (HoS summer: 2.6%; winter: 4.7%) increased (Fig. <a href="#Fig2" class="usa-link">2I, J</a>). Compared to the captive snub-nosed monkeys, however, the proportion of DL (DL wild: 92.5–92.6%; captive: 79–82%) increased in the wild, while the proportion of HoS (HoS wild: 2.6–4.7%; captive: 15–19%) decreased. The prevailing view is that captive animals live in smaller and more homogeneous environments, which may increase the influence of HoS on their gut microbiota<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. In contrast, wild populations experience greater ecological freedom and environmental variability, which could lead to more pronounced DL<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, as reflected by the more scattered clustering patterns in the PCoA plots (Fig. <a href="#Fig2" class="usa-link">2A</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Seasonal fluctuations of the gut microbiome composition</h3>
<p id="Par16">For captive group, our results did not detect any microbiota taxa with significant seasonal differences (Supplementary Table <a href="#MOESM2" class="usa-link">7</a>), consistent with our findings on enterotype and community structure, which indicated that the gut microbiota of captive golden snub-nosed monkeys exhibited minimal fluctuations across seasons.</p>
<p id="Par17">In the wild monkeys, 24 MAGs exhibited significantly higher abundance in the summer compared to the winter (pFDR &lt; 0.05) (Supplementary Table <a href="#MOESM2" class="usa-link">8</a>) (see <em>Methods 2.8</em>) (Fig. <a href="#Fig3" class="usa-link">3A</a>, Supplementary Table <a href="#MOESM2" class="usa-link">9</a>). This suggested that these 24 MAGs may better reflect the effects of seasonal fluctuations on the host’s gut microbiota, which primarily included Oscillospiraceae (12 MAGs, mean completeness 94.16, mean contamination 1.58), Lachnospiraceae (4 MAGs, mean completeness 91.41, mean contamination 1.58), and Eggerthellaceae (2 MAGs, mean completeness 99.0, mean contamination 0.85) (Fig. <a href="#Fig3" class="usa-link">3A</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Similarly, in the wild winter group, we identified 21 MAGs with significantly higher abundance compared to the wild summer group (pFDR &lt; 0.05) (Supplementary Table <a href="#MOESM2" class="usa-link">8</a>) (see <em>Methods 2.8</em>). Most of these belonged to the taxonomic groups Lachnospiraceae (9 MAGs, mean completeness 93.99, mean contamination 2.62) and Oscillospirales (3 MAGs, mean completeness 96.49, mean contamination 0) <strong>(</strong>Fig. <a href="#Fig3" class="usa-link">3A</a>, Supplementary Figure <a href="#MOESM1" class="usa-link">2</a><strong>)</strong>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Seasonal differences in MAGs of golden snub-nosed monkeys.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373803_41522_2025_806_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a343/12373803/8ff422ffa411/41522_2025_806_Fig3_HTML.jpg" loading="lazy" id="d33e644" height="399" width="799" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The circos barplot reveals enriched MAGs between wild summer (red) and wild winter (blue) groups. The bars indicate abundance. The barplot backgrounds are colored according to the taxonomic phylum. <strong>B</strong> The Sankey diagram reveals that 9 MAGs belong to the family Lachnospiraceae and share several unique genes and pathways. <strong>C</strong> The circos barplot reveals the enriched fungi between wild summer (red) and wild winter (blue) group.</p></figcaption></figure><p id="Par18">In response to winter low temperatures and seasonal variation in food distribution and availability, golden snub-nosed monkeys appear to rely on fat metabolism to maintain energy balance<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Our results suggested that Lachnospiraceae may play a significant role in the energy metabolism of the golden snub-nosed monkeys in the winter. The core genes of Lachnospiraceae (shared among the nine enriched MAGs from the winter) and Oscillospirales (shared among the 12 enrichweds MAGs from the summer) revealed that the enriched MAGs from Lachnospiraceae in winter shared several unique genes, including <a href="https://www.ncbi.nlm.nih.gov/nuccore/K00074" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K00074</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/K00817" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K00817</a>, and K11358 (ko00360; phenylalanine metabolism); <a href="https://www.ncbi.nlm.nih.gov/nuccore/K01610" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K01610</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/K01647" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K01647</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/K01681" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K01681</a> (ko00020; citrate cycle); <a href="https://www.ncbi.nlm.nih.gov/nuccore/K00937" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K00937</a> (oxidative phosphorylation); <a href="https://www.ncbi.nlm.nih.gov/nuccore/K00248" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K00248</a> (ko00071; fatty acid degradation); and <a href="https://www.ncbi.nlm.nih.gov/nuccore/K00626" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K00626</a> (involved in ko04975; fat digestion and absorption, ko00071; fatty acid degradation, and ko00380; tryptophan metabolism) (Fig. <a href="#Fig3" class="usa-link">3B</a>). Our results show that these genes were predominantly associated with pathways, such as map00020 (Citrate cycle (TCA cycle)), map00360 (Phenylalanine metabolism), map00071 (Fatty acid degradation), map00190 (Oxidative phosphorylation), map00380 (Tryptophan metabolism), and map04975 (Fat digestion and absorption) (Fig. <a href="#Fig3" class="usa-link">3B</a>), suggesting that these genes may aid the golden snub-nosed monkeys to more efficiently utilize fat for energy and heat production during the winter.</p>
<p id="Par19">We also analyzed the fungi in the gut microbiota of golden snub-nosed monkeys using the same methods. In accordance with results of the MAGs, no significantly different fungal taxa at the family level were observed in the captive group (Supplementary Table <a href="#MOESM2" class="usa-link">10</a>). In the wild group, we found that Synchytriaceae was significantly more abundant in the summer, while Clavicipitaceae, Parmeliaceae, Enterocytozoonidae, Dacrymycetaceae, and Ramalinaceae were significantly more abundant in the winter (Fig. <a href="#Fig3" class="usa-link">3C</a>, Supplementary Table <a href="#MOESM2" class="usa-link">11</a>, <a href="#MOESM2" class="usa-link">12</a>). Studies have found that Parmeliaceae<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> and Ramalinaceae<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> belong to lichen-forming fungi, thus our results indicated that fungi in the gut microbiota may play an important role in facilitating dietary adaptation as the golden snub-nosed monkeys shift their diet from leaves, buds, and fruits in the summer to lichen, tree bark, and pine seeds in the winter<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The effect of seasonal fluctuation on functional pathways and CAZymes</h3>
<p id="Par20">Based on our analysis of the KEGG database functional annotation, we found significant enrichment in 29 pathways during the winter and in 69 pathways during the summer (Kruskal–Wallis test, <em>p</em> &lt; 0.05) (Supplementary Table <a href="#MOESM2" class="usa-link">13</a>). During the summer, the gut microbiome of wild golden snub-nosed monkeys was enriched in functional pathways related to the digestion and metabolism of potential plant secondary compounds (Fig. <a href="#Fig4" class="usa-link">4A</a>). These pathways include “folate biosynthesis”, “porphyrin and chlorophyll metabolism”, “synthesis and degradation of ketone bodies”, and “terpenoid backbone biosynthesis”. In winter, we found that the “biosynthesis of unsaturated fatty acids”, “longevity regulating pathways”, and “taurine and hypotaurine metabolism” were significantly enriched (Fig. <a href="#Fig4" class="usa-link">4A</a>). These pathways are essential for host immune health, energy storage, and physical development.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Seasonal differences in gut microbiome functional pathways and gene expression activity of golden snub-nosed monkeys.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373803_41522_2025_806_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a343/12373803/2c8c69c9e770/41522_2025_806_Fig4_HTML.jpg" loading="lazy" id="d33e722" height="914" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Clustering heatmap of the abundance of level 3 pathways based on the KEGG database. <strong>B</strong> Up- and down-regulated pathways in the gut metatranscriptome of golden snub-nosed monkeys, and the significantly lower pathway in the winter metagenomic group. <strong>C</strong> Clustering heatmap (left) and histograms (right) of the abundance of CAZy gene families. Yellow asterisk corresponds to summer and is significantly higher than winter; blue asterisk corresponds to winter and is significantly higher than summer ∗:<em>p</em> &lt; 0.05. <strong>D</strong> Up- and down-regulated Metabolic pathways in the gut metatranscriptome of golden snub-nosed monkeys. Each scatterplot illustrates the average gene (DNA) and transcript (RNA) relative abundance for 2883 KOs from all samples. Blue circles correspond to KOs where RNA &gt; DNA; red circles correspond to KOs where DNA &gt; RNA. Marks on the x or y axis margins represent KOs with zero measured abundance in one dataset but non-zero abundances in the other.</p></figcaption></figure><p id="Par21">In the wild snub-nosed monkeys, we identified significant seasonal variation in enzyme levels. In the summer, the levels of GH11, GH77, GT1, GT2, and GT66 were significantly higher than in the winter (<em>p</em> &lt; 0.05). Alternatively, the levels of lichen-degrading enzymes GH16, GH26, GH5, and GH76 in the gut microbiome of wild golden snub-nosed monkeys were significantly higher in winter compared to summer (<em>p</em> &lt; 0.05). This supports field observations that lichen is an important food source for golden snub-nosed monkeys in winter (Fig. <a href="#Fig4" class="usa-link">4C</a>).</p>
<p id="Par22">We also examined the expression of functional genes in the golden snub-nosed monkey gut microbiome. In the winter, metatranscriptomic analysis revealed that gene expression in the gut microbiome of wild golden snub-nosed monkeys was downregulated in metabolism-related genes (Fig. <a href="#Fig4" class="usa-link">4D</a>), including those associated with Lysine biosynthesis, Pyrimidine metabolism, Peptidoglycan biosynthesis, Terpenoid backbone biosynthesis, Folate biosynthesis, Pentose and glucuronate interconversions, the Pentose phosphate pathway and Thiamine metabolism (Fig. <a href="#Fig4" class="usa-link">4B</a>). These pathways were also significantly lower in winter based on our metagenomic analysis, indicating a consistent relationship between gene abundance and gene expression (i.e., significantly downregulated genes in the metagenome also show significantly decreased expression in activity in the metatranscriptome). In addition, <a href="https://www.ncbi.nlm.nih.gov/nuccore/K02794" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K02794</a> [EC:2.7.1.191] and <a href="https://www.ncbi.nlm.nih.gov/nuccore/K02800" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K02800</a> [EC:2.7.1.197] were found to be significantly upregulated in the winter metatranscriptome of wild golden snub-nosed monkeys (Fig. <a href="#Fig4" class="usa-link">4B</a>). These genes primarily participate in encoding enzymes involved in the degradation and digestion of mannitol and mannose, which are major components of lichens<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, this is consistent with our findings in the CAZymes analysis.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Using carbon cycling and metatranscriptomic analyses to compare the methane metabolism of the golden snub-nosed monkey in different seasons</h3>
<p id="Par23">Due to the critical role of carbon in metabolism<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, we constructed a carbon cycling process within the gut microbiome of the golden snub-nosed monkeys (Fig. <a href="#Fig5" class="usa-link">5A</a>). In the wild summer group, we observed that the abundance of Fermentation (pyruvate -&gt; formate <em>pf1D</em> and formate -&gt; CO<sub>2</sub> &amp; H<sub>2</sub>
<em>fdh</em>) and Methanogenesis (acetate -&gt; methane <em>cdhCDE</em>, <em>mcrABG</em>, and methanol -&gt; methane <em>mtaABC</em>) was significantly higher than in winter (<em>p</em> &lt; 0.05) (Fig. <a href="#Fig5" class="usa-link">5B</a>). In winter, the processes of Gluconeogenesis (<em>fbp, pck</em>) and Fermentation to succinate were significantly higher than in summer (<em>p</em> &lt; 0.05) (Fig. <a href="#Fig5" class="usa-link">5B</a>). Furthermore, the metatranscriptomic results from the wild winter group demonstrated significant upregulation of the transcriptional activity of genes related to methane metabolism (Fig. <a href="#Fig5" class="usa-link">5C</a>), primarily involving pathways M00356 and M00567 (Fig. <a href="#Fig5" class="usa-link">5D</a>). It has been suggested that methanogens can drive metabolism forward by removing the gaseous fermentation by-products, carbon dioxide and hydrogen, implying that the gut microbiome contributes to maintaining the metabolic processes of golden snub-nosed monkeys</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Seasonal differences in the gut microbiome carbon cycle pathways and methane metabolism of golden snub-nosed monkeys.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373803_41522_2025_806_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a343/12373803/97e2f92ffd95/41522_2025_806_Fig5_HTML.jpg" loading="lazy" id="d33e830" height="479" width="798" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Relative abundances of the pathways involved in the carbon cycle. The pie chart indicates the relative abundance of each pathway in each metagenomic sample. The size of pie charts represents the total relative abundance of each pathway. CBB, Calvin-Benson-Bassham cycle; rTCA, reductive citric acid cycle; wL, Wood-Ljungdahl pathway; 3HB, 3-hydroxypropionate bicycle; DHc, dicarboxylate-hydroxybutyrate cycle. <strong>B</strong> Bar plot of the pathway involved in the carbon cycle for different seasons. ∗:<em>p</em> &lt; 0.05, ∗∗:<em>p</em> &lt; 0.01. <strong>C</strong> Up- and down-regulated genes in Methane metabolism, with the scatterplot illustrating the average gene (DNA) and transcript (RNA) relative abundance. <strong>D</strong> Reconstruction of the metabolic pathway associated with Methanogenesis in the gut microbiome of golden snub-nosed monkeys. All K numbers and EC numbers were obtained from the KEGG database.</p></figcaption></figure></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">In this study, we constructed a genomic catalog comprising 3602 MAGs of wild and captive golden snub-nosed monkeys. After dereplication, we obtained 578 NR-MAGs, among which 442 MAGs could not be assigned to any known reference genome in the GTDB-Tk database. This finding is consistent with previous large-scale studies reconstructing the gut microbiome genomes of non-human primates and highlights the underexplored microbiome diversity in these hosts<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Collectively, our results revealed substantial “dark matter” in the gut microbiome of non-human primates, which could emphasize the value of primate-focused metagenomic research in significantly broadening our understanding of microbiome diversity and function beyond existing reference databases. Furthermore, the MAG-based approach also enhanced our ability to link functional genes with specific microbiota taxa, moving beyond traditional correlation-based inferences. Although MAG-based approaches have greatly advanced microbiome research, they still have certain limitations<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. For example, low-abundance taxa are often underrepresented, and the fragmentation of genomes, especially those derived from wildlife samples, poses additional challenges. Therefore, we conducted rigorous quality assessments to retain only high- and medium-quality MAGs for taxonomic and functional analyses. This quality control step was crucial for improving genome completeness, reducing contamination, and enhancing the reliability of microbiota classification and gene-function mapping.</p>
<p id="Par25">In their natural habitat, golden snub-nosed monkeys encounter considerable seasonal variation in the distribution and availability of food resources, leading to major shifts in diet<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Our study confirmed that the gut microbiome of wild golden snub-nosed monkeys exhibits plasticity in its response to seasonal fluctuation in diet and ambient temperature in a multidimensional manner, encompassing community structure, microbiota taxa, functional genes, and expression activities. As a highly folivorous foregut fermenting primate inhabiting high-latitude regions, wild golden snub-nosed monkeys face significant seasonal challenges that require them to alter their diet and adjust their energy requirements, especially during periods of extreme cold winter temperatures. During winter, the snub-nosed monkey's diet was dominated by lichen, pine seeds, and bark. During summer, their diet was dominated by leaves, buds, and fruits. In contrast, the diet of captive golden snub-nosed monkeys is similar across seasons (Supplementary Table <a href="#MOESM2" class="usa-link">3</a>), resulting in a more stable and consistent microbiome. Previous research indicated that wild golden snub-nosed monkeys experienced significant seasonal variation in their diet, with abundant and diverse food available in summer, while their diet became more limited and monotonous in winter<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par26">Our study explored differences in host-microbe interactions between wild and captive golden snub-nosed monkeys, revealing the response mechanisms of the gut microbiota to seasonal fluctuations in the host’s dietary diversity and richness. We found that the gut diversity of wild golden snub-nosed monkeys was higher in the summer, which is consistent with previous studies of wild white-faced capuchins (<em>Cebus capucinus imitator</em>)<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> and wild geladas<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Increased microbial diversity may contribute to more efficient nutrient uptake and promote the health of wild primate populations. A study of the gut microbiota of snails found that as the degree of HoS increases, the diversity of the microbial communities decreases<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. These results support a strong interaction mechanism between host diet and host gut microbiome.</p>
<p id="Par27">With the aid of more precise sequencing methods, we found that gut microbiome composition and function exhibited plasticity in response to seasonal changes in food consumption, at both the community level and in specific functional genes and carbohydrate enzymes. From the perspective of functional genes, we found that the microbiome community of wild golden snub-nosed monkeys during the summer aided in plant compound metabolism (Fig. <a href="#Fig4" class="usa-link">4A</a>), enhancing the host’s ability to digest leaves and degrade plant secondary compounds. This plasticity enabled the monkeys to consume large amounts of leaves during the summer, helping them accumulate fat reserves essential for surviving winter conditions<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Similarly, studies on geladas have shown that their gut microbiome can assist in digesting large amounts of carbohydrates from underground plant parts during the dry season<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Moreover, during the summer when female golden snub-nosed monkeys are nursing infants, they require an extremely efficient process for denaturing plant secondary compounds that may be harmless to adults but potentially toxic to their nursing infants<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>.</p>
<p id="Par28">During the winter, we identified an abundance of lichen-forming fungi such as Parmeliaceae<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> and Ramalinaceae<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> in the gut microbiota of the wild golden snub-nosed monkeys (Fig. <a href="#Fig3" class="usa-link">3C</a>). This coincided with an increase in lichen consumption during the winter. These lichen fungi are rich in complex polysaccharides, such as mannopolysaccharides and lichenin, which require an extended period of fermentation for digestion and are generally considered to be of low nutritional quality<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. We found that the gut microbiome of golden snub-nosed monkeys is rich in lichen-degrading enzymes during winter, with GH16 coding for licheninase enzymes, and GH26, GH5, and GH76 coding for mannanase enzymes (Fig. <a href="#Fig4" class="usa-link">4C</a>). In addition, <a href="https://www.ncbi.nlm.nih.gov/nuccore/K02794" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K02794</a> [EC:2.7.1.191] and <a href="https://www.ncbi.nlm.nih.gov/nuccore/K02800" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K02800</a> [EC:2.7.1.197], which are involved in the degradation and digestion of mannitol and mannose, also exhibited high expression in winter (Fig. <a href="#Fig4" class="usa-link">4B</a>). This suggests that the gut microbiota of golden snub-nosed monkeys can help their host digest complex polysaccharides during the winter through lichen-degrading enzymes. This strategy is similar to that employed by Tibetan macaques that rely on their gut microbiome to degrade glycans derived from cellulose and hemicellulose in winter and geladas that efficiently digest starches present in underground plant parts during the dry season<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Moreover, lichens are used in traditional medicine across various cultures worldwide and are believed to possess antiseptic and antibacterial properties<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Thus, it is possible that lichens play a role in the golden snub-nosed monkey’s self-medication. We note that our wild golden snub-monkey population was provisioned with a small amount of food throughout the year. This supplemental feeding, which may have reduced their reliance on natural foods, suggests that wild non-provisioned individuals are likely to exhibit even greater seasonal variation in diet and microbiome composition (see Methods for information on the provisioning protocol of our study animals).</p>
<p id="Par29">In addition to adapting to seasonal variation in the nutrient composition and digestibility of their diet, maintaining energy balance during the winter is also crucial for the golden snub-nosed monkeys. Studies have demonstrated that golden snub-nosed monkeys lose approximately 14% of their body weight (primarily fat) in response to the energy requirements of remaining thermoneutral during the long and cold winter<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. We found that the gut microbiome of wild golden snub-nosed monkeys in the winter was enriched with bacteria of the family Lachnospiraceae, which was positively correlated with leaf consumption in the Skywalker hoolock gibbon (<em>Hoolock tianxing</em>)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. During the rainy season, when grasses dominated their diet, the gelada gut microbiome was enriched in Lachnospiraceae<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. In the present study, based on the Lachnospiraceae species genomes we assembled, we found that the Lachnospiraceae was enriched with functional genes enabling efficient utilization of host fat stores for energy production, including the tricarboxylic acid (TCA) cycle<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, fat digestion and absorption<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, fatty acid degradation<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, oxidative phosphorylation<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, phenylalanine metabolism<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, and tryptophan metabolism<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> (Fig. <a href="#Fig4" class="usa-link">4B</a>). Similar results have been observed in Brandt’s voles (<em>Lasiopodomys brandtii</em>). Huddling voles exposed to cold stress exhibited upregulation of Lachnospiraceae abundance compared to solitary individuals, which resulted in higher concentrations of short-chain fatty acids (SCFAs) to cope with the host’s cold stress<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. In addition, several studies of the mammalian gut microbiome (i.e., brown bears, <em>Ursus arctos</em><sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, Tibetan macaques<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, and geladas<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>) indicate that the microbiome can assist hosts in efficiently maintaining energy balance in response to seasonal challenges in food availability and environmental conditions. In our study, wild golden snub-nosed monkeys were found to exhibit marked seasonal shifts in microbiome composition and function, despite receiving a small amount of provisioned foods.</p>
<p id="Par30">Finally, a recent study of humans found that many taxa of microbiota of the family Lachnospiraceae play important roles in bile acid conversion, short-chain fatty acid production, and antibiotic production<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. In this regard, our results serve to deepen our understanding of the possible functions of the Lachnospiraceae in lipid metabolism, and for next-generation probiotics<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p>
<p id="Par31">In conclusion, we reconstructed a total of 578 MAGs from wild and captive golden snub-nosed monkeys across seasons and environmental conditions (Fig. <a href="#Fig1" class="usa-link">1C</a>). We found significant differences in the gut microbiome of wild golden snub-nosed monkeys that appear to closely correspond with both seasonal shifts in diet and periods of cold stress that occur during the winter in their high elevation habitat. Moreover, a pairwise comparison of metagenomic and metatranscriptomic data revealed that pathways associated with lichen degradation (Fig. <a href="#Fig4" class="usa-link">4B</a>) and methane metabolism (Fig. <a href="#Fig5" class="usa-link">5C</a>) were upregulated during winter, potentially facilitating more efficient food digestion and energy balance needed to remain thermoneutral.</p>
<p id="Par32">Overall, this study presents a comprehensive, multi-omics investigation of the gut microbiome of an Endangered species of primate across distinct seasonal and environmental conditions. By integrating metagenomic and metatranscriptomic analyses, we reveal how microbiome community composition and function exhibit ecological plasticity in response to seasonal shifts in diet. These findings not only enhance our understanding of gut microbiome dynamics but also have broader implications for primate ecology and conservation. An expanding body of research posits that the gut microbiome is an essential contributor to host nutrition, detoxification of plant secondary compounds, enhances energy metabolism, and plays a crucial role in its host’s ability to adapt to changing environmental conditions. Our study demonstrated that, through the use of multiple omic approaches, regular monitoring of the fecal gut microbiome can serve as an important tool in the field of conservation biology to assess wild animal population health and to gain a more detailed understanding of host-microbe interactions under changing environmental conditions. Finally, host-microbe co-evolution represents an important line of research in understanding the ability of both threatened and non-threatened species to adjust to marked shifts in food availability and environmental perturbations associated with climate change and extreme weather events that characterize the Anthropocene.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Sample collection</h3>
<p id="Par33">During 2021 and 2022, we collected fresh fecal samples from the wild population of golden snub-nosed monkeys inhabiting Shennongjia National Park (SNJ), Hubei Province, China. The natural diet of the golden snub-nosed monkeys at this site is supplemented twice per day (totaling ∼40 kg or 0.5 kg per monkey), primarily with sweet potatoes, apples, and peaches. The amount of daily provisioned food was 20 kg of sweet potatoes, 10 kg of apples and 10 kg of peaches in summer, and the food was supplemented with 30 kg of sweet potatoes, 5 kg of apples, and 5 kg peaches in the winter, which can provide approximately 10–20% of their daily energy intake <strong>(</strong>Supplementary Table <a href="#MOESM2" class="usa-link">2</a><strong>)</strong>.</p>
<p id="Par34">We collected 29 fecal samples from the wild population in winter Dec 2020–Jan 2021.1) and 17 fecal samples in summer (June 2021–July 2021). Based on individual identification, we were able to obtain both winter and summer fecal samples for 12 golden snub-nosed monkeys. These samples were designated as the “same individual group” (Fig. <a href="#Fig1" class="usa-link">1B</a>). Also, information on sampling location, time and individual age (adult vs. juvenile) was recorded during fecal collection.</p>
<p id="Par35">In addition, we collected fecal samples (ten samples in winter and seven samples in summer) from captive golden snub-nosed monkeys in Beijing Wildlife Park. For these individuals we integrated dietary records from the wildlife park with nutritional information from the Center for Food Safety (<a href="https://www.cfs.gov.hk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cfs.gov.hk/</a>) to calculate the nutritional composition of their diet (Supplementary Table <a href="#MOESM2" class="usa-link">3</a>). Given changes in the number of golden snub-nosed monkeys available to zoo visitors, we were not able to collect winter and summer fecal samples from the same captive individual. All samples were temporarily preserved and transported to the laboratory in dry ice, then subsequently stored in a −80 °C freezer before nucleic acid extraction. The procedures and protocols of all experiments in this study have been approved by the Institute of Zoology, Chinese Academy of Sciences</p>
<p id="Par36">Based on the sampling season and habitat conditions, the samples were divided into four groups: Wild Summer Group, Wild Winter Group, Captive Summer Group, and Captive Winter Group.</p></section><section id="Sec11"><h3 class="pmc_sec_title">DNA and RNA extraction, and sequencing</h3>
<p id="Par37">Following the manufacturer’s instructions, microbiome genomic DNA was extracted using a cetyltrimethylammonium bromide (CTAB) reagent kit (Macklin, China). To remove RNA contamination, RNase A (omega BIO-TEK, USA) was added and incubated at 37 °C for 15 min. The concentration, purity, and integrity of the extracted DNA were assessed using an Agilent 5400 Fragment Analyzer (Agilent Technologies, USA), and a total of 63 qualified samples were selected for metagenomic sequencing. The NEB Next Ultra DNA Library Prep Kit (NEB, USA) was used to construct a single library, and DNA sequencing was carried out using the 2 × 150 bp paired-end read protocol on the Illumina NovaSeq 6000 platform (Illumina, USA). The amount of raw data obtained from each sample was at least 6 Gb.</p>
<p id="Par38">Compared to metagenomics, metatranscriptomics provides a more accurate interpretation of the functional activity of the gut microbiome. Due to RNA’s susceptibility to degradation, only 22 of the 29 winter fecal samples from the wild monkeys met the quality standards for metatranscriptomic sequencing and were included in this analysis. We used the BioTeke RP8001 Fecal Total RNA Rapid Extraction Kit (BioTeke, China) to extract total RNA from fecal samples. Agarose gel electrophoresis was used to evaluate RNA integrity and to detect potential genomic DNA contamination; a NanoPhotometer spectrophotometer (Implen, Germany) was used to assess RNA purity based on OD260/280 and OD260/230 ratios; a Qubit 2.0 Fluorometer (Life Technologies, USA) was employed for accurate quantification of RNA concentration; and an Agilent 2100 Bioanalyzer (Agilent Technologies, USA) was used to precisely determine RNA integrity. Samples that passed quality inspection were treated with the TIANSeq rRNA Depletion Kit (TIANGEN BIOTECH, China) to remove rRNA and enrich for mRNA. Next, a standard method was used to prepare the RNA library. Specifically, fragmented mRNA was used as the template, and first-strand cDNA synthesis was performed using M-MuLV reverse transcriptase with random hexamer primers. Subsequently, the RNA strand was degraded using RNase H, and the second-strand cDNA was synthesized using DNA polymerase I and dNTPs. The resulting double-stranded cDNA was purified, followed by end repair, A-tailing, and adapter ligation. Approximately 200 bp fragments were selected using AMPure XP beads. The libraries were then amplified by PCR and further purified using AMPure XP beads. Sequencing was performed on the Illumina NovaSeq 6000 platform (Illumina, USA) using a paired-end 150 nt (PE150) high-throughput sequencing strategy, after the constructed libraries had passed the quality control process. Each sample generated at least 5 Gb of raw data.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Metagenome assembly, genome binning, and taxonomy annotation</h3>
<p id="Par39">To ensure the accuracy and reliability of the subsequent analyses, we pre-processed the raw sequencing data. The original data were filtered using Trimmomatic (version 0.39, parameter: ILLUMINACLIP: adapters_path:2:30:10) to remove adapter sequences<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Trimmed sequences with a quality score below 20 (parameter: SLIDINGWINDOW: 4:20) and sequences shorter than 50 bp (parameter: MINLEN:50) were eliminated. This process results in higher-quality and cleaner reads. To remove host contaminants, we used Bowtie 2 (version 2.3.5.1, parameter: --very-sensitive)<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> to map the reads to the <em>Rhinopithecus roxellana</em> genome (Genome assembly: Rrox_v1 GCA_000769185.1).</p>
<p id="Par40">The host-removed clean reads of each sample were individually assembled into contigs using MEGAHIT (version1.1.3)<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Metagenome binning into MAGs was then conducted using MetaBAT2 (version2.12.1) (parameters: minimum contig length to bin -l 1500)<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The dRep (version 3.5.0) was used to dereplicate bins generated from different samples and ensure representatives and diversity of all MAGs<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Completeness and contamination of the final bins were determined using CheckM (parameters: dRep dereplicate -comp 80 -conn 10)<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. MAGs annotation was carried out using GTDB-tk (version 2.3.2) (parameters: gtdbtk classify_wf, --extension fa --skip_ani_screen) and used to construct a species phylogenetic tree<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. The phylogenetic tree of the bins was then visualized using tvBOT<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Using coverM (version 0.7.0) ((<a href="https://github.com/wwood/CoverM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/wwood/CoverM</a>), the non-redundant MAGs were mapped back to clean reads to calculate abundance, with statistical units in count and Reads Per Kilobase Million (RPKM) (parameters: coverm genome -m count, -m rpkm). Gene prediction was performed on the obtained bins using Prokka (version 1.11)<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. KEGG (Kyoto Encyclopedia of Genes and Genomes) database annotation was conducted using DIAMOND BLASTP (version 2.1.8)<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> to obtain functional information at the bin level, with a primary focus on the families Oscillospiraceae and Lachnospiraceae.</p>
<p id="Par41">Given that reconstructing the eukaryotic genome (particularly fungi) from host fecal samples is highly challenging<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>,<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> and considering the crucial role fungi play in the environmental adaptation of golden snub-nosed monkeys, taxonomic annotation methods based on reads offer valuable insights into the classification and abundance of fungi within these samples. Therefore, we utilized Kraken 2 (version 2.0.7-beta) to generate classification tags for the metagenomic DNA sequences of the clean reads<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. After classification, we used Bracken (Bayesian Re-estimation of Abundance after Classification with Kraken) (version 2.0) to estimate the abundance of each sample at various phylogenetic levels, including phylum, class, order, family, genus, and species<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>, and only retained the result related to fungi (k_Fungi). The workflow of data processing is shown in Fig. <a href="#Fig1" class="usa-link">1A</a>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Metagenomic and metatranscriptomic function annotation</h3>
<p id="Par42">The obtained clean reads were compared to the UniRef90 database using DIAMOND (version 0.8.22)<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. We then employed the HUMAnN3 (version 3.6)<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> hierarchical search strategy to quantify the abundance of KEGG Orthology (Kyoto Encyclopedia of Genes and Genomes, KO) and CAZy families annotated from the KEGG and CAZy (Carbohydrate-Active enZYmes) databases, based on the relationship between UniRef90 ID and these databases<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. The functional gene abundance was then converted into relative abundance in units of copies per million (CPM) using the ‘humann_renorm_table’ script provided by HUMAnN3.</p>
<p id="Par43">For metatranscriptomic data, RNA clean reads were obtained using the same quality control filter method as for the metagenome. These reads were then indexed with Bowtie2<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, based on the ChocoPhlan database (v 0.1.1). Using HUMAnN3<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> software we calculated gene families, path abundance, and path coverage. We then mapped the resulting data onto the UniRef90 and KEGG databases<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. We derived the abundance of functional gene expression in the golden snub-nosed monkeys’ gut microbiome using the same methodology for the metagenomic data. The workflow of data processing is shown in Fig. <a href="#Fig1" class="usa-link">1A</a>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Co-occurrence network analysis</h3>
<p id="Par44">Co-occurrence networks for the gut microbiome of golden snub-nosed monkeys were constructed based on MAGs abundance. We used R (version 3.8) to conduct Spearman’s correlation calculations on the MAGs. The statistically robust correlations were used in the analysis of co-occurrence networks under the <em>P</em> value &gt; 0.05 and |correlation coefficient | &gt; 0.7. Gephi (version 0.10.1) software was used to calculate the network diagram parameters and visualize the network<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Microbial Assembly Mechanism Analysis</h3>
<p id="Par45">According to Ning et al. (2020), “infer community assembly mechanisms using the Phylogenetic-bin-based null model analysis” (iCAMP)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> were applied to test the gut microbial community data of the golden snub-nosed monkeys. We aimed to determine the relative importance of stochastic and deterministic processes in microbial community assembly, and to further classify them into different subprocesses. Specifically, a pairwise βNRI value &lt; −1.96 or &gt; 1.96 indicates HoS or heterogeneous selection (HeS), respectively. The fraction of pairwise comparisons with |βNRI | ≤ 1.96 and RC &gt; 0.95 indicates DL, while |βNRI | ≤ 1.96 and RC &lt; −0.95 represents homogenizing dispersal (HD). The remaining |βNRI | ≤ 1.96 and |RC | ≤ 0.95 are used to identify the influence of drift (DR) or undominant processes.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Enterotype classification</h3>
<p id="Par46">Enterotype clustering was conducted at the bin level. Based on the Jensen-Shannon distance between samples, clustering was performed using the Partitioning Around Medoids (PAM) algorithm (R package). The optimal number of clusters was determined using the Calinski-Harabasz (CH) index<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Statistical Analysis</h3>
<p id="Par47">In this study, we used vegan (R package) to compute and assess alpha diversity richness and evenness of microbial communities in each sample. We carried out differential analyses using the K-W dunn.test (R package) to compare across groups. Beta diversity was assessed using principal coordinates analysis (PCoA) based on Bray-Curtis distances to compare the differences in gut microbiota structure between seasons and environmental conditions. Differences in the taxonomic abundance of microbiota, functional genes, functional pathways, and carbohydrate enzyme families across environmental conditions and seasons were analyzed using the Kruskal-Wallis test.</p>
<p id="Par48">To eliminate different seasonal microbiome variations caused by differences in host genetic backgrounds, we employed the following statistical methods: We conducted a Kruskal-Wallis test to identify differentially abundant taxa between all winter samples (<em>N</em> = 29) and summer samples (<em>N</em> = 17), with FDR correction applied to P-values. Effect sizes (η² and Cohen’s f²) and statistical power were also calculated to assess the robustness of the results. Additionally, we performed the same analysis of the samples from the same individuals group (<em>N</em> = 12) across different seasons. We compared the results obtained from the two analyses to identify the shared differential microbiota taxa (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). This provided a level of control not commonly found in wild field studies, allowing us to minimize the variability caused by different individuals represented in the two seasons.</p>
<p id="Par49">For the expression of functional genes and to better approximate normality, we performed an arcsine square root-transformation on the relative abundance values of functional genes obtained from the annotation of metagenomes and metatranscriptomes to variance-stabilize data. Then, we used the <em>t</em> test and fold change to compare differences in gene expression. The resulting data were visualized as scatter plots using R (ggpubr)<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. For all analyses, <em>P</em> value was set at 0.05.</p>
<p id="Par50">The analysis of key genes involved in carbon metabolism is based on the DiTing software<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. Statistical tests of pathways involved in carbon metabolism were performed using Kruskal-Wallis test.</p></section></section><section id="Sec18"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373803/bin/41522_2025_806_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplement Figure-revision.</a><sup> (1,008.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373803/bin/41522_2025_806_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Tables 1-13.</a><sup> (1.1MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The work was supported by National Natural Science Foundation of China (32330015, 32070404 and 31821001). P.A.G wishes to acknowledge Chrissie, Sara, Jenni, Dax, and Saffron for their support during the writing of this manuscript. We also thank all the anonymous reviewers for their helpful comments. Sequencing services were provided by Wekemo Tech Group Co.,Ltd. Shenzhen China.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>H.P. and M.L. designed the study. M.Z. and X.W. analyzed the data and wrote the manuscript. H.Y., Y.S., and Y.T. conducted the sampling and helped with data analysis. P.A.G., H.P., and M.L. improved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data that support the findings of this study are openly available in NGDC at <a href="https://www.cncb.ac.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cncb.ac.cn/</a>, reference number: PRJCA023520. The metagenome-assembled genomes from this study were submitted to Figshare (10.6084/m9.figshare.28520399.v2).</p></section><section id="notes3"><h2 class="pmc_sec_title">Code availability</h2>
<p>The code was provided on GitHub at <a href="https://github.com/ZhangMYioz/Adaptive-Strategies-of-the-golden-snub-nosed-monkeys" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZhangMYioz/Adaptive-Strategies-of-the-golden-snub-nosed-monkeys</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par51">The authors declare no competing interests.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Ethics Declarations</h2>
<p id="Par52">Methods of noninvasive fecal sample collection were applied, keeping the golden snub-nosed monkeys from direct disturbance and contact, and thus no review from the ethics committee was required. All experimental procedures and protocols in this study were approved by the Institute of Zoology, Chinese Academy of Sciences, and the study was authorized by Shennongjia National Park and Beijing Wildlife Park.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Mingyi Zhang, Xiaochen Wang.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Huijuan Pan, Email: phjjanine@bjfu.edu.cn.</p>
<p>Ming Li, Email: lim@ioz.ac.cn.</p></section><section id="sec19"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41522-025-00806-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Herpertz-Dahlmann, B., Seitz, J. &amp; Baines, J. Food matters: how the microbiome and gut–brain interaction might impact the development and course of anorexia nervosa. <em>Eur. Child Adolesc. Psychiatry</em><strong>26</strong>, 1031–1041 (2017).
</cite> [<a href="https://doi.org/10.1007/s00787-017-0945-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5591351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28144744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herpertz-Dahlmann,%20B.,%20Seitz,%20J.%20&amp;%20Baines,%20J.%20Food%20matters:%20how%20the%20microbiome%20and%20gut%E2%80%93brain%20interaction%20might%20impact%20the%20development%20and%20course%20of%20anorexia%20nervosa.%20Eur.%20Child%20Adolesc.%20Psychiatry26,%201031%E2%80%931041%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Lee, A. H. &amp; Dixit, V. D. Dietary regulation of immunity. <em>Immunity</em><strong>53</strong>, 510–523 (2020).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2020.08.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7491384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32937152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20A.%20H.%20&amp;%20Dixit,%20V.%20D.%20Dietary%20regulation%20of%20immunity.%20Immunity53,%20510%E2%80%93523%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Guglielmo, C. G. Obese super athletes: fat-fueled migration in birds and bats. <em>J. Exp. Biol.</em><strong>221</strong>, jeb165753 (2018).
</cite> [<a href="https://doi.org/10.1242/jeb.165753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29514885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guglielmo,%20C.%20G.%20Obese%20super%20athletes:%20fat-fueled%20migration%20in%20birds%20and%20bats.%20J.%20Exp.%20Biol.221,%20jeb165753%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Olsen, L., Thum, E. &amp; Rohner, N. Lipid metabolism in adaptation to extreme nutritional challenges. <em>Dev. Cell</em><strong>56</strong>, 1417–1429 (2021).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2021.02.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33730548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olsen,%20L.,%20Thum,%20E.%20&amp;%20Rohner,%20N.%20Lipid%20metabolism%20in%20adaptation%20to%20extreme%20nutritional%20challenges.%20Dev.%20Cell56,%201417%E2%80%931429%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Van Schaik, C. P. &amp; Brockman, D. K. Seasonality in primate ecology, reproduction, and life history: an overview. <em>Camb. Stud. Biol. Evol. Anthropol.</em><strong>44</strong>, 3 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Van%20Schaik,%20C.%20P.%20&amp;%20Brockman,%20D.%20K.%20Seasonality%20in%20primate%20ecology,%20reproduction,%20and%20life%20history:%20an%20overview.%20Camb.%20Stud.%20Biol.%20Evol.%20Anthropol.44,%203%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wells, C. P. et al. Life history consequences of climate change in hibernating mammals: a review. <em>Ecography</em><strong>2022</strong>, e06056 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wells,%20C.%20P.%20et%20al.%20Life%20history%20consequences%20of%20climate%20change%20in%20hibernating%20mammals:%20a%20review.%20Ecography2022,%20e06056%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Williams, C. T. et al. Seasonal reproductive tactics: annual timing and the capital-to-income breeder continuum. <em>Philos. Trans. R. Soc. B Biol. Sci.</em><strong>372</strong>, 20160250 (2017).</cite> [<a href="https://doi.org/10.1098/rstb.2016.0250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5647277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28993494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williams,%20C.%20T.%20et%20al.%20Seasonal%20reproductive%20tactics:%20annual%20timing%20and%20the%20capital-to-income%20breeder%20continuum.%20Philos.%20Trans.%20R.%20Soc.%20B%20Biol.%20Sci.372,%2020160250%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Bronson, F. H. Climate change and seasonal reproduction in mammals. <em>Philos. Trans. R Soc. Lond B Biol. Sci.</em><strong>364</strong>, 3331–3340 (2009).
</cite> [<a href="https://doi.org/10.1098/rstb.2009.0140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2781850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19833645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bronson,%20F.%20H.%20Climate%20change%20and%20seasonal%20reproduction%20in%20mammals.%20Philos.%20Trans.%20R%20Soc.%20Lond%20B%20Biol.%20Sci.364,%203331%E2%80%933340%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Amato, K. R. et al. The gut microbiota appears to compensate for seasonal diet variation in the wild black howler monkey (<em>Alouatta pigr</em>a). <em>Microb. Ecol.</em><strong>69</strong>, 434–443 (2015).
</cite> [<a href="https://doi.org/10.1007/s00248-014-0554-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25524570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amato,%20K.%20R.%20et%20al.%20The%20gut%20microbiota%20appears%20to%20compensate%20for%20seasonal%20diet%20variation%20in%20the%20wild%20black%20howler%20monkey%20(Alouatta%20pigra).%20Microb.%20Ecol.69,%20434%E2%80%93443%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Huang, G. Seasonal shift of the gut microbiome synchronizes host peripheral circadian rhythm for physiological adaptation to a low-fat diet in the giant panda. <em>Cell Rep</em><strong>38</strong>, 110203 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.110203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35045306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20G.%20Seasonal%20shift%20of%20the%20gut%20microbiome%20synchronizes%20host%20peripheral%20circadian%20rhythm%20for%20physiological%20adaptation%20to%20a%20low-fat%20diet%20in%20the%20giant%20panda.%20Cell%20Rep38,%20110203%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Sommer, F. et al. The gut microbiota modulates energy metabolism in the hibernating brown bear Ursus arctos. <em>Cell Rep.</em><strong>14</strong>, 1655–1661 (2016).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.01.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26854221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sommer,%20F.%20et%20al.%20The%20gut%20microbiota%20modulates%20energy%20metabolism%20in%20the%20hibernating%20brown%20bear%20Ursus%20arctos.%20Cell%20Rep.14,%201655%E2%80%931661%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Tang, K. et al. Metagenomics reveals seasonal functional adaptation of the gut microbiome to host feeding and fasting in the Chinese alligator. <em>Front. Microbiol.</em><strong>10</strong>, 2409 (2019).
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6824212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31708889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20K.%20et%20al.%20Metagenomics%20reveals%20seasonal%20functional%20adaptation%20of%20the%20gut%20microbiome%20to%20host%20feeding%20and%20fasting%20in%20the%20Chinese%20alligator.%20Front.%20Microbiol.10,%202409%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Wei, X. et al. Seasonal diets supersede host species in shaping the distal gut microbiota of Yaks and Tibetan sheep. <em>Sci. Rep.</em><strong>11</strong>, 22626 (2021).
</cite> [<a href="https://doi.org/10.1038/s41598-021-99351-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8604981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34799677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20X.%20et%20al.%20Seasonal%20diets%20supersede%20host%20species%20in%20shaping%20the%20distal%20gut%20microbiota%20of%20Yaks%20and%20Tibetan%20sheep.%20Sci.%20Rep.11,%2022626%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Yan, M. &amp; Yu, Z. Viruses contribute to microbial diversification in the rumen ecosystem and are associated with certain animal production traits. <em>Microbiome</em><strong>12</strong>, 82 (2024).
</cite> [<a href="https://doi.org/10.1186/s40168-024-01791-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11080232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38725064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20M.%20&amp;%20Yu,%20Z.%20Viruses%20contribute%20to%20microbial%20diversification%20in%20the%20rumen%20ecosystem%20and%20are%20associated%20with%20certain%20animal%20production%20traits.%20Microbiome12,%2082%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Yan, M. et al. Interrogating the viral dark matter of the rumen ecosystem with a global virome database. <em>Nat. Commun.</em><strong>14</strong>, 5254 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-41075-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10465536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37644066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20M.%20et%20al.%20Interrogating%20the%20viral%20dark%20matter%20of%20the%20rumen%20ecosystem%20with%20a%20global%20virome%20database.%20Nat.%20Commun.14,%205254%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Sun, B. et al. Marked variation between winter and spring gut microbiota in free-ranging Tibetan Macaques (Macaca thibetana). <em>Sci. Rep.</em><strong>6</strong>, 26035 (2016).
</cite> [<a href="https://doi.org/10.1038/srep26035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4867428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27180722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20B.%20et%20al.%20Marked%20variation%20between%20winter%20and%20spring%20gut%20microbiota%20in%20free-ranging%20Tibetan%20Macaques%20(Macaca%20thibetana).%20Sci.%20Rep.6,%2026035%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Baniel, A. et al. Seasonal shifts in the gut microbiome indicate plastic responses to diet in wild geladas. <em>Microbiome</em><strong>9</strong>, 26 (2021).
</cite> [<a href="https://doi.org/10.1186/s40168-020-00977-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7828014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33485388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baniel,%20A.%20et%20al.%20Seasonal%20shifts%20in%20the%20gut%20microbiome%20indicate%20plastic%20responses%20to%20diet%20in%20wild%20geladas.%20Microbiome9,%2026%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Chevalier, C. et al. Gut microbiota orchestrates energy homeostasis during cold. <em>Cell</em><strong>163</strong>, 1360–1374 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26638070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chevalier,%20C.%20et%20al.%20Gut%20microbiota%20orchestrates%20energy%20homeostasis%20during%20cold.%20Cell163,%201360%E2%80%931374%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ren, B. et al. Inter-population variation of diets of golden snub-nosed monkeys (Rhinopithecus roxellana) in China. <em>Acta Theriologica Sin.</em><strong>30</strong>, 357 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ren,%20B.%20et%20al.%20Inter-population%20variation%20of%20diets%20of%20golden%20snub-nosed%20monkeys%20(Rhinopithecus%20roxellana)%20in%20China.%20Acta%20Theriologica%20Sin.30,%20357%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Colobine monkeys: their ecology, behaviour and evolution. (Cambridge University Press, 1994).</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Jablonski N. G. The natural history of the doucs and snub-nosed monkeys. (World Scientific, 1998).</cite>
</li>
<li id="CR22">
<span class="label">22.</span><cite>Wang, Y., Jiang, X. &amp; Li, D. Classification and distribution of the extant subspecies of golden snub-nosed monkey (Rhinopithecus [Rhinopithecus] roxellana). <em>The Natural History of the Doucs and Snub-Nosed Monkeys</em><strong>10</strong>, 53–64 (1998).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.,%20Jiang,%20X.%20&amp;%20Li,%20D.%20Classification%20and%20distribution%20of%20the%20extant%20subspecies%20of%20golden%20snub-nosed%20monkey%20(Rhinopithecus%20%5BRhinopithecus%5D%20roxellana).%20The%20Natural%20History%20of%20the%20Doucs%20and%20Snub-Nosed%20Monkeys10,%2053%E2%80%9364%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Tie, J. et al. Feeding habits of Rhinopithecus roxellana in Shennongjia Nature Reserve of China in winter and spring. <em>Chin. J. Ecol.</em><strong>29</strong>, 62–68 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tie,%20J.%20et%20al.%20Feeding%20habits%20of%20Rhinopithecus%20roxellana%20in%20Shennongjia%20Nature%20Reserve%20of%20China%20in%20winter%20and%20spring.%20Chin.%20J.%20Ecol.29,%2062%E2%80%9368%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Tie J. The Plant Composition in the Habitat and Food Resources Evaluation of Rhinopithecus roxellana in Shennongjia, China. Doctor. (Beijing Forestry University, 2009).</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Guo, S. et al. Nutrient-specific compensation for seasonal cold stress in a free-ranging temperate colobine monkey. <em>Funct. Ecol.</em><strong>32</strong>, 2170–2180 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Guo,%20S.%20et%20al.%20Nutrient-specific%20compensation%20for%20seasonal%20cold%20stress%20in%20a%20free-ranging%20temperate%20colobine%20monkey.%20Funct.%20Ecol.32,%202170%E2%80%932180%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hou, R. et al. Cold and hungry: combined effects of low temperature and resource scarcity on an edge-of-range temperate primate, the golden snub-nose monkey. <em>Ecography</em><strong>43</strong>, 1672–1682 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hou,%20R.%20et%20al.%20Cold%20and%20hungry:%20combined%20effects%20of%20low%20temperature%20and%20resource%20scarcity%20on%20an%20edge-of-range%20temperate%20primate,%20the%20golden%20snub-nose%20monkey.%20Ecography43,%201672%E2%80%931682%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Qin, J. et al. A human gut microbial gene catalogue established by metagenomic sequencing. <em>Nature</em><strong>464</strong>, 59–U70 (2010).
</cite> [<a href="https://doi.org/10.1038/nature08821" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3779803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20203603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin,%20J.%20et%20al.%20A%20human%20gut%20microbial%20gene%20catalogue%20established%20by%20metagenomic%20sequencing.%20Nature464,%2059%E2%80%93U70%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Jorth, P. et al. Metatranscriptomics of the human oral microbiome during health and disease. <em>Mbio</em><strong>5</strong>, e01012 (2014).
</cite> [<a href="https://doi.org/10.1128/mBio.01012-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3977359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24692635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jorth,%20P.%20et%20al.%20Metatranscriptomics%20of%20the%20human%20oral%20microbiome%20during%20health%20and%20disease.%20Mbio5,%20e01012%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Bickhart, D. et al. Generating lineage-resolved, complete metagenome-assembled genomes from complex microbial communities. <em>Nat. Biotechnol.</em><strong>40</strong>, 711–719 (2022).
</cite> [<a href="https://doi.org/10.1038/s41587-021-01130-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34980911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bickhart,%20D.%20et%20al.%20Generating%20lineage-resolved,%20complete%20metagenome-assembled%20genomes%20from%20complex%20microbial%20communities.%20Nat.%20Biotechnol.40,%20711%E2%80%93719%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Kashaf, S. et al. Recovering prokaryotic genomes from host-associated, short-read shotgun metagenomic sequencing data. <em>Nat. Protoc.</em><strong>16</strong>, 2520–2541 (2021).
</cite> [<a href="https://doi.org/10.1038/s41596-021-00508-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33864056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kashaf,%20S.%20et%20al.%20Recovering%20prokaryotic%20genomes%20from%20host-associated,%20short-read%20shotgun%20metagenomic%20sequencing%20data.%20Nat.%20Protoc.16,%202520%E2%80%932541%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Bowers, R. et al. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. <em>Nat. Biotechnol.</em><strong>35</strong>, 725–731 (2017).
</cite> [<a href="https://doi.org/10.1038/nbt.3893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6436528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28787424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bowers,%20R.%20et%20al.%20Minimum%20information%20about%20a%20single%20amplified%20genome%20(MISAG)%20and%20a%20metagenome-assembled%20genome%20(MIMAG)%20of%20bacteria%20and%20archaea.%20Nat.%20Biotechnol.35,%20725%E2%80%93731%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Bulygin, I. et al. Absence of enterotypes in the human gut microbiomes reanalyzed with non-linear dimensionality reduction methods. <em>PeerJ</em><strong>11</strong>, e15838 (2023).
</cite> [<a href="https://doi.org/10.7717/peerj.15838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10494839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37701837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bulygin,%20I.%20et%20al.%20Absence%20of%20enterotypes%20in%20the%20human%20gut%20microbiomes%20reanalyzed%20with%20non-linear%20dimensionality%20reduction%20methods.%20PeerJ11,%20e15838%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Röttjers, L. &amp; Faust, K. From hairballs to hypotheses–biological insights from microbial networks. <em>FEMS Microbiol. Rev.</em><strong>42</strong>, 761–780 (2018).
</cite> [<a href="https://doi.org/10.1093/femsre/fuy030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6199531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30085090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?R%C3%B6ttjers,%20L.%20&amp;%20Faust,%20K.%20From%20hairballs%20to%20hypotheses%E2%80%93biological%20insights%20from%20microbial%20networks.%20FEMS%20Microbiol.%20Rev.42,%20761%E2%80%93780%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Ning, D. et al. A quantitative framework reveals ecological drivers of grassland microbial community assembly in response to warming. <em>Nat. Commun.</em><strong>11</strong>, 4717 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-18560-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7501310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32948774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ning,%20D.%20et%20al.%20A%20quantitative%20framework%20reveals%20ecological%20drivers%20of%20grassland%20microbial%20community%20assembly%20in%20response%20to%20warming.%20Nat.%20Commun.11,%204717%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Fan, P. et al. Home range variation of two different-sized groups of golden snub-nosed monkeys (<em>Rhinopithecus roxellana</em>) in Shennongjia, China: implications for feeding competition. <em>Zool. Res.</em><strong>40</strong>, 121 (2019).
</cite> [<a href="https://doi.org/10.24272/j.issn.2095-8137.2018.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6378561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29955030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fan,%20P.%20et%20al.%20Home%20range%20variation%20of%20two%20different-sized%20groups%20of%20golden%20snub-nosed%20monkeys%20(Rhinopithecus%20roxellana)%20in%20Shennongjia,%20China:%20implications%20for%20feeding%20competition.%20Zool.%20Res.40,%20121%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Kang L. et al. Metagenomic insights into microbial community structure and metabolism in alpine permafrost on the Tibetan Plateau. <em>Nat. Commun.</em> 15, 5920 (2024).</cite> [<a href="https://doi.org/10.1038/s41467-024-50276-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11247091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39004662/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Thell, A. et al. A review of the lichen family Parmeliaceae–history, phylogeny and current taxonomy. <em>Nord. J. Bot.</em><strong>30</strong>, 641–664 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Thell,%20A.%20et%20al.%20A%20review%20of%20the%20lichen%20family%20Parmeliaceae%E2%80%93history,%20phylogeny%20and%20current%20taxonomy.%20Nord.%20J.%20Bot.30,%20641%E2%80%93664%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kondratyuk, S. Y. et al. Three new genera of the Ramalinaceae (lichen-forming Ascomycota) and the phenomenon of presence of ‘extraneous mycobiont DNA’in lichen associations. <em>Acta Botanica Hungarica</em><strong>61</strong>, 275–323 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kondratyuk,%20S.%20Y.%20et%20al.%20Three%20new%20genera%20of%20the%20Ramalinaceae%20(lichen-forming%20Ascomycota)%20and%20the%20phenomenon%20of%20presence%20of%20%E2%80%98extraneous%20mycobiont%20DNA%E2%80%99in%20lichen%20associations.%20Acta%20Botanica%20Hungarica61,%20275%E2%80%93323%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Pichler, G. et al. Enhanced culturing techniques for the mycobiont isolated from the lichen Xanthoria parietina. <em>Mycol. Prog.</em><strong>20</strong>, 797–808 (2021).
</cite> [<a href="https://doi.org/10.1007/s11557-021-01707-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8550697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34720793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pichler,%20G.%20et%20al.%20Enhanced%20culturing%20techniques%20for%20the%20mycobiont%20isolated%20from%20the%20lichen%20Xanthoria%20parietina.%20Mycol.%20Prog.20,%20797%E2%80%93808%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Yoshino, K. et al. Polyol-assimilation capacities of lichen-inhabiting fungi. <em>Lichenologist</em><strong>52</strong>, 49–59 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yoshino,%20K.%20et%20al.%20Polyol-assimilation%20capacities%20of%20lichen-inhabiting%20fungi.%20Lichenologist52,%2049%E2%80%9359%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Oliphant, K. &amp; Allen-Vercoe, E. Macronutrient metabolism by the human gut microbiome: major fermentation by-products and their impact on host health. <em>Microbiome</em><strong>7</strong>, 1–15 (2019).
</cite> [<a href="https://doi.org/10.1186/s40168-019-0704-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6567490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31196177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oliphant,%20K.%20&amp;%20Allen-Vercoe,%20E.%20Macronutrient%20metabolism%20by%20the%20human%20gut%20microbiome:%20major%20fermentation%20by-products%20and%20their%20impact%20on%20host%20health.%20Microbiome7,%201%E2%80%9315%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Manara, S. et al. Microbial genomes from non-human primate gut metagenomes expand the primate-associated bacterial tree of life with over 1000 novel species. <em>Genome Biol.</em><strong>20</strong>, 299 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1923-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6935492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31883524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manara,%20S.%20et%20al.%20Microbial%20genomes%20from%20non-human%20primate%20gut%20metagenomes%20expand%20the%20primate-associated%20bacterial%20tree%20of%20life%20with%20over%201000%20novel%20species.%20Genome%20Biol.20,%20299%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Almeida, A. et al. A new genomic blueprint of the human gut microbiota. <em>Nature</em><strong>568</strong>, 499–504 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-0965-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6784870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30745586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeida,%20A.%20et%20al.%20A%20new%20genomic%20blueprint%20of%20the%20human%20gut%20microbiota.%20Nature568,%20499%E2%80%93504%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Guo, S., Li, B. &amp; Watanabe, K. Diet and activity budget of Rhinopithecus roxellana in the Qinling Mountains, China. <em>Primates</em><strong>48</strong>, 268–276 (2007).
</cite> [<a href="https://doi.org/10.1007/s10329-007-0048-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17522758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20S.,%20Li,%20B.%20&amp;%20Watanabe,%20K.%20Diet%20and%20activity%20budget%20of%20Rhinopithecus%20roxellana%20in%20the%20Qinling%20Mountains,%20China.%20Primates48,%20268%E2%80%93276%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Orkin, J. D. et al. Seasonality of the gut microbiota of free-ranging white-faced capuchins in a tropical dry forest. <em>ISME J.</em><strong>13</strong>, 183–196 (2019).
</cite> [<a href="https://doi.org/10.1038/s41396-018-0256-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6298967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30135468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orkin,%20J.%20D.%20et%20al.%20Seasonality%20of%20the%20gut%20microbiota%20of%20free-ranging%20white-faced%20capuchins%20in%20a%20tropical%20dry%20forest.%20ISME%20J.13,%20183%E2%80%93196%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Shi, Z. et al. More deterministic assembly constrains the diversity of gut microbiota in freshwater snails. <em>Front. Microbiol.</em><strong>15</strong>, 1394463 (2024).
</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1394463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11260827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39040899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi,%20Z.%20et%20al.%20More%20deterministic%20assembly%20constrains%20the%20diversity%20of%20gut%20microbiota%20in%20freshwater%20snails.%20Front.%20Microbiol.15,%201394463%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Panter, K. &amp; James, L. Natural plant toxicants in milk: a review. <em>J. Anim. Sci.</em><strong>68</strong>, 892–904 (1990).
</cite> [<a href="https://doi.org/10.2527/1990.683892x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2180885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Panter,%20K.%20&amp;%20James,%20L.%20Natural%20plant%20toxicants%20in%20milk:%20a%20review.%20J.%20Anim.%20Sci.68,%20892%E2%80%93904%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Crawford S. D. Lichens used in traditional medicine. <em>Lichen Secondary Metabolites: Bioactive Properties and Pharmaceutical Potential</em>, 31–97 (Springer, 2019).</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Li, Q. et al. Gut microbiome responds compositionally and functionally to the seasonal diet variations in wild gibbons. <em>npj Biofilms Microbiomes</em><strong>9</strong>, 21 (2023).
</cite> [<a href="https://doi.org/10.1038/s41522-023-00388-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10121652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37085482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Q.%20et%20al.%20Gut%20microbiome%20responds%20compositionally%20and%20functionally%20to%20the%20seasonal%20diet%20variations%20in%20wild%20gibbons.%20npj%20Biofilms%20Microbiomes9,%2021%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Donohoe, D. R. et al. The microbiome and butyrate regulate energy metabolism and autophagy in the mammalian colon. <em>Cell Metab.</em><strong>13</strong>, 517–526 (2011).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2011.02.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3099420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21531334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Donohoe,%20D.%20R.%20et%20al.%20The%20microbiome%20and%20butyrate%20regulate%20energy%20metabolism%20and%20autophagy%20in%20the%20mammalian%20colon.%20Cell%20Metab.13,%20517%E2%80%93526%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Krajmalnik-Brown, R. et al. Effects of gut microbes on nutrient absorption and energy regulation. <em>Nutr. Clin. Pract.</em><strong>27</strong>, 201–214 (2012).
</cite> [<a href="https://doi.org/10.1177/0884533611436116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3601187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22367888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krajmalnik-Brown,%20R.%20et%20al.%20Effects%20of%20gut%20microbes%20on%20nutrient%20absorption%20and%20energy%20regulation.%20Nutr.%20Clin.%20Pract.27,%20201%E2%80%93214%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Amabebe, E. et al. Microbial dysbiosis-induced obesity: role of gut microbiota in homoeostasis of energy metabolism. <em>Br. J. Nutr.</em><strong>123</strong>, 1127–1137 (2020).
</cite> [<a href="https://doi.org/10.1017/S0007114520000380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32008579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amabebe,%20E.%20et%20al.%20Microbial%20dysbiosis-induced%20obesity:%20role%20of%20gut%20microbiota%20in%20homoeostasis%20of%20energy%20metabolism.%20Br.%20J.%20Nutr.123,%201127%E2%80%931137%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Liu, Y. et al. Gut microbial metabolites of aromatic amino acids as signals in host–microbe interplay. <em>Trends Endocrinol. Metab.</em><strong>31</strong>, 818–834 (2020).
</cite> [<a href="https://doi.org/10.1016/j.tem.2020.02.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32284282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Gut%20microbial%20metabolites%20of%20aromatic%20amino%20acids%20as%20signals%20in%20host%E2%80%93microbe%20interplay.%20Trends%20Endocrinol.%20Metab.31,%20818%E2%80%93834%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Agus, A., Planchais, J. &amp; Sokol, H. Gut microbiota regulation of tryptophan metabolism in health and disease. <em>Cell Host Microbe</em><strong>23</strong>, 716–724 (2018).
</cite> [<a href="https://doi.org/10.1016/j.chom.2018.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29902437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agus,%20A.,%20Planchais,%20J.%20&amp;%20Sokol,%20H.%20Gut%20microbiota%20regulation%20of%20tryptophan%20metabolism%20in%20health%20and%20disease.%20Cell%20Host%20Microbe23,%20716%E2%80%93724%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Zhang, X. et al. Huddling remodels gut microbiota to reduce energy requirements in a small mammal species during cold exposure. <em>Microbiome</em><strong>6</strong>, 1–14 (2018).
</cite> [<a href="https://doi.org/10.1186/s40168-018-0473-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5994089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29884232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20X.%20et%20al.%20Huddling%20remodels%20gut%20microbiota%20to%20reduce%20energy%20requirements%20in%20a%20small%20mammal%20species%20during%20cold%20exposure.%20Microbiome6,%201%E2%80%9314%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Lin, X. et al. Isolation of potentially novel species expands the genomic and functional diversity of Lachnospiraceae. <em>iMeta</em><strong>3</strong>, e174.56 (2024).
</cite> [<a href="https://doi.org/10.1002/imt2.174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11170972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38882499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20X.%20et%20al.%20Isolation%20of%20potentially%20novel%20species%20expands%20the%20genomic%20and%20functional%20diversity%20of%20Lachnospiraceae.%20iMeta3,%20e174.56%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <em>Bioinformatics</em><strong>30</strong>, 2114–2120 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolger,%20A.%20M.,%20Lohse,%20M.%20&amp;%20Usadel,%20B.%20Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data.%20Bioinformatics30,%202114%E2%80%932120%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with bowtie 2. <em>Nat. Methods</em><strong>9</strong>, 357–359 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20Fast%20gapped-read%20alignment%20with%20bowtie%202.%20Nat.%20Methods9,%20357%E2%80%93359%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Li, D. et al. MEGAHIT v1. 0: a fast and scalable metagenome assembler driven by advanced methodologies and community practices. <em>Methods</em><strong>102</strong>, 3–11 (2016).
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2016.02.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27012178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20D.%20et%20al.%20MEGAHIT%20v1.%200:%20a%20fast%20and%20scalable%20metagenome%20assembler%20driven%20by%20advanced%20methodologies%20and%20community%20practices.%20Methods102,%203%E2%80%9311%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Kang, D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. <em>PeerJ.</em><strong>7</strong>, e7359 (2019).
</cite> [<a href="https://doi.org/10.7717/peerj.7359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6662567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31388474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang,%20D.%20et%20al.%20MetaBAT%202:%20an%20adaptive%20binning%20algorithm%20for%20robust%20and%20efficient%20genome%20reconstruction%20from%20metagenome%20assemblies.%20PeerJ.7,%20e7359%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Olm, M. R. et al. dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. <em>ISME J.</em><strong>11</strong>, 2864–2868 (2017).
</cite> [<a href="https://doi.org/10.1038/ismej.2017.126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5702732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28742071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olm,%20M.%20R.%20et%20al.%20dRep:%20a%20tool%20for%20fast%20and%20accurate%20genomic%20comparisons%20that%20enables%20improved%20genome%20recovery%20from%20metagenomes%20through%20de-replication.%20ISME%20J.11,%202864%E2%80%932868%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Parks, D. H. et al. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <em>Genome Res.</em><strong>25</strong>, 1043–1055 (2015).
</cite> [<a href="https://doi.org/10.1101/gr.186072.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4484387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25977477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parks,%20D.%20H.%20et%20al.%20CheckM:%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates,%20single%20cells,%20and%20metagenomes.%20Genome%20Res.25,%201043%E2%80%931055%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Chaumeil, P. A. et al. GTDB-Tk: a toolkit to classify genomes with the genome taxonomy database. <em>Bioinformatics</em><strong>36</strong>, 1925–1927 (2020).</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7703759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31730192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaumeil,%20P.%20A.%20et%20al.%20GTDB-Tk:%20a%20toolkit%20to%20classify%20genomes%20with%20the%20genome%20taxonomy%20database.%20Bioinformatics36,%201925%E2%80%931927%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Xie, J. et al. Tree Visualization By One Table (tvBOT): a web application for visualizing, modifying and annotating phylogenetic trees. <em>Nucleic Acids Res.</em><strong>51</strong>, W587–W592 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37144476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20J.%20et%20al.%20Tree%20Visualization%20By%20One%20Table%20(tvBOT):%20a%20web%20application%20for%20visualizing,%20modifying%20and%20annotating%20phylogenetic%20trees.%20Nucleic%20Acids%20Res.51,%20W587%E2%80%93W592%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Seemann, T. Prokka: rapid prokaryotic genome annotation. <em>Bioinformatics</em><strong>30</strong>, 2068–2069 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24642063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seemann,%20T.%20Prokka:%20rapid%20prokaryotic%20genome%20annotation.%20Bioinformatics30,%202068%E2%80%932069%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Buchfink, B., Xie, C. &amp; Huson, D. H. Fast and sensitive protein alignment using DIAMOND. <em>Nat. Methods</em><strong>12</strong>, 59–60 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buchfink,%20B.,%20Xie,%20C.%20&amp;%20Huson,%20D.%20H.%20Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND.%20Nat.%20Methods12,%2059%E2%80%9360%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Nilsson, R. H. et al. Mycobiome diversity: high-throughput sequencing and identification of fungi. <em>Nat. Rev. Microbiol.</em><strong>17</strong>, 95–109 (2019).
</cite> [<a href="https://doi.org/10.1038/s41579-018-0116-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30442909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nilsson,%20R.%20H.%20et%20al.%20Mycobiome%20diversity:%20high-throughput%20sequencing%20and%20identification%20of%20fungi.%20Nat.%20Rev.%20Microbiol.17,%2095%E2%80%93109%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Stewart, R. D. et al. Compendium of 4941 rumen metagenome-assembled genomes for rumen microbiome biology and enzyme discovery. <em>Nat. Biotechnol.</em><strong>37</strong>, 953–961 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0202-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6785717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stewart,%20R.%20D.%20et%20al.%20Compendium%20of%204941%20rumen%20metagenome-assembled%20genomes%20for%20rumen%20microbiome%20biology%20and%20enzyme%20discovery.%20Nat.%20Biotechnol.37,%20953%E2%80%93961%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Wood, D. E. &amp; Salzberg, S. L. Kraken: ultrafast metagenomic sequence classification using exact alignments. <em>Genome Biol.</em><strong>15</strong>, R46 (2014).
</cite> [<a href="https://doi.org/10.1186/gb-2014-15-3-r46" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4053813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24580807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wood,%20D.%20E.%20&amp;%20Salzberg,%20S.%20L.%20Kraken:%20ultrafast%20metagenomic%20sequence%20classification%20using%20exact%20alignments.%20Genome%20Biol.15,%20R46%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Lu, J. et al. Bracken: estimating species abundance in metagenomics data. <em>PeerJ. Comp. Sci.</em><strong>3</strong>, e104 (2017).</cite> [<a href="https://doi.org/10.7717/peerj-cs.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12016282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40271438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20J.%20et%20al.%20Bracken:%20estimating%20species%20abundance%20in%20metagenomics%20data.%20PeerJ.%20Comp.%20Sci.3,%20e104%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Beghini, F. et al. Integrating taxonomic, functional, and strain-level profiling of diverse microbial communities with bioBakery 3. <em>Elife</em><strong>10</strong>, e65088 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.65088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8096432/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33944776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beghini,%20F.%20et%20al.%20Integrating%20taxonomic,%20functional,%20and%20strain-level%20profiling%20of%20diverse%20microbial%20communities%20with%20bioBakery%203.%20Elife10,%20e65088%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Franzosa, E. A. et al. Species-level functional profiling of metagenomes and metatranscriptomes. <em>Nat. Methods</em><strong>15</strong>, 962–968 (2018).
</cite> [<a href="https://doi.org/10.1038/s41592-018-0176-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6235447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30377376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Franzosa,%20E.%20A.%20et%20al.%20Species-level%20functional%20profiling%20of%20metagenomes%20and%20metatranscriptomes.%20Nat.%20Methods15,%20962%E2%80%93968%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Bastian, M., Heymann, S. &amp; Jacomy, M. Gephi: an open source software for exploring and manipulating networks. <em>Proc. Int. AAAI Conf. Web Soc. Media</em><strong>3</strong>, 361–362 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bastian,%20M.,%20Heymann,%20S.%20&amp;%20Jacomy,%20M.%20Gephi:%20an%20open%20source%20software%20for%20exploring%20and%20manipulating%20networks.%20Proc.%20Int.%20AAAI%20Conf.%20Web%20Soc.%20Media3,%20361%E2%80%93362%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Costea, P. I. et al. Enterotypes in the landscape of gut microbial community composition. <em>Nat. Microbiol.</em><strong>3</strong>, 8–16 (2018).
</cite> [<a href="https://doi.org/10.1038/s41564-017-0072-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5832044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29255284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costea,%20P.%20I.%20et%20al.%20Enterotypes%20in%20the%20landscape%20of%20gut%20microbial%20community%20composition.%20Nat.%20Microbiol.3,%208%E2%80%9316%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Franzosa, E. A. et al. Relating the metatranscriptome and metagenome of the human gut. <em>Proc. Natl. Acad. Sci. USA</em><strong>111</strong>, E2329–E2338 (2014).
</cite> [<a href="https://doi.org/10.1073/pnas.1319284111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4050606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24843156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Franzosa,%20E.%20A.%20et%20al.%20Relating%20the%20metatranscriptome%20and%20metagenome%20of%20the%20human%20gut.%20Proc.%20Natl.%20Acad.%20Sci.%20USA111,%20E2329%E2%80%93E2338%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Xue, C. et al. DiTing: a pipeline to infer and compare biogeochemical pathways from metagenomic and metatranscriptomic data. <em>Front. Microbiol.</em><strong>12</strong>, 698286 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.698286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8367434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34408730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xue,%20C.%20et%20al.%20DiTing:%20a%20pipeline%20to%20infer%20and%20compare%20biogeochemical%20pathways%20from%20metagenomic%20and%20metatranscriptomic%20data.%20Front.%20Microbiol.12,%20698286%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373803/bin/41522_2025_806_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplement Figure-revision.</a><sup> (1,008.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373803/bin/41522_2025_806_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Tables 1-13.</a><sup> (1.1MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are openly available in NGDC at <a href="https://www.cncb.ac.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cncb.ac.cn/</a>, reference number: PRJCA023520. The metagenome-assembled genomes from this study were submitted to Figshare (10.6084/m9.figshare.28520399.v2).</p>
<p>The code was provided on GitHub at <a href="https://github.com/ZhangMYioz/Adaptive-Strategies-of-the-golden-snub-nosed-monkeys" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZhangMYioz/Adaptive-Strategies-of-the-golden-snub-nosed-monkeys</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Biofilms and Microbiomes are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41522-025-00806-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41522_2025_Article_806.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373803/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373803/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373803%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373803/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373803/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373803/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846855/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373803/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846855/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373803/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373803/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="YWU1ar67FjLbtILrfRIVRO1pugXtLpnWIlcqQJlkr4XFi3PqlrCnlwQmSz6eMQgY">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
