
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Probing the extent of importin‐α targeting of the TAF8 NLS by eliminating its cationic net‐charge - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152546AB8AF32E130546AB005AAFECCE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375970/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="Probing the extent of importin‐α targeting of the TAF8 NLS by eliminating its cationic net‐charge">
<meta name="citation_author" content="Amirabbas Abdoli">
<meta name="citation_author_institution" content="Département de médecine nucléaire et radiobiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada">
<meta name="citation_author" content="Zhihan Yang">
<meta name="citation_author_institution" content="Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada">
<meta name="citation_author_institution" content="Centre de Recherche en Biologie Structurale (CRBS), Montreal, Québec, Canada">
<meta name="citation_author" content="Abdullah Odeh‐Ahmed">
<meta name="citation_author_institution" content="Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada">
<meta name="citation_author" content="Olga Bednova">
<meta name="citation_author_institution" content="Département de médecine nucléaire et radiobiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada">
<meta name="citation_author" content="Bruno Lemieux">
<meta name="citation_author_institution" content="Département de microbiologie et infectiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada">
<meta name="citation_author" content="Leanne Dawe">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada">
<meta name="citation_author" content="Aymeric Ravel‐Chapuis">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada">
<meta name="citation_author_institution" content="Department of Cellular and Molecular Medicine, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada">
<meta name="citation_author_institution" content="Eric Poulin Centre for Neuromuscular Disease, Ottawa, Ontario, Canada">
<meta name="citation_author" content="Pierre Lavigne">
<meta name="citation_author_institution" content="Département de pharmacologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada">
<meta name="citation_author" content="Natalie Zeytuni">
<meta name="citation_author_institution" content="Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada">
<meta name="citation_author_institution" content="Centre de Recherche en Biologie Structurale (CRBS), Montreal, Québec, Canada">
<meta name="citation_author" content="Jeffrey V Leyton">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada">
<meta name="citation_author_institution" content="Department of Cellular and Molecular Medicine, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70272">
<meta name="citation_doi" content="10.1002/pro.70272">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375970/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375970/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375970/pdf/PRO-34-e70272.pdf">
<meta name="description" content="The nucleus, as the control center of the eukaryotic cell, is a prime target for therapeutic interventions due to its role in regulating genetic material. Importin‐α is critical for successful nuclear import as it recognizes and binds to cargo ...">
<meta name="og:title" content="Probing the extent of importin‐α targeting of the TAF8 NLS by eliminating its cationic net‐charge">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The nucleus, as the control center of the eukaryotic cell, is a prime target for therapeutic interventions due to its role in regulating genetic material. Importin‐α is critical for successful nuclear import as it recognizes and binds to cargo ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375970/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375970">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70272"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70272.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375970%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375970/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375970/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375970/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 25;34(9):e70272. doi: <a href="https://doi.org/10.1002/pro.70272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70272</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Probing the extent of importin‐α targeting of the TAF8 NLS by eliminating its cationic net‐charge</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdoli%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Amirabbas Abdoli</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Amirabbas Abdoli</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Département de médecine nucléaire et radiobiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdoli%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Amirabbas Abdoli</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Zhihan Yang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zhihan Yang</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Centre de Recherche en Biologie Structurale (CRBS), Montreal, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhihan Yang</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Odeh%E2%80%90Ahmed%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Abdullah Odeh‐Ahmed</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Abdullah Odeh‐Ahmed</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Odeh%E2%80%90Ahmed%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abdullah Odeh‐Ahmed</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bednova%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Olga Bednova</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Olga Bednova</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Département de médecine nucléaire et radiobiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bednova%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olga Bednova</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lemieux%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Bruno Lemieux</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Bruno Lemieux</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Département de microbiologie et infectiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lemieux%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bruno Lemieux</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dawe%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Leanne Dawe</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Leanne Dawe</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dawe%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Leanne Dawe</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ravel%E2%80%90Chapuis%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Aymeric Ravel‐Chapuis</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Aymeric Ravel‐Chapuis</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Cellular and Molecular Medicine, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>Eric Poulin Centre for Neuromuscular Disease, Ottawa, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ravel%E2%80%90Chapuis%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aymeric Ravel‐Chapuis</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lavigne%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Pierre Lavigne</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Pierre Lavigne</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Département de pharmacologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lavigne%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pierre Lavigne</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeytuni%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Natalie Zeytuni</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Natalie Zeytuni</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Centre de Recherche en Biologie Structurale (CRBS), Montreal, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeytuni%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Natalie Zeytuni</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leyton%20JV%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jeffrey V Leyton</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jeffrey V Leyton</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Cellular and Molecular Medicine, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leyton%20JV%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeffrey V Leyton</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70272-aff-0001">
<sup>
<sup>1</sup>
</sup>Département de médecine nucléaire et radiobiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada</div>
<div id="pro70272-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada</div>
<div id="pro70272-aff-0003">
<sup>
<sup>3</sup>
</sup>Centre de Recherche en Biologie Structurale (CRBS), Montreal, Québec, Canada</div>
<div id="pro70272-aff-0004">
<sup>
<sup>4</sup>
</sup>Département de microbiologie et infectiologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada</div>
<div id="pro70272-aff-0005">
<sup>
<sup>5</sup>
</sup>School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada</div>
<div id="pro70272-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Cellular and Molecular Medicine, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada</div>
<div id="pro70272-aff-0007">
<sup>
<sup>7</sup>
</sup>Eric Poulin Centre for Neuromuscular Disease, Ottawa, Ontario, Canada</div>
<div id="pro70272-aff-0008">
<sup>
<sup>8</sup>
</sup>Département de pharmacologie, Faculté de médecine et sciences des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Natalie Zeytuni, Department of Anatomy and Cell Biology, McGill University, Montreal, Québec, Canada. 
Email: <span>natalie.zeytuni@mcgill.ca</span>
, 
Jeffrey V. Leyton, School of Pharmaceutical Sciences, Faculty of Medicine, University of Ottawa, Ottawa, Ontario, Canada. 
Email: <span>vleyton@uottawa.ca</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 29; Received 2024 Jul 15; Accepted 2025 Aug 4; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375970</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The nucleus, as the control center of the eukaryotic cell, is a prime target for therapeutic interventions due to its role in regulating genetic material. Importin‐α is critical for successful nuclear import as it recognizes and binds to cargo proteins bearing a classical nuclear localization signal (NLS), which facilitates their transport from the cytoplasm into the nucleus. NLS tagging to ‘actively’ import therapeutics provides the most effective means of maximizing nuclear localization and therapeutic efficacy. However, traditional NLSs are highly cationic due to the recognition and binding requirements with importin‐α. Because of their highly ‘super‐charged’ nature, NLS‐tagged therapeutics face significant challenges, including poor pharmacokinetics due to non‐specific interactions. In this study, we engineered novel NLS tags with zero net charge to potentially overcome this limitation. Computational modeling and experimental validation revealed that these net‐neutral NLSs bind to importin‐α with similar modes and energies as their cationic counterpart. High‐resolution structural determination and analysis by X‐ray crystallography then confirmed their binding modes. Biophysical methods using circular dichroism, microscale thermophoresis, and cellular localization studies demonstrated that these NLSs maintain sufficiently stable complexes and acceptable binding to importin‐α and are functional. Additionally, this study revealed that the minor NLS‐binding site of importin‐α, with its extensive cationic surface area, was particularly suited for interactions with the acidic residues of the net‐neutral NLSs. This study provides a foundational understanding of NLS‐importin interactions and presents net‐neutral NLSs as viable candidates for next‐generation NLS‐therapeutic development and expands the scope of nuclear‐targeting therapies.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> importin‐α, microscale thermophoresis, NLS, nuclear transport, peptide engineering, TAF8, targeted therapeutics, zero net‐charge</p></section></section><section id="pro70272-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Nuclear import relies on the presence of a nuclear localization signal (NLS) and its recognition by proteins known as importins (Goswami et al. <a href="#pro70272-bib-0020" class="usa-link" aria-describedby="pro70272-bib-0020">2024</a>). Importins are a multigene family of soluble receptors that ferry NLS‐containing cargos in the cytoplasm across nuclear pore complexes (NPCs) and into the nucleus. Classical nuclear transport involves the initial binding of importin‐α to the NLS of cargo proteins in the cytoplasm. This is then followed by the association with importin‐β, which facilitates passage through the nuclear pore complex in a RanGTP‐dependent manner (Cingolani et al. <a href="#pro70272-bib-0010" class="usa-link" aria-describedby="pro70272-bib-0010">1999</a>; Kobe <a href="#pro70272-bib-0031" class="usa-link" aria-describedby="pro70272-bib-0031"><sup>1999</sup></a>; Melchior et al. <a href="#pro70272-bib-0039" class="usa-link" aria-describedby="pro70272-bib-0039"><sup>1993</sup></a>; Moore and Blobel <a href="#pro70272-bib-0040" class="usa-link" aria-describedby="pro70272-bib-0040"><sup>1993</sup></a>).</p>
<p>Importin‐α possesses an α‐solenoid tertiary structure characterized by a deep concave surface ideal for NLS‐cargo binding (Fournier et al. <a href="#pro70272-bib-0019" class="usa-link" aria-describedby="pro70272-bib-0019">2013</a>). Specifically, importin‐α consists of armadillo (ARM) domains, repetitive units of three α‐helices (termed H1, H2, and H3) arranged in tandem repeats (Fournier et al. <a href="#pro70272-bib-0019" class="usa-link" aria-describedby="pro70272-bib-0019">2013</a>). Importin‐α consists of 10 ARM repeats forming an overall contorted cylindrical shape, with a ~30° right‐hand super‐helical twist (Kobe <a href="#pro70272-bib-0031" class="usa-link" aria-describedby="pro70272-bib-0031">1999</a>). Notably, the twist causes the concave surface to become deeper at the N‐ and C‐termini, resulting in what is commonly known as the major (ARMs 2–4) and minor (ARMs 6–8) NLS‐binding grooves, respectively. They provide large surface areas, mostly negatively charged, that enable multivalent interactions with the elongated and positively charged NLSs.</p>
<p>Classical nuclear transport relies on NLSs that are categorized as monopartite or bipartite, based on the presence of one or two NLS motifs (Chang et al. <a href="#pro70272-bib-0007" class="usa-link" aria-describedby="pro70272-bib-0007">2013</a>; Fontes et al. <a href="#pro70272-bib-0017" class="usa-link" aria-describedby="pro70272-bib-0017"><sup>2000</sup></a>; Zheng et al. <a href="#pro70272-bib-0056" class="usa-link" aria-describedby="pro70272-bib-0056"><sup>2018</sup></a>). Typically, monopartite NLSs can bind to both NLS‐binding grooves but prefer the major site, while bipartite NLSs require binding to both grooves for efficient nuclear transport (Chelsky et al. <a href="#pro70272-bib-0008" class="usa-link" aria-describedby="pro70272-bib-0008">1989</a>; Fontes et al. <a href="#pro70272-bib-0017" class="usa-link" aria-describedby="pro70272-bib-0017"><sup>2000</sup></a>; Marfori et al. <a href="#pro70272-bib-0037" class="usa-link" aria-describedby="pro70272-bib-0037"><sup>2011</sup></a>; Marfori et al. <a href="#pro70272-bib-0036" class="usa-link" aria-describedby="pro70272-bib-0036"><sup>2012</sup></a>). Central to classical recognition are specific amino acids on an NLS that can be mapped to positions (P)1 through P6 within the major NLS‐binding groove (Goswami et al. <a href="#pro70272-bib-0020" class="usa-link" aria-describedby="pro70272-bib-0020">2024</a>). Classical NLS models, exemplified by the ‘SV40’ large T‐antigen NLS (126‐PKKKRKV‐132) (Kalderon et al. <a href="#pro70272-bib-0029" class="usa-link" aria-describedby="pro70272-bib-0029">1984</a>), emphasize NLSs need to be linearly elongated to fit well within the binding grooves. Additionally, the basic residue lysine is required at positions P2, while a lysine or arginine must bind at positions P3 and P5 (Conti and Kuriyan <a href="#pro70272-bib-0011" class="usa-link" aria-describedby="pro70272-bib-0011">2000</a>; Fontes et al. <a href="#pro70272-bib-0017" class="usa-link" aria-describedby="pro70272-bib-0017"><sup>2000</sup></a>; Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052"><sup>2015</sup></a>). As a result, the consensus of a classical monopartite NLS has been defined as K(K/R)x(K/R), with D/E/G being particularly disfavored at the P4 site (Chelsky et al. <a href="#pro70272-bib-0008" class="usa-link" aria-describedby="pro70272-bib-0008">1989</a>; Hodel et al. <a href="#pro70272-bib-0025" class="usa-link" aria-describedby="pro70272-bib-0025"><sup>2001</sup></a>; Smith et al. <a href="#pro70272-bib-0049" class="usa-link" aria-describedby="pro70272-bib-0049"><sup>2018</sup></a>).</p>
<p>The nucleus being the control center of the cell, it is also the major susceptibility center and, hence, a desired target for drug designers (Bavelaar et al. <a href="#pro70272-bib-0002" class="usa-link" aria-describedby="pro70272-bib-0002">2018</a>; Munsell et al. <a href="#pro70272-bib-0042" class="usa-link" aria-describedby="pro70272-bib-0042"><sup>2016</sup></a>; Shin et al. <a href="#pro70272-bib-0048" class="usa-link" aria-describedby="pro70272-bib-0048"><sup>2020</sup></a>). Drug targets include the DNA double helix and DNA‐interacting proteins (Bjornsti and Kaufmann <a href="#pro70272-bib-0004" class="usa-link" aria-describedby="pro70272-bib-0004">2019</a>; Canavese et al. <a href="#pro70272-bib-0005" class="usa-link" aria-describedby="pro70272-bib-0005"><sup>2012</sup></a>; Martin et al. <a href="#pro70272-bib-0038" class="usa-link" aria-describedby="pro70272-bib-0038"><sup>2020</sup></a>). The deregulation of the nuclear import machinery is a key factor in many diseases (Hung and Link <a href="#pro70272-bib-0026" class="usa-link" aria-describedby="pro70272-bib-0026">2011</a>; Mor et al. <a href="#pro70272-bib-0041" class="usa-link" aria-describedby="pro70272-bib-0041"><sup>2014</sup></a>), with increased importin gene expression often driving tumor cell proliferation and progression (Goswami et al. <a href="#pro70272-bib-0020" class="usa-link" aria-describedby="pro70272-bib-0020">2024</a>). Therefore, understanding nuclear import principles, including their role in disease, offers potential avenues for advancing therapeutic strategies targeting the nucleus.</p>
<p>Because the NPC is a permeability barrier and limits the passage of macromolecules, it is also a formidable blockade for therapeutic nuclear access. ‘Active’ nuclear‐directed import and efficient translocations across NPCs are crucial for maximizing nuclear localization (Goswami et al. <a href="#pro70272-bib-0020" class="usa-link" aria-describedby="pro70272-bib-0020">2024</a>). Despite advancements in NLS‐based nuclear targeting systems, additional insight is needed to elucidate the complexities of nuclear transport, disease cellular contexts, and pharmaceutical aspects like the mode of administration.</p>
<p>We previously developed diverse NLS‐tagged antibody‐conjugate therapeutics targeting multiple tumor types, demonstrating their ability to eradicate cancer cells in vivo (Fasih et al. <a href="#pro70272-bib-0016" class="usa-link" aria-describedby="pro70272-bib-0016">2012</a>; Leyton et al. <a href="#pro70272-bib-0033" class="usa-link" aria-describedby="pro70272-bib-0033"><sup>2011</sup></a>; Leyton et al. <a href="#pro70272-bib-0032" class="usa-link" aria-describedby="pro70272-bib-0032"><sup>2015</sup></a>; Paquette et al. <a href="#pro70272-bib-0044" class="usa-link" aria-describedby="pro70272-bib-0044"><sup>2018</sup></a>). Upon intravenous administration, the NLS‐antibody conjugates cleared significantly faster than the unmodified antibody counterparts (Fasih et al. <a href="#pro70272-bib-0016" class="usa-link" aria-describedby="pro70272-bib-0016">2012</a>; Paquette et al. <a href="#pro70272-bib-0044" class="usa-link" aria-describedby="pro70272-bib-0044"><sup>2018</sup></a>). This was most likely caused by the highly cationic net charge (5+) of the SV40 NLS. Studies have shown that antibody therapeutics with increased net‐positive molecular charge have poor pharmacokinetics due to increased non‐specific ionic interactions with cell membranes that, in part, increase the rate of degradation (Datta‐Mannan et al. <a href="#pro70272-bib-0012" class="usa-link" aria-describedby="pro70272-bib-0012"><sup>2015</sup></a>; Hinton et al. <a href="#pro70272-bib-0024" class="usa-link" aria-describedby="pro70272-bib-0024"><sup>2006</sup></a>; Igawa et al. <a href="#pro70272-bib-0027" class="usa-link" aria-describedby="pro70272-bib-0027"><sup>2010</sup></a>; Li et al. <a href="#pro70272-bib-0034" class="usa-link" aria-describedby="pro70272-bib-0034"><sup>2014</sup></a>). Tagging methods typically append 10–20 NLSs per antibody (Beaudoin et al. <a href="#pro70272-bib-0003" class="usa-link" aria-describedby="pro70272-bib-0003">2016</a>; Chen et al. <a href="#pro70272-bib-0009" class="usa-link" aria-describedby="pro70272-bib-0009"><sup>2006</sup></a>), making NLS‐therapeutics ‘super‐charged’ cationic agents with limited pharmacokinetics.</p>
<p>The general transcription factor TFIID is essential for RNA polymerase II transcription initiation in eukaryotic cells. This megadalton protein complex, composed of several TATA‐binding‐associated factors (TAFs), is assembled into a functional transcription factor in the nucleus (Thomas and Chiang <a href="#pro70272-bib-0050" class="usa-link" aria-describedby="pro70272-bib-0050">2006</a>). RNA polymerase II and its associated transcription factors are upregulated in cancer (Sadurni <a href="#pro70272-bib-0045" class="usa-link" aria-describedby="pro70272-bib-0045">2023</a>). The Berger group recently elucidated how an essential TFIID subcomplex consisting of TAF2, TAF8, and TAF10 is assembled in the cytoplasm (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>). Using a peptide containing TAF8's amino acid sequence 297‐PVKKPKIRRKKSLS‐310, its structure bound to importin‐α was solved, revealing the PVKKPKIRR sequence bound the major groove, while only the KKS sequence bound the minor groove, which indicated that while the sequence most likely independently sampled both binding grooves, only the PVKKPKIRR sequence bound preferentially at the major groove. Silencing TAF8 expression prevented TAF2 and TAF10 from localizing in the nucleus. Isothermal titration calorimetry (ITC) studies of TAF8 residues 297–310 determined a dissociation constant (<em>K</em>
<sub>D</sub>) of 10.4 ± 0.8 μM, supporting a 1:1 stoichiometry and further suggesting that the PVKKPKIRR sequence constituted the functional TAF8 NLS and that it was major groove specific.</p>
<p>Published crystal structures and theoretical energy estimations suggest that the molecular recognition of an NLS sequence by importin‐α in the major binding groove has an estimated free energy of at least Δ<em>G</em> ~ −10 kcal/mol (Delfing et al. <a href="#pro70272-bib-0014" class="usa-link" aria-describedby="pro70272-bib-0014">2023</a>; Hodel et al. <a href="#pro70272-bib-0025" class="usa-link" aria-describedby="pro70272-bib-0025"><sup>2001</sup></a>). The P2 site interactions are dominant and the energetic contributions of the P3–P5 range from 25%–66% of the P2 site contribution, with the P4 site interactions contributing the lowest among the consensus (Delfing et al. <a href="#pro70272-bib-0014" class="usa-link" aria-describedby="pro70272-bib-0014">2023</a>). The amino acids that bind at the P1 and P6 sites confer shape to the NLS bound pose within the pocket, which contributes to specificity for different importin‐α subtypes (Fontes et al. <a href="#pro70272-bib-0017" class="usa-link" aria-describedby="pro70272-bib-0017">2000</a>) or minor versus major groove preferences (Chang et al. <a href="#pro70272-bib-0007" class="usa-link" aria-describedby="pro70272-bib-0007">2013</a>). The Klimov group recently showed that the minimum sequence of KKPK matching exactly TAF8 NLS P2–P5 binding residues, transitioned between diverse binding poses due to multiple free energy states in the major binding groove favorable for binding (Delfing et al. <a href="#pro70272-bib-0014" class="usa-link" aria-describedby="pro70272-bib-0014">2023</a>). Interestingly, adding a C‐terminal glutamic acid significantly strengthened binding to importin‐α. Therefore, targeting the nucleus by exploiting the increased nuclear import associated with overexpressed RNA polymerase II and/or its associated transcription factors and the discovery that acidic amino acids potentially play a role in TAF8 NLS binding to importin‐α provides an opportunity for NLS‐therapeutic development by specifically reducing its highly cationic nature.</p>
<p>However, nuclear import is an intricate process that must be carefully considered. Nuclear import rates correlate with the formation of the importin‐α/β/NLS‐cargo complex in the cytoplasm (Timney et al. <a href="#pro70272-bib-0051" class="usa-link" aria-describedby="pro70272-bib-0051">2006</a>). Additionally, nuclear import kinetics are influenced by the cytoplasmic concentrations of both NLS‐cargo proteins and importin‐α, as well as NLS affinity. It is tempting to speculate that increased NLS affinity will proportionally result in improved therapeutic efficacy. However, to the best of our knowledge, this has never been demonstrated and underscores the deep complexity of NLS‐therapeutic development. Therefore, the early design stages of potential NLS candidates with the net cationic charge eliminated must maintain comparable interaction strengths and binding modes to importin‐α, and nuclear localization as the natural NLS counterpart to minimize any potential disturbance of the common thermodynamics and kinetics of nuclear import.</p>
<p>In this study, we describe a combined computational and biophysical approach to design TAF8 NLS analogs, with zero net charge, and test their ability to target importin‐α. The analogs were not explicitly designed to bind with higher affinity but to have comparable interaction strengths and binding modes. To validate the computational modeling design, the analogs were co‐crystallized with importin‐α, and their structures were experimentally determined by X‐ray crystallography. Circular dichroism (CD) and microscale thermophoresis (MST) assays were performed to quantify these interactions and calculate their stoichiometry, complexation stabilities, and <em>K</em>
<sub>D</sub> values. Cellular studies were performed to evaluate nuclear localization. Our results reveal that it is possible to develop NLS analogs with a zero net charge based on TAF8 NLS that still maintain comparable binding strength and modes and can localize to the nucleus. Additionally, potential alternative amino acid interactions are discovered, as well as minor groove binding preference that provides avenues for future synthetic NLS development and/or searching for rare NLS types in nature.</p></section><section id="pro70272-sec-0002"><h2 class="pmc_sec_title">2. RESULTS</h2>
<section id="pro70272-sec-0003"><h3 class="pmc_sec_title">2.1. 
TAF8 NLS binding mode characteristics in the major groove of importin‐α</h3>
<p>A visual and interaction energy‐based inspection of TAF8 NLS–importin‐α complex (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>) was performed to identify suitable positions within the NLS sequence to switch to generate zero net‐charged analogs. TAF8 NLS binds to the major groove with a total contact energy of −76.2 kcal/mol. K299, K300, and K302 provide 87.9% of the total NLS binding energy, with K299 and K302 making the strongest energetic interactions (Table <a href="#pro70272-tbl-0001" class="usa-link">1</a>). Apart from the previously identified contacts made by K299 with the side chains D192, G150, and T155 (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>), several Van der Waals (VdW) force interactions are made with residues A148, S149, T151, S152, N188, G191, and W231. K300 forms hydrogen bonds with N188 and N228, cation–π and hydrogen bonds with W184, and VdW force interactions with S149 and W231.</p>
<section class="tw xbox font-sm" id="pro70272-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Contact energy distributions of the crystal structure analogs complexed to the major NLS‐binding groove of importin‐α.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th rowspan="2" align="left" valign="bottom" colspan="1">NLS (net‐charge)</th>
<th colspan="9" align="left" valign="bottom" rowspan="1">Individual energetic contribution per residue (%)</th>
<th rowspan="2" align="left" valign="bottom" colspan="1">Total (kcal/mol)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">297</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">298</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">299</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">300</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">301</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">302</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">303</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">304</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">305</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TAF8<a href="#pro70272-note-0003" class="usa-link">
<sup>a</sup>
</a> (5+)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (7.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (7.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (31.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (22.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (1.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (33.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (2.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">R (<a href="#pro70272-note-0004" class="usa-link">
<sup>b</sup>
</a>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">R (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−76.2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A1 (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (4.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (5.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (25.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (27.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (0.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (26.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (5.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">D (2.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (2.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−74.93</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A2<a href="#pro70272-note-0005" class="usa-link">
<sup>c</sup>
</a> (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (31.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (333.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (−78.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (−88.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (−5.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (−69.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (−12)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (−7.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (−3.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">28.24</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A3 (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (2.2)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (38.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (23.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (1.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (29.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (3.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">D<a href="#pro70272-note-0006" class="usa-link">
<sup>d</sup>
</a> (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (3.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−67.67</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A4 (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (3.8)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (6.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (32.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (26.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (<a href="#pro70272-note-0004" class="usa-link">
<sup>b</sup>
</a>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (26.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (5.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">D (2.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (1.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−75.46</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70272-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="pro70272-note-0003">
<sup>
<sup>a</sup>
</sup><p class="display-inline">TAF8 NLS is complexed to human importin‐α1 (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>), while the analog structures were determined complexed to murine importin‐α2.</p>
</div>
<div class="fn" id="pro70272-note-0004">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Amino acid interactions clashed with importin‐α side chains and positive contact energies. These energies were included in the total contact energy for TAF8 NLS and analogs but are not calculated in the % individual energetic contribution per residue.</p>
</div>
<div class="fn" id="pro70272-note-0005">
<sup>
<sup>c</sup>
</sup><p class="display-inline">A2 had an overall positive contact energy. Positive and negative percentages reflect contact energies that contributed to repulsive and favorable binding forces, respectively.</p>
</div>
<div class="fn" id="pro70272-note-0006">
<sup>
<sup>d</sup>
</sup><p class="display-inline">Based on the in silico design and the significant amount of contact energy contributed by E304 for C3, it was changed for an aspartic acid residue.</p>
</div>
</div></section><p>The TAF8 NLS residues R304 and R305, at the C‐terminus, are stabilized by water molecules in the crystal cell (space group P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>). Upon deeper inspection, R304 and R305 form contacts with five water molecules (Figure <a href="#pro70272-fig-0001" class="usa-link">1a</a>). As a result, R304 and R305 are oriented up and out of the major groove (Figure <a href="#pro70272-fig-0001" class="usa-link">1b</a>). R305 does not engage with any importin‐α side chains, while R304 sterically clashes with side chains R101 and K102 (Figure <a href="#pro70272-fig-0001" class="usa-link">1b</a>) in ARM1 and suggests that both arginine residues are mobile, can possibly conform to additional configurations, and are suitable for substituting with acidic amino acids.</p>
<figure class="fig xbox font-sm" id="pro70272-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c39a/12375970/57421d5e08c8/PRO-34-e70272-g006.jpg" loading="lazy" id="jats-graphic-1" height="247" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70272-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>C‐terminal arginine and N‐terminal proline binding mode of TAF8 NLS. (a) Top and (b) bottom views of TAF8 NLS (cyan) in the major groove defined by the H3 helices of ARMs 2, 3, and 4 colored yellow, pink, and lime, respectively, of human importin‐α1. The five waters that stabilize the R304 and R305 at the C‐terminus are shown as blue spheres. (c) Surface rendering showing the positioning of P297 and V298 of TAF8 NLS at the P1 site of the major groove. Dashed lines depict the hydrogen bonds between P297 and W231 and V298 and N235. V298 also forms a VdW interaction with R238. The colors indicate the surfaces for the H3 helices of ARMs 2–4, as indicated above. Adapted from PDB: 4WV6 (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>).</p></figcaption></figure><p>A common but less understood feature of NLSs, which is the N‐terminal proline, was also evaluated. Several NLS sequences contain N‐terminal prolines that bind at the P1 site of the major NLS‐binding groove (several are listed in de Oliveira et al. <a href="#pro70272-bib-0013" class="usa-link" aria-describedby="pro70272-bib-0013"><sup>2021</sup></a>) and influence binding affinity. The P1 site for human importin‐α is characterized by a small valley‐shaped‐like pocket with sharp rising peaks composed of W231 and R238 on the N‐ and C‐terminal sides, respectively (Figure <a href="#pro70272-fig-0001" class="usa-link">1c</a>). R238 rises sharply from the P1 site floor and points inwards towards the NLS‐binding groove, while the indole of W231 lies sideways, forming a broad shallow peak. R238 comprises a portion of the hinge that connects ARMs 4 and 5 and defines the C‐terminal boundary of the major groove (Kelley et al. <a href="#pro70272-bib-0030" class="usa-link" aria-describedby="pro70272-bib-0030">2010</a>), while W231 is part of the H3 helix of ARM4.</p>
<p>V298 of TAF8 NLS forms a hydrogen bond and VdW interaction with N235 and R238, respectively. Importantly, the interaction of R238 and V298 prevents P297 from extending further outside the major groove and results in P297 forming a ‘hairpin‐turn,’ common to NLSs with N‐terminal proline residues, which is stabilized by a hydrogen bond with W231. The C‐terminal carboxyl group of P297 forms the hydrogen bond with the nitrogen atom in the indole side chain of W231. As a result, the pyrrolidine side chain “floats out” of the NLS‐binding pocket and does not make any contacts (Figure <a href="#pro70272-fig-0001" class="usa-link">1c</a>). To further support this, other NLSs with this N‐terminal proline motif, such as SV40 and c‐Myc, which also have the hairpin‐turn binding modes (Conti and Kuriyan <a href="#pro70272-bib-0011" class="usa-link" aria-describedby="pro70272-bib-0011">2000</a>; Fontes et al. <a href="#pro70272-bib-0017" class="usa-link" aria-describedby="pro70272-bib-0017"><sup>2000</sup></a>), were evaluated (Figure <a href="#pro70272-supitem-0001" class="usa-link">S1</a>, Supporting Information). These NLSs bind to importin‐α from different species (TAF8:human, SV40:mouse, and c‐Myc:yeast) and were modeled onto the mouse importin‐α paralog 2 (mImpα2) as it is almost identical to its human importin‐α1 counterpart in amino acid sequence similarity (Delfing et al. <a href="#pro70272-bib-0014" class="usa-link" aria-describedby="pro70272-bib-0014">2023</a>), which was subsequently used for crystallization experiments. P297 and V298 side chains, including the modeled counterparts, were not involved in hydrogen bonds or salt bridge formations with the side chains of importin‐α (Figure <a href="#pro70272-supitem-0001" class="usa-link">S1</a>), opening the plausibility to introduce mutations to generate zero net‐charge analogs that can also interact at the P1 site.</p></section><section id="pro70272-sec-0004"><h3 class="pmc_sec_title">2.2. In silico design of TAF8 NLS‐based zero net‐charge analogs</h3>
<p>The in silico design of TAF8 NLS‐based zero net‐charge analogs resulted in the successful creation of four variants (A1–A4) maintaining statistically significant comparable docking scores and binding modes to TAF8 NLS within the major NLS‐binding groove of human importin‐α. As anticipated, alanine scanning highlighted the sensitivity of amino acids K299, K300, and K302 at the P2, P3, and P5 sites. Substituting K299 for an alanine residue impaired the overall NLS binding up to 10% (Figure <a href="#pro70272-supitem-0001" class="usa-link">S2a</a>). In contrast, alanine residue substitutions at positions P297, V298, P301, and I202 resulted in ≤2% reduced docking scores, while there was an improvement for R304 (Figure <a href="#pro70272-supitem-0001" class="usa-link">S2a</a>).</p>
<p>The subsequent substitutions of negatively charged residues at the P1, P4, and P6 sites revealed that R305, protruding outside the major binding groove, could be substituted with glutamic or aspartic acid. The most impaired docking, a 3.5% reduced score, was with a glutamic acid substitution at I303, marking our threshold for acceptable loss in binding strength. Acidic residue substitutions at the other positions resulted in ≤3.0% reduction, allowing for diverse combinations of zero net‐charge analogs at sites P297, V298, and R304 for testing.</p>
<p>Figure <a href="#pro70272-supitem-0001" class="usa-link">S2b</a> lists the frequency of distribution of the 100 docked poses of A1–A4 with statistically significant docking scores comparable to the in silico version of TAF8 NLS. A3 exhibited the best frequency of distribution with superior median (−28.09 kcal/mol) and mean (−28.10 ± 7.33 kcal/mol) scores, suggesting it formed the most stable interactions with the major groove. Additionally, the binding modes for A1–A4 were very similar to one another and the crystal structure of TAF8 NLS (Figure <a href="#pro70272-fig-0002" class="usa-link">2a</a>).</p>
<figure class="fig xbox font-sm" id="pro70272-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c39a/12375970/79982cccaa13/PRO-34-e70272-g004.jpg" loading="lazy" id="jats-graphic-3" height="759" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70272-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Novel contacts for the modeled zero net‐charge analogs. The analogs (A1: yellow, A2: green, A3: marine, A4: magenta) formed a network of contacts with amino acid side chains that are outside the major NLS‐binding pocket of human importin‐α1. (a) Analog binding modes at the major groove. Black NLS is the TAF8 NLS pose from the resolved crystal structure (PDB: 4WV6; Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052"><sup>2015</sup></a>). ARMs 2–4 are colored as in Figure <a href="#pro70272-fig-0001" class="usa-link">1</a>. (b) E297 of A1 formed a hydrogen bond (dashed black line) with D270. (c) E298 of A3 formed a salt bridge (dashed red line) with R238. (d) E304 of A2 forms two hydrogen bonds with R101 and K102 and an additional salt bridge (no line shown) with K102.</p></figcaption></figure></section><section id="pro70272-sec-0005"><h3 class="pmc_sec_title">2.3. Zero net‐charge analogs form novel interactions with human importin‐α</h3>
<p>Table <a href="#pro70272-tbl-0002" class="usa-link">2</a> lists the contact energy distribution of TAF8 NLS and the zero net‐charge analog top poses docked at the major groove of human importin‐α. The analogs made 6–8 hydrogen bonds, 1–4 salt bridges, 29–33 VdW interactions, and 1–2 cation–π bonds. In comparison, the top pose TAF8 NLS made 7 hydrogen bonds, 2 salt bridges, 33 VdW interactions, and 1 cation–π bond.</p>
<section class="tw xbox font-sm" id="pro70272-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Contact energy distribution of top docked poses for TAF8 NLS and the zero net‐charge analogs at the major NLS‐binding groove of human importin‐α.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">NLS</th>
<th colspan="9" align="left" valign="bottom" rowspan="1">Individual energetic contribution per residue (%)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Total (kcal/mol)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TAF8</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (16.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (31.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (42.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (13.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (2.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">R (<a href="#pro70272-note-0007" class="usa-link">
<sup>a</sup>
</a>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">R (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−51.1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A1</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (4.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (18.8)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (40.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (20.8)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (<a href="#pro70272-note-0007" class="usa-link">
<sup>a</sup>
</a>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (19.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">D (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−38.44</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A2</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (4.2)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (8.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (9.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (4.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (39.0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (3.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (30.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−54.15</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A3</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (26.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (34.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (7.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (<a href="#pro70272-note-0007" class="usa-link">
<sup>a</sup>
</a>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (11.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (3.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (17.7)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−49.3</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A4</td>
<td align="left" valign="top" rowspan="1" colspan="1">P (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">V (14.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (49.2)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (10.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (<a href="#pro70272-note-0007" class="usa-link">
<sup>a</sup>
</a>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">K (26.3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">I (3.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">D (0.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">E (0)</td>
<td align="left" valign="top" rowspan="1" colspan="1">−45.2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70272-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="pro70272-note-0007">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Amino acid interactions clashed with importin‐α side chains and positive contact energies. These energies were included in the total contact energy for TAF8 NLS and analogs but cannot be calculated in the % individual energetic contribution per residue.</p>
</div></div></section><p>Notably, the N‐terminal amine of E297 of A1 formed a hydrogen bond with the side chain of D270, which is located outside the binding groove on ARM5 but in the vicinity of the P1 site (Figure <a href="#pro70272-fig-0002" class="usa-link">2b</a>). For A3, the side chain of E298 formed extensive bonds with nearby amino acids of importin‐α and contributed the second highest amount of contact energy, behind K299 (Table <a href="#pro70272-tbl-0002" class="usa-link">2</a>). E298 formed a novel salt bridge with R238 at the C‐terminal edge of the P1 site (Figure <a href="#pro70272-fig-0002" class="usa-link">2c</a>), while maintaining the important hydrogen bonds with the side chains of W231 and N235 deep in the binding groove. For A2 and A3, the side chain of E304 extended out of the major groove as was the case for R304 of TAF8 NLS (Figure <a href="#pro70272-fig-0001" class="usa-link">1a,b</a>). However, E304 formed multiple new hydrogen bonds and salt bridges with R101 and K102 in ARM 1 (Figure <a href="#pro70272-fig-0002" class="usa-link">2d</a>), whereas the natural R304 previously clashed with these residues (Figure <a href="#pro70272-fig-0001" class="usa-link">1b</a>).</p>
<p>Taken together, in silico design of potential NLS analogs with zero net charge derived from TAF8 NLS is possible for at least three reasons. One, the C‐terminal R305 residue does not play a role in binding importin‐α and can be substituted for an acidic residue. Two, glutamic acid substitutions at positions P297, V298, and E304 can form novel contacts with amino acid side chains at the edges (at the P1 and P6 sites) of the major NLS‐binding groove. Three, the A1–A4 analogs, with the diverse interspersed acidic amino acids, can bind the major groove with comparable docking scores and maintain very similar binding modes compared to TAF8 NLS.</p></section><section id="pro70272-sec-0006"><h3 class="pmc_sec_title">2.4. Atomic structures of the four zero net‐charge analogs bound to mImpα2 and a comparative analysis between the crystallized and modeled structures</h3>
<p>The crystal structures of A1–A4 in complex with mImpα2 at resolutions ranging from 2.11 to 2.42 Å were determined (Table <a href="#pro70272-tbl-0003" class="usa-link">3</a>). The electron densities of the 10 ARM domains of mImpα2 without the importin‐β‐binding (IBB)‐domain were clearly resolved including the P1–P6 and P′1–P′5 sites for the major and minor NLS‐binding grooves, respectively. The crystal structures had the same space groups as the original TAF8 NLS/importin‐α complex and every amino acid was resolved for all analogs bound at the major groove (Figure <a href="#pro70272-fig-0003" class="usa-link">3a</a>). Accordingly, the analogs bury an average surface area of 629.6 ± 21.2 Å<sup>2</sup> in the major groove.</p>
<section class="tw xbox font-sm" id="pro70272-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Data refinement statistics of the crystals.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">NLS Analogs</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">A1</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">A2</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">A3</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">A4</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">PDB code</td>
<td align="left" valign="top" rowspan="1" colspan="1">9CL8</td>
<td align="left" valign="top" rowspan="1" colspan="1">9CLE</td>
<td align="left" valign="top" rowspan="1" colspan="1">9CLF</td>
<td align="left" valign="top" rowspan="1" colspan="1">9CLG</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Space group</td>
<td align="left" valign="top" rowspan="1" colspan="1">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Molecules/unit</td>
<td align="left" valign="top" rowspan="1" colspan="1">1</td>
<td align="left" valign="top" rowspan="1" colspan="1">1</td>
<td align="left" valign="top" rowspan="1" colspan="1">1</td>
<td align="left" valign="top" rowspan="1" colspan="1">1</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">Cell dimensions</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>a</em>, <em>b</em>, <em>c</em> (Å)</td>
<td align="left" valign="top" rowspan="1" colspan="1">78.41, 89.70, 99.42</td>
<td align="left" valign="top" rowspan="1" colspan="1">78.43, 89.82, 99.66</td>
<td align="left" valign="top" rowspan="1" colspan="1">78.40, 89.75, 99.52</td>
<td align="left" valign="top" rowspan="1" colspan="1">78.01, 89.96, 99.16</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>α</em>, <em>β</em>, <em>γ</em> (°)</td>
<td align="left" valign="top" rowspan="1" colspan="1">90, 90, 90</td>
<td align="left" valign="top" rowspan="1" colspan="1">90, 90, 90</td>
<td align="left" valign="top" rowspan="1" colspan="1">90, 90, 90</td>
<td align="left" valign="top" rowspan="1" colspan="1">90, 90, 90</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Resolution (Å)</td>
<td align="left" valign="top" rowspan="1" colspan="1">50–2.11</td>
<td align="left" valign="top" rowspan="1" colspan="1">50–2.34</td>
<td align="left" valign="top" rowspan="1" colspan="1">50–2.36</td>
<td align="left" valign="top" rowspan="1" colspan="1">50–2.42</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>R</em>
<sub>merge</sub> (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">7.3 (50.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">8.0 (54.1)</td>
<td align="left" valign="top" rowspan="1" colspan="1">9.2 (55.8)</td>
<td align="left" valign="top" rowspan="1" colspan="1">7.2 (48.3)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>I</em>/<em>σI</em>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">21.8 (2.5)</td>
<td align="left" valign="top" rowspan="1" colspan="1">20.67 (2.58)</td>
<td align="left" valign="top" rowspan="1" colspan="1">18.65 (2.37)</td>
<td align="left" valign="top" rowspan="1" colspan="1">20.37 (2.71)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Completeness (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">100 (100)</td>
<td align="left" valign="top" rowspan="1" colspan="1">100 (100)</td>
<td align="left" valign="top" rowspan="1" colspan="1">100 (100)</td>
<td align="left" valign="top" rowspan="1" colspan="1">99.9 (99.8)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Redundancy</td>
<td align="left" valign="top" rowspan="1" colspan="1">6.6 (6.9)</td>
<td align="left" valign="top" rowspan="1" colspan="1">6.4 (6.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">6.4 (6.6)</td>
<td align="left" valign="top" rowspan="1" colspan="1">6.4 (6.3)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CC1/2</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.999 (0.901)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.997 (0.862)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.995 (0.870)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.997 (0.928)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CC*</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.000 (0.974)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.999 (0.962)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.999 (0.964)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.999 (0.981)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Wavelength (Å)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.9537</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.9537</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.9537</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.9537</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">Refinement</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Resolution (Å)</td>
<td align="left" valign="top" rowspan="1" colspan="1">49.761–2.11</td>
<td align="left" valign="top" rowspan="1" colspan="1">49.880–2.34</td>
<td align="left" valign="top" rowspan="1" colspan="1">49.811–2.36</td>
<td align="left" valign="top" rowspan="1" colspan="1">49.628–2.42</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">No. reflections (unique)</td>
<td align="left" valign="top" rowspan="1" colspan="1">41,131</td>
<td align="left" valign="top" rowspan="1" colspan="1">30,438</td>
<td align="left" valign="top" rowspan="1" colspan="1">29,674</td>
<td align="left" valign="top" rowspan="1" colspan="1">27,207</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>R</em>
<sub>work</sub>/<em>R</em>
<sub>free</sub> (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">15.95/19.16</td>
<td align="left" valign="top" rowspan="1" colspan="1">15.72/20.50</td>
<td align="left" valign="top" rowspan="1" colspan="1">15.64/20.78</td>
<td align="left" valign="top" rowspan="1" colspan="1">16.07/20.24</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">No. of atoms</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">mImpɑ2</td>
<td align="left" valign="top" rowspan="1" colspan="1">3285</td>
<td align="left" valign="top" rowspan="1" colspan="1">3287</td>
<td align="left" valign="top" rowspan="1" colspan="1">3288</td>
<td align="left" valign="top" rowspan="1" colspan="1">3265</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ligand/ion</td>
<td align="left" valign="top" rowspan="1" colspan="1">32</td>
<td align="left" valign="top" rowspan="1" colspan="1">58</td>
<td align="left" valign="top" rowspan="1" colspan="1">34</td>
<td align="left" valign="top" rowspan="1" colspan="1">52</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Water</td>
<td align="left" valign="top" rowspan="1" colspan="1">313</td>
<td align="left" valign="top" rowspan="1" colspan="1">260</td>
<td align="left" valign="top" rowspan="1" colspan="1">246</td>
<td align="left" valign="top" rowspan="1" colspan="1">190</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">NLS analogs (major groove)</td>
<td align="left" valign="top" rowspan="1" colspan="1">75</td>
<td align="left" valign="top" rowspan="1" colspan="1">76</td>
<td align="left" valign="top" rowspan="1" colspan="1">75</td>
<td align="left" valign="top" rowspan="1" colspan="1">75</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">NLS analogs (minor groove)</td>
<td align="left" valign="top" rowspan="1" colspan="1">9</td>
<td align="left" valign="top" rowspan="1" colspan="1">41</td>
<td align="left" valign="top" rowspan="1" colspan="1">50</td>
<td align="left" valign="top" rowspan="1" colspan="1">50</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">B‐factors (Å)</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">mImpɑ2</td>
<td align="left" valign="top" rowspan="1" colspan="1">42.47</td>
<td align="left" valign="top" rowspan="1" colspan="1">45.65</td>
<td align="left" valign="top" rowspan="1" colspan="1">46.34</td>
<td align="left" valign="top" rowspan="1" colspan="1">47.17</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ligand/ion</td>
<td align="left" valign="top" rowspan="1" colspan="1">70.08</td>
<td align="left" valign="top" rowspan="1" colspan="1">79.75</td>
<td align="left" valign="top" rowspan="1" colspan="1">81.6</td>
<td align="left" valign="top" rowspan="1" colspan="1">79.20</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Water</td>
<td align="left" valign="top" rowspan="1" colspan="1">47.83</td>
<td align="left" valign="top" rowspan="1" colspan="1">49.26</td>
<td align="left" valign="top" rowspan="1" colspan="1">47.32</td>
<td align="left" valign="top" rowspan="1" colspan="1">48.66</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">NLS analogs (major groove)</td>
<td align="left" valign="top" rowspan="1" colspan="1">59.26</td>
<td align="left" valign="top" rowspan="1" colspan="1">73.32</td>
<td align="left" valign="top" rowspan="1" colspan="1">73.56</td>
<td align="left" valign="top" rowspan="1" colspan="1">74.02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">NLS analogs (minor groove)</td>
<td align="left" valign="top" rowspan="1" colspan="1">81.47</td>
<td align="left" valign="top" rowspan="1" colspan="1">108.51</td>
<td align="left" valign="top" rowspan="1" colspan="1">117.36</td>
<td align="left" valign="top" rowspan="1" colspan="1">110.486</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MolProbity score</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.21</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.34</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.44</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.20</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Rotamer outliers (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.27</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.32</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.58</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.8</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ramachandran favored (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">98.84</td>
<td align="left" valign="top" rowspan="1" colspan="1">97.92</td>
<td align="left" valign="top" rowspan="1" colspan="1">98.16</td>
<td align="left" valign="top" rowspan="1" colspan="1">98.39</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ramachandran allowed (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.16</td>
<td align="left" valign="top" rowspan="1" colspan="1">2.08</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.84</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.61</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ramachandran outliers (%)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.23</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.23</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.23</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">RMS deviations</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Bond lengths (Å)</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.0090</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.0071</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.0070</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.0074</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Bond angles (°)</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.5319</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.3672</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.4036</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.4706</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Buffer condition</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.8M NaCit, 0.1M HEPES pH 6.4, 60% glycerol, 10 mM DTT</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.7M NaCit, 0.1M HEPES pH 6.6, 60% glycerol, 10 mM DTT</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.65M NaCit, 0.1M HEPES pH 7.0, 60% glycerol, 10 mM DTT</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.8M NaCit, 0.1M HEPES pH 6.2, 60% glycerol, 10 mM DTT</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70272-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="pro70272-note-0008"><p>
<em>Note</em>: Statistic values present in parentheses correspond to the highest resolution shell data.</p></div></div></section><figure class="fig xbox font-sm" id="pro70272-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c39a/12375970/b066cd72c7c7/PRO-34-e70272-g005.jpg" loading="lazy" id="jats-graphic-5" height="559" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70272-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Resolved crystal structures of the zero net‐charge analogs bound to the major groove of mImpα2. (a) The Fo–Fc difference maps of A1–A4 at the major groove of mImpα2. The superposition of the (b) Cα's and (c) side chains of the determined crystal structures of A1 (yellow), A2 (green), A3 (marine), and A4 (magenta) in the major groove of mImpα2, along with TAF8 NLS (black) from PDB: 4WV6 (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>).</p></figcaption></figure><p>Superpositioning the four analog structures at the major groove showed that the sequences also share almost identical binding modes, based on the alpha‐carbon's (Cα's) as the crystal structure of TAF8 NLS (Figure <a href="#pro70272-fig-0003" class="usa-link">3b</a>). The Cα backbone of the analogs exhibited RMSD values of 0.485, 0.529, 0.487, and 0.513 Å for A1, A2, A3, and A4, respectively, at the major groove.</p>
<p>Interestingly, the analog side chains displayed diverse avidities, all markedly different from TAF8 NLS. In the published structure of human importin‐ɑ bound to TAF8 NLS, the resolved number of atoms at the major and minor NLS‐binding grooves was 3,264 for importin‐ɑ, 78 for the major groove ligand, and 24 for the minor groove ligand (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>). The analog crystal structures had highly similar atom counts (Table <a href="#pro70272-tbl-0003" class="usa-link">3</a>). The B‐factors, which reflect atomic mobility or flexibility in a crystal structure,  were lower for human importin‐ɑ1 and both bound ligands compared to the analogs and their corresponding mImpɑ2 complexes (Table <a href="#pro70272-tbl-0003" class="usa-link">3</a>). Among the analogs, the A1/mImpα2 complex exhibited the greatest stability based on B‐factors. The analogs formed a total of 8–10 hydrogen bonds, 1 salt bridge, 37–40 VdW interactions, and 1–2 cation–π bonds. In comparison, TAF8 NLS formed 11 hydrogen bonds, 1 salt bridge, 35 VdW interactions, and 2 cation–π bonds. As the total number and types of bonds formed by the analogs were similar to TAF8 NLS, it was not apparent why the analogs had varying B‐factors that were notably higher and reflecting less stability. Most likely, subtle yet vital contacts account for these differences.</p>
<p>Visual inspection revealed that although the side chains for K299, K300, and K301 residues aligned at the P2, P3, and P4 sites, indicating that the analogs also relied on the consensus sequences for their binding (Figure <a href="#pro70272-fig-0003" class="usa-link">3c</a>), there were notable spatial orientational differences for some other residues and the amino acid contributions for A1–A4 (Table <a href="#pro70272-tbl-0001" class="usa-link">1</a>) changed significantly from the computational modeling (Table <a href="#pro70272-tbl-0002" class="usa-link">2</a>). For example, the P297 residues of each analog were positioned differently from each other and from TAF8 NLS (Figure <a href="#pro70272-fig-0003" class="usa-link">3c</a>). A1, the only analog with an N‐terminal glutamic acid, was positioned over W231, matching the spatial position of P297 of TAF8 NLS and the hydrogen bond formed with W231 (Figure <a href="#pro70272-supitem-0001" class="usa-link">S3a</a> vs. Figure <a href="#pro70272-fig-0002" class="usa-link">2b</a>). However, instead of forming a hydrogen bond with the side chain of D270, a hydrogen bond was formed with the solvent molecule DTT, derived from the crystallization buffer (Figure <a href="#pro70272-supitem-0001" class="usa-link">S3a</a>). E297 was further stabilized by forming hydrogen bonds with two water molecules. DTT also interfered with the N‐terminal P297 of A3. Instead of forming a bond, DTT had an energetically repulsive effect on P297 and pushed it into the binding groove (Figure <a href="#pro70272-supitem-0001" class="usa-link">S3b</a>). As a result, E298 was shifted too far away to form a salt bridge with the side chain of R238 (Figure <a href="#pro70272-supitem-0001" class="usa-link">S3b</a> vs. Figure <a href="#pro70272-fig-0002" class="usa-link">2c</a>) and was stabilized by a hydrogen bond with a water molecule. As previously mentioned, it was not an aim to generate zero net‐charged NLSs with superior binding affinity, as it is unknown how this would affect natural classical nuclear transport. Hence, E304 of A3 was changed to D304 as the former contributed 17.7% of the total contact energy (Table <a href="#pro70272-tbl-0002" class="usa-link">2</a>). D304 in the crystal structure of A3 complexed to mImpα2 was not able to form bonds with R101 and K102, previously observed with E304 (Figure <a href="#pro70272-supitem-0001" class="usa-link">S3c</a> vs. Figure <a href="#pro70272-fig-0002" class="usa-link">2d</a>). However, E304 of A2 was also unable to form bonds with R101 and K102 and, again, was due to DTT interference (Figure <a href="#pro70272-supitem-0001" class="usa-link">S3c</a>). DTT was slotted near the P1 site. An energetic analysis revealed that DTT formed a hydrogen bond with the pyyrolidine amine of P297. As a result, P297 was not able to position itself further up the N‐terminus of the H3 helix of ARM4, as observed for the modeled A2 (Figure <a href="#pro70272-supitem-0001" class="usa-link">S4</a>). This resulted in an overall displacement of the Cα backbone of A2 in the crystal structure. As a result, E304 was also positioned too far away from binding to R101 and K102.</p>
<p>Therefore, the individual amino acid contributions for A1–A4 to the total contact energies were very different from the modeled versus the crystal structures (Table <a href="#pro70272-tbl-0001" class="usa-link">1</a> vs. Table <a href="#pro70272-tbl-0002" class="usa-link">2</a>). Importantly, the three identified ‘hot spots,’ namely positions 297, 298, and 304 of the natural TAF8 NLS, lost their novel contacts formed in the modeled structures. These changes resulted in overall less contact energies being formed for mImpα2 in the crystal structures compared to the energies from the modeled analogs for human importin‐α1 (Table <a href="#pro70272-tbl-0001" class="usa-link">1</a> vs. Table <a href="#pro70272-tbl-0002" class="usa-link">2</a>). Nevertheless, the binding modes were highly similar and A1, A3, and A4 had cumulative contact energies to mImpα2 comparable to the contact energy that TAF8 NLS had with human importin‐α1 (Table <a href="#pro70272-tbl-0001" class="usa-link">1</a>). In addition, it cannot be excluded that these deviations in binding energies were driven by crystallization conditions (and that in solution these binding energies will be similar to those calculated in the original in silico model).</p></section><section id="pro70272-sec-0007"><h3 class="pmc_sec_title">2.5. Binding of the zero net‐charge analogs at the minor groove</h3>
<p>In the original TAF8 NLS–human importin‐α1 complex, the electron density in the minor groove was insufficient to resolve the NLS, and because the ITC measurement determined a <em>K</em>
<sub>D</sub> value on the low end of the affinity spectrum for monopartite NLSs, it was concluded to be an exclusively major groove‐specific binder (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>). By contrast, residues 297–302 for A3 and A4 were clearly defined in the minor groove (Figure <a href="#pro70272-fig-0004" class="usa-link">4a</a>). A3 had the best cumulative interaction energy (−43.46 kcal/mol) at the minor groove and formed 9 hydrogen bonds, 2 salt bridges, 27 VdW interactions, and 1 cation–π bond. A3 buried 504.2 Å<sup>2</sup> of surface area.</p>
<figure class="fig xbox font-sm" id="pro70272-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c39a/12375970/030dffb7095a/PRO-34-e70272-g007.jpg" loading="lazy" id="jats-graphic-7" height="547" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70272-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Resolved crystal structures of the zero net‐charge analogs at the minor groove of mImpα2. (a) The Fo–Fc difference maps of A1–A4 at the major groove of mImpα2. The A2–A4 analogs in the minor NLS‐binding groove have increased prominence and rigidity compared to TAF8 NLS (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>). (b) Notable differences in the binding modes between the analogs (A4 shown) in the minor groove as opposed to the major groove (Figure <a href="#pro70272-fig-0004" class="usa-link">4</a>) of mImpα2. ARMs 6–8 colored lime, pink, and lime green, respectively.</p></figcaption></figure><p>The P′0 site (counterpart of P1 of the major groove) of the NLS‐binding minor groove is defined by a pocket with two sharp peaks formed by T409 and W399. In contrast to the hairpin turn motif made by N‐terminal proline residue‐containing NLSs at the major groove, P297 for both A3 and A4 was straight and positioned near S406 and formed hydrogen bonds (Figure <a href="#pro70272-fig-0004" class="usa-link">4b</a>). As anticipated, K299 bound at the critical P′1 site (counterpart of P2). However, the binding energies are dominated by K300 and K301 at positions P′2 and P′4. E298 and E301 from A3 and A4, respectively, were the only acidic amino acids introduced that were resolved at the minor groove, and they formed multiple VdW interactions. The core NLS consensus motif dominated the interaction energies, accounting for nearly 100% and 91% of the total binding energy for A3 and A4, respectively.</p></section><section id="pro70272-sec-0008"><h3 class="pmc_sec_title">2.6. Zero net‐charge analogs demonstrate effective binding to mImpα2 and drive localization to the cell nucleus, potentially favoring the minor groove</h3>
<p>In the presence of TAF8 NLS, mImpα2 was stabilized, evidenced by the increase in temperature denaturation, shift in stability curves, and the reduction of the unfolded state (Figures <a href="#pro70272-supitem-0001" class="usa-link">S5a–c</a> and <a href="#pro70272-fig-0005" class="usa-link">5a</a>). All thermodynamic results are listed in Table <a href="#pro70272-supitem-0001" class="usa-link">S1</a>. The analogs also stabilized mImpα2, albeit to a lesser extent than TAF8 NLS. The stabilization order was A3 &gt; A1 &gt; A4 ≈ A2.</p>
<figure class="fig xbox font-sm" id="pro70272-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c39a/12375970/42d3cd3d96e1/PRO-34-e70272-g003.jpg" loading="lazy" id="jats-graphic-9" height="249" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70272-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Circular dichroism (CD) and microscale thermophoresis (MST). (a) The proportion of the unfolded state as a function of temperature. Data was fitted from the thermal denaturation profile (Figure <a href="#pro70272-supitem-0001" class="usa-link">S5b</a>). (b) The logarithmic scale binding affinities using a one‐binding site model.</p></figcaption></figure><p>Fitting the MST data to a one‐binding site model (Figures <a href="#pro70272-supitem-0001" class="usa-link">S5d</a> and <a href="#pro70272-fig-0005" class="usa-link">5b</a>), TAF8 NLS exhibited the highest apparent affinity, with <em>K</em>
<sub>D</sub> values approximately 10‐ to 16‐fold higher than those of the analogs (Table <a href="#pro70272-supitem-0001" class="usa-link">S2</a>). Trowitzsch et al. originally reported <em>K</em>
<sub>D</sub> value for TAF8 NLS of ~10 μM with 1:1 stoichiometry (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>). Due to our crystallography findings revealing the presence of the analogs at both binding grooves, in contrast to TAF8 NLS strictly in the minor groove, the TAF8 NLS and the zero net‐charge analog binding isotherms were also fitted with a two‐binding site model. In contrast to Trowitzsch et al., the binding affinities were both ~1 μM for both sites and suggest TAF8 NLS actually binds at both grooves.</p>
<p>Here, the <em>K</em>
<sub>D</sub> values for the analogs showed significant disparity between the two sites (Table <a href="#pro70272-supitem-0001" class="usa-link">S2</a>). The <em>K</em>
<sub>D</sub> values at site 1 (<em>K</em>
<sub>D</sub>1) ranged from 3‐ to 6‐fold weaker, while the <em>K</em>
<sub>D</sub> values at site 2 (K<sub>D</sub>2) were approximately 28‐fold weaker than TAF8 NLS. To graphically discern two binding phases to two different binding sites, the two combined <em>K</em>
<sub>D</sub> values of 1 μM for <em>K</em>
<sub>D</sub>1 and <em>K</em>
<sub>D</sub>2 amounted to 100% of bound mImpα2 (Figure <a href="#pro70272-supitem-0001" class="usa-link">S6</a>). Hill coefficient analyses supported the fitted data (Tso et al. <a href="#pro70272-bib-0053" class="usa-link" aria-describedby="pro70272-bib-0053">2018</a>), indicating that TAF8 NLS binds mImpα2 with positive cooperativity, while the analogs bind with lower cooperativity (Table <a href="#pro70272-supitem-0001" class="usa-link">S2</a> and Figure <a href="#pro70272-supitem-0001" class="usa-link">S7</a>).</p>
<p>To test the functionality of the zero net‐charge approach in generating novel NLSs, mammalian cell lines were transfected with constructs encoding enhanced green fluorescent protein (EGFP) fused to canonical NLSs (SV40 and TAF8), or the TAF8‐derived A3 and A4 analogs. Nuclear localization was assessed by fluorescence microscopy (Figure <a href="#pro70272-fig-0006" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="pro70272-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c39a/12375970/d141c4598053/PRO-34-e70272-g001.jpg" loading="lazy" id="jats-graphic-11" height="819" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70272-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Fluorescence microscopy. (a) The overlaid line fluorescence intensities for C2C12 myoblasts transfected with the different pCIneoEGFP plasmids. (b–f) Representative C2C12 myoblasts fluorescence microscopy images transfected with different constructs.</p></figcaption></figure><p>Line overlay analysis of fluorescence intensity across C2C12 myoblasts revealed sharp increases in fluorescent nuclear accumulation for EGFP‐A3 and EGFP‐A4, with peak fluorescence localized within the ~20–40 μm zone corresponding to nuclear regions (Figure <a href="#pro70272-fig-0006" class="usa-link">6a</a>). This nuclear accumulation was comparable to that observed in cells expressing EGFP‐TAF8 NLS and EGFP‐SV40 NLS fusion proteins. In contrast, myoblasts expressing EGFP alone exhibited only a shallow increase within the nuclear region, indicative of passive diffusion.</p>
<p>Peak fluorescence intensity was highest in cells transfected with EGFP‐TAF8 NLS and EGFP‐A3, both of which displayed strong nuclear‐specific fluorescence and sharp contrast relative to the cytoplasm (Figure <a href="#pro70272-fig-0006" class="usa-link">6d,f</a>). EGFP‐SV40 NLS showed robust and crisp nuclear‐specific fluorescence intensity, consistent with the known strength of this canonical NLS. Conversely, EGFP alone remained diffusely distributed throughout the cell (Figure <a href="#pro70272-fig-0006" class="usa-link">6b</a>), further supporting its lack of nuclear targeting and passive diffusion. Fluorescence imaging also indicated active nuclear localization of EGFP‐A4 but to a lesser extent than EGFP‐A3 (Figure <a href="#pro70272-fig-0006" class="usa-link">6e</a>).</p>
<p>Experiments were also performed to determine whether nuclear localization was energy‐dependent, where the RanGDP/GTP gradient is deactivated (details in Data <a href="#pro70272-supitem-0001" class="usa-link">S1</a>) and previously shown to inhibit active nuclear import (Schwoebel et al. <a href="#pro70272-bib-0047" class="usa-link" aria-describedby="pro70272-bib-0047">2002</a>). However, no differences in nuclear localization were observed across constructs or at multiple time points post‐transfection to the results obtained with standard transfection and imaging, as shown in Figure <a href="#pro70272-fig-0006" class="usa-link">6</a>.</p>
<p>Similar trends were observed in Hela cells, where EGFP‐NLS constructs exhibited sharply elevated nuclear fluorescence relative to EGFP alone (Figure <a href="#pro70272-supitem-0001" class="usa-link">S8a–f</a>). Interestingly, the total nuclear localization intensities were less than half observed in C2C12 myoblasts across constructs. In this context, A4 outperformed A3 in promoting nuclear localization. Additional wide‐field images, including DAPI staining and representative fields of broader cell populations, are shown in Figure <a href="#pro70272-supitem-0001" class="usa-link">S8g,h</a>. Taken together, these findings demonstrate that the TAF8‐derived zero net‐charge analogs, A3 and A4, are functional NLSs in the cells evaluated in this study.</p>
<p>Analysis of the minor and major NLS‐binding grooves revealed very interesting aspects regarding their respective cationic surfaces. The minor groove, with three cationic patches, has a combined surface area 342% larger than the major groove (Figure <a href="#pro70272-fig-0007" class="usa-link">7</a>). The three cationic patches are all grouped at the N‐terminus that contains positions P′4 and extend to the edges of the binding groove and suggest they can engage with the acidic amino acids at the C‐termini of the NLS analogs. This is supported by analogs A3 and A4 having most of their residues resolved at the minor groove (Figure <a href="#pro70272-fig-0005" class="usa-link">5</a>). This also supports why TAF8 NLS was not resolved at the minor groove, as its C‐terminal arginine residues could have strongly been repulsed by the cationic surfaces.</p>
<figure class="fig xbox font-sm" id="pro70272-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c39a/12375970/45f8ce14062e/PRO-34-e70272-g002.jpg" loading="lazy" id="jats-graphic-13" height="241" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/pro70272-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The cationic surface of the major and minor NLS‐binding groove of mImpα2. Surface renderings of the (a) major and (b) minor NLS‐binding grooves of mImpα2. The blue regions are the cationic surfaces. Cationic patches 1–3 are labeled in the minor site. The resolved crystal structure of A4 (pink) in the minor site is shown. The major groove cationic patch is composed of residues G72, T73, V74, Q98, R101, K102, and W142 and has a surface area of 70 Å<sup>2</sup>. The minor site has three cationic patches. Patch 1 consists of R238, N239, K240, N241, and Y277. Patch 2 consists of K348, T349, N350, and K353. Patch 3 consists of F391, K392, K395, W399, K435, Q438, and V439. Cationic surfaces were determined using MOE and images rendered in Pymol.</p></figcaption></figure><p>Although specific <em>K</em>
<sub>D</sub> values cannot be assigned to either the major or the minor NLS‐binding grooves, these findings demonstrate that the zero net‐charge analogs effectively bind to and stabilize mImpα2, with affinities indicating a preference for the minor groove. The crystal structures suggest that while TAF8 NLS binds exclusively to the major groove, the consistent <em>K</em>
<sub>D</sub> values suggest that it too can interact with both sites; this is sufficient to target the nucleus in the cell lines evaluated in this study.</p></section></section><section id="pro70272-sec-0009"><h2 class="pmc_sec_title">3. DISCUSSION</h2>
<p>In this study, novel NLS tags with zero net‐charge were developed with the aim to potentially improve the pharmacokinetics and reduce the non‐specific interactions associated with traditional NLS‐tagged therapeutics (Beaudoin et al. <a href="#pro70272-bib-0003" class="usa-link" aria-describedby="pro70272-bib-0003">2016</a>; Chen et al. <a href="#pro70272-bib-0009" class="usa-link" aria-describedby="pro70272-bib-0009"><sup>2006</sup></a>). Four zero net‐charge NLS analogs were successfully designed using structure‐guided computational modeling, achieving docking scores and binding modes at the major site of importin‐α comparable to TAF8 NLS. Crystal structures of these analogs complexed with mImpα2 confirmed their binding modes, albeit the novel interactions formed in the modeling procedure were disrupted by DTT used during crystallization. CD and MST experiments revealed the analogs bound mImpα2, albeit with reduced <em>K</em>
<sub>D</sub> values. Among these, analogs A3 and A4 demonstrated functional NLS activity in both C2C12 myoblasts and Hela cells and favored the minor groove. These results reveal the feasibility of engineering noncanonical zero net‐charge NLSs that retain importin‐α binding capacity and nuclear import function.</p>
<p>The positively charged amino acids that interact at the P2, P3, and P5 sites, whose side chains are deeply buried in negatively charged cavities in the major binding groove, are critical for TAF8 NLS binding. The binding of these NLS amino acids at these P sites of the major groove of importin‐α is foundational in classical nuclear transport. Currently, TAF8 NLS is considered a major groove‐specific binder, even though its canonical consensus sequence would also suggest binding at the minor groove. One can argue TAF8 NLS was too mobile to be modeled entirely at the minor groove in the original study (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>). In support of this perspective, the N‐terminus of TAF8 NLS dominates the crystal packing interactions with the C‐terminal arginine residues notably located outside the pocket and stabilized by water molecules with no interactions to importin‐α (Figure <a href="#pro70272-fig-0001" class="usa-link">1a,b</a>). TAF8 NLS would appear to bind even less at the minor groove considering C‐terminal arginine residues would interact right at the area of extensive cationic surfaces (Figure <a href="#pro70272-fig-0007" class="usa-link">7</a>).</p>
<p>However, our MST data provide evidence that TAF8 NLS binds both sites with affinities of 1 μM, which is 10‐fold stronger than previously reported (Trowitzsch et al. <a href="#pro70272-bib-0052" class="usa-link" aria-describedby="pro70272-bib-0052">2015</a>). This discrepancy may be due to methodological differences, where the previous study used ITC and where the binding of two sites with equivalent affinity could have been interpreted as binding to only one site.</p>
<p>The following question naturally arises—How can we reconcile the fact that more total (major + minor grooves) electron density translated into lower apparent binding affinities for the zero net‐charge analogs? Fitting a two‐binding site model resulted in <em>K</em>
<sub>D</sub> values ranging from <em>K</em>
<sub>D</sub>1 = 5 to 7 μM and <em>K</em>
<sub>D</sub>2 = 30 to 40 μM. Although it was not possible to assign which <em>K</em>
<sub>D</sub> value belongs to which binding site, due to general electrostatics principles, the highly positively charged TAF8 NLS may have a much higher association rate constant and/or lower dissociation rate constant (<em>k</em>
<sub>off</sub>) than the analogs at the major groove. But at the minor groove, our results suggest that the analogs with acidic amino acids at their C‐termini can bind better at the minor groove due to the extensive cationic surface, which provides more interaction opportunities, despite these residues not being resolved in the crystal structures and might provide them with at least lower <em>k</em>
<sub>off</sub> values. Therefore, this suggests that the <em>K</em>
<sub>D</sub>1 values reflect binding of the analogs at the minor groove (Table <a href="#pro70272-supitem-0001" class="usa-link">S2</a>).</p>
<p>Nuclear accumulation of A3 and A4 in cell‐based assays supports their classification as active NLSs (Figures <a href="#pro70272-fig-0006" class="usa-link">6</a> and <a href="#pro70272-supitem-0001" class="usa-link">S8</a>). These analogs, when fused to EGFP, localized sharply to the nucleus in both myeloblasts and Hela cell contexts. It is possible that a portion of the nuclear accumulation exhibited by the EGFP‐NLS constructs was due to passive diffusion due to the molecular weight of EGFP being under the threshold of the NPC (Timney et al. <a href="#pro70272-bib-0051" class="usa-link" aria-describedby="pro70272-bib-0051">2006</a>). Nevertheless, the clear differential localization relative to EGFP alone further confirms importin‐α‐mediated import.</p>
<p>From a therapeutic perspective, it is important to note that there are typically 10–20 NLSs per therapeutic antibodies (Beaudoin et al. <a href="#pro70272-bib-0003" class="usa-link" aria-describedby="pro70272-bib-0003">2016</a>; Chen et al. <a href="#pro70272-bib-0009" class="usa-link" aria-describedby="pro70272-bib-0009"><sup>2006</sup></a>), which allows multiple NLS sampling at the minor and major grooves, and the potential cumulative cooperativity effect would enable major and minor groove sampling redundancy and further enable effective binding and nuclear localization. It has been demonstrated that low‐affinity molecules targeting minor pockets of importin‐α can exert antiviral activity, where synthesized minor‐specific small‐molecule compounds exhibiting millimolar <em>K</em>
<sub>D</sub> blocked virus nuclear localization (Nakada et al. <a href="#pro70272-bib-0043" class="usa-link" aria-describedby="pro70272-bib-0043">2015</a>). The zero net‐charge analogs displayed low micromolar affinities to, most likely, the minor groove and, thus, are very promising for the development of NLS therapeutics.</p>
<p>However, the relationship between affinity and nuclear import remains complex. The field of nuclear import faces a critical challenge in harmonizing NLS affinity measurements and their impact on <em>K</em>
<sub>D</sub> values with nuclear import efficiency. Small differences (3‐fold) in NLS/importin‐α affinities have been shown to significantly affect the formation of transport complexes (Timney et al. <a href="#pro70272-bib-0051" class="usa-link" aria-describedby="pro70272-bib-0051">2006</a>; Wirthmueller et al. <a href="#pro70272-bib-0055" class="usa-link" aria-describedby="pro70272-bib-0055"><sup>2013</sup></a>). Conversely, it has also been shown that efficient nuclear import can tolerate NLS mutations diminishing importin‐α binding affinity by up to 200‐fold (Alvisi and Jans <a href="#pro70272-bib-0001" class="usa-link" aria-describedby="pro70272-bib-0001">2014</a>; Harreman et al. <a href="#pro70272-bib-0023" class="usa-link" aria-describedby="pro70272-bib-0023"><sup>2004</sup></a>; Hodel et al. <a href="#pro70272-bib-0025" class="usa-link" aria-describedby="pro70272-bib-0025"><sup>2001</sup></a>). The SV40 NLS, which is a foundational NLS for understanding binding principles (Fontes et al. <a href="#pro70272-bib-0017" class="usa-link" aria-describedby="pro70272-bib-0017">2000</a>) and widely utilized in NLS‐therapeutic design (Goswami et al. <a href="#pro70272-bib-0020" class="usa-link" aria-describedby="pro70272-bib-0020">2024</a>), has a perplexing disparity (1500‐fold difference) in the reported <em>K</em>
<sub>D</sub> values (10 nM–15 μM) measured by ELISA, ITC, fluorescence depolarization, and surface plasmon resonance (Catimel et al. <a href="#pro70272-bib-0006" class="usa-link" aria-describedby="pro70272-bib-0006">2001</a>; Fanara et al. <a href="#pro70272-bib-0015" class="usa-link" aria-describedby="pro70272-bib-0015"><sup>2000</sup></a>; Fontes et al. <a href="#pro70272-bib-0018" class="usa-link" aria-describedby="pro70272-bib-0018"><sup>2003</sup></a>; Sankhala et al. <a href="#pro70272-bib-0046" class="usa-link" aria-describedby="pro70272-bib-0046"><sup>2016</sup></a>). In this study, the isotherm fits were of good statistical quality as the MST data had low standards of deviations between replicates. Additionally, MST was able to process multiple samples near‐concurrently in a short time.</p>
<p>Another important consideration is the role of the IBB. The IBB domain exhibits micromolar affinity for importin‐α (<em>K</em>
<sub>D</sub> ≈ 14 ± 5 μM), which varies between importin‐α members and nanomolar affinity for importin‐β (<em>K</em>
<sub>D</sub> ≈ 10 nM) (Harreman et al. <a href="#pro70272-bib-0022" class="usa-link" aria-describedby="pro70272-bib-0022">2003</a>). The presence of importin‐β enhances binding affinity for NLS by ~30‐fold (Harreman et al. <a href="#pro70272-bib-0022" class="usa-link" aria-describedby="pro70272-bib-0022">2003</a>).</p>
<p>A growing body of work is linking NLS charge with biological outcomes. Although it is currently unknown how natural eukaryotic NLS‐containing proteins determine their rates and levels of nuclear localization with respect to charge, there is evidence from viral pathogenicity. For example, the NLS charge is associated with coronaviruses pathogenicity. Coronaviruses with highly cationic classical NLSs on their nucleocapsid proteins, including human SARS Coronavirus 2, ranked highest in pathogenicity and transmission compared to coronaviruses with NLSs with less cationic charge (Gussow et al. <a href="#pro70272-bib-0021" class="usa-link" aria-describedby="pro70272-bib-0021">2020</a>).</p>
<p>In conclusion, this study analyzed the structural and energetic landscape of the NLS of human TAF8, shedding deeper insight on its binding to importin‐α. We used these insights to design zero net‐charge analogs and showed their ability to effectively target mImpα2 with a preference for the minor groove. These findings support a revised model of NLS design that emphasizes groove selectivity, charge modulation, and tunable affinity. Future studies will explore integrating zero net‐charge NLSs into antibody‐based therapeutic platforms.</p></section><section id="pro70272-sec-0010"><h2 class="pmc_sec_title">4. MATERIALS AND METHODS</h2>
<section id="pro70272-sec-0011"><h3 class="pmc_sec_title">4.1. In silico TAF8 NLS‐based analog design and docking</h3>
<p>The Protein Data Bank (PDB) file 4WV6 for the TAF8 NLS complexed with human importin‐α1 was loaded into Molecular Operating Environment (MOE; Chemical Computing Group, Montreal, Canada). Individual residues underwent alanine substitution followed by docking to generate 100 docking poses and associated dAffinity scores (kcal/mol). The distribution of differences in dAffinity scores between the natural NLS and the derivatives containing alanine substitutions at different positions was determined. If the distributions followed the normal law, the Welch <em>t</em> test was utilized to determine if there was significance (<em>p</em> ≤ 0.05) between the means. In non‐normality scenarios, the Mann–Whitney <em>U</em> and Hodges‐Lehmann tests were also utilized (Lumley et al. <a href="#pro70272-bib-0035" class="usa-link" aria-describedby="pro70272-bib-0035">2002</a>; Vaux <a href="#pro70272-bib-0054" class="usa-link" aria-describedby="pro70272-bib-0054"><sup>2014</sup></a>). As a result, only amino acids that were not sensitive during alanine scanning were pursued for substitution with different amino acids, with the goal of generating sequences with net zero charge. Residue scanning was then performed on the non‐sensitive amino acids for the given NLSs, and docking and statistical analysis were performed as previously described.</p>
<p>It was crucial for zero net‐charge analogs to match the binding mode of the natural TAF8 NLS. Therefore, prior to testing different zero net‐charge analogs, the docking procedure had to ensure that it would not cause non‐natural binding modes. Docking of the in silico designed natural TAF8 NLS was performed. The Cα's of the in silico TAF8 NLS superposed very well to its NLS counterpart crystal structure. The NLS analogs were then docked and the best candidates with comparable docking scores to TAF8 NLS were selected for advancement.</p></section><section id="pro70272-sec-0012"><h3 class="pmc_sec_title">4.2. Analog synthesis</h3>
<p>TAF8 NLS and analogs were synthesized at the Peptide and Imaging Probe Synthesis Platform (Université de Sherbrooke) based on a similar method (Beaudoin et al. <a href="#pro70272-bib-0003" class="usa-link" aria-describedby="pro70272-bib-0003">2016</a>); further details are described in Data <a href="#pro70272-supitem-0001" class="usa-link">S1</a> and their characteristics are reported in Table <a href="#pro70272-supitem-0001" class="usa-link">S3</a>.</p></section><section id="pro70272-sec-0013"><h3 class="pmc_sec_title">4.3. 
DNA constructs, protein production and purification</h3>
<p>For biophysical investigations, the pET30a vector plasmid containing mImpα2 cDNA, lacking the importin‐β‐binding domain, and featuring the E396R mutation, was kindly provided by Professor Bostjan Kobe (The University of Queensland, Australia). Site‐directed mutagenesis was applied to revert the deletion to its native sequence. Transformation, expression, and purification details are described in Data <a href="#pro70272-supitem-0001" class="usa-link">S1</a>.</p></section><section id="pro70272-sec-0014"><h3 class="pmc_sec_title">4.4. Crystallization and structure determination</h3>
<p>MImpα2, residues 70–529, was co‐crystallized with each NLS analog using the sitting drop vapor diffusion method at 20°C. Briefly, 1 μL of protein (18.3 mg/mL) was mixed with 1 μL of reservoir solution and 0.7 μL of each peptide (3 mg/mL) and suspended over 0.5 mL of reservoir solution, resulting in a molar ratio of 5.2:1 for each analog over mImpα2 protein (Fontes et al. <a href="#pro70272-bib-0017" class="usa-link" aria-describedby="pro70272-bib-0017">2000</a>). Crystals were grown for 4 weeks and were harvested with the addition of 1 μL cryoprotectant (0.6M sodium citrate, 100 mM HEPES pH 6.4 and 20% glycerol) and flash‐frozen in liquid nitrogen. Detailed crystallization conditions for each crystal are listed in Table <a href="#pro70272-tbl-0003" class="usa-link">3</a> and details on the collection of diffraction datasets and structure determination can be found in Data <a href="#pro70272-supitem-0001" class="usa-link">S1</a>.</p></section><section id="pro70272-sec-0015"><h3 class="pmc_sec_title">4.5. Apparent NLS–mImpα2 complex stability monitored by CD
</h3>
<p>CD measurements were conducted as previously described, including the determination of free energy using the Van't Hoff method (Jean‐Francois et al. <a href="#pro70272-bib-0028" class="usa-link" aria-describedby="pro70272-bib-0028"><sup>2003</sup></a>). Fitting details are in Data <a href="#pro70272-supitem-0001" class="usa-link">S1</a>.</p></section><section id="pro70272-sec-0016"><h3 class="pmc_sec_title">4.6. Microscale thermophoresis</h3>
<p>Microscale thermophoresis (MST) experiments were conducted using the Monolith NT.115 (Nanotemper, Germany). The RED‐NHS 2nd generation Nanotemper labeling kit (MO‐L011) was used to label mImpα2 in 20 mM Tris–HCl and 127 mM NaCl at pH 8.0. Ten microliter of RED‐NHS‐reacted mImpα2 at 60 nM was titrated against a serial 1:1 dilution (250 μM–7.6 nM) of 10 μL of NLS containing 0.5% Tween‐20, and the mixture was pipetted into 16 MST capillaries contained in a single cartridge. The samples were processed using default settings and followed the manufacturer's protocol. The binding model fits are detailed in Data <a href="#pro70272-supitem-0001" class="usa-link">S1</a>.</p></section><section id="pro70272-sec-0017"><h3 class="pmc_sec_title">4.7. Fluorescence microscopy</h3>
<p>The plasmids, cloning, and cell culture transfection methods are described in Data <a href="#pro70272-supitem-0001" class="usa-link">S1</a>. Cells were fixed with 3% paraformaldehyde (Electron Microscopy Sciences, Cat#15710) for 15 min at room temperature; then rinsed thoroughly with PBS. Nuclear staining was performed using 1 μg/mL DAPI (Sigma, Cat#D9542) in PBS. Following staining, cells were rinsed again with PBS and mounted with glass coverslips using Vectashield Antifade Mounting Medium (Vector Laboratories, Cat#H‐1000).</p>
<p>Fluorescence imaging was performed using a Zeiss AxioImager.M2 microscope (RRID:SCR_024706) equipped with a 40× EC Plan‐NeoFluar (1.3 NA; oil) objective, X‐Cite XYLIS LED illumination system, and a Zeiss AxioCam mRm CCD monochrome camera. Image acquisition and initial processing were performed using ZEN Blue software (RRID:SCR_013672), and additional analyses were performed using Fiji (ImageJ; RRID:SCR_002285). For each experimental condition, 15 randomly selected fields of view were imaged. Line overlay fluorescence intensity was quantified using the line profile tool in Fiji. A straight line across the entire cell length was drawn and centered on the nucleus. The “Plot Profile” function was used to extract pixel intensity values along the line. Three cells were analyzed per condition, and the data were presented as mean fluorescence intensity values.</p></section><section id="pro70272-sec-0018"><h3 class="pmc_sec_title">4.8. Coordinates</h3>
<p>Structures and maps have been deposited in the PDB (9CL8, 9CLE, 9CLF, and 9CLG).</p></section></section><section id="pro70272-sec-0022"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Amirabbas Abdoli:</strong> Investigation; methodology; writing – review and editing; formal analysis. <strong>Zhihan Yang:</strong> Investigation; methodology; writing – review and editing; formal analysis. <strong>Abdullah Odeh‐Ahmed:</strong> Methodology. <strong>Olga Bednova:</strong> Methodology. <strong>Bruno Lemieux:</strong> Methodology. <strong>Leanne Dawe:</strong> Methodology; visualization. <strong>Aymeric Ravel‐Chapuis:</strong> Methodology; writing – review and editing; visualization; formal analysis. <strong>Pierre Lavigne:</strong> Conceptualization; writing – review and editing; supervision; formal analysis; resources. <strong>Natalie Zeytuni:</strong> Conceptualization; funding acquisition; writing – review and editing; formal analysis; supervision; resources; visualization; software. <strong>Jeffrey V. Leyton:</strong> Conceptualization; funding acquisition; writing – original draft; visualization; writing – review and editing; formal analysis; project administration; supervision; software.</p></section><section id="sec20"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70272-supitem-0001"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375970/bin/PRO-34-e70272-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70272-s001.docx</a><sup> (19.9MB, docx) </sup>
</div></div></section></section><section id="pro70272-sec-0019" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank the CLS (University of Saskatchewan) beamline CMCF‐ID and staff for remote diffraction data collection. The CLS is supported by the Canada Foundation for Innovation, the Natural Sciences and Engineering Research Council, the National Research Council, the Canadian Institutes of Health Research (CIHR), the Government of Saskatchewan, and the University of Saskatchewan. Imaging was performed at the University of Ottawa Cell Biology and Image Acquisition Core Facility (RRID:SCR_021845), and DNA sequencing was performed by the StemCore Laboratories Core Facility at the Ottawa Hospital Research Institute (RRID:SCR_012601). This work was supported by the Fonds de Recherche du Québec‐Santé (NZ), Centre de Recherche en Biologie Structurale Blue sky program (NZ and JVL), and by Canadian Institutes of Health Research (JVL). A. Rioux‐Chevalier for help using MOE. Marc‐André Bonin at the Université de Sherbrooke Peptide and Imaging Probe Synthesis Platform. AA was a recipient of a graduate student scholarship from the Université de Sherbrooke.</p></section><section id="notes1"><p>

Abdoli A, Yang Z, Odeh‐Ahmed A, Bednova O, Lemieux B, Dawe L, et al. Probing the extent of importin‐α targeting of the TAF8 NLS by eliminating its cationic net‐charge. Protein Science. 2025;34(9):e70272. 10.1002/pro.70272

</p>
<section id="pro70272-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="pro70272-note-0001"><p>Amirabbas Abdoli and Zhihan Yang contributed equally to this study.</p></div>
<div class="fn p" id="pro70272-note-0002"><p>
<strong>Review Editor:</strong> Aitziber L. Cortajarena</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Natalie Zeytuni, Email: natalie.zeytuni@mcgill.ca.</p>
<p>Jeffrey V. Leyton, Email: vleyton@uottawa.ca.</p></section><section id="pro70272-sec-0021"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that support the findings of this study are available on request from the corresponding author. The data are not publicly available due to privacy or ethical restrictions.</p></section><section id="pro70272-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70272-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70272-bib-0001">
<cite>
Alvisi G, Jans DA. Comment on phosphorylation adjacent to the nuclear localization signal of human dUTPase abolishes nuclear import: structural and mechanistic insights by Rona et al. (2013). Acta Crystallogr D Biol Crystallogr. 2014;70(Pt 10):2775–2776.
</cite> [<a href="https://doi.org/10.1107/S1399004714007081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25286860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Comment%20on%20phosphorylation%20adjacent%20to%20the%20nuclear%20localization%20signal%20of%20human%20dUTPase%20abolishes%20nuclear%20import:%20structural%20and%20mechanistic%20insights%20by%20Rona%20et%20al.%20(2013)&amp;author=G%20Alvisi&amp;author=DA%20Jans&amp;volume=70&amp;issue=Pt%2010&amp;publication_year=2014&amp;pages=2775-2776&amp;pmid=25286860&amp;doi=10.1107/S1399004714007081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0002">
<cite>
Bavelaar BM, Lee BQ, Gill MR, Falzone N, Vallis KA. Subcellular targeting of theranostic radionuclides. Front Pharmacol. 2018;9:996.
</cite> [<a href="https://doi.org/10.3389/fphar.2018.00996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6131480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30233374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&amp;title=Subcellular%20targeting%20of%20theranostic%20radionuclides&amp;author=BM%20Bavelaar&amp;author=BQ%20Lee&amp;author=MR%20Gill&amp;author=N%20Falzone&amp;author=KA%20Vallis&amp;volume=9&amp;publication_year=2018&amp;pages=996&amp;pmid=30233374&amp;doi=10.3389/fphar.2018.00996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0003">
<cite>
Beaudoin S, Rondeau A, Martel O, Bonin MA, van Lier JE, Leyton JV. ChAcNLS, a novel modification to antibody‐conjugates permitting target cell‐specific endosomal escape, localization to the nucleus, and enhanced total intracellular accumulation. Mol Pharm. 2016;13(6):1915–1926.
</cite> [<a href="https://doi.org/10.1021/acs.molpharmaceut.6b00075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27112376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Pharm&amp;title=ChAcNLS,%20a%20novel%20modification%20to%20antibody%E2%80%90conjugates%20permitting%20target%20cell%E2%80%90specific%20endosomal%20escape,%20localization%20to%20the%20nucleus,%20and%20enhanced%20total%20intracellular%20accumulation&amp;author=S%20Beaudoin&amp;author=A%20Rondeau&amp;author=O%20Martel&amp;author=MA%20Bonin&amp;author=JE%20van%20Lier&amp;volume=13&amp;issue=6&amp;publication_year=2016&amp;pages=1915-1926&amp;pmid=27112376&amp;doi=10.1021/acs.molpharmaceut.6b00075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0004">
<cite>
Bjornsti MA, Kaufmann SH. Topoisomerases and cancer chemotherapy: recent advances and unanswered questions. F1000Res. 2019;8:F1000 Faculty Rev‐1704.</cite> [<a href="https://doi.org/10.12688/f1000research.20201.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6774054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31602296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Res&amp;title=Topoisomerases%20and%20cancer%20chemotherapy:%20recent%20advances%20and%20unanswered%20questions&amp;author=MA%20Bjornsti&amp;author=SH%20Kaufmann&amp;volume=8&amp;publication_year=2019&amp;pages=F1000%20Faculty%20Rev%E2%80%901704&amp;pmid=31602296&amp;doi=10.12688/f1000research.20201.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0005">
<cite>
Canavese M, Santo L, Raje N. Cyclin dependent kinases in cancer: potential for therapeutic intervention. Cancer Biol Ther. 2012;13(7):451–457.
</cite> [<a href="https://doi.org/10.4161/cbt.19589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22361734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&amp;title=Cyclin%20dependent%20kinases%20in%20cancer:%20potential%20for%20therapeutic%20intervention&amp;author=M%20Canavese&amp;author=L%20Santo&amp;author=N%20Raje&amp;volume=13&amp;issue=7&amp;publication_year=2012&amp;pages=451-457&amp;pmid=22361734&amp;doi=10.4161/cbt.19589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0006">
<cite>
Catimel B, Teh T, Fontes MR, Jennings IG, Jans DA, Howlett GJ, et al. Biophysical characterization of interactions involving importin‐alpha during nuclear import. J Biol Chem. 2001;276(36):34189–34198.
</cite> [<a href="https://doi.org/10.1074/jbc.M103531200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11448961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Biophysical%20characterization%20of%20interactions%20involving%20importin%E2%80%90alpha%20during%20nuclear%20import&amp;author=B%20Catimel&amp;author=T%20Teh&amp;author=MR%20Fontes&amp;author=IG%20Jennings&amp;author=DA%20Jans&amp;volume=276&amp;issue=36&amp;publication_year=2001&amp;pages=34189-34198&amp;pmid=11448961&amp;doi=10.1074/jbc.M103531200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0007">
<cite>
Chang CW, Couñago RM, Williams SJ, Bodén M, Kobe B. Distinctive conformation of minor site‐specific nuclear localization signals bound to importin‐alpha. Traffic. 2013;14(11):1144–1154.
</cite> [<a href="https://doi.org/10.1111/tra.12098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23910026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Distinctive%20conformation%20of%20minor%20site%E2%80%90specific%20nuclear%20localization%20signals%20bound%20to%20importin%E2%80%90alpha&amp;author=CW%20Chang&amp;author=RM%20Cou%C3%B1ago&amp;author=SJ%20Williams&amp;author=M%20Bod%C3%A9n&amp;author=B%20Kobe&amp;volume=14&amp;issue=11&amp;publication_year=2013&amp;pages=1144-1154&amp;pmid=23910026&amp;doi=10.1111/tra.12098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0008">
<cite>
Chelsky D, Ralph R, Jonak G. Sequence requirements for synthetic peptide‐mediated translocation to the nucleus. Mol Cell Biol. 1989;9(6):2487–2492.
</cite> [<a href="https://doi.org/10.1128/mcb.9.6.2487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC362321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2668735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&amp;title=Sequence%20requirements%20for%20synthetic%20peptide%E2%80%90mediated%20translocation%20to%20the%20nucleus&amp;author=D%20Chelsky&amp;author=R%20Ralph&amp;author=G%20Jonak&amp;volume=9&amp;issue=6&amp;publication_year=1989&amp;pages=2487-2492&amp;pmid=2668735&amp;doi=10.1128/mcb.9.6.2487&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0009">
<cite>
Chen P, Wang J, Hope K, Jin L, Dick J, Cameron R, et al. Nuclear localizing sequences promote nuclear translocation and enhance the radiotoxicity of the anti‐CD33 monoclonal antibody HuM195 labeled with 111In in human myeloid leukemia cells. J Nucl Med. 2006;47(5):827–836.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/16644753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Nucl%20Med&amp;title=Nuclear%20localizing%20sequences%20promote%20nuclear%20translocation%20and%20enhance%20the%20radiotoxicity%20of%20the%20anti%E2%80%90CD33%20monoclonal%20antibody%20HuM195%20labeled%20with%20111In%20in%20human%20myeloid%20leukemia%20cells&amp;author=P%20Chen&amp;author=J%20Wang&amp;author=K%20Hope&amp;author=L%20Jin&amp;author=J%20Dick&amp;volume=47&amp;issue=5&amp;publication_year=2006&amp;pages=827-836&amp;pmid=16644753&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0010">
<cite>
Cingolani G, Petosa C, Weis K, Müller CW. Structure of importin‐beta bound to the IBB domain of importin‐alpha. Nature. 1999;399(6733):221–229.
</cite> [<a href="https://doi.org/10.1038/20367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10353244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20importin%E2%80%90beta%20bound%20to%20the%20IBB%20domain%20of%20importin%E2%80%90alpha&amp;author=G%20Cingolani&amp;author=C%20Petosa&amp;author=K%20Weis&amp;author=CW%20M%C3%BCller&amp;volume=399&amp;issue=6733&amp;publication_year=1999&amp;pages=221-229&amp;pmid=10353244&amp;doi=10.1038/20367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0011">
<cite>
Conti E, Kuriyan J. Crystallographic analysis of the specific yet versatile recognition of distinct nuclear localization signals by karyopherin alpha. Structure. 2000;8(3):329–338.
</cite> [<a href="https://doi.org/10.1016/s0969-2126(00)00107-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10745017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Crystallographic%20analysis%20of%20the%20specific%20yet%20versatile%20recognition%20of%20distinct%20nuclear%20localization%20signals%20by%20karyopherin%20alpha&amp;author=E%20Conti&amp;author=J%20Kuriyan&amp;volume=8&amp;issue=3&amp;publication_year=2000&amp;pages=329-338&amp;pmid=10745017&amp;doi=10.1016/s0969-2126(00)00107-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0012">
<cite>
Datta‐Mannan A, Thangaraju A, Leung D, Tang Y, Witcher DR, Lu J, et al. Balancing charge in the complementarity‐determining regions of humanized mAbs without affecting pI reduces non‐specific binding and improves the pharmacokinetics. MAbs. 2015;7(3):483–493.
</cite> [<a href="https://doi.org/10.1080/19420862.2015.1016696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4622971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25695748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MAbs&amp;title=Balancing%20charge%20in%20the%20complementarity%E2%80%90determining%20regions%20of%20humanized%20mAbs%20without%20affecting%20pI%20reduces%20non%E2%80%90specific%20binding%20and%20improves%20the%20pharmacokinetics&amp;author=A%20Datta%E2%80%90Mannan&amp;author=A%20Thangaraju&amp;author=D%20Leung&amp;author=Y%20Tang&amp;author=DR%20Witcher&amp;volume=7&amp;issue=3&amp;publication_year=2015&amp;pages=483-493&amp;pmid=25695748&amp;doi=10.1080/19420862.2015.1016696&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0013">
<cite>
de Oliveira HC, da Silva TD, Salvador GHM, Moraes IR, Fukuda CA, de Barros AC, et al. Structural and calorimetric studies reveal specific determinants for the binding of a high‐affinity NLS to mammalian importin‐alpha. Biochem J. 2021;478(13):2715–2732.
</cite> [<a href="https://doi.org/10.1042/BCJ20210401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34195786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Structural%20and%20calorimetric%20studies%20reveal%20specific%20determinants%20for%20the%20binding%20of%20a%20high%E2%80%90affinity%20NLS%20to%20mammalian%20importin%E2%80%90alpha&amp;author=HC%20de%20Oliveira&amp;author=TD%20da%20Silva&amp;author=GHM%20Salvador&amp;author=IR%20Moraes&amp;author=CA%20Fukuda&amp;volume=478&amp;issue=13&amp;publication_year=2021&amp;pages=2715-2732&amp;pmid=34195786&amp;doi=10.1042/BCJ20210401&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0014">
<cite>
Delfing BM, Laracuente XE, Olson A, Foreman KW, Paige M, Kehn‐Hall K, et al. Binding of viral nuclear localization signal peptides to importin‐alpha nuclear transport protein. Biophys J. 2023;122(17):3476–3488.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2023.07.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10502480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37542371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Binding%20of%20viral%20nuclear%20localization%20signal%20peptides%20to%20importin%E2%80%90alpha%20nuclear%20transport%20protein&amp;author=BM%20Delfing&amp;author=XE%20Laracuente&amp;author=A%20Olson&amp;author=KW%20Foreman&amp;author=M%20Paige&amp;volume=122&amp;issue=17&amp;publication_year=2023&amp;pages=3476-3488&amp;pmid=37542371&amp;doi=10.1016/j.bpj.2023.07.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0015">
<cite>
Fanara P, Hodel MR, Corbett AH, Hodel AE. Quantitative analysis of nuclear localization signal (NLS)‐importin alpha interaction through fluorescence depolarization. Evidence for auto‐inhibitory regulation of NLS binding. J Biol Chem. 2000;275(28):21218–21223.
</cite> [<a href="https://doi.org/10.1074/jbc.M002217200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10806202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Quantitative%20analysis%20of%20nuclear%20localization%20signal%20(NLS)%E2%80%90importin%20alpha%20interaction%20through%20fluorescence%20depolarization.%20Evidence%20for%20auto%E2%80%90inhibitory%20regulation%20of%20NLS%20binding&amp;author=P%20Fanara&amp;author=MR%20Hodel&amp;author=AH%20Corbett&amp;author=AE%20Hodel&amp;volume=275&amp;issue=28&amp;publication_year=2000&amp;pages=21218-21223&amp;pmid=10806202&amp;doi=10.1074/jbc.M002217200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0016">
<cite>
Fasih A, Fonge H, Cai Z, Leyton JV, Tikhomirov I, Done SJ, et al. In‐Bn‐DTPA‐nimotuzumab with/without modification with nuclear translocation sequence (NLS) peptides: an Auger electron‐emitting radioimmunotherapeutic agent for EGFR‐positive and trastuzumab (Herceptin)‐resistant breast cancer. Breast Cancer Res Treat. 2012;135(1):189–200.
</cite> [<a href="https://doi.org/10.1007/s10549-012-2137-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22736376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&amp;title=In%E2%80%90Bn%E2%80%90DTPA%E2%80%90nimotuzumab%20with/without%20modification%20with%20nuclear%20translocation%20sequence%20(NLS)%20peptides:%20an%20Auger%20electron%E2%80%90emitting%20radioimmunotherapeutic%20agent%20for%20EGFR%E2%80%90positive%20and%20trastuzumab%20(Herceptin)%E2%80%90resistant%20breast%20cancer&amp;author=A%20Fasih&amp;author=H%20Fonge&amp;author=Z%20Cai&amp;author=JV%20Leyton&amp;author=I%20Tikhomirov&amp;volume=135&amp;issue=1&amp;publication_year=2012&amp;pages=189-200&amp;pmid=22736376&amp;doi=10.1007/s10549-012-2137-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0017">
<cite>
Fontes MR, Teh T, Kobe B. Structural basis of recognition of monopartite and bipartite nuclear localization sequences by mammalian importin‐alpha. J Mol Biol. 2000;297(5):1183–1194.
</cite> [<a href="https://doi.org/10.1006/jmbi.2000.3642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10764582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Structural%20basis%20of%20recognition%20of%20monopartite%20and%20bipartite%20nuclear%20localization%20sequences%20by%20mammalian%20importin%E2%80%90alpha&amp;author=MR%20Fontes&amp;author=T%20Teh&amp;author=B%20Kobe&amp;volume=297&amp;issue=5&amp;publication_year=2000&amp;pages=1183-1194&amp;pmid=10764582&amp;doi=10.1006/jmbi.2000.3642&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0018">
<cite>
Fontes MR, Teh T, Toth G, John A, Pavo I, Jans DA, et al. Role of flanking sequences and phosphorylation in the recognition of the simian‐virus‐40 large T‐antigen nuclear localization sequences by importin‐alpha. Biochem J. 2003;375(Pt 2):339–349.
</cite> [<a href="https://doi.org/10.1042/BJ20030510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1223685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12852786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Role%20of%20flanking%20sequences%20and%20phosphorylation%20in%20the%20recognition%20of%20the%20simian%E2%80%90virus%E2%80%9040%20large%20T%E2%80%90antigen%20nuclear%20localization%20sequences%20by%20importin%E2%80%90alpha&amp;author=MR%20Fontes&amp;author=T%20Teh&amp;author=G%20Toth&amp;author=A%20John&amp;author=I%20Pavo&amp;volume=375&amp;issue=Pt%202&amp;publication_year=2003&amp;pages=339-349&amp;pmid=12852786&amp;doi=10.1042/BJ20030510&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0019">
<cite>
Fournier D, Palidwor GA, Shcherbinin S, Szengel A, Schaefer MH, Perez‐Iratxeta C, et al. Functional and genomic analyses of alpha‐solenoid proteins. PLoS One. 2013;8(11):e79894.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0079894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3837014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24278209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Functional%20and%20genomic%20analyses%20of%20alpha%E2%80%90solenoid%20proteins&amp;author=D%20Fournier&amp;author=GA%20Palidwor&amp;author=S%20Shcherbinin&amp;author=A%20Szengel&amp;author=MH%20Schaefer&amp;volume=8&amp;issue=11&amp;publication_year=2013&amp;pages=e79894&amp;pmid=24278209&amp;doi=10.1371/journal.pone.0079894&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0020">
<cite>
Goswami R, Gupta A, Bednova O, Coulombe G, Patel D, Rotello VM, et al. Nuclear localization signal‐tagged systems: relevant nuclear import principles in the context of current therapeutic design. Chem Soc Rev. 2024;53(1):204–226.
</cite> [<a href="https://doi.org/10.1039/d1cs00269d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10798298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38031452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Soc%20Rev&amp;title=Nuclear%20localization%20signal%E2%80%90tagged%20systems:%20relevant%20nuclear%20import%20principles%20in%20the%20context%20of%20current%20therapeutic%20design&amp;author=R%20Goswami&amp;author=A%20Gupta&amp;author=O%20Bednova&amp;author=G%20Coulombe&amp;author=D%20Patel&amp;volume=53&amp;issue=1&amp;publication_year=2024&amp;pages=204-226&amp;pmid=38031452&amp;doi=10.1039/d1cs00269d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0021">
<cite>
Gussow AB, Auslander N, Faure G, Wolf YI, Zhang F, Koonin EV. Genomic determinants of pathogenicity in SARS‐CoV‐2 and other human coronaviruses. Proc Natl Acad Sci U S A. 2020;117(26):15193–15199.
</cite> [<a href="https://doi.org/10.1073/pnas.2008176117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32522874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Genomic%20determinants%20of%20pathogenicity%20in%20SARS%E2%80%90CoV%E2%80%902%20and%20other%20human%20coronaviruses&amp;author=AB%20Gussow&amp;author=N%20Auslander&amp;author=G%20Faure&amp;author=YI%20Wolf&amp;author=F%20Zhang&amp;volume=117&amp;issue=26&amp;publication_year=2020&amp;pages=15193-15199&amp;pmid=32522874&amp;doi=10.1073/pnas.2008176117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0022">
<cite>
Harreman MT, Hodel MR, Fanara P, Hodel AE, Corbett AH. The auto‐inhibitory function of importin alpha is essential in vivo. J Biol Chem. 2003;278(8):5854–5863.
</cite> [<a href="https://doi.org/10.1074/jbc.M210951200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12486120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=The%20auto%E2%80%90inhibitory%20function%20of%20importin%20alpha%20is%20essential%20in%20vivo&amp;author=MT%20Harreman&amp;author=MR%20Hodel&amp;author=P%20Fanara&amp;author=AE%20Hodel&amp;author=AH%20Corbett&amp;volume=278&amp;issue=8&amp;publication_year=2003&amp;pages=5854-5863&amp;pmid=12486120&amp;doi=10.1074/jbc.M210951200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0023">
<cite>
Harreman MT, Kline TM, Milford HG, Harben MB, Hodel AE, Corbett AH. Regulation of nuclear import by phosphorylation adjacent to nuclear localization signals. J Biol Chem. 2004;279(20):20613–20621.
</cite> [<a href="https://doi.org/10.1074/jbc.M401720200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14998990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Regulation%20of%20nuclear%20import%20by%20phosphorylation%20adjacent%20to%20nuclear%20localization%20signals&amp;author=MT%20Harreman&amp;author=TM%20Kline&amp;author=HG%20Milford&amp;author=MB%20Harben&amp;author=AE%20Hodel&amp;volume=279&amp;issue=20&amp;publication_year=2004&amp;pages=20613-20621&amp;pmid=14998990&amp;doi=10.1074/jbc.M401720200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0024">
<cite>
Hinton PR, Xiong JM, Johlfs MG, Tang MT, Keller S, Tsurushita N. An engineered human IgG1 antibody with longer serum half‐life. J Immunol. 2006;176(1):346–356.
</cite> [<a href="https://doi.org/10.4049/jimmunol.176.1.346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16365427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunol&amp;title=An%20engineered%20human%20IgG1%20antibody%20with%20longer%20serum%20half%E2%80%90life&amp;author=PR%20Hinton&amp;author=JM%20Xiong&amp;author=MG%20Johlfs&amp;author=MT%20Tang&amp;author=S%20Keller&amp;volume=176&amp;issue=1&amp;publication_year=2006&amp;pages=346-356&amp;pmid=16365427&amp;doi=10.4049/jimmunol.176.1.346&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0025">
<cite>
Hodel MR, Corbett AH, Hodel AE. Dissection of a nuclear localization signal. J Biol Chem. 2001;276(2):1317–1325.
</cite> [<a href="https://doi.org/10.1074/jbc.M008522200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11038364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Dissection%20of%20a%20nuclear%20localization%20signal&amp;author=MR%20Hodel&amp;author=AH%20Corbett&amp;author=AE%20Hodel&amp;volume=276&amp;issue=2&amp;publication_year=2001&amp;pages=1317-1325&amp;pmid=11038364&amp;doi=10.1074/jbc.M008522200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0026">
<cite>
Hung MC, Link W. Protein localization in disease and therapy. J Cell Sci. 2011;124(Pt 20):3381–3392.
</cite> [<a href="https://doi.org/10.1242/jcs.089110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22010196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=Protein%20localization%20in%20disease%20and%20therapy&amp;author=MC%20Hung&amp;author=W%20Link&amp;volume=124&amp;issue=Pt%2020&amp;publication_year=2011&amp;pages=3381-3392&amp;pmid=22010196&amp;doi=10.1242/jcs.089110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0027">
<cite>
Igawa T, Tsunoda H, Tachibana T, Maeda A, Mimoto F, Moriyama C, et al. Reduced elimination of IgG antibodies by engineering the variable region. Protein Eng Des Sel. 2010;23(5):385–392.
</cite> [<a href="https://doi.org/10.1093/protein/gzq009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20159773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Eng%20Des%20Sel&amp;title=Reduced%20elimination%20of%20IgG%20antibodies%20by%20engineering%20the%20variable%20region&amp;author=T%20Igawa&amp;author=H%20Tsunoda&amp;author=T%20Tachibana&amp;author=A%20Maeda&amp;author=F%20Mimoto&amp;volume=23&amp;issue=5&amp;publication_year=2010&amp;pages=385-392&amp;pmid=20159773&amp;doi=10.1093/protein/gzq009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0028">
<cite>
Jean‐Francois N, Frédéric G, Raymund W, Benoit C, Lavigne P. Improving the thermodynamic stability of the leucine zipper of max increases the stability of its b‐HLH‐LZ:E‐box complex. J Mol Biol. 2003;326(5):1577–1595.
</cite> [<a href="https://doi.org/10.1016/s0022-2836(03)00029-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12595267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Improving%20the%20thermodynamic%20stability%20of%20the%20leucine%20zipper%20of%20max%20increases%20the%20stability%20of%20its%20b%E2%80%90HLH%E2%80%90LZ:E%E2%80%90box%20complex&amp;author=N%20Jean%E2%80%90Francois&amp;author=G%20Fr%C3%A9d%C3%A9ric&amp;author=W%20Raymund&amp;author=C%20Benoit&amp;author=P%20Lavigne&amp;volume=326&amp;issue=5&amp;publication_year=2003&amp;pages=1577-1595&amp;pmid=12595267&amp;doi=10.1016/s0022-2836(03)00029-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0029">
<cite>
Kalderon D, Richardson WD, Markham AF, Smith AE. Sequence requirements for nuclear location of simian virus 40 large‐T antigen. Nature. 1984;311(5981):33–38.
</cite> [<a href="https://doi.org/10.1038/311033a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6088992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Sequence%20requirements%20for%20nuclear%20location%20of%20simian%20virus%2040%20large%E2%80%90T%20antigen&amp;author=D%20Kalderon&amp;author=WD%20Richardson&amp;author=AF%20Markham&amp;author=AE%20Smith&amp;volume=311&amp;issue=5981&amp;publication_year=1984&amp;pages=33-38&amp;pmid=6088992&amp;doi=10.1038/311033a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0030">
<cite>
Kelley JB, Talley AM, Spencer A, Gioeli D, Paschal BM. Karyopherin alpha7 (KPNA7), a divergent member of the importin alpha family of nuclear import receptors. BMC Cell Biol. 2010;11:63.
</cite> [<a href="https://doi.org/10.1186/1471-2121-11-63" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2929220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20701745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Cell%20Biol&amp;title=Karyopherin%20alpha7%20(KPNA7),%20a%20divergent%20member%20of%20the%20importin%20alpha%20family%20of%20nuclear%20import%20receptors&amp;author=JB%20Kelley&amp;author=AM%20Talley&amp;author=A%20Spencer&amp;author=D%20Gioeli&amp;author=BM%20Paschal&amp;volume=11&amp;publication_year=2010&amp;pages=63&amp;pmid=20701745&amp;doi=10.1186/1471-2121-11-63&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0031">
<cite>
Kobe B. Autoinhibition by an internal nuclear localization signal revealed by the crystal structure of mammalian importin alpha. Nat Struct Biol. 1999;6(4):388–397.
</cite> [<a href="https://doi.org/10.1038/7625" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10201409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Biol&amp;title=Autoinhibition%20by%20an%20internal%20nuclear%20localization%20signal%20revealed%20by%20the%20crystal%20structure%20of%20mammalian%20importin%20alpha&amp;author=B%20Kobe&amp;volume=6&amp;issue=4&amp;publication_year=1999&amp;pages=388-397&amp;pmid=10201409&amp;doi=10.1038/7625&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0032">
<cite>
Leyton JV, Gao C, Williams B, Keating A, Minden M, Reilly RM. A radiolabeled antibody targeting CD123(+) leukemia stem cells—initial radioimmunotherapy studies in NOD/SCID mice engrafted with primary human AML. Leuk Res Rep. 2015;4(2):55–59.
</cite> [<a href="https://doi.org/10.1016/j.lrr.2015.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4588397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26500848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Leuk%20Res%20Rep&amp;title=A%20radiolabeled%20antibody%20targeting%20CD123(+)%20leukemia%20stem%20cells%E2%80%94initial%20radioimmunotherapy%20studies%20in%20NOD/SCID%20mice%20engrafted%20with%20primary%20human%20AML&amp;author=JV%20Leyton&amp;author=C%20Gao&amp;author=B%20Williams&amp;author=A%20Keating&amp;author=M%20Minden&amp;volume=4&amp;issue=2&amp;publication_year=2015&amp;pages=55-59&amp;pmid=26500848&amp;doi=10.1016/j.lrr.2015.07.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0033">
<cite>
Leyton JV, Hu M, Gao C, Turner PV, Dick JE, Minden M, et al. Auger electron radioimmunotherapeutic agent specific for the CD123+/CD131− phenotype of the leukemia stem cell population. J Nucl Med. 2011;52(9):1465–1473.
</cite> [<a href="https://doi.org/10.2967/jnumed.111.087668" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21816968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Nucl%20Med&amp;title=Auger%20electron%20radioimmunotherapeutic%20agent%20specific%20for%20the%20CD123+/CD131%E2%88%92%20phenotype%20of%20the%20leukemia%20stem%20cell%20population&amp;author=JV%20Leyton&amp;author=M%20Hu&amp;author=C%20Gao&amp;author=PV%20Turner&amp;author=JE%20Dick&amp;volume=52&amp;issue=9&amp;publication_year=2011&amp;pages=1465-1473&amp;pmid=21816968&amp;doi=10.2967/jnumed.111.087668&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0034">
<cite>
Li B, Tesar D, Boswell CA, Cahaya HS, Wong A, Zhang J, et al. Framework selection can influence pharmacokinetics of a humanized therapeutic antibody through differences in molecule charge. MAbs. 2014;6(5):1255–1264.
</cite> [<a href="https://doi.org/10.4161/mabs.29809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4623330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25517310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MAbs&amp;title=Framework%20selection%20can%20influence%20pharmacokinetics%20of%20a%20humanized%20therapeutic%20antibody%20through%20differences%20in%20molecule%20charge&amp;author=B%20Li&amp;author=D%20Tesar&amp;author=CA%20Boswell&amp;author=HS%20Cahaya&amp;author=A%20Wong&amp;volume=6&amp;issue=5&amp;publication_year=2014&amp;pages=1255-1264&amp;pmid=25517310&amp;doi=10.4161/mabs.29809&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0035">
<cite>
Lumley T, Diehr P, Emerson S, Chen L. The importance of the normality assumption in large public health data sets. Annu Rev Public Health. 2002;23:151–169.
</cite> [<a href="https://doi.org/10.1146/annurev.publhealth.23.100901.140546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11910059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Public%20Health&amp;title=The%20importance%20of%20the%20normality%20assumption%20in%20large%20public%20health%20data%20sets&amp;author=T%20Lumley&amp;author=P%20Diehr&amp;author=S%20Emerson&amp;author=L%20Chen&amp;volume=23&amp;publication_year=2002&amp;pages=151-169&amp;pmid=11910059&amp;doi=10.1146/annurev.publhealth.23.100901.140546&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0036">
<cite>
Marfori M, Lonhienne TG, Forwood J, Kobe B. Structural basis of high‐affinity nuclear localization signal interactions with importin‐alpha. Traffic. 2012;13(4):532–548.
</cite> [<a href="https://doi.org/10.1111/j.1600-0854.2012.01329.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22248489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Structural%20basis%20of%20high%E2%80%90affinity%20nuclear%20localization%20signal%20interactions%20with%20importin%E2%80%90alpha&amp;author=M%20Marfori&amp;author=TG%20Lonhienne&amp;author=J%20Forwood&amp;author=B%20Kobe&amp;volume=13&amp;issue=4&amp;publication_year=2012&amp;pages=532-548&amp;pmid=22248489&amp;doi=10.1111/j.1600-0854.2012.01329.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0037">
<cite>
Marfori M, Mynott A, Ellis JJ, Mehdi AM, Saunders NFW, Curmi PM, et al. Molecular basis for specificity of nuclear import and prediction of nuclear localization. Biochim Biophys Acta. 2011;1813(9):1562–1577.
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2010.10.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20977914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Molecular%20basis%20for%20specificity%20of%20nuclear%20import%20and%20prediction%20of%20nuclear%20localization&amp;author=M%20Marfori&amp;author=A%20Mynott&amp;author=JJ%20Ellis&amp;author=AM%20Mehdi&amp;author=NFW%20Saunders&amp;volume=1813&amp;issue=9&amp;publication_year=2011&amp;pages=1562-1577&amp;pmid=20977914&amp;doi=10.1016/j.bbamcr.2010.10.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0038">
<cite>
Martin RD, Hebert TE, Tanny JC. Therapeutic targeting of the general RNA polymerase II transcription machinery. Int J Mol Sci. 2020;21(9):3354.
</cite> [<a href="https://doi.org/10.3390/ijms21093354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7246882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32397434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Therapeutic%20targeting%20of%20the%20general%20RNA%20polymerase%20II%20transcription%20machinery&amp;author=RD%20Martin&amp;author=TE%20Hebert&amp;author=JC%20Tanny&amp;volume=21&amp;issue=9&amp;publication_year=2020&amp;pages=3354&amp;pmid=32397434&amp;doi=10.3390/ijms21093354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0039">
<cite>
Melchior F, Paschal B, Evans J, Gerace L. Inhibition of nuclear protein import by nonhydrolyzable analogues of GTP and identification of the small GTPase Ran/TC4 as an essential transport factor. J Cell Biol. 1993;123(6):1649–1659.
</cite> [<a href="https://doi.org/10.1083/jcb.123.6.1649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2290879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8276887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=Inhibition%20of%20nuclear%20protein%20import%20by%20nonhydrolyzable%20analogues%20of%20GTP%20and%20identification%20of%20the%20small%20GTPase%20Ran/TC4%20as%20an%20essential%20transport%20factor&amp;author=F%20Melchior&amp;author=B%20Paschal&amp;author=J%20Evans&amp;author=L%20Gerace&amp;volume=123&amp;issue=6&amp;publication_year=1993&amp;pages=1649-1659&amp;pmid=8276887&amp;doi=10.1083/jcb.123.6.1649&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0040">
<cite>
Moore MS, Blobel G. The GTP‐binding protein Ran/TC4 is required for protein import into the nucleus. Nature. 1993;365(6447):661–663.
</cite> [<a href="https://doi.org/10.1038/365661a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8413630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20GTP%E2%80%90binding%20protein%20Ran/TC4%20is%20required%20for%20protein%20import%20into%20the%20nucleus&amp;author=MS%20Moore&amp;author=G%20Blobel&amp;volume=365&amp;issue=6447&amp;publication_year=1993&amp;pages=661-663&amp;pmid=8413630&amp;doi=10.1038/365661a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0041">
<cite>
Mor A, White MA, Fontoura BM. Nuclear trafficking in health and disease. Curr Opin Cell Biol. 2014;28:28–35.
</cite> [<a href="https://doi.org/10.1016/j.ceb.2014.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4061247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24530809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Cell%20Biol&amp;title=Nuclear%20trafficking%20in%20health%20and%20disease&amp;author=A%20Mor&amp;author=MA%20White&amp;author=BM%20Fontoura&amp;volume=28&amp;publication_year=2014&amp;pages=28-35&amp;pmid=24530809&amp;doi=10.1016/j.ceb.2014.01.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0042">
<cite>
Munsell EV, Ross NL, Sullivan MO. Journey to the center of the cell: current nanocarrier design strategies targeting biopharmaceuticals to the cytoplasm and nucleus. Curr Pharm Des. 2016;22(9):1227–1244.
</cite> [<a href="https://doi.org/10.2174/1381612822666151216151420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4792758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26675220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&amp;title=Journey%20to%20the%20center%20of%20the%20cell:%20current%20nanocarrier%20design%20strategies%20targeting%20biopharmaceuticals%20to%20the%20cytoplasm%20and%20nucleus&amp;author=EV%20Munsell&amp;author=NL%20Ross&amp;author=MO%20Sullivan&amp;volume=22&amp;issue=9&amp;publication_year=2016&amp;pages=1227-1244&amp;pmid=26675220&amp;doi=10.2174/1381612822666151216151420&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0043">
<cite>
Nakada R, Hirano H, Matsuura Y. Structure of importin‐alpha bound to a non‐classical nuclear localization signal of the influenza A virus nucleoprotein. Sci Rep. 2015;5:15055.
</cite> [<a href="https://doi.org/10.1038/srep15055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4601014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26456934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Structure%20of%20importin%E2%80%90alpha%20bound%20to%20a%20non%E2%80%90classical%20nuclear%20localization%20signal%20of%20the%20influenza%20A%20virus%20nucleoprotein&amp;author=R%20Nakada&amp;author=H%20Hirano&amp;author=Y%20Matsuura&amp;volume=5&amp;publication_year=2015&amp;pages=15055&amp;pmid=26456934&amp;doi=10.1038/srep15055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0044">
<cite>
Paquette M, Beaudoin S, Tremblay M‐A, Jean S, Lopez AF, Lecomte R, et al. NLS‐cholic acid conjugation to IL‐5Ralpha‐specific antibody improves cellular accumulation and in vivo tumor‐targeting properties in a bladder cancer model. Bioconjug Chem. 2018;29(4):1352–1363.
</cite> [<a href="https://doi.org/10.1021/acs.bioconjchem.8b00077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29433309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconjug%20Chem&amp;title=NLS%E2%80%90cholic%20acid%20conjugation%20to%20IL%E2%80%905Ralpha%E2%80%90specific%20antibody%20improves%20cellular%20accumulation%20and%20in%20vivo%20tumor%E2%80%90targeting%20properties%20in%20a%20bladder%20cancer%20model&amp;author=M%20Paquette&amp;author=S%20Beaudoin&amp;author=M%E2%80%90A%20Tremblay&amp;author=S%20Jean&amp;author=AF%20Lopez&amp;volume=29&amp;issue=4&amp;publication_year=2018&amp;pages=1352-1363&amp;pmid=29433309&amp;doi=10.1021/acs.bioconjchem.8b00077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0045">
<cite>
Sadurni MMS. Deregulations of RNA Pol II subunits in cancer. Appl Biosci. 2023;2:459–476.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Biosci&amp;title=Deregulations%20of%20RNA%20Pol%20II%20subunits%20in%20cancer&amp;author=MMS%20Sadurni&amp;volume=2&amp;publication_year=2023&amp;pages=459-476&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0046">
<cite>
Sankhala RS, Lokareddy RK, Cingolani G. Divergent evolution of nuclear localization signal sequences in herpesvirus terminase subunits. J Biol Chem. 2016;291(21):11420–11433.
</cite> [<a href="https://doi.org/10.1074/jbc.M116.724393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4900285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27033706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Divergent%20evolution%20of%20nuclear%20localization%20signal%20sequences%20in%20herpesvirus%20terminase%20subunits&amp;author=RS%20Sankhala&amp;author=RK%20Lokareddy&amp;author=G%20Cingolani&amp;volume=291&amp;issue=21&amp;publication_year=2016&amp;pages=11420-11433&amp;pmid=27033706&amp;doi=10.1074/jbc.M116.724393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0047">
<cite>
Schwoebel ED, Ho TH, Moore MS. The mechanism of inhibition of Ran‐dependent nuclear transport by cellular ATP depletion. J Cell Biol. 2002;157(6):963–974.
</cite> [<a href="https://doi.org/10.1083/jcb.200111077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2174045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12058015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=The%20mechanism%20of%20inhibition%20of%20Ran%E2%80%90dependent%20nuclear%20transport%20by%20cellular%20ATP%20depletion&amp;author=ED%20Schwoebel&amp;author=TH%20Ho&amp;author=MS%20Moore&amp;volume=157&amp;issue=6&amp;publication_year=2002&amp;pages=963-974&amp;pmid=12058015&amp;doi=10.1083/jcb.200111077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0048">
<cite>
Shin MD, Shukla S, Chung YH, Beiss V, Chan SK, Ortega‐Rivera OA, et al. COVID‐19 vaccine development and a potential nanomaterial path forward. Nat Nanotechnol. 2020;15(8):646–655.
</cite> [<a href="https://doi.org/10.1038/s41565-020-0737-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Nanotechnol&amp;title=COVID%E2%80%9019%20vaccine%20development%20and%20a%20potential%20nanomaterial%20path%20forward&amp;author=MD%20Shin&amp;author=S%20Shukla&amp;author=YH%20Chung&amp;author=V%20Beiss&amp;author=SK%20Chan&amp;volume=15&amp;issue=8&amp;publication_year=2020&amp;pages=646-655&amp;pmid=32669664&amp;doi=10.1038/s41565-020-0737-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0049">
<cite>
Smith KM, di Antonio V, Bellucci L, Thomas DR, Caporuscio F, Ciccarese F, et al. Contribution of the residue at position 4 within classical nuclear localization signals to modulating interaction with importins and nuclear targeting. Biochim Biophys Acta Mol Cell Res. 2018;1865(8):1114–1129.
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2018.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20Mol%20Cell%20Res&amp;title=Contribution%20of%20the%20residue%20at%20position%204%20within%20classical%20nuclear%20localization%20signals%20to%20modulating%20interaction%20with%20importins%20and%20nuclear%20targeting&amp;author=KM%20Smith&amp;author=V%20di%20Antonio&amp;author=L%20Bellucci&amp;author=DR%20Thomas&amp;author=F%20Caporuscio&amp;volume=1865&amp;issue=8&amp;publication_year=2018&amp;pages=1114-1129&amp;pmid=29750988&amp;doi=10.1016/j.bbamcr.2018.05.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0050">
<cite>
Thomas MC, Chiang CM. The general transcription machinery and general cofactors. Crit Rev Biochem Mol Biol. 2006;41(3):105–178.
</cite> [<a href="https://doi.org/10.1080/10409230600648736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16858867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Biochem%20Mol%20Biol&amp;title=The%20general%20transcription%20machinery%20and%20general%20cofactors&amp;author=MC%20Thomas&amp;author=CM%20Chiang&amp;volume=41&amp;issue=3&amp;publication_year=2006&amp;pages=105-178&amp;pmid=16858867&amp;doi=10.1080/10409230600648736&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0051">
<cite>
Timney BL, Tetenbaum‐Novatt J, Agate DS, Williams R, Zhang W, Chait BT, et al. Simple kinetic relationships and nonspecific competition govern nuclear import rates in vivo. J Cell Biol. 2006;175(4):579–593.
</cite> [<a href="https://doi.org/10.1083/jcb.200608141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2064595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17116750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=Simple%20kinetic%20relationships%20and%20nonspecific%20competition%20govern%20nuclear%20import%20rates%20in%20vivo&amp;author=BL%20Timney&amp;author=J%20Tetenbaum%E2%80%90Novatt&amp;author=DS%20Agate&amp;author=R%20Williams&amp;author=W%20Zhang&amp;volume=175&amp;issue=4&amp;publication_year=2006&amp;pages=579-593&amp;pmid=17116750&amp;doi=10.1083/jcb.200608141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0052">
<cite>
Trowitzsch S, Viola C, Scheer E, Conic S, Chavant V, Fournier M, et al. Cytoplasmic TAF2‐TAF8‐TAF10 complex provides evidence for nuclear holo‐TFIID assembly from preformed submodules. Nat Commun. 2015;6:6011.
</cite> [<a href="https://doi.org/10.1038/ncomms7011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4309443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25586196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Cytoplasmic%20TAF2%E2%80%90TAF8%E2%80%90TAF10%20complex%20provides%20evidence%20for%20nuclear%20holo%E2%80%90TFIID%20assembly%20from%20preformed%20submodules&amp;author=S%20Trowitzsch&amp;author=C%20Viola&amp;author=E%20Scheer&amp;author=S%20Conic&amp;author=V%20Chavant&amp;volume=6&amp;publication_year=2015&amp;pages=6011&amp;pmid=25586196&amp;doi=10.1038/ncomms7011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0053">
<cite>
Tso SC, Chen Q, Vishnivetskiy SA, Gurevich VV, Iverson TM, Brautigam CA. Using two‐site binding models to analyze microscale thermophoresis data. Anal Biochem. 2018;540–541:64–75.</cite> [<a href="https://doi.org/10.1016/j.ab.2017.10.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5906060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29054528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&amp;title=Using%20two%E2%80%90site%20binding%20models%20to%20analyze%20microscale%20thermophoresis%20data&amp;author=SC%20Tso&amp;author=Q%20Chen&amp;author=SA%20Vishnivetskiy&amp;author=VV%20Gurevich&amp;author=TM%20Iverson&amp;volume=540%E2%80%93541&amp;publication_year=2018&amp;pages=64-75&amp;pmid=29054528&amp;doi=10.1016/j.ab.2017.10.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0054">
<cite>
Vaux DL. Basic statistics in cell biology. Annu Rev Cell Dev Biol. 2014;30:23–37.
</cite> [<a href="https://doi.org/10.1146/annurev-cellbio-100913-013303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25000992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Cell%20Dev%20Biol&amp;title=Basic%20statistics%20in%20cell%20biology&amp;author=DL%20Vaux&amp;volume=30&amp;publication_year=2014&amp;pages=23-37&amp;pmid=25000992&amp;doi=10.1146/annurev-cellbio-100913-013303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0055">
<cite>
Wirthmueller L, Roth C, Banfield MJ, Wiermer M. Hop‐on hop‐off: importin‐alpha‐guided tours to the nucleus in innate immune signaling. Front Plant Sci. 2013;4:149.
</cite> [<a href="https://doi.org/10.3389/fpls.2013.00149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3659281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23734157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&amp;title=Hop%E2%80%90on%20hop%E2%80%90off:%20importin%E2%80%90alpha%E2%80%90guided%20tours%20to%20the%20nucleus%20in%20innate%20immune%20signaling&amp;author=L%20Wirthmueller&amp;author=C%20Roth&amp;author=MJ%20Banfield&amp;author=M%20Wiermer&amp;volume=4&amp;publication_year=2013&amp;pages=149&amp;pmid=23734157&amp;doi=10.3389/fpls.2013.00149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70272-bib-0056">
<cite>
Zheng W, Wang R, Liu X, Tian S, Yao B, Chen A, et al. Structural insights into the nuclear import of the histone acetyltransferase males‐absent‐on‐the‐first by importin alpha1. Traffic. 2018;19(1):19–28.
</cite> [<a href="https://doi.org/10.1111/tra.12534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28991411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Structural%20insights%20into%20the%20nuclear%20import%20of%20the%20histone%20acetyltransferase%20males%E2%80%90absent%E2%80%90on%E2%80%90the%E2%80%90first%20by%20importin%20alpha1&amp;author=W%20Zheng&amp;author=R%20Wang&amp;author=X%20Liu&amp;author=S%20Tian&amp;author=B%20Yao&amp;volume=19&amp;issue=1&amp;publication_year=2018&amp;pages=19-28&amp;pmid=28991411&amp;doi=10.1111/tra.12534&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375970/bin/PRO-34-e70272-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70272-s001.docx</a><sup> (19.9MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available on request from the corresponding author. The data are not publicly available due to privacy or ethical restrictions.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70272"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70272.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375970/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375970/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375970%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375970/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375970/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375970/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375970/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375970/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375970/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MoaqPjTuq0Yj78x5rR31mJ152PK0AWa40qgiGPMtY5sdAe1ubkdX8sn74f4turWa">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
