
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Control of Golgi- V-ATPase through Sac1-dependent co-regulation of PI(4)P and cholesterol - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A3588AF2073305A3580056EDEBF6.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371084/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Control of Golgi- V-ATPase through Sac1-dependent co-regulation of PI(4)P and cholesterol">
<meta name="citation_author" content="Xin Zhou">
<meta name="citation_author_institution" content="Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland">
<meta name="citation_author_institution" content="Minerva Foundation Institute for Medical Research, Helsinki, Finland">
<meta name="citation_author" content="Miesje M van der Stoel">
<meta name="citation_author_institution" content="Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland">
<meta name="citation_author_institution" content="Minerva Foundation Institute for Medical Research, Helsinki, Finland">
<meta name="citation_author" content="Shreyas Kaptan">
<meta name="citation_author_institution" content="Department of Physics, University of Helsinki, Helsinki, Finland">
<meta name="citation_author" content="Haoran Li">
<meta name="citation_author_institution" content="Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland">
<meta name="citation_author_institution" content="Minerva Foundation Institute for Medical Research, Helsinki, Finland">
<meta name="citation_author" content="Shiqian Li">
<meta name="citation_author_institution" content="Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland">
<meta name="citation_author_institution" content="Minerva Foundation Institute for Medical Research, Helsinki, Finland">
<meta name="citation_author" content="Maarit Hölttä">
<meta name="citation_author_institution" content="Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland">
<meta name="citation_author_institution" content="Minerva Foundation Institute for Medical Research, Helsinki, Finland">
<meta name="citation_author" content="Helena Vihinen">
<meta name="citation_author_institution" content="Institute of Biotechnology, University of Helsinki, Helsinki, Finland">
<meta name="citation_author" content="Eija Jokitalo">
<meta name="citation_author_institution" content="Institute of Biotechnology, University of Helsinki, Helsinki, Finland">
<meta name="citation_author" content="Christoph Thiele">
<meta name="citation_author_institution" content="Life and Medical Science Institute, University of Bonn, Bonn, Germany">
<meta name="citation_author" content="Olli Pietiläinen">
<meta name="citation_author_institution" content="Neuroscience Center, Helsinki Institute of Life Science, University of Helsinki, Helsinki, Finland">
<meta name="citation_author" content="Shin Morioka">
<meta name="citation_author_institution" content="Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan">
<meta name="citation_author" content="Junko Sasaki">
<meta name="citation_author_institution" content="Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan">
<meta name="citation_author" content="Takehiko Sasaki">
<meta name="citation_author_institution" content="Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan">
<meta name="citation_author" content="Ilpo Vattulainen">
<meta name="citation_author_institution" content="Department of Physics, University of Helsinki, Helsinki, Finland">
<meta name="citation_author" content="Elina Ikonen">
<meta name="citation_author_institution" content="Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland">
<meta name="citation_author_institution" content="Minerva Foundation Institute for Medical Research, Helsinki, Finland">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7808">
<meta name="citation_doi" content="10.1038/s41467-025-63125-7">
<meta name="citation_pmid" content="40841558">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371084/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371084/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371084/pdf/41467_2025_Article_63125.pdf">
<meta name="description" content="Sac1 is a conserved phosphoinositide phosphatase, whose loss-of-function compromises cell and organism viability. Here, we employ acute auxin-inducible Sac1 degradation to identify its immediate downstream effectors in human cells. Most of Sac1 is ...">
<meta name="og:title" content="Control of Golgi- V-ATPase through Sac1-dependent co-regulation of PI(4)P and cholesterol">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Sac1 is a conserved phosphoinositide phosphatase, whose loss-of-function compromises cell and organism viability. Here, we employ acute auxin-inducible Sac1 degradation to identify its immediate downstream effectors in human cells. Most of Sac1 is ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371084/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371084">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63125-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63125.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371084%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371084/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371084/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371084/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7808. doi: <a href="https://doi.org/10.1038/s41467-025-63125-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63125-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Control of Golgi- V-ATPase through Sac1-dependent co-regulation of PI(4)P and cholesterol</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xin Zhou</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xin Zhou</span></h3>
<div class="p">
<sup>1</sup>Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland </div>
<div class="p">
<sup>2</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin Zhou</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20der%20Stoel%20MM%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Miesje M van der Stoel</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Miesje M van der Stoel</span></h3>
<div class="p">
<sup>1</sup>Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland </div>
<div class="p">
<sup>2</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20der%20Stoel%20MM%22%5BAuthor%5D" class="usa-link"><span class="name western">Miesje M van der Stoel</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaptan%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Shreyas Kaptan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Shreyas Kaptan</span></h3>
<div class="p">
<sup>3</sup>Department of Physics, University of Helsinki, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaptan%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shreyas Kaptan</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Haoran Li</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Haoran Li</span></h3>
<div class="p">
<sup>1</sup>Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland </div>
<div class="p">
<sup>2</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haoran Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shiqian Li</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shiqian Li</span></h3>
<div class="p">
<sup>1</sup>Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland </div>
<div class="p">
<sup>2</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shiqian Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%C3%B6ltt%C3%A4%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Maarit Hölttä</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Maarit Hölttä</span></h3>
<div class="p">
<sup>1</sup>Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland </div>
<div class="p">
<sup>2</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%C3%B6ltt%C3%A4%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maarit Hölttä</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vihinen%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Helena Vihinen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Helena Vihinen</span></h3>
<div class="p">
<sup>4</sup>Institute of Biotechnology, University of Helsinki, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vihinen%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Helena Vihinen</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jokitalo%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Eija Jokitalo</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Eija Jokitalo</span></h3>
<div class="p">
<sup>4</sup>Institute of Biotechnology, University of Helsinki, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jokitalo%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eija Jokitalo</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiele%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Christoph Thiele</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Christoph Thiele</span></h3>
<div class="p">
<sup>5</sup>Life and Medical Science Institute, University of Bonn, Bonn, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thiele%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christoph Thiele</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pietil%C3%A4inen%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Olli Pietiläinen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Olli Pietiläinen</span></h3>
<div class="p">
<sup>6</sup>Neuroscience Center, Helsinki Institute of Life Science, University of Helsinki, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pietil%C3%A4inen%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olli Pietiläinen</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morioka%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Shin Morioka</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Shin Morioka</span></h3>
<div class="p">
<sup>7</sup>Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morioka%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shin Morioka</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sasaki%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Junko Sasaki</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Junko Sasaki</span></h3>
<div class="p">
<sup>7</sup>Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sasaki%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junko Sasaki</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sasaki%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Takehiko Sasaki</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Takehiko Sasaki</span></h3>
<div class="p">
<sup>7</sup>Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sasaki%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Takehiko Sasaki</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vattulainen%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Ilpo Vattulainen</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Ilpo Vattulainen</span></h3>
<div class="p">
<sup>3</sup>Department of Physics, University of Helsinki, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vattulainen%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ilpo Vattulainen</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ikonen%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Elina Ikonen</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Elina Ikonen</span></h3>
<div class="p">
<sup>1</sup>Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland </div>
<div class="p">
<sup>2</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ikonen%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elina Ikonen</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Stem Cells and Metabolism Research Program and Department of Anatomy, Faculty of Medicine, University of Helsinki, Helsinki, Finland </div>
<div id="Aff2">
<sup>2</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div>
<div id="Aff3">
<sup>3</sup>Department of Physics, University of Helsinki, Helsinki, Finland </div>
<div id="Aff4">
<sup>4</sup>Institute of Biotechnology, University of Helsinki, Helsinki, Finland </div>
<div id="Aff5">
<sup>5</sup>Life and Medical Science Institute, University of Bonn, Bonn, Germany </div>
<div id="Aff6">
<sup>6</sup>Neuroscience Center, Helsinki Institute of Life Science, University of Helsinki, Helsinki, Finland </div>
<div id="Aff7">
<sup>7</sup>Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 22; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371084  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841558/" class="usa-link">40841558</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Sac1 is a conserved phosphoinositide phosphatase, whose loss-of-function compromises cell and organism viability. Here, we employ acute auxin-inducible Sac1 degradation to identify its immediate downstream effectors in human cells. Most of Sac1 is degraded in ~1 h, paralleled by increased PI(4)P and decreased cholesterol in the <em>trans-</em>Golgi network (TGN) during the following hour, and superseded by Golgi fragmentation, impaired glycosylation, and selective degradation of TGN proteins by ~4 h. The TGN disintegration results from its acute deacidification caused by disassembly of the Golgi V-ATPase. Mechanistically, Sac1 mediated TGN membrane composition maintains an assembly-promoting conformation of the V<sub>0</sub>a2 subunit. Key phenotypes of acute Sac1 degradation are recapitulated in human differentiated trophoblasts, causing processing defects of chorionic gonadotropin, in line with loss-of-function intolerance of the human <em>SACM1L</em> gene. Collectively, our findings reveal that the assembly of the Golgi V-ATPase is controlled by the TGN membrane via Sac1 fuelled lipid exchange.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Golgi, Lipidomics, Computational biophysics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">This study employs auxin-inducible degradation of Sac1. The authors reveal that acute Sac1 depletion changes the Golgi membrane lipid composition, causing disassembly of the Golgi V-ATPase and eventually resulting in cargo processing defects.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">PI(4)P is the most abundant species of phosphoinositides that serve as major regulators of organelle identity and dynamics<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. It is asymmetrically distributed in the cell, with high levels in the <em>trans-</em>Golgi network (TGN) and the plasma membrane (PM)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>, regulated by an interplay between organelle specific PI4-kinases (PI4Ks) and the endoplasmic reticulum (ER)/Golgi integral phosphatase Sac1<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In the Golgi, PI(4)P plays an important role by controlling the localization, signaling and activity of PI(4)P binding proteins and subsequently lipid homeostasis and vesicular trafficking<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Defects in the regulation of cellular PI(4)P levels are linked to neurodegenerative diseases<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> and cancer<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par4">Importantly, PI(4)P in the TGN serves as counter-exchange cargo for lipid transfer reactions at membrane contact sites. Here, PI(4)P regulates the docking of lipid transfer proteins, such as oxysterol-binding protein (OSBP), which transfers PI(4)P from the TGN to the ER and in exchange relocates cholesterol to the opposite direction, from the ER to the TGN<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. At the ER, Sac1 hydrolyzes PI(4)P into phosphatidylinositol (PI), thereby creating an intracellular PI(4)P gradient that drives anterograde cholesterol transport and reduces PI(4)P levels in the TGN<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Besides OSBP, other lipid transporters such as ORP9 and Asters/GRAMDs, are also involved in maintaining the cholesterol distribution between the ER and TGN<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Sac1 depletion causes an increase in PI(4)P<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and results in enlargement of the Golgi, mislocalization of glycosylation enzymes and glycosylation defects<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Additionally, by locally controlling Golgi PI(4)P levels, Sac1 can regulate cargo trafficking and secretion<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. However, how Sac1 mediated cholesterol and PI(4)P distribution affects Golgi functioning and which critical effectors are involved in this, are currently unknown.</p>
<p id="Par5">Sac1, or suppressor of actin-1, was first discovered in yeast, where it regulates actin cytoskeletal organization and Golgi secretion<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Sac1 is the only known PI(4)P phosphatase in mammalian cells and its catalytic domain is highly conserved across species<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Constitutive loss of Sac1 compromises the viability of cells and organisms, causing early embryonic lethality in e.g. <em>Drosophila</em> and mice<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. As long-term loss of Sac1 compromises viability, the precise mechanisms of how Sac1 fuelled lipid exchange affects the function of the Golgi have been challenging to study. Indeed, due to the rapid turnover and broad distribution of phosphoinositide species in cellular membranes, most if not all membrane compartments may become afflicted in chronic Sac1 depletion models.</p>
<p id="Par6">Here, we employed auxin-inducible degradation (AID) of Sac1 to rapidly deplete the protein in several human cell types, allowing us to investigate early effects after Sac1 loss. We identified vacuolar-type ATPase (V-ATPase) as a key effector of Sac1 controlled PI(4)P and cholesterol content in the TGN membrane, impacting TGN pH and resulting in defects in Golgi terminal glycosylation and secretion, with physiological implications for early embryonic development. Mechanistically, increased PI(4)P levels in the TGN characteristic of Sac1 loss, promoted a conformational change of the V<sub>0</sub>a2 subunit of the Golgi V-ATPase, preventing the assembly of the V-ATPase holo-complex. Taken together, this study reveals the Golgi V-ATPase as a critical effector of Sac1 fuelled PI(4)P/cholesterol exchange to regulate Golgi integrity and cargo processing.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Sac1 exerts acute control of Golgi morphology</h3>
<p id="Par7">To investigate the acute consequences of Sac1 depletion on the Golgi, we took advantage of the AID system that we recently established in several human cell types<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> (Fig. <a href="#Fig1" class="usa-link">1a</a>). Within 1 h after IAA addition, about 90% of endogenous Sac1 was removed from A431 cells (Fig. <a href="#Fig1" class="usa-link">1b</a>). At this time, cellular PI(4)P was already moderately increased, as assessed by mass spectrometry<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, and after 2 h of IAA treatment, the PI(4)P content was robustly elevated, to ~3-fold from the starting level (Fig. <a href="#Fig1" class="usa-link">1c</a>). Instead, no significant or very minor changes were observed in PI(3)P, PI(4,5)P<sub>2</sub>, and PI(3,5)P<sub>2</sub> species at this time (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a–c</a>), highlighting PI(4)P as the primary substrate of Sac1 in cells. All acyl chain compositions of PI(4)P species were similarly affected, whereas no changes in the acyl chain composition of the other PIP species were observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">1d</a>). By 4 h of Sac1 depletion, all detected phosphoinositide species were elevated compared to the starting situation (Supplementary Fig. <a href="#MOESM1" class="usa-link">1e–h</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Rapid Sac1 depletion disturbs Golgi morphology.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371084_41467_2025_63125_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f66c/12371084/091e9dc2f58d/41467_2025_63125_Fig1_HTML.jpg" loading="lazy" id="d33e513" height="832" width="699" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic overview of the genetic manipulations for auxin-inducible degradation of Sac1. Sac1 is endogenously tagged with a miniIAA7-3xFlag tag and the auxin receptor <em>At</em>AFB2(F74A) mutant is exogenously expressed in cells. After addition of an IAA analogue, miniIAA7-3xFlag-Sac1 is recruited to the AtAFB2(F74A)-Cullin E3 ligase complex, resulting in polyubiquitination and proteasomal degradation of Sac1. <strong>b</strong> Western blot analysis of Sac1-degron A431 cell lysates after IAA addition for 0 to 4 h in A431 cells, blotted for miniIAA7 to detect Sac1. Normalized relative intensity of each band is marked under the blot. <strong>c</strong> Bar graph showing total PI(4)P levels ±S.E.M. (pmol/x10<sup>6</sup> cells) of Sac1-degron A431 cells treated with DMSO for 2 h (0) or IAA for 0.5, 1 or 2 h as measured by PRMC-MS. n = 3 biological replicates. One-way ANOVA with Dunnett’s multiple comparisons test. n.s. p = 0.7464, * p = 0.0167, **** p &lt; 0.0001. <strong>d</strong> Representative confocal images of Sac1-degron A431 cells treated with DMSO (4 h) or IAA for 2 or 4 h and stained for GM130 (<em>cis-</em>Golgi) or TGN46 (<em>trans-</em>Golgi). <strong>e</strong> Bar graph depicting the percentage (mean ±S.E.M) of cells with intact, partially fragmented, or fully fragmented Golgi. n = 3 independent experiments. <strong>f</strong> Western blot analysis of Sac1-degron A431 cells treated with DMSO or IAA for 4 h and blotted for TGN46, Golgin-97, GM130 and GRASP55. <strong>g</strong> Representative TEM images of Sac1-degron A431 cells treated with DMSO or IAA for 2 h. The yellow arrows indicate the Golgi stacks.</p></figcaption></figure><p id="Par8">Previously, Sac1 depletion was shown to affect Golgi morphology<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. To investigate the effect of Sac1 degradation on Golgi integrity, we performed immunofluorescence staining using Golgi markers. After 2 h of IAA induction, the normally tight perinuclear clustering of <em>cis-</em> and <em>trans-</em>Golgi elements (GM130 and TGN46 as markers, respectively) was lost and the ratio of cells with aberrantly fragmented Golgi increased (Fig. <a href="#Fig1" class="usa-link">1d, e</a>). By 4 h, GM130-positive structures were largely fragmented and TGN46 was hardly detectable by fluorescence microscopy (Fig. <a href="#Fig1" class="usa-link">1d, e</a>). This was reflected by reduced TGN46 protein levels, while the protein levels of Golgin-97 (<em>trans-</em>), GRASP55 (medial-) and GM130 (<em>cis-</em>Golgi) remained unchanged (Fig. <a href="#Fig1" class="usa-link">1f</a>).</p>
<p id="Par9">Consistent with these results, thin-section transmission electron microscopy (TEM) revealed a rapid transformation of Golgi ribbons to tubular-vesicular networks in 2 h of IAA induction (Fig. <a href="#Fig1" class="usa-link">1g</a>). While the Golgi had a typical ribbon-like organization in 80% of the DMSO treated cells, the Golgi elements were dispersed in 70% of the Sac1 degraded cells: the occurrence and length of stacks decreased, and the Golgi elements were mainly compromised of tubular networks and tubulo-vesicular clusters (Fig. <a href="#Fig1" class="usa-link">1g</a> and Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Importantly, the rapid fragmentation of the Golgi upon Sac1 removal was also evident in other Sac1-degron models, such as HEK293 and A549 cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">1i, j</a>). These results show that Sac1 is important to maintain Golgi integrity.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Sac1 phosphatase activity is required for terminal glycosylation</h3>
<p id="Par10">To obtain an unbiased overview of the effects of acute Sac1 removal at the protein level, we performed quantitative proteomic analysis of A431 Sac1-degron cells. After 3 h of IAA induction, a total of 8040 proteins were identified, of which 203 were upregulated (fold change ≥ 1.3, IAA to DMSO) and 103 downregulated (fold change ≤ 0.77, IAA to DMSO) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>, Supplementary Data <a href="#MOESM4" class="usa-link">2</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Among the most upregulated proteins in the Sac1-degron cell proteomics were PI(4)P binding proteins, such as OSBP, ORP9 and CERT1 (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>). Gene ontology (GO) analysis revealed that the most downregulated proteins were enriched in the biological processes and cellular components related to glycosylation, Golgi organization and membrane (Fig. <a href="#Fig2" class="usa-link">2a</a> and Supplementary Data <a href="#MOESM4" class="usa-link">2</a>), and the most upregulated proteins were enriched in membrane trafficking and organelle organization (Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a> and Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). Interestingly, while the levels of ER and <em>cis-</em>/medial-Golgi glycosylation enzymes were unaltered or moderately downregulated, the enzymes of the <em>trans-</em>Golgi cisternae, in particular galactosyl- and sialyltransferases, were strongly downregulated (Fig. <a href="#Fig2" class="usa-link">2b</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Terminal glycosylation is impaired by Sac1 depletion.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371084_41467_2025_63125_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f66c/12371084/0e854e46e092/41467_2025_63125_Fig2_HTML.jpg" loading="lazy" id="d33e645" height="504" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Top terms in gene ontology enrichment analysis of the downregulated proteins in Sac1-degron A431 cells treated with IAA for 3 h compared to DMSO control (-Log<sub>10</sub> False Discovery Rate (FDR) with a fold change threshold of ≥ 1.30 or ≤ 0.77 and an adjusted p-value of ≤ 0.05). <strong>b</strong> Schematic overview illustrating the changes in protein levels of glycosylation enzymes in Sac1-degron A431 cells after 3 h IAA treatment. The Log<sub>2</sub> fold change (Log<sub>2</sub>FC) of individual enzyme is presented using a color gradient from magenta (−2) to green ( + 2). Glycosylation starts in the ER, where precursor glycan chains are added to the protein. The glycans are modified sequentially by glycosidases and glycosyltransferases along the ER and Golgi stacks. <strong>c</strong> Western blot analysis of TGN46 and B4GALT1 in Sac1-degron A431 cells treated with IAA for 0 to 4 h. <strong>d</strong> Bar graph showing normalized intensity of B4GALT1 and TGN46 (mean ±S.E.M) quantified from (<strong>c</strong>). n = 3 independent experiments. One-way ANOVA with Dunnett’s multiple comparisons test. TGN46: n.s. p = 0.9964, p = 0.5307, *** p = 0.0004, p = 0.0001. B4GALT1: n.s. p = 0.8336, p = 0.9872, p = 0.2245, ** p = 0.0006. <strong>e</strong> Western blot analysis of lysates of HEK293A Sac1-degron cells transfected with pcDNA3.1 (empty vector), EGFP-Sac1-WT (Sac1<sup>wt</sup>) or EGFP-Sac1-C389S (Sac1<sup>mut</sup>), treated for 6 h with DMSO or IAA and blotted for B4GALT1 and GFP (Sac1). <strong>f</strong> Graph depicts the B4GALT1 degradation efficiency (mean ±S.E.M) in Sac1-degron HEK293A cells treated for 6 h with IAA and rescued with Sac1<sup>wt</sup> or Sac1<sup>mut</sup>. n = 5 independent experiments. One-way ANOVA with Tukey’s multiple comparisons test. n.s. p = 0.6379, ****p &lt; 0.0001.</p></figcaption></figure><p id="Par11">Western blotting confirmed the robust reduction of beta-1,4-galactosyltransferase 1 (B4GALT1) and of TGN46 (Fig. <a href="#Fig2" class="usa-link">2c, d</a>) by 4 h of IAA induction, while the mRNA levels of B4GALT1 and TGN46 remained unchanged (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>). To investigate if lysosomal degradation plays a role in the loss of TGN46, we neutralized lysosomes using Bafilomycin A1 (BafA1) during IAA treatment. This effectively restored TGN46 protein levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">2d</a>), suggesting that TGN46 undergoes lysosomal proteolysis upon Sac1 depletion. Interestingly, at 8 h the mature form of TGN46 was replaced by an immature form of the protein (Supplementary Fig. <a href="#MOESM1" class="usa-link">2e</a>), and PNGaseF and α−2-3,6,8,9-Neuraminidase A treatments indicated defects in its terminal glycosylation (Supplementary Fig. <a href="#MOESM1" class="usa-link">2f, g</a>). Of note, 4 h treatment of OSW-1 also reduced TGN46 protein levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">2h, i</a>), indicating that OSBP inhibition phenocopied acute Sac1 depletion.</p>
<p id="Par12">Sac1-dependent functions rely on its catalytic activity<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. To scrutinize if the effect of Sac1 on TGN integrity was dependent on its phosphatase activity, we reintroduced EGFP-tagged wild-type Sac1 (Sac1<sup>wt</sup>) and catalytically inactive mutant Sac1-C389S (Sac1<sup>mut</sup>) to Sac1 degraded cells, using the well transfectable HEK293A Sac1-degron cells. After 24 h of transient transfection, endogenously degron-tagged Sac1 was removed by 6 h of IAA induction, while the levels of exogenous Sac1<sup>wt</sup> and Sac1<sup>mut</sup> remained stable (Fig. <a href="#Fig2" class="usa-link">2e</a>). In this setting, the reduction of B4GALT1 levels caused by Sac1 degradation could be rescued by reintroduction of Sac1<sup>wt</sup> but not by Sac1<sup>mut</sup> (Fig. <a href="#Fig2" class="usa-link">2e, f</a>). Collectively, these results suggest that Sac1 mediated PI(4)P hydrolysis maintains TGN integrity and terminal glycosylation.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Sac1 is needed to maintain TGN pH and Golgi V-ATPase assembly status</h3>
<p id="Par13">The Golgi maintains a proton gradient, from neutral pH at the <em>cis-</em>face to pH ~6 at the TGN. This is important to maintain Golgi integrity and protein glycosylation<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. To assess if acute Sac1 degradation affects the pH of the TGN, we transfected Sac1-degron cells with TGN-targeting GalT-RpHLuorin2, a pH-sensitive GFP mutant fused to B4GALT1<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). Fluorescence lifetime values of GalT-RpHLuorin2 were obtained from A431 Sac1-degron cells and converted to pH values according to a standard curve (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>).</p>
<p id="Par14">The average pH of the TGN of A431 cells remained consistently ~5.9 in the DMSO control group over time, while it was elevated to ~6.1 at 2 h and ~6.3 at 3 h of IAA induction (Fig. <a href="#Fig3" class="usa-link">3a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">4c</a>). However, when we applied EN6, an activator of vacuole-type ATPase (V-ATPase)<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> to counteract the pH elevation, the TGN pH dropped to ~6.0 in Sac1-depleted cells (Fig. <a href="#Fig3" class="usa-link">3b</a>). Although EN6 marginally affected inducible Sac1 depletion, re-acidification of the TGN efficiently prevented Sac1 depletion induced B4GALT1 and TGN46 degradation (Fig. <a href="#Fig3" class="usa-link">3c, d</a>), and partially restored the TGN morphology (Fig. <a href="#Fig3" class="usa-link">3e, f</a>). These data argue that acute Sac1 depletion deacidifies the TGN, thereby inducing selective TGN morphological changes and degradation.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Acute Sac1 depletion results in TGN deacidification caused by V-ATPase disassembly.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371084_41467_2025_63125_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f66c/12371084/9cf7e1f7789f/41467_2025_63125_Fig3_HTML.jpg" loading="lazy" id="d33e827" height="994" width="785" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Violin plot showing the distribution of TGN pH values of Sac1-degron A431 cells transiently transfected with GalT-RpHLuorin2 and treated with DMSO or IAA for 0 to 3 h. n = 3 independent experiments. One-way ANOVA with Tukey’s multiple comparisons test. n.s. p &gt; 0.9999, p = 0.9848, * p = 0.0480, **** p &lt; 0.0001. <strong>b</strong> Violin plot depicting the distribution of TGN pH values of Sac1-degron A431 cells treated for 3 h with DMSO or IAA with or without 50 µM EN6. n = 3 independent experiments. One-way ANOVA with Tukey’s multiple comparison test. n.s. p = 0.4775, ** p = 0.0039, **** p &lt; 0.0001. <strong>c</strong> Western blot analysis of Sac1-degron A431 cells treated for 4 h with DMSO or IAA with or without 50 µM EN6. Immunoblotted for TGN46, B4GALT1 and miniIAA7 (Sac1). <strong>d</strong> Bar graph showing average normalized immunoblot intensities from (<strong>c</strong>). n = 4 independent experiments. One-way ANOVA with Tukey’s multiple comparisons test. TGN46: *** p = 0.0004, p = 0.0002, **** p &lt; 0.0001. B4GALT1: * p = 0.0127, ** p = 0.0022, p = 0.0026. miniIAA7: *** p = 0.0001, **** p &lt; 0.0001. n.s. p = 0.7763. <strong>e</strong> Representative confocal images of Sac1-degron A431 cells treated for 4 h with DMSO or IAA with or without 50 µM EN6. Stained for DAPI, Golgin-97 and TGN46. <strong>f</strong> Violin plot depicts the distribution of the TGN46 intensity in the Golgi area calculated from (<strong>e</strong>). n = 3 independent experiments. One-way ANOVA with Tukey’s multiple comparisons test. n.s. p = 0.6182, **** p &lt; 0.0001. <strong>g</strong> Cartoon illustrating the reversible assembly of the V<sub>0</sub> and V<sub>1</sub> regions of the V-ATPase complex. Upon assembly, ATP hydrolysis of the V<sub>1</sub> region drives the proton pumping ability of the V<sub>0</sub> region. <strong>h</strong> Representative fluorescence micrographs of Sac1-degron A431 cells treated with DMSO for 4 h or IAA for 2 or 4 h and stained for ATP6V<sub>0</sub>a2 or ATP6V<sub>1</sub>A. <strong>i</strong> Violin plot showing the distribution of the Pearson’s correlation coefficient between ATP6V<sub>0</sub>a2 and ATP6V<sub>1</sub>A from (<strong>h</strong>). Representative data from one of three different experiments. One-way ANOVA with Dunnett’s multiple comparisons test. **** p &lt; 0.0001. Representative confocal images of Sac1-degron A549 cells treated with DMSO or IAA for 2 h and stained for ATP6V<sub>1</sub>A and Golgin-97 (<strong>j</strong>) or ATP6V<sub>1</sub>A and LAMP2 (<strong>l</strong>). <strong>k</strong>, <strong>m</strong> Violin plot showing the Pearson’s correlation coefficient between ATP6V<sub>1</sub>A and Golgin-97 (<strong>j</strong>) and ATP6V<sub>1</sub>A and LAMP2 (<strong>l</strong>). n = 3 independent experiments. Two-tailed unpaired Student’s t-test. **** p &lt; 0.0001, n.s. p = 0.1287.</p></figcaption></figure><p id="Par15">V-ATPase is a ubiquitous and essential regulator of organelle pH, with the cytoplasmic V<sub>1</sub> region consuming ATP to facilitate proton pumping through the membrane-embedded V<sub>0</sub> region. The reversible assembly and disassembly of the V<sub>0</sub> and V<sub>1</sub> regions orchestrates proton translocation to adapt to various physiological demands (Fig. <a href="#Fig3" class="usa-link">3g</a>)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. We therefore studied if the V-ATPase complex is affected by Sac1 depletion. The total amounts of V<sub>0</sub>- and V<sub>1</sub>-subunits were not altered by acute Sac1 degradation, as assessed by specific antibodies (Supplementary Fig. <a href="#MOESM1" class="usa-link">4d</a>). However, membrane-cytosol fractionation suggested that upon 2 h of Sac1 degradation the membrane-association of the V<sub>1</sub>A subunit was reduced (Supplementary Fig. <a href="#MOESM1" class="usa-link">4e, f</a>).</p>
<p id="Par16">We next studied the subcellular localization of the V<sub>1</sub> region. For this, we employed Sac1-degron A549 cells, due to the prominent cytoplasmic V<sub>1</sub>A immunoreactivity in A431 cells that complicated the assessment. We found substantial colocalization of the V<sub>1</sub>A subunit with the Golgi-specific V<sub>0</sub>a2 subunit in control conditions, as expected<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Remarkably, this colocalization diminished gradually during 2 – 4 h after inducing Sac1 degradation (Fig. <a href="#Fig3" class="usa-link">3h, i</a>). Co-staining with Golgin-97 (Fig. <a href="#Fig3" class="usa-link">3j, k</a>) and LAMP2 (Fig. <a href="#Fig3" class="usa-link">3l, m</a>) antibodies revealed that at 2 h after inducing Sac1 degradation, the TGN localization of the V<sub>1</sub> region was reduced, while its lysosomal localization was maintained. In parallel, the Golgi specific V<sub>0</sub>a2 subunit clearly overlapped with the Golgi marker GM130, while no colocalization with the lysosomal marker LAMP2 was observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">4g</a>). Together, these data suggest that the V<sub>0</sub> and V<sub>1</sub> regions of the Golgi V-ATPase complex become disassembled rapidly after Sac1 depletion, providing a mechanism for the deacidification of the TGN.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Sac1 controls V-ATPase complex integrity via its lipid environment</h3>
<p id="Par17">As Sac1 fuels PI(4)P/cholesterol lipid exchange by lipid transfer proteins, i.e. OSBP<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, we assessed the role of Sac1 in controlling PI(4)P and cholesterol distribution in the TGN. Consistently with the mass spectrometry results (Fig. <a href="#Fig1" class="usa-link">1c</a>), expression of mCherry-P4M-SidM and immunostaining with PI(4)P antibody in A431 Sac1-degron cells revealed an increasing PI(4)P signal over time, starting perinuclearly at early time points after degradation and extending later on throughout the cell (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a–c</a>). The latter agrees with findings from chronic Sac1 depletion systems<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Remarkably, already at 1 h of Sac1 degradation, we observed a prominent accumulation of PI(4)P in the TGN region (Fig. <a href="#Fig4" class="usa-link">4a, b</a>). Staining of live cells with mNeonGreen-ALOD4, recognizing PM accessible cholesterol<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, showed that after 1 h of Sac1 removal the PM accessible cholesterol levels were slightly reduced, and by 4 h, only ~20% of control levels remained (Supplementary Fig. <a href="#MOESM1" class="usa-link">5d, e</a>). Filipin staining and thin-layer chromatography showed a minor reduction in total cellular free cholesterol over time (Supplementary Fig. <a href="#MOESM1" class="usa-link">5f–h</a>). However, by 2 h of Sac1 degradation, the filipin signal in the TGN area was significantly reduced (Fig. <a href="#Fig4" class="usa-link">4c, d</a>) and in the lysosomes slightly increased (Supplementary Fig. <a href="#MOESM1" class="usa-link">5i, j</a>). Meanwhile, cholesterol esterification was increased 2.5-fold by 2 h (Supplementary Fig. <a href="#MOESM1" class="usa-link">5k</a>) and total cholesteryl ester content elevated 1.5-fold by 4 h (Supplementary Fig. <a href="#MOESM1" class="usa-link">5l</a>), likely due to reduced cholesterol delivery from the ER to the TGN, resulting in cholesterol accumulation in the ER. Together, these results provide evidence that degradation of Sac1 rapidly alters both the PI(4)P and cholesterol content of the TGN.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Sac1 fuelled PI(4)P/cholesterol exchange maintains V-ATPase activity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371084_41467_2025_63125_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f66c/12371084/8b06dad88e66/41467_2025_63125_Fig4_HTML.jpg" loading="lazy" id="d33e999" height="994" width="760" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative confocal images of Sac1-degron A431 cells treated with IAA or DMSO for 1 h and stained for PI(4)P and TGN46. <strong>b</strong> Violin plot showing the average PI(4)P intensity in the Golgi area (TGN46). Representative data from one of three independent experiments. Two-tailed unpaired Student’s t-test. **** p &lt; 0.0001. <strong>c</strong> Representative confocal images of Sac1-degron A431 cell treated with IAA or DMSO for 2 h and stained with Filipin and Golgin-97. <strong>d</strong> Violin plot showing the normalized filipin intensity in the Golgi area (Golgin-97). Data from 3 independent experiments. Two-tailed Mann-Whitney U test. **** p &lt; 0.0001. <strong>e</strong> Western blot analysis of sucrose gradient fractionations (Sucrose concentration from 0.3 M (fraction 1) to 2.1 M (loading fractions 9 – 12)) of A549 Sac1-degron cells treated for 2 h with DMSO, IAA, IAA with cholesterol, or IAA with PI4K inhibitors (NC03 and PIK93) and blotted for ATP6V<sub>0</sub>a2. <strong>f</strong> Line graph depicting the average percentage (mean±S.E.M.) of floating ATP6V<sub>0</sub>a2 distributed between fraction 1 to 7 of A549 Sac1-degron cells treated for 2 h with DMSO, IAA, IAA plus cholesterol, or IAA plus PI4K inhibitors (NC03 and PIK93). n = 3 technical replicates. <strong>g</strong> Representative confocal images of A549 Sac1-degron cells treated for 2 h with DMSO, IAA, IAA in combination with cholesterol or PI4K inhibitors. Stained for ATP6V<sub>1</sub>A and ATP6V<sub>0</sub>a2. <strong>h</strong> Violin plot depicting the distribution of the Pearson’s correlation coefficient between ATP6V<sub>1</sub>A and ATP6V<sub>0</sub>a2 from (<strong>g</strong>). Data from one of three independent experiments. One-way ANOVA with Tukey’s multiple comparisons tests. *** p = 0.0008, **** p &lt; 0.0001.</p></figcaption></figure><p id="Par18">Next, we investigated if the changes in the TGN lipid environment induced by Sac1 degradation affect the Golgi V-ATPase by using sucrose density gradient fractionation of cellular post-nuclear supernatants (PNS). We found that the floating Golgi V-ATPase V<sub>0</sub> region peaked in the light membrane fraction 2, as probed by an anti-V<sub>0</sub>a2 antibody (Fig. <a href="#Fig4" class="usa-link">4e, f</a>). Upon Sac1 depletion for 2 h, the flotation of the V<sub>0</sub> region became less efficient, suggesting that the membrane environment of the V<sub>0</sub> region shifted towards higher equilibrium density (Fig. <a href="#Fig4" class="usa-link">4e, f</a>). Interestingly, cellular cholesterol depletion phenocopied the effect of Sac1 depletion on V<sub>0</sub> flotation (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>), suggesting that the cholesterol reduction in the TGN membrane contributes to the behavior of the V<sub>0</sub> region in Sac1 depleted cells.</p>
<p id="Par19">To probe this more directly, we replenished cells with cholesterol while Sac1 was being degraded. This effectively rescued the flotation of the V<sub>0</sub> region (Fig. <a href="#Fig4" class="usa-link">4e, f</a>). We also tested the effect of inhibiting PI(4)P generation by PI4K inhibitors (PIK93 for PI4KIIIβ and NC03 for PI4KIIα), the main PI4Ks in the Golgi<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, during Sac1 degradation. This did not revert the V<sub>0</sub> region to low-density membrane fractions (Fig. <a href="#Fig4" class="usa-link">4e, f</a>), suggesting that PI(4)P, as a minor membrane constituent, does not affect the overall flotation behavior of the V<sub>0</sub> region. Together, these findings suggest that cholesterol reduction in the TGN induced by Sac1 removal affects the TGN membrane environment of the V<sub>0</sub> region.</p>
<p id="Par20">Next, we analyzed if cholesterol supplementation or PI4K inhibitor treatment during Sac1 degradation can rescue V<sub>0</sub>-V<sub>1</sub> complex assembly. Interestingly, both increasing membrane cholesterol or decreasing PI(4)P generation improved the V<sub>0</sub>-V<sub>1</sub> colocalization in Sac1 depleted cells (Fig. <a href="#Fig4" class="usa-link">4g, h</a>), while only PI4K inhibition could partially rescue Golgi morphology (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b–d</a>). Collectively, the data suggest that Sac1 degradation alters the TGN membrane lipid composition resulting in V-ATPase complex disassembly, providing a mechanism for the TGN deacidification and glycosylation defects.</p></section><section id="Sec7"><h3 class="pmc_sec_title">The conformation of the V<sub>0</sub> region is influenced by the membrane lipid composition</h3>
<p id="Par21">To investigate how the membrane lipid composition affects the Golgi V-ATPase complex, we carried out atomic-level molecular dynamics simulations of the V<sub>0</sub> region containing the Golgi specific V<sub>0</sub>a2 subunit (Fig. <a href="#Fig5" class="usa-link">5a</a>), focusing on the role of PI(4)P and cholesterol (Fig. <a href="#Fig5" class="usa-link">5b</a> and Table <a href="#Tab1" class="usa-link">1</a>). We analyzed the simulation data using deep learning techniques, looking for structural classes where the V<sub>0</sub> region conformations are similar. Using this approach, the conformational space was first reduced to two dimensions using an autoencoder, after which these states were clustered with a Bayesian Gaussian Mixture Model, searching for structural classes representing distinct conformations. The analysis revealed four separate clusters (Fig. <a href="#Fig5" class="usa-link">5c</a>) characterized by balanced populations (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a</a>) and high structural similarity within each cluster (Supplementary Fig. <a href="#MOESM1" class="usa-link">7b</a>). A representative conformation of the V<sub>0</sub> region from each of the four clusters is depicted in Fig. <a href="#Fig5" class="usa-link">5d</a> (top view) and Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a> (side view). Based on structural data of the V<sub>0</sub>-V<sub>1</sub> complex, the assembly of the V<sub>0</sub> and V<sub>1</sub> regions requires the detachment of the cytosolic head domain of the V<sub>0</sub>a2 subunit from the d1 subunit, otherwise the a2 head sterically hinders V<sub>0</sub>-V<sub>1</sub> complex assembly. Similar structural rearrangements of the yeast V<sub>0</sub>a cytosolic domain have been described to be essential for V<sub>0</sub>-V<sub>1</sub> assembly/disassembly<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Atomic-level molecular dynamics simulations of the V<sub>0</sub> region in different lipid membrane compositions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371084_41467_2025_63125_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f66c/12371084/532ac1760d83/41467_2025_63125_Fig5_HTML.jpg" loading="lazy" id="d33e1222" height="995" width="663" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Top and side view of the Golgi V-ATPase V<sub>0</sub> region. The V<sub>0</sub>a2 subunit is divided into three regions: a2-body (grey), a2-arm (yellow), and a2-head (green). The d1 subunit is shown in magenta. The c-ring is shown in cyan. <strong>b</strong> Graphical representation of the different membrane lipid compositions used in the simulations. <strong>c</strong> Clustering of the simulation data using Bayesian Gaussian Mixture Models (BGMM) reveals four distinct clusters, each representing a different structural class. <strong>d</strong> Representative top view of the V<sub>0</sub> region from each cluster. For each cluster, the structure with the smallest distance from the mean of the Gaussian in the BGMM is presented. <strong>e</strong> Scatter plot (red line = mean) showing number of interactions formed by the V<sub>0</sub>a2-head with the V<sub>0</sub>d1 subunit in each cluster. <strong>f</strong> Schematic representation of the active conformation (cluster 2) and the inactive conformations, where the V<sub>0</sub>a2-head interacts with the V<sub>0</sub>d1 subunit. <strong>g</strong> Scatter plot (red line = mean) showing number of interactions between the V<sub>0</sub>a2-head and PI(4)P molecules in each cluster. <strong>h</strong> Probability (relative population) of cluster 2 with each membrane lipid composition. <strong>i</strong> Scatter plot (red line = mean) showing number of interactions between the V<sub>0</sub>a2-head and the V<sub>0</sub>d1 subunit in each lipid composition. <strong>j</strong> Occupational density maps of PI(4)P (orange) and cholesterol (green) densities in the vicinity of the V<sub>0</sub> region and c-ring in a representative inactive and active conformation from a top view. Insets from representative images shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">6d</a>. Red arrows indicate the pocket between the a2 subunit and the c-ring. <strong>k</strong> Schematic representation of the potentially clashing regions between the V<sub>0</sub>a2 arm and the V<sub>1</sub> region. The V<sub>0</sub> region is based on the atomistic simulations data, which is fitted one by one on the cryo-EM structure 6WM2 of the human V-ATPase. <strong>l</strong> Scatter plot (red line = mean) showing number of clashes between the V<sub>0</sub>a2-arm from each cluster with the V<sub>1</sub> region. <strong>m</strong> Scatter plot (red line = mean) showing number of clashes between the V<sub>0</sub>a2-arm from each lipid composition with the V<sub>1</sub> region.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Description of the simulated model systems</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Simulation model</th>
<th colspan="1" rowspan="1">POPC (mol%)</th>
<th colspan="1" rowspan="1">PI(4)P (mol%)</th>
<th colspan="1" rowspan="1">Cholesterol (mol%)</th>
<th colspan="1" rowspan="1">Simulation time (ns)</th>
<th colspan="1" rowspan="1">Replicas</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">V<sub>0</sub>
</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">750</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">V<sub>0</sub>−20PI4</td>
<td align="left" colspan="1" rowspan="1">80</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">750</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">V<sub>0</sub>−20PI4-10Chol</td>
<td align="left" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">750</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">V<sub>0</sub>−10PI4-20Chol</td>
<td align="left" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">750</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par22">Analysis of the simulation data showed that among the four conformational clusters detected, cluster 2 plays a crucial role in the assembly process, as in these conformations the a2 head formed the fewest interactions with the d1 subunit (Fig. <a href="#Fig5" class="usa-link">5e</a>). Thus, cluster 2 characterizes conformations of the V<sub>0</sub> region that are prone to bind to the V<sub>1</sub> region to form a functional V<sub>0</sub>-V<sub>1</sub> complex. Notably, these conformations differ from those in the other clusters (Fig. <a href="#Fig5" class="usa-link">5f</a>). PLS-based discriminant analysis of the simulation data (Supplementary Movie <a href="#MOESM5" class="usa-link">1</a>, <a href="#MOESM6" class="usa-link">2</a>) showed that, compared to the other clusters, in cluster 2 the arm and head of the a2 subunit are positioned close to the membrane and do not sterically interfere with the binding orientation of the V<sub>1</sub> region. The V<sub>0</sub>a2 head can interact with PI(4)P in the membrane, as described previously<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. In line with this, our data show that in cluster 2 the number of interactions between the a2 head and PI(4)P molecules in the membrane is the highest (Fig. <a href="#Fig5" class="usa-link">5g</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">High level of PI(4)P suppresses the conformational state of the V<sub>0</sub> region that promotes V<sub>0</sub>–V<sub>1</sub> complex assembly</h3>
<p id="Par23">We next analyzed the probability of occurrence of cluster 2 in membranes of different lipid compositions (Fig. <a href="#Fig5" class="usa-link">5h</a>). We found that the cluster 2 population is minimized (&lt;10% of conformations) when a cholesterol-free membrane is enriched with ample (20 mol%) PI(4)P. When 10 mol% cholesterol is added to this membrane, the population of cluster 2 increases to ~24%. The population of cluster 2 is at its highest (40%) when the concentration of PI(4)P is reduced to 10 mol% and in parallel cholesterol is increased to 20 mol%. At 10 mol% PI(4)P and 20 mol% cholesterol, the V<sub>0</sub>a2 head has the least interactions with the d1 subunit (Fig. <a href="#Fig5" class="usa-link">5i</a>), thus supporting a conformation that can form a complex with the V<sub>1</sub> region. These data imply that in the absence of cholesterol, high PI(4)P suppresses the assembly-promoting conformational state of V<sub>0</sub> region, while cholesterol plays an assembly-promoting, rescuing role.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Cholesterol blocks the access of PI(4)P deep into the pocket between the a2-subunit and the c-ring</h3>
<p id="Par24">From the atomic-level simulation data, we elucidated the mechanisms of PI(4)P and cholesterol action by focusing on their densities in the immediate vicinity of the V<sub>0</sub> region (Supplementary Fig. <a href="#MOESM1" class="usa-link">7d</a>). In membranes with high PI(4)P and without cholesterol (mimicking Sac1 degradation), we found that PI(4)P in the cytosolic membrane leaflet occupies the pocket between the a2 body and the c-ring, where it can interact with the a2 arm, a2 body, and the c-ring (Fig. <a href="#Fig5" class="usa-link">5j</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7d</a>). In cholesterol and PI(4)P containing membranes, cholesterol prevents PI(4)P from entering the pocket between the a2 body and the c-ring (Fig. <a href="#Fig5" class="usa-link">5j</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7d</a>), promoting an active conformation of the V<sub>0</sub> region. Under these conditions, potential salt bridges can be observed in the pocket between the a2 subunit and the c-ring [between E81 (a2 arm) – R436 (a2 body) and D76 (a2 arm) – R126 (c-ring), Supplementary Fig. <a href="#MOESM1" class="usa-link">6e, f</a>]. While these salt bridges appear most frequently in cluster 2 conformations (Supplementary Fig. <a href="#MOESM1" class="usa-link">7e, f</a>), their role(s) in the conformational changes of the V<sub>0</sub>a2-head remain to be elucidated.</p>
<p id="Par25">To further investigate how the lipid dependent conformational changes of the V<sub>0</sub>a2 arm affect V<sub>0</sub>-V<sub>1</sub> complex formation, we assessed the compatibility of the V<sub>0</sub> sector to form a docked V<sub>0</sub>-V<sub>1</sub> complex. We fitted the simulated V<sub>0</sub> conformations to the cryo-EM structure<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> and counted the number of clashes, i.e. atom-atom contacts, between the simulated V<sub>0</sub> structure and the V<sub>1</sub> region of the cryo-EM structure (Fig. <a href="#Fig5" class="usa-link">5k</a>). This showed that the active conformation promoting cluster 2 has the lowest number of clashes between the V<sub>0</sub> and V<sub>1</sub> sectors (Fig. <a href="#Fig5" class="usa-link">5l</a>), as would be expected. Moreover, a combination of 10 mol% PI(4)P with 20 mol% cholesterol in the membrane is associated with the least clashes (Fig. <a href="#Fig5" class="usa-link">5m</a>), thus favoring V<sub>0</sub>-V<sub>1</sub> complex assembly. Together, these data suggest that the Golgi V-ATPase assembly is highly sensitive to its membrane environment. In membranes mimicking the Golgi lipid environment upon Sac1 degradation, PI(4)P accumulates in the pocket between the a2 body and the c-ring, due to insufficient cholesterol to block it. This reduces the propensity to reach a V<sub>0</sub> conformation that is capable of associating with the V<sub>1</sub> region.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Sac1 safeguards hCG processing in human trophoblasts</h3>
<p id="Par26">Our experimental findings identified an essential role for Sac1 in maintaining Golgi morphology and terminal glycosylation via V-ATPase mediated pH regulation. We next challenged the physiological relevance of this idea. As Sac1 loss in mouse is embryonically lethal<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, variations in the <em>SACM1L</em> gene might not be well tolerated in humans. Indeed, based on the Genome Aggregation Database (gnomAD) the probability of the <em>SACM1L</em> gene for being loss-of-function intolerant (lof.pLI) is high, 0.97 (Fig. <a href="#Fig6" class="usa-link">6a</a>). Additionally, the observed number of loss-of-function <em>SACM1L</em> variants is only 27% of the expected (o/e = 0.27, 90% CI: 0.15-0.51), consistent with negative selection acting on these variants in humans and with an essential role of Sac1 also in human development.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Sac1 depletion impairs hCG secretion in trophoblasts.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371084_41467_2025_63125_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f66c/12371084/2a968b56b396/41467_2025_63125_Fig6_HTML.jpg" loading="lazy" id="d33e1524" height="814" width="667" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Density plot of gnomAD dataset showing the frequency of genes (y-axis) of having the probability of being loss-of-function intolerant (lof.pLI, x-axis). The orange dotted line indicates the 0.9 threshold commonly used for Mendelian diseases and the red dotted line the location of the <em>SACM1L</em> gene. <strong>b</strong> Schematic overview of differentiation of trophoblasts from Sac1-degron hESCs. Sac1-degron hESCs are subsequently differentiated into trophoblasts (TB) and syncytiotrophoblasts (STB). IAA treatment can induce Sac1 degradation in both TB and STB. <strong>c</strong> Representative confocal images of Sac1-degron STB treated for 4 h with IAA and stained for GM130 and TGN46. <strong>d</strong> Western blot analysis of Sac1-degron STB treated for 4 h with DMSO or IAA and 50 µM EN6 and blotted for TGN46 and B4GALT1. <strong>e</strong> Bar graph depicting the TGN46 or B4GALT1 degradation efficiency calculated from (<strong>d</strong>). Data from 3 biological replicates. One-way ANOVA with Tukey’s multiple comparisons test. TGN46: n.s. p = 0.7079, * p = 0.0112, p = 0.0278. B4GALT1: ** p = 0.005, **** p &lt; 0.0001. <strong>f</strong> Violin plot showing the distribution of average PI(4)P staining intensity in the TGN. n &gt; 100 cells. One-way ANOVA with Dunnett’s multiple comparisons test. **** p &lt; 0.0001. <strong>g</strong> Violin plot showing normalized filipin intensity in the TGN area (excluding lysosomal signal). Data from 3 independent experiments. Two-tailed Mann-Whitney U test. **** p &lt; 0.0001. <strong>h</strong> Representative confocal images of Sac1-degron STBs treated for 2 h with DMSO or IAA, stained for ATP6V<sub>0</sub>a2 (green) and ATP6V<sub>1</sub>A (magenta). <strong>i</strong> Western blot analysis of cell lysate and secreted medium of Sac1-degron STB treated with DMSO or IAA for 15 h, with or without Neuraminidase A (Neu A) digestion and blotted for hCGβ. n = 3 independent experiments. <strong>j</strong> Western blot analysis of secreted medium of Sac1-degron STBs treated with DMSO, IAA or IAA for 15 h, with or without Neuraminidase A digestion. Blotted for hCGβ. n = 3 independent experiments.</p></figcaption></figure><p id="Par27">During the early stages of mammalian embryogenesis, secretion of growth factors, hormones and cytokines orchestrates close communication between the blastocyst and the uterine wall. Particularly, human chorionic gonadotropin (hCG), a trophoblast-secreted glycoprotein, is required for implantation and gestation<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. To investigate if Sac1 is important for hCG secretion by trophoblasts, we differentiated human embryonic stem cells (hESCs) carrying the Sac1-degron system<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> into trophoblasts (TB) (Fig. <a href="#Fig6" class="usa-link">6b</a>). After 10 days of differentiation, the trophoblast markers hCGβ (beta-subunit of hCG), GATA3 and Dab2 were readily detected (Supplementary Fig. <a href="#MOESM1" class="usa-link">8a</a>), and multi-nucleated syncytiotrophoblasts (STB) were obtained after an additional 6 days of differentiation. Importantly, the levels of hCGβ increased drastically during differentiation, as anticipated (Supplementary Fig. <a href="#MOESM1" class="usa-link">8b</a>).</p>
<p id="Par28">Acute depletion of Sac1 from trophoblasts led to a severe fragmentation of the Golgi by 4 h (Fig. <a href="#Fig6" class="usa-link">6c</a>), followed by reduced protein levels of TGN46 and B4GALT1 (Supplementary Fig. <a href="#MOESM1" class="usa-link">8c, d</a>). Remarkably, when EN6 was added to the trophoblasts together with IAA, the TGN46 protein levels could be restored and B4GALT1 levels were partially rescued (Fig. <a href="#Fig6" class="usa-link">6d, e</a>). At 1-2 h of Sac1 degradation, PI(4)P levels rapidly increased in the TGN (Fig. <a href="#Fig6" class="usa-link">6f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">8e</a>), while TGN cholesterol levels decreased (Fig. <a href="#Fig6" class="usa-link">6g</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">8f, g</a>). Sac1 degradation in trophoblasts also acutely induced V-ATPase disassembly within 2 h (Fig. <a href="#Fig6" class="usa-link">6h</a>).</p>
<p id="Par29">Next, we assessed the impact of Sac1 depletion on trophoblast hCGβ processing and secretion. At 15 h upon Sac1 removal, the cellular levels of hCGβ were increased and the highly glycosylated hCGβ, abundantly secreted in normal conditions, was massively reduced in the medium (Fig. <a href="#Fig6" class="usa-link">6i, j</a>). Neuraminidase treatment of hCGβ increased the antibody reactivity and showed that hCGβ was abundantly sialylated in control trophoblasts and that some residual sialylation took place in Sac1 depleted cells. However, upon Sac1 degradation, hCGβ with a lower molecular weight was secreted, implicating a glycosylation defect (Fig. <a href="#Fig6" class="usa-link">6j</a>). Overall, this demonstrates that key phenotypes observed upon acute Sac1 degradation are recapitulated in differentiated trophoblasts and highlights the Golgi V-ATPase complex as a key effector of Sac1 activity that safeguards hCGβ processing and secretion in trophoblasts.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par30">This study employs the potential of AID in elucidating the early functional consequences of eliminating a vital protein, Sac1. The rapid, tightly controlled degradation of Sac1 enabled us to systematically follow the time course of ensuing defects, thereby revealing underlying causalities. In approximately one hour, Sac1 was fully degraded, followed by a ~ 3-fold increase in PI(4)P concentrated in the TGN area and a decrease in free cholesterol levels in the TGN during the following hour, likely due to OSBP inhibition. These changes in the TGN lipid environment prevent the assembly of the Golgi V-ATPase complex as observed from 2 h onwards, subsequently elevating TGN pH from ~5.9 to ~6.3 and resulting in disruption of terminal glycosylation and selective TGN degradation by 4 h (Fig. <a href="#Fig7" class="usa-link">7</a>). Mechanistically, the changes in the lipid environment of the TGN upon acute Sac1 depletion induce conformational changes in the V-ATPase V<sub>0</sub> region, inhibiting V<sub>0</sub>-V<sub>1</sub> complex assembly. Importantly, the key phenotypes ensuing from Sac1 degradation could be recapitulated in human differentiated trophoblasts, resulting in defects in hCGβ glycosylation and secretion. This may in part explain the preimplantation lethality of Sac1 null mice and rationalizes the intolerance for Sac1 loss-of-function during mammalian development.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. Working model for Sac1 controlled TGN V-ATPase activity and downstream effects.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371084_41467_2025_63125_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f66c/12371084/fdb0625369fb/41467_2025_63125_Fig7_HTML.jpg" loading="lazy" id="d33e1604" height="504" width="672" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Sac1 fuelled PI(4)P/cholesterol exchange maintains the composition of the Golgi membrane, allowing V-ATPase assembly for TGN acidification, proper terminal glycosylation and cargo secretion. Rapidly after Sac1 degradation, PI(4)P is accumulated in the TGN while cholesterol is depleted. This changes the lipid microenvironment of the V<sub>0</sub> region, resulting in a conformational change in the V<sub>0</sub>a2 subunit which prevents the assembly of the Golgi V-ATPase complex. This results in the deacidification of the TGN, Golgi fragmentation and defects in cargo processing and secretion.</p></figcaption></figure><p id="Par31">The most direct effect of loss of Sac1 function is the accumulation of its ligand PI(4)P that facilitates cargo trafficking from the TGN<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Importantly, we provide evidence that Sac1 is needed not only to maintain PI(4)P levels, but also the high cholesterol content in the TGN, in line with the critical role of Sac1 in fuelling PI(4)P/cholesterol exchange by OSBP at ER-TGN contact sites<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. As complex interactions between lipid transfer proteins control cholesterol and PI(4)P levels in the TGN<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, Sac1 most likely affects a variety of lipid transfer proteins beyond OSBP. The upregulation of several lipid transporters, observed in our proteomics data, probably serves to compensate for the acute loss of Sac1. However, this is clearly insufficient to maintain the normal ER-TGN (and ER-plasma membrane) cholesterol gradient without Sac1 burning off PI(4)P. Hence, the critical role of Sac1 in governing cellular cholesterol distribution contributes to its vital importance. Interestingly, Sac1 can interact with the ER cholesterol sensor SCAP, suggesting an intricate coregulation between Sac1 fuelled PI(4)P/cholesterol exchange and transcriptional control of cholesterol homeostasis<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Besides cholesterol, PI(4)P is involved in counter exchange with other lipids, such as phosphatidylserine<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Thus, over time Sac1 depletion will challenge lipid homeostasis broadly.</p>
<p id="Par32">The present study highlights the TGN membrane lipid dependent control of Golgi V-ATPase activity via Sac1 as an early, shared denominator and driver for the later phenotypes, i.e. Golgi deacidification, fragmentation and degradation, as well as cargo glycosylation and secretion defects. This notion is supported by the observations that TGN fragmentation, protein degradation and defective cargo secretion could be partly rescued by pharmacological re-acidification. It also agrees with the established role of V-ATPase in safeguarding Golgi glycosylation and secretion<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Interestingly, mutations in V<sub>0</sub>a2 are also linked to trafficking and glycosylation defects and can lead to connective tissue disorders, such as wrinkly skin syndrome and cutis laxa<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. As Sac1 activity has also been implicated in cargo secretion from the TGN<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, the coupling of TGN carrier formation to V-ATPase activity may help to coordinate terminal glycosylation and secretion.</p>
<p id="Par33">V-ATPases are key regulators of organelle acidification conserved from plants to humans. The regulation of V-ATPases has been best characterized in <em>Saccharomyces cerevisiae</em>, where the assembly/disassembly of V<sub>0</sub> and V<sub>1</sub> regions rapidly adjusts proton pumping activity, with e.g. glucose starvation stimulating complex disassembly and conservation of energy<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. In mammalian cells, the cues controlling the reversible association of V<sub>0</sub> and V<sub>1</sub> regions may be more context dependent, with e.g. glucose removal promoting either assembly or dissociation<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, and studies have mainly focused on the lysosomal V-ATPase<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p>
<p id="Par34">Despite V-ATPases being large integral membrane protein complexes, there is limited understanding on how they are functionally regulated by lipids. In yeast, V-ATPase activity can be modulated by phosphoinositides, phospholipids and ergosterol<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>–<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. The human V<sub>0</sub>a-subunits were shown to bind to distinct phosphoinositides, with the endo-lysosomal isoform binding to PI(3)P and PI(3,5)P<sub>2</sub> and the Golgi isoform to PI(4)P in vitro<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Importantly, the cryo-EM structures of human V-ATPase reveal several lipids, including phospholipids and cholesterol, as integral parts of the V<sub>0</sub> region<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>.</p>
<p id="Par35">The present study complements and extends these findings by identifying Sac1 as a gatekeeper whose activity controls, via effects on the TGN membrane lipid environment, the conformation and activity of Golgi V-ATPase in the cellular context. Specifically, we provide evidence that the dissociation of the V<sub>0</sub> and V<sub>1</sub> regions is promoted by increased PI(4)P and decreased cholesterol in the TGN membrane upon Sac1 removal. Atomic-level simulations illuminate the mechanistic role of cholesterol in stabilizing the assembly-promoting conformation of the V<sub>0</sub> region. In the absence of cholesterol, PI(4)P can move into the recess between the a2 subunit and the c-ring of the V<sub>0</sub> region and thus hinder the assembly-promoting conformation. The assembly-disrupting conformations lead to binding of the a2 head to the d1 subunit, which sterically prevents the V<sub>1</sub> region from assembling with the V<sub>0</sub> region. Cholesterol, on the other hand, prevents PI(4)P from entering deep into this recess, thereby promoting assembly.</p>
<p id="Par36">Based on our data, the V-ATPase assembly status was more sensitive to loss of Sac1 function at Golgi than lysosomal membranes: upon 2 h of Sac1 removal - when PI(4)P was the only phosphoinositide increased – the V<sub>1</sub> region became disassembled from the Golgi V<sub>0</sub> region but the localization of the V<sub>1</sub> region in lysosomal membranes appeared intact. Moreover, at 4 h of Sac1 removal, TGN46 degradation was still sensitive to lysosomal V<sub>0</sub>-V<sub>1</sub> dissociation by BafA1. The early affliction of Golgi V-ATPase may be related to different phosphoinositide sensitivities of the V-ATPases, but possibly also to a predominant activity of Sac1 at Golgi contacts.</p>
<p id="Par37">The initial effects of Sac1 removal that we focused on, centred on the TGN. Yet, other compartments, such as endo-lysosomes, are likely affected over time. Interestingly, recent data implicate lysosomal PI(4)P, generated by local synthesis and turned over by Sac1, in lysosomal membrane repair<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>,<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> and rewiring of lysosomal degradation during starvation<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. We found all the major PIPs to be elevated by 4 h of Sac1 removal. As PIPs are rapidly hydrolyzed and phosphorylated by specific kinases and phosphatases, the initial increase of PI(4)P is expected to result in the interconversion to other PIP species over time<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. The subcellular localization of individual PIP species is important for organelle specific membrane properties<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, and its dysregulation leads to defects in migration, autophagy, endo-lysosomal trafficking and signaling via PIP binding proteins<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>,<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. Thus, a broad PIP imbalance affecting multiple endomembranes and PIP dependent lipid transfer processes may contribute to the long-term effects of Sac1 depletion. Moreover, considering the coupling between PIP and cholesterol distribution, it is likely that cholesterol and PIPs co-operate in the regulation of several membrane proteins, including V-ATPase, in other compartments.</p>
<p id="Par38">In conclusion, this study reveals that in human cells, the Golgi V-ATPase assembly status is dynamically modulated by membrane lipids, specifically by the TGN PI(4)P and cholesterol pools that are physiologically co-regulated by Sac1 activity. The multiple modes of lipid – V-ATPase interactions and how they regulate V-ATPase activity will be important topics for future work.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Cultivation of A431, HEK293A and A549 cells and generation of cell lines</h3>
<p id="Par39">A431 (ATCC CRL-1555) and HEK293A (Invitrogen <a href="https://www.ncbi.nlm.nih.gov/nuccore/R70507" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">R70507</a>) cells were cultured in DMEM (high-glucose, Gibco, 21969) and A549 (ATCC CCL-185) cells were cultured in F-12 Nutrient Mixture (Gibco, 21127-022). Media was supplemented with 10% FBS, penicillin/streptomycin (100 U ml<sup>−1</sup>) and L-glutamine (2 mM). Cells were cultured at 37 °C in 5% CO<sub>2</sub>. Human Embryonic Stem cells (hESCs) (H9, WiCell, WIC-WA09-RB-001) were cultured in mTeSR Plus culture medium (Stem Cell Technologies, 100-0276) and plated on Matrigel (Corning, 356231) coated dishes, diluted 1:200 in DMEM/F-12 (Gibco, 31331-028). For passaging, hESCs were washed 2x with 1x PBS (Corning, 21-040-cv) and treated with 0.5 µM EDTA (Invitrogen, 15575020) for 5 min at room temperature (RT). Sac1-degron cell lines (A431, A549, HEK293A and hESCs) were generated as previously described<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">hESC culture and trophoblasts differentiation</h3>
<p id="Par40">Wild-type and Sac1-degron H9 hESCs were maintained in mTeSR Plus medium on Matrigel-coated plates. Differentiation into trophoblasts was performed by using the differentiation protocol previously described<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. In brief, hESCs were seeded at 1:4 ratio onto Matrigel-coated 6-well plates in the presence of 10 μM ROCK inhibitor (Selleckchem, Y-27632). The following day the cells were rinsed with DMEM/F-12 and transferred into TB differentiation medium (DMEM/F-12 supplemented with penicillin/streptomycin 100 U ml<sup>−1</sup> each, 0.2% FBS, 0.3% fatty acid free BSA (Sigma-Aldrich, A-3803), 1% Insulin-Transferrin-Selenium-Ethanolamine (ITS-X) (Gibco, 51500056), 0.1 mM β-Mercaptoethanol (Roth, 4227.1), 1.5 µg ml<sup>−1</sup> L-ascorbic acid (Sigma-Aldrich, A92902), 50 ng ml<sup>−1</sup> EGF (PreproTech, AF-100-15), 2 µM CHIR99021 (Tocris, 4423), 0.5 µM A83-01 (Selleckchem, S7692), 1 µM SB431524 (Selleckchem, S1067), 0.8 mM Valproic acid (MedChemExpress, HY-10585), 5 μM ROCK inhibitor, and 10 ng ml<sup>−1</sup> BMP4 (R&amp;D System,s 314-BP)). The medium was replaced every 2 days and the cells routinely passaged with StemPro Accutase (ThermoFisher, A1110501) at 1:3 ratio after reaching 90-100% confluency. From day 9-10 onwards, the cells were used for experiments or further differentiated to syncytiotrophoblasts (STB). For STB differentiation, trophoblasts were dissociated with StemPro Accutase into 1-cell suspension in TB medium, counted, centrifuged at 200x<em>g</em> for 3 min and resuspended in STB medium (DMEM/F-12 supplemented with penicillin/streptomycin 100 U ml<sup>−1</sup> each, 0.3% fatty acid free BSA, 1% ITS-X, 0.1 mM β-Mercaptoethanol, 10 mM Forskolin (Sigma-Aldrich, F6886), 4% KnockOut Serum Replacement (Gibco, 10828028), 50 ng ml<sup>−1</sup> EGF, 1 µM cAMP (Sigma-Aldrich, D0260), and 5 μM ROCK inhibitor and seeded at 1x10<sup>5</sup> cells per well on Matrigel-coated 6-well plates. The medium was replaced after 3 days, and the cells were used for experiments on day 6 after STB differentiation.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Antibodies and reagents</h3>
<p id="Par41">Primary antibodies for western blot (WB) or immunofluorescence (IF) analysis: Rabbit polyclonal anti-TGN46 (Abcam, ab50595, IF 1:400, WB 1:1000), mouse purified anti-PI4P IgM (Echelon, Z-P004, IF 1:200), mouse monoclonal anti-GM130 (BD Biosciences, G56120-150-1, IF 1:500. WB 1:2000), rabbit polyclonal anti-GM130 (Sigma-Aldrich, G7295, IF 1:300), rabbit polyclonal anti-GRASP55 (Proteintech 10598-1-AP, WB 1:1000), polyclonal goat anti-B4GALT1 (R&amp;D systems, AF3609, IF 1:100, WB: 1:500), anti-GFP (Abcam, ab290, WB 1:5000), mouse anti-miniIAA7 (customized from Genscript<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>), rabbit monoclonal anti-ATP6V<sub>1</sub>A1 (Abcam, ab199326, IF 1:250, WB 1:2000), rabbit polyclonal anti-ATP6V<sub>0</sub>d1 (ProteinTech, 18274-1-AP, WB 1:500), mouse polyclonal anti-ATP6V<sub>0</sub>a2 (Abnova, H00023545-A01, IF 1:200), rabbit polyclonal anti-ATP6V<sub>0</sub>a2 (Invitrogen, PA5-85284, WB 1:2000), goat polyclonal anti-LAMP2 (R&amp;D systems, AF6228, IF 1:40), rabbit polyclonal anti-TMEM192 (Sigma-Aldrich, HPA014717, IF 1:200), DAPI (Sigma-Aldrich, D9542, IF 1:1000), rabbit polyclonal anti-hCGβ (ProteinTech, 11615-1-AP, IF 1:200), Rabbit monoclonal anti-GATA-3 (Cell Signaling Technology, #5852, IF 1:500), rabbit monoclonal anti-DAB2 (Cell Signaling Technology, #12906, IF 1:500).</p>
<p id="Par42">Secondary antibodies for IF: Alexa Fluor<sup>TM</sup> 488 and 568 Goat anti-Mouse and anti-Rabbit IgG (Molecular Probes, A11008, A11011, A11001, A11004, 1:250), Alexa Fluor<sup>TM</sup> 647 Donkey anti-Rabbit and anti-Mouse 647 IgG (Molecular probes, A31571, <a href="https://www.ncbi.nlm.nih.gov/protein/A31573" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A31573</a>, 1:250), Alexa Fluor<sup>TM</sup> 488 Goat anti-Mouse IgM (ThermoFisher, A21042, 1:250). Secondary antibodies for WB analysis: HRP conjugated antibodies Goat anti-Mouse (Bio-Rad, 1706516, 1:2000), Goat anti-Rabbit (Bio-Rad, 1706515, 1:1000) and Donkey anti-Goat (Jackson Immuno research, 705-035-147 1:5000). Reagents: cvx_IAA (final concentration 5 µM) and cvx_picoIAA (final concentration 0.5 µM)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, DMSO (Santa Cruz, sc-358801), 1 µM Bafilomycin A1 (AppliChem c22229915), 20 nM OSW-1 (Bocsci 145075-81-6, stock 10 mM), 50 µM EN6 (Sigma-Aldrich, SML2689), 10 µM PIK93 (TargetMol, T-12454), 10 µM NC03 (Aobious, <a href="https://www.ncbi.nlm.nih.gov/protein/AOB17420" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AOB17420</a>), 4 µM Methyl-β cyclodextrin (Sigma-Aldrich, C-4555), 50 µM 8.2:1 methyl-β-cyclodextrin/cholesterol complex (Sigma-Aldrich C-4555, C-8667).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Sac1<sup>wt</sup> and Sac1<sup>mut</sup> plasmid generation</h3>
<p id="Par43">Plasmids harboring wild-type Sac1 (Sac1<sup>wt</sup>) and a phosphatase-dead mutant Sac1 (C389S, Sac1<sup>mut</sup>) were obtained from Addgene (#108127 and #108128, respectively). The Sac1 fragments were amplified by PCR (Fw: GGAAGATCTATGGCGACGGCGGCCTA, Rev: AAAACTGCAGTCAGTCTATCTTTTCTTTCTGGACCAGTCTGG) and cloned into the pEGFP-C1 vector. The resulting plasmids, along with pcDNA3.1 as control, were transfected into Sac1-degron HEK293A cells using X-tremeGENE™ HP DNA Transfection Reagent (Roche, 06 366 236 001). The transfection occurred overnight, after which IAA or DMSO was added for 6 h. Subsequently, cells were lysed and subjected to Western blot analysis.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Phosphoinositide regioisomer measurement by chiral column chromatography and mass spectrometry (PRMC-MS)</h3>
<p id="Par44">A431 cells were washed twice with cold 1x PBS. Cells were scraped in 500 ml cold 1x PBS. 2.5 × 10<sup>6</sup> cells were divided over 1.5 ml tubes and centrifuged for 5 min at 1000x<em>g</em> at 4 °C. Cell pellets were frozen down and transferred to Tokyo Medical and Dental University (TMDU) for further processing for PRMC-MS analysis. Cellular phosphoinositide levels were measured according to<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Briefly, acidic phospholipids extracted from control and Sac1-depleted A431 cells were derivatized by a methylation reaction with trimethylsilyl diazomethane (Tokyo Chemical Industry, #T1146), followed by phosphoinositide regioisomer measurement by chiral chromatography and mass spectrometry. LC-MS/MS was performed using a triple quadruple mass spectrometer QTRAP6500 (ABSciex) and a Nexera X2 HPLC system (Shimadzu) combined with a PAL HTC-xt (CTC Analytics) autosampler. The obtained chromatographic data were analyzed by MultiQuant 3.0.2 software (ABSciex). Concentrations of phosphoinositide species were estimated based on the concentration of C37:4 species of the corresponding phosphoinositide regioisomers that had been added to the cell pellets as internal standards before the lipid extraction procedure.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Lipid extraction and CAMAG TLC</h3>
<p id="Par45">Lipid extraction according to Bligh and Dyer<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. In short, cells are washed 2x with cold 1x PBS and scraped in 2% NaCl. Subsequently, samples were homogenized using methanol (Fluka, 65543), chloroform (Merck, 34854) and ultra-pure water. Samples were centrifuged for phase separation of the lipids in the organic phase. Finally, samples were evaporated and resuspended in a chloroform/methanol (2:1) resuspension solution. 100 µg of protein was used to run a HPTLC Silica gel plate (Sigma-Aldrich, 1.05642.0001) in combination with a neutral lipid standard (1 mg ml<sup>−1</sup> FC:CE:TG) on a CAMAG Automatic TLC Sampler 4 using the visionCATS software. A running solution for neutral lipids containing 80:20:1 hexane-diethyl ether-acetic acid (Honeywell, 34859, J.T. Baker 8254, Fluka 33209) was used and the silica plate was subsequently stained with 3% CuSO<sub>4</sub> / 8% H<sub>3</sub>PO<sub>4</sub> (Merck 1.02790.0250, Fisher Scientific, 10685082). The silica plate was charred on a stove up to 180 °C, and the plate was scanned to assess free cholesterol (FC), cholesteryl ester (CE) and triglyceride (TG) lipid content.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Click-labeling</h3>
<p id="Par46">Quadruplicates of Sac1-degron A431 cells treated with DMSO or IAA for 2 h were labelled with 100 µM alkyne-Oleic Acid-BSA complex (0.33 mM stock complexed with BSA in 8:1 molar ratio, prepared in serum free DMEM) in CO<sub>2</sub> independent medium (Gibco, 18045) for 30 min. After labeling, cells were washed 1x with 1% FA-free BSA, 1x with ice-cold 1x PBS and 1x with 155 mM ammonium acetate. Lipids were extracted with a chloroform:methanol (6:1) mix with internal standards (D8-Click IS, 13C3 IS and OMICS Click IS<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>) and sonicated for 60 seconds. Chloroform and 1% acetic acid were added, and samples were vortexed and centrifuged. Lower phase was evaporated and analyzed by Mass Spectrometry<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">PI(4)P staining</h3>
<p id="Par47">Cells grown on coverslip were fixed with 2% paraformaldehyde (Electron Microscopy Sciences, 15710) in 1x PBS for 15 min at RT. Subsequently, cells were permeabilized with 20 µM digitonin (Sigma-Aldrich, 11024-24-1) for 5 min and blocked for 1 h in 1% fatty acid free BSA in 1x PBS (Sigma-Aldrich, A3803). Cells were incubated for 2 h at 37 °C with mouse anti-PI(4)P IgM antibody, washed 3x with 1x PBS and incubated with Alexa Fluor<sup>TM</sup> 488 Goat anti-Mouse IgM secondary antibody. Coverslips were mounted on microscopy slides using Prolong glass antifade reagent (Invitrogen, <a href="https://www.ncbi.nlm.nih.gov/protein/P36980" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P36980</a>).</p></section><section id="Sec21"><h3 class="pmc_sec_title">mCherry-P4M-SidM live-imaging</h3>
<p id="Par48">Cells were transfected with mCherry-P4M-SidM using X-tremeGENE HP DNA transfection reagent according to manufacturer’s protocol. The day after transfection, live-cell imaging was performed on Zeiss LSM 880 confocal microscopy at 37 °C and 5% CO<sub>2</sub>. The grayscale images were visualized using ImageJ software with the ‘Physics’ lookup table applied to enhance contrast and highlight specific intensity ranges.</p></section><section id="Sec22"><h3 class="pmc_sec_title">ALOD4 plasma membrane staining</h3>
<p id="Par49">Recombinant mNeonGreen-ALOD4 was a gift from Arun Radhakrishnan (UT Southwestern). For plasma membrane ALOD4 staining, cells were grown on ibidi 15 µ-slide 8-well chamber slides (Ibidi, 80826) and stained with 0.5 µM mNeonGreen-ALOD4 (diluted in 1% fatty acid free BSA in cold 1x PBS) for 30 min on ice. Cells were washed twice with cold 1x PBS, fixed with 4% PFA for 15 min, and quenched with 50 mM NH<sub>4</sub>Cl in 1x PBS for 10 min at RT.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Filipin staining</h3>
<p id="Par50">For filipin staining, cells were grown on glass coverslips and fixed with 4% PFA. Cells were washed 2x with 1x PBS and incubated for 30 min at 37 °C with 0.5 mg ml<sup>−1</sup> filipin (Sigma-Aldrich, F9765) diluted in 1x PBS. Cells were washed 3x with 1x PBS and mounted on microscopy slides with Mowiol-DABCO (CalBioChem, 475904, Sigma D-2522). For better intracellular visibility of filipin staining, cells were treated for 30 min with 10 mM methyl-β-cyclodextrin at 37 °C according to<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. Cells were incubated for 30 min at RT with 0.5 mg ml<sup>−1</sup> filipin, blocked with 1% BSA for 30 min, and subsequently incubated with primary and secondary antibodies. After antibody staining, the cells were re-incubated with 0.5 mg ml<sup>−1</sup> filipin for 30 min at RT and mounted on microscopy slides with Mowiol-DABCO.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Immunofluorescence stainings</h3>
<p id="Par51">Cells grown on glass coverslips were fixed for 15 min at RT with 4% PFA (Sigma-Aldrich, P-441244) or 10 min with ice-cold methanol, permeabilized for 10 min with 0.1 – 0.2% Triton X-100 (Sigma-Aldrich, t-8787) in 1x PBS and blocked for 30 min with 1 to 3 % BSA (Sigma-Aldrich, A-3803) in 1x PBS. Cells were incubated with primary antibodies diluted in blocking buffer for 1 to 2 h at RT or 37 °C, followed by secondary antibody incubation for 1 h at RT. Coverslips were mounted on microscopy slides using Prolong glass antifade reagent (Invitrogen, <a href="https://www.ncbi.nlm.nih.gov/protein/P36980" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P36980</a>).</p></section><section id="Sec25"><h3 class="pmc_sec_title">Image acquisition</h3>
<p id="Par52">Imaging was performed at the Biomedicum Imaging Unit at the University of Helsinki, supported by the Helsinki Institute of Life Science (HiLIFE) and Biocenter Finland. For confocal imaging, a Zeiss LSM 880 inverted confocal microscope with Airyscan detector, 63x/1.40 PL Apo oil objective, Diode and Argon lasers was used. For FLIM and confocal imaging a Leica Stellaris 8 Falcon / DLS inverted confocal microscope, equipped with HyD detectors, 405 nm diode laser, and supercontinuum laser, with 63x/1.40 HC PL APO CS2 oil or 63x/1.20 HC PL APO CS2 water objective was used. For widefield imaging a NIKON Eclipse Ti-E inverted wide-field microscope, equipped with Led-DAPI-A, TxRed-4040C, LED-Cy5-A, LED-FITC-A filter cubes, Hamamatsu Orca Flash 4.0 V2 B&amp;W camera and Lumencor Spectra X light engine, with 40x/0.75 Plan Fluor dry objective, 60x/1.40 Plan Apo Oil or 100x/1.40 Plan Apo VC oil objective with Perfect Focus System was used. A Leica Upright DM6 B epifluorescence microscope was used for immunofluorescent staining of TB differentiation markers.</p></section><section id="Sec26"><h3 class="pmc_sec_title">TGN pH measurements using fluorescence lifetime microscopy (FLIM)</h3>
<p id="Par53">FLIM pH measurements were performed according to<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Briefly, Sac1-degron cells were transfected with GalT-RpHLuorin2, for <em>trans-</em>Golgi expression of pHLuorin2, (Addgene, #171719) using X-tremeGENE HP DNA transfection reagent according to manufacturer’s protocol. 5 h after transfection cells were seeded onto an 8-well Lab-Tek chambered cover glass (Thermo Fisher, 155411) coated with 5 µg ml<sup>−1</sup> fibronectin (Roche, 11051407001). FLIM imaging was performed on a Leica Stellaris 8 Falcon / DLS with an OKO lab full-enclosure incubator set at 37 °C, equipped with a 63x/1.20 HC PL APO CS2 water objective. GalT-pHLuorin2 was excited at 488 nm with a pulsed white light laser at 80 mHz. An HyD X detector was set at 502-530 nm, and photons were collected for 1 min. Lifetime histograms were fitted with a mono-exponential decay function. pH calibrations were performed before every experiment using an Intracellular pH Calibration Buffer kit (Thermo Fisher Scientific, <a href="https://www.ncbi.nlm.nih.gov/protein/P35379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P35379</a>) to generate a standard curve to determine pH levels.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Transmission electron microscopy</h3>
<p id="Par54">Cells grown on cover slips (thickness #1) were fixed with 2.5% glutaraldehyde (EM-grade, Sigma-Aldrich, G-7651) in 0.1 M sodium cacodylate buffer, pH 7.4, and flat embedded according to<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. Briefly, after 30 min fixation at RT, the cells were post-fixed with reduced 1% osmium tetroxide for 1 h on ice, gradually dehydrated in ethanol and acetone, and infiltrated into epoxy (TAAB 812 resin, T030). Sections ca. at height of Golgi were cut from the polymerized sample blocks, picked-up on single slot grids, and post-stained with uranyl acetate and lead citrate. TEM imaging was performed using Hitachi HT7800 microscope (Hitachi High-Technologies, Tokyo, Japan) operated at 100 kV. For unbiased analysis 10 cells were selected at low magnification and montage images were acquired covering the whole Golgi area at nominal magnification of 6000x (pixel size 1.84 nm) using a bottom mounted Rio9 CMOS-camera (AMETEK Gatan Inc., Pleasanton, CA) and SerialEM software (<a href="https://bio3d.colorado.edu/SerialEM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bio3d.colorado.edu/SerialEM</a>, version 3.8.1). The analysis of the Golgi morphology was done using pseudonymized TEM micrographs (MIB software<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>).</p></section><section id="Sec28"><h3 class="pmc_sec_title">Image quantification</h3>
<p id="Par55">Whole cell area, Golgi region or nuclei were segmented with interactive machine learning algorithms based software Ilastik (version 1.3.2)<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. Cellular PI4P (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>), ALOD4 (Supplementary Fig. <a href="#MOESM1" class="usa-link">4d</a>), Filipin (Supplementary Fig. <a href="#MOESM1" class="usa-link">4f</a>) and TGN46 (Fig. <a href="#Fig3" class="usa-link">3e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>), Golgi PI4P (Fig. <a href="#Fig4" class="usa-link">4a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7e</a>) and cholesterol (Fig. <a href="#Fig4" class="usa-link">4c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7f</a>), lysosomal cholesterol (Supplementary Figs. <a href="#MOESM1" class="usa-link">4i</a> and <a href="#MOESM1" class="usa-link">7</a>f), and Western blot intensities were measured using Fiji ImageJ (version 1.54 f). For colocalization analysis, binary images of ATP6V<sub>1</sub>A and ATP6V<sub>0</sub>a2 subunits (Figs. <a href="#Fig3" class="usa-link">3h</a> and <a href="#Fig4" class="usa-link">4</a>g) were obtained with Ilastik. Pearson’s correlation coefficient was calculated using the command ‘colocalization’ in Fiji ImageJ. The co-localization of ATP6V<sub>1</sub>A and ATP6V<sub>0</sub>a2 with the lysosomal marker LAMP2 and the Golgi marker GM130 was assessed in a similar way (Fig. <a href="#Fig3" class="usa-link">3j, l</a>).</p></section><section id="Sec29"><h3 class="pmc_sec_title">Quantitative proteomic analysis</h3>
<section id="Sec30"><h4 class="pmc_sec_title">Mass spectrometry</h4>
<p id="Par56">Mass spectrometry analysis was performed at the Turku Proteomics Facility, University of Turku and Åbo Akademi University, supported by Biocenter Finland. Quantitative proteomic analysis of 3 h DMSO and 5 µM cvxIAA treated A431 Sac1-degron cells was performed using biological triplicates. Cell samples were lysed, and proteins were denatured with 8 M urea in 50 mM Tris-HCl, pH 8. Samples were reduced with 10 mM D,L-dithiothreitol and alkylated with 40 mM iodoacetamide. Samples were digested overnight with sequencing grade modified trypsin (Promega). After digestion, peptides were desalted with a SepPak C18 96-well plate (Waters), evaporated to dryness and stored at −20 °C. Digested peptides were dissolved in 0.1% formic acid and peptide concentration was determined with a NanoDrop device (Thermo Fisher Scientific). The Liquid Chromatography Electrospray Ionization Tandem Mass Spectrometric (LC-ESI-MS/MS) analysis was performed on a nanoflow HPLC system (Thermo Fisher Scientific, Easy-nLC1200) coupled to the Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a nano-electrospray ionization source. 800 ng peptides samples were loaded and separated inline on a 25 cm C18 column (75 μm x 25 cm, ReproSil-Pur 1.9 μm 120 Å C18-AQ, Dr. Maisch HPLC GmbH, Ammerbuch-Entringen, Germany) heated to 60 °C. The mobile phase consisted of water with 0.1% formic acid (solvent A) or acetonitrile/water (80:20 (v/v)) with 0.1% formic acid (solvent B). A 170 min gradient was used to elute peptides (95 min from 5% to 21% solvent B followed by 75 min from 21% to 36% solvent B).</p>
<p id="Par57">Samples were analyzed by a data independent acquisition (DIA) method. MS data was acquired automatically by using Thermo Xcalibur 4.2 software (Thermo Fisher Scientific). In a FAIMS-DIA method a duty cycle contained two compensation voltages (−50V and −70V) with one full scan (400 −1000 m/z) and 30 DIA MS/MS scans covering the mass range 400 – 1000 with variable width isolation windows in each of the compensation voltages.</p></section><section id="Sec31"><h4 class="pmc_sec_title">Data processing parameters</h4>
<p id="Par58">Data analysis consisted of protein identifications and label free quantifications of protein abundances. Data was analyzed by Spectronaut software (Biognosys, version 15.6.2). DirectDIA approach was used to identify proteins and label-free quantifications were performed with MaxLFQ.</p>
<p id="Par59">Main data analysis parameters in Spectronaut:</p>
<p id="Par60">- Enzyme: Trypsin/P</p>
<p id="Par61">- Missed cleavages: 2</p>
<p id="Par62">- Fixed modifications: Carbamidomethyl (Peptides were alkylated with iodoacetamide)</p>
<p id="Par63">- Variable modifications: Acetyl (protein N-term) and oxidation (M)</p>
<p id="Par64">- Protein database: Swiss-Prot 2021_04 Homo sapiens</p>
<p id="Par65">- Precursor FDR Cutoff: 0.01</p>
<p id="Par66">- Protein FDR Cutoff: 0.01</p>
<p id="Par67">- Quantification MS level: MS2</p>
<p id="Par68">- Quantification type: Area under the curve within integration boundaries for each targeted ion</p>
<p id="Par69">- Normalization: Local normalization (Based on RT dependent local regression model described by<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>)</p>
<p id="Par70">- Differential abundance analysis: Unpaired Student’s t-test with combined MS1 + MS2 statistical model<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup> and multiple-testing correction of the p-values with Benjamini-Hochberg method (q-values).</p></section><section id="Sec32"><h4 class="pmc_sec_title">Gene Ontology (GO) Enrichment Analysis</h4>
<p id="Par71">GO Enrichment Analysis were performed online (<a href="https://geneontology.org/docs/go-enrichment-analysis/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://geneontology.org/docs/go-enrichment-analysis/</a>)<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>,<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. This service connects to the analysis tool from the PANTHER Classification System, which is kept up to date with GO annotations<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>. In brief, lists of significantly upregulated (p &lt; 0.05, FC &gt; 1.33 (cvxIAA 4 h/DMSO 4 h)) and downregulated (p &lt; 0.05, FC &lt; 0.75 (cvxIAA 4 h/DMSO 4 h)) proteins were subjected to GO enrichment analysis and categorized into two main GO categories: biological process (BP) and cellular component (CC). As a reference, the list of all genes of Homo sapiens was used. Fisher’s Exact Test was employed to identify significant enrichments with the False Discovery Rate (FDR) correction method to adjust p-value for multiple testing. FDR &lt; 0.05 was set as significance threshold.</p></section></section><section id="Sec33"><h3 class="pmc_sec_title">Western blotting</h3>
<p id="Par72">Cells were washed twice in 1x PBS and lysed in cold RIPA lysis buffer (1% NP-40, 0.1% SDS, 0.5% Sodium Deoxycholate in 1x TBS) with freshly added protease inhibitors (25 µg ml<sup>−1</sup> chymostatin (Sigma-Aldrich, C-7268), 25 µg ml<sup>−1</sup> leupeptin (Sigma-Aldrich, L-2023), 25 µg ml<sup>−1</sup> antipain hydrochloride (Sigma-Aldrich, A-6191) and 25 µg ml<sup>−1</sup> pepstatin A (Sigma-Aldrich, P-5318)). DC<sup>TM</sup> protein Assay Kit I (Bio-Rad, 5000111) was used to determine protein concentration. 5x Sample buffer containing β-Mercaptoethanol was added to the cell lysates to a final dilution of 1x and lysates were denatured for 1 h at 37 °C. 10 – 20 µg of protein was loaded on Mini-Protein TGX Stain-Free gels (Bio-Rad, 1610181, 1610183, 1610185) and run at 120 V in 1x SDS-Page Running buffer. Pageruler Plus prestained protein ladder (Thermo Fisher, 26619) was used. Gels were activated with the ChemiDoc MD Imaging System (Bio-Rad) and transferred to a 0.45 µm Low Fluorescence PVDF membrane (Bio-Rad, 1704274). Membranes were blocked for 45 min at RT with 5% skim milk (Valio) in 0.1% Tween-20 (Fisher Scientific, BP337) in TBS (TBS-t), and subsequently incubated with the primary antibody overnight at 4 °C. Membranes were washed 3x with TBS-t and incubated for 45 min at RT with an HRP-conjugated secondary antibody. Membranes were washed 3x with TBS-t, incubated for 1 min with Clarity Western ECL substrate (Bio-Rad, 1705061) and imaged using a ChemiDoc MD Imaging system (BioRad). Uncropped and unprocessed blots are provided in the Source Data.</p></section><section id="Sec34"><h3 class="pmc_sec_title">RT-qPCR</h3>
<p id="Par73">Total RNA was extracted using the NucleoSpin RNA Isolation Kit (Macherey-Nagel, Cat. 740955-250). Subsequently, 1 μg of RNA was reverse transcribed into cDNA according to the SuperScript VILO cDNA Synthesis Kit (Invitrogen, Cat. 11754050). RT-qPCR was then performed on a LightCycler 480 II system (Roche) using LightCycler 480 SYBR Green I Master Mix (Roche, 04707516001). The amplification protocol consisted of an initial denaturation at 95 °C for 5 minutes, followed by 40 cycles with the following conditions: 95 °C for 10 s, 64 °C for 15 s, and 72 °C for 20 s. The expression levels of target mRNAs were quantified relative to 18S ribosomal RNA using 2<sup>-ΔΔCT</sup> method.</p>
<p id="Par74">Primer pairs were designed for the amplification of the genes of interest. Specifically, TGN46: AGCTGGAGTACGGCCTTCT (fw) and GATGCGACTTGGTAGAGCCTC (rev); B4GALT1: GTATTTTGGAGGTGTCTCTGCTC (fw) and GGGCGAGATATAGACATGCCTC (rev); 18S rRNA: CGGCTACCACATCCAAGGAA (fw) and GCTGGAATTACCGCGGCT (rev).</p></section><section id="Sec35"><h3 class="pmc_sec_title">Subcellular fractionations</h3>
<section id="Sec36"><h4 class="pmc_sec_title">Membrane/cytosol fractionation</h4>
<p id="Par75">To investigate membrane association of the ATP6V<sub>1</sub>A subunit, A431 cells were scraped from 10-cm dishes and transferred to 1.5 ml tubes. Cell pellets were centrifuged at 800x<em>g</em> for 3 min at 4 °C and washed with cold 1x PBS to remove residual culture medium. Cells were re-suspended into 500 µl fractionation buffer (20 mM HEPES pH 7.4, 10 mM KCl, 2 mM MgCl<sub>2</sub>, 1 mM EDTA, 1 mM EGTA, 1 mM DTT and protease inhibitors) and passed 100x through a 26-gauge needle using a 1 ml syringe. 5 µl cell homogenate was observed with inverted phase contrast microscope (Olympus) to ensure over 90% cells were broken. Pellets containing nuclei were removed by 5 min 1,000x<em>g</em> centrifugation at 4 °C. The post-nuclear supernatant (PNS) was subjected to 100,000x<em>g</em> ultracentrifugation for 1 h at 4 °C to obtain organellar membrane fraction pellet. Membrane fraction was dissolved in 2x SDS-PAGE sample buffer. Supernatant containing cytosolic proteins was mixed with equal amount of 20% cold trichloroacetic acid (TCA) and placed on ice for 2 h. After 15 min, samples were centrifuged at 16,200x<em>g</em> at 4 °C, and denatured protein sediment was collected and washed with cold acetone twice. Protein sediment was dissolved in 2x SDS-PAGE sample buffer. ATP6V<sub>1</sub>A levels in the membrane and cytosol were analyzed by Western blotting.</p></section><section id="Sec37"><h4 class="pmc_sec_title">Sucrose density gradient fractionation</h4>
<p id="Par76">To investigate the membrane lipid environment surrounding ATP6V<sub>0</sub>a2 subunit, cells were scraped from 10-cm dishes and transferred to 1.5 ml tubes. Cell pellets were centrifuged at 800x<em>g</em> for 3 min at 4 °C and washed with cold 1x PBS to remove residual culture medium. Cells were re-suspended into 600 µl hypotonic buffer (10 mM HEPES pH 7.4, 0.3 M sucrose, protease inhibitors) and passed 100x through a 26-gauge needle using a 1 ml syringe. 5 ml cell homogenate was observed with inverted phase contrast microscope (Olympus) to ensure over 90% cells were broken. Pellets containing nuclei were removed by centrifugation for 5 min at 1,000x<em>g</em> at 4 °C. 500 µl PNS was mixed with 3 ml 2.5 M sucrose (10 mM HEPES pH 7.4) and placed on the bottom of the tube (Beckman, 331374). Then, 1 ml 1.9 M, 1 ml 1.4 M, 2 ml 1.2 M 3 ml 1.0 M and 1.5 ml 0.3 M sucrose solutions in 10 mM HEPES pH 7.4 were placed above PNS mixture, sequentially from bottom to the top to create a sucrose gradient. Samples were ultracentrifuged in a Beckman SW40 Ti rotor at 264,000x<em>g</em> for 16 h. Afterwards, twelve fractions of 1 ml were collected from the top and mixed with equal amounts of 20% cold TCA and placed on ice for 2 h. After centrifugation for 15 min at 16,200x<em>g</em> at 4 °C, denatured protein sediment was collected and washed with cold acetone twice. Protein sediment was dissolved in 2x SDS-PAGE sample buffer. Distribution of ATP6V<sub>0</sub>a2 across the fractions was analyzed by Western blotting.</p></section></section><section id="Sec38"><h3 class="pmc_sec_title">PNGaseF and neuraminidase A digestions</h3>
<p id="Par77">PNGaseF (NEB, P0704) digestions were performed according to manufacturer’s protocol. Sac1-degron cells were lysed in RIPA lysis buffer (1% NP-40, 0,1% SDS, 0.5% Sodium Deoxycholate, in 1x TBS). After protein concentration determination using the DC™ Protein Assay Kit I (Bio-Rad, 5000111), 10 µg of glycoprotein was mixed with 2 µl of 10x Glycobuffer. H<sub>2</sub>O was added to make a 20 µl total reaction volume. Finally, 2 µl of PNGaseF was added and samples were incubated for 16 h at 37 °C.</p>
<p id="Par78">α2-3,6,8,9 Neuraminidase A (NEB, P0722) reaction was performed according to manufacturer’s protocol. In short, 2 µg of glycoprotein were mixed with H<sub>2</sub>O to make a total reaction volume of 18 µl. 2 µl of 10x GlycoBuffer 1 was added, followed by 2 µl of α2-3,6,8,9 Neuraminidase A. The reaction mix was incubated at 37 °C for 1 h.</p>
<p id="Par79">5x sample buffer was added to the samples and the samples were analyzed on a 7.5% Mini-Protein TGX Stain-Free gels (Bio-Rad, 1610181). A similar amount of undigested sample was loaded on the same gel to determine PNGaseF and α2-3,6,8,9 Neuraminidase A activity.</p></section><section id="Sec39"><h3 class="pmc_sec_title">Analysis of secreted proteins</h3>
<p id="Par80">Sac1-degron TBs were seeded on 10 cm dishes. Cells were washed 3x with 1x PBS and medium was changed to serum-free medium containing DMSO or IAA with or without EN6. After 15 h medium was collected and centrifuged for 10 min at 2,300x<em>g</em>. Meanwhile, cells were lysed according to western blot protocol. After centrifugation, 5x SDS-PAGE sample buffer was added to secreted medium, and samples were denatured for 10 min at 95 °C and analyzed by Western blotting.</p></section><section id="Sec40"><h3 class="pmc_sec_title">gnomAD lof.pLI analysis</h3>
<p id="Par81">From the Genome Aggregation Database (gnomAD) dataset v4.0.0 (gnomAD (broadinstitute.org)), the publicly available genetic constraint metrics TSV data was downloaded containing the lof.pLI score for genes, indicating the probability of a gene of being loss-of-function intolerant. A probability of 0 indicates a gene is highly tolerant to loss-of-function mutations, while a lof.pLI 1 indicates that a gene does not tolerate loss-of-function mutations. Data was plotted as a density plot using RStudio highlighting the lof.pLI score of <em>SACM1L</em>.</p></section><section id="Sec41"><h3 class="pmc_sec_title">Atomistic molecular dynamics simulations</h3>
<p id="Par82">The simulation model is based on the V<sub>0</sub> region hosted in a lipid membrane. As the initial structure of V<sub>0</sub>, we used the PDB ID: 6WM2<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. The cryo-EM structure includes the lysosomal V<sub>0</sub>a1 subunit that was replaced by the V<sub>0</sub>a2 subunit structure given by the AlphaFold database entry <a href="https://www.ncbi.nlm.nih.gov/protein/Q9Y487" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9Y487</a><sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>,<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. Using CHARMM-GUI<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>, the modified V<sub>0</sub> structure was then inserted in a 1-oleoyl-2-palmitoyl-glycero-3-phosphocholine (POPC) membrane, with varied concentrations of phosphatidylinositol 4-phosphate (PI(4)P) and cholesterol (Table <a href="#Tab1" class="usa-link">1</a>). Membranes were comprised of a total number of 980 lipids that were symmetrically distributed in the two leaflets. In additional simulations, an asymmetric distribution of PI(4)P was also explored but was observed not to affect conclusions. Further, while a number of additional mixtures of POPC, PI(4)P and cholesterol were investigated, they did not bring forth additional insight. Hence, for the sake of clarity, the focus here is on the systems in Table <a href="#Tab1" class="usa-link">1</a>. The membrane complex was solvated in a solution of counterions to neutralize the system, with 150 mM KCl. The CHARMM36m force field<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup> was used for the protein and lipids, the TIP3P model<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup> for water molecules, and a compatible parameter set for the ions<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>.</p>
<p id="Par83">The models were simulated using the GROMACS 2022.3 engine<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup> using the Verlet scheme and the hydrogen mass repartitioning method<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup> to achieve a 4 fs timestep. Periodic boundary conditions were applied in all three dimensions. All production simulations were performed in the NpT ensemble. The Nosé-Hoover thermostat<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>,<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup> was used to maintain the temperature at 310 K with the protein, the membrane, and the solvent (water and KCl) coupled to separate temperature baths with a time constant of 1.0 ps. The Parrinello-Rahman barostat<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup> was used for semi-isotropic pressure coupling at 1 atm with a time constant of 5 ps and a compressibility value of 4.5 x 10<sup>-5</sup> bar<sup>-1</sup>. The LINCS algorithm<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup> was used to constrain all bonds. For electrostatic interactions, a real space cut-off of 1.2 nm was used. Long-range electrostatic interactions were computed using the Particle-Mesh Ewald method<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup> with a Fourier spacing of 0.12 nm and a fourth-order interpolation. For the van der Waals interactions, a Lennard-Jones potential with a force-switch between 1.0 and 1.2 nm was used.</p>
<p id="Par84">All visualizations were implemented with VMD<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a></sup>. The analysis of the simulated data (except for deep learning-based analyses) was carried out using the MDAnalysis python package<sup><a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>,<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup>.</p></section><section id="Sec42"><h3 class="pmc_sec_title">Autoencoder-based dimensionality reduction</h3>
<p id="Par85">To detect conformational states in the simulated protein data, the dimensionality of the raw configurational data was reduced using a denoising autoencoder architecture<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup> as implemented in PyTorch<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. The L1-loss (Mean Absolute Error loss) function was used for assessing the reconstruction quality. Only the C-alpha atoms of the protein structure were used for the dimensionality reduction. The encoded 2D space was mapped on a 3 × 3 grid. Each grid point acted as a cluster forming a region with the constraint that the Euclidean similarity of the input data in its immediate vicinity was maximized using the L2-loss (Mean Squared Error Loss) function. This additional loss function assisted in creating preformed clusters of highly similar structures. The model was trained on only half of the (shuffled) simulation frames to avoid overfitting. The reconstruction error was then evaluated on the remaining holdout data. To make the data rotationally and translationally invariant, the total rotations and translational movement was removed from the entire data set by fitting it on a reference structure. The model was further regularized with a 20% dropout rate on each layer. After manually testing a wide variety of architectures, an autoencoder with two hidden layers in the encoder module and two hidden layers in the decoder module was finally chosen. The encoder and decoder modules had a mirrored structure with the former module with 1024 neurons in the first hidden layer, and a reversed arrangement in the latter module. The encoded layer was created with only two neurons to obtain a 2D reduced dimensionality. The non-linearity in encoding was provided by the ReLU<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup> activation function. The model was then trained for 500 epochs using a fixed seed with the intent of maintaining reproducibility. At the end of training, a mean absolute reconstruction error (MAE) of 1.01 Å in the holdout data was obtained. This model was then saved and used for further analysis.</p></section><section id="Sec43"><h3 class="pmc_sec_title">Gaussian Mixture model-based clustering</h3>
<p id="Par86">To find clusters in the dimensionality-reduced space obtained from the autoencoder, a Bayesian Gaussian Mixture Model (BGMM) as implemented in the scikit-learn package<sup><a href="#CR104" class="usa-link" aria-describedby="CR104">104</a></sup> was used. A full covariance matrix for the 2D Gaussians was used to allow for flexible fitting of the Gaussian density on the data. The number of clusters present in the data was selected to be 4, based on first using a large value for this parameter and allowing the BGMM to regularize the number. The clusters are divided by their decision boundaries, which represent the region where the uncertainty of a cluster label approaches 50%.</p></section><section id="Sec44"><h3 class="pmc_sec_title">Partial least squares-based discriminant analysis</h3>
<p id="Par87">The functional difference between the assembly-promoting cluster 2 and the other three clusters is clearly revealed through Partial Least Squares-based discriminant analysis (PLS-DA). This was done by creating a vector whose length corresponds to the total number of structures in all simulations and setting the value of this vector to 1 for structures classified in cluster 2, and 0 for structures classified in other clusters. The Scikit-learn PLS package was then used to determine whether it was possible to create a linear model that would separate cluster 2 from the other clusters. Half of the shuffled data was used to train this model. The rest of the data was left for model validation. A six-component model was found to have a robust ability to predict the labels in the validation set with R<sup>2</sup> &gt; 0.75. This model was then visualized by linear interpolation between values 0 and 1 to interpret the differences between the clusters 1, 3, and 4 at one end, and the cluster 2 at the other end.</p></section><section id="Sec45"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par88">Statistical analyses were performed and plotted using Graphpad Prism (version 9.4.1). Data were checked for normal distribution. For comparisons of two groups, Student’s t-test was used. For comparisons of two or more groups, a One-way ANOVA with multiple comparisons test was used.</p></section><section id="Sec46"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par89">Further information on research design is available in the <a href="#MOESM7" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec47"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (70.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63125_MOESM2_ESM.pdf</a><sup> (84.9KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (2.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM5_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (5.4MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM6_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (5.6MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (131.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (99.7KB, pdf) </sup>
</div></div></section></section><section id="Sec48"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (5.4MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the HiLIFE and Biocenter Finland supported Biomedicum Imaging and Electron Microscopy Units and Turku Proteomics Facility for infrastructure support, Giray Enkavi and Waldemar Kulig for discussions on molecular dynamics simulations, Ábel Szkalisity for support on data analysis and the generation of the interactive volcano plot, and Anna Uro for technical assistance. X.Z. was supported by Academy of Finland grant 332096, M.M.v.d.S. and H.L. received funding from the European Union’s Horizon 2020 Research and Innovation programmes under the Marie Sklodowska-Curie Personal Fellowship (M.M.v.d.S.) no. 101059424 and the Marie Sklodowska-Curie Grant Agreement (H.L.) no. 953489. E.I. was supported by the Academy of Finland (grant no. 324929), Foundation Leducq 19CVD04, Jane and Aatos Erkko Foundation and Sigrid Jusélius Foundation, T.S. was supported by AMED (grant no. 24gm1710007), by TMDU under Multilayered Stress Diseases (JPMXP1323015483) and Medical Research Center Initiative for High Depth Omics. T.S. was partly supported by AMED-CREST (JP25gm1710001h0003), Nanken-Kyoten, The Multilayered Stress Diseases program and the Initiative for High Depth Omics, IST. I.V. thanks the Helsinki Institute of Life Science (HiLIFE) Fellow program, Sigrid Jusélius Foundation, Lundbeck Foundation, and Academy of Finland (project IDs: 335527, 331349, 336234, 364185) for financial support. The authors thank CSC–IT Center for Science Ltd (Espoo, Finland) for providing computational resources.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>E.I. conceived and oversaw the project. X.Z., M.M.v.d.S., H.L. and M.H. performed experiments and analyzed data. S.L. provided Sac1-degron cell lines and technical support for cell engineering. H.V. and E.J. performed electron microscopic analysis. C.T. performed click-lipid MS experiments. O.P. performed gnomAD dataset analysis. S.M., J.S. and T.S. performed PRMC-MS experiments and analysis. S.K. and I.V. performed and analyzed atomistic molecular dynamics simulations. X.Z., M.M.v.d.S. and E.I. wrote the manuscript, and all authors edited the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par90"><em>Nature Communications</em> thanks Kartik Venkatachalam and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The gnomAD constraint metrics TSV data set v4.0.0 is publicly available via: gnomAD (broadinstitute.org). The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup> partner repository with the dataset identifier <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD065794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDX065794</a>. <a href="#Sec48" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par91">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Xin Zhou, Miesje M. van der Stoel.</p></div>
</div></section><section id="sec49"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63125-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Balla, T. Phosphoinositides: tiny lipids with giant impact on cell regulation. <em>Physiol. Rev.</em><strong>93</strong>, 1019–1137 (2013).
</cite> [<a href="https://doi.org/10.1152/physrev.00028.2012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3962547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23899561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Balla,%20T.%20Phosphoinositides:%20tiny%20lipids%20with%20giant%20impact%20on%20cell%20regulation.%20Physiol.%20Rev.93,%201019%E2%80%931137%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Hammond, G. R. V., Machner, M. P. &amp; Balla, T. A novel probe for phosphatidylinositol 4-phosphate reveals multiple pools beyond the Golgi. <em>J. Cell Biol.</em><strong>205</strong>, 113–126 (2014).
</cite> [<a href="https://doi.org/10.1083/jcb.201312072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3987136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24711504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hammond,%20G.%20R.%20V.,%20Machner,%20M.%20P.%20&amp;%20Balla,%20T.%20A%20novel%20probe%20for%20phosphatidylinositol%204-phosphate%20reveals%20multiple%20pools%20beyond%20the%20Golgi.%20J.%20Cell%20Biol.205,%20113%E2%80%93126%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Balla, A., Tuymetova, G., Barshishat, M., Geiszt, M. &amp; Balla, T. Characterization of type II phosphatidylinositol 4-kinase isoforms reveals association of the enzymes with endosomal vesicular compartments*. <em>J. Biol. Chem.</em><strong>277</strong>, 20041–20050 (2002).
</cite> [<a href="https://doi.org/10.1074/jbc.M111807200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11923287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Balla,%20A.,%20Tuymetova,%20G.,%20Barshishat,%20M.,%20Geiszt,%20M.%20&amp;%20Balla,%20T.%20Characterization%20of%20type%20II%20phosphatidylinositol%204-kinase%20isoforms%20reveals%20association%20of%20the%20enzymes%20with%20endosomal%20vesicular%20compartments*.%20J.%20Biol.%20Chem.277,%2020041%E2%80%9320050%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Tan, J. &amp; Brill, J. A. Cinderella story: PI4P goes from precursor to key signaling molecule. <em>Crit. Rev. Biochem. Mol. Biol.</em><strong>49</strong>, 33–58 (2014).
</cite> [<a href="https://doi.org/10.3109/10409238.2013.853024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24219382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20J.%20&amp;%20Brill,%20J.%20A.%20Cinderella%20story:%20PI4P%20goes%20from%20precursor%20to%20key%20signaling%20molecule.%20Crit.%20Rev.%20Biochem.%20Mol.%20Biol.49,%2033%E2%80%9358%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Balla, A. et al. Maintenance of hormone-sensitive phosphoinositide pools in the plasma membrane requires phosphatidylinositol 4-kinase IIIα. <em>Mol. Biol. Cell</em><strong>19</strong>, 711–721 (2008).
</cite> [<a href="https://doi.org/10.1091/mbc.E07-07-0713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2230591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18077555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Balla,%20A.%20et%20al.%20Maintenance%20of%20hormone-sensitive%20phosphoinositide%20pools%20in%20the%20plasma%20membrane%20requires%20phosphatidylinositol%204-kinase%20III%CE%B1.%20Mol.%20Biol.%20Cell19,%20711%E2%80%93721%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Graham, T. R. &amp; Burd, C. G. Coordination of Golgi functions by phosphatidylinositol 4-kinases. <em>Trends Cell Biol.</em><strong>21</strong>, 113–121 (2011).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2010.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3053015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21282087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Graham,%20T.%20R.%20&amp;%20Burd,%20C.%20G.%20Coordination%20of%20Golgi%20functions%20by%20phosphatidylinositol%204-kinases.%20Trends%20Cell%20Biol.21,%20113%E2%80%93121%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lolicato, F., Nickel, W., Haucke, V. &amp; Ebner, M. Phosphoinositide switches in cell physiology—from molecular mechanisms to disease. <em>J. Biol. Chem</em>. 10.1016/j.jbc.2024.105757 (2024).</cite> [<a href="https://doi.org/10.1016/j.jbc.2024.105757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10944118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38364889/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Wang, Y. J. et al. Phosphatidylinositol 4 phosphate regulates targeting of clathrin adaptor AP-1 complexes to the golgi. <em>Cell</em><strong>114</strong>, 299–310 (2003).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(03)00603-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12914695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20J.%20et%20al.%20Phosphatidylinositol%204%20phosphate%20regulates%20targeting%20of%20clathrin%20adaptor%20AP-1%20complexes%20to%20the%20golgi.%20Cell114,%20299%E2%80%93310%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Dippold, H. C. et al. GOLPH3 bridges phosphatidylinositol-4- phosphate and actomyosin to stretch and shape the golgi to promote budding. <em>Cell</em><strong>139</strong>, 337–351 (2009).
</cite> [<a href="https://doi.org/10.1016/j.cell.2009.07.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2779841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19837035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dippold,%20H.%20C.%20et%20al.%20GOLPH3%20bridges%20phosphatidylinositol-4-%20phosphate%20and%20actomyosin%20to%20stretch%20and%20shape%20the%20golgi%20to%20promote%20budding.%20Cell139,%20337%E2%80%93351%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wakana, Y. et al. The ER cholesterol sensor SCAP promotes CARTS biogenesis at ER–Golgi membrane contact sites. <em>J. Cell Biol</em>. <strong>220</strong>, e202002150 (2020).</cite> [<a href="https://doi.org/10.1083/jcb.202002150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7654440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33156328/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Kovács, D. et al. Lipid exchange at ER–trans-Golgi contact sites governs polarized cargo sorting. <em>J. Cell Biol.</em><strong>223</strong>, e202307051 (2023).
</cite> [<a href="https://doi.org/10.1083/jcb.202307051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10664280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37991810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kov%C3%A1cs,%20D.%20et%20al.%20Lipid%20exchange%20at%20ER%E2%80%93trans-Golgi%20contact%20sites%20governs%20polarized%20cargo%20sorting.%20J.%20Cell%20Biol.223,%20e202307051%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Phillips, T. E. J. &amp; Maguire, E. Phosphoinositides: roles in the development of microglial-mediated neuroinflammation and neurodegeneration. <em>Front. Cell. Neurosci</em>. <strong>15</strong>, 652593 (2021).</cite> [<a href="https://doi.org/10.3389/fncel.2021.652593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8032904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33841102/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Tokuda, E. et al. Phosphatidylinositol 4-phosphate in the golgi apparatus regulates cell–cell adhesion and invasive cell migration in human breast cancer. <em>Cancer Res.</em><strong>74</strong>, 3054 (2014).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-13-2441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24706697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tokuda,%20E.%20et%20al.%20Phosphatidylinositol%204-phosphate%20in%20the%20golgi%20apparatus%20regulates%20cell%E2%80%93cell%20adhesion%20and%20invasive%20cell%20migration%20in%20human%20breast%20cancer.%20Cancer%20Res.74,%203054%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Ijuin, T. et al. Regulation of CD44 expression and focal adhesion by Golgi phosphatidylinositol 4-phosphate in breast cancer. <em>Cancer Sci.</em><strong>107</strong>, 981–990 (2016).
</cite> [<a href="https://doi.org/10.1111/cas.12968" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4946718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27178239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ijuin,%20T.%20et%20al.%20Regulation%20of%20CD44%20expression%20and%20focal%20adhesion%20by%20Golgi%20phosphatidylinositol%204-phosphate%20in%20breast%20cancer.%20Cancer%20Sci.107,%20981%E2%80%93990%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Moser von Filseck, J. et al. Phosphatidylserine transport by ORP/Osh proteins is driven by phosphatidylinositol 4-phosphate. <em>Science</em><strong>349</strong>, 432 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aab1346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26206936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moser%20von%20Filseck,%20J.%20et%20al.%20Phosphatidylserine%20transport%20by%20ORP/Osh%20proteins%20is%20driven%20by%20phosphatidylinositol%204-phosphate.%20Science349,%20432%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Mesmin, B. et al. A four-step cycle driven by PI(4)P hydrolysis directs sterol/PI(4)P exchange by the ER-golgi tether OSBP. <em>Cell</em><strong>155</strong>, 830–843 (2013).
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.09.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24209621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mesmin,%20B.%20et%20al.%20A%20four-step%20cycle%20driven%20by%20PI(4)P%20hydrolysis%20directs%20sterol/PI(4)P%20exchange%20by%20the%20ER-golgi%20tether%20OSBP.%20Cell155,%20830%E2%80%93843%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Mesmin, B. et al. Sterol transfer, PI4P consumption, and control of membrane lipid order by endogenous OSBP. <em>EMBO J.</em><strong>36</strong>, 3156–3174 (2017).
</cite> [<a href="https://doi.org/10.15252/embj.201796687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5666618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28978670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mesmin,%20B.%20et%20al.%20Sterol%20transfer,%20PI4P%20consumption,%20and%20control%20of%20membrane%20lipid%20order%20by%20endogenous%20OSBP.%20EMBO%20J.36,%203156%E2%80%933174%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Zewe, J. P., Wills, R. C., Sangappa, S., Goulden, B. D. &amp; Hammond, G. R. V. SAC1 degrades its lipid substrate PtdIns4P in the endoplasmic reticulum to maintain a steep chemical gradient with donor membranes. <em>Elife</em><strong>7</strong>, e35588 (2018).
</cite> [<a href="https://doi.org/10.7554/eLife.35588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5829913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29461204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zewe,%20J.%20P.,%20Wills,%20R.%20C.,%20Sangappa,%20S.,%20Goulden,%20B.%20D.%20&amp;%20Hammond,%20G.%20R.%20V.%20SAC1%20degrades%20its%20lipid%20substrate%20PtdIns4P%20in%20the%20endoplasmic%20reticulum%20to%20maintain%20a%20steep%20chemical%20gradient%20with%20donor%20membranes.%20Elife7,%20e35588%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Venditti, R. et al. The activity of Sac1 across ER-TGN contact sites requires the four-phosphate-adaptor-protein-1. <em>J. Cell Biol.</em><strong>218</strong>, 783–797 (2019).
</cite> [<a href="https://doi.org/10.1083/jcb.201812021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6400556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30659099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Venditti,%20R.%20et%20al.%20The%20activity%20of%20Sac1%20across%20ER-TGN%20contact%20sites%20requires%20the%20four-phosphate-adaptor-protein-1.%20J.%20Cell%20Biol.218,%20783%E2%80%93797%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Naito, T. et al. Regulation of cellular cholesterol distribution via non-vesicular lipid transport at ER-Golgi contact sites. <em>Nat. Commun.</em><strong>14</strong>, 5867 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-41213-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10514280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37735529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naito,%20T.%20et%20al.%20Regulation%20of%20cellular%20cholesterol%20distribution%20via%20non-vesicular%20lipid%20transport%20at%20ER-Golgi%20contact%20sites.%20Nat.%20Commun.14,%205867%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Dong, R. et al. Endosome-ER contacts control actin nucleation and retromer function through VAP-dependent regulation of PI4P. <em>Cell</em><strong>166</strong>, 408–423 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.06.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4963242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27419871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20R.%20et%20al.%20Endosome-ER%20contacts%20control%20actin%20nucleation%20and%20retromer%20function%20through%20VAP-dependent%20regulation%20of%20PI4P.%20Cell166,%20408%E2%80%93423%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Liu, Y. et al. The Sac1 phosphoinositide phosphatase regulates golgi membrane morphology and mitotic spindle organization in mammals. <em>Mol. Biol. Cell</em><strong>19</strong>, 3080–3096 (2008).
</cite> [<a href="https://doi.org/10.1091/mbc.E07-12-1290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2441670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18480408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20The%20Sac1%20phosphoinositide%20phosphatase%20regulates%20golgi%20membrane%20morphology%20and%20mitotic%20spindle%20organization%20in%20mammals.%20Mol.%20Biol.%20Cell19,%203080%E2%80%933096%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Cheong, F. Y. et al. Spatial regulation of golgi phosphatidylinositol-4-phosphate is required for enzyme localization and glycosylation fidelity. <em>Traffic</em><strong>11</strong>, 1180–1190 (2010).
</cite> [<a href="https://doi.org/10.1111/j.1600-0854.2010.01092.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2921771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20573065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheong,%20F.%20Y.%20et%20al.%20Spatial%20regulation%20of%20golgi%20phosphatidylinositol-4-phosphate%20is%20required%20for%20enzyme%20localization%20and%20glycosylation%20fidelity.%20Traffic11,%201180%E2%80%931190%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Blagoveshchenskaya, A. et al. Integration of Golgi trafficking and growth factor signaling by the lipid phosphatase SAC1. <em>J. Cell Biol.</em><strong>180</strong>, 803–812 (2008).
</cite> [<a href="https://doi.org/10.1083/jcb.200708109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2265582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18299350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blagoveshchenskaya,%20A.%20et%20al.%20Integration%20of%20Golgi%20trafficking%20and%20growth%20factor%20signaling%20by%20the%20lipid%20phosphatase%20SAC1.%20J.%20Cell%20Biol.180,%20803%E2%80%93812%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Novick, P., Osmond, B. C. &amp; Botstein, D. Suppressors of yeast actin mutations. <em>Genetics</em><strong>121</strong>, 659–674 (1989).
</cite> [<a href="https://doi.org/10.1093/genetics/121.4.659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1203651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2656401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Novick,%20P.,%20Osmond,%20B.%20C.%20&amp;%20Botstein,%20D.%20Suppressors%20of%20yeast%20actin%20mutations.%20Genetics121,%20659%E2%80%93674%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Cleves, A. E., Novick, P. J. &amp; Bankaitis, V. A. Mutations in the SAC1 gene suppress defects in yeast Golgi and yeast actin function. <em>J. Cell Biol.</em><strong>109</strong>, 2939–2950 (1989).
</cite> [<a href="https://doi.org/10.1083/jcb.109.6.2939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2115899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2687291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cleves,%20A.%20E.,%20Novick,%20P.%20J.%20&amp;%20Bankaitis,%20V.%20A.%20Mutations%20in%20the%20SAC1%20gene%20suppress%20defects%20in%20yeast%20Golgi%20and%20yeast%20actin%20function.%20J.%20Cell%20Biol.109,%202939%E2%80%932950%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Liu, Y., Boukhelifa, M., Tribble, E. &amp; Bankaitis, V. A. Functional studies of the mammalian Sac1 phosphoinositide phosphatase. <em>Adv. Enzym. Regul.</em><strong>49</strong>, 75–86 (2009).</cite> [<a href="https://doi.org/10.1016/j.advenzreg.2009.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2895967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19534026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.,%20Boukhelifa,%20M.,%20Tribble,%20E.%20&amp;%20Bankaitis,%20V.%20A.%20Functional%20studies%20of%20the%20mammalian%20Sac1%20phosphoinositide%20phosphatase.%20Adv.%20Enzym.%20Regul.49,%2075%E2%80%9386%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Wei, H.-C. et al. The Sac1 lipid phosphatase regulates cell shape change and the JNK cascade during dorsal closure in Drosophila. <em>Curr. Biol.</em><strong>13</strong>, 1882–1887 (2003).
</cite> [<a href="https://doi.org/10.1016/j.cub.2003.09.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14588244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20H.-C.%20et%20al.%20The%20Sac1%20lipid%20phosphatase%20regulates%20cell%20shape%20change%20and%20the%20JNK%20cascade%20during%20dorsal%20closure%20in%20Drosophila.%20Curr.%20Biol.13,%201882%E2%80%931887%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Li, S., Prasanna, X., Salo, V. T., Vattulainen, I. &amp; Ikonen, E. An efficient auxin-inducible degron system with low basal degradation in human cells. <em>Nat. Methods</em><strong>16</strong>, 866–869 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0512-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31451765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.,%20Prasanna,%20X.,%20Salo,%20V.%20T.,%20Vattulainen,%20I.%20&amp;%20Ikonen,%20E.%20An%20efficient%20auxin-inducible%20degron%20system%20with%20low%20basal%20degradation%20in%20human%20cells.%20Nat.%20Methods16,%20866%E2%80%93869%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Li, S. et al. HiHo-AID2: boosting homozygous knock-in efficiency enables robust generation of human auxin-inducible degron cells. <em>Genome Biol.</em><strong>25</strong>, 58 (2024).
</cite> [<a href="https://doi.org/10.1186/s13059-024-03187-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10895734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38409044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.%20et%20al.%20HiHo-AID2:%20boosting%20homozygous%20knock-in%20efficiency%20enables%20robust%20generation%20of%20human%20auxin-inducible%20degron%20cells.%20Genome%20Biol.25,%2058%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Morioka, S. et al. A mass spectrometric method for in-depth profiling of phosphoinositide regioisomers and their disease-associated regulation. <em>Nat. Commun.</em><strong>13</strong>, 1–9 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-021-27648-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8749000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35013169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morioka,%20S.%20et%20al.%20A%20mass%20spectrometric%20method%20for%20in-depth%20profiling%20of%20phosphoinositide%20regioisomers%20and%20their%20disease-associated%20regulation.%20Nat.%20Commun.13,%201%E2%80%939%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Rohde, H. M. et al. The human phosphatidylinositol phosphatase SAC1 interacts with the coatomer I complex*. <em>J. Biol. Chem.</em><strong>278</strong>, 52689–52699 (2003).
</cite> [<a href="https://doi.org/10.1074/jbc.M307983200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14527956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rohde,%20H.%20M.%20et%20al.%20The%20human%20phosphatidylinositol%20phosphatase%20SAC1%20interacts%20with%20the%20coatomer%20I%20complex*.%20J.%20Biol.%20Chem.278,%2052689%E2%80%9352699%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Rivinoja, A., Hassinen, A., Kokkonen, N., Kauppila, A. &amp; Kellokumpu, S. Elevated Golgi pH impairs terminal N-glycosylation by inducing mislocalization of Golgi glycosyltransferases. <em>J. Cell. Physiol.</em><strong>220</strong>, 144–154 (2009).
</cite> [<a href="https://doi.org/10.1002/jcp.21744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19277980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rivinoja,%20A.,%20Hassinen,%20A.,%20Kokkonen,%20N.,%20Kauppila,%20A.%20&amp;%20Kellokumpu,%20S.%20Elevated%20Golgi%20pH%20impairs%20terminal%20N-glycosylation%20by%20inducing%20mislocalization%20of%20Golgi%20glycosyltransferases.%20J.%20Cell.%20Physiol.220,%20144%E2%80%93154%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Linders, P. T. A., Ioannidis, M., ter Beest, M. &amp; van den Bogaart, G. Fluorescence lifetime imaging of pH along the secretory pathway. <em>ACS Chem. Biol.</em><strong>17</strong>, 240–251 (2022).
</cite> [<a href="https://doi.org/10.1021/acschembio.1c00907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8787756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35000377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Linders,%20P.%20T.%20A.,%20Ioannidis,%20M.,%20ter%20Beest,%20M.%20&amp;%20van%20den%20Bogaart,%20G.%20Fluorescence%20lifetime%20imaging%20of%20pH%20along%20the%20secretory%20pathway.%20ACS%20Chem.%20Biol.17,%20240%E2%80%93251%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Chung, C. Y.-S. et al. Covalent targeting of the vacuolar H(+)-ATPase activates autophagy via mTORC1 inhibition. <em>Nat. Chem. Biol.</em><strong>15</strong>, 776–785 (2019).
</cite> [<a href="https://doi.org/10.1038/s41589-019-0308-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6641988/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31285595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chung,%20C.%20Y.-S.%20et%20al.%20Covalent%20targeting%20of%20the%20vacuolar%20H(+)-ATPase%20activates%20autophagy%20via%20mTORC1%20inhibition.%20Nat.%20Chem.%20Biol.15,%20776%E2%80%93785%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>McGuire, C. M. &amp; Forgac, M. Glucose starvation increases V-ATPase assembly and activity in mammalian cells through AMP kinase and phosphatidylinositide 3-kinase/Akt signaling. <em>J. Biol. Chem.</em><strong>293</strong>, 9113–9123 (2018).
</cite> [<a href="https://doi.org/10.1074/jbc.RA117.001327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5995505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29540478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGuire,%20C.%20M.%20&amp;%20Forgac,%20M.%20Glucose%20starvation%20increases%20V-ATPase%20assembly%20and%20activity%20in%20mammalian%20cells%20through%20AMP%20kinase%20and%20phosphatidylinositide%203-kinase/Akt%20signaling.%20J.%20Biol.%20Chem.293,%209113%E2%80%939123%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Banerjee, S. &amp; Kane, P. M. Direct interaction of the Golgi V-ATPase a-subunit isoform with PI(4)P drives localization of Golgi V-ATPases in yeast. <em>Mol. Biol. Cell</em><strong>28</strong>, 2518–2530 (2017).
</cite> [<a href="https://doi.org/10.1091/mbc.E17-05-0316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5597324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28720663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee,%20S.%20&amp;%20Kane,%20P.%20M.%20Direct%20interaction%20of%20the%20Golgi%20V-ATPase%20a-subunit%20isoform%20with%20PI(4)P%20drives%20localization%20of%20Golgi%20V-ATPases%20in%20yeast.%20Mol.%20Biol.%20Cell28,%202518%E2%80%932530%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Infante, R. E. &amp; Radhakrishnan, A. Continuous transport of a small fraction of plasma membrane cholesterol to endoplasmic reticulum regulates total cellular cholesterol. <em>Elife</em><strong>6</strong>, e25466 (2017).
</cite> [<a href="https://doi.org/10.7554/eLife.25466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5433840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28414269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Infante,%20R.%20E.%20&amp;%20Radhakrishnan,%20A.%20Continuous%20transport%20of%20a%20small%20fraction%20of%20plasma%20membrane%20cholesterol%20to%20endoplasmic%20reticulum%20regulates%20total%20cellular%20cholesterol.%20Elife6,%20e25466%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Couoh-Cardel, S., Milgrom, E. &amp; Wilkens, S. Affinity purification and structural features of the yeast vacuolar ATPase V<sub>o</sub> membrane sector *. <em>J. Biol. Chem.</em><strong>290</strong>, 27959–27971 (2015).
</cite> [<a href="https://doi.org/10.1074/jbc.M115.662494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4646912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26416888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Couoh-Cardel,%20S.,%20Milgrom,%20E.%20&amp;%20Wilkens,%20S.%20Affinity%20purification%20and%20structural%20features%20of%20the%20yeast%20vacuolar%20ATPase%20Vo%20membrane%20sector%20*.%20J.%20Biol.%20Chem.290,%2027959%E2%80%9327971%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wang, L., Wu, D., Robinson, C. V., Wu, H. &amp; Fu, T.-M. Structures of a complete human V-ATPase reveal mechanisms of its assembly. <em>Mol. Cell</em><strong>80</strong>, 501–511.e3 (2020).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2020.09.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7655608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33065002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20L.,%20Wu,%20D.,%20Robinson,%20C.%20V.,%20Wu,%20H.%20&amp;%20Fu,%20T.-M.%20Structures%20of%20a%20complete%20human%20V-ATPase%20reveal%20mechanisms%20of%20its%20assembly.%20Mol.%20Cell80,%20501%E2%80%93511.e3%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Nwabuobi, C. et al. hCG: biological functions and clinical applications. <em>Int. J. Mol. Sci</em>. <strong>18</strong>, 2037 (2017).</cite> [<a href="https://doi.org/10.3390/ijms18102037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5666719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28937611/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>von Blume, J. &amp; Hausser, A. Lipid-dependent coupling of secretory cargo sorting and trafficking at the trans-Golgi network. <em>FEBS Lett.</em><strong>593</strong>, 2412–2427 (2019).
</cite> [<a href="https://doi.org/10.1002/1873-3468.13552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8048779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31344259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?von%20Blume,%20J.%20&amp;%20Hausser,%20A.%20Lipid-dependent%20coupling%20of%20secretory%20cargo%20sorting%20and%20trafficking%20at%20the%20trans-Golgi%20network.%20FEBS%20Lett.593,%202412%E2%80%932427%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Péresse, T. et al. Molecular and cellular dissection of the oxysterol-binding protein cycle through a fluorescent inhibitor. <em>J. Biol. Chem.</em><strong>295</strong>, 4277–4288 (2020).
</cite> [<a href="https://doi.org/10.1074/jbc.RA119.012012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7105299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32075908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%A9resse,%20T.%20et%20al.%20Molecular%20and%20cellular%20dissection%20of%20the%20oxysterol-binding%20protein%20cycle%20through%20a%20fluorescent%20inhibitor.%20J.%20Biol.%20Chem.295,%204277%E2%80%934288%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Kawasaki, A. et al. PI4P/PS countertransport by ORP10 at ER–endosome membrane contact sites regulates endosome fission. <em>J. Cell Biol</em>. <strong>221</strong>, e202103141 (2021).</cite> [<a href="https://doi.org/10.1083/jcb.202103141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8624802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34817532/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Kornak, U. et al. Impaired glycosylation and cutis laxa caused by mutations in the vesicular H+-ATPase subunit ATP6V0A2. <em>Nat. Genet.</em><strong>40</strong>, 32–34 (2008).
</cite> [<a href="https://doi.org/10.1038/ng.2007.45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18157129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kornak,%20U.%20et%20al.%20Impaired%20glycosylation%20and%20cutis%20laxa%20caused%20by%20mutations%20in%20the%20vesicular%20H+-ATPase%20subunit%20ATP6V0A2.%20Nat.%20Genet.40,%2032%E2%80%9334%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Oh-hashi, K., Kanamori, Y., Hirata, Y. &amp; Kiuchi, K. Characterization of V-ATPase inhibitor-induced secretion of cysteine-rich with EGF-like domains 2. <em>Cell Biol. Toxicol.</em><strong>30</strong>, 127–136 (2014).
</cite> [<a href="https://doi.org/10.1007/s10565-014-9274-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24687431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oh-hashi,%20K.,%20Kanamori,%20Y.,%20Hirata,%20Y.%20&amp;%20Kiuchi,%20K.%20Characterization%20of%20V-ATPase%20inhibitor-induced%20secretion%20of%20cysteine-rich%20with%20EGF-like%20domains%202.%20Cell%20Biol.%20Toxicol.30,%20127%E2%80%93136%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Guillard, M. et al. Vacuolar H+-ATPase meets glycosylation in patients with cutis laxa. <em>Biochim. Biophys. Acta Mol. Basis Dis.</em><strong>1792</strong>, 903–914 (2009).</cite> [<a href="https://doi.org/10.1016/j.bbadis.2008.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19171192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guillard,%20M.%20et%20al.%20Vacuolar%20H+-ATPase%20meets%20glycosylation%20in%20patients%20with%20cutis%20laxa.%20Biochim.%20Biophys.%20Acta%20Mol.%20Basis%20Dis.1792,%20903%E2%80%93914%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Serio, M. C. et al. Mutations in the V-ATPase assembly factor VMA21 cause a congenital disorder of glycosylation with autophagic liver disease. <em>Hepatology.</em><strong>72</strong>, 1968–1986 (2020).</cite> [<a href="https://doi.org/10.1002/hep.31218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7483274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32145091/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Hucthagowder, V. et al. Loss-of-function mutations in ATP6V0A2 impair vesicular trafficking, tropoelastin secretion and cell survival. <em>Hum. Mol. Genet.</em><strong>18</strong>, 2149–2165 (2009).
</cite> [<a href="https://doi.org/10.1093/hmg/ddp148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2685755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19321599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hucthagowder,%20V.%20et%20al.%20Loss-of-function%20mutations%20in%20ATP6V0A2%20impair%20vesicular%20trafficking,%20tropoelastin%20secretion%20and%20cell%20survival.%20Hum.%20Mol.%20Genet.18,%202149%E2%80%932165%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Levic, D. S. et al. Distinct roles for luminal acidification in apical protein sorting and trafficking in zebrafish. <em>J. Cell Biol.</em><strong>219</strong>, e201908225 (2020).
</cite> [<a href="https://doi.org/10.1083/jcb.201908225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7147097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32328632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levic,%20D.%20S.%20et%20al.%20Distinct%20roles%20for%20luminal%20acidification%20in%20apical%20protein%20sorting%20and%20trafficking%20in%20zebrafish.%20J.%20Cell%20Biol.219,%20e201908225%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Karacan, İ. et al. A novel ATP6V0A2 mutation causing recessive cutis laxa with unusual manifestations of bleeding diathesis and defective wound healing. <em>Turk. J. Haematol.</em><strong>36</strong>, 29–36 (2019).
</cite> [<a href="https://doi.org/10.4274/tjh.galenos.2018.2018.0325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6373508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30474613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karacan,%20%C4%B0.%20et%20al.%20A%20novel%20ATP6V0A2%20mutation%20causing%20recessive%20cutis%20laxa%20with%20unusual%20manifestations%20of%20bleeding%20diathesis%20and%20defective%20wound%20healing.%20Turk.%20J.%20Haematol.36,%2029%E2%80%9336%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Chu, A., Yao, Y., Glibowicka, M., Deber, C. M. &amp; Manolson, M. F. The human mutation K237_V238del in a putative lipid binding motif within the V-ATPase a2 isoform suggests a molecular mechanism underlying cutis laxa. <em>Int. J. Mol. Sci</em>. <strong>25</strong>, 2170 (2024).</cite> [<a href="https://doi.org/10.3390/ijms25042170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10889665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38396846/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wakana, Y. et al. CARTS biogenesis requires VAP-lipid transfer protein complexes functioning at the endoplasmic reticulum-Golgi interface. <em>Mol. Biol. Cell</em><strong>26</strong>, 4686–4699 (2015).
</cite> [<a href="https://doi.org/10.1091/mbc.E15-08-0599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4678024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26490117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wakana,%20Y.%20et%20al.%20CARTS%20biogenesis%20requires%20VAP-lipid%20transfer%20protein%20complexes%20functioning%20at%20the%20endoplasmic%20reticulum-Golgi%20interface.%20Mol.%20Biol.%20Cell26,%204686%E2%80%934699%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Kane, P. M. Disassembly and reassembly of the yeast vacuolar H(+)-ATPase in vivo. <em>J. Biol. Chem.</em><strong>270</strong>, 17025–17032 (1995).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7622524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kane,%20P.%20M.%20Disassembly%20and%20reassembly%20of%20the%20yeast%20vacuolar%20H(+)-ATPase%20in%20vivo.%20J.%20Biol.%20Chem.270,%2017025%E2%80%9317032%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Sautin, Y. Y., Lu, M., Gaugler, A., Zhang, L. &amp; Gluck, S. L. Phosphatidylinositol 3-kinase-mediated effects of glucose on vacuolar H+-ATPase assembly, translocation, and acidification of intracellular compartments in renal epithelial cells. <em>Mol. Cell. Biol.</em><strong>25</strong>, 575–589 (2005).
</cite> [<a href="https://doi.org/10.1128/MCB.25.2.575-589.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC543406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15632060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sautin,%20Y.%20Y.,%20Lu,%20M.,%20Gaugler,%20A.,%20Zhang,%20L.%20&amp;%20Gluck,%20S.%20L.%20Phosphatidylinositol%203-kinase-mediated%20effects%20of%20glucose%20on%20vacuolar%20H+-ATPase%20assembly,%20translocation,%20and%20acidification%20of%20intracellular%20compartments%20in%20renal%20epithelial%20cells.%20Mol.%20Cell.%20Biol.25,%20575%E2%80%93589%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Trombetta, E. S., Ebersold, M., Garrett, W., Pypaert, M. &amp; Mellman, I. Activation of lysosomal function during dendritic cell maturation. <em>Science</em><strong>299</strong>, 1400–1403 (2003).
</cite> [<a href="https://doi.org/10.1126/science.1080106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12610307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trombetta,%20E.%20S.,%20Ebersold,%20M.,%20Garrett,%20W.,%20Pypaert,%20M.%20&amp;%20Mellman,%20I.%20Activation%20of%20lysosomal%20function%20during%20dendritic%20cell%20maturation.%20Science299,%201400%E2%80%931403%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Zoncu, R. et al. mTORC1 senses lysosomal amino acids through an inside-out mechanism that requires the vacuolar H+-ATPase. <em>Science</em><strong>334</strong>, 678–683 (2011).
</cite> [<a href="https://doi.org/10.1126/science.1207056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3211112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22053050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zoncu,%20R.%20et%20al.%20mTORC1%20senses%20lysosomal%20amino%20acids%20through%20an%20inside-out%20mechanism%20that%20requires%20the%20vacuolar%20H+-ATPase.%20Science334,%20678%E2%80%93683%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Ratto, E. et al. Direct control of lysosomal catabolic activity by mTORC1 through regulation of V-ATPase assembly. <em>Nat. Commun.</em><strong>13</strong>, 4848 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-32515-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9385660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35977928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ratto,%20E.%20et%20al.%20Direct%20control%20of%20lysosomal%20catabolic%20activity%20by%20mTORC1%20through%20regulation%20of%20V-ATPase%20assembly.%20Nat.%20Commun.13,%204848%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Hooper, K. M. et al. V-ATPase is a universal regulator of LC3-associated phagocytosis and non-canonical autophagy. <em>J. Cell Biol.</em><strong>221</strong>, e202105112 (2022).
</cite> [<a href="https://doi.org/10.1083/jcb.202105112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9082624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35511089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hooper,%20K.%20M.%20et%20al.%20V-ATPase%20is%20a%20universal%20regulator%20of%20LC3-associated%20phagocytosis%20and%20non-canonical%20autophagy.%20J.%20Cell%20Biol.221,%20e202105112%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Li, S. C. et al. The signaling lipid PI(3,5)P<sub>2</sub> stabilizes V<sub>1</sub>-V(o) sector interactions and activates the V-ATPase. <em>Mol. Biol. Cell</em><strong>25</strong>, 1251–1262 (2014).
</cite> [<a href="https://doi.org/10.1091/mbc.E13-10-0563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3982991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24523285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.%20C.%20et%20al.%20The%20signaling%20lipid%20PI(3,5)P2%20stabilizes%20V1-V(o)%20sector%20interactions%20and%20activates%20the%20V-ATPase.%20Mol.%20Biol.%20Cell25,%201251%E2%80%931262%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Banerjee, S., Clapp, K., Tarsio, M. &amp; Kane, P. M. Interaction of the late endo-lysosomal lipid PI(3,5)P2 with the Vph1 isoform of yeast V-ATPase increases its activity and cellular stress tolerance. <em>J. Biol. Chem.</em><strong>294</strong>, 9161–9171 (2019).
</cite> [<a href="https://doi.org/10.1074/jbc.RA119.008552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6556579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31023825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee,%20S.,%20Clapp,%20K.,%20Tarsio,%20M.%20&amp;%20Kane,%20P.%20M.%20Interaction%20of%20the%20late%20endo-lysosomal%20lipid%20PI(3,5)P2%20with%20the%20Vph1%20isoform%20of%20yeast%20V-ATPase%20increases%20its%20activity%20and%20cellular%20stress%20tolerance.%20J.%20Biol.%20Chem.294,%209161%E2%80%939171%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Vasanthakumar, T. et al. Structural comparison of the vacuolar and Golgi V-ATPases from Saccharomyces cerevisiae. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 7272–7277 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1814818116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6462096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30910982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vasanthakumar,%20T.%20et%20al.%20Structural%20comparison%20of%20the%20vacuolar%20and%20Golgi%20V-ATPases%20from%20Saccharomyces%20cerevisiae.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%207272%E2%80%937277%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Zhang, Y.-Q. et al. Requirement for ergosterol in V-ATPase function underlies antifungal activity of azole drugs. <em>PLoS Pathog.</em><strong>6</strong>, e1000939 (2010).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1000939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2880581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20532216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.-Q.%20et%20al.%20Requirement%20for%20ergosterol%20in%20V-ATPase%20function%20underlies%20antifungal%20activity%20of%20azole%20drugs.%20PLoS%20Pathog.6,%20e1000939%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Mitra, C., Winkley, S. &amp; Kane, P. M. Human V-ATPase a-subunit isoforms bind specifically to distinct phosphoinositide phospholipids. <em>J. Biol. Chem.</em><strong>299</strong>, 105473 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2023.105473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10755780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37979916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitra,%20C.,%20Winkley,%20S.%20&amp;%20Kane,%20P.%20M.%20Human%20V-ATPase%20a-subunit%20isoforms%20bind%20specifically%20to%20distinct%20phosphoinositide%20phospholipids.%20J.%20Biol.%20Chem.299,%20105473%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Chu, A. et al. Characterization of a PIP binding site in the N-terminal domain of V-ATPase a4 and its role in plasma membrane association. <em>Int. J. Mol. Sci.</em><strong>24</strong>, 4867 (2023).</cite> [<a href="https://doi.org/10.3390/ijms24054867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10002524/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36902293/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Coupland, C. E. et al. High-resolution electron cryomicroscopy of V-ATPase in native synaptic vesicles. <em>Science</em><strong>385</strong>, 168–174 (2024).
</cite> [<a href="https://doi.org/10.1126/science.adp5577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38900912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coupland,%20C.%20E.%20et%20al.%20High-resolution%20electron%20cryomicroscopy%20of%20V-ATPase%20in%20native%20synaptic%20vesicles.%20Science385,%20168%E2%80%93174%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Tan, J. X. &amp; Finkel, T. A phosphoinositide signalling pathway mediates rapid lysosomal repair. <em>Nature</em>10.1038/s41586-022-05164-4 (2022).</cite> [<a href="https://doi.org/10.1038/s41586-022-05164-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9450835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36071159/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Radulovic, M. et al. Cholesterol transfer via endoplasmic reticulum contacts mediates lysosome damage repair. <em>EMBO J.</em><strong>41</strong>, e112677 (2022).
</cite> [<a href="https://doi.org/10.15252/embj.2022112677" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9753466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36408828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Radulovic,%20M.%20et%20al.%20Cholesterol%20transfer%20via%20endoplasmic%20reticulum%20contacts%20mediates%20lysosome%20damage%20repair.%20EMBO%20J.41,%20e112677%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Ebner, M. et al. Nutrient-regulated control of lysosome function by signaling lipid conversion. <em>Cell</em><strong>186</strong>, 5328–5346.e26 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.09.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37883971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ebner,%20M.%20et%20al.%20Nutrient-regulated%20control%20of%20lysosome%20function%20by%20signaling%20lipid%20conversion.%20Cell186,%205328%E2%80%935346.e26%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Hasegawa, J., Strunk, B. S. &amp; Weisman, L. S. PI5P and PI(3,5)P2: minor, but essential phosphoinositides. <em>Cell Struct. Funct.</em><strong>42</strong>, 49–60 (2017).
</cite> [<a href="https://doi.org/10.1247/csf.17003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5846621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28302928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hasegawa,%20J.,%20Strunk,%20B.%20S.%20&amp;%20Weisman,%20L.%20S.%20PI5P%20and%20PI(3,5)P2:%20minor,%20but%20essential%20phosphoinositides.%20Cell%20Struct.%20Funct.42,%2049%E2%80%9360%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Ghosh, A. et al. PI3P-dependent regulation of cell size and autophagy by phosphatidylinositol 5-phosphate 4-kinase. <em>Life Sci. Alliance</em><strong>6</strong>, e202301920 (2023).</cite> [<a href="https://doi.org/10.26508/lsa.202301920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10267561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37316298/" class="usa-link">PubMed</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Wei, Y. et al. Protocol to derive human trophoblast stem cells directly from primed pluripotent stem cells. <em>STAR Protoc.</em><strong>3</strong>, 101638 (2022).
</cite> [<a href="https://doi.org/10.1016/j.xpro.2022.101638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9420535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36042882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20Y.%20et%20al.%20Protocol%20to%20derive%20human%20trophoblast%20stem%20cells%20directly%20from%20primed%20pluripotent%20stem%20cells.%20STAR%20Protoc.3,%20101638%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Bligh, E. G. &amp; Dyer, W. J. A rapid method of total lipid extraction and purification. <em>Can. J. Biochem. Physiol.</em><strong>37</strong>, 911–917 (1959).
</cite> [<a href="https://doi.org/10.1139/o59-099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13671378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bligh,%20E.%20G.%20&amp;%20Dyer,%20W.%20J.%20A%20rapid%20method%20of%20total%20lipid%20extraction%20and%20purification.%20Can.%20J.%20Biochem.%20Physiol.37,%20911%E2%80%93917%20(1959)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Wunderling, K., Zurkovic, J., Zink, F., Kuerschner, L. &amp; Thiele, C. Triglyceride cycling enables modification of stored fatty acids. <em>Nat. Metab.</em><strong>5</strong>, 699–709 (2023).
</cite> [<a href="https://doi.org/10.1038/s42255-023-00769-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10132980/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37012495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wunderling,%20K.,%20Zurkovic,%20J.,%20Zink,%20F.,%20Kuerschner,%20L.%20&amp;%20Thiele,%20C.%20Triglyceride%20cycling%20enables%20modification%20of%20stored%20fatty%20acids.%20Nat.%20Metab.5,%20699%E2%80%93709%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Thiele, C., Wunderling, K. &amp; Leyendecker, P. Multiplexed and single cell tracing of lipid metabolism. <em>Nat. Methods</em><strong>16</strong>, 1123–1130 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0593-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31611692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thiele,%20C.,%20Wunderling,%20K.%20&amp;%20Leyendecker,%20P.%20Multiplexed%20and%20single%20cell%20tracing%20of%20lipid%20metabolism.%20Nat.%20Methods16,%201123%E2%80%931130%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Wilhelm, L. P. et al. STARD3 mediates endoplasmic reticulum-to-endosome cholesterol transport at membrane contact sites. <em>EMBO J.</em><strong>36</strong>, 1412–1433 (2017).
</cite> [<a href="https://doi.org/10.15252/embj.201695917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5430228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28377464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilhelm,%20L.%20P.%20et%20al.%20STARD3%20mediates%20endoplasmic%20reticulum-to-endosome%20cholesterol%20transport%20at%20membrane%20contact%20sites.%20EMBO%20J.36,%201412%E2%80%931433%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Jokitalo, E., Cabrera-Poch, N., Warren, G. &amp; Shima, D. T. Golgi clusters and vesicles mediate mitotic inheritance independently of the endoplasmic reticulum. <em>J. Cell Biol.</em><strong>154</strong>, 317–330 (2001).
</cite> [<a href="https://doi.org/10.1083/jcb.200104073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2150754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11470821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jokitalo,%20E.,%20Cabrera-Poch,%20N.,%20Warren,%20G.%20&amp;%20Shima,%20D.%20T.%20Golgi%20clusters%20and%20vesicles%20mediate%20mitotic%20inheritance%20independently%20of%20the%20endoplasmic%20reticulum.%20J.%20Cell%20Biol.154,%20317%E2%80%93330%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Belevich, I., Joensuu, M., Kumar, D., Vihinen, H. &amp; Jokitalo, E. Microscopy image browser: a platform for segmentation and analysis of multidimensional datasets. <em>PLoS Biol.</em><strong>14</strong>, e1002340 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1002340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4699692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26727152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Belevich,%20I.,%20Joensuu,%20M.,%20Kumar,%20D.,%20Vihinen,%20H.%20&amp;%20Jokitalo,%20E.%20Microscopy%20image%20browser:%20a%20platform%20for%20segmentation%20and%20analysis%20of%20multidimensional%20datasets.%20PLoS%20Biol.14,%20e1002340%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Berg, S. et al. ilastik: interactive machine learning for (bio)image analysis. <em>Nat. Methods</em><strong>16</strong>, 1226–1232 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0582-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berg,%20S.%20et%20al.%20ilastik:%20interactive%20machine%20learning%20for%20(bio)image%20analysis.%20Nat.%20Methods16,%201226%E2%80%931232%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Callister, S. J. et al. Normalization approaches for removing systematic biases associated with mass spectrometry and label-free proteomics. <em>J. Proteome Res.</em><strong>5</strong>, 277–286 (2006).
</cite> [<a href="https://doi.org/10.1021/pr050300l" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1992440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16457593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callister,%20S.%20J.%20et%20al.%20Normalization%20approaches%20for%20removing%20systematic%20biases%20associated%20with%20mass%20spectrometry%20and%20label-free%20proteomics.%20J.%20Proteome%20Res.5,%20277%E2%80%93286%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Huang, T. et al. Combining precursor and fragment information for improved detection of differential abundance in data independent acquisition*. <em>Mol. Cell. Proteom.</em><strong>19</strong>, 421–430 (2020).</cite> [<a href="https://doi.org/10.1074/mcp.RA119.001705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7000113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31888964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20T.%20et%20al.%20Combining%20precursor%20and%20fragment%20information%20for%20improved%20detection%20of%20differential%20abundance%20in%20data%20independent%20acquisition*.%20Mol.%20Cell.%20Proteom.19,%20421%E2%80%93430%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Ashburner, M. et al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <em>Nat. Genet.</em><strong>25</strong>, 25–29 (2000).
</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashburner,%20M.%20et%20al.%20Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium.%20Nat.%20Genet.25,%2025%E2%80%9329%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Aleksander, S. A. et al. The Gene Ontology knowledgebase in 2023. <em>Genetics</em><strong>224</strong>, iyad031 (2023).
</cite> [<a href="https://doi.org/10.1093/genetics/iyad031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10158837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36866529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aleksander,%20S.%20A.%20et%20al.%20The%20Gene%20Ontology%20knowledgebase%20in%202023.%20Genetics224,%20iyad031%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Thomas, P. D. et al. PANTHER: making genome-scale phylogenetics accessible to all. <em>Protein Sci.</em><strong>31</strong>, 8–22 (2022).
</cite> [<a href="https://doi.org/10.1002/pro.4218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8740835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34717010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thomas,%20P.%20D.%20et%20al.%20PANTHER:%20making%20genome-scale%20phylogenetics%20accessible%20to%20all.%20Protein%20Sci.31,%208%E2%80%9322%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Varadi, M. et al. AlphaFold Protein Structure Database in 2024: providing structure coverage for over 214 million protein sequences. <em>Nucleic Acids Res.</em><strong>52</strong>, D368–D375 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37933859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varadi,%20M.%20et%20al.%20AlphaFold%20Protein%20Structure%20Database%20in%202024:%20providing%20structure%20coverage%20for%20over%20214%20million%20protein%20sequences.%20Nucleic%20Acids%20Res.52,%20D368%E2%80%93D375%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. <em>J. Comput. Chem.</em><strong>29</strong>, 1859–1865 (2008).
</cite> [<a href="https://doi.org/10.1002/jcc.20945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18351591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jo,%20S.,%20Kim,%20T.,%20Iyer,%20V.%20G.%20&amp;%20Im,%20W.%20CHARMM-GUI:%20a%20web-based%20graphical%20user%20interface%20for%20CHARMM.%20J.%20Comput.%20Chem.29,%201859%E2%80%931865%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. <em>Nat. Methods</em><strong>14</strong>, 71–73 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5199616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27819658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20et%20al.%20CHARMM36m:%20an%20improved%20force%20field%20for%20folded%20and%20intrinsically%20disordered%20proteins.%20Nat.%20Methods14,%2071%E2%80%9373%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. &amp; Klein, M. L. Comparison of simple potential functions for simulating liquid water. <em>J. Chem. Phys.</em><strong>79</strong>, 926–935 (1983).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jorgensen,%20W.%20L.,%20Chandrasekhar,%20J.,%20Madura,%20J.%20D.,%20Impey,%20R.%20W.%20&amp;%20Klein,%20M.%20L.%20Comparison%20of%20simple%20potential%20functions%20for%20simulating%20liquid%20water.%20J.%20Chem.%20Phys.79,%20926%E2%80%93935%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Beglov, D. &amp; Roux, B. Finite representation of an infinite bulk system: solvent boundary potential for computer simulations. <em>J. Chem. Phys.</em><strong>100</strong>, 9050–9063 (1994).</cite> [<a href="https://scholar.google.com/scholar_lookup?Beglov,%20D.%20&amp;%20Roux,%20B.%20Finite%20representation%20of%20an%20infinite%20bulk%20system:%20solvent%20boundary%20potential%20for%20computer%20simulations.%20J.%20Chem.%20Phys.100,%209050%E2%80%939063%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. <em>SoftwareX</em><strong>1–2</strong>, 19–25 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abraham,%20M.%20J.%20et%20al.%20GROMACS:%20high%20performance%20molecular%20simulations%20through%20multi-level%20parallelism%20from%20laptops%20to%20supercomputers.%20SoftwareX1%E2%80%932,%2019%E2%80%9325%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Hopkins, C. W., Le Grand, S., Walker, R. C. &amp; Roitberg, A. E. Long-time-step molecular dynamics through hydrogen mass repartitioning. <em>J. Chem. Theory Comput.</em><strong>11</strong>, 1864–1874 (2015).
</cite> [<a href="https://doi.org/10.1021/ct5010406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hopkins,%20C.%20W.,%20Le%20Grand,%20S.,%20Walker,%20R.%20C.%20&amp;%20Roitberg,%20A.%20E.%20Long-time-step%20molecular%20dynamics%20through%20hydrogen%20mass%20repartitioning.%20J.%20Chem.%20Theory%20Comput.11,%201864%E2%80%931874%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Nosé, S. A molecular dynamics method for simulations in the canonical ensemble. <em>Mol. Phys.</em><strong>52</strong>, 255–268 (1984).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nos%C3%A9,%20S.%20A%20molecular%20dynamics%20method%20for%20simulations%20in%20the%20canonical%20ensemble.%20Mol.%20Phys.52,%20255%E2%80%93268%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Hoover, W. G. Canonical dynamics: equilibrium phase-space distributions. <em>Phys. Rev. A</em><strong>31</strong>, 1695–1697 (1985).</cite> [<a href="https://doi.org/10.1103/physreva.31.1695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9895674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoover,%20W.%20G.%20Canonical%20dynamics:%20equilibrium%20phase-space%20distributions.%20Phys.%20Rev.%20A31,%201695%E2%80%931697%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Parrinello, M. &amp; Rahman, A. Polymorphic transitions in single crystals: a new molecular dynamics method. <em>J. Appl. Phys.</em><strong>52</strong>, 7182–7190 (1981).</cite> [<a href="https://scholar.google.com/scholar_lookup?Parrinello,%20M.%20&amp;%20Rahman,%20A.%20Polymorphic%20transitions%20in%20single%20crystals:%20a%20new%20molecular%20dynamics%20method.%20J.%20Appl.%20Phys.52,%207182%E2%80%937190%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Hess, B. P-LINCS:  a parallel linear constraint solver for molecular simulation. <em>J. Chem. Theory Comput.</em><strong>4</strong>, 116–122 (2008).
</cite> [<a href="https://doi.org/10.1021/ct700200b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26619985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hess,%20B.%20P-LINCS:%E2%80%89%20a%20parallel%20linear%20constraint%20solver%20for%20molecular%20simulation.%20J.%20Chem.%20Theory%20Comput.4,%20116%E2%80%93122%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Essmann, U. et al. A smooth particle mesh Ewald method. <em>J. Chem. Phys.</em><strong>103</strong>, 8577–8593 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Essmann,%20U.%20et%20al.%20A%20smooth%20particle%20mesh%20Ewald%20method.%20J.%20Chem.%20Phys.103,%208577%E2%80%938593%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Humphrey, W., Dalke, A. &amp; Schulten, K. V. M. D. Visual molecular dynamics. <em>J. Mol. Graph.</em><strong>14</strong>, 33–38 (1996).
</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Humphrey,%20W.,%20Dalke,%20A.%20&amp;%20Schulten,%20K.%20V.%20M.%20D.%20Visual%20molecular%20dynamics.%20J.%20Mol.%20Graph.14,%2033%E2%80%9338%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Gowers, R. J. et al. MDAnalysis: a Python package for the rapid analysis of molecular dynamics simulations MDAnalysis. In <em>Proc. 15th Python Sci. Conf</em>. 98–105 10.25080/Majora-629e541a-00e (SciPy Proceedings, 2016).</cite>
</li>
<li id="CR100">
<span class="label">100.</span><cite>Michaud-Agrawal, N., Denning, E. J., Woolf, T. B. &amp; Beckstein, O. MDAnalysis: a toolkit for the analysis of molecular dynamics simulations. <em>J. Comput. Chem.</em><strong>32</strong>, 2319–2327 (2011).
</cite> [<a href="https://doi.org/10.1002/jcc.21787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3144279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21500218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michaud-Agrawal,%20N.,%20Denning,%20E.%20J.,%20Woolf,%20T.%20B.%20&amp;%20Beckstein,%20O.%20MDAnalysis:%20a%20toolkit%20for%20the%20analysis%20of%20molecular%20dynamics%20simulations.%20J.%20Comput.%20Chem.32,%202319%E2%80%932327%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Kramer, M. A. Autoassociative neural networks. <em>Comput. Chem. Eng.</em><strong>16</strong>, 313–328 (1992).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kramer,%20M.%20A.%20Autoassociative%20neural%20networks.%20Comput.%20Chem.%20Eng.16,%20313%E2%80%93328%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Paszke, A. et al. PyTorch: an imperative style, high-performance deep learning library. <em>arXiv</em>10.48550/arXiv.1912.01703 (2019).</cite>
</li>
<li id="CR103">
<span class="label">103.</span><cite>Agarap, A. F. M. Deep learning using rectified linear units (ReLU). <em>arXiv</em>10.48550/arXiv.1803.08375 (2019).</cite>
</li>
<li id="CR104">
<span class="label">104.</span><cite>Pedregosa, F. et al. Generating the blood exposome database using a comprehensive text mining and database fusion approach. <em>J. Mach. Learn. Res.</em><strong>12</strong>, 2825–2830 (2011).</cite> [<a href="https://doi.org/10.1289/EHP4713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6794490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31557052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pedregosa,%20F.%20et%20al.%20Generating%20the%20blood%20exposome%20database%20using%20a%20comprehensive%20text%20mining%20and%20database%20fusion%20approach.%20J.%20Mach.%20Learn.%20Res.12,%202825%E2%80%932830%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Perez-Riverol, Y. et al. The PRIDE database at 20 years: 2025 update. <em>Nucleic Acids Res.</em><strong>53</strong>, D543–D553 (2025).
</cite> [<a href="https://doi.org/10.1093/nar/gkae1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39494541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez-Riverol,%20Y.%20et%20al.%20The%20PRIDE%20database%20at%2020%20years:%202025%20update.%20Nucleic%20Acids%20Res.53,%20D543%E2%80%93D553%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (70.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63125_MOESM2_ESM.pdf</a><sup> (84.9KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (2.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM5_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (5.4MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM6_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (5.6MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (131.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (99.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371084/bin/41467_2025_63125_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (5.4MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The gnomAD constraint metrics TSV data set v4.0.0 is publicly available via: gnomAD (broadinstitute.org). The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup> partner repository with the dataset identifier <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD065794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDX065794</a>. <a href="#Sec48" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63125-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63125.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371084/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371084/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371084%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371084/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371084/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371084/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841558/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371084/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841558/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371084/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371084/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Y6kEDALJpp5o6BoW8pdGCa078Jl9ppJOeOnAEWpRO3cqWPBWmk3Q87HpKkt1dGmR">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
