
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Chromosome‐level haplotype‐resolved genome assembly provides insights into the highly heterozygous genome of Italian ryegrass (Lolium multiflorum Lam.) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533A6A8AF33173053A6A004DEDCE93.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376112/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Chromosome‐level haplotype‐resolved genome assembly provides insights into the highly heterozygous genome of Italian ryegrass (Lolium multiflorum Lam.)">
<meta name="citation_author" content="Yutang Chen">
<meta name="citation_author_institution" content="Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Jenny Kiesbauer">
<meta name="citation_author_institution" content="Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland">
<meta name="citation_author_institution" content="Fodder Plant Breeding, Division of Plant Breeding, Agroscope, Zurich, Switzerland">
<meta name="citation_author" content="Dario Copetti">
<meta name="citation_author_institution" content="Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland">
<meta name="citation_author_institution" content="Arizona Genomics Institute, School of Plant Sciences, University of Arizona, Tucson, Arizona, USA">
<meta name="citation_author" content="Daniel Frei">
<meta name="citation_author_institution" content="Department of Methods Development and Analytics, Agroscope, Wädenswil, Switzerland">
<meta name="citation_author_institution" content="Institute of Medical Virology, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Jürg E Frey">
<meta name="citation_author_institution" content="Department of Methods Development and Analytics, Agroscope, Wädenswil, Switzerland">
<meta name="citation_author" content="Christoph Grieder">
<meta name="citation_author_institution" content="Fodder Plant Breeding, Division of Plant Breeding, Agroscope, Zurich, Switzerland">
<meta name="citation_author" content="Roland Kölliker">
<meta name="citation_author_institution" content="Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Bruno Studer">
<meta name="citation_author_institution" content="Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70079">
<meta name="citation_doi" content="10.1002/tpg2.70079">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376112/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376112/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376112/pdf/TPG2-18-e70079.pdf">
<meta name="description" content="Italian ryegrass (Lolium multiflorum Lam.) is an important forage grass, providing a major source of feed for ruminants in temperate regions. Due to its highly heterozygous and repeat‐rich genome, high‐quality chromosome‐level genome assemblies are ...">
<meta name="og:title" content="Chromosome‐level haplotype‐resolved genome assembly provides insights into the highly heterozygous genome of Italian ryegrass (Lolium multiflorum Lam.)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Italian ryegrass (Lolium multiflorum Lam.) is an important forage grass, providing a major source of feed for ruminants in temperate regions. Due to its highly heterozygous and repeat‐rich genome, high‐quality chromosome‐level genome assemblies are ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376112/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12376112">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70079"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70079.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376112%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12376112/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12376112/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376112/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70079" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 25;18(3):e70079. doi: <a href="https://doi.org/10.1002/tpg2.70079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70079</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Chromosome‐level haplotype‐resolved genome assembly provides insights into the highly heterozygous genome of Italian ryegrass (<em>Lolium multiflorum</em> Lam.)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yutang Chen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yutang Chen</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yutang Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiesbauer%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jenny Kiesbauer</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jenny Kiesbauer</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Fodder Plant Breeding, Division of Plant Breeding, Agroscope, Zurich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiesbauer%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jenny Kiesbauer</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Copetti%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Dario Copetti</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Dario Copetti</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Arizona Genomics Institute, School of Plant Sciences, University of Arizona, Tucson, Arizona, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Copetti%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dario Copetti</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Frei%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Daniel Frei</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Daniel Frei</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Methods Development and Analytics, Agroscope, Wädenswil, Switzerland</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Institute of Medical Virology, University of Zurich, Zurich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Frei%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Frei</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Frey%20JE%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jürg E Frey</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jürg E Frey</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Methods Development and Analytics, Agroscope, Wädenswil, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Frey%20JE%22%5BAuthor%5D" class="usa-link"><span class="name western">Jürg E Frey</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grieder%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Christoph Grieder</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Christoph Grieder</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Fodder Plant Breeding, Division of Plant Breeding, Agroscope, Zurich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grieder%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christoph Grieder</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6lliker%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Roland Kölliker</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Roland Kölliker</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6lliker%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Roland Kölliker</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Studer%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Bruno Studer</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Bruno Studer</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Studer%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bruno Studer</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270079-aff-0001">
<sup>
<sup>1</sup>
</sup>Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, Zurich, Switzerland</div>
<div id="tpg270079-aff-0002">
<sup>
<sup>2</sup>
</sup>Fodder Plant Breeding, Division of Plant Breeding, Agroscope, Zurich, Switzerland</div>
<div id="tpg270079-aff-0003">
<sup>
<sup>3</sup>
</sup>Arizona Genomics Institute, School of Plant Sciences, University of Arizona, Tucson, Arizona, USA</div>
<div id="tpg270079-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Methods Development and Analytics, Agroscope, Wädenswil, Switzerland</div>
<div id="tpg270079-aff-0005">
<sup>
<sup>5</sup>
</sup>Institute of Medical Virology, University of Zurich, Zurich, Switzerland</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Bruno Studer, Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, 8092, Zurich, Switzerland. Email: <span>bruno.studer@usys.ethz.ch</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 13; Received 2024 Oct 21; Accepted 2025 Jun 18; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12376112</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Italian ryegrass (<em>Lolium multiflorum</em> Lam.) is an important forage grass, providing a major source of feed for ruminants in temperate regions. Due to its highly heterozygous and repeat‐rich genome, high‐quality chromosome‐level genome assemblies are scarce for Italian ryegrass. Here, we sequenced the genome of a genotype from the Italian ryegrass cultivar ‘Rabiosa’ (hereafter referred to as Rabiosa), and we obtained Oxford Nanopore Technologies long reads (∼60‐fold coverage), Illumina short reads (∼85‐fold coverage), and high‐throughput chromosome conformation capture data (∼60‐fold coverage). With Rabiosa as one of the parents, we constructed an F<sub>1</sub> population consisting of 304 individuals, which were genotyped by reduced representation sequencing for linkage map construction and quantitative trait locus (QTL) analysis. Using whole‐genome sequencing data of Rabiosa and the genetic linkage map, we first generated a chromosome‐level unphased haploid assembly (scaffold N50 of 338.75 Mb, total Benchmarking Universal Single‐Copy Orthologs [BUSCO] score of 94.60%). Then, based on the unphased assembly and a reference‐based phasing approach, we generated a chromosome‐level haplotype‐resolved assembly containing both haplotypes (scaffold N50 of ∼250 Mb and total BUSCO score of ∼90% for each haplome). Between the two haplotypes of Rabiosa, we observed a highly collinear gene order at the chromosome level and a high sequence variation at the local level. With a graph‐based reference built from the unphased and the haplotype‐resolved assemblies of Rabiosa, we conducted a QTL analysis, and two QTL significantly associated with stem rust resistance were detected. The genome assemblies of Rabiosa will serve as invaluable genomic resources to facilitate genomic applications in forage grass research and breeding.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270079-list-10001" class="list" style="list-style-type:disc">
<li><p>The first chromosome‐level haplotype‐resolved assembly for highly heterozygous Italian ryegrass was generated.</p></li>
<li><p>A customized haplotig purging pipeline and a reference‐based phasing pipeline were established.</p></li>
<li><p>Highly consistent gene order but low local sequence identity was observed between the two haplotypes.</p></li>
<li><p>Two quantitative trait loci (QTL) significantly associated with stem rust resistance were detected using a graph‐based reference.</p></li>
</ul></section><section class="abstract" id="abstract3"><h2>Plain Language Summary</h2>
<p>Italian ryegrass is an important forage crop used to feed livestock in temperate regions worldwide. However, the nature and complexity of its genome limit the application of genomics‐based breeding strategies. Using a combination of advanced sequencing technologies, including long and short DNA reads and long‐range linkage information, we provide the first genome assembly of Italian ryegrass that is separated for the two sets of chromosomes inherited from each parent (i.e., haplotype‐resolved). We validated the usefulness of our genome assemblies for genetic analyses and identified two genome regions that are associated with resistance to stem rust, an major disease for Italian ryegrass. Our genome assemblies provide a valuable resource for future research and breeding efforts aimed at improving Italian ryegrass and other forage crops.</p></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270079-dl-0001">
<dt>BLUE</dt>
<dd><p>best linear unbiased estimator</p></dd>
<dt>BUSCO</dt>
<dd><p>Benchmarking Universal Single‐Copy Orthologs</p></dd>
<dt>CIM</dt>
<dd><p>composite interval mapping</p></dd>
<dt>GBS</dt>
<dd><p>genotyping‐by‐sequencing</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>Hi‐C</dt>
<dd><p>high‐throughput chromosome conformation capture</p></dd>
<dt>INDEL</dt>
<dd><p>insertion and deletion</p></dd>
<dt>LG</dt>
<dd><p>linkage group</p></dd>
<dt>LOD</dt>
<dd><p>logarithm of odds</p></dd>
<dt>ONT</dt>
<dd><p>Oxford Nanopore Technologies</p></dd>
<dt>PacBio HiFi</dt>
<dd><p>Pacific Biosciences' high‐fidelity sequencing technology</p></dd>
<dt>PAV</dt>
<dd><p>presence and absence variation</p></dd>
<dt>PGGB</dt>
<dd><p>pangenome graph builder</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus/loci</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>TE</dt>
<dd><p>transposable element</p></dd>
<dt>WGA</dt>
<dd><p>whole‐genome alignment</p></dd>
<dt>WGS</dt>
<dd><p>whole‐genome sequencing</p></dd>
</dl>
<section id="tpg270079-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Italian ryegrass (<em>Lolium multiflorum</em> Lam.) is one of the most important forage grasses widely grown in temperate regions across the world, providing highly digestible and palatable feed for livestock (Boller et al., <a href="#tpg270079-bib-0008" class="usa-link" aria-describedby="tpg270079-bib-0008">2010</a>). It is a diploid, outcrossing species containing seven pairs of chromosomes (2<em>n</em> = 2<em>x</em> = 14) with a very high level of heterozygosity (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>). Its genome size is estimated to be around 2.5 Gb (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>), of which 70% are repetitive sequences (Zwyrtková et al., <a href="#tpg270079-bib-0102" class="usa-link" aria-describedby="tpg270079-bib-0102">2020</a>). The relatively large genome size, together with the high degree of repetitiveness and heterozygosity, makes genome assembly of this species very challenging. As a result, so far only two draft genome assemblies have been reported with one being highly fragmented (Knorst et al., <a href="#tpg270079-bib-0049" class="usa-link" aria-describedby="tpg270079-bib-0049">2019</a>) and the other being haplotype‐redundant (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>). Thanks to recent advances in long‐read sequencing technologies, a chromosome‐level haploid assembly of Italian ryegrass has been generated with Pacific Biosciences’ high‐fidelity sequencing technology (PacBio HiFi) long reads (Brunharo et al., <a href="#tpg270079-bib-0011" class="usa-link" aria-describedby="tpg270079-bib-0011">2025</a>). While this haploid assembly showed a high quality in terms of contiguity, completeness, and correctness, it may miss some important alleles of its heterozygous diploid genotype. Biased results might be obtained for downstream genomic applications when relying on a haploid reference. Therefore, to further advance forage grass research and breeding with the rapidly advancing long‐read sequencing technologies, haplotype‐resolved diploid assemblies containing both haplotypes are now needed.</p>
<p>Traditionally, when de novo assembling a heterozygous diploid genome, the assembly algorithm randomly selects only one of the two alleles to be present in the output sequence, leading to a mosaic haploid assembly with alleles among loci randomly alternating between haplotypes (Chin et al., <a href="#tpg270079-bib-0018" class="usa-link" aria-describedby="tpg270079-bib-0018">2016</a>; Garg et al., <a href="#tpg270079-bib-0028" class="usa-link" aria-describedby="tpg270079-bib-0028">2021</a>). With half of the genetic variation missing, a mosaic haploid assembly is never the best representation of a highly heterozygous diploid genome. Ideally, a heterozygous diploid genome should always be represented as a haplotype‐resolved assembly where alleles from both homologous chromosomes are not only present but also correctly combined or phased (Cheng et al., <a href="#tpg270079-bib-0017" class="usa-link" aria-describedby="tpg270079-bib-0017">2021</a>; Garg et al., <a href="#tpg270079-bib-0028" class="usa-link" aria-describedby="tpg270079-bib-0028">2021</a>; Koren et al., <a href="#tpg270079-bib-0051" class="usa-link" aria-describedby="tpg270079-bib-0051">2018</a>; Llamas et al., <a href="#tpg270079-bib-0058" class="usa-link" aria-describedby="tpg270079-bib-0058">2021</a>). To generate haplotype‐resolved assemblies with long‐read sequencing technologies such as Oxford Nanopore Technologies (ONT) or PacBio HiFi, so far, three approaches have been developed and adopted: bin‐then‐assemble, such as trio‐binning (Koren et al., <a href="#tpg270079-bib-0051" class="usa-link" aria-describedby="tpg270079-bib-0051">2018</a>) and reference‐based phasing (Garg et al., <a href="#tpg270079-bib-0028" class="usa-link" aria-describedby="tpg270079-bib-0028">2021</a>; Porubsky et al., <a href="#tpg270079-bib-0075" class="usa-link" aria-describedby="tpg270079-bib-0075">2021</a>); assemble‐then‐bin, such as ALLHiC (Guk et al., <a href="#tpg270079-bib-0037" class="usa-link" aria-describedby="tpg270079-bib-0037">2022</a>), HapHiC (Zeng et al., <a href="#tpg270079-bib-0097" class="usa-link" aria-describedby="tpg270079-bib-0097">2024</a>), and phasing with pollen or pedigree sequencing data (Bao et al., <a href="#tpg270079-bib-0004" class="usa-link" aria-describedby="tpg270079-bib-0004">2022</a>; Serra Mari et al., <a href="#tpg270079-bib-0083" class="usa-link" aria-describedby="tpg270079-bib-0083">2024</a>; Sun et al., <a href="#tpg270079-bib-0088" class="usa-link" aria-describedby="tpg270079-bib-0088">2022</a>; Q. Zhou et al., <a href="#tpg270079-bib-0099" class="usa-link" aria-describedby="tpg270079-bib-0099">2020</a>); and graph‐based phasing, such as Hifiasm (Cheng et al., <a href="#tpg270079-bib-0017" class="usa-link" aria-describedby="tpg270079-bib-0017">2021</a>) and Verkko (Rautiainen et al., <a href="#tpg270079-bib-0077" class="usa-link" aria-describedby="tpg270079-bib-0077">2023</a>). These methods have generated promising haplotype‐resolved assemblies, suggesting that it is now feasible to generate haplotype‐resolved assemblies also for highly heterozygous grass species such as Italian ryegrass.</p>
<p>Recently, a graph‐based human pangenome reference was published (Liao et al., <a href="#tpg270079-bib-0056" class="usa-link" aria-describedby="tpg270079-bib-0056">2023</a>), showing the advantage of overcoming reference bias using a pangenome reference instead of a single reference. In plant sciences, more and more pangenomic studies for grass species such as wheat (<em>Triticum aestivum</em> L., Walkowiak et al., <a href="#tpg270079-bib-0093" class="usa-link" aria-describedby="tpg270079-bib-0093">2020</a>), barley (<em>Hordeum vulgare</em> L., Jayakodi et al., <a href="#tpg270079-bib-0045" class="usa-link" aria-describedby="tpg270079-bib-0045">2020</a>), maize (<em>Zea mays</em> L., Gui et al., <a href="#tpg270079-bib-0036" class="usa-link" aria-describedby="tpg270079-bib-0036">2022</a>), rice (<em>Oryza sativa</em> L., Shang et al., <a href="#tpg270079-bib-0084" class="usa-link" aria-describedby="tpg270079-bib-0084">2022</a>), and various species of millet (<em>Panicum miliaceum</em> L., J. Chen et al., <a href="#tpg270079-bib-0014" class="usa-link" aria-describedby="tpg270079-bib-0014">2023</a>; <em>Setaria italica</em> (L.) P. Beauv., He et al., <a href="#tpg270079-bib-0041" class="usa-link" aria-describedby="tpg270079-bib-0041">2023</a>; <em>Pennisetum glaucum</em> (L.) R. Br., Yan et al., <a href="#tpg270079-bib-0096" class="usa-link" aria-describedby="tpg270079-bib-0096">2023</a>) have also emerged. All these studies illustrate the transition from single‐reference‐dependent genomics to pangenomic approaches. Given its high level of genetic diversity and its importance as a source of forage, pangenomic resources for Italian ryegrass should be generated. Moreover, to pave the way for future pangenomic studies for <em>Lolium</em> species, methods such as pangenome graph construction, haplotype‐aware read mapping, and variant genotyping with the graph‐based reference could be tested with Italian ryegrass.</p>
<p>In this work, we first aimed at generating a haplotype‐resolved genome assembly for a highly heterozygous Italian ryegrass genotype using ONT long reads with high‐throughput chromosome conformation capture (Hi‐C) data and a genetic linkage map. Then, based on the resulting haplomes, we aimed at understanding the high level of heterozygosity in the genome and addressing how high heterozygosity might affect genome assembly and phasing. Finally, to explore the potential of current pangenomic methods, we aimed at building a graph‐based reference based on the haplotype‐resolved assembly and conducting quantitative trait locus (QTL) analysis for resistance to stem rust with the graph‐based reference.</p></section><section id="tpg270079-sec-0020"><h2 class="pmc_sec_title">2. METHODS</h2>
<section id="tpg270079-sec-0030"><h3 class="pmc_sec_title">2.1. Plant material</h3>
<p>A genotype (M.02402/16) of the Swiss Italian ryegrass cultivar ‘Rabiosa’ (Agroscope) was selected for whole‐genome sequencing (WGS). This genotype, referred to as Rabiosa, was crossed with a genotype (M.2002/18) of another Italian ryegrass cultivar ‘Sikem’ (DLF Seeds A/S), resulting in a bi‐parental population of 304 F<sub>1</sub> individuals. Both Rabiosa and Sikem are highly heterozygous genotypes.</p></section><section id="tpg270079-sec-0040"><h3 class="pmc_sec_title">2.2. Plant material for QTL analysis</h3>
<p>In total, 304 F<sub>1</sub> seeds, collected from both Rabiosa and Sikem spikes, were germinated for 2 days in Petri dishes. The seedlings were transplanted to the greenhouse and were grown there for approximately 4 weeks. Two clonal replicates of a subset of 135 plants (133 F<sub>1</sub> individuals and the two parents) were planted in autumn 2018 in a first environment at Zurich (47.427° N, 8.516° E) using an alpha design. Because of severe drought, only one replicate of each plant could be phenotyped in 2019. After phenotyping in 2019, the plants were dug out and divided into three clonal replicates and planted in a second environment at Zurich (47.427° N, 8.516° E) using an alpha design. After phenotyping in 2020, these plants were again transplanted to a third environment at Zurich (47.427° N, 8.516° E) using an alpha design and again phenotyped in three clonal replicates in 2021.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270079-sec-0050"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270079-list-0001" class="list" style="list-style-type:disc">
<li><p>The first chromosome‐level haplotype‐resolved assembly for highly heterozygous Italian ryegrass was generated.</p></li>
<li><p>A customized haplotig purging pipeline and a reference‐based phasing pipeline were established.</p></li>
<li><p>Highly consistent gene order but low local sequence identity was observed between the two haplotypes.</p></li>
<li><p>Two quantitative trait loci (QTL) significantly associated with stem rust resistance were detected using a graph‐based reference.</p></li>
</ul></section></section></section><section id="tpg270079-sec-0060"><h3 class="pmc_sec_title">2.3. Genome sequencing</h3>
<p>ONT sequencing for Rabiosa was conducted following the method described in Frei et al. (<a href="#tpg270079-bib-0027" class="usa-link" aria-describedby="tpg270079-bib-0027">2021</a>). Hi‐C library preparation and sequencing for Rabiosa were performed at the Functional Genomic Center Zurich (FGCZ), Switzerland. Illumina WGS data of Rabiosa were generated by NRGene as described previously (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>). For the bi‐parental population, genotyping‐by‐sequencing (GBS) libraries were prepared with the restriction enzyme <em>Pst</em>I as described by Begheyn et al. (<a href="#tpg270079-bib-0006" class="usa-link" aria-describedby="tpg270079-bib-0006">2018</a>) and sequenced with Illumina HiSeq 2500 at FGCZ.</p></section><section id="tpg270079-sec-0070"><h3 class="pmc_sec_title">2.4. Rabiosa genome assembly</h3>
<p>For Rabiosa v1, ONT long reads were first assembled using Flye 2.8.3 (Kolmogorov et al., <a href="#tpg270079-bib-0050" class="usa-link" aria-describedby="tpg270079-bib-0050">2019</a>) with –min‐overlap 10,000 and –iterations 2, then the contigs were polished with accurate short reads using Polca (Zimin &amp; Salzberg, <a href="#tpg270079-bib-0101" class="usa-link" aria-describedby="tpg270079-bib-0101">2020</a>) for one round. After polishing, haplotigs were removed from the polished assembly using a customized haplotig purge pipeline (Section <a href="#tpg270079-sec-0100" class="usa-link">2.7</a>). Hi‐C data were mapped to the haplotig‐purged assembly using the Arima Hi‐C mapping pipeline (<a href="https://github.com/ArimaGenomics/mapping_pipeline" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ArimaGenomics/mapping_pipeline</a>), and SALSA2 (Ghurye et al., <a href="#tpg270079-bib-0031" class="usa-link" aria-describedby="tpg270079-bib-0031">2019</a>) was used to scaffold the haplotig‐purged assembly with the Hi‐C mapping results. The scaffolds generated by SALSA2 were further anchored to pseudo‐chromosomes using ALLMAPS (Tang et al., <a href="#tpg270079-bib-0089" class="usa-link" aria-describedby="tpg270079-bib-0089">2015</a>) based on the genetic linkage map generated with the GBS data from the bi‐parental F<sub>1</sub> population (Section <a href="#tpg270079-sec-0090" class="usa-link">2.6</a>). The resulting pseudo‐chromosomes from ALLMAPS were used to anchor the scaffolds generated by SALSA2 again using the Tritex pipeline (Monat et al., <a href="#tpg270079-bib-0065" class="usa-link" aria-describedby="tpg270079-bib-0065">2019</a>) with Hi‐C data, producing the Rabiosa v1 assembly.</p>
<p>For Rabiosa v2, ONT long reads were first corrected with accurate short reads using FMLRC2 (Mak et al., <a href="#tpg270079-bib-0059" class="usa-link" aria-describedby="tpg270079-bib-0059">2023</a>). Then, the corrected ONT long reads were assembled using Canu 2.2 (Koren et al., <a href="#tpg270079-bib-0052" class="usa-link" aria-describedby="tpg270079-bib-0052">2017</a>) with batOptions = ‐eg 0.12 ‐sb 0.01 ‐dg 3 ‐db 3 ‐dr 1 ‐ca 500 ‐cp 50. One round of short‐read polishing was applied to the Canu assembly using Polca (Zimin &amp; Salzberg, <a href="#tpg270079-bib-0101" class="usa-link" aria-describedby="tpg270079-bib-0101">2020</a>). After polishing, the same customized haplotig purge pipeline was applied, and then the haplotig‐purged contigs were scaffolded to pseudo‐chromosomes using the Tritex pipeline with Hi‐C data and Rabiosa v1 as the reference. For the resulting chromosome‐level assembly, a Hi‐C contact map was generated using Juicer (Durand, Shamim, et al., <a href="#tpg270079-bib-0024" class="usa-link" aria-describedby="tpg270079-bib-0024">2016</a>) and 3d‐DNA (Dudchenko et al., <a href="#tpg270079-bib-0022" class="usa-link" aria-describedby="tpg270079-bib-0022">2017</a>), and manual curation was conducted using Juicebox (Durand, Robinson, et al., <a href="#tpg270079-bib-0023" class="usa-link" aria-describedby="tpg270079-bib-0023">2016</a>). This resulted in the Rabiosa v2 assembly.</p></section><section id="tpg270079-sec-0080"><h3 class="pmc_sec_title">2.5. GBS data pre‐processing</h3>
<p>GBS data from the bi‐parental F<sub>1</sub> population were first demultiplexed using Sabre (Najoshi, <a href="#tpg270079-bib-0068" class="usa-link" aria-describedby="tpg270079-bib-0068">2022</a>), and quality check was performed using FastQC (Andrews, <a href="#tpg270079-bib-0003" class="usa-link" aria-describedby="tpg270079-bib-0003">2015</a>). Later, adapters were trimmed off using Trimmomatic (Bolger et al., <a href="#tpg270079-bib-0007" class="usa-link" aria-describedby="tpg270079-bib-0007">2014</a>).</p></section><section id="tpg270079-sec-0090"><h3 class="pmc_sec_title">2.6. Genetic linkage map construction</h3>
<p>Several genetic linkage maps were constructed in this work. The first genetic linkage map was the one used for scaffolding Rabiosa v1. Clean GBS reads obtained from the above methods were mapped to the scaffolds generated with SALSA2 using the short‐read aligner BWA‐MEM (Li, <a href="#tpg270079-bib-0053" class="usa-link" aria-describedby="tpg270079-bib-0053">2013</a>). Then, single nucleotide polymorphisms (SNPs) were called using SAMtools and BCFtools (Danecek et al., <a href="#tpg270079-bib-0021" class="usa-link" aria-describedby="tpg270079-bib-0021">2021</a>). Ungenotyped SNPs were imputed using LinkImputeR (Money et al., <a href="#tpg270079-bib-0066" class="usa-link" aria-describedby="tpg270079-bib-0066">2017</a>), and the genetic linkage map was built using Lep‐MAP3 (Rastas, <a href="#tpg270079-bib-0076" class="usa-link" aria-describedby="tpg270079-bib-0076">2017</a>) by sorting and ordering markers based on the recombination fraction between SNP markers. When building the genetic linkage map with Lep‐MAP3, Rabiosa was considered as parent 1 and Sikem was considered as parent 2 for all F<sub>1</sub> individuals, as these were derived from a bi‐directional pair‐wise cross. Informative SNP markers (markers that are heterozygous in either or both parents) were selected for map construction. These SNP markers were separated into seven linkage groups (LGs) using SeparateChromosomes2 from Lep‐MAP3, with lodLimit = 20, <em>θ</em> = 0.2.</p>
<p>A second batch of genetic linkage maps was generated based on SNPs called against pseudo‐chromosomes of Rabisoa v2 and the haplotype assemblies (Rabiosa h1 and h2, Section <a href="#tpg270079-sec-0110" class="usa-link">2.8</a>), following the same read mapping and variant calling methods. Prior to linkage map construction, no imputation was made to fill missing genotypes in the variant calling file, and F<sub>1</sub> individuals were filtered if more than 30% of their SNPs were not genotyped. Lep‐MAP3 (Rastas, <a href="#tpg270079-bib-0076" class="usa-link" aria-describedby="tpg270079-bib-0076">2017</a>) was again used for linkage map construction. Informative markers from both parents were used for constructing consensus maps, aiming to include as many SNPs as possible into the maps for follow‐up validation. The genetic map distance of the SNP markers was calculated with the given order (the physical position) of the SNP markers from the assemblies (OrderMarkers2 with improveOrder = 0). By comparing the physical position with the genetic map position of the SNP markers, these maps were used to validate the structural correctness of the pseudo‐chromosomes in the assemblies (Figure <a href="#tpg270079-fig-0004" class="usa-link">4a</a>, track B). If the structure of the pseudo‐chromosome is correct, then the physical position of the markers should correlate well with the corresponding genetic map position. Discrepancies between the physical position and the genetic map position, such as a steep slope in the correlation plot (a short distance on the <em>x</em>‐axis, the physical position, corresponds to a much longer distance on the <em>y</em>‐axis, the genetic linkage map position, between two markers), might indicate structural mistakes in the assembly.</p>
<figure class="fig xbox font-sm" id="tpg270079-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a81a/12376112/4dd7370e92af/TPG2-18-e70079-g002.jpg" loading="lazy" id="jats-graphic-7" height="608" width="700" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270079-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Circos plot showing genomic features of Rabiosa v2, h1, and h2. Track A shows pseudo‐chromosomes of Rabiosa v2 (gray), h1 (red), and h2 (blue), respectively; track B shows the base pair position in the assembly versus genetic linkage map position; track C shows the number of high‐confidence genes per 10 Mb; track D shows the number of low‐confidence genes per 10 Mb; track E shows the fraction of repetitive sequences per 10 Mb; and track F shows the GC content per 10 Mb. (b) Genome synteny between Rabiosa and barley cultivar Morex. (c) Sequence variation between Rabiosa h1 and Rabiosa h2. The distribution of gaps, single nucleotide polymorphisms (SNPs), insertions and deletions (INDELs), and presence and absence variations (PAVs) are displayed. Sequence identity per 100 kb was revealed by the track at the top with the red dashed line indicating 30% identity.</p></figcaption></figure><p>For the QTL analysis with the graph‐based reference (Section <a href="#tpg270079-sec-0180" class="usa-link">2.15</a>), SNPs were first identified from the graph‐based reference, and then these SNPs were projected to Rabiosa v2 to obtain their physical positions. These SNPs were genotyped with GBS reads from all F<sub>1</sub> individuals (Section <a href="#tpg270079-sec-0170" class="usa-link">2.14</a>), and SNP markers that were only heterozygous in Rabiosa were considered for computing the genetic map position (OrderMarkers2 with improveOrder = 0) based on the projected physical position in Rabiosa v2 using Lep‐MAP3 (Rastas, <a href="#tpg270079-bib-0076" class="usa-link" aria-describedby="tpg270079-bib-0076">2017</a>). To verify the resulting genetic linkage map, a recombination fraction plot was generated using R/qtl (Broman et al., <a href="#tpg270079-bib-0009" class="usa-link" aria-describedby="tpg270079-bib-0009">2003</a>). For the QTL analysis with the single reference (Rabiosa v2), the SNPs obtained when generating the second batch of genetic linkage maps were used for map construction. With Lep‐MAP3 (Rastas, <a href="#tpg270079-bib-0076" class="usa-link" aria-describedby="tpg270079-bib-0076">2017</a>), two genetic linkage maps were constructed with SNP markers that were only heterozygous in Rabiosa or Sikem, respectively. Similarly, the position of SNP markers in the genetic linkage maps was calculated with the given order of these SNP markers in Rabiosa v2.</p></section><section id="tpg270079-sec-0100"><h3 class="pmc_sec_title">2.7. Customized haplotig purge pipeline</h3>
<p>A customized pipeline named PurgeGrass (available at GitHub, <a href="https://github.com/Yutang-ETH/PhaseGrass/tree/main/PurgeGrass" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Yutang‐ETH/PhaseGrass/tree/main/PurgeGrass</a>) was developed to purge both Flye and Canu assemblies in this work. The pipeline pairs allelic contigs based on evidence from all‐by‐all alignment, micro‐synteny, and Benchmarking Universal Single‐Copy Orthologs (BUSCO) genes. All‐by‐all alignment was conducted using Purge Haplotigs (Roach et al., <a href="#tpg270079-bib-0078" class="usa-link" aria-describedby="tpg270079-bib-0078">2018</a>), and micro‐synteny between contigs was detected using a pipeline consisting of GMAP (Wu &amp; Watanabe, <a href="#tpg270079-bib-0095" class="usa-link" aria-describedby="tpg270079-bib-0095">2005</a>), GffRead (G. Pertea &amp; Pertea, <a href="#tpg270079-bib-0071" class="usa-link" aria-describedby="tpg270079-bib-0071">2020</a>), DIAMOND (Buchfink et al., <a href="#tpg270079-bib-0012" class="usa-link" aria-describedby="tpg270079-bib-0012">2015</a>), and MCScanX (Wang et al., <a href="#tpg270079-bib-0094" class="usa-link" aria-describedby="tpg270079-bib-0094">2012</a>) with transcripts generated in a previous study (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>). BUSCO genes were detected using BUSCO v4.1.4 (Manni et al., <a href="#tpg270079-bib-0060" class="usa-link" aria-describedby="tpg270079-bib-0060">2021</a>) with database embryophyta_odb10. A contig is considered as a haplotig of a longer contig if it fulfills one of the following criteria: (1) having alignment coverage greater than 70% with a longer contig, (2) sharing at least five collinear genes with a longer contig, and (3) sharing at least one single‐copy complete BUSCO gene with a longer contig.</p></section><section id="tpg270079-sec-0110"><h3 class="pmc_sec_title">2.8. Reference‐based phasing and haplotype‐resolved genome assembly</h3>
<p>First, WGS Illumina short reads were mapped to Rabiosa v2 pseudo‐chromosomes using BWA‐MEM (Li, <a href="#tpg270079-bib-0053" class="usa-link" aria-describedby="tpg270079-bib-0053">2013</a>), and then SNPs were called using SAMtools and BCFtools (Danecek et al., <a href="#tpg270079-bib-0021" class="usa-link" aria-describedby="tpg270079-bib-0021">2021</a>). Next, Hi‐C data were mapped to Rabiosa v2 pseudo‐chromosomes to phase the SNPs using Hapcut2 (Edge et al., <a href="#tpg270079-bib-0025" class="usa-link" aria-describedby="tpg270079-bib-0025">2016</a>), following the pipeline described here, <a href="https://github.com/vibansal/HapCUT2/tree/master/HiC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/vibansal/HapCUT2/tree/master/HiC</a>. This resulted in a sparse chromosome‐level phase block per pseudo‐chromosome. In parallel, ONT long reads were mapped to the pseudo‐chromosomes using Winnowmap2 (Jain et al., <a href="#tpg270079-bib-0044" class="usa-link" aria-describedby="tpg270079-bib-0044">2022</a>). By combining the local phase information provided by the ONT read alignment with the long‐range phase information provided by the sparse chromosome‐level phase block derived from Hi‐C data, SNPs were phased through pseudo‐chromosomes using WhatsHap (Martin et al., <a href="#tpg270079-bib-0062" class="usa-link" aria-describedby="tpg270079-bib-0062">2016</a>). This resulted in a dense chromosome‐level phase block per pseudo‐chromosome. Finally, ONT reads were binned to different haplotypes using WhatsHap based on the chromosome‐level phase blocks with high‐density SNPs, resulting in four sets of reads including haplotype 1, haplotype 2, untagged, and unaligned. Only haplotype 1 and 2 reads were used subsequently to generate haplotype assemblies.</p>
<p>Flye (Kolmogorov et al., <a href="#tpg270079-bib-0050" class="usa-link" aria-describedby="tpg270079-bib-0050">2019</a>) was used to assemble reads of each haplotype separately. Then, one round of short‐read polishing was applied to each haplotype assembly using Polca (Zimin &amp; Salzberg, <a href="#tpg270079-bib-0101" class="usa-link" aria-describedby="tpg270079-bib-0101">2020</a>). Next, each assembly was scaffolded with Hi‐C using SALSA2 (Ghurye et al., <a href="#tpg270079-bib-0031" class="usa-link" aria-describedby="tpg270079-bib-0031">2019</a>), and then mis‐joins in the resulting scaffolds were corrected using Tritex (Monat et al., <a href="#tpg270079-bib-0065" class="usa-link" aria-describedby="tpg270079-bib-0065">2019</a>). The corrected scaffolds were aligned to Rabiosa v2 to construct pseudo‐chromosomes using RagTag (Alonge et al., <a href="#tpg270079-bib-0001" class="usa-link" aria-describedby="tpg270079-bib-0001">2022</a>). Additional manual curation to both haplotype assemblies was conducted using Juicer (Durand, Robinson, et al., <a href="#tpg270079-bib-0023" class="usa-link" aria-describedby="tpg270079-bib-0023">2016</a>), 3D‐DNA (Dudchenko et al., <a href="#tpg270079-bib-0022" class="usa-link" aria-describedby="tpg270079-bib-0022">2017</a>), and Juicebox (Durand, Robinson, et al., <a href="#tpg270079-bib-0023" class="usa-link" aria-describedby="tpg270079-bib-0023">2016</a>), resulting in Rabiosa h1 and h2. The two haplotype assemblies were concatenated to a diploid assembly (Rabiosa dip), and a diploid Hi‐C contact map was also generated with Juicer.</p></section><section id="tpg270079-sec-0120"><h3 class="pmc_sec_title">2.9. Assembly quality assessment</h3>
<p>The general assembly statistics were calculated with assembly‐stats (<a href="https://github.com/sanger-pathogens/assembly-stats" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/sanger‐pathogens/assembly‐stats</a>). The completeness of the assemblies was assessed using BUSCO v4.1.4 (Manni et al., <a href="#tpg270079-bib-0060" class="usa-link" aria-describedby="tpg270079-bib-0060">2021</a>) with database embryophyta_odb10. The k‐mer profile of each assembly was generated using the k‐mer analysis toolkit (KAT; Mapleson et al., <a href="#tpg270079-bib-0061" class="usa-link" aria-describedby="tpg270079-bib-0061">2017</a>). To calculate the ONT read alignment depth, all ONT long reads were aligned to Rabiosa h1, h2, and dip separately using Winowmap2 (Jain et al., <a href="#tpg270079-bib-0044" class="usa-link" aria-describedby="tpg270079-bib-0044">2022</a>), and the alignment depth was calculated using Mosdepth (Pedersen &amp; Quinlan, <a href="#tpg270079-bib-0070" class="usa-link" aria-describedby="tpg270079-bib-0070">2018</a>). RNA‐seq‐read mapping was done with the methods described in Section <a href="#tpg270079-sec-0130" class="usa-link">2.10</a>, and the mapping rate was reported from HISAT2 (Kim et al., <a href="#tpg270079-bib-0048" class="usa-link" aria-describedby="tpg270079-bib-0048">2015</a>).</p></section><section id="tpg270079-sec-0130"><h3 class="pmc_sec_title">2.10. Evidence‐based genome annotation</h3>
<p>First, a custom repeat library was generated using RepeatModeler2 (Flynn et al., <a href="#tpg270079-bib-0026" class="usa-link" aria-describedby="tpg270079-bib-0026">2020</a>) and ProtExcluder.pl (Campbell et al., <a href="#tpg270079-bib-0013" class="usa-link" aria-describedby="tpg270079-bib-0013">2014</a>) based on Rabiosa v2. Then, transposable elements (TEs) in Rabiosa v2, h1, and h2 were annotated using RepeatMasker (Smit et al., <a href="#tpg270079-bib-0086" class="usa-link" aria-describedby="tpg270079-bib-0086">2013</a>) with the custom repeat library.</p>
<p>Gene models were built using EVidenceModeler (EVM) (Haas et al., <a href="#tpg270079-bib-0039" class="usa-link" aria-describedby="tpg270079-bib-0039">2008</a>) with evidence from ab initio gene prediction, protein alignment, and transcript alignment following the method described in Tritex (Monat et al., <a href="#tpg270079-bib-0065" class="usa-link" aria-describedby="tpg270079-bib-0065">2019</a>). Ab initio gene prediction was performed using BRAKER2 (Brůna et al., <a href="#tpg270079-bib-0010" class="usa-link" aria-describedby="tpg270079-bib-0010">2021</a>) with protein sequences from Kyuss (Frei et al., <a href="#tpg270079-bib-0027" class="usa-link" aria-describedby="tpg270079-bib-0027">2021</a>). Protein sequences collected from closely related species (<a href="#tpg270079-supinfo-0001" class="usa-link">Methods S1</a>) were mapped to the assembly using GenomeThreader (Gremme et al., <a href="#tpg270079-bib-0033" class="usa-link" aria-describedby="tpg270079-bib-0033">2005</a>), and alignment results were merged using GenomeTools (Gremme et al., <a href="#tpg270079-bib-0034" class="usa-link" aria-describedby="tpg270079-bib-0034">2013</a>). Transcripts collected from closely related species (Methods <a href="#tpg270079-supinfo-0001" class="usa-link">S2</a>) were mapped to the assembly using GMAP (Wu &amp; Watanabe, <a href="#tpg270079-bib-0095" class="usa-link" aria-describedby="tpg270079-bib-0095">2005</a>), and mapped transcripts were merged using Cuffcompare (Trapnell et al., <a href="#tpg270079-bib-0091" class="usa-link" aria-describedby="tpg270079-bib-0091">2012</a>) and StringTie merge (M. Pertea et al., <a href="#tpg270079-bib-0073" class="usa-link" aria-describedby="tpg270079-bib-0073">2015</a>). Coding sequences were identified using TransDecoder (Haas, BJ, <a href="#tpg270079-bib-0038" class="usa-link" aria-describedby="tpg270079-bib-0038">2018</a>) with the merged mapped transcripts. Additionally, RNA‐seq data of Rabiosa generated previously (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>) were also used for genome annotation in this work. RNA‐seq reads were first trimmed using fastp (S. Chen et al., <a href="#tpg270079-bib-0015" class="usa-link" aria-describedby="tpg270079-bib-0015">2018</a>), then transcripts were constructed using the alignment‐based approach with HISAT2 (Kim et al., <a href="#tpg270079-bib-0048" class="usa-link" aria-describedby="tpg270079-bib-0048">2015</a>) and StringTie2 (M. Pertea et al., <a href="#tpg270079-bib-0073" class="usa-link" aria-describedby="tpg270079-bib-0073">2015</a>) following the instructions detailed in M. Pertea et al. (<a href="#tpg270079-bib-0072" class="usa-link" aria-describedby="tpg270079-bib-0072">2016</a>). All evidence was integrated by EVM to generate consensus gene models.</p>
<p>Using BLAST (Altschul et al., <a href="#tpg270079-bib-0002" class="usa-link" aria-describedby="tpg270079-bib-0002">1990</a>), gene models were searched for against The TRansposable Elements Platform (TREP), a TEs database (Schlagenhauf &amp; Wicker, <a href="#tpg270079-bib-0079" class="usa-link" aria-describedby="tpg270079-bib-0079">2016</a>), the UniProt reviewed database (The UniProt Consortium et al., <a href="#tpg270079-bib-0090" class="usa-link" aria-describedby="tpg270079-bib-0090">2021</a>), and the Pfam protein families database (Mistry et al., <a href="#tpg270079-bib-0064" class="usa-link" aria-describedby="tpg270079-bib-0064">2021</a>). Based on the BLAST results, gene models were classified into high‐ and low‐confidence groups. High‐confidence genes were defined as those with a hit in either the UniProt or Pfam database and no more than 25% overlap with a TE gene in TREP. In contrast, low‐confidence genes were defined as those without a BLAST hit in either the UniProt or Pfam database and no more than 25% overlap with a TE gene in TREP. Notably, since the TREP database may not represent all the repetitive sequences of Italian ryegrass, there might be some TE coding genes remaining in the high‐ and low‐confidence groups. A more comprehensive TE database could be used to improve the classification of the functional protein‐coding genes, or TE could be masked prior to gene annotation.</p>
<p>Functional annotation was performed using AHRD (<a href="https://github.com/groupschoof/AHRD" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/groupschoof/AHRD</a>) and InterProScan 5 (Jones et al., <a href="#tpg270079-bib-0046" class="usa-link" aria-describedby="tpg270079-bib-0046">2014</a>).</p></section><section id="tpg270079-sec-0140"><h3 class="pmc_sec_title">2.11. Gene‐based synteny analysis</h3>
<p>Using Last (Kiełbasa et al., <a href="#tpg270079-bib-0047" class="usa-link" aria-describedby="tpg270079-bib-0047">2011</a>), the high‐confidence genes of Rabiosa v2, h1, and h2 were aligned against genes from the barley cultivar ‘Morex’ (Mascher et al., <a href="#tpg270079-bib-0063" class="usa-link" aria-describedby="tpg270079-bib-0063">2021</a>), Kyuss, a doubled haploid perennial ryegrass genotype (Frei et al., <a href="#tpg270079-bib-0027" class="usa-link" aria-describedby="tpg270079-bib-0027">2021</a>), and Lp261, a self‐compatible perennial ryegrass genotype (Nagy et al., <a href="#tpg270079-bib-0067" class="usa-link" aria-describedby="tpg270079-bib-0067">2022</a>). After alignment, gene‐based synteny was constructed using MCscan (Python version <a href="https://doi.org/10.5281/zenodo.594205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.594205</a>). The same analysis was applied to check the synteny between Rabiosa v2 and the two haplotype assemblies, and the synteny between Rabiosa h1 and h2. The mean alignment identity of the gene coding sequence between Rabiosa h1 and h2 was calculated with awk commands (see Data Availability Statement) based on the gene coding sequence alignment file generated by Last when running the MCscan pipeline.</p></section><section id="tpg270079-sec-0150"><h3 class="pmc_sec_title">2.12. Sequence identity between two haplotypes</h3>
<p>Pseudo‐chromosomes of Rabiosa h2 (query) were aligned against pseudo‐chromosomes of Rabiosa h1 (reference) using AnchorWave (Song et al., <a href="#tpg270079-bib-0087" class="usa-link" aria-describedby="tpg270079-bib-0087">2022</a>) allowing inversions. The resulting alignments were converted to the Sequence Alignment/Map (SAM) format with the maf‐convert script provided by AnchorWave. Then, the SAM alignment file was further converted to the Pairwise mApping Format (PAF) by paftools.js from Minimap2 (Li, <a href="#tpg270079-bib-0054" class="usa-link" aria-describedby="tpg270079-bib-0054">2018</a>). Next, with the PAF file as input, SyRi (Goel et al., <a href="#tpg270079-bib-0032" class="usa-link" aria-describedby="tpg270079-bib-0032">2019</a>) was used to call SNPs, small insertions and deletions (INDELs), large presence and absence variations (PAVs), and inversions.</p>
<p>In parallel, short reads were aligned to Rabiosa h1 to call SNPs and small INDELs with BWA (Li, <a href="#tpg270079-bib-0053" class="usa-link" aria-describedby="tpg270079-bib-0053">2013</a>) and BCFtools (Danecek et al., <a href="#tpg270079-bib-0021" class="usa-link" aria-describedby="tpg270079-bib-0021">2021</a>). ONT long reads were aligned to Rabiosa h1 with Winnomap2 (Jain et al., <a href="#tpg270079-bib-0044" class="usa-link" aria-describedby="tpg270079-bib-0044">2022</a>), and PAVs were called with Sniffles (Sedlazeck et al., <a href="#tpg270079-bib-0082" class="usa-link" aria-describedby="tpg270079-bib-0082">2018</a>). Gaps in Rabiosa h1 and h2 were detected using a custom Python script. Common SNPs, small INDELs, and large PAVs that were detected by both whole‐genome alignment (WGA) and read alignment were identified by custom R scripts. Common SNPs and INDELs between two variant calling methods were defined as SNPs and INDELs sharing the exact start position, while common PAVs between two variant calling methods were defined as PAVs that overlap each other, or the start position of one PAV is within 10 bp downstream of the end position of the other PAV. Only common variants were used for sequence identity calculation and PAVs containing gaps were filtered. Sequence identity was calculated per 100 kb along the pseudo‐chromosome as (100 kb − total length of common variants − gaps − unaligned sequences reported by SyRi)/100 kb.</p></section><section id="tpg270079-sec-0160"><h3 class="pmc_sec_title">2.13. Constructing graph‐based reference</h3>
<p>Pair‐wise WGA was first performed among Rabiosa v2, h1, and h2 using AnchorWave (Song et al., <a href="#tpg270079-bib-0087" class="usa-link" aria-describedby="tpg270079-bib-0087">2022</a>). Then, the resulting alignments were converted to PAF format as described above, and one PAF file was obtained for each pair‐wise alignment. All PAF files were concatenated into one PAF file using a custom R script and then fed into pangenome graph builder (PGGB, Garrison et al., <a href="#tpg270079-bib-0029" class="usa-link" aria-describedby="tpg270079-bib-0029">2023</a>) to construct the graph‐based reference.</p></section><section id="tpg270079-sec-0170"><h3 class="pmc_sec_title">2.14. Genotyping of SNPs in the graph‐based reference</h3>
<p>SNPs in the graph‐based reference were detected by deconstructing the graph‐based reference using vg deconstruct (Garrison et al., <a href="#tpg270079-bib-0030" class="usa-link" aria-describedby="tpg270079-bib-0030">2018</a>). Subsequently, the resulting vcf file from deconstruction was filtered using vcfbub (<a href="https://github.com/pangenome/vcfbub" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/pangenome/vcfbub</a>), resulting in a vcf file containing only SNPs. GBS data of each F<sub>1</sub> individual in the bi‐parental population were mapped to the graph‐based reference using Giraffe (Sirén et al., <a href="#tpg270079-bib-0085" class="usa-link" aria-describedby="tpg270079-bib-0085">2021</a>), and the resulting alignment files were further processed with the vg toolkit (Garrison et al., <a href="#tpg270079-bib-0030" class="usa-link" aria-describedby="tpg270079-bib-0030">2018</a>). Based on the alignment, any SNPs in the previous vcf file covered by at least five GBS reads of one F<sub>1</sub> individual were genotyped by the vg toolkit, resulting in one vcf file with the genotypes of this individual. All individual vcf files were finally merged to one vcf file using BCFtools merge (Danecek et al., <a href="#tpg270079-bib-0021" class="usa-link" aria-describedby="tpg270079-bib-0021">2021</a>).</p></section><section id="tpg270079-sec-0180"><h3 class="pmc_sec_title">2.15. Phenotyping and QTL analysis of stem rust resistance</h3>
<p>The F<sub>1</sub> population was grown at the same location (47.427° N, 8.516° E, Zurich, Switzerland) but in a different neighboring field (environment) for each of the three phenotyping years (2019, 2020, and 2021). This design was intended to maximize the likelihood of exposure to the same stem rust strain across years, although the composition of the field isolates was not determined. In spring 2019, the F<sub>1</sub> plants were cut after the end of heading. In the second growth, the seed harvesting date was determined based on the cumulative average daily temperature measured 5 cm above the ground after the start of flowering for each plant individually. Once a plant reached the temperature sum of around 590°C, the plant was harvested by cutting with a sickle and phenotyped for stem rust occurrence. Stem rust occurrence was scored on each plant using a scale from 1 to 9, where 1 = no rust occurrence, 2 = trace of rust, 3 = 5%, 4 = 10%, 5 = 25%, 6 = 40%, 7 = 60%, 8 = 75%, and 9 = more than 75% of the plant is covered with rust (Schubiger &amp; Boller, <a href="#tpg270079-bib-0080" class="usa-link" aria-describedby="tpg270079-bib-0080">2016</a>). To analyze the phenotypic data across environments, the following linear mixed model was used:</p>
<table class="disp-formula p" id="tpg270079-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mrow><mi>e</mi><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>b</mi><mrow><mi>l</mi><mi>k</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <em>y<sub>ijkl</sub>
</em> represents the measurement for stem rust occurrence on a single plant basis, <em>μ</em> denotes the overall mean, <em>g<sub>i</sub>
</em> the effect of genotype <em>i</em>, <em>e<sub>j</sub>
</em> the effect of environment <em>j</em>, <em>ge<sub>ij</sub>
</em> the interaction effect of genotype <em>i</em> with environment <em>j</em>, <em>b<sub>lkj</sub>
</em> the effect of the <em>l</em>th incomplete block nested within the <em>k</em>th complete block, and <em>ε<sub>ijkl</sub>
</em> the residual error. The best linear unbiased estimators (BLUEs) for each genotype across all environments were calculated by using all factors within Equation (<a href="#tpg270079-disp-0001" class="usa-link">1</a>) as random effect, except <em>b<sub>lkj</sub>
</em>, which was considered as fixed effect. All the phenotypic data analyses were conducted using R version 4.3.1 (R Core Team, <a href="#tpg270079-bib-0103" class="usa-link" aria-describedby="tpg270079-bib-0103">2023</a>). Package lm4 (Bates et al., <a href="#tpg270079-bib-0005" class="usa-link" aria-describedby="tpg270079-bib-0005">2015</a>) was used for fitting mixed‐effect models. Package emmeans (Searle et al., <a href="#tpg270079-bib-0081" class="usa-link" aria-describedby="tpg270079-bib-0081">1980</a>) was used for calculating the BLUEs. Resistance data were obtained for 133 F<sub>1</sub> individuals along with their two parents, and the BLUEs calculated for the 133 F1 individuals were used for QTL analysis.</p>
<p>The genetic linkage maps (Section <a href="#tpg270079-sec-0090" class="usa-link">2.6</a>) and the BLUEs were integrated into a CSV file as input for QTL analysis using R/qtl (Broman et al., <a href="#tpg270079-bib-0009" class="usa-link" aria-describedby="tpg270079-bib-0009">2003</a>). Composite interval mapping (CIM) was used to calculate the marker‐trait association (function cim in R/qtl with n.marcovar = 5, window = 10). The suggested logarithm of odds (LOD) threshold was defined as the 95th percentile of the distribution of the genome‐wide maximum LOD score. The distribution was obtained by conducting a permutation test using the function scanone from R/qtl with n.perm = 200. The percentage of variance explained by each QTL was calculated using the function makeqtl from R/qtl.</p></section></section><section id="tpg270079-sec-0190"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270079-sec-0200"><h3 class="pmc_sec_title">3.1. Whole‐genome sequencing and reduced representation sequencing of Italian ryegrass</h3>
<p>For Rabiosa, the following sequencing data were obtained: ONT long reads (total length of 153 Gb, corresponding to ∼60‐fold coverage of the haploid genome; read N50 of 21 kb), Hi‐C reads (540 million 150 bp pair‐end reads, corresponding to ∼60‐fold coverage of the haploid genome), and Illumina short reads (818 million 150 bp pair‐end reads, corresponding to ∼85‐fold coverage of the haploid genome) from a previous study (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>).</p>
<p>The F<sub>1</sub> population, together with the two parental genotypes Rabiosa and Sikem, was subjected to GBS, yielding on average 2.6 million 90‐bp single‐end reads per individual (ranging from 0.1 to 6 million) across the 304 F1 individuals.</p></section><section id="tpg270079-sec-0210"><h3 class="pmc_sec_title">3.2. Chromosome‐level unphased haploid assembly of Rabiosa</h3>
<p>For the chromosome‐level haploid assembly (Rabiosa v1), the raw Flye assembly was first polished with accurate short reads, resulting in contigs with a total length of 3.39 Gb (Table <a href="#tpg270079-supinfo-0001" class="usa-link">S1</a>) and a contig N50 of 660.61 kb (Table <a href="#tpg270079-supinfo-0001" class="usa-link">S1</a>). Next, haplotigs in the polished assembly were removed with a customized pipeline, resulting in a haplotig‐purged assembly with its total length reduced to 2.75 Gb (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>). Based on the haplotig‐purged assembly, subsequent scaffolding with a consensus genetic linkage map containing 26,203 informative SNP markers (2083 and 2293 markers with a unique genetic map position in the map of parent 1 and 2, respectively, Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S1</a>) and Hi‐C data generated a chromosome‐level assembly with a scaffold N50 of 275.65 Mb and a 77.82% anchoring rate (2.14 out of 2.75 Gb were anchored to seven pseudo‐chromosomes, Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="tpg270079-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Assembly statistics of Rabiosa assemblies.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">Rabiosa v1</th>
<th align="left" rowspan="1" colspan="1">Rabiosa v2</th>
<th align="left" rowspan="1" colspan="1">Rabiosa h1</th>
<th align="left" rowspan="1" colspan="1">Rabiosa h2</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Total assembly size (Gb)</td>
<td align="center" rowspan="1" colspan="1">2.75</td>
<td align="center" rowspan="1" colspan="1">2.46</td>
<td align="center" rowspan="1" colspan="1">1.78</td>
<td align="center" rowspan="1" colspan="1">1.74</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Contig N50 (kb)/L50 (#)</td>
<td align="center" rowspan="1" colspan="1">660.61/1350</td>
<td align="center" rowspan="1" colspan="1">1332.60/563</td>
<td align="center" rowspan="1" colspan="1">276.06/1807</td>
<td align="center" rowspan="1" colspan="1">265.82/1808</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Contig N90 (kb)/L90 (#)</td>
<td align="center" rowspan="1" colspan="1">125.25/5908</td>
<td align="center" rowspan="1" colspan="1">396.90/1838</td>
<td align="center" rowspan="1" colspan="1">77.53/6528</td>
<td align="center" rowspan="1" colspan="1">74.79/6573</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Scaffold N50 (Mb)/L50 (#)</td>
<td align="center" rowspan="1" colspan="1">275.65/5</td>
<td align="center" rowspan="1" colspan="1">338.75/4</td>
<td align="center" rowspan="1" colspan="1">247.62/4</td>
<td align="center" rowspan="1" colspan="1">245.51/4</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Scaffold N90 (Mb)/L90 (#)</td>
<td align="center" rowspan="1" colspan="1">0.18/786</td>
<td align="center" rowspan="1" colspan="1">244.92/7</td>
<td align="center" rowspan="1" colspan="1">186.44/7</td>
<td align="center" rowspan="1" colspan="1">178.87/7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total length anchored to pseudo‐chromosomes (Gb)</td>
<td align="center" rowspan="1" colspan="1">2.14</td>
<td align="center" rowspan="1" colspan="1">2.24</td>
<td align="center" rowspan="1" colspan="1">1.67</td>
<td align="center" rowspan="1" colspan="1">1.63</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Anchoring rate (%)</td>
<td align="center" rowspan="1" colspan="1">77.79</td>
<td align="center" rowspan="1" colspan="1">91.16</td>
<td align="center" rowspan="1" colspan="1">94.04</td>
<td align="center" rowspan="1" colspan="1">93.85</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total BUSCO (%)</td>
<td align="center" rowspan="1" colspan="1">94.00</td>
<td align="center" rowspan="1" colspan="1">94.60</td>
<td align="center" rowspan="1" colspan="1">89.70</td>
<td align="center" rowspan="1" colspan="1">89.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Single‐copy BUSCO (%)</td>
<td align="center" rowspan="1" colspan="1">88.20</td>
<td align="center" rowspan="1" colspan="1">89.00</td>
<td align="center" rowspan="1" colspan="1">83.60</td>
<td align="center" rowspan="1" colspan="1">83.60</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Duplicated BUSCO (%)</td>
<td align="center" rowspan="1" colspan="1">5.80</td>
<td align="center" rowspan="1" colspan="1">5.60</td>
<td align="center" rowspan="1" colspan="1">6.10</td>
<td align="center" rowspan="1" colspan="1">5.40</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fragmented (%)</td>
<td align="center" rowspan="1" colspan="1">0.80</td>
<td align="center" rowspan="1" colspan="1">0.40</td>
<td align="center" rowspan="1" colspan="1">0.30</td>
<td align="center" rowspan="1" colspan="1">0.60</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Missing (%)</td>
<td align="center" rowspan="1" colspan="1">5.20</td>
<td align="center" rowspan="1" colspan="1">5.00</td>
<td align="center" rowspan="1" colspan="1">10.00</td>
<td align="center" rowspan="1" colspan="1">10.40</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270079-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270079-tbl1-note-0001"><p>Abbreviation: BUSCO, Benchmarking Universal Single‐Copy Orthologs.</p></div></div></section><p>The more contiguous chromosome‐level haploid assembly (Rabiosa v2) was generated by scaffolding based on Rabiosa v1 with contigs produced by Canu (Koren et al., <a href="#tpg270079-bib-0052" class="usa-link" aria-describedby="tpg270079-bib-0052">2017</a>), using the same ONT data. Contigs generated by Canu were first polished, resulting in a total length of 4.18 Gb (Table <a href="#tpg270079-supinfo-0001" class="usa-link">S1</a>) and a higher contig N50 of 966.70 kb (Table <a href="#tpg270079-supinfo-0001" class="usa-link">S1</a>) compared to Rabiosa v1. Removing redundant haplotigs resulted in a haplotig‐purged assembly with a total length of 2.46 Gb (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>) and a higher contig N50 of 1.33 Mb (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>). Using Rabiosa v1 as the reference together with Hi‐C data, this haplotig‐purged assembly was scaffolded, resulting in a chromosome‐level assembly with a scaffold N50 of 338.75 Mb and a 91.16% anchoring rate (2.24 out of 2.46 Gb was anchored to seven pseudo‐chromosomes, Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>). Rabiosa v2 was used for all further analyses.</p></section><section id="tpg270079-sec-0220"><h3 class="pmc_sec_title">3.3. Chromosome‐level haplotype‐resolved assembly of Rabiosa</h3>
<p>With the unphased haploid assembly Rabiosa v2, a reference‐based phasing approach was applied to separate the reads belonging to the two Rabiosa haplotypes. Seven chromosome‐level phase blocks together with some small phase blocks that could not be integrated into the chromosome‐level phase blocks were constructed with high‐density SNPs (Figure <a href="#tpg270079-fig-0001" class="usa-link">1a</a>; Table <a href="#tpg270079-supinfo-0001" class="usa-link">S2</a>). Ideally, one chromosome‐level phase block should be obtained for each pseudo‐chromosome, and the small phase blocks might lead to haplotype switches in downstream haplotype partitioning (assigning long reads to corresponding haplotypes). Based on the haplotypes of all the phase blocks, ONT reads were assigned to one of the four groups: haplotype 1, haplotype 2, untagged, and unaligned (Figure <a href="#tpg270079-fig-0001" class="usa-link">1b</a>, Table <a href="#tpg270079-supinfo-0001" class="usa-link">S3</a>). Roughly 50 Gb ONT long reads (∼20‐fold coverage) were assigned to either haplotype, and based on these haplotype‐specific read sets, two phased haploid assemblies were generated. Each haplotype assembly showed a total size of ∼1.80 Gb and a contig N50 of ∼270.00 kb (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>). By further scaffolding, around 1.60 Gb of each assembly was anchored to seven pseudo‐chromosomes, resulting in two chromosome‐level assemblies with a scaffold N50 of around 250.00 Mb (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>). The two chromosome‐level phased assemblies were named Rabiosa h1 and Rabiosa h2, respectively.</p>
<figure class="fig xbox font-sm" id="tpg270079-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a81a/12376112/0044d046862d/TPG2-18-e70079-g004.jpg" loading="lazy" id="jats-graphic-1" height="283" width="800" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270079-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Phase blocks generated by reference‐based phasing. The <em>y</em>‐axis indicates the seven pseudo‐chromosomes (Chr 1–7), and the <em>x</em>‐axis indicates the base pair position (in Mb) in the pseudo‐chromosome. In each pseudo‐chromosome, the red line represents the largest phase block, and black vertical lines indicate small phase blocks that could not be integrated into the largest phase block. The blue vertical line indicates the length of the pseudo‐chromosome, showing whether the largest phase block spans the whole pseudo‐chromosome and reaches chromosome‐level coverage. (b) Read binning results from reference‐based phasing. Reads assigned to haplotypes were classified as tagged, reads mapped to pseudo‐chromosomes of Rabiosa v2 but not assigned to either haplotype were classified as untagged, and reads not mapped to any pseudo‐chromosomes were classified as unaligned. The sum of tagged, untagged, and unaligned reads is equal to the total length of the Oxford Nanopore Technologies (ONT) reads used for reference‐based phasing.</p></figcaption></figure></section><section id="tpg270079-sec-0230"><h3 class="pmc_sec_title">3.4. Quality check of the Rabiosa assemblies</h3>
<p>Quality assessments were conducted for the unphased haploid assembly (Rabiosa v2) and the two phased assemblies (Rabiosa h1 and h2). All three assemblies showed a high completeness as indicated by their total BUSCO scores (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>) and the k‐mer comparison results (Figure <a href="#tpg270079-fig-0002" class="usa-link">2a–c</a>). All three assemblies reached a chromosome‐level contiguity as indicated by their scaffold N50 (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>). Notably, Rabiosa v2 showed a higher completeness and contiguity compared to the two‐phased assemblies, reflected by its larger assembly size and higher scaffold N50 (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270079-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a81a/12376112/370ec8b1a65d/TPG2-18-e70079-g005.jpg" loading="lazy" id="jats-graphic-3" height="745" width="800" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270079-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a–d) K‐mer comparison between assembly and whole‐genome sequencing (WGS) short reads for Rabiosa v2, Rabiosa h1, Rabiosa h2, and Rabiosa dip, respectively. The plots show whether the presence and copy number of the k‐mers in the assembly match the distribution of the k‐mers in WGS short reads. The <em>x</em>‐axis represents k‐mer frequency in the WGS short reads, and the <em>y</em>‐axis shows the corresponding count of the distinct k‐mers from the reads. For a heterozygous diploid genome, two peaks are expected for the distribution of the k‐mers from reads. The first peak (heterozygous peak) at the lower frequency represents k‐mers from the two alleles at heterozygous loci, and the second peak (homozygous peak) at the doubled frequency of the first peak represents k‐mers from homozygous loci. The colors in the plot show the copy number of k‐mers from the reads in the assembly, with black showing no presence, red showing one copy, and purple showing two copies. For a haploid assembly with only one allele present for each locus, the first peak should be half black and half red, and the second should be full red. For a diploid assembly with both alleles present for each locus, the first peak should be full red, and the second peak should be full purple. (a–c) suggest that Rabiosa v2, h1, and h2 are haploid assemblies with some homozygous sequences missing (small black area under the homozygous peak). (d) suggests that Rabiosa dip is a diploid assembly with some alleles from heterozygous regions missing (small black area under the heterozygous peak) and some homozygous regions collapsed as one copy (red area under the homozygous peak). (e), Oxford Nanopore Technologies (ONT) read alignment coverage for Rabiosa h1 (blue), haplotype 1 in Rabiosa dip (red), Rabiosa h2 (dark blue), and haplotype 2 in Rabiosa dip (dark red). The read alignment coverage for a haplotype in the diploid assembly is expected to be half of the alignment coverage of a haploid assembly. (f) RNA‐seq read mapping rate (<em>y</em>‐axis) in Rabiosa v2, h1, h2, and dip. RNA‐seq data were obtained from multiple tissues with three replicates (<em>x</em>‐axis).</p></figcaption></figure><p>Rabiosa h1 and h2 were concatenated to form a diploid assembly (Rabiosa dip). Rabiosa dip showed a typical diploid assembly k‐mer profile (Figure <a href="#tpg270079-fig-0002" class="usa-link">2d</a>), suggesting that it was a good representation of the diploid genome. For each haplotype in the diploid assembly, a halved ONT read alignment coverage was observed compared to the haploid assemblies (Figure <a href="#tpg270079-fig-0002" class="usa-link">2e</a>), suggesting that the haplotypes of Rabiosa were indeed separated. Compared to the three haploid assemblies, Rabiosa dip showed a higher total BUSCO score and, as expected, a much higher duplicated BUSCO score (Table <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>) as well as a higher RNA‐seq read mapping rate (Figure <a href="#tpg270079-fig-0002" class="usa-link">2f</a>). This suggested that the haplotype‐resolved diploid assembly was a more complete representation of the diploid genome when compared to the haploid assemblies.</p>
<p>The order and orientation of the contigs in the pseudo‐chromosomes of Rabiosa v2, h1, and h2 were found to be correct based on the Hi‐C contact maps (Figure <a href="#tpg270079-fig-0003" class="usa-link">3a,b</a>). Complementary to the Hi‐C contact maps, large‐scale structural correctness of the pseudo‐chromosomes was also supported by the high consistency between marker order in the genetic map and their physical position in the assembly (Figure <a href="#tpg270079-fig-0004" class="usa-link">4a</a>, track B).</p>
<figure class="fig xbox font-sm" id="tpg270079-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a81a/12376112/f696e115aa96/TPG2-18-e70079-g001.jpg" loading="lazy" id="jats-graphic-5" height="296" width="600" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270079-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) High‐throughput chromosome conformation capture (Hi‐C) contact map of Rabiosa v2. (b) Hi‐C contact map of pseudo‐chromosomes of both haplotypes in Rabiosa dip. Each red pixel in the plot indicates one interaction between two loci within or between chromosomes, and each blue box represents one pseudo‐chromosome. If the sequence in the pseudo‐chromosome is correctly ordered and oriented, then more interactions should be observed between closely linked loci within chromosomes, resulting in a “smooth” red diagonal line.</p></figcaption></figure></section><section id="tpg270079-sec-0240"><h3 class="pmc_sec_title">3.5. Genome annotation and gene‐based synteny analyses</h3>
<p>With the evidence‐based genome annotation pipeline, a total of 72,069, 56,142, and 55,928 high‐confidence genes were identified for Rabiosa v2, h1, and h2, respectively (Figure <a href="#tpg270079-fig-0004" class="usa-link">4a</a>, track C). In addition, 113,004, 91,753, and 90,448 low‐confidence genes were identified for Rabiosa v2, h1, and h2, respectively (Figure <a href="#tpg270079-fig-0004" class="usa-link">4a</a>, track D). With all the gene models (including high‐ and low‐confidence genes), a high completeness of the gene space was observed in Rabiosa v2, h1, and h2, as indicated by the total complete BUSCO scores of 91.8%, 85.7%, and 85.5%, respectively (Table <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>). The number of complete BUSCO genes found in high‐confidence and low‐confidence genes was 83.3% and 9.2% in Rabiosa v2, respectively (Table <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>), indicating the presence of true genes in the low‐confidence gene set. Around 70% of each assembly was found to be composed of repetitive sequences (Figure <a href="#tpg270079-fig-0004" class="usa-link">4a</a>, track E; Table <a href="#tpg270079-supinfo-0001" class="usa-link">S5</a>), and an average GC (guanine‐cytosin) content of 44% was observed for each assembly (Figure <a href="#tpg270079-fig-0004" class="usa-link">4a</a>, track F).</p>
<p>Based on the high‐confidence gene models, a highly consistent gene‐based synteny was observed between Rabiosa and closely related species, including perennial ryegrass (<em>Lolium perenne</em> L.) and barley (Figure <a href="#tpg270079-fig-0004" class="usa-link">4b</a>; Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S2</a>). The known translocation resulting in chromosome 4 of <em>Lolium</em> spp. having synteny with chromosome 4 and a segment of chromosome 5 of barley (Pfeifer et al., <a href="#tpg270079-bib-0074" class="usa-link" aria-describedby="tpg270079-bib-0074">2013</a>) was observed (Figure <a href="#tpg270079-fig-0004" class="usa-link">4b</a>), validating the correctness of the pseudo‐chromosome structure in the Rabiosa assemblies. Notably, the inverted orientation of chromosomes 6 and 7 between Rabiosa and Kyuss (Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S2</a>) does not indicate true chromosome rearrangements between Italian and perennial ryegrass but is indicative for the wrong orientation of pseudo‐chromosomes in the first version of Kyuss (Frei et al., <a href="#tpg270079-bib-0027" class="usa-link" aria-describedby="tpg270079-bib-0027">2021</a>), as revealed by Y. Chen et al. (<a href="#tpg270079-bib-0016" class="usa-link" aria-describedby="tpg270079-bib-0016">2024</a>). A highly consistent chromosome‐level gene order was observed between the Rabiosa haplotypes h1 and h2 (Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S3</a>), with a mean alignment identity of 98.22% across 13,308 gene pairs.</p></section><section id="tpg270079-sec-0250"><h3 class="pmc_sec_title">3.6. High sequence variation between the two haplotypes of Rabiosa</h3>
<p>With a k‐mer‐based analysis, we first estimated the level of heterozygosity of Rabiosa, and a high level of heterozygosity of 3.43% was observed (Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>). This suggests a high sequence variation between the two haplotypes of Rabiosa with three to four SNPs per 100 bp. In parallel to the k‐mer‐based analysis, we also tried to estimate the sequence variation between the two haplotypes of Rabiosa based on not only SNPs, but also other types of variants called from WGA and read mapping. A total of 2,205,246 SNPs, 181,734 small INDELs (length shorter than 50 bp), and 52,578 large PAVs (length equal to or greater than 50 bp) were detected between Rabiosa h1 and h2 (Figure <a href="#tpg270079-fig-0004" class="usa-link">4c</a>; Table <a href="#tpg270079-supinfo-0001" class="usa-link">S6</a>). With these variants, sequence identity between haplotypes was calculated per 100 kb window along the pseudo‐chromosomes, and sequence identity as low as 30% was observed between haplotypes in many 100 kb windows (Figure <a href="#tpg270079-fig-0004" class="usa-link">4c</a>; Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S5</a>). This low sequence identity suggests that there was a much higher level of sequence variation between the two haplotypes of Rabiosa when structural variants were also considered.</p></section><section id="tpg270079-sec-0260"><h3 class="pmc_sec_title">3.7. QTL analysis for stem rust resistance with a graph‐based reference of Rabiosa</h3>
<p>QTL analysis for stem rust resistance was conducted with the genetic linkage map constructed based on the graph‐based reference and compared to QTL results from a genetic linkage map constructed based on a single reference (Rabiosa v2).</p>
<p>The graph‐based reference was constructed based on the three Rabiosa assemblies (Rabiosa v2, h1, and h2). Rabiosa v2 was included in the graph‐based reference so that variants detected from the graph could be projected to Rabiosa v2 to find their corresponding physical positions in Rabiosa v2. The physical position from the same coordinate system (the same assembly) allowed us to compare the QTL analysis results between the single‐reference and the graph‐based reference. The graph‐based reference contained 21 paths, corresponding to the 21 pseudo‐chromosomes of the three assemblies. The total size of the graph‐based reference was 3.66 Gb, and there were 243,499,034 nodes and 334,693,667 edges in the graph (Figure <a href="#tpg270079-fig-0005" class="usa-link">5a</a>).</p>
<figure class="fig xbox font-sm" id="tpg270079-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a81a/12376112/52664f0bde89/TPG2-18-e70079-g003.jpg" loading="lazy" id="jats-graphic-9" height="457" width="700" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270079-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) The 1D visualization of the graph‐based reference of Rabiosa showing multiple alignments for Rabiosa v2, h1, and h2. Each horizontal colored rectangle corresponds to a pseudo‐chromosome in one assembly (one path in the graph) containing nodes in the graph. The name of the pseudo‐chromosome was shown by the <em>y</em>‐axis label (rabiosa_0, rabiosa_1, and rabiosa_2 correspond to Rabiosa v2, h1, and h2, respectively). The black lines at the bottom are the edges in the graph indicating how nodes were connected. (b) Pair‐wise recombination fraction between markers in the Rabiosa linkage map generated with single nucleotide polymorphisms (SNPs) from the graph‐based reference. A low recombination fraction (indicated by the yellow color) was observed within linkage groups, and a high recombination fraction (no linkage, indicated by the dark blue color) was observed between linkage groups and distant parts of the same linkage group. (c) Results of quantitative trait locus (QTL) analysis for stem rust resistance with a graph‐based reference and a single reference (Rabiosa v2). The brown vertical lines in the gray horizontal bars (chromosomes) represent SNPs used for QTL analysis, and black lines connect the same SNP between the graph‐based reference and the single reference. The numbers linked to the significant QTL peaks show the base pair position (in Rabiosa v2) of the most significant SNP in the QTL.</p></figcaption></figure><p>Since the graph‐based reference is essentially a form of non‐redundant multiple sequence alignment, variants can be directly detected from the graph‐based reference. With the graph‐based reference of Rabiosa, 54,614 SNPs with alleles differentiating haplotypes were obtained and projected to Rabiosa v2 to have the corresponding physical positions in Rabiosa v2. These SNPs were further genotyped with GBS reads from all F<sub>1</sub> individuals and both parents. Phenotypic data of stem rust resistance were available for 133 F<sub>1</sub> individuals and the two parents (Rabiosa and Sikem), and a clear segregation of stem rust resistance was observed as a bimodal distribution (Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S6</a>). After genotyping, 304 F<sub>1</sub> individuals with 1,237 SNPs that were only heterozygous in Rabiosa were selected to build a genetic linkage map (Figure <a href="#tpg270079-fig-0005" class="usa-link">5b</a>). Combining the genetic linkage map and the phenotypic data of stem rust resistance resulted in 132 F<sub>1</sub> individuals for subsequent QTL analysis. Two significant QTL associated with stem rust resistance were detected, with one located on LG 2 and the other located on LG 7, corresponding to pseudo‐chromosomes 2 and 7 in Rabiosa v2, respectively (Figure <a href="#tpg270079-fig-0005" class="usa-link">5c</a>; Table <a href="#tpg270079-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270079-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Quantitative trait locus (QTL) analysis results for stem rust resistance.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">QTL</th>
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Left boundary (bp)</th>
<th align="left" rowspan="1" colspan="1">Right boundary (bp)</th>
<th align="left" rowspan="1" colspan="1">Highest LOD</th>
<th align="left" rowspan="1" colspan="1">LOD threshold</th>
<th align="left" rowspan="1" colspan="1">Variance explained (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">qtl.chr2</td>
<td align="left" rowspan="1" colspan="1">Rabiosa v2</td>
<td align="left" rowspan="1" colspan="1">220,462,590</td>
<td align="left" rowspan="1" colspan="1">237,911,985</td>
<td align="center" rowspan="1" colspan="1">4.64</td>
<td align="center" rowspan="1" colspan="1">2.71</td>
<td align="center" rowspan="1" colspan="1">5.93</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qtl.chr7</td>
<td align="left" rowspan="1" colspan="1">Rabiosa v2</td>
<td align="left" rowspan="1" colspan="1">233,515,340</td>
<td align="left" rowspan="1" colspan="1">259,843,203</td>
<td align="center" rowspan="1" colspan="1">27.52</td>
<td align="center" rowspan="1" colspan="1">2.71</td>
<td align="center" rowspan="1" colspan="1">47.18</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qtl.chr2</td>
<td align="left" rowspan="1" colspan="1">Graph based</td>
<td align="left" rowspan="1" colspan="1">174,459,966</td>
<td align="left" rowspan="1" colspan="1">231,403,834</td>
<td align="center" rowspan="1" colspan="1">5.66</td>
<td align="center" rowspan="1" colspan="1">2.71</td>
<td align="center" rowspan="1" colspan="1">4.83</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qtl.chr7</td>
<td align="left" rowspan="1" colspan="1">Graph based</td>
<td align="left" rowspan="1" colspan="1">224,224,230</td>
<td align="left" rowspan="1" colspan="1">255,175,175</td>
<td align="center" rowspan="1" colspan="1">18.39</td>
<td align="center" rowspan="1" colspan="1">2.71</td>
<td align="center" rowspan="1" colspan="1">35.30</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270079-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270079-tbl2-note-0001"><p>Abbreviation: LOD, logarithm of odds.</p></div></div></section><p>For QTL analysis with the single reference, a genetic linkage map was constructed based on Rabiosa v2. GBS reads from all F<sub>1</sub> individuals and both parents were aligned to Rabiosa v2 to call SNPs. In total, 264 F<sub>1</sub> individuals with 5,034 SNPs that were only heterozygous in Rabiosa were used to construct the genetic linkage map. Combining this genetic linkage map with the phenotypic data of stem rust resistance resulted in 120 individuals for QTL analysis. The same two QTL that were associated with stem rust resistance were detected (Figure <a href="#tpg270079-fig-0005" class="usa-link">5c</a>; Table <a href="#tpg270079-tbl-0002" class="usa-link">2</a>), validating the results of the QTL analysis with the graph‐based reference. In addition, a QTL analysis was also conducted using a linkage map consisting of 6,778 SNPs that were only heterozygous in Sikem. In this analysis, both QTL, although detectable, did not exceed the significance threshold (Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S7</a>). Combining this result with the phenotypic distribution of resistance to stem rust (Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S6</a>), we hypothesize that complementing resistance sources, either another dominantly inherited resistance allele of the QTL on chromosome 7 or the independently segregating resistance source chromosome 2 from Sikem, are segregating in this population.</p></section></section><section id="tpg270079-sec-0270"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Rabiosa v2, the chromosome‐level unphased haploid genome assembly of the highly heterozygous Italian ryegrass genotype Rabiosa, showed a much better quality compared to the reported draft assemblies for this genotype (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>) or other genotypes of this species (Knorst et al., <a href="#tpg270079-bib-0049" class="usa-link" aria-describedby="tpg270079-bib-0049">2019</a>). It showed a comparable quality to the recently reported reference‐level assembly (Brunharo et al., <a href="#tpg270079-bib-0011" class="usa-link" aria-describedby="tpg270079-bib-0011">2025</a>) with regard to the scaffold contiguity and assembly completeness (Rabiosa v2 vs. Brunharo's assembly, Scaffold N50, 338.75 Mb vs. 363.00 Mb, total BUSCO score, 94.60% vs. 92.80%). Notably, for Rabiosa v2, 60‐fold error‐prone ONT data were used to generate the assembly, while for the assembly of Brunharo et al. (<a href="#tpg270079-bib-0011" class="usa-link" aria-describedby="tpg270079-bib-0011">2025</a>), nearly 100‐fold accurate PacBio HiFi data were used. More importantly, we generated two haplotype assemblies (Rabiosa h1 and h2, Figures <a href="#tpg270079-fig-0001" class="usa-link">1</a>, <a href="#tpg270079-fig-0002" class="usa-link">2</a>, and <a href="#tpg270079-fig-0003" class="usa-link">3b</a>) with the error‐prone ONT data, which is more challenging to work with compared to the highly accurate PacBio HiFi data (Li &amp; Durbin, <a href="#tpg270079-bib-0055" class="usa-link" aria-describedby="tpg270079-bib-0055">2024</a>). Although the contigs of each haplotype assembly we generated were rather fragmented (contig N50 270 kb, Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>), we still managed to anchor more than 93% of the total length of the contigs onto pseudo‐chromosomes (Table <a href="#tpg270079-tbl-0001" class="usa-link">1</a>). Besides, when concatenating the two haplotype assemblies into a diploid assembly (Rabiosa dip) and comparing it with the haploid assemblies, we found that the diploid assembly was a more complete representation of the gene space of the diploid genome (higher total BUSCO score and RNA‐seq read mapping rate, Table <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>; Figure <a href="#tpg270079-fig-0002" class="usa-link">2f</a>). This suggests that a haplotype‐resolved assembly should be considered when generating genome assemblies for highly heterozygous species such as Italian ryegrass.</p>
<p>A high level of heterozygosity was estimated with k‐mers for Rabiosa (3.43%, Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>), which is consistent with the high SNP frequency (1 per 20 bp) observed in ryegrass (Hayes et al., <a href="#tpg270079-bib-0040" class="usa-link" aria-describedby="tpg270079-bib-0040">2013</a>). Compared to other published highly heterozygous plant genomes, such as tea (<em>Camellia sinensis</em>, heterozygosity 2.31%, Zhang et al., <a href="#tpg270079-bib-0098" class="usa-link" aria-describedby="tpg270079-bib-0098">2021</a>), lychee (<em>Litchi chinensis</em>, heterozygosity 2.27%, Hu et al., <a href="#tpg270079-bib-0043" class="usa-link" aria-describedby="tpg270079-bib-0043">2022</a>), and diploid potato (<em>Solanum tuberosum</em>, heterozygosity 2.10%, Q. Zhou et al., <a href="#tpg270079-bib-0099" class="usa-link" aria-describedby="tpg270079-bib-0099">2020</a>), Rabiosa showed the highest level of heterozygosity, suggesting that there is a high level of sequence variation between the two haplotypes of Rabiosa. Indeed, a very high level of sequence variation (30% sequence similarity in some 100 kb regions) was observed between the two haplotypes of Rabiosa when estimating the sequence variation with different types of variants, including SNPs, INDELs, and PAVs (Figure <a href="#tpg270079-fig-0004" class="usa-link">4c</a>). As the mean alignment identity of the gene coding sequence between haplotypes is around 98%, as revealed by the gene‐based synteny analysis, the high sequence variation between the haplotypes is mainly from intergenic regions. This is consistent with the high intergenic sequence variation found previously in Rabiosa based on a single locus analysis (Copetti et al., <a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>). Such a high level of sequence variation might explain some of the challenges or technical limitations we encountered when assembling Rabiosa.</p>
<p>We found that the high level of sequence variation between the two haplotypes of Rabiosa may cause sequence‐alignment‐based haplotig‐purge tools, such as Purge Haplotigs (Roach et al., <a href="#tpg270079-bib-0078" class="usa-link" aria-describedby="tpg270079-bib-0078">2018</a>) and purge_dups (Guan et al., <a href="#tpg270079-bib-0035" class="usa-link" aria-describedby="tpg270079-bib-0035">2020</a>), to fail to align allelic contigs, leaving more redundant allelic sequences (false duplication) in the assembly after purging (indicated by the duplicated BUSCO score, Table <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>). The same situation was observed in other <em>Lolium</em> genome assemblies. For example, a haploid <em>L. rigidum</em> assembly with 70% duplicated BUSCO score (Paril et al., <a href="#tpg270079-bib-0069" class="usa-link" aria-describedby="tpg270079-bib-0069">2022</a>) was generated using Purge Haplotigs (Roach et al., <a href="#tpg270079-bib-0078" class="usa-link" aria-describedby="tpg270079-bib-0078">2018</a>), and the new reference‐level Italian ryegrass assembly with 20% duplicated BUSCO score (Brunharo et al., <a href="#tpg270079-bib-0011" class="usa-link" aria-describedby="tpg270079-bib-0011">2025</a>) was generated using purge_dups (Guan et al., <a href="#tpg270079-bib-0035" class="usa-link" aria-describedby="tpg270079-bib-0035">2020</a>). Notably, for the Brunharo's assembly, even though 100‐fold high‐quality PacBio HiFi data were used for assembly, the relatively high unexpected rate of false duplication remained. This suggests that the false duplication in the assembly is not a limitation from the input sequencing data but a technical limitation of the bioinformatic methods that could not handle the high sequence variation between haplotypes. To overcome the technical limitation and purge as many redundant haplotigs as possible for Rabiosa, in addition to sequence‐alignment‐based methods, we also used micro‐synteny and BUSCO genes to pair allelic contigs, which led to better purging results compared to using the sequence‐alignment‐based tools alone (lower duplicated BUSCO scores, Table <a href="#tpg270079-supinfo-0001" class="usa-link">S4</a>).</p>
<p>Genome phasing was also hampered by the high sequence variation between haplotypes. A reference‐based phasing approach was chosen in this work to separate the haplotypes of Rabiosa as no parental datasets or highly accurate long reads were available. Based on the unphased haploid assembly (Rabiosa v2) as the reference, we found that around 26% (40 Gb out of the total 153 Gb) of the ONT reads were left unassigned to any haplotypes based on the long‐read alignment to the reference. This high proportion of unassigned reads could be attributed to reference bias caused by the high sequence variation between haplotypes rather than sequences from homozygous regions (Porubsky et al., <a href="#tpg270079-bib-0075" class="usa-link" aria-describedby="tpg270079-bib-0075">2021</a>). As the haploid assembly (Rabiosa v2) was a mosaic representation of the diploid genome with alleles alternating between haplotypes in the pseudo‐chromosomes, when mapping long reads from both haplotypes to the reference, reads from one haplotype might be aligned to the reference, but reads from the other haplotype may not be aligned or incorrectly aligned to the reference (Jain et al., <a href="#tpg270079-bib-0044" class="usa-link" aria-describedby="tpg270079-bib-0044">2022</a>). If reads were incorrectly aligned to the wrong position due to repetitive sequences (Jain et al., <a href="#tpg270079-bib-0044" class="usa-link" aria-describedby="tpg270079-bib-0044">2022</a>), they might be left unassigned to any haplotypes because of no variants called in the repetitive sequences (Figure <a href="#tpg270079-supinfo-0001" class="usa-link">S8</a>). The high proportion of unassigned reads led to a low coverage (∼20‐fold) of haplotype‐tagged reads in each haplotype, which compromised the contiguity and completeness of the phased haplotype assemblies (Rabiosa h1 and h2). For the reference‐based phasing approach, due to the reference bias caused by the high sequence variation between haplotypes, long reads can only be assigned to haplotypes when they are correctly aligned to the reference. More future work is needed to overcome this limitation when applying reference‐based phasing to highly heterozygous genomes. Nonetheless, to the best of our knowledge, the two haplotype assemblies (Rabiosa h1 and h2) generated in this work are the first and the only haplotype‐resolved assembly reported for Italian ryegrass so far.</p>
<p>A graph‐based reference can be built by adding structural variations (SVs) into a single reference, and this type of graph‐based reference has been used for many genome‐wide association studies (GWAS) in plants (J. Chen et al., <a href="#tpg270079-bib-0014" class="usa-link" aria-describedby="tpg270079-bib-0014">2023</a>; Cochetel et al., <a href="#tpg270079-bib-0019" class="usa-link" aria-describedby="tpg270079-bib-0019">2023</a>; Liu et al., <a href="#tpg270079-bib-0057" class="usa-link" aria-describedby="tpg270079-bib-0057">2020</a>; Y. Zhou et al., <a href="#tpg270079-bib-0100" class="usa-link" aria-describedby="tpg270079-bib-0100">2022</a>). A graph‐based reference can also be built by aligning chromosome‐level genome assemblies, and this approach is more accurate than adding a catalog of SVs to a single reference (Hickey et al., <a href="#tpg270079-bib-0042" class="usa-link" aria-describedby="tpg270079-bib-0042">2020</a>). However, so far, not many attempts have been made to perform GWAS or QTL analysis in plants with this type of graph‐based reference built from de novo assemblies (Vaughn et al., <a href="#tpg270079-bib-0092" class="usa-link" aria-describedby="tpg270079-bib-0092">2022</a>). To explore the potential of the graph‐based reference built from de novo assemblies for future pangenomic studies in <em>Lolium</em> spp., we generated a graph‐based reference with all three Rabiosa assemblies (Rabiosa v2, h1, and h2) and performed QTL analysis for stem rust resistance with the graph‐based reference. Based on this graph‐based reference, we identified two significant QTL for stem rust resistance, and the same two QTL were also found by the QTL analysis with a single reference (Rabiosa v2, Figure <a href="#tpg270079-fig-0005" class="usa-link">5c</a>; Table <a href="#tpg270079-tbl-0002" class="usa-link">2</a>). This suggests that the graph‐based reference together with the current pangenomic tools, such as the PGGB (Garrison et al., <a href="#tpg270079-bib-0029" class="usa-link" aria-describedby="tpg270079-bib-0029">2023</a>), haplotype‐aware read mapper (Giraffe, Sirén et al., <a href="#tpg270079-bib-0085" class="usa-link" aria-describedby="tpg270079-bib-0085">2021</a>), and the toolkit to manipulate the graph‐based reference (vg toolkit, Garrison et al., <a href="#tpg270079-bib-0030" class="usa-link" aria-describedby="tpg270079-bib-0030">2018</a>), can be reliably applied to highly heterozygous <em>Lolium</em> genomes. However, several challenges with the pangenomic approach should be noted. First, due to the high sequence variation between haplotypes, haplomes are difficult to align, and a more suitable WGA tool is needed. For example, instead of using the standard aligner wfmash in PGGB, we used AnchorWave (Song et al., <a href="#tpg270079-bib-0087" class="usa-link" aria-describedby="tpg270079-bib-0087">2022</a>), which is a synteny‐based aligner for pair‐wise WGA as it is specifically designed for highly heterozygous plant genomes. However, even with AnchorWave, we found that the two haplotypes were not well aligned, leaving the graph‐based reference 1.42 Gb larger than the total size of the pseudo‐chromosomes of Rabiosa v2 (3.66 vs. 2.24 Gb). In contrast, for a pangenome of human built from 90 haplotypes with PGGB, the size of the graph was only 5 Gb, adding just 2 Gb more sequence variations from 90 haplotypes to the haploid genome (genome size 3 Gb, Liao et al., <a href="#tpg270079-bib-0056" class="usa-link" aria-describedby="tpg270079-bib-0056">2023</a>). Second, although QTL analysis worked with the graph‐based reference, we found that the QTL detected on LG 7 with the graph‐based reference showed a lower LOD score compared to the same QTL detected with the single reference. However, this does not necessarily indicate that the QTL analysis with the graph‐based reference performed worse than QTL analysis with the single reference. A possible reason for the lower LOD score with the graph‐based reference could be that fewer SNPs were genotyped in the QTL region with the graph‐based reference due to the difficulty of aligning the two divergent haplotypes (Figure <a href="#tpg270079-fig-0005" class="usa-link">5c</a>). To improve QTL analysis using the graph‐based reference, small INDELs and large SVs present in the graph‐based reference need to be considered in addition to SNPs (Vaughn et al., <a href="#tpg270079-bib-0092" class="usa-link" aria-describedby="tpg270079-bib-0092">2022</a>) as SNPs may not be called or genotyped if the two haplotypes could not be aligned.</p>
<p>In conclusion, the chromosome‐level unphased haploid assembly (Rabiosa v2) and the two phased haplotype assemblies (Rabiosa h1 and h2) that we have generated provide invaluable genomic resources for the forage and turf grass research and breeding community. They will serve as references for Italian ryegrass for genomic applications such as reference‐assisted scaffolding, reference‐based phasing, QTL analysis, and genome‐wide association studies. The two QTL linked to stem rust resistance were found with the graph‐based reference, and the comparison between the QTL analyses done with the graph‐based reference and the single reference emphasizes the potential of graph‐based reference and pangenomic tools for future research in forage grasses and other highly heterozygous plant species. While the value of such graph‐based references and pangenomic methods may not be fully exploited with the genomic resources presented here, it will increase in the future as more and more complex genomes are becoming available and the community's needs will develop beyond single reference sequences.</p></section><section id="tpg270079-sec-0280"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Yutang Chen</strong>: Data curation; formal analysis; investigation; methodology; visualization; writing—original draft; writing—review and editing. <strong>Jenny Kiesbauer</strong>: Formal analysis; investigation; methodology; writing—review and editing. <strong>Dario Copetti</strong>: Data curation; methodology; supervision; writing—review and editing. <strong>Daniel Frei</strong>: Data curation; methodology; writing—review and editing. <strong>Jürg E. Frey</strong>: Data curation; methodology; writing—review and editing. <strong>Christoph Grieder</strong>: Methodology; resources; writing—review and editing. <strong>Roland Kölliker</strong>: Formal analysis; methodology; supervision; writing—review and editing. <strong>Bruno Studer</strong>: Conceptualization; funding acquisition; supervision; writing—review and editing.</p></section><section id="tpg270079-sec-0300"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec30"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270079-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270079-supitem-0001"><div class="caption p">
<p>Figure S1. Consensus genetic linkage maps used for scaffolding Rabiosa v1, as output by Lep‐MAP3 (Rastas, <a href="#tpg270079-bib-0076" class="usa-link" aria-describedby="tpg270079-bib-0076">2017</a>).</p>
<p>Figure S2. Gene‐based synteny between Rabiosa and other Pooideae genomes.</p>
<p>Figure S3. Pair‐wise gene‐based synteny between Rabiosa v2, Rabiosa h1, and h2.</p>
<p>Figure S4. K‐mer profile of the Rabiosa diploid genome generated using GenomeScope2 with k‐mers from whole‐genome sequencing short reads.</p>
<p>Figure S5. Sequence identity between Rabiosa h1 and Rabiosa h2 for the chromosomes Chr2 to Chr7 (the comparison between the two haplotypes of Chr1 is given in the main text).</p>
<p>Figure S6. Histogram of the phenotypic distribution of resistance to stem rust based on the best linear unbiased estimators (BLUEs).</p>
<p>Figure S7. QTL analysis for resistance to stem rust using the genetic linkage map of Sikem.</p>
<p>Figure S8. Reference bias affects reference‐based phasing.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12376112/bin/TPG2-18-e70079-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70079-s001.docx</a><sup> (1.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270079-supitem-0002"><div class="caption p">
<p>Table S1: Assembly statistics of Rabiosa</p>
<p>Table S2: Statistics of phase blocks</p>
<p>Table S3: Statistics of binned reads from reference‐based phasing</p>
<p>Table S4: BUSCO (v4.1.4) assessment results of genome assemblies</p>
<p>Table S5: Transposable elements (TE) annotation results</p>
<p>Table S6: Number of variants for calculating sequence variation between the two haplotypes of Rabiosa</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12376112/bin/TPG2-18-e70079-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70079-s002.xlsx</a><sup> (23.8KB, xlsx) </sup>
</div></div></section></section><section id="tpg270079-sec-0290" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This study was supported by the European Union's Horizon 2020 research and innovation program under the Marie Skłodowska‐Curie grant agreement No 847585 – RESPONSE. We would like to sincerely thank Ingo Lenk and Elisabeth Veeckman from DLF Seeds A/S for their valuable input and discussion in genome assembly, genome annotation, and other bioinformatic analyses. We would also like to thank ISG‐HEST at ETH Zurich for providing computational resources as well as their IT service for this work</p></section><section id="notes1"><p>


Chen, Y.
, 
Kiesbauer, J.
, 
Copetti, D.
, 
Frei, D.
, 
Frey, J. E.
, 
Grieder, C.
, 
Kölliker, R.
, &amp; 
Studer, B.
 (2025). Chromosome‐level haplotype‐resolved genome assembly provides insights into the highly heterozygous genome of Italian ryegrass (<em>Lolium multiflorum</em> Lam.). The Plant Genome, 18, e70079. 10.1002/tpg2.70079

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270079-note-0001"><p>Assigned to Associate Editor Katrien Devos.</p></div></div></section></section><section id="tpg270079-sec-0320"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The unphased haploid genome assembly Rabiosa v2 is publicly available on NCBI under the accession: <a href="https://www.ncbi.nlm.nih.gov/nuccore/JAUUTY000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JAUUTY000000000</a>. The two haplotype assemblies are publicly available on NCBI under the accession: <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBHEQO000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBHEQO000000000</a>. The transcripts used in our haplotig purge pipeline generated previously by Copetti et al. (<a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>), together with the genome annotation results including the GFF files and the functional annotation results, are available at <a href="https://doi.org/10.5281/zenodo.13641219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.13641219</a>. All genetic linkage maps generated in this work together with the stem rust resistance scores (BLUEs) used for QTL analysis are available at <a href="https://doi.org/10.5281/zenodo.15009016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15009016</a>. All sequencing data (ONT data, WGS short reads, Hi‐C data, and GBS data) generated in this work have been submitted to NCBI under BioProject accession: PRJNA990649. The RNA‐Seq data generated previously by Copetti et al. (<a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>) are available on NCBI with the BioProject accession: PRJNA1156394. Code used for bioinformatic analyses in this work can be found at GitHub, <a href="https://github.com/Yutang-ETH/Rabiosa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Yutang‐ETH/Rabiosa</a>.</p></section><section id="tpg270079-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270079-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270079-bib-0001">
<cite>

Alonge, M.
, 
Lebeigle, L.
, 
Kirsche, M.
, 
Jenike, K.
, 
Ou, S.
, 
Aganezov, S.
, 
Wang, X.
, 
Lippman, Z. B.
, 
Schatz, M. C.
, &amp; 
Soyk, S.
 (2022). Automated assembly scaffolding using RagTag elevates a new tomato system for high‐throughput genome editing. Genome Biology, 23, Article 258. 10.1186/s13059-022-02823-7

</cite> [<a href="https://doi.org/10.1186/s13059-022-02823-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9753292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36522651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Automated%20assembly%20scaffolding%20using%20RagTag%20elevates%20a%20new%20tomato%20system%20for%20high%E2%80%90throughput%20genome%20editing&amp;author=M.%20Alonge&amp;author=L.%20Lebeigle&amp;author=M.%20Kirsche&amp;author=K.%20Jenike&amp;author=S.%20Ou&amp;volume=23&amp;publication_year=2022&amp;pages=258&amp;pmid=36522651&amp;doi=10.1186/s13059-022-02823-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0002">
<cite>

Altschul, S. F.
, 
Gish, W.
, 
Miller, W.
, 
Myers, E. W.
, &amp; 
Lipman, D. J.
 (1990). Basic local alignment search tool. Journal of Molecular Biology, 215, 403–410. 10.1016/S0022-2836(05)80360-2

</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;volume=215&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;doi=10.1016/S0022-2836(05)80360-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0003">
<cite>

Andrews, S.
 (2015). 
FastQC
. Babraham Bioinformatics. <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Andrews,%20S.%20(2015).%20FastQC%20.%20Babraham%20Bioinformatics.%20https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0004">
<cite>

Bao, Z.
, 
Li, C.
, 
Li, G.
, 
Wang, P.
, 
Peng, Z.
, 
Cheng, L.
, 
Li, H.
, 
Zhang, Z.
, 
Li, Y.
, 
Huang, W. U.
, 
Ye, M.
, 
Dong, D.
, 
Cheng, Z.
, 
Vanderzaag, P.
, 
Jacobsen, E.
, 
Bachem, C. W. B.
, 
Dong, S.
, 
Zhang, C.
, 
Huang, S.
, &amp; 
Zhou, Q.
 (2022). Genome architecture and tetrasomic inheritance of autotetraploid potato. Molecular Plant, 15, 1211–1226. 10.1016/j.molp.2022.06.009

</cite> [<a href="https://doi.org/10.1016/j.molp.2022.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35733345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Genome%20architecture%20and%20tetrasomic%20inheritance%20of%20autotetraploid%20potato&amp;author=Z.%20Bao&amp;author=C.%20Li&amp;author=G.%20Li&amp;author=P.%20Wang&amp;author=Z.%20Peng&amp;volume=15&amp;publication_year=2022&amp;pages=1211-1226&amp;pmid=35733345&amp;doi=10.1016/j.molp.2022.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0090">
<cite>

Bateman, A.
, 
Martin, M.‐J.
, 
Orchard, S.
, 
Magrane, M.
, 
Agivetova, R.
, 
Ahmad, S.
, 
Alpi, E.
, 
Bowler‐Barnett, E. H.
, 
Britto, R.
, 
Bursteinas, B.
, 
Bye‐A‐Jee, H.
, 
Coetzee, R.
, 
Cukura, A.
, 
Da Silva, A.
, 
Denny, P.
, 
Dogan, T.
, 
Ebenezer, T.
, 
Fan, J.
, … 
Teodoro, D.
, The UniProt Consortium
. (2021). UniProt: The universal protein knowledgebase in 2021. Nucleic Acids Research, 49, D480–D489. 10.1093/nar/gkaa1100

</cite> [<a href="https://doi.org/10.1093/nar/gkaa1100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33237286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=UniProt:%20The%20universal%20protein%20knowledgebase%20in%202021&amp;author=A.%20Bateman&amp;author=M.%E2%80%90J.%20Martin&amp;author=S.%20Orchard&amp;author=M.%20Magrane&amp;author=R.%20Agivetova&amp;volume=49&amp;publication_year=2021&amp;pages=D480-D489&amp;pmid=33237286&amp;doi=10.1093/nar/gkaa1100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0005">
<cite>

Bates, D.
, 
Mächler, M.
, 
Bolker, B.
, &amp; 
Walker, S.
 (2015). Fitting linear mixed‐effects models using lme4. Journal of Statistical Software, 67, 1–48. 10.18637/jss.v067.i01
</cite> [<a href="https://doi.org/10.18637/jss.v067.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Fitting%20linear%20mixed%E2%80%90effects%20models%20using%20lme4&amp;author=D.%20Bates&amp;author=M.%20M%C3%A4chler&amp;author=B.%20Bolker&amp;author=S.%20Walker&amp;volume=67&amp;publication_year=2015&amp;pages=1-48&amp;doi=10.18637/jss.v067.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0006">
<cite>

Begheyn, R. F.
, 
Yates, S. A.
, 
Sykes, T.
, &amp; 
Studer, B.
 (2018). Genetic loci governing androgenic capacity in perennial ryegrass (<em>Lolium perenne</em> L.). G3 Genes|Genomes|Genetics, 8, 1897–1908. 10.1534/g3.117.300550

</cite> [<a href="https://doi.org/10.1534/g3.117.300550" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5982819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29626084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Genetic%20loci%20governing%20androgenic%20capacity%20in%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;author=R.%20F.%20Begheyn&amp;author=S.%20A.%20Yates&amp;author=T.%20Sykes&amp;author=B.%20Studer&amp;volume=8&amp;publication_year=2018&amp;pages=1897-1908&amp;pmid=29626084&amp;doi=10.1534/g3.117.300550&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0007">
<cite>

Bolger, A. M.
, 
Lohse, M.
, &amp; 
Usadel, B.
 (2014). Trimmomatic: A flexible trimmer for Illumina sequence data. Bioinformatics, 30, 2114–2120. 10.1093/bioinformatics/btu170

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20A%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0008">
<cite>

Boller, B.
, 
Posselt, U. K.
, &amp; 
Veronesi, F.
 (Eds.). (2010). Fodder crops and amenity grasses (Vol. 212). Springer. 10.1007/978-1-4419-0760-8
</cite> [<a href="https://doi.org/10.1007/978-1-4419-0760-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Fodder%20crops%20and%20amenity%20grasses&amp;publication_year=2010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0009">
<cite>

Broman, K. W.
, 
Wu, H.
, 
Sen, Ś.
, &amp; 
Churchill, G. A.
 (2003). R/qtl: QTL mapping in experimental crosses. Bioinformatics, 19, 889–890. 10.1093/bioinformatics/btg112

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btg112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12724300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=R/qtl:%20QTL%20mapping%20in%20experimental%20crosses&amp;author=K.%20W.%20Broman&amp;author=H.%20Wu&amp;author=%C5%9A.%20Sen&amp;author=G.%20A.%20Churchill&amp;volume=19&amp;publication_year=2003&amp;pages=889-890&amp;pmid=12724300&amp;doi=10.1093/bioinformatics/btg112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0010">
<cite>

Brůna, T.
, 
Hoff, K. J.
, 
Lomsadze, A.
, 
Stanke, M.
, &amp; 
Borodovsky, M.
 (2021). BRAKER2: Automatic eukaryotic genome annotation with GeneMark‐EP+ and AUGUSTUS supported by a protein database. NAR Genomics and Bioinformatics, 3, lqaa108. 10.1093/nargab/lqaa108

</cite> [<a href="https://doi.org/10.1093/nargab/lqaa108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7787252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33575650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Genomics%20and%20Bioinformatics&amp;title=BRAKER2:%20Automatic%20eukaryotic%20genome%20annotation%20with%20GeneMark%E2%80%90EP+%20and%20AUGUSTUS%20supported%20by%20a%20protein%20database&amp;author=T.%20Br%C5%AFna&amp;author=K.%20J.%20Hoff&amp;author=A.%20Lomsadze&amp;author=M.%20Stanke&amp;author=M.%20Borodovsky&amp;volume=3&amp;publication_year=2021&amp;pages=lqaa108&amp;pmid=33575650&amp;doi=10.1093/nargab/lqaa108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0011">
<cite>

Brunharo, C. A.
, 
Short, A. W.
, 
Bobadilla, L. K.
, &amp; 
Streisfeld, M. A.
 (2025). The genome of <em>Lolium multiflorum</em> reveals the genetic architecture of paraquat resistance. Molecular Ecology, 34, e17775. 10.1111/mec.17775

</cite> [<a href="https://doi.org/10.1111/mec.17775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12051776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40285737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=The%20genome%20of%20Lolium%20multiflorum%20reveals%20the%20genetic%20architecture%20of%20paraquat%20resistance&amp;author=C.%20A.%20Brunharo&amp;author=A.%20W.%20Short&amp;author=L.%20K.%20Bobadilla&amp;author=M.%20A.%20Streisfeld&amp;volume=34&amp;publication_year=2025&amp;pages=e17775&amp;pmid=40285737&amp;doi=10.1111/mec.17775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0012">
<cite>

Buchfink, B.
, 
Xie, C.
, &amp; 
Huson, D. H.
 (2015). Fast and sensitive protein alignment using DIAMOND. Nature Methods, 12, 59–60. 10.1038/nmeth.3176

</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND&amp;author=B.%20Buchfink&amp;author=C.%20Xie&amp;author=D.%20H.%20Huson&amp;volume=12&amp;publication_year=2015&amp;pages=59-60&amp;pmid=25402007&amp;doi=10.1038/nmeth.3176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0013">
<cite>

Campbell, M. S.
, 
Law, M.
, 
Holt, C.
, 
Stein, J. C.
, 
Moghe, G. D.
, 
Hufnagel, D. E.
, 
Lei, J.
, 
Achawanantakun, R.
, 
Jiao, D.
, 
Lawrence, C. J.
, 
Ware, D.
, 
Shiu, S.‐H.
, 
Childs, K. L.
, 
Sun, Y.
, 
Jiang, N.
, &amp; 
Yandell, M.
 (2014). MAKER‐P: A tool kit for the rapid creation, management, and quality control of plant genome annotations. Plant Physiology, 164, 513–524. 10.1104/pp.113.230144

</cite> [<a href="https://doi.org/10.1104/pp.113.230144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3912085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24306534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=MAKER%E2%80%90P:%20A%20tool%20kit%20for%20the%20rapid%20creation,%20management,%20and%20quality%20control%20of%20plant%20genome%20annotations&amp;author=M.%20S.%20Campbell&amp;author=M.%20Law&amp;author=C.%20Holt&amp;author=J.%20C.%20Stein&amp;author=G.%20D.%20Moghe&amp;volume=164&amp;publication_year=2014&amp;pages=513-524&amp;pmid=24306534&amp;doi=10.1104/pp.113.230144&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0014">
<cite>

Chen, J.
, 
Liu, Y.
, 
Liu, M.
, 
Guo, W.
, 
Wang, Y.
, 
He, Q.
, 
Chen, W.
, 
Liao, Y.
, 
Zhang, W.
, 
Gao, Y.
, 
Dong, K.
, 
Ren, R.
, 
Yang, T.
, 
Zhang, L.
, 
Qi, M.
, 
Li, Z.
, 
Zhao, M.
, 
Wang, H.
, 
Wang, J.
, … 
Diao, X.
 (2023). Pangenome analysis reveals genomic variations associated with domestication traits in broomcorn millet. Nature Genetics, 55, 2243–2254. 10.1038/s41588-023-01571-z

</cite> [<a href="https://doi.org/10.1038/s41588-023-01571-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10703678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38036791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Pangenome%20analysis%20reveals%20genomic%20variations%20associated%20with%20domestication%20traits%20in%20broomcorn%20millet&amp;author=J.%20Chen&amp;author=Y.%20Liu&amp;author=M.%20Liu&amp;author=W.%20Guo&amp;author=Y.%20Wang&amp;volume=55&amp;publication_year=2023&amp;pages=2243-2254&amp;pmid=38036791&amp;doi=10.1038/s41588-023-01571-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0015">
<cite>

Chen, S.
, 
Zhou, Y.
, 
Chen, Y.
, &amp; 
Gu, J.
 (2018). fastp: An ultra‐fast all‐in‐one FASTQ preprocessor. Bioinformatics, 34, i884–i890. 10.1093/bioinformatics/bty560

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=fastp:%20An%20ultra%E2%80%90fast%20all%E2%80%90in%E2%80%90one%20FASTQ%20preprocessor&amp;author=S.%20Chen&amp;author=Y.%20Zhou&amp;author=Y.%20Chen&amp;author=J.%20Gu&amp;volume=34&amp;publication_year=2018&amp;pages=i884-i890&amp;pmid=30423086&amp;doi=10.1093/bioinformatics/bty560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0016">
<cite>

Chen, Y.
, 
Kölliker, R.
, 
Mascher, M.
, 
Copetti, D.
, 
Himmelbach, A.
, 
Stein, N.
, &amp; 
Studer, B.
 (2024). An improved chromosome‐level genome assembly of perennial ryegrass (<em>Lolium perenne</em> L.). Gigabyte, 2024, 1–11. 10.46471/gigabyte.112
</cite> [<a href="https://doi.org/10.46471/gigabyte.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10940895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38496214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gigabyte&amp;title=An%20improved%20chromosome%E2%80%90level%20genome%20assembly%20of%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;author=Y.%20Chen&amp;author=R.%20K%C3%B6lliker&amp;author=M.%20Mascher&amp;author=D.%20Copetti&amp;author=A.%20Himmelbach&amp;volume=2024&amp;publication_year=2024&amp;pages=1-11&amp;pmid=38496214&amp;doi=10.46471/gigabyte.112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0017">
<cite>

Cheng, H.
, 
Concepcion, G. T.
, 
Feng, X.
, 
Zhang, H.
, &amp; 
Li, H.
 (2021). Haplotype‐resolved de novo assembly using phased assembly graphs with hifiasm. Nature Methods, 18, 170–175. 10.1038/s41592-020-01056-5

</cite> [<a href="https://doi.org/10.1038/s41592-020-01056-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7961889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33526886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Haplotype%E2%80%90resolved%20de%20novo%20assembly%20using%20phased%20assembly%20graphs%20with%20hifiasm&amp;author=H.%20Cheng&amp;author=G.%20T.%20Concepcion&amp;author=X.%20Feng&amp;author=H.%20Zhang&amp;author=H.%20Li&amp;volume=18&amp;publication_year=2021&amp;pages=170-175&amp;pmid=33526886&amp;doi=10.1038/s41592-020-01056-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0018">
<cite>

Chin, C.‐S.
, 
Peluso, P.
, 
Sedlazeck, F. J.
, 
Nattestad, M.
, 
Concepcion, G. T.
, 
Clum, A.
, 
Dunn, C.
, 
O'Malley, R.
, 
Figueroa‐Balderas, R.
, 
Morales‐Cruz, A.
, 
Cramer, G. R.
, 
Delledonne, M.
, 
Luo, C.
, 
Ecker, J. R.
, 
Cantu, D.
, 
Rank, D. R.
, &amp; 
Schatz, M. C.
 (2016). Phased diploid genome assembly with single‐molecule real‐time sequencing. Nature Methods, 13, 1050–1054. 10.1038/nmeth.4035

</cite> [<a href="https://doi.org/10.1038/nmeth.4035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5503144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27749838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Phased%20diploid%20genome%20assembly%20with%20single%E2%80%90molecule%20real%E2%80%90time%20sequencing&amp;author=C.%E2%80%90S.%20Chin&amp;author=P.%20Peluso&amp;author=F.%20J.%20Sedlazeck&amp;author=M.%20Nattestad&amp;author=G.%20T.%20Concepcion&amp;volume=13&amp;publication_year=2016&amp;pages=1050-1054&amp;pmid=27749838&amp;doi=10.1038/nmeth.4035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0019">
<cite>

Cochetel, N.
, 
Minio, A.
, 
Guarracino, A.
, 
Garcia, J. F.
, 
Figueroa‐Balderas, R.
, 
Massonnet, M.
, 
Kasuga, T.
, 
Londo, J. P.
, 
Garrison, E.
, 
Gaut, B. S.
, &amp; 
Cantu, D.
 (2023). A super‐pangenome of the North American wild grape species. Genome Biology, 24, Article 290. 10.1186/s13059-023-03133-2

</cite> [<a href="https://doi.org/10.1186/s13059-023-03133-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10729490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38111050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=A%20super%E2%80%90pangenome%20of%20the%20North%20American%20wild%20grape%20species&amp;author=N.%20Cochetel&amp;author=A.%20Minio&amp;author=A.%20Guarracino&amp;author=J.%20F.%20Garcia&amp;author=R.%20Figueroa%E2%80%90Balderas&amp;volume=24&amp;publication_year=2023&amp;pages=290&amp;pmid=38111050&amp;doi=10.1186/s13059-023-03133-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0020">
<cite>

Copetti, D.
, 
Yates, S. A.
, 
Vogt, M. M.
, 
Russo, G.
, 
Grieder, C.
, 
Kölliker, R.
, &amp; 
Studer, B.
 (2021). Evidence for high intergenic sequence variation in heterozygous Italian ryegrass (Lolium multiflorum Lam.) genome revealed by a high‐quality draft diploid genome assembly. BioRxiv. 10.1101/2021.05.05.442707
</cite> [<a href="https://doi.org/10.1101/2021.05.05.442707" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270079-bib-0021">
<cite>

Danecek, P.
, 
Bonfield, J. K.
, 
Liddle, J.
, 
Marshall, J.
, 
Ohan, V.
, 
Pollard, M. O.
, 
Whitwham, A.
, 
Keane, T.
, 
Mccarthy, S. A.
, 
Davies, R. M.
, &amp; 
Li, H.
 (2021). Twelve years of SAMtools and BCFtools. GigaScience, 10, giab008. 10.1093/gigascience/giab008

</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&amp;author=P.%20Danecek&amp;author=J.%20K.%20Bonfield&amp;author=J.%20Liddle&amp;author=J.%20Marshall&amp;author=V.%20Ohan&amp;volume=10&amp;publication_year=2021&amp;pages=giab008&amp;pmid=33590861&amp;doi=10.1093/gigascience/giab008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0022">
<cite>

Dudchenko, O.
, 
Batra, S. S.
, 
Omer, A. D.
, 
Nyquist, S. K.
, 
Hoeger, M.
, 
Durand, N. C.
, 
Shamim, M. S.
, 
Machol, I.
, 
Lander, E. S.
, 
Aiden, A. P.
, &amp; 
Aiden, E. L.
 (2017). De novo assembly of the <em>Aedes aegypti</em> genome using Hi‐C yields chromosome‐length scaffolds. Science, 356, 92–95. 10.1126/science.aal3327

</cite> [<a href="https://doi.org/10.1126/science.aal3327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5635820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28336562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=De%20novo%20assembly%20of%20the%20Aedes%20aegypti%20genome%20using%20Hi%E2%80%90C%20yields%20chromosome%E2%80%90length%20scaffolds&amp;author=O.%20Dudchenko&amp;author=S.%20S.%20Batra&amp;author=A.%20D.%20Omer&amp;author=S.%20K.%20Nyquist&amp;author=M.%20Hoeger&amp;volume=356&amp;publication_year=2017&amp;pages=92-95&amp;pmid=28336562&amp;doi=10.1126/science.aal3327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0023">
<cite>

Durand, N. C.
, 
Robinson, J. T.
, 
Shamim, M. S.
, 
Machol, I.
, 
Mesirov, J. P.
, 
Lander, E. S.
, &amp; 
Aiden, E. L.
 (2016). Juicebox provides a visualization system for Hi‐C contact maps with unlimited zoom. Cell Systems, 3, 99–101. 10.1016/j.cels.2015.07.012

</cite> [<a href="https://doi.org/10.1016/j.cels.2015.07.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5596920/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27467250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=Juicebox%20provides%20a%20visualization%20system%20for%20Hi%E2%80%90C%20contact%20maps%20with%20unlimited%20zoom&amp;author=N.%20C.%20Durand&amp;author=J.%20T.%20Robinson&amp;author=M.%20S.%20Shamim&amp;author=I.%20Machol&amp;author=J.%20P.%20Mesirov&amp;volume=3&amp;publication_year=2016&amp;pages=99-101&amp;pmid=27467250&amp;doi=10.1016/j.cels.2015.07.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0024">
<cite>

Durand, N. C.
, 
Shamim, M. S.
, 
Machol, I.
, 
Rao, S. S. P.
, 
Huntley, M. H.
, 
Lander, E. S.
, &amp; 
Aiden, E. L.
 (2016). Juicer provides a one‐click system for analyzing loop‐resolution Hi‐C experiments. Cell Systems, 3, 95–98. 10.1016/j.cels.2016.07.002

</cite> [<a href="https://doi.org/10.1016/j.cels.2016.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5846465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=Juicer%20provides%20a%20one%E2%80%90click%20system%20for%20analyzing%20loop%E2%80%90resolution%20Hi%E2%80%90C%20experiments&amp;author=N.%20C.%20Durand&amp;author=M.%20S.%20Shamim&amp;author=I.%20Machol&amp;author=S.%20S.%20P.%20Rao&amp;author=M.%20H.%20Huntley&amp;volume=3&amp;publication_year=2016&amp;pages=95-98&amp;pmid=27467249&amp;doi=10.1016/j.cels.2016.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0025">
<cite>

Edge, P.
, 
Bafna, V.
, &amp; 
Bansal, V.
 (2016). HapCUT2: Robust and accurate haplotype assembly for diverse sequencing technologies. Genome Research, 27, 801–812. 10.1101/gr.213462.116

</cite> [<a href="https://doi.org/10.1101/gr.213462.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27940952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=HapCUT2:%20Robust%20and%20accurate%20haplotype%20assembly%20for%20diverse%20sequencing%20technologies&amp;author=P.%20Edge&amp;author=V.%20Bafna&amp;author=V.%20Bansal&amp;volume=27&amp;publication_year=2016&amp;pages=801-812&amp;pmid=27940952&amp;doi=10.1101/gr.213462.116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0026">
<cite>

Flynn, J. M.
, 
Hubley, R.
, 
Goubert, C.
, 
Rosen, J.
, 
Clark, A. G.
, 
Feschotte, C.
, &amp; 
Smit, A. F.
 (2020). RepeatModeler2 for automated genomic discovery of transposable element families. Proceedings of the National Academy of Sciences of the USA, 117, 9451–9457. 10.1073/pnas.1921046117

</cite> [<a href="https://doi.org/10.1073/pnas.1921046117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20USA&amp;title=RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families&amp;author=J.%20M.%20Flynn&amp;author=R.%20Hubley&amp;author=C.%20Goubert&amp;author=J.%20Rosen&amp;author=A.%20G.%20Clark&amp;volume=117&amp;publication_year=2020&amp;pages=9451-9457&amp;pmid=32300014&amp;doi=10.1073/pnas.1921046117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0027">
<cite>

Frei, D.
, 
Veekman, E.
, 
Grogg, D.
, 
Stoffel‐Studer, I.
, 
Morishima, A.
, 
Shimizu‐Inatsugi, R.
, 
Yates, S.
, 
Shimizu, K. K.
, 
Frey, J. E.
, 
Studer, B.
, &amp; 
Copetti, D.
 (2021). Ultralong Oxford Nanopore reads enable the development of a reference‐grade perennial ryegrass genome assembly. Genome Biology and Evolution, 13, evab159. 10.1093/gbe/evab159

</cite> [<a href="https://doi.org/10.1093/gbe/evab159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8358221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34247248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=Ultralong%20Oxford%20Nanopore%20reads%20enable%20the%20development%20of%20a%20reference%E2%80%90grade%20perennial%20ryegrass%20genome%20assembly&amp;author=D.%20Frei&amp;author=E.%20Veekman&amp;author=D.%20Grogg&amp;author=I.%20Stoffel%E2%80%90Studer&amp;author=A.%20Morishima&amp;volume=13&amp;publication_year=2021&amp;pages=evab159&amp;pmid=34247248&amp;doi=10.1093/gbe/evab159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0028">
<cite>

Garg, S.
, 
Fungtammasan, A.
, 
Carroll, A.
, 
Chou, M.
, 
Schmitt, A.
, 
Zhou, X.
, 
Mac, S.
, 
Peluso, P.
, 
Hatas, E.
, 
Ghurye, J.
, 
Maguire, J.
, 
Mahmoud, M.
, 
Cheng, H.
, 
Heller, D.
, 
Zook, J. M.
, 
Moemke, T.
, 
Marschall, T.
, 
Sedlazeck, F. J.
, 
Aach, J.
, … 
Li, H.
 (2021). Chromosome‐scale, haplotype‐resolved assembly of human genomes. Nature Biotechnology, 39, 309–312. 10.1038/s41587-020-0711-0
</cite> [<a href="https://doi.org/10.1038/s41587-020-0711-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7954703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33288905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Chromosome%E2%80%90scale,%20haplotype%E2%80%90resolved%20assembly%20of%20human%20genomes&amp;author=S.%20Garg&amp;author=A.%20Fungtammasan&amp;author=A.%20Carroll&amp;author=M.%20Chou&amp;author=A.%20Schmitt&amp;volume=39&amp;publication_year=2021&amp;pages=309-312&amp;pmid=33288905&amp;doi=10.1038/s41587-020-0711-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0029">
<cite>

Garrison, E.
, 
Guarracino, A.
, 
Heumos, S.
, 
Villani, F.
, 
Bao, Z.
, 
Tattini, L.
, 
Hagmann, J.
, 
Vorbrugg, S.
, 
Marco‐Sola, S.
, 
Kubica, C.
, 
Ashbrook, D. G.
, 
Thorell, K.
, 
Rusholme‐Pilcher, R. L.
, 
Liti, G.
, 
Rudbeck, E.
, 
Golicz, A. A.
, 
Nahnsen, S.
, 
Yang, Z.
, 
Mwaniki, M. N.
, … 
Prins, P.
 (2023). Building pangenome graphs. BioRxiv.</cite> [<a href="https://doi.org/10.1038/s41592-024-02430-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39433878/" class="usa-link">PubMed</a>]</li>
<li id="tpg270079-bib-0030">
<cite>

Garrison, E.
, 
Sirén, J.
, 
Novak, A. M.
, 
Hickey, G.
, 
Eizenga, J. M.
, 
Dawson, E. T.
, 
Jones, W.
, 
Garg, S.
, 
Markello, C.
, 
Lin, M. F.
, 
Paten, B.
, &amp; 
Durbin, R.
 (2018). Variation graph toolkit improves read mapping by representing genetic variation in the reference. Nature Biotechnology, 36, 875–879. 10.1038/nbt.4227
</cite> [<a href="https://doi.org/10.1038/nbt.4227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6126949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30125266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Variation%20graph%20toolkit%20improves%20read%20mapping%20by%20representing%20genetic%20variation%20in%20the%20reference&amp;author=E.%20Garrison&amp;author=J.%20Sir%C3%A9n&amp;author=A.%20M.%20Novak&amp;author=G.%20Hickey&amp;author=J.%20M.%20Eizenga&amp;volume=36&amp;publication_year=2018&amp;pages=875-879&amp;pmid=30125266&amp;doi=10.1038/nbt.4227&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0031">
<cite>

Ghurye, J.
, 
Rhie, A.
, 
Walenz, B. P.
, 
Schmitt, A.
, 
Selvaraj, S.
, 
Pop, M.
, 
Phillippy, A. M.
, &amp; 
Koren, S.
 (2019). Integrating Hi‐C links with assembly graphs for chromosome‐scale assembly. PLoS Computational Biology, 15, e1007273. 10.1371/journal.pcbi.1007273

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1007273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6719893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31433799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Computational%20Biology&amp;title=Integrating%20Hi%E2%80%90C%20links%20with%20assembly%20graphs%20for%20chromosome%E2%80%90scale%20assembly&amp;author=J.%20Ghurye&amp;author=A.%20Rhie&amp;author=B.%20P.%20Walenz&amp;author=A.%20Schmitt&amp;author=S.%20Selvaraj&amp;volume=15&amp;publication_year=2019&amp;pages=e1007273&amp;pmid=31433799&amp;doi=10.1371/journal.pcbi.1007273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0032">
<cite>

Goel, M.
, 
Sun, H.
, 
Jiao, W.‐B.
, &amp; 
Schneeberger, K.
 (2019). SyRI: Finding genomic rearrangements and local sequence differences from whole‐genome assemblies. Genome Biology, 20, Article 277. 10.1186/s13059-019-1911-0

</cite> [<a href="https://doi.org/10.1186/s13059-019-1911-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6913012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31842948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=SyRI:%20Finding%20genomic%20rearrangements%20and%20local%20sequence%20differences%20from%20whole%E2%80%90genome%20assemblies&amp;author=M.%20Goel&amp;author=H.%20Sun&amp;author=W.%E2%80%90B.%20Jiao&amp;author=K.%20Schneeberger&amp;volume=20&amp;publication_year=2019&amp;pages=277&amp;pmid=31842948&amp;doi=10.1186/s13059-019-1911-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0033">
<cite>

Gremme, G.
, 
Brendel, V.
, 
Sparks, M. E.
, &amp; 
Kurtz, S.
 (2005). Engineering a software tool for gene structure prediction in higher organisms. Information and Software Technology, 47, 965–978. 10.1016/j.infsof.2005.09.005
</cite> [<a href="https://doi.org/10.1016/j.infsof.2005.09.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Information%20and%20Software%20Technology&amp;title=Engineering%20a%20software%20tool%20for%20gene%20structure%20prediction%20in%20higher%20organisms&amp;author=G.%20Gremme&amp;author=V.%20Brendel&amp;author=M.%20E.%20Sparks&amp;author=S.%20Kurtz&amp;volume=47&amp;publication_year=2005&amp;pages=965-978&amp;doi=10.1016/j.infsof.2005.09.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0034">
<cite>

Gremme, G.
, 
Steinbiss, S.
, &amp; 
Kurtz, S.
 (2013). GenomeTools: A comprehensive software library for efficient processing of structured genome annotations. IEEE/ACM Transactions on Computational Biology and Bioinformatics, 10, 645–656. 10.1109/TCBB.2013.68

</cite> [<a href="https://doi.org/10.1109/TCBB.2013.68" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24091398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE/ACM%20Transactions%20on%20Computational%20Biology%20and%20Bioinformatics&amp;title=GenomeTools:%20A%20comprehensive%20software%20library%20for%20efficient%20processing%20of%20structured%20genome%20annotations&amp;author=G.%20Gremme&amp;author=S.%20Steinbiss&amp;author=S.%20Kurtz&amp;volume=10&amp;publication_year=2013&amp;pages=645-656&amp;pmid=24091398&amp;doi=10.1109/TCBB.2013.68&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0035">
<cite>

Guan, D.
, 
McCarthy, S. A.
, 
Wood, J.
, 
Howe, K.
, 
Wang, Y.
, &amp; 
Durbin, R.
 (2020). Identifying and removing haplotypic duplication in primary genome assemblies. Bioinformatics, 36, 2896–2898. 10.1093/bioinformatics/btaa025

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7203741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31971576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Identifying%20and%20removing%20haplotypic%20duplication%20in%20primary%20genome%20assemblies&amp;author=D.%20Guan&amp;author=S.%20A.%20McCarthy&amp;author=J.%20Wood&amp;author=K.%20Howe&amp;author=Y.%20Wang&amp;volume=36&amp;publication_year=2020&amp;pages=2896-2898&amp;pmid=31971576&amp;doi=10.1093/bioinformatics/btaa025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0036">
<cite>

Gui, S.
, 
Wei, W.
, 
Jiang, C.
, 
Luo, J.
, 
Chen, L. U.
, 
Wu, S.
, 
Li, W.
, 
Wang, Y.
, 
Li, S.
, 
Yang, N.
, 
Li, Q.
, 
Fernie, A. R.
, &amp; 
Yan, J.
 (2022). A pan‐Zea genome map for enhancing maize improvement. Genome biology, 23, 178. 10.1186/s13059-022-02742-7

</cite> [<a href="https://doi.org/10.1186/s13059-022-02742-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9396798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35999561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20biology&amp;title=A%20pan%E2%80%90Zea%20genome%20map%20for%20enhancing%20maize%20improvement&amp;author=S.%20Gui&amp;author=W.%20Wei&amp;author=C.%20Jiang&amp;author=J.%20Luo&amp;author=L.%20U.%20Chen&amp;volume=23&amp;publication_year=2022&amp;pages=178&amp;pmid=35999561&amp;doi=10.1186/s13059-022-02742-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0037">
<cite>

Guk, J.‐Y.
, 
Jang, M.‐J.
, 
Choi, J.‐W.
, 
Lee, Y. M.
, &amp; 
Kim, S.
 (2022). De novo phasing resolves haplotype sequences in complex plant genomes. Plant Biotechnology Journal, 20, 1031–1041. 10.1111/pbi.13815

</cite> [<a href="https://doi.org/10.1111/pbi.13815" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9129073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35332665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=De%20novo%20phasing%20resolves%20haplotype%20sequences%20in%20complex%20plant%20genomes&amp;author=J.%E2%80%90Y.%20Guk&amp;author=M.%E2%80%90J.%20Jang&amp;author=J.%E2%80%90W.%20Choi&amp;author=Y.%20M.%20Lee&amp;author=S.%20Kim&amp;volume=20&amp;publication_year=2022&amp;pages=1031-1041&amp;pmid=35332665&amp;doi=10.1111/pbi.13815&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0038">
<cite>

Haas, B. J.
 (2018). 
TransDecoder
. GitHub. <a href="https://github.com/TransDecoder/TransDecoder" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/TransDecoder/TransDecoder</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Haas,%20B.%20J.%20(2018).%20TransDecoder%20.%20GitHub.%20https://github.com/TransDecoder/TransDecoder" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0039">
<cite>

Haas, B. J.
, 
Salzberg, S. L.
, 
Zhu, W.
, 
Pertea, M.
, 
Allen, J. E.
, 
Orvis, J.
, 
White, O.
, 
Buell, C. R.
, &amp; 
Wortman, J. R.
 (2008). Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments. Genome Biology, 9, Article R7. 10.1186/gb-2008-9-1-r7

</cite> [<a href="https://doi.org/10.1186/gb-2008-9-1-r7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2395244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18190707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;author=M.%20Pertea&amp;author=J.%20E.%20Allen&amp;volume=9&amp;publication_year=2008&amp;pages=R7&amp;pmid=18190707&amp;doi=10.1186/gb-2008-9-1-r7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0040">
<cite>

Hayes, B. J.
, 
Cogan, N. O. I.
, 
Pembleton, L. W.
, 
Goddard, M. E.
, 
Wang, J.
, 
Spangenberg, G. C.
, &amp; 
Forster, J. W.
 (2013). Prospects for genomic selection in forage plant species. Plant Breeding, 132, 133–143. 10.1111/pbr.12037
</cite> [<a href="https://doi.org/10.1111/pbr.12037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Prospects%20for%20genomic%20selection%20in%20forage%20plant%20species&amp;author=B.%20J.%20Hayes&amp;author=N.%20O.%20I.%20Cogan&amp;author=L.%20W.%20Pembleton&amp;author=M.%20E.%20Goddard&amp;author=J.%20Wang&amp;volume=132&amp;publication_year=2013&amp;pages=133-143&amp;doi=10.1111/pbr.12037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0041">
<cite>

He, Q.
, 
Tang, S.
, 
Zhi, H.
, 
Chen, J.
, 
Zhang, J.
, 
Liang, H.
, 
Alam, O.
, 
Li, H.
, 
Zhang, H.
, 
Xing, L.
, 
Li, X.
, 
Zhang, W.
, 
Wang, H.
, 
Shi, J.
, 
Du, H.
, 
Wu, H.
, 
Wang, L.
, 
Yang, P.
, 
Xing, L.
, … 
Diao, X.
 (2023). A graph‐based genome and pan‐genome variation of the model plant Setaria. Nature Genetics, 55, 1232–1242. 10.1038/s41588-023-01423-w

</cite> [<a href="https://doi.org/10.1038/s41588-023-01423-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10335933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37291196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=A%20graph%E2%80%90based%20genome%20and%20pan%E2%80%90genome%20variation%20of%20the%20model%20plant%20Setaria&amp;author=Q.%20He&amp;author=S.%20Tang&amp;author=H.%20Zhi&amp;author=J.%20Chen&amp;author=J.%20Zhang&amp;volume=55&amp;publication_year=2023&amp;pages=1232-1242&amp;pmid=37291196&amp;doi=10.1038/s41588-023-01423-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0042">
<cite>

Hickey, G.
, 
Heller, D.
, 
Monlong, J.
, 
Sibbesen, J. A.
, 
Sirén, J.
, 
Eizenga, J.
, 
Dawson, E. T.
, 
Garrison, E.
, 
Novak, A. M.
, &amp; 
Paten, B.
 (2020). Genotyping structural variants in pangenome graphs using the vg toolkit. Genome Biology, 21, Article 35. 10.1186/s13059-020-1941-7

</cite> [<a href="https://doi.org/10.1186/s13059-020-1941-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7017486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32051000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Genotyping%20structural%20variants%20in%20pangenome%20graphs%20using%20the%20vg%20toolkit&amp;author=G.%20Hickey&amp;author=D.%20Heller&amp;author=J.%20Monlong&amp;author=J.%20A.%20Sibbesen&amp;author=J.%20Sir%C3%A9n&amp;volume=21&amp;publication_year=2020&amp;pages=35&amp;pmid=32051000&amp;doi=10.1186/s13059-020-1941-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0043">
<cite>

Hu, G.
, 
Feng, J.
, 
Xiang, X.
, 
Wang, J.
, 
Salojärvi, J.
, 
Liu, C.
, 
Wu, Z.
, 
Zhang, J.
, 
Liang, X.
, 
Jiang, Z.
, 
Liu, W.
, 
Ou, L.
, 
Li, J.
, 
Fan, G.
, 
Mai, Y.
, 
Chen, C.
, 
Zhang, X.
, 
Zheng, J.
, 
Zhang, Y.
, … 
Li, J.
 (2022). Two divergent haplotypes from a highly heterozygous lychee genome suggest independent domestication events for early and late‐maturing cultivars. Nature Genetics, 54, 73–83. 10.1038/s41588-021-00971-3

</cite> [<a href="https://doi.org/10.1038/s41588-021-00971-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8755541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34980919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Two%20divergent%20haplotypes%20from%20a%20highly%20heterozygous%20lychee%20genome%20suggest%20independent%20domestication%20events%20for%20early%20and%20late%E2%80%90maturing%20cultivars&amp;author=G.%20Hu&amp;author=J.%20Feng&amp;author=X.%20Xiang&amp;author=J.%20Wang&amp;author=J.%20Saloj%C3%A4rvi&amp;volume=54&amp;publication_year=2022&amp;pages=73-83&amp;pmid=34980919&amp;doi=10.1038/s41588-021-00971-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0044">
<cite>

Jain, C.
, 
Rhie, A.
, 
Hansen, N. F.
, 
Koren, S.
, &amp; 
Phillippy, A. M.
 (2022). Long‐read mapping to repetitive reference sequences using Winnowmap2. Nature Methods, 19, 705–710. 10.1038/s41592-022-01457-8

</cite> [<a href="https://doi.org/10.1038/s41592-022-01457-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10510034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35365778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Long%E2%80%90read%20mapping%20to%20repetitive%20reference%20sequences%20using%20Winnowmap2&amp;author=C.%20Jain&amp;author=A.%20Rhie&amp;author=N.%20F.%20Hansen&amp;author=S.%20Koren&amp;author=A.%20M.%20Phillippy&amp;volume=19&amp;publication_year=2022&amp;pages=705-710&amp;pmid=35365778&amp;doi=10.1038/s41592-022-01457-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0045">
<cite>

Jayakodi, M.
, 
Padmarasu, S.
, 
Haberer, G.
, 
Bonthala, V. S.
, 
Gundlach, H.
, 
Monat, C.
, 
Lux, T.
, 
Kamal, N.
, 
Lang, D.
, 
Himmelbach, A.
, 
Ens, J.
, 
Zhang, X.‐Q.
, 
Angessa, T. T.
, 
Zhou, G.
, 
Tan, C.
, 
Hill, C.
, 
Wang, P.
, 
Schreiber, M.
, 
Boston, L. B.
, … 
Stein, N.
 (2020). The barley pan‐genome reveals the hidden legacy of mutation breeding. Nature, 588, 284–289. 10.1038/s41586-020-2947-8

</cite> [<a href="https://doi.org/10.1038/s41586-020-2947-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7759462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33239781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20barley%20pan%E2%80%90genome%20reveals%20the%20hidden%20legacy%20of%20mutation%20breeding&amp;author=M.%20Jayakodi&amp;author=S.%20Padmarasu&amp;author=G.%20Haberer&amp;author=V.%20S.%20Bonthala&amp;author=H.%20Gundlach&amp;volume=588&amp;publication_year=2020&amp;pages=284-289&amp;pmid=33239781&amp;doi=10.1038/s41586-020-2947-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0046">
<cite>

Jones, P.
, 
Binns, D.
, 
Chang, H.‐Y.
, 
Fraser, M.
, 
Li, W.
, 
Mcanulla, C.
, 
Mcwilliam, H.
, 
Maslen, J.
, 
Mitchell, A.
, 
Nuka, G.
, 
Pesseat, S.
, 
Quinn, A. F.
, 
Sangrador‐Vegas, A.
, 
Scheremetjew, M.
, 
Yong, S.‐Y.
, 
Lopez, R.
, &amp; 
Hunter, S.
 (2014). InterProScan 5: Genome‐scale protein function classification. Bioinformatics, 30, 1236–1240. 10.1093/bioinformatics/btu031

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=InterProScan%205:%20Genome%E2%80%90scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.%E2%80%90Y.%20Chang&amp;author=M.%20Fraser&amp;author=W.%20Li&amp;volume=30&amp;publication_year=2014&amp;pages=1236-1240&amp;pmid=24451626&amp;doi=10.1093/bioinformatics/btu031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0047">
<cite>

Kiełbasa, S. M.
, 
Wan, R.
, 
Sato, K.
, 
Horton, P.
, &amp; 
Frith, M. C.
 (2011). Adaptive seeds tame genomic sequence comparison. Genome Research, 21, 487–493. 10.1101/gr.113985.110

</cite> [<a href="https://doi.org/10.1101/gr.113985.110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3044862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21209072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Adaptive%20seeds%20tame%20genomic%20sequence%20comparison&amp;author=S.%20M.%20Kie%C5%82basa&amp;author=R.%20Wan&amp;author=K.%20Sato&amp;author=P.%20Horton&amp;author=M.%20C.%20Frith&amp;volume=21&amp;publication_year=2011&amp;pages=487-493&amp;pmid=21209072&amp;doi=10.1101/gr.113985.110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0048">
<cite>

Kim, D.
, 
Langmead, B.
, &amp; 
Salzberg, S. L.
 (2015). HISAT: A fast spliced aligner with low memory requirements. Nature Methods, 12, 357–360. 10.1038/nmeth.3317

</cite> [<a href="https://doi.org/10.1038/nmeth.3317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4655817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=HISAT:%20A%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&amp;author=D.%20Kim&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;volume=12&amp;publication_year=2015&amp;pages=357-360&amp;pmid=25751142&amp;doi=10.1038/nmeth.3317&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0049">
<cite>

Knorst, V.
, 
Yates, S.
, 
Byrne, S.
, 
Asp, T.
, 
Widmer, F.
, 
Studer, B.
, &amp; 
Kölliker, R.
 (2019). First assembly of the gene‐space of <em>Lolium multiflorum</em> and comparison to other Poaceae genomes. Grassland Science, 65, 125–134. 10.1111/grs.12225
</cite> [<a href="https://doi.org/10.1111/grs.12225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Grassland%20Science&amp;title=First%20assembly%20of%20the%20gene%E2%80%90space%20of%20Lolium%20multiflorum%20and%20comparison%20to%20other%20Poaceae%20genomes&amp;author=V.%20Knorst&amp;author=S.%20Yates&amp;author=S.%20Byrne&amp;author=T.%20Asp&amp;author=F.%20Widmer&amp;volume=65&amp;publication_year=2019&amp;pages=125-134&amp;doi=10.1111/grs.12225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0050">
<cite>

Kolmogorov, M.
, 
Yuan, J.
, 
Lin, Y.
, &amp; 
Pevzner, P. A.
 (2019). Assembly of long, error‐prone reads using repeat graphs. Nature Biotechnology, 37, 540–546. 10.1038/s41587-019-0072-8
</cite> [<a href="https://doi.org/10.1038/s41587-019-0072-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30936562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Assembly%20of%20long,%20error%E2%80%90prone%20reads%20using%20repeat%20graphs&amp;author=M.%20Kolmogorov&amp;author=J.%20Yuan&amp;author=Y.%20Lin&amp;author=P.%20A.%20Pevzner&amp;volume=37&amp;publication_year=2019&amp;pages=540-546&amp;pmid=30936562&amp;doi=10.1038/s41587-019-0072-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0051">
<cite>

Koren, S.
, 
Rhie, A.
, 
Walenz, B. P.
, 
Dilthey, A. T.
, 
Bickhart, D. M.
, 
Kingan, S. B.
, 
Hiendleder, S.
, 
Williams, J. L.
, 
Smith, T. P. L.
, &amp; 
Phillippy, A. M.
 (2018). De novo assembly of haplotype‐resolved genomes with trio binning. Nature Biotechnology, 36, 1174–1182. 10.1038/nbt.4277
</cite> [<a href="https://doi.org/10.1038/nbt.4277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6476705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30346939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=De%20novo%20assembly%20of%20haplotype%E2%80%90resolved%20genomes%20with%20trio%20binning&amp;author=S.%20Koren&amp;author=A.%20Rhie&amp;author=B.%20P.%20Walenz&amp;author=A.%20T.%20Dilthey&amp;author=D.%20M.%20Bickhart&amp;volume=36&amp;publication_year=2018&amp;pages=1174-1182&amp;pmid=30346939&amp;doi=10.1038/nbt.4277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0052">
<cite>

Koren, S.
, 
Walenz, B. P.
, 
Berlin, K.
, 
Miller, J. R.
, 
Bergman, N. H.
, &amp; 
Phillippy, A. M.
 (2017). Canu: Scalable and accurate long‐read assembly via adaptive <em>k</em> ‐mer weighting and repeat separation. Genome Research, 27, 722–736. 10.1101/gr.215087.116

</cite> [<a href="https://doi.org/10.1101/gr.215087.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28298431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Canu:%20Scalable%20and%20accurate%20long%E2%80%90read%20assembly%20via%20adaptive%20k%20%E2%80%90mer%20weighting%20and%20repeat%20separation&amp;author=S.%20Koren&amp;author=B.%20P.%20Walenz&amp;author=K.%20Berlin&amp;author=J.%20R.%20Miller&amp;author=N.%20H.%20Bergman&amp;volume=27&amp;publication_year=2017&amp;pages=722-736&amp;pmid=28298431&amp;doi=10.1101/gr.215087.116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0053">
<cite>

Li, H.
 (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA‐MEM. arXiv. 10.48550/arXiv.1303.3997
</cite> [<a href="https://doi.org/10.48550/arXiv.1303.3997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270079-bib-0054">
<cite>

Li, H.
 (2018). Minimap2: Pairwise alignment for nucleotide sequences. Bioinformatics, 34, 3094–3100. 10.1093/bioinformatics/bty191

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Minimap2:%20Pairwise%20alignment%20for%20nucleotide%20sequences&amp;author=H.%20Li&amp;volume=34&amp;publication_year=2018&amp;pages=3094-3100&amp;pmid=29750242&amp;doi=10.1093/bioinformatics/bty191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0055">
<cite>

Li, H.
, &amp; 
Durbin, R.
 (2024). Genome assembly in the telomere‐to‐telomere era. Nature Reviews Genetics, 25, 658–670. 10.1038/s41576-024-00718-w
</cite> [<a href="https://doi.org/10.1038/s41576-024-00718-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38649458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Genome%20assembly%20in%20the%20telomere%E2%80%90to%E2%80%90telomere%20era&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;volume=25&amp;publication_year=2024&amp;pages=658-670&amp;pmid=38649458&amp;doi=10.1038/s41576-024-00718-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0056">
<cite>

Liao, W.‐W.
, 
Asri, M.
, 
Ebler, J.
, 
Doerr, D.
, 
Haukness, M.
, 
Hickey, G.
, 
Lu, S.
, 
Lucas, J. K.
, 
Monlong, J.
, 
Abel, H. J.
, 
Buonaiuto, S.
, 
Chang, X. H.
, 
Cheng, H.
, 
Chu, J.
, 
Colonna, V.
, 
Eizenga, J. M.
, 
Feng, X.
, 
Fischer, C.
, 
Fulton, R. S.
, … 
Paten, B.
 (2023). A draft human pangenome reference. Nature, 617, 312–324. 10.1038/s41586-023-05896-x

</cite> [<a href="https://doi.org/10.1038/s41586-023-05896-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10172123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37165242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20draft%20human%20pangenome%20reference&amp;author=W.%E2%80%90W.%20Liao&amp;author=M.%20Asri&amp;author=J.%20Ebler&amp;author=D.%20Doerr&amp;author=M.%20Haukness&amp;volume=617&amp;publication_year=2023&amp;pages=312-324&amp;pmid=37165242&amp;doi=10.1038/s41586-023-05896-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0057">
<cite>

Liu, Y.
, 
Du, H.
, 
Li, P.
, 
Shen, Y.
, 
Peng, H.
, 
Liu, S.
, 
Zhou, G.‐A.
, 
Zhang, H.
, 
Liu, Z.
, 
Shi, M.
, 
Huang, X.
, 
Li, Y.
, 
Zhang, M.
, 
Wang, Z.
, 
Zhu, B.
, 
Han, B.
, 
Liang, C.
, &amp; 
Tian, Z.
 (2020). Pan‐genome of wild and cultivated soybeans. Cell, 182, 162–176. E13. 10.1016/j.cell.2020.05.023

</cite> [<a href="https://doi.org/10.1016/j.cell.2020.05.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32553274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Pan%E2%80%90genome%20of%20wild%20and%20cultivated%20soybeans&amp;author=Y.%20Liu&amp;author=H.%20Du&amp;author=P.%20Li&amp;author=Y.%20Shen&amp;author=H.%20Peng&amp;volume=182&amp;publication_year=2020&amp;pages=162-176&amp;pmid=32553274&amp;doi=10.1016/j.cell.2020.05.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0058">
<cite>

Llamas, B.
, 
Narzisi, G.
, 
Schneider, V.
, 
Audano, P. A.
, 
Biederstedt, E.
, 
Blauvelt, L.
, 
Bradbury, P.
, 
Chang, X.
, 
Chin, C.‐S.
, 
Fungtammasan, A.
, 
Clarke, W. E.
, 
Cleary, A.
, 
Ebler, J.
, 
Eizenga, J.
, 
Sibbesen, J. A.
, 
Markello, C. J.
, 
Garrison, E.
, 
Garg, S.
, 
Hickey, G.
, … 
Busby, B.
 (2021). A strategy for building and using a human reference pangenome. F1000Research, 8, 1751. 10.12688/f1000research.19630.2
</cite> [<a href="https://doi.org/10.12688/f1000research.19630.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8350888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34386196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=A%20strategy%20for%20building%20and%20using%20a%20human%20reference%20pangenome&amp;author=B.%20Llamas&amp;author=G.%20Narzisi&amp;author=V.%20Schneider&amp;author=P.%20A.%20Audano&amp;author=E.%20Biederstedt&amp;volume=8&amp;publication_year=2021&amp;pages=1751&amp;pmid=34386196&amp;doi=10.12688/f1000research.19630.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0059">
<cite>

Mak, Q. X. C.
, 
Wick, R. R.
, 
Holt, J. M.
, &amp; 
Wang, J. R.
 (2023). Polishing de novo nanopore assemblies of bacteria and eukaryotes With FMLRC2. Molecular Biology and Evolution, 40, msad048. 10.1093/molbev/msad048

</cite> [<a href="https://doi.org/10.1093/molbev/msad048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10015616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36869750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Polishing%20de%20novo%20nanopore%20assemblies%20of%20bacteria%20and%20eukaryotes%20With%20FMLRC2&amp;author=Q.%20X.%20C.%20Mak&amp;author=R.%20R.%20Wick&amp;author=J.%20M.%20Holt&amp;author=J.%20R.%20Wang&amp;volume=40&amp;publication_year=2023&amp;pages=msad048&amp;pmid=36869750&amp;doi=10.1093/molbev/msad048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0060">
<cite>

Manni, M.
, 
Berkeley, M. R.
, 
Seppey, M.
, &amp; 
Zdobnov, E. M.
 (2021). BUSCO: Assessing genomic data quality and beyond. Current Protocols, 1, e323. 10.1002/cpz1.323

</cite> [<a href="https://doi.org/10.1002/cpz1.323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34936221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Protocols&amp;title=BUSCO:%20Assessing%20genomic%20data%20quality%20and%20beyond&amp;author=M.%20Manni&amp;author=M.%20R.%20Berkeley&amp;author=M.%20Seppey&amp;author=E.%20M.%20Zdobnov&amp;volume=1&amp;publication_year=2021&amp;pages=e323&amp;pmid=34936221&amp;doi=10.1002/cpz1.323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0061">
<cite>

Mapleson, D.
, 
Garcia Accinelli, G.
, 
Kettleborough, G.
, 
Wright, J.
, &amp; 
Clavijo, B. J.
 (2017). KAT: A K‐mer analysis toolkit to quality control NGS datasets and genome assemblies. Bioinformatics, 33, 574–576. 10.1093/bioinformatics/btw663

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5408915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27797770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=KAT:%20A%20K%E2%80%90mer%20analysis%20toolkit%20to%20quality%20control%20NGS%20datasets%20and%20genome%20assemblies&amp;author=D.%20Mapleson&amp;author=G.%20Garcia%20Accinelli&amp;author=G.%20Kettleborough&amp;author=J.%20Wright&amp;author=B.%20J.%20Clavijo&amp;volume=33&amp;publication_year=2017&amp;pages=574-576&amp;pmid=27797770&amp;doi=10.1093/bioinformatics/btw663&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0062">
<cite>

Martin, M.
, 
Patterson, M.
, 
Garg, S.
, 
Fischer, S. O.
, 
Pisanti, N.
, 
Klau, G. W.
, 
Schöenhuth, A.
, &amp; 
Marschall, T.
 (2016). WhatsHap: Fast and accurate read‐based phasing. BioRxiv. 10.1101/085050
</cite> [<a href="https://doi.org/10.1101/085050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270079-bib-0063">
<cite>

Mascher, M.
, 
Wicker, T.
, 
Jenkins, J.
, 
Plott, C.
, 
Lux, T.
, 
Koh, C. S.
, 
Ens, J.
, 
Gundlach, H.
, 
Boston, L. B.
, 
Tulpová, Z.
, 
Holden, S.
, 
Hernández‐Pinzón, I.
, 
Scholz, U.
, 
Mayer, K. F. X.
, 
Spannagl, M.
, 
Pozniak, C. J.
, 
Sharpe, A. G.
, 
Šimková, H.
, 
Moscou, M. J.
, … 
Stein, N.
 (2021). Long‐read sequence assembly: A technical evaluation in barley. Plant Cell, 33, 1888–1906. 10.1093/plcell/koab077

</cite> [<a href="https://doi.org/10.1093/plcell/koab077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8290290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33710295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Long%E2%80%90read%20sequence%20assembly:%20A%20technical%20evaluation%20in%20barley&amp;author=M.%20Mascher&amp;author=T.%20Wicker&amp;author=J.%20Jenkins&amp;author=C.%20Plott&amp;author=T.%20Lux&amp;volume=33&amp;publication_year=2021&amp;pages=1888-1906&amp;pmid=33710295&amp;doi=10.1093/plcell/koab077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0064">
<cite>

Mistry, J.
, 
Chuguransky, S.
, 
Williams, L.
, 
Qureshi, M.
, 
Salazar, G. A.
, 
Sonnhammer, E. L. L.
, 
Tosatto, S. C. E.
, 
Paladin, L.
, 
Raj, S.
, 
Richardson, L. J.
, 
Finn, R. D.
, &amp; 
Bateman, A.
 (2021). Pfam: The protein families database in 2021. Nucleic Acids Research, 49, D412–D419. 10.1093/nar/gkaa913

</cite> [<a href="https://doi.org/10.1093/nar/gkaa913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33125078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Pfam:%20The%20protein%20families%20database%20in%202021&amp;author=J.%20Mistry&amp;author=S.%20Chuguransky&amp;author=L.%20Williams&amp;author=M.%20Qureshi&amp;author=G.%20A.%20Salazar&amp;volume=49&amp;publication_year=2021&amp;pages=D412-D419&amp;pmid=33125078&amp;doi=10.1093/nar/gkaa913&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0065">
<cite>

Monat, C.
, 
Padmarasu, S.
, 
Lux, T.
, 
Wicker, T.
, 
Gundlach, H.
, 
Himmelbach, A.
, 
Ens, J.
, 
Li, C.
, 
Muehlbauer, G. J.
, 
Schulman, A. H.
, 
Waugh, R.
, 
Braumann, I.
, 
Pozniak, C.
, 
Scholz, U.
, 
Mayer, K. F. X.
, 
Spannagl, M.
, 
Stein, N.
, &amp; 
Mascher, M.
 (2019). TRITEX: Chromosome‐scale sequence assembly of Triticeae genomes with open‐source tools. Genome Biology, 20, Article 284. 10.1186/s13059-019-1899-5

</cite> [<a href="https://doi.org/10.1186/s13059-019-1899-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6918601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31849336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=TRITEX:%20Chromosome%E2%80%90scale%20sequence%20assembly%20of%20Triticeae%20genomes%20with%20open%E2%80%90source%20tools&amp;author=C.%20Monat&amp;author=S.%20Padmarasu&amp;author=T.%20Lux&amp;author=T.%20Wicker&amp;author=H.%20Gundlach&amp;volume=20&amp;publication_year=2019&amp;pages=284&amp;pmid=31849336&amp;doi=10.1186/s13059-019-1899-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0066">
<cite>

Money, D.
, 
Migicovsky, Z.
, 
Gardner, K.
, &amp; 
Myles, S.
 (2017). LinkImputeR: User‐guided genotype calling and imputation for non‐model organisms. BMC Genomics, 18, Article 523. 10.1186/s12864-017-3873-5

</cite> [<a href="https://doi.org/10.1186/s12864-017-3873-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5504746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28693460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=LinkImputeR:%20User%E2%80%90guided%20genotype%20calling%20and%20imputation%20for%20non%E2%80%90model%20organisms&amp;author=D.%20Money&amp;author=Z.%20Migicovsky&amp;author=K.%20Gardner&amp;author=S.%20Myles&amp;volume=18&amp;publication_year=2017&amp;pages=523&amp;pmid=28693460&amp;doi=10.1186/s12864-017-3873-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0067">
<cite>

Nagy, I.
, 
Veeckman, E.
, 
Liu, C.
, 
Bel, M. V.
, 
Vandepoele, K.
, 
Jensen, C. S.
, 
Ruttink, T.
, &amp; 
Asp, T.
 (2022). Chromosome‐scale assembly and annotation of the perennial ryegrass genome. BMC Genomics, 23, Article 505. 10.1186/s12864-022-08697-0

</cite> [<a href="https://doi.org/10.1186/s12864-022-08697-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9281035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35831814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Chromosome%E2%80%90scale%20assembly%20and%20annotation%20of%20the%20perennial%20ryegrass%20genome&amp;author=I.%20Nagy&amp;author=E.%20Veeckman&amp;author=C.%20Liu&amp;author=M.%20V.%20Bel&amp;author=K.%20Vandepoele&amp;volume=23&amp;publication_year=2022&amp;pages=505&amp;pmid=35831814&amp;doi=10.1186/s12864-022-08697-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0068">
<cite>
Najoshi
. (2022). 
Sabre—A barcode demultiplexing and trimming tool for FastQ files
. GitHub, Inc. <a href="https://github.com/najoshi/sabre" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/najoshi/sabre</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Najoshi%20.%20(2022).%20Sabre%E2%80%94A%20barcode%20demultiplexing%20and%20trimming%20tool%20for%20FastQ%20files%20.%20GitHub,%20Inc.%20https://github.com/najoshi/sabre" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0069">
<cite>

Paril, J.
, 
Pandey, G.
, 
Barnett, E. M.
, 
Rane, R. V.
, 
Court, L.
, 
Walsh, T.
, &amp; 
Fournier‐Level, A.
 (2022). Rounding up the annual ryegrass genome: High‐quality reference genome of <em>Lolium rigidum</em>
. Frontiers in Genetics, 13, 1012694. 10.3389/fgene.2022.1012694

</cite> [<a href="https://doi.org/10.3389/fgene.2022.1012694" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9664059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36386808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Rounding%20up%20the%20annual%20ryegrass%20genome:%20High%E2%80%90quality%20reference%20genome%20of%20Lolium%20rigidum&amp;author=J.%20Paril&amp;author=G.%20Pandey&amp;author=E.%20M.%20Barnett&amp;author=R.%20V.%20Rane&amp;author=L.%20Court&amp;volume=13&amp;publication_year=2022&amp;pages=1012694&amp;pmid=36386808&amp;doi=10.3389/fgene.2022.1012694&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0070">
<cite>

Pedersen, B. S.
, &amp; 
Quinlan, A. R.
 (2018). Mosdepth: Quick coverage calculation for genomes and exomes. Bioinformatics, 34, 867–868. 10.1093/bioinformatics/btx699

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6030888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29096012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Mosdepth:%20Quick%20coverage%20calculation%20for%20genomes%20and%20exomes&amp;author=B.%20S.%20Pedersen&amp;author=A.%20R.%20Quinlan&amp;volume=34&amp;publication_year=2018&amp;pages=867-868&amp;pmid=29096012&amp;doi=10.1093/bioinformatics/btx699&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0071">
<cite>

Pertea, G.
, &amp; 
Pertea, M.
 (2020). GFF Utilities: GffRead and GffCompare. F1000Research, 9, 304. 10.12688/f1000research.23297.1
</cite> [<a href="https://doi.org/10.12688/f1000research.23297.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7222033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32489650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=GFF%20Utilities:%20GffRead%20and%20GffCompare&amp;author=G.%20Pertea&amp;author=M.%20Pertea&amp;volume=9&amp;publication_year=2020&amp;pages=304&amp;pmid=32489650&amp;doi=10.12688/f1000research.23297.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0072">
<cite>

Pertea, M.
, 
Kim, D.
, 
Pertea, G. M.
, 
Leek, J. T.
, &amp; 
Salzberg, S. L.
 (2016). Transcript‐level expression analysis of RNA‐seq experiments with HISAT, StringTie and Ballgown. Nature Protocols, 11, 1650–1667. 10.1038/nprot.2016.095

</cite> [<a href="https://doi.org/10.1038/nprot.2016.095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5032908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27560171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Transcript%E2%80%90level%20expression%20analysis%20of%20RNA%E2%80%90seq%20experiments%20with%20HISAT,%20StringTie%20and%20Ballgown&amp;author=M.%20Pertea&amp;author=D.%20Kim&amp;author=G.%20M.%20Pertea&amp;author=J.%20T.%20Leek&amp;author=S.%20L.%20Salzberg&amp;volume=11&amp;publication_year=2016&amp;pages=1650-1667&amp;pmid=27560171&amp;doi=10.1038/nprot.2016.095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0073">
<cite>

Pertea, M.
, 
Pertea, G. M.
, 
Antonescu, C. M.
, 
Chang, T.‐C.
, 
Mendell, J. T.
, &amp; 
Salzberg, S. L.
 (2015). StringTie enables improved reconstruction of a transcriptome from RNA‐seq reads. Nature Biotechnology, 33, 290–295. 10.1038/nbt.3122
</cite> [<a href="https://doi.org/10.1038/nbt.3122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4643835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA%E2%80%90seq%20reads&amp;author=M.%20Pertea&amp;author=G.%20M.%20Pertea&amp;author=C.%20M.%20Antonescu&amp;author=T.%E2%80%90C.%20Chang&amp;author=J.%20T.%20Mendell&amp;volume=33&amp;publication_year=2015&amp;pages=290-295&amp;pmid=25690850&amp;doi=10.1038/nbt.3122&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0074">
<cite>

Pfeifer, M.
, 
Martis, M.
, 
Asp, T.
, 
Mayer, K. F. X.
, 
Lübberstedt, T.
, 
Byrne, S.
, 
Frei, U.
, &amp; 
Studer, B.
 (2013). The perennial ryegrass genomezipper: Targeted use of genome resources for comparative grass genomics. Plant Physiology, 161, 571–582. 10.1104/pp.112.207282

</cite> [<a href="https://doi.org/10.1104/pp.112.207282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3561004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23184232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20perennial%20ryegrass%20genomezipper:%20Targeted%20use%20of%20genome%20resources%20for%20comparative%20grass%20genomics&amp;author=M.%20Pfeifer&amp;author=M.%20Martis&amp;author=T.%20Asp&amp;author=K.%20F.%20X.%20Mayer&amp;author=T.%20L%C3%BCbberstedt&amp;volume=161&amp;publication_year=2013&amp;pages=571-582&amp;pmid=23184232&amp;doi=10.1104/pp.112.207282&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0075">
<cite>

Porubsky, D.
, 
Ebert, P.
, 
Audano, P. A.
, 
Vollger, M. R.
, 
Harvey, W. T.
, 
Marijon, P.
, 
Ebler, J.
, 
Munson, K. M.
, 
Sorensen, M.
, 
Sulovari, A.
, 
Haukness, M.
, 
Ghareghani, M.
, 
Lansdorp, P. M.
, 
Paten, B.
, 
Devine, S. E.
, 
Sanders, A. D.
, 
Lee, C.
, 
Chaisson, M. J. P.
, 
Korbel, J. O.
, … 
Marschall, T.
 (2021). Fully phased human genome assembly without parental data using single‐cell strand sequencing and long reads. Nature Biotechnology, 39, 302–308. 10.1038/s41587-020-0719-5
</cite> [<a href="https://doi.org/10.1038/s41587-020-0719-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7954704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33288906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Fully%20phased%20human%20genome%20assembly%20without%20parental%20data%20using%20single%E2%80%90cell%20strand%20sequencing%20and%20long%20reads&amp;author=D.%20Porubsky&amp;author=P.%20Ebert&amp;author=P.%20A.%20Audano&amp;author=M.%20R.%20Vollger&amp;author=W.%20T.%20Harvey&amp;volume=39&amp;publication_year=2021&amp;pages=302-308&amp;pmid=33288906&amp;doi=10.1038/s41587-020-0719-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0103"><cite>
R Core Team
. (2023). 
R: A language and environment for statistical computing
. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a>
</cite></li>
<li id="tpg270079-bib-0076">
<cite>

Rastas, P.
 (2017). Lep‐MAP3: Robust linkage mapping even for low‐coverage whole genome sequencing data. Bioinformatics, 33, 3726–3732. 10.1093/bioinformatics/btx494

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29036272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Lep%E2%80%90MAP3:%20Robust%20linkage%20mapping%20even%20for%20low%E2%80%90coverage%20whole%20genome%20sequencing%20data&amp;author=P.%20Rastas&amp;volume=33&amp;publication_year=2017&amp;pages=3726-3732&amp;pmid=29036272&amp;doi=10.1093/bioinformatics/btx494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0077">
<cite>

Rautiainen, M.
, 
Nurk, S.
, 
Walenz, B. P.
, 
Logsdon, G. A.
, 
Porubsky, D.
, 
Rhie, A.
, 
Eichler, E. E.
, 
Phillippy, A. M.
, &amp; 
Koren, S.
 (2023). Telomere‐to‐telomere assembly of diploid chromosomes with Verkko. Nature Biotechnology, 41, 1474–1482. 10.1038/s41587-023-01662-6
</cite> [<a href="https://doi.org/10.1038/s41587-023-01662-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10427740/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36797493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Telomere%E2%80%90to%E2%80%90telomere%20assembly%20of%20diploid%20chromosomes%20with%20Verkko&amp;author=M.%20Rautiainen&amp;author=S.%20Nurk&amp;author=B.%20P.%20Walenz&amp;author=G.%20A.%20Logsdon&amp;author=D.%20Porubsky&amp;volume=41&amp;publication_year=2023&amp;pages=1474-1482&amp;pmid=36797493&amp;doi=10.1038/s41587-023-01662-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0078">
<cite>

Roach, M. J.
, 
Schmidt, S. A.
, &amp; 
Borneman, A. R.
 (2018). Purge Haplotigs: Allelic contig reassignment for third‐gen diploid genome assemblies. BMC Bioinformatics, 19, Article 460. 10.1186/s12859-018-2485-7

</cite> [<a href="https://doi.org/10.1186/s12859-018-2485-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6267036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30497373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Purge%20Haplotigs:%20Allelic%20contig%20reassignment%20for%20third%E2%80%90gen%20diploid%20genome%20assemblies&amp;author=M.%20J.%20Roach&amp;author=S.%20A.%20Schmidt&amp;author=A.%20R.%20Borneman&amp;volume=19&amp;publication_year=2018&amp;pages=460&amp;pmid=30497373&amp;doi=10.1186/s12859-018-2485-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0079">
<cite>

Schlagenhauf, E.
, &amp; 
Wicker, T.
 (2016). 
The TREP platform: A curated database of transposable elements
. University of Zurich. <a href="https://trep-db.uzh.ch" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://trep‐db.uzh.ch</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Schlagenhauf,%20E.%20,%20&amp;%20Wicker,%20T.%20(2016).%20The%20TREP%20platform:%20A%20curated%20database%20of%20transposable%20elements%20.%20University%20of%20Zurich.%20https://trep%E2%80%90db.uzh.ch" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0080">
<cite>

Schubiger, F. X.
, &amp; 
Boller, B.
 (2016). Virulence of crown rust isolates (<em>Puccinia coronata</em> f.sp. <em>lolii</em>) on genotypes of Italian and perennial ryegrass (<em>Lolium multiflorum</em> and <em>L. perenne</em>). European Journal of Plant Pathology, 144, 141–154. 10.1007/s10658-015-0758-9
</cite> [<a href="https://doi.org/10.1007/s10658-015-0758-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Plant%20Pathology&amp;title=Virulence%20of%20crown%20rust%20isolates%20(Puccinia%20coronata%20f.sp.%20lolii)%20on%20genotypes%20of%20Italian%20and%20perennial%20ryegrass%20(Lolium%20multiflorum%20and%20L.%20perenne)&amp;author=F.%20X.%20Schubiger&amp;author=B.%20Boller&amp;volume=144&amp;publication_year=2016&amp;pages=141-154&amp;doi=10.1007/s10658-015-0758-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0081">
<cite>

Searle, S. R.
, 
Speed, F. M.
, &amp; 
Milliken, G. A.
 (1980). Population marginal means in the linear model: An alternative to least squares means. The American Statistician, 34, 216–221. 10.1080/00031305.1980.10483031
</cite> [<a href="https://doi.org/10.1080/00031305.1980.10483031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20American%20Statistician&amp;title=Population%20marginal%20means%20in%20the%20linear%20model:%20An%20alternative%20to%20least%20squares%20means&amp;author=S.%20R.%20Searle&amp;author=F.%20M.%20Speed&amp;author=G.%20A.%20Milliken&amp;volume=34&amp;publication_year=1980&amp;pages=216-221&amp;doi=10.1080/00031305.1980.10483031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0082">
<cite>

Sedlazeck, F. J.
, 
Rescheneder, P.
, 
Smolka, M.
, 
Fang, H.
, 
Nattestad, M.
, 
von Haeseler, A.
, &amp; 
Schatz, M. C.
 (2018). Accurate detection of complex structural variations using single‐molecule sequencing. Nature Methods, 15, 461–468. 10.1038/s41592-018-0001-7

</cite> [<a href="https://doi.org/10.1038/s41592-018-0001-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5990442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29713083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Accurate%20detection%20of%20complex%20structural%20variations%20using%20single%E2%80%90molecule%20sequencing&amp;author=F.%20J.%20Sedlazeck&amp;author=P.%20Rescheneder&amp;author=M.%20Smolka&amp;author=H.%20Fang&amp;author=M.%20Nattestad&amp;volume=15&amp;publication_year=2018&amp;pages=461-468&amp;pmid=29713083&amp;doi=10.1038/s41592-018-0001-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0083">
<cite>

Serra Mari, R.
, 
Schrinner, S.
, 
Finkers, R.
, 
Ziegler, F. M. R.
, 
Arens, P.
, 
Schmidt, M. H.‐W.
, 
Usadel, B.
, 
Klau, G. W.
, &amp; 
Marschall, T.
 (2024). Haplotype‐resolved assembly of a tetraploid potato genome using long reads and low‐depth offspring data. Genome Biology, 25, Article 26. 10.1186/s13059-023-03160-z

</cite> [<a href="https://doi.org/10.1186/s13059-023-03160-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10797741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38243222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Haplotype%E2%80%90resolved%20assembly%20of%20a%20tetraploid%20potato%20genome%20using%20long%20reads%20and%20low%E2%80%90depth%20offspring%20data&amp;author=R.%20Serra%20Mari&amp;author=S.%20Schrinner&amp;author=R.%20Finkers&amp;author=F.%20M.%20R.%20Ziegler&amp;author=P.%20Arens&amp;volume=25&amp;publication_year=2024&amp;pages=26&amp;pmid=38243222&amp;doi=10.1186/s13059-023-03160-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0084">
<cite>

Shang, L.
, 
Li, X.
, 
He, H.
, 
Yuan, Q.
, 
Song, Y.
, 
Wei, Z.
, 
Lin, H.
, 
Hu, M.
, 
Zhao, F.
, 
Zhang, C.
, 
Li, Y.
, 
Gao, H.
, 
Wang, T.
, 
Liu, X.
, 
Zhang, H.
, 
Zhang, Y.
, 
Cao, S.
, 
Yu, X.
, 
Zhang, B.
, … 
Qian, Q.
 (2022). A super pan‐genomic landscape of rice. Cell Research, 32, 878–896. 10.1038/s41422-022-00685-z

</cite> [<a href="https://doi.org/10.1038/s41422-022-00685-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9525306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35821092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Research&amp;title=A%20super%20pan%E2%80%90genomic%20landscape%20of%20rice&amp;author=L.%20Shang&amp;author=X.%20Li&amp;author=H.%20He&amp;author=Q.%20Yuan&amp;author=Y.%20Song&amp;volume=32&amp;publication_year=2022&amp;pages=878-896&amp;pmid=35821092&amp;doi=10.1038/s41422-022-00685-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0085">
<cite>

Sirén, J.
, 
Monlong, J.
, 
Chang, X.
, 
Novak, A. M.
, 
Eizenga, J. M.
, 
Markello, C.
, 
Sibbesen, J. A.
, 
Hickey, G.
, 
Chang, P.‐C.
, 
Carroll, A.
, 
Gupta, N.
, 
Gabriel, S.
, 
Blackwell, T. W.
, 
Ratan, A.
, 
Taylor, K. D.
, 
Rich, S. S.
, 
Rotter, J. I.
, 
Haussler, D.
, 
Garrison, E.
, &amp; 
Paten, B.
 (2021). Pangenomics enables genotyping of known structural variants in 5202 diverse genomes. Science, 374, abg8871. 10.1126/science.abg8871

</cite> [<a href="https://doi.org/10.1126/science.abg8871" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9365333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34914532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Pangenomics%20enables%20genotyping%20of%20known%20structural%20variants%20in%205202%20diverse%20genomes&amp;author=J.%20Sir%C3%A9n&amp;author=J.%20Monlong&amp;author=X.%20Chang&amp;author=A.%20M.%20Novak&amp;author=J.%20M.%20Eizenga&amp;volume=374&amp;publication_year=2021&amp;pages=abg8871&amp;pmid=34914532&amp;doi=10.1126/science.abg8871&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0086">
<cite>

Smit, A.
, 
Hubley, R.
, &amp; 
Green, P.
 (2013). 
RepeatMasker Open‐4.0. 2013–2015
. Institute for Systems Biology. <a href="http://www.repeatmasker.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.repeatmasker.org</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Smit,%20A.%20,%20Hubley,%20R.%20,%20&amp;%20Green,%20P.%20(2013).%20RepeatMasker%20Open%E2%80%904.0.%202013%E2%80%932015%20.%20Institute%20for%20Systems%20Biology.%20http://www.repeatmasker.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0087">
<cite>

Song, B.
, 
Marco‐Sola, S.
, 
Moreto, M.
, 
Johnson, L.
, 
Buckler, E. S.
, &amp; 
Stitzer, M. C.
 (2022). AnchorWave: Sensitive alignment of genomes with high sequence diversity, extensive structural polymorphism, and whole‐genome duplication. Proceedings of the National Academy of Sciences of the USA, 119, e2113075119. 10.1073/pnas.2113075119

</cite> [<a href="https://doi.org/10.1073/pnas.2113075119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8740769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34934012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20USA&amp;title=AnchorWave:%20Sensitive%20alignment%20of%20genomes%20with%20high%20sequence%20diversity,%20extensive%20structural%20polymorphism,%20and%20whole%E2%80%90genome%20duplication&amp;author=B.%20Song&amp;author=S.%20Marco%E2%80%90Sola&amp;author=M.%20Moreto&amp;author=L.%20Johnson&amp;author=E.%20S.%20Buckler&amp;volume=119&amp;publication_year=2022&amp;pages=e2113075119&amp;pmid=34934012&amp;doi=10.1073/pnas.2113075119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0088">
<cite>

Sun, H.
, 
Jiao, W.‐B.
, 
Krause, K.
, 
Campoy, J. A.
, 
Goel, M.
, 
Folz‐Donahue, K.
, 
Kukat, C.
, 
Huettel, B.
, &amp; 
Schneeberger, K.
 (2022). Chromosome‐scale and haplotype‐resolved genome assembly of a tetraploid potato cultivar. Nature Genetics, 54, 342–348. 10.1038/s41588-022-01015-0

</cite> [<a href="https://doi.org/10.1038/s41588-022-01015-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8920897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35241824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Chromosome%E2%80%90scale%20and%20haplotype%E2%80%90resolved%20genome%20assembly%20of%20a%20tetraploid%20potato%20cultivar&amp;author=H.%20Sun&amp;author=W.%E2%80%90B.%20Jiao&amp;author=K.%20Krause&amp;author=J.%20A.%20Campoy&amp;author=M.%20Goel&amp;volume=54&amp;publication_year=2022&amp;pages=342-348&amp;pmid=35241824&amp;doi=10.1038/s41588-022-01015-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0089">
<cite>

Tang, H.
, 
Zhang, X.
, 
Miao, C.
, 
Zhang, J.
, 
Ming, R.
, 
Schnable, J. C.
, 
Schnable, P. S.
, 
Lyons, E.
, &amp; 
Lu, J.
 (2015). ALLMAPS: Robust scaffold ordering based on multiple maps. Genome Biology, 16, Article 3. 10.1186/s13059-014-0573-1

</cite> [<a href="https://doi.org/10.1186/s13059-014-0573-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4305236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25583564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=ALLMAPS:%20Robust%20scaffold%20ordering%20based%20on%20multiple%20maps&amp;author=H.%20Tang&amp;author=X.%20Zhang&amp;author=C.%20Miao&amp;author=J.%20Zhang&amp;author=R.%20Ming&amp;volume=16&amp;publication_year=2015&amp;pages=3&amp;pmid=25583564&amp;doi=10.1186/s13059-014-0573-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0091">
<cite>

Trapnell, C.
, 
Roberts, A.
, 
Goff, L.
, 
Pertea, G.
, 
Kim, D.
, 
Kelley, D. R.
, 
Pimentel, H.
, 
Salzberg, S. L.
, 
Rinn, J. L.
, &amp; 
Pachter, L.
 (2012). Differential gene and transcript expression analysis of RNA‐seq experiments with TopHat and Cufflinks. Nature Protocols, 7, 562–578. 10.1038/nprot.2012.016

</cite> [<a href="https://doi.org/10.1038/nprot.2012.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3334321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22383036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA%E2%80%90seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;author=G.%20Pertea&amp;author=D.%20Kim&amp;volume=7&amp;publication_year=2012&amp;pages=562-578&amp;pmid=22383036&amp;doi=10.1038/nprot.2012.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0092">
<cite>

Vaughn, J. N.
, 
Branham, S. E.
, 
Abernathy, B.
, 
Hulse‐Kemp, A. M.
, 
Rivers, A. R.
, 
Levi, A.
, &amp; 
Wechter, W. P.
 (2022). Graph‐based pangenomics maximizes genotyping density and reveals structural impacts on fungal resistance in melon. Nature Communications, 13, Article 7897. 10.1038/s41467-022-35621-7
</cite> [<a href="https://doi.org/10.1038/s41467-022-35621-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9780226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36550124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Graph%E2%80%90based%20pangenomics%20maximizes%20genotyping%20density%20and%20reveals%20structural%20impacts%20on%20fungal%20resistance%20in%20melon&amp;author=J.%20N.%20Vaughn&amp;author=S.%20E.%20Branham&amp;author=B.%20Abernathy&amp;author=A.%20M.%20Hulse%E2%80%90Kemp&amp;author=A.%20R.%20Rivers&amp;volume=13&amp;publication_year=2022&amp;pages=7897&amp;pmid=36550124&amp;doi=10.1038/s41467-022-35621-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0093">
<cite>

Walkowiak, S.
, 
Gao, L.
, 
Monat, C.
, 
Haberer, G.
, 
Kassa, M. T.
, 
Brinton, J.
, 
Ramirez‐Gonzalez, R. H.
, 
Kolodziej, M. C.
, 
Delorean, E.
, 
Thambugala, D.
, 
Klymiuk, V.
, 
Byrns, B.
, 
Gundlach, H.
, 
Bandi, V.
, 
Siri, J. N.
, 
Nilsen, K.
, 
Aquino, C.
, 
Himmelbach, A.
, 
Copetti, D.
, … 
Pozniak, C. J.
 (2020). Multiple wheat genomes reveal global variation in modern breeding. Nature, 588, 277–283. 10.1038/s41586-020-2961-x

</cite> [<a href="https://doi.org/10.1038/s41586-020-2961-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7759465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33239791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Multiple%20wheat%20genomes%20reveal%20global%20variation%20in%20modern%20breeding&amp;author=S.%20Walkowiak&amp;author=L.%20Gao&amp;author=C.%20Monat&amp;author=G.%20Haberer&amp;author=M.%20T.%20Kassa&amp;volume=588&amp;publication_year=2020&amp;pages=277-283&amp;pmid=33239791&amp;doi=10.1038/s41586-020-2961-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0094">
<cite>

Wang, Y.
, 
Tang, H.
, 
Debarry, J. D.
, 
Tan, X.
, 
Li, J.
, 
Wang, X.
, 
Lee, T.‐H.
, 
Jin, H.
, 
Marler, B.
, 
Guo, H.
, 
Kissinger, J. C.
, &amp; 
Paterson, A. H.
 (2012). MCScanX: A toolkit for detection and evolutionary analysis of gene synteny and collinearity. Nucleic Acids Research, 40, e49–e49. 10.1093/nar/gkr1293

</cite> [<a href="https://doi.org/10.1093/nar/gkr1293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3326336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22217600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=MCScanX:%20A%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20Debarry&amp;author=X.%20Tan&amp;author=J.%20Li&amp;volume=40&amp;publication_year=2012&amp;pages=e49-e49&amp;pmid=22217600&amp;doi=10.1093/nar/gkr1293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0095">
<cite>

Wu, T. D.
, &amp; 
Watanabe, C. K.
 (2005). GMAP: A genomic mapping and alignment program for mRNA and EST sequences. Bioinformatics, 21, 1859–1875. 10.1093/bioinformatics/bti310

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15728110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GMAP:%20A%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.%20D.%20Wu&amp;author=C.%20K.%20Watanabe&amp;volume=21&amp;publication_year=2005&amp;pages=1859-1875&amp;pmid=15728110&amp;doi=10.1093/bioinformatics/bti310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0096">
<cite>

Yan, H.
, 
Sun, M.
, 
Zhang, Z.
, 
Jin, Y.
, 
Zhang, A.
, 
Lin, C.
, 
Wu, B.
, 
He, M.
, 
Xu, B.
, 
Wang, J.
, 
Qin, P.
, 
Mendieta, J. P.
, 
Nie, G.
, 
Wang, J.
, 
Jones, C. S.
, 
Feng, G.
, 
Srivastava, R. K.
, 
Zhang, X.
, 
Bombarely, A.
, … 
Huang, L.
 (2023). Pangenomic analysis identifies structural variation associated with heat tolerance in pearl millet. Nature Genetics, 55, 507–518. 10.1038/s41588-023-01302-4

</cite> [<a href="https://doi.org/10.1038/s41588-023-01302-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10011142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36864101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Pangenomic%20analysis%20identifies%20structural%20variation%20associated%20with%20heat%20tolerance%20in%20pearl%20millet&amp;author=H.%20Yan&amp;author=M.%20Sun&amp;author=Z.%20Zhang&amp;author=Y.%20Jin&amp;author=A.%20Zhang&amp;volume=55&amp;publication_year=2023&amp;pages=507-518&amp;pmid=36864101&amp;doi=10.1038/s41588-023-01302-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0097">
<cite>

Zeng, X.
, 
Yi, Z.
, 
Zhang, X.
, 
Du, Y.
, 
Li, Y.
, 
Zhou, Z.
, 
Chen, S.
, 
Zhao, H.
, 
Yang, S.
, 
Wang, Y.
, &amp; 
Chen, G.
 (2024). Chromosome‐level scaffolding of haplotype‐resolved assemblies using Hi‐C data without reference genomes. Nature Plants, 10, 1184–1200. 10.1038/s41477-024-01755-3

</cite> [<a href="https://doi.org/10.1038/s41477-024-01755-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39103456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Chromosome%E2%80%90level%20scaffolding%20of%20haplotype%E2%80%90resolved%20assemblies%20using%20Hi%E2%80%90C%20data%20without%20reference%20genomes&amp;author=X.%20Zeng&amp;author=Z.%20Yi&amp;author=X.%20Zhang&amp;author=Y.%20Du&amp;author=Y.%20Li&amp;volume=10&amp;publication_year=2024&amp;pages=1184-1200&amp;pmid=39103456&amp;doi=10.1038/s41477-024-01755-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0098">
<cite>

Zhang, X.
, 
Chen, S.
, 
Shi, L.
, 
Gong, D.
, 
Zhang, S.
, 
Zhao, Q.
, 
Zhan, D.
, 
Vasseur, L.
, 
Wang, Y.
, 
Yu, J.
, 
Liao, Z.
, 
Xu, X.
, 
Qi, R.
, 
Wang, W.
, 
Ma, Y.
, 
Wang, P.
, 
Ye, N.
, 
Ma, D.
, 
Shi, Y.
, … 
You, M.
 (2021). Haplotype‐resolved genome assembly provides insights into evolutionary history of the tea plant <em>Camellia sinensis</em>
. Nature Genetics, 53, 1250–1259. 10.1038/s41588-021-00895-y

</cite> [<a href="https://doi.org/10.1038/s41588-021-00895-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8346365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34267370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Haplotype%E2%80%90resolved%20genome%20assembly%20provides%20insights%20into%20evolutionary%20history%20of%20the%20tea%20plant%20Camellia%20sinensis&amp;author=X.%20Zhang&amp;author=S.%20Chen&amp;author=L.%20Shi&amp;author=D.%20Gong&amp;author=S.%20Zhang&amp;volume=53&amp;publication_year=2021&amp;pages=1250-1259&amp;pmid=34267370&amp;doi=10.1038/s41588-021-00895-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0099">
<cite>

Zhou, Q.
, 
Tang, D.
, 
Huang, W. U.
, 
Yang, Z.
, 
Zhang, Y.
, 
Hamilton, J. P.
, 
Visser, R. G. F.
, 
Bachem, C. W. B.
, 
Robin Buell, C.
, 
Zhang, Z.
, 
Zhang, C.
, &amp; 
Huang, S.
 (2020). Haplotype‐resolved genome analyses of a heterozygous diploid potato. Nature Genetics, 52, 1018–1023. 10.1038/s41588-020-0699-x

</cite> [<a href="https://doi.org/10.1038/s41588-020-0699-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7527274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32989320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Haplotype%E2%80%90resolved%20genome%20analyses%20of%20a%20heterozygous%20diploid%20potato&amp;author=Q.%20Zhou&amp;author=D.%20Tang&amp;author=W.%20U.%20Huang&amp;author=Z.%20Yang&amp;author=Y.%20Zhang&amp;volume=52&amp;publication_year=2020&amp;pages=1018-1023&amp;pmid=32989320&amp;doi=10.1038/s41588-020-0699-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0100">
<cite>

Zhou, Y.
, 
Zhang, Z.
, 
Bao, Z.
, 
Li, H.
, 
Lyu, Y.
, 
Zan, Y.
, 
Wu, Y.
, 
Cheng, L.
, 
Fang, Y.
, 
Wu, K.
, 
Zhang, J.
, 
Lyu, H.
, 
Lin, T.
, 
Gao, Q.
, 
Saha, S.
, 
Mueller, L.
, 
Fei, Z.
, 
Städler, T.
, 
Xu, S.
, … 
Huang, S.
 (2022). Graph pangenome captures missing heritability and empowers tomato breeding. Nature, 606, 527–534. 10.1038/s41586-022-04808-9

</cite> [<a href="https://doi.org/10.1038/s41586-022-04808-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9200638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35676474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Graph%20pangenome%20captures%20missing%20heritability%20and%20empowers%20tomato%20breeding&amp;author=Y.%20Zhou&amp;author=Z.%20Zhang&amp;author=Z.%20Bao&amp;author=H.%20Li&amp;author=Y.%20Lyu&amp;volume=606&amp;publication_year=2022&amp;pages=527-534&amp;pmid=35676474&amp;doi=10.1038/s41586-022-04808-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0101">
<cite>

Zimin, A. V.
, &amp; 
Salzberg, S. L.
 (2020). The genome polishing tool POLCA makes fast and accurate corrections in genome assemblies. PloS Computational Biology, 16, e1007981. 10.1371/journal.pcbi.1007981

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1007981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7347232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32589667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PloS%20Computational%20Biology&amp;title=The%20genome%20polishing%20tool%20POLCA%20makes%20fast%20and%20accurate%20corrections%20in%20genome%20assemblies&amp;author=A.%20V.%20Zimin&amp;author=S.%20L.%20Salzberg&amp;volume=16&amp;publication_year=2020&amp;pages=e1007981&amp;pmid=32589667&amp;doi=10.1371/journal.pcbi.1007981&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270079-bib-0102">
<cite>

Zwyrtková, J.
, 
Němečková, A.
, 
Čížková, J.
, 
Holušová, K.
, 
Kapustová, V.
, 
Svačina, R.
, 
Kopecký, D.
, 
Till, B. J.
, 
Doležel, J.
, &amp; 
Hřibová, E.
 (2020). Comparative analyses of DNA repeats and identification of a novel Fesreba centromeric element in fescues and ryegrasses. BMC Plant Biology, 20, Article 280. 10.1186/s12870-020-02495-0

</cite> [<a href="https://doi.org/10.1186/s12870-020-02495-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7302162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32552738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Comparative%20analyses%20of%20DNA%20repeats%20and%20identification%20of%20a%20novel%20Fesreba%20centromeric%20element%20in%20fescues%20and%20ryegrasses&amp;author=J.%20Zwyrtkov%C3%A1&amp;author=A.%20N%C4%9Bme%C4%8Dkov%C3%A1&amp;author=J.%20%C4%8C%C3%AD%C5%BEkov%C3%A1&amp;author=K.%20Holu%C5%A1ov%C3%A1&amp;author=V.%20Kapustov%C3%A1&amp;volume=20&amp;publication_year=2020&amp;pages=280&amp;pmid=32552738&amp;doi=10.1186/s12870-020-02495-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>Figure S1. Consensus genetic linkage maps used for scaffolding Rabiosa v1, as output by Lep‐MAP3 (Rastas, <a href="#tpg270079-bib-0076" class="usa-link" aria-describedby="tpg270079-bib-0076">2017</a>).</p>
<p>Figure S2. Gene‐based synteny between Rabiosa and other Pooideae genomes.</p>
<p>Figure S3. Pair‐wise gene‐based synteny between Rabiosa v2, Rabiosa h1, and h2.</p>
<p>Figure S4. K‐mer profile of the Rabiosa diploid genome generated using GenomeScope2 with k‐mers from whole‐genome sequencing short reads.</p>
<p>Figure S5. Sequence identity between Rabiosa h1 and Rabiosa h2 for the chromosomes Chr2 to Chr7 (the comparison between the two haplotypes of Chr1 is given in the main text).</p>
<p>Figure S6. Histogram of the phenotypic distribution of resistance to stem rust based on the best linear unbiased estimators (BLUEs).</p>
<p>Figure S7. QTL analysis for resistance to stem rust using the genetic linkage map of Sikem.</p>
<p>Figure S8. Reference bias affects reference‐based phasing.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12376112/bin/TPG2-18-e70079-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70079-s001.docx</a><sup> (1.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<p>Table S1: Assembly statistics of Rabiosa</p>
<p>Table S2: Statistics of phase blocks</p>
<p>Table S3: Statistics of binned reads from reference‐based phasing</p>
<p>Table S4: BUSCO (v4.1.4) assessment results of genome assemblies</p>
<p>Table S5: Transposable elements (TE) annotation results</p>
<p>Table S6: Number of variants for calculating sequence variation between the two haplotypes of Rabiosa</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12376112/bin/TPG2-18-e70079-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70079-s002.xlsx</a><sup> (23.8KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The unphased haploid genome assembly Rabiosa v2 is publicly available on NCBI under the accession: <a href="https://www.ncbi.nlm.nih.gov/nuccore/JAUUTY000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JAUUTY000000000</a>. The two haplotype assemblies are publicly available on NCBI under the accession: <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBHEQO000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBHEQO000000000</a>. The transcripts used in our haplotig purge pipeline generated previously by Copetti et al. (<a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>), together with the genome annotation results including the GFF files and the functional annotation results, are available at <a href="https://doi.org/10.5281/zenodo.13641219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.13641219</a>. All genetic linkage maps generated in this work together with the stem rust resistance scores (BLUEs) used for QTL analysis are available at <a href="https://doi.org/10.5281/zenodo.15009016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15009016</a>. All sequencing data (ONT data, WGS short reads, Hi‐C data, and GBS data) generated in this work have been submitted to NCBI under BioProject accession: PRJNA990649. The RNA‐Seq data generated previously by Copetti et al. (<a href="#tpg270079-bib-0020" class="usa-link" aria-describedby="tpg270079-bib-0020">2021</a>) are available on NCBI with the BioProject accession: PRJNA1156394. Code used for bioinformatic analyses in this work can be found at GitHub, <a href="https://github.com/Yutang-ETH/Rabiosa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Yutang‐ETH/Rabiosa</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70079"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70079.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12376112/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12376112/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376112%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376112/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12376112/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12376112/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12376112/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12376112/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12376112/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="TiN0q1GX5YpA4sxHyBW2ofDI0iORy0IrKSvGBahigjK3QYdZRQOaJtW3RIIRh1JN">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
