
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Identity‐By‐Descent Mapping Using Multi‐Individual IBD With Genome‐Wide Multiple Testing Adjustment - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484E18AEF1C030584E10012061C1E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327188/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genetic Epidemiology">
<meta name="citation_title" content="Identity‐By‐Descent Mapping Using Multi‐Individual IBD With Genome‐Wide Multiple Testing Adjustment">
<meta name="citation_author" content="Ruoyi Cai">
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington, USA">
<meta name="citation_author" content="Sharon R Browning">
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington, USA">
<meta name="citation_publication_date" content="2025 Jul 28">
<meta name="citation_volume" content="49">
<meta name="citation_issue" content="6">
<meta name="citation_firstpage" content="e70015">
<meta name="citation_doi" content="10.1002/gepi.70015">
<meta name="citation_pmid" content="40719186">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327188/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327188/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327188/pdf/GEPI-49-0.pdf">
<meta name="description" content="We present an identity‐by‐descent mapping approach to test the association between genome‐wide loci and complex traits. Our method evaluates whether levels of genetic similarities at specific genomic locations, captured by local relatedness matrices ...">
<meta name="og:title" content="Identity‐By‐Descent Mapping Using Multi‐Individual IBD With Genome‐Wide Multiple Testing Adjustment">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="We present an identity‐by‐descent mapping approach to test the association between genome‐wide loci and complex traits. Our method evaluates whether levels of genetic similarities at specific genomic locations, captured by local relatedness matrices ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327188/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12327188">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/gepi.70015"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/GEPI-49-0.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12327188%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12327188/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12327188/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327188/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/gepi.70015" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genet Epidemiol</button></div>. 2025 Jul 28;49(6):e70015. doi: <a href="https://doi.org/10.1002/gepi.70015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/gepi.70015</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genet%20Epidemiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genet%20Epidemiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genet%20Epidemiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genet%20Epidemiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Identity‐By‐Descent Mapping Using Multi‐Individual IBD With Genome‐Wide Multiple Testing Adjustment</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ruoyi Cai</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ruoyi Cai</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biostatistics, University of Washington, Seattle, Washington, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruoyi Cai</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Browning%20SR%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Sharon R Browning</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Sharon R Browning</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biostatistics, University of Washington, Seattle, Washington, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Browning%20SR%22%5BAuthor%5D" class="usa-link"><span class="name western">Sharon R Browning</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="gepi70015-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Biostatistics, University of Washington, Seattle, Washington, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong> Ruoyi Cai (<span>rcai2@uw.edu</span>), 
Sharon R. Browning (<span>sguy@uw.edu</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 12; Received 2025 Feb 3; Accepted 2025 Jul 15; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Genetic Epidemiology</em> published by Wiley Periodicals LLC.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12327188  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40719186/" class="usa-link">40719186</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>We present an identity‐by‐descent mapping approach to test the association between genome‐wide loci and complex traits. Our method evaluates whether levels of genetic similarities at specific genomic locations, captured by local relatedness matrices derived from multi‐individual IBD sharing, are associated with phenotypic variation in complex traits. In addition, we propose an approach to adjust for multiple testing in genome‐wide IBD mapping scans based on the correlation structure between test statistics across the genome. Through simulation studies, we demonstrate that our test has a well‐controlled genome‐wide type I error rate and superior power to detect rare and untyped variants compared to standard single‐variant tests. We applied our method to systolic blood pressure data from White British individuals in the UK Biobank.</p></section><section id="gepi70015-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>With the development of sequencing technology, there has been a surge in understanding the association between genetic variants and complex traits over the past two decades. Historically, there are two major classes of methods for association analysis: linkage mapping and genome‐wide association studies (GWAS). Linkage mapping identifies regions of the genome that are co‐inherited with a trait within families, which is particularly powerful for detecting rare variants (Amos <a href="#gepi70015-bib-0003" class="usa-link" aria-describedby="gepi70015-bib-0003">1994</a>; Blangero and Almasy <a href="#gepi70015-bib-0007" class="usa-link" aria-describedby="gepi70015-bib-0007">1997</a>; Almasy and Blangero <a href="#gepi70015-bib-0001" class="usa-link" aria-describedby="gepi70015-bib-0001">1998</a>; Blangero et al. <a href="#gepi70015-bib-0008" class="usa-link" aria-describedby="gepi70015-bib-0008">2000</a>; Day‐Williams et al. <a href="#gepi70015-bib-0022" class="usa-link" aria-describedby="gepi70015-bib-0022">2011</a>). However, linkage mapping is often limited by its low resolution and requirement for family data. In contrast, as a more commonly used method at present, GWAS can analyze common genetic variants across large, unrelated populations, offering high resolution and the ability to detect associations with complex traits at a genome‐wide level (Hirschhorn and Daly <a href="#gepi70015-bib-0031" class="usa-link" aria-describedby="gepi70015-bib-0031">2005</a>; McCarthy and Hirschhorn <a href="#gepi70015-bib-0041" class="usa-link" aria-describedby="gepi70015-bib-0041">2008</a>; Visscher et al. <a href="#gepi70015-bib-0056" class="usa-link" aria-describedby="gepi70015-bib-0056">2017</a>).</p>
<p>While single‐variant tests in GWAS are powerful for detecting common variants, they often struggle to detect structural variants or rare variants that are more likely to be population‐specific, untyped in the genotype data, and have small individual effect sizes (McCarthy et al. <a href="#gepi70015-bib-0040" class="usa-link" aria-describedby="gepi70015-bib-0040">2008</a>; Eichler et al. <a href="#gepi70015-bib-0023" class="usa-link" aria-describedby="gepi70015-bib-0023">2010</a>; Tam et al. <a href="#gepi70015-bib-0053" class="usa-link" aria-describedby="gepi70015-bib-0053">2019</a>). Variant‐set tests improve upon single‐variant tests by aggregating effects across multiple variants within a genomic region or a set of related genes, thus increasing power to detect rare variants (Madsen and Browning <a href="#gepi70015-bib-0038" class="usa-link" aria-describedby="gepi70015-bib-0038">2009</a>; Wu et al. <a href="#gepi70015-bib-0060" class="usa-link" aria-describedby="gepi70015-bib-0060">2011</a>; Lee et al. <a href="#gepi70015-bib-0036" class="usa-link" aria-describedby="gepi70015-bib-0036">2012</a>; Liu et al. <a href="#gepi70015-bib-0037" class="usa-link" aria-describedby="gepi70015-bib-0037">2019</a>). However, these methods often rely on pre‐defined models about the genetic architecture underlying complex traits, the sparsity of causal variants, and the choice of weights on the effects of variants. In comparison, population‐based identity‐by‐descent (IBD) mapping offers a complementary approach for linkage mapping, GWAS, and variant‐set tests, providing a more flexible and comprehensive framework for association testing.</p>
<p>IBD mapping identifies genomic regions containing potential causal variants by searching for genomic positions where levels of IBD sharing are associated with phenotypic variation in a group of individuals for a trait of interest (Purcell et al. <a href="#gepi70015-bib-0046" class="usa-link" aria-describedby="gepi70015-bib-0046">2007</a>; Gusev et al. <a href="#gepi70015-bib-0028" class="usa-link" aria-describedby="gepi70015-bib-0028">2011</a>; Browning and Thompson <a href="#gepi70015-bib-0015" class="usa-link" aria-describedby="gepi70015-bib-0015">2012</a>; Browning and Browning <a href="#gepi70015-bib-0013" class="usa-link" aria-describedby="gepi70015-bib-0013">2012</a>; Qian et al. <a href="#gepi70015-bib-0047" class="usa-link" aria-describedby="gepi70015-bib-0047">2014</a>; Vacic et al. <a href="#gepi70015-bib-0054" class="usa-link" aria-describedby="gepi70015-bib-0054">2014</a>; Westerlind et al. <a href="#gepi70015-bib-0058" class="usa-link" aria-describedby="gepi70015-bib-0058">2015</a>; Belbin et al. <a href="#gepi70015-bib-0005" class="usa-link" aria-describedby="gepi70015-bib-0005">2017</a>; Hsueh et al. <a href="#gepi70015-bib-0033" class="usa-link" aria-describedby="gepi70015-bib-0033">2017</a>; Henden et al. <a href="#gepi70015-bib-0030" class="usa-link" aria-describedby="gepi70015-bib-0030">2020</a>; Chen et al. <a href="#gepi70015-bib-0019" class="usa-link" aria-describedby="gepi70015-bib-0019">2023</a>). By leveraging long stretches of IBD segments shared between individuals, IBD mapping can capture the combined effects of co‐occurring proximate rare alleles, even when individual variants have small or modest effects (Browning and Thompson <a href="#gepi70015-bib-0015" class="usa-link" aria-describedby="gepi70015-bib-0015">2012</a>; Browning and Browning <a href="#gepi70015-bib-0013" class="usa-link" aria-describedby="gepi70015-bib-0013">2012</a>; Chen et al. <a href="#gepi70015-bib-0019" class="usa-link" aria-describedby="gepi70015-bib-0019">2023</a>). In addition, IBD mapping approaches can indirectly recover signals of untyped rare variants, structural variants, or population‐specific variants tagged by IBD haplotypes (Purcell et al. <a href="#gepi70015-bib-0046" class="usa-link" aria-describedby="gepi70015-bib-0046">2007</a>; Gusev et al. <a href="#gepi70015-bib-0028" class="usa-link" aria-describedby="gepi70015-bib-0028">2011</a>; Browning and Thompson <a href="#gepi70015-bib-0015" class="usa-link" aria-describedby="gepi70015-bib-0015">2012</a>; Browning and Browning <a href="#gepi70015-bib-0013" class="usa-link" aria-describedby="gepi70015-bib-0013">2012</a>; Chen et al. <a href="#gepi70015-bib-0019" class="usa-link" aria-describedby="gepi70015-bib-0019">2023</a>). Furthermore, IBD mapping is not reliant on assumptions about the underlying genetic structure or models on the effects of causal variants, which contributes to its applicability and robustness in analysis of complex traits with unknown or intricate genetic architectures (Madsen and Browning <a href="#gepi70015-bib-0038" class="usa-link" aria-describedby="gepi70015-bib-0038">2009</a>; Wu et al. <a href="#gepi70015-bib-0060" class="usa-link" aria-describedby="gepi70015-bib-0060">2011</a>; Lee et al. <a href="#gepi70015-bib-0036" class="usa-link" aria-describedby="gepi70015-bib-0036">2012</a>; Liu et al. <a href="#gepi70015-bib-0037" class="usa-link" aria-describedby="gepi70015-bib-0037">2019</a>).</p>
<p>Previous IBD mapping studies have mostly focused on using IBD segments shared between pairs of individuals (Purcell et al. <a href="#gepi70015-bib-0046" class="usa-link" aria-describedby="gepi70015-bib-0046">2007</a>; Browning and Thompson <a href="#gepi70015-bib-0015" class="usa-link" aria-describedby="gepi70015-bib-0015">2012</a>; Vacic et al. <a href="#gepi70015-bib-0054" class="usa-link" aria-describedby="gepi70015-bib-0054">2014</a>; Westerlind et al. <a href="#gepi70015-bib-0058" class="usa-link" aria-describedby="gepi70015-bib-0058">2015</a>; Belbin et al. <a href="#gepi70015-bib-0005" class="usa-link" aria-describedby="gepi70015-bib-0005">2017</a>; Hsueh et al. <a href="#gepi70015-bib-0033" class="usa-link" aria-describedby="gepi70015-bib-0033">2017</a>; Henden et al. <a href="#gepi70015-bib-0030" class="usa-link" aria-describedby="gepi70015-bib-0030">2020</a>; Chen et al. <a href="#gepi70015-bib-0019" class="usa-link" aria-describedby="gepi70015-bib-0019">2023</a>). However, IBD information can extend beyond the pairwise level by considering clusters of haplotypes that are all IBD with each other (Gusev et al. <a href="#gepi70015-bib-0028" class="usa-link" aria-describedby="gepi70015-bib-0028">2011</a>; Qian et al. <a href="#gepi70015-bib-0047" class="usa-link" aria-describedby="gepi70015-bib-0047">2014</a>; Browning and Browning <a href="#gepi70015-bib-0014" class="usa-link" aria-describedby="gepi70015-bib-0014">2024</a>; He <a href="#gepi70015-bib-0029" class="usa-link" aria-describedby="gepi70015-bib-0029">2013</a>). Using IBD haplotypes shared among groups of individuals may enhance power for IBD mapping due to more accurate IBD calls and the additional information provided by specific IBD group effects that are ignored in pairwise IBD analysis. Most existing approaches detect multi‐individual IBD sharing from pairwise IBD segments by either using probabilistic techniques to model the states of potential IBD tracts across multiple individuals or by aggregating overlapping pairwise IBD segments through graph‐based clustering or hidden Markov models (Gusev et al. <a href="#gepi70015-bib-0028" class="usa-link" aria-describedby="gepi70015-bib-0028">2011</a>; Qian et al. <a href="#gepi70015-bib-0047" class="usa-link" aria-describedby="gepi70015-bib-0047">2014</a>; He <a href="#gepi70015-bib-0029" class="usa-link" aria-describedby="gepi70015-bib-0029">2013</a>; Moltke et al. <a href="#gepi70015-bib-0043" class="usa-link" aria-describedby="gepi70015-bib-0043">2011</a>). However, these approaches face substantial computational challenges in biobank‐scale datasets because the number of pairwise IBD segments increases quadratically with sample size, limiting the applicability of multi‐individual IBD information for association testing in large‐scale studies. In contrast, Browning and Browning (<a href="#gepi70015-bib-0014" class="usa-link" aria-describedby="gepi70015-bib-0014">2024</a>) utilize PBWT sorting to directly merge haplotypes sharing the same allele sequence over a specified minimum length threshold within a targeted region, avoiding exhaustive pairwise comparisons (Browning and Browning <a href="#gepi70015-bib-0014" class="usa-link" aria-describedby="gepi70015-bib-0014">2024</a>). This approach reliably identifies clusters of shared ancestry and scales linearly with sample size, making it well‐suited for biobank‐scale analyses.</p>
<p>In this study, we develop an IBD mapping test leveraging multi‐individual IBD sharing between distantly related individuals in large, outbred populations. Motivated by variance component models in linkage analysis, we construct local relatedness matrices from multi‐individual IBD sharing to quantify genetic similarities at consecutive genomic locations. Using a likelihood ratio framework, we test for associations between genomic regions and a complex trait of interest by evaluating whether local genetic similarities significantly contribute to the phenotypic variation of the trait. Additionally, we develop an approach to adjust for multiple testing based on an analytical formula for the correlation between test statistics across the genome. Through simulation studies, we demonstrate that our test maintains a well‐controlled family‐wise type I error rate and exhibits superior power to detect the effects of rare and untyped variants in SNP array data compared to standard single‐variant tests. Applying this method to White British individuals in the UK Biobank, we identified signals associated with systolic blood pressure in this cohort.</p></section><section id="gepi70015-sec-0020"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="gepi70015-sec-0030"><h3 class="pmc_sec_title">2.1. Variance Component Test</h3>
<p>We use a quantitative trait model that is commonly used in linkage analysis (Blangero and Almasy <a href="#gepi70015-bib-0007" class="usa-link" aria-describedby="gepi70015-bib-0007">1997</a>; Almasy and Blangero <a href="#gepi70015-bib-0001" class="usa-link" aria-describedby="gepi70015-bib-0001">1998</a>; Day‐Williams et al. <a href="#gepi70015-bib-0022" class="usa-link" aria-describedby="gepi70015-bib-0022">2011</a>; Almasy and Blangero <a href="#gepi70015-bib-0002" class="usa-link" aria-describedby="gepi70015-bib-0002">2010</a>). Denote <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">Y</mi></mrow></mrow></mrow></math></span> as the vector of quantitative trait values observed from a population sample. We model the trait values with the following linear mixed‐effects model:
</p>
<table class="disp-formula p" id="gepi70015-disp-0001"><tr>
<td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><mrow><mi mathvariant="bold-italic">Y</mi><mo>=</mo><mi mathvariant="bold">X</mi><mi mathvariant="bold-italic">β</mi><mo>+</mo><mi mathvariant="bold-italic">G</mi><mo>+</mo><msub><mi mathvariant="bold-italic">Q</mi><mi>l</mi></msub><mo>+</mo><mi mathvariant="bold-italic">ε</mi><mo>.</mo></mrow></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>In the above formula, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">X</mi></mrow></mrow></mrow></math></span> is a matrix of fixed‐effects covariates with effect sizes <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">G</mi></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold-italic">Q</mi><mi mathvariant="bold-italic">l</mi></msub></mrow></mrow></mrow></math></span> are random‐effects terms that represent the genome‐wide additive effect and the location‐specific effect at position <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> respectively, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></mrow></math></span> is the environmental effect. The genome‐wide additive effect, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">G</mi><mo>,</mo></mrow></mrow></mrow></math></span> is assumed to follow a multivariate normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mn mathvariant="bold">0</mn><mo>,</mo><mi mathvariant="bold">Ψ</mi><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">Ψ</mi></mrow></mrow></mrow></math></span> is the genome‐wide relatedness matrix that can be estimated from genome‐wide IBD sharing. We refer to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">Ψ</mi></mrow></mrow></mrow></math></span> as the global IBD matrix. The location‐specific effect <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold-italic">Q</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> at position <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> is assumed to follow the multivariate normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn mathvariant="bold">0</mn><mo>,</mo><msub><mrow><mspace width=".01em"></mspace><mi mathvariant="bold">Φ</mi></mrow><mi>l</mi></msub><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> is the location‐specific relatedness matrix that characterizes the proportion of alleles shared IBD between each pair of individuals at a specific genomic location <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span>. We refer to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> as the local IBD matrix at position <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span>. The environmental effect is assumed to be independent between individuals and follow a normal distribution with mean 0 and variance <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mrow><mrow><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup></mrow></mrow></mrow></math></span>.</p>
<p>Given the mixed‐effects model in (1), the phenotypic variance of the quantitative trait <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mrow><mrow><mi>Y</mi></mrow></mrow></mrow></math></span> is represented by the sum of three variance components:
</p>
<table class="disp-formula p" id="gepi70015-disp-0002"><tr>
<td class="formula"><math id="jats-math-22" display="block"><mrow><mrow><mrow><mi mathvariant="bold">V</mi><mo>≔</mo><mi mathvariant="italic">Var</mi><mrow><mo>(</mo><mi mathvariant="bold-italic">Y</mi><mo>)</mo></mrow><mo>=</mo><mi mathvariant="bold">Ψ</mi><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>+</mo><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub><msubsup><mi>σ</mi><mrow><msub><mi>Q</mi><mi>l</mi></msub></mrow><mn>2</mn></msubsup><mo>+</mo><mi mathvariant="bold">I</mi><msubsup><mi mathvariant="normal">σ</mi><mi mathvariant="normal">ε</mi><mn>2</mn></msubsup><mo>.</mo></mrow></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>The parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mrow><mrow><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><mrow><mrow><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-25" display="inline"><mrow><mrow><mrow><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup></mrow></mrow></mrow></math></span> represent the effect sizes of the three variance components. The structuring matrices <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">Ψ</mi></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">I</mi></mrow></mrow></mrow></math></span> predict the covariances among individuals attributable to the effect of each variance component, respectively. Testing whether a locus at position <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> is associated with the quantitative trait is equivalent to testing the variance component hypothesis <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><mrow><mrow><msubsup><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><mi>σ</mi></mrow><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>=</mo><mn>0</mn></mrow></mrow></mrow></math></span> versus <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mrow><mrow><msubsup><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><mi>σ</mi></mrow><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>&gt;</mo><mn>0</mn></mrow></mrow></mrow></math></span>.</p>
<p>Many different forms of test statistics have been proposed for variance component tests. In this study, we use the log‐of‐odds (LOD) score (Almasy and Blangero <a href="#gepi70015-bib-0001" class="usa-link" aria-describedby="gepi70015-bib-0001">1998</a>; Page et al. <a href="#gepi70015-bib-0044" class="usa-link" aria-describedby="gepi70015-bib-0044">1998</a>) as the test statistic to take advantage of the maximal information considered under the likelihood ratio framework. To test <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mrow><mrow><msubsup><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><mi>σ</mi></mrow><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>=</mo><mn>0</mn></mrow></mrow></mrow></math></span> versus <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mrow><mrow><msubsup><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><mi>σ</mi></mrow><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>&gt;</mo><mn>0</mn></mrow></mrow></mrow></math></span>, we consider the test statistic <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> defined as:
</p>
<table class="disp-formula p" id="gepi70015-disp-0003"><tr><td class="formula"><math id="jats-math-35" display="block"><mrow><mrow><mrow><msub><mrow><mspace width="1em"></mspace><mi>W</mi></mrow><mi>l</mi></msub><mo>=</mo><mn>2</mn><mi>ln</mi><mrow><mo>(</mo><mn>10</mn><mo>)</mo></mrow><mo>×</mo><mtext>LOD score</mtext><mo>=</mo><mn>2</mn><mi>ln</mi><mrow><mo>(</mo><mn>10</mn><mo>)</mo></mrow><mo>×</mo><msub><mi>log</mi><mn>10</mn></msub><mfenced close=")" open="("><mfrac><mrow><munder><mi>max</mi><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup></mrow></munder><mrow><msub><mi>L</mi><mi mathvariant="bold-italic">Y</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi mathvariant="normal">;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow></mfenced></mrow></mrow><mrow><munder><mi>max</mi><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup></mrow></munder><mrow><msub><mi>L</mi><mi mathvariant="bold-italic">Y</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>=</mo><mn>0</mn><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi mathvariant="normal">;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow></mfenced></mrow></mrow></mfrac></mfenced></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70015-disp-0004"><tr>
<td class="formula"><math id="jats-math-36" display="block"><mrow><mrow><mrow><mo>=</mo><mn>2</mn><mo stretchy="true">(</mo><munder><mi>max</mi><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup></mrow></munder><mrow><msub><mi>l</mi><mi mathvariant="bold-italic">Y</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi>;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow></mfenced></mrow><mo>−</mo><munder><mi>max</mi><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup></mrow></munder><mrow><msub><mi>l</mi><mi mathvariant="bold-italic">Y</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>=</mo><mn>0</mn><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi>;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow></mfenced></mrow><mo stretchy="true">)</mo></mrow></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mi mathvariant="bold-italic">Y</mi></msub><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi>;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><mi>ln</mi><mrow><msub><mi>L</mi><mi mathvariant="bold-italic">Y</mi></msub><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi>;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></mrow></math></span> is the restricted model log‐likelihood that takes the following form, with the variance matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-38" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">V</mi></mrow></mrow></mrow></math></span> defined in (2):
</p>
<table class="disp-formula p" id="gepi70015-disp-0005"><tr>
<td class="formula"><math id="jats-math-39" display="block"><mrow><mrow><mrow><msub><mi>l</mi><mi mathvariant="bold-italic">Y</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi mathvariant="normal">;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow></mfenced><mo>=</mo><mi mathvariant="italic">const</mi><mo>.</mo><mo>−</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi>ln</mi><mo mathvariant="normal">|</mo><mi mathvariant="bold">V</mi><mo mathvariant="normal">|</mo><mo>−</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi>ln</mi><mrow><msup><mrow><mo mathvariant="normal">|</mo><mi mathvariant="bold">X</mi></mrow><mi>T</mi></msup><msup><mi mathvariant="bold">V</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mi mathvariant="bold">X</mi><mo mathvariant="normal">|</mo></mrow><mo>−</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><msup><mrow><mo>(</mo><mrow><mi mathvariant="bold-italic">Y</mi><mo>−</mo><mi mathvariant="bold">X</mi><mi mathvariant="bold-italic">β</mi></mrow><mo>)</mo></mrow><mi>T</mi></msup><msup><mi mathvariant="bold">V</mi><mrow><mo mathvariant="bold">−</mo><mn mathvariant="bold">1</mn></mrow></msup><mrow><mo>(</mo><mrow><mi mathvariant="bold-italic">Y</mi><mo mathvariant="bold">−</mo><mi mathvariant="bold">X</mi><mi mathvariant="bold-italic">β</mi></mrow><mo>)</mo></mrow><mo mathvariant="bold">.</mo></mrow></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>To calculate the test statistic, we find restricted maximum likelihood estimators (REML) of the fixed effect <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></mrow></math></span> and variance effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-41" display="inline"><mrow><mrow><mrow><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><mrow><mrow><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup></mrow></mrow></mrow></math></span>. Under the null hypothesis, the distribution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> is a <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mrow><mrow><mfrac><mn>1</mn><mn>2</mn></mfrac><mo>:</mo><mfrac><mn>1</mn><mn>2</mn></mfrac></mrow></mrow></mrow></math></span> mixture of a point mass at <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-45" display="inline"><mrow><mrow><mrow><mn>0</mn></mrow></mrow></mrow></math></span> and a <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-46" display="inline"><mrow><mrow><mrow><msup><mi>χ</mi><mn>2</mn></msup></mrow></mrow></mrow></math></span> distribution with 1 degree of freedom (Appendix <a href="#gepi70015-app-0001" class="usa-link">A</a>) (Almasy and Blangero <a href="#gepi70015-bib-0001" class="usa-link" aria-describedby="gepi70015-bib-0001">1998</a>).</p>
<p>The application of the likelihood ratio statistic in variance component tests faces two major challenges. First, without closed‐form solutions for REML estimates of variance components, an efficient and accurate numerical optimization algorithm is essential. To address this, we use the BFGS algorithm (Broyden <a href="#gepi70015-bib-0016" class="usa-link" aria-describedby="gepi70015-bib-0016">1970</a>; Fletcher <a href="#gepi70015-bib-0024" class="usa-link" aria-describedby="gepi70015-bib-0024">1970</a>; Goldfarb <a href="#gepi70015-bib-0026" class="usa-link" aria-describedby="gepi70015-bib-0026">1970</a>; Shanno <a href="#gepi70015-bib-0050" class="usa-link" aria-describedby="gepi70015-bib-0050">1970</a>; Byrd et al. <a href="#gepi70015-bib-0018" class="usa-link" aria-describedby="gepi70015-bib-0018">1995</a>) from the SciPy (Virtanen et al. <a href="#gepi70015-bib-0055" class="usa-link" aria-describedby="gepi70015-bib-0055">2020</a>) optimize library in Python due to its robustness and efficiency in handling large‐scale optimization problems. Second, calculating the log‐likelihood for multiple iterations during optimization can become computationally expensive with large sample sizes <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-47" display="inline"><mrow><mrow><mrow><mo>(</mo><mi>N</mi><mo>)</mo></mrow></mrow></mrow></math></span>, as inverting <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-48" display="inline"><mrow><mrow><mrow><mi>N</mi><mo>×</mo><mi>N</mi></mrow></mrow></mrow></math></span> matrices typically has <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-49" display="inline"><mrow><mrow><mrow><mi>O</mi><mrow><mo>(</mo><msup><mi>N</mi><mn>3</mn></msup><mo>)</mo></mrow></mrow></mrow></mrow></math></span> complexity. However, in our application, we can reduce the computational burden because the variance‐covariance matrix has a sparse block‐diagonal structure, which arises because individuals only share IBD within small clusters in large outbred populations (see the section “Global and local IBD matrices”). We stored the global and local IBD matrices as sparse matrices using data structures and functions provided by the SciPy (Virtanen et al. <a href="#gepi70015-bib-0055" class="usa-link" aria-describedby="gepi70015-bib-0055">2020</a>) sparse library. When calculating the inverse, we perform sparse matrix Cholesky decomposition followed by forward substitution, with utilities implemented in the scikit‐sparse library (Shor et al. <a href="#gepi70015-bib-0051" class="usa-link" aria-describedby="gepi70015-bib-0051">2019</a>; Davis <a href="#gepi70015-bib-0021" class="usa-link" aria-describedby="gepi70015-bib-0021">2021</a>). This approach takes advantage of the block structure in the sparse matrix by performing calculations only on blocks with non‐zero entries, so the overall computational complexity scales in cubic terms with the largest block size, rather than with the full sample size.</p></section><section id="gepi70015-sec-0040"><h3 class="pmc_sec_title">2.2. Multi‐Individual Identity‐by‐Descent Inference</h3>
<p>In this study, we focus on haplotypes that are identical by descent across multiple individuals instead of between pairs of individuals, which provides more detailed information about IBD sharing across the genome. We use the recently developed software ibd‐cluster that scans locations across the genome to directly cluster haplotypes that are IBD at each specific genomic location without identifying pairwise IBD segments (Browning and Browning <a href="#gepi70015-bib-0014" class="usa-link" aria-describedby="gepi70015-bib-0014">2024</a>).</p>
<p>The ibd‐cluster algorithm first assigns each haplotype to an individual cluster and then merges clusters when a pair of haplotypes, one from each cluster, share the same allele sequence in a genomic region of at least <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-50" display="inline"><mrow><mrow><mrow><mi>L</mi></mrow></mrow></mrow></math></span> cM that contains the markers of interest and extends at least <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-51" display="inline"><mrow><mrow><mrow><mi>T</mi></mrow></mrow></mrow></math></span> cM (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-52" display="inline"><mrow><mrow><mrow><mi>T</mi><mo>&lt;</mo><mi>L</mi><mo>/</mo><mn>2</mn></mrow></mrow></mrow></math></span>) in both direction from the markers of interest. We refer to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-53" display="inline"><mrow><mrow><mrow><mi>L</mi></mrow></mrow></mrow></math></span> as the haplotype length threshold and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-54" display="inline"><mrow><mrow><mrow><mi>T</mi></mrow></mrow></mrow></math></span> as the trimming threshold. Due to IBD transitivity, only clusters for pairs of haplotypes that are adjacent in the positional Burrows‐Wheeler Transform sorting need to be merged, so this approach scales linearly with the number of individuals in the dataset in terms of computation time, memory requirements, and output size. This makes it feasible for using with very large datasets, such as those in biobanks.</p>
<p>At a given genomic location, a group identifier representing the IBD state is assigned to each of the two haplotypes for each individual in the sample. Haplotypes sharing the same group identifier are identical by descent with each other at the given location. This format is highly efficient for computing local relatedness matrices, as it avoids iterating through a potentially quadratic number of IBD segments relative to the sample size. Instead, computations are performed directly on a vector of haplotype cluster indices, which scales linearly with the sample size.</p>
<p>Using smaller haplotype length thresholds <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-55" display="inline"><mrow><mrow><mrow><mi>L</mi></mrow></mrow></mrow></math></span> or smaller trimming thresholds <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-56" display="inline"><mrow><mrow><mrow><mi>T</mi></mrow></mrow></mrow></math></span> to detect multi‐individual IBD will lead to inclusion of shorter shared haplotypes in clustering, which will potentially lead to larger multi‐individual IBD clusters. As a result, the local IBD matrix will tend to be less sparse and also more likely to include false‐positive IBD. In contrast, using larger <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-57" display="inline"><mrow><mrow><mrow><mi>L</mi></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-58" display="inline"><mrow><mrow><mrow><mi>T</mi></mrow></mrow></mrow></math></span> to detect multi‐individual IBD imposes a more stringent standard when clustering IBD haplotypes, and thus the resulting multi‐individual clusters will tend to contain longer but fewer segments, leading to a sparser local IBD matrix. Through simulation studies, we evaluate how different choices of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-59" display="inline"><mrow><mrow><mrow><mi>L</mi></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-60" display="inline"><mrow><mrow><mrow><mi>T</mi></mrow></mrow></mrow></math></span> for multi‐individual IBD inference affect the performance of our IBD mapping test.</p></section><section id="gepi70015-sec-0050"><h3 class="pmc_sec_title">2.3. Global and Local IBD Matrices</h3>
<p>The global IBD matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-61" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">Ψ</mi></mrow></mrow></mrow></math></span> is a symmetric matrix, with 1's on the diagonal, and the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-62" display="inline"><mrow><mrow><mrow><mo>(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>)</mo></mrow></mrow></mrow></math></span>‐th entry being the coefficient of relatedness between the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-63" display="inline"><mrow><mrow><mrow><mi>i</mi></mrow></mrow></mrow></math></span>‐th and the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-64" display="inline"><mrow><mrow><mrow><mi>j</mi></mrow></mrow></mrow></math></span>‐th individuals, which equals twice the kinship coefficient of this pair of individuals. To construct the global IBD matrix, we infer pairwise IBD segments using the software hap‐ibd (Zhou et al. <a href="#gepi70015-bib-0061" class="usa-link" aria-describedby="gepi70015-bib-0061">2020a</a>) from phased genotype or sequence data and estimate kinship coefficients using the software IBDkin (Zhou et al. <a href="#gepi70015-bib-0062" class="usa-link" aria-describedby="gepi70015-bib-0062">2020b</a>). IBDkin calculates kinship coefficients based on the proportion of the genome shared between pairs of individuals with 1 or 2 IBD haplotypes. Unlike averaging local relatedness matrices across genomic locations, which can be computationally intensive and susceptible to noise from sparse or unevenly distributed local IBD data, IBDkin leverages genome‐wide information to yield a robust and efficient estimate of relatedness. To maintain the sparsity of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-65" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">Ψ</mi></mrow></mrow></mrow></math></span>, we set kinship coefficients below 0.044 (corresponding to relationships beyond third‐degree relatives) to zero.</p>
<p>The local IBD matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-66" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> characterizes the genetic similarities between individuals at a specific position <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-67" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> on the genome. We utilize multi‐individual IBD inferred using the ibd‐cluster software to obtain the localized relatedness coefficient at position <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-68" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> between each pair of individuals in the local IBD matrix. We summarize the proportion of each individual's chromosome in each multi‐individual IBD group at the given location <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-69" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> with a matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-70" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">A</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span>. We define <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-71" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">A</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> as a <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-72" display="inline"><mrow><mrow><mrow><mi>N</mi><mo>×</mo><msub><mi>K</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> matrix, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-73" display="inline"><mrow><mrow><mrow><mi>N</mi></mrow></mrow></mrow></math></span> is the number of individuals in the sample and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-74" display="inline"><mrow><mrow><mrow><msub><mi>K</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> is the total number of IBD clusters at the given location <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-75" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span>. The <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-76" display="inline"><mrow><mrow><mrow><mo>(</mo><mi>i</mi><mo>,</mo><mi mathvariant="italic">k</mi><mo>)</mo></mrow></mrow></mrow></math></span>‐th entry of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-77" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">A</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> is the proportion of individual <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-78" display="inline"><mrow><mrow><mrow><mi>i</mi></mrow></mrow></mrow></math></span>'s chromosomes (0, ½, or 1) that are assigned to IBD group <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-79" display="inline"><mrow><mrow><mrow><mi>k</mi></mrow></mrow></mrow></math></span> at locus <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-80" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span>.</p>
<p>The local IBD matrix at genomic location <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-81" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> is then obtained as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-82" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub><mo>=</mo><mn>2</mn><mo>×</mo><msub><mi mathvariant="bold">A</mi><mi>l</mi></msub><msubsup><mi mathvariant="bold">A</mi><mi>l</mi><mi>T</mi></msubsup></mrow></mrow></mrow></math></span>. This means that we sum the proportion of IBD haplotypes over all clusters at <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-83" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> for each pair of individuals. As a result, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-84" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> is a symmetric matrix with diagonal of 1's and the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-85" display="inline"><mrow><mrow><mrow><mo>(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>)</mo></mrow></mrow></mrow></math></span>‐th entry as the proportion of alleles shared IBD by individuals <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-86" display="inline"><mrow><mrow><mrow><mi>i</mi></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-87" display="inline"><mrow><mrow><mrow><mi>j</mi></mrow></mrow></mrow></math></span> at position <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-88" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span>, which would be non‐zero only if individuals <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-89" display="inline"><mrow><mrow><mrow><mi>i</mi></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-90" display="inline"><mrow><mrow><mrow><mi>j</mi></mrow></mrow></mrow></math></span> have haplotypes in the same IBD cluster. Local IBD matrices constructed in this way are guaranteed to be positive (semi)definite, which ensures that Cholesky decomposition can be performed on the variance matrix in our optimization algorithm to efficiently optimize the model likelihood.</p></section><section id="gepi70015-sec-0060"><h3 class="pmc_sec_title">2.4. Genome‐Wide Multiple Testing Adjustment</h3>
<p>To access the significance of test results in IBD mapping scan across the genome, correcting for multiple testing is essential. The commonly used Bonferroni threshold would be conservative for IBD mapping tests due to correlations between tests at nearby locations that are covered by the same IBD segment. To obtain an appropriate genome‐wide significance threshold, we need to consider the joint distribution of test statistics when performing IBD mapping at a series of locations across the genome.</p>
<p>Consider standardized test statistics <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-91" display="inline"><mrow><mrow><mrow><msub><mrow><mo>{</mo><mi>Z</mi></mrow><mi>l</mi></msub><mo>}</mo></mrow></mrow></mrow></math></span> that are approximately normally distributed and follow an Ornstein‐Uhlenbeck (OU) process under the null hypothesis of no genetic associations. The mean <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-92" display="inline"><mrow><mrow><mrow><mi>E</mi><mrow><mo>[</mo><msub><mi>Z</mi><mi>l</mi></msub><mo>]</mo></mrow><mo>=</mo><mn>0</mn></mrow></mrow></mrow></math></span>, and the covariance between any pair of test statistics <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-93" display="inline"><mrow><mrow><mrow><msub><msub><mi>Z</mi><mi>l</mi></msub><mi>i</mi></msub></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-94" display="inline"><mrow><mrow><mrow><msub><msub><mi>Z</mi><mi>l</mi></msub><mi>j</mi></msub></mrow></mrow></mrow></math></span> is given by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-95" display="inline"><mrow><mrow><mrow><mtext>cov</mtext><mrow><mo>(</mo><mrow><msub><msub><mi>Z</mi><mi>l</mi></msub><mi>i</mi></msub><mo>,</mo><msub><msub><mi>Z</mi><mi>l</mi></msub><mi>j</mi></msub></mrow><mo>)</mo></mrow><mo>=</mo><mtext>exp</mtext><mrow><mo stretchy="false">(</mo><mrow><mo>−</mo><mi>α</mi><mrow><mo>|</mo><mrow><msub><mi>l</mi><mi>i</mi></msub><mo>−</mo><msub><mi>l</mi><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math></span>. (Lander and Botstein <a href="#gepi70015-bib-0035" class="usa-link" aria-describedby="gepi70015-bib-0035">1989</a>; Siegmund and Yakir <a href="#gepi70015-bib-0052" class="usa-link" aria-describedby="gepi70015-bib-0052">2007</a>; Grinde et al. <a href="#gepi70015-bib-0027" class="usa-link" aria-describedby="gepi70015-bib-0027">2019</a>). The parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-96" display="inline"><mrow><mrow><mrow><mi>α</mi><mo>&gt;</mo><mn>0</mn></mrow></mrow></mrow></math></span> is the rate of correlation decay as genomic distances increase between test statistics, and we refer to it as the decay parameter of the OU process.</p>
<p>We show in Appendix <a href="#gepi70015-app-0002" class="usa-link">B</a> that the IBD mapping statistics <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-97" display="inline"><mrow><mrow><mrow><msub><mrow><mo>{</mo><msub><mi>W</mi><mi>l</mi></msub><mo>}</mo></mrow><mrow><msub><mi>l</mi><mn>1</mn></msub><mo>:</mo><msub><mi>l</mi><mi>p</mi></msub></mrow></msub></mrow></mrow></mrow></math></span> for testing genomic locations <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-98" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mn>1</mn></msub><mo>,</mo><msub><mi>l</mi><mn>2</mn></msub><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><msub><mi>l</mi><mi>p</mi></msub></mrow></mrow></mrow></math></span> can be define in terms of standard normal <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-99" display="inline"><mrow><mrow><mrow><mrow><mo>{</mo><msub><mi>Z</mi><mi>l</mi></msub><mo>}</mo></mrow></mrow></mrow></mrow></math></span> as:
</p>
<table class="disp-formula p" id="gepi70015-disp-0006"><tr>
<td class="formula"><math id="jats-math-100" display="block"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub><mo>=</mo><mfenced open="{" close=""><mtable displaystyle="true" columnalign="left"><mtr><mtd><msubsup><mi>Z</mi><mi>l</mi><mn>2</mn></msubsup><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mi mathvariant="italic">if</mi><mspace width="1em"></mspace><msub><mi>Z</mi><mi>l</mi></msub><mo>&gt;</mo><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mi>i</mi><mi>f</mi><mspace width="1em"></mspace><msub><mi>Z</mi><mi>l</mi></msub><mo>≤</mo><mn>0</mn></mtd></mtr></mtable></mfenced><mo>,</mo></mrow></mrow></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>If the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-101" display="inline"><mrow><mrow><mrow><msub><mrow><mo>{</mo><msub><mi>Z</mi><mi>l</mi></msub><mo>}</mo></mrow><mrow><msub><mi>l</mi><mn>1</mn></msub><mo>:</mo><msub><mi>l</mi><mi>p</mi></msub></mrow></msub></mrow></mrow></mrow></math></span>form an OU process, then we refer to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-102" display="inline"><mrow><mrow><mrow><msub><mrow><mo>{</mo><msub><mi>W</mi><mi>l</mi></msub><mo>}</mo></mrow><mrow><msub><mi>l</mi><mn>1</mn></msub><mo>:</mo><msub><mi>l</mi><mi>p</mi></msub></mrow></msub></mrow></mrow></mrow></math></span> as the modified OU process. We expect that the IBD mapping statistics will be approximately memoryless as a process along the genome, so that the modified OU process may provide a good approximation to the joint distribution of the test statistics. In what follows, we assume that the IBD mapping statistics do indeed follow a modified OU process.</p>
<p>Define <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-103" display="inline"><mrow><mrow><mrow><mi>d</mi><mo>=</mo><mrow><mo>|</mo><mrow><msub><mi>l</mi><mi>i</mi></msub><mo>−</mo><msub><mi>l</mi><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-104" display="inline"><mrow><mrow><mrow><mi>ρ</mi><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow><mo>=</mo><mtext>exp</mtext><mrow><mo>(</mo><mrow><mo>−</mo><mi>α</mi><mi>d</mi></mrow><mo>)</mo></mrow><mo>.</mo></mrow></mrow></mrow></math></span>It can be shown that at any locations <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-105" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-106" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow></mrow></mrow></math></span> that are spaced at intervals of length <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-107" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span>, the correlation between <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-108" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-109" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow></mrow></mrow></math></span> depends only on the decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-110" display="inline"><mrow><mrow><mrow><mi>α</mi></mrow></mrow></mrow></math></span> and the distance <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-111" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span> (Appendix <a href="#gepi70015-app-0002" class="usa-link">B</a>):
</p>
<table class="disp-formula p" id="gepi70015-disp-0007"><tr>
<td class="formula"><math id="jats-math-112" display="block" linebreak="nobreak" mathsize="8pt"><mrow><mrow><mrow><mi>f</mi><mo>(</mo><mi>α</mi><mo>,</mo><mi>d</mi><mo>)</mo><mi mathvariant="normal">≔</mi><mi mathvariant="italic">Corr</mi><mrow><mo>(</mo><mrow><msub><mi>W</mi><msub><mi>l</mi><mn>1</mn></msub></msub><mo>,</mo><msub><mi>W</mi><msub><mi>l</mi><mn>2</mn></msub></msub></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>5</mn></mfrac><mfenced close="}" open="{"><mfrac><mrow><mn>2</mn><mi>π</mi><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow><mo>+</mo><mn>6</mn><mi>ρ</mi><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow></mrow></msqrt><mo>+</mo><mo>(</mo><mn>2</mn><mo>+</mo><mn>4</mn><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow><mo>)</mo><msup><mi>sin</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mi>ρ</mi><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow></mrow></mrow><mi>π</mi></mfrac></mfenced><mo>.</mo></mrow></mrow></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>Based on (6), we can estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-113" display="inline"><mrow><mrow><mrow><mi>α</mi></mrow></mrow></mrow></math></span> from observed correlations between LOD scores under the null hypothesis. To do so, we simulate phenotype values under the null hypothesis and use pairs of LOD scores obtained at any two locations spaced at distance <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-114" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span> to calculate the empirical correlation <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-115" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>f</mi><mo>ˆ</mo></mover><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></span>, and then estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-116" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>ρ</mi><mo>ˆ</mo></mover><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></span> by solving <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-117" display="inline"><mrow><mrow><mrow><mi>f</mi><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow><mo>−</mo><mover accent="true"><mi>f</mi><mo>ˆ</mo></mover><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow><mo>=</mo><mn>0</mn><mspace width="0.25em"></mspace></mrow></mrow></mrow></math></span>for a given <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-118" display="inline"><mrow><mrow><mrow><mi>d</mi><mo>.</mo></mrow></mrow></mrow></math></span> Since <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-119" display="inline"><mrow><mrow><mrow><mi>ρ</mi><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow><mo>=</mo><mtext>exp</mtext><mrow><mo>(</mo><mrow><mo>−</mo><mi>α</mi><mi>d</mi></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></span>, we have <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-120" display="inline"><mrow><mrow><mrow><mo>−</mo><mi>log</mi><mrow><mo stretchy="false">(</mo><mrow><mi>ρ</mi><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><mi>α</mi><mi>d</mi><mo>.</mo></mrow></mrow></mrow></math></span> Hence, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-121" display="inline"><mrow><mrow><mrow><mi>α</mi></mrow></mrow></mrow></math></span> can be estimated by the slope of a linear regression line without intercept between <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-122" display="inline"><mrow><mrow><mrow><mo>−</mo><mi>log</mi><mrow><mo stretchy="false">(</mo><mrow><mover accent="true"><mi>ρ</mi><mo>ˆ</mo></mover><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>d</mi></mrow><mo>)</mo></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-123" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span> at a series of different values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-124" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span>.</p>
<p>Given <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-125" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span>, we implemented a Monte Carlo approach to find the significance threshold for genome‐wide multiple testing based on the distribution for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-126" display="inline"><mrow><mrow><mrow><munder><mi>max</mi><mi>l</mi></munder><mo>{</mo><msub><mi>W</mi><mi>l</mi></msub><mo>}</mo></mrow></mrow></mrow></math></span>. We simulate observations <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-127" display="inline"><mrow><mrow><mrow><mrow><mo stretchy="false">{</mo><msubsup><mi>W</mi><mi>l</mi><mo accent="true">′</mo></msubsup><mo stretchy="false">}</mo></mrow></mrow></mrow></mrow></math></span> from the modified OU process, starting from <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-128" display="inline"><mrow><mrow><mrow><mi>l</mi><mo>=</mo><mn>0</mn></mrow></mrow></mrow></math></span> and incrementing <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-129" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> by the same space between test locations selected in genome‐wide IBD mapping, until <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-130" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span> reaches the end position on the genome. The simulated process <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-131" display="inline"><mrow><mrow><mrow><mrow><mo stretchy="false">{</mo><msubsup><mi>W</mi><mi>l</mi><mo accent="true">′</mo></msubsup><mo stretchy="false">}</mo></mrow></mrow></mrow></mrow></math></span> represents a single replicate of the genome‐wide IBD mapping test under the null hypothesis. An empirical distribution for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-132" display="inline"><mrow><mrow><mrow><munder><mi>max</mi><mi>l</mi></munder><mo>{</mo><msub><mi>W</mi><mi>l</mi></msub><mo>}</mo></mrow></mrow></mrow></math></span> can be obtained by repeating the simulation thousands of times and recording <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-133" display="inline"><mrow><mrow><mrow><munder accentunder="false"><mi>max</mi><mi>l</mi></munder><mo form="prefix" stretchy="false">{</mo><msubsup><mi>W</mi><mi>l</mi><mo lspace="0em" rspace="0em" accent="true">′</mo></msubsup><mo form="postfix" stretchy="false">}</mo></mrow></mrow></mrow></math></span> from each replicate. We take the 95% quantile from this empirical distribution as the 95% significance threshold of genome‐wide IBD mapping on the given data. The detailed algorithm to simulate the modified OU process along the genome is given in Appendix <a href="#gepi70015-app-0003" class="usa-link">C</a>. We note that the genome‐wide multiple testing correction depends on the specific settings used to detect multi‐individual IBD, because the length distribution of IBD haplotypes considered in the algorithm will affect the covariance between test statistics.</p></section><section id="gepi70015-sec-0070"><h3 class="pmc_sec_title">2.5. Analysis Pipeline</h3>
<p>The required input data for our IBD mapping test include phased genotype sequence or array data, a quantitative trait of interest, fixed‐effect covariates if applicable, and a genetic map.</p>
<p>As the first step, we construct a global IBD matrix and location‐specific relatedness matrices. For the global IBD matrix, we use kinship coefficients that are estimated by the IBDkin program based on pairwise IBD segments identified from the sample using the hap‐ibd program, as described above. For the hap‐ibd analysis on sequence data, we set the minimum seed length to 0.5 cM, the minimum extension length to 0.2 cM, and the minimum output segment length to 2 cM. We also exclude rare variants by setting the minimum minor allele count filter to 100. For the hap‐ibd analysis on SNP array data, due to the lower marker density, we set the minimum seed length to 1 cM, the minimum extension length to 0.1 cM, the minimum number of markers in a seed IBS segment to 50, and the minimum output segment length to 3 cM. Other parameters are left at their default values. When estimating kinship coefficients, we use all default parameters of the IBDkin program.</p>
<p>Next, we run the ibd‐cluster software to obtain multi‐individual IBD clusters. In our simulation studies, we applied different haplotype length thresholds at 2 cM or 3 cM and different trimming thresholds at 0.25 cM, 0.5 cM, or 1 cM to investigate how different IBD clustering thresholds affect the performance of our test. When analyzing UK Biobank data, we set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-134" display="inline"><mrow><mrow><mrow><mi>L</mi><mo>=</mo><mn>2</mn><mspace width="0.25em"></mspace><mtext>cM</mtext></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-135" display="inline"><mrow><mrow><mrow><mi>T</mi><mo>=</mo><mn>0.5</mn><mspace width="0.25em"></mspace><mtext>cM</mtext></mrow></mrow></mrow></math></span>, which is the setting that we found through our simulation studies to give good power while producing reasonably sparse local IBD matrices that are computationally feasible for large‐scale data.</p>
<p>We then select a collection of locations across the genome as candidates to compute the local IBD matrices and perform the IBD mapping test. The test locations do not need to be directly available in the array data or in the IBD cluster output, as we can simply use the nearest location to the testing position in the IBD cluster output to perform the test. As IBD states do not change immediately across nearby markers, we recommend running tests at 0.1 cM intervals when considering IBD clusters based on haplotypes that are at least 2 cM long.</p>
<p>For the simulation studies, and to estimate the genome‐wide multiple testing threshold in the UK Biobank data, we ran IBD mapping tests at 0.1 cM intervals. When analyzing phenotypes from the UK Biobank, to save computational resources given the large sample size, we chose a two‐step approach. On each chromosome, we first ran IBD mapping test at locations spaced 1 cM apart across the genome, and we identified the top 10 locations ranked by ascending test <em>p</em>‐values. We then zoomed into 1 cM regions centered around these locations by conducting finer‐scale tests at 0.1 cM intervals.</p>
<p>Our method is also applicable to genomic regions in addition to single genomic locations. We compute the local IBD matrix over a genomic region by averaging the local IBD matrices computed at a few selected locations within the given region. In the simulation studies for power estimation, we tested each 0.05 cM region containing simulated causal variants using the local IBD matrix computed by averaging the IBD matrices at the start and the end positions of the region.</p>
<p>Finally, we compare test <em>p</em>‐values to the genome‐wide significance threshold. To derive the multiple testing adjustment for genome‐wide IBD mapping, we first estimate the decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-136" display="inline"><mrow><mrow><mrow><mi>α</mi></mrow></mrow></mrow></math></span> of the modified OU process formed by IBD mapping statistics across the genome under the null hypothesis, as described above. Null phenotypes are simulated as the sum of independent random variables drawn from the standard normal distribution and random variables drawn from the multivariate normal distribution with mean 0 and covariance matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-137" display="inline"><mrow><mrow><mrow><mover accent="true"><mi mathvariant="bold">Ψ</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-138" display="inline"><mrow><mrow><mrow><mover accent="true"><mi mathvariant="bold">Ψ</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> is the global IBD matrix estimated from the data. We conduct IBD mapping tests at 0.1 cM intervals across the genome and calculate the empirical correlation <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-139" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>f</mi><mo>ˆ</mo></mover><mo>(</mo><mi>α</mi><mo>,</mo><mi>d</mi><mo>)</mo></mrow></mrow></mrow></math></span> between pairs of test statistics spaced at <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-140" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span> cM for a sequence of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-141" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span> from 0.1 cM to 1 cM, spacing at 0.1 cM intervals. We then estimated <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-142" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>ρ</mi><mo>ˆ</mo></mover><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> from <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-143" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>f</mi><mo>ˆ</mo></mover><mo>(</mo><mi>α</mi><mo>,</mo><mi>d</mi><mo>)</mo></mrow></mrow></mrow></math></span> based on Equation (<a href="#gepi70015-disp-0007" class="usa-link">6</a>) for each <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-144" display="inline"><mrow><mrow><mrow><mi>d</mi><mo>.</mo></mrow></mrow></mrow></math></span> Finally, we fit a regression on <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-145" display="inline"><mrow><mrow><mrow><mo>−</mo><mi>log</mi><mi mathvariant="normal">⁡</mi><mo>(</mo><mover accent="true"><mi>ρ</mi><mo>ˆ</mo></mover><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo>)</mo></mrow></mrow></mrow></math></span> against <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-146" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span> and use the slope of the fitted line as the estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-147" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> for the decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-148" display="inline"><mrow><mrow><mrow><mi>α</mi><mo>.</mo></mrow></mrow></mrow></math></span>
</p>
<p>We perform a bootstrap analysis across chromosomes to obtain confidence intervals for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-149" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span>. Specifically, we set the number of bootstrap replicates to 10,000, and we resample chromosomes with replacement in each replicate. We take all pairs of test statistics spaced at <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-150" display="inline"><mrow><mrow><mrow><mi>d</mi></mrow></mrow></mrow></math></span> cM within each chromosome, and combine pairs over all sampled chromosomes to calculate the sample correlations and estimate the decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-151" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover><mo>.</mo></mrow></mrow></mrow></math></span> We calculate the 2.5th and 97.5th percentile of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-152" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> across all bootstrap replicates to construct the 95% confidence interval for the estimated decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-153" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover><mo>.</mo></mrow></mrow></mrow></math></span>
</p>
<p>We then use <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-154" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> as the decay parameter to simulate observations at 0.1 cM intervals from a modified OU process that spans the same distance as the total size of the genome for 10,000 replicates, and we take the 95% quantile of maximum test statistics from all simulations as the genome‐wide significance threshold.</p></section><section id="gepi70015-sec-0080"><h3 class="pmc_sec_title">2.6. Studies on Simulated Data</h3>
<p>We used msprime (Kelleher and Lohse <a href="#gepi70015-bib-0034" class="usa-link" aria-describedby="gepi70015-bib-0034">2020</a>; Baumdicker et al. <a href="#gepi70015-bib-0004" class="usa-link" aria-describedby="gepi70015-bib-0004">2022</a>) to simulate whole‐genome sequence data for 5000 individuals, with each genome consisting of 30 chromosomes, all 100 cM in length. The demographic model for the simulation, adopted from previous studies, resembles the demographic history of the UK population (Browning and Browning <a href="#gepi70015-bib-0011" class="usa-link" aria-describedby="gepi70015-bib-0011">2013</a>). In the msprime simulations, mutation occurred at a rate of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-155" display="inline"><mrow><mrow><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>8</mn></mrow></msup></mrow></mrow></mrow></math></span> per base pair per generation, and recombination at a rate of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-156" display="inline"><mrow><mrow><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>8</mn></mrow></msup></mrow></mrow></mrow></math></span> per base pair per meiosis.</p>
<p>We estimated the genome‐wide type I error rate in the simulated sequence data by simulating 1000 replicates of phenotypes without genetic associations on random samples of 1000 individuals from the original dataset. On the simulated sequence data, we generated phenotypes following <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-157" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">Y</mi><mo>=</mo><mi mathvariant="bold-italic">g</mi><mo>+</mo><mi mathvariant="bold-italic">ε</mi></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-158" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow></mrow></mrow></math></span> follows a multivariate <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-159" display="inline"><mrow><mrow><mrow><mi>N</mi><mo>(</mo><mn mathvariant="bold-italic">0</mn><mo>,</mo><mover accent="true"><mi mathvariant="bold">Ψ</mi><mo>ˆ</mo></mover><mo>)</mo></mrow></mrow></mrow></math></span> distribution, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-160" display="inline"><mrow><mrow><mrow><mover accent="true"><mi mathvariant="bold">Ψ</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> being the genome‐wide relatedness matrix constructed from estimated kinship coefficients of all pairs of individuals in the sample, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-161" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></mrow></math></span> is a vector of standard normal observations. For each replicate, we performed genome‐wide IBD mapping at 0.1 cM intervals and obtained a genome‐wide significance threshold based on the observed test scores. We calculated the genome‐wide type I error rate as the proportion of replicates that contain at least one test score achieving the corresponding genome‐wide significance threshold.</p>
<p>We estimate the power of our test by simulating phenotypes associated with four types of causal variants using simulated sequence data. Variants in the sequence data are classified based on minor allele frequencies (MAF) as common (MAF &gt; 10%), low‐frequency (1% &lt; MAF &lt; 10%), rare (0.05% &lt; MAF &lt; 1%), and ultra‐rare (MAF &lt; 0.05% but appearing at least once in the data). We simulate 1000 replicates of phenotypes associated with a 0.05 cM region containing <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-162" display="inline"><mrow><mrow><mrow><mi>k</mi></mrow></mrow></mrow></math></span> causal variants in each of these four classes.</p>
<p>For phenotypes associated with common or low‐frequency variants, we consider regions that contain at least 4 variants of the target class, and designate half of them as causal. For rare and ultra‐rare variants, we consider regions that contain at least 8 variants of the target class, and randomly select a quarter of them as causal. The simulated phenotype for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-163" display="inline"><mrow><mrow><mrow><msup><mi>i</mi><mi mathvariant="italic">th</mi></msup></mrow></mrow></mrow></math></span> individual is constructed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-164" display="inline"><mrow><mrow><mrow><msub><mi>Y</mi><mi>i</mi></msub><mo>=</mo><msub><mi>g</mi><mi>i</mi></msub><mo>+</mo><msubsup><mrow><msubsup><mo>∑</mo><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mi>k</mi></msubsup><msub><mi>θ</mi><mi>l</mi></msub><mi>q</mi></mrow><mi>l</mi><mrow><mo>(</mo><mi>i</mi><mo>)</mo></mrow></msubsup><mo>+</mo><msub><mi>ε</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-165" display="inline"><mrow><mrow><mrow><msub><mi>g</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-166" display="inline"><mrow><mrow><mrow><msub><mi>ε</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span> are the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-167" display="inline"><mrow><mrow><mrow><msup><mi>i</mi><mi mathvariant="italic">th</mi></msup></mrow></mrow></mrow></math></span> observations of the vectors <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-168" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-169" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></mrow></math></span> that are defined as above, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-170" display="inline"><mrow><mrow><mrow><msubsup><mi>q</mi><mi>l</mi><mrow><mo>(</mo><mi>i</mi><mo>)</mo></mrow></msubsup><mo>=</mo><mn>0</mn><mo>,</mo><mn>1</mn></mrow></mrow></mrow></math></span> or 2 is the number of copies of the minor allele at marker <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-171" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span>, and the effect size of each causal marker depends on its minor allele frequency as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-172" display="inline"><mrow><mrow><mrow><msub><mi>θ</mi><mi>l</mi></msub><mo>=</mo><msqrt><mrow><mn>0.05</mn><mo>/</mo><mo>(</mo><mn>2</mn><mi mathvariant="italic">MA</mi><msub><mi>F</mi><mi>l</mi></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi mathvariant="italic">MA</mi><msub><mi>F</mi><mi>l</mi></msub></mrow><mo>)</mo></mrow><mo>)</mo></mrow></msqrt></mrow></mrow></mrow></math></span>.</p>
<p>From the simulated sequence data, we generate a simulated SNP array dataset for each chromosome by excluding all variants with MAF below 1% and all variants associated with the simulated phenotypes, and then randomly selecting 30,000 variants per chromosome. We evaluate the power of our test to detect variants associated with the simulated phenotypes using both the full simulated sequence data, where all genetic variants are directly available, and the simulated SNP array data, where none of the causal variants are directly present.</p>
<p>The power of our IBD mapping test is calculated using both the Bonferroni threshold (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-173" display="inline"><mrow><mrow><mrow><mn>0.05</mn><mo>/</mo><mn>30000</mn><mo>=</mo><mn>1.7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup><mo>)</mo></mrow></mrow></mrow></math></span> and the genome‐wide significance threshold derived from our proposed multiple testing adjustment. On simulated sequence data, we also conducted the SKAT (Wu et al. <a href="#gepi70015-bib-0060" class="usa-link" aria-describedby="gepi70015-bib-0060">2011</a>) test using default parameter settings to benchmark the performance of our IBD mapping test. On the SNP array data, we compare the performance of our test to the traditional single‐variant test used in GWAS. Within each 0.05 cM region containing causal variants, we test every marker in the SNP data using simple linear regression between the phenotype value and the copy number of minor alleles. The minimum <em>p</em>‐value across all tests within a region is used as the overall single‐variant test <em>p</em>‐value for that region. For the SKAT test, power is calculated using the significance threshold <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-174" display="inline"><mrow><mrow><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span>, as recommended in Wu et al. (<a href="#gepi70015-bib-0060" class="usa-link" aria-describedby="gepi70015-bib-0060">2011</a>) for genome‐wide testing (Wu et al. <a href="#gepi70015-bib-0060" class="usa-link" aria-describedby="gepi70015-bib-0060">2011</a>). For the single‐variant test, power is calculated using the traditional GWAS significance threshold <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-175" display="inline"><mrow><mrow><mrow><mn>5</mn><msup><mrow><mo>×</mo><mn>10</mn></mrow><mrow><mo>−</mo><mn>8</mn></mrow></msup></mrow></mrow></mrow></math></span>, which is approximately 0.05 divided by the total number of markers in our simulated SNP array data. We note that in the simulated data, it is not necessary to account for relatedness in the single‐variant and SKAT tests because there is no real relatedness among simulated individuals; each chromosome was simulated independently in msprime, and therefore relatedness is not preserved across the genome.</p></section><section id="gepi70015-sec-0090"><h3 class="pmc_sec_title">2.7. UK Biobank Data Analysis</h3>
<p>The UK Biobank SNP array data were collected on half a million UK participants using the UK Biobank Axiom array that assays approximately 850,000 genetic variants across genome (Bycroft et al. <a href="#gepi70015-bib-0017" class="usa-link" aria-describedby="gepi70015-bib-0017">2018</a>). We analyzed the systolic blood pressure of White British UK Biobank individuals.</p>
<p>We analyzed 124,376 White British individuals with no missing data in age, sex, the top 10 genetic PCs, the history of medication, and two measurements of systolic blood pressure at the initial assessment visit. The final phenotypic values for systolic blood pressure are obtained as the mean of the two measurements at initial assessment visit and adjusted for medication record. For individuals with a history of taking medications to control blood pressure, we increase their systolic blood pressure by 15, as suggested by previous genetic association studies on blood pressure in the UK Biobank White British cohort (Warren et al. <a href="#gepi70015-bib-0057" class="usa-link" aria-describedby="gepi70015-bib-0057">2017</a>). We used the GRCh37 deCODE map developed by Bhérer et al. (<a href="#gepi70015-bib-0006" class="usa-link" aria-describedby="gepi70015-bib-0006">2017</a>) in the analyses of the UK Biobank data (Bhérer et al. <a href="#gepi70015-bib-0006" class="usa-link" aria-describedby="gepi70015-bib-0006">2017</a>).</p>
<p>We first estimated the genome‐wide type I error rate when applying our IBD mapping test on the UK Biobank dataset. We selected random subsets of 1000 individuals and simulated 1000 replicates of phenotypes with no genetic associations using the model <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-176" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">Y</mi><mo>=</mo><mn>0.05</mn><mo>*</mo><mi mathvariant="italic">age</mi><mo>+</mo><mn>0.5</mn><mo>*</mo><mi mathvariant="italic">sex</mi><mo>+</mo><mi mathvariant="bold-italic">g</mi><mo>+</mo><mi mathvariant="bold-italic">ε</mi></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-177" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-178" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">ε</mi></mrow></mrow></mrow></math></span> are defined in the same way as the studies on the simulated data, and observed age and sex are included as fixed effects. For each replicate, we conducted genome‐wide IBD mapping at 0.1 cM intervals and derived the genome‐wide significance threshold based on the random subset of 1000 individuals. We calculated the genome‐wide type I error rates based on results of 1000 replicates as described previously.</p>
<p>Next, to analyze the genetic association of systolic blood pressure among White British individuals in UK Biobank, we conducted IBD mapping tests adjusted for age, sex, age squared, their interactions, and the first 10 genetic principal components as fixed effects in the model. We used 2 cM as the haplotype length threshold and 0.5 cM as the trimming threshold in multi‐individual IBD clustering. We assessed the test results against the genome‐wide multiple testing threshold derived from the genotype data of all 124,376 individuals in the analysis.</p>
<p>We also analyzed the data using FiMAP, a recently proposed IBD mapping test that employs the same variance component model but utilizes a score‐type statistic for testing the variance component hypothesis with pairwise IBD segments (Chen et al. <a href="#gepi70015-bib-0019" class="usa-link" aria-describedby="gepi70015-bib-0019">2023</a>). To balance power and computational efficiency, FiMAP recommends an IBD threshold of 3 cM because using a shorter threshold (such as 2 cM) would substantially increase the number of IBD segments, resulting in denser local IBD matrices and significantly higher computational demands (Chen et al. <a href="#gepi70015-bib-0019" class="usa-link" aria-describedby="gepi70015-bib-0019">2023</a>). Following the analysis procedure outlined in Chen et al. (<a href="#gepi70015-bib-0019" class="usa-link" aria-describedby="gepi70015-bib-0019">2023</a>), we used hap‐ibd to identify IBD segments of at least 3 cM in length and performed the FiMAP test at 3,363 consecutive, nonoverlapping 1 cM regions across the genome, adjusting the same set of fixed effects as in our IBD mapping test. FiMAP <em>p</em>‐values are evaluated against a Bonferroni‐adjusted significance threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-179" display="inline"><mrow><mrow><mrow><mn>0.05</mn><mo>/</mo><mn>3363</mn><mo>=</mo><mn>1.5</mn><msup><mrow><mo>×</mo><mn>10</mn></mrow><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></mrow></math></span>.</p></section></section><section id="gepi70015-sec-0100"><h2 class="pmc_sec_title">3. Results</h2>
<section id="gepi70015-sec-0110"><h3 class="pmc_sec_title">3.1. Genome‐Wide Multiple‐Testing Correction</h3>
<p>We derived the genome‐wide multiple testing threshold for IBD mapping tests on the simulated sequence data and the simulated SNP array data using different haplotype length and trimming thresholds for multi‐individual IBD clustering. For each test scenario, the estimated decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-180" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> of the modified OU process (see Methods) is shown in Supporting Information S1: Figure <a href="#gepi70015-suppl-0001" class="usa-link">S1</a> along with its 95% bootstrap confidence interval. The 95% genome‐wide multiple testing thresholds for test <em>p</em>‐values are summarized in Table <a href="#gepi70015-tbl-0001" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="gepi70015-tbl-0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Estimated decay parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-181" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> and the corresponding genome‐wide 95% significance threshold for test p‐values under different haplotype length thresholds (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-182" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">L</mi></mrow></mrow></mrow></math></span>) and trimming thresholds (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-183" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">T</mi></mrow></mrow></mrow></math></span>) for multi‐individual IBD detection.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom">
<tr valign="bottom">
<th align="left" colspan="2" valign="bottom" rowspan="1">
<table class="disp-formula p" id="jats-math-184_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-184" display="inline"><mrow><mrow><mrow><mi>L</mi></mrow></mrow></mrow></math></span></td></tr></table>
</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="center" valign="bottom" rowspan="1">2 cM</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="center" valign="bottom" rowspan="1">3 cM</th>
</tr>
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="left" colspan="2" valign="bottom" rowspan="1">
<table class="disp-formula p" id="jats-math-185_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-185" display="inline"><mrow><mrow><mrow><mi>T</mi></mrow></mrow></mrow></math></span></td></tr></table>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.25 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.5 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.25 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.5 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1 cM</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td rowspan="2" align="left" valign="top" colspan="1">Simulated sequence data</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-186_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-186" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.03</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.17</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.29</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.39</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.46</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.72</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">Threshold</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-187_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-187" display="inline"><mrow><mrow><mrow><mn>2.9</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-188_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-188" display="inline"><mrow><mrow><mrow><mn>2.8</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-189_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-189" display="inline"><mrow><mrow><mrow><mn>2.7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-190_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-190" display="inline"><mrow><mrow><mrow><mn>5.2</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-191_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-191" display="inline"><mrow><mrow><mrow><mn>4.7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-192_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-192" display="inline"><mrow><mrow><mrow><mn>3.5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="top" colspan="1">Simulated SNP array data</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-193_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-193" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.02</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.14</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.3</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.41</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.48</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">Threshold</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-194_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-194" display="inline"><mrow><mrow><mrow><mn>3.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-195_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-195" display="inline"><mrow><mrow><mrow><mn>2.8</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-196_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-196" display="inline"><mrow><mrow><mrow><mn>2.7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-197_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-197" display="inline"><mrow><mrow><mrow><mn>5.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-198_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-198" display="inline"><mrow><mrow><mrow><mn>4.5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-199_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-199" display="inline"><mrow><mrow><mrow><mn>3.5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70015-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>For each simulated test scenario, the estimated <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-200" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> from the simulated sequence data closely matched those from the simulated SNP array data. Using a 3 cM haplotype length threshold for detecting multi‐individual IBD resulted in smaller <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-201" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> compared to using a 2 cM threshold. At the same haplotype length threshold, increasing the trimming threshold for IBD clustering led to higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-202" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> estimates. Larger <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-203" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> values corresponded to smaller genome‐wide multiple‐testing thresholds for <em>p</em>‐values.</p>
<p>We calculated the theoretical correlations between random variables in each modified OU process using Equation (<a href="#gepi70015-disp-0007" class="usa-link">6</a>) and the estimated <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-204" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> for each test scenario. Across all test scenarios, the observed correlations between test statistics were consistent with the theoretical correlations of the modified OU process for both the simulated sequence data (Supporting Information S1: Figure <a href="#gepi70015-suppl-0001" class="usa-link">S2</a>) and the simulated SNP data (Supporting Information S1: Figure <a href="#gepi70015-suppl-0001" class="usa-link">S3</a>), especially for pairs of test statistics less than 1 cM apart. These results confirm that the modified OU process provides a reliable approximation to the correlation structure of the IBD mapping test statistics.</p>
<p>For the UK Biobank SNP array data, we estimated the genome‐wide multiple testing threshold for IBD mapping tests using a haplotype length threshold of 2 cM and a trimming threshold of 0.5 cM in multi‐individual IBD clustering. The estimated decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-205" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover></mrow></mrow></mrow></math></span> of the corresponding modified OU process was 1.62, with a 95% bootstrap confidence interval of 1.35 and 1.92. The 95% bootstrap confidence interval of the observed correlations overlapped with the trajectory of theoretical correlation of the modified OU process with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-206" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>α</mi><mo>ˆ</mo></mover><mo>=</mo><mn>1.62</mn></mrow></mrow></mrow></math></span> (Supporting Information S1: Figure <a href="#gepi70015-suppl-0001" class="usa-link">S4</a>). The estimated multiple‐testing <em>p</em>‐value threshold for a 5% genome‐wide type I error rate was <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-207" display="inline"><mrow><mrow><mrow><mn>2.3</mn><msup><mrow><mo>×</mo><mn>10</mn></mrow><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span>.</p></section><section id="gepi70015-sec-0120"><h3 class="pmc_sec_title">3.2. Genome‐Wide Type I Error Rates</h3>
<p>Table <a href="#gepi70015-tbl-0002" class="usa-link">2</a> summarizes the estimated genome‐wide type I error rates of our IBD mapping test using the proposed genome‐wide multiple testing adjustment on both the simulated sequence data and the UK Biobank SNP array data when different haplotype length and trimming thresholds are used for multi‐individual IBD detection. As a comparison, we also calculated the proportion of genome‐wide IBD mapping replicates where at least one test exceeded the significance threshold based on the Bonferroni correction, which is <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-208" display="inline"><mrow><mrow><mrow><mn>0.05</mn><mo>/</mo><mn>30000</mn></mrow></mrow></mrow></math></span> for IBD mapping tests on the simulated sequence data and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-209" display="inline"><mrow><mrow><mrow><mn>0.05</mn><mo>/</mo><mn>33498</mn></mrow></mrow></mrow></math></span> for tests on the UK Biobank data.</p>
<section class="tw xbox font-sm" id="gepi70015-tbl-0002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Genome‐wide type I error rates for the proposed genome‐wide multiple‐testing adjustment and the Bonferroni adjustment for IBD mapping tests at a genome‐wide significance level of 0.05. Results are based on simulated phenotypes under the null hypothesis of no genetic association, using both the simulated sequence data and the UK Biobank SNP‐array data, with different haplotype length thresholds (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-210" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">L</mi></mrow></mrow></mrow></math></span>) and trimming thresholds (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-211" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">T</mi></mrow></mrow></mrow></math></span>) for multi‐individual IBD detection.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom">
<tr valign="bottom">
<th align="left" colspan="2" valign="bottom" rowspan="1">
<table class="disp-formula p" id="jats-math-212_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-212" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">L</mi></mrow></mrow></mrow></math></span></td></tr></table>
</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="center" valign="bottom" rowspan="1">2 cM</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="center" valign="bottom" rowspan="1">3 cM</th>
</tr>
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="left" colspan="2" valign="bottom" rowspan="1">
<table class="disp-formula p" id="jats-math-213_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-213" display="inline"><mrow><mrow><mrow><mi>T</mi></mrow></mrow></mrow></math></span></td></tr></table>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.25 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.5 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.25 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.5 cM</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1 cM</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td rowspan="2" align="left" valign="top" colspan="1">Simulated data</td>
<td align="center" valign="top" rowspan="1" colspan="1">Proposed adjustment</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.046</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.057</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.065</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.049</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.064</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.059</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">Bonferroni adjustment</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.03</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.04</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.046</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.019</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.026</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.032</td>
</tr>
<tr>
<td rowspan="2" align="left" valign="top" colspan="1">UK Biobank SNP‐array data</td>
<td align="center" valign="top" rowspan="1" colspan="1">Proposed adjustment</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.04</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.035</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.052</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.065</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.048</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.038</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">Bonferroni adjustment</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.028</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.023</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.033</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.024</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.026</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.018</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70015-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In most test scenarios, our proposed genome‐wide multiple‐testing adjustment maintains a well‐controlled genome‐wide type I error rate close to the nominal level of 5%. In contrast, the genome‐wide type I error rates based on the Bonferroni threshold are lower than the nominal level, especially with a 3 cM haplotype length threshold, indicating that as expected the Bonferroni correction is conservative.</p></section><section id="gepi70015-sec-0130"><h3 class="pmc_sec_title">3.3. Power</h3>
<p>The power of our IBD mapping test to detect different types of causal variants, calculated using the corresponding genome‐wide significance threshold from Table <a href="#gepi70015-tbl-0002" class="usa-link">2</a> under various parameter settings for multi‐individual IBD clustering, is compared in Figure <a href="#gepi70015-fig-0001" class="usa-link">1</a>. Our IBD mapping test demonstrates similar power to detect all classes of causal variants when applied to both the simulated sequence data and the simulated SNP array data.</p>
<figure class="fig xbox font-sm" id="gepi70015-fig-0001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5b2d/12327188/886e2b3c9569/GEPI-49-0-g001.jpg" loading="lazy" id="jats-graphic-1" height="691" width="702" alt="Figure 1"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70015-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The power of our IBD mapping test for detecting common (&gt; 10% MAF), low‐frequency (1%–10% MAF), rare (0.05%‐1% MAF), and ultra‐rare (&lt; 0.05% MAF) causal variants using the proposed genome‐wide multiple testing threshold. On the simulated sequence data, we compared the IBD mapping test to the sequence kernel association test (SKAT). On the simulated SNP array data, we compared the IBD mapping test to the traditional single‐variant test used in GWAS. The tick labels on the <em>x</em>‐axis indicate the haplotype length threshold (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-214" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">L</mi></mrow></mrow></mrow></math></span>) and the trimming threshold (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-215" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">T</mi></mrow></mrow></mrow></math></span>) used for multi‐individual IBD detection in the corresponding IBD mapping test.</p></figcaption></figure><p>Across all parameter settings for multi‐individual IBD clustering, the highest power to detect causal variants is achieved for rare variants with MAFs between 0.05% and 1%. When using multi‐individual IBD detected from haplotypes of length at least 2 cM, trimmed by 0.25 cM or 0.5 cM, the IBD mapping test reaches over 97% power to detect rare variants associated with the simulated phenotypes in both the simulated sequence and SNP‐array data. These settings also yield good power (70%–80%) to detect common, low‐frequency, or ultra‐rare variants. Additionally, the IBD mapping test using a minimum haplotype length of 3 cM with trimming thresholds of 0.25 or 0.5 cM for multi‐individual IBD clustering also provides good power for detecting rare and ultra‐rare variants.</p>
<p>When using the simulated sequence data as the testing dataset, the IBD mapping test does not outperform the SKAT test, which achieved 90% power for detecting common causal variants and 100% power for detecting the other three types of causal variants. When performing tests on the simulated SNP array data, where none of the causal variants are directly available, the single‐variant test achieved higher power for detecting common variants (96%). However, the single‐variant test showed similar power to detect low‐frequency variants as the IBD mapping test, and it achieved only 11% power for detecting rare variants and completely failed to detect ultra‐rare variants. Across all parameter settings, the IBD mapping test significantly outperformed the single‐variant test in detecting rare and ultra‐rare causal variants. The highest power for detecting rare causal variants reaches 98.6% when using a haplotype threshold of 2 cM and a trimming threshold of 0.25 cM for multi‐individual IBD detection, and 78% for detecting ultra‐rare causal variants when a haplotype threshold of 3 cM and a trimming threshold of 0.25 cM are used for multi‐individual IBD clustering.</p>
<p>Lastly, we evaluated the power gain achieved by the proposed genome‐wide multiple‐testing adjustment. We calculated the power of our IBD mapping test using the naïve Bonferroni correction <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-216" display="inline"><mrow><mrow><mrow><mfrac><mn>0.05</mn><mn>30000</mn></mfrac><mo>=</mo><mn>1.7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span> and compared it to results obtained using our proposed genome‐wide multiple‐testing adjustment on both the simulated sequence data and the simulated SNP array data (Supporting Information S1: Figure <a href="#gepi70015-suppl-0001" class="usa-link">S5</a>). In most test scenarios, we observed an absolute increase of at least 2%–3% in power using our proposed multiple‐testing adjustment compared to using the Bonferroni correction on both the simulated sequence data and the simulated SNP data. Notably, when using a haplotype threshold of 3 cM and a trimming threshold of 0.25 cM for multi‐individual IBD clustering, the power to detect low‐frequency causal variants is improved by about 8% (from 45.8% to 53.7%) on simulated sequence data and by about 6% (from 47.0% to 52.8%) on simulated SNP array data using our proposed multiple‐testing adjustment.</p></section><section id="gepi70015-sec-0140"><h3 class="pmc_sec_title">3.4. UK Biobank Data Analysis</h3>
<p>For the genome‐wide scan of systolic blood pressure from more than 120k White British individuals in the UK Biobank (Figure <a href="#gepi70015-fig-0002" class="usa-link">2</a>), the IBD mapping test at 17.759 Mb on chromosome 19 (GRCh37) achieved the genome‐wide multiple‐testing threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-217" display="inline"><mrow><mrow><mrow><mn>2.3</mn><msup><mrow><mo>×</mo><mn>10</mn></mrow><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span> with <em>p</em>‐value = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-218" display="inline"><mrow><mrow><mrow><mn>2.21</mn><msup><mrow><mo>×</mo><mn>10</mn></mrow><mrow><mo>−</mo><mn>7</mn></mrow></msup></mrow></mrow></mrow></math></span>. In addition, a near‐significant signal is observed at 3.790 Mb on chromosome 22 (<em>p</em>‐value = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-219" display="inline"><mrow><mrow><mrow><mn>2.51</mn><msup><mrow><mo>×</mo><mn>10</mn></mrow><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></mrow></math></span>).</p>
<figure class="fig xbox font-sm" id="gepi70015-fig-0002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5b2d/12327188/d05be941c346/GEPI-49-0-g002.jpg" loading="lazy" id="jats-graphic-3" height="214" width="702" alt="Figure 2"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70015-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide IBD mapping on systolic blood pressure data from 124k White British individuals in the UK Biobank. The negative log10 transformed <em>p</em>‐values are plotted against tested positions on each chromosome along the genome. The result that achieved the genome‐wide multiple‐testing threshold is highlighted in red.</p></figcaption></figure><p>For comparison, we applied the FiMAP test to this data set at 1 cM intervals across the genome (Supporting Information S1: Figure <a href="#gepi70015-suppl-0001" class="usa-link">S6</a>). No significant signals were observed relative to the Bonferroni <em>p</em>‐value threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-220" display="inline"><mrow><mrow><mrow><mn>0.05</mn><mo>/</mo><mn>3363</mn><mo>=</mo><mn>1.5</mn><msup><mrow><mo>×</mo><mn>10</mn></mrow><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></mrow></math></span>.</p></section></section><section id="gepi70015-sec-0150"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>In this study, we developed an IBD mapping approach that leverages multi‐individual IBD sharing among distantly related individuals in large, outbred populations. Our approach constructs local relatedness matrices from multi‐individual IBD sharing and employs a likelihood ratio statistic to assess the contribution of local genetic similarities to the total phenotypic variation. To address genome‐wide multiple‐testing, we proposed a <em>p</em>‐value adjustment obtained by modeling the correlations between test statistics across the genome.</p>
<p>Through simulation studies, we evaluated the performance of our IBD mapping test across different types of testing datasets and a range of parameter settings for multi‐individual IBD clustering. Our results demonstrated that when appropriate haplotype length thresholds and trimming thresholds are chosen for multi‐individual IBD detection, the IBD mapping test maintains a well‐controlled genome‐wide type I error rate and exhibits robust power to detect various types of causal variants, with particular strength in detecting rare or untyped variants compared to traditional GWAS single‐variant tests. The power of IBD mapping was similar when applied to both SNP array and sequence data, making it a versatile approach that is suitable for use with either type of dataset. The proposed genome‐wide multiple testing adjustment further improved power for our IBD mapping test compared to the Bonferroni threshold.</p>
<p>However, in our simulation study, the IBD mapping test did not outperform the SKAT test when applied to sequence data in which the causal variants were genotyped. It was also less powerful than single‐variant GWAS for variants with MAF &gt; 10% on SNP array data. These findings emphasize that the strengths of our method lie in its complementary nature rather than as a replacement for existing approaches. Nevertheless, the IBD mapping test may still provide additional perspectives in scenarios involving structural variants or other genetic features that are not directly captured in the sequence data.</p>
<p>Another strength of this study is the genome‐wide multiple‐testing adjustment we proposed, which serves as an alternative to the commonly used Bonferroni correction. In previous studies, resampling methods such as permutation tests have been employed to derive multiple‐testing adjustments that account for the correlations between test statistics (Browning and Thompson <a href="#gepi70015-bib-0015" class="usa-link" aria-describedby="gepi70015-bib-0015">2012</a>; Browning <a href="#gepi70015-bib-0009" class="usa-link" aria-describedby="gepi70015-bib-0009">2008</a>; Marees et al. <a href="#gepi70015-bib-0039" class="usa-link" aria-describedby="gepi70015-bib-0039">2018</a>). However, such approaches are computationally intensive, especially in large‐scale GWAS. In contrast, our proposed analytical adjustment is more computationally efficient while still ensuring reliable control of the family‐wise type I error rate by addressing the inherent correlation structure in test statistics. This feature is particularly advantageous for large‐scale studies where computational efficiency is crucial.</p>
<p>Applications of our IBD mapping approach to the UK Biobank systolic blood pressure data revealed a locus on chromosome 19 with IBD mapping <em>p</em>‐value that surpassed the genome‐wide threshold. This location is close to the <em>MYO9B</em> gene and the <em>USHBP1</em> gene on chromosome 19, which has been found to harbor variants associated with blood pressure traits among European‐ancestry individuals (Hoffmann et al. <a href="#gepi70015-bib-0032" class="usa-link" aria-describedby="gepi70015-bib-0032">2017</a>; Giri et al. <a href="#gepi70015-bib-0025" class="usa-link" aria-describedby="gepi70015-bib-0025">2019</a>; Sakaue et al. <a href="#gepi70015-bib-0048" class="usa-link" aria-describedby="gepi70015-bib-0048">2021</a>; Plotnikov et al. <a href="#gepi70015-bib-0045" class="usa-link" aria-describedby="gepi70015-bib-0045">2022</a>; Zhu et al. <a href="#gepi70015-bib-0063" class="usa-link" aria-describedby="gepi70015-bib-0063">2022</a>). This result illustrates the potential of our method for uncovering biologically relevant signals in large‐scale datasets. In comparison, the FiMAP test did not find any significant signals in the same dataset. However, our method required hundreds of hours to run, whereas FiMAP completed in just a couple of hours, indicating a trade‐off between sensitivity and computational cost. While the FiMAP test is well‐suited for efficient genome‐wide scans in large‐scale biobank cohorts, our method may be more advantageous for smaller datasets, where the higher power it offers is more attainable given manageable computational cost.</p>
<p>The performance of our IBD mapping approach is inherently dependent on the quality of the inferred multi‐individual IBD sharing. The accuracy and resolution of multi‐individual IBD detection, influenced by parameters such as haplotype length and trimming thresholds, directly impact both the type I error rate and the power of the mapping tests. In practice, the direct evaluation of power and false discovery rates of multi‐individual IBD detection may not always be feasible, as the true IBD states are typically unknown. Nonetheless, Browning and Browning (<a href="#gepi70015-bib-0014" class="usa-link" aria-describedby="gepi70015-bib-0014">2024</a>) demonstrated through simulation studies that both power and false discovery rates generally decrease as the length threshold or the trimming threshold increases. This highlights the importance of conducting further simulation studies to better understand the trade‐offs among sensitivity, error control, and computational efficiency in multi‐individual IBD detection, thereby refining the selection strategies for algorithm parameters used within our IBD mapping framework. An additional area for improvement in the use of multi‐individual IBD information within our IBD mapping approach is the derivation of kinship coefficients directly from multi‐individual IBD clusters across genome‐wide markers. Such an approach could reduce the computational complexity of the current implementation, which relies on pairwise IBD information to construct the global IBD matrix, although it would require further effort in algorithm development and testing to ensure accuracy and robustness.</p>
<p>In addition to the challenges related to multi‐individual IBD detection, our study also highlights several areas for further improvement. First, our method requires a significant amount of computational time on biobank‐scale cohorts, limiting its scalability to sample size compared to studies using simpler statistical tests, such as single‐variant GWAS or SKAT‐type tests. These tests generally rely on more straightforward calculations, such as comparing summary statistics, fitting linear models, or computing score‐type statistics under the null hypothesis, which are computationally efficient even for large sample sizes. In contrast, our approach is based on a likelihood ratio framework, where optimizing model likelihoods is computationally intensive and becomes increasingly demanding as the sample size grows, thus restricting the scalability of our method. Future work could explore strategies to improve computational efficiency for our IBD mapping application, particularly regarding the inversion of the variance matrix. A key constraint is that the local IBD matrices in our data typically remain full‐rank because they consist of many small IBD clusters involving only one or a few individuals, rather than a few large, strongly related clusters. As a result, common strategies to accelerate matrix computations by reducing matrix rank may not yield significant efficiency gains in this setting. Nonetheless, some matrix randomization techniques to approximate the solution of matrix inversion might still be promising, although their impact on the power of the likelihood ratio test would require further investigation. Additionally, moving the current algorithm pipeline to GPU‐accelerated sparse linear algebra libraries could provide another avenue for improving computational performance.</p>
<p>Furthermore, the resolution of the significant signals identified by our IBD mapping approach was relatively low because IBD status changes slowly along the chromosome. This suggests that future work could focus on methods such as haplotype testing to refine the IBD signals and pinpoint the causal variants under complex traits (Browning <a href="#gepi70015-bib-0012" class="usa-link" aria-describedby="gepi70015-bib-0012">2006</a>; Browning and Browning <a href="#gepi70015-bib-0010" class="usa-link" aria-describedby="gepi70015-bib-0010">2007</a>). Another limitation of our current approach is its focus on quantitative traits with continuous values; future research extending this method to binary or categorical traits would be useful. Likewise, extending this method to test gene‐environment interaction effects by incorporating interaction terms into the model framework could further enhance its applicability for complex trait analyses. Additionally, our IBD mapping test is currently designed for large outbred populations, such as the White British population in the UK Biobank data. However, it would be useful to extend the approach to scenarios involving inbreeding to enable its application in studies of small, inbred populations. Specifically, in the presence of inbreeding, the structure of the local relatedness matrix at the inbred locus would differ, with the diagonal entries being 2 for inbred individuals rather than 1. Such a scenario would necessitate modification in the current methodology to explicitly account for this altered matrix structure to ensure valid variance component modeling.</p>
<p>In summary, the IBD mapping approach developed in this study offers a versatile and powerful tool for detecting associations between genomic regions and complex traits, particularly those involving rare or untyped variants. Its robust performance across both SNP array and sequence data underscores its broad applicability to diverse datasets. Future improvements in computational efficiency, integration with higher‐resolution methods, and extensions to binary traits, gene‐environment interactions, and other complex genetic features could further enhance the utility of IBD information in advancing our understanding of the genetic architecture underlying complex traits across diverse populations.</p></section><section id="sec16"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="gepi70015-suppl-0001"><div class="caption p">
<p>
<strong>Figure S1:</strong> Estimated decay parameter for genome‐wide IBD mapping tests under different algorithm parameters for multi‐individual IBD detection.</p>
<p>
<strong>Figure S2:</strong> Empirical correlation between test statistics spaced at cM from genome‐wide IBD mapping tests with phenotypes simulated under the null hypothesis on simulated sequence data, using different haplotype length thresholds (<strong>
<em>L</em>
</strong>) and trimming thresholds (<strong>
<em>T</em>
</strong>) for multi‐individual IBD detection.</p>
<p>
<strong>Figure S3:</strong> Empirical correlation between test statistics spaced at cM from genome‐wide IBD mapping tests with phenotypes simulated under the null hypothesis on simulated SNP array data, using different haplotype length thresholds (<strong>
<em>L</em>
</strong>) and trimming thresholds (<strong>
<em>T</em>
</strong>) for multi‐individual IBD detection.</p>
<p>
<strong>Figure S4:</strong> Empirical correlation between test statistics from genome‐wide IBD mapping tests with phenotypes simulated under the null hypothesis on the array data of 124,376 White British individuals in the UK Biobank.</p>
<p>
<strong>Figure S5:</strong> The power of our IBD mapping test for detecting common (&gt;10% MAF), low‐frequency (1‐10% MAF), rare (0.05‐1% MAF), and ultra‐rare (&lt;0.05% MAF) causal variants using the Bonferroni correction. Figure S6: Results of applying the FiMAP test at 1 cM intervals across the genome to systolic blood pressure data from 124k White British individuals in the UK Biobank.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12327188/bin/GEPI-49-0-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">GEPI-49-0-s001.docx</a><sup> (812.8KB, docx) </sup>
</div></div></section></section><section id="gepi70015-sec-0160" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Research reported in this publication was supported by the National Human Genome Research Institute of the National Institutes of Health under award number HG007501. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. This study has been conducted using the UK Biobank Resource under Application Number 19934.</p></section><section id="gepi70015-app-0001" class="app"><h2 class="pmc_sec_title">Appendix A. The Null distribution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-221" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span>
</h2>
<p>We consider the mixed‐effects model
</p>
<table class="disp-formula p" id="gepi70015-disp-0008"><tr><td class="formula"><math id="jats-math-222" display="block"><mrow><mrow><mrow><mi mathvariant="bold-italic">Y</mi><mo>=</mo><mi mathvariant="bold">X</mi><mi mathvariant="bold-italic">β</mi><mo>+</mo><mi mathvariant="bold-italic">G</mi><mo>+</mo><msub><mi mathvariant="bold-italic">Q</mi><mi>l</mi></msub><mo>+</mo><mi mathvariant="bold-italic">ε</mi></mrow></mrow></mrow></math></td></tr></table>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">−</span><p class="display-inline">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-223" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">Y</mi></mrow></mrow></mrow></math></span> is a vector of length <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-224" display="inline"><mrow><mrow><mrow><mi>N</mi></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-225" display="inline"><mrow><mrow><mrow><mi>N</mi></mrow></mrow></mrow></math></span> is the sample size</p>
</li>
<li>
<span class="label">−</span><p class="display-inline">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-226" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">X</mi></mrow></mrow></mrow></math></span> is a <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-227" display="inline"><mrow><mrow><mrow><mi>N</mi><mo>×</mo><mi>p</mi></mrow></mrow></mrow></math></span> matrix, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-228" display="inline"><mrow><mrow><mrow><mi>p</mi></mrow></mrow></mrow></math></span> is the number of fixed‐effects covariates</p>
</li>
<li>
<span class="label">−</span><p class="display-inline">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-229" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></mrow></math></span> is a vector of length <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-230" display="inline"><mrow><mrow><mrow><mi>p</mi></mrow></mrow></mrow></math></span>
</p>
</li>
<li>
<span class="label">−</span><p class="display-inline">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-231" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">G</mi><mo>~</mo><mi>N</mi><mo stretchy="true">(</mo><mn mathvariant="bold">0</mn><mo>,</mo><mi mathvariant="bold">Ψ</mi><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo stretchy="true">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-232" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">Ψ</mi></mrow></mrow></mrow></math></span> is the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-233" display="inline"><mrow><mrow><mrow><mi>N</mi><mo>×</mo><mi>N</mi></mrow></mrow></mrow></math></span> genome‐wide relatedness matrix</p>
</li>
<li>
<span class="label">−</span><p class="display-inline">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-234" display="inline"><mrow><mrow><mrow><msub><mi mathvariant="bold-italic">Q</mi><mi>l</mi></msub><mo>~</mo><mi>N</mi><mrow><mo stretchy="true">(</mo><mrow><mn mathvariant="bold">0</mn><mo>,</mo><msub><mi mathvariant="bold">Φ</mi><mi>l</mi></msub><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup></mrow><mo stretchy="true">)</mo></mrow></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-235" display="inline"><mrow><mrow><mrow><msub><mrow><mi mathvariant="bold">Φ</mi></mrow><mi>l</mi></msub></mrow></mrow></mrow></math></span> is the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-236" display="inline"><mrow><mrow><mrow><mi>N</mi><mo>×</mo><mi>N</mi></mrow></mrow></mrow></math></span> local relatedness matrix at a genomic location <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-237" display="inline"><mrow><mrow><mrow><mi>l</mi></mrow></mrow></mrow></math></span>
</p>
</li>
<li>
<span class="label">−</span><p class="display-inline">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-238" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">ε</mi><mo>~</mo><mi>N</mi><mo stretchy="true">(</mo><mn mathvariant="bold">0</mn><mo>,</mo><mi mathvariant="bold">I</mi><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mo stretchy="true">)</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-239" display="inline"><mrow><mrow><mrow><mi mathvariant="bold">I</mi></mrow></mrow></mrow></math></span> is the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-240" display="inline"><mrow><mrow><mrow><mi>N</mi><mo>×</mo><mi>N</mi></mrow></mrow></mrow></math></span> identity matrix</p>
</li>
</ul>
<p>Denote
</p>
<table class="disp-formula p" id="gepi70015-disp-0009"><tr><td class="formula"><math id="jats-math-241" display="block"><mrow><mrow><mrow><mi mathvariant="bold">V</mi><mo>≔</mo><mi mathvariant="italic">Var</mi><mrow><mo>(</mo><mi mathvariant="bold-italic">Y</mi><mo>)</mo></mrow><mo>=</mo><mi mathvariant="bold">Ψ</mi><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>+</mo><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><msubsup><mi>σ</mi><mrow><msub><mi>Q</mi><mi>l</mi></msub></mrow><mn>2</mn></msubsup><mo>+</mo><mi mathvariant="bold">I</mi><msubsup><mi mathvariant="normal">σ</mi><mi mathvariant="normal">ε</mi><mn>2</mn></msubsup><mo>.</mo></mrow></mrow></mrow></math></td></tr></table>
<p>The restricted log likelihood of this model has the following form:
</p>
<table class="disp-formula p" id="gepi70015-disp-0010"><tr><td class="formula"><math id="jats-math-242" display="block"><mrow><mrow><mrow><msub><mi>l</mi><mi mathvariant="bold-italic">Y</mi></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mi mathvariant="normal">;</mi><msub><mi mathvariant="bold">Φ</mi><mi mathvariant="bold-italic">l</mi></msub><mo>,</mo><mi mathvariant="bold">Ψ</mi></mrow></mfenced><mo>=</mo><mi mathvariant="italic">const</mi><mo>−</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi>ln</mi><mo mathvariant="normal">|</mo><mi mathvariant="bold">V</mi><mo mathvariant="normal">|</mo><mo>−</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi>ln</mi><mrow><msup><mrow><mo mathvariant="normal">|</mo><mi mathvariant="bold">X</mi></mrow><mi>T</mi></msup><msup><mi mathvariant="bold">V</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mi mathvariant="bold">X</mi><mo mathvariant="normal">|</mo></mrow><mo>−</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><msup><mrow><mo>(</mo><mrow><mi mathvariant="bold-italic">Y</mi><mo>−</mo><mi mathvariant="bold">X</mi><mi mathvariant="bold-italic">β</mi></mrow><mo>)</mo></mrow><mi>T</mi></msup><msup><mi mathvariant="bold">V</mi><mrow><mo mathvariant="bold">−</mo><mn mathvariant="bold">1</mn></mrow></msup><mrow><mo>(</mo><mrow><mi mathvariant="bold-italic">Y</mi><mo mathvariant="bold">−</mo><mi mathvariant="bold">X</mi><mi mathvariant="bold-italic">β</mi></mrow><mo>)</mo></mrow><mo mathvariant="bold">.</mo></mrow></mrow></mrow></math></td></tr></table>
<p>Denote <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-243" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">γ</mi><mi>≔</mi><mo stretchy="true">(</mo><mi mathvariant="bold-italic">β</mi><mo>,</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>,</mo><msubsup><mi>σ</mi><mi>ε</mi><mn>2</mn></msubsup><mo stretchy="true">)</mo><mo>∈</mo><mi mathvariant="normal">Γ</mi><mo>⊂</mo><msup><mi mathvariant="normal">ℝ</mi><mrow><mi>p</mi><mo>+</mo><mn>2</mn></mrow></msup></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-244" display="inline"><mrow><mrow><mrow><mi>λ</mi><mi>≔</mi><msubsup><mi>σ</mi><msub><mi>Q</mi><mi>l</mi></msub><mn>2</mn></msubsup><mo>∈</mo><mo stretchy="true">(</mo><mn>0</mn><mo>,</mo><mi>∞</mi><mo stretchy="true">)</mo></mrow></mrow></mrow></math></span>. We are interested in testing the hypothesis:
</p>
<table class="disp-formula p" id="gepi70015-disp-0011"><tr><td class="formula"><math id="jats-math-245" display="block"><mrow><mrow><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><mi>λ</mi><mo>=</mo><mn>0</mn><mo> </mo><mo> </mo><mtext>vs.</mtext><mo> </mo><mo> </mo><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><mi>λ</mi><mo>&gt;</mo><mn>0</mn><mo>.</mo></mrow></mrow></mrow></math></td></tr></table>
<p>Note that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-246" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">λ</mi><mo>=</mo><mn mathvariant="bold-italic">0</mn></mrow></mrow></mrow></math></span> is a boundary point of its parameter space.</p>
<p>For each fixed <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-247" display="inline"><mrow><mrow><mrow><mi>λ</mi><mo>≥</mo><mn>0</mn></mrow></mrow></mrow></math></span>, define
</p>
<table class="disp-formula p" id="gepi70015-disp-0012"><tr><td class="formula"><math id="jats-math-248" display="block"><mrow><mrow><mrow><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow><mo>≔</mo><munder accentunder="false"><mtext>sup</mtext><mrow><mi mathvariant="bold-italic">γ</mi><mo>∈</mo><mi mathvariant="normal">Γ</mi></mrow></munder><mo> </mo><msub><mi>l</mi><mi mathvariant="bold-italic">Y</mi></msub><mrow><mo>(</mo><mrow><mi mathvariant="bold-italic">γ</mi><mo>,</mo><mi>λ</mi></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></td></tr></table>
<p>In terms of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-249" display="inline"><mrow><mrow><mrow><msup><mi>l</mi><mo>*</mo></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span>, the test statistic <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-250" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> can be written as:
</p>
<table class="disp-formula p" id="gepi70015-disp-0013"><tr><td class="formula"><math id="jats-math-251" display="block"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub><mo>=</mo><mn>2</mn><mrow><mo stretchy="false">[</mo><mrow><munder><mi>max</mi><mrow><mi>λ</mi><mo>≥</mo><mn>0</mn></mrow></munder><mrow><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow></mrow><mo>−</mo><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mn>0</mn><mo>)</mo></mrow></mrow><mo stretchy="false">]</mo></mrow><mo>.</mo></mrow></mrow></mrow></math></td></tr></table>
<p>Assume <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-252" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mi mathvariant="bold-italic">Y</mi></msub><mo>(</mo><mi>γ</mi><mo>,</mo><mi>λ</mi><mo>)</mo></mrow></mrow></mrow></math></span> is sufficiently smooth and satisfies usual regularity and identifiability conditions. Denote
</p>
<table class="disp-formula p" id="gepi70015-disp-0014"><tr><td class="formula"><math id="jats-math-253" display="block"><mrow><mrow><mrow><mi>U</mi><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow><mi mathvariant="normal">≔</mi><mfrac><mi>d</mi><mrow><mi>d</mi><mi>λ</mi></mrow></mfrac><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow><mo>,</mo><mspace width="1em"></mspace><mi>I</mi><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow><mi mathvariant="normal">≔</mi><mo>−</mo><mfrac><msup><mi>d</mi><mn>2</mn></msup><mrow><mi>d</mi><msup><mi>λ</mi><mn>2</mn></msup></mrow></mfrac><msup><mrow><mi>l</mi><mspace width="0.25em"></mspace></mrow><mrow><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></td></tr></table>
<p>Note that the solution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-254" display="inline"><mrow><mrow><mrow><mfrac><mi mathvariant="bold-italic">d</mi><mrow><mi mathvariant="bold-italic">d</mi><mi mathvariant="bold-italic">λ</mi></mrow></mfrac><msup><mi mathvariant="bold-italic">l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi mathvariant="bold-italic">λ</mi><mo>)</mo></mrow><mo>=</mo><mn mathvariant="bold-italic">0</mn></mrow></mrow></mrow></math></span> is the unconstrained maximum likelihood estimator, i.e.,
</p>
<table class="disp-formula p" id="gepi70015-disp-0015"><tr><td class="formula"><math id="jats-math-255" display="block"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mi>~</mi></mover><mo>=</mo><munder accentunder="false"><mtext>argmax</mtext><mrow><mi>λ</mi><mo>∈</mo><mi>ℝ</mi></mrow></munder><mo> </mo><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></td></tr></table>
<p>Thus,
</p>
<table class="disp-formula p" id="gepi70015-disp-0016"><tr><td class="formula"><math id="jats-math-256" display="block"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mo>ˆ</mo></mover><mo>≔</mo><munder accentunder="false"><mtext>argmax</mtext><mrow><mi>λ</mi><mo>≥</mo><mn>0</mn></mrow></munder><mo> </mo><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow><mo>=</mo><mi>max</mi><mo>⁡</mo><mrow><mo>{</mo><mrow><mn>0</mn><mo>,</mo><mover accent="true"><mi>λ</mi><mi>~</mi></mover></mrow><mo>}</mo></mrow></mrow></mrow></mrow></math></td></tr></table>
<p>Consider the first derivative of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-257" display="inline"><mrow><mrow><mrow><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> at <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-258" display="inline"><mrow><mrow><mrow><mn>0</mn></mrow></mrow></mrow></math></span>:
</p>
<table class="disp-formula p" id="gepi70015-disp-0017"><tr><td class="formula"><math id="jats-math-259" display="block"><mrow><mrow><mrow><mi>U</mi><mrow><mo>(</mo><mn>0</mn><mo>)</mo></mrow><mo>=</mo><mfrac><mi>d</mi><mrow><mi>d</mi><mi>λ</mi></mrow></mfrac><msub><mrow><mo stretchy="false"></mo><mrow><msup><mi>l</mi><mrow><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace></mrow></msup><mrow><mo>(</mo><mi>λ</mi><mo>)</mo></mrow></mrow><mo stretchy="false">|</mo></mrow><mrow><mi>λ</mi><mo>=</mo><mn>0</mn></mrow></msub><mo>,</mo></mrow></mrow></mrow></math></td></tr></table>
<p>we see
</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">−</span><p class="display-inline">If <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-260" display="inline"><mrow><mrow><mrow><mi>U</mi><mrow><mo>(</mo><mn>0</mn><mo>)</mo></mrow><mo>&gt;</mo><mn>0</mn></mrow></mrow></mrow></math></span>, then <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-261" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mo>~</mo></mover><mo>&gt;</mo><mn>0</mn><mo>,</mo></mrow></mrow></mrow></math></span> so <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-262" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mo>ˆ</mo></mover><mo>=</mo><mover accent="true"><mi>λ</mi><mo>~</mo></mover><mo>.</mo></mrow></mrow></mrow></math></span>
</p>
</li>
<li>
<span class="label">−</span><p class="display-inline">If <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-263" display="inline"><mrow><mrow><mrow><mi>U</mi><mrow><mo>(</mo><mn>0</mn><mo>)</mo></mrow><mo>&lt;</mo><mn>0</mn></mrow></mrow></mrow></math></span>, then <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-264" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mo>~</mo></mover><mo>&lt;</mo><mn>0</mn></mrow></mrow></mrow></math></span>, so <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-265" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mo>ˆ</mo></mover><mo>=</mo><mn>0</mn><mo>.</mo></mrow></mrow></mrow></math></span>
</p>
</li>
</ul>
<p>Note that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-266" display="inline"><mrow><mrow><mrow><mi>U</mi><mrow><mo>(</mo><mn>0</mn><mo>)</mo></mrow><mover><mo>→</mo><mi>d</mi></mover><mi mathvariant="italic">N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>I</mi><mrow><mo>(</mo><mn>0</mn><mo>)</mo></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math></span>. Thus, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-267" display="inline"><mrow><mrow><mrow><mtext>Pr</mtext><mrow><mo>[</mo><mrow><mover accent="true"><mi>λ</mi><mo>~</mo></mover><mo>&lt;</mo><mn>0</mn></mrow><mo>]</mo></mrow><mo>=</mo><mtext>Pr</mtext><mrow><mo stretchy="false">[</mo><mrow><mi>U</mi><mrow><mo>(</mo><mn>0</mn><mo>)</mo></mrow><mo>&lt;</mo><mn>0</mn></mrow><mo stretchy="false">]</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mo>.</mo></mrow></mrow></mrow></math></span>
</p>
<p>When <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-268" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mo>ˆ</mo></mover><mo>=</mo><mn>0</mn><mo>,</mo><msub><mi>W</mi><mi>l</mi></msub><mo>=</mo><mn>0</mn></mrow></mrow></mrow></math></span>. When <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-269" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>λ</mi><mo>ˆ</mo></mover><mo>=</mo><mover accent="true"><mi>λ</mi><mo>~</mo></mover></mrow></mrow></mrow></math></span>, by asymptotic properties of the maximum likelihood estimator, we have that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-270" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub><mover><mo>→</mo><mi>d</mi></mover><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup></mrow></mrow></mrow></math></span>. Therefore, the distribution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-271" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub></mrow></mrow></mrow></math></span> is a 50:50 mixture of a point mass at 0 and a <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-272" display="inline"><mrow><mrow><mrow><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup></mrow></mrow></mrow></math></span> under <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-273" display="inline"><mrow><mrow><mrow><msub><mi>H</mi><mn>0</mn></msub></mrow></mrow></mrow></math></span>, as shown in Chernoff (<a href="#gepi70015-bib-0020" class="usa-link" aria-describedby="gepi70015-bib-0020">1954</a>), Miller (<a href="#gepi70015-bib-0042" class="usa-link" aria-describedby="gepi70015-bib-0042">1977</a>), and Self and Liang (<a href="#gepi70015-bib-0049" class="usa-link" aria-describedby="gepi70015-bib-0049">1987</a>) (Chernoff <a href="#gepi70015-bib-0020" class="usa-link" aria-describedby="gepi70015-bib-0020">1954</a>; Miller <a href="#gepi70015-bib-0042" class="usa-link" aria-describedby="gepi70015-bib-0042">1977</a>; Self and Liang <a href="#gepi70015-bib-0049" class="usa-link" aria-describedby="gepi70015-bib-0049">1987</a>).</p></section><section id="gepi70015-app-0002" class="app"><h2 class="pmc_sec_title">Appendix B. Correlation structure of the modified OU process</h2>
<p>Consider <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-274" display="inline"><mrow><mrow><mrow><msub><mi>Z</mi><mi>l</mi></msub><mo>~</mo><mi>N</mi><mo>(</mo><mn>0</mn><mo>,</mo><mn>1</mn><mo>)</mo></mrow></mrow></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-275" display="inline"><mrow><mrow><mrow><mtext>Pr</mtext><mo>[</mo><msub><mi>Z</mi><mi>l</mi></msub><mo>&gt;</mo><mn>0</mn><mo>]</mo><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mo>,</mo></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-276" display="inline"><mrow><mrow><mrow><msubsup><mi>Z</mi><mi>l</mi><mn>2</mn></msubsup><mo>~</mo><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup></mrow></mrow></mrow></math></span>. From results shown in Appendix A, it is easily seen that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-277" display="inline"><mrow><mrow><mrow><msub><mi>W</mi><mi>l</mi></msub><mo>=</mo></mrow></mrow></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-278" display="inline"><mrow><mrow><mrow><msubsup><mi>Z</mi><mi>l</mi><mn>2</mn></msubsup><msub><mi>I</mi><mrow><mo>[</mo><msub><mi>Z</mi><mi>l</mi></msub><mo>&gt;</mo><mn>0</mn><mo>]</mo></mrow></msub></mrow></mrow></mrow></math></span>.</p>
<p>Now suppose that observations <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-279" display="inline"><mrow><mrow><mrow><msub><mrow><mo>{</mo><msub><mi>Z</mi><mi>l</mi></msub><mo>}</mo></mrow><mrow><msub><mi>l</mi><mn>1</mn></msub><mo>:</mo><msub><mi>l</mi><mi>p</mi></msub></mrow></msub></mrow></mrow></mrow></math></span> are drawn from an Orenstein‐Uhlenbeck process, where
</p>
<table class="disp-formula p" id="gepi70015-disp-0018"><tr><td class="formula"><math id="jats-math-280" display="block"><mrow><mrow><mrow><mi mathvariant="italic">Cov</mi><mrow><mo>(</mo><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>,</mo><msub><mi>Z</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>)</mo></mrow><mo>=</mo><mtext>exp</mtext><mrow><mo stretchy="false">(</mo><mrow><mo>−</mo><mi>α</mi><mrow><mo>|</mo><mrow><msub><mi>l</mi><mi>i</mi></msub><mo>−</mo><msub><mi>l</mi><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70015-disp-0019"><tr><td class="formula"><math id="jats-math-281" display="block"><mrow><mrow><mrow><mi>E</mi><mrow><mo>[</mo><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>]</mo></mrow><mo>=</mo><mi>E</mi><mo>[</mo><msub><mi>Z</mi><msub><mi>l</mi><mi>j</mi></msub></msub><mo>]</mo><mo>=</mo><mn>0</mn></mrow></mrow></mrow></math></td></tr></table>
<p>for any two observations <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-282" display="inline"><mrow><mrow><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>,</mo><msub><mi>Z</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow></mrow></mrow></math></span> taking at locations <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-283" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span> and time <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-284" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span> from this process. We refer to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-285" display="inline"><mrow><mrow><mrow><msub><mrow><mo>{</mo><msub><mi>W</mi><mi>l</mi></msub><mo>}</mo></mrow><mrow><msub><mi>l</mi><mn>1</mn></msub><mo>:</mo><msub><mi>l</mi><mi>p</mi></msub></mrow></msub></mrow></mrow></mrow></math></span> as the modified OU process, i.e., for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-286" display="inline"><mrow><mrow><mrow><msub><mi>l</mi><mi>i</mi></msub><mo>∈</mo><mrow><mo>{</mo><mrow><msub><mi>l</mi><mn>1</mn></msub><mo>,</mo><msub><mi>l</mi><mn>2</mn></msub><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><msub><mi>l</mi><mi>p</mi></msub></mrow><mo>}</mo></mrow><mo>,</mo></mrow></mrow></mrow></math></span>
</p>
<table class="disp-formula p" id="gepi70015-disp-0020"><tr><td class="formula"><math id="jats-math-287" display="block"><mrow><mrow><mrow><msub><mi>W</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo>=</mo><mfenced open="{" close=""><mrow><mtable displaystyle="true"><mtr><mtd><msubsup><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow><mn>2</mn></msubsup><mo> </mo><mo> </mo><mi mathvariant="italic">if</mi><mspace width="1em"></mspace><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo>&gt;</mo><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mi mathvariant="italic">if</mi><mspace width="1em"></mspace><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo>≤</mo><mn>0</mn></mtd></mtr></mtable><mo>.</mo></mrow></mfenced></mrow></mrow></mrow></math></td></tr></table>
<p>Our goal is to derive <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-288" display="inline"><mrow><mrow><mrow><mi mathvariant="italic">Corr</mi><mrow><mo>(</mo><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>,</mo><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>)</mo></mrow><mo>.</mo></mrow></mrow></mrow></math></span>
</p>
<p>First, we derive <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-289" display="inline"><mrow><mrow><mrow><mi mathvariant="italic">Cov</mi><mrow><mo>(</mo><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>,</mo><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>)</mo></mrow><mo>=</mo><mi>E</mi><mrow><mo>[</mo><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>]</mo></mrow><mo>−</mo><mi>E</mi><mrow><mo>[</mo><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>]</mo></mrow><mi>E</mi><mrow><mo>[</mo><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub><mo>]</mo></mrow><mo>.</mo></mrow></mrow></mrow></math></span> Since the marginal distribution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-290" display="inline"><mrow><mrow><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub></mrow></mrow></mrow></math></span> is standard normal, we have:
</p>
<table class="disp-formula p" id="gepi70015-disp-0021"><tr><td class="formula"><math id="jats-math-291" display="block"><mrow><mrow><mrow><mi>E</mi><mrow><mo>[</mo><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>]</mo></mrow><mo>=</mo><mi>E</mi><mfenced close="]" open="["><msubsup><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub><mn>2</mn></msubsup></mfenced><mo>*</mo><mi>P</mi><mrow><mo>(</mo><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>&gt;</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70015-disp-0022"><tr><td class="formula"><math id="jats-math-292" display="block"><mrow><mrow><mrow><mi mathvariant="italic">Var</mi><mrow><mo>[</mo><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>]</mo></mrow><mo>=</mo><mi>E</mi><mfenced close="]" open="["><msubsup><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub><mn>2</mn></msubsup></mfenced><mo>−</mo><msup><mrow><mo stretchy="false">(</mo><mrow><mi>E</mi><mrow><mo>[</mo><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>]</mo></mrow></mrow><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>=</mo><mi>E</mi><mfenced close="]" open="["><msubsup><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub><mn>4</mn></msubsup></mfenced><mi>P</mi><mrow><mo>(</mo><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>&gt;</mo><mn>0</mn></mrow><mo>)</mo></mrow><mo>−</mo><mfrac><mn>1</mn><mn>4</mn></mfrac><mo>=</mo><mfrac><mn>3</mn><mn>2</mn></mfrac><mo>−</mo><mfrac><mn>1</mn><mn>4</mn></mfrac><mo>=</mo><mfrac><mn>5</mn><mn>4</mn></mfrac><mo>.</mo></mrow></mrow></mrow></math></td></tr></table>
<p>Furthermore,
</p>
<table class="disp-formula p" id="gepi70015-disp-0023"><tr><td class="formula"><math id="jats-math-293" display="block"><mrow><mrow><mrow><mi>E</mi><mo form="prefix" stretchy="false">[</mo><msub><mi>W</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><msub><mi>W</mi><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow></msub><mo form="postfix" stretchy="false">]</mo><mo>=</mo><mi>E</mi><mfenced open="{" close="}"><mrow><msup><mrow><mo form="prefix" stretchy="false">(</mo><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow></msub><mo form="postfix" stretchy="false">)</mo></mrow><mrow><mn>2</mn></mrow></msup><mo>|</mo><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo>&gt;</mo><mn>0</mn><mo> </mo><mi mathvariant="normal">and</mi><mo> </mo><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow></msub><mo>&gt;</mo><mn>0</mn></mrow></mfenced></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70015-disp-0024"><tr><td class="formula"><math id="jats-math-294" display="block"><mrow><mrow><mrow><mo>=</mo><mo movablelimits="false">∬</mo><msubsup><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow><mn>2</mn></msubsup><msubsup><mi>Z</mi><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow><mn>2</mn></msubsup><mi>f</mi><mo form="prefix" stretchy="false">(</mo><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo separator="true">,</mo><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow></msub><mo form="postfix" stretchy="false">)</mo><msub><mi>I</mi><mrow><mo form="prefix" stretchy="false">[</mo><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo>&gt;</mo><mn>0</mn><mo separator="true">,</mo><msub><mi>Z</mi><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow></msub><mo>&gt;</mo><mn>0</mn><mo form="postfix" stretchy="false">]</mo></mrow></msub><mi>d</mi><msub><mi>z</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mi>d</mi><msub><mi>z</mi><mrow><msub><mi>l</mi><mi>j</mi></msub></mrow></msub></mrow></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-295" display="inline"><mrow><mrow><mrow><mi>f</mi><mrow><mo>(</mo><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>,</mo><msub><mi>Z</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></span> is the probability density function of the joint binormal distribution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-296" display="inline"><mrow><mrow><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-297" display="inline"><mrow><mrow><mrow><msub><mi>Z</mi><msub><mi>l</mi><mi>i</mi></msub></msub></mrow></mrow></mrow></math></span>. We evaluated this integral using Mathematica (Wolfram Research I <a href="#gepi70015-bib-0059" class="usa-link" aria-describedby="gepi70015-bib-0059">2024</a>), and we obtained that
</p>
<table class="disp-formula p" id="gepi70015-disp-0025"><tr><td class="formula"><math id="jats-math-298" display="block"><mrow><mrow><mrow><mi>E</mi><mrow><mo>[</mo><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>]</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>4</mn></mfrac><mo>+</mo><mfrac><mrow><mn>2</mn><mi>π</mi><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo>+</mo><mn>6</mn><mi>ρ</mi><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow></mrow></msqrt><mo>+</mo><mo>(</mo><mn>2</mn><mo>+</mo><mn>4</mn><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo>)</mo><msup><mi>sin</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mi>ρ</mi><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow></mrow></mrow><mrow><mn>4</mn><mi>π</mi></mrow></mfrac><mo>,</mo></mrow></mrow></mrow></math></td></tr></table>
<p>where we define <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-299" display="inline"><mrow><mrow><mrow><mi>d</mi><mi mathvariant="normal">≔</mi><mrow><mo>|</mo><mrow><msub><mi>l</mi><mi>i</mi></msub><mo>−</mo><msub><mi>l</mi><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-300" display="inline"><mrow><mrow><mrow><mi>ρ</mi><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mi mathvariant="normal">≔</mi><mtext>exp</mtext><mrow><mo>(</mo><mrow><mo>−</mo><mi>α</mi><mi>d</mi></mrow><mo>)</mo></mrow><mo>.</mo></mrow></mrow></mrow></math></span> Hence,
</p>
<table class="disp-formula p" id="gepi70015-disp-0026"><tr><td class="formula"><math id="jats-math-301" display="block"><mrow><mrow><mrow><mi mathvariant="italic">Cov</mi><mrow><mo>(</mo><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><mo>,</mo><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mn>2</mn><mi>π</mi><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo>+</mo><mn>6</mn><mi>ρ</mi><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow></mrow></msqrt><mo>+</mo><mo>(</mo><mn>2</mn><mo>+</mo><mn>4</mn><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo>)</mo><msup><mi>sin</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mi>ρ</mi><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow></mrow></mrow><mrow><mn>4</mn><mi>π</mi></mrow></mfrac><mo>.</mo></mrow></mrow></mrow></math></td></tr></table>
<p>And so
</p>
<table class="disp-formula p" id="gepi70015-disp-0027"><tr><td class="formula"><math id="jats-math-302" display="block"><mrow><mrow><mrow><mi mathvariant="italic">Corr</mi><mrow><mo>(</mo><mrow><msub><mi>W</mi><msub><mi>l</mi><mi>i</mi></msub></msub><msub><mi>W</mi><msub><mi>l</mi><mi>j</mi></msub></msub></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>5</mn></mfrac><mfenced close="}" open="{"><mfrac><mrow><mn>2</mn><mi>π</mi><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo>+</mo><mn>6</mn><mi>ρ</mi><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow></mrow></msqrt><mo>+</mo><mo>(</mo><mn>2</mn><mo>+</mo><mn>4</mn><msup><mi>ρ</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo>)</mo><msup><mi>sin</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mi>ρ</mi><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow></mrow></mrow><mi>π</mi></mfrac></mfenced><mo>.</mo></mrow></mrow></mrow></math></td></tr></table></section><section id="gepi70015-app-0003" class="app"><h2 class="pmc_sec_title">Appendix C. Simulation algorithm of the modified OU process along a chromosome</h2>
<p>
<strong>Input:</strong>
</p>
<ul class="list" style="list-style-type:disc">
<li><p>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-303" display="inline"><mrow><mrow><mrow><mtext>chrlen</mtext></mrow></mrow></mrow></math></span>: total length of chromosome in centiMorgan</p></li>
<li><p>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-304" display="inline"><mrow><mrow><mrow><mtext>alpha</mtext><mo>:</mo></mrow></mrow></mrow></math></span> value of the decay parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-305" display="inline"><mrow><mrow><mrow><mi>α</mi></mrow></mrow></mrow></math></span>
</p></li>
<li><p>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-306" display="inline"><mrow><mrow><mrow><mtext>stepsize</mtext><mo>:</mo></mrow></mrow></mrow></math></span> size of the interval between consecutive obervations</p></li>
<li><p>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-307" display="inline"><mrow><mrow><mrow><mtext>n_iter</mtext><mo>:</mo></mrow></mrow></mrow></math></span> total number of replicates generated</p></li>
</ul>
<p>
<strong>Output:</strong>
</p>
<ul class="list" style="list-style-type:disc"><li><p>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-308" display="inline"><mrow><mrow><mrow><mtext>maxval</mtext><mo>:</mo></mrow></mrow></mrow></math></span> a vector of maximum simulated observations in each of the replicates</p></li></ul>
<p>
<strong>Initialization:</strong>
</p>
<ul class="list" style="list-style-type:disc">
<li><p>Initialize <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-309" display="inline"><mrow><mrow><mrow><mi mathvariant="normal">x</mi></mrow></mrow></mrow></math></span> as an array of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-310" display="inline"><mrow><mrow><mrow><mtext>n_iter</mtext></mrow></mrow></mrow></math></span> random normal values</p></li>
<li><p>Initialize <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-311" display="inline"><mrow><mrow><mrow><mtext>modified_x</mtext></mrow></mrow></mrow></math></span> as an array where:</p></li>
</ul>
<table class="disp-formula p" id="gepi70015-disp-0028"><tr><td class="formula"><math id="jats-math-312" display="block"><mrow><mrow><mrow><mtext>modified_x</mtext><mrow><mo>[</mo><mi mathvariant="normal">i</mi><mo>]</mo></mrow><mo>=</mo><mn>0</mn><mo> </mo><mtext>if x</mtext><mrow><mo>[</mo><mi mathvariant="normal">i</mi><mo>]</mo></mrow><mo>&lt;</mo><mn>0</mn></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70015-disp-0029"><tr><td class="formula"><math id="jats-math-313" display="block"><mrow><mrow><mrow><mtext>modified_x</mtext><mrow><mo>[</mo><mi mathvariant="normal">i</mi><mo>]</mo></mrow><mo>=</mo><mi mathvariant="normal">x</mi><msup><mrow><mo>[</mo><mi mathvariant="normal">i</mi><mo>]</mo></mrow><mrow><mn>2</mn><mo> </mo></mrow></msup><mtext>if x</mtext><mrow><mo>[</mo><mi mathvariant="normal">i</mi><mo>]</mo></mrow><mo>≥</mo><mn>0</mn></mrow></mrow></mrow></math></td></tr></table>
<ul class="list" style="list-style-type:disc"><li><p>Set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-314" display="inline"><mrow><mrow><mrow><mtext>maxval</mtext><mo>=</mo><mtext>modified_x</mtext></mrow></mrow></mrow></math></span>
</p></li></ul>
<p>
<strong>Iterative process:</strong>
</p>
<p>For <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-315" display="inline"><mrow><mrow><mrow><mi>i</mi></mrow></mrow></mrow></math></span> from <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-316" display="inline"><mrow><mrow><mrow><mn>2</mn></mrow></mrow></mrow></math></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-317" display="inline"><mrow><mrow><mrow><mfenced close="⌋" open="⌊"><mfrac><mi mathvariant="italic">chrlen</mi><mi mathvariant="italic">stepsize</mi></mfrac></mfenced></mrow></mrow></mrow></math></span>:
</p>
<ul class="list" style="list-style-type:disc">
<li><div class="p">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-318" display="inline"><mrow><mrow><mrow><mtext>newx</mtext></mrow></mrow></mrow></math></span> = array of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-319" display="inline"><mrow><mrow><mrow><mtext>n_iter</mtext></mrow></mrow></mrow></math></span> random normal values with:
<table class="disp-formula p" id="gepi70015-disp-0030"><tr><td class="formula"><math id="jats-math-320" display="block"><mrow><mrow><mrow><mtext>mean</mtext><mspace width="0.25em"></mspace><mo>=</mo><mi mathvariant="normal">x</mi><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace><mtext>exp</mtext><mrow><mo>(</mo><mrow><mo>−</mo><mtext>stepsize</mtext><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace><mtext>alpha</mtext></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70015-disp-0031"><tr><td class="formula"><math id="jats-math-321" display="block"><mrow><mrow><mrow><mspace width="4.3em"></mspace><mtext>sd</mtext><mspace width="0.25em"></mspace><mo>=</mo><msqrt><mrow><mn>1</mn><mo>−</mo><mtext>exp</mtext><mrow><mo>(</mo><mrow><mo>−</mo><mn>2</mn><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace><mtext>stepsize</mtext><mspace width="0.25em"></mspace><mo>*</mo><mspace width="0.25em"></mspace><mtext>alpha</mtext></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></mrow></mrow></math></td></tr></table>
</div></li>
<li><p>Initialize <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-322" display="inline"><mrow><mrow><mrow><mtext>new_modified_x</mtext></mrow></mrow></mrow></math></span> as an array where:</p></li>
</ul>
<table class="disp-formula p" id="gepi70015-disp-0032"><tr><td class="formula"><math id="jats-math-323" display="block"><mrow><mrow><mrow><mtext>new_modified_x</mtext><mrow><mo>[</mo><mi mathvariant="normal">j</mi><mo>]</mo></mrow><mo>=</mo><mn>0</mn><mo> </mo><mtext>if newx</mtext><mrow><mo>[</mo><mi mathvariant="normal">j</mi><mo>]</mo></mrow><mo>&lt;</mo><mn>0</mn></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70015-disp-0033"><tr><td class="formula"><math id="jats-math-324" display="block"><mrow><mrow><mrow><mtext>new_modified_x</mtext><mrow><mo>[</mo><mi mathvariant="normal">j</mi><mo>]</mo></mrow><mo>=</mo><mtext>newx</mtext><msup><mrow><mo>[</mo><mi mathvariant="normal">j</mi><mo>]</mo></mrow><mn>2</mn></msup><mo> </mo><mtext>if newx</mtext><mrow><mo>[</mo><mi mathvariant="normal">j</mi><mo>]</mo></mrow><mo>≥</mo><mn>0</mn></mrow></mrow></mrow></math></td></tr></table>
<ul class="list" style="list-style-type:disc">
<li><p>Update <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-325" display="inline"><mrow><mrow><mrow><mtext>maxval</mtext></mrow></mrow></mrow></math></span> as the element‐wise maximum of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-326" display="inline"><mrow><mrow><mrow><mtext>maxval</mtext></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-327" display="inline"><mrow><mrow><mrow><mtext>new_modified_x</mtext></mrow></mrow></mrow></math></span>
</p></li>
<li><p>Set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-328" display="inline"><mrow><mrow><mrow><mi mathvariant="normal">x</mi><mo>=</mo><mspace width="0.25em"></mspace><mtext>newx</mtext></mrow></mrow></mrow></math></span>
</p></li>
</ul>
<p>Return <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-329" display="inline"><mrow><mrow><mrow><mtext>maxval</mtext></mrow></mrow></mrow></math></span>
</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Ruoyi Cai, Email: rcai2@uw.edu.</p>
<p>Sharon R. Browning, Email: sguy@uw.edu.</p></section><section id="gepi70015-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="gepi70015-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="gepi70015-bib-0001">
<cite>

Almasy, L.
, and Blangero J.. 1998. “Multipoint Quantitative‐Trait Linkage Analysis in General Pedigrees.” American Journal of Human Genetics
62, no. 5: 1198–1211.
</cite> [<a href="https://doi.org/10.1086/301844" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1377101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9545414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Multipoint%20Quantitative%E2%80%90Trait%20Linkage%20Analysis%20in%20General%20Pedigrees&amp;author=L.%20Almasy&amp;author=J.%20Blangero&amp;volume=62&amp;issue=5&amp;publication_year=1998&amp;pages=1198-1211&amp;pmid=9545414&amp;doi=10.1086/301844&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0002">
<cite>

Almasy, L.
, and Blangero J.. 2010. “Variance Component Methods for Analysis of Complex Phenotypes.” Cold Spring Harbor Protocols
2010, no. 5: pdb. top77.
</cite> [<a href="https://doi.org/10.1101/pdb.top77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3064490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20439422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Protocols&amp;title=Variance%20Component%20Methods%20for%20Analysis%20of%20Complex%20Phenotypes&amp;author=L.%20Almasy&amp;author=J.%20Blangero&amp;volume=2010&amp;issue=5&amp;publication_year=2010&amp;pages=pdb.%20top77&amp;pmid=20439422&amp;doi=10.1101/pdb.top77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0003">
<cite>

Amos, C. I.

1994. “Robust Variance‐Components Approach for Assessing Genetic Linkage in Pedigrees.” American Journal of Human Genetics
54, no. 3: 535–543.
</cite> [<a href="/articles/PMC1918121/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8116623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Robust%20Variance%E2%80%90Components%20Approach%20for%20Assessing%20Genetic%20Linkage%20in%20Pedigrees&amp;author=C.%20I.%20Amos&amp;volume=54&amp;issue=3&amp;publication_year=1994&amp;pages=535-543&amp;pmid=8116623&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0004">
<cite>

Baumdicker, F.
, Bisschop G., Goldstein D., et al. 2022. “Efficient Ancestry and Mutation Simulation With Msprime 1.0.” Genetics
220, no. 3: iyab229.
</cite> [<a href="https://doi.org/10.1093/genetics/iyab229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9176297/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34897427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Efficient%20Ancestry%20and%20Mutation%20Simulation%20With%20Msprime%201.0&amp;author=F.%20Baumdicker&amp;author=G.%20Bisschop&amp;author=D.%20Goldstein&amp;volume=220&amp;issue=3&amp;publication_year=2022&amp;pages=iyab229&amp;pmid=34897427&amp;doi=10.1093/genetics/iyab229&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0005">
<cite>

Belbin, G. M.
, Odgis J., Sorokin E. P., et al. 2017. “Genetic Identification of a Common Collagen Disease in Puerto Ricans Via Identity‐By‐Descent Mapping in a Health System.” eLife
6: e25060.
</cite> [<a href="https://doi.org/10.7554/eLife.25060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5595434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28895531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Genetic%20Identification%20of%20a%20Common%20Collagen%20Disease%20in%20Puerto%20Ricans%20Via%20Identity%E2%80%90By%E2%80%90Descent%20Mapping%20in%20a%20Health%20System&amp;author=G.%20M.%20Belbin&amp;author=J.%20Odgis&amp;author=E.%20P.%20Sorokin&amp;volume=6&amp;publication_year=2017&amp;pages=e25060&amp;pmid=28895531&amp;doi=10.7554/eLife.25060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0006">
<cite>

Bhérer, C.
, Campbell C. L., and Auton A.. 2017. “Refined Genetic Maps Reveal Sexual Dimorphism in Human Meiotic Recombination At Multiple Scales.” Nature Communications
8, no. 1: 14994.</cite> [<a href="https://doi.org/10.1038/ncomms14994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5414043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28440270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Refined%20Genetic%20Maps%20Reveal%20Sexual%20Dimorphism%20in%20Human%20Meiotic%20Recombination%20At%20Multiple%20Scales&amp;author=C.%20Bh%C3%A9rer&amp;author=C.%20L.%20Campbell&amp;author=A.%20Auton&amp;volume=8&amp;issue=1&amp;publication_year=2017&amp;pages=14994&amp;pmid=28440270&amp;doi=10.1038/ncomms14994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0007">
<cite>

Blangero, J.
, and Almasy L.. 1997. “Multipoint Oligogenic Linkage Analysis of Quantitative Traits.” Genetic Epidemiology
14, no. 6: 959–964.
</cite> [<a href="https://doi.org/10.1002/(SICI)1098-2272(1997)14:6&lt;959::AID-GEPI66&gt;3.0.CO;2-K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9433607/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&amp;title=Multipoint%20Oligogenic%20Linkage%20Analysis%20of%20Quantitative%20Traits&amp;author=J.%20Blangero&amp;author=L.%20Almasy&amp;volume=14&amp;issue=6&amp;publication_year=1997&amp;pages=959-964&amp;pmid=9433607&amp;doi=10.1002/(SICI)1098-2272(1997)14:6&lt;959::AID-GEPI66&gt;3.0.CO;2-K&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0008">
<cite>

Blangero, J.
, Williams J. T., and Almasy L.. 2000. “Quantitative Trait Locus Mapping Using Human Pedigrees.” Human Biology
72: 35–62.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10721613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Biology&amp;title=Quantitative%20Trait%20Locus%20Mapping%20Using%20Human%20Pedigrees&amp;author=J.%20Blangero&amp;author=J.%20T.%20Williams&amp;author=L.%20Almasy&amp;volume=72&amp;publication_year=2000&amp;pages=35-62&amp;pmid=10721613&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0009">
<cite>

Browning, B. L.

2008. “Presto: Rapid Calculation of Order Statistic Distributions and Multiple‐Testing Adjusted P‐Values Via Permutation for One and Two‐Stage Genetic Association Studies.” BMC Bioinformatics
9: 309.
</cite> [<a href="https://doi.org/10.1186/1471-2105-9-309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2483288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18620604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Presto:%20Rapid%20Calculation%20of%20Order%20Statistic%20Distributions%20and%20Multiple%E2%80%90Testing%20Adjusted%20P%E2%80%90Values%20Via%20Permutation%20for%20One%20and%20Two%E2%80%90Stage%20Genetic%20Association%20Studies&amp;author=B.%20L.%20Browning&amp;volume=9&amp;publication_year=2008&amp;pages=309&amp;pmid=18620604&amp;doi=10.1186/1471-2105-9-309&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0010">
<cite>

Browning, B. L.
, and Browning S. R.. 2007. “Efficient Multilocus Association Testing for Whole Genome Association Studies Using Localized Haplotype Clustering.” Genetic Epidemiology
31, no. 5: 365–375.
</cite> [<a href="https://doi.org/10.1002/gepi.20216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17326099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&amp;title=Efficient%20Multilocus%20Association%20Testing%20for%20Whole%20Genome%20Association%20Studies%20Using%20Localized%20Haplotype%20Clustering&amp;author=B.%20L.%20Browning&amp;author=S.%20R.%20Browning&amp;volume=31&amp;issue=5&amp;publication_year=2007&amp;pages=365-375&amp;pmid=17326099&amp;doi=10.1002/gepi.20216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0011">
<cite>

Browning, B. L.
, and Browning S. R.. 2013. “Detecting Identity by Descent and Estimating Genotype Error Rates in Sequence Data.” The American Journal of Human Genetics
93, no. 5: 840–851.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2013.09.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3824133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24207118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&amp;title=Detecting%20Identity%20by%20Descent%20and%20Estimating%20Genotype%20Error%20Rates%20in%20Sequence%20Data&amp;author=B.%20L.%20Browning&amp;author=S.%20R.%20Browning&amp;volume=93&amp;issue=5&amp;publication_year=2013&amp;pages=840-851&amp;pmid=24207118&amp;doi=10.1016/j.ajhg.2013.09.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0012">
<cite>

Browning, S. R.

2006. “Multilocus Association Mapping Using Variable‐Length Markov Chains.” American Journal of Human Genetics
78, no. 6: 903–913.
</cite> [<a href="https://doi.org/10.1086/503876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1474089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16685642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Multilocus%20Association%20Mapping%20Using%20Variable%E2%80%90Length%20Markov%20Chains&amp;author=S.%20R.%20Browning&amp;volume=78&amp;issue=6&amp;publication_year=2006&amp;pages=903-913&amp;pmid=16685642&amp;doi=10.1086/503876&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0013">
<cite>

Browning, S. R.
, and Browning B. L.. 2012. “Identity by Descent Between Distant Relatives: Detection and Applications.” Annual Review of Genetics
46: 617–633.</cite> [<a href="https://doi.org/10.1146/annurev-genet-110711-155534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22994355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Genetics&amp;title=Identity%20by%20Descent%20Between%20Distant%20Relatives:%20Detection%20and%20Applications&amp;author=S.%20R.%20Browning&amp;author=B.%20L.%20Browning&amp;volume=46&amp;publication_year=2012&amp;pages=617-633&amp;pmid=22994355&amp;doi=10.1146/annurev-genet-110711-155534&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0014">
<cite>

Browning, S. R.
, and Browning B. L.. 2024. “Biobank‐Scale Inference of Multi‐Individual Identity by Descent and Gene Conversion.” American Journal of Human Genetics
111, no. 4: 691–700.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2024.02.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11023918/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38513668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Biobank%E2%80%90Scale%20Inference%20of%20Multi%E2%80%90Individual%20Identity%20by%20Descent%20and%20Gene%20Conversion&amp;author=S.%20R.%20Browning&amp;author=B.%20L.%20Browning&amp;volume=111&amp;issue=4&amp;publication_year=2024&amp;pages=691-700&amp;pmid=38513668&amp;doi=10.1016/j.ajhg.2024.02.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0015">
<cite>

Browning, S. R.
, and Thompson E. A.. 2012. “Detecting Rare Variant Associations by Identity‐By‐Descent Mapping in Case‐Control Studies.” Genetics
190, no. 4: 1521–1531.
</cite> [<a href="https://doi.org/10.1534/genetics.111.136937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3316661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22267498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Detecting%20Rare%20Variant%20Associations%20by%20Identity%E2%80%90By%E2%80%90Descent%20Mapping%20in%20Case%E2%80%90Control%20Studies&amp;author=S.%20R.%20Browning&amp;author=E.%20A.%20Thompson&amp;volume=190&amp;issue=4&amp;publication_year=2012&amp;pages=1521-1531&amp;pmid=22267498&amp;doi=10.1534/genetics.111.136937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0016">
<cite>

Broyden, C. G.

1970. “The Convergence of a Class of Double‐Rank Minimization Algorithms: 2. The New Algorithm.” IMA Journal of applied mathematics
6, no. 3: 222–231.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IMA%20Journal%20of%20applied%20mathematics&amp;title=The%20Convergence%20of%20a%20Class%20of%20Double%E2%80%90Rank%20Minimization%20Algorithms:%202.%20The%20New%20Algorithm&amp;author=C.%20G.%20Broyden&amp;volume=6&amp;issue=3&amp;publication_year=1970&amp;pages=222-231&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0017">
<cite>

Bycroft, C.
, Freeman C., Petkova D., et al. 2018. “The UK Biobank Resource With Deep Phenotyping and Genomic Data.” Nature
562, no. 7726: 203–209.
</cite> [<a href="https://doi.org/10.1038/s41586-018-0579-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6786975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30305743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20UK%20Biobank%20Resource%20With%20Deep%20Phenotyping%20and%20Genomic%20Data&amp;author=C.%20Bycroft&amp;author=C.%20Freeman&amp;author=D.%20Petkova&amp;volume=562&amp;issue=7726&amp;publication_year=2018&amp;pages=203-209&amp;pmid=30305743&amp;doi=10.1038/s41586-018-0579-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0018">
<cite>

Byrd, R. H.
, Lu P., Nocedal J., and Zhu C.. 1995. “A Limited Memory Algorithm for Bound Constrained Optimization.” SIAM Journal on Scientific Computing
16, no. 5: 1190–1208.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=SIAM%20Journal%20on%20Scientific%20Computing&amp;title=A%20Limited%20Memory%20Algorithm%20for%20Bound%20Constrained%20Optimization&amp;author=R.%20H.%20Byrd&amp;author=P.%20Lu&amp;author=J.%20Nocedal&amp;author=C.%20Zhu&amp;volume=16&amp;issue=5&amp;publication_year=1995&amp;pages=1190-1208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0019">
<cite>

Chen, H.
, Naseri A., and Zhi D.. 2023. “Fimap: A Fast Identity‐By‐Descent Mapping Test for Biobank‐Scale Cohorts.” PLoS Genetics
19, no. 12: e1011057.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1011057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10718418/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38039339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Fimap:%20A%20Fast%20Identity%E2%80%90By%E2%80%90Descent%20Mapping%20Test%20for%20Biobank%E2%80%90Scale%20Cohorts&amp;author=H.%20Chen&amp;author=A.%20Naseri&amp;author=D.%20Zhi&amp;volume=19&amp;issue=12&amp;publication_year=2023&amp;pages=e1011057&amp;pmid=38039339&amp;doi=10.1371/journal.pgen.1011057&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0020">
<cite>

Chernoff, H.

1954. “On the Distribution of the Likelihood Ratio.” Annals of Mathematical Statistics
25: 573–578.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Mathematical%20Statistics&amp;title=On%20the%20Distribution%20of%20the%20Likelihood%20Ratio&amp;author=H.%20Chernoff&amp;volume=25&amp;publication_year=1954&amp;pages=573-578&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0021"><cite>

Davis, T. A.
 Scikit‐Sparse: Sparse Matrix Tools Extending Scipy.sparse. 2021.</cite></li>
<li id="gepi70015-bib-0022">
<cite>

Day‐Williams, A. G.
, Blangero J., Dyer T. D., Lange K., and Sobel E. M.. 2011. “Linkage Analysis Without Defined Pedigrees.” Genetic Epidemiology
35, no. 5: 360–370.
</cite> [<a href="https://doi.org/10.1002/gepi.20584" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3290352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21465549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&amp;title=Linkage%20Analysis%20Without%20Defined%20Pedigrees&amp;author=A.%20G.%20Day%E2%80%90Williams&amp;author=J.%20Blangero&amp;author=T.%20D.%20Dyer&amp;author=K.%20Lange&amp;author=E.%20M.%20Sobel&amp;volume=35&amp;issue=5&amp;publication_year=2011&amp;pages=360-370&amp;pmid=21465549&amp;doi=10.1002/gepi.20584&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0023">
<cite>

Eichler, E. E.
, Flint J., Gibson G., et al. 2010. “Missing Heritability and Strategies for Finding the Underlying Causes of Complex Disease.” Nature Reviews Genetics
11, no. 6: 446–450.</cite> [<a href="https://doi.org/10.1038/nrg2809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2942068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20479774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Missing%20Heritability%20and%20Strategies%20for%20Finding%20the%20Underlying%20Causes%20of%20Complex%20Disease&amp;author=E.%20E.%20Eichler&amp;author=J.%20Flint&amp;author=G.%20Gibson&amp;volume=11&amp;issue=6&amp;publication_year=2010&amp;pages=446-450&amp;pmid=20479774&amp;doi=10.1038/nrg2809&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0024">
<cite>

Fletcher, R.

1970. “A New Approach to Variable Metric Algorithms.” Computer Journal
13, no. 3: 317–322.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Computer%20Journal&amp;title=A%20New%20Approach%20to%20Variable%20Metric%20Algorithms&amp;author=R.%20Fletcher&amp;volume=13&amp;issue=3&amp;publication_year=1970&amp;pages=317-322&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0025">
<cite>

Giri, A.
, Hellwege J. N., Keaton J. M., et al. 2019. “Trans‐Ethnic Association Study of Blood Pressure Determinants in Over 750,000 Individuals.” Nature Genetics
51, no. 1: 51–62.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0303-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6365102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30578418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Trans%E2%80%90Ethnic%20Association%20Study%20of%20Blood%20Pressure%20Determinants%20in%20Over%20750,000%20Individuals&amp;author=A.%20Giri&amp;author=J.%20N.%20Hellwege&amp;author=J.%20M.%20Keaton&amp;volume=51&amp;issue=1&amp;publication_year=2019&amp;pages=51-62&amp;pmid=30578418&amp;doi=10.1038/s41588-018-0303-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0026">
<cite>

Goldfarb, D.

1970. “A Family of Variable Metric Updates Derived by Variational Means.” Mathematics of Computation
v. 24: 21–55.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Mathematics%20of%20Computation&amp;title=A%20Family%20of%20Variable%20Metric%20Updates%20Derived%20by%20Variational%20Means&amp;author=D.%20Goldfarb&amp;volume=v.%2024&amp;publication_year=1970&amp;pages=21-55&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0027">
<cite>

Grinde, K. E.
, Brown L. A., Reiner A. P., Thornton T. A., and Browning S. R.. 2019. “Genome‐Wide Significance Thresholds for Admixture Mapping Studies.” American Journal of Human Genetics
104, no. 3: 454–465.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2019.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6407497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30773276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Genome%E2%80%90Wide%20Significance%20Thresholds%20for%20Admixture%20Mapping%20Studies&amp;author=K.%20E.%20Grinde&amp;author=L.%20A.%20Brown&amp;author=A.%20P.%20Reiner&amp;author=T.%20A.%20Thornton&amp;author=S.%20R.%20Browning&amp;volume=104&amp;issue=3&amp;publication_year=2019&amp;pages=454-465&amp;pmid=30773276&amp;doi=10.1016/j.ajhg.2019.01.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0028">
<cite>

Gusev, A.
, Kenny E. E., Lowe J. K., et al. 2011. “Dash: A Method for Identical‐by‐Descent Haplotype Mapping Uncovers Association With Recent Variation.” American Journal of Human Genetics
88, no. 6: 706–717.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2011.04.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3113343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21620352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Dash:%20A%20Method%20for%20Identical%E2%80%90by%E2%80%90Descent%20Haplotype%20Mapping%20Uncovers%20Association%20With%20Recent%20Variation&amp;author=A.%20Gusev&amp;author=E.%20E.%20Kenny&amp;author=J.%20K.%20Lowe&amp;volume=88&amp;issue=6&amp;publication_year=2011&amp;pages=706-717&amp;pmid=21620352&amp;doi=10.1016/j.ajhg.2011.04.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0029">
<cite>

He, D.

2013. “Ibd‐Groupon: An Efficient Method for Detecting Group‐Wise Identity‐By‐Descent Regions Simultaneously in Multiple Individuals Based on Pairwise IBD Relationships.” Bioinformatics
29, no. 13: i162–i170.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3694672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23812980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Ibd%E2%80%90Groupon:%20An%20Efficient%20Method%20for%20Detecting%20Group%E2%80%90Wise%20Identity%E2%80%90By%E2%80%90Descent%20Regions%20Simultaneously%20in%20Multiple%20Individuals%20Based%20on%20Pairwise%20IBD%20Relationships&amp;author=D.%20He&amp;volume=29&amp;issue=13&amp;publication_year=2013&amp;pages=i162-i170&amp;pmid=23812980&amp;doi=10.1093/bioinformatics/btt237&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0030">
<cite>

Henden, L.
, Twine N. A., Szul P., et al. 2020. “Identity by Descent Analysis Identifies Founder Events and Links SOD1 Familial and Sporadic ALS Cases.” NPJ Genomic Medicine
5, no. 1: 32.
</cite> [<a href="https://doi.org/10.1038/s41525-020-00139-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7414871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32789025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Genomic%20Medicine&amp;title=Identity%20by%20Descent%20Analysis%20Identifies%20Founder%20Events%20and%20Links%20SOD1%20Familial%20and%20Sporadic%20ALS%20Cases&amp;author=L.%20Henden&amp;author=N.%20A.%20Twine&amp;author=P.%20Szul&amp;volume=5&amp;issue=1&amp;publication_year=2020&amp;pages=32&amp;pmid=32789025&amp;doi=10.1038/s41525-020-00139-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0031">
<cite>

Hirschhorn, J. N.
, and Daly M. J.. 2005. “Genome‐Wide Association Studies for Common Diseases and Complex Traits.” Nature Reviews Genetics
6, no. 2: 95–108.</cite> [<a href="https://doi.org/10.1038/nrg1521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15716906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Genome%E2%80%90Wide%20Association%20Studies%20for%20Common%20Diseases%20and%20Complex%20Traits&amp;author=J.%20N.%20Hirschhorn&amp;author=M.%20J.%20Daly&amp;volume=6&amp;issue=2&amp;publication_year=2005&amp;pages=95-108&amp;pmid=15716906&amp;doi=10.1038/nrg1521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0032">
<cite>

Hoffmann, T. J.
, Ehret G. B., Nandakumar P., et al. 2017. “Genome‐Wide Association Analyses Using Electronic Health Records Identify New Loci Influencing Blood Pressure Variation.” Nature Genetics
49, no. 1: 54–64.
</cite> [<a href="https://doi.org/10.1038/ng.3715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5370207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27841878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Genome%E2%80%90Wide%20Association%20Analyses%20Using%20Electronic%20Health%20Records%20Identify%20New%20Loci%20Influencing%20Blood%20Pressure%20Variation&amp;author=T.%20J.%20Hoffmann&amp;author=G.%20B.%20Ehret&amp;author=P.%20Nandakumar&amp;volume=49&amp;issue=1&amp;publication_year=2017&amp;pages=54-64&amp;pmid=27841878&amp;doi=10.1038/ng.3715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0033">
<cite>

Hsueh, W.‐C.
, Nair A. K., Kobes S., et al. 2017. “Identity‐by‐Descent Mapping Identifies Major Locus for Serum Triglycerides in Amerindians Largely Explained by an APOC3 Founder Mutation.” Circulation: Cardiovascular Genetics
10, no. 6: e001809.
</cite> [<a href="https://doi.org/10.1161/CIRCGENETICS.117.001809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5736502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29237685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Circulation:%20Cardiovascular%20Genetics&amp;title=Identity%E2%80%90by%E2%80%90Descent%20Mapping%20Identifies%20Major%20Locus%20for%20Serum%20Triglycerides%20in%20Amerindians%20Largely%20Explained%20by%20an%20APOC3%20Founder%20Mutation&amp;author=W.%E2%80%90C.%20Hsueh&amp;author=A.%20K.%20Nair&amp;author=S.%20Kobes&amp;volume=10&amp;issue=6&amp;publication_year=2017&amp;pages=e001809&amp;pmid=29237685&amp;doi=10.1161/CIRCGENETICS.117.001809&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0034">
<cite>

Kelleher, J.
, and Lohse K.. 2020. “Coalescent Simulation With Msprime.” Statistical Population Genomics
986: 191–230.</cite> [<a href="https://doi.org/10.1007/978-1-0716-0199-0_9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31975169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Statistical%20Population%20Genomics&amp;title=Coalescent%20Simulation%20With%20Msprime&amp;author=J.%20Kelleher&amp;author=K.%20Lohse&amp;volume=986&amp;publication_year=2020&amp;pages=191-230&amp;pmid=31975169&amp;doi=10.1007/978-1-0716-0199-0_9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0035">
<cite>

Lander, E. S.
, and Botstein D.. 1989. “Mapping Mendelian Factors Underlying Quantitative Traits Using Rflp Linkage Maps.” Genetics
121, no. 1: 185–199.
</cite> [<a href="https://doi.org/10.1093/genetics/121.1.185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1203601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2563713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Mapping%20Mendelian%20Factors%20Underlying%20Quantitative%20Traits%20Using%20Rflp%20Linkage%20Maps&amp;author=E.%20S.%20Lander&amp;author=D.%20Botstein&amp;volume=121&amp;issue=1&amp;publication_year=1989&amp;pages=185-199&amp;pmid=2563713&amp;doi=10.1093/genetics/121.1.185&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0036">
<cite>

Lee, S.
, Emond M. J., Bamshad M. J., et al. 2012. “Optimal Unified Approach for Rare‐Variant Association Testing With Application to Small‐Sample Case‐Control Whole‐Exome Sequencing Studies.” American Journal of Human Genetics
91, no. 2: 224–237.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2012.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3415556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22863193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Optimal%20Unified%20Approach%20for%20Rare%E2%80%90Variant%20Association%20Testing%20With%20Application%20to%20Small%E2%80%90Sample%20Case%E2%80%90Control%20Whole%E2%80%90Exome%20Sequencing%20Studies&amp;author=S.%20Lee&amp;author=M.%20J.%20Emond&amp;author=M.%20J.%20Bamshad&amp;volume=91&amp;issue=2&amp;publication_year=2012&amp;pages=224-237&amp;pmid=22863193&amp;doi=10.1016/j.ajhg.2012.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0037">
<cite>

Liu, Y.
, Chen S., Li Z., Morrison A. C., Boerwinkle E., and Lin X.. 2019. “ACAT: A Fast and Powerful P Value Combination Method for Rare‐Variant Analysis in Sequencing Studies.” American Journal of Human Genetics
104, no. 3: 410–421.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2019.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6407498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30849328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=ACAT:%20A%20Fast%20and%20Powerful%20P%20Value%20Combination%20Method%20for%20Rare%E2%80%90Variant%20Analysis%20in%20Sequencing%20Studies&amp;author=Y.%20Liu&amp;author=S.%20Chen&amp;author=Z.%20Li&amp;author=A.%20C.%20Morrison&amp;author=E.%20Boerwinkle&amp;volume=104&amp;issue=3&amp;publication_year=2019&amp;pages=410-421&amp;pmid=30849328&amp;doi=10.1016/j.ajhg.2019.01.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0038">
<cite>

Madsen, B. E.
, and Browning S. R.. 2009. “A Groupwise Association Test for Rare Mutations Using a Weighted Sum Statistic.” PLoS Genetics
5, no. 2: e1000384.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1000384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2633048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19214210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=A%20Groupwise%20Association%20Test%20for%20Rare%20Mutations%20Using%20a%20Weighted%20Sum%20Statistic&amp;author=B.%20E.%20Madsen&amp;author=S.%20R.%20Browning&amp;volume=5&amp;issue=2&amp;publication_year=2009&amp;pages=e1000384&amp;pmid=19214210&amp;doi=10.1371/journal.pgen.1000384&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0039">
<cite>

Marees, A. T.
, De Kluiver H., Stringer S., et al. 2018. “A Tutorial on Conducting Genome‐Wide Association Studies: Quality Control and Statistical Analysis.” International Journal of Methods in Psychiatric Research
27, no. 2: e1608.
</cite> [<a href="https://doi.org/10.1002/mpr.1608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6001694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29484742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Methods%20in%20Psychiatric%20Research&amp;title=A%20Tutorial%20on%20Conducting%20Genome%E2%80%90Wide%20Association%20Studies:%20Quality%20Control%20and%20Statistical%20Analysis&amp;author=A.%20T.%20Marees&amp;author=H.%20De%20Kluiver&amp;author=S.%20Stringer&amp;volume=27&amp;issue=2&amp;publication_year=2018&amp;pages=e1608&amp;pmid=29484742&amp;doi=10.1002/mpr.1608&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0040">
<cite>

McCarthy, M. I.
, Abecasis G. R., Cardon L. R., et al. 2008. “Genome‐Wide Association Studies for Complex Traits: Consensus, Uncertainty and Challenges.” Nature Reviews Genetics
9, no. 5: 356–369.</cite> [<a href="https://doi.org/10.1038/nrg2344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18398418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Genome%E2%80%90Wide%20Association%20Studies%20for%20Complex%20Traits:%20Consensus,%20Uncertainty%20and%20Challenges&amp;author=M.%20I.%20McCarthy&amp;author=G.%20R.%20Abecasis&amp;author=L.%20R.%20Cardon&amp;volume=9&amp;issue=5&amp;publication_year=2008&amp;pages=356-369&amp;pmid=18398418&amp;doi=10.1038/nrg2344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0041">
<cite>

McCarthy, M. I.
, and Hirschhorn J. N.. 2008. “Genome‐Wide Association Studies: Potential Next Steps on a Genetic Journey.” Human Molecular Genetics
17, no. R2: R156–R165.
</cite> [<a href="https://doi.org/10.1093/hmg/ddn289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2782356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18852205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&amp;title=Genome%E2%80%90Wide%20Association%20Studies:%20Potential%20Next%20Steps%20on%20a%20Genetic%20Journey&amp;author=M.%20I.%20McCarthy&amp;author=J.%20N.%20Hirschhorn&amp;volume=17&amp;issue=R2&amp;publication_year=2008&amp;pages=R156-R165&amp;pmid=18852205&amp;doi=10.1093/hmg/ddn289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0042">
<cite>

Miller, J. J.

1977. “Asymptotic Properties of Maximum Likelihood Estimates in the Mixed Model of the Analysis of Variance.” Annals of Statistics
5: 746–762.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Statistics&amp;title=Asymptotic%20Properties%20of%20Maximum%20Likelihood%20Estimates%20in%20the%20Mixed%20Model%20of%20the%20Analysis%20of%20Variance&amp;author=J.%20J.%20Miller&amp;volume=5&amp;publication_year=1977&amp;pages=746-762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0043">
<cite>

Moltke, I.
, Albrechtsen A., Hansen T. O., Nielsen F. C., and Nielsen R.. 2011. “A Method for Detecting Ibd Regions Simultaneously in Multiple Individuals—With Applications to Disease Genetics.” Genome Research
21, no. 7: 1168–1180.
</cite> [<a href="https://doi.org/10.1101/gr.115360.110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3129259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21493780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=A%20Method%20for%20Detecting%20Ibd%20Regions%20Simultaneously%20in%20Multiple%20Individuals%E2%80%94With%20Applications%20to%20Disease%20Genetics&amp;author=I.%20Moltke&amp;author=A.%20Albrechtsen&amp;author=T.%20O.%20Hansen&amp;author=F.%20C.%20Nielsen&amp;author=R.%20Nielsen&amp;volume=21&amp;issue=7&amp;publication_year=2011&amp;pages=1168-1180&amp;pmid=21493780&amp;doi=10.1101/gr.115360.110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0044">
<cite>

Page, G. P.
, Amos C. I., and Boerwinkle E.. 1998. “The Quantitative LOD Score: Test Statistic and Sample Size for Exclusion and Linkage of Quantitative Traits in Human Sibships.” American Journal of Human Genetics
62, no. 4: 962–968.
</cite> [<a href="https://doi.org/10.1086/301783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1377023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9529341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=The%20Quantitative%20LOD%20Score:%20Test%20Statistic%20and%20Sample%20Size%20for%20Exclusion%20and%20Linkage%20of%20Quantitative%20Traits%20in%20Human%20Sibships&amp;author=G.%20P.%20Page&amp;author=C.%20I.%20Amos&amp;author=E.%20Boerwinkle&amp;volume=62&amp;issue=4&amp;publication_year=1998&amp;pages=962-968&amp;pmid=9529341&amp;doi=10.1086/301783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0045">
<cite>

Plotnikov, D.
, Huang Y., Khawaja A. P., et al. 2022. “High Blood Pressure and Intraocular Pressure: A Mendelian Randomization Study.” Investigative Ophthalmology &amp; Visual Science
63, no. 6: 29.</cite> [<a href="https://doi.org/10.1167/iovs.63.6.29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9251815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35762941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Investigative%20Ophthalmology%20&amp;%20Visual%20Science&amp;title=High%20Blood%20Pressure%20and%20Intraocular%20Pressure:%20A%20Mendelian%20Randomization%20Study&amp;author=D.%20Plotnikov&amp;author=Y.%20Huang&amp;author=A.%20P.%20Khawaja&amp;volume=63&amp;issue=6&amp;publication_year=2022&amp;pages=29&amp;pmid=35762941&amp;doi=10.1167/iovs.63.6.29&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0046">
<cite>

Purcell, S.
, Neale B., Todd‐Brown K., et al. 2007. “Plink: A Tool Set for Whole‐Genome Association and Population‐Based Linkage Analyses.” American Journal of Human Genetics
81, no. 3: 559–575.
</cite> [<a href="https://doi.org/10.1086/519795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Plink:%20A%20Tool%20Set%20for%20Whole%E2%80%90Genome%20Association%20and%20Population%E2%80%90Based%20Linkage%20Analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd%E2%80%90Brown&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0047">
<cite>

Qian, Y.
, Browning B. L., and Browning S. R.. 2014. “Efficient Clustering of Identity‐by‐Descent Between Multiple Individuals.” Bioinformatics
30, no. 7: 915–922.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3967111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24363374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Efficient%20Clustering%20of%20Identity%E2%80%90by%E2%80%90Descent%20Between%20Multiple%20Individuals&amp;author=Y.%20Qian&amp;author=B.%20L.%20Browning&amp;author=S.%20R.%20Browning&amp;volume=30&amp;issue=7&amp;publication_year=2014&amp;pages=915-922&amp;pmid=24363374&amp;doi=10.1093/bioinformatics/btt734&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0048">
<cite>

Sakaue, S.
, Kanai M., Tanigawa Y., et al. 2021. “A Cross‐Population Atlas of Genetic Associations for 220 Human Phenotypes.” Nature Genetics
53, no. 10: 1415–1424.
</cite> [<a href="https://doi.org/10.1038/s41588-021-00931-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12208603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34594039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=A%20Cross%E2%80%90Population%20Atlas%20of%20Genetic%20Associations%20for%20220%20Human%20Phenotypes&amp;author=S.%20Sakaue&amp;author=M.%20Kanai&amp;author=Y.%20Tanigawa&amp;volume=53&amp;issue=10&amp;publication_year=2021&amp;pages=1415-1424&amp;pmid=34594039&amp;doi=10.1038/s41588-021-00931-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0049">
<cite>

Self, S. G.
, and Liang K.‐Y.. 1987. “Asymptotic Properties of Maximum Likelihood Estimators and Likelihood Ratio Tests Under Nonstandard Conditions.” Journal of the American Statistical Association
82, no. 398: 605–610.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=Asymptotic%20Properties%20of%20Maximum%20Likelihood%20Estimators%20and%20Likelihood%20Ratio%20Tests%20Under%20Nonstandard%20Conditions&amp;author=S.%20G.%20Self&amp;author=K.%E2%80%90Y.%20Liang&amp;volume=82&amp;issue=398&amp;publication_year=1987&amp;pages=605-610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0050">
<cite>

Shanno, D. F.

1970. “Conditioning of Quasi‐Newton Methods for Function Minimization.” Mathematics of Computation
24, no. 111: 647–656.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Mathematics%20of%20Computation&amp;title=Conditioning%20of%20Quasi%E2%80%90Newton%20Methods%20for%20Function%20Minimization&amp;author=D.%20F.%20Shanno&amp;volume=24&amp;issue=111&amp;publication_year=1970&amp;pages=647-656&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0051">
<cite>

Shor, T.
, Kalka I., Geiger D., Erlich Y., and Weissbrod O.. 2019. “Estimating Variance Components in Population Scale Family Trees.” PLoS Genetics
15, no. 5: e1008124.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6529016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31071088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Estimating%20Variance%20Components%20in%20Population%20Scale%20Family%20Trees&amp;author=T.%20Shor&amp;author=I.%20Kalka&amp;author=D.%20Geiger&amp;author=Y.%20Erlich&amp;author=O.%20Weissbrod&amp;volume=15&amp;issue=5&amp;publication_year=2019&amp;pages=e1008124&amp;pmid=31071088&amp;doi=10.1371/journal.pgen.1008124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0052">
<cite>

Siegmund, D.
, and Yakir B.. 2007. The Statistics of Gene Mapping (1). Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20Statistics%20of%20Gene%20Mapping&amp;author=D.%20Siegmund&amp;author=B.%20Yakir&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0053">
<cite>

Tam, V.
, Patel N., Turcotte M., Bossé Y., Paré G., and Meyre D.. 2019. “Benefits and Limitations of Genome‐Wide Association Studies.” Nature Reviews Genetics
20, no. 8: 467–484.</cite> [<a href="https://doi.org/10.1038/s41576-019-0127-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31068683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Benefits%20and%20Limitations%20of%20Genome%E2%80%90Wide%20Association%20Studies&amp;author=V.%20Tam&amp;author=N.%20Patel&amp;author=M.%20Turcotte&amp;author=Y.%20Boss%C3%A9&amp;author=G.%20Par%C3%A9&amp;volume=20&amp;issue=8&amp;publication_year=2019&amp;pages=467-484&amp;pmid=31068683&amp;doi=10.1038/s41576-019-0127-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0054">
<cite>

Vacic, V.
, Ozelius L. J., Clark L. N., et al. 2014. “Genome‐Wide Mapping of IBD Segments in an Ashkenazi PD Cohort Identifies Associated Haplotypes.” Human Molecular Genetics
23, no. 17: 4693–4702.
</cite> [<a href="https://doi.org/10.1093/hmg/ddu158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4119402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24842889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&amp;title=Genome%E2%80%90Wide%20Mapping%20of%20IBD%20Segments%20in%20an%20Ashkenazi%20PD%20Cohort%20Identifies%20Associated%20Haplotypes&amp;author=V.%20Vacic&amp;author=L.%20J.%20Ozelius&amp;author=L.%20N.%20Clark&amp;volume=23&amp;issue=17&amp;publication_year=2014&amp;pages=4693-4702&amp;pmid=24842889&amp;doi=10.1093/hmg/ddu158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0055">
<cite>

Virtanen, P.
, Gommers R., Oliphant T. E., et al. 2020. “SciPy 1.0: Fundamental Algorithms for Scientific Computing in Python.” Nature Methods
17, no. 3: 261–272.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0686-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7056644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32015543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=SciPy%201.0:%20Fundamental%20Algorithms%20for%20Scientific%20Computing%20in%20Python&amp;author=P.%20Virtanen&amp;author=R.%20Gommers&amp;author=T.%20E.%20Oliphant&amp;volume=17&amp;issue=3&amp;publication_year=2020&amp;pages=261-272&amp;pmid=32015543&amp;doi=10.1038/s41592-019-0686-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0056">
<cite>

Visscher, P. M.
, Wray N. R., Zhang Q., et al. 2017. “10 Years of GWAS Discovery: Biology, Function, and Translation.” American Journal of Human Genetics
101, no. 1: 5–22.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2017.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5501872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28686856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=10%20Years%20of%20GWAS%20Discovery:%20Biology,%20Function,%20and%20Translation&amp;author=P.%20M.%20Visscher&amp;author=N.%20R.%20Wray&amp;author=Q.%20Zhang&amp;volume=101&amp;issue=1&amp;publication_year=2017&amp;pages=5-22&amp;pmid=28686856&amp;doi=10.1016/j.ajhg.2017.06.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0057">
<cite>

Warren, H. R.
, Evangelou E., Cabrera C. P., et al. 2017. “Genome‐Wide Association Analysis Identifies Novel Blood Pressure Loci and Offers Biological Insights Into Cardiovascular Risk.” Nature Genetics
49, no. 3: 403–415.
</cite> [<a href="https://doi.org/10.1038/ng.3768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5972004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28135244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Genome%E2%80%90Wide%20Association%20Analysis%20Identifies%20Novel%20Blood%20Pressure%20Loci%20and%20Offers%20Biological%20Insights%20Into%20Cardiovascular%20Risk&amp;author=H.%20R.%20Warren&amp;author=E.%20Evangelou&amp;author=C.%20P.%20Cabrera&amp;volume=49&amp;issue=3&amp;publication_year=2017&amp;pages=403-415&amp;pmid=28135244&amp;doi=10.1038/ng.3768&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0058">
<cite>

Westerlind, H.
, Imrell K., Ramanujam R., et al. 2015. “Identity‐by‐Descent Mapping in a Scandinavian Multiple Sclerosis Cohort.” European Journal of Human Genetics
23, no. 5: 688–692.
</cite> [<a href="https://doi.org/10.1038/ejhg.2014.155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25159868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Human%20Genetics&amp;title=Identity%E2%80%90by%E2%80%90Descent%20Mapping%20in%20a%20Scandinavian%20Multiple%20Sclerosis%20Cohort&amp;author=H.%20Westerlind&amp;author=K.%20Imrell&amp;author=R.%20Ramanujam&amp;volume=23&amp;issue=5&amp;publication_year=2015&amp;pages=688-692&amp;pmid=25159868&amp;doi=10.1038/ejhg.2014.155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0059"><cite>
Wolfram Research I
. Mathematica. Version 14.1 ed. Champaign, Illinois: Wolfram Research, Inc.; 2024.</cite></li>
<li id="gepi70015-bib-0060">
<cite>

Wu, M. C.
, Lee S., Cai T., Li Y., Boehnke M., and Lin X.. 2011. “Rare‐Variant Association Testing for Sequencing Data With the Sequence Kernel Association Test.” American Journal of Human Genetics
89, no. 1: 82–93.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2011.05.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3135811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21737059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Rare%E2%80%90Variant%20Association%20Testing%20for%20Sequencing%20Data%20With%20the%20Sequence%20Kernel%20Association%20Test&amp;author=M.%20C.%20Wu&amp;author=S.%20Lee&amp;author=T.%20Cai&amp;author=Y.%20Li&amp;author=M.%20Boehnke&amp;volume=89&amp;issue=1&amp;publication_year=2011&amp;pages=82-93&amp;pmid=21737059&amp;doi=10.1016/j.ajhg.2011.05.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0061">
<cite>

Zhou, Y.
, Browning S. R., and Browning B. L.. 2020a. “A Fast and Method for Detecting Identity‐by‐Descent Segments in Large‐Scale Data.” American Journal of Human Genetics
106, no. 4: 426–437.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2020.02.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7118582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32169169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=A%20Fast%20and%20Method%20for%20Detecting%20Identity%E2%80%90by%E2%80%90Descent%20Segments%20in%20Large%E2%80%90Scale%20Data&amp;author=Y.%20Zhou&amp;author=S.%20R.%20Browning&amp;author=B.%20L.%20Browning&amp;volume=106&amp;issue=4&amp;publication_year=2020a&amp;pages=426-437&amp;pmid=32169169&amp;doi=10.1016/j.ajhg.2020.02.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0062">
<cite>

Zhou, Y.
, Browning S. R., and Browning B. L.. 2020b. “IBDkin: Fast Estimation of Kinship Coefficients From Identity by Descent Segments.” Bioinformatics
36, no. 16: 4519–4520.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7750976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32844204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=IBDkin:%20Fast%20Estimation%20of%20Kinship%20Coefficients%20From%20Identity%20by%20Descent%20Segments&amp;author=Y.%20Zhou&amp;author=S.%20R.%20Browning&amp;author=B.%20L.%20Browning&amp;volume=36&amp;issue=16&amp;publication_year=2020b&amp;pages=4519-4520&amp;pmid=32844204&amp;doi=10.1093/bioinformatics/btaa569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70015-bib-0063">
<cite>

Zhu, X.
, Zhu L., Wang H., Cooper R. S., and Chakravarti A.. 2022. “Genome‐Wide Pleiotropy Analysis Identifies Novel Blood Pressure Variants and Improves Its Polygenic Risk Scores.” Genetic Epidemiology
46, no. 2: 105–121.
</cite> [<a href="https://doi.org/10.1002/gepi.22440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8863647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34989438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&amp;title=Genome%E2%80%90Wide%20Pleiotropy%20Analysis%20Identifies%20Novel%20Blood%20Pressure%20Variants%20and%20Improves%20Its%20Polygenic%20Risk%20Scores&amp;author=X.%20Zhu&amp;author=L.%20Zhu&amp;author=H.%20Wang&amp;author=R.%20S.%20Cooper&amp;author=A.%20Chakravarti&amp;volume=46&amp;issue=2&amp;publication_year=2022&amp;pages=105-121&amp;pmid=34989438&amp;doi=10.1002/gepi.22440&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>
<strong>Figure S1:</strong> Estimated decay parameter for genome‐wide IBD mapping tests under different algorithm parameters for multi‐individual IBD detection.</p>
<p>
<strong>Figure S2:</strong> Empirical correlation between test statistics spaced at cM from genome‐wide IBD mapping tests with phenotypes simulated under the null hypothesis on simulated sequence data, using different haplotype length thresholds (<strong>
<em>L</em>
</strong>) and trimming thresholds (<strong>
<em>T</em>
</strong>) for multi‐individual IBD detection.</p>
<p>
<strong>Figure S3:</strong> Empirical correlation between test statistics spaced at cM from genome‐wide IBD mapping tests with phenotypes simulated under the null hypothesis on simulated SNP array data, using different haplotype length thresholds (<strong>
<em>L</em>
</strong>) and trimming thresholds (<strong>
<em>T</em>
</strong>) for multi‐individual IBD detection.</p>
<p>
<strong>Figure S4:</strong> Empirical correlation between test statistics from genome‐wide IBD mapping tests with phenotypes simulated under the null hypothesis on the array data of 124,376 White British individuals in the UK Biobank.</p>
<p>
<strong>Figure S5:</strong> The power of our IBD mapping test for detecting common (&gt;10% MAF), low‐frequency (1‐10% MAF), rare (0.05‐1% MAF), and ultra‐rare (&lt;0.05% MAF) causal variants using the Bonferroni correction. Figure S6: Results of applying the FiMAP test at 1 cM intervals across the genome to systolic blood pressure data from 124k White British individuals in the UK Biobank.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12327188/bin/GEPI-49-0-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">GEPI-49-0-s001.docx</a><sup> (812.8KB, docx) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genetic Epidemiology are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/gepi.70015"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/GEPI-49-0.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (901.0 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12327188/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12327188/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12327188%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327188/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12327188/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12327188/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40719186/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12327188/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40719186/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12327188/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12327188/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="gb3Fe9o8abDEsoHMyR8FSg9cBhCAX3vEV8LGI2ZmVMsS6slNOSLqc2QKOPQYDDyP">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
