
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Halomonas sp for sustainable agriculture a potential halo bio fertilizer for tomato plants with biocontrol activity against Fusarium wilt under saline environments - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525444D8AF3005305444D0004F40156.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370916/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Halomonas sp for sustainable agriculture a potential halo bio fertilizer for tomato plants with biocontrol activity against Fusarium wilt under saline environments">
<meta name="citation_author" content="Ahmed Ahmed Abdelmonaem Mousa">
<meta name="citation_author_institution" content="Botany Department, Faculty of Agriculture, Menoufia University, Shibin El Kom, Egypt">
<meta name="citation_author" content="Wafaa Hanafy Mahmoud">
<meta name="citation_author_institution" content="Botany Department, Faculty of Agriculture, Menoufia University, Shibin El Kom, Egypt">
<meta name="citation_author" content="Hosam Easa Elsaied">
<meta name="citation_author_institution" content="Department of Genetics and Genetic Engineering, National Institute of Oceanography and Fisheries, NIOF, Cairo, Egypt">
<meta name="citation_author" content="Adel Elsayed Elbeltagy">
<meta name="citation_author_institution" content="Botany Department, Faculty of Agriculture, Menoufia University, Shibin El Kom, Egypt">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30748">
<meta name="citation_doi" content="10.1038/s41598-025-12974-9">
<meta name="citation_pmid" content="40841562">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370916/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370916/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370916/pdf/41598_2025_Article_12974.pdf">
<meta name="description" content="Halophilic bacteria are remarkable microorganisms that excel in hypersaline environments. Their significant potential in various fields, such as industry and agriculture, positions them as vital players in advancing our technological and ecological ...">
<meta name="og:title" content="Halomonas sp for sustainable agriculture a potential halo bio fertilizer for tomato plants with biocontrol activity against Fusarium wilt under saline environments">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Halophilic bacteria are remarkable microorganisms that excel in hypersaline environments. Their significant potential in various fields, such as industry and agriculture, positions them as vital players in advancing our technological and ecological ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370916/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370916">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-12974-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_12974.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370916%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370916/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370916/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370916/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 21;15:30748. doi: <a href="https://doi.org/10.1038/s41598-025-12974-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-12974-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
<em>Halomonas</em> sp for sustainable agriculture a potential halo bio fertilizer for tomato plants with biocontrol activity against <em>Fusarium</em> wilt under saline environments</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mousa%20AAA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ahmed Ahmed Abdelmonaem Mousa</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ahmed Ahmed Abdelmonaem Mousa</span></h3>
<div class="p">
<sup>1</sup>Botany Department, Faculty of Agriculture, Menoufia University, Shibin El Kom, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mousa%20AAA%22%5BAuthor%5D" class="usa-link"><span class="name western">Ahmed Ahmed Abdelmonaem Mousa</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahmoud%20WH%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wafaa Hanafy Mahmoud</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wafaa Hanafy Mahmoud</span></h3>
<div class="p">
<sup>1</sup>Botany Department, Faculty of Agriculture, Menoufia University, Shibin El Kom, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahmoud%20WH%22%5BAuthor%5D" class="usa-link"><span class="name western">Wafaa Hanafy Mahmoud</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elsaied%20HE%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hosam Easa Elsaied</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hosam Easa Elsaied</span></h3>
<div class="p">
<sup>2</sup>Department of Genetics and Genetic Engineering, National Institute of Oceanography and Fisheries, NIOF, Cairo, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elsaied%20HE%22%5BAuthor%5D" class="usa-link"><span class="name western">Hosam Easa Elsaied</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elbeltagy%20AE%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Adel Elsayed Elbeltagy</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Adel Elsayed Elbeltagy</span></h3>
<div class="p">
<sup>1</sup>Botany Department, Faculty of Agriculture, Menoufia University, Shibin El Kom, Egypt </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elbeltagy%20AE%22%5BAuthor%5D" class="usa-link"><span class="name western">Adel Elsayed Elbeltagy</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Botany Department, Faculty of Agriculture, Menoufia University, Shibin El Kom, Egypt </div>
<div id="Aff2">
<sup>2</sup>Department of Genetics and Genetic Engineering, National Institute of Oceanography and Fisheries, NIOF, Cairo, Egypt </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 12; Accepted 2025 Jul 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370916  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841562/" class="usa-link">40841562</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Halophilic bacteria are remarkable microorganisms that excel in hypersaline environments. Their significant potential in various fields, such as industry and agriculture, positions them as vital players in advancing our technological and ecological efforts. In this study, three bacterial strains (QSLA1, QSLA2, and QSLA3) were successfully isolated from solar saltern ponds, attached to Qarun Lake, Fayoum governorate, Egypt, using nutrient agar (NA) culture medium derived from pond water. Morphological and physiological characterization revealed that these isolates are rod-shaped, gram-negative, catalase-positive, and motile. All isolates were identified as not spore-forming bacteria. The halo tolerance assay demonstrated that QSLA1 and QSLA2 are extremely halophilic, whereas QSLA3 is classified as moderately halophilic. Through 16S rRNA sequence analysis, it was determined that QSLA1 shares 91.26% similarity with <em>Halomonas</em> sp. RS-17, while QSLA2 exhibits 96.6% similarity with <em>Halomonas</em> sp. strain LR2-3. QSLA3 shows even greater similarity at 97.33% to <em>Halomonas</em> sp. GQ30. All isolates are capable of producing indole-3-acetic acid (IAA), but only QSLA2 has the ability to fix atmospheric nitrogen and solubilize insoluble phosphate. Additionally, QSLA1 demonstrates antifungal activity against <em>Fusarium oxysporium f.sp. lycopersici</em> in vitro under saline environment. Given these promising traits, we explored the potential of QSLA1 as a bio-control agent under greenhouse conditions at 1.5% salinity. These findings suggest that these bacterial strains could be used to develop sustainable agricultural practices, enhancing crop yields and reducing the reliance on chemical fertilizers and pesticides. Future applications of these strains could provide a valuable solution for improving agricultural productivity in saline environments.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Hypersaline ecosystems are dynamic and diverse habitats that include various terrestrial lakes and deep-sea basins, exhibiting salt concentrations that exceed three times that of seawater, reaching saturation levels. These habitats are effectively categorized into two main types: thalassohaline and athalassohaline waters<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Solar salterns, widely distributed across arid and semi-arid regions, can be found at sea level, either as natural formations or as human-made innovations<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>.</p>
<p id="Par3">Qarun Lake, strategically located in the northern region of Fayoum Governorate, is characterized by its saline, turbid waters and lack of surface outflow<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. The solar salterns surrounding Qarun Lake feature a well-structured multi-pond system that is interconnected. These ponds are actively utilized by local companies for the efficient production of salt and minerals.</p>
<p id="Par4">Salinity stress significantly challenges agriculture by reducing productivity, limiting arable land, and causing ionic and osmotic imbalances in plants, which disrupts physiological processes and leads to soil degradation. However, halophilic/halotolerant plant growth-promoting bacteria (PGPB) provide sustainable solutions to mitigate saline stress and enhance plant growth due to their ability to adapt to extreme environments through mechanisms such as ion homeostasis, osmotic balance, and production of compatible solutes<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par5">Halophiles, the remarkable salt-loving microorganisms that thrive in saline environments, are integral components of hypersaline ecosystems. They span all three domains of life: Archaea, Bacteria, and Eukarya<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. These halophilic bacteria present significant opportunities in agriculture, where they can be employed for bio-control of phytopathogens, the solubilization of essential nutrients, and the stimulation of plant growth through the production of beneficial growth factors<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p>
<p id="Par6">Tomato fruits (<em>Lycopersicon esculentum</em>) rank among the most popular and widely consumed vegetables globally, especially in Egypt<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. However, tomato plants face challenges from various fungal diseases, with <em>Fusarium</em> wilt being one of the most critical. This disease caused by the soil-borne fungus <em>Fusarium oxysporium f.sp. lycopersici</em> (Fol), can lead to significant yield losses in both greenhouse and field conditions<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. In Egypt, <em>Fusarium</em> wilt can reduce yields by as much as 25%<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. The pathogen infiltrates the plant through the roots, colonizes the vascular tissues, clogs the xylem, and induces water stress, which manifests as wilt-like symptoms<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p>
<p id="Par7">Research by<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> has demonstrated the effectiveness of moderately halophilic bacteria, particularly <em>Bacillus subtilis</em> J9 and <em>Halomonas</em> sp. K2-5, in managing stem canker in greenhouse tomatoes. <em>Halomonas</em> sp., isolated from a saline habitat in northeastern Algeria, has shown a broad spectrum of antifungal activity against pathogens such as <em>Fusarium oxysporium, Botrytis cinerea, Phytophthora capsici,</em> and <em>F. verticillioides</em><sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par8">In this study, halophilic bacterial strains were isolated from the solar saltern ponds of a local salt and mineral company. The strains were rigorously tested for their potential to enhance plant growth and exhibit antimicrobial activities against plant pathogens. We meticulously assessed the impact of the isolate QSLA1 on Fusarium wilt disease as well as its effects on the growth of tomato seedlings under saline conditions in controlled greenhouse environments.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Bacterial isolates</h3>
<p id="Par9">In September 2019, water samples were collected from solar saltern ponds, ensuring a comprehensive analysis of their properties. After filtering to remove impurities, the salt concentration was accurately assessed using a refractometer and pH levels were determined with precision using a pH meter. Ten liters of water were gathered from each pond in sterile plastic jars, which were meticulously placed in ice packs for transportation to the biotechnology laboratory in the Botany Department at Menoufia University.</p>
<p id="Par10">To isolate bacteria, we employed sterilized Nutrient Agar (NA) medium prepared using pond water as follow: 5 gm peptone, 3 gm yeast extract, 15 gm agar agar, and 1 L pond water. A 100 µl aliquot of each water sample was streaked onto NA plates and incubated at 30 °C for periods ranging from 7 to 30 days<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Subsequent streaking and sub-culturing were conducted on NA culture medium prepared from the pond water, successfully purifying the isolates, which were then preserved in 30% glycerol and stored in a freezer for future analysis.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Morphological and physiological characterization of the isolates</h3>
<p id="Par11">The colonies that developed on the plates underwent thorough morphological examination, focusing on their shape, pigmentation, elevation, and optical properties. Additional parameters including shape analysis, gram staining, endospore formation<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> were rigorously tested as well as bacterial motility using the hanging drop method<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, and catalase activity<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Salinity tolerance of the isolates</h3>
<p id="Par12">The bacterial isolates were effectively screened for salt tolerance by using a NA culture medium supplemented with varying concentrations of NaCl (0%, 7%, 12.5%, 20%, and 22%). The plates were incubated for 7 days at 30 °C, and the growth results were meticulously recorded<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Molecular identification of the isolates via 16S rDNA</h3>
<section id="Sec7"><h4 class="pmc_sec_title">DNA extraction and polymerase chain reaction (PCR)</h4>
<p id="Par13">Genomic DNA was extracted from bacterial cells cultured aerobically in nutrient broth<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. The extracted DNA was purified, visualized under UV light with ethidium bromide staining<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, and securely stored at −20 °C until needed<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Utilizing bacterial primers 27F and 1492R, as specified in Table <a href="#Tab1" class="usa-link">1</a>, the 16S rRNA gene sequence was successfully amplified using a model PTC-100 thermal cycler (MJ Research Inc., USA)<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. The presence of amplified products was confirmed by applying 7 μl of the PCR product onto a 1% agarose gel in 1X TAE buffer containing ethidium bromide, and results were visualized with a gel documentation system (Bio-Rad Laboratories)<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The PCR products were purified using the QIA quick PCR purification kit protocol (Qiagen) and promptly sent for sequencing.</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Primers used for PCR 16S rRNA sequencing analysis<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Primer name</th>
<th align="left" colspan="1" rowspan="1">Orientation</th>
<th align="left" colspan="1" rowspan="1">Priming site</th>
<th align="left" colspan="1" rowspan="1">Sequence (5´–3´)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">27F</td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">8–27</td>
<td align="left" colspan="1" rowspan="1">AGAGTTTGATCCTGGCTCAG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1492R</td>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">1492–1513</td>
<td align="left" colspan="1" rowspan="1">GGTTACCTTGTTACGACTT</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec8"><h4 class="pmc_sec_title">16S rRNA sequencing</h4>
<p id="Par14">The sequencing was expertly conducted by Colors Laboratories at El-Etihad Square in Maadi, Cairo, Egypt, following rigorous laboratory protocols. The sequences were thoroughly edited using the Complete Deletion option to eliminate all gaps, utilizing CHROMAS PRO software, version 1.5.</p></section><section id="Sec9"><h4 class="pmc_sec_title">Evolutionary relationships of taxa</h4>
<p id="Par15">The resultant 16S rRNA gene sequences were successfully compared with those available in the GenBank databases using the highly effective Basic Local Alignment Search Tool (BLAST) on the National Center for Biotechnology Information (NCBI) website (<a href="http://www.ncbi.nih.gov" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nih.gov</a>). For alignment, CLUSTAL W 1.6 software was employed<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>, ensuring precise and reliable results. The evolutionary history was robustly inferred using the Neighbor-Joining method<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Evolutionary analyses were carried out with confidence using MEGA X<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. The sequences were meticulously aligned using the embedded MUSCLE algorithm, and the resulting output was leveraged to construct a phylogenetic tree by calculating distance matrices for Neighbor-Joining (NJ) analysis.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Evaluation of plant growth under laboratory conditions</h4>
<p id="Par16">The isolates were rigorously evaluated for their growth-promoting properties, with all analyses conducted in duplicate to ensure accuracy. The inoculum for screening was expertly prepared by cultivating halo-bacteria in 20 ml of nutrient broth culture medium, enriched with a 12.5% salt concentration, and subjected to shaking at 120 rpm for three days.</p></section><section id="Sec11"><h4 class="pmc_sec_title">Nitrogen fixation test</h4>
<p id="Par17">The capability of bacterial isolates to fix free nitrogen was thoroughly tested using Ashby-free N-agar medium<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Bacteria that successfully grew in this medium are unequivocally identified as nitrogen-fixing organisms<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Phosphate solubilization assay</h4>
<p id="Par18">The potential of isolates to solubilize inorganic phosphate was determined using the disk diffusion method on a modified Pikovskaya agar culture medium<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. Colonies of phosphate-solubilizing bacteria (PSB) were clearly identified by the distinct diffusion zones surrounding them, as highlighted by<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>.</p></section><section id="Sec13"><h4 class="pmc_sec_title">Indole acetic acid (iaa) production assay</h4>
<p id="Par19">Isolates were confidently screened for IAA production utilizing a refined quantification method<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The production of IAA by bacterial isolates was clearly indicated by a distinct pink color change upon the addition of Salkowski’s reagent, as illustrated in Fig. <a href="#Fig3" class="usa-link">3</a>. The IAA concentration was accurately estimated against a well-prepared standard curve derived from various concentrations of IAA (Sigma-Aldrich, Germany)<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/0d80c06bbefa/41598_2025_12974_Fig3_HTML.jpg" loading="lazy" id="MO3" height="581" width="664" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>demonstrates the vibrant color development resulting from the Indole-Salkowski reagent reaction in nutrient broth medium enriched with 1% L-tryptophan.</p></figcaption></figure></section><section id="Sec14"><h4 class="pmc_sec_title">Antagonistic activity of the isolates against plant pathogens</h4>
<p id="Par20">The antifungal activity of bacterial isolates was rigorously assessed through co-cultivation of macro-colonies against key pathogens, including <em>Fusarium oxysporium f.sp. lycopersici</em>, a well-known cause of tomato vascular wilt disease<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>; <em>Alternaria solani</em>, responsible for early blight in tomato and potato plants<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>; and <em>Botryodiplodia theobromae</em>, which leads to die-back and stem-end rot in mango fruits<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. For this assessment, the pathogens were strategically inoculated onto NA culture medium (with a concentration of 12.5% NaCl) at the center of the plate, while the halophilic bacteria inoculated around the fungus. The antagonistic properties of the bacteria were determined by measuring the zones of inhibited fungal growth, while NA plates inoculated solely with the pathogens served as an effective negative control. All cultures were incubated at a consistent temperature of 25 °C for 7 days<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<p id="Par21">Also an impactful evaluation of the antibacterial activity of the isolates against <em>Ralstonia solanacearum</em> NRRL B-3211, a notorious cause of bacterial wilt affecting a broad spectrum of host plants<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, was conducted using the well-established disc diffusion method<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p></section><section id="Sec15"><h4 class="pmc_sec_title">Determination of bioactive chemical constituents produced by QSLA1 isolate using GC–MS analysis</h4>
<p id="Par22">The chemical compounds produced by the selected isolate QSLA1 were thoroughly evaluated and analyzed using Gas Chromatography-Mass Spectrometry (GC–MS). Extraction of bioactive compounds was executed following the liquid–liquid extraction protocol<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Briefly, an aliquot of 5 mL sample was placed in 15-mL centrifuge tubes, and a mixture of extraction solvent (0.29 mL chloroform in 0.28 mL acetone) was introduced rapidly into the sample. The extractant was dispersed into the sample solution via vortexing for 0.5 min resulting in the formation of a cloudy solution (water/acetone/chloroform). The analytes were extracted into the fine droplets of chloroform<strong>.</strong> Centrifugation was applied to separate the two immiscible layers at 4400 rpm for 3 min. The segmented organic layer (chloroform) was thereafter quantitatively transferred into a 2-mL vial and evaporated to dryness at 60 °C. Thereafter, the residue was reconstituted in 1 mL mobile phase and vortexed prior to injection into the GC MS.</p></section><section id="Sec16"><h4 class="pmc_sec_title">Gas chromatography–mass spectrometry (GC–MS) analysis</h4>
<p id="Par23">The chemical composition of samples performed using GC-TSQ mass spectrometer (Thermo Scientific, Austin, TX, USA) with a direct capillary column TG–5MS (30 m × 0.25 mm × 0.25 µm film thickness). The column oven temperature was initially held at 60 °C and then increased by 5 °C/min to 250 °C withhold 2 min then increased to 300 with 30 °C/min. The injector temperature was kept at 270 °C. Helium was used as a carrier gas at a constant flow rate of 1 ml/min. The solvent delay was 4 min and diluted samples of 1 µl were injected automatically using Autosampler AS3000 coupled with GC in the split mode. EI mass spectra were collected at 70 eV ionization voltages over the range of m/z 50–650 in full scan mode. The ion source and transfer line were set at 200 °C and 280 °C respectively. The components were identified by comparison of their mass spectra with those of WILEY 09 and NIST14 mass spectral database<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par24">The comprehensive chemical composition of metabolites extracted was detected using a GC–TSQ mass spectrometer (Thermo Scientific, Austin, TX, USA) with a direct capillary column TG–5MS (30 m × 0.25 mm × 0.25 µm film thickness), as detailed by<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. The percentage composition of each compound was diligently calculated as the ratio of the peak area to the total chromatographic area, with GC–MS peaks confidently assigned through comparison with established data, achieving similarity percentages from the Wiley 275 libraries<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p></section></section><section id="Sec17"><h3 class="pmc_sec_title">Controlling tomato <em>Fusarium</em> wilt pathogen using QSLA1 (<em>Halomonas</em> sp.) under greenhouse conditions</h3>
<section id="Sec18"><h4 class="pmc_sec_title">Preparation of fungal inoculum</h4>
<p id="Par25">Tomato seedlings (halophyte strain 023) that were 35 days old were utilized for the study. The fungal inoculum was prepared as follows: The fungal strain <em>Fusarium oxysporium subsp. lycopersici</em> was cultivated on sterilized barley grain medium, consisting of 100 g of washed, dried barley grains mixed with 65 ml of tap water per bottle. Inoculation of the barley was performed with uniform 5 mm agar discs of <em>Fusarium oxysporium</em>, which had been grown on a PDA medium with 3.5% salt for a period of 4 days. The bottles were incubated at 28 °C for two weeks, allowing ample growth of the fungal isolates<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p></section><section id="Sec19"><h4 class="pmc_sec_title">Pots preparation and inoculation</h4>
<p id="Par26">Fertile soil was sourced from the surface layer of the experimental farm at the Faculty of Agriculture, Menoufia University, and was sterilized using a 5% formalin solution. The preparation of pots and inoculation was executed as follows: The formalin-disinfected clay pots (30 cm in diameter) were filled with a mixture of sterilized soil and compost (at a 1:3 ratio) to a weight of 3 kg per pot. The potted soil was artificially infested with the inoculum at 3% (w/w) and was watered twice a week with saline water (1.5%) for seven days prior to planting. Control pots containing soil and compost (1:3) without inoculation were also prepared<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par27">Tomato seedlings of cultivar 023, which had been grown for 35 days in seed boxes filled with a peat-moss vermiculite mixture (1:1 w/w), were uprooted and transplanted into the pots at a rate of 2 seedlings per pot. Each treatment included three replicates. Immediately after transplanting, the pots were irrigated and subsequently, each seedling received 20 ml of saline water (1.5%) daily.</p>
<p id="Par28">The treatments implemented were as follows: Treatment S = Tomato pots not treated with QSLA1isolate, and not infected with <em>Fusarium oxysporium subsp. lycopersici</em>; Treatment HS = Tomato seedlings treated with QSLA1 isolate (10<sup>–2</sup>) and not infected with <em>Fusarium oxysporium subsp. lycopersici</em>; Treatment FS = Tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium subsp. Lycopersici</em> and not treated with QSLA1isolate; Treatment HFS = Tomato seedlings infected with 3% (w/w) of Fusarium oxysporium subsp. lycopersici and inoculated with QSLA1 (10<sup>–2</sup>). All treatments were irrigated with salt water (1.5%).</p>
<p id="Par29">The wilt disease incidence percentage (WDI%) and severity percentage (WDS%) were determined and calculated after 28 days of transplanting, based on a 0–4 scale<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, where: 0 = No infection, 1 = Slight infection (approximately 25% of the total), characterized by one or two yellowed leaves, 2 = Moderate infection (two or three yellowed leaves, 50% wilted), 3 = Extensive infection (all leaves yellowed, 75% wilted, growth inhibited), and 4 = Complete infection (the entire plant yellowed, 100% wilted, leading to plant death). Disease severity was calculated accordingly.</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/375b045bc71d/d33e583.gif" loading="lazy" id="d33e583" alt="graphic file with name d33e583.gif"></td></tr></table>
<p>where (a) represents the number of infected plants in each category, (b) is the numerical value of that category, (N) is the total number of examined plants, and (K) signifies the highest degree of infection category. We recorded disease incidence for each individual treatment using the appropriate formula.</p>
<p id="Par30">To evaluate the effectiveness of QSLA1 in controlling <em>Fusarium</em> wilt under salinity stress, root length, shoot height, and shoot infected length were measured with precision. The shoot infected length was defined as the length of the browned area at the bottom of the shoot, conclusively indicating the presence of the pathogen<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p></section></section><section id="Sec20"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par31">The data underwent rigorous statistical analysis using analysis of variance (ANOVA). Differences between means were firmly evaluated using a high-range statistical domain with Tukey’s post hoc analysis, enabling us to distinguish between homogeneous and heterogeneous groups across various variables. We established significance for multiple comparisons of means at a probability level of (<em>p</em> = 0.05). The results are presented as average means ± standard deviations (SD) from triplicate measurements, ensuring robust and reliable findings.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec22"><h3 class="pmc_sec_title">Isolation and purification of halophilic bacteria from solar saltern water</h3>
<section id="Sec23"><h4 class="pmc_sec_title">Bacterial isolates</h4>
<p id="Par32">Bacterial isolates, QSLA1 and QSLA3, were successfully obtained from the second pond with an 8.2% salt concentration, while the QSLA2 strain was isolated from the third pond, exhibiting a salinity of 17.2%. These isolates were effectively cultured on nutrient agar (NA) culture medium prepared with saline water sourced directly from the pond samples, rather than distilled water.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Cultural, morphological, and biochemical characterization of the isolates</h4>
<p id="Par33">Cultural, morphological, and biochemical characterizations, which play a critical role in the partial identification of microorganisms<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> were comprehensively conducted. It was confirmed that all three isolates were gram-negative, motile, catalase-positive, and rod-shaped. Notably, they were not spore forming bacteria, as documented in Tables <a href="#Tab2" class="usa-link">2</a> and <a href="#Tab3" class="usa-link">3</a>. These findings reinforce our understanding of the isolates’ properties and their potential applications.</p>
<section class="tw xbox font-sm" id="Tab2"><h5 class="obj_head">Table 2.</h5>
<div class="caption p"><p>Colony morphology of isolates on nutrient agar plates (12.5% NaCl).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1">Tested Characters</th>
<th align="left" rowspan="2" colspan="1">Colony shape</th>
<th align="left" rowspan="2" colspan="1">Colony edge</th>
<th align="left" rowspan="2" colspan="1">Opacity</th>
<th align="left" rowspan="2" colspan="1">Elevation</th>
<th align="left" rowspan="2" colspan="1">Size</th>
<th align="left" rowspan="2" colspan="1">Pigmentation</th>
<th align="left" rowspan="2" colspan="1">Appearance</th>
</tr>
<tr><th align="left" colspan="1" rowspan="1">Isolates</th></tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1</td>
<td align="left" colspan="1" rowspan="1">Round</td>
<td align="left" colspan="1" rowspan="1">Lobate</td>
<td align="left" colspan="1" rowspan="1">Translucent</td>
<td align="left" colspan="1" rowspan="1">Flat</td>
<td align="left" colspan="1" rowspan="1">Pin point</td>
<td align="left" colspan="1" rowspan="1">Creamy</td>
<td align="left" colspan="1" rowspan="1">Shinny</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA2</td>
<td align="left" colspan="1" rowspan="1">Round</td>
<td align="left" colspan="1" rowspan="1">Lobate</td>
<td align="left" colspan="1" rowspan="1">Opaque</td>
<td align="left" colspan="1" rowspan="1">Raised</td>
<td align="left" colspan="1" rowspan="1">Small</td>
<td align="left" colspan="1" rowspan="1">White</td>
<td align="left" colspan="1" rowspan="1">Shinny</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA3</td>
<td align="left" colspan="1" rowspan="1">Irregular</td>
<td align="left" colspan="1" rowspan="1">Ragged</td>
<td align="left" colspan="1" rowspan="1">Translucent</td>
<td align="left" colspan="1" rowspan="1">Flat</td>
<td align="left" colspan="1" rowspan="1">Small</td>
<td align="left" colspan="1" rowspan="1">Creamy</td>
<td align="left" colspan="1" rowspan="1">Shinny</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab3"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p>Morphological and biochemical characterization of the isolates.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1">Tested Characters</th>
<th align="left" rowspan="2" colspan="1">Shape</th>
<th align="left" rowspan="2" colspan="1">Concentration of the isolation pond(% NaCl)</th>
<th align="left" rowspan="2" colspan="1">Gram stain</th>
<th align="left" rowspan="2" colspan="1">Catalase test</th>
<th align="left" rowspan="2" colspan="1">Motility</th>
<th align="left" rowspan="2" colspan="1">Spore forming</th>
</tr>
<tr><th align="left" colspan="1" rowspan="1">Isolates</th></tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1</td>
<td align="left" colspan="1" rowspan="1">Rod shape</td>
<td align="center" colspan="1" rowspan="1">8.2</td>
<td align="left" colspan="1" rowspan="1">G<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> − </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA2</td>
<td align="left" colspan="1" rowspan="1">Rod shape</td>
<td align="center" colspan="1" rowspan="1">17.2</td>
<td align="left" colspan="1" rowspan="1">G<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> − </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA3</td>
<td align="left" colspan="1" rowspan="1">Rod shape</td>
<td align="center" colspan="1" rowspan="1">8.2</td>
<td align="left" colspan="1" rowspan="1">G<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> − </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec25"><h4 class="pmc_sec_title">Salinity tolerance of the isolates</h4>
<p id="Par34">The salinity tolerance tests clearly demonstrated that none of the isolates could grow without salt (0% NaCl). In contrast, all isolates thrived on NA culture medium containing 7% and 12.5% NaCl. Notably, QSLA1 and QSLA2 exhibited remarkable growth on NA culture medium with 20% NaCl, with QSLA2 even thriving in medium with 22% NaCl (see Table <a href="#Tab4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h5 class="obj_head">Table 4.</h5>
<div class="caption p"><p>Salinity tolerance assay of the isolates (NaCl %).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1">Isolates</th>
<th align="left" colspan="5" rowspan="1">NaCl concentrations (%)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">Zero</th>
<th align="left" colspan="1" rowspan="1">7</th>
<th align="left" colspan="1" rowspan="1">12.5</th>
<th align="left" colspan="1" rowspan="1">20</th>
<th align="left" colspan="1" rowspan="1">22</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1</td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> − </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA2</td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA3</td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> − </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par35">Moderate halophiles typically prosper in environments with 0.5 to 2.5 M NaCl (approximately 3% to 15% NaCl), while extreme halophiles flourish in conditions with 2.5 to 5.2 M (saturated) NaCl (15% to 30% NaCl) and cannot survive without salt. Thus, the inability of these isolates to grow in the absence of NaCl decisively confirms their classification as halophilic bacteria. Among them, QSLA1 and QSLA3 are confidently identified as moderately halophilic, while QSLA2 is undoubtedly recognized as an extreme halophilic bacterium<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p>
<p id="Par36">Halophilic bacterial strains were successfully isolated from three distinct basins of Lake Meyghan, each characterized by varying salinity levels: the green brine with approximately 50 g/L salinity, the red brine with around 180 g/L salinity, and the white brine with about 300 g/L salinity<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. In a separate study, moderately halophilic bacteria were isolated from hypersaline environments, specifically solar salterns and salt lakes in Alexandria, Egypt. Notably, 85% of their isolates were Gram-negative, while the remaining 15% were Gram-positive<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Forty-six halo-bacterial isolates were identified from soil and water samples at Sambhar Lake. Remarkably, all isolates demonstrated tolerance to 10% NaCl, with forty-four cultures exhibiting resilience to 15% NaCl, and three out of ten selected cultures tolerating as much as 25% salt<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Molecular identification of the isolates via 16S rDNA</h4>
<p id="Par37">In addition to performing cultural, morphological, and biochemical characterization, the 16S rDNA genes of the isolates were sequenced and analyzed for molecular identification. The resultant sequences were meticulously compared against those in the NCBI BLAST database. The analysis revealed that isolate QSLA1 is closely related to the <em>Halomonas</em> sp. strain RS-17, with a strong similarity of 91.26%. Isolate QSLA2 was identified as <em>Halomonas</em> sp. strain LR2-3, showing 96.6% similarity, while QSLA3 was confirmed as <em>Halomonas</em> sp. GQ30, exhibiting an impressive 97.33% similarity.</p></section><section id="Sec27"><h4 class="pmc_sec_title">Evolutionary relationships of taxa</h4>
<p id="Par38">Using the Clustal W program, the sequences were aligned and the MEGA X program was employed to construct a phylogenetic tree. Results clearly demonstrate that these isolates belong to the class Gammaproteobacteria, as depicted in Fig. <a href="#Fig1" class="usa-link">1</a>. This robust phylogenetic affiliation underscores the significance of our research in understanding halophilic microbial diversity.</p>
<figure class="fig xbox font-sm" id="Fig1"><h5 class="obj_head">Fig. 1.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/6728cb53e6f7/41598_2025_12974_Fig1_HTML.jpg" loading="lazy" id="MO1" height="545" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree constructed using neighbor-joining analysis based on 16S rDNA sequences from the isolates and closely related sequences deposited in GenBank.</p></figcaption></figure><p id="Par39">The three isolates have been deposited in the NCBI GenBank under the accession numbers listed in Table <a href="#Tab5" class="usa-link">5</a>.</p>
<section class="tw xbox font-sm" id="Tab5"><h5 class="obj_head">Table 5.</h5>
<div class="caption p"><p>Accession numbers of the sequences of submitted isolates deposited in NCBI GenBank.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolates</th>
<th align="left" colspan="1" rowspan="1">Organisms</th>
<th align="left" colspan="1" rowspan="1">Strains in genebank</th>
<th align="left" colspan="1" rowspan="1">Accession numbers</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1</td>
<td align="left" colspan="1" rowspan="1"><em>Halomonas</em></td>
<td align="left" colspan="1" rowspan="1">
<em>Halomonas</em> sp. strain QSLA1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442496" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442496</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA2</td>
<td align="left" colspan="1" rowspan="1"><em>Halomonas</em></td>
<td align="left" colspan="1" rowspan="1">
<em>Halomonas</em> sp. strain QSLA2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442497" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442497</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA3</td>
<td align="left" colspan="1" rowspan="1"><em>Halomonas</em></td>
<td align="left" colspan="1" rowspan="1">
<em>Halomonas</em> sp. strain QSLA3</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442498" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442498</a></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par40">361 halo bacterial strains were isolated from three distinct basins of Lake Meyghan, each exhibiting varying salinities. These strains are classified into several key classes, including Gammaproteobacteria, Alphaproteobacteria, Bacteroidetes, and Firmicutes, representing a diverse range of genera<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<p id="Par41">74 halophilic bacteria were identified from the saline ecosystems of Algeria’s Sebkha and Chott lakes, which are located in arid and semi-arid eco climate zones. Notably, 16 of these isolates were closely related to the genus <em>Halomonas</em><sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Moderate halophilic strains were isolated from solar salterns, and their phylogenetic analysis confirmed affiliations with five notable genera: <em>Bacillus, Halobacillus, Planococcus, Salinicoccus,</em> and <em>Halomonas</em><sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par42">Taxonomic analyses of moderately halophilic bacteria isolated from hypersaline habitats, specifically solar salterns and salt lakes in Alexandria, Egypt, revealed that a remarkable 85% of the isolates belonged to the γ-Proteobacteria. Five genera were precisely identified: <em>Pseudoalteromonas, Flavobacterium, Chromohalobacter, Halomonas,</em> and <em>Salegentibacter</em><sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par43">Romania’s salt lakes, which have salinities exceeding 70 g/L, are home to bacteria from three predominant phyla: Firmicutes, Proteobacteria, and Actinobacteria, with <em>Halomonas</em> standing out as the most representative genus within the Proteobacteria phylum<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>.</p>
<p id="Par44">Importantly, in vitro assessments of plant growth-promoting traits underscore the significant potential of halophilic and halotolerant bacteria to enhance plant growth. This approach is essential for the development of effective bio-inoculants for saline soils<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. All isolates demonstrated robust plant growth-promoting activities, such as phosphorus solubilization, production of indole-3-acetic acid (IAA), and nitrogen fixation (Table <a href="#Tab6" class="usa-link">6</a>).</p>
<section class="tw xbox font-sm" id="Tab6"><h5 class="obj_head">Table 6.</h5>
<div class="caption p"><p>Plant growth promoting ability results of the isolates.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolates</th>
<th align="left" colspan="1" rowspan="1">Nitrogen fixation (Ashby medium)</th>
<th align="left" colspan="1" rowspan="1">IAA production</th>
<th align="left" colspan="1" rowspan="1">Phosphate solubilization</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1</td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> − </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA2</td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA3</td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> − </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec28"><h4 class="pmc_sec_title">Nitrogen fixation and phosphate solubilization assay</h4>
<p id="Par45">The results clearly demonstrate that the QSLA1 and QSLA3 isolates do not have the capability to fix nitrogen, as they failed to grow on Ashby N-free culture medium. Furthermore, they do not solubilize phosphate, as indicated by the lack of clear zones around their colonies on Pikovskaya (PVK) agar plates (Fig. <a href="#Fig2" class="usa-link">2</a>). In stark contrast, the QSLA2 isolate has proven to be effective in both solubilizing phosphate (Fig. <a href="#Fig2" class="usa-link">2</a>) and fixing nitrogen.</p>
<figure class="fig xbox font-sm" id="Fig2"><h5 class="obj_head">Fig. 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/376a2a8a1842/41598_2025_12974_Fig2_HTML.jpg" loading="lazy" id="MO2" height="391" width="679" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PVK agar plates showing distinctly clear zones around the positive bacterial isolates with 12.5% NaCl.</p></figcaption></figure></section></section><section id="Sec29"><h3 class="pmc_sec_title">Indole acetic acid (IAA) production assay</h3>
<p id="Par46">All isolates tested demonstrated the ability to produce Indole acetic acid (IAA) in NA culture medium enriched with L-tryptophan (see Table <a href="#Tab7" class="usa-link">7</a> and Fig. <a href="#Fig3" class="usa-link">3</a>). Notably, the QSLA1 isolate achieved the highest IAA production at 15.45 µg/ml, followed closely by QSLA3 at 14.30 µg/ml (refer to Table <a href="#Tab7" class="usa-link">7</a>). The differences in IAA production between the QSLA1 isolate and the other two isolates were statistically significant, while the QSLA2 and QSLA3 isolates showed no significant differences in their production levels (see Table <a href="#Tab7" class="usa-link">7</a> and Fig. <a href="#Fig4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>IAA concentration produced by the isolates (means + standard deviation).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolates</th>
<th align="left" colspan="1" rowspan="1">IAA concentration (µg/ml) ± S.D</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1</td>
<td align="center" colspan="1" rowspan="1">0.15 ± 0.09<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA2</td>
<td align="center" colspan="1" rowspan="1">15.45 ± 0.91<sup>b</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA3</td>
<td align="center" colspan="1" rowspan="1">14.30 ± 0.97<sup>b</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p57">
<p>Values are means ± standard deviation of three replicates.</p>
<p>b = higher value, a = lower value.</p>
</div></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/2c586f30b0c5/41598_2025_12974_Fig4_HTML.jpg" loading="lazy" id="MO4" height="466" width="722" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Concentrations of IAA produced in NB medium supplemented with 1% l-tryptophan.</p></figcaption></figure><p id="Par47">120 salt-tolerant bacterial strains were successfully isolated from various locations along Portugal’s coastline. Among these, <em>Halomonas titanicae</em> was particularly noteworthy for its capacity to fix nitrogen, solubilize phosphate, and produce IAA at a concentration 12.22 µg/ml<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Furthermore, a salt-tolerant strain, <em>Halomonas</em> sp. that exhibited impressive plant growth-promoting traits including phosphate solubilization and nitrogen fixation in high-salinity habitats in India was isolated<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. The exceptional growth-promoting potential of the moderate halophile <em>Halomonas</em> sp. MAN5 was underscored, and it was found to have the potential to produce 95.3 µg/ml of IAA and solubilize 53 parts per million (ppm) of phosphates in the presence of 15% NaCl<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>.</p>
<p id="Par48">The variation in IAA concentrations among the different bacterial isolates can be attributed to their distinct abilities to utilize tryptophan or to the diverse pathways involved in IAA biosynthesis, including Indole-3-Pyruvic Acid, Tryptamine, and Indole-3-Acetonitrile<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>.</p></section><section id="Sec30"><h3 class="pmc_sec_title">In vitro antagonistic activity of the isolates against plant pathogens</h3>
<p id="Par49">Isolate QSLA1 exhibited notable antifungal activity against <em>Fusarium oxysporium f.sp. Lycopersici</em> and <em>Alternaria solani</em>, effectively limiting mycelial growth in a co-cultivation plate assay. Additionally, it demonstrated significant antibacterial activity against <em>Ralstonia solanacearum</em> using the disc diffusion method at a 12.5% salt concentration. In contrast, isolates QSLA2 and QSLA3 displayed antibacterial activity exclusively against <em>Ralstonia solanacearum</em> (see Table <a href="#Tab8" class="usa-link">8</a> and Fig. <a href="#Fig5" class="usa-link">5</a>). Consequently, isolate QSLA1 has been selected for further studies due to its impressive antagonistic activity against multiple pathogens.</p>
<section class="tw xbox font-sm" id="Tab8"><h4 class="obj_head">Table 8.</h4>
<div class="caption p"><p>Results of antagonistic activity of the isolates against plant pathogens in vitro.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolate</th>
<th align="left" colspan="1" rowspan="1"><em>Fusarium oxysporium</em></th>
<th align="left" colspan="1" rowspan="1"><em>Alternaria solani</em></th>
<th align="left" colspan="1" rowspan="1"><em>Ralstonia solanacearum</em></th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1</td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
<td align="left" colspan="1" rowspan="1"> + </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA2</td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> + </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA3</td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> − </td>
<td align="left" colspan="1" rowspan="1"> + </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/2b03f939ddeb/41598_2025_12974_Fig5_HTML.jpg" loading="lazy" id="MO5" height="419" width="695" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>the antagonistic effects of the isolates against <em>Fusarium oxysporium</em> (<strong>A</strong>), <em>Alternaria solani</em> (<strong>B</strong>), and <em>Ralstonia solanacearum</em> (<strong>C</strong>) were compellingly demonstrated at a 12.5% NaCl concentration.</p></figcaption></figure></section><section id="Sec31"><h3 class="pmc_sec_title">Determination of bioactive chemical constituents produced by QSLA1 isolate using GC–MS analysis</h3>
<p id="Par50">As its antagonistic activity against all used phyto-pathogens, the bioactive chemical constituents produced by the QSLA1 isolate were successfully identified through comprehensive GC–MS analysis. Insights from unpublished bioinformatics data prompted a decisive investigation into the bioactive compounds released by halophilic isolates. As a result, the extract from QSLA1 underwent thorough GC–MS analysis (see Fig. <a href="#Fig6" class="usa-link">6</a>), revealing its metabolic profile and identifying the specific chemical compounds present (refer to Table <a href="#Tab9" class="usa-link">9</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/07678bd2f01b/41598_2025_12974_Fig6_HTML.jpg" loading="lazy" id="MO6" height="376" width="776" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The GC–MS chromatogram from the supernatant of the QSLA1 isolate reveals a rich array of bioactive compounds.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab9"><h4 class="obj_head">Table 9.</h4>
<div class="caption p"><p>The chemical compounds detected in the culture supernatant of QSLA1 after centrifugation, as revealed by the GC–MS analysis, clearly underscore the remarkable bioactive potential of this isolate.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">No</th>
<th align="left" colspan="1" rowspan="1">Compound name</th>
<th align="left" colspan="1" rowspan="1">Rt (Min)</th>
<th align="left" colspan="1" rowspan="1">Area %</th>
<th align="left" colspan="1" rowspan="1">Molecular formula</th>
<th align="left" colspan="1" rowspan="1">Molecular weight</th>
<th align="left" colspan="1" rowspan="1">Function (PubMed)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">Desulphosinigrin</td>
<td align="center" colspan="1" rowspan="1">13.70</td>
<td align="left" colspan="1" rowspan="1">0.23</td>
<td align="left" colspan="1" rowspan="1">C<sub>10</sub>H<sub>17</sub>NO<sub>6</sub>S</td>
<td align="left" colspan="1" rowspan="1">279</td>
<td align="left" colspan="1" rowspan="1">Antibacterial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">Decanoic acid, ethyl ester</td>
<td align="center" colspan="1" rowspan="1">15.29</td>
<td align="left" colspan="1" rowspan="1">0.04</td>
<td align="left" colspan="1" rowspan="1">C<sub>12</sub>H<sub>24</sub>O2</td>
<td align="left" colspan="1" rowspan="1">200</td>
<td align="left" colspan="1" rowspan="1">Antibcterial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">Undeca-2,4,6,8,10-pentaenal, 11-(2-Furyl)-, oxime</td>
<td align="center" colspan="1" rowspan="1">16.35</td>
<td align="left" colspan="1" rowspan="1">0.10</td>
<td align="left" colspan="1" rowspan="1">C<sub>12</sub>H<sub>26</sub>O<sub>5</sub>S</td>
<td align="left" colspan="1" rowspan="1">282</td>
<td align="left" colspan="1" rowspan="1">Antibacterial/Antifungal</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">Hexadecanoic acid, methyl ester</td>
<td align="center" colspan="1" rowspan="1">23.11</td>
<td align="left" colspan="1" rowspan="1">33.22</td>
<td align="left" colspan="1" rowspan="1">C<sub>17</sub>H<sub>34</sub>O<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">270</td>
<td align="left" colspan="1" rowspan="1">Antimicrobial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">Strychane, 1-acetyl-20à-hydroxy-16-methylene</td>
<td align="center" colspan="1" rowspan="1">23.97</td>
<td align="left" colspan="1" rowspan="1">0.11</td>
<td align="left" colspan="1" rowspan="1">C<sub>21</sub>H<sub>26</sub>N<sub>2</sub>O<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">338</td>
<td align="left" colspan="1" rowspan="1">Antimicrobial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">Curan-17-oic acid, 19,20-dihydroxy-, methyl ester, (19S)</td>
<td align="center" colspan="1" rowspan="1">24.94</td>
<td align="left" colspan="1" rowspan="1">1.39</td>
<td align="left" colspan="1" rowspan="1">C<sub>20</sub>H<sub>26</sub>N<sub>2</sub>O<sub>4</sub>
</td>
<td align="left" colspan="1" rowspan="1">358</td>
<td align="left" colspan="1" rowspan="1">Antimicrobial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">Cyclopropaneoctanoic acid, 2-[[2-[(2-ethyl cyclopropyl) methyl] cyclopropyl]methyl]-, methyl ester</td>
<td align="center" colspan="1" rowspan="1">26.19</td>
<td align="left" colspan="1" rowspan="1">0.071</td>
<td align="left" colspan="1" rowspan="1">C<sub>22</sub>H<sub>38</sub>O<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">334</td>
<td align="left" colspan="1" rowspan="1">Antimicrobial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">Palmitic acid</td>
<td align="center" colspan="1" rowspan="1">30.16</td>
<td align="left" colspan="1" rowspan="1">0.59</td>
<td align="left" colspan="1" rowspan="1">C<sub>16</sub>H<sub>32</sub>O<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">256</td>
<td align="left" colspan="1" rowspan="1">Antibacterial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">cis-Vaccenic acid</td>
<td align="center" colspan="1" rowspan="1">30.63</td>
<td align="left" colspan="1" rowspan="1">0.10</td>
<td align="left" colspan="1" rowspan="1">C<sub>18</sub>H<sub>34</sub>O<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">282</td>
<td align="left" colspan="1" rowspan="1">Antimicrobial</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">Ethanimidothioic Acid, 2-(Dimethylamino)-N-[[(Methylamino)Carbonyl] Oxy]-2-Oxo-, Methyl Ester</td>
<td align="center" colspan="1" rowspan="1">34.26</td>
<td align="left" colspan="1" rowspan="1">0.27</td>
<td align="left" colspan="1" rowspan="1">C<sub>7</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>S</td>
<td align="left" colspan="1" rowspan="1">219</td>
<td align="left" colspan="1" rowspan="1">Insecticide</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par51">Notably, the main chemical constituents identified include Hexadecanoic Acid, Methyl Ester (33.22%), 9-Octadecenoic Acid, (2-Phenyl-1,3-Dioxolan-4-Yl)Methyl Ester, Cis (6%), and d-Lyxo-d-manno-nononic-1,4-lactone (2.94%). Other significant compounds are Curan-17-oic acid, 19,20-dihydroxy-, methyl ester, (19S) (1.39%), Cyclopropanedodecanoic acid, 2-octyl-, methyl ester (1.39%), and several others, including 13,16-Octadecadiynoic Acid, Methyl Ester (1.27%) and Palmitic Acid (0.59%). The majority of these constituents are fatty acids and fatty acid derivatives, recognized for their biosurfactant properties and antimicrobial activity <sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Additionally, compounds such as Reynosin (lactone) (0.25%) and Leukotriene F4 (0.19%) showcase notable bioactive potential, contributing to antifungal, antitumor, antibacterial, and antioxidant properties<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The dominant presence of fatty acids in the extract underscores their therapeutic significance. Fatty acids and their derivatives were emphasized as versatile agents against various health issues, including cancer and bacterial infections<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Furthermore, acids like 3-hydroxydecanoic acid and decanoic acid, sourced from <em>Lactobacillus plantarum</em>, exhibit strong antibacterial properties<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Fatty acids have also been shown to disrupt bacterial growth by altering membrane permeability and inhibiting fatty acid synthesis<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Moreover, the compounds (antibiotics and biosurfactants) derived from <em>Halomonas meridiana</em> BK-AB4 playing a vital role in combating pathogens<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>.</p></section><section id="Sec32"><h3 class="pmc_sec_title">Controlling tomato fusarium wilt pathogen with QSLA1 (<em>Halomonas</em> sp.)</h3>
<p id="Par52">A pot experiment has successfully demonstrated the efficacy of QSLA1 against <em>Fusarium oxysporium f.sp. lycopersici</em>, showcasing its remarkable ability to control <em>Fusarium</em> wilt disease in tomato seedlings, particularly under saline conditions (1.5% salt water) (Fig. <a href="#Fig7" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/41fe3f9e1a98/41598_2025_12974_Fig7_HTML.jpg" loading="lazy" id="MO7" height="501" width="690" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A greenhouse experiment clearly demonstrated significant differences among treatments after 28 days. Treatment S = tomato seedlings irrigated with salt water (1.5%), not treated with QSLA1and not infected with <em>Fusarium</em>. Treatment HS = tomato seedlings treated with QSLA1 isolate (10<sup>–2</sup>) and not infected with <em>Fusarium</em>. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>). All treatments irrigated with salt water (1.5%).</p></figcaption></figure><p id="Par53">Throughout the 28-day experiment, notable differences in symptoms emerged between the seedlings inoculated solely with <em>Fusarium</em> (treatment FS) and those treated with both QSLA1 isolate and <em>Fusarium</em> (treatment HFS). After just 7 days post-transplanting, yellowing of the leaves was observed, often concentrated on one side of the plant, resulting from impaired lateral water translocation<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. This symptom was evident in both treatment F and treatment HF. By the 21-day mark, seedlings treated with <em>Fusarium</em> alone (treatment FS) displayed severe symptoms, including wilting, browning of the above-ground portion, and drooping of lower leaves (Fig. <a href="#Fig8" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/b3e499990835/41598_2025_12974_Fig8_HTML.jpg" loading="lazy" id="MO8" height="475" width="735" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clearly demonstrates the progression of <em>Fusarium</em> wilt symptoms throughout the experiment. Treatment S = tomato seedlings irrigated with salt water (1.5%), not treated with QSLA1and not infected with <em>Fusarium</em>. Treatment HS = tomato seedlings treated with QSLA1 isolate (10<sup>–2</sup>) and not infected with <em>Fusarium</em>. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>). All treatments irrigated with salt water (1.5%).</p></figcaption></figure><p id="Par54">The results for the other treatments reveal that seedlings inoculated with QSLA1 alone (Treatment HS) remained remarkably healthy, displaying vibrant green leaves. In stark contrast, the untreated control group (Treatment S) showed no symptoms in any of the plants, as illustrated in Fig. <a href="#Fig9" class="usa-link">9</a>.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/f77458522ccd/41598_2025_12974_Fig9_HTML.jpg" loading="lazy" id="MO9" height="479" width="646" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Impact of inoculation with isolate QSLA1 on tomato seedlings after 28 days. Treatment S = tomato seedlings irrigated with salt water (1.5%), not treated with QSLA1and not infected with <em>Fusarium</em>. Treatment HS = tomato seedlings treated with QSLA1 isolate (10<sup>–2</sup>) and not infected with <em>Fusarium</em>. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>). All treatments irrigated with salt water (1.5%).</p></figcaption></figure><p id="Par55">Our findings clearly demonstrate that after 28 days of transplanting under greenhouse conditions, QSLA1 is highly effective in reducing the incidence of <em>Fusarium</em> wilt in tomatoes by an impressive 33.3% in seedlings treated with both QSLA1 isolate and <em>Fusarium</em> (treatment HF). In contrast, seedlings treated with <em>Fusarium</em> alone (treatment F) exhibited a 100% disease incidence (refer to Table <a href="#Tab10" class="usa-link">10</a> and Fig. <a href="#Fig10" class="usa-link">10</a>). Moreover, the severity of <em>Fusarium</em> disease in seedlings treated with the QSLA1 isolate combined with <em>Fusarium</em> (treatment HF) was significantly reduced by 12.5% compared to those treated solely with <em>Fusarium</em> (treatment F), which showed an 83.3% disease severity, as confirmed by the established equation (see Table <a href="#Tab10" class="usa-link">10</a> and Fig. <a href="#Fig11" class="usa-link">11</a>).</p>
<section class="tw xbox font-sm" id="Tab10"><h4 class="obj_head">Table 10.</h4>
<div class="caption p"><p>The effects of inoculation with isolate QSLA1 on tomato seedlings under various treatment conditions.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Treatments</th>
<th align="left" colspan="1" rowspan="1">Root length (cm)</th>
<th align="left" colspan="1" rowspan="1">Shoot height (cm)</th>
<th align="left" colspan="1" rowspan="1">Shoot infection length (cm)</th>
<th align="left" colspan="1" rowspan="1">Disease incidence (%)</th>
<th align="left" colspan="1" rowspan="1">Disease severity (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Only fresh water</td>
<td align="center" colspan="1" rowspan="1">5.50 ± 0.50<sup>a</sup>
</td>
<td align="center" colspan="1" rowspan="1">15.97 ± 0.25<sup>d</sup>
</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Only salt water (1.5%) (S)</td>
<td align="center" colspan="1" rowspan="1">6.43 ± 0.25<sup>b</sup>
</td>
<td align="center" colspan="1" rowspan="1">12.80 ± 0.20<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1 only + Salt water (1.5%) (HS)</td>
<td align="center" colspan="1" rowspan="1">8.73 ± 0.46<sup>d</sup>
</td>
<td align="center" colspan="1" rowspan="1">15.80 ± 0.20<sup>d</sup>
</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Salt water (1.5%) + <em>Fusarium</em> only (FS)</td>
<td align="center" colspan="1" rowspan="1">5.07 ± 0.12<sup>a</sup>
</td>
<td align="center" colspan="1" rowspan="1">11.17 ± 0.15<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.63 ± 0.12<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">100<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">83.3<sup>b</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QSLA1 + Salt water (1.5%) + <em>Fusarium</em> (HFS)</td>
<td align="center" colspan="1" rowspan="1">7.50 ± 0.36<sup>c</sup>
</td>
<td align="center" colspan="1" rowspan="1">14.27 ± 0.25<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.60 ± 0.36<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">33.3<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">12.5<sup>a</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p78">
<p>Values are means ± standard deviation of three replicates.</p>
<p>d = higher value, a = lower value, ND = not detected.</p>
</div></div></section><figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/4b53897222b7/41598_2025_12974_Fig10_HTML.jpg" loading="lazy" id="MO10" height="469" width="723" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of inoculation with isolate QSLA1 on disease incidence of tomato seedlings infected with <em>Fusarium oxysporium</em> sub sp. <em>lycopersci</em> after 28 days.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/8e9da2b7fb43/41598_2025_12974_Fig11_HTML.jpg" loading="lazy" id="MO11" height="486" width="723" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of inoculation with isolate QSLA1 on disease severity percentage on tomato seedlings infected with <em>Fusarium oxysporium</em> sub sp. <em>lycopersici</em> after 28 days. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>). All treatments irrigated with salt water (1.5%).</p></figcaption></figure><p id="Par56">Plants treated with the QSLA1 isolate in conjunction with <em>Fusarium</em> (treatment HF) and those treated with <em>Fusarium</em> alone (treatment F) exhibited a clear browning zone at the base of the seedling shoots, resulting from pathogen colonization of the tissues. This browning was quantitatively assessed as shoot infection length (see Fig. <a href="#Fig12" class="usa-link">12</a>). The results clearly demonstrated a significant difference in shoot infection length, with tomato seedlings treated with the QSLA1 isolate and <em>Fusarium</em> (treatment HF) showing a length of 1.6 cm, compared to 3.63 cm for those treated with <em>Fusarium</em> only (treatment F) (refer to Table <a href="#Tab10" class="usa-link">10</a> and Fig. <a href="#Fig13" class="usa-link">13</a>).</p>
<figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/b253dd5a03eb/41598_2025_12974_Fig12_HTML.jpg" loading="lazy" id="MO12" height="447" width="775" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The impact of the QSLA1 isolate on shoot height (1), root length (2), and shoot infection length (3) after 28 days. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>). All treatments irrigated with salt water (1.5%).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig13"><h4 class="obj_head">Fig. 13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/b13d3bbaeaf9/41598_2025_12974_Fig13_HTML.jpg" loading="lazy" id="MO13" height="445" width="723" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The significant impact of inoculation with isolate QSLA1 on the length of shoot infection in tomato seedlings infected with <em>Fusarium oxysporium subsp. lycopersici</em> after 28 days. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>). All treatments irrigated with salt water (1.5%).</p></figcaption></figure><p id="Par57">Moreover, inoculation with QSLA1 significantly stimulated both shoot height and root length. The treated tomato plants achieved a shoot height of 14.3 cm and a root length of 7.5 cm, whereas seedlings treated with <em>Fusarium</em> alone (treatment F) measured 11.1 cm in height and 5.1 cm in root length (see Table <a href="#Tab10" class="usa-link">10</a>). These differences were statistically significant and underscored the beneficial effects of QSLA1 inoculation (refer to Figs. <a href="#Fig14" class="usa-link">14</a> and <a href="#Fig15" class="usa-link">15</a>).</p>
<figure class="fig xbox font-sm" id="Fig14"><h4 class="obj_head">Fig. 14.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig14_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/93bda972bc34/41598_2025_12974_Fig14_HTML.jpg" loading="lazy" id="MO14" height="451" width="723" alt="Fig. 14"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The significant impact of inoculation with isolate QSLA1 on the shoot height of tomato seedlings infected with <em>Fusarium oxysporium f. sp. lycopersici</em> after 28 days. Treatment S = tomato seedlings irrigated with salt water (1.5%), not treated with QSLA1and not infected with <em>Fusarium</em>. Treatment HS = tomato seedlings treated with QSLA1 isolate (10<sup>–2</sup>) and not infected with <em>Fusarium</em>. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>). All treatments irrigated with salt water (1.5%).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig15"><h4 class="obj_head">Fig. 15.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370916_41598_2025_12974_Fig15_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/200d/12370916/411de50adb12/41598_2025_12974_Fig15_HTML.jpg" loading="lazy" id="MO15" height="461" width="723" alt="Fig. 15"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Impact of Inoculation with Isolate QSLA1 on root length of Tomato Seedlings Infected with <em>Fusarium oxysporium f.sp. lycopersici</em> after 28 Days. -Treatment S = tomato seedlings irrigated with salt water (1.5%), not treated with QSLA1and not infected with <em>Fusarium</em>. Treatment HS = tomato seedlings treated with QSLA1 isolate (10<sup>–2</sup>) and not infected with <em>Fusarium</em>. Treatment FS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. lycopersici</em>, and not treated with QSLA1. Treatment HFS = tomato seedlings infected with 3% (w/w) of <em>Fusarium oxysporium sub sp. Lycopersci</em> and inoculated with QSLA1 (10<sup>–2</sup>).—All treatments irrigated with salt water (1.5%).</p></figcaption></figure><p id="Par58">These results underscore the significant effects of the QSLA1 isolate on plant growth and infection dynamics, also highlights the potential of isolate QSLA1 in promoting seedling growth even in the presence of this pathogen.</p>
<p id="Par59">The infection process begins with fungal hyphae adhering to and penetrating the root surface. The mycelium invades the root cortical cells intercellularly, subsequently entering the vascular system through the xylem. This fungus exhibits a distinct infection pathway, effectively colonizing exclusively inside the xylem vessels, allowing it to rapidly infiltrate the host. Within these vessels, the fungus produces microconidia, which ascend through the sap stream. The germination of these microconidia facilitates mycelial penetration of the upper vessels. Characteristic wilt symptoms arise from vessel blockage caused by accumulating fungal hyphae, compounded by host–pathogen interactions involving the release of toxins (such as fusaric acid and lycomarasmin), gums, gels, and the formation of tyloses. Symptoms including leaf drooping, vessel obstruction, wilting, and defoliation eventually lead to host plant death<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p>
<p id="Par60">The management of tomato wilt disease caused by <em>Fusarium oxysporium f.sp. lycopersici</em> through chemical fungicides presents several challenges, including residual toxicity, environmental pollution, and the development of pathogen resistance to repeatedly used fungicides<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. In contrast, employing biocontrol agents such as halophilic bacteria effectively mitigates these issues. Numerous microbes, recognized as biocontrol agents such as <em>Bacillus spp., Pseudomonas spp</em>., <em>Streptomyces spp</em>., and <em>Trichoderma spp</em>. have demonstrated significant efficacy against soil-borne pathogens<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. These agents compete for ecological substrates by producing antibiotics, hydrogen cyanide, releasing siderophores, and secreting enzymes that lyse fungal cell walls, thereby functioning as effective biocontrol agents<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Additionally, they activate induced systemic resistance (ISR) across various crops against multiple diseases, utilizing signaling pathways involving jasmonic acid (JA), ethylene (ET), and salicylic acid (SA)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The antagonistic effect of isolate QSLA1 against <em>Fusarium</em> is likely attributed to one or more of these mechanisms.</p>
<p id="Par61">Isolate QSLA1 has been confirmed to produce IAA (0.15 µg/ml) and possesses nitrogen-fixing capabilities, contributing to improved plant growth, notably in shoot height and root length. Additionally, this strain can produce lipase, protease, and chitinase enzymes recognized for their antimicrobial<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> and antifungal<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> properties. GC–MS analysis indicates that strain QSLA1 can synthesize metabolites such as Desulphosinigrin, Undeca-2, 4, 6, 8, 10-pentaenal, 11-(2-furyl)-oxime, and Strychane, 1-acetyl-20α-hydroxy-16-methylene, all of which are acknowledged as potent antifungal and antimicrobial constituents<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<p id="Par62">Strains from the <em>Bacillus, Virgibacillus,</em> and <em>Halomonas</em> genera, were isolated from saline habitats in northeastern Algeria, demonstrate remarkable activity against pathogenic fungi including <em>Botrytis cinerea, Fusarium oxysporium, F. verticillioides,</em> and <em>Phytophthora capsici</em><sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Furthermore, tomato plants treated with moderately halophilic bacteria <em>Halomonas</em> sp. K2-5, isolated from various Tunisian Sebkhas (hypersaline soils), exhibited reduced stem canker lesions under greenhouse conditions<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p></section></section><section id="Sec33"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par63">This study demonstrates the potential of halophilic bacterial strains isolated from solar saltern ponds, attached to Qarun Lake, Fayoum governorate, Egypt, to promote plant growth and resist tomato wilt disease under saline conditions. The three isolated strains showed promising results in producing plant growth-promoting substances and one strain exhibited significant antagonistic activity against _<em>Fusarium</em>_ fungus, a pathogen affecting tomato seedlings. These findings suggest that these bacterial strains could be used as biofertilizers or biocontrol agents to improve crop productivity and resistance to disease in salt-affected soils.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Methodology &amp; statistical analysis, A. A.A.M. Results and discussions, W.H.M. Abstract &amp; introduction, A.E.E. Phylogenetic analysis &amp; figures preparing, H.E.E. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by The Science, Technology &amp; Innovation Funding Authority (STDF) in cooperation with The Egyptian Knowledge Bank (EKB). If the paper is accepted the publishing will be funded according to the Open Access Agreement for Egypt between Springer Nature and Science, Technology&amp; Innovation Funding Authority (STDF) in cooperation with Egyptian Knowledge Bank (EKB).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Sequence data that support the findings of this study have been deposited in NCBI with the primary accession code <a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442496" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442496</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442497" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442497</a>, &amp;<a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442498" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442498</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par64">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Abdel-Monaim, M. F. Induced systemic resistance in tomato plants against <em>Fusarium</em> wilt disease. <em>Int. Res. J. Microbiol.</em><strong>3</strong>(1), 014–023 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abdel-Monaim,%20M.%20F.%20Induced%20systemic%20resistance%20in%20tomato%20plants%20against%20Fusarium%20wilt%20disease.%20Int.%20Res.%20J.%20Microbiol.3(1),%20014%E2%80%93023%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Altschul, S. F. et al. Gapped BLAST and PSI-BLAST a new generation of protein database search programs. <em>Nucleic Acids Res.</em><strong>25</strong>, 3389–3402 (1997).
</cite> [<a href="https://doi.org/10.1093/nar/25.17.3389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altschul,%20S.%20F.%20et%20al.%20Gapped%20BLAST%20and%20PSI-BLAST%20a%20new%20generation%20of%20protein%20database%20search%20programs.%20Nucleic%20Acids%20Res.25,%203389%E2%80%933402%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Amer, M. A., El-Samra, I. A., Abou-El-Seoud, I. I., Sawsan, M. &amp; Shawertamimi, N. K. Induced systemic resistance in tomato plants against <em>Fusarium</em> wilt disease using biotic inducers. <em>Middle East J. Agric. Res.</em><strong>3</strong>(4), 1090–1103 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Amer,%20M.%20A.,%20El-Samra,%20I.%20A.,%20Abou-El-Seoud,%20I.%20I.,%20Sawsan,%20M.%20&amp;%20Shawertamimi,%20N.%20K.%20Induced%20systemic%20resistance%20in%20tomato%20plants%20against%20Fusarium%20wilt%20disease%20using%20biotic%20inducers.%20Middle%20East%20J.%20Agric.%20Res.3(4),%201090%E2%80%931103%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Bajpai, R. et al. Biocontrol of fusarium wilt in tomato an eco-friendly and cost effective approach. <em>Biol. Forum Int. J.</em><strong>13</strong>(2), 62–69 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bajpai,%20R.%20et%20al.%20Biocontrol%20of%20fusarium%20wilt%20in%20tomato%20an%20eco-friendly%20and%20cost%20effective%20approach.%20Biol.%20Forum%20Int.%20J.13(2),%2062%E2%80%9369%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Benito, J. M., Lovrich, G. A., Siñeriz, F. &amp; Abate, C. M. Isolation and molecular characterization of seawater bacteria. <em>Environ. Microbiol. Methods Protoc</em><strong>3</strong>, 10 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Benito,%20J.%20M.,%20Lovrich,%20G.%20A.,%20Si%C3%B1eriz,%20F.%20&amp;%20Abate,%20C.%20M.%20Isolation%20and%20molecular%20characterization%20of%20seawater%20bacteria.%20Environ.%20Microbiol.%20Methods%20Protoc3,%2010%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Boussada, O. et al. Chemical composition and antimicrobial activity of volatile components from capitula and aerial parts of Rhaponticum acaule DC growing wild in Tunisia. <em>Microbiol. Res.</em><strong>163</strong>, 87–95 (2008).
</cite> [<a href="https://doi.org/10.1016/j.micres.2007.02.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17482441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boussada,%20O.%20et%20al.%20Chemical%20composition%20and%20antimicrobial%20activity%20of%20volatile%20components%20from%20capitula%20and%20aerial%20parts%20of%20Rhaponticum%20acaule%20DC%20growing%20wild%20in%20Tunisia.%20Microbiol.%20Res.163,%2087%E2%80%9395%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Broderick, N. A., Raffa, K. F., Goodman, R. M. &amp; Handelsman, J. Census of the bacterial community of the gypsy moth larval midgut by using culturing and cultureindependent methods. <em>Appl. Environ. Microbiol.</em><strong>70</strong>(1), 293–300 (2004).
</cite> [<a href="https://doi.org/10.1128/AEM.70.1.293-300.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC321235/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14711655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Broderick,%20N.%20A.,%20Raffa,%20K.%20F.,%20Goodman,%20R.%20M.%20&amp;%20Handelsman,%20J.%20Census%20of%20the%20bacterial%20community%20of%20the%20gypsy%20moth%20larval%20midgut%20by%20using%20culturing%20and%20cultureindependent%20methods.%20Appl.%20Environ.%20Microbiol.70(1),%20293%E2%80%93300%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Chen, L. et al. Phylogenetic analysis &amp; screening of antimicrobial &amp; cytotoxic activities of moderately halophilic bacteria isolated from the Weihai solar Saltern (China). <em>World J. Microbiol. Biotechnol.</em><strong>26</strong>, 879–888 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen,%20L.%20et%20al.%20Phylogenetic%20analysis%20&amp;%20screening%20of%20antimicrobial%20&amp;%20cytotoxic%20activities%20of%20moderately%20halophilic%20bacteria%20isolated%20from%20the%20Weihai%20solar%20Saltern%20(China).%20World%20J.%20Microbiol.%20Biotechnol.26,%20879%E2%80%93888%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Corden, M. E. &amp; Chambers, H. L. Vascular dysfunction in <em>Fusarium</em> wilt of tomato. <em>Amer. J. Bot.</em><strong>53</strong>(3), 284–287 (1966).</cite> [<a href="https://scholar.google.com/scholar_lookup?Corden,%20M.%20E.%20&amp;%20Chambers,%20H.%20L.%20Vascular%20dysfunction%20in%20Fusarium%20wilt%20of%20tomato.%20Amer.%20J.%20Bot.53(3),%20284%E2%80%93287%20(1966)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Darwesh, O. M., Mahmoud, M. S., Barakat, K. M., Abuellil, A. &amp; Ahmad, M. E. Improving the bioremediumtion technology of contaminated wastewater using biosurfactants produced by novel <em>Bacillus</em> isolates. <em>Heliyon</em><strong>7</strong>, e08616 (2021).
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2021.e08616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8703238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34988315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Darwesh,%20O.%20M.,%20Mahmoud,%20M.%20S.,%20Barakat,%20K.%20M.,%20Abuellil,%20A.%20&amp;%20Ahmad,%20M.%20E.%20Improving%20the%20bioremediumtion%20technology%20of%20contaminated%20wastewater%20using%20biosurfactants%20produced%20by%20novel%20Bacillus%20isolates.%20Heliyon7,%20e08616%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Desale, P., Patel, B., Singh, S., Malhotra, A. &amp; Nawani, N. Plant growth promoting properties of <em>Halobacillus</em> sp. and <em>Halomonas</em> sp. in presence of salinity and heavy metals. <em>J. Basic Microbiol.</em><strong>54</strong>, 781–791 (2014).
</cite> [<a href="https://doi.org/10.1002/jobm.201200778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23775888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Desale,%20P.,%20Patel,%20B.,%20Singh,%20S.,%20Malhotra,%20A.%20&amp;%20Nawani,%20N.%20Plant%20growth%20promoting%20properties%20of%20Halobacillus%20sp.%20and%20Halomonas%20sp.%20in%20presence%20of%20salinity%20and%20heavy%20metals.%20J.%20Basic%20Microbiol.54,%20781%E2%80%93791%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Edbeib, M. F., Abdul Wahab, R. &amp; Huyop, F. Halophiles biology, adaptation, and their role in decontamination of hypersaline environments. <em>World J. Microbiol. Biotechnol.</em><strong>32</strong>, 135. 10.1007/s11274-016-2081-9 (2016).
</cite> [<a href="https://doi.org/10.1007/s11274-016-2081-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27344438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edbeib,%20M.%20F.,%20Abdul%20Wahab,%20R.%20&amp;%20Huyop,%20F.%20Halophiles%20biology,%20adaptation,%20and%20their%20role%20in%20decontamination%20of%20hypersaline%20environments.%20World%20J.%20Microbiol.%20Biotechnol.32,%20135.%2010.1007/s11274-016-2081-9%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Essghaier, B. Antimicrobial behavior of Intracellular proteins from two moderately halophilic bacteria strain J31 of <em>Terribacillus halophilus</em> and strain M3–23 of <em>Virgibacillus marismortui</em>. <em>J. Plant. Pathol. Microbiol</em><strong>5</strong>, 1000214. 10.4172/2157-7471.1000214 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Essghaier,%20B.%20Antimicrobial%20behavior%20of%20Intracellular%20proteins%20from%20two%20moderately%20halophilic%20bacteria%20strain%20J31%20of%20Terribacillus%20halophilus%20and%20strain%20M3%E2%80%9323%20of%20Virgibacillus%20marismortui.%20J.%20Plant.%20Pathol.%20Microbiol5,%201000214.%2010.4172/2157-7471.1000214%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Farajzadeh, M. A., Bahram, M., Vardast, M. R. &amp; Bamorowat, M. Dispersive liquid-liquid microextraction for the analysis of three organophosphorus pesticides in real samples by high-performance liquid chromatography–ultraviolet detection and its optimization by experimental design. <em>Microchim. Acta</em><strong>172</strong>, 465–470. 10.1007/s00604-010-0451-9 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Farajzadeh,%20M.%20A.,%20Bahram,%20M.,%20Vardast,%20M.%20R.%20&amp;%20Bamorowat,%20M.%20Dispersive%20liquid-liquid%20microextraction%20for%20the%20analysis%20of%20three%20organophosphorus%20pesticides%20in%20real%20samples%20by%20high-performance%20liquid%20chromatography%E2%80%93ultraviolet%20detection%20and%20its%20optimization%20by%20experimental%20design.%20Microchim.%20Acta172,%20465%E2%80%93470.%2010.1007/s00604-010-0451-9%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Ferreira, M. J. et al. The root microbiome of <em>Salicornia ramosissima</em> as a seedbank for plant-growth promoting halotolerant bacteria. <em>Appl. Sci.</em><strong>11</strong>, 2233. 10.3390/app11052233 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ferreira,%20M.%20J.%20et%20al.%20The%20root%20microbiome%20of%20Salicornia%20ramosissima%20as%20a%20seedbank%20for%20plant-growth%20promoting%20halotolerant%20bacteria.%20Appl.%20Sci.11,%202233.%2010.3390/app11052233%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Gayathiri, E. et al. Biosurfactants potential and eco-friendly material for sustainable agriculture and environmental safety: A review. <em>Agronomy</em><strong>12</strong>, 662. 10.3390/agronomy12030662 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gayathiri,%20E.%20et%20al.%20Biosurfactants%20potential%20and%20eco-friendly%20material%20for%20sustainable%20agriculture%20and%20environmental%20safety:%20A%20review.%20Agronomy12,%20662.%2010.3390/agronomy12030662%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Ghozlan, H., Deif, H., Kandil, R. A. &amp; Sabry, S. Biodiversity of moderately halophilic bacteria in hypersaline habitats. <em>J. Gen Appl. Microbiol.</em><strong>52</strong>, 63–72 (2006).
</cite> [<a href="https://doi.org/10.2323/jgam.52.63" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16778349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghozlan,%20H.,%20Deif,%20H.,%20Kandil,%20R.%20A.%20&amp;%20Sabry,%20S.%20Biodiversity%20of%20moderately%20halophilic%20bacteria%20in%20hypersaline%20habitats.%20J.%20Gen%20Appl.%20Microbiol.52,%2063%E2%80%9372%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Gomaa, et al. Control of tomato <em>Fusarium</em> wilt caused by <em>Fusarium oxysporium</em> f.sp. lycopersici by grafting and silver nanoparticles under greenhouse conditions. <em>Benha J. Appl. Sci. BJAS</em><strong>7</strong>(5), 37–50 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gomaa,%20et%20al.%20Control%20of%20tomato%20Fusarium%20wilt%20caused%20by%20Fusarium%20oxysporium%20f.sp.%20lycopersici%20by%20grafting%20and%20silver%20nanoparticles%20under%20greenhouse%20conditions.%20Benha%20J.%20Appl.%20Sci.%20BJAS7(5),%2037%E2%80%9350%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Gordon, S. A. &amp; Weber, R. P. Colorimetric estimation of indole acetic acid. <em>Plant Physiol.</em><strong>26</strong>(1), 192–195 (1950).</cite> [<a href="https://doi.org/10.1104/pp.26.1.192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC437633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16654351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gordon,%20S.%20A.%20&amp;%20Weber,%20R.%20P.%20Colorimetric%20estimation%20of%20indole%20acetic%20acid.%20Plant%20Physiol.26(1),%20192%E2%80%93195%20(1950)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Goszczynska, T., Serfontein, J. J. &amp; Serfontein, S. Introduction to practical phytobacteriology a manual for phytobacteriology, safrinet. <em>South Afr. Sadc Loop Bionet-Int.</em><strong>83</strong>, 1112 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Goszczynska,%20T.,%20Serfontein,%20J.%20J.%20&amp;%20Serfontein,%20S.%20Introduction%20to%20practical%20phytobacteriology%20a%20manual%20for%20phytobacteriology,%20safrinet.%20South%20Afr.%20Sadc%20Loop%20Bionet-Int.83,%201112%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Gulzar, N., Kamili, A. N. &amp; Shah, M. A. Tomato early blight (<em>Alternaria solani</em>), pathogen, disease development &amp; defense response phytohormone signaling. <em>Int. Res. J. Plant Sci.</em><strong>12</strong>(4), 1–10 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gulzar,%20N.,%20Kamili,%20A.%20N.%20&amp;%20Shah,%20M.%20A.%20Tomato%20early%20blight%20(Alternaria%20solani),%20pathogen,%20disease%20development%20&amp;%20defense%20response%20phytohormone%20signaling.%20Int.%20Res.%20J.%20Plant%20Sci.12(4),%201%E2%80%9310%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Hopwood, D. A. The phase-contrast observations on <em>Streptomyces coelicolor</em>. <em>J. Gen. Microbiol.</em><strong>22</strong>, 295–302 (1960).
</cite> [<a href="https://doi.org/10.1099/00221287-22-1-295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14403062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hopwood,%20D.%20A.%20The%20phase-contrast%20observations%20on%20Streptomyces%20coelicolor.%20J.%20Gen.%20Microbiol.22,%20295%E2%80%93302%20(1960)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite><strong>Hucker, G. J., &amp; Conn, H. J. (1923).</strong> Methods of Gram Staining. <em>Technical Bulletin</em>, Vol. <strong>93</strong>, 3–38 (New York Agricultural Experiment Station, Geneva).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hucker,%20G.%20J.,%20&amp;%20Conn,%20H.%20J.%20(1923).%20Methods%20of%20Gram%20Staining.%20Technical%20Bulletin,%20Vol.%2093,%203%E2%80%9338%20(New%20York%20Agricultural%20Experiment%20Station,%20Geneva)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite><em>Int. J. Anal. Mass Spectrom. Chromatogr.</em><strong>4</strong>, 14–25 (2016).</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Jiang, H., Okoye, C. O., Ezenwanne, B. C., Wu, Y. &amp; Jiang, J. Synergistic potential of halophytes and halophilic/halotolerant plant growth-promoting bacteria in saline soil remediation: Adaptive mechanisms, challenges, and sustainable solutions. <em>Microbiol. Res.</em><strong>298</strong>, 12877. 10.1016/j.micres.2025.128227 (2025).</cite> [<a href="https://doi.org/10.1016/j.micres.2025.128227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40440870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20H.,%20Okoye,%20C.%20O.,%20Ezenwanne,%20B.%20C.,%20Wu,%20Y.%20&amp;%20Jiang,%20J.%20Synergistic%20potential%20of%20halophytes%20and%20halophilic/halotolerant%20plant%20growth-promoting%20bacteria%20in%20saline%20soil%20remediation:%20Adaptive%20mechanisms,%20challenges,%20and%20sustainable%20solutions.%20Microbiol.%20Res.298,%2012877.%2010.1016/j.micres.2025.128227%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kesaulya, H., Talahaturuson, A., Kalay, A. M., Matatula, E., Lawalatta, I. J., Hehanussa, M. L. &amp; Nendissa, S. J. Characterization of plant growth promoting rhizobacteria of Maize. In <em>International Seminar on Agriculture, Biodiversity, Food Security &amp; Health IOP Conference Series Earth &amp; Environmental Science</em> 883 (2021).</cite>
</li>
<li id="CR27">
<span class="label">27.</span><cite>Kizilkaya, R. Nitrogen fixation capacity of <em>Azotobacter</em> spp. strains isolated from soils in different ecosystems &amp; relationship between them &amp; the microbiological properties of soils. <em>J. Environ. Biol.</em><strong>30</strong>(1), 73–82 (2009).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/20112866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kizilkaya,%20R.%20Nitrogen%20fixation%20capacity%20of%20Azotobacter%20spp.%20strains%20isolated%20from%20soils%20in%20different%20ecosystems%20&amp;%20relationship%20between%20them%20&amp;%20the%20microbiological%20properties%20of%20soils.%20J.%20Environ.%20Biol.30(1),%2073%E2%80%9382%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Krishnaveni, M. Docking, simulation studies of desulphosinigrin-cyclin dependent kinase 2, an anticancer drug target. <em>Int. J. Pharm. Sci. Rev. Res.</em><strong>30</strong>(2), 115–118 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Krishnaveni,%20M.%20Docking,%20simulation%20studies%20of%20desulphosinigrin-cyclin%20dependent%20kinase%202,%20an%20anticancer%20drug%20target.%20Int.%20J.%20Pharm.%20Sci.%20Rev.%20Res.30(2),%20115%E2%80%93118%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kumar, S., Stecher, G., Li, M., Knyaz, C. &amp; Tamura, K. MEGA X molecular evolutionary genetics analysis across computing platforms. <em>Mol. Biol. Evol.</em><strong>35</strong>, 1547–1549 (2018).
</cite> [<a href="https://doi.org/10.1093/molbev/msy096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5967553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20S.,%20Stecher,%20G.,%20Li,%20M.,%20Knyaz,%20C.%20&amp;%20Tamura,%20K.%20MEGA%20X%20molecular%20evolutionary%20genetics%20analysis%20across%20computing%20platforms.%20Mol.%20Biol.%20Evol.35,%201547%E2%80%931549%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Li, X. &amp; Yu, Y. H. Biodiversity &amp; screening of halophilic bacteria with hydrolytic &amp; antimicrobial activities from Yuncheng Salt Lake, China. <em>Biologia</em><strong>70</strong>, 151–156. 10.1515/biolog-2015-0033 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Li,%20X.%20&amp;%20Yu,%20Y.%20H.%20Biodiversity%20&amp;%20screening%20of%20halophilic%20bacteria%20with%20hydrolytic%20&amp;%20antimicrobial%20activities%20from%20Yuncheng%20Salt%20Lake,%20China.%20Biologia70,%20151%E2%80%93156.%2010.1515/biolog-2015-0033%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Managamuri, U. et al. Isolation, identification, optimization &amp; metabolite profiling of <em>Streptomyces sparsus</em> VSM-30. <em>3 Biotech.</em><strong>7</strong>, 217. 10.1007/s13205-017-0835-1 (2017).
</cite> [<a href="https://doi.org/10.1007/s13205-017-0835-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5494031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28669076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Managamuri,%20U.%20et%20al.%20Isolation,%20identification,%20optimization%20&amp;%20metabolite%20profiling%20of%20Streptomyces%20sparsus%20VSM-30.%203%20Biotech.7,%20217.%2010.1007/s13205-017-0835-1%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Mao, Y., Zhang, X. &amp; Xu, Z. Identification of antibacterial substances of <em>Lactobacillus plantarum</em> DY-6 for bacteriostatic action. <em>Food Sci. Nutr.</em><strong>8</strong>(6), 2854–2863 (2020).
</cite> [<a href="https://doi.org/10.1002/fsn3.1585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7300085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32566203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao,%20Y.,%20Zhang,%20X.%20&amp;%20Xu,%20Z.%20Identification%20of%20antibacterial%20substances%20of%20Lactobacillus%20plantarum%20DY-6%20for%20bacteriostatic%20action.%20Food%20Sci.%20Nutr.8(6),%202854%E2%80%932863%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Meah, M. B., Plumbley, R. A. &amp; Jeger, M. J. Growth &amp; infectivity of <em>Botryodiplodia theobromae</em> causingstem-end rot of mango. <em>Mycol. Res.</em><strong>9S</strong>(4), 405–408 (1991).</cite> [<a href="https://scholar.google.com/scholar_lookup?Meah,%20M.%20B.,%20Plumbley,%20R.%20A.%20&amp;%20Jeger,%20M.%20J.%20Growth%20&amp;%20infectivity%20of%20Botryodiplodia%20theobromae%20causingstem-end%20rot%20of%20mango.%20Mycol.%20Res.9S(4),%20405%E2%80%93408%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Menasria, T. et al. Diversity &amp; bioprospecting of extremely halophilic archaea isolated from Algerian arid &amp; semi-arid wetl &amp; ecosystems for halophilic-activehydrolytic enzymes. <em>Microbiol. Res.</em><strong>207</strong>, 289–298 (2018).
</cite> [<a href="https://doi.org/10.1016/j.micres.2017.12.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29458865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menasria,%20T.%20et%20al.%20Diversity%20&amp;%20bioprospecting%20of%20extremely%20halophilic%20archaea%20isolated%20from%20Algerian%20arid%20&amp;%20semi-arid%20wetl%20&amp;%20ecosystems%20for%20halophilic-activehydrolytic%20enzymes.%20Microbiol.%20Res.207,%20289%E2%80%93298%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Menasria, T. et al. Culturable halophilic bacteria inhabiting Algerian saline ecosystems a source of promising features &amp; potentialities. <em>World J. Microbiol. Biotechnol.</em><strong>35</strong>, 132 (2019).
</cite> [<a href="https://doi.org/10.1007/s11274-019-2705-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31432260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menasria,%20T.%20et%20al.%20Culturable%20halophilic%20bacteria%20inhabiting%20Algerian%20saline%20ecosystems%20a%20source%20of%20promising%20features%20&amp;%20potentialities.%20World%20J.%20Microbiol.%20Biotechnol.35,%20132%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Mohamedin, A. H., Mowafy, A. M., Elsayed, A. A. &amp; Ghanim, S. O. Potential applications of some moderate halophilic bacteria. <em>Egypt. J. Aquat. Biol. Fish.</em><strong>22</strong>(5), 537–550 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mohamedin,%20A.%20H.,%20Mowafy,%20A.%20M.,%20Elsayed,%20A.%20A.%20&amp;%20Ghanim,%20S.%20O.%20Potential%20applications%20of%20some%20moderate%20halophilic%20bacteria.%20Egypt.%20J.%20Aquat.%20Biol.%20Fish.22(5),%20537%E2%80%93550%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Mohammadipanah, F., Hamedi, J. &amp; Dehhaghi, M. (2015). Halophilic bacteria potentials &amp; applications in biotechnology. In <em>Halophiles</em><strong>,</strong><em> Biodiversity &amp; Sustainable Exploitation. Library of Congress</em> (ed. Maheshwari, D, Saraf, M.) Vol. 6, 277–321. 10.1007/978-3-319-14595-2_11</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Mwirichia, R., Muigai, A. W., Cousin, S., Boga, H. I. &amp; Stackebrandt, E. Bacterial diversity in the Haloalkaline lake Elmenteita in Kenya. <em>Curr. Microbiol.</em><strong>62</strong>(1), 209–221 (2010).
</cite> [<a href="https://doi.org/10.1007/s00284-010-9692-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20582590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mwirichia,%20R.,%20Muigai,%20A.%20W.,%20Cousin,%20S.,%20Boga,%20H.%20I.%20&amp;%20Stackebrandt,%20E.%20Bacterial%20diversity%20in%20the%20Haloalkaline%20lake%20Elmenteita%20in%20Kenya.%20Curr.%20Microbiol.62(1),%20209%E2%80%93221%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Naghoni, A. et al. Microbial diversity in the hypersaline lake Meyghan, Iran. <em>Sci. Rep.</em><strong>7</strong>, 11522. 10.1038/s41598-017-11585-3 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-11585-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5599592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28912589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naghoni,%20A.%20et%20al.%20Microbial%20diversity%20in%20the%20hypersaline%20lake%20Meyghan,%20Iran.%20Sci.%20Rep.7,%2011522.%2010.1038/s41598-017-11585-3%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Paul, S. et al. Isolation &amp; characterization of salt tolerant bacteria from Sambhar Lake for plant growth promoting activities Sangeeta. <em>Environ. Ecol.</em><strong>34</strong>(1A), 365–371 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Paul,%20S.%20et%20al.%20Isolation%20&amp;%20characterization%20of%20salt%20tolerant%20bacteria%20from%20Sambhar%20Lake%20for%20plant%20growth%20promoting%20activities%20Sangeeta.%20Environ.%20Ecol.34(1A),%20365%E2%80%93371%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Peeters, N., Guidot, A., Vailleau, F. &amp; Valls, M. <em>Ralstonia Solanacearum</em>, a widespread bacterial plant pathogen inthe post-genomic era. <em>Mol. Plant Pathol.</em><strong>14</strong>(7), 651–662. 10.1111/Mpp.12038 (2013).
</cite> [<a href="https://doi.org/10.1111/mpp.12038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6638647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23718203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peeters,%20N.,%20Guidot,%20A.,%20Vailleau,%20F.%20&amp;%20Valls,%20M.%20Ralstonia%20Solanacearum,%20a%20widespread%20bacterial%20plant%20pathogen%20inthe%20post-genomic%20era.%20Mol.%20Plant%20Pathol.14(7),%20651%E2%80%93662.%2010.1111/Mpp.12038%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Puchkova, E. P., Gaas, M. V., Khizhnyak, S. V., Ivchenko, V. K. &amp; Polosina, V. A. The occurrence of antagonists microorganisms to phytopathogenic fungi in consideration of various tillages. <em>IOP Conf. Ser. Earth Environ. Sci.</em><strong>421</strong>, 062037. 10.1088/1755-1315/421/6/062037 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Puchkova,%20E.%20P.,%20Gaas,%20M.%20V.,%20Khizhnyak,%20S.%20V.,%20Ivchenko,%20V.%20K.%20&amp;%20Polosina,%20V.%20A.%20The%20occurrence%20of%20antagonists%20microorganisms%20to%20phytopathogenic%20fungi%20in%20consideration%20of%20various%20tillages.%20IOP%20Conf.%20Ser.%20Earth%20Environ.%20Sci.421,%20062037.%2010.1088/1755-1315/421/6/062037%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Ramadoss, D., Lakkineni, V. K., Bose, P., Ali, S. &amp; Annapurna, K. Mitigation of salt stress in wheat seedlings by halotolerant bacteria isolated from saline habitats. <em>Springerplus</em><strong>26</strong>, 2–6. 10.1186/2193-1801 (2013).</cite> [<a href="https://doi.org/10.1186/2193-1801-2-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3579424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23449812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramadoss,%20D.,%20Lakkineni,%20V.%20K.,%20Bose,%20P.,%20Ali,%20S.%20&amp;%20Annapurna,%20K.%20Mitigation%20of%20salt%20stress%20in%20wheat%20seedlings%20by%20halotolerant%20bacteria%20isolated%20from%20saline%20habitats.%20Springerplus26,%202%E2%80%936.%2010.1186/2193-1801%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Rani, H. S. M. &amp; Kalaiselvam, M. Antibacterial activity of halophilic fungi isolated from solar Saltern at Tuticorin. <em>World J. Pharm. Pharm. Sci.</em><strong>2</strong>(2), 536–545 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rani,%20H.%20S.%20M.%20&amp;%20Kalaiselvam,%20M.%20Antibacterial%20activity%20of%20halophilic%20fungi%20isolated%20from%20solar%20Saltern%20at%20Tuticorin.%20World%20J.%20Pharm.%20Pharm.%20Sci.2(2),%20536%E2%80%93545%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Roux, K. H. <em>Optimization &amp; Trouble Shooting in PCR</em> 53–62 (Cold Spring Harbor Laboratory Press, New York, 1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Roux,%20K.%20H.%20Optimization%20&amp;%20Trouble%20Shooting%20in%20PCR%2053%E2%80%9362%20(Cold%20Spring%20Harbor%20Laboratory%20Press,%20New%20York,%201995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ruginescu, R. et al. Bioprospecting for novel halophilic &amp; halotolerant sources of hydrolytic enzymes in brackish, saline &amp; hypersaline lakes of Romania. <em>Microorganisms</em><strong>8</strong>, 1903 (2020).
</cite> [<a href="https://doi.org/10.3390/microorganisms8121903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7760675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33266166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruginescu,%20R.%20et%20al.%20Bioprospecting%20for%20novel%20halophilic%20&amp;%20halotolerant%20sources%20of%20hydrolytic%20enzymes%20in%20brackish,%20saline%20&amp;%20hypersaline%20lakes%20of%20Romania.%20Microorganisms8,%201903%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Saitou, N. &amp; Nei, M. The neighbor-joining method a new method for reconstructing phylogenetic trees. <em>Mol. Biol. Evol.</em><strong>4</strong>, 406–425 (1987).
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a040454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3447015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saitou,%20N.%20&amp;%20Nei,%20M.%20The%20neighbor-joining%20method%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees.%20Mol.%20Biol.%20Evol.4,%20406%E2%80%93425%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sambrook, K. J., Fritsch, E. F. &amp; Maniatis, T. <em>Molecular Cloning a Laboratory Manual</em> 2nd edn. (Cold Spring Harbor, New York, 1989).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sambrook,%20K.%20J.,%20Fritsch,%20E.%20F.%20&amp;%20Maniatis,%20T.%20Molecular%20Cloning%20a%20Laboratory%20Manual%202nd%20edn.%20(Cold%20Spring%20Harbor,%20New%20York,%201989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Saravanakumar, D., Vijayakumar, C., Kumar, N. &amp; Samiyappan, R. PGPR-induced defense responses in the tea plant against blister blight disease. <em>Crop Prot.</em><strong>26</strong>, 556–565 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Saravanakumar,%20D.,%20Vijayakumar,%20C.,%20Kumar,%20N.%20&amp;%20Samiyappan,%20R.%20PGPR-induced%20defense%20responses%20in%20the%20tea%20plant%20against%20blister%20blight%20disease.%20Crop%20Prot.26,%20556%E2%80%93565%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Sari, C. N., Hertadi, R., Harahap, A. F. P., Ramadhan, M. Y. A. &amp; Gozan, M. Process optimization of palm oil mill E-uent-based biosurfactant of <em>Halomonas meridiana</em> BK-AB4 originated from Bledug Kuwu mud volcano in central java for microbial enhanced oil recovery. <em>Processes</em><strong>8</strong>, 716 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sari,%20C.%20N.,%20Hertadi,%20R.,%20Harahap,%20A.%20F.%20P.,%20Ramadhan,%20M.%20Y.%20A.%20&amp;%20Gozan,%20M.%20Process%20optimization%20of%20palm%20oil%20mill%20E-uent-based%20biosurfactant%20of%20Halomonas%20meridiana%20BK-AB4%20originated%20from%20Bledug%20Kuwu%20mud%20volcano%20in%20central%20java%20for%20microbial%20enhanced%20oil%20recovery.%20Processes8,%20716%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Sarker, A. &amp; Al-Rashid, J. <em>Analytical Protocol for Determination of Indole 3 Acetic Acid (IAA) Production by Plant Growth Promoting Bacteria (PGPB) Based on the Colorimetric Method of Gordon &amp; Weber (1951)</em> (2013).</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Segarra, G., Casanova, E., Avilés, M. &amp; Trillas, I. <em>Trichoderma asperellum</em> strain t34 controls <em>Fusarium</em> wilt disease in tomato plants in soilless culture through competition for iron. <em>Microb Ecol.</em><strong>59</strong>, 141–149. 10.1007/s00248-009-9545-5 (2009).
</cite> [<a href="https://doi.org/10.1007/s00248-009-9545-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19536588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Segarra,%20G.,%20Casanova,%20E.,%20Avil%C3%A9s,%20M.%20&amp;%20Trillas,%20I.%20Trichoderma%20asperellum%20strain%20t34%20controls%20Fusarium%20wilt%20disease%20in%20tomato%20plants%20in%20soilless%20culture%20through%20competition%20for%20iron.%20Microb%20Ecol.59,%20141%E2%80%93149.%2010.1007/s00248-009-9545-5%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Srinivas, C. et al. <em>Fusarium oxysporium</em> f.sp. <em>lycopersici</em> causal agent of vascular wilt disease of tomato Biology to diversity. A review. <em>Saudi J. Biol. Sci.</em><strong>26</strong>, 1315–1324 (2019).
</cite> [<a href="https://doi.org/10.1016/j.sjbs.2019.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6864208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31762590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Srinivas,%20C.%20et%20al.%20Fusarium%20oxysporium%20f.sp.%20lycopersici%20causal%20agent%20of%20vascular%20wilt%20disease%20of%20tomato%20Biology%20to%20diversity.%20A%20review.%20Saudi%20J.%20Biol.%20Sci.26,%201315%E2%80%931324%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Suthar, M. P., Hajoori, M. A., Chaudhari, R. R. &amp; Desai, S. A. Isolation, screening &amp; characterization of potent biosurfactant producing bacteria from oil contaminated site. <em>Biosci. Discov.</em><strong>8</strong>(3), 375–381 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Suthar,%20M.%20P.,%20Hajoori,%20M.%20A.,%20Chaudhari,%20R.%20R.%20&amp;%20Desai,%20S.%20A.%20Isolation,%20screening%20&amp;%20characterization%20of%20potent%20biosurfactant%20producing%20bacteria%20from%20oil%20contaminated%20site.%20Biosci.%20Discov.8(3),%20375%E2%80%93381%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Tanvir, R., Javeed, A. &amp; Rehman, Y. Fatty acids &amp; their amide derivatives from endophytes new therapeutic possibilities from a hidden source. <em>FEMS Microbiol. Lett.</em><strong>365</strong>(12), 114. 10.1093/femsle/fny114 (2018).</cite> [<a href="https://doi.org/10.1093/femsle/fny114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29733374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanvir,%20R.,%20Javeed,%20A.%20&amp;%20Rehman,%20Y.%20Fatty%20acids%20&amp;%20their%20amide%20derivatives%20from%20endophytes%20new%20therapeutic%20possibilities%20from%20a%20hidden%20source.%20FEMS%20Microbiol.%20Lett.365(12),%20114.%2010.1093/femsle/fny114%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Teh, C. H., Nazni, W. A., Norazah, A. &amp; Lee, H. L. Determination of antibacterial activity &amp; minimum inhibitory concentration of larval extract of fly via resazurin-based turbidometric assay. <em>BMC Microbiol.</em><strong>17</strong>(1), 1–8 (2017).
</cite> [<a href="https://doi.org/10.1186/s12866-017-0936-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5314469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28209130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teh,%20C.%20H.,%20Nazni,%20W.%20A.,%20Norazah,%20A.%20&amp;%20Lee,%20H.%20L.%20Determination%20of%20antibacterial%20activity%20&amp;%20minimum%20inhibitory%20concentration%20of%20larval%20extract%20of%20fly%20via%20resazurin-based%20turbidometric%20assay.%20BMC%20Microbiol.17(1),%201%E2%80%938%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Tiwari, S. et al. Salt-tolerant rhizobacteria-mediated induced tolerance in wheat (<em>Triticum aestivum</em>) &amp; chemical diversity in rhizosphere enhance plant growth. <em>Biol. Fertil. Soils</em><strong>2011</strong>(47), 907–916 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tiwari,%20S.%20et%20al.%20Salt-tolerant%20rhizobacteria-mediated%20induced%20tolerance%20in%20wheat%20(Triticum%20aestivum)%20&amp;%20chemical%20diversity%20in%20rhizosphere%20enhance%20plant%20growth.%20Biol.%20Fertil.%20Soils2011(47),%20907%E2%80%93916%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Upadhyay, A. K., Bandi, S. R. &amp; Peddaguad, D. M. Bio-control agents-antagonistic magicians against soil borne pathogens a review. Biological forum. <em>Int. J.</em><strong>13</strong>, 232–242 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Upadhyay,%20A.%20K.,%20Bandi,%20S.%20R.%20&amp;%20Peddaguad,%20D.%20M.%20Bio-control%20agents-antagonistic%20magicians%20against%20soil%20borne%20pathogens%20a%20review.%20Biological%20forum.%20Int.%20J.13,%20232%E2%80%93242%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Van Der Velie, D. et al. Use of single-point genome signature tags as a universal tagging method for microbial genome surveys@@@. <em>Appl. Environ. Microbiol.</em><strong>72</strong>(3), 2092–2101 (2011).</cite> [<a href="https://doi.org/10.1128/AEM.72.3.2092-2101.2006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1393173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16517658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Der%20Velie,%20D.%20et%20al.%20Use%20of%20single-point%20genome%20signature%20tags%20as%20a%20universal%20tagging%20method%20for%20microbial%20genome%20surveys@@@.%20Appl.%20Environ.%20Microbiol.72(3),%202092%E2%80%932101%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Ventosa, A. &amp; Arahal, D. R. Physico-chemical characteristics of hypersaline environments &amp; their biodiversity. Encyclopedia of life support systems (EOLSS). <em>Extremophiles</em><strong>2</strong>, 247 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ventosa,%20A.%20&amp;%20Arahal,%20D.%20R.%20Physico-chemical%20characteristics%20of%20hypersaline%20environments%20&amp;%20their%20biodiversity.%20Encyclopedia%20of%20life%20support%20systems%20(EOLSS).%20Extremophiles2,%20247%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Whittenbury, R. Hydrogen peroxide formation &amp; catalase activity in the lactic acid bacteria. <em>J. Gen. Microbiol.</em><strong>35</strong>, 13–26 (1964).
</cite> [<a href="https://doi.org/10.1099/00221287-35-1-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14167645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whittenbury,%20R.%20Hydrogen%20peroxide%20formation%20&amp;%20catalase%20activity%20in%20the%20lactic%20acid%20bacteria.%20J.%20Gen.%20Microbiol.35,%2013%E2%80%9326%20(1964)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Widawati, S. S. Isolation &amp; identification of phosphate solubilizing &amp; nitrogen fixing bacteria from soil in Wamena biological garden, Jayawijaya, Papua. <em>Biodiversitas</em><strong>6</strong>(5), 175–177 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Widawati,%20S.%20S.%20Isolation%20&amp;%20identification%20of%20phosphate%20solubilizing%20&amp;%20nitrogen%20fixing%20bacteria%20from%20soil%20in%20Wamena%20biological%20garden,%20Jayawijaya,%20Papua.%20Biodiversitas6(5),%20175%E2%80%93177%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Zouaoui, S. N., Essghaier, B., Hannachi, I., Hajlaoui, M. R. &amp; Boudabous, A. First report on the use of moderately halophilic bacteria against stem canker of greenhouse tomatoes caused by <em>Botrytis cinerea</em>. <em>Ann. Microbiol.</em><strong>57</strong>(3), 337–339 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zouaoui,%20S.%20N.,%20Essghaier,%20B.,%20Hannachi,%20I.,%20Hajlaoui,%20M.%20R.%20&amp;%20Boudabous,%20A.%20First%20report%20on%20the%20use%20of%20moderately%20halophilic%20bacteria%20against%20stem%20canker%20of%20greenhouse%20tomatoes%20caused%20by%20Botrytis%20cinerea.%20Ann.%20Microbiol.57(3),%20337%E2%80%93339%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Sequence data that support the findings of this study have been deposited in NCBI with the primary accession code <a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442496" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442496</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442497" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442497</a>, &amp;<a href="https://www.ncbi.nlm.nih.gov/nuccore/OP442498" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP442498</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-12974-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_12974.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370916/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370916/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370916%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370916/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370916/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370916/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841562/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370916/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841562/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370916/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370916/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="h0Olpw5HraRh9l5gZW7SvSXrmGrYYhjKeRZs8rlMfDZXjyl3sDdr1bzCbOshOp0U">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
