
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Modulating Antimicrobial Activity and Structure of the Peptide Esc(1‐21) via Site‐Specific Isopeptide Bond Formation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5310448AEEFB13051044003118BB64.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328261/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Peptide Science">
<meta name="citation_title" content="Modulating Antimicrobial Activity and Structure of the Peptide Esc(1‐21) via Site‐Specific Isopeptide Bond Formation">
<meta name="citation_author" content="Bruno Casciaro">
<meta name="citation_author_institution" content="Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy">
<meta name="citation_author" content="Daniel Ben Hur">
<meta name="citation_author_institution" content="Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_author" content="Daniela Roversi">
<meta name="citation_author_institution" content="Department of Chemical Science and Technologies, University of Rome Tor Vergata, Rome, Italy">
<meta name="citation_author" content="Carlo Vetrano">
<meta name="citation_author_institution" content="Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy">
<meta name="citation_author" content="Edo Kiper">
<meta name="citation_author_institution" content="Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_author" content="Giacomo Cappella">
<meta name="citation_author_institution" content="Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy">
<meta name="citation_author" content="Federico Carneri">
<meta name="citation_author_institution" content="Department of Chemical Science and Technologies, University of Rome Tor Vergata, Rome, Italy">
<meta name="citation_author_institution" content="Photoinduced Processes and Technologies Doctoral School, Department of Chemistry, Biology and Biotechnology, Perugia University, Perugia, Italy">
<meta name="citation_author" content="Eeva Tortellini">
<meta name="citation_author_institution" content="Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy">
<meta name="citation_author" content="Lorenzo Stella">
<meta name="citation_author_institution" content="Department of Chemical Science and Technologies, University of Rome Tor Vergata, Rome, Italy">
<meta name="citation_author" content="Neta Regev‐Rudzki">
<meta name="citation_author_institution" content="Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_author" content="Yechiel Shai">
<meta name="citation_author_institution" content="Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_author" content="Maria Luisa Mangoni">
<meta name="citation_author_institution" content="Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy">
<meta name="citation_publication_date" content="2025 Aug 6">
<meta name="citation_volume" content="31">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70048">
<meta name="citation_doi" content="10.1002/psc.70048">
<meta name="citation_pmid" content="40769954">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328261/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328261/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328261/pdf/PSC-31-e70048.pdf">
<meta name="description" content="Antimicrobial peptides (AMPs) represent valid alternatives to conventional antibiotics primarily due to their mechanism of action, which consists of cytoplasmic membrane disruption. However, their clinical application is often limited by ...">
<meta name="og:title" content="Modulating Antimicrobial Activity and Structure of the Peptide Esc(1‐21) via Site‐Specific Isopeptide Bond Formation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Antimicrobial peptides (AMPs) represent valid alternatives to conventional antibiotics primarily due to their mechanism of action, which consists of cytoplasmic membrane disruption. However, their clinical application is often limited by ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328261/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12328261">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/psc.70048"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PSC-31-e70048.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12328261%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12328261/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12328261/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328261/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/psc.70048" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Pept Sci</button></div>. 2025 Aug 6;31(9):e70048. doi: <a href="https://doi.org/10.1002/psc.70048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/psc.70048</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Pept%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Pept%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Pept%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Pept%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Modulating Antimicrobial Activity and Structure of the Peptide Esc(1‐21) via Site‐Specific Isopeptide Bond Formation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Casciaro%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Bruno Casciaro</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Bruno Casciaro</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Casciaro%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bruno Casciaro</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ben%C2%A0Hur%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Daniel Ben Hur</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Daniel Ben Hur</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ben%C2%A0Hur%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Ben Hur</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roversi%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Daniela Roversi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Daniela Roversi</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemical Science and Technologies, University of Rome Tor Vergata, Rome, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roversi%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniela Roversi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vetrano%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Carlo Vetrano</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Carlo Vetrano</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vetrano%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carlo Vetrano</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiper%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Edo Kiper</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Edo Kiper</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiper%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Edo Kiper</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cappella%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Giacomo Cappella</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Giacomo Cappella</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cappella%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Giacomo Cappella</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carneri%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Federico Carneri</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Federico Carneri</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemical Science and Technologies, University of Rome Tor Vergata, Rome, Italy</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Photoinduced Processes and Technologies Doctoral School, Department of Chemistry, Biology and Biotechnology, Perugia University, Perugia, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carneri%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Federico Carneri</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tortellini%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Eeva Tortellini</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Eeva Tortellini</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tortellini%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eeva Tortellini</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stella%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lorenzo Stella</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lorenzo Stella</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemical Science and Technologies, University of Rome Tor Vergata, Rome, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stella%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lorenzo Stella</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Regev%E2%80%90Rudzki%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Neta Regev‐Rudzki</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Neta Regev‐Rudzki</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Regev%E2%80%90Rudzki%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Neta Regev‐Rudzki</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shai%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yechiel Shai</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yechiel Shai</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shai%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yechiel Shai</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mangoni%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Maria Luisa Mangoni</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Maria Luisa Mangoni</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mangoni%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria Luisa Mangoni</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="psc70048-aff-0001">
<sup>
<sup>1</sup>
</sup>Laboratory Affiliated to Pasteur Italia‐Fondazione Cenci Bolognetti, Department of Biochemical Sciences, Sapienza University of Rome, Rome, Italy</div>
<div id="psc70048-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Biomolecular Sciences, Faculty of Biochemistry, Weizmann Institute of Science, Rehovot, Israel</div>
<div id="psc70048-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Chemical Science and Technologies, University of Rome Tor Vergata, Rome, Italy</div>
<div id="psc70048-aff-0004">
<sup>
<sup>4</sup>
</sup>Photoinduced Processes and Technologies Doctoral School, Department of Chemistry, Biology and Biotechnology, Perugia University, Perugia, Italy</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Bruno Casciaro (<span>bruno.casciaro@uniroma1.it</span>), 
Yechiel Shai (<span>yechiel.shai@weizmann.ac.il</span>), 
Maria Luisa Mangoni (<span>marialuisa.mangoni@uniroma1.it</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 18; Received 2025 May 7; Accepted 2025 Jul 23; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Journal of Peptide Science</em> published by European Peptide Society and John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12328261  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40769954/" class="usa-link">40769954</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Antimicrobial peptides (AMPs) represent valid alternatives to conventional antibiotics primarily due to their mechanism of action, which consists of cytoplasmic membrane disruption. However, their clinical application is often limited by cytotoxicity at high concentrations and low intrinsic biostability. To address these limitations, various biochemical approaches have been explored. In recent years, the frog‐skin derived AMP Esc(1‐21) has been extensively characterized for its potent antimicrobial activity, especially against Gram‐negative bacteria, both in vitro and in vivo. In this study, we designed and synthesized novel Esc(1‐21) analogs in which a single isopeptide bond was introduced in place of a conventional peptide bond at specific positions within the sequence. The resulting five analogs were evaluated for their (i) chemical and structural properties, (ii) resistance to proteolytic degradation, (iii) antimicrobial and antibiofilm activities, (iv) hemolytic and cytotoxic effects, and (v) ability to perturb bacterial cytoplasmic membranes. Among these, Esc(1‐21)ε20 showed the most promising features, maintaining antimicrobial and antibiofilm activities comparable to those of the parent peptide while exhibiting lower cytotoxicity towards eukaryotic cells at higher concentrations and greater resistance to enzymatic degradation. These findings highlight Esc(1‐21)ε20 as an attractive lead candidate for the development of new antibiotic therapeutics.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>The introduction of an isopeptide bond at the lysine residues in the backbone of Esc(1‐21) led to the generation of five analogs. Among these, Esc(1‐21)ε20 showed the most promising features, having antimicrobial activity comparable with those of the parent peptide, a lower cytotoxicity, and a higher stability to proteolytic degradation.</p>
<section class="bt xbox font-sm" id="psc70048-blkfxd-0001"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/8619386dc669/PSC-31-e70048-g008.jpg" loading="lazy" id="jats-graphic-1" height="271" width="394" alt="graphic file with name PSC-31-e70048-g008.jpg"></p></section></section><section id="psc70048-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Bioactive peptides are a class of compounds consisting of short amino acid sequences, capable of performing various physiological functions, including the regulation of blood pressure, immune activity, glucose metabolism, protection against oxidative stress, and, last but not least, antimicrobial activity [<a href="#psc70048-bib-0001" class="usa-link" aria-describedby="psc70048-bib-0001">1</a>]. Among these bioactive compounds, antimicrobial peptides (AMPs) hold great promise in the current post‐antibiotic era, during which only a small number of new antibiotics are being introduced to the market [<a href="#psc70048-bib-0002" class="usa-link" aria-describedby="psc70048-bib-0002">2</a>]. AMPs are gene‐encoded cationic molecules, typically composed of about 10–50 amino acids and with various amphipathic secondary structures [<a href="#psc70048-bib-0003" class="usa-link" aria-describedby="psc70048-bib-0003">3</a>]. Their cationicity and amphipathicity are considered key physicochemical properties that make AMPs active against biological membranes [<a href="#psc70048-bib-0004" class="usa-link" aria-describedby="psc70048-bib-0004">4</a>], particularly those of bacteria, which are rich in anionic phospholipids [<a href="#psc70048-bib-0005" class="usa-link" aria-describedby="psc70048-bib-0005">5</a>]. Through a mechanism involving membrane destruction by pore formation or micellization, AMPs represent a valid alternative to conventional antibiotics, which generally act via specific molecular targets prone to mutation [<a href="#psc70048-bib-0006" class="usa-link" aria-describedby="psc70048-bib-0006">6</a>, <a href="#psc70048-bib-0007" class="usa-link" aria-describedby="psc70048-bib-0007">7</a>, <a href="#psc70048-bib-0008" class="usa-link" aria-describedby="psc70048-bib-0008">8</a>]. In recent years, a frog‐skin derived AMP, named Esculentin‐1a(1‐21)NH<sub>2</sub> [Esc(1‐21)], has been characterized for its potent antimicrobial activity, especially against Gram‐negative bacteria, both in vitro and in vivo [<a href="#psc70048-bib-0009" class="usa-link" aria-describedby="psc70048-bib-0009">9</a>, <a href="#psc70048-bib-0010" class="usa-link" aria-describedby="psc70048-bib-0010">10</a>, <a href="#psc70048-bib-0011" class="usa-link" aria-describedby="psc70048-bib-0011">11</a>]. Furthermore, several chemical modification strategies have been adopted to broaden the spectrum of action of the peptide (e.g., against Gram‐positive bacteria), to increase its resistance to proteolytic degradation, and to reduce its cytotoxicity at high concentrations. These include the replacement of two <span class="font-variant-small-caps">l</span>‐residues with their corresponding <span class="font-variant-small-caps">d</span>‐enantiomers [<a href="#psc70048-bib-0012" class="usa-link" aria-describedby="psc70048-bib-0012">12</a>]; the replacement of certain residues with the nonproteinogenic α‐aminoisobutyric acid (Aib) at positions 1, 10, and 18 [<a href="#psc70048-bib-0013" class="usa-link" aria-describedby="psc70048-bib-0013">13</a>], as well as the strategic position 8 [<a href="#psc70048-bib-0014" class="usa-link" aria-describedby="psc70048-bib-0014">14</a>]. In this work, we adopted a different approach by synthesizing Esc(1‐21) analogs in which the typical peptide bond between two residues was replaced with an isopeptide bond. Isopeptide bond formation involves lysine residues forming covalent linkages via their ε‐amino groups. Generally, this noncanonical bond enhances resistance to proteolytic degradation, stabilizes tertiary and quaternary structures, and facilitates the covalent attachment of proteins to biological surfaces [<a href="#psc70048-bib-0015" class="usa-link" aria-describedby="psc70048-bib-0015">15</a>, <a href="#psc70048-bib-0016" class="usa-link" aria-describedby="psc70048-bib-0016">16</a>]. Naturally occurring isopeptide bonds have been identified in systems such as the HK97 bacteriophage capsid, Gram‐positive bacterial pili, and the ubiquitin‐proteasome pathway in eukaryotic cells [<a href="#psc70048-bib-0017" class="usa-link" aria-describedby="psc70048-bib-0017">17</a>, <a href="#psc70048-bib-0018" class="usa-link" aria-describedby="psc70048-bib-0018">18</a>, <a href="#psc70048-bib-0019" class="usa-link" aria-describedby="psc70048-bib-0019">19</a>]. In these contexts, isopeptide bonds are typically formed between separate protein domains or subunits. In contrast, our work explores the direct incorporation of this noncanonical bond within the backbone of the AMP sequence, representing an innovative application of isopeptide bond chemistry in peptide engineering. In fact, to the best of our knowledge, very few studies have explored isopeptide bond incorporation in AMPs [<a href="#psc70048-bib-0020" class="usa-link" aria-describedby="psc70048-bib-0020">20</a>, <a href="#psc70048-bib-0021" class="usa-link" aria-describedby="psc70048-bib-0021">21</a>, <a href="#psc70048-bib-0022" class="usa-link" aria-describedby="psc70048-bib-0022">22</a>, <a href="#psc70048-bib-0023" class="usa-link" aria-describedby="psc70048-bib-0023">23</a>], confirming the intriguing strategy of this biochemical approach. Considering the presence of five lysine residues in the primary structure of Esc(1‐21), five peptides with site‐specific isopeptide bond switches have been synthesized and characterized for their (i) chemical properties, (ii) structural properties, (iii) stability to proteolytic degradation, (iv) antimicrobial and antibiofilm activity, (v) hemolytic and cytotoxic effect, and (vi) ability to perturb bacterial cytoplasmic membranes.</p></section><section id="psc70048-sec-0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="psc70048-sec-0003"><h3 class="pmc_sec_title">2.1. Materials</h3>
<p>9‐Fluorenylmethoxylcarbonyl (Fmoc)‐rink amide methyl‐benzhydryl amine (MBHA) resin, ethyl cyanohydroxyiminoacetate (Oxyma) and Fmoc amino acids were purchased from Calibochem/Novabiochem AG (Laufelfinger, Switzerland). Acetonitrile (ACN), methanol, <em>N</em>,<em>N</em>‐dimethylformamide (DMF), dichloromethane (DCM), diethyl ether (Et<sub>2</sub>O), piperidine, and trifluoroacetic acid (TFA) were provided by Bio‐Lab Ltd, (Israel). <em>N</em>,<em>N</em>′‐Diisopropylcarbodiimide (DIC) and triethylsilane (TES) were purchased from Sigma‐Aldrich (Israel). Sodium‐dodecylsulfate (SDS) was obtained from Cambridge Isotope Laboratories (Tewksbury, MA, USA); 3‐(4,5‐dimethylthiazol‐2‐yl)‐2,5‐diphenyltetrazolium bromide (MTT) was supplied by Merck (Darmstadt, Germany) and Cell Counting Kit 8 (CCK‐8) was purchased from MedChemExpress (Monmouth Junction, NJ, USA).</p></section><section id="psc70048-sec-0004"><h3 class="pmc_sec_title">2.2. Peptide Synthesis</h3>
<p>Peptides were synthesized using an automated microwave‐assisted solid‐phase peptide synthesizer (Liberty Blue, CEM Corporation) on 0.57 mmol/g MBHA resin, using the Fmoc solid‐phase strategy [<a href="#psc70048-bib-0024" class="usa-link" aria-describedby="psc70048-bib-0024">24</a>, <a href="#psc70048-bib-0025" class="usa-link" aria-describedby="psc70048-bib-0025">25</a>]. Isopeptide bond formation was achieved by incorporating Boc‐L‐Lys(Fmoc)‐OH, enabling coupling through the ε‐amino group of lysine, in contrast to the conventional Fmoc‐L‐Lys(Boc)‐OH used for α‐peptide bond formation. The resin‐bound peptide was washed thoroughly with DMF, then with DCM, and dried under nitrogen flow. Cleavage of the peptide from the resin was done by adding 95% TFA, 2.5% ddH<sub>2</sub>O, and 2.5% TES. The resin was filtered from the mixture, and the peptide was precipitated from the mixture using cold Et<sub>2</sub>O.</p></section><section id="psc70048-sec-0005"><h3 class="pmc_sec_title">2.3. Peptide Purification and Characterization</h3>
<p>The purification of the peptides was performed by reverse‐phase high‐performance liquid chromatography (RP‐HPLC) on an Agilent Technologies 1260 Infinity II spectrometer with a reversed‐phase Vydac C<sub>4</sub> column at a flow rate of 1.8 mL/min and monitored with an ultraviolet (UV) detector at 215 nm. Linear gradients of 10%–90% ACN in ddH<sub>2</sub>O containing 0.1% (v/v) TFA were used for peptide purification for 40 min. Final products were obtained by freeze‐drying the collected pure fractions. The purity of the peptides was validated with a C18 reversed‐phase column (Thermo Fisher Scientific, 250 mm × 4.6 mm, 5 μm particle size) at a flow rate of 0.6 mL/min using a gradient of 10%–90% ACN in ddH<sub>2</sub>O containing 0.1% (v/v) TFA for 40 min with UV detection at 215 nm. Incorporation of the isopeptide bond was verified as done previously [<a href="#psc70048-bib-0022" class="usa-link" aria-describedby="psc70048-bib-0022">22</a>], based on differences in hydrophobicity compared between the peptide parental and analogs, resulting in distinct retention times under identical conditions. Molecular weights were determined by time‐of‐flight mass spectrometry (TOF‐MS), confirming the expected mass and structural integrity of each peptide. The purity of all peptides examined was &gt; 95%.</p></section><section id="psc70048-sec-0006"><h3 class="pmc_sec_title">2.4. Microbial and Eukaryotic Cells</h3>
<p>The following strains from American Type Culture Collection (ATCC) were used in this study: the Gram‐negatives 
<em>Escherichia coli</em>
 ATCC 25922, 
<em>Pseudomonas aeruginosa</em>
 ATCC 27853, 
<em>Pseudomonas aeruginosa</em>
 ATCC 15692; and the Gram‐positives 
<em>Staphylococcus epidermidis</em>
 ATCC 12228 and 
<em>Staphylococcus aureus</em>
 ATCC 25923. For the cytotoxicity assay, HaCaT cells (AddexBio San Diego, CA, United States) were used: the cells were grown in Dulbecco's modified Eagle's medium supplemented with 4 mM glutamine (DMEMg), 10% heat‐inactivated fetal bovine serum (FBS), and 0.1 mg/mL penicillin and streptomycin. The cell culture was maintained in a humidified incubator at 37°C and 5% CO<sub>2</sub>.</p></section><section id="psc70048-sec-0007"><h3 class="pmc_sec_title">2.5. Antimicrobial and Antibiofilm Activity</h3>
<p>The antimicrobial activity of Esc(1‐21) analogs was assessed by determining the minimal inhibitory concentration (MIC) against a panel of representative Gram‐negative and Gram‐positive bacteria. Bacteria were grown at 37°C until an optical density (OD) of 0.8 was reached at 590 nm. The bacteria were then diluted in Mueller‐Hinton (MH) broth at a concentration of 2 × 10<sup>6</sup> colony‐forming units (CFU)/mL. Aliquots (50 μL) were added to 50 μL of MH containing serial two‐fold dilutions of the peptides, which had been previously prepared in the wells of a 96‐well plate, achieving a final cell concentration of 10<sup>6</sup> CFU/mL. The plate was incubated at 37°C for 16–18 h and the MIC was defined as the lowest concentration capable of visually inhibiting microbial growth. To evaluate the antibiofilm activity of the tested peptides, bacteria were diluted to a cell density of 10<sup>6</sup> CFU/mL and aliquots of 100 μL were dispensed into the wells of a 96‐multiwell plate and incubated for 20 h at 37°C to allow biofilm formation. Planktonic cells were then removed, and each well was washed twice with 150 μL of phosphate buffered saline (PBS). Each well was filled with PBS supplemented with different 2‐fold serial dilutions of the tested peptides and the plate was incubated for 2 h at 37°C. After two washes with PBS, biofilm viability was assessed by adding 150 μL of MTT (0.5 mg/mL). The plate was incubated for 4 h at 37°C and the reaction was then stopped by adding SDS (final concentration equal to 5% v/v). The absorbance of each well was measured at 570 nm with a microplate reader (Infinite M200; Tecan, Salzburg, Austria), and the percentage of biofilm viability was calculated with respect to the untreated samples.</p></section><section id="psc70048-sec-0008"><h3 class="pmc_sec_title">2.6. Membrane Permeabilization Assay</h3>
<p>To investigate the potential of Esc(1‐21) analogs to permeabilize cytoplasmic membranes of Gram‐positive and Gram‐negative bacteria, a Sytox Green assay was performed using 
<em>E. coli</em>
 ATCC 25922 and 
<em>S. epidermidis</em>
 ATCC 12228 as representative strains. Bacterial suspensions were prepared at approximately 10<sup>7</sup> CFU/mL in PBS and incubated with 1 μM Sytox Green for 5 min in the dark. After that, peptides were added at concentrations ranging from 0.78 to 50 μM (time = 0). Fluorescence intensity (λ<sub>exc</sub> = 485 nm, λ<sub>ems</sub> = 535 nm), which reflects the binding of the dye to intracellular DNA after membrane permeabilization, was monitored over 30 min using a microplate reader (Infinite M200, Tecan, Salzburg, Austria) at 37°C. Experiments were performed in triplicate. Bacterial cells treated with vehicle (water) were used as a control.</p></section><section id="psc70048-sec-0009"><h3 class="pmc_sec_title">2.7. Hemolytic Assay</h3>
<p>The hemolytic activity of Esc(1‐21) analogs was assessed by quantifying the release of hemoglobin resulting from the disruption of defibrinated sheep red blood cells (RBCs) (OXOID, SR0051D, Milan, Italy). RBCs were washed twice in 0.9% (w/v) NaCl. Aliquots of red blood cell suspension were adjusted to an OD of 0.5 at 500 nm in 0.9% (w/v) NaCl and incubated with peptide solutions at concentrations ranging from 0.78 to 50 μM for 30 min at 37°C with gentle agitation. After centrifugation, the samples were centrifuged at 900 × <em>g</em> for 5 min, and hemoglobin released in the supernatant was quantified by measuring absorbance at 415 nm using a microplate reader (Infinite M200; Tecan, Salzburg, Austria) and compared with the complete lysis of erythrocytes (100%) in ddH<sub>2</sub>O. Cells treated with vehicle (ddH<sub>2</sub>O) served as the negative control. The data were reported as the mean ± the standard error of the mean (SEM) of three independent experiments.</p></section><section id="psc70048-sec-0010"><h3 class="pmc_sec_title">2.8. Cell Viability Assay</h3>
<p>The viability of HaCaT cells was assessed using the CCK‐8 assay as previously reported [<a href="#psc70048-bib-0009" class="usa-link" aria-describedby="psc70048-bib-0009">9</a>, <a href="#psc70048-bib-0026" class="usa-link" aria-describedby="psc70048-bib-0026">26</a>]. Briefly, 100 μL of cell suspensions (10<sup>4</sup> cells) were added to each well of a 96‐well plate and incubated for approximately 24 h at 37°C and 5% CO<sub>2</sub>. Afterward, the medium was replaced by 100 μL of DMEMg, containing Esc(1‐21), Esc(1‐21)ε12, and Esc(1‐21)ε20, at two different high concentrations, that is, 100 and 200 μM. The plate was incubated for 24 h, and afterward, the CCK‐8 reagent was added to each well (1:10 dilution in DMEMg), and after 90 min of incubation, the production of orange‐colored formazan dye, related to the number of living cells, was measured by evaluating the absorbance of each well at 450 nm using the microplate reader (Infinite M200; Tecan, Salzburg, Austria). The cell viability was expressed as a percentage with respect to the untreated control cells.</p></section><section id="psc70048-sec-0011"><h3 class="pmc_sec_title">2.9. Plasma Stability Testing</h3>
<p>Peptides were prepared as a 1 mM solution in PBS and diluted in plasma (20% v/v) that was provided by Magen David Adom Blood Donations (IRB #SMC‐13‐0954) and separated from human red blood via centrifugation to the final volume 100 μL. The mixture was incubated at 37°C for various time points. To stop further degradation, 100 μL of a stopping solution (80% ACN, 10% methanol, and 10% ddH<sub>2</sub>O) was added. This caused the solution to become cloudy, which was then cooled to 4°C for 1 h and centrifuged at 10,000 g for 10 min to remove plasma proteins. The supernatant (10 μL) was analyzed using an RP‐HPLC system with a Vydac C<sub>18</sub> (250 × 4.6 mm, 5 μm particle size) column (Grace Discovery Sciences, Deerfield, I). The column was eluted over 30 min using a linear gradient of 20%–80% ACN in ddH<sub>2</sub>O containing 0.1% TFA at a flow rate of 0.6 mL/min. Absorbance was detected at 215 nm, and the decrease in the chromatographic peak area of the untreated peptide determined the percentage of the remaining peptide.</p></section><section id="psc70048-sec-0012"><h3 class="pmc_sec_title">2.10. Large Unilamellar Vesicles Preparation</h3>
<p>Lipid films of PEG‐PE/POPE/POPG were prepared by dissolving lipids (POPE/POPG mixture, 7:3 mol/mol + 2% PEG‐PE) in chloroform/methanol (1:1, v/v). Solvents were evaporated in a rotary vacuum system until a thin film was formed. Complete evaporation of the organic solvent was ensured by applying a rotary vacuum pump for at least 2 h. The lipid film was then hydrated with a phosphate buffer (10 mM phosphate, 140 mM NaF, 0.1 mM EDTA, pH 7.4). After 10 freeze and thaw cycles, the liposome solution was extruded 31 times by two stacked polycarbonate membranes with 50 nm pores to obtain unilamellar large vesicles (LUVs). The final lipid concentration was measured by the Stewart phospholipid assay [<a href="#psc70048-bib-0027" class="usa-link" aria-describedby="psc70048-bib-0027">27</a>].</p></section><section id="psc70048-sec-0013"><h3 class="pmc_sec_title">2.11. Circular Dichroism Spectroscopy</h3>
<p>Circular dichroism (CD) experiments were carried out using a Jasco J‐1500 spectrometer (Jasco International Co. Ltd., Tokyo, Japan). CD spectra were measured for each peptide at a concentration of 20 μM in buffer. LUVs were added to have a final lipid concentration of 50, 100, 200, 500, and 1000 μM. CD spectra were scanned over a range of 190–260 nm with a 1 nm data interval, scan speed 20 nm/min, bandwidth 2 nm, and averaged over 4 scans. Blank sample spectra were subtracted from the raw data, and the CD values were converted to per residue molar ellipticity([θ]) (deg cm<sup>2</sup> dmol<sup>−1</sup>). The secondary structure content was calculated with BeStSel [<a href="#psc70048-bib-0028" class="usa-link" aria-describedby="psc70048-bib-0028">28</a>].</p></section><section id="psc70048-sec-0014"><h3 class="pmc_sec_title">2.12. Ethics Statement</h3>
<p>Human sample studies were carried out in strict accordance with Israeli law. All samples for experiments were conducted at the Weizmann Institute of Science and approved by Israel Institutional Review Board #SMC‐13‐0954.</p></section></section><section id="psc70048-sec-0015"><h2 class="pmc_sec_title">3. Results</h2>
<section id="psc70048-sec-0016"><h3 class="pmc_sec_title">3.1. Esc(1‐21) Analogs: Synthesis and Chemical Properties</h3>
<p>Esc(1‐21) and its analogs containing a single isopeptide bond were synthesized by an automated peptide solid‐phase synthesizer. Crude peptides were purified by RP‐HPLC, and their identity and purity were confirmed by TOF‐MS (Figure <a href="#psc70048-supitem-0001" class="usa-link">S1A</a>) and analytical RP‐HPLC using a C<sub>18</sub> column (Figure <a href="#psc70048-supitem-0001" class="usa-link">S1B</a>), as detailed in Section <a href="#psc70048-sec-0002" class="usa-link">2</a>. All peptides were 21 residues long, with a molecular weight of 2184.4 g/mol and had a net charge of +6 at neutral pH, due to the presence of the five lysine residues and an amidated C‐terminus. The primary structures and the main chemical information, including delta epsilon (Δε) from the center, retention time, and relative hydrophobicity, are reported in Table <a href="#psc70048-tbl-0001" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="psc70048-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Primary structures and chemical characteristics of Esc(1‐21) and its newly designed analogs.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Peptide</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Sequence<a href="#psc70048-note-0003" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Δε from center<a href="#psc70048-note-0004" class="usa-link">
<sup>b</sup>
</a>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Retention time<a href="#psc70048-note-0005" class="usa-link">
<sup>c</sup>
</a> (min)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Relative hydrophobicity<a href="#psc70048-note-0006" class="usa-link">
<sup>d</sup>
</a> (% ACN)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)</td>
<td align="center" valign="top" rowspan="1" colspan="1">GIFSKLAGKKIKNLLISGLKG</td>
<td align="center" valign="top" rowspan="1" colspan="1">NA</td>
<td align="center" valign="top" rowspan="1" colspan="1">22.66</td>
<td align="center" valign="top" rowspan="1" colspan="1">55.33</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)ε5</td>
<td align="center" valign="top" rowspan="1" colspan="1">GIFS<span class="text-underline">
<strong>K</strong>
</span>LAGKKIKNLLISGLKG</td>
<td align="center" valign="top" rowspan="1" colspan="1">6</td>
<td align="center" valign="top" rowspan="1" colspan="1">21.41</td>
<td align="center" valign="top" rowspan="1" colspan="1">52.83</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)ε9</td>
<td align="center" valign="top" rowspan="1" colspan="1">GIFSKLAG<span class="text-underline">
<strong>K</strong>
</span>KIKNLLISGLKG</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">21.65</td>
<td align="center" valign="top" rowspan="1" colspan="1">53.31</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)ε10</td>
<td align="center" valign="top" rowspan="1" colspan="1">GIFSKLAGK<span class="text-underline">
<strong>K</strong>
</span>IKNLLISGLKG</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">21.59</td>
<td align="center" valign="top" rowspan="1" colspan="1">53.18</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)ε12</td>
<td align="center" valign="top" rowspan="1" colspan="1">GIFSKLAGKKI<span class="text-underline">
<strong>K</strong>
</span>NLLISGLKG</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">20.99</td>
<td align="center" valign="top" rowspan="1" colspan="1">51.98</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)ε20</td>
<td align="center" valign="top" rowspan="1" colspan="1">GIFSKLAGKKIKNLLISGL<span class="text-underline">
<strong>K</strong>
</span>G</td>
<td align="center" valign="top" rowspan="1" colspan="1">9</td>
<td align="center" valign="top" rowspan="1" colspan="1">22.80</td>
<td align="center" valign="top" rowspan="1" colspan="1">55.61</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/psc70048-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="psc70048-note-0003">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Underlined and bold residues are involved in isopeptide bond formation. All the peptides were amidated at their C‐terminus.</p>
</div>
<div class="fn" id="psc70048-note-0004">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Delta epsilon (Δε) from the center is defined as the distance between the isopeptide bond position and the center of the peptide sequence.</p>
</div>
<div class="fn" id="psc70048-note-0005">
<sup>
<sup>c</sup>
</sup><p class="display-inline">Relative hydrophobicity is reflected by the percentage of ACN at the retention time.</p>
</div>
<div class="fn" id="psc70048-note-0006">
<sup>
<sup>d</sup>
</sup><p class="display-inline">The reversed‐phase HPLC retention time in the C<sub>18</sub> column using a gradient of 10%–90% ACN in ddH<sub>2</sub>O for 40 min.</p>
</div>
</div></section><p>The chemical structure of Esc(1‐21) and the site‐specific introduction of the isopeptide bond in each designed analog are shown in Figure <a href="#psc70048-fig-0001" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="psc70048-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/4aa4bb5491a0/PSC-31-e70048-g005.jpg" loading="lazy" id="jats-graphic-3" height="714" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/psc70048-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Chemical structures of Esc(1‐21) peptide and its isopeptide bond analogs. Amino acid residues in black are L‐amino acids in α‐peptide bond. Amino acid residues involved in isopeptide bond formation are highlighted in red. In the analogs, lysine residues form covalent linkages via their ε‐amino groups to the carboxyl group of the adjacent amino acid, resulting in isopeptide bonds. (A) Esc(1‐21); (B) Esc(1‐21)ε5; (C) Esc(1‐21)ε9; (D) Esc(1‐21)ε10; (E) Esc(1‐21)ε12; (F) Esc(1‐21)ε20. The chemical structures of the peptides were generated using ChemDraw software.</p></figcaption></figure></section><section id="psc70048-sec-0017"><h3 class="pmc_sec_title">3.2. Biological Characterization</h3>
<section id="psc70048-sec-0018"><h4 class="pmc_sec_title">3.2.1. Position‐Dependent Effects of Isopeptide Bond Incorporation on Antimicrobial Activity</h4>
<p>The antimicrobial activity of all synthesized peptides was evaluated against a panel of Gram‐positive and Gram‐negative bacteria to determine the MIC, as reported in Table <a href="#psc70048-tbl-0002" class="usa-link">2</a>. The peptides were tested starting from 0.78 to 50 μM, and MIC values were compared with those already published for Esc(1‐21).</p>
<section class="tw xbox font-sm" id="psc70048-tbl-0002"><h5 class="obj_head">TABLE 2.</h5>
<div class="caption p"><p>Antimicrobial activity of Esc(1‐21) and its isopeptide bond analogs.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th rowspan="2" style="border-bottom:solid 1px #000000" align="left" valign="bottom" colspan="1">Strain</th>
<th align="center" colspan="6" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1">Antimicrobial activity (μM)<a href="#psc70048-note-0007" class="usa-link">
<sup>a</sup>
</a>
</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">Esc(1‐21)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Esc(1‐21)ε5</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Esc(1‐21)ε9</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Esc(1‐21)ε10</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Esc(1‐21)ε12</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Esc(1‐21)ε20</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>Gram‐negative</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td style="padding-left:10%" align="left" valign="top" rowspan="1" colspan="1">

<em>E. coli</em>
 ATCC 25922</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.56<a href="#psc70048-note-0008" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.12</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.12</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.25</td>
<td align="center" valign="top" rowspan="1" colspan="1">25</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.56</td>
</tr>
<tr>
<td style="padding-left:10%" align="left" valign="top" rowspan="1" colspan="1">

<em>P. aeruginosa</em>
 ATCC 27853</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.12<a href="#psc70048-note-0008" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">12.5</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.25</td>
<td align="center" valign="top" rowspan="1" colspan="1">12.5</td>
<td align="center" valign="top" rowspan="1" colspan="1">25</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.56</td>
</tr>
<tr>
<td style="padding-left:10%" align="left" valign="top" rowspan="1" colspan="1">

<em>P. aeruginosa</em>
 ATCC 15692</td>
<td align="center" valign="top" rowspan="1" colspan="1">2<a href="#psc70048-note-0009" class="usa-link">
<sup>c</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.25</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.25</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.25</td>
<td align="center" valign="top" rowspan="1" colspan="1">25</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.56</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>Gram‐positive</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td style="padding-left:10%" align="left" valign="top" rowspan="1" colspan="1">

<em>S. epidermidis</em>
 ATCC 12228</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.12<a href="#psc70048-note-0008" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">25</td>
<td align="center" valign="top" rowspan="1" colspan="1">12.5</td>
<td align="center" valign="top" rowspan="1" colspan="1">25</td>
<td align="center" valign="top" rowspan="1" colspan="1">&gt; 50</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.12</td>
</tr>
<tr>
<td style="padding-left:10%" align="left" valign="top" rowspan="1" colspan="1">

<em>S. aureus</em>
 ATCC 25923</td>
<td align="center" valign="top" rowspan="1" colspan="1">&gt; 100<a href="#psc70048-note-0008" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">&gt; 50</td>
<td align="center" valign="top" rowspan="1" colspan="1">&gt; 50</td>
<td align="center" valign="top" rowspan="1" colspan="1">&gt; 50</td>
<td align="center" valign="top" rowspan="1" colspan="1">&gt; 50</td>
<td align="center" valign="top" rowspan="1" colspan="1">&gt; 50</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/psc70048-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="psc70048-note-0007">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Values were obtained from at least two identical readings out of three independent experiments.</p>
</div>
<div class="fn" id="psc70048-note-0008">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Values taken from ref. [<a href="#psc70048-bib-0014" class="usa-link" aria-describedby="psc70048-bib-0014">14</a>].</p>
</div>
<div class="fn" id="psc70048-note-0009">
<sup>
<sup>c</sup>
</sup><p class="display-inline">Values taken from ref. [<a href="#psc70048-bib-0008" class="usa-link" aria-describedby="psc70048-bib-0008">8</a>].</p>
</div>
</div></section><p>As reported in Table <a href="#psc70048-tbl-0002" class="usa-link">2</a>, all isopeptide bond‐containing analogs were inactive against 
<em>S. aureus</em>
, consistent with previous reports for Esc(1‐21) [<a href="#psc70048-bib-0014" class="usa-link" aria-describedby="psc70048-bib-0014">14</a>]. Meanwhile, analogs containing the isopeptide bond in positions 5, 9, and 10 showed only a slight reduction in antimicrobial activity against all the tested strains. In contrast, the antimicrobial activity of Esc(1‐21)ε12 was notably impaired by the presence of the isopeptide bond, as reflected by increased MIC values (25 or &gt; 50 μM) against all strains. Conversely, Esc(1‐21)ε20 exhibited MIC values comparable to or slightly better than those of the parent peptide. This difference may be attributed to the position of the isopeptide bond, which, in Esc(1‐21)ε20, is closer to the C‐terminus, or possibly to changes in peptide hydrophobicity resulting from isopeptide bond formation. Specifically, Esc(1‐21)ε20 appears to have an increase in hydrophobicity that enhances its antimicrobial activity, while Esc(1‐21)ε12 likely experiences a reduction in hydrophobicity leading to lower activity (Table <a href="#psc70048-tbl-0001" class="usa-link">1</a>).</p></section><section id="psc70048-sec-0019"><h4 class="pmc_sec_title">3.2.2. Position of the Site‐Specific Isopeptide Bond Influences Membrane‐Disruptive Activity of Esc(1‐21) Analogs</h4>
<p>Perturbation of the cytoplasmic membrane has previously been identified as the principal mechanism underlying the antimicrobial activity of Esc(1‐21) against both Gram‐negative and Gram‐positive bacteria [<a href="#psc70048-bib-0014" class="usa-link" aria-describedby="psc70048-bib-0014">14</a>, <a href="#psc70048-bib-0029" class="usa-link" aria-describedby="psc70048-bib-0029">29</a>]. To verify whether the weaker activity observed in Table <a href="#psc70048-tbl-0002" class="usa-link">2</a> was related to a reduced capacity to perturb the cytoplasmic bacterial membranes, fluorescence studies were performed using the Sytox Green probe on two different bacterial strains, i.e. the Gram‐negative 
<em>E. coli</em>
 ATCC 25922 and the Gram‐positive 
<em>S. epidermidis</em>
 ATCC 12228. Membrane perturbation was assessed by measuring the increase in fluorescence during 30 min after peptide addition at different concentrations, ranging from 1.56 to 50 μM and results were compared with those of untreated control cells. Esc(1‐21)ε5, Esc(1‐21)ε9, and Esc(1‐21)ε10 provoked only a slight increase in fluorescence intensity against 
<em>E. coli</em>
 (Figure <a href="#psc70048-fig-0002" class="usa-link">2</a>). Moreover, they did not provoke any significant increase in the fluorescence intensity after peptide addition (time = 0) against 
<em>S. epidermidis</em>
 (Figure <a href="#psc70048-fig-0003" class="usa-link">3</a>). However, the most active analog, Esc(1‐21)ε20, induced a dose‐dependent membrane perturbation already from the first minutes of treatment, with kinetics comparable to those previously reported for Esc(1‐21) (Figures <a href="#psc70048-fig-0002" class="usa-link">2</a> and <a href="#psc70048-fig-0003" class="usa-link">3</a>) [<a href="#psc70048-bib-0014" class="usa-link" aria-describedby="psc70048-bib-0014">14</a>, <a href="#psc70048-bib-0029" class="usa-link" aria-describedby="psc70048-bib-0029">29</a>].</p>
<figure class="fig xbox font-sm" id="psc70048-fig-0002"><h5 class="obj_head">FIGURE 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/1f12994c99b5/PSC-31-e70048-g003.jpg" loading="lazy" id="jats-graphic-5" height="849" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/psc70048-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Kinetics of cytoplasmic membrane permeabilization of 
<em>E. coli</em>
 ATCC 25922 induced by the addition (time = 0) of Esc(1‐21)ε5, Esc(1‐21)ε9, Esc(1‐21)ε10, Esc(1‐21)ε12, and Esc(1‐21)ε20 at different concentrations (from 1.56 to 50 μM) evaluated by the Sytox Green assay. Controls (Ctrl) are microbial cells without the addition of any peptide. The values are from one representative experiment out of three.</p></figcaption></figure><figure class="fig xbox font-sm" id="psc70048-fig-0003"><h5 class="obj_head">FIGURE 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/2be5f25119d6/PSC-31-e70048-g002.jpg" loading="lazy" id="jats-graphic-7" height="854" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/psc70048-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Kinetics of cytoplasmic membrane permeabilization of 
<em>S. epidermidis</em>
 ATCC 12228 induced by the addition (time = 0) of Esc(1‐21)ε5, Esc(1‐21)ε9, Esc(1‐21)ε10, Esc(1‐21)ε12, and Esc(1‐21)ε20 at different concentrations (from 1.56 to 50 μM) evaluated by the Sytox Green assay. Controls (Ctrl) are microbial cells without the addition of any peptide. The reported values are from one representative experiment out of three.</p></figcaption></figure></section><section id="psc70048-sec-0020"><h4 class="pmc_sec_title">3.2.3. Esc(1‐21)ε20 Exhibits Potent, Dose‐Dependent Antibiofilm Activity</h4>
<p>Pathogenic bacteria such as <em>Staphylococcus</em> and <em>Pseudomonas</em> pose a serious threat to human health primarily due to their ability to transition from a planktonic form (individual, free‐floating bacteria in liquid environments) to a more complex and structured form, known as biofilm [<a href="#psc70048-bib-0030" class="usa-link" aria-describedby="psc70048-bib-0030">30</a>]. In these sessile communities, bacteria are embedded in a self‐produced extracellular matrix that protects them from antibiotic molecules and facilitates surface colonization [<a href="#psc70048-bib-0031" class="usa-link" aria-describedby="psc70048-bib-0031">31</a>]. Consequently, it is crucial for antimicrobial compounds to not only inhibit or kill planktonic bacteria but also effectively eradicate biofilms [<a href="#psc70048-bib-0032" class="usa-link" aria-describedby="psc70048-bib-0032">32</a>]. Esc(1‐21) has been extensively characterized for its capability to kill both planktonic and biofilm cells [<a href="#psc70048-bib-0009" class="usa-link" aria-describedby="psc70048-bib-0009">9</a>, <a href="#psc70048-bib-0014" class="usa-link" aria-describedby="psc70048-bib-0014">14</a>, <a href="#psc70048-bib-0029" class="usa-link" aria-describedby="psc70048-bib-0029">29</a>], and for this reason, we evaluated the antibiofilm potency of the most active Esc(1‐21) analog i.e., Esc(1‐21)ε20 in comparison with the less active Esc(1‐21)ε12. Both peptides were tested at different concentrations against a representative Gram‐negative bacterium (i.e., 
<em>P. aeruginosa</em>
 ATCC 15692, Figure <a href="#psc70048-fig-0004" class="usa-link">4A</a>) and a Gram‐positive bacterium (i.e., 
<em>S. epidermidis</em>
 ATCC 12228, Figure <a href="#psc70048-fig-0004" class="usa-link">4B</a>).</p>
<figure class="fig xbox font-sm" id="psc70048-fig-0004"><h5 class="obj_head">FIGURE 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/e66c8b6e7607/PSC-31-e70048-g007.jpg" loading="lazy" id="jats-graphic-9" height="223" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/psc70048-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Activity of Esc(1‐21)ε12 and Esc(1‐21)ε20 against 20 h‐preformed biofilm of 
<em>P. aeruginosa</em>
 ATCC 15692 (A) and 
<em>S. epidermidis</em>
 ATCC 12228 (B) after 2 h of treatment. Biofilm viability was evaluated by measuring the reduction of the yellow MTT to its purple insoluble formazan and expressed as a percentage compared to untreated samples (bacterial biofilm not treated with the peptide, 100% viability). Data are the mean ± standard error of the mean (SEM) of three independent experiments. Statistical analysis was conducted using the two‐way ANOVA to determine the significance between the two peptides. *<em>p</em> &lt; 0.05; ***<em>p</em> &lt; 0.001; ****<em>p</em> &lt; 0.0001; ns, not significant.</p></figcaption></figure><p>Against 
<em>P. aeruginosa</em>
, neither peptide exhibited significant activity at low concentrations (i.e., 1.56–12.55 μM). However, Esc(1‐21)ε20 showed a dose‐dependent antibiofilm activity at concentrations of 25 and 50 μM with a reduction in biofilm viability of about 60% and 80%, respectively. In contrast, Esc(1‐21)ε12 remained inactive at all concentrations against 
<em>P. aeruginosa</em>
 and was also inactive against 
<em>S. epidermidis</em>
. In comparison, against this latter, Esc(1‐21)ε20 showed a stronger antibiofilm activity with more than 90% reduction in biofilm viability across the concentration range 6.25–50 μM.</p></section><section id="psc70048-sec-0021"><h4 class="pmc_sec_title">3.2.4. Site‐Specific Isopeptide Bond Switch in Esc(1‐21) Analogs Exhibit Low Hemolytic Activity</h4>
<p>To assess the potential short‐term cytotoxicity of Esc(1‐21) analogs, their ability to lyse mammalian RBCs was evaluated after 30 min of peptide treatment at 37°C. The peptides were tested at the same concentrations used in the microbiological assay (Figure <a href="#psc70048-fig-0005" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="psc70048-fig-0005"><h5 class="obj_head">FIGURE 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/fc2690fbbccc/PSC-31-e70048-g006.jpg" loading="lazy" id="jats-graphic-11" height="386" width="664" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/psc70048-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of Esc(1‐21)ε5, Esc(1‐21)ε9, Esc(1‐21)ε10, Esc(1‐21)ε12, and Esc(1‐21)ε20 on mammalian RBCs after 30 min of peptide treatment at 37°C. The percentage of hemolysis was calculated with respect to the control (cells treated with vehicle). Data are the means ± SEM of three independent experiments.</p></figcaption></figure><p>All peptides showed a safe profile at all the concentrations tested, inducing less than 20% hemoglobin release even at 50 μM, a concentration that, for the most active Esc(1‐21)ε20 peptide, is 16–32 times higher than its MIC. These data are fully consistent with previous studies on Esc(1‐21) which demonstrated about 10%–15% hemolysis at the highest tested concentration of 64 μM [<a href="#psc70048-bib-0033" class="usa-link" aria-describedby="psc70048-bib-0033">33</a>].</p></section><section id="psc70048-sec-0022"><h4 class="pmc_sec_title">3.2.5. Esc(1‐21)ε20 Displays Reduced Cytotoxicity Compared With the Parental Peptide</h4>
<p>The long‐term cytotoxicity of Esc(1‐21) on nucleated cells was extensively described in earlier studies, demonstrating the absence of cytotoxicity up to the concentration of 64 μM [<a href="#psc70048-bib-0009" class="usa-link" aria-describedby="psc70048-bib-0009">9</a>]. Based on these findings, the most active Esc(1‐21)ε20 analog and the less active Esc(1‐21)ε12 were tested at two higher concentrations, that is, 100 and 200 μM using the CCK‐8 assay to investigate the potential cytotoxic effect after 24 h treatment on HaCaT cells (Figure <a href="#psc70048-fig-0006" class="usa-link">6</a>). The results were compared with those obtained with Esc(1‐21), considering that these two high concentrations were not tested in the previous study [<a href="#psc70048-bib-0009" class="usa-link" aria-describedby="psc70048-bib-0009">9</a>].</p>
<figure class="fig xbox font-sm" id="psc70048-fig-0006"><h5 class="obj_head">FIGURE 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/bbb7cf74d6be/PSC-31-e70048-g004.jpg" loading="lazy" id="jats-graphic-13" height="485" width="665" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/psc70048-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Viability of HaCaT cells after 24 h treatment with two different concentrations of Esc(1‐21), Esc(1‐21)ε12, and Esc(1‐21)ε20. Cells not treated with peptides were used as controls. All data are the means of three replicates ± SEM. Statistical analysis was conducted using two‐way ANOVA to determine the significance between the peptides. *<em>p</em> &lt; 0.05; ***<em>p</em> &lt; 0.001; ****<em>p</em> &lt; 0.0001; ns, not significant.</p></figcaption></figure><p>Esc(1‐21)ε12 peptide was not cytotoxic towards HaCaT cells at either 100 or 200 μM. In contrast, the parent peptide provoked about 50% reduction in cell viability at 100 μM and nearly complete cell death at 200 μM (Figure <a href="#psc70048-fig-0006" class="usa-link">6</a>). Interestingly, Esc(1‐21)ε20 provoked a similar reduction in cell viability (~50%) at 100 μM. However, at 200 μM it exhibited lower cytotoxicity compared to the parent Esc(1‐21) with a reduction of cell viability of about 65% versus 95% of Esc(1‐21).</p></section><section id="psc70048-sec-0023"><h4 class="pmc_sec_title">3.2.6. Substitution With Site‐Specific Isopeptide Bond Enhances Proteolytic Stability in Human Plasma</h4>
<p>One of the major challenges limiting the clinical application of AMPs is their susceptibility to proteolytic degradation [<a href="#psc70048-bib-0034" class="usa-link" aria-describedby="psc70048-bib-0034">34</a>]. To assess the proteolytic stability of the least and most active modified peptides, Esc(1‐21)ε12 and Esc(1‐21)ε20, as well as the parental Esc(1‐21), were incubated in fresh human plasma at 37°C for 6 and 24 h (see Section <a href="#psc70048-sec-0002" class="usa-link">2</a>). The degradation profiles revealed that both isopeptide bond‐containing analogs exhibited improved stability compared to the wild‐type peptide (Table <a href="#psc70048-tbl-0003" class="usa-link">3</a> and Figure <a href="#psc70048-supitem-0001" class="usa-link">S2</a>).</p>
<section class="tw xbox font-sm" id="psc70048-tbl-0003"><h5 class="obj_head">TABLE 3.</h5>
<div class="caption p"><p>Peptide amount after 6 and 24 h incubation with human fresh plasma at 37°C.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th rowspan="2" style="border-bottom:solid 1px #000000" align="left" valign="bottom" colspan="1">Peptide</th>
<th align="center" colspan="2" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1">Peptide amount (%)<a href="#psc70048-note-0010" class="usa-link">
<sup>a</sup>
</a>
</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">6 h</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">24 h</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.55</td>
<td align="center" valign="top" rowspan="1" colspan="1">15.59</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)ε12</td>
<td align="center" valign="top" rowspan="1" colspan="1">73.34</td>
<td align="center" valign="top" rowspan="1" colspan="1">44.95</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Esc(1‐21)ε20</td>
<td align="center" valign="top" rowspan="1" colspan="1">65.61</td>
<td align="center" valign="top" rowspan="1" colspan="1">39.64</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/psc70048-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="psc70048-note-0010">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Peptide amounts were determined by the peak areas of the RP‐HPLC relative to those of the control peptide at 0 min (set as 100%).</p>
</div></div></section><p>Notably, after 6 and 24 h, the remaining intact peptide of Esc(1‐21) was only 30.55% and 15.59%, respectively, highlighting its rapid degradation in plasma. In contrast, both Esc(1‐21)ε12 and Esc(1‐21)ε20 showed significantly higher resistance to proteolysis over the same time periods, with 73.34% and 44.95% for Esc(1‐21)ε12 and 65.61% and 39.64% for Esc(1‐21)ε20. These findings demonstrate that the site‐specific isopeptide bond switch is improving the proteolytic stability of peptides.</p></section></section><section id="psc70048-sec-0024"><h3 class="pmc_sec_title">3.3. Structural Studies</h3>
<section id="psc70048-sec-0025"><h4 class="pmc_sec_title">3.3.1. Membrane‐Induced Folding of Esc(1‐21) Analogs Depends on Isopeptide Bond Position</h4>
<p>In principle, replacing a canonical peptide bond with an isopeptide bond could significantly affect the peptide's folding and, consequently, its ability to interact with the pathogen membrane. To investigate the impact of isopeptide bond positioning on the conformation and membrane association of the analogs, we measured the CD spectra of all the analogs in the presence of phospholipid vesicles mimicking the composition of bacterial membranes (POPE/POPG mixture, 7:3 mol/mol + 2% PEG‐PE, Figures <a href="#psc70048-supitem-0001" class="usa-link">S3–S6</a>).</p>
<p>All the peptides are mostly unstructured in aqueous solution (Figure <a href="#psc70048-fig-0007" class="usa-link">7A</a>), displaying characteristic CD spectra with a minimum in molar ellipticity near 200 nm, indicative of a random coil/other conformation. Upon addition of LUVs, only the parent peptide Esc(1‐21) and the analog Esc(1‐21)ε20 exhibited an increase in secondary structure content (Table <a href="#psc70048-supitem-0001" class="usa-link">S1</a>).</p>
<figure class="fig xbox font-sm" id="psc70048-fig-0007"><h5 class="obj_head">FIGURE 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6aff/12328261/d7d844826313/PSC-31-e70048-g001.jpg" loading="lazy" id="jats-graphic-15" height="194" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/psc70048-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>CD spectra of Esc(1–21)<sup>2</sup> and its analogues (A) in aqueous solution and (B) after LUVs addition (lipid concentration 1 mM, peptide concentration 20 μM).</p></figcaption></figure><p>In contrast, the remaining analogues—particularly those containing substitutions closer to the central region of the sequence (i.e., ε9, ε10, and ε12)—showed only minimal structural changes, even at the highest lipid concentration tested (i.e., 1 mM, Figure <a href="#psc70048-fig-0007" class="usa-link">7B</a>), suggesting a markedly reduced capacity for membrane‐induced folding and insertion.</p>
<p>Unlike other analogues, Esc(1‐21)ε20 is characterized by the presence of the isopeptide bond near the C‐terminal region of the sequence. This substitution may affect the peptide folding upon membrane association only to a minor extent compared to the ε5, ε9, ε10, and ε12 analogues. These results suggest that the structural and functional integrity of membrane‐active peptides can be influenced by the precise location of isopeptide bond backbone modification.</p></section></section></section><section id="psc70048-sec-0026"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>With the growing threat of antibiotic resistance, the discovery of new compounds with antimicrobial activity has become extremely urgent [<a href="#psc70048-bib-0035" class="usa-link" aria-describedby="psc70048-bib-0035">35</a>]. Among the candidates considered as potential new antibiotics are AMPs of innate immunity [<a href="#psc70048-bib-0036" class="usa-link" aria-describedby="psc70048-bib-0036">36</a>]. Over the years, numerous peptides have been isolated and characterized from a variety of natural sources. However, their susceptibility to proteolytic degradation results in a short half‐life [<a href="#psc70048-bib-0037" class="usa-link" aria-describedby="psc70048-bib-0037">37</a>]. In addition, although these peptides preferentially target and rapidly disrupt the anionic membranes of microorganisms, their characteristic α‐helical conformation in contact with biological membranes can lead to cytotoxicity at high concentrations [<a href="#psc70048-bib-0038" class="usa-link" aria-describedby="psc70048-bib-0038">38</a>]. It is therefore critical to explore chemical modification strategies aimed at optimizing the biological properties of these peptides. In recent years, different approaches have been employed to improve the biological properties of Esc(1‐21), a nontoxic derivative of the amphibian‐derived Esculentin‐1a [<a href="#psc70048-bib-0039" class="usa-link" aria-describedby="psc70048-bib-0039">39</a>], including the incorporation of unconventional amino acids or the application of nanotechnologies to enhance peptide stability [<a href="#psc70048-bib-0040" class="usa-link" aria-describedby="psc70048-bib-0040">40</a>].</p>
<p>However, the incorporation of site‐specific isopeptide bonds into the backbone sequence of nontoxic AMP, such as Esc(1‐21), has not been previously explored. This modification involves replacing conventional peptide bonds with isopeptide linkages through the ε‐amino group of lysine residues. This strategy has been used only in a few cases: in the first study, three isopeptide bonds were incorporated into two peptides, the highly toxic Amp1L and MSI‐78 (Pexiganan), generating the modified peptides Amp1EP and MSIEP, respectively. They retained key physicochemical properties while displaying reduced cytotoxicity, enhanced proteolytic stability, and minimal immune modulation. Despite adopting random coil conformations, they maintained strong antibacterial activity through membrane permeabilization [<a href="#psc70048-bib-0023" class="usa-link" aria-describedby="psc70048-bib-0023">23</a>]. In a subsequent study, a site‐specific strategy was used to introduce a single isopeptide bond into the leucine‐ and lysine‐rich Amp1L. They showed that introducing the isopeptide bond rendered the peptide nontoxic and nonhemolytic while maintaining activity and increasing protease resistance. This loss of toxicity and hemolysis is likely due to disruption of the alpha‐helical structure [<a href="#psc70048-bib-0022" class="usa-link" aria-describedby="psc70048-bib-0022">22</a>]. In another recent study, Wang et al. modified the peptide IDR‐1018 by substituting an arginine residue with lysine and introducing a lysine‐mediated isopeptide bond. The resulting peptide, 1018KI11, showed enhanced resistance to bromelain and papain digestion, along with improved antimicrobial and antibiofilm activities [<a href="#psc70048-bib-0020" class="usa-link" aria-describedby="psc70048-bib-0020">20</a>]. Taking these studies as starting points, in our work, we introduced isopeptide bonds at five distinct lysine residues (K5, K9, K10, K12, and K20). This positional switch to isopeptide bonding altered the hydrophobicity and antimicrobial activity of the resulting analogs (Tables <a href="#psc70048-tbl-0001" class="usa-link">1</a> and <a href="#psc70048-tbl-0002" class="usa-link">2</a>). In fact, isopeptide substitutions at positions K5, K9, and K10 led to only minor reductions in antibacterial activity as indicated by the MIC values reported in Table <a href="#psc70048-tbl-0002" class="usa-link">2</a>. Incorporation of an isopeptide bond at the central residue K12, however, resulted in a more pronounced loss of activity, suggesting that central positioning may disrupt critical structural or functional aspects of the peptide. Conversely, modification at K20 slightly enhanced hydrophobicity and improved antimicrobial efficacy (Tables <a href="#psc70048-tbl-0001" class="usa-link">1</a> and <a href="#psc70048-tbl-0002" class="usa-link">2</a>). Among all the analogs, Esc(1‐21)ε20 demonstrated the highest effectiveness, particularly in membrane permeabilization and antibiofilm assays (Figures <a href="#psc70048-fig-0002" class="usa-link">2</a>, <a href="#psc70048-fig-0003" class="usa-link">3</a>, and <a href="#psc70048-fig-0004" class="usa-link">4</a>). Importantly, all isopeptide‐containing analogs exhibited low hemolytic activity up to 50 μM, indicating minimal cytotoxicity toward RBCs (Figure <a href="#psc70048-fig-0005" class="usa-link">5</a>). Furthermore, susceptibility of antimicrobial peptides to degradation in human serum remains a key barrier to their therapeutic application. Given that proteases such as plasmin, kallikrein, neutrophil elastase, and cathepsin G target lysine‐rich and hydrophobic regions, we modified cleavage‐prone lysine residues to probe their role in proteolytic vulnerability [<a href="#psc70048-bib-0041" class="usa-link" aria-describedby="psc70048-bib-0041">41</a>, <a href="#psc70048-bib-0042" class="usa-link" aria-describedby="psc70048-bib-0042">42</a>, <a href="#psc70048-bib-0043" class="usa-link" aria-describedby="psc70048-bib-0043">43</a>, <a href="#psc70048-bib-0044" class="usa-link" aria-describedby="psc70048-bib-0044">44</a>]. The least active Esc(1‐21)ε12 and the most active Esc(1‐21)ε20 showed enhanced proteolytic stability in human plasma compared to the unmodified Esc(1‐21), confirming that the introduction of a noncanonical peptide bond contributed to increasing peptide half‐life. The strategic incorporation of isopeptide bonds may not only disrupt canonical cleavage motifs but also offer a generalizable approach to improving peptide stability in protease‐rich physiological environments. It is well established that the conformational features are crucial for activity and cytotoxicity of cationic AMPs, and variations in the α‐helical content can influence their biological properties [<a href="#psc70048-bib-0045" class="usa-link" aria-describedby="psc70048-bib-0045">45</a>, <a href="#psc70048-bib-0046" class="usa-link" aria-describedby="psc70048-bib-0046">46</a>, <a href="#psc70048-bib-0047" class="usa-link" aria-describedby="psc70048-bib-0047">47</a>]. Interestingly, the relative positioning of the isopeptide bond appears to play a key role in modulating the peptide's ability to adopt an α‐helical conformation in a membrane environment. Specifically, the closer the isopeptide is to the center of the peptide sequence, the greater its disruptive effect on α‐helical folding (Figure <a href="#psc70048-fig-0007" class="usa-link">7</a>). This trend is consistent with the corresponding loss of antibacterial activity and reduced cytotoxicity at high concentrations (Figure <a href="#psc70048-fig-0006" class="usa-link">6</a>). Conversely, the Esc(1‐21)ε20, which retains considerable helicity due to the peripheral location of the isopeptide bond, demonstrated antimicrobial activity comparable to that of the parent Esc(1‐21), but with a lower cytotoxicity. Similar results were obtained by Wani et al., who demonstrated that peptides incorporating isopeptide bonds exhibited reduced cytotoxicity, and that introducing an isopeptide bond near the center of the sequence caused greater disruption to the AMP structure [<a href="#psc70048-bib-0022" class="usa-link" aria-describedby="psc70048-bib-0022">22</a>].</p>
<p>In conclusion, in this work we have introduced an isopeptide bond at the lysine residues in the backbone of the Esc(1‐21) leading to the generation of five analogues with distinct chemical and biological properties. Among these, Esc(1‐21)ε20 showed the most promising features, having (i) antimicrobial and antibiofilm activity comparable to those of the parent peptide; (ii) a lower cytotoxicity towards eukaryotic cells at high concentrations; (iii) a higher stability to proteolytic degradation. All these characteristics make Esc(1‐21)ε20 an attractive candidate for the development of new antibiotic therapies, underscoring the potential of isopeptide bond incorporation to improve AMP pharmacological properties.</p></section><section id="psc70048-sec-0029"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>B.C., D.B.H., D.R., C.V., E.K., G.C., F.C., and E.T. performed the experiments and analyzed the data. B.C. and D.B.H. wrote the manuscript. L.S., N.R.R., Y.S., and M.L.M. revised it critically.</p></section><section id="psc70048-sec-0028"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="psc70048-supitem-0001"><div class="caption p">
<p>
<strong>Figure S1.</strong> Mass spectrometry and analytical RP‐HPLC traces. (A) Representative MALDI‐TOF mass spectrometry spectra. (B) Analytical RP‐HPLC traces demonstrate peptide purity, with single peaks indicating high purity. RP‐HPLC was conducted using an Agilent Technologies 1260 Infinity II spectrometer with a reversed‐phase C<sub>18</sub> column at a flow rate of 1.8 mL/min and monitored at 215 nm.</p>
<p>
<strong>Figure S2.</strong> Isopeptide bond‐substitution enhanced the proteolytic stability of Esc(1‐21) in human plasma. HPLC chromatograms of Esc(1‐21), Esc(1‐21)ε12, and Esc(1‐21)ε20 after incubation in human plasma at 37°C for 0, 6, and 24 h. To aid visualization, traces were offset by +1, +2, and +3 min (<em>x</em>‐axis) and +100, +200, and +300 units (<em>y</em>‐axis) for 0, 6, and 24 h, respectively. Peptide separation was performed on a C<sub>18</sub> column over 30 min using a linear gradient of 20%–80% ACN in ddH<sub>2</sub>O containing 0.1% TFA at a flow rate of 0.6 mL/min. Absorbance was monitored at 215 nm, human plasma without peptide served as a blank and the percentage of remaining peptide was quantified based on the decrease in peak area relative to the untreated sample.</p>
<p>
<strong>Figure S3.</strong> CD spectra of Esc1‐21ε5 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S4.</strong> CD spectra of Esc1‐21ε9 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S5.</strong> CD spectra of Esc1‐21ε10 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S6.</strong> CD spectra of Esc1‐21ε12 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S7.</strong> CD spectra of Esc1‐21ε20 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Table S1.</strong> Estimated secondary structure of Esc(1‐21) and its analogs in aqueous and LUVs solution.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328261/bin/PSC-31-e70048-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PSC-31-e70048-s001.docx</a><sup> (583.4KB, docx) </sup>
</div></div></section></section><section id="psc70048-sec-0027" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This research was supported by PRIN2022‐2022XFFTH5 grant of the Italian Ministry of University and Research (M.L.M) and by Sapienza University (RG124190CBD34B9D). The research of Carlo Vetrano was supported by Project ECS 00000024 Rome Technopole, ‐ CUP B83C22002820006, PNRR Missione 4 Componente 2 Investimento 1.5, funded by European Union—NextGenerationEU. This work was also supported by the Italian Ministry for University (grant PRIN 2020833Y75_005) (L.S.). N. R. R. is deeply grateful for the support of the Dr. Barry Sherman Institute for Medicinal Chemistry and BINA, the translational research and innovation unit of the Weizmann Institute. The research of Y. S. is supported by the Israeli Ministry of Science and Technology (application no.: 3‐14316) and the ISF (application no.: 1944/20).</p></section><section id="notes1"><p>

Casciaro B., Ben Hur D., Roversi D., et al., “Modulating Antimicrobial Activity and Structure of the Peptide Esc(1‐21) via Site‐Specific Isopeptide Bond Formation,” Journal of Peptide Science
31, no. 9 (2025): e70048, 10.1002/psc.70048.

</p>
<section id="psc70048-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="psc70048-note-0001"><p>
<strong>Funding:</strong> This research was supported by Ministero dell'Istruzione, dell'Università e della Ricerca (PRIN2022‐2022XFFTH5 and PRIN 2020833Y75_005), Sapienza Università di Roma (RG124190CBD34B9D), Rome Technopole, ‐ CUP B83C22002820006, PNRR Missione 4 Componente 2 Investimento 1.5, funded by European Union ‐ NextGenerationEU (ECS 00000024), Dr. Barry Sherman Institute for Medicinal Chemistry, BINA ‐ the translational research and innovation unit of the Weizmann Institute, Israeli Ministry of Science and Technology (3‐14316) and Israel Science Foundation (ISF) (1944/20).</p></div>
<div class="fn p" id="psc70048-note-0002"><p>Bruno Casciaro and Daniel Ben Hur have equal contribution.</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Bruno Casciaro, Email: bruno.casciaro@uniroma1.it.</p>
<p>Yechiel Shai, Email: yechiel.shai@weizmann.ac.il.</p>
<p>Maria Luisa Mangoni, Email: marialuisa.mangoni@uniroma1.it.</p></section><section id="psc70048-sec-0031"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section><section id="psc70048-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="psc70048-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="psc70048-bib-0001">
<span class="label">1.</span><cite>
Zaky A. A., Simal‐Gandara J., Eun J. B., Shim J. H., and Abd El‐Aty A. M., “Bioactivities, Applications, Safety, and Health Benefits of Bioactive Peptides From Food and By‐Products: A Review,” Frontiers in Nutrition
8 (2021): 815640.
</cite> [<a href="https://doi.org/10.3389/fnut.2021.815640" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8810531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35127796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Nutrition&amp;title=Bioactivities,%20Applications,%20Safety,%20and%20Health%20Benefits%20of%20Bioactive%20Peptides%20From%20Food%20and%20By%E2%80%90Products:%20A%20Review&amp;author=A.%20A.%20Zaky&amp;author=J.%20Simal%E2%80%90Gandara&amp;author=J.%20B.%20Eun&amp;author=J.%20H.%20Shim&amp;author=A.%20M.%20Abd%20El%E2%80%90Aty&amp;volume=8&amp;publication_year=2021&amp;pages=815640&amp;pmid=35127796&amp;doi=10.3389/fnut.2021.815640&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0002">
<span class="label">2.</span><cite>
Brussow H., “The Antibiotic Resistance Crisis and the Development of New Antibiotics,” Microbial Biotechnology
17, no. 7 (2024): e14510.
</cite> [<a href="https://doi.org/10.1111/1751-7915.14510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11226406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38970161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbial%20Biotechnology&amp;title=The%20Antibiotic%20Resistance%20Crisis%20and%20the%20Development%20of%20New%20Antibiotics&amp;author=H.%20Brussow&amp;volume=17&amp;issue=7&amp;publication_year=2024&amp;pages=e14510&amp;pmid=38970161&amp;doi=10.1111/1751-7915.14510&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0003">
<span class="label">3.</span><cite>
Fathi F., Alizadeh B., Tabarzad M. V., and Tabarzad M., “Important Structural Features of Antimicrobial Peptides Towards Specific Activity: Trends in the Development of Efficient Therapeutics,” Bioorganic Chemistry
149 (2024): 107524.
</cite> [<a href="https://doi.org/10.1016/j.bioorg.2024.107524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38850782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorganic%20Chemistry&amp;title=Important%20Structural%20Features%20of%20Antimicrobial%20Peptides%20Towards%20Specific%20Activity:%20Trends%20in%20the%20Development%20of%20Efficient%20Therapeutics&amp;author=F.%20Fathi&amp;author=B.%20Alizadeh&amp;author=M.%20V.%20Tabarzad&amp;author=M.%20Tabarzad&amp;volume=149&amp;publication_year=2024&amp;pages=107524&amp;pmid=38850782&amp;doi=10.1016/j.bioorg.2024.107524&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0004">
<span class="label">4.</span><cite>
Carneri F., Troiano C., Giaquinto G., Roversi D., Franzyk H., and Stella L., “Water‐Membrane Partition and the Mutant Selection Window of Antimicrobial Peptides: Insights From Liposome Studies,” Journal of Colloid and Interface Science
683, no. Pt 1 (2025): 1078–1086.
</cite> [<a href="https://doi.org/10.1016/j.jcis.2024.12.099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39721079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Colloid%20and%20Interface%20Science&amp;title=Water%E2%80%90Membrane%20Partition%20and%20the%20Mutant%20Selection%20Window%20of%20Antimicrobial%20Peptides:%20Insights%20From%20Liposome%20Studies&amp;author=F.%20Carneri&amp;author=C.%20Troiano&amp;author=G.%20Giaquinto&amp;author=D.%20Roversi&amp;author=H.%20Franzyk&amp;volume=683&amp;issue=Pt%201&amp;publication_year=2025&amp;pages=1078-1086&amp;pmid=39721079&amp;doi=10.1016/j.jcis.2024.12.099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0005">
<span class="label">5.</span><cite>
Avci F. G., Akbulut B. S., and Ozkirimli E., “Membrane Active Peptides and Their Biophysical Characterization,” Biomolecules
8, no. 3 (2018): 77.
</cite> [<a href="https://doi.org/10.3390/biom8030077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6164437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30135402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Membrane%20Active%20Peptides%20and%20Their%20Biophysical%20Characterization&amp;author=F.%20G.%20Avci&amp;author=B.%20S.%20Akbulut&amp;author=E.%20Ozkirimli&amp;volume=8&amp;issue=3&amp;publication_year=2018&amp;pages=77&amp;pmid=30135402&amp;doi=10.3390/biom8030077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0006">
<span class="label">6.</span><cite>
Ji S., An F., Zhang T., et al., “Antimicrobial Peptides: An Alternative to Traditional Antibiotics,” European Journal of Medicinal Chemistry
265 (2024): 116072.
</cite> [<a href="https://doi.org/10.1016/j.ejmech.2023.116072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38147812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Medicinal%20Chemistry&amp;title=Antimicrobial%20Peptides:%20An%20Alternative%20to%20Traditional%20Antibiotics&amp;author=S.%20Ji&amp;author=F.%20An&amp;author=T.%20Zhang&amp;volume=265&amp;publication_year=2024&amp;pages=116072&amp;pmid=38147812&amp;doi=10.1016/j.ejmech.2023.116072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0007">
<span class="label">7.</span><cite>
Shai Y., “Mode of Action of Membrane Active Antimicrobial Peptides,” Biopolymers
66, no. 4 (2002): 236–248.
</cite> [<a href="https://doi.org/10.1002/bip.10260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12491537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biopolymers&amp;title=Mode%20of%20Action%20of%20Membrane%20Active%20Antimicrobial%20Peptides&amp;author=Y.%20Shai&amp;volume=66&amp;issue=4&amp;publication_year=2002&amp;pages=236-248&amp;pmid=12491537&amp;doi=10.1002/bip.10260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0008">
<span class="label">8.</span><cite>
Casciaro B., Loffredo M. R., Luca V., Verrusio W., Cacciafesta M., and Mangoni M. L., “Esculentin‐1a Derived Antipseudomonal Peptides: Limited Induction of Resistance and Synergy With Aztreonam,” Protein and Peptide Letters
25, no. 12 (2018): 1155–1162.
</cite> [<a href="https://doi.org/10.2174/0929866525666181101104649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30381056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20and%20Peptide%20Letters&amp;title=Esculentin%E2%80%901a%20Derived%20Antipseudomonal%20Peptides:%20Limited%20Induction%20of%20Resistance%20and%20Synergy%20With%20Aztreonam&amp;author=B.%20Casciaro&amp;author=M.%20R.%20Loffredo&amp;author=V.%20Luca&amp;author=W.%20Verrusio&amp;author=M.%20Cacciafesta&amp;volume=25&amp;issue=12&amp;publication_year=2018&amp;pages=1155-1162&amp;pmid=30381056&amp;doi=10.2174/0929866525666181101104649&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0009">
<span class="label">9.</span><cite>
Loffredo M. R., Cappiello F., Cappella G., et al., “The pH‐Insensitive Antimicrobial and Antibiofilm Activities of the Frog Skin Derived Peptide Esc(1–21): Promising Features for Novel Anti‐Infective Drugs,” Antibiotics
13, no. 8 (2024): 701.
</cite> [<a href="https://doi.org/10.3390/antibiotics13080701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11350779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39200001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antibiotics&amp;title=The%20pH%E2%80%90Insensitive%20Antimicrobial%20and%20Antibiofilm%20Activities%20of%20the%20Frog%20Skin%20Derived%20Peptide%20Esc(1%E2%80%9321):%20Promising%20Features%20for%20Novel%20Anti%E2%80%90Infective%20Drugs&amp;author=M.%20R.%20Loffredo&amp;author=F.%20Cappiello&amp;author=G.%20Cappella&amp;volume=13&amp;issue=8&amp;publication_year=2024&amp;pages=701&amp;pmid=39200001&amp;doi=10.3390/antibiotics13080701&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0010">
<span class="label">10.</span><cite>
Mangoni M. L., Loffredo M. R., Casciaro B., Ferrera L., and Cappiello F., “An Overview of Frog Skin‐Derived esc Peptides: Promising Multifunctional Weapons Against Pseudomonas Aeruginosa‐Induced Pulmonary and Ocular Surface Infections,” International Journal of Molecular Sciences
25 (2024): 25 (8).</cite> [<a href="https://doi.org/10.3390/ijms25084400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11049899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38673985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=An%20Overview%20of%20Frog%20Skin%E2%80%90Derived%20esc%20Peptides:%20Promising%20Multifunctional%20Weapons%20Against%20Pseudomonas%20Aeruginosa%E2%80%90Induced%20Pulmonary%20and%20Ocular%20Surface%20Infections&amp;author=M.%20L.%20Mangoni&amp;author=M.%20R.%20Loffredo&amp;author=B.%20Casciaro&amp;author=L.%20Ferrera&amp;author=F.%20Cappiello&amp;volume=25&amp;publication_year=2024&amp;pages=25%20(8)&amp;pmid=38673985&amp;doi=10.3390/ijms25084400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0011">
<span class="label">11.</span><cite>
Casciaro B., Cappiello F., Loffredo M. R., Ghirga F., and Mangoni M. L., “The Potential of Frog Skin Peptides for Anti‐Infective Therapies: The Case of Esculentin‐1a(1‐21)NH2,” Current Medicinal Chemistry
27, no. 9 (2020): 1405–1419.
</cite> [<a href="https://doi.org/10.2174/0929867326666190722095408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31333082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Medicinal%20Chemistry&amp;title=The%20Potential%20of%20Frog%20Skin%20Peptides%20for%20Anti%E2%80%90Infective%20Therapies:%20The%20Case%20of%20Esculentin%E2%80%901a(1%E2%80%9021)NH2&amp;author=B.%20Casciaro&amp;author=F.%20Cappiello&amp;author=M.%20R.%20Loffredo&amp;author=F.%20Ghirga&amp;author=M.%20L.%20Mangoni&amp;volume=27&amp;issue=9&amp;publication_year=2020&amp;pages=1405-1419&amp;pmid=31333082&amp;doi=10.2174/0929867326666190722095408&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0012">
<span class="label">12.</span><cite>
Di Grazia A., Cappiello F., Cohen H., et al., “D‐Amino Acids Incorporation in the Frog Skin‐Derived Peptide Esculentin‐1a(1‐21)NH2 Is Beneficial for Its Multiple Functions,” Amino Acids
47, no. 12 (2015): 2505–2519.
</cite> [<a href="https://doi.org/10.1007/s00726-015-2041-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26162435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Amino%20Acids&amp;title=D%E2%80%90Amino%20Acids%20Incorporation%20in%20the%20Frog%20Skin%E2%80%90Derived%20Peptide%20Esculentin%E2%80%901a(1%E2%80%9021)NH2%20Is%20Beneficial%20for%20Its%20Multiple%20Functions&amp;author=A.%20Di%20Grazia&amp;author=F.%20Cappiello&amp;author=H.%20Cohen&amp;volume=47&amp;issue=12&amp;publication_year=2015&amp;pages=2505-2519&amp;pmid=26162435&amp;doi=10.1007/s00726-015-2041-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0013">
<span class="label">13.</span><cite>
Biondi B., Casciaro B., Di Grazia A., et al., “Effects of Aib Residues Insertion on the Structural‐Functional Properties of the Frog Skin‐Derived Peptide Esculentin‐1a(1‐21)NH<sub>2</sub>
,” Amino Acids
49, no. 1 (2017): 139–150.
</cite> [<a href="https://doi.org/10.1007/s00726-016-2341-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27726008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Amino%20Acids&amp;title=Effects%20of%20Aib%20Residues%20Insertion%20on%20the%20Structural%E2%80%90Functional%20Properties%20of%20the%20Frog%20Skin%E2%80%90Derived%20Peptide%20Esculentin%E2%80%901a(1%E2%80%9021)NH2&amp;author=B.%20Biondi&amp;author=B.%20Casciaro&amp;author=A.%20Di%20Grazia&amp;volume=49&amp;issue=1&amp;publication_year=2017&amp;pages=139-150&amp;pmid=27726008&amp;doi=10.1007/s00726-016-2341-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0014">
<span class="label">14.</span><cite>
Loffredo M. R., Casciaro B., Bellavita R., et al., “Strategic Single‐Residue Substitution in the Antimicrobial Peptide Esc(1‐21) Confers Activity Against Staphylococcus Aureus, Including Drug‐Resistant and Biofilm Phenotype,” ACS Infectious Diseases
10, no. 7 (2024): 2403–2418.
</cite> [<a href="https://doi.org/10.1021/acsinfecdis.4c00130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11250030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38848266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Infectious%20Diseases&amp;title=Strategic%20Single%E2%80%90Residue%20Substitution%20in%20the%20Antimicrobial%20Peptide%20Esc(1%E2%80%9021)%20Confers%20Activity%20Against%20Staphylococcus%20Aureus,%20Including%20Drug%E2%80%90Resistant%20and%20Biofilm%20Phenotype&amp;author=M.%20R.%20Loffredo&amp;author=B.%20Casciaro&amp;author=R.%20Bellavita&amp;volume=10&amp;issue=7&amp;publication_year=2024&amp;pages=2403-2418&amp;pmid=38848266&amp;doi=10.1021/acsinfecdis.4c00130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0015">
<span class="label">15.</span><cite>
Kang H. J. and Baker E. N., “Intramolecular Isopeptide Bonds: Protein Crosslinks Built for Stress?,” Trends in Biochemical Sciences
36, no. 4 (2011): 229–237.
</cite> [<a href="https://doi.org/10.1016/j.tibs.2010.09.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21055949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Biochemical%20Sciences&amp;title=Intramolecular%20Isopeptide%20Bonds:%20Protein%20Crosslinks%20Built%20for%20Stress?&amp;author=H.%20J.%20Kang&amp;author=E.%20N.%20Baker&amp;volume=36&amp;issue=4&amp;publication_year=2011&amp;pages=229-237&amp;pmid=21055949&amp;doi=10.1016/j.tibs.2010.09.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0016">
<span class="label">16.</span><cite>
Srisantitham S., Walker A. L., Markel U., and Tezcan F. A., “De Novo Design of Proteins for Autocatalytic Isopeptide Bond Formation,” Journal of the American Chemical Society
147 (2025): 12338–12346.
</cite> [<a href="https://doi.org/10.1021/jacs.5c03319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12065602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40138671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Chemical%20Society&amp;title=De%20Novo%20Design%20of%20Proteins%20for%20Autocatalytic%20Isopeptide%20Bond%20Formation&amp;author=S.%20Srisantitham&amp;author=A.%20L.%20Walker&amp;author=U.%20Markel&amp;author=F.%20A.%20Tezcan&amp;volume=147&amp;publication_year=2025&amp;pages=12338-12346&amp;pmid=40138671&amp;doi=10.1021/jacs.5c03319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0017">
<span class="label">17.</span><cite>
Wikoff W. R., Liljas L., Duda R. L., Tsuruta H., Hendrix R. W., and Johnson J. E., “Topologically Linked Protein Rings in the Bacteriophage HK97 Capsid,” Science
289, no. 5487 (2000): 2129–2133.
</cite> [<a href="https://doi.org/10.1126/science.289.5487.2129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11000116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Topologically%20Linked%20Protein%20Rings%20in%20the%20Bacteriophage%20HK97%20Capsid&amp;author=W.%20R.%20Wikoff&amp;author=L.%20Liljas&amp;author=R.%20L.%20Duda&amp;author=H.%20Tsuruta&amp;author=R.%20W.%20Hendrix&amp;volume=289&amp;issue=5487&amp;publication_year=2000&amp;pages=2129-2133&amp;pmid=11000116&amp;doi=10.1126/science.289.5487.2129&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0018">
<span class="label">18.</span><cite>
Vertegaal A. C., “Uncovering Ubiquitin and Ubiquitin‐Like Signaling Networks,” Chemical Reviews
111, no. 12 (2011): 7923–7940.
</cite> [<a href="https://doi.org/10.1021/cr200187e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3238414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22004258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Reviews&amp;title=Uncovering%20Ubiquitin%20and%20Ubiquitin%E2%80%90Like%20Signaling%20Networks&amp;author=A.%20C.%20Vertegaal&amp;volume=111&amp;issue=12&amp;publication_year=2011&amp;pages=7923-7940&amp;pmid=22004258&amp;doi=10.1021/cr200187e&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0019">
<span class="label">19.</span><cite>
Rivas‐Pardo J. A., Badilla C. L., Tapia‐Rojo R., Alonso‐Caballero A., and Fernandez J. M., “Molecular Strategy for Blocking Isopeptide Bond Formation in Nascent Pilin Proteins,” Proceedings of the National Academy of Sciences of the United States of America
115, no. 37 (2018): 9222–9227.
</cite> [<a href="https://doi.org/10.1073/pnas.1807689115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6140542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30150415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Molecular%20Strategy%20for%20Blocking%20Isopeptide%20Bond%20Formation%20in%20Nascent%20Pilin%20Proteins&amp;author=J.%20A.%20Rivas%E2%80%90Pardo&amp;author=C.%20L.%20Badilla&amp;author=R.%20Tapia%E2%80%90Rojo&amp;author=A.%20Alonso%E2%80%90Caballero&amp;author=J.%20M.%20Fernandez&amp;volume=115&amp;issue=37&amp;publication_year=2018&amp;pages=9222-9227&amp;pmid=30150415&amp;doi=10.1073/pnas.1807689115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0020">
<span class="label">20.</span><cite>
Wang Q., Yang Y., Dong X., Wang H., Ding L., and Wang X., “Design of a Novel Lysine Isopeptide 1018KI11 With Potent Antimicrobial Activity as a Safe and Stable Food Preservative Candidate,” Journal of Agricultural and Food Chemistry
72, no. 14 (2024): 7894–7905.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.3c09484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38551085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural%20and%20Food%20Chemistry&amp;title=Design%20of%20a%20Novel%20Lysine%20Isopeptide%201018KI11%20With%20Potent%20Antimicrobial%20Activity%20as%20a%20Safe%20and%20Stable%20Food%20Preservative%20Candidate&amp;author=Q.%20Wang&amp;author=Y.%20Yang&amp;author=X.%20Dong&amp;author=H.%20Wang&amp;author=L.%20Ding&amp;volume=72&amp;issue=14&amp;publication_year=2024&amp;pages=7894-7905&amp;pmid=38551085&amp;doi=10.1021/acs.jafc.3c09484&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0021">
<span class="label">21.</span><cite>
Kiper E., Ben Hur D., Alfandari D., et al., “Antimicrobial Peptides Selectively Target Malaria Parasites by a Cholesterol‐Dependent Mechanism,” Journal of Biological Chemistry
301, no. 4 (2025): 108298.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2025.108298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11993164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39971158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Antimicrobial%20Peptides%20Selectively%20Target%20Malaria%20Parasites%20by%20a%20Cholesterol%E2%80%90Dependent%20Mechanism&amp;author=E.%20Kiper&amp;author=D.%20Ben%20Hur&amp;author=D.%20Alfandari&amp;volume=301&amp;issue=4&amp;publication_year=2025&amp;pages=108298&amp;pmid=39971158&amp;doi=10.1016/j.jbc.2025.108298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0022">
<span class="label">22.</span><cite>
Wani N. A., Stolovicki E., Hur D. B., and Shai Y., “Site‐Specific Isopeptide Bond Formation: A Powerful Tool for the Generation of Potent and Nontoxic Antimicrobial Peptides,” Journal of Medicinal Chemistry
65, no. 6 (2022): 5085–5094.
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.2c00061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8958506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35290038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Medicinal%20Chemistry&amp;title=Site%E2%80%90Specific%20Isopeptide%20Bond%20Formation:%20A%20Powerful%20Tool%20for%20the%20Generation%20of%20Potent%20and%20Nontoxic%20Antimicrobial%20Peptides&amp;author=N.%20A.%20Wani&amp;author=E.%20Stolovicki&amp;author=D.%20B.%20Hur&amp;author=Y.%20Shai&amp;volume=65&amp;issue=6&amp;publication_year=2022&amp;pages=5085-5094&amp;pmid=35290038&amp;doi=10.1021/acs.jmedchem.2c00061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0023">
<span class="label">23.</span><cite>
Wani N. A., Ben Hur D., Kapach G., Stolovicki E., Rotem E., and Shai Y., “Switching Bond: Generation of New Antimicrobial Peptides via the Incorporation of an Intramolecular Isopeptide Bond,” ACS Infectious Diseases
7, no. 6 (2021): 1702–1712.
</cite> [<a href="https://doi.org/10.1021/acsinfecdis.1c00037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8634383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34043312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Infectious%20Diseases&amp;title=Switching%20Bond:%20Generation%20of%20New%20Antimicrobial%20Peptides%20via%20the%20Incorporation%20of%20an%20Intramolecular%20Isopeptide%20Bond&amp;author=N.%20A.%20Wani&amp;author=D.%20Ben%20Hur&amp;author=G.%20Kapach&amp;author=E.%20Stolovicki&amp;author=E.%20Rotem&amp;volume=7&amp;issue=6&amp;publication_year=2021&amp;pages=1702-1712&amp;pmid=34043312&amp;doi=10.1021/acsinfecdis.1c00037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0024">
<span class="label">24.</span><cite>
Merrifield R. B., Vizioli L. D., and Boman H. G., “Synthesis of the Antibacterial Peptide Cecropin a (1‐33),” Biochemistry
21, no. 20 (1982): 5020–5031.
</cite> [<a href="https://doi.org/10.1021/bi00263a028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6814482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Synthesis%20of%20the%20Antibacterial%20Peptide%20Cecropin%20a%20(1%E2%80%9033)&amp;author=R.%20B.%20Merrifield&amp;author=L.%20D.%20Vizioli&amp;author=H.%20G.%20Boman&amp;volume=21&amp;issue=20&amp;publication_year=1982&amp;pages=5020-5031&amp;pmid=6814482&amp;doi=10.1021/bi00263a028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0025">
<span class="label">25.</span><cite>
Klug Y. A., Ashkenazi A., Viard M., Porat Z., Blumenthal R., and Shai Y., “Early and Late HIV‐1 Membrane Fusion Events Are Impaired by Sphinganine Lipidated Peptides That Target the Fusion Site,” Biochemical Journal
461, no. 2 (2014): 213–222.
</cite> [<a href="https://doi.org/10.1042/BJ20140189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4072049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24766462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20Journal&amp;title=Early%20and%20Late%20HIV%E2%80%901%20Membrane%20Fusion%20Events%20Are%20Impaired%20by%20Sphinganine%20Lipidated%20Peptides%20That%20Target%20the%20Fusion%20Site&amp;author=Y.%20A.%20Klug&amp;author=A.%20Ashkenazi&amp;author=M.%20Viard&amp;author=Z.%20Porat&amp;author=R.%20Blumenthal&amp;volume=461&amp;issue=2&amp;publication_year=2014&amp;pages=213-222&amp;pmid=24766462&amp;doi=10.1042/BJ20140189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0026">
<span class="label">26.</span><cite>
Mi B., Liu J., Liu G., et al., “Icariin Promotes Wound Healing by Enhancing the Migration and Proliferation of Keratinocytes via the AKT and ERK Signaling Pathway,” International Journal of Molecular Medicine
42, no. 2 (2018): 831–838.
</cite> [<a href="https://doi.org/10.3892/ijmm.2018.3676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6034939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29767263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Medicine&amp;title=Icariin%20Promotes%20Wound%20Healing%20by%20Enhancing%20the%20Migration%20and%20Proliferation%20of%20Keratinocytes%20via%20the%20AKT%20and%20ERK%20Signaling%20Pathway&amp;author=B.%20Mi&amp;author=J.%20Liu&amp;author=G.%20Liu&amp;volume=42&amp;issue=2&amp;publication_year=2018&amp;pages=831-838&amp;pmid=29767263&amp;doi=10.3892/ijmm.2018.3676&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0027">
<span class="label">27.</span><cite>
Stewart J. C., “Colorimetric Determination of Phospholipids With Ammonium Ferrothiocyanate,” Analytical Biochemistry
104, no. 1 (1980): 10–14.
</cite> [<a href="https://doi.org/10.1016/0003-2697(80)90269-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6892980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Biochemistry&amp;title=Colorimetric%20Determination%20of%20Phospholipids%20With%20Ammonium%20Ferrothiocyanate&amp;author=J.%20C.%20Stewart&amp;volume=104&amp;issue=1&amp;publication_year=1980&amp;pages=10-14&amp;pmid=6892980&amp;doi=10.1016/0003-2697(80)90269-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0028">
<span class="label">28.</span><cite>
Micsonai A., Wien F., Kernya L., et al., “Accurate Secondary Structure Prediction and Fold Recognition for Circular Dichroism Spectroscopy,” Proceedings of the National Academy of Sciences of the United States of America
112, no. 24 (2015): E3095–E3103.
</cite> [<a href="https://doi.org/10.1073/pnas.1500851112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4475991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26038575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Accurate%20Secondary%20Structure%20Prediction%20and%20Fold%20Recognition%20for%20Circular%20Dichroism%20Spectroscopy&amp;author=A.%20Micsonai&amp;author=F.%20Wien&amp;author=L.%20Kernya&amp;volume=112&amp;issue=24&amp;publication_year=2015&amp;pages=E3095-E3103&amp;pmid=26038575&amp;doi=10.1073/pnas.1500851112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0029">
<span class="label">29.</span><cite>
Luca V., Stringaro A., Colone M., Pini A., and Mangoni M. L., “Esculentin(1‐21), an Amphibian Skin Membrane‐Active Peptide With Potent Activity on Both Planktonic and Biofilm Cells of the Bacterial Pathogen <em>Pseudomonas aeruginosa</em>
,” Cellular and Molecular Life Sciences
70, no. 15 (2013): 2773–2786.
</cite> [<a href="https://doi.org/10.1007/s00018-013-1291-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11113931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23503622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cellular%20and%20Molecular%20Life%20Sciences&amp;title=Esculentin(1%E2%80%9021),%20an%20Amphibian%20Skin%20Membrane%E2%80%90Active%20Peptide%20With%20Potent%20Activity%20on%20Both%20Planktonic%20and%20Biofilm%20Cells%20of%20the%20Bacterial%20Pathogen%20Pseudomonas%20aeruginosa&amp;author=V.%20Luca&amp;author=A.%20Stringaro&amp;author=M.%20Colone&amp;author=A.%20Pini&amp;author=M.%20L.%20Mangoni&amp;volume=70&amp;issue=15&amp;publication_year=2013&amp;pages=2773-2786&amp;pmid=23503622&amp;doi=10.1007/s00018-013-1291-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0030">
<span class="label">30.</span><cite>
Sharma S., Mohler J., Mahajan S. D., Schwartz S. A., Bruggemann L., and Aalinkeel R., “Microbial Biofilm: A Review on Formation, Infection, Antibiotic Resistance, Control Measures, and Innovative Treatment,” Microorganisms
11, no. 6 (2023): 1614.
</cite> [<a href="https://doi.org/10.3390/microorganisms11061614" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10305407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37375116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Microbial%20Biofilm:%20A%20Review%20on%20Formation,%20Infection,%20Antibiotic%20Resistance,%20Control%20Measures,%20and%20Innovative%20Treatment&amp;author=S.%20Sharma&amp;author=J.%20Mohler&amp;author=S.%20D.%20Mahajan&amp;author=S.%20A.%20Schwartz&amp;author=L.%20Bruggemann&amp;volume=11&amp;issue=6&amp;publication_year=2023&amp;pages=1614&amp;pmid=37375116&amp;doi=10.3390/microorganisms11061614&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0031">
<span class="label">31.</span><cite>
Ben Hur D., Kapach G., Wani N. A., et al., “Antimicrobial Peptides Against Multidrug‐Resistant Pseudomonas Aeruginosa Biofilm From Cystic Fibrosis Patients,” Journal of Medicinal Chemistry
65, no. 13 (2022): 9050–9062.
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.2c00270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9289885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35759644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Medicinal%20Chemistry&amp;title=Antimicrobial%20Peptides%20Against%20Multidrug%E2%80%90Resistant%20Pseudomonas%20Aeruginosa%20Biofilm%20From%20Cystic%20Fibrosis%20Patients&amp;author=D.%20Ben%20Hur&amp;author=G.%20Kapach&amp;author=N.%20A.%20Wani&amp;volume=65&amp;issue=13&amp;publication_year=2022&amp;pages=9050-9062&amp;pmid=35759644&amp;doi=10.1021/acs.jmedchem.2c00270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0032">
<span class="label">32.</span><cite>
Nadar S., Khan T., Patching S. G., and Omri A., “Development of Antibiofilm Therapeutics Strategies to Overcome Antimicrobial Drug Resistance,” Microorganisms
10, no. 2 (2022): 303.
</cite> [<a href="https://doi.org/10.3390/microorganisms10020303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8879831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35208758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Development%20of%20Antibiofilm%20Therapeutics%20Strategies%20to%20Overcome%20Antimicrobial%20Drug%20Resistance&amp;author=S.%20Nadar&amp;author=T.%20Khan&amp;author=S.%20G.%20Patching&amp;author=A.%20Omri&amp;volume=10&amp;issue=2&amp;publication_year=2022&amp;pages=303&amp;pmid=35208758&amp;doi=10.3390/microorganisms10020303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0033">
<span class="label">33.</span><cite>
Casciaro B., Loffredo M. R., Cappiello F., Verrusio W., Corleto V. D., and Mangoni M. L., “Frog Skin‐Derived Peptides Against <em>Corynebacterium jeikeium</em>: Correlation Between Antibacterial and Cytotoxic Activities,” Antibiotics (Basel)
9, no. 8 (2020): 448.
</cite> [<a href="https://doi.org/10.3390/antibiotics9080448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7459541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32722535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antibiotics%20(Basel)&amp;title=Frog%20Skin%E2%80%90Derived%20Peptides%20Against%20Corynebacterium%20jeikeium:%20Correlation%20Between%20Antibacterial%20and%20Cytotoxic%20Activities&amp;author=B.%20Casciaro&amp;author=M.%20R.%20Loffredo&amp;author=F.%20Cappiello&amp;author=W.%20Verrusio&amp;author=V.%20D.%20Corleto&amp;volume=9&amp;issue=8&amp;publication_year=2020&amp;pages=448&amp;pmid=32722535&amp;doi=10.3390/antibiotics9080448&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0034">
<span class="label">34.</span><cite>
Yang R., Ma X., Peng F., et al., “Advances in Antimicrobial Peptides: From Mechanistic Insights to Chemical Modifications,” Biotechnology Advances
81 (2025): 108570.
</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2025.108570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40154761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20Advances&amp;title=Advances%20in%20Antimicrobial%20Peptides:%20From%20Mechanistic%20Insights%20to%20Chemical%20Modifications&amp;author=R.%20Yang&amp;author=X.%20Ma&amp;author=F.%20Peng&amp;volume=81&amp;publication_year=2025&amp;pages=108570&amp;pmid=40154761&amp;doi=10.1016/j.biotechadv.2025.108570&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0035">
<span class="label">35.</span><cite>
Ahmed S. K., Hussein S., Qurbani K., et al., “Antimicrobial Resistance: Impacts, Challenges, and Future Prospects,” Journal of Medicine, Surgery, and Public Health
2 (2024): 100081.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Medicine,%20Surgery,%20and%20Public%20Health&amp;title=Antimicrobial%20Resistance:%20Impacts,%20Challenges,%20and%20Future%20Prospects&amp;author=S.%20K.%20Ahmed&amp;author=S.%20Hussein&amp;author=K.%20Qurbani&amp;volume=2&amp;publication_year=2024&amp;pages=100081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0036">
<span class="label">36.</span><cite>
Browne K., Chakraborty S., Chen R., et al., “A New Era of Antibiotics: The Clinical Potential of Antimicrobial Peptides,” International Journal of Molecular Sciences
21 (2020): 21 (19).</cite> [<a href="https://doi.org/10.3390/ijms21197047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7582481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32987946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=A%20New%20Era%20of%20Antibiotics:%20The%20Clinical%20Potential%20of%20Antimicrobial%20Peptides&amp;author=K.%20Browne&amp;author=S.%20Chakraborty&amp;author=R.%20Chen&amp;volume=21&amp;publication_year=2020&amp;pages=21%20(19)&amp;pmid=32987946&amp;doi=10.3390/ijms21197047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0037">
<span class="label">37.</span><cite>
Lai Z., Yuan X., Chen H., Zhu Y., Dong N., and Shan A., “Strategies Employed in the Design of Antimicrobial Peptides With Enhanced Proteolytic Stability,” Biotechnology Advances
59 (2022): 107962.
</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2022.107962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35452776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20Advances&amp;title=Strategies%20Employed%20in%20the%20Design%20of%20Antimicrobial%20Peptides%20With%20Enhanced%20Proteolytic%20Stability&amp;author=Z.%20Lai&amp;author=X.%20Yuan&amp;author=H.%20Chen&amp;author=Y.%20Zhu&amp;author=N.%20Dong&amp;volume=59&amp;publication_year=2022&amp;pages=107962&amp;pmid=35452776&amp;doi=10.1016/j.biotechadv.2022.107962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0038">
<span class="label">38.</span><cite>
Inui Kishi R. N., Stach‐Machado D., Singulani J. L., et al., “Evaluation of Cytotoxicity Features of Antimicrobial Peptides With Potential to Control Bacterial Diseases of Citrus,” PLoS ONE
13, no. 9 (2018): e0203451.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0203451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6128562/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30192822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Evaluation%20of%20Cytotoxicity%20Features%20of%20Antimicrobial%20Peptides%20With%20Potential%20to%20Control%20Bacterial%20Diseases%20of%20Citrus&amp;author=R.%20N.%20Inui%20Kishi&amp;author=D.%20Stach%E2%80%90Machado&amp;author=J.%20L.%20Singulani&amp;volume=13&amp;issue=9&amp;publication_year=2018&amp;pages=e0203451&amp;pmid=30192822&amp;doi=10.1371/journal.pone.0203451&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0039">
<span class="label">39.</span><cite>
Casciaro B., Cappiello F., Cacciafesta M., and Mangoni M. L., “Promising Approaches to Optimize the Biological Properties of the Antimicrobial Peptide Esculentin‐1a(1‐21)NH<sub>2</sub>: Amino Acids Substitution and Conjugation to Nanoparticles,” Frontiers in Chemistry
5 (2017): 26.
</cite> [<a href="https://doi.org/10.3389/fchem.2017.00026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5404639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28487853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Chemistry&amp;title=Promising%20Approaches%20to%20Optimize%20the%20Biological%20Properties%20of%20the%20Antimicrobial%20Peptide%20Esculentin%E2%80%901a(1%E2%80%9021)NH2:%20Amino%20Acids%20Substitution%20and%20Conjugation%20to%20Nanoparticles&amp;author=B.%20Casciaro&amp;author=F.%20Cappiello&amp;author=M.%20Cacciafesta&amp;author=M.%20L.%20Mangoni&amp;volume=5&amp;publication_year=2017&amp;pages=26&amp;pmid=28487853&amp;doi=10.3389/fchem.2017.00026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0040">
<span class="label">40.</span><cite>
Tan P., Fu H., and Ma X., “Design, Optimization, and Nanotechnology of Antimicrobial Peptides: From Exploration to Applications,” Nano Today
39 (2021): 101229.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nano%20Today&amp;title=Design,%20Optimization,%20and%20Nanotechnology%20of%20Antimicrobial%20Peptides:%20From%20Exploration%20to%20Applications&amp;author=P.%20Tan&amp;author=H.%20Fu&amp;author=X.%20Ma&amp;volume=39&amp;publication_year=2021&amp;pages=101229&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0041">
<span class="label">41.</span><cite>
Korkmaz B., Horwitz M. S., Jenne D. E., and Gauthier F., “Neutrophil Elastase, Proteinase 3, and Cathepsin G as Therapeutic Targets in Human Diseases,” Pharmacological Reviews
62, no. 4 (2010): 726–759.
</cite> [<a href="https://doi.org/10.1124/pr.110.002733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2993259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21079042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacological%20Reviews&amp;title=Neutrophil%20Elastase,%20Proteinase%203,%20and%20Cathepsin%20G%20as%20Therapeutic%20Targets%20in%20Human%20Diseases&amp;author=B.%20Korkmaz&amp;author=M.%20S.%20Horwitz&amp;author=D.%20E.%20Jenne&amp;author=F.%20Gauthier&amp;volume=62&amp;issue=4&amp;publication_year=2010&amp;pages=726-759&amp;pmid=21079042&amp;doi=10.1124/pr.110.002733&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0042">
<span class="label">42.</span><cite>
Starr C. G. and Wimley W. C., “Antimicrobial Peptides Are Degraded by the Cytosolic Proteases of Human Erythrocytes,” Biochimica et Biophysica Acta ‐ Biomembranes
1859, no. 12 (2017): 2319–2326.
</cite> [<a href="https://doi.org/10.1016/j.bbamem.2017.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5659893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28912099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20%E2%80%90%20Biomembranes&amp;title=Antimicrobial%20Peptides%20Are%20Degraded%20by%20the%20Cytosolic%20Proteases%20of%20Human%20Erythrocytes&amp;author=C.%20G.%20Starr&amp;author=W.%20C.%20Wimley&amp;volume=1859&amp;issue=12&amp;publication_year=2017&amp;pages=2319-2326&amp;pmid=28912099&amp;doi=10.1016/j.bbamem.2017.09.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0043">
<span class="label">43.</span><cite>
Hollands A., Gonzalez D., Leire E., et al., “A Bacterial Pathogen Co‐Opts Host Plasmin to Resist Killing by Cathelicidin Antimicrobial Peptides,” Journal of Biological Chemistry
287, no. 49 (2012): 40891–40897.
</cite> [<a href="https://doi.org/10.1074/jbc.M112.404582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3510793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23038245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=A%20Bacterial%20Pathogen%20Co%E2%80%90Opts%20Host%20Plasmin%20to%20Resist%20Killing%20by%20Cathelicidin%20Antimicrobial%20Peptides&amp;author=A.%20Hollands&amp;author=D.%20Gonzalez&amp;author=E.%20Leire&amp;volume=287&amp;issue=49&amp;publication_year=2012&amp;pages=40891-40897&amp;pmid=23038245&amp;doi=10.1074/jbc.M112.404582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0044">
<span class="label">44.</span><cite>
Sotiropoulou G. and Pampalakis G., “Kallikrein‐Related Peptidases: Bridges Between Immune Functions and Extracellular Matrix Degradation,” Biological Chemistry
391, no. 4 (2010): 321–331.
</cite> [<a href="https://doi.org/10.1515/BC.2010.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20180637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Chemistry&amp;title=Kallikrein%E2%80%90Related%20Peptidases:%20Bridges%20Between%20Immune%20Functions%20and%20Extracellular%20Matrix%20Degradation&amp;author=G.%20Sotiropoulou&amp;author=G.%20Pampalakis&amp;volume=391&amp;issue=4&amp;publication_year=2010&amp;pages=321-331&amp;pmid=20180637&amp;doi=10.1515/BC.2010.036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0045">
<span class="label">45.</span><cite>
Zhang S. K., Song J. W., Gong F., et al., “Design of an Alpha‐Helical Antimicrobial Peptide With Improved Cell‐Selective and Potent Anti‐Biofilm Activity,” Scientific Reports
6 (2016): 27394.
</cite> [<a href="https://doi.org/10.1038/srep27394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4897634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27271216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Design%20of%20an%20Alpha%E2%80%90Helical%20Antimicrobial%20Peptide%20With%20Improved%20Cell%E2%80%90Selective%20and%20Potent%20Anti%E2%80%90Biofilm%20Activity&amp;author=S.%20K.%20Zhang&amp;author=J.%20W.%20Song&amp;author=F.%20Gong&amp;volume=6&amp;publication_year=2016&amp;pages=27394&amp;pmid=27271216&amp;doi=10.1038/srep27394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0046">
<span class="label">46.</span><cite>
Zhang Q. Y., Yan Z. B., Meng Y. M., et al., “Antimicrobial Peptides: Mechanism of Action, Activity and Clinical Potential,” Military Medical Research
8, no. 1 (2021): 48.
</cite> [<a href="https://doi.org/10.1186/s40779-021-00343-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8425997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34496967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Military%20Medical%20Research&amp;title=Antimicrobial%20Peptides:%20Mechanism%20of%20Action,%20Activity%20and%20Clinical%20Potential&amp;author=Q.%20Y.%20Zhang&amp;author=Z.%20B.%20Yan&amp;author=Y.%20M.%20Meng&amp;volume=8&amp;issue=1&amp;publication_year=2021&amp;pages=48&amp;pmid=34496967&amp;doi=10.1186/s40779-021-00343-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="psc70048-bib-0047">
<span class="label">47.</span><cite>
Huang Y., He L., Li G., Zhai N., Jiang H., and Chen Y., “Role of Helicity of Alpha‐Helical Antimicrobial Peptides to Improve Specificity,” Protein &amp; Cell
5, no. 8 (2014): 631–642.
</cite> [<a href="https://doi.org/10.1007/s13238-014-0061-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4130925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24805306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20&amp;%20Cell&amp;title=Role%20of%20Helicity%20of%20Alpha%E2%80%90Helical%20Antimicrobial%20Peptides%20to%20Improve%20Specificity&amp;author=Y.%20Huang&amp;author=L.%20He&amp;author=G.%20Li&amp;author=N.%20Zhai&amp;author=H.%20Jiang&amp;volume=5&amp;issue=8&amp;publication_year=2014&amp;pages=631-642&amp;pmid=24805306&amp;doi=10.1007/s13238-014-0061-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>
<strong>Figure S1.</strong> Mass spectrometry and analytical RP‐HPLC traces. (A) Representative MALDI‐TOF mass spectrometry spectra. (B) Analytical RP‐HPLC traces demonstrate peptide purity, with single peaks indicating high purity. RP‐HPLC was conducted using an Agilent Technologies 1260 Infinity II spectrometer with a reversed‐phase C<sub>18</sub> column at a flow rate of 1.8 mL/min and monitored at 215 nm.</p>
<p>
<strong>Figure S2.</strong> Isopeptide bond‐substitution enhanced the proteolytic stability of Esc(1‐21) in human plasma. HPLC chromatograms of Esc(1‐21), Esc(1‐21)ε12, and Esc(1‐21)ε20 after incubation in human plasma at 37°C for 0, 6, and 24 h. To aid visualization, traces were offset by +1, +2, and +3 min (<em>x</em>‐axis) and +100, +200, and +300 units (<em>y</em>‐axis) for 0, 6, and 24 h, respectively. Peptide separation was performed on a C<sub>18</sub> column over 30 min using a linear gradient of 20%–80% ACN in ddH<sub>2</sub>O containing 0.1% TFA at a flow rate of 0.6 mL/min. Absorbance was monitored at 215 nm, human plasma without peptide served as a blank and the percentage of remaining peptide was quantified based on the decrease in peak area relative to the untreated sample.</p>
<p>
<strong>Figure S3.</strong> CD spectra of Esc1‐21ε5 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S4.</strong> CD spectra of Esc1‐21ε9 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S5.</strong> CD spectra of Esc1‐21ε10 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S6.</strong> CD spectra of Esc1‐21ε12 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Figure S7.</strong> CD spectra of Esc1‐21ε20 in aqueous solution (lipid concentration = 0) and after liposomes addition (lipid concentrations 50, 100, 200, 500, and 1000 μM, peptide concentration 20 μM).</p>
<p>
<strong>Table S1.</strong> Estimated secondary structure of Esc(1‐21) and its analogs in aqueous and LUVs solution.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328261/bin/PSC-31-e70048-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PSC-31-e70048-s001.docx</a><sup> (583.4KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Peptide Science are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/psc.70048"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PSC-31-e70048.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12328261/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12328261/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12328261%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328261/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12328261/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12328261/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40769954/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12328261/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40769954/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12328261/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12328261/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="BdlHsMkFbW2H881dhQO52Mm0aqnnQtpqo5Uq2uWZOQXIKlPg3asrTUvG1UmjBOyL">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
