
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Deep fusion of incomplete multi-omic data for molecular mechanism of Alzheimer’s disease - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BA508AF4113305BA5000390DB94D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361453/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Deep fusion of incomplete multi-omic data for molecular mechanism of Alzheimer’s disease">
<meta name="citation_author" content="Linhui Xie">
<meta name="citation_author_institution" content="Purdue University, Indianapolis, Department of Electrical and Computer Engineering, Indianapolis, 46204 U.S.">
<meta name="citation_author" content="Yash Raj">
<meta name="citation_author_institution" content="Indiana University, Indianapolis, Department of Biomedical Engineering and Informatics, Indianapolis, 46204 U.S.">
<meta name="citation_author" content="Mingzhao Tong">
<meta name="citation_author_institution" content="Indiana University, Indianapolis, Department of Computer Science, Indianapolis, 46204 U.S.">
<meta name="citation_author" content="Kwangsik Nho">
<meta name="citation_author_institution" content="Indiana University School of Medicine, Department of Radiology and Imaging Sciences, Indianapolis, 46204 U.S.">
<meta name="citation_author" content="Paul Salama">
<meta name="citation_author_institution" content="Purdue University, Indianapolis, Department of Electrical and Computer Engineering, Indianapolis, 46204 U.S.">
<meta name="citation_author" content="Andrew J Saykin">
<meta name="citation_author_institution" content="Indiana University School of Medicine, Department of Radiology and Imaging Sciences, Indianapolis, 46204 U.S.">
<meta name="citation_author" content="Shiaofen Fang">
<meta name="citation_author_institution" content="Indiana University, Indianapolis, Department of Computer Science, Indianapolis, 46204 U.S.">
<meta name="citation_author" content="Jingwen Yan">
<meta name="citation_author_institution" content="Indiana University, Indianapolis, Department of Biomedical Engineering and Informatics, Indianapolis, 46204 U.S.">
<meta name="citation_author_institution" content="Indiana University School of Medicine, Department of Radiology and Imaging Sciences, Indianapolis, 46204 U.S.">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30182">
<meta name="citation_doi" content="10.1038/s41598-025-14636-2">
<meta name="citation_pmid" content="40825989">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361453/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361453/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361453/pdf/41598_2025_Article_14636.pdf">
<meta name="description" content="Multi-omics data provides a comprehensive view of biological systems and enables researchers to uncover intricate molecular mechanisms underlying complex diseases. However, multi-omic data is often incomplete and joint modeling of multi-omics data ...">
<meta name="og:title" content="Deep fusion of incomplete multi-omic data for molecular mechanism of Alzheimer’s disease">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Multi-omics data provides a comprehensive view of biological systems and enables researchers to uncover intricate molecular mechanisms underlying complex diseases. However, multi-omic data is often incomplete and joint modeling of multi-omics data ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361453/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12361453">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-14636-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_14636.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12361453%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12361453/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12361453/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361453/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 18;15:30182. doi: <a href="https://doi.org/10.1038/s41598-025-14636-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-14636-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Deep fusion of incomplete multi-omic data for molecular mechanism of Alzheimer’s disease</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Linhui Xie</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Linhui Xie</span></h3>
<div class="p">
<sup>1</sup>Purdue University, Indianapolis, Department of Electrical and Computer Engineering, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Linhui Xie</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raj%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yash Raj</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yash Raj</span></h3>
<div class="p">
<sup>2</sup>Indiana University, Indianapolis, Department of Biomedical Engineering and Informatics, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raj%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yash Raj</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tong%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Mingzhao Tong</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Mingzhao Tong</span></h3>
<div class="p">
<sup>3</sup>Indiana University, Indianapolis, Department of Computer Science, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tong%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingzhao Tong</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nho%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kwangsik Nho</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kwangsik Nho</span></h3>
<div class="p">
<sup>4</sup>Indiana University School of Medicine, Department of Radiology and Imaging Sciences, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nho%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kwangsik Nho</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salama%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Paul Salama</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Paul Salama</span></h3>
<div class="p">
<sup>1</sup>Purdue University, Indianapolis, Department of Electrical and Computer Engineering, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salama%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul Salama</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saykin%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Andrew J Saykin</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Andrew J Saykin</span></h3>
<div class="p">
<sup>4</sup>Indiana University School of Medicine, Department of Radiology and Imaging Sciences, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saykin%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew J Saykin</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Shiaofen Fang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Shiaofen Fang</span></h3>
<div class="p">
<sup>3</sup>Indiana University, Indianapolis, Department of Computer Science, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shiaofen Fang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jingwen Yan</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jingwen Yan</span></h3>
<div class="p">
<sup>2</sup>Indiana University, Indianapolis, Department of Biomedical Engineering and Informatics, Indianapolis, 46204 U.S. </div>
<div class="p">
<sup>4</sup>Indiana University School of Medicine, Department of Radiology and Imaging Sciences, Indianapolis, 46204 U.S. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jingwen Yan</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Purdue University, Indianapolis, Department of Electrical and Computer Engineering, Indianapolis, 46204 U.S. </div>
<div id="Aff2">
<sup>2</sup>Indiana University, Indianapolis, Department of Biomedical Engineering and Informatics, Indianapolis, 46204 U.S. </div>
<div id="Aff3">
<sup>3</sup>Indiana University, Indianapolis, Department of Computer Science, Indianapolis, 46204 U.S. </div>
<div id="Aff4">
<sup>4</sup>Indiana University School of Medicine, Department of Radiology and Imaging Sciences, Indianapolis, 46204 U.S. </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 14; Accepted 2025 Aug 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12361453  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40825989/" class="usa-link">40825989</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Multi-omics data provides a comprehensive view of biological systems and enables researchers to uncover intricate molecular mechanisms underlying complex diseases. However, multi-omic data is often incomplete and joint modeling of multi-omics data will lead to exclusion of a large portion of subjects. Furthermore, most current multi-omics studies pinpoint individual -omics markers, which may not interact, posing challenges for interpretation. In this study, we developed an interpretable deep trans-omic fusion neural network, TransFuse, to include incomplete -omic data for training of prediction models. When evaluated using the data from two Alzheimer’s disease cohorts, TransFuse generally showed superior or comparable performance over competing methods in a wide range of metrics like classification accuracy and F1 score. In addition, TransFuse yielded a subset of multi-omics features forming functional disease network modules, providing valuable insights into underlying molecular mechanism. In addition, almost all the genetic variants identified by TransFuse are expression quantitative trait locus (eQTLs) specific to frontal cortex tissue, from which the gene and protein expression data were collected. This highlights the great potential of TransFuse in capturing the tissue-specific information flow. Top pathways enriched include VEGF and EPH pathways, both influencing neural development and synaptic formation.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-14636-2.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Transfer learning, Multi-omic network, Alzheimer’s Disease, Systems biology, Deep learning</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Computational biology and bioinformatics, Systems biology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Multi-omics data provides a comprehensive view of biological systems by integrating various molecular layers, such as genomics, proteomics, and metabolomics. This comprehensive approach enables researchers to uncover intricate disease mechanisms and identify potential therapeutic targets<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. On the other hand, databases such as Reactome and SNP2TFBS provide critical prior knowledge of regulatory links between SNPs, genes, and proteins. Although not entirely accurate nor tissue specific, this prior knowledge could still be leveraged as an additional source of evidence on top of multi-omics data to improve the precision in discovery of molecular mechanism. Recently, a growing effort has been dedicated to the integrative modeling of multi-omic data together with a rich resource of prior knowledge in biological interactions like protein interactions and biochemical interactions in the pathways. Several studies have verified that machine learning models informed by prior biological interactions can lead to improved prediction performance and model interpretability<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. A subset of those are integrative -omics analysis that jointly modeled multi-omics data and prior biological interactions<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. This approach has yielded functionally connected -omics features, such as genetic variants, genes, and proteins, offering valuable insights into molecular mechanisms and providing potential targets for treatment.</p>
<p id="Par3">However, a major challenge in integrative modeling of multi-omics data is the prevalence of missing data across -omics layers<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In disease cohorts, incomplete multi-omic profiles are common due to various factors such as technical limitations, limited tissue availability, and patient dropout. Most established integrative -omics modeling techniques rely on concatenated multi-omics features and thus require complete multi-omic profiles from all subjects<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. For instance, MOGONET is a graph neural network framework designed to integrate omics-specific subject similarity networks for patient classification<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. In addition, our previous work, MoFNet, is a graph neural network that integrates multi-omics data with prior trans-omic interactions to model the flow of information from DNA to gene and protein. However, these methods necessitate that all subjects have a complete set of omics data, leading to the exclusion of subjects with missing -omics types and restricting analysis to a small subset of the cohort. This limitation reduces the full potential of large multi-omic datasets, thereby diminishing predictive power and the ability to identify robust -omics features associated with complex diseases.</p>
<p id="Par4">While imputation methods can be used to estimate missing values, they typically perform well only when a few values are missing and rely on the presence of partial data and observable patterns within each data type<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. However, when an entire -omics data type is missing, imputation must depend solely on other -omics types, which is often insufficient because shared information across -omics is limited and may fail to capture modality-specific biological signals. Although some methods attempt to predict one -omics type from another<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, such predictions risk amplifying noise or biases present in the observed data, thereby reducing robustness for downstream tasks such as classification or biomarker discovery. Imputation methods for substantial missingness remain underdeveloped and should be used with caution; while they may capture global patterns, they are generally unsuitable for precision tasks like patient classification.</p>
<p id="Par5">In this study, we introduce TransFuse, an interpretable deep trans-omic fusion neural network that to enable the inclusion of subjects with incomplete -omics data during model training, without requiring reconstruction of large missing data chunks. TransFuse integrates multi-omics data with prior knowledge of functional interactions among proteins, genes, and their upstream regulatory SNPs. As an extension of MoFNet, TransFuse adopts a modular network architecture consisting of three separate modules, each dedicated to processing only one omics type (i.e., SNPs, gene expression, or proteins). This modular design allows each component to be pre-trained independently using subjects with missing -omics types. We evaluated TransFuse on multi-omics data from the ROS/MAP cohort using Alzheimer’s disease as a test case. Transfuse fine-tuned from pre-trained modules demonstrated improved/comparable prediction performance compared to other state-of-the-art methods, while also identifying biologically meaningful disease subnetworks associated with Alzheimer’s disease.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Classification performance</h3>
<p id="Par6">Shown in Fig. <a href="#Fig1" class="usa-link">1</a> is the performance metrics for the proposed TransFuse model and other competing classification methods. We reported not only accuracy and AUC, but also F1 score, precision, and specificity metrics to provide a comprehensive comparison of performance. Notably, the F1 score combines precision and sensitivity (recall) into a single metric and is widely utilized as a primary evaluation metric for imbalanced datasets. We found that fine-tuned TransFuse achieved the best average performance in 4 out of 5 evaluation metrics, with consistently low variation across 5 folds (Supplementary Table <a href="#MOESM2" class="usa-link">S1</a>). Following the approach in<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, we conducted paired t-test to assess the significance of this improvement. It showed that fine-tuned TransFuse delivered significantly higher or comparable performance relative to other methods (FDR-corrected p&lt; 0.05; Fig. 1). In terms of accuracy, it significantly outperformed baseline TransFuse, GraphNet-constrained logistic regression, MOGONET, and MLP, and performed marginally better than the Lasso model. For specificity, it significantly outperformed competing methods including baseline TransFuse, MLP, Lasso, GraphNet, and Lasso-constrained logistic regression. Non-significant differences in other metrics are mainly due to high variability in the performance of competing methods, as reflected by large error bars in Fig. <a href="#Fig1" class="usa-link">1</a>. Additionally, we performed paired t-tests to assess whether any competing method significantly outperformed the fine-tuned TransFuse on any metric. All p-values were greater than 0.92 (Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>), indicating that none of the competing methods demonstrated statistically superior performance compared to fine-tuned TransFuse.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361453_41598_2025_14636_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/6d8f0996cf2e/41598_2025_14636_Fig1_HTML.jpg" loading="lazy" id="MO1" height="350" width="750" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance comparison across models on the test sets. FDR corrected pvalues and effect size are calcualted from paired t-test.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Multi-omic sub-networks for AD</h3>
<p id="Par7">Across all methods, Laplacian-logistic and Lasso-logistic selected only a small number of features, with few prior connections (Fig. S3 and S4). Other methods, such as modularity-constrained logistic regression, elastic net, random forest, and MLP, identified a much larger set of features without much prioritization (Fig. S1, S2, S5 and S7). In contrast, TransFuse (Fig.  <a href="#Fig2" class="usa-link">2</a>) and MOGONET (Fig. S6) returned a smaller, more connected set of disease-related modules, which are more manageable for downstream functional interpretation. Fine-tuned TransFuse model identified 20 peptides, 107 genes and 7 SNPs, most of which are part of one big cohesive sub-network and three smaller connected sub-networks (Fig.  <a href="#Fig2" class="usa-link">2</a>). Smaller groups highlighted localized functional interactions within the broader prior network, consisting of important genes and proteins related to AD like the microtubule associated protein tau (<em>MAPT</em>) gene and the tau_PHF1_S404 peptide, which confirmed the important role of tau phosphorylation in neurodegeneration. The paired helical filament 1 (<em>PHF1</em>) antibody targets the specific phosphorylated epitope of tau at Ser396/404. Phosphorylation at this <em>PHF1</em> site disrupts tau’s structural folding, promoting polymerization and contributing to neurofibrillary tangle formation<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361453_41598_2025_14636_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/540e4bb9e8ba/41598_2025_14636_Fig2_HTML.jpg" loading="lazy" id="MO2" height="586" width="714" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TransFuse identified multi-omic network modules associated with AD, highlighting the information flow from SNPs to genes and proteins. Node size is proportional to the importance score. Edge thickness is proportional to the weights learned from TransFuse.</p></figcaption></figure><p id="Par8">The largest sub-network includes 16 peptides, 104 genes, and 5 SNPs. The protein apolipoprotein E (<em>APOE</em>), the top risk factor of Alzheimer’s Disease, was identified within this sub-network, directly connected to the early growth response protein 1 (<em>EGR1</em>) gene. <em>EGR1</em> transcripts were found elevated in <em>APOE</em>-deficient mice, pointing to an inverse relationship between <em>EGR1</em> and <em>APOE</em> levels<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Pathway connecting amyloid precursor protein (<em>APP_2</em>), aph-1 homolog A/B (<em>APH1A</em>/<em>APH1B</em>), cluster of differentiation-44 (<em>CD44_2</em>), and <em>EGR1</em> covers another potential critical aspects of AD. <em>APP</em> processing by the gamma-secretase complex (containing <em>APH1A</em>/<em>APH1B</em>) produces amyloid-beta, a primary driver of AD<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. <em>CD44</em>, a mediator of cell adhesion, plays a complex role in neuroinflammation by controlling T-cell differentiation, adhesion, and blood-brain barrier permeability. In <em>CD44</em>-deficient mice, increased permeability and pro-inflammatory T-cell profiles highlight <em>CD44</em>’s role as a negative regulator of inflammation<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. <em>EGR1</em>, a transcription factor, is involved in synaptic health and is impacted by neuroinflammation and amyloid-beta processing<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, contributing to synaptic dysfunction and cognitive decline. Interactions between <em>APP</em>, <em>APH1A</em>/<em>APH1B</em>, <em>CD44</em>, and <em>EGR1</em> reveal a pathological interaction network which may provide important hypothesis for therapeutic development.</p>
<p id="Par9">Within the largest sub-network, hub peptides like phosphoinositide-3-kinase regulatory subunit 1 (<em>PIK3R1_1</em>) and growth factor receptor bound protein 2 (<em>GRB2_1</em>) exhibit high importance scores, demonstrating their role in integrating signals from upstream genes and other interacting proteins. These peptides are well known for their contribution to neuroinflammation, vascular permeability, and impaired cell signaling in AD<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The largest gene node angiopoietin 2 (<em>ANGPT2</em>) and its connection to hub peptide <em>PIK3R1_1</em> suggest strong functional connectivity in signaling networks. Specifically, <em>PIK3R1</em> modulates neuroinflammation through the PI3K/Akt pathway<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, while <em>ANGPT2</em> promotes vascular permeability and leukocyte infiltration by antagonizing Tie2 receptor signaling. A study showed that blocking <em>ANGPT2</em> improved blood-brain barrier integrity, reducing proinflammatory cell activity and improving central nervous system function<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Expression quantitative trait locus (eQTL) analysis</h3>
<p id="Par10">Considering that gene and protein expression data are both from prefrontal cortex, we further examined the tissue-specificity of the functional effect associated with the identified SNPs. That is, whether these SNPs are expression quantitative trait loci (eQTLs) associated with the downstream transcriptomic changes in the corresponding tissue. Based on the Brain eQTL Almanac (BRAINEAC) database, we found that 5 out of the 7 identified SNPs were significant eQTLs in the frontal cortex (Table. <a href="#Tab1" class="usa-link">1</a>) and one SNP, <em>rs10135521</em>, could not be found in the database. The remaining SNP <em>rs9835340</em> returned a marginal eQTL effect with adjusted p-value of 0.053.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of eQTL analysis for the identified SNPs and their associated genes in BRAINEAC database.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Chromosome</th>
<th align="left" colspan="1" rowspan="1">Gene</th>
<th align="left" colspan="1" rowspan="1">SNP</th>
<th align="left" colspan="1" rowspan="1">Adjusted p-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">
<em>PPIL5</em>,<em>RHOQ</em>
</td>
<td align="left" colspan="1" rowspan="1">rs3126179</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/4753043ebe7e/d33e1002.gif" loading="lazy" id="d33e1002" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1"><em>PDE7B</em></td>
<td align="left" colspan="1" rowspan="1">rs12215787</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/40264eb46c07/d33e1018.gif" loading="lazy" id="d33e1018" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1"><em>IQCG</em></td>
<td align="left" colspan="1" rowspan="1">rs9873700</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/e3b66018f291/d33e1034.gif" loading="lazy" id="d33e1034" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"><em>ST7L</em></td>
<td align="left" colspan="1" rowspan="1">rs11102304</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/32dd1fc54e06/d33e1050.gif" loading="lazy" id="d33e1050" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"><em>LOC100129269</em></td>
<td align="left" colspan="1" rowspan="1">rs2789535</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/6ab2f9256ced/d33e1066.gif" loading="lazy" id="d33e1066" alt="Inline graphic"></span></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec6"><h3 class="pmc_sec_title">Pathway enrichment analysis</h3>
<p id="Par11">Pathway enrichment analysis was performed on the largest sub-network of Fig.  <a href="#Fig2" class="usa-link">2</a>, which includes 16 peptides, 104 genes and 5 SNPs, using the g:Profiler platform. We identified several clusters of enriched functional pathways, like vascular endothelial growth factor (VEGF) signaling, erythropoietin-producing hepatocellular (EPH) signaling, cell communication and transcriptional regulation (Fig.  <a href="#Fig3" class="usa-link">3</a>). The largest cluster of enriched pathways is VEGF signaling pathways. This signaling family is involved in neuroprotection and speculated to contribute to the neurodegenerative process in AD<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Some research indicates that VEGF levels are decreased in the cerebrospinal fluid and brain tissue of Alzheimer patients, which might reflect impaired angiogenesis and neuroprotective functions<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. In addition, our enrichment results highlighted the important role of EPH signaling pathway in AD (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/ee2e2530da6d/d33e490.gif" loading="lazy" id="d33e490" alt="Inline graphic"></span>). The EPH signaling pathway, involving EPH receptors and ephrin ligands, is crucial for synaptic function and plasticity, impacting synapse formation and maintenance. Dysregulation of this pathway can lead to synaptic dysfunction and loss, which are the key features of AD<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Additionally, EPH signaling may influence amyloid-beta production and tau pathology, contributing to the progression of AD<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361453_41598_2025_14636_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/b18f3a64d5f9/41598_2025_14636_Fig3_HTML.jpg" loading="lazy" id="MO3" height="421" width="750" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clusters of top pathways enriched by the largest multi-omic sub-network identified in TransFuse. Displayed are the most representative enriched terms for each cluster, with edge thickness indicating the number of shared genes and node color for FDR corrected p-values.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Ablation analysis</h3>
<p id="Par12">In the ablation analysis, we evaluated the contribution of each -omics type to the final performance of the fine-tuned TransFuse model. Specifically, we tested six different input configurations by either removing one -omics type at a time (by setting its input values to zero) or retaining only one single -omics type. We used paired t-tests with FDR correction to assess the statistical significance of performance differences across these configurations (Supplementary Figure <a href="#MOESM1" class="usa-link">S8</a>). When removing individual -omics types, we observed that omitting SNP or gene expression data led to only a slight decrease in performance. In contrast, removing protein data resulted in a significant drop, underscoring its critical importance to the predictive power of TransFuse. Similarly, among the single-omics models, the protein-only configuration consistently outperformed the SNP-only and gene-only models. Notably, the protein-only model achieved performance levels close to that of the full multi-omics model, further emphasizing the dominant role of protein data in this task. This finding is biologically plausible, as proteins are the functional products of gene expression and genetic variation, and thus may inherently encapsulate much of the downstream disease-relevant information. Our TransFuse model captures the information flow from SNP to gene and protein that likely contributes to the strong performance of protein features.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Replication analysis</h3>
<p id="Par13">After module pre-training using ROS/MAP samples with missing -omics types, the learned weights from the three pre-trained modules were used to initialize the corresponding components of the TransFuse model, while the fully connected layers were initialized with random weights. The fine-tuning process was performed separately using complete samples from the ROS/MAP and MSBB cohorts, allowing TransFuse to adapt independently to each dataset. TransFuse model fine-tuned using MSBB complete samples identified 3 SNPs, 64 genes, and 21 peptides associated with Alzheimer’s disease, among which 1 SNP, 17 genes, and 8 peptides overlapped with those previously identified in the ROS/MAP cohort.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par14">We proposed a new deep multi-omic fusion model that models the dynamic information flow from DNA to RNA and protein, and at the same time enables the training using subjects with missing -omics types. TransFuse, after fine-tuning, demonstrated significantly improved performance over a wide range of performance metrics. Compared to other competing methods, it also yielded a subset of multi-omics features with functional interactions mostly known in the prior knowledge, with reasonable success when replicated in an independent cohort. In addition, almost all the SNPs identified by TransFuse are eQTLs specific to frontal cortex tissue, from which the gene expression and protein expression data were collected. This highlights the great potential of TransFuse in capturing tissue-specific information flow from SNP to RNA. Our findings together underscore the important role of the VEGF and EPH pathways in AD. Both pathways influence neural development, with VEGF promoting neuron growth and survival, while Eph signaling is involved in axon guidance and synaptic formation. Their crosstalk is crucial in coordinating neural functions, and dysregulation of these pathways could contribute to neurological disorders, including Alzheimer’s disease.</p>
<p id="Par15">There are several limitations of this work that merit further consideration. The prior SNP-gene interactions used in this study are limited to promoter regions. However, many of the AD risk SNPs identified in the genome wide association studies (GWASs) are intergenic and located in non-coding regions. Therefore, our prior network limits the capability of TransFuse to connect AD risk SNPs with AD-related proteins. Instead, it only searches for upstream regulatory SNPs nearby candidate genes, which likely lead to the small number of identified SNPs in both discovery and replication analyses. This could be improved by incorporating emerging high-throughput chromosome conformation capture (Hi-C) data, which provide distant SNP-gene interactions and will likely reach intergenic regions with AD GWAS findings.Furthermore, while TransFuse achieved the highest average F1 score, its effectiveness in handling imbalanced data remains unclear, as the dataset in this study is not highly imbalanced. A more thorough evaluation would require testing on more imbalanced data when it becomes available.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Multi-omic data</h3>
<p id="Par16">Multi-omics data used this study were downloaded from a cohort of Alzheimer’s disease (AD), the Religious Orders Study (ROS) and Memory and Aging Project (MAP) cohort<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Multi-omic data of 1717 participants were downloaded from the AMP-AD portal, including imputed genotypes, pre-processed RNA-Seq gene expression, and protein expression data, along with diagnostic information. RNA-seq gene expression and protein expression data were collected from the prefrontal cortex region of postmortem brains. Table <a href="#Tab2" class="usa-link">2</a> outlines the demographics of the study participants, including the breakdown of participants with cognitive normals (CN) and AD across all -omics data types. This dataset consists a substantial number of subjects with brain tissue data, totaling 683 CN and 581 AD in the genomics group, 354 CN and 268 AD in the transcriptomics group, and 641 CN and 541 AD in the proteomics group. Among those 1717 subjects, only 464 of them have complete sets of -omics types including 263 CN and 201 AD cases. Considering that information flow from SNPs to genes and proteins, this rich multi-omics dataset allows for a multi-layered analysis of the genetic and molecular factors associated with AD. Detailed information regarding the ROS/MAP cohort, and processing steps of genotype, gene expression and protein expression can be found in the <a href="#MOESM3" class="usa-link">supplementary text</a>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Demographic information of ROS/MAP participants in the discovery study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="6" rowspan="1">Individual -omics</th>
<th align="left" colspan="2" rowspan="1">ROS/MAP full -omics</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="2" rowspan="1">ROS/MAP Genotype</th>
<th align="left" colspan="2" rowspan="1">ROS/MAP RNA-Seq</th>
<th align="left" colspan="2" rowspan="1">ROS/MAP Protein</th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"> Diagnosis</th>
<th align="left" colspan="1" rowspan="1">CN</th>
<th align="left" colspan="1" rowspan="1">AD</th>
<th align="left" colspan="1" rowspan="1">CN</th>
<th align="left" colspan="1" rowspan="1">AD</th>
<th align="left" colspan="1" rowspan="1">CN</th>
<th align="left" colspan="1" rowspan="1">AD</th>
<th align="left" colspan="1" rowspan="1">CN</th>
<th align="left" colspan="1" rowspan="1">AD</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Subject Number</td>
<td align="left" colspan="1" rowspan="1">683</td>
<td align="left" colspan="1" rowspan="1">581</td>
<td align="left" colspan="1" rowspan="1">354</td>
<td align="left" colspan="1" rowspan="1">268</td>
<td align="left" colspan="1" rowspan="1">641</td>
<td align="left" colspan="1" rowspan="1">541</td>
<td align="left" colspan="1" rowspan="1">263</td>
<td align="left" colspan="1" rowspan="1">201</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Male/Female</td>
<td align="left" colspan="1" rowspan="1">235/448</td>
<td align="left" colspan="1" rowspan="1">183/398</td>
<td align="left" colspan="1" rowspan="1">132/222</td>
<td align="left" colspan="1" rowspan="1">91/177</td>
<td align="left" colspan="1" rowspan="1">218/423</td>
<td align="left" colspan="1" rowspan="1">160/381</td>
<td align="left" colspan="1" rowspan="1">98/165</td>
<td align="left" colspan="1" rowspan="1">68/133</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Education (mean ± std.)</td>
<td align="left" colspan="1" rowspan="1">16.4 ± 3.6</td>
<td align="left" colspan="1" rowspan="1">16.3 ± 3.7</td>
<td align="left" colspan="1" rowspan="1">16.4 ± 3.5</td>
<td align="left" colspan="1" rowspan="1">16.5 ± 3.5</td>
<td align="left" colspan="1" rowspan="1">16.2 ± 3.5</td>
<td align="left" colspan="1" rowspan="1">16.2 ± 3.6</td>
<td align="left" colspan="1" rowspan="1">16.3 ± 3.4</td>
<td align="left" colspan="1" rowspan="1">16.5 ± 3.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Age (mean ± std.)</td>
<td align="left" colspan="1" rowspan="1">88.1 ± 7.3</td>
<td align="left" colspan="1" rowspan="1">90.9 ± 6.0</td>
<td align="left" colspan="1" rowspan="1">87.4 ± 6.5</td>
<td align="left" colspan="1" rowspan="1">90.5 ± 6.3</td>
<td align="left" colspan="1" rowspan="1">88.5 ± 6.5</td>
<td align="left" colspan="1" rowspan="1">90.7 ± 6.1</td>
<td align="left" colspan="1" rowspan="1">87.7 ± 6.4</td>
<td align="left" colspan="1" rowspan="1">90.5 ± 6.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PMI (mean ± std.)</td>
<td align="left" colspan="1" rowspan="1">9.4 ± 8.8</td>
<td align="left" colspan="1" rowspan="1">8.7 ± 6.8</td>
<td align="left" colspan="1" rowspan="1">7.5 ± 4.9</td>
<td align="left" colspan="1" rowspan="1">7.1 ± 4.7</td>
<td align="left" colspan="1" rowspan="1">8.7 ± 6.3</td>
<td align="left" colspan="1" rowspan="1">8.0 ± 5.2</td>
<td align="left" colspan="1" rowspan="1">7.3 ± 4.7</td>
<td align="left" colspan="1" rowspan="1">7.4 ± 5.0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Pre-filtering of -omics features</h3>
<p id="Par17">This multi-omics analysis was designed to target the molecular mechanisms underlying 186 peptides measured in the ROS/MAP study, selected for their high relevance to AD<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Similar to our previous research, we employed a bottom-up approach to pre-filter SNPs and genes, which is also expected to mitigate the effect of small sample size<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Specifically, we mapped the 186 peptides to 126 unique genes (gene set A), which demonstrated functional interactions with 954 genes (gene set B) in the Reactome database. Among these 1080 (126 + 954) genes, 743 of them with available RNA-seq data were included in the subsequent analysis. Upstream SNPs within 5K base pairs of these genes were identified, and further filtered to include only those significantly associated with transcription factor-binding activity as per the SNP2TFBS database. This resulted in a comprehensive dataset comprising 822 SNPs, 743 genes, and 186 peptides (Fig. <a href="#Fig4" class="usa-link">4</a>a). The functional relationships used to filter the genes and SNPs formed a prior multi-omic network, and will be embedded into the modules of TransFuse to guide the search for molecular sub-networks related to AD.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12361453_41598_2025_14636_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/26f74f450ee0/41598_2025_14636_Fig4_HTML.jpg" loading="lazy" id="MO4" height="730" width="732" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of study cohorts and TransFuse architecture. (<strong>a</strong>) Pre-filtered multi-omic data collection in the ROS/MAP and MSBB cohorts. The image of brain surface was generated using BrainNet Viewer 1.7 in Matlab 2022b. (<strong>b</strong>) Architecture of complete TransFuse model. Shaded areas are pre-trainable modules that require input of only single -omics type.</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Prediction outcome</h3>
<p id="Par18">The SNP genotype, gene expression, and protein expression data were extracted and utilized for the classification of AD patients from cognitive normal (CN) individuals. The clinical diagnosis of all participants at the time of brain tissue collection was used as the indicator of their disease status. Notably, the diagnosis time aligns with the time of data collection for the -omics datasets.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Architecture of TransFuse</h3>
<p id="Par19">The proposed deep multi-omic fusion model is a graph neural network that enables transfer learning with a modular architecture, namely TransFuse. It is composed of 3 primary modules, each modeling one type of information flow or information integration (Fig. <a href="#Fig4" class="usa-link">4</a>b). Module 1 takes genotype of pre-filtered SNPs as input and output nodes are pseudogenes, each corresponding to one pre-filtered gene. Links in module 1 are simply the SNP-gene connections in the prior network. Each pseudo-gene in module 1 integrates information from upstream SNPs of corresponding gene. Module 2 takes expression of pre-filtered genes as input and output nodes are pseudo-proteins, each corresponding to one input protein. Links in module 2 are gene-protein functional interactions in the prior network. Each pseudo-protein in module 2 integrates information from interacting genes and then integrated with the output of module 1. Module 3 takes the expression of pre-filtered proteins as input and output nodes are pseudo-proteins, same as the output of module 2. Each pseudo-protein in this module integrates information from connected proteins, including itself. Finally, output from these modules will be integrated and passed to the fully connected layers. </p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par20"><strong>Input layers:</strong> Input data was separated into 3 modalities as SNP genotype, gene expression and protein expression, denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/4059d55e8c9f/d33e572.gif" loading="lazy" id="d33e572" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/2f4c5b39433d/d33e578.gif" loading="lazy" id="d33e578" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/79e84a0f2833/d33e584.gif" loading="lazy" id="d33e584" alt="Inline graphic"></span> respectively. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/b446deaaddaf/d33e590.gif" loading="lazy" id="d33e590" alt="Inline graphic"></span> is the number of subjects.</p></li>
<li><p id="Par21"><strong>Graph fusion layers:</strong> Three graph fusion layers (i.e., modules) were directly wired with prior biological interactions to encode functional connectivity among SNPs, genes, and proteins. For example, the adjacency matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/914bd2186f68/d33e603.gif" loading="lazy" id="d33e603" alt="Inline graphic"></span> represents interactions between proteins, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/b45a6461a025/d33e609.gif" loading="lazy" id="d33e609" alt="Inline graphic"></span> if protein <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/a086866e2160/d33e615.gif" loading="lazy" id="d33e615" alt="Inline graphic"></span> has interactions with protein <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/c4ddf470d238/d33e621.gif" loading="lazy" id="d33e621" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/f58c88ff4727/d33e627.gif" loading="lazy" id="d33e627" alt="Inline graphic"></span> otherwise. Self-connections within proteins are denoted by an identity matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/e7812d7e7c0a/d33e634.gif" loading="lazy" id="d33e634" alt="Inline graphic"></span>. The output of module 3, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/5b36ec837965/d33e640.gif" loading="lazy" id="d33e640" alt="Inline graphic"></span>, is computed as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/121c0aa18400/d33e646.gif" loading="lazy" id="d33e646" alt="Inline graphic"></span> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/19a719214a21/d33e652.gif" loading="lazy" id="d33e652" alt="Inline graphic"></span> represents the Hadamard product for element-wise multiplication of two matrices, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/7cc0292d73f8/d33e658.gif" loading="lazy" id="d33e658" alt="Inline graphic"></span> is the bias term. The matrices <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/3de7bb95bb10/d33e664.gif" loading="lazy" id="d33e664" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/f590f2cc2c1a/d33e671.gif" loading="lazy" id="d33e671" alt="Inline graphic"></span> encode connections between SNPs and genes, and between genes and proteins respectively. Similarly, for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/7d1e211ba589/d33e677.gif" loading="lazy" id="d33e677" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/e0c3ac14b4de/d33e683.gif" loading="lazy" id="d33e683" alt="Inline graphic"></span>.</p></li>
<li><p id="Par22"><strong>Graph bridge Layer:</strong> The graph bridge layer is designed to integrate the outputs from modules 1 and 2, i.e., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/d7b75ac3c0ac/d33e696.gif" loading="lazy" id="d33e696" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/ac479f2736ea/d33e702.gif" loading="lazy" id="d33e702" alt="Inline graphic"></span>. These outputs are concatenated and fed into the graph bridge layer with the structure used in the fusion layers. A combined adjacency matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/350c83e591ad/d33e708.gif" loading="lazy" id="d33e708" alt="Inline graphic"></span> is derived from the concatenation of gene-protein interactions <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/f590f2cc2c1a/d33e714.gif" loading="lazy" id="d33e714" alt="Inline graphic"></span> and the self-connections among proteins <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/b742a047e1c2/d33e720.gif" loading="lazy" id="d33e720" alt="Inline graphic"></span>, i.e., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/9937ddea9c58/d33e727.gif" loading="lazy" id="d33e727" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/fb52e2e1af52/d33e733.gif" loading="lazy" id="d33e733" alt="Inline graphic"></span> stands for row concatenation. The output of the bridge layer, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/978bfeb6551e/d33e739.gif" loading="lazy" id="d33e739" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/7d68b4f99dd2/d33e745.gif" loading="lazy" id="d33e745" alt="Inline graphic"></span> represents the weight matrix specific to the bridge layer and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/00217bb22f8a/d33e751.gif" loading="lazy" id="d33e751" alt="Inline graphic"></span> is the bias term in this layer.</p></li>
<li><p id="Par23"><strong>Fully connected layers:</strong> the architecture incorporates several fully connected layers for disease status classification, with dropout regularization to prevent overfitting. The initial fully connected layer takes the concatenated outputs from module 3 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/5b36ec837965/d33e764.gif" loading="lazy" id="d33e764" alt="Inline graphic"></span>) and graph bridge layer (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/602caf5c82be/d33e770.gif" loading="lazy" id="d33e770" alt="Inline graphic"></span>) as its input. Following this first layer, two additional fully connected layers further process the features before reaching the final prediction layer. The output of the last layer, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/d0edda76f254/d33e776.gif" loading="lazy" id="d33e776" alt="Inline graphic"></span>, uses a sigmoid activation function to classify the samples: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/782aa9ee1950/d33e782.gif" loading="lazy" id="d33e782" alt="Inline graphic"></span> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/6cee83f5ffe0/d33e788.gif" loading="lazy" id="d33e788" alt="Inline graphic"></span> indicates the layer number of the final prediction layer, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/e0013f9151eb/d33e795.gif" loading="lazy" id="d33e795" alt="Inline graphic"></span> stands for the weight of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/6cee83f5ffe0/d33e801.gif" loading="lazy" id="d33e801" alt="Inline graphic"></span>-th layer. The binary cross-entropy loss, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/ba9a29679b63/d33e807.gif" loading="lazy" id="d33e807" alt="Inline graphic"></span> is used to measure the classification error across all input samples.</p></li>
</ol></section><section id="Sec15"><h3 class="pmc_sec_title">Omics-specific modules</h3>
<p id="Par24">Each module or graph fusion layer requires the input of only one -omics type, and therefore can be pre-trained using subjects without complete -omics types. Specifically, we pre-trained three graph neural networks for binary classification, each composed of a graph fusion layer followed with fully connected layers. Each single-modality graph neural network was pre-trained independently using subjects from each -omics layer, and only subjects with missing -omics types were included for pre-training. For each modality-specific graph neural network, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/ca37f8fb11b3/d33e817.gif" loading="lazy" id="d33e817" alt="Inline graphic"></span>, the loss function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/588fd6da5b14/d33e823.gif" loading="lazy" id="d33e823" alt="Inline graphic"></span> is defined using binary cross-entropy, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/e65fc859d7d4/d33e829.gif" loading="lazy" id="d33e829" alt="Inline graphic"></span> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/e268b352f1bb/d33e835.gif" loading="lazy" id="d33e835" alt="Inline graphic"></span> stands for the subjects available for pre-training of each modality, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/fd4d81d1438b/d33e841.gif" loading="lazy" id="d33e841" alt="Inline graphic"></span> represents the predicted diagnosis. The loss functions for SNP-, gene- and protein-specific neural networks were denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/15bf9489d485/d33e848.gif" loading="lazy" id="d33e848" alt="Inline graphic"></span> respectively.</p>
<p id="Par25">Taking SNP-specific graph neural network as an example, the optimization problem is formulated as,</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/e97ed86d745f/d33e856.gif" loading="lazy" id="d33e856" alt="graphic file with name d33e856.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/67dfdcc643fd/d33e862.gif" loading="lazy" id="d33e862" alt="Inline graphic"></span> represents all the weights in the SNP-specific graph neural network, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/425cbc65effb/d33e868.gif" loading="lazy" id="d33e868" alt="Inline graphic"></span> is the regularization parameter. Since the prior network is not completely accurate and lacks tissue specificity, not all prior interactions hold the same importance. For instance, certain interactions may not occur in brain tissues, and thus, the corresponding information flow should not be modeled with brain multi-omics data. To address this, we employ a penalty term <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/9e82f751509f/d33e874.gif" loading="lazy" id="d33e874" alt="Inline graphic"></span> to enforce zero weight on prior connections in the graph fusion layers, which helps exclude functional connections irrelevant to brain expression data and disease outcomes. The goal is to minimize the loss <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/a6bce05642e3/d33e880.gif" loading="lazy" id="d33e880" alt="Inline graphic"></span> while promoting sparsity in the connections of the graph fusion layer. Weights from the graph fusion layers of all modality-specific neural networks are then transferred to the TransFuse model. Initially, these transferred weights were frozen to preserve the learned representations and stabilize the early phase of training on the complex integrated data. Later, all the modules were unfrozen, and the entire TransFuse model was fine-tuned using 464 subjects with a complete set of -omics types.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Parameter optimization</h3>
<p id="Par26">During the module pre-training stage, three omics-specific graph neural networks–for proteins, genes, and SNPs–were independently trained using the Adam optimizer<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, with optimal hyperparameters selected via a grid search. The predefined hyperparameter ranges included: dropout rate (0.1, 0.3, 0.5, 0.7), L1 regularization (0.0001, 0.0003, 0.0005, 0.0008, 0.0010, 0.0030, 0.0050), initial learning rate (0.0002, 0.0004, 0.0006, 0.0008, 0.0010, 0.0015, 0.0020), and weight decay (0.0001, 0.0002). The best-performing configuration included a dropout rate of 0.5, L1 regularization of 0.0005, an initial learning rate of 0.0006, and weight decay of 0.0002. The resulting modality-specific networks achieved prediction accuracies of 0.75 for proteins, 0.80 for genes, and 0.59 for SNPs. The architectures of the final two fully connected layers in each network were set to (186, 16) for proteins, (186, 32) for genes, and (743, 48) for SNPs.</p>
<p id="Par27">During the TransFuse fine-tuning stage, the learned weights from the three pre-trained modules were used to initialize the corresponding parts of TransFuse, while the fully connected layers were initialized with random weights. Fine-tuning was performed separately on the ROS/MAP and MSBB datasets using the same training pipeline. Initially, three modules in TransFuse were frozen, and only the fully connected layers and inter-module connections were trained using a relatively high learning rate of 0.0015. After this phase, the modules were unfrozen, and the entire network was fine-tuned using a smaller learning rate of 0.00006, selected from the range (0.00002, 0.00004, 0.00006, 0.00008). Early stopping and dropout (rate = 0.5) were applied to prevent overfitting. TransFuse was trained for up to 100 epochs, with early stopping triggered if validation loss failed to improve for 10 consecutive epochs or if training loss increased while validation loss decreased. The final two fully connected layers had dimensions of (186, 64) and (64, 1), respectively. All training was conducted in a Google Colab environment using an L4 GPU, PyTorch version 2.2.1, and CUDA version cu121.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Performance comparison</h3>
<p id="Par28">We compared the performance of TransFuse against MOGONET, vanilla neural network Multilayer Perceptron (MLP), random forest and three other logistic regression based classification models, using modularity, elastic net, and Lasso as penalty terms respectively<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. These sparse logistic regression models were selected because they are designed for both classification and feature selection. Modularity and GraphNet constrained logistic regression (M-logistic) were implemented using Matlab. Elastic net constrained logistic regression, traditional logistic regression with lasso penalty, and random forest were implemented using the Python scikit-learn package. In addition, we evaluated the performance of both the baseline and fine-tuned TransFuse models. In the baseline model, initial weights were randomly assigned, whereas the fine-tuned model used weights initialized from the pre-trained modules. The ROS/MAP cohort includes 1,717 subjects, of whom 464 with complete multi-omics data were split into five folds for training and testing the baseline TransFuse and competing methods. The remaining subjects, missing at least one -omics type, were used for pre-training the individual modules. Classification accuracy and the Area Under the Receiver Operating Characteristic Curve (AUC) served as the primary evaluation metrics, complemented by F1 score, precision, and specificity for a more comprehensive performance assessment.</p>
<p id="Par29">To ensure a robust and fair comparison, all methods underwent one single 5-fold cross-validation procedure for hyperparameter tuning and training, with identical training and test partitions. This approach was chosen because the limited sample size made nested cross-validation impractical, as smaller training folds led to unstable performance estimates. While we acknowledge that this may introduce some optimistic bias in the performance metrics, all competing methods were evaluated using the same single cross-validation strategy. Therefore, the comparisons between methods and the resulting conclusions should be fair and consistent. All methods were provided with the same input features, including genotype data for 822 SNPs, expression levels of 743 genes, and 186 peptides. Hyperparameters for all methods were carefully tuned across a comprehensive search space, detailed in Supplementary Table <a href="#MOESM2" class="usa-link">S3</a>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Model interpretation</h3>
<p id="Par30">TransFuse was interpreted in two ways. First, with the application of the L1 penalty, most links in the graph fusion layers or modules will receive zero coefficients. Since these links are uniquely mapped to the prior network, the weights within these modules can be used to prune the prior network. Second, each node will obtain an importance score using integrated gradient<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. This importance score measures the impact of a unit change in that node on the prediction outcome, providing a relative measure to prioritize the contribution of SNPs, genes, and proteins. The cut-off threshold for pruning prior network edges is set at 0.00003, which was determined by identifying the point at which there is a significant increase in the number of connected components. This is based on the assumption that the top multi-omic features should be primarily functionally connected in a few large connected components, while less important ones are selected randomly and emerge as many small connected components or individual nodes. For other competing methods, features assigned a non-zero weight are regarded as contributing features.</p>
<p id="Par31">Finally, we performed an ablation analysis to evaluate the relative contribution of each -omics type within the fine-tuned TransFuse model. In this analysis, we systematically removed or retained one -omics type at a time and recorded the corresponding changes in model performance. This approach enabled us to assess the unique predictive value each data type contributes to the overall effectiveness of TransFuse.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Replication analysis</h3>
<p id="Par32">The replication analysis utilized genotype, RNA-Seq gene expression, and protein expression data from the Mount Sinai Brain Bank (MSBB) cohort. Due to differences in protein quantification methods between the MSBB and the initial ROS/MAP cohort, direct peptide matching was not feasible. Consequently, the study included all the peptides corresponding to the 126 unique proteins previously identified in ROS/MAP. Finally, a total of 107 peptides, 648 genes, and 695 SNPs across 159 MSBB participants were included. Detailed information regarding MSBB cohort, and processing steps of genotype, gene expression and protein expression can be found in the <a href="#MOESM3" class="usa-link">supplementary text</a>.</p></section></section><section id="Sec20"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12361453/bin/41598_2025_14636_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (2.7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12361453/bin/41598_2025_14636_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (14.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12361453/bin/41598_2025_14636_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (52.2KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This research was supported by NIH grants R01 AG081951, R21 AG072101, U19 AG074879, U01 AG068057, and NSF 2345235, 1942394. The results published here are in whole or in part based on data obtained from the AD Knowledge Portal. <strong>ROS/MAP</strong>: Study data was provided by the Rush Alzheimer’s Disease Center, Rush University Medical Center, Chicago. Data collection was supported through funding by NIA grants P30AG10161, R01AG15819, R01AG17917, R01AG30146, R01AG36836, U01AG32984, U01AG46152, the Illinois Department of Public Health, and the Translational Genomics Research Institute.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.X.: Conceptualization, Methodology, Visualization, Formal analysis, Validation, Writing - original draft, Writing review &amp; editing. Y.R.: Methodology, Formal analysis, Validation. M.T.: Methodology, Formal analysis, Validation. K.N.: Data curation, Writing – review &amp; editing. P.S.: Methodology, Supervision, Writing – review &amp; editing. S.F.: Supervision, Writing – review &amp; editing. A.S.: Data curation, Resource, Writing – review &amp; editing. J.Y.: Conceptualization, Methodology, Visualization, Writing - original draft, Writing review &amp; editing, Supervision, Funding acquisition. All authors reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The source code is freely available through GitHub (<a href="https://github.com/JW-Yan/TransFuse" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/JW-Yan/TransFuse</a>). The data underlying this article is available via the AD Knowledge Portal (<a href="https://adknowledgeportal.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://adknowledgeportal.org</a>). The AD Knowledge Portal is a platform for accessing data, analyses, and tools generated by the Accelerating Medicines Partnership (AMP-AD) Target Discovery Program and other National Institute on Aging (NIA)-supported programs to enable open-science practices and accelerate translational learning. The data, analyses and tools are shared early in the research cycle without a publication embargo on secondary use. Data is available for general research use according to the following requirements for data access and data attribution (https://adknowledgeportal.synapse.org/Data%20Access). Data downloaded for the proposed analysis include: ROS/MAP protein expression (<a href="https://www.synapse.org/Synapse:syn21448467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn21448467</a>), ROS/MAP gene expression (<a href="https://www.synapse.org/Synapse:syn8456704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn8456704</a>), ROS/MAP genetic variants (<a href="https://www.synapse.org/Synapse:syn3221153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn3221153</a>), MSBB protein expression (<a href="https://www.synapse.org/Synapse:syn6100410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn6100410</a>), MSBB gene expression (<a href="https://www.synapse.org/Synapse:syn7391833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn7391833</a>), MSBB genetic variants (<a href="https://www.synapse.org/Synapse:syn11707204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn11707204</a>).</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par36">Dr. Saykin has received support from Avid Radiopharmaceuticals, a subsidiary of Eli Lilly (in kind contribution of PET tracer precursor) and holds advisory roles with Siemens Medical Solutions USA, Inc., NIH NHLBI, and Eisai. His editorial commitments include serving as Editor-in-Chief for the journal “Brain Imaging and Behavior”, and he participates in various NIH/NIA advisory committees. All the remaining authors declare no conflict of interest.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Linhui Xie, Yash Raj and Mingzhao Tong: These authors contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hasin, Y., Seldin, M. &amp; Lusis, A. Multi-omics approaches to disease. <em>Genome Biol.</em><strong>18</strong>, 1–15 (2017).
</cite> [<a href="https://doi.org/10.1186/s13059-017-1215-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5418815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28476144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hasin,%20Y.,%20Seldin,%20M.%20&amp;%20Lusis,%20A.%20Multi-omics%20approaches%20to%20disease.%20Genome%20Biol.18,%201%E2%80%9315%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Fortelny, N. &amp; Bock, C. Knowledge-primed neural networks enable biologically interpretable deep learning on single-cell sequencing data. <em>Genome Biol.</em><strong>21</strong>, 1–36 (2020).</cite> [<a href="https://doi.org/10.1186/s13059-020-02100-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7397672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32746932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fortelny,%20N.%20&amp;%20Bock,%20C.%20Knowledge-primed%20neural%20networks%20enable%20biologically%20interpretable%20deep%20learning%20on%20single-cell%20sequencing%20data.%20Genome%20Biol.21,%201%E2%80%9336%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Chandrashekar, P. B. et al. Deepgami: deep biologically guided auxiliary learning for multimodal integration and imputation to improve genotype-phenotype prediction. <em>Genome Med.</em><strong>15</strong>, 88 (2023).
</cite> [<a href="https://doi.org/10.1186/s13073-023-01248-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10617196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37904203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chandrashekar,%20P.%20B.%20et%20al.%20Deepgami:%20deep%20biologically%20guided%20auxiliary%20learning%20for%20multimodal%20integration%20and%20imputation%20to%20improve%20genotype-phenotype%20prediction.%20Genome%20Med.15,%2088%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Xie, L., He, B., Varathan, P. et al. Integrative-omics for discovery of network-level disease biomarkers: a case study in alzheimer’s disease. <em>Briefings in bioinformatics</em><strong>22</strong>, bbab121 (2021).</cite> [<a href="https://doi.org/10.1093/bib/bbab121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8574309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33971669/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Flores, J. E. et al. Missing data in multi-omics integration: Recent advances through artificial intelligence. <em>Front. Artif. Intell.</em><strong>6</strong>, 1098308 (2023).
</cite> [<a href="https://doi.org/10.3389/frai.2023.1098308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9949722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36844425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flores,%20J.%20E.%20et%20al.%20Missing%20data%20in%20multi-omics%20integration:%20Recent%20advances%20through%20artificial%20intelligence.%20Front.%20Artif.%20Intell.6,%201098308%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wang, T. et al. Mogonet integrates multi-omics data using graph convolutional networks allowing patient classification and biomarker identification. <em>Nat. Commun.</em><strong>12</strong>, 3445 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23774-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8187432/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34103512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20T.%20et%20al.%20Mogonet%20integrates%20multi-omics%20data%20using%20graph%20convolutional%20networks%20allowing%20patient%20classification%20and%20biomarker%20identification.%20Nat.%20Commun.12,%203445%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Thung, K.-H. et al. Neurodegenerative disease diagnosis using incomplete multi-modality data via matrix shrinkage and completion. <em>NeuroImage</em><strong>91</strong>, 386–400 (2014).
</cite> [<a href="https://doi.org/10.1016/j.neuroimage.2014.01.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4096013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24480301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thung,%20K.-H.%20et%20al.%20Neurodegenerative%20disease%20diagnosis%20using%20incomplete%20multi-modality%20data%20via%20matrix%20shrinkage%20and%20completion.%20NeuroImage91,%20386%E2%80%93400%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hastie, T. et al. Imputing missing data for gene expression arrays. Technical report, Stanford University Statistics Department (1999).</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>Demšar, J. Statistical comparisons of classifiers over multiple data sets. <em>J. Mach. Learn. Res.</em><strong>7</strong>, 1–30 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dem%C5%A1ar,%20J.%20Statistical%20comparisons%20of%20classifiers%20over%20multiple%20data%20sets.%20J.%20Mach.%20Learn.%20Res.7,%201%E2%80%9330%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Jeganathan, S. et al. Proline-directed pseudo-phosphorylation at at8 and phf1 epitopes induces a compaction of the paperclip folding of tau and generates a pathological (mc-1) conformation. <em>J. Biol. Chem.</em><strong>283</strong>, 32066–32076 (2008).
</cite> [<a href="https://doi.org/10.1074/jbc.M805300200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18725412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeganathan,%20S.%20et%20al.%20Proline-directed%20pseudo-phosphorylation%20at%20at8%20and%20phf1%20epitopes%20induces%20a%20compaction%20of%20the%20paperclip%20folding%20of%20tau%20and%20generates%20a%20pathological%20(mc-1)%20conformation.%20J.%20Biol.%20Chem.283,%2032066%E2%80%9332076%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Qin, X., Wang, Y. &amp; Paudel, H. K. Early growth response 1 (egr-1) is a transcriptional activator of <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/dcae87c9e622/41598_2025_14636_Article_IEq55.gif" loading="lazy" alt="Inline graphic">-secretase 1 (bace-1) in the brain. <em>J. Biol. Chem.</em><strong>291</strong>, 22276–22287 (2016).
</cite> [<a href="https://doi.org/10.1074/jbc.M116.738849" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5064006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27576688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin,%20X.,%20Wang,%20Y.%20&amp;%20Paudel,%20H.%20K.%20Early%20growth%20response%201%20(egr-1)%20is%20a%20transcriptional%20activator%20of%20-secretase%201%20(bace-1)%20in%20the%20brain.%20J.%20Biol.%20Chem.291,%2022276%E2%80%9322287%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>O’brien, R. J. &amp; Wong, P. C. Amyloid precursor protein processing and alzheimer’s disease. <em>Annu. Rev. Neurosci.</em><strong>34</strong>, 185–204 (2011).</cite> [<a href="https://doi.org/10.1146/annurev-neuro-061010-113613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3174086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21456963/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Dzwonek, J. &amp; Wilczynski, G. M. Cd44: molecular interactions, signaling and functions in the nervous system. <em>Front. Cell. Neurosci.</em><strong>9</strong>, 175 (2015).
</cite> [<a href="https://doi.org/10.3389/fncel.2015.00175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4423434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25999819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dzwonek,%20J.%20&amp;%20Wilczynski,%20G.%20M.%20Cd44:%20molecular%20interactions,%20signaling%20and%20functions%20in%20the%20nervous%20system.%20Front.%20Cell.%20Neurosci.9,%20175%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Duclot, F. &amp; Kabbaj, M. The role of early growth response 1 (egr1) in brain plasticity and neuropsychiatric disorders. <em>Front. Behav. Neurosci.</em><strong>11</strong>, 35 (2017).
</cite> [<a href="https://doi.org/10.3389/fnbeh.2017.00035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5337695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28321184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duclot,%20F.%20&amp;%20Kabbaj,%20M.%20The%20role%20of%20early%20growth%20response%201%20(egr1)%20in%20brain%20plasticity%20and%20neuropsychiatric%20disorders.%20Front.%20Behav.%20Neurosci.11,%2035%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Chu, E., Mychasiuk, R., Hibbs, M. L. &amp; Semple, B. D. Dysregulated phosphoinositide 3-kinase signaling in microglia: shaping chronic neuroinflammation. <em>J. Neuroinflammation</em><strong>18</strong>, 1–17 (2021).
</cite> [<a href="https://doi.org/10.1186/s12974-021-02325-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8627624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34838047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chu,%20E.,%20Mychasiuk,%20R.,%20Hibbs,%20M.%20L.%20&amp;%20Semple,%20B.%20D.%20Dysregulated%20phosphoinositide%203-kinase%20signaling%20in%20microglia:%20shaping%20chronic%20neuroinflammation.%20J.%20Neuroinflammation18,%201%E2%80%9317%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Takata, F., Nakagawa, S., Matsumoto, J. &amp; Dohgu, S. Blood-brain barrier dysfunction amplifies the development of neuroinflammation: understanding of cellular events in brain microvascular endothelial cells for prevention and treatment of bbb dysfunction. <em>Front. Cell. Neurosci.</em><strong>15</strong>, 661838 (2021).
</cite> [<a href="https://doi.org/10.3389/fncel.2021.661838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8475767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34588955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takata,%20F.,%20Nakagawa,%20S.,%20Matsumoto,%20J.%20&amp;%20Dohgu,%20S.%20Blood-brain%20barrier%20dysfunction%20amplifies%20the%20development%20of%20neuroinflammation:%20understanding%20of%20cellular%20events%20in%20brain%20microvascular%20endothelial%20cells%20for%20prevention%20and%20treatment%20of%20bbb%20dysfunction.%20Front.%20Cell.%20Neurosci.15,%20661838%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Li, Z. et al. Angiopoietin-2 blockade ameliorates autoimmune neuroinflammation by inhibiting leukocyte recruitment into the cns. <em>J. Clin. Investig.</em><strong>130</strong>, 1977–1990 (2020).
</cite> [<a href="https://doi.org/10.1172/JCI130308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7108925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32149735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.%20et%20al.%20Angiopoietin-2%20blockade%20ameliorates%20autoimmune%20neuroinflammation%20by%20inhibiting%20leukocyte%20recruitment%20into%20the%20cns.%20J.%20Clin.%20Investig.130,%201977%E2%80%931990%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Moore, A. M. et al. Apoe <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa4a/12361453/7659cf671118/41598_2025_14636_Article_IEq57.gif" loading="lazy" alt="Inline graphic">4-specific associations of vegf gene family expression with cognitive aging and alzheimer’s disease. <em>Neurobiol. Aging</em><strong>87</strong>, 18–25 (2020).
</cite> [<a href="https://doi.org/10.1016/j.neurobiolaging.2019.10.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7064375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31791659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moore,%20A.%20M.%20et%20al.%20Apoe%204-specific%20associations%20of%20vegf%20gene%20family%20expression%20with%20cognitive%20aging%20and%20alzheimer%E2%80%99s%20disease.%20Neurobiol.%20Aging87,%2018%E2%80%9325%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Hohman, T. J. et al. The role of vascular endothelial growth factor in neurodegeneration and cognitive decline: exploring interactions with biomarkers of alzheimer disease. <em>JAMA Neurol.</em><strong>72</strong>, 520–529 (2015).
</cite> [<a href="https://doi.org/10.1001/jamaneurol.2014.4761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4428948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hohman,%20T.%20J.%20et%20al.%20The%20role%20of%20vascular%20endothelial%20growth%20factor%20in%20neurodegeneration%20and%20cognitive%20decline:%20exploring%20interactions%20with%20biomarkers%20of%20alzheimer%20disease.%20JAMA%20Neurol.72,%20520%E2%80%93529%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Rosenthal, S. B. et al. Mapping the gene network landscape of alzheimer’s disease through integrating genomics and transcriptomics. <em>PLoS Comput. Biol.</em><strong>18</strong>, e1009903 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8906581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35213535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenthal,%20S.%20B.%20et%20al.%20Mapping%20the%20gene%20network%20landscape%20of%20alzheimer%E2%80%99s%20disease%20through%20integrating%20genomics%20and%20transcriptomics.%20PLoS%20Comput.%20Biol.18,%20e1009903%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Rosenberger, A. F. et al. Altered distribution of the epha4 kinase in hippocampal brain tissue of patients with alzheimer’s disease correlates with pathology. <em>Acta neuropathol. commun.</em><strong>2</strong>, 1–13 (2014).
</cite> [<a href="https://doi.org/10.1186/s40478-014-0079-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4149234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25027113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenberger,%20A.%20F.%20et%20al.%20Altered%20distribution%20of%20the%20epha4%20kinase%20in%20hippocampal%20brain%20tissue%20of%20patients%20with%20alzheimer%E2%80%99s%20disease%20correlates%20with%20pathology.%20Acta%20neuropathol.%20commun.2,%201%E2%80%9313%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>A Bennett, D., A Schneider, J., Arvanitakis, Z. et al. Overview and findings from the religious orders study. <em>Curr. Alzheimer Res.</em><strong>9</strong>, 628–645. (2012).</cite> [<a href="https://doi.org/10.2174/156720512801322573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3409291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22471860/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Bennett, D. A. et al. Religious orders study and rush memory and aging project. <em>J. Alzheimers Dis.</em><strong>64</strong>, S161–S189 (2018).
</cite> [<a href="https://doi.org/10.3233/JAD-179939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6380522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29865057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bennett,%20D.%20A.%20et%20al.%20Religious%20orders%20study%20and%20rush%20memory%20and%20aging%20project.%20J.%20Alzheimers%20Dis.64,%20S161%E2%80%93S189%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kingma, D. P. &amp; Ba, J. Adam: A method for stochastic optimization. <em>arXiv preprint</em><a href="http://arxiv.org/abs/1412.6980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:1412.6980</a> (2014).</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Zou, H. &amp; Hastie, T. Regularization and variable selection via the elastic net. <em>J. R. Stat. Soc. Series B Stat. Methodol.</em><strong>67</strong>, 301–320 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zou,%20H.%20&amp;%20Hastie,%20T.%20Regularization%20and%20variable%20selection%20via%20the%20elastic%20net.%20J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol.67,%20301%E2%80%93320%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Tibshirani, R. Regression shrinkage and selection via the lasso. <em>J. R. Stat. Soc. Series B Methodol.</em><strong>58</strong>, 267–288 (1996).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tibshirani,%20R.%20Regression%20shrinkage%20and%20selection%20via%20the%20lasso.%20J.%20R.%20Stat.%20Soc.%20Series%20B%20Methodol.58,%20267%E2%80%93288%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kokhlikyan, N., Miglani, V., Martin, M. et al. Captum: A unified and generic model interpretability library for pytorch. <em>arXiv preprint</em><a href="http://arxiv.org/abs/2009.07896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:2009.07896</a> (2020).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361453/bin/41598_2025_14636_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (2.7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361453/bin/41598_2025_14636_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (14.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12361453/bin/41598_2025_14636_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (52.2KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The source code is freely available through GitHub (<a href="https://github.com/JW-Yan/TransFuse" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/JW-Yan/TransFuse</a>). The data underlying this article is available via the AD Knowledge Portal (<a href="https://adknowledgeportal.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://adknowledgeportal.org</a>). The AD Knowledge Portal is a platform for accessing data, analyses, and tools generated by the Accelerating Medicines Partnership (AMP-AD) Target Discovery Program and other National Institute on Aging (NIA)-supported programs to enable open-science practices and accelerate translational learning. The data, analyses and tools are shared early in the research cycle without a publication embargo on secondary use. Data is available for general research use according to the following requirements for data access and data attribution (https://adknowledgeportal.synapse.org/Data%20Access). Data downloaded for the proposed analysis include: ROS/MAP protein expression (<a href="https://www.synapse.org/Synapse:syn21448467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn21448467</a>), ROS/MAP gene expression (<a href="https://www.synapse.org/Synapse:syn8456704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn8456704</a>), ROS/MAP genetic variants (<a href="https://www.synapse.org/Synapse:syn3221153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn3221153</a>), MSBB protein expression (<a href="https://www.synapse.org/Synapse:syn6100410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn6100410</a>), MSBB gene expression (<a href="https://www.synapse.org/Synapse:syn7391833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn7391833</a>), MSBB genetic variants (<a href="https://www.synapse.org/Synapse:syn11707204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.synapse.org/Synapse:syn11707204</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-14636-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_14636.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12361453/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12361453/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12361453%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12361453/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12361453/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12361453/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40825989/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12361453/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40825989/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12361453/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12361453/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="bMgcH5pYE5BfBp2hLzxk2GRVtns8h3LbA8GUop2FD8pC3YQvs2fcLVvxU4oBetJq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
