
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Current progress of digital twin construction using medical imaging - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4813E8AEEC3D305813E001AB51E23.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="jacmp">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370374/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Applied Clinical Medical Physics">
<meta name="citation_title" content="Current progress of digital twin construction using medical imaging">
<meta name="citation_author" content="Feng Zhao">
<meta name="citation_author_institution" content="Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA">
<meta name="citation_author_institution" content="School of Computational Science and Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author" content="Yizhou Wu">
<meta name="citation_author_institution" content="Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA">
<meta name="citation_author_institution" content="School of Electrical and Computer Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author" content="Mingzhe Hu">
<meta name="citation_author_institution" content="Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA">
<meta name="citation_author_institution" content="Department of Computer Science and Informatics, Emory University, Atlanta, Georgia, USA">
<meta name="citation_author" content="Chih‐Wei Chang">
<meta name="citation_author_institution" content="Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA">
<meta name="citation_author" content="Ruirui Liu">
<meta name="citation_author_institution" content="Department of Radiation Oncology, University of Nebraska Medical Center, Omaha, Nebraska, USA">
<meta name="citation_author" content="Richard Qiu">
<meta name="citation_author_institution" content="Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA">
<meta name="citation_author" content="Xiaofeng Yang">
<meta name="citation_author_institution" content="Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA">
<meta name="citation_author_institution" content="Department of Computer Science and Informatics, Emory University, Atlanta, Georgia, USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70226">
<meta name="citation_doi" content="10.1002/acm2.70226">
<meta name="citation_pmid" content="40841176">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370374/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370374/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370374/pdf/ACM2-26-e70226.pdf">
<meta name="description" content="Medical imaging is fundamental to digital twin technology, enabling patient‐specific virtual models of anatomy and physiology. By integrating high‐resolution modalities (Magnetic Resonance Imaging (MRI), Computed Tomography (CT), Positron Emission ...">
<meta name="og:title" content="Current progress of digital twin construction using medical imaging">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Medical imaging is fundamental to digital twin technology, enabling patient‐specific virtual models of anatomy and physiology. By integrating high‐resolution modalities (Magnetic Resonance Imaging (MRI), Computed Tomography (CT), Positron Emission ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370374/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370374">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/acm2.70226"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ACM2-26-e70226.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370374%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370374/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370374/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370374/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-jacmp.jpg" alt="Journal of Applied Clinical Medical Physics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Journal of Applied Clinical Medical Physics" title="Link to Journal of Applied Clinical Medical Physics" shape="default" href="http://aapm.onlinelibrary.wiley.com/hub/journal/10.1002/(ISSN)1526-9914/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Appl Clin Med Phys</button></div>. 2025 Aug 21;26(9):e70226. doi: <a href="https://doi.org/10.1002/acm2.70226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/acm2.70226</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Appl%20Clin%20Med%20Phys%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Appl%20Clin%20Med%20Phys%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Appl%20Clin%20Med%20Phys%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Appl%20Clin%20Med%20Phys%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Current progress of digital twin construction using medical imaging</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Feng Zhao</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Feng Zhao</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>School of Computational Science and Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Feng Zhao</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yizhou Wu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yizhou Wu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>School of Electrical and Computer Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yizhou Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Mingzhe Hu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Mingzhe Hu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Computer Science and Informatics, Emory University, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingzhe Hu</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20CW%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Chih‐Wei Chang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Chih‐Wei Chang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20CW%22%5BAuthor%5D" class="usa-link"><span class="name western">Chih‐Wei Chang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ruirui Liu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ruirui Liu</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Radiation Oncology, University of Nebraska Medical Center, Omaha, Nebraska, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruirui Liu</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiu%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Richard Qiu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Richard Qiu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiu%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard Qiu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xiaofeng Yang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xiaofeng Yang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Computer Science and Informatics, Emory University, Atlanta, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaofeng Yang</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="acm270226-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, Georgia, USA</div>
<div id="acm270226-aff-0002">
<sup>
<sup>2</sup>
</sup>School of Computational Science and Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div id="acm270226-aff-0003">
<sup>
<sup>3</sup>
</sup>School of Electrical and Computer Engineering, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div id="acm270226-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Computer Science and Informatics, Emory University, Atlanta, Georgia, USA</div>
<div id="acm270226-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Radiation Oncology, University of Nebraska Medical Center, Omaha, Nebraska, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Xiaofeng Yang, Department of Radiation Oncology and Winship Cancer Institute, Emory University, Atlanta, GA 30322, USA.  Email: <span>xiaofeng.yang@emory.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 3; Received 2024 Dec 3; Accepted 2025 Jul 21; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Journal of Applied Clinical Medical Physics</em> published by Wiley Periodicals, LLC on behalf of The American Association of Physicists in Medicine.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370374  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841176/" class="usa-link">40841176</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Medical imaging is fundamental to digital twin technology, enabling patient‐specific virtual models of anatomy and physiology. By integrating high‐resolution modalities (Magnetic Resonance Imaging (MRI), Computed Tomography (CT), Positron Emission Tomography (PET), ultrasound) with computational frameworks, recent imaging advances now support real‐time simulation, predictive modeling, and earlier disease detection. Such capabilities directly inform individualized treatment planning and contribute to more precise, personalized care. Despite remaining challenges—complex anatomical modeling, multimodal integration, and high computational demands—recent advances in imaging and machine learning have significantly enhanced the accuracy and clinical utility of digital twins. The main contributions of our review are: (1) a system‐by‐system classification of methodologies; (2) evidence that advanced imaging modalities have improved diagnostic accuracy, treatment effectiveness, and patient outcomes beyond conventional approaches; and (3) identification of remaining technical bottlenecks. We further analyze key technical barriers—such as data scarcity and computational complexity—and outline future directions (e.g., AI‐driven data augmentation, real‐time model optimization) to unlock digital twins’ full potential in precision medicine.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> computational models, diagnostic accuracy, digital twin, machine learning, medical imaging, personalized treatment, predictive analytics, real‐time simulation</p></section></section><section id="acm270226-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Originally emerging in the manufacturing and aerospace sectors,<a href="#acm270226-bib-0001" class="usa-link" aria-describedby="acm270226-bib-0001">
<sup>1</sup>
</a> digital twin technology has now entered healthcare, transforming the field of medical imaging. A digital twin is a dynamic, virtual representation of a physical entity, created using real‐time data and sophisticated computational models. With advancements in medical imaging, digital twins allow for the creation of highly detailed and precise representations of a patient's anatomy and physiological processes.</p>
<p>The effectiveness and reliability of digital twin models hinge heavily on the quality of the medical imaging data they're built upon—especially when it comes to spatial resolution and dynamic range. Spatial resolution, influenced by the specific imaging modality and parameters like voxel size in MRI or CT, plays a direct role in how precisely anatomical structures can be recreated digitally. The sharper the resolution, the more detail is captured, but this also means a significant rise in computational load due to the volume and intricacy of the data. On the other hand, dynamic range—referring to the spectrum of light and dark values an imaging system can detect—impacts how well tissue contrasts and physiological nuances are represented. A wider dynamic range allows for better distinction between subtle tissue variations, which is vital for accurate physiological modeling. As a result, achieving a workable balance between high‐quality imaging and computational manageability remains a key challenge in deploying digital twins effectively in clinical environments.</p>
<p>One significant application of digital twins in healthcare is their ability to support personalized treatment for each patient, with the added advantage of dynamically updating the model based on longitudinal measurements and clinical data, thereby capturing the patient's disease trajectory over time. Digital twins enable healthcare professionals to forecast disease progression, predict responses to interventions such as drugs, and anticipate adverse events associated with treatments. As a result, there is an increasing need for advanced tools and approaches to meet these demands effectively. Digital twin technology is emerging as a powerful tool in healthcare, opening new possibilities for enhancing patient outcomes, optimizing treatment plans, and advancing preventive care strategies.</p>
<p>Digital twin technology is revolutionizing personalized healthcare across various medical domains. For instance, a cardiovascular digital‐twin system<a href="#acm270226-bib-0002" class="usa-link" aria-describedby="acm270226-bib-0002">
<sup>2</sup>
</a> had been integrated using MRI and CT with hemodynamic data to simulate heart and blood flow dynamics, aiding in the prediction of arrhythmias or blockages and facilitating precise intervention planning. Figure <a href="#acm270226-fig-0001" class="usa-link">1</a> illustrates the comprehensive integration of digital twin technology in healthcare. On the left side, representing the physical world, are key entities: the human body, medical imaging devices, and healthcare facilities. Data is continuously collected from these sources, including genetic information, lab results, imaging data, and other biomedical data from patients; sensor readings, physical properties, and performance metrics from medical devices; and resource usage, staff allocation, processes, and workflows from hospitals. This real‐world data is then transferred to the digital realm, depicted on the right side of Figure <a href="#acm270226-fig-0001" class="usa-link">1</a>. Digital twins of the human body, medical devices, and healthcare facilities are created and regularly updated. The digital twin of a human aggregates all the collected data, forming a dynamic virtual representation that enables detailed analysis and simulation of patient‐specific conditions. Similarly, digital twins of machines and hospital systems incorporate operational and environmental data, providing a complete view of their performance and usage. Together, these digital twins offer an integrated perspective on healthcare operations, enhancing patient care, resource management, and overall efficiency.</p>
<figure class="fig xbox font-sm" id="acm270226-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/329a/12370374/a9980eb7d184/ACM2-26-e70226-g004.jpg" loading="lazy" id="jats-graphic-1" height="431" width="602" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/acm270226-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comprehensive integration of digital twin technology in healthcare.</p></figcaption></figure><p>Medical imaging is fundamental to constructing digital twins, serving as the critical link between the real patient and their virtual representation. While these models are continuously enhanced with data from multiple sources, imaging technologies such as MRI, CT, and PET provide the essential, non‐invasive, high‐resolution, and dynamic data needed for precise digital replication. By capturing detailed anatomical structures, identifying abnormalities, and documenting physiological processes, these imaging modalities enable ongoing monitoring and simulation of disease progression and treatment responses. Beyond simple visualization, medical imaging provides deep anatomical and functional insights that reveal each patient's unique characteristics. Utilizing these advanced imaging technologies addresses the complexities of replicating the intricate human body within a virtual environment, driving forward the possibilities in personalized medicine.</p>
<p>As shown in Table <a href="#acm270226-tbl-0001" class="usa-link">1</a>, we have compiled a comprehensive list of technical terms and their corresponding abbreviations commonly encountered in the fields of medical imaging and digital twins. This table includes a wide range of concepts, from clinical diseases and imaging modalities to data processing techniques and analytical models. By providing a clear reference for these terms, we aim to help readers navigate the technical details and maintain consistency when interpreting the results and methodologies presented throughout the manuscript.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0001"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Technical abbreviations.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Full term</th>
<th align="left" rowspan="1" colspan="1">Abbreviation</th>
<th align="left" rowspan="1" colspan="1">Full term</th>
<th align="left" rowspan="1" colspan="1">Abbreviation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Abdominal aortic aneurysm</td>
<td align="left" rowspan="1" colspan="1">AAA</td>
<td align="left" rowspan="1" colspan="1">Hypergraph convolutional neural network</td>
<td align="left" rowspan="1" colspan="1">HGCNN</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Alzheimer's disease</td>
<td align="left" rowspan="1" colspan="1">AD</td>
<td align="left" rowspan="1" colspan="1">High‐performance computing</td>
<td align="left" rowspan="1" colspan="1">HPC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Aortic stenosis</td>
<td align="left" rowspan="1" colspan="1">AS</td>
<td align="left" rowspan="1" colspan="1">Image reconstruction network</td>
<td align="left" rowspan="1" colspan="1">IR‐Net</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Artificial intelligence</td>
<td align="left" rowspan="1" colspan="1">AI</td>
<td align="left" rowspan="1" colspan="1">Implantable cardioverter defibrillator</td>
<td align="left" rowspan="1" colspan="1">ICD</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Atrial fibrillation</td>
<td align="left" rowspan="1" colspan="1">AF</td>
<td align="left" rowspan="1" colspan="1">Internet of things</td>
<td align="left" rowspan="1" colspan="1">IoT</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Augmented reality</td>
<td align="left" rowspan="1" colspan="1">AR</td>
<td align="left" rowspan="1" colspan="1">Intrahepatic portosystemic shunts</td>
<td align="left" rowspan="1" colspan="1">IPSS</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Blood oxygen level dependent</td>
<td align="left" rowspan="1" colspan="1">BOLD</td>
<td align="left" rowspan="1" colspan="1">Inverse kinematics</td>
<td align="left" rowspan="1" colspan="1">IK</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cardiac resynchronization therapy</td>
<td align="left" rowspan="1" colspan="1">CRT</td>
<td align="left" rowspan="1" colspan="1">Level set method</td>
<td align="left" rowspan="1" colspan="1">LSM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Common objects in context</td>
<td align="left" rowspan="1" colspan="1">COCO</td>
<td align="left" rowspan="1" colspan="1">Linear regression</td>
<td align="left" rowspan="1" colspan="1">LR</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Computational fluid dynamics</td>
<td align="left" rowspan="1" colspan="1">CFD</td>
<td align="left" rowspan="1" colspan="1">Long short‐term memory</td>
<td align="left" rowspan="1" colspan="1">LSTM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Computed tomography</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">Magnetic resonance</td>
<td align="left" rowspan="1" colspan="1">MR</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Computed tomography angiography</td>
<td align="left" rowspan="1" colspan="1">CTA</td>
<td align="left" rowspan="1" colspan="1">Magnetic resonance imaging</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Computer‐aided manufacturing</td>
<td align="left" rowspan="1" colspan="1">CAM</td>
<td align="left" rowspan="1" colspan="1">Magnetic resonance spectroscopy</td>
<td align="left" rowspan="1" colspan="1">MRS</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cone beam computed tomography</td>
<td align="left" rowspan="1" colspan="1">CBCT</td>
<td align="left" rowspan="1" colspan="1">Metabolic dysfunction‐associated fatty liver disease</td>
<td align="left" rowspan="1" colspan="1">MAFLD</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Convolutional neural network</td>
<td align="left" rowspan="1" colspan="1">CNN</td>
<td align="left" rowspan="1" colspan="1">Mixture of products of the multinomial</td>
<td align="left" rowspan="1" colspan="1">MPoM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Central nervous system</td>
<td align="left" rowspan="1" colspan="1">CNS</td>
<td align="left" rowspan="1" colspan="1">Multi‐core fiber</td>
<td align="left" rowspan="1" colspan="1">MCF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Coronary artery disease</td>
<td align="left" rowspan="1" colspan="1">CAD</td>
<td align="left" rowspan="1" colspan="1">Multiple sclerosis</td>
<td align="left" rowspan="1" colspan="1">MS</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Decision tree regression</td>
<td align="left" rowspan="1" colspan="1">DTR</td>
<td align="left" rowspan="1" colspan="1">Neoadjuvant systemic therapy</td>
<td align="left" rowspan="1" colspan="1">NAST</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Deep convolutional generative adversarial network</td>
<td align="left" rowspan="1" colspan="1">DCGAN</td>
<td align="left" rowspan="1" colspan="1">Neural ordinary differential equations</td>
<td align="left" rowspan="1" colspan="1">NeuralODEs</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Deep convolutional neural network</td>
<td align="left" rowspan="1" colspan="1">DCNN</td>
<td align="left" rowspan="1" colspan="1">Nonalcoholic fatty liver disease</td>
<td align="left" rowspan="1" colspan="1">NAFLD</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Deep neural network</td>
<td align="left" rowspan="1" colspan="1">DNN</td>
<td align="left" rowspan="1" colspan="1">Optical coherence tomography</td>
<td align="left" rowspan="1" colspan="1">OCT</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dice similarity coefficient</td>
<td align="left" rowspan="1" colspan="1">DSC</td>
<td align="left" rowspan="1" colspan="1">Peak signal‐to‐noise ratio</td>
<td align="left" rowspan="1" colspan="1">PSNR</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diffusion tensor imaging</td>
<td align="left" rowspan="1" colspan="1">DTI</td>
<td align="left" rowspan="1" colspan="1">Portal hypertension</td>
<td align="left" rowspan="1" colspan="1">PHT</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diffusion‐weighted MRI</td>
<td align="left" rowspan="1" colspan="1">DW‐MRI</td>
<td align="left" rowspan="1" colspan="1">Principal component analysis</td>
<td align="left" rowspan="1" colspan="1">PCA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Digital imaging and communications in medicine</td>
<td align="left" rowspan="1" colspan="1">DICOM</td>
<td align="left" rowspan="1" colspan="1">Proton density fat fraction</td>
<td align="left" rowspan="1" colspan="1">PDFF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Digital twin brain</td>
<td align="left" rowspan="1" colspan="1">DTB</td>
<td align="left" rowspan="1" colspan="1">Preferred reporting items for systematic reviews and meta‐analyses</td>
<td align="left" rowspan="1" colspan="1">PRISMA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Digital twin brain visualization</td>
<td align="left" rowspan="1" colspan="1">DTBVis</td>
<td align="left" rowspan="1" colspan="1">Random forest regression</td>
<td align="left" rowspan="1" colspan="1">RFR</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Discrete multiphysics</td>
<td align="left" rowspan="1" colspan="1">DMP</td>
<td align="left" rowspan="1" colspan="1">Recurrent neural network</td>
<td align="left" rowspan="1" colspan="1">RNN</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Doppler ultrasound</td>
<td align="left" rowspan="1" colspan="1">DUS</td>
<td align="left" rowspan="1" colspan="1">Reduced‐Order Model</td>
<td align="left" rowspan="1" colspan="1">ROM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Double‐outlet right ventricle</td>
<td align="left" rowspan="1" colspan="1">DORV</td>
<td align="left" rowspan="1" colspan="1">Relative technical error of measurement</td>
<td align="left" rowspan="1" colspan="1">rTEM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dynamic colon model</td>
<td align="left" rowspan="1" colspan="1">DCM</td>
<td align="left" rowspan="1" colspan="1">Robotic‐assisted radical prostatectomy</td>
<td align="left" rowspan="1" colspan="1">RARP</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dynamic contrast‐enhanced magnetic resonance imaging</td>
<td align="left" rowspan="1" colspan="1">DCE‐MRI</td>
<td align="left" rowspan="1" colspan="1">Robust auxiliary classifier GAN</td>
<td align="left" rowspan="1" colspan="1">rAC‐GAN</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Electrical impedance Tomography</td>
<td align="left" rowspan="1" colspan="1">EIT</td>
<td align="left" rowspan="1" colspan="1">Semi‐supervised support Vector machines</td>
<td align="left" rowspan="1" colspan="1">S3VM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Electrocardiogram</td>
<td align="left" rowspan="1" colspan="1">ECG</td>
<td align="left" rowspan="1" colspan="1">Smoothed particle hydrodynamics</td>
<td align="left" rowspan="1" colspan="1">SPH</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Electromyography</td>
<td align="left" rowspan="1" colspan="1">EMG</td>
<td align="left" rowspan="1" colspan="1">Standard tessellation language</td>
<td align="left" rowspan="1" colspan="1">STL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Electrophysiology</td>
<td align="left" rowspan="1" colspan="1">EP</td>
<td align="left" rowspan="1" colspan="1">Statistical shape model</td>
<td align="left" rowspan="1" colspan="1">SSM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Enhanced deep super‐resolution</td>
<td align="left" rowspan="1" colspan="1">EDSR</td>
<td align="left" rowspan="1" colspan="1">Structural similarity index measure</td>
<td align="left" rowspan="1" colspan="1">SSIM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Extended reality</td>
<td align="left" rowspan="1" colspan="1">XR</td>
<td align="left" rowspan="1" colspan="1">Sudden cardiac death</td>
<td align="left" rowspan="1" colspan="1">SCD</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Findable, accessible, interoperable, and reusable</td>
<td align="left" rowspan="1" colspan="1">FAIR</td>
<td align="left" rowspan="1" colspan="1">Support vector machine</td>
<td align="left" rowspan="1" colspan="1">SVM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Finite element analysis</td>
<td align="left" rowspan="1" colspan="1">FEA</td>
<td align="left" rowspan="1" colspan="1">t‐distributed Stochastic neighbor embedding</td>
<td align="left" rowspan="1" colspan="1">t‐SNE</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Finite element modeling</td>
<td align="left" rowspan="1" colspan="1">FEM</td>
<td align="left" rowspan="1" colspan="1">Three‐dimensional</td>
<td align="left" rowspan="1" colspan="1">3D</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Four‐dimensional computed tomography</td>
<td align="left" rowspan="1" colspan="1">4D‐CT</td>
<td align="left" rowspan="1" colspan="1">Three‐dimensional virtual modeling</td>
<td align="left" rowspan="1" colspan="1">3DVM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fully convolutional network</td>
<td align="left" rowspan="1" colspan="1">FCN</td>
<td align="left" rowspan="1" colspan="1">Triple‐negative breast cancer</td>
<td align="left" rowspan="1" colspan="1">TNBC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Functional connectivity matrices</td>
<td align="left" rowspan="1" colspan="1">FC matrices</td>
<td align="left" rowspan="1" colspan="1">Two‐dimensional</td>
<td align="left" rowspan="1" colspan="1">2D</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">functional MRI</td>
<td align="left" rowspan="1" colspan="1">fMRI</td>
<td align="left" rowspan="1" colspan="1">Ventricular tachycardia</td>
<td align="left" rowspan="1" colspan="1">VT</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Gaussian process regression</td>
<td align="left" rowspan="1" colspan="1">GPR</td>
<td align="left" rowspan="1" colspan="1">Verification, validation, and uncertainty quantification</td>
<td align="left" rowspan="1" colspan="1">VVUQ</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Geographic atrophy</td>
<td align="left" rowspan="1" colspan="1">GA</td>
<td align="left" rowspan="1" colspan="1">You only look once</td>
<td align="left" rowspan="1" colspan="1">YOLO</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Generative artificial intelligence</td>
<td align="left" rowspan="1" colspan="1">GAI</td>
<td align="left" rowspan="1" colspan="1">Zero‐dimensional</td>
<td align="left" rowspan="1" colspan="1">0D</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Generative adversarial networks</td>
<td align="left" rowspan="1" colspan="1">GANs</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Gradient boosting algorithm</td>
<td align="left" rowspan="1" colspan="1">GBA</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Graph Neural network</td>
<td align="left" rowspan="1" colspan="1">GNN</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Graphics processing unit</td>
<td align="left" rowspan="1" colspan="1">GPU</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Heart failure</td>
<td align="left" rowspan="1" colspan="1">HF</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Hounsfield unit</td>
<td align="left" rowspan="1" colspan="1">HU</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Hybrid pyramid <em>U</em>‐Net</td>
<td align="left" rowspan="1" colspan="1">HPU‐Net</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Hyperbolic graph neural networks</td>
<td align="left" rowspan="1" colspan="1">HGNN</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The evolution of digital twin technology has reached a pivotal stage in precision medicine, driven by improvements in computational power, high‐performance computing, and cloud‐based platforms that facilitate complex simulations. This progress has been further accelerated by the development of deep learning models, particularly foundational models, which overcome significant limitations of traditional analytical or physical models. This is especially valuable in complex systems like the human circulatory system, where many physiological parameters are difficult or impossible to measure directly, complicating accurate modeling. Foundational models, pre‐trained on extensive datasets, have made the construction of digital twins more feasible by enabling robust modeling even with incomplete or diverse data. Moreover, the integration of multimodal data—including medical imaging, genomic information, wearable sensor data, and electronic health records—enables a comprehensive and dynamic representation of individual patients. This integration supports real‐time monitoring, predictive modeling, and personalized treatment planning. Machine learning techniques, such as CNN,<a href="#acm270226-bib-0003" class="usa-link" aria-describedby="acm270226-bib-0003">
<sup>3</sup>
</a><sup>, </sup><a href="#acm270226-bib-0004" class="usa-link" aria-describedby="acm270226-bib-0004">
<sup>4</sup>
</a><sup>, </sup><a href="#acm270226-bib-0005" class="usa-link" aria-describedby="acm270226-bib-0005">
<sup>5</sup>
</a> GAN,<a href="#acm270226-bib-0006" class="usa-link" aria-describedby="acm270226-bib-0006">
<sup>6</sup>
</a><sup>, </sup><a href="#acm270226-bib-0007" class="usa-link" aria-describedby="acm270226-bib-0007">
<sup>7</sup>
</a><sup>, </sup><a href="#acm270226-bib-0008" class="usa-link" aria-describedby="acm270226-bib-0008">
<sup>8</sup>
</a> DDPM,<a href="#acm270226-bib-0009" class="usa-link" aria-describedby="acm270226-bib-0009">
<sup>9</sup>
</a><sup>, </sup><a href="#acm270226-bib-0010" class="usa-link" aria-describedby="acm270226-bib-0010">
<sup>10</sup>
</a><sup>, </sup><a href="#acm270226-bib-0011" class="usa-link" aria-describedby="acm270226-bib-0011">
<sup>11</sup>
</a> and GAI,<a href="#acm270226-bib-0012" class="usa-link" aria-describedby="acm270226-bib-0012">
<sup>12</sup>
</a><sup>, </sup><a href="#acm270226-bib-0013" class="usa-link" aria-describedby="acm270226-bib-0013">
<sup>13</sup>
</a><sup>, </sup><a href="#acm270226-bib-0014" class="usa-link" aria-describedby="acm270226-bib-0014">
<sup>14</sup>
</a> enhance the adaptability and accuracy of digital twins by efficiently handling complex datasets and uncovering insights beyond the capabilities of traditional models. These converging advancements have transformed digital twins from a visionary concept into a practical, scalable tool, supporting a proactive, patient‐centered approach that significantly elevates the precision and personalization of modern healthcare.</p>
<section id="acm270226-sec-0020"><h3 class="pmc_sec_title">1.1. Importance and advantages of this review</h3>
<p>As digital twin technology advances rapidly and its applications in healthcare broaden, it becomes increasingly important to understand its current development as shown in Figure <a href="#acm270226-fig-0002" class="usa-link">2</a>. This review aims to meet that need by examining the system‐specific applications of digital twins and highlighting their potential to revolutionize healthcare through personalized medicine. Our primary focus is on the latest computational models used in creating digital twins, providing a detailed exploration of the methods for constructing these twins and refining the associated algorithms and model architectures. Our analysis covers the entire lifecycle—from data collection and simulation to predictive analytics—offering a comprehensive yet accessible overview for both computer scientists seeking technical depth and medical professionals interested in clinical applications.</p>
<figure class="fig xbox font-sm" id="acm270226-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/329a/12370374/da1df871e71a/ACM2-26-e70226-g005.jpg" loading="lazy" id="jats-graphic-3" height="330" width="602" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/acm270226-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Steady growth in research focused on digital twin technology in medical imaging.</p></figcaption></figure><p>Central to this paper is a comprehensive analysis of the computational modeling techniques identified in the selected body of research. These techniques encompass image processing algorithms tailored to extract critical features from medical images, time‐series analysis for detecting temporal patterns within patient health data, and advanced simulation methodologies such as FEA (finite element analysis, a numerical method for simulating mechanical behavior) and CFD (computational fluid dynamics, used for simulating fluid flow). These sophisticated tools substantially enhance the precision and functionality of digital twins, facilitating real‐time monitoring and predictive capabilities. By incorporating these state‐of‐the‐art approaches, this review offers a rigorous and analytically sound examination that highlights the transformative potential of digital twins in the realm of personalized medicine.</p>
<p>One particularly novel aspect of our review is the focus on digital‐twin human system development based on medical imaging, a perspective not commonly addressed in current literature. Creating a digital twin of the entire human body is a complex and challenging endeavor, comparable to assembling an intricate puzzle. To tackle this challenge, researchers have shifted toward focusing on system‐specific digital twins, a more practical and manageable approach. We examine studies where researchers have broken the body down into smaller, more manageable components, developing detailed, actionable models for individual organs or systems. This focused approach allows for a level of precision and insight that broader, whole‐body models cannot achieve. Each system‐specific twin serves as a specialist, tailored to address the unique challenges of its respective system—whether in heart function, liver health, or other processes—contributing to a more comprehensive understanding of human health.</p>
<p>In conclusion, this review synthesizes current progress and technical innovations across multiple domains of digital twin research in healthcare. Our goal is to provide a comprehensive and rigorous reference that can inform future investigations, particularly in the context of medical imaging. By highlighting both methodological challenges and emerging opportunities, we aim to support continued advancements that contribute to more effective, personalized, and data‐driven patient care.</p></section><section id="acm270226-sec-0030"><h3 class="pmc_sec_title">1.2. Outline of the review</h3>
<p>In this review, we have organized our discussion as follows. Section <a href="#acm270226-sec-0010" class="usa-link">1</a> introduces the role of medical imaging in digital twin development and highlights key technical considerations, including spatial resolution and data integration. Section <a href="#acm270226-sec-0040" class="usa-link">2</a> presents a system‐specific analysis across seven major systems—cardiovascular, central nervous, musculoskeletal, breast, urinary, digestive, and respiratory—detailing the imaging modalities, computational methods, and clinical applications in each. Section <a href="#acm270226-sec-0270" class="usa-link">3</a> synthesizes cross‐system challenges (e.g., data scarcity, computational complexity) and proposes prospective research directions, such as AI‐driven data augmentation and real‐time model optimization. Finally, Section <a href="#acm270226-sec-0280" class="usa-link">4</a> concludes with key takeaways and outlines future pathways for advancing digital twins in precision medicine.</p></section></section><section id="acm270226-sec-0040"><h2 class="pmc_sec_title">2. SYSTEM‐SPECIFIC REVIEW</h2>
<p>In this section, we explore the system‐specific aspects of our review, providing an in‐depth analysis of various bodily systems. By focusing on the construction of digital twins using medical imaging data and the advancements in the algorithms and model architectures that support and refine them for higher precision, we aim to present a detailed understanding of the latest developments and emerging research trends in this application. It offers valuable insights for researchers and healthcare practitioners working to advance the field of personalized medicine.</p>
<section id="acm270226-sec-0050"><h3 class="pmc_sec_title">2.1. Methodology for literature review</h3>
<p>To comprehensively gather relevant literature, we conducted a systematic search following the PRISMA<a href="#acm270226-bib-0015" class="usa-link" aria-describedby="acm270226-bib-0015">
<sup>15</sup>
</a> protocol to the peer‐reviewed journal articles across Scopus, PubMed, Web of Science, and Google Scholar, using targeted queries that combined terms such as “Digital Twin” and “Digital Replica” with medical imaging modalities, including MRI and CT. This approach yielded 117 records from Scopus, 115 from PubMed, and 17 from Web of Science. Although the Google Scholar search initially returned over 800 records, only 80 were deemed highly relevant. After removing duplicates, we obtained a total of 260 peer‐reviewed journal articles as our initial identification. After removing duplicates, we obtained a total of 260 peer‐reviewed journal articles as initial identification as shown in Figure <a href="#acm270226-fig-0003" class="usa-link">3</a>.</p>
<figure class="fig xbox font-sm" id="acm270226-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/329a/12370374/47de86d49ede/ACM2-26-e70226-g003.jpg" loading="lazy" id="jats-graphic-5" height="438" width="602" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/acm270226-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PRISMA flow diagram of papers reviewed.</p></figcaption></figure><p>Following the initial screening, we subsequently assessed 181 full‐text articles for eligibility, and excluded 97 articles for various reasons: three were not in English, six focused solely on ethical concerns, one was pre‐clinical, 15 were unrelated to medical imaging, and 87 lacked sufficient technical detail. Specifically, these 87 articles were excluded due to insufficient reporting of critical methodological aspects, detailed as follows: 37 articles lacked sufficient details on imaging acquisition parameters (e.g., voxel resolution, imaging protocols), 29 articles provided inadequate descriptions of computational modeling techniques (e.g., algorithm configurations, model architectures, parameter settings), and 21 articles did not clearly report validation procedures or performance metrics, making reproducibility and clinical applicability unclear. Ultimately, after applying these criteria, 69 studies met our inclusion standards and were selected for detailed qualitative analysis.</p>
<p>Based on the qualified research work from Figure <a href="#acm270226-fig-0003" class="usa-link">3</a>, we further classified various digital‐twin systems into Figure <a href="#acm270226-fig-0004" class="usa-link">4</a>, which visually represents the diverse applications in medical fields. Each segment corresponds to a specific medical domain, such as cardiovascular, respiratory, breast, digestive, liver, musculoskeletal, and others. Within each segment, subdivisions outline particular application scenarios of digital twin technology within that field, including diagnostic enhancement, personalized treatment planning, image processing, simulation, cognitive modeling, and more. The chart provides a comprehensive visual overview of the classification methodology, offering insight into the extensive applications and potential research trajectories for digital twin technology in healthcare. This paper is structured around a classification analysis of different systems within the human body, which forms the foundation of this system‐specific review. By adopting this classification framework, we systematically examine the applications and advancements of digital twin technology across various medical domains, allowing for a focused and detailed exploration of its impact and potential in healthcare.</p>
<figure class="fig xbox font-sm" id="acm270226-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/329a/12370374/7947a68ebad3/ACM2-26-e70226-g002.jpg" loading="lazy" id="jats-graphic-7" height="600" width="600" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/acm270226-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Classification of digital twin applications across medical fields.</p></figcaption></figure></section><section id="acm270226-sec-0060"><h3 class="pmc_sec_title">2.2. Cardiovascular system</h3>
<p>Table <a href="#acm270226-tbl-0002" class="usa-link">2</a> presents an overview of digital twin research centered on the cardiovascular system, encompassing a wide range of clinical conditions, various imaging techniques, data volumes, and computational methodologies. It highlights the diversity in study designs and approaches, illustrating how cardiovascular digital twins are advancing the field by enhancing diagnostic capabilities, informing treatment strategies, and improving overall patient care. MRI and CT imaging are predominantly used as primary data sources, especially when investigating arterial stenosis and various cardiac diseases. Occasionally, additional imaging modalities like CTA, DUS, and x‐ray are incorporated to provide supplementary diagnostic information. The majority of datasets utilized in these studies are not publicly accessible, limiting widespread availability; however, certain initiatives are working toward sharing specific datasets. From a computational perspective, there is a strong dependence on conventional modeling techniques such as CFD and FEA for simulating hemodynamic processes. A significant shift toward employing advanced machine learning models—like deep neural networks and generative algorithms—is evident to more effectively capture complex physiological interactions. Furthermore, the combination of mechanistic and data‐driven modeling approaches is increasingly adopted to enhance the accuracy and adaptability of cardiovascular digital twins.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Summary of digital‐twin studies for cardiovascular system.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Clinical site</th>
<th align="left" rowspan="1" colspan="1">Data modality</th>
<th align="left" rowspan="1" colspan="1">Data quantity</th>
<th align="left" rowspan="1" colspan="1">Data publicity</th>
<th align="left" rowspan="1" colspan="1">Computational models</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0016" class="usa-link" aria-describedby="acm270226-bib-0016">16</a>
</td>
<td align="left" rowspan="1" colspan="1">Myocardial infarction</td>
<td align="left" rowspan="1" colspan="1">Cine MRI</td>
<td align="left" rowspan="1" colspan="1">20 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">FCN, SSM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0017" class="usa-link" aria-describedby="acm270226-bib-0017">17</a>
</td>
<td align="left" rowspan="1" colspan="1">Carotid artery stenosis</td>
<td align="left" rowspan="1" colspan="1">Cta, dus</td>
<td align="left" rowspan="1" colspan="1">26 patients, 37 bifurcations</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">CFD</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0018" class="usa-link" aria-describedby="acm270226-bib-0018">18</a>
</td>
<td align="left" rowspan="1" colspan="1">Arrhythmias, HF</td>
<td align="left" rowspan="1" colspan="1">MRI, CT, electrophysiology</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">CFD, EP</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0019" class="usa-link" aria-describedby="acm270226-bib-0019">19</a>
</td>
<td align="left" rowspan="1" colspan="1">Aortic stenosis</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">35 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">
<em>U</em>‐Net, DCNN</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0020" class="usa-link" aria-describedby="acm270226-bib-0020">20</a>
</td>
<td align="left" rowspan="1" colspan="1">Coronary artery stenosis</td>
<td align="left" rowspan="1" colspan="1">MRI, x‐ray</td>
<td align="left" rowspan="1" colspan="1">24 phantoms</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Computer‐aided design, CAM, stereolithography</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0021" class="usa-link" aria-describedby="acm270226-bib-0021">21</a>
</td>
<td align="left" rowspan="1" colspan="1">Cardiac EP</td>
<td align="left" rowspan="1" colspan="1">MRI, CT</td>
<td align="left" rowspan="1" colspan="1">24 patients</td>
<td align="left" rowspan="1" colspan="1">Available on zenodo</td>
<td align="left" rowspan="1" colspan="1">Cardiac EP, VVUQ</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0022" class="usa-link" aria-describedby="acm270226-bib-0022">22</a>
</td>
<td align="left" rowspan="1" colspan="1">DORV</td>
<td align="left" rowspan="1" colspan="1">CT, Echocardiography</td>
<td align="left" rowspan="1" colspan="1">6 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Virtual models</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0023" class="usa-link" aria-describedby="acm270226-bib-0023">23</a>
</td>
<td align="left" rowspan="1" colspan="1">Ischaemic cardiomyopathy</td>
<td align="left" rowspan="1" colspan="1">ECG, MRI, CT</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Data‐driven models, mechanistic models</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0024" class="usa-link" aria-describedby="acm270226-bib-0024">24</a>
</td>
<td align="left" rowspan="1" colspan="1">Coronary artery disease</td>
<td align="left" rowspan="1" colspan="1">OCT</td>
<td align="left" rowspan="1" colspan="1">28 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">FEA, CNN, 3D reconstruction</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0025" class="usa-link" aria-describedby="acm270226-bib-0025">25</a>
</td>
<td align="left" rowspan="1" colspan="1">AF</td>
<td align="left" rowspan="1" colspan="1">MRI, CT, ECG</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">DNN, YOLO, GANs, GNNs</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0026" class="usa-link" aria-describedby="acm270226-bib-0026">26</a>
</td>
<td align="left" rowspan="1" colspan="1">Myocardial infarction, hypertension, dyslipidemia</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">23 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Thematic analysis</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0027" class="usa-link" aria-describedby="acm270226-bib-0027">27</a>
</td>
<td align="left" rowspan="1" colspan="1">AAA</td>
<td align="left" rowspan="1" colspan="1">MRI, CT, blood pressure waveforms</td>
<td align="left" rowspan="1" colspan="1">4137 virtual patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">LSTM, CNN, inverse analysis</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0002" class="usa-link" aria-describedby="acm270226-bib-0002">2</a>
</td>
<td align="left" rowspan="1" colspan="1">AF, V, SCD</td>
<td align="left" rowspan="1" colspan="1">MRI, ECG</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">ML models, mechanistic models</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0028" class="usa-link" aria-describedby="acm270226-bib-0028">28</a>
</td>
<td align="left" rowspan="1" colspan="1">Coronary artery disease, myocardial ischemia</td>
<td align="left" rowspan="1" colspan="1">ECG, 3D imaging</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Aliev‐panfilov model, voxel models, 3d reconstruction</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0029" class="usa-link" aria-describedby="acm270226-bib-0029">29</a>
</td>
<td align="left" rowspan="1" colspan="1">Left bundle branch block, myocardial infarction</td>
<td align="left" rowspan="1" colspan="1">MRI, echocardiography</td>
<td align="left" rowspan="1" colspan="1">13 patients</td>
<td align="left" rowspan="1" colspan="1">GitHub repository available</td>
<td align="left" rowspan="1" colspan="1">CircAdapt model, particle swarm optimization</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0030" class="usa-link" aria-describedby="acm270226-bib-0030">30</a>
</td>
<td align="left" rowspan="1" colspan="1">Ebstein anomaly</td>
<td align="left" rowspan="1" colspan="1">3D ultrasound</td>
<td align="left" rowspan="1" colspan="1">1 patient</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">3D virtual, 3d printing</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0031" class="usa-link" aria-describedby="acm270226-bib-0031">31</a>
</td>
<td align="left" rowspan="1" colspan="1">Tetralogy of fallot</td>
<td align="left" rowspan="1" colspan="1">Echocardiograph, MRI, CT</td>
<td align="left" rowspan="1" colspan="1">2 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">CircAdapt</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="acm270226-sec-0070"><h4 class="pmc_sec_title">2.2.1. Cardiovascular disease modeling and risk stratification</h4>
<p>In recent years, the integration of advanced technologies has catalyzed significant improvements in cardiovascular healthcare.</p>
<p>Advanced modeling techniques, combined with digital twin technology, offer substantial advancements in CAD system (Computer‐Aided Design, a computer‐based system for designing and simulating complex structures), enabling precise risk stratification and supporting personalized clinical decision‐making, ultimately contributing to better patient outcomes. The study by Renaudin et al.<a href="#acm270226-bib-0020" class="usa-link" aria-describedby="acm270226-bib-0020">
<sup>20</sup>
</a> exemplifies this transformation through a novel approach involving stereolithography‐based Computer‐Aided Design/CAM to create highly precise three‐dimensional arterial phantoms. These phantoms, characterized by sub‐0.1 mm accuracy, provide a critical foundation for accurately modeling coronary arteries, facilitating the integration of these detailed models into digital twin frameworks.</p>
<p>Straughan et al.<a href="#acm270226-bib-0024" class="usa-link" aria-describedby="acm270226-bib-0024">
<sup>24</sup>
</a> extended these efforts by automating the construction of finite element simulations using OCT data. This innovation allows for efficient and precise stress analysis of coronary lesions, showcasing the potential to transform patient‐specific imaging into actionable digital twin models for CAD (Coronary Artery Disease) assessment. Compared to traditional methods that rely on manual segmentation of lesion geometries from imaging modalities like intravascular ultrasound or CT, Straughan's OCT‐based automated finite element modeling circumvents labor‐intensive processes, directly converting OCT data into precise 3D lesion models. This not only shortens the model construction time from hours to minutes but also reduces observer variability, thereby enabling more consistent and prompt decision‐making in clinical settings. Similarly, Chakshu et al.<a href="#acm270226-bib-0027" class="usa-link" aria-describedby="acm270226-bib-0027">
<sup>27</sup>
</a> employed LSTM‐based inverse analysis to derive blood pressure waveforms non‐invasively, effectively integrating deep learning with digital twins for real‐time cardiovascular monitoring. The contributions by Koopsen et al.<a href="#acm270226-bib-0029" class="usa-link" aria-describedby="acm270226-bib-0029">
<sup>29</sup>
</a> further demonstrate the power of digital twins by creating personalized models of left ventricular mechanical discoordination, enhancing the reliability of these models through parameter subset reduction and identifiability analysis.</p>
<p>Further evidence of the transformative power of digital twin technology in heart disease modeling and treatment is provided by studies in related cardiovascular domains. Loewe et al.<a href="#acm270226-bib-0018" class="usa-link" aria-describedby="acm270226-bib-0018">
<sup>18</sup>
</a> developed a highly personalized computational heart model using CT and MRI data to simulate electrical activity and assess SCD risk, demonstrating a hazard ratio significantly higher than that of traditional clinical metrics. This underscores the superiority of digital twins in heart failure modeling and precise risk stratification. Moreover, Banerjee et al.<a href="#acm270226-bib-0016" class="usa-link" aria-describedby="acm270226-bib-0016">
<sup>16</sup>
</a> applied deep learning techniques to 2D cine MR images for arrhythmia treatment, generating accurate 3D heart models by correcting misalignments, which enhanced the precision of electrophysiological simulations and diagnosis. Additionally, de Lepper et al.<a href="#acm270226-bib-0023" class="usa-link" aria-describedby="acm270226-bib-0023">
<sup>23</sup>
</a> introduced a hybrid modeling approach that combined mechanistic and data‐driven models, enabling better identification of high‐risk VT patients compared to traditional metrics, thereby reducing unnecessary ICD implantation.</p></section><section id="acm270226-sec-0080"><h4 class="pmc_sec_title">2.2.2. Enhancement of diagnostic and treatment modalities</h4>
<p>Recent studies demonstrate how digital twin technology is revolutionizing diagnostic imaging, making substantial contributions to both clinical research and practical applications. Bodin et al.<a href="#acm270226-bib-0028" class="usa-link" aria-describedby="acm270226-bib-0028">
<sup>28</sup>
</a> exemplify this integration by demonstrating the use of digital twins through the transformation of ECG data into visualizations of electrical activity within a 3D heart model. This innovative approach not only enabled precise simulation of myocardial damage but also provided a tangible framework for understanding the spatial dynamics of cardiac conditions, thereby helps clinicians better identify pathological regions and directly enhancing diagnostic accuracy and informing clinical decision‐making.</p>
<p>Rouhollahi et al.<a href="#acm270226-bib-0019" class="usa-link" aria-describedby="acm270226-bib-0019">
<sup>19</sup>
</a> built upon this foundation, highlighting the role of digital twins in cardiovascular diagnostics through a fully automated deep learning tool, CardioVision. Utilizing CT images, CardioVision effectively generated precise 3D reconstructions of the aortic valve, achieving highly accurate calcification quantification. This digital twin model significantly reduced manual effort and expedited the analysis process, demonstrating the feasibility of large‐scale clinical applications. The ability to automate image segmentation and produce detailed 3D reconstructions is pivotal in advancing the scalability of cardiovascular diagnostics, particularly in complex cases such as aortic stenosis.</p>
<p>Pires et al.<a href="#acm270226-bib-0030" class="usa-link" aria-describedby="acm270226-bib-0030">
<sup>30</sup>
</a> extended these capabilities further into the realm of congenital heart disease by leveraging obstetrical ultrasound data to create 3D digital twins of fetal hearts. These digital models provided an exceptionally detailed visualization of congenital anomalies, such as Ebstein anomaly, which played a crucial role in aiding pre‐surgical planning and family counseling. The ability to faithfully replicate the intricate anatomical structures of fetal hearts enabled medical teams to observe and assess abnormalities with a level of clarity that traditional 2D ultrasound could not provide. Such detailed digital twins bridge the gap between diagnostic imaging and actionable clinical insights, underscoring their value in personalized medicine.</p>
<p>In addition to surgical planning, the role of digital twin technology extends to non‐invasive diagnostics and validation. Dubs et al.<a href="#acm270226-bib-0017" class="usa-link" aria-describedby="acm270226-bib-0017">
<sup>17</sup>
</a> leveraged CFD to construct patient‐specific digital twins of carotid bifurcations, enabling a more precise, non‐invasive assessment of CAD severity. The CFD‐based digital twin models offered detailed insights into hemodynamic parameters, such as pressure drops across stenoses under varying flow conditions, which significantly enhanced diagnostic accuracy compared to traditional diagnostic tools. Meanwhile, Galappaththige et al.<a href="#acm270226-bib-0021" class="usa-link" aria-describedby="acm270226-bib-0021">
<sup>21</sup>
</a> expanded on the application of digital twins by developing personalized ventricular models for cardiac electrophysiological applications. By focusing on patient‐specific differences and employing VVUQ (Verification, Validation, and Uncertainty Quantification, a suite of techniques used to assess the reliability and accuracy of computational models) methods, the study emphasized the necessity of personalized modeling for ensuring the credibility and applicability of cardiac simulations across different patient profiles. Collectively, these studies demonstrate how digital twin technology enables the creation of highly individualized cardiac models and provides a robust framework for validating cardiovascular interventions.</p></section><section id="acm270226-sec-0090"><h4 class="pmc_sec_title">2.2.3. Personalized surgical planning and intervention</h4>
<p>Emerging studies demonstrate that digital twin technology is reshaping personalized treatment plans, surgical planning, and rehabilitation, contributing to remarkable advancements in cardiovascular disease management. Trayanova and Prakosa<a href="#acm270226-bib-0002" class="usa-link" aria-describedby="acm270226-bib-0002">
<sup>2</sup>
</a> exemplified this transformation by utilizing multimodal data—including ECG, CT, and MRI—to create personalized digital twin models of the heart. These models significantly enhanced the precision of arrhythmia treatment by accurately predicting optimal ablation targets, leading to improved outcomes for patients with atrial fibrillation. This study highlights the power of digital twins to bridge the gap between diagnostic imaging and actionable clinical interventions. Similarly, Ložek et al.<a href="#acm270226-bib-0031" class="usa-link" aria-describedby="acm270226-bib-0031">
<sup>31</sup>
</a> applied digital twin technology to model right ventricular dyssynchrony in patients post‐Tetralogy of Fallot repair. Their findings demonstrated that using digital twin simulations to personalize CRT led to notable improvements in ventricular function and efficiency, offering compelling evidence in favor of data‐driven, individualized therapeutic decisions.</p>
<p>In addition to treatment planning, digital twin technology has demonstrated significant value in surgical planning and rehabilitation, thereby expanding its influence across various stages of cardiovascular disease management. Rudnicka et al.<a href="#acm270226-bib-0025" class="usa-link" aria-describedby="acm270226-bib-0025">
<sup>25</sup>
</a> illustrated the application of digital twins in surgical planning by leveraging AI and XR technologies to model cardiac functions. This integration enhanced diagnostic precision, optimized targeted therapy selection, and reduced uncertainty in surgical procedures, thereby improving the reliability of treatment outcomes. Meanwhile, Vanzella et al.<a href="#acm270226-bib-0026" class="usa-link" aria-describedby="acm270226-bib-0026">
<sup>26</sup>
</a> highlighted the use of digital twins in cardiac rehabilitation, employing hybrid and virtual models to address the unique challenges posed by the COVID‐19 pandemic. These models not only improved patient adherence but also enhanced quality of life by overcoming barriers such as accessibility and convenience.</p>
<p>Recent studies have shown that advancements in digital twin technology significantly enhance the accuracy and dependability of surgical planning, testing, and validation in cardiovascular disease management. Liang et al.<a href="#acm270226-bib-0022" class="usa-link" aria-describedby="acm270226-bib-0022">
<sup>22</sup>
</a> introduced an innovative approach combining virtual models with 3D printing to simulate the DORV surgical procedure, resulting in a substantial reduction in diagnostic uncertainties and errors. By employing these digital twins as auxiliary tools, the study achieved a significant decrease in the rate of uncertain or incorrect diagnoses from 42.5% to 4.6%. This finding illustrates the transformative capacity of digital twins to offer a clear and tangible representation of complex cardiac anatomies, ultimately guiding surgical interventions with greater accuracy and confidence.</p></section></section><section id="acm270226-sec-0100"><h3 class="pmc_sec_title">2.3. Central nervous systems</h3>
<p>Among the 12 digital‐twin studies reviewed for central nervous systems, as presented in Table <a href="#acm270226-tbl-0003" class="usa-link">3</a>, MRI emerges as the primary imaging modality, providing essential insights into brain anatomy and activity,<a href="#acm270226-bib-0032" class="usa-link" aria-describedby="acm270226-bib-0032">
<sup>32</sup>
</a> and often combined with other techniques such as DTI and fMRI. While abundant datasets exist for common conditions like Alzheimer's disease, tumors, and multiple sclerosis—offering rich opportunities for in‐depth research—whole‐brain simulations highlight the need for more publicly available data. The computational methods employed range from traditional approaches like K‐Means clustering to advanced techniques including NeuralODEs and Spiking Neuronal Networks, with models selected according to specific clinical requirements. By summarizing this body of work, Table <a href="#acm270226-tbl-0003" class="usa-link">3</a> presents a comprehensive overview of the imaging modalities, data sources, and computational strategies used to address various neurological disorders. These investigations integrate methodologies from traditional machine learning and computer vision to cutting‐edge deep learning, transfer learning, and graphical modeling.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Summary of digital‐twin studies for central nervous systems.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Clinical site</th>
<th align="left" rowspan="1" colspan="1">Data modality</th>
<th align="left" rowspan="1" colspan="1">Data quantity</th>
<th align="left" rowspan="1" colspan="1">Data publicity</th>
<th align="left" rowspan="1" colspan="1">Computational models</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0033" class="usa-link" aria-describedby="acm270226-bib-0033">33</a>
</td>
<td align="left" rowspan="1" colspan="1">Multiple sclerosis</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
<td align="left" rowspan="1" colspan="1">2572 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">Mixed splines</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0034" class="usa-link" aria-describedby="acm270226-bib-0034">34</a>
</td>
<td align="left" rowspan="1" colspan="1">Tumor—general</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">HPU‐Net</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0035" class="usa-link" aria-describedby="acm270226-bib-0035">35</a>
</td>
<td align="left" rowspan="1" colspan="1">Tumor—general</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">SVM + AlexNet</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0036" class="usa-link" aria-describedby="acm270226-bib-0036">36</a>
</td>
<td align="left" rowspan="1" colspan="1">Tumor—general</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
<td align="left" rowspan="1" colspan="1">247 patients</td>
<td align="left" rowspan="1" colspan="1">Available at radiopaedia and kaggle</td>
<td align="left" rowspan="1" colspan="1">K‐means</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0037" class="usa-link" aria-describedby="acm270226-bib-0037">37</a>
</td>
<td align="left" rowspan="1" colspan="1">Tumor—general</td>
<td align="left" rowspan="1" colspan="1">
<p>Optical imaging (MCF endoscopy),</p>
<p>autofluorescence imaging</p>
</td>
<td align="left" rowspan="1" colspan="1">50 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>U</em>‐Net,</p>
<p>EDSR network,</p>
<p>transfer learning</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0038" class="usa-link" aria-describedby="acm270226-bib-0038">38</a>
</td>
<td align="left" rowspan="1" colspan="1">Tumor—gliomas</td>
<td align="left" rowspan="1" colspan="1">MRI + DTI</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Graph theory</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0039" class="usa-link" aria-describedby="acm270226-bib-0039">39</a>
</td>
<td align="left" rowspan="1" colspan="1">Tumor—gliomas</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
<td align="left" rowspan="1" colspan="1">100 In silico patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">Bayesian model</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0040" class="usa-link" aria-describedby="acm270226-bib-0040">40</a>
</td>
<td align="left" rowspan="1" colspan="1">Whole brain simulation</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">CNNs + GAN</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0041" class="usa-link" aria-describedby="acm270226-bib-0041">41</a>
</td>
<td align="left" rowspan="1" colspan="1">Whole brain</td>
<td align="left" rowspan="1" colspan="1">MRI + DTI + PET</td>
<td align="left" rowspan="1" colspan="1">1 patient</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Spiking neuronal networks</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0042" class="usa-link" aria-describedby="acm270226-bib-0042">42</a>
</td>
<td align="left" rowspan="1" colspan="1">Whole brain</td>
<td align="left" rowspan="1" colspan="1">MRI</td>
<td align="left" rowspan="1" colspan="1">1 patient</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Hierarchical mesoscale data assimilation</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0043" class="usa-link" aria-describedby="acm270226-bib-0043">43</a>
</td>
<td align="left" rowspan="1" colspan="1">Whole brain</td>
<td align="left" rowspan="1" colspan="1">MRI + fMRI</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Hierarchical mesoscopic data assimilation</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0044" class="usa-link" aria-describedby="acm270226-bib-0044">44</a>
</td>
<td align="left" rowspan="1" colspan="1">Alzheimer's disease</td>
<td align="left" rowspan="1" colspan="1">MRI + OCT</td>
<td align="left" rowspan="1" colspan="1">876 patients</td>
<td align="left" rowspan="1" colspan="1">Available partially at ADNI and TADPOLE Dataset</td>
<td align="left" rowspan="1" colspan="1">NeuralODEs</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The CNS, encompassing the brain and spinal cord, is one of the most complex and delicate systems in the human body. Accurately simulating and representing CNS functions is therefore essential for advancing our understanding of neurological diseases, improving diagnostic capabilities, and enabling personalized treatment strategies. Some, like Wan et al.,<a href="#acm270226-bib-0035" class="usa-link" aria-describedby="acm270226-bib-0035">
<sup>35</sup>
</a> utilize digital twin technology with semi‐supervised learning model to enhance brain image fusion and diagnostic accuracy, enabling digital twins to perform feature recognition and forecasting tasks by mapping real‐time brain imaging data into virtual space. In addition to diagnostic improvement, people are exploring the use of digital twins to support precise diagnostics, optimize personalized treatment plans, enhance medical imaging, facilitate cognitive simulations, and enhance visualization and neurosurgical applications, revolutionizing CNS healthcare.</p>
<section id="acm270226-sec-0110"><h4 class="pmc_sec_title">2.3.1. Advanced imaging techniques for digital twin models</h4>
<p>Just as a human doctor relies on fine‐detailed images for accurate diagnosis, digital twins also benefit from high‐resolution imaging to capture anatomical information for precise modeling. With this objective, Wang and Qiao<a href="#acm270226-bib-0040" class="usa-link" aria-describedby="acm270226-bib-0040">
<sup>40</sup>
</a> conducted a novel approach to enhance MRI quality using several super‐resolution techniques, including Deep Transfer Learning and GANs. Their model integrates an adaptive decomposition fusion algorithm with several super‐resolution techniques—most notably deep transfer learning and GANs<a href="#acm270226-bib-0006" class="usa-link" aria-describedby="acm270226-bib-0006">
<sup>6</sup>
</a>—to improve MRI image resolution. The process begins by breaking down MRI scans into multiple frequency sub‐bands, each capturing distinct structural layers: broader anatomical shapes are preserved in the low‐frequency bands, while high‐frequency bands retain fine‐grained pathological textures. Enhancement is performed within each band to sharpen contrast and detail, tailored to the specific properties of each frequency range. These refined components are then seamlessly fused into a single high‐resolution image, restoring spatial precision and improving the clarity of tissue boundaries. This comprehensive approach allows for more detailed visualization of subtle brain structures, ultimately aiding in more accurate diagnoses and treatment planning. Additionally, they utilized metamaterial technology to further boost MRI signal quality, achieving metrics such as a PSNR of 34.11 dB and an SSIM of 85.24%, primarily on brain MRI images. These values indicate substantial improvement in both pixel‐level fidelity and structural similarity compared to baseline MRI scans, reflecting clearer visualization of anatomical details that are critical for diagnostic precision. Therefore, this study suggests broader applications in digital twin models, offering enhanced spatial and spectral fidelity for improved diagnostics.</p>
<p>Similarly, Wang and Dremel<a href="#acm270226-bib-0037" class="usa-link" aria-describedby="acm270226-bib-0037">
<sup>37</sup>
</a> explored a deep learning approach to enhance tumor imaging resolution with MCF endoscopes. Using a cascaded network architecture with <em>U</em>‐Net and an EDSR Network, they pre‐trained the model on simulated MCF images and fine‐tuned it on glioblastoma tissue data, achieving PSNR values up to 31.6 dB and SSIM of 0.97. This method contributes to digital twin development by providing higher‐resolution visualizations crucial for tumor analysis.</p>
<p>Building on these refined images, digital twin models can now be employed for a variety of applications, including precise disease modeling, progression prediction, and cognitive simulation.</p></section><section id="acm270226-sec-0120"><h4 class="pmc_sec_title">2.3.2. Disease modeling and prediction</h4>
<p>One of the purposes of having a digital twin for the central nervous system is to detect the disease, model its progression, and make predictions for proactive treatment planning. In 2023, Sarris et al.<a href="#acm270226-bib-0036" class="usa-link" aria-describedby="acm270226-bib-0036">
<sup>36</sup>
</a> investigated the use of K‐means clustering within digital twin technology to enhance brain tumor detection. A 3D model of the brain was constructed using open‐source computer‐aided design, where tumor tissues were differentiated from normal brain tissues through clustering of pixel intensity variations. Similarly, Cen and colleagues<a href="#acm270226-bib-0033" class="usa-link" aria-describedby="acm270226-bib-0033">
<sup>33</sup>
</a> tackled the challenge of predicting disease‐specific brain atrophy in MS by employing digital twin technology. Due to the scarcity of extensive longitudinal brain MRI data, the study utilized a spline model to create normal aging trajectories as a baseline for comparison against MS patient data, allowing for the prediction of thalamic atrophy onset up to 5–6 years before clinical symptoms appear.</p>
<p>Building on these detection techniques, Hu et al.<a href="#acm270226-bib-0034" class="usa-link" aria-describedby="acm270226-bib-0034">
<sup>34</sup>
</a> introduce an advanced fuzzy system within digital twin technology to enhance brain MRI diagnostics, particularly for brain tumors. Unlike traditional binary logic systems, fuzzy systems accommodate more nuanced values, mimicking human reasoning and improving predictive accuracy. They combined this enhanced fuzzy clustering with HPU‐Net for MRI segmentation, achieving high metrics such as a DSC of 0.936 and a Jaccard coefficient of 0.845. Powered by CNN and RNN, their system offers improved segmentation accuracy and significantly reduces noise. These advanced techniques help digital twin models better handle real‐world complexity, laying a strong foundation for predictive disease modeling.</p>
<p>Following disease detection and diagnosis, ongoing monitoring and prediction become essential. Wan and Dong<a href="#acm270226-bib-0035" class="usa-link" aria-describedby="acm270226-bib-0035">
<sup>35</sup>
</a> applied semi‐supervised learning with digital twin technology to enhance brain tumor analysis. By combining S3VM and an enhanced AlexNet model, the study achieved 92.52% accuracy in tumor segmentation and feature extraction from MRI data.</p>
<p>After predicting the presence of tumors, tracking their progression is highly valuable. For instance, Guillevin et al.<a href="#acm270226-bib-0038" class="usa-link" aria-describedby="acm270226-bib-0038">
<sup>38</sup>
</a> used multimodal MRI, including fMRI, DTI, and MRS, to develop a digital replica of gliomas. These models simulate glioma progression and therapeutic response, aiding in personalized treatment planning. Chaudhuri et al.<a href="#acm270226-bib-0039" class="usa-link" aria-describedby="acm270226-bib-0039">
<sup>39</sup>
</a> also presented a progression‐predictive digital twin framework for high‐grade gliomas, dynamically incorporating updated patient MRI data with a Bayesian model to predict tumor growth. This framework not only forecasts tumor progression but also adapts radiotherapy regimens based on patient‐specific data, accounting for uncertainties in tumor biology. With digital twins, we can not only detect brain diseases such as tumors early on but also model their progression over time, enabling continuous monitoring and personalized treatment planning.</p></section><section id="acm270226-sec-0130"><h4 class="pmc_sec_title">2.3.3. Cognitive modeling and visualization</h4>
<p>Apart from neoplastic diseases like tumors, digital twins are also valuable for studying neurodegenerative conditions. A solid understanding of brain and disease simulations is enhanced by visualizing the brain's digital twin, making it more practical for clinical and research applications. To simulate human brain function, Lu et al.<a href="#acm270226-bib-0042" class="usa-link" aria-describedby="acm270226-bib-0042">
<sup>42</sup>
</a> developed a comprehensive DTB model, starting with their 2022 paper, which established a framework to replicate brain states and perform tasks like virtual deep brain stimulation. This model incorporates mesoscale data assimilation to simulate both resting and active states, laying the groundwork for cognitive task modeling. In 2023, Lu et al.<a href="#acm270226-bib-0041" class="usa-link" aria-describedby="acm270226-bib-0041">
<sup>41</sup>
</a> expanded on this foundation by improving computational efficiency and applying the DTB to real‐world cognitive experiments, demonstrating its potential for advanced research and clinical applications. Capable of simulating 86 billion neurons, this model requires substantial computational resources: over 3,500 computer nodes and more than 10,000 GPUs.</p>
<p>Similarly, Li et al.<a href="#acm270226-bib-0043" class="usa-link" aria-describedby="acm270226-bib-0043">
<sup>43</sup>
</a> introduced the DTBVis system, an interactive platform for comparing human and digital twin brains. DTBVis includes four visualization modules: the main 3D view for topological connectivity and BOLD signals, the auxiliary view for FC matrices or t‐SNE plots, the detailed view for in‐depth BOLD signals, and the timeline view for tracking temporal changes. This system allows users to explore individual brains or directly compare two, highlighting differences in BOLD signals to assess the DTB's accuracy against real human data. By advancing simulation and cognitive modeling, DTBVis is a valuable tool in digital twin and medical imaging research within neuroscience.</p>
<p>Being able to simulate general brain functions, digital twin models are applied to neurodegenerative diseases. AD, a significant public health concern, affects millions worldwide.<a href="#acm270226-bib-0045" class="usa-link" aria-describedby="acm270226-bib-0045">
<sup>45</sup>
</a> The paper by Lachinov et al.<a href="#acm270226-bib-0044" class="usa-link" aria-describedby="acm270226-bib-0044">
<sup>44</sup>
</a> addresses disease progression in AD, using MRI to monitor anatomical changes. Lachinov et al. utilized CNNs to extract features from MRI images and then applied NeuralODEs to predict the continuous evolution of disease‐related anatomical changes. Their framework leverages longitudinal data to simulate dynamic progression, offering a robust model for neurodegenerative diseases. Notably, their model extends beyond AD, applying to other conditions such as GA, where OCT captures retinal degeneration. This versatility underscores the digital twin model's potential for broad applications in predicting the progression of various degenerative diseases.</p></section></section><section id="acm270226-sec-0140"><h3 class="pmc_sec_title">2.4. Musculoskeletal system</h3>
<p>The musculoskeletal system, encompassing bones, muscles, tendons, and ligaments, is essential for movement, stability, and support. Its health is crucial for physical function and quality of life. Digital twin technology offers significant benefits for the musculoskeletal system by enabling precise, personalized models for diagnosing conditions, planning surgeries, and monitoring recovery. Same as other systems, medical imaging, such as CT, MRI, and 3D scanning, plays a key role in constructing these digital twins, allowing accurate visualization and simulation of musculoskeletal structures and their biomechanical behaviors.</p>
<p>Many studies rely on CT or its variants (e.g., Cone Beam CT, Micro‐CT) for digital twin construction in bones, joints, and musculoskeletal structures, as shown in Table <a href="#acm270226-tbl-0004" class="usa-link">4</a>, due to CT's superior capability for capturing detailed anatomy.<a href="#acm270226-bib-0046" class="usa-link" aria-describedby="acm270226-bib-0046">
<sup>46</sup>
</a> Other modalities like Optical Imaging and EMG are used selectively, but CT remains the primary choice. There are some dataset available for extremities and spine studies, but there is limited availability for dental applications. For computational models, dental studies favor traditional models like FEA, extremities use mixed statistical and 3D modeling approaches, and spine studies increasingly apply neural networks and generative models.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Summary of digital‐twin studies for musculoskeletal system.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Clinical site</th>
<th align="left" rowspan="1" colspan="1">Data modality</th>
<th align="left" rowspan="1" colspan="1">Data quantity</th>
<th align="left" rowspan="1" colspan="1">Data publicity</th>
<th align="left" rowspan="1" colspan="1">Computational models</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0047" class="usa-link" aria-describedby="acm270226-bib-0047">47</a>
</td>
<td align="left" rowspan="1" colspan="1">Dental—implants and surrounding bone</td>
<td align="left" rowspan="1" colspan="1">Cone beam CT (CBCT)</td>
<td align="left" rowspan="1" colspan="1">25 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Parametric ROM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0048" class="usa-link" aria-describedby="acm270226-bib-0048">48</a>
</td>
<td align="left" rowspan="1" colspan="1">Dental—mandible</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">13 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Non‐linear FEA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0049" class="usa-link" aria-describedby="acm270226-bib-0049">49</a>
</td>
<td align="left" rowspan="1" colspan="1">Dental—teeth</td>
<td align="left" rowspan="1" colspan="1">3D dental scans</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Statistical analysis—Spearman's correlation analysis</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0050" class="usa-link" aria-describedby="acm270226-bib-0050">50</a>
</td>
<td align="left" rowspan="1" colspan="1">Extremities—bone allografts</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">3D modeling based on DICOM standards</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0051" class="usa-link" aria-describedby="acm270226-bib-0051">51</a>
</td>
<td align="left" rowspan="1" colspan="1">Extremities—finger joints</td>
<td align="left" rowspan="1" colspan="1">Optical imaging</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Available upon request from authors</td>
<td align="left" rowspan="1" colspan="1">Spatiotemporal fourier component analysis, Monte Carlo simulations</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0052" class="usa-link" aria-describedby="acm270226-bib-0052">52</a>
</td>
<td align="left" rowspan="1" colspan="1">Extremities—foot and ankle</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">5 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request from hospital</td>
<td align="left" rowspan="1" colspan="1">CAD Catia™</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0053" class="usa-link" aria-describedby="acm270226-bib-0053">53</a>
</td>
<td align="left" rowspan="1" colspan="1">Extremities—shoulder</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Shapekey function in blender, unity engine</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0054" class="usa-link" aria-describedby="acm270226-bib-0054">54</a>
</td>
<td align="left" rowspan="1" colspan="1">Extremities—shoulder</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">24 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">3D modeling and segmentation</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0055" class="usa-link" aria-describedby="acm270226-bib-0055">55</a>
</td>
<td align="left" rowspan="1" colspan="1">Extremities—upper arm</td>
<td align="left" rowspan="1" colspan="1">EMG, thermography</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Available</td>
<td align="left" rowspan="1" colspan="1">SVM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0056" class="usa-link" aria-describedby="acm270226-bib-0056">56</a>
</td>
<td align="left" rowspan="1" colspan="1">Spine vertebral body—cervical vertebrae</td>
<td align="left" rowspan="1" colspan="1">x‐ray</td>
<td align="left" rowspan="1" colspan="1">319 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request from authors</td>
<td align="left" rowspan="1" colspan="1">MobileNetV2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0057" class="usa-link" aria-describedby="acm270226-bib-0057">57</a>
</td>
<td align="left" rowspan="1" colspan="1">Spine vertebral body—lumbar spine (L2‐L5)</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Available</td>
<td align="left" rowspan="1" colspan="1">FEM, GPR</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0058" class="usa-link" aria-describedby="acm270226-bib-0058">58</a>
</td>
<td align="left" rowspan="1" colspan="1">Spine vertebral body—vertebra (Spine)</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">1 patient</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">ReconGAN (DCGAN with finite element analysis)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0059" class="usa-link" aria-describedby="acm270226-bib-0059">59</a>
</td>
<td align="left" rowspan="1" colspan="1">Spine vertebral body—vertebra (spine)</td>
<td align="left" rowspan="1" colspan="1">CT, micro‐CT</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">ReconGAN (DCGAN with finite element analysis)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0060" class="usa-link" aria-describedby="acm270226-bib-0060">60</a>
</td>
<td align="left" rowspan="1" colspan="1">Dental—maxillary canine</td>
<td align="left" rowspan="1" colspan="1">CBCT</td>
<td align="left" rowspan="1" colspan="1">100 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request from authors</td>
<td align="left" rowspan="1" colspan="1">3D UNet</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In the following sections, we will explore current digital twin applications in Dentistry, extremities, and spine vertebral body.</p>
<section id="acm270226-sec-0150"><h4 class="pmc_sec_title">2.4.1. Dental</h4>
<p>In dentistry, digital twin technology has undergone a transformative evolution, with pioneering research extending its applications from basic clinical practices to complex personalized treatments. It commenced with the foundational work by Zilberman et al.,<a href="#acm270226-bib-0061" class="usa-link" aria-describedby="acm270226-bib-0061">
<sup>61</sup>
</a> which validated the accuracy and clinical relevance of virtual orthodontic models by comparing measurements from digital twins against traditional caliper measurements on plaster models. This initial validation set the stage for more intricate applications in clinics. Such applications address conditions like periodontal disease, which affects 20%–50% of the global population.<a href="#acm270226-bib-0062" class="usa-link" aria-describedby="acm270226-bib-0062">
<sup>62</sup>
</a>
</p>
<p>Building on these initial validations, subsequent research has focused on digital twins in dentistry focusing on enhancing diagnostic and surgical precision. For instance, Kim et al.<a href="#acm270226-bib-0049" class="usa-link" aria-describedby="acm270226-bib-0049">
<sup>49</sup>
</a> used the Opto TOP‐HE 3D scanner to create detailed virtual models for measuring tooth wear. By repeating scans 10–20 times, they generated full‐mouth dental casts, which were then rendered and analyzed with Rapidform software. In contrast to the statistical methods used by them, which involved repeated scanning, the study by Swaitly et al.<a href="#acm270226-bib-0060" class="usa-link" aria-describedby="acm270226-bib-0060">
<sup>60</sup>
</a> represents a significant advancement in dental imaging through deep learning. Their approach utilizes a <em>U</em>‐Net architecture to automate the segmentation of maxillary impacted canines directly from CBCT images. The model achieved a DSC of 0.99, which is nearly perfect between inference and ground truth. This method enhances the precision of the 3D models generated and drastically reduces the time required for segmentation. Swaitly asserts that the creation of fast and reliable 3D models supports the implementation of digital twins in personalized dental care.</p>
<p>Beyond orthodontics, digital twins have expanded their applications to implant dentistry, where the recent study by Ahn et al.<a href="#acm270226-bib-0047" class="usa-link" aria-describedby="acm270226-bib-0047">
<sup>47</sup>
</a> introduces a novel parametric ROM that leverages finite element analysis to simulate real‐time stress distributions in dental implants and surrounding bone structures. By integrating patient‐specific medical imaging data from CBCT scans into a digital twin framework, this model enables dentists to visualize and modify implant placement plans dynamically, enhancing the precision and success of implant surgeries.</p>
<p>In the mandible, Demir et al.<a href="#acm270226-bib-0048" class="usa-link" aria-describedby="acm270226-bib-0048">
<sup>48</sup>
</a> used CT scans to create a digital twin of the human lower jaw, employing the Mimics Innovation Suite to segment cortical and cancellous bone based on HU thresholds. They optimized the mesh for efficiency and anatomical accuracy, and then assigned material properties, such as elasticity modulus and Poisson's ratio, to simulate biomechanical behavior accurately. This approach allowed the digital twin to predict the mandible's biomechanical response under impact conditions, enhancing its clinical relevance for maxillofacial trauma analysis.</p></section><section id="acm270226-sec-0160"><h4 class="pmc_sec_title">2.4.2. Extremities</h4>
<p>Extremities disorders, such as arthritis and fractures, significantly impact mobility and quality of life, with arthritis affecting 21.2% of adults in the United States.<a href="#acm270226-bib-0063" class="usa-link" aria-describedby="acm270226-bib-0063">
<sup>63</sup>
</a> Digital twin technology is transforming the management of these conditions by providing clinicians with precise visualization and simulation tools to better understand and treat complex musculoskeletal issues. This technology enables detailed anatomical insights, enhances surgical precision, optimizes allograft matching, and supports real‐time rehabilitation and monitoring. The following subsections explore four critical areas where digital twins are advancing limb healthcare: Visualization and Simulation, Precision in Surgical Interventions, Allograft Matching, and Real‐Time Rehabilitation and Monitoring.</p>
<p>Visualization and simulation are crucial for understanding musculoskeletal anatomy, providing clinicians with dynamic tools to manage complex conditions effectively. In shoulder treatment, Kim et al.<a href="#acm270226-bib-0053" class="usa-link" aria-describedby="acm270226-bib-0053">
<sup>53</sup>
</a> created interactive 3D visualizations of shoulder mechanics, using CT scans converted to Standard Tessellation Language (STL) format for detailed modeling in Blender. Through animation, the models simulated shoulder movements and were integrated into Unity, enabling an interactive application that provides comprehensive insights into shoulder muscles such as the Deltoid, Infraspinatus, and Supraspinatus. This detailed simulation aids clinicians and patients in visualizing shoulder mechanics for more effective management and rehabilitation. Similarly, Verweij et al.<a href="#acm270226-bib-0054" class="usa-link" aria-describedby="acm270226-bib-0054">
<sup>54</sup>
</a> developed virtual models of the humerus and scapula to analyze shoulder instability after anterior dislocations. Using 3D segmentation and registration, they positioned the humeral head within a glenoid coordinate system, enabling precise measurements of joint space and positioning.</p>
<p>Building on the anatomical insights gained from visualization and simulation, digital twins are instrumental in enhancing surgical precision, particularly in lower limb procedures like total ankle arthroplasty and knee surgery. Hernigou et al.<a href="#acm270226-bib-0052" class="usa-link" aria-describedby="acm270226-bib-0052">
<sup>52</sup>
</a> employed CT scans to refine implant placement, converting CT images to DICOM format before creating detailed 3D models using CAD tools like Catia™. This digitalization process seamlessly integrates high‐fidelity anatomical data into the digital twin environment, where artificial intelligence and machine learning analyze and simulate kinematic behaviors. Such precision in modeling and simulation optimized implant placement and contributing to personalized orthopedic care.</p>
<p>After precise surgical interventions, ensuring accurate anatomical compatibility in bone grafting is essential for successful outcomes. Vila et al.<a href="#acm270226-bib-0050" class="usa-link" aria-describedby="acm270226-bib-0050">
<sup>50</sup>
</a> introduced the BeST‐Graft Viewer, a digital twin‐based tool that optimizes the matching of bone allografts to recipient sites, specifically in femoral applications for lower‐limb surgeries. This system digitizes grafts through CT scans, storing data in DICOM format, and reconstructs 3D models for detailed anatomical assessments in 2D, 3D, and multiplanar views. The BeST‐Graft Viewer automates the graft‐recipient matching protocol, facilitating precise measurements and minimizing manual errors. By integrating real‐time data management with interactive visualization, this system exemplifies how digital twins enhance orthopedic care, where anatomical matching precision is crucial to the success of grafting procedures.</p>
<p>After surgery and precise allograft matching, rehabilitation and continuous monitoring are crucial for ensuring successful recovery and long‐term functional improvement. Digital twin technology enhances these processes by providing real‐time feedback and personalized insights, allowing clinicians to adjust rehabilitation protocols effectively. Ha et al.<a href="#acm270226-bib-0055" class="usa-link" aria-describedby="acm270226-bib-0055">
<sup>55</sup>
</a> introduced a hybrid upper‐arm‐geared exoskeleton system with an anatomical digital twin model, integrating VR torque feedback and MR remote healthcare monitoring through EMG and thermographic sensors to assess muscle activity and temperature distribution, thus ensuring safety. By simulating physical interactions, the system offers immersive training with realistic torque feedback, supporting rehabilitation through interactive virtual environments. Similarly, Ioussoufovitch and Diop<a href="#acm270226-bib-0051" class="usa-link" aria-describedby="acm270226-bib-0051">
<sup>51</sup>
</a> developed a Time‐Domain Diffuse Optical Imaging system for monitoring rheumatoid arthritis in finger joints, specifically targeting the proximal interphalangeal joints. Using tissue‐mimicking phantoms and spatiotemporal Fourier analysis, their system distinguishes between different RA states, enhancing monitoring precision and enabling earlier detection of treatment failures.</p>
<p>Together, these approaches illustrate how digital twins create a comprehensive pipeline in limb orthopedic care, from understanding anatomy and enhancing surgical precision to optimizing allograft matching and supporting effective rehabilitation, ultimately facilitating a seamless and personalized continuum of care.</p></section><section id="acm270226-sec-0170"><h4 class="pmc_sec_title">2.4.3. Spine vertebral body</h4>
<p>Spinal health is essential, with conditions like disc degeneration affecting over 70% of people under 50 and more than 90% of those over 50.<a href="#acm270226-bib-0064" class="usa-link" aria-describedby="acm270226-bib-0064">
<sup>64</sup>
</a> Digital twin technology plays a critical role in managing spine‐related disorders by enabling accurate simulations, real‐time monitoring, and personalized treatment planning. This section discusses three key applications: ReconGAN for Orthopedic Interventions, Integration with Real‐Time Systems, and Digital Twins in Forensic Anthropology and Morphological Analysis.</p>
<p>Ahmadian et al.<a href="#acm270226-bib-0058" class="usa-link" aria-describedby="acm270226-bib-0058">
<sup>58</sup>
</a> used ReconGAN to simulate vertebroplasty, crucial for vertebral stability in cancer patients, by generating realistic trabecular bone microstructures and predicting the effects of bone cement injections. In another study,<a href="#acm270226-bib-0059" class="usa-link" aria-describedby="acm270226-bib-0059">
<sup>59</sup>
</a> they applied ReconGAN to predict fracture risks from micro‐CT images, producing high‐fidelity models to simulate fracture responses. These applications demonstrate ReconGAN's ability to enhance clinical outcomes by personalizing spinal interventions and assessing risks in musculoskeletal treatment.</p>
<p>Significant advancements in digital twin technology have enabled real‐time prediction and analysis for lumbar and cervical spine health. He et al.<a href="#acm270226-bib-0057" class="usa-link" aria-describedby="acm270226-bib-0057">
<sup>57</sup>
</a> developed a real‐time lumbar spine model integrating FEM, motion capture, IK, and machine learning. This framework uses FEM to simulate lumbar biomechanics, motion capture for movement data, and IK for accurate joint modeling. A GPR algorithm predicts biomechanical responses such as facet contact force and intradiscal pressure, visualized dynamically in Unity3D for enhanced understanding of spine mechanics.</p>
<p>Similarly, Moztarzadeh et al.<a href="#acm270226-bib-0056" class="usa-link" aria-describedby="acm270226-bib-0056">
<sup>56</sup>
</a> leveraged the MobileNetV2 algorithm for real‐time diagnosis of cervical vertebral maturation via lateral cephalometric radiographs. Integrated within a blockchain‐based metaverse platform, this lightweight algorithm ensures rapid analysis and secure data management. Together, these systems highlight the use of digital twins for accurate, real‐time insights in spine diagnostics and treatment planning, enhancing patient outcomes in spine‐related healthcare.</p></section></section><section id="acm270226-sec-0180"><h3 class="pmc_sec_title">2.5. Breast</h3>
<p>Studies focusing on digital twins for breast conditions primarily concentrate on breast cancer, including specific subtypes like TNBC.<a href="#acm270226-bib-0065" class="usa-link" aria-describedby="acm270226-bib-0065">
<sup>65</sup>
</a> Table <a href="#acm270226-tbl-0005" class="usa-link">5</a> provides an overview of these studies. To obtain detailed imaging data, these investigations mainly employ MRI and ultrasound‐based techniques such as elastography. However, the majority of datasets remain non‐public. In some instances, data access is possible through permission requests or by registration. Additionally, there is a growing trend toward integrating diverse data types—including clinical records and biological samples—to enhance the precision and practical applicability of digital twins in breast health. The computational modeling methods utilized are highly varied, spanning from traditional finite‐element models and graph‐based approaches to advanced machine learning algorithms and biologically inspired models.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0005"><h4 class="obj_head">TABLE 5.</h4>
<div class="caption p"><p>Summary of digital‐twin studies for breast.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Clinical site</th>
<th align="left" rowspan="1" colspan="1">Data modality</th>
<th align="left" rowspan="1" colspan="1">Data quantity</th>
<th align="left" rowspan="1" colspan="1">Data publicity</th>
<th align="left" rowspan="1" colspan="1">Computational models</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0066" class="usa-link" aria-describedby="acm270226-bib-0066">66</a>
</td>
<td align="left" rowspan="1" colspan="1">Breast cancer</td>
<td align="left" rowspan="1" colspan="1">3D ultrasound</td>
<td align="left" rowspan="1" colspan="1">2 phantoms</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">3D annotation, spatial sensor</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0067" class="usa-link" aria-describedby="acm270226-bib-0067">67</a>
</td>
<td align="left" rowspan="1" colspan="1">Breast cancer</td>
<td align="left" rowspan="1" colspan="1">Elastography, ultrasound</td>
<td align="left" rowspan="1" colspan="1">10 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Finite‐element modeling</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0056" class="usa-link" aria-describedby="acm270226-bib-0056">56</a>
</td>
<td align="left" rowspan="1" colspan="1">Breast cancer</td>
<td align="left" rowspan="1" colspan="1">MRI, blood samples</td>
<td align="left" rowspan="1" colspan="1">64 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">LR, DTR, RFR, GBA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0068" class="usa-link" aria-describedby="acm270226-bib-0068">68</a>
</td>
<td align="left" rowspan="1" colspan="1">TNBC</td>
<td align="left" rowspan="1" colspan="1">DCE‐MRI, DW‐MRI</td>
<td align="left" rowspan="1" colspan="1">56 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Biologically‐based models</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0069" class="usa-link" aria-describedby="acm270226-bib-0069">69</a>
</td>
<td align="left" rowspan="1" colspan="1">Breast cancer</td>
<td align="left" rowspan="1" colspan="1">SEER cancer registry</td>
<td align="left" rowspan="1" colspan="1">87,674 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon registration</td>
<td align="left" rowspan="1" colspan="1">Graph‐based approach, MPoM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0070" class="usa-link" aria-describedby="acm270226-bib-0070">70</a>
</td>
<td align="left" rowspan="1" colspan="1">Breast cancer, HF</td>
<td align="left" rowspan="1" colspan="1">MRI, clinical data</td>
<td align="left" rowspan="1" colspan="1">922 patients</td>
<td align="left" rowspan="1" colspan="1">Public</td>
<td align="left" rowspan="1" colspan="1">Biomechanics, machine learning</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Digital twin technology is playing a pivotal role in breast healthcare, with diverse applications in imaging, diagnosis, treatment, and clinical data integration. For instance, Jiang et al.<a href="#acm270226-bib-0066" class="usa-link" aria-describedby="acm270226-bib-0066">
<sup>66</sup>
</a> developed an innovative semi‐automated 3D annotation method for breast ultrasound using a digital twin model, which significantly enhanced both the accuracy and repeatability of annotations. This work beautifully illustrates how digital twins can serve as a bridge between conventional 2D ultrasound and advanced 3D anatomical visualization, offering a more precise, dynamic representation of breast anatomy. The resulting improvements in accuracy and clinical feasibility underscore the immense value of integrating digital twins into routine breast imaging, paving the way for more standardized and efficient diagnostic procedures.</p>
<p>Building upon these advancements in imaging, the role of digital twins in breast cancer diagnosis is further underscored by the work of Sayed et al.,<a href="#acm270226-bib-0067" class="usa-link" aria-describedby="acm270226-bib-0067">
<sup>67</sup>
</a> who applied FEM to simulate breast tissue behavior during elastography. This study demonstrated that 3D FEM could serve as a functional biomarker for distinguishing between benign and malignant tumors, providing a non‐invasive, data‐driven approach to early cancer detection. Such applications reinforce the practical relevance of digital twins in replicating complex biomechanical properties of breast tissues. Similarly, Moztarzadeh et al.<a href="#acm270226-bib-0056" class="usa-link" aria-describedby="acm270226-bib-0056">
<sup>56</sup>
</a> utilized multimodal data and machine learning to construct a digital twin model for breast cancer diagnosis and treatment, significantly improving diagnostic accuracy and enabling real‐time disease monitoring. This work shows how digital twins can integrate diverse data sources to create dynamic, patient‐specific models.</p>
<p>As illustrated in Figure <a href="#acm270226-fig-0005" class="usa-link">5</a>, the process flow provides a detailed overview of the FEM steps used for simulating breast tissue during multicompression in ultrasound elastography. The figure outlines the comprehensive workflow, from defining the problem as a solid structure specifying material properties and building the object's geometry. The process includes selecting element type and size, meshing, and ensuring element quality. Once the quality is acceptable, boundary conditions and contact elements are applied, followed by the multi‐compression application and solution phase. The graphical results are evaluated to ensure they are adequate, and if so, the tumor and normal tissue strain calculations are performed. If the results are unsatisfactory, the process revisits the geometry for adjustments. This structured approach assists in differentiating benign from malignant tumors based on their biomechanical properties.</p>
<figure class="fig xbox font-sm" id="acm270226-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/329a/12370374/4326ee44db58/ACM2-26-e70226-g001.jpg" loading="lazy" id="jats-graphic-9" height="338" width="600" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/acm270226-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of FEM for simulating breast tissue multicompression in ultrasound elastography. Adapted from.<a href="#acm270226-bib-0067" class="usa-link" aria-describedby="acm270226-bib-0067">
<sup>67</sup>
</a>
</p></figcaption></figure><p>Further advancing the use of digital twins, Moore et al.<a href="#acm270226-bib-0069" class="usa-link" aria-describedby="acm270226-bib-0069">
<sup>69</sup>
</a> introduced SynTwin, a model that leverages synthetic data for clinical outcome prediction. The enhanced predictive accuracy achieved with SynTwin, particularly in breast cancer mortality prediction, demonstrates the immense value of digital twins in synthesizing clinical data to support personalized treatment strategies. Additionally, Wu et al.<a href="#acm270226-bib-0068" class="usa-link" aria-describedby="acm270226-bib-0068">
<sup>68</sup>
</a> developed a biologically‐based digital twin model using multiparametric MRI to predict individual responses to NAST in TNBC patients, achieving high predictive accuracy. Finally, Gamage et al.<a href="#acm270226-bib-0070" class="usa-link" aria-describedby="acm270226-bib-0070">
<sup>70</sup>
</a> introduced the 12 LABOURS Digital Twin Platform, which integrates multiple data sources to automate computational physiology workflows, thereby enhancing the efficiency of clinical trials and supporting breast cancer diagnosis and treatment.</p></section><section id="acm270226-sec-0190"><h3 class="pmc_sec_title">2.6. Urinary system</h3>
<p>Digital twin research focusing on the urinary tract utilizes various imaging techniques—including CT scans, MRI, and endoscopic images—to model conditions such as renal masses, kidney tumors, and prostate cancer, as highlighted in Table <a href="#acm270226-tbl-0006" class="usa-link">6</a>. Besides, some researchers investigated the impact of digital twin technology in urologic oncology, specifically focusing on the treatments like RARP<a href="#acm270226-bib-0071" class="usa-link" aria-describedby="acm270226-bib-0071">
<sup>71</sup>
</a>. Moreover, three‐dimensional virtual and printed models are widely employed, especially in surgical planning and complex anatomical reconstructions. While most datasets in this domain are not publicly accessible, a notable large‐scale kidney condition dataset is available through Kaggle, demonstrating efforts to improve data availability. The computational modeling approaches used are diverse, ranging from traditional methods like 3DVM analysis to some newly‐invented deep learning algorithms. The integration of such diverse modeling and imaging modalities demonstrates the field's growing commitment to improving precision and clinical applicability in urinary health.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0006"><h4 class="obj_head">TABLE 6.</h4>
<div class="caption p"><p>Summary of digital‐twin studies for urinary system.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Clinical site</th>
<th align="left" rowspan="1" colspan="1">Data modality</th>
<th align="left" rowspan="1" colspan="1">Data quantity</th>
<th align="left" rowspan="1" colspan="1">Data publicity</th>
<th align="left" rowspan="1" colspan="1">Computational models</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0072" class="usa-link" aria-describedby="acm270226-bib-0072">72</a>
</td>
<td align="left" rowspan="1" colspan="1">Renal mass</td>
<td align="left" rowspan="1" colspan="1">CT, endoscopic images</td>
<td align="left" rowspan="1" colspan="1">20 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">CNN, computer vision</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0073" class="usa-link" aria-describedby="acm270226-bib-0073">73</a>
</td>
<td align="left" rowspan="1" colspan="1">Kidney cancer</td>
<td align="left" rowspan="1" colspan="1">CT, MRI, 3D virtual models</td>
<td align="left" rowspan="1" colspan="1">9 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Virtual reality, surgical planning</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0074" class="usa-link" aria-describedby="acm270226-bib-0074">74</a>
</td>
<td align="left" rowspan="1" colspan="1">Renal mass</td>
<td align="left" rowspan="1" colspan="1">CT, 3D virtual models</td>
<td align="left" rowspan="1" colspan="1">152 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">Propensity‐score matching, 3DVM analysis</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0075" class="usa-link" aria-describedby="acm270226-bib-0075">75</a>
</td>
<td align="left" rowspan="1" colspan="1">Kidney stones, cysts, tumor, chronic kidney disease</td>
<td align="left" rowspan="1" colspan="1">CT‐radiography</td>
<td align="left" rowspan="1" colspan="1">12,446 images</td>
<td align="left" rowspan="1" colspan="1">Public dataset (Kaggle)</td>
<td align="left" rowspan="1" colspan="1">HGCNN, EfficientNetB0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0076" class="usa-link" aria-describedby="acm270226-bib-0076">76</a>
</td>
<td align="left" rowspan="1" colspan="1">Prostate cancer</td>
<td align="left" rowspan="1" colspan="1">mp‐MRI, 3D printed models</td>
<td align="left" rowspan="1" colspan="1">1075 patients</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">3D virtual models, meta‐analysis</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="acm270226-sec-0200"><h4 class="pmc_sec_title">2.6.1. Kidney</h4>
<p>Recent advancements in digital twin technology have fundamentally transformed surgical planning, simulation, and diagnostic imaging, offering unprecedented insights and precision in kidney healthcare. Amparore and Sica<a href="#acm270226-bib-0072" class="usa-link" aria-describedby="acm270226-bib-0072">
<sup>72</sup>
</a> provide a compelling illustration of this transformation through the application of computer vision and machine learning in AR‐guided robotic partial nephrectomy. Their work demonstrates the power of automatic 3D virtual image overlays in real‐time surgical environments. By integrating intraoperative endoscopic images with patient‐specific 3D models derived from CT scans, they showed that digital twins not only improved registration accuracy but also minimized the need for manual adjustments, thereby enhancing both surgical efficiency and precision.</p>
<p>In the context of preoperative surgical planning, Checcucci and Amparore<a href="#acm270226-bib-0073" class="usa-link" aria-describedby="acm270226-bib-0073">
<sup>73</sup>
</a> employed three‐dimensional virtual models and metaverse technology to enhance minimally invasive partial nephrectomy procedures. This innovative approach allowed surgeons to interact with hyperaccurate 3D models within a virtual reality environment, facilitating comprehensive preoperative discussions and optimizing clamping strategies. The integration of digital twins into the planning phase led to greater precision in surgical execution by providing a detailed visualization of patient‐specific anatomy and supporting data‐driven decision‐making. Likewise, Grosso et al.<a href="#acm270226-bib-0074" class="usa-link" aria-describedby="acm270226-bib-0074">
<sup>74</sup>
</a> demonstrated that employing 3D virtual models during robot‐assisted partial nephrectomy significantly improved functional preservation and reduced surgical complications. Furthermore, Sasikaladevi and Revathi<a href="#acm270226-bib-0075" class="usa-link" aria-describedby="acm270226-bib-0075">
<sup>75</sup>
</a> utilized CT imaging to construct a digital twin model of the renal system aimed at early disease detection. Their HGNN achieved high accuracy in classifying renal pathologies, highlighting the efficacy of digital twins as non‐invasive and reliable diagnostic tools that facilitate early detection and effective management of kidney diseases.</p></section><section id="acm270226-sec-0210"><h4 class="pmc_sec_title">2.6.2. Prostate</h4>
<p>Recent advancements in digital twin technology have fundamentally transformed surgical planning, simulation, and diagnostic imaging, offering unprecedented insights and precision in prostate healthcare.</p>
<p>Sarhan et al.<a href="#acm270226-bib-0076" class="usa-link" aria-describedby="acm270226-bib-0076">
<sup>76</sup>
</a> explored the impact of digital twin technology in urologic oncology, particularly in the context of RARP. Their systematic review and meta‐analysis leveraged patient‐specific mp‐MRI data to create both virtual and 3D‐printed prostate models. These models played a crucial role in preoperative planning and intraoperative navigation, significantly reducing positive surgical margin rates. Specifically, the group using 3D models had an 8.1% positive surgical margin rate compared to 22.2% in the conventional surgery group. This study exemplifies how digital twin technology allows for the generation of highly detailed prostate models, demonstrating its efficacy in enhancing surgical precision and minimizing complications.</p></section></section><section id="acm270226-sec-0220"><h3 class="pmc_sec_title">2.7. Digestive system</h3>
<p>Digital twin research related to the digestive system employs a broad spectrum of data modalities—including MRI, CT imaging, histopathological slides, and clinical records—to investigate conditions such as colorectal cancer, NAFLD, and type 2 diabetes, as summarized in Table <a href="#acm270226-tbl-0007" class="usa-link">7</a>. Additionally, less conventional imaging techniques like 4D‐CT and holographic AR are utilized, particularly in the study of liver tumors. While most datasets are not publicly accessible or require special requests for access, some are shared under specific conditions to facilitate research. The computational models applied in these studies are diverse, ranging from traditional approaches like Monte Carlo simulations<a href="#acm270226-bib-0077" class="usa-link" aria-describedby="acm270226-bib-0077">
<sup>77</sup>
</a> and closed‐loop systems to advanced techniques such as machine learning algorithms and augmented reality integration<a href="#acm270226-bib-0078" class="usa-link" aria-describedby="acm270226-bib-0078">
<sup>78</sup>
</a>. This progression signifies a growing trend toward incorporating modern computational methods—such as ML and the IoT<a href="#acm270226-bib-0079" class="usa-link" aria-describedby="acm270226-bib-0079">
<sup>79</sup>
</a>—to enhance the accuracy and functionality of digital twins in digestive health.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0007"><h4 class="obj_head">TABLE 7.</h4>
<div class="caption p"><p>Summary of digital‐twin studies for digestive system.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Clinical site</th>
<th align="left" rowspan="1" colspan="1">Data modality</th>
<th align="left" rowspan="1" colspan="1">Data quantity</th>
<th align="left" rowspan="1" colspan="1">Data publicity</th>
<th align="left" rowspan="1" colspan="1">Computational models</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0080" class="usa-link" aria-describedby="acm270226-bib-0080">80</a>
</td>
<td align="left" rowspan="1" colspan="1">Colorectal cancer</td>
<td align="left" rowspan="1" colspan="1">Histological slides, clinical data</td>
<td align="left" rowspan="1" colspan="1">118 cases</td>
<td align="left" rowspan="1" colspan="1">Not public</td>
<td align="left" rowspan="1" colspan="1">QuPath Software, ML</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0081" class="usa-link" aria-describedby="acm270226-bib-0081">81</a>
</td>
<td align="left" rowspan="1" colspan="1">Ascending colon conditions</td>
<td align="left" rowspan="1" colspan="1">MRI, DMP</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">DMP, SPH, LSM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0082" class="usa-link" aria-describedby="acm270226-bib-0082">82</a>
</td>
<td align="left" rowspan="1" colspan="1">IPSS</td>
<td align="left" rowspan="1" colspan="1">CTA, 3D modeling</td>
<td align="left" rowspan="1" colspan="1">32 Dogs</td>
<td align="left" rowspan="1" colspan="1">Contact corresponding author</td>
<td align="left" rowspan="1" colspan="1">Materialise Mimics, 3‐matic</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0083" class="usa-link" aria-describedby="acm270226-bib-0083">83</a>
</td>
<td align="left" rowspan="1" colspan="1">Post‐hepatectomy PHT</td>
<td align="left" rowspan="1" colspan="1">MRI, flowmetry, CT</td>
<td align="left" rowspan="1" colspan="1">47 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">0D closed‐loop model</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0084" class="usa-link" aria-describedby="acm270226-bib-0084">84</a>
</td>
<td align="left" rowspan="1" colspan="1">Respiratory liver tumor</td>
<td align="left" rowspan="1" colspan="1">4D‐CT, holographic AR</td>
<td align="left" rowspan="1" colspan="1">2 dogs</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">CNN, AR fusion</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0085" class="usa-link" aria-describedby="acm270226-bib-0085">85</a>
</td>
<td align="left" rowspan="1" colspan="1">NAFLD</td>
<td align="left" rowspan="1" colspan="1">MRI, Monte Carlo</td>
<td align="left" rowspan="1" colspan="1">68 samples</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">Single‐R2*, Dual‐R2*, Monte Carlo</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0086" class="usa-link" aria-describedby="acm270226-bib-0086">86</a>
</td>
<td align="left" rowspan="1" colspan="1">Type 2 diabetes, MAFLD</td>
<td align="left" rowspan="1" colspan="1">MRI‐PDFF, clinical data</td>
<td align="left" rowspan="1" colspan="1">319 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">ML, IoT</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="acm270226-sec-0230"><h4 class="pmc_sec_title">2.7.1. Liver</h4>
<p>Recent research underscores the transformative potential of digital twin technology in liver disease management, encompassing imaging diagnosis, surgical planning, and personalized metabolic care. Jackson et al.<a href="#acm270226-bib-0082" class="usa-link" aria-describedby="acm270226-bib-0082">
<sup>82</sup>
</a> demonstrated the clinical applicability of 3D models for IPSS in dogs, illustrating that the use of 3D digital twin models significantly enhances procedure planning by providing detailed visualization of liver anatomy. This improvement in visualization translates to enhanced accuracy and safety of interventions compared to traditional 2D imaging techniques. In a similar vein, Shrestha and Esparza<a href="#acm270226-bib-0085" class="usa-link" aria-describedby="acm270226-bib-0085">
<sup>85</sup>
</a> employed digital twins to simulate hepatic steatosis by generating MRI signals based on fat droplet morphology and comparing different models under various conditions. Their findings emphasized the robustness of the single‐R2* model for fat fraction estimation in clinical settings, highlighting the value of digital twins in providing reliable, non‐invasive quantification of liver disease—a crucial component for accurate diagnosis and ongoing monitoring.</p>
<p>In the realm of liver surgery, Golse et al.<a href="#acm270226-bib-0083" class="usa-link" aria-describedby="acm270226-bib-0083">
<sup>83</sup>
</a> employed a zero‐dimensional closed‐loop mathematical model to create patient‐specific digital twins capable of accurately predicting surgical risks, such as postoperative portal hypertension. This capability enhances personalized surgical planning by allowing clinicians to anticipate complications and tailor interventions to the specific needs of individual patients, thereby improving surgical outcomes. Furthermore, Shi et al.<a href="#acm270226-bib-0084" class="usa-link" aria-describedby="acm270226-bib-0084">
<sup>84</sup>
</a> integrated digital twin models with AR to address the challenges posed by respiratory motion during liver puncture procedures. Their approach led to significant improvements in accuracy and real‐time navigation, demonstrating the practical benefits of digital twins in complex surgical environments where precision is paramount. In the context of personalized metabolic care, Joshi et al.<a href="#acm270226-bib-0086" class="usa-link" aria-describedby="acm270226-bib-0086">
<sup>86</sup>
</a> uses continuous glucose monitoring, IoT sensors, and AI algorithms to create digital twins for managing metabolic diseases, resulting in significant improvements in glycemic control and liver health outcomes compared to standard care practices.</p></section><section id="acm270226-sec-0240"><h4 class="pmc_sec_title">2.7.2. Intestine</h4>
<p>Digital twin technology in healthcare is significantly advancing our understanding and management of digestive system functions, particularly through innovations in colorectal cancer modeling and simulations of intestinal dynamics.</p>
<p>To begin with, the study by Peshkova et al.<a href="#acm270226-bib-0080" class="usa-link" aria-describedby="acm270226-bib-0080">
<sup>80</sup>
</a> offers a compelling example of how a digital twin‐based approach can revolutionize pathological analysis in colorectal cancer. Collaborating with a multidisciplinary team to evaluate and annotate samples, they developed a highly detailed digital model that integrates histological data with advanced imaging techniques. This model, and the resulting colorectal cancer case database, improved the efficiency of pathological evaluations by 50% while increasing diagnostic accuracy by 20%. These advances significantly streamlined the diagnostic process, enhancing both research capabilities and clinical decision‐making.</p>
<p>Additionally, the work of Schutt et al.<a href="#acm270226-bib-0081" class="usa-link" aria-describedby="acm270226-bib-0081">
<sup>81</sup>
</a> represents another critical leap forward in simulating intestinal dynamics and optimizing drug delivery methods. Utilizing a DCM, the researchers created a digital twin that replicates the hydrodynamic conditions of the human colon. By employing advanced simulation techniques such as DMP and SPH, they achieved a precise simulation of flow rates and shear stresses, providing a physiologically accurate environment for testing drug dissolution. This method surpasses traditional approaches, signaling a paradigm shift in how we model and understand gastrointestinal processes.</p></section></section><section id="acm270226-sec-0250"><h3 class="pmc_sec_title">2.8. Respiratory system</h3>
<p>Digital twin research for the respiratory system, as detailed in Table <a href="#acm270226-tbl-0008" class="usa-link">8</a>, spans a wide range of conditions, such as COVID‐19, lung cancer, pneumonia, and pulmonary embolism. To investigate these diseases, primary data modalities like CT scans, MRI, chest x‐rays, and various clinical data inputs are utilized. Moreover, the incorporation of IoT sensors in respiratory studies highlights the emphasis on real‐time monitoring capabilities. The computational models applied are diverse, ranging from advanced deep learning architectures—such as <em>U</em>‐Net and YOLOv8<a href="#acm270226-bib-0087" class="usa-link" aria-describedby="acm270226-bib-0087">
<sup>87</sup>
</a>—to frameworks involving mixed reality and the IoMT.<a href="#acm270226-bib-0088" class="usa-link" aria-describedby="acm270226-bib-0088">
<sup>88</sup>
</a> Data availability is inconsistent; while some datasets are publicly accessible—for example, a substantial collection of chest x‐ray images on Kaggle—others require special access requests. This progression indicates a growing focus on leveraging machine learning and interconnected technologies to enhance the accuracy and practical applications of digital twins in respiratory health monitoring and diagnostics.</p>
<section class="tw xbox font-sm" id="acm270226-tbl-0008"><h4 class="obj_head">TABLE 8.</h4>
<div class="caption p"><p>Summary of digital‐twin studies for respiratory system.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1">Clinical site</th>
<th align="left" rowspan="1" colspan="1">Data modality</th>
<th align="left" rowspan="1" colspan="1">Data quantity</th>
<th align="left" rowspan="1" colspan="1">Data publicity</th>
<th align="left" rowspan="1" colspan="1">Computational models</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0089" class="usa-link" aria-describedby="acm270226-bib-0089">89</a>
</td>
<td align="left" rowspan="1" colspan="1">Lung cancer, pulmonary embolism</td>
<td align="left" rowspan="1" colspan="1">Clinical data, CT, MR</td>
<td align="left" rowspan="1" colspan="1">90 patients, 1372 controls</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">rAC‐GAN, IoMT, mixed reality</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0090" class="usa-link" aria-describedby="acm270226-bib-0090">90</a>
</td>
<td align="left" rowspan="1" colspan="1">Lung diseases</td>
<td align="left" rowspan="1" colspan="1">EIT, CT</td>
<td align="left" rowspan="1" colspan="1">20 patients</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">IR‐Net (<em>U</em>‐Net), DL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0091" class="usa-link" aria-describedby="acm270226-bib-0091">91</a>
</td>
<td align="left" rowspan="1" colspan="1">COVID‐19, lung opacity, pneumonia, tuberculosis</td>
<td align="left" rowspan="1" colspan="1">Chest x‐ray, IoT sensors</td>
<td align="left" rowspan="1" colspan="1">23,442 images</td>
<td align="left" rowspan="1" colspan="1">Public datasets (Kaggle)</td>
<td align="left" rowspan="1" colspan="1">YOLOv8, AI</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="#acm270226-bib-0092" class="usa-link" aria-describedby="acm270226-bib-0092">92</a>
</td>
<td align="left" rowspan="1" colspan="1">Pneumonia</td>
<td align="left" rowspan="1" colspan="1">CT</td>
<td align="left" rowspan="1" colspan="1">6012 subjects</td>
<td align="left" rowspan="1" colspan="1">Available upon request</td>
<td align="left" rowspan="1" colspan="1">Anchor‐free object detection, multi‐branch network</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/acm270226-tbl-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>To truly understand the transformative potential of digital twin technology in respiratory healthcare, it is essential to consider its diverse applications in lung health monitoring, cancer diagnosis, and surgical simulation. The reviewed studies serve as compelling examples of how this technology is pushing the boundaries of traditional medical practices, creating more dynamic and responsive healthcare solutions. The paper by Zhu et al.<a href="#acm270226-bib-0090" class="usa-link" aria-describedby="acm270226-bib-0090">
<sup>90</sup>
</a> introduces a novel combination of EIT, digital twin technology, and deep learning algorithms for lung monitoring. By utilizing a 3D digital twin lung model and a deep learning‐based image reconstruction algorithm, the study demonstrates a significant improvement in image quality compared to traditional EIT methods, achieving an SSIM of 0.737 under noisy conditions. This improvement underscores the potential of digital twin technology in creating realistic, dynamic models of specific organs—here, the lung—that can be used for more accurate and robust diagnostic purposes.</p>
<p>Building on these advancements, another notable example is the work by Li,<a href="#acm270226-bib-0092" class="usa-link" aria-describedby="acm270226-bib-0092">
<sup>92</sup>
</a> which proposes an innovative framework for real‐time object detection using digital twin technology. This framework leverages digital twins to create a dynamic and responsive environment for medical imaging analysis. By incorporating a five‐level feature pyramid to capture fine and coarse details and employing a dual‐branch detection head for accurate localization and sizing, the study demonstrates superior performance on various datasets, including COCO and pneumonia detection datasets, surpassing state‐of‐the‐art methods like RetinaNet. This example illustrates how digital twin technology enhances object detection capabilities in medical applications, providing both higher accuracy and real‐time efficiency, which are critical for timely and precise healthcare interventions. Li's contributions emphasize the versatility of digital twin technology in handling complex medical imaging tasks.</p>
<p>In addition to these diagnostic and imaging advancements, Avanzato et al.<a href="#acm270226-bib-0091" class="usa-link" aria-describedby="acm270226-bib-0091">
<sup>91</sup>
</a> illustrate the application of digital twin technology for real‐time lung health monitoring and disease diagnosis through the Lung‐DT framework. This system integrates IoT sensors and AI algorithms to create a comprehensive digital representation of a patient's respiratory health, effectively generating a digital twin of the respiratory system that can continuously monitor health status. The YOLOv8 neural network enables accurate classification of chest x‐rays into different lung disease categories with high precision and recall, achieving an average accuracy of 96.8%. This framework highlights the advantage of using digital twins for personalized and continuous lung health monitoring. This work showcases the potential of digital twins to transform personalized health monitoring into a continuous, proactive practice, ultimately contributing to better patient outcomes.</p>
<p>Lastly, moving towards applications in surgical practice, the study by Tai et al.<a href="#acm270226-bib-0089" class="usa-link" aria-describedby="acm270226-bib-0089">
<sup>89</sup>
</a> focuses on the use of digital twin technology in remote lung cancer surgery simulation, showcasing its capacity to enhance surgical precision and immersion. By enhancing a CNN network and introducing the DL‐CNN model, the study achieves high accuracy in quantifying clot burden in pulmonary embolism patients, supported by advanced haptic feedback for real‐time surgical simulation. This approach leverages the digital twin to create a realistic, data‐driven surgical environment, allowing for real‐time visualization and tactile feedback, which vastly surpasses traditional methods. It emphasizes their potential for improving both the management of clinical data and the effectiveness of remote surgical training, making it possible to simulate complex procedures in a highly controlled and immersive manner.</p></section><section id="acm270226-sec-0260"><h3 class="pmc_sec_title">2.9. Cross‐system comparative summary</h3>
<p>To conclude this system‐specific review, we synthesize several key findings that emerged across different organ systems, highlighting both common technical bottlenecks and system‐specific modeling challenges.</p>
<p>Despite the diversity of clinical applications and anatomical focus, several key findings consistently emerge across systems. First, computational burden remains a major limitation in physics‐based modeling frameworks, particularly in cardiovascular and musculoskeletal applications that rely heavily on CFD and FEM. These approaches, while precise, are often computationally prohibitive for real‐time clinical use. Second, the challenge of multimodal data integration is especially evident in CNS models, where the alignment of MRI, fMRI, and DTI data is hindered by inconsistent formats and varying resolution quality. Third, dynamic systems such as the lungs and gastrointestinal tract suffer from motion‐related artifacts, which compromise the fidelity of simulations due to temporally and spatially variable organ deformation. Finally, different systems require varying levels of abstraction in their modeling approaches: cardiovascular twins generally emphasize macro‐structural fidelity through 3D geometric reconstruction, whereas CNS twins must incorporate higher‐order features such as functional connectivity<a href="#acm270226-bib-0093" class="usa-link" aria-describedby="acm270226-bib-0093">
<sup>93</sup>
</a> and network‐level dynamics. These observations point toward the need for flexible frameworks that can accommodate both high‐resolution physical modeling and abstract system‐level representations.</p>
<p>Data integration also remains a substantial challenge, particularly in CNS applications where combining MRI, fMRI, and DTI<a href="#acm270226-bib-0094" class="usa-link" aria-describedby="acm270226-bib-0094">
<sup>94</sup>
</a> requires complex co‐registration and harmonization pipelines. In practice, inconsistent data quality and lack of shared formats across modalities often hinder such integration. Developing standardized multi‐modal datasets and promoting FAIR (Findable, Accessible, Interoperable, and Reusable)<a href="#acm270226-bib-0095" class="usa-link" aria-describedby="acm270226-bib-0095">
<sup>95</sup>
</a> data principles may help address this issue.</p>
<p>Meanwhile, motion‐related artifacts are particularly problematic in systems such as the lungs and gastrointestinal tract, where organ deformation introduces temporal and spatial inconsistencies. While some studies attempt to compensate using deformable models or gated imaging, these methods remain in early stages and are rarely generalizable. More robust modeling of dynamic boundaries or motion‐aware training data could improve generalizability.</p>
<p>Another key difference lies in the level of abstraction required by each system. Cardiovascular models tend to work at the macro‐structural scale using patient‐specific 3D imaging (e.g., CT or OCT), whereas CNS models must integrate structural, functional, and sometimes network‐level information, often requiring high‐level abstractions like connectomics.<a href="#acm270226-bib-0096" class="usa-link" aria-describedby="acm270226-bib-0096">
<sup>96</sup>
</a> This diversity complicates the development of unified modeling pipelines and highlights the importance of tailoring model architectures to the physiological complexity of each system.</p>
<p>In summary, while system‐specific approaches are necessary, many foundational challenges—such as computational scalability, robust multimodal data integration, and anatomical dynamics—are shared. Addressing these through cross‐system methodological innovations (e.g., modular simulation engines,<a href="#acm270226-bib-0097" class="usa-link" aria-describedby="acm270226-bib-0097">
<sup>97</sup>
</a> federated data sharing,<a href="#acm270226-bib-0098" class="usa-link" aria-describedby="acm270226-bib-0098">
<sup>98</sup>
</a> or multimodal AI architectures) may accelerate the realization of generalizable and clinically viable digital twins. These findings highlight the need for both system‐specific optimization and generalizable methodological frameworks, paving the way toward scalable and impactful clinical applications.</p></section></section><section id="acm270226-sec-0270"><h2 class="pmc_sec_title">3. DISCUSSION</h2>
<p>This review identified several domain‐spanning and system‐specific findings that reflect both current limitations and innovation trends in medical digital twin development.</p>
<p>Advancing digital twin technology in the medical field with the support of medical imaging presents a highly promising direction. However, realizing its full potential requires overcoming several technical challenges. These challenges can be broadly categorized into computational performance bottlenecks, data scarcity, accuracy and validation, and data integration issues, providing a structured framework for addressing the obstacles in this field.</p>
<p>Data scarcity presents a critical challenge in developing digital twin models, fundamentally limiting the diversity and representativeness of the datasets available for training. This scarcity introduces biases, as models trained on narrow population samples struggle to generalize effectively, often resulting in overfitting—where a model performs well on specific training data but fails with broader, more diverse populations. For instance, similar challenges arise in musculoskeletal systems: Swaitly et al.<a href="#acm270226-bib-0060" class="usa-link" aria-describedby="acm270226-bib-0060">
<sup>60</sup>
</a> claimed that their maxillary impacted canines segmentation model encountered overfitting due to insufficient and non‐diverse training data. Same in neurological modeling, limited data led Cen et al.<a href="#acm270226-bib-0033" class="usa-link" aria-describedby="acm270226-bib-0033">
<sup>33</sup>
</a> and Lachinov et al.<a href="#acm270226-bib-0044" class="usa-link" aria-describedby="acm270226-bib-0044">
<sup>44</sup>
</a> to employ simpler methodologies like spline functions and NeuralODEs instead of deep learning models to avoid overfitting, which constrained their ability to represent multiple sclerosis and Alzheimer's disease comprehensively. These adaptations, while addressing immediate data limitations, come at the expense of model precision and applicability. Thus, data scarcity compels researchers to strike a balance between feasibility and model sophistication.</p>
<p>While this is a universal challenge in the medical imaging modeling field, positively, this challenge also drives innovation. Emerging strategies—particularly those driven by AI and machine learning—have begun to address the limitations of small and imbalanced datasets. While traditional data augmentation methods involve: flip, rotate, crop, zoom and so on,<a href="#acm270226-bib-0004" class="usa-link" aria-describedby="acm270226-bib-0004">
<sup>4</sup>
</a> advanced methods like synthetic data using GAN<a href="#acm270226-bib-0006" class="usa-link" aria-describedby="acm270226-bib-0006">
<sup>6</sup>
</a> and Diffusion Model<a href="#acm270226-bib-0099" class="usa-link" aria-describedby="acm270226-bib-0099">
<sup>99</sup>
</a> start getting popular. This novel approach was first brought into the medical field by Frid‐Adar for liver lesion classification.<a href="#acm270226-bib-0100" class="usa-link" aria-describedby="acm270226-bib-0100">
<sup>100</sup>
</a> He reported that GAN‐generated synthetic liver lesion images improved CNN classification performance, increasing sensitivity from 78.6% to 85.7% and specificity from 88.4% to 92.4%, demonstrating the tangible benefits of data‐driven augmentation in medical imaging. Soon this method has been explored and advanced by others in medical image,<a href="#acm270226-bib-0101" class="usa-link" aria-describedby="acm270226-bib-0101">
<sup>101</sup>
</a><sup>, </sup><a href="#acm270226-bib-0102" class="usa-link" aria-describedby="acm270226-bib-0102">
<sup>102</sup>
</a><sup>, </sup><a href="#acm270226-bib-0103" class="usa-link" aria-describedby="acm270226-bib-0103">
<sup>103</sup>
</a> and physiological data.<a href="#acm270226-bib-0027" class="usa-link" aria-describedby="acm270226-bib-0027">
<sup>27</sup>
</a> Given the promising results in general medical data generation, GAN‐ and diffusion‐based models offer valuable opportunities for future researchers to enhance digital twin construction, providing diverse and synthetic data that can help overcome limitations in real‐world datasets.</p>
<p>To advance the use of GANs and diffusion models for medical data augmentation in digital twin systems, it is crucial to move beyond conceptual mentions and establish reproducible experimental standards. To further harness the power of these generative models, future studies could implement controlled experimental pipelines where synthetic datasets generated by GANs or diffusion models are evaluated alongside real patient data using consistent performance metrics (e.g., Dice similarity coefficient,<a href="#acm270226-bib-0104" class="usa-link" aria-describedby="acm270226-bib-0104">
<sup>104</sup>
</a> FID score,<a href="#acm270226-bib-0105" class="usa-link" aria-describedby="acm270226-bib-0105">
<sup>105</sup>
</a> classification accuracy). It would also be beneficial to standardize key parameters such as latent vector size, augmentation ratio, and image resolution to enable reproducibility across studies. For instance, GANs trained on organ‐specific datasets (e.g., brain MRI or cardiac CT) should report architecture type (e.g., StyleGAN2,<a href="#acm270226-bib-0106" class="usa-link" aria-describedby="acm270226-bib-0106">
<sup>106</sup>
</a> cGAN<a href="#acm270226-bib-0107" class="usa-link" aria-describedby="acm270226-bib-0107">
<sup>107</sup>
</a>), training stability measures, and synthetic‐real mixing ratios during training. Similarly, diffusion models could be assessed on their ability to retain anatomical realism and modality fidelity across different organ systems. By encouraging such standardized protocols and reporting guidelines, the field can more effectively benchmark generative augmentation strategies and accelerate the development of robust, data‐rich digital twins.</p>
<p>Computational complexity remains a core challenge in implementing digital twin technology within medical imaging. Even with abundant data, the immense computational resources required to run these models limits their practicality in clinical settings. Digital twins aim to replicate complex physiological systems with high fidelity, which inherently demands extensive computational power—especially critical for real‐time applications. For example, Lu et al.<a href="#acm270226-bib-0041" class="usa-link" aria-describedby="acm270226-bib-0041">
<sup>41</sup>
</a> demonstrated that creating a digital twin of the brain required over 3500 nodes and 10,000 GPUs, a level of demand that is impractical for most healthcare facilities. Similarly, Ahn et al.<a href="#acm270226-bib-0047" class="usa-link" aria-describedby="acm270226-bib-0047">
<sup>47</sup>
</a> faced substantial difficulties in applying FEA for biomechanical modeling due to its intense computational requirements, making real‐time clinical integration exceptionally challenging. HPC (high‐performance computing, which involves the use of powerful computing clusters and parallel processing techniques to perform complex calculations) and parallel processing have been adopted to address these computational demands, but they come with limitations of their own. The high costs and infrastructure requirements of HPC resources restrict their accessibility, creating a barrier to widespread digital twin implementation in diverse healthcare settings. For instance, Ahmadian et al.<a href="#acm270226-bib-0059" class="usa-link" aria-describedby="acm270226-bib-0059">
<sup>59</sup>
</a> found that even with access to HPC, reconstructing vertebral microstructures was computationally prohibitive, demonstrating that simply adding computational power is not always a viable solution. These cases illustrate that computational constraints are not just technical hurdles; they are crucial issues that impact the scalability and integration of digital twins, particularly where real‐time model updates are necessary to support dynamic clinical decision‐making. Another notable example of clinical application can be seen in cardiovascular interventions. Studies utilizing patient‐specific digital twins constructed from OCT or CT data have demonstrated the ability to simulate coronary blood flow and mechanical stress distribution, aiding in the assessment of lesion severity and treatment planning. For instance, Morris<a href="#acm270226-bib-0108" class="usa-link" aria-describedby="acm270226-bib-0108">
<sup>108</sup>
</a> used a digital twin model to simulate fractional flow reserve non‐invasively, and reported a diagnostic accuracy comparable to invasive its measurement, significantly reducing procedural risks and improving patient triage. Such applications illustrate how digital twin models, when integrated into clinical workflows, can lead to faster decision‐making, reduced procedural burden, and ultimately better patient outcomes. To address these challenges, researchers are rethinking using latest ML algorithms to improve the model structures and workflow to balance fidelity with feasibility rather than relying solely on computational power. For example, in model wise, Zhou et al.<a href="#acm270226-bib-0109" class="usa-link" aria-describedby="acm270226-bib-0109">
<sup>109</sup>
</a> achieved a 32.2% reduction in inference time through model pruning, decreasing UNet model depth and complexity while preserving segmentation accuracy above 90%. This efficiency gain supports the feasibility of deploying deep learning models for near‐real‐time inference in clinical environments. Similarly, Koopsen et al.<a href="#acm270226-bib-0029" class="usa-link" aria-describedby="acm270226-bib-0029">
<sup>29</sup>
</a> reduced parameters to simplify cardiovascular models, and lowered computational demands while retaining essential accuracy. From workflow perspective, Banerjee et al.<a href="#acm270226-bib-0016" class="usa-link" aria-describedby="acm270226-bib-0016">
<sup>16</sup>
</a> optimized their pipeline by constructing 3D heart models from 2D cine MR images, enhancing efficiency. Though not universally applicable, these strategies highlight a direction worth exploring for constructing efficient, accurate models.</p>
<p>Achieving precise registration and alignment is a crucial challenge in implementing digital twin DT technology in medical imaging, especially for patient‐specific modeling, where anatomical accuracy is essential. This challenge is particularly evident in musculoskeletal applications, where aligning digital twin replicas with skeletal structures is complicated by the rigidity and anatomical variability of bones. Any misalignment between the digital twin and the patient's actual anatomy can lead to significant errors in diagnosis and treatment planning. During the construction phase, precise registration ensures that the digital twin accurately mirrors the patient's unique anatomical features, which allows clinicians to simulate and plan interventions with a higher degree of confidence. In dynamic contexts, such as the shoulder joint, studies by Verweij et al.<a href="#acm270226-bib-0054" class="usa-link" aria-describedby="acm270226-bib-0054">
<sup>54</sup>
</a> and Kim et al.<a href="#acm270226-bib-0053" class="usa-link" aria-describedby="acm270226-bib-0053">
<sup>53</sup>
</a> emphasize the need for adaptive algorithms to manage movement variability, which is vital for real‐time updates in musculoskeletal digital twins. Similarly, Vila et al.<a href="#acm270226-bib-0050" class="usa-link" aria-describedby="acm270226-bib-0050">
<sup>50</sup>
</a> and Ahmadian et al.<a href="#acm270226-bib-0059" class="usa-link" aria-describedby="acm270226-bib-0059">
<sup>59</sup>
</a> highlight the importance of precise alignment in procedures like bone grafting and vertebroplasty, where successful outcomes heavily rely on exact imaging data alignment. Although approaches like that of Amparore et al.<a href="#acm270226-bib-0072" class="usa-link" aria-describedby="acm270226-bib-0072">
<sup>72</sup>
</a> show improved registration by integrating intraoperative endoscopic images with CT‐based 3D models, robust solutions are still required to tackle alignment challenges effectively across diverse applications.</p>
<p>DL advancements are emerging as promising solutions for enhancing registration accuracy in complex medical imaging tasks. Models such as Segment Anything Model 2,<a href="#acm270226-bib-0110" class="usa-link" aria-describedby="acm270226-bib-0110">
<sup>110</sup>
</a> VoxelMorph,<a href="#acm270226-bib-0111" class="usa-link" aria-describedby="acm270226-bib-0111">
<sup>111</sup>
</a> nn<em>U</em>‐Net,<a href="#acm270226-bib-0112" class="usa-link" aria-describedby="acm270226-bib-0112">
<sup>112</sup>
</a> and DeepReg<a href="#acm270226-bib-0113" class="usa-link" aria-describedby="acm270226-bib-0113">
<sup>113</sup>
</a> have shown considerable success in aligning multimodal datasets like MRI‐to‐CT and in applications like tumor tracking and soft tissue registration. These DL models leverage advanced learning techniques to manage the inherent variability in medical imaging, making them more adaptable than traditional registration methods. Although not yet widely applied to skeletal digital twins, the demonstrated adaptability and precision of these models suggest they could help bridge current gaps, particularly in managing the rigid and intricate nature of bone structures in digital twin applications.</p>
<p>While existing literature has explored key applications of digital twins in medical imaging, several significant gaps remain. First, the limited availability of longitudinal, multimodal datasets blocks robust model generalization across diverse patient populations, as most studies rely on small, single‐institution cohorts. Second, insufficient research on real‐time model updating hinders seamless clinical integration, particularly for dynamic organs such as the lungs and gastrointestinal tract, where continuous data acquisition and model adaptation are essential. Third, the lack of standardized evaluation benchmarks prevents fair comparison of competing methods and slows consensus on best practices.</p>
<p>Beyond these core technical issues, additional challenges warrant attention. Data privacy and security remain critical when aggregating large‐scale imaging data; future work should explore privacy preserving strategies, such as federated learning<a href="#acm270226-bib-0114" class="usa-link" aria-describedby="acm270226-bib-0114">
<sup>114</sup>
</a> and secure multiparty computation, to build robust models without compromising confidentiality.</p>
<p>Moreover, explainability and transparency in AI‐driven digital twin models are currently limited. Many existing approaches operate as “black boxes,” which undermines clinician trust and hampers adoption. Research should focus on integrating explainable AI techniques, such as Grad‐CAM,<a href="#acm270226-bib-0115" class="usa-link" aria-describedby="acm270226-bib-0115">
<sup>115</sup>
</a> saliency maps,<a href="#acm270226-bib-0116" class="usa-link" aria-describedby="acm270226-bib-0116">
<sup>116</sup>
</a> and SHAP,<a href="#acm270226-bib-0117" class="usa-link" aria-describedby="acm270226-bib-0117">
<sup>117</sup>
</a> to provide interpretable insights into model decisions. At the same time, interdisciplinary collaboration is essential: successfully developing clinically viable digital twins requires close coordination among radiologists, surgeons, engineers, data scientists, ethicists, and health economists to ensure that models address real‐world clinical workflows and constraints.</p>
<p>Despite these hurdles, ongoing advancements in AI offer promising solutions to address many of these challenges and drive innovation in digital twin development.</p>
<p>Looking ahead, artificial intelligence is poised to play an increasingly central role in digital twin development and application within medical imaging. Deep learning models such as CNNs and Transformer‐based architectures facilitate advanced image reconstruction and feature extraction, enabling more accurate anatomical and functional representations. Generative models—including GANs<a href="#acm270226-bib-0006" class="usa-link" aria-describedby="acm270226-bib-0006">
<sup>6</sup>
</a> and diffusion models—address data scarcity by producing synthetic multimodal datasets, while domain adaptation techniques help harmonize inputs from MRI, CT, ultrasound, and other sources. Furthermore, reinforcement learning can optimize real‐time decision‐making in dynamic simulation scenarios, such as modeling respiratory and cardiac cycles. As AI frameworks continue to mature, we anticipate fully automated pipelines—from image acquisition through personalized simulation—becoming feasible, thereby ushering in a new era of data‐driven precision medicine. In summary, while implementing digital twin technology in medical imaging presents numerous challenges—from data scarcity and computational complexity to registration—each obstacle also serves as an impetus for innovation. As we navigate these complexities, the advancements made bring us closer to the ultimate goal of improving personalized medicine and patient outcomes, demonstrating the vast potential of digital twin technology in healthcare.</p></section><section id="acm270226-sec-0280"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>The exploration of medical imaging applications within digital twin technology reveals its transformative potential across numerous healthcare fields. Through a comprehensive analysis of advancements, challenges, and emerging research, this review illustrates that digital twins represent more than mere technological innovation; they are pivotal tools poised to revolutionize personalized medicine. By integrating research across cardiovascular, neurological, musculoskeletal, and other systems, we demonstrate the profound impact of digital twins in fostering a future where predictive, precision‐based healthcare becomes the standard. These virtual replicas, formed by combining high‐resolution imaging with advanced computational models, enable patient‐specific treatments and diagnostics. Despite challenges such as data scarcity and computational complexity, these obstacles have catalyzed innovative solutions, leading to practical clinical applications like optimized cardiovascular care, neurological disease modeling, and precision‐guided musculoskeletal surgeries. Looking forward, the ongoing development of digital twin technologies holds the promise of a proactive, personalized healthcare future, driving enhanced patient outcomes and advancing the field of medicine. This review, therefore, serves as a guide for researchers and clinicians, encouraging them to further harness the transformative potential of digital twin technology in healthcare.</p></section><section id="acm270226-sec-0290"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTION</h2>
<p>
<strong>Feng Zhao</strong>: Conceptualization; methodology; investigation; data curation; formal analysis; writing—original draft; writing—review &amp; editing; visualization; project administration. <strong>Yizhou Wu</strong>: Conceptualization; methodology; investigation; data curation; formal analysis; writing—partial original draft; writing—review &amp; editing; visualization; project administration. <strong>Mingzhe Hu</strong>: Writing—review &amp; editing; supervision. <strong>Chih‐Wei Chang</strong>: Writing—review &amp; editing; supervision. <strong>Ruirui Liu</strong>: Writing—review &amp; editing; supervision. <strong>Richard Qiu</strong>: Writing—review &amp; editing; supervision. <strong>Xiaofeng Yang</strong>: Conceptualization; writing—review &amp; editing; supervision; funding acquisition.</p></section><section id="acm270226-sec-0310"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="acm270226-sec-0300" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research is supported in part by the National Institutes of Health under Award Number R01CA272991, R01EB032680, and R01DE033512.</p></section><section id="notes1"><p>

Zhao F, Wu Y, Hu M, et al. Current progress of digital twin construction using medical imaging. J Appl Clin Med Phys. 2025;26:e70226. 10.1002/acm2.70226

</p></section><section id="acm270226-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="acm270226-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="acm270226-bib-0001">
<span class="label">1.</span><cite>
Glaessgen E, Stargel D. The digital twin paradigm for future NASA and US Air Force vehicles. Presented at: 53rd <em>AIAA/ASME/ASCE/AHS/ASC Structures, Structural Dynamics and Materials Conference</em>
. Honolulu, Hawaii; 2012: April 23–26, doi: 10.2514/6.2012-1818
</cite> [<a href="https://doi.org/10.2514/6.2012-1818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0002">
<span class="label">2.</span><cite>
Trayanova NA, Prakosa A, Up digital and personal: how heart digital twins can transform heart patient care. Heart Rhythm
2024;21(1):89‐99. doi: 10.1016/j.hrthm.2023.10.019

</cite> [<a href="https://doi.org/10.1016/j.hrthm.2023.10.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10872898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37871809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heart%20Rhythm&amp;title=Up%20digital%20and%20personal:%20how%20heart%20digital%20twins%20can%20transform%20heart%20patient%20care&amp;author=NA%20Trayanova&amp;author=A%20Prakosa&amp;volume=21&amp;issue=1&amp;publication_year=2024&amp;pages=89-99&amp;pmid=37871809&amp;doi=10.1016/j.hrthm.2023.10.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0003">
<span class="label">3.</span><cite>
LeCun Y, Lo Bottou, Bengio Y, Haffner P, Gradient‐based learning applied to document recognition. Proc IEEE. 1998;86(11):2278‐2324. doi: 10.1109/5.726791
</cite> [<a href="https://doi.org/10.1109/5.726791" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20IEEE&amp;title=Gradient%E2%80%90based%20learning%20applied%20to%20document%20recognition&amp;author=Y%20LeCun&amp;author=Bottou%20Lo&amp;author=Y%20Bengio&amp;author=P%20Haffner&amp;volume=86&amp;issue=11&amp;publication_year=1998&amp;pages=2278-2324&amp;doi=10.1109/5.726791&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0004">
<span class="label">4.</span><cite>
Krizhevsky A, Sutskever I, Hinton GE, Imagenet classification with deep convolutional neural networks. Commun ACM. 2012;60(6):84‐90. doi: 10.1145/3065386
</cite> [<a href="https://doi.org/10.1145/3065386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20ACM&amp;title=Imagenet%20classification%20with%20deep%20convolutional%20neural%20networks&amp;author=A%20Krizhevsky&amp;author=I%20Sutskever&amp;author=GE%20Hinton&amp;volume=60&amp;issue=6&amp;publication_year=2012&amp;pages=84-90&amp;doi=10.1145/3065386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0005">
<span class="label">5.</span><cite>
Simonyan K, Zisserman A. Very deep convolutional networks for large‐scale image recognition. Presented at: 
3rd International Conference on Learning Representations (ICLR)
. San Diego, CA; 2015: May 7–9.</cite>
</li>
<li id="acm270226-bib-0006">
<span class="label">6.</span><cite>
Goodfellow I, Pouget‐Abadie J, Mirza M, et al. Generative adversarial networks. Commun ACM. 2020;63(11):139‐144. doi: 10.1145/3422622
</cite> [<a href="https://doi.org/10.1145/3422622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20ACM&amp;title=Generative%20adversarial%20networks&amp;author=I%20Goodfellow&amp;author=J%20Pouget%E2%80%90Abadie&amp;author=M%20Mirza&amp;volume=63&amp;issue=11&amp;publication_year=2020&amp;pages=139-144&amp;doi=10.1145/3422622&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0007">
<span class="label">7.</span><cite>
Radford A, Metz L, Chintala S, Unsupervised representation learning with deep convolutional generative adversarial networks. CoRR. 2015;doi:abs/1511.06434</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=CoRR&amp;title=Unsupervised%20representation%20learning%20with%20deep%20convolutional%20generative%20adversarial%20networks&amp;author=A%20Radford&amp;author=L%20Metz&amp;author=S%20Chintala&amp;publication_year=2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0008">
<span class="label">8.</span><cite>
Zhu J‐Y, Park T, Isola P, Efros AA, Unpaired image‐to‐image translation using cycle‐consistent adversarial networks. IEEE International Conference on Computer Vision. 2017:2223‐2232. doi: 10.1109/ICCV.2017.244
</cite> [<a href="https://doi.org/10.1109/ICCV.2017.244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0009">
<span class="label">9.</span><cite>
Ho J, Jain A, Abbeel P, Denoising diffusion probabilistic models. Advances in Neural Information Processing Systems. 2020;33:6840‐6851. doi: 10.48550/arXiv.2006.11239
</cite> [<a href="https://doi.org/10.48550/arXiv.2006.11239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Ho%20J,%20Jain%20A,%20Abbeel%20P,%20Denoising%20diffusion%20probabilistic%20models.%20Advances%20in%20Neural%20Information%20Processing%20Systems.%202020;33:6840%E2%80%906851.%20doi:%2010.48550/arXiv.2006.11239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0010">
<span class="label">10.</span><cite>
Song J, Meng C, Ermon S. Denoising diffusion implicit models. Presented at: <em>International Conference on Learning Representations (ICLR)</em>
; 2021: May 3–7, Virtual Conference.</cite>
</li>
<li id="acm270226-bib-0011">
<span class="label">11.</span><cite>
Nichol AQ, Dhariwal P, Improved denoising diffusion probabilistic models. Proceedings of the 38th International Conference on Machine Learning (ICML 2021). 2021;139:8162‐8171.</cite> [<a href="https://scholar.google.com/scholar_lookup?Nichol%20AQ,%20Dhariwal%20P,%20Improved%20denoising%20diffusion%20probabilistic%20models.%20Proceedings%20of%20the%2038th%20International%20Conference%20on%20Machine%20Learning%20(ICML%202021).%202021;139:8162%E2%80%908171." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0012">
<span class="label">12.</span><cite>
Vaswani A, Attention is all you need. Advances in Neural Information Processing Systems, 30 (NIPS 2017). 2017:6000‐6010. doi: 10.48550/arXiv.1706.03762
</cite> [<a href="https://doi.org/10.48550/arXiv.1706.03762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Vaswani%20A,%20Attention%20is%20all%20you%20need.%20Advances%20in%20Neural%20Information%20Processing%20Systems,%2030%20(NIPS%202017).%202017:6000%E2%80%906010.%20doi:%2010.48550/arXiv.1706.03762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0013">
<span class="label">13.</span><cite>
Bert DevlinJ, : Pre‐training of deep bidirectional transformers for language understanding. Proceedings of the 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies (NAACL‐HLT 2019). 2019:4171‐4186. doi: 10.48550/arXiv.1810.04805
</cite> [<a href="https://doi.org/10.48550/arXiv.1810.04805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0014">
<span class="label">14.</span><cite>
Brown TB, Language models are few‐shot learners. Advances in Neural Information Processing Systems (NeurIPS 2020). 2020:1877‐1901. doi: 10.48550/arXiv.2005.14165
</cite> [<a href="https://doi.org/10.48550/arXiv.2005.14165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0015">
<span class="label">15.</span><cite>
Page MJ, McKenzie JE, Bossuyt PM, et al. The PRISMA 2020 statement: an updated guideline for reporting systematic reviews. BMJ. 2021;372:n71. doi: 10.1136/bmj.n71

</cite> [<a href="https://doi.org/10.1136/bmj.n71" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8005924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33782057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=The%20PRISMA%202020%20statement:%20an%20updated%20guideline%20for%20reporting%20systematic%20reviews&amp;author=MJ%20Page&amp;author=JE%20McKenzie&amp;author=PM%20Bossuyt&amp;volume=372&amp;publication_year=2021&amp;pages=n71&amp;pmid=33782057&amp;doi=10.1136/bmj.n71&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0016">
<span class="label">16.</span><cite>
Banerjee A, Camps J, Zacur E, et al. A completely automated pipeline for 3D reconstruction of human heart from 2D cine magnetic resonance slices. Philos Trans A Math Phys Eng Sci. 2021;379(2212)20200257. doi: 10.1098/rsta.2020.0257

</cite> [<a href="https://doi.org/10.1098/rsta.2020.0257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8543046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34689630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philos%20Trans%20A%20Math%20Phys%20Eng%20Sci&amp;title=A%20completely%20automated%20pipeline%20for%203D%20reconstruction%20of%20human%20heart%20from%202D%20cine%20magnetic%20resonance%20slices&amp;author=A%20Banerjee&amp;author=J%20Camps&amp;author=E%20Zacur&amp;volume=379&amp;issue=2212&amp;publication_year=2021&amp;pages=20200257&amp;pmid=34689630&amp;doi=10.1098/rsta.2020.0257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0017">
<span class="label">17.</span><cite>
Dubs L, Charitatos V, Buoso S, et al. Assessment of extracranial carotid artery disease using digital twins—A pilot study. Neuroimage Clin. 2023;38:103435. doi: 10.1016/j.nicl.2023.103435

</cite> [<a href="https://doi.org/10.1016/j.nicl.2023.103435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10238877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37245493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage%20Clin&amp;title=Assessment%20of%20extracranial%20carotid%20artery%20disease%20using%20digital%20twins%E2%80%94A%20pilot%20study&amp;author=L%20Dubs&amp;author=V%20Charitatos&amp;author=S%20Buoso&amp;volume=38&amp;publication_year=2023&amp;pages=103435&amp;pmid=37245493&amp;doi=10.1016/j.nicl.2023.103435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0018">
<span class="label">18.</span><cite>
Loewe A, Martínez Díaz P, Nagel C, Sánchez J, Cardiac digital twin modeling. Innovative Treatment Strategies for Clinical Electrophysiology. 2022:111‐134. doi: 10.1007/978-981-19-6649-1_7
</cite> [<a href="https://doi.org/10.1007/978-981-19-6649-1_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Innovative%20Treatment%20Strategies%20for%20Clinical%20Electrophysiology&amp;author=A%20Loewe&amp;author=P%20Mart%C3%ADnez%20D%C3%ADaz&amp;author=C%20Nagel&amp;author=J%20S%C3%A1nchez&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0019">
<span class="label">19.</span><cite>
Rouhollahi A, Willi JN, Haltmeier S, et al. CardioVision: a fully automated deep learning package for medical image segmentation and reconstruction generating digital twins for patients with aortic stenosis. Comput Med Imaging Graph. 2023;109:102289. doi: 10.1016/j.compmedimag.2023.102289

</cite> [<a href="https://doi.org/10.1016/j.compmedimag.2023.102289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10599298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37633032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Med%20Imaging%20Graph&amp;title=CardioVision:%20a%20fully%20automated%20deep%20learning%20package%20for%20medical%20image%20segmentation%20and%20reconstruction%20generating%20digital%20twins%20for%20patients%20with%20aortic%20stenosis&amp;author=A%20Rouhollahi&amp;author=JN%20Willi&amp;author=S%20Haltmeier&amp;volume=109&amp;publication_year=2023&amp;pages=102289&amp;pmid=37633032&amp;doi=10.1016/j.compmedimag.2023.102289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0020">
<span class="label">20.</span><cite>
Renaudin CP, Barbier B, Roriz R, Revel D, Amiel M, Coronary arteries: new design for three‐dimensional arterial phantoms. Article. Radiology. 1994;190(2):579‐582. doi: 10.1148/radiology.190.2.8284422

</cite> [<a href="https://doi.org/10.1148/radiology.190.2.8284422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8284422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Radiology&amp;title=Coronary%20arteries:%20new%20design%20for%20three%E2%80%90dimensional%20arterial%20phantoms.%20Article&amp;author=CP%20Renaudin&amp;author=B%20Barbier&amp;author=R%20Roriz&amp;author=D%20Revel&amp;author=M%20Amiel&amp;volume=190&amp;issue=2&amp;publication_year=1994&amp;pages=579-582&amp;pmid=8284422&amp;doi=10.1148/radiology.190.2.8284422&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0021">
<span class="label">21.</span><cite>
Galappaththige S, Gray RA, Costa CM, Niederer S, Pathmanathan P, Credibility assessment of patient‐specific computational modeling using patient‐specific cardiac modeling as an exemplar. PLoS Comput Biol. 2022;18(10):e1010541. doi: 10.1371/journal.pcbi.1010541

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1010541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9550052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36215228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Credibility%20assessment%20of%20patient%E2%80%90specific%20computational%20modeling%20using%20patient%E2%80%90specific%20cardiac%20modeling%20as%20an%20exemplar&amp;author=S%20Galappaththige&amp;author=RA%20Gray&amp;author=CM%20Costa&amp;author=S%20Niederer&amp;author=P%20Pathmanathan&amp;volume=18&amp;issue=10&amp;publication_year=2022&amp;pages=e1010541&amp;pmid=36215228&amp;doi=10.1371/journal.pcbi.1010541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0022">
<span class="label">22.</span><cite>
Liang J, Lu B, Zhao X, et al. Feasibility analyses of virtual models and 3D printing for surgical simulation of the double‐outlet right ventricle. Med Biol Eng Comput. 2022;60(10):3029‐3040. doi: 10.1007/s11517-022-02660-7

</cite> [<a href="https://doi.org/10.1007/s11517-022-02660-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36053430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med%20Biol%20Eng%20Comput&amp;title=Feasibility%20analyses%20of%20virtual%20models%20and%203D%20printing%20for%20surgical%20simulation%20of%20the%20double%E2%80%90outlet%20right%20ventricle&amp;author=J%20Liang&amp;author=B%20Lu&amp;author=X%20Zhao&amp;volume=60&amp;issue=10&amp;publication_year=2022&amp;pages=3029-3040&amp;pmid=36053430&amp;doi=10.1007/s11517-022-02660-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0023">
<span class="label">23.</span><cite>
de Lepper AGW, Buck CMA, van ’t Veer M, Huberts W, van de Vosse FN, Dekker LRC, From evidence‐based medicine to digital twin technology for predicting ventricular tachycardia in ischaemic cardiomyopathy. J R Soc Interface. 2022;19(194):20220317. doi: 10.1098/rsif.2022.0317

</cite> [<a href="https://doi.org/10.1098/rsif.2022.0317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9490337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36128708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20R%20Soc%20Interface&amp;title=From%20evidence%E2%80%90based%20medicine%20to%20digital%20twin%20technology%20for%20predicting%20ventricular%20tachycardia%20in%20ischaemic%20cardiomyopathy&amp;author=AGW%20de%20Lepper&amp;author=CMA%20Buck&amp;author=M%20van%20%E2%80%99t%20Veer&amp;author=W%20Huberts&amp;author=FN%20van%20de%20Vosse&amp;volume=19&amp;issue=194&amp;publication_year=2022&amp;pages=20220317&amp;pmid=36128708&amp;doi=10.1098/rsif.2022.0317&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0024">
<span class="label">24.</span><cite>
Straughan R, Kadry K, Parikh SA, Edelman ER, Nezami FR, Fully automated construction of three‐dimensional finite element simulations from optical coherence tomography. Comput Biol Med. 2023;165:107341. doi: 10.1016/j.compbiomed.2023.107341

</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2023.107341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10528179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37611423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Biol%20Med&amp;title=Fully%20automated%20construction%20of%20three%E2%80%90dimensional%20finite%20element%20simulations%20from%20optical%20coherence%20tomography&amp;author=R%20Straughan&amp;author=K%20Kadry&amp;author=SA%20Parikh&amp;author=ER%20Edelman&amp;author=FR%20Nezami&amp;volume=165&amp;publication_year=2023&amp;pages=107341&amp;pmid=37611423&amp;doi=10.1016/j.compbiomed.2023.107341&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0025">
<span class="label">25.</span><cite>
Rudnicka Z, Proniewska K, Perkins M, Pregowska A. Cardiac healthcare digital twins supported by artificial intelligence‐based algorithms and extended reality—a systematic review. Electronics. 2024;13(5):866. doi: 10.3390/electronics13050866
</cite> [<a href="https://doi.org/10.3390/electronics13050866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Electronics&amp;title=Cardiac%20healthcare%20digital%20twins%20supported%20by%20artificial%20intelligence%E2%80%90based%20algorithms%20and%20extended%20reality%E2%80%94a%20systematic%20review&amp;author=Z%20Rudnicka&amp;author=K%20Proniewska&amp;author=M%20Perkins&amp;author=A%20Pregowska&amp;volume=13&amp;issue=5&amp;publication_year=2024&amp;pages=866&amp;doi=10.3390/electronics13050866&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0026">
<span class="label">26.</span><cite>
Vanzella LM, Cotie LM, Flores‐Hukom M, Marzolini S, Konidis R, Ghisi GLM, Patients' perceptions of hybrid and virtual‐only care models during the cardiac rehabilitation patient journey: a qualitative study. J Cardiovasc Nurs. 2024:i. doi: 10.1097/jcn.0000000000001076
</cite> [<a href="https://doi.org/10.1097/jcn.0000000000001076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38206327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cardiovasc%20Nurs&amp;title=Patients'%20perceptions%20of%20hybrid%20and%20virtual%E2%80%90only%20care%20models%20during%20the%20cardiac%20rehabilitation%20patient%20journey:%20a%20qualitative%20study&amp;author=LM%20Vanzella&amp;author=LM%20Cotie&amp;author=M%20Flores%E2%80%90Hukom&amp;author=S%20Marzolini&amp;author=R%20Konidis&amp;publication_year=2024&amp;pmid=38206327&amp;doi=10.1097/jcn.0000000000001076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0027">
<span class="label">27.</span><cite>
Chakshu NK, Sazonov I, Nithiarasu P, Towards enabling a cardiovascular digital twin for human systemic circulation using inverse analysis. Biomech Model Mechanobiol. 2021;20(2):449‐465. doi: 10.1007/s10237-020-01393-6

</cite> [<a href="https://doi.org/10.1007/s10237-020-01393-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7979679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33064221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomech%20Model%20Mechanobiol&amp;title=Towards%20enabling%20a%20cardiovascular%20digital%20twin%20for%20human%20systemic%20circulation%20using%20inverse%20analysis&amp;author=NK%20Chakshu&amp;author=I%20Sazonov&amp;author=P%20Nithiarasu&amp;volume=20&amp;issue=2&amp;publication_year=2021&amp;pages=449-465&amp;pmid=33064221&amp;doi=10.1007/s10237-020-01393-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0028">
<span class="label">28.</span><cite>
Bodin ON, Chekaikin SV, Rakhmatullov FK, Rakhmatullov RF, Kramm MN, Bodin AY, Visualization of a Digital Twin of the Heart
. 2021 IEEE 22nd International Conference of Young Professionals in Electron Devices and Materials (EDM). 2021:419‐423. doi: 10.1109/edm52169.2021.9507615
</cite> [<a href="https://doi.org/10.1109/edm52169.2021.9507615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0029">
<span class="label">29.</span><cite>
Koopsen T, van Osta N, van Loon T, et al. Parameter subset reduction for imaging‐based digital twin generation of patients with left ventricular mechanical discoordination. BioMed Eng Online. 2024;23(1):46. doi: 10.1186/s12938-024-01232-0

</cite> [<a href="https://doi.org/10.1186/s12938-024-01232-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11089736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38741182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioMed%20Eng%20Online&amp;title=Parameter%20subset%20reduction%20for%20imaging%E2%80%90based%20digital%20twin%20generation%20of%20patients%20with%20left%20ventricular%20mechanical%20discoordination&amp;author=T%20Koopsen&amp;author=N%20van%20Osta&amp;author=T%20van%20Loon&amp;volume=23&amp;issue=1&amp;publication_year=2024&amp;pages=46&amp;pmid=38741182&amp;doi=10.1186/s12938-024-01232-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0030">
<span class="label">30.</span><cite>
Pires G, CdO Nieblas, Bravo‐Valenzuela NJ, Castro P, Werner H, Ebstein anomaly: 3D virtual and physical models from obstetrical ultrasound data. Echocardiography. 2024;41(4):e15806. doi: 10.1111/echo.15806

</cite> [<a href="https://doi.org/10.1111/echo.15806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38549419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Echocardiography&amp;title=Ebstein%20anomaly:%203D%20virtual%20and%20physical%20models%20from%20obstetrical%20ultrasound%20data&amp;author=G%20Pires&amp;author=Nieblas%20CdO&amp;author=NJ%20Bravo%E2%80%90Valenzuela&amp;author=P%20Castro&amp;author=H%20Werner&amp;volume=41&amp;issue=4&amp;publication_year=2024&amp;pages=e15806&amp;pmid=38549419&amp;doi=10.1111/echo.15806&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0031">
<span class="label">31.</span><cite>
Ložek M, Kovanda J, Kubuš P, et al. How to assess and treat right ventricular electromechanical dyssynchrony in post‐repair tetralogy of Fallot: insights from imaging, invasive studies, and computational modelling. Europace. 2024;26(2):euae024. doi: 10.1093/europace/euae024

</cite> [<a href="https://doi.org/10.1093/europace/euae024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10838147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38266248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Europace&amp;title=How%20to%20assess%20and%20treat%20right%20ventricular%20electromechanical%20dyssynchrony%20in%20post%E2%80%90repair%20tetralogy%20of%20Fallot:%20insights%20from%20imaging,%20invasive%20studies,%20and%20computational%20modelling&amp;author=M%20Lo%C5%BEek&amp;author=J%20Kovanda&amp;author=P%20Kubu%C5%A1&amp;volume=26&amp;issue=2&amp;publication_year=2024&amp;pages=euae024&amp;pmid=38266248&amp;doi=10.1093/europace/euae024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0032">
<span class="label">32.</span><cite>
Glover GH, Overview of functional magnetic resonance imaging. Neurosurg Clin N Am. 2011;22(2):133‐139. doi: 10.1016/j.nec.2011.02.001

</cite> [<a href="https://doi.org/10.1016/j.nec.2011.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3073717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21435566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurosurg%20Clin%20N%20Am&amp;title=Overview%20of%20functional%20magnetic%20resonance%20imaging&amp;author=GH%20Glover&amp;volume=22&amp;issue=2&amp;publication_year=2011&amp;pages=133-139&amp;pmid=21435566&amp;doi=10.1016/j.nec.2011.02.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0033">
<span class="label">33.</span><cite>
Cen S, Gebregziabher M, Moazami S, Azevedo CJ, Pelletier D, Toward precision medicine using a “digital twin” approach: modeling the onset of disease‐specific brain atrophy in individuals with multiple sclerosis. Sci Rep. 2023;13(1):16279. doi: 10.1038/s41598-023-43618-5

</cite> [<a href="https://doi.org/10.1038/s41598-023-43618-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10539386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37770560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Toward%20precision%20medicine%20using%20a%20%E2%80%9Cdigital%20twin%E2%80%9D%20approach:%20modeling%20the%20onset%20of%20disease%E2%80%90specific%20brain%20atrophy%20in%20individuals%20with%20multiple%20sclerosis&amp;author=S%20Cen&amp;author=M%20Gebregziabher&amp;author=S%20Moazami&amp;author=CJ%20Azevedo&amp;author=D%20Pelletier&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=16279&amp;pmid=37770560&amp;doi=10.1038/s41598-023-43618-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0034">
<span class="label">34.</span><cite>
Hu M, Zhong Y, Xie S, Lv H, Lv Z, Fuzzy system based medical image processing for brain disease prediction. Front Neurosci
2021;15714318. doi: 10.3389/fnins.2021.714318
</cite> [<a href="https://doi.org/10.3389/fnins.2021.714318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8361453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34393718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Neurosci&amp;title=Fuzzy%20system%20based%20medical%20image%20processing%20for%20brain%20disease%20prediction&amp;author=M%20Hu&amp;author=Y%20Zhong&amp;author=S%20Xie&amp;author=H%20Lv&amp;author=Z%20Lv&amp;publication_year=2021&amp;pages=15714318&amp;pmid=34393718&amp;doi=10.3389/fnins.2021.714318&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0035">
<span class="label">35.</span><cite>
Wan Z, Dong Y, Yu Z, Lv H, Lv Z, Semi‐supervised support vector machine for digital twins based brain image Fusion. Front Neurosci
2021;15705323. doi: 10.3389/fnins.2021.705323
</cite> [<a href="https://doi.org/10.3389/fnins.2021.705323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8298822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34305523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Neurosci&amp;title=Semi%E2%80%90supervised%20support%20vector%20machine%20for%20digital%20twins%20based%20brain%20image%20Fusion&amp;author=Z%20Wan&amp;author=Y%20Dong&amp;author=Z%20Yu&amp;author=H%20Lv&amp;author=Z%20Lv&amp;publication_year=2021&amp;pages=15705323&amp;pmid=34305523&amp;doi=10.3389/fnins.2021.705323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0036">
<span class="label">36.</span><cite>
Sarris AL, Sidiropoulos E, Paraskevopoulos E, Bamidis P, Towards a digital twin in human brain: brain tumor detection using K‐means. Stud Health Technol Inform. 2023;302:1052‐1056. doi: 10.3233/SHTI230345

</cite> [<a href="https://doi.org/10.3233/SHTI230345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37203579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stud%20Health%20Technol%20Inform&amp;title=Towards%20a%20digital%20twin%20in%20human%20brain:%20brain%20tumor%20detection%20using%20K%E2%80%90means&amp;author=AL%20Sarris&amp;author=E%20Sidiropoulos&amp;author=E%20Paraskevopoulos&amp;author=P%20Bamidis&amp;volume=302&amp;publication_year=2023&amp;pages=1052-1056&amp;pmid=37203579&amp;doi=10.3233/SHTI230345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0037">
<span class="label">37.</span><cite>
Wang T, Dremel J, Richter S, et al. Resolution‐enhanced multi‐core fiber imaging learned on a digital twin for cancer diagnosis. Neurophotonics
2024;11(S1):S11505. doi: 10.1117/1.NPh.11.S1.S11505

</cite> [<a href="https://doi.org/10.1117/1.NPh.11.S1.S11505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10828892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38298866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurophotonics&amp;title=Resolution%E2%80%90enhanced%20multi%E2%80%90core%20fiber%20imaging%20learned%20on%20a%20digital%20twin%20for%20cancer%20diagnosis&amp;author=T%20Wang&amp;author=J%20Dremel&amp;author=S%20Richter&amp;volume=11&amp;issue=S1&amp;publication_year=2024&amp;pages=S11505&amp;pmid=38298866&amp;doi=10.1117/1.NPh.11.S1.S11505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0038">
<span class="label">38.</span><cite>
Guillevin R, Naudin M, Fayolle P, et al. Diagnostic and therapeutic issues in glioma using imaging data: the challenge of numerical twinning. J Clin Med
2023;12(24)7706. doi: 10.3390/jcm12247706

</cite> [<a href="https://doi.org/10.3390/jcm12247706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10744312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38137775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med&amp;title=Diagnostic%20and%20therapeutic%20issues%20in%20glioma%20using%20imaging%20data:%20the%20challenge%20of%20numerical%20twinning&amp;author=R%20Guillevin&amp;author=M%20Naudin&amp;author=P%20Fayolle&amp;volume=12&amp;issue=24&amp;publication_year=2023&amp;pages=7706&amp;pmid=38137775&amp;doi=10.3390/jcm12247706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0039">
<span class="label">39.</span><cite>
Chaudhuri A, Pash G, Hormuth DA, 2nd, et al. Predictive digital twin for optimizing patient‐specific radiotherapy regimens under uncertainty in high‐grade gliomas. Front Artif Intell. 2023;6:1222612. doi: 10.3389/frai.2023.1222612

</cite> [<a href="https://doi.org/10.3389/frai.2023.1222612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10598726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37886348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Artif%20Intell&amp;title=Predictive%20digital%20twin%20for%20optimizing%20patient%E2%80%90specific%20radiotherapy%20regimens%20under%20uncertainty%20in%20high%E2%80%90grade%20gliomas&amp;author=A%20Chaudhuri&amp;author=G%20Pash&amp;author=DA%20Hormuth&amp;volume=6&amp;publication_year=2023&amp;pages=1222612&amp;pmid=37886348&amp;doi=10.3389/frai.2023.1222612&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0040">
<span class="label">40.</span><cite>
Wang J, Qiao L, Lv H, Lv Z, Deep transfer learning‐based multi‐modal digital twins for enhancement and diagnostic analysis of brain MRI image. EEE/ACM Trans Comput Biol Bioinform. 2023;20(4):2407‐2419. doi: 10.1109/TCBB.2022.3168189
</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3168189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35439137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EEE/ACM%20Trans%20Comput%20Biol%20Bioinform&amp;title=Deep%20transfer%20learning%E2%80%90based%20multi%E2%80%90modal%20digital%20twins%20for%20enhancement%20and%20diagnostic%20analysis%20of%20brain%20MRI%20image&amp;author=J%20Wang&amp;author=L%20Qiao&amp;author=H%20Lv&amp;author=Z%20Lv&amp;volume=20&amp;issue=4&amp;publication_year=2023&amp;pages=2407-2419&amp;pmid=35439137&amp;doi=10.1109/TCBB.2022.3168189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0041">
<span class="label">41.</span><cite>
Lu W, Zeng L, Du X, et al. Digital twin brain: a simulation and assimilation platform for whole human brain. arXiv: arXiv:2308.01241. Preprint posted online, 2023.</cite>
</li>
<li id="acm270226-bib-0042">
<span class="label">42.</span><cite>
Lu W, Du X, Wang J, et al; DTB Consortium. The human digital twin brain in the resting state and in action. Nat Comput Sci. 2024;4(11):840‐857. doi: 10.1038/s43588-024-00731-3
39455825
</cite> [<a href="https://doi.org/10.1038/s43588-024-00731-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Comput%20Sci&amp;title=DTB%20Consortium.%20The%20human%20digital%20twin%20brain%20in%20the%20resting%20state%20and%20in%20action&amp;author=W%20Lu&amp;author=X%20Du&amp;author=J%20Wang&amp;volume=4&amp;issue=11&amp;publication_year=2024&amp;pages=840-857&amp;doi=10.1038/s43588-024-00731-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0043">
<span class="label">43.</span><cite>
Li Y, Li X, Shen S, et al. DTBVis: an interactive visual comparison system for digital twin brain and human brain. Visual Informatics. 2023;7(2):41‐53. doi: 10.1016/j.visinf.2023.02.002
</cite> [<a href="https://doi.org/10.1016/j.visinf.2023.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Visual%20Informatics&amp;title=DTBVis:%20an%20interactive%20visual%20comparison%20system%20for%20digital%20twin%20brain%20and%20human%20brain&amp;author=Y%20Li&amp;author=X%20Li&amp;author=S%20Shen&amp;volume=7&amp;issue=2&amp;publication_year=2023&amp;pages=41-53&amp;doi=10.1016/j.visinf.2023.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0044">
<span class="label">44.</span><cite>
Lachinov D, Chakravarty A, Grechenig C, Schmidt‐Erfurth U, Bogunovic H, Learning spatio‐temporal model of disease progression with NeuralODEs from longitudinal volumetric data. IEEE Trans Med Imaging. 2023;doi: 10.1109/tmi.2023.3330576
</cite> [<a href="https://doi.org/10.1109/tmi.2023.3330576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37934647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Med%20Imaging&amp;title=Learning%20spatio%E2%80%90temporal%20model%20of%20disease%20progression%20with%20NeuralODEs%20from%20longitudinal%20volumetric%20data&amp;author=D%20Lachinov&amp;author=A%20Chakravarty&amp;author=C%20Grechenig&amp;author=U%20Schmidt%E2%80%90Erfurth&amp;author=H%20Bogunovic&amp;publication_year=2023&amp;pmid=37934647&amp;doi=10.1109/tmi.2023.3330576&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0045">
<span class="label">45.</span><cite>
Knopman DS, Amieva H, Petersen RC, et al. Alzheimer disease. Nat Rev Dis Primers. 2021;7(1):33. doi: 10.1038/s41572-021-00269-y

</cite> [<a href="https://doi.org/10.1038/s41572-021-00269-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8574196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33986301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Dis%20Primers&amp;title=Alzheimer%20disease&amp;author=DS%20Knopman&amp;author=H%20Amieva&amp;author=RC%20Petersen&amp;volume=7&amp;issue=1&amp;publication_year=2021&amp;pages=33&amp;pmid=33986301&amp;doi=10.1038/s41572-021-00269-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0046">
<span class="label">46.</span><cite>
Engelke K, Museyko O, Wang L, Laredo J‐D, Quantitative analysis of skeletal muscle by computed tomography imaging—State of the art. J Orthop Transl. 2018;15:91‐103. doi: 10.1016/j.jot.2018.10.004
</cite> [<a href="https://doi.org/10.1016/j.jot.2018.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6260391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30533385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Orthop%20Transl&amp;title=Quantitative%20analysis%20of%20skeletal%20muscle%20by%20computed%20tomography%20imaging%E2%80%94State%20of%20the%20art&amp;author=K%20Engelke&amp;author=O%20Museyko&amp;author=L%20Wang&amp;author=J%E2%80%90D%20Laredo&amp;volume=15&amp;publication_year=2018&amp;pages=91-103&amp;pmid=30533385&amp;doi=10.1016/j.jot.2018.10.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0047">
<span class="label">47.</span><cite>
Ahn S, Kim J, Baek S, Kim C, Jang H, Lee S, Toward digital twin development for implant placement planning using a parametric reduced‐order model. Bioengineering. 2024;11(1):84. doi: 10.3390/bioengineering11010084

</cite> [<a href="https://doi.org/10.3390/bioengineering11010084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10813277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38247961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioengineering&amp;title=Toward%20digital%20twin%20development%20for%20implant%20placement%20planning%20using%20a%20parametric%20reduced%E2%80%90order%20model&amp;author=S%20Ahn&amp;author=J%20Kim&amp;author=S%20Baek&amp;author=C%20Kim&amp;author=H%20Jang&amp;volume=11&amp;issue=1&amp;publication_year=2024&amp;pages=84&amp;pmid=38247961&amp;doi=10.3390/bioengineering11010084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0048">
<span class="label">48.</span><cite>
Demir O, Uslan I, Buyuk M, Salamci MU, Development and validation of a digital twin of the human lower jaw under impact loading by using non‐linear finite element analyses. J Mech Behav Biomed Mater. 2023;148:106207. doi: 10.1016/j.jmbbm.2023.106207

</cite> [<a href="https://doi.org/10.1016/j.jmbbm.2023.106207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37922761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mech%20Behav%20Biomed%20Mater&amp;title=Development%20and%20validation%20of%20a%20digital%20twin%20of%20the%20human%20lower%20jaw%20under%20impact%20loading%20by%20using%20non%E2%80%90linear%20finite%20element%20analyses&amp;author=O%20Demir&amp;author=I%20Uslan&amp;author=M%20Buyuk&amp;author=MU%20Salamci&amp;volume=148&amp;publication_year=2023&amp;pages=106207&amp;pmid=37922761&amp;doi=10.1016/j.jmbbm.2023.106207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0049">
<span class="label">49.</span><cite>
Kim SH, Park YS, Kim MK, Kim S, Lee SP, Methods for quantitative measurement of tooth wear using the area and volume of virtual model cusps. J Periodontal Implant Sci. 2018;48(2):124‐134. doi: 10.5051/jpis.2018.48.2.124

</cite> [<a href="https://doi.org/10.5051/jpis.2018.48.2.124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5944223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29770241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Periodontal%20Implant%20Sci&amp;title=Methods%20for%20quantitative%20measurement%20of%20tooth%20wear%20using%20the%20area%20and%20volume%20of%20virtual%20model%20cusps&amp;author=SH%20Kim&amp;author=YS%20Park&amp;author=MK%20Kim&amp;author=S%20Kim&amp;author=SP%20Lee&amp;volume=48&amp;issue=2&amp;publication_year=2018&amp;pages=124-134&amp;pmid=29770241&amp;doi=10.5051/jpis.2018.48.2.124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0050">
<span class="label">50.</span><cite>
Vila M, Xiberta P, Ruiz M, et al. BeST‐Graft viewer, a new system to improve the bone allograft–recipient matching process. Comput Biol Med. 2024;176:108553. doi: 10.1016/j.compbiomed.2024.108553

</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2024.108553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38723397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Biol%20Med&amp;title=BeST%E2%80%90Graft%20viewer,%20a%20new%20system%20to%20improve%20the%20bone%20allograft%E2%80%93recipient%20matching%20process&amp;author=M%20Vila&amp;author=P%20Xiberta&amp;author=M%20Ruiz&amp;volume=176&amp;publication_year=2024&amp;pages=108553&amp;pmid=38723397&amp;doi=10.1016/j.compbiomed.2024.108553&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0051">
<span class="label">51.</span><cite>
Ioussoufovitch S, Diop M. Time‐domain diffuse optical imaging technique for monitoring rheumatoid arthritis disease activity: experimental validation in tissue‐mimicking finger phantoms. Phys Med Biol. 2024;69(12):125021. doi: 10.1088/1361-6560/ad53a0
</cite> [<a href="https://doi.org/10.1088/1361-6560/ad53a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38830365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Med%20Biol&amp;title=Time%E2%80%90domain%20diffuse%20optical%20imaging%20technique%20for%20monitoring%20rheumatoid%20arthritis%20disease%20activity:%20experimental%20validation%20in%20tissue%E2%80%90mimicking%20finger%20phantoms&amp;author=S%20Ioussoufovitch&amp;author=M%20Diop&amp;volume=69&amp;issue=12&amp;publication_year=2024&amp;pages=125021&amp;pmid=38830365&amp;doi=10.1088/1361-6560/ad53a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0052">
<span class="label">52.</span><cite>
Hernigou P, Olejnik R, Safar A, Martinov S, Hernigou J, Ferre B, Digital twins, artificial intelligence, and machine learning technology to identify a real personalized motion axis of the tibiotalar joint for robotics in total ankle arthroplasty. Int Orthop. 2021;45:2209‐2217. doi: 10.1007/s00264-021-05175-2

</cite> [<a href="https://doi.org/10.1007/s00264-021-05175-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34351462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20Orthop&amp;title=Digital%20twins,%20artificial%20intelligence,%20and%20machine%20learning%20technology%20to%20identify%20a%20real%20personalized%20motion%20axis%20of%20the%20tibiotalar%20joint%20for%20robotics%20in%20total%20ankle%20arthroplasty&amp;author=P%20Hernigou&amp;author=R%20Olejnik&amp;author=A%20Safar&amp;author=S%20Martinov&amp;author=J%20Hernigou&amp;volume=45&amp;publication_year=2021&amp;pages=2209-2217&amp;pmid=34351462&amp;doi=10.1007/s00264-021-05175-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0053">
<span class="label">53.</span><cite>
Kim Y‐H, Park I, Cho SB, et al. Three‐dimensional virtual reconstructions of shoulder movements using computed tomography images: model development. Interact J Med Res. 2023;12(1):e48381. doi: 10.2196/48381

</cite> [<a href="https://doi.org/10.2196/48381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10587804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37796554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Interact%20J%20Med%20Res&amp;title=Three%E2%80%90dimensional%20virtual%20reconstructions%20of%20shoulder%20movements%20using%20computed%20tomography%20images:%20model%20development&amp;author=Y%E2%80%90H%20Kim&amp;author=I%20Park&amp;author=SB%20Cho&amp;volume=12&amp;issue=1&amp;publication_year=2023&amp;pages=e48381&amp;pmid=37796554&amp;doi=10.2196/48381&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0054">
<span class="label">54.</span><cite>
Verweij LP, Dobbe JG, Kerkhoffs GM, et al. Minimal but potentially clinically relevant anteroinferior position of the humeral head following traumatic anterior shoulder dislocations: a 3D‐CT analysis. J Orthop Res. 2024;42(8):1641‐1652. doi: 10.1002/jor.25831

</cite> [<a href="https://doi.org/10.1002/jor.25831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38465730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Orthop%20Res&amp;title=Minimal%20but%20potentially%20clinically%20relevant%20anteroinferior%20position%20of%20the%20humeral%20head%20following%20traumatic%20anterior%20shoulder%20dislocations:%20a%203D%E2%80%90CT%20analysis&amp;author=LP%20Verweij&amp;author=JG%20Dobbe&amp;author=GM%20Kerkhoffs&amp;volume=42&amp;issue=8&amp;publication_year=2024&amp;pages=1641-1652&amp;pmid=38465730&amp;doi=10.1002/jor.25831&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0055">
<span class="label">55.</span><cite>
Ha M, Lee J, Cho Y, et al. A hybrid upper‐arm‐geared exoskeleton with anatomical digital twin for tangible metaverse feedback and communication. Artic Adv Mater Technol. 2024;9(2):2301404. doi: 10.1002/admt.202301404
</cite> [<a href="https://doi.org/10.1002/admt.202301404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artic%20Adv%20Mater%20Technol&amp;title=A%20hybrid%20upper%E2%80%90arm%E2%80%90geared%20exoskeleton%20with%20anatomical%20digital%20twin%20for%20tangible%20metaverse%20feedback%20and%20communication&amp;author=M%20Ha&amp;author=J%20Lee&amp;author=Y%20Cho&amp;volume=9&amp;issue=2&amp;publication_year=2024&amp;pages=2301404&amp;doi=10.1002/admt.202301404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0056">
<span class="label">56.</span><cite>
Moztarzadeh O, Jamshidi MB, Sargolzaei S, et al. Metaverse and medical diagnosis: a blockchain‐based digital twinning approach based on MobileNetV2 algorithm for cervical vertebral maturation. Diagnostics. 2023;13(8):1485.doi: 10.3390/diagnostics13081485

</cite> [<a href="https://doi.org/10.3390/diagnostics13081485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10137959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37189587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diagnostics&amp;title=Metaverse%20and%20medical%20diagnosis:%20a%20blockchain%E2%80%90based%20digital%20twinning%20approach%20based%20on%20MobileNetV2%20algorithm%20for%20cervical%20vertebral%20maturation&amp;author=O%20Moztarzadeh&amp;author=MB%20Jamshidi&amp;author=S%20Sargolzaei&amp;volume=13&amp;issue=8&amp;publication_year=2023&amp;pages=1485&amp;pmid=37189587&amp;doi=10.3390/diagnostics13081485&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0057">
<span class="label">57.</span><cite>
He X, Qiu Y, Lai X, et al. Towards a shape‐performance integrated digital twin for lumbar spine analysis. Digital Twin. 2021;1:8. doi: 10.12688/digitaltwin.17478.1
</cite> [<a href="https://doi.org/10.12688/digitaltwin.17478.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Digital%20Twin&amp;title=Towards%20a%20shape%E2%80%90performance%20integrated%20digital%20twin%20for%20lumbar%20spine%20analysis&amp;author=X%20He&amp;author=Y%20Qiu&amp;author=X%20Lai&amp;volume=1&amp;publication_year=2021&amp;pages=8&amp;doi=10.12688/digitaltwin.17478.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0058">
<span class="label">58.</span><cite>
Ahmadian H, Mageswaran P, Walter BA, et al. A digital twin for simulating the vertebroplasty procedure and its impact on mechanical stability of vertebra in cancer patients. Int J Numer Method Biomed Eng. 2022;38(6):e3600. doi: 10.1002/cnm.3600

</cite> [<a href="https://doi.org/10.1002/cnm.3600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9287026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35347880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Numer%20Method%20Biomed%20Eng&amp;title=A%20digital%20twin%20for%20simulating%20the%20vertebroplasty%20procedure%20and%20its%20impact%20on%20mechanical%20stability%20of%20vertebra%20in%20cancer%20patients&amp;author=H%20Ahmadian&amp;author=P%20Mageswaran&amp;author=BA%20Walter&amp;volume=38&amp;issue=6&amp;publication_year=2022&amp;pages=e3600&amp;pmid=35347880&amp;doi=10.1002/cnm.3600&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0059">
<span class="label">59.</span><cite>
Ahmadian H, Mageswaran P, Walter BA, et al. Toward an artificial intelligence‐assisted framework for reconstructing the digital twin of vertebra and predicting its fracture response. Int J Numer Method Biomedical Eng. 2022;38(6):e3601. doi: 10.1002/cnm.3601
</cite> [<a href="https://doi.org/10.1002/cnm.3601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9285948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35403831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Numer%20Method%20Biomedical%20Eng&amp;title=Toward%20an%20artificial%20intelligence%E2%80%90assisted%20framework%20for%20reconstructing%20the%20digital%20twin%20of%20vertebra%20and%20predicting%20its%20fracture%20response&amp;author=H%20Ahmadian&amp;author=P%20Mageswaran&amp;author=BA%20Walter&amp;volume=38&amp;issue=6&amp;publication_year=2022&amp;pages=e3601&amp;pmid=35403831&amp;doi=10.1002/cnm.3601&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0060">
<span class="label">60.</span><cite>
Swaity A, Elgarba BM, Morgan N, et al. Deep learning driven segmentation of maxillary impacted canine on cone beam computed tomography images. Sci Rep. 2024;14(1):369. doi: 10.1038/s41598-023-49613-0

</cite> [<a href="https://doi.org/10.1038/s41598-023-49613-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10764895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38172136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Deep%20learning%20driven%20segmentation%20of%20maxillary%20impacted%20canine%20on%20cone%20beam%20computed%20tomography%20images&amp;author=A%20Swaity&amp;author=BM%20Elgarba&amp;author=N%20Morgan&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=369&amp;pmid=38172136&amp;doi=10.1038/s41598-023-49613-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0061">
<span class="label">61.</span><cite>
Zilberman O, Huggare JA, Parikakis KA, Evaluation of the validity of tooth size and arch width measurements using conventional and three‐dimensional virtual orthodontic models. Angle Orthod. 2003;73(3):301‐6. doi:

</cite> [<a href="https://doi.org/10.1043/0003-3219(2003)073&amp;#x0003c;0301:Eotvot&amp;#x0003e;2.0.Co;2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12828439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angle%20Orthod&amp;title=Evaluation%20of%20the%20validity%20of%20tooth%20size%20and%20arch%20width%20measurements%20using%20conventional%20and%20three%E2%80%90dimensional%20virtual%20orthodontic%20models&amp;author=O%20Zilberman&amp;author=JA%20Huggare&amp;author=KA%20Parikakis&amp;volume=73&amp;issue=3&amp;publication_year=2003&amp;pages=301-6&amp;pmid=12828439&amp;doi=10.1043/0003-3219(2003)073&amp;#x0003c;0301:Eotvot&amp;#x0003e;2.0.Co;2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0062">
<span class="label">62.</span><cite>
Nazir MA, Prevalence of periodontal disease, its association with systemic diseases and prevention. Int J Health Sci. 2017;11(2):72. doi: 10.12816/0048722
</cite> [<a href="https://doi.org/10.12816/0048722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5426403/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28539867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Health%20Sci&amp;title=Prevalence%20of%20periodontal%20disease,%20its%20association%20with%20systemic%20diseases%20and%20prevention&amp;author=MA%20Nazir&amp;volume=11&amp;issue=2&amp;publication_year=2017&amp;pages=72&amp;pmid=28539867&amp;doi=10.12816/0048722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0063">
<span class="label">63.</span><cite>
Fallon EA, Prevalence of diagnosed arthritis—United States, 2019–2021. MMWR Morb Mortal Wkly Rep. 2023;72. doi: 10.15585/mmwr.mm7241a1
</cite> [<a href="https://doi.org/10.15585/mmwr.mm7241a1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10578950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37824422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MMWR%20Morb%20Mortal%20Wkly%20Rep&amp;title=Prevalence%20of%20diagnosed%20arthritis%E2%80%94United%20States,%202019%E2%80%932021&amp;author=EA%20Fallon&amp;publication_year=2023&amp;pages=72&amp;pmid=37824422&amp;doi=10.15585/mmwr.mm7241a1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0064">
<span class="label">64.</span><cite>
Teraguchi M, Yoshimura N, Hashizume H, et al. Prevalence and distribution of intervertebral disc degeneration over the entire spine in a population‐based cohort: the Wakayama spine study. Osteoarthritis Cartilage. 2014;22(1):104‐110. doi: 10.1016/j.joca.2013.10.019

</cite> [<a href="https://doi.org/10.1016/j.joca.2013.10.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24239943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Osteoarthritis%20Cartilage&amp;title=Prevalence%20and%20distribution%20of%20intervertebral%20disc%20degeneration%20over%20the%20entire%20spine%20in%20a%20population%E2%80%90based%20cohort:%20the%20Wakayama%20spine%20study&amp;author=M%20Teraguchi&amp;author=N%20Yoshimura&amp;author=H%20Hashizume&amp;volume=22&amp;issue=1&amp;publication_year=2014&amp;pages=104-110&amp;pmid=24239943&amp;doi=10.1016/j.joca.2013.10.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0065">
<span class="label">65.</span><cite>
Lehmann BD, Bauer JA, Chen X, et al. Identification of human triple‐negative breast cancer subtypes and preclinical models for selection of targeted therapies. J Clin Invest. 2011;121(7):2750‐2767. doi: 10.1172/JCI45014

</cite> [<a href="https://doi.org/10.1172/JCI45014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3127435/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21633166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&amp;title=Identification%20of%20human%20triple%E2%80%90negative%20breast%20cancer%20subtypes%20and%20preclinical%20models%20for%20selection%20of%20targeted%20therapies&amp;author=BD%20Lehmann&amp;author=JA%20Bauer&amp;author=X%20Chen&amp;volume=121&amp;issue=7&amp;publication_year=2011&amp;pages=2750-2767&amp;pmid=21633166&amp;doi=10.1172/JCI45014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0066">
<span class="label">66.</span><cite>
Jiang WW, Li AH, Zheng YP, A semi‐automated 3‐D annotation method for breast ultrasound imaging: system development and feasibility study on phantoms. Ultrasound Med Biol Feb
2014;40(2):434‐46. doi: 10.1016/j.ultrasmedbio.2013.10.001
</cite> [<a href="https://doi.org/10.1016/j.ultrasmedbio.2013.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24315389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ultrasound%20Med%20Biol%20Feb&amp;title=A%20semi%E2%80%90automated%203%E2%80%90D%20annotation%20method%20for%20breast%20ultrasound%20imaging:%20system%20development%20and%20feasibility%20study%20on%20phantoms&amp;author=WW%20Jiang&amp;author=AH%20Li&amp;author=YP%20Zheng&amp;volume=40&amp;issue=2&amp;publication_year=2014&amp;pages=434-46&amp;pmid=24315389&amp;doi=10.1016/j.ultrasmedbio.2013.10.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0067">
<span class="label">67.</span><cite>
Sayed AM, Naser MA, Wahba AA, Eldosoky MAA, Breast tumor diagnosis using finite‐element modeling based on clinical in vivo elastographic data. J Ultrasound Med. 2020;39(12):2351‐2363. doi: 10.1002/jum.15344

</cite> [<a href="https://doi.org/10.1002/jum.15344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32472949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Ultrasound%20Med&amp;title=Breast%20tumor%20diagnosis%20using%20finite%E2%80%90element%20modeling%20based%20on%20clinical%20in%20vivo%20elastographic%20data&amp;author=AM%20Sayed&amp;author=MA%20Naser&amp;author=AA%20Wahba&amp;author=MAA%20Eldosoky&amp;volume=39&amp;issue=12&amp;publication_year=2020&amp;pages=2351-2363&amp;pmid=32472949&amp;doi=10.1002/jum.15344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0068">
<span class="label">68.</span><cite>
Wu C, Jarrett AM, Zhou Z, et al. MRI‐based digital models forecast patient‐specific treatment responses to neoadjuvant chemotherapy in triple‐negative breast cancer. Cancer Res. 2022;82(18):3394‐3404. doi: 10.1158/0008-5472.CAN-22-1329

</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-22-1329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9481712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35914239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=MRI%E2%80%90based%20digital%20models%20forecast%20patient%E2%80%90specific%20treatment%20responses%20to%20neoadjuvant%20chemotherapy%20in%20triple%E2%80%90negative%20breast%20cancer&amp;author=C%20Wu&amp;author=AM%20Jarrett&amp;author=Z%20Zhou&amp;volume=82&amp;issue=18&amp;publication_year=2022&amp;pages=3394-3404&amp;pmid=35914239&amp;doi=10.1158/0008-5472.CAN-22-1329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0069">
<span class="label">69.</span><cite>
Moore JH, Li X, Chang JH, et al. SynTwin: a graph‐based approach for predicting clinical outcomes using digital twins derived from synthetic patients. Pac Symp Biocomput. 2024;29:96‐107. doi: 10.1142/9789811286421_0008

</cite> [<a href="https://doi.org/10.1142/9789811286421_0008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10827004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38160272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pac%20Symp%20Biocomput&amp;title=SynTwin:%20a%20graph%E2%80%90based%20approach%20for%20predicting%20clinical%20outcomes%20using%20digital%20twins%20derived%20from%20synthetic%20patients&amp;author=JH%20Moore&amp;author=X%20Li&amp;author=JH%20Chang&amp;volume=29&amp;publication_year=2024&amp;pages=96-107&amp;pmid=38160272&amp;doi=10.1142/9789811286421_0008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0070">
<span class="label">70.</span><cite>
Gamage TPB, Elsayed A, Lin C, et al. Vision for the 12 LABOURS Digital Twin Platform
. 2023 45th Annual International Conference of the IEEE Engineering in Medicine &amp; Biology Society (EMBC). 2023:1‐4. doi: 10.1109/embc40787.2023.10341138
</cite> [<a href="https://doi.org/10.1109/embc40787.2023.10341138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38083471/" class="usa-link">PubMed</a>]</li>
<li id="acm270226-bib-0071">
<span class="label">71.</span><cite>
Novara G, Ficarra V, Rosen RC, et al. Systematic review and meta‐analysis of perioperative outcomes and complications after robot‐assisted radical prostatectomy. European Urology. 2012;62(3):431‐452. doi: 10.1016/j.eururo.2012.05.044

</cite> [<a href="https://doi.org/10.1016/j.eururo.2012.05.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22749853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Urology&amp;title=Systematic%20review%20and%20meta%E2%80%90analysis%20of%20perioperative%20outcomes%20and%20complications%20after%20robot%E2%80%90assisted%20radical%20prostatectomy&amp;author=G%20Novara&amp;author=V%20Ficarra&amp;author=RC%20Rosen&amp;volume=62&amp;issue=3&amp;publication_year=2012&amp;pages=431-452&amp;pmid=22749853&amp;doi=10.1016/j.eururo.2012.05.044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0072">
<span class="label">72.</span><cite>
Amparore D, Sica M, Verri P, et al. Computer vision and machine‐learning techniques for automatic 3D virtual images overlapping during augmented reality guided robotic partial nephrectomy. TECHNOL CANCER RES TREAT. 2024;23:15330338241229368. doi: 10.1177/15330338241229368

</cite> [<a href="https://doi.org/10.1177/15330338241229368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10878218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38374643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=TECHNOL%20CANCER%20RES%20TREAT&amp;title=Computer%20vision%20and%20machine%E2%80%90learning%20techniques%20for%20automatic%203D%20virtual%20images%20overlapping%20during%20augmented%20reality%20guided%20robotic%20partial%20nephrectomy&amp;author=D%20Amparore&amp;author=M%20Sica&amp;author=P%20Verri&amp;volume=23&amp;publication_year=2024&amp;pages=15330338241229368&amp;pmid=38374643&amp;doi=10.1177/15330338241229368&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0073">
<span class="label">73.</span><cite>
Checcucci E, Amparore D, Volpi G, et al. Metaverse surgical planning with three‐dimensional virtual models for minimally invasive partial nephrectomy. Eur Urol. 2024;85(4):320‐325. doi: 10.1016/j.eururo.2023.07.015

</cite> [<a href="https://doi.org/10.1016/j.eururo.2023.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37673751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20Urol&amp;title=Metaverse%20surgical%20planning%20with%20three%E2%80%90dimensional%20virtual%20models%20for%20minimally%20invasive%20partial%20nephrectomy&amp;author=E%20Checcucci&amp;author=D%20Amparore&amp;author=G%20Volpi&amp;volume=85&amp;issue=4&amp;publication_year=2024&amp;pages=320-325&amp;pmid=37673751&amp;doi=10.1016/j.eururo.2023.07.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0074">
<span class="label">74.</span><cite>
Grosso AA, Di Maida F, Lambertini L, et al. Three‐dimensional virtual model for robot‐assisted partial nephrectomy: a propensity‐score matching analysis with a contemporary control group. World J Urol. 2024;42(1):338. doi: 10.1007/s00345-024-05043-9

</cite> [<a href="https://doi.org/10.1007/s00345-024-05043-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11106151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38767673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J%20Urol&amp;title=Three%E2%80%90dimensional%20virtual%20model%20for%20robot%E2%80%90assisted%20partial%20nephrectomy:%20a%20propensity%E2%80%90score%20matching%20analysis%20with%20a%20contemporary%20control%20group&amp;author=AA%20Grosso&amp;author=F%20Di%20Maida&amp;author=L%20Lambertini&amp;volume=42&amp;issue=1&amp;publication_year=2024&amp;pages=338&amp;pmid=38767673&amp;doi=10.1007/s00345-024-05043-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0075">
<span class="label">75.</span><cite>
Sasikaladevi N, Revathi A, Digital twin of renal system with CT‐radiography for the early diagnosis of chronic kidney diseases. Biomed Signal Process Control. 2024;88. doi: 10.1016/j.bspc.2023.105632
</cite> [<a href="https://doi.org/10.1016/j.bspc.2023.105632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomed%20Signal%20Process%20Control&amp;title=Digital%20twin%20of%20renal%20system%20with%20CT%E2%80%90radiography%20for%20the%20early%20diagnosis%20of%20chronic%20kidney%20diseases&amp;author=N%20Sasikaladevi&amp;author=A%20Revathi&amp;publication_year=2024&amp;pages=88&amp;doi=10.1016/j.bspc.2023.105632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0076">
<span class="label">76.</span><cite>
Sarhan K, Khan N, Prezzi D, et al. Reduction of surgical complications via 3D models during robotic assisted radical prostatectomy: review of current evidence and meta‐analysis. J Robot Surg. 2024;18(1):304. doi: 10.1007/s11701-024-02041-2

</cite> [<a href="https://doi.org/10.1007/s11701-024-02041-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11303509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39105931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Robot%20Surg&amp;title=Reduction%20of%20surgical%20complications%20via%203D%20models%20during%20robotic%20assisted%20radical%20prostatectomy:%20review%20of%20current%20evidence%20and%20meta%E2%80%90analysis&amp;author=K%20Sarhan&amp;author=N%20Khan&amp;author=D%20Prezzi&amp;volume=18&amp;issue=1&amp;publication_year=2024&amp;pages=304&amp;pmid=39105931&amp;doi=10.1007/s11701-024-02041-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0077">
<span class="label">77.</span><cite>
Metropolis N, Rosenbluth AW, Rosenbluth MN, Teller AH, Teller E, Equation of state calculations by fast computing machines. J Chem Phys. 1953;21(6):1087‐1092. doi: 10.1063/1.1699114
</cite> [<a href="https://doi.org/10.1063/1.1699114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=Equation%20of%20state%20calculations%20by%20fast%20computing%20machines&amp;author=N%20Metropolis&amp;author=AW%20Rosenbluth&amp;author=MN%20Rosenbluth&amp;author=AH%20Teller&amp;author=E%20Teller&amp;volume=21&amp;issue=6&amp;publication_year=1953&amp;pages=1087-1092&amp;doi=10.1063/1.1699114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0078">
<span class="label">78.</span><cite>
Azuma RT. A survey of augmented reality. Presence Teleoper Virtual Environ. 1997;6(4):355‐385. doi: 10.1162/pres.1997.6.4.355
</cite> [<a href="https://doi.org/10.1162/pres.1997.6.4.355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Presence%20Teleoper%20Virtual%20Environ&amp;title=A%20survey%20of%20augmented%20reality&amp;author=RT%20Azuma&amp;volume=6&amp;issue=4&amp;publication_year=1997&amp;pages=355-385&amp;doi=10.1162/pres.1997.6.4.355&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0079">
<span class="label">79.</span><cite>
Atzori L, Iera A, Morabito G, The Internet of things: a survey. Computer Networks. 2010;54(15):2787‐2805. doi: 10.1016/j.comnet.2010.05.010
</cite> [<a href="https://doi.org/10.1016/j.comnet.2010.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Computer%20Networks&amp;title=The%20Internet%20of%20things:%20a%20survey&amp;author=L%20Atzori&amp;author=A%20Iera&amp;author=G%20Morabito&amp;volume=54&amp;issue=15&amp;publication_year=2010&amp;pages=2787-2805&amp;doi=10.1016/j.comnet.2010.05.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0080">
<span class="label">80.</span><cite>
Peshkova M, Yumasheva V, Rudenko E, Kretova N, Timashev P, Demura T, Digital twin concept: healthcare, education, research. J Pathol Inform. 2023;14:100313. doi: 10.1016/j.jpi.2023.100313

</cite> [<a href="https://doi.org/10.1016/j.jpi.2023.100313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10165159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37168801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Pathol%20Inform&amp;title=Digital%20twin%20concept:%20healthcare,%20education,%20research&amp;author=M%20Peshkova&amp;author=V%20Yumasheva&amp;author=E%20Rudenko&amp;author=N%20Kretova&amp;author=P%20Timashev&amp;volume=14&amp;publication_year=2023&amp;pages=100313&amp;pmid=37168801&amp;doi=10.1016/j.jpi.2023.100313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0081">
<span class="label">81.</span><cite>
Schutt M, O'Farrell C, Stamatopoulos K, et al. Simulating the hydrodynamic conditions of the human ascending colon: a digital twin of the dynamic colon model. Pharmaceutics. 2022;14(1)doi: 10.3390/pharmaceutics14010184
</cite> [<a href="https://doi.org/10.3390/pharmaceutics14010184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8778200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35057077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&amp;title=Simulating%20the%20hydrodynamic%20conditions%20of%20the%20human%20ascending%20colon:%20a%20digital%20twin%20of%20the%20dynamic%20colon%20model&amp;author=M%20Schutt&amp;author=C%20O'Farrell&amp;author=K%20Stamatopoulos&amp;volume=14&amp;issue=1&amp;publication_year=2022&amp;pmid=35057077&amp;doi=10.3390/pharmaceutics14010184&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0082">
<span class="label">82.</span><cite>
Jackson E, Markovic LE, Perlini M, Comparison of two‐dimensional imaging to three‐dimensional modeling of intrahepatic portosystemic shunts using computed tomography angiography. Vet Radiol Ultrasound. 2024;doi: 10.1111/vru.13337
</cite> [<a href="https://doi.org/10.1111/vru.13337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38279771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet%20Radiol%20Ultrasound&amp;title=Comparison%20of%20two%E2%80%90dimensional%20imaging%20to%20three%E2%80%90dimensional%20modeling%20of%20intrahepatic%20portosystemic%20shunts%20using%20computed%20tomography%20angiography&amp;author=E%20Jackson&amp;author=LE%20Markovic&amp;author=M%20Perlini&amp;publication_year=2024&amp;pmid=38279771&amp;doi=10.1111/vru.13337&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0083">
<span class="label">83.</span><cite>
Golse N, Joly F, Combari P, et al. Predicting the risk of post‐hepatectomy portal hypertension using a digital twin: a clinical proof of concept. J Hepatology. 2021;74(3):661‐669. doi: 10.1016/j.jhep.2020.10.036
</cite> [<a href="https://doi.org/10.1016/j.jhep.2020.10.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33212089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Hepatology&amp;title=Predicting%20the%20risk%20of%20post%E2%80%90hepatectomy%20portal%20hypertension%20using%20a%20digital%20twin:%20a%20clinical%20proof%20of%20concept&amp;author=N%20Golse&amp;author=F%20Joly&amp;author=P%20Combari&amp;volume=74&amp;issue=3&amp;publication_year=2021&amp;pages=661-669&amp;pmid=33212089&amp;doi=10.1016/j.jhep.2020.10.036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0084">
<span class="label">84.</span><cite>
Shi Y, Deng X, Tong Y, et al. Synergistic digital twin and holographic augmented‐reality‐guided percutaneous puncture of respiratory liver tumor. IEEE Tran Hum ‐Mach Syst. 2022;52(6):1364‐1374. doi: 10.1109/THMS.2022.3185541
</cite> [<a href="https://doi.org/10.1109/THMS.2022.3185541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Tran%20Hum%20%E2%80%90Mach%20Syst&amp;title=Synergistic%20digital%20twin%20and%20holographic%20augmented%E2%80%90reality%E2%80%90guided%20percutaneous%20puncture%20of%20respiratory%20liver%20tumor&amp;author=Y%20Shi&amp;author=X%20Deng&amp;author=Y%20Tong&amp;volume=52&amp;issue=6&amp;publication_year=2022&amp;pages=1364-1374&amp;doi=10.1109/THMS.2022.3185541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0085">
<span class="label">85.</span><cite>
Shrestha U, Esparza JP, Satapathy SK, Vanatta JM, Abramson ZR, Tipirneni‐Sajja A, Hepatic steatosis modeling and MRI signal simulations for comparison of single‐and dual‐R2* models and estimation of fat fraction at 1.5 T and 3T. Comput Biol Med. 2024;174:108448. doi: 10.1016/j.compbiomed.2024.108448

</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2024.108448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12149756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38626508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Biol%20Med&amp;title=Hepatic%20steatosis%20modeling%20and%20MRI%20signal%20simulations%20for%20comparison%20of%20single%E2%80%90and%20dual%E2%80%90R2*%20models%20and%20estimation%20of%20fat%20fraction%20at%201.5%20T%20and%203T&amp;author=U%20Shrestha&amp;author=JP%20Esparza&amp;author=SK%20Satapathy&amp;author=JM%20Vanatta&amp;author=ZR%20Abramson&amp;volume=174&amp;publication_year=2024&amp;pages=108448&amp;pmid=38626508&amp;doi=10.1016/j.compbiomed.2024.108448&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0086">
<span class="label">86.</span><cite>
Joshi S, Shamanna P, Dharmalingam M, et al. Digital twin‐enabled personalized nutrition improves metabolic dysfunction‐associated fatty liver disease in type 2 diabetes: results of a 1‐year randomized controlled study. Artic Endocr Pract
2023;29(12):960‐970. doi: 10.1016/j.eprac.2023.08.016
</cite> [<a href="https://doi.org/10.1016/j.eprac.2023.08.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37778441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artic%20Endocr%20Pract&amp;title=Digital%20twin%E2%80%90enabled%20personalized%20nutrition%20improves%20metabolic%20dysfunction%E2%80%90associated%20fatty%20liver%20disease%20in%20type%202%20diabetes:%20results%20of%20a%201%E2%80%90year%20randomized%20controlled%20study&amp;author=S%20Joshi&amp;author=P%20Shamanna&amp;author=M%20Dharmalingam&amp;volume=29&amp;issue=12&amp;publication_year=2023&amp;pages=960-970&amp;pmid=37778441&amp;doi=10.1016/j.eprac.2023.08.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0087">
<span class="label">87.</span><cite>
Terven J, Diana‐Margarita C‐E, Romero‐González J‐A, A comprehensive review of yolo architectures in computer vision: from YOLOv1 to YOLOv8 and Yolo‐NAS. Mach Learn Knowl Extr. 2023;5(4):1680‐1716. doi: 10.3390/make5040083
</cite> [<a href="https://doi.org/10.3390/make5040083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mach%20Learn%20Knowl%20Extr&amp;title=A%20comprehensive%20review%20of%20yolo%20architectures%20in%20computer%20vision:%20from%20YOLOv1%20to%20YOLOv8%20and%20Yolo%E2%80%90NAS&amp;author=J%20Terven&amp;author=C%E2%80%90E%20Diana%E2%80%90Margarita&amp;author=J%E2%80%90A%20Romero%E2%80%90Gonz%C3%A1lez&amp;volume=5&amp;issue=4&amp;publication_year=2023&amp;pages=1680-1716&amp;doi=10.3390/make5040083&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0088">
<span class="label">88.</span><cite>
Joyia GJ, Liaqat RM, Farooq A, Rehman S, Internet of medical things (IoMT): applications, benefits and future challenges in healthcare domain. J Commun. 2017;12(4):240‐247. doi: 10.12720/JCM.12.4.240-247
</cite> [<a href="https://doi.org/10.12720/JCM.12.4.240-247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Commun&amp;title=Internet%20of%20medical%20things%20(IoMT):%20applications,%20benefits%20and%20future%20challenges%20in%20healthcare%20domain&amp;author=GJ%20Joyia&amp;author=RM%20Liaqat&amp;author=A%20Farooq&amp;author=S%20Rehman&amp;volume=12&amp;issue=4&amp;publication_year=2017&amp;pages=240-247&amp;doi=10.12720/JCM.12.4.240-247&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0089">
<span class="label">89.</span><cite>
Tai Y, Zhang L, Li Q, et al. Digital‐twin‐enabled IoMT system for surgical simulation using rAC‐GAN. IEEE Internet Things J. 2022;9(21):20918‐20931. doi: 10.1109/JIOT.2022.3176300
</cite> [<a href="https://doi.org/10.1109/JIOT.2022.3176300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Internet%20Things%20J&amp;title=Digital%E2%80%90twin%E2%80%90enabled%20IoMT%20system%20for%20surgical%20simulation%20using%20rAC%E2%80%90GAN&amp;author=Y%20Tai&amp;author=L%20Zhang&amp;author=Q%20Li&amp;volume=9&amp;issue=21&amp;publication_year=2022&amp;pages=20918-20931&amp;doi=10.1109/JIOT.2022.3176300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0090">
<span class="label">90.</span><cite>
Zhu L, Lu W, Soleimani M, Li Z, Zhang M, Electrical impedance tomography guided by digital twins and deep learning for lung monitoring. IEEE Trans Instrum Meas. 2023;doi: 10.1109/TIM.2023.3298389
</cite> [<a href="https://doi.org/10.1109/TIM.2023.3298389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Instrum%20Meas&amp;title=Electrical%20impedance%20tomography%20guided%20by%20digital%20twins%20and%20deep%20learning%20for%20lung%20monitoring&amp;author=L%20Zhu&amp;author=W%20Lu&amp;author=M%20Soleimani&amp;author=Z%20Li&amp;author=M%20Zhang&amp;publication_year=2023&amp;doi=10.1109/TIM.2023.3298389&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0091">
<span class="label">91.</span><cite>
Avanzato R, Beritelli F, Lombardo A, Ricci C, Lung‐DT: an AI‐powered digital twin framework for thoracic health monitoring and diagnosis. Sensors. 2024;24(3)doi: 10.3390/s24030958
</cite> [<a href="https://doi.org/10.3390/s24030958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10857717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38339678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sensors&amp;title=Lung%E2%80%90DT:%20an%20AI%E2%80%90powered%20digital%20twin%20framework%20for%20thoracic%20health%20monitoring%20and%20diagnosis&amp;author=R%20Avanzato&amp;author=F%20Beritelli&amp;author=A%20Lombardo&amp;author=C%20Ricci&amp;volume=24&amp;issue=3&amp;publication_year=2024&amp;pmid=38339678&amp;doi=10.3390/s24030958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0092">
<span class="label">92.</span><cite>
Li X, Real‐time digital twins end‐to‐end multi‐branch object detection with feature level selection for healthcare. J Real‐Time Image Process. 2022;doi: 10.1007/s11554-022-01233-z
</cite> [<a href="https://doi.org/10.1007/s11554-022-01233-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Real%E2%80%90Time%20Image%20Process&amp;title=Real%E2%80%90time%20digital%20twins%20end%E2%80%90to%E2%80%90end%20multi%E2%80%90branch%20object%20detection%20with%20feature%20level%20selection%20for%20healthcare&amp;author=X%20Li&amp;publication_year=2022&amp;doi=10.1007/s11554-022-01233-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0093">
<span class="label">93.</span><cite>
Friston KJ, Frith CD, Liddle PF, Frackowiak RS, Functional connectivity: the principal‐component analysis of large (PET) data sets. J Cereb Blood Flow Metab. 1993;13(1):5‐14. doi: 10.1038/jcbfm.1993.4

</cite> [<a href="https://doi.org/10.1038/jcbfm.1993.4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8417010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cereb%20Blood%20Flow%20Metab&amp;title=Functional%20connectivity:%20the%20principal%E2%80%90component%20analysis%20of%20large%20(PET)%20data%20sets&amp;author=KJ%20Friston&amp;author=CD%20Frith&amp;author=PF%20Liddle&amp;author=RS%20Frackowiak&amp;volume=13&amp;issue=1&amp;publication_year=1993&amp;pages=5-14&amp;pmid=8417010&amp;doi=10.1038/jcbfm.1993.4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0094">
<span class="label">94.</span><cite>
Pierpaoli C, Jezzard P, Basser PJ, Barnett A, Di Chiro G, Diffusion tensor MR imaging of the human brain. Radiology. 1996;201(3):637‐648. doi: 10.1148/radiology.201.3.8939209

</cite> [<a href="https://doi.org/10.1148/radiology.201.3.8939209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8939209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Radiology&amp;title=Diffusion%20tensor%20MR%20imaging%20of%20the%20human%20brain&amp;author=C%20Pierpaoli&amp;author=P%20Jezzard&amp;author=PJ%20Basser&amp;author=A%20Barnett&amp;author=G%20Di%20Chiro&amp;volume=201&amp;issue=3&amp;publication_year=1996&amp;pages=637-648&amp;pmid=8939209&amp;doi=10.1148/radiology.201.3.8939209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0095">
<span class="label">95.</span><cite>
Wilkinson MD, Dumontier M, Aalbersberg IJ, et al. The FAIR guiding principles for scientific data management and stewardship. Sci Data. 2016;3(1):1‐9. doi: 10.1038/sdata.2016.18
</cite> [<a href="https://doi.org/10.1038/sdata.2016.18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4792175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26978244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Data&amp;title=The%20FAIR%20guiding%20principles%20for%20scientific%20data%20management%20and%20stewardship&amp;author=MD%20Wilkinson&amp;author=M%20Dumontier&amp;author=IJ%20Aalbersberg&amp;volume=3&amp;issue=1&amp;publication_year=2016&amp;pages=1-9&amp;pmid=26978244&amp;doi=10.1038/sdata.2016.18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0096">
<span class="label">96.</span><cite>
Sporns O, Tononi G, Kötter R, The human connectome: a structural description of the human brain. PLoS Comput Biol. 2005;1(4):e42. doi: 10.1371/journal.pcbi.0010042

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.0010042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1239902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16201007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=The%20human%20connectome:%20a%20structural%20description%20of%20the%20human%20brain&amp;author=O%20Sporns&amp;author=G%20Tononi&amp;author=R%20K%C3%B6tter&amp;volume=1&amp;issue=4&amp;publication_year=2005&amp;pages=e42&amp;pmid=16201007&amp;doi=10.1371/journal.pcbi.0010042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0097">
<span class="label">97.</span><cite>
Pinciroli C, Trianni V, O'Grady R, et al. ARGoS: a modular, parallel, multi‐engine simulator for multi‐robot systems. Swarm Intell. 2012;6:271‐295. doi: 10.1007/s11721-012-0072-5
</cite> [<a href="https://doi.org/10.1007/s11721-012-0072-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Swarm%20Intell&amp;title=ARGoS:%20a%20modular,%20parallel,%20multi%E2%80%90engine%20simulator%20for%20multi%E2%80%90robot%20systems&amp;author=C%20Pinciroli&amp;author=V%20Trianni&amp;author=R%20O'Grady&amp;volume=6&amp;publication_year=2012&amp;pages=271-295&amp;doi=10.1007/s11721-012-0072-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0098">
<span class="label">98.</span><cite>
McMahan B, Moore E, Ramage D, Hampson S, y Arcas BA, Communication‐efficient learning of deep networks from decentralized data. Artif Intell Stat. 2017:1273‐1282. doi: 10.48550/arXiv.1602.05629
</cite> [<a href="https://doi.org/10.48550/arXiv.1602.05629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artif%20Intell%20Stat&amp;title=Communication%E2%80%90efficient%20learning%20of%20deep%20networks%20from%20decentralized%20data&amp;author=B%20McMahan&amp;author=E%20Moore&amp;author=D%20Ramage&amp;author=S%20Hampson&amp;author=BA%20y%20Arcas&amp;publication_year=2017&amp;pages=1273-1282&amp;doi=10.48550/arXiv.1602.05629&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0099">
<span class="label">99.</span><cite>
Sohl‐Dickstein J, Weiss E, Maheswaranathan N, Ganguli S, Deep unsupervised learning using nonequilibrium thermodynamics. International conference on machine learning. 2015:2256‐2265. doi: 10.48550/arXiv.1503.03585
</cite> [<a href="https://doi.org/10.48550/arXiv.1503.03585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0100">
<span class="label">100.</span><cite>
Frid‐Adar M, Diamant I, Klang E, Amitai M, Goldberger J, Greenspan H, GAN‐based synthetic medical image augmentation for increased CNN performance in liver lesion classification. Neurocomputing. 2018;321:321‐331. doi: 10.1016/j.neurocomputing.2018.09.013
</cite> [<a href="https://doi.org/10.1016/j.neurocomputing.2018.09.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurocomputing&amp;title=GAN%E2%80%90based%20synthetic%20medical%20image%20augmentation%20for%20increased%20CNN%20performance%20in%20liver%20lesion%20classification&amp;author=M%20Frid%E2%80%90Adar&amp;author=I%20Diamant&amp;author=E%20Klang&amp;author=M%20Amitai&amp;author=J%20Goldberger&amp;volume=321&amp;publication_year=2018&amp;pages=321-331&amp;doi=10.1016/j.neurocomputing.2018.09.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0101">
<span class="label">101.</span><cite>
Shin H‐C, Tenenholtz NA, Rogers JK, et al. Medical image synthesis for data augmentation and anonymization using generative adversarial networks. Simulation and Synthesis in Medical Imaging: Third International Workshop, SASHIMI 2018, Held in Conjunction with MICCAI 2018, 2018, Proceedings 3. 2018:1‐11. doi: 10.1007/978-3-030-00536-8_1
</cite> [<a href="https://doi.org/10.1007/978-3-030-00536-8_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Simulation%20and%20Synthesis%20in%20Medical%20Imaging:%20Third%20International%20Workshop,%20SASHIMI%202018,%20Held%20in%20Conjunction%20with%20MICCAI%202018&amp;author=H%E2%80%90C%20Shin&amp;author=NA%20Tenenholtz&amp;author=JK%20Rogers&amp;volume=3&amp;publication_year=2018&amp;pages=1-11&amp;doi=10.1007/978-3-030-00536-8_1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0102">
<span class="label">102.</span><cite>
Sundaram S, Hulkund N. GAN‐based data augmentation for chest x‐ray classification. Presented at: 
Applied Data Science for Healthcare Workshop, KDD 2021
; 2021: 14–18. Virtual Conference. arXiv:2107.02970</cite>
</li>
<li id="acm270226-bib-0103">
<span class="label">103.</span><cite>
Yoon JS, Zhang C, Suk H‐I, Guo J, Li X, Sadm: sequence‐aware diffusion model for longitudinal medical image generation. International Conference on Information Processing in Medical Imaging. 2023:388‐400. doi: 10.1007/978-3-031-34048-2_30
</cite> [<a href="https://doi.org/10.1007/978-3-031-34048-2_30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0104">
<span class="label">104.</span><cite>
Dice LR, Measures of the amount of ecologic association between species. Ecology. 1945;26(3):297‐302. doi: 10.2307/1932409
</cite> [<a href="https://doi.org/10.2307/1932409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology&amp;title=Measures%20of%20the%20amount%20of%20ecologic%20association%20between%20species&amp;author=LR%20Dice&amp;volume=26&amp;issue=3&amp;publication_year=1945&amp;pages=297-302&amp;doi=10.2307/1932409&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0105">
<span class="label">105.</span><cite>
Heusel M, Ramsauer H, Unterthiner T, Nessler B, Hochreiter S. GANs trained by a two time‐scale update rule converge to a local Nash equilibrium. Presented at: 
31st Conference on Neural Information Processing Systems (NeurIPS 2017)
. Long Beach, CA; 2017: December.</cite> [<a href="https://scholar.google.com/scholar_lookup?Heusel%20M,%20Ramsauer%20H,%20Unterthiner%20T,%20Nessler%20B,%20Hochreiter%20S.%20GANs%20trained%20by%20a%20two%20time%E2%80%90scale%20update%20rule%20converge%20to%20a%20local%20Nash%20equilibrium.%20Presented%20at:%2031st%20Conference%20on%20Neural%20Information%20Processing%20Systems%20(NeurIPS%202017)%20.%20Long%20Beach,%20CA;%202017:%20December." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0106">
<span class="label">106.</span><cite>
Karras T, Laine S, Aittala M, Hellsten J, Lehtinen J, Aila T. Analyzing and improving the image quality of StyleGAN. Presented at: IEEE/CVF <em>Conference on Computer Vision and Pattern Recognition (CVPR)</em>
; 2020, June 13–19; 2020; 8107–8116. Virtual Conference. doi: 10.1109/CVPR42600.2020.00813
</cite> [<a href="https://doi.org/10.1109/CVPR42600.2020.00813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="acm270226-bib-0107">
<span class="label">107.</span><cite>
Mirza M, Osindero S. Conditional generative adversarial nets. arXiv: arXiv:1411.1784. Preprint posted online November 6, 2014.</cite>
</li>
<li id="acm270226-bib-0108">
<span class="label">108.</span><cite>
Morris PD, van de Vosse FN, Lawford PV, Hose DR, Gunn JP, “Virtual” (Computed) fractional flow reserve: current challenges and limitations. JACC: Cardiovas Interv
2015;8(8):1009‐1017. doi: 10.1016/j.jcin.2015.04.006
</cite> [<a href="https://doi.org/10.1016/j.jcin.2015.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4726733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26117471/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JACC:%20Cardiovas%20Interv&amp;title=Virtual%E2%80%9D%20(Computed)%20fractional%20flow%20reserve:%20current%20challenges%20and%20limitations&amp;author=PD%20Morris&amp;author=FN%20van%20de%20Vosse&amp;author=PV%20Lawford&amp;author=DR%20Hose&amp;author=JP%20Gunn&amp;volume=8&amp;issue=8&amp;publication_year=2015&amp;pages=1009-1017&amp;pmid=26117471&amp;doi=10.1016/j.jcin.2015.04.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0109">
<span class="label">109.</span><cite>
Zhou Z, Rahman Siddiquee MM, Tajbakhsh N, Liang J, Unet++: A nested u‐net architecture for medical image segmentation. Deep Learning in Medical Image Analysis and Multimodal Learning for Clinical Decision Support: 4th International Workshop, DLMIA 2018, and 8th International Workshop, ML‐CDS 2018, Held in Conjunction with MICCAI 2018, September 20, 2018, Proceedings 4. 2018:3‐11. doi: 10.1007/978-3-030-00889-5_1
</cite> [<a href="https://doi.org/10.1007/978-3-030-00889-5_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7329239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32613207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=DLMIA%202018,%20and%208th%20International%20Workshop,%20ML%E2%80%90CDS%202018,%20Held%20in%20Conjunction%20with%20MICCAI%202018,%20September%2020,%202018&amp;author=Z%20Zhou&amp;author=MM%20Rahman%20Siddiquee&amp;author=N%20Tajbakhsh&amp;author=J%20Liang&amp;volume=4&amp;publication_year=2018&amp;pages=3-11&amp;pmid=32613207&amp;doi=10.1007/978-3-030-00889-5_1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0110">
<span class="label">110.</span><cite>
Ravi N, Gabeur V, Hu YT, et al. SAM 2: segment anything in images and videos. arXiv: arXiv:2408.00714. Preprint posted online 2024.</cite>
</li>
<li id="acm270226-bib-0111">
<span class="label">111.</span><cite>
Balakrishnan G, Zhao A, Sabuncu MR, Guttag J, Dalca AV, Voxelmorph: a learning framework for deformable medical image registration. IEEE Trans. Med Imag. 2019;38(8):1788‐1800. doi: 10.1109/TMI.2019.2897538
</cite> [<a href="https://doi.org/10.1109/TMI.2019.2897538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30716034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans.%20Med%20Imag&amp;title=Voxelmorph:%20a%20learning%20framework%20for%20deformable%20medical%20image%20registration&amp;author=G%20Balakrishnan&amp;author=A%20Zhao&amp;author=MR%20Sabuncu&amp;author=J%20Guttag&amp;author=AV%20Dalca&amp;volume=38&amp;issue=8&amp;publication_year=2019&amp;pages=1788-1800&amp;pmid=30716034&amp;doi=10.1109/TMI.2019.2897538&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0112">
<span class="label">112.</span><cite>
Isensee F, Jaeger PF, Kohl SA, Petersen J, Maier‐Hein KH, nnU‐Net: a self‐configuring method for deep learning‐based biomedical image segmentation. Nat Methods. 2021;18(2):203‐211. doi: 10.1038/s41592-020-01008-z

</cite> [<a href="https://doi.org/10.1038/s41592-020-01008-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33288961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=nnU%E2%80%90Net:%20a%20self%E2%80%90configuring%20method%20for%20deep%20learning%E2%80%90based%20biomedical%20image%20segmentation&amp;author=F%20Isensee&amp;author=PF%20Jaeger&amp;author=SA%20Kohl&amp;author=J%20Petersen&amp;author=KH%20Maier%E2%80%90Hein&amp;volume=18&amp;issue=2&amp;publication_year=2021&amp;pages=203-211&amp;pmid=33288961&amp;doi=10.1038/s41592-020-01008-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0113">
<span class="label">113.</span><cite>
Ledesma‐Dominguez L, Carbajal‐Degante E, Moreno‐Hagelsieb G, Perez‐Rueda E, DeepReg: a deep learning hybrid model for predicting transcription factors in eukaryotic and prokaryotic genomes. Sci Reports. 2024;14(1):9155. doi: 10.1038/s41598-024-59487-5
</cite> [<a href="https://doi.org/10.1038/s41598-024-59487-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11551149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38644393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Reports&amp;title=DeepReg:%20a%20deep%20learning%20hybrid%20model%20for%20predicting%20transcription%20factors%20in%20eukaryotic%20and%20prokaryotic%20genomes&amp;author=L%20Ledesma%E2%80%90Dominguez&amp;author=E%20Carbajal%E2%80%90Degante&amp;author=G%20Moreno%E2%80%90Hagelsieb&amp;author=E%20Perez%E2%80%90Rueda&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=9155&amp;pmid=38644393&amp;doi=10.1038/s41598-024-59487-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0114">
<span class="label">114.</span><cite>
Li T, Sahu AK, Talwalkar A, Smith V, Federated learning: challenges, methods, and future directions. IEEE Signal Process Magazine. 2020;37(3):50‐60.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Signal%20Process%20Magazine&amp;title=Federated%20learning:%20challenges,%20methods,%20and%20future%20directions&amp;author=T%20Li&amp;author=AK%20Sahu&amp;author=A%20Talwalkar&amp;author=V%20Smith&amp;volume=37&amp;issue=3&amp;publication_year=2020&amp;pages=50-60&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="acm270226-bib-0115">
<span class="label">115.</span><cite>
Selvaraju RR, Cogswell M, Das A, Vedantam R, Parikh D, Batra D, Grad‐cam: Visual explanations from deep networks via gradient‐based localization. 2017:618‐626.</cite>
</li>
<li id="acm270226-bib-0116">
<span class="label">116.</span><cite>
Simonyan K, Vedaldi A, Zisserman A, Deep inside convolutional networks: Visualising image classification models and saliency maps. arXiv preprint arXiv:13126034. 2013.</cite>
</li>
<li id="acm270226-bib-0117">
<span class="label">117.</span><cite>
Lundberg SM, Lee S‐I, A unified approach to interpreting model predictions. Adv Neural Inf Process Syst. 2017;30.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Neural%20Inf%20Process%20Syst&amp;title=A%20unified%20approach%20to%20interpreting%20model%20predictions&amp;author=SM%20Lundberg&amp;author=S%E2%80%90I%20Lee&amp;publication_year=2017&amp;pages=30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Applied Clinical Medical Physics are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/acm2.70226"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ACM2-26-e70226.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370374/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370374/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370374%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370374/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370374/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370374/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841176/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370374/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841176/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370374/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370374/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="qVfkMYhNSB8HRNICCEwpkAWLSyrwAUtZy0ne0tvNCUzi5d60Tjbp5rrFMpwa7O9v">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
