
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genome-Wide identification and salt stress-responsive expression dynamics of the HMGR gene family in Ziziphus jujuba var. spinosa - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152544D18AF30D330544D10013BAC700.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367118/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Genome-Wide identification and salt stress-responsive expression dynamics of the HMGR gene family in Ziziphus jujuba var. spinosa">
<meta name="citation_author" content="Xiaohan Tang">
<meta name="citation_author_institution" content="College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China">
<meta name="citation_author" content="Xiaojun Ma">
<meta name="citation_author_institution" content="College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China">
<meta name="citation_author" content="Jun Cao">
<meta name="citation_author_institution" content="College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China">
<meta name="citation_author" content="Xinhong Wang">
<meta name="citation_author_institution" content="College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China">
<meta name="citation_author" content="Xuexiang Li">
<meta name="citation_author_institution" content="College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China">
<meta name="citation_author" content="Xiaozhou Yang">
<meta name="citation_author_institution" content="College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China">
<meta name="citation_author" content="Jing Shu">
<meta name="citation_author_institution" content="College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330439">
<meta name="citation_doi" content="10.1371/journal.pone.0330439">
<meta name="citation_pmid" content="40833966">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367118/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367118/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367118/pdf/pone.0330439.pdf">
<meta name="description" content="Terpenoids are critical components of plant environmental adaptation mechanisms. They also exhibit significant therapeutic potential in herbal medicine. 3-Hydroxy-3-methylglutaryl coenzyme A reductase (HMGR), a pivotal rate-limiting enzyme governing ...">
<meta name="og:title" content="Genome-Wide identification and salt stress-responsive expression dynamics of the HMGR gene family in Ziziphus jujuba var. spinosa">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Terpenoids are critical components of plant environmental adaptation mechanisms. They also exhibit significant therapeutic potential in herbal medicine. 3-Hydroxy-3-methylglutaryl coenzyme A reductase (HMGR), a pivotal rate-limiting enzyme governing ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367118/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367118">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330439"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330439.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367118%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367118/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367118/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367118/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330439" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 20;20(8):e0330439. doi: <a href="https://doi.org/10.1371/journal.pone.0330439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330439</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genome-Wide identification and salt stress-responsive expression dynamics of the HMGR gene family in <em>Ziziphus jujuba</em> var. <em>spinosa</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xiaohan Tang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xiaohan Tang</span></h3>
<div class="p">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div>Data curation, Formal analysis, Funding acquisition, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaohan Tang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xiaojun Ma</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiaojun Ma</span></h3>
<div class="p">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div>Data curation, Formal analysis, Methodology, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaojun Ma</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jun Cao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jun Cao</span></h3>
<div class="p">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div>Data curation, Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jun Cao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Xinhong Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Xinhong Wang</span></h3>
<div class="p">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div>Formal analysis, Funding acquisition, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinhong Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Xuexiang Li</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xuexiang Li</span></h3>
<div class="p">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div>Data curation, Methodology, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xuexiang Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Xiaozhou Yang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Xiaozhou Yang</span></h3>
<div class="p">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div>Methodology, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaozhou Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jing Shu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jing Shu</span></h3>
<div class="p">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div>Conceptualization, Data curation, Funding acquisition, Supervision, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jing Shu</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Diaa Abd El-Moneim</span><sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>College of Forestry Engineering, Shandong Agriculture and Engineering University, Jinan, P R China</div>
<div id="edit1">
<sup>2</sup>Arish university, Faculty of agricultural and environmental sciences, EGYPT</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>shujing79@163.com</span></p>
</div>
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Xiaohan Tang</span></strong>: <span class="role">Data curation, Formal analysis, Funding acquisition, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiaojun Ma</span></strong>: <span class="role">Data curation, Formal analysis, Methodology, Visualization, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jun Cao</span></strong>: <span class="role">Data curation, Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xinhong Wang</span></strong>: <span class="role">Formal analysis, Funding acquisition, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xuexiang Li</span></strong>: <span class="role">Data curation, Methodology, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiaozhou Yang</span></strong>: <span class="role">Methodology, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jing Shu</span></strong>: <span class="role">Conceptualization, Data curation, Funding acquisition, Supervision, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Diaa Abd El-Moneim</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 11; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Tang et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367118  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40833966/" class="usa-link">40833966</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Terpenoids are critical components of plant environmental adaptation mechanisms. They also exhibit significant therapeutic potential in herbal medicine. 3-Hydroxy-3-methylglutaryl coenzyme A reductase (HMGR), a pivotal rate-limiting enzyme governing the initial stage of the mevalonate (MVA) pathway in triterpene saponin biosynthesis, remains uncharacterized in <em>Ziziphus jujuba</em> var. <em>spinosa</em>. Through genome-wide and molecular analysis, we systematically identified ZjHMGR isoforms and revealed differential tissue-specific expression patterns and significant salt stress-responsive regulation across identified isoforms. Our findings reveal three evolutionarily conserved ZjHMGR isoforms with a complete HMG-CoA reductase domain and closely related to <em>Populus trichocarpa</em>. Collinearity analysis revealed two collinear gene pairs, and purifying selection was identified as the primary evolutionary force acting on the <em>ZjHMGR</em> gene family. Cis-acting element analysis revealed that <em>ZjHMGR</em> gene family enriched MYB-related, TC-rich repeats, light- and hormone-responsive elements, suggesting transcriptional regulation by environmental stimuli and phytohormones. Spatiotemporal expression analysis via qRT-PCR revealed differential transcriptional patterns of <em>ZjHMGR</em> members, with pronounced upregulation under ABA, MeJA, and light induction. Saline stress disrupted the growth of wild jujube seedlings while activating <em>ZjHMGR</em> expression alongside other MVA pathway genes. Overexpression of <em>ZjHMGR</em> enhances salt stress resistance in <em>Arabidopsis thaliana</em>. This study lays the foundation for further investigations into the molecular mechanisms of the <em>ZjHMGR</em> gene family concerning saponin biosynthesis, phytohormone interactions, and salt tolerance in wild jujube.</p></section><section id="sec001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p><em>Ziziphus jujuba</em> var. <em>spinosa</em> (Rhamnaceae) is a diploid progenitor of cultivated jujube indigenous to China. It represents a phylogenetically and economically significant species with dual medicinal-agronomic value [<a href="#pone.0330439.ref001" class="usa-link" aria-describedby="pone.0330439.ref001">1</a>]. This hardy perennial, colloquially termed “wild jujube,” serves as a critical genetic resource for arid-land agriculture due to its exceptional abiotic stress resilience, maintaining physiological homeostasis under saline conditions (5‰ NaCl) and nutrient-deficient soils [<a href="#pone.0330439.ref002" class="usa-link" aria-describedby="pone.0330439.ref002">2</a>]. Wild jujube serves as a key pioneer species for ecological restoration and soil conservation in harsh environments, evidenced by its capacity to thrive in marginal ecosystems, including saline-alkaline lands and barren mountainous regions [<a href="#pone.0330439.ref003" class="usa-link" aria-describedby="pone.0330439.ref003">3</a>]. The <em>Shennong Bencao Jing</em> (Divine Farmer’s Materia Medica, circa 200–300 CE), an ancient Chinese pharmacopeia, documents the phytotherapeutic efficacy of wild jujube in “regulating visceral homeostasis, enhancing metabolic regulation, and promoting longevity” [<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>]. Phytochemical profiling reveals therapeutic metabolites distributed across its leaves, fruits, flowers, and seeds, with particular pharmacological interest in its mature seeds (<em>suanzaoren</em>), a traditional Chinese medicinal material [<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>–<a href="#pone.0330439.ref006" class="usa-link" aria-describedby="pone.0330439.ref006">6</a>]. Modern pharmacological studies mechanistically link these traditional claims to the species’ bioactive triterpenoids and polysaccharides, which demonstrate multi-target regulatory effects on plant defense-immune networks, metabolism and oxidative stress pathways [<a href="#pone.0330439.ref001" class="usa-link" aria-describedby="pone.0330439.ref001">1</a>,<a href="#pone.0330439.ref002" class="usa-link" aria-describedby="pone.0330439.ref002">2</a>,<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>]. The species’ unique combination of stress tolerance traits and bioactive compound biosynthesis underscores its importance in sustainable crop development and natural product discovery.</p>
<p>Plants regulate complex adaptive mechanisms to counter dynamic environmental perturbations through evolutionarily conserved stress perception systems. These coordinated responses exhibit fine-tune in maintaining cellular osmotic equilibrium, redox homeostasis, and metabolic balance – fundamental requirements for environmental acclimatization [<a href="#pone.0330439.ref007" class="usa-link" aria-describedby="pone.0330439.ref007">7</a>–<a href="#pone.0330439.ref010" class="usa-link" aria-describedby="pone.0330439.ref010">10</a>]. The ability of plants to withstand extreme environmental challenges stems primarily from phylogenetically conserved specialized metabolic systems [<a href="#pone.0330439.ref011" class="usa-link" aria-describedby="pone.0330439.ref011">11</a>,<a href="#pone.0330439.ref012" class="usa-link" aria-describedby="pone.0330439.ref012">12</a>]. This evolutionary mechanism has given rise to three major groups of metabolites, phenolic derivatives including flavonoids, terpenoids including saponins, and alkaloids, which collectively mediate key physiological functions ranging from scavenging of reactive oxygen species (ROS) to phytohormone regulation, thereby maintaining cellar homeostasis in the face of abiotic and biotic stress [<a href="#pone.0330439.ref011" class="usa-link" aria-describedby="pone.0330439.ref011">11</a>,<a href="#pone.0330439.ref013" class="usa-link" aria-describedby="pone.0330439.ref013">13</a>,<a href="#pone.0330439.ref014" class="usa-link" aria-describedby="pone.0330439.ref014">14</a>]. Terpenoids represent the phylogenetically conserved secondary metabolites. They coordinate important plant-environment interactions through dual chemical-ecological functions, regulate interactions through organic compounds, and act as a regulatory link between phytohormone signaling [<a href="#pone.0330439.ref011" class="usa-link" aria-describedby="pone.0330439.ref011">11</a>].</p>
<p>The dual functionality of terpenoid metabolism in <em>Ziziphus jujuba</em> var. <em>spinosa</em> encompasses both medicinal biosynthesis [<a href="#pone.0330439.ref001" class="usa-link" aria-describedby="pone.0330439.ref001">1</a>,<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>] and stress adaptogenesis [<a href="#pone.0330439.ref015" class="usa-link" aria-describedby="pone.0330439.ref015">15</a>–<a href="#pone.0330439.ref017" class="usa-link" aria-describedby="pone.0330439.ref017">17</a>]. Structural elucidation identifies jujuboside saponins – dammarane triterpenoids isoprenylated as pharmacologically active analogs to panaxosides [<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>,<a href="#pone.0330439.ref005" class="usa-link" aria-describedby="pone.0330439.ref005">5</a>]. Simultaneously, terpenoid-based hormonal regulators, notably abscisic acid (ABA) and gibberellin (GA3) mediate cross-talk between phytohormone signaling, reinforcing membrane integrity through sterol-lipid integration, and direct ROS scavenging through conjugated isoprene units, thereby coordinating growth-defense tradeoffs under stressful constraints [<a href="#pone.0330439.ref009" class="usa-link" aria-describedby="pone.0330439.ref009">9</a>,<a href="#pone.0330439.ref018" class="usa-link" aria-describedby="pone.0330439.ref018">18</a>–<a href="#pone.0330439.ref020" class="usa-link" aria-describedby="pone.0330439.ref020">20</a>]. Evolutionary analyses reveal terpenoid indispensable role in plant stress resilience, particularly through compartmentalized biosynthesis during abiotic/biotic stressors. Crucially, terpenoid diversification patterns correlate with stressful adaptation, demonstrating selective pressure maintenance of their biosynthetic gene clusters [<a href="#pone.0330439.ref011" class="usa-link" aria-describedby="pone.0330439.ref011">11</a>]. The metabolic crosstalk between terpenoid pathways and phenolic antioxidant networks establishes a robust defense continuum, positioning these compounds as fundamental components of plant adaptive arsenals.</p>
<p>Plant terpenoid biosynthesis operates through two compartmentalized pathways, the cytosolic mevalonate (MVA) route and plastidial methylerythritol phosphate (MEP) system [<a href="#pone.0330439.ref021" class="usa-link" aria-describedby="pone.0330439.ref021">21</a>]. 3-Hydroxy-3-methylglutaryl-CoA reductase (HMGR), the principal flux-controlling enzyme in MVA metabolism, catalyzes the NADPH-dependent irreversible reduction of HMG-CoA to mevalonate [<a href="#pone.0330439.ref022" class="usa-link" aria-describedby="pone.0330439.ref022">22</a>,<a href="#pone.0330439.ref023" class="usa-link" aria-describedby="pone.0330439.ref023">23</a>]. This committed step governs carbon partitioning into sesquiterpenoids, phytosterols, and bioactive triterpenes through substrate channeling mechanisms, while spatial segregation of MEP-derived isoprenoids prevents metabolic cross-talk between organelles [<a href="#pone.0330439.ref011" class="usa-link" aria-describedby="pone.0330439.ref011">11</a>,<a href="#pone.0330439.ref024" class="usa-link" aria-describedby="pone.0330439.ref024">24</a>,<a href="#pone.0330439.ref025" class="usa-link" aria-describedby="pone.0330439.ref025">25</a>].</p>
<p>HMGR multigene family exhibits evolutionary diversification across plant taxa, with molecular characterization extending from model <em>Arabidopsis thaliana</em> [<a href="#pone.0330439.ref026" class="usa-link" aria-describedby="pone.0330439.ref026">26</a>,<a href="#pone.0330439.ref027" class="usa-link" aria-describedby="pone.0330439.ref027">27</a>], to pharmacologically significant species (<em>Panax ginseng</em> [<a href="#pone.0330439.ref028" class="usa-link" aria-describedby="pone.0330439.ref028">28</a>]<em>, Salvia miltiorrhiza</em> [<a href="#pone.0330439.ref029" class="usa-link" aria-describedby="pone.0330439.ref029">29</a>]<em>, Ginkgo biloba</em> [<a href="#pone.0330439.ref030" class="usa-link" aria-describedby="pone.0330439.ref030">30</a>], <em>Lithospermum erythrorhizon</em> [<a href="#pone.0330439.ref023" class="usa-link" aria-describedby="pone.0330439.ref023">23</a>] and <em>Taxus</em> [<a href="#pone.0330439.ref024" class="usa-link" aria-describedby="pone.0330439.ref024">24</a>]) and agronomically vital crops (<em>Glycine max</em> [<a href="#pone.0330439.ref031" class="usa-link" aria-describedby="pone.0330439.ref031">31</a>] and <em>Vitis vinifera</em> [<a href="#pone.0330439.ref032" class="usa-link" aria-describedby="pone.0330439.ref032">32</a>]). Functional genomics reveal HMGR’s dual regulatory capacity, transgenic overexpression enhances terpenoid biosynthetic flux while conferring growth-defense tradeoff advantages [<a href="#pone.0330439.ref022" class="usa-link" aria-describedby="pone.0330439.ref022">22</a>,<a href="#pone.0330439.ref028" class="usa-link" aria-describedby="pone.0330439.ref028">28</a>,<a href="#pone.0330439.ref031" class="usa-link" aria-describedby="pone.0330439.ref031">31</a>,<a href="#pone.0330439.ref033" class="usa-link" aria-describedby="pone.0330439.ref033">33</a>,<a href="#pone.0330439.ref034" class="usa-link" aria-describedby="pone.0330439.ref034">34</a>]. Photoregulation mechanisms were demonstrated in <em>Solanum tuberosum</em>, where HMGR activity undergoes light-dependent post-translational modulation – dark-induced suppression and phytochrome-mediated recovery [<a href="#pone.0330439.ref035" class="usa-link" aria-describedby="pone.0330439.ref035">35</a>]. Cross-species analyses in <em>Malus domestica</em> domestica illustrate HMGR’s pleiotropic effects, <em>MdHMGR5</em> ectopic expression coordinately upregulates MVA/MEP pathway genes, enhances ROS scavenging capacity, and promotes stress-adaptive morphogenesis [<a href="#pone.0330439.ref034" class="usa-link" aria-describedby="pone.0330439.ref034">34</a>]. <em>Taraxacum kok-saghyz</em> genome analysis identified HMGR promoters enriched in cis-regulatory elements (containing methyl jasmonate (MeJA)/ethylene-responsive motifs), with hormonal induction triggering coordinated metabolic reprogramming – HMGR upregulation paralleled by osmoprotectant accumulation and photosynthetic adaptation [<a href="#pone.0330439.ref022" class="usa-link" aria-describedby="pone.0330439.ref022">22</a>].</p>
<p>The halophytic adaptation of <em>Ziziphus jujuba</em> var. <em>spinosa</em>, a halophytic species with dual medicinal-edible value, manifested through its oligotrophic resilience and xerophytic endurance, remains mechanistically enigmatic despite extensive characterization of its triterpenoid biosynthesis since genome sequencing [<a href="#pone.0330439.ref036" class="usa-link" aria-describedby="pone.0330439.ref036">36</a>]. The dynamic interplay between defensive terpenogenesis and environmental sensing in medicinal plants manifests through two regulatory axes: biotic stress-responsive synthesis of phytoanticipins and abiotic stress-activated metabolic channeling via phytohormone-ROS signaling hubs. Cross-kingdom RNAi studies reveal UVR8-dependent MAPK cascades transduce light stress into monoterpene synthase activation [<a href="#pone.0330439.ref037" class="usa-link" aria-describedby="pone.0330439.ref037">37</a>], while drought/salinity co-opt ABA-JAZ modules may to redirect MEP/MVA flux toward stress-protective triterpenoids [<a href="#pone.0330439.ref011" class="usa-link" aria-describedby="pone.0330439.ref011">11</a>,<a href="#pone.0330439.ref018" class="usa-link" aria-describedby="pone.0330439.ref018">18</a>]. Notably, the isoprenoid-derived phytohormone ABA orchestrates multilevel stress adaptation via guard cell dynamics regulation, ionic equilibrium maintenance, and stress-responsive transcriptome reprogramming [<a href="#pone.0330439.ref009" class="usa-link" aria-describedby="pone.0330439.ref009">9</a>].</p>
<p>This study focus on a comprehensive investigation of the <em>ZjHMGR</em> gene family, a potential genetic locus underlying the exceptional stress resilience of <em>Ziziphus jujuba</em> var. <em>spinosa</em>. Our genome-wide characterization systematically delineates chromosomal distribution patterns, protein structural dynamics and physicochemical profiles, evolutionary conservation of functional motifs/domains, stress-responsive cis-regulatory architectures in promoter regions, phylogenetic divergence across angiosperm taxa, and tissue-specific spatiotemporal expression signatures. Functional interrogation via heterologous overexpression models analyzed specific functional stratification, with <em>ZjHMGR1/2/3</em> exhibiting distinct salt tolerance modulation capacities. Through integrated multi-omics analysis, we have elucidated a tripartite regulatory cascade encompassing upstream transcriptional orchestration, dynamic expression fine-tuning, and downstream metabolic network recalibration. The mechanistic dissection reveals ZjHMGRs’ multifunctionality in abiotic stress perception and signaling transduction, with particular emphasis on their pleiotropic regulation through isoprenoid biosynthesis pathways. Our findings establish novel regulatory modules that bridge genetic architecture with metabolic reprogramming under stress conditions. These discoveries providean evolutionary framework for extremophyte genetic resource exploitation and synthetic biology blueprints for salinity-resilient crop engineering. By resolving the ZjHMGR-mediated stress adaptation paradigm, this work advances both fundamental understanding of plant-environment interactions and translational applications in sustainable agriculture.</p></section><section id="sec002"><h2 class="pmc_sec_title">2. Materials and methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">2.1. Genome-wide identification of <em>HMGR</em> genes in <em>Ziziphus jujuba</em> var. <em>spinosa</em>
</h3>
<p>Based on the published whole genome data of <em>Ziziphus jujuba</em> var. <em>spinosa</em> [<a href="#pone.0330439.ref036" class="usa-link" aria-describedby="pone.0330439.ref036">36</a>], the genome sequence and annotation information of wild jujube were downloaded from NCBI to build a local BLAST database. Using the <em>Arabidopsis</em> HMGR protein sequence as the query sequence, local BLASTp was performed to obtain the <em>HMGR</em> gene family members of <em>Ziziphus jujuba</em> var. <em>spinosa</em> using BioEdit software. Then, the Pfam database (<a href="https://www.ebi.ac.uk/jdispatcher/pfa/pfamscan" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/jdispatcher/pfa/pfamscan</a>) and NCBI-CDD database (<a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi</a>) were used to identify the HMGR family members of <em>Ziziphus jujuba</em> var. <em>spinosa</em>. Chromosomal localisation and gene structure analysis of <em>ZjHMGRs</em> were performed using Tbtools software [<a href="#pone.0330439.ref038" class="usa-link" aria-describedby="pone.0330439.ref038">38</a>].</p></section><section id="sec004"><h3 class="pmc_sec_title">2.2. Physicochemical properties, multiple sequence alignments, motif, conserved domain analysis and protein structure prediction</h3>
<p>The physicochemical properties of the ZjHMGR proteins, including their molecular weight and isoelectric point, were analysed by the Protparam tool, which is available online via Expasy (v2023.1) [<a href="#pone.0330439.ref039" class="usa-link" aria-describedby="pone.0330439.ref039">39</a>] (<a href="https://web.expasy.org/protparam/).The" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://web.expasy.org/protparam/).The</a> conserved domains and conserved motifs of the three ZjHMGR proteins were analyzed via the SMART DOMAIN [<a href="#pone.0330439.ref040" class="usa-link" aria-describedby="pone.0330439.ref040">40</a>] (<a href="https://smart.embl.de/smart/set_mode.cgi?NORMAL=1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://smart.embl.de/smart/set_mode.cgi?NORMAL=1</a>) and MEME suit [<a href="#pone.0330439.ref041" class="usa-link" aria-describedby="pone.0330439.ref041">41</a>] (<a href="http://memesuite.org/tools/meme" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://memesuite.org/tools/meme</a>), and the maximum number of motifs was 10. Visualization of conserved motifs and structural domains by Tbtools software. The transmembrane structural domains, signal peptide and subcellular localisation of ZjHMGR proteins were predicted and analysed via DeepTMHMM v2.0 [<a href="#pone.0330439.ref042" class="usa-link" aria-describedby="pone.0330439.ref042">42</a>] (<a href="https://dtu.biolib.com/DeepTMHMM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dtu.biolib.com/DeepTMHMM</a>), SignalP 6.0 [<a href="#pone.0330439.ref043" class="usa-link" aria-describedby="pone.0330439.ref043">43</a>] (<a href="https://services.healthtech.dtu.dk/services/SignalP-6.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/SignalP-6.0/</a>) and the Plant-mPLoc tool of Cell-PLoc 2.0 [<a href="#pone.0330439.ref044" class="usa-link" aria-describedby="pone.0330439.ref044">44</a>] (<a href="https://www.csbio.sjtu.edu.cn/bioinf/plant-multi/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.csbio.sjtu.edu.cn/bioinf/plant-multi/</a>), respectively. Multiple sequence alignments was analyzed by ESPript 3.0 [<a href="#pone.0330439.ref045" class="usa-link" aria-describedby="pone.0330439.ref045">45</a>] (<a href="https://espript.ibcp.fr/ESPript/ESPript/index.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://espript.ibcp.fr/ESPript/ESPript/index.php</a>). The secondary structure and 3D structure of the remaining proteins was predicted from the SOPMA (<a href="https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=npsa%20_sopma.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=npsa%20_sopma.html</a>) and the AlphaFold3 [<a href="#pone.0330439.ref046" class="usa-link" aria-describedby="pone.0330439.ref046">46</a>] online website (<a href="https://deepmind.google/technologies/alphafold/alphafold-server/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://deepmind.google/technologies/alphafold/alphafold-server/</a>), separately.</p></section><section id="sec005"><h3 class="pmc_sec_title">2.3. Phylogenetic analysis, collinearity analysis and selective pressure discovery</h3>
<p>The protein sequences of HMGRs in date palm and <em>Arabidopsis thaliana</em>, soybean, grape, apple, rice, mauve poplar, and corn were aligned using MUCLE in MEGA 11.0, and a phylogenetic tree was constructed with the maximum likelihood method with 1000 replications [<a href="#pone.0330439.ref047" class="usa-link" aria-describedby="pone.0330439.ref047">47</a>], to explore the evolutionary relationship of ZjHMGRs. The phylogenetic tree was beautified by the online website iTOL (<a href="https://itol.embl.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://itol.embl.de/</a>). Intra- and inter-species collinearity analyses of the ZjHMGR gene were performed using the McScanX plug-in of the TBtools software. And the Ks values of collinear gene pairs were calculated and the divergence time was calculated with the formula T = Ks/2r × 10<sup>−6</sup> (million years Mya), where Ks is one synonymous substitution at each locus and r is 1.5 × 10<sup>−8</sup> per year at each locus for dicotyledons synonymous substitutions [<a href="#pone.0330439.ref048" class="usa-link" aria-describedby="pone.0330439.ref048">48</a>].</p></section><section id="sec006"><h3 class="pmc_sec_title">2.4. Cis-acting element analysis</h3>
<p>Potential promoter sequences, 2000 bp upstream of the transcription start site of the ZjHMGR genes, were extracted by TBtools, analyzed for promoter region cis-acting element prediction by PlantCARE database [<a href="#pone.0330439.ref049" class="usa-link" aria-describedby="pone.0330439.ref049">49</a>] (<a href="http://bioinformatics.psb.ugent.be/webtools/plantcare/html/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bioinformatics.psb.ugent.be/webtools/plantcare/html/</a>), and subsequently visualized by TBtools.</p></section><section id="sec007"><h3 class="pmc_sec_title">2.5. <em>Ziziphus jujuba</em> var. <em>spinosa</em> growth and sample collection</h3>
<p>The seeds (ZH#1 and ZH#2) of <em>Ziziphus jujuba</em> var. <em>spinosa</em> were obtained from Zanhuang Country, Hebei, China. <em>Ziziphus jujuba</em> var. <em>spinosa</em> (wild jujube) is widely distributed and cultivated in China. The ZH#1 and ZH#2 germplasm used are not listed on any protected plant or animal conservation lists. Unless otherwise specified, the ZH#1 germplasm served as the primary plant material in this study. Seeds were soaked at 30 <sup>◦</sup>C for 2–3 days before germination, and plants were grown under continuous white light at 23 ± 2 <sup>◦</sup>C in soil. Roots, stems, leaves and flowers were collected from five -year-old plants grown in a greenhouse. Four- to six-leaf stage seedings grown at hydroponic tank were selected for treatment analysis. Seedings were treated with NaCl/Na<sub>2</sub>SO4/NaHCO<sub>3</sub>/Na<sub>2</sub>CO<sub>3</sub> 1:1:1:1 M ratio for 0–4 d to simulate salinity stress. For hormone treatment, seedings were irrigated with 100 μmol/L ABA or 1 mmol/ MeJA, and samples were collected at different periods of 0 h, 6 h, 12 h, 24 h, respectively. Following a regular growth stage, the seedlings were transferred to darkness for 24 h and then returned to light for a further 0–24 h recovery, to analyze effects of light on <em>ZjHMGR</em> expression. All samples were quick-frozen in liquid nitrogen and stored at −80 ◦C.</p></section><section id="sec008"><h3 class="pmc_sec_title">2.6. Physiological indicators and antioxidant enzyme assays</h3>
<p>Four to six leaf stage seedings grown at hydroponic tank were treated with NaCl/Na<sub>2</sub>SO4/NaHCO<sub>3</sub>/Na<sub>2</sub>CO<sub>3</sub> 1:1:1:1 M ratio for 0−8 d to simulate salinity stress. Quantification of proline, soluble sugar, chlorophyll and antioxidant enzyme was conducted on ~50 mg of fresh sample. The chlorophyll contents of seedings were determined spectrophotometrically from the absorbance at 663 nm and 649 nm following standard extraction. The contents of proline and soluble sugar in samples was determined in accordance with the assay kits (Nanjing Jiancheng Bioengineering Institute, cat#A107-1–1 and cat#A145-1–1, China).CAT, SOD, and POD activity of seedings were analyzed in accordance with the assay kits (Nanjing Jiancheng Bioengineering Institute, cat# A007-1–1, cat# A001-1–2, and cat# A084-3–1). All spectrophotometric analyses were conducted on a BMG SPECTRO star omega microplate reader (BMG Labtech, Germany). All data were obtained from at least three biological replicates.</p></section><section id="sec009"><h3 class="pmc_sec_title">2.7. RNA extraction and qRT-PCR analysis</h3>
<p>Total RNA Extraction Reagent (Vazyme, R401), reverse transcription kit HiScript III RT SuperMix for qRTPCR (+gDNA wiper) (Vazyme, R323) and ChamQ Universal SYBR qRT-PCR Master Mix (Vazyme, Q711) were used for qRT-PCR following the manufacturer’s instructions. mRNA-sequencing was conducted using the high-throughput Genomics Core facility at Novomagic (<a href="https://www.novogene.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.novogene.cn</a>). Gene expression was normalized to that of <em>AtACTIN8</em> or <em>ZjACTIN7</em> and then the expression level in the different timepoint was determined relative to the level in the control plants. qRT-PCR analysis was performed with 3 independent biological replicates, and the data were analysed using the 2<sup>–ΔΔ</sup>CT method [<a href="#pone.0330439.ref050" class="usa-link" aria-describedby="pone.0330439.ref050">50</a>] with mean <em>± </em>standard deviation (SD).Statistical differences were shown by different letters according to One-Way ANOVA followed by Tukey’s HSD test, *<em>P</em> &lt; 0.05. The specific primers used for this paper are listed in Supplementary <a href="#pone.0330439.s003" class="usa-link">S2 Table</a>.</p></section><section id="sec010"><h3 class="pmc_sec_title">2.8. Plasmid construction and plant transformation</h3>
<p>To construct gene fusions with <em>pROKII-X-eGFP</em>, the coding sequences (removed the stop codon) of <em>ZjHMGRs</em> were each individually amplified through PCR, cloned, and inserted into the corresponding vectors using a ClonExpress II One Step Cloning Kit (Vazyme, C112). The construct <em>pROKII-ZjHMGR1/2/3-eGFP</em> was transformed into Col-0 (<em>Arabidopsis thaliana</em>) by the floral dip method. Based on the selection with antibiotics (using 50 milligrams per liter of kanamycin), a T1 generation transgenic population was cultivated. Subsequent generations were screened until complete homozygosity was achieved. The transformed individuals were then used for subsequent functional studies. After vernalizing at 4 °C for 24 h, the transgenic lines of <em>A. thaliana</em> grew in 1/2MS medium at 21°C under 16-h-light/8-h-dark conditions. Transgenic homozygous lines were utilized to detect biomass and observe root and shoot growth. The main root’s length was measured by ImageJ. The fresh weight and dry weight was analyzed by using fresh seedings of 8-day-old <em>A. thaliana</em>. The primer pairs are listed in Supplementary <a href="#pone.0330439.s003" class="usa-link">S3 Table</a>, and all the generated constructs were confirmed by sequencing.</p></section></section><section id="sec011"><h2 class="pmc_sec_title">3. Results</h2>
<section id="sec012"><h3 class="pmc_sec_title">3.1. Identification of HMGR Family Members in <em>Ziziphus jujuba</em> var. <em>spinosa</em>
</h3>
<p>The <em>Ziziphus jujuba</em> var. <em>spinosa</em> genome was systematically interrogated through homology-based BLASTp analysis employing the <em>Arabidopsis thaliana</em> HMGR protein sequence (AT2G17370) as the query. A total of three <em>HMGRs</em> were identified in the <em>Ziziphus jujuba</em> var. <em>spinosa</em> genome, and the members were named <em>ZjHMGR1</em> ~ <em>ZjHMGR3</em> according to the order of localization in the chromosome.</p>
<p>Chromosome distribution mapping revealed the ZjHMGR paralogs reside on chromosomes 6, 8, and 9, with ZjHMGR1-ZjHMGR3 names assigned according to their linear genomic arrangement (S1 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Comparative analysis of gene architecture demonstrated substantial structural diversities among family members, with gene lengths ranging from 2,914 bp to 4,012 bp, corresponding to coding sequence (CDS) lengths of 1,710−1,812 bp (<a href="#pone.0330439.t001" class="usa-link">Table 1</a>). Strikingly, ZjHMGR3 exhibits a paradoxical genomic architecture – despite possessing the most compact genomic, it contains the longest CDS to code protein (<a href="#pone.0330439.t001" class="usa-link">Table 1</a>). This intriguing structural feature suggests ZjHMGR3 may have evolved distinct regulatory mechanisms, potentially through intron loss or alternative splicing patterns not observed in ZjHMGR1 or ZjHMGR2.</p>
<section class="tw xbox font-sm" id="pone.0330439.t001"><h4 class="obj_head">Table 1. The detail information of <em>HMGR</em> gene family members in <em>Ziziphus jujuba</em> var. <em>spinosa.</em>
</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Gene name</th>
<th align="left" rowspan="1" colspan="1">Gene ID</th>
<th align="left" rowspan="1" colspan="1">Chromosomal localization</th>
<th align="left" rowspan="1" colspan="1">Gene length/bp</th>
<th align="left" rowspan="1" colspan="1">Location</th>
<th align="left" rowspan="1" colspan="1">CDS length/bp</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR1</td>
<td align="left" rowspan="1" colspan="1">ZspiChr6G00030740.1</td>
<td align="left" rowspan="1" colspan="1">Chr6</td>
<td align="left" rowspan="1" colspan="1">3864</td>
<td align="left" rowspan="1" colspan="1">782618-786481</td>
<td align="left" rowspan="1" colspan="1">1710</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR2</td>
<td align="left" rowspan="1" colspan="1">ZspiChr8G00149430.1</td>
<td align="left" rowspan="1" colspan="1">Chr8</td>
<td align="left" rowspan="1" colspan="1">4012</td>
<td align="left" rowspan="1" colspan="1">8050134-8054145</td>
<td align="left" rowspan="1" colspan="1">1758</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR3</td>
<td align="left" rowspan="1" colspan="1">ZspiChr9G00131940.1</td>
<td align="left" rowspan="1" colspan="1">Chr9</td>
<td align="left" rowspan="1" colspan="1">2914</td>
<td align="left" rowspan="1" colspan="1">19404627-19407540</td>
<td align="left" rowspan="1" colspan="1">1812</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330439.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Comprehensive biophysical characterization of ZjHMGR isoforms was performed through computational physiochemical profiling using the Expasy ProtParam platform (v2023.1). The analysis revealed conserved polypeptide lengths (569–603 residues) across family members, corresponding to predicted molecular masses of 60.88–64.68 kDa. Intriguingly, ZjHMGR1 (pI 7.59) and ZjHMGR2 (pI 7.17) exhibited alkaline isoelectric points with elevated instability indices (42.6 and 41.75 respectively), classifying them as intrinsically disordered proteins through threshold &gt;40. In contrast, ZjHMGR3 demonstrated divergent biophysical behavior with a mildly acidic pI (6) and enhanced thermodynamic stability (instability index 38.25), suggesting potential structural stabilization through distinct hydrophobic moment analysis. Subcellular compartmentalization predictions unanimously localized all three paralogs to the endoplasmic reticulum membrane system, corroborated by conserved dual transmembrane helices (TMHMM v2.0) – a structural hallmark of sterol biosynthetic enzymes (<a href="#pone.0330439.t002" class="usa-link">Table 2</a>). This membrane-anchored configuration aligns with HMGR’s established role in mevalonate pathway channeling, though ZjHMGR3’s atypical C-terminal amphipathic helix implies potential isoform-specific regulatory interactions.</p>
<section class="tw xbox font-sm" id="pone.0330439.t002"><h4 class="obj_head">Table 2. The information of physicochemical properties of ZjHMGRs family proteins.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Gene name</th>
<th align="left" rowspan="1" colspan="1">Protein Length/aa</th>
<th align="left" rowspan="1" colspan="1">Molecular Weight/kDa</th>
<th align="left" rowspan="1" colspan="1">pI<sup>1</sup>
</th>
<th align="left" rowspan="1" colspan="1">Instability Index</th>
<th align="left" rowspan="1" colspan="1">Predicted Location(s)</th>
<th align="left" rowspan="1" colspan="1">Number of TMHs<sup>3</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR1</td>
<td align="left" rowspan="1" colspan="1">569</td>
<td align="left" rowspan="1" colspan="1">60.88</td>
<td align="left" rowspan="1" colspan="1">7.59</td>
<td align="left" rowspan="1" colspan="1">42.6</td>
<td align="left" rowspan="1" colspan="1">ER<sup>2</sup>
</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR2</td>
<td align="left" rowspan="1" colspan="1">585</td>
<td align="left" rowspan="1" colspan="1">63.16</td>
<td align="left" rowspan="1" colspan="1">7.17</td>
<td align="left" rowspan="1" colspan="1">41.75</td>
<td align="left" rowspan="1" colspan="1">ER</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR3</td>
<td align="left" rowspan="1" colspan="1">603</td>
<td align="left" rowspan="1" colspan="1">64.68</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">38.25</td>
<td align="left" rowspan="1" colspan="1">ER</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330439.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t002fn001"><p><sup>1</sup> PI: Isoelectric point. <sup>2</sup> ER: Endoplasmic reticulum. <sup>3</sup> TMH: Transmembrane helices.</p></div></div></section></section><section id="sec013"><h3 class="pmc_sec_title">3.2. Variation across the ZjHMGR Family in terms of domain, motif composition, and conserved structure</h3>
<p>This computational screening revealed three putative ZjHMGR homologs (ZjHMGR1–3) that were subsequently subjected to rigorous domain architecture validation using both Pfam (v35.0) and NCBI’s Conserved Domain Database (CDD v3.20). All candidate proteins exhibited the characteristic sterol-sensing domain (SSD, PF00368) and catalytic HMG-CoA reductase domain (cl00205), confirming their classification as bona fide members of the plant HMGR class I family (cd00643), as determined by iterative profile-profile alignment. To elucidate structure-function relationships within the ZjHMGR proteoforms, we performed integrative structural analyses combining multiple sequence alignment, secondary structure prediction, and 3D modeling. Domain architecture interrogation revealed all isoforms possess the canonical plant HMGR organization – ten evolutionarily conserved motifs and the HMG-CoA reductase І catalytic core (<a href="#pone.0330439.g001" class="usa-link">Fig 1A</a> and <a href="#pone.0330439.g001" class="usa-link">1B</a>). Phylogenetic conservation of gene architecture is frequently observed among paralogous enzymes, yet comparative exon-intron mapping uncovered atypical structural compression in ZjHMGR3. While ZjHMGR1 (4 exons; 3,864 bp) and ZjHMGR2 (4 exons; 4,012 bp) maintain conserved splicing patterns, ZjHMGR3 (3 exons; 2,914 bp) achieves comparable coding capacity (1,812 bp CDS) through intronic optimization – a phenomenon potentially arising from retrotransposon-mediated intron loss (<a href="#pone.0330439.g001" class="usa-link">Fig 1C</a>). Critical functional dissection of conserved motifs identified three catalytic hotspots: Motif 1 (NADP(H)-binding fingerprint), Motif 2 (dual HMG-CoA recognition pockets), and Motif 3 (NADP<sup>+</sup> coordination) (<a href="#pone.0330439.g001" class="usa-link">Fig 1D</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330439.g001"><h4 class="obj_head">Fig 1. Conserved domain, motif analysis, and gene structure of the <em>ZjHMGR</em> family members in <em>Ziziphus jujuba</em> var. <em>spinosa.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/4954216d09f9/pone.0330439.g001.jpg" loading="lazy" height="183" width="750" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) The top ten conserved motifs distribution of ZjHMGR proteins; the color boxes represent different conserved motifs, as shown in the scheme on the lower right side of the figure. (B) Predicted the distribution of conserved domains in ZjHMGR proteins. The green and yellow boxes represent the transmembrane domain and HMG-CoA reductase class I domains, respectively. (C) The exon–intron structures of <em>ZjHMGR</em> genes. The utilisation of pink boxes and light blue boxes to denote UTR and CDS, respectively, alongside the representation of introns by thin grey lines, serves to facilitate comprehension of the intricacies of the gene structure. (D) Three conserved motif logos including HMG-CoA binding sites and NADP(H) binding sites. Amino acids are represented by one-letter codes and presented in different colors.</p></figcaption></figure><p>Phylogenetic sequence analysis revealed pronounced evolutionary divergence between ZjHMGR paralogs, with the two N-terminal transmembrane domains at residues 43−64 and residues 84−107 exhibiting similarities (<a href="#pone.0330439.g002" class="usa-link">Fig 2</a>). In contrast, the C-terminal catalytic domain demonstrated exceptional conservation, preserving two canonical HMG-CoA binding pockets and dual NADP(H)-interacting sites (<a href="#pone.0330439.g001" class="usa-link">Fig 1D</a> and <a href="#pone.0330439.g002" class="usa-link">Fig 2</a>), suggesting strong purifying selection pressure to maintain catalytic fidelity. Multiple sequence alignment using DeepTMHMM corroborated dual transmembrane helices in all isoforms, while SignalP-6.0 excluded secretory pathway involvement (<a href="#pone.0330439.g002" class="usa-link">Fig 2</a> and <a href="#pone.0330439.t002" class="usa-link">Table 2</a>). These computational findings, combined with endoplasmic reticulum (ER) membrane localization predictions, support HMGR membrane protein configuration anchored to the ER membrane by two transmembrane movements (<a href="#pone.0330439.g002" class="usa-link">Fig 2</a> and <a href="#pone.0330439.t002" class="usa-link">Table 2</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330439.g002"><h4 class="obj_head">Fig 2. Multiple sequence alignment of the ZjHMGR proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/7f3450b33995/pone.0330439.g002.jpg" loading="lazy" height="863" width="760" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The secondary structural elements predicted with ZjHMGR1 are shown above. Red box, white character represents strict identity; red character represents similarity in a group; blue frame represents similarity across groups. The η symbol refers to 3<sub>10</sub>-helixs; α-helixs and 3<sub>10</sub>-helixs are displayed as solenoids with numbers, respectively. β-strands are displayed as black arrows; strict β-turns are denoted as TT letters. The orange line represents three different transmembrane helices. The blue boxes with letters represent two transmembrane helices, two HMG-CoA binding sites and two NADP(H) binding sites, respectively.</p></figcaption></figure><p>The secondary structure predicted via ZjHMGR1 was shown above the multiple sequence alignments, with twenty α-helices, eleven β-strands, six strict β-turns, and five 3<sub>10</sub>-helixs (<a href="#pone.0330439.g002" class="usa-link">Fig 2</a>). The first 3<sub>10</sub>-helixs is more divergent in the three ZjHMGR proteins. And the third 3<sub>10</sub>-helixs in ZjHMGR3 is different from other two ZjHMGR proteins, but the second, the fourth and the fifth 3<sub>10</sub>-helixs are the same in all three ZjHMGR proteins (<a href="#pone.0330439.g002" class="usa-link">Fig 2</a>). The prediction of the secondary structure of the ZjHMGR protein family (<a href="#pone.0330439.t003" class="usa-link">Table 3</a>) via SOPMA revealed that all ZjHMGR family members consisted of α-helices, β-extended strands, β-turns, and irregular curls. Among them, α-helices (41.96%−46.05%) had the highest percentage and followed by random curl structures (36.03%−38.47%) indicating that the secondary structure of ZjHMGR was mainly composed of α-helices and random curl and exercised the main functions, highlighting their potential functional significance in catalytic mechanisms and substrate interactions.</p>
<section class="tw xbox font-sm" id="pone.0330439.t003"><h4 class="obj_head">Table 3. The secondary structure of ZjHMGR family proteins.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Gene name</th>
<th align="left" rowspan="1" colspan="1">α-helix</th>
<th align="left" rowspan="1" colspan="1">β-strands</th>
<th align="left" rowspan="1" colspan="1">β-turn</th>
<th align="left" rowspan="1" colspan="1">Random coil</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR1</td>
<td align="left" rowspan="1" colspan="1">45.34%</td>
<td align="left" rowspan="1" colspan="1">13.71%</td>
<td align="left" rowspan="1" colspan="1">4.92%</td>
<td align="left" rowspan="1" colspan="1">36.03%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR2</td>
<td align="left" rowspan="1" colspan="1">46.05%</td>
<td align="left" rowspan="1" colspan="1">12.20%</td>
<td align="left" rowspan="1" colspan="1">4.81%</td>
<td align="left" rowspan="1" colspan="1">36.94%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR3</td>
<td align="left" rowspan="1" colspan="1">41.96%</td>
<td align="left" rowspan="1" colspan="1">14.59%</td>
<td align="left" rowspan="1" colspan="1">4.98%</td>
<td align="left" rowspan="1" colspan="1">38.47%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330439.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The tertiary structure-function relationship of proteins constitutes a cornerstone of regulatory mechanism. Based on the highly conserved protein sequences of the ZjHMGR family members, we constructed and mapped 3D protein structural models to distinguish structural divergence patterns across this enzyme family through AlphaFold3 to facilitate a better understanding of ZjHMGR family protein functions. Comparative analysis of predicted models demonstrated striking structural conservation among ZjHMGR isoforms, with all members adopting a tripartite domain organization: a N-domain with small helical structure of N-terminal site mediating membrane association, a L-domain housing large helical structure containing conserved HMG-COA and NADP(H) binding motifs, and a small helical structure with NADP(H) binding motif on catalytic S-domain (<a href="#pone.0330439.g003" class="usa-link">Fig 3</a>). ZjHMGR are highly conserved, extending from primary sequences to tertiary structural features, during the evolutionary process and may perform analogous functions, suggesting strong purifying selection acting on ZjHMGR paralogs during Ziziphus jujuba speciation. These structural conservation features of ZjHMGR may underpin functional redundancy in isoprenoid biosynthesis, potentially serving as evolutionary safeguards to maintain metabolic flux homeostasis.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g003"><h4 class="obj_head">Fig 3. Predicted tertiary structures of ZjHMGR isoforms.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/e475972f539e/pone.0330439.g003.jpg" loading="lazy" height="247" width="784" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ZjHMGR had three domains. N-domain (blue; small helical structure with N-terminal site, membrane anchoring), L-domain (green; ZjHMGR proteins structural center assembly with large helical structure containing HMG-COA and NADP(H) binding motifs), and S-domain (orange; small helical structures with NADP(H) binding motif, auxiliary NADPH coordination). Structural variations in the L-domain (e.g., loop topology, surface charge distribution) may facilitate isoform-specific regulatory mechanisms or protein-protein interactions under stress. Models generated via AlphaFold. Image coloured by rainbow N to C terminus.</p></figcaption></figure></section><section id="sec014"><h3 class="pmc_sec_title">3.3. Phylogenetic relationships, collinearity analysis and selective pressure discovery of hmgr family members in <em>Ziziphus jujuba</em> var. <em>spinosa</em>
</h3>
<p>To elucidate the evolutionary relationships of HMGR in <em>Ziziphus jujuba</em> var. <em>spinosa</em> relative to other plants, we performed maximum likelihood (ML) phylogenetic analysis using protein sequences from eight representative species. The dataset comprised three ZjHMGRs along with orthologs from <em>Arabidopsis thaliana</em> (2 AtHMGRs), <em>Glycine max</em> (7 GmHMGRs), <em>Malus domestica</em> (9 MdHMGRs), <em>Oryza sativa</em> (3 OsHMGRs), <em>Populus trichocarpa</em> (6 PtHMGRs), <em>Vitis vinifera</em> (3 VvHMGRs), and <em>Zea mays</em> (3 ZmHMGRs), totaling 36 HMGR sequences encompassing both monocotyledonous and dicotyledonous species. HMGR proteins were classed into five well-supported clades (I-V) with significant bootstrap values (<a href="#pone.0330439.g004" class="usa-link">Fig 4</a> and <a href="#pone.0330439.s004" class="usa-link">S4 Table</a>). Notably, ZjHMGR1 exhibited closest phylogenetic affinity to VvHMGR3 within clade II, demonstrating the strongest sequence identity. The remaining isoforms ZjHMGR2 and ZjHMGR3 were clustered in clade I with PtHMGR homologs, showing highly conservation. This grouping pattern suggests conserved evolutionary approaches between ZjHMGRs and those from woody perennials in <em>Populus trichocarpa</em> and <em>Vitis vinifera</em>.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g004"><h4 class="obj_head">Fig 4. Evolution analysis of <em>HMGR</em> genes in <em>Ziziphus jujuba</em> var. <em>spinosa</em> (Zj), <em>Arabidopsis thaliana</em> (At), <em>Glycine max</em> (Gm), <em>Oryza sativa</em> (Os), <em>Malus domestica</em> (Md), <em>Populus trichocarpa</em> (Pt), <em>Vitis vinifera</em> (Vv), and <em>Zea mays</em> (Zm).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/afc8b41abe52/pone.0330439.g004.jpg" loading="lazy" height="718" width="718" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The tree was constructed with MEGA 11.0 using the maximum likelihood (ML) method with 1000 bootstrap replications. Detailed Gene ID and protein sequences in Supplementary <a href="#pone.0330439.s004" class="usa-link">S4 Table</a>.</p></figcaption></figure><p>Gene duplication mechanisms, fundamental drivers of genomic and gene family diversification, were systematically investigated in the ZjHMGR family. Phylogenomic reconstruction revealed that all ZjHMGR paralogs originated through either whole-genome duplication (WGD) or segmental duplication events. Two paralog pairs (<em>ZjHMGR1–ZjHMGR2</em> and <em>ZjHMGR1–ZjHMGR3</em>) exhibit significant collinearity, localized to chromosomes 6, 8, and 9 (<a href="#pone.0330439.g005" class="usa-link">Fig 5A</a>). This spatial separation confirms their origin via segmental duplication rather than tandem duplication. To quantify evolutionary constraints, we calculated pairwise non-synonymous (Ka) and synonymous (Ks) substitution rates, deriving purifying selection pressure indices (Ka/Ks ratios). Both paralog pairs (ZjHMGR1–ZjHMGR2 and ZjHMGR1–ZjHMGR3) exhibit Ka/Ks ratios significantly below 1 (0.0565 and 0.0597, respectively; <em>P</em> &lt; 0.001). This unequivocally indicates stringent purifying selection, preserving essential functional domains of HMGR across evolutionary time. Such conservation underscores HMGR’s indispensable role in the mevalonate pathway, where mutations in catalytic residues are likely deleterious (<a href="#pone.0330439.t004" class="usa-link">Table 4</a>). The paralogs arose via segmental duplication at distinct timepoints. ZjHMGR1–ZjHMGR2 diverged 83.84 Mya (Late Cretaceous), ZjHMGR1–ZjHMGR3 diverged 82.96 Mya (<a href="#pone.0330439.t004" class="usa-link">Table 4</a>). The ≈ 0.9-Mya lag between events aligns with independent genomic rearrangements, potentially coinciding with ancestral whole-genome duplication (WGD) episodes common in eudicots.. To elucidate the evolutionary dynamics of the HMGR gene family in <em>Ziziphus jujuba var. spinosa</em> (wild jujube), a comprehensive covariation analysis was conducted across phylogenetically diverse species, including <em>Arabidopsis thaliana</em>, <em>Glycine max</em>, <em>Malus domestica</em>, <em>Oryza sativa</em>, <em>Populus trichocarpa</em>, <em>Vitis vinifera</em>, and <em>Zea mays</em>. The analysis revealed 4, 8, 7, 2, 14, 7, and 1 collinear gene pairs between wild jujube and the aforementioned species, respectively (<a href="#pone.0330439.g005" class="usa-link">Fig 5B</a>). Wild jujube shares the highest number of collinear <em>HMGR</em> pairs (14) with <em>P. trichocarpa,</em> underscoring evolutionary conservation of terpenoid pathway genes in perennial woody taxa. <em>ZjHMGR1</em> displays collinearity with monocots (<em>Oryza sativa</em>: 2 pairs; <em>Zea mays</em>: 1 pair), suggesting its emergence prior to monocot–eudicot divergence (~200 Mya). This positions <em>ZjHMGR1</em> as the ancestral lineage in the <em>ZjHMGR</em> family. Fewer collinear pairs with <em>Arabidopsis thaliana</em> (4) and <em>Glycine max</em> (8) highlight lineage-specific gene family expansions in eudicots. The ancient emergence and remarkable conservation of the <em>ZjHMGR</em> gene family may imply strong association with HMGR’s crucial role in MVA pathway regulation across divergent plant lineages.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g005"><h4 class="obj_head">Fig 5. Collinearity Analysis of <em>ZjHMGR</em> genes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/55adbe8b5951/pone.0330439.g005.jpg" loading="lazy" height="1439" width="797" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Chromosomal distribution and intraspecific covariance analysis of the <em>ZjHMGR</em> genes. Red lines connected to blue boxes represent <em>ZjHMGR</em> gene pairs, and gray lines represent homologous modules in the background of the wild jujube genome. (B) Covariance analysis of HMGR across <em>Ziziphus jujuba</em> var. <em>spinosa</em> (Zj), <em>Arabidopsis thaliana</em> (At), <em>Glycine max</em> (Gm), <em>Oryza sativa</em> (Os), <em>Malus domestica</em> (Md), <em>Populus trichocarpa</em> (Pt), <em>Vitis vinifera</em> (Vv), and <em>Zea mays</em> (Zm) chromosomes. The colored lines highlight the collinear blocks in the genomes of <em>Ziziphus jujuba</em> var. <em>spinosa</em> and the other seven plant species. Various plant species were marked as boxes with different colors.</p></figcaption></figure><section class="tw xbox font-sm" id="pone.0330439.t004"><h4 class="obj_head">Table 4. Detailed information on the Ka, Ks, Ka/Ks ratio and divergence-time in wild jujube.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Homologous gene pair</th>
<th align="left" rowspan="1" colspan="1">Ka</th>
<th align="left" rowspan="1" colspan="1">Ks</th>
<th align="left" rowspan="1" colspan="1">Ka/Ks</th>
<th align="left" rowspan="1" colspan="1">Divergence-Time/Mya<sup>1</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR1-ZjHMGR2</td>
<td align="left" rowspan="1" colspan="1">0.142035</td>
<td align="left" rowspan="1" colspan="1">2.515159</td>
<td align="left" rowspan="1" colspan="1">0.056472</td>
<td align="left" rowspan="1" colspan="1">83.8386333</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ZjHMGR1-ZjHMGR3</td>
<td align="left" rowspan="1" colspan="1">0.148599</td>
<td align="left" rowspan="1" colspan="1">2.488694</td>
<td align="left" rowspan="1" colspan="1">0.059709</td>
<td align="left" rowspan="1" colspan="1">82.9564667</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330439.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t004fn001"><p>Mya<sup>1</sup>: million years, T =Ks/2r ×10<sup>-6</sup>, r =1.5 × 10<sup>−8</sup></p></div></div></section></section><section id="sec015"><h3 class="pmc_sec_title">3.4. Cis-acting element analysis of HMGR gene family in <em>Ziziphus jujuba</em> var. <em>spinosa</em>
</h3>
<p>To investigate the functional potential of the ZjHMGR gene family in wild jujube, a systematic identification of <em>cis</em>-acting regulatory elements within its promoter regions was performed using the PlantCARE database. As illustrated in <a href="#pone.0330439.g006" class="usa-link">Fig 6</a>, four major functional categories of <em>cis</em>-acting elements were identified after excluding ubiquitous TATA-boxes, CAAT-boxes, and one uncharacterized motif, including 35 light-responsive motifs (ACE, AE-box, Box 4, Box II, chs-CMA2a, G-box, GT1-motif, GTGGC-motif, L-box, Sp1, and TCT-motif), 19 hormone-responsive elements (abscisic acid-responsive element ABRE, growth hormone-responsive element TGA-element, gibberellin-responsive element P-box, methyl jasmonate-responsive elements CGTCA-motif and TGACG-motif, and salicylic acid-responsive element TCA-element), 12 stress-associated motifs (anaerobic induction element ARE, low-temperature responsiveness element LTR, defense/stress element TC-rich repeats, and MYBHv1 binding element CCAAT-box), and 4 growth/developmental regulators (meristem expression related CAT-box, and maize alkyd metabolism-regulatory element O2-site) (<a href="#pone.0330439.g006" class="usa-link">Fig 6A</a>). Notably, three ZjHMGR isoforms exhibited conserved enrichment of key regulatory motifs, including light-responsive Box 4, G-box, and GT1-motif; abscisic acid-responsive ABRE; and anaerobic-inducible ARE (<a href="#pone.0330439.g006" class="usa-link">Fig 6B</a>). These findings collectively suggest that ZjHMGR members may function as integrators of photomorphogenic signaling, abscisic acid-mediated stress adaptation, and hypoxia-responsive regulation. Furthermore, the co-occurrence of auxin (TGA-element), jasmonate (CGTCA/TGACG), and salicylic acid (TCA) response motifs across all three isoforms implies potential cross-talk between ZjHMGR activity and phytohormone signaling networks.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g006"><h4 class="obj_head">Fig 6. Prediction of cis-acting elements of promoters of the <em>ZjHMGR</em> gene family.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/828dce46e7f8/pone.0330439.g006.jpg" loading="lazy" height="373" width="742" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PlantCARE was used to analyze the promoter sequences (2000 bp) of three <em>ZjHMGR</em> genes. (A) The numerical values denote the quantity of cis-acting elements of each type. (B)Vertical bars display the positional distribution of the cis-acting elements on the <em>ZjHMGR</em> promoters. In this legend, each color represented different cis-acting elements.</p></figcaption></figure></section><section id="sec016"><h3 class="pmc_sec_title">3.5. Tissue-specific expression of <em>ZjHMGR</em> and analysis of expression patterns upon different stresses and exogenous phytohormone treatments</h3>
<p>To elucidate the functional roles of <em>ZjHMGR</em> gene family members across various tissues and developmental phases in wild jujube, we conducted a comprehensive expression profiling analysis through qRT-PCR. The investigation revealed distinct tissue-specific expression profiles and significant inter-member transcriptional variations among the three ZjHMGR paralogs in root, stem, leaf, floral, and seedling tissues. Notably, comparative expression analysis demonstrated substantial transcriptional divergence between ZjHMGR isoforms. While ZjHMGR2 maintained relatively basal expression levels across all examined tissues, ZjHMGR1 and ZjHMGR3 exhibited comparable transcriptional abundance with significantly elevated expression intensities (<a href="#pone.0330439.g007" class="usa-link">Fig 7A</a>). A particularly striking expression pattern emerged in root tissues, where all three ZjHMGR homologs displayed markedly higher transcript accumulation compared to other organ systems, suggesting potential isoform-specific functional specialization in belowground metabolic processes. Given the presence of phytohormone-responsive (ABA/MeJA) and light-regulatory <em>cis</em>-elements in all ZjHMGR paralog promoters, we systematically investigated the transcriptional dynamics of these isoforms under exogenous abscisic acid (ABA), methyl jasmonate (MeJA), and photoperiod treatments in <em>wild jujube.</em> To account for tissue-specific expression patterns observed in previous analyses (<a href="#pone.0330439.g007" class="usa-link">Fig 7A</a>), experimental materials were standardized using wild jujube seedlings at the 4–6 true leaf developmental stage, with parallel expression profiling conducted in root and leaf tissues through qRT-PCR. Notably, ABA induction triggered coordinated transcriptional responses across the ZjHMGR family members, though with distinct pattern variations. All three paralogs exhibited significant upregulation following ABA treatment, with ZjHMGR2 demonstrating both the most rapid response and the highest magnitude induction. This differential regulation pattern suggests potential functional diversification among paralogs in ABA-mediated stress adaptation mechanisms. Quantitative profiling revealed temporally coordinated induction patterns of ZjHMGR1 in both root and leaf tissues during early ABA exposure (<a href="#pone.0330439.g007" class="usa-link">Fig 7B</a> and <a href="#pone.0330439.g007" class="usa-link">7C</a>). In contrast, ZjHMGR3 displayed tissue-specific regulatory divergence, root tissues maintained progressive transcriptional activation throughout the treatment period, whereas foliar expression peaked at 12 h followed by a significant attenuation, suggesting potential feedback inhibition or post-transcriptional regulation in leaves. MeJA responsiveness uncovered striking disparities between hormone signaling pathways in ZjHMGR members. Root tissues exhibited delayed MeJA responsiveness relative to ABA stimulation, with ZjHMGR2 demonstrating primary response activation at 12 h, while ZjHMGR1 and ZjHMGR3 showed deferred upregulation until 24 h (<a href="#pone.0330439.g007" class="usa-link">Fig 7D</a> and <a href="#pone.0330439.g007" class="usa-link">7E</a>). Notably, ZjHMGR3 achieved maximal induction amplitude despite its delayed activation, implicating isoform-specific roles in sustained JA signaling. This delayed responsiveness in ZjHMGR3 may be correlated with promoter architecture of higher abundance of ABA-responsive <em>cis</em>-elements compared to MeJA-responsive elements across ZjHMGR promoters. The differential response pattern with rapid ABA and delayed MeJA coupled with fine modulation among ZjHMGR paralogs suggest functional diversification in ABA-JA hormone signaling integration.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g007"><h4 class="obj_head">Fig 7. Tissue-specific expression of <em>ZjHMGR</em> and analysis of expression patterns upon different stresses and exogenous hormone treatments.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/37cde183e9a8/pone.0330439.g007.jpg" loading="lazy" height="958" width="755" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) The transcription level of <em>ZjHMGR</em> in different tissues of 5-year-old plants and four- to six-leaf stage seedings assayed by qRT-PCR. (B–E) The transcription level of <em>ZjHMGR</em> in four- to six-leaf stage seedings treated with exogenous ABA or MeJA assayed by qRT-PCR. The levels in roots or leaves of seedlings in the absence or presence of 100 μM ABA or 1 mM MeJA for 0-24 h. (F-G) The transcription level of <em>ZjHMGR</em> in four- to six-leaf stage seedings protected from light for 24 h, and then recovered to light for 12 h or 24 h. The relative gene expression was normalized to the level of the control and was normalized to Zj<em>ACTIN7</em> expression. Data are from three biological replicates (± SD). Statistical differences were shown by different letters or asterisks according to One-Way ANOVA followed by Tukey’s HSD test, *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.01.</p></figcaption></figure><p>Based on the pronounced enrichment of light-responsive <em>cis</em>-elements in <em>ZjHMGR</em> promoters, we detected expression pattern of this gene family through controlled dark adaptation (24 h) and subsequent light recovery assays. In roots, dark-induced transcriptional attenuation manifested differentially across ZjHMGR paralogs, with ZjHMGR1 and ZjHMGR2 exhibiting significant downregulation, while ZjHMGR3 maintained basal expression levels under darkness (<a href="#pone.0330439.g007" class="usa-link">Fig 7F</a> and <a href="#pone.0330439.g007" class="usa-link">7G</a>). This specific sensitivity to light suggests functional stratification, where ZjHMGR1/2 may act as light sensors in subterranean organs. All three paralogs showed light upregulation during dark treatment in leaves. This paradoxical induction correlates with the presence of chloroplasts or ROS bursts. Light restoration elicited transcriptional activation across all tissues, with ZjHMGR2 demonstrating exceptional photosensitivity. The coordinated pattern of ZjHMGR1 and ZjHMGR3 expression suggests complementary roles in photomorphogenic recovery (<a href="#pone.0330439.g007" class="usa-link">Fig 7F</a> and <a href="#pone.0330439.g007" class="usa-link">7G</a>). To minimize bias from a single accession, we conducted identical expression pattern analyses in an additional wild jujube accession (ZH#2). Results demonstrated that ZjHMGR1/2/3 maintained high root expression, while ZjHMGR2 exhibited constitutively low expression in stems, leaves, and flowers. ZjHMGR1/3 displayed ubiquitous expression across tissues—broadly consistent with <a href="#pone.0330439.g007" class="usa-link">Fig 7A</a>. Following hormone and light treatments, ZjHMGR1/2/3 expression profiles in ZH#2 roots and leaves remained essentially identical to initial observations (S2 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). The tripartite regulatory module implies that ZjHMGR1 as a candidate mediating root photomorphogenesis, ZjHMGR3 potentially involved in light responses and ZjHMGR2 serving as the master switch through its hyperresponsive light-responsive elements.</p></section><section id="sec017"><h3 class="pmc_sec_title">3.6. Inhibitive effects of saline-alkaline on seedlings growth of wild jujube</h3>
<p>The phytohormone abscisic acid (ABA) serves as a central regulator of ionic homeostasis and osmotic adjustment in plant salinity tolerance networks, primarily through orchestrated activation of SOS pathway components and stress-responsive transcription factors (bZIP/NAC families) [<a href="#pone.0330439.ref009" class="usa-link" aria-describedby="pone.0330439.ref009">9</a>,<a href="#pone.0330439.ref051" class="usa-link" aria-describedby="pone.0330439.ref051">51</a>]. The robust transcriptional responsiveness of ZjHMGR paralogs to ABA induction strongly implicates this gene family in mediating wild jujube’s adaptive reprogramming under saline-alkaline stress conditions (<a href="#pone.0330439.g007" class="usa-link">Fig 7B</a> and <a href="#pone.0330439.g007" class="usa-link">7C</a>). To mechanistically dissect ZjHMGR functionality in saline-alkaline stress adaptation, we recorded the seed germination rates of two varieties of wild jujube (ZH#1 and ZH#2) and established a controlled salinity-alkalinity composite stress system using hydroponically cultivated <em>Z. jujuba</em> var. <em>spinosa</em> seedlings (ZH#1). Germination assays of ZH#1 and ZH#2 sour jujube seeds under 0–200 mM NaCl gradients revealed marked dose dependency. Relative to controls, 50 mM NaCl exerted no significant impact on germination rates. However, 100 mM NaCl reduced germination to 38%, while 200 mM NaCl further suppressed germination to approximately 25% after 15 days (S3 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a> and <a href="#pone.0330439.s005" class="usa-link">S5 Table</a>). At early phase (0–12 h post-treatment), Absence of visible morphological changes despite massive transcriptomic remodeling, with 12422 genes exhibiting ≥1-fold expression shifts (S4 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>).</p>
<p>Integrated analysis of transcriptomic dynamics and phenotypic transitions enabled systematic classification of the composite stress response into four distinct adaptive satges: (1) response stag<strong>e</strong> (0–12 h), (2) adaptation stage (12 h-2 d), (3) resistance regulation stage (2–4 d), and (4) the death stage (&gt;4 d) (<a href="#pone.0330439.g008" class="usa-link">Fig 8A</a>). This multi-phase progression reflects an evolutionary-conserved adaption mechanism balancing energy allocation between stress perception and survival strategies. At response stage (0–12 h), the primary stress recognition window exhibited massive transcriptomic turbulence preceding visible symptomatology (S4 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Key biomarkers included photosynthetic collapse with significant reduction in chlorophyll a and ROS scavenging enzyme activation in superoxide dismutase (SOD) at this stage (<a href="#pone.0330439.g008" class="usa-link">Fig 8D</a> and <a href="#pone.0330439.g008" class="usa-link">8F</a>). The salinity-alkalinity adaptation phase manifested between 12 h and 2 d post-treatment initiation (<a href="#pone.0330439.g008" class="usa-link">Fig 8A</a>), characterized by sustained transcriptional activation of stress-responsive genetic elements (S4 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). During this critical period, physiological analyses revealed three hallmark responses: (1) pronounced accumulation of osmolytes including proline and soluble sugars (<a href="#pone.0330439.g008" class="usa-link">Fig 8B</a> and <a href="#pone.0330439.g008" class="usa-link">8C</a>); (2) progressive chlorophyll degradation (<a href="#pone.0330439.g008" class="usa-link">Fig 8D</a>) and main root length shortened (S5 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>); and (3) transient activation of antioxidant defense systems. Notably, catalase (CAT), superoxide dismutase (SOD), and peroxidase (POD) activities exhibited stress-induced upregulation, peaking at 24 h (about 2.8-, 3.1-, and 2.1-fold increases, respectively) before subsequent attenuation (<a href="#pone.0330439.g008" class="usa-link">Fig 8E</a>, <a href="#pone.0330439.g008" class="usa-link">8F</a> and <a href="#pone.0330439.g008" class="usa-link">8G</a>), indicative of dynamic redox homeostasis regulation. The resistance regulation phase (2–4 d) represented a pivotal transition in stress acclimatization. While phenotypic observations revealed partial seedling wilting (<a href="#pone.0330439.g008" class="usa-link">Fig 8A</a>), molecular profiling demonstrated persistent upregulation of 5663 salinity-responsive genes (S4 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>), suggesting active transcriptional reprogramming in <em>Ziziphus jujuba</em> var. <em>spinosa</em>. This phase likely constitutes the functional window for salinity resistance gene networks in wild jujube, as evidenced by maintenance of delayed necrosis progression. Post 8-day exposure, seedlings entered the terminal stress phase marked by catastrophic physiological collapse: severe hydric deficit, severe chlorophyll depletion (<a href="#pone.0330439.g008" class="usa-link">Fig 8A</a> and <a href="#pone.0330439.g008" class="usa-link">8D</a>), and massive leaf abscission. Meanwhile, osmolyte concentrations reached maximal levels (<a href="#pone.0330439.g008" class="usa-link">Fig 8B</a> and <a href="#pone.0330439.g008" class="usa-link">8C</a>), and antioxidant enzyme activities plummeted to low levels (<a href="#pone.0330439.g008" class="usa-link">Fig 8E</a>, <a href="#pone.0330439.g008" class="usa-link">8F</a> and <a href="#pone.0330439.g008" class="usa-link">8G</a>). Concurrent transcriptional profiling revealed aberrant gene activation patterns (S4 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>), suggesting that stress-induced senescence and death genes were highly upregulated. This phase was thus characterised as a state of death. These findings demonstrate that prolonged saline-alkaline exposure induces systemic dysfunction in wild jujube seedlings, disrupting the growth and cellular homeostasis through multi-level transcriptional and metabolic dysregulation.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g008"><h4 class="obj_head">Fig 8. Analysis of phenotypic and physiological change patterns in <em>Ziziphus jujuba</em> var. <em>spinosa</em> seedlings under sustained salinity-alkalinity stress.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/e7342c0cec5c/pone.0330439.g008.jpg" loading="lazy" height="1078" width="776" alt="Fig 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Phenotypes of wild jujube seedlings treated continuously with a saline-alkaline mixture of six parts per thousand. (B-D) The contents of proline (B), soluble sugar (C), and chlorophyll <strong>(D)</strong> of wild jujube seedlings treated continuously with a saline-alkaline mixture of six parts per thousand for 0-8 d. (E-G) The enzyme activity of CAT (E), SOD (F), and POD <strong>(G)</strong> of wild jujube seedlings treated continuously with a saline-alkaline mixture of six parts per thousand for 0-8 d. The saline-alkaline mixture of six parts per thousand consists of NaCl/Na<sub>2</sub>SO4/NaHCO<sub>3</sub>/Na<sub>2</sub>CO<sub>3</sub> in the concentration ratio 1:1:1:1 M. Data are from three biological replicates (± SD). Statistical differences were shown by different letters according to One-Way ANOVA followed by Tukey’s HSD test, *<em>P</em> &lt; 0.05, n = 10.</p></figcaption></figure></section><section id="sec018"><h3 class="pmc_sec_title">3.7. Transcriptional reprogramming of MVA pathway genes under saline-alkali stress in wild jujube adaptation</h3>
<p>To elucidate the regulatory interplay between <em>ZjHMGR</em> paralogs and saline-alkaline stress adaptation, we conducted temporal expression profiling of the <em>ZjHMGR</em> gene family across saline-alkali treatment gradients. qRT-PCR analysis revealed that all ZjHMGR isoforms exhibited significant stress induction (<a href="#pone.0330439.g009" class="usa-link">Fig 9A</a>), with ZjHMGR2/3 displaying coordinated induction. These paralogs (ZjHMGR2 and ZjHMGR3) demonstrated rapid transcriptional activation, achieving peak expression at 6 h post-treatment followed by progressive attenuation (<a href="#pone.0330439.g009" class="usa-link">Fig 9A</a>). Intriguingly, sustained stress exposure (&gt;2 d) triggered a biphasic expression pattern, where transcript levels remained elevated relative to control despite post-peak decline, suggesting persistent metabolic demand for isoprenoid precursors under chronic stress. Intriguingly, comparative expression revealed functional divergence among <em>ZjHMGR</em> paralogs. While <em>ZjHMGR2/3</em> exhibited rapid stress responsiveness (6 h peak), ZjHMGR1 displayed delayed transcriptional activation, initiating significant upregulation only after 2 d sustained saline-alkali exposure (<a href="#pone.0330439.g009" class="usa-link">Fig 9A</a>). This temporal specialization suggests stage-specific roles in ZjHMGR paralogs. ZjHMGR2/3 likely mediate acute saline-alkali adjustment, whereas ZjHMGR1 may orchestrate chronic stress adaptation. Mechanistically, the lack of co-expression correlation between ZjHMGR1 and ZjHMGR2/3 implies distinct regulatory architectures, potentially involving lineage-specific cis-regulatory elements or differential transcription factor binding. Collectively, these findings position the ZjHMGR family as central regulatory nodes in wild jujube’s salinity resistance network. The observed paralog-specific induction patterns and ZjHMGR1 independent regulatory decoupling exemplify evolutionary innovation in stress-responsive metabolic channeling.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g009"><h4 class="obj_head">Fig 9. Expression analysis of MVA pathways-related genes in wild jujube seedings upon salinity-alkalinity treatment.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/d549662074d5/pone.0330439.g009.jpg" loading="lazy" height="542" width="716" alt="Fig 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A-B) The transcription levels of <em>ZjHMGR</em>
<strong>(A)</strong> and other MVA pathways-related genes <strong>(B)</strong> in four- to six-leaf stage seedings of wild jujube treated with a saline-alkaline mixture of six parts per thousand for 0-4 d. The relative gene expression was normalized to the level of the control and was normalized to <em>ZjACTIN7</em> expression. Data are from three biological replicates (± SD). Statistical differences were shown by different letters or numbers according to One-way ANOVA followed by Tukey’s HSD test, *<em>P</em> &lt; 0.05.</p></figcaption></figure><p>As the key rate-limiting enzymatic step governing metabolic flux through the MVA pathway in <em>Ziziphus jujuba</em> var. <em>spinosa</em>, the ZjHMGR gene family may emerge as a critical regulatory node coordinating triterpenoid saponin biosynthesis and salinity adaptation. To dissect whether this evolutionarily conserved pathway exhibits stress-responsive rewiring, we performed time-resolved transcription profiling of core MVA pathway upstream enzymes – acetyl-CoA acyltransferases (<em>AACTs</em>) and 3-Hydroxy-3-methylglutaryl coenzyme A synthase (<em>HMGS</em>), and downstream of HMGR in the MVA pathway, squalene synthase (<em>SQS</em>) and geranylpyrophosphate synthetase (<em>GPPS</em>). The initial enzymatic component of MVA biosynthetic pathway, AACT, positioned upstream of HMGR, demonstrated immediate transcriptional activation under saline-alkaline stress conditions, catalyzing the condensation of acetyl-CoA to acetoacetyl-CoA (<a href="#pone.0330439.g009" class="usa-link">Fig 9B</a>). In contrast, HMGS functioning upstream in the MVA pathway, exhibited delayed upregulation with attenuated expression magnitude relative to AACT under identical stress condition. Our analysis extended to critical downstream enzymatic targets – SQS and GPPS – both essential for pivotal triterpenoid saponin precursor. Quantitative analysis revealed distinct temporal expression patterns, SQS displayed rapid transcriptional induction within 12 h post-stress initiation, whereas GPPS mirrored HMGS’s delayed response profile, achieving significant upregulation only after sustained 24 h saline exposure (<a href="#pone.0330439.g009" class="usa-link">Fig 9B</a>). The close relationship between activation of the MVA pathway from AACT to SQS/GPPS and stress response suggests that MVA-derived specialized metabolites, especially triterpenoid saponins, play the role of monitors during salinity stress tolerance in <em>Ziziphus jujuba</em> var. <em>spinosa</em>. These coordinated transcriptional dynamics suggest that HMGR serving as the central regulatory node of MVA pathway components in <em>Ziziphus jujuba</em> var. <em>spinosa</em> exhibit differential sensitivity to saline-alkaline stressors. In ZH#2 wild jujube seedings, MVA pathway-associated genes also exhibited time-dependent regulation under NaCl exposure (S6 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). The observed transcriptional reprogramming provides evidence linking triterpenoid saponin biosynthesis with plant adaptation to saline-alkaline edaphic conditions by ZjHMGR.</p></section><section id="sec019"><h3 class="pmc_sec_title">3.8. Heterologous Overexpression of <em>ZjHMGR</em> Confers Enhanced Salinity Tolerance in <em>Arabidopsis</em>
</h3>
<p>Functionally validating ZjHMGR as a central regulator of the MVA pathway under saline-alkaline stress, we generated Arabidopsis lines overexpressing <em>ZjHMGR1</em>, <em>ZjHMGR2</em>, or <em>ZjHMGR3</em> (S7 and S8 Figs in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Primary root elongation assays revealed that while transgenic seedlings exhibited root lengths comparable to wild-type (Col-0) under control conditions, they displayed significantly enhanced tolerance under NaCl stress. Following 6-day exposure to 100 mM or 150 mM NaCl, root elongation in Col-0 was drastically attenuated. In contrast, all ZjHMGR-overexpressing lines maintained significantly longer root (<a href="#pone.0330439.g010" class="usa-link">Fig 10</a>). Quantitative analysis showed a root length inhibition rate of 50.16% in Col-0 under 100 mM NaCl, markedly higher than the inhibition observed in ZjHMGR1 (16.32–34.77%), ZjHMGR2 (37.58–37.72%), and ZjHMGR3 (29.97–42.16%) transformants (S9 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). A similar protective effect was evident at 150 mM NaCl, demonstrating a concentration-dependent response. Notably, at the severe stress level of 200 mM NaCl, <em>ZjHMGR3</em>-overexpressing lines uniquely sustained significantly longer roots compared to Col-0 and the other transgenics, where root lengths were comparable to wild-type, likely due to overwhelming ionic toxicity. Critically, <em>ZjHMGR</em> overexpression also enhanced shoot resilience. Transgenic lines exhibited increased biomass under non-stress conditions and maintained significantly higher fresh weight than Col-0 across all NaCl concentrations (100–200 mM; S10 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Collectively, these results demonstrate that <em>ZjHMGR</em> acts as a genetic determinant of salinity adaptation, conferring significant enhancement of whole-plant stress tolerance in <em>Arabidopsis</em>.</p>
<figure class="fig xbox font-sm" id="pone.0330439.g010"><h4 class="obj_head">Fig 10. <em>A. thaliana</em> primary root growth during a 6-day period for the Col-0, <em>ZjHMGR1, ZjHMGR2</em> and <em>ZjHMGR3.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367118_pone.0330439.g010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c3db/12367118/00dbc248d82f/pone.0330439.g010.jpg" loading="lazy" height="673" width="740" alt="Fig 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330439.g010/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Growth phenotypes of <em>A. thaliana</em> grown for 6 days in a vertically placed 1/2 MS medium. <strong>(B)</strong> Primary root length between Col-0 and <em>ZjHMGRs</em> transgenic lines. Data are from biological replicates (± SD). Statistical differences were shown by different letters or numbers according to One-Way ANOVA followed by Tukey’s HSD test, *<em>P</em> &lt; 0.05, n ≥ 40 roots per overexpression strain.</p></figcaption></figure></section></section><section id="sec020"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>Plants are exposed to numerous environmental changes throughout their life cycle, and the harshness of the survival environment makes cells unusually sensitive to stress signals and stimuli. Plant cells induce stress under a variety of adversity conditions such as heat, salinity, and drought. Salinity stress has become a type of abiotic stress that is in crisis globally and has a significant impact on plant survival. Soil salinization constitutes a pressing global ecological challenge that severely compromises agricultural sustainability by impairing plant growth trajectories, diminishing land productivity, and threatening food security networks. Nevertheless, China confronts multifaceted germplasm limitations in saline-alkaline land rehabilitation, characterized by three interdependent constraints: extensive spatial distribution of affected areas, paucity of halophytic cultivars with agronomic value, and suboptimal salt-tolerance thresholds in conventional crops. Recent climatic anomalies have precipitated a critical ecological transition in Chinese saline-alkaline ecosystems, with soil salt contents escalating to 6 ‰. This salt toxicity threshold now exceeds the adaptive capacity of most of commercial cultivars under controlled salinity trials. The strategic development of stress-resilient cultivars through advanced breeding technologies and germplasm innovation has emerged as a pivotal approach for maximizing saline soil productivity.</p>
<p><em>Ziziphus jujuba</em> var. <em>spinosa</em> (wild jujube) emerges as a multifunctional phytoresource with potential encompassing pharmacotherapeutic, nutraceutical, ecological resilience, and agroeconomic dimensions [<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>,<a href="#pone.0330439.ref006" class="usa-link" aria-describedby="pone.0330439.ref006">6</a>]. Kernels of wild jujube are enriched with bioactive specialized metabolites, notably jujuboside A/B (triterpenoid saponins) and flavonoids, which mechanistically demonstrate superior anxiolytic efficacy, establishing this botanical as a first-line phytotherapeutic agent for neuropsychiatric disorders in traditional Chinese medical treatment [<a href="#pone.0330439.ref001" class="usa-link" aria-describedby="pone.0330439.ref001">1</a>,<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>,<a href="#pone.0330439.ref052" class="usa-link" aria-describedby="pone.0330439.ref052">52</a>]. This xerophytic perennial exhibits exceptional edaphic adaptability, demonstrating halophytic tolerance and drought resistance, but the molecular determinants underlying its extremophytic adaptation remain enigmatic. Jujuboside A has been identified as the principal bioactive constituent in <em>Ziziphus jujuba</em> var. <em>spinosa</em> seeds [<a href="#pone.0330439.ref001" class="usa-link" aria-describedby="pone.0330439.ref001">1</a>,<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>]. However, the biosynthetic confinement of these triterpenoid saponins presents three critical production bottlenecks, limited biomass yield exacerbated by climatic volatility, structural complexity and high price. Within the paradigm of low-abundance, high-value specialized metabolite engineering, this investigation proposes a synergistic strategy integrating mechanistic studies on the nexus between triterpene saponin biosynthesis in sour jujube and its halophytic adaptation. The methodology encompasses: (1) genome-wide identification and functional validation of rate-limiting enzymes within the mevalonate pathway (triterpenoid backbone biosynthesis) via multi-omics triangulation; (2) structure-activity relationship analyses coupled with molecular simulations to anticipate catalytic mechanisms and stress-responsive regulatory networks; and (3) establishment of high-fidelity homologous/heterologous expression chassis systems, thereby enabling secondary metabolism and adaptation to stress conditions. Such fundamental investigations will provide critical molecular insights for technological innovation in phytochemical production, efficient utilization of saline resources and ecological agriculture development.</p>
<section id="sec021"><h3 class="pmc_sec_title">4.1. Structural and Functional Conservation of HMGR: A Rate-Limiting Enzyme Governing Mevalonate Pathway Evolution in <em>Ziziphus jujuba</em> var. <em>spinosa</em>
</h3>
<p>Jujuboside triterpenoids, classified as dammarane-type tetracyclic compounds, are biosynthesized through the mevalonate (MVA) pathway, with 3-hydroxy-3-methylglutaryl-CoA reductase (HMGR) serving as the pivotal rate-limiting enzyme [<a href="#pone.0330439.ref004" class="usa-link" aria-describedby="pone.0330439.ref004">4</a>,<a href="#pone.0330439.ref025" class="usa-link" aria-describedby="pone.0330439.ref025">25</a>]. Contemporary molecular investigations demonstrate that HMGR gene characterization and functional elucidation significantly enhance triterpenoid saponin and terpenoid secondary metabolite yields in medicinal taxa including <em>Salvia miltiorrhiza</em> [<a href="#pone.0330439.ref029" class="usa-link" aria-describedby="pone.0330439.ref029">29</a>], <em>Panax ginseng</em> [<a href="#pone.0330439.ref028" class="usa-link" aria-describedby="pone.0330439.ref028">28</a>], <em>Tripterygium wilfordii</em> [<a href="#pone.0330439.ref053" class="usa-link" aria-describedby="pone.0330439.ref053">53</a>], and <em>Artemisia annua</em> [<a href="#pone.0330439.ref054" class="usa-link" aria-describedby="pone.0330439.ref054">54</a>]. Coordinated upregulation of mevalonate (MVA) pathway genes concurrently validates HMGR’s phylogenetically conserved regulatory mechanism in triterpenoid biosynthesis across divergent lineages. Despite <em>Ziziphus jujuba</em> var. <em>spinosa</em>’s medicinal prominence, its HMGR-mediated triterpenoid regulation remains underexplored. Systematic characterization of <em>ZjHMGR</em> isoforms, including structural phylogenetics and interactome mapping, will decode evolutionary function on metabolic networks for precision engineering of jujuboside biosynthesis.</p>
<p>In this paper, three members of the <em>ZjHMGR</em> gene family were identified in the wild jujube genome. To resolve potential genome annotation discrepancies, we performed orthogonal validation through integration of strand-specific RNA-seq data with molecular cloning followed by Sanger sequencing. This multimodal approach uncovered a critical misannotation in the original genome assembly, particularly in the exon-intron architecture of ZjHMGR1. The revised gene models, featuring corrected exon boundaries and phase consistency, have been deposited in the Supplementary Materials (<a href="#pone.0330439.s002" class="usa-link">S1 Table</a>).<em>ZjHMGR1</em> and <em>ZjHMGR2</em>, which have similar gene structures, have similar protein physicochemical properties, whereas <em>ZjHMGR3</em>, which has lost one intron, resulting in a compact genomic architecture despite encoding the longest CDS (1,812 bp), has a large difference in protein physicochemical properties (<a href="#pone.0330439.t001" class="usa-link">Tables 1</a> and <a href="#pone.0330439.t002" class="usa-link">2</a>).ZjHMGR3’s intron loss correlates with distinct biophysical properties—acidic pI, lower instability index, and reduced α-helix content—suggesting structural stabilization (<a href="#pone.0330439.t002" class="usa-link">Tables 2</a> and <a href="#pone.0330439.t003" class="usa-link">3</a>). This may facilitate rapid transcriptional induction under saline stress (Fig. 9A), positioning ZjHMGR3 as a primary responder. All isoforms retained two catalytic signatures, HMG-CoA-binding motifs (EMPVGYVQIP, TTEGCLVA) and NADP(H)-binding domains (DAMGMNM, GTVGGGGT), confirming structural conservation (<a href="#pone.0330439.g001" class="usa-link">Fig 1</a> and <a href="#pone.0330439.g002" class="usa-link">2</a>). AlphaFold3 predictions (<a href="#pone.0330439.g003" class="usa-link">Fig 3</a>) confirm all isoforms retain the conserved tripartite domain (N-terminal anchor, catalytic L-domain, S-domain). However, ZjHMGR3’s atypical C-terminal amphipathic helix may alter protein-protein interactions or subcellular dynamics. ZjHMGR protein structure and interspecific evolutionary relationships are highly conserved (<a href="#pone.0330439.g003" class="usa-link">Fig 3</a> and <a href="#pone.0330439.g004" class="usa-link">4</a>). Genome-wide covariate analysis detected two segmentally duplicated <em>ZjHMGR</em> pairs localized to distinct chromosomes (<a href="#pone.0330439.g005" class="usa-link">Fig 5A</a>). The ancient origin (≈83 Mya) and persistent purifying selection highlight HMGR’s evolutionary stability as a rate-limiting enzyme in isoprenoid biosynthesis (<a href="#pone.0330439.t004" class="usa-link">Table 4</a>). The duplication chronology further positions ZjHMGR1 as the ancestral lineage predating monocot–eudicot divergence, consistent with its collinearity to monocot HMGRs (<a href="#pone.0330439.g005" class="usa-link">Fig 5B</a>). These dynamics exemplify how essential metabolic genes maintain functional fidelity despite genomic upheaval, ensuring terpenoid-mediated stress adaptation in extremophytes like <em>Z. jujuba</em> var. <em>spinosa</em>. Despite high sequence identity (&gt;90% in catalytic domains), the differential intron architecture of ZjHMGR3 (loss of one intron; <a href="#pone.0330439.t001" class="usa-link">Table 1</a>) suggests subfunctionalization. However, the minimal Ka/Ks values reject neofunctionalization, implying that divergent expression patterns (e.g., tissue specificity; <a href="#pone.0330439.g007" class="usa-link">Fig 7</a>) arose via cis-regulatory evolution rather than coding-sequence changes. Thus, intron loss likely fine-tunes expression kinetics (e.g., rapid stress induction) rather than enzymatic function. Conserved synteny with stress-adapted species (<em>P. trichocarpa</em>, <em>V. vinifera</em>) implies selective maintenance of HMGR-mediated isoprenoid biosynthesis for abiotic stress adaptation in perennial woody plants.</p>
<p>Cis-regulatory element architecture in promoter regions critically governs spatiotemporal gene regulation. Photoresponsive HMGR expression modulation is evolutionarily conserved, evidenced by dark-inducible <em>HMGR1</em> upregulation in <em>Arabidopsis thaliana</em> immature leaves and light-repressed paclitaxel biosynthesis in fungal systems [<a href="#pone.0330439.ref055" class="usa-link" aria-describedby="pone.0330439.ref055">55</a>,<a href="#pone.0330439.ref056" class="usa-link" aria-describedby="pone.0330439.ref056">56</a>]. Promoter analysis of <em>Ziziphus jujuba</em> var. <em>spinosa</em> HMGR isoforms revealed abundant light-responsive cis-elements (<a href="#pone.0330439.g006" class="usa-link">Fig 6</a>), corroborated by qRT-PCR demonstrating photoperiod-dependent regulation: leaf-specific induction versus root suppression (<a href="#pone.0330439.g007" class="usa-link">Fig 7</a>). This tissue-divergent expression pattern suggests a novel regulatory paradigm where light modulates HMGR-mediated triterpenoid biosynthesis through isoform-specific transcriptional control, potentially driving metabolic regulation in medicinal plants. Promoter architecture analysis revealed <em>ZjHMGR</em> isoforms harbor enriched cis-regulatory modules, including phytohormone-responsive elements (ABRE, CGTCA/TGACG motifs), defense/stress-related motifs (TC-rich repeats, MYBHv1, CCAAT-box), and develomental indicators, suggesting HMGR’s integration into ABA-light-oxygen signaling networks (<a href="#pone.0330439.g006" class="usa-link">Fig 6</a>). Acute <em>ZjHMGR</em> responsiveness, with pronounced transcriptional activation observed following ABA, MeJA, and salinity treatments were analyzed by qRT-PCR (<a href="#pone.0330439.g007" class="usa-link">Fig 7</a> and S2 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Specifically, ZjHMGR2 may serve as a primary responder to acute stress signals via its rapid ABA activation, and ZjHMGR3 potentially mediates prolonged adaptive responses through high-level induction. ZjHMGR3 displays unique spatiotemporal regulation—e.g., sustained ABA induction in roots but transient leaf response (<a href="#pone.0330439.g007" class="usa-link">Fig 7B</a> and <a href="#pone.0330439.g007" class="usa-link">7C</a>). Its promoter harbors fewer MeJA-responsive elements, correlating with delayed MeJA response (24 h; <a href="#pone.0330439.g007" class="usa-link">Fig 7D</a> and <a href="#pone.0330439.g007" class="usa-link">7E</a>). Given HMGR’s rate-limiting role in isoprenoid biosynthesis and ABA’s centrality in tripartite stress (salt/drought/osmotic) adaptation, we propose an evolutionary model where ZjHMGR isoforms functionally bridge specialized metabolism with systemic acquired resilience through terpenoid-hormone crosstalk. This multiple induction pattern suggests a complicated regulatory mechanism coordinating HMGR-mediated isoprenoid biosynthesis with environmental adaptation and stress resilience.</p></section><section id="sec022"><h3 class="pmc_sec_title">4.2. Elucidating the Regulatory Role of HMGR in Salinity Stress Adaptation of <em>Ziziphus jujuba</em> var. <em>spinosa</em>
</h3>
<p><em>Ziziphus jujuba</em> var. <em>spinosa</em> exhibits sustained growth vigor in soils with high salinity, establishing its pioneer species status for phytoremediation in coastal salt marshes and secondary salinized inland ecosystems through physiological homeostasis regulation [<a href="#pone.0330439.ref057" class="usa-link" aria-describedby="pone.0330439.ref057">57</a>]. Under saline-alkaline stress, this adaptive mechanism not only mitigates ionic toxicity but also enables osmotic adjustment through compatible solute biosynthesis. Critical knowledge gaps persist regarding the temporal dynamics of stress-responsive phytohormones (e.g., ABA, JA) and the transcriptional regulatory cascades governing HMGR-mediated isoprenoid biosynthesis activation in this species. Integrative phenotyping, physiological profiling, and transcriptomic dissection revealed four sequential stress-response phases in <em>Ziziphus jujuba var. spinosa</em> seedlings under combined salinity-alkalinity stress: (1) response stag<strong>e</strong>, (2) adaptation stage, (3) resistance regulation stage, and (4) the death stage. Stage-specific phenotypic signatures and transcriptional reprogramming patterns were systematically characterized, with seedlings at the 4–6 true leaf stage demonstrating remarkable survival capacity (≥8 days) under 6‰ saline-alkaline co-stress (<a href="#pone.0330439.g008" class="usa-link">Fig 8</a>, S4 and S5 Figs in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Combined salinity-alkalinity stress triggered multilevel physiological disruptions in wild jujube seedlings, manifesting as growth inhibition, reduced chlorophyll content, and osmotic accumulation. The combined salinity-alkalinity stress elicited a dual-phase osmotic response, marked by progressive proline accrual and soluble sugar elevation, consistent with osmolyte biosynthesis activation (<a href="#pone.0330439.g008" class="usa-link">Fig 8B</a> and <a href="#pone.0330439.g008" class="usa-link">8C</a>). Paradoxically, this co-stress triggered temporal dysregulation of ROS homeostasis, while superoxide dismutase (SOD), peroxidase (POD), and catalase (CAT) activities exhibited transient enhancement (peaking at 24 h), systemic antioxidant capacity declined precipitously beyond this threshold, culminating in a redox collapse (<a href="#pone.0330439.g008" class="usa-link">Fig 8E</a>,<a href="#pone.0330439.g008" class="usa-link">8F</a> and <a href="#pone.0330439.g008" class="usa-link">8G</a>).As the central regulatory hub of plant stress adaptation, the endoplasmic reticulum (ER) orchestrates sophisticated cellular transcriptional reprogramming through activation of the unfolded protein folding and enhancement of co-translational processing, dynamic membrane lipid trafficking and remodeling [<a href="#pone.0330439.ref007" class="usa-link" aria-describedby="pone.0330439.ref007">7</a>]. This organelle integrates environmental sensing through phytohormone-mediated crosstalk with plasma membrane-localized receptors, triggering stress signaling cascades that modulate metabolic flux redistribution to maintain cell homeostasis [<a href="#pone.0330439.ref051" class="usa-link" aria-describedby="pone.0330439.ref051">51</a>].</p>
<p>Abscisic acid (ABA) exerts central regulatory control over plant salinity-alkalinity adaptation through coordinated phytohormonal signaling. ABA accumulation triggers an ABA-dependent signaling cascade that (1) activates stress-responsive transcriptional networks (e.g., <em>RD29B</em>, <em>ABI5</em>), (2) induces stomatal closure via ionic modulation, and (3) enhances osmoprotectant biosynthesis, collectively mitigating ionic imbalance under co-stress conditions. Strikingly, the <em>ZjHMGR</em> gene family exhibited ABA hypersensitization (induction within 6 h post-treatment; <a href="#pone.0330439.g007" class="usa-link">Fig 7B</a> and <a href="#pone.0330439.g007" class="usa-link">7C</a>), positioning it as an early signaling hub in wild jujube’s salinity tolerance. Tissue-specific profiling revealed constitutive expression of <em>ZjHMGR1</em> and <em>ZjHMGR3</em> across organs (<a href="#pone.0330439.g007" class="usa-link">Fig 7A</a>), with <em>ZjHMGR3</em>’s intronless architecture potentially enhancing transcriptional efficiency under acute stress (faster induction kinetics vs intron-containing paralogs; <a href="#pone.0330439.g009" class="usa-link">Fig 9A</a> and S6A Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>).Crucially, salinity stress coordinately upregulated the mevalonate (MVA) pathway (<a href="#pone.0330439.g009" class="usa-link">Fig 9B</a> and S6B Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>), demonstrating dual potential functionality in synthesizing triterpenoid saponins (key osmoprotectants) and responding to salinity through intermediate metabolites modulation or phytohormone-mediated crosstalk. This metabolic rewiring suggests evolutionary selection pressure favoring MVA pathway plasticity in halophytic <em>Ziziphus jujuba</em> var. <em>spinosa</em>. Targeted screening of high-saponin germplasm with <em>ZjHMGRs</em> allelic variants may thus accelerate breeding of salinity-resilient cultivars.</p>
<p>Phytohormones are central regulators of plant developmental processes and environmental stress adaptation. These endogenous signaling molecules are synthesised primarily by endogenous metabolic pathways. The ABA-, JA- and GAs-mediated phytohormone pathways directly regulate the transcriptional network for the synthesis of medicinal plant triterpenoids [<a href="#pone.0330439.ref058" class="usa-link" aria-describedby="pone.0330439.ref058">58</a>–<a href="#pone.0330439.ref062" class="usa-link" aria-describedby="pone.0330439.ref062">62</a>]. This transcriptional activation may be mediated by enhanced interactions with specific transcription factors, in particular ABRE, MYB and MYC2, ultimately promoting triterpenoid accumulation. The ZjHMGRs’ promoters contain several hormone-responsive cis-regulatory elements. The use of exogenous hormones during ZjHMGR-mediated regulation of triterpenoid biosynthesis may disrupt the homeostasis of the endogenous phytohormone network, thereby altering plant metabolic physiology. However, the potential functions of HMGR, either as a regulatory link between phytohormone signaling and terpenoid accumulation or as a quantitative switch, remain unresolved due to the complexity of hormonal interactions, the different expression patterns of terpenoid synthase genes and the dynamic interactions between transcription factors and the promoter in this multi-faceted regulatory process.</p>
<p>To functionally characterize the <em>ZjHMGR</em> gene family in salinity adaptation, we engineered three binary expression vectors (<em>pROKII-35S::ZjHMGR1/2/3-eGFP</em>) (S7 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>) and generated stable transgenic <em>Arabidopsis</em> lines through <em>Agrobacterium</em>-mediated floral dip transformation (S8A Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Antibiotic-based selection yielded a T1 transgenic population (HMGR1: n = 9; HMGR2: n = 3; HMGR3: n = 21, independent lines per construct) for subsequent functional investigations(S8B Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Upon establishing homozygous T3 transgenic lines, qRT-PCR confirmed effective ZjHMGR overexpression, with rigorous validation ensuring reproducibility across replicates (S8C Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Based on the robust phenotypic validation in heterologous systems, we demonstrate that <em>ZjHMGR1/2/3</em> overexpression confers enhanced salinity tolerance in <em>Arabidopsis</em>, significantly mitigating NaCl-induced root growth inhibition (16–42% vs. 50% in WT at 100 mM NaCl) and preserving shoot biomass (<a href="#pone.0330439.g010" class="usa-link">Fig 10</a>, S9 and S10 Fig in <a href="#pone.0330439.s001" class="usa-link">S1 File</a>). Crucially, <em>ZjHMGR3</em> uniquely maintained root elongation under extreme stress (200 mM NaCl), aligning with its distinct structural features (intron loss; <a href="#pone.0330439.t001" class="usa-link">Table 1</a>) and rapid stress-induction kinetics in native jujube (<a href="#pone.0330439.g009" class="usa-link">Fig 9A</a>). These findings functionally anchor <em>ZjHMGRs</em> as key genetic determinants of halophytic adaptation in <em>Z. jujuba</em> var. <em>spinosa</em>. Crucially, this dual-functional function enables multi-omics dissection of ZjHMGR-mediated triterpenoid saponin biosynthesis, providing a chassis for metabolic engineering, and development of an allele swapping platform where ZjHMGR can be functionally evaluated to identify salinity-resilient alleles for marker-assisted breeding. Moreover, these studies will be instrumental in formulating a strategy to enhance the resilience of wild jujube to unfavourable environmental conditions, thereby ensuring its survival and the development of superior and highly efficient anticlimactic wild jujube germplasm resources.</p></section></section><section id="sec023"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0330439.s001"><div class="caption p">
<span>S1 File. Supplementary figures.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s001.docx</a><sup> (11.1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330439.s002"><div class="caption p">
<span>S1 Table. Details of HMGR corrected sequences used in this study.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s002.xlsx</a><sup> (26.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330439.s003"><div class="caption p">
<span>S2 Table and S3 Table. Primers for quantitative real-time PCR and vector construction.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s003.docx</a><sup> (17.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330439.s004"><div class="caption p">
<span>S4 Table. Gene ID and Sequences of HMGR in evolution tree.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s004.xlsx</a><sup> (16.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330439.s005"><div class="caption p">
<span>S5 Table. Germination rates of two varieties of wild jujube under salt stress.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s005.xlsx</a><sup> (13.4KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Prof. Xiaofang Li (School of Life Sciences, East China Normal University, China) for kindly supplying the seeds of Col-0. We also thank Dr. Shizhong Zhang (School of Life Sciences, Shandong Agricultural University, China) for providing the <em>pROKII-X-eGFP</em> vector.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>Sequence data that support the findings of this study have been deposited in NCBl project PRJNA974227 (<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA974227/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA974227/</a>) and been stored in figshare, an online data repository, at <a href="https://figshare.com/s/ad5d747ccc2ccbb2b65b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/s/ad5d747ccc2ccbb2b65b</a>. Corrected sequence data is provided within the Supplementary Table 1. The datasets generated during and/or analysed during the current study are fully available without restriction.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by Shandong Agriculture and Engineering University Start-Up Fund for Talented Scholars (2024GCCZR-14), Jinan Agricultural Science and Technology Research Project (GG202402) and Shandong Province Science and Technology-based Small and Medium-sized Enterprises Innovation Capacity Enhancement Project (2023TSGC0281). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330439.ref001">
<span class="label">1.</span><cite>Wu Y, Zhang J, Wang D, Liu J-G, Hu Y. Triterpenoid saponins from Ziziphus jujuba var. spinosa. Chem Nat Compd. 2013;49(4):677–81. doi: 10.1007/s10600-013-0705-y</cite> [<a href="https://doi.org/10.1007/s10600-013-0705-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Nat%20Compd&amp;title=Triterpenoid%20saponins%20from%20Ziziphus%20jujuba%20var.%20spinosa&amp;author=Y%20Wu&amp;author=J%20Zhang&amp;author=D%20Wang&amp;author=J-G%20Liu&amp;author=Y%20Hu&amp;volume=49&amp;issue=4&amp;publication_year=2013&amp;pages=677-81&amp;doi=10.1007/s10600-013-0705-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref002">
<span class="label">2.</span><cite>Ma Z, Zhao X, He A, Cao Y, Han Q, Lu Y, et al. Mycorrhizal symbiosis reprograms ion fluxes and fatty acid metabolism in wild jujube during salt stress. Plant Physiol. 2022;189(4):2481–99. doi: 10.1093/plphys/kiac239

</cite> [<a href="https://doi.org/10.1093/plphys/kiac239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9342988/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35604107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Mycorrhizal%20symbiosis%20reprograms%20ion%20fluxes%20and%20fatty%20acid%20metabolism%20in%20wild%20jujube%20during%20salt%20stress&amp;author=Z%20Ma&amp;author=X%20Zhao&amp;author=A%20He&amp;author=Y%20Cao&amp;author=Q%20Han&amp;volume=189&amp;issue=4&amp;publication_year=2022&amp;pages=2481-99&amp;pmid=35604107&amp;doi=10.1093/plphys/kiac239&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref003">
<span class="label">3.</span><cite>Wang Y, An Y, Yu J, Zhou Z, He S, Ru M. Different responses of photosystem II and antioxidants to drought stress in two contrasting populations of Sour jujube from the Loess Plateau, China. Ecol Res. 2016;31(6):761–75.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ecol%20Res&amp;title=Different%20responses%20of%20photosystem%20II%20and%20antioxidants%20to%20drought%20stress%20in%20two%20contrasting%20populations%20of%20Sour%20jujube%20from%20the%20Loess%20Plateau,%20China&amp;author=Y%20Wang&amp;author=Y%20An&amp;author=J%20Yu&amp;author=Z%20Zhou&amp;author=S%20He&amp;volume=31&amp;issue=6&amp;publication_year=2016&amp;pages=761-75&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref004">
<span class="label">4.</span><cite>Hua Y, Xu X-X, Guo S, Xie H, Yan H, Ma X-F, et al. Wild Jujube (Ziziphus jujuba var. spinosa): A Review of Its Phytonutrients, Health Benefits, Metabolism, and Applications. J Agric Food Chem. 2022;70(26):7871–86. doi: 10.1021/acs.jafc.2c01905

</cite> [<a href="https://doi.org/10.1021/acs.jafc.2c01905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35731918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Agric%20Food%20Chem&amp;title=Wild%20Jujube%20(Ziziphus%20jujuba%20var.%20spinosa):%20A%20Review%20of%20Its%20Phytonutrients,%20Health%20Benefits,%20Metabolism,%20and%20Applications&amp;author=Y%20Hua&amp;author=X-X%20Xu&amp;author=S%20Guo&amp;author=H%20Xie&amp;author=H%20Yan&amp;volume=70&amp;issue=26&amp;publication_year=2022&amp;pages=7871-86&amp;pmid=35731918&amp;doi=10.1021/acs.jafc.2c01905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref005">
<span class="label">5.</span><cite>Qian-Qian Z, Wen QI, Wei-Ning W, Ming-Xia LI, Fang D, Xiao-Qiu L. Isolation and identification of chemical constituents from Ziziphi Spinosae Branch and Folium. J Shenyang Pharmaceutical University. 2013;30(12):917–96.</cite>
</li>
<li id="pone.0330439.ref006">
<span class="label">6.</span><cite>Guo S, Duan J-A, Li Y, Wang R, Yan H, Qian D, et al. Comparison of the Bioactive Components in Two Seeds of Ziziphus Species by Different Analytical Approaches Combined with Chemometrics. Front Pharmacol. 2017;8:609. doi: 10.3389/fphar.2017.00609

</cite> [<a href="https://doi.org/10.3389/fphar.2017.00609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5591821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28928663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&amp;title=Comparison%20of%20the%20Bioactive%20Components%20in%20Two%20Seeds%20of%20Ziziphus%20Species%20by%20Different%20Analytical%20Approaches%20Combined%20with%20Chemometrics&amp;author=S%20Guo&amp;author=J-A%20Duan&amp;author=Y%20Li&amp;author=R%20Wang&amp;author=H%20Yan&amp;volume=8&amp;publication_year=2017&amp;pages=609&amp;pmid=28928663&amp;doi=10.3389/fphar.2017.00609&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref007">
<span class="label">7.</span><cite>Zhang H, Zhu J, Gong Z, Zhu J-K. Abiotic stress responses in plants. Nat Rev Genet. 2022;23(2):104–19. doi: 10.1038/s41576-021-00413-0

</cite> [<a href="https://doi.org/10.1038/s41576-021-00413-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34561623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Abiotic%20stress%20responses%20in%20plants&amp;author=H%20Zhang&amp;author=J%20Zhu&amp;author=Z%20Gong&amp;author=J-K%20Zhu&amp;volume=23&amp;issue=2&amp;publication_year=2022&amp;pages=104-19&amp;pmid=34561623&amp;doi=10.1038/s41576-021-00413-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref008">
<span class="label">8.</span><cite>Nadarajah KK. ROS homeostasis in abiotic stress tolerance in plants. Int J Mol Sci. 2020;21(15):5208. doi: 10.3390/ijms21155208

</cite> [<a href="https://doi.org/10.3390/ijms21155208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7432042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32717820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=ROS%20homeostasis%20in%20abiotic%20stress%20tolerance%20in%20plants&amp;author=KK%20Nadarajah&amp;volume=21&amp;issue=15&amp;publication_year=2020&amp;pmid=32717820&amp;doi=10.3390/ijms21155208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref009">
<span class="label">9.</span><cite>Singh A, Roychoudhury A. Abscisic acid in plants under abiotic stress: crosstalk with major phytohormones. Plant Cell Rep. 2023;42(6):961–74. doi: 10.1007/s00299-023-03013-w

</cite> [<a href="https://doi.org/10.1007/s00299-023-03013-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37079058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Rep&amp;title=Abscisic%20acid%20in%20plants%20under%20abiotic%20stress:%20crosstalk%20with%20major%20phytohormones&amp;author=A%20Singh&amp;author=A%20Roychoudhury&amp;volume=42&amp;issue=6&amp;publication_year=2023&amp;pages=961-74&amp;pmid=37079058&amp;doi=10.1007/s00299-023-03013-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref010">
<span class="label">10.</span><cite>Yang L, Zhang R, Zhang H, Yang Y, Fu L. TOR Mediates Stress Responses Through Global Regulation of Metabolome in Plants. Int J Mol Sci. 2025;26(5). doi: 10.3390/ijms26052095

</cite> [<a href="https://doi.org/10.3390/ijms26052095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11900525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40076716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=TOR%20Mediates%20Stress%20Responses%20Through%20Global%20Regulation%20of%20Metabolome%20in%20Plants&amp;author=L%20Yang&amp;author=R%20Zhang&amp;author=H%20Zhang&amp;author=Y%20Yang&amp;author=L%20Fu&amp;volume=26&amp;issue=5&amp;publication_year=2025&amp;pmid=40076716&amp;doi=10.3390/ijms26052095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref011">
<span class="label">11.</span><cite>Boncan DAT, Tsang SSK, Li CD, Lee IHT, Lam HM, Chan TF, et al. Terpenes and terpenoids in plants: interactions with environment and insects. Int J Mol Sci. 2020;21(19). doi: 10.3390/ijms21197382

</cite> [<a href="https://doi.org/10.3390/ijms21197382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7583029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33036280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Terpenes%20and%20terpenoids%20in%20plants:%20interactions%20with%20environment%20and%20insects&amp;author=DAT%20Boncan&amp;author=SSK%20Tsang&amp;author=CD%20Li&amp;author=IHT%20Lee&amp;author=HM%20Lam&amp;volume=21&amp;issue=19&amp;publication_year=2020&amp;pmid=33036280&amp;doi=10.3390/ijms21197382&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref012">
<span class="label">12.</span><cite>Bashir K, Todaka D, Sako K, Ueda M, Aziz F, Seki M. Chemical application improves stress resilience in plants. Plant Mol Biol. 2025;115(2):47. doi: 10.1007/s11103-025-01566-w

</cite> [<a href="https://doi.org/10.1007/s11103-025-01566-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11922999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40105987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol%20Biol&amp;title=Chemical%20application%20improves%20stress%20resilience%20in%20plants&amp;author=K%20Bashir&amp;author=D%20Todaka&amp;author=K%20Sako&amp;author=M%20Ueda&amp;author=F%20Aziz&amp;volume=115&amp;issue=2&amp;publication_year=2025&amp;pages=47&amp;pmid=40105987&amp;doi=10.1007/s11103-025-01566-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref013">
<span class="label">13.</span><cite>Gershenzon J, Dudareva N. The function of terpene natural products in the natural world. Nat Chem Biol. 2007;3(7):408–14. doi: 10.1038/nchembio.2007.5

</cite> [<a href="https://doi.org/10.1038/nchembio.2007.5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17576428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Chem%20Biol&amp;title=The%20function%20of%20terpene%20natural%20products%20in%20the%20natural%20world&amp;author=J%20Gershenzon&amp;author=N%20Dudareva&amp;volume=3&amp;issue=7&amp;publication_year=2007&amp;pages=408-14&amp;pmid=17576428&amp;doi=10.1038/nchembio.2007.5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref014">
<span class="label">14.</span><cite>Dudareva N, Negre F, Nagegowda DA, Orlova I. Plant Volatiles: Recent Advances and Future Perspectives. Critical Reviews in Plant Sciences. 2006;25(5):417–40. doi: 10.1080/07352680600899973</cite> [<a href="https://doi.org/10.1080/07352680600899973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Critical%20Reviews%20in%20Plant%20Sciences&amp;title=Plant%20Volatiles:%20Recent%20Advances%20and%20Future%20Perspectives&amp;author=N%20Dudareva&amp;author=F%20Negre&amp;author=DA%20Nagegowda&amp;author=I%20Orlova&amp;volume=25&amp;issue=5&amp;publication_year=2006&amp;pages=417-40&amp;doi=10.1080/07352680600899973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref015">
<span class="label">15.</span><cite>Sun Y, Zhang Y, Qi W, Xie J, Cui X. Saponins extracted by ultrasound from Zizyphus jujuba Mil var. spinosa leaves exert resistance to oxidative damage in Caenorhabditis elegans. J Food Meas Charact. 2020;15(1):541–54.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Food%20Meas%20Charact&amp;title=Saponins%20extracted%20by%20ultrasound%20from%20Zizyphus%20jujuba%20Mil%20var.%20spinosa%20leaves%20exert%20resistance%20to%20oxidative%20damage%20in%20Caenorhabditis%20elegans&amp;author=Y%20Sun&amp;author=Y%20Zhang&amp;author=W%20Qi&amp;author=J%20Xie&amp;author=X%20Cui&amp;volume=15&amp;issue=1&amp;publication_year=2020&amp;pages=541-54&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref016">
<span class="label">16.</span><cite>Song L, Zhang L, Xu L, Ma Y, Lian W, Liu Y, et al. Optimized Extraction of Total Triterpenoids from Jujube (Ziziphus jujuba Mill.) and Comprehensive Analysis of Triterpenic Acids in Different Cultivars. Plants (Basel). 2020;9(4):412. doi: 10.3390/plants9040412

</cite> [<a href="https://doi.org/10.3390/plants9040412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7238538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32230740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants%20(Basel)&amp;title=Optimized%20Extraction%20of%20Total%20Triterpenoids%20from%20Jujube%20(Ziziphus%20jujuba%20Mill.)%20and%20Comprehensive%20Analysis%20of%20Triterpenic%20Acids%20in%20Different%20Cultivars&amp;author=L%20Song&amp;author=L%20Zhang&amp;author=L%20Xu&amp;author=Y%20Ma&amp;author=W%20Lian&amp;volume=9&amp;issue=4&amp;publication_year=2020&amp;pages=412&amp;pmid=32230740&amp;doi=10.3390/plants9040412&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref017">
<span class="label">17.</span><cite>Ruan W, Liu J, Zhang S, Huang Y, Zhang Y, Wang Z. Sour Jujube (Ziziphus jujuba var. spinosa): A Bibliometric Review of Its Bioactive Profile, Health Benefits and Trends in Food and Medicine Applications. Foods. 2024;13(5):636. doi: 10.3390/foods13050636

</cite> [<a href="https://doi.org/10.3390/foods13050636" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10930723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38472749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=Sour%20Jujube%20(Ziziphus%20jujuba%20var.%20spinosa):%20A%20Bibliometric%20Review%20of%20Its%20Bioactive%20Profile,%20Health%20Benefits%20and%20Trends%20in%20Food%20and%20Medicine%20Applications&amp;author=W%20Ruan&amp;author=J%20Liu&amp;author=S%20Zhang&amp;author=Y%20Huang&amp;author=Y%20Zhang&amp;volume=13&amp;issue=5&amp;publication_year=2024&amp;pages=636&amp;pmid=38472749&amp;doi=10.3390/foods13050636&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref018">
<span class="label">18.</span><cite>Wang Q, Zhao X, Jiang Y, Jin B, Wang L. Functions of Representative Terpenoids and Their Biosynthesis Mechanisms in Medicinal Plants. Biomolecules. 2023;13(12):1725. doi: 10.3390/biom13121725

</cite> [<a href="https://doi.org/10.3390/biom13121725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10741589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38136596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Functions%20of%20Representative%20Terpenoids%20and%20Their%20Biosynthesis%20Mechanisms%20in%20Medicinal%20Plants&amp;author=Q%20Wang&amp;author=X%20Zhao&amp;author=Y%20Jiang&amp;author=B%20Jin&amp;author=L%20Wang&amp;volume=13&amp;issue=12&amp;publication_year=2023&amp;pages=1725&amp;pmid=38136596&amp;doi=10.3390/biom13121725&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref019">
<span class="label">19.</span><cite>Wang P, Qi L, Song J, Zhu R, Han X, Liu Y, et al. Functional characterization of ZjPYL8 from sour jujube: enhancing the sensitivity of stomata and roots to ABA in Arabidopsis thaliana. Acta Sci Pol Hortorum Cultus. 2023;22(6):79–89. doi: 10.24326/asp.hc.2023.5154</cite> [<a href="https://doi.org/10.24326/asp.hc.2023.5154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Sci%20Pol%20Hortorum%20Cultus&amp;title=Functional%20characterization%20of%20ZjPYL8%20from%20sour%20jujube:%20enhancing%20the%20sensitivity%20of%20stomata%20and%20roots%20to%20ABA%20in%20Arabidopsis%20thaliana&amp;author=P%20Wang&amp;author=L%20Qi&amp;author=J%20Song&amp;author=R%20Zhu&amp;author=X%20Han&amp;volume=22&amp;issue=6&amp;publication_year=2023&amp;pages=79-89&amp;doi=10.24326/asp.hc.2023.5154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref020">
<span class="label">20.</span><cite>Gupta A, Rico-Medina A, Caño-Delgado AI. The physiology of plant responses to drought. Science. 2020;368(6488):266–9. doi: 10.1126/science.aaz7614

</cite> [<a href="https://doi.org/10.1126/science.aaz7614" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32299946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20physiology%20of%20plant%20responses%20to%20drought&amp;author=A%20Gupta&amp;author=A%20Rico-Medina&amp;author=AI%20Ca%C3%B1o-Delgado&amp;volume=368&amp;issue=6488&amp;publication_year=2020&amp;pages=266-9&amp;pmid=32299946&amp;doi=10.1126/science.aaz7614&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref021">
<span class="label">21.</span><cite>Bergman ME, Kortbeek RWJ, Gutensohn M, Dudareva N. Plant terpenoid biosynthetic network and its multiple layers of regulation. Prog Lipid Res. 2024;95:101287. doi: 10.1016/j.plipres.2024.101287

</cite> [<a href="https://doi.org/10.1016/j.plipres.2024.101287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38906423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog%20Lipid%20Res&amp;title=Plant%20terpenoid%20biosynthetic%20network%20and%20its%20multiple%20layers%20of%20regulation&amp;author=ME%20Bergman&amp;author=RWJ%20Kortbeek&amp;author=M%20Gutensohn&amp;author=N%20Dudareva&amp;volume=95&amp;publication_year=2024&amp;pages=101287&amp;pmid=38906423&amp;doi=10.1016/j.plipres.2024.101287&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref022">
<span class="label">22.</span><cite>Du P, He H, Wang J, Wang L, Meng Z, Jin X, et al. Genome-Wide Identification and Characterization of the HMGR Gene Family in Taraxacum kok-saghyz Provide Insights into Its Regulation in Response to Ethylene and Methyl Jsamonate Treatments. Plants (Basel). 2024;13(18):2646. doi: 10.3390/plants13182646

</cite> [<a href="https://doi.org/10.3390/plants13182646" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11435204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39339620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants%20(Basel)&amp;title=Genome-Wide%20Identification%20and%20Characterization%20of%20the%20HMGR%20Gene%20Family%20in%20Taraxacum%20kok-saghyz%20Provide%20Insights%20into%20Its%20Regulation%20in%20Response%20to%20Ethylene%20and%20Methyl%20Jsamonate%20Treatments&amp;author=P%20Du&amp;author=H%20He&amp;author=J%20Wang&amp;author=L%20Wang&amp;author=Z%20Meng&amp;volume=13&amp;issue=18&amp;publication_year=2024&amp;pages=2646&amp;pmid=39339620&amp;doi=10.3390/plants13182646&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref023">
<span class="label">23.</span><cite>Wang X, Wang C, Yang M, Jie W, Fazal A, Fu J, et al. Genome-Wide Comparison and Functional Characterization of HMGR Gene Family Associated with Shikonin Biosynthesis in Lithospermum erythrorhizon. Int J Mol Sci. 2023;24(15):12532. doi: 10.3390/ijms241512532

</cite> [<a href="https://doi.org/10.3390/ijms241512532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10419935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37569907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Genome-Wide%20Comparison%20and%20Functional%20Characterization%20of%20HMGR%20Gene%20Family%20Associated%20with%20Shikonin%20Biosynthesis%20in%20Lithospermum%20erythrorhizon&amp;author=X%20Wang&amp;author=C%20Wang&amp;author=M%20Yang&amp;author=W%20Jie&amp;author=A%20Fazal&amp;volume=24&amp;issue=15&amp;publication_year=2023&amp;pages=12532&amp;pmid=37569907&amp;doi=10.3390/ijms241512532&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref024">
<span class="label">24.</span><cite>Liao Z, Tan Q, Chai Y, Zuo K, Chen M, Gong Y, et al. Cloning and characterisation of the gene encoding HMG-CoA reductase from Taxus media and its functional identification in yeast. Funct Plant Biol. 2004;31(1):73–81. doi: 10.1071/FP03153

</cite> [<a href="https://doi.org/10.1071/FP03153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32688882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Funct%20Plant%20Biol&amp;title=Cloning%20and%20characterisation%20of%20the%20gene%20encoding%20HMG-CoA%20reductase%20from%20Taxus%20media%20and%20its%20functional%20identification%20in%20yeast&amp;author=Z%20Liao&amp;author=Q%20Tan&amp;author=Y%20Chai&amp;author=K%20Zuo&amp;author=M%20Chen&amp;volume=31&amp;issue=1&amp;publication_year=2004&amp;pages=73-81&amp;pmid=32688882&amp;doi=10.1071/FP03153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref025">
<span class="label">25.</span><cite>Li W, Liu W, Wei H, He Q, Chen J, Zhang B, et al. Species-specific expansion and molecular evolution of the 3-hydroxy-3-methylglutaryl coenzyme A reductase (HMGR) gene family in plants. PLoS One. 2014;9(4):e94172. doi: 10.1371/journal.pone.0094172

</cite> [<a href="https://doi.org/10.1371/journal.pone.0094172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3983158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24722776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Species-specific%20expansion%20and%20molecular%20evolution%20of%20the%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase%20(HMGR)%20gene%20family%20in%20plants&amp;author=W%20Li&amp;author=W%20Liu&amp;author=H%20Wei&amp;author=Q%20He&amp;author=J%20Chen&amp;volume=9&amp;issue=4&amp;publication_year=2014&amp;pmid=24722776&amp;doi=10.1371/journal.pone.0094172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref026">
<span class="label">26.</span><cite>Bach TJ. Hydroxymethylglutaryl-CoA reductase, a key enzyme in phytosterol synthesis?. Lipids. 1986;21(1):82–8. doi: 10.1007/BF02534307

</cite> [<a href="https://doi.org/10.1007/BF02534307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3959769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lipids&amp;title=Hydroxymethylglutaryl-CoA%20reductase,%20a%20key%20enzyme%20in%20phytosterol%20synthesis?&amp;author=TJ%20Bach&amp;volume=21&amp;issue=1&amp;publication_year=1986&amp;pages=82-8&amp;pmid=3959769&amp;doi=10.1007/BF02534307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref027">
<span class="label">27.</span><cite>Caelles C, Ferrer A, Balcells L, Hegardt FG, Boronat A. Isolation and structural characterization of a cDNA encoding Arabidopsis thaliana 3-hydroxy-3-methylglutaryl coenzyme A reductase. Plant Mol Biol. 1989;13(6):627–38. doi: 10.1007/BF00016018

</cite> [<a href="https://doi.org/10.1007/BF00016018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2491679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol%20Biol&amp;title=Isolation%20and%20structural%20characterization%20of%20a%20cDNA%20encoding%20Arabidopsis%20thaliana%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase&amp;author=C%20Caelles&amp;author=A%20Ferrer&amp;author=L%20Balcells&amp;author=FG%20Hegardt&amp;author=A%20Boronat&amp;volume=13&amp;issue=6&amp;publication_year=1989&amp;pages=627-38&amp;pmid=2491679&amp;doi=10.1007/BF00016018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref028">
<span class="label">28.</span><cite>Kim Y-J, Lee OR, Oh JY, Jang M-G, Yang D-C. Functional analysis of 3-hydroxy-3-methylglutaryl coenzyme a reductase encoding genes in triterpene saponin-producing ginseng. Plant Physiol. 2014;165(1):373–87. doi: 10.1104/pp.113.222596

</cite> [<a href="https://doi.org/10.1104/pp.113.222596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4012596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24569845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Functional%20analysis%20of%203-hydroxy-3-methylglutaryl%20coenzyme%20a%20reductase%20encoding%20genes%20in%20triterpene%20saponin-producing%20ginseng&amp;author=Y-J%20Kim&amp;author=OR%20Lee&amp;author=JY%20Oh&amp;author=M-G%20Jang&amp;author=D-C%20Yang&amp;volume=165&amp;issue=1&amp;publication_year=2014&amp;pages=373-87&amp;pmid=24569845&amp;doi=10.1104/pp.113.222596&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref029">
<span class="label">29.</span><cite>Dai Z, Cui G, Zhou S-F, Zhang X, Huang L. Cloning and characterization of a novel 3-hydroxy-3-methylglutaryl coenzyme A reductase gene from Salvia miltiorrhiza involved in diterpenoid tanshinone accumulation. J Plant Physiol. 2011;168(2):148–57. doi: 10.1016/j.jplph.2010.06.008

</cite> [<a href="https://doi.org/10.1016/j.jplph.2010.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20637524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Plant%20Physiol&amp;title=Cloning%20and%20characterization%20of%20a%20novel%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase%20gene%20from%20Salvia%20miltiorrhiza%20involved%20in%20diterpenoid%20tanshinone%20accumulation&amp;author=Z%20Dai&amp;author=G%20Cui&amp;author=S-F%20Zhou&amp;author=X%20Zhang&amp;author=L%20Huang&amp;volume=168&amp;issue=2&amp;publication_year=2011&amp;pages=148-57&amp;pmid=20637524&amp;doi=10.1016/j.jplph.2010.06.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref030">
<span class="label">30.</span><cite>Rao S, Meng X, Liao Y, Yu T, Cao J, Tan J, et al. Characterization and functional analysis of two novel 3-hydroxy-3-methylglutaryl-coenzyme A reductase genes (GbHMGR2 and GbHMGR3) from Ginkgo biloba. Sci Rep. 2019;9(1):14109. doi: 10.1038/s41598-019-50629-8

</cite> [<a href="https://doi.org/10.1038/s41598-019-50629-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6773693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31575936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Characterization%20and%20functional%20analysis%20of%20two%20novel%203-hydroxy-3-methylglutaryl-coenzyme%20A%20reductase%20genes%20(GbHMGR2%20and%20GbHMGR3)%20from%20Ginkgo%20biloba&amp;author=S%20Rao&amp;author=X%20Meng&amp;author=Y%20Liao&amp;author=T%20Yu&amp;author=J%20Cao&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=14109&amp;pmid=31575936&amp;doi=10.1038/s41598-019-50629-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref031">
<span class="label">31.</span><cite>Wang S, Feng Y, Lou Y, Niu J, Yin C, Zhao J, et al. 3-Hydroxy-3-methylglutaryl coenzyme A reductase genes from Glycine max regulate plant growth and isoprenoid biosynthesis. Sci Rep. 2023;13(1):3902. doi: 10.1038/s41598-023-30797-4

</cite> [<a href="https://doi.org/10.1038/s41598-023-30797-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9995466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36890158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=3-Hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase%20genes%20from%20Glycine%20max%20regulate%20plant%20growth%20and%20isoprenoid%20biosynthesis&amp;author=S%20Wang&amp;author=Y%20Feng&amp;author=Y%20Lou&amp;author=J%20Niu&amp;author=C%20Yin&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=3902&amp;pmid=36890158&amp;doi=10.1038/s41598-023-30797-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref032">
<span class="label">32.</span><cite>Zheng T, Guan L, Yu K, Haider MS, Nasim M, Liu Z, et al. Expressional diversity of grapevine 3-Hydroxy-3-methylglutaryl-CoA reductase (VvHMGR) in different grapes genotypes. BMC Plant Biol. 2021;21(1):279. doi: 10.1186/s12870-021-03073-8

</cite> [<a href="https://doi.org/10.1186/s12870-021-03073-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8214791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34147088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol&amp;title=Expressional%20diversity%20of%20grapevine%203-Hydroxy-3-methylglutaryl-CoA%20reductase%20(VvHMGR)%20in%20different%20grapes%20genotypes&amp;author=T%20Zheng&amp;author=L%20Guan&amp;author=K%20Yu&amp;author=MS%20Haider&amp;author=M%20Nasim&amp;volume=21&amp;issue=1&amp;publication_year=2021&amp;pages=279&amp;pmid=34147088&amp;doi=10.1186/s12870-021-03073-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref033">
<span class="label">33.</span><cite>Deng B, Zhang P, Ge F, Liu DQ, Chen CY. Enhancement of triterpenoid saponins biosynthesis in Panax notoginseng cells by co-overexpressions of 3-hydroxy-3-methylglutaryl CoA reductase and squalene synthase genes. Biochem Eng J. 2017;122:38–46.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Eng%20J&amp;title=Enhancement%20of%20triterpenoid%20saponins%20biosynthesis%20in%20Panax%20notoginseng%20cells%20by%20co-overexpressions%20of%203-hydroxy-3-methylglutaryl%20CoA%20reductase%20and%20squalene%20synthase%20genes&amp;author=B%20Deng&amp;author=P%20Zhang&amp;author=F%20Ge&amp;author=DQ%20Liu&amp;author=CY%20Chen&amp;volume=122&amp;publication_year=2017&amp;pages=38-46&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref034">
<span class="label">34.</span><cite>Zhang M, Liu H, Wang Q, Liu S, Zhang Y. The 3-hydroxy-3-methylglutaryl-coenzyme A reductase 5 gene from Malus domestica enhances oxidative stress tolerance in Arabidopsis thaliana. Plant Physiol Biochem. 2020;146:269–77. doi: 10.1016/j.plaphy.2019.11.031

</cite> [<a href="https://doi.org/10.1016/j.plaphy.2019.11.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31783202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol%20Biochem&amp;title=The%203-hydroxy-3-methylglutaryl-coenzyme%20A%20reductase%205%20gene%20from%20Malus%20domestica%20enhances%20oxidative%20stress%20tolerance%20in%20Arabidopsis%20thaliana&amp;author=M%20Zhang&amp;author=H%20Liu&amp;author=Q%20Wang&amp;author=S%20Liu&amp;author=Y%20Zhang&amp;volume=146&amp;publication_year=2020&amp;pages=269-77&amp;pmid=31783202&amp;doi=10.1016/j.plaphy.2019.11.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref035">
<span class="label">35.</span><cite>Korth KL, Jaggard DA, Dixon RA. Developmental and light-regulated post-translational control of 3-hydroxy-3-methylglutaryl-CoA reductase levels in potato. Plant J. 2000;23(4):507–16. doi: 10.1046/j.1365-313x.2000.00821.x

</cite> [<a href="https://doi.org/10.1046/j.1365-313x.2000.00821.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10972877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Developmental%20and%20light-regulated%20post-translational%20control%20of%203-hydroxy-3-methylglutaryl-CoA%20reductase%20levels%20in%20potato&amp;author=KL%20Korth&amp;author=DA%20Jaggard&amp;author=RA%20Dixon&amp;volume=23&amp;issue=4&amp;publication_year=2000&amp;pages=507-16&amp;pmid=10972877&amp;doi=10.1046/j.1365-313x.2000.00821.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref036">
<span class="label">36.</span><cite>Li K, Chen R, Abudoukayoumu A, Wei Q, Ma Z, Wang Z, et al. Haplotype-resolved T2T reference genomes for wild and domesticated accessions shed new insights into the domestication of jujube. Hortic Res. 2024;11(5):uhae071. doi: 10.1093/hr/uhae071

</cite> [<a href="https://doi.org/10.1093/hr/uhae071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11079485/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38725458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hortic%20Res&amp;title=Haplotype-resolved%20T2T%20reference%20genomes%20for%20wild%20and%20domesticated%20accessions%20shed%20new%20insights%20into%20the%20domestication%20of%20jujube&amp;author=K%20Li&amp;author=R%20Chen&amp;author=A%20Abudoukayoumu&amp;author=Q%20Wei&amp;author=Z%20Ma&amp;volume=11&amp;issue=5&amp;publication_year=2024&amp;pmid=38725458&amp;doi=10.1093/hr/uhae071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref037">
<span class="label">37.</span><cite>Demkura PV, Ballaré CL. UVR8 mediates UV-B-induced Arabidopsis defense responses against Botrytis cinerea by controlling sinapate accumulation. Mol Plant. 2012;5(3):642–52. doi: 10.1093/mp/sss025

</cite> [<a href="https://doi.org/10.1093/mp/sss025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22447155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&amp;title=UVR8%20mediates%20UV-B-induced%20Arabidopsis%20defense%20responses%20against%20Botrytis%20cinerea%20by%20controlling%20sinapate%20accumulation&amp;author=PV%20Demkura&amp;author=CL%20Ballar%C3%A9&amp;volume=5&amp;issue=3&amp;publication_year=2012&amp;pages=642-52&amp;pmid=22447155&amp;doi=10.1093/mp/sss025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref038">
<span class="label">38.</span><cite>Chen C, Chen H, Zhang Y, Thomas HR, Frank MH, He Y, et al. TBtools: An Integrative Toolkit Developed for Interactive Analyses of Big Biological Data. Mol Plant. 2020;13(8):1194–202. doi: 10.1016/j.molp.2020.06.009

</cite> [<a href="https://doi.org/10.1016/j.molp.2020.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32585190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&amp;title=TBtools:%20An%20Integrative%20Toolkit%20Developed%20for%20Interactive%20Analyses%20of%20Big%20Biological%20Data&amp;author=C%20Chen&amp;author=H%20Chen&amp;author=Y%20Zhang&amp;author=HR%20Thomas&amp;author=MH%20Frank&amp;volume=13&amp;issue=8&amp;publication_year=2020&amp;pages=1194-202&amp;pmid=32585190&amp;doi=10.1016/j.molp.2020.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref039">
<span class="label">39.</span><cite>Duvaud S, Gabella C, Lisacek F, Stockinger H, Ioannidis V, Durinx C. Expasy, the Swiss Bioinformatics Resource Portal, as designed by its users. Nucleic Acids Res. 2021;49(W1):W216–27. doi: 10.1093/nar/gkab225

</cite> [<a href="https://doi.org/10.1093/nar/gkab225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8265094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33849055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Expasy,%20the%20Swiss%20Bioinformatics%20Resource%20Portal,%20as%20designed%20by%20its%20users&amp;author=S%20Duvaud&amp;author=C%20Gabella&amp;author=F%20Lisacek&amp;author=H%20Stockinger&amp;author=V%20Ioannidis&amp;volume=49&amp;publication_year=2021&amp;pmid=33849055&amp;doi=10.1093/nar/gkab225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref040">
<span class="label">40.</span><cite>Letunic I, Khedkar S, Bork P. SMART: recent updates, new developments and status in 2020. Nucleic Acids Res. 2021;49(D1):D458–60. doi: 10.1093/nar/gkaa937

</cite> [<a href="https://doi.org/10.1093/nar/gkaa937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33104802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=SMART:%20recent%20updates,%20new%20developments%20and%20status%20in%202020&amp;author=I%20Letunic&amp;author=S%20Khedkar&amp;author=P%20Bork&amp;volume=49&amp;publication_year=2021&amp;pmid=33104802&amp;doi=10.1093/nar/gkaa937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref041">
<span class="label">41.</span><cite>Bailey TL, Johnson J, Grant CE, Noble WS. The MEME Suite. Nucleic Acids Research. 2015;43(W1):W39–49. doi: 10.1093/nar/gkv416

</cite> [<a href="https://doi.org/10.1093/nar/gkv416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4489269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25953851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20MEME%20Suite&amp;author=TL%20Bailey&amp;author=J%20Johnson&amp;author=CE%20Grant&amp;author=WS%20Noble&amp;volume=43&amp;publication_year=2015&amp;pmid=25953851&amp;doi=10.1093/nar/gkv416&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref042">
<span class="label">42.</span><cite>Hallgren J, Tsirigos KD, Pedersen MD, Almagro Armenteros JJ, Marcatili P, Nielsen H. DeepTMHMM predicts alpha and beta transmembrane proteins using deep neural networks. bioRxiv. 2022.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=DeepTMHMM%20predicts%20alpha%20and%20beta%20transmembrane%20proteins%20using%20deep%20neural%20networks&amp;author=J%20Hallgren&amp;author=KD%20Tsirigos&amp;author=MD%20Pedersen&amp;author=JJ%20Almagro%20Armenteros&amp;author=P%20Marcatili&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref043">
<span class="label">43.</span><cite>Teufel F, Almagro Armenteros JJ, Johansen AR, Gíslason MH, Pihl SI, Tsirigos KD, et al. SignalP 6.0 predicts all five types of signal peptides using protein language models. Nat Biotechnol. 2022;40(7):1023–5. doi: 10.1038/s41587-021-01156-3

</cite> [<a href="https://doi.org/10.1038/s41587-021-01156-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9287161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34980915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=SignalP%206.0%20predicts%20all%20five%20types%20of%20signal%20peptides%20using%20protein%20language%20models&amp;author=F%20Teufel&amp;author=JJ%20Almagro%20Armenteros&amp;author=AR%20Johansen&amp;author=MH%20G%C3%ADslason&amp;author=SI%20Pihl&amp;volume=40&amp;issue=7&amp;publication_year=2022&amp;pages=1023-5&amp;pmid=34980915&amp;doi=10.1038/s41587-021-01156-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref044">
<span class="label">44.</span><cite>Chou K-C, Shen H-B. Cell-PLoc: a package of Web servers for predicting subcellular localization of proteins in various organisms. Nat Protoc. 2008;3(2):153–62. doi: 10.1038/nprot.2007.494

</cite> [<a href="https://doi.org/10.1038/nprot.2007.494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18274516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=Cell-PLoc:%20a%20package%20of%20Web%20servers%20for%20predicting%20subcellular%20localization%20of%20proteins%20in%20various%20organisms&amp;author=K-C%20Chou&amp;author=H-B%20Shen&amp;volume=3&amp;issue=2&amp;publication_year=2008&amp;pages=153-62&amp;pmid=18274516&amp;doi=10.1038/nprot.2007.494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref045">
<span class="label">45.</span><cite>Robert X, Gouet P. Deciphering key features in protein structures with the new ENDscript server. Nucleic Acids Res. 2014;42(Web Server issue):W320-4. doi: 10.1093/nar/gku316

</cite> [<a href="https://doi.org/10.1093/nar/gku316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24753421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Deciphering%20key%20features%20in%20protein%20structures%20with%20the%20new%20ENDscript%20server&amp;author=X%20Robert&amp;author=P%20Gouet&amp;volume=42&amp;publication_year=2014&amp;pmid=24753421&amp;doi=10.1093/nar/gku316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref046">
<span class="label">46.</span><cite>Abramson J, Adler J, Dunger J, Evans R, Green T, Pritzel A, et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Nature. 2024;630(8016):493–500. doi: 10.1038/s41586-024-07487-w

</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203&amp;author=J%20Abramson&amp;author=J%20Adler&amp;author=J%20Dunger&amp;author=R%20Evans&amp;author=T%20Green&amp;volume=630&amp;issue=8016&amp;publication_year=2024&amp;pages=493-500&amp;pmid=38718835&amp;doi=10.1038/s41586-024-07487-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref047">
<span class="label">47.</span><cite>Tamura K, Stecher G, Kumar S. MEGA11 Molecular Evolutionary Genetics Analysis Version 11. Mol Biol Evol. 2021;38(7):3022–7.
</cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=MEGA11%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%2011&amp;author=K%20Tamura&amp;author=G%20Stecher&amp;author=S%20Kumar&amp;volume=38&amp;issue=7&amp;publication_year=2021&amp;pages=3022-7&amp;pmid=33892491&amp;doi=10.1093/molbev/msab120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref048">
<span class="label">48.</span><cite>Koch MA, Haubold B, Mitchell-Olds T. Comparative evolutionary analysis of chalcone synthase and alcohol dehydrogenase loci in Arabidopsis, Arabis, and related genera (Brassicaceae). Mol Biol Evol. 2000;17(10):1483–98. doi: 10.1093/oxfordjournals.molbev.a026248

</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a026248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11018155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Comparative%20evolutionary%20analysis%20of%20chalcone%20synthase%20and%20alcohol%20dehydrogenase%20loci%20in%20Arabidopsis,%20Arabis,%20and%20related%20genera%20(Brassicaceae)&amp;author=MA%20Koch&amp;author=B%20Haubold&amp;author=T%20Mitchell-Olds&amp;volume=17&amp;issue=10&amp;publication_year=2000&amp;pages=1483-98&amp;pmid=11018155&amp;doi=10.1093/oxfordjournals.molbev.a026248&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref049">
<span class="label">49.</span><cite>Lescot M, Déhais P, Thijs G, Marchal K, Moreau Y, Van de Peer Y, et al. PlantCARE, a database of plant cis-acting regulatory elements and a portal to tools for in silico analysis of promoter sequences. Nucleic Acids Res. 2002;30(1):325–7. doi: 10.1093/nar/30.1.325

</cite> [<a href="https://doi.org/10.1093/nar/30.1.325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC99092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11752327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=PlantCARE,%20a%20database%20of%20plant%20cis-acting%20regulatory%20elements%20and%20a%20portal%20to%20tools%20for%20in%20silico%20analysis%20of%20promoter%20sequences&amp;author=M%20Lescot&amp;author=P%20D%C3%A9hais&amp;author=G%20Thijs&amp;author=K%20Marchal&amp;author=Y%20Moreau&amp;volume=30&amp;issue=1&amp;publication_year=2002&amp;pages=325-7&amp;pmid=11752327&amp;doi=10.1093/nar/30.1.325&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref050">
<span class="label">50.</span><cite>Schmittgen TD, Livak KJ. Analyzing real-time PCR data by the comparative C(T) method. Nat Protoc. 2008;3(6):1101–8. doi: 10.1038/nprot.2008.73

</cite> [<a href="https://doi.org/10.1038/nprot.2008.73" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18546601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=Analyzing%20real-time%20PCR%20data%20by%20the%20comparative%20C(T)%20method&amp;author=TD%20Schmittgen&amp;author=KJ%20Livak&amp;volume=3&amp;issue=6&amp;publication_year=2008&amp;pages=1101-8&amp;pmid=18546601&amp;doi=10.1038/nprot.2008.73&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref051">
<span class="label">51.</span><cite>Waadt R, Seller CA, Hsu P-K, Takahashi Y, Munemasa S, Schroeder JI. Plant hormone regulation of abiotic stress responses. Nat Rev Mol Cell Biol. 2022;23(10):680–94. doi: 10.1038/s41580-022-00479-6

</cite> [<a href="https://doi.org/10.1038/s41580-022-00479-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9592120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35513717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Plant%20hormone%20regulation%20of%20abiotic%20stress%20responses&amp;author=R%20Waadt&amp;author=CA%20Seller&amp;author=P-K%20Hsu&amp;author=Y%20Takahashi&amp;author=S%20Munemasa&amp;volume=23&amp;issue=10&amp;publication_year=2022&amp;pages=680-94&amp;pmid=35513717&amp;doi=10.1038/s41580-022-00479-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref052">
<span class="label">52.</span><cite>Shergis JL, Ni X, Sarris J, Zhang AL, Guo X, Xue CC, et al. Ziziphus spinosa seeds for insomnia: A review of chemistry and psychopharmacology. Phytomedicine. 2017;34:38–43. doi: 10.1016/j.phymed.2017.07.004

</cite> [<a href="https://doi.org/10.1016/j.phymed.2017.07.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28899507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytomedicine&amp;title=Ziziphus%20spinosa%20seeds%20for%20insomnia:%20A%20review%20of%20chemistry%20and%20psychopharmacology&amp;author=JL%20Shergis&amp;author=X%20Ni&amp;author=J%20Sarris&amp;author=AL%20Zhang&amp;author=X%20Guo&amp;volume=34&amp;publication_year=2017&amp;pages=38-43&amp;pmid=28899507&amp;doi=10.1016/j.phymed.2017.07.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref053">
<span class="label">53.</span><cite>Wei H, Xu C, Movahedi A, Sun W, Li D, Zhuge Q. Characterization and Function of 3-Hydroxy-3-Methylglutaryl-CoA Reductase in Populus trichocarpa: Overexpression of PtHMGR Enhances Terpenoids in Transgenic Poplar. Front Plant Sci. 2019;10:1476. doi: 10.3389/fpls.2019.01476

</cite> [<a href="https://doi.org/10.3389/fpls.2019.01476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6872958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31803212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&amp;title=Characterization%20and%20Function%20of%203-Hydroxy-3-Methylglutaryl-CoA%20Reductase%20in%20Populus%20trichocarpa:%20Overexpression%20of%20PtHMGR%20Enhances%20Terpenoids%20in%20Transgenic%20Poplar&amp;author=H%20Wei&amp;author=C%20Xu&amp;author=A%20Movahedi&amp;author=W%20Sun&amp;author=D%20Li&amp;volume=10&amp;publication_year=2019&amp;pages=1476&amp;pmid=31803212&amp;doi=10.3389/fpls.2019.01476&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref054">
<span class="label">54.</span><cite>Aquil S, Husaini AM, Abdin MZ, Rather GM. Overexpression of the HMG-CoA reductase gene leads to enhanced artemisinin biosynthesis in transgenic Artemisia annua plants. Planta Med. 2009;75(13):1453–8. doi: 10.1055/s-0029-1185775

</cite> [<a href="https://doi.org/10.1055/s-0029-1185775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19551613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta%20Med&amp;title=Overexpression%20of%20the%20HMG-CoA%20reductase%20gene%20leads%20to%20enhanced%20artemisinin%20biosynthesis%20in%20transgenic%20Artemisia%20annua%20plants&amp;author=S%20Aquil&amp;author=AM%20Husaini&amp;author=MZ%20Abdin&amp;author=GM%20Rather&amp;volume=75&amp;issue=13&amp;publication_year=2009&amp;pages=1453-8&amp;pmid=19551613&amp;doi=10.1055/s-0029-1185775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref055">
<span class="label">55.</span><cite>Learned RM, Connolly EL. Light modulates the spatial patterns of 3-hydroxy-3-methylglutaryl coenzyme A reductase gene expression in Arabidopsis thaliana. Plant J. 1997;11(3):499–511. doi: 10.1046/j.1365-313x.1997.11030499.x

</cite> [<a href="https://doi.org/10.1046/j.1365-313x.1997.11030499.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9107038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Light%20modulates%20the%20spatial%20patterns%20of%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase%20gene%20expression%20in%20Arabidopsis%20thaliana&amp;author=RM%20Learned&amp;author=EL%20Connolly&amp;volume=11&amp;issue=3&amp;publication_year=1997&amp;pages=499-511&amp;pmid=9107038&amp;doi=10.1046/j.1365-313x.1997.11030499.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref056">
<span class="label">56.</span><cite>Soliman SSM, Raizada MN. Darkness: A Crucial Factor in Fungal Taxol Production. Front Microbiol. 2018;9:353. doi: 10.3389/fmicb.2018.00353

</cite> [<a href="https://doi.org/10.3389/fmicb.2018.00353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5840228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29552002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Darkness:%20A%20Crucial%20Factor%20in%20Fungal%20Taxol%20Production&amp;author=SSM%20Soliman&amp;author=MN%20Raizada&amp;volume=9&amp;publication_year=2018&amp;pages=353&amp;pmid=29552002&amp;doi=10.3389/fmicb.2018.00353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref057">
<span class="label">57.</span><cite>C-Xiang X. Biological responses and resistances of five cultivars of Chinese jujube and sour date under iso-osmotic drought, salt and alkaline stresses. J Cent South Univ For Techn. 2012.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cent%20South%20Univ%20For%20Techn&amp;title=Biological%20responses%20and%20resistances%20of%20five%20cultivars%20of%20Chinese%20jujube%20and%20sour%20date%20under%20iso-osmotic%20drought,%20salt%20and%20alkaline%20stresses&amp;author=X%20C-Xiang&amp;publication_year=2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref058">
<span class="label">58.</span><cite>Linh NT, Cuong LK, Tam HT, Tung HT, Luan VQ, Hien VT, et al. Improvement of bioactive saponin accumulation in adventitious root cultures of Panax vietnamensis via culture periods and elicitation. PCTOC. 2019;137(1):101–13.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=PCTOC&amp;title=Improvement%20of%20bioactive%20saponin%20accumulation%20in%20adventitious%20root%20cultures%20of%20Panax%20vietnamensis%20via%20culture%20periods%20and%20elicitation&amp;author=NT%20Linh&amp;author=LK%20Cuong&amp;author=HT%20Tam&amp;author=HT%20Tung&amp;author=VQ%20Luan&amp;volume=137&amp;issue=1&amp;publication_year=2019&amp;pages=101-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref059">
<span class="label">59.</span><cite>Yin J, Yang J, Ma H, Liang T, Li Y, Xiao J, et al. Expression characteristics and function of CAS and a new beta-amyrin synthase in triterpenoid synthesis in birch (Betula platyphylla Suk.). Plant Sci. 2020;294:110433. doi: 10.1016/j.plantsci.2020.110433

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2020.110433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32234222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci&amp;title=Expression%20characteristics%20and%20function%20of%20CAS%20and%20a%20new%20beta-amyrin%20synthase%20in%20triterpenoid%20synthesis%20in%20birch%20(Betula%20platyphylla%20Suk.)&amp;author=J%20Yin&amp;author=J%20Yang&amp;author=H%20Ma&amp;author=T%20Liang&amp;author=Y%20Li&amp;volume=294&amp;publication_year=2020&amp;pages=110433&amp;pmid=32234222&amp;doi=10.1016/j.plantsci.2020.110433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref060">
<span class="label">60.</span><cite>Kochan E, Balcerczak E, Szymczyk P, Sienkiewicz M, Zielińska-Bliźniewska H, Szymańska G. Abscisic Acid Regulates the 3-Hydroxy-3-methylglutaryl CoA Reductase Gene Promoter and Ginsenoside Production in Panax quinquefolium Hairy Root Cultures. Int J Mol Sci. 2019;20(6):1310. doi: 10.3390/ijms20061310

</cite> [<a href="https://doi.org/10.3390/ijms20061310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6471273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30875925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Abscisic%20Acid%20Regulates%20the%203-Hydroxy-3-methylglutaryl%20CoA%20Reductase%20Gene%20Promoter%20and%20Ginsenoside%20Production%20in%20Panax%20quinquefolium%20Hairy%20Root%20Cultures&amp;author=E%20Kochan&amp;author=E%20Balcerczak&amp;author=P%20Szymczyk&amp;author=M%20Sienkiewicz&amp;author=H%20Zieli%C5%84ska-Bli%C5%BAniewska&amp;volume=20&amp;issue=6&amp;publication_year=2019&amp;pages=1310&amp;pmid=30875925&amp;doi=10.3390/ijms20061310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref061">
<span class="label">61.</span><cite>Yin J, Sun L, Li Y, Xiao J, Wang S, Yang J, et al. Functional identification of BpMYB21 and BpMYB61 transcription factors responding to MeJA and SA in birch triterpenoid synthesis. BMC Plant Biol. 2020;20(1):374. doi: 10.1186/s12870-020-02521-1

</cite> [<a href="https://doi.org/10.1186/s12870-020-02521-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7422618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32787836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol&amp;title=Functional%20identification%20of%20BpMYB21%20and%20BpMYB61%20transcription%20factors%20responding%20to%20MeJA%20and%20SA%20in%20birch%20triterpenoid%20synthesis&amp;author=J%20Yin&amp;author=L%20Sun&amp;author=Y%20Li&amp;author=J%20Xiao&amp;author=S%20Wang&amp;volume=20&amp;issue=1&amp;publication_year=2020&amp;pages=374&amp;pmid=32787836&amp;doi=10.1186/s12870-020-02521-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330439.ref062">
<span class="label">62.</span><cite>Li YP, Yu CX, Qiao J, Zang YM, Xiang Y, Ren GX. Effect of exogenous phytohormones treatment on glycyrrhizic acid accumulation and preliminary exploration of the chemical control network based on glycyrrhizic acid in root of Glycyrrhiza uralensis. Rev Bras Farmacogn. 2016;26(4):490–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Rev%20Bras%20Farmacogn&amp;title=Effect%20of%20exogenous%20phytohormones%20treatment%20on%20glycyrrhizic%20acid%20accumulation%20and%20preliminary%20exploration%20of%20the%20chemical%20control%20network%20based%20on%20glycyrrhizic%20acid%20in%20root%20of%20Glycyrrhiza%20uralensis&amp;author=YP%20Li&amp;author=CX%20Yu&amp;author=J%20Qiao&amp;author=YM%20Zang&amp;author=Y%20Xiang&amp;volume=26&amp;issue=4&amp;publication_year=2016&amp;pages=490-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 File. Supplementary figures.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s001.docx</a><sup> (11.1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Table. Details of HMGR corrected sequences used in this study.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s002.xlsx</a><sup> (26.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S2 Table and S3 Table. Primers for quantitative real-time PCR and vector construction.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s003.docx</a><sup> (17.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S4 Table. Gene ID and Sequences of HMGR in evolution tree.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s004.xlsx</a><sup> (16.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S5 Table. Germination rates of two varieties of wild jujube under salt stress.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367118/bin/pone.0330439.s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330439.s005.xlsx</a><sup> (13.4KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Sequence data that support the findings of this study have been deposited in NCBl project PRJNA974227 (<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA974227/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA974227/</a>) and been stored in figshare, an online data repository, at <a href="https://figshare.com/s/ad5d747ccc2ccbb2b65b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/s/ad5d747ccc2ccbb2b65b</a>. Corrected sequence data is provided within the Supplementary Table 1. The datasets generated during and/or analysed during the current study are fully available without restriction.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330439"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330439.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367118/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367118/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367118%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367118/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367118/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367118/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40833966/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367118/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40833966/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367118/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367118/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="PqCxdJZbNIzaRKOoByDOHjXuVGlex4VPJw1JMJmJifjUnqmge16hjF1HqiIQbHPm">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
