
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Discovery of YTHDF2 Ligands by Fragment-Based Design - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5310448AEEFB13051044001BA7ADED.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371494/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="Discovery of YTHDF2 Ligands by Fragment-Based Design">
<meta name="citation_author" content="Annalisa Invernizzi">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="Francesco Nai">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="Rajiv Kumar Bedi">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="Pablo Andrés Vargas-Rosales">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="Yaozong Li">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="Elena Bochenkova">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="Marcin Herok">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="František Zálešák">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_author" content="Amedeo Caflisch">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Zurich, CH-8057 Zurich, Switzerland">
<meta name="citation_publication_date" content="2025 Jun 27">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="753">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.5c00099">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371494/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371494/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371494/pdf/bg5c00099.pdf">
<meta name="description" content="N 6-Adenosine methylation is the most abundant modification of mRNA. The three members of the YTH domain family proteins (YTHDF1–3) recognize in the cytoplasm the m6A-RNA modification. We screened a library of about 500,000 fragments (i.e., ...">
<meta name="og:title" content="Discovery of YTHDF2 Ligands by Fragment-Based Design">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="N 6-Adenosine methylation is the most abundant modification of mRNA. The three members of the YTH domain family proteins (YTHDF1–3) recognize in the cytoplasm the m6A-RNA modification. We screened a library of about 500,000 fragments (i.e., ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371494/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371494">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.5c00099"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg5c00099.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371494%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371494/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371494/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371494/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Jun 27;5(4):753–765. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.5c00099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.5c00099</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Discovery
of YTHDF2 Ligands by Fragment-Based Design</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Invernizzi%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Annalisa Invernizzi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Annalisa Invernizzi</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Invernizzi%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Annalisa Invernizzi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nai%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Francesco Nai</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Francesco Nai</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nai%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Francesco Nai</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bedi%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Rajiv Kumar Bedi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Rajiv Kumar Bedi</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bedi%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Rajiv Kumar Bedi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vargas-Rosales%20PA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Pablo Andrés Vargas-Rosales</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Pablo Andrés Vargas-Rosales</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vargas-Rosales%20PA%22%5BAuthor%5D" class="usa-link"><span class="name western">Pablo Andrés Vargas-Rosales</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yaozong Li</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yaozong Li</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yaozong Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bochenkova%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Elena Bochenkova</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Elena Bochenkova</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bochenkova%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elena Bochenkova</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Herok%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Marcin Herok</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Marcin Herok</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Herok%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marcin Herok</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Z%C3%A1le%C5%A1%C3%A1k%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">František Zálešák</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">František Zálešák</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Z%C3%A1le%C5%A1%C3%A1k%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">František Zálešák</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Caflisch%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Amedeo Caflisch</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Amedeo Caflisch</span></h3>
<div class="p">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Caflisch%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Amedeo Caflisch</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
Department of Biochemistry, University of Zurich, CH-8057
Zurich, Switzerland
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>caflisch@bioc.uzh.ch</span>. Phone: +41 44 635 55 21.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 15; Accepted 2025 Jun 4; Revised 2025 Jun 4; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371494</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>
<em>N</em>
<sup>6</sup>-Adenosine methylation
is the most
abundant modification of mRNA. The three members of the YTH domain
family proteins (YTHDF1–3) recognize in the cytoplasm the m<sup>6</sup>A-RNA modification. We screened a library of about 500,000
fragments (i.e., molecules with 11–20 non-hydrogen atoms) by
docking into YTHDF2, which resulted in the identification of six active
compounds among 47 tested in vitro (hit rate of 13%). The acquisition
of 28 analogues of the docking hits provided an additional set of
10 active compounds (IC<sub>50</sub> &lt; 100 μM). Protein crystallography-guided
optimization of a ligand-efficient fragment by the synthesis of 32
derivatives culminated in a series of YTHDF2 ligands, which show low-micromolar
affinity measured by a fluorescence polarization (FP) assay and a
homogeneous time-resolved fluorescence-based (HTRF) assay. The series
is characterized by very favorable ligand efficiency (of about 0.3–0.4
kcal/mol per non-hydrogen atom). Compound <strong>23</strong> binds to
YTHDF2 according to the FP and HTRF assays with a <em>K</em>
<sub>d</sub> value of 1.3 μM and an IC<sub>50</sub> value of
11 μM, respectively, and it is selective against all of the
other YTH reader proteins. Several compounds of the series bind to
the three YTHDF proteins with similar low-micromolar affinity, while
they are less potent for YTHDC1 and YTHDC2. In contrast, compounds <strong>17</strong> and <strong>30</strong> bind also to YTHDC2, with <em>K</em>
<sub>d</sub> of 6.3 and 4.9 μM, respectively. We also disclose
six crystal structures of YTHDF2 in the complex with the fragments
identified by docking.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> epitranscriptomics, m<sup>6</sup>A readers, docking, FRET binding assay, molecular dynamics, protein crystallography</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371494_bg5c00099_0012.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/0375c344f284/bg5c00099_0012.jpg" loading="lazy" height="265" width="646" alt="graphic file with name bg5c00099_0012.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371494_bg5c00099_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/e9801c96ac5f/bg5c00099_0010.jpg" loading="lazy" id="tgr1" height="265" width="646" alt="graphic file with name bg5c00099_0010.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>The
YTH (YT521-B homology) domain-containing proteins are a family
of RNA-binding proteins that specifically recognize <em>N</em>
<sup>6</sup>-methyladenosine (m<sup>6</sup>A), the most abundant
internal modification in eukaryotic RNA. These proteins play critical roles in various biological processes,
including mRNA metabolism, splicing, stability, and translation, influencing
gene expression and cellular functions.<em>
−

</em> The YTH family consists
of five members: YTHDF1 (from now on termed DF1), YTHDF2 (DF2), YTHDF3
(DF3), YTHDC1 (DC1), and YTHDC2 (DC2). DC1 is primarily nuclear, where it participates in mRNA splicing,
processing, and export. In contrast, the
DF proteins (DF1, DF2, and DF3) are mainly cytoplasmic and play essential
roles in mRNA translation, stability, and degradation. Finally, DC2 is the latest discovered member of this protein
family and is mostly cytoplasmic, even though it has been found to
interact with nuclear components, suggesting a dual role in RNA processing
and regulation.<em>
,
</em> Its functions appear to overlap
with those of DF proteins, particularly in regulating RNA translation
and stability. While other family members are broadly expressed across
various cell types, DC2 is notably enriched in the testes, where it
plays a critical role in germ cell development and maturation.
</p>
<p>While it has been established that each
YTH protein possesses unique
functions, there is evidence indicating that they can compensate for
one another under certain conditions, leading to functional redundancy.<em>
,,
</em> Research has demonstrated that
DF proteins can engage in context-dependent functional compensation.
For instance, when one DF protein is knocked down, the others can
partially compensate for the loss, maintaining the overall regulation
of mRNA metabolism.<em>
,
</em> This phenomenon highlights the
complexity of the YTH protein family, where the precise roles of individual
members may vary depending on the cellular context, the specific mRNA
targets, and the presence of other regulatory factors.<em>
,,
</em>
</p>
<p>Given their crucial role
in gene expression regulation, it is not
surprising that YTH proteins and m<sup>6</sup>A regulation are heavily
implicated in various diseases, especially cancer. Our study focuses
on DF2, which is involved in multiple types of cancer, including prostate
cancer, MYC-driven breast cancer, and acute myeloid leukemia (AML). This makes DF2 a highly attractive target for
drug discovery and is gaining more and more attention. Even though
we focus mainly on DF2, the highly conserved m<sup>6</sup>A binding
site of the DF proteins hinders the development of a DF2-selective
ligand. Furthermore, the discussed compensatory
effects make a pan-DF ligand desirable.</p>
<p>Only a few small-molecule
ligands have been identified for the
DF protein family. Among them are Ebselen, Tegaserod, and Salvianolic Acid C, previously known compounds repurposed from other
targets. Reviews of known inhibitors of the YTH proteins can be found
in refs  and . In our earlier publication, we reported the first small-molecule binders
of DF2; the X-ray crystal structure of one of them (compound <strong>11</strong>, IC<sub>50</sub> = 174 μM) is the starting point of this work. Subsequently, Wang et al. reported
the discovery of DC-Y13–27, a DF2 inhibitor with an IC<sub>50</sub> of 21.8 μM (measured using an AlphaScreen assay) and
a <em>K</em>
<sub>d</sub> of 38 μM (determined by microscale
thermophoresis). The compound showed
weaker activity on DF1 (IC<sub>50</sub> = 165 μM in the AlphaScreen
assay), but was not tested on DF3. A
more recent study identified a series of functionalized pyrazoles
as selective DF2 binders. The most potent
compound, CK-75, exhibited an IC<sub>50</sub> of 13.2 μM in
an AlphaScreen assay and was found to be inactive against all other
members of the YTH protein family. Notably, CK-75 induced cell cycle
arrest and apoptosis in the K567 leukemia cell line, further supporting
DF2 as a promising therapeutic target.
</p>
<p>Here we present a new series of DF2 binders identified by
docking,
followed by structure–activity relationship (SAR)-by-catalog.
We discovered new chemotypes that compete with m<sup>6</sup>A-RNA
for binding to DF2. Medicinal chemistry optimization of a ligand-efficient
scaffold resulted in a series of low-micromolar binders of DF2. Most
compounds of the series show a preference for DF proteins against
DC1 and DC2. Compound <strong>23</strong> binds only to DF2, being selective
against all of the other YTH proteins.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results and Discussion</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Docking
of a Library of Fragments</h3>
<p>The chemical diversity
represented by a library of <em>N</em> fragments (with up
to 18–20 non-hydrogen atoms) is substantially larger than the
diversity of a library of <em>N</em> molecules with more than
25–30 heavy atoms. As a consequence,
fragment screening by docking is an efficient alternative to the screening
of ultralarge libraries of compounds. Thus, we decided to start with a fragment docking campaign in DF2.
A library of 500,000 fragments was docked using the program SEED.<em>
,
</em> The fragments were selected from the ZINC20 database according to the following rules: between 11
and 20 heavy atoms, at least one ring, and at least one sp<sup>3</sup>-hybridized carbon. The first two rules reflect the properties of
the program SEED, which was developed for docking mainly rigid fragments.
The third rule was selected as our first campaign had identified ligand
fragments with methyl, ethyl, or cyclopropyl in the bottom of the
aromatic pocket. For each of the extracted
compounds, up to 20 conformers were generated using a distance geometry-based
algorithm, and docked by SEED. Two structures
of the m<sup>6</sup>A-RNA recognition domain of DF2 were used for
docking (<a href="#fig1" class="usa-link">Figure 
</a>).
The crystal structure in the complex with 6-cyclopropylpyrimidine-2,4-diol
(compound <strong>11</strong> in ref  PDB ID: <a href="https://www.rcsb.org/structure/7R5W" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7R5W</a>) and a snapshot obtained by molecular
dynamics (MD) simulations started from the same crystal structure.
The MD snapshot was selected by analysis of the time series of the
volume of the recognition pocket. It has a more open recognition loop with a pocket volume of 600 Å<sup>3</sup>, which is significantly larger than the volume of 324 Å<sup>3</sup> in the crystal structure <a href="https://www.rcsb.org/structure/7R5W" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7R5W</a> (<a href="#fig1" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/b4c498f1e5f6/bg5c00099_0001.jpg" loading="lazy" id="gr4" height="1459" width="750" alt="1"></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Two structures of the DF2 reader employed for docking.
(A) X-ray
crystal structure of DF2 in the complex with 6-cyclopropylpyrimidine-2,4-diol
(compound <strong>11</strong> of ref  PDB: <a href="https://www.rcsb.org/structure/7R5W" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7R5W</a>). (B) MD-simulation snapshot with a larger aperture
of the m<sup>6</sup>A-recognition pocket (see <a href="#sec4" class="usa-link">Materials
and Methods</a>). The surface of DF2 is colored by the electrostatic
potential (red, negative; blue, positive), and 6-cyclopropylpyrimidine-2,4-diol
(carbon atoms in green) and the binding site residues are shown by
sticks (carbon atoms in yellow).</p></figcaption></figure><p>The two protein structures were kept rigid during
docking and evaluation
of the binding energy. SEED calculates the binding energy by a force
field with implicit treatment of the electrostatic effects of the
solvent. The docked compounds were ranked according to two energy
terms, namely, the total binding energy and the difference between
the electrostatic contribution to the intermolecular interaction energy
in the solvent and the solvation energy of the ligand. The top-scoring
compounds were then selected if they showed the crucial hydrogen bond
with the backbone carbonyl of C433, which is the acceptor of the <em>N</em>
<sup>6</sup> of the natural ligand m<sup>6</sup>A, and
another polar interaction within the binding site (see <a href="#sec4" class="usa-link">Materials and Methods</a>). If an interesting compound was not
commercially available, then a structurally similar analogue was chosen.
Finally, 25 and 22 compounds were selected from the docking campaigns
that made use of the crystal structure and MD snapshot, respectively
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>).</p></section><section id="sec2.2"><h3 class="pmc_sec_title">In Vitro Validation</h3>
<p>A previously reported homogeneous
time-resolved fluorescence (HTRF)-based assay was used to measure
the binding affinity of the 47 ordered compounds (see <a href="#sec4" class="usa-link">Materials and Methods</a>). For the
nine compounds with residual signal at 1 mM smaller than 60% (with
respect to dimethyl sulfoxide (DMSO) control), the IC<sub>50</sub> value was determined by dose–response experiments (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Among these, the thiobarbiturate derivatives <strong>1</strong> and <strong>2</strong> were the strongest binders of DF2, with
IC<sub>50</sub> values of 19 and 170 μM, respectively (<a href="#tbl1" class="usa-link">Table 
</a>). The thiobarbiturate
derivative <strong>1</strong> shows a very favorable ligand efficiency
(LE) of 0.50 kcal/(mol HAC) [HAC = heavy atoms count, i.e., number
of non-hydrogen atoms] while the toluene group of compound <strong>2</strong> does not seem to contribute to binding.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. m<sup>6</sup>-Adenine and 16 Ligands
of DF2 Identified by Docking, Followed by SAR by Catalogue<sup></sup>
.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371494_bg5c00099_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/721458c329de/bg5c00099_0007.jpg" loading="lazy" id="fx1" height="1302" width="760" alt="graphic file with name bg5c00099_0007.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">The NH group interacting as a hydrogen-bond
donor with the backbone carbonyl of C433 (blue) and the group in the
bottom of the tryptophan cage (red) are emphasized for the compounds
with a crystal structure in the complex with DF2.</p>
</div>
<div class="fn" id="t1fn2">
<sup>b</sup><p class="display-inline">The residual signal at 1 mM compound
concentration is measured using an HTRF-based binding assay as previously
reported. The signal decreases (with
respect to buffer-only measurement) when the fragment competes with
the binding of the natural ligand, i.e., m<sup>6</sup>A-oligoRNA.
Thus, the lower the signal, the higher the affinity of the fragment.
The reported values are the average of two technical replicates.</p>
</div>
<div class="fn" id="t1fn3">
<sup>c</sup><p class="display-inline">We use the term IC<sub>50</sub> for
the concentration of the compound that reduces the signal by 50% with
respect to buffer-only. YTH readers are not enzymes, but we still
prefer the more frequently used inhibitory concentration (IC<sub>50</sub>) rather than effective concentration. The IC<sub>50</sub> value
for the DF2 reader domain was measured only for the fragments that,
at a concentration of 1 mM, decrease the signal by more than 60%.</p>
</div>
<div class="fn" id="t1fn4">
<sup>d</sup><p class="display-inline">The ligand efficiency is calculated
as <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e523" display="inline"><mi>LE</mi><mo>=</mo><mo form="prefix">−</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><mi>G</mi></mrow><mrow><mi mathvariant="normal">H</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">C</mi></mrow></mfrac><mo>≈</mo><mo form="prefix">−</mo><mi mathvariant="italic">RT</mi><mfrac><mrow><mi>ln</mi><mo linebreak="nobreak" lspace="0em" rspace="0.25em">⁡</mo><msub><mrow><mi mathvariant="normal">I</mi><mi mathvariant="normal">C</mi></mrow><mn>50</mn></msub></mrow><mrow><mi mathvariant="normal">H</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">C</mi></mrow></mfrac></math>
</span> and the values are reported in kcal/(mol
HAC), HAC = heavy atom count.</p>
</div>
<div class="fn" id="t1fn5">
<sup>e</sup><p class="display-inline">Compounds <strong>1</strong>–<strong>6</strong> were identified by
docking, while compounds <strong>7</strong>–<strong>16</strong> were
selected by SAR by catalogue. * Interference
or poor solubility observed at 1 mM. <sup>#</sup> Interference or
poor solubility observed at higher concentrations, IC<sub>50</sub> could not be determined.</p>
</div>
</div></section><p>The X-ray crystal structure was solved for compounds <strong>3</strong>–<strong>6</strong> at high resolution (<a href="#tbl1" class="usa-link">Table 
</a> and <a href="#fig2" class="usa-link">Figure 
</a>B–E). The unique binding mode of compound <strong>3</strong> provided evidence that the ene-thiobarbiturate substructure
of compounds <strong>1</strong>–<strong>3</strong> specifically and
noncovalently binds to DF2 despite the potential Michael acceptor
reactivity.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371494_bg5c00099_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/71c389de710d/bg5c00099_0002.jpg" loading="lazy" id="gr5" height="1221" width="646" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Crystal structures of DF2/fragment complexes. (A–G) Binding
modes of m<sup>6</sup>-adenine (compound <strong>1</strong> in ref , PDB: <a href="https://www.rcsb.org/structure/7YWB" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7YWB</a>), compounds <strong>3</strong> (PDB: <a href="https://www.rcsb.org/structure/9QEM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QEM</a>), <strong>4</strong> (PDB: <a href="https://www.rcsb.org/structure/9QEL" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QEL</a>), <strong>5</strong> (PDB: <a href="https://www.rcsb.org/structure/9QEO" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QEO</a>), <strong>6</strong> (PDB: <a href="https://www.rcsb.org/structure/9QFI" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QFI</a>), <strong>13</strong> (PDB: <a href="https://www.rcsb.org/structure/9QIU" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QIU</a>), and <strong>15</strong> (PDB: <a href="https://www.rcsb.org/structure/9QFL" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QFL</a>), respectively. The conserved water molecule
(red sphere) and the hydrogen bonds (yellow dashed lines) are also
shown. (H) Structural overlap of panels (A)–(G). The carbon
atoms of the ligands are colored green and those of the protein in
cyan.</p></figcaption></figure><p>The six binders <strong>1</strong>–<strong>6</strong> (<a href="#tbl1" class="usa-link">Table 
</a>) belong
to four distinct
chemotypes: thiobarbiturates (<strong>1</strong>–<strong>3</strong>), uracil (<strong>4</strong>), nicotinamide (<strong>5</strong>), and pyrazolopyrimidine
(<strong>6</strong>). Despite their chemical diversity, these chemotypes
feature an NH group that is involved as a hydrogen-bond donor with
the backbone carbonyl of C433 (<a href="#fig2" class="usa-link">Figure 
</a>B–E). Furthermore, they act as hydrogen-bond
acceptor of the conserved water that forms hydrogen bonds with the
side chains of W432 and D528. Notably, compounds <strong>1</strong>–<strong>3</strong> occupy the bottom of the aromatic cage with their sulfur
atom, while compounds <strong>4</strong>–<strong>6</strong> have a
methyl group as in the natural ligand. Compounds <strong>1</strong>–<strong>4</strong> and <strong>6</strong> act as hydrogen-bond acceptors for the
backbone NH of Y418, and they also form a hydrogen bond with the side
chain of D422. This interaction may offer selectivity against the
nuclear reader DC1, which features the N367 hydrogen-bond donor NH<sub>2</sub> in this position. In our first
screening campaign, we described a series
of uracil analogues; compound <strong>4</strong> is a new member of this
series with a modest IC<sub>50</sub> value of 250 μM. The amide
nitrogen atom of compound <strong>5</strong> is involved in a hydrogen
bond with the side chain of D528, but it does not form favorable interactions
with D422 and the backbone NH of Y418. The 70° relative orientation
and 4.8 Å distance between the ring centers indicate that the
aromatic side chain of Y418 engages in an edge-to-face π-stacking
interaction with <strong>5</strong>.<em>
,
</em>
</p></section><section id="sec2.3"><h3 class="pmc_sec_title">SAR by Catalogue</h3>
<p>From the information gained in the
first screening, a second set of 28 compounds was ordered (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>). This set consisted of 14 top-ranking
docked molecules with a chemotype similar to that of compounds <strong>1–3</strong>, and other 14 molecules that are closely related
to the discovered binders <strong>1</strong>–<strong>6</strong> but
were not present in the library used for docking. A total of 16 of
the ordered compounds belong to the thiobarbiturate chemotype, which
was considered very promising from the previous results. Ten of the
28 compounds (ligands <strong>7–16</strong>) showed an IC<sub>50</sub> &lt; 100 μM.</p>
<p>Among the thiobarbiturate derivatives <strong>7–10</strong>, ligand <strong>8</strong> is the most potent (IC<sub>50</sub> = 6 μM and LE = 0.36). The carboxylic acid substituent
is likely to contribute to the binding affinity, as the solvent-exposed
portion of the DF2 binding pocket is rich in positively charged residues
(K416, K490, and R527; <a href="#fig2" class="usa-link">Figure 
</a>H), which typically interact with the negatively charged phosphates
of RNA, its natural substrate.</p>
<p>Other molecules closely related
to the thiobarbiturates were identified
as interesting binders: barbiturates <strong>11</strong> and <strong>12</strong>, triazine <strong>13</strong>, and condensed bicycles <strong>14</strong>–<strong>16</strong>. The X-ray crystal structures of compounds <strong>13</strong> and <strong>15</strong> confirm their binding (<a href="#fig2" class="usa-link">Figure 
</a>F,G), again showing the sulfur
atom interacting within the lipophilic tryptophan cage.</p>
<p>Compound <strong>15</strong> can establish all of the interactions
previously discussed: in addition to the interactions of m<sup>6</sup>-adenine, it can interact with both D422 and D528. (Note though that
m<sup>6</sup>-adenine is likely to be protonated in the bound state
as suggested by the distance of 2.6 Å between its N1 atom and
the nearest O atom of the D422 side chain (PDB entry <a href="https://www.rcsb.org/structure/7YWB" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7YWB</a>).) Unfortunately,
the multiple interactions of compound <strong>15</strong> do not translate
into a very strong binding (IC<sub>50</sub> = 86 μM). This might
be a consequence of the major tautomer represented in <a href="#tbl1" class="usa-link">Table 
</a> being in equilibrium with other
tautomeric forms, which are not ideal for efficient binding (∼67%
of the desired tautomer, as calculated with the Chemaxon tautomers
generator plugin <a href="https://plugins.calculators.cxn.io/tautomers/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plugins.calculators.cxn.io/tautomers/</a>). Compounds <strong>13</strong> and <strong>14</strong> are characterized
by very favorable LE values of 0.73 and 0.60, respectively. We decided
to further explore the derivatization of compound <strong>14</strong> because
of its favorable LE, the two possible vectors for substitution at
the two carbon atoms of its thiophene ring, respectively, and the
relatively accessible synthesis (see below). We could not determine
the crystal structure of DF2 in complex with <strong>14</strong>. The
choice of compound <strong>14</strong> was supported by its pose predicted
by docking, which overlapped with the binding mode of compound <strong>15</strong> in the high-resolution crystal structure with DF2 (PDB
entry <a href="https://www.rcsb.org/structure/9QFL" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QFL</a>).</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Derivatization of Compound <strong>14</strong>
</h3>
<p>The exploration
of analogues of <strong>14</strong> has been pursued by synthesizing new
derivatives (<a href="#tbl2" class="usa-link">Tables 
</a> and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S4</a>) and ordering commercially available
variants (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>). The replacement of
the exocyclic sulfur atom with oxygen was detrimental to the potency,
from 18 to 290 μM (compound <strong>S61</strong>). Several 5- and
6-membered ring alternatives to the thiophene were explored (compounds <strong>15</strong>, <strong>16</strong>, <strong>S5</strong>, <strong>S62–S64</strong>), and only compound <strong>16</strong> resulted in an IC<sub>50</sub> comparable to that of fragment <strong>14</strong> (30 vs 18 μM).</p>
<section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. Expansion of Hit Fragment <strong>14</strong>
<sup></sup>
.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/a98b12abf5a1/bg5c00099_0008.jpg" loading="lazy" id="fx2" height="344" width="489" alt="graphic file with name bg5c00099_0008.jpg"></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/daa65c53e719/bg5c00099_0009.jpg" loading="lazy" id="fx3" height="4700" width="3465" alt="graphic file with name bg5c00099_0009.jpg"></p>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="t2fn1">
<sup>a</sup><p class="display-inline">For compounds with
a single value,
it refers to DF2. The dose–response curves and replicates are
shown in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>.</p>
</div>
<div class="fn" id="t2fn2">
<sup>b</sup><p class="display-inline">The LE formula is shown in the caption
of <a href="#tbl1" class="usa-link">Table 
</a>.</p>
</div>
<div class="fn" id="t2fn3">
<sup>c</sup><p class="display-inline">The <em>K</em>
<sub>d</sub> values
were determined with the Cheng–Prusoff equation using
a <em>K</em>
<sub>d</sub> value of 5 nM (DF2), 7 nM (DC2),
and 5 nM (DC1) for the 5′-fluorescein-labeled m<sup>6</sup>A-DNA probe. The reported values are the averages of one (DC1) or
two (DF2 and DC2) biological replicates, and each replicate is the
average of four technical replicates. The dose–response curves
are in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S2</a> (DF2), <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S3</a> (DC2), and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S4</a> (DC1).</p>
</div>
<div class="fn" id="t2fn4">
<sup>d</sup><p class="display-inline">Compounds <strong>17</strong>–<strong>31</strong> are more active for DF2 than compound <strong>14</strong> in
the HTRF and/or the FP assays.</p>
</div>
</div></section><p>The medicinal chemistry campaign focused on derivatizing
the thiophene
ring of <strong>14</strong> through the addition of R<sup>1</sup> and/or
R<sup>2</sup> groups (<a href="#tbl2" class="usa-link">Table 
</a> and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a>). A total of 32 molecules
were synthesized (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a>). <a href="#tbl2" class="usa-link">Table 
</a> shows the molecules with higher
affinity than compound <strong>14</strong> as measured by the HTRF-based
binding assay and/or FP. As mentioned above, we were not able to determine
the crystal structure of <strong>14</strong> in complex with DF2. Thus
we hypothesized a similar binding mode as in the crystal structure
with compound <strong>15</strong> (<a href="#fig3" class="usa-link">Figure 
</a>A). The putative binding poses will be further analyzed
and discussed in the next section.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371494_bg5c00099_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/051d8f7d0b7f/bg5c00099_0003.jpg" loading="lazy" id="gr6" height="350" width="696" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predicted binding modes of compounds <strong>14</strong>, <strong>17</strong>, <strong>23</strong>, and <strong>27</strong> in
DF2. (A) The pseudosymmetry
of fragment 14 (carbon atoms in gray) results in two binding modes
called here A (top) and B (bottom). The crystal structure of DF2 (cyan,
PDB: <a href="https://www.rcsb.org/structure/9QFL" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9QFL</a>) in
complex with fragment 15 (carbon atoms in green) is overlapped for
comparison. (B–D) The most populated pose of compounds 17 (pose
B), 23 (pose A), and 27 (pose A). Their alternative poses are shown
in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>. (E) Analysis of the MD simulations
started from the two potential poses (pose A, left; pose B, right).
The time series shows the median ligand root-mean-square deviation
(RMSD) with respect to the first frame, with a colored band corresponding
to one median absolute deviation around the median. (F) Principal
component analysis (PCA) projections of the distances between atoms
of the ligand and representative residues of the DF2 binding site.
Data for MD snapshots saved every 1 ns are shown for the two poses
(A, cross; B, diamond), and the centers of the two clusters are emphasized
(yellow star).</p></figcaption></figure><p>Compound <strong>S74</strong> (R<sup>2</sup> = phenyl)
and its derivatives
(<strong>17</strong>–<strong>19</strong>, <strong>S75</strong>, and <strong>S76</strong>) were synthesized to try to obtain a favorable π–π
stacking interaction with Y418, as seen in the X-ray crystal structure
of the <em>N</em>-methyl-3-phenyl-1<em>H</em>-pyrazolo­[4,3-<em>d</em>]­pyrimidin-7-amine (compound <strong>7</strong> in ref , PDB: <a href="https://www.rcsb.org/structure/7YXE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7YXE</a>). Among these compounds, <strong>17</strong> and <strong>18</strong> resulted in a 3-fold improved IC<sub>50</sub> in the HTRF-based binding assay (<a href="#tbl2" class="usa-link">Table 
</a>), possibly due to a hydrogen bond between
the carbonyl group and the hydroxyl of Y418 or the −NH<sub>2</sub> of N462. At R<sup>1</sup>, we started with small polar substituents
to try to establish interactions with D528 and/or with the structural
water molecule or to try to replace the water molecule (compounds <strong>20</strong>, <strong>21</strong>, <strong>S79</strong>, and <strong>S80</strong>). The ethyl ester of <strong>21</strong> at R<sup>1</sup> enhanced the
binding (IC<sub>50</sub> = 6.3 μM), as also the bulkier benzyl
(<strong>22</strong>, IC<sub>50</sub> = 10 μM) and methylpyridine
(<strong>23</strong>, IC<sub>50</sub> = 11 μM) groups, both connected
to the thiophene via an amide bond. We hypothesized that the −CH<sub>2</sub>– linker enables the aromatic ring to orient toward
the solvent-exposed region of the pocket, which is enriched with positively
charged residues that facilitate binding to the negatively charged
RNA. Based on this, we explored modifications such as adding a carboxylic
acid (<strong>24</strong>) and further increasing the flexibility by replacing
the benzyl group with an alkyl chain (<strong>25</strong> and <strong>S83</strong>). However, none of these changes led to an improvement in the potency.</p>
<p>We continued the exploration of R<sup>1</sup> with substituted
benzyl and phenyl rings directly connected to the thiophene (compounds <strong>26</strong>–<strong>30</strong> and <strong>S84</strong>–<strong>S87</strong>). Compounds <strong>26</strong> and <strong>27</strong> resulted
in an improvement of 3- to 4-fold compared to fragment <strong>14</strong>.</p>
<p>An FP competition assay was used to further validate the
binding
of the compounds to DF2. The main difference with respect to a previously
published FP assay is the use of an oligo-DNA
as competitor ligand (see <a href="#sec4" class="usa-link">Materials and Methods</a> section). For most compounds, there is a factor of 2–5 difference
between the IC<sub>50</sub> values for DF2 measured by HTRF and FP
(<a href="#tbl2" class="usa-link">Table 
</a>). The largest
discrepancy is a factor of 16 for compound <strong>19</strong> (32 and
2 μM by HTRF and FP, respectively; note that <a href="#tbl2" class="usa-link">Table 
</a> shows the <em>K</em>
<sub>d</sub> value for FP, which is equal to the IC<sub>50</sub>/1.6
for DF2). These differences might originate from the varying conditions
in the two assays, such as the competitor mRNA (5′-biotin-AAGAACCGG­(m6A)­CUAAGCU-3′)
in HTRF and DNA (5′-FAM-AAGAACCGG­(m6A)­CTAAGCT-3′) in
FP, the salt concentration (150 mM NaCl and 100 mM KF in HTRF, and
150 mM NaCl in FP), and the pH (7.5 in HTRF and 7.4 in FP). The FP-based
assay was also employed to assess the selectivity against DC1 and
DC2 (see <a href="#sec2.6" class="usa-link">Selectivity</a> section).</p></section><section id="sec2.5"><h3 class="pmc_sec_title">Computational
Analysis</h3>
<p>We could not determine the crystal
structure of the complex of DF2 with compound <strong>14</strong> or any
of its derivatives by soaking the ligands into apo DF2 crystals or
cocrystallization. Thus, we decided to run MD simulations to investigate
the binding mode of <strong>14</strong> and its derivatives <strong>17</strong>, <strong>23</strong>, and <strong>27</strong> (<a href="#fig3" class="usa-link">Figure 
</a>). As already observed during the docking
campaign, the symmetry of the thiourea substructure is congruent with
two distinct poses (A and B), which are flipped by a rotation of 180°
around the SC double bond. For each compound and pose, eight
independent 0.2-μs MD simulations were performed for a cumulative
sampling of 3.2 μs per compound, starting from the docked poses
of compound <strong>14</strong>, or the alignment of the derivatives <strong>17</strong>, <strong>23</strong>, and <strong>27</strong> to it.</p>
<p>The
analysis of the MD trajectories was carried out by adapting the protocol
described in ref  (see <a href="#sec4" class="usa-link">Materials and Methods</a>). Only the second
half of each MD run, i.e., the trajectory segments from 100 to 200
ns, was used for the analysis to allow for sufficient ligand relaxation.
From these segments, we extracted the simulation frames where the
compound is bound, defined as having a distance lower than 5 Å
between the exocyclic sulfur atom and the N atom of C433. Then we
calculated a set of 10 distances between the heavy atoms of the compounds
(all located in the scaffold, i.e., present in compound <strong>14</strong>) and the binding pocket. Finally, principal component analysis (PCA)
was used to project the multidimensional space into two dimensions.
The two-dimensional data were then clustered using the Gaussian Mixture
algorithm. The trajectory frame closest to the center of each cluster
(centroid) was extracted and used as the reference pose (<a href="#fig3" class="usa-link">Figure 
</a>F).</p>
<p>
<a href="#fig3" class="usa-link">Figure 
</a>A shows
the two reference poses obtained for compound <strong>14</strong> compared
to the crystal structure of compound <strong>15</strong>. In both poses,
the exocyclic sulfur atom is positioned within the lipophilic tryptophan
cage. As mentioned above, compound <strong>14</strong> can adopt two distinct
orientations, which are related by a 180-degree rotation. This flipping
rearranges the hydrogen-bond interactions while preserving the same
number of favorable polar contacts. In both poses, the two NH groups
of the thiourea act as hydrogen-bond donors for the backbone carbonyl
of C433 and the side chain of D422, respectively. Moreover, the carbonyl
group and the thiophene sulfur atom are inverted between the two poses.
In pose A, the carbonyl oxygen interacts with the backbone NH of Y418,
as observed for the crystal structure with compound <strong>15</strong> (<a href="#fig2" class="usa-link">Figures 
</a>G and A top). In pose B, it instead forms a hydrogen bond
with the conserved water molecule (<a href="#fig3" class="usa-link">Figure 
</a>A, bottom). The sulfur atom in the thiophene
ring points toward the structural water in pose A and the backbone
NH of Y418 in pose B. In both poses, it acts as a weak hydrogen-bond
acceptor.<em>
−

</em> The root-mean-square deviation (RMSD) analysis of
the MD runs started from the two poses suggests that pose A is slightly
more stable (<a href="#fig3" class="usa-link">Figure 
</a>E, top).</p>
<p>A similar analysis was performed for compounds <strong>17</strong>, <strong>23</strong>, and <strong>27</strong> to investigate the
impact of bulky
R<sup>1</sup> and/or R<sup>2</sup> substituents on the stability of
the two poses. For compound <strong>17</strong>, a smaller RMSD is observed
for the B pose in comparison to A, and the opposite was observed for
compound <strong>23</strong> (<a href="#fig3" class="usa-link">Figure 
</a>E). A smaller difference emerges for compound <strong>27</strong>, with pose A being slightly more stable.</p>
<p>We also
calculated the population of each cluster (<a href="#fig3" class="usa-link">Figure 
</a>F). As expected, the most populated
cluster corresponds to the most stable pose based on RMSD analysis.
Specifically, 55% of the bound frames of compound <strong>14</strong> belong
to the cluster of pose A (<a href="#fig3" class="usa-link">Figure 
</a>A, top); 61% of the bound frames of compound <strong>17</strong> to pose B (<a href="#fig3" class="usa-link">Figure 
</a>B); 56% of compound <strong>23</strong> to pose A (<a href="#fig3" class="usa-link">Figure 
</a>C); and 53% of compound <strong>27</strong> to pose A (<a href="#fig3" class="usa-link">Figure 
</a>D).</p></section><section id="sec2.6"><h3 class="pmc_sec_title">Selectivity</h3>
<p>The HTRF-based assay
was employed to evaluate
the selectivity of the series against other DF proteins. The compounds
were tested at a concentration of 100 μM and most of them also
inhibit DF1 and DF3 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S5</a>).</p>
<p>A
dose–response experiment on DF1 and DF3 was conducted for compounds <strong>17</strong>, <strong>18</strong>, <strong>23</strong>, and <strong>27</strong> (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5</a>). Compounds <strong>17</strong>, <strong>18</strong>, and <strong>27</strong> are active on three DF proteins. Their
IC<sub>50</sub> values for DF1 are approximately twice as high as
those for DF2, while for DF3, they are 4–12 times higher. In
contrast, compound <strong>23</strong> is selective for DF2, with residual
signals (at the highest employed concentration of 62.5 μM) of
51 and 87% for DF1 and DF3, respectively.</p>
<p>The binding affinity
for DC1 could not be evaluated by HTRF because
of interference of the compounds with the assay, which has a smaller
assay window when performed with DC1 compared to the DF proteins.
Thus, the FP assay was used to evaluate binding to DC1 and DC2. The
affinity of the compounds for DC2 varied significantly depending on
the substituents, with <em>K</em>
<sub>d</sub> values ranging
from 5 μM for compound <strong>30</strong> to over 400 μM
for compounds <strong>23</strong>, <strong>S74</strong>, <strong>S78</strong>, <strong>S79</strong>, <strong>S82</strong>, <strong>S83</strong>, and <strong>S89</strong> (<a href="#tbl2" class="usa-link">Tables 
</a> and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S4</a>).</p>
<p>The parent scaffold, compound <strong>14</strong>, exhibits approximately
a 20-fold and 8-fold higher binding affinity for DF2 compared to DC2
and DC1, with <em>K</em>
<sub>d</sub> values of 3.8, 88, and
32 μM, respectively, as measured by FP (<a href="#tbl2" class="usa-link">Table 
</a>). Many derivatives maintain a strong preference
for DF2. Notably, compound <strong>23</strong> shows high selectivity,
being about 350 and 100 times more potent for DF2 than DC2 and DC1,
respectively. MD simulations of compound <strong>23</strong> in complex
with DC1 were performed following the same strategy described for
DF2 (see the <a href="#sec2.5" class="usa-link">Computational Analysis</a> section).
The RMSD analysis (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>) indicates
a higher stability (lower RMSD) of both poses A and B in DF2 compared
to DC1. These results are consistent with the selectivity of compound <strong>23</strong> for DF2 and against DC1.</p>
<p>In contrast, compounds with
a substituted phenyl ring directly
connected to the thiophene (R<sup>1</sup> or R<sup>2</sup>) exhibited
little to no selectivity against the DC proteins (e.g., ratio <em>K</em>
<sub>d</sub> DC1/DF2 and DC2/DF2 of only 2–4 for
ligands <strong>29</strong> and <strong>30</strong>).</p>
<p>Compounds <strong>17</strong>, <strong>18</strong>, and <strong>28</strong>–<strong>30</strong> are the most potent DC binders of the series.
They feature benzoic acid (at R<sup>2</sup> in compound <strong>18</strong> and R<sup>1</sup> in compound <strong>30</strong>), benzoic ester (at
R<sup>2</sup> in compound <strong>17</strong> and R<sup>1</sup> in compound <strong>29</strong>), or para-methoxyphenyl (at R<sup>1</sup> in compound <strong>28</strong>). Their similar behavior indicates that the previously
discussed two poses (A and B, see previous section) may also be populated
in the DC1 and DC2 binding sites. These results suggest that the substituent,
at either R<sup>1</sup> or R<sup>2</sup>, likely occupies the same
region of the binding pocket. To the best of our knowledge, the only
previously identified DC2-binder in the literature is the pan-YTH
binder “N-7”, with a reported IC<sub>50</sub> of 30
μM (as measured by FP). Thus, compounds <strong>17</strong> and <strong>30</strong> are the most potent DC2 ligands as
of today (<em>K</em>
<sub>d</sub> values of 6.3 and 4.9 μM,
respectively, measured by FP).</p></section><section id="sec2.7"><h3 class="pmc_sec_title">Chemistry</h3>
<p>The
general synthetic approach begins with
the synthesis of the ethyl 2-aminothiophene-3-carboxylate derivatives
(<strong>37a</strong>–<strong>g</strong>), with the corresponding R<sup>1</sup> and/or R<sup>2</sup> substituents. For compounds <strong>37a</strong>–<strong>e</strong> this was achieved via a classic one-pot Gewald
reaction (<a href="#sch1" class="usa-link">Scheme 
</a>).<em>
,
</em>
</p>
<figure class="fig xbox font-sm" id="sch1"><h4 class="obj_head">1. Synthesis Route for Compounds <strong>37a</strong>–<strong>e</strong>

.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371494_bg5c00099_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/974433debe1a/bg5c00099_0004.jpg" loading="lazy" id="gr1" height="684" width="773" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="s1fn1"><sup>a</sup> Reagents and conditions:
(a)
for <strong>37a</strong>–<strong>d</strong>: S<sub>8</sub>, morpholine,
EtOH, 70 °C; for <strong>37e</strong>: S<sub>8</sub>, morpholine,
room temperature (rt).</p></figcaption></figure><p>For the preparation
of compounds <strong>37g</strong>–<strong>m</strong>, the substituted
phenyl group was added via a Suzuki–Miyaura
coupling from compound <strong>39</strong> (<a href="#sch2" class="usa-link">Scheme 
</a>), prepared by bromination and protection
of compound <strong>38</strong>.<em>
,
</em> Finally, the amino
group was deprotected, affording ethyl 2-aminothiophene-3-carboxylate
intermediates <strong>35g</strong>–<strong>i</strong>.</p>
<figure class="fig xbox font-sm" id="sch2"><h4 class="obj_head">2. Synthesis
Route for Compounds <strong>37f</strong>–<strong>g</strong>

.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371494_bg5c00099_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/79023dd0edac/bg5c00099_0005.jpg" loading="lazy" id="gr2" height="420" width="827" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="s2fn1"><sup>a</sup> Reagents and conditions:
(a)
(i) Boc<sub>2</sub>O, 4-dimethylaminopyridine (DMAP), dioxane, 0–80
°C; (ii) <em>N</em>-bromosuccinimide (NBS), AcOH/dichloromethane
(DCM), −15 °C (b) boronic acid, K<sub>2</sub>PO<sub>3</sub>, Pd­(PPh<sub>3</sub>)<sub>4</sub>, dimethylformamide (DMF)/H<sub>2</sub>O, 80 °C; (c) trifluoroacetic acid (TFA), DCM; (d) HCl
4 N in dioxane, MeOH.</p></figcaption></figure><p>Finally, the synthesized
2-aminothiophene-3-carboxylate intermediates
(<strong>37a</strong>–<strong>g</strong>), and the commercially available
ones (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information (SI)</a>),
were reacted with benzoyl isothiocyanate <strong>42</strong> (<a href="#sch3" class="usa-link">Scheme 
</a>). The ring closure
was then performed in basic conditions under reflux. The synthesis of some final compounds required additional
transformations, including hydrolysis (<strong>18</strong>, <strong>24</strong>, <strong>25</strong>, <strong>30</strong>, <strong>S80</strong>, and <strong>S87</strong>), reduction (<strong>27</strong> and <strong>S79</strong>), and amide coupling
(<strong>22–25</strong>, <strong>S81</strong>, and <strong>S83</strong>). Full experimental details are provided in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a>.</p>
<figure class="fig xbox font-sm" id="sch3"><h4 class="obj_head">3. Synthesis Route for the Bicyclic Final
Compounds
.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a93/12371494/5941eb140c61/bg5c00099_0006.jpg" loading="lazy" id="gr3" height="746" width="3500" alt="3"></p>
<div class="p text-right font-secondary"><a href="figure/sch3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="s3fn1"><sup>a</sup> Reagents and conditions:
(a)
CH<sub>3</sub>CN, 45 °C; (b) EtONa, EtOH, reflux.</p></figcaption></figure></section></section><section id="sec3"><h2 class="pmc_sec_title">Conclusions</h2>
<p>We employed a fragment-based approach for
identifying ligand-efficient
small molecules that occupy the m<sup>6</sup>A-RNA recognition pocket
of the DF2 reader domain. A crystal structure of a previously disclosed
ligand-efficient fragment binder of DF2 (PDB <a href="https://www.rcsb.org/structure/7R5W" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7R5W</a>) and a molecular dynamics snapshot were used for fragment
docking. Each of the two docking campaigns yielded three active compounds
and, thus, an overall hit rate of 13% (6/47). SAR by catalogue and
the synthesis of 32 derivatives of the thioxo-dihydrothienopyridinone
scaffold <strong>14</strong> resulted in a series of ligand-efficient,
low-micromolar binders of DF2. Most members of the series are selective
against DC1 and DC2, while they bind with low-micromolar affinity
also to DF1 and DF3. In contrast, compound <strong>23</strong> displays
distinct selectivity as it binds exclusively to DF2 (<em>K</em>
<sub>d</sub> = 1.3 μM and IC<sub>50</sub> = 11 μM measured
by FP and HTRF, respectively). Using a similar fragment-based strategy,
in our previous screening campaign for DF2 we identified 6-cyclopropyluracil
as hit compound with an IC<sub>50</sub> value of 170 μM. Compound <strong>23</strong> is more potent for
DF2 than 6-cyclopropyluracil by a factor of 15, and is significantly
more selective against DF1 and DF3. It is important to note that protein
crystallography played a key role in both screening campaigns, as
crystal structures were used for docking and molecular dynamics simulations.
Moreover, in the present campaign the crystal structure of DF2 in
the complex with compound <strong>15</strong> guided the derivatization
of compound <strong>14</strong>, which has a similar scaffold and binding
mode.</p>
<p>A few compounds (e.g., <strong>17</strong> and <strong>18</strong>) have
a comparable affinity for the five YTH-containing proteins. Among
them, compounds <strong>17</strong> and <strong>30</strong> are currently
the most potent ligands of DC2 (<em>K</em>
<sub>d</sub> values
of 6.3 and 4.9 μM, respectively, measured by FP). We have also
presented the crystal structures of DF2 in complex with six ligands,
which represent six distinct chemotypes (compounds <strong>3</strong>–<strong>6</strong>, <strong>13</strong>, and <strong>15</strong>). This structural
data is useful for the development of a new series of ligands of the
YTHDF m<sup>6</sup>A-RNA readers, and for the further training of
machine learning models.
</p></section><section id="sec4"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">Fragment Docking and Ranking</h3>
<p>We
used force field-based
docking to identify small-molecule binders of the m<sup>6</sup>A reader
YTHDF2. The structure of the YTHDF2 domain used for docking is the
one in the complex with the ligand 6-cyclopropyl-1<em>H</em>-pyrimidine-2,4-dione (PDB code: <a href="https://www.rcsb.org/structure/7R5W" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7R5W</a>). We prepared
the protein structure for docking using CAMPARIv5. The SEED<em>
,,
</em> docking program was used for rigid docking to the crystal structure
itself and a snapshot obtained by MD simulations. These MD simulations
were described in a previous study. After
clustering the snapshots, a representative pose with a large aperture
of the m<sup>6</sup>A binding site was selected (volume of 600 vs
324 Å<sup>3</sup> in the crystal structure). The pocket volume
was obtained by structural alignment of the crystal structure (PDB <a href="https://www.rcsb.org/structure/7R5W" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7R5W</a>) and the MD snapshot,
and using the dpocket functionality of the fpocket tool with the ligand
as reference. A library of 500,000 small
molecules was considered for screening by SEED. The molecules were
selected from the ZINC2020 database with
the number of non-hydrogen atoms between 11 and 20, at least one ring
and one sp<sup>3</sup> carbon in the structure. For each of the extracted
compounds, up to 20 conformers were generated using a distance geometry-based
algorithm.
</p>
<p>The two protein structures
were kept rigid during docking and evaluation of the binding energy.
The residue C433 was selected for posing the fragments by SEED, which
is similar to a pharmacophoric constraint. For the evaluation of the
SEED energy, the binding site consisted of all of the DF2 residues
within 20 Å of C433 and all charged side chains of DF2. A structural
water molecule was also considered as part of the binding site because
it is consistently resolved in all of the crystal structures obtained
and is involved as a hydrogen-bond acceptor and donor with the side
chains of W432 and D528, respectively.<em>
,
</em> The partial
charges and van der Waals parameters for the atoms in the protein
and the small molecules were taken from the CHARMM36 all-atom force
field<em>
−

</em> and the CHARMM general force field (CGenFF), respectively. Importantly, the CHARMM36 force
field and CGenFF are fully consistent in their partial charges and
van der Waals parameters. The evaluation of the binding energy in
the program SEED consists of a force field-based energy function with
a continuum dielectric approximation of desolvation penalties by the
generalized Born model. The values of
the dielectric constant were 2.0 and 78.5 for the regions of the space
occupied by the solute and solvent, respectively. Fragment screening
by SEED requires about 1 s per fragment. SEED is available as an open-source
code from GitLab (<a href="https://gitlab.com/CaflischLab" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gitlab.com/CaflischLab</a>).</p>
<p>From both docking campaigns, the compounds were ranked according
to two energy terms calculated by SEED, namely, the total binding
energy (SEED total) and the difference between the electrostatic contribution
to the intermolecular interaction energy in the solvent and the solvation
energy of the ligand (Delec). The top-scoring compounds were then
selected if they showed the crucial hydrogen bond with the backbone
carbonyl of C433 and a favorable interaction with the backbone NH
of Y418, the side chain of D422, and/or the conserved water molecule.
Finally, 47 compounds were purchased on the basis of commercial availability
and structural diversity: 25 selected from the docking performed on
the crystal structure and 22 from that on the MD snapshot with a large
aperture of the binding site.</p></section><section id="sec4.2"><h3 class="pmc_sec_title">HTRF Assay</h3>
<p>GST-YTHDC1,
GST-YTHDF1, GST-YTHDF2, and
GST-YTHDF3 were purified as previously reported. The HTRF assay was assembled as detailed in ref  with the only difference
being that the starting concentration of the dose–response
experiments used for the IC<sub>50</sub> determination was variated
dependently from the tested compound. The same protocol was applied
to the four proteins. The competitive inhibition data of GST-YTHDF1
(single dose experiment at 100 μM compound concentration and
dose–response curves), GST-YTHDF3 (single dose experiment at
100 μM compound concentration and dose–response curves),
and GST-YTHDF2 (single dose experiment at 100 μM compound concentration)
were normalized as described in ref  to mitigate interference. The signal was measured
as described in ref .</p></section><section id="sec4.3"><h3 class="pmc_sec_title">GST-YTHDC2 Production</h3>
<p>The N-terminally GST-tagged YTH
domain of YTHDC2 (residues 1285–1424, cloned into the pGEX-6P-1
vector) was overexpressed in Rosetta (DE3) cells grown overnight at
20 °C following induction with 0.4 mM isopropyl-β-<span class="font-variant-small-caps">d</span>-thiogalactopyranoside (IPTG) at an OD<sub>600</sub> of 0.8. The
cells were harvested and resuspended in a lysis buffer containing
100 mM Tris–HCl (pH 8.0) and 500 mM NaCl. After cell lysis,
the lysate was clarified by centrifugation at 48,000<em>g</em> for 1 h, and the soluble proteins were loaded onto a column packed
with Glutathione Sepharose 4B (GE Healthcare), then eluted with 20
mM reduced glutathione in lysis buffer. Finally, a size-exclusion
chromatography step (HiLoad 16/600 Superdex 200 pg column, GE Healthcare)
was performed to further purify the protein in 20 mM Tris–HCl
(pH 7.4) and 150 mM NaCl.</p></section><section id="sec4.4"><h3 class="pmc_sec_title">DNA-Fluorescence Polarization (FP) Assay</h3>
<p>For fluorescence
polarization (FP) experiments, a 5′-fluorescein-labeled m<sup>6</sup>A-DNA probe (5′-FAM-AAGAACCGG­(m6A)­CTAAGCT-3′)
was synthesized by Microsynth AG.</p>
<p>The final concentration of
the fluorescein-labeled DNA was kept constant at 3 nM. For YTHDC2
measurements, the protein concentration was set to 25 nM, while for
YTHDF2 measurements, it was set to 10 nM, and for YTHDC1, it was set
at 15 nM. The compound concentration was serially diluted to obtain
the dose–response curve. The competition experiments were conducted
in a final volume of 20 μL in a buffer containing 20 mM Tris–HCl
(pH 7.4), 150 mM NaCl, and 0.01% bovine serum albumin (BSA), using
a 384-well black flat-bottomed microplate (Corning 3575).</p>
<p>After
incubating the mixture for 1 h, the anisotropy values were
measured using a Tecan SPARK plate reader with 485/20 nm excitation
and 535/25 nm emission polarization filters, suitable for fluorescein,
at 25 °C. To obtain the equilibrium dissociation constant (<em>K</em>
<sub>d</sub>), the IC<sub>50</sub> values were first derived
by fitting a dose–response curve to the data using nonlinear
regression analysis in GraphPad Prism. These IC<sub>50</sub> values
were then converted to <em>K</em>
<sub>d</sub> by using the
Cheng–Prusoff equation:</p>
<table class="disp-formula p" id="ueq1"><tr><td class="formula"><math id="d1e1690" display="block"><msub><mi>K</mi><mi mathvariant="normal">d</mi></msub><mo>=</mo><mfrac><msub><mi>IC</mi><mn>50</mn></msub><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><mo stretchy="false">[</mo><mi mathvariant="normal">L</mi><mo stretchy="false">]</mo></mrow><msubsup><mi>K</mi><mi mathvariant="normal">d</mi><mi>probe</mi></msubsup></mfrac></mrow></mfrac></math></td></tr></table>
<p>IC<sub>50</sub> is the observed value
from FP assay, [L] is
the
probe concentration used, and <em>K</em>
<sub>d</sub>
 is the binding affinity of
probe for the target.</p></section><section id="sec4.5"><h3 class="pmc_sec_title">YTHDF2 Protein Crystallography and Soaking</h3>
<p>The YTH
domain of YTHDF2 was expressed, purified, crystallized, and soaked
as described in ref . The X-ray diffraction experiment was performed on the X06DA beamline
of Paul Scherrer Institute’s Swiss Light Source. The resulting
data were analyzed as described in ref .</p></section><section id="sec4.6"><h3 class="pmc_sec_title">Molecular Dynamics Simulations and Clustering</h3>
<p>We used
molecular dynamics (MD) simulations to analyze the interactions between
the YTHDF2 domain and compounds <strong>14</strong>, <strong>17</strong>, <strong>23</strong>, and <strong>27</strong>. Compound <strong>14</strong> (and its
derivatives <strong>17</strong>, <strong>23</strong> and <strong>27</strong>)
can, in principle, bind in two poses due to the symmetry of its thiourea
ring. This was also observed in the docking results of compound <strong>14</strong>. Therefore, we defined two poses, A and B, depending respectively
on the orientation of the exocyclic sulfur atom pointing toward the
tryptophan cage or outward. We used the ParaLig software to modify the two docked poses of compound <strong>14</strong> to obtain derivatives <strong>17</strong> and <strong>27</strong>. The protein/ligand structures were then prepared using the software
CAMPARIv5. Simulations were run using
TIP3P (CHARMM) water model, with a 0.15
M concentration of Na<sup>+</sup> and Cl<sup>–</sup> ions.
We equilibrated the systems first using an NPT ensemble to reach 300
K and 1 bar under 10 kJ/(mol Å<sup>2</sup>) positional restraints.
We then applied four successive 1 ns NVT equilibrations with weakening
restraints of 10, 5, 2.5, and 1.25 kJ/(mol Å<sup>2</sup>), respectively.
All simulations were done using the CHARMM36m force field with the July 2022 GROMACS port. Production
MD simulations consisted of eight independent runs for each compound
and pose and a sampling of 200 ns per run. Production simulations
were performed at the Swiss Supercomputing Center (CSCS) with the
support of grant s1272 using GROMACS 2021.5.
</p>
<p>For clustering, we first subsampled the trajectories by selecting
MD frames at every nanosecond of the simulation segments from 100
to 200 ns. In other words, we discarded the first half of each run
for allowing the ligand to equilibrate in the binding pocket. The
bound frames were selected according to the distance between the N
atom of C433 and the thiourea sulfur atom and choosing frames with
a distance lower than 5 Å. We applied PCA to a set of 10 protein–ligand
distances to reduce the data to two dimensions. These 10 distances
involved atoms of the thiourea ring and the residues C433 (four distances),
D422 (two distances), and Y418, W432, W486, and W491 (one distance
each). The Gaussian mixture algorithm with full covariance was employed
for clustering the two-dimensional data in PC space. The MD snapshot
closest to the center of the cluster (centroid) was used as a representative
pose. The root-mean-square deviation (RMSD) of the ligand in the binding
pocket was calculated for all of the trajectories. First, all MD snapshots
were overlapped with the equilibrated starting structure using the
Cα atoms of the protein. Then the coordinates of the heavy atoms
of the ligand were employed for the calculation of the RMSD. All analyses
were done using MDTraj and SciKit learn.
</p></section></section><section id="sec18"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371494/bin/bg5c00099_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00099_si_001.pdf</a><sup> (8.4MB, pdf) </sup>
</div></div></section></section><section id="ACK-d14e1768-autogenerated" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors
thank Beat Blattmann and Görkem Kurtuldu
at the Protein Crystallization Center of UZH for the assistance with
the crystallization, and the beamline scientists at the Swiss Light
Source at Paul Scherrer Institute for their help with the X-ray diffraction
experiments. We thank Maria Paula Flores Espinoza and Thomas Frei
for their technical assistance. This work was financially supported
by the Swiss National Science Foundation to A.C. (Grant number 310030_212195),
the Swiss Cancer Research Foundation to A.C. (Grant number KFS 5748-02-2023),
the Swiss National Supercomputing Center (CSCS, project ID s1272 on
Piz Daint), and the UZH Candoc Grant to A.I. (Grant number FK-24-031).</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Glossary</h2>
<h3>Abbreviations</h3>
<dl class="def-list" id="dl1">
<dt>YTHDF</dt>
<dd><p>YT521-B
homology domain family</p></dd>
<dt>m<sup>6</sup>A</dt>
<dd><p>
<em>N</em>
<sup>6</sup>-methyladenosine</p></dd>
<dt>YTHDC</dt>
<dd><p>YTH domain containing</p></dd>
<dt>oligoRNA</dt>
<dd><p>oligoribonucleotide</p></dd>
<dt>HTRF</dt>
<dd><p>homogeneous time-resolved
fluorescence</p></dd>
<dt>LE</dt>
<dd><p>ligand efficiency</p></dd>
<dt>FP</dt>
<dd><p>fluorescence polarization</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>RMSD</dt>
<dd><p>root-mean-square deviation</p></dd>
</dl></section><section id="notes-1"><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00099?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00099</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Supplementary
figures, tables, materials, synthetic
procedures, characterization data, <sup>1</sup>H and <sup>13</sup>C NMR spectra, HPLC traces for all final compounds, and X-ray data
collection and refinement statistics for the 6 complex YTH-YTHDF2
crystal structures (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00099/suppl_file/bg5c00099_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes-3"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Shi H., Wei J., He C.. Where, When, and How: Context-Dependent Functions of
RNA Methylation Writers, Readers, and Erasers. Mol. Cell. 2019;74(4):640–650. doi: 10.1016/j.molcel.2019.04.025.</cite> [<a href="https://doi.org/10.1016/j.molcel.2019.04.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6527355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31100245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Where,%20When,%20and%20How:%20Context-Dependent%20Functions%20of%20RNA%20Methylation%20Writers,%20Readers,%20and%20Erasers&amp;author=H.%20Shi&amp;author=J.%20Wei&amp;author=C.%20He&amp;volume=74&amp;issue=4&amp;publication_year=2019&amp;pages=640-650&amp;pmid=31100245&amp;doi=10.1016/j.molcel.2019.04.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Liao J., Wei Y., Liang J., Wen J., Chen X., Zhang B., Chu L.. Insight into the Structure,
Physiological Function, and Role in Cancer
of M6A ReadersYTH Domain-Containing Proteins. Cell Death Discovery. 2022;8:137.  doi: 10.1038/s41420-022-00947-0.</cite> [<a href="https://doi.org/10.1038/s41420-022-00947-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8964710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35351856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Discovery&amp;title=Insight%20into%20the%20Structure,%20Physiological%20Function,%20and%20Role%20in%20Cancer%20of%20M6A%20Readers%EE%97%B8YTH%20Domain-Containing%20Proteins&amp;author=J.%20Liao&amp;author=Y.%20Wei&amp;author=J.%20Liang&amp;author=J.%20Wen&amp;author=X.%20Chen&amp;volume=8&amp;publication_year=2022&amp;pages=137&amp;pmid=35351856&amp;doi=10.1038/s41420-022-00947-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Chen L., Gao Y., Xu S., Yuan J., Wang M., Li T., Gong J.. N6-Methyladenosine
Reader YTHDF Family in Biological Processes: Structures,
Roles, and Mechanisms. Front. Immunol. 2023;14:1162607.  doi: 10.3389/fimmu.2023.1162607.</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1162607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10043241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36999016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&amp;title=N6-Methyladenosine%20Reader%20YTHDF%20Family%20in%20Biological%20Processes:%20Structures,%20Roles,%20and%20Mechanisms&amp;author=L.%20Chen&amp;author=Y.%20Gao&amp;author=S.%20Xu&amp;author=J.%20Yuan&amp;author=M.%20Wang&amp;volume=14&amp;publication_year=2023&amp;pages=1162607&amp;pmid=36999016&amp;doi=10.3389/fimmu.2023.1162607&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Dai X. Y., Shi L., Li Z., Yang H. Y., Wei J. F., Ding Q.. Main N6-Methyladenosine
Readers: YTH Family Proteins in Cancers. Front.
Oncol. 2021;11:635329.  doi: 10.3389/fonc.2021.635329.</cite> [<a href="https://doi.org/10.3389/fonc.2021.635329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8076607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33928028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol.&amp;title=Main%20N6-Methyladenosine%20Readers:%20YTH%20Family%20Proteins%20in%20Cancers&amp;author=X.%20Y.%20Dai&amp;author=L.%20Shi&amp;author=Z.%20Li&amp;author=H.%20Y.%20Yang&amp;author=J.%20F.%20Wei&amp;volume=11&amp;publication_year=2021&amp;pages=635329&amp;pmid=33928028&amp;doi=10.3389/fonc.2021.635329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Xiao W., Adhikari S., Dahal U., Chen Y. S., Hao Y. J., Sun B. F., Sun H. Y., Li A., Ping X. L., Lai W. Y., Wang X., Ma H. L., Huang C. M., Yang Y., Huang N., Jiang G. B., Wang H. L., Zhou Q., Wang X. J., Zhao Y. L., Yang Y. G.. Nuclear
M6A Reader YTHDC1 Regulates MRNA Splicing. Mol.
Cell. 2016;61(4):507–519. doi: 10.1016/j.molcel.2016.01.012.</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26876937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Nuclear%20M6A%20Reader%20YTHDC1%20Regulates%20MRNA%20Splicing&amp;author=W.%20Xiao&amp;author=S.%20Adhikari&amp;author=U.%20Dahal&amp;author=Y.%20S.%20Chen&amp;author=Y.%20J.%20Hao&amp;volume=61&amp;issue=4&amp;publication_year=2016&amp;pages=507-519&amp;pmid=26876937&amp;doi=10.1016/j.molcel.2016.01.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Hsu P. J., Zhu Y., Ma H., Guo Y., Shi X., Liu Y., Qi M., Lu Z., Shi H., Wang J., Cheng Y., Luo G., Dai Q., Liu M., Guo X., Sha J., Shen B., He C.. Ythdc2 Is
an N6-Methyladenosine Binding
Protein That Regulates Mammalian Spermatogenesis. Cell Res. 2017;27(9):1115–1127. doi: 10.1038/cr.2017.99.</cite> [<a href="https://doi.org/10.1038/cr.2017.99" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5587856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28809393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=Ythdc2%20Is%20an%20N6-Methyladenosine%20Binding%20Protein%20That%20Regulates%20Mammalian%20Spermatogenesis&amp;author=P.%20J.%20Hsu&amp;author=Y.%20Zhu&amp;author=H.%20Ma&amp;author=Y.%20Guo&amp;author=X.%20Shi&amp;volume=27&amp;issue=9&amp;publication_year=2017&amp;pages=1115-1127&amp;pmid=28809393&amp;doi=10.1038/cr.2017.99&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Kretschmer J., Rao H., Hackert P., Sloan K. E., Höbartner C., Bohnsack M. T.. The M6A Reader Protein YTHDC2 Interacts with the Small
Ribosomal Subunit and the 5′–3′ Exoribonuclease
XRN1. RNA. 2018;24(10):1339–1350. doi: 10.1261/rna.064238.117.</cite> [<a href="https://doi.org/10.1261/rna.064238.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6140455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29970596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=The%20M6A%20Reader%20Protein%20YTHDC2%20Interacts%20with%20the%20Small%20Ribosomal%20Subunit%20and%20the%205%E2%80%B2%E2%80%933%E2%80%B2%20Exoribonuclease%20XRN1&amp;author=J.%20Kretschmer&amp;author=H.%20Rao&amp;author=P.%20Hackert&amp;author=K.%20E.%20Sloan&amp;author=C.%20H%C3%B6bartner&amp;volume=24&amp;issue=10&amp;publication_year=2018&amp;pages=1339-1350&amp;pmid=29970596&amp;doi=10.1261/rna.064238.117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Lasman L., Krupalnik V., Viukov S., Mor N., Aguilera-Castrejon A., Schneir D., Bayerl J., Mizrahi O., Peles S., Tawil S., Sathe S., Nachshon A., Shani T., Zerbib M., Kilimnik I., Aigner S., Shankar A., Mueller J. R., Schwartz S., Stern-Ginossar N., Yeo G. W., Geula S., Novershtern N., Hanna J. H.. Context-Dependent Functional Compensation between Ythdf
M6A Reader Proteins. Genes Dev. 2020;34(19–20):1373–1391. doi: 10.1101/gad.340695.120.</cite> [<a href="https://doi.org/10.1101/gad.340695.120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7528697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32943573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Context-Dependent%20Functional%20Compensation%20between%20Ythdf%20M6A%20Reader%20Proteins&amp;author=L.%20Lasman&amp;author=V.%20Krupalnik&amp;author=S.%20Viukov&amp;author=N.%20Mor&amp;author=A.%20Aguilera-Castrejon&amp;volume=34&amp;issue=19%E2%80%9320&amp;publication_year=2020&amp;pages=1373-1391&amp;pmid=32943573&amp;doi=10.1101/gad.340695.120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Li F., Zeng C., Liu J., Wang L., Yuan X., Yuan L., Xia X., Huang W.. The YTH Domain-Containing
Protein Family: Emerging Players in Immunomodulation and Tumour Immunotherapy
Targets. Clin. Transl. Med. 2024;14(8):e1784.  doi: 10.1002/ctm2.1784.</cite> [<a href="https://doi.org/10.1002/ctm2.1784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11319238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39135292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Med.&amp;title=The%20YTH%20Domain-Containing%20Protein%20Family:%20Emerging%20Players%20in%20Immunomodulation%20and%20Tumour%20Immunotherapy%20Targets&amp;author=F.%20Li&amp;author=C.%20Zeng&amp;author=J.%20Liu&amp;author=L.%20Wang&amp;author=X.%20Yuan&amp;volume=14&amp;issue=8&amp;publication_year=2024&amp;pages=e1784&amp;pmid=39135292&amp;doi=10.1002/ctm2.1784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Li J., Meng S., Xu M., Wang S., He L., Xu X., Wang X., Xie L.. Downregulation of N6-Methyladenosine
Binding YTHDF2 Protein Mediated by MiR-493–3p Suppresses Prostate
Cancer by Elevating N6-Methyladenosine Levels. Oncotarget. 2018;9(3):3752–3764. doi: 10.18632/oncotarget.23365.</cite> [<a href="https://doi.org/10.18632/oncotarget.23365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5790497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29423080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Downregulation%20of%20N6-Methyladenosine%20Binding%20YTHDF2%20Protein%20Mediated%20by%20MiR-493%E2%80%933p%20Suppresses%20Prostate%20Cancer%20by%20Elevating%20N6-Methyladenosine%20Levels&amp;author=J.%20Li&amp;author=S.%20Meng&amp;author=M.%20Xu&amp;author=S.%20Wang&amp;author=L.%20He&amp;volume=9&amp;issue=3&amp;publication_year=2018&amp;pages=3752-3764&amp;pmid=29423080&amp;doi=10.18632/oncotarget.23365&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Einstein J. M., Perelis M., Chaim I. A., Meena J. K., Nussbacher J. K., Tankka A. T., Yee B. A., Li H., Madrigal A. A., Neill N. J., Shankar A., Tyagi S., Westbrook T. F., Yeo G. W.. Inhibition of YTHDF2 Triggers Proteotoxic Cell Death
in MYC-Driven Breast Cancer. Mol. Cell. 2021;81(15):3048.e9–3064.e9. doi: 10.1016/j.molcel.2021.06.014.</cite> [<a href="https://doi.org/10.1016/j.molcel.2021.06.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8359670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34216543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Inhibition%20of%20YTHDF2%20Triggers%20Proteotoxic%20Cell%20Death%20in%20MYC-Driven%20Breast%20Cancer&amp;author=J.%20M.%20Einstein&amp;author=M.%20Perelis&amp;author=I.%20A.%20Chaim&amp;author=J.%20K.%20Meena&amp;author=J.%20K.%20Nussbacher&amp;volume=81&amp;issue=15&amp;publication_year=2021&amp;pages=3048.e9-3064.e9&amp;pmid=34216543&amp;doi=10.1016/j.molcel.2021.06.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Wang J.-y., Lu A.-q.. The Biological Function
of M6A Reader YTHDF2 and Its Role in Human
Disease. Cancer Cell Int. 2021;21:109.  doi: 10.1186/S12935-021-01807-0.</cite> [<a href="https://doi.org/10.1186/S12935-021-01807-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7885220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33593354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int.&amp;title=The%20Biological%20Function%20of%20M6A%20Reader%20YTHDF2%20and%20Its%20Role%20in%20Human%20Disease&amp;author=J.-y.%20Wang&amp;author=A.-q.%20Lu&amp;volume=21&amp;publication_year=2021&amp;pages=109&amp;pmid=33593354&amp;doi=10.1186/S12935-021-01807-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Nai F., Nachawati R., Zálešák F., Wang X., Li Y., Caflisch A.. Fragment Ligands of
the M6A-RNA Reader YTHDF2. ACS Med. Chem. Lett. 2022;13(9):1500–1509. doi: 10.1021/acsmedchemlett.2c00303.</cite> [<a href="https://doi.org/10.1021/acsmedchemlett.2c00303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9466600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36110386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Med.%20Chem.%20Lett.&amp;title=Fragment%20Ligands%20of%20the%20M6A-RNA%20Reader%20YTHDF2&amp;author=F.%20Nai&amp;author=R.%20Nachawati&amp;author=F.%20Z%C3%A1le%C5%A1%C3%A1k&amp;author=X.%20Wang&amp;author=Y.%20Li&amp;volume=13&amp;issue=9&amp;publication_year=2022&amp;pages=1500-1509&amp;pmid=36110386&amp;doi=10.1021/acsmedchemlett.2c00303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Micaelli M., Dalle Vedove A., Cerofolini L., Vigna J., Sighel D., Zaccara S., Bonomo I., Poulentzas G., Rosatti E. F., Cazzanelli G., Alunno L., Belli R., Peroni D., Dassi E., Murakami S., Jaffrey S. R., Fragai M., Mancini I., Lolli G., Quattrone A., Provenzani A.. Small-Molecule Ebselen Binds to YTHDF Proteins Interfering
with the Recognition of N6-Methyladenosine-Modified RNAs. ACS Pharmacol. Transl. Sci. 2022;5(10):872–891. doi: 10.1021/acsptsci.2c00008.</cite> [<a href="https://doi.org/10.1021/acsptsci.2c00008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9578143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36268123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Pharmacol.%20Transl.%20Sci.&amp;title=Small-Molecule%20Ebselen%20Binds%20to%20YTHDF%20Proteins%20Interfering%20with%20the%20Recognition%20of%20N6-Methyladenosine-Modified%20RNAs&amp;author=M.%20Micaelli&amp;author=A.%20Dalle%20Vedove&amp;author=L.%20Cerofolini&amp;author=J.%20Vigna&amp;author=D.%20Sighel&amp;volume=5&amp;issue=10&amp;publication_year=2022&amp;pages=872-891&amp;pmid=36268123&amp;doi=10.1021/acsptsci.2c00008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Madia V. N., Messore A., Saccoliti F., Tudino V., De Leo A., De Vita D., Bortolami M., Scipione L., Pindinello I., Costi R., Di Santo R.. Tegaserod
for the Treatment of Irritable
Bowel Syndrome. Anti-Inflammatory Anti-Allergy
Agents Med. Chem. 2020;19(4):342–369. doi: 10.2174/1871523018666190911121306.</cite> [<a href="https://doi.org/10.2174/1871523018666190911121306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7579269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31518227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anti-Inflammatory%20Anti-Allergy%20Agents%20Med.%20Chem.&amp;title=Tegaserod%20for%20the%20Treatment%20of%20Irritable%20Bowel%20Syndrome&amp;author=V.%20N.%20Madia&amp;author=A.%20Messore&amp;author=F.%20Saccoliti&amp;author=V.%20Tudino&amp;author=A.%20De%20Leo&amp;volume=19&amp;issue=4&amp;publication_year=2020&amp;pages=342-369&amp;pmid=31518227&amp;doi=10.2174/1871523018666190911121306&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Zou Z., Wei J., Chen Y., Jin P., Luo C., Kang Y., Shi H., Yang F., Shi Z., Chen S., Zhou Y., Sepich-Poore C., Zhuang X., Zhou X., Jiang H., Wen Z., He C.. FMRP Phosphorylation Modulates Neuronal Translation
through YTHDF1. Mol. Cell. 2023;83:4304.e8–4317.e8. doi: 10.1016/j.molcel.2023.10.028.</cite> [<a href="https://doi.org/10.1016/j.molcel.2023.10.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10872974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37949069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=FMRP%20Phosphorylation%20Modulates%20Neuronal%20Translation%20through%20YTHDF1&amp;author=Z.%20Zou&amp;author=J.%20Wei&amp;author=Y.%20Chen&amp;author=P.%20Jin&amp;author=C.%20Luo&amp;volume=83&amp;publication_year=2023&amp;pages=4304.e8-4317.e8&amp;pmid=37949069&amp;doi=10.1016/j.molcel.2023.10.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Feng G., Wu Y., Hu Y., Shuai W., Yang X., Li Y., Ouyang L., Wang G.. Small Molecule
Inhibitors Targeting
M6A Regulators. J. Hematol. Oncol. 2024;17:30.  doi: 10.1186/s13045-024-01546-5.</cite> [<a href="https://doi.org/10.1186/s13045-024-01546-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11075261/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38711100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Hematol.%20Oncol.&amp;title=Small%20Molecule%20Inhibitors%20Targeting%20M6A%20Regulators&amp;author=G.%20Feng&amp;author=Y.%20Wu&amp;author=Y.%20Hu&amp;author=W.%20Shuai&amp;author=X.%20Yang&amp;volume=17&amp;publication_year=2024&amp;pages=30&amp;pmid=38711100&amp;doi=10.1186/s13045-024-01546-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>

Nai, F.

Development of Chemical
Probes for Epitranscriptomic Proteins; University
of Zurich, Faculty of Science: Zurich, 2024; pp 5–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20Chemical%20Probes%20for%20Epitranscriptomic%20Proteins&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Wang L., Dou X., Chen S., Yu X., Huang X., Zhang L., Chen Y., Wang J., Yang K., Bugno J., Pitroda S., Ding X., Piffko A., Si W., Chen C., Jiang H., Zhou B., Chmura S. J., Luo C., Liang H. L., He C., Weichselbaum R. R.. YTHDF2
Inhibition Potentiates Radiotherapy Antitumor Efficacy. Cancer Cell. 2023;41(7):1294.e8–1308.e8. doi: 10.1016/j.ccell.2023.04.019.</cite> [<a href="https://doi.org/10.1016/j.ccell.2023.04.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10524856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37236197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=YTHDF2%20Inhibition%20Potentiates%20Radiotherapy%20Antitumor%20Efficacy&amp;author=L.%20Wang&amp;author=X.%20Dou&amp;author=S.%20Chen&amp;author=X.%20Yu&amp;author=X.%20Huang&amp;volume=41&amp;issue=7&amp;publication_year=2023&amp;pages=1294.e8-1308.e8&amp;pmid=37236197&amp;doi=10.1016/j.ccell.2023.04.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Qiu X., Kemker C., Goebel G. L., Lampe P., Wallis N., Schiller D., Bigler K., Jiang M., Sievers S., Yeo G. W., Wu P.. Phenylpyrazoles as Inhibitors of
the M6A RNA-Binding Protein YTHDF2. JACS Au. 2025;5(2):618–630. doi: 10.1021/jacsau.4c00754.</cite> [<a href="https://doi.org/10.1021/jacsau.4c00754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11862924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40017738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JACS%20Au&amp;title=Phenylpyrazoles%20as%20Inhibitors%20of%20the%20M6A%20RNA-Binding%20Protein%20YTHDF2&amp;author=X.%20Qiu&amp;author=C.%20Kemker&amp;author=G.%20L.%20Goebel&amp;author=P.%20Lampe&amp;author=N.%20Wallis&amp;volume=5&amp;issue=2&amp;publication_year=2025&amp;pages=618-630&amp;pmid=40017738&amp;doi=10.1021/jacsau.4c00754&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Kolb P., Kipouros C. B., Huang D., Caflisch A.. Structure-Based Tailoring
of Compound Libraries for High-Throughput Screening: Discovery of
Novel EphB4 Kinase Inhibitors. Proteins: Struct.,
Funct., Bioinf. 2008;73(1):11–18. doi: 10.1002/prot.22028.</cite> [<a href="https://doi.org/10.1002/prot.22028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18384152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:%20Struct.,%20Funct.,%20Bioinf.&amp;title=Structure-Based%20Tailoring%20of%20Compound%20Libraries%20for%20High-Throughput%20Screening:%20Discovery%20of%20Novel%20EphB4%20Kinase%20Inhibitors&amp;author=P.%20Kolb&amp;author=C.%20B.%20Kipouros&amp;author=D.%20Huang&amp;author=A.%20Caflisch&amp;volume=73&amp;issue=1&amp;publication_year=2008&amp;pages=11-18&amp;pmid=18384152&amp;doi=10.1002/prot.22028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Marchand J. R., Caflisch A.. In Silico Fragment-Based Drug Design with SEED. Eur. J. Med. Chem. 2018;156:907–917. doi: 10.1016/j.ejmech.2018.07.042.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2018.07.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30064119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=In%20Silico%20Fragment-Based%20Drug%20Design%20with%20SEED&amp;author=J.%20R.%20Marchand&amp;author=A.%20Caflisch&amp;volume=156&amp;publication_year=2018&amp;pages=907-917&amp;pmid=30064119&amp;doi=10.1016/j.ejmech.2018.07.042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Majeux N., Scarsi M., Apostolakis J., Ehrhardt C., Caflisch A.. Exhaustive
Docking of Molecular Fragments With Electrostatic Solvation. Proteins: Struct., Funct., Genet. 1999;37:88–105. doi: 10.1002/(SICI)1097-0134(19991001)37:1&amp;#x0003c;88::AID-PROT9&amp;#x0003e;3.0.CO;2-O.</cite> [<a href="https://doi.org/10.1002/(SICI)1097-0134(19991001)37:1&amp;#x0003c;88::AID-PROT9&amp;#x0003e;3.0.CO;2-O" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10451553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:%20Struct.,%20Funct.,%20Genet.&amp;title=Exhaustive%20Docking%20of%20Molecular%20Fragments%20With%20Electrostatic%20Solvation&amp;author=N.%20Majeux&amp;author=M.%20Scarsi&amp;author=J.%20Apostolakis&amp;author=C.%20Ehrhardt&amp;author=A.%20Caflisch&amp;volume=37&amp;publication_year=1999&amp;pages=88-105&amp;pmid=10451553&amp;doi=10.1002/(SICI)1097-0134(19991001)37:1&amp;#x0003c;88::AID-PROT9&amp;#x0003e;3.0.CO;2-O&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Goossens K., Wroblowski B., Langini C., van Vlijmen H., Caflisch A., de Winter H.. Assessment
of the Fragment Docking
Program SEED. J. Chem. Inf. Model. 2020;60(10):4881–4893. doi: 10.1021/acs.jcim.0c00556.</cite> [<a href="https://doi.org/10.1021/acs.jcim.0c00556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32820916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=Assessment%20of%20the%20Fragment%20Docking%20Program%20SEED&amp;author=K.%20Goossens&amp;author=B.%20Wroblowski&amp;author=C.%20Langini&amp;author=H.%20van%20Vlijmen&amp;author=A.%20Caflisch&amp;volume=60&amp;issue=10&amp;publication_year=2020&amp;pages=4881-4893&amp;pmid=32820916&amp;doi=10.1021/acs.jcim.0c00556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Irwin J. J., Tang K. G., Young J., Dandarchuluun C., Wong B. R., Khurelbaatar M., Moroz Y. S., Mayfield J., Sayle R. A.. ZINC20 - A Free Ultralarge-Scale Chemical Database
for Ligand Discovery. J. Chem. Inf. Model. 2020;60(12):6065–6073. doi: 10.1021/acs.jcim.0c00675.</cite> [<a href="https://doi.org/10.1021/acs.jcim.0c00675" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8284596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33118813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=ZINC20%20-%20A%20Free%20Ultralarge-Scale%20Chemical%20Database%20for%20Ligand%20Discovery&amp;author=J.%20J.%20Irwin&amp;author=K.%20G.%20Tang&amp;author=J.%20Young&amp;author=C.%20Dandarchuluun&amp;author=B.%20R.%20Wong&amp;volume=60&amp;issue=12&amp;publication_year=2020&amp;pages=6065-6073&amp;pmid=33118813&amp;doi=10.1021/acs.jcim.0c00675&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Riniker S., Landrum G. A.. Better Informed Distance Geometry: Using What We Know
to Improve Conformation Generation. J. Chem.
Inf. Model. 2015;55(12):2562–2574. doi: 10.1021/acs.jcim.5b00654.</cite> [<a href="https://doi.org/10.1021/acs.jcim.5b00654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26575315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=Better%20Informed%20Distance%20Geometry:%20Using%20What%20We%20Know%20to%20Improve%20Conformation%20Generation&amp;author=S.%20Riniker&amp;author=G.%20A.%20Landrum&amp;volume=55&amp;issue=12&amp;publication_year=2015&amp;pages=2562-2574&amp;pmid=26575315&amp;doi=10.1021/acs.jcim.5b00654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Li Y., Bedi R. K., Moroz-Omori E. V., Caflisch A.. Structural and Dynamic
Insights into Redundant Function of YTHDF Proteins. J. Chem. Inf. Model. 2020;60(12):5932–5935. doi: 10.1021/acs.jcim.0c01029.</cite> [<a href="https://doi.org/10.1021/acs.jcim.0c01029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33073985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=Structural%20and%20Dynamic%20Insights%20into%20Redundant%20Function%20of%20YTHDF%20Proteins&amp;author=Y.%20Li&amp;author=R.%20K.%20Bedi&amp;author=E.%20V.%20Moroz-Omori&amp;author=A.%20Caflisch&amp;volume=60&amp;issue=12&amp;publication_year=2020&amp;pages=5932-5935&amp;pmid=33073985&amp;doi=10.1021/acs.jcim.0c01029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Wiedmer L., Eberle S. A., Bedi R. K., Śledź P., Caflisch A.. A Reader-Based Assay for M6A Writers
and Erasers. Anal. Chem. 2019;91(4):3078–3084. doi: 10.1021/acs.analchem.8b05500.</cite> [<a href="https://doi.org/10.1021/acs.analchem.8b05500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30715855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=A%20Reader-Based%20Assay%20for%20M6A%20Writers%20and%20Erasers&amp;author=L.%20Wiedmer&amp;author=S.%20A.%20Eberle&amp;author=R.%20K.%20Bedi&amp;author=P.%20%C5%9Aled%C5%BA&amp;author=A.%20Caflisch&amp;volume=91&amp;issue=4&amp;publication_year=2019&amp;pages=3078-3084&amp;pmid=30715855&amp;doi=10.1021/acs.analchem.8b05500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Zálešák F., Nai F., Herok M., Bochenkova E., Bedi R. K., Li Y., Errani F., Caflisch A.. Structure-Based Design of a Potent
and Selective YTHDC1 Ligand. J. Med. Chem. 2024;67(11):9516–9535. doi: 10.1021/acs.jmedchem.4c00599.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.4c00599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11181329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38787793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Structure-Based%20Design%20of%20a%20Potent%20and%20Selective%20YTHDC1%20Ligand&amp;author=F.%20Z%C3%A1le%C5%A1%C3%A1k&amp;author=F.%20Nai&amp;author=M.%20Herok&amp;author=E.%20Bochenkova&amp;author=R.%20K.%20Bedi&amp;volume=67&amp;issue=11&amp;publication_year=2024&amp;pages=9516-9535&amp;pmid=38787793&amp;doi=10.1021/acs.jmedchem.4c00599&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Sinnokrot M. O., Valeev E. F., Sherrill C. D.. Estimates of the Ab Initio Limit
for π-π Interactions: The Benzene Dimer. J. Am. Chem. Soc. 2002;124(36):10887–10893. doi: 10.1021/ja025896h.</cite> [<a href="https://doi.org/10.1021/ja025896h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12207544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Estimates%20of%20the%20Ab%20Initio%20Limit%20for%20%CF%80-%CF%80%20Interactions:%20The%20Benzene%20Dimer&amp;author=M.%20O.%20Sinnokrot&amp;author=E.%20F.%20Valeev&amp;author=C.%20D.%20Sherrill&amp;volume=124&amp;issue=36&amp;publication_year=2002&amp;pages=10887-10893&amp;pmid=12207544&amp;doi=10.1021/ja025896h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Zhao Y., Li J., Gu H., Wei D., Xu Y.-c., Fu W., Yu Z.. Conformational Preferences
of π–π Stacking Between
Ligand and Protein, Analysis Derived from Crystal Structure Data Geometric
Preference of π–π Interaction. Interdiscip. Sci.: Comput. Life Sci. 2015;7(3):211–220. doi: 10.1007/s12539-015-0263-z.</cite> [<a href="https://doi.org/10.1007/s12539-015-0263-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26370211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Interdiscip.%20Sci.:%20Comput.%20Life%20Sci.&amp;title=Conformational%20Preferences%20of%20%CF%80%E2%80%93%CF%80%20Stacking%20Between%20Ligand%20and%20Protein,%20Analysis%20Derived%20from%20Crystal%20Structure%20Data%20Geometric%20Preference%20of%20%CF%80%E2%80%93%CF%80%20Interaction&amp;author=Y.%20Zhao&amp;author=J.%20Li&amp;author=H.%20Gu&amp;author=D.%20Wei&amp;author=Y.-c.%20Xu&amp;volume=7&amp;issue=3&amp;publication_year=2015&amp;pages=211-220&amp;pmid=26370211&amp;doi=10.1007/s12539-015-0263-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Wang C. H., Zhou H.. Discovery of a New Inhibitor for
YTH Domain-Containing M6A RNA Readers. RSC Chem.
Biol. 2024;5(9):914–923. doi: 10.1039/D4CB00105B.</cite> [<a href="https://doi.org/10.1039/D4CB00105B" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11353026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39211476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Chem.%20Biol.&amp;title=Discovery%20of%20a%20New%20Inhibitor%20for%20YTH%20Domain-Containing%20M6A%20RNA%20Readers&amp;author=C.%20H.%20Wang&amp;author=H.%20Zhou&amp;volume=5&amp;issue=9&amp;publication_year=2024&amp;pages=914-923&amp;pmid=39211476&amp;doi=10.1039/D4CB00105B&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Nai F., Flores Espinoza M. P., Invernizzi A., Vargas-Rosales P. A., Bobileva O., Herok M., Caflisch A.. Small-Molecule Inhibitors
of the M7G-RNA Writer METTL1. ACS Bio Med Chem
Au. 2024;4(2):100–110. doi: 10.1021/acsbiomedchemau.3c00030.</cite> [<a href="https://doi.org/10.1021/acsbiomedchemau.3c00030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11027120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38645929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Bio%20Med%20Chem%20Au&amp;title=Small-Molecule%20Inhibitors%20of%20the%20M7G-RNA%20Writer%20METTL1&amp;author=F.%20Nai&amp;author=M.%20P.%20Flores%20Espinoza&amp;author=A.%20Invernizzi&amp;author=P.%20A.%20Vargas-Rosales&amp;author=O.%20Bobileva&amp;volume=4&amp;issue=2&amp;publication_year=2024&amp;pages=100-110&amp;pmid=38645929&amp;doi=10.1021/acsbiomedchemau.3c00030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Van
Bergen L. A. H., Alonso M., Palló A., Nilsson L., De Proft F., Messens J.. Revisiting Sulfur H-Bonds
in Proteins: The Example of Peroxiredoxin AhpE. Sci. Rep. 2016;6:30369.  doi: 10.1038/srep30369.</cite> [<a href="https://doi.org/10.1038/srep30369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4965862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27468924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Revisiting%20Sulfur%20H-Bonds%20in%20Proteins:%20The%20Example%20of%20Peroxiredoxin%20AhpE&amp;author=L.%20A.%20H.%20Van%20Bergen&amp;author=M.%20Alonso&amp;author=A.%20Pall%C3%B3&amp;author=L.%20Nilsson&amp;author=F.%20De%20Proft&amp;volume=6&amp;publication_year=2016&amp;pages=30369&amp;pmid=27468924&amp;doi=10.1038/srep30369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Gregoret L.
M., Rader S. D., Fletterick R. J., Cohen F. E.. Hydrogen Bonds Involving
Sulfur Atoms in Proteins. Proteins: Struct.,
Funct., Bioinf. 1991;9(2):99–107. doi: 10.1002/prot.340090204.</cite> [<a href="https://doi.org/10.1002/prot.340090204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1755867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:%20Struct.,%20Funct.,%20Bioinf.&amp;title=Hydrogen%20Bonds%20Involving%20Sulfur%20Atoms%20in%20Proteins&amp;author=L.%20M.%20Gregoret&amp;author=S.%20D.%20Rader&amp;author=R.%20J.%20Fletterick&amp;author=F.%20E.%20Cohen&amp;volume=9&amp;issue=2&amp;publication_year=1991&amp;pages=99-107&amp;pmid=1755867&amp;doi=10.1002/prot.340090204&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Mundlapati V. R., Ghosh S., Bhattacherjee A., Tiwari P., Biswal H. S.. Critical
Assessment of the Strength of Hydrogen Bonds between the Sulfur Atom
of Methionine/Cysteine and Backbone Amides in Proteins. J. Phys. Chem. Lett. 2015;6(8):1385–1389. doi: 10.1021/acs.jpclett.5b00491.</cite> [<a href="https://doi.org/10.1021/acs.jpclett.5b00491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26263139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20Lett.&amp;title=Critical%20Assessment%20of%20the%20Strength%20of%20Hydrogen%20Bonds%20between%20the%20Sulfur%20Atom%20of%20Methionine/Cysteine%20and%20Backbone%20Amides%20in%20Proteins&amp;author=V.%20R.%20Mundlapati&amp;author=S.%20Ghosh&amp;author=A.%20Bhattacherjee&amp;author=P.%20Tiwari&amp;author=H.%20S.%20Biswal&amp;volume=6&amp;issue=8&amp;publication_year=2015&amp;pages=1385-1389&amp;pmid=26263139&amp;doi=10.1021/acs.jpclett.5b00491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Sabnis R. W., Rangnekar D. W., Sonawane N. D.. 2-Aminothiophenes
by the Gewald Reaction. J. Heterocycl. Chem. 1999;36(2):333–345. doi: 10.1002/jhet.5570360203.</cite> [<a href="https://doi.org/10.1002/jhet.5570360203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Heterocycl.%20Chem.&amp;title=2-Aminothiophenes%20by%20the%20Gewald%20Reaction&amp;author=R.%20W.%20Sabnis&amp;author=D.%20W.%20Rangnekar&amp;author=N.%20D.%20Sonawane&amp;volume=36&amp;issue=2&amp;publication_year=1999&amp;pages=333-345&amp;doi=10.1002/jhet.5570360203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>El-Mekabaty A.. Chemistry
of 2-Amino-3-Carbethoxythiophene and Related Compounds. Synth. Commun. 2014;44(1):1–31. doi: 10.1080/00397911.2013.821618.</cite> [<a href="https://doi.org/10.1080/00397911.2013.821618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Synth.%20Commun.&amp;title=Chemistry%20of%202-Amino-3-Carbethoxythiophene%20and%20Related%20Compounds&amp;author=A.%20El-Mekabaty&amp;volume=44&amp;issue=1&amp;publication_year=2014&amp;pages=1-31&amp;doi=10.1080/00397911.2013.821618&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Dong Y., Navarathne D., Bolduc A., McGregor N., Skene W. G.. α,A′-N-Boc-Substituted
Bi- and Terthiophenes: Fluorescent Precursors for Functional Materials. J. Org. Chem. 2012;77(12):5429–5433. doi: 10.1021/jo300687d.</cite> [<a href="https://doi.org/10.1021/jo300687d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22606937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Org.%20Chem.&amp;title=%CE%B1,A%E2%80%B2-N-Boc-Substituted%20Bi-%20and%20Terthiophenes:%20Fluorescent%20Precursors%20for%20Functional%20Materials&amp;author=Y.%20Dong&amp;author=D.%20Navarathne&amp;author=A.%20Bolduc&amp;author=N.%20McGregor&amp;author=W.%20G.%20Skene&amp;volume=77&amp;issue=12&amp;publication_year=2012&amp;pages=5429-5433&amp;pmid=22606937&amp;doi=10.1021/jo300687d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Aurelio L., Figler H., Flynn B. L., Linden J., Scammells P. J.. 5-Substituted
2-Aminothiophenes as A1 Adenosine Receptor Allosteric Enhancers. Bioorg. Med. Chem. 2008;16(3):1319–1327. doi: 10.1016/j.bmc.2007.10.065.</cite> [<a href="https://doi.org/10.1016/j.bmc.2007.10.065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17980606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.&amp;title=5-Substituted%202-Aminothiophenes%20as%20A1%20Adenosine%20Receptor%20Allosteric%20Enhancers&amp;author=L.%20Aurelio&amp;author=H.%20Figler&amp;author=B.%20L.%20Flynn&amp;author=J.%20Linden&amp;author=P.%20J.%20Scammells&amp;volume=16&amp;issue=3&amp;publication_year=2008&amp;pages=1319-1327&amp;pmid=17980606&amp;doi=10.1016/j.bmc.2007.10.065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Wang T., Zheng C. H., Liu S., Chen H. Z.. Synthesis and Biological
Activity of a Series of New Thieno­[2,3-d]­Pyrimidines. Phosphorus, Sulfur Silicon Relat. Elem. 2010;185(7):1543–1549. doi: 10.1080/10426500903127565.</cite> [<a href="https://doi.org/10.1080/10426500903127565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phosphorus,%20Sulfur%20Silicon%20Relat.%20Elem.&amp;title=Synthesis%20and%20Biological%20Activity%20of%20a%20Series%20of%20New%20Thieno%C2%AD%5B2,3-d%5D%C2%ADPyrimidines&amp;author=T.%20Wang&amp;author=C.%20H.%20Zheng&amp;author=S.%20Liu&amp;author=H.%20Z.%20Chen&amp;volume=185&amp;issue=7&amp;publication_year=2010&amp;pages=1543-1549&amp;doi=10.1080/10426500903127565&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Vargas-Rosales P. A., Caflisch A.. The Physics-AI Dialogue
in Drug Design. RSC Med. Chem. 2025;16(4):1499–1515. doi: 10.1039/D4MD00869C.</cite> [<a href="https://doi.org/10.1039/D4MD00869C" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11788922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39906313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Med.%20Chem.&amp;title=The%20Physics-AI%20Dialogue%20in%20Drug%20Design&amp;author=P.%20A.%20Vargas-Rosales&amp;author=A.%20Caflisch&amp;volume=16&amp;issue=4&amp;publication_year=2025&amp;pages=1499-1515&amp;pmid=39906313&amp;doi=10.1039/D4MD00869C&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Vitalis A., Pappu R. V.. Chapter 3 Methods
for Monte Carlo Simulations of Biomacromolecules. Annu. Rep. Comput. Chem. 2009;5:49–76. doi: 10.1016/S1574-1400(09)00503-9.</cite> [<a href="https://doi.org/10.1016/S1574-1400(09)00503-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2860296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20428473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rep.%20Comput.%20Chem.&amp;title=Chapter%203%20Methods%20for%20Monte%20Carlo%20Simulations%20of%20Biomacromolecules&amp;author=A.%20Vitalis&amp;author=R.%20V.%20Pappu&amp;volume=5&amp;publication_year=2009&amp;pages=49-76&amp;pmid=20428473&amp;doi=10.1016/S1574-1400(09)00503-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Majeux N., Scarsi M., Caflisch A.. Efficient Electrostatic
Solvation
Model for Protein-Fragment Docking. Proteins:
Struct., Funct., Genet. 2001;42:256–268. doi: 10.1002/1097-0134(20010201)42:2&amp;#x0003c;256::AID-PROT130&amp;#x0003e;3.0.CO;2-4.</cite> [<a href="https://doi.org/10.1002/1097-0134(20010201)42:2&amp;#x0003c;256::AID-PROT130&amp;#x0003e;3.0.CO;2-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11119650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:%20Struct.,%20Funct.,%20Genet.&amp;title=Efficient%20Electrostatic%20Solvation%20Model%20for%20Protein-Fragment%20Docking&amp;author=N.%20Majeux&amp;author=M.%20Scarsi&amp;author=A.%20Caflisch&amp;volume=42&amp;publication_year=2001&amp;pages=256-268&amp;pmid=11119650&amp;doi=10.1002/1097-0134(20010201)42:2&amp;#x0003c;256::AID-PROT130&amp;#x0003e;3.0.CO;2-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Le
Guilloux V., Schmidtke P., Tuffery P.. Fpocket: An Open Source
Platform for Ligand Pocket Detection. BMC Bioinf. 2009;10(1):168.  doi: 10.1186/1471-2105-10-168.</cite> [<a href="https://doi.org/10.1186/1471-2105-10-168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2700099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19486540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinf.&amp;title=Fpocket:%20An%20Open%20Source%20Platform%20for%20Ligand%20Pocket%20Detection&amp;author=V.%20Le%20Guilloux&amp;author=P.%20Schmidtke&amp;author=P.%20Tuffery&amp;volume=10&amp;issue=1&amp;publication_year=2009&amp;pages=168&amp;pmid=19486540&amp;doi=10.1186/1471-2105-10-168&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>MacKerell A. D., Bashford D., Bellott M., Dunbrack R. L., Evanseck J. D., Field M. J., Fischer S., Gao J., Guo H., Ha S., Joseph-McCarthy D., Kuchnir L., Kuczera K., Lau F. T. K., Mattos C., Michnick S., Ngo T., Nguyen D. T., Prodhom B., Reiher W. E., Roux B., Schlenkrich M., Smith J. C., Stote R., Straub J., Watanabe M., Wiórkiewicz-Kuczera J., Yin D., Karplus M.. All-Atom Empirical Potential for Molecular Modeling
and Dynamics Studies of Proteins. J. Phys. Chem.
B. 1998;102(18):3586–3616. doi: 10.1021/jp973084f.</cite> [<a href="https://doi.org/10.1021/jp973084f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24889800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20B&amp;title=All-Atom%20Empirical%20Potential%20for%20Molecular%20Modeling%20and%20Dynamics%20Studies%20of%20Proteins&amp;author=A.%20D.%20MacKerell&amp;author=D.%20Bashford&amp;author=M.%20Bellott&amp;author=R.%20L.%20Dunbrack&amp;author=J.%20D.%20Evanseck&amp;volume=102&amp;issue=18&amp;publication_year=1998&amp;pages=3586-3616&amp;pmid=24889800&amp;doi=10.1021/jp973084f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>MacKerell A. D., Feig M., Brooks C. L.. Improved Treatment of the Protein
Backbone in Empirical Force Fields. J. Am. Chem.
Soc. 2004;126(3):698–699. doi: 10.1021/ja036959e.</cite> [<a href="https://doi.org/10.1021/ja036959e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14733527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Improved%20Treatment%20of%20the%20Protein%20Backbone%20in%20Empirical%20Force%20Fields&amp;author=A.%20D.%20MacKerell&amp;author=M.%20Feig&amp;author=C.%20L.%20Brooks&amp;volume=126&amp;issue=3&amp;publication_year=2004&amp;pages=698-699&amp;pmid=14733527&amp;doi=10.1021/ja036959e&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Best R. B., Zhu X., Shim J., Lopes P. E. M., Mittal J., Feig M., MacKerell A. D.. Optimization of the Additive CHARMM All-Atom Protein
Force Field Targeting Improved Sampling of the Backbone φ, ψ
and Side-Chain X1 and X2 Dihedral Angles. J.
Chem. Theory Comput. 2012;8(9):3257–3273. doi: 10.1021/ct300400x.</cite> [<a href="https://doi.org/10.1021/ct300400x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3549273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23341755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Optimization%20of%20the%20Additive%20CHARMM%20All-Atom%20Protein%20Force%20Field%20Targeting%20Improved%20Sampling%20of%20the%20Backbone%20%CF%86,%20%CF%88%20and%20Side-Chain%20X1%20and%20X2%20Dihedral%20Angles&amp;author=R.%20B.%20Best&amp;author=X.%20Zhu&amp;author=J.%20Shim&amp;author=P.%20E.%20M.%20Lopes&amp;author=J.%20Mittal&amp;volume=8&amp;issue=9&amp;publication_year=2012&amp;pages=3257-3273&amp;pmid=23341755&amp;doi=10.1021/ct300400x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Vanommeslaeghe K., Hatcher E., Acharya C., Kundu S., Zhong S., Shim J., Darian E., Guvench O., Lopes P., Vorobyov I., Mackerell A. D.. CHARMM
General Force Field: A Force
Field for Drug-like Molecules Compatible with the CHARMM All-Atom
Additive Biological Force Fields. J. Comput.
Chem. 2010;31(4):671–690. doi: 10.1002/jcc.21367.</cite> [<a href="https://doi.org/10.1002/jcc.21367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2888302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19575467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=CHARMM%20General%20Force%20Field:%20A%20Force%20Field%20for%20Drug-like%20Molecules%20Compatible%20with%20the%20CHARMM%20All-Atom%20Additive%20Biological%20Force%20Fields&amp;author=K.%20Vanommeslaeghe&amp;author=E.%20Hatcher&amp;author=C.%20Acharya&amp;author=S.%20Kundu&amp;author=S.%20Zhong&amp;volume=31&amp;issue=4&amp;publication_year=2010&amp;pages=671-690&amp;pmid=19575467&amp;doi=10.1002/jcc.21367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Scarsi M., Apostolakis J., Caflisch A.. Continuum Electrostatic Energies
of Macromolecules in Aqueous Solutions. J. Phys.
Chem. A. 1997;101(43):8098–8106. doi: 10.1021/jp9714227.</cite> [<a href="https://doi.org/10.1021/jp9714227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20A&amp;title=Continuum%20Electrostatic%20Energies%20of%20Macromolecules%20in%20Aqueous%20Solutions&amp;author=M.%20Scarsi&amp;author=J.%20Apostolakis&amp;author=A.%20Caflisch&amp;volume=101&amp;issue=43&amp;publication_year=1997&amp;pages=8098-8106&amp;doi=10.1021/jp9714227&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Vitalis A., Parker D., Radler F., Vargas-Rosales P. A., Zhang Y., Marchand J.-R., Caflisch A.. Parallelized Tools
for the Preparation and Curation of Large Libraries for Virtual Screening. J. Comput. Chem. 2025;46(7):e70074.  doi: 10.1002/jcc.70074.</cite> [<a href="https://doi.org/10.1002/jcc.70074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40048625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=Parallelized%20Tools%20for%20the%20Preparation%20and%20Curation%20of%20Large%20Libraries%20for%20Virtual%20Screening&amp;author=A.%20Vitalis&amp;author=D.%20Parker&amp;author=F.%20Radler&amp;author=P.%20A.%20Vargas-Rosales&amp;author=Y.%20Zhang&amp;volume=46&amp;issue=7&amp;publication_year=2025&amp;pages=e70074&amp;pmid=40048625&amp;doi=10.1002/jcc.70074&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Boonstra S., Onck P. R., Van Der
Giessen E.. CHARMM TIP3P Water Model Suppresses
Peptide Folding by Solvating the Unfolded State. J. Phys. Chem. B. 2016;120(15):3692–3698. doi: 10.1021/acs.jpcb.6b01316.</cite> [<a href="https://doi.org/10.1021/acs.jpcb.6b01316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27031562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20B&amp;title=CHARMM%20TIP3P%20Water%20Model%20Suppresses%20Peptide%20Folding%20by%20Solvating%20the%20Unfolded%20State&amp;author=S.%20Boonstra&amp;author=P.%20R.%20Onck&amp;author=E.%20Van%20Der%20Giessen&amp;volume=120&amp;issue=15&amp;publication_year=2016&amp;pages=3692-3698&amp;pmid=27031562&amp;doi=10.1021/acs.jpcb.6b01316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Huang J., Rauscher S., Nawrocki G., Ran T., Feig M., De Groot B. L., Grubmüller H., MacKerell A. D.. CHARMM36m:
An Improved Force Field for Folded and Intrinsically Disordered Proteins. Nat. Methods. 2017;14(1):71–73. doi: 10.1038/nmeth.4067.</cite> [<a href="https://doi.org/10.1038/nmeth.4067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5199616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27819658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=CHARMM36m:%20An%20Improved%20Force%20Field%20for%20Folded%20and%20Intrinsically%20Disordered%20Proteins&amp;author=J.%20Huang&amp;author=S.%20Rauscher&amp;author=G.%20Nawrocki&amp;author=T.%20Ran&amp;author=M.%20Feig&amp;volume=14&amp;issue=1&amp;publication_year=2017&amp;pages=71-73&amp;pmid=27819658&amp;doi=10.1038/nmeth.4067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Abraham M. J., Murtola T., Schulz R., Páll S., Smith J. C., Hess B., Lindah E.. GROMACS: High Performance
Molecular Simulations through Multi-Level Parallelism from Laptops
to Supercomputers. SoftwareX. 2015;1–2:19–25. doi: 10.1016/j.softx.2015.06.001.</cite> [<a href="https://doi.org/10.1016/j.softx.2015.06.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=SoftwareX&amp;title=GROMACS:%20High%20Performance%20Molecular%20Simulations%20through%20Multi-Level%20Parallelism%20from%20Laptops%20to%20Supercomputers&amp;author=M.%20J.%20Abraham&amp;author=T.%20Murtola&amp;author=R.%20Schulz&amp;author=S.%20P%C3%A1ll&amp;author=J.%20C.%20Smith&amp;volume=1%E2%80%932&amp;publication_year=2015&amp;pages=19-25&amp;doi=10.1016/j.softx.2015.06.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>McGibbon R. T., Beauchamp K. A., Harrigan M. P., Klein C., Swails J. M., Hernández C. X., Schwantes C. R., Wang L. P., Lane T. J., Pande V. S.. MDTraj: A Modern
Open Library for the Analysis of Molecular
Dynamics Trajectories. Biophys. J. 2015;109(8):1528–1532. doi: 10.1016/j.bpj.2015.08.015.</cite> [<a href="https://doi.org/10.1016/j.bpj.2015.08.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4623899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26488642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=MDTraj:%20A%20Modern%20Open%20Library%20for%20the%20Analysis%20of%20Molecular%20Dynamics%20Trajectories&amp;author=R.%20T.%20McGibbon&amp;author=K.%20A.%20Beauchamp&amp;author=M.%20P.%20Harrigan&amp;author=C.%20Klein&amp;author=J.%20M.%20Swails&amp;volume=109&amp;issue=8&amp;publication_year=2015&amp;pages=1528-1532&amp;pmid=26488642&amp;doi=10.1016/j.bpj.2015.08.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Pedregosa F., Varoquaux G., Gramfort A., Michel V., Thirion B., Grisel O., Blondel M., Prettenhofer P., Weiss R., Dubourg V., Vanderplas J., Passos A., Cournapeau D., Brucher M., Perrot M., Duchesnay É.. Scikit-Learn: Machine Learning in Python. J. Mach. Learn. Res. 2011;12(85):2825–2830. doi: 10.5555/1953048.2078195.</cite> [<a href="https://doi.org/10.5555/1953048.2078195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mach.%20Learn.%20Res.&amp;title=Scikit-Learn:%20Machine%20Learning%20in%20Python&amp;author=F.%20Pedregosa&amp;author=G.%20Varoquaux&amp;author=A.%20Gramfort&amp;author=V.%20Michel&amp;author=B.%20Thirion&amp;volume=12&amp;issue=85&amp;publication_year=2011&amp;pages=2825-2830&amp;doi=10.5555/1953048.2078195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371494/bin/bg5c00099_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00099_si_001.pdf</a><sup> (8.4MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.5c00099"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg5c00099.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371494/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371494/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371494%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371494/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371494/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371494/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371494/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371494/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371494/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="E1oYxtj3ykQ8JBuwgn6mtx30RatnYGVgS699mh4eYXogUR6fed8hH3TQbor1SDpv">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
