
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Integrated multi-omics analysis identifies lipid metabolism biomarkers in ONFH and reveals therapeutic potential of retinoic acid - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53328C8AF2760305328C0000F1EA76.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371116/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Integrated multi-omics analysis identifies lipid metabolism biomarkers in ONFH and reveals therapeutic potential of retinoic acid">
<meta name="citation_author" content="Chuan Wang">
<meta name="citation_author_institution" content="Department of Orthopedics, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China">
<meta name="citation_author" content="Chaode Cen">
<meta name="citation_author_institution" content="Department of Orthopedics, The Beijing Jishuitan Hospital Guizhou Hospital, Guiyang, 550014 Guizhou China">
<meta name="citation_author" content="Huachuan Su">
<meta name="citation_author_institution" content="Internal Medicine, Luodian of Hospital of Traditional Chinese Medicine, Qiannan 550100 Guizhou, China">
<meta name="citation_author" content="Yuhui Peng">
<meta name="citation_author_institution" content="Key Laboratory of Endemic and Ethnic Diseases, Ministry of Education &amp; Key Laboratory of Medical Molecular Biology of Guizhou Province, School of Basic Medical Science, Guizhou Medical University, Guiyang, 550004 Guizhou China">
<meta name="citation_author" content="Tao Wang">
<meta name="citation_author_institution" content="Department of Orthopedics, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China">
<meta name="citation_author_institution" content="Emergency Surgery, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China">
<meta name="citation_author" content="Jian Li">
<meta name="citation_author_institution" content="Department of Orthopedics, The Beijing Jishuitan Hospital Guizhou Hospital, Guiyang, 550014 Guizhou China">
<meta name="citation_author" content="Fei Zhang">
<meta name="citation_author_institution" content="Emergency Surgery, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China">
<meta name="citation_author" content="Wuxun Peng">
<meta name="citation_author_institution" content="Emergency Surgery, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30716">
<meta name="citation_doi" content="10.1038/s41598-025-13703-y">
<meta name="citation_pmid" content="40841396">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371116/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371116/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371116/pdf/41598_2025_Article_13703.pdf">
<meta name="description" content="Osteonecrosis of the femoral head (ONFH) is a debilitating condition frequently associated with dysregulation in lipid metabolism. The objective of this study was to identify potential biomarkers for ONFH through various analytical methods and ...">
<meta name="og:title" content="Integrated multi-omics analysis identifies lipid metabolism biomarkers in ONFH and reveals therapeutic potential of retinoic acid">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Osteonecrosis of the femoral head (ONFH) is a debilitating condition frequently associated with dysregulation in lipid metabolism. The objective of this study was to identify potential biomarkers for ONFH through various analytical methods and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371116/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371116">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-13703-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_13703.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371116%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371116/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371116/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371116/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 21;15:30716. doi: <a href="https://doi.org/10.1038/s41598-025-13703-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-13703-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Integrated multi-omics analysis identifies lipid metabolism biomarkers in ONFH and reveals therapeutic potential of retinoic acid</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Chuan Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Chuan Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Orthopedics, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chuan Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cen%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chaode Cen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chaode Cen</span></h3>
<div class="p">
<sup>2</sup>Department of Orthopedics, The Beijing Jishuitan Hospital Guizhou Hospital, Guiyang, 550014 Guizhou China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cen%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chaode Cen</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Su%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Huachuan Su</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Huachuan Su</span></h3>
<div class="p">
<sup>3</sup>Internal Medicine, Luodian of Hospital of Traditional Chinese Medicine, Qiannan 550100 Guizhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Su%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huachuan Su</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yuhui Peng</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yuhui Peng</span></h3>
<div class="p">
<sup>4</sup>Key Laboratory of Endemic and Ethnic Diseases, Ministry of Education &amp; Key Laboratory of Medical Molecular Biology of Guizhou Province, School of Basic Medical Science, Guizhou Medical University, Guiyang, 550004 Guizhou China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuhui Peng</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Tao Wang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Tao Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Orthopedics, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China </div>
<div class="p">
<sup>5</sup>Emergency Surgery, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tao Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jian Li</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jian Li</span></h3>
<div class="p">
<sup>2</sup>Department of Orthopedics, The Beijing Jishuitan Hospital Guizhou Hospital, Guiyang, 550014 Guizhou China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jian Li</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Fei Zhang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Fei Zhang</span></h3>
<div class="p">
<sup>5</sup>Emergency Surgery, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fei Zhang</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Wuxun Peng</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Wuxun Peng</span></h3>
<div class="p">
<sup>5</sup>Emergency Surgery, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wuxun Peng</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Orthopedics, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China </div>
<div id="Aff2">
<sup>2</sup>Department of Orthopedics, The Beijing Jishuitan Hospital Guizhou Hospital, Guiyang, 550014 Guizhou China </div>
<div id="Aff3">
<sup>3</sup>Internal Medicine, Luodian of Hospital of Traditional Chinese Medicine, Qiannan 550100 Guizhou, China </div>
<div id="Aff4">
<sup>4</sup>Key Laboratory of Endemic and Ethnic Diseases, Ministry of Education &amp; Key Laboratory of Medical Molecular Biology of Guizhou Province, School of Basic Medical Science, Guizhou Medical University, Guiyang, 550004 Guizhou China </div>
<div id="Aff5">
<sup>5</sup>Emergency Surgery, Affiliated Hospital of Guizhou Medical University, Guiyang, 550001 Guizhou China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 28; Accepted 2025 Jul 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371116  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841396/" class="usa-link">40841396</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Osteonecrosis of the femoral head (ONFH) is a debilitating condition frequently associated with dysregulation in lipid metabolism. The objective of this study was to identify potential biomarkers for ONFH through various analytical methods and experimental verification, thereby providing a robust reference for disease treatment. Utilizing public datasets, differential analysis was conducted and integrated with weighted gene co-expression network analysis to pinpoint key genes. Subsequently, biomarkers related to lipid metabolism in ONFH were identified by combining machine learning techniques with receiver operating characteristic curves. Furthermore, based on these biomarkers, a nomogram model was developed for diagnostic prediction. Immune infiltration was assessed, and drug predictions involved molecular docking studies. Finally, the relationship between biomarkers and immune cells was investigated through single-cell analysis. Then the system experiment is carried out to verify. Thirty key genes were identified, primarily enriched in "phospholipid metabolic processes" and “lysosome”. A total of three biomarkers—CREBBP, GLB1, and PSAP—were recognized as exhibiting strong diagnostic efficacy. The nomogram incorporating these biomarkers demonstrated accurate predictive capabilities. Immune infiltration analysis revealed altered levels of neutrophils and activated dendritic cells in ONFH that correlated with biomarker expression. Additionally, molecular docking studies indicated stable interactions between the identified biomarkers and potential therapeutic agents such as estradiol and retinoic acid. Single-cell sequencing analysis uncovered significant differences in biomarker expression across various immune cell types including T cells. Finally, it was verified that retinoic acid improved ONFH by hsa-miR-320a/GLB1 in vitro and in vivo. This study identifies CREBBP, GLB1, and PSAP as promising biomarkers for ONFH while underscoring their potential roles in lipid metabolism and immune modulation. Moreover, we confirmed the important conclusion that retinoic acid improves ONFH through hsa-miR-320a/GLB1. Future investigations into the therapeutic implications of these findings may offer new avenues for ONFH management.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Osteonecrosis of the femoral head, Lipid metabolism, Biomarkers, Single-cell RNA transcriptome</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Computational biology and bioinformatics, Medical research, Molecular medicine</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Osteonecrosis of the femoral head (ONFH) is a localized bone metabolic disorder caused by abnormal blood supply to the femoral head<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. With complete collapse of the femoral head in 80% of untreated patients, ONFH has become one of the most serious challenges faced by orthopedic surgeons<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Globally, there are over 30 million individuals with ONFH, with approximately 22,000 new cases annually in the United States<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The incidence of ONFH in adults in China is relatively high, predominantly affecting middle-aged and young individuals. The majority of patients present at a stage where the bone necrosis has advanced to the mid-to-late stages, with collapse of the femoral head. Despite numerous studies on the pathogenesis of ONFH, a consensus has not yet been reached<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Joint replacement surgery may offer the best therapeutic outcome. However, artificial prostheses have a finite lifespan, and young patients who undergo joint replacement may require one or more revisions later in life. Exploring early non-surgical treatment methods to delay or avoid joint replacement is currently a research direction for ONFH. Therefore, the development of diagnostic biomarkers for ONFH is crucial.</p>
<p id="Par3">The effective regulation and maintenance of lipid homeostasis is crucial for the normal function of cells and tissues. Patients with ONFH often have severe lipid metabolism disorders and fat accumulation in the damaged bone marrow cavity<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Therefore, lipid metabolism disorders are often recognized as a key factor in the pathogenesis of ONFH<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. In addition, dysregulation of lipid metabolism may lead to ischemia by increasing intraosseous pressure and reducing blood flow, ultimately resulting in non-traumatic ONFH (NONFH)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Decreased levels of adiponectin (APN) have been detected in patients with ONFH, and lower APN levels may reflect the severity of NONFH<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. In animal experimental models of ONFH, the first change observed is an increase in blood lipid levels<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Long-term and high-dose use of glucocorticoids can lead to lipid metabolism disorders. These disorders can result in the appearance of small fat droplets in the blood, which are prone to form thrombi within the small vessels in the femoral head area, thereby affecting the microvascular circulation of the femoral head. Concurrently, hormones also have a proliferative effect on adipose tissue in the body, which leads to significant proliferation and hypertrophy of intramedullary fat cells, and causes an increase in intramedullary pressureand ultimately ONFH. Given the critical role of lipid metabolism in ONFH, investigating its potential as a biomarker of ONFH is essential for the development of innovative therapeutic strategies.</p>
<p id="Par4">Building on previous findings and leveraging data from multiple public databases, this study aims to utilize transcriptomic data in conjunction with advanced machine learning techniques to identify biomarkers associated with lipid metabolism in ONFH. The characteristics of these biomarkers at the cellular level were further analyzed by single-cell RNA sequencing (scRNA-seq). Subsequently, we will experimentally validate the key genes that regulate lipid metabolism and further elucidate the critical molecular pathways involved in the pathogenesis of ONFH. The flowchart of this study visualizing multiple analytical methods was shown as Fig. <a href="#Fig1" class="usa-link">1</a>. This comprehensive approach will aid in a better understanding of the disease and provide more nuanced perspective for the clinical diagnosis of ONFH.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/1db050c4159c/41598_2025_13703_Fig1_HTML.jpg" loading="lazy" id="MO1" height="473" width="668" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The flowchart of this study.</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Data collection</h3>
<p id="Par5">This study utilized several datasets, including mRNA transcriptome datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a>), and the microRNA (miRNA) transcriptome dataset <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89587" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE89587</a>, all obtained from the Gene Expression Omnibus (GEO) database (<a href="https://www.ncbi.nlm.nih.gov/geo" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo</a>). The <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> dataset was based on the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL15207" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL15207</a> platform and contained 30 samples of ONFH and 10 control samples. The <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a> cohort, using the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13497" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL13497</a> platform, comprised 4 samples of ONFH and 4 control samples. The <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89587" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE89587</a> cohort, employing the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21439" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL21439</a> platform, included 10 samples of ONFH alongside 10 controls. Additionally, we extracted 720 lipid metabolism-related genes (LMRGs) from the Reactome database (<a href="https://reactome.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://reactome.org/</a>) (<a href="#MOESM1" class="usa-link">Table S1</a>). The scRNA-seq dataset SRP361778, sourced from the SRA database (<a href="https://www.ncbi.nlm.nih.gov/sra" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra</a>), included 6 samples of ONFH and 5 controls, providing a robust foundation for further analysis. The comprehensive table summarizing the key metadata for each dataset was exhibited in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>The specific information of the four datasets used in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"><strong>GEO accession numbers</strong></th>
<th align="left" colspan="1" rowspan="1"><strong>platform information</strong></th>
<th align="left" colspan="1" rowspan="1"><strong>sample sizes per group</strong></th>
<th align="left" colspan="1" rowspan="1"><strong>dataset publication dates</strong></th>
<th align="left" colspan="1" rowspan="1"></th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL15207" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL15207</a></td>
<td align="left" colspan="1" rowspan="1">30 SONFH patients and 10 non-SONFH patients (following steroid administration)</td>
<td align="left" colspan="1" rowspan="1">Public on Dec 31, 2019</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13497" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL13497</a></td>
<td align="left" colspan="1" rowspan="1">12 NFH patients and 12 healthy controls</td>
<td align="left" colspan="1" rowspan="1">Public on Jun 01, 2016</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89587" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE89587</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21439" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL21439</a></td>
<td align="left" colspan="1" rowspan="1">10 SONFH patients and 10 non-SONFH patients</td>
<td align="left" colspan="1" rowspan="1">Public on Sep 14, 2017</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SRP361778</td>
<td align="left" colspan="1" rowspan="1">ILLUMINA (Illumina NovaSeq 6000)</td>
<td align="left" colspan="1" rowspan="1">Control: ONFH = 5: 6</td>
<td align="left" colspan="1" rowspan="1">unknown</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Differential expression analysis</h3>
<p id="Par6">Differentially expressed genes (DEGs) between the ONFH and control groups were identified in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a> datasets using the limma package (v 3.54.0)<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. The selection criteria was set at |log2-fold change (FC)|&gt; 0.5 and FDR-adjusted <em>P</em>-value &lt; 0.05. The results from both datasets were then integrated to obtain common DEGs<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. For visualization, volcano plots and heatmaps were generated using the ggplot2 (v 3.3.3)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> and ComplexHeatmap (v 2.14.0)<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> packages, facilitating an intuitive representation of gene expression changes.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Weighted gene co-expression network analysis (WGCNA)</h3>
<p id="Par7">Co-expression network construction using the WGCNA package (v 1.70.3)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,</sup> with ONFH as a trait, was performed for the whole gene expression profile of the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> cohort. First, the samples were clustered and outlier samples were removed. The scale-free R<sup>2</sup> is close to 0.8 and the average connectivity is close to 0. The optimal soft threshold is chosen, at which point the network approaches a scale-free distribution. The minimum number of genes per gene module was set to 200 to divide the gene modules. Pearson correlation analysis was performed on the modules from the WGCNA results with ONFH, selecting modules with |cor| greater than 0.5 and <em>P </em>value less than 0.05, thereby extracting key module genes associated with ONFH.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Identification of key genes and functional enrichment analysis</h3>
<p id="Par8">The common DEGs were intersected with LMRGs and key module genes from the WGCNA to define the key genes in ONFH. Subsequently, Gene Ontology (GO) (FDR-adjusted <em>P</em>-value &lt; 0.05) and Kyoto Encyclopedia of Genes and Genomes (KEGG) (<em>P</em> value &lt; 0.05) functional enrichment analysis were conducted using the clusterProfiler package (v 3.18.0)<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. The KEGG pathway database is copyrighted by Kanehisa laboratories and we have obtained formal permission from them to publish this material commercially under an Open Access license<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.The results were visualized with ggplot2 and GOplot (v 3.6.1)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, allowing for an in-depth understanding of these genes’potential roles in biological processes.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Biomarker selection, chromosomal localization, and tissue expression analysis</h3>
<p id="Par9">The key genes were analyzed in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> cohort using the Glmnet package (v 3.6.0)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, the Boruta package (v 7.0.0)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, and the e1071 package (v1.7.0)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> for least absolute shrinkage and selection operator (LASSO) regression analysis, Boruta, and support vector machine-recursive feature elimination (SVM-RFE) machine learning algorithms, respectively. The ggvenn package (v 0.1.9)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> was utilized to extract the intersection of genes identified by the three algorithms as candidate biomarkers. Receiver operating characteristic (ROC) curves were plotted in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a> cohorts using the pROC package (v 1.13.0)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, and area under the curve (AUC) values were calculated to select candidate biomarkers with AUC values greater than 0.7 in both cohorts as biomarkers. The RCircos package (v 4.1.1)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> facilitated chromosomal localization analysis of these biomarkers, illustrating their positions and expression changes on chromosomes. To explore the tissue specificity of these feature genes, data from the Human Protein Atlas (<a href="http://www.proteinatlas.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.proteinatlas.org/</a>), Bgee (<a href="https://bgee.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bgee.org/</a>), and BioGPS (<a href="http://biogps.org/#goto=welcome" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://biogps.org/#goto=welcome</a>) were analyzed for biomarker distribution in human bone marrow, adipose tissue, and blood.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Construction of nomogram</h3>
<p id="Par10">Based on the expression levels of the biomarkers, a nomogram model was constructed in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> cohort using the rms package (v 3.5.0)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> to predict the probability of ONFH diagnosis. Each biomarker’s expression corresponded to a score, when summed, a total point was produced which used to estimate disease diagnostic probability. The accuracy of the nomogram model was evaluated by plotting a calibration curve. Additionally, ROC curves were generated using the pROC package to assess diagnostic capability, while decision curve analysis (DCA) was performed with the rmda package (v 3.4.4)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> to evaluate the clinical value of the nomogram model.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Immune infiltration analysis</h3>
<p id="Par11">Utilizing the CIBERSORT algorithm (v 1.03)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> and LM22 gene set<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, we calculated the proportions of 22 immune cell types in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> cohort. Bar plot illustrating the abundance percentages of immune cell infiltration was generated using ggplot2, and comparisons between ONFH and control groups were conducted through Wilcoxon test, as depicted in box plot. Spearman correlation analysis was conducted using the psych package (v 2.1.6)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> to assess the relationship between biomarkers and 22 immune cells, with results visualized in heatmap via the ggplot2 package.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Construction of regulatory network and prediction of transcription factors (TFs)</h3>
<p id="Par12">Differentially expressed miRNAs were identified in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89587" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE89587</a> dataset using the same methods as for <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a> cohort. Interactions between biomarkers and miRNAs were predicted via the starBase database (<a href="https://rnasysu.com/encori/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://rnasysu.com/encori/</a>), followed by predictions of relationships between miRNAs and long non-coding RNAs (lncRNAs)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. TFs involved in biomarkers regulation were also predicted using the ChEA3 database (<a href="https://maayanlab.cloud/chea3/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://maayanlab.cloud/chea3/</a>), selecting TFs supported by ChIP-seq data from the ENCODE database (<a href="https://www.encodeproject.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.encodeproject.org/</a>). Network diagrams were visualized using the Cytoscape software (v3.9.1). Combined with the CytoHubba plugin, centrality indices such as Degree, Betweenness, and Eccentricity were calculated to identify key nodes<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Drug prediction and molecular docking</h3>
<p id="Par13">Potential targeted drugs for biomarkers were predicted through the DSigDB database (<a href="https://dsigdb.tanlab.org/DSigDBv1.0/geneSearch.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dsigdb.tanlab.org/DSigDBv1.0/geneSearch.html</a>) to explore possible therapeutic strategies. To validate the binding stability of predicted drugs with biomarkers, the highest-scoring drug was selected for molecular docking. Protein structures corresponding to biomarkers were obtained from the PDB database (<a href="https://www.rcsb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/</a>), while the 3D molecular structures of corresponding compounds were sourced from PubChem (<a href="https://pubchem.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubchem.ncbi.nlm.nih.gov/</a>) and AlphaFold (<a href="https://alphafold.ebi.ac.uk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alphafold.ebi.ac.uk/</a>). Molecular docking was performed using Autodock software to calculate the binding energy between ligands and receptors, with binding energies ≤ − 5.0 kcal/mol considered stable<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Gene set enrichment analysis (GSEA) and gene set variation analysis (GSVA)</h3>
<p id="Par14">Gene sets (c2.cp.kegg_legacy.v2024.1.Hs.symbols.gmt, h.all.v2023.2.Hs.symbols.gmt) were utilized for Spearman correlation analysis between each biomarker and all genes. GSEA based on the KEGG and Hallmark databases was conducted using the clusterProfiler package, with criteria set at |NES|&gt; 1 and <em>P</em>-adjust &lt; 0.05. To further understand the differences in pathways during disease progression between groups, the ONFH and control groups were differentiated by median based on the height of the nomogram scores, and then GSVA analysis was performed using GSVA (v1.42.0)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and the limma package, which utilizes thresholds |t|&gt; 2 and <em>P</em> value &lt; 0.05 to show the top 10 up- and down-regulated pathways.</p></section><section id="Sec13"><h3 class="pmc_sec_title">ScRNA-seq analysis</h3>
<p id="Par15">Quality control was first performed on the scRNA-seq dataset SRP361778 using the Seurat package (v4.1.1)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, filtering out genes covered by fewer than 200 cells and those with over 10% mitochondrial gene content. Cells with ≤ 200 and ≥ 8000 genes in the cell were excluded, and genes with count numbers ≤ 200 and ≥ 80,000 were removed. Data normalization was conducted with the NormalizeData function, followed by extraction of the top 2000 highly variable genes (HVGs) using the FindVariableFeatures function. Data scaling was performed using ScaleData, and significant principal components were determined with the JackStrawPlot function. Neighbor identification and clustering of small cell groups were conducted via FindNeighbors and FindClusters, employing a resolution of 0.2 and utilizing the t-SNE clustering and UMAP method for visualization with RunTSNE and RunUMAP, respectively. The FindAllMarkers function was used to identify the top 100–200 HVGs in each cluster, which were cross-referenced with the Cell Marker (<a href="http://bio-bigdata.hrbmu.edu.cn/CellMarker/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bio-bigdata.hrbmu.edu.cn/CellMarker/index.html</a>) and PanglaoDB (<a href="https://panglaodb.se/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://panglaodb.se/</a>) databases (accessed on 25 June 2025) for cell annotation<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Finally, biomarker expression in cells was analyzed using FindMarkers and Wilcox.test, applying thresholds of |logFC|&gt; 2 and Bonferroni-corrected <em>P</em>-values &lt; 0.05.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Ethical Approval</h3>
<p id="Par16">The experimental animals, specifically SD rats, sourced from the Experimental Animal Center of Guizhou Medical University, where the animal experiments were conducted. The facility holds a certificate numbered SYXK (Qian) 2018–0001. All procedures involving the animals adhered strictly to the guidelines outlined in the "Guide for the Care and Use of Laboratory Animals" published by the National Institutes of Health (NIH; No.85–23, revised 1996). The experimental protocol underwent rigorous review and received approval from the Ethical Committee of Guizhou Medical University, under reference number 2304177. Compliance with the International Council for Laboratory Animal Science (ICLAS) standards for the protection of laboratory animals used for scientific purposes was ensured, as well as adherence to the ARRIVE guidelines (Animal Research: Reporting In Vivo Experiments). Every endeavor was made to minimize both the suffering of the animals and the number of animals utilized in this study.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Early ONFH model</h3>
<p id="Par17">The SD rats that remained were utilized to establish the model of ONFH, while an additional 20 healthy adult male rats were randomly selected to serve as the healthy control group. Subsequently, the rats were stratified into three distinct groups: normal, glucocorticoid (GC) treated, and retinoic acid treated, with each group comprising 20 SD rats. Prior to the daily administration of methylprednisolone (MP; 60 mg/kg; Pfizer, USA) into the gluteal muscle for a duration of 10 days<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, the buttocks of the rats were meticulously shaved and disinfected with alcohol. The body weight of each rat was recorded before each injection. Four weeks following the completion of MP treatment, samples were taken for subsequent experiments.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Anesthesia of experimental animals and euthanasia</h3>
<p id="Par18">All experimental animals were anesthetized via intraperitoneal injection prior to any surgical procedures to mitigate pain and discomfort. The anesthesia dosage was standardized at 0.25 ml per 100 g of body weight for rats. Specifically, the anesthesia cocktail consisted of a mixture of pentobarbital sodium (50 mg/kg) and ketamine (25 mg/kg), administered through intraperitoneal injection [i.p.]. Following the completion of experiments, the rats were euthanized through an overdose of sevoflurane.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Cellular immunofluorescence</h3>
<p id="Par19">Bone marrow mesenchymal stem cells (BMSCs) treated with glucocorticoids were rinsed in phosphate-buffered saline (PBS) and subsequently fixed with 10% neutral-buffered formalin for a duration at 21 °C for 30 min. To facilitate membrane permeabilization, the cells were incubated with Triton X-100 (sourced from Beyotime Biotechnology, Beijing, China) for 10 min. Blocking of nonspecific binding sites was achieved through a 1-h incubation with donkey serum (also from Beyotime Biotechnology). The cells were then incubated overnight at 4 °C with the primary antibody GLB1 (ab305174; Abcam; diluted 1:100). After three washes with PBS, the cells were incubated for 1 h at 21 °C with Alexa Fluor 594-conjugated goat anti-rabbit secondary antibody (A-11012, Thermo Fisher Scientific; diluted 2 µg/mL). Nuclei staining was accomplished with DAPI (Beyotime Institute of Biotechnology, Jiangsu, China) for 10 min. The samples were then rinsed with PBS and visualized using a confocal microscope (Olympus Life Science, Tokyo, Japan).</p></section><section id="Sec18"><h3 class="pmc_sec_title">RNA extraction and real-time quantitative reverse transcription polymerase chain reaction (qRT-PCR)</h3>
<p id="Par20">Primary cultured BMSCs were subjected to total RNA extraction using TRIzol reagent sourced from Invitrogen (Carlsbad, CA, USA). Subsequently, 2 µg of purified RNA underwent reverse transcription utilizing the RevertAid First Strand cDNA Synthesis Kit provided by Thermo Fisher Scientific (Boston, MA, USA). The qRT-PCR was executed on the Applied Biosystems StepOnePlus Real-Time PCR System (Foster City, CA, USA), employing SYBR Green PCR Master Mix obtained from Toyobo (Japan). The expression levels of the target mRNAs were normalized against the expression of 18S ribosomal RNA (Rn18s). The relative gene expression levels were calculated using the 2<sup> − ΔΔCT</sup> method and presented as fold-changes. Each qRT-PCR experiment was conducted in triplicate, with a minimum of three distinct biological replicates included. The primer sequences utilized in this study are detailed in Table <a href="#Tab2" class="usa-link">2</a>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>The qRT-PCR primer sequences used in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"><strong>Gene</strong></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"><strong>Sequence</strong></th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">GLB1</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">GCACGGCATCTATAATGTCACC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">R</td>
<td align="left" colspan="1" rowspan="1">GTATCGGAATGGCTGTCCATC</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">CREBBP</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">AAGAGCCCTCTGAACCAAGGA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">R</td>
<td align="left" colspan="1" rowspan="1">CTGTGATGTGGCAGGACTACT</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">PSAP</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">TGCTGAAAGATAATGCTACGCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">R</td>
<td align="left" colspan="1" rowspan="1">GCAGGTAAGAGTCAACCACCTC</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">hsa-miR-320a</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">CGGGGAGAGGGCGAAA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">R</td>
<td align="left" colspan="1" rowspan="1">AGTGCAGGGTCCGAGGTATT</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">GAPDH</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">CTCCTCCACCTTTGACGC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">R</td>
<td align="left" colspan="1" rowspan="1">CCACCACCCTGTTGCTGT</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec19"><h3 class="pmc_sec_title">H&amp;E staining</h3>
<p id="Par21">The bone tissue underwent decalcification using 10% EDTA supplied by Solarbio. Following this, dehydration was achieved through an alcohol gradient sourced from Sinopharm (China). Transparency was induced using xylene, also from Sinopharm, and the tissue was subsequently embedded in paraffin, also sourced from Sinopharm. The bone tissue was sectioned using a Leica RM2016 pathological slicer (Leica, Germany) and baked at 60 °C for 3 h. The slices were dewaxed sequentially in xylene and a gradient of alcohols. The dewaxed sections were stained with Mayer’s hematoxylin and 1% water-soluble eosin solution, both obtained from Solarbio. The stained slices were then dehydrated in a graded series of alcohols, rendered transparent in xylene, and finally mounted with neutral gum sourced from Sinopharm. Images were captured using a BX53 microscope manufactured by Olympus (Japan).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Tissue immunofluorescence</h3>
<p id="Par22">Bone tissue was decalcified, dehydrated, rendered transparent, waxed, embedded, sliced, baked, and dewaxed (the procedure was the same as that outlined for H&amp;E staining). Antigen retrieval was performed using the microwave method, and bone tissue antigen retrieval solution was purchased from BioShun (Shanghai, China). Bone tissues were blocked with goat serum (Solarbio) at 25 °C for 30 min. Anti-GLB1 antibody (ab305174; Abcam; diluted 1:100) was used for the primary antibody reaction (incubated overnight at 4 °C). After three washes with PBS, the cells were incubated for 1 h at 21 °C with either Alexa Fluor 594-conjugated goat anti-rabbit secondary antibody (A-11012, Thermo Fisher Scientific; diluted 2 µg/mL). Nuclei staining was accomplished with DAPI (Beyotime Institute of Biotechnology, Jiangsu, China) for 10 min. The samples were then rinsed with PBS and visualized using a confocal microscope (Olympus Life Science, Tokyo, Japan).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par23">All statistical analyses were performed using R version 4.4.1, inter-group differences were evaluated via the Wilcoxon test, with statistical significance set at <em>P</em> &lt; 0.05. Data analysis was further facilitated by SPSS statistical software version 22.0, while statistical graphs were generated using GraphPad Prism 6.0. For quantitative data, normality was assessed using the Kolmogorov–Smirnov test, and variance homogeneity between groups was tested using analysis of variance. Data that exhibited normal distribution and homogeneous variance were reported as the mean ± standard deviation (SD). When comparing only two groups, a two-tailed unpaired Student’s t-test was employed. For analysis involving more than two groups, one-way ANOVA with Tukey’s post hoc test was utilized. Data that did not conform to normal distribution were described by their median (P25, P75). For comparisons between such groups, the Kruskal–Wallis rank-sum test with the DSCF method was applied. <em>P</em> value &lt; 0.05 was considered indicative of a statistically significant difference.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Result</h2>
<section id="Sec23"><h3 class="pmc_sec_title">Exploring the potential functions of key genes</h3>
<p id="Par24">In our study on the lipid metabolism in ONFH, we conducted a comprehensive differential expression analysis between the ONFH and control group across two datasets. The analysis of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> identified 1,977 DEGs, with 1,343 showing upregulation and 634 downregulation (<a href="#MOESM2" class="usa-link">Table SCite ESM.2</a>). Meanwhile, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a> cohort revealed a total of 8,717 DEGs (<a href="#MOESM3" class="usa-link">Table S3</a>), including 5,481 upregulated and 3,236 downregulated genes (Fig. <a href="#Fig2" class="usa-link">2</a>A). By integrating the differential analysis results from both datasets, we obtained 596 common DEGs as shown in Fig. <a href="#Fig2" class="usa-link">2</a>B, among which 530 were upregulated and 66 downregulated. To identify genes with a strong correlation to ONFH, we performed a module gene selection process. Cluster analysis confirmed the integrity of our sample statuses, and eliminated the need to remove any outliers (Fig. <a href="#Fig2" class="usa-link">2</a>C). We established an optimal soft threshold of 18 for achieving a scale-free distribution and then set the minimum number of genes per module to 200 thereby identifying 10 modules (Fig. <a href="#Fig2" class="usa-link">2</a>D-E). Notably, the MEbrown (cor = 0.68, <em>P</em> &lt; 0.001), MEgreen (cor = −0.75, <em>P</em> &lt; 0.001), and MEyellow modules (cor = 0.57, <em>P</em> &lt; 0.001) emerged as critical, collectively encompassing 2,510 key module genes (Fig. <a href="#Fig2" class="usa-link">2</a>F, <a href="#MOESM4" class="usa-link">Table S4</a>). Ultimately, we intersected the identified common DEGs, key module genes from WGCNA, and LMRGs, resulting in 30 key genes (Fig. <a href="#Fig3" class="usa-link">3</a>A).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/372609bfdbca/41598_2025_13703_Fig2_HTML.jpg" loading="lazy" id="MO2" height="944" width="743" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential expression analysis and weighted gene co-expression network analysis (WGCNA). (<strong>A</strong>) volcano plot of differentially expressed genes (DEGs) and heatmap of the expression of the top 5 DEGs in datasets <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a>. Red dots represent up-regulated DEGs, blue dots represent down-regulated DEGs, and gray dots are genes that are not statistically significant. (<strong>B</strong>) the Venn diagram visualizing the common items between these two datasets. (<strong>C</strong>) sample clustering graph. <strong>(</strong><strong>D</strong>) screening for soft thresholds to ensure the network conforms to a scale-free topology. (<strong>E</strong>) module dynamic shear tree. <strong>F:</strong> heatmap of the correlation of the module with osteonecrosis of the ONFH.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/3159f3e5b415/41598_2025_13703_Fig3_HTML.jpg" loading="lazy" id="MO3" height="342" width="668" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Acquisition and enrichment analysis of key genes. (<strong>A</strong>) DEGs, key module genes from the WGCNA and 720 lipid metabolism-related genes (LMRGs) common to both <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a> datasets intersected in 30 key genes. (<strong>B</strong>) Gene Ontology (GO) enrichment analysis of key genes. MF, Molecular Function; BP, Biological Process; CC, Cellular Component. (<strong>C</strong>) Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis of key genes.</p></figcaption></figure><p id="Par25">To delve into the functional roles of these key genes, we conducted enrichment analyses. A total of 284 significant entries were enriched in GO enrichment, of which 172 entries were significantly enriched in biological processes, 12 in cellular components, and 100 in molecular functions. Specifically, it was primarily enriched in the"regulation of lipid metabolic processes","phospholipid metabolic processes", and“lipid droplets”(Fig. <a href="#Fig3" class="usa-link">3</a>B, <a href="#MOESM5" class="usa-link">Table S5</a>). These pathways align closely with our research focus. Furthermore, the KEGG analysis highlighted 17 significant pathways, including“Sphingolipid metabolism”,“Lysosome”, and“Glycerophospholipid metabolism”(Fig. <a href="#Fig3" class="usa-link">3</a>C, <a href="#MOESM7" class="usa-link">Table S6</a>). These findings indicate potential functional mechanisms underlying ONFH in patients.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Three biomarkers (CREBBP, GLB1, and PSAP) were identified</h3>
<p id="Par26">In a bid to identify potential biomarkers, we employed three machine learning techniques. LASSO regression analysis yielded 3 candidate genes (CREBBP, GLB1, and PSAP) (Fig. <a href="#Fig4" class="usa-link">4</a>A), while Boruta analysis identified 14 genes (such as LPCAT1, MID1IP1, PLIN3, EP300, MFSD2B, and so on) (Fig. <a href="#Fig4" class="usa-link">4</a>B). SVM-RFE further refined our selection to 24 genes (including PSAP, ORMDA3, OSBPL8, MFSD2B, GPS2, and etc.) (Fig. <a href="#Fig4" class="usa-link">4</a>C). By intersecting the results from these analyses, we identified 3 candidate biomarkers: CREBBP, GLB1, and PSAP (Fig. <a href="#Fig4" class="usa-link">4</a>D). To evaluate the diagnostic potential of these candidate biomarkers, we generated ROC curves across both cohorts. Notably, all candidate biomarkers demonstrated the AUC values exceeding 0.9 (Fig. <a href="#Fig4" class="usa-link">4</a>E), indicating strong diagnostic capacity, thus qualifying them as potential biomarkers. Further analysis of the chromosomal locations revealed that CREBBP is located on chromosome 16, GLB1 on chromosome 3, and PSAP on chromosome 10 (Fig. <a href="#Fig4" class="usa-link">4</a>F). We also examined the expression of these biomarkers in human bone marrow, adipose tissue, and blood, highlighting their tissue-specific expression (Fig. <a href="#Fig4" class="usa-link">4</a>G). Particularly, PSAP exhibited high expression across all three databases, while GLB1 showed elevated levels in two of the databases. These findings suggest that these biomarkers may play significant roles in tissues such as bone marrow and adipose tissue.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/6edf5d3cf7f7/41598_2025_13703_Fig4_HTML.jpg" loading="lazy" id="MO4" height="341" width="668" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Acquisition of biomarkers, chromosomal localization, and tissue expression analysis. (<strong>A</strong>) least absolute shrinkage and selection operator (LASSO) regression analysis. (<strong>B</strong>) boruta analysis. In the left panel, the box-and-line plot indicate the minimum, average, and maximum Z-values of the traits. The right panel shows the evolution of the Z-Score of the attributes during the Boruta run. (<strong>C</strong>) support vector machine-recursive feature elimination (SVM-RFE) screening of candidate genes. The horizontal coordinate indicates the number of genes and the vertical coordinate indicates the error rate. (<strong>D</strong>) feature genes are obtained after 3 types of machine learning on key genes, and the intersection was taken to obtain the candidate biomarkers. (<strong>E</strong>) receiver operating characteristic (ROC) curves were plotted for the candidate genesin two datasets, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a>, and the candidate genes with high diagnostic ability [area under the curve (AUC) values greater than 0.7] were selected as biomarkers. (<strong>F</strong>) localization of biomarkers on chromosomes. (<strong>G</strong>) expression of biomarkers in tissues was analyzed using three databases [Human Protein Atlas Project (HPA), Bgee, and BioGPS].</p></figcaption></figure></section><section id="Sec25"><h3 class="pmc_sec_title">A nomogram model for predicting ONFH risk was constructed</h3>
<p id="Par27">The nomogram is a visual statistical tool designed to provide quantitative disease risk assessment for individual patients by combining information from multiple clinical and biomarkers. To synthesize our findings, we constructed a nomogram model utilizing the three identified biomarkers as independent diagnostic factors to predict the risk of ONFH in patients (Fig. <a href="#Fig5" class="usa-link">5</a>A). The calibration curve corresponding to the nomogram illustrated a slope close to 1, indicating accurate predictions (Fig. <a href="#Fig5" class="usa-link">5</a>B). Additionally, the ROC curve demonstrated that the combined AUC value for the three biomarkers was 0.95, which was greater than the value of each of the three biomarkers [CREBBP (0.940), GLB1 (0.913), PSAP (0.923)], further validated the robustness of our diagnostic model (Fig. <a href="#Fig5" class="usa-link">5</a>C). The DCA indicated the clinical applicability of our diagnostic model (Fig. <a href="#Fig5" class="usa-link">5</a>D). These results affirm that the biomarkers can effectively predict the risk of ONFH, providing a theoretical foundation for early disease intervention.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/d33fd7f559dc/41598_2025_13703_Fig5_HTML.jpg" loading="lazy" id="MO5" height="455" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Constructing the nomogram. (<strong>A</strong>) construction of the nomogram. (<strong>B</strong>) calibration chart of the nomogram. (<strong>C</strong>) ROC curve of the nomogram. (<strong>D</strong>) decision curve analysis (DCA) analysis of the nomogram.</p></figcaption></figure></section><section id="Sec26"><h3 class="pmc_sec_title">Immune microenvironment in ONFH</h3>
<p id="Par28">In ONFH, infiltration of immune cells not only affects the local inflammatory response, but may also be involved in bone metabolism and remodeling processes, thus influencing the pathological process. To investigate the immune cell infiltration between the control and ONFH groups, we depicted the abundance percentage of 22 immune cell types (Fig. <a href="#Fig6" class="usa-link">6</a>A). Notably, “Neutrophils” were highly infiltrated in ONFH, while "Dendritic cells activated" and "B cells memory" were less infiltrated in ONFH (Fig. <a href="#Fig6" class="usa-link">6</a>B). We further explored the relationships between immune cells and biomarkers, revealing a strong negative correlation between PSAP and activated dendritic cells (cor = −0.59, <em>P</em> &lt; 0.001), as well as between GLB1 and activated Dendritic cells (cor = −0.58, <em>P</em> &lt; 0.001), while CREBBP showed a robust positive correlation with neutrophils (cor = 0.68, <em>P</em> &lt; 0.001) (Fig. <a href="#Fig6" class="usa-link">6</a>C). We can observe a high degree of CREBBP infiltration in neutrophils. By analyzing the microenvironment in which ONFH’s immune cells exist and exploring the role that biomarkers play in it, we can help predict the prognosis of the disease and provide early defense.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/fa3c811cefbb/41598_2025_13703_Fig6_HTML.jpg" loading="lazy" id="MO6" height="1180" width="695" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Immune infiltration analysis. (<strong>A</strong>) stacked bar graph of immune cell scores. (<strong>B</strong>) box line graph of immune cell scores in control and ONFH groups. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01. (<strong>C</strong>) bubble plot of the correlation of biomarkers and immune cells. The size of the dots represents the significance of the correlation, and the color of the dots as well as the shade represents the direction and size of the correlation.</p></figcaption></figure></section><section id="Sec27"><h3 class="pmc_sec_title">Exploring the regulatory relationship between biomarkers and miRNAs</h3>
<p id="Par29">To further elucidate the regulatory mechanisms of our identified biomarkers, we conducted differential expression analysis of miRNAs between ONFH and control samples (<a href="#MOESM8" class="usa-link">Table S7</a>). This analysis yielded 6 differentially expressed miRNAs, with 4 showing upregulation and 2 downregulation (Fig. <a href="#Fig7" class="usa-link">7</a>A-B). We explored potential interactions between the 3 biomarkers and the 6 differentially expressed miRNAs, identifying 3 pairs of mRNA-miRNA interactions. Subsequently, we predicted interactions involving 17 miRNAs and lncRNAs, suggesting multiple regulatory pathways, such as MALAT1 and NEAT1 indirectly influencing GLB1 via hsa-miR-320a (<a href="#MOESM9" class="usa-link">Table S8</a>). Moreover, we predicted TFs corresponding to our 3 biomarkers. Notably, TCF7L2, CTCF, and EGR1 were predicted to be common across all 3 biomarkers (<a href="#MOESM10" class="usa-link">Table S9</a>). A comprehensive ceRNA network was constructed by integrating miRNA-lncRNA/mRNA and mRNA-TF regulatory interactions, and comprehensive scoring-based network analysis revealed that hub nodes with comprehensive scores greater than 1 were identified, such as GLB1, PSAP, hsa-miR-93-5p, hsa-let-7i-5p, CTCF, EGR1, TCF7L2, CREBBP, hsa-miR-320a, PBX3, which were consistent with the above findings (Fig. <a href="#Fig7" class="usa-link">7</a>C, <a href="#MOESM11" class="usa-link">Table S10</a>). Moreover, it was found that 10 TFs including PBX3, REST, and IRF1 not only regulate the expression of biomarkers but are also regulated by hsa-miR-93-5p and hsa-let-7i-5p (<a href="#MOESM12" class="usa-link">Table S11</a>). These results provide new understanding of the complex regulatory mechanisms of biomarkers.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/0029ef1069fb/41598_2025_13703_Fig7_HTML.jpg" loading="lazy" id="MO7" height="421" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction of the regulatory network and drug prediction. (<strong>A</strong>) volcano plot of differentially expressed microRNAs (miRNAs). (<strong>B</strong>) heatmap of the expression of the top 20 differentially expressed miRNAs. (<strong>C</strong>) the ceRNA and transcription factors (TFs) interaction network of biomarkers. From red to blue represents the comprehensive score from high to low. (<strong>D</strong>) drug target action network of biomarkers. Red diamonds represent drugs and blue circles represent biomarkers.</p></figcaption></figure></section><section id="Sec28"><h3 class="pmc_sec_title">Potential drug treatment for ONFH</h3>
<p id="Par30">To further explore potential targeted therapies for ONFH, we conducted a drug prediction analysis involving our three identified biomarkers. This analysis yielded a total of 42 potential drugs with 56 interaction relationships identified (Fig. <a href="#Fig7" class="usa-link">7</a>D). Notably, CREBBP was linked to the highest number of drugs. Strikingly, both estradiol and retinoic acid were predicted to interact with all three biomarkers: CREBBP, GLB1, and PSAP. Following this, we selected the top-scoring drugs for molecular docking studies. Generally, a molecular binding energy below −5.0 kcal/mol indicates a favorable binding affinity. It was not difficult to see that the molecular binding energies of CREBBP, GLB1 and PSAP to estradiol and retinoic acid, respectively, were all less than −7 kcal/mol (<a href="#MOESM13" class="usa-link">Table S12</a>), indicating strong binding potential. Specifically, the docking results revealed hydrogen bonds forming between the residues GLN-411 and TYR-413 of GLB1 with estradiol and retinoic acid, while CREBBP showed hydrogen bonding between residue MET-1160 and retinoic acid (Fig. <a href="#Fig8" class="usa-link">8</a>). These findings suggest that the predicted drugs may form stable complexes with the biomarkers which highlight their potential as therapeutic targets.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/854ce91e9fcb/41598_2025_13703_Fig8_HTML.jpg" loading="lazy" id="MO8" height="359" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Diagram of molecular docking binding sites. In the diagram cyan is the protein molecular structure, green is the compound and yellow dashed line is the hydrogen bonding junction.</p></figcaption></figure></section><section id="Sec29"><h3 class="pmc_sec_title">Potential functional exploration</h3>
<p id="Par31">To delve deeper into the functional roles associated with our three biomarkers, we conducted GSEA. In the KEGG context, both GLB1 and PSAP were enriched in similar pathways, including “lysosome”,“proteasome”, and “spliceosome”. In the Hallmark context, all 3 biomarkers were activated in "interferon gamma (IFN-γ) response", while GLB1 and PSAP were activated in "MYC targets V1" and “oxidative phosphorylation” (Fig. <a href="#Fig9" class="usa-link">9</a>A-C). To further understand the differential pathway between ONFH and control groups, we identified 21 pathways that exhibited significant differences in the KEGG and Hallmark backgrounds. Notably, the “inflammatory response”, "fatty acid metabolism", and “spliceosome” were activated in the ONFH group, while other pathways, including "VEGF signaling pathway" was suppressed (Fig. <a href="#Fig9" class="usa-link">9</a>D). Based on the analysis results, we can identify key biological pathways and functional modules, explore biomarker-related functions, and further understand the mechanism of the disease.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/a5d10f6005db/41598_2025_13703_Fig9_HTML.jpg" loading="lazy" id="MO9" height="629" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene set enrichment analysis (GSEA) and gene set variation analysis (GSVA). (<strong>A</strong>) GSEA of CREBBP, GLB1, and PSAP in KEGG background, respectively. (<strong>B</strong>) GSEA of CREBBP, GLB1, and PSAP in Hallmark background, respectively. (<strong>C</strong>) GSVA between ONFH and control groups in Hallmark background. (<strong>D</strong>) GSVA between ONFH and control groups in KEGG background.</p></figcaption></figure></section><section id="Sec30"><h3 class="pmc_sec_title">Characterization of biomarkers by scRNA-seq analysis</h3>
<p id="Par32">Utilizing PCA permutation testing and elbow plots, we selected the top 50 principal components for subsequent analysis (Fig. <a href="#Fig10" class="usa-link">10</a>A, <a href="#MOESM6" class="usa-link">S1</a>). Clustering revealed 17 distinct cell clusters, which were characterized against marker genes to identify their respective cell types (Fig. <a href="#Fig10" class="usa-link">10</a>B-C). Ultimately, we categorized the clusters into 15 cell types (Fig. <a href="#Fig10" class="usa-link">10</a>D and <a href="#MOESM14" class="usa-link">Table S13</a>), including T cells, myeloid cells, fibroblasts, B cells, and others. On the basis of t-SNE clustering, UMAP has validated the favorable spatial distribution distinction of these cell types (<a href="#MOESM6" class="usa-link">Figure S2A</a>). We further analyzed the expression levels and distribution of our biomarkers across different cell types (Fig. <a href="#Fig10" class="usa-link">10</a>E-F, <a href="#MOESM6" class="usa-link">S2B</a>). The results indicated that all 3 biomarkers were widely distributed among most cell types, with PSAP predominantly expressed in myeloid cells. Additionally, significant differential expression of the biomarkers was observed across various cell classifications, co-expressed in multiple cells, e.g. T cells, myeloid cells, plasma cells, etc. This reinforces their relevance in the context of ONFH.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/0da9d595587e/41598_2025_13703_Fig10_HTML.jpg" loading="lazy" id="MO10" height="436" width="669" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Single-cell analysis in SRP361778. (<strong>A</strong>) principal component analysis (PCA) dimensionality analysis. The left graph demonstrates the degree of explanation of each principal component on the variance of the data; the right graph demonstrates the significance of different PCs in explaining the dataset compared to the random data. (<strong>B</strong>) cell clusters for cluster analysis. Each point in the figure represents a single cell. (<strong>C</strong>) plot of top100 ~ 200 of highly variable genes compared with marker genes of each cell type. (<strong>D</strong>) cell annotation plot using t-SNE. (<strong>E</strong>) distribution of biomarkers in cell types (t-SNE). (<strong>F</strong>) expression of biomarkers in cell types. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; *** <em>P</em> &lt; 0.001; **** <em>P</em> &lt; 0.0001.</p></figcaption></figure></section><section id="Sec31"><h3 class="pmc_sec_title">Retinoic acid could regulate hsa-miR-320a/GLB1 to improve ONFH</h3>
<p id="Par33">Based on the results of the above multiple analyses, we first verified the expression of GLB1, CREBBP and PSAP in the normal femoral head and ONFH group, and the results showed that the expression of these three genes was different, among which GLB1 showed the most significant trend of high expression in the ONFH group (Fig. <a href="#Fig11" class="usa-link">11</a>A). Therefore, we selected GLB1 for immunofluorescence detection in BMSCs and femoral head tissues, and showed a tendency of high expression in the ONFH group, consistent with qRT-PCR results (Fig. <a href="#Fig11" class="usa-link">11</a>B,E). After treatment with retinoic acid, it was found that GLB1 expression in the ONFH group could be significantly reduced, while ONFH was significantly improved (Fig. <a href="#Fig11" class="usa-link">11</a>C-F). In order to further explore the mechanism, we continued to verify the relationship between GLB1 and hsa-miR-320a. High expression of GLB1 could significantly inhibit the expression of hsa-miR-320a, while treatment with retinoic acid could significantly increase the expression of hsa-miR-320a (Fig. <a href="#Fig11" class="usa-link">11</a>G). Subsequently, we sorted out the genome sequence and found that GLB1 and hsa-miR-320a were perfectly matched at multiple sites (<a href="#MOESM6" class="usa-link">Figure S3A</a>), so we confirmed that retinoic acid could regulate hsa-miR-320a/GLB1 to improve ONFH.</p>
<figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371116_41598_2025_13703_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5328/12371116/21e424b93ca3/41598_2025_13703_Fig11_HTML.jpg" loading="lazy" id="MO11" height="940" width="687" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Retinoic acid improves ONFH by hsa-miR-320a/GLB1. (<strong>A</strong>) qRT-PCR was used to verify the expression of GLB1, CREBBP and PSAP in the healthy femoral head group and the ONFH group. (<strong>B</strong>,<strong>E</strong>) Immunofluorescence confirmed the expression of GLB1 in normal femoral head and ONFH bone marrow mesenchymal stem cells and conducted quantitative analysis. (<strong>C</strong>,<strong>F</strong>) Immunofluorescence method confirmed the expression and quantitative analysis of GLB1 in femoral head tissue; (<strong>D</strong>) HE staining suggested that retinoic acid treatment can improve the pathological progression of ONFH. (<strong>G</strong>) The expression of hsa-miR-320a in the healthy femoral head group, ONFH group and retinoic acid treatment group was verified by qRT-PCR.<sup>ns</sup><em>P</em> &gt; 0.05;*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; *** <em>P</em> &lt; 0.001; **** <em>P</em> &lt; 0.0001.</p></figcaption></figure></section></section><section id="Sec32"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par34">The relationship between ONFH and lipid metabolism disorders has garnered increasing attention. Relevant research has shown a significant correlation between lipid metabolism disorders and NONFH, with abnormal lipid metabolic states being considered one of the risk factors for the progression of ONFH<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. It has also been discovered that certain gene polymorphisms related to lipid metabolism are associated with an increased risk of ONFH<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. These genes may induce or exacerbate ischemic damage to the femoral head by affecting lipid metabolic pathways. These findings underscore the critical importance of identifying lipid metabolism biomarkers in ONFH for enhancing disease management and facilitating early diagnosis. In our study, we employed a comprehensive suite of analytical techniques to achieve this goal. By conducting differential expression analysis, WGCNA, machine learning, and ROC, we successfully identified three lipid metabolism biomarkers associated with ONFH: CREBBP, GLB1, and PSAP. Combining network pharmacology analysis, we have screened out drugs such as estradiol and retinoic acid that regulate the CREBBP, GLB1, and PSAP , this drugs have regulatory effects on genes and on ONFH. Gaining a deeper understanding of their roles in ONFH can shed light on the underlying mechanisms of the disease, potentially leading to more earlier detection strategies.</p>
<p id="Par35">Galactosidase beta 1 (GLB1) encodes a member of the 35-protein family of glycosyl hydrolases. It was shown that excess bioactive sphingolipids are also affected by GLB1 and B4GALT1 proteins, which are up-regulated to affect the balance of osteoblast and osteoclast activity<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. In some cases, CREBBP mutations may lead to abnormal bone metabolism, thereby increasing the risk of ONFH<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.Notably, the higher incidence of ONFH in patients with Rubinstein-Taybi syndrome further emphasizes the importance of CREBBP in the development of ONFH<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. β−1,4-galactosyltransferase 1 (B4GALT1) is an important gene in the human genome, primarily involved in the glycosylation process. B4GALT1 plays a significant role in the glycosylation process, and abnormalities in glycosylation may lead to reduced development and repair capabilities of bone tissue, thereby increasing the risk of ONFH<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Glycosylation modifications can affect intercellular signaling, disrupt normal bone metabolic processes, and cause bone cell dysfunction, which may act as a catalyst in the occurrence of ONFH. These findings suggest that the selected biomarkers may play key contributing roles in the onset and development of ONFH. To gain a deeper understanding of the biological processes or signaling pathways that these biomarkers may influence, we performed GSEA. This analysis aims to reveal how these biomarkers affect the pathological process of ONFH through specific molecular pathways.</p>
<p id="Par36">GSEA is capable of identifying the most significant pathways or functions under specific biological conditions, thereby revealing the overall patterns of biological processes, not just local changes. Through GSEA, we had uncovered three biomarkers were all involved in the activation process of IFN-γ response. IFN-γ plays a central role in immune regulation: it not only enhances the bactericidal capacity of macrophages and promotes the activation of T cells but also regulates antibody production, these functions are crucial for maintaining immune balance<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Furthermore, IFN-γ has a direct impact on bone metabolism by inhibiting the function of osteoblasts and reducing bone formation, while simultaneously promoting the activity of osteoclasts and increasing bone resorption, thus playing a role in the remodeling process of bone tissue<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. The role of IFN-γ in inflammatory responses is also not to be overlooked: it can both promote the production of inflammatory factors and suppress inflammation by regulating the expression of other cytokines, a bidirectional regulatory effect that is vital for controlling inflammatory responses and maintaining tissue homeostasis<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Additionally, we have found that GLB1 and PSAP are activated in lysosomes, which is closely related to the process of autophagy. Autophagy is an essential mechanism for cells to degrade dysfunctional cellular components through lysosomes, maintaining cellular homeostasis and ensuring cell survival under stress conditions. Dysfunction of autophagy plays a pathogenic role in a variety of human diseases, including steroid-induced osteonecrosis of the femoral head<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Moreover, autophagy also affects the interaction between osteoblasts and osteoclasts in ONFH, which has profound implications for the balance of bone tissue and the progression of the disease. These findings provide new ideas for understanding the pathways involved in biomarkers.</p>
<p id="Par37">The combination of bone biology and immunology has recently become an important focus of ONFH research in the field of bone immunology<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. The proper functioning of the immune system is crucial for maintaining the health of bone tissue. We found that activated dendritic cells and memory B cells infiltrated less in ONFH by immune infiltration analysis. In a healthy state, there is a balance between osteoblasts and osteoclasts, with immune cells playing a regulatory role in this process. However, when the immune system is imbalanced, this equilibrium is disrupted, leading to the death of bone cells and the degeneration of bone tissue. Research has found that specific types of immune cells exhibit abnormal expression patterns in ONFH. For example, memory B cells, activated dendritic cells, M1 macrophages, monocytes, and neutrophils are all considered to play significant roles in ONFH. These cells interact by secreting various cytokines and regulating immune responses, a process that plays a key role in the pathophysiology of ONFH. Cytokines secreted by immune cells have a significant impact on the health of bone tissue<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. For instance, certain cytokines may promote bone resorption, inhibit bone formation, leading to the destruction and necrosis of bone tissue. The dysregulation of these cytokines is a key factor in the development of ONFH. In our study, we observed significant differences in neutrophil infiltration between the ONFH and control groups. ScRNA-seq analysis revealed that these three biomarkers were highly infiltrated in ONFH, suggesting the crucial role of neutrophils in the pathogenesis of ONFH. Neutrophils, which are produced in the bone marrow, play a significant role in immune functions within the systemic circulation<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Recent studies have indicated that the functions of neutrophils are heterogeneous and can be influenced by lipid metabolism<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Following early infiltration in ONFH, neutrophils participate in bone remodeling, and the rate of necrosis may be associated with the active immune defense and necrotic tissue clearance by neutrophils in the early stages of bone necrosis<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Furthermore, the percentage of neutrophils is significantly correlated with the incidence of ONFH<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, indicating that neutrophil levels may play an important role in the pathogenesis of ONFH.</p>
<p id="Par38">Deciphering the potential relationships between biomarkers and other miRNAs, etc. may provide new insights into biological functions as well as the pathogenesis of complex diseases. Hsa-miR-320a is a miRNA that plays a regulatory role in various physiological and pathological processes. It binds to the mRNA of specific target genes, inhibiting their expression and thereby affecting multiple cellular functions. In a study on ONFH, hsa-miR-320a levels were elevated in blood samples, and it was discovered that it may block osteoblast differentiation by targeting RUNX2 while promoting adipocyte differentiation of BMSCs<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Our research has found that hsa-miR-320a might inhibit glucocorticoid-induced ONFH by targeting GLB1. GLB1 is an important bioenzyme that may be associated with the metabolism and repair processes of bone tissue. With the over-expression of GLB1, the level of hsa-miR-320a also increased significantly, which may have an effect on bone necrosis-related pathways or mechanisms as previously discovered. Following retinoic acid treatment, the level of hsa-miR-320a decreased significantly, suggesting that hsa-miR-320a may serve as a potential therapeutic target, influencing the progression of the disease by modulating the expression of GLB1. On the other hand, our study found that the key gene GLB1 is enriched in biological processes such as “lysosome”, “proteasome”, and "IFN-γ response", which are closely related to inflammation and immune responses-key factors in the pathogenesis of ONFH<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Existing evidence also shows that hsa-miR-320a is involved in these processes, such as by regulating Arf1 and PBX3 to influence inflammation and cartilage degradation<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Based on this, we propose that hsa-miR-320a may regulate GLB1 expression and participate in the inflammatory or immune pathways in ONFH. Meantime, gain/loss-of-function experiments, ChIP-qPCR, and dual-luciferase reporter gene assays, might be utilized to further clarify the synergistic or temporal relationship between the transcriptional (TFs) and post-transcriptional (hsa-miR-320a) levels in GLB1 regulation so as to define their biological significance in ONFH.</p></section><section id="Sec33"><h2 class="pmc_sec_title">Conclusion and limitation</h2>
<p id="Par39">Summarizing our current findings, CREBBP, GLB1 and PSAP play key roles in regulating the development of ONFH and can be used as current biomarkers for the diagnosis of ONFH. These findings not only provided us with an in-depth understanding of the disease mechanism, but also enabled the successful construction of an efficient diagnostic model based on these biomarkers. In addition, we further analyzed in depth the influence pathways of these biomarkers in the development of ONFH, which helped us to grasp the developmental dynamics of the disease more comprehensively. We also explored the specific features and roles of the biomarkers at the cellular level.</p>
<p id="Par40">However, there are limitations to our study, The datasets used in this study were derived from a single source, which may not fully reflect the model performance in real-world clinical settings. In the future, we plan to collect more clinical samples, expand the study cohort, and validate the model with multi-center data to further improve its clinical applicability and reliability. For the follow-up, we will also detect the protein levels of the three biomarkers in serum by ELISA to explore the feasibility of non-invasive detection. Besides, in the KEGG analysis, uncorrected <em>P</em>-values (<em>P</em> &lt; 0.05) were used as the significance criterion, which may increase the risk of false-positive results. However, given that we focused on pathways highly relevant to the research focus and these pathways also showed significant enrichment in the GO analysis, these KEGG pathways can be considered to have high biological significance. If more pathway data are mined in the future, we will supplement additional experiments (such as qPCR and WB) for further independent verification. These studies have enhanced our understanding of the pathological process of ONFH, and they provide potential targets and biomarkers for future therapeutic strategies.</p></section><section id="Sec34"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (36.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (13.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM3_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (261.5KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM4_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 4.</a><sup> (851B, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 5.</a><sup> (10.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 6.</a><sup> (453KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 7.</a><sup> (16.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 8.</a><sup> (10.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 9.</a><sup> (12.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 10.</a><sup> (10.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 11.</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM12_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 12.</a><sup> (2.1MB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM13_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 13.</a><sup> (2.4MB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 14.</a><sup> (97.9KB, xlsx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>All authors contributed to the study conception and design. Research design: Fei Zhang and Wuxun Peng. Data analysis: Chuan Wang, Huachuan Su and ChaoDe Cen. Resources: Yuhui Peng, Tao Wang, Jian Li. Funding acquisition and supervision: Chuan Wang, Fei Zhang. All authors commented on previous versions of the manuscript. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study were funded by the Science foundation of Guizhou Provincial Health Commission(gzwkj2023–177) and the Cultivation Project of National Natural Science Foundation of Guizhou Medical University (Grant No. gyfynsfc[2023]-02).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>This study utilized several datasets, including mRNA transcriptome datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a>), and the microRNA (miRNA) transcriptome dataset <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89587" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE89587</a>, all obtained from the Gene Expression Omnibus (GEO) database (<a href="https://www.ncbi.nlm.nih.gov/geo)." class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo).</a>  The lipid metabolism-related genes (LMRGs) from the Reactome database (<a href="https://reactome.org/)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://reactome.org/).</a>  The scRNA-seq dataset SRP361778, sourced from the SRA database (<a href="https://www.ncbi.nlm.nih.gov/sra" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra</a>).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar8"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par41">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Chuan Wang, Chaode Cen, and Huachuan Su authors contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Fei Zhang, Email: 1426287482@qq.com.</p>
<p>Wuxun Peng, Email: GZMU2022@126.com.</p></section><section id="sec35"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>The online version contains supplementary material available at 10.1038/s41598-025-13703-y.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Liu, X. et al. Research progress on exosomes in osteonecrosis of the femoral head. <em>Orthop. Surg.</em><strong>14</strong>(9), 1951–1957 (2022).
</cite> [<a href="https://doi.org/10.1111/os.13393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9483046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35924692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20X.%20et%20al.%20Research%20progress%20on%20exosomes%20in%20osteonecrosis%20of%20the%20femoral%20head.%20Orthop.%20Surg.14(9),%201951%E2%80%931957%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Seamon, J., Keller, T., Saleh, J. &amp; Cui, Q. The Pathogenesis of Nontraumatic Osteonecrosis. <em>Arthritis</em><strong>2012</strong>(1), 601763 (2012).
</cite> [<a href="https://doi.org/10.1155/2012/601763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3518945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23243507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seamon,%20J.,%20Keller,%20T.,%20Saleh,%20J.%20&amp;%20Cui,%20Q.%20The%20Pathogenesis%20of%20Nontraumatic%20Osteonecrosis.%20Arthritis2012(1),%20601763%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Zhao, D. W. et al. Prevalence of Nontraumatic Osteonecrosis of the Femoral Head and Its Associated Risk Factors in the Chinese Population: Results from a Nationally Representative Survey. <em>Chin. Med. J. (Engl.)</em><strong>128</strong>(21), 2843–2850 (2015).
</cite> [<a href="https://doi.org/10.4103/0366-6999.168017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4756878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26521779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20D.%20W.%20et%20al.%20Prevalence%20of%20Nontraumatic%20Osteonecrosis%20of%20the%20Femoral%20Head%20and%20Its%20Associated%20Risk%20Factors%20in%20the%20Chinese%20Population:%20Results%20from%20a%20Nationally%20Representative%20Survey.%20Chin.%20Med.%20J.%20(Engl.)128(21),%202843%E2%80%932850%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Ma, M. et al. Osteoimmunology and Osteonecrosis of the Femoral Head. <em>Bone Joint Res.</em><strong>11</strong>(1), 26–28 (2022).
</cite> [<a href="https://doi.org/10.1302/2046-3758.111.BJR-2021-0467.R1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8801166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35045723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20M.%20et%20al.%20Osteoimmunology%20and%20Osteonecrosis%20of%20the%20Femoral%20Head.%20Bone%20Joint%20Res.11(1),%2026%E2%80%9328%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Nan, K. et al. Exosomes from Mirna-378-Modified Adipose-Derived Stem Cells Prevent Glucocorticoid-Induced Osteonecrosis of the Femoral Head by Enhancing Angiogenesis and Osteogenesis Via Targeting Mir-378 Negatively Regulated Suppressor of Fused (Sufu). <em>Stem Cell Res. Ther.</em><strong>12</strong>(1), 331 (2021).
</cite> [<a href="https://doi.org/10.1186/s13287-021-02390-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8186190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34099038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nan,%20K.%20et%20al.%20Exosomes%20from%20Mirna-378-Modified%20Adipose-Derived%20Stem%20Cells%20Prevent%20Glucocorticoid-Induced%20Osteonecrosis%20of%20the%20Femoral%20Head%20by%20Enhancing%20Angiogenesis%20and%20Osteogenesis%20Via%20Targeting%20Mir-378%20Negatively%20Regulated%20Suppressor%20of%20Fused%20(Sufu).%20Stem%20Cell%20Res.%20Ther.12(1),%20331%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Hao, Y. et al. The Relationship between Apolipoprotein Genes Polymorphisms and Susceptibility to Osteonecrosis of the Femoral Head: A Meta-Analysis. <em>Lipid. Health Dis.</em><strong>17</strong>(1), 192 (2018).</cite> [<a href="https://doi.org/10.1186/s12944-018-0827-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6098662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30119683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20Y.%20et%20al.%20The%20Relationship%20between%20Apolipoprotein%20Genes%20Polymorphisms%20and%20Susceptibility%20to%20Osteonecrosis%20of%20the%20Femoral%20Head:%20A%20Meta-Analysis.%20Lipid.%20Health%20Dis.17(1),%20192%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Yan, Y. et al. Plasma Lipidomics Analysis Reveals Altered Lipids Signature in Patients with Osteonecrosis of the Femoral Head. <em>Metabolomic</em><strong>18</strong>(2), 14 (2022).</cite> [<a href="https://doi.org/10.1007/s11306-022-01872-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35147763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20Y.%20et%20al.%20Plasma%20Lipidomics%20Analysis%20Reveals%20Altered%20Lipids%20Signature%20in%20Patients%20with%20Osteonecrosis%20of%20the%20Femoral%20Head.%20Metabolomic18(2),%2014%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Yue, Y. B., Liu, X. &amp; Wei, B. F. Attenuated Serum Adiponectin Levels Are Associated with Disease Severity in Patients with Non-Traumatic Osteonecrosis of the Femoral Head. <em>J. Pain Res.</em><strong>10</strong>, 2387–2393 (2017).
</cite> [<a href="https://doi.org/10.2147/JPR.S143036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5633271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29042814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yue,%20Y.%20B.,%20Liu,%20X.%20&amp;%20Wei,%20B.%20F.%20Attenuated%20Serum%20Adiponectin%20Levels%20Are%20Associated%20with%20Disease%20Severity%20in%20Patients%20with%20Non-Traumatic%20Osteonecrosis%20of%20the%20Femoral%20Head.%20J.%20Pain%20Res.10,%202387%E2%80%932393%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Nishida, K., Yamamoto, T., Motomura, G., Jingushi, S. &amp; Iwamoto, Y. Pitavastatin May Reduce Risk of Steroid-Induced Osteonecrosis in Rabbits: A Preliminary Histological Study. <em>Clin. Orthop. Relat. Res.</em><strong>466</strong>(5), 1054–1058 (2008).
</cite> [<a href="https://doi.org/10.1007/s11999-008-0189-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2311459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18350345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nishida,%20K.,%20Yamamoto,%20T.,%20Motomura,%20G.,%20Jingushi,%20S.%20&amp;%20Iwamoto,%20Y.%20Pitavastatin%20May%20Reduce%20Risk%20of%20Steroid-Induced%20Osteonecrosis%20in%20Rabbits:%20A%20Preliminary%20Histological%20Study.%20Clin.%20Orthop.%20Relat.%20Res.466(5),%201054%E2%80%931058%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Li, Y., Yu, J., Li, R., Zhou, H. &amp; Chang, X. New Insights into the Role of Mitochondrial Metabolic Dysregulation and Immune Infiltration in Septic Cardiomyopathy by Integrated Bioinformatics Analysis and Experimental Validation. <em>Cell Mol. Biol. Lett.</em><strong>29</strong>(1), 21 (2024).
</cite> [<a href="https://doi.org/10.1186/s11658-024-00536-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10826082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38291374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.,%20Yu,%20J.,%20Li,%20R.,%20Zhou,%20H.%20&amp;%20Chang,%20X.%20New%20Insights%20into%20the%20Role%20of%20Mitochondrial%20Metabolic%20Dysregulation%20and%20Immune%20Infiltration%20in%20Septic%20Cardiomyopathy%20by%20Integrated%20Bioinformatics%20Analysis%20and%20Experimental%20Validation.%20Cell%20Mol.%20Biol.%20Lett.29(1),%2021%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Ma, H., Zhang, W. &amp; Shi, J. Differentially Expressed Genes Reveal the Biomarkers and Molecular Mechanism of Osteonecrosis. <em>J. Healthc. Eng.</em><strong>2022</strong>, 8684137 (2022).
</cite> [<a href="https://doi.org/10.1155/2022/8684137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8759865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35035862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20H.,%20Zhang,%20W.%20&amp;%20Shi,%20J.%20Differentially%20Expressed%20Genes%20Reveal%20the%20Biomarkers%20and%20Molecular%20Mechanism%20of%20Osteonecrosis.%20J.%20Healthc.%20Eng.2022,%208684137%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Zhuang, X. Y. et al. Comprehensive Analysis of Pain Genes in Prognosis of Kidney Renal Clear Cell Carcinoma and Tumor Immunotherapy: A Comprehensive Bioinformatic Study. <em>Health Sci. Rep.</em><strong>7</strong>(2), e1884 (2024).
</cite> [<a href="https://doi.org/10.1002/hsr2.1884" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10862147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38352696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhuang,%20X.%20Y.%20et%20al.%20Comprehensive%20Analysis%20of%20Pain%20Genes%20in%20Prognosis%20of%20Kidney%20Renal%20Clear%20Cell%20Carcinoma%20and%20Tumor%20Immunotherapy:%20A%20Comprehensive%20Bioinformatic%20Study.%20Health%20Sci.%20Rep.7(2),%20e1884%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Langfelder, P. &amp; Horvath, S. Wgcna: An R Package for Weighted Correlation Network Analysis. <em>BMC Bioinform.</em><strong>9</strong>, 559 (2008).</cite> [<a href="https://doi.org/10.1186/1471-2105-9-559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2631488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19114008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langfelder,%20P.%20&amp;%20Horvath,%20S.%20Wgcna:%20An%20R%20Package%20for%20Weighted%20Correlation%20Network%20Analysis.%20BMC%20Bioinform.9,%20559%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Mao, W., Ding, J., Li, Y., Huang, R. &amp; Wang, B. Inhibition of Cell Survival and Invasion by Tanshinone Iia Via Fth1: A Key Therapeutic Target and Biomarker in Head and Neck Squamous Cell Carcinoma. <em>Exp. Ther. Med.</em><strong>24</strong>(2), 521 (2022).
</cite> [<a href="https://doi.org/10.3892/etm.2022.11449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9257971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35837069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao,%20W.,%20Ding,%20J.,%20Li,%20Y.,%20Huang,%20R.%20&amp;%20Wang,%20B.%20Inhibition%20of%20Cell%20Survival%20and%20Invasion%20by%20Tanshinone%20Iia%20Via%20Fth1:%20A%20Key%20Therapeutic%20Target%20and%20Biomarker%20in%20Head%20and%20Neck%20Squamous%20Cell%20Carcinoma.%20Exp.%20Ther.%20Med.24(2),%20521%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kanehisa, M., Furumichi, M., Sato, Y., Matsuura, Y. &amp; Ishiguro-Watanabe, M. Kegg: Biological Systems Database as a Model of the Real World. <em>Nucl. Acid. Res.</em><strong>53</strong>(D1), D672–D677 (2024).</cite> [<a href="https://doi.org/10.1093/nar/gkae909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39417505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.,%20Furumichi,%20M.,%20Sato,%20Y.,%20Matsuura,%20Y.%20&amp;%20Ishiguro-Watanabe,%20M.%20Kegg:%20Biological%20Systems%20Database%20as%20a%20Model%20of%20the%20Real%20World.%20Nucl.%20Acid.%20Res.53(D1),%20D672%E2%80%93D677%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kanehisa, M. &amp; Goto, S. Kegg: Kyoto Encyclopedia of Genes and Genomes. <em>Nucl. Acid. Res.</em><strong>28</strong>(1), 27–30 (2000).</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20&amp;%20Goto,%20S.%20Kegg:%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes.%20Nucl.%20Acid.%20Res.28(1),%2027%E2%80%9330%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kanehisa, M. Toward Understanding the Origin and Evolution of Cellular Organisms. <em>Protein Sci.</em><strong>28</strong>(11), 1947–1951 (2019).
</cite> [<a href="https://doi.org/10.1002/pro.3715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6798127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31441146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20Toward%20Understanding%20the%20Origin%20and%20Evolution%20of%20Cellular%20Organisms.%20Protein%20Sci.28(11),%201947%E2%80%931951%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Guo, S. et al. Identification and Analysis of Key Genes Associated with Acute Myocardial Infarction by Integrated Bioinformatics Methods. <em>Medicine (Baltimore)</em><strong>100</strong>(15), e25553 (2021).
</cite> [<a href="https://doi.org/10.1097/MD.0000000000025553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8052032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33847684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20S.%20et%20al.%20Identification%20and%20Analysis%20of%20Key%20Genes%20Associated%20with%20Acute%20Myocardial%20Infarction%20by%20Integrated%20Bioinformatics%20Methods.%20Medicine%20(Baltimore)100(15),%20e25553%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Raoufi, S., Jafarinejad-Farsangi, S., Dehesh, T. &amp; Hadizadeh, M. Investigating Unique Genes of Five Molecular Subtypes of Breast Cancer Using Penalized Logistic Regression. <em>J. Cancer Res. Ther.</em><strong>19</strong>(Supplement), S126–S137 (2023).
</cite> [<a href="https://doi.org/10.4103/jcrt.jcrt_811_21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37147992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raoufi,%20S.,%20Jafarinejad-Farsangi,%20S.,%20Dehesh,%20T.%20&amp;%20Hadizadeh,%20M.%20Investigating%20Unique%20Genes%20of%20Five%20Molecular%20Subtypes%20of%20Breast%20Cancer%20Using%20Penalized%20Logistic%20Regression.%20J.%20Cancer%20Res.%20Ther.19(Supplement),%20S126%E2%80%93S137%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Aldoss I. et al. Outcomes of Allogeneic Hematopoietic Cell Transplantation in Adults with Fusions Associated with Ph-Like All. <em>Blood Adv.</em><strong>6</strong>(17), 4936–4948 (2022).</cite> [<a href="https://doi.org/10.1182/bloodadvances.2022007597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9631622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35816633/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Yamamoto, Y. et al. Automated Acquisition of Explainable Knowledge from Unannotated Histopathology Images. <em>Nat. Commun.</em><strong>10</strong>(1), 5642 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-13647-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6920352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31852890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamamoto,%20Y.%20et%20al.%20Automated%20Acquisition%20of%20Explainable%20Knowledge%20from%20Unannotated%20Histopathology%20Images.%20Nat.%20Commun.10(1),%205642%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Adzibolosu, N. et al. Immunological Modifications Following Chemotherapy Are Associated with Delayed Recurrence of Ovarian Cancer. <em>Front. Immunol.</em><strong>14</strong>, 1204148 (2023).
</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1204148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10331425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37435088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adzibolosu,%20N.%20et%20al.%20Immunological%20Modifications%20Following%20Chemotherapy%20Are%20Associated%20with%20Delayed%20Recurrence%20of%20Ovarian%20Cancer.%20Front.%20Immunol.14,%201204148%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Liang, L. et al. Foxp3 Contributes to Tmz Resistance, Prognosis, and Immune Infiltration in Gbm from a Novel Pyroptosis-Associated Risk Signature. <em>Dis. Mark.</em><strong>2022</strong>, 4534080 (2022).</cite> [<a href="https://doi.org/10.1155/2022/4534080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8993549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35401877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang,%20L.%20et%20al.%20Foxp3%20Contributes%20to%20Tmz%20Resistance,%20Prognosis,%20and%20Immune%20Infiltration%20in%20Gbm%20from%20a%20Novel%20Pyroptosis-Associated%20Risk%20Signature.%20Dis.%20Mark.2022,%204534080%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Liang, X. Z. et al. Irf8 and Its Related Molecules as Potential Diagnostic Biomarkers or Therapeutic Candidates and Immune Cell Infiltration Characteristics in Steroid-Induced Osteonecrosis of the Femoral Head. <em>J. Orthop. Surg. Res.</em><strong>18</strong>(1), 27 (2023).
</cite> [<a href="https://doi.org/10.1186/s13018-022-03381-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9832881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36627660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang,%20X.%20Z.%20et%20al.%20Irf8%20and%20Its%20Related%20Molecules%20as%20Potential%20Diagnostic%20Biomarkers%20or%20Therapeutic%20Candidates%20and%20Immune%20Cell%20Infiltration%20Characteristics%20in%20Steroid-Induced%20Osteonecrosis%20of%20the%20Femoral%20Head.%20J.%20Orthop.%20Surg.%20Res.18(1),%2027%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Feng, S. S. et al. Clinical Characteristics and Disease-Specific Prognostic Nomogram for Primary Gliosarcoma: A Seer Population-Based Analysis. <em>Sci. Rep.</em><strong>9</strong>(1), 10744 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-47211-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6656887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng,%20S.%20S.%20et%20al.%20Clinical%20Characteristics%20and%20Disease-Specific%20Prognostic%20Nomogram%20for%20Primary%20Gliosarcoma:%20A%20Seer%20Population-Based%20Analysis.%20Sci.%20Rep.9(1),%2010744%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>He, W. et al. Radiomics Based on Hrct Can Predict Rp-Ild and Mortality in Anti-Mda5 + Dermatomyositis Patients: A Multi-Center Retrospective Study. <em>Respir. Res.</em><strong>25</strong>(1), 252 (2024).
</cite> [<a href="https://doi.org/10.1186/s12931-024-02843-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11191144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38902680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20W.%20et%20al.%20Radiomics%20Based%20on%20Hrct%20Can%20Predict%20Rp-Ild%20and%20Mortality%20in%20Anti-Mda5%20+%20Dermatomyositis%20Patients:%20A%20Multi-Center%20Retrospective%20Study.%20Respir.%20Res.25(1),%20252%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Chen, B., Khodadoust, M. S., Liu, C. L., Newman, A. M. &amp; Alizadeh, A. A. Profiling Tumor Infiltrating Immune Cells with Cibersort. <em>Method. Mol. Biol.</em><strong>1711</strong>, 243–259 (2018).</cite> [<a href="https://doi.org/10.1007/978-1-4939-7493-1_12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5895181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29344893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20B.,%20Khodadoust,%20M.%20S.,%20Liu,%20C.%20L.,%20Newman,%20A.%20M.%20&amp;%20Alizadeh,%20A.%20A.%20Profiling%20Tumor%20Infiltrating%20Immune%20Cells%20with%20Cibersort.%20Method.%20Mol.%20Biol.1711,%20243%E2%80%93259%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Newman, A. M. et al. Robust Enumeration of Cell Subsets from Tissue Expression Profiles. <em>Nat. Method.</em><strong>12</strong>(5), 453–457 (2015).</cite> [<a href="https://doi.org/10.1038/nmeth.3337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4739640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25822800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Newman,%20A.%20M.%20et%20al.%20Robust%20Enumeration%20of%20Cell%20Subsets%20from%20Tissue%20Expression%20Profiles.%20Nat.%20Method.12(5),%20453%E2%80%93457%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Orifjon, S. et al. Translation and Adaptation of the Adult Developmental Coordination Disorder/Dyspraxia Checklist (Adc) into Asian Uzbekistan. <em>Sports (Basel)</em><strong>11</strong>, 7 (2023).</cite> [<a href="https://doi.org/10.3390/sports11070135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10383954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37505622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orifjon,%20S.%20et%20al.%20Translation%20and%20Adaptation%20of%20the%20Adult%20Developmental%20Coordination%20Disorder/Dyspraxia%20Checklist%20(Adc)%20into%20Asian%20Uzbekistan.%20Sports%20(Basel)11,%207%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Zhong, M. E. et al. Lncrna H19 Regulates Pi3k-Akt Signal Pathway by Functioning as a Cerna and Predicts Poor Prognosis in Colorectal Cancer: Integrative Analysis of Dysregulated Ncrna-Associated Cerna Network. <em>Cancer Cell. Int.</em><strong>19</strong>, 148 (2019).
</cite> [<a href="https://doi.org/10.1186/s12935-019-0866-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6543669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31164794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong,%20M.%20E.%20et%20al.%20Lncrna%20H19%20Regulates%20Pi3k-Akt%20Signal%20Pathway%20by%20Functioning%20as%20a%20Cerna%20and%20Predicts%20Poor%20Prognosis%20in%20Colorectal%20Cancer:%20Integrative%20Analysis%20of%20Dysregulated%20Ncrna-Associated%20Cerna%20Network.%20Cancer%20Cell.%20Int.19,%20148%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Rafi, M. O. et al. Design of a Multi-Epitope Vaccine against Sars-Cov-2: Immunoinformatic and Computational Methods. <em>RSC Adv.</em><strong>12</strong>(7), 4288–4310 (2022).
</cite> [<a href="https://doi.org/10.1039/d1ra06532g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8981096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35425433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rafi,%20M.%20O.%20et%20al.%20Design%20of%20a%20Multi-Epitope%20Vaccine%20against%20Sars-Cov-2:%20Immunoinformatic%20and%20Computational%20Methods.%20RSC%20Adv.12(7),%204288%E2%80%934310%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Saikia, S. &amp; Bordoloi, M. Molecular Docking: Challenges, Advances and Its Use in Drug Discovery Perspective. <em>Curr. Drug Target.</em><strong>20</strong>(5), 501–521 (2019).</cite> [<a href="https://doi.org/10.2174/1389450119666181022153016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30360733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saikia,%20S.%20&amp;%20Bordoloi,%20M.%20Molecular%20Docking:%20Challenges,%20Advances%20and%20Its%20Use%20in%20Drug%20Discovery%20Perspective.%20Curr.%20Drug%20Target.20(5),%20501%E2%80%93521%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hänzelmann, S., Castelo, R. &amp; Guinney, J. Gsva: Gene Set Variation Analysis for Microarray and Rna-Seq Data. <em>BMC Bioinform.</em><strong>14</strong>, 7 (2013).</cite> [<a href="https://doi.org/10.1186/1471-2105-14-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3618321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23323831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%A4nzelmann,%20S.,%20Castelo,%20R.%20&amp;%20Guinney,%20J.%20Gsva:%20Gene%20Set%20Variation%20Analysis%20for%20Microarray%20and%20Rna-Seq%20Data.%20BMC%20Bioinform.14,%207%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Luo, C. et al. Increased Cyr61 Expression Activates Ccnd1/C-Myc Pathway to Promote Nasal Epithelial Cells Proliferation in Chronic Rhinosinusitis with Nasal Polyps. <em>Clin. Immunol.</em><strong>247</strong>, 109235 (2023).
</cite> [<a href="https://doi.org/10.1016/j.clim.2023.109235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36681101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo,%20C.%20et%20al.%20Increased%20Cyr61%20Expression%20Activates%20Ccnd1/C-Myc%20Pathway%20to%20Promote%20Nasal%20Epithelial%20Cells%20Proliferation%20in%20Chronic%20Rhinosinusitis%20with%20Nasal%20Polyps.%20Clin.%20Immunol.247,%20109235%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Xia, P. et al. Macrophage-Related Testicular Inflammation in Individuals with Idiopathic Non-Obstructive Azoospermia: A Single-Cell Analysis. <em>Int. J. Mol. Sci.</em><strong>24</strong>(10), 8819 (2023).
</cite> [<a href="https://doi.org/10.3390/ijms24108819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10218437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37240164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20P.%20et%20al.%20Macrophage-Related%20Testicular%20Inflammation%20in%20Individuals%20with%20Idiopathic%20Non-Obstructive%20Azoospermia:%20A%20Single-Cell%20Analysis.%20Int.%20J.%20Mol.%20Sci.24(10),%208819%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Zhang, F. et al. Far591 Promotes the Pathogenesis and Progression of Sonfh by Regulating Fos Expression to Mediate the Apoptosis of Bone Microvascular Endothelial Cells. <em>Bone Res.</em><strong>11</strong>(1), 27 (2023).
</cite> [<a href="https://doi.org/10.1038/s41413-023-00259-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10203311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37217464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20F.%20et%20al.%20Far591%20Promotes%20the%20Pathogenesis%20and%20Progression%20of%20Sonfh%20by%20Regulating%20Fos%20Expression%20to%20Mediate%20the%20Apoptosis%20of%20Bone%20Microvascular%20Endothelial%20Cells.%20Bone%20Res.11(1),%2027%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Yan, Y., Yu, Y., Liu, J., Zhao, H. &amp; Wang, J. Lipid Metabolism Analysis for Peripheral Blood in Patients with Alcohol-Induced and Steroid-Induced Osteonecrosis of the Femoral Head. <em>Zhong Nan Da Xue Xue Bao Yi Xue Ban</em><strong>47</strong>(7), 872–880 (2022).
</cite> [<a href="https://doi.org/10.11817/j.issn.1672-7347.2022.210567" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10930281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36039583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20Y.,%20Yu,%20Y.,%20Liu,%20J.,%20Zhao,%20H.%20&amp;%20Wang,%20J.%20Lipid%20Metabolism%20Analysis%20for%20Peripheral%20Blood%20in%20Patients%20with%20Alcohol-Induced%20and%20Steroid-Induced%20Osteonecrosis%20of%20the%20Femoral%20Head.%20Zhong%20Nan%20Da%20Xue%20Xue%20Bao%20Yi%20Xue%20Ban47(7),%20872%E2%80%93880%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Baek, S. H. et al. Abnormal Lipid Profiles in Nontraumatic Osteonecrosis of the Femoral Head: A Comparison with Osteoarthritis Using Propensity Score Matching. <em>J. Bone Joint Surg. Am.</em><strong>104</strong>(Suppl 2), 19–24 (2022).
</cite> [<a href="https://doi.org/10.2106/JBJS.20.00520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35389903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baek,%20S.%20H.%20et%20al.%20Abnormal%20Lipid%20Profiles%20in%20Nontraumatic%20Osteonecrosis%20of%20the%20Femoral%20Head:%20A%20Comparison%20with%20Osteoarthritis%20Using%20Propensity%20Score%20Matching.%20J.%20Bone%20Joint%20Surg.%20Am.104(Suppl%202),%2019%E2%80%9324%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Song, Y. et al. Association of Srebp2 Gene Polymorphisms with the Risk of Osteonecrosis of the Femoral Head Relates to Gene Expression and Lipid Metabolism Disorders. <em>Mol. Med. Rep.</em><strong>16</strong>(5), 7145–7153 (2017).
</cite> [<a href="https://doi.org/10.3892/mmr.2017.7473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28901487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20Y.%20et%20al.%20Association%20of%20Srebp2%20Gene%20Polymorphisms%20with%20the%20Risk%20of%20Osteonecrosis%20of%20the%20Femoral%20Head%20Relates%20to%20Gene%20Expression%20and%20Lipid%20Metabolism%20Disorders.%20Mol.%20Med.%20Rep.16(5),%207145%E2%80%937153%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Zhong, Y. et al. Dissecting the Effect of Sphingolipid Metabolism Gene in Progression and Microenvironment of Osteosarcoma to Develop a Prognostic Signature. <em>Front. Endocrinol. (Lausanne)</em><strong>13</strong>, 1030655 (2022).
</cite> [<a href="https://doi.org/10.3389/fendo.2022.1030655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9613955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36313783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong,%20Y.%20et%20al.%20Dissecting%20the%20Effect%20of%20Sphingolipid%20Metabolism%20Gene%20in%20Progression%20and%20Microenvironment%20of%20Osteosarcoma%20to%20Develop%20a%20Prognostic%20Signature.%20Front.%20Endocrinol.%20(Lausanne)13,%201030655%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Hughes, D. et al. Gaucher Disease in Bone: From Pathophysiology to Practice. <em>J. Bone Miner Res.</em><strong>34</strong>(6), 996–1013 (2019).
</cite> [<a href="https://doi.org/10.1002/jbmr.3734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6852006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31233632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hughes,%20D.%20et%20al.%20Gaucher%20Disease%20in%20Bone:%20From%20Pathophysiology%20to%20Practice.%20J.%20Bone%20Miner%20Res.34(6),%20996%E2%80%931013%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Xu, Y. et al. Circular Rna Crebbp Modulates Cartilage Degradation by Activating the Smad1/5 Pathway through the Tgfβ2/Alk1 Axis. <em>Exp. Mol. Med.</em><strong>54</strong>(10), 1727–1740 (2022).
</cite> [<a href="https://doi.org/10.1038/s12276-022-00865-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9636424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36224344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20Y.%20et%20al.%20Circular%20Rna%20Crebbp%20Modulates%20Cartilage%20Degradation%20by%20Activating%20the%20Smad1/5%20Pathway%20through%20the%20Tgf%CE%B22/Alk1%20Axis.%20Exp.%20Mol.%20Med.54(10),%201727%E2%80%931740%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Gittens, O. et al. Ep131: High Prevalence of Bony Abnormalities in Patients with Rubinstein-Taybi Syndrome: A Case Series. <em>Genet. Med.</em><strong>24</strong>(3), S82 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gittens,%20O.%20et%20al.%20Ep131:%20High%20Prevalence%20of%20Bony%20Abnormalities%20in%20Patients%20with%20Rubinstein-Taybi%20Syndrome:%20A%20Case%20Series.%20Genet.%20Med.24(3),%20S82%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Weich, N., Rivadeneyra, L., Glabere, S., Weiler, H. &amp; Hoffmeister, K. M. B4galt1-Dependent Glycosylation Is Essential to Bone Marrow Reconstitution Following Chemotherapeutic Injury. <em>Blood</em><strong>142</strong>(Supplement 1), 3933–4033 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Weich,%20N.,%20Rivadeneyra,%20L.,%20Glabere,%20S.,%20Weiler,%20H.%20&amp;%20Hoffmeister,%20K.%20M.%20B4galt1-Dependent%20Glycosylation%20Is%20Essential%20to%20Bone%20Marrow%20Reconstitution%20Following%20Chemotherapeutic%20Injury.%20Blood142(Supplement%201),%203933%E2%80%934033%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Taylor, G. A., Feng, C. G. &amp; Sher, A. P47 Gtpases: Regulators of Immunity to Intracellular Pathogens. <em>Nat. Rev. Immunol.</em><strong>4</strong>(2), 100–109 (2004).
</cite> [<a href="https://doi.org/10.1038/nri1270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15040583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taylor,%20G.%20A.,%20Feng,%20C.%20G.%20&amp;%20Sher,%20A.%20P47%20Gtpases:%20Regulators%20of%20Immunity%20to%20Intracellular%20Pathogens.%20Nat.%20Rev.%20Immunol.4(2),%20100%E2%80%93109%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Alsamraae, M. &amp; Cook, L. M. Emerging Roles for Myeloid Immune Cells in Bone Metastasis. <em>Cancer Metastasis Rev.</em><strong>40</strong>(2), 413–425 (2021).
</cite> [<a href="https://doi.org/10.1007/s10555-021-09965-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33855680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alsamraae,%20M.%20&amp;%20Cook,%20L.%20M.%20Emerging%20Roles%20for%20Myeloid%20Immune%20Cells%20in%20Bone%20Metastasis.%20Cancer%20Metastasis%20Rev.40(2),%20413%E2%80%93425%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Rosain, J. et al. Human Irf1 Governs Macrophagic Ifn-Γ Immunity to Mycobacteria. <em>Cell</em><strong>186</strong>(3), 621 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.12.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9907019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36736301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosain,%20J.%20et%20al.%20Human%20Irf1%20Governs%20Macrophagic%20Ifn-%CE%93%20Immunity%20to%20Mycobacteria.%20Cell186(3),%20621%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Luo, P., Gao, F., Han, J., Sun, W. &amp; Li, Z. The Role of Autophagy in Steroid Necrosis of the Femoral Head: A Comprehensive Research Review. <em>Int. Orthop.</em><strong>42</strong>(7), 1747–1753 (2018).
</cite> [<a href="https://doi.org/10.1007/s00264-018-3994-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29797168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo,%20P.,%20Gao,%20F.,%20Han,%20J.,%20Sun,%20W.%20&amp;%20Li,%20Z.%20The%20Role%20of%20Autophagy%20in%20Steroid%20Necrosis%20of%20the%20Femoral%20Head:%20A%20Comprehensive%20Research%20Review.%20Int.%20Orthop.42(7),%201747%E2%80%931753%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Goodman, S. B. &amp; Maruyama, M. Inflammation, Bone Healing and Osteonecrosis: From Bedside to Bench. <em>J. Inflamm. Res.</em><strong>13</strong>, 913–923 (2020).
</cite> [<a href="https://doi.org/10.2147/JIR.S281941" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7671464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33223846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goodman,%20S.%20B.%20&amp;%20Maruyama,%20M.%20Inflammation,%20Bone%20Healing%20and%20Osteonecrosis:%20From%20Bedside%20to%20Bench.%20J.%20Inflamm.%20Res.13,%20913%E2%80%93923%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Duan, X. et al. Bioinformatic Analysis of Related Immune Cell Infiltration and Key Genes in the Progression of Osteonecrosis of the Femoral Head. <em>Front. Immunol.</em><strong>14</strong>, 1340446 (2023).
</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1340446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10811953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38283345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan,%20X.%20et%20al.%20Bioinformatic%20Analysis%20of%20Related%20Immune%20Cell%20Infiltration%20and%20Key%20Genes%20in%20the%20Progression%20of%20Osteonecrosis%20of%20the%20Femoral%20Head.%20Front.%20Immunol.14,%201340446%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Jablonska, J., Lang, S., Sionov, R. V. &amp; Granot, Z. The Regulation of Pre-Metastatic Niche Formation by Neutrophils. <em>Oncotarget</em><strong>8</strong>(67), 112132–112144 (2017).
</cite> [<a href="https://doi.org/10.18632/oncotarget.22792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5762385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29340117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jablonska,%20J.,%20Lang,%20S.,%20Sionov,%20R.%20V.%20&amp;%20Granot,%20Z.%20The%20Regulation%20of%20Pre-Metastatic%20Niche%20Formation%20by%20Neutrophils.%20Oncotarget8(67),%20112132%E2%80%93112144%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Jiang, J., Tu, H. &amp; Li, P. Lipid Metabolism and Neutrophil Function. <em>Cell Immunol.</em><strong>377</strong>, 104546 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cellimm.2022.104546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35688009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20J.,%20Tu,%20H.%20&amp;%20Li,%20P.%20Lipid%20Metabolism%20and%20Neutrophil%20Function.%20Cell%20Immunol.377,%20104546%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Jiang, J. et al. Correlational Analysis between Neutrophil Granulocyte Levels and Osteonecrosis of the Femoral Head. <em>BMC Musculoskelet Disord.</em><strong>20</strong>(1), 393 (2019).
</cite> [<a href="https://doi.org/10.1186/s12891-019-2778-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6717348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31470845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20J.%20et%20al.%20Correlational%20Analysis%20between%20Neutrophil%20Granulocyte%20Levels%20and%20Osteonecrosis%20of%20the%20Femoral%20Head.%20BMC%20Musculoskelet%20Disord.20(1),%20393%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Zhang, Y. et al. Mirna-320a-5p Contributes to the Homeostasis of Osteogenesis and Adipogenesis in Bone Marrow Mesenchymal Stem Cell. <em>Regen. Ther.</em><strong>20</strong>, 32–40 (2022).
</cite> [<a href="https://doi.org/10.1016/j.reth.2022.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8968203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35402661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Mirna-320a-5p%20Contributes%20to%20the%20Homeostasis%20of%20Osteogenesis%20and%20Adipogenesis%20in%20Bone%20Marrow%20Mesenchymal%20Stem%20Cell.%20Regen.%20Ther.20,%2032%E2%80%9340%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Li, T. et al. Identification of Inflammation-Related Genes and Exploration of Regulatory Mechanisms in Patients with Osteonecrosis of the Femoral Head. <em>Biomed. Res. Int.</em><strong>2022</strong>, 4501186 (2022).
</cite> [<a href="https://doi.org/10.1155/2022/4501186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9526608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36193326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20T.%20et%20al.%20Identification%20of%20Inflammation-Related%20Genes%20and%20Exploration%20of%20Regulatory%20Mechanisms%20in%20Patients%20with%20Osteonecrosis%20of%20the%20Femoral%20Head.%20Biomed.%20Res.%20Int.2022,%204501186%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Jin, Y. et al. The Role of Mir-320a and Il-1β in Human Chondrocyte Degradation. <em>Bone Joint Res.</em><strong>6</strong>(4), 196–203 (2017).
</cite> [<a href="https://doi.org/10.1302/2046-3758.64.BJR-2016-0224.R1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5415906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28404547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20Y.%20et%20al.%20The%20Role%20of%20Mir-320a%20and%20Il-1%CE%B2%20in%20Human%20Chondrocyte%20Degradation.%20Bone%20Joint%20Res.6(4),%20196%E2%80%93203%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Ou, M. et al. Identification of Potential Microrna-Target Pairs Associated with Osteopetrosis by Deep Sequencing, Itraq Proteomics and Bioinformatics. <em>Eur. J. Hum. Genet.</em><strong>22</strong>(5), 625–632 (2014).
</cite> [<a href="https://doi.org/10.1038/ejhg.2013.221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3992578/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24084574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ou,%20M.%20et%20al.%20Identification%20of%20Potential%20Microrna-Target%20Pairs%20Associated%20with%20Osteopetrosis%20by%20Deep%20Sequencing,%20Itraq%20Proteomics%20and%20Bioinformatics.%20Eur.%20J.%20Hum.%20Genet.22(5),%20625%E2%80%93632%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (36.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (13.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM3_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (261.5KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM4_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 4.</a><sup> (851B, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 5.</a><sup> (10.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 6.</a><sup> (453KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 7.</a><sup> (16.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 8.</a><sup> (10.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 9.</a><sup> (12.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 10.</a><sup> (10.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 11.</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM12_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 12.</a><sup> (2.1MB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM13_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 13.</a><sup> (2.4MB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371116/bin/41598_2025_13703_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 14.</a><sup> (97.9KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>This study utilized several datasets, including mRNA transcriptome datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123568" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123568</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74089" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE74089</a>), and the microRNA (miRNA) transcriptome dataset <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89587" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE89587</a>, all obtained from the Gene Expression Omnibus (GEO) database (<a href="https://www.ncbi.nlm.nih.gov/geo)." class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo).</a>  The lipid metabolism-related genes (LMRGs) from the Reactome database (<a href="https://reactome.org/)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://reactome.org/).</a>  The scRNA-seq dataset SRP361778, sourced from the SRA database (<a href="https://www.ncbi.nlm.nih.gov/sra" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-13703-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_13703.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (8.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371116/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371116/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371116%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371116/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371116/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371116/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841396/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371116/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841396/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371116/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371116/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xjJ577O1t4HRLZB7Q4AngWDzV9KljuD3wTjifmwMpsevhza4LNcopvLgCuJ5JbSG">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
